INSERT INTO enzyme VALUES ( 'EC 1.1.1.1','Alcohol dehydrogenase','Aldehyde reductase','Zinc or iron.','Acts on primary or secondary alcohols or hemi-acetalss with verybroad specificity; however the enzyme oxidizes methanol much morepoorly than ethanol.The animal, but not the yeast, enzyme acts also on cyclic secondaryalcohols.','(1) An alcohol + NAD(+) = an aldehyde or ketone + NADH.(2) A secondary alcohol + NAD(+) = a ketone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.2','Alcohol dehydrogenase (NADP(+))','Aldehyde reductase (NADPH)','Zinc.','Some members of this group oxidize only primary alcohols; others actalso on secondary alcohols.May be identical with EC 1.1.1.19, EC 1.1.1.33 and EC 1.1.1.55.Re-specific with respect to NADPH.','An alcohol + NADP(+) = an aldehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.3','Homoserine dehydrogenase','NULL','NULL','The enzyme from Saccharomyces cerevisiae acts most rapidly withNAD(+); the Neurospora enzyme with NADP(+).The enzyme from Escherichia coli is a multifunctional protein, whichalso catalyzes the reaction of EC 2.7.2.4.','L-homoserine + NAD(P)(+) = L-aspartate 4-semialdehyde + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.4','(R,R)-butanediol dehydrogenase','(R)-2,3-butanediol dehydrogenase','NULL','Also converts diacetyl into acetoin with NADH as reductant.','(R,R)-butane-2,3-diol + NAD(+) = (R)-acetoin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.6','Glycerol dehydrogenase','NAD-linked glycerol dehydrogenase','NULL','Also acts on 1,2-propanediol.','Glycerol + NAD(+) = glycerone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.7','Propanediol-phosphate dehydrogenase','NULL','NULL','NULL','Propane-1,2-diol 1-phosphate + NAD(+) = hydroxyacetone phosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.8','Glycerol-3-phosphate dehydrogenase (NAD(+))','Alpha-glycerol phosphate dehydrogenase (NAD)','NULL','Also acts on propane-1,2-diol phosphate and glycerone sulfate (butwith a much lower affinity).','sn-glycerol 3-phosphate + NAD(+) = glycerone phosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.9','D-xylulose reductase','Xylitol dehydrogenase','NULL','Also acts as an L-erythrulose reductase.','Xylitol + NAD(+) = D-xylulose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.10','L-xylulose reductase','NULL','NULL','NULL','Xylitol + NADP(+) = L-xylulose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.11','D-arabinitol 4-dehydrogenase','NULL','NULL','NULL','D-arabinitol + NAD(+) = D-xylulose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.12','L-arabinitol 4-dehydrogenase','NULL','NULL','NULL','L-arabinitol + NAD(+) = L-xylulose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.13','L-arabinitol 2-dehydrogenase','L-arabinitol 2-dehydrogenase (ribulose forming)','NULL','NULL','L-arabinitol + NAD(+) = L-ribulose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.14','L-iditol 2-dehydrogenase','Glucitol dehydrogenase','NULL','This enzyme is widely distributed and has been described in archaea,bacteria, yeast, plants and animals.It acts on a number of sugar alcohols, including (but not limited to)L-iditol, D-glucitol, D-xylitol, and D-galactitol.Enzymes from different organisms or tissues display differentsubstrate specificity.The enzyme is specific to NAD(+) and can not use NADP(+).','L-iditol + NAD(+) = L-sorbose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.15','D-iditol 2-dehydrogenase','NULL','NULL','Also converts xylitol into L-xylulose and L-glucitol into L-fructose.','D-iditol + NAD(+) = D-sorbose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.16','Galactitol 2-dehydrogenase','NULL','NULL','Also converts other alditols containing an L-threo-configurationadjacent to a primary alcohol group into the corresponding sugars.','Galactitol + NAD(+) = D-tagatose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.17','Mannitol-1-phosphate 5-dehydrogenase','NULL','NULL','NULL','D-mannitol 1-phosphate + NAD(+) = D-fructose 6-phosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.18','Inositol 2-dehydrogenase','Myo-inositol 2-dehydrogenase','NULL','NULL','Myo-inositol + NAD(+) = 2,4,6/3,5-pentahydroxycyclohexanone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.19','Glucuronate reductase','Aldehyde reductase','NULL','Also reduces D-galacturonate.May be identical with EC 1.1.1.2.','L-gulonate + NADP(+) = D-glucuronate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.20','Glucuronolactone reductase','NULL','NULL','NULL','L-gulono-1,4-lactone + NADP(+) = D-glucurono-3,6-lactone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.21','Aldehyde reductase','Aldose reductase','NULL','Wide specificity.Formerly EC 1.1.1.139.','Alditol + NAD(P)(+) = aldose + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.22','UDP-glucose 6-dehydrogenase','NULL','NULL','Also acts on UDP-2-deoxyglucose.','UDP-glucose + 2 NAD(+) + H(2)O = UDP-glucuronate + 2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.23','Histidinol dehydrogenase','NULL','NULL','Also oxidizes L-histidinal.The Neurospora enzyme also catalyzes the reactions of EC 3.5.4.19 andEC 3.6.1.31.','L-histidinol + H(2)O + 2 NAD(+) = L-histidine + 2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.24','Quinate dehydrogenase','Quinate 5-dehydrogenase','NULL','The enzyme is specific for quinate as substrate; phenylpyruvate,phenylalanine, cinnamate and shikimate will not act as substrates.NAD(+) cannot be replaced by NADP(+).','Quinate + NAD(+) = 3-dehydroquinate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.25','Shikimate dehydrogenase','5-dehydroshikimate reductase','NULL','NAD(+) cannot replace NADP(+).In higher organisms, this enzyme forms part of a multienzyme complexwith EC 4.2.1.10.','Shikimate + NADP(+) = 3-dehydroshikimate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.26','Glyoxylate reductase','Glycolate reductase','NULL','Reduces glyoxylate to glycolate or hydroxypyruvate to D-glycerate.','Glycolate + NAD(+) = glyoxylate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.27','L-lactate dehydrogenase','L-lactic acid dehydrogenase','NULL','Also oxidizes other (S)-2-hydroxymonocarboxylic acids.NADP(+) acts, more slowly, with the animal, but not the bacterial,enzyme.','(S)-lactate + NAD(+) = pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.28','D-lactate dehydrogenase','D-lactic acid dehydrogenase','NULL','NULL','(R)-lactate + NAD(+) = pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.29','Glycerate dehydrogenase','Hydroxypyruvate dehydrogenase','NULL','NULL','D-glycerate + NAD(+) = hydroxypyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.30','3-hydroxybutyrate dehydrogenase','D-beta-hydroxybutyrate dehydrogenase','NULL','Also oxidizes other 3-hydroxymonocarboxylic acids.','(R)-3-hydroxybutanoate + NAD(+) = acetoacetate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.31','3-hydroxyisobutyrate dehydrogenase','NULL','NULL','NULL','3-hydroxy-2-methylpropanoate + NAD(+) = 2-methyl-3-oxopropanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.32','Mevaldate reductase','NULL','NULL','NULL','(R)-mevalonate + NAD(+) = mevaldate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.33','Mevaldate reductase (NADPH)','NULL','NULL','May be identical with EC 1.1.1.2.','(R)-mevalonate + NADP(+) = mevaldate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.34','Hydroxymethylglutaryl-CoA reductase (NADPH)','3-hydroxy-3-methylglutaryl-coenzyme A reductase','NULL','The enzyme is inactivated by EC 2.7.11.31 and reactivated byEC 3.1.3.47.','(R)-mevalonate + CoA + 2 NADP(+) = (S)-3-hydroxy-3-methylglutaryl-CoA +2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.35','3-hydroxyacyl-CoA dehydrogenase','Beta-hydroxyacyl dehydrogenase','NULL','Also oxidizes S-3-hydroxyacyl-N-acylthioethanolamine and S-3-hydroxyacylhydrolipoate.Some enzymes act, more slowly, with NADP(+).Broad specificity to acyl chain-length (cf. EC 1.1.1.211).','(S)-3-hydroxyacyl-CoA + NAD(+) = 3-oxoacyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.36','Acetoacetyl-CoA reductase','NULL','NULL','NULL','(R)-3-hydroxyacyl-CoA + NADP(+) = 3-oxoacyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.37','Malate dehydrogenase','Malic dehydrogenase','NULL','Also oxidizes some other 2-hydroxydicarboxylic acids.','(S)-malate + NAD(+) = oxaloacetate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.38','Malate dehydrogenase (oxaloacetate-decarboxylating)','Malic enzyme','NULL','Unlike EC 1.1.1.39, this enzyme can also decarboxylate oxaloacetate,cf. EC 1.1.1.40.','(1) (S)-malate + NAD(+) = pyruvate + CO(2) + NADH.(2) Oxaloacetate = pyruvate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.39','Malate dehydrogenase (decarboxylating)','Malic enzyme','NULL','Does not decarboxylate added oxaloacetate.','(S)-malate + NAD(+) = pyruvate + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.40','Malate dehydrogenase (oxaloacetate-decarboxylating) (NADP(+))','Malic enzyme','NULL','The enzyme catalyzes the oxidative decarboxylation of (S)-malate inthe presence of NADP(+) and divalent metal ions, and decarboxylationof oxaloacetate, cf. EC 1.1.1.38 and EC 1.1.1.39.','(1) (S)-malate + NADP(+) = pyruvate + CO(2) + NADPH.(2) Oxaloacetate = pyruvate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.41','Isocitrate dehydrogenase (NAD(+))','Beta-ketoglutaric-isocitric carboxylase','Manganese or magnesium.','Unlike EC 1.1.1.42, oxalosuccinate cannot be used as a substrate.In eukaryotes, isocitrate dehydrogenase exists in two forms: anNAD(+)-linked enzyme found only in mitochondria and displayingallosteric properties, and a non-allosteric, NADP(+)-linked enzymethat is found in both mitochondria and cytoplasm.The enzyme from some species can also use NADP(+) but much moreslowly.','Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.42','Isocitrate dehydrogenase (NADP(+))','Dual-cofactor-specific isocitrate dehydrogenase','Manganese or magnesium.','Unlike EC 1.1.1.41, oxalosuccinate can be used as a substrate.In eukaryotes, isocitrate dehydrogenase exists in two forms: anNAD(+)-linked enzyme found only in mitochondria and displayingallosteric properties, and a non-allosteric, NADP(+)-linked enzymethat is found in both mitochondria and cytoplasm.The enzyme from some species can also use NAD(+) but much moreslowly.','Isocitrate + NADP(+) = 2-oxoglutarate + CO(2) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.43','Phosphogluconate 2-dehydrogenase','6-phosphogluconate 2-dehydrogenase','NULL','NULL','6-phospho-D-gluconate + NAD(P)(+) = 6-phospho-2-dehydro-D-gluconate +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.44','Phosphogluconate dehydrogenase (NADP(+)-dependent, decarboxylating)','6-phosphogluconic carboxylase','NULL','The enzyme participates in the oxidative branch of the pentosephosphate pathway, whose main purpose is to produce NADPH and pentosefor biosynthetic reactions.Highly specific for NADP(+).Cf. EC 1.1.1.343.','6-phospho-D-gluconate + NADP(+) = D-ribulose 5-phosphate + CO(2) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.45','L-gulonate 3-dehydrogenase','L-3-aldonate dehydrogenase','NULL','Also oxidizes other L-3-hydroxyacids.','L-gulonate + NAD(+) = 3-dehydro-L-gulonate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.46','L-arabinose 1-dehydrogenase','NULL','NULL','NULL','L-arabinose + NAD(+) = L-arabinono-1,4-lactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.47','Glucose 1-dehydrogenase (NAD(P)(+))','NULL','NULL','This enzyme has similar activity with either NAD(+) or NADP(+).Cf. EC 1.1.1.118 and EC 1.1.1.119.','Beta-D-glucose + NAD(P)(+) = D-glucono-1,5-lactone + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.48','D-galactose 1-dehydrogenase','NULL','NULL','This enzyme is part of the De Ley-Doudoroff pathway, which is used bysome bacteria during growth on D-galactose.','D-galactose + NAD(+) = D-galactono-1,4-lactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.49','Glucose-6-phosphate dehydrogenase (NADP(+))','G6PD','NULL','The enzyme catalyzes a step of the pentose phosphate pathway.The enzyme is specific for NADP(+), cf. EC 1.1.1.363.','D-glucose 6-phosphate + NADP(+) = 6-phospho-D-glucono-1,5-lactone +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.50','3-alpha-hydroxysteroid 3-dehydrogenase (Si-specific)','3-alpha-HSD','NULL','The enzyme acts on androsterone and other 3-alpha-hydroxysteroids andon 9-, 11- and 15-hydroxyprostaglandin.Si-specific with respect to NAD(+) or NADP(+) (cf. EC 1.1.1.213).','A 3-alpha-hydroxysteroid + NAD(P)(+) = a 3-oxosteroid + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.51','3(or 17)-beta-hydroxysteroid dehydrogenase','NULL','NULL','Also acts on other 3-beta- or 17-beta-hydroxysteroids (cf.EC 1.1.1.209).','Testosterone + NAD(P)(+) = androst-4-ene-3,17-dione + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.52','3-alpha-hydroxycholanate dehydrogenase','NULL','NULL','Also acts on other 3-alpha-hydroxysteroids with an acidic side-chain.','Lithocholate + NAD(+) = 3-oxo-5-beta-cholan-24-oate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.53','3-alpha-(or 20-beta)-hydroxysteroid dehydrogenase','(R)-20-hydroxysteroid dehydrogenase','NULL','The 3-alpha-hydroxy group or 20-beta-hydroxy group of pregnane andandrostane steroids can act as donor.','Androstan-3-alpha,17-beta-diol + NAD(+) = 17-beta-hydroxyandrostan-3-one+ NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.54','Allyl-alcohol dehydrogenase','NULL','NULL','Also acts on saturated primary alcohols.','Allyl alcohol + NADP(+) = acrolein + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.55','Lactaldehyde reductase (NADPH)','NULL','NULL','May be identical with EC 1.1.1.2.','Propane-1,2-diol + NADP(+) = L-lactaldehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.56','Ribitol 2-dehydrogenase','NULL','NULL','NULL','Ribitol + NAD(+) = D-ribulose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.57','Fructuronate reductase','D-mannonate dehydrogenase','NULL','Also reduces D-tagaturonate.','D-mannonate + NAD(+) = D-fructuronate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.58','Tagaturonate reductase','Altronate dehydrogenase','NULL','NULL','D-altronate + NAD(+) = D-tagaturonate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.59','3-hydroxypropionate dehydrogenase','NULL','NULL','NULL','3-hydroxypropanoate + NAD(+) = 3-oxopropanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.60','2-hydroxy-3-oxopropionate reductase','Tartronate semialdehyde reductase','NULL','NULL','D-glycerate + NAD(P)(+) = 2-hydroxy-3-oxopropanoate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.61','4-hydroxybutyrate dehydrogenase','NULL','NULL','NULL','4-hydroxybutanoate + NAD(+) = succinate semialdehyde + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.62','17-beta-estradiol 17-dehydrogenase','17-beta-estradiol dehydrogenase','NULL','The enzyme oxidizes or reduces the hydroxy/keto group on C(17) ofestrogens and androgens in mammals and regulates the biologicalpotency of these steroids.The mammalian enzyme is bifunctional and also catalyzes EC 1.1.1.270.The enzyme also acts on (S)-20-hydroxypregn-4-en-3-one and relatedcompounds, oxidizing the (S)-20-group, but unlike EC 1.1.1.149, it isSi-specific with respect to NAD(P)(+).','17-beta-estradiol + NAD(P)(+) = estrone + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.63','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.64','Testosterone 17-beta-dehydrogenase (NADP(+))','17-ketoreductase','NULL','Also oxidizes 3-hydroxyhexobarbital to 3-oxohexobarbital.','Testosterone + NADP(+) = androst-4-ene-3,17-dione + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.65','Pyridoxine 4-dehydrogenase','PL reductase','NULL','Also oxidizes pyridoxine phosphate.','Pyridoxine + NADP(+) = pyridoxal + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.66','Omega-hydroxydecanoate dehydrogenase','NULL','NULL','Also acts, more slowly, on 9-hydroxynonanoate and11-hydroxyundecanoate.','10-hydroxydecanoate + NAD(+) = 10-oxodecanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.67','Mannitol 2-dehydrogenase','Mannitol dehydrogenase','NULL','NULL','D-mannitol + NAD(+) = D-fructose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.68','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.69','Gluconate 5-dehydrogenase','5-keto-D-gluconate 5-reductase','NULL','NULL','D-gluconate + NAD(P)(+) = 5-dehydro-D-gluconate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.70','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.71','Alcohol dehydrogenase (NAD(P)(+))','Retinal reductase','NULL','Reduces aliphatic aldehydes of chain length from C2 to C14, withgreatest activity on C4, C6 and C8 aldehydes.Also reduces retinal to retinol.','An alcohol + NAD(P)(+) = an aldehyde + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.72','Glycerol dehydrogenase (NADP(+))','NULL','NULL','NULL','Glycerol + NADP(+) = D-glyceraldehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.73','Octanol dehydrogenase','NULL','NULL','Acts, less rapidly, on other long-chain alcohols.','1-octanol + NAD(+) = 1-octanal + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.74','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.75','(R)-aminopropanol dehydrogenase','NULL','Potassium.','NULL','(R)-1-aminopropan-2-ol + NAD(+) = aminoacetone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.76','(S,S)-butanediol dehydrogenase','NULL','NULL','This enzyme catalyzes the reversible reduction of (S)-acetoin to(S,S)-butane-2,3-diol.It can also catalyze the irreversible reduction of diacetyl to (S)-acetoin.','(2S,3S)-butane-2,3-diol + NAD(+) = (S)-acetoin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.77','Lactaldehyde reductase','Propanediol oxidoreductase','NULL','NULL','(1) (R)-propane-1,2-diol + NAD(+) = (R)-lactaldehyde + NADH.(2) (S)-propane-1,2-diol + NAD(+) = (S)-lactaldehyde + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.78','Methylglyoxal reductase (NADH)','D-lactaldehyde dehydrogenase','NULL','This mammalian enzyme differs from the yeast enzyme, EC 1.1.1.283,in preferring NADH over NADPH as reductant.','(R)-lactaldehyde + NAD(+) = methylglyoxal + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.79','Glyoxylate reductase (NADP(+))','NULL','NULL','Also reduces hydroxypyruvate to glycerate.Has some affinity for NAD(+).','Glycolate + NADP(+) = glyoxylate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.80','Isopropanol dehydrogenase (NADP(+))','NULL','NULL','Also acts on other short-chain secondary alcohols and, slowly,on primary alcohols.','Propan-2-ol + NADP(+) = acetone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.81','Hydroxypyruvate reductase','D-glycerate dehydrogenase','NULL','NULL','D-glycerate + NAD(P)(+) = hydroxypyruvate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.82','Malate dehydrogenase (NADP(+))','NADP-linked malate dehydrogenase','NULL','Activated by light.','(S)-malate + NADP(+) = oxaloacetate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.83','D-malate dehydrogenase (decarboxylating)','NULL','NULL','NULL','(R)-malate + NAD(+) = pyruvate + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.84','Dimethylmalate dehydrogenase','NULL','Cobalt or manganese; Potassium or NH(4)(+).','Also acts on (R)-malate.','(R)-3,3-dimethylmalate + NAD(+) = 3-methyl-2-oxobutanoate + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.85','3-isopropylmalate dehydrogenase','Beta-IPM dehydrogenase','NULL','NULL','(2R,3S)-3-isopropylmalate + NAD(+) = 4-methyl-2-oxopentanoate + CO(2) +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.86','Ketol-acid reductoisomerase','Acetohydroxy acid isomeroreductase','NULL','Formerly EC 1.1.1.89.','(1) (R)-2,3-dihydroxy-3-methylbutanoate + NADP(+) = (S)-2-hydroxy-2-methyl-3-oxobutanoate + NADPH.(2) (2R,3R)-2,3-dihydroxy-3-methylpentanoate + NADP(+) = (S)-2-hydroxy-2-ethyl-3-oxobutanoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.87','Homoisocitrate dehydrogenase','(-)-1-hydroxy-1,2,4-butanetricarboxylate:NAD(+) oxidoreductase(decarboxylating)','NULL','Forms part of the lysine biosynthesis pathway in fungi.Formerly EC 1.1.1.155.','(1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate + NAD(+) = 2-oxoadipate +CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.88','Hydroxymethylglutaryl-CoA reductase','3-hydroxy-3-methylglutaryl-coenzyme A reductase','NULL','NULL','(R)-mevalonate + CoA + 2 NAD(+) = 3-hydroxy-3-methylglutaryl-CoA +2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.89','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.90','Aryl-alcohol dehydrogenase','Benzyl alcohol dehydrogenase','NULL','A group of enzymes with broad specificity toward primary alcoholswith an aromatic or cyclohex-1-ene ring, but with low or no activitytoward short-chain aliphatic alcohols.','An aromatic alcohol + NAD(+) = an aromatic aldehyde + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.91','Aryl-alcohol dehydrogenase (NADP(+))','NULL','NULL','Also acts on some aliphatic aldehydes, but cinnamaldehyde was thebest substrate found.','An aromatic alcohol + NADP(+) = an aromatic aldehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.92','Oxaloglycolate reductase (decarboxylating)','NULL','NULL','Also reduces hydroxypyruvate to D-glycerate and glyoxylate toglycolate.','D-glycerate + NAD(P)(+) + CO(2) = 2-hydroxy-3-oxosuccinate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.93','Tartrate dehydrogenase','Mesotartrate dehydrogenase','Manganese; Monovalent cation.','NULL','(1) Tartrate + NAD(+) = oxaloglycolate + NADH.(2) Meso-tartrate + NAD(+) = oxaloglycolate + NADH.(3) (R,R)-tartrate + NAD(+) = oxaloglycolate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.94','Glycerol-3-phosphate dehydrogenase (NAD(P)(+))','Glycerol 3-phosphate dehydrogenase (NAD(P))','NULL','The enzyme from Escherichia coli shows specificity for the B side ofNADPH.','sn-glycerol 3-phosphate + NAD(P)(+) = glycerone phosphate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.95','Phosphoglycerate dehydrogenase','3-phosphoglycerate dehydrogenase','NULL','Catalyzes the first committed step in the phosphoserine pathway ofserine biosynthesis in Escherichia coli.Reaction (1) occurs predominantly in the reverse direction and isinhibited by serine and glycine.The enzyme is unusual in that it also acts as a D- and L-2-hydroxyglutarate dehydrogenase (with the D-form being the bettersubstrate) and as a 2-oxoglutarate reductase.','(1) 3-phospho-D-glycerate + NAD(+) = 3-phosphonooxypyruvate + NADH.(2) 2-hydroxyglutarate + NAD(+) = 2-oxoglutarate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.96','Diiodophenylpyruvate reductase','NULL','NULL','Substrates contain an aromatic ring with a pyruvate side chain.The most active substrates are halogenated derivatives.Compounds with hydroxy or amino groups in the 3 or 5 position areinactive.','3-(3,5-diiodo-4-hydroxyphenyl)lactate + NAD(+) = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.97','3-hydroxybenzyl-alcohol dehydrogenase','m-hydroxybenzyl alcohol dehydrogenase','NULL','NULL','3-hydroxybenzyl alcohol + NADP(+) = 3-hydroxybenzaldehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.98','(R)-2-hydroxy-fatty-acid dehydrogenase','D-2-hydroxy fatty acid dehydrogenase','NULL','NULL','(R)-2-hydroxystearate + NAD(+) = 2-oxostearate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.99','(S)-2-hydroxy-fatty-acid dehydrogenase','L-2-hydroxy fatty acid dehydrogenase','NULL','NULL','(S)-2-hydroxystearate + NAD(+) = 2-oxostearate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.100','3-oxoacyl-[acyl-carrier-protein] reductase','NULL','NULL','Exhibits a marked preference for [acyl-carrier-protein] derivativesover CoA derivatives as substrates.','(3R)-3-hydroxyacyl-[acyl-carrier-protein] + NADP(+) = 3-oxoacyl-[acyl-carrier-protein] + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.101','Acylglycerone-phosphate reductase','Palmitoyldihydroxyacetone-phosphate reductase','NULL','Also acts on alkylglycerone 3-phosphate and alkylglycerol3-phosphate.','1-palmitoylglycerol 3-phosphate + NADP(+) = palmitoylglycerone phosphate+ NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.102','3-dehydrosphinganine reductase','3-ketosphinganine reductase','NULL','NULL','Sphinganine + NADP(+) = 3-dehydrosphinganine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.103','L-threonine 3-dehydrogenase','TDH','NULL','Acts in concert with EC 2.3.1.29 in the degradation of threonine toglycine.This threonine-degradation pathway is common to prokaryotic andeukaryotic cells and the two enzymes involved form a complex.In aqueous solution, the product L-2-amino-3-oxobutanoate canspontaneously decarboxylate to form aminoacetone.','L-threonine + NAD(+) = L-2-amino-3-oxobutanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.104','4-oxoproline reductase','Hydroxyproline oxidase','NULL','NULL','4-hydroxy-L-proline + NAD(+) = 4-oxoproline + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.105','All-trans-retinol dehydrogenase (NAD(+))','NULL','NULL','The enzyme recognizes all-trans-retinol and all-trans-retinal assubstrates and exhibits a strong preference for NAD(+)/NADH ascofactors.Recognizes the substrate both in free form and when bound tocellular-retinol-binding-protein (CRBP1), but has higher affinity forthe bound form.No activity with 11-cis-retinol or 11-cis-retinal (cf. EC 1.1.1.315).Also active with 3-alpha-hydroxysteroids.','All-trans-retinol-[cellular-retinol-binding-protein] + NAD(+) = all-trans-retinal-[cellular-retinol-binding-protein] + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.106','Pantoate 4-dehydrogenase','Panthothenase','NULL','NULL','(R)-pantoate + NAD(+) = (R)-4-dehydropantoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.107','Pyridoxal 4-dehydrogenase','NULL','NULL','The enzyme acts on the hemiacetal form of the substrate.','Pyridoxal + NAD(+) = 4-pyridoxolactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.108','Carnitine 3-dehydrogenase','NULL','NULL','NULL','Carnitine + NAD(+) = 3-dehydrocarnitine + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.109','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.110','Indolelactate dehydrogenase','Indole-3-lactate dehydrogenase','NULL','NULL','(Indol-3-yl)lactate + NAD(+) = (indol-3-yl)pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.111','3-(imidazol-5-yl)lactate dehydrogenase','NULL','NULL','NULL','(S)-3-(imidazol-5-yl)lactate + NAD(P)(+) = 3-(imidazol-5-yl)pyruvate +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.112','Indanol dehydrogenase','NULL','NULL','3(20)-alpha-hydroxysteroids are also oxidized, more slowly.','Indan-1-ol + NAD(P)(+) = indanone + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.113','L-xylose 1-dehydrogenase','NULL','NULL','Also oxidizes D-arabinose and D-lyxose.','L-xylose + NADP(+) = L-xylono-1,4-lactone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.114','Apiose 1-reductase','D-apiitol reductase','NULL','NULL','D-apiitol + NAD(+) = D-apiose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.115','Ribose 1-dehydrogenase (NADP(+))','NADP-pentose-dehydrogenase','NULL','Also acts, more slowly, on D-xylose and other pentoses.','D-ribose + NADP(+) + H(2)O = D-ribonate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.116','D-arabinose 1-dehydrogenase (NAD(+))','NULL','NULL','NULL','D-arabinose + NAD(+) = D-arabinono-1,4-lactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.117','D-arabinose 1-dehydrogenase (NAD(P)(+))','NULL','NULL','Also acts on L-galactose, 6-deoxy- and 3,6-dideoxy-L-galactose.','D-arabinose + NAD(P)(+) = D-arabinono-1,4-lactone + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.118','Glucose 1-dehydrogenase (NAD(+))','NULL','NULL','NULL','D-glucose + NAD(+) = D-glucono-1,5-lactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.119','Glucose 1-dehydrogenase (NADP(+))','NULL','NULL','Also oxidizes D-mannose, 2-deoxy-D-glucose and 2-amino-2-deoxy-D-mannose.','D-glucose + NADP(+) = D-glucono-1,5-lactone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.120','Galactose 1-dehydrogenase (NADP(+))','NULL','NULL','Also acts on L-arabinose, 6-deoxy- and 2-deoxy-D-galactose.','D-galactose + NADP(+) = D-galactono-1,5-lactone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.121','Aldose 1-dehydrogenase (NAD(+))','NULL','NULL','Acts on D-glucose, 2-deoxy- and 6-deoxy-D-glucose, D-galactose,6-deoxy-D-galactose, 2-deoxy-L-arabinose and D-xylose.','D-aldose + NAD(+) = D-aldonolactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.122','D-threo-aldose 1-dehydrogenase','(2S,3R)-aldose dehydrogenase','NULL','Acts on L-fucose, D-arabinose and L-xylose.The animal enzyme was also shown to act on L-arabinose, and theenzyme from Pseudomonas caryophylli on L-glucose.','A D-threo-aldose + NAD(+) = a D-threo-aldono-1,5-lactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.123','Sorbose 5-dehydrogenase (NADP(+))','5-ketofructose reductase','NULL','NULL','L-sorbose + NADP(+) = 5-dehydro-D-fructose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.124','Fructose 5-dehydrogenase (NADP(+))','5-ketofructose reductase (NADP(+))','NULL','NULL','D-fructose + NADP(+) = 5-dehydro-D-fructose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.125','2-deoxy-D-gluconate 3-dehydrogenase','2-keto-3-deoxygluconate oxidoreductase','NULL','NULL','2-deoxy-D-gluconate + NAD(+) = 3-dehydro-2-deoxy-D-gluconate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.126','2-dehydro-3-deoxy-D-gluconate 6-dehydrogenase','2-keto-3-deoxy-D-gluconate dehydrogenase','NULL','NULL','2-dehydro-3-deoxy-D-gluconate + NADP(+) = (4S,5S)-4,5-dihydroxy-2,6-dioxohexanoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.127','2-dehydro-3-deoxy-D-gluconate 5-dehydrogenase','2-keto-3-deoxygluconate dehydrogenase','NULL','The enzyme from Pseudomonas acts equally well on NAD(+) or NADP(+),while those from Erwinia chrysanthemi and Escherichia coli are morespecific for NAD(+).','2-dehydro-3-deoxy-D-gluconate + NAD(+) = (4S)-4,6-dihydroxy-2,5-dioxohexanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.128','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.129','L-threonate 3-dehydrogenase','L-threonic acid dehydrogenase','NULL','NULL','L-threonate + NAD(+) = 3-dehydro-L-threonate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.130','3-dehydro-L-gulonate 2-dehydrogenase','2,3-diketo-L-gulonate reductase','NULL','NULL','3-dehydro-L-gulonate + NAD(P)(+) = (4R,5S)-4,5,6-trihydroxy-2,3-dioxohexanoate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.131','Mannuronate reductase','Mannonate dehydrogenase','NULL','Formerly EC 1.1.1.180 and EC 1.2.1.34.','D-mannonate + NAD(P)(+) = D-mannuronate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.132','GDP-mannose 6-dehydrogenase','NULL','NULL','Also uses the corresponding deoxynucleoside diphosphate derivative asa substrate.','GDP-D-mannose + 2 NAD(+) + H(2)O = GDP-D-mannuronate + 2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.133','dTDP-4-dehydrorhamnose reductase','dTDP-4-keto-L-rhamnose reductase','NULL','In the reverse direction, reduction on the 4-position of the hexosemoiety takes place only while the substrate is bound to anotherenzyme that catalyzes epimerization at C-3 and C-5; the complex hasbeen referred to as dTDP-L-rhamnose synthase.','dTDP-beta-L-rhamnose + NADP(+) = dTDP-4-dehydro-beta-L-rhamnose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.134','dTDP-6-deoxy-L-talose 4-dehydrogenase','NULL','NULL','Oxidation on the 4-position of the hexose moiety takes place onlywhile the substrate is bound to another enzyme that catalyzesepimerization at C-3 and C-5.','dTDP-6-deoxy-beta-L-talose + NADP(+) = dTDP-4-dehydro-beta-L-rhamnose +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.135','GDP-6-deoxy-D-talose 4-dehydrogenase','NULL','NULL','NULL','GDP-6-deoxy-alpha-D-talose + NAD(P)(+) = GDP-4-dehydro-alpha-D-rhamnose +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.136','UDP-N-acetylglucosamine 6-dehydrogenase','UDP-2-acetamido-2-deoxy-D-glucose:NAD oxidoreductase','NULL','This enzyme participates in the biosynthetic pathway for UDP-alpha-D-ManNAc(3)NAcA (UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-mannuronicacid), an important precursor of B-band lipopolysaccharide.','UDP-N-acetyl-alpha-D-glucosamine + 2 NAD(+) + H(2)O = UDP-2-acetamido-2-deoxy-alpha-D-glucuronate + 2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.137','Ribitol-5-phosphate 2-dehydrogenase','NULL','NULL','NULL','D-ribitol 5-phosphate + NAD(P)(+) = D-ribulose 5-phosphate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.138','Mannitol 2-dehydrogenase (NADP(+))','NADP-dependent mannitol dehydrogenase','NULL','NULL','D-mannitol + NADP(+) = D-fructose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.139','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.140','Sorbitol-6-phosphate 2-dehydrogenase','Glucitol-6-phosphate dehydrogenase','NULL','NULL','D-sorbitol 6-phosphate + NAD(+) = D-fructose 6-phosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.141','15-hydroxyprostaglandin dehydrogenase (NAD(+))','15-hydroxyprostaglandin dehydrogenase','NULL','Acts on prostaglandins E(2), F(2-alpha) and B(1), but not onprostaglandin D(2) (cf. EC 1.1.1.196 and EC 1.1.1.197).','(5Z,13E,15S)-11-alpha,15-dihydroxy-9-oxoprost-5,13-dienoate + NAD(+) =(5Z,13E)-11-alpha-hydroxy-9,15-dioxoprost-5,13-dienoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.142','D-pinitol dehydrogenase','NULL','NULL','NULL','1D-3-O-methyl-chiro-inositol + NADP(+) = 2D-5-O-methyl-2,3,5/4,6-pentahydroxycyclohexanone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.143','Sequoyitol dehydrogenase','NULL','NULL','NULL','5-O-methyl-myo-inositol + NAD(+) = 2D-5-O-methyl-2,3,5/4,6-pentahydroxycyclohexanone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.144','Perillyl-alcohol dehydrogenase','NULL','NULL','Oxidizes a number of primary alcohols with the alcohol group allylicto an endocyclic double bond and a 6-membered ring, either aromaticor hydroaromatic.','Perillyl alcohol + NAD(+) = perillyl aldehyde + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.145','3-beta-hydroxy-Delta(5)-steroid dehydrogenase','3-beta-hydroxy-5-ene steroid dehydrogenase','NULL','Acts on 3-beta-hydroxyandrost-5-en-17-one to form androst-4-ene-3,17-dione and on 3-beta-hydroxypregn-5-en-20-one to form progesterone.','A 3-beta-hydroxy-Delta(5)-steroid + NAD(+) = a 3-oxo-Delta(5)-steroid +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.146','11-beta-hydroxysteroid dehydrogenase','Corticosteroid 11-beta-dehydrogenase','NULL','NULL','An 11-beta-hydroxysteroid + NADP(+) = an 11-oxosteroid + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.147','16-alpha-hydroxysteroid dehydrogenase','NULL','NULL','NULL','A 16-alpha-hydroxysteroid + NAD(P)(+) = a 16-oxosteroid + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.148','Estradiol 17-alpha-dehydrogenase','NULL','NULL','NULL','Estradiol-17-alpha + NAD(P)(+) = estrone + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.149','20-alpha-hydroxysteroid dehydrogenase','NULL','NULL','Re-specific with respect to NAD(P)(+) (cf. EC 1.1.1.62).','17-alpha,20-alpha-dihydroxypregn-4-en-3-one + NAD(P)(+) = 17-alpha-hydroxyprogesterone + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.150','21-hydroxysteroid dehydrogenase (NAD(+))','NULL','NULL','Acts on a number of 21-hydroxycorticosteroids.','Pregnan-21-ol + NAD(+) = pregnan-21-al + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.151','21-hydroxysteroid dehydrogenase (NADP(+))','21-hydroxy steroid (nicotinamide adenine dinucleotide phosphate)dehydrogenase','NULL','Acts on a number of 21-hydroxycorticosteroids.','Pregnan-21-ol + NADP(+) = pregnan-21-al + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.152','3-alpha-hydroxy-5-beta-androstane-17-one 3-alpha-dehydrogenase','Etiocholanolone 3-alpha-dehydrogenase','NULL','NULL','3-alpha-hydroxy-5-beta-androstane-17-one + NAD(+) = 5-beta-androstane-3,17-dione + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.153','Sepiapterin reductase (L-erythro-7,8-dihydrobiopterin forming)','NULL','NULL','This enzyme catalyzes the final step in the de novo synthesis oftetrahydrobiopterin from GTP.The enzyme, which is found in higher animals and some fungi andbacteria, produces the erythro form of tetrahydrobiopterin.Cf. EC 1.1.1.325.','(1) L-erythro-7,8-dihydrobiopterin + NADP(+) = sepiapterin + NADPH.(2) L-erythro-tetrahydrobiopterin + 2 NADP(+) = 6-pyruvoyl-5,6,7,8-tetrahydropterin + 2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.154','Ureidoglycolate dehydrogenase','NULL','NULL','NULL','(S)-ureidoglycolate + NAD(P)(+) = oxalureate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.155','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.156','Glycerol 2-dehydrogenase (NADP(+))','Dihydroxyacetone reductase','NULL','NULL','Glycerol + NADP(+) = glycerone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.157','3-hydroxybutyryl-CoA dehydrogenase','Beta-hydroxybutyryl-CoA dehydrogenase','NULL','NULL','(S)-3-hydroxybutanoyl-CoA + NADP(+) = 3-acetoacetyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.158','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.159','7-alpha-hydroxysteroid dehydrogenase','NULL','NULL','Catalyzes the oxidation of the 7-alpha-hydroxy group of bile acidsand alcohols both in their free and conjugated forms.The Bacteroides fragilis and Clostridium enzymes can also utilizeNADP(+).','3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholanate + NAD(+) =3-alpha,12-alpha-dihydroxy-7-oxo-5-beta-cholanate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.160','Dihydrobunolol dehydrogenase','Bunolol reductase','NULL','Also acts, more slowly, with NAD(+).','(+-)-5-((tert-butylamino)-2-hydroxypropoxy)-1,2,3,4-tetrahydro-1-naphthol + NADP(+) = (+-)-5-((tert-butylamino)-2-hydroxypropoxy)-3,4-dihydro-1(2H)-naphthalenone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.161','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.162','Erythrulose reductase','D-erythrulose reductase','NULL','NAD(+) is also utilized, but more slowly.','D-threitol + NADP(+) = D-erythrulose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.163','Cyclopentanol dehydrogenase','NULL','NULL','4-methylcyclohexanol and cyclohexanol can also act as substrates.','Cyclopentanol + NAD(+) = cyclopentanone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.164','Hexadecanol dehydrogenase','NULL','NULL','The liver enzyme acts on long-chain alcohols from C(8) to C(16).The Euglena enzyme also oxidizes the corresponding aldehydes to fattyacids.','Hexadecanol + NAD(+) = hexadecanal + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.165','2-alkyn-1-ol dehydrogenase','NULL','NULL','Acts on a variety of 2-alkyn-1-ols, and also on 1,4-butanediol.NADP(+) also acts as acceptor, but more slowly.','2-butyne-1,4-diol + NAD(+) = 4-hydroxy-2-butynal + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.166','Hydroxycyclohexanecarboxylate dehydrogenase','NULL','NULL','Acts on hydroxycyclohexanecarboxylates that have an equatorialcarboxy group at C-1, an axial hydroxy group at C-3 and an equatorialhydroxy or carbonyl group at C-4, including (-)-quinate and (-)-shikimate.','(1S,3R,4S)-3,4-dihydroxycyclohexane-1-carboxylate + NAD(+) = (1S,4S)-4-hydroxy-3-oxocyclohexane-1-carboxylate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.167','Hydroxymalonate dehydrogenase','NULL','NULL','NULL','Hydroxymalonate + NAD(+) = oxomalonate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.168','2-dehydropantolactone reductase (Re-specific)','2-dehydropantolactone reductase (A-specific)','NULL','The enzyme from Saccharomyces cerevisiae differs from that fromEscherichia coli (EC 1.1.1.214), which is specific for the Si-face ofNADP, and in receptor requirements from EC 1.1.99.26.','(R)-pantolactone + NADP(+) = 2-dehydropantolactone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.169','2-dehydropantoate 2-reductase','Ketopantoate reductase','NULL','NULL','(R)-pantoate + NADP(+) = 2-dehydropantoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.170','3-beta-hydroxysteroid-4-alpha-carboxylate 3-dehydrogenase(decarboxylating)','3-beta-hydroxy-4-alpha-methylcholestenecarboxylate 3-dehydrogenase(decarboxylating)','NULL','The enzyme catalyzes the decarboxylation of the C-4 carbon and thedehydrogenation of a 3-beta hydroxyl at the C-3 carbon of 3-beta-hydroxysteroid-4-alpha-carboxylates.It is involved in zymosterol and cholesterol biosynthesis.','A 3-beta-hydroxysteroid-4-alpha-carboxylate + NAD(P)(+) = a 3-oxosteroid+ CO(2) + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.171','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.172','2-oxoadipate reductase','2-ketoadipate reductase','NULL','NULL','2-hydroxyadipate + NAD(+) = 2-oxoadipate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.173','L-rhamnose 1-dehydrogenase','NULL','NULL','NULL','L-rhamnofuranose + NAD(+) = L-rhamno-1,4-lactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.174','Cyclohexane-1,2-diol dehydrogenase','NULL','NULL','Also oxidizes, more slowly, the cis isomer and2-hydroxycyclohexanone.','Trans-cyclohexane-1,2-diol + NAD(+) = 2-hydroxycyclohexan-1-one + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.175','D-xylose 1-dehydrogenase','D-xylose dehydrogenase','NULL','NULL','D-xylose + NAD(+) = D-xylonolactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.176','12-alpha-hydroxysteroid dehydrogenase','NULL','NULL','Catalyzes the oxidation of the 12-alpha-hydroxy group of bile acids,both in their free and conjugated form.Also acts on bile alcohols.','3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholanate + NADP(+) =3-alpha,7-alpha-dihydroxy-12-oxo-5-beta-cholanate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.177','Glycerol-3-phosphate 1-dehydrogenase (NADP(+))','Glycerin-3-phosphate dehydrogenase','NULL','NULL','sn-glycerol 3-phosphate + NADP(+) = D-glyceraldehyde 3-phosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.178','3-hydroxy-2-methylbutyryl-CoA dehydrogenase','2-methyl-3-hydroxybutyryl-CoA dehydrogenase','NULL','Also acts, more slowly, on (2S,3S)-2-hydroxy-3-methylpentanoyl-CoA.','(2S,3S)-3-hydroxy-2-methylbutanoyl-CoA + NAD(+) = 2-methylacetoacetyl-CoA+ NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.179','D-xylose 1-dehydrogenase (NADP(+))','D-xylose-NADP dehydrogenase','NULL','Also acts, more slowly, on L-arabinose and D-ribose.','D-xylose + NADP(+) = D-xylono-1,5-lactone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.180','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.181','Cholest-5-ene-3-beta,7-alpha-diol 3-beta-dehydrogenase','3-beta-hydroxy-Delta(5)-C(27)-steroid oxidoreductase','NULL','Highly specific for 3-beta-hydroxy-C(27)-steroids with Delta(5)-double bond.','Cholest-5-ene-3-beta,7-alpha-diol + NAD(+) = 7-alpha-hydroxycholest-4-en-3-one + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.182','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.183','Geraniol dehydrogenase (NADP(+))','NULL','NULL','Also acts, more slowly on farnesol but not on nerol.The enzyme produces a mixture known as citral, which includesgeranial and neral.It is still not known whether neral is produced directly by theenzyme, or by isomerization of geranial.','Geraniol + NADP(+) = geranial + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.184','Carbonyl reductase (NADPH)','Aldehyde reductase I','NULL','Acts on a wide range of carbonyl compounds, including quinones,aromatic aldehydes, ketoaldehydes, daunorubicin, and prostaglandinsE and F, reducing them to the corresponding alcohol.Si-specific with respect to NADPH (cf. EC 1.1.1.2).','R-CHOH-R + NADP(+) = R-CO-R + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.185','L-glycol dehydrogenase','NULL','NULL','The 2-hydroxycarbonyl compound formed can be further oxidized to avicinal dicarbonyl compound.In the reverse direction, vicinal diketones, glyceraldehyde, glyoxal,methylglyoxal, 2-oxo-hydroxyketones and 2-ketoacid esters can bereduced.','An L-glycol + NAD(P)(+) = a 2-hydroxycarbonyl compound + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.186','dTDP-galactose 6-dehydrogenase','Thymidine-diphosphate-galactose dehydrogenase','NULL','NULL','dTDP-D-galactose + 2 NADP(+) + H(2)O = dTDP-D-galacturonate + 2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.187','GDP-4-dehydro-D-rhamnose reductase','GDP-4-keto-6-deoxy-D-mannose reductase','NULL','The enzyme, which operates in the opposite direction to that shown,forms a mixture of GDP-alpha-D-rhamnose and its C-4 epimer, GDP-6-deoxy-alpha-D-talose.Cf. EC 1.1.1.281 and EC 1.1.1.135.','(1) GDP-alpha-D-rhamnose + NAD(P)(+) = GDP-4-dehydro-alpha-D-rhamnose +NAD(P)H.(2) GDP-6-deoxy-alpha-D-talose + NAD(P)(+) = GDP-4-dehydro-alpha-D-rhamnose + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.188','Prostaglandin-F synthase','PGD(2) 11-ketoreductase','NULL','Reduces prostaglandin D(2) and prostaglandin H(2) to prostaglandinF(2); prostaglandin D(2) is not an intermediate in the reduction ofprostaglandin H(2).Also catalyzes the reduction of a number of carbonyl compounds, suchas 9,10-phenanthroquinone and 4-nitroacetophenone.','(5Z,13E)-(15S)-9-alpha,11-alpha,15-trihydroxyprosta-5,13-dienoate +NADP(+) = (5Z,13E)-(15S)-9-alpha,15-dihydroxy-11-oxoprosta-5,13-dienoate+ NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.189','Prostaglandin-E(2) 9-reductase','9-keto-prostaglandin E(2) reductase','NULL','Reduces prostaglandin E(2) to prostaglandin F(2)-alpha.A number of other 9-oxo- and 15-oxo-prostaglandin derivatives canalso be reduced to the corresponding hydroxy compounds.May be identical with EC 1.1.1.197.','(5Z,13E)-(15S)-9-alpha,11-alpha,15-trihydroxyprosta-5,13-dienoate +NADP(+) = (5Z,13E)-(15S)-11-alpha,15-dihydroxy-9-oxoprosta-5,13-dienoate+ NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.190','Indole-3-acetaldehyde reductase (NADH)','Indoleacetaldehyde reductase','NULL','NULL','(Indol-3-yl)ethanol + NAD(+) = (indol-3-yl)acetaldehyde + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.191','Indole-3-acetaldehyde reductase (NADPH)','NULL','NULL','NULL','(Indol-3-yl)ethanol + NADP(+) = (indol-3-yl)acetaldehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.192','Long-chain-alcohol dehydrogenase','Fatty alcohol oxidoreductase','NULL','Hexadecanol is a good substrate.','A long-chain alcohol + 2 NAD(+) + H(2)O = a long-chain carboxylate +2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.193','5-amino-6-(5-phosphoribosylamino)uracil reductase','NULL','NULL','NULL','5-amino-6-(5-phospho-D-ribitylamino)uracil + NADP(+) = 5-amino-6-(5-phospho-D-ribosylamino)uracil + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.194','Coniferyl-alcohol dehydrogenase','NULL','NULL','Specific for coniferyl alcohol; does not act on cinnamyl alcohol,4-coumaryl alcohol or sinapyl alcohol.','Coniferyl alcohol + NADP(+) = coniferyl aldehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.195','Cinnamyl-alcohol dehydrogenase','CAD','NULL','Acts on coniferyl alcohol, sinapyl alcohol, 4-coumaryl alcohol andcinnamyl alcohol (cf. EC 1.1.1.194).','Cinnamyl alcohol + NADP(+) = cinnamaldehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.196','15-hydroxyprostaglandin-D dehydrogenase (NADP(+))','Prostaglandin-D 15-dehydrogenase (NADP(+))','NULL','Specific for prostaglandins D (cf. EC 1.1.1.141 and EC 1.1.1.197).','(5Z,13E)-(15S)-9-alpha,15-dihydroxy-11-oxoprosta-5,13-dienoate + NADP(+)= (5Z,13E)-9-alpha-hydroxy-11,15-dioxoprosta-5,13-dienoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.197','15-hydroxyprostaglandin dehydrogenase (NADP(+))','NULL','NULL','Acts on prostaglandins E(2), F(2)-alpha and B(1), but not onprostaglandin D(2) (cf. EC 1.1.1.141 and EC 1.1.1.196).May be identical with EC 1.1.1.189.','(13E)-(15S)-11-alpha,15-dihydroxy-9-oxoprost-13-enoate + NADP(+) = (13E)-11-alpha-hydroxy-9,15-dioxoprost-13-enoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.198','(+)-borneol dehydrogenase','NULL','NULL','NADP(+) can also act, more slowly.Formerly EC 1.1.1.182.','(+)-borneol + NAD(+) = (+)-camphor + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.199','(S)-usnate reductase','NULL','NULL','NULL','(6R)-2-acetyl-6-(3-acetyl-2,4,6-trihydroxy-5-methylphenyl)-3-hydroxy-6-methyl-2,4-cyclohexadien-1-one + NAD(+) = (S)-usnate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.200','Aldose-6-phosphate reductase (NADPH)','NADP-dependent D-sorbitol-6-phosphate dehydrogenase','NULL','In the reverse reaction, acts also on D-galactose 6-phosphate and,more slowly, on D-mannose 6-phosphate and 2-deoxy-D-glucose6-phosphate.','D-sorbitol 6-phosphate + NADP(+) = D-glucose 6-phosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.201','7-beta-hydroxysteroid dehydrogenase (NADP(+))','NADP-dependent 7-beta-hydroxysteroid dehydrogenase','NULL','Catalyzes the oxidation of the 7-beta-hydroxy group of bile acidssuch as ursodeoxycholate.','A 7-beta-hydroxysteroid + NADP(+) = a 7-oxosteroid + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.202','1,3-propanediol dehydrogenase','1,3-propanediol oxidoreductase','NULL','NULL','Propane-1,3-diol + NAD(+) = 3-hydroxypropanal + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.203','Uronate dehydrogenase','Uronic acid dehydrogenase','NULL','Also acts on D-glucuronate.Formerly EC 1.2.1.35.','D-galacturonate + NAD(+) + H(2)O = D-galactarate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.204','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.205','IMP dehydrogenase','IMP oxidoreductase','NULL','The enzyme acts on the hydroxy group of the hydrated derivative ofthe substrate.Formerly EC 1.2.1.14.','Inosine 5-phosphate + NAD(+) + H(2)O = xanthosine 5-phosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.206','Tropinone reductase I','TR-I','NULL','Also oxidizes other tropan-3-alpha-ols, but not the correspondingbeta-derivatives.Represents, along with EC 1.1.1.236, a branch point in tropanealkaloid metabolism.Tropine (the product of EC 1.1.1.206) is incorporated intohyoscyamine and scopolamine whereas pseudotropine (the product ofEC 1.1.1.236) is the first specific metabolite on the pathway to thecalystegines.Both enzymes are always found together in any given tropane-alkaloid-producing species, have a common substrate, tropinone, and arestrictly stereospecific.','Tropine + NADP(+) = tropinone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.207','(-)-menthol dehydrogenase','Monoterpenoid dehydrogenase','NULL','Acts on a number of other cyclohexanols and cyclohexenols.Not identical with EC 1.1.1.208.','(-)-menthol + NADP(+) = (-)-menthone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.208','(+)-neomenthol dehydrogenase','Monoterpenoid dehydrogenase','NULL','Acts on a number of other cyclohexanols and cyclohexenols.Not identical with EC 1.1.1.207.','(+)-neomenthol + NADP(+) = (-)-menthone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.209','3(or 17)-alpha-hydroxysteroid dehydrogenase','NULL','NULL','Acts on the 3-alpha-hydroxy group of androgens of the 5-alpha-androstane series; and also, more slowly, on the 17-alpha-hydroxylgroup of both androgenic and estrogenic substrates (cf. EC 1.1.1.51).','Androsterone + NAD(P)(+) = 5-alpha-androstane-3,17-dione + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.210','3-beta-(or 20-alpha)-hydroxysteroid dehydrogenase','3-beta-HSD','NULL','Also acts on 20-alpha-hydroxysteroids.','5-alpha-androstan-3-beta,17-beta-diol + NADP(+) = 17-beta-hydroxy-5-alpha-androstan-3-one + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.211','Long-chain-3-hydroxyacyl-CoA dehydrogenase','Beta-hydroxyacyl-CoA dehydrogenase','NULL','This enzyme was purified from the mitochondrial inner membrane.The enzyme has a preference for long-chain substrates, and activitywith C(16) substrates was 6- to 15-fold higher than with C(4)substrates (cf. EC 1.1.1.35).','A long-chain (S)-3-hydroxyacyl-CoA + NAD(+) = a long-chain 3-oxoacyl-CoA+ NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.212','3-oxoacyl-[acyl-carrier-protein] reductase (NADH)','NULL','NULL','Forms part of the fatty acid synthase system in plants.Can be separated from EC 1.1.1.100.','(3R)-3-hydroxyacyl-[acyl-carrier-protein] + NAD(+) = 3-oxoacyl-[acyl-carrier-protein] + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.213','3-alpha-hydroxysteroid dehydrogenase (Re-specific)','3-alpha-hydroxysteroid dehydrogenase','NULL','The enzyme acts on multiple 3-alpha-hydroxysteroids.Re-specific with respect to NAD(+) or NADP(+) (cf. EC 1.1.1.50).Enzymes whose stereo-specificity with respect to NAD(+) or NADP(+) isnot known are described by EC 1.1.1.357.','A 3-alpha-hydroxysteroid + NAD(P)(+) = a 3-oxosteroid + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.214','2-dehydropantolactone reductase (Si-specific)','2-dehydropantolactone reductase (B-specific)','NULL','The Escherichia coli enzyme differs from that from Saccharomycescerevisiae (EC 1.1.1.168), which is specific for the Re-face of NADP,and in receptor requirements from EC 1.1.99.26.','(R)-pantolactone + NADP(+) = 2-dehydropantolactone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.215','Gluconate 2-dehydrogenase','2-keto-D-gluconate reductase','NULL','Also acts on L-idonate, D-galactonate and D-xylonate.','D-gluconate + NADP(+) = 2-dehydro-D-gluconate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.216','Farnesol dehydrogenase','NULL','NULL','Also acts, more slowly, on (2Z,6E)-farnesol, geraniol, citronerol andnerol.','(2E,6E)-farnesol + NADP(+) = (2E,6E)-farnesal + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.217','Benzyl-2-methyl-hydroxybutyrate dehydrogenase','Benzyl 2-methyl-3-hydroxybutyrate dehydrogenase','NULL','Also acts on benzyl (2S,3S)-2-methyl-3-hydroxybutanoate; otherwisehighly specific.','Benzyl (2R,3S)-2-methyl-3-hydroxybutanoate + NADP(+) = benzyl 2-methyl-3-oxobutanoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.218','Morphine 6-dehydrogenase','Naloxone reductase','NULL','Also acts on some other alkaloids, including codeine, normorphine andethylmorphine, but only very slowly on 7,8-saturated derivatives suchas dihydromorphine and dihydrocodeine.In the reverse direction, also reduces naloxone to the 6-alpha-hydroxy analog.Activated by 2-mercaptoethanol.','Morphine + NAD(P)(+) = morphinone + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.219','Dihydrokaempferol 4-reductase','Dihydroflavanol 4-reductase','NULL','Also acts, in the reverse direction, on (+)-dihydroquercetin and (+)-dihydromyricetin; each dihydroflavonol is reduced to thecorresponding cis-flavan-3,4-diol.NAD(+) can act instead of NADP(+), but more slowly.Involved in the biosynthesis of anthocyanidins in plants.','Cis-3,4-leucopelargonidin + NADP(+) = (+)-dihydrokaempferol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.220','6-pyruvoyltetrahydropterin 2-reductase','6-pyruvoyl tetrahydropterin (2-oxo)reductase','NULL','Not identical with EC 1.1.1.153.','6-lactoyl-5,6,7,8-tetrahydropterin + NADP(+) = 6-pyruvoyltetrahydropterin+ NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.221','Vomifoliol dehydrogenase','Vomifoliol 4-dehydrogenase','NULL','Oxidizes vomifoliol to dehydrovomifoliol; involved in the metabolismof abscisic acid in Corynebacterium sp.','(6S,9R)-6-hydroxy-3-oxo-alpha-ionol + NAD(+) = (6R)-6-hydroxy-3-oxo-alpha-ionone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.222','(R)-4-hydroxyphenyllactate dehydrogenase','(R)-aromatic lactate dehydrogenase','NULL','Also acts, more slowly, on (R)-3-phenyllactate, (R)-3-(indol-3-yl)lactate and (R)-lactate.','(R)-3-(4-hydroxyphenyl)lactate + NAD(P)(+) = 3-(4-hydroxyphenyl)pyruvate+ NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.223','Isopiperitenol dehydrogenase','NULL','NULL','Acts on (-)-trans-isopiperitenol, (+)-trans-piperitenol and (+)-trans-pulegol.Involved in the biosynthesis of menthol and related monoterpenes inpeppermint (Mentha piperita) leaves.','(-)-trans-isopiperitenol + NAD(+) = (-)-isopiperitenone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.224','Mannose-6-phosphate 6-reductase','NADPH-dependent M6P reductase','NULL','Involved in the biosynthesis of mannitol in celery (Apium graveolens)leaves.','D-mannitol 1-phosphate + NADP(+) = D-mannose 6-phosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.225','Chlordecone reductase','CDR','NULL','Chlordecone, an organochlorine pesticide, is 1,1a,3,3a,4,5,5a,5b,6-decachlorooctahydro-1,3,4-metheno-2H-cyclobuta[cd]pentalen-2-one.','Chlordecone alcohol + NADP(+) = chlordecone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.226','4-hydroxycyclohexanecarboxylate dehydrogenase','NULL','NULL','The enzyme from Corynebacterium cyclohexanicum is highly specific forthe trans-4-hydroxy derivative.','Trans-4-hydroxycyclohexanecarboxylate + NAD(+) =4-oxocyclohexanecarboxylate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.227','(-)-borneol dehydrogenase','NULL','NULL','NADP(+) can also act, more slowly.Formerly EC 1.1.1.182.','(-)-borneol + NAD(+) = (-)-camphor + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.228','(+)-sabinol dehydrogenase','NULL','NULL','NADP(+) can also act, more slowly.Involved in the biosynthesis of (+)-3-thujone and (-)-3-isothujone.Formerly EC 1.1.1.182.','(+)-cis-sabinol + NAD(+) = (+)-sabinone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.229','Diethyl 2-methyl-3-oxosuccinate reductase','NULL','NULL','Also acts on diethyl (2S,3R)-2-methyl-3-hydroxysuccinate; and on thecorresponding dimethyl esters.','Diethyl (2R,3R)-2-methyl-3-hydroxysuccinate + NADP(+) = diethyl 2-methyl-3-oxosuccinate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.230','3-alpha-hydroxyglycyrrhetinate dehydrogenase','NULL','NULL','Highly specific to 3-alpha-hydroxy derivatives of glycyrrhetinate andits analogs.Not identical to EC 1.1.1.50.','3-alpha-hydroxyglycyrrhetinate + NADP(+) = 3-oxoglycyrrhetinate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.231','15-hydroxyprostaglandin-I dehydrogenase (NADP(+))','PG I2 dehydrogenase','NULL','Specific for prostaglandin I(2).','(5Z,13E)-(15S)-6,9-alpha-epoxy-11-alpha,15-dihydroxyprosta-5,13-dienoate+ NADP(+) = (5Z,13E)-6,9-alpha-epoxy-11-alpha-hydroxy-15-oxoprosta-5,13-dienoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.232','15-hydroxyicosatetraenoate dehydrogenase','NULL','NULL','NULL','(15S)-15-hydroxy-5,8,11-cis-13-trans-icosatetraenoate + NAD(P)(+) =15-oxo-5,8,11-cis-13-trans-icosatetraenoate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.233','N-acylmannosamine 1-dehydrogenase','N-acetyl-D-mannosamine dehydrogenase','NULL','Acts on acetyl-D-mannosamine and glycolyl-D-mannosamine.Highly specific.','N-acyl-D-mannosamine + NAD(+) = N-acyl-D-mannosaminolactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.234','Flavanone 4-reductase','NULL','NULL','Involved in the biosynthesis of 3-deoxyanthocyanidins from flavanonessuch as naringenin or eriodictyol.','(2S)-flavan-4-ol + NADP(+) = (2S)-flavanone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.235','8-oxocoformycin reductase','NULL','NULL','Si-specific with respect to NADPH.Also reduces 8-oxodeoxy-coformycin to the nucleoside antibioticdeoxycoformycin.','Coformycin + NADP(+) = 8-oxocoformycin + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.236','Tropinone reductase II','Pseudotropine forming tropinone reductase','NULL','Represents, along with EC 1.1.1.206, a branch point in tropanealkaloid metabolism.Tropine (the product of EC 1.1.1.206) is incorporated intohyoscyamine and scopolamine whereas pseudotropine (the product ofEC 1.1.1.236) is the first specific metabolite on the pathway to thecalystegines.Both enzymes are always found together in any given tropane-alkaloid-producing species, have a common substrate, tropinone, and arestrictly stereospecific.','Pseudotropine + NADP(+) = tropinone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.237','Hydroxyphenylpyruvate reductase','NULL','NULL','Also acts on 3-(3,4-dihydroxyphenyl)lactate.Involved with EC 2.3.1.140 in the biosynthesis of rosmarinic acid.','3-(4-hydroxyphenyl)lactate + NAD(+) = 3-(4-hydroxyphenyl)pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.238','12-beta-hydroxysteroid dehydrogenase','NULL','NULL','Acts on a number of bile acids, both in their free and conjugatedforms.','3-alpha,7-alpha,12-beta-trihydroxy-5-beta-cholanate + NADP(+) =3-alpha,7-alpha-dihydroxy-12-oxo-5-beta-cholanate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.239','3-alpha-(17-beta)-hydroxysteroid dehydrogenase (NAD(+))','17-beta-HSD','NULL','Also acts on other 17-beta-hydroxysteroids, on the 3-alpha-hydroxygroup of pregnanes and bile acids.Different from EC 1.1.1.50 or EC 1.1.1.213.Formerly EC 1.1.1.63.','Testosterone + NAD(+) = androstenedione + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.240','N-acetylhexosamine 1-dehydrogenase','NULL','NULL','Also acts on N-acetylgalactosamine and, more slowly,on N-acetylmannosamine.Anomeric specificity was tested with N-acetyl-D-glucosamine, and itwas shown that the enzyme is specific for the alpha anomer.','N-acetyl-alpha-D-glucosamine + H(2)O + NAD(+) = N-acetyl-D-glucosaminate+ NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.241','6-endo-hydroxycineole dehydrogenase','NULL','NULL','NULL','6-endo-hydroxycineole + NAD(+) = 6-oxocineole + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.242','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.243','Carveol dehydrogenase','NULL','NULL','NULL','(-)-trans-carveol + NADP(+) = (-)-carvone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.244','Methanol dehydrogenase','NULL','NULL','NULL','Methanol + NAD(+) = formaldehyde + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.245','Cyclohexanol dehydrogenase','NULL','NULL','Also oxidizes some other alicyclic alcohols and diols.','Cyclohexanol + NAD(+) = cyclohexanone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.246','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.247','Codeinone reductase (NADPH)','NULL','NULL','Catalyzes the reversible reduction of codeinone to codeine, which isa direct precursor of morphine in the opium poppy plant, Papaversomniferum.','Codeine + NADP(+) = codeinone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.248','Salutaridine reductase (NADPH)','NULL','NULL','Catalyzes the reversible reduction of salutaridine to salutaridinol,which is a direct precursor of morphinan alkaloids in the poppyplant.','Salutaridinol + NADP(+) = salutaridine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.249','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.250','D-arabinitol 2-dehydrogenase','D-arabinitol 2-dehydrogenase (ribulose-forming)','NULL','NULL','D-arabinitol + NAD(+) = D-ribulose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.251','Galactitol-1-phosphate 5-dehydrogenase','NULL','Zinc.','NULL','Galactitol-1-phosphate + NAD(+) = L-tagatose 6-phosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.252','Tetrahydroxynaphthalene reductase','T4HN reductase','NULL','Reduces 1,3,6,8-tetrahydroxynaphthalene to scytalone and also reduces1,3,8-trihydroxynaphthalene to vermelone.Involved with EC 4.2.1.94 in the biosynthesis of melanin inpathogenic fungi.Formerly EC 1.3.1.50.','Scytalone + NADP(+) = 1,3,6,8-tetrahydroxynaphthalene + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.253','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.254','(S)-carnitine 3-dehydrogenase','D-carnitine dehydrogenase','NULL','Specific for the (S)-enantiomer of carnitine, i.e., the enantiomer ofthe substrate of EC 1.1.1.108.','(S)-carnitine + NAD(+) = 3-dehydrocarnitine + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.255','Mannitol dehydrogenase','MTD','NULL','The enzyme from Apium graveolens (celery) oxidizes alditols with aminimum requirement of 2R chirality at the carbon adjacent to theprimary carbon undergoing the oxidation.The enzyme is specific for NAD(+) and does not use NADP(+).','D-mannitol + NAD(+) = D-mannose + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.256','Fluoren-9-ol dehydrogenase','NULL','NULL','Involved in the pathway for fluorene metabolism in Arthrobacter sp.','Fluoren-9-ol + NAD(P)(+) = fluoren-9-one + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.257','4-(hydroxymethyl)benzenesulfonate dehydrogenase','NULL','NULL','Involved in the toluene-4-sulfonate degradation pathway in Comamonastestosteroni.','4-(hydroxymethyl)benzenesulfonate + NAD(+) = 4-formylbenzenesulfonate +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.258','6-hydroxyhexanoate dehydrogenase','NULL','NULL','Involved in the cyclohexanol degradation pathway in AcinetobacterNCIB 9871.','6-hydroxyhexanoate + NAD(+) = 6-oxohexanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.259','3-hydroxypimeloyl-CoA dehydrogenase','NULL','NULL','Involved in the anaerobic pathway of benzoate degradation inbacteria.','3-hydroxypimeloyl-CoA + NAD(+) = 3-oxopimeloyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.260','Sulcatone reductase','NULL','NULL','Studies on the effects of growth-stage and nutrient supply on thestereochemistry of sulcatone reduction in Clostridia pasteurianum,C.tyrobutyricum and Lactobacillus brevis suggest that there may be atleast two sulcatone reductases with different stereospecificities.','Sulcatol + NAD(+) = sulcatone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.261','sn-glycerol-1-phosphate dehydrogenase','Glycerol-1-phosphate dehydrogenase (NAD(P)(+))','Zinc or nickel.','Responsible for the formation of archaea-specific sn-glycerol-1-phosphate, the first step in the biosynthesis of polar lipids inarchaea.It is the enantiomer of sn-glycerol 3-phosphate, the form ofglycerophosphate found in bacteria and eukaryotes.The other enzymes involved in the biosynthesis of polar lipids inarchaea are EC 2.5.1.41 and EC 2.5.1.42, which together alkylate thehydroxy groups of glycerol 1-phosphate to give unsaturatedarchaetidic acid, which is acted upon by EC 2.7.7.67 to form CDP-unsaturated archaeol.The final step in the pathway involves the addition of L-serine, withconcomitant removal of CMP, leading to the production of unsaturatedarchaetidylserine.Activity of the enzyme is stimulated by K(+).','sn-glycerol 1-phosphate + NAD(P)(+) = glycerone phosphate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.262','4-hydroxythreonine-4-phosphate dehydrogenase','4-(phosphohydroxy)-L-threonine dehydrogenase','NULL','The product of the reaction undergoes decarboxylation to give3-amino-2-oxopropyl phosphate.The enzyme is part of the biosynthesis pathway of the coenzymepyridoxal 5-phosphate found in anaerobic bacteria.','4-phosphonooxy-L-threonine + NAD(+) = 3-amino-2-oxopropyl phosphate +CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.263','1,5-anhydro-D-fructose reductase','AF reductase','NULL','Also reduces pyridine-3-aldehyde and 2,3-butanedione.Acetaldehyde, 2-dehydroglucose (glucosone) and glucuronate are poorsubstrates, but there is no detectable action on glucose, mannose andfructose.','1,5-anhydro-D-glucitol + NADP(+) = 1,5-anhydro-D-fructose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.264','L-idonate 5-dehydrogenase (NAD(P)(+))','L-idonate 5-dehydrogenase','Zinc.','The enzyme from the bacterium Escherichia coli is specific for5-dehydro-D-gluconate.Cf. EC  1.1.1.366.','L-idonate + NAD(P)(+) = 5-dehydrogluconate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.265','3-methylbutanal reductase','NULL','NULL','The enzyme purified from Saccharomyces cerevisiae catalyzes thereduction of a number of straight-chain and branched aldehydes,as well as some aromatic aldehydes.','3-methylbutanol + NAD(P)(+) = 3-methylbutanal + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.266','dTDP-4-dehydro-6-deoxyglucose reductase','dTDP-4-keto-6-deoxyglucose reductase','NULL','The enzymes from the Gram-negative bacteria Actinobacillusactinomycetemcomitans and Escherichia coli O52 are involved inactivation of fucose for incorporation into capsular polysaccharideO-antigens.The enzyme from the Gram-positive bacterium Anoxybacillus tepidamans(Geobacillus tepidamans) is involved in activation of fucose forincorporation into the organisms S-layer.The enzyme from E.coli O52 has a higher catalytic efficiency withNADH than with NADPH.','dTDP-alpha-D-fucopyranose + NAD(P)(+) = dTDP-4-dehydro-6-deoxy-alpha-D-glucose + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.267','1-deoxy-D-xylulose-5-phosphate reductoisomerase','1-deoxyxylulose-5-phosphate reductoisomerase','Cobalt or manganese or magnesium.','The enzyme from several eubacteria, including Escherichia coli, formspart of an alternative nonmevalonate pathway for terpenoidbiosynthesis.The mechanism has been shown to be a retroaldol/aldol reaction.','2-C-methyl-D-erythritol 4-phosphate + NADP(+) = 1-deoxy-D-xylulose5-phosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.268','2-(R)-hydroxypropyl-CoM dehydrogenase','2-(2-(R)-hydroxypropylthio)ethanesulfonate dehydrogenase','NULL','Highly specific for (R)-2-hydroxyalkyl thioethers of CoM, in contrastto EC 1.1.1.269, which is highly specific for the (S)-enantiomer.Forms component III of a four-component enzyme system (comprisingEC 4.4.1.23 (component I), EC 1.8.1.5 (component II), EC 1.1.1.268(component III) and EC 1.1.1.269 (component IV)) that is involved inepoxyalkane carboxylation in Xanthobacter sp. strain Py2.','2-(R)-hydroxypropyl-CoM + NAD(+) = 2-oxopropyl-CoM + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.269','2-(S)-hydroxypropyl-CoM dehydrogenase','2-(2-(S)-hydroxypropylthio)ethanesulfonate dehydrogenase','NULL','Highly specific for (S)-2-hydroxyalkyl thioethers of CoM, in contrastto EC 1.1.1.268, which is highly specific for the (R)-enantiomer.Forms component IV of a four-component enzyme system (comprisingEC 4.4.1.23 (component I), EC 1.8.1.5 (component II), EC 1.1.1.268(component III) and EC 1.1.1.269 (component IV)) that is involved inepoxyalkane carboxylation in Xanthobacter sp. strain Py2.','2-(S)-hydroxypropyl-CoM + NAD(+) = 2-oxopropyl-CoM + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.270','3-beta-hydroxysteroid 3-dehydrogenase','3-keto-steroid reductase','NULL','The enzyme acts on multiple 3-beta-hydroxysteroids.Participates in the biosynthesis of zemosterol and cholesterol, whereit catalyzes the reaction in the opposite direction to that shown.The mammalian enzyme is bifunctional and also catalyzes EC 1.1.1.62.','A 3-beta-hydroxysteroid + NADP(+) = a 3-oxosteroid + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.271','GDP-L-fucose synthase','GDP-4-keto-6-deoxy-D-mannose-3,5-epimerase-4-reductase','NULL','Both human and Escherichia coli enzymes can use NADH in place ofNADPH to a slight extent.','GDP-beta-L-fucose + NADP(+) = GDP-4-dehydro-6-deoxy-alpha-D-mannose +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.272','D-2-hydroxyacid dehydrogenase (NADP(+))','NULL','NULL','This enzyme, characterized from the halophilic archaeon Haloferaxmediterranei, catalyzes a reversible stereospecific reduction of2-ketocarboxylic acids into the corresponding D-2-hydroxycarboxylicacids.The enzyme prefers substrates with a main chain of 5 carbons (such as4-methyl-2-oxopentanoate) to those with a shorter chain, and can useNADH with much lower efficiency.cf. EC 1.1.1.345.','An (R)-2-hydroxycarboxylate + NADP(+) = a 2-oxocarboxylate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.273','Vellosimine dehydrogenase','NULL','NULL','Also acts on related alkaloids with an endo-aldehyde group such asvellosimine (same stereochemistry at C-16) but only slight activitywith exo-aldehydes (epimeric at C-16).Detected in many cell suspension cultures of plants from the familyApocynaceae.','10-deoxysarpagine + NADP(+) = vellosimine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.274','2,5-didehydrogluconate reductase (2-dehydro-D-gluconate-forming)','2,5-diketo-D-gluconate reductase','NULL','The enzyme is involved in the catabolism of 2,5-didehydrogluconate.cf. EC 1.1.1.346.','2-dehydro-D-gluconate + NADP(+) = 2,5-didehydro-D-gluconate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.275','(+)-trans-carveol dehydrogenase','Carveol dehydrogenase','NULL','NADP(+) cannot replace NAD(+).Forms part of the monoterpenoid biosynthesis pathway in Carum carvi(caraway) seeds.','(+)-trans-carveol + NAD(+) = (+)-(S)-carvone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.276','Serine 3-dehydrogenase','NULL','NULL','The product 2-ammoniomalonate semialdehyde is spontaneously convertedinto 2-aminoacetaldehyde and CO(2).NAD(+) cannot replace NADP(+).','L-serine + NADP(+) = 2-ammoniomalonate semialdehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.277','3-beta-hydroxy-5-beta-steroid dehydrogenase','3-beta-hydroxysteroid 5-beta-oxidoreductase','NULL','NULL','3-beta-hydroxy-5-beta-pregnane-20-one + NADP(+) = 5-beta-pregnan-3,20-dione + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.278','3-beta-hydroxy-5-alpha-steroid dehydrogenase','NULL','NULL','NULL','3-beta-hydroxy-5-alpha-pregnane-20-one + NADP(+) = 5-alpha-pregnan-3,20-dione + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.279','(R)-3-hydroxyacid-ester dehydrogenase','3-oxo ester (R)-reductase','NULL','Also acts on ethyl (R)-3-oxobutanoate and some other (R)-3-hydroxyacid esters.The (R)- symbol is allotted on the assumption that no substituentschange the order of priority from O-3','Ethyl (R)-3-hydroxyhexanoate + NADP(+) = ethyl 3-oxohexanoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.280','(S)-3-hydroxyacid-ester dehydrogenase','3-oxo ester (S)-reductase','NULL','Also acts on 4-oxo- and 5-oxo-fatty acids and their esters.Cf. EC 1.1.1.279.Formerly EC 1.2.1.56.','Ethyl (S)-3-hydroxyhexanoate + NADP(+) = ethyl 3-oxohexanoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.281','GDP-4-dehydro-6-deoxy-D-mannose reductase','GDP-4-keto-6-deoxy-D-mannose reductase','NULL','Differs from EC 1.1.1.187 in that the only product formed is GDP-alpha-D-rhamnose (GDP-6-deoxy-alpha-D-mannose).D-rhamnose is a constituent of lipopolysaccharides of Gram-negativeplant and human pathogenic bacteria.','GDP-alpha-D-rhamnose + NAD(P)(+) = GDP-4-dehydro-alpha-D-rhamnose +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.282','Quinate/shikimate dehydrogenase','NULL','NULL','This is the second shikimate dehydrogenase enzyme found inEscherichia coli and differs from EC 1.1.1.25, in that it can useboth quinate and shikimate as substrate and either NAD(+) or NADP(+)as acceptor.','(1) L-quinate + NAD(P)(+) = 3-dehydroquinate + NAD(P)H.(2) Shikimate + NAD(P)(+) = 3-dehydroshikimate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.283','Methylglyoxal reductase (NADPH)','Lactaldehyde dehydrogenase (NADP(+))','NULL','The enzyme from the yeast Saccharomyces cerevisiae catalyzes thereduction of a keto group in a number of compounds, formingenantoiopure products.Among the substrates are methylglyoxal (which is reduced to (S)-lactaldehyde), 3-methylbutanal, hexane-2,5-dione and 3-chloro-1-phenylpropan-1-one.It differs in coenzyme requirement from EC 1.1.1.78, which is foundin mammals.','(S)-lactaldehyde + NADP(+) = methylglyoxal + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.284','S-(hydroxymethyl)glutathione dehydrogenase','ADH3','NULL','The substrate, S-(hydroxymethyl)glutathione, forms spontaneously fromglutathione and formaldehyde; its rate of formation is increased insome bacteria by the presence of EC 4.4.1.22.Forms part of the pathway that detoxifies formaldehyde, since theproduct is hydrolyzed by EC 3.1.2.12.Also specifically reduces S-nitrosylglutathione.Formerly EC 1.2.1.1.','S-(hydroxymethyl)glutathione + NAD(P)(+) = S-formylglutathione + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.285','3-deamino-3-oxonicotianamine reductase','NULL','NULL','NULL','2-deoxymugineic acid + NAD(P)(+) = 3-deamino-3-oxonicotianamine +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.286','Isocitrate--homoisocitrate dehydrogenase','Homoisocitrate--isocitrate dehydrogenase','Manganese; Potassium or NH(4)(+).','Unlike EC 1.1.1.41 and EC 1.1.1.87, this enzyme, from Pyrococcushorikoshii, can use both isocitrate and homoisocitrate as substrate.May play a role in both the lysine and glutamate biosynthesispathways.','(1) Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH.(2) (1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate + NAD(+) = 2-oxoadipate+ CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.287','D-arabinitol dehydrogenase (NADP(+))','D-arabinitol dehydrogenase 1','NULL','The enzyme from the rust fungus Uromyces fabae can use D-arabinitoland D-mannitol as substrates in the forward direction and D-xylulose,D-ribulose and, to a lesser extent, D-fructose as substrates in thereverse direction.This enzyme carries out the reactions of both EC 1.1.1.11 andEC 1.1.1.250, but unlike them, uses NADP(+) rather than NAD(+) ascofactor.D-arabinitol is capable of quenching reactive oxygen species involvedin defense reactions of the host plant.','(1) D-arabinitol + NADP(+) = D-xylulose + NADPH.(2) D-arabinitol + NADP(+) = D-ribulose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.288','Xanthoxin dehydrogenase','Xanthoxin oxidase','Molybdenum.','NADP(+) cannot replace NAD(+) and short-chain alcohols such asethanol, isopropanol, butanol and cyclohexanol cannot replacexanthoxin as substrate.Involved in the abscisic-acid biosynthesis pathway in plants, alongwith EC 1.2.3.14, EC 1.13.11.51 and EC 1.14.13.93.','Xanthoxin + NAD(+) = abscisic aldehyde + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.289','Sorbose reductase','NULL','NULL','The reaction occurs predominantly in the reverse direction.This enzyme can also convert D-fructose into D-mannitol, but moreslowly.','D-glucitol + NADP(+) = L-sorbose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.290','4-phosphoerythronate dehydrogenase','4-O-phosphoerythronate dehydrogenase','NULL','This enzyme catalyzes the second step in the biosynthesis of thecoenzyme pyridoxal 5-phosphate in Escherichia coli.The reaction occurs predominantly in the reverse direction.Other enzymes involved in this pathway are EC 1.2.1.72, EC 2.6.1.52,EC 1.1.1.262, EC 2.6.99.2 and EC 1.4.3.5.','4-phospho-D-erythronate + NAD(+) = (3R)-3-hydroxy-2-oxo-4-phosphonooxybutanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.291','2-hydroxymethylglutarate dehydrogenase','NULL','NULL','NADP(+) cannot replace NAD(+).Forms part of the nicotinate-fermentation catabolism pathway inEubacterium barkeri.Other enzymes involved in this pathway are EC 1.17.1.5, EC 1.3.7.1,EC 3.5.2.18, EC 5.4.99.4, EC 5.3.3.6, EC 4.2.1.85 and EC 4.1.3.32.','(S)-2-hydroxymethylglutarate + NAD(+) = 2-formylglutarate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.292','1,5-anhydro-D-fructose reductase (1,5-anhydro-D-mannitol-forming)','1,5-anhydro-D-fructose reductase','NULL','Present in some but not all Rhizobium species.Differs from hepatic 1,5-anhydro-D-fructose reductase, which yields1,5-anhydro-D-glucitol as the product (see EC 1.1.1.263).In Sinorhizobium morelense, the product of the reaction, 1,5-anhydro-D-mannitol, can be further metabolized to D-mannose.Also reduces 1,5-anhydro-D-erythro-hexo-2,3-diulose and 2-ketoaldoses(called osones), such as D-glucosone (D-arabino-hexos-2-ulose) and6-deoxy-D-glucosone.Does not reduce common aldoses and ketoses, or non-sugar aldehydesand ketones.','1,5-anhydro-D-mannitol + NADP(+) = 1,5-anhydro-D-fructose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.293','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.294','Chlorophyll(ide) b reductase','Chl b reductase','NULL','Carries out the first step in the conversion of chlorophyll b tochlorophyll a.Involved in chlorophyll degradation, which occurs during leafsenescence and it also forms part of the chlorophyll cycle, whichinterconverts chlorophyll a and b in response to changing lightconditions.While both chlorophyll a and chlorophyll b are found in higherplants, only breakdown products derived from chlorophyll a are foundin the end products, i.e. non-fluorescent chlorophyll catabolites(NCCs).The transition from chlorophyll b, which contains a formyl group onC7, to chlorophyll a, which contains a methyl group on C7, proceedsthrough the intermediate 7(1)-hydroxychlorophyll.Another enzyme, which requires ferredoxin but has not beencharacterized fully, is required to complete the conversion.','7(1)-hydroxychlorophyllide a + NAD(P)(+) = chlorophyllide b + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.295','Momilactone-A synthase','Momilactone A synthase','NULL','The rice phytoalexin momilactone A is a diterpenoid secondarymetabolite that is involved in the defense mechanism of the plant.Momilactone A is produced in response to attack by a pathogen throughthe perception of elicitor signal molecules such as chitinoligosaccharide, or after exposure to UV irradiation.The enzyme, which catalyzes the last step in the biosynthesis ofmomilactone A, can use both NAD(+) and NADP(+) but activity is higherwith NAD(+).','3-beta-hydroxy-9-beta-pimara-7,15-diene-19,6-beta-olide + NAD(P)(+) =momilactone A + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.296','Dihydrocarveol dehydrogenase','Carveol dehydrogenase','NULL','This enzyme from the Gram-positive bacterium Rhodococcus erythropolisDCL14 forms part of the carveol and dihydrocarveol degradationpathway.Accepts all eight stereoisomers of menth-8-en-2-ol as substrate,although some isomers are converted faster than others.The preferred substrates are (+)-neoisodihydrocarveol, (+)-isodihydrocarveol, (+)-dihydrocarveol and (-)-isodihydrocarveol.','Menth-8-en-2-ol + NAD(+) = menth-8-en-2-one + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.297','Limonene-1,2-diol dehydrogenase','NULL','NULL','While the enzyme from the Gram-positive bacterium Rhodococcuserythropolis DCL14 can use both (1S,2S,4R)- and (1R,2R,4S)-menth-8-ene-1,2-diol as substrate, activity is higher with (1S,2S,4R)-menth-8-ene-1,2-diol as substrate.','Menth-8-ene-1,2-diol + NAD(+) = 1-hydroxymenth-8-en-2-one + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.298','3-hydroxypropionate dehydrogenase (NADP(+))','NULL','NULL','Catalyzes the reduction of 3-oxopropanoate to 3-hydroxypropanoate,a key step in the 3-hydroxypropanoate and the3-hydroxypropanoate/4-hydroxybutyrate cycles, autotrophic CO(2)fixation pathways found in some green non-sulfur phototrophicbacteria and archaea, respectively.The enzyme from Chloroflexus aurantiacus is bifunctional, and alsocatalyzes the upstream reaction in the pathway, EC 1.2.1.75.Different from EC 1.1.1.59 by cofactor preference.','3-hydroxypropanoate + NADP(+) = 3-oxopropanoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.299','Malate dehydrogenase (NAD(P)(+))','NULL','NULL','This enzyme, which was characterized from the methanogenic archaeonMethanobacterium thermoautotrophicum, catalyzes only the reduction ofoxaloacetate, and can use NAD(+) and NADP(+) with similar specificactivity.Different from EC 1.1.1.37, EC 1.1.1.82 and EC 1.1.5.4.','(S)-malate + NAD(P)(+) = oxaloacetate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.300','NADP-retinol dehydrogenase','All-trans retinal reductase','NULL','Greater catalytic efficiency in the reductive direction.This observation, and the enzymes localization at the entrance tothe mitochondrial matrix, suggest that it may function to protectmitochondria against oxidative stress associated with the highlyreactive retinal produced from dietary beta-carotene byEC 1.14.99.36.Km-values for NADP(+) and NADPH are at least 800-fold lower thanthose for NAD(+) and NADH.This enzyme differs from EC 1.1.1.105 which prefers NAD(+) and NADH.Formerly EC 1.1.1.n2.','All-trans-retinol + NADP(+) = all-trans-retinal + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.301','D-arabitol-phosphate dehydrogenase','APDH','NULL','This enzyme participates in arabitol catabolism.The enzyme also converts D-arabitol 5-phosphate to D-ribulose5-phosphate at a lower rate.Formerly EC 1.1.1.n10.','D-arabinitol 1-phosphate + NAD(+) = D-xylulose 5-phosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.302','2,5-diamino-6-(ribosylamino)-4(3H)-pyrimidinone 5-phosphate reductase','2,5-diamino-6-ribosylamino-4(3H)-pyrimidinone 5-phosphate reductase','NULL','The reaction proceeds in the opposite direction.A step in riboflavin biosynthesis, NADPH and NADH function equallywell as reductant.Differs from EC 1.1.1.193 since it does not catalyze the reduction of5-amino-6-ribosylaminopyrimidine-2,4(1H,3H)-dione 5-phosphate.','2,5-diamino-6-(5-phospho-D-ribitylamino)pyrimidin-4(3H)-one + NAD(P)(+) =2,5-diamino-6-(5-phospho-D-ribosylamino)pyrimidin-4(3H)-one + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.303','Diacetyl reductase ((R)-acetoin forming)','(R)-acetoin dehydrogenase','NULL','The reaction is catalyzed in the reverse direction.This activity is usually associated with butanediol dehydrogenaseactivity (EC 1.1.1.4 or EC 1.1.1.76).While the butanediol dehydrogenase activity is reversible, diacetylreductase activity is irreversible.This enzyme has been reported in the yeast Saccharomyces cerevisiae.Different from EC 1.1.1.304.Formerly EC 1.1.1.5.','(R)-acetoin + NAD(+) = diacetyl + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.304','Diacetyl reductase ((S)-acetoin forming)','(S)-acetoin dehydrogenase','NULL','The reaction is catalyzed in the reverse direction.This activity is usually associated with butanediol dehydrogenaseactivity (EC 1.1.1.4 or EC 1.1.1.76).While the butanediol dehydrogenase activity is reversible, diacetylreductase activity is irreversible.This enzyme has been reported in the bacteria Geobacillusstearothermophilus, Enterobacter aerogenes and Klebsiella pneumoniae.Different from EC 1.1.1.303.Formerly EC 1.1.1.5.','(S)-acetoin + NAD(+) = diacetyl + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.305','UDP-glucuronic acid oxidase (UDP-4-keto-hexauronic acid decarboxylating)','ArnADH','NULL','The activity is part of a bifunctional enzyme also performing thereaction of EC 2.1.2.13.Formerly EC 1.1.1.n1.','UDP-alpha-D-glucuronate + NAD(+) = UDP-beta-L-threo-pentapyranos-4-ulose+ CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.306','S-(hydroxymethyl)mycothiol dehydrogenase','Mycothiol-dependent formaldehyde dehydrogenase','Zinc.','S-hydroxymethylmycothiol is believed to form spontaneously fromformaldehyde and mycothiol.This enzyme oxidizes the product of this spontaneous reaction toS-formylmycothiol, in a reaction that is analogous to EC 1.1.1.284,S-(hydroxymethyl)glutathione dehydrogenase.Formerly EC 1.2.1.66.','S-(hydroxymethyl)mycothiol + NAD(+) = S-formylmycothiol + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.307','D-xylose reductase','NULL','NULL','Xylose reductase catalyzes the initial reaction in the xyloseutilization pathway, the NAD(P)H dependent reduction of xylose toxylitol.','Xylitol + NAD(P)(+) = D-xylose + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.308','Sulfopropanediol 3-dehydrogenase','Dihydroxypropanesulfonate 3-dehydrogenase','NULL','The enzyme is involved in degradation of (R)-2,3-dihydroxypropanesulfonate.','(R)-2,3-dihydroxypropane-1-sulfonate + 2 NAD(+) + H(2)O = (R)-3-sulfolactate + 2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.309','Phosphonoacetaldehyde reductase (NADH)','NULL','NULL','The enzyme from Streptomyces viridochromogenes catalyzes a step inthe biosynthesis of phosphinothricin tripeptide, the reduction ofphosphonoacetaldehyde to 2-hydroxyethylphosphonate.The preferred cofactor is NADH, lower activity with NADPH.','2-hydroxyethylphosphonate + NAD(+) = phosphonoacetaldehyde + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.310','(S)-sulfolactate dehydrogenase','(2S)-3-sulfolactate dehydrogenase','NULL','This enzyme, isolated from the bacterium Chromohalobacter salexigensDSM 3043, acts only on the (S)-enantiomer of 3-sulfolactate.Combined with EC 1.1.1.338, it provides a racemase system thatconverts (2S)-3-sulfolactate to (2R)-3-sulfolactate, which isdegraded further by EC 4.4.1.24.The enzyme is specific for NAD(+).','(2S)-3-sulfolactate + NAD(+) = 3-sulfopyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.311','(S)-1-phenylethanol dehydrogenase','PED','NULL','The enzyme is involved in degradation of ethylbenzene.','(S)-1-phenylethanol + NAD(+) = acetophenone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.312','2-hydroxy-4-carboxymuconate semialdehyde hemiacetal dehydrogenase','4-carboxy-2-hydroxymuconate-6-semialdehyde dehydrogenase','NULL','The enzyme does not act on unsubstituted aliphatic or aromaticaldehydes or glucose; NAD(+) can replace NADP(+), but with loweraffinity.The enzyme was initially believed to act on 4-carboxy-2-hydroxy-cis,cis-muconate 6-semialdehyde and produce 4-carboxy-2-hydroxy-cis,cis-muconate.However, later studies showed that the substrate is the hemiacetalform, and the product is 2-oxo-2H-pyran-4,6-dicarboxylate.Formerly EC 1.2.1.45.','4-carboxy-2-hydroxymuconate semialdehyde hemiacetal + NADP(+) = 2-oxo-2H-pyran-4,6-dicarboxylate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.313','Sulfoacetaldehyde reductase','NULL','NULL','Catalyzes the reaction only in the opposite direction.Involved in taurine degradation.The bacterium Chromohalobacter salexigens strain DSM 3043 possessestwo enzymes that catalyze this reaction, a constitutive enzyme(encoded by isfD2) and an inducible enzyme (encoded by isfD).The latter is induced by taurine, and is responsible for most of theactivity observed in taurine-grown cells.','Isethionate + NADP(+) = 2-sulfoacetaldehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.314','Germacrene A alcohol dehydrogenase','NULL','NULL','In Lactuca sativa EC 1.1.1.314 is a mutifunctional enzyme withEC 1.14.13.123.','Germacra-1(10),4,11(13)-trien-12-ol + 2 NADP(+) + H(2)O = germacra-1(10),4,11(13)-trien-12-oate + 2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.315','11-cis-retinol dehydrogenase','NULL','NULL','This enzyme, abundant in the retinal pigment epithelium, catalyzesthe reduction of 11-cis-retinol to 11-cis-retinal while the substrateis bound to the retinal-binding protein.This is a crucial step in the regeneration of 11-cis-retinal,the chromophore of rhodopsin.The enzyme can also accept other cis forms of retinol.','11-cis-retinol-[retinal-binding-protein] + NAD(+) = 11-cis-retinal-[retinol-binding-protein] + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.316','L-galactose 1-dehydrogenase','L-galactose dehydrogenase','NULL','The enzyme catalyzes a step in the ascorbate biosynthesis in higherplants (Smirnoff-Wheeler pathway).The activity with NADP(+) is less than 10% of the activity withNAD(+).','L-galactose + NAD(+) = L-galactono-1,4-lactone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.317','Perakine reductase','NULL','NULL','The biosynthesis of raucaffrinoline from perakine is a side route ofthe ajmaline biosynthesis pathway.The enzyme is a member of the aldo-keto reductase enzyme superfamilyfrom higher plants.','Raucaffrinoline + NADP(+) = perakine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.318','Eugenol synthase','EGS2','NULL','The enzyme acts in the opposite direction.The enzymes from the plants Ocimum basilicum (sweet basil), Clarkiabreweri and Petunia hybrida only accept coniferyl acetate and formeugenol.The enzyme from Pimpinella anisum (anise) forms anol (from 4-coumarylacetate) in vivo, although the recombinant enzyme can form eugenolfrom coniferyl acetate.The enzyme from Larrea tridentata (creosote bush) also forms chavicolfrom a coumaryl ester and can use NADH.','Eugenol + a carboxylate + NADP(+) = a coniferyl ester + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.319','Isoeugenol synthase','T-anol/isoeugenol synthase 1','NULL','The enzyme acts in the opposite direction.In Ocimum basilicum (sweet basil), Clarkia breweri, and Petuniahybrida only isoeugenol is formed.However in Pimpinella anisum (anise) only anol is formed in vivo,although the cloned enzyme does produce isoeugenol.','Isoeugenol + acetate + NADP(+) = coniferyl acetate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.320','Benzil reductase ((S)-benzoin forming)','NULL','NULL','The enzyme also reduces 1-phenylpropane-1,2-dione.The enzyme from Bacillus cereus in addition reduces 1,4-naphthoquinone and 1-(4-methylphenyl)-2-phenylethane-1,2-dione withhigh efficiency.Formerly EC 1.1.1.n7.','(S)-benzoin + NADP(+) = benzil + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.321','Benzil reductase ((R)-benzoin forming)','NULL','NULL','The enzyme from the bacterium Xanthomonas oryzae is able to reduceenantioselectively only one of the two carbonyl groups of benzil togive optically active (R)-benzoin.','(R)-benzoin + NADP(+) = benzil + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.322','(-)-endo-fenchol dehydrogenase','NULL','NULL','Isolated from the plant Foeniculum vulgare (fennel).NADH is slightly preferred to NADPH.','(-)-endo-fenchol + NAD(P)(+) = (+)-fenchone + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.323','(+)-thujan-3-ol dehydrogenase','D-3-thujanol dehydrogenase','NULL','Isolated from the plant Tanacetum vulgare (tansy).NADH is preferred to NADPH.','(+)-thujan-3-ol + NAD(P)(+) = (+)-thujan-3-one + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.324','8-hydroxygeraniol dehydrogenase','8-hydroxygeraniol oxidoreductase','Zinc.','The enzyme catalyzes the oxidation of (6E)-8-hydroxygeraniol to (6E)-8-oxogeranial via either (6E)-8-hydroxygeranial or (6E)-8-oxogeraniol.Also acts on geraniol, nerol and citronellol.May be identical to EC 1.1.1.183.The recommended numbering of geraniol gives 8-hydroxygeraniol as thesubstrate rather than 10-hydroxygeraniol.','(6E)-8-hydroxygeraniol + 2 NADP(+) = (6E)-8-oxogeranial + 2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.325','Sepiapterin reductase (L-threo-7,8-dihydrobiopterin forming)','NULL','NULL','This enzyme, isolated from the bacterium Chlorobium tepidum,catalyzes the final step in the de novo synthesis oftetrahydrobiopterin from GTP.Cf. EC 1.1.1.153.','(1) L-threo-7,8-dihydrobiopterin + NADP(+) = sepiapterin + NADPH.(2) L-threo-tetrahydrobiopterin + 2 NADP(+) = 6-pyruvoyl-5,6,7,8-tetrahydropterin + 2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.326','Zerumbone synthase','NULL','NULL','The enzyme was cloned from shampoo ginger, Zingiber zerumbet.','10-hydroxy-alpha-humulene + NAD(+) = zerumbone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.327','5-exo-hydroxycamphor dehydrogenase','NULL','Zinc.','Isolated from Pseudomonas putida, and involved in degradation of (+)-camphor.','5-exo-hydroxycamphor + NAD(+) = bornane-2,5-dione + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.328','Nicotine blue oxidoreductase','NULL','NULL','The enzyme, characterized from the nicotine degrading bacteriumArthrobacter nicotinovorans, catalyzes the reduction of nicotineblue to its hydroquinone form (the opposite direction from thatshown).Nicotine blue is the name given to the compound formed by theautocatalytic condensation of two molecules of 2,3,6-trihydroxypyridine, an intermediate in the nicotine degradationpathway.The main role of the enzyme may be to prevent the intracellularformation of nicotine blue semiquinone radicals, which by redoxcycling would lead to the formation of toxic reactive oxygen species.The enzyme possesses a slight preference for NADH over NADPH.','3,3-bipyridine-2,2,5,5,6,6-hexol + NAD(P)(+) = (E)-2,2,5,5-tetrahydroxy-6H,6H-(3,3-bipyridinylidene)-6,6-dione +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.329','2-deoxy-scyllo-inosamine dehydrogenase','NULL','Zinc.','Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including kanamycin, neomycin andribostamycin.Cf. EC 1.1.99.38.','2-deoxy-scyllo-inosamine + NAD(P)(+) = 3-amino-2,3-dideoxy-scyllo-inosose+ NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.330','Very-long-chain 3-oxoacyl-CoA reductase','Very-long-chain 3-ketoacyl-CoA reductase','NULL','The second component of the elongase, a microsomal protein complexresponsible for extending palmitoyl-CoA and stearoyl-CoA (andmodified forms thereof) to very-long-chain acyl CoAs.The enzyme is active with substrates with chain length of C(16) toC(34), depending on the species.Cf. EC 2.3.1.199, EC 4.2.1.134 and EC 1.3.1.93.Formerly EC 1.1.1.n13.','A very-long-chain (3R)-3-hydroxyacyl-CoA + NADP(+) = a very-long-chain3-oxoacyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.331','Secoisolariciresinol dehydrogenase','NULL','NULL','Isolated from the plants Forsythia intermedia and Podophyllumpeltatum.An intermediate lactol is detected in vitro.','(-)-secoisolariciresinol + 2 NAD(+) = (-)-matairesinol + 2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.332','Chanoclavine-I dehydrogenase','NULL','NULL','The enzyme catalyzes a step in the pathway of ergot alkaloidbiosynthesis in certain fungi.','Chanoclavine-I + NAD(+) = chanoclavine-I aldehyde + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.333','Decaprenylphospho-beta-D-erythro-pentofuranosid-2-ulose 2-reductase','Decaprenylphospho-beta-D-ribofuranose 2-epimerase','NULL','The reaction is catalyzed in the reverse direction.The enzyme, isolated from the bacterium Mycobacterium smegmatis,is involved, along with EC 1.1.98.3, in the epimerization oftrans,octacis-decaprenylphospho-beta-D-ribofuranose to trans,octacis-decaprenylphospho-beta-D-arabinoofuranose, the arabinosyl donor forthe biosynthesis of mycobacterial cell wall arabinan polymers.','Trans,octacis-decaprenylphospho-beta-D-arabinofuranose + NAD(+) =trans,octacis-decaprenylphospho-beta-D-erythro-pentofuranosid-2-ulose +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.334','Methylecgonone reductase','NULL','NULL','The enzyme from the plant Erythroxylum coca catalyzes the penultimatestep in the biosynthesis of cocaine.In vivo the reaction proceeds in the opposite direction.With NADH instead of NADPH the reaction rate is reduced to 14%.The enzyme also reduces tropinone, nortropinone and6-hydroxytropinone but with lower reaction rates.','Ecgonine methyl ester + NADP(+) = ecgonone methyl ester + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.335','UDP-N-acetyl-2-amino-2-deoxyglucuronate dehydrogenase','NULL','NULL','This enzyme participates in the biosynthetic pathway for UDP-alpha-D-ManNAc3NAcA (UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-mannuronicacid), an important precursor of B-band lipopolysaccharide.The enzymes from Pseudomonas aeruginosa serotype O5 and Thermusthermophilus form a complex with the the enzyme catalyzing the nextstep the pathway (EC 2.6.1.98).The enzyme also possesses an EC 1.1.99.2 activity, and utilizes the2-oxoglutarate produced by EC 2.6.1.98 to regenerate the tightlybound NAD(+).The enzymes from Bordetella pertussis and Chromobacterium violaceumdo not bind NAD(+) as tightly and do not require 2-oxoglutarate tofunction.','UDP-N-acetyl-2-amino-2-deoxy-alpha-D-glucuronate + NAD(+) = UDP-2-acetamido-2-deoxy-alpha-D-ribo-hex-3-uluronate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.336','UDP-N-acetyl-D-mannosamine dehydrogenase','UDP-ManNAc 6-dehydrogenase','NULL','Part of the pathway for acetamido sugar biosynthesis in bacteria andarchaea.The enzyme has no activity with NADP(+).Formerly EC 1.1.1.n3.','UDP-N-acetyl-alpha-D-mannosamine + 2 NAD(+) + H(2)O = UDP-N-acetyl-alpha-D-mannosaminuronate + 2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.337','L-2-hydroxycarboxylate dehydrogenase (NAD(+))','(R)-sulfolactate dehydrogenase','NULL','The enzyme from the archaeon Methanocaldococcus jannaschii acts onmultiple L-2-hydroxycarboxylates including (2R)-3-sulfolactate, (S)-malate, (S)-lactate, and (S)-2-hydroxyglutarate.Note that (2R)-3-sulfolactate has the same stereo configuration as(2S)-2-hydroxycarboxylates.','A (2S)-2-hydroxycarboxylate + NAD(+) = a 2-oxocarboxylate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.338','(2R)-3-sulfolactate dehydrogenase (NADP(+))','(R)-sulfolactate dehydrogenase','NULL','The enzyme from the bacterium Chromohalobacter salexigens can onlyutilize NADP(+).It functions both biosynthetically in coenzyme M biosynthesis anddegradatively, in the degradation of sulfolactate.It can not use (S)-malate and (S)-lactate.','(2R)-3-sulfolactate + NADP(+) = 3-sulfopyruvate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.339','dTDP-6-deoxy-L-talose 4-dehydrogenase (NAD(+))','NULL','NULL','The enzyme has been characterized from the bacterium Aggregatibacteractinomycetemcomitans, in which it participates in the biosynthesisof the serotype c-specific polysaccharide antigen.Shows no activity with NADP(+).','dTDP-6-deoxy-beta-L-talose + NAD(+) = dTDP-4-dehydro-beta-L-rhamnose +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.340','1-deoxy-11-beta-hydroxypentalenate dehydrogenase','1-deoxy-11-beta-hydroxypentalenic acid dehydrogenase','NULL','Isolated from the bacterium Streptomyces avermitilis and present inmany other Streptomyces species.Part of the pathway for pentalenolactone biosynthesis.','1-deoxy-11-beta-hydroxypentalenate + NAD(+) = 1-deoxy-11-oxopentalenate +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.341','CDP-abequose synthase','NULL','NULL','Isolated from Yersinia pseudotuberculosis and Salmonella enterica.','CDP-alpha-D-abequose + NADP(+) = CDP-4-dehydro-3,6-dideoxy-alpha-D-glucose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.342','CDP-paratose synthase','NULL','NULL','The enzyme is involved in synthesis of paratose and tyvelose, unusual3,6-dideoxyhexose sugars that form part of the O-antigen in thelipopolysaccharides of several enteric bacteria.Isolated from Salmonella enterica subsp. enterica serovar Typhi(Salmonella typhi).','CDP-alpha-D-paratose + NADP(+) = CDP-4-dehydro-3,6-dideoxy-alpha-D-glucose + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.343','Phosphogluconate dehydrogenase (NAD(+)-dependent, decarboxylating)','6-PGDH','NULL','Highly specific for NAD(+).The enzyme catalyzes both the oxidation and decarboxylation of6-phospho-D-gluconate.In the bacterium Methylobacillus flagellatus the enzyme participatesin a formaldehyde oxidation pathway.Cf. EC 1.1.1.44.','6-phospho-D-gluconate + NAD(+) = D-ribulose 5-phosphate + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.344','dTDP-6-deoxy-L-talose 4-dehydrogenase (NAD(P)(+))','NULL','NULL','The enzyme works equally well with NAD(+) and NADP(+).','dTDP-6-deoxy-beta-L-talose + NAD(P)(+) = dTDP-4-dehydro-beta-L-rhamnose +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.345','D-2-hydroxyacid dehydrogenase (NAD(+))','NULL','NULL','This enzyme, characterized from the halophilic archaeon Haloferaxmediterranei, catalyzes a reversible stereospecific reduction of2-ketocarboxylic acids into the corresponding D-2-hydroxycarboxylicacids.The enzyme is strictly NAD-dependent and prefers substrates with amain chain of 3-4 carbons (pyruvate and 2-oxobutanoate).Activity with 4-methyl-2-oxopentanoate is 10-fold lower.Cf. EC 1.1.1.272.','An (R)-2-hydroxcarboxylate + NAD(+) = a 2-oxocarboxylate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.346','2,5-didehydrogluconate reductase (2-dehydro-L-gulonate-forming)','2,5-diketo-D-gluconate-reductase','NULL','The enzyme is involved in ketogluconate metabolism, and catalyzes thereaction in vivo in the reverse direction to that shown.It is used in the commercial microbial production of ascorbate.cf. EC 1.1.1.274.','2-dehydro-L-gulonate + NADP(+) = 2,5-didehydro-D-gluconate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.347','Geraniol dehydrogenase (NAD(+))','GeDH','NULL','The enzyme from the bacterium Castellaniella defragrans is mostactive in vitro with perillyl alcohol.The enzyme from the prune mite Carpoglyphus lactis also acts (moreslowly) on farnesol but not on nerol.Formerly EC 1.1.1.n14.','Geraniol + NAD(+) = geranial + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.348','Vestitone reductase','NULL','NULL','This plant enzyme catalyzes the penultimate step in the biosynthesisof the pterocarpin phytoalexins medicarpin and maackiain.This activity was previously classified as part of EC 1.1.1.246,which is now known to be catalyzed by two enzymes, vestitonereductase and EC 4.2.1.139.Formerly EC 1.1.1.246.','(3R,4R)-4-methoxyisoflavan-2,4,7-triol + NADP(+) = (3R)-vestitone +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.349','Norsolorinic acid ketoreductase','NULL','NULL','Involved in the synthesis of aflatoxins in the fungus Aspergillusparasiticus.','(1S)-averantin + NADP(+) = norsolorinic acid + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.350','Ureidoglycolate dehydrogenase (NAD(+))','NULL','NULL','Involved in catabolism of purines.The enzyme from the bacterium Escherichia coli is specific forNAD(+).Cf. EC 1.1.1.154.','(S)-ureidoglycolate + NAD(+) = oxalurate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.351','Phosphogluconate dehydrogenase (NAD(P)(+)-dependent, decarboxylating)','NULL','NULL','The enzyme participates in the oxidative branch of the pentosephosphate pathway, whose main purpose is to produce reducing powerand pentose for biosynthetic reactions.Unlike EC 1.1.1.44, it is not specific for NADP(+) and can acceptboth cofactors with similar efficiency.Cf. EC 1.1.1.343.','6-phospho-D-gluconate + NAD(P)(+) = D-ribulose 5-phosphate + CO(2) +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.352','5-hydroxyaverantin dehydrogenase','HAVN dehydrogenase','NULL','Isolated from the aflatoxin-producing mold Aspergillus parasiticus.Involved in aflatoxin biosynthesis.5-oxoaverantin will spontaneously form averufin by intramolecularketalisation.Cf. EC 4.2.1.142.','(1) (1S,5S)-hydroxyaverantin + NAD(+) = 5-oxoaverantin + NADH.(2) (1S,5R)-hydroxyaverantin + NAD(+) = 5-oxoaverantin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.353','Versiconal hemiacetal acetate reductase','VHA reductase','NULL','Isolated from the mold Aspergillus parasiticus.Involved in a metabolic grid that leads to aflatoxin biosynthesis.','(1) Versicolorone + NADP(+) = 1-hydroxyversicolorone + NADPH.(2) Versiconol acetate + NADP(+) = versiconal hemiacetal acetate + NADPH.(3) Versiconol + NADP(+) = versiconal + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.354','Farnesol dehydrogenase (NAD(+))','NAD(+)-farnesol dehydrogenase','NULL','The enzyme from the prune mite Carpoglyphus lactis also acts ongeraniol with greater activity (cf. 1.1.1.347).Unlike EC 1.1.1.216, this enzyme cannot use NADP(+) as cofactor.','(2E,6E)-farnesol + NAD(+) = (2E,6E)-farnesal + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.355','2-dehydrokanamycin reductase','NULL','NULL','Found in the bacterium Streptomyces kanamyceticus where it isinvolved in the conversion of kanamycin B to kanamycin A.','Kanamycin A + NADP(+) = 2-dehydrokanamycin A + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.356','GDP-L-colitose synthase','NULL','NULL','The enzyme is involved in biosynthesis of L-colitose, a 3,6-dideoxyhexose found in the O-antigen of Gram-negativelipopolysaccharides, where it catalyzes the reaction in the reversedirection.The enzyme also performs the NAD(P)H-dependent epimerisation at C-5of the sugar.The enzyme from Yersinia pseudotuberculosis is Si-specific withrespect to NAD(P)H.','GDP-beta-L-colitose + NAD(P)(+) = GDP-4-dehydro-3,6-dideoxy-alpha-D-mannose + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.357','3-alpha-hydroxysteroid 3-dehydrogenase','3-alpha-hydroxysteroid dehydrogenase','NULL','The enzyme acts on multiple 3-alpha-hydroxysteroids, such asandrosterone and 5 alpha-dihydrotestosterone.The mammalian enzymes are involved in inactivation of steroidhormones, while the bacterial enzymes are involved in steroiddegradation.This entry stands for enzymes whose stereo-specificity with respectto NAD(+) or NADP(+) is not known (cf. EC 1.1.1.50 and EC 1.1.1.213).','A 3-alpha-hydroxysteroid + NAD(P)(+) = a 3-oxosteroid + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.358','2-dehydropantolactone reductase','2-dehydropantoyl-lactone reductase','NULL','The enzyme participates in an alternative pathway for biosynthesis of(R)-pantothenate (vitamin B(5)).This entry covers enzymes whose stereo specificity for NADP(+) is notknown (cf. EC 1.1.1.168 and EC 1.1.1.214).','(R)-pantolactone + NADP(+) = 2-dehydropantolactone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.359','Aldose 1-dehydrogenase (NAD(P)(+))','NULL','NULL','The enzyme from the archaeon Sulfolobus solfataricus shows broadspecificity toward aldoses (D-glucose, D-galactose, D-xylose,L-arabinose, 6-deoxy-D-glucose, D-fucose) and can utilize NAD(+) andNADP(+) with similar catalytic efficiency.It is involved in aldose catabolism via the branched variant of theEntner-Doudoroff pathway.','An aldopyranose + NAD(P)(+) = an aldono-1,5-lactone + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.360','Glucose/galactose 1-dehydrogenase','Dual-specific glucose/galactose dehydrogenase','Zinc.','The enzyme from the archaeon Picrophilus torridus is involved inglucose and galactose catabolism via the nonphosphorylative variantof the Entner-Doudoroff pathway.It shows 20-fold higher activity with NADP(+) compared to NAD(+).The oxidation of D-glucose and D-galactose is catalyzed at acomparable rate (cf. EC 1.1.1.119 and EC 1.1.1.120).','(1) D-glucopyranose + NADP(+) = D-glucono-1,5-lactone + NADPH.(2) D-galactopyranose + NADP(+) = D-galactono-1,5-lactone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.361','Glucose-6-phosphate 3-dehydrogenase','NULL','NULL','The enzyme, found in the bacterium Bacillus subtilis, is involved ina kanosamine biosynthesis pathway.','D-glucose 6-phosphate + NAD(+) = 3-dehydro-D-glucose 6-phosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.362','Aklaviketone reductase','NULL','NULL','The enzyme is involved in the synthesis of the aklavinone aglycone,a common precursor for several anthracycline antibiotics includingaclacinomycins, daunorubicin and doxorubicin.The enzyme from the Gram-negative bacterium Streptomyces sp. C5produces daunomycin.','Aklavinone + NADP(+) = aklaviketone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.363','Glucose-6-phosphate dehydrogenase (NAD(P)(+))','G6PD','NULL','The enzyme catalyzes a step of the pentose phosphate pathway.The enzyme from the Gram-positive bacterium Leuconostoc mesenteroidesprefers NADP(+) while the enzyme from the Gram-negative bacteriumGluconacetobacter xylinus prefers NAD(+).Cf. EC 1.1.1.49.','D-glucose 6-phosphate + NAD(P)(+) = 6-phospho-D-glucono-1,5-lactone +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.364','dTDP-4-dehydro-6-deoxy-alpha-D-gulose 4-ketoreductase','dTDP-4-dehydro-6-deoxygulose reductase','NULL','The enzyme forms an activated deoxy-alpha-D-allose, which isconverted to mycinose after attachment to the aglycones of severalmacrolide antibiotics, including tylosin, chalcomycin,dihydrochalcomycin, and mycinamicin II.','dTDP-6-deoxy-alpha-D-allose + NAD(P)(+) = dTDP-4-dehydro-6-deoxy-alpha-D-allose + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.365','D-galacturonate reductase','GalUR','NULL','The enzyme from plants is involved in ascorbic acid (vitamin C)biosynthesis.The enzyme from the fungus Trichoderma reesei (Hypocrea jecorina) isinvolved in a eukaryotic degradation pathway of D-galacturonate.It is also active with D-glucuronate and glyceraldehyde.Neither enzyme shows any activity with NADH.Formerly EC 1.1.1.n9.','L-galactonate + NADP(+) = D-galacturonate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.366','L-idonate 5-dehydrogenase (NAD(+))','NULL','NULL','Involved in the catabolism of ascorbate (vitamin C) to tartrate.No activity is observed with NADP(+) (cf. EC 1.1.1.264).Formerly EC 1.1.1.n8.','L-idonate + NAD(+) = 5-dehydro-D-gluconate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.367','UDP-2-acetamido-2,6-beta-L-arabino-hexul-4-ose reductase','NULL','NULL','Part of the biosynthesis of UDP-N-acetyl-L-fucosamine.Isolated from the bacteria Pseudomonas aeruginosa and Staphylococcusaureus.','UDP-2-acetamido-2,6-dideoxy-beta-L-talose + NAD(P)(+) = UDP-2-acetamido-2,6-beta-L-arabino-hexul-4-ose + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.368','6-hydroxycyclohex-1-ene-1-carbonyl-CoA dehydrogenase','NULL','NULL','The enzyme participates in the central benzoyl-CoA degradationpathway of some anaerobic bacteria such as Thauera aromatica.','6-hydroxycyclohex-1-ene-1-carbonyl-CoA + NAD(+) = 6-oxocyclohex-1-ene-1-carbonyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.369','D-chiro-inositol 1-dehydrogenase','DCI 1-dehydrogenase','NULL','The enzyme, found in the bacterium Bacillus subtilis, also catalyzesthe reaction of EC 1.1.1.18 and can also use D-glucose and D-xylose.It shows trace activity with D-ribose and D-fructose.It is part of a myo-inositol/D-chiro-inositol degradation pathwayleading to acetyl-CoA.Formerly EC 1.1.1.n6.','1D-chiro-inositol + NAD(+) = 2D-2,3,5/4,6-pentahydroxycyclohexanone +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.370','Scyllo-inositol 2-dehydrogenase (NAD(+))','NULL','NULL','The enzyme, found in the bacterium Bacillus subtilis, has no activitywith NADP(+) (cf. EC 1.1.1.371).It is part of a scyllo-inositol degradation pathway leading toacetyl-CoA.','Scyllo-inositol + NAD(+) = 2,4,6/3,5-pentahydroxycyclohexanone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.371','Scyllo-inositol 2-dehydrogenase (NADP(+))','NULL','NULL','The enzyme, found in the bacterium Bacillus subtilis, has no activitywith NAD(+) (cf. EC 1.1.1.370).','Scyllo-inositol + NADP(+) = 2,4,6/3,5-pentahydroxycyclohexanone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n4','(-)-trans-carveol dehydrogenase','(4R,6S)-carveol dehydrogenase','NULL','Catalyzes the oxidation of carveol to carvone, with a strongstereoselectivity since it efficiently converts only the (6S)-stereoisomers, of which (-)-(4R,6S)-trans-carveol is the bettersubstrate.Displays a broad substrate specificity with a preference forsubstituted cyclohexanols, and does not catalyze the oxidation ofprimary or short chain aliphatic secondary alcohols.Is also able, albeit more slowly, to oxidize limonene-1,2-diol into1-hydroxy-2-oxolimonene.','(1) (-)-(4R,6S)-trans-carveol + NAD(+) = (-)-(4R)-carvone + NADH.(2) (+)-(4S,6S)-cis-carveol + NAD(+) = (+)-(4S)-carvone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n5','3-methylmalate dehydrogenase','NULL','Manganese or magnesium.','The enzyme from Methanocaldococcus jannaschii also catalyzes thereaction of EC 1.1.1.85 and EC 1.1.1.83.','(2R,3S)-3-methylmalate + NAD(+) = 2-oxobutanoate + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n11','Succinic semialdehyde reductase','NULL','NULL','NULL','4-hydroxybutanoate + NADP(+) = succinate semialdehyde + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n12','(3R)-hydroxyacyl-CoA dehydrogenase','NULL','NULL','NULL','(R)-3-hydroxyacyl-CoA + NAD(+) = 3-oxoacyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n13','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.1.n14','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.2.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.2.2','Mannitol dehydrogenase (cytochrome)','Polyol dehydrogenase','NULL','Acts on polyols with a D-lyxo configuration, such as D-mannitol andD-sorbitol.','D-mannitol + ferricytochrome c = D-fructose + ferrocytochrome c + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.2.3','L-lactate dehydrogenase (cytochrome)','Cytochrome b2','FMN; Protoheme IX.','NULL','(S)-lactate + 2 ferricytochrome c = pyruvate + 2 ferrocytochrome c +2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.2.4','D-lactate dehydrogenase (cytochrome)','D-lactate ferricytochrome c oxidoreductase','FAD.','NULL','(R)-lactate + 2 ferricytochrome c = pyruvate + 2 ferrocytochrome c +2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.2.5','D-lactate dehydrogenase (cytochrome c-553)','NULL','NULL','From Desulfovibrio vulgaris.','(R)-lactate + 2 ferricytochrome c-553 = pyruvate + 2 ferrocytochromec-553 + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.2.6','Polyvinyl alcohol dehydrogenase (cytochrome)','PVA dehydrogenase','Pyrroloquinoline quinone.','The enzyme is involved in bacterial polyvinyl alcohol degradation.Some Gram-negative bacteria degrade polyvinyl alcohol by importing itinto the periplasmic space, where it is oxidized by polyvinyl alcoholdehydrogenase, an enzyme that is coupled to the respiratory chain viacytochrome c.Formerly EC 1.1.99.23.','Polyvinyl alcohol + ferricytochrome c = oxidized polyvinyl alcohol +ferrocytochrome c + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.2.7','Methanol dehydrogenase (cytochrome c)','MDH','Pyrroloquinoline quinone.','A periplasmic quinoprotein alcohol dehydrogenase that only occurs inmethylotrophic bacteria.It uses the novel specific cytochrome c(L) as acceptor.Acts on a wide range of primary alcohols, including ethanol,duodecanol, chloroethanol, cinnamyl alcohol, and also formaldehyde.Activity is stimulated by ammonia or methylamine.It is usually assayed with phenazine methosulfate.Like all other quinoprotein alcohol dehydrogenases it has an 8-bladedpropeller structure, a calcium ion bound to the PQQ in the activesite and an unusual disulfide ring structure in close proximity tothe PQQ.It differs from EC 1.1.2.8, alcohol dehydrogenase (cytochrome c),in having a high affinity for methanol and in having a secondessential small subunit (no known function).Formerly EC 1.1.99.8.','A primary alcohol + 2 cytochrome c(L)  = an aldehyde + 2 reducedcytochrome c(L) + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.2.8','Alcohol dehydrogenase (cytochrome c)','NULL','Pyrroloquinoline quinone.','A periplasmic PQQ-containing quinoprotein.Occurs in Pseudomonas and Rhodopseudomonas.The enzyme from Pseudomonas aeruginosa uses a specific induciblecytochrome c(550) as electron acceptor.Acts on a wide range of primary and secondary alcohols, but notmethanol.It has a homodimeric structure (contrasting with the heterotetramericstructure of EC 1.1.2.7, methanol dehydrogenase (cytochrome c)).It is routinely assayed with phenazine methosulfate as electronacceptor.Activity is stimulated by ammonia or amines.Like all other quinoprotein alcohol dehydrogenases it has an 8-bladedpropeller structure, a calcium ion bound to the PQQ in the activesite and an unusual disulfide ring structure in close proximity tothe PQQ.Formerly EC 1.1.99.8.','A primary alcohol + 2 cytochrome c = an aldehyde + 2 reduced cytochrome c+ 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.3','Malate oxidase','Malic oxidase','FAD.','NULL','(S)-malate + O(2) = oxaloacetate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.4','Glucose oxidase','Beta-D-glucose:oxygen 1-oxido-reductase','FAD.','NULL','Beta-D-glucose + O(2) = D-glucono-1,5-lactone + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.5','Hexose oxidase','NULL','Copper.','Also oxidizes D-galactose, D-mannose, maltose, lactose andcellobiose.','Beta-D-glucose + O(2) = D-glucono-1,5-lactone + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.6','Cholesterol oxidase','Cholesterol-O(2) oxidoreductase','FAD.','Cholesterol oxidases are secreted bacterial bifunctional enzymes thatcatalyze the first two steps in the degradation of cholesterol.The enzyme catalyzes the oxidation of the 3-beta-hydroxyl group to aketo group, and the isomerization of the double bond in the oxidizedsteroid ring system from the Delta(5) position to the Delta(6)position (cf. EC 5.3.3.1).','Cholesterol + O(2) = cholest-5-en-3-one + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.7','Aryl-alcohol oxidase','Veratryl alcohol oxidase','NULL','Oxidizes many primary alcohols containing an aromatic ring; bestsubstrates are (2-naphthyl)methanol and 3-methoxybenzyl alcohol.','An aromatic primary alcohol + O(2) = an aromatic aldehyde + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.8','L-gulonolactone oxidase','GLO','FAD.','The product spontaneously isomerizes to L-ascorbate.While most higher animals can synthesize ascorbic acid, primates andguinea pigs cannot.','L-gulono-1,4-lactone + O(2) = L-ascorbate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.9','Galactose oxidase','Beta-galactose oxidase','Copper.','NULL','D-galactose + O(2) = D-galacto-hexodialdose + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.10','Pyranose oxidase','Glucose 2-oxidase','FAD.','Also oxidizes D-xylose, L-sorbose and D-glucono-1,5-lactone, whichhave the same ring conformation and configuration at C-2, C-3 andC-4.','D-glucose + O(2) = 2-dehydro-D-glucose + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.11','L-sorbose oxidase','NULL','NULL','Also acts on D-glucose, D-galactose and D-xylose, but not onD-fructose.2,6-dichloroindophenol can act as acceptor.','L-sorbose + O(2) = 5-dehydro-D-fructose + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.12','Pyridoxine 4-oxidase','Pyridoxin 4-oxidase','FAD.','2,6-dichloroindophenol can act as acceptor.','Pyridoxine + O(2) = pyridoxal + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.13','Alcohol oxidase','AOX','FAD.','Acts on lower primary alcohols and unsaturated alcohols but branched-chain and secondary alcohols are not attacked.','A primary alcohol + O(2) = an aldehyde + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.14','Catechol oxidase (dimerizing)','NULL','NULL','NULL','4 catechol + 3 O(2) = 2 dibenzo[1,4]dioxin-2,3-dione + 6 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.15','(S)-2-hydroxy-acid oxidase','Glycolate oxidase','FMN.','Exists as two major isoenzymes; the A form preferentially oxidizesshort-chain aliphatic hydroxy acids; the B form preferentiallyoxidizes long-chain and aromatic hydroxy acids.The rat isoenzyme B also acts as EC 1.4.3.2.Formerly EC 1.1.3.1.','(S)-2-hydroxy acid + O(2) = 2-oxo acid + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.16','Ecdysone oxidase','NULL','NULL','2,6-dichloroindophenol can act as acceptor.','Ecdysone + O(2) = 3-dehydroecdysone + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.17','Choline oxidase','NULL','FAD.','In many bacteria, plants and animals, the osmoprotectant betaine issynthesized using different enzymes to catalyze the conversion of (1)choline into betaine aldehyde and (2) betaine aldehyde into betaine.In plants, the first reaction is catalyzed by EC 1.14.15.7, whereasin animals and many bacteria, it is catalyzed by either EC 1.1.99.1or EC 1.1.3.17.The enzyme involved in the second step, EC 1.2.1.8, appears to be thesame in those plants, animals and bacteria that use two separateenzymes.','Choline + 2 O(2) + H(2)O = betaine + 2 H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.18','Secondary-alcohol oxidase','NULL','Iron.','Acts on secondary alcohols with five or more carbons, and polyvinylalcohols with molecular mass over 300 Da.','A secondary alcohol + O(2) = a ketone + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.19','4-hydroxymandelate oxidase','NULL','FAD; Manganese.','NULL','(S)-2-hydroxy-2-(4-hydroxyphenyl)acetate + O(2) = 4-hydroxybenzaldehyde +CO(2) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.20','Long-chain-alcohol oxidase','NULL','NULL','Oxidizes long-chain fatty alcohols; best substrate is dodecylalcohol.','A long-chain alcohol + O(2) = a long-chain aldehyde + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.21','Glycerol-3-phosphate oxidase','Alpha-glycerophosphate oxidase','FAD.','NULL','sn-glycerol 3-phosphate + O(2) = glycerone phosphate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.22','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.23','Thiamine oxidase','Thiamine dehydrogenase','FAD.','The product differs from thiamine in replacement of -CH(2)-CH(2)-OHby -CH(2)-COOH; the two-step oxidation proceeds without the releaseof the intermediate aldehyde from the enzyme.','Thiamine + 2 O(2) + H(2)O = thiamine acetic acid + 2 H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.24','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.25','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.26','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.27','Hydroxyphytanate oxidase','NULL','NULL','NULL','L-2-hydroxyphytanate + O(2) = 2-oxophytanate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.28','Nucleoside oxidase','NULL','NULL','Other purine and pyrimidine nucleosides (as well as2-deoxynucleosides) are substrates, but ribose and nucleotides arenot substrates.The overall reaction takes place in two separate steps:(1) 2 inosine + O(2) = 2 5-dehydroinosine + 2 H(2)O.(2) 2 5-dehydroinosine + O(2) = 2 9-riburonosylhypoxanthine +2 H(2)O.The 5-dehydro nucleoside is being released from the enzyme to serveas substrate for the second reaction.Differs from EC 1.1.3.39 as it produces water rather than hydrogenperoxide.','Inosine + O(2) = 9-riburonosylhypoxanthine + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.29','N-acylhexosamine oxidase','NULL','NULL','Also acts on N-glycolylglucosamine, N-acetylgalactosamine and, moreslowly, on N-acetylmannosamine.','N-acetyl-D-glucosamine + O(2) = N-acetyl-D-glucosaminate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.30','Polyvinyl-alcohol oxidase','PVA oxidase','NULL','NULL','Polyvinyl alcohol + O(2) = oxidized polyvinyl alcohol + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.31','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.32','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.33','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.34','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.35','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.36','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.37','D-arabinono-1,4-lactone oxidase','D-arabinono-gamma-lactone oxidase','FAD.','NULL','D-arabinono-1,4-lactone + O(2) = dehydro-D-arabinono-1,4-lactone +H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.38','Vanillyl-alcohol oxidase','4-hydroxy-2-methoxybenzyl alcohol oxidase','FAD.','Converts a wide range of 4-hydroxybenzyl alcohols and4-hydroxybenzylamines into the corresponding aldehydes.The allyl group of 4-allylphenols is also converted into the-CH=CH-CH(2)OH group.','Vanillyl alcohol + O(2) = vanillin + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.39','Nucleoside oxidase (H(2)O(2)-forming)','NULL','FAD; Heme.','Other purine and pyrimidine nucleosides (as well as2-deoxynucleosides and arabinosides) are substrates, but ribose andnucleotides are not substrates.The 5-dehydro nucleoside is being released from the enzyme to serveas substrate for the second reaction.Differs from EC 1.1.3.28 as it produces hydrogen peroxide rather thanwater.','Adenosine + 2 O(2) + H(2)O = 9-riburonosyladenine + 2 H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.40','D-mannitol oxidase','D-arabinitol oxidase','NULL','Also catalyzes the oxidation of D-arabinitol and, to a lesser extent,D-glucitol (sorbitol), whereas L-arabinitol is not a good substrate.The enzyme from the snails Helix aspersa and Arion ater is found in aspecialized tubular organelle that has been termed the mannosome.','Mannitol + O(2) = mannose + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.41','Alditol oxidase','Xylitol oxidase','FAD.','While xylitol (five carbons) and sorbitol (6 carbons) are thepreferred substrates, other alditols, including L-threitol (fourcarbons), D-arabitol (five carbons), D-galactitol (six carbons) andD-mannitol (six carbons) can also act as substrates, but more slowly.','An alditol + O(2) = an aldose + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.42','Prosolanapyrone-II oxidase','Prosolanapyrone II oxidase/prosolanapyrone III cycloisomerase','NULL','The enzyme is involved in the biosynthesis of the phytotoxinsolanapyrone by some fungi.The bifunctional enzyme catalyzes the oxidation of prosolanapyrone IIand the subsequent Diels Alder cycloisomerization of the productprosolanapyrone III to (-)-solanapyrone A (cf. EC 5.5.1.20).','Prosolanapyrone II + O(2) = prosolanapyrone III + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.43','Paromamine 6-oxidase','NULL','FAD.','Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including kanamycin, butirosin, neomycinand ribostamycin.Works in combination with EC 2.6.1.93 to replace the 6-hydroxy groupof paromamine with an amino group.The enzyme from the bacterium Streptomyces fradiae also catalyzesEC 1.1.3.44.','Paromamine + O(2) = 6-dehydroparomamine + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.44','6-hydroxyneomycin C oxidase','NULL','FAD.','Involved in the biosynthetic pathway of aminoglycoside antibiotics ofthe neomycin family.Works in combination with EC 2.6.1.95 to replace the 6-hydroxygroup of 6-hydroxyneomycin C with an amino group.Also catalyzes EC 1.1.3.43.','6-deamino-6-hydroxyneomycin C + O(2) = 6-deamino-6-oxoneomycin C + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.3.45','Aclacinomycin-N oxidase','Aclacinomycin oxidoreductase','FAD.','This bifunctional enzyme is a secreted flavin-dependent enzyme thatis involved in the modification of the terminal sugar residues in thebiosynthesis of aclacinomycins.The enzyme utilizes the same active site to catalyze the oxidation ofthe rhodinose moiety of aclacinomycin N to the cinerulose A moiety ofaclacinomycin A and the oxidation of the latter to the L-aculosemoiety of aclacinomycin Y (cf. EC 1.3.3.14).','Aclacinomycin N + O(2) = aclacinomycin A + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.4.1','Vitamin-K-epoxide reductase (warfarin-sensitive)','Phylloquinone epoxide reductase','NULL','In the reverse reaction, vitamin K 2,3-epoxide is reduced to vitaminK and possibly to vitamin K hydroquinone by 1,4-dithioerythritol,which is oxidized to a disulfide; some other dithiols and4-butanethiol can also act.Inhibited strongly by warfarin (cf. EC 1.1.4.2).','2-methyl-3-phytyl-1,4-naphthoquinone + oxidized dithiothreitol = 2,3-epoxy-2,3-dihydro-2-methyl-3-phytyl-1,4-naphthoquinone + 1,4-dithiothreitol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.4.2','Vitamin-K-epoxide reductase (warfarin-insensitive)','Vitamin K 2,3-epoxide reductase','NULL','In the reverse reaction, vitamin K 2,3-epoxide is reduced to3-hydroxy- (and 2-hydroxy-) vitamin K by 1,4-dithioerythritol, whichis oxidized to a disulfide.Not inhibited by warfarin (cf. EC 1.1.4.1).','3-hydroxy-2-methyl-3-phytyl-2,3-dihydronaphthoquinone + oxidizeddithiothreitol = 2,3-epoxy-2,3-dihydro-2-methyl-3-phytyl-1,4-naphthoquinone + 1,4-dithiothreitol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.1','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.2','Quinoprotein glucose dehydrogenase (PQQ, quinone)','D-glucose:(pyrroloquinoline-quinone) 1-oxidoreductase','Calcium or magnesium; Pyrroloquinoline quinone.','Integral membrane protein containing PQQ as prosthetic group.It also contains bound ubiquinone and Mg(2+) or Ca(2+).Electron acceptor is membrane ubiquinone but usually assayed withphenazine methosulfate.Like in all other quinoprotein alcohol dehydrogenases the catalyticdomain has an 8-bladed propeller structure.It occurs in a wide range of bacteria.Catalyzes a direct oxidation of the pyranose form of D-glucose to thelactone and thence to D-gluconate in the periplasm.Oxidizes other monosaccharides including the pyranose forms ofpentoses.Formerly EC 1.1.99.17.','D-glucose + ubiquinone = D-glucono-1,5-lactone + ubiquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.3','Glycerol-3-phosphate dehydrogenase','FAD-dependent glycerol-3-phosphate dehydrogenase','Flavin.','An essential membrane enzyme, functioning at the central junction ofglycolysis, respiration and phospholipid biosynthesis.In bacteria, the enzyme is localized to the cytoplasmic membrane,while in eukaryotes it is tightly bound to the outer surface of theinner mitochondrial membrane.In eukaryotes, this enzyme, together with the cytosolic enzymeEC 1.1.1.8 forms the glycerol-3-phosphate shuttle by which NADHproduced in the cytosol, primarily from glycolysis, can be reoxidizedto NAD(+) by the mitochondrial electron-transport chain.This shuttle plays a critical role in transferring reducingequivalents from cytosolic NADH into the mitochondrial matrix.Insect flight muscle uses only CoQ(10) as the physiological quinonewhereas hamster and rat mitochondria use mainly CoQ(9).The enzyme is activated by calcium.Formerly EC 1.1.2.1 and EC 1.1.99.5.','sn-glycerol 3-phosphate + a quinone = glycerone phosphate + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.4','Malate dehydrogenase (quinone)','FAD-dependent malate dehydrogenase','FAD.','Vitamin K and several other quinones can act as acceptors.Different from EC 1.1.1.37, EC 1.1.1.82 and EC 1.1.1.299.Formerly EC 1.1.99.16.','(S)-malate + a quinone = oxaloacetate + reduced quinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.5','Alcohol dehydrogenase (quinone)','PQQ alcohol dehydrogenase','Pyrroloquinoline quinone.','Only described in acetic acid bacteria where it is involved in aceticacid production.Associated with membrane.Electron acceptor is membrane ubiquinone.A model structure suggests that, like all other quinoprotein alcoholdehydrogenases, the catalytic subunit has an 8-bladed propellerstructure, a calcium ion bound to the PQQ in the active site and anunusual disulfide ring structure in close proximity to the PQQ; thecatalytic subunit also has a heme c in the C-terminal domain.The enzyme has two additional subunits, one of which contains threemolecules of heme c.It does not require amines for activation.It has a restricted substrate specificity, oxidizing a few primaryalcohols (C2 to C6), but not methanol, secondary alcohols and somealdehydes.It is assayed with phenazine methosulfate or with ferricyanide.','Ethanol + ubiquinone = acetaldehyde + ubiquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.6','Formate dehydrogenase-N','Formate dehydrogenase N','Heme; Iron-sulfur; Molybdopterin guanine dinucleotide.','Formate dehydrogenase-N oxidizes formate in the periplasm,transferring electrons via the menaquinone pool in the cytoplasmicmembrane to a dissimilatory nitrate reductase (EC 1.7.5.1), whichtransfers electrons to nitrate in the cytoplasm.The system generates proton motive force under anaerobic conditions.','Formate + a quinone = CO(2) + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.7','Cyclic alcohol dehydrogenase (quinone)','Cyclic alcohol dehydrogenase','NULL','This enzyme oxidizes a wide variety of cyclic alcohols.Some minor enzyme activity is found with aliphatic secondary alcoholsand sugar alcohols, but not primary alcohols.The enzyme is unable to catalyze the reverse reaction of cyclicketones or aldehydes to cyclic alcohols.This enzyme differs from EC 1.1.5.5, which shows activity withethanol.','A cyclic alcohol + a quinone = a cyclic ketone + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.8','Quinate dehydrogenase (quinone)','NAD(P)(+)-independent quinate dehydrogenase','Pyrroloquinoline quinone.','The enzyme is membrane-bound.Does not use NAD(P)(+) as acceptor.Formerly EC 1.1.99.25.','Quinate + quinone = 3-dehydroquinate + quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.9','Glucose 1-dehydrogenase (FAD, quinone)','FAD-dependent glucose dehydrogenase','FAD.','2,6-dichloroindophenol can act as acceptor.Cf. EC 1.1.5.2.Formerly EC 1.1.99.10.','D-glucose + a quinone = D-glucono-1,5-lactone + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.10','D-2-hydroxyacid dehydrogenase (quinone)','(R)-2-hydroxy acid dehydrogenase','FAD.','(R)-lactate, (R)-malate and meso-tartrate are good substrates.Ubiquinone-1 and the dye 2,6-dichloroindophenol can act as acceptors;NAD(+) and NADP(+) are not acceptors.','(R)-2-hydroxyacid + a quinone = 2-oxoacid + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.5.n1','Quinoprotein inositol dehydrogenase','Membrane-bound quinoprotein inositol dehydrogenase','Pyrroloquinoline quinone.','NULL','Myo-inositol + pyrroloquinoline quinone = 2,4,6/3,5-pentahydroxycyclohexanone + pyrroloquinoline quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.9.1','Alcohol dehydrogenase (azurin)','QHEDH','Heme; Pyrroloquinoline quinone.','Occurs in Comamonas and Pseudomonas.Does not require an amine activator.Oxidizes a wide range of primary and secondary alcohols, and alsoaldehydes and large substrates such as sterols; methanol is not asubstrate.Usually assayed with phenazine methosulfate or ferricyanide.Like all other quinoprotein alcohol dehydrogenases it has an 8-bladedpropeller structure, a calcium ion bound to the PQQ in the activesite and an unusual disulfide ring structure in close proximity tothe PQQ.Formerly EC 1.1.98.1.','A primary alcohol + azurin = an aldehyde + reduced azurin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.98.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.98.2','Glucose-6-phosphate dehydrogenase (coenzyme-F420)','Coenzyme F420-dependent glucose-6-phosphate dehydrogenase','NULL','The enzyme is very specific for D-glucose 6-phosphate.No activity with NAD(+), NADP(+), flavin adenine dinucleotide andflavin mononucleotide.Formerly EC 1.1.99.34.','D-glucose 6-phosphate + oxidized coenzyme F420 = D-glucono-1,5-lactone6-phosphate + reduced coenzyme F420.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.98.3','Decaprenylphospho-beta-D-ribofuranose 2-oxidase','Decaprenylphosphoryl-beta-D-ribofuranose 2-epimerase','NULL','The enzyme, isolated from the bacterium Mycobacterium smegmatis,is involved, along with EC 1.1.1.333 in the epimerization oftrans,octacis-decaprenylphospho-beta-D-ribofuranose to trans,octacis-decaprenylphospho-beta-D-arabinoofuranose, the arabinosyl donor forthe biosynthesis of mycobacterial cell wall arabinan polymers.','Trans,octacis-decaprenylphospho-beta-D-ribofuranose + FAD =trans,octacis-decaprenylphospho-beta-D-erythro-pentofuranosid-2-ulose +FADH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.98.4','F420H(2):quinone oxidoreductase','NULL','FAD; Iron-sulfur.','NULL','A quinol + oxidized coenzyme F420 = a quinone + reduced coenzyme F420.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.98.5','Secondary-alcohol dehydrogenase (coenzyme-F420)','F420-dependent alcohol dehydrogenase','NULL','The enzyme isolated from the methanogenic archaea Methanogeniumliminatans catalyzes the reversible oxidation of various secondaryand cyclic alcohols to the corresponding ketones.','R-CHOH-R+ oxidized coenzyme F420 = R-CO-R+ reduced coenzyme F420.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.1','Choline dehydrogenase','Choline oxidase','Pyrroloquinoline quinone.','In many bacteria, plants and animals, the osmoprotectant betaine issynthesized using different enzymes to catalyze the conversion of (1)choline into betaine aldehyde and (2) betaine aldehyde into betaine.In plants, the first reaction is catalyzed by EC 1.14.15.7 whereas inanimals and many bacteria, it is catalyzed by either EC 1.1.99.1 orEC 1.1.3.17.The enzyme involved in the second step, EC 1.2.1.8, appears to be thesame in plants, animals and bacteria.','Choline + acceptor = betaine aldehyde + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.2','L-2-hydroxyglutarate dehydrogenase','Alpha-hydroxyglutarate dehydrogenase','FAD.','NULL','(S)-2-hydroxyglutarate + acceptor = 2-oxoglutarate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.3','Gluconate 2-dehydrogenase (acceptor)','NULL','FAD.','NULL','D-gluconate + acceptor = 2-dehydro-D-gluconate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.4','Dehydrogluconate dehydrogenase','Ketogluconate dehydrogenase','Flavoprotein.','NULL','2-dehydro-D-gluconate + acceptor = 2,5-didehydro-D-gluconate + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.6','D-lactate dehydrogenase (acceptor)','NULL','FAD; Zinc.','The enzyme from the archaeon Archaeoglobus fulgidus cannot utilizeNAD(+), cytochrome C, methylene blue or dimethylnaphthoquinone asacceptors.In vitro it is active with artificial electron acceptors such as 2,6-dichlorophenolindophenol, but the physiological acceptor is not yetknown.','(R)-lactate + acceptor = pyruvate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.7','Lactate--malate transhydrogenase','NULL','NULL','Catalyzes hydrogen transfer from C(3) or C(4) (S)-2-hydroxy acids to2-oxo acids.Contains tightly bound nicotinamide nucleotide in its active center.This prosthetic group cannot be removed without denaturation of theprotein.','(S)-lactate + oxaloacetate = pyruvate + malate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.9','Pyridoxine 5-dehydrogenase','Pyridoxol 5-dehydrogenase','FAD.','NULL','Pyridoxine + acceptor = isopyridoxal + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.11','Fructose 5-dehydrogenase','D-fructose dehydrogenase','NULL','2,6-dichloroindophenol can act as acceptor.','D-fructose + acceptor = 5-dehydro-D-fructose + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.12','Sorbose dehydrogenase','NULL','NULL','2,6-dichloroindophenol can act as acceptor.','L-sorbose + acceptor = 5-dehydro-D-fructose + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.13','Glucoside 3-dehydrogenase','D-aldohexopyranoside dehydrogenase','FAD.','The enzyme acts on D-glucose, D-galactose, D-glucosides andD-galactosides, but D-glucosides react more rapidly thanD-galactosides.','Sucrose + acceptor = 3-dehydro-alpha-D-glucosyl-beta-D-fructofuranoside +reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.14','Glycolate dehydrogenase','Glycolate oxidoreductase','NULL','Also acts on (R)-lactate.2,6-dichloroindophenol and phenazine methosulfate can act asacceptors.','Glycolate + acceptor = glyoxylate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.15','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.16','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.17','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.18','Cellobiose dehydrogenase (acceptor)','CBOR','FAD; Heme.','2,6-dichloroindophenol can act as acceptor.Also acts, more slowly, on cello-oligosaccharides, lactose andD-glucosyl-1,4-beta-D-mannose.Includes EC 1.1.5.1, which is now known to be a proteolytic productof this enzyme.The enzyme from the white rot fungus Phanerochaete chrysosporium isunusual in having two redoxin domains, one containing a flavin andthe other a protoheme group.It transfers reducing equivalents from cellobiose to two types ofredox acceptor: two-electron oxidants, including redox dyes,benzoquinones and molecular oxygen and one-electron oxidants,including semiquinone species, Fe(2+) complexes and the modelacceptor cytochrome c.Formerly EC 1.1.3.25.','Cellobiose + acceptor = cellobiono-1,5-lactone + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.19','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.20','Alkan-1-ol dehydrogenase (acceptor)','Polyethylene glycol dehydrogenase','Pyrroloquinoline quinone.','Acts on C(3)-C(16) linear-chain saturated primary alcohols, C(4)-C(7)aldehydes and on non-ionic surfactants containing polyethylene glycolresidues, such as Tween 40 and 60, but not on methanol and only veryslowly on ethanol.2,6-dichloroindophenol can act as acceptor.','Primary alcohol + acceptor = aldehyde + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.21','D-sorbitol dehydrogenase (acceptor)','NULL','FAD.','NULL','D-sorbitol + acceptor = L-sorbose + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.22','Glycerol dehydrogenase (acceptor)','NULL','Pyrroloquinoline quinone.','Also acts, more slowly, on a number of other polyols includingD-sorbitol, D-arabinitol, meso-erythritol, adonitol and propyleneglycol.','Glycerol + acceptor = glycerone + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.23','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.24','Hydroxyacid-oxoacid transhydrogenase','NULL','NULL','NULL','(1) (S)-3-hydroxybutanoate + 2-oxoglutarate = acetoacetate + (R)-2-hydroxyglutarate.(2) 4-hydroxybutanoate + 2-oxoglutarate = succinic semialdehyde + (R)-2-hydroxyglutarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.25','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.26','3-hydroxycyclohexanone dehydrogenase','NULL','NULL','2,6-dichloroindophenol and methylene blue can act as acceptors.','3-hydroxycyclohexanone + acceptor = cyclohexane-1,3-dione + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.27','(R)-pantolactone dehydrogenase (flavin)','(R)-pantoyllactone dehydrogenase (flavin)','FMN.','High specificity for (R)-pantolactone.Phenazine methosulfate (PMS) can act as acceptor.The enzyme has been studied in Nocardia asteroides and shown to bemembrane-bound and induced by 1,2-propanediol.','(R)-pantolactone + acceptor = 2-dehydropantolactone + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.28','Glucose-fructose oxidoreductase','NULL','NULL','D-mannose, D-xylose, D-galactose, 2-deoxy-D-glucose and L-arabinosewill function as aldose substrates, but with low affinities.The ketose substrate must be in the open-chain form.The apparent affinity for fructose is low, because little of thefructose substrate is in the open-chain form.Xylulose and glycerone (dihydroxyacetone) will replace fructose,but they are poor substrates.The enzyme from Zymomonas mobilis contains tightly bound NADP(+).','D-glucose + D-fructose = D-gluconolactone + D-glucitol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.29','Pyranose dehydrogenase (acceptor)','PDH','FAD.','A number of aldoses and ketoses in pyranose form, as well asglycosides, gluco-oligosaccharides, sucrose and lactose can act as adonor.1,4-benzoquinone or ferricenium ion (ferrocene oxidized by removal ofone electron) can serve as acceptor.Unlike EC 1.1.3.10, this fungal enzyme does not interact with O(2)and exhibits extremely broad substrate tolerance with variableregioselectivity (C-3, C-2 or C-3 + C-2 or C-3 + C-4) for(di)oxidation of different sugars.D-glucose is exclusively or preferentially oxidized at C-3 (dependingon the enzyme source), but can also be oxidized at C-2 + C-3.The enzyme also acts on 1-','(1) Pyranose + acceptor = pyranos-2-ulose + reduced acceptor.(2) Pyranose + acceptor = pyranos-3-ulose + reduced acceptor.(3) Pyranose + acceptor = pyranos-2,3-diulose + reduced acceptor.(4) A pyranoside + acceptor = a pyranosid-3-ulose + reduced acceptor.(5) A pyranoside + acceptor = a pyranosid-3,4-diulose + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.30','2-oxo-acid reductase','(2R)-hydroxycarboxylate-viologen-oxidoreductase','Iron-sulfur; Molybdenum cofactor.','Has broad substrate specificity, with 2-oxo-monocarboxylates and2-oxo-dicarboxylates acting as substrates.Branching in a substrate at the C-3 position results in loss ofactivity.The enzyme from Proteus sp. is inactivated by oxygen.','A (2R)-hydroxy-carboxylate + acceptor = a 2-oxo-carboxylate + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.31','(S)-mandelate dehydrogenase','L(+)-mandelate dehydrogenase','NULL','While all enzymes of this family oxidize the (S)-enantiomer of analpha-hydroxy acid to an alpha-oxo acid, the ultimate oxidant(oxygen, intramolecular heme or some other acceptor) depends on theparticular enzyme.Transfers the electron pair from FMNH(2) to a component of theelectron transport chain, most probably ubiquinone.It is part of a metabolic pathway in Pseudomonads that allows theseorganisms to utilize mandelic acid, derivatized from the common soilmetabolite amygdalin, as the sole source of carbon and energy.Has a large active-site pocket and preferentially binds substrateswith longer sidechains, e.g. 2-hydroxyoctanoate rather than2-hydroxybutyrate.It also prefers substrates that, like (S)-mandelate, have betaunsaturation, e.g. (indol-3-yl)glycolate compared with (indol-3-yl)lactate.Esters of mandelate, such as methyl (S)-mandelate, are alsosubstrates.','(S)-2-hydroxy-2-phenylacetate + acceptor = 2-oxo-2-phenylacetate +reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.32','L-sorbose 1-dehydrogenase','SDH','NULL','The product, 1-dehydro-L-sorbose, is an intermediate in bacterial2-keto-L-gulonic-acid formation.The activity of this membrane-bound enzyme is stimulated by Fe(III)or Co(2+) but is inhibited by Cu(2+).Highly specific for L-sorbose as other sugars, such as glucose,mannitol and sorbitol, are not substrates.Phenazine methosulfate and DCIP can act as artificial acceptors.','L-sorbose + acceptor = 1-dehydro-L-sorbose + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.33','Formate dehydrogenase (acceptor)','Formate dehydrogenase H','NULL','Formate dehydrogenase H is a cytoplasmic enzyme that oxidizes formatewithout oxygen transfer, transferring electrons to a hydrogenase.The two enzymes form the formate-hydrogen lyase complex.The enzyme contains a selenocysteine residue.','Formate + acceptor = CO(2) + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.34','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.35','Soluble quinoprotein glucose dehydrogenase','Glucose dehydrogenase (PQQ-dependent)','NULL','Soluble periplasmic enzyme containing PQQ as prosthetic group, boundto a calcium ion.Electron acceptor is not known.It is assayed with Wursters Blue or phenazine methosulfate.It has negligible sequence or structure similarity to otherquinoproteins.It catalyzes an exceptionally high rate of oxidation of a wide rangeof aldose sugars, including D-glucose, galactose, arabinose andxylose, and also the disaccharides lactose, cellobiose and maltose.It has been described only in Acinetobacter calcoaceticus.','D-glucose + acceptor = D-glucono-1,5-lactone + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.36','Alcohol dehydrogenase (nicotinoprotein)','NDMA-dependent alcohol dehydrogenase','Zinc.','Nicotinoprotein alcohol dehydrogenases are unique medium-chaindehydrogenases/reductases (MDR) alcohol dehydrogenases that have atightly bound NAD(+)/NADH cofactor that does not dissociate duringthe catalytic process.Instead, the cofactor is regenerated by a second substrate orelectron carrier.While the in vivo electron acceptor is not known, N,N-dimethyl-4-nitrosoaniline (NDMA) can serve this function in vitro.The enzyme from the Gram-positive bacterium Amycolatopsis methanolicacan accept many primary alcohols as substrates, includingbenzylalcohol.','Ethanol + acceptor = acetaldehyde + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.37','Methanol dehydrogenase (nicotinoprotein)','NDMA-dependent methanol dehydrogenase','Magnesium; Zinc.','Nicotinoprotein methanol dehydrogenases have a tightly boundNADP+/NADPH cofactor that does not dissociate during the catalyticprocess.Instead, the cofactor is regenerated by a second substrate orelectron carrier.While the in vivo electron acceptor is not known, N,N-dimethyl-4-nitrosoaniline (NDMA) can serve this function in vitro.The enzyme has been detected in several Gram-positive methylotrophicbacteria, including Amycolatopsis methanolica, Rhodococcusrhodochrous and Rhodococcus erythropolis.These enzymes are decameric, and possess a 5-fold symmetry.Some of the enzymes can also dismutate formaldehyde to methanol andformate.','Methanol + acceptor = formaldehyde + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.38','2-deoxy-scyllo-inosamine dehydrogenase (AdoMet-dependent)','NULL','Iron-sulfur.','Involved in the biosynthetic pathway of the aminoglycosideantibiotics of the butirosin family.The enzyme from Bacillus circulans was shown to be a radicalS-adenosyl-L-methionine (SAM) enzyme.Cf. EC 1.1.1.329.','2-deoxy-scyllo-inosamine + S-adenosyl-L-methionine = 3-amino-2,3-dideoxy-scyllo-inosose + 5-deoxyadenosine + L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.1.99.39','D-2-hydroxyglutarate dehydrogenase','NULL','NULL','The enzyme, which has been characterized from the plant Arabidopsisthaliana, is highly specific for (R)-2-hydroxyglutarate.It has low activity with (R)-lactate, (R)-2-hydroxybutyrate and meso-tartrate, and no activity with the (S) isomers.','(R)-2-hydroxyglutarate + acceptor = 2-oxoglutarate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.2','Formate dehydrogenase','NULL','Flavin; Iron-sulfur; Molybdenum.','The enzyme from most aerobic organisms is devoid of redox-activecenters but that from the proteobacterium Methylosinus trichosporiumcontains iron-sulfur centers, flavin and a molybdenum center.Together with EC 1.12.1.2, forms a system previously known as formatehydrogenlyase.','Formate + NAD(+) = CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.3','Aldehyde dehydrogenase (NAD(+))','NULL','NULL','Wide specificity, including oxidation of D-glucuronolactone toD-glucarate.Formerly EC 1.1.1.70.','An aldehyde + NAD(+) + H(2)O = a carboxylate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.4','Aldehyde dehydrogenase (NADP(+))','NULL','NULL','NULL','An aldehyde + NADP(+) + H(2)O = a carboxylate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.5','Aldehyde dehydrogenase (NAD(P)(+))','NULL','NULL','NULL','An aldehyde + NAD(P)(+) + H(2)O = a carboxylate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.6','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.7','Benzaldehyde dehydrogenase (NADP(+))','NULL','NULL','NULL','Benzaldehyde + NADP(+) + H(2)O = benzoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.8','Betaine-aldehyde dehydrogenase','BADH','NULL','In many bacteria, plants and animals, the osmoprotectant betaine issynthesized in two steps: (1) choline to betaine aldehyde and (2)betaine aldehyde to betaine.This enzyme is involved in the second step and appears to be the samein plants, animals and bacteria.In contrast, different enzymes are involved in the first reaction.In plants, this reaction is catalyzed by EC 1.14.15.7, whereas inanimals and many bacteria, it is catalyzed by either membrane-boundEC 1.1.99.1 or soluble EC 1.1.3.17.In some bacteria, betaine is synthesized from glycine through theactions of EC 2.1.1.156 and EC 2.1.1.157.','Betaine aldehyde + NAD(+) + H(2)O = betaine + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.9','Glyceraldehyde-3-phosphate dehydrogenase (NADP(+))','Triosephosphate dehydrogenase','NULL','NULL','D-glyceraldehyde 3-phosphate + NADP(+) + H(2)O = 3-phospho-D-glycerate +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.10','Acetaldehyde dehydrogenase (acetylating)','Acylating acetaldehyde dehydrogenase','NULL','Also acts, more slowly, on glycolaldehyde, propanal and butanal.In several bacterial species this enzyme forms a bifunctional complexwith EC 4.1.3.39.The enzymes from the bacteria Burkholderia xenovorans and Thermusthermophilus also perform the reaction of EC 1.2.1.87.Involved in the meta-cleavage pathway for the degradation of phenols,methylphenols and catechols.NADP(+) can replace NAD(+) but the rate of reaction is much slower.','Acetaldehyde + CoA + NAD(+) = acetyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.11','Aspartate-semialdehyde dehydrogenase','ASA dehydrogenase','NULL','NULL','L-aspartate 4-semialdehyde + phosphate + NADP(+) = L-4-aspartyl phosphate+ NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.12','Glyceraldehyde-3-phosphate dehydrogenase (phosphorylating)','GAPDH','NULL','Also acts very slowly on D-glyceraldehyde and some other aldehydes.Thiols can replace phosphate.','D-glyceraldehyde 3-phosphate + phosphate + NAD(+) = 3-phospho-D-glyceroylphosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.13','Glyceraldehyde-3-phosphate dehydrogenase (NADP(+)) (phosphorylating)','NADP-dependent glyceraldehyde-3-phosphate dehydrogenase','NULL','NULL','D-glyceraldehyde 3-phosphate + phosphate + NADP(+) = 3-phospho-D-glyceroyl phosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.14','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.15','Malonate-semialdehyde dehydrogenase','NULL','NULL','NULL','3-oxopropanoate + NAD(P)(+) + H(2)O = malonate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.16','Succinate-semialdehyde dehydrogenase (NAD(P)(+))','NULL','NULL','NULL','Succinate semialdehyde + NAD(P)(+) + H(2)O = succinate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.17','Glyoxylate dehydrogenase (acylating)','NULL','NULL','NULL','Glyoxylate + CoA + NADP(+) = oxalyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.18','Malonate-semialdehyde dehydrogenase (acetylating)','NULL','NULL','NULL','3-oxopropanoate + CoA + NAD(P)(+) = acetyl-CoA + CO(2) + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.19','Aminobutyraldehyde dehydrogenase','1-pyrroline dehydrogenase','NULL','The enzyme from some species exhibits broad substrate specificity andhas a marked preference for straight-chain aldehydes (up to 7 carbonatoms) as substrates.The plant enzyme also acts on 4-guanidinobutanal (cf. EC 1.2.1.54).As 1-pyrroline and 4-aminobutanal are in equilibrium and can beinterconverted spontaneously, 1-pyrroline may act as the startingsubstrate.Formerly EC 1.5.1.35.','4-aminobutanal + NAD(+) + H(2)O = 4-aminobutanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.20','Glutarate-semialdehyde dehydrogenase','NULL','NULL','NULL','Glutarate semialdehyde + NAD(+) + H(2)O = glutarate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.21','Glycolaldehyde dehydrogenase','NULL','NULL','NULL','Glycolaldehyde + NAD(+) + H(2)O = glycolate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.22','Lactaldehyde dehydrogenase','NULL','NULL','NULL','(S)-lactaldehyde + NAD(+) + H(2)O = (S)-lactate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.23','2-oxoaldehyde dehydrogenase (NAD(+))','Alpha-ketoaldehyde dehydrogenase','NULL','Not identical with EC 1.2.1.49.','A 2-oxoaldehyde + NAD(+) + H(2)O = a 2-oxo acid + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.24','Succinate-semialdehyde dehydrogenase (NAD(+))','NULL','NULL','This enzyme participates in the degradation of glutamate and4-aminobutyrate.It is similar to EC 1.2.1.79 and EC 1.2.1.16 but is specific forNAD(+).','Succinate semialdehyde + NAD(+) + H(2)O = succinate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.25','2-oxoisovalerate dehydrogenase (acylating)','NULL','NULL','Also acts on (S)-3-methyl-2-oxopentanoate and 4-methyl-2-oxopentanoate.','3-methyl-2-oxobutanoate + CoA + NAD(+) = 2-methylpropanoyl-CoA + CO(2) +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.26','2,5-dioxovalerate dehydrogenase','NULL','NULL','NULL','2,5-dioxopentanoate + NADP(+) + H(2)O = 2-oxoglutarate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.27','Methylmalonate-semialdehyde dehydrogenase (acylating)','MMSA dehydrogenase','NULL','Also converts 3-oxopropanoate into acetyl-CoA.The reaction occurs in two steps with the decarboxylation processpreceding CoA-binding.Bicarbonate rather than CO(2) is released as a final product.','2-methyl-3-oxopropanoate + CoA + H(2)O + NAD(+) = propanoyl-CoA +HCO(3)(-) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.28','Benzaldehyde dehydrogenase (NAD(+))','NULL','NULL','NULL','Benzaldehyde + NAD(+) + H(2)O = benzoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.29','Aryl-aldehyde dehydrogenase','NULL','NULL','Oxidizes a number of aromatic aldehydes, but not aliphatic aldehydes.','An aromatic aldehyde + NAD(+) + H(2)O = an aromatic acid + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.30','Aryl-aldehyde dehydrogenase (NADP(+))','NULL','NULL','NULL','An aromatic aldehyde + NADP(+) + AMP + diphosphate + H(2)O = an aromaticacid + NADPH + ATP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.31','L-aminoadipate-semialdehyde dehydrogenase','2-aminoadipate semialdehyde dehydrogenase','NULL','NULL','(S)-2-amino-6-oxohexanoate + NAD(P)(+) + H(2)O = L-2-aminoadipate +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.32','Aminomuconate-semialdehyde dehydrogenase','NULL','NULL','Also acts on 2-hydroxymuconate semialdehyde.','2-aminomuconate 6-semialdehyde + NAD(+) + H(2)O = 2-aminomuconate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.33','(R)-dehydropantoate dehydrogenase','D-aldopantoate dehydrogenase','NULL','NULL','(R)-4-dehydropantoate + NAD(+) + H(2)O = (R)-3,3-dimethylmalate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.34','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.35','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.36','Retinal dehydrogenase','NULL','FAD.','Acts on both the 11-trans- and 13-cis-forms of retinal.','Retinal + NAD(+) + H(2)O = retinoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.37','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.38','N-acetyl-gamma-glutamyl-phosphate reductase','N-acetyl-glutamate semialdehyde dehydrogenase','NULL','NULL','N-acetyl-L-glutamate 5-semialdehyde + NADP(+) + phosphate = N-acetyl-5-glutamyl phosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.39','Phenylacetaldehyde dehydrogenase','NULL','NULL','NULL','Phenylacetaldehyde + NAD(+) + H(2)O = phenylacetate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.40','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.41','Glutamate-5-semialdehyde dehydrogenase','Beta-glutamylphosphate reductase','NULL','NULL','L-glutamate 5-semialdehyde + phosphate + NADP(+) = L-glutamyl 5-phosphate+ NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.42','Hexadecanal dehydrogenase (acylating)','Fatty acyl-CoA reductase','NULL','Also acts, more slowly, on octadecanoyl-CoA.','Hexadecanal + CoA + NAD(+) = hexadecanoyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.43','Formate dehydrogenase (NADP(+))','NULL','Iron; Selenium; Tungsten.','NULL','Formate + NADP(+) = CO(2) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.44','Cinnamoyl-CoA reductase','NULL','NULL','Also acts on a number of substituted cinnamoyl esters of coenzyme A.','Cinnamaldehyde + CoA + NADP(+) = cinnamoyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.45','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.46','Formaldehyde dehydrogenase','NULL','NULL','NULL','Formaldehyde + NAD(+) + H(2)O = formate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.47','4-trimethylammoniobutyraldehyde dehydrogenase','NULL','NULL','NULL','4-trimethylammoniobutanal + NAD(+) + H(2)O = 4-trimethylammoniobutanoate+ NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.48','Long-chain-aldehyde dehydrogenase','NULL','NULL','The best substrate is dodecylaldehyde.','A long-chain aldehyde + NAD(+) + H(2)O = a long-chain acid anion + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.49','2-oxoaldehyde dehydrogenase (NADP(+))','Alpha-ketoaldehyde dehydrogenase','NULL','Not identical with EC 1.2.1.23.','A 2-oxoaldehyde + NADP(+) + H(2)O = a 2-oxo acid + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.50','Long-chain-fatty-acyl-CoA reductase','Acyl-CoA reductase','NULL','Together with EC 6.2.1.19 forms a fatty acid reductase system whichproduces the substrate of EC 1.14.14.3, thus being part of thebacterial luciferase system.','A long-chain aldehyde + CoA + NADP(+) = a long-chain acyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.51','Pyruvate dehydrogenase (NADP(+))','Pyruvate:NADP(+) oxidoreductase','NULL','The Euglena enzyme can also use FAD or methyl viologen as acceptor,more slowly.Inhibited by oxygen.','Pyruvate + CoA + NADP(+) = acetyl-CoA + CO(2) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.52','Oxoglutarate dehydrogenase (NADP(+))','NULL','NULL','The Euglena enzyme can also use NAD(+) as acceptor, but more slowly.','2-oxoglutarate + CoA + NADP(+) = succinyl-CoA + CO(2) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.53','4-hydroxyphenylacetaldehyde dehydrogenase','NULL','NULL','With EC 4.2.1.87, brings about the metabolism of octopamine inPseudomonas.','4-hydroxyphenylacetaldehyde + NAD(+) + H(2)O = 4-hydroxyphenylacetate +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.54','Gamma-guanidinobutyraldehyde dehydrogenase','NULL','NULL','Involved in the degradation of arginine in Pseudomonas putida (cf.EC 1.2.1.19).','4-guanidinobutanal + NAD(+) + H(2)O = 4-guanidinobutanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.55','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.56','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.57','Butanal dehydrogenase','NULL','NULL','Also acts on acetaldehyde, but more slowly.','Butanal + CoA + NAD(P)(+) = butanoyl-CoA + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.58','Phenylglyoxylate dehydrogenase (acylating)','NULL','FAD; Iron-sulfur; Thiamine diphosphate.','The enzyme from Azoarcus evansii is specific for phenylglyoxylate.2-oxoisovalerate is oxidized at 15% of the rate for phenylglyoxylate.Also reduces viologen dyes.','Phenylglyoxylate + NAD(+) + CoA = benzoyl-S-CoA + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.59','Glyceraldehyde-3-phosphate dehydrogenase (NAD(P)(+)) (phosphorylating)','NAD(P)-dependent glyceraldehyde-3-phosphate dehydrogenase','NULL','NAD(+) and NADP(+) can be used as cofactors with similar efficiency,unlike EC 1.2.1.12 and EC 1.2.1.13, which are NAD(+)- and NADP(+)-dependent, respectively.','D-glyceraldehyde 3-phosphate + phosphate + NAD(P)(+) = 3-phospho-D-glyceroyl phosphate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.60','5-carboxymethyl-2-hydroxymuconic-semialdehyde dehydrogenase','Carboxymethylhydroxymuconic semialdehyde dehydrogenase','NULL','Involved in the tyrosine degradation pathway in Arthrobacter sp.','5-carboxymethyl-2-hydroxymuconate semialdehyde + H(2)O + NAD(+) =5-carboxymethyl-2-hydroxymuconate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.61','4-hydroxymuconic-semialdehyde dehydrogenase','NULL','NULL','Involved in the 4-nitrophenol degradation pathway.','4-hydroxymuconic semialdehyde + NAD(+) + H(2)O = maleylacetate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.62','4-formylbenzenesulfonate dehydrogenase','NULL','NULL','Involved in the toluene-4-sulfonate degradation pathway.','4-formylbenzenesulfonate + NAD(+) + H(2)O = 4-sulfobenzoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.63','6-oxohexanoate dehydrogenase','NULL','NULL','Last step in the cyclohexanol degradation pathway in Acinetobactersp.','6-oxohexanoate + NADP(+) + H(2)O = adipate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.64','4-hydroxybenzaldehyde dehydrogenase','p-hydroxybenzaldehyde dehydrogenase','NULL','Involved in the toluene degradation pathway in Pseudomonas mendocina.','4-hydroxybenzaldehyde + NAD(+) + H(2)O = 4-hydroxybenzoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.65','Salicylaldehyde dehydrogenase','NULL','NULL','Involved in the naphthalene degradation pathway in some bacteria.','Salicylaldehyde + NAD(+) + H(2)O = salicylate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.66','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.67','Vanillin dehydrogenase','NULL','NULL','NULL','Vanillin + NAD(+) + H(2)O = vanillate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.68','Coniferyl-aldehyde dehydrogenase','NULL','NULL','Also oxidizes other aromatic aldehydes, but not aliphatic aldehydes.','Coniferyl aldehyde + H(2)O + NAD(P)(+) = ferulate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.69','Fluoroacetaldehyde dehydrogenase','NULL','NULL','The enzyme from Streptomyces cattleya has a high affinity forfluoroacetate and glycolaldehyde but not for acetaldehyde.','Fluoroacetaldehyde + NAD(+) + H(2)O = fluoroacetate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.70','Glutamyl-tRNA reductase','NULL','NULL','Forms part of the pathway for the biosynthesis of 5-aminolevulinatefrom glutamate, known as the C5 pathway, which is used in mosteubacteria, and in all archaebacteria, algae and plants.However, in the alpha-proteobacteria EC 2.3.1.37 is used in analternative route to produce the product 5-aminolevulinate fromsuccinyl-CoA and glycine.This route is found in the mitochondria of fungi and animals,organelles that are considered to be derived from an endosymbioticalpha-proteobacterium.Although higher plants do not possess EC 2.3.1.37, the protistanEuglena gracilis possesses both the C5 pathway and EC 2.3.1.37.','L-glutamate 1-semialdehyde + NADP(+) + tRNA(Glu) = L-glutamyl-tRNA(Glu) +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.71','Succinylglutamate-semialdehyde dehydrogenase','N-succinylglutamate 5-semialdehyde dehydrogenase','NULL','This is the fourth enzyme in the arginine succinyltransferase (AST)pathway for the catabolism of arginine.This pathway converts the carbon skeleton of arginine into glutamate,with the concomitant production of ammonia and conversion ofsuccinyl-CoA into succinate and CoA.The five enzymes involved in this pathway are EC 2.3.1.109,EC 3.5.3.23, EC 2.6.1.11, EC 1.2.1.71 and EC 3.5.1.96.','N-succinyl-L-glutamate 5-semialdehyde + NAD(+) + H(2)O = N-succinyl-L-glutamate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.72','Erythrose-4-phosphate dehydrogenase','E4P dehydrogenase','NULL','Was originally thought to be an EC 1.2.1.12, but this has since beendisproved, as glyceraldehyde 3-phosphate is not a substrate.Forms part of the pyridoxal-5-phosphate coenzyme biosynthesispathway in Escherichia coli, along with EC 1.1.1.290, EC 2.6.1.52,EC 1.1.1.262, EC 2.6.99.2 and EC 1.4.3.5.','D-erythrose 4-phosphate + NAD(+) + H(2)O = 4-phosphoerythronate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.73','Sulfoacetaldehyde dehydrogenase','NULL','NULL','Part of a bacterial pathway that can utilize the amino group oftaurine as a sole source of nitrogen for growth.At physiological concentrations, NAD(+) cannot be replaced byNADP(+).Specific for sulfoacetaldehyde, as formaldehyde, acetaldehyde,betaine aldehyde, propanal, glyceraldehyde, phosphonoaldehyde,glyoxylate, glycolaldehyde and 2-oxobutyrate are not substrates.','2-sulfoacetaldehyde + H(2)O + NAD(+) = sulfoacetate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.74','Abieta-7,13-dien-18-al dehydrogenase','Abietadienal dehydrogenase','NULL','Abietic acid is the principle component of conifer resin.This enzyme catalyzes the last step of the pathway of abietic acidbiosynthesis in Abies grandis (grand fir).The activity has been demonstrated in cell-free stem extracts ofA.grandis, was present in the cytoplasm, and required NAD(+) ascofactor.The enzyme is expressed constitutively at a high level, and is notinducible by wounding of the plant tissue.','Abieta-7,13-diene-18-al + H(2)O + NAD(+) = abieta-7,13-diene-18-oate +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.75','Malonyl CoA reductase (malonate semialdehyde-forming)','Malonyl CoA reductase (NADP)','Magnesium.','Catalyzes the reduction of malonyl-CoA to malonate semialdehyde,a key step in the 3-hydroxypropanoate and the3-hydroxypropanoate/4-hydroxybutyrate cycles, autotrophic CO(2)fixation pathways found in some green non-sulfur phototrophicbacteria and some thermoacidophilic archaea, respectively.The enzyme from Sulfolobus tokodaii has been purified, and found tocontain one RNA molecule per two subunits.The enzyme from Chloroflexus aurantiacus is bifunctional, and alsocatalyzes the next reaction in the pathway, EC 1.1.1.298.','Malonate semialdehyde + CoA + NADP(+) = malonyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.76','Succinate-semialdehyde dehydrogenase (acetylating)','Coenzyme-A-dependent succinate-semialdehyde dehydrogenase','NULL','Catalyzes the NADPH-dependent reduction of succinyl-CoA to succinatesemialdehyde.The enzyme has been described in Clostridium kluyveri, where itparticipates in succinate fermentation, and in Metallosphaera sedula,where it participates in the 3-hydroxypropanoate/4-hydroxybutyratecycle, an autotrophic CO(2) fixation pathway found in somethermoacidophilic archaea.','Succinate semialdehyde + CoA + NADP(+) = succinyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.77','3,4-dehydroadipyl-CoA semialdehyde dehydrogenase (NADP(+))','3,4-dehydroadipyl-CoA semialdehyde dehydrogenase','NULL','This enzyme catalyzes a step in the aerobic benzoyl-coenzyme Acatabolic pathway in Azoarcus evansii and Burkholderia xenovorans.Formerly EC 1.2.1.n1.','Cis-3,4-dehydroadipyl-CoA semialdehyde + NADP(+) + H(2)O = cis-3,4-dehydroadipyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.78','2-formylbenzoate dehydrogenase','2-carboxybenzaldehyde dehydrogenase','NULL','The enzyme is involved in phenanthrene degradation.','2-formylbenzoate + NAD(+) + H(2)O = phthalate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.79','Succinate-semialdehyde dehydrogenase (NADP(+))','NULL','NULL','This enzyme participates in the degradation of glutamate and4-aminobutyrate.It is similar to EC 1.2.1.24 and EC 1.2.1.16 but is specific forNADP(+).The enzyme from Escherichia coli is 20-fold more active with NADP(+)than NAD(+).','Succinate semialdehyde + NADP(+) + H(2)O = succinate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.80','Long-chain acyl-[acyl-carrier-protein] reductase','Fatty acyl-[acyl-carrier-protein] reductase','Magnesium.','Catalyzes the reaction in the opposite direction.This enzyme, purified from the cyanobacterium Synechococcus elongatusPCC 7942, catalyzes the NAD(P)H-dependent reduction of an activatedfatty acid (acyl-[acp]) to the corresponding aldehyde.Together with EC 4.1.99.5, it is involved in alkane biosynthesis.The natural substrates of the enzyme are C16 to C18 activated fattyacids.','A long-chain aldehyde + an [acyl-carrier protein] + NAD(P)(+) = a long-chain acyl-[acyl-carrier protein] + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.81','Sulfoacetaldehyde dehydrogenase (acylating)','NULL','NULL','The enzyme is involved in degradation of sulfoacetate.In this pathway the reaction is catalyzed in the reverse direction.The enzyme is specific for sulfoacetaldehyde and NADP(+).','2-sulfoacetaldehyde + CoA + NADP(+) = sulfoacetyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.82','Beta-apo-4-carotenal oxygenase','Beta-apo-4-carotenal dehydrogenase','NULL','Neurosporaxanthin is responsible for the orange color of ofNeurospora.','4-apo-beta,psi-caroten-4-al + NAD(+) + H(2)O = neurosporaxanthin +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.83','3-succinoylsemialdehyde-pyridine dehydrogenase','NULL','NULL','The enzyme has been characterized from the soil bacterium Pseudomonassp. HZN6.It participates in the nicotine degradation pathway.','4-oxo-4-(pyridin-3-yl)butanal + NADP(+) + H(2)O = 4-oxo-4-(pyridin-3-yl)butanoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.84','Alcohol-forming fatty acyl-CoA reductase','NULL','NULL','The enzyme has been characterized from the plant Simmondsia chinensis(jojoba).The alcohol is formed by a four-electron reduction of fatty acyl-CoA.Although the reaction proceeds through an aldehyde intermediate,a free aldehyde is not released.The recombinant enzyme was shown to accept saturated and mono-unsaturated fatty acyl-CoAs of 16 to 22 carbons.','A long-chain acyl-CoA + 2 NADPH = CoA + a long-chain alcohol + 2 NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.85','2-hydroxymuconate-6-semialdehyde dehydrogenase','NULL','NULL','This substrate for this enzyme is formed by meta ring cleavage ofcatechol (EC 1.13.11.2), and is an intermediate in the bacterialdegradation of several aromatic compounds.Has lower activity with benzaldehyde.Activity with NAD(+) is more than 10-fold higher than with NADP(+).Cf. EC 1.2.1.32.','2-hydroxymuconate-6-semialdehyde + NAD(+) + H(2)O = (2Z,4E)-2-hydroxyhexa-2,4-dienedioate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.86','Geranial dehydrogenase','GaDH','NULL','Does not act on neral.Formerly EC 1.2.1.n3.','Geranial + H(2)O + NAD(+) = geranate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.87','Propanal dehydrogenase (CoA-propanoylating)','NULL','NULL','The enzyme forms a bifunctional complex with EC 4.1.3.43, with atight channel connecting the two subunits.Also acts, more slowly, on glycolaldehyde and butanal.In Pseudomonas species the enzyme forms a bifunctional complex withEC 4.1.3.39.The enzymes from the bacteria Burkholderia xenovorans and Thermusthermophilus also perform the reaction of EC 1.2.1.10.NADP(+) can replace NAD(+) with a much slower rate.','Propanal + CoA + NAD(+) = propanoyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.88','L-glutamate gamma-semialdehyde dehydrogenase','1-pyrroline-5-carboxylate dehydrogenase','NULL','This enzyme catalyzes the irreversible oxidation of glutamate-gamma-semialdehyde to glutamate as part of the proline degradation pathway.(S)-1-pyrroline-5-carboxylate, the product of the first enzyme of thepathway (EC 1.5.5.2) is in spontaneous equilibrium with its tautomerL-glutamate gamma-semialdehyde.In many bacterial species, both activities are carried out by asingle bifunctional enzyme.The enzyme can also oxidize other 1-pyrrolines, e.g. 3-hydroxy-1-pyrroline-5-carboxylate is converted into 4-hydroxyglutamate and (R)-1-pyrroline-5-carboxylate is converted into D-glutamate.NADP(+) can also act as acceptor, but with lower activity.Formerly EC 1.5.1.12.','L-glutamate 5-semialdehyde + NAD(+) + H(2)O = L-glutamate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.89','D-glyceraldehyde dehydrogenase (NADP(+))','GADH','NULL','The enzyme from the archaea Thermoplasma acidophilum and Picrophilustorridus is involved in the non-phosphorylative Entner-Doudoroffpathway, cf. EC 1.2.99.8.','D-glyceraldehyde + NADP(+) + H(2)O = D-glycerate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.n2','Fatty acyl-CoA reductase','Hexadecanal dehydrogenase (acylating, NADP dependent)','NULL','NULL','Hexadecanal + CoA + NADP(+) = hexadecanoyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.1.n3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.2.1','Formate dehydrogenase (cytochrome)','NULL','NULL','NULL','Formate + 2 ferricytochrome b1 = CO(2) + 2 ferrocytochrome b1 + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.2.2','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.2.3','Formate dehydrogenase (cytochrome c-553)','NULL','NULL','Saccharomyces cerevisiae cytochrome c, ferricyanide and phenazinemethosulfate can act as acceptors.','Formate + ferricytochrome c-553 = CO(2) + ferrocytochrome c-553.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.2.4','Carbon-monoxide dehydrogenase (cytochrome b-561)','Carbon monoxide oxidase','FAD; Iron-sulfur; Molybdopterin cytosine dinucleotide.','Oxygen, methylene blue and iodonitrotetrazolium chloride can act asnonphysiological electron acceptors.','CO + H(2)O + 2 ferricytochrome b-561 = CO(2) + 2 H(+) + 2 ferrocytochromeb-561.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.1','Aldehyde oxidase','Quinoline oxidase','FAD; Iron-sulfur; Molybdenum.','The enzyme from liver exhibits a broad substrate specificity, and isinvolved in the metabolism of xenobiotics, including the oxidation ofN-heterocycles and aldehydes and the reduction of N-oxides,nitrosamines, hydroxamic acids, azo dyes, nitropolycyclic aromatichydrocarbons, and sulfoxides.The enzyme is also responsible for the oxidation of retinal,an activity that was initially attributed to a distinct enzyme(EC 1.2.3.11).Formerly EC 1.2.3.11.','An aldehyde + H(2)O + O(2) = a carboxylate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.3','Pyruvate oxidase','Phosphate-dependent pyruvate oxidase','FAD; Thiamine diphosphate.','Two reducing equivalents are transferred from the resonantcarbanion/enamine forms of 2-hydroxyethyl-thiamine-diphosphate to theadjacent flavin cofactor, yielding 2-acetyl-thiamine diphosphate(AcThDP) and reduced flavin.FADH is reoxidized by O(2) to yield H(2)O(2) and FAD and AcThDP iscleaved phosphorolytically to acetyl phosphate and thiaminediphosphate.','Pyruvate + phosphate + O(2) = acetyl phosphate + CO(2) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.4','Oxalate oxidase','NULL','Manganese.','NULL','Oxalate + O(2) + 2 H(+) = 2 CO(2) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.5','Glyoxylate oxidase','NULL','NULL','NULL','Glyoxylate + H(2)O + O(2) = oxalate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.6','Pyruvate oxidase (CoA-acetylating)','NULL','FAD.','May be identical with EC 1.2.7.1.','Pyruvate + CoA + O(2) = acetyl-CoA + CO(2) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.7','Indole-3-acetaldehyde oxidase','IAA oxidase','FAD; Heme; Molybdenum.','Isoform of EC 1.2.3.1.Has a preference for aldehydes having an indole-ring structure assubstrate.May play a role in plant hormone biosynthesis as its activity ishigher in the auxin-overproducing mutant, super-root1, than in wild-type Arabidopsis thaliana.While (indol-3-yl)acetaldehyde is the preferred substrate, it alsooxidizes indole-3-carbaldehyde and acetaldehyde, but more slowly.','(Indol-3-yl)acetaldehyde + H(2)O + O(2) = (indol-3-yl)acetate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.8','Pyridoxal oxidase','NULL','Molybdenum.','NULL','Pyridoxal + H(2)O + O(2) = 4-pyridoxate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.9','Aryl-aldehyde oxidase','NULL','NULL','Acts on benzaldehyde, vanillin and a number of other aromaticaldehydes, but not on aliphatic aldehydes or sugars.','An aromatic aldehyde + O(2) + H(2)O = an aromatic carboxylate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.10','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.11','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.12','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.13','4-hydroxyphenylpyruvate oxidase','NULL','NULL','Involved in tyrosine degradation pathway in Arthrobacter sp.','2 4-hydroxyphenylpyruvate + O(2) = 2 4-hydroxyphenylacetate + 2 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.3.14','Abscisic-aldehyde oxidase','AO-delta','NULL','Acts on both (+)- and (-)-abscisic aldehyde.Involved in the abscisic-acid biosynthesis pathway in plants, alongwith EC 1.1.1.288, EC 1.13.11.51 and EC 1.14.13.93.While abscisic aldehyde is the best substrate, the enzyme also actswith indole-3-aldehyde, 1-naphthaldehyde and benzaldehyde assubstrates, but more slowly.','Abscisic aldehyde + H(2)O + O(2) = abscisate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.4.1','Pyruvate dehydrogenase (acetyl-transferring)','MtPDC (mitochondrial pyruvate dehydogenase complex)','Thiamine diphosphate.','It is a component (in multiple copies) of the multienzyme pyruvatedehydrogenase complex in which it is bound to a core of molecules ofEC 2.3.1.12, which also binds multiple copies of EC 1.8.1.4.It does not act on free lipoamide or lipoyllysine, but only on thelipoyllysine residue in EC 2.3.1.12.','Pyruvate + [dihydrolipoyllysine-residue acetyltransferase] lipoyllysine =[dihydrolipoyllysine-residue acetyltransferase]S-acetyldihydrolipoyllysine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.4.2','Oxoglutarate dehydrogenase (succinyl-transferring)','2-ketoglutarate dehydrogenase','Thiamine diphosphate.','It is a component of the multienzyme 2-oxoglutarate dehydrogenasecomplex in which multiple copies of it are bound to a core ofmolecules of EC 2.3.1.61, which also binds multiple copies ofEC 1.8.1.4.It does not act on free lipoamide or lipoyllysine, but only on thelipoyllysine residue in EC 2.3.1.61.','2-oxoglutarate + [dihydrolipoyllysine-residue succinyltransferase]lipoyllysine = [dihydrolipoyllysine-residue succinyltransferase]S-succinyldihydrolipoyllysine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.4.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.4.4','3-methyl-2-oxobutanoate dehydrogenase (2-methylpropanoyl-transferring)','2-oxoisocaproate dehydrogenase','Thiamine diphosphate.','It acts not only on 3-methyl-2-oxobutanaoate, but also on 4-methyl-2-oxopentanoate and (S)-3-methyl-2-oxopentanoate, so that it acts onthe 2-oxo acids that derive from the action of transaminases onvaline, leucine and isoleucine.It is a component of the multienzyme 3-methyl-2-oxobutanoatedehydrogenase complex in which multiple copies of it are bound to acore of molecules of EC 2.3.1.168, which also binds multiple copiesof EC 1.8.1.4.It does not act on free lipoamide or lipoyllysine, but only on thelipoyllysine residue in EC 2.3.1.168.Formerly EC 1.2.4.3.','3-methyl-2-oxobutanoate + [dihydrolipoyllysine-residue(2-methylpropanoyl)transferase] lipoyllysine = [dihydrolipoyllysine-residue (2-methylpropanoyl)transferase]S-(2-methylpropanoyl)dihydrolipoyllysine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.5.1','Pyruvate dehydrogenase (quinone)','Pyruvate dehydrogenase','FAD.','This bacterial enzyme is located on the inner surface of thecytoplasmic membrane and coupled to the respiratory chain viaubiquinone.Does not accept menaquinone.Activity is greatly enhanced by lipids.Requires thiamine diphosphate.The enzyme can also form acetoin.','Pyruvate + ubiquinone + H(2)O = acetate + CO(2) + ubiquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.1','Pyruvate synthase','2-oxobutyrate synthase','Iron-sulfur; Thiamine diphosphate.','The enzyme also decarboxylates 2-oxobutyrate with lower efficiency,but shows no activity with 2-oxoglutarate.This enzyme is a member of the 2-oxoacid oxidoreductases, a family ofenzymes that oxidatively decarboxylate different 2-oxoacids to formtheir CoA derivatives, and are differentiated based on theirsubstrate specificity.For examples of other members of this family, see EC 1.2.7.3 andEC 1.2.7.7.','Pyruvate + CoA + 2 oxidized ferredoxin = acetyl-CoA + CO(2) + 2 reducedferredoxin + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.2','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.3','2-oxoglutarate synthase','2-ketoglutarate ferredoxin oxidoreductase','Iron-sulfur; Thiamine diphosphate.','Highly specific for 2-oxoglutarate.This enzyme is a member of the 2-oxoacid oxidoreductases, a family ofenzymes that oxidatively decarboxylate different 2-oxoacids to formtheir CoA derivatives, and are differentiated based on theirsubstrate specificity.For examples of other members of this family, see EC 1.2.7.1 andEC 1.2.7.7.Formerly EC 1.2.7.9.','2-oxoglutarate + CoA + 2 oxidized ferredoxin = succinyl-CoA + CO(2) +2 reduced ferredoxin + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.4','Carbon-monoxide dehydrogenase (ferredoxin)','NULL','Iron; Nickel; Zinc.','Methyl viologen can act as acceptor.The enzyme from Moorella thermoacetica exists as a complex withEC 2.3.1.169, which catalyzes the overall reaction: methylcorrinoidprotein + CoA + CO(2) + reduced ferredoxin = acetyl-CoA + corrinoidprotein + H(2)O + oxidized ferredoxin.','CO + H(2)O + 2 oxidized ferredoxin = CO(2) + 2 reduced ferredoxin +2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.5','Aldehyde ferredoxin oxidoreductase','AOR','Iron-sulfur; Tungsten-molybdopterin.','This is an oxygen-sensitive enzyme.Catalyzes the oxidation of aldehydes (including crotonaldehyde,acetaldehyde, formaldehyde and glyceraldehyde) to their correspondingacids.However, it does not oxidize glyceraldehyde 3-phosphate (seeEC 1.2.7.6).Can use ferredoxin or methyl viologen but not NAD(P)(+) as electronacceptor.','An aldehyde + H(2)O + 2 oxidized ferredoxin = a carboxylate + 2 H(+) +2 reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.6','Glyceraldehyde-3-phosphate dehydrogenase (ferredoxin)','GAPOR','Iron-sulfur; Tungsten-molybdopterin.','Thought to function in place of glyceraldehyde-3-phosphatedehydrogenase and possibly phosphoglycerate kinase in the novelEmbden-Meyerhof-type glycolytic pathway found in Pyrococcus furiosus.Specific for glyceraldehyde-3-phosphate.','D-glyceraldehyde-3-phosphate + H(2)O + 2 oxidized ferredoxin = 3-phospho-D-glycerate + 2 H(+) + 2 reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.7','3-methyl-2-oxobutanoate dehydrogenase (ferredoxin)','2-ketoisovalerate ferredoxin reductase','Iron-sulfur; Tungsten-molybdopterin.','Preferentially utilizes 2-oxo-acid derivatives of branched chainamino acids, e.g. 3-methyl-2-oxopentanoate, 4-methyl-2-oxo-pentanoate, 2-oxobutyrate and 3-methylthiopropanamine.This enzyme is a member of the 2-oxoacid oxidoreductases, a family ofenzymes that oxidatively decarboxylate different 2-oxoacids to formtheir CoA derivatives, and are differentiated based on theirsubstrate specificity.For examples of other members of this family, see EC 1.2.7.1 andEC 1.2.7.3.','3-methyl-2-oxobutanoate + CoA + 2 oxidized ferredoxin =S-(2-methylpropanoyl)-CoA + CO(2) + 2 reduced ferredoxin + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.8','Indolepyruvate ferredoxin oxidoreductase','3-(indol-3-yl)pyruvate synthase (ferredoxin)','Iron-sulfur; Thiamine diphosphate.','Preferentially utilizes the transaminated forms of aromatic aminoacids and can use phenylpyruvate and 4-hydroxyphenylpyruvate assubstrates.This enzyme, which is found in archaea, is a member of the 2-oxoacidoxidoreductases, a family of enzymes that oxidatively decarboxylatedifferent 2-oxoacids to form their CoA derivatives, and aredifferentiated based on their substrate specificity.For examples of other members of this family, see EC 1.2.7.3 andEC 1.2.7.7.','(Indol-3-yl)pyruvate + CoA + 2 oxidized ferredoxin = S-2-(indol-3-yl)acetyl-CoA + CO(2) + 2 reduced ferredoxin + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.10','Oxalate oxidoreductase','NULL','Iron-sulfur; Thiamine diphosphate.','NULL','Oxalate + oxidized ferredoxin = 2 CO(2) + reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.7.11','2-oxoacid oxidoreductase (ferredoxin)','OFOR','Iron-sulfur; Thiamine diphosphate.','This enzyme is a member of the 2-oxoacid oxidoreductases, a family ofenzymes that oxidatively decarboxylate different 2-oxoacids to formtheir CoA derivatives, and are differentiated based on theirsubstrate specificity.For example, see EC 1.2.7.3 and EC 1.2.7.7.','A 2-oxocarboxylate + CoA + 2 oxidized ferredoxin = an acyl-CoA + CO(2) +2 reduced ferredoxin + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.99.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.99.2','Carbon-monoxide dehydrogenase (acceptor)','Anaerobic carbon monoxide dehydrogenase','Iron-sulfur; Nickel.','It uses many electron acceptors, including ferredoxin, methylviologen and benzyl viologen and flavins, but not pyridinenucleotides.Forms part of a membrane-bound multienzyme complex with EC 1.12.99.6,which catalyzes the overall reaction: CO + H(2)O = CO(2) + H(2).','CO + H(2)O + A = CO(2) + AH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.99.3','Aldehyde dehydrogenase (pyrroloquinoline-quinone)','Aldehyde dehydrogenase (acceptor)','Pyrroloquinoline quinone.','Wide specificity; acts on straight-chain aldehydes up to C(10),aromatic aldehydes, glyoxylate and glyceraldehyde.','An aldehyde + acceptor + H(2)O = a carboxylate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.99.4','Formaldehyde dismutase','NULL','Magnesium; NAD(+) or NADP(+); Zinc.','The enzyme from Mycobacterium sp. DSM 3803 also catalyzes thereactions of EC 1.1.99.36 and EC 1.1.99.37.Formaldehyde and acetaldehyde can act as donors; formaldehyde,acetaldehyde and propanal can act as acceptors.','2 formaldehyde + H(2)O = formate + methanol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.99.5','Formylmethanofuran dehydrogenase','NULL','Molybdenum.','Tungstate can substitute for molybdate.The enzyme catalyzes a reversible reaction in methanogenic bacteria,and is involved in methanogenesis from CO(2) as well as the oxidationof methyl-coenzyme M to CO(2).Methyl viologen can act as acceptor.Also oxidizes N-furfurylformamide.','Formylmethanofuran + H(2)O + acceptor = CO(2) + methanofuran + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.99.6','Carboxylate reductase','NULL','Tungsten.','Methyl viologen can act as acceptor.In the reverse direction, non-activated acids are reduced by reducedviologens to aldehydes, but not to the corresponding alcohols.','An aldehyde + acceptor + H(2)O = a carboxylate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.99.7','Aldehyde dehydrogenase (FAD-independent)','Aldehyde oxidase','Iron-sulfur; Molybdopterin cytosine dinucleotide.','NULL','An aldehyde + H(2)O + acceptor = a carboxylate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.2.99.8','Glyceraldehyde dehydrogenase (FAD-containing)','Glyceraldehyde oxidoreductase','FAD; Iron-sulfur; Molybdopterin guanine dinucleotide.','The enzyme from the archaeon Sulfolobus acidocaldarius catalyzes theoxidation of D-glyceraldehyde in the nonphosphorylative Entner-Doudoroff pathway.With 2,6-dichlorophenolindophenol as artificial electron acceptor,the enzyme shows a broad substrate range, but is most active withD-glyceraldehyde.It is not known which acceptor is utilized in vivo.','D-glyceraldehyde + H(2)O + acceptor = D-glycerate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.1','Dihydrouracil dehydrogenase (NAD(+))','NULL','Flavoprotein; Iron-sulfur.','The enzyme was originally discovered in the uracil-fermentingbacterium, Clostridium uracilicum, which utilizes uracil and thymineas nitrogen and carbon sources for growth.Since then the enzyme was found in additional organisms includingAlcaligenes eutrophus, Pseudomonas strains and Escherichia coli.','(1) 5,6-dihydrouracil + NAD(+) = uracil + NADH.(2) 5,6-dihydrothymine + NAD(+) = thymine + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.2','Dihydropyrimidine dehydrogenase (NADP(+))','Dihydrothymine dehydrogenase','NULL','Also acts on dihydrothymine.','5,6-dihydrouracil + NADP(+) = uracil + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.3','Delta(4)-3-oxosteroid 5-beta-reductase','3-oxo-Delta(4)-steroid 5-beta-reductase','NULL','The enzyme from human efficiently catalyzes the reduction ofprogesterone, androstenedione, 17-alpha-hydroxyprogesterone andtestosterone to 5-beta-reduced metabolites; it can also act onaldosterone, corticosterone and cortisol, but to a lesser extent.The bile acid intermediates 7-alpha,12-alpha-dihydroxy-4-cholesten-3-one and 7-alpha-hydroxy-4-cholesten-3-one can also act as substrates.','(1) 5-beta-cholestan-3-one + NADP(+) = cholest-4-en-3-one + NADPH.(2) 17,21-dihydroxy-5-beta-pregnane-3,11,20-trione + NADP(+) = cortisone+ NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.5','Cucurbitacin Delta(23)-reductase','NULL','Manganese.','Iron or zinc can replace manganese to some extent.Cucurbitacin is a plant triterpenoid.','23,24-dihydrocucurbitacin + NAD(P)(+) = cucurbitacin + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.6','Fumarate reductase (NADH)','NADH-dependent fumarate reductase','NULL','NULL','Succinate + NAD(+) = fumarate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.7','Meso-tartrate dehydrogenase','NULL','NULL','NULL','Meso-tartrate + NAD(+) = dihydroxyfumarate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.8','Acyl-CoA dehydrogenase (NADP(+))','2-enoyl-CoA reductase','NULL','The liver enzyme acts on enoyl-CoA derivatives of carbon chain length4 to 16, with optimum activity on 2-hexenoyl-CoA.In Escherichia coli, cis-specific and trans-specific enzymes exist(cf. EC 1.3.1.37 and EC 1.3.1.38).','Acyl-CoA + NADP(+) = 2,3-dehydroacyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.9','Enoyl-[acyl-carrier-protein] reductase (NADH)','Enoyl-ACP reductase','NULL','The enzyme catalyzes an essential step in fatty acid biosynthesis,the reduction of the 2,3-double bond in enoyl-acyl-[acyl-carrier-protein] derivatives of the elongating fatty acid moiety.The enzyme from the bacterium Escherichia coli accepts substrateswith carbon chain length from 4 to 18.The enzyme from the bacterium Mycobacterium tuberculosis preferssubstrates with carbon chain length from 12 to 24 carbons.','An acyl-[acyl-carrier protein] + NAD(+) = a trans-2,3-dehydroacyl-[acyl-carrier protein] + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.10','Enoyl-[acyl-carrier-protein] reductase (NADPH, Si-specific)','Acyl-ACP dehydrogenase','NULL','One of the activities of EC 2.3.1.86, an enzyme found in yeasts(Ascomycota and the Basidiomycota).Catalyzes the reduction of enoyl-acyl-[acyl-carrier protein]derivatives of carbon chain length from 4 to 16.The yeast enzyme is Si-specific with respect to NADP(+).cf. EC 1.3.1.39 and EC 1.3.1.9.','An acyl-[acyl-carrier protein] + NADP(+) = a trans-2,3-dehydroacyl-[acyl-carrier protein] + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.11','2-coumarate reductase','Melilotate dehydrogenase','NULL','NULL','3-(2-hydroxyphenyl)propanoate + NAD(+) = 2-coumarate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.12','Prephenate dehydrogenase','Chorismate mutase--prephenate dehydrogenase','NULL','This enzyme in the enteric bacteria also possesses EC 5.4.99.5activity and converts chorismate into prephenate.','Prephenate + NAD(+) = 4-hydroxyphenylpyruvate + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.13','Prephenate dehydrogenase (NADP(+))','NULL','NULL','NULL','Prephenate + NADP(+) = 4-hydroxyphenylpyruvate + CO(2) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.14','Dihydroorotate dehydrogenase (NAD(+))','DHOD','FAD; FMN; Iron-sulfur.','The enzyme consists of two subunits, an FMN binding catalytic subunitand a FAD and iron-sulfur binding electron transfer subunit.The reaction, which takes place in the cytosol, is the only redoxreaction in the de-novo biosynthesis of pyrimidine nucleotides.Other class 1 dihydroorotate dehydrogenases use either fumarate(EC 1.3.98.1) or NADP(+) (EC 1.3.1.15) as electron acceptor.The membrane bound class 2 dihydroorotate dehydrogenase (EC 1.3.5.2)uses quinone as electron acceptor.','(S)-dihydroorotate + NAD(+) = orotate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.15','Dihydroorotate dehydrogenase (NADP(+))','NULL','FAD; FMN.','Other class 1 dihydroorotate dehydrogenases use either fumarate(EC 1.3.98.1) or NAD(+) (EC 1.3.1.14) as electron acceptor.The membrane bound class 2 dihydroorotate dehydrogenase (EC 1.3.5.2)uses quinone as electron acceptor.','(S)-dihydroorotate + NADP(+) = orotate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.16','Beta-nitroacrylate reductase','NULL','NULL','NULL','3-nitropropanoate + NADP(+) = 3-nitroacrylate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.17','3-methyleneoxindole reductase','NULL','NULL','NULL','3-methyl-1,3-dihydroindol-2-one + NADP(+) = 3-methylene-1,3-dihydro-2H-indol-2-one + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.18','Kynurenate-7,8-dihydrodiol dehydrogenase','NULL','NULL','NULL','7,8-dihydro-7,8-dihydroxykynurenate + NAD(+) = 7,8-dihydroxykynurenate +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.19','Cis-1,2-dihydrobenzene-1,2-diol dehydrogenase','Cis-benzene glycol dehydrogenase','NULL','NULL','Cis-1,2-dihydrobenzene-1,2-diol + NAD(+) = catechol + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.20','Trans-1,2-dihydrobenzene-1,2-diol dehydrogenase','NULL','NULL','NULL','Trans-1,2-dihydrobenzene-1,2-diol + NADP(+) = catechol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.21','7-dehydrocholesterol reductase','7-dehydrocholesterol dehydrogenase/cholesterol oxidase','NULL','In mammals the enzyme is part of the cholesterol biosynthesis pathwayand catalyzes the reduction reaction.In the nematode Caenorhabditis elegans the enzyme catalyzes theoxidation reaction as part of the biosynthesis of dafachronic acidsthat are important for control of developmental timing and longevity.','Cholesterol + NADP(+) = cholesta-5,7-dien-3-beta-ol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.22','3-oxo-5-alpha-steroid 4-dehydrogenase (NADP(+))','3-oxo-5-alpha-steroid 4-dehydrogenase','NULL','The enzyme catalyzes the conversion of assorted 3-oxo-Delta(4)steroids into their corresponding 5-alpha form.Substrates for the mammalian enzyme include testosterone,progesterone, and corticosterone.Substrates for the plant enzyme are brassinosteroids such as campest-4-en-3-one and (22-alpha)-hydroxy-campest-4-en-3-one.Cf. EC 1.3.99.5.Formerly EC 1.3.1.4 and EC 1.3.1.30.','A 3-oxo-5-alpha-steroid + NADP(+) = a 3-oxo-Delta(4)-steroid + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.23','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.24','Biliverdin reductase','NULL','NULL','NULL','Bilirubin + NAD(P)(+) = biliverdin + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.25','1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate dehydrogenase','2-hydro-1,2-dihydroxybenzoate dehydrogenase','NULL','NULL','(1R,6S)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate + NAD(+) =catechol + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.26','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.27','2-hexadecenal reductase','2-alkenal reductase','NULL','Specific for long chain 2-trans- and 2-cis-alkenals, with chainlength optimum around 14 to 16 carbon atoms.','Hexadecanal + NADP(+) = 2-trans-hexadecenal + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.28','2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase','2,3-DHB dehydrogenase','NULL','Formerly EC 1.1.1.109.','2,3-dihydro-2,3-dihydroxybenzoate + NAD(+) = 2,3-dihydroxybenzoate +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.29','Cis-1,2-dihydro-1,2-dihydroxynaphthalene dehydrogenase','Cis-dihydrodiol naphthalene dehydrogenase','NULL','Also acts, at half the rate, on cis-anthracene dihydrodiol and cis-phenanthrene dihydrodiol.','Cis-1,2-dihydronaphthalene-1,2-diol + NAD(+) = naphthalene-1,2-diol +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.30','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.31','2-enoate reductase','Enoate reductase','FAD; Iron-sulfur.','Acts, in the reverse direction, on a wide range of alkyl and arylalpha-beta-unsaturated carboxylate ions; 2-butenoate was the bestsubstrate tested.','Butanoate + NAD(+) = but-2-enoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.32','Maleylacetate reductase','Maleolylacetate reductase','NULL','NULL','3-oxoadipate + NAD(P)(+) = 2-maleylacetate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.33','Protochlorophyllide reductase','NADPH-protochlorophyllide oxidoreductase','NULL','Catalyzes a light-dependent trans-reduction of the D-ring ofprotochlorophyllide; the product has the (7S,8S)-configuration.','Chlorophyllide a + NADP(+) = protochlorophyllide + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.34','2,4-dienoyl-CoA reductase (NADPH)','4-enoyl-CoA reductase (NADPH)','NULL','Best substrates for reduction contain a 2,4-diene structure withchain length C(8) or C(10).','Trans-2,3-didehydroacyl-CoA + NADP(+) = trans,trans-2,3,4,5-tetradehydroacyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.35','Phosphatidylcholine desaturase','Linoleate synthase','NULL','Also desaturates phosphatidylcholine containing the oleoyl group onO-2 of the glycerol residue.','1-acyl-2-oleoyl-sn-glycero-3-phosphocholine + NAD(+) = 1-acyl-2-linoleoyl-sn-glycero-3-phosphocholine + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.36','Geissoschizine dehydrogenase','NULL','NULL','Involved in the interconversion of heteroyohimbine alkaloids inCatharanthus roseus.','Geissoschizine + NADP(+) = 4,21-dehydrogeissoschizine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.37','Cis-2-enoyl-CoA reductase (NADPH)','NULL','NULL','Not identical with EC 1.3.1.38 (cf. EC 1.3.1.8).','Acyl-CoA + NADP(+) = cis-2,3-dehydroacyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.38','Trans-2-enoyl-CoA reductase (NADPH)','NULL','NULL','Not identical with EC 1.3.1.37 (cf. EC 1.3.1.8).','Acyl-CoA + NADP(+) = trans-2,3-dehydroacyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.39','Enoyl-[acyl-carrier-protein] reductase (NADPH, Re-specific)','Acyl-ACP dehydrogenase','NULL','This enzyme completes each cycle of fatty acid elongation bycatalyzing the stereospecific reduction of the double bond atposition 2 of a growing fatty acid chain, while linked to an acyl-carrier protein.It is one of the activities of EC 2.3.1.85.The mammalian enzyme is Re-specific with respect to NADP(+) (cf.EC 1.3.1.10 and and EC 1.3.1.104).','An acyl-[acyl-carrier protein] + NADP(+) = a trans-2,3-dehydroacyl-[acyl-carrier protein] + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.40','2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate reductase','NULL','NULL','Broad specificity; reduces a number of compounds produced byPseudomonas from aromatic hydrocarbons by ring fission.','2,6-dioxo-6-phenylhexanoate + NADP(+) = 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.41','Xanthommatin reductase','NULL','NULL','From Drosophila melanogaster.','5,12-dihydroxanthommatin + NAD(+) = xanthommatin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.42','12-oxophytodienoate reductase','NULL','NULL','Involved in the conversion of linolenate into jasmonate in Zea mays.','8-((1R,2R)-3-oxo-2-((Z)-pent-2-enyl)cyclopentyl)octanoate + NADP(+) =(15Z)-12-oxophyto-10,15-dienoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.43','Arogenate dehydrogenase','Arogenic dehydrogenase','NULL','See also EC 1.3.1.12, EC 1.3.1.78 and EC 1.3.1.79.','L-arogenate + NAD(+) = L-tyrosine + NADH + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.44','Trans-2-enoyl-CoA reductase (NAD(+))','NULL','NULL','The enzyme from Euglena gracilis acts on crotonoyl-CoA and, moreslowly, on trans-hex-2-enoyl-CoA and trans-oct-2-enoyl-CoA.','Acyl-CoA + NAD(+) = trans-didehydroacyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.45','2-hydroxyisoflavone reductase','NULL','NULL','In the reverse reaction, a 2-hydroxyisoflavone is reduced to anisoflavanone; 2-hydroxypseudobaptigenin also acts.Involved in the biosynthesis of the pterocarpin phytoalexinsmedicarpin and maackiain.','Vestitone + NADP(+) = 2-hydroxyformononetin + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.46','Biochanin-A reductase','NULL','NULL','Some other isoflavones are reduced to the correspondingisoflavanones.','Dihydrobiochanin A + NADP(+) = biochanin A + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.47','Alpha-santonin 1,2-reductase','NULL','NULL','NULL','1,2-dihydrosantonin + NAD(P)(+) = alpha-santonin + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.48','15-oxoprostaglandin 13-oxidase','15-oxo-Delta(13)-prostaglandin reductase','NULL','Reduces 15-oxoprostaglandins to 13,14-dihydro derivatives.The enzyme from placenta is specific for NAD(+).','11-alpha-hydroxy-9,15-dioxoprost-5-enoate + NAD(P)(+) = (5Z)-(13E)-11-alpha-hydroxy-9,15-dioxoprosta-5,13-dienoate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.49','Cis-3,4-dihydrophenanthrene-3,4-diol dehydrogenase','NULL','NULL','NULL','(+)-cis-3,4-dihydrophenanthrene-3,4-diol + NAD(+) = phenanthrene-3,4-diol+ NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.50','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.51','2-hydroxydaidzein reductase','2-hydroxydihydrodaidzein:NADP(+) 2-oxidoreductase','NULL','In the reverse reaction, the 2-hydroxyisoflavone(2-hydroxydaidzein) is reduced to an isoflavanone.Also acts on 2-hydroxyformononetin and to a small extent on2-hydroxygenistein.Involved in the biosynthesis of the phytoalexin glyceollin.The isoflavones biochanin A, daidzein and genestein as well as theflavonoids apigenin, kaempferol and quercetin do not act assubstrates.','2-hydroxy-2,3-dihydrodaidzein + NADP(+) = 2-hydroxydaidzein + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.52','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.53','(3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate dehydrogenase','(1R,2S)-dihydroxy-3,5-cyclohexadiene-1,4-dicarboxylate dehydrogenase','Fe(2+).','Involved in the terephthalate degradation pathway in bacteria.Formerly EC 1.3.1.61.','(3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate + NAD(+) =3,4-dihydroxybenzoate + CO(2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.54','Precorrin-6A reductase','Precorrin-6X reductase','NULL','NULL','Precorrin-6B + NADP(+) = precorrin-6A + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.55','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.56','Cis-2,3-dihydrobiphenyl-2,3-diol dehydrogenase','2,3-dihydro-2,3-dihydroxybiphenyl dehydrogenase','NULL','Catalyzes the second step in the biphenyl degradation pathway inbacteria.','Cis-3-phenylcyclohexa-3,5-diene-1,2-diol + NAD(+) = biphenyl-2,3-diol +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.57','Phloroglucinol reductase','NULL','NULL','Involved in the gallate anaerobic degradation pathway in bacteria.','Dihydrophloroglucinol + NADP(+) = phloroglucinol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.58','2,3-dihydroxy-2,3-dihydro-p-cumate dehydrogenase','Biphenyl-2,3-dihydro-2,3-diol dehydrogenase','NULL','Involved in the p-cymene and p-cumate degradation pathways inPseudomonas putida.','Cis-5,6-dihydroxy-4-isopropylcyclohexa-1,3-dienecarboxylate + NAD(+) =2,3-dihydroxy-p-cumate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.59','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.60','Dibenzothiophene dihydrodiol dehydrogenase','NULL','NULL','Involved in the dibenzothiophene degradation pathway in bacteria.','Cis-1,2-dihydroxy-1,2-dihydrodibenzothiophene + NAD(+) = 1,2-dihydroxydibenzothiophene + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.61','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.62','Pimeloyl-CoA dehydrogenase','NULL','NULL','Involved in the benzoate degradation (anaerobic) pathway in bacteria.','Pimeloyl-CoA + NAD(+) = 6-carboxyhex-2-enoyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.63','2,4-dichlorobenzoyl-CoA reductase','NULL','NULL','Acts in the reverse direction to form part of the 2,4-dichlorobenzoate degradation pathway in bacteria.','4-chlorobenzoyl-CoA + NADP(+) + chloride = 2,4-dichlorobenzoyl-CoA +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.64','Phthalate 4,5-cis-dihydrodiol dehydrogenase','NULL','NULL','Involved in the phthalate degradation pathway in bacteria.','Cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,2-dicarboxylate + NAD(+) = 4,5-dihydroxyphthalate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.65','5,6-dihydroxy-3-methyl-2-oxo-1,2,5,6-tetrahydroquinoline dehydrogenase','NULL','NULL','Acts in the reverse direction to form part of the 3-methylquinolinedegradation pathway in bacteria.','5,6-dihydroxy-3-methyl-2-oxo-1,2,5,6-tetrahydroquinoline + NAD(+) = 5,6-dihydroxy-3-methyl-2-oxo-1,2-dihydroquinoline + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.66','Cis-dihydroethylcatechol dehydrogenase','NULL','NULL','Involved in the ethylbenzene degradation pathway in bacteria.','Cis-1,2-dihydro-3-ethylcatechol + NAD(+) = 3-ethylcatechol + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.67','Cis-1,2-dihydroxy-4-methylcyclohexa-3,5-diene-1-carboxylatedehydrogenase','NULL','NULL','Involved in the p-xylene degradation pathway in bacteria.','Cis-1,2-dihydroxy-4-methylcyclohexa-3,5-diene-1-carboxylate + NAD(P)(+) =4-methylcatechol + NAD(P)H + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.68','1,2-dihydroxy-6-methylcyclohexa-3,5-dienecarboxylate dehydrogenase','NULL','NULL','Involved in the o-xylene degradation pathway in bacteria.','1,2-dihydroxy-6-methylcyclohexa-3,5-dienecarboxylate + NAD(+) =3-methylcatechol + NADH + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.69','Zeatin reductase','NULL','NULL','Formerly EC 1.1.1.242.','Dihydrozeatin + NADP(+) = zeatin + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.70','Delta(14)-sterol reductase','C-14 sterol reductase','NULL','Acts on a range of steroids with a 14(15)-double bond.','4,4-dimethyl-5-alpha-cholesta-8,24-dien-3-beta-ol + NADP(+) = 4,4-dimethyl-5-alpha-cholesta-8,14,24-trien-3-beta-ol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.71','Delta(24(24(1)))-sterol reductase','C-24(28) sterol reductase','NULL','Acts on a range of steroids with a 24(24(1))-double bond.','Ergosterol + NADP(+) = ergosta-5,7,22,24(24(1))-tetraen-3-beta-ol +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.72','Delta(24)-sterol reductase','Lanosterol Delta(24)-reductase','NULL','Acts on a range of steroids with a 24(25)-double bond, includinglanosterol, desmosterol and zymosterol.','5-alpha-cholest-7-en-3-beta-ol + NADP(+) = 5-alpha-cholesta-7,24-dien-3-beta-ol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.73','1,2-dihydrovomilenine reductase','NULL','NULL','Forms part of the ajmaline biosynthesis pathway.','17-O-acetylnorajmaline + NADP(+) = 1,2-dihydrovomilenine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.74','2-alkenal reductase (NAD(P)(+))','2-alkenal reductase','NULL','Highly specific for 4-hydroxynon-2-enal and non-2-enal.Alk-2-enals of shorter chain have lower affinities.Exhibits high activities also for alk-2-enones such as but-3-en-2-oneand pent-3-en-2-one.Inactive with cyclohex-2-en-1-one and 12-oxophytodienoic acid.Involved in the detoxification of alpha,beta-unsaturated aldehydesand ketones (cf. EC 1.3.1.102).','A n-alkanal + NAD(P)(+) = an alk-2-enal + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.75','Divinyl chlorophyllide a 8-vinyl-reductase','4VCR','NULL','Also reduces divinyl protochlorophyllide to protochlorophyllide insome species, providing an alternative pathway.','Chlorophyllide a + NADP(+) = divinyl chlorophyllide a + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.76','Precorrin-2 dehydrogenase','Precorrin-2 oxidase','NULL','Catalyzes the second of three steps leading to the formation ofsiroheme from uroporphyrinogen III.The first step involves the donation of two S-adenosyl-L-methionine-derived methyl groups to carbons 2 and 7 of uroporphyrinogen III toform precorrin-2 (EC 2.1.1.107) and the third step involves thechelation of ferrous iron to sirohydrochlorin to form siroheme(EC 4.99.1.4).In Saccharomyces cerevisiae, the last two steps are carried out by asingle bifunctional enzyme, Met8p.In some bacteria, steps 1-3 are catalyzed by a single multifunctionalprotein called CysG, whereas in Bacillus megaterium, three separateenzymes carry out each of the steps, with SirC being responsible forthe above reaction.','Precorrin-2 + NAD(+) = sirohydrochlorin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.77','Anthocyanidin reductase','ANR','NULL','Forms 2,3-cis-flavan-3-ols.The isomeric 2,3-trans-flavan-3-ols are formed from flavan-3,4-diolsby EC 1.17.1.3.While the enzyme from the legume Medicago truncatula (MtANR) usesboth NADPH and NADH as reductant, that from the crucifer Arabidopsisthaliana (AtANR) uses only NADPH.Also, while the substrate preference of MtANR iscyanidin','A flavan-3-ol + 2 NAD(P)(+) = an anthocyanidin + 2 NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.78','Arogenate dehydrogenase (NADP(+))','Arogenic dehydrogenase','NULL','Arogenate dehydrogenases may utilize NAD(+) (EC 1.3.1.43), NADP(+)(EC 1.3.1.78), or both (EC 1.3.1.79).NADP(+)-dependent enzymes usually predominate in higher plants.The enzyme from the cyanobacterium Synechocystis sp. PCC 6803 and theTyrAAT1 isoform of the plant Arabidopsis thaliana cannot useprephenate as a substrate, while the Arabidopsis isoform TyrAAT2 canuse it very poorly.','L-arogenate + NADP(+) = L-tyrosine + NADPH + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.79','Arogenate dehydrogenase (NAD(P)(+))','Arogenic dehydrogenase','NULL','Arogenate dehydrogenases may utilize NAD(+) (EC 1.3.1.43), NADP(+)(EC 1.3.1.78), or both (EC 1.3.1.79).Enzymes that can utilize both cofactors have been reported from someProteobacteria, including Burkholderia caryophylli, Burkholderiacepacia, Pseudomonas marginata and Delftia acidovorans.','L-arogenate + NAD(P)(+) = L-tyrosine + NAD(P)H + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.80','Red chlorophyll catabolite reductase','RCC reductase','NULL','Chlorophyll degradation is a characteristic symptom of leafsenescence and fruit ripening.The reaction catalyzed by this enzyme requires reduced ferredoxin,which is generated either by NADPH through the pentose-phosphatepathway or by the action of photosystem I.This reaction takes place without release of the substrate fromEC 1.14.12.20.Depending on the plant species used as the source of enzyme, one oftwo possible C-1 epimers of primary fluorescent chlorophyllcatabolite (pFCC), pFCC-1 or pFCC-2, is normally formed, with allgenera or species within a family producing the same isomer.After modification and export, pFCCs are eventually imported into thevacuole, where the acidic environment causes their non-enzymicconversion into colorless breakdown products called non-fluorescentchlorophyll catabolites (NCCs).','Primary fluorescent chlorophyll catabolite + NADP(+) = red chlorophyllcatabolite + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.81','(+)-pulegone reductase','NULL','NULL','NADH cannot replace NADPH as reductant.The Delta(8,9)-double bond of (+)-cis-isopulegone and the Delta(1,2)-double bond of (+-)-piperitone are not substrates.The enzyme from peppermint (Mentha x piperita) converts (+)-pulegoneinto both (-)-menthone and (+)-isomenthone at a ratio of 70:30 fornative enzyme but it does not catalyze the reverse reaction.','(1) (-)-menthone + NADP(+) = (+)-pulegone + NADPH.(2) (+)-isomenthone + NADP(+) = (+)-pulegone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.82','(-)-isopiperitenone reductase','NULL','NULL','The reaction occurs in the opposite direction to that shown above.Participates in the menthol-biosynthesis pathway of Mentha plants.(+)-pulegone, (+)-cis-isopulegone and (-)-menthone are notsubstrates.Has a preference for NADPH as the reductant, with NADH being a poorsubstitute.Highly regioselective for the reduction of the endocyclic 1,2-doublebond, and is stereoselective, producing only the 1R-configuredproduct.','(+)-cis-isopulegone + NADP(+) = (-)-isopiperitenone + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.83','Geranylgeranyl diphosphate reductase','Geranylgeranyl reductase','NULL','This enzyme also acts on geranylgeranyl-chlorophyll a.The reaction occurs in three steps.Which order the three double bonds are reduced is not known.','Phytyl diphosphate + 3 NADP(+) = geranylgeranyl diphosphate + 3 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.84','Acrylyl-CoA reductase (NADPH)','NULL','Zinc.','Catalyzes a step in the 3-hydroxypropanoate/4-hydroxybutyrate cycle,an autotrophic CO(2) fixation pathway found in some thermoacidophilicarchaea.The reaction is catalyzed in the opposite direction to that shown.The enzyme from Sulfolobus tokodaii does not act on either NADH orcrotonyl-CoA.Different from EC 1.3.1.8, which acts only on enoyl-CoA derivativesof carbon chain length 4 to 16.','Propanoyl-CoA + NADP(+) = acrylyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.85','Crotonyl-CoA carboxylase/reductase','CCR','NULL','The reaction is catalyzed in the reverse direction.This enzyme, isolated from the bacterium Rhodobacter sphaeroides,catalyzes (E)-but-2-enoyl-CoA-dependent oxidation of NADPH in thepresence of CO(2).When CO(2) is absent, the enzyme catalyzes the reduction of (E)-but-2-enoyl-CoA to butanoyl-CoA, but with only 10% of maximal activity(relative to (E)-but-2-enoyl-CoA carboxylation).','(2S)-ethylmalonyl-CoA + NADP(+) = (E)-but-2-enoyl-CoA + CO(2) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.86','Crotonyl-CoA reductase','3-hydroxyacyl CoA reductase','NULL','Catalyzes the reaction in the reverse direction.This enzyme from Streptomyces collinus is specific for (E)-but-2-enoyl-CoA, and is proposed to provide butanoyl-CoA as a starter unitfor straight-chain fatty acid biosynthesis.','Butanoyl-CoA + NADP(+) = (E)-but-2-enoyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.87','3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)propanoate dehydrogenase','2,3-dihydroxy-2,3-dihydro-phenylpropionate dehydrogenase','NULL','This enzyme catalyzes a step in the pathway of phenylpropanoidcompounds degradation.Formerly EC 1.3.1.n1.','(1) 3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)propanoate + NAD(+) =3-(2,3-dihydroxyphenyl)propanoate + NADH.(2) (2E)-3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)prop-2-enoate +NAD(+) = (2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.88','tRNA-dihydrouridine(16/17) synthase (NAD(P)(+))','tRNA-dihydrouridine synthase 1','FAD.','The enzyme specifically modifies uracil(16) and uracil(17) in tRNA.','(1) 5,6-dihydrouracil(16) in tRNA + NAD(P)(+) = uracil(16) in tRNA +NAD(P)H.(2) 5,6-dihydrouracil(17) in tRNA + NAD(P)(+) = uracil(17) in tRNA +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.89','tRNA-dihydrouridine(47) synthase (NAD(P)(+))','tRNA-dihydrouridine synthase 3','FAD.','The enzyme specifically modifies uracil(47) in tRNA.','5,6-dihydrouracil(47) in tRNA + NAD(P)(+) = uracil(47) in tRNA + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.90','tRNA-dihydrouridine(20a/20b) synthase (NAD(P)(+))','NULL','FAD.','The enzyme specifically modifies uracil(20a) and uracil(20b) in tRNA.','(1) 5,6-dihydrouracil(20a) in tRNA + NAD(P)(+) = uracil(20a) in tRNA +NAD(P)H.(2) 5,6-dihydrouracil(20b) in tRNA + NAD(P)(+) = uracil(20b) in tRNA +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.91','tRNA-dihydrouridine(20) synthase (NAD(P)(+))','tRNA-dihydrouridine synthase 2','FAD.','The enzyme specifically modifies uracil(20) in tRNA.','5,6-dihydrouracil(20) in tRNA + NAD(P)(+) = uracil(20) in tRNA + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.92','Artemisinic aldehyde Delta(11(13)) reductase','NULL','NULL','Cloned from Artemisia annua.In addition to the reduction of artemisinic aldehyde it is also ableto a lesser extent to reduce artemisinic alcohol and artemisinicacid.Part of the biosyntheis of artemisinin.','(11R)-dihydroartemisinic aldehyde + NADP(+) = artemisinic aldehyde +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.93','Very-long-chain enoyl-CoA reductase','NULL','NULL','This is the fourth component of the elongase, a microsomal proteincomplex responsible for extending palmitoyl-CoA and stearoyl-CoA (andmodified forms thereof) to very-long-chain acyl CoAs.Cf. EC 2.3.1.199, EC 1.1.1.330 and EC 4.2.1.134.','A very-long-chain acyl-CoA + NADP(+) = a very-long-chain trans-2,3-dehydroacyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.94','Polyprenol reductase','NULL','NULL','The reaction occurs in the reverse direction with reduction of theterminal double bond next to the alcohol group.Isolated from human fetal brain tissue but present in all eukaryotes.In mammalian cells dolichols are predominantly 18-21 isoprene unitsin length.','Ditrans,polycis-dolichol + NADP(+) = ditrans,polycis-polyprenol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.95','Acryloyl-CoA reductase (NADH)','NULL','FAD.','The reaction is catalyzed in the opposite direction to that shown.The enzyme from the bacterium Clostridium propionicum is a complexthat includes an electron-transfer flavoprotein (ETF).The ETF is reduced by NADH and transfers the electrons to the activesite.Catalyzes a step in a pathway for L-alanine fermentation topropanoate.Cf. EC 1.3.1.84.','Propanoyl-CoA + NAD(+) = acryloyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.96','Botryococcus squalene synthase','NULL','NULL','Isolated from the green alga Botryococcus braunii BOT22.Acts in the reverse direction.cf. EC 2.5.1.21, where squalene is formed directly from farnesyldiphosphate.','Squalene + diphosphate + NADP(+) = presqualene diphosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.97','Botryococcene synthase','NULL','NULL','Isolated from the green alga Botryococcus braunii BOT22.Acts in the reverse direction.Involved in the production of botryococcenes, which are triterpenoidhydrocarbons of isoprenoid origin produced in large amount by thisalga.','C(30) botryococcene + NADP(+) + diphosphate = presqualene diphosphate +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.98','UDP-N-acetylmuramate dehydrogenase','UDP-GlcNAc-enoylpyruvate reductase','FAD.','NADH can to a lesser extent replace NADPH.Formerly EC 1.1.1.158.','UDP-N-acetyl-alpha-D-muramate + NADP(+) = UDP-N-acetyl-3-O-(1-carboxyvinyl)-alpha-D-glucosamine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.99','Iridoid synthase','NULL','NULL','Isolated from the plant Catharanthus roseus.The reaction may involve cyclization via a Diels-Alder or Michaelreaction.Iridoids are involved in the biosynthesis of many indole alkaloids.The cyclic hemiacetal is readily hydrolyzed to the correspondingdial.','(6E)-8-oxogeranial + NAD(P)H = cis-trans-nepetalactol + NAD(P)(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.100','Chanoclavine-I aldehyde reductase','NULL','FMN.','The enzyme participates in the biosynthesis of fumigaclavine C,an ergot alkaloid produced by some fungi of the Trichocomaceaefamily.The enzyme catalyzes the reduction of chanoclavine-I aldehyde todihydrochanoclavine-I aldehyde.This hydrolyzes spontaneously to form 6,8-dimethyl-6,7-didehydroergoline, which is converted to festuclavine by EC 1.5.1.44.','Dihydrochanoclavine-I aldehyde + NADP(+) = chanoclavine-I aldehyde +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.101','2,3-bis-O-geranylgeranyl-sn-glycerol 1-phosphate reductase (NAD(P)H)','2,3-digeranylgeranylglycerophospholipid reductase','FAD.','The enzyme from the archaeon Thermoplasma acidophilum is involved inthe biosynthesis of membrane lipids.In vivo the reaction occurs in the reverse direction with theformation of 2,3-bis-O-phytanyl-sn-glycerol 1-phosphate.cf. EC 1.3.99.34.','2,3-bis-(O-phytanyl)-sn-glycerol 1-phosphate + 8 NAD(P)(+) = 2,3-bis-(O-geranylgeranyl)-sn-glycerol 1-phosphate + 8 NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.102','2-alkenal reductase (NADP(+))','NADPH-dependent alkenal/one oxidoreductase','NULL','Shows highest activity with 1-nitrocyclohexene but also hassignificant activity with 2-methylpentenal and trans-cinnamaldehyde.Involved in the detoxification of alpha,beta-unsaturated aldehydesand ketones.Has very low activity with NAD as reductant (cf. EC 1.3.1.74).','An n-alkanal + NADP(+) = an alk-2-enal + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.103','2-haloacrylate reductase','NULL','NULL','The enzyme acts in the degradation pathway of unsaturatedorganohalogen compounds by the bacterium Burkholderia sp. WS.','(S)-2-chloropropanoate + NADP(+) = 2-chloroacrylate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.104','Enoyl-[acyl-carrier-protein] reductase','Acyl-ACP dehydrogenase','NULL','The enzyme completes each cycle of fatty acid elongation bycatalyzing the stereospecific reduction of the double bond atposition 2 of a growing fatty acid chain, while linked to the acyl-carrier protein, in an NADPH-dependent manner.This entry stands for enzymes whose stereo-specificity with respectto NADP(+) is not known (cf. EC 1.3.1.39, EC 1.3.1.10 andEC 1.3.1.9).','An acyl-[acyl-carrier protein] + NADP(+) = a trans-2,3-dehydroacyl-[acyl-carrier protein] + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.105','2-methylene-furan-3-one reductase','Enone oxidoreductase','NULL','In the fruit-ripening process of strawberry (Fragaria x ananassa) thereaction is catalyzed in the reverse direction from that shown.4-hydroxy-2,5-dimethylfuran-3(2H)-one is one of the major aromacompounds in the fruits.The enzyme has also been detected in tomato (Solanum lycopersicum)fruits.NADPH is the preferred cofactor.','4-hydroxy-2,5-dimethylfuran-3(2H)-one + NAD(P)(+) = 4-hydroxy-5-methyl-2-methylenefuran-3(2H)-one + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.106','Cobalt-precorrin-6A reductase','NULL','NULL','The enzyme catalyzes a step in the anaerobic (early cobalt insertion)pathway of adenosylcobalamin biosynthesis.The enzyme from the bacterium Bacillus megaterium has no activitywith NADPH.The equivalent enzyme in the aerobic pathway is EC 1.3.1.54.','Cobalt-precorrin-6B + NAD(+) = cobalt-precorrin-6A + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.1.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.2.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.2.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.2.3','L-galactonolactone dehydrogenase','Galactonolactone dehydrogenase','NULL','Catalyzes the final step in the biosynthesis of L-ascorbic acid inhigher plants and in nearly all higher animals with the exception ofprimates and some birds.Very specific for its substrate L-galactono-1,4-lactone asD-galactono-gamma-lactone, D-gulono-gamma-lactone, L-gulono-gamma-lactone, D-erythronic-gamma-lactone, D-xylonic-gamma-lactone,L-mannono-gamma-lactone, D-galactonate, D-glucuronate and D-gluconateare not substrates.FAD, NAD(+), NADP(+) and O(2) (cf. EC 1.3.3.12) cannot act aselectron acceptor.','L-galactono-1,4-lactone + 4 ferricytochrome c = L-dehydroascorbate +4 ferrocytochrome c + 4 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.3','Coproporphyrinogen oxidase','Coprogen oxidase','Iron.','NULL','Coproporphyrinogen-III + O(2) + 2 H(+) = protoporphyrinogen-IX + 2 CO(2)+ 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.4','Protoporphyrinogen oxidase','Protoporphyrinogen IX oxidase','FAD.','Also slowly oxidizes mesoporphyrinogen IX.','Protoporphyrinogen-IX + 3 O(2) = protoporphyrin-IX + 3 H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.5','Bilirubin oxidase','NULL','NULL','NULL','2 bilirubin + O(2) = 2 biliverdin + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.6','Acyl-CoA oxidase','NULL','FAD.','Acts on CoA derivatives of fatty acids with chain length from C(8) toC(18).','Acyl-CoA + O(2) = trans-2,3-dehydroacyl-CoA + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.7','Dihydrouracil oxidase','NULL','FMN.','Also oxidizes dihydrothymine to thymine.','5,6-dihydrouracil + O(2) = uracil + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.8','Tetrahydroberberine oxidase','THB oxidase','Flavoprotein.','(R)-tetrahydroberberines are not oxidized.Formerly EC 1.5.3.8.','(S)-tetrahydroberberine + 2 O(2) = berberine + 2 H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.9','Secologanin synthase','NULL','Heme-thiolate.','Secologanin is the precursor of the monoterpenoid indole alkaloidsand ipecac alkaloids.','Loganin + NADPH + O(2) = secologanin + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.10','Tryptophan alpha,beta-oxidase','L-tryptophan 2,3-oxidase','Heme.','The enzyme from Chromobacterium violaceum is specific for tryptophanderivatives possessing its carboxyl group free or as an amide orester, and an unsubstituted indole ring.Also catalyzes the alpha,beta dehydrogenation of L-tryptophan sidechains in peptides.The product of the reaction can hydrolyze spontaneously to formindolepyruvate.Formerly EC 1.4.3.17.','L-tryptophan + O(2) = alpha,beta-didehydrotryptophan + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.11','Pyrroloquinoline-quinone synthase','NULL','NULL','So far only a single turnover of the enzyme has been observed,and the pyrroloquinoline quinone remains bound to it; it is not yetknown what releases the product in the bacterium.','6-(2-amino-2-carboxyethyl)-7,8-dioxo-1,2,3,4,7,8-hexahydroquinoline-2,4-dicarboxylate + 3 O(2) = 4,5-dioxo-4,5-dihydro-1H-pyrrolo[2,3-f]quinoline-2,7,9-tricarboxylate + 2 H(2)O(2) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.12','L-galactonolactone oxidase','L-xylono-1,4-lactone oxidase','FAD.','Acts on the 1,4-lactones of L-galactonic, D-altronic, L-fuconic,D-arabinic and D-threonic acids.Not identical with EC 1.1.3.8 (cf. EC 1.3.2.3).Formerly EC 1.1.3.24.','L-galactono-1,4-lactone + O(2) = L-ascorbate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.13','Albonoursin synthase','Cyclic dipeptide oxidase','Flavoprotein.','The enzyme can also oxidize several other cyclo dipeptides, the bestbeing cyclo(L-tryptophyl-L-tryptophyl) and cyclo(L-phenylalanyl-L-phenylalanyl).','Cyclo(L-leucyl-L-phenylalanyl) + 2 O(2) = albonoursin + 2 H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.3.14','Aclacinomycin-A oxidase','Aclacinomycin oxidoreductase','FAD.','This bifunctional enzyme is a secreted flavin-dependent enzyme thatis involved in the modification of the terminal sugar residues in thebiosynthesis of aclacinomycins.The enzyme utilizes the same active site to catalyze the oxidation ofthe rhodinose moiety of aclacinomycin N to the cinerulose A moiety ofaclacinomycin A (cf. EC 1.1.3.45) and the oxidation of the latter tothe L-aculose moiety of aclacinomycin Y.','Aclacinomycin A + O(2) = aclacinomycin Y + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.5.1','Succinate dehydrogenase (quinone)','Succinate dehydrogenase (ubiquinone)','FAD; Iron-sulfur.','The enzyme is found in the inner mitochondrial membrane in eukaryotesand the plasma membrane of many aerobic or facultative bacteria.It catalyzes succinate oxidation in the citric acid cycle andtransfers the electrons to quinones in the membrane, thusconstituting a part of the aerobic respiratory chain (known ascomplex II).Eukaryotic enzymes utilize ubioquinone while bacterial enzymesutilize either ubiquinone or menaquinone, depending on the organism.Cf. EC 1.3.5.4.','Succinate + a quinone = fumarate + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.5.2','Dihydroorotate dehydrogenase (quinone)','DHOD','FMN.','This Class 2 dihydroorotate dehydrogenase enzyme contains FMN.The enzyme is found in eukaryotes in the mitochondrial membrane,and in some Gram negative bacteria associated with the cytoplasmicmembrane.The reaction is the only redox reaction in the de-novo biosynthesisof pyrimidine nucleotides.The best quinone electron acceptors for the enzyme from bovine liverare ubiquinone-6 and ubiquinone-7, although simple quinones, such asbenzoquinone, can also act as acceptor at lower rates.Methyl-, ethyl-, tert-butyl and benzyl-(S)-dihydroorotates are alsosubstrates, but 1- and 3-methyl and 1,3-dimethyl methyl-(S)-dihydroorotates are not.Class 1 dihydroorotate dehydrogenases use either fumarate(EC 1.3.98.1), NAD(+) (EC 1.3.1.14) or NADP(+) (EC 1.3.1.15) aselectron acceptor.Formerly EC 1.3.99.11.','(S)-dihydroorotate + a quinone = orotate + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.5.3','Protoporphyrinogen IX dehydrogenase (menaquinone)','NULL','NULL','This enzyme enables Escherichia coli to synthesize heme in bothaerobic and anaerobic environments.','Protoporphyrinogen IX + 3 menaquinone = protoporphyrin IX + 3 menaquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.5.4','Fumarate reductase (quinol)','Fumarate reductase (menaquinone)','NULL','The enzyme, which is found in anaerobic and facultative organismssuch as bacteria, parasitic helminthes, and lower marine organisms,utilizes low potential quinols, such as menaquinol and rhodoquinol,to reduce fumarate as the final step of an anaerobic respiratorychain.The enzyme is known as complex II of the electron transfer chain,similarly to EC 1.3.5.1, to which it is closely related.','Succinate + a quinone = fumarate + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.5.5','15-cis-phytoene desaturase','PDS','NULL','This enzyme is involved in carotenoid biosynthesis in plants andcyanobacteria.The enzyme from Synechococcus can also use NAD(+) and NADP(+) aselectron acceptor under anaerobic conditions.The enzyme from Gentiana lutea shows no activity with NAD(+) orNADP(+).','15-cis-phytoene + 2 plastoquinone = 9,15,9-tri-cis-zeta-carotene +2 plastoquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.5.6','9,9-di-cis-zeta-carotene desaturase','ZDS','NULL','This enzyme is involved in carotenoid biosynthesis in plants andcyanobacteria.Formerly EC 1.14.99.30.','9,9-di-cis-zeta-carotene + 2 quinone = 7,9,7,9-tetra-cis-lycopene +2 quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.1','6-hydroxynicotinate reductase','6-hydroxynicotinic reductase','NULL','Forms part of the nicotinate-fermentation catabolism pathway inEubacterium barkeri.Other enzymes involved in this pathway are EC 1.17.5.1, EC 3.5.2.18,EC 1.1.1.291, EC 5.4.99.4, EC 5.3.3.6, EC 4.2.1.85 and EC 4.1.3.32.','6-oxo-1,4,5,6-tetrahydronicotinate + oxidized ferredoxin =6-hydroxynicotinate + reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.2','15,16-dihydrobiliverdin:ferredoxin oxidoreductase','NULL','NULL','Catalyzes the two-electron reduction of biliverdin IX-alpha at theC15 methine bridge.It has been proposed that this enzyme and EC 1.3.7.3 function as adual enzyme complex in the conversion of biliverdin IX-alpha intophycoerythrobilin.','15,16-dihydrobiliverdin + oxidized ferredoxin = biliverdin IX-alpha +reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.3','Phycoerythrobilin:ferredoxin oxidoreductase','NULL','NULL','Catalyzes the two-electron reduction of the C2 and C3(1) diene systemof 15,16-dihydrobiliverdin.Specific for 15,16-dihydrobiliverdin.It has been proposed that this enzyme and EC 1.3.7.2 function as adual enzyme complex in the conversion of biliverdin IX-alpha intophycoerythrobilin.','(3Z)-phycoerythrobilin + oxidized ferredoxin = 15,16-dihydrobiliverdin +reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.4','Phytochromobilin:ferredoxin oxidoreductase','P-Phi-B synthase','NULL','Catalyzes the two-electron reduction of biliverdin IX-alpha.Can use [2Fe-2S] ferredoxins from a number of sources as acceptor butnot the [4Fe-4S] ferredoxin from Clostridium pasteurianum.The isomerization of (3Z)-phytochromobilin to (3E)-phytochromobilinis thought to occur prior to covalent attachment to apophytochrome inthe plant cell cytoplasm.Flavodoxins can be used instead of ferredoxin.','(3Z)-phytochromobilin + oxidized ferredoxin = biliverdin IX-alpha +reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.5','Phycocyanobilin:ferredoxin oxidoreductase','NULL','NULL','Catalyzes the four-electron reduction of biliverdin IX-alpha(2-electron reduction at both the A and D rings).Reaction proceeds via an isolatable 2-electron intermediate,18(1),18(2)-dihydrobiliverdin.Flavodoxins can be used instead of ferredoxin.The direct conversion of biliverdin IX-alpha (BV) to (3Z)-phycocyanobilin (PCB) in the cyanobacteria Synechocystis sp.PCC 6803, Anabaena sp. PCC7120 and Nostoc punctiforme is in contrastto the proposed pathways of PCB biosynthesis in the red algaCyanidium caldarium, which involves (3Z)-phycoerythrobilin (PEB) asan intermediate and in the green alga Mesotaenium caldariorum,in which PCB is an isolable intermediate.','(3Z)-phycocyanobilin + oxidized ferredoxin = biliverdin IX-alpha +reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.6','Phycoerythrobilin synthase','NULL','NULL','This enzyme, from a cyanophage infecting oceanic cyanobacteria of theProchlorococcus genus, uses a four-electron reduction to carry outthe reactions catalyzed by EC 1.3.7.2 and EC 1.3.7.3.15,16-dihydrobiliverdin is formed as a bound intermediate.Free 15,16-dihydrobiliverdin can also act as a substrate to formphycoerythrobilin.','(3Z)-phycoerythrobilin + 2 oxidized ferredoxin = biliverdin IX-alpha +2 reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.7','Ferredoxin:protochlorophyllide reductase (ATP-dependent)','Light-independent protochlorophyllide reductase','Iron-sulfur.','Occurs in photosynthetic bacteria, cyanobacteria, green algae andgymnosperms.The enzyme catalyzes trans-reduction of the D-ring ofprotochlorophyllide; the product has the (7S,8S)-configuration.Unlike EC 1.3.1.33 light is not required.The enzyme contains a [4Fe-4S] cluster, and structurally resemblesthe Fe protein/MoFe protein complex of EC 1.18.6.1, which catalyzesan ATP-driven reduction.','Protochlorophyllide a + reduced ferredoxin + 2 ATP + 2 H(2)O =chlorophyllide a + oxidized ferredoxin + 2 ADP + 2 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.8','Benzoyl-CoA reductase','NULL','Iron-sulfur; Magnesium or manganese.','Inactive toward aromatic acids that are not CoA esters but will alsocatalyze the reaction: NH(3) + acceptor + 2 ADP + 2 phosphate =hydroxylamine + reduced acceptor + 2 ATP + H(2)O.In the presence of reduced acceptor, but in the absence of oxidizablesubstrate, the enzyme catalyzes the hydrolysis of ATP to ADP plusphosphate.Formerly EC 1.3.99.15.','Cyclohexa-1,5-diene-1-carbonyl-CoA + oxidized ferredoxin + 2 ADP +2 phosphate = benzoyl-CoA + reduced ferredoxin + 2 ATP + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.9','4-hydroxybenzoyl-CoA reductase','NULL','Flavin; Iron-sulfur; Molybdenum.','Involved in the anaerobic pathway of phenol metabolism in bacteria.A ferredoxin with two [4Fe-4S] clusters functions as the naturalelectron donor.Formerly EC 1.3.99.20.','Benzoyl-CoA + oxidized ferredoxin = 4-hydroxybenzoyl-CoA +reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.7.10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.1','Short-chain acyl-CoA dehydrogenase','Butanoyl-CoA dehydrogenase','FAD.','One of several enzymes that catalyze the first step in fatty acidsbeta-oxidation.The enzyme catalyzes the oxidation of saturated short-chain acyl-CoAthioesters to give a trans 2,3-unsaturated product by removal of thetwo pro-R-hydrogen atoms.The enzyme from beef liver accepts substrates with acyl chain lengthsof 3 to 8 carbon atoms.The highest activity was reported with either butanoyl-CoA orpentanoyl-CoA.The enzyme from rat has only 10% activity with hexanoyl-CoA (comparedto butanoyl-CoA) and no activity with octanoyl-CoA.Cf. EC 1.3.8.7, EC 1.3.8.8 and EC 1.3.8.9.Formerly EC 1.3.2.1 and EC 1.3.99.2.','Butanoyl-CoA + electron-transfer flavoprotein = 2-butenoyl-CoA + reducedelectron-transfer flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.2','4,4-diapophytoene desaturase','Dehydrosqualene desaturase','NULL','Typical of Staphylococcus aureus and some other bacteria such asHeliobacillus sp.Responsible for four successive dehydrogenations.In some species it only proceeds as far as all-trans-4,4-diaponeurosporene.','15-cis-4,4-diapophytoene + 4 FAD = all-trans-4,4-diapolycopene +4 FADH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.3','(R)-benzylsuccinyl-CoA dehydrogenase','NULL','FAD.','Unlike other acyl-CoA dehydrogenases, this enzyme exhibits highsubstrate- and enantiomer specificity it is highly specific for (R)-benzylsuccinyl-CoA and is inhibited by (S)-benzylsuccinyl-CoA.Forms the third step in the anaerobic toluene metabolic pathway inThauera aromatica.Ferricenium ion is an effective artificial electron acceptor.Formerly EC 1.3.99.21.','(R)-2-benzylsuccinyl-CoA + electron-transferring flavoprotein = (E)-2-benzylidenesuccinyl-CoA + reduced electron-transferring flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.4','Isovaleryl-CoA dehydrogenase','NULL','FAD.','Pentanoate can act as donor.Formerly EC 1.3.99.10.','Isovaleryl-CoA + electron-transfer flavoprotein = 3-methylcrotonyl-CoA +reduced electron-transfer flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.5','2-methyl-branched-chain-enoyl-CoA reductase','NULL','FAD.','From Ascaris suum.The reaction functions in shuttling reducing power from the electron-transport chain to 2-methyl branched-chain enoyl CoA.Formerly EC 1.3.1.52.','2-methylbutanoyl-CoA + electron-transfer flavoprotein = (E)-2-methylbut-2-enoyl-CoA + reduced electron-transfer flavoprotein + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.6','Glutaryl-CoA dehydrogenase (ETF)','Glutaryl-CoA dehydrogenase','FAD.','The enzyme catalyzes the oxidation of glutaryl-CoA to glutaconyl-CoA(which remains bound to the enzyme), and the decarboxylation of thelatter to crotonyl-CoA (cf. EC 4.1.1.70).FAD is the electron acceptor in the oxidation of the substrate,and its reoxidation by electron-transfer flavoprotein completes thecatalytic cycle.The anaerobic, sulfate-reducing bacterium Desulfococcus multivoranscontains two glutaryl-CoA dehydrogenases: a decarboxylating enzyme(this entry), and a non-decarboxylating enzyme that only catalyzesthe oxidation to glutaconyl-CoA (EC 1.3.99.32).Formerly EC 1.3.99.7.','Glutaryl-CoA + electron-transfer flavoprotein = crotonyl-CoA + CO(2) +reduced electron-transfer flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.7','Medium-chain acyl-CoA dehydrogenase','Acyl CoA dehydrogenase','NULL','One of several enzymes that catalyze the first step in fatty acidsbeta-oxidation.The enzyme from pig liver can accept substrates with acyl chainlengths of 4 to 16 carbon atoms, but is most active with C(8) toC(12) compounds.The enzyme from rat does not accept C(16) at all and is most activewith C(6)-C(8) compounds.cf. EC 1.3.8.1, EC 1.3.8.8 and EC 1.3.8.9.Formerly EC 1.3.2.2 and EC 1.3.99.3.','A medium-chain acyl-CoA + electron-transfer flavoprotein = a medium-chaintrans-2,3-dehydroacyl-CoA + reduced electron-transfer flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.8','Long-chain-acyl-CoA dehydrogenase','NULL','FAD.','One of several enzymes that catalyze the first step in fatty acidsbeta-oxidation.The enzyme from pig liver can accept substrates with acyl chainlengths of 6 to at least 16 carbon atoms.The highest activity was found with C(12), and the rates with C(8)and C(16) were 80% and 70%, respectively.The enzyme from rat can accept substrates with C(8)-C(22).It is most active with C(14) and C(16), and has no activity withC(4), C(6) or C(24).cf. EC 1.3.8.1, EC 1.3.8.8 and EC 1.3.8.9.Formerly EC 1.3.99.3 and EC 1.3.99.13.','Long-chain-acyl-CoA + electron-transfer flavoprotein = long-chain-2,3-dehydroacyl-CoA + reduced electron-transfer flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.9','Very-long-chain acyl-CoA dehydrogenase','NULL','FAD.','One of several enzymes that catalyze the first step in fatty acidsbeta-oxidation.The enzyme is most active toward long-chain acyl-CoAs such as C(14),C(16) and C(18), but is also active with very-long-chain acyl-CoAs upto 24 carbons.It shows no activity for substrates of less than 12 carbons.Its specific activity toward palmitoyl-CoA is more than 10-fold thatof the long-chain acyl-CoA dehydrogenase.cf. EC 1.3.8.1, EC 1.3.8.7 and EC 1.3.8.8.Formerly EC 1.3.99.3.','A very-long-chain acyl-CoA + electron-transfer flavoprotein = a very-long-chain trans-2,3-dehydroacyl-CoA + reduced electron-transferflavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.10','Cyclohex-1-ene-1-carbonyl-CoA dehydrogenase','NULL','FAD.','The enzyme, characterized from the strict anaerobic bacteriumSyntrophus aciditrophicus, is involved in production of cyclohexane-1-carboxylate, a byproduct produced by that organism duringfermentation of benzoate and crotonate to acetate.','Cyclohex-1-ene-1-carbonyl-CoA + electron-transfer flavoprotein = (E)-2-cyclohex-1,5-diene-1-carbonyl-CoA + reduced electron-transferflavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.8.11','Cyclohexane-1-carbonyl-CoA dehydrogenase','NULL','FAD.','The enzyme, characterized from the strict anaerobic bacteriumSyntrophus aciditrophicus, is involved in production of cyclohexane-1-carboxylate, a byproduct produced by that organism duringfermentation of benzoate and crotonate to acetate.','Cyclohexane-1-carbonyl-CoA + electron-transfer flavoprotein = cyclohex-1-ene-1-carbonyl-CoA + reduced electron-transfer flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.98.1','Dihydroorotate oxidase (fumarate)','DHOD','FMN.','The reaction, which takes place in the cytosol, is the only redoxreaction in the de novo biosynthesis of pyrimidine nucleotides.Molecular oxygen can replace fumarate in vitro.Other class 1 dihydroorotate dehydrogenases use either NAD(+)(EC 1.3.1.14) or NADP(+) (EC 1.3.1.15) as electron acceptor.The membrane bound class 2 dihydroorotate dehydrogenase (EC 1.3.5.2)uses quinone as electron acceptor.Formerly EC 1.3.3.1.','(S)-dihydroorotate + fumarate = orotate + succinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.98.2','Fumarate reductase (CoM/CoB)','Thiol:fumarate reductase','NULL','The enzyme, isolated from the archaeon Methanobacteriumthermoautotrophicum, is very oxygen sensitive.It cannot use reduced flavins, reduced coenzyme F420, or NAD(P)H asan electron donor.Distinct from EC 1.3.1.6, EC 1.3.5.1, and EC 1.3.5.4.','Fumarate + CoM + CoB = succinate + CoM-S-S-CoB.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.1','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.4','3-oxosteroid 1-dehydrogenase','NULL','NULL','NULL','A 3-oxosteroid + acceptor = a 3-oxo-Delta(1)-steroid + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.5','3-oxo-5-alpha-steroid 4-dehydrogenase (acceptor)','Steroid 5-alpha-reductase','Flavoprotein.','This bacterial enzyme, characterized from Comamonas testosteroni,is involved in androsterone degradation.Cf. EC 1.3.1.22.','A 3-oxo-5-alpha-steroid + acceptor = a 3-oxo-Delta(4)-steroid + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.6','3-oxo-5-beta-steroid 4-dehydrogenase','Delta(4)-3-ketosteroid 5-beta-reductase','NULL','NULL','A 3-oxo-5-beta-steroid + acceptor = a 3-oxo-Delta(4)-steroid + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.8','2-furoyl-CoA dehydrogenase','Furoyl-CoA hydroxylase','Copper.','The oxygen atom of the -OH produced is derived from water, not O(2).The actual oxidative step is probably dehydrogenation of a hydratedform -CHOH-CH(2)- to -C(OH)=CH-, which tautomerizes non-enzymicallyto -CO-CH(2)-, giving (5-oxo-4,5-dihydro-2-furoyl)-CoA.Methylene blue, nitro blue tetrazolium and a membrane fraction fromPseudomonas putida can act as acceptors.','2-furoyl-CoA + H(2)O + acceptor = S-(5-hydroxy-2-furoyl)-CoA + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.11','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.12','2-methylacyl-CoA dehydrogenase','Branched-chain acyl-CoA dehydrogenase','FAD.','Also oxidizes 2-methylpropanoyl-CoA.Not identical with EC 1.3.8.1, EC 1.3.8.7, EC 1.3.8.8 or EC 1.3.8.9.','2-methylbutanoyl-CoA + acceptor = 2-methylbut-2-enoyl-CoA + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.13','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.14','Cyclohexanone dehydrogenase','NULL','NULL','2,6-dichloroindophenol can act as acceptor.The corresponding ketones of cyclopentane and cycloheptane cannot actas donors.','Cyclohexanone + acceptor = cyclohex-2-enone + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.15','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.16','Isoquinoline 1-oxidoreductase','NULL','NULL','The enzyme from Pseudomonas diminuta is specific toward N-containingN-heterocyclic substrates, including isoquinoline, isoquinolin-5-ol,phthalazine and quinazoline.Electron acceptors include 1,2-benzoquinone, cytochrome c,ferricyanide, iodonitrotetrazolium chloride (INT), nitrobluetetrazolium (NBT), Meldola blue and phenazine methosulfate.','Isoquinoline + acceptor + H(2)O = isoquinolin-1(2H)-one + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.17','Quinoline 2-oxidoreductase','NULL','FAD; Iron-sulfur; Molybdenum.','In addition to quinoline, quinolin-2-ol, quinolin-7-ol, quinolin-8-ol, 3-, 4- and 8-methylquinolines and 8-chloroquinoline aresubstrates.Iodonitrotetrazolium chloride (INT) can act as an electron acceptor.','Quinoline + acceptor + H(2)O = isoquinolin-1(2H)-one + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.18','Quinaldate 4-oxidoreductase','Quinaldic acid 4-oxidoreductase','NULL','The enzyme from Pseudomonas sp. AK2 also acts on quinoline-8-carboxylate, whereas that from Serratia marcescens 2CC-1 will oxidizenicotinate; quinaldate is a substrate for both of these enzymes.2,4,6-trinitrobenzene sulfonate acid, 1,4-benzoquinone, 1,2-naphthoquinone, nitroblue tetrazolium (NBT), thionine and menadionewill serve as an electron acceptor for the former enzyme andferricyanide for the latter; Meldola blue, iodonitrotetrazoliumchloride (INT), phenazine methosulfate (PMS), 2,6-dichlorophenolindophenol (DCPIP) and cytochrome c will act aselectron acceptors for both.','Quinaldate + acceptor + H(2)O = kynurenate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.19','Quinoline-4-carboxylate 2-oxidoreductase','Quinaldic acid 4-oxidoreductase','Flavin; Iron-sulfur; Molybdopterin cytosine dinucleotide.','Quinoline, 4-methylquinoline and 4-chloroquinoline can also serve assubstrates for the enzyme from Agrobacterium sp. 1B.Iodonitrotetrazolium chloride, thionine, menadione and 2,6-dichlorophenolindophenol can act as electron acceptors.','Quinoline-4-carboxylate + acceptor + H(2)O = 2-oxo-1,2-dihydroquinoline-4-carboxylate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.20','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.21','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.22','Coproporphyrinogen dehydrogenase','Coproporphyrinogen III oxidase','Iron-sulfur.','Differs from EC 1.3.3.3 by using S-adenosyl-L-methionine (AdoMet)instead of oxygen as oxidant.Occurs mainly in bacteria, whereas eukaryotes use the oxygen-dependent oxidase.The reaction starts by using an electron from the reduced form of theenzymes [4Fe-4S] cluster to split AdoMet into methionine and theradical 5-deoxyadenosin-5-yl; this radical initiates attack on the2-carboxyethyl groups, leading to their conversion into vinyl groups.This conversion, -.CH-CH(2)-COO- -','Coproporphyrinogen-III + 2 S-adenosyl-L-methionine = protoporphyrinogen-IX + 2 CO(2) + 2 L-methionine + 2 5-deoxyadenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.23','All-trans-retinol 13,14-reductase','(13,14)-all-trans-retinol saturase','NULL','The reaction is only known to occur in the opposite direction to thatgiven above, with the enzyme being specific for all-trans-retinol assubstrate.Neither all-trans-retinoic acid nor 9-cis, 11-cis or 13-cis-retinolisomers are substrates.May play a role in the metabolism of vitamin A.','All-trans-13,14-dihydroretinol + acceptor = all-trans-retinol + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.24','2-amino-4-deoxychorismate dehydrogenase','2-amino-2-deoxyisochorismate dehydrogenase','NULL','The sequential action of EC 2.6.1.86 and this enzyme leads to theformation of the benzoxazolinate moiety of the enediyne antitumorantibiotic C-1027.','(2S)-2-amino-4-deoxychorismate + FMN = 3-(1-carboxyvinyloxy)anthranilate+ FMNH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.25','Carvone reductase','NULL','NULL','Participates in the carveol and dihydrocarveol degradation pathway ofthe Gram-positive bacterium Rhodococcus erythropolis DCL14.Has not been purified, and requires an unknown cofactor, which isdifferent from NAD(+), NADP(+) or a flavin.','(1) (+)-dihydrocarvone + acceptor = (-)-carvone + reduced acceptor.(2) (-)-isodihydrocarvone + acceptor = (+)-carvone + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.26','All-trans-zeta-carotene desaturase','2-step phytoene desaturase','NULL','This enzyme is involved in carotenoid biosynthesis.','All-trans-zeta-carotene + 2 acceptor = all-trans-lycopene + 2 reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.27','1-hydroxycarotenoid 3,4-desaturase','1-hydroxycarotenoid 3,4-dehydrogenase','NULL','The enzymes from Rubrivivax gelatinosus and Rhodobacter sphaeroidesprefer the acyclic carotenoids (e.g. 1-hydroxy-1,2-dihydroneurosporene, 1-hydroxy-1,2-dihydrolycopene) as substrates.The conversion rate for the 3,4-desaturation of the monocyclic1-hydroxy-1,2-dihydro-gamma-carotene is lower.The enzyme from the marine bacterium strain P99-3 shows high activitywith the monocyclic carotenoid 1-hydroxy-1,2-dihydro-gamma-carotene.The enzyme from Rhodobacter sphaeroides utilizes molecular oxygen asthe electron acceptor in vitro.However, oxygen is unlikely to be the natural electron acceptor underanaerobic conditions.','1-hydroxy-1,2-dihydrolycopene + acceptor = 1-hydroxy-3,4-didehydro-1,2-dihydrolycopene + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.28','Phytoene desaturase (neurosporene-forming)','3-step phytoene desaturase','NULL','This enzyme is involved in carotenoid biosynthesis and catalyzes upto three desaturation steps (cf. EC 1.3.99.29, EC 1.3.99.30,EC 1.3.99.31).The enzyme is activated by FAD.NAD(+), NADP(+) or ATP show no activating effect.','15-cis-phytoene + 3 acceptor = all-trans-neurosporene +3 reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.29','Phytoene desaturase (zeta-carotene-forming)','2-step phytoene desaturase','NULL','The enzyme is involved in carotenoid biosynthesis and catalyzes up totwo desaturation steps (cf. EC 1.3.99.28, EC 1.3.99.30 andEC 1.3.99.31).','15-cis-phytoene + 2 acceptor = all-trans-zeta-carotene + 2 reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.30','Phytoene desaturase (3,4-didehydrolycopene-forming)','5-step phytoene desaturase','NULL','This enzyme is involved in carotenoid biosynthesis and catalyzes upto five desaturation steps (cf. EC 1.3.99.28, EC 1.3.99.29 andEC 1.3.99.31).','15-cis-phytoene + 5 acceptor = all-trans-3,4-didehydrolycopene +5 reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.31','Phytoene desaturase (lycopene-forming)','4-step phytoene desaturase','FAD.','The enzyme is involved in carotenoid biosynthesis and catalyzes up tofour desaturation steps (cf. EC 1.3.99.28, EC 1.3.99.29 andEC 1.3.99.30).','15-cis-phytoene + 4 acceptor = all-trans-lycopene + 4 reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.32','Glutaryl-CoA dehydrogenase (acceptor)','NULL','FAD.','The anaerobic, sulfate-reducing bacterium Desulfococcus multivoranscontains two glutaryl-CoA dehydrogenases: a decarboxylating enzyme(EC 1.3.8.6), and a nondecarboxylating enzyme (this entry).The two enzymes cause different structural changes around theglutaconyl carboxylate group, primarily due to the presence of eithera tyrosine or a valine residue, respectively, at the active site.','Glutaryl-CoA + acceptor = (E)-glutaconyl-CoA + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.33','Urocanate reductase','NULL','FAD; FMN.','The enzyme functions as part of an anaerobic electron transfer chainthat utilizes urocanate as the terminal electron acceptor.The activity has been demonstrated with the artificial donor reducedmethyl viologen.','Dihydrourocanate + acceptor = urocanate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.34','2,3-bis-O-geranylgeranyl-sn-glycerol 1-phosphate reductase (donor)','NULL','FAD.','The enzyme is involved in the biosynthesis of archaeal membranelipids.It catalyzes the reduction of all 8 double bonds in 2,3-bis-O-geranylgeranyl-sn-glycerol 1-phosphate and all 4 double bonds in 3-O-geranylgeranyl-sn-glycerol 1-phosphate with comparable activity.Geranylgeranyl diphosphate is partially reduced to phytyl diphosphatewith 10% of the activity (cf. EC 1.3.1.83).Unlike EC 1.3.1.101, the enzyme shows no activity with NADPH.In vitro the enzyme can utilize sodium dithionite as the electrondonor.','2,3-bis-(O-phytanyl)-sn-glycerol 1-phosphate + acceptor = 2,3-bis-(O-geranylgeranyl)-sn-glycerol 1-phosphate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.35','Chlorophyllide a reductase','NULL','Iron-sulfur.','Found in the purple non-sulfur bacterium Rhodobacter capsulatus.The enzyme catalyzes trans-reduction of the B-ring of chlorophyllidea; the product has the (7R,8R)-configuration.','3-deacetyl-3-vinylbacteriochlorophyllide a + A + ADP + phosphate =chlorophyllide a + AH(2) + ATP + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.n1','3-hydroxybenzoyl-CoA reductase','3-hydroxybenzoyl-CoA reductase (dearomatising)','Iron-sulfur; Magnesium or manganese.','Reduced methyl viologen can act as electron donor.The enzyme also catalyzes the reaction of EC 1.3.7.8.The reaction is catalyzed by a heterotetramer.','3-hydroxybenzoyl-CoA + reduced acceptor + 2 ATP + 2 H(2)O =3-hydroxycyclohexa-1,5-diene-1-carbonyl-CoA + acceptor + 2 ADP +2 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.n2','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.3.99.n3','Cypemycin cysteine dehydrogenase','Cypemycin decarboxylase','NULL','Cypemycin, isolated from the bacterium Streptomyces sp. OH-4156, is apeptide antibiotic, member of the linaridins, a class ofposttranslationally modified ribosomally synthesized peptides.The enzyme decarboxylates and reduces the C-terminal L-cysteineresidue, producing an active ethenethiol.The resulting group reacts with a second dethiolated cysteine to forman aminovinyl-methyl-cysteine loop that is important for theantibiotic activity of the mature peptide.','(1) L-cysteinate group + A + H(+) = (Z)-N-(2-mercaptoethenyl)amino group+ AH(2) + CO(2).(2) [Cypemycin]-Cys-C-term + acceptor = [cypemycin]-N-(Z)-ethenethiol +CO(2) + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.1','Alanine dehydrogenase','NULL','NULL','NULL','L-alanine + H(2)O + NAD(+) = pyruvate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.2','Glutamate dehydrogenase','Glutamic dehydrogenase','NULL','NULL','L-glutamate + H(2)O + NAD(+) = 2-oxoglutarate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.3','Glutamate dehydrogenase (NAD(P)(+))','Glutamic dehydrogenase','NULL','NULL','L-glutamate + H(2)O + NAD(P)(+) = 2-oxoglutarate + NH(3) + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.4','Glutamate dehydrogenase (NADP(+))','Glutamic dehydrogenase','NULL','NULL','L-glutamate + H(2)O + NADP(+) = 2-oxoglutarate + NH(3) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.5','L-amino-acid dehydrogenase','NULL','NULL','Acts on aliphatic amino acids.','An L-amino acid + H(2)O + NAD(+) = a 2-oxo acid + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.7','Serine 2-dehydrogenase','L-serine:NAD oxidoreductase (deaminating)','NULL','NULL','L-serine + H(2)O + NAD(+) = 3-hydroxypyruvate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.8','Valine dehydrogenase (NADP(+))','NULL','NULL','NULL','L-valine + H(2)O + NADP(+) = 3-methyl-2-oxobutanoate + NH(3) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.9','Leucine dehydrogenase','LeuDH','NULL','Also acts on isoleucine, valine, norvaline and norleucine.','L-leucine + H(2)O + NAD(+) = 4-methyl-2-oxopentanoate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.10','Glycine dehydrogenase','NULL','NULL','NULL','Glycine + H(2)O + NAD(+) = glyoxylate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.11','L-erythro-3,5-diaminohexanoate dehydrogenase','NULL','NULL','NULL','L-erythro-3,5-diaminohexanoate + H(2)O + NAD(+) = (S)-5-amino-3-oxohexanoate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.12','2,4-diaminopentanoate dehydrogenase','NULL','NULL','Also acts, more slowly, on 2,5-diaminohexanoate forming 2-amino-5-oxohexanoate, which then cyclizes non-enzymically to 1-pyrroline-2-methyl-5-carboxylate.','2,4-diaminopentanoate + H(2)O + NAD(P)(+) = 2-amino-4-oxopentanoate +NH(3) + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.13','Glutamate synthase (NADPH)','Glutamate (reduced nicotinamide adenine dinucleotide phosphate) synthase','FAD; FMN; Iron-sulfur.','The reaction takes place in the opposite direction.The protein is composed of two subunits, alpha and beta.The alpha subunit is composed of two domains, one hydrolyzingL-glutamine to NH(3) and L-glutamate (cf. EC 3.5.1.2), the othercombining the produced NH(3) with 2-oxoglutarate to produce a secondmolecule of L-glutamate (cf. EC 1.4.1.4).The beta subunit transfers electrons to the cosubstrate.The NH(3) is channeled through a 31 A channel in the active protein.In the absence of the beta subunit, coupling between the two domainsof the alpha subunit is compromised and some ammonium can beproduced.In the intact alpha-beta complex, ammonia production only takes placeas part of the overall reaction.Formerly EC 2.6.1.53.','2 L-glutamate + NADP(+) = L-glutamine + 2-oxoglutarate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.14','Glutamate synthase (NADH)','L-glutamate synthase','FMN.','NULL','2 L-glutamate + NAD(+) = L-glutamine + 2-oxoglutarate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.15','Lysine dehydrogenase','NULL','NULL','NULL','L-lysine + NAD(+) = 1,2-didehydropiperidine-2-carboxylate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.16','Diaminopimelate dehydrogenase','Meso-diaminopimelate D-dehydrogenase','NULL','NULL','Meso-2,6-diaminoheptanedioate + H(2)O + NADP(+) = L-2-amino-6-oxoheptanedioate + NH(3) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.17','N-methylalanine dehydrogenase','NULL','NULL','NULL','N-methyl-L-alanine + H(2)O + NADP(+) = pyruvate + methylamine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.18','Lysine 6-dehydrogenase','L-lysine 6-dehydrogenase','NULL','Highly specific for L-lysine as substrate, although (S)-(2-aminoethyl)-L-cysteine can act as a substrate, but more slowly.While the enzyme from Agrobacterium tumefaciens can use only NAD(+),that from the thermophilic bacterium Geobacillus stearothermophiluscan also use NADP(+), but more slowly.','L-lysine + NAD(+) = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate + NADH +NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.19','Tryptophan dehydrogenase','L-Trp-dehydrogenase','Calcium.','NULL','L-tryptophan + NAD(P)(+) = (indol-3-yl)pyruvate + NH(3) + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.20','Phenylalanine dehydrogenase','L-phenylalanine dehydrogenase','NULL','The enzymes from Bacillus badius and Sporosarcina ureae are highlyspecific for L-phenylalanine; that from Bacillus sphaericus also actson L-tyrosine.','L-phenylalanine + H(2)O + NAD(+) = phenylpyruvate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.21','Aspartate dehydrogenase','NAD-dependent aspartate dehydrogenase','NULL','Catalyzes the first step in NAD biosynthesis from aspartate.Strictly specific for L-aspartate as substrate.Has a higher affinity for NAD(+) than NADP(+).','L-aspartate + H(2)O + NAD(P)(+) = oxaloacetate + NH(3) + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.23','Valine dehydrogenase (NAD(+))','NULL','NULL','The enzyme from Streptomyces spp. has no activity with NADP(+) (cf.EC 1.4.1.8).','L-valine + H(2)O + NAD(+) = 3-methyl-2-oxobutanoate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.1.24','3-dehydroquinate synthase II','DHQ synthase II','NULL','The enzyme, which was isolated from the archaeon Methanocaldococcusjannaschii, plays a key role in an alternative pathway for thebiosynthesis of 3-dehydroquinate (DHQ), an intermediate of thecanonical pathway for the biosynthesis of aromatic amino acids.The enzyme catalyzes a two-step reaction - an oxidative deamination,followed by cyclization.','2-amino-3,7-dideoxy-D-threo-hept-6-ulosonate + H(2)O + NAD(+) =3-dehydroquinate + NH(3) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.2.1','Glycine dehydrogenase (cytochrome)','Glycine-cytochrome c reductase','NULL','NULL','Glycine + H(2)O + 2 ferricytochrome c = glyoxylate + NH(3) +2 ferrocytochrome c + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.1','D-aspartate oxidase','Aspartic oxidase','FAD.','NULL','D-aspartate + H(2)O + O(2) = oxaloacetate + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.2','L-amino-acid oxidase','Ophio-amino-acid oxidase','FAD.','NULL','An L-amino acid + H(2)O + O(2) = a 2-oxo acid + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.3','D-amino-acid oxidase','NULL','FAD.','Wide specificity for D-amino acids.Also acts on glycine.','A D-amino acid + H(2)O + O(2) = a 2-oxo acid + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.4','Monoamine oxidase','Adrenaline oxidase','FAD.','Catalyzes the oxidative deamination of neurotransmitters and biogenicamines.Acts on primary amines, and also on some secondary and tertiaryamines.It differs from EC 1.4.3.21, as it can oxidize secondary and tertiaryamines but not methylamine.Inhibited by acetylenic compounds such as chlorgyline, 1-deprenyl andpargyline but, unlike EC 1.4.3.21 and EC 1.4.3.22, it is notinhibited by semicarbazide.Formerly EC 1.4.3.9.','RCH(2)NHR + H(2)O + O(2) = RCHO + RNH(2) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.5','Pyridoxal 5-phosphate synthase','PMP oxidase','FMN.','In Escherichia coli, the coenzyme pyridoxal 5-phosphate issynthesized de novo by a pathway that involves EC 1.2.1.72,EC 1.1.1.290, EC 2.6.1.52, EC 1.1.1.262, EC 2.6.99.2 and EC 1.4.3.5.','(1) Pyridoxamine 5-phosphate + H(2)O + O(2) = pyridoxal 5-phosphate +NH(3) + H(2)O(2).(2) Pyridoxine 5-phosphate + O(2) = pyridoxal 5-phosphate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.7','D-glutamate oxidase','D-glutamic oxidase','NULL','NULL','D-glutamate + H(2)O + O(2) = 2-oxoglutarate + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.8','Ethanolamine oxidase','NULL','Cobalt.','NULL','Ethanolamine + H(2)O + O(2) = glycolaldehyde + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.10','Putrescine oxidase','NULL','FAD.','4-aminobutanal condenses non-enzymically to 1-pyrroline.','Putrescine + O(2) + H(2)O = 4-aminobutanal + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.11','L-glutamate oxidase','NULL','FAD.','The enzyme from Azotobacter previously listed under this number,which did not produce H(2)O(2), was a crude cell-free extract whichprobably contained catalase.','L-glutamate + O(2) + H(2)O = 2-oxoglutarate + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.12','Cyclohexylamine oxidase','NULL','FAD.','Some other cyclic amines can act instead of cyclohexylamine, but notsimple aliphatic and aromatic amides.','Cyclohexylamine + O(2) + H(2)O = cyclohexanone + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.13','Protein-lysine 6-oxidase','Lysyl oxidase','NULL','Also acts on protein 5-hydroxylysine.Catalyzes the final known enzymic step required for collagen andelastin cross-linking in the biosynthesis of normal matureextracellular matrices.These reactions play an important role for the development,elasticity and extensibility of connective tissue.Also active on free amines, such as cadaverine or benzylamine.','Peptidyl-L-lysyl-peptide + O(2) + H(2)O = peptidyl-allysyl-peptide +NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.14','L-lysine oxidase','NULL','FAD.','Also acts, more slowly, on L-ornithine, L-phenylalanine, L-arginineand L-histidine.','L-lysine + O(2) + H(2)O = 6-amino-2-oxohexanoate + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.15','D-glutamate(D-aspartate) oxidase','NULL','FAD.','D-glutamate and D-aspartate are oxidized at the same rate.Other D-monoaminodicarboxylates, and other D- and L-amino acids,are not oxidized.Cf. EC 1.4.3.7 and EC 1.4.3.1.','(1) D-glutamate + H(2)O + O(2) = 2-oxoglutarate + NH(3) + H(2)O(2).(2) D-aspartate + H(2)O + O(2) = oxaloacetate + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.16','L-aspartate oxidase','LASPO','FAD.','L-aspartate oxidase catalyzes the first step in the de novobiosynthesis of NAD(+) in some bacteria.O(2) can be replaced by fumarate as electron acceptor, yieldingsuccinate.The ability of the enzyme to use both O(2) and fumarate in cofactorreoxidation enables it to function under both aerobic and anaerobicconditions.Iminosuccinate can either be hydrolyzed to form oxaloacetate andNH(3) or can be used by EC 2.5.1.72 in the production of quinolinate.','L-aspartate + O(2) = iminosuccinate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.17','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.18','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.19','Glycine oxidase','NULL','FAD.','The enzyme from Bacillus subtilis is active with glycine, sarcosine,N-ethylglycine, D-alanine, D-alpha-aminobutyrate, D-proline,D-pipecolate and N-methyl-D-alanine.It differs from EC 1.4.3.3 due to its activity on sarcosine andD-pipecolate.The intermediate 2-iminoacetate is used directly by EC 2.8.1.10.','Glycine + H(2)O + O(2) = glyoxylate + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.20','L-lysine 6-oxidase','L-lysine-epsilon-oxidase','NULL','Differs from EC 1.4.3.13 by using free L-lysine rather than theprotein-bound form.N(2)-acetyl-L-lysine is also a substrate, but N(6)-acetyl-L-lysine,which has an acetyl group at position 6, is not.Also acts on L-ornithine, D-lysine and 4-hydroxy-L-lysine, but moreslowly.The amines cadaverine and putrescine are not substrates.','L-lysine + O(2) + H(2)O = (S)-2-amino-6-oxohexanoate + H(2)O(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.21','Primary-amine oxidase','Amine oxidase','NULL','A group of enzymes that oxidize primary monoamines but have little orno activity toward diamines, such as histamine, or toward secondaryand tertiary amines.They are copper quinoproteins (2,4,5-trihydroxyphenylalanine quinone)and, unlike EC 1.4.3.4, are sensitive to inhibition by carbonyl-groupreagents, such as semicarbazide.In some mammalian tissues the enzyme also functions as a vascular-adhesion protein (VAP-1).Formerly EC 1.4.3.6.','RCH(2)NH(2) + H(2)O + O(2) = RCHO + NH(3) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.22','Diamine oxidase','Amine oxidase','NULL','A group of enzymes that oxidize diamines, such as histamine, and alsosome primary monoamines but have little or no activity towardsecondary and tertiary amines.They are copper quinoproteins (2,4,5-trihydroxyphenylalanine quinone)and, like EC 1.4.3.21 but unlike EC 1.4.3.4, they are sensitive toinhibition by carbonyl-group reagents, such as semicarbazide.Formerly EC 1.4.3.6.','Histamine + H(2)O + O(2) = (imidazol-4-yl)acetaldehyde + NH(3) +H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.23','7-chloro-L-tryptophan oxidase','NULL','NULL','This enzyme catalyzes a step in the biosynthesis of rebeccamycin,an indolocarbazole alkaloid produced by the actinobacteriumLechevalieria aerocolonigenes.During catalysis, the bound FAD is reoxidized at the expense ofmolecular oxygen, producing one molecule of hydrogen peroxide.The enzyme shows significant preference for 7-chloro-L-tryptophanover L-tryptophan.','7-chloro-L-tryptophan + O(2) = 2-imino-3-(7-chloroindol-3-yl)propanoate +H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.3.24','Pseudooxynicotine oxidase','NULL','FAD.','This enzyme, characterized from the soil bacterium Pseudomonas sp.HZN6, is involved the nicotine degradation.','4-(methylamino)-1-(pyridin-3-yl)butan-1-one + H(2)O + O(2) = 4-oxo-4-(pyridin-3-yl)butanal + methylamine + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.4.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.4.2','Glycine dehydrogenase (aminomethyl-transferring)','Glycine cleavage system P-protein','Pyridoxal 5-phosphate.','A component of the glycine cleavage system, which is composed of fourcomponents that only loosely associate: the P protein (EC 1.4.4.2),the T protein (EC 2.1.2.10), the L protein (EC 1.8.1.4) and thelipoyl-bearing H protein.Previously known as glycine synthase.','Glycine + [glycine-cleavage complex H protein]-N(6)-lipoyl-L-lysine =[glycine-cleavage complex H protein]-S-aminomethyl-N(6)-dihydrolipoyl-L-lysine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.5.1','D-amino acid dehydrogenase (quinone)','DadA','FAD; Iron-sulfur.','This enzyme may be the same as EC 1.4.99.1.','A D-amino acid + H(2)O + a quinone = a 2-oxo acid + NH(3) + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.7.1','Glutamate synthase (ferredoxin)','Ferredoxin-dependent glutamate synthase','FAD; FMN; Iron-sulfur.','The protein is composed of two domains, one hydrolyzing L-glutamineto NH(3) and L-glutamate (cf. EC 3.5.1.2), the other combining theproduced NH(3) with 2-oxoglutarate to produce a second molecule ofL-glutamate.The NH(3) is channeled through a 24 A channel in the active protein.No hydrolysis of glutamine takes place without ferredoxin and2-oxoglutarate being bound to the protein.','2 L-glutamate + 2 oxidized ferredoxin = L-glutamine + 2-oxoglutarate +2 reduced ferredoxin + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.9.1','Methylamine dehydrogenase (amicyanin)','Amine dehydrogenase','Tryptophan tryptophylquinone.','The enzyme oxidizes aliphatic monoamines and diamines, histamine andethanolamine, but not secondary and tertiary amines, quaternaryammonium salts or aromatic amines.Formerly EC 1.4.98.1 and EC 1.4.99.3.','Methylamine + H(2)O + amicyanin = formaldehyde + ammonia + reducedamicyanin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.9.2','Aralkylamine dehydrogenase (azurin)','Aromatic amine dehydrogenase','NULL','Phenazine methosulfate can act as acceptor.Acts on aromatic amines and, more slowly, on some long-chainaliphatic amines, but not on methylamine or ethylamine.Formerly EC 1.4.99.4.','ArCH(2)NH(2) + H(2)O + 2 azurin = ArCHO + NH(3) + 2 reduced azurin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.98.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.99.1','D-amino-acid dehydrogenase','NULL','FAD.','Acts to some extent on all D-amino acids except D-aspartate andD-glutamate.','A D-amino acid + H(2)O + acceptor = a 2-oxo acid + NH(3) + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.99.2','Taurine dehydrogenase','NULL','NULL','NULL','Taurine + H(2)O + acceptor = sulfoacetaldehyde + NH(3) + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.99.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.99.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.4.99.5','Glycine dehydrogenase (cyanide-forming)','HCN synthase','FAD.','The enzyme is membrane-bound, and the 2-electron acceptor is acomponent of the respiratory chain.The enzyme can act with various artificial electron acceptors,including phenazine methosulfate.','Glycine + 2 A = HCN + CO(2) + 2 AH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.1','Pyrroline-2-carboxylate reductase','NULL','NULL','Reduces 1-pyrroline-2-carboxylate to L-proline and also 1,2-didehydropiperidine-2-carboxylate to L-pipecolate.','L-proline + NAD(P)(+) = 1-pyrroline-2-carboxylate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.2','Pyrroline-5-carboxylate reductase','P5CR','NULL','Also reduces 1-pyrroline-3-hydroxy-5-carboxylate to L-hydroxyproline.','L-proline + NAD(P)(+) = 1-pyrroline-5-carboxylate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.3','Dihydrofolate reductase','Tetrahydrofolate dehydrogenase','NULL','The enzyme from animals and some micro-organisms also slowly reducesfolate to 5,6,7,8-tetrahydrofolate.Formerly EC 1.5.1.4.','5,6,7,8-tetrahydrofolate + NADP(+) = 7,8-dihydrofolate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.5','Methylenetetrahydrofolate dehydrogenase (NADP(+))','NULL','NULL','NULL','5,10-methylenetetrahydrofolate + NADP(+) = 5,10-methenyltetrahydrofolate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.6','Formyltetrahydrofolate dehydrogenase','10-formyltetrahydrofolate dehydrogenase','NULL','NULL','10-formyltetrahydrofolate + NADP(+) + H(2)O = tetrahydrofolate + CO(2) +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.7','Saccharopine dehydrogenase (NAD(+), L-lysine-forming)','Lysine-2-oxoglutarate reductase','NULL','NULL','N(6)-(L-1,3-dicarboxypropyl)-L-lysine + NAD(+) + H(2)O = L-lysine +2-oxoglutarate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.8','Saccharopine dehydrogenase (NADP(+), L-lysine-forming)','Lysine-2-oxoglutarate reductase','NULL','NULL','N(6)-(L-1,3-dicarboxypropyl)-L-lysine + NADP(+) + H(2)O = L-lysine +2-oxoglutarate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.9','Saccharopine dehydrogenase (NAD(+), L-glutamate-forming)','Aminoadipic semialdehyde synthase','NULL','NULL','N(6)-(L-1,3-dicarboxypropyl)-L-lysine + NAD(+) + H(2)O = L-glutamate +(S)-2-amino-6-oxohexanoate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.10','Saccharopine dehydrogenase (NADP(+), L-glutamate-forming)','Aminoadipate semialdehyde-glutamate reductase','NULL','NULL','N(6)-(L-1,3-dicarboxypropyl)-L-lysine + NADP(+) + H(2)O = L-glutamate +(S)-2-amino-6-oxohexanoate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.11','D-octopine dehydrogenase','D-octopine synthase','NULL','Also acts, in the reverse direction, on L-ornithine, L-lysine andL-histidine.','N(2)-(D-1-carboxyethyl)-L-arginine + NAD(+) + H(2)O = L-arginine +pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.12','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.13','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.14','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.15','Methylenetetrahydrofolate dehydrogenase (NAD(+))','NULL','NULL','NULL','5,10-methylenetetrahydrofolate + NAD(+) = 5,10-methenyltetrahydrofolate +NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.16','D-lysopine dehydrogenase','D-lysopine synthase','NULL','In the reverse reaction, a number of L-amino acids can act instead ofL-lysine, and 2-oxobutanoate and, to a lesser extent, glyoxylate canact instead of pyruvate.','N(2)-(D-1-carboxyethyl)-L-lysine + NADP(+) + H(2)O = L-lysine + pyruvate+ NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.17','Alanopine dehydrogenase','NULL','NULL','In the reverse reaction, L-alanine can be replaced by L-cysteine,L-serine, or L-threonine; glycine acts very slowly (cf. EC 1.5.1.22).','2,2-iminodipropanoate + NAD(+) + H(2)O = L-alanine + pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.18','Ephedrine dehydrogenase','NULL','NULL','The product immediately hydrolyzes to methylamine and 1-hydroxy-1-phenylpropan-2-one.Acts on a number of related compounds including (-)-sympatol, (+)-pseudoephedrine and (+)-norephedrine.','(-)-ephedrine + NAD(+) = (R)-2-methylimino-1-phenylpropan-1-ol + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.19','D-nopaline dehydrogenase','D-nopaline synthase','NULL','In the reverse direction, forms D-nopaline from L-arginine andD-ornaline from L-ornithine.','N(2)-(D-1,3-dicarboxypropyl)-L-arginine + NADP(+) + H(2)O = L-arginine +2-oxoglutarate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.20','Methylenetetrahydrofolate reductase (NAD(P)H)','5,10-CH(2)-H(4)folate reductase','FAD.','Menadione can also serve as an electron acceptor.Formerly EC 1.1.1.68, EC 1.1.1.171, EC 1.1.99.15 and EC 1.7.99.5.','5-methyltetrahydrofolate + NAD(P)(+) = 5,10-methylenetetrahydrofolate +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.21','Delta(1)-piperideine-2-carboxylate reductase','NULL','NULL','Formerly EC 1.5.1.14.','L-pipecolate + NADP(+) = Delta(1)-piperideine-2-carboxylate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.22','Strombine dehydrogenase','NULL','NULL','Catalyzes the reaction of EC 1.5.1.17, more slowly.Does not act on L-strombine.','N-(carboxymethyl)-D-alanine + NAD(+) + H(2)O = glycine + pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.23','Tauropine dehydrogenase','NULL','NULL','In the reverse reaction, alanine can act instead of taurine, moreslowly, and 2-oxobutanoate and 2-oxopentanoate can act instead ofpyruvate.','Tauropine + NAD(+) + H(2)O = taurine + pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.24','N(5)-(carboxyethyl)ornithine synthase','NULL','NULL','In the reverse direction L-lysine can act instead of L-ornithine,more slowly.Acts on the amino group (cf. EC 1.5.1.16).','N(5)-(L-1-carboxyethyl)-L-ornithine + NADP(+) + H(2)O = L-ornithine +pyruvate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.25','Thiomorpholine-carboxylate dehydrogenase','Ketimine reductase','NULL','The product is the cyclic imine of the 2-oxoacid corresponding toS-(2-aminoethyl)cysteine.In the reverse direction, a number of other cyclic unsaturatedcompounds can act as substrates, more slowly.','Thiomorpholine 3-carboxylate + NAD(P)(+) = 3,4-dehydro-thiomorpholine-3-carboxylate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.26','Beta-alanopine dehydrogenase','NULL','NULL','NULL','Beta-alanopine + NAD(+) + H(2)O = beta-alanine + pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.27','1,2-dehydroreticulinium reductase (NADPH)','1,2-dehydroreticulinium ion reductase','NULL','Reduces the 1,2-dehydroreticulinium ion to (R)-reticuline, which is adirect precursor of morphinan alkaloids in the poppy plant.The enzyme does not catalyze the reverse reaction to any significantextent under physiological conditions.','(R)-reticuline + NADP(+) = 1,2-dehydroreticulinium + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.28','Opine dehydrogenase','NULL','NULL','In the forward direction, the enzyme from Arthrobacter sp. acts alsoon secondary amine dicarboxylates such asN-(1-carboxyethyl)methionine and N-(1-carboxyethyl)phenylalanine.Dehydrogenation forms an imine, which dissociates to the amino acidand pyruvate.In the reverse direction, the enzyme acts also on neutral amino acidsas an amino donor.They include L-amino acids such as 2-aminopentanoic acid,2-aminobutyric acid, 2-aminohexanoic acid, 3-chloroalanine,O-acetylserine, methionine, isoleucine, valine, phenylalanine,leucine and alanine.The amino acceptors include 2-oxoacids such as pyruvate,oxaloacetate, glyoxylate and 2-oxobutyrate.','(2S)-2-((1-(R)-carboxyethyl)amino)pentanoate + NAD(+) + H(2)O = L-2-aminopentanoic acid + pyruvate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.29','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.30','Flavin reductase (NADPH)','Flavin mononucleotide reductase','NULL','The enzyme from Entamoeba histolytica reduces riboflavin andgalactoflavin, and, more slowly, FMN and FAD.NADH is oxidized more slowly than NADPH.Formerly EC 1.6.8.2.','Reduced riboflavin + NADP(+) = riboflavin + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.31','Berberine reductase','(R)-canadine synthase','NULL','Involved in alkaloid biosynthesis in Corydalis cava to give (R)-canadine with the opposite configuration to the precursor ofberberine (see EC 1.3.3.8).Also acts on 7,8-dihydroberberine.','(R)-canadine + 2 NADP(+) = berberine + 2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.32','Vomilenine reductase','NULL','NULL','Forms part of the ajmaline biosynthesis pathway.','1,2-dihydrovomilenine + NADP(+) = vomilenine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.33','Pteridine reductase','Pteridine reductase 1','NULL','The enzyme from Leishmania (both amastigote and promastigote forms)catalyzes the NADPH-dependent reduction of folate and a wide varietyof unconjugated pterins, including biopterin, to their tetrahydroforms.It also catalyzes the reduction of 7,8-dihydropterins and 7,8-dihydrofolate to their tetrahydro forms.In contrast to EC 1.5.1.3 and EC 1.5.1.34, pteridine reductase willnot catalyze the reduction of the quinonoid form of dihydrobiopterin.The enzyme is specific for NADPH; no activity has been detected withNADH.It also differs from EC 1.5.1.3 in being specific for the B side ofNADPH.Formerly EC 1.1.1.253.','5,6,7,8-tetrahydrobiopterin + 2 NADP(+) = biopterin + 2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.34','6,7-dihydropteridine reductase','6,7-dihydropteridine:NAD(P)H oxidoreductase','NULL','The substrate is the quinonoid form of dihydropteridine.Not identical with EC 1.5.1.3.Formerly EC 1.6.99.7 and EC 1.6.99.10.','A 5,6,7,8-tetrahydropteridine + NAD(P)(+) = a 6,7-dihydropteridine +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.35','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.36','Flavin reductase (NADH)','NADH-dependent flavin reductase','NULL','The enzyme from Escherichia coli W catalyzes the reduction of freeflavins by NADH.The enzyme has similar affinity to FAD, FMN and riboflavin.Activity with NADPH is more than 2 orders of magnitude lower thanactivity with NADH.','Reduced flavin + NAD(+) = flavin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.37','FAD reductase (NADH)','NADH-dependent FAD reductase','NULL','The enzyme from Burkholderia phenoliruptrix can reduce either FAD orflavin mononucleotide (FMN) but prefers FAD.Unlike EC 1.5.1.36, the enzyme can not reduce riboflavin.The enzyme does not use NADPH as acceptor.','FADH(2) + NAD(+) = FAD + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.38','FMN reductase (NADPH)','Flavin reductase P','FMN.','The enzymes from bioluminescent bacteria contain FMN, while theenzyme from Escherichia coli does not.The enzyme often forms a two-component system with monooxygenasessuch as luciferase.Unlike EC 1.5.1.39, this enzyme does not use NADH as acceptor.While FMN is the preferred substrate, the enzyme can also use FAD andriboflavin with lower activity.Formerly EC 1.5.1.29 and EC 1.6.8.1.','FMNH(2) + NADP(+) = FMN + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.39','FMN reductase (NAD(P)H)','NULL','FMN.','The enzyme can utilize NADH and NADPH with similar reaction rates.Different from EC 1.5.1.42 and EC 1.5.1.38.The luminescent bacterium Vibrio harveyi possesses all three enzymes,while the bacterium Aliivibrio fischeri contains only this non-specific type.Also reduces riboflavin and FAD, but more slowly.Formerly EC 1.5.1.29 and EC 1.6.8.1.','FMNH(2) + NAD(P)(+) = FMN + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.40','8-hydroxy-5-deazaflavin:NADPH oxidoreductase','NULL','NULL','The enzyme has an absolute requirement for both the 5-deazaflavinstructure and the presence of an 8-hydroxy group in the substrate.','Reduced coenzyme F420 + NADP(+) = oxidized coenzyme F420 + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.41','Riboflavin reductase (NAD(P)H)','Flavin mononucleotide reductase','NULL','Catalyzes the reduction of soluble flavins by reduced pyridinenucleotides.Highest activity with riboflavin.When NADH is used as acceptor, the enzyme can also utilize FMN andFAD as substrates, with lower activity than riboflavin.When NADPH is used as acceptor, the enzyme has a very low activitywith FMN and no activity with FAD.Formerly EC 1.5.1.29 and EC 1.6.8.1.','Reduced riboflavin + NAD(P)(+) = riboflavin + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.42','FMN reductase (NADH)','NADH-dependent FMN reductase','NULL','The enzyme often forms a two-component system with monooxygenases.Unlike EC 1.5.1.38 and EC 1.5.1.39, this enzyme has a strongpreference for NADH over NADPH, although some activity with thelatter is observed.While FMN is the preferred substrate, FAD can also be used with muchlower activity.','FMNH(2) + NAD(+) = FMN + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.43','Carboxynorspermidine synthase','Carboxynorspermidine dehydrogenase','NULL','The reaction takes place in the opposite direction.Part of a bacterial polyamine biosynthesis pathway.L-aspartate 4-semialdehyde and propane-1,3-diamine/putrescine form aSchiff base that is reduced to formcarboxynorspermidine/carboxyspermidine, respectively.The enzyme from the bacterium Vibrio cholerae is essential forbiofilm formation.The enzyme from Campylobacter jejuni only produces carboxyspermidinein vivo even though it also can produce carboxynorspermidine invitro.','(1) Carboxynorspermidine + H(2)O + NADP(+) = L-aspartate 4-semialdehyde +propane-1,3-diamine + NADPH.(2) Carboxyspermidine + H(2)O + NADP(+) = L-aspartate 4-semialdehyde +putrescine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.44','Festuclavine dehydrogenase','Festuclavine synthase','NULL','The enzyme participates in the biosynthesis of fumigaclavine C,an ergot alkaloid produced by some fungi of the Trichocomaceaefamily.The reaction proceeds in vivo in the opposite direction to the oneshown here.','Festuclavine + NAD(+) = 6,8-dimethyl-6,7-didehydroergoline + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.45','FAD reductase (NAD(P)H)','NULL','NULL','This enzyme, isolated from the bacterium Geobacillusthermodenitrificans, participates in the pathway of tryptophandegradation.The enzyme is part of a system that also includes a bifunctionalriboflavin kinase/FMN adenylyltransferase and EC 1.14.14.8.It can reduce either FAD or flavin mononucleotide (FMN) but prefersFAD.The enzyme has a slight preference for NADPH as acceptor.Cf. EC 1.5.1.37.','FADH(2) + NAD(P)(+) = FAD + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.46','Agroclavine dehydrogenase','NULL','NULL','The enzyme participates in the biosynthesis of ergotamine, an ergotalkaloid produced by some fungi of the Clavicipitaceae family.The reaction is catalyzed in the opposite direction to that shown.The substrate for the enzyme is an iminium intermediate that isformed spontaneously from chanoclavine-I aldehyde in the presence ofglutathione.','Agroclavine + NADP(+) = 6,8-dimethyl-6,7,8,9-tetradehydroergoline +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.1.47','Dihydromethanopterin reductase','H(2)MPT reductase','NULL','The enzyme, characterized from the bacterium Methylobacteriumextorquens, is involved in tetrahydromethanopterin biosynthesis.The specific activity with NADH is 15% of that with NADPH at the sameconcentration.It does not reduce 7,8-dihydrofolate (cf. EC 1.5.1.3).','5,6,7,8-tetrahydromethanopterin + NAD(P)(+) = 7,8-dihydromethanopterin +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.1','Sarcosine oxidase','NULL','FAD.','The flavin is both covalently and non-covalently bound in a molarratio of 1:1.','Sarcosine + H(2)O + O(2) = glycine + formaldehyde + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.2','N-methyl-L-amino-acid oxidase','NULL','Flavoprotein.','NULL','An N-methyl-L-amino acid + H(2)O + O(2) = an L-amino acid + formaldehyde+ H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.3','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.4','N(6)-methyl-lysine oxidase','Epsilon-alkyllysinase','NULL','NULL','N(6)-methyl-L-lysine + H(2)O + O(2) = L-lysine + formaldehyde + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.5','(S)-6-hydroxynicotine oxidase','NULL','FAD.','NULL','(S)-6-hydroxynicotine + H(2)O + O(2) = 1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.6','(R)-6-hydroxynicotine oxidase','NULL','FAD.','NULL','(R)-6-hydroxynicotine + H(2)O + O(2) = 1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.7','L-pipecolate oxidase','NULL','NULL','The product reacts with water to form (S)-2-amino-6-oxohexanoate.','L-pipecolate + O(2) = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate +H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.10','Dimethylglycine oxidase','NULL','FAD.','Does not oxidize sarcosine.','N,N-dimethylglycine + H(2)O + O(2) = sarcosine + formaldehyde + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.11','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.12','Dihydrobenzophenanthridine oxidase','NULL','Copper.','Found in higher plants.Produces oxidized forms of the benzophenanthridine alkaloids.','(1) Dihydrosanguinarine + O(2) = sanguinarine + H(2)O(2).(2) Dihydrochelirubine + O(2) = chelirubine + H(2)O(2).(3) Dihydromacarpine + O(2) = macarpine + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.13','N(1)-acetylpolyamine oxidase','PAO','FAD.','The enzyme also catalyzes the reaction: N(1),N(12)-diacetylspermine +O(2) + H(2)O = N(1)-acetylspermidine + 3-acetamamidopropanal +H(2)O(2).No or very weak activity with spermine, or spermidine in absence ofaldehydes.In presence of aldehydes the enzyme catalyzes the reactions: 1.spermine + O(2) + H(2)O = spermidine + 3-aminopropanal + H(2)O(2),and with weak efficiency 2.spermidine + O(2) + H(2)O = putrescine + 3-aminopropanal + H(2)O(2).Found in mammalian peroxisomes and oxidizes N(1)-acetylatedpolyamines at the exo (three-carbon) side of the secondary amine,forming 3-acetamamidopropanal.Since the products of the reactions are deacetylated polyamines, thisprocess is known as polyamine back-conversion.Differs in specificity from EC 1.5.3.14, EC 1.5.3.15, EC 1.5.3.16 andEC 1.5.3.17.Formerly EC 1.5.3.11, EC 1.5.3.n3, EC 1.5.3.n4 and EC 1.5.3.n10.','(1) N(1)-acetylspermidine + O(2) + H(2)O = putrescine +3-acetamidopropanal + H(2)O(2).(2) N(1)-acetylspermine + O(2) + H(2)O = spermidine + 3-acetamidopropanal+ H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.14','Polyamine oxidase (propane-1,3-diamine-forming)','Maize PAO','FAD.','As the products of the reaction cannot be converted directly to otherpolyamines, this class of polyamine oxidases is considered to beinvolved in the terminal catabolism of polyamines.Catalyzes less efficiently the oxidation of N(1)-acetylspermine andspermine.Differs in specificity from EC 1.5.3.13, EC 1.5.3.15, EC 1.5.3.16 andEC 1.5.3.17.Formerly EC 1.5.3.11, EC 1.5.3.n6, EC 1.5.3.n7, EC 1.5.3.n8 andEC 1.5.3.n9.','Spermidine + O(2) + H(2)O = propane-1,3-diamine + 4-aminobutanal +H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.15','N(8)-acetylspermidine oxidase (propane-1,3-diamine-forming)','NULL','FAD.','Also active with N(1)-acetylspermine, weak activity with N(1),N(12)-diacetylspermine.No activity with diaminopropane, putrescine, cadaverine,diaminohexane, norspermidine, spermine and spermidine.Absence of monoamine oxidase (EC 1.4.3.4) activity.Differs in specificity from EC 1.5.3.13, EC 1.5.3.14, EC 1.5.3.16 andEC 1.5.3.17.Formerly EC 1.5.3.11, EC 1.5.3.n6, EC 1.5.3.n7, EC 1.5.3.n8 andEC 1.5.3.n9.','N(8)-acetylspermidine + O(2) + H(2)O = propane-1,3-diamine +4-acetamidobutanal + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.16','Spermine oxidase','SMO','FAD.','The enzyme from Arabidopsis thaliana (AtPAO1) oxidizes norspermine tonorspermidine with high efficiency.The mammalian enzyme, encoded by the SMOX gene, is a cytosolic enzymethat catalyzes the oxidation of spermine at the exo (three-carbon)side of the tertiary amine.No activity with spermidine.Weak activity with N(1)-acetylspermine.Differs in specificity from EC 1.5.3.13, EC 1.5.3.14, EC 1.5.3.15 andEC 1.5.3.17.Formerly EC 1.5.3.11, 1.5.3.n1, EC 1.5.3.n2, EC 1.5.3.n3 andEC 1.5.3.n5.','Spermine + O(2) + H(2)O = spermidine + 3-aminopropanal + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.17','Non-specific polyamine oxidase','Polyamine oxidase','FAD.','The non-specific polyamine oxidases may differ from each otherconsiderably.The enzyme from Saccharomyces cerevisiae shows a rather broadspecificity and also oxidizes N(8)-acetylspermidine.The enzyme from Ascaris suum shows high activity with spermine andspermidine, but also oxidizes norspermine.The enzyme from Arabidopsis thaliana shows high activity withspermidine, but also oxidizes other polyamines.The specific polyamine oxidases are classified as EC 1.5.3.13,EC 1.5.3.14, EC 1.5.3.15 and EC 1.5.3.16.Formerly EC 1.5.3.11, EC 1.5.3.n1, EC 1.5.3.n2 and EC 1.5.3.n4.','(1) Spermine + O(2) + H(2)O = spermidine + 3-aminopropanal + H(2)O(2).(2) Spermidine + O(2) + H(2)O = putrescine + 3-aminopropanal + H(2)O(2).(3) N(1)-acetylspermine + O(2) + H(2)O = spermidine + 3-acetamidopropanal+ H(2)O(2).(4) N(1)-acetylspermidine + O(2) + H(2)O = putrescine +3-acetamidopropanal + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.18','L-saccharopine oxidase','FAP2','FAD.','The enzyme is involved in pipecolic acid biosynthesis.','N(6)-(L-1,3-dicarboxypropyl)-L-lysine + H(2)O + O(2) = (S)-2-amino-6-oxohexanoate + L-glutamate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.19','4-methylaminobutanoate oxidase (formaldehyde-forming)','NULL','FAD.','In the enzyme from the soil bacterium Arthrobacter nicotinovorans thecofactor is covalently bound.Participates in the nicotine degradation pathway of this organism.','4-methylaminobutanoate + O(2) + H(2)O = 4-aminobutanoate + formaldehyde +H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.20','N-alkylglycine oxidase','N-carboxymethylalkylamine:oxygen oxidoreductase (decarboxymethylating)','NULL','Isolated from the mold Cladosporium sp. G-10.Acts on N(6)-(carboxymethyl)lysine, 6-((carboxymethy)amino)hexanoicacid, sarcosine and N-ethylglycine.It has negligible action on glycine (cf. EC 1.4.3.19).','N-alkylglycine + H(2)O + O(2) = alkylamine + glyoxalate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.21','4-methylaminobutanoate oxidase (methylamine-forming)','NULL','NULL','The enzyme participates in the nicotine degradation pathway of thesoil bacterium Arthrobacter nicotinovorans.Has a very weak EC 1.4.3.4 activity with 4-aminobutanoate.','4-methylaminobutanoate + O(2) + H(2)O = succinate semialdehyde +methylamine + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.22','Coenzyme F420H(2) oxidase','NULL','FMN; Iron.','The enzyme from the archaeon Methanothermobacter marburgensis isSi-face specific with respect to C(5) of coenzyme F420.','2 reduced coenzyme F420 + O(2) = 2 oxidized coenzyme F420 + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.3.n10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.4.1','Pyrimidodiazepine synthase','NULL','NULL','In the reverse direction of reaction, the reduction of 6-pyruvoyl-tetrahydropterin is accompanied by the opening of the 6-memberedpyrazine ring and the formation of the 7-membered diazepine ring.The pyrimidodiazepine involved is an acetyldihydro derivative.Involved in the formation of the eye pigment drosopterin inDrosophila melanogaster.','A pyrimidodiazepine + glutathione disulfide + H(2)O = 6-pyruvoyl-tetrahydropterin + 2 glutathione.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.5.1','Electron-transferring-flavoprotein dehydrogenase','Electron transfer flavoprotein-ubiquinone oxidoreductase','FAD; Iron-sulfur.','Forms part of the mitochondrial electron-transfer system.','Reduced electron-transferring flavoprotein + ubiquinone = electron-transferring flavoprotein + ubiquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.5.2','Proline dehydrogenase','NULL','FAD.','The electrons from L-proline are transferred to the FAD cofactor,and from there to a quinone acceptor.In many organisms, ranging from bacteria to mammals, proline isoxidized to glutamate in a two-step process involving this enzyme andEC 1.2.1.88.Both activities are carried out by the same enzyme in enterobacteria.Formerly EC 1.5.99.8.','L-proline + a quinone = (S)-1-pyrroline-5-carboxylate + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.7.1','Methylenetetrahydrofolate reductase (ferredoxin)','5,10-methylenetetrahydrofolate reductase','Flavoprotein; Iron-sulfur; Zinc.','The enzyme from Clostridium formicoaceticum catalyzes the reductionof methylene blue, menadione, benzyl viologen, rubredoxin or FAD with5-methyltetrahydrofolate and the oxidation of reduced ferredoxin orFADH(2) with 5,10-methylenetetrahydrofolate.However, unlike EC 1.5.1.20, there is no activity with NAD(P)H.','5-methyltetrahydrofolate + 2 oxidized ferredoxin = 5,10-methylenetetrahydrofolate + 2 reduced ferredoxin + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.7.2','Coenzyme F420 oxidoreductase (ferredoxin)','NULL','FAD; Iron-sulfur.','NULL','Reduced coenzyme F420 + 2 oxidized ferredoxin = oxidized coenzyme F420 +2 reduced ferredoxin + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.8.1','Dimethylamine dehydrogenase','DMADh','FAD; Iron-sulfur.','Formerly EC 1.5.99.10.','Dimethylamine + H(2)O + electron-transferring flavoprotein = methylamine+ formaldehyde + reduced electron-transferring flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.8.2','Trimethylamine dehydrogenase','TMADh','FMN; Iron-sulfur.','A number of alkyl-substituted derivatives of trimethylamine can alsoact as electron donors.Phenazine methosulfate and 2,6-dichloroindophenol can act as electronacceptors.Formerly EC 1.5.99.7.','Trimethylamine + H(2)O + electron-transferring flavoprotein =dimethylamine + formaldehyde + reduced electron-transferringflavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.8.3','Sarcosine dehydrogenase','NULL','FMN.','Tetrahydrofolate is also a substrate, being converted to N(5),N(10)-methylenetetrahydrofolate.Formerly EC 1.5.99.1.','Sarcosine + H(2)O + electron-transfer flavoprotein = glycine +formaldehyde + reduced electron-transfer flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.8.4','Dimethylglycine dehydrogenase','Me2GlyDH','FAD.','Formerly EC 1.5.99.2.','N,N-dimethylglycine + electron-transfer flavoprotein + H(2)O = sarcosine+ formaldehyde + reduced electron-transfer flavoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.98.1','Methylenetetrahydromethanopterin dehydrogenase','5,10-methylenetetrahydromethanopterin dehydrogenase','NULL','Coenzyme F420 is a 7,8-didemethyl-8-hydroxy-5-deazariboflavinderivative; methanopterin is a pterin analog.The enzyme is involved in the formation of methane from CO(2) inMethanobacterium thermoautotrophicum.Formerly EC 1.5.99.9.','5,10-methylenetetrahydromethanopterin + oxidized coenzyme F420 = 5,10-methenyltetrahydromethanopterin + reduced coenzyme F420.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.98.2','5,10-methylenetetrahydromethanopterin reductase','5,10-methylenetetrahydromethanopterin cyclohydrolase','NULL','Catalyzes an intermediate step in methanogenesis from CO(2) and H(2)in archaea.Formerly EC 1.5.99.11.','5-methyltetrahydromethanopterin + oxidized coenzyme F420 = 5,10-methylenetetrahydromethanopterin + reduced coenzyme F420.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.3','L-pipecolate dehydrogenase','NULL','NULL','The product reacts with water to form (S)-2-amino-6-oxohexanoate.','L-pipecolate + acceptor = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate +reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.4','Nicotine dehydrogenase','NULL','FMN.','Acts on both (R)- and (S)-isomers.','Nicotine + acceptor + H(2)O = (S)-6-hydroxynicotine + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.5','Methylglutamate dehydrogenase','NULL','NULL','A number of N-methyl-substituted amino acids can act as donors.2,6-dichloroindophenol is the best acceptor.','N-methyl-L-glutamate + acceptor + H(2)O = L-glutamate + formaldehyde +reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.6','Spermidine dehydrogenase','NULL','FAD; Heme.','Ferricyanide, 2,6-dichloroindophenol and cytochrome c can act asacceptor.4-aminobutanal condenses non-enzymically to 1-pyrroline.','Spermidine + acceptor + H(2)O = propane-1,3-diamine + 4-aminobutanal +reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.11','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.12','Cytokinin dehydrogenase','Cytokinin oxidase','FAD.','Converts zeatin and related cytokinins.Catalyzes the oxidation of cytokinins, a family of N(6)-substitutedadenine derivatives that are plant hormones, where the substituent isa dimethylallyl or other prenyl group.Although this activity was previously thought to be catalyzed by ahydrogen-peroxide-forming oxidase, this enzyme does not requireoxygen for activity and does not form hydrogen peroxide.2,6-dichloroindophenol, methylene blue, nitroblue tetrazolium,phenazine methosulfate and Cu(2+) in the presence of imidazole canact as acceptors.','N(6)-dimethylallyladenine + acceptor + H(2)O = adenine + 3-methylbut-2-enal + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.13','D-proline dehydrogenase','D-pro dehydrogenase','FAD.','The enzyme prefers D-proline and acts on other D-amino acid withlower efficiency.','D-proline + acceptor = 1-pyrroline-2-carboxylate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.5.99.14','6-hydroxypseudooxynicotine dehydrogenase','NULL','Molybdopterin cytosine dinucleotide.','Contains a cytidylyl molybdenum cofactor.The enzyme, which participates in the nicotine degradation pathway,has been characterized from the soil bacterium Arthrobacternicotinovorans.','1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one + acceptor + H(2)O =1-(2,6-dihydroxypyridin-3-yl)-4-(methylamino)butan-1-one + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.1.1','NAD(P)(+) transhydrogenase (Si-specific)','NAD(P)(+) transhydrogenase (B-specific)','FAD.','Si-specific with respect to both NAD(+) and NADP(+).Also acts with deamino coenzymes.','NADPH + NAD(+) = NADP(+) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.1.2','NAD(P)(+) transhydrogenase (Re/Si-specific)','NAD(P)(+) transhydrogenase (AB-specific)','NULL','The enzyme from heart mitochondria is Re-specific with respect toNAD(+) and Si-specific with respect to NADP(+) (cf. EC 1.6.1.1.).','NADPH + NAD(+) = NADP(+) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.1.3','NAD(P)(+) transhydrogenase','NAD(+) transhydrogenase','NULL','The enzyme catalyzes the NADPH-driven reduction of NAD(+).This entry stands for enzymes whose stereo-specificity with respectto NADPH is not known (cf. EC 1.6.1.1 and EC 1.6.1.2).','NADPH + NAD(+) = NADP(+) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.2.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.2.2','Cytochrome-b5 reductase','NULL','FAD.','NULL','NADH + 2 ferricytochrome b5 = NAD(+) + H(+) + 2 ferrocytochrome b5.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.2.3','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.2.4','NADPH--hemoprotein reductase','Aldehyde reductase (NADPH-dependent)','FAD; FMN.','The number n in the equation is 1 if the hemoprotein undergoes a2-electron reduction, and is 2 if it undergoes a 1-electronreduction.The enzyme catalyzes the reduction of the heme-thiolate-dependentmonooxygenases, such as EC 1.14.14.1, and reduction of EC 1.14.99.3.It is part of the microsomal hydroxylating system.It also reduces cytochrome b5 and cytochrome c.','NADPH + n oxidized hemoprotein = NADP(+) + n reduced hemoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.2.5','NADPH--cytochrome-c2 reductase','NULL','FAD.','NULL','NADPH + 2 ferricytochrome c2 = NADP(+) + H(+) + 2 ferrocytochrome c2.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.2.6','Leghemoglobin reductase','NULL','NULL','NULL','NAD(P)H + 2 ferrileghemoglobin = NAD(P)(+) + 2 ferroleghemoglobin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.3.1','NAD(P)H oxidase (H(2)O(2)-forming)','Dual oxidase','Calcium; FAD; Heme.','When calcium is present, this transmembrane glycoprotein generatesH(2)O(2) by transfering electrons from intracellular NAD(P)H toextracellular molecular oxygen.The electron bridge within the enzyme contains one molecule of FADand probably two heme groups.This flavoprotein is expressed at the apical membrane of thyrocytes,and provides H(2)O(2) for the thyroid peroxidase-catalyzedbiosynthesis of thyroid hormones.','NAD(P)H + O(2) = NAD(P)(+) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.3.2','NAD(P)H oxidase (H(2)O-forming)','NULL','FAD.','NADPH is a better substrate than NADH.By removal of oxygen the enzyme is involved in aerobic tolerance inthe thermophilic anaerobic archaeon Thermococcus profundus and inGiardia intestinalis, a microaerophilic single-celled parasite of theorder Diplomonadida.','2 NAD(P)H + O(2) = 2 NAD(P)(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.3.3','NADH oxidase (H(2)O(2)-forming)','H(2)O(2)-forming NADH oxidase','FAD.','The bacterium Streptococcus mutans contains two distinct NADHoxidases, a H(2)O(2)-forming enzyme and a H(2)O-forming enzyme (cf.EC 1.6.3.4).The enzymes from the anaerobic archaea Methanocaldococcus jannaschiiand Pyrococcus furiosus also produce low amounts of H(2)O.Unlike EC 1.6.3.1 it has no activity toward NADPH.','NADH + O(2) = NAD(+) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.3.4','NADH oxidase (H(2)O-forming)','H(2)O-forming NADH oxidase','FAD.','The bacterium Streptococcus mutans contains two distinct NADHoxidases, a H(2)O-forming enzyme and a H(2)O(2)-forming enzyme (cf.EC 1.6.3.3).','2 NADH + O(2) = 2 NAD(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.3.5','Renalase','Alpha-NAD(P)H oxidase/anomerase','FAD.','Renalase is a flavoprotein hormone secreted into the blood by thekidney that is reported to lower blood pressure and slow heart rate.','Alpha-NAD(P)H + O(2) = beta-NAD(P)(+) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.4.10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.1','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.2','NAD(P)H dehydrogenase (quinone)','Azoreductase','FAD.','The enzyme catalyzes a two-electron reduction and has a preferencefor short-chain acceptor quinones, such as ubiquinone, benzoquinone,juglone and duroquinone.The animal, but not the plant, form of the enzyme is inhibited bydicoumarol.Formerly EC 1.6.99.2.','NAD(P)H + a quinone = NAD(P)(+) + a hydroquinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.3','NADH:ubiquinone reductase (H(+)-translocating)','Coenzyme Q reductase','FMN; Iron-sulfur.','The complex is present in mitochondria and aerobic bacteria.Breakdown of the complex can release EC 1.6.99.3.In photosynthetic bacteria, reversed electron transport through thisenzyme can reduce NAD(+) to NADH.','NADH + ubiquinone + 5 H(+)(In) = NAD(+) + ubiquinol + 4 H(+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.4','Monodehydroascorbate reductase (NADH)','NULL','NULL','NULL','NADH + 2 monodehydroascorbate = NAD(+) + 2 ascorbate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.5','NADPH:quinone reductase','Quinone oxidoreductase','Zinc.','Specific for NADPH.Catalyzes the one-electron reduction of certain quinones, with theorthoquinones 1,2-naphthoquinone and 9,10-phenanthrenequinone beingthe best substrates.Dicoumarol (cf. EC 1.6.5.2) and nitrofurantoin are competitiveinhibitors with respect to the quinone substrate.The semiquinone free-radical product may be non-enzymically reducedto the hydroquinone or oxidized back to quinone in the presence ofO(2).Abundant in the lens of the eye of some mammalian species.','NADPH + 2 quinone = NADP(+) + 2 semiquinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.6','p-benzoquinone reductase (NADPH)','NULL','NULL','Involved in the 4-nitrophenol degradation pathway in bacteria.','NADPH + p-benzoquinone = NADP(+) + hydroquinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.7','2-hydroxy-1,4-benzoquinone reductase','Hydroxybenzoquinone reductase','FMN.','The enzyme differs in substrate speificity from other quinonereductases.The enzyme in Burkholderia cepacia is inducible by 2,4,5-trichlorophenoxyacetate.','2-hydroxy-1,4-benzoquinone + NADH = 1,2,4-trihydroxybenzene + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.8','NADH:ubiquinone reductase (Na(+)-transporting)','Na(+)-NQR','FAD; FMN; Iron-sulfur; Riboflavin.','NULL','NADH + ubiquinone + n Na(+)(In) = NAD(+) + ubiquinol + n Na(+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.9','NADH:ubiquinone reductase (non-electrogenic)','Coenzyme Q reductase','FAD.','Occurs in mitochondria of yeast and plants, and in aerobic bacteria.Has low activity with NADPH.','NADH + ubiquinone = NAD(+) + ubiquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.5.10','NADPH dehydrogenase (quinone)','NULL','Flavoprotein.','The enzyme from Escherichia coli is specific for NADPH and is mostactive with quinone derivatives and ferricyanide as electronacceptors.Menaquinone can act as acceptor.The enzyme from hog liver is inhibited by dicoumarol and folic acidderivatives but not by 2,4-dinitrophenol.Formerly EC 1.6.99.6.','NADPH + a quinone = NADP(+) + a quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.9','Trimethylamine-N-oxide reductase','Trimethylamine oxidase','NULL','NULL','NADH + trimethylamine N-oxide = NAD(+) + trimethylamine + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.11','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.12','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.6.13','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.7.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.7.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.8.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.8.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.1','NADPH dehydrogenase','NADPH diaphorase','FMN or FAD.','FMN in yeasts; FAD in plants.','NADPH + acceptor = NADP(+) + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.3','NADH dehydrogenase','Beta-NADH dehydrogenase dinucleotide','Flavoprotein; Iron-sulfur.','After preparations have been subjected to certain treatmentscytochrome c may act as acceptor.Under normal conditions, two protons are extruded from the cytoplasmor the intramitochondrial or stromal compartment.Present in a mitochondrial complex as EC 1.6.5.3.Formerly EC 1.6.2.1.','NADH + acceptor = NAD(+) + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.5','NADH dehydrogenase (quinone)','NULL','NULL','Menaquinone or other quinones can act as acceptor.Inhibited by AMP and 2,4-dinitrophenol but not by dicoumarol or folicacid derivatives.','NADH + acceptor = NAD(+) + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.11','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.12','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.13','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.6.99.n1','NADPH dehydrogenase (coenzyme F420 dependent)','F420-dependent NADP reductase','NULL','NULL','NADPH + coenzyme F420 = NADP(+) + reduced coenzyme F420.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.1','Nitrate reductase (NADH)','Assimilatory NADH:nitrate reductase','FAD or FMN; Heme; Molybdenum.','Formerly EC 1.6.6.1.','Nitrite + NAD(+) + H(2)O = nitrate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.2','Nitrate reductase (NAD(P)H)','Assimilatory NAD(P)H-nitrate reductase','FAD or FMN; Heme; Molybdenum.','Formerly EC 1.6.6.2.','Nitrite + NAD(P)(+) + H(2)O = nitrate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.3','Nitrate reductase (NADPH)','Assimilatory NADPH-nitrate reductase','FAD; Heme; Iron-sulfur; Molybdenum.','Formerly EC 1.6.6.3.','Nitrite + NADP(+) + H(2)O = nitrate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.4','Nitrite reductase (NAD(P)H)','Assimilatory nitrite reductase','FAD; Iron; Iron-sulfur; Siroheme.','The enzymes from the fungi Neurospora crassa, Emericella nidulans andCandida nitratophila and the bacterium Aliivibrio fischeri can useeither NADPH or NADH as electron donor.Cf. EC 1.7.1.15.Formerly EC 1.6.6.4.','Ammonia + 3 NAD(P)(+) + 2 H(2)O = nitrite + 3 NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.5','Hyponitrite reductase','NADH:hyponitrite oxidoreductase','Metal ions.','Formerly EC 1.6.6.6.','2 hydroxylamine + 2 NAD(+) = hyponitrous acid + 2 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.6','Azobenzene reductase','Azo reductase','NULL','The reaction occurs in the reverse direction to that shown above.Other azo dyes, such as Methyl Red, Rocceline, Solar Orange andSumifix Black B can also be reduced.Formerly EC 1.6.6.7.','N,N-dimethyl-1,4-phenylenediamine + aniline + 2 NADP(+) =4-(dimethylamino)azobenzene + 2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.7','GMP reductase','Guanosine 5-monophosphate oxidoreductase','NULL','Formerly EC 1.6.6.8.','Inosine 5-phosphate + NH(3) + NADP(+) = guanosine 5-phosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.8','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.9','Nitroquinoline-N-oxide reductase','4-nitroquinoline 1-oxide reductase','NULL','Formerly EC 1.6.6.10.','4-(hydroxyamino)quinoline N-oxide + 2 NAD(P)(+) + H(2)O =4-nitroquinoline N-oxide + 2 NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.10','Hydroxylamine reductase (NADH)','Ammonium dehydrogenase','NULL','Also acts on some hydroxamates.Formerly EC 1.6.6.11.','NH(3) + NAD(+) + H(2)O = hydroxylamine + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.11','4-(dimethylamino)phenylazoxybenzene reductase','Dimethylaminoazobenzene N-oxide reductase','NULL','Formerly EC 1.6.6.12.','4-(dimethylamino)phenylazobenzene + NADP(+) + H(2)O =4-(dimethylamino)phenylazoxybenzene + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.12','N-hydroxy-2-acetamidofluorene reductase','N-hydroxy-2-acetylaminofluorene reductase','NULL','Also acts, more slowly, on N-hydroxy-4-acetamidobiphenyl.Formerly EC 1.6.6.13.','2-acetamidofluorene + NAD(P)(+) + H(2)O = N-hydroxy-2-acetamidofluorene +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.13','PreQ(1) synthase','7-cyano-7-deazaguanine reductase','NULL','The reaction occurs in the reverse direction.This enzyme catalyzes one of the early steps in the synthesis ofqueosine (Q-tRNA), and is followed by the action of EC 2.4.2.29.Queosine is found in the wobble position of tRNA(GUN) in eukaryotesand bacteria and is thought to be involved in translationalmodulation.','7-aminomethyl-7-carbaguanine + 2 NADP(+) = 7-cyano-7-carbaguanine +2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.14','Nitric oxide reductase (NAD(P)(+), nitrous oxide-forming)','Cytochrome P450nor','Heme-thiolate.','The enzyme from Fusarium oxysporum utilizes only NADH, but theisozyme from Trichosporon cutaneum utilizes both NADH and NADPH.The electron transfer from NAD(P)H to heme occurs directly,not requiring flavin or other redox cofactors.','Nitrous oxide + NAD(P)(+) + H(2)O = 2 nitric oxide + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.1.15','Nitrite reductase (NADH)','Assimilatory nitrite reductase','FAD; Iron-sulfur; Siroheme.','This prokaryotic enzyme is specific for NADH.In addition to catalyzing the 6-electron reduction of nitrite toammonia, the enzyme from Escherichia coli can also catalyze the2-electron reduction of hydroxylamine to ammonia.cf. EC 1.7.1.4.','Ammonia + 3 NAD(+) + 2 H(2)O = nitrite + 3 NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.2.1','Nitrite reductase (NO-forming)','cd-cytochrome nitrite reductase','Copper or iron; FAD.','The reaction is catalyzed by two types of enzymes, found in theperimplasm of denitrifying bacteria.One type comprises proteins containing multiple copper centers,the other a heme protein, cytochrome cd1.Acceptors include c-type cytochromes such as cytochrome c-550 orcytochrome c-551 from Paracoccus denitrificans or Pseudomonasaeruginosa, and small blue copper proteins such as azurin andpseudoazurin.Cytochrome cd1 also has oxidase and hydroxylamine reductaseactivities.May also catalyze the reaction of EC 1.7.99.1 since this is a well-known activity of cytochrome cd1.Formerly EC 1.6.6.5, EC 1.7.99.3 and EC 1.9.3.2.','Nitric oxide + H(2)O + ferricytochrome c = nitrite + ferrocytochrome c +2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.2.2','Nitrite reductase (cytochrome; ammonia-forming)','Cytochrome c nitrite reductase','Calcium; Heme.','The enzyme also reduces nitric oxide and hydroxylamine to ammonia,and sulfite to sulfide.','NH(3) + 2 H(2)O + 6 ferricytochrome c = nitrite + 6 ferrocytochrome c +7 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.2.3','Trimethylamine-N-oxide reductase (cytochrome c)','TMAO reductase','Bis(molybdopterin guanine dinucleotide)molybdenum cofactor.','The cytochrome c involved in photosynthetic bacteria is a pentahemeprotein.The reductant is a membrane-bound multiheme cytochrome c.Also reduces dimethyl sulfoxide to dimethyl sulfide.','Trimethylamine + 2 (ferricytochrome c)-subunit + H(2)O = trimethylamineN-oxide + 2 (ferrocytochrome c)-subunit + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.2.4','Nitrous-oxide reductase','NULL','Copper.','The reaction is observed only in the direction of nitrous oxidereduction.Contains the mixed-valent dinuclear CuA species at the electron entrysite of the enzyme, and the tetranuclear Cu-Z center in the activesite.In Paracoccus pantotrophus, the electron donor is cytochrome c552.Formerly EC 1.7.99.6.','Nitrogen + H(2)O + 2 ferricytochrome c = nitrous oxide +2 ferrocytochrome c + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.2.5','Nitric-oxide reductase (cytochrome c)','NULL','Calcium; Heme; Iron.','The enzyme from Pseudomonas aeruginosa contains a dinuclear centercomprising a non-heme iron center and heme b3, plus heme c, heme band calcium; the acceptor is cytochrome c551.Formerly EC 1.7.99.7.','Nitrous oxide + 2 ferricytochrome c + H(2)O = 2 nitric oxide +2 ferrocytochrome c + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.2.6','Hydroxylamine dehydrogenase','Hydroxylamine oxidase','Heme.','The enzymes from the nitrifying bacterium Nitrosomonas europaea andthe methylotrophic bacterium Methylococcus capsulatus arehemoproteins with seven c-type hemes and one specialized P460-typeheme per subunit.The enzyme converts hydroxylamine to nitrite via an enzyme-boundnitroxyl intermediate.While nitrite is the main product, the enzyme from Nitrosomonaseuropaea can produce nitric oxide as well.Formerly EC 1.7.3.4.','(1) Hydroxylamine + H(2)O + 2 ferricytochrome c = nitrite +2 ferrocytochrome c + 5 H(+).(2) Hydroxylamine + ferricytochrome c = nitric oxide + ferrocytochrome c+ 3 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.3.1','Nitroalkane oxidase','NAO','FAD.','While nitroethane may be the physiological substrate, the enzyme alsoacts on several other nitroalkanes, including 1-nitropropane,2-nitropropane, 1-nitrobutane, 1-nitropentane, 1-nitrohexane,nitrocyclohexane and some nitroalkanols.Differs from EC 1.13.12.16 in that the preferred substrates areneutral nitroalkanes rather than anionic nitronates.','A nitroalkane + H(2)O + O(2) = an aldehyde or ketone + nitrite +H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.3.2','Acetylindoxyl oxidase','NULL','NULL','NULL','N-acetylindoxyl + O(2) = N-acetylisatin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.3.3','Factor independent urate hydroxylase','Urate oxidase','NULL','Was previously thought to be a copper protein, but it is now knownthat the enzymes from soybean (Glycine max), the mold Aspergillusflavus and Bacillus subtilis contains no copper nor any othertransition-metal ion.The 5-hydroxyisourate formed decomposes spontaneously to formallantoin and CO(2), although there is an enzyme-catalyzed pathway inwhich EC 3.5.2.17 catalyzes the first step.The enzyme is different from EC 1.14.13.113.','Urate + O(2) + H(2)O = 5-hydroxyisourate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.3.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.3.5','3-aci-nitropropanoate oxidase','NULL','FMN.','The primary products of the enzymatic reaction are probably thenitropropanoate free radical and superoxide.Also acts, more slowly, on 4-aci-nitrobutanoate.','3-aci-nitropropanoate + O(2) = 3-oxopropanoate + nitrite + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.3.6','Hydroxylamine oxidase (cytochrome)','HAO','Iron.','The enzyme from the heterotrophic nitrifying bacterium Paracoccusdenitrificans contains three to five non-heme, non-iron-sulfur ironatoms and interacts with cytochrome C(556) and pseudoazurin.Formerly EC 1.7.3.4.','Hydroxylamine + O(2) + ferricytochrome ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.5.1','Nitrate reductase (quinone)','Dissimilatory nitrate reductase','Heme; Iron-sulfur; Molybdopterin guanine dinucleotide.','A membrane-bound enzyme which supports anaerobic respiration onnitrate under anaerobic conditions and in the presence of nitrateEscherichia coli expresses two forms NarA and NarZ, both beingcomprised of three subunits.','Nitrate + a quinol = nitrite + a quinone + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.5.2','Nitric oxide reductase (menaquinol)','NULL','Copper.','NULL','2 nitric oxide + menaquinol = nitrous oxide + menaquinone + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.6.1','Nitrite dismutase','Nitrophorin 7','Ferriheme b.','The enzyme is one of the nitrophorins from the salivary gland of theblood-feeding insect Rhodnius prolixus.Nitric oxide produced induces vasodilation after injection.Nitrophorins 2 and 4 can also catalyze this reaction.','3 nitrite + 2 H(+) = 2 nitric oxide + nitrate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.7.1','Ferredoxin--nitrite reductase','NULL','Iron-sulfur; Siroheme.','NULL','NH(3) + 2 H(2)O + 6 oxidized ferredoxin = nitrite + 6 reduced ferredoxin+ 7 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.7.2','Ferredoxin--nitrate reductase','Assimilatory nitrate reductase','Iron-sulfur; Molybdenum.','NULL','Nitrite + H(2)O + 2 oxidized ferredoxin = nitrate + 2 reduced ferredoxin+ 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.99.1','Hydroxylamine reductase','Hydroxylamine (acceptor) reductase','Flavoprotein.','Reduced pyocyanine, methylene blue and flavins act as donors for thereduction of hydroxylamine.May be identical to EC 1.7.2.1.','NH(3) + H(2)O + acceptor = hydroxylamine + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.99.2','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.99.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.99.4','Nitrate reductase','Respiratory nitrate reductase','NULL','The Pseudomonas enzyme is a cytochrome, but the enzyme fromMicrococcus halodenitrificans is an iron protein containingmolybdenum.Reduced benzyl viologen and other dyes bring about the reduction ofnitrate.','Nitrite + acceptor = nitrate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.99.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.99.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.99.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.7.99.8','Hydrazine oxidoreductase','NULL','NULL','The enzyme is involved in the pathway of anaerobic ammonium oxidationin anammox bacteria.The conversion of hydroxylamine by the enzyme from Brocadiaanammoxidans results in the formation of NO and N(2)O.Hydroxylamine is not a substrate for the enzyme from planctomyceteKSU-1.','Hydrazine + acceptor = N(2) + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.1','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.2','Sulfite reductase (NADPH)','NULL','FAD; FMN; Heme.','NULL','H(2)S + 3 NADP(+) + 3 H(2)O = sulfite + 3 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.3','Hypotaurine dehydrogenase','NULL','Heme; Molybdenum.','NULL','Hypotaurine + H(2)O + NAD(+) = taurine + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.4','Dihydrolipoyl dehydrogenase','Dehydrolipoate dehydrogenase','FAD.','A component of the multienzyme 2-oxo-acid dehydrogenase complexes.In the pyruvate dehydrogenase complex, it binds to the core ofEC 2.3.1.12 and catalyzes oxidation of its dihydrolipoyl groups.It plays a similar role in the oxoglutarate and 3-methyl-2-oxobutanoate dehydrogenase complexes.Another substrate is the dihydrolipoyl group in the H-protein of theglycine-cleavage system, in which it acts, together with EC 1.4.4.2and EC 2.1.2.10 to break down glycine.It can also use free dihydrolipoate, dihydrolipoamide ordihydrolipoyllysine as substrate.Was first shown to catalyze the oxidation of NADH by methylene blue;this activity was called diaphorase.The glycine cleavage system is composed of four components that onlyloosely associate: the P protein (EC 1.4.4.2), the T protein(EC 2.1.2.10), the L protein (EC 1.8.1.4) and the lipoyl-bearing Hprotein.Formerly EC 1.6.4.3.','Protein N(6)-(dihydrolipoyl)lysine + NAD(+) = protein N(6)-(lipoyl)lysine+ NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.5','2-oxopropyl-CoM reductase (carboxylating)','2-KPCC','NULL','Also acts on thioethers longer in chain length on the oxo side, e.g.2-oxobutyl-CoM, but this portion must be attached to CoM(2-mercaptoethanesulfonate); no CoM analogs will substitute.Forms component II of a four-component enzyme system (comprisingEC 4.4.1.23 (component I), EC 1.8.1.5 (component II), EC 1.1.1.268(component III) and EC 1.1.1.269 (component IV)) that is involved inepoxyalkane carboxylation in Xanthobacter sp. strain Py2.','2-mercaptoethanesulfonate + acetoacetate + NADP(+) =2-(2-oxopropylthio)ethanesulfonate + CO(2) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.6','Cystine reductase','Cystine reductase (NADH)','NULL','Formerly EC 1.6.4.1.','2 L-cysteine + NAD(+) = L-cystine + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.7','Glutathione-disulfide reductase','Glutathione reductase','FAD.','Activity is dependent on a redox-active disulfide in each of theactive centers.Formerly EC 1.6.4.2.','2 glutathione + NADP(+) = glutathione disulfide + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.8','Protein-disulfide reductase','Disulfide reductase','NULL','Formerly EC 1.6.4.4.','Protein dithiol + NAD(P)(+) = protein disulfide + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.9','Thioredoxin-disulfide reductase','NADP--thioredoxin reductase','NULL','May be identical to EC 1.8.1.10.Formerly EC 1.6.4.5.','Thioredoxin + NADP(+) = thioredoxin disulfide + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.10','CoA-glutathione reductase','CoA-glutathione reductase (NADPH)','Flavoprotein.','The substrate is a mixed disulfide.May be identical to EC 1.8.1.9.Formerly EC 1.6.4.6.','CoA + glutathione + NADP(+) = CoA-glutathione + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.11','Asparagusate reductase','Asparagusate dehydrogenase','NULL','Also acts on lipoate.Formerly EC 1.6.4.7.','3-mercapto-2-mercaptomethylpropanoate + NAD(+) = asparagusate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.12','Trypanothione-disulfide reductase','N(1),N(8)-bis(glutathionyl)spermidine reductase','FAD.','Trypanothione disulfide is the oxidized form of N(1),N(8)-bis(glutathionyl)-spermidine from the insect-parasitic trypanosomatidCrithidia fasciculata.The activity is dependent on a redox-active cystine at the activecenter (cf. EC 1.8.1.7).Formerly EC 1.6.4.8.','Trypanothione + NADP(+) = trypanothione disulfide + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.13','Bis-gamma-glutamylcystine reductase','Bis-gamma-glutamylcystine reductase (NADPH)','FAD.','The enzyme, which is found only in halobacteria, maintains theconcentration of gamma-glutamylcysteine, the major low molecularweight thiol in halobacteria.Not identical with EC 1.8.1.7 or EC 1.8.1.14.Formerly EC 1.6.4.9.','2 gamma-glutamylcysteine + NADP(+) = bis-gamma-glutamylcystine + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.14','CoA-disulfide reductase','CoA-disulfide reductase (NAD(P)H)','FAD.','Not identical with EC 1.8.1.6, EC 1.8.1.7 or EC 1.8.1.13.The enzyme from the bacterium Staphylococcus aureus has a strongpreference for NADPH, while the bacterium Bacillus megateriumcontains both NADH and NADPH-dependent enzymes.Formerly EC 1.6.4.10.','2 CoA + NADP(+) = CoA-disulfide + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.15','Mycothione reductase','Mycothiol-disulfide reductase','FAD.','No activity with glutathione, trypanothione or coenzyme A assubstrate.','2 mycothiol + NAD(P)(+) = mycothione + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.16','Glutathione amide reductase','GAR','NULL','The enzyme restores glutathione amide disulfide, which is producedduring the reduction of peroxide by EC 1.11.1.17 back to glutathioneamide (it catalyzes the reaction in the opposite direction to thatshown).The enzyme belongs to the family of flavoprotein disulfideoxidoreductases, but unlike other members of the family, which arespecific for NADPH, it prefers NADH.','2 glutathione amide + NAD(+) = glutathione amide disulfide + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.17','Dimethylsulfone reductase','NULL','Molybdopterin.','NULL','Dimethyl sulfoxide + NAD(+) + H(2)O = dimethyl sulfone + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.18','NAD(P)H sulfur oxidoreductase (CoA-dependent)','Coenzyme A-dependent NADPH sulfur oxidoreductase','FAD.','This enzyme, characterized from the archaeon Pyrococcus furiosus,is responsible for NAD(P)H-linked sulfur reduction.The activity with NADH is about half of that with NADPH.The reaction is dependent on CoA, although the nature of thisdependency is not well understood.','Hydrogen sulfide + NAD(P)(+) = sulfur + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.1.19','Sulfide dehydrogenase','NULL','Flavoprotein; Iron-sulfur.','In the archaeon Pyrococcus furiosus the enzyme is involved in theoxidation of NADPH which is produced in peptide degradation.The enzyme also catalyzes the reduction of sulfur with loweractivity.','Hydrogen sulfide + (sulfide)(n) + NADP(+) = (sulfide)(n+1) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.2.1','Sulfite dehydrogenase','NULL','NULL','Associated with cytochrome c-551.','Sulfite + 2 ferricytochrome c + H(2)O = sulfate + 2 ferrocytochrome c +2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.2.2','Thiosulfate dehydrogenase','Tetrathionate synthase','NULL','NULL','2 thiosulfate + 2 ferricytochrome c = tetrathionate +2 ferrocytochrome c.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.2.3','Sulfide-cytochrome-c reductase (flavocytochrome c)','NULL','FAD; Heme c.','NULL','Hydrogen sulfide + 2 ferricytochrome c = sulfur + 2 ferrocytochrome c.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.2.4','Dimethyl sulfide:cytochrome c2 reductase','NULL','Heme b; Iron-sulfur; Molybdopterin guanine dinucleotide.','The enzyme from Rhodovulum sulfidophilum binds molybdopterin guaninedinucleotide, heme b and [4Fe-4S] clusters.','Dimethyl sulfide + 2 ferricytochrome c2 + H(2)O = dimethyl sulfoxide +2 ferrocytochrome c2 + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.3.1','Sulfite oxidase','NULL','Heme; Molybdenum.','NULL','Sulfite + O(2) + H(2)O = sulfate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.3.2','Thiol oxidase','Sulfhydryl oxidase','NULL','R may be =S or =O, or a variety of other groups.The enzyme is not specific for R.','2 RC(R)SH + O(2) = RC(R)S-S(R)CR + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.3.3','Glutathione oxidase','NULL','FAD.','Also acts, more slowly, on L-cysteine and several other thiols.','2 glutathione + O(2) = glutathione disulfide + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.3.4','Methanethiol oxidase','Methylmercaptan oxidase','NULL','NULL','Methanethiol + O(2) + H(2)O = formaldehyde + H(2)S + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.3.5','Prenylcysteine oxidase','Prenylcysteine lyase','FAD.','Cleaves the thioether bond of S-prenyl-L-cysteines, such asS-farnesylcysteine and S-geranylgeranylcysteine.N-acetyl-prenylcysteine and prenylcysteinyl peptides are notsubstrates.May represent the final step in the degradation of prenylatedproteins in mammalian tissues.Originally thought to be a simple lyase, EC 4.4.1.18.','An S-prenyl-L-cysteine + O(2) + H(2)O = a prenal + L-cysteine + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.3.6','Farnesylcysteine lyase','FC lyase','FAD.','In contrast to mammalian EC 1.8.3.5 the farnesylcysteine lyase fromArabidopsis is specific for S-farnesyl-L-cysteine and shows noactivity with S-geranylgeranyl-L-cysteine.','S-(2E,6E)-farnesyl-L-cysteine + O(2) + H(2)O = (2E,6E)-farnesal +L-cysteine + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.1','Glutathione--homocystine transhydrogenase','NULL','NULL','The reactions catalyzed by this enzyme and by others in this subclassmay be similar to those catalyzed by EC 2.5.1.18.','2 glutathione + homocystine = glutathione disulfide + 2 homocysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.2','Protein-disulfide reductase (glutathione)','Glutathione--insulin transhydrogenase','NULL','Reduces insulin and some other proteins.','2 glutathione + protein-disulfide = glutathione disulfide + protein-dithiol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.3','Glutathione--CoA-glutathione transhydrogenase','NULL','NULL','NULL','CoA + glutathione disulfide = CoA-glutathione + glutathione.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.4','Glutathione--cystine transhydrogenase','NULL','NULL','NULL','2 glutathione + cystine = oxidized glutathione + 2 cysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.7','Enzyme-thiol transhydrogenase (glutathione-disulfide)','[Xanthine-dehydrogenase]:oxidized-glutathione S-oxidoreductase','NULL','Converts EC 1.17.1.4 into EC 1.17.3.2 in the presence of glutathionedisulfide.Also reduces the disulfide bond of ricin.Not inhibited by Cu(2+) or thiol reagents.','[Xanthine dehydrogenase] + glutathione disulfide = [xanthine oxidase] +2 glutathione.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.8','Phosphoadenylyl-sulfate reductase (thioredoxin)','3-phosphoadenylylsulfate reductase','NULL','Specific for PAPS.The enzyme from Escherichia coli will use thioredoxins from otherspecies.Formerly EC 1.8.99.4.','Adenosine 3,5-bisphosphate + sulfite + thioredoxin disulfide =3-phosphoadenylyl sulfate + thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.9','Adenylyl-sulfate reductase (glutathione)','5-adenylylsulfate reductase','NULL','Differs from EC 1.8.99.2, in using glutathione as the reductant.Glutathione can be replaced by gamma-glutamylcysteine ordithiothreitol, but not by thioredoxin, glutaredoxin ormercaptoethanol.The enzyme from Arabidopsis thaliana contains a glutaredoxin-likedomain.Also found in other photosynthetic eukaryotes, e.g., the Madagascarperiwinkle, Catharanthus roseus and the hollow green seaweed,Enteromorpha intestinalis.','AMP + sulfite + glutathione disulfide = adenylyl sulfate + 2 glutathione.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.10','Adenylyl-sulfate reductase (thioredoxin)','Thioredoxin-dependent 5-adenylylsulfate reductase','NULL','Uses adenylyl sulfate, not phosphoadenylyl sulfate, distinguishingthis enzyme from EC 1.8.4.8.Uses thioredoxin as electron donor, not glutathione or other donors,distinguishing it from EC 1.8.4.9 and EC 1.8.99.2.','AMP + sulfite + thioredoxin disulfide = 5-adenylyl sulfate +thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.11','Peptide-methionine (S)-S-oxide reductase','Methionine S-oxide reductase','NULL','The reaction occurs in the reverse direction to that shown above.Exhibits high specificity for the reduction of the S-form ofL-methionine S-oxide, acting faster on the residue in a peptide thanon the free amino acid.On the free amino acid, it can also reduce D-methionine (S)-S-oxidebut more slowly.Plays a role in preventing oxidative-stress damage caused by reactiveoxygen species by reducing the oxidized form of methionine back tomethionine and thereby reactivating peptides that had been damaged.The reaction proceeds via a sulfenic-acid intermediate.Formerly EC 1.8.4.6.','(1) Peptide-L-methionine + thioredoxin disulfide + H(2)O = peptide-L-methionine (S)-S-oxide + thioredoxin.(2) L-methionine + thioredoxin disulfide + H(2)O = L-methionine (S)-S-oxide + thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.12','Peptide-methionine (R)-S-oxide reductase','Methionine S-oxide reductase','Selenium; Zinc.','The reaction occurs in the reverse direction to that shown above.Exhibits high specificity for reduction of the R-form of methionineS-oxide, with higher activity being observed with L-methionineS-oxide than with D-methionine S-oxide.While both free and protein-bound methionine (R)-S-oxide act assubstrates, the activity with the peptide-bound form is far greater.Plays a role in preventing oxidative-stress damage caused by reactiveoxygen species by reducing the oxidized form of methionine back tomethionine and thereby reactivating peptides that had been damaged.The reaction proceeds via a sulfenic-acid intermediate.For MsrB2 and MsrB3, thioredoxin is a poor reducing agent butthionein works well.','Peptide-L-methionine + thioredoxin disulfide + H(2)O = peptide-L-methionine (R)-S-oxide + thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.13','L-methionine (S)-S-oxide reductase','Acetylmethionine sulfoxide reductase','NADPH.','The reaction occurs in the opposite direction to that given above.Dithiothreitol can replace reduced thioredoxin.L-methionine (R)-S-oxide is not a substrate (see EC 1.8.4.14).Formerly EC 1.8.4.5.','L-methionine + thioredoxin disulfide + H(2)O = L-methionine (S)-S-oxide +thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.4.14','L-methionine (R)-S-oxide reductase','Free met-R-(o) reductase','NADPH.','Unlike EC 1.8.4.12 this enzyme cannot use peptide-bound methionine(R)-S-oxide as a substrate.Differs from EC 1.8.4.13, in that L-methionine (S)-S-oxide is not asubstrate.Formerly EC 1.8.4.5.','L-methionine + thioredoxin disulfide + H(2)O = L-methionine (R)-S-oxide +thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.5.1','Glutathione dehydrogenase (ascorbate)','NULL','NULL','NULL','2 glutathione + dehydroascorbate = glutathione disulfide + ascorbate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.5.2','Thiosulfate dehydrogenase (quinone)','Thiosulfate oxidoreductase, tetrathionate-forming','NULL','The reaction can also proceed with ferricyanide as the electronacceptor, but more slowly.Unlike EC 1.8.2.2, this enzyme cannot utilize cytochrome c as anacceptor.','2 thiosulfate + 6-decylubiquinone = tetrathionate + 6-decylubiquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.5.3','Dimethylsulfoxide reductase','DMSO reductase','Iron-sulfur; Molybdopterin.','Also reduces pyridine N-oxide and trimethylamine N-oxide, with loweractivity, to the corresponding amines.','Dimethylsulfide + menaquinone + H(2)O = dimethylsulfoxide + menaquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.5.4','Sulfide:quinone reductase','NULL','FAD.','Ubiquinone, plastoquinone or menaquinone can act as acceptor indifferent species.This enzyme catalyzes the formation of sulfur globules.It is also an important step in anoxygenic bacterial photosynthesis.','n sulfide + n quinone = polysulfide + n quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.6.1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.7.1','Sulfite reductase (ferredoxin)','NULL','Iron.','NULL','Hydrogen sulfide + 6 oxidized ferredoxin + 3 H(2)O = sulfite + 6 reducedferredoxin + 6 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.7.2','Ferredoxin:thioredoxin reductase','NULL','Iron-sulfur.','The enzyme contains a [4Fe-4S] cluster and internal disulfide.It forms a mixed disulfide with thioredoxin on one side, and docksferredoxin on the other side, enabling two one-electron transfers.The reduced thioredoxins generated by the enzyme activate the Calvincycle enzymes EC 3.1.3.11, EC 3.1.3.37 and EC 2.7.1.19 as well asother chloroplast enzymes by disulfide reduction.','2 reduced ferredoxin + thioredoxin disulfide = 2 oxidized ferredoxin +thioredoxin + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.98.1','CoB--CoM heterodisulfide reductase','Heterodisulfide reductase','Heme; Iron-sulfur.','Found in methanogenic archaea, particularly Methanosarcina species,and regenerates coenzyme M and coenzyme B after the action ofEC 2.8.4.1.Highly specific for both coenzyme M and coenzyme B.Reacts with various phenazine derivatives, including2-hydroxyphenazine and 2-bromophenazine.Formerly EC 1.12.99.2.','Coenzyme B + coenzyme M + methanophenazine =N-(7-((2-sulfoethyl)dithio)heptanoyl)-O(3)-phospho-L-threonine +dihydromethanophenazine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.98.2','Sulfiredoxin','Peroxiredoxin-(S-hydroxy-S-oxocysteine) reductase','NULL','In the course of the reaction of EC 1.11.1.15, its cysteine residueis alternately oxidized to the sulfenic acid, S-hydroxycysteine,and reduced back to cysteine.Occasionally the S-hydroxycysteine residue is further oxidized to thesulfinic acid S-hydroxy-S-oxocysteine, thereby inactivating theenzyme.The reductase provides a mechanism for regenerating the active formof peroxiredoxin, i.e. the peroxiredoxin-(S-hydroxycysteine) form.Apparently the reductase first catalyzes the phosphorylation of the-S(O)-OH group by ATP to give -S(O)-O-P, which is attached to theperoxiredoxin by a cysteine residue, forming an -S(O)-S- link betweenthe two enzymes.Attack by a thiol splits this bond, leaving the peroxiredoxin as thesulfenic acid and the reductase as the thiol.','Peroxiredoxin-(S-hydroxy-S-oxocysteine) + ATP + 2 R-SH = peroxiredoxin-(S-hydroxycysteine) + ADP + phosphate + R-S-S-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.99.1','Sulfite reductase','NULL','Iron.','A stoichiometry of six molecules of reduced methyl viologen permolecule of sulfide formed was found.','H(2)S + acceptor + 3 H(2)O = SO(3)(2-) + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.99.2','Adenylyl-sulfate reductase','NULL','FAD; Iron.','Methyl viologen can act as acceptor.','AMP + sulfite + acceptor = adenylyl sulfate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.99.3','Hydrogensulfite reductase','Bisulfite reductase','Iron-sulfur; Siroheme.','Methyl viologen can act as acceptor.A group of sirohemoproteins containing iron-sulfur centers (P-582).','(O(3)S.S.SO(3))(2-) + acceptor + 2 H(2)O + OH(-) = 3 HSO(3)(-) + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.8.99.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.9.3.1','Cytochrome-c oxidase','Complex IV (mitochondrial electron transport)','Copper.','The reduction of O(2) to water is accompanied by the extrusion offour protons from the intramitochondrial compartment.Several bacteria appear to contain analogous oxidases.','4 ferrocytochrome c + O(2) + 4 H(+) = 4 ferricytochrome c + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.9.3.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.9.6.1','Nitrate reductase (cytochrome)','NULL','NULL','NULL','2 ferrocytochrome + nitrate + 2 H(+) = 2 ferricytochrome + nitrite.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.9.99.1','Iron--cytochrome-c reductase','NULL','Iron.','NULL','Ferrocytochrome c + Fe(3+) = ferricytochrome c + Fe(2+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.1.1','Trans-acenaphthene-1,2-diol dehydrogenase','NULL','NULL','Some preparations also utilize NAD(+).','(+-)-trans-acenaphthene-1,2-diol + 2 NADP(+) = acenaphthenequinone +2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.2.1','L-ascorbate--cytochrome-b5 reductase','NULL','NULL','NULL','L-ascorbate + ferricytochrome b5 = monodehydroascorbate +ferrocytochrome b5 + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.2.2','Quinol--cytochrome-c reductase','Complex III (mitochondrial electron transport)','NULL','The enzyme, often referred to as the cytochrome bc1 complex orcomplex III, is the third complex in the electron transport chain.It is present in the mitochondria of all aerobic eukaryotes and inthe inner membranes of most bacteria.The mammalian enzyme contains cytochromes b-562, b-566 and C(1),and a 2-iron ferredoxin.Depending on the organism and physiological conditions, the enzymeextrudes either two or four protons from the cytoplasmic to the non-cytoplasmic compartment (cf. EC 1.6.99.3).','Quinol + 2 ferricytochrome c = quinone + 2 ferrocytochrome c + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.1','Catechol oxidase','Diphenol oxidase','Copper.','A type 3 copper protein that catalyzes exclusively the oxidation ofcatechols (i.e., o-diphenols) to the corresponding o-quinones.The enzyme also acts on a variety of substituted catechols.It is different from tyrosinase, EC 1.14.18.1, which can catalyzeboth the monooxygenation of monophenols and the oxidation ofcatechols.','2 catechol + O(2) = 2 1,2-benzoquinone + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.2','Laccase','Urishiol oxidase','Copper.','A group of multi-copper proteins of low specificity.Acts on both o- and p-quinols, and often acting also on aminophenolsand phenylenediamine.The semiquinone may react further either enzymically or non-enzymically.','4 benzenediol + O(2) = 4 benzosemiquinone + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.3','L-ascorbate oxidase','Ascorbase','Copper.','NULL','4 L-ascorbate + O(2) = 4 monodehydroascorbate + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.4','o-aminophenol oxidase','Isophenoxazine synthase','Flavoprotein; Manganese or copper.','NULL','4 2-aminophenol + 3 O(2) = 2 2-aminophenoxazin-3-one + 6 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.5','3-hydroxyanthranilate oxidase','3-hydroxyanthranilic acid oxidase','NULL','NULL','3-hydroxyanthranilate + O(2) = 6-imino-5-oxocyclohexa-1,3-dienecarboxylate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.6','Rifamycin-B oxidase','NULL','NULL','Also acts on benzene-1,4-diol and, more slowly, on some otherp-quinols.Not identical with EC 1.10.3.1, EC 1.10.3.2, EC 1.10.3.4 orEC 1.10.3.5.','Rifamycin B + O(2) = rifamycin O + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.9','Photosystem II','NULL','NULL','Contains chlorophyll a, beta-carotene, pheophytin, plastoquinone,a Mn(4)Ca cluster, heme and non-heme iron.Four successive photoreactions, resulting in a storage of fourpositive charges, are required to oxidize two water molecules to oneoxygen molecule.','2 H(2)O + 2 plastoquinone + 4 light = O(2) + 2 plastoquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.10','Ubiquinol oxidase (H(+)-transporting)','Cytochrome bb3 oxidase','Copper; Heme.','Contains a dinuclear center comprising two hemes, or heme and copper.Reduction of O(2) on the periplasmic side of the membrane leads torelease of protons in the periplasm, generating a transmembraneproton gradient.In addition, the bb3 and bo oxidases, but not the bd oxidases,contribute to the proton gradient by actively pumping protons acrossthe membrane.','2 ubiquinol + O(2) + n H(+)(Side 1) = 2 ubiquinone + 2 H(2)O +n H(+)(Side 2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.11','Ubiquinol oxidase (non-electrogenic)','Cyanide-insensitive oxidase','Iron.','The enzyme, described from the mitochondria of plants and some fungiand protists, is an alternative terminal oxidase that is notsensitive to cyanide inhibition and does not generate a proton motiveforce.The function of this oxidase is believed to be dissipating excessreducing power, minimizing oxidative stress, and optimizingphotosynthesis in response to changing conditions.','2 ubiquinol + O(2) = 2 ubiquinone + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.12','Menaquinol oxidase (H(+)-transporting)','Cytochrome aa3-600 oxidase','Copper; Heme.','Cytochrome aa3-600, one of the principal respiratory oxidases fromBacillus subtilis, is a member of the heme-copper superfamily ofoxygen reductases, and is a close homolog of the cytochrome bo3ubiquinol oxidase from Escherichia coli, but uses menaquinol insteadof ubiquinol as a substrate.The enzyme also pumps protons across the membrane bilayer, generatinga proton motive force.','2 menaquinol + O(2) = 2 menaquinone + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.13','Caldariellaquinol oxidase (H(+)-transporting)','Aa(3) quinol oxidase','Copper.','The enzyme from thermophilic archaea is part of the terminal oxidaseand catalyzes the reduction of O(2) to water, accompanied by theextrusion of protons across the cytoplasmic membrane.','2 caldariellaquinol + O(2) + n H(+)(Side 1) = 2 caldariellaquinone +2 H(2)O + n H(+)(Side 2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.3.14','Ubiquinol oxidase (electrogenic, non H(+)-transporting)','Cytochrome bd-I oxidase','NULL','This terminal oxidase enzyme is unable to pump protons but generatesa proton motive force by transmembrane charge separation resultingfrom utilizing protons and electrons originating from opposite sidesof the membrane to generate water.The bioenergetic efficiency (the number of charges driven across themembrane per electron used to reduce oxygen to water) is 1.The bd-I oxidase from the bacterium Escherichia coli is thepredominant respiratory oxygen reductase that functions undermicroaerophilic conditions in that organism.Cf. EC 1.10.3.10.','2 ubiquinol + O(2) + 4 H(+)(Side 1) = 2 ubiquinone + 2 H(2)O +4 H(+)(Side 1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.9.1','Plastoquinol--plastocyanin reductase','Cytochrome b6f complex','NULL','Contains two b-type cytochromes, two c-type cytochromes (cn and f),and a [2Fe-2S] Rieske cluster.Cytochrome c-552 can act as acceptor instead of plastocyanin,but more slowly.In chloroplasts, protons are translocated through the thylakoidmembrane from the lumen to the stroma.Formerly EC 1.10.99.1.','Plastoquinol + 2 oxidized plastocyanin + 2 H(+)(Side 1) = plastoquinone +2 reduced plastocyanin + 2 H(+)(Side 2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.99.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.99.2','Ribosyldihydronicotinamide dehydrogenase (quinone)','N-ribosyldihydronicotinamide dehydrogenase (quinone)','FAD; Zinc.','Unlike EC 1.6.5.2, this quinone reductase cannot use NADH or NADPH;instead it uses N-ribosyl- and N-alkyldihydronicotinamides.Polycyclic aromatic hydrocarbons, such as benz[a]anthracene, and theestrogens 17-beta-estradiol and diethylstilbestrol are potentinhibitors, but dicoumarol is only a very weak inhibitor.This enzyme can catalyze both 2-electron and 4-electron reductions,but one-electron acceptors, such as potassium ferricyanide, cannot bereduced.','1-(beta-D-ribofuranosyl)-1,4-dihydronicotinamide + a quinone = 1-(beta-D-ribofuranosyl)nicotinamide + a hydroquinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.10.99.3','Violaxanthin de-epoxidase','VDE','NULL','Along with EC 1.14.13.90, this enzyme forms part of the xanthophyll(or violaxanthin) cycle for controlling the concentration ofzeaxanthin in chloroplasts.It is activated by a low pH of the thylakoid lumen (produced by highlight intensity).Zeaxanthin induces the dissipation of excitation energy in thechlorophyll of the light-harvesting protein complex of photosystemII.In higher plants the enzyme reacts with all-trans-diepoxides, such asviolaxanthin, and all-trans-monoepoxides, but in the alga Mantoniellasquamata, only the diepoxides are good substrates.','Violaxanthin + 2 ascorbate = antheraxanthin + 2 dehydroascorbate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.1','NADH peroxidase','NULL','FAD.','Ferricyanide, quinones, etc., can replace H(2)O(2).','NADH + H(2)O(2) = NAD(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.2','NADPH peroxidase','NULL','NULL','NULL','NADPH + H(2)O(2) = NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.3','Fatty-acid peroxidase','NULL','NULL','Acts on long-chain fatty acids from dodecanoic to octadecanoic acid.','Palmitate + 2 H(2)O(2) = pentadecanal + CO(2) + 3 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.5','Cytochrome-c peroxidase','NULL','Heme.','NULL','2 ferrocytochrome c + H(2)O(2) = 2 ferricytochrome c + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.6','Catalase','NULL','Heme; Manganese.','A manganese protein containing Mn(III) in the resting state, whichalso belongs here, is often called pseudocatalase.The enzymes from some organisms, such as Penicillium simplicissimum,can also act as a peroxidase (EC 1.11.1.7) for which several organicsubstances, especially ethanol, can act as a hydrogen donor.Enzymes that exhibit both catalase and peroxidase activity belongunder EC 1.11.1.21.','2 H(2)O(2) = O(2) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.7','Peroxidase','Lactoperoxidase','Heme.','NULL','2 phenolic donor + H(2)O(2) = 2 phenoxyl radical of the donor + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.8','Iodide peroxidase','Iodinase','Heme.','Thyroid peroxidase catalyzes the biosynthesis of the thyroid hormonesL-thyroxine and triiodo-L-thyronine.It catalyzes both the iodination of tyrosine residues inthyroglobulin (forming mono- and di-iodinated forms) and theircoupling to form either L-thyroxine or triiodo-L-thyronine.','(1) 2 iodide + H(2)O(2) + 2 H(+) = diiodine + 2 H(2)O.(2) [Thyroglobulin]-L-tyrosine + iodide + H(2)O(2) = [thyroglobulin]-3-iodo-L-tyrosine + 2 H(2)O.(3) [Thyroglobulin]-3-iodo-L-tyrosine + iodide + H(2)O(2) =[thyroglobulin]-3,5-diiodo-L-tyrosine + 2 H(2)O.(4) 2 [thyroglobulin]-3,5-diiodo-L-tyrosine + H(2)O(2) = [thyroglobulin]-L-thyroxine + [thyroglobulin]-aminoacrylate + 2 H(2)O.(5) [Thyroglobulin]-3-iodo-L-tyrosine + [thyroglobulin]-3,5-diiodo-L-tyrosine + H(2)O(2) = [thyroglobulin]-3,5,3-triiodo-L-thyronine +[thyroglobulin]-aminoacrylate + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.9','Glutathione peroxidase','NULL','Selenium.','Steroid and lipid hydroperoxides, but not the product of reaction ofEC 1.13.11.12 on phospholipids, can act as acceptor, but more slowlythan H(2)O(2) (cf. EC 1.11.1.12).','2 glutathione + H(2)O(2) = glutathione disulfide + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.10','Chloride peroxidase','Chloroperoxidase','Heme.','Brings about the chlorination of a range of organic molecules,forming stable C-Cl bonds.Also oxidizes bromide and iodide.Enzymes of this type are either heme-thiolate proteins, or containvanadate.A secreted enzyme produced by the ascomycetous fungus Caldariomycesfumago (Leptoxyphium fumago) is an example of the heme-thiolate type.It catalyzes the production of hypochlorous acid by transferring oneoxygen atom from H(2)O(2) to chloride.At a separate site it catalyzes the chlorination of activatedaliphatic and aromatic substrates, via HClO and derived chlorinespecies.In the absence of halides, it shows peroxidase (e.g. phenoloxidation) and peroxygenase activities.The latter inserts oxygen from H(2)O(2) into, for example, styrene(side chain epoxidation) and toluene (benzylic hydroxylation),however, these activities are less pronounced than its activity withhalides.Has little activity with non-activated substrates such as aromaticrings, ethers or saturated alkanes.The chlorinating peroxidase produced by ascomycetous fungi (e.g.Curvularia inaequalis) is an example of a vanadium chloroperoxidase,and is related to bromide peroxidase (EC 1.11.1.18).It contains vanadate and oxidizes chloride, bromide and iodide intohypohalous acids.In the absence of halides, it peroxygenates organic sulfides andoxidizes ABTS (2,2-azinobis(3-ethylbenzthiazoline-6-sulfonic acid))but no phenols.','RH + Cl(-) + H(2)O(2) = RCl + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.11','L-ascorbate peroxidase','Ascorbic acid peroxidase','Heme.','Oxidizes ascorbate and low molecular weight aromatic substrates.The monodehydroascorbate radical produced is either directly reducedback to ascorbate by EC 1.6.5.4 or undergoes non-enzymaticdisproportionation to ascorbate and dehydroascorbate.','2 L-ascorbate + H(2)O(2) + 2 H(+) = L-ascorbate + L-dehydroascorbate +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.12','Phospholipid-hydroperoxide glutathione peroxidase','Peroxidation-inhibiting protein','Selenium.','The products of action of EC 1.13.11.12 on phospholipids can act asacceptor.Also acts on H(2)O(2), but much more slowly (cf. EC 1.11.1.9).','2 glutathione + a lipid hydroperoxide = glutathione disulfide + lipid +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.13','Manganese peroxidase','Mn-dependent peroxidase','Heme.','Involved in the oxidative degradation of lignin in white rotBasidiomycetes.','2 Mn(2+) + 2 H(+) + H(2)O(2) = 2 Mn(3+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.14','Lignin peroxidase','Diarylpropane oxygenase','Heme.','Involved in the oxidative breakdown of lignin by white-rotbasidiomycete fungi.The enzyme from Phanerochaete chrysosporium oxidizes typicalperoxidase dye substrates at the heme iron, a reaction involving theformation of Compound II (Fe(IV)=O); it also oxidizes (3,4-dimethoxyphenyl)methanol (veratryl alcohol) to the radical cation.The bound veratryl alcohol radical is proposed to bring about theoxidative cleavage of C-C and ether (C-O-C) bonds in lignin modelcompounds of the diarylpropane and arylpropane-aryl ether type.','(3,4-dimethoxyphenyl)methanol + H(2)O(2) = 3,4-dimethoxybenzaldehyde +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.15','Peroxiredoxin','AhpC','NULL','Peroxiredoxins (Prxs) are a ubiquitous family of antioxidantproteins.They can be divided into three classes: typical 2-Cys, atypical 2-Cysand 1-Cys peroxiredoxins.The peroxidase reaction comprises two steps centered around a redox-active cysteine called the peroxidatic cysteine.All three peroxiredoxin classes have the first step in common,in which the peroxidatic cysteine attacks the peroxide substrate andis oxidized to S-hydroxycysteine (a sulfenic acid).The second step of the peroxidase reaction, the regeneration ofcysteine from S-hydroxycysteine, distinguishes the threeperoxiredoxin classes.For typical 2-Cys Prxs, in the second step, the peroxidaticS-hydroxycysteine from one subunit is attacked by the resolvingcysteine located in the C-terminus of the second subunit, to form anintersubunit disulfide bond, which is then reduced by one of severalcell-specific thiol-containing reductants (R-SH) (e.g. thioredoxin,AhpF, tryparedoxin or AhpD), completing the catalytic cycle.In the atypical 2-Cys Prxs, both the peroxidatic cysteine and itsresolving cysteine are in the same polypeptide, so their reactionforms an intrachain disulfide bond.To recycle the disulfide, known atypical 2-Cys Prxs appear to usethioredoxin as an electron donor.The 1-Cys Prxs conserve only the peroxidatic cysteine, so that itsoxidized form is directly reduced to cysteine by the reductantmolecule.','2 R-SH + ROOH = R-S-S-R + H(2)O + ROH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.16','Versatile peroxidase','Hybrid peroxidase','Heme.','This ligninolytic peroxidase combines the substrate-specificitycharacteristics of the two other ligninolytic peroxidases,EC 1.11.1.13 and EC 1.11.1.14.Also able to oxidize phenols, hydroquinones and both low- and high-redox-potential dyes, due to a hybrid molecular architecture thatinvolves multiple binding sites for substrates.','(1) Reactive Black 5 + H(2)O(2) = oxidized Reactive Black 5 + 2 H(2)O.(2) Donor + H(2)O(2) = oxidized donor + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.17','Glutathione amide-dependent peroxidase','NULL','NULL','This enzyme, which has been characterized from the proteobacteriumMarichromatium gracile, is a chimeric protein, containing aperoxiredoxin-like N-terminus and a glutaredoxin-like C terminus.The enzyme has peroxidase activity toward hydrogen peroxide andseveral small alkyl hydroperoxides, and is thought to represent anearly adaptation for fighting oxidative stress.The glutathione amide disulfide produced by this enzyme can berestored to glutathione amide by EC 1.8.1.16.','2 glutathione amide + H(2)O(2) = glutathione amide disulfide + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.18','Bromide peroxidase','Bromoperoxidase','NULL','Bromoperoxidases of red and brown marine algae (Rhodophyta andPhaeophyta) contain vanadate.They catalyze the bromination of a range of organic molecules such assesquiterpenes, forming stable C-Br bonds.Bromoperoxidases also oxidize iodides.','RH + HBr + H(2)O(2) = RBr + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.19','Dye decolorizing peroxidase','DyP','NULL','Heme proteins with proximal histidine secreted by basidiomycetousfungi and eubacteria.They are similar to EC 1.11.1.16 versatile peroxidase (oxidation ofReactive Black 5, phenols, veratryl alcohol), but differ from thelatter in their ability to efficiently oxidize a number ofrecalcitrant anthraquinone dyes, and inability to oxidize Mn(II).The model substrate Reactive Blue 5 is converted with high efficiencyvia a so far unique mechanism that combines oxidative and hydrolyticsteps and leads to the formation of phthalic acid.Bacterial TfuDyP catalyzes sulfoxidation.','Reactive Blue 5 + H(2)O(2) + 2 H(+) = oxidized Reactive Blue 5 + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.20','Prostamide/prostaglandin F(2-alpha) synthase','Prostamide F synthase','NULL','The enzyme contains a thioredoxin-type disulfide as a catalyticgroup.Prostamide H(2) and prostaglandin H(2) are the best substrates; thelatter is converted to prostaglandin F(2-alpha).The enzyme also reduces tert-butyl hydroperoxide, cumenehydroperoxide and H(2)O(2), but not prostaglandin D(2) orprostaglandin E(2).','Thioredoxin + prostaglandin H(2) = thioredoxin disulfide + prostaglandinF(2-alpha).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.21','Catalase peroxidase','NULL','NULL','Differs from EC 1.11.1.7, peroxidase, in having a relatively highcatalase (EC 1.11.1.6) activity with H(2)O(2) as donor, releasingO(2); both activities use the same heme active site.In Mycobacterium tuberculosis it is responsible for activation of thecommonly used antitubercular drug, isoniazid.','(1) Donor + H(2)O(2) = oxidized donor + 2 H(2)O.(2) 2 H(2)O(2) = O(2) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.1.22','Hydroperoxy fatty acid reductase','NULL','NULL','The enzyme, characterized from the cyanobacterium SynechocystisPCC 6803, can reduce unsaturated fatty acid hydroperoxides and alkylhydroperoxides.The enzyme, which utilizes NADPH generated by the photosyntheticelectron transfer system, protects the cells from lipid peroxidation.','A hydroperoxy fatty acid + NADPH = a hydroxy fatty acid + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.2.1','Unspecific peroxygenase','Haloperoxidase-peroxygenase','Heme-thiolate.','Enzymes of this type include glycoproteins secreted by agaricbasidiomycetes.They catalyze the insertion of an oxygen atom from H(2)O(2) into awide variety of substrates, including aromatic rings such asnaphthalene, toluene, phenanthrene, pyrene and p-nitrophenol,recalcitrant heterocycles such as pyridine, dibenzofuran, variousethers (resulting in O-dealkylation) and alkanes such as propane,hexane and cyclohexane.Reactions catalyzed include hydroxylation, epoxidation, N-oxidation,sulfooxidation, O- and N-dealkylation, bromination and one-electronoxidations.They have little or no activity toward chloride.Mechanistically, the catalytic cycle of unspecific (mono)-peroxygenases combines elements of the shunt pathway of cytochromeP450s (a side activity that utilizes a peroxide in place of dioxygenand NAD(P)H) and the classic heme peroxidase cycle.','RH + H(2)O(2) = ROH + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.2.2','Myeloperoxidase','MPO','Calcium; Heme.','Is present in phagosomes of neutrophils and monocytes, where thehypochlorite produced is strongly bactericidal.It differs from EC 1.11.1.10 in its preference for formation ofhypochlorite over the chlorination of organic substrates underphysiological conditions (pH 5-8).Hypochlorite in turn forms a number of antimicrobial products (Cl(2),chloramines, hydroxyl radical, singlet oxygen).MPO also oxidizes bromide, iodide and thiocyanate.In the absence of halides, it oxidizes phenols and has a moderateperoxygenase activity toward styrene.','Cl(-) + H(2)O(2) + H(+) = HClO + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.2.3','Plant seed peroxygenase','Plant peroxygenase','Heme.','A heme protein with calcium binding motif (caleosin-type).Enzymes of this type include membrane-bound proteins found in seedsof different plants.They catalyze the direct transfer of one oxygen atom from an organichydroperoxide, which is reduced into its corresponding alcohol to asubstrate which will be oxidized.Reactions catalyzed include hydroxylation, epoxidation andsulfoxidation.Preferred substrate and cosubstrate are unsaturated fatty acids andfatty acid hydroperoxides, respectively.Plant seed peroxygenase is involved in the synthesis of cutin.','RH + ROOH = ROH + ROH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.11.2.4','Fatty-acid peroxygenase','CYP152A1','Heme-thiolate.','A cytosolic heme-thiolate protein with sequence homology to P450monooxygenases.Unlike the latter, it needs neither NAD(P)H, dioxygen nor specificreductases for function.Enzymes of this type are produced by bacteria (e.g. Sphingomonaspaucimobilis, Bacillus subtilis).Catalytic turnover rates are high compared with those ofmonooxygenation reactions as well as peroxide shunt reactionscatalyzed by the common P450s.A model substrate is myristate, but other saturated and unsaturatedfatty acids are also hydroxylated.Oxidizes the peroxidase substrate 3,3,5,5-tetramethylbenzidine (TMB)and peroxygenates aromatic substrates in a fatty-acid-dependentreaction.','Fatty acid + H(2)O(2) = 3- or 2-hydroxy fatty acid + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.1.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.1.2','Hydrogen dehydrogenase','Bidirectional hydrogenase','FAD or FMN; Iron-sulfur; Nickel.','NULL','H(2) + NAD(+) = H(+) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.1.3','Hydrogen dehydrogenase (NADP(+))','NADP(+)-linked hydrogenase','Iron; Iron-sulfur; Nickel.','The protein from the bacterium Desulfovibrio fructosovoransexclusively functions as a hydrogen dehydrogenase, while the enzymefrom the archaeon Pyrococcus furiosus is part of a heterotetramericcomplex where the alpha and delta subunits function as a hydrogenasewhile the beta and gamma subunits function as sulfur reductase(EC 1.12.98.4).Different from EC 1.12.1.5.','H(2) + NADP(+) = H(+) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.1.4','Hydrogenase (NAD(+), ferredoxin)','Bifurcating (FeFe) hydrogenase','NULL','The enzyme from Thermotoga maritima contains a (FeFe) cluster(H-cluster) and iron-sulfur clusters.It works in the direction evolving hydrogen as a means of eliminatingexcess reducing equivalents.','2 H(2) + NAD(+) + 2 oxidized ferredoxin = 5 H(+) + NADH + 2 reducedferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.1.5','Hydrogen dehydrogenase (NAD(P)(+))','NULL','Iron; Iron-sulfur; Nickel.','The enzyme from the archaeon Pyrococcus furiosus is part of aheterotetrameric complex where the alpha and delta subunits functionas a hydrogenase while the beta and gamma subunits function as sulfurreductase (EC 1.12.98.4).Different from EC 1.12.1.3.','H(2) + NAD(P)(+) = H(+) + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.2.1','Cytochrome-c3 hydrogenase','Cytochrome c3 reductase','Iron-sulfur; Nickel.','Some forms of the enzyme contain nickel ([NiFe]-hydrogenases) and,of these, some contain selenocysteine ([NiFeSe]-hydrogenases).Methylene blue and other acceptors can also be reduced.','2 H(2) + ferricytochrome c3 = 4 H(+) + ferrocytochrome c3.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.5.1','Hydrogen:quinone oxidoreductase','Hydrogen-ubiquinone oxidoreductase','Iron-sulfur; Nickel.','Also catalyzes the reduction of water-soluble quinones (e.g. 2,3-dimethylnaphthoquinone) or viologen dyes (benzyl viologen or methylviologen) by H(2).Formerly EC 1.12.99.3.','H(2) + menaquinone = menaquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.7.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.7.2','Ferredoxin hydrogenase','Bidirectional hydrogenase','Iron-sulfur; Nickel.','Can use molecular hydrogen for the reduction of a variety ofsubstances.Formerly EC 1.12.1.1, EC 1.12.7.1, EC 1.18.3.1, EC 1.18.99.1,and EC 1.98.1.1.','H(2) + 2 oxidized ferredoxin = 2 reduced ferredoxin + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.98.1','Coenzyme F420 hydrogenase','8-hydroxy-5-deazaflavin-reducing hydrogenase','FAD; Iron; Nickel.','The enzyme from some sources contains selenocysteine.The enzyme also reduces the riboflavin analog of F420, flavins andmethylviologen, but to a lesser extent.The hydrogen acceptor coenzyme F420 is a deazaflavin derivative.Formerly EC 1.12.99.1.','H(2) + oxidized coenzyme F420 = reduced coenzyme F420.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.98.2','5,10-methenyltetrahydromethanopterin hydrogenase','H(2)-dependent methylene-H(4)MPT dehydrogenase','NULL','Does not catalyze the reduction of artificial dyes.Does not by itself catalyze a H(2)/H(+) exchange reaction.Does not contain nickel or iron-sulfur clusters.Formerly EC 1.12.99.4.','H(2) + 5,10-methenyltetrahydromethanopterin = H(+) + 5,10-methylenetetrahydromethanopterin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.98.3','Methanosarcina-phenazine hydrogenase','Methanophenazine hydrogenase','Iron-sulfur; Nickel.','The enzyme from some sources contains selenocysteine.Formerly EC 1.12.99.2.','H(2) + 2-(2,3-dihydropentaprenyloxy)phenazine =2-dihydropentaprenyloxyphenazine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.98.4','Sulfhydrogenase','Sulfur reductase','Iron-sulfur.','The enzyme from the hyperthermophilic archaeon Pyrococcus furiosus ispart of two heterotetrameric complexes where the beta and gammasubunits function as sulfur reductase and the alpha and deltasubunits function as hydrogenases (EC 1.12.1.3 and EC 1.12.1.4).Sulfur can also be used as substrate, but since it is insoluble inaqueous solution and polysulfide is generated abiotically by thereaction of hydrogen sulfide and sulfur, polysulfide is believed tobe the true substrate.Formerly EC 1.97.1.3.','H(2) + polysulfide(n) = hydrogen sulfide + polysulfide(n-1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.99.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.99.2','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.99.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.99.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.99.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.12.99.6','Hydrogenase (acceptor)','H(2) producing hydrogenase','Iron-sulfur; Nickel.','Uses molecular hydrogen for the reduction of a variety of substances.','H(2) + A = AH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.7','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.11','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.12','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.1.13','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.1','Catechol 1,2-dioxygenase','Catechase','Iron.','Involved in the metabolism of nitro-aromatic compounds by a strain ofPseudomonas putida.Formerly EC 1.13.1.1 and EC 1.99.2.2.','Catechol + O(2) = cis,cis-muconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.2','Catechol 2,3-dioxygenase','2,3-pyrocatechase','Fe(2+).','The enzyme initiates the meta-cleavage pathway of catecholdegradation.Formerly EC 1.13.1.2.','Catechol + O(2) = 2-hydroxymuconate-6-semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.3','Protocatechuate 3,4-dioxygenase','Protocatechuate oxygenase','Iron.','The enzyme, which participates in the degradation of aromaticcompounds, catalyzes the intradiol addition of both oxygen atoms frommolecular oxygen, resulting in ortho-cleavage of the aromatic ring.The type of cleavage leads to mineralization via the intermediate3-oxoadipate.Formerly EC 1.13.1.3 and EC 1.99.2.3.','3,4-dihydroxybenzoate + O(2) = 3-carboxy-cis,cis-muconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.4','Gentisate 1,2-dioxygenase','Gentisate oxygenase','Iron.','Formerly EC 1.13.1.4 and EC 1.99.2.4.','2,5-dihydroxybenzoate + O(2) = maleylpyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.5','Homogentisate 1,2-dioxygenase','Homogentisate oxygenase','Iron.','Formerly EC 1.13.1.5 and EC 1.99.2.5.','Homogentisate + O(2) = 4-maleylacetoacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.6','3-hydroxyanthranilate 3,4-dioxygenase','3-hydroxyanthranilate oxygenase','Iron.','The product of the reaction spontaneously rearrange to quinolinicacid (quin).Formerly EC 1.13.1.6.','3-hydroxyanthranilate + O(2) = 2-amino-3-carboxymuconate semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.7','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.8','Protocatechuate 4,5-dioxygenase','Protocatechuate 4,5-oxygenase','Iron.','Formerly EC 1.13.1.8.','Protocatechuate + O(2) = 4-carboxy-2-hydroxymuconate semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.9','2,5-dihydroxypyridine 5,6-dioxygenase','2,5-dihydroxypyridine oxygenase','Iron.','Formerly EC 1.13.1.9.','2,5-dihydroxypyridine + O(2) = N-formylmaleamate + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.10','7,8-dihydroxykynurenate 8,8a-dioxygenase','7,8-dihydroxykynurenate oxygenase','Iron.','Formerly EC 1.13.1.10.','7,8-dihydroxykynurenate + O(2) = 5-(3-carboxy-3-oxopropenyl)-4,6-dihydroxypyridine-2-carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.11','Tryptophan 2,3-dioxygenase','Indolamine 2,3-dioxygenase','Heme.','Catalyzes, together with EC 1.13.11.52, the first and rate-limitingstep in the kynurenine pathway, the major pathway of tryptophanmetabolism.Specific for tryptophan as substrate, but is far more active withL-tryptophan than with D-tryptophan.Formerly EC 1.11.1.4 and EC 1.13.1.12.','L-tryptophan + O(2) = N-formyl-L-kynurenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.12','Linoleate 13S-lipoxygenase','13-lipoperoxidase','Iron.','A common plant lipoxygenase that oxidizes linoleate and alpha-linolenate, the two most common polyunsaturated fatty acids inplants, by inserting molecular oxygen at the C(13) position with (S)-configuration.This enzyme produces precursors for several important compounds,including the plant hormone jasmonic acid.EC 1.13.11.58 catalyzes a similar reaction at the second availableposition of these fatty acids.Formerly EC 1.13.1.13 and EC 1.99.2.1.','(1) Linoleate + O(2) = (9Z,11E,13S)-13-hydroperoxyoctadeca-9,11-dienoate.(2) Alpha-linolenate + O(2) = (9Z,11E,13S,15Z)-13-hydroperoxyoctadeca-9,11,15-trienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.13','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.14','2,3-dihydroxybenzoate 3,4-dioxygenase','2,3-dihydroxybenzoic oxygenase','NULL','NULL','2,3-dihydroxybenzoate + O(2) = 3-carboxy-2-hydroxymuconate semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.15','3,4-dihydroxyphenylacetate 2,3-dioxygenase','Homoprotocatechuate 2,3-dioxygenase','Iron.','NULL','3,4-dihydroxyphenylacetate + O(2) = 2-hydroxy-5-carboxymethylmuconatesemialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.16','3-carboxyethylcatechol 2,3-dioxygenase','2,3-dihydroxy-beta-phenylpropionate oxygenase','Iron.','This enzyme catalyzes a step in the pathway of phenylpropanoidcompounds degradation.','(1) 3-(2,3-dihydroxyphenyl)propanoate + O(2) = (2Z,4E)-2-hydroxy-6-oxonona-2,4-diene-1,9-dioate.(2) (2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate + O(2) = (2Z,4E,7E)-2-hydroxy-6-oxonona-2,4,7-triene-1,9-dioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.17','Indole 2,3-dioxygenase','Indole oxidase','Copper; Flavoprotein.','Enzymes from the plants Tecoma stans, Jasminum grandiflorum and Zeamays are flavoproteins containing copper.They are part of enzyme systems that form either anthranil (3,4-benzoisoxazole) (Tecoma stans), anthranilate (Jasminum grandiflorum)or both (Zea mays) as the final product.A second enzyme from Tecoma stans is not a flavoprotein, does notrequire copper, and is part of a system that forms anthranilate asthe final product.','Indole + O(2) = 2-formylaminobenzaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.18','Sulfur dioxygenase','Sulfur oxygenase','Iron.','Glutathione (GSH) plays a catalytic role in elemental sulfuractivation, but is not consumed during the enzymic reaction.GSH and elemental sulfur react non-enzymically to yieldS-sulfanylglutathione and it is only in this form that the sulfur canbe oxidized to yield sulfite.The sulfite can be further converted into sulfate, thiosulfate orS-sulfoglutathione (GSSO(3)(-)) non-enzymically.','Sulfur + O(2) + H(2)O = sulfite + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.19','Cysteamine dioxygenase','Cysteamine oxygenase','Iron.','Involved in the biosynthesis of taurine.Requires catalytic amounts of a cofactor-like compound, such assulfur, sufide, selenium or methylene blue for maximal activity.3-aminopropanethiol (homocysteamine) and 2-mercaptoethanol can alsoact as substrates, but glutathione, cysteine, and cysteine ethyl- andmethyl esters are not good substrates.','2-aminoethanethiol + O(2) = hypotaurine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.20','Cysteine dioxygenase','NULL','Iron; NAD(P)H.','NULL','L-cysteine + O(2) = 3-sulfinoalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.21','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.22','Caffeate 3,4-dioxygenase','NULL','NULL','NULL','3,4-dihydroxy-trans-cinnamate + O(2) = 3-(2-carboxyethenyl)-cis,cis-muconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.23','2,3-dihydroxyindole 2,3-dioxygenase','NULL','NULL','NULL','2,3-dihydroxyindole + O(2) = anthranilate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.24','Quercetin 2,3-dioxygenase','NULL','Iron or copper.','Quercetin is a flavonol (5,7,3,4-tetrahydroxyflavonol).','Quercetin + O(2) = 2-(3,4-dihydroxybenzoyloxy)-4,6-dihydroxybenzoate +CO + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.25','3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione 4,5-dioxygenase','3-alkylcatechol 2,3-dioxygenase','Iron.','Also acts on 3-isopropylcatechol and 3-tert-butyl-5-methylcatechol.','3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + O(2) =3-hydroxy-5,9,17-trioxo-4,5:9,10-disecoandrosta-1(10),2-dien-4-oate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.26','Peptide-tryptophan 2,3-dioxygenase','Pyrrolooxygenase','NULL','Also acts on tryptophan.','[Protein]-L-tryptophan + O(2) = [protein]-N-formyl-L-kynurenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.27','4-hydroxyphenylpyruvate dioxygenase','4-hydroxyphenylpyruvate hydroxylase','Iron.','Formerly EC 1.14.2.2 and EC 1.99.1.14.','4-hydroxyphenylpyruvate + O(2) = homogentisate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.28','2,3-dihydroxybenzoate 2,3-dioxygenase','NULL','NULL','Also acts, more slowly, with 2,3-dihydroxy-4-methylbenzoate and 2,3-dihydroxy-4-isopropylbenzoate.','2,3-dihydroxybenzoate + O(2) = 2-carboxy-cis,cis-muconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.29','Stizolobate synthase','NULL','Zinc.','The intermediate product undergoes ring closure and oxidation, withNAD(P)(+) as acceptor, to stizolobic acid.','L-dopa + O(2) = 4-(L-alanin-3-yl)-2-hydroxy-cis,cis-muconate6-semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.30','Stizolobinate synthase','NULL','Zinc.','The intermediate product undergoes ring closure and oxidation, withNAD(P)(+) as acceptor, to stizolobinic acid.','L-dopa + O(2) = 5-(L-alanin-3-yl)-2-hydroxy-cis,cis-muconate6-semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.31','Arachidonate 12-lipoxygenase','12-lipoxygenase','Iron.','The product is rapidly reduced to the corresponding 12S-hydroxycompound.','Arachidonate + O(2) = (5Z,8Z,10E,14Z)-(12S)-12-hydroperoxyicosa-5,8,10,14-tetraenoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.32','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.33','Arachidonate 15-lipoxygenase','15-lipoxygenase','Iron.','The product is rapidly converted to the corresponding 15S-hydroxycompound.','Arachidonate + O(2) = (5Z,8Z,11Z,13E)-(15S)-15-hydroperoxyicosa-5,8,11,13-tetraenoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.34','Arachidonate 5-lipoxygenase','5-Delta-lipoxygenase','Iron.','NULL','Arachidonate + O(2) = leukotriene A(4) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.35','Pyrogallol 1,2-oxygenase','NULL','NULL','NULL','1,2,3-trihydroxybenzene + O(2) = (2Z,4E)-2-hydroxyhexa-2,4-dienedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.36','Chloridazon-catechol dioxygenase','NULL','Iron.','Involved in the breakdown of the herbicide chloridazon.Not identical with EC 1.13.11.1, EC 1.13.11.2 or EC 1.13.11.5.','5-amino-4-chloro-2-(2,3-dihydroxyphenyl)-3(2H)-pyridazinone + O(2) =5-amino-4-chloro-2-(2-hydroxymuconoyl)-3(2H)-pyridazinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.37','Hydroxyquinol 1,2-dioxygenase','NULL','Iron.','Highly specific; catechol and pyrogallol are acted on at less than 1%of the rate at which hydroxyquinol is oxidized.','Benzene-1,2,4-triol + O(2) = maleylacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.38','1-hydroxy-2-naphthoate 1,2-dioxygenase','1-hydroxy-2-naphthoate-degrading enzyme','Iron.','Involved, with EC 4.1.2.34, in the metabolism of phenanthrene inbacteria.','1-hydroxy-2-naphthoate + O(2) = (3Z)-4-(2-carboxyphenyl)-2-oxobut-3-enoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.39','Biphenyl-2,3-diol 1,2-dioxygenase','2,3-dihydroxybiphenyl dioxygenase','Manganese or iron.','This enzyme participates in the degradation pathway of biphenyl andPCB (poly chlorinated biphenyls), and catalyzes the first ringcleavage step by incorporating two oxygen atoms into the catecholring formed by EC 1.3.1.56.The enzyme from the bacterium Burkholderia xenovorans LB400 can alsoprocess catechol, 3-methylcatechol, and 4-methylcatechol, but lessefficiently.The enzyme from the carbazole-degrader Pseudomonas resinovoransstrain CA10 also accepts 2-aminobiphenyl-2,3-diol.The enzyme from Ralstonia sp. SBUG 290 can also accept 1,2-dihydroxydibenzofuran and 1,2-dihydroxynaphthalene.The enzyme is strongly inhibited by the substrate.Not identical with EC 1.13.11.2.','Biphenyl-2,3-diol + O(2) = 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.40','Arachidonate 8-lipoxygenase','8-lipoxygenase','Iron.','An enzyme from the coral Pseudoplexaura porosa.','Arachidonate + O(2) = (5Z,9E,11Z,14Z)-(8R)-8-hydroperoxyicosa-5,9,11,14-tetraenoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.41','2,4-dihydroxyacetophenone dioxygenase','NULL','NULL','NULL','2,4-dihydroxyacetophenone + O(2) = 4-hydroxybenzoate + formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.42','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.43','Lignostilbene alpha-beta-dioxygenase','NULL','Iron.','The enzyme catalyzes oxidative cleavage of the interphenyl doublebond in the synthetic substrate and lignin-derived stilbenes.It is responsible for the degradation of a diarylpropane-typestructure in lignin.','1,2-bis(4-hydroxy-3-methoxyphenyl)ethylene + O(2) = 2 vanillin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.44','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.45','Linoleate 11-lipoxygenase','Linoleate dioxygenase','NULL','The product (9Z,12Z)-(11S)-11-hydroperoxyoctadeca-9,12-dienoate,is converted, more slowly, into (9Z,11E)-(13R)-13-hydroperoxyoctadeca-9,11-dienoate.The enzyme from the fungus Gaeumannomyces graminis requires Mn(2+).It also acts on alpha-linolenate, whereas gamma-linolenate is a poorsubstrate.Oleate and arachidonate are not substrates.','Linoleate + O(2) = (9Z,12Z)-(11S)-11-hydroperoxyoctadeca-9,12-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.46','4-hydroxymandelate synthase','4-hydroxyphenylpyruvate dioxygenase II','Iron.','Involved in the biosynthesis of the vancomycin group of glycopeptideantibiotics.','4-hydroxyphenylpyruvate + O(2) = 4-hydroxymandelate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.47','3-hydroxy-4-oxoquinoline 2,4-dioxygenase','(1H)-3-hydroxy-4-oxoquinoline 2,4-dioxygenase','NULL','Does not contain a metal center or organic cofactor.Fission of two C-C bonds: 2,4-dioxygenolytic cleavage withconcomitant release of carbon monoxide.The enzyme from Pseudomonas putida is highly specific for thissubstrate.Formerly EC 1.12.99.5 and EC 1.13.99.5.','3-hydroxy-1H-quinolin-4-one + O(2) = N-formylanthranilate + CO.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.48','3-hydroxy-2-methylquinolin-4-one 2,4-dioxygenase','(1H)-3-hydroxy-4-oxoquinaldine 2,4-dioxygenase','NULL','Does not contain a metal center or organic cofactor.Fission of two C-C bonds: 2,4-dioxygenolytic cleavage withconcomitant release of carbon monoxide.The enzyme from Arthrobacter sp. can also act on 3-hydroxy-4-oxoquinoline, forming N-formylanthranilate and CO (cf.EC 1.13.11.47), but more slowly.','3-hydroxy-2-methyl-1H-quinolin-4-one + O(2) = N-acetylanthranilate + CO.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.49','Chlorite O(2)-lyase','Chlorite dismutase','Iron; Protoheme IX.','Reaction occurs in the reverse direction in chlorate- andperchlorate-reducing bacteria.There is no activity when chlorite is replaced by hydrogen peroxide,perchlorate, chlorate or nitrite.The term chlorite dismutase is misleading as the reaction does notinvolve dismutation/disproportionation.','Chloride + O(2) = chlorite.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.50','Acetylacetone-cleaving enzyme','Acetylacetone dioxygenase','Iron.','Forms the first step in the acetylacetone degradation pathway ofAcinetobacter johnsonii.While acetylacetone is by far the best substrate, heptane-3,5-dione,octane-2,4-dione, 2-acetylcyclohexanone and ethyl acetoacetate canalso act as substrates.','Pentane-2,4-dione + O(2) = acetate + methylglyoxal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.51','9-cis-epoxycarotenoid dioxygenase','NCED','Iron.','Acts on 9-cis-violaxanthin and 9-cis-neoxanthin but not on the all-trans isomers.In vitro, it will cleave 9-cis-zeaxanthin.Catalyzes the first step of abscisic-acid biosynthesis fromcarotenoids in chloroplasts, in response to water stress.The other enzymes involved in the abscisic-acid biosynthesis pathwayare EC 1.1.1.288, EC 1.2.3.14 and EC 1.14.13.93.','(1) A 9-cis-epoxycarotenoid + O(2) = 2-cis,4-trans-xanthoxin + a 12-apo-carotenal.(2) 9-cis-violaxanthin + O(2) = 2-cis,4-trans-xanthoxin + (3S,5R,6S)-5,6-epoxy-3-hydroxy-5,6-dihydro-12-apo-beta-caroten-12-al.(3) 9-cis-neoxanthin + O(2) = 2-cis,4-trans-xanthoxin + (3S,5R,6R)-5,6-dihydroxy-6,7-didehydro-5,6-dihydro-12-apo-beta-caroten-12-al.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.52','Indoleamine 2,3-dioxygenase','IDO','Heme.','Requires ascorbic acid and methylene blue for activity.Has broader substrate specificity than EC 1.13.11.11.Is induced in response to pathological conditions and host-defensemechanisms.While the enzyme is more active with D-tryptophan than L-tryptophan,its only known function to date is in the metabolism of L-tryptophan.Superoxide radicals can replace O(2) as oxygen donor.','(1) D-tryptophan + O(2) = N-formyl-D-kynurenine.(2) L-tryptophan + O(2) = N-formyl-L-kynurenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.53','Acireductone dioxygenase (Ni(2+)-requiring)','2-hydroxy-3-keto-5-thiomethylpent-1-ene dioxygenase','Nickel.','If Fe(2+) is bound instead of Ni(2+), the reaction catalyzed byEC 1.13.11.54 occurs instead.The enzyme from Klebsiella oxytoca (formerly Klebsiella pneumoniae)ATCC 8724 is involved in the methionine salvage pathway.','1,2-dihydroxy-5-(methylthio)pent-1-en-3-one + O(2) =3-(methylthio)propanoate + formate + CO.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.54','Acireductone dioxygenase (Fe(2+)-requiring)','2-hydroxy-3-keto-5-thiomethylpent-1-ene dioxygenase','Fe(2+).','If Ni(2+) is bound instead of Fe(2+), the reaction catalyzed byEC 1.13.11.53 occurs instead.The enzyme from Klebsiella oxytoca (formerly Klebsiella pneumoniae)ATCC 8724 is involved in the methionine salvage pathway.','1,2-dihydroxy-5-(methylthio)pent-1-en-3-one + O(2) = 4-(methylthio)-2-oxobutanoate + formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.55','Sulfur oxygenase/reductase','Sulfur oxygenase','Iron.','Elemental sulfur is both the electron donor and one of the two knownacceptors, the other being oxygen.Another reaction product is thiosulfate, but this is probably formednon-enzymically at elevated temperature from sulfite and sulfur.This enzyme differs from EC 1.13.11.18 and EC 1.12.98.4 in that bothactivities are found together.','4 sulfur + 4 H(2)O + O(2) = 2 H(2)S + 2 HSO(3)(-) + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.56','1,2-dihydroxynaphthalene dioxygenase','1,2-DHN dioxygenase','Iron.','This enzyme is involved in naphthalene degradation.','Naphthalene-1,2-diol + O(2) = 2-hydroxy-2H-chromene-2-carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.57','Gallate dioxygenase','NULL','Iron.','The enzyme is a ring-cleavage dioxygenase that acts specifically ongallate to produce the keto-tautomer of 4-oxalomesaconate.','Gallate + O(2) = (1E)-4-oxobut-1-ene-1,2,4-tricarboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.58','Linoleate 9S-lipoxygenase','9-lipoxygenase','Iron.','A common plant lipoxygenase that oxidizes linoleate and alpha-linolenate, the two most common polyunsaturated fatty acids inplants, by inserting molecular oxygen at the C(9) position with (S)-configuration.The enzyme plays a physiological role during the early stages ofseedling growth.The enzyme from Arabidopsis thaliana shows comparable activity towardlinoleate and linolenate.EC 1.13.11.12 catalyzes a similar reaction at another position ofthese fatty acids.','Linoleate + O(2) = (9S,10E,12Z)-9-hydroperoxy-10,12-octadecadienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.59','Torulene dioxygenase','NULL','NULL','It is assumed that 3-methylbut-2-enal is formed.The enzyme cannot cleave the saturated 3,4-bond of gamma-carotenewhich implies that a 3,4-double bond is neccessary for thisreaction.','Torulene + O(2) = 4-apo-beta,psi-caroten-4-al + 3-methylbut-2-enal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.60','Linoleate 8R-lipoxygenase','5,8-linoleate diol synthase (bifunctional enzyme)','Heme.','The bifunctional enzyme from Aspergillus nidulans uses different hemedomains to catalyze two separate reactions.Linoleic acid is oxidized within the N-terminal heme peroxidasedomain to (8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate, which issubsequently isomerized by the C-terminal P450 heme thiolate domainto (5S,8R,9Z,12Z)-5,8-dihydroxyoctadeca-9,12-dienoate (cf.EC 5.4.4.5).The bifunctional enzyme from Gaeumannomyces graminis also catalyzesthe oxidation of linoleic acid to (8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate, but its second domain isomerizes it to (7S,8S,9Z,12Z)-5,8-dihydroxyoctadeca-9,12-dienoate (cf. EC 5.4.4.6).Formerly EC 1.13.11.44.','Linoleate + O(2) = (8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.61','Linolenate 9R-lipoxygenase','(9R)-LOX','NULL','In cyanobacteria the enzyme is involved in oxylipin biosynthesis.The enzyme also converts linoleate to (9R,10E,12Z)-9-hydroperoxyoctadeca-10,12-dienoate.','Alpha-linolenate + O(2) = (9R,10E,12Z,15Z)-9-hydroperoxyoctadeca-10,12,15-trienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.62','Linoleate 10R-lipoxygenase','(10R)-dioxygenase','NULL','The enzyme is involved in biosynthesis of oxylipins, which affectsporulation, development, and pathogenicity of Aspergillus spp.','Linoleate + O(2) = (8E,10R,12Z)-10-hydroperoxyoctadeca-8,12-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.63','Beta-carotene 15,15-dioxygenase','NULL','Iron.','The enzyme, which has been characterized from the uncultured marinebacterium 66A03, is involved in the proteorhodopsin system, whichuses retinal as its chromophore.The enzyme cleaves beta-carotene symmetrically, producing twomolecules of all-trans-retinal.Both atoms of the oxygen molecule are incorporated into the products.It requires a substrate greater than C(35) that contains at least oneunsubstituted beta-end group.Cf. EC 1.14.99.36.Formerly EC 1.13.11.n2.','Beta-carotene + O(2) = 2 all-trans-retinal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.64','5-nitrosalicylate dioxygenase','NULL','NULL','The enzyme, characterized from the soil bacterium Bradyrhizobium sp.JS329, is involved in the pathway of 5-nitroanthranilate degradation.It is unusual in being able to catalyze the ring fission without therequirement for prior removal of the nitro group.The product undergoes spontaneous lactonization, with concurrentelimination of the nitro group.','5-nitrosalicylate + O(2) = 2-oxo-3-(5-oxofuran-2-ylidene)propanoate +nitrite.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.65','Carotenoid isomerooxygenase','NULL','NULL','The enzyme, characterized from the moth Galleria mellonella and thefruit fly Drosophila melanogaster, is involved in the synthesis ofretinal from dietary caroteoids in insects.The enzyme accepts different all-trans carotenoids, including beta-carotene, alpha-carotene and lutein, and catalyzes the symmetricalcleavage of the carotenoid and the simultaneous isomerization of onlyone of the products to a cis configuration.When the substrate is hydroxylated only in one side (as incryptoxanthin), the enzyme preferentially isomerizes the hydroxylatedpart of the molecule.Formerly EC 1.14.13.164.','Zeaxanthin + O(2) = (3R)-11-cis-3-hydroxyretinal + (3R)-all-trans-3-hydroxyretinal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.66','Hydroquinone 1,2-dioxygenase','Hydroquinone dioxygenase','Iron.','The enzyme is an extradiol-type dioxygenase, and is a member of thenonheme-iron(II)-dependent dioxygenase family.It catalyzes the ring cleavage of a wide range of hydroquinonesubstrates to produce the corresponding 4-hydroxymuconicsemialdehydes.','Benzene-1,4-diol + O(2) = (2E,4Z)-4-hydroxy-6-oxohexa-2,4-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.67','8-apo-beta-carotenoid 14,13-cleaving dioxygenase','NULL','NULL','A thiol-dependent enzyme.Unlike EC 1.14.99.36, it is not active toward beta-carotene.The secondary product has not been characterized, but may be (3E,5E)-7-hydroxy-6-methylhepta-3,5-dien-2-one.Formerly EC 1.13.12.12.','8-apo-beta-carotenol + O(2) = 14-apo-beta-carotenal +an uncharacterized product.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.68','9-cis-beta-carotene 9,10-cleaving dioxygenase','NULL','Iron.','The enzyme participates in a pathway leading to biosynthesis ofstrigolactones, plant hormones involved in promotion of symbioticassociations known as arbuscular mycorrhiza.','9-cis-beta-carotene + O(2) = 9-cis-10-apo-beta-carotenal + beta-ionone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.69','Carlactone synthase','NULL','Iron.','The enzyme participates in a pathway leading to biosynthesis ofstrigolactones, plant hormones involved in promotion of symbioticassociations known as arbuscular mycorrhiza.Also catalyzes EC 1.13.11.70, but 10-fold slower.','9-cis-10-apo-beta-carotenal + 2 O(2) = carlactone + (2E,4E,6E)-7-hydroxy-4-methylhepta-2,4,6-trienal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.70','All-trans-10-apo-beta-carotenal 13,14-cleaving dioxygenase','Carotenoid cleavage dioxygenase 8','Iron.','The enzyme from the plant Arabidopsis thaliana also catalyzesEC 1.13.11.69, 10-fold faster.Formerly EC 1.14.99.n5.','All-trans-10-apo-beta-carotenal + O(2) = 13-apo-beta-carotenone +(2E,4E,6E)-4-methylocta-2,4,6-trienedial.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.71','Carotenoid-9,10-cleaving dioxygenase','Beta-carotene 9,10-monooxygenase','Iron.','The enzyme catalyzes the asymmetric oxidative cleavage ofcarotenoids.The mammalian enzyme can also cleave all-trans-lycopene.Formerly EC 1.14.99.n2.','All-trans-beta-carotene + O(2) = all-trans-10-apo-beta-carotenal + beta-ionone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.72','2-hydroxyethylphosphonate dioxygenase','HEPD','Iron.','Isolated from some bacteria including Streptomyces hygroscopicus andStreptomyces viridochromogenes.The pro-R hydrogen at C-2 of the ethyl group is retained by theformate ion.Any stereochemistry at C-1 of the ethyl group is lost.One atom from dioxygen is present in each product.Involved in phosphinothricin biosynthesis.','2-hydroxyethylphosphonate + O(2) = hydroxymethylphosphonate + formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.73','Methylphosphonate synthase','NULL','NULL','Isolated from the marine archaeon Nitrosopumilus maritimus.','2-hydroxyethylphosphonate + O(2) = methylphosphonate + HCO(3)(-).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.74','2-aminophenol 1,6-dioxygenase','NULL','Fe(2+).','The enzyme, is an extradiol-type dioxygenase that utilizes a non-hemeferrous iron to cleave the aromatic ring at the meta position(relative to the hydroxyl substituent).The enzyme also has some activity with 2-amino-5-methylphenol and2-amino-4-methylphenol.The enzyme from the bacterium Comamonas testosteroni CNB-1 also hasthe activity of 2-amino-5-chlorophenol 1,6-dioxygenase.Formerly EC 1.13.11.n1.','2-aminophenol + O(2) = 2-aminomuconate 6-semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.75','All-trans-8-apo-beta-carotenal 15,15-oxygenase','8-apo-beta-carotenal 15,15-oxygenase','Fe(2+).','The enzyme is involved in retinal biosynthesis in bacteria.Formerly EC 1.14.99.41 and EC 1.14.99.n1.','All-trans-8-apo-beta-carotenal + O(2) = all-trans-retinal + (2E,4E,6E)-2,6-dimethylocta-2,4,6-trienedial.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.76','2-amino-5-chlorophenol 1,6-dioxygenase','NULL','Fe(2+).','The enzyme, a member of the nonheme-iron(II)-dependent dioxygenasefamily, is an extradiol-type dioxygenase that utilizes a non-hemeferrous iron to cleave the aromatic ring at the meta position(relative to the hydroxyl substituent).The enzyme from the bacterium Comamonas testosteroni CNB-1 also hasthe activity of EC 1.3.1.74.','2-amino-5-chlorophenol + O(2) = 2-amino-5-chloromuconate 6-semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.77','Oleate 10S-lipoxygenase','(10S)-dioxygenase','Fe(2+).','The enzyme isolated from the bacterium Pseudomonas sp. 42A2 hassimilar activity with all the three Delta(9) fatty acids (cf.EC 1.13.11.62).','(1) Oleate + O(2) = (8E,10S)-10-hydroperoxyoctadeca-8-enoate.(2) Linoleate + O(2) = (8E,10S,12Z)-10-hydroperoxyoctadeca-8,12-dienoate.(3) Alpha-linolenate + O(2) = (8E,10S,12Z,15Z)-10-hydroperoxyoctadeca-8,12,15-trienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.11.n2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.1','Arginine 2-monooxygenase','Arginine decarboxy-oxidase','Flavoprotein.','Also acts on canavanine and homoarginine.','L-arginine + O(2) = 4-guanidinobutanamide + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.2','Lysine 2-monooxygenase','Lysine oxygenase','FAD.','Also acts on other diamino acids.','L-lysine + O(2) = 5-aminopentanamide + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.3','Tryptophan 2-monooxygenase','NULL','NULL','NULL','L-tryptophan + O(2) = (indol-3-yl)acetamide + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.4','Lactate 2-monooxygenase','Lactate oxidase','FMN.','Formerly EC 1.1.3.2.','(S)-lactate + O(2) = acetate + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.5','Renilla-luciferin 2-monooxygenase','Aequorin','NULL','From the soft coral coelenterate Renilla reniformis.The luciferin is bound to a luciferin-binding protein (BP-LH2).The bioluminescent reaction between the luciferin complex, luciferaseand oxygen is triggered by calcium ions.In vivo, the excited state luciferin--luciferase complex undergoesthe process of nonradiative energy transfer to an accessory protein,Renilla green fluorescent protein, which results in greenbioluminescence.In vitro, Renilla luciferase emits blue light in the absence of anygreen fluorescent protein.','Renilla luciferin + O(2) = oxidized Renilla luciferin + CO(2) + light.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.6','Cypridina-luciferin 2-monooxygenase','Cypridina-type luciferase','NULL','Cypridina luciferin is a bioluminescent crustacea.The enzyme may be assayed by measurement of light emission.','Cypridina luciferin + O(2) = oxidized Cypridina luciferin + CO(2) +light.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.7','Photinus-luciferin 4-monooxygenase (ATP-hydrolyzing)','Firefly luciferase','NULL','Photinus (firefly) is a bioluminescent insect.The first step in the reaction is the formation of an acid anhydridebetween the carboxylic group and AMP, with the release ofdiphosphate.The enzyme may be assayed by measurement of light emission.','Photinus luciferin + O(2) + ATP = oxidized Photinus luciferin + CO(2) +AMP + diphosphate + light.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.8','Watasenia-luciferin 2-monooxygenase','Luciferase','NULL','The enzyme may be assayed by measurement of light emission.','Watasenia luciferin + O(2) = oxidized Watasenia luciferin + CO(2) +light.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.9','Phenylalanine 2-monooxygenase','L-phenylalanine oxidase (deaminating and decarboxylating)','NULL','The reaction shown above is about 80% of the reaction catalyzed; theremaining 20% is: L-phenylalanine + O(2) + H(2)O = 3-phenylpyruvicacid + NH(3) + H(2)O(2), a reaction similar to that of EC 1.4.3.2.','L-phenylalanine + O(2) = 2-phenylacetamide + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.10','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.11','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.12','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.13','Oplophorus-luciferin 2-monooxygenase','Oplophorus luciferase','NULL','The luciferase from the deep sea shrimp Oplophorus gracilorostris isa complex composed of more than one protein.The enzymes specificity is quite broad, with both coelenterazine andbisdeoxycoelenterazine being good substrates.','Oplophorus luciferin + O(2) = oxidized Oplophorus luciferin + CO(2) +light.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.14','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.15','3,4-dihydroxyphenylalanine oxidative deaminase','DOPA oxidative deaminase','NULL','One of the three enzymes involved in L-dopa (3,4-dihydroxyphenylalanine) catabolism in the non-oxygenic phototrophicbacterium Rubrivivax benzoatilyticus OU5 (and not Rhodobactersphaeroides OU5 as had been thought), the other two being EC 4.3.1.22and EC 2.6.1.49.In addition to DOPA, the enzyme can also use L-tyrosine,L-phenylalanine, L-tryptophan and glutamate as substrate, but moreslowly.Inhibited by NADH and 2-oxoglutarate.','2 L-dopa + O(2) = 2 3,4-dihydroxyphenylpyruvate + 2 NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.16','Nitronate monooxygenase','2-nitropropane dioxygenase','FMN.','Previously classified as EC 1.13.11.32, but it is now recognized thatthis was the result of the slow ionization of nitroalkanes to theirnitronate (anionic) forms.The enzymes from the fungus Neurospora crassa and the yeastWilliopsis saturnus var. mrakii (formerly classified as Hansenulamrakii) contain non-covalently bound FMN as the cofactor.Neither hydrogen peroxide nor superoxide were detected during enzymeturnover.Active toward linear alkyl nitronates of lengths between 2 and 6carbon atoms and, with lower activity, toward propyl-2-nitronate.The enzyme from N.crassa can also utilize neutral nitroalkanes,but with lower activity.Formerly EC 1.13.11.32.','Ethylnitronate + O(2) = acetaldehyde + nitrite + other products.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.17','Dichloroarcyriaflavin A synthase','NULL','NULL','The conversion of dichlorochromopyrrolate to dichloroarcyriaflavin Ais a complex process that involves two enzyme components.RebP is an NAD-dependent cytochrome P(450) oxygenase that performs anaryl-aryl bond formation yielding the six-ring indolocarbazolescaffold.Along with RebC, a flavin-dependent hydroxylase, it also catalyzesthe oxidative decarboxylation of both carboxyl groups.The presence of RebC ensures that the only product is therebeccamycin aglycone dichloroarcyriaflavin A.The enzymes are similar, but not identical, to StaP and StaC, whichare involved in the synthesis of staurosporine.','Dichlorochromopyrrolate + 4 O(2) + 4 NADH = dichloroarcyriaflavin A +2 CO(2) + 6 H(2)O + 4 NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.18','Dinoflagellate luciferase','NULL','NULL','A luciferase from dinoflagelates such as Gonyaulax polyhedra,Lingulodinium polyedrum, Noctiluca scintillans, and Pyrocystislunula.It is a single protein with three luciferase domains.The luciferin is strongly bound by a luciferin binding protein abovea pH of 7.','Dinoflagellate luciferin + O(2) = oxidized dinoflagellate luciferin +H(2)O + light.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.19','2-oxuglutarate dioxygenase (ethylene-forming)','Ethylene-forming enzyme','NULL','This is one of two simultaneous reactions catalyzed by the enzyme,which is responsible for ethylene production in bacteria of thePseudomonas syringae group.In the other reaction (EC 1.14.11.34) the enzyme catalyzes the mono-oxygenation of both 2-oxoglutarate and L-arginine, forming succinate,carbon dioxide and L-hydroxyarginine, which is subsequently cleavedinto guanidine and (S)-1-pyrroline-5-carboxylate.The enzymes catalyze two cycles of the ethylene-forming reaction foreach cycle of the succinate-forming reaction, so that thestoichiometry of the products ethylene and succinate is 2:1.','2-oxoglutarate + O(2) = ethylene + 3 CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.20','Noranthrone monooxygenase','Norsolorinate anthrone oxidase','NULL','Involved in the synthesis of aflatoxins in the fungus Aspergillusparasiticus.','Norsolorinic acid anthrone + O(2) = norsolorinic acid + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.12.21','Tetracenomycin-F1 monooxygenase','NULL','NULL','The enzyme is involved in biosynthesis of the anthracyclineantibiotic tetracenomycin C by the bacterium Streptomycesglaucescens.','Tetracenomycin F1 + O(2) = tetracenomycin D3 + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.99.1','Inositol oxygenase','Meso-inositol oxygenase','Iron.','Formerly EC 1.13.1.11 and EC 1.99.2.6.','Myo-inositol + O(2) = D-glucuronate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.99.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.99.3','Tryptophan 2-dioxygenase','Indole-3-alkane alpha-hydroxylase','Heme.','Acts on a number of indole-3-alkane derivatives, oxidizing the3-side-chain in the 2-position.Best substrates are L-tryptophan and 5-hydroxy-L-tryptophan.','L-tryptophan + O(2) = (indol-3-yl)glycolaldehyde + CO(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.99.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.13.99.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.9','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.10','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.1.11','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.2.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.2.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.3.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.1','Gamma-butyrobetaine dioxygenase','Gamma-BBH','Iron; L-ascorbate.','NULL','4-trimethylammoniobutanoate + 2-oxoglutarate + O(2) = 3-hydroxy-4-trimethylammoniobutanoate + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.2','Procollagen-proline dioxygenase','Procollagen-proline 4-dioxygenase','Iron; L-ascorbate.','NULL','L-proline-[procollagen] + 2-oxoglutarate + O(2) = trans-4-hydroxy-L-proline-[procollagen] + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.3','Pyrimidine-deoxynucleoside 2-dioxygenase','Deoxyuridine 2-dioxygenase','Iron; L-ascorbate.','Also acts on thymidine (cf. EC 1.14.11.10).','2-deoxyuridine + 2-oxoglutarate + O(2) = uridine + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.4','Procollagen-lysine 5-dioxygenase','Lysine hydroxylase','Iron; L-ascorbate.','NULL','L-lysine-[procollagen] + 2-oxoglutarate + O(2) = (2S,5R)-5-hydroxy-L-lysine-[procollagen] + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.6','Thymine dioxygenase','Thymine 7-hydroxylase','Iron; L-ascorbate.','Also acts on 5-hydroxymethyluracil to oxidize its -CH(2)-OH groupfirst to -CHO and then to -COOH.Formerly EC 1.14.11.5.','Thymine + 2-oxoglutarate + O(2) = 5-hydroxymethyluracil + succinate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.7','Procollagen-proline 3-dioxygenase','Procollagen-proline,2-oxoglutarate 3-dioxygenase','Iron; L-ascorbate.','NULL','L-proline-[procollagen] + 2-oxoglutarate + O(2) = trans-3-hydroxy-L-proline-[procollagen] + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.8','Trimethyllysine dioxygenase','Epsilon-trimethyllysine 2-oxoglutarate dioxygenase','Fe(2+); L-ascorbate.','NULL','N(6),N(6),N(6)-trimethyl-L-lysine + 2-oxoglutarate + O(2) = 3-hydroxy-N(6),N(6),N(6)-trimethyl-L-lysine + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.9','Flavanone 3-dioxygenase','(2S)-flavanone 3-hydroxylase','Iron; L-ascorbate.','NULL','A flavanone + 2-oxoglutarate + O(2) = a dihydroflavonol + succinate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.10','Pyrimidine-deoxynucleoside 1-dioxygenase','Deoxyuridine-uridine 1-dioxygenase','Iron; L-ascorbate.','Cf. EC 1.14.11.3.','2-deoxyuridine + 2-oxoglutarate + O(2) = uracil + 2-deoxyribonolactone +succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.11','Hyoscyamine (6S)-dioxygenase','Hyoscyamine (6S)-hydroxylase','Iron; L-ascorbate.','NULL','L-hyoscyamine + 2-oxoglutarate + O(2) = (6S)-hydroxyhyoscyamine +succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.12','Gibberellin-44 dioxygenase','Gibberellin A44 oxidase','Iron.','NULL','Gibberellin 44 + 2-oxoglutarate + O(2) = gibberellin 19 + succinate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.13','Gibberellin 2-beta-dioxygenase','Gibberellin 2-beta-hydroxylase','Iron.','Also acts on a number of other gibberellins.','Gibberellin 1 + 2-oxoglutarate + O(2) = 2-beta-hydroxygibberellin 1 +succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.14','6-beta-hydroxyhyoscyamine epoxidase','Hydroxyhyoscyamine dioxygenase','Iron; L-ascorbate.','NULL','(6S)-6-hydroxyhyoscyamine + 2-oxoglutarate + O(2) = scopolamine +succinate + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.15','Gibberellin 3-beta-dioxygenase','Gibberellin 3-beta-hydroxylase','Iron; L-ascorbate.','NULL','Gibberellin 20 + 2-oxoglutarate + O(2) = gibberellin 1 + succinate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.16','Peptide-aspartate beta-dioxygenase','Aspartate beta-hydroxylase','Iron.','Some vitamin K-dependent coagulation factors, as well as syntheticpeptides based on the structure of the first epidermal growth factordomain of human coagulation factor IX or X, can act as acceptors.','Peptide L-aspartate + 2-oxoglutarate + O(2) = peptide 3-hydroxy-L-aspartate + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.17','Taurine dioxygenase','2-aminoethanesulfonate dioxygenase','Iron; L-ascorbate.','The enzyme from Escherichia coli also acts on pentanesulfonate,3-(N-morpholino)propanesulfonate and 2-(1,3-dioxoisoindolin-2-yl)ethanesulfonate, but at lower rates.','Taurine + 2-oxoglutarate + O(2) = sulfite + aminoacetaldehyde + succinate+ CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.18','Phytanoyl-CoA dioxygenase','Phytanoyl-CoA 2 oxoglutarate dioxygenase','Iron; L-ascorbate.','Part of the peroxisomal phytanic acid alpha-oxidation pathway.','Phytanoyl-CoA + 2-oxoglutarate + O(2) = 2-hydroxyphytanoyl-CoA +succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.19','Leucocyanidin oxygenase','Anthocyanidin synthase','Iron; L-ascorbate.','Involved in the pathway by which many flowering plants makeanthocyanin (glycosylated anthocyandin) flower pigments.The intermediates are transformed into cis- and trans-dihydroquercetin, which the enzyme can also oxidize to quercetin.Acidification of the products gives anthocyanidin, which, however,may not be a natural precursor of the anthocyanins.','Leucocyanidin + 2-oxoglutarate + O(2) = cis- and trans-dihydroquercetins+ succinate + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.20','Deacetoxyvindoline 4-hydroxylase','D17H','Iron; L-ascorbate.','Also acts, to a lesser extent, on 16-methoxy-2,3-dihydrotabersonine.','Deacetoxyvindoline + 2-oxoglutarate + O(2) = deacetylvindoline +succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.21','Clavaminate synthase','Clavaminic acid synthase','Iron.','Catalyzes three separate oxidative reactions in the pathway for thebiosythesis of the beta-lactamase inhibitor clavulanate inStreptomyces clavuligerus.The first step (hydroxylation) is separated from the latter two(oxidative cyclization and desaturation) by the action ofEC 3.5.3.22.The three reactions are all catalyzed at the same nonheme iron site.','(1) Deoxyamidinoproclavaminate + 2-oxoglutarate + O(2) =amidinoproclavaminate + succinate + CO(2).(2) Proclavaminate + 2-oxoglutarate + O(2) = dihydroclavaminate +succinate + CO(2) + H(2)O.(3) Dihydroclavaminate + 2-oxoglutarate + O(2) = clavaminate + succinate+ CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.22','Flavone synthase','Flavone synthase I','Iron; L-ascorbate.','NULL','A flavanone + 2-oxoglutarate + O(2) = a flavone + succinate + CO(2) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.23','Flavonol synthase','Flavonoid 2-oxoglutarate-dependent dioxygenase','Iron.','In addition to the desaturation of (2R,3R)-dihydroflavonols toflavonols, the enzyme from the satsuma Citrus unshiu also has a non-specific activity that trans-hydroxylates the flavanones (2S)-naringenin and the unnatural (2R)-naringenin at C-3 to kaempferol and(2R,3R)-dihydrokaempferol, respectively.','A dihydroflavonol + 2-oxoglutarate + O(2) = a flavonol + succinate +CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.24','2-deoxymugineic-acid 2-dioxygenase','IDS3','Iron.','It is also likely that this enzyme can catalyze the hydroxylation of3-epihydroxy-2-deoxymugineic acid to form 3-epihydroxymugineic acid.','2-deoxymugineic acid + 2-oxoglutarate + O(2) = mugineic acid + succinate+ CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.25','Mugineic-acid 3-dioxygenase','IDS2','Fe(2+).','NULL','(1) Mugineic acid + 2-oxoglutarate + O(2) = 3-epihydroxymugineic acid +succinate + CO(2).(2) 2-deoxymugineic acid + 2-oxoglutarate + O(2) = 3-epihydroxy-2-deoxymugineic acid + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.26','Deacetoxycephalosporin-C hydroxylase','3-methylcephem hydroxylase','Iron.','The enzyme can also use 3-exomethylenecephalosporin C as a substrateto form deacetoxycephalosporin C, although more slowly.In Acremonium chrysogenum, the enzyme forms part of a bifunctionalprotein along with EC 1.14.20.1.It is a separate enzyme in Streptomyces clavuligerus.','Deacetoxycephalosporin C + 2-oxoglutarate + O(2) = deacetylcephalosporinC + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.27','[Histone H3]-lysine-36 demethylase','H3-K36-specific demethylase','Fe(2+).','Of the seven potential methylation sites in histones H3 (K4, K9, K27,K36, K79) and H4 (K20, R3) from HeLa cells, the enzyme is specificfor Lys-36.Lysine residues exist in three methylation states (mono-, di- andtrimethylated).The enzyme preferentially demethylates the dimethyl form of Lys-36(K36me2), which is its natural substrate, to form the monomethyl andunmethylated forms of Lys-36.It can also demethylate the monomethyl- but not the trimethyl form ofLys-36.','(1) Protein N(6),N(6)-dimethyl-L-lysine + 2-oxoglutarate + O(2) = proteinN(6)-methyl-L-lysine + succinate + formaldehyde + CO(2).(2) Protein N(6)-methyl-L-lysine + 2-oxoglutarate + O(2) = proteinL-lysine + succinate + formaldehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.28','Proline 3-hydroxylase','P-3-H','Fe(2+).','Unlike the proline hydroxylases involved in collagen biosynthesis(EC 1.14.11.2 and EC 1.14.11.7), this enzyme does not requireascorbate for activity although it does increase the activity of theenzyme.The enzyme is specific for L-proline as D-proline, trans-4-hydroxy-L-proline, cis-4-hydroxy-L-proline and 3,4-dehydro-DL-proline are notsubstrates.','L-proline + 2-oxoglutarate + O(2) = cis-3-hydroxy-L-proline + succinate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.29','Hypoxia-inducible factor-proline dioxygenase','HIF hydroxylase','Iron; L-ascorbate.','Contains iron, and requires ascorbate.Specifically hydroxylates a proline residue in HIF-alpha, the alphasubunit of the transcriptional regulator HIF (hypoxia-induciblefactor), which targets HIF for proteasomal destruction.The requirement of oxygen for the hydroxylation reaction enablesanimals to respond to hypoxia.','Hypoxia-inducible factor-L-proline + 2-oxoglutarate + O(2) = hypoxia-inducible factor-trans-4-hydroxy-L-proline + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.30','Hypoxia-inducible factor-asparagine dioxygenase','HIF hydroxylase','Iron; L-ascorbate.','Contains iron, and requires ascorbate.Catalyzes hydroxylation of an asparagine in the C-terminaltranscriptional activation domain of HIF-alpha, the alpha subunit ofthe transcriptional regulator HIF (hypoxia-inducible factor), whichreduces its interaction with the transcriptional coactivator proteinp300.The requirement of oxygen for the hydroxylation reaction enablesanimals to respond to hypoxia.','Hypoxia-inducible factor-L-asparagine + 2-oxoglutarate + O(2) = hypoxia-inducible factor-(3S)-3-hydroxy-L-asparagine + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.31','Thebaine 6-O-demethylase','NULL','Iron.','Requires Fe(2+).Catalyzes a step in morphine biosynthesis.The product neopinione spontaneously rearranges to the more stablecodeinone.The enzyme also catalyzes the 6-O-demethylation of oripavine tomorphinone, with lower efficiency.','Thebaine + 2-oxoglutarate + O(2) = neopinone + formaldehyde + succinate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.32','Codeine 3-O-demethylase','Codeine O-demethylase','Iron.','Requires Fe(2+).Catalyzes a step in morphine biosynthesis.The enzyme also catalyzes the 3-O-demethylation of thebaine tooripavine, with lower efficiency.','Codeine + 2-oxoglutarate + O(2) = morphine + formaldehyde + succinate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.33','DNA oxidative demethylase','Alkylated DNA repair protein','Iron.','Activity is slightly stimulated by ascorbate.Catalyzes oxidative demethylation of the DNA base lesions N(1)-methyladenine, N(3)-methylcytosine, N(1)-methylguanine, and N(3)-methylthymine.It works better on single-stranded DNA (ssDNA) and is capable ofrepairing damaged bases in RNA.','DNA-base-CH(3) + 2-oxoglutarate + O(2) = DNA-base + formaldehyde +succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.34','2-oxoglutarate/L-arginine monooxygenase/decarboxylase (succinate-forming)','Ethylene-forming enzyme','NULL','This is one of two simultaneous reactions catalyzed by the enzyme,which is responsible for ethylene production in bacteria of thePseudomonas syringae group.In the other reaction (EC 1.13.12.19) the enzyme catalyzes thedioxygenation of 2-oxoglutarate forming ethylene and three moleculesof carbon dioxide.The enzyme catalyzes two cycles of the ethylene-forming reaction foreach cycle of the succinate-forming reaction, so that thestoichiometry of the products ethylene and succinate is 2:1.','2-oxoglutarate + L-arginine + O(2) = succinate + CO(2) + guanidine + (S)-1-pyrroline-5-carboxylate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.35','1-deoxypentalenic acid 11-beta-hydroxylase','NULL','Iron; L-ascorbate.','Isolated from the bacterium Streptomyces avermitilis.Part of the pathway for pentalenolactone biosynthesis.','1-deoxypentalenate + 2-oxoglutarate + O(2) = 1-deoxy-11-beta-hydroxypentalenate + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.36','Pentalenolactone F synthase','NULL','Iron; L-ascorbate.','Isolated from the bacteria Streptomyces exfoliatus, Streptomycesarenae and Streptomyces avermitilis.Part of the pentalenolactone biosynthesis pathway.','Pentalenolactone D + 2 2-oxoglutarate + 2 O(2) = pentalenolactone F +2 succinate + 2 CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.37','Kanamycin B dioxygenase','NULL','Ascorbate; Fe(2+).','Found in the bacterium Streptomyces kanamyceticus where it isinvolved in the conversion of the aminoglycoside antibiotic kanamycinB to kanamycin A.','Kanamycin B + 2-oxoglutarate + O(2) = 2-dehydrokanamycin A + succinate +NH(3) + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.38','Verruculogen synthase','NULL','Fe(2+).','Found in the fungus Aspergillus fumigatus.Both atoms of a dioxygen molecule are incorporated into verruculogen.Involved in the biosynthetic pathways of several indole alkaloidssuch as fumitremorgin A.','Fumitremorgin B + 2-oxoglutarate + 2 O(2) + AH(2) = verruculogen +succinate + CO(2) + H(2)O + A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.39','L-asparagine oxygenase','L-asparagine 3-hydroxylase','Fe(2+).','The enzyme is only able to hydroxylate free L-asparagine.It is not active toward D-asparagine.The beta-hydroxylated asparagine produced is incorporated at position9 of the calcium-dependent antibiotic (CDA), an 11-residue non-ribosomally synthesized acidic lipopeptide lactone.Formerly EC 1.14.11.n1.','L-asparagine + 2-oxoglutarate + O(2) = (2S,3S)-3-hydroxyasparagine +succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.40','Enduracididine beta-hydroxylase','NULL','NULL','The enzyme is involved in biosynthesis of the nonproteinogenic aminoacid beta-hydroxyenduracididine, a component of the mannopeptimycins(cyclic glycopeptide antibiotic), produced by Streptomyceshygroscopicus NRRL 30439.','L-enduracididine + 2-oxoglutarate + O(2) = (3S)-3-hydroxy-L-enduracididine + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.41','L-arginine hydroxylase','NULL','Fe(2+).','The enzyme is involved in the biosynthesis of the cyclic pentapeptideantibiotic viomycin.It differs from EC 1.14.11.34, because it does not form guanidine and(S)-1-pyrroline-5-carboxylate from 3-hydroxy-L-arginine.','L-arginine + 2-oxoglutarate + O(2) = (3S)-3-hydroxy-L-arginine +succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.42','tRNA(Phe) (7-(3-amino-3-carboxypropyl)wyosine(37)-C(2))-hydroxylase','NULL','Fe(2+).','The enzyme is not active with wybutosine.','7-(3-amino-3-carboxypropyl)wyosine(37) in tRNA(Phe) + 2-oxoglutarate +O(2) = 7-(2-hydroxy-3-amino-3-carboxypropyl)wyosine(37) in tRNA(Phe) +succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.43','(S)-dichlorprop dioxygenase (2-oxoglutarate)','(S)-mecoprop dioxygenase','Fe(2+).','The enzymes from the Gram-negative bacteria Delftia acidovorans MC1and Sphingomonas herbicidovorans MH are involved in the degradationof the (S)-enantiomer of the phenoxyalkanoic acid herbicides mecopropand dichlorprop.','(1) (S)-2-(4-chloro-2-methylphenoxy)propanoate + 2-oxoglutarate + O(2) =4-chloro-2-methylphenol + pyruvate + succinate + CO(2).(2) (S)-(2,4-dichlorophenoxy)propanoate + 2-oxoglutarate + O(2) = 2,4-dichlorophenol + pyruvate + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.44','(R)-dichlorprop dioxygenase (2-oxoglutarate)','(R)-mecoprop dioxygenase','Fe(2+).','The enzymes from the Gram-negative bacteria Delftia acidovorans MC1and Sphingomonas herbicidovorans MH are involved in the degradationof the (R)-enantiomer of the phenoxyalkanoic acid herbicides mecopropand dichlorprop.','(1) (R)-2-(4-chloro-2-methylphenoxy)propanoate + 2-oxoglutarate + O(2) =4-chloro-2-methylphenol + pyruvate + succinate + CO(2).(2) (R)-(2,4-dichlorophenoxy)propanoate + 2-oxoglutarate + O(2) = 2,4-dichlorophenol + pyruvate + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.n2','Methylcytosine dioxygenase','NULL','NULL','Dioxygenase that catalyzes the conversion of the modified genomicbase 5-methylcytosine (5mC) into 5-hydroxymethylcytosine (5hmC).Might initiate a process leading to cytosine demethylation throughdeamination into 5-hydroxymethyluracil (5hmU) and subsequentreplacement by unmethylated cytosine by the base excision repairsystem.Methylation at the C5 position of cytosine bases is an epigeneticmodification of the mammalian genome which plays an important role intranscriptional regulation.','DNA 5-methylcytosine + 2-oxoglutarate + O(2) =DNA 5-hydroxymethylcytosine + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.n3','L-proline cis-4-hydroxylase','NULL','NULL','NULL','L-proline + 2-oxoglutarate + O(2) = cis-4-hydroxy-L-proline + succinate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.11.n4','Ankyrin-repeat-histidine dioxagenase','NULL','NULL','NULL','Ankyrin-repeat-L-histidine + 2-oxoglutarate + O(2) = ankyrin-repeat-(3S)-3-hydroxy-L-histidine + succinate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.1','Anthranilate 1,2-dioxygenase (deaminating, decarboxylating)','Anthranilate hydroxylase','Iron.','NULL','Anthranilate + NAD(P)H + O(2) = catechol + CO(2) + NAD(P)(+) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.3','Benzene 1,2-dioxygenase','Benzene hydroxylase','FAD; Iron; Iron-sulfur.','A system, containing a reductase which is an iron-sulfur flavoprotein(FAD), an iron-sulfur oxygenase and ferredoxin.','Benzene + NADH + O(2) = cis-cyclohexa-3,5-diene-1,2-diol + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.4','3-hydroxy-2-methylpyridinecarboxylate dioxygenase','Methylhydroxypyridine carboxylate dioxygenase','FAD.','NULL','3-hydroxy-2-methylpyridine-5-carboxylate + NAD(P)H + O(2) =2-(acetamidomethylene)succinate + NAD(P)(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.5','5-pyridoxate dioxygenase','5-pyridoxate oxidase','Flavoprotein.','NULL','3-hydroxy-4-hydroxymethyl-2-methylpyridine-5-carboxylate + NADPH + O(2) =2-(acetamidomethylene)-3-(hydroxymethyl)succinate + NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.7','Phthalate 4,5-dioxygenase','PDO','FMN; Iron; Iron-sulfur.','A system, containing a reductase which is an iron-sulfur flavoprotein(FMN), an iron-sulfur oxygenase, and no independent ferredoxin.','Phthalate + NADH + O(2) = cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,2-dicarboxylate + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.8','4-sulfobenzoate 3,4-dioxygenase','NULL','FMN; Iron; Iron-sulfur.','A system, containing a reductase which is an iron-sulfur flavoprotein(FMN), an iron-sulfur oxygenase, and no independent ferredoxin.','4-sulfobenzoate + NADH + O(2) = 3,4-dihydroxybenzoate + sulfite + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.9','4-chlorophenylacetate 3,4-dioxygenase','NULL','Iron.','A system, containing a reductase and an iron-sulfur oxygenase, and noindependent ferredoxin.Also acts on 4-bromophenyl acetate.Formerly EC 1.13.99.4.','4-chlorophenylacetate + NADH + O(2) = 3,4-dihydroxyphenylacetate +chloride + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.10','Benzoate 1,2-dioxygenase','Benzoate hydroxylase','Fe(2+).','A system, containing a reductase which is an iron-sulfur flavoprotein(FAD), and an iron-sulfur oxygenase.Formerly EC 1.13.99.2.','Benzoate + NADH + O(2) = (1R,6S)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.11','Toluene dioxygenase','Toluene 1,2-dioxygenase','NULL','A system, containing a reductase which is an iron-sulfur flavoprotein(FAD), an iron-sulfur oxygenase, and a ferredoxin.Some other aromatic compounds, including ethylbenzene, 4-xylene andsome halogenated toluenes, are converted into the corresponding cis-dihydrodiols.','Toluene + NADH + O(2) = (1S,2R)-3-methylcyclohexa-3,5-diene-1,2-diol +NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.12','Naphthalene 1,2-dioxygenase','NULL','Iron.','Comprises a multicomponent system, containing a reductase that is aniron-sulfur flavoprotein (FAD; EC 1.18.1.3), an iron-sulfuroxygenase, and ferredoxin.','Naphthalene + NADH + O(2) = (1R,2S)-1,2-dihydronaphthalene-1,2-diol +NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.13','2-halobenzoate 1,2-dioxygenase','2-chlorobenzoate 1,2-dioxygenase','Iron.','A multicomponent enzyme system composed of a dioxygenase componentand an electron transfer component.The latter contains FAD.The enzyme, characterized from the bacterium Burkholderia cepacia2CBS, has a broad substrate specificity.Substrates include 2-fluorobenzoate, 2-chlorobenzoate,2-bromobenzoate, and 2-iodobenzoate, which are processed in thisorder of preference.','A 2-halobenzoate + NADH + O(2) = catechol + a halide anion + NAD(+) +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.14','2-aminobenzenesulfonate 2,3-dioxygenase','2-aminosulfobenzene 2,3-dioxygenase','NULL','NULL','2-aminobenzenesulfonate + NADH + O(2) = 2,3-dihydroxybenzenesulfonate +NH(3) + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.15','Terephthalate 1,2-dioxygenase','1,4-dicarboxybenzoate 1,2-dioxygenase','NULL','In Comamonas testoteroni, contains a Rieske [2Fe-2S] center.','Terephthalate + NADH + O(2) = (1R,6S)-dihydroxycyclohexa-2,4-diene-1,4-dicarboxylate + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.16','2-hydroxyquinoline 5,6-dioxygenase','2-oxo-1,2-dihydroquinoline 5,6-dioxygenase','NULL','3-methylquinolin-2-ol, quinolin-8-ol and quinolin-2,8-diol are alsosubstrates.Quinolin-2-ols exist largely as their quinolin-2(1H)-one tautomers.','Quinolin-2-ol + NADH + O(2) = 2,5,6-trihydroxy-5,6-dihydroquinoline +NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.17','Nitric oxide dioxygenase','NOD','FAD; Heme.','It has been proposed that FAD functions as the electron carrier fromNADPH to the ferric heme prosthetic group.','2 nitric oxide + 2 O(2) + NAD(P)H = 2 nitrate + NAD(P)(+) + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.18','Biphenyl 2,3-dioxygenase','Biphenyl dioxygenase','Iron.','The enzyme from Burkholderia fungorum strain LB400 (previouslyPseudomonas sp.) is part of a multicomponent system composed of anNADH:ferredoxin oxidoreductase (FAD cofactor), a [2Fe-2S] Rieske-typeferredoxin, and a terminal oxygenase that contains a [2Fe-2S] Rieske-type iron-sulfur cluster and a catalytic mononuclear nonheme ironcenter.Chlorine-substituted biphenyls can also act as substrates.Similar to the three-component enzyme systems EC 1.14.12.3 andEC 1.14.12.11.','Biphenyl + NADH + O(2) = (1S,2R)-3-phenylcyclohexa-3,5-diene-1,2-diol +NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.19','3-phenylpropanoate dioxygenase','3-phenylpropionate dioxygenase','Iron.','This enzyme catalyzes a step in the pathway of phenylpropanoidcompounds degradation.It catalyzes the insertion of both atoms of molecular oxygen intopositions 2 and 3 of the phenyl ring of 3-phenylpropanoate or (2E)-3-phenylprop-2-enoate.','(1) 3-phenylpropanoate + NADH + O(2) = 3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)propanoate + NAD(+).(2) (2E)-3-phenylprop-2-enoate + NADH + O(2) = (2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.20','Pheophorbide a oxygenase','PaO','NULL','Catalyzes a key reaction in chlorophyll degradation, which occursduring leaf senescence and fruit ripening in higher plants.Requires reduced ferredoxin, which is generated either by NADPHthrough the pentose-phosphate pathway or by the action of photosystemI.While still attached to this enzyme, the product is rapidly convertedinto primary fluorescent chlorophyll catabolite by the action ofEC 1.3.1.80.Pheophorbide b acts as an inhibitor.In (18)O(2) labelling experiments, only the aldehyde oxygen islabelled, suggesting that the other oxygen atom may originate fromH(2)O.','Pheophorbide a + NADPH + O(2) = red chlorophyll catabolite + NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.21','Benzoyl-CoA 2,3-dioxygenase','Benzoyl-CoA dioxygenase/reductase','FAD; Iron; Iron-sulfur.','The enzyme is involved in aerobic benzoate metabolism in Azoarcusevansii.BoxB functions as the oxygenase part of benzoyl-CoA oxygenase inconjunction with BoxA, the reductase component, which upon binding ofbenzoyl-CoA, transfers two electrons to the ring in the course ofdioxygenation.Formerly EC 1.14.12.n1.','Benzoyl-CoA + NADPH + O(2) = 2,3-dihydro-2,3-dihydroxybenzoyl-CoA +NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.22','Carbazole 1,9a-dioxygenase','CARDO','Iron; Iron-sulfur.','This enzyme catalyzes the first reaction in the pathway of carbazoledegradation.The enzyme attacks at the 1 and 9a positions of carbazole, resultingin the formation of a highly unstable hemiaminal intermediate thatundergoes a spontaneous cleavage and rearomatization, resulting in2-aminobiphenyl-2,3-diol.In most bacteria the enzyme is a complex composed of a terminaloxygenase, a ferredoxin, and a ferredoxin reductase.The terminal oxygenase component contains a nonheme iron center and aRieske [2Fe-2S] iron-sulfur cluster.','9H-carbazole + NAD(P)H + O(2) = 2-aminobiphenyl-2,3-diol + NAD(P)(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.12.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.1','Salicylate 1-monooxygenase','Salicylate hydroxylase','FAD.','NULL','Salicylate + NADH + O(2) = catechol + NAD(+) + H(2)O + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.2','4-hydroxybenzoate 3-monooxygenase','p-hydroxybenzoate hydroxylase','FAD.','Most enzymes from Pseudomonas are highly specific for NAD(P)H (cf.EC 1.14.13.33).','4-hydroxybenzoate + NADPH + O(2) = protocatechuate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.4','Melilotate 3-monooxygenase','2-hydroxyphenylpropionate hydroxylase','FAD.','NULL','3-(2-hydroxyphenyl)propanoate + NADH + O(2) = 3-(2,3-dihydroxyphenyl)propanoate + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.5','Imidazoleacetate 4-monooxygenase','Imidazoleacetate hydroxylase','FAD.','Formerly EC 1.14.1.5.','4-imidazoleacetate + NADH + O(2) = 5-hydroxy-4-imidazoleacetate + NAD(+)+ H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.6','Orcinol 2-monooxygenase','Orcinol hydroxylase','FAD.','NULL','Orcinol + NADH + O(2) = 2,3,5-trihydroxytoluene + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.7','Phenol 2-monooxygenase','Phenol hydroxylase','FAD.','Also active with resorcinol and 2-methylphenol.','Phenol + NADPH + O(2) = catechol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.8','Flavin-containing monooxygenase','Dimethylaniline monooxygenase (N-oxide-forming)','FAD.','A broad spectrum monooxygenase that accepts substrates as diverse ashydrazines, phosphines, boron-containing compounds, sulfides,selenides, iodide, as well as primary, secondary and tertiary amines.Is distinct from other monooxygenases in that the enzyme forms arelatively stable hydroperoxy flavin intermediate.Generally converts nucleophilic heteroatom-containing chemicals anddrugs into harmless, readily excreted metabolites.For example, N-oxygenation is largely responsible for thedetoxification of the dopaminergic neurotoxin 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine (MPTP).Formerly EC 1.13.12.11.','N,N-dimethylaniline + NADPH + O(2) = N,N-dimethylaniline N-oxide +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.9','Kynurenine 3-monooxygenase','Kynurenine 3-hydroxylase','FAD.','Formerly EC 1.14.1.2 and EC 1.99.1.5.','L-kynurenine + NADPH + O(2) = 3-hydroxy-L-kynurenine + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.10','2,6-dihydroxypyridine 3-monooxygenase','NULL','Flavoprotein.','NULL','2,6-dihydroxypyridine + NADH + O(2) = 2,3,6-trihydroxypyridine + NAD(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.11','Trans-cinnamate 4-monooxygenase','CA4H','Heme-thiolate.','Also acts on NADH, more slowly.','Trans-cinnamate + NADPH + O(2) = 4-hydroxycinnamate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.12','Benzoate 4-monooxygenase','Benzoate 4-hydroxylase','5,6,7,8-tetrahydropteridine; Iron.','NULL','Benzoate + NADPH + O(2) = 4-hydroxybenzoate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.13','Calcidiol 1-monooxygenase','25-hydroxycholecalciferol 1-monooxygenase','Heme-thiolate.','NULL','Calcidiol + NADPH + O(2) = calcitriol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.14','Trans-cinnamate 2-monooxygenase','Cinnamic 2-hydroxylase','NULL','NULL','Trans-cinnamate + NADPH + O(2) = 2-hydroxycinnamate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.15','Cholestanetriol 26-monooxygenase','5-beta-cholestane-3-alpha,7-alpha,12-alpha-triol 26-hydroxylase','Heme-thiolate.','The enzyme requires ferredoxin and ferredoxin reductase.It catalyzes the first three sterol side chain oxidations in bileacid biosynthesis via the neutral (classic) pathway.Can also act on cholesterol, cholest-5-en-3-beta,7-alpha-diol,7-alpha-hydroxycholest-4-en-3-one, and 5-beta-cholestane-3-alpha,7-alpha-diol.The enzyme can also hydroxylate cholesterol at positions 24 and 25.','5-beta-cholestane-3-alpha,7-alpha,12-alpha-triol + 3 NADPH + 3 O(2) =(25R)-3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestan-26-oate +3 NADP(+) + 4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.16','Cyclopentanone monooxygenase','Cyclopentanone 1,2-monooxygenase','NULL','NULL','Cyclopentanone + NADPH + O(2) = 5-valerolactone + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.17','Cholesterol 7-alpha-monooxygenase','Cholesterol 7-alpha-hydroxylase','Heme-thiolate.','NULL','Cholesterol + NADPH + O(2) = 7-alpha-hydroxycholesterol + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.18','4-hydroxyphenylacetate 1-monooxygenase','4-HPA 1-hydroxylase','FAD.','Also acts on 4-hydroxyhydratropate forming 2-methylhomogentisate andon 4-hydroxyphenoxyacetate forming hydroquinone and glycolate.','4-hydroxyphenylacetate + NAD(P)H + O(2) = homogentisate + NAD(P)(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.19','Taxifolin 8-monooxygenase','NULL','Flavoprotein.','Converts a flavanol into a flavanone.Also acts on fustin, but not on catechin, quercetin ormollisacacidin.','Taxifolin + NAD(P)H + O(2) = 2,3-dihydrogossypetin + NAD(P)(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.20','2,4-dichlorophenol 6-monooxygenase','2,4-dichlorophenol hydroxylase','FAD.','Also acts, more slowly, on 4-chlorophenol and 4-chloro-2-methylphenol.NADH can act instead of NADPH, but more slowly.','2,4-dichlorophenol + NADPH + O(2) = 3,5-dichlorocatechol + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.21','Flavonoid 3-monooxygenase','Flavonoid 3-hydroxylase','NULL','Acts on a number of flavonoids, including naringenin anddihydrokaempferol.Does not act on 4-coumarate or 4-coumaroyl-CoA.','A flavonoid + NADPH + O(2) = a 3-hydroxyflavonoid + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.22','Cyclohexanone monooxygenase','Cyclohexanone 1,2-monooxygenase','FAD.','In the catalytic mechanism of this enzyme, the nucleophilic speciesthat attacks the carbonyl group is a peroxyflavin intermediate thatis generated by reaction of the enzyme-bound flavin cofactor withNAD(P)H and oxygen.This enzyme is able to catalyze a wide range of oxidative reactions,including enantioselective Baeyer-Villiger reactions, sulfoxidations,amine oxidations and epoxidations.','Cyclohexanone + NADPH + O(2) = hexano-6-lactone + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.23','3-hydroxybenzoate 4-monooxygenase','3-hydroxybenzoate 4-hydroxylase','FAD.','Also acts on a number of analogs of 3-hydroxybenzoate substituted inthe 2, 4, 5 and 6 positions.Formerly EC 1.14.99.13.','3-hydroxybenzoate + NADPH + O(2) = 3,4-dihydroxybenzoate + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.24','3-hydroxybenzoate 6-monooxygenase','3-hydroxybenzoate 6-hydroxylase','FAD.','Also acts on a number of analogs of 3-hydroxybenzoate substituted inthe 2, 4, 5 and 6 positions.NADPH can act instead of NADH, more slowly.','3-hydroxybenzoate + NADH + O(2) = 2,5-dihydroxybenzoate + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.25','Methane monooxygenase (soluble)','Methane hydroxylase','NULL','The enzyme is soluble, in contrast to the particulate enzyme,EC 1.14.18.3.Broad specificity; many alkanes can be hydroxylated, and alkenes areconverted into the corresponding epoxides; CO is oxidized to CO(2),ammonia is oxidized to hydroxylamine, and some aromatic compounds andcyclic alkanes can also be hydroxylated, but more slowly.','Methane + NAD(P)H + O(2) = methanol + NAD(P)(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.26','Phosphatidylcholine 12-monooxygenase','Oleate Delta(12)-hydroxylase','NULL','NULL','1-acyl-2-oleoyl-sn-glycero-3-phosphocholine + NADH + O(2) = 1-acyl-2-((S)-12-hydroxyoleoyl)-sn-glycero-3-phosphocholine + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.27','4-aminobenzoate 1-monooxygenase','4-aminobenzoate dehydrogenase','FAD.','Acts on anthranilate and 4-aminosalicylate but not on salicylate (cf.EC 1.14.13.1).','4-aminobenzoate + NAD(P)H + O(2) = 4-hydroxyaniline + NAD(P)(+) + H(2)O +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.28','3,9-dihydroxypterocarpan 6A-monooxygenase','3,9-dihydroxypterocarpan 6A-hydroxylase','Heme-thiolate.','The product of the reaction is the biosynthetic precursor of thephytoalexin glyceollin in soybean.','(6aR,11aR)-3,9-dihydroxypterocarpan + NADPH + O(2) = (6aS,11aS)-3,6a,9-trihydroxypterocarpan + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.29','4-nitrophenol 2-monooxygenase','4-nitrophenol hydroxylase','FAD.','NULL','4-nitrophenol + NADH + O(2) = 4-nitrocatechol + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.30','Leukotriene-B(4) 20-monooxygenase','Leukotriene-B(4) 20-hydroxylase','Heme-thiolate.','Not identical with EC 1.14.13.34.','(6Z,8E,10E,14Z)-(5S,12R)-5,12-dihydroxyicosa-6,8,10,14-tetraenoate +NADPH + O(2) = (6Z,8E,10E,14Z)-(5S,12R)-5,12,20-trihydroxyicosa-6,8,10,14-tetraenoate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.31','2-nitrophenol 2-monooxygenase','Nitrophenol oxygenase','NULL','Involved in the metabolism of nitro-aromatic compounds by a strain ofPseudomonas putida.','2-nitrophenol + NADPH + O(2) = catechol + nitrite + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.32','Albendazole monooxygenase','Albendazole sulfoxidase','FAD.','NULL','Albendazole + NADPH + O(2) = albendazole S-oxide + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.33','4-hydroxybenzoate 3-monooxygenase (NAD(P)H)','4-hydroxybenzoate 3-hydroxylase','FAD.','The enzyme from Corynebacterium cyclohexanicum is highly specific for4-hydroxybenzoate, but uses NADH and NADPH at approximately equalrates (cf. EC 1.14.13.2).It is less specific for NADPH than EC 1.14.13.2.','4-hydroxybenzoate + NAD(P)H + O(2) = 3,4-dihydroxybenzoate + NAD(P)(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.34','Leukotriene-E(4) 20-monooxygenase','Leukotriene-E(4) omega-hydroxylase','NULL','Also acts on N-acetyl-leukotriene E(4), more slowly.Not identical with EC 1.14.13.30.','(7E,9E,11Z,14Z)-(5S,6R)-6-(cystein-S-yl)-5-hydroxyicosa-7,9,11,14-tetraenoate + NADPH + O(2) = 20-hydroxyleukotriene E(4) + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.35','Anthranilate 3-monooxygenase (deaminating)','Anthranilate 2,3-hydroxylase (deaminating)','NULL','The enzyme from Aspergillus niger is an iron protein; that from theyeast Trichosporon cutaneum is a flavoprotein (FAD).Formerly EC 1.14.12.2.','Anthranilate + NADPH + O(2) = 2,3-dihydroxybenzoate + NADP(+) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.36','5-O-(4-coumaroyl)-D-quinate 3-monooxygenase','5-O-(4-coumaroyl)-D-quinate/shikimate 3-hydroxylase','NULL','Also acts on trans-5-O-(4-coumaroyl)shikimate.','Trans-5-O-(4-coumaroyl)-D-quinate + NADPH + O(2) = trans-5-O-caffeoyl-D-quinate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.37','Methyltetrahydroprotoberberine 14-monooxygenase','(S)-cis-N-methyltetrahydroprotoberberine-14-hydroxylase','Heme-thiolate.','NULL','(S)-N-methylcanadine + NADPH + O(2) = allocryptopine + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.38','Anhydrotetracycline monooxygenase','Anhydrotetracycline oxygenase','NULL','Involved in the biosynthesis of tetracyclines in Streptomyces sp.','Anhydrotetracycline + NADPH + O(2) = 12-dehydrotetracycline + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.39','Nitric-oxide synthase (NADPH dependent)','Endothelium-derived relaxation factor-forming enzyme','NULL','This eukaryotic enzyme, which is found in plants and animals,consists of oxygenase and reductase domains that are linked via aregulatory calmodulin-binding domain.Upon calcium-induced calmodulin binding, the reductase and oxygenasedomains form a complex, allowing electrons to flow from NADPH via FADand FMN to the active center.May produce superoxide under certain conditions.Cf. EC 1.14.13.165.','2 L-arginine + 3 NADPH + 4 O(2) = 2 L-citrulline + 2 nitric oxide +3 NADP(+) + 4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.40','Anthraniloyl-CoA monooxygenase','2-aminobenzoyl-CoA monooxygenase/reductase','FAD.','The non-aromatic product is unstable and releases CO(2) and NH(3),forming 1,4-cyclohexanedione.','Anthraniloyl-CoA + 2 NAD(P)H + O(2) = 2-amino-5-oxocyclohex-1-enecarboxyl-CoA + H(2)O + 2 NAD(P)(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.41','Tyrosine N-monooxygenase','CYP79A1','Heme-thiolate.','Involved in the biosynthesis of the cyanogenic glucoside dhurrin insorghum, along with EC 1.14.13.68 and EC 2.4.1.85.Some 2-(4-hydroxyphenyl)-1-nitroethane is formed as a side product.','L-tyrosine + 2 O(2) + 2 NADPH = (Z)-(4-hydroxyphenylacetaldehyde oxime) +2 NADP(+) + CO(2) + 3 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.42','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.43','Questin monooxygenase','Questin oxygenase','NULL','The enzyme cleaves the anthraquinone ring of questin to form abenzophenone.Involved in the biosynthesis of the seco-anthraquinone (+)-geodin.','Questin + NADPH + O(2) = demethylsulochrin + NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.44','2-hydroxybiphenyl 3-monooxygenase','NULL','NULL','Also converts 2,2-dihydroxybiphenyl into 2,2,3-trihydroxy-biphenyl.','2-hydroxybiphenyl + NADH + O(2) = 2,3-dihydroxybiphenyl + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.45','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.46','(-)-menthol monooxygenase','NULL','NULL','NULL','(-)-menthol + NADPH + O(2) = p-menthane-3,8-diol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.47','(S)-limonene 3-monooxygenase','(-)-limonene 3-hydroxylase','Heme-thiolate.','High specificity, but NADH can act instead of NADPH, although moreslowly.','(-)-(S)-limonene + NADPH + O(2) = (-)-trans-isopiperitenol + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.48','(S)-limonene 6-monooxygenase','(-)-limonene 6-hydroxylase','Heme-thiolate.','High specificity, but NADH can act instead of NADPH, although moreslowly.','(-)-(S)-limonene + NADPH + O(2) = (-)-trans-carveol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.49','(S)-limonene 7-monooxygenase','(-)-limonene 7-monooxygenase','Heme-thiolate.','High specificity, but NADH can act instead of NADPH, although moreslowly.','(-)-(S)-limonene + NADPH + O(2) = (-)-perillyl alcohol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.50','Pentachlorophenol monooxygenase','PCB 4-monooxygenase','FAD.','The enzyme displaces a diverse range of substituents from the4-position of polyhalogenated phenols but requires that a halogensubstituent be present at the 2-position.The enzyme converts many polyhalogenated phenols into hydroquinones,and requires that a halogen substituent be present at C-2.If C-4 carries a halogen substituent, reaction 1 is catalyzed, e.g.2,4,6-triiodophenol is oxidized to 2,6-diiodohydroquinone; if C-4 isunsubstituted, reaction 2 is catalyzed.','(1) Pentachlorophenol + 2 NADPH + O(2) = 2,3,5,6-tetrachlorohydroquinone+ 2 NADP(+) + chloride + H(2)O.(2) 2,3,5,6-tetrachlorophenol + NADPH + O(2) = 2,3,5,6-tetrachlorohydroquinone + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.51','6-oxocineole dehydrogenase','NULL','NULL','The product undergoes non-enzymic cleavage and subsequent ringclosure to form the lactone 4,5-dihydro-5,5-dimethyl-4-(3-oxobutyl)furan-2(3H)-one.','6-oxocineole + NADPH + O(2) = 1,6,6-trimethyl-2,7-dioxabicyclo[3.2.2]nonan-3-one + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.52','Isoflavone 3-hydroxylase','NULL','Heme-thiolate.','Also acts on biochanin A and other isoflavones with a 4-methoxygroup.Involved in the biosynthesis of the pterocarpin phytoalexinsmedicarpin and maackiain.','Formononetin + NADPH + O(2) = calycosin + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.53','4-methoxyisoflavone 2-hydroxylase','Isoflavone 2-hydroxylase','Heme-thiolate.','Acts on isoflavones with a 4-methoxy group, such as formononetin andbiochanin A.Involved in the biosynthesis of the pterocarpin phytoalexinsmedicarpin and maackiain.EC 1.14.13.89 is less specific and acts on other isoflavones as wellas 4-methoxyisoflavones.','Formononetin + NADPH + O(2) = 2-hydroxyformononetin + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.54','Ketosteroid monooxygenase','Steroid-ketone monooxygenase','FAD.','A single FAD-containing enzyme catalyzes three types of monooxygenase(Baeyer-Villiger oxidation) reaction.The oxidative esterification of a number of derivatives ofprogesterone to produce the corresponding 17-alpha-hydroxysteroid17-acetate ester, such as testosterone acetate, is shown in reaction(2).The oxidative lactonization of a number of derivatives ofandrostenedione to produce the 13,17-secoandrosteno-17,13-alpha-lactone, such as testololactone, is shown in reaction (3).The oxidative cleavage of the 17-beta-side-chain of 17-alpha-hydroxyprogesterone to produce androstenedione and acetate is shownin reaction (4).reaction (2) is also catalyzed by EC 1.14.99.4 and reactions (3) and(4) correspond to that catalyzed by EC 1.14.99.12.The possibility that a single enzyme is responsible for the reactionsascribed to EC 1.14.99.4 and EC 1.14.99.12 in other tissues cannot beexcluded.','(1) Ketosteroid + NADPH + O(2) = steroid ester/lactone + NADP(+) + H(2)O.(2) Progesterone + NADPH + O(2) = testosterone acetate + NADP(+) + H(2)O.(3) Androstenedione + NADPH + O(2) = testololactone + NADP(+) + H(2)O.(4) 17-alpha-hydroxyprogesterone + NADPH + O(2) = androstenedione +NADP(+) + H(2)O + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.55','Protopine 6-monooxygenase','Protopine 6-hydroxylase','Heme-thiolate.','Involved in benzophenanthridine alkaloid synthesis in higher plants.','Protopine + NADPH + O(2) = 6-hydroxyprotopine + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.56','Dihydrosanguinarine 10-monooxygenase','Dihydrosanguinarine 10-hydroxylase','Heme-thiolate.','Involved in benzophenanthridine alkaloid synthesis in higher plants.','Dihydrosanguinarine + NADPH + O(2) = 10-hydroxydihydrosanguinarine +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.57','Dihydrochelirubine 12-monooxygenase','Dihydrochelirubine 12-hydroxylase','Heme-thiolate.','NULL','Dihydrochelirubine + NADPH + O(2) = 12-hydroxydihydrochelirubine +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.58','Benzoyl-CoA 3-monooxygenase','Benzoyl-CoA 3-hydroxylase','FMN or FAD.','Benzoate is not a substrate.','Benzoyl-CoA + NADPH + O(2) = 3-hydroxybenzoyl-CoA + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.59','L-lysine N(6)-monooxygenase (NADPH)','Lysine N(6)-hydroxylase','FAD.','The enzyme from strain EN 222 of Escherichia coli is highly specificfor L-lysine; L-ornithine and L-homolysine are, for example,not substrates.','L-lysine + NADPH + O(2) = N(6)-hydroxy-L-lysine + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.60','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.61','2-hydroxyquinoline 8-monooxygenase','2-oxo-1,2-dihydroquinoline 8-monooxygenase','Iron.','Quinolin-2-ol exists largely as the quinolin-2(1H)-one tautomer.','Quinolin-2-ol + NADH + O(2) = quinolin-2,8-diol + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.62','4-hydroxyquinoline 3-monooxygenase','1-H-4-oxoquinoline 3-monooxygenase','NULL','Quinolin-4-ol exists largely as the quinolin-4(1H)-one tautomer.','Quinolin-4-ol + NADH + O(2) = quinolin-3,4-diol + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.63','3-hydroxyphenylacetate 6-hydroxylase','3-hydroxyphenylacetate 6-monooxygenase','FAD.','3-hydroxyphenylacetate 6-hydroxylase from Flavobacterium sp. ishighly specific for 3-hydroxyphenylacetate and uses NADH and NADPH aselectron donors with similar efficiency.','3-hydroxyphenylacetate + NAD(P)H + O(2) = 2,5-dihydroxyphenylacetate +NAD(P)(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.64','4-hydroxybenzoate 1-hydroxylase','4-hydroxybenzoate 1-monooxygenase','FAD.','The enzyme from Candida parapsilosis is specific for4-hydroxybenzoate derivatives and prefers NADH to NADPH as electrondonor.','4-hydroxybenzoate + NAD(P)H + O(2) = hydroquinone + NAD(P)(+) + H(2)O +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.65','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.66','2-hydroxycyclohexanone 2-monooxygenase','NULL','NULL','The product decomposes spontaneously to 6-oxohexanoic acid (adipicsemialdehyde).Formerly EC 1.14.12.6.','2-hydroxycyclohexan-1-one + NADPH + O(2) = 6-hydroxyhexan-6-olide +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.67','Quinine 3-monooxygenase','Nifedipine oxidase','Heme-thiolate.','NULL','Quinine + NADPH + O(2) = 3-hydroxyquinine + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.68','4-hydroxyphenylacetaldehyde oxime monooxygenase','4-hydroxybenzeneacetaldehyde oxime monooxygenase','Heme-thiolate.','Involved in the biosynthesis of the cyanogenic glucoside dhurrin insorghum, along with EC 1.14.13.41 and EC 2.4.1.85.','(Z)-4-hydroxyphenylacetaldehyde oxime + NADPH + O(2) = (S)-4-hydroxymandelonitrile + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.69','Alkene monooxygenase','Alkene epoxygenase','Iron.','The enzyme from Xanthobacter sp. strain Py2 is a multicomponentenzyme comprising:(1) an NADH reductase, which provides the reductant for O(2)activation.(2) a Rieske-type ferredoxin, which is an electron-transfer protein.(3) an oxygenase, which contains the catalytic center for alkeneepoxidation.(4) a small protein of unknown function that is essential foractivity.The enzyme oxygenates C(2) to C(6) aliphatic alkenes.With 1,2-epoxypropane as substrate, the stereospecificity of theepoxypropane formed is 95% (R) and 5% (S).','Propene + NADH + O(2) = 1,2-epoxypropane + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.70','Sterol 14-alpha-demethylase','Cytochrome P450 51','Heme-thiolate.','This heme-thiolate (P450) enzyme acts on a range of steroids with a14-alpha-methyl group, such as obtusifoliol and lanosterol.The enzyme catalyzes two successive hydroxylations of the 14-alpha-methyl group, followed by elimination as formate, leaving a 14(15)double bond.','A 14-alpha-methylsteroid + 3 O(2) + 3 NADPH = a Delta(14)-steroid +formate + 3 NADP(+) + 4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.71','N-methylcoclaurine 3-monooxygenase','(S)-N-methylcoclaurine 3-hydroxylase','Heme-thiolate.','A P450 protein involved in benzylisoquinoline alkaloid synthesis inhigher plants.','(S)-N-methylcoclaurine + NADPH + O(2) = (S)-3-hydroxy-N-methylcoclaurine+ NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.72','Methylsterol monooxygenase','4-methylsterol oxidase','NULL','Requires cytochrome b5.Also acts on 4-alpha-methyl-5-alpha-cholest-7-en-3-beta-ol.The sterol can be based on cycloartenol as well as lanosterol.Formerly EC 1.14.99.16.','(1) 4,4-dimethyl-5-alpha-cholest-7-en-3-beta-ol + NAD(P)H + O(2) =4-beta-hydroxymethyl-4-alpha-methyl-5-alpha-cholest-7-en-3-beta-ol +NAD(P)(+) + H(2)O.(2) 4-beta-hydroxymethyl-4-alpha-methyl-5-alpha-cholest-7-en-3-beta-ol +NAD(P)H + O(2) = 3-beta-hydroxy-4-beta-methyl-5-alpha-cholest-7-ene-4-alpha-carbaldehyde + NAD(P)(+) + 2 H(2)O.(3) 3-beta-hydroxy-4-beta-methyl-5-alpha-cholest-7-ene-4-alpha-carbaldehyde + NAD(P)H + O(2) = 3-beta-hydroxy-4-beta-methyl-5-alpha-cholest-7-ene-4-alpha-carboxylate + NAD(P)(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.73','Tabersonine 16-hydroxylase','NULL','Heme-thiolate.','NULL','Tabersonine + NADPH + O(2) = 16-hydroxytabersonine + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.74','7-deoxyloganin 7-hydroxylase','NULL','Heme-thiolate.','NULL','7-deoxyloganin + NADPH + O(2) = loganin + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.75','Vinorine hydroxylase','NULL','Heme-thiolate.','Forms a stage in the biosynthesis of the indole alkaloid ajmaline.','Vinorine + NADPH + O(2) = vomilenine + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.76','Taxane 10-beta-hydroxylase','5-alpha-taxadienol-10-beta-hydroxylase','Heme-thiolate.','Involved in the biosynthesis of the diterpenoid antineoplastic drugtaxol (paclitaxel).','Taxa-4(20),11-dien-5-alpha-yl acetate + NADPH + O(2) = 10-beta-hydroxytaxa-4(20),11-dien-5-alpha-yl acetate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.77','Taxane 13-alpha-hydroxylase','NULL','Heme-thiolate.','Involved in the biosynthesis of the diterpenoid antineoplastic drugtaxol (paclitaxel).','Taxa-4(20),11-dien-5-alpha-ol + NADPH + O(2) = taxa-4(20),11-dien-5-alpha,13-alpha-diol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.78','Ent-kaurene oxidase','NULL','NULL','Requires cytochrome P450.Catalyzes three sucessive oxidations of the 4-methyl group of ent-kaurene giving kaurenoic acid.','(1) Ent-kaur-16-ene + NADPH + O(2) = ent-kaur-16-en-19-ol + NADP(+) +H(2)O.(2) Ent-kaur-16-en-19-ol + NADPH + O(2) = ent-kaur-16-en-19-al + NADP(+)+ 2 H(2)O.(3) Ent-kaur-16-en-19-al + NADPH + O(2) = ent-kaur-16-en-19-oate +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.79','Ent-kaurenoic acid oxidase','NULL','NULL','Requires cytochrome P450.Catalyzes three sucessive oxidations of ent-kaurenoic acid.The second step includes a ring-B contraction giving the gibbaneskeleton.In Cucurbita maxima, ent-6-alpha,7-alpha-dihydroxykaur-16-en-19-oateis also formed.','Ent-kaur-16-en-19-oate + 3 NADPH + 3 O(2) = gibberellin A(12) + 3 NADP(+)+ 4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.80','(R)-limonene 6-monooxygenase','(+)-limonene 6-monooxygenase','Heme-thiolate.','The reaction is stereospecific with over 95% yield of (+)-trans-carveol from (R)-limonene.(S)-Limonene, the substrate for EC 1.14.13.48 is not a substrate.Forms part of the carvone biosynthesis pathway in Carum carvi(caraway) seeds.','(+)-(R)-limonene + NADPH + O(2) = (+)-trans-carveol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.81','Magnesium-protoporphyrin IX monomethyl ester (oxidative) cyclase','Mg-protoporphyrin IX monomethyl ester oxidative cyclase','Iron.','The cyclase activity in Chlamydomonas reinhardtii is associatedexclusively with the membranes, whereas that from cucumber cotyledonsrequires both membrane and soluble fractions for activity.','Magnesium-protoporphyrin IX 13-monomethyl ester + 3 NADPH + 3 O(2) =divinylprotochlorophyllide + 3 NADP(+) + 5 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.82','Vanillate monooxygenase','4-hydroxy-3-methoxybenzoate demethylase','NULL','Forms part of the vanillin degradation pathway in Arthrobacter sp.Formerly EC 1.2.3.12.','Vanillate + O(2) + NADH = 3,4-dihydroxybenzoate + NAD(+) + H(2)O +formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.83','Precorrin-3B synthase','Precorrin-3X synthase','Iron-sulfur.','An oxygen atom from dioxygen is incorporated into the macrocycle atC-20.In the aerobic cobalamin biosynthesis pathway, four enzymes areinvolved in the conversion of precorrin-3A to precorrin-6A.This enzyme carries out the first of the four steps, yieldingprecorrin-3B as the product.This is followed by three methylation reactions, which introduce amethyl group at C-17 (CobJ; EC 2.1.1.131), C-11 (CobM; 2.1.1.133) andC-1 (CobF; EC 2.1.1.152) of the macrocycle, giving rise to precorrin-4, precorrin-5 and precorrin-6A, respectively.','Precorrin-3A + NADH + O(2) = precorrin-3B + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.84','4-hydroxyacetophenone monooxygenase','HAPMO','FAD.','The enzyme from Pseudomonas fluorescens ACB catalyzes the conversionof a wide range of acetophenone derivatives.Highest activity occurs with compounds bearing an electron-donatingsubstituent at the para position of the aromatic ring.In the absence of substrate, the enzyme can act as an NADPH oxidase(EC 1.6.3.1).','(4-hydroxyphenyl)ethan-1-one + NADPH + O(2) = 4-hydroxyphenyl acetate +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.85','Glyceollin synthase','Dimethylallyl-3,6a,9-trihydroxypterocarpan cyclase','Heme-thiolate.','Glyceollins II and III are formed from 2-dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan whereas glyceollin I is formed from the4-isomer.','2-(or 4-)dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan + NADPH +O(2) = glyceollin + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.86','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.87','Licodione synthase','(2S)-flavanone 2-hydroxylase','Heme-thiolate.','It probably forms 2-hydroxyliquiritigenin which spontaneously formslicodione.NADH can act instead of NADPH, but more slowly.','Liquiritigenin + NADPH + O(2) = licodione + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.88','Flavonoid 3,5-hydroxylase','F3,5H','Heme-thiolate.','The 3,5-dihydroxyflavanone is formed via the 3-hydroxyflavanone.In Petunia hybrida the enzyme acts on naringenin, eriodictyol,dihydroquercetin (taxifolin) and dihydrokaempferol (aromadendrin).The enzyme catalyzes the hydroxylation of 5,7,4-trihydroxyflavanone(naringenin) at either the 3 position to form eriodictyol or at boththe 3 and 5 positions to form 5,7,3,4,5-pentahydroxyflavanone(dihydrotricetin).The enzyme also catalyzes the hydroxylation of3,5,7,3,4-pentahydroxyflavanone (taxifolin) at the 5 position,forming ampelopsin.NADH is not a good substitute for NADPH.','A flavanone + 2 NADPH + 2 O(2) = a 3,5-dihydroxyflavanone + 2 NADP(+) +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.89','Isoflavone 2-hydroxylase','CYP Ge-3','Heme-thiolate.','Acts on daidzein, formononetin and genistein.EC 1.14.13.53 has the same reaction but is more specific as itrequires a 4-methoxyisoflavone.','An isoflavone + NADPH + O(2) = a 2-hydroxyisoflavone + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.90','Zeaxanthin epoxidase','Zea-epoxidase','FAD.','Active under conditions of low light.Along with EC 1.10.99.3, this enzyme forms part of the xanthophyll(or violaxanthin) cycle, which is involved in protecting the plantagainst damage by excess light.It will also epoxidize lutein in some higher-plant species.','Zeaxanthin + 2 NAD(P)H + 2 O(2) = violaxanthin + 2 NAD(P)(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.91','Deoxysarpagine hydroxylase','DOSH','Heme-thiolate.','NULL','10-deoxysarpagine + NADPH + O(2) = sarpagine + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.92','Phenylacetone monooxygenase','PAMO','FAD.','NADH cannot replace NADPH as coenzyme.In addition to phenylacetone, which is the best substrate found todate, this Baeyer-Villiger monooxygenase can oxidize other aromaticketones (1-(4-hydroxyphenyl)propan-2-one, 1-(4-hydroxyphenyl)propan-2-one and 3-phenylbutan-2-one), some alipatic ketones (e.g. dodecan-2-one) and sulfides (e.g. 1-methyl-4-(methylsulfanyl)benzene).','Phenylacetone + NADPH + O(2) = benzyl acetate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.93','(+)-abscisic acid 8-hydroxylase','(+)-ABA 8-hydroxylase','Heme-thiolate.','Catalyzes the first step in the oxidative degradation of abscisicacid and is considered to be the pivotal enzyme in controlling therate of degradation of this plant hormone.CO inhibits the reaction, but its effects can be reversed by thepresence of blue light.The 8-hydroxyabscisate formed can be converted into (-)-phaseicacid, most probably spontaneously.Other enzymes involved in the abscisic-acid biosynthesis pathway areEC 1.1.1.288, EC 1.2.3.14 and EC 1.13.11.51.','(+)-abscisate + NADPH + O(2) = 8-hydroxyabscisate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.94','Lithocholate 6-beta-hydroxylase','6-beta-hydroxylase','Heme-thiolate.','Expression of the gene for this enzyme is 50-fold higher in malecompared to female hamsters.','Lithocholate + NADPH + O(2) = 6-beta-hydroxylithocholate + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.95','7-alpha-hydroxycholest-4-en-3-one 12-alpha-hydroxylase','7-alpha-hydroxy-4-cholesten-3-one 12-alpha-monooxygenase','Heme-thiolate.','Requires EC 1.6.2.4 and cytochrome b5 for maximal activity.Important in bile acid biosynthesis, being responsible for thebalance between the formation of cholic acid and chenodeoxycholicacid.','7-alpha-hydroxycholest-4-en-3-one + NADPH + O(2) = 7-alpha,12-alpha-dihydroxycholest-4-en-3-one + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.96','5-beta-cholestane-3-alpha,7-alpha-diol 12-alpha-hydroxylase','5-beta-cholestane-3-alpha,7-alpha-diol 12-alpha-monooxygenase','Heme-thiolate.','Key enzyme in the biosynthesis of the bile acid cholic acid(3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholanoic acid).The activity determines the biosynthetic ratio between cholic acidand chenodeoxycholic acid.Can also hydroxylate the substrate at the 25 and 26 position, but toa lesser extent.','5-beta-cholestane-3-alpha,7-alpha-diol + NADPH + O(2) = 5-beta-cholestane-3-alpha,7-alpha,12-alpha-triol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.97','Taurochenodeoxycholate 6-alpha-hydroxylase','Taurochenodeoxycholate 6-alpha-monooxygenase','Heme-thiolate.','Requires cytochrome b5 for maximal activity.Acts on taurochenodeoxycholate, taurodeoxycholate and less readily onlithocholate and chenodeoxycholate.In adult pig (Sus scrofa), hyocholic acid replaces cholic acid as aprimary bile acid.','(1) Taurochenodeoxycholate + NADPH + O(2) = taurohyocholate + NADP(+) +H(2)O.(2) Lithocholate + NADPH + O(2) = hyodeoxycholate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.98','Cholesterol 24-hydroxylase','Cholesterol 24-monooxygenase','Heme-thiolate.','Can also produce 25-hydroxycholesterol.In addition, it can further hydroxylate the product to 24,25-dihydroxycholesterol and 24,27-dihydroxycholesterol.This reaction is the first step in the enzymatic degradation ofcholesterol in the brain as hydroxycholesterol can pass the blood-brain barrier whereas cholesterol cannot.','Cholesterol + NADPH + O(2) = (24S)-24-hydroxycholesterol + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.99','24-hydroxycholesterol 7-alpha-hydroxylase','24-hydroxycholesterol 7-alpha-monooxygenase','Heme-thiolate.','Found in liver microsomes and in ciliary non-pigmented epithelium.Specific for (24R)-cholest-5-ene-3-beta,24-diol as substrate.','(24R)-cholest-5-ene-3-beta,24-diol + NADPH + O(2) = (24R)-cholest-5-ene-3-beta,7-alpha,24-triol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.100','25/26-hydroxycholesterol 7-alpha-hydroxylase','25-hydroxycholesterol 7-alpha-hydroxylase','Heme-thiolate.','Unlike EC 1.14.13.99 which is specific for its oxysterol substrate,this enzyme can also metabolize the oxysterols 24,25-epoxycholesterol, 22-hydroxycholesterol and 24-hydroxycholesterol,but to a lesser extent.Formerly EC 1.14.13.60.','(1) Cholest-5-ene-3-beta,25-diol + NADPH + O(2) = cholest-5-ene-3-beta,7-alpha,25-triol + NADP(+) + H(2)O.(2) (25R)-cholest-5-ene-3-beta,26-diol + NADPH + O(2) = (25R)-cholest-5-ene-3-beta,7-alpha,26-triol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.101','Senecionine N-oxygenase','Senecionine monooxygenase (N-oxide-forming)','NULL','NADH cannot replace NADPH.While pyrrolizidine alkaloids of the senecionine and monocrotalinetypes are generally good substrates (e.g. senecionine, retrorsine andmonocrotaline), the enzyme does not use ester alkaloids lacking anhydroxy group at C-7 (e.g. supinine and phalaenopsine), 1,2-dihydro-alkaloids (e.g. sarracine) or unesterified necine bases (e.g.senkirkine) as substrates.Senecionine N-oxide is used by insects as a chemical defense: it isnon-toxic, but it is bioactivated to a toxic form by the action ofcytochrome P-450 oxidase when absorbed by insectivores.','Senecionine + NADPH + O(2) = senecionine N-oxide + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.102','Psoralen synthase','NULL','NULL','Specific for (+)-marmesin, and to a much lesser extent5-hydroxymarmesin, as substrate.Furanocoumarins protect plants from fungal invasion and herbivoreattack.(+)-columbianetin, the angular furanocoumarin analog of the linearfuranocoumarin (+)-marmesin, is not a substrate for the enzyme but itis a competitive inhibitor.','(+)-marmesin + NADPH + O(2) = psoralen + NADP(+) + acetone + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.103','8-dimethylallylnaringenin 2-hydroxylase','8-DMAN 2-hydroxylase','NULL','Specific for sophoraflavanone B as substrate.NADPH cannot be replaced by NADH, FAD or FMN.Along with EC 2.5.1.70 and EC 2.5.1.71, this enzyme forms part of thesophoraflavanone-G biosynthesis pathway.','Sophoraflavanone B + NADPH + O(2) = leachianone G + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.104','(+)-menthofuran synthase','(+)-MFS','Heme-thiolate.','The conversion of substrate into product involves the hydroxylationof the syn-methyl (C9), intramolecular cyclization to the hemiketaland dehydration to the furan.This is the second cytochrome P-450-mediated step of monoterpenemetabolism in peppermint, with the other step being catalyzed byEC 1.14.13.47.','(+)-pulegone + NADPH + O(2) = (+)-menthofuran + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.105','Monocyclic monoterpene ketone monooxygenase','1-hydroxy-2-oxolimonene 1,2-monooxygenase','FAD.','This Baeyer-Villiger monooxygenase enzyme from the Gram-positivebacterium Rhodococcus erythropolis DCL14 has wide substratespecificity, catalyzing the lactonization of a large number ofmonocyclic monoterpene ketones and substituted cyclohexanones.Both (1R,4S)- and (1S,4R)-1-hydroxymenth-8-en-2-one are metabolized,with the lactone product spontaneously rearranging to form3-isopropenyl-6-oxoheptanoate.','(1) (-)-menthone + NADPH + O(2) = (4R,7S)-7-isopropyl-4-methyloxepan-2-one + NADP(+) + H(2)O.(2) Dihydrocarvone + NADPH + O(2) = 4-isopropenyl-7-methyloxepan-2-one +NADP(+) + H(2)O.(3) (Iso)-dihydrocarvone + NADPH + O(2) = 6-isopropenyl-3-methyloxepan-2-one + NADP(+) + H(2)O.(4) 1-hydroxymenth-8-en-2-one + NADPH + O(2) = 3-isopropenyl-6-oxoheptanoate + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.106','Epi-isozizaene 5-monooxygenase','CYP170A1','NULL','This enzyme, from the soil-dwelling bacterium Streptomyces coelicolorA3(2), catalyzes two sequential allylic oxidation reactions.The substrate epi-isozizaene, which is formed by the action ofEC 4.2.3.37 is first oxidized to yield the epimeric intermediates(5R)-albaflavenol and (5S)-albaflavenol, which can be furtheroxidized to yield the sesquiterpenoid antibiotic albaflavenone.','(+)-epi-isozizaene + 2 NADPH + 2 O(2) = albaflavenone + 2 NADP(+) +3 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.107','Limonene 1,2-monooxygenase','NULL','FAD.','Limonene is the most widespread terpene and is formed by more than300 plants.Rhodococcus erythropolis DCL14, a Gram-positive bacterium, is able togrow on both (S)-limonene and (R)-limonene as the sole source ofcarbon and energy.NADPH can act instead of NADH, although more slowly.It has not been established if the product formed is optically pureor a mixture of two enantiomers.','(1) (S)-limonene + NAD(P)H + O(2) = 1,2-epoxymenth-8-ene + NAD(P)++ H(2)O.(2) (R)-limonene + NAD(P)H + O(2) = 1,2-epoxymenth-8-ene + NAD(P)++ H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.108','Abieta-7,13-diene hydroxylase','Abietadiene hydroxylase','Heme-thiolate.','This enzyme catalyzes a step in the pathway of abietic acidbiosynthesis.The activity has been demonstrated in cell-free stem extracts ofAbies grandis (grand fir) and Pinus contorta (lodgepole pine).Activity is induced by wounding of the plant tissue.','Abieta-7,13-diene + NADPH + O(2) = abieta-7,13-dien-18-ol + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.109','Abieta-7,13-dien-18-ol hydroxylase','Abietadienol hydroxylase','Heme-thiolate.','Catalyzes a step in the pathway of abietic acid biosynthesis.The activity has been demonstrated in cell-free stem extracts ofAbies grandis (grand fir) and Pinus contorta (lodgepole pine),and the gene encoding the enzyme has been identified in Pinus taeda(loblolly pine).The recombinant enzyme catalyzed the oxidation of multiple diterpenealcohol and aldehydes, including levopimaradienol, isopimara-7,15-dienol, isopimara-7,15-dienal, dehydroabietadienol,and dehydroabietadienal.It is not able to oxidize abietadiene.','Abieta-7,13-dien-18-ol + NADPH + O(2) = abieta-7,13-dien-18-al + NADP(+)+ 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.110','Geranylgeraniol 18-hydroxylase','GGOH-18-hydroxylase','Heme-thiolate.','NULL','Geranylgeraniol + NADPH + O(2) = 18-hydroxygeranylgeraniol + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.111','Methanesulfonate monooxygenase','Mesylate monooxygenase','Flavoprotein; Iron-sulfur.','Methanesulfonate is the simplest of the sulfonates and is a substratefor the growth of certain methylotrophic microorganisms.Compared with EC 1.14.14.5, this enzyme has a restricted substraterange that includes only the short-chain aliphatic sulfonates(methane- to butanesulfonate) and excludes all larger molecules, suchas arylsulfonates and aromatic sulfonates.The enzyme from the bacterium Methylosulfonomonas methylovora is amulticomponent system comprising an hydroxylase, a reductase (MsmD)and a ferredoxin (MsmC).The hydroxylase has both large (MsmA) and small (MsmB) subunits, witheach large subunit containing a Rieske-type [2Fe-2S] center.Formerly EC 1.14.14.6.','Methanesulfonate + NADH + O(2) = formaldehyde + NAD(+) + sulfite + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.112','3-epi-6-deoxocathasterone 23-monooxygenase','CYP90C1','NULL','This enzyme is involved in brassinosteroid biosynthesis.C-23 hydroxylation shortcuts bypass campestanol, 6-deoxocathasterone,and 6-deoxoteasterone and lead directly from (22S,24R)-22-hydroxy-5-alpha-ergostan-3-one and 3-epi-6-deoxocathasterone to 3-dehydro-6-deoxoteasterone and 6-deoxotyphasterol.','(1) 3-epi-6-deoxocathasterone + NADPH + O(2) = 6-deoxotyphasterol +NADP(+) + H(2)O.(2) (22S,24R)-22-hydroxy-5-alpha-ergostan-3-one + NADPH + O(2) =3-dehydro-6-deoxoteasterone + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.113','FAD-dependent urate hydroxylase','FAD-dependent urate oxidase','Flavoprotein.','The reaction is part of the purine catabolic pathway in the bacteriumKlebsiella pneumoniae.The enzyme is different from EC 1.7.3.3, found in most plants, whichproduces hydrogen peroxide.The product of the enzyme is a substrate for EC 3.5.2.17.','Urate + NADH + O(2) = 5-hydroxyisourate + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.114','6-hydroxynicotinate 3-monooxygenase','6HNA monooxygenase','FAD.','The reaction is involved in the aerobic catabolism of nicotinic acid.','6-hydroxynicotinate + NADH + O(2) = 2,5-dihydroxypyridine + NAD(+) +H(2)O + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.115','Angelicin synthase','CYP71AJ4','Heme-thiolate.','This P450 monooxygenase enzyme is involved in the formation ofangular furanocoumarins.Attacks its substrate by syn-elimination of hydrogen from C-3.','(+)-columbianetin + NADPH + O(2) = angelicin + NADP(+) + acetone +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.116','Geranylhydroquinone 3-hydroxylase','GHQ 3-hydroxylase','NULL','Contains cytochrome P450.','Geranylhydroquinone + NADPH + O(2) = 3-hydroxygeranylhydroquinone +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.117','Isoleucine N-monooxygenase','NULL','Heme-thiolate.','This enzyme catalyzes two successive N-hydroxylations ofL-isoleucine, the first committed steps in the biosynthesis of thecyanogenic glucoside lotaustralin in the plant Lotus japonicus.The product of the two hydroxylations, N,N-dihydroxy-L-isoleucine,is extremely labile and dehydrates spontaneously.The dehydrated product is then subject to a decarboxylation thatproduces the oxime.It is still not known whether the decarboxylation is spontaneous orcatalyzed by the enzyme.The product, (E)-2-methylbutanal oxime, undergoes a spontaneousisomerization to the (Z) form.The enzyme can also accept L-valine as substrate, with a loweractivity.It is different from EC 1.14.13.118 which prefers L-valine.','L-isoleucine + 2 H(+) + 2 NADPH + 2 O(2) = (E)-2-methylbutanal oxime +CO(2) + 3 H(2)O + 2 NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.118','Valine N-monooxygenase','NULL','Heme-thiolate.','This enzyme catalyzes two successive N-hydroxylations of L-valine,the first committed steps in the biosynthesis of the cyanogenicglucoside linamarin in the plant Manihot esculenta (cassava).The product of the two hydroxylations, N,N-dihydroxy-L-valine,is extremely labile and dehydrates spontaneously.The dehydrated product is then subject to a decarboxylation thatproduces the oxime.It is still not known whether the decarboxylation is spontaneous orcatalyzed by the enzyme.The product, (E)-2-methylpropanal oxime, undergoes a spontaneousisomerization to the (Z) form.The enzyme can also accept L-isoleucine as substrate, with a loweractivity.It is different from EC 1.14.13.117 which prefers L-isoleucine.','L-valine + 2 H(+) + 2 NADPH + 2 O(2) = (Z)-2-methylpropanal oxime + CO(2)+ 3 H(2)O + 2 NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.119','5-epiaristolochene 1,3-dihydroxylase','5-epi-aristolochene 1,3-dihydroxylase','Heme-thiolate.','Kinetic studies suggest that 1-beta-hydroxyepiaristolochene is mainlyformed first followed by hydroxylation at C-3.However the reverse order via 3-alpha-hydroxyepiaristolochene doesoccur.','5-epiaristolochene + 2 NADPH + 2 O(2) = capsidiol + 2 NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.120','Costunolide synthase','NULL','Heme-thiolate.','The enzyme hydroxylates carbon C-6 of germacra-1(10),4,11(13)-trien-12-oate to give 6-alpha-hydroxygermacra-1(10),4,11(13)-trien-12-oate,which probably spontaneously cyclises to form the lactone ring.','Germacra-1(10),4,11(13)-trien-12-oate + NADPH + O(2) = (+)-costunolide +NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.121','Premnaspirodiene oxygenase','HPO','Heme-thiolate.','The enzyme from the plant Hyoscymus muticus also hydroxylatesvalencene at C-2 to give the alpha-hydroxy compound, nootkatol,and this is converted into nootkatone.5-Epiaristolochene and epieremophilene are hydroxylated at C-2 togive a 2-beta-hydroxy derivative which is not further oxidized.','(-)-vetispiradiene + 2 NADPH + 2 O(2) = solavetivone + 2 NADP(+) +3 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.122','Chlorophyllide-a oxygenase','CAO','Iron.','Catalyzes two successive hydroxylations at the 7-methyl group ofchlorophyllide a.The second step yields the aldehyde hydrate, which loses H(2)Ospontaneously to form chlorophyllide b.Chlorophyll a and protochlorophyllide a are not substrates.Formerly EC 1.13.12.14.','Chlorophyllide a + 2 O(2) + 2 NADPH = chlorophyllide b + 3 H(2)O +2 NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.123','Germacrene A hydroxylase','NULL','Heme-thiolate.','This is probably part of the biosynthesis of many sesquiterpenoidlactones.In Lactuca sativa EC 1.14.13.213 is a mutifunctional enzyme withEC 1.1.1.314.','(+)-germacrene A + NADPH + O(2) = germacra-1(10),4,11(13)-trien-12-ol +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.124','Phenylalanine N-monooxygenase','CYP79A2','Heme-thiolate.','This enzyme catalyzes two successive N-hydroxylations ofL-phenylalanine, the first committed steps in the biosynthesis ofbenzylglucosinolate.The product of the two hydroxylations, N,N-dihydroxy-L-phenylalanine,is extremely labile and dehydrates spontaneously.The dehydrated product is then subject to a decarboxylation thatproduces the oxime.It is still not known whether the decarboxylation is spontaneous orcatalyzed by the enzyme.The product, (E)-phenylacetaldoxime, undergoes a spontaneousisomerization to the (Z) form.Formerly EC 1.14.13.n1.','L-phenylalanine + 2 NADPH + 2 O(2) = (E)-phenylacetaldoxime + 2 NADP(+) +3 H(2)O + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.125','Tryptophan N-monooxygenase','CYP79B1','Heme-thiolate.','This enzyme catalyzes two successive N-hydroxylations ofL-tryptophan, the first steps in the biosynthesis of the both auxinand the indole alkaloid phytoalexin camalexin.The product of the two hydroxylations, N,N-dihydroxy-L-tryptophan,is extremely labile and dehydrates spontaneously.The dehydrated product is then subject to a decarboxylation thatproduces the oxime.It is still not known whether the decarboxylation is spontaneous orcatalyzed by the enzyme.The product, (E)-indol-3-ylacetaldoxime, undergoes a spontaneousisomerization to the (Z) form.Formerly EC 1.14.13.n2.','L-tryptophan + 2 NADPH + 2 O(2) = indole-3-acetaldoxime + 2 NADP(+) +3 H(2)O + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.126','Vitamin D(3) 24-hydroxylase','CYP24A1','Heme-thiolate.','The second donor, NADPH, donates electrons through EC 1.18.1.2 and a[2Fe-2S] ferredoxin.The enzyme can perform up to 6 rounds of hydroxylation of thesubstrate calcitriol leading to calcitroic acid.The human enzyme also shows 23-hydroxylating activity leading to1-alpha,25 dihydroxyvitamin D(3)-26,23-lactone as end product whilethe mouse and rat enzymes do not.Formerly EC 1.14.13.n4.','(1) Calcitriol + NADPH + O(2) = calcitetrol + NADP(+) + H(2)O.(2) Calcidiol + NADPH + O(2) = secalciferol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.127','3-(3-hydroxy-phenyl)propanoic acid hydroxylase','NULL','FAD.','This enzyme participates in a meta-cleavage pathway employed by thebacterium Escherichia coli for the degradation of variousphenylpropanoid compounds.Formerly EC 1.14.13.n3.','(1) 3-(3-hydroxyphenyl)propanoate + NADH + O(2) = 3-(2,3-dihydroxyphenyl)propanoate + H(2)O + NAD(+).(2) (2E)-3-(3-hydroxyphenyl)prop-2-enoate + NADH + O(2) = (2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate + H(2)O + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.128','7-methylxanthine demethylase','NULL','Iron.','NADH is the preferred cofactor.Part of the caffeine degradation pathway in Pseudomonas putida.','7-methylxanthine + O(2) + NAD(P)H = xanthine + NAD(P)(+) + H(2)O +formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.129','Beta-carotene 3-hydroxylase','Beta-carotene 3,3-monooxygenase','Iron.','Also acts on other carotenoids with a beta-end group.In some species canthaxanthin is the preferred substrate.','Beta-carotene + 2 NADH + 2 O(2) = zeaxanthin + 2 NAD(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.130','Pyrrole-2-carboxylate monooxygenase','Pyrrole-2-carboxylate ooxygenase','FAD.','The enzyme initiates the degradation of pyrrole-2-carboxylate.','Pyrrole-2-carboxylate + NADH + O(2) = 5-hydroxypyrrole-2-carboxylate +NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.131','Dimethyl-sulfide monooxygenase','Dimethylsulfide monooxygenase','NULL','The enzyme has lower activity with diethyl sulfide and other short-chain alkyl methyl sulfides.Its activity is stimulated by combined addition of FMN, and, afterdepletion of cations, of Mg(2+) and Fe(2+).The enzyme from Hyphomicrobium is a two component system thatincludes an FMN-dependent reductase subunit and a monooxygenasesubunit.','Dimethyl sulfide + O(2) + NADH = methanethiol + formaldehyde + NAD(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.132','Squalene monooxygenase','Squalene epoxidase','FAD.','This enzyme, together with EC 5.4.99.7, was formerly known assqualene oxidocyclase.The electron donor, NADPH, is coupled via EC 1.6.2.4.Formerly EC 1.14.1.3, EC 1.14.99.7 and EC 1.99.1.13.','Squalene + NADPH + O(2) = (3S)-2,3-epoxy-2,3-dihydrosqualene + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.133','Pentalenene oxygenase','NULL','Heme-thiolate.','The enzyme is involved in the biosynthesis of pentalenolactone andrelated antibiotics.','Pentalenene + 2 NADPH + 2 O(2) = pentalen-13-al + 2 NADP(+) + 3 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.134','Beta-amyrin 11-oxidase','CYP88D6','NULL','Requires cytochrome P450.Part of the glycyrrhizin biosynthesis pathway.The enzyme is also able to oxidize 30-hydroxy-beta-amyrin to11-alpha,30-dihydroxy-beta-amyrin but this is not thought to be partof glycyrrhizin biosynthesis.','Beta-amyrin + 2 O(2) + 2 NADPH = 11-oxo-beta-amyrin + 3 H(2)O +2 NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.135','1-hydroxy-2-naphthoate hydroxylase','1-hydroxy-2-naphthoic acid hydroxylase','NULL','The enzyme is involved in the catabolic pathway for the degradationof chrysene in some bacteria.','1-hydroxy-2-naphthoate + NAD(P)H + O(2) = 1,2-dihydroxynaphthalene +NAD(P)(+) + H(2)O + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.136','2-hydroxyisoflavanone synthase','CYT93C','NULL','The reaction involves the migration of the 2-phenyl group of theflavanone to the 3-position of the isoflavanone.The 2-hydroxyl group is derived from the oxygen molecule.EC 4.2.1.105, acts on the products with loss of water and formationof genistein and daidzein, respectively.Formerly EC 1.14.13.86.','(1) Liquiritigenin + O(2) + NADPH = 2,7,4-trihydroxyisoflavanone + H(2)O+ NADP(+).(2) (2S)-naringenin + O(2) + NADPH = 2,4,5,7-tetrahydroxyisoflavanone +H(2)O + NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.137','Indole-2-monooxygenase','CYP71C4','NULL','The enzyme is involved in the biosynthesis of protective andallelophatic benzoxazinoids in some plants, most commonly from thefamily of Poacea (grasses).It is a member of the cytochrome P450 dependent monooxygenases.','Indole + NAD(P)H + O(2) = indolin-2-one + NAD(P)(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.138','Indolin-2-one monooxygenase','CYP71C2','NULL','The enzyme is involved in the biosynthesis of protective andallelophatic benzoxazinoids in some plants, most commonly from thefamily of Poacea (grasses).It is a member of the cytochrome P450 dependent monooxygenases.','Indolin-2-one + NAD(P)H + O(2) = 3-hydroxyindolin-2-one + NAD(P)(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.139','3-hydroxyindolin-2-one monooxygenase','CYP71C1','NULL','The enzyme is involved in the biosynthesis of protective andallelophatic benzoxazinoids in some plants, most commonly from thefamily of Poacea (grasses).It is a member of the cytochrome P450 dependent monooxygenases.','3-hydroxyindolin-2-one + NAD(P)H + O(2) = 2-hydroxy-2H-1,4-benzoxazin-3(4H)-one + NAD(P)(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.140','2-hydroxy-1,4-benzoxazin-3-one monooxygenase','CYP71C3','NULL','The enzyme is involved in the biosynthesis of protective andallelophatic benzoxazinoids in some plants, most commonly from thefamily of Poacea (grasses).It is a member of the cytochrome P450 dependent monooxygenases.','2-hydroxy-2H-1,4-benzoxazin-3(4H)-one + NAD(P)H + O(2) = 2,4-dihydroxy-2H-1,4-benzoxazin-3(4H)-one + NAD(P)(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.141','Cholest-4-en-3-one 26-monooxygenase','Cholest-4-en-3-one 27-monooxygenase','Heme-thiolate.','This enzyme, found in several bacterial pathogens, is involved indegradation of the host cholesterol.It catalyzes the hydroxylation of the C26 carbon, followed byoxidation of the alcohol to the carboxylic acid via the aldehydeintermediate.These activities are required to initiate the degradation of thealkyl side-chain of cholesterol.The enzyme also accepts cholesterol as a substrate, but unlikeEC 1.14.13.15, this enzyme is specific for C26 and prefers cholest-4-en-3-one.','Cholest-4-en-3-one + NADH + O(2) = 26-hydroxycholest-4-en-3-one + NAD(+)+ H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.142','3-ketosteroid 9-alpha-monooxygenase','3-ketosteroid 9-alpha-hydroxylase','Iron; Iron-sulfur.','The enzyme is involved in the cholesterol degradation pathway ofseveral bacterial pathogens, such as Mycobacterium tuberculosis.It is a two-component system consisting of a terminal oxygenase(KshA) and a ferredoxin reductase (KshB).The oxygenase contains a Rieske-type iron-sulfur center and non-hemeiron.The reductase component is a flavoprotein containing an NAD-bindingdomain and a plant-type iron-sulfur cluster.The product of the enzyme is unstable, and spontaneously converts to3-hydroxy-9,10-seconandrost-1,3,5(10)-triene-9,17-dione.','Androsta-1,4-diene-3,17-dione + NADH + O(2) = 9-alpha-hydroxyandrosta-1,4-diene-3,17-dione + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.143','Ent-isokaurene C2-hydroxylase','CYP71Z6','NULL','This is the initial step in the conversion of ent-isokaurene to theantibacterial oryzalides in rice, Oryza sativa.','Ent-isokaurene + O(2) + NADPH = ent-2-alpha-hydroxyisokaurene + H(2)O +NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.144','9-beta-pimara-7,15-diene oxidase','CYP99A3','NULL','A rice, Oryza sativa, enzyme involved in the phytoalexin momilactonebiosynthesis.It also acts similarly on 9-beta-stemod-13(17)-ene.','9-beta-pimara-7,15-diene + 3 O(2) + 3 NADPH = 9-beta-pimara-7,15-dien-19-oate + 3 NADP(+) + 4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.145','Ent-cassa-12,15-diene 11-hydroxylase','CYP76M7','NULL','A rice, Oryza sativa, enzyme involved in the biosynthesis of theantifungal phytocassanes.','Ent-cassa-12,15-diene + O(2) + NADPH = ent-11-beta-hydroxycassa-12,15-diene + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.146','Taxoid 14-beta-hydroxylase','NULL','NULL','Requires cytochrome P450.From the yew Taxus cuspidata.Also acts on taxa-4(20),11-dien-5-alpha-yl acetate.','10-beta-hydroxytaxa-4(20),11-dien-5-alpha-yl acetate + O(2) + NADPH =10-beta,14-beta-dihydroxytaxa-4(20),11-dien-5?-yl acetate + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.147','Taxoid 7-beta-hydroxylase','NULL','NULL','Requires cytochrome P450.From the yew tree Taxus cuspidata.Does not act on earlier intermediates in taxol biosynthesis.','Taxusin + O(2) + NADPH = 7-beta-hydroxytaxusin + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.148','Trimethylamine monooxygenase','Flavin-containing monooxygenase 3','Flavoprotein.','The bacterial enzyme enables bacteria to use trimethylamine as thesole source of carbon and energy.The mammalian enzyme is involved in detoxification of trimethylamine.Mutations in the human enzyme cause the inheritable disease known astrimethylaminuria (fish odor syndrome).','N,N,N-trimethylamine + NADPH + O(2) = N,N,N-trimethylamine N-oxide +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.149','Phenylacetyl-CoA 1,2-epoxidase','Phenylacetyl-CoA monooxygenase','NULL','Part of the aerobic pathway of phenylacetate catabolism inEscherichia coli and Pseudomonas putida.Formerly EC 1.14.13.n9.','Phenylacetyl-CoA + NADPH + O(2) = 2-(1,2-epoxy-1,2-dihydrophenyl)acetyl-CoA + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.150','Alpha-humulene 10-hydroxylase','CYP71BA1','NULL','The recommended numbering of humulene gives 10-hydroxy-alpha-humuleneas the product rather than 8-hydroxy-alpha-humulene as used by theliterature.','Alpha-humulene + O(2) + NADPH = 10-hydroxy-alpha-humulene + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.151','Linalool 8-monooxygenase','CYP111','Heme-thiolate.','The secondary electron donor is a specific [2Fe-2S] ferredoxin fromthe same bacterial strain.Formerly EC 1.14.99.28.','Linalool + 2 NADH + 2 O(2) = (6E)-8-oxolinalool + 2 NAD(+) + 3  H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.152','Geraniol 8-hydroxylase','CYP76B10','NULL','Also hydroxylates nerol and citronellol, cf. EC 1.14.13.151.The recommended numbering of geraniol gives 8-hydroxygeraniol as theproduct rather than 10-hydroxygeraniol.The cloned enzyme also catalyzed, but less efficiently,the 3-hydroxylation of naringenin (cf. EC 1.14.13.21).','Geraniol + NADPH + O(2) = (6E)-8-hydroxygeraniol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.153','(+)-sabinene 3-hydroxylase','NULL','NULL','The enzyme has been characterized from Salvia officinalis (sage).','(+)-sabinene + NADPH + O(2) = (+)-cis-sabinol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.154','Erythromycin 12 hydroxylase','NULL','NULL','The enzyme is responsible for the C-12 hydroxylation of themacrolactone ring, one of the last steps in erythromycinbiosynthesis.It shows 1200-1900-fold preference for erythromycin D over thealternative substrate erythromycin B.','Erythromycin D + NADPH + O(2) = erythromycin C + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.155','Alpha-pinene monooxygenase','NULL','NULL','Involved in the catabolism of alpha-pinene.','(-)-alpha-pinene + NADH + O(2) = alpha-pinene oxide + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.156','1,8-cineole 2-endo-monooxygenase','CYP176A','Heme-thiolate.','Isolated from the bacterium Citrobacter braakii, which can use 1,8-cineole as the sole source of carbon.','1,8-cineole + NADPH + O(2) = 2-endo-hydroxy-1,8-cineole + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.157','1,8-cineole 2-exo-monooxygenase','CYP3A4','Heme-thiolate.','The mammalian enzyme, expressed in liver microsomes, performs avariety of oxidation reactions of structurally unrelated compounds,including stereoids, fatty acids, and xenobiotics.Cf. EC 1.14.13.97, EC 1.14.13.67 and EC 1.14.13.32.','1,8-cineole + NADPH + O(2) = 2-exo-hydroxy-1,8-cineole + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.158','Amorpha-4,11-diene 12 monooxygenase','CYP71AV1','Heme-thiolate.','Cloned from the plant Artemisia annua (sweet wormwood).Part of the biosynthetic pathway of artemisinin.','Amorpha-4,11-diene + 3 O(2) + 3 NADPH = artemisinate + 3 NADP(+) +4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.159','Vitamin D 25-hydroxylase','CYP2R1','NULL','A microsomal enzyme isolated from human and mouse liver whichbioactivates vitamin D(3).While multiple isoforms (CYP27A1, CYP2J2/3, CYP3A4, CYP2D25 andCYP2C11) are able to catalyze the reaction in vitro, only CYP2R1 isthought to catalyze the reaction in humans in vivo.','Calciol + O(2) + NADPH = calcidiol + NADP(+) H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.160','(2,2,3-trimethyl-5-oxocyclopent-3-enyl)acetyl-CoA 1,5-monooxygenase','2-oxo-Delta(3)-4,5,5-trimethylcyclopentenylacetyl-CoA 1,2-monooxygenase','FAD.','Forms part of the catabolism of camphor.It acts on the CoA ester in preference to the free acid.','((1R)-2,2,3-trimethyl-5-oxocyclopent-3-enyl)acetyl-CoA + O(2) + NADPH =((2R)-3,3,4-trimethyl-6-oxo-3,6-dihydro-1H-pyran-2-yl)acetyl-CoA +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.161','(+)-camphor 6-exo-hydroxylase','(+)-camphor 6-hydroxylase','NULL','A cytochrome P-450 monooxygenase isolated from Salvia officinalis(sage).Involved in the catabolism of camphor in senescent tissue.','(+)-camphor + NADPH + O(2) = (+)-6-exo-hydroxycamphor + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.162','2,5-diketocamphane 1,2-monooxygenase','2,5-diketocamphane 1,2-monooxygenase oxygenating component','FMN; Iron.','A Baeyer-Villiger monooxygenase isolated from camphor-grown strainsof Pseudomonas putida and encoded on the cam plasmid.Involved in the degradation of (+)-camphor.Requires a dedicated NADH-FMN reductase (cf. EC 1.5.1.42).Can accept several bicyclic ketones including (+)- and (-)-camphorand adamantanone.The product spontaneously converts to ((1R)-5-oxo-2,2,3-trimethylcyclopent-3-enyl)acetate.Formerly EC 1.14.15.2.','(+)-bornane-2,5-dione + O(2) + NADH = (+)-5-oxo-1,2-campholide + NAD(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.163','6-hydroxy-3-succinoylpyridine 3-monooxygenase','6-hydroxy-3-succinoylpyridine hydroxylase','FAD.','The enzyme catalyzes a reaction in the nicotine degradation pathwayof Pseudomonas species.','4-(6-hydroxypyridin-3-yl)-4-oxobutanoate + 2 NADH + O(2) = 2,5-dihydroxypyridine + succinate semialdehyde + 2 NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.164','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.165','Nitric-oxide synthase (NAD(P)H-dependent)','Nitric oxide synthetase','5,6,7,8-tetrahydrobiopterin; Ferriprotoporphyrin IX.','Most of the bacterial and archaeal enzymes consist of only an oxidasedomain and function together with bacterial ferredoxins.The enzyme from the delta-proteobacterium Sorangium cellulosum alsoincludes a reductase domain that binds FAD, FMN and a [2Fe-2S]cluster.The similar enzymes from plants and animals use only NADPH asacceptor (cf. EC 1.14.13.39).','2 L-arginine + 3 NAD(P)H + 4 O(2) = 2 L-citrulline + 2 nitric oxide +3 NAD(P)(+) + 4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.166','4-nitrocatechol 4-monooxygenase','NULL','FAD.','The enzyme catalyzes the oxidation of 4-nitrocatechol with theconcomitant removal of the nitro group as nitrite.Forms a two-component system with a flavoprotein reductase.The enzymes from the bacteria Lysinibacillus sphaericus JS905 andRhodococcus sp. strain PN1 were shown to also catalyze EC 1.14.13.29,while the enzyme from Pseudomonas sp. WBC-3 was shown to alsocatalyze EC 1.14.13.167.','4-nitrocatechol + NAD(P)H + O(2) = 2-hydroxy-1,4-benzoquinone + nitrite +NAD(P)(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.167','4-nitrophenol 4-monooxygenase','NULL','FAD.','The enzyme catalyzes the first step in a degradation pathway for4-nitrophenol, the oxidation of 4-nitrophenol at position 4 with theconcomitant removal of the nitro group as nitrite.The enzyme from the bacterium Pseudomonas sp. WBC-3 also catalyzesEC 1.14.13.166.','4-nitrophenol + NADPH + O(2) = 1,4-benzoquinone + nitrite + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.168','Indole-3-pyruvate monooxygenase','NULL','NULL','This plant enzyme, along with EC 2.6.1.99, is responsible for thebiosynthesis of the plant hormone indole-3-acetate from L-tryptophan.','(Indol-3-yl)pyruvate + NADPH + O(2) = (indol-3-yl)acetate + NADP(+) +H(2)O + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.169','Sphinganine C(4)-monooxygenase','Sphingolipid C4-hydroxylase','NULL','The enzyme is involved in the biosynthesis of sphingolipids in yeastand plants.','Sphinganine + NADPH + O(2) = phytosphingosine + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.170','Pentalenolactone D synthase','NULL','FAD.','Isolated from the bacteria Streptomyces exfoliatus and Streptomycesarenae.The ketone undergoes a biological Baeyer-Villiger reaction.Part of the pathway of pentalenolactone biosynthesis.','1-deoxy-11-oxopentalenate + NADPH + O(2) = pentalenolactone D + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.171','Neopentalenolactone D synthase','NULL','FAD.','Isolated from the bacterium Streptomyces avermitilis.The ketone undergoes a biological Baeyer-Villiger reaction.','1-deoxy-11-oxopentalenate + NADPH + O(2) = neopentalenolactone D +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.172','Salicylate 5-hydroxylase','NULL','NULL','This enzyme, which was characterized from the bacterium Ralstonia sp.U2, comprises a multicomponent system, containing a reductase that isan iron-sulfur flavoprotein (FAD; EC 1.18.1.7), an iron-sulfuroxygenase, and ferredoxin.','Salicylate + NADH + O(2) = 2,5-dihydroxybenzoate + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.173','11-oxo-beta-amyrin 30-oxidase','CYP72A','Heme-thiolate.','The enzyme is involved in the biosynthesis of the triterpenoidsaponin glycyrrhizin in the plant Glycyrrhiza uralensis (licorice).The enzyme from the plant Medicago truncatula can also hydroxylatebeta-amyrin.','11-oxo-beta-amyrin + 3 O(2) + 3 NADPH = glycyrrhetinate + 3 NADP(+) +4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.174','Averantin hydroxylase','AVN hydroxylase','Heme-thiolate.','A P450 monooxygenase isolated from Aspergillus parasiticus.There is no reaction with (1R)-averantin.Involved in aflatoxin biosynthesis.','(1) (1S)-averantin + NADPH + O(2) = (1S,5S)-5-hydroxyaverantin +NADP(+) + H(2)O.(2) (1S)-averantin + NADPH + O(2) = (1S,5R)-5-hydroxyaverantin +NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.175','Aflatoxin B synthase','NULL','Heme-thiolate.','Isolated from the mold Aspergillus parasiticus.','(1) 8-O-methylsterigmatocystin + 2 NADPH + 2 O(2) = aflatoxin B(1) +2 NADP(+) + H(2)O + methanol + CO(2).(2) 8-O-methyldihydrosterigmatocystin + 2 NADPH + 2 O(2) = aflatoxin B(2)+ 2 NADP(+) + H(2)O + methanol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.176','Tryprostatin B 6-hydroxylase','NULL','Heme-thiolate.','Involved in the biosynthetic pathways of several indole alkaloidssuch as tryprostatins, fumitremorgins and verruculogen.','Tryprostatin B + NADPH + O(2) = 6-hydroxytryprostatin B + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.177','Fumitremorgin C monooxygenase','NULL','NULL','Involved in the biosynthetic pathway of the indole alkaloidverruculogen.','Fumitremorgin C + 2 NADPH + 2 O(2) = 12-alpha,13-alpha-dihydroxyfumitremorgin C + 2 NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.178','Methylxanthine N(1)-demethylase','NULL','Iron.','The enzyme from the bacterium Pseudomonas putida shares anNAD(P)H-FMN reductase subunit with EC 1.14.13.179, and has a 5-foldhigher activity with NADH than with NADPH.Also demethylate 1-methylxantine with lower efficiency.Forms part of the degradation pathway of methylxanthines.','(1) Caffeine + O(2) + NAD(P)H = theobromine + NAD(P)(+) + H(2)O +formaldehyde.(2) Theophylline + O(2) + NAD(P)H = 3-methylxanthine + NAD(P)(+) + H(2)O+ formaldehyde.(3) Paraxanthine + O(2) + NAD(P)H = 7-methylxanthine + NAD(P)(+) + H(2)O+ formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.179','Methylxanthine N(3)-demethylase','NULL','Iron.','The enzyme from the bacterium Pseudomonas putida shares anNAD(P)H-FMN reductase subunit with EC 1.14.13.178, and has higheractivity with NADH than with NADPH.Also demethylates caffeine and theophylline with lower efficiency.Forms part of the degradation pathway of methylxanthines.','(1) Theobromine + O(2) + NAD(P)H = 7-methylxanthine + NAD(P)(+) + H(2)O +formaldehyde.(2) 3-methylxanthine + O(2) + NAD(P)H = xanthine + NAD(P)(+) + H(2)O +formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.180','Aklavinone 12-hydroxylase','Aklavinone 11-hydroxylase','FAD.','The enzymes from the Gram-positive bacteria Streptomyces peucetiusand Streptomyces purpurascens participate in the biosynthesis ofdaunorubicin, doxorubicin and rhodomycins.','Aklavinone + NADPH + O(2) = epsilon-rhodomycinone + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.181','13-deoxydaunorubicin hydroxylase','NULL','NULL','The enzymes from the Gram-positive bacteria Streptomyces sp. C5 andStreptomyces peucetius show broad substrate specificity forstructures based on an anthracycline aglycone, but have a strongpreference for 4-methoxy anthracycline intermediates(13-deoxydaunorubicin and 13-dihydrodaunorubicin) over their4-hydroxy analogs (13-deoxycarminomycin and 13-dihydrocarminomycin),as well as a preference for substrates hydroxylated at the C-13rather than the C-14 position.','(1) 13-deoxydaunorubicin + NADPH + O(2) = 13-dihydrodaunorubicin +NADP(+) + H(2)O.(2) 13-dihydrodaunorubicin + NADPH + O(2) = daunorubicin + NADP(+) +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.182','2-heptyl-3-hydroxy-4(1H)-quinolone synthase','2-heptyl-3,4-dihydroxyquinoline synthase','NULL','The enzyme from the bacterium Pseudomonas aeruginosa catalyzes theterminal step in biosynthesis of the signal molecule 2-heptyl-3,4-dihydroxyquinoline that plays a role in regulation of virulencegenes.','2-heptyl-4(1H)-quinolone + NADH + O(2) = 2-heptyl-3-hydroxy-4(1H)-quinolone + NAD(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.183','Dammarenediol 12-hydroxylase','CYP716A47','NULL','Involved in the biosynthetic pathway of ginsenosides.Isolated from ginseng (Panax ginseng).','Dammarenediol-II + NADPH + O(2) = protopanaxadiol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.184','Protopanaxadiol 6-hydroxylase','CYP716A53v2','Heme-thiolate.','Involved in the biosynthetic pathway of ginsenosides.Isolated from the rhizomes of ginseng (Panax ginseng).','Protopanaxadiol + NADPH + O(2) = protopanaxatriol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.185','Pikromycin synthase','CYP107L1','Heme-thiolate.','Involved in the biosynthesis of a number of bacterial macrolideantibiotics containing a desosamine glycoside unit.With 10-deoxymethymycin it hydroxylates at either C-10 to givemethymycin or C-12 to give neomethymycin.','(1) Narbomycin + NADPH + O(2) = pikromycin + NADP(+) + H(2)O.(2) Narbomycin + NADPH + O(2) = neopikromycin + NADP(+) + H(2)O.(3) Narbomycin + 2 NADPH + 2 O(2) = novapikromyin + 2 NADP(+) + 2 H(2)O.(4) 10-deoxymethymycin + NADPH + O(2) = methymycin + NADP(+) + H(2)O.(5) 10-deoxymethymycin + NADPH + O(2) = neomethymycin + NADP(+) + H(2)O.(6) 10-deoxymethymycin + 2 NADPH + 2 O(2) = novamethymycin + 2 NADP(+) +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.186','20-oxo-5-O-mycaminosyltylactone 23-monooxygenase','NULL','Heme-thiolate.','Involved in the biosynthetic pathway of the macrolide antibiotictylosin, which is produced by several species of Streptomycesbacteria.','20-oxo-5-O-beta-mycaminosyltylactone + NADPH + O(2) = 5-O-beta-mycaminosyltylonolide + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.187','L-evernosamine nitrososynthase','NULL','FAD.','Isolated from the bacterium Micromonospora carbonacea var. africana.The nitroso group is probably spontaneously oxidized to a nitro groupgiving dTDP-beta-L-evernitrose, which is involved in the biosynthesisof the antibiotic everninomycin.The reaction was studied using dTDP-beta-L-4-epi-vancosamine (dTDP-4-O-desmethyl-beta-L-evernitrosamine).','dTDP-beta-L-evernosamine + 2 NADPH + 2 O(2) = dTDP-2,3,6-trideoxy-3-C-methyl-4-O-methyl-3-nitroso-beta-L-arabino-hexopyranose + 2 NADP(+) +3 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.188','6-deoxyerythronolide B hydroxylase','CYP107A1','Heme-thiolate.','Isolated from the bacterium Saccharopolyspora erythraea.The enzyme is involved in the biosynthesis of the antibioticerythromycin.','6-deoxyerythronolide B + NADPH + O(2) = erythronolide B + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.n2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.n3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.n4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.n5','Dihomomethionine N-hydroxylase','CYP79F1','Heme-thiolate.','The enzyme also catalyzes the N-hydroxylation of longer chainelongated methionines up to hexahomomethionine.','(1) Dihomomethionine + 2 O(2) + 2 NADPH = 5-methylthiopentanaldoxime +3 H(2)O + CO(2) + 2 NADP(+).(2) Trihomomethionine + 2 O(2) + 2 NADPH = 6-methylthiohexanaldoxime +3 H(2)O + CO(2) + 2 NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.n6','Hexahomomethionine N-hydroxylase','NULL','Heme-thiolate.','NULL','(1) Hexahomomethionine + 2 O(2) + 2 NADPH = 9-methylthiononanaldoxime +3 H(2)O + CO(2) + NADP(+).(2) Pentahomomethionine + 2 O(2) + 2 NADPH = 8-methylthiooctanaldoxime +3 H(2)O + CO(2) + NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.n7','4-nitrophenol 2-hydroxylase','CYP2E1','Heme-thiolate.','NULL','4-nitrophenol + NADPH + O(2) = 4-nitrocatechol + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.n8','2-methylbutanal oxime monooxygenase','CYP71E7','NULL','Part of the biosynthesis pathway of cyanogenic glucosides in theplant  Manihot esculenta (Cassava).','(Z)-2-methylbutanal oxime + NADPH + O(2) = 2-hydroxy-2-methylbutyronitrile + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.13.n9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.1','Unspecific monooxygenase','Aryl hydrocarbon hydroxylase','Heme-thiolate.','Acts on a wide range of substrates including many xenobiotics,steroids, fatty acids, vitamins and prostaglandins.Reactions catalyzed include hydroxylation, epoxidation, N-oxidation,sulfooxidation, N-, S- and O-dealkylations, desulfation, deamination,and reduction of azo, nitro, and N-oxide groups.Formerly EC 1.14.1.1, EC 1.14.14.2, EC 1.14.99.8 and EC 1.99.1.1.','RH + reduced flavoprotein + O(2) = ROH + oxidized flavoprotein + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.3','Alkanal monooxygenase (FMN-linked)','Aldehyde monooxygenase','NULL','The reaction involves incorporation of a molecule of oxygen intoreduced FMN, and subsequent reaction with the aldehyde to form anactivated FMN.H(2)O complex which breaks down with emission of light.The enzyme is highly specific for reduced FMN, and for long-chainaliphatic aldehydes with 8 carbons or more.','RCHO + reduced FMN + O(2) = RCOOH + FMN + H(2)O + light.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.5','Alkanesulfonate monooxygenase','FMNH(2)-dependent aliphatic sulfonate monooxygenase','NULL','The enzyme from Escherichia coli catalyzes the desulfonation of awide range of aliphatic sulfonates (unsubstituted C(1)- to C(14)-sulfonates as well as substituted C(2)-sulfonates).Does not desulfonate taurine (2-aminoethanesulfonate) or aromaticsulfonates.Does not use FMN as a bound cofactor.Instead, it uses reduced FMN (i.e., FMNH(2)) as a substrate.FMNH(2) is provided by SsuE, the associated FMN reductase(EC 1.5.1.38).','An alkanesulfonate (R-CH(2)-SO(3)H) + FMNH(2) + O(2) = an aldehyde(R-CHO) + FMN + sulfite + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.7','Tryptophan 7-halogenase','NULL','NULL','In Lechevalieria aerocolonigenes the enzyme catalyzes the initialstep in the biosynthesis of rebeccamycin.','Tryptophan + FADH(2) + Cl(-) + O(2) + H(+) = 7-chloro-L-tryptophan +FAD + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.8','Anthranilate 3-monooxygenase (FAD)','Anthranilate 3-hydroxylase','NULL','This enzyme, isolated from the bacterium Geobacillusthermodenitrificans, participates in the pathway of tryptophandegradation.The enzyme is part of a system that also includes a bifunctionalriboflavin kinase/FMN adenylyltransferase and an FAD reductase, whichensures ample supply of FAD to the monooxygenase.','Anthranilate + FADH(2) + O(2) = 3-hydroxyanthranilate + FAD + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.9','4-hydroxyphenylacetate 3-monooxygenase','4 HPA 3-hydroxylase','FAD.','The enzyme from Escherichia coli attacks a broad spectrum of phenoliccompounds.The enzyme uses FADH(2) as a substrate rather than a cofactor.FADH(2) is provided by EC 1.5.1.36.Formerly EC 1.14.13.3.','4-hydroxyphenylacetate + FADH(2) + O(2) = 3,4-dihydroxyphenylacetate +FAD + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.10','Nitrilotriacetate monooxygenase','NULL','Magnesium.','The enzyme from Aminobacter aminovorans (previously Chelatobacterheintzii) is part of a two component system that also includesEC 1.5.1.42 (FMN reductase), which provides reduced flavinmononucleotide for this enzyme.','Nitrilotriacetate + FMNH(2) + H(+) + O(2) = iminodiacetate + glyoxylate +FMN + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.11','Styrene monooxygenase','NULL','NULL','The enzyme catalyzes the first step in the aerobic styrenedegradation pathway.It forms a two-component system with a reductase (StyB) that utilizesNADH to reduce FAD, which is then transferred to the oxygenase.','Styrene + FADH(2) + O(2) = (S)-2-phenyloxirane + FAD + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.12','3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione monooxygenase','NULL','NULL','This bacterial enzyme participates in the degradation of severalsteroids, including cholesterol and testosterone.It can use either FADH or FMNH(2) as flavin cofactor.The enzyme forms a two-component system with a reductase (HsaB) thatutilizes NADH to reduce the flavin, which is then transferred to theoxygenase subunit.','3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + FMNH(2) + O(2)= 3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + FMN +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.13','4-(gamma-L-glutamylamino)butanoyl-[BtrI acyl-carrier protein]monooxygenase','NULL','FMNH(2).','Catalyzes a step in the biosynthesis of the side chain of theaminoglycoside antibiotics of the butirosin family.Forms a complex with a dedicated NAD(P)H:FMN oxidoreductase.The enzyme is not able to hydroxylate free substrates, activation bythe acyl-carrier protein is mandatory.Octanoyl-S-[BtrI acyl-carrier protein] is also accepted.','4-(gamma-L-glutamylamino)butanoyl-[BtrI acyl-carrier protein] + FMNH(2) +O(2) = 4-(gamma-L-glutamylamino)-(2S)-2-hydroxybutanoyl-[BtrI acyl-carrier protein] + FMN + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.14','Aromatase','CYP19A1','NULL','The enzyme catalyzes three sequential hydroxylations of the androgensandrost-4-ene-3,17-dione and testosterone, resulting in theiraromatization and forming the estrogens estrone and 17-beta-estradiol, respectively.','(1) Testosterone + 3 O(2) + 3 reduced flavoproteins = 17-beta-estradiol +formate + 4 H(2)O + 3 oxidized flavoproteins.(2) Androst-4-ene-3,17-dione + 3 O(2) + 3 reduced flavoproteins = estrone+ formate + 4 H(2)O + 3 oxidized flavoproteins.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.14.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.1','Camphor 5-monooxygenase','Camphor 5-exo-methylene hydroxylase','Heme-thiolate.','Also acts on (-)-camphor and 1,2-campholide, forming 5-exo-hydroxy-1,2-campholide.','(+)-camphor + reduced putidaredoxin + O(2) = (+)-exo-5-hydroxycamphor +oxidized putidaredoxin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.3','Alkane 1-monooxygenase','Alkane 1-hydroxylase','NULL','Some enzyme of this group are heme-thiolated proteins (P450).Also hydroxylates fatty acids in the omega-position.','Octane + reduced rubredoxin + O(2) = 1-octanol + oxidized rubredoxin +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.4','Steroid 11-beta-monooxygenase','Cytochrome p450 XIB1','Heme-thiolate.','Also hydroxylates steroids at the 18-position, and converts18-hydroxycorticosterone into aldosterone.Formerly EC 1.14.1.6 and EC 1.99.1.7.','A steroid + reduced adrenodoxin + O(2) = an 11-beta-hydroxysteroid +oxidized adrenodoxin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.5','Corticosterone 18-monooxygenase','Corticosterone 18-hydroxylase','Heme-thiolate.','NULL','Corticosterone + reduced adrenodoxin + O(2) = 18-hydroxycorticosterone +oxidized adrenodoxin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.6','Cholesterol monooxygenase (side-chain-cleaving)','Cholesterol 20-22-desmolase','Heme-thiolate.','The reaction proceeds in three stages, with two hydroxylations atC-22 and C-20 preceding scission of the side-chain between carbons 20and 22.The initial source of the electrons is NADPH, which transfers theelectrons to the adrenodoxin via EC 1.18.1.6.','Cholesterol + 6 reduced adrenodoxin + 3 O(2) = pregnenolone +4-methylpentanal + 6 oxidized adrenodoxin + 4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.7','Choline monooxygenase','NULL','Iron-sulfur; Magnesium.','Catalyzes the first step of glycine betaine synthesis.In many bacteria, plants and animals, the osmoprotectant betaine issynthesized in two steps: (1) choline to betaine aldehyde and (2)betaine aldehyde to betaine.Different enzymes are involved in the first reaction.In plants, the reaction is catalyzed by this enzyme whereas inanimals and many bacteria, it is catalyzed by either membrane-boundEC 1.1.99.1 or soluble EC 1.1.3.17.The enzyme involved in the second step, EC 1.2.1.8, appears to be thesame in plants, animals and bacteria.In some bacteria, betaine is synthesized from glycine through theactions of EC 2.1.1.156 and EC 2.1.1.157.Formerly EC 1.14.14.4.','Choline + O(2) + 2 reduced ferredoxin + 2 H(+) = betaine aldehyde hydrate+ H(2)O + 2 oxidized ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.8','Steroid 15-beta-monooxygenase','CYP106A2','NULL','The enzyme from Bacillus megaterium hydroxylates a variety of 3-oxo-Delta-4-steroids in position 15-beta.Ring A-reduced, aromatic, and 3-beta-hydroxy-Delta-4-steroids do notserve as substrates.','Progesterone + reduced ferredoxin + O(2) = 15-beta-hydroxyprogesterone +oxidized ferredoxin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.9','Spheroidene monooxygenase','2-oxo-spirilloxanthin monooxygenase','Heme.','The enzyme is involved in spheroidenone biosynthesis and in2,2-dioxospirilloxanthin biosynthesis.The enzyme from Rhodobacter sphaeroides contains heme at its activesite.','(1) Spheroidene + reduced ferredoxin + O(2) = spheroiden-2-one + oxidizedferredoxin + H(2)O.(2) Spirilloxantin + reduced ferredoxin + O(2) = 2-oxospirilloxanthin +oxidized ferredoxin + H(2)O.(3) 2-oxospirilloxanthin + reduced ferredoxin + O(2) =2,2-dioxospirilloxanthin + oxidized ferredoxin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.10','(+)-camphor 6-endo-hydroxylase','P450(camr)','NULL','A cytochrome P-450 monooxygenase from the bacterium Rhodococcus sp.NCIMB 9784.','(+)-camphor + reduced putidaredoxin + O(2) = (+)-6-endo-hydroxycamphor +oxidized putidaredoxin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.11','Pentalenic acid synthase','CYP105D7','Heme-thiolate.','Isolated from the bacterium Streptomyces avermitilis.The product, pentalenate, is a co-metabolite from pentalenolactonebiosynthesis.','1-deoxypentalenate + reduced ferredoxin + O(2) = pentalenate + oxidizedferredoxin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.12','Pimeloyl-[acyl-carrier protein] synthase','CYP107H1','Heme-thiolate.','The enzyme catalyzes an oxidative C-C bond cleavage of long-chainacyl-[acyl-carrier protein]s of various lengths to generate pimeloyl-[acyl-carrier protein], an intermediate in the biosynthesis ofbiotin.The preferred substrate of the enzyme from the bacterium Bacillussubtilis is palmitoyl-[acyl-carrier protein] which then givesheptanal as the alkanal.The mechanism is similar to EC 1.14.15.6, followed by a hydroxylationstep, which may occur spontaneously.','A long-chain acyl-[acyl-carrier protein] + 2 reduced flavodoxin + 3 O(2)= pimeloyl-[acyl-carrier protein] + an n-alkanal + 2 oxidized flavodoxin+ 3 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.15.13','Pulcherriminic acid synthase','Cyclo-L-leucyl-L-leucyl dipeptide oxidase','Heme-thiolate.','A P450 enzyme from the bacterium Bacillus subtilis.The order of events during the overall reaction is unknown.Pulcherrimic acid spontaneously forms an iron chelate with Fe(3+) toform the red pigment pulcherrimin.','Cyclo(L-leucyl-L-leucyl) + 6 reduced ferredoxin + 3 O(2) = pulcherriminicacid + 6 oxidized ferredoxin + 4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.16.1','Phenylalanine 4-monooxygenase','PAH','Iron.','The reaction involves an arene oxide which rearranges to give thephenolic hydroxy group.This results in the hydrogen at C-4 migrating to C-3 and in partbeing retained, a process known as the NIH-shift.The 4a-hydroxytetrahydrobiopterin formed can dehydrate to 6,7-dihydrobiopterin, both spontaneously and by the action ofEC 4.2.1.96.The 6,7-dihydrobiopterin can be enzymically reduced back totetrahydrobiopterin, by EC 1.5.1.34, or slowly rearranges into themore stable compound 7,8-dihydrobiopterin.Formerly EC 1.14.3.1 and EC 1.99.1.2.','L-phenylalanine + tetrahydrobiopterin + O(2) = L-tyrosine +4a-hydroxytetrahydrobiopterin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.16.2','Tyrosine 3-monooxygenase','L-tyrosine hydroxylase','Iron.','Activated by phosphorylation, catalyzed by EC 2.7.11.27.The 4a-hydroxytetrahydrobiopterin formed can dehydrate to 6,7-dihydrobiopterin, both spontaneously and by the action ofEC 4.2.1.96.The 6,7-dihydrobiopterin can be enzymically reduced back totetrahydrobiopterin, by EC 1.5.1.34, or slowly rearranges into themore stable compound 7,8-dihydrobiopterin.','L-tyrosine + tetrahydrobiopterin + O(2) = L-dopa +4a-hydroxytetrahydrobiopterin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.16.3','Anthranilate 3-monooxygenase','Anthranilate 3-hydroxylase','Iron.','NULL','Anthranilate + tetrahydrobiopterin + O(2) = 3-hydroxyanthranilate +dihydrobiopterin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.16.4','Tryptophan 5-monooxygenase','Indoleacetic acid-5-hydroxylase','Iron.','Activated by phosphorylation, catalyzed by a Ca(2+)-activated proteinkinase.The 4a-hydroxytetrahydrobiopterin formed can dehydrate to 6,7-dihydrobiopterin, both spontaneously and by the action ofEC 4.2.1.96.The 6,7-dihydrobiopterin can be enzymically reduced back totetrahydrobiopterin, by EC 1.5.1.34, or slowly rearranges into themore stable compound 7,8-dihydrobiopterin.','L-tryptophan + tetrahydrobiopterin + O(2) = 5-hydroxy-L-tryptophan +4a-hydroxytetrahydrobiopterin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.16.5','Alkylglycerol monooxygenase','Glyceryl etherase','Glutathione.','The enzyme cleaves alkylglycerols, but does not cleavealkenylglycerols (plasmalogens).Requires phospholipids for full activity.The product spontaneously breaks down to form a fatty aldehyde andglycerol.Formerly EC 1.14.99.17.','1-alkyl-sn-glycerol + tetrahydrobiopterin + O(2) = 1-O-alkyl-sn-glycerol+ dihydrobiopterin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.16.6','Mandelate 4-monooxygenase','L-mandelate 4-hydroxylase','Iron.','NULL','(S)-2-hydroxy-2-phenylacetate + tetrahydrobiopterin + O(2) = (S)-4-hydroxymandelate + dihydrobiopterin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.17.1','Dopamine beta-monooxygenase','Dopamine beta-hydroxylase','Copper.','Stimulated by fumarate.Formerly EC 1.14.2.1.','3,4-dihydroxyphenethylamine + ascorbate + O(2) = noradrenaline +dehydroascorbate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.17.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.17.3','Peptidylglycine monooxygenase','PAM','Copper.','Peptidylglycines with a neutral amino acid residue in the penultimateposition are the best substrates for the enzyme.The product is unstable and dismutates to glyoxylate and thecorresponding desglycine peptide amide, a reaction catalyzed byEC 4.3.2.5.Involved in the final step of biosynthesis of alpha-melanotropin andrelated biologically active peptides.','Peptidylglycine + ascorbate + O(2) = peptidyl(2-hydroxyglycine) +dehydroascorbate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.17.4','Aminocyclopropanecarboxylate oxidase','1-aminocyclopropane-1-carboxylate oxidase','Iron.','Requires CO(2) for activity.In the enzyme from plants, the ethylene has signaling functions suchas stimulation of fruit-ripening.','1-aminocyclopropane-1-carboxylate + ascorbate + O(2) = ethylene + cyanide+ dehydroascorbate + CO(2) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.18.1','Tyrosinase','Cresolase','Copper.','Found in a broad variety of bacteria, fungi, plants, insects,crustaceans, and mammals, which is involved in the synthesis ofbetalains and melanin.The enzyme, which is activated upon binding molecular oxygen,can catalyze both a monophenolase reaction cycle or a diphenolasereaction cycle.During the monophenolase cycle, one of the bound oxygen atoms istransferred to a monophenol (such as L-tyrosine), generating anO-diphenol intermediate, which is subsequently oxidized to ano-quinone and released, along with a water molecule.The enzyme remains in an inactive deoxy state, and is restored to theactive oxy state by the binding of a new oxygen molecule.During the diphenolase cycle the enzyme binds an external diphenolmolecule (such as L-dopa) and oxidizes it to an O-quinone that isreleased along with a water molecule, leaving the enzyme in theintermediate met state.The enzyme then binds a second diphenol molecule and repeats theprocess, ending in a deoxy state.The second reaction is identical to that catalyzed by the relatedenzyme catechol oxidase (EC 1.10.3.1).However, the latter can not catalyze the hydroxylation ormonooxygenation of monophenols.Formerly EC 1.14.17.2.','(1) 2 L-dopa + O(2) = 2 dopaquinone + 2 H(2)O.(2) L-tyrosine + O(2) = dopaquinone + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.18.2','CMP-N-acetylneuraminate monooxygenase','CMP-N-acetylneuraminate hydroxylase','Iron; Iron-sulfur.','The ferricytochrome b5 produced is reduced by NADH and EC 1.6.2.2.The enzyme can be activated by Fe(2+) or Fe(3+).Formerly EC 1.14.13.45 and EC 1.14.99.18.','CMP-N-acetylneuraminate + 2 ferrocytochrome b5 + O(2) + 2 H(+) = CMP-N-glycoloylneuraminate + 2 ferricytochrome b5 + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.18.3','Methane monooxygenase (particulate)','NULL','Copper.','It is membrane-bound, in contrast to the soluble EC 1.14.13.25.','Methane + quinol + O(2) = methanol + quinone + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.1','Stearoyl-CoA 9-desaturase','Acyl-CoA desaturase','Iron.','The liver enzyme is an enzyme system involving cytochrome b5 andEC 1.6.2.2.The ferricytochrome b5 produced is reduced by NADH and EC 1.6.2.2.Formerly EC 1.14.99.5.','Stearoyl-CoA + 2 ferrocytochrome b5 + O(2) + 2 H(+) = oleoyl-CoA +2 ferricytochrome b5 + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.2','Acyl-[acyl-carrier-protein] desaturase','Stearyl acyl carrier protein desaturase','NULL','The enzyme from safflower is specific for stearoyl-[acyl-carrierprotein]; that from Euglena acts on derivatives of a number of long-chain fatty acids.Requires ferredoxin.Formerly EC 1.14.99.6.','Stearoyl-[acyl-carrier-protein] + reduced acceptor + O(2) = oleoyl-[acyl-carrier-protein] + acceptor + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.3','Linoleoyl-CoA desaturase','Delta(6)-acyl CoA desaturase','Iron.','The rat liver enzyme is an enzyme system involving cytochrome b5 andEC 1.6.2.2.Formerly EC 1.14.99.25.','Linoleoyl-CoA + AH(2) + O(2) = gamma-linolenoyl-CoA + A + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.4','Delta(8)-fatty-acid desaturase','Delta(8)-desaturase','NULL','This enzyme, which has been found mainly in plants, introduces adouble bond at Delta(8) of C(18) and C(20) fatty acids.The enzyme from the marine microalga Euglena gracilis requires adouble bond to be present at Delta(11) and is most active with 20:3Delta(11,14,17) and 20:2 Delta(11,14) as substrates, although it canalso desaturate 20:1 Delta(11).The Delta(8)-desaturation pathway represents an alternate pathway forthe synthesis of the polyunsaturated fatty acids arachidonate (C20:4Delta(5,14)) and eicosapentaenoate (C20:5 Delta(5,17)) in organismslacking a Delta(6)-desaturase.Formerly EC 1.14.19.n1.','Phytosphinganine + reduced acceptor + O(2) = Delta(8)-phytosphingenine +acceptor + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.5','Delta(11)-fatty-acid desaturase','Delta(11) desaturase','NULL','In common with front-end desaturases involved in the synthesis ofpolyunsaturated fatty acids (PUFAs), this membrane-bound enzyme has acytochrome b5-like domain at the N-terminus and contains threehistidine boxes that are critical for desaturase activity.The enzyme from the marine microalga Thalassiosira pseudonanaspecifically desaturates palmitic acid 16:0 to 16:1-Delta(11) whereasthat from the leafroller moth Choristoneura rosaceana desaturatesmyristic acid 14:0 to 14:1-Delta(11).16:1-Delta(11) represents an important precursor for pheromonesynthesis in insect cells, although its function in microalgae iscurrently unknown.The enzyme from the Egyptian cotton leafworm Spodoptera littoralishas a preference for palmitoyl-CoA as substrate and for NADH ratherthan NADPH as reduced acceptor.Formerly EC 1.14.19.n2.','Acyl-CoA + reduced acceptor + O(2) = Delta(11)-acyl-CoA + acceptor +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.6','Delta(12)-fatty-acid desaturase','Delta(12) desaturase','NULL','In the yeast Lipomyces starkeyi and in the American cockroachPeriplaneta americana, this microsomal enzyme converts oleoyl-CoAinto linoleoyl-CoA.In the moths Cadra cautella and Spodoptera, the enzyme converts (Z)-9-tetradecenoic acid into (9Z,12E)-tetradeca-9,12-dienoic acid, whichis reduced and acetylated to form the acetate ester pheromone.Formerly EC 1.14.19.n3.','Acyl-CoA + reduced acceptor + O(2) = Delta(12)-acyl-CoA + acceptor +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.7','(S)-2-hydroxypropylphosphonic acid epoxidase','(S)-2-hydroxypropylphosphonate epoxidase','Iron.','FMN is required to mediate the transfer of reducing equivalents fromNADH to the active site iron.This is the last enzyme in the biosynthetic pathway of fosfomycin,a broad-spectrum antibiotic produced by certain Streptomyces species.','(S)-2-hydroxypropylphosphonate + NADH + O(2) = (1R,2S)-epoxypropylphosphonate + 2 H(2)O + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.8','Pentalenolactone synthase','NULL','Heme-thiolate.','Isolated from the bacteria Streptomyces exfoliatus and Streptomycesarenae.Formerly EC 1.3.7.10.','Pentalenolactone F + O(2) + 2 reduced ferredoxin + 2 H(+) =pentalenolactone + 2 oxidized ferredoxin + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.n2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.n3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.n4','Stearoyl-CoA 9-desaturase','NULL','NULL','NULL','Stearoyl-CoA + NADPH + O(2) = oleoyl-CoA + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.19.n5','Versicolorin B desaturase','NULL','Heme-thiolate.','The enzyme is specific for NADP.Involved in the synthesis of aflatoxins in the fungus Aspergillusparasiticus.','Versicolorin B + NADPH + O(2) = versicolorin A + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.20.1','Deacetoxycephalosporin-C synthase','DAOC synthase','NULL','Forms part of the penicillin biosynthesis pathway.','Penicillin N + 2-oxoglutarate + O(2) = deacetoxycephalosporin C +succinate + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.20.2','2,4-dihydroxy-1,4-benzoxazin-3-one-glucoside dioxygenase','DIBOA-Glc dioxygenase','NULL','The enzyme is involved in the biosynthesis of protective andallelophatic benzoxazinoids in some plants, most commonly from thefamily of Poacea (grasses).','(2R)-4-hydroxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside + 2-oxoglutarate + O(2) = (2R)-4,7-dihydroxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside + succinate + CO(2)+ H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.20.3','(5R)-carbapenem-3-carboxylate synthase','NULL','Fe(2+).','The enzyme is involved in the biosynthesis of the carbapenem beta-lactam antibiotic (5R)-carbapen-2-em-3-carboxylate in the bacteriumPectobacterium carotovorum.It catalyzes a stereoinversion at C(5) and introduces a double bondbetween C-2 and C-3.','(3S,5S)-carbapenam-3-carboxylate + 2-oxoglutarate + O(2) = (5R)-carbapen-2-em-3-carboxylate + succinate + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.1','(S)-stylopine synthase','(S)-cheilanthifoline oxidase (methylenedioxy-bridge-forming)','Heme-thiolate.','Catalyzes an oxidative reaction that does not incorporate oxygen intothe product.Forms the second methylenedioxy bridge of the protoberberine alkaloidstylopine from oxidative ring closure of adjacent phenolic andmethoxy groups of cheilanthifoline.Formerly EC 1.1.3.32.','(S)-cheilanthifoline + NADPH + O(2) = (S)-stylopine + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.2','(S)-cheilanthifoline synthase','(S)-scoulerine oxidase (methylenedioxy-bridge-forming)','Heme-thiolate.','Catalyzes an oxidative reaction that does not incorporate oxygen intothe product.Forms the methylenedioxy bridge of the protoberberine alkaloidcheilanthifoline from oxidative ring closure of adjacent phenolic andmethoxy groups of scoulerine.Formerly EC 1.1.3.33.','(S)-scoulerine + NADPH + O(2) = (S)-cheilanthifoline + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.3','Berbamunine synthase','(S)-N-methylcoclaurine oxidase (C-O phenol-coupling)','Heme-thiolate.','Forms the bisbenzylisoquinoline alkaloid berbamunine by phenoloxidation of N-methylcoclaurine without the incorporation of oxygeninto the product.Reaction of 2 molecules of (R)-N-methylcoclaurine gives the dimerguattagaumerine.Formerly EC 1.1.3.34.','(S)-N-methylcoclaurine + (R)-N-methylcoclaurine + NADPH + O(2) =berbamunine + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.4','Salutaridine synthase','(R)-reticuline oxidase (C-C phenol-coupling)','Heme-thiolate.','Forms the morphinan alkaloid salutaridine by intramolecular phenoloxidation of reticuline without the incorporation of oxygen into theproduct.Formerly EC 1.1.3.35.','(R)-reticuline + NADPH + O(2) = salutaridine + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.5','(S)-canadine synthase','(S)-tetrahydroberberine synthase','Heme-thiolate.','Catalyzes an oxidative reaction that does not incorporate oxygen intothe product.Oxidation of the methoxyphenol group of the alkaloidtetrahydrocolumbamine results in the formation of the methylenedioxybridge of canadine.Formerly EC 1.1.3.36.','(S)-tetrahydrocolumbamine + NADPH + O(2) = (S)-canadine + NADP(+) +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.6','Lathosterol oxidase','5-DES','NULL','This enzyme catalyzes the introduction of a C5 double bond into the Bring of Delta(7)-sterols to yield the corresponding Delta(5,7)-sterols in mammals, yeast and plants.Forms part of the plant sterol biosynthesis pathway.Formerly EC 1.3.3.2.','5-alpha-cholest-7-en-3-beta-ol + NAD(P)H + O(2) = cholesta-5,7-dien-3-beta-ol + NAD(P)(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.7','Biflaviolin synthase','CYP158A2','NULL','This cytochrome-P450 enzyme, from the soil-dwelling bacteriumStreptomyces coelicolor A3(2), catalyzes a phenol oxidationC-C coupling reaction, which results in the polymerization offlaviolin to form biflaviolin or triflaviolin without theincorporation of oxygen into the product.The products are highly conjugated pigments that protect thebacterium from the deleterious effects of UV irradiation.','(1) 2 flaviolin + NADPH + O(2) = 3,3-biflaviolin + NADP(+) + 2 H(2)O.(2) 2 flaviolin + NADPH + O(2) = 3,8-biflaviolin + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.8','Pseudobaptigenin synthase','NULL','Heme-thiolate.','Catalyzes an oxidative reaction that does not incorporate oxygen intothe product.Catalyzes a step in the biosynthesis of (-)-maackiain, the mainpterocarpan phytoalexin in chickpea (Cicer arietinum).','(1) Calycosin + NADPH + O(2) = pseudobaptigenin + NADP(+) + 2 H(2)O.(2) Pratensein + NADPH + O(2) = 5-hydroxypseudobaptigenin + NADP(+) +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.9','Mycocyclosin synthase','CYP121','Heme-thiolate.','A P450 enzyme from the bacterium Mycobacterium tuberculosiscatalyzing an oxidative reaction that does not incorporate oxygeninto the product.','Cyclo(L-tyrosyl-L-tyrosyl) + NADPH + O(2) = mycocyclosin + NADP(+) +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.21.10','Fumitremorgin C synthase','NULL','Heme-thiolate.','The protein from the fungus Aspergillus fumigatus also has activitywith tryprostatin B forming demethoxyfumitremorgin C.Involved in the biosynthetic pathways of several indole alkaloidssuch as fumitremorgins and verruculogen.','Tryprostatin A + NADPH + O(2) = fumitremorgin C + NADP(+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.1','Prostaglandin-endoperoxide synthase','PG synthetase','NULL','Acts both as a dioxygenase and as a peroxidase.','Arachidonate + AH(2) + 2 O(2) = prostaglandin H(2) + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.2','Kynurenine 7,8-hydroxylase','NULL','NULL','Formerly EC 1.14.1.4.','Kynurenate + AH(2) + O(2) = 7,8-dihydro-7,8-dihydroxykynurenate + A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.3','Heme oxygenase (biliverdin-producing)','Heme oxidase','NULL','Requires NAD(P)H and EC 1.6.2.4.The terminal oxygen atoms that are incorporated into the carbonylgroups of pyrrole rings A and B of biliverdin are derived from twoseparate oxygen molecules.The third oxygen molecule provides the oxygen atom that converts thealpha-carbon to CO.The central iron is kept in the reduced state by NAD(P)H.','Protoheme + 3 AH(2) + 3 O(2) = biliverdin + Fe(2+) + CO + 3 A + 3 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.4','Progesterone monooxygenase','Progesterone hydroxylase','NULL','Has a wide specificity.A single enzyme from Cylindrocarpon radicicola (EC 1.14.13.54)catalyzes both this reaction and that catalyzed by EC 1.14.99.12.','Progesterone + AH(2) + O(2) = testosterone acetate + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.8','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.9','Steroid 17-alpha-monooxygenase','CYP17','Heme-thiolate.','Requires NADPH and EC 1.6.2.4.Catalyzes two independent reactions at the same active site - the17-alpha-hydroxylation of pregnenolone and progesterone, which ispart of glucocorticoid hormones biosynthesis, and the conversion ofthe 17-alpha-hydroxylated products via a 17,20-lyase reaction to formandrostenedione and dehydroepiandrosterone, leading to sex hormonebiosynthesis (EC 4.1.2.30).The ratio of the 17-alpha-hydroxylase and 17,20-lyase activities isan important factor in determining the directions of steroid hormonebiosynthesis toward biosynthesis of glucocorticoid or sex hormones.Formerly EC 1.14.1.7 and EC 1.99.1.9.','A C(21)-steroid + (reduced NADPH--hemoprotein reductase) + O(2) =a 17-alpha-hydroxy-C(21)-steroid + (oxidized NADPH--hemoproteinreductase) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.10','Steroid 21-monooxygenase','CYP21A2','Heme-thiolate.','Requires NADPH and EC 1.6.2.4.Responsible for the conversion of progesterone and 17-alpha-hydroxyprogesterone to their respective 21-hydroxylated derivatives,11-deoxycorticosterone and 11-deoxycortisol.Involved in the biosynthesis of the hormones aldosterone andcortisol.Formerly EC 1.14.1.8 and EC 1.99.1.11.','A C(21) steroid + (reduced NADPH--hemoprotein reductase) + O(2) =a 21-hydroxy-C(21)-steroid + (oxidized NADPH--hemoprotein reductase) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.11','Estradiol 6-beta-monooxygenase','Estradiol 6-beta-hydroxylase','NULL','Formerly EC 1.14.1.10.','Estradiol-17-beta + AH(2) + O(2) = 6-beta-hydroxyestradiol-17-beta + A +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.12','Androst-4-ene-3,17-dione monooxygenase','4-androstene-3,17-dione monooxygenase','NULL','Has a wide specificity.A single enzyme from Cylindrocarpon radicicola (EC 1.14.13.54)catalyzes both this reaction and that catalyzed by EC 1.14.99.4.','Androstenedione + AH(2) + O(2) = testololactone + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.13','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.14','Progesterone 11-alpha-monooxygenase','Progesterone 11-alpha-hydroxylase','NULL','NULL','Progesterone + AH(2) + O(2) = 11-alpha-hydroxyprogesterone + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.15','4-methoxybenzoate monooxygenase (O-demethylating)','4-methoxybenzoate O-demethylase','Flavoprotein; Iron-sulfur.','The bacterial enzyme consists of a ferredoxin-type protein and aniron-sulfur flavoprotein (FMN).Also acts on 4-ethoxybenzoate, N-methyl-4-aminobenzoate and toluate.The fungal enzyme acts best on veratrate.','4-methoxybenzoate + AH(2) + O(2) = 4-hydroxybenzoate + formaldehyde + A +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.16','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.17','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.18','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.19','Plasmanylethanolamine desaturase','Alkylacylglycerophosphoethanolamine desaturase','Magnesium.','Either NADPH or NADH can act as AH(2).May involve cytochrome b5.Requires ATP.','O-1-alkyl-2-acyl-sn-glycero-3-phosphoethanolamine + AH(2) + O(2) = O-1-alk-1-enyl-2-acyl-sn-glycero-3-phosphoethanolamine + A + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.20','Phylloquinone monooxygenase (2,3-epoxidizing)','Phylloquinone epoxidase','NULL','NULL','Phylloquinone + AH(2) + O(2) = 2,3-epoxyphylloquinone + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.21','Latia-luciferin monooxygenase (demethylating)','NULL','Flavoprotein.','Latia luciferin is (E)-2-methyl-4-(2,6,6-trimethyl-1-cyclohex-1-yl)-1-buten-1-ol formate.The reaction possibly involves two enzymes, an oxygenase followed bya monooxygenase for the actual light-emitting step.','Latia luciferin + AH(2) + 2 O(2) = oxidized Latia luciferin + CO(2) +formate + A + H(2)O + light.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.22','Ecdysone 20-monooxygenase','NULL','Heme-thiolate.','An enzyme from insect fat body or malpighian tubules.NADPH can act as ultimate hydrogen donor.','Ecdysone + AH(2) + O(2) = 20-hydroxyecdysone + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.23','3-hydroxybenzoate 2-monooxygenase','3-hydroxybenzoate 2-hydroxylase','NULL','NULL','3-hydroxybenzoate + AH(2) + O(2) = 2,3-dihydroxybenzoate + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.24','Steroid 9-alpha-monooxygenase','Steroid 9-alpha-hydroxylase','FMN; Iron-sulfur.','An enzyme system involving a flavoprotein (FMN) and two iron-sulfurproteins.','Pregna-4,9(11)-diene-3,20-dione + AH(2) + O(2) = 9,11-alpha-epoxypregn-4-ene-3,20-dione + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.25','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.26','2-hydroxypyridine 5-monooxygenase','2-hydroxypyridine oxygenase','NULL','Also oxidizes 2,5-dihydroxypyridine, but does not act on3-hydroxypyridine, 4-hydroxypyridine or 2,6-dihydroxypyridine.','2-hydroxypyridine + AH(2) + O(2) = 2,5-dihydroxypyridine + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.27','Juglone 3-monooxygenase','NULL','NULL','Also acts on 1,4-naphthoquinone, naphthazarin and 2-chloro-1,4-naphthoquinone, but not on other related compounds.','5-hydroxy-1,4-naphthoquinone + AH(2) + O(2) = 3,5-dihydroxy-1,4-naphthoquinone + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.28','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.29','Deoxyhypusine monooxygenase','Deoxyhypusine dioxygenase','NULL','Catalyzes the final step in the formation of the amino acid hypusinein eukaryotic initiation factor 5A (eIF-5A) (formerly eIF-4D).','[eIF5A]-deoxyhypusine + AH(2) + O(2) = [eIF5A]-hypusine + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.30','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.31','Myristoyl-CoA 11-(E) desaturase','NULL','NULL','Involved in sex pheromone synthesis in Spodoptera littoralis.(E)-11-tetradecenoyl-CoA is formed by stereospecific removal of thepro-R H at C-11 and the pro-S H at C-12.EC 1.14.99.32 forms the Z-isomer by stereospecific cleavage of pro-RH at C-11 and C-12.','Myristoyl-CoA + NAD(P)H + O(2) = (E)-11-tetradecenoyl-CoA + NAD(P)(+) +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.32','Myristoyl-CoA 11-(Z) desaturase','NULL','NULL','Involved in sex pheromone synthesis in Spodoptera littoralis.(Z)-11-tetradecenoyl-CoA is formed by stereospecific removal of Hfrom the pro-R positions at C-11 and C-12.EC 1.14.99.31 forms the E-isomer by removing the pro-R H group atC-11 and the pro-S H group at C-12.','Myristoyl-CoA + NAD(P)H + O(2) = (Z)-11-tetradecenoyl-CoA + NAD(P)(+) +2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.33','Delta(12)-fatty acid dehydrogenase','Crepenynate synthase','Iron.','NULL','Linoleate + AH(2) + O(2) = crepenynate + A + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.34','Monoprenyl isoflavone epoxidase','Monoprenyl isoflavone monooxygenase','FAD.','The enzyme has a high specificity for monoprenyl isoflavone.The product of the prenyl epoxidation reaction contains an oxygenatom derived from O(2), but not from H(2)O.It is slowly and nonenzymically converted into the correspondingdihydrofurano derivative.','7-O-methylluteone + NADPH + O(2) = dihydrofurano derivatives + NADP(+) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.35','Thiophene-2-carbonyl-CoA monooxygenase','Thiophene-2-carboxyl-CoA dehydrogenase','Molybdenum.','Highly specific for thiophene-2-carbonyl-CoA.Tetrazolium salts can act as electron acceptors.','Thiophene-2-carbonyl-CoA + AH(2) + O(2) = 5-hydroxythiophene-2-carbonyl-CoA + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.36','Beta-carotene 15,15-monooxygenase','Beta-carotene 15,15-dioxygenase','Bile salt; Iron.','The reaction proceeds in three stages, epoxidation of the15,15-double bond, hydration of the double bond leading to ringopening, and oxidative cleavage of the diol formed (cf.EC 1.14.15.6).Thus only one atom of the dioxygen is incorporated into retinal.Formerly EC 1.13.11.21 and EC 1.18.3.1.','Beta-carotene + O(2) = 2 all-trans-retinal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.37','Taxadiene 5-alpha-hydroxylase','NULL','Heme-thiolate.','Involved in the biosynthesis of the diterpenoid antineoplastic drugTaxol (paclitaxel).The reaction includes rearrangement of the 4(5)-double bond to a4(20)-double bond, possibly through allylic oxidation.','Taxa-4,11-diene + AH(2) + O(2) = taxa-4(20),11-dien-5-alpha-ol + A +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.38','Cholesterol 25-hydroxylase','Cholesterol 25-monooxygenase','Iron.','Unlike most other sterol hydroxylases, this enzyme is not acytochrome P-450.Instead, it uses diiron cofactors to catalyze the hydroxylation ofhydrophobic substrates.The diiron cofactor can be either Fe-O-Fe or Fe-OH-Fe and is bound tothe enzyme through interactions with clustered histidine or glutamateresidues.In cell cultures, this enzyme down-regulates cholesterol synthesisand the processing of sterol regulatory element binding proteins(SREBPs).','Cholesterol + AH(2) + O(2) = 25-hydroxycholesterol + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.39','Ammonia monooxygenase','AMO','Copper; Iron.','The donor is membrane-bound.Electrons are derived indirectly from ubiquinol.','Ammonia + AH(2) + O(2) = NH(2)OH + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.40','5,6-dimethylbenzimidazole synthase','NULL','NULL','The C-2 of 5,6-dimethylbenzimidazole is derived from C-1 of theribityl group of FMNH(2) and 2-H from the ribityl 1-pro-(S)hydrogen.The other product may be barbituric acid.The stoichiometery of the reaction is not known.','FMNH(2) + NADH + O(2) = 5,6-dimethylbenzimidazole + D-erythrose4-phosphate + NAD(+) + other product.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.41','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.42','Zeaxanthin 7,8-dioxygenase','Zeaxanthin 7,8(7,8)-cleavage dioxygenase','NULL','Presumably the enzyme acts twice on zeaxanthin cleaving3-hydroxycyclocitral off each 3-hydroxy end group.Formerly EC 1.14.99.n3.','Zeaxanthin + 2 O(2) = crocetin dialdehyde + 2 (3S)-3-hydroxycyclocitral.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.43','Beta-amyrin 24-hydroxylase','CYP93E1','Heme-thiolate.','NULL','(1) Beta-amyrin + AH(2) + O(2) = 24-hydroxy-beta-amyrin + A + H(2)O.(2) Sophoradiol + AH(2) + O(2) = 24-hydroxysophoradiol + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.44','Diapolycopene oxygenase','NULL','NULL','Little activity with neurosporene or lycopene.Involved in the biosynthesis of C(30) carotenoids such asstaphyloxanthin.The enzyme oxidizes each methyl group to the hydroxymethyl and then adihydroxymethyl group, followed by the spontaneous loss of water togive an aldehyde group.','4,4-diapolycopene + 4 AH(2) + 4 O(2) = 4,4-diapolycopenedial + 4 A +6 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.45','Carotene epsilon-monooxygenase','CYP97C1','Heme-thiolate.','Also acts on zeinoxanthin to give lutein.','Alpha-carotene + O(2) + AH(2) = alpha-cryptoxanthin + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.46','Pyrimidine monooxygenase','NULL','NULL','In vitro the product (Z)-3-ureidoacrylate peracid is spontaneouslyreduced into ureidoacrylate.Part of the Rut pyrimidine catabolic pathway.Formerly EC 1.14.14.n1.','(1) Uracil + FMNH(2) + O(2) = (Z)-3-ureidoacrylate peracid + FMN + H(2)O.(2) Thymine + FMNH(2) + O(2) = (Z)-2-methylureidoacrylate peracid + FMN +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.47','(+)-larreatricin hydroxylase','NULL','NULL','Isolated from the plant Larrea tridentata (creosote bush).The enzyme has a strong preference for the 3 position of (+)-larreatricin.','(+)-larreatricin + O(2) + AH(2) = (+)-3-hydroxylarreatricin + A + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.48','Heme oxygenase (staphylobilin-producing)','Heme oxidase','NULL','This enzyme, which is found in pathogenic bacteria, is involved in aniron acquisition system that catabolizes the hosts hemoglobin.The two enzymes from the bacterium Staphylococcus aureus, encoded bythe isdG and isdI genes, produce 67.5 % and 56.2 % 5-oxo-delta-bilirubin, respectively.','(1) Protoheme + 4 AH(2) + 4 O(2) = 5-oxo-delta-bilirubin + Fe(2+) + CO +4 A + 4 H(2)O.(2) Protoheme + 4 AH(2) + 4 O(2) = 15-oxo-beta-bilirubin + Fe(2+) + CO +4 A + 4 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.n1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.n2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.n3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.n4','Carotenoid 9,10-dioxygenase','Carotenoid 9,10(9,10)-cleavage dioxygenase','NULL','Presumably the enzyme acts twice on zeaxanthin cleaving 3-hydroxy-beta-ionone off each 3-hydroxy end group.','Zeaxanthin + 2 O(2) = 4,9-dimethyl-dodeca-2,4,6,8,10-pentaen-dial +2 (3R)-3-hydroxy-beta-ionone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.14.99.n5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.15.1.1','Superoxide dismutase','NULL','Iron or manganese or (zinc and copper).','NULL','2 superoxide + 2 H(+) = O(2) + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.15.1.2','Superoxide reductase','Desulfoferrodoxin','Iron.','Formerly EC 1.18.96.1.','Reduced rubredoxin + superoxide + 2 H(+) = rubredoxin + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.1','Mercury(II) reductase','Mercuric reductase','NULL','NULL','Hg + NADP(+) + H(+) = Hg(2+) + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.2','Diferric-transferrin reductase','Transferrin reductase','NULL','NULL','Transferrin-[Fe(2+)](2) + NAD(+) = transferrin-[Fe(3+)](2) + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.3','Aquacobalamin reductase','Aquocobalamin reductase','Flavoprotein.','Formerly EC 1.6.99.8.','2 cob(II)alamin + NAD(+) = 2 aquacob(III)alamin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.4','Cob(II)alamin reductase','B(12r) reductase','Flavoprotein.','Formerly EC 1.6.99.9.','2 cob(I)alamin + NAD(+) = 2 cob(II)alamin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.5','Aquacobalamin reductase (NADPH)','Aquacobalamin (reduced nicotinamide adenine dinucleotide phosphate)reductase','Flavoprotein.','Acts on aquacob(III)alamin and hydroxycobalamin, but not oncyanocobalamin.Formerly EC 1.6.99.11.','2 cob(II)alamin + NADP(+) = 2 aquacob(III)alamin + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.6','Cyanocobalamin reductase (cyanide-eliminating)','Cyanocobalamin reductase','Flavoprotein.','Formerly EC 1.6.99.12.','Cob(I)alamin + cyanide + NADP(+) = cyanocob(III)alamin + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.7','Ferric-chelate reductase (NADH)','Ferric chelate reductase','FAD.','The enzyme catalyzes the reduction of bound ferric iron in a varietyof iron chelators (siderophores), resulting in the release of ferrousiron.The plant enzyme is involved in the transport of iron across plantplasma membranes.The enzyme from the bacterium Paracoccus denitrificans can alsoreduce chromate.Cf. EC 1.16.1.9 and EC 1.16.1.10.Formerly EC 1.6.99.13.','2 Fe(II)-siderophore + NAD(+) + H(+) = 2 Fe(III)-siderophore + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.8','[Methionine synthase] reductase','[Methionine synthase]-cobalamin methyltransferase (cob(II)alaminreducing)','FAD; FMN.','The substrate of the enzyme is the inactivated [Co(II)] form ofEC 2.1.1.13.Electrons are transferred from NADPH to FAD to FMN.Formerly EC 2.1.1.135.','2 [methionine synthase]-methylcob(I)alamin + 2 S-adenosylhomocysteine +NADP(+) = 2 [methionine synthase]-cob(II)alamin + NADPH + 2 S-adenosyl-L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.9','Ferric-chelate reductase (NADPH)','Ferric chelate reductase','FAD.','The enzyme, which is widespread among bacteria, catalyzes thereduction of ferric iron bound to a variety of iron chelators(siderophores), including ferric triscatecholates and ferricdicitrate, resulting in the release of ferrous iron.The enzyme from the bacterium Escherichia coli has the highestefficiency with the hydrolyzed ferric enterobactin complex ferricN-(2,3-dihydroxybenzoyl)-L-serine.Cf. EC 1.16.1.7 and EC 1.16.1.10.','2 Fe(II)-siderophore + NADP(+) + H(+) = 2 Fe(III)-siderophore + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.1.10','Ferric-chelate reductase (NAD(P)H)','Ferric reductase','Flavoprotein.','The enzyme catalyzes the reduction of bound ferric iron in a varietyof iron chelators (siderophores), resulting in the release of ferrousiron.The enzyme from the hyperthermophilic archaeon Archaeoglobus fulgidusis not active with uncomplexed Fe(III).Cf. EC 1.16.1.7 and EC 1.16.1.9.','2 Fe(II)-siderophore + NAD(P)(+) + H(+) = 2 Fe(III)-siderophore +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.3.1','Ferroxidase','Ceruloplasmin','Copper.','NULL','4 Fe(2+) + 4 H(+) + O(2) = 4 Fe(3+) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.5.1','Ascorbate ferrireductase (transmembrane)','Cytochrome b(561)','Heme.','A diheme cytochrome that transfers electrons across a singlemembrane, such as the outer membrane of the enterocyte, or thetonoplast membrane of the plant cell vacuole.Acts on hexacyanoferrate(III) and other ferric chelates.','Ascorbate(Side 1) + Fe(III)(Side 2) = monodehydroascorbate(Side 1) +Fe(II)(Side 2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.8.1','Cob(II)yrinic acid a,c-diamide reductase','NULL','NULL','This enzyme also catalyzes the reduction of cob(II)yric acid,cob(II)inamide, cob(II)inamide phosphate, GDP-cob(II)inamide andcob(II)alamin although cob(II)yrinic acid a,c-diamide is thought tobe the physiological substrate.Also uses FAD and NADH but not NADPH.','2 cob(I)yrinic acid a,c-diamide + FMN + 2 H(+) = 2 cob(II)yrinic acida,c-diamide + FMNH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.9.1','Iron:rusticyanin reductase','NULL','Heme c.','The enzyme in Acidithiobacillus ferrooxidans is a component of anelectron transfer chain from Fe(II), comprising this enzyme,the copper protein rusticyanin, cytochrome c4, and cytochrome coxidase (EC 1.9.3.1).Formerly EC 1.16.98.1.','Fe(II) + rusticyanin = Fe(III) + reduced rusticyanin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.16.98.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.1.1','CDP-4-dehydro-6-deoxyglucose reductase','CDP-4-keto-6-deoxy-D-glucose-3-dehydrogenase system','NULL','The enzyme consists of two proteins.One forms an enzyme-bound adduct of the CDP-4-dehydro-6-deoxyglucosewith pyridoxamine phosphate, in which the 3-hydroxy group has beenremoved.The second catalyzes the reduction of this adduct by NAD(P)H andrelease of the CDP-4-dehydro-3,6-dideoxy-D-glucose and pyridoxaminephosphate.','CDP-4-dehydro-3,6-dideoxy-D-glucose + NAD(P)(+) + H(2)O = CDP-4-dehydro-6-deoxy-D-glucose + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.1.2','4-hydroxy-3-methylbut-2-enyl diphosphate reductase','(E)-4-hydroxy-3-methylbut-2-en-1-yl diphosphate reductase','Iron-sulfur.','Comprises a system in which ferredoxin is first reduced andsubsequently reoxidized by an NAD(P)(+)-dependent reductase.This is the last enzyme in the non-mevalonate pathway for isoprenoidbiosynthesis.Acts in the reverse direction, producing a 5:1 mixture of isopentenyldiphosphate and dimethyallyl diphosphate.','(1) Isopentenyl diphosphate + NAD(P)(+) + H(2)O = (E)-4-hydroxy-3-methylbut-2-en-1-yl diphosphate + NAD(P)H.(2) Dimethylallyl diphosphate + NAD(P)(+) + H(2)O = (E)-4-hydroxy-3-methylbut-2-en-1-yl diphosphate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.1.3','Leucoanthocyanidin reductase','Leucocyanidin reductase','NULL','Catalyzes the synthesis of catechin, catechin-4-beta-ol(leucocyanidin) and the related flavan-3-ols afzelechin andgallocatechin, which are initiating monomers in the synthesis ofplant polymeric proanthocyanidins or condensed tannins.While 2,3-trans-3,4-cis-leucocyanidin is the preferred flavan-3,4-diol substrate, 2,3-trans-3,4-cis-leucodelphinidin and 2,3-trans-3,4-cis-leucopelargonidin can also act as substrates, but more slowly.NADH can replace NADPH but is oxidized more slowly.','(2R,3S)-catechin + NADP(+) + H(2)O = 2,3-trans-3,4-cis-leucocyanidin +NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.1.4','Xanthine dehydrogenase','NAD-xanthine dehydrogenase','FAD; Iron-sulfur; Molybdenum.','Acts on a variety of purines and aldehydes, including hypoxanthine.The mammalian enzyme can also convert all-trans retinol to all-trans-retinoate, while the substrate is bound to a retinoid-bindingprotein.The enzyme from eukaryotes contains [2Fe-2S], FAD and a molybdenumcenter.The mammallian enzyme predominantly exists as the NAD-dependentdehydrogenase (EC 1.17.1.4).During purification the enzyme is largely converted to an O(2)-dependent form, EC 1.17.3.2.The conversion can be triggered by several mechanisms, including theoxidation of cysteine thiols to form disulfide bonds (which can becatalyzed by EC 1.8.4.7 in the presence of glutathione disulfide) orlimited proteolysis, which results in irreversible conversion.The conversion can also occur in vivo.Formerly EC 1.2.1.37 and EC 1.1.1.204.','(1) Xanthine + NAD(+) + H(2)O = urate + NADH.(2) Hypoxanthine + NAD(+) + H(2)O = xanthine + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.1.5','Nicotinate dehydrogenase','Nicotinate hydroxylase','Flavoprotein; Iron.','The enzyme is capable of acting on a variety of nicotinate analogs tovarying degrees, including pyrazine-2-carboxylate, pyrazine 2,3-dicarboxylate, trigonelline and 6-methylnicotinate.The enzyme from Clostridium barkeri also possesses a catalyticallyessential, labile selenium that can be removed by reaction withcyanide.Forms part of the nicotinate-fermentation catabolism pathway inEubacterium barkeri.Other enzymes involved in this pathway are EC 1.3.7.1, EC 3.5.2.18,EC 1.1.1.291, EC 5.4.99.4, EC 5.3.3.6, EC 4.2.1.85 and EC 4.1.3.32.Formerly EC 1.5.1.13.','Nicotinate + H(2)O + NADP(+) = 6-hydroxynicotinate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.1.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.1.7','3-oxo-5,6-dehydrosuberyl-CoA semialdehyde dehydrogenase','NULL','NULL','The enzyme from Escherichia coli is a bifunctional fusion proteinthat also catalyzes EC 3.3.2.12.Combined the two activities result in a two-step conversion ofoxepin-CoA to 3-oxo-5,6-dehydrosuberyl-CoA, part of an aerobicphenylacetate degradation pathway.','3-oxo-5,6-dehydrosuberyl-CoA semialdehyde + NADP(+) + H(2)O = 3-oxo-5,6-dehydrosuberyl-CoA + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.1.8','4-hydroxy-tetrahydrodipicolinate reductase','Dihydrodipicolinate reductase','NULL','Studies of the enzyme from the bacterium Escherichia coli have shownthat the enzyme accepts (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate and not (S)-2,3-dihydrodipicolinate asoriginally thought.Formerly EC 1.3.1.26.','(S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate + NAD(P)(+) + H(2)O =(2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.2.1','Nicotinate dehydrogenase (cytochrome)','Nicotinate hydroxylase','Iron-sulfur; Molybdopterin cytosine dinucleotide.','This two-component enzyme from Pseudomonas belongs to the family ofxanthine dehydrogenases, but differs from most other members of thisfamily.While most members contain an FAD cofactor, the large subunit of thisenzyme contains three c-type cytochromes, enabling it to interactwith the electron transfer chain, probably by delivering theelectrons to a cytochrome oxidase.','Nicotinate + a ferricytochrome + H(2)O = 6-hydroxynicotinate +a ferrocytochrome + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.2.2','Lupanine 17-hydroxylase (cytochrome c)','Lupanine dehydrogenase (cytochrome c)','Pyrroloquinoline quinone.','NULL','Lupanine + 2 ferricytochrome c + H(2)O = 17-hydroxylupanine +2 ferrocytochrome c + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.3.1','Pteridine oxidase','NULL','NULL','Does not act on hypoxanthine.Different from EC 1.17.3.2.','2-amino-4-hydroxypteridine + O(2) = 2-amino-4,7-dihydroxypteridine + (?).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.3.2','Xanthine oxidase','Hypoxanthine oxidase','FAD; Iron-sulfur; Molybdopterin.','Also oxidizes hypoxanthine, some other purines and pterins,and aldehydes, but is distinct from EC 1.2.3.1.Under some conditions the product is mainly superoxide rather thanperoxide: R-H + H(2)O + 2 O(2) = ROH + 2 O(2)(.-) + 2 H(+).The mammallian enzyme predominantly exists as an NAD-dependentdehydrogenase (EC 1.17.1.4).During purification the enzyme is largely converted to the O(2)-dependent xanthine oxidase form (EC 1.17.3.2).The conversion can be triggered by several mechanisms, including theoxidation of cysteine thiols to form disulfide bonds (which can becatalyzed by EC 1.8.4.7 in the presence of glutathione disulfide) orlimited proteolysis, which results in irreversible conversion.The conversion can also occur in vivo.Formerly EC 1.1.3.22 and EC 1.2.3.2.','Xanthine + H(2)O + O(2) = urate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.3.3','6-hydroxynicotinate dehydrogenase','6-hydroxynicotinate hydroxylase','FAD; Iron-sulfur; Molybdopterin.','The enzyme also has a catalytically essential, labile selenium thatcan be removed by reaction with cyanide.In Bacillus niacini, this enzyme is required for growth on nicotinicacid.','6-hydroxynicotinate + H(2)O + O(2) = 2,6-dihydroxynicotinate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.4.1','Ribonucleoside-diphosphate reductase','Ribonucleotide reductase','Iron.','Responsible for the de novo conversion of ribonucleoside diphosphatesinto deoxyribonucleoside diphosphates, which are essential for DNAsynthesis and repair.While the enzyme is activated by ATP, it is inhibited by dATP.','2-deoxyribonucleoside diphosphate + thioredoxin disulfide + H(2)O =ribonucleoside diphosphate + thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.4.2','Ribonucleoside-triphosphate reductase','Ribonucleotide reductase','Cobalt.','NULL','2-deoxyribonucleoside triphosphate + thioredoxin disulfide + H(2)O =ribonucleoside triphosphate + thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.4.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.5.1','Phenylacetyl-CoA dehydrogenase','Phenylacetyl-CoA:acceptor oxidoreductase','Iron-sulfur; Molybdopterin.','The enzyme is specific for phenylacetyl-CoA as substrate.Phenylacetate, acetyl-CoA, benzoyl-CoA, propanoyl-CoA, crotonyl-CoA,succinyl-CoA and 3-hydroxybenzoyl-CoA cannot act as substrates.The oxygen atom introduced into the product, phenylglyoxylyl-CoA,is derived from water and not molecular oxygen.Duroquinone, menaquinone and 2,6-dichlorophenolindophenol (DCPIP) canact as acceptor, but the likely physiological acceptor is ubiquinone.A second enzyme, EC 3.1.2.25 converts the phenylglyoxylyl-CoA formedinto phenylglyoxylate.','Phenylacetyl-CoA + H(2)O + 2 quinone = phenylglyoxylyl-CoA + 2 quinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.5.2','Caffeine dehydrogenase','NULL','NULL','This enzyme, characterized from the soil bacterium Pseudomonas sp.CBB1, catalyzes the incorporation of an oxygen atom originating froma water molecule into position C-8 of caffeine.The enzyme utilizes short-tail ubiquinones as the preferred electronacceptor.','Caffeine + ubiquinone + H(2)O = 1,3,7-trimethylurate + ubiquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.7.1','(E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase','(E)-4-hydroxy-3-methylbut-2-enyl diphosphate synthase','Iron-sulfur.','Forms, in the reverse direction, part of an alternative non-mevalonate pathway for isoprenoid biosynthesis that is found in mostbacteria and in plant chloroplasts.The enzyme from the plant Arabidopsis thaliana is active withphotoreduced 5-deazaflavin but not with the flavodoxin/flavodoxinreductase/NADPH system that can be used as an electron donor byEscherichia coli.Metabolites derived from isoprenoids play important roles in systemssuch as electron transport, photosynthesis, plant defense responses,hormonal regulation of development and membrane fluidity.Formerly EC 1.17.4.3.','(E)-4-hydroxy-3-methylbut-2-en-1-yl diphosphate + H(2)O + 2 oxidizedferredoxin = 2-C-methyl-D-erythritol 2,4-cyclodiphosphate + 2 reducedferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.7.2','7-hydroxymethyl chlorophyll a reductase','NULL','FAD; Iron-sulfur.','This enzyme, which is present in plant chloroplasts, carries out thesecond step in the conversion of chlorophyll b to chlorophyll a (cf.EC 1.1.1.294).','7(1)-hydroxychlorophyllide a + 2 reduced ferredoxin + 2 H(+) =chlorophyll a + 2 oxidized ferredoxin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.99.1','4-methylphenol dehydrogenase (hydroxylating)','4-cresol dehydrogenase (hydroxylating)','FAD.','Phenazine methosulfate can act as acceptor.A quinone methide is probably formed as intermediate.The first hydroxylation forms 4-hydroxybenzyl alcohol; a secondhydroxylation converts this into 4-hydroxybenzaldehyde.','4-methylphenol + acceptor + H(2)O = 4-hydroxybenzaldehyde + reducedacceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.99.2','Ethylbenzene hydroxylase','Ethylbenzene dehydrogenase','Heme; Iron-sulfur; Molybdopterin.','Involved in the anaerobic catabolism of ethylbenzene by denitrifyingbacteria.Ethylbenzene is the preferred substrate; the enzyme from some strainsoxidizes propylbenzene, 1-ethyl-4-fluorobenzene, 3-methylpent-2-eneand ethylidenecyclohexane.Toluene is not oxidized.p-benzoquinone or ferrocenium can act as electron acceptor.','Ethylbenzene + H(2)O + acceptor = (S)-1-phenylethanol + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.99.3','3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestanoyl-CoA24-hydroxylase','3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestan-26-oate24-hydroxylase','NULL','Requires ATP.The reaction in mammals possibly involves dehydrogenation to give a24(25)-double bond followed by hydration.However, in amphibians such as the Oriental fire-bellied toad(Bombina orientalis), it is probable that the product is formed viadirect hydroxylation of the saturated side chain of (25R)-3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestan-26-oate and not viahydration of a 24(25) double bond.In microsomes, the free acid is preferred to the coenzyme A ester,whereas in mitochondria, the coenzyme A ester is preferred to thefree-acid form of the substrate.','(25R)-3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestan-26-oyl-CoA +H(2)O + acceptor = (24R,25R)-3-alpha,7-alpha,12-alpha,24-tetrahydroxy-5-beta-cholestan-26-oyl-CoA + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.99.4','Uracil/thymine dehydrogenase','Uracil dehydrogenase','NULL','Forms part of the oxidative pyrimidine-degrading pathway in somemicroorganisms, along with EC 3.5.2.1 and EC 3.5.1.95.Mammals, plants and other microorganisms utilize the reductivepathway, comprising EC 1.3.1.1 or EC 1.3.1.2, EC 3.5.2.2 andEC 3.5.1.6, with the ultimate degradation products being an L-aminoacid, NH(3) and CO(2).Formerly EC 1.1.99.19 and EC 1.2.99.1.','(1) Uracil + H(2)O + acceptor = barbiturate + reduced acceptor.(2) Thymine + H(2)O + acceptor = 5-methylbarbiturate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.17.99.5','Bile-acid 7-alpha-dehydroxylase','7-alpha-dehydroxylase','NULL','Under physiological conditions, the reactions occur in the reversedirection to that shown above.Highly specific for bile-acid substrates and requires a free C-24carboxy group and an unhindered 7-alpha-hydroxy group on the B-ringof the steroid nucleus for activity, as found in cholate andchenodeoxycholate.The reaction is stimulated by the presence of NAD(+) but is inhibitedby excess NADH.This unusual regulation by the NAD(+)/NADH ratio is most likely theresult of the intermediates being linked at C-24 by an anhydride bondto the 5-diphosphate of 3-phospho-ADP.Allo-deoxycholate is also formed as a side-product of the 7-alpha-dehydroxylation of cholate.Formerly EC 1.17.1.6.','(1) Deoxycholate + FAD + H(2)O = cholate + FADH(2).(2) Lithocholate + FAD + H(2)O = chenodeoxycholate + FADH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.1.1','Rubredoxin--NAD(+) reductase','Dihydronicotinamide adenine dinucleotide-rubredoxin reductase','FAD; Iron.','The enzyme from Clostridium acetobutylicum reduces rubredoxin,ferricyanide and dichlorophenolindophenol, but not ferredoxin orflavodoxin.The reaction does not occur when NADPH is substituted for NADH.Formerly EC 1.6.7.2.','2 reduced rubredoxin + NAD(+) + H(+) = 2 oxidized rubredoxin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.1.2','Ferredoxin--NADP(+) reductase','NULL','FAD.','In chloroplasts and cyanobacteria the enzyme acts on plant-type [2Fe-2S] ferredoxins, but in other bacteria it can also reduce bacterial2[4Fe-4S] ferredoxins and flavodoxin.Formerly EC 1.6.7.1 and EC 1.6.99.4.','2 reduced ferredoxin + NADP(+) + H(+) = 2 oxidized ferredoxin + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.1.3','Ferredoxin--NAD(+) reductase','NULL','FAD.','Reaction (1) is written for a [2Fe-2S] ferredoxin, which ischaracteristic of some mono- and dioxygenase systems.The alternative reaction (2) is written for a 2[4Fe-4S] ferredoxin,which transfers two electrons, and occurs in metabolism of anaerobicbacteria.','Reduced ferredoxin + NAD(+) = oxidized ferredoxin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.1.4','Rubredoxin--NAD(P)(+) reductase','Dinucleotide phosphate reductase','FAD.','The enzyme reduces a number of electron carriers, including benzylviologen, menadione and 2,6-dichloroindophenol, but rubredoxin is themost efficient.Ferredoxin is not utilized.','2 reduced rubredoxin + NAD(P)(+) + H(+) = 2 oxidized rubredoxin +NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.1.5','Putidaredoxin--NAD(+) reductase','Putidaredoxin reductase','NULL','The enzyme from Pseudomonas putida reduces putidaredoxin.It contains a [2Fe-2S] cluster.Involved in the camphor monooxygenase system (see EC 1.14.15.1).','Reduced putidaredoxin + NAD(+) = oxidized putidaredoxin + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.1.6','Adrenodoxin-NADP(+) reductase','Adrenodoxin reductase','FAD.','The enzyme is the first component in the mitochondrial cytochromeP450 electron transfer systems, and is involved in the biosynthesisof all steroid hormones.','2 reduced adrenodoxin + NADP(+) = 2 oxidized adrenodoxin + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.1.7','Ferredoxin--NAD(P)(+) reductase (naphthalene dioxygenase ferredoxin-specific)','NADH-ferredoxin(NAP) reductase','FAD; Iron-sulfur.','The enzyme from the aerobic bacterium Ralstonia sp. U2 donateselectrons to both EC 1.14.12.12 and EC 1.14.13.172.The enzyme from Pseudomonas NCIB 9816 is specific for the ferredoxinassociated with naphthalene dioxygenase.','2 reduced [2Fe-2S] ferredoxin + NAD(P)(+) + H(+) = 2 oxidized [2Fe-2S]ferredoxin + NAD(P)H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.2.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.3.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.6.1','Nitrogenase','NULL','Iron-sulfur; Vanadium or molybdenum.','Composed of two proteins that can be separated but are both requiredfor nitrogenase activity.Dinitrogen reductase is a [4Fe-4S] protein, which, with two moleculesof ATP and ferredoxin, generates an electron.The electron is transferred to the other protein, dinitrogenase(molybdoferredoxin).Dinitrogenase is a molybdenum-iron protein that reduces dinitrogen inthree succesive two-electron reductions from nitrogen to diimine tohydrazine to two molecules of ammonia; the molybdenum may be replacedby vanadium or iron.The reduction is initiated by formation of hydrogen in stoichiometricamounts.Acetylene is reduced to ethylene (but only very slowly to ethane),azide to nitrogen and ammonia, and cyanide to methane and ammonia.In the absence of a suitable substrate, hydrogen is slowly formed.Ferredoxin may be replaced by flavodoxin (see EC 1.19.6.1).Formerly EC 1.18.2.1.','8 reduced ferredoxin + 8 H(+) + N(2) + 16 ATP + 16 H(2)O = 8 oxidizedferredoxin + H(2) + 2 NH(3) + 16 ADP + 16 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.96.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.18.99.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.19.6.1','Nitrogenase (flavodoxin)','NULL','Iron-sulfur; Molybdenum.','NULL','6 reduced flavodoxin + 6 H(+) + N(2) + n ATP = 6 oxidized flavodoxin +2 NH(3) + n ADP + n phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.20.1.1','Phosphonate dehydrogenase','NAD-dependent phosphite dehydrogenas','NULL','NADP(+) is a poor substitute for NAD(+) in the enzyme fromPseudomonas stutzeri WM88.','Phosphonate + NAD(+) + H(2)O = phosphate + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.20.2.1','Arsenate reductase (cytochrome c)','Arsenite oxidase','Iron-sulfur; Molybdopterin.','Isolated from alpha-proteobacteria.Unlike EC 1.20.9.1 it does not use azurin as acceptor.','Arsenite + H(2)O + 2 oxidized cytochrome c = arsenate + 2 reducedcytochrome c + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.20.4.1','Arsenate reductase (glutaredoxin)','NULL','Molybdenum.','The glutaredoxins catalyze glutathione-disulfide oxidoreductions andhave a redox-active disulfide/dithiol in the active site (-Cys-Pro-Tyr-Cys-) that forms a disulfide bond in the oxidized form.Glutaredoxins have a binding site for glutathione, which is requiredto reduce them to the dithiol form.Thioredoxins reduced by NADPH and thioredoxin reductase can act asalternative substrates.The enzyme is part of a system for detoxifying arsenate.Although the arsenite formed is more toxic than arsenate, it can beextruded from some bacteria by EC 3.6.3.16.In other organisms, arsenite can be methylated by EC 2.1.1.137 in apathway to non-toxic organoarsenical compounds.Formerly EC 1.97.1.5.','Arsenate + glutaredoxin = arsenite + glutaredoxin disulfide + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.20.4.2','Methylarsonate reductase','MMA(V) reductase','NULL','The product, Me-As(OH)(2) (methylarsonous acid), is biologicallymethylated by EC 2.1.1.137 to form cacodylic acid (dimethylarsinicacid).Formerly EC 1.97.1.7.','Methylarsonate + 2 glutathione = methylarsonite + glutathione disulfide +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.20.4.3','Mycoredoxin','NULL','NULL','Reduction of arsenate is part of a defense mechanism of the cellagainst toxic arsenate.The substrate arseno-mycothiol is formed by EC 2.8.4.2.A second mycothiol recycles mycoredoxin and forms mycothione.','Arseno-mycothiol + mycoredoxin = arsenite + mycothiol-mycoredoxin disulfide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.20.9.1','Arsenate reductase (azurin)','Arsenite oxidase','Iron-sulfur; Molybdenum; Molybdopterin.','The enzyme also uses a c-type cytochrome or O(2) as acceptors.Formerly EC 1.20.98.1.','Arsenite + H(2)O + 2 oxidized azurin = arsenate + 2 reduced azurin +2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.20.98.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.20.99.1','Arsenate reductase (donor)','NULL','NULL','Benzyl viologen can act as an acceptor.Unlike EC 1.20.4.1 reduced glutaredoxin cannot serve as a reductant.Formerly EC 1.97.1.6.','Arsenite + acceptor = arsenate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.3.1','Isopenicillin-N synthase','NULL','NULL','Forms part of the penicillin biosynthesis pathway.','N-((5S)-5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine + O(2) =isopenicillin N + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.3.2','Columbamine oxidase','Berberine synthase','Iron.','Oxidation of the O-methoxyphenol structure forms the methylenedioxygroup of berberine.Formerly EC 1.1.3.26.','2 columbamine + O(2) = 2 berberine + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.3.3','Reticuline oxidase','Berberine bridge enzyme','FAD.','Acts on (S)-reticuline and related compounds, converting the N-methylgroup into the methylene bridge (berberine bridge) of (S)-tetrahydroprotoberberines.The product of the reaction, (S)-scoulerine, is a precursor ofprotopine, protoberberine and benzophenanthridine alkaloidbiosynthesis in plants.Formerly EC 1.5.3.9.','(S)-reticuline + O(2) = (S)-scoulerine + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.3.4','Sulochrin oxidase ((+)-bisdechlorogeodin-forming)','Sulochrin oxidase','NULL','Also acts on several diphenols and phenylenediamines, but has lowaffinity for these substrates.Involved in the biosynthesis of mold metabolites related to theantibiotic griseofulvin.Formerly EC 1.10.3.7.','2 sulochrin + O(2) = 2 (+)-bisdechlorogeodin + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.3.5','Sulochrin oxidase ((-)-bisdechlorogeodin-forming)','Sulochrin oxidase','NULL','Also acts on several diphenols and phenylenediamines, but has lowaffinity for these substrates.Involved in the biosynthesis of mold metabolites related to theantibiotic griseofulvin.Formerly EC 1.10.3.8.','2 sulochrin + O(2) = 2 (-)-bisdechlorogeodin + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.3.6','Aureusidin synthase','NULL','Copper.','Plays a key role in the yellow coloration of flowers such asAntirrhinum majus (snapdragon).The enzyme is a homolog of plant polyphenol oxidase and catalyzes twoseparate chemical transformations, i.e. 3-hydroxylation and oxidativecyclization (2,-dehydrogenation).H(2)O(2) activates reaction (1) but inhibits reaction (2).Originally considered to act on the phenol but now thought to mainlyact on the 4-O-beta-glucoside in vivo.','(1) 2,4,4,6-tetrahydroxychalcone 4-O-beta-glucoside + O(2) =aureusidin 6-O-beta-glucoside + H(2)O.(2) 2 2,3,4,4,6-pentahydroxychalcone 4-O-beta-glucoside + O(2) =2 aureusidin 6-O-beta-glucoside + 2 H(2)O.(3) 2,3,4,4,6-pentahydroxychalcone 4-O-beta-glucoside + O(2) =bracteatin 6-O-beta-glucoside + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.3.7','Tetrahydrocannabinolic acid synthase','Delta(1)-tetrahydrocannabinolic acid synthase','FAD.','The cofactor is covalently bound.Part of the cannabinoids biosynthetic pathway in the plant Cannabissativa.The enzyme can also convert cannabinerolate (the (Z)-isomer ofcannabigerolate) to Delta(9)-THCA with lower efficiency.The traditional numbering called Delta(9)-tetrahydrocannabinolate,Delta(1)-tetrahydrocannabinolate.Systematic peripheral numbering is now recommended.','Cannabigerolate + O(2) = Delta(9)-tetrahydrocannabinolate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.3.8','Cannabidiolic acid synthase','CBDA synthase','FAD.','Part of the cannabinoids biosynthetic pathway of the plant Cannabissativa.The enzyme can also convert cannabinerolate to cannabidiolate withlower efficiency.','Cannabigerolate + O(2) = cannabidiolate + H(2)O(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.3.9','Dichlorochromopyrrolate synthase','NULL','Heme; Iron.','This enzyme catalyzes a step in the biosynthesis of rebeccamycin,an indolocarbazole alkaloid produced by the actinobacteriumLechevalieria aerocolonigenes.The enzyme is a dimeric heme-protein oxidase that catalyzes theoxidative dimerization of two L-tryptophan-derived molecules to formdichlorochromopyrrolic acid, the precursor for the fused six-ringindolocarbazole scaffold of rebeccamycin.The enzyme also possesses catalase activity, suggesting that thesecondary product may be hydrogen peroxide rather than water.The nature of the secondary product has not been confirmed as of2013.Formerly EC 4.3.1.26.','4 2-imino-3-(7-chloroindol-3-yl)propanoate + O(2) =2 dichlorochromopyrrolate + 2 NH(3) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.4.1','D-proline reductase (dithiol)','NULL','Pyruvate.','The reaction is observed only in the direction of D-prolinereduction.Other dithiols can function as reducing agents; the enzyme contains apyruvoyl group and a selenocysteine residue, both essential foractivity.Formerly EC 1.4.1.6 and EC 1.4.4.1.','5-aminopentanoate + lipoate = D-proline + dihydrolipoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.4.2','Glycine reductase','NULL','NULL','The reaction is observed only in the direction of glycine reduction.The enzyme from Eubacterium acidaminophilum consists of subunits A,B and C.Subunit B contains selenocysteine and a pyruvoyl group, and isresponsible for glycine binding and ammonia release.Subunit A, which also contains selenocysteine, is reduced bythioredoxin, and is needed to convert the carboxymethyl group into aketene equivalent, in turn used by subunit C to produce acetylphosphate.Only subunit B distinguishes this enzyme from EC 1.21.4.3 andEC 1.21.4.4.','Acetyl phosphate + NH(3) + thioredoxin disulfide + H(2)O = glycine +phosphate + thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.4.3','Sarcosine reductase','NULL','NULL','The reaction is observed only in the direction of sarcosinereduction.The enzyme from Eubacterium acidaminophilum consists of subunits A,B and C.Subunit B contains selenocysteine and a pyruvoyl group, and isresponsible for sarcosine binding and methylamine release.Subunit A, which also contains selenocysteine, is reduced bythioredoxin, and is needed to convert the carboxymethyl group into aketene equivalent, in turn used by subunit C to produce acetylphosphate.Only subunit B distinguishes this enzyme from EC 1.21.4.2 andEC 1.21.4.4.','Acetyl phosphate + methylamine + thioredoxin disulfide + H(2)O =N-methylglycine + phosphate + thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.4.4','Betaine reductase','NULL','NULL','The reaction is observed only in the direction of betaine reduction.The enzyme from Eubacterium acidaminophilum consists of subunits A,B and C.Subunit B contains selenocysteine and a pyruvoyl group, and isresponsible for glycine binding and ammonia release.Subunit A, which also contains selenocysteine, is reduced bythioredoxin, and is needed to convert the carboxymethyl group into aketene equivalent, in turn used by subunit C to produce acetylphosphate.Only subunit B distinguishes this enzyme from EC 1.21.4.2 andEC 1.21.4.3.','Acetyl phosphate + trimethylamine + thioredoxin disulfide + H(2)O =betaine + phosphate + thioredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.99.1','Beta-cyclopiazonate dehydrogenase','Beta-cyclopiazonate oxidocyclase','FAD.','Cytochrome c and various dyes can act as acceptor.Cyclopiazonate is a microbial toxin.Formerly EC 1.3.99.9.','Beta-cyclopiazonate + acceptor = alpha-cyclopiazonate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.21.99.2','Cyclic dehypoxanthinyl futalosine synthase','Dehypoxanthinyl futalosine cyclase','Iron-sulfur.','This enzyme is a member of the AdoMet radical (radical SAM) family.The enzyme, found in several bacterial species, is part of thefutalosine pathway for menaquinone biosynthesis.','Dehypoxanthine futalosine + S-adenosyl-L-methionine = cyclicdehypoxanthinyl futalosine + 5-deoxyadenosine + L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.22.1.1','Iodotyrosine deiodinase','Iodotyrosine dehalogenase 1','FMN.','The enzyme activity has only been demonstrated in the direction of3-deiodination.Present in a transmembrane flavoprotein.','L-tyrosine + 2 NADP(+) + 2 I(-) = 3,5-diiodo-L-tyrosine + 2 NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.23.1.1','(+)-pinoresinol reductase','(+)-pinoresinol-(+)-lariciresinol reductase','NULL','The reaction is catalyzed in vivo in the opposite direction to thatshown.A multifunctional enzyme that further reduces the product to thelignan (-)-secoisolariciresinol (EC 1.23.1.2).Isolated from the plants Forsythia intermedia, Thuja plicata (westernred cedar), Linum perenne (perennial flax) and Linum corymbulosum.The 4-pro-R hydrogen of NADH is transferred to the 7-pro-R positionof lariciresinol.','(+)-lariciresinol + NADP(+) = (+)-pinoresinol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.23.1.2','(+)-lariciresinol reductase','(+)-pinoresinol-(+)-lariciresinol reductase','NULL','The reaction is catalyzed in vivo in the opposite direction to thatshown.A multifunctional enzyme that also reduces (+)-pinoresinol(EC 1.23.1.1).Isolated from the plants Forsythia intermedia, Thuja plicata (westernred cedar), Linum perenne (perennial flax) and Linum corymbulosum.','(-)-secoisolariciresinol + NADP(+) = (+)-lariciresinol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.23.1.3','(-)-pinoresinol reductase','(-)-pinoresinol-(-)-lariciresinol reductase','NULL','The reaction is catalyzed in vivo in the opposite direction to thatshown.A multifunctional enzyme that usually further reduces the product to(+)-secoisolariciresinol (EC 1.23.1.4).Isolated from the plants Thuja plicata (western red cedar), Linumperenne (perennial flax) and Arabidopsis thaliana (thale cress).','(-)-lariciresinol + NADP(+) = (-)-pinoresinol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.23.1.4','(-)-lariciresinol reductase','(-)-pinoresinol-(-)-lariciresinol reductase','NULL','The reaction is catalyzed in vivo in the opposite direction to thatshown.A multifunctional enzyme that also reduces (-)-pinoresinol(EC 1.23.1.3).Isolated from the plants Thuja plicata (western red cedar) and Linumcorymbulosum.','(+)-secoisolariciresinol + NADP(+) = (-)-lariciresinol + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.1','Chlorate reductase','NULL','NULL','Flavins or benzylviologen can act as acceptor.','AH(2) + chlorate = A + H(2)O + chlorite.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.2','Pyrogallol hydroxytransferase','Pyrogallol hydroxyltransferase','Molybdenum.','1,2,3,5-tetrahydroxybenzene acts as a cosubstrate for the conversionof pyrogallol into phloroglucinol, and for a number of similarisomerizations.The enzyme is provisionally listed here, but might be considered asthe basis for a new class in the transferases, analogous to theaminotransferases.','1,2,3,5-tetrahydroxybenzene + 1,2,3-trihydroxybenzene = 1,3,5-trihydroxybenzene + 1,2,3,5-tetrahydroxybenzene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.4','[Formate-C-acetyltransferase]-activating enzyme','[Pyruvate formate-lyase]-activating enzyme','Iron-sulfur.','A single glycine residue in EC 2.3.1.54 is oxidized to thecorresponding radical by transfer of H from its CH(2) to AdoMet withconcomitant cleavage of the latter.The first stage is reduction of the AdoMet to give methionine and the5-deoxyadenosin-5-yl radical, which then abstracts a hydrogenradical from the glycine residue.','S-adenosyl-L-methionine + dihydroflavodoxin + [formateC-acetyltransferase]-glycine = 5-deoxyadenosine + L-methionine +flavodoxin semiquinone + [formate C-acetyltransferase]-glycin-2-ylradical.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.8','Tetrachloroethene reductive dehalogenase','Tetrachloroethene reductase','Iron-sulfur.','This enzyme allows the common pollutant tetrachloroethene to supportbacterial growth and is responsible for disposal of a number ofchlorinated hydrocarbons by this organism.The reaction occurs in the reverse direction.The enzyme also reduces trichloroethene to dichloroethene.Although the physiological reductant is unknown, the supply ofreductant in some organisms is via reduced menaquinone, itself formedfrom molecular hydrogen, via EC 1.12.99.3.Methyl viologen can act as electron donor.','Trichloroethene + chloride + acceptor = tetrachloroethene +reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.9','Selenate reductase','NULL','Heme; Iron-sulfur; Molybdenum.','The periplasmic enzyme from Thauera selenatis is a complex comprisingthree heterologous subunits (alpha, beta and gamma).Nitrate, nitrite, chlorate and sulfate are not substrates.A number of compounds, including acetate, lactate, pyruvate,and certain sugars, amino acids, fatty acids, di- and tricarboxylicacids, and benzoate can serve as electron donors.','Selenite + H(2)O + acceptor = selenate + reduced acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.10','Thyroxine 5-deiodinase','Diiodothyronine 5-deiodinase','NULL','The enzyme activity has only been demonstrated in the direction of5-deiodination, which renders the thyroid hormone more active.The enzyme consists of type I and type II enzymes, both containingselenocysteine, but with different kinetics.For the type I enzyme the first reaction is a reductive deiodinationconverting the -Se-H group of the enzyme into an -Se-I group; thereductant then reconverts this into -Se-H, releasing iodide.Formerly EC 3.8.1.4.','3,5,3-triiodo-L-thyronine + iodide + A + H(+) = L-thyroxine + AH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.11','Thyroxine 5-deiodinase','Diiodothyronine 5-deiodinase','NULL','The enzyme activity has only been demonstrated in the direction of5-deiodination.This removal of the 5-iodine, i.e. from the inner ring, largelyinactivates the hormone thyroxine.','3,3,5-triiodo-L-thyronine + iodide + A + H(+) = L-thyroxine + AH(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.97.1.12','Photosystem I','NULL','NULL','Contains chlorophyll, phylloquinones, carotenoids and [4Fe-4S]clusters.Cytochrome c6 can act as an alternative electron donor,and flavodoxin as an alternative acceptor in some species.','Reduced plastocyanin + oxidized ferredoxin + light =oxidized plastocyanin + reduced ferredoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.98.1.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.3','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.4','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.6','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.7','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.8','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.9','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.10','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.11','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.12','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.13','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.1.14','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.2.1','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.2.2','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.2.3','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.2.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.2.5','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 1.99.2.6','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.1','Nicotinamide N-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + nicotinamide = S-adenosyl-L-homocysteine +1-methylnicotinamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.2','Guanidinoacetate N-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + guanidinoacetate = S-adenosyl-L-homocysteine +creatine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.3','Thetin--homocysteine S-methyltransferase','NULL','NULL','NULL','Dimethylsulfonioacetate + L-homocysteine = S-methylthioglycolate +L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.4','Acetylserotonin O-methyltransferase','Hydroxyindole O-methyltransferase','NULL','Some other hydroxyindoles also act as acceptor, more slowly.','S-adenosyl-L-methionine + N-acetylserotonin = S-adenosyl-L-homocysteine +melatonin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.5','Betaine--homocysteine S-methyltransferase','NULL','Zinc.','NULL','Trimethylammonioacetate + L-homocysteine = dimethylglycine +L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.6','Catechol O-methyltransferase','NULL','NULL','The mammalian enzymes act more rapidly on catecholamines such asadrenaline or noradrenaline than on catechols.','S-adenosyl-L-methionine + a catechol = S-adenosyl-L-homocysteine +a guaiacol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.7','Nicotinate N-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + nicotinate = S-adenosyl-L-homocysteine +N-methylnicotinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.8','Histamine N-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + histamine = S-adenosyl-L-homocysteine + N(tau)-methylhistamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.9','Thiol S-methyltransferase','NULL','NULL','H(2)S and a variety of alkyl, aryl and heterocyclic thiols andhydroxy thiols can act as acceptors.','S-adenosyl-L-methionine + a thiol = S-adenosyl-L-homocysteine +a thioether.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.10','Homocysteine S-methyltransferase','NULL','NULL','The enzyme uses S-adenosyl-L-methionine as methyl donor less activelythan S-methyl-L-methionine.','S-methyl-L-methionine + L-homocysteine = 2 L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.11','Magnesium protoporphyrin IX methyltransferase','(-)-S-adenosyl-L-methionine:magnesium-protoporphyrin IXmethyltransferase','NULL','NULL','S-adenosyl-L-methionine + magnesium protoporphyrin IX = S-adenosyl-L-homocysteine + magnesium protoporphyrin IX 13-methyl ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.12','Methionine S-methyltransferase','NULL','Zinc or manganese.','NULL','S-adenosyl-L-methionine + L-methionine = S-adenosyl-L-homocysteine +S-methyl-L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.13','Methionine synthase','5-methyltetrahydrofolate--homocysteine S-methyltransferase','Cobalamin; Zinc.','The enzyme becomes inactivated occasionally during its cycle byoxidation of Co(I) to Co(II).Reactivation by reductive methylation is catalyzed by the enzymeitself, with S-adenosyl-L-methionine as the methyl donor and areducing system.For the mammalian enzyme, the reducing system involves NADPH andEC 1.16.1.8.In bacteria, the reducing agent is flavodoxin, and no furthercatalyst is needed (the flavodoxin is kept in the reduced state byNADPH and EC 1.18.1.2).Acts on the monoglutamate as well as the triglutamate folate,in contrast with EC 2.1.1.14, which acts only on the triglutamate.','5-methyltetrahydrofolate + L-homocysteine = tetrahydrofolate +L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.14','5-methyltetrahydropteroyltriglutamate--homocysteine S-methyltransferase','Cobalamin-independent methionine synthase','Zinc.','Requires phosphate.The enzyme from Escherichia coli also requires a reducing system.Unlike EC 2.1.1.13 this enzyme does not contain cobalamin.','5-methyltetrahydropteroyltri-L-glutamate + L-homocysteine =tetrahydropteroyltri-L-glutamate + L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.15','Fatty-acid O-methyltransferase','NULL','NULL','Oleic acid is the most effective fatty acid acceptor.','S-adenosyl-L-methionine + a fatty acid = S-adenosyl-L-homocysteine +a fatty acid methyl ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.16','Methylene-fatty-acyl-phospholipid synthase','Cyclopropane synthetase','NULL','The enzyme transfers a methyl group to the 10-position of a Delta(9)-olefinic acyl chain in phosphatidylglycerol, phosphatidylinositol,or, more slowly, phosphatidylethanolamine; subsequent proton transferproduces a 10-methylene group (cf. EC 2.1.1.79).','S-adenosyl-L-methionine + phospholipid olefinic fatty acid = S-adenosyl-L-homocysteine + phospholipid methylene fatty acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.17','Phosphatidylethanolamine N-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + phosphatidylethanolamine = S-adenosyl-L-homocysteine + phosphatidyl-N-methylethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.18','Polysaccharide O-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + a (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.19','Trimethylsulfonium--tetrahydrofolate N-methyltransferase','NULL','NULL','NULL','Trimethylsulfonium + tetrahydrofolate = dimethylsulfide +5-methyltetrahydrofolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.20','Glycine N-methyltransferase','Glycine methyltransferase','NULL','Thought to play an important role in the regulation of methyl groupmetabolism in the liver and pancreas by regulating the ratio betweenS-adenosyl-L-methionine and S-adenosyl-L-homocysteine.Inhibited by 5-methyltetrahydrofolate pentaglutamate.Sarcosine, which has no physiological role, is converted back intoglycine by the action of EC 1.5.8.3.','S-adenosyl-L-methionine + glycine = S-adenosyl-L-homocysteine +sarcosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.21','Methylamine--glutamate N-methyltransferase','N-methylglutamate synthase','NULL','NULL','Methylamine + L-glutamate = NH(3) + N-methyl-L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.22','Carnosine N-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + carnosine = S-adenosyl-L-homocysteine +anserine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.23','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.24','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.25','Phenol O-methyltransferase','NULL','NULL','Acts on a wide variety of simple alkyl-, methoxy- and halo-phenols.','S-adenosyl-L-methionine + phenol = S-adenosyl-L-homocysteine + anisole.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.26','Iodophenol O-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + 2-iodophenol = S-adenosyl-L-homocysteine +2-iodophenol methyl ether.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.27','Tyramine N-methyltransferase','NULL','NULL','Has some activity on phenylethylamine analogs.','S-adenosyl-L-methionine + tyramine = S-adenosyl-L-homocysteine +N-methyltyramine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.28','Phenylethanolamine N-methyltransferase','Noradrenaline N-methyltransferase','NULL','Acts on various phenylethanolamines; converts noradrenaline intoadrenaline.','S-adenosyl-L-methionine + phenylethanolamine = S-adenosyl-L-homocysteine+ N-methylphenylethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.29','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.30','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.31','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.32','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.33','tRNA (guanine(46)-N(7))-methyltransferase','N(7)-methylguanine methylase','NULL','The enzyme specifically methylates guanine(46) at N(7) in tRNA.','S-adenosyl-L-methionine + guanine(46) in tRNA = S-adenosyl-L-homocysteine+ N(7)-methylguanine(46) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.34','tRNA (guanosine(18)-2-O)-methyltransferase','tRNA (Gm18) 2-O-methyltransferase','NULL','The enzyme catalyzes the methylation of guanosine(18) in tRNA.','S-adenosyl-L-methionine + guanosine(18) in tRNA = S-adenosyl-L-homocysteine + 2-O-methylguanosine(18) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.35','tRNA (uracil(54)-C(5))-methyltransferase','5-methyluridine(54) tRNA methyltransferase','NULL','Unlike this enzyme, EC 2.1.1.74 uses 5,10-methylenetetrahydrofolateand FADH(2) to supply the atoms for methylation of U54.','S-adenosyl-L-methionine + uracil(54) in tRNA = S-adenosyl-L-homocysteine+ 5-methyluracil(54) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.36','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.37','DNA (cytosine-5-)-methyltransferase','Cytosine 5-methyltransferase','NULL','See the REBASE database for a complete list of these enzymes:http://rebase.neb.com/rebase/Formerly EC 2.1.1.73.','S-adenosyl-L-methionine + DNA = S-adenosyl-L-homocysteine +DNA containing 5-methylcytosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.38','O-demethylpuromycin O-methyltransferase','NULL','NULL','Puromycin is the antibiotic derived from N(6)-dimethyladenosine byreplacing the 3-hydroxy group with an amino group and acylating thiswith 4-O-methyltyrosine.','S-adenosyl-L-methionine + O-demethylpuromycin = S-adenosyl-L-homocysteine+ puromycin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.39','Inositol 3-methyltransferase','Inositol L-1-methyltransferase','NULL','NULL','S-adenosyl-L-methionine + myo-inositol = S-adenosyl-L-homocysteine +1D-3-O-methyl-myo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.40','Inositol 1-methyltransferase','Inositol 3-O-methyltransferase','NULL','NULL','S-adenosyl-L-methionine + myo-inositol = S-adenosyl-L-homocysteine +1D-1-O-methyl-myo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.41','Sterol 24-C-methyltransferase','24-sterol C-methyltransferase','Glutathione.','Acts on a range of sterols with a 24(25)-double bond in the sidechain.While zymosterol is the preferred substrate it also acts ondesmosterol, 5-alpha-cholesta-7,24-dien-3-beta-ol, 5-alpha-cholesta-5,7,24-trien-3-beta-ol, 4-alpha-methylzymosterol and others.S-adenosyl-L-methionine attacks the Si face of the 24(25) double bondand the C-24 hydrogen is transferred to C-25 on the Re face of thedouble bond.','S-adenosyl-L-methionine + 5-alpha-cholesta-8,24-dien-3-beta-ol =S-adenosyl-L-homocysteine + 24-methylene-5-alpha-cholest-8-en-3-beta-ol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.42','Flavone 3-O-methyltransferase','o-dihydric phenol methyltransferase','NULL','The enzyme prefers flavones with vicinal 3,4-dihydroxyl groups.','S-adenosyl-L-methionine + 3-hydroxyflavone = S-adenosyl-L-homocysteine +3-methoxyflavone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.43','Histone-lysine N-methyltransferase','Protein methylase 3','NULL','NULL','S-adenosyl-L-methionine + L-lysine-[histone] = S-adenosyl-L-homocysteine+ N(6)-methyl-L-lysine-[histone].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.44','L-histidine N(alpha)-methyltransferase','Dimethylhistidine N-methyltransferase','NULL','Part of the biosynthetic pathway of ergothioneine.','3 S-adenosyl-L-methionine + L-histidine = 3 S-adenosyl-L-homocysteine +hercynine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.45','Thymidylate synthase','NULL','NULL','NULL','5,10-methylenetetrahydrofolate + dUMP = dihydrofolate + dTMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.46','Isoflavone 4-O-methyltransferase','NULL','Magnesium.','The enzyme catalyzes the methylation of daidzein and genistein.It does not methylate naringenin, apigenin, luteolin or kaempferol.','S-adenosyl-L-methionine + a 4-hydroxyisoflavone = S-adenosyl-L-homocysteine + a 4-methoxyisoflavone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.47','Indolepyruvate C-methyltransferase','Indolepyruvate 3-methyltransferase','NULL','NULL','S-adenosyl-L-methionine + (indol-3-yl)pyruvate = S-adenosyl-L-homocysteine + (S)-3-(indol-3-yl)-2-oxobutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.48','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.49','Amine N-methyltransferase','Arylamine N-methyltransferase','NULL','An enzyme of very broad specificity; many primary, secondary andtertiary amines can act as acceptors, including tryptamine, aniline,nicotine and a variety of drugs and other xenobiotics.Formerly EC 2.1.1.81.','S-adenosyl-L-methionine + an amine = S-adenosyl-L-homocysteine +a methylated amine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.50','Loganate O-methyltransferase','NULL','NULL','Also acts on secologanate.Methylates the 11-carboxy group of the monoterpenoid loganate.','S-adenosyl-L-methionine + loganate = S-adenosyl-L-homocysteine + loganin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.51','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.52','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.53','Putrescine N-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + putrescine = S-adenosyl-L-homocysteine +N-methylputrescine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.54','Deoxycytidylate C-methyltransferase','NULL','NULL','dCMP is methylated by formaldehyde in the presence oftetrahydrofolate.CMP, dCTP and CTP can act as acceptor, more slowly.','5,10-methylenetetrahydrofolate + dCMP = dihydrofolate + deoxy-5-methylcytidylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.55','tRNA (adenine-N(6)-)-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNAcontaining N(6)-methyladenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.56','mRNA (guanine-N(7)-)-methyltransferase','Guanine-7-methyltransferase','NULL','Adds an N(7)-methylguanine cap to mRNA.The nucleoside next to the terminal guanosine may be either guanosineor adenosine.','S-adenosyl-L-methionine + G(5)pppR-RNA = S-adenosyl-L-homocysteine +m(7)G(5)pppR-RNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.57','Methyltransferase cap1','Cap1-MTase','NULL','This enzyme catalyzes the methylation of the ribose on the firsttranscribed nucleotide of mRNA or snRNA molecules, which may beeither guanosine or adenosine.This methylation event is known as cap1, and occurs in all mRNAs andsnRNAs of higher eukaryotes, including insects, vertebrates and theirviruses.The human enzyme can also methylate mRNA molecules that lackmethylation on the capping 5-triphosphoguanosine.','S-adenosyl-L-methionine + a 5-(N(7)-methyl 5-triphosphoguanosine)-(purine-ribonucleotide)-(mRNA) = S-adenosyl-L-homocysteine + a 5-(N(7)-methyl 5-triphosphoguanosine)-(2-O-methyl-purine-ribonucleotide)-(mRNA).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.58','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.59','[Cytochrome c]-lysine N-methyltransferase','Cytochrome c (lysine) methyltransferase','NULL','NULL','S-adenosyl-L-methionine + [cytochrome c]-L-lysine = S-adenosyl-L-homocysteine + [cytochrome c]-N(6)-methyl-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.60','Calmodulin-lysine N-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + calmodulin L-lysine = S-adenosyl-L-homocysteine+ calmodulin N(6)-methyl-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.61','tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase','NULL','NULL','Specific for the terminal methyl group of 5-methylaminomethyl-2-thiouridylate.','S-adenosyl-L-methionine + tRNA containing 5-aminomethyl-2-thiouridine =S-adenosyl-L-homocysteine + tRNA containing 5-methylaminomethyl-2-thiouridylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.62','mRNA (2-O-methyladenosine-N(6)-)-methyltransferase','NULL','NULL','mRNA containing an N(6),2-O-dimethyladenosine cap.','S-adenosyl-L-methionine + m(7)G(5)pppAm = S-adenosyl-L-homocysteine +m(7)G(5)pppm(6)Am.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.63','Methylated-DNA--[protein]-cysteine S-methyltransferase','6-O-methylguanine-DNA methyltransferase','NULL','This protein is involved in the repair of alkylated DNA.It acts only on the alkylated DNA (cf. EC 3.2.2.20 and EC 3.2.2.21).This enzyme catalyzes only one turnover and therefore is not strictlycatalytic.','DNA (containing 6-O-methylguanine) + protein L-cysteine = DNA (without6-O-methylguanine) + protein S-methyl-L-cysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.64','3-demethylubiquinol 3-O-methyltransferase','NULL','NULL','This enzyme is involved in ubiquinone biosynthesis.Ubiquinones from different organisms have a different number ofprenyl units (for example, ubiquinone-6 in Saccharomyces, ubiquinone-9 in rat and ubiquinone-10 in human), and thus the natural substratefor the enzymes from different organisms has a different number ofprenyl units.However, the enzyme usually shows a low degree of specificityregarding the number of prenyl units.For example, the human COQ3 enzyme can restore biosynthesis ofubiquinone-6 in coq3 deletion mutants of yeast.The enzymes from yeast, Escherichia coli and rat also catalyze themethylation of 3,4-dihydroxy-5-all-trans-polyprenylbenzoate (areaction that is classified as EC 2.1.1.114).','S-adenosyl-L-methionine + 3-demethylubiquinone-n = S-adenosyl-L-homocysteine + ubiquinone-n.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.65','Licodione 2-O-methyltransferase','NULL','NULL','As well as licodione (1-(2,4-dihydroxyphenyl)-3-(4-hydroxyphenyl)-1,3-propanedione), the 2-hydroxy-derivative and isoliquiritigenincan act as acceptors, more slowly.','S-adenosyl-L-methionine + licodione = S-adenosyl-L-homocysteine + 2-O-methyllicodione.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.66','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.67','Thiopurine S-methyltransferase','NULL','NULL','Also acts, more slowly, on thiopyrimidines and aromatic thiols.Not identical with EC 2.1.1.9.','S-adenosyl-L-methionine + a thiopurine = S-adenosyl-L-homocysteine +a thiopurine S-methylether.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.68','Caffeate O-methyltransferase','NULL','NULL','3,4-dihydroxybenzaldehyde and catechol can act as acceptor, moreslowly.','S-adenosyl-L-methionine + 3,4-dihydroxy-trans-cinnamate = S-adenosyl-L-homocysteine + 3-methoxy-4-hydroxy-trans-cinnamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.69','5-hydroxyfuranocoumarin 5-O-methyltransferase','Bergaptol 5-O-methyltransferase','NULL','Converts bergaptol into bergapten, which has therapeutic potential inthe treatment of psoriasis as it has photosensitizing andantiproliferative activities.Methylates the 5-hydroxy group of some hydroxy- and methylcoumarins,such as 5-hydroxyxanthotoxin, but has little activity on non-coumarinphenols; caffeate, 5-hydroxyferulate and daphnetin are notsubstrates.Cu(2+), Zn(2+) and Co(2+) cause enzyme inhibition.See also EC 2.1.1.70.Formerly EC 2.1.1.92.','(1) S-adenosyl-L-methionine + a 5-hydroxyfurocoumarin = S-adenosyl-L-homocysteine + a 5-methoxyfurocoumarin.(2) S-adenosyl-L-methionine + bergaptol = S-adenosyl-L-homocysteine +bergapten.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.70','8-hydroxyfuranocoumarin 8-O-methyltransferase','Furanocoumarin 8-methyltransferase','NULL','Converts xanthotoxol into xanthotoxin, which has therapeuticpotential in the treatment of psoriasis as it has photosensitizingand antiproliferative activities.Methylates the 8-hydroxy group of some hydroxy- and methylcoumarins,but has little activity on non-coumarin phenols.See also EC 2.1.1.69.Formerly EC 2.1.1.93.','(1) S-adenosyl-L-methionine + an 8-hydroxyfurocoumarin = S-adenosyl-L-homocysteine + an 8-methoxyfurocoumarin.(2) S-adenosyl-L-methionine + xanthotoxol = S-adenosyl-L-homocysteine +xanthotoxin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.71','Phosphatidyl-N-methylethanolamine N-methyltransferase','NULL','NULL','Also catalyzes the transfer of a further methylgroup, producingphosphatidylcholine.','S-adenosyl-L-methionine + phosphatidyl-N-methylethanolamine = S-adenosyl-L-homocysteine + phosphatidyl-N-dimethylethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.72','Site-specific DNA-methyltransferase (adenine-specific)','Modification methylase','NULL','This is a large group of enzymes, the majority of which, with enzymesof similar site specificity listed as EC 3.1.21.3, EC 3.1.21.4 andEC 3.1.21.5, form so-called restriction-modification systems.See the REBASE database for a complete list of these enzymes:http://rebase.neb.com/rebase/','S-adenosyl-L-methionine + DNA adenine = S-adenosyl-L-homocysteine +DNA 6-methylaminopurine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.73','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.74','Methylenetetrahydrofolate--tRNA-(uracil(54)-C(5))-methyltransferase(FADH(2)-oxidizing)','Folate-dependent ribothymidyl synthase','NULL','Up to 25% of the bases in mature tRNA are post-translationallymodified or hypermodified.One almost universal post-translational modification is theconversion of U54 into ribothymidine in the T-Psi-C loop, and thismodification is found in most species studied to date.Unlike this enzyme, which uses 5,10-methylenetetrahydrofolate andFADH(2) to supply the atoms for methylation of U54, EC 2.1.1.35 usesS-adenosyl-L-methionine.Formerly EC 2.1.2.12.','5,10-methylenetetrahydrofolate + uracil(54) in tRNA + FADH(2) =tetrahydrofolate + 5-methyluracil(54) in tRNA + FAD.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.75','Apigenin 4-O-methyltransferase','Flavonoid O-methyltransferase','NULL','Converts apigenin into acacetin.Naringenin (5,7,4-trihydroxyflavanone) can also act as an acceptor,but more slowly.','S-adenosyl-L-methionine + apigenin = S-adenosyl-L-homocysteine +acacetin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.76','Quercetin 3-O-methyltransferase','Flavonol 3-O-methyltransferase','NULL','Specific for quercetin.Related enzymes bring about the 3-O-methylation of other flavonols,such as galangin and kaempferol.','S-adenosyl-L-methionine + 3,5,7,3,4-pentahydroxyflavone = S-adenosyl-L-homocysteine + 3-methoxy-5,7,3,4-tetrahydroxyflavone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.77','Protein-L-isoaspartate(D-aspartate) O-methyltransferase','L-isoaspartyl protein carboxyl methyltransferase','NULL','D-aspartate (but not L-aspartate) residues in proteins can also actas acceptors.Formerly EC 2.1.1.24.','S-adenosyl-L-methionine + protein L-isoaspartate = S-adenosyl-L-homocysteine + protein L-isoaspartate alpha-methyl ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.78','Isoorientin 3-O-methyltransferase','NULL','NULL','Also acts on isoorientin 2-O-rhamnoside.Involved in the biosynthesis of flavones.','S-adenosyl-L-methionine + isoorientin = S-adenosyl-L-homocysteine +isoscoparin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.79','Cyclopropane-fatty-acyl-phospholipid synthase','CFA synthase','NULL','Adds a methylene group across the 9,10 position of a Delta(9)-olefinic acyl chain in phosphatidylethanolamine or, more slowly,phosphatidylglycerol or phosphatidylinositol forming a cyclopropanederivative (cf. EC 2.1.1.16).','S-adenosyl-L-methionine + phospholipid olefinic fatty acid = S-adenosyl-L-homocysteine + phospholipid cyclopropane fatty acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.80','Protein-glutamate O-methyltransferase','Methyl-accepting chemotaxis protein O-methyltransferase','NULL','Forms ester groups with L-glutamate residues in a number of membraneproteins.Formerly EC 2.1.1.24.','S-adenosyl-L-methionine + protein L-glutamate = S-adenosyl-L-homocysteine+ protein L-glutamate methyl ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.81','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.82','3-methylquercetin 7-O-methyltransferase','3-methylquercitin 7-O-methyltransferase','NULL','Involved with EC 2.1.1.76 and EC 2.1.1.83 in the methylation ofquercetin to 3,7,4-trimethylquercetin in Chrysosplenium americanum.Does not act on flavones, dihydroflavonols, or their glucosides.','S-adenosyl-L-methionine + 3,4,5,7-tetrahydroxy-3-methoxyflavone =S-adenosyl-L-homocysteine + 3,4,5-trihydroxy-3,7-dimethoxyflavone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.83','3,7-dimethylquercetin 4-O-methyltransferase','3,7-dimethylquercitin 4-O-methyltransferase','NULL','3,7-dimethylquercetagetin can also act as acceptor.Involved with EC 2.1.1.76 and EC 2.1.1.82 in the methylation ofquercetin to 3,7,4-trimethylquercetin in Chrysosplenium americanum.Does not act on flavones, dihydroflavonols, or their glucosides.','S-adenosyl-L-methionine + 3,4,5-trihydroxy-3,7-dimethoxyflavone =S-adenosyl-L-homocysteine + 3,5-dihydroxy-3,4,7-trimethoxyflavone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.84','Methylquercetagetin 6-O-methyltransferase','Flavonol 6-O-methyltransferase','NULL','The enzymes from Chrysosplenium americanum also methylates3,7,3-trimethylquercetagetin at the 6-position.Does not act on flavones, dihydroflavonols, or their glucosides.','S-adenosyl-L-methionine + 3,4,5,6-tetrahydroxy-3,7-dimethoxyflavone =S-adenosyl-L-homocysteine + 3,4,5-trihydroxy-3,6,7-trimethoxyflavone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.85','Protein-histidine N-methyltransferase','Protein methylase IV','NULL','Highly specific for histidine residues, for example, in actin.','S-adenosyl-L-methionine + protein L-histidine = S-adenosyl-L-homocysteine+ protein N(tau)-methyl-L-histidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.86','Tetrahydromethanopterin S-methyltransferase','N(5)-methyltetrahydromethanopterin--coenzyme M methyltransferase','NULL','Involved in the formation of methane from CO in methanogenic archaea.','5-methyl-5,6,7,8-tetrahydromethanopterin + 2-mercaptoethanesulfonate =5,6,7,8-tetrahydromethanopterin + 2-(methylthio)ethanesulfonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.87','Pyridine N-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + pyridine = S-adenosyl-L-homocysteine +N-methylpyridinium.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.88','8-hydroxyquercetin 8-O-methyltransferase','8-hydroxyquercitin 8-O-methyltransferase','NULL','Also acts on 8-hydroxykaempferol, but not on the glycosides of8-hydroxyflavonols.An enzyme from the flower buds of Lotus corniculatus.','S-adenosyl-L-methionine + 3,3,4,5,7,8-hexahydroxyflavone = S-adenosyl-L-homocysteine + 3,3,4,5,7-pentahydroxy-8-methoxyflavone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.89','Tetrahydrocolumbamine 2-O-methyltransferase','NULL','NULL','Involved in the biosynthesis of the berberine alkaloids.','S-adenosyl-L-methionine + 5,8,13,13a-tetrahydrocolumbamine = S-adenosyl-L-homocysteine + tetrahydropalmatine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.90','Methanol--corrinoid protein Co-methyltransferase','Methanol--5-hydroxybenzimidazolylcobamide Co-methyltransferase','NULL','The enzyme, which catalyzes the transfer of methyl groups frommethanol to a methanol-specific corrinoid protein (MtaC), is involvedin methanogenesis from methanol.Methylation of the corrinoid protein requires the central cobalt tobe in the Co(I) state.During methylation the cobalt is oxidized to the Co(III) state.Free cob(I)alamin can substitute for the corrinoid protein in vitro.Inactivated by oxygen and other oxidizing agents, and reactivated bycatalytic amounts of ATP and hydrogen.','Methanol + a [Co(I) methanol-specific corrinoid protein] = a [methyl-Co(III) methanol-specific corrinoid protein] + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.91','Isobutyraldoxime O-methyltransferase','Aldoxime methyltransferase','NULL','Oximes of C(4) to C(6) aldehydes can act as acceptors.The most active substrate is 2-methylbutyraldoxime.','S-adenosyl-L-methionine + 2-methylpropanal oxime = S-adenosyl-L-homocysteine + 2-methylpropanal O-methyloxime.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.92','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.93','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.94','Tabersonine 16-O-methyltransferase','11-demethyl-17-deacetylvindoline 11-methyltransferase','NULL','Involved in the biosynthesis of vindoline from tabersonine in theMadagascar periwinkle, Catharanthus roseus.','S-adenosyl-L-methionine + 16-hydroxytabersonine = S-adenosyl-L-homocysteine + 16-methoxytabersonine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.95','Tocopherol O-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + gamma-tocopherol = S-adenosyl-L-homocysteine +alpha-tocopherol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.96','Thioether S-methyltransferase','NULL','NULL','Also acts on dimethyl selenide, dimethyl telluride, diethyl sulfide,1,4-dithiane and many other thioethers.','S-adenosyl-L-methionine + dimethyl sulfide = S-adenosyl-L-homocysteine +trimethylsulfonium.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.97','3-hydroxyanthranilate 4-C-methyltransferase','NULL','NULL','Involved in the biosynthesis of the antibiotic actinomycin inStreptomyces antibioticus.','S-adenosyl-L-methionine + 3-hydroxyanthranilate = S-adenosyl-L-homocysteine + 3-hydroxy-4-methylanthranilate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.98','Diphthine synthase','NULL','NULL','The trimethylated product, diphthine, is converted into diphthamideby EC 6.3.1.14.','3 S-adenosyl-L-methionine + 2-(3-carboxy-3-aminopropyl)-L-histidine-[translation elongation factor 2] = 3 S-adenosyl-L-homocysteine +2-(3-carboxy-3-(trimethylammonio)propyl)-L-histidine-[translationelongation factor 2].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.99','3-hydroxy-16-methoxy-2,3-dihydrotabersonine N-methyltransferase','16-methoxy-2,3-dihydro-3-hydroxytabersonine methyltransferase','NULL','Involved in the biosynthesis of vindoline from tabersonine in theMadagascar periwinkle Catharanthus roseus.','S-adenosyl-L-methionine + 3-hydroxy-16-methoxy-2,3-dihydrotabersonine =S-adenosyl-L-homocysteine + deacetoxyvindoline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.100','Protein-S-isoprenylcysteine O-methyltransferase','Isoprenylcysteine carboxylmethyltransferase','NULL','C-terminal S-geranylgeranylcysteine and S-geranylcysteine residuesare also methylated, more slowly.Formerly EC 2.1.1.24.','S-adenosyl-L-methionine + protein C-terminal S-farnesyl-L-cysteine =S-adenosyl-L-homocysteine + protein C-terminal S-farnesyl-L-cysteinemethyl ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.101','Macrocin O-methyltransferase','MOMT','Magnesium.','The 3-hydroxyl group of a 2-O-methyl-6-deoxy-D-allose residue in themacrolide antibiotic macrocin acts as methyl acceptor.Isolated from Streptomyces fradiae.Tylosin is a macrolide antibiotic.Not identical with EC 2.1.1.102.','S-adenosyl-L-methionine + macrocin = S-adenosyl-L-homocysteine + tylosin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.102','Demethylmacrocin O-methyltransferase','DMOMT','Magnesium.','The 2-hydroxyl group of a 6-deoxy-D-allose residue indemethylmacrocin acts as methyl acceptor.Isolated from Streptomyces fradiae.Also acts on demethyllactenocin giving lactenocin.Involved in the biosynthesis of the macrolide antibiotic tylosin.Not identical with EC 2.1.1.101.','S-adenosyl-L-methionine + demethylmacrocin = S-adenosyl-L-homocysteine +macrocin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.103','Phosphoethanolamine N-methyltransferase','NULL','NULL','The enzyme may catalyze the transfer of two further methyl groups tothe product.','S-adenosyl-L-methionine + ethanolamine phosphate = S-adenosyl-L-homocysteine + N-methylethanolamine phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.104','Caffeoyl-CoA O-methyltransferase','Trans-caffeoyl-CoA 3-O-methyltransferase','NULL','NULL','S-adenosyl-L-methionine + caffeoyl-CoA = S-adenosyl-L-homocysteine +feruloyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.105','N-benzoyl-4-hydroxyanthranilate 4-O-methyltransferase','NULL','NULL','Involved in the biosynthesis of phytoalexins.','S-adenosyl-L-methionine + N-benzoyl-4-hydroxyanthranilate = S-adenosyl-L-homocysteine + N-benzoyl-4-methoxyanthranilate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.106','Tryptophan 2-C-methyltransferase','NULL','NULL','D-tryptophan and (indol-3-yl)pyruvate can also act as acceptors,but more slowly.','S-adenosyl-L-methionine + L-tryptophan = S-adenosyl-L-homocysteine + L-2-methyltryptophan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.107','Uroporphyrinogen-III C-methyltransferase','Adenosylmethionine-uroporphyrinogen III methyltransferase','NULL','This enzyme catalyzes two sequential methylation reactions, the firstforming precorrin-1 and the second leading to the formation ofprecorrin-2.It is the first of three steps leading to the formation of sirohemefrom uroporphyrinogen III.The second step involves an NAD(+)-dependent dehydrogenation to formsirohydrochlorin from precorrin-2 (EC 1.3.1.76) and the third stepinvolves the chelation of Fe(2+) to sirohydrochlorin to form siroheme(EC 4.99.1.4).In Saccharomyces cerevisiae, the last two steps are carried out by asingle bifunctional enzyme, Met8p.In some bacteria, steps 1-3 are catalyzed by a single multifunctionalprotein called CysG, whereas in Bacillus megaterium, three separateenzymes carry out each of the steps, with SirA being responsible forthe above reaction.Also involved in the biosynthesis of cobalamin.','(1) S-adenosyl-L-methionine + uroporphyrinogen III = S-adenosyl-L-homocysteine + precorrin-1.(2) S-adenosyl-L-methionine + precorrin-1 = S-adenosyl-L-homocysteine +precorrin-2.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.108','6-hydroxymellein O-methyltransferase','NULL','NULL','3,4-dehydro-6-hydroxymellein can also act as acceptor.6-methoxymellein is a phytoalexin produced by carrot tissue.','S-adenosyl-L-methionine + 6-hydroxymellein = S-adenosyl-L-homocysteine +6-methoxymellein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.109','Demethylsterigmatocystin 6-O-methyltransferase','NULL','NULL','Dihydrodemethylsterigmatocystin can also act as acceptor.Involved in the biosynthesis of aflatoxins in fungi.','S-adenosyl-L-methionine + 6-demethylsterigmatocystin = S-adenosyl-L-homocysteine + sterigmatocystin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.110','Sterigmatocystin 8-O-methyltransferase','O-methyltransferase II','NULL','Dihydrosterigmatocystin can also act as acceptor.Involved in the biosynthesis of aflatoxins in fungi.','(1) S-adenosyl-L-methionine + sterigmatocystin = S-adenosyl-L-homocysteine + 8-O-methylsterigmatocystin.(2) S-adenosyl-L-methionine + dihydrosterigmatocystin = S-adenosyl-L-homocysteine + 8-O-methyldihydrosterigmatocystin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.111','Anthranilate N-methyltransferase','NULL','NULL','Involved in the biosynthesis of acridine alkaloids in plant tissues.','S-adenosyl-L-methionine + anthranilate = S-adenosyl-L-homocysteine +N-methylanthranilate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.112','Glucuronoxylan 4-O-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + glucuronoxylan D-glucuronate = S-adenosyl-L-homocysteine + glucuronoxylan 4-O-methyl-D-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.113','Site-specific DNA-methyltransferase (cytosine-N(4)-specific)','Modification methylase','NULL','This is a large group of enzymes, the majority of which, with enzymesof similar site specificity listed as EC 3.1.21.3, EC 3.1.21.4 andEC 3.1.21.5, form so-called restriction-modification systems.See the REBASE database for a complete list of these enzymes:http://rebase.neb.com/rebase/','S-adenosyl-L-methionine + DNA cytosine = S-adenosyl-L-homocysteine +DNA N(4)-methylcytosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.114','Polyprenyldihydroxybenzoate methyltransferase','3,4-dihydroxy-5-hexaprenylbenzoate methyltransferase','NULL','This enzyme is involved in ubiquinone biosynthesis.Ubiquinones from different organisms have a different number ofprenyl units (for example, ubiquinone-6 in Saccharomyces, ubiquinone-9 in rat and ubiquinone-10 in human), and thus the natural substratefor the enzymes from different organisms has a different number ofprenyl units.However, the enzyme usually shows a low degree of specificityregarding the number of prenyl units.For example, the human COQ3 enzyme can restore biosynthesis ofubiquinone-6 in coq3 deletion mutants of yeast.The enzymes from yeast and rat also catalyze the methylation of3-demethylubiquinol-6 and 3-demethylubiquinol-9, respectively (thisactivity is classified as EC 2.1.1.64).','S-adenosyl-L-methionine + 3,4-dihydroxy-5-all-trans-polyprenylbenzoate =S-adenosyl-L-homocysteine + 3-methoxy-4-hydroxy-5-all-trans-polyprenylbenzoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.115','(RS)-1-benzyl-1,2,3,4-tetrahydroisoquinoline N-methyltransferase','(RS)-tetrahydrobenzylisoquinoline N-methyltransferase','NULL','Broad substrate specificity for (RS)-1-benzyl-1,2,3,4-tetrahydroisoquinolines; including coclaurine, norcoclaurine,isococlaurine, norarmepavine, norreticuline and tetrahydropapaverine.Both R- and S-enantiomers are methylated.Participates in the pathway leading to benzylisoquinoline alkaloidsynthesis in plants.The physiological substrate is likely to be coclaurine.Was earlier termed norreticuline N-methyltransferase.However, norreticuline has not been found to occur in nature and thatname does not reflect the broad specificity of the enzyme for (RS)-1-benzyl-1,2,3,4-tetrahydroisoquinolines.','S-adenosyl-L-methionine + (RS)-1-benzyl-1,2,3,4-tetrahydroisoquinoline =S-adenosyl-L-homocysteine + N-methyl-(RS)-1-benzyl-1,2,3,4-tetrahydroisoquinoline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.116','3-hydroxy-N-methyl-(S)-coclaurine 4-O-methyltransferase','NULL','NULL','Involved in isoquinoline alkaloid metabolism in plants.Has also been shown to catalyze the methylation of (RS)-laudanosoline, (S)-3-hydroxycoclaurine and (RS)-7-O-methylnoraudanosoline.','S-adenosyl-L-methionine + 3-hydroxy-N-methyl-(S)-coclaurine =S-adenosyl-L-homocysteine + (S)-reticuline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.117','(S)-scoulerine 9-O-methyltransferase','NULL','NULL','The product of this reaction is a precursor for protoberberinealkaloids in plants.','S-adenosyl-L-methionine + (S)-scoulerine = S-adenosyl-L-homocysteine +(S)-tetrahydrocolumbamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.118','Columbamine O-methyltransferase','NULL','NULL','The product of this reaction is a protoberberine alkaloid that iswidely distributed in the plant kingdom.Distinct in specificity from EC 2.1.1.88.','S-adenosyl-L-methionine + columbamine = S-adenosyl-L-homocysteine +palmatine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.119','10-hydroxydihydrosanguinarine 10-O-methyltransferase','NULL','NULL','Part of the pathway for synthesis of benzophenanthridine alkaloids inplants.','S-adenosyl-L-methionine + 10-hydroxydihydrosanguinarine = S-adenosyl-L-homocysteine + dihydrochelirubine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.120','12-hydroxydihydrochelirubine 12-O-methyltransferase','NULL','NULL','Part of the pathway for synthesis of benzophenanthridine alkaloidmacarpine in plants.','S-adenosyl-L-methionine + 12-hydroxydihydrochelirubine = S-adenosyl-L-homocysteine + dihydromacarpine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.121','6-O-methylnorlaudanosoline 5-O-methyltransferase','NULL','NULL','Nororientaline is a precursor of the alkaloid papaverine.','S-adenosyl-L-methionine + 6-O-methylnorlaudanosoline = S-adenosyl-L-homocysteine + nororientaline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.122','(S)-tetrahydroprotoberberine N-methyltransferase','Tetrahydroprotoberberine cis-N-methyltransferase','NULL','Involved in the biosynthesis of isoquinoline alkaloids in plants.','S-adenosyl-L-methionine + (S)-7,8,13,14-tetrahydroprotoberberine =S-adenosyl-L-homocysteine + cis-N-methyl-(S)-7,8,13,14-tetrahydroprotoberberine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.123','[Cytochrome c]-methionine S-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + [cytochrome c]-methionine = S-adenosyl-L-homocysteine + [cytochrome c]-S-methyl-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.124','[Cytochrome c]-arginine N-methyltransferase','NULL','NULL','Formerly EC 2.1.1.23.','S-adenosyl-L-methionine + [cytochrome c]-arginine = S-adenosyl-L-homocysteine + [cytochrome c]-N(omega)-methyl-arginine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.125','Histone-arginine N-methyltransferase','Histone protein methylase','NULL','Forms the N(omega)-monomethyl- and N(omega),N(omega)-dimethyl,but not the N(omega),N(omega)-dimethyl-arginine derivatives.The name protein methylase I is misleading since it has been used fora number of enzymes with different specificities.Formerly EC 2.1.1.23.','S-adenosyl-L-methionine + arginine-[histone] = S-adenosyl-L-homocysteine+ N(omega)-methyl-arginine-[histone].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.126','[Myelin basic protein]-arginine N-methyltransferase','Myelin basic protein methylase','NULL','Forms the N(omega)-monomethyl-, N(omega),N(omega)-dimethyl- andN(omega),N(omega)-dimethyl-arginine derivatives.The name protein methylase I is misleading since it has been used fora number of enzymes with different specificities.Formerly EC 2.1.1.23.','S-adenosyl-L-methionine + [myelin basic protein]-arginine = S-adenosyl-L-homocysteine + [myelin basic protein]-N(omega)-methyl-arginine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.127','[Ribulose-bisphosphate carboxylase]-lysine N-methyltransferase','Ribulose-bisphosphate-carboxylase/oxygenase N-methyltransferase','NULL','The enzyme catalyzes three successive methylations of Lys-14 in thelarge subunits of hexadecameric higher plant ribulose-bisphosphate-carboxylase (EC 4.1.1.39).Only the three methylated form is observed.The enzyme from pea (Pisum sativum) also three-methylates a specificlysine in the chloroplastic isoforms of fructose-bisphosphatealdolase (EC 4.1.2.13).','3 S-adenosyl-L-methionine + [ribulose-bisphosphate carboxylase]-L-lysine= 3 S-adenosyl-L-homocysteine + [ribulose-bisphosphate carboxylase]-N(6),N(6),N(6)-trimethyl-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.128','(RS)-norcoclaurine 6-O-methyltransferase','NULL','NULL','The enzyme will also catalyze the 6-O-methylation of (RS)-norlaudanosoline to form 6-O-methyl-norlaudanosoline, but thisalkaloid has not been found to occur in plants.','S-adenosyl-L-methionine + (RS)-norcoclaurine = S-adenosyl-L-homocysteine+ (RS)-coclaurine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.129','Inositol 4-methyltransferase','Myo-inositol 4-O-methyltransferase','NULL','The enzyme from Vigna umbellata is highly specific for S-adenosyl-L-methionine.The enzyme also methylates 1L-1,2,4/3,5-cyclohexanepentol, 2,4,6/3,5-pentahydroxycyclohexanone, D,L-2,3,4,6/5-pentacyclohexanone and2,2-anhydro-2-C-hydroxymethyl-myo-inositol, but at lower rates thanthat of myo-inositol.Formerly EC 2.1.1.134.','S-adenosyl-L-methionine + myo-inositol = S-adenosyl-L-homocysteine +1D-4-O-methyl-myo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.130','Precorrin-2 C(20)-methyltransferase','Precorrin-2 C20-methyltransferase','NULL','NULL','S-adenosyl-L-methionine + precorrin-2 = S-adenosyl-L-homocysteine +precorrin-3A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.131','Precorrin-3B C(17)-methyltransferase','Precorrin-3 methylase','NULL','In the aerobic cobalamin biosythesis pathway, four enzymes areinvolved in the conversion of precorrin-3A to precorrin-6A.The first of the four steps is carried out by EC 1.14.13.83 (CobG),yielding precorrin-3B as the product.This is followed by three methylation reactions, which introduce amethyl group at C-17 (CobJ; EC 2.1.1.131), C-11 (CobM; 2.1.1.133) andC-1 (CobF; EC 2.1.1.152) of the macrocycle, giving rise to precorrin-4, precorrin-5 and precorrin-6A, respectively.','S-adenosyl-L-methionine + precorrin-3B = S-adenosyl-L-homocysteine +precorrin-4.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.132','Precorrin-6B C(5,15)-methyltransferase (decarboxylating)','Precorrin-6 methyltransferase','NULL','The enzyme, which participates in the aerobic adenosylcobalaminbiosynthesis pathway, has S-adenosyl-L-methionine-dependentmethyltransferase and decarboxylase activities.The enzyme is a fusion protein with two active sites; one catalyzesthe methylation at C(15) and the decarboxylation, while the othercatalyzes the methylation at C(5).','2 S-adenosyl-L-methionine + precorrin-6B = 2 S-adenosyl-L-homocysteine +precorrin-8X + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.133','Precorrin-4 C(11)-methyltransferase','Precorrin-3 methylase','NULL','In the aerobic cobalamin biosythesis pathway, four enzymes areinvolved in the conversion of precorrin-3A to precorrin-6A.The first of the four steps is carried out by EC 1.14.13.83 (CobG),yielding precorrin-3B as the product.This is followed by three methylation reactions, which introduce amethyl group at C-17 (CobJ; EC 2.1.1.131), C-11 (CobM; 2.1.1.133) andC-1 (CobF; EC 2.1.1.152) of the macrocycle, giving rise to precorrin-4, precorrin-5 and precorrin-6A, respectively.','S-adenosyl-L-methionine + precorrin-4 = S-adenosyl-L-homocysteine +precorrin-5.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.134','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.135','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.136','Chlorophenol O-methyltransferase','Halogenated phenol O-methyltransferase','NULL','The enzyme from Trichoderma virgatum, when cultured in the presenceof halogenated phenol, also acts on a range of mono-, di- andtrichlorophenols.','S-adenosyl-L-methionine + trichlorophenol = S-adenosyl-L-homocysteine +trichloroanisole.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.137','Arsenite methyltransferase','Methylarsonite methyltransferase','NULL','An enzyme of the biotransformation pathway that formsdimethylarsinate from inorganic arsenite and arsenate.It methylates arsenite to form methylarsonate, Me-AsO(3)H(2), whichis reduced by EC 1.20.4.2 to methylarsonite, Me-As(OH)(2).Methylarsonite is also a substrate for this enzyme which converts itinto the much less toxic compound dimethylarsinate (cacodylate),Me(2)As(O)-OH.','(1) S-adenosyl-L-methionine + arsenite = S-adenosyl-L-homocysteine +methylarsonate.(2) S-adenosyl-L-methionine + methylarsonite = S-adenosyl-L-homocysteine+ dimethylarsinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.138','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.139','3-demethylstaurosporine O-methyltransferase','3-demethoxy-3-hydroxystaurosporine O-methyltransferase','NULL','Catalyzes the final step in the biosynthesis of staurosporine,an alkaloidal antibiotic that is a potent inhibitor of proteinkinases, especially protein kinase C.','S-adenosyl-L-methionine + 3-demethylstaurosporine = S-adenosyl-L-homocysteine + staurosporine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.140','(S)-coclaurine-N-methyltransferase','NULL','NULL','The enzyme is specific for the (S)-isomer of coclaurine.Norcoclaurine can also act as an acceptor.','S-adenosyl-L-methionine + (S)-coclaurine = S-adenosyl-L-homocysteine +(S)-N-methylcoclaurine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.141','Jasmonate O-methyltransferase','Jasmonic acid carboxyl methyltransferase','NULL','9,10-dihydrojasmonic acid is a poor substrate for the enzyme.The enzyme does not convert 12-oxo-phytodienoic acid (a precursor ofjasmonic acid), salicylic acid, benzoic acid, linolenic acid orcinnamic acid into their corresponding methyl esters.Enzyme activity is inhibited by the presence of divalent cations,e.g., Ca(2+), Cu(2+), Mg(2+) and Zn(2+).','S-adenosyl-L-methionine + jasmonate = S-adenosyl-L-homocysteine + methyljasmonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.142','Cycloartenol 24-C-methyltransferase','Sterol C-methyltransferase','NULL','S-adenosyl-L-methionine methylates the Si face of the 24(25)-doublebond with elimination of a hydrogen atom from the pro-Z methyl groupat C-25.','S-adenosyl-L-methionine + cycloartenol = S-adenosyl-L-homocysteine +(24R)-24-methylcycloart-25-en-3-beta-ol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.143','24-methylenesterol C-methyltransferase','24-methylenelophenol C-24(1)-methyltransferase','NULL','This is the second methylation step of plant sterol biosynthesis (cf.EC 2.1.1.142).','S-adenosyl-L-methionine + 24-methylenelophenol = S-adenosyl-L-homocysteine + (Z)-24-ethylidenelophenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.144','Trans-aconitate 2-methyltransferase','NULL','NULL','Also catalyzes the formation of the methyl monoester of cis-aconitate, isocitrate and citrate, but more slowly.While the enzyme from Escherichia coli forms (E)-3-(methoxycarbonyl)-pent-2-enedioate as the product, that from Saccharomyces cerevisiaeforms (E)-2-(methoxycarbonylmethyl)butenedioate and is thereforeclassified as a separate enzyme (cf. EC 2.1.1.145).','S-adenosyl-L-methionine + trans-aconitate = S-adenosyl-L-homocysteine +(E)-3-(methoxycarbonyl)pent-2-enedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.145','Trans-aconitate 3-methyltransferase','NULL','NULL','Also catalyzes the formation of the methyl monoester of cis-aconitate, isocitrate and citrate, but more slowly.While the enzyme from Saccharomyces cerevisiae forms (E)-2-(methoxycarbonylmethyl)butenedioate as the product, that fromEscherichia coli forms (E)-3-(methoxycarbonyl)-pent-2-enedioate andis therefore classified as a separate enzyme (cf. EC 2.1.1.144).','S-adenosyl-L-methionine + trans-aconitate = S-adenosyl-L-homocysteine +(E)-2-(methoxycarbonylmethyl)butenedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.146','(Iso)eugenol O-methyltransferase','NULL','NULL','Acts on eugenol and chavicol as well as isoeugenol.','S-adenosyl-L-methionine + isoeugenol = S-adenosyl-L-homocysteine +isomethyleugenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.147','Corydaline synthase','NULL','NULL','Also acts on 7,8-dihydropalmatine.','S-adenosyl-L-methionine + palmatine + 2 NADPH = S-adenosyl-L-homocysteine+ corydaline + 2 NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.148','Thymidylate synthase (FAD)','FDTS','NULL','Contains FAD.All thymidylate synthases catalyze a reductive methylation involvingthe transfer of the methylene group of 5,10-methylenetetrahydrofolateto the C5-position of dUMP and a two electron reduction of themethylene group to a methyl group.Unlike the classical thymidylate synthase, ThyA (EC 2.1.1.45), whichuses folate as both a 1-carbon donor and a source of reducingequivalents, this enzyme uses a flavin coenzyme as a source ofreducing equivalents, which are derived from NADPH.','5,10-methylenetetrahydrofolate + dUMP + NADPH = dTMP + tetrahydrofolate +NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.149','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.150','Isoflavone 7-O-methyltransferase','NULL','NULL','The enzyme from alfalfa can methylate daidzein, genistein and6,7,4-trihydroxyisoflavone but not flavones or flavanones.','S-adenosyl-L-methionine + a 7-hydroxyisoflavone = S-adenosyl-L-homocysteine + a 7-methoxyisoflavone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.151','Cobalt-factor II C(20)-methyltransferase','Cobalt-precorrin-2 C(20)-methyltransferase','NULL','Involved in the anaerobic biosynthesis of vitamin B12.','S-adenosyl-L-methionine + cobalt-factor II = S-adenosyl-L-homocysteine +cobalt-factor III.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.152','Precorrin-6A synthase (deacetylating)','Precorrin-6X synthase (deacetylating)','NULL','In the aerobic cobalamin biosythesis pathway, four enzymes areinvolved in the conversion of precorrin-3A to precorrin-6A.The first of the four steps is carried out by EC 1.14.13.83 (CobG),yielding precorrin-3B as the product.This is followed by three methylation reactions, which introduce amethyl group at C-17 (CobJ; EC 2.1.1.131), C-11 (CobM; 2.1.1.133) andC-1 (CobF; EC 2.1.1.152) of the macrocycle, giving rise to precorrin-4, precorrin-5 and precorrin-6A, respectively.','S-adenosyl-L-methionine + precorrin-5 + H(2)O = S-adenosyl-L-homocysteine+ precorrin-6A + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.153','Vitexin 2-O-rhamnoside 7-O-methyltransferase','NULL','NULL','The flavonoids vitexin and isovitexin 2-O-arabinoside do not act assubstrates for the enzyme from oats (Avena sativa).','S-adenosyl-L-methionine + vitexin 2-O-beta-L-rhamnoside = S-adenosyl-L-homocysteine + 7-O-methylvitexin 2-O-beta-L-rhamnoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.154','Isoliquiritigenin 2-O-methyltransferase','Chalcone OMT','NULL','Not identical to EC 2.1.1.65.While EC 2.1.1.154 can use licodione as a substrate, EC 2.1.1.65cannot use isoliquiritigenin as a substrate.','S-adenosyl-L-methionine + isoliquiritigenin = S-adenosyl-L-homocysteine +2-O-methylisoliquiritigenin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.155','Kaempferol 4-O-methyltransferase','F 4-OMT','NULL','The enzyme acts on the hydroxy group in the 4-position of someflavones, flavanones and isoflavones.Kaempferol, apigenin and kaempferol triglucoside are substrates,as is genistein, which reacts more slowly.Compounds with an hydroxy group in the 3 and 4 positions, such asquercetin and eriodictyol, do not act as substrates.Similar to EC 2.1.1.75 and EC 2.1.1.83.','S-adenosyl-L-methionine + kaempferol = S-adenosyl-L-homocysteine +kaempferide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.156','Glycine/sarcosine N-methyltransferase','Glycine sarcosine N-methyltransferase','NULL','Cells of the oxygen-evolving halotolerant cyanobacterium Aphanocthecehalophytica synthesize betaine from glycine by a three-stepmethylation process.This is the first enzyme and it leads to the formation of eithersarcosine or N,N-dimethylglycine, which is further methylated toyield betaine (N,N,N-trimethylglycine) by the action of EC 2.1.1.157.Differs from EC 2.1.1.20, as it can further methylate the product ofthe first reaction.Acetate, dimethylglycine and S-adenosyl-L-homocysteine can inhibitthe reaction.','2 S-adenosyl-L-methionine + glycine = 2 S-adenosyl-L-homocysteine +N,N-dimethylglycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.157','Sarcosine/dimethylglycine N-methyltransferase','ApDMT','NULL','Cells of the oxygen-evolving halotolerant cyanobacterium Aphanocthecehalophytica synthesize betaine from glycine by a three-stepmethylation process.The first enzyme, EC 2.1.1.156, leads to the formation of eithersarcosine or N,N-dimethylglycine, which is further methylated toyield betaine (N,N,N-trimethylglycine) by the action of this enzyme.Both of these enzymes can catalyze the formation ofN,N-dimethylglycine from sarcosine.The reactions are strongly inhibited by S-adenosyl-L-homocysteine.','2 S-adenosyl-L-methionine + sarcosine = 2 S-adenosyl-L-homocysteine +betaine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.158','7-methylxanthosine synthase','Xanthosine methyltransferase','NULL','The enzyme is specific for xanthosine, as XMP and xanthine cannot actas substrates.The enzyme does not have N(1)- or N(3)- methylation activity.This is the first methylation step in the production of caffeine.','S-adenosyl-L-methionine + xanthosine = S-adenosyl-L-homocysteine +7-methylxanthosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.159','Theobromine synthase','Monomethylxanthine methyltransferase','NULL','Can also catalyze the conversion of paraxanthine into caffeine,although the paraxanthine pathway is considered to be a minor pathwayfor caffeine biosynthesis.This is the third enzyme in the caffeine biosynthesis pathway.','S-adenosyl-L-methionine + 7-methylxanthine = S-adenosyl-L-homocysteine +3,7-dimethylxanthine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.160','Caffeine synthase','3N-methyltransferase','NULL','Paraxanthine is the best substrate for this enzyme but theparaxanthine pathway is considered to be a minor pathway for caffeinebiosynthesis.','(1) S-adenosyl-L-methionine + 3,7-dimethylxanthine = S-adenosyl-L-homocysteine + 1,3,7-trimethylxanthine.(2) S-adenosyl-L-methionine + 1,7-dimethylxanthine = S-adenosyl-L-homocysteine + 1,3,7-trimethylxanthine.(3) S-adenosyl-L-methionine + 7-methylxanthine = S-adenosyl-L-homocysteine + 3,7-dimethylxanthine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.161','Dimethylglycine N-methyltransferase','DMT','NULL','This enzyme, from the marine cyanobacterium Synechococcus sp. strainWH8102, differs from EC 2.1.1.157 in that it cannot use sarcosine asan alternative substrate.Betaine is a compatible solute that enables cyanobacteria to copewith osmotic stress by maintaining a positive cellular turgor.','S-adenosyl-L-methionine + N,N-dimethylglycine = S-adenosyl-L-homocysteine+ betaine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.162','Glycine/sarcosine/dimethylglycine N-methyltransferase','Glycine sarcosine dimethylglycine N-methyltransferase','NULL','Unlike EC 2.1.1.156, EC 2.1.1.157 and EC 2.1.1.161, this enzyme, fromthe halophilic methanoarchaeon Methanohalophilus portucalensis,can methylate glycine and all of its intermediates to form thecompatible solute betaine.','3 S-adenosyl-L-methionine + glycine = 3 S-adenosyl-L-homocysteine +betaine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.163','Demethylmenaquinone methyltransferase','2-demethylmenaquinone methyltransferase','NULL','The enzyme catalyzes the last step in menaquinone biosynthesis.It is able to accept substrates with varying polyprenyl side chainlength (the chain length is determined by polyprenyl diphosphatesynthase).The enzyme from Escherichia coli also catalyzes the conversion of2-methoxy-6-octaprenyl-1,4-benzoquinone to 5-methoxy-2-methyl-3-octaprenyl-1,4-benzoquinone during the biosynthesis of ubiquinone.The enzyme probably acts on menaquinol rather than menaquinone.','A demethylmenaquinone + S-adenosyl-L-methionine = a menaquinol +S-adenosyl-L-homocysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.164','Demethylrebeccamycin-D-glucose O-methyltransferase','NULL','NULL','Catalyzes the last step in the biosynthesis of rebeccamycin,an indolocarbazole alkaloid produced by the actinobacteriumLechevalieria aerocolonigenes.The enzyme is able to use a wide variety substrates, toleratingvariation on the imide heterocycle, deoxygenation of the sugarmoiety, and even indolocarbazole glycoside anomers.','4-demethylrebeccamycin + S-adenosyl-L-methionine = rebeccamycin +S-adenosyl-L-homocysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.165','Methyl halide transferase','Methyl chloride transferase','NULL','This enzyme contributes to the methyl halide emissions fromArabidopsis.','S-adenosyl-L-methionine + iodide = S-adenosyl-L-homocysteine + methyliodide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.166','23S rRNA (uridine(2552)-2-O)-methyltransferase','FTSJ','NULL','The enzyme catalyzes the 2-O-methylation of the universallyconserved U2552 in the A loop of 23S rRNA.','S-adenosyl-L-methionine + uridine(2552) in 23S rRNA = S-adenosyl-L-homocysteine + 2-O-methyluridine(2552) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.167','27S pre-rRNA (guanosine(2922)-2-O)-methyltransferase','NULL','NULL','Spb1p is a site-specific 2-O-ribose RNA methyltransferase thatcatalyzes the formation of 2-O-methylguanosine(2922), a universallyconserved position of the catalytic center of the ribosome that isessential for translation.2-O-Methylguanosine(2922) is formed at a later stage of theprocessing, during the maturation of of the 27S pre-rRNA.In absence of snR52, Spb1p can also catalyze the formation ofuridine(2921).','S-adenosyl-L-methionine + guanosine(2922) in 27S pre-rRNA = S-adenosyl-L-homocysteine + 2-O-methylguanosine(2922) in 27S pre-rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.168','21S rRNA (uridine(2791)-2-O)-methyltransferase','Mitochondrial rRNA MTase 2','NULL','The enzyme catalyzes the methylation of uridine(2791) ofmitochondrial 21S rRNA.','S-adenosyl-L-methionine + uridine(2791) in 21S rRNA = S-adenosyl-L-homocysteine + 2-O-methyluridine(2791) in 21S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.169','Tricetin 3,4,5-O-trimethyltransferase','NULL','NULL','The enzyme from Triticum aestivum catalyzes the sequentialO-methylation of tricetin via 3-O-methyltricetin, 3,5-O-methyltricetin to 3,4,5-O-trimethyltricetin (2).','3 S-adenosyl-L-methionine + tricetin = 3 S-adenosyl-L-homocysteine +3,4,5-O-trimethyltricetin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.170','16S rRNA (guanine(527)-N(7))-methyltransferase','NULL','NULL','The enzyme specifically methylates guanine(527) at N(7) in 16S rRNA.','S-adenosyl-L-methionine + guanine(527) in 16S rRNA = S-adenosyl-L-homocysteine + N(7)-methylguanine(527) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.171','16S rRNA (guanine(966)-N(2))-methyltransferase','M(2)G966 methyltransferase','NULL','The enzyme efficiently methylates guanine(966) of the assembled 30Ssubunits in vitro.Protein-free 16S rRNA is not a substrate for RsmD.The enzyme specifically methylates guanine(966) at N(2) in 16S rRNA.Formerly EC 2.1.1.52.','S-adenosyl-L-methionine + guanine(966) in 16S rRNA = S-adenosyl-L-homocysteine + N(2)-methylguanine(966) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.172','16S rRNA (guanine(1207)-N(2))-methyltransferase','M(2)G1207 methyltransferase','NULL','The enzyme reacts well with 30S subunits reconstituted from 16S RNAtranscripts and 30S proteins but is almost inactive with thecorresponding free RNA.The enzyme specifically methylates guanine(1207) at N(2) in 16S rRNA.Formerly EC 2.1.1.52.','S-adenosyl-L-methionine + guanine(1207) in 16S rRNA = S-adenosyl-L-homocysteine + N(2)-methylguanine(1207) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.173','23S rRNA (guanine(2445)-N(2))-methyltransferase','NULL','NULL','The enzyme methylates 23S rRNA in vitro, assembled 50S subunits arenot a substrate.The enzyme specifically methylates guanine(2445) at N(2) in 23S rRNA.Formerly EC 2.1.1.52.','S-adenosyl-L-methionine + guanine(2445) in 23S rRNA = S-adenosyl-L-homocysteine + N(2)-methylguanine(2445) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.174','23S rRNA (guanine(1835)-N(2))-methyltransferase','Ribosomal RNA large subunit methyltransferase G','NULL','The enzyme methylates 23S rRNA in vitro, assembled 50S subunits arenot a substrate.The enzyme specifically methylates guanine(1835) at N(2) in 23S rRNA.Formerly EC 2.1.1.52.','S-adenosyl-L-methionine + guanine(1835) in 23S rRNA = S-adenosyl-L-homocysteine + N(2)-methylguanine(1835) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.175','Tricin synthase','NULL','NULL','The enzymes from Oryza sativa (ROMT-15 and ROMT-17) catalyzes thestepwise methylation of tricetin to its 3-mono- and 3,5-dimethylethers.In contrast with the wheat enzyme (EC 2.1.1.167, tricetin 3,4,5-O-trimethyltransferase), tricetin dimethyl ether is not converted toits 3,4,5-trimethylated ether derivative.The enzymes from Hordeum vulgare (HvOMT1) and from Zea mays (ZmOMT1)form the 3,5-dimethyl derivative as the major product.','2 S-adenosyl-L-methionine + tricetin = 2 S-adenosyl-L-homocysteine +3,5-O-dimethyltricetin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.176','16S rRNA (cytosine(967)-C(5))-methyltransferase','NULL','NULL','The enzyme specifically methylates cytosine(967) at C(5) in 16S rRNA.','S-adenosyl-L-methionine + cytosine(967) in 16S rRNA = S-adenosyl-L-homocysteine + 5-methylcytosine(967) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.177','23S rRNA (pseudouridine(1915)-N(3))-methyltransferase','Pseudouridine methyltransferase','NULL','YbeA does not methylate uridine at position 1915.','S-adenosyl-L-methionine + pseudouridine(1915) in 23S rRNA = S-adenosyl-L-homocysteine + N(3)-methylpseudouridine(1915) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.178','16S rRNA (cytosine(1407)-C(5))-methyltransferase','NULL','NULL','The enzyme specifically methylates cytosine(1407) at C(5) in 16SrRNA.','S-adenosyl-L-methionine + cytosine(1407) in 16S rRNA = S-adenosyl-L-homocysteine + 5-methylcytosine(1407) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.179','16S rRNA (guanine(1405)-N(7))-methyltransferase','Sisomicin-gentamicin methyltransferase','NULL','The enzyme specifically methylates guanine(1405) at N(7) in 16S rRNA.The enzyme from the antibiotic-producing bacterium Micromonosporazionensis methylates guanine(1405) in 16S rRNA to 7-methylguanine,thereby rendering the ribosome resistant to 4,6-disubstituteddeoxystreptamine aminoglycosides, which include gentamicins andkanamycins.','S-adenosyl-L-methionine + guanine(1405) in 16S rRNA = S-adenosyl-L-homocysteine + 7-methylguanine(1405) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.180','16S rRNA (adenine(1408)-N(1))-methyltransferase','NULL','NULL','The enzyme provides a panaminoglycoside-resistant nature throughinterference with the binding of aminoglycosides toward the A site of16S rRNA through N(1)-methylation at position adenine(1408).','S-adenosyl-L-methionine + adenine(1408) in 16S rRNA = S-adenosyl-L-homocysteine + N(1)-methyladenine(1408) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.181','23S rRNA (adenine(1618)-N(6))-methyltransferase','rRNA large subunit methyltransferase F','NULL','The recombinant YbiN protein is able to methylate partiallydeproteinized 50 S ribosomal subunit, but neither the completelyassembled 50 S subunits nor completely deproteinized 23 S rRNA.Formerly EC 2.1.1.48.','S-adenosyl-L-methionine + adenine(1618) in 23S rRNA = S-adenosyl-L-homocysteine + rRNA containing N(6)-methyladenine(1618) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.182','16S rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase','NULL','NULL','KsgA introduces the most highly conserved ribosomal RNA modification,the dimethylation of adenine(1518) and adenine(1519) in 16S rRNA.Strains lacking the methylase are resistant to kasugamycin.Formerly EC 2.1.1.48.','4 S-adenosyl-L-methionine + adenine(1518)/adenine(1519) in 16S rRNA =4 S-adenosyl-L-homocysteine + N(6)-dimethyladenine(1518)/N(6)-dimethyladenine(1519) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.183','18S rRNA (adenine(1779)-N(6)/adenine(1780)-N(6))-dimethyltransferase','NULL','NULL','DIM1 is involved in pre-rRNA processing.Formerly EC 2.1.1.48.','4 S-adenosyl-L-methionine + adenine(1779)/adenine(1780) in 18S rRNA =4 S-adenosyl-L-homocysteine + N(6)-dimethyladenine(1779)/N(6)-dimethyladenine(1780) in 18S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.184','23S rRNA (adenine(2085)-N(6))-dimethyltransferase','NULL','NULL','ErmC is a methyltransferase that confers resistance to the macrolide-lincosamide-streptogramin B group of antibiotics by catalyzing themethylation of 23S rRNA at adenine(2085).Formerly EC 2.1.1.48.','2 S-adenosyl-L-methionine + adenine(2085) in 23S rRNA = 2 S-adenosyl-L-homocysteine + N(6)-dimethyladenine(2085) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.185','23S rRNA (guanosine(2251)-2-O)-methyltransferase','NULL','NULL','The enzyme catalyzes the methylation of guanosine(2251),a modification conserved in the peptidyltransferase domain of 23SrRNA.','S-adenosyl-L-methionine + guanosine(2251) in 23S rRNA = S-adenosyl-L-homocysteine + 2-O-methylguanosine(2251) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.186','23S rRNA (cytidine(2498)-2-O)-methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + cytidine(2498) in 23S rRNA = S-adenosyl-L-homocysteine + 2-O-methylcytidine(2498) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.187','23S rRNA (guanine(745)-N(1))-methyltransferase','23S rRNA m(1)G(745) methyltransferase','NULL','The enzyme specifically methylates guanine(745) at N(1) in 23S rRNA.Formerly EC 2.1.1.51.','S-adenosyl-L-methionine + guanine(745) in 23S rRNA = S-adenosyl-L-homocysteine + N(1)-methylguanine(745) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.188','23S rRNA (guanine(748)-N(1))-methyltransferase','23S rRNA m(1)G(748) methyltransferase','NULL','The enzyme specifically methylates guanine(748) at N(1) in 23S rRNA.The methyltransferase RlmA(II) confers resistance to the macrolideantibiotic tylosin in the drug-producing strain Streptomyces fradiae.Formerly EC 2.1.1.51.','S-adenosyl-L-methionine + guanine(748) in 23S rRNA = S-adenosyl-L-homocysteine + N(1)-methylguanine(748) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.189','23S rRNA (uracil(747)-C(5))-methyltransferase','NULL','NULL','The enzyme specifically methylates uracil(747) at C(5) in 23S rRNA.','S-adenosyl-L-methionine + uracil(747) in 23S rRNA = S-adenosyl-L-homocysteine + 5-methyluracil(747) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.190','23S rRNA (uracil(1939)-C(5))-methyltransferase','RNA uridine methyltransferase A','NULL','The enzyme specifically methylates uracil(1939) at C(5) in 23S rRNA.The enzyme contains an [4Fe-4S] cluster coordinated by four conservedcysteine residues.','S-adenosyl-L-methionine + uracil(1939) in 23S rRNA = S-adenosyl-L-homocysteine + 5-methyluracil(1939) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.191','23S rRNA (cytosine(1962)-C(5))-methyltransferase','rRNA large subunit methyltransferase I','NULL','The enzyme specifically methylates cytosine(1962) at C(5) in 23SrRNA.','S-adenosyl-L-methionine + cytosine(1962) in 23S rRNA = S-adenosyl-L-homocysteine + 5-methylcytosine(1962) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.192','23S rRNA (adenine(2503)-C(2))-methyltransferase','NULL','Iron-sulfur.','This enzyme is a member of the AdoMet radical (radical SAM) family.S-adenosyl-L-methionine acts as both a radical generator and as thesource of the appended methyl group.RlmN is an endogenous enzyme used by the cell to refine functions ofthe ribosome in protein synthesis.The enzyme methylates adenosine by a radical mechanism with CH(2)from the S-adenosyl-L-methionine and retention of the hydrogen at C-2of adenosine(2503) of 23S rRNA.It will also methylate 8-methyladenosine(2503) of 23S rRNA.Cf. EC 2.1.1.224.','2 S-adenosyl-L-methionine + adenine(2503) in 23S rRNA = S-adenosyl-L-homocysteine + L-methionine + 5-deoxyadenosine + 2-methyladenine(2503)in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.193','16S rRNA (uracil(1498)-N(3))-methyltransferase','M(3)U(1498) specific methyltransferase','NULL','The enzyme specifically methylates uracil(1498) at N(3) in 16S rRNA.','S-adenosyl-L-methionine + uracil(1498) in 16S rRNA = S-adenosyl-L-homocysteine + N(3)-methyluracil(1498) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.194','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.195','Cobalt-precorrin-5B (C(1))-methyltransferase','Cobalt-precorrin-6A synthase','NULL','This enzyme catylses the C-1 methylation of cobalt-precorrin-5B inthe anaerobic (early cobalt insertion) pathway of adenosylcobalaminbiosynthesis.','Cobalt-precorrin-5B + S-adenosyl-L-methionine = cobalt-precorrin-6A +S-adenosyl-L-homocysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.196','Cobalt-precorrin-6B (C(15))-methyltransferase (decarboxylating)','Cobalt-precorrin-6Y (C(15))-methyltransferase (decarboxylating)','NULL','This enzyme catalyzes both methylation at C-15 and decarboxylation ofthe C-12 acetate side chain of cobalt-precorrin-6B, a step in theanaerobic (early cobalt insertion) adenosylcobalamin biosynthesispathway.','Cobalt-precorrin-6B + S-adenosyl-L-methionine = cobalt-precorrin-7 +S-adenosyl-L-homocysteine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.197','Malonyl-[acyl-carrier protein] O-methyltransferase','NULL','NULL','Involved in an early step of biotin biosynthesis in Gram-negativebacteria.This enzyme catalyzes the transfer of a methyl group to the omega-carboxyl group of malonyl-[acyl-carrier protein] forming a methylester.The methyl ester is recognized by the fatty acid synthetic enzymes,which process it via the fatty acid elongation cycle to give pimelyl-[acyl-carrier-protein] methyl ester.While the enzyme can also accept malonyl-CoA, it has a much higheractivity with malonyl-[acyl-carrier protein].','S-adenosyl-L-methionine + malonyl-[acyl-carrier protein] = S-adenosyl-L-homocysteine + malonyl-[acyl-carrier protein] methyl ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.198','16S rRNA (cytidine(1402)-2-O)-methyltransferase','NULL','NULL','RsmI catalyzes the 2-O-methylation of cytidine(1402) and RsmH(EC 2.1.1.199) catalyzes the N(4)-methylation of cytidine(1402) in16S rRNA.Both methylations are necessary for efficient translation initiationat the UUG and GUG codons.','S-adenosyl-L-methionine + cytidine(1402) in 16S rRNA = S-adenosyl-L-homocysteine + 2-O-methylcytidine(1402) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.199','16S rRNA (cytosine(1402)-N(4))-methyltransferase','NULL','NULL','RsmH catalyzes the N(4)-methylation of cytosine(1402) and RsmI(EC 2.1.1.198) catalyzes the 2-O-methylation of cytosine(1402) in16S rRNA.Both methylations are necessary for efficient translation initiationat the UUG and GUG codons.','S-adenosyl-L-methionine + cytosine(1402) in 16S rRNA = S-adenosyl-L-homocysteine + N(4)-methylcytosine(1402) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.200','tRNA (cytidine(32)/uridine(32)-2-O)-methyltransferase','TrMet(Xm32)','NULL','In Escherichia coli TrmJ is the only methyltransferase responsiblefor the formation of 2-O-methylcytidine(32) in tRNA.No methylation of cytosine(34) in tRNA(Leu)(CAA).In vitro the enzyme 2-O-methylates cytidine(34) of tRNA(Ser1) anduridine(32) of tRNA(Gln2).','(1) S-adenosyl-L-methionine + cytidine(32) in tRNA = S-adenosyl-L-homocysteine + 2-O-methylcytidine(32) in tRNA.(2) S-adenosyl-L-methionine + uridine(32) in tRNA = S-adenosyl-L-homocysteine + 2-O-methyluridine(32) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.201','2-methoxy-6-polyprenyl-1,4-benzoquinol methylase','NULL','NULL','This enzyme is involved in ubiquinone biosynthesis.Ubiquinones from different organisms have a different number ofprenyl units (for example, ubiquinone-6 in Saccharomyces, ubiquinone-9 in rat and ubiquinone-10 in human), and thus the natural substratefor the enzymes from different organisms has a different number ofprenyl units.However, the enzyme usually shows a low degree of specificityregarding the number of prenyl units.For example, when the COQ5 gene from Saccharomyces cerevisiae isintroduced into Escherichia coli, it complements the respiratorydeficiency of an ubiE mutant.The bifunctional enzyme from E.coli also catalyzes the methylation ofdemethylmenaquinol-8 (this activity is classified as EC 2.1.1.163).','S-adenosyl-L-methionine + 2-methoxy-6-all-trans-polyprenyl-1,4-benzoquinol = S-adenosyl-L-homocysteine + 6-methoxy-3-methyl-2-all-trans-polyprenyl-1,4-benzoquinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.202','Multisite-specific tRNA:(cytosine-C(5))-methyltransferase','NULL','NULL','The enzyme from Saccharomyces cerevisiae is responsible for complete5-methylcytosine methylations of yeast tRNA.The incidence of modification depends on the cytosine position intRNA.At positions 34 and 40, 5-methylcytosine is found only in two yeasttRNAs (tRNA(Leu)(CUA) and tRNA(Phe)(GAA), respectively), whereas mostother elongator yeast tRNAs bear either 5-methylcytosine(48) or5-methylcytosine(49), but never both in the same tRNA molecule.The formation of 5-methylcytosine(34) and 5-methylcytosine(40) is astrictly intron-dependent process, whereas the formation of5-methylcytosine(48) and 5-methylcytosine(49) is an intron-independent process.Formerly EC 2.1.1.29.','(1) S-adenosyl-L-methionine + cytosine(34) in tRNA precursor =S-adenosyl-L-homocysteine + 5-methylcytosine(34) in tRNA precursor.(2) S-adenosyl-L-methionine + cytosine(40) in tRNA precursor =S-adenosyl-L-homocysteine + 5-methylcytosine(40) in tRNA precursor.(3) S-adenosyl-L-methionine + cytosine(48) in tRNA precursor =S-adenosyl-L-homocysteine + 5-methylcytosine(48) in tRNA precursor.(4) S-adenosyl-L-methionine + cytosine(49) in tRNA precursor =S-adenosyl-L-homocysteine + 5-methylcytosine(49) in tRNA precursor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.203','tRNA (cytosine(34)-C(5))-methyltransferase','NULL','NULL','The human enzyme is specific for C(5)-methylation of cytosine(34) intRNA precursors.The intron in the human pre-tRNA(Leu)(CAA) is indispensable for theC(5)-methylation of cytosine in the first position of the anticodon.It is not able to form 5-methylcytosine at positions 48 and 49 ofhuman and yeast tRNA precursors.Formerly EC 2.1.1.29.','S-adenosyl-L-methionine + cytosine(34) in tRNA precursor = S-adenosyl-L-homocysteine + 5-methylcytosine(34) in tRNA precursor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.204','tRNA (cytosine(38)-C(5))-methyltransferase','NULL','NULL','The eukaryotic enzyme catalyzes methylation of cytosine(38) in theanti-codon loop of tRNA(Asp)(GTC), tRNA(Val)(AAC) and tRNA(Gly)(GCC).Methylation by Dnmt2 protects tRNAs against stress-induced cleavageby ribonuclease.Formerly EC 2.1.1.29.','S-adenosyl-L-methionine + cytosine(38) in tRNA = S-adenosyl-L-homocysteine + 5-methylcytosine(38) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.205','tRNA (cytidine(32)/guanosine(34)-2-O)-methyltransferase','NULL','NULL','The enzyme from Saccharomyces cerevisiae catalyzes the formation of2-O-methylnucleotides at positions 32 and 34 of the yeast tRNA(Phe),tRNA(Trp) and, possibly, tRNA(Leu).','S-adenosyl-L-methionine + cytidine(32)/guanosine(34) in tRNA =S-adenosyl-L-homocysteine + 2-O-methylcytidine(32)/2-O-methylguanosine(34) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.206','tRNA (cytidine(56)-2-O)-methyltransferase','NULL','NULL','The archaeal enzyme specifically catalyzes the S-adenosyl-L-methionine dependent 2-O-ribose methylation of cytidine at position56 in tRNA transcripts.','S-adenosyl-L-methionine + cytidine(56) in tRNA = S-adenosyl-L-homocysteine + 2-O-methylcytidine(56) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.207','tRNA (cytidine(34)-2-O)-methyltransferase','tRNA (cytidine(34)/5-carboxymethylaminomethyluridine(34)-2-O)-methyltransferase','NULL','The enzyme from Escherichia coli catalyzes the 2-O-methylation ofcytidine or 5-carboxymethylaminomethyluridine at the wobble positionat nucleotide 34 in tRNA(Leu)CmAA and tRNA(Leu)cmnm(5)UmAA.The enzyme is selective for the two tRNA(Leu) isoacceptors and onlymethylates these when they present the correct anticodon loopsequence and modification pattern.Specifically, YibK requires a pyrimidine nucleoside at position 34,it has a clear preference for an adenosine at position 35, and itfails to methylate without prior addition of the N(6)-(isopentenyl)-2-methylthioadenosine modification at position 37.','(1) S-adenosyl-L-methionine + cytidine(34) in tRNA = S-adenosyl-L-homocysteine + 2-O-methylcytidine(34) in tRNA.(2) S-adenosyl-L-methionine + 5-carboxymethylaminomethyluridine(34) intRNA(Leu) = S-adenosyl-L-homocysteine + 5-carboxymethylaminomethyl-2-O-methyluridine(34) in tRNA(Leu).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.208','23S rRNA (uridine(2479)-2-O)-methyltransferase','NULL','NULL','Streptomyces viridochromogenes produces the antibiotic avilamycin Awhich binds to the 50S ribosomal subunit to inhibit proteinsynthesis.To protect itself from the antibiotic, Streptomyces viridochromogenesutilizes two methyltransferases, 23S rRNA (uridine(2479)-2-O)-methyltransferase and EC 2.1.1.209 whose actions confer avilamycinresistance to the RNA.','S-adenosyl-L-methionine + uridine(2479) in 23S rRNA = S-adenosyl-L-homocysteine + 2-O-methyluridine(2479) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.209','23S rRNA (guanine(2535)-N(1))-methyltransferase','NULL','NULL','Streptomyces viridochromogenes produces the antibiotic avilamycin Awhich binds to the 50S ribosomal subunit to inhibit proteinsynthesis.To protect itself from the antibiotic, Streptomyces viridochromogenesutilizes two methyltransferases, 23S rRNA (guanine(2535)-N(1))-methyltransferase and EC 2.1.1.208 whose actions confer avilamycinresistance to the RNA.','S-adenosyl-L-methionine + guanine(2535) in 23S rRNA = S-adenosyl-L-homocysteine + N(1)-methylguanine(2535) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.210','Demethylspheroidene O-methyltransferase','1-HO-carotenoid methylase','NULL','In Rhodopseudomonas capsulata and Rubrivivax gelatinosus the enzymeis involved in biosynthesis of spheroidene.In Rubrivivax gelatinosus the enzyme also catalyzes the methylationof demethylspirilloxanthin to spirilloxanthin and the methylation of3,4-didehydrorhodopin to anhydrorhodovibrin.','S-adenosyl-L-methionine + demethylspheroidene = S-adenosyl-L-homocysteine+ spheroidene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.211','tRNA(Ser) (uridine(44)-2-O)-methyltransferase','NULL','NULL','The 2-O-methylation of uridine(44) contributes to stability oftRNA(Ser)(CGA).Formerly EC 2.1.1.n2.','S-adenosyl-L-methionine + uridine(44) in tRNA(Ser) = S-adenosyl-L-homocysteine + 2-O-methyluridine(44) in tRNA(Ser).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.212','2,7,4-trihydroxyisoflavanone 4-O-methyltransferase','NULL','NULL','Specifically methylates 2,4,7-trihydroxyisoflavanone on the4-position.No activity with isoflavones.The enzyme is involved in formononetin biosynthesis in legumes.The protein from pea (Pisum sativum) also methylates (+)-6a-hydroxymaackiain at the 3-position (cf. EC 2.1.1.270).','S-adenosyl-L-methionine + 2,4,7-trihydroxyisoflavanone = S-adenosyl-L-homocysteine + 2,7-dihydroxy-4-methoxyisoflavanone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.213','tRNA (guanine(10)-N(2))-dimethyltransferase','N(2),N(2)-dimethylguanosine tRNA methyltransferase','NULL','Formerly EC 2.1.1.32.','2 S-adenosyl-L-methionine + guanine(10) in tRNA = 2 S-adenosyl-L-homocysteine + N(2)-dimethylguanine(10) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.214','tRNA (guanine(10)-N(2))-methyltransferase','NULL','NULL','In contrast to the archaeal enzyme EC 2.1.1.213, tRNA (guanine(10)-N(2))-methyltransferase from yeast does not catalyze the methylationfrom N(2)-methylguanine(10) to N(2)-dimethylguanine(10) in tRNA.Formerly EC 2.1.1.32.','S-adenosyl-L-methionine + guanine(10) in tRNA = S-adenosyl-L-homocysteine+ N(2)-methylguanine(10) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.215','tRNA (guanine(26)-N(2)/guanine(27)-N(2))-dimethyltransferase','NULL','NULL','The enzyme from Aquifex aeolicus is similar to the TRM1methyltransferases of archaea and eukarya (see EC 2.1.1.216).However, it catalyzes the double methylation of guanines at bothpositions 26 and 27 of tRNA.Formerly EC 2.1.1.32.','4 S-adenosyl-L-methionine + guanine(26)/guanine(27) in tRNA =4 S-adenosyl-L-homocysteine + N(2)-dimethylguanine(26)/N(2)-dimethylguanine(27) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.216','tRNA (guanine(26)-N(2))-dimethyltransferase','tRNA (m(2)(2)G(26))dimethyltransferase','NULL','The enzyme dissociates from its tRNA substrate between the twoconsecutive methylation reactions.In contrast to EC 2.1.1.215, this enzyme does not catalyze themethylation of guanine(27) in tRNA.Formerly EC 2.1.1.32.','2 S-adenosyl-L-methionine + guanine(26) in tRNA = 2 S-adenosyl-L-homocysteine + N(2)-dimethylguanine(26) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.217','tRNA (adenine(22)-N(1))-methyltransferase','NULL','NULL','The enzyme specifically methylates adenine(22) in tRNA.Formerly EC 2.1.1.36.','S-adenosyl-L-methionine + adenine(22) in tRNA = S-adenosyl-L-homocysteine+ N(1)-methyladenine(22) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.218','tRNA (adenine(9)-N(1))-methyltransferase','tRNA m(1)A(9)-methyltransferase','NULL','The enzyme from Sulfolobus acidocaldarius specifically methylatesadenine(9) in tRNA.The bifunctional enzyme from Thermococcus kodakaraensis alsocatalyzes the methylation of guanine(9) in tRNA (cf. EC 2.1.1.221).Formerly EC 2.1.1.36.','S-adenosyl-L-methionine + adenine(9) in tRNA = S-adenosyl-L-homocysteine+ N(1)-methyladenine(9) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.219','tRNA (adenine(57)-N(1)/adenine(58)-N(1))-methyltransferase','NULL','NULL','The enzyme catalyzes the formation of N(1)-methyladenine at twoadjacent positions (57 and 58) in the T-loop of certain tRNAs (e.g.tRNA(Asp)).Methyladenosine at position 57 is an obligatory intermediate for thesynthesis of methylinosine, which is commonly found at position 57 ofarchaeal tRNAs.Formerly EC 2.1.1.36.','2 S-adenosyl-L-methionine + adenine(57)/adenine(58) in tRNA =2 S-adenosyl-L-homocysteine + N(1)-methyladenine(57)/N(1)-methyladenine(58) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.220','tRNA (adenine(58)-N(1))-methyltransferase','tRNA (m(1)A(58)) methyltransferase','NULL','The enzyme specifically methylates adenine(58) in tRNA.The methylation of A(58) is critical for maintaining the stability ofinitiator tRNA(Met) in yeast.Formerly EC 2.1.1.36.','S-adenosyl-L-methionine + adenine(58) in tRNA = S-adenosyl-L-homocysteine+ N(1)-methyladenine(58) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.221','tRNA (guanine(9)-N(1))-methyltransferase','tRNA m(1)G(9) MTase','NULL','The enzyme from Saccharomyces cerevisiae specifically methylatesguanine(9).The bifunctional enzyme from Thermococcus kodakaraensis alsocatalyzes the methylation of adenine(9) in tRNA (cf. EC 2.1.1.218).Formerly EC 2.1.1.31.','S-adenosyl-L-methionine + guanine(9) in tRNA = S-adenosyl-L-homocysteine+ N(1)-methylguanine(9) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.222','2-polyprenyl-6-hydroxyphenol methylase','2-octaprenyl-6-hydroxyphenol methylase','NULL','UbiG catalyzes both methylation steps in ubiquinone biosynthesis inEscherichia coli.The second methylation is classified as EC 2.1.1.64.In eukaryotes Coq3 catalyzes the two methylation steps in ubiquinonebiosynthesis.However, while the second methylation is common to both enzymes,the first methylation by Coq3 occurs at a different position withinthe pathway, and thus involves a different substrate and isclassified as EC 2.1.1.114.The substrate of the eukaryotic enzyme (3,4-dihydroxy-5-all-trans-polyprenylbenzoate) differs by an additional carboxylate moiety.','S-adenosyl-L-methionine + 3-(all-trans-polyprenyl)benzene-1,2-diol =S-adenosyl-L-homocysteine + 2-methoxy-6-(all-trans-polyprenyl)phenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.223','tRNA(1)(Val) (adenine(37)-N(6))-methyltransferase','NULL','NULL','The enzyme specifically methylates adenine(37) in tRNA(1)(Val)(anticodon cmo(5)UAC).','S-adenosyl-L-methionine + adenine(37) in tRNA(1)(Val) = S-adenosyl-L-homocysteine + N(6)-methyladenine(37) in tRNA(1)(Val).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.224','23S rRNA (adenine(2503)-C(8))-methyltransferase','NULL','Iron-sulfur.','This enzyme is a member of the AdoMet radical (radical SAM) family.S-adenosyl-L-methionine acts as both a radical generator and as thesource of the appended methyl group.Cfr is an plasmid-acquired methyltransferase that protects cells fromthe action of antibiotics.The enzyme methylates adenosine at position 2503 of 23S rRNA by aradical mechanism, transferring a CH(2) group from S-adenosyl-L-methionine while retaining the hydrogen at the C-8 position of theadenine.It will also methylate 2-methyladenine produced by the action ofEC 2.1.1.192.','2 S-adenosyl-L-methionine + adenine(2503) in 23S rRNA = S-adenosyl-L-homocysteine + L-methionine + 5-deoxyadenosine + 8-methyladenine(2503)in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.225','tRNA:m(4)X modification enzyme','NULL','NULL','The enzyme from Saccharomyces cerevisiae 2-O-methylates cytidine(4)in tRNA(Pro) and tRNA(Gly)(GCC), and adenosine(4) in tRNA(His).','(1) S-adenosyl-L-methionine + cytidine(4) in tRNA(Pro) = S-adenosyl-L-homocysteine + 2-O-methylcytidine(4) in tRNA(Pro).(2) S-adenosyl-L-methionine + cytidine(4) in tRNA(Gly)(GCC) = S-adenosyl-L-homocysteine + 2-O-methylcytidine(4) in tRNA(Gly)(GCC).(3) S-adenosyl-L-methionine + adenosine(4) in tRNA(His) = S-adenosyl-L-homocysteine + 2-O-methyladenosine(4) in tRNA(His).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.226','23S rRNA (cytidine(1920)-2-O)-methyltransferase','NULL','NULL','The bifunctional enzyme from Mycobacterium tuberculosis 2-O-methylates cytidine(1920) in helix 69 of 23S rRNA and cytidine(1409)in helix 44 of 16S rRNA (cf. EC 2.1.1.227).These methylations result in increased susceptibility to theantibiotics capreomycin and viomycin.','S-adenosyl-L-methionine + cytidine(1920) in 23S rRNA = S-adenosyl-L-homocysteine + 2-O-methylcytidine(1920) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.227','16S rRNA (cytidine(1409)-2-O)-methyltransferase','NULL','NULL','The bifunctional enzyme from Mycobacterium tuberculosis 2-O-methylates cytidine(1409) in helix 44 of 16S rRNA and cytidine(1920)in helix 69 of 23S rRNA (cf. EC 2.1.1.226).','S-adenosyl-L-methionine + cytidine(1409) in 16S rRNA = S-adenosyl-L-homocysteine + 2-O-methylcytidine(1409) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.228','tRNA (guanine(37)-N(1))-methyltransferase','Transfer RNA (m(1)G(37)) methyltransferase','NULL','This enzyme is important for the maintenance of the correct readingframe during translation.Unlike TrmD from Escherichia coli, which recognizes the G(36)pG(37)motif preferentially, the human enzyme (encoded by TRMT5) alsomethylates inosine at position 37.Formerly EC 2.1.1.31.','S-adenosyl-L-methionine + guanine(37) in tRNA = S-adenosyl-L-homocysteine+ N(1)-methylguanine(37) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.229','tRNA (carboxymethyluridine(34)-5-O)-methyltransferase','tRNA methyltransferase 9','NULL','The enzyme catalyzes the posttranslational modification of uridineresidues at the wobble position 34 of the anticodon loop of tRNA.','S-adenosyl-L-methionine + carboxymethyluridine(34) in tRNA = S-adenosyl-L-homocysteine + 5-(2-methoxy-2-oxoethyl)uridine(34) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.230','23S rRNA (adenosine(1067)-2-O)-methyltransferase','23S rRNA A(1067) 2-methyltransferase','NULL','The methylase that is responsible for autoimmunity in thethiostrepton producer Streptomyces azureus, renders ribosomescompletely resistant to thiostrepton.Formerly EC 2.1.1.66.','S-adenosyl-L-methionine + adenosine(1067) in 23S rRNA = S-adenosyl-L-homocysteine + 2-O-methyladenosine(1067) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.231','Flavonoid 4-O-methyltransferase','4-hydroxyisoflavone methyltransferase','NULL','The enzyme catalyzes the 4-methylation of naringenin.In vitro it catalyzes the 4-methylation of apigenin, quercetin,daidzein and genistein.','S-adenosyl-L-methionine + a 4-hydroxyflavanone = S-adenosyl-L-homocysteine + a 4-methoxyflavanone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.232','Naringenin 7-O-methyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of the sakuranetin,an inducible defense mechanism of Oryza sativa against pathogenattack.','S-adenosyl-L-methionine + (2S)-naringenin = S-adenosyl-L-homocysteine +(2S)-sakuranetin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.233','[Phosphatase 2A protein]-leucine-carboxy methyltransferase','Leucine carboxy methyltransferase-1','NULL','Methylates the C-terminal leucine of phosphatase 2A.A key regulator of protein phosphatase 2A.The methyl ester is hydrolyzed by EC 3.1.1.89.Occurs mainly in the cytoplasm, Golgi region and late endosomes.','S-adenosyl-L-methionine + [phosphatase 2A protein]-leucine = S-adenosyl-L-homocysteine + [phosphatase 2A protein]-leucine methyl ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.234','dTDP-3-amino-3,4,6-trideoxy-alpha-D-glucopyranoseN,N-dimethyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of desosamine,a 3-(dimethylamino)-3,4,6-trideoxyhexose found in certain macrolideantibiotics such as erthyromycin, azithromycin, and clarithromycin.','2 S-adenosyl-L-methionine + dTDP-3-amino-3,4,6-trideoxy-alpha-D-glucopyranose = 2 S-adenosyl-L-homocysteine + dTDP-3-dimethylamino-3,4,6-trideoxy-alpha-D-glucopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.235','dTDP-3-amino-3,6-dideoxy-alpha-D-glucopyranose N,N-dimethyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of mycaminose,an essential structural component of the macrolide antibiotictylosin, which is produced by the bacterium Streptomyces fradiae.','2 S-adenosyl-L-methionine + dTDP-3-amino-3,6-dideoxy-alpha-D-glucopyranose = 2 S-adenosyl-L-homocysteine + dTDP-3-dimethylamino-3,6-dideoxy-alpha-D-glucopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.236','dTDP-3-amino-3,6-dideoxy-alpha-D-galactopyranose N,N-dimethyltransferase','NULL','NULL','The enzyme is involved in the synthesis of dTDP-D-ravidosamine,the amino sugar moiety of the antibiotic ravidomycin V, which isproduced by the bacterium Streptomyces ravidus.','2 S-adenosyl-L-methionine + dTDP-3-amino-3,6-dideoxy-alpha-D-galactopyranose = 2 S-adenosyl-L-homocysteine + dTDP-3-dimethylamino-3,6-dideoxy-alpha-D-galactopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.237','Mycinamicin III 3-O-methyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of mycinamicin macrolideantibiotics.','S-adenosyl-L-methionine + mycinamicin III = S-adenosyl-L-homocysteine +mycinamicin IV.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.238','Mycinamicin VI 2-O-methyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of mycinamicin macrolideantibiotics.Requires Mg(2+) for optimal activity.','S-adenosyl-L-methionine + mycinamicin VI = S-adenosyl-L-homocysteine +mycinamicin III.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.239','L-olivosyl-oleandolide 3-O-methyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of the macrolideantibiotic oleandomycin in Streptomyces antibioticus.It can also act on other monoglycosylated macrolactones, includingL-rhamnosyl-erythronolide B and L-mycarosyl-erythronolide B.','S-adenosyl-L-methionine + L-olivosyl-oleandolide = S-adenosyl-L-homocysteine + L-oleandrosyl-oleandolide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.240','Trans-resveratrol di-O-methyltransferase','Pterostilbene synthase','NULL','The enzyme catalyzes the biosynthesis of pterostilbene fromresveratrol.','2 S-adenosyl-L-methionine + trans-resveratrol = 2 S-adenosyl-L-homocysteine + pterostilbene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.241','2,4,7-trihydroxy-1,4-benzoxazin-3-one-glucoside 7-O-methyltransferase','OMT BX7','NULL','The enzyme is involved in the biosynthesis of the protective andallelophatic benzoxazinoid DIMBOA ((2R)-4-hydroxy-7-methoxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin) in some plants, most commonly from thefamily of Poacea (grasses).','S-adenosyl-L-methionine + (2R)-4,7-dihydroxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside = S-adenosyl-L-homocysteine +(2R)-4-hydroxy-7-methoxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.242','16S rRNA (guanine(1516)-N(2))-methyltransferase','M(2)G(1516) methyltransferase','NULL','The enzyme specifically methylates guanine(1516) at N(2) in 16S rRNA.','S-adenosyl-L-methionine + guanine(1516) in 16S rRNA = S-adenosyl-L-homocysteine + N(2)-methylguanine(1516) in 16S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.243','2-ketoarginine methyltransferase','NULL','NULL','The enzyme is involved in production of the rare amino acid3-methylarginine, which is used by the epiphytic bacteriumPseudomonas syringae pv. syringae as an antibiotic against therelated pathogenic species Pseudomonas syringae pv. glycinea.','S-adenosyl-L-methionine + 5-guanidino-2-oxopentanoate = S-adenosyl-L-homocysteine + 5-guanidino-3-methyl-2-oxopentanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.244','Protein N-terminal methyltransferase','NULL','NULL','This enzyme methylates the N-terminus of target proteins containingthe N-terminal motif (Ala/Pro/Ser)-Pro-Lys after the initiatorL-methionine is cleaved.When the terminal amino acid is L-proline, the enzyme catalyzes twosuccessive methylations of its alpha-amino group.When the first amino acid is either L-alanine or L-serine, the enzymecatalyzes three successive methylations.The Pro-Lys in positions 2-3 cannot be exchanged for other aminoacids.Formerly EC 2.1.1.n5.','(1) 3 S-adenosyl-L-methionine + N-terminal-(A,S)PK-[protein] =3 S-adenosyl-L-homocysteine + N-terminal-N,N,N-trimethyl-N-(A,S)PK-[protein].(2) 2 S-adenosyl-L-methionine + N-terminal-PPK-[protein] = 2 S-adenosyl-L-homocysteine + N-terminal-N,N-dimethyl-N-PPK-[protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.245','5-methyltetrahydrosarcinapterin:corrinoid/iron-sulfur proteinCo-methyltransferase','NULL','NULL','Catalyzes the transfer of a methyl group from the cobamide cofactorof a corrinoid/Fe-S protein to the N5 group oftetrahydrosarcinapterin.Forms, together with EC 1.2.7.4 and EC 2.3.1.169 the acetyl-CoAdecarbonylase/synthase complex that catalyzes the demethylation ofacetyl-CoA in a reaction that also forms CO(2).This reaction is a key step in methanogenesis from acetate.','A [methyl-Co(III) corrinoid Fe-S protein] + tetrahydrosarcinapterin =a [Co(I) corrinoid Fe-S protein] + 5-methyltetrahydrosarcinapterin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.246','[Methyl-Co(III) methanol-specific corrinoid protein]:coenzyme Mmethyltransferase','NULL','NULL','The enzyme, which is involved in methanogenesis from methanol,catalyzes the transfer of a methyl group from a corrinoid protein(see EC 2.1.1.90), where it is bound to the cobalt cofactor,to coenzyme M, forming the substrate for EC 2.8.4.1, the enzyme thatcatalyzes the final step in methanogenesis.Free methylcob(I)alamin can substitute for the corrinoid protein invitro.','A [methyl-Co(III) methanol-specific corrinoid protein] + coenzyme M =methyl-CoM + a [Co(I) methanol-specific corrinoid protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.247','[Methyl-Co(III) methylamine-specific corrinoid protein]:coenzyme Mmethyltransferase','MT2-A','Zinc.','The enzyme, which is involved in methanogenesis from mono-, di-,and trimethylamine, catalyzes the transfer of a methyl group bound tothe cobalt cofactor of several corrinoid proteins (mono-, di-,and trimethylamine-specific corrinoid proteins, cf. EC 2.1.1.248,EC 2.1.1.249 and EC 2.1.1.250) to coenzyme M, forming the substratefor EC 2.8.4.1, the enzyme that catalyzes the final step inmethanogenesis.','A [methyl-Co(III) methylamine-specific corrinoid protein] + coenzyme M =methyl-CoM + a [Co(I) methylamine-specific corrinoid protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.248','[Methylamine--corrinoid protein] Co-methyltransferase','Monomethylamine methyltransferase','NULL','The enzyme, which catalyzes the transfer of a methyl group frommethylamine to a methylamine-specific corrinoid protein (MtmC),is involved in methanogenesis from methylamine.The enzyme contains the unusual amino acid pyrrolysine.Methylation of the corrinoid protein requires the central cobalt tobe in the Co(I) state.During methylation the cobalt is oxidized to the Co(III) state.The methylated corrinoid protein is substrate for EC 2.1.1.247.','Methylamine + a [Co(I) methylamine-specific corrinoid protein] =a [methyl-Co(III) methylamine-specific corrinoid protein] + ammonia.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.249','[Dimethylamine--corrinoid protein] Co-methyltransferase','Dimethylamine methyltransferase','NULL','The enzyme, which catalyzes the transfer of a methyl group fromdimethylamine to a dimethylamine-specific corrinoid protein (MtbC),is involved in methanogenesis from dimethylamine.The enzyme contains the unusual amino acid pyrrolysine.Methylation of the corrinoid protein requires the central cobalt tobe in the Co(I) state.During methylation the cobalt is oxidized to the Co(III) state.The methylated corrinoid protein is substrate for EC 2.1.1.247.','Dimethylamine + a [Co(I) dimethylamine-specific corrinoid protein] =a [methyl-Co(III) dimethylamine-specific corrinoid protein] +methylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.250','[Trimethylamine--corrinoid protein] Co-methyltransferase','Trimethylamine methyltransferase','NULL','The enzyme, which catalyzes the transfer of a methyl group fromtrimethylamine to a trimethylamine-specific corrinoid protein (MttC),is involved in methanogenesis from trimethylamine.The enzyme contains the unusual amino acid pyrrolysine.Methylation of the corrinoid protein requires the central cobalt tobe in the Co(I) state.During methylation the cobalt is oxidized to the Co(III) state.The methylated corrinoid protein is substrate for EC 2.1.1.247.','Trimethylamine + a [Co(I) trimethylamine-specific corrinoid protein] =a [methyl-Co(III) trimethylamine-specific corrinoid protein] +dimethylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.251','Methylated-thiol--coenzyme M methyltransferase','NULL','NULL','The enzyme, which is involved in methanogenesis from methylatedthiols, such as methane thiol, dimethyl sulfide, and 3-S-methylmercaptopropionate, catalyzes two successive steps - thetransfer of a methyl group from the substrate to the cobalt cofactorof a methylated-thiol-specific corrinoid protein (MtsB), and thesubsequent transfer of the methyl group from the corrinoid protein tocoenzyme M.With most other methanogenesis substrates this process is carried outby two different enzymes (for example, EC 2.1.1.90 and EC 2.1.1.246).The cobalt is oxidized during methylation from the Co(I) state to theCo(III) state, and is reduced back to the Co(I) form duringdemethylation.','Methanethiol + coenzyme M = methyl-CoM + hydrogen sulfide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.252','Tetramethylammonium--corrinoid protein Co-methyltransferase','Tetramethylammonium methyltransferase','NULL','The enzyme, which catalyzes the transfer of a methyl group fromtetramethylammonium to a tetramethylammonium-specific corrinoidprotein (MtqC), is involved in methanogenesis fromtetramethylammonium.Methylation of the corrinoid protein requires the central cobalt tobe in the Co(I) state.During methylation the cobalt is oxidized to the Co(III) state.The methylated corrinoid protein is substrate for EC 2.1.1.253.','Tetramethylammonium + a [Co(I) tetramethylammonium-specific corrinoidprotein] = a [methyl-Co(III) tetramethylammonium-specific corrinoidprotein] + trimethylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.253','[Methyl-Co(III) tetramethylammonium-specific corrinoid protein]:coenzymeM methyltransferase','NULL','NULL','The enzyme, which is involved in methanogenesis fromtetramethylammonium, catalyzes the transfer of a methyl group from acorrinoid protein (see EC 2.1.1.252), where it is bound to the cobaltcofactor, to coenzyme M, forming the substrate for EC 2.8.4.1,the enzyme that catalyzes the final step in methanogenesis.','A [methyl-Co(III) tetramethylammonium-specific corrinoid protein] +coenzyme M = methyl-CoM + a [Co(I) tetramethylammonium-specific corrinoidprotein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.254','Erythromycin 3-O-methyltransferase','NULL','NULL','The enzyme methylates the 3 position of the mycarosyl moiety oferythromycin C, forming the most active form of the antibiotic,erythromycin A.It can also methylate the precursor erythromycin D, formingerythromycin B, which is then converted to erythromycin A byEC 1.14.13.154.','(1) S-adenosyl-L-methionine + erythromycin C = S-adenosyl-L-homocysteine+ erythromycin A.(2) S-adenosyl-L-methionine + erythromycin D = S-adenosyl-L-homocysteine+ erythromycin B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.255','Geranyl diphosphate 2-C-methyltransferase','2-methyl-GPP synthase','Magnesium.','This enzyme, along with EC 4.2.3.118, produces 2-methylisoborneol,an odiferous compound produced by soil microorganisms with a strongearthy/musty odor.Formerly EC 2.1.1.n6.','S-adenosyl-L-methionine + geranyl diphosphate = S-adenosyl-L-homocysteine+ (E)-2-methylgeranyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.256','tRNA (guanine(6)-N(2))-methyltransferase','M(2)G6 methyltransferase','NULL','The enzyme specifically methylates guanine(6) at N(2) in tRNA.','S-adenosyl-L-methionine + guanine(6) in tRNA = S-adenosyl-L-homocysteine+ N(2)-methylguanine(6) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.257','tRNA (pseudouridine(54)-N(1))-methyltransferase','M(1)-Psi methyltransferase','NULL','While this archaeal enzyme is specific for the 54 position and doesnot methylate pseudouridine at position 55, the presence ofpseudouridine at position 55 is necessary for the efficientmethylation of pseudouridine at position 54.','S-adenosyl-L-methionine + pseudouridine(54) in tRNA = S-adenosyl-L-homocysteine + N(1)-methylpseudouridine(54) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.258','5-methyltetrahydrofolate:corrinoid/iron-sulfur proteinCo-methyltransferase','NULL','NULL','Catalyzes the transfer of a methyl group from the N(5) group ofmethyltetrahydrofolate to the 5-methoxybenzimidazolylcobamidecofactor of a corrinoid/Fe-S protein.Involved, together with EC 1.2.7.4 and EC 2.3.1.169, in the reductiveacetyl coenzyme A (Wood-Ljungdahl) pathway of autotrophic carbonfixation in various bacteria and archaea.','A [methyl-Co(III) corrinoid Fe-S protein] + tetrahydrofolate = a [Co(I)corrinoid Fe-S protein] + 5-methyltetrahydrofolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.259','[Fructose-bisphosphate aldolase]-lysine N-methyltransferase','Ribulose-bisphosphate-carboxylase/oxygenase N-methyltransferase','NULL','The enzyme methylates a conserved lysine in the C-terminal part ofhigher plant fructose-bisphosphate aldolase (EC 4.1.2.13).The enzyme from pea (Pisum sativum) also methylates Lys-14 in thelarge subunits of hexadecameric higher plant ribulose-bisphosphate-carboxylase (EC 4.1.1.39), but that from Arabidopsis thaliana doesnot.','3 S-adenosyl-L-methionine + [fructose-bisphosphate aldolase]-L-lysine =3 S-adenosyl-L-homocysteine + [fructose-bisphosphate aldolase]-N(6),N(6),N(6)-trimethyl-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.260','rRNA small subunit pseudouridine methyltransferase Nep1','NULL','NULL','This enzyme, which occurs in both prokaryotes and eukaryotes,recognizes specific pseudouridine residues (Psi) in small subunits ofribosomal RNA based on the local RNA structure.It recognizes Psi(914) in 16S rRNA from the archaeonMethanocaldococcus jannaschii, Psi(1191) in yeast 18S rRNA,and Psi(1248) in human 18S rRNA.','S-adenosyl-L-methionine + pseudouridine(1191) in yeast 18S rRNA =S-adenosyl-L-homocysteine + N(1)-methylpseudouridine(1191) in yeast 18SrRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.261','4-dimethylallyltryptophan N-methyltransferase','NULL','NULL','The enzyme catalyzes a step in the pathway leading to biosynthesis ofergot alkaloids in certain fungi.','S-adenosyl-L-methionine + 4-dimethylallyl-L-tryptophan = S-adenosyl-L-homocysteine + 4-dimethylallyl-L-abrine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.262','Squalene methyltransferase','NULL','NULL','Two isoforms differing in their specificity were isolated from thegreen alga Botryococcus braunii BOT22.TMT-1 gave more of the dimethylated form whereas TMT2 gave more ofthe monomethylated form.','2 S-adenosyl-L-methionine + squalene = 2 S-adenosyl-L-homocysteine +3,22-dimethyl-1,2,23,24-tetradehydro-2,3,22,23-tetrahydrosqualene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.263','Botryococcene C-methyltransferase','NULL','NULL','Isolated from the green alga Botryococcus braunii BOT22.Shows a very weak activity with squalene.','2 S-adenosyl-L-methionine + C(30) botryococcene = 2 S-adenosyl-L-homocysteine + 3,20-dimethyl-1,2,21,22-tetradehydro-2,3,20,21-tetrahydrobotryococcene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.264','23S rRNA (guanine(2069)-N(7))-methyltransferase','23S rRNA m(7)G(2069) methyltransferase','NULL','The enzyme specifically methylates guanine(2069) at position N7 in23S rRNA.In gamma-proteobacteria the enzyme also catalyzes EC 2.1.1.173, whilein beta-proteobacteria the activities are carried out by separateproteins.The enzyme from the gamma-proteobacterium Escherichia coli has RNAunwinding activity as well.','S-adenosyl-L-methionine + guanine(2069) in 23S rRNA = S-adenosyl-L-homocysteine + N(7)-methylguanine(2069) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.265','Tellurite methyltransferase','NULL','NULL','The enzyme is involved in the detoxification of tellurite.It can also methylate selenite and selenium dioxide.','S-adenosyl-L-methionine + tellurite = S-adenosyl-L-homocysteine +methanetelluronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.266','23S rRNA (adenine(2030)-N(6))-methyltransferase','M(6)A(2030) methyltransferase','NULL','The recombinant RlmJ protein is most active in methylatingdeproteinized 23S ribosomal subunit, and does not methylate thecompletely assembled 50S subunits.','S-adenosyl-L-methionine + adenine(2030) in 23S rRNA = S-adenosyl-L-homocysteine + N(6)-methyladenine(2030) in 23S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.267','Flavonoid 3,5-methyltransferase','NULL','NULL','Isolated from Vitis vinifera (grape).Most active with delphinidin 3-glucoside but also acts on cyanidin3-glucoside, cyanidin, myricetin, quercetin and quercetin3-glucoside.The enzyme from Catharanthus roseus was most active with myricetin.Formerly EC 2.1.1.149.','(1) S-adenosyl-L-methionine + a 3-hydroxyflavonoid = S-adenosyl-L-homocysteine + a 3-methoxyflavonoid.(2) S-adenosyl-L-methionine + a 5-hydroxy-3-methoxyflavonoid =S-adenosyl-L-homocysteine + a 3,5-dimethoxyflavonoid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.268','tRNA(Thr) (cytosine(32)-N(3))-methyltransferase','NULL','NULL','The enzyme from Saccharomyces cerevisiae specifically methylatescytosine(32) in tRNA(Thr) and in tRNA(Ser).','(1) S-adenosyl-L-methionine + cytosine(32) in tRNA(Thr) = S-adenosyl-L-homocysteine + N(3)-methylcytosine(32) in tRNA(Thr).(2) S-adenosyl-L-methionine + cytosine(32) in tRNA(Ser) = S-adenosyl-L-homocysteine + N(3)-methylcytosine(32) in tRNA(Ser).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.269','Dimethylsulfoniopropionate demethylase','Dimethylsulfoniopropionate-dependent demethylase A','NULL','The enzyme from the marine bacteria Pelagibacter ubique and Ruegeriapomeroyi are specific toward S,S-dimethyl-beta-propiothetin.They do not demethylate glycine-betaine.','S,S-dimethyl-beta-propiothetin + tetrahydrofolate =3-(methylthio)propanoate + 5-methyltetrahydrofolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.270','(+)-6a-hydroxymaackiain 3-O-methyltransferase','HM3OMT','NULL','The protein from the plant Pisum sativum (garden pea) methylates (+)-6a-hydroxymaackiain at the 3-position.It also methylates 2,7,4-trihydroxyisoflavanone on the 4-position(cf. EC 2.1.1.212).','S-adenosyl-L-methionine + (+)-6a-hydroxymaackiain = S-adenosyl-L-homocysteine + (+)-pisatin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.271','Cobalt-precorrin-4 methyltransferase','NULL','NULL','Part of the anaerobic route to adenosylcobalamin.','S-adenosyl-L-methionine + cobalt-precorrin-4 = S-adenosyl-L-homocysteine+ cobalt-precorrin-5A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.272','Cobalt-factor III methyltransferase','NULL','NULL','Isolated from Bacillus megaterium.The enzyme catalyzes both methylation at C-17 and ring contraction.Contains a [4Fe-4S] cluster.It can also convert cobalt-precorrin-3 to cobalt-precorrin-4.The reductant may be thioredoxin.','S-adenosyl-L-methionine + cobalt-factor III + reduced acceptor =S-adenosyl-L-homocysteine + cobalt-precorrin-4 + acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.273','Benzoate O-methyltransferase','NULL','NULL','While the enzyme from the plant Zea mays is specific for benzoate,the enzymes from Arabidopsis species and Clarkia breweri alsocatalyze the reaction of EC 2.1.1.274.In snapdragon (Antirrhinum majus) two isoforms are found,one specific for benzoate and one that is also active towardsalicylate.The volatile product is an important scent compound in some floweringspecies.','S-adenosyl-L-methionine + benzoate = S-adenosyl-L-homocysteine + methylbenzoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.274','Salicylate carboxymethyltransferase','Salicylate O-methyltransferase','NULL','The enzyme, which is found in flowering plants, also has the activityof EC 2.1.1.273.Formerly EC 2.1.1.n9.','S-adenosyl-L-methionine + salicylate = S-adenosyl-L-homocysteine + methylsalicylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.275','Gibberellin A(9) O-methyltransferase','NULL','NULL','The enzyme also methylates gibberellins A(20) (95%), A(3) (80%), A(4)(69%) and A(34) (46%) with significant activity.','S-adenosyl-L-methionine + gibberellin A(9) = S-adenosyl-L-homocysteine +methyl gibberellin A(9).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.276','Gibberellin A(4) carboxyl methyltransferase','Gibberellin A(4) O-methyltransferase','NULL','The enzyme also methylates gibberellins A(34) (80%), A(9) (60%),and A(3) (45%) with significant activity.','S-adenosyl-L-methionine + gibberellin A(4) = S-adenosyl-L-homocysteine +methyl gibberellin A(4).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.277','Anthranilate O-methyltransferase','NULL','NULL','In the plant maize (Zea mays), the isoforms AAMT1 and AAMT2 arespecific for anthranilate while AAMT3 also has the activity ofEC 2.1.1.273.','S-adenosyl-L-methionine + anthranilate = S-adenosyl-L-homocysteine +O-methyl anthranilate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.278','Indole-3-acetate O-methyltransferase','IAA carboxylmethyltransferase','Magnesium.','The enzyme is found in plants and is important for regulation of theplant hormone (indol-3-yl)acetate.The product, methyl (indol-3-yl)acetate is inactive as hormone.Formerly EC 2.1.1.n10.','S-adenosyl-L-methionine + (indol-3-yl)acetate = S-adenosyl-L-homocysteine+ methyl (indol-3-yl)acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.279','Trans-anol O-methyltransferase','T-anol O-methyltransferase','NULL','The enzyme from anise (Pimpinella anisum) is highly specific forsubstrates in which the double bond in the propenyl side chain islocated between C(7) and C(8), and, in contrast to EC 2.1.1.146, doesnot have activity with eugenol or chavicol.','(1) S-adenosyl-L-methionine + trans-anol = S-adenosyl-L-homocysteine +trans-anethole.(2) S-adenosyl-L-methionine + isoeugenol = S-adenosyl-L-homocysteine +isomethyleugenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.280','Selenocysteine Se-methyltransferase','NULL','NULL','The enzyme uses S-adenosyl-L-methionine as methyl donor less activelythan S-methyl-L-methionine.The enzyme from broccoli (Brassica oleracea var. italica) also hasthe activity of EC 2.1.1.10.Formerly EC 2.1.1.n3.','S-methyl-L-methionine + L-selenocysteine = L-methionine + Se-methyl-L-selenocysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.281','Phenylpyruvate C(3)-methyltransferase','Phenylpyruvate C-beta-methyltransferase','NULL','The enzyme from the bacterium Streptomyces hygroscopicus NRRL3085 isinvolved in synthesis of the nonproteinogenic amino acid (2S,3S)-beta-methyl-phenylalanine, a building block of the antibioticmannopeptimycin.','S-adenosyl-L-methionine + 3-phenylpyruvate = S-adenosyl-L-homocysteine +(3S)-2-oxo-3-phenylbutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.282','tRNA(Phe) 7-((3-amino-3-carboxypropyl)-4-demethylwyosine(37)-N(4))-methyltransferase','tRNA-yW synthesizing enzyme-3','NULL','The enzyme is involved in the biosynthesis of hypermodified tricyclicbases found at position 37 of certain tRNAs.These modifications are important for translational reading-framemaintenance.The enzyme is found in all eukaryotes and in some archaea, but not inbacteria.The eukaryotic enzyme is involved in the biosynthesis of wybutosine.','S-adenosyl-L-methionine + 7-((3S)-(3-amino-3-carboxypropyl))-4-demethylwyosine(37) in tRNA(Phe) = S-adenosyl-L-homocysteine + 7-((3S)-(3-amino-3-carboxypropyl))wyosine(37) in tRNA(Phe).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.283','Emodin O-methyltransferase','NULL','NULL','The enzyme is involved in biosynthesis of the seco-anthraquinone (+)-geodin.','S-adenosyl-L-methionine + emodin = S-adenosyl-L-homocysteine + questin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.284','8-demethylnovobiocic acid C(8)-methyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of the aminocoumarinantibiotic novobiocin.','S-adenosyl-L-methionine + 8-demethylnovobiocic acid = S-adenosyl-L-homocysteine + novobiocic acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.285','Demethyldecarbamoylnovobiocin O-methyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of the aminocoumarinantibiotic novobiocin.','S-adenosyl-L-methionine + demethyldecarbamoylnovobiocin = S-adenosyl-L-homocysteine + decarbamoylnovobiocin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.286','25S rRNA (adenine(2142)-N(1))-methyltransferase','25S rRNA m(1)A(2142) methyltransferase','NULL','In the yeast Saccharomyces cerevisiae this methylation is importantfor resistance toward hydrogen peroxide and the antibioticanisomycin.','S-adenosyl-L-methionine + adenine(2142) in 25S rRNA = S-adenosyl-L-homocysteine + N(1)-methyladenine(2142) in 25S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.287','25S rRNA (adenine(645)-N(1))-methyltransferase','25S rRNA m(1)A(645) methyltransferase','NULL','The enzyme is found in eukaryotes.The adenine position refers to tRNA the yeast Saccharomycescerevisiae, in which the enzyme is important for ribosome biogenesis.','S-adenosyl-L-methionine + adenine(645) in 25S rRNA = S-adenosyl-L-homocysteine + N(1)-methyladenine(645) in 25S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.288','Aklanonic acid methyltransferase','NULL','NULL','The enzyme from the Gram-positive bacterium Streptomyces sp. C5 isinvolved in the biosynthesis of the anthracycline daunorubicin.','S-adenosyl-L-methionine + aklanonate = S-adenosyl-L-homocysteine + methylaklanonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.289','Cobalt-precorrin-7 (C(5))-methyltransferase','NULL','NULL','This enzyme catalyzes the methylation at C-5 of cobalt-precorrin-7,a step in the anaerobic (early cobalt insertion) adenosylcobalaminbiosynthesis pathway.','Cobalt-precorrin-7 + S-adenosyl-L-methionine = cobalt-precorrin-8 +S-adenosyl-L-homocysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.290','tRNA(Phe) (7-(3-amino-3-carboxypropyl)wyosine(37)-O)-methyltransferase','tRNA-yW synthesizing enzyme-4','NULL','The enzyme is found only in eukaryotes, where it is involved in thebiosynthesis of wybutosine, a hypermodified tricyclic base found atposition 37 of certain tRNAs.The modification is important for translational reading-framemaintenance.In some species that produce hydroxywybutosine the enzyme uses7-(2-hydroxy-3-amino-3-carboxypropyl)wyosine(37) in tRNA(Phe) assubstrate.The enzyme also has the activity of EC 2.3.1.231.','S-adenosyl-L-methionine + 7-((3S)-3-amino-3-carboxypropyl)wyosine(37) intRNA(Phe) = S-adenosyl-L-homocysteine + 7-((3S)-3-amino-3-(methoxycarbonyl)propyl)wyosine(37) in tRNA(Phe).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.291','(R,S)-reticuline 7-O-methyltransferase','NULL','NULL','The enzyme from the plant Papaver somniferum (opium poppy) methylates(S)- and (R)-reticuline with equal efficiency and is involved in thebiosynthesis of tetrahydrobenzylisoquinoline alkaloids.','(1) S-adenosyl-L-methionine + (S)-reticuline = S-adenosyl-L-homocysteine+ (S)-laudanine.(2) S-adenosyl-L-methionine + (R)-reticuline = S-adenosyl-L-homocysteine+ (R)-laudanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.292','Carminomycin 4-O-methyltransferase','NULL','NULL','The enzymes from the Gram-positive bacteria Streptomyces sp. C5 andStreptomyces peucetius are involved in the biosynthesis of theanthracycline daunorubicin.In vitro the enzyme from Streptomyces sp. C5 also catalyzes the 4-O-methylation of 13-dihydrocarminomycin, rhodomycin D and 10-carboxy-13-deoxycarminomycin.','S-adenosyl-L-methionine + carminomycin = S-adenosyl-L-homocysteine +daunorubicin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.293','6-hydroxytryprostatin B O-methyltransferase','NULL','NULL','Involved in the biosynthetic pathways of several indole alkaloidssuch as tryprostatins, fumitremorgins and verruculogen.','S-adenosyl-L-methionine + 6-hydroxytryprostatin B = S-adenosyl-L-homocysteine + tryprostatin A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.294','3-O-phospho-polymannosyl GlcNAc-diphospho-ditrans,octacis-undecaprenol3-phospho-methyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of the polymannoseO-polysaccharide in the outer leaflet of the membrane of Escherichiacoli serotype O9a.O-polysaccharide structures vary extensively because of differencesin the number and type of sugars in the repeat unit.The dual kinase/methylase WbdD also catalyzes the precedingphosphorylation of alpha-D-Man-(1-','S-adenosyl-L-methionine + 3-O-phospho-alpha-D-Man-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.295','2-methyl-6-phytyl-1,4-hydroquinone methyltransferase','2-methyl-6-solanyl-1,4-hydroquinone methyltransferase','NULL','Involved in the biosynthesis of plastoquinol, as well as vitaminE (tocopherols and tocotrienols).','(1) S-adenosyl-L-methionine + 2-methyl-6-phytylbenzene-1,4-diol =S-adenosyl-L-homocysteine + 2,3-dimethyl-6-phytylbenzene-1,4-diol.(2) S-adenosyl-L-methionine + 2-methyl-6-all-trans-nonaprenylbenzene-1,4-diol = S-adenosyl-L-homocysteine + plastoquinol.(3) S-adenosyl-L-methionine + 6-geranylgeranyl-2-methylbenzene-1,4-diol =S-adenosyl-L-homocysteine + 6-geranylgeranyl-2,3-dimethylbenzene-1,4-diol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.296','Methyltransferase cap2','Cap2-MTase','NULL','The enzyme, found in higher eukaryotes including insects andvertebrates, and their viruses, methylates the ribose of thenucleoside at the second transcribed position of mRNAs and snRNAs.This methylation is known as cap2.The human enzyme can also methylate mRNA molecules where the upstreampurine ribonucleotide is not methylated (see EC 2.1.1.57), but withlower efficiency.','S-adenosyl-L-methionine + a 5-(N(7)-methyl 5-triphosphoguanosine)-(2-O-methyl-purine-ribonucleotide)-(ribonucleotide)-(mRNA) = S-adenosyl-L-homocysteine + a 5-(N(7)-methyl 5-triphosphoguanosine)-(2-O-methyl-purine-ribonucleotide)-(2-O-methyl-ribonucleotide)-(mRNA).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.297','Peptide chain release factor N(5)-glutamine methyltransferase','N(5)-glutamine MTase','NULL','Modifies the glutamine residue in the universally conservedglycylglycylglutamine (GGQ) motif of peptide chain release factor,resulting in almost complete loss of release activity.Formerly EC 2.1.1.n16.','S-adenosyl-L-methionine + [peptide chain release factor 1 or 2]-L-glutamine = S-adenosyl-L-homocysteine + [peptide chain release factor 1or 2]-N(5)-methyl-L-glutamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.298','Ribosomal protein L3 N(5)-glutamine methyltransferase','NULL','NULL','Modifies the glutamine residue in the glycylglycylglutamine (GGQ)motif of ribosomal protein L3 (Gln(150) in the protein from thebacterium Escherichia coli).The enzyme does not act on peptide chain release factor 1 or 2.Formerly EC 2.1.1.n15.','S-adenosyl-L-methionine + [ribosomal protein L3]-L-glutamine =S-adenosyl-L-homocysteine + [ribosomal protein L3]-N(5)-methyl-L-glutamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.299','Protein N-terminal monomethyltransferase','N-terminal monomethylase','NULL','This enzyme methylates the N-terminus of target proteins containingthe N-terminal motif (Ala/Pro/Ser)-Pro-Lys after the initiatorL-methionine is cleaved.In contrast to EC 2.1.1.244, the protein only adds one methyl groupto the N-terminal.','S-adenosyl-L-methionine + N-terminal-(A,P,S)PK-[protein] = S-adenosyl-L-homocysteine + N-terminal-N-methyl-N-(A,P,S)PK-[protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.300','Pavine N-methyltransferase','PavNMT','NULL','The enzyme, isolated from the plant Thalictrum flavum, alsomethylates (R,S)-stylopine and (S)-scoulerine (11%) with loweractivity (14% and 11%, respectively).Formerly EC 2.1.1.n14.','S-adenosyl-L-methionine + (+-)-pavine = S-adenosyl-L-homocysteine +N-methylpavine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n1','Resorcinol O-methyltransferase','O-methyltransferase 12','NULL','The enzyme from Dictyostelium discoideum is involed in modifying theresorcinolic back-bone of the differentiation regulating factor4-methyl-5-pentylbenzene-1,3-diol.It also shows activity toward 5-pentylbenzene-1,3-diol.','S-adenosyl-L-methionine + resorcinol = S-adenosyl-L-homocysteine +3-methoxy phenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n2','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n3','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n4','Thiocyanate methyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + thiocyanate = S-adenosyl-L-homocysteine +methyl thiocyanate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n5','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n6','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n7','5-pentadecatrienyl resorcinol O-methyltransferase','NULL','NULL','In Sorghum, this enzyme is part of the biosynthetic pathway of thephytotoxin sorgoleone.','S-adenosyl-L-methionine + (8Z,11Z)-5-(pentadeca-8,11,14-trien-1-yl)resorcinol = S-adenosyl-L-homocysteine + (8Z,11Z)-5-(pentadeca-8,11,14-trien-1-yl)resorcinol-3-methyl ether.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n8','Small RNA 2-O-methyltransferase','NULL','NULL','Adds a methyl group to the ribose of the last nucleotide of smallRNAs (sRNAs).This protects the 3-end of sRNAs from uridylation activity andsubsequent degradation.Can methylates 3-end of microRNAs (miRNAs), small interfering RNAs(siRNas) and trans-acting small interfering RNAs (ta-siRNAs).','S-adenosyl-L-methionine + small RNA = S-adenosyl-L-homocysteine + smallRNA containing a 3-terminal 2-O-methylnucleotide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n9','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n10','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n11','Methylphosphotriester-DNA--[protein]-cysteine S-methyltransferase','NULL','Zinc.','This protein is involved in the repair of Sp diastereomers of DNAmethylphosphotriester lesions.This enzyme catalyzes only one turnover and therefore is not strictlycatalytic.The enzyme from the bacterium Escherichia coli also has the activityof EC 2.1.1.63 while the enzyme from Bacillus subtilis does not.','DNA (containing Sp-methylphosphotriester) + protein L-cysteine =DNA (without Sp-methylphosphotriester) + protein S-methyl-L-cysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n12','2-phytyl-1,4-naphtoquinone methyltransferase','NULL','NULL','The enzyme is onvolved in the biosynthesis pathway of phyyloquinone(vitamin K1).It is found in the plant Arabidopsis thaliana and the algaSynechocystis PCC6803.','S-adenosyl-L-methionine + demethylphylloquinone = S-adenosyl-L-homocysteine + phylloquinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n13','Cypemycin N-terminal methyltransferase','NULL','NULL','The enzyme, isolated from Streptomyces sp. OH-4156, can methylate avariety of oligopeptides, cyclic peptides such as nisin andhaloduracin, and the epsilon-amino group of lysine.Cypemycin is a peptide antibiotic, a member of the linaridins,a class of posttranslationally modified ribosomally synthesizedpeptides.','(1) 2 S-adenosyl-L-methionine + L-alanyl group = N,N-dimethyl-L-alaninegroup + 2 S-adenosyl-L-homocysteine + 2 H(+).(2) 2 S-adenosyl-L-methionine + [cypemycin]-Ala-N-term = 2 S-adenosyl-L-homocysteine + [cypemycin]-N,N-dimethylalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n14','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n15','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.1.n16','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.1','Glycine hydroxymethyltransferase','Serine aldolase','Pyridoxal 5-phosphate.','Also catalyzes the reaction of glycine with acetaldehyde to formL-threonine, and with 4-trimethylammoniobutanal to form 3-hydroxy-N(6),N(6),N(6)-trimethyl-L-lysine.','5,10-methylenetetrahydrofolate + glycine + H(2)O = tetrahydrofolate +L-serine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.2','Phosphoribosylglycinamide formyltransferase','2-amino-N-ribosylacetamide 5-phosphate transformylase','NULL','NULL','10-formyltetrahydrofolate + N(1)-(5-phospho-D-ribosyl)glycinamide =tetrahydrofolate + N(2)-formyl-N(1)-(5-phospho-D-ribosyl)glycinamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.3','Phosphoribosylaminoimidazolecarboxamide formyltransferase','10-formyltetrahydrofolate:5-phosphoribosyl-5-amino-4-imidazolecarboxamide formyltransferase','NULL','NULL','10-formyltetrahydrofolate + 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide = tetrahydrofolate + 5-formamido-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.4','Glycine formimidoyltransferase','FIG formiminotransferase','NULL','NULL','5-formimidoyltetrahydrofolate + glycine = tetrahydrofolate +N-formimidoylglycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.5','Glutamate formimidoyltransferase','Formiminoglutamic acid transferase','Pyridoxal 5-phosphate.','In eukaryotes occurs as a bifunctional enzyme also having EC 4.3.1.4activity.Formerly EC 2.1.2.6.','(1) 5-formimidoyltetrahydrofolate + L-glutamate = tetrahydrofolate +N-formimidoyl-L-glutamate.(2) 5-formyltetrahydrofolate + L-glutamate = tetrahydrofolate + N-formyl-L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.6','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.7','D-alanine 2-hydroxymethyltransferase','2-methylserine hydroxymethyltransferase','NULL','Also acts on 2-hydroxymethylserine.','5,10-methylenetetrahydrofolate + D-alanine + H(2)O = tetrahydrofolate +2-methylserine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.8','Deoxycytidylate 5-hydroxymethyltransferase','dCMP hydroxymethylase','NULL','NULL','5,10-methylenetetrahydrofolate + H(2)O + deoxycytidylate =tetrahydrofolate + 5-hydroxymethyldeoxycytidylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.9','Methionyl-tRNA formyltransferase','Formylmethionyl-transfer ribonucleic synthetase','NULL','NULL','10-formyltetrahydrofolate + L-methionyl-tRNA(fMet) = tetrahydrofolate +N-formylmethionyl-tRNA(fMet).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.10','Aminomethyltransferase','Glycine synthase','NULL','A component, with EC 1.4.4.2 and EC 1.8.1.4, of the glycine cleavagesystem, formerly known as glycine synthase.The glycine cleavage system is composed of four components that onlyloosely associate: the P protein (EC 1.4.4.2), the T protein(EC 2.1.2.10), the L protein (EC 1.8.1.4) and the lipoyl-bearing Hprotein.','[Protein]-S(8)-aminomethyldihydrolipoyllysine + tetrahydrofolate =[protein]-dihydrolipoyllysine + 5,10-methylenetetrahydrofolate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.11','3-methyl-2-oxobutanoate hydroxymethyltransferase','Alpha-ketoisovalerate hydroxymethyltransferase','NULL','NULL','5,10-methylenetetrahydrofolate + 3-methyl-2-oxobutanoate + H(2)O =tetrahydrofolate + 2-dehydropantoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.12','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.13','UDP-4-amino-4-deoxy-L-arabinose formyltransferase','ArnAFT','NULL','The activity is part of a bifunctional enzyme also performing thereaction of EC 1.1.1.305.Formerly EC 2.1.2.n1.','10-formyltetrahydrofolate + UDP-4-amino-4-deoxy-beta-L-arabinose =5,6,7,8-tetrahydrofolate + UDP-4-deoxy-4-formamido-beta-L-arabinose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.2.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.1','Methylmalonyl-CoA carboxytransferase','Methylmalonyl-CoA carboxyltransferase','Biotin; Cobalt; Zinc.','NULL','(S)-methylmalonyl-CoA + pyruvate = propanoyl-CoA + oxaloacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.2','Aspartate carbamoyltransferase','Aspartate transcarbamylase','NULL','NULL','Carbamoyl phosphate + L-aspartate = phosphate + N-carbamoyl-L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.3','Ornithine carbamoyltransferase','Citrulline phosphorylase','NULL','The plant enzyme also catalyzes the reactions of EC 2.1.3.6,EC 2.7.2.2 and EC 3.5.3.12, thus acting as putrescine synthase,converting agmatine and ornithine into putrescine and citrullinerespectively.','Carbamoyl phosphate + L-ornithine = phosphate + L-citrulline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.4','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.5','Oxamate carbamoyltransferase','Oxamic transcarbamylase','NULL','NULL','Carbamoyl phosphate + oxamate = phosphate + oxalureate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.6','Putrescine carbamoyltransferase','NULL','NULL','The plant enzyme also catalyzes the reactions of EC 2.1.3.3,EC 2.7.2.2 and EC 3.5.3.12, thus acting as putrescine synthase,converting agmatine and ornithine into putrescine and citrullinerespectively.','Carbamoyl phosphate + putrescine = phosphate + N-carbamoylputrescine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.7','3-hydroxymethylcephem carbamoyltransferase','NULL','NULL','Acts on a wide range of 3-hydroxymethylcephems (a subclass of thecephalosporin antibiotics).Activated by ATP.','Carbamoyl phosphate + a 3-hydroxymethylceph-3-em-4-carboxylate =phosphate + a 3-carbamoyloxymethylcephem.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.8','Lysine carbamoyltransferase','Lysine transcarbamylase','NULL','Not identical with EC 2.1.3.3.','Carbamoyl phosphate + L-lysine = phosphate + L-homocitrulline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.9','N-acetylornithine carbamoyltransferase','Acetylornithine transcarbamylase','NULL','Differs from EC 2.1.3.3.This enzyme replaces EC 2.1.3.3 in the canonic arginine biosyntheticpathway of several eubacteria and has no catalytic activity withL-ornithine as substrate.','Carbamoyl phosphate + N(2)-acetyl-L-ornithine = phosphate + N-acetyl-L-citrulline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.10','Malonyl-S-ACP:biotin-protein carboxyltransferase','NULL','NULL','Derived from the components MadC and MadD of the anaerobic bacteriumMalonomonas rubra, this enzyme is a component of EC 4.1.1.89.The carboxy group is transferred from malonate to the prostheticgroup of the biotin protein (MadF) with retention of configuration.Similar to EC 4.1.1.87, which forms part of the biotin-independentEC 4.1.1.88, this enzyme also follows on from EC 2.3.1.187,and results in the regeneration of the acetyl-[acyl-carrier protein].','A malonyl-[acyl-carrier protein] + a biotinyl-[protein] = an acetyl-[acyl-carrier protein] + a carboxybiotinyl-[protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.11','N-succinylornithine carbamoyltransferase','N-succinyl-L-ornithine transcarbamylase','NULL','Specific for N-succinyl-L-ornithine and cannot use either L-ornithine(see EC 2.1.3.3) or N-acetyl-L-ornithine (see EC 2.1.3.9) assubstrate.However, a single amino-acid substitution (Pro90Glu) is sufficient toswitch the enzyme to one that uses N-acetyl-L-ornithine as substrate.It is essential for de novo arginine biosynthesis in the obligateanaerobe Bacteroides fragilis, suggesting that this organism uses analternative pathway for synthesizing arginine.','Carbamoyl phosphate + N(2)-succinyl-L-ornithine = phosphate + N-succinyl-L-citrulline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.12','Decarbamoylnovobiocin carbamoyltransferase','NULL','NULL','The enzyme catalyzes the last step in the biosynthesis of theaminocoumarin antibiotic novobiocin.The reaction is activated by ATP.','Carbamoyl phosphate + decarbamoylnovobiocin = phosphate + novobiocin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.13','ATP carbamoyltransferase','NULL','NULL','The enzyme from the bacterium Streptoalloteichus tenebrariuscatalyzes the activation of carbamoyl phosphate toO-carbamoyladenylate, and the subsequent carbamoylation of kanamycinand tobramycin (cf. EC 2.1.3.14).','Carbamoyl phosphate + ATP + H(2)O = diphosphate + O-carbamoyladenylate +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.3.14','Tobramycin carbamoyltransferase','NULL','Iron.','The enzyme from the bacterium Streptoalloteichus tenebrariuscatalyzes the activation of carbamoyl phosphate toO-carbamoyladenylate (see EC 2.1.3.13), and the subsequentcarbamoylation of kanamycin and tobramycin.','(1) O-carbamoyladenylate + tobramycin = AMP + nebramycin 5.(2) O-carbamoyladenylate + kanamycin A = AMP + 6-O-carbamoylkanamycinA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.4.1','Glycine amidinotransferase','L-arginine:glycine amidinotransferase','NULL','Canavanine can act instead of arginine.Formerly EC 2.6.2.1.','L-arginine + glycine = L-ornithine + guanidinoacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.1.4.2','Scyllo-inosamine-4-phosphate amidinotransferase','Inosamine-P amidinotransferase','NULL','1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol 6-phosphate,streptamine phosphate and 2-deoxystreptamine phosphate can also actas acceptors.Canavanine can act as donor.','L-arginine + 1-amino-1-deoxy-scyllo-inositol 4-phosphate = L-ornithine +1-guanidino-1-deoxy-scyllo-inositol 4-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.1','Transketolase','Glycoaldehyde transferase','Thiamine diphosphate.','Wide specificity for both reactants, e.g. converts hydroxypyruvateand R-CHO into CO(2) and R-CHOH-CO-CH(2)OH.Transketolase from Alcaligenes faecalis shows high activity withD-erythrose as acceptor.','Sedoheptulose 7-phosphate + D-glyceraldehyde 3-phosphate = D-ribose5-phosphate + D-xylulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.2','Transaldolase','Dihydroxyacetone transferase','NULL','NULL','Sedoheptulose 7-phosphate + D-glyceraldehyde 3-phosphate = D-erythrose4-phosphate + D-fructose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.3','Formaldehyde transketolase','DHAS','Thiamine diphosphate.','Also converts hydroxypyruvate and formaldehyde into glycerone andCO(2).Not identical with EC 2.2.1.1.','D-xylulose 5-phosphate + formaldehyde = D-glyceraldehyde 3-phosphate +glycerone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.4','Acetoin--ribose-5-phosphate transaldolase','1-deoxy-D-altro-heptulose-7-phosphate synthetase','Thiamine diphosphate.','NULL','3-hydroxybutan-2-one + D-ribose 5-phosphate = acetaldehyde + 1-deoxy-D-altro-heptulose 7-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.5','2-hydroxy-3-oxoadipate synthase','2-hydroxy-3-oxoadipate glyoxylate-lyase (carboxylating)','Thiamine diphosphate.','The product decarboxylates to 5-hydroxy-4-oxopentanoate.The enzyme can decarboxylate 2-oxoglutarate.Acetaldehyde can replace glyoxylate.Formerly EC 4.1.3.15.','2-oxoglutarate + glyoxylate = 2-hydroxy-3-oxoadipate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.6','Acetolactate synthase','Acetohydroxy acid synthetase','Thiamine diphosphate.','The reaction shown is in the pathway of biosynthesis of valine.The enzyme can also transfer the acetaldehyde from pyruvate to2-oxobutanoate, forming 2-ethyl-2-hydroxy-3-oxobutanoate, also knownas 2-aceto-2-hydroxybutanoate, a reaction in the biosynthesis ofisoleucine.Formerly EC 4.1.3.18.','2 pyruvate = 2-acetolactate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.7','1-deoxy-D-xylulose-5-phosphate synthase','1-deoxy-D-xylulose-5-phosphate pyruvate-lyase (carboxylating)','Thiamine diphosphate.','The enzyme forms part of an alternative nonmevalonate pathway forterpenoid biosynthesis.Formerly EC 4.1.3.37.','Pyruvate + D-glyceraldehyde 3-phosphate = 1-deoxy-D-xylulose 5-phosphate+ CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.8','Fluorothreonine transaldolase','NULL','Pyridoxal 5-phosphate.','Can also convert chloroacetaldehyde into 4-chloro-L-threonine.Unlike EC 2.1.2.1 does not use glycine as a substrate.','L-threonine + fluoroacetaldehyde = acetaldehyde + 4-fluoro-L-threonine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.9','2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic-acidsynthase','SEPHCHC synthase','Magnesium.','Involved in the biosynthesis of vitamin K(2) (menaquinone).In most anaerobes and all Gram-positive aerobes, menaquinone is thesole electron transporter in the respiratory chain and is essentialfor their survival.It had previously been thought that the products of the reaction were(1R,6R)-6-hydroxy-2-succinylcyclohexa-2,4-diene-1-carboxylate(SHCHC), pyruvate and CO(2) but it is now known that two separateenzymes are involved: this enzyme and EC 4.2.99.20.Under basic conditions, the product can spontaneously lose pyruvateto form SHCHC.Formerly EC 2.5.1.64 and EC 2.5.1.n1.','Isochorismate + 2-oxoglutarate = 5-enolpyruvoyl-6-hydroxy-2-succinyl-cyclohex-3-ene-1-carboxylate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.10','2-amino-3,7-dideoxy-D-threo-hept-6-ulosonate synthase','ADH synthase','NULL','The enzyme plays a key role in an alternative pathway of thebiosynthesis of 3-dehydroquinate (DHQ), which is involved in thecanonical pathway for the biosynthesis of aromatic amino acids.The enzyme can also catalyze the cleavage of fructose-1,6-bisphosphate to glyceraldehyde-3-phosphate and dihydroxyacetonephosphate.Formerly EC 4.1.2.n5.','L-aspartate 4-semialdehyde + 1-deoxy-D-threo-hexo-2,5-diulose 6-phosphate= 2-amino-2,3,7-trideoxy-D-lyxo-hept-6-ulosonate + 2,3-dioxopropylaldehyde phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.2.1.11','6-deoxy-5-ketofructose 1-phosphate synthase','DKFP synthase','NULL','The enzyme plays a key role in an alternative pathway of thebiosynthesis of 3-dehydroquinate (DHQ), which is involved in thecanonical pathway for the biosynthesis of aromatic amino acids.The enzyme can also catalyze the reaction of EC 4.1.2.13.','(1) Methylglyoxal + D-fructose 1,6-bisphosphate = D-glyceraldehyde3-phosphate + 1-deoxy-D-threo-hexo-2,5-diulose 6-phosphate.(2) Methylglyoxal + D-fructose 1-phosphate = D-glyceraldehyde + 1-deoxy-D-threo-hexo-2,5-diulose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.1','Amino-acid N-acetyltransferase','N-acetylglutamate synthase','NULL','Also acts with L-aspartate and, more slowly, with some other aminoacids.','Acetyl-CoA + L-glutamate = CoA + N-acetyl-L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.2','Imidazole N-acetyltransferase','Imidazole acetylase','NULL','Also acts with propanoyl-CoA.','Acetyl-CoA + imidazole = CoA + N-acetylimidazole.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.3','Glucosamine N-acetyltransferase','Glucosamine acetylase','NULL','NULL','Acetyl-CoA + D-glucosamine = CoA + N-acetyl-D-glucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.4','Glucosamine-phosphate N-acetyltransferase','Aminodeoxyglucosephosphate acetyltransferase','NULL','NULL','Acetyl-CoA + D-glucosamine 6-phosphate = CoA + N-acetyl-D-glucosamine6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.5','Arylamine N-acetyltransferase','Arylamine acetylase','NULL','Wide specificity for aromatic amines, including serotonin; alsocatalyzes acetyl-transfer between arylamines without CoA.','Acetyl-CoA + an arylamine = CoA + an N-acetylarylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.6','Choline O-acetyltransferase','CHOACTase','NULL','Propanoyl-CoA can act, more slowly, in place of acetyl-CoA.','Acetyl-CoA + choline = CoA + O-acetylcholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.7','Carnitine O-acetyltransferase','Carnitine acetylase','NULL','Also acts on propanoyl-CoA and butanoyl-CoA (cf. EC 2.3.1.21 andEC 2.3.1.137).','Acetyl-CoA + carnitine = CoA + O-acetylcarnitine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.8','Phosphate acetyltransferase','Phosphoacylase','NULL','Also acts with other short-chain acyl-CoAs.','Acetyl-CoA + phosphate = CoA + acetyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.9','Acetyl-CoA C-acetyltransferase','Acetoacetyl-CoA thiolase','NULL','NULL','2 acetyl-CoA = CoA + acetoacetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.10','Hydrogen-sulfide S-acetyltransferase','NULL','NULL','NULL','Acetyl-CoA + hydrogen sulfide = CoA + thioacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.11','Thioethanolamine S-acetyltransferase','Thioethanolamine acetyltransferase','NULL','2-sulfanylethanol (2-mercaptoethanol) can act as a substrate.','Acetyl-CoA + 2-aminoethanethiol = CoA + S-(2-aminoethyl)thioacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.12','Dihydrolipoyllysine-residue acetyltransferase','Acetyl-CoA:dihydrolipoamide S-acetyltransferase','NULL','A multimer (24-mer or 60-mer, depending on the source) of this enzymeforms the core of the pyruvate dehydrogenase multienzyme complex,and binds tightly both EC 1.2.4.1 and EC 1.8.1.4.The lipoyl group of this enzyme is reductively acetylated byEC 1.2.4.1, and the only observed direction catalyzed by EC 2.3.1.12is that where the acetyl group is passed to coenzyme A.','Acetyl-CoA + enzyme N(6)-(dihydrolipoyl)lysine = CoA + enzyme N(6)-(S-acetyldihydrolipoyl)lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.13','Glycine N-acyltransferase','NULL','NULL','The CoA derivatives of a number of aliphatic and aromatic acids,but not phenylacetyl-CoA or (indol-3-yl)acetyl-CoA, can act as donor.Not identical with EC 2.3.1.68 or EC 2.3.1.71.','Acyl-CoA + glycine = CoA + N-acylglycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.14','Glutamine N-phenylacetyltransferase','NULL','NULL','NULL','Phenylacetyl-CoA + L-glutamine = CoA + alpha-N-phenylacetyl-L-glutamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.15','Glycerol-3-phosphate 1-O-acyltransferase','Glycerol-3-phosphate O-acyltransferase','NULL','Acyl-[acyl-carrier-protein] can also act as acyl donor.Acts only with derivatives of fatty acids of chain length aboveC(10).','Acyl-CoA + sn-glycerol 3-phosphate = CoA + 1-acyl-sn-glycerol3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.16','Acetyl-CoA C-acyltransferase','3-ketoacyl-CoA thiolase','NULL','NULL','Acyl-CoA + acetyl-CoA = CoA + 3-oxoacyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.17','Aspartate N-acetyltransferase','NULL','NULL','NULL','Acetyl-CoA + L-aspartate = CoA + N-acetyl-L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.18','Galactoside O-acetyltransferase','Thiogalactoside acetyltransferase','NULL','Acts on thiogalactosides and phenylgalactoside.','Acetyl-CoA + a beta-D-galactoside = CoA + a 6-acetyl-beta-D-galactoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.19','Phosphate butyryltransferase','Phosphotransbutyrylase','NULL','NULL','Butanoyl-CoA + phosphate = CoA + butanoylphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.20','Diacylglycerol O-acyltransferase','Diglyceride acyltransferase','NULL','Palmitoyl-CoA and other long-chain acyl-CoAs can act as donors.','Acyl-CoA + 1,2-diacylglycerol = CoA + triacylglycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.21','Carnitine O-palmitoyltransferase','NULL','NULL','Broad specificity to acyl group, over the range C(8) to C(18);optimal activity with palmitoyl-CoA (cf. EC 2.3.1.7 andEC 2.3.1.137).','Palmitoyl-CoA + L-carnitine = CoA + L-palmitoylcarnitine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.22','2-acylglycerol O-acyltransferase','Acylglycerol palmitoyltransferase','NULL','Various 2-acylglycerols can act as acceptor.Palmitoyl-CoA and other long-chain acyl-CoAs can act as donors.The sn-1 position and the sn-3 position are both acylated, at aboutthe same rate.','Acyl-CoA + 2-acylglycerol = CoA + diacylglycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.23','1-acylglycerophosphocholine O-acyltransferase','Lysolecithin acyltransferase','NULL','Acts preferentially with unsaturated acyl-CoA derivatives.1-acyl-sn-glycero-3-phosphoinositol can also act as acceptor.','Acyl-CoA + 1-acyl-sn-glycero-3-phosphocholine = CoA + 1,2-diacyl-sn-glycero-3-phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.24','Sphingosine N-acyltransferase','NULL','NULL','Acts on sphingosine or its 2-epimer.','Acyl-CoA + sphingosine = CoA + N-acylsphingosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.25','Plasmalogen synthase','NULL','NULL','Formerly EC 2.3.1.104.','Acyl-CoA + 1-O-(alk-1-enyl)glycero-3-phosphocholine = CoA +plasmenylcholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.26','Sterol O-acyltransferase','Cholesterol acyltransferase','NULL','The animal enzyme is highly specific for transfer of acyl groups witha single cis double bond 9 carbon atoms distant from the carboxygroup.','Acyl-CoA + cholesterol = CoA + cholesterol ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.27','Cortisol O-acetyltransferase','NULL','NULL','NULL','Acetyl-CoA + cortisol = CoA + cortisol 21-acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.28','Chloramphenicol O-acetyltransferase','NULL','NULL','Chloramphenicol is an antibiotic.','Acetyl-CoA + chloramphenicol = CoA + chloramphenicol 3-acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.29','Glycine C-acetyltransferase','2-amino-3-ketobutyrate coenzyme A ligase','Pyridoxal 5-phosphate.','Acts in concert with EC 1.1.1.103 in the degradation of threonine toform glycine.This threonine degradation pathway is common to prokaryotic andeukaryotic cells and the two enzymes involved form a complex.','Acetyl-CoA + glycine = CoA + 2-amino-3-oxobutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.30','Serine O-acetyltransferase','NULL','NULL','NULL','Acetyl-CoA + L-serine = CoA + O-acetyl-L-serine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.31','Homoserine O-acetyltransferase','Homoserine O-trans-acetylase','NULL','NULL','Acetyl-CoA + L-homoserine = CoA + O-acetyl-L-homoserine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.32','Lysine N-acetyltransferase','LAT','NULL','NULL','Acetyl phosphate + L-lysine = phosphate + N(6)-acetyl-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.33','Histidine N-acetyltransferase','NULL','NULL','NULL','Acetyl-CoA + L-histidine = CoA + N-acetyl-L-histidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.34','D-tryptophan N-acetyltransferase','NULL','NULL','NULL','Acetyl-CoA + D-tryptophan = CoA + N-acetyl-D-tryptophan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.35','Glutamate N-acetyltransferase','Acetylglutamate synthetase','NULL','Also has some hydrolytic activity on acetyl-L-ornithine, but the rateis 1% of that of transferase activity.','N(2)-acetyl-L-ornithine + L-glutamate = L-ornithine + N-acetyl-L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.36','D-amino-acid N-acetyltransferase','NULL','NULL','NULL','Acetyl-CoA + a D-amino acid = CoA + an N-acetyl-D-amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.37','5-aminolevulinate synthase','5-aminolevulinic acid synthase','Pyridoxal 5-phosphate.','NULL','Succinyl-CoA + glycine = 5-aminolevulinate + CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.38','[Acyl-carrier-protein] S-acetyltransferase','ACAT','NULL','Essential, along with EC 2.3.1.39, for the initiation of fatty-acidbiosynthesis in bacteria.The substrate acetyl-CoA protects the enzyme against inhibition byN-ethylmaleimide or iodoacetamide.This is one of the activities associated with EC 2.3.1.180.','Acetyl-CoA + [acyl-carrier-protein] = CoA + acetyl-[acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.39','[Acyl-carrier-protein] S-malonyltransferase','Acyl carrier protein malonyltransferase','NULL','Essential, along with EC 2.3.1.38, for the initiation of fatty-acidbiosynthesis in bacteria.Also provides the malonyl groups for polyketide biosynthesis.The product of the reaction, malonyl-ACP, is an elongation substratein fatty-acid biosynthesis.In Mycobacterium tuberculosis, holo-ACP (the product of EC 2.7.8.7)is the preferred substrate.This enzyme also forms part of the multienzyme complexes EC 4.1.1.88and EC 4.1.1.89.Malonylation of ACP is immediately followed by decarboxylation withinthe malonate-decarboxylase complex to yield acetyl-ACP,the catalytically active species of the decarboxylase.In the enzyme from Klebsiella pneumoniae, methylmalonyl-CoA can alsoact as a substrate but acetyl-CoA cannot whereas the enzyme fromPseudomonas putida can use both as substrates.The ACP subunit found in fatty-acid biosynthesis contains apantetheine-4-phosphate prosthetic group; that from malonatedecarboxylase also contains pantetheine-4-phosphate but in the formof a 2-(5-triphosphoribosyl)-3-dephospho-CoA prosthetic group.','Malonyl-CoA + [acyl-carrier-protein] = CoA + malonyl-[acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.40','Acyl-[acyl-carrier-protein]--phospholipid O-acyltransferase','NULL','NULL','NULL','Acyl-[acyl-carrier-protein] + O-(2-acyl-sn-glycero-3-phospho)ethanolamine= [acyl-carrier-protein] + O-(1-beta-acyl-2-acyl-sn-glycero-3-phospho)ethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.41','Beta-ketoacyl-[acyl-carrier-protein] synthase I','3-ketoacyl-acyl carrier protein synthase','NULL','Responsible for the chain-elongation step of dissociated (type II)fatty-acid biosynthesis, i.e. the addition of two C atoms to thefatty-acid chain.Escherichia coli mutants that lack this enzyme are deficient inunsaturated fatty acids.Can use fatty acyl thioesters of ACP (C(2) to C(16)) as substrates,as well as fatty acyl thioesters of Co-A (C(4) to C(16)).The substrate specificity is very similar to that of EC 2.3.1.179with the exception that the latter enzyme is far more active withpalmitoleoyl-ACP (C(16)-Delta(9)) as substrate, allowing the organismto regulate its fatty-acid composition with changes in temperature.','Acyl-[acyl-carrier-protein] + malonyl-[acyl-carrier-protein] = 3-oxoacyl-[acyl-carrier-protein] + CO(2) + [acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.42','Glycerone-phosphate O-acyltransferase','Dihydroxyacetone phosphate acyltransferase','NULL','Uses CoA derivatives of palmitate, stearate, and oleate, with highestactivity on palmitoyl-CoA.','Acyl-CoA + glycerone phosphate = CoA + acylglycerone phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.43','Phosphatidylcholine--sterol O-acyltransferase','LCAT','NULL','Palmitoyl, oleoyl, and linoleoyl can be transferred; a number ofsterols, including cholesterol, can act as acceptors.The bacterial enzyme also catalyzes the reactions of EC 3.1.1.4 andEC 3.1.1.5.','Phosphatidylcholine + a sterol = 1-acylglycerophosphocholine +a sterol ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.44','N-acetylneuraminate 4-O-acetyltransferase','NULL','NULL','Both free and glycosidically bound N-acetyl- and N-glycolyl-neuraminates can act as O-acetyl acceptors.','Acetyl-CoA + N-acetylneuraminate = CoA + N-acetyl-4-O-acetylneuraminate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.45','N-acetylneuraminate 7-O(or 9-O)-acetyltransferase','NULL','NULL','Both free and glycosidically bound N-acetyl- andN-glycolylneuraminates can act as O-acetyl acceptors.','Acetyl-CoA + N-acetylneuraminate = CoA + N-acetyl-7-O(or 9-O)-acetylneuraminate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.46','Homoserine O-succinyltransferase','Homoserine O-transsuccinylase','NULL','NULL','Succinyl-CoA + L-homoserine = CoA + O-succinyl-L-homoserine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.47','8-amino-7-oxononanoate synthase','7-KAP synthetase','Pyridoxal 5-phosphate.','The enzyme catalyzes the decarboxylative condensation of L-alanineand pimeloyl-[acyl-carrier protein], a key step in the pathway forbiotin biosynthesis.Pimeloyl-CoA can be used with lower efficiency.','Pimeloyl-[acyl-carrier protein] + L-alanine = 8-amino-7-oxononanoate +CO(2) + holo-[acyl-carrier protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.48','Histone acetyltransferase','NULL','NULL','A group of enzymes with differing specificities toward histoneacceptors.','Acetyl-CoA + [histone] = CoA + acetyl-[histone].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.49','Deacetyl-[citrate-(pro-3S)-lyase] S-acetyltransferase','NULL','NULL','Both this enzyme and EC 6.2.1.22 acetylate and activate EC 4.1.3.6.','S-acetylphosphopantotheine + deacetyl-[citrate-oxaloacetate-lyase((pro-3S)-CH(2)COO(-)-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.50','Serine C-palmitoyltransferase','NULL','Pyridoxal 5-phosphate.','NULL','Palmitoyl-CoA + L-serine = CoA + 3-dehydro-D-sphinganine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.51','1-acylglycerol-3-phosphate O-acyltransferase','NULL','NULL','Acyl-[acyl-carrier-protein] can also act as acyl donor.The animal enzyme is specific for the transfer of unsaturated fattyacyl groups.','Acyl-CoA + 1-acyl-sn-glycerol 3-phosphate = CoA + 1,2-diacyl-sn-glycerol3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.52','2-acylglycerol-3-phosphate O-acyltransferase','NULL','NULL','Saturated acyl-CoA thioesters are the most effective acyl donors.','Acyl-CoA + 2-acyl-sn-glycerol 3-phosphate = CoA + 1,2-diacyl-sn-glycerol3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.53','Phenylalanine N-acetyltransferase','NULL','NULL','Also acts, more slowly, on L-histidine and L-alanine.','Acetyl-CoA + L-phenylalanine = CoA + N-acetyl-L-phenylalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.54','Formate C-acetyltransferase','Pyruvate formate-lyase','NULL','NULL','Acetyl-CoA + formate = CoA + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.55','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.56','Aromatic-hydroxylamine O-acetyltransferase','NULL','NULL','Transfers the N-acetyl group of some aromatic acethydroxamates to theO-position of some aromatic hydroxylamines.','N-hydroxy-4-acetylaminobiphenyl + N-hydroxy-4-aminobiphenyl = N-hydroxy-4-aminobiphenyl + N-acetoxy-4-aminobiphenyl.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.57','Diamine N-acetyltransferase','Acetyl-coenzyme A-1,4-diaminobutane N-acetyltransferase','NULL','Acts on 1,3-diaminopropane, 1,5-diaminopentane, putrescine,spermidine (forming N(1)- and N(8)-acetylspermidine), spermine, N(1)-acetylspermidine and N(8)-acetylspermidine.','Acetyl-CoA + an alkane-alpha,omega-diamine = CoA + an N-acetyldiamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.58','2,3-diaminopropionate N-oxalyltransferase','ODAP synthase','NULL','NULL','Oxalyl-CoA + L-2,3-diaminopropanoate = CoA + N(3)-oxalyl-L-2,3-diaminopropanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.59','Gentamicin 2-N-acetyltransferase','Gentamycin acetyltransferase II','NULL','The antibiotics gentamicin A, sisomicin, tobramycin, paromomycin,neomycin B, kanamycin B and kanamycin C can also act as acceptors.','Acetyl-CoA + gentamicin C(1a) = CoA + N(2)-acetylgentamicin C(1a).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.60','Gentamicin 3-N-acetyltransferase','Aminoglycoside acetyltransferase AAC(3)-I','NULL','Also acetylates sisomicin.','Acetyl-CoA + gentamicin C = CoA + N(3)-acetylgentamicin C.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.61','Dihydrolipoyllysine-residue succinyltransferase','Dihydrolipoamide S-succinyltransferase','NULL','A multimer (24-mer) of this enzyme forms the core of the multienzymecomplex, and binds tightly both EC 1.2.4.2 and EC 1.8.1.4.The lipoyl group of this enzyme is reductively succinylated byEC 1.2.4.2, and the only observed direction catalyzed by EC 2.3.1.61is that where this succinyl group is passed to coenzyme A.','Succinyl-CoA + enzyme N(6)-(dihydrolipoyl)lysine = CoA + enzyme N(6)-(S-succinyldihydrolipoyl)lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.62','2-acylglycerophosphocholine O-acyltransferase','NULL','NULL','NULL','Acyl-CoA + 2-acyl-sn-glycero-3-phosphocholine = CoA +phosphatidylcholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.63','1-alkylglycerophosphocholine O-acyltransferase','NULL','NULL','May be identical with EC 2.3.1.23.','Acyl-CoA + 1-alkyl-sn-glycero-3-phosphocholine = CoA + 2-acyl-1-alkyl-sn-glycero-3-phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.64','Agmatine N(4)-coumaroyltransferase','Agmatine coumaroyltransferase','NULL','NULL','4-coumaroyl-CoA + agmatine = CoA + N-(4-guanidinobutyl)-4-hydroxycinnamamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.65','Bile acid-CoA:amino acid N-acyltransferase','Amino acid N-choloyltransferase','NULL','Also acts on CoA derivatives of other bile acids.Taurine and 2-fluoro-beta-alanine can act as substrates, but moreslowly.Can also conjugate fatty acids to glycine and can act as a very-long-chain acyl-CoA thioesterase.Bile-acid--amino-acid conjugates serve as detergents in thegastrointestinal tract, solubilizing long chain fatty acids, mono-and diglycerides, fat-soluble vitamins and cholesterol.This is the second enzyme in a two-step process leading to theconjugation of bile acids with amino acids; the first step is theconversion of bile acids into their acyl-CoA thioesters, which iscatalyzed by EC 6.2.1.7.','Choloyl-CoA + glycine = CoA + glycocholate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.66','Leucine N-acetyltransferase','NULL','NULL','Propanoyl-CoA can act, more slowly, as donor.L-arginine, L-valine, L-phenylalanine and peptides containingL-leucine can act as acceptor.','Acetyl-CoA + L-leucine = CoA + N-acetyl-L-leucine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.67','1-alkylglycerophosphocholine O-acetyltransferase','1-alkyl-2-lyso-sn-glycero-3-phosphocholine acetyltransferase','NULL','NULL','Acetyl-CoA + 1-alkyl-sn-glycero-3-phosphocholine = CoA + 2-acetyl-1-alkyl-sn-glycero-3-phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.68','Glutamine N-acyltransferase','NULL','NULL','Phenylacetyl-CoA and (indol-3-yl)acetyl-CoA, but not benzoyl-CoA,can act as acyl donor.Not identical with EC 2.3.1.13 or EC 2.3.1.71.','Acyl-CoA + L-glutamine = CoA + N-acyl-L-glutamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.69','Monoterpenol O-acetyltransferase','Menthol transacetylase','NULL','(-)-menthol, (+)-neomenthol, borneol, cyclohexanol and N-decanol canbe acetylated.','Acetyl-CoA + a monoterpenol = CoA + a monoterpenol acetate ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.70','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.71','Glycine N-benzoyltransferase','NULL','NULL','Not identical with EC 2.3.1.13 or EC 2.3.1.68.','Benzoyl-CoA + glycine = CoA + hippurate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.72','Indoleacetylglucose--inositol O-acyltransferase','NULL','NULL','The position of acylation is indeterminate because of the ease ofacyl transfer between hydroxy groups.','1-O-(indol-3-yl)acetyl-beta-D-glucose + myo-inositol = D-glucose +O-(indol-3-yl)acetyl-myo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.73','Diacylglycerol--sterol O-acyltransferase','NULL','NULL','Cholesterol, sitosterol, campesterol and diacylglycerol can act asacceptors.Transfers a number of long-chain fatty acyl groups.','1,2-diacyl-sn-glycerol + a sterol = monoacylglycerol + a sterol ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.74','Naringenin-chalcone synthase','6-deoxychalcone synthase','NULL','In the presence of NADH and a reductase, 6-deoxychalcone isproduced.','3 malonyl-CoA + 4-coumaroyl-CoA = 4 CoA + naringenin chalcone + 3 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.75','Long-chain-alcohol O-fatty-acyltransferase','Wax synthase','NULL','Transfers saturated or unsaturated acyl residues of chain lengthC(18) to C(20) to long-chain alcohols, forming waxes.The best acceptor is cis-icos-11-en-1-ol.','Acyl-CoA + a long-chain alcohol = CoA + a long-chain ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.76','Retinol O-fatty-acyltransferase','NULL','NULL','Acts on palmitoyl-CoA and other long-chain fatty-acyl derivatives ofCoA.','Acyl-CoA + retinol = CoA + retinyl ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.77','Triacylglycerol--sterol O-acyltransferase','NULL','NULL','Tripalmitoylglycerol can act as donor, and, more slowly, othertriacylglycerols containing C(6) to C(22) fatty acids.The best acceptors are 3-beta-hydroxysteroids with a planar ringsystem.','Triacylglycerol + a 3-beta-hydroxysteroid = diacylglycerol + a 3-beta-hydroxysteroid ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.78','Heparan-alpha-glucosaminide N-acetyltransferase','NULL','NULL','Brings about the acetylation of glucosamine groups of heparan sulfateand heparin from which the sulfate has been removed.Also acts on heparin.Not identical with EC 2.3.1.3 or EC 2.3.1.4.','Acetyl-CoA + heparan sulfate alpha-D-glucosaminide = CoA + heparansulfate N-acetyl-alpha-D-glucosaminide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.79','Maltose O-acetyltransferase','NULL','NULL','Not identical with EC 2.3.1.18.','Acetyl-CoA + maltose = CoA + acetyl-maltose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.80','Cysteine-S-conjugate N-acetyltransferase','NULL','NULL','S-benzyl-L-cysteine and, in decreasing order of activity, S-butyl-L-cysteine, S-propyl-L-cysteine, O-benzyl-L-serine and S-ethyl-L-cysteine, can act as acceptor.','Acetyl-CoA + an L-cysteine-S-conjugate = CoA + an N-acetyl L-cysteine-S-conjugate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.81','Aminoglycoside N(3)-acetyltransferase','Gentamicin-(3)-N-acetyltransferase','NULL','A wide range of antibiotics containing the 2-deoxystreptamine ringcan act as acceptor, including gentamicin, kanamycin, tobramycin,neomycin and apramycin.Different from EC 2.3.1.60.','Acetyl-CoA + a 2-deoxystreptamine antibiotic = CoA + N(3)-acetyl-2-deoxystreptamine antibiotic.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.82','Aminoglycoside N(6)-acetyltransferase','6-aminoglycoside-N-acetyltransferase','NULL','The antibiotics kanamycin A, kanamycin B, neomycin, gentamicin C(1a),gentamicin C(2), sisomicin and tobramycin are substrates.The antibiotic paromomycin is not a substrate.The 6-amino group of the purpurosamine ring is acetylated.Formerly EC 2.3.1.55.','Acetyl-CoA + kanamycin-B = CoA + N(6)-acetylkanamycin-B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.83','Phosphatidylcholine--dolichol O-acyltransferase','NULL','NULL','NULL','3-sn-phosphatidylcholine + dolichol = 1-acyl-sn-glycero-3-phosphocholine+ acyldolichol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.84','Alcohol O-acetyltransferase','AATASE','NULL','Acts on a range of short-chain aliphatic alcohols, including methanoland ethanol.','Acetyl-CoA + an alcohol = CoA + an acetyl ester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.85','Fatty-acid synthase','NULL','NULL','The animal enzyme is a multifunctional protein catalyzing thereactions of EC 2.3.1.38, EC 2.3.1.39, EC 2.3.1.41, EC 1.1.1.100,EC 4.2.1.59, EC 1.3.1.39 and EC 3.1.2.14.','Acetyl-CoA + n malonyl-CoA + 2n NADPH = a long-chain fatty acid + (n+1)CoA + n CO(2) + 2n NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.86','Fatty-acyl-CoA synthase','Yeast fatty acid synthase','NULL','The enzyme from yeasts (Ascomycota and Basidiomycota) is a multi-functional protein complex composed of two subunits.One subunit catalyzes the reactions EC 1.1.1.100 and EC 2.3.1.41,while the other subunit catalyzes the reactions of EC 2.3.1.38,EC 2.3.1.39, EC 4.2.1.59, EC 1.3.1.10 and EC 1.1.1.279.The enzyme differs from the animal enzyme (EC 2.3.1.85) in that theenoyl reductase domain requires FMN as a cofactor, and the ultimateproduct is an acyl-CoA (usually palmitoyl-CoA) instead of a freefatty acid.','Acetyl-CoA + n malonyl-CoA + 2n NADPH = long-chain-acyl-CoA + n CoA +n CO(2) + 2n NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.87','Aralkylamine N-acetyltransferase','AANAT','NULL','Narrow specificity toward 2-arylethylamines, including serotonin(5-hydroxytryptamine), tryptamine, 5-methoxytryptamine andphenylethylamine.This is the penultimate enzyme in the production of melatonin(5-methoxy-N-acetyltryptamine) and controls its synthesis (cf.EC 2.1.1.4).Differs from EC 2.3.1.5.','Acetyl-CoA + a 2-arylethylamine = CoA + an N-acetyl-2-arylethylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.88','Peptide alpha-N-acetyltransferase','Amino-terminal amino acid-acetylating enzyme','NULL','Acetylates N-terminal alanine, serine, methionine and glutamateresidues in a number of peptides and proteins, including beta-endorphin, corticotropins and melanotropin (cf. EC 2.3.1.108).','Acetyl-CoA + peptide = N(alpha)-acetylpeptide + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.89','Tetrahydrodipicolinate N-acetyltransferase','Tetrahydrodipicolinate acetylase','NULL','NULL','Acetyl-CoA + (S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate + H(2)O =CoA + L-2-acetamido-6-oxoheptanedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.90','Beta-glucogallin O-galloyltransferase','NULL','NULL','Beta-glucogallin can act as donor and as acceptor.Digalloylglucose can also act as acceptor, with the formation of1-O,2-O,6-O-trigalloylglucose.','2 1-O-galloyl-beta-D-glucose = D-glucose + 1-O,6-O-digalloyl-beta-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.91','Sinapoylglucose--choline O-sinapoyltransferase','Sinapine synthase','NULL','NULL','1-O-sinapoyl-beta-D-glucose + choline = D-glucose + sinapoylcholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.92','Sinapoylglucose--malate O-sinapoyltransferase','NULL','NULL','NULL','1-O-sinapoyl-beta-D-glucose + (S)-malate = D-glucose + sinapoyl-(S)-malate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.93','13-hydroxylupanine O-tigloyltransferase','NULL','NULL','Benzoyl-CoA and, more slowly, pentanoyl-CoA, 3-methylbutanoyl-CoA andbutanoyl-CoA can act as acyl donors.Involved in the synthesis of lupanine alkaloids.','(E)-2-methylcrotonoyl-CoA + 13-hydroxylupanine = CoA + 13-((E)-2-methylcrotonoyl)oxylupanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.94','6-deoxyerythronolide-B synthase','6-deoxyerythronolide B synthase','NULL','The product, 6-deoxyerythronolide B, contains a 14-membered lactonering and is an intermediate in the biosynthesis of erythromycinantibiotics.Biosynthesis of 6-deoxyerythronolide B requires 28 active sites thatare precisely arranged along three large polypeptides, denoted DEBS1,-2 and -3.The polyketide product is synthesized by the processive action of aloading didomain, six extension modules and a terminal thioesterasedomain.Each extension module contains a minimum of a ketosynthase (KS),an acyltransferase (AT) and an acyl-carrier protein (ACP).The KS domain both accepts the growing polyketide chain from theprevious module and catalyzes the subsequent decarboxylativecondensation between this substrate and an ACP-bound methylmalonylextender unit, introduce by the AT domain.This combined effort gives rise to a new polyketide intermediate thathas been extended by two carbon atoms.','Propanoyl-CoA + 6 (2S)-methylmalonyl-CoA + 6 NADPH = 6-deoxyerythronolideB + 7 CoA + 6 CO(2) + H(2)O + 6 NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.95','Trihydroxystilbene synthase','Resveratrol synthase','NULL','Not identical with EC 2.3.1.74 or EC 2.3.1.146.','3 malonyl-CoA + 4-coumaroyl-CoA = 4 CoA + trans-resveratrol + 4 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.96','Glycoprotein N-palmitoyltransferase','NULL','NULL','Acts on glycoprotein from mucus.Activated by 1,4-dithiothreitol.','Palmitoyl-CoA + glycoprotein = CoA + N-palmitoylglycoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.97','Glycylpeptide N-tetradecanoyltransferase','Peptide N-myristoyltransferase','NULL','The enzyme from Saccharomyces cerevisiae is highly specific fortetradecanoyl-CoA, and for N-terminal glycine in oligopeptidescontaining serine in the 5-position.The enzyme from mammalian heart transfers acyl groups to a specificacceptor protein of 51 kDa.','Tetradecanoyl-CoA + glycylpeptide = CoA + N-tetradecanoylglycylpeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.98','Chlorogenate--glucarate O-hydroxycinnamoyltransferase','NULL','NULL','Galactarate can act as acceptor, more slowly.Involved with EC 2.3.1.99 in the formation of caffeoylglucarate intomato.','Chlorogenate + glucarate = quinate + 2-O-caffeoylglucarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.99','Quinate O-hydroxycinnamoyltransferase','Hydroxycinnamoyl coenzyme A-quinate transferase','NULL','Caffeoyl-CoA and 4-coumaroyl-CoA can also act as donors, more slowly.Involved in the biosynthesis of chlorogenic acid in sweet potato and,with EC 2.3.1.98 in the formation of caffeoyl-CoA in tomato.','Feruloyl-CoA + quinate = CoA + O-feruloylquinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.100','[Myelin-proteolipid] O-palmitoyltransferase','Acyl-protein synthase','NULL','The enzyme in brain transfers long-chain acyl residues to theendogenous myelin proteolipid.','Palmitoyl-CoA + [myelin proteolipid] = CoA + O-palmitoyl-[myelinproteolipid].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.101','Formylmethanofuran--tetrahydromethanopterin N-formyltransferase','NULL','NULL','Methanofuran is a complex 4-substituted furfurylamine and is involvedin the formation of methane from CO(2) in Methanobacteriumthermoautotrophicum.','Formylmethanofuran + 5,6,7,8-tetrahydromethanopterin = methanofuran +5-formyl-5,6,7,8-tetrahydromethanopterin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.102','N(6)-hydroxylysine O-acetyltransferase','N(6)-hydroxylysine acetylase','NULL','Involved in the synthesis of aerobactin from lysine in a strain ofEscherichia coli.','Acetyl-CoA + N(6)-hydroxy-L-lysine = CoA + N(6)-acetyl-N(6)-hydroxy-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.103','Sinapoylglucose--sinapoylglucose O-sinapoyltransferase','NULL','NULL','NULL','2 1-O-sinapoyl beta-D-glucoside = D-glucose + 1,2-bis-O-sinapoyl beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.104','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.105','Alkylglycerophosphate 2-O-acetyltransferase','NULL','NULL','Involved in the biosynthesis of thrombocyte activating factor inanimal tissues.','Acetyl-CoA + 1-alkyl-sn-glycero-3-phosphate = CoA + 1-alkyl-2-acetyl-sn-glycero-3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.106','Tartronate O-hydroxycinnamoyltransferase','Hydroxycinnamoyl-coenzyme-A:tartronate hydroxycinnamoyltransferase','NULL','4-coumaroyl-CoA (4-hydroxycinnamoyl-CoA), caffeoyl-CoA (3,4-dihydroxycinnamoyl-CoA) and feruloyl-CoA (4-hydroxy-3-methoxycinnamoyl-CoA) can also act as donors for the enzyme from themung bean (Vigna radiata).','Sinapoyl-CoA + 2-hydroxymalonate = CoA + sinapoyltartronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.107','Deacetylvindoline O-acetyltransferase','17-O-deacetylvindoline O-acetyltransferase','NULL','Catalyzes the final step in the biosynthesis of vindoline fromtabersonine in the Madagascar periwinkle, Catharanthus roseus.','Acetyl-CoA + deacetylvindoline = CoA + vindoline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.108','Alpha-tubulin N-acetyltransferase','Alpha-tubulin acetylase','NULL','The enzyme from Chlamydomonas flagella also acetylates mammalianbrain alpha-tubulin.','Acetyl-CoA + [alpha-tubulin]-L-lysine = CoA + [alpha-tubulin]-N(6)-acetyl-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.109','Arginine N-succinyltransferase','AOST','NULL','Also acts on L-ornithine.This is the first enzyme in the arginine succinyltransferase (AST)pathway for the catabolism of arginine.This pathway converts the carbon skeleton of arginine into glutamate,with the concomitant production of ammonia and conversion ofsuccinyl-CoA into succinate and CoA.The five enzymes involved in this pathway are EC 2.3.1.109,EC 3.5.3.23, EC 2.6.1.81, EC 1.2.1.71 and EC 3.5.1.96.','Succinyl-CoA + L-arginine = CoA + N(2)-succinyl-L-arginine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.110','Tyramine N-feruloyltransferase','NULL','NULL','Cinnamoyl-CoA, 4-coumaroyl-CoA and sinapoyl-CoA can also act asdonors.Some aromatic amines can act as acceptors.','Feruloyl-CoA + tyramine = CoA + N-feruloyltyramine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.111','Mycocerosate synthase','NULL','NULL','Elongates CoA esters of fatty acids from C(6) to C(20) byincorporation of methylmalonyl, but not malonyl, residues, to formmultimethyl-branched fatty-acyl-CoA such as 2,4,6,8-tetramethyl-octosanoyl-CoA.','Acyl-CoA + n methylmalonyl-CoA + 2n NADPH = multi-methyl-branched acyl-CoA + n CoA + n CO(2) + 2n NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.112','D-tryptophan N-malonyltransferase','NULL','NULL','1-aminocyclopropane-1-carboxylate can act instead of malonyl-CoA.','Malonyl-CoA + D-tryptophan = CoA + N(2)-malonyl-D-tryptophan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.113','Anthranilate N-malonyltransferase','NULL','NULL','NULL','Malonyl-CoA + anthranilate = CoA + N-malonylanthranilate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.114','3,4-dichloroaniline N-malonyltransferase','NULL','NULL','NULL','Malonyl-CoA + 3,4-dichloroaniline = CoA + N-(3,4-dichlorophenyl)-malonamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.115','Isoflavone-7-O-beta-glucoside 6-O-malonyltransferase','Flavone/flavonol 7-O-beta-D-glucoside malonyltransferase','NULL','The 6-position of the glucose residue of formononetin can also act asacceptor; some other 7-O-glucosides of isoflavones, flavones andflavonols can also act, but more slowly.','Malonyl-CoA + biochanin A 7-O-beta-D-glucoside = CoA + biochanin A 7-O-(6-O-malonyl-beta-D-glucoside).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.116','Flavonol-3-O-beta-glucoside O-malonyltransferase','NULL','NULL','NULL','Malonyl-CoA + flavonol 3-O-beta-D-glucoside = CoA + flavonol 3-O-(6-O-malonyl-beta-D-glucoside).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.117','2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase','Succinyl-CoA:tetrahydrodipicolinate N-succinyltransferase','NULL','Involved in the biosynthesis of lysine in bacteria, cyanobacteria andhigher plants.Earlier erroneously called 2,3,4,5-tetrahydropyridine-2-carboxylateN-succinyltransferase.','Succinyl-CoA + (S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate + H(2)O =CoA + N-succinyl-L-2-amino-6-oxoheptanedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.118','N-hydroxyarylamine O-acetyltransferase','Arylamine N-acetyltransferase','NULL','The enzyme from liver, but not that from bacteria, can also catalyzeN-acetylation of arylamines and N,O-acetylation of arylhydroxamates.','Acetyl-CoA + an N-hydroxyarylamine = CoA + an N-acetoxyarylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.119','Icosanoyl-CoA synthase','Acyl-CoA elongase','NULL','Icosanoyl-CoA can act in place of stearoyl-CoA.The membrane enzyme brings about the elongation of these long-chainfatty-acyl-CoAs.','Stearoyl-CoA + malonyl-CoA + 2 NAD(P)H = icosanoyl-CoA + CO(2) + CoA +2 NAD(P)(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.120','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.121','1-alkenylglycerophosphoethanolamine O-acyltransferase','NULL','NULL','Long-chain unsaturated acyl-CoAs are the best substrates.','Acyl-CoA + 1-alkenylglycerophosphoethanolamine = CoA + 1-alkenyl-2-acylglycerophosphoethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.122','Trehalose O-mycolyltransferase','NULL','NULL','Catalyzes the exchange of mycolic acid between trehalose, trehalosemycolate and trehalose bismycolate.Trehalose 6-palmitate can also act as donor.','2 alpha,alpha-trehalose 6-mycolate = alpha,alpha-trehalose +alpha,alpha-trehalose 6,6-bismycolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.123','Dolichol O-acyltransferase','NULL','NULL','Other acyl-CoAs can act, more slowly.Alpha-saturated dolichols are acylated more rapidly than the alpha-unsaturated analogs.','Palmitoyl-CoA + dolichol = CoA + dolichyl palmitate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.124','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.125','1-alkyl-2-acetylglycerol O-acyltransferase','1-hexadecyl-2-acetylglycerol acyltransferase','NULL','A number of acyl-CoAs can act as acyl donor; maximum activity isobtained with linoleoyl-CoA.Not identical with EC 2.3.1.20.','Acyl-CoA + 1-O-alkyl-2-acetyl-sn-glycerol = CoA + 1-O-alkyl-2-acetyl-3-acyl-sn-glycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.126','Isocitrate O-dihydroxycinnamoyltransferase','NULL','NULL','Feruloyl-CoA and 4-coumaroyl-CoA can also act as donors.','Caffeoyl-CoA + isocitrate = CoA + 2-O-caffeoylisocitrate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.127','Ornithine N-benzoyltransferase','NULL','NULL','NULL','2 benzoyl-CoA + L-ornithine = 2 CoA + N(2),N(5)-dibenzoyl-L-ornithine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.128','Ribosomal-protein-alanine N-acetyltransferase','NULL','NULL','A groups of enzymes in Escherichia coli that acetylate the N-terminalalanine residues of specific ribosomal proteins (cf. EC 2.3.1.88).','Acetyl-CoA + ribosomal-protein L-alanine = CoA + ribosomal-proteinN-acetyl-L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.129','Acyl-[acyl-carrier-protein]--UDP-N-acetylglucosamine O-acyltransferase','Acyl-[acyl-carrier-protein]-UDP-N-acetylglucosamine O-acyltransferase','NULL','Involved with EC 2.4.1.182 and EC 2.7.1.130 in the biosynthesis ofthe phosphorylated glycolipid, lipid A, in the outer membrane ofEscherichia coli.','(R)-3-hydroxytetradecanoyl-[acyl-carrier-protein] + UDP-N-acetyl-alpha-D-glucosamine = [acyl-carrier-protein] + UDP-3-O-(3-hydroxytetradecanoyl)-N-acetyl-alpha-D-glucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.130','Galactarate O-hydroxycinnamoyltransferase','NULL','NULL','Sinapoyl-CoA and 4-coumaroyl-CoA can also act as donors.','Feruloyl-CoA + galactarate = CoA + O-feruloylgalactarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.131','Glucarate O-hydroxycinnamoyltransferase','NULL','NULL','4-coumaroyl-CoA, feruloyl-CoA and caffeoyl-CoA can also act asdonors, more slowly.','Sinapoyl-CoA + glucarate = CoA + O-sinapoylglucarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.132','Glucarolactone O-hydroxycinnamoyltransferase','NULL','NULL','4-coumaroyl-CoA, feruloyl-CoA and caffeoyl-CoA can also act asdonors, more slowly.','Sinapoyl-CoA + glucarolactone = CoA + O-sinapoylglucarolactone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.133','Shikimate O-hydroxycinnamoyltransferase','NULL','NULL','Caffeoyl-CoA, feruloyl-CoA and sinapoyl-CoA can also act as donors,more slowly.','4-coumaroyl-CoA + shikimate = CoA + 4-coumaroylshikimate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.134','Galactolipid O-acyltransferase','Galactolipid:galactolipid acyltransferase','NULL','Di-D-galactosyldiacylglycerol can also act as acceptor.','2 mono-beta-D-galactosyldiacylglycerol = acylmono-beta-D-galactosyldiacylglycerol + mono-beta-D-galactosylacylglycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.135','Phosphatidylcholine--retinol O-acyltransferase','Lecithin retinol acyl transferase','NULL','A key enzyme in retinoid metabolism, catalyzing the transfer of anacyl group from the sn-1 position of phosphatidylcholine to retinol,forming retinyl esters which are then stored.Recognizes the substrate both in free form and when bound tocellular-retinol-binding-protein, but has higher affinity for thebound form.Can also esterify 11-cis-retinol.','Phosphatidylcholine + retinol--[cellular-retinol-binding-protein] =2-acylglycerophosphocholine + retinyl-ester--[cellular-retinol-binding-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.136','Polysialic-acid O-acetyltransferase','NULL','NULL','Acts only on substrates containing more than 14 sialosyl residues.Catalyzes the modification of capsular polysaccharides in somestrains of Escherichia coli.','Acetyl-CoA + an alpha-2,8-linked polymer of sialic acid = CoA +polysialic acid acetylated at O-7 or O-9.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.137','Carnitine O-octanoyltransferase','NULL','NULL','Acts on a range of acyl-CoAs, with optimal activity with C(6) or C(8)acyl groups.Cf. EC 2.3.1.7 and EC 2.3.1.21.','Octanoyl-CoA + L-carnitine = CoA + L-octanoylcarnitine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.138','Putrescine N-hydroxycinnamoyltransferase','NULL','NULL','Feruloyl-CoA, cinnamoyl-CoA and sinapoyl-CoA can also act as donors,more slowly.','Caffeoyl-CoA + putrescine = CoA + N-caffeoylputrescine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.139','Ecdysone O-acyltransferase','NULL','NULL','NULL','Palmitoyl-CoA + ecdysone = CoA + ecdysone palmitate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.140','Rosmarinate synthase','NULL','NULL','Involved with EC 1.1.1.237 in the biosynthesis of rosmarinic acid.Characterized from the plant Melissa officinalis L. (lemon balm).','Caffeoyl-CoA + (R)-3-(3,4-dihydroxyphenyl)lactate = CoA + rosmarinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.141','Galactosylacylglycerol O-acyltransferase','NULL','NULL','Transfers long-chain acyl groups to the sn-1 position of the glycerolresidues.','Acyl-[acyl-carrier-protein] + sn-3-D-galactosyl-sn-2-acylglycerol =[acyl-carrier-protein] + D-galactosyldiacylglycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.142','Glycoprotein O-fatty-acyltransferase','NULL','NULL','NULL','Palmitoyl-CoA + mucus glycoprotein = CoA + O-palmitoylglycoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.143','Beta-glucogallin--tetrakisgalloylglucose O-galloyltransferase','NULL','NULL','NULL','1-O-galloyl-beta-D-glucose + 1,2,3,6-tetrakis-O-galloyl-beta-D-glucose =D-glucose + 1,2,3,4,6-pentakis-O-galloyl-beta-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.144','Anthranilate N-benzoyltransferase','NULL','NULL','Cinnamoyl-CoA, 4-coumaroyl-CoA and salicyloyl-CoA can act as donors,more slowly.Involved in the biosynthesis of phytoalexins.','Benzoyl-CoA + anthranilate = CoA + N-benzoylanthranilate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.145','Piperidine N-piperoyltransferase','NULL','NULL','Pyrrolidine and 3-pyrroline can also act as acceptors, more slowly.','(E,E)-piperoyl-CoA + piperidine = CoA + N-((E,E)-piperoyl)-piperidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.146','Pinosylvin synthase','Stilbene synthase','NULL','Not identical with EC 2.3.1.74 or EC 2.3.1.95.','3 malonyl-CoA + cinnamoyl-CoA = 4 CoA + pinosylvin + 4 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.147','Glycerophospholipid arachidonoyl-transferase (CoA-independent)','NULL','NULL','Catalyzes the transfer of arachidonate and other polyenoic fattyacids from intact choline or ethanolamine-containingglycerophospholipids to the sn-2 position of a lyso-glycerophospholipid.The organyl group on sn-1 of the donor or acceptor molecule can bealkyl, acyl or alk-1-enyl.The term radyl has sometimes been used to refer to suchsubstituting groups.Differs from EC 2.3.1.148 in not requiring CoA and in its specificityfor poly-unsaturated acyl groups.','1-organyl-2-arachidonoyl-sn-glycero-3-phosphocholine + 1-organyl-2-lyso-sn-glycero-3-phosphoethanolamine = 1-organyl-2-arachidonoyl-sn-glycero-3-phosphoethanolamine + 1-organyl-2-lyso-sn-glycero-3-phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.148','Glycerophospholipid acyltransferase (CoA-dependent)','NULL','CoA.','Catalyzes the transfer of fatty acids from intact choline- orethanolamine-containing glycerophospholipids to the sn-2 position ofa lyso-glycerophospholipid.The organyl group on sn-1 of the donor or acceptor molecule can bealkyl, acyl or alk-1-enyl.The term radyl has sometimes been used to refer to suchsubstituting groups.Differs from EC 2.3.1.147 in requiring CoA and not favoring thetransfer of polyunsaturated acyl groups.','1-organyl-2-acyl-sn-glycero-3-phosphocholine + 1-organyl-2-lyso-sn-glycero-3-phosphoethanolamine = 1-organyl-2-acyl-sn-glycero-3-phosphoethanolamine + 1-organyl-2-lyso-sn-glycero-3-phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.149','Platelet-activating factor acetyltransferase','PAF acetyltransferase','NULL','Catalyzes the transfer of the acetyl group from 1-alkyl-2-acyl-sn-glycero-3-phosphocholine (platelet-activating factor) to the sn-2position of lyso-glycerophospholipids containing ethanolamine,choline, serine, inositol or phosphate groups at the sn-3 position aswell as to sphingosine and long-chain fatty alcohols.The organyl group can be alkyl, acyl or alk-1-enyl (sometimes alsocollectively referred to as radyl).','1-alkyl-2-acetyl-sn-glycero-3-phosphocholine + 1-organyl-2-lyso-sn-glycero-3-phospholipid = 1-organyl-2-lyso-sn-glycero-3-phosphocholine+ 1-alkyl-2-acetyl-sn-glycero-3-phospholipid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.150','Salutaridinol 7-O-acetyltransferase','NULL','NULL','At higher pH values the product, 7-O-acetylsalutaridinol,spontaneously closes the 4-','Acetyl-CoA + salutaridinol = CoA + 7-O-acetylsalutaridinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.151','2,3,4,6-tetrahydroxybenzophenone synthase','NULL','NULL','Involved in the biosynthesis of plant xanthones.Benzoyl-CoA can replace 3-hydroxybenzoyl-CoA (cf. EC 2.3.1.220).','3 malonyl-CoA + 3-hydroxybenzoyl-CoA = 4 CoA + 2,3,4,6-tetrahydroxybenzophenone + 3 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.152','Alcohol O-cinnamoyltransferase','NULL','NULL','Acceptor alcohols (ROH) include methanol, ethanol and propanol.No cofactors are required as 1-O-trans-cinnamoyl-beta-D-glucopyranoseitself is an energy-rich (activated) acyl-donor, comparable to CoA-thioesters.1-O-trans-cinnamoyl-beta-D-gentobiose can also act as the acyl donor,but with much less affinity.','1-O-trans-cinnamoyl-beta-D-glucopyranose + ROH = alkyl cinnamate +glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.153','Anthocyanin 5-(6-hydroxycinnamoyltransferase)','NULL','NULL','Isolated from the plant Gentiana triflora.Transfers the hydroxycinnamoyl group only to the C-5 glucoside ofanthocyanin.Caffeoyl-CoA, but not malonyl-CoA, can substitute as an acyl donor.','4-hydroxycinnamoyl-CoA + anthocyanidin 3,5-diglucoside = CoA +anthocyanidin 3-glucoside 5-(6-O-4-hydroxycinnamoylglucoside).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.154','Propionyl-CoA C(2)-trimethyltridecanoyltransferase','3-oxopristanoyl-CoA hydrolase','NULL','The peroxisomal protein sterol carrier protein X (SCPx) combines thisthiolase activity with its carrier function, and is involved inbranched chain fatty acid beta-oxidation in peroxisomes.It also acts on 3-oxopalmitoyl-CoA as a substrate but differs fromEC 2.3.1.16 which has little activity toward the 3-oxoacyl-CoA estersof 2-methyl-branched chain fatty acids such as 3-oxopristanoyl-CoA.','4,8,12-trimethyltridecanoyl-CoA + propanoyl-CoA = 3-oxopristanoyl-CoA +CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.155','Acetyl-CoA C-myristoyltransferase','3-oxopalmitoyl-CoA hydrolase','NULL','A peroxisomal enzyme involved in branched chain fatty acid beta-oxidation in peroxisomes.It differs from EC 2.3.1.154 in not being active toward3-oxopristanoyl-CoA.','Myristoyl-CoA + acetyl-CoA = 3-oxopalmitoyl-CoA + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.156','Phloroisovalerophenone synthase','3-methyl-1-(trihydroxyphenyl)butan-1-one synthase','NULL','Closely related to EC 2.3.1.74.The product, 3-methyl-1-(2,4,6-trihydroxyphenyl)butan-1-one,is phloroisovalerophenone.Also acts on isobutyryl-CoA as substrate to givephlorisobutyrophenone.The products are intermediates in the biosynthesis of the bitter(alpha) acids in hops (Humulus lupulus).','Isovaleryl-CoA + 3 malonyl-CoA = 4 CoA + 3 CO(2) + 3-methyl-1-(2,4,6-trihydroxyphenyl)butan-1-one.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.157','Glucosamine-1-phosphate N-acetyltransferase','NULL','NULL','The enzyme from several bacteria has been shown to be bifunctionaland also to possess the activity of EC 2.7.7.23.','Acetyl-CoA + alpha-D-glucosamine 1-phosphate = CoA + N-acetyl-alpha-D-glucosamine 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.158','Phospholipid:diacylglycerol acyltransferase','PDAT','NULL','This enzyme differs from EC 2.3.1.20 by synthesising triacylglycerolusing an acyl-CoA-independent mechanism.The specificity of the enzyme for the acyl group in the phospholipidvaries with species, e.g., the enzyme from castor bean (Ricinuscommunis) preferentially incorporates vernoloyl (12,13-epoxyoctadec-9-enoyl) groups into triacylglycerol, whereas that from the hawksbeard (Crepis palaestina) incorporates both ricinoleoyl(12-hydroxyoctadec-9-enoyl) and vernoloyl groups.The enzyme from the yeast Saccharomyces cerevisiae specificallytransfers acyl groups from the sn-2 position of the phospholipid todiacylglycerol, thus forming an sn-1-lysophospholipid.','Phospholipid + 1,2-diacylglycerol = lysophospholipid + triacylglycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.159','Acridone synthase','NULL','NULL','NULL','3 malonyl-CoA + N-methylanthraniloyl-CoA = 4 CoA + 1,3-dihydroxy-N-methylacridone + 3 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.160','Vinorine synthase','NULL','NULL','The reaction proceeds in two stages.The indole nitrogen of 16-epivellosimine interacts with its aldehydegroup giving an hydroxy-substituted new ring.This alcohol is then acetylated.Also acts on gardneral (11-methoxy-16-epivellosimine).Generates the ajmalan skeleton, which forms part of the route toajmaline.','Acetyl-CoA + 16-epivellosimine = CoA + vinorine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.161','Lovastatin nonaketide synthase','NULL','NULL','The microbial enzyme is a multi-functional protein catalyzing many ofthe chain building reactions of EC 2.3.1.85 as well as a reductivemethylation and a Diels-Alder reaction.','Acetyl-CoA + 8 malonyl-CoA + 11 NADPH + S-adenosyl-L-methionine =dihydromonacolin L + 9 CoA + 8 CO(2) + 11 NADP(+) + S-adenosyl-L-homocysteine + 6 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.162','Taxadien-5-alpha-ol O-acetyltransferase','Taxa-4(20),11(12)-dien-5-alpha-ol O-acetyltransferase','NULL','This is the third enzyme in the biosynthesis of the diterpenoidantineoplastic drug taxol (paclitaxel), which is widely used in thetreatment of carcinomas, sarcomas and melanomas.','Acetyl-CoA + taxa-4(20),11-dien-5-alpha-ol = CoA + taxa-4(20),11-dien-5-alpha-yl acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.163','10-hydroxytaxane O-acetyltransferase','Acetyl coenzyme A:10-hydroxytaxane O-acetyltransferase','NULL','Acts on a number of related taxane diterpenoids with a free 10-beta-hydroxy group.May be identical to EC 2.3.1.167.','Acetyl-CoA + 10-desacetyltaxuyunnanin C = CoA + taxuyunnanin C.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.164','Isopenicillin-N N-acyltransferase','Acyl-coenzyme A:6-aminopenicillanic-acid-acyltransferase','NULL','Proceeds by a two stage mechanism via 6-aminopenicillanic acid.Different from EC 3.5.1.11.','Phenylacetyl-CoA + isopenicillin N + H(2)O = CoA + penicillin G + L-2-aminohexanedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.165','6-methylsalicylic acid synthase','6-MSAS','NULL','A multienzyme complex with a 4-phosphopantetheine prosthetic groupon the acyl carrier protein.It has a similar sequence to vertebrate type I fatty acid synthase.Acetoacetyl-CoA can also act as a starter molecule.','Acetyl-CoA + 3 malonyl-CoA + NADPH = 6-methylsalicylate + 4 CoA + 3 CO(2)+ NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.166','2-alpha-hydroxytaxane 2-O-benzoyltransferase','2-debenzoyl-7,13-diacetylbaccatin III-2-O-benzoyl transferase','NULL','The enzyme was studied using the semisynthetic substrate 2-debenzoyl-7,13-diacetylbaccatin III.It will not acylate the hydroxy group at 1-beta, 7-beta, 10-beta or13-alpha of 10-deacetylbaccatin III, or at 2-alpha or 5-alpha oftaxa-4(20),11-diene-2-alpha,5-alpha-diol.','Benzoyl-CoA + 10-deacetyl-2-debenzoylbaccatin III = CoA +10-deacetylbaccatin III.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.167','10-deacetylbaccatin III 10-O-acetyltransferase','NULL','NULL','The enzyme will not acylate the hydroxy group at 1-beta, 7-beta,13-alpha of 10-deacetylbaccatin III, or at 5-alpha of taxa-4(20),11-dien-5-alpha-ol.May be identical to EC 2.3.1.163.','Acetyl-CoA + 10-deacetylbaccatin III = CoA + baccatin III.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.168','Dihydrolipoyllysine-residue (2-methylpropanoyl)transferase','Dihydrolipoyl transacylase','NULL','A multimer (24-mer) of this enzyme forms the core of the multienzyme3-methyl-2-oxobutanoate dehydrogenase complex, and binds tightly bothEC 1.2.4.4 and EC 1.8.1.4.The lipoyl group of this enzyme is reductively 2-methylpropanoylatedby EC 1.2.4.4, and the only observed direction catalyzed byEC 2.3.1.168 is that where this 2-methylpropanoyl is passed tocoenzyme A.In addition to the 2-methylpropanoyl group, formed when EC 1.2.4.4acts on the oxoacid that corresponds with valine, this enzyme alsotransfers the 3-methylbutanoyl and S-2-methylbutanoyl groups, donatedto it when EC 1.2.4.4 acts on the oxo acids corresponding withleucine and isoleucine.','2-methylpropanoyl-CoA + enzyme N(6)-(dihydrolipoyl)lysine = CoA + enzymeN(6)-(S-(2-methylpropanoyl)dihydrolipoyl)lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.169','CO-methylating acetyl-CoA synthase','ACS','Iron-sulfur; Nickel.','Also catalyzes exchange reactions of carbon between C-1 of acetyl-CoAand CO, and between C-2 of acetyl-CoA and methyl corrinoid protein.Involved, together with EC 1.2.7.4, in the synthesis of acetyl-CoAfrom CO(2) and H(2).To follow its stoichiometry, the reaction can be written as: CH(3)-CO-S-CoA + protein Co(+) + H(+) = CO + protein Co(2+)-CH(3) + HS-CoA.','Acetyl-CoA + corrinoid protein = CO + methylcorrinoid protein + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.170','6-deoxychalcone synthase','NULL','NULL','Isoliquiritigenin is the precursor of liquiritigenin,a 5-deoxyflavanone.','3 malonyl-CoA + 4-coumaroyl-CoA + NADPH = 4 CoA + isoliquiritigenin +3 CO(2) + NADP(+) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.171','Anthocyanin 6-O-malonyltransferase','3MaT','NULL','Acts on pelargonidin 3-O-glucoside in dahlia (Dahlia variabilis),delphinidin 3-O-glucoside, and on cyanidin 3-O-glucoside intransgenic petunia (Petunia hybrida).','Malonyl-CoA + an anthocyanidin 3-O-beta-D-glucoside = CoA +an anthocyanidin 3-O-(6-O-malonyl-beta-D-glucoside).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.172','Anthocyanin 5-O-glucoside 6-O-malonyltransferase','Ss5MaT1','NULL','Specific for the penultimate step in salvianin biosynthesis.The enzyme also catalyzes the malonylation of shisonin tomalonylshisonin (cyanidin 3-O-(6-O-p-coumaryl-beta-D-glucoside)-5-(6-O-malonyl-beta-D-glucoside)).The compounds 4-demalonylsalvianin, salvianin, pelargonidin 3,5-diglucoside and delphinidin 3,5-diglucoside cannot act as substrates.','Malonyl-CoA + pelargonidin 3-O-(6-caffeoyl-beta-D-glucoside) 5-O-beta-D-glucoside = CoA + 4-demalonylsalvianin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.173','Flavonol-3-O-triglucoside O-coumaroyltransferase','NULL','NULL','Acylates kaempferol 3-O-triglucoside on the terminal glucosyl unit,almost certainly at C-6.','4-coumaroyl-CoA + a flavonol 3-O-(beta-D-glucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.174','3-oxoadipyl-CoA thiolase','Beta-ketoadipyl-CoA thiolase','NULL','The enzyme from the bacterium Escherichia coli also has the activityof EC 2.3.1.223.','Succinyl-CoA + acetyl-CoA = CoA + 3-oxoadipyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.175','Deacetylcephalosporin-C acetyltransferase','Acetyl coenzyme A:DAC acetyltransferase','NULL','This enzyme catalyzes the final step in the biosynthesis ofcephalosporin C.','Acetyl-CoA + deacetylcephalosporin C = CoA + cephalosporin C.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.176','Propanoyl-CoA C-acyltransferase','Peroxisomal thiolase 2','NULL','Also acts on dihydroxy-5-beta-cholestanoyl-CoA and other branchedchain acyl-CoA derivatives.Catalyzes the penultimate step in the formation of bile acids.The bile acid moiety is transferred from the acyl-CoA thioester (RCO-SCoA) to either glycine or taurine (NH(2)R) by EC 2.3.1.65.','3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholanoyl-CoA + propanoyl-CoA= CoA + 3-alpha,7-alpha,12-alpha-trihydroxy-24-oxo-5-beta-cholestanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.177','3,5-dihydroxybiphenyl synthase','Biphenyl synthase','NULL','A polyketide synthase that is involved in the production of thephytoalexin aucuparin.2-hydroxybenzoyl-CoA can also act as substrate but it leads to thederailment product 4-hydroxycoumarin (cf. EC 2.3.1.208).This enzyme uses the same starter substrate as EC 2.3.1.151.','3 malonyl-CoA + benzoyl-CoA = 4 CoA + 3,5-dihydroxybiphenyl + 4 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.178','Diaminobutyrate acetyltransferase','2,4-diaminobutanoate acetyltransferase','Sodium or potassium.','Ornithine, lysine, aspartate, and alpha-, beta- and gamma-aminobutanoate cannot act as substrates.However, acetyl-CoA can be replaced by propanoyl-CoA, although thereaction proceeds more slowly.Forms part of the ectoine biosynthesis pathway, the other enzymesinvolved being EC 2.6.1.76 and EC 4.2.1.108.','Acetyl-CoA + L-2,4-diaminobutanoate = CoA + N(4)-acetyl-L-2,4-diaminobutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.179','Beta-ketoacyl-[acyl-carrier-protein] synthase II','3-oxoacyl-acyl carrier protein synthase I','NULL','Involved in the dissociated (or type II) fatty acid biosynthesissystem that occurs in plants and bacteria.While the substrate specificity of this enzyme is very similar tothat of EC 2.3.1.41, it differs in that palmitoleoyl-ACP is not agood substrate of EC 2.3.1.41 but is an excellent substrate of thisenzyme.The fatty-acid composition of Escherichia coli changes as a functionof growth temperature, with the proportion of unsaturated fatty acidsincreasing with lower growth temperature.Controls the temperature-dependent regulation of fatty-acidcomposition, with mutants lacking this acivity being deficient in theelongation of palmitoleate to cis-vaccenate at low temperatures.','(Z)-hexadec-11-enoyl-[acyl-carrier-protein] + malonyl-[acyl-carrier-protein] = (Z)-3-oxooctadec-13-enoyl-[acyl-carrier-protein] + CO(2) +[acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.180','Beta-ketoacyl-[acyl-carrier-protein] synthase III','3-ketoacyl-acyl carrier protein synthase III','NULL','Involved in the dissociated (or type II) fatty-acid biosynthesissystem that occurs in plants and bacteria.In contrast to EC 2.3.1.41 and EC 2.3.1.179, this enzyme specificallyuses CoA thioesters rather than acyl-ACP as the primer.In addition to the above reaction, the enzyme can also catalyze thereaction of EC 2.3.1.38, but to a much lesser extent.Responsible for initiating both straight- and branched-chain fatty-acid biosynthesis, with the substrate specificity in an organismreflecting the fatty-acid composition found in that organism.For example, Streptococcus pneumoniae, a Gram-positive bacterium,is able to use both straight- and branched-chain (C4--C6) acyl-CoAprimers whereas Escherichia coli, a Gram-negative organism, usesprimarily short straight-chain acyl CoAs, with a preference foracetyl-CoA.','Acetyl-CoA + malonyl-[acyl-carrier-protein] = acetoacetyl-[acyl-carrier-protein] + CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.181','Lipoyl(octanoyl) transferase','Lipoate/octanoate transferase','NULL','The first committed step in the biosynthesis of lipoyl cofactor.The lipoyl cofactor is essential for the function of several keyenzymes involved in oxidative metabolism, as it converts apoproteininto the biologically active holoprotein.Examples of such lipoylated proteins include pyruvate dehydrogenase(E(2) domain), 2-oxoglutarate dehydrogenase (E(2) domain),the branched-chain 2-oxoacid dehydrogenases and the glycine cleavagesystem (H protein).Lipoyl-ACP can also act as a substrate although octanoyl-ACP islikely to be the true substrate.The other enzyme involved in the biosynthesis of lipoyl cofactor isEC 2.8.1.8.An alternative lipoylation pathway involves EC 2.7.7.63, which canlipoylate apoproteins using exogenous lipoic acid (or its analogs).','Octanoyl-[acyl-carrier-protein] + protein = protein N(6)-(octanoyl)lysine+ [acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.182','(R)-citramalate synthase','NULL','NULL','Involved in a novel pyruvate pathway for isoleucine biosynthesis thatis found mainly in archaea.Can be inhibited by isoleucine, the end-product of the pathway,but not by leucine.Highly specific for pyruvate as substrate, as the 2-oxo acids3-methyl-2-oxobutanoate, 2-oxobutanoate, 4-methyl-2-oxopentanoate,2-oxohexanoate and 2-oxoglutarate cannot act as substrate.','Acetyl-CoA + pyruvate + H(2)O = CoA + (2R)-2-hydroxy-2-methylbutanedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.183','Phosphinothricin acetyltransferase','PPT acetyltransferase','NULL','The substrate phosphinothricin is used as a nonselective herbicideand is a potent inhibitor of EC 6.3.1.2, a key enzyme of nitrogenmetabolism in plants.','Acetyl-CoA + phosphinothricin = CoA + N-acetylphosphinothricin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.184','Acyl-homoserine-lactone synthase','Acyl homoserine lactone synthase','NULL','Acyl-homoserine lactones (AHLs) are produced by a number of bacterialspecies and are used by them to regulate the expression of virulencegenes in a process known as quorum-sensing.Each bacterial cell has a basal level of AHL and, once the populationdensity reaches a critical level, it triggers AHL-signaling which,in turn, initiates the expression of particular virulence genes.N-(3-oxohexanoyl)-[acyl-carrier-protein] and hexanoyl-[acyl-carrier-protein] are the best substrates.The fatty-acyl substrate is derived from fatty-acid biosynthesisthrough acyl-[acyl-carrier-protein] rather than from fatty-aciddegradation through acyl-CoA.S-adenosyl-L-methionine cannot be replaced by methionine,S-adenosylhomocysteine, homoserine or homoserine lactone.','An acyl-[acyl-carrier-protein] + S-adenosyl-L-methionine = [acyl-carrier-protein] + S-methyl-5-thioadenosine + an N-acyl-L-homoserine lactone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.185','Tropine acyltransferase','TAT','NULL','Exhibits absolute specificity for the endo/3-alpha configurationfound in tropine as pseudotropine (tropan-3-beta-ol; seeEC 2.3.1.186) is not a substrate.Acts on a wide range of aliphatic acyl-CoA derivatives, with tigloyl-CoA and acetyl-CoA being the best substrates.It is probably involved in the formation of the tropane alkaloidlittorine, which is a precursor of hyoscyamine.','An acyl-CoA + tropine = CoA + an O-acyltropine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.186','Pseudotropine acyltransferase','PAT','NULL','Exhibits absolute specificity for the exo/3-beta configuration foundin pseudotropine as tropine (tropan-3-alpha-ol; see EC 2.3.1.185) andnortropine are not substrates.Acts on a wide range of aliphatic acyl-CoA derivatives, includingacetyl-CoA, beta-methylcrotonyl-CoA and tigloyl-CoA.','An acyl-CoA + pseudotropine = CoA + an O-acylpseudotropine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.187','Acetyl-S-ACP:malonate ACP transferase','ACP transferase','NULL','This is the first step in the catalysis of malonate decarboxylationand involves the exchange of an acetyl thioester residue bound to theactivated acyl-carrier protein (ACP) subunit of the malonatedecarboxylase complex for a malonyl thioester residue.This enzyme forms the alpha subunit of the multienzyme complexesEC 4.1.1.88 and EC 4.1.1.89.Can also use acetyl-CoA as a substrate but more slowly.','An acetyl-[acyl-carrier protein] + malonate = a malonyl-[acyl-carrierprotein] + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.188','Omega-hydroxypalmitate O-feruloyl transferase','HHT','NULL','p-coumaroyl-CoA and sinapoyl-CoA also act as substrates.The enzyme is widely distributed in roots of higher plants.','Feruloyl-CoA + 16-hydroxypalmitate = CoA + 16-feruloyloxypalmitate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.189','Mycothiol synthase','NULL','NULL','This enzyme catalyzes the last step in the biosynthesis of mycothiol,the major thiol in most actinomycetes, including Mycobacterium.The enzyme is a member of a large family of GCN5-relatedN-acetyltransferases (GNATs).The enzyme has been purified from Mycobacterium tuberculosis H37Rv.Acetyl-CoA is the preferred CoA thioester but propionyl-CoA is also asubstrate.','Desacetylmycothiol + acetyl-CoA = CoA + mycothiol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.190','Acetoin dehydrogenase','NULL','Thiamine phosphate.','This enzyme, which belongs to the family of 2-oxo acid dehydrogenasecomplexes, catalyzes the oxidative-hydrolytic cleavage of acetoin toacetaldehyde and acetyl-CoA in many bacterial strains, both aerobicand anaerobic.The enzyme is composed of multiple copies of three enzymaticcomponents:acetoin oxidoreductase (E1), dihydrolipoamideacetyltransferase (E2) and dihydrolipoyl dehydrogenase (E3).','Acetoin + CoA + NAD(+) = acetaldehyde + acetyl-CoA + NADH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.191','UDP-3-O-(3-hydroxymyristoyl)glucosamine N-acyltransferase','UDP-3-O-(3-hydroxymyristoyl)-D-glucosamine N-acyltransferase','NULL','The enzyme catalyzes a step of lipid A biosynthesis.LpxD from Escherichia coli prefers (R,S)-3-hydroxymyristoyl-[acyl-carrier protein] over (R,S)-3-hydroxypalmitoyl-[acyl-carrierprotein].E.coli lipid A acyltransferases do not have an absolute specificityfor 14-carbon hydroxy fatty acids but can transfer fatty acidsdiffering by one carbon unit if the fatty acid substrates areavailable.When grown on 1% propionic acid, lipid A also contains the odd-chainfatty acids tridecanoic acid, pentadecanoic acid, hydroxytridecanoicacid, and hydroxypentadecanoic acid.Formerly EC 2.3.1.n1.','(3R)-3-hydroxymyristoyl-[acyl-carrier-protein] + UDP-3-O-((3R)-hydroxymyristoyl)-alpha-D-glucosamine = UDP-2,3-bis(O-(3R)-3-hydroxymyristoyl)-alpha-D-glucosamine + a holo-[acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.192','Glycine N-phenylacetyltransferase','Arylacetyl-CoA N-acyltransferase','NULL','Not identical with EC 2.3.1.13.This enzyme was purified from bovine liver mitochondria.L-asparagine, L-glutamine and L-arginine are alternative substratesto glycine, but have higher Km values.','Phenylacetyl-CoA + glycine = phenylacetylglycine + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.193','tRNA(Met) cytidine acetyltransferase','NULL','NULL','The enzyme acetylates the wobble base C34 of the CAU anticodon ofelongation-specific tRNA(Met).Escherichia coli TmcA strictly discriminates elongator tRNA(Met) fromtRNA(Ile), which is structurally similar and has the same anticodonloop, mainly by recognizing the C27-G43 pair in the anticodon stem.The enzyme can use GTP in place of ATP for formation of N(4)-acetylcytidine.','(Elongator tRNA(Met))-cytidine(34) + ATP + acetyl-CoA = (elongatortRNA(Met))-N(4)-acetylcytidine(34) + ADP + phosphate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.194','Acetoacetyl-CoA synthase','NphT7','NULL','The enzyme from the soil bacterium Streptomyces sp. CL190 producesacetoacetyl-CoA to be used for mevalonate production via themevalonate pathway.Unlike the homologous EC 2.3.1.180 this enzyme does not acceptmalonyl-[acyl-carrier-protein] as a substrate.','Acetyl-CoA + malonyl-CoA = acetoacetyl-CoA + CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.195','(Z)-3-hexen-1-ol acetyltransferase','NULL','NULL','The enzyme is resonsible for the production of (3Z)-hex-3-en-1-ylacetate, the major volatile compound released upon mechanicalwounding of the leaves of Arabidopsis thaliana.','Acetyl-CoA + (3Z)-hex-3-en-ol = (3Z)-hex-3-en-1-yl acetate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.196','Benzyl alcohol O-benzoyltransferase','NULL','NULL','The enzyme is involved in volatile benzenoid and benzoic acidbiosynthesis.The enzyme from Petunia hybrida also catalyzes the formation of2-phenylethyl benzoate from benzoyl-CoA and 2-phenylethanol.The apparent catalytic efficiency of the enzyme from Petunia hybridawith benzoyl-CoA is almost 6-fold higher than with acetyl-CoA.','Benzoyl-CoA + benzyl alcohol = benzyl benzoate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.197','dTDP-3-amino-3,6-dideoxy-alpha-D-galactopyranose 3-N-acetyltransferase','dTDP-D-Fucp3N acetylase','NULL','The product, dTDP-3-acetamido-3,6-dideoxy-alpha-D-galactose, is acomponent of the glycan chain of the of the crystalline bacterialcell surface layer protein (S-layer glycoprotein) of Aneurinibacillusthermoaerophilus.','Acetyl-CoA + dTDP-3-amino-3,6-dideoxy-alpha-D-galactopyranose = CoA +dTDP-3-acetamido-3,6-dideoxy-alpha-D-galactopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.198','Glycerol-3-phosphate 2-O-acyltransferase','sn-2-glycerol-3-phosphate O-acyltransferase','NULL','A membrane-associated enzyme required for suberin or cutin synthesisin plants.Active with a wide range of acyl-CoA substrates (C16:0-C24:0).The enzyme from some sources has much higher activity with omega-oxidized acyl-CoAs.Some enzymes are bifunctional and have an additional phosphataseactivity producing sn-2-monoacylglycerols.','An acyl-CoA + sn-glycerol 3-phosphate = CoA + a 2-acyl-sn-glycerol3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.199','Very-long-chain 3-oxoacyl-CoA synthase','Condensing enzyme','NULL','This is the first component of the elongase, a microsomal proteincomplex responsible for extending palmitoyl-CoA and stearoyl-CoA (andmodified forms thereof) to very-long-chain acyl CoAs.Multiple forms exist with differing preferences for the substrate,and thus the specific form expressed determines the local compositionof very-long-chain fatty acids.For example, the FAE1 form from the plant Arabidopsis thalianaaccepts only 16 and 18 carbon substrates, with oleoyl-CoA (18:1)being the preferred substrate, while CER6 from the same plant preferssubstrates with chain length of C(22) to C(32).Cf. EC 1.1.1.330, EC 4.2.1.134 and EC 1.3.1.93.Formerly EC 2.3.1.n8.','A very-long-chain acyl-CoA + malonyl-CoA = CoA + a very-long-chain3-oxoacyl-CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.200','Lipoyl amidotransferase','NULL','NULL','In the bacterium Listeria monocytogenes the enzyme takes part in apathway for scavenging of lipoic acid.The enzyme is bound to 2-oxo-acid dehydrogenases such as the pyruvatedehydrogenase complex, where it transfers the lipoyl moiety fromlipoyl-[glycine cleavage system H] to the E2 subunits of thecomplexes.','[Glycine cleavage system H]-N(6)-lipoyl-L-lysine + a [lipoyl-carrierprotein] = [glycine cleavage system H] + a [lipoyl-carrier protein]-N(6)-lipoyl-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.201','UDP-2-acetamido-3-amino-2,3-dideoxy-glucuronate N-acetyltransferase','NULL','NULL','This enzyme participates in the biosynthetic pathway for UDP-alpha-D-ManNAc3NAcA (UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-mannuronicacid), an important precursor of B-band lipopolysaccharide.','Acetyl-CoA + UDP-2-acetamido-3-amino-2,3-dideoxy-alpha-D-glucuronate =CoA + UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.202','UDP-4-amino-4,6-dideoxy-N-acetyl-beta-L-altrosamine N-acetyltransferase','NULL','NULL','Isolated from Helicobacter pylori.The enzyme is involved in the biosynthesis of pseudaminic acid.','Acetyl-CoA + UDP-4-amino-4,6-dideoxy-N-acetyl-beta-L-altrosamine = CoA +UDP-2,4-diacetamido-2,4,6-trideoxy-beta-L-altropyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.203','UDP-N-acetylbacillosamine N-acetyltransferase','UDP-4-amino-4,6-dideoxy-N-acetyl-alpha-D-glucosamine N-acetyltransferase','NULL','The product, UDP-N,N-diacetylbacillosamine, is an intermediate inprotein glycosylation pathways in several bacterial species,including N-linked glycosylation of certain L-asparagine residues inCampylobacter species and O-linked glycosylation of certain L-serineresidues in Neisseria species.','Acetyl-CoA + UDP-N-acetylbacillosamine = CoA + UDP-N,N-diacetylbacillosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.204','Octanoyl-[GcvH]:protein N-octanoyltransferase','NULL','NULL','In the bacterium Bacillus subtilis it has been shown that the enzymecatalyzes the amidotransfer of the octanoyl moiety from [glycinecleavage system H]-N(6)-octanoyl-L-lysine (i.e. octanoyl-GcvH) to theE2 subunit (dihydrolipoamide acetyltransferase) of pyruvatedehydrogenase.','[Glycine cleavage system H]-N(6)-octanoyl-L-lysine + a [lipoyl-carrierprotein] = glycine cleavage system H + a [lipoyl-carrier protein]-N(6)-octanoyl-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.205','Fumigaclavine B O-acetyltransferase','NULL','NULL','The enzyme participates in the biosynthesis of fumigaclavine C,an ergot alkaloid produced by some fungi of the Trichocomaceaefamily.','Acetyl-CoA + fumigaclavine B = CoA + fumigaclavine A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.206','3,5,7-trioxododecanoyl-CoA synthase','Olivetol synthase','NULL','A polyketide synthase catalyzing the first committed step in thecannabinoids biosynthetic pathway of the plant Cannabis sativa.The enzyme was previously thought to also function as a cyclase,but the cyclization is now known to be catalyzed by EC 4.4.1.26.','3 malonyl-CoA + hexanoyl-CoA = 3 CoA + 3,5,7-trioxododecanoyl-CoA +3 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.207','Beta-ketodecanoyl-[acyl-carrier-protein] synthase','NULL','NULL','This enzyme, which has been characterized from the bacteriumPseudomonas aeruginosa PAO1, catalyzes the condensation of octanoyl-CoA, obtained from exogenously supplied fatty acids via beta-oxidation, with malonyl-[acp], forming 3-oxodecanoyl-[acp],an intermediate of the fatty acid elongation cycle.The enzyme provides a shunt for beta-oxidation degradationintermediates into de novo fatty acid biosynthesis.','Octanoyl-CoA + a malonyl-[acyl-carrier protein] = a 3-oxodecanoyl-[acyl-carrier protein] + CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.208','4-hydroxycoumarin synthase','NULL','NULL','The enzyme, a polyketide synthase, can also accept benzoyl-CoA assubstrate, which it condenses with 3 malonyl-CoA molecules to form3,5-dihydroxybiphenyl (cf. EC 2.3.1.177).','Malonyl-CoA + 2-hydroxybenzoyl-CoA = 2 CoA + 4-hydroxycoumarin + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.209','dTDP-4-amino-4,6-dideoxy-D-glucose acyltransferase','NULL','NULL','The non-activated product, 4-acetamido-4,6-dideoxy-alpha-D-glucose,is part of the O antigens of Shigella dysenteriae type 7 andEscherichia coli O7.','Acetyl-CoA + dTDP-4-amino-4,6-dideoxy-alpha-D-glucose = CoA + dTDP-4-acetamido-4,6-dideoxy-alpha-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.210','dTDP-4-amino-4,6-dideoxy-D-galactose acyltransferase','TDP-fucosamine acetyltransferase','NULL','The product, TDP-4-acetamido-4,6-dideoxy-D-galactose, is utilized inthe biosynthesis of Enterobacterial Common Antigen (ECA).','Acetyl-CoA + dTDP-4-amino-4,6-dideoxy-alpha-D-galactose = CoA + dTDP-4-acetamido-4,6-dideoxy-alpha-D-galactose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.211','Bisdemethoxycurcumin synthase','Curcuminoid synthase','NULL','A polyketide synthase characterized from the plant Oryza sativa(rice) that catalyzes the formation of the C(6)-C(7)-C(6)diarylheptanoid scaffold of bisdemethoxycurcumin.Unlike the process in the plant Curcuma longa (turmeric), where theconversion is carried out via a diketide intermediate by twodifferent enzymes, the diketide intermediate formed by this enzymeremains within the enzymes cavity and is not released to theenvironment.Formerly EC 2.3.1.n9.','2 4-coumaroyl-CoA + malonyl-CoA + H(2)O = 3 CoA + bisdemethoxycurcumin +2 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.212','Benzalacetone synthase','NULL','NULL','A polyketide synthase that catalyzes the C(6)-C(4) skeleton ofphenylbutanoids in higher plants.','4-coumaroyl-CoA + malonyl-CoA + H(2)O = 2 CoA + 4-hydroxybenzalacetone +2 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.213','Cyanidin 3-O-(6-O-glucosyl-2-O-xylosylgalactoside) 6-O-hydroxycinnamoyltransferase','NULL','NULL','Isolated from the plant Daucus carota (Afghan cultivar carrot).In addition to 1-O-(4-hydroxycinnamoyl)-beta-D-glucose, the enzymecan use the 1-O-sinapoyl- and 1-O-feruloyl- derivatives of beta-D-glucose.','1-O-(4-hydroxycinnamoyl)-beta-D-glucose + cyanidin 3-O-(6-O-beta-D-glucosyl-2-O-beta-D-xylosyl-beta-D-galactoside) = beta-D-glucose +cyanidin 3-O-(6-O-(6-O-4-hydroxycinnamoyl-beta-D-glucosyl)-2-O-beta-D-xylosyl-beta-D-galactoside).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.214','Pelargonidin 3-O-(6-caffeoylglucoside) 5-O-(6-O-malonylglucoside)4-malonyltransferase','NULL','NULL','Isolated from the plant Salvia splendens (scarlet sage).','Malonyl-CoA + 4-demalonylsalvianin = CoA + salvianin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.215','Anthocyanidin 3-O-glucoside 6-O-acyltransferase','NULL','NULL','Isolated from the plants Perilla frutescens and Gentiana triflora(clustered gentian).Acts on a range of anthocyanidin 3-O-glucosides, 3,5-di-O-glucosidesand cyanidin 3-rutinoside.It did not act on delphinidin 3,3,7-tri-O-glucoside.Recombinant Perilla frutescens enzyme could utilize caffeoyl-CoA butnot malonyl-CoA as alternative acyl donor.','4-hydroxycinnamoyl-CoA + an anthocyanidin 3-O-beta-D-glucoside = CoA +an anthocyanidin 3-O-(6-O-(4-hydroxycinnamoyl)-beta-D-glucoside).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.216','5,7-dihydroxy-2-methylchromone synthase','Pentaketide chromone synthase','NULL','A polyketide synthase from the plant Aloe arborescens (aloe).','5 malonyl-CoA = 5 CoA + 5,7-dihydroxy-2-methyl-4H-chromen-4-one + 5 CO(2)+ H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.217','Curcumin synthase','NULL','NULL','A polyketide synthase from the plant Curcuma longa (turmeric).Three isoforms exist, CURS1, CURS2 and CURS3.While CURS1 and CURS2 prefer feruloyl-CoA as a starter substrate,CURS3 can accept 4-coumaroyl-CoA equally well (see EC 2.3.1.219).','Feruloyl-CoA + feruloylacetyl-CoA + H(2)O = 2 CoA + curcumin + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.218','Phenylpropanoylacetyl-CoA synthase','Phenylpropanoyl-diketide-CoA synthase','NULL','The enzyme has been characterized from the plant Curcuma longa(turmeric).It prefers feruloyl-CoA, and has no activity with cinnamoyl-CoA.','(1) Feruloyl-CoA + malonyl-CoA = feruloylacetyl-CoA + CO(2) + CoA.(2) 4-coumaroyl-CoA + malonyl-CoA = (4-coumaroyl)acetyl-CoA + CO(2) +CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.219','Demethoxycurcumin synthase','NULL','NULL','A polyketide synthase from the plant Curcuma longa (turmeric).Three isoforms exist, CURS1, CURS2 and CURS3.While CURS1 and CURS2 prefer feruloyl-CoA as a starter substrate (cf.EC 2.3.1.217), CURS3 can accept 4-coumaroyl-CoA equally well.','(1) 4-coumaroyl-CoA + feruloylacetyl-CoA + H(2)O = 2 CoA +demethoxycurcumin + CO(2).(2) 4-coumaroyl-CoA + (4-coumaroyl)acetyl-CoA + H(2)O = 2 CoA +bisdemethoxycurcumin + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.220','2,4,6-trihydroxybenzophenone synthase','Benzophenone synthase','NULL','Involved in the biosynthesis of plant xanthones.The enzyme from the plant Hypericum androsaemum L can use3-hydroxybenzoyl-CoA instead of benzoyl-CoA, but with lower activity(cf. EC 2.3.1.151).','3 malonyl-CoA + benzoyl-CoA = 4 CoA + 2,4,6-trihydroxybenzophenone +3 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.221','Noranthrone synthase','Norsolorinic acid anthrone synthase','NULL','A multi-domain polyketide synthase involved in the synthesis ofaflatoxins in the fungus Aspergillus parasiticus.The hexanoyl starter unit is provided to the acyl-carrier protein(ACP) domain by a dedicated fungal fatty acid synthase.','7 malonyl-CoA + hexanoyl-[acyl-carrier protein] = 7 CoA + norsolorinateanthrone + [acyl-carrier protein] + 7 CO(2) + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.222','Phosphate propanoyltransferase','NULL','NULL','Part of the degradation pathway for 1,2-propanediol.Formerly EC 2.3.1.n10.','Propanoyl-CoA + phosphate = CoA + propanoyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.223','3-oxo-5,6-didehydrosuberyl-CoA thiolase','NULL','NULL','The enzyme acts in the opposite direction.The enzymes from the bacteria Escherichia coli and Pseudomonas sp. Y2also have the activity of EC 2.3.1.174.Formerly EC 2.3.1.n11.','2,3-didehydroadipyl-CoA + acetyl-CoA = CoA + 3-oxo-5,6-didehydrosuberyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.224','Acetyl-CoA-benzylalcohol acetyltransferase','NULL','NULL','The enzyme is found in flowers like Clarkia breweri, where it isimportant for floral scent production.Unlike EC 2.3.1.84 this enzyme is active with alcohols that contain abenzyl ring.','(1) Acetyl-CoA + benzyl alcohol = CoA + benzyl acetate.(2) Acetyl-CoA + cinnamyl alcohol = CoA + cinnamyl acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.225','Protein S-acyltransferase','DHHC palmitoyl transferase','NULL','The enzyme catalyzes the posttranslational protein palmitoylationthat plays a role in protein-membrane interactions, proteintrafficking, and enzyme activity.Palmitoylation increases the hydrophobicity of proteins or proteindomains and contributes to their membrane association.','Palmitoyl-CoA + [protein]-L-cysteine = [protein]-S-palmitoyl-L-cysteine +CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.226','Carboxymethylproline synthase','NULL','NULL','The enzyme is involved in the biosynthesis of the carbapenem beta-lactam antibiotic (5R)-carbapen-2-em-3-carboxylate in the bacteriumPectobacterium carotovorum.','Malonyl-CoA + (S)-1-pyrroline-5-carboxylate + H(2)O = CoA + (2S,5S)-5-carboxymethylproline + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.227','GDP-perosamine N-acetyltransferase','GDP-alpha-D-perosamine N-acetyltransferase','NULL','D-perosamine is one of several dideoxy sugars found in the O-antigencomponent of the outer membrane lipopolysaccharides of Gram-negativebacteria.','Acetyl-CoA + GDP-4-amino-4,6-dideoxy-alpha-D-mannose = CoA + GDP-4-acetamido-4,6-dideoxy-alpha-D-mannose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.228','Isovaleryl-homoserine lactone synthase','IV-HSL synthase','NULL','The enzyme, found in the bacterium Bradyrhizobium japonicum, does notaccept isovaleryl-[acyl-carrier protein] as acyl donor (cf.EC 2.3.1.184).','Isovaleryl-CoA + S-adenosyl-L-methionine = CoA + S-methyl-5-thioadenosine + N-isovaleryl-L-homoserine lactone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.229','4-coumaroyl-homoserine lactone synthase','p-coumaryl-homoserine lactone synthase','NULL','The enzyme is found in the bacterium Rhodopseudomonas palustris,which produces N-(4-coumaroyl)-L-homoserine lactone as a quorum-sensing signal.','4-coumaroyl-CoA + S-adenosyl-L-methionine = CoA + S-methyl-5-thioadenosine + N-(4-coumaroyl)-L-homoserine lactone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.230','2-heptyl-4(1H)-quinolone synthase','NULL','NULL','The enzyme from the Gram-negative bacterium Pseudomonas aeruginosa isinvolved in the biosynthesis of 2-heptyl-4-hydroxyquinoline and 2,4-dihydroxyquinoline.2-heptyl-4-hydroxyquinoline is a signal molecule, that is involved inregulation of virulence factor production and biofilm formation.The enzyme shows a broad specificity and is involved in the synthesisof a wide array of additional 2-alkyl-4(1H)-quinolones synthesized bythe organism.','(1) 3-oxodecanoate + anthraniloyl-CoA = CoA + 2-heptyl-4(1H)-quinolone +CO(2) + H(2)O.(2) Malonyl-CoA + anthraniloyl-CoA = 2 CoA + 4-hydroxy-2(1H)-quinolone +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.231','tRNA(Phe) (7-(3-amino-3-(methoxycarbonyl)propyl)wyosine(37)-N)-methoxycarbonyltransferase','tRNA-yW synthesizing enzyme-4','NULL','The enzyme is found only in eukaryotes, where it is involved in thebiosynthesis of wybutosine, a hypermodified tricyclic base found atposition 37 of certain tRNAs.The modification is important for translational reading-framemaintenance.In some species that produce hydroxywybutosine the enzyme uses7-(2-hydroxy-3-amino-3-(methoxycarbonyl)propyl)wyosine(37) intRNA(Phe) as substrate.The enzyme also has the activity of EC 2.1.1.290.','S-adenosyl-L-methionine + 7-((3S)-3-amino-3-(methoxycarbonyl)propyl)wyosine(37) in tRNA(Phe) + CO(2) = S-adenosyl-L-homocysteine + wybutosine(37) in tRNA(Phe).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.232','Methanol O-anthraniloyltransferase','NULL','NULL','The enzyme from Concord grape (Vitis labrusca) is solely responsiblefor the production of O-methyl anthranilate, an important aroma andflavor compound in the grape.The enzyme has a broad substrate specificity, and can use a range ofalcohols with substantial activity, the best being butanol, benzylalcohol, iso-pentanol, octanol and 2-propanol.It can use benzoyl-CoA and acetyl-CoA as acyl donors with lowerefficiency.In addition to O-methyl anthranilate, the enzyme might be responsiblefor the production of ethyl butanoate, methyl-3-hydroxy butanoate andethyl-3-hydroxy butanoate, which are present in large quantities inthe grapes.Also catalyzes EC 2.3.1.196.','Anthraniloyl-CoA + methanol = CoA + O-methyl anthranilate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n2','Phosphate acyltransferase','Acyl-[acyl-carrier-protein]--phosphate acyltransferase','NULL','This enzyme can utilize acyl-ACP as fatty acyl donor, but not acyl-CoA.','Acyl-[acyl-carrier-protein] + phosphate = acyl-phosphate + [acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n3','Glycerol-3-phosphate acyltransferase (acyl-phosphate transferring)','Acyl-phosphate--glycerol-3-phosphate acyltransferase','NULL','This enzyme can use acyl-phosphate as fatty acyl donor, but not acyl-CoA or acyl-ACP.','Acyl-phosphate + sn-glycerol 3-phosphate = 1-acyl-sn-glycerol 3-phosphate+ phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n4','1-acyl-sn-glycerol-3-phosphate acyltransferase','NULL','NULL','NULL','Acyl-[acyl-carrier-protein] + 1-acyl-sn-glycerol 3-phosphate = [acyl-carrier-protein] + 1,2-diacyl-sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n5','Glycerol-3-phosphate acyltransferase (acyl-[acyl-carrier-protein]-transferring)','NULL','NULL','NULL','Acyl-[acyl-carrier-protein] + sn-glycerol 3-phosphate = [acyl-carrier-protein] + 1-acyl-sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n6','1-acylglycerophosphoserine O-acyltransferase','Lysophosphatidylserine acyltransferase','NULL','Acyltransferase which mediates the conversion oflysophosphatidylserine into phosphatidylserine.The enzyme also catalyzes the acylation oflysophosphatidylethanolamine into phosphatidylethanolamine (cf.EC 2.3.1.n7).Prefers oleoyl-CoA as the acyl donor.','Acyl-CoA + 1-acyl-sn-glycero-3-phosphatidylserine = CoA + 1,2-diacyl-sn-glycero-3-phosphatidylserine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n7','1-acylglycerophosphoethanolamine O-acyltransferase','Lysophosphatidylethanolamine acyltransferase','NULL','The enzyme mediates the conversion of lysophosphatidylethanolamineinto phosphatidylethanolamine.The enzyme also catalyzes the acylation of lysophosphatidylserineinto phosphatidylserine (cf. EC 2.3.1.n6).Prefers oleoyl-CoA as the acyl donor.','Acyl-CoA + 1-acyl-sn-glycero-3-phosphoethanolamine = CoA + 1,2-diacyl-sn-glycero-3-phosphoethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n8','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n9','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n10','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n11','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.1.n12','Sinapoyl-beta-D-glucose:anthocyanin sinapoyltransferase','NULL','NULL','The product is the most abundant anthocyanin in the plant Arabidopsisthaliana.The enzyme likely sinapoylates other anthocyanins in the plant.','1-O-sinapoyl-beta-D-glucose + cyanidin 3-O-(6-O-(4-O-beta-D-glucosyl-p-coumaroyl)-2-O-(beta-D-xylosyl)-beta-D-glucosyl)-5-O-(6-O-malonyl-beta-D-glucoside) = D-glucose + cyanidin 3-O-(6-O-(4-O-beta-D-glucosyl-p-coumaroyl)-2-O-(2-O-sinapoyl-beta-D-xylosyl)-beta-D-glucosyl)-5-O-(6-O-malonyl-beta-D-glucoside).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.1','D-glutamyltransferase','D-glutamyl transpeptidase','NULL','The enzyme catalyzes two reactions; the first is the transfer of aglutamyl residue from L- or D-glutamine to D-glutamate via a gammalinkage, forming gamma-glutamyl-D-glutamate, and the second is thetransfer of additional glutamyl residues to the peptide, extendingthe polypeptide chain.','(1) D-glutamine + D-glutamate = ammonia + gamma-D-glutamyl-D-glutamate.(2) L(or D)-glutamine + (gamma-D-glutamyl)(n)-[peptide] = ammonia +(gamma-D-glutamyl)(n+1)-[peptide].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.2','Gamma-glutamyltransferase','Gamma-glutamyl transpeptidase','NULL','The mammlian enzyme is part of the cell antioxidant defensemechanism.It initiates extracellular glutathione (GSH) breakdown, providescells with a local cysteine supply and contributes to maintainintracelular GSH levels.The protein also has EC 3.4.19.13 activity.The enzyme consists of two chains that are created by the proteolyticcleavage of a single precursor polypeptide.The N-terminal L-threonine of the C-terminal subunit functions as theactive site for both the cleavage and the hydrolysis reactions.','A (5-L-glutamyl)-peptide + an amino acid = a peptide + a 5-L-glutamylamino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.3','Lysyltransferase','NULL','NULL','NULL','L-lysyl-tRNA(Lys) + phosphatidylglycerol = tRNA(Lys) + 3-O-L-lysyl-1-O-phosphatidylglycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.4','Gamma-glutamylcyclotransferase','NULL','NULL','Acts on derivatives of L-glutamate, L-2-aminobutanoate, L-alanine andglycine.','(Gamma-L-glutamyl)-L-amino acid = 5-oxoproline + L-amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.5','Glutaminyl-peptide cyclotransferase','Glutaminyl cyclase','NULL','Involved in the formation of thyrotropin-releasing hormone and otherbiologically active peptides containing N-terminal pyroglutamylresidues.The enzyme from papaya also acts on glutaminyl-tRNA.','L-glutaminyl-peptide = 5-oxoprolyl-peptide + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.6','Leucyltransferase','L/F transferase','Monovalent cation.','Also transfers phenylalanyl groups.Peptides and proteins containing an N-terminal arginine, lysine orhistidine residue can act as acceptors.','(1) L-leucyl-tRNA(Leu) + [protein] = tRNA(Leu) + L-leucyl-[protein].(2) L-phenylalanine-tRNA(Phe) + [protein] = tRNA(Phe) + L-phenylalanyl-[protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.7','Aspartyltransferase','NULL','NULL','NULL','L-asparagine + hydroxylamine = NH(3) + beta-L-aspartylhydroxamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.8','Arginyltransferase','Arginyl-tRNA--protein transferase','NULL','Requires mercaptoethanol and a univalent cation.Peptides and proteins containing an N-terminal glutamate, aspartateor cystine residue can act as acceptors.','L-arginyl-tRNA(Arg) + [protein] = tRNA(Arg) + L-arginyl-[protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.9','Agaritine gamma-glutamyltransferase','NULL','NULL','4-hydroxyaniline, cyclohexylamine, 1-naphthylhydrazine and similarcompounds can act as acceptors.Also catalyzes the hydrolysis of agaritine.','Agaritine + acceptor = 4-hydroxymethylphenylhydrazine + gamma-L-glutamyl-acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.10','UDP-N-acetylmuramoylpentapeptide-lysine N(6)-alanyltransferase','Alanyl-transfer ribonucleate-uridine diphosphoacetylmuramoylpentapeptidetransferase','NULL','Also acts on L-seryl-tRNA(Ser).','L-alanyl-tRNA(Ala) + UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamyl-L-lysyl-D-alanyl-D-alanine = tRNA(Ala) + UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamyl-N(6)-(L-alanyl)-L-lysyl-D-alanyl-D-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.11','Alanylphosphatidylglycerol synthase','NULL','NULL','NULL','L-alanyl-tRNA(Ala) + phosphatidylglycerol = tRNA(Ala) + 3-O-L-alanyl-1-O-phosphatidylglycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.12','Peptidyltransferase','NULL','NULL','The enzyme is a ribozyme.Two non-equivlant ribonucleoprotein subunits operate in non-concertedfashion in peptide elongation.The small subunit forms the mRNA-binding machinery and decodingcenter, the large subunit performs the main ribosomal catalyticfunction in the peptidyl-transferase center.','Peptidyl-tRNA(1) + aminoacyl-tRNA(2) = tRNA(1) + peptidyl(aminoacyl-tRNA(2)).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.13','Protein-glutamine gamma-glutamyltransferase','Fibrinoligase','Calcium.','The gamma-carboxymide groups of peptide-bound glutamine residues actas acyl donors, and the 6-amino-groups of protein- and peptide-boundlysine residues act as acceptors, to give intra- and inter-molecularN(6)-(5-glutamyl)lysine crosslinks.','Protein glutamine + alkylamine = protein N(5)-alkylglutamine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.14','D-alanine gamma-glutamyltransferase','NULL','NULL','D-phenylalanine and D-2-aminobutyrate can also act as acceptors, moreslowly.Also catalyzes some of the reactions of EC 2.3.2.2.','L-glutamine + D-alanine = NH(3) + gamma-L-glutamyl-D-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.15','Glutathione gamma-glutamylcysteinyltransferase','Phytochelatin synthase','NULL','NULL','Glutathione + (Glu(-Cys))(n)-Gly = Gly + (Glu(-Cys))(n+1)-Gly.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.16','Lipid II:glycine glycyltransferase','NULL','NULL','This enzyme from Staphylococcus aureus catalyzes the transfer ofglycine from a charged tRNA to MurNAc-L-Ala-D-isoglutaminyl-L-Lys-D-Ala-D-Ala-diphosphoundecaprenyl-GlcNAc (lipid II), attaching it tothe N(6) of the L-lysine at position 3 of the pentapeptide.This is the first step in the synthesis of the pentaglycineinterpeptide bridge that is used in S.aureus for the crosslinking ofdifferent glycan strands to each other.Four additional glycine residues are subsequently attached byEC 2.3.2.17 (FemA) and EC 2.3.2.18 (FemB).','MurNAc-L-Ala-D-isoglutaminyl-L-Lys-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc + glycyl-tRNA(Gly) = MurNAc-L-Ala-D-isoglutaminyl-L-Lys-(N(6)-Gly)-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc+ tRNA(Gly).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.17','N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysyl-(N(6)-glycyl)-D-alanyl-D-alanine-diphosphoundecaprenyl-N-acetylglucosamine:glycineglycyltransferase','NULL','NULL','This enzyme catalyzes the successive transfer of two glycine moietiesfrom charged tRNAs to MurNAc-L-Ala-D-isoglutaminyl-L-Lys-(N(6)-Gly)-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc, attachingthem to a glycine residue previously attached by EC 2.3.2.16 (FemX)to the N(6) of the L-lysine at position 3 of the pentapeptide.This is the second step in the synthesis of the pentaglycineinterpeptide bridge that is used by Staphylococcus aureus for thecrosslinking of different glycan strands to each other.The next step is catalyzed by EC 2.3.2.18 (FemB).This enzyme is essential for methicillin resistance.','MurNAc-L-Ala-D-isoglutaminyl-L-Lys-(N(6)-Gly)-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc + 2 glycyl-tRNA = MurNAc-L-Ala-D-isoglutaminyl-L-Lys-(N(6)-tri-Gly)-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc + 2 tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.18','N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysyl-(N(6)-triglycine)-D-alanyl-D-alanine-diphosphoundecaprenyl-N-acetylglucosamine:glycineglycyltransferase','NULL','NULL','This Staphylococcus aureus enzyme catalyzes the successive transferof two glycine moieties from charged tRNAs to MurNAc-L-Ala-D-isoglutaminyl-L-Lys-(N(6)-tri-Gly)-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc, attaching them to the threeglycine molecules that were previously attached to the N(6) of theL-lysine at position 3 of the pentapeptide by EC 2.3.2.16 (FemX) andEC 2.3.2.17 (FemA).This is the last step in the synthesis of the pentaglycineinterpeptide bridge that is used in this organism for thecrosslinking of different glycan strands to each other.','MurNAc-L-Ala-D-isoglutaminyl-L-Lys-(N(6)-tri-Gly)-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc + 2 glycyl-tRNA = MurNAc-L-Ala-D-isoglutaminyl-L-Lys-(N(6)-penta-Gly)-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc + 2 tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.19','Ribostamycin:4-(gamma-L-glutamylamino)-(S)-2-hydroxybutanoyl-[BtrI acyl-carrier protein] 4-(gamma-L-glutamylamino)-(S)-2-hydroxybutanoatetransferase','NULL','NULL','The enzyme attaches the side chain of the aminoglycoside antibioticsof the butirosin family.The side chain confers resistance against several aminoglycoside-modifying enzymes.','4-(gamma-L-glutamylamino)-(S)-2-hydroxybutanoyl-[BtrI acyl-carrierprotein] + ribostamycin = gamma-L-glutamyl-butirosin B + BtrI acyl-carrier protein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.20','Cyclo(L-leucyl-L-phenylalanyl) synthase','NULL','NULL','The reaction proceeds following a ping-pong mechanism forming acovalent intermediate between an active site serine and theL-phenylalanine residue.The protein, found in the bacterium Streptomyces noursei, also formscyclo(L-phenylalanyl-L-phenylalanyl), cyclo(L-methionyl-L-phenylalanyl), cyclo(L-phenylalanyl-L-tyrosyl) and cyclo(L-methionyl-L-tyrosyl).','L-leucyl-tRNA(Leu) + L-phenylalanyl-tRNA(Phe) = tRNA(Leu) + tRNA(Phe) +cyclo(L-leucyl-L-phenylalanyl).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.21','Cyclo(L-tyrosyl-L-tyrosyl) synthase','Cyclodityrosine synthase','NULL','The reaction proceeds following a ping-pong mechanism forming acovalent intermediate between an active site serine and the firstL-tyrosine residue.The protein, from the bacterium Mycobacterium tuberculosis, alsoforms small amounts of cyclo(L-tyrosyl-L-phenylalanyl).','2 L-tyrosyl-tRNA(Tyr) = 2 tRNA(Tyr) + cyclo(L-tyrosyl-L-tyrosyl).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.2.22','Cyclo(L-leucyl-L-leucyl) synthase','Cyclodileucine synthase','NULL','The reaction proceeds following a ping-pong mechanism forming acovalent intermediate between an active site serine and the firstL-leucine residue.The proteins from bacteria of the genus Bacillus also form smallamounts of cyclo(L-phenylalanyl-L-leucyl) and cyclo(L-leucyl-L-methionyl).','2 L-leucyl-tRNA(Leu) = 2 tRNA(Leu) + cyclo(L-leucyl-L-leucyl).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.1','Citrate (Si)-synthase','(R)-citrate synthase','NULL','The stereospecificity of this enzyme is opposite to that ofEC 2.3.3.3, which is found in some anaerobes.Formerly EC 4.1.3.7.Citrate synthase for which the stereospecificity with respect to C(2)of oxaloacetate has not been established are included in EC 2.3.3.16.','Acetyl-CoA + H(2)O + oxaloacetate = citrate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.2','Decylcitrate synthase','(2S,3S)-2-hydroxytridecane-1,2,3-tricarboxylate oxaloacetate-lyase (CoA-acylating)','NULL','Formerly EC 4.1.3.23.','Lauroyl-CoA + H(2)O + oxaloacetate = (2S,3S)-2-hydroxytridecane-1,2,3-tricarboxylate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.3','Citrate (Re)-synthase','(R)-citrate synthase','NULL','This enzyme is inactivated by oxygen and is found in some anaerobes.Its stereospecificity is opposite to that of EC 2.3.3.1.Formerly EC 4.1.3.28.','Acetyl-CoA + H(2)O + oxaloacetate = citrate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.4','Decylhomocitrate synthase','2-decylhomocitrate synthase','NULL','Decanoyl-CoA can act instead of dodecanoyl-CoA, but 2-oxoglutaratecannot be replaced by oxaloacetate or pyruvate.Formerly EC 4.1.3.29.','Dodecanoyl-CoA + H(2)O + 2-oxoglutarate = (3S,4S)-3-hydroxytetradecane-1,3,4-tricarboxylate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.5','2-methylcitrate synthase','2-methylcitrate oxaloacetate-lyase','NULL','The enzyme acts on acetyl-CoA, propanoyl-CoA, butanoyl-CoA andpentanoyl-CoA.The relative rate of condensation of acetyl-CoA and oxaloacetate is140% of that of propanoyl-CoA and oxaloacetate, but the enzyme hasbeen separated from EC 2.3.3.1.Oxaloacetate cannot be replaced by glyoxylate, pyruvate or2-oxoglutarate.Formerly EC 4.1.3.31.','Propanoyl-CoA + H(2)O + oxaloacetate = (2R,3S)-2-hydroxybutane-1,2,3-tricarboxylate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.6','2-ethylmalate synthase','(R)-2-ethylmalate 2-oxobutanoyl-lyase (CoA-acetylating)','NULL','Also acts on (R)-2-(n-propyl)-malate.Formerly EC 4.1.3.33.','Acetyl-CoA + H(2)O + 2-oxobutanoate = (R)-2-ethylmalate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.7','3-ethylmalate synthase','2-ethyl-3-hydroxybutanedioate synthase','NULL','Formerly EC 4.1.3.10.','Butanoyl-CoA + H(2)O + glyoxylate = 3-ethylmalate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.8','ATP citrate synthase','Acetyl-CoA:oxaloacetate acetyltransferase (isomerizing; ADP-phosphorylating)','NULL','The enzyme can be dissociated into components, two of which areidentical with EC 4.1.3.34 and EC 6.2.1.18.Formerly EC 4.1.3.8.','ADP + phosphate + acetyl-CoA + oxaloacetate = ATP + citrate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.9','Malate synthase','Glyoxylate transacetase','NULL','Formerly EC 4.1.3.2.','Acetyl-CoA + H(2)O + glyoxylate = (S)-malate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.10','Hydroxymethylglutaryl-CoA synthase','(S)-3-hydroxy-3-methylglutaryl-CoA acetoacetyl-CoA-lyase (CoA-acetylating)','NULL','Formerly EC 4.1.3.5.','Acetyl-CoA + H(2)O + acetoacetyl-CoA = (S)-3-hydroxy-3-methylglutaryl-CoA+ CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.11','2-hydroxyglutarate synthase','2-hydroxyglutarate glyoxylate-lyase (CoA-propanoylating)','NULL','Formerly EC 4.1.3.9.','Propanoyl-CoA + H(2)O + glyoxylate = 2-hydroxyglutarate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.12','3-propylmalate synthase','3-(n-propyl)-malate synthase','NULL','Formerly EC 4.1.3.11.','Pentanoyl-CoA + H(2)O + glyoxylate = 3-propylmalate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.13','2-isopropylmalate synthase','3-carboxy-3-hydroxy-4-methylpentanoate 3-methyl-2-oxobutanoate-lyase(CoA-acetylating)','Potassium.','Formerly EC 4.1.3.12.','Acetyl-CoA + 3-methyl-2-oxobutanoate + H(2)O = (2S)-2-isopropylmalate +CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.14','Homocitrate synthase','2-hydroxybutane-1,2,4-tricarboxylate 2-oxoglutarate-lyase (CoA-acetylating)','NULL','Belongs in the alpha-aminoadipate pathway of lysine synthesis, alongwith EC 4.2.1.36.Also acts with oxaloacetate as substrate, but more slowly.Formerly EC 4.1.3.21.','Acetyl-CoA + H(2)O + 2-oxoglutarate = (R)-2-hydroxybutane-1,2,4-tricarboxylate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.15','Sulfoacetaldehyde acetyltransferase','NULL','Magnesium; Thiamine diphosphate.','The reaction occurs in the reverse direction to that shown above.','Acetyl phosphate + sulfite = 2-sulfoacetaldehyde + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.3.3.16','Citrate synthase (unknown stereospecificity)','Citrate condensing enzyme','NULL','This entry has been included to accommodate those citrate synthasesfor which the stereospecificity with respect to C(2) of oxaloacetatehas not been established (cf. EC 2.3.3.1 and EC 2.3.3.3).','Acetyl-CoA + H(2)O + oxaloacetate = citrate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.1','Glycogen phosphorylase','Amylophosphorylase','NULL','This entry covers several enzymes from different sources that act invivo on different forms of (1-','((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.2','Dextrin dextranase','Dextrin 6-glucosyltransferase','NULL','NULL','((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.3','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.4','Amylosucrase','Sucrose--glucan glucosyltransferase','NULL','NULL','Sucrose + ((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.5','Dextransucrase','Sucrose 6-glucosyltransferase','NULL','NULL','Sucrose + ((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.6','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.7','Sucrose phosphorylase','Disaccharide glucosyltransferase','NULL','In the forward reaction, arsenate may replace phosphate.In the reverse reaction, various ketoses and L-arabinose may replaceD-fructose.','Sucrose + phosphate = D-fructose + alpha-D-glucose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.8','Maltose phosphorylase','NULL','NULL','NULL','Maltose + phosphate = D-glucose + beta-D-glucose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.9','Inulosucrase','Sucrose 1-fructosyltransferase','NULL','Converts sucrose into inulin and D-glucose.Some other sugars can act as D-fructosyl acceptors.','Sucrose + ((2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.10','Levansucrase','Sucrose 6-fructosyltransferase','NULL','Some other sugars can act as D-fructosyl acceptors.','Sucrose + (6)-beta-D-fructofuranosyl-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.11','Glycogen(starch) synthase','UDP-glucose--glycogen glucosyltransferase','NULL','The description (official name) varies according to the source of theenzyme and the nature of its synthetic product.Glycogen synthase from animal tissues is a complex of a catalyticsubunit and the protein glycogenin.The enzyme requires glucosylated glycogenin as a primer; this is thereaction product of EC 2.4.1.186.A similar enzyme utilizes ADP-glucose (cf. EC 2.4.1.21).','UDP-glucose ((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.12','Cellulose synthase (UDP-forming)','UDP-glucose--beta-glucan glucosyltransferase','NULL','Involved in the synthesis of cellulose.A similar enzyme utilizes GDP-glucose (cf. EC 2.4.1.29).','UDP-glucose + (1,4-beta-D-glucosyl)(n) = UDP + (1,4-beta-D-glucosyl)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.13','Sucrose synthase','Sucrose synthetase','NULL','Although UDP is generally considered to be the preferred nucleosidediphosphate for sucrose synthase, numerous studies have shown thatADP serves as an effective acceptor molecule to produce ADP-glucose.Sucrose synthase has a dual role in producing both UDP-glucose(necessary for cell wall and glycoprotein biosynthesis) and ADP-glucose (necessary for starch biosynthesis).','NDP-glucose + D-fructose = NDP + sucrose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.14','Sucrose-phosphate synthase','Sucrosephosphate--UDP glucosyltransferase','Manganese or magnesium.','The enzyme from Synechocystis sp. strain PCC 6803 is not specific forUDP-glucose as it can use ADP-glucose and, to a lesser extent, GDP-glucose as substrates.The enzyme from rice leaves is activated by glucose 6-phosphate butthat from cyanobacterial species is not.While the reaction catalyzed by this enzyme is reversible, the enzymeusually works in concert with EC 3.1.3.24 to form sucrose, making theabove reaction essentially irreversible.The F in sucrose 6(F)-phosphate is used to indicate that the fructoseresidue of sucrose carries the substituent.','UDP-glucose + D-fructose 6-phosphate = UDP + sucrose 6(F)-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.15','Alpha,alpha-trehalose-phosphate synthase (UDP-forming)','Transglucosylase','NULL','See also EC 2.4.1.36.','UDP-glucose + D-glucose 6-phosphate = UDP + alpha,alpha-trehalose6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.16','Chitin synthase','Chitin-UDP acetyl-glucosaminyl transferase','NULL','Converts UDP-N-acetyl-D-glucosamine into chitin and UDP.','UDP-N-acetyl-alpha-D-glucosamine + (1,4-(N-acetyl-beta-D-glucosaminyl))(n) = UDP + (1,4-(N-acetyl-beta-D-glucosaminyl))(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.17','Glucuronosyltransferase','1-naphthol glucuronyltransferase','NULL','Family of enzymes accepting a wide range of substrates, includingphenols, alcohols, amines and fatty acids.Some of the activities catalyzed were previously listed separately asEC 2.4.1.42, EC 2.4.1.59, EC 2.4.1.61, EC 2.4.1.76, EC 2.4.1.77,EC 2.4.1.84, EC 2.4.1.107 and EC 2.4.1.108.A temporary nomenclature for the various forms whose delineation isin a state of flux.','UDP-glucuronate + acceptor = UDP + acceptor beta-D-glucuronoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.18','1,4-alpha-glucan branching enzyme','Amylo-(1,4 to 1,6)transglucosidase','NULL','Converts amylose into amylopectin.The description (official name) requires a qualification depending onthe product, glycogen or amylopectin, e.g. glycogen branching enzyme,amylopectin branching enzyme.The latter has frequently been termed Q-enzyme.','Transfers a segment of a (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.19','Cyclomaltodextrin glucanotransferase','Bacillus macerans amylase','NULL','Cyclomaltodextrins (Schardinger dextrins) of various sizes (6, 7,8, etc. glucose units) are formed reversibly from starch and similarsubstrates.Also disproportionates linear maltodextrins without cyclizing (cf.EC 2.4.1.25).','Cyclizes part of a (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.20','Cellobiose phosphorylase','NULL','NULL','NULL','Cellobiose + phosphate = alpha-D-glucose 1-phosphate + D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.21','Starch synthase','ADP-glucose--starch glucosyltransferase','NULL','The description (official name) various according to the source ofthe enzyme and the nature of its synthetic product, e.g. starchsynthase, bacterial glycogen synthase.A similar enzyme utilizes UDP-glucose (cf. EC 2.4.1.11).','ADP-glucose + (1,4-alpha-D-glucosyl)(n) = ADP + (1,4-alpha-D-glucosyl)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.22','Lactose synthase','N-acetyllactosamine synthase','NULL','The enzyme is a complex of 2 proteins A and B.In the absence of the B protein (alpha-lactalbumin) the enzymecatalyzes the transfer of galactose from UDP-galactose toN-acetylglucosamine (cf. EC 2.4.1.90).','UDP-alpha-D-galactose + D-glucose = UDP + lactose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.23','Sphingosine beta-galactosyltransferase','Psychosine--UDP galactosyltransferase','NULL','NULL','UDP-alpha-D-galactose + sphingosine = UDP + psychosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.24','1,4-alpha-glucan 6-alpha-glucosyltransferase','Oligoglucan-branching glycosyltransferase','NULL','NULL','Transfers an alpha-D-glucosyl residue in a (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.25','4-alpha-glucanotransferase','D-enzyme','NULL','An enzymic activity of this nature forms part of the mammalian andSaccharomyces cerevisiae glycogen branching system (see EC 3.2.1.33).Formerly EC 2.4.1.3.','Transfers a segment of a (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.26','DNA alpha-glucosyltransferase','NULL','NULL','NULL','Transfers an alpha-D-glucosyl residue from UDP-glucose to anhydroxymethylcytosine residue in DNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.27','DNA beta-glucosyltransferase','NULL','NULL','NULL','Transfers a beta-D-glucosyl residue from UDP-glucose to anhydroxymethylcytosine residue in DNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.28','Glucosyl-DNA beta-glucosyltransferase','NULL','NULL','NULL','Transfers a beta-D-glucosyl residue from UDP-glucose to aglucosylhydroxymethylcytosine residue in DNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.29','Cellulose synthase (GDP-forming)','GDP-glucose-beta-D-glucan glucosyltransferase','NULL','Involved in the synthesis of cellulose.A similar enzyme utilizes UDP-glucose (cf. EC 2.4.1.12).','GDP-glucose + ((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.30','1,3-beta-oligoglucan phosphorylase','Beta-1,3-oligoglucan:orthophosphate glucosyltransferase II','NULL','Does not act on laminarin.Differs in specificity from EC 2.4.1.31 and EC 2.4.1.97.','((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.31','Laminaribiose phosphorylase','NULL','NULL','Also acts on 1,3-beta-D-oligoglucans.Differs in specificity from EC 2.4.1.30 and EC 2.4.1.97.','3-beta-D-glucosyl-D-glucose + phosphate = D-glucose + alpha-D-glucose1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.32','Glucomannan 4-beta-mannosyltransferase','Glucomannan-synthase','NULL','NULL','GDP-mannose + (glucomannan)(n) = GDP + (glucomannan)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.33','Alginate synthase','Mannuronosyl transferase','NULL','NULL','GDP-D-mannuronate + (alginate)(n) = GDP + (alginate)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.34','1,3-beta-glucan synthase','1,3-beta-D-glucan--UDP glucosyltransferase','NULL','NULL','UDP-glucose + ((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.35','Phenol beta-glucosyltransferase','UDP-glucosyltransferase','NULL','Acts on a wide range of phenols.','UDP-glucose + a phenol = UDP + an aryl beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.36','Alpha,alpha-trehalose-phosphate synthase (GDP-forming)','GDP-glucose-glucosephosphate glucosyltransferase','NULL','See also EC 2.4.1.15.','GDP-glucose + glucose 6-phosphate = GDP + alpha,alpha-trehalose6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.37','Fucosylgalactoside 3-alpha-galactosyltransferase','[Blood group substance] alpha-galactosyltransferase','NULL','Acts on blood group substance, and can use a number of 2-fucosyl-galactosides as acceptors.','UDP-alpha-D-galactose + alpha-L-fucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.38','Beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase','GalT','NULL','Terminal N-acetyl-beta-D-glucosaminyl residues in polysaccharides,glycoproteins and glycopeptides can act as acceptor.High activity is shown toward such residues in branched-chainpolysaccharides when these are linked by beta-1,6 links to galactoseresidues; lower activity toward residues linked to galactose by beta-1,3 links.A component of EC 2.4.1.22.','UDP-alpha-D-galactose + N-acetyl-beta-D-glucosaminylglycopeptide = UDP +beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.39','Steroid N-acetylglucosaminyltransferase','Sucrose-glucan glucosyltransferase','NULL','NULL','UDP-N-acetyl-alpha-D-glucosamine + estradiol-17-alpha 3-D-glucuronoside =UDP + 17-alpha-(N-acetyl-alpha-D-glucosaminyl)-estradiol 3-D-glucuronoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.40','Glycoprotein-fucosylgalactoside alpha-N-acetylgalactosaminyltransferase','A transferase','NULL','Acts on blood group substance H.Can use a number of 2-fucosyl-galactosides as acceptors.','UDP-N-acetyl-alpha-beta-D-galactosamine + glycoprotein-alpha-L-fucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.41','Polypeptide N-acetylgalactosaminyltransferase','Protein-UDP acetylgalactosaminyltransferase','Calcium; Manganese.','The glycosyl residue is transferred to threonine or serine hydroxygroups on the polypeptide core of submaxillary mucin, kappa-casein,apofetuin and some other acceptors of high molecular mass.','UDP-N-acetyl-alpha-D-galactosamine + polypeptide = UDP + N-acetyl-alpha-D-galactosaminyl-polypeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.42','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.43','Polygalacturonate 4-alpha-galacturonosyltransferase','NULL','NULL','NULL','UDP-D-galacturonate + ((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.44','Lipopolysaccharide 3-alpha-galactosyltransferase','Lipopolysaccharide 1,3-galactosyltransferase','NULL','Transfers D-galactosyl residues to D-glucose in the partiallycompleted core of lipopolysaccharide (cf. EC 2.4.1.56, EC 2.4.1.58,and EC 2.4.1.73).','UDP-alpha-D-galactose + lipopolysaccharide = UDP + 3-alpha-D-galactosyl-[lipopolysaccharide glucose].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.45','2-hydroxyacylsphingosine 1-beta-galactosyltransferase','Cerebroside synthase','NULL','Highly specific.','UDP-alpha-D-galactose + 2-(2-hydroxyacyl)sphingosine = UDP + 1-(beta-D-galactosyl)-2-(2-hydroxyacyl)sphingosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.46','Monogalactosyldiacylglycerol synthase','1,2-diacylglycerol 3-beta-galactosyltransferase','NULL','This enzyme adds only one galactosyl group to the diacylglycerol;EC 2.4.1.241 adds a galactosyl group to the product of the abovereaction.There are three isoforms in Arabidopsis that can be divided into twotypes, A-type (MGD1) and B-type (MGD2 and MGD3).MGD1 is the isoform responsible for the bulk ofmonogalactosyldiacylglycerol (MGDG) synthesis in Arabidopsis.','UDP-alpha-D-galactose + 1,2-diacyl-sn-glycerol = UDP + 3-beta-D-galactosyl-1,2-diacyl-sn-glycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.47','N-acylsphingosine galactosyltransferase','NULL','NULL','NULL','UDP-alpha-D-galactose + N-acylsphingosine = UDP + D-galactosylceramide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.48','Heteroglycan alpha-mannosyltransferase','NULL','NULL','The acceptor is a heteroglycan primer containing mannose, galactoseand xylose.1,2- and 1,3-mannosyl bonds are formed.','GDP-mannose + heteroglycan = GDP + 2(or 3)-alpha-D-mannosyl-heteroglycan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.49','Cellodextrin phosphorylase','NULL','NULL','NULL','((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.50','Procollagen galactosyltransferase','Hydroxylysine galactosyltransferase','NULL','Probably involved in the synthesis of carbohydrate units incomplement (cf. EC 2.4.1.66).','UDP-alpha-D-galactose + 5-hydroxy-L-lysine-[procollagen] = UDP +5-(D-galactosyloxy)-L-lysine-[procollagen].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.51','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.52','Poly(glycerol-phosphate) alpha-glucosyltransferase','NULL','NULL','NULL','UDP-glucose + poly(glycerol phosphate) = UDP + O-(alpha-D-glucosyl)poly(glycerol phosphate).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.53','Poly(ribitol-phosphate) beta-glucosyltransferase','NULL','NULL','NULL','UDP-glucose + poly(ribitol phosphate) = UDP + (beta-D-glucosyl)poly(ribitol phosphate).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.54','Undecaprenyl-phosphate mannosyltransferase','NULL','NULL','Requires phosphatidylglycerol.','GDP-mannose + undecaprenyl phosphate = GDP + D-mannosyl-1-phosphoundecaprenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.55','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.56','Lipopolysaccharide N-acetylglucosaminyltransferase','NULL','NULL','Transfers N-acetylglucosaminyl residues to a D-galactose residue inthe partially completed lipopolysaccharide core (cf. EC 2.4.1.44,EC 2.4.1.58, and EC 2.4.1.73).','UDP-N-acetyl-alpha-D-glucosamine + lipopolysaccharide = UDP + N-acetyl-alpha-D-glucosaminyllipopolysaccharide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.57','Phosphatidylinositol alpha-mannosyltransferase','GDP mannose-phosphatidyl-myo-inositol alpha-mannosyltransferase','NULL','NULL','Transfers one or more alpha-D-mannose residues from GDP-mannose topositions 2,6 and others in 1-phosphatidyl-myo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.58','Lipopolysaccharide glucosyltransferase I','NULL','NULL','Transfers glucosyl residues to the backbone portion oflipopolysaccharide (cf. EC 2.4.1.44, EC 2.4.1.56, and EC 2.4.1.73).','UDP-glucose + lipopolysaccharide = UDP + D-glucosyl-lipopolysaccharide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.59','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.60','Abequosyltransferase','NULL','NULL','NULL','CDP-D-abequose + alpha-D-mannopyranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.61','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.62','Ganglioside galactosyltransferase','NULL','NULL','The substrate is also known as gangloside GM2, the product asgangloside GM1a.','UDP-alpha-D-galactose + an N-acetyl-beta-D-galactosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.63','Linamarin synthase','NULL','NULL','Glucosylates the cyanohydrins of butanone and pentan-3-one as well asthat of acetone.','UDP-glucose + 2-hydroxy-2-methylpropanenitrile = UDP + linamarin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.64','Alpha,alpha-trehalose phosphorylase','NULL','NULL','NULL','Alpha,alpha-trehalose + phosphate = D-glucose + beta-D-glucose1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.65','3-galactosyl-N-acetylglucosaminide 4-alpha-L-fucosyltransferase','(Le(a))-dependent (alpha-3/4)-fucosyltransferase','NULL','Product of the Lewis blood group gene.Normally acts on a glycoconjugate where R (see CA line) is aglycoprotein or glycolipid.Although it is a 4-fucosyltransferase, it has a persistent3-fucosyltransferase activity toward the glucose residue in freelactose.Fucosylates on O-4 of an N-acetylglucosamine that carries agalactosyl group on O-3, unlike EC 2.4.1.152 which fucosylates on O-3of an N-acetylglucosamine that carries a galactosyl group on O-4.Enzymes catalyzing the 4-alpha-fucosylation of the GlcNAc in beta-D-Gal-(1-','GDP-beta-L-fucose + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.66','Procollagen glucosyltransferase','Galactosylhydroxylysine-glucosyltransferase','NULL','Probably involved in the synthesis of carbohydrate units incomplement (cf. EC 2.4.1.50).','UDP-glucose + (2S,5R)-5-O-(beta-D-galactosyl)-5-hydroxy-lysine-[procollagen] = UDP + (2S,5R)-5-O-(alpha-D-glucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.67','Galactinol--raffinose galactosyltransferase','Galactinol-raffinose galactosyltransferase','NULL','This enzyme also catalyzes galactosyl transfer from stachyose toraffinose (shown by labelling).For synthesis of the substrate, see EC 2.4.1.123.See also EC 2.4.1.82.','Alpha-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.68','Glycoprotein 6-alpha-L-fucosyltransferase','GDP-fucose--glycoprotein fucosyltransferase','NULL','Catalyzes a reaction similar to that of EC 2.4.1.214, but transfersthe L-fucosyl group from GDP-beta-L-fucose to form an alpha-1,6-linkage rather than an alpha-1,3-linkage.','GDP-beta-L-fucose + N(4)-(N-acetyl-beta-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.69','Galactoside 2-alpha-L-fucosyltransferase','Alpha-(1-','NULL','Free lactose can act as acceptor.Formerly EC 2.4.1.89.','GDP-beta-L-fucose + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.70','Poly(ribitol-phosphate) N-acetylglucosaminyltransferase','NULL','NULL','Involved in the synthesis of teichoic acids.','UDP-N-acetyl-alpha-D-glucosamine + poly(ribitol phosphate) = UDP +(N-acetyl-alpha-D-glucosaminyl)poly(ribitol phosphate).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.71','Arylamine glucosyltransferase','UDP glucose-arylamine glucosyltransferase','NULL','NULL','UDP-glucose + an arylamine = UDP + an N-D-glucosylarylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.72','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.73','Lipopolysaccharide glucosyltransferase II','NULL','NULL','Transfers glucosyl residues to the D-galactosyl-D-glucosyl side-chains in the partially completed core of lipopolysaccharide (cf.EC 2.4.1.44, EC 2.4.1.56, and EC 2.4.1.58).','UDP-glucose + lipopolysaccharide = UDP + alpha-D-glucosyl-lipopolysaccharide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.74','Glycosaminoglycan galactosyltransferase','NULL','NULL','Involved in the biosynthesis of galactose-containingglycosaminoglycan of Dictyostelium discoideum.','UDP-alpha-D-galactose + glycosaminoglycan = UDP +D-galactosylglycosaminoglycan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.75','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.76','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.77','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.78','Phosphopolyprenol glucosyltransferase','NULL','NULL','Ficaprenyl phosphate is the best substrate; other polyprenols canalso act as substrates, more slowly.','UDP-glucose + polyprenyl phosphate = UDP + polyprenylphosphate-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.79','Globotriaosylceramide 3-beta-N-acetylgalactosaminyltransferase','Beta-3GalNAc-T1','Manganese.','Globoside is a neutral glycosphingolipid in human erythrocytes andhas blood-group-P-antigen activity.UDP-GalNAc is the only sugar donor that is used efficiently by theenzyme: UDP-Gal and UDP-GlcNAc result in very low enzyme activity.Lactosylceramide, globoside and gangliosides GM3 and GD3 are notsubstrates.','UDP-N-acetyl-alpha-D-galactosamine + alpha-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.80','Ceramide glucosyltransferase','Glucosylceramide synthase','NULL','Sphingosine and dihydrosphingosine can also act as acceptors; CDP-glucose can act as donor.','UDP-glucose + N-acylsphingosine = UDP + D-glucosyl-N-acylsphingosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.81','Flavone 7-O-beta-glucosyltransferase','UDP-glucose-apigenin beta-glucosyltransferase','NULL','A number of flavones, flavonones and flavonols can function asacceptors.Different from EC 2.4.1.91.','UDP-glucose + 5,7,3,4-tetrahydroxyflavone = UDP + 7-O-beta-D-glucosyl-5,7,3,4-tetrahydroxyflavone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.82','Galactinol--sucrose galactosyltransferase','1-alpha-D-galactosyl-myo-inositol:sucrose 6-alpha-D-galactosyltransferase','NULL','4-nitrophenyl-alpha-D-galactopyranoside can also act as donor.Also catalyzes an exchange reaction between raffinose and sucrose(cf. EC 2.4.1.123).','Alpha-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.83','Dolichyl-phosphate beta-D-mannosyltransferase','Dolichol-phosphate mannose synthase','NULL','Acts only on long-chain polyprenyl phosphates and alpha-dihydropolyprenyl phosphates, larger than C(35).','GDP-mannose + dolichyl phosphate = GDP + dolichyl D-mannosyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.84','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.85','Cyanohydrin beta-glucosyltransferase','UDP-glucose-p-hydroxymandelonitrile glucosyltransferase','NULL','Acts on a wide range of substrates in vitro, including cyanohydrins,terpenoids, phenolics, hexanol derivatives and plant hormones, in aregiospecific manner.Involved in the biosynthesis of the cyanogenic glucoside dhurrin insorghum, along with EC 1.14.13.41 and EC 1.14.13.68.This reaction prevents the disocciation and release of toxic hydrogencyanide.','UDP-D-glucose + (S)-4-hydroxymandelonitrile = UDP + (S)-4-hydroxymandelonitrile beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.86','Glucosaminylgalactosylglucosylceramide beta-galactosyltransferase','Glucosaminylgalactosylglucosylceramide 4-beta-galactosyltransferase','NULL','NULL','UDP-alpha-D-galactose + N-acetyl-beta-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.87','N-acetyllactosaminide 3-alpha-galactosyltransferase','Alpha-galactosyltransferase','NULL','Acts on beta-galactosyl-1,4-N-acetylglucosaminyl termini on asialo-alpha(1)-acid glycoprotein and N-acetyllactosamine (beta-D-galactosyl-1,4-N-acetyl-beta-D-glucosamine), but not on2-fucosylated-N-acetyllactosamine.The non-reducing terminal N-acetyllactosamine residues ofglycoproteins can also act as acceptor.Now includes EC 2.4.1.124 and EC 2.4.1.151.','UDP-alpha-D-galactose + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.88','Globoside alpha-N-acetylgalactosaminyltransferase','Forssman synthase','NULL','NULL','UDP-N-acetyl-alpha-D-galactosamine + N-acetyl-D-galactosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.89','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.90','N-acetyllactosamine synthase','Acetyllactosamine synthetase','NULL','The reaction is catalyzed by a component of EC 2.4.1.22, which isidentical with EC 2.4.1.38, and by an enzyme from the Golgi apparatusof animal tissues.Formerly EC 2.4.1.98.','UDP-alpha-D-galactose + N-acetyl-D-glucosamine = UDP +N-acetyllactosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.91','Flavonol 3-O-glucosyltransferase','UDP-glucose flavonol 3-O-glucosyltransferase','NULL','Acts on a variety of flavonols, including quercetin and quercetin7-O-glucoside.Different from EC 2.4.1.81.','UDP-glucose + a flavonol = UDP + a flavonol 3-O-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.92','(N-acetylneuraminyl)-galactosylglucosylceramideN-acetylgalactosaminyltransferase','Asialo-GM2 synthase','NULL','Catalyzes the formation of the gangliosides (i.e. sialic-acid-containing glycosphingolipids) GM2, GD2 and SM2 from GM3, GD3 andSM3, respectively.Asialo-GM3 and lactosylceramide are also substrates,but glycoproteins and oligosaccharides are not substrates.','UDP-N-acetyl-alpha-D-galactosamine + O-(N-acetyl-alpha-neuraminyl)-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.93','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.94','Protein N-acetylglucosaminyltransferase','N-GlcNAc transferase','NULL','The acceptor is the asparagine residue in a sequence of the form Asn-Xaa-(Ser/Thr).','UDP-N-acetyl-D-glucosamine + [protein]-L-asparagine = UDP + [protein]-N(4)-(N-acetyl-D-glucosaminyl)-L-asparagine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.95','Bilirubin-glucuronoside glucuronosyltransferase','Bilirubin monoglucuronide transglucuronidase','NULL','NULL','2 bilirubin-glucuronoside = bilirubin + bilirubin-bisglucuronoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.96','sn-glycerol-3-phosphate 1-galactosyltransferase','Isofloridoside-phosphate synthase','NULL','The product is hydrolyzed by a phosphatase to isofloridoside, whichis involved in osmoregulation (cf. EC 2.4.1.137).','UDP-alpha-D-galactose + sn-glycerol 3-phosphate = UDP + 1-O-alpha-D-galactosyl-sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.97','1,3-beta-D-glucan phosphorylase','Laminarin phosphorylase','NULL','Acts on a range of beta-1,3-oligoglucans, and on glucans of laminarintype.Different from EC 2.4.1.30 and EC 2.4.1.31.','((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.98','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.99','Sucrose:sucrose fructosyltransferase','SST','NULL','NULL','2 sucrose = D-glucose + beta-D-fructofuranosyl-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.100','2,1-fructan:2,1-fructan 1-fructosyltransferase','1,2-beta-D-fructan 1(F)-fructosyltransferase','NULL','NULL','(Beta-D-fructosyl-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.101','Alpha-1,3-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase','Alpha-1,3-mannosyl-glycoprotein beta-1,2-N-acetylglucosaminyltransferase','NULL','R represents the remainder of the N-linked oligosaccharide in theglycoprotein acceptor.Note that this enzyme acts before N-acetylglucosaminyltransferases II(EC 2.4.1.143), III (EC 2.4.1.144), IV (EC 2.4.1.145),V (EC 2.4.1.155) and VI (EC 2.4.1.201).Formerly EC 2.4.1.51.','UDP-N-acetyl-D-glucosamine + 3-(alpha-D-mannosyl)-beta-D-mannosyl-R =UDP + 3-(2-(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl)-beta-D-mannosyl-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.102','Beta-1,3-galactosyl-O-glycosyl-glycoprotein beta-1,6-N-acetylglucosaminyltransferase','Beta(6)-N-acetylglucosaminyltransferase','NULL','Cf. EC 2.4.1.146, EC 2.4.1.147, and EC 2.4.1.148.','UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-1,3-N-acetyl-D-galactosaminyl-R = UDP + beta-D-galactosyl-1,3-(N-acetyl-beta-D-glucosaminyl-1,6)-N-acetyl-D-galactosaminyl-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.103','Alizarin 2-beta-glucosyltransferase','NULL','NULL','Acts on other hydroxy- and dihydroxy-derivatives of 9,10-anthraquinone.','UDP-glucose + alizarin = UDP + 1-hydroxy-2-(beta-D-glucosyloxy)-9,10-anthraquinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.104','o-dihydroxycoumarin 7-O-glucosyltransferase','NULL','NULL','Converts the aglycone daphetin into daphnin and, more slowly,esculetin into cichoriin, umbelliferone into skimmin, hydrangetininto hydrangin and scopoletin into scopolin.','UDP-glucose + 7,8-dihydroxycoumarin = UDP + daphnin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.105','Vitexin beta-glucosyltransferase','NULL','NULL','Vitexin is a flavonoid from Cannabis sativa (hemp) and somepopulations of Silene alba.','UDP-glucose + vitexin = UDP + vitexin 2-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.106','Isovitexin beta-glucosyltransferase','NULL','NULL','Isovitexin is a flavonoid from petals of Silene alba.','UDP-glucose + isovitexin = UDP + isovitexin 2-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.107','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.108','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.109','Dolichyl-phosphate-mannose-protein mannosyltransferase','NULL','NULL','Transfers mannosyl residues to the hydroxy of serine or threonineresidues, producing cell-wall mannoproteins.Acts only on long-chain alpha-dihydropolyprenyl derivatives, largerthan C(35).','Dolichyl phosphate D-mannose + protein = dolichyl phosphate + O-D-mannosylprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.110','tRNA-queuosine beta-mannosyltransferase','NULL','NULL','NULL','GDP-mannose + tRNA(Asp)-queuosine = GDP + tRNA(Asp)-O-5-beta-D-mannosylqueuosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.111','Coniferyl-alcohol glucosyltransferase','NULL','NULL','Sinapyl alcohol can also act as acceptor.','UDP-glucose + coniferyl alcohol = UDP + coniferin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.112','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.113','Alpha-1,4-glucan-protein synthase (ADP-forming)','NULL','NULL','Builds up alpha-1,4-glucan chains covalently bound to protein, thusacting as an initiator of glycogen synthesis.','ADP-glucose + protein = ADP + alpha-D-glucosyl-protein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.114','2-coumarate O-beta-glucosyltransferase','NULL','NULL','Coumarinate (cis-2-hydroxycinnamate) does not act as acceptor.','UDP-glucose + trans-2-hydroxycinnamate = UDP + trans-beta-D-glucosyl-2-hydroxycinnamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.115','Anthocyanidin 3-O-glucosyltransferase','3-GT','NULL','The anthocyanidin compounds cyanidin, delphinidin, peonidin and to alesser extent pelargonidin can act as substrates.Does not catalyze glucosylation of the 5-position of cyanidin anddoes not act on flavanols such as quercetin and kaempferol (cf.EC 2.4.1.91).In conjunction with EC 1.14.11.19, it is involved in the conversionof leucoanthocyanidin into anthocyanidin 3-glucoside.It may act on the pseudobase precursor of the anthocyanidin ratherthan on the anthocyanidin itself.Formerly EC 2.4.1.233.','UDP-D-glucose + an anthocyanidin = UDP + an anthocyanidin-3-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.116','Cyanidin 3-O-rutinoside 5-O-glucosyltransferase','Cyanidin-3-rhamnosylglucoside 5-O-glucosyltransferase','NULL','Isolated from the plants Silene dioica (red campion), Iris ensata(Japanese iris) and Iris hollandica (Dutch iris).Also acts on the 3-O-rutinosides of pelargonidin, delphinidin andmalvidin, but not the corresponding glucosides or 6-acylglucosides.The enzyme does not catalyze the glucosylation of the 5-hydroxy groupof cyanidin 3-glucoside.','UDP-alpha-D-glucose + cyanidin-3-O-rutinoside = UDP + cyanidin 3-O-rutinoside 5-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.117','Dolichyl-phosphate beta-glucosyltransferase','NULL','NULL','Solanesyl phosphate and ficaprenyl phosphate can act as acceptors,more slowly.','UDP-glucose + dolichyl phosphate = UDP + dolichyl beta-D-glucosylphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.118','Cytokinin 7-beta-glucosyltransferase','Cytokinin 7-glucosyltransferase','NULL','Acts on a range of N(6)-substituted adenines, including zeatin andN(6)-benzylaminopurine, but not N(6)-benzyladenine.With some acceptors, 9-beta-D-glucosides are also formed.','UDP-glucose + N(6)-alkylaminopurine = UDP + N(6)-alkylaminopurine-7-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.119','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.120','Sinapate 1-glucosyltransferase','NULL','NULL','Some other hydroxycinnamates, including 4-coumarate, ferulate andcaffeate, can also act as acceptors, more slowly.Only glucose esters, not glucosides, are formed (cf. EC 2.4.1.126).','UDP-glucose + sinapate = UDP + 1-sinapoyl-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.121','Indole-3-acetate beta-glucosyltransferase','IAA-Glu synthetase','NULL','NULL','UDP-glucose + (indol-3-yl)acetate = UDP + 1-O-(indol-3-yl)acetyl-beta-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.122','Glycoprotein-N-acetylgalactosamine 3-beta-galactosyltransferase','NULL','NULL','The non-reducing O-serine-linked N-acetylgalactosamine residues inmucin glycoproteins can act as acceptors.','UDP-alpha-D-galactose + glycoprotein N-acetyl-D-galactosamine = UDP +glycoprotein D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.123','Inositol 3-alpha-galactosyltransferase','Galactinol synthase','NULL','An enzyme from plants involved in the formation of raffinose andstachyose (cf. EC 2.4.1.67 and EC 2.4.1.82).','UDP-alpha-D-galactose + myo-inositol = UDP + O-alpha-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.124','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.125','Sucrose--1,6-alpha-glucan 3(6)-alpha-glucosyltransferase','Water-soluble-glucan synthase','NULL','The enzyme was characterized from the dental caries bacteriumStreptococcus mutans.It transfers glucosyl residues from sucrose to either the 6- or the3-positions of glucose residues in glucans, producing a highly-branched extracellular D-glucan polymers that promote attachment ofthe bacteria to teeth.Three types of the enzyme have been described; the insoluble polymersproduced by GTF-I and GTF-SI contain 85% alpha(1-','(1) Sucrose + ((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.126','Hydroxycinnamate 4-beta-glucosyltransferase','NULL','NULL','Acts on 4-coumarate, ferulate, caffeate and sinapate, forming amixture of 4-glucosides and glucose esters (cf. EC 2.4.1.120).','UDP-glucose + trans-4-hydroxycinnamate = UDP + 4-O-beta-D-glucosyl-4-hydroxycinnamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.127','Monoterpenol beta-glucosyltransferase','NULL','NULL','(+)-neomenthol can also act as acceptor.','UDP-glucose + (-)-menthol = UDP + (-)-menthyl O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.128','Scopoletin glucosyltransferase','NULL','NULL','NULL','UDP-glucose + scopoletin = UDP + scopolin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.129','Peptidoglycan glycosyltransferase','Bactoprenyldiphospho-N-acetylmuramoyl-(N-acetyl-D-glucosaminyl)-pentapeptide:peptidoglycan N-acetylmuramoyl-N-acetyl-D-glucosaminyltransferase','NULL','The enzyme also works when the lysine residue is replaced by meso-2,6-diaminoheptanedioate (meso-2,6-diaminopimelate, A2pm) combinedwith adjacent residues through its L-center, as it is in Gram-negative and some Gram-positive organisms.The undecaprenol involved is ditrans,octacis-undecaprenol.Involved in the synthesis of cell-wall peptidoglycan.','(GlcNAc-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.130','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.131','GDP-Man:Man(3)GlcNAc(2)-PP-dolichol alpha-1,2-mannosyltransferase','NULL','NULL','The biosynthesis of asparagine-linked glycoproteins (N-linked proteinglycosylation) utilizes a dolichyl diphosphate-linked glycosyl donor,which is assembled by the series of membrane-boundglycosyltransferases that comprise the dolichol pathway.ALG11 mannosyltransferase from Saccharomyces cerevisiae carries outtwo sequential steps in the formation of the lipid-linked coreoligosaccharide, adding two mannose residues in alpha(1-','2 GDP-alpha-D-mannose + D-Man-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.132','GDP-Man:Man(1)GlcNAc(2)-PP-dolichol alpha-1,3-mannosyltransferase','GDP-Man:Man(1)GlcNAc(2)-PP-Dol alpha-1,3-mannosyltransferase','NULL','The biosynthesis of asparagine-linked glycoproteins utilizes adolichyl diphosphate-linked glycosyl donor, which is assembled by theseries of membrane-bound glycosyltransferases that comprise thedolichol pathway.Alg2 mannosyltransferase from Saccharomyces cerevisiae carries out analpha-1,3-mannosylation of D-Man-beta-(1-','GDP-D-mannose + D-Man-beta-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.133','Xylosylprotein 4-beta-galactosyltransferase','Galactosyltransferase I','Manganese.','Involved in the biosynthesis of the linkage region ofglycosaminoglycan chains as part of proteoglycan biosynthesis(chondroitin, dermatan and heparan sulfates).','UDP-alpha-D-galactose + O-beta-D-xylosyl-[protein] = UDP + 4-beta-D-galactosyl-O-beta-D-xylosyl-[protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.134','Galactosylxylosylprotein 3-beta-galactosyltransferase','Galactosyltransferase II','Manganese.','Involved in the biosynthesis of the linkage region ofglycosaminoglycan chains as part of proteoglycan biosynthesis(chondroitin, dermatan and heparan sulfates).','UDP-alpha-D-galactose + 4-beta-D-galactosyl-O-beta-D-xylosyl-[protein] =UDP + 3-beta-D-galactosyl-4-beta-D-galactosyl-O-beta-D-xylosyl-[protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.135','Galactosylgalactosylxylosylprotein 3-beta-glucuronosyltransferase','Glucuronosyltransferase I','Manganese.','Involved in the biosynthesis of the linkage region ofglycosaminoglycan chains as part of proteoglycan biosynthesis(chondroitin, dermatan and heparan sulfates).','UDP-glucuronate + 3-beta-D-galactosyl-4-beta-D-galactosyl-O-beta-D-xylosylprotein = UDP + 3-beta-D-glucuronosyl-3-beta-D-galactosyl-4-beta-D-galactosyl-O-beta-D-xylosylprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.136','Gallate 1-beta-glucosyltransferase','UDP-glucose--vanillate 1-glucosyltransferase','NULL','A number of substituted benzoic acids and, more slowly, cinnamicacids, can act as acceptors.Vanillin is the best acceptor investigated.','UDP-glucose + gallate = UDP + 1-galloyl-beta-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.137','sn-glycerol-3-phosphate 2-alpha-galactosyltransferase','Floridoside-phosphate synthase','NULL','The product is hydrolyzed by a phosphatase to floridoside (cf.EC 2.4.1.96).','UDP-alpha-D-galactose + sn-glycerol 3-phosphate = UDP + 2-(alpha-D-galactosyl)-sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.138','Mannotetraose 2-alpha-N-acetylglucosaminyltransferase','Alpha-N-acetylglucosaminyltransferase','NULL','NULL','UDP-N-acetyl-D-glucosamine + (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.139','Maltose synthase','NULL','NULL','Neither free phosphate nor maltose 1-phosphate is an intermediate inthe reaction.','2 alpha-D-glucose 1-phosphate + H(2)O = maltose + 2 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.140','Alternansucrase','Sucrose-1,6(3)-alpha-glucan 6(3)-alpha-glucosyltransferase','NULL','The product, which has quite different properties from otherdextrans, has been called alternan.','Transfers alternately an alpha-D-glucosyl residue from sucrose to the6-position and the 3-position of the non-reducing terminal residue of analpha-D-glucan, thus producing a glucan having alternating alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.141','N-acetylglucosaminyldiphosphodolichol N-acetylglucosaminyltransferase','NULL','NULL','NULL','UDP-N-acetyl-D-glucosamine + N-acetyl-D-glucosaminyl-diphosphodolichol =UDP + N,N-diacetylchitobiosyl-diphosphodolichol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.142','Chitobiosyldiphosphodolichol beta-mannosyltransferase','GDP-mannose-dolichol diphosphochitobiose mannosyltransferase','NULL','NULL','GDP-mannose + chitobiosyldiphosphodolichol = GDP + beta-1,4-D-mannosylchitobiosyldiphosphodolichol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.143','Alpha-1,6-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase','Acetylglucosaminyltransferase II','NULL','R represents the remainder of the N-linked oligosaccharide in theglycoprotein acceptor.Note that this enzyme acts after N-acetylglucosaminyltransferase I(EC 2.4.1.101) but before N-acetylglucosaminyltransferases III(EC 2.4.1.144), IV (EC 2.4.1.145), V (EC 2.4.1.155) and VI(EC 2.4.1.201).Formerly EC 2.4.1.51.','UDP-N-acetyl-D-glucosamine + 6-(alpha-D-mannosyl)-beta-D-mannosyl-R =UDP + 6-(2-(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl)-beta-D-mannosyl-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.144','Beta-1,4-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase','Beta-1,4-mannosyl-glycoprotein beta-1,4-N-acetylglucosaminyltransferase','NULL','R represents the remainder of the N-linked oligosaccharide in theglycoprotein acceptor.The action of this enzyme probably prevents further attachment ofN-acetylglucosamine residues to the growing carbohydrate chain.Formerly EC 2.4.1.51.','UDP-N-acetyl-D-glucosamine + beta-D-mannosyl-R = UDP + 4-(N-acetyl-beta-D-glucosaminyl)-beta-D-mannosyl-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.145','Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase','Alpha-1,3-mannosylglycoprotein beta-1,4-N-acetylglucosaminyltransferase','NULL','R represents the remainder of the N-linked oligosaccharide in theglycoprotein acceptor.The best acceptor for this enzyme is probably the same as thatfavored by EC 2.4.1.144.Formerly EC 2.4.1.51.','UDP-N-acetyl-D-glucosamine + 3-(2-(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl)-beta-D-mannosyl-R = UDP + 3-(2,4-bis(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl)-beta-D-mannosyl-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.146','Beta-1,3-galactosyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase','Elongation 3-beta-GalNAc-transferase','NULL','Cf. EC 2.4.1.102, EC 2.4.1.147, and EC 2.4.1.148.','UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.147','Acetylgalactosaminyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase','Core 3-beta-GlcNAc-transferase','NULL','Cf. EC 2.4.1.102, EC 2.4.1.146, and EC 2.4.1.148.','UDP-N-acetyl-D-glucosamine + N-acetyl-beta-D-galactosaminyl-R = UDP +N-acetyl-beta-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.148','Acetylgalactosaminyl-O-glycosyl-glycoprotein beta-1,6-N-acetylglucosaminyltransferase','Core 6-beta-GalNAc-transferase B','NULL','Cf. EC 2.4.1.102, EC 2.4.1.146, and EC 2.4.1.147.','UDP-N-acetyl-D-glucosamine + N-acetyl-beta-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.149','N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase','GnTE','NULL','Acts on beta-galactosyl-1,4-N-acetylglucosaminyl termini on asialo-alpha(1)-acid glycoprotein and other glycoproteins andoligosaccharides.','UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.150','N-acetyllactosaminide beta-1,6-N-acetylglucosaminyl-transferase','N-acetylglucosaminyltransferase','NULL','Acts on beta-galactosyl-1,4-N-acetylglucosaminyl termini on asialo-alpha(1)-acid glycoprotein.','UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-1,4-N-acetyl-D-glucosaminyl-R = UDP + N-acetyl-beta-D-glucosaminyl-1,6-beta-D-galactosyl-1,4-N-acetyl-D-glucosaminyl-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.151','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.152','4-galactosyl-N-acetylglucosaminide 3-alpha-L-fucosyltransferase','Galactoside 3-fucosyltransferase','NULL','Normally acts on a glycoconjugate where R (see CA line) is aglycoprotein or glycolipid.This enzyme fucosylates on O-3 of an N-acetylglucosamine that carriesa galactosyl group on O-4, unlike EC 2.4.1.65, which fucosylates onO-4 of an N-acetylglucosamine that carries a galactosyl group on O-3.','GDP-beta-L-fucose + (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.153','Dolichyl-phosphate alpha-N-acetylglucosaminyltransferase','Dolichyl-phosphate acetylglucosaminyltransferase','NULL','NULL','UDP-N-acetyl-D-glucosamine + dolichyl phosphate = UDP + dolichylN-acetyl-alpha-D-glucosaminyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.154','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.155','Alpha-1,6-mannosyl-glycoprotein 6-beta-N-acetylglucosaminyltransferase','Alpha-1,3(6)-mannosylglycoprotein beta-1,6-N-acetylglucosaminyltransferase','NULL','R represents the remainder of the N-linked oligosaccharide in theglycoprotein acceptor.','UDP-N-acetyl-D-glucosamine + 6-(2-(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl)-beta-D-mannosyl-R = UDP + 6-(2,6-bis(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl)-beta-D-mannosyl-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.156','Indolylacetyl-myo-inositol galactosyltransferase','Indol-3-ylacetyl-myo-inositol galactoside synthase','NULL','NULL','UDP-alpha-D-galactose + (indol-3-yl)acetyl-myo-inositol = UDP + 5-O-(indol-3-yl)acetyl-myo-inositol D-galactoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.157','1,2-diacylglycerol 3-glucosyltransferase','UDP-glucose-diacylglycerol glucosyltransferase','NULL','Many diacylglycerols with long-chain acyl groups can act asacceptors.','UDP-glucose + 1,2-diacyl-sn-glycerol = UDP + 3-D-glucosyl-1,2-diacyl-sn-glycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.158','13-hydroxydocosanoate 13-beta-glucosyltransferase','13-glucosyloxydocosanoate 2-beta-glucosyltransferase','NULL','13-beta-D-glucosyloxydocosanoate can also act as acceptor, leading tothe formation by Candida bogoriensis of the extracellular glycolipid,hydroxydocosanoate sophoroside diacetate.','UDP-glucose + 13-hydroxydocosanoate = UDP + 13-beta-D-glucosyloxydocosanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.159','Flavonol-3-O-glucoside L-rhamnosyltransferase','UDP-rhamnose:flavonol 3-O-glucoside rhamnosyltransferase','NULL','Converts flavonol 3-O-glucosides to 3-O-rutinosides.Also acts, more slowly, on rutin, quercetin 3-O-galactoside andflavonol 3-O-rhamnosides.','UDP-L-rhamnose + a flavonol 3-O-D-glucoside = UDP + a flavonol 3-O-(beta-L-rhamnosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.160','Pyridoxine 5-O-beta-D-glucosyltransferase','UDP-glucose-pyridoxine glucosyltransferase','NULL','4-deoxypyridoxine and pyridoxamine can also act as acceptors, moreslowly.','UDP-glucose + pyridoxine = UDP + 5-O-beta-D-glucosylpyridoxine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.161','Oligosaccharide 4-alpha-D-glucosyltransferase','Alpha-1,4-transglucosylase','NULL','The enzyme acts on amylose, amylopectin, glycogen andmaltooligosaccharides.No detectable free glucose is formed, indicating the enzyme does notact as a hydrolase.The enzyme from the bacterium Cellvibrio japonicus has the highestactivity with maltotriose as a donor, and also accepts maltose, whilethe enzyme from amoeba does not accept maltose.Oligosaccharides with 1-','Transfers the non-reducing terminal alpha-D-glucose residue from a(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.162','Aldose beta-D-fructosyltransferase','NULL','NULL','NULL','Alpha-D-aldosyl(1) beta-D-fructoside + D-aldose(2) = D-aldose(1) + alpha-D-aldosyl(2) beta-D-fructoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.163','Beta-galactosyl-N-acetylglucosaminylgalactosylglucosyl-ceramide beta-1,3-acetylglucosaminyltransferase','Poly-N-acetyllactosamine extension enzyme','Manganese.','NULL','UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.164','Galactosyl-N-acetylglucosaminylgalactosylglucosyl-ceramide beta-1,6-N-acetylglucosaminyltransferase','NULL','Manganese.','NULL','UDP-N-acetyl-D-glucosamine + D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.165','N-acetylneuraminylgalactosylglucosylceramide beta-1,4-N-acetylgalactosaminyltransferase','NULL','Manganese.','Only substances containing sialic acid residues can act as acceptors;bovine fetuin is the best acceptor tested.','UDP-N-acetyl-alpha-D-galactosamine + alpha-N-acetylneuraminyl-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.166','Raffinose--raffinose alpha-galactosyltransferase','NULL','NULL','The 3(F) position of raffinose can also act as galactosyl acceptor.Involved in the accumulation of the tetrasaccharides lychnose andisolychnose in the leaves of Cerastium arvense and other plants ofthe family Caryophyllaceae during late autumn.','2 raffinose = 1(F)-alpha-D-galactosylraffinose + sucrose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.167','Sucrose 6(F)-alpha-galactosyltransferase','NULL','NULL','Involved in the synthesis of the trisaccharide planteose and higheranalogs in the seeds of Plantago and Sesamum species.','UDP-alpha-D-galactose + sucrose = UDP + 6(F)-alpha-D-galactosylsucrose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.168','Xyloglucan 4-glucosyltransferase','NULL','NULL','In association with EC 2.4.2.39, brings about the synthesis ofxyloglucan.Concurrent transfers of glucose and xylose are essential for thissynthesis.Not identical with EC 2.4.1.12.','Transfers a beta-D-glucosyl residue from UDP-glucose on to a glucoseresidue in xyloglucan, forming a beta-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.169','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.170','Isoflavone 7-O-glucosyltransferase','NULL','NULL','The 4-methoxy isoflavones biochanin A and formononetin and, moreslowly, the 4-hydroxy isoflavones genistein and daidzein, can act asacceptors.The enzyme does not act on isoflavanones, flavones, flavanones,flavanols or coumarins.','UDP-glucose + an isoflavone = UDP + an isoflavone 7-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.171','Methyl-ONN-azoxymethanol beta-D-glucosyltransferase','Cycasin synthase','NULL','Brings about the biosynthesis of the toxic substance cycasin in theleaves of Japanese cycad, Cycas revoluta.','UDP-glucose + CH(3)-N(O)=N-CH(2)OH = UDP + cycasin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.172','Salicyl-alcohol beta-D-glucosyltransferase','NULL','NULL','NULL','UDP-glucose + salicyl alcohol = UDP + salicin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.173','Sterol 3-beta-glucosyltransferase','NULL','NULL','Not identical with EC 2.4.1.192 or EC 2.4.1.193.','UDP-glucose + a sterol = UDP + a sterol 3-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.174','Glucuronylgalactosylproteoglycan 4-beta-N-acetylgalactosaminyltransferase','Glucuronylgalactosylproteoglycan beta-1,4-N-acetylgalactosaminyltransferase','NULL','Involved in the biosynthesis of chondroitin sulfate.Key enzyme activity for the initiation of chondroitin and dermatansulfates, transferring GalNAc to the GlcA-Gal-Gal-Xyl-Ser core.','UDP-N-acetyl-D-galactosamine + beta-D-glucuronyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.175','Glucuronosyl-N-acetylgalactosaminyl-proteoglycan 4-beta-N-acetylgalactosaminyltransferase','Chondroitin synthase','NULL','Involved in the biosynthesis of chondroitin sulfate.The human form of this enzyme is a bifunctional glycosyltransferase,which also has the EC 2.4.1.226 activity required for the synthesisof the chondroitin sulfate disaccharide repeats.Similar chondroitin synthase copolymerases can be found inPasteurella multocida and Escherichia coli.','UDP-N-acetyl-alpha-D-galactosamine + beta-D-glucuronosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.176','Gibberellin beta-D-glucosyltransferase','NULL','NULL','Acts on the plant hormone gibberellin GA(3) and related compounds.','UDP-glucose + gibberellin = UDP + gibberellin 2-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.177','Cinnamate beta-D-glucosyltransferase','NULL','NULL','4-coumarate, 2-coumarate, benzoate, feruloate and caffeate can alsoact as acceptors, more slowly.Involved in the biosynthesis of chlorogenic acid in the root of thesweet potato.','UDP-glucose + trans-cinnamate = UDP + trans-cinnamoyl beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.178','Hydroxymandelonitrile glucosyltransferase','Cyanohydrin glucosyltransferase','NULL','3,4-dihydroxymandelonitrile can also act as acceptor.','UDP-glucose + 4-hydroxymandelonitrile = UDP + taxiphyllin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.179','Lactosylceramide beta-1,3-galactosyltransferase','NULL','NULL','R may be an oligosaccharide or a glycolipid.Lactose can also act as acceptor, more slowly.Involved in the elongation of oligosaccharide chains, especially inglycolipids.','UDP-alpha-D-galactose + D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.180','Lipopolysaccharide N-acetylmannosaminouronosyltransferase','NULL','NULL','Involved in the biosynthesis of common antigen in Enterobacteriaceae.','UDP-N-acetyl-alpha-D-mannosaminouronate + N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol = UDP + N-acetyl-beta-D-mannosaminouronyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.181','Hydroxyanthraquinone glucosyltransferase','NULL','NULL','A range of anthraquinones and some flavones can act as acceptors;best substrates are emodin, anthrapurpurin, quinizarin, 2,6-dihydroxyanthraquinone and 1,8-dihydroxyanthraquinone.','UDP-glucose + an hydroxyanthraquinone = UDP + a glucosyloxyanthraquinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.182','Lipid-A-disaccharide synthase','NULL','NULL','Involved with EC 2.3.1.129 and EC 2.7.1.130 in the biosynthesis ofthe phosphorylated glycolipid, lipid A, in the outer membrane ofEscherichia coli.','UDP-2,3-bis(3-hydroxytetradecanoyl)glucosamine + 2,3-bis(3-hydroxytetradecanoyl)-beta-D-glucosaminyl 1-phosphate = UDP + 2,3-bis(3-hydroxytetradecanoyl)-D-glucosaminyl-1,6-beta-D-2,3-bis(3-hydroxytetradecanoyl)-beta-D-glucosaminyl 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.183','Alpha-1,3-glucan synthase','NULL','NULL','A glucan primer is needed to begin the reaction, which brings aboutelongation of the glucan chains.','UDP-glucose + (alpha-D-glucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.184','Galactolipid galactosyltransferase','DGDG synthase','NULL','By further transfers of galactosyl residues to thedigalactosyldiacylglycerol, trigalactosyldiacylglycerol andtetragalactosyldiacylglycerol are also formed.Was originally thought to be the major enzyme involved in theproduction of digalactosyldiacylglycerol in plants as it masked theeffect of the true enzyme (EC 2.4.1.241).Its activity is localized to chloroplast envelope membranes, but itdoes not contribute to net galactolipid synthesis in plants.','2 3-(beta-D-galactosyl)-1,2-diacyl-sn-glycerol = 3-(alpha-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.185','Flavanone 7-O-beta-glucosyltransferase','NULL','NULL','Naringenin and hesperetin can act as acceptors.No action on flavones or flavonols.','UDP-glucose + a flavanone = UDP + a flavanone 7-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.186','Glycogenin glucosyltransferase','Glycogenin','Manganese.','The first reaction of this enzyme is to catalyze its ownglucosylation, normally at a specific Tyr of the protein if thisgroup is free; when the Tyr is replaced by Thr or Phe, the enzymesself-glucosylation activity is lost but its intermoleculartransglucosylation ability remains.It continues to glucosylate an existing glucosyl group until a lengthof about 5-13 residues has been formed.Further lengthening of the glycogen chain is then carried out byEC 2.4.1.11.Not highly specific for the donor, using UDP-xylose in addition toUDP-glucose (although not glucosylating or xylosylating a xylosylgroup so added).It can also use CDP-glucose and TDP-glucose, but not ADP-glucose orGDP-glucose.Similarly it is not highly specific for the acceptor, using water(i.e. hydrolyzing UDP-glucose) among others.Various forms of the enzyme exist, and different forms predominate indifferent organs.Thus primate liver contains glycogenin-2, of molecular mass 66 kDa,whereas the more widespread form is glycogenin-1, with a molecularmass of 38 kDa.Formerly EC 2.4.1.112.','UDP-alpha-D-glucose + glycogenin = UDP + alpha-D-glucosylglycogenin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.187','N-acetylglucosaminyldiphosphoundecaprenol N-acetyl-beta-D-mannosaminyltransferase','NULL','NULL','Involved in the biosynthesis of teichoic acid linkage units inbacterial cell walls.','UDP-N-acetyl-alpha-D-mannosamine + N-acetyl-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol = UDP + N-acetyl-beta-D-mannosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.188','N-acetylglucosaminyldiphosphoundecaprenol glucosyltransferase','NULL','NULL','NULL','UDP-alpha-D-glucose + N-acetyl-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol = UDP + beta-D-glucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.189','Luteolin 7-O-glucuronosyltransferase','NULL','NULL','NULL','UDP-glucuronate + luteolin = UDP + luteolin 7-O-beta-D-glucuronide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.190','Luteolin-7-O-glucuronide 2-O-glucuronosyltransferase','LMT','NULL','NULL','UDP-glucuronate + luteolin 7-O-beta-D-glucuronide = UDP + luteolin 7-O-(beta-D-glucuronosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.191','Luteolin-7-O-diglucuronide 4-O-glucuronosyltransferase','NULL','NULL','NULL','UDP-glucuronate + luteolin 7-O-(beta-D-glucuronosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.192','Nuatigenin 3-beta-glucosyltransferase','NULL','NULL','Some other sapogenins can act as glucosyl acceptors.Involved in the biosynthesis of plant saponins.Not identical with EC 2.4.1.173 or EC 2.4.1.193.','UDP-glucose + (20S,22S,25S)-22,25-epoxyfurost-5-ene-3-beta,26-diol =UDP + (20S,22S,25S)-22,25-epoxyfurost-5-ene-3-beta,26-diol 3-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.193','Sarsapogenin 3-beta-glucosyltransferase','NULL','NULL','Specific to 5-beta-spirostanols.Involved in the biosynthesis of plant saponins.Not identical with EC 2.4.1.173 or EC 2.4.1.192.','UDP-glucose + (25S)-5-beta-spirostan-3-beta-ol = UDP + (25S)-5-beta-spirostan-3-beta-ol 3-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.194','4-hydroxybenzoate 4-O-beta-D-glucosyltransferase','NULL','NULL','NULL','UDP-glucose + 4-hydroxybenzoate = UDP + 4-(beta-D-glucosyloxy)benzoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.195','N-hydroxythioamide S-beta-glucosyltransferase','Desulfoglucosinolate-uridine diphosphate glucosyltransferase','NULL','Involved with EC 2.8.2.24 in the biosynthesis of thioglucosides incruciferous plants.','UDP-glucose + N-hydroxy-2-phenylethanethioamide = UDP +desulfoglucotropeolin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.196','Nicotinate glucosyltransferase','NULL','NULL','NULL','UDP-glucose + nicotinate = UDP + N-glucosylnicotinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.197','High-mannose-oligosaccharide beta-1,4-N-acetylglucosaminyltransferase','NULL','NULL','The activity of the intersecting mannose residue as acceptor isdependent on two other mannose residues attached by alpha-1,3 andalpha-1,6 links.','Transfers an N-acetyl-D-glucosamine residue from UDP-N-acetyl-D-glucosamine to the 4-position of a mannose linked alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.198','Phosphatidylinositol N-acetylglucosaminyltransferase','UDP-N-acetyl-D-glucosamine:phosphatidylinositol N-acetyl-D-glucosaminyltransferase','NULL','Involved in the first step of glycosylphosphatidylinositol (GPI)anchor formation in all eukaryotes.In mammalian cells, the enzyme is composed of at least five subunits(PIG-A, PIG-H, PIG-C, GPI1 and PIG-P).PIG-A subunit is the catalytic subunit.In some species, the long-chain acyl groups of the phosphatidyl groupare partly replaced by long-chain alkyl or alk-1-enyl groups.','UDP-N-acetyl-D-glucosamine + 1-phosphatidyl-1D-myo-inositol = UDP +6-(N-acetyl-alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.199','Beta-mannosylphosphodecaprenol--mannooligosaccharide6-mannosyltransferase','NULL','NULL','Involved in the formation of mannooligosaccharides in the membrane ofMycobacterium smegmatis.','Beta-D-mannosylphosphodecaprenol + (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.200','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.201','Alpha-1,6-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase','GnTVI','NULL','R represents the remainder of the N-linked oligosaccharide in theglycoprotein acceptor.','UDP-N-acetyl-D-glucosamine + 2,6-bis(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl-R = UDP + 2,4,6-tris(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.202','2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one 2-D-glucosyltransferase','DIMBOA glucosyltransferase','NULL','The enzyme is involved in the detoxification of the benzoxazinoidsDIBOA (2,4-dihydroxy-2H-1,4-benzoxazin-3(4H)-one) and DIMBOA (2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one) which are stored asthe respective non-toxic glucosides in the vacuoles in some plants,most commonly from the family of Poaceae (grasses).Benzoxazinoids are known to exhibit antimicrobial, antifeedant,and antiinsecticidal effects and are involved in the interaction ofplants with other plants, insects, or microorganisms.','(1) UDP-alpha-D-glucose + 2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one = UDP + (2R)-4-hydroxy-7-methoxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside.(2) UDP-alpha-D-glucose + 2,4-dihydroxy-2H-1,4-benzoxazin-3(4H)-one =UDP + (2R)-4-hydroxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.203','Trans-zeatin O-beta-D-glucosyltransferase','Uridine diphosphoglucose-zeatin O-glucosyltransferase','NULL','Unlike EC 2.4.1.215, UDP-xylose can also act as donor (cf.EC 2.4.2.40).','UDP-glucose + trans-zeatin = UDP + O-beta-D-glucosyl-trans-zeatin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.204','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.205','Galactogen 6-beta-galactosyltransferase','1,6-D-galactosyltransferase','NULL','Galactogen from Helix pomatia is the most effective acceptor.','UDP-beta-D-galactose + galactogen = UDP + (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.206','Lactosylceramide 1,3-N-acetyl-beta-D-glucosaminyltransferase','NULL','NULL','NULL','UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.207','Xyloglucan:xyloglucosyl transferase','Endo-xyloglucan transferase','NULL','Does not use cello-oligosaccharides as either donor or acceptor.','Breaks a beta-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.208','Diglucosyl diacylglycerol synthase','DGlcDAG synthase','Magnesium.','NULL','UDP-glucose + 1,2-diacyl-3-O-(alpha-D-glucopyranosyl)-sn-glycerol = 1,2-diacyl-3-O-(alpha-D-glucopyranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.209','Cis-p-coumarate glucosyltransferase','NULL','NULL','Cis-caffeic acid also serves as a glucosyl acceptor with the enzymefrom Sphagnum fallax kinggr.The corresponding trans-isomers are not substrates.','UDP-glucose + cis-p-coumarate = 4-O-beta-D-glucosyl-cis-p-coumarate +UDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.210','Limonoid glucosyltransferase','LGTase','NULL','The enzyme purified from navel orange albedo tissue also acts on therelated tetranortriterpenoid nomilin.','UDP-glucose + limonin = glucosyl-limonin + UDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.211','1,3-beta-galactosyl-N-acetylhexosamine phosphorylase','Galacto-N-biose phosphorylase','NULL','Reaction also occurs with beta-D-galactopyranosyl-(1-','Beta-D-galactopyranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.212','Hyaluronan synthase','HAS','Magnesium.','Adds GlcNAc to nascent hyaluronan when the non-reducing end is GlcA,but it adds GlcA when the non-reducing end is GlcNAc.Highly specific for UDP-GlcNAc and UDP-GlcA; no copolymerization isobserved if either is replaced by UDP-Glc, UDP-Gal, UDP-GalNAc orUDP-GalA.','(1) UDP-alpha-N-acetyl-D-glucosamine + beta-D-glucuronosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.213','Glucosylglycerol-phosphate synthase','GG-phosphate synthase','NULL','Acts with EC 3.1.3.69 to form glucosylglycerol, an osmolyte thatendows cyanobacteria with resistance to salt.','ADP-glucose + sn-glycerol 3-phosphate = 2-(beta-D-glucosyl)-sn-glycerol3-phosphate + ADP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.214','Glycoprotein 3-alpha-L-fucosyltransferase','GDP-fucose:beta-N-acetylglucosamine (Fuc to (Fuc-alpha-1-','Manganese.','The enzyme transfers to N-linked oligosaccharide structures(N-glycans), generally with a specificity for N-glycans with oneunsubstituted non-reducing terminal GlcNAc residue.This enzyme catalyzes a reaction similar to that of EC 2.4.1.68,but transferring the L-fucosyl group from GDP-beta-L-fucose to forman alpha-1,3-linkage rather than an alpha-1,6-linkage.The N-glycan products of this enzyme are present in plants, insectsand some other invertebrates (e.g., Schistosoma, Haemonchus,Lymnaea).','GDP-L-fucose + N(4)-(N-acetyl-beta-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.215','Cis-zeatin O-beta-D-glucosyltransferase','NULL','NULL','The enzyme from maize can use cis-zeatin and UDPglucose assubstrates, but not cis-ribosylzeatin, trans-zeatin or trans-ribosylzeatin.Unlike EC 2.4.1.203 UDP-xylose cannot act as a donor.','UDP-glucose + cis-zeatin = UDP + O-beta-D-glucosyl-cis-zeatin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.216','Trehalose 6-phosphate phosphorylase','Trehalose 6-phosphate:phosphate beta-D-glucosyltransferase','NULL','The enzyme from Lactococcus lactis is specific for trehalose6-phosphate.Differs from EC 2.4.1.64 in that trehalose is not a substrate.','Alpha,alpha-trehalose 6-phosphate + phosphate = glucose 6-phosphate +beta-D-glucose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.217','Mannosyl-3-phosphoglycerate synthase','MPG synthase','Magnesium.','The enzyme is absolutely specific for GDP-mannose and3-phosphoglycerate, and transfers the mannosyl group with retentionof configuration.In the hyperthermophilic archaeon Pyrococcus horikoshii,the mannosyl-3-phosphoglycerate formed is subsequentlydephosphorylated by a specific phosphatase, EC 3.1.3.70, producingmannosylglycerate.','GDP-mannose + 3-phospho-D-glycerate = GDP + 2-(alpha-D-mannosyl)-3-phosphoglycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.218','Hydroquinone glucosyltransferase','Arbutin synthase','NULL','Hydroquinone is the most effective acceptor, but over 40 phenoliccompounds are also glucosylated, but at lower rates.','UDP-glucose + hydroquinone = UDP + hydroquinone-O-beta-D-glucopyranoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.219','Vomilenine glucosyltransferase','UDPG:vomilenine 21-beta-D-glucosyltransferase','NULL','The indole alkaloid raucaffricine accumulates during the culture ofRauvolfia cell suspensions.','UDP-glucose + vomilenine = UDP + raucaffricine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.220','Indoxyl-UDPG glucosyltransferase','Indoxyl-UDPG-glucosyltransferase','NULL','Also acts to a limited extent on 4-, 5-, 6- and 7-hydroxyindole.After enzymic or chemical hydrolysis, indican forms indoxyl, which,in turn, is converted in the presence of oxygen to the dye indigo.','UDP-glucose + indoxyl = UDP + indican.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.221','Peptide-O-fucosyltransferase','GDP-fucose protein O-fucosyltransferase','NULL','Involved in the biosynthesis of O-fucosylated epidermal growth factor(EGF) and thrombospondin type 1 repeats.The attachment of O-linked fucose to serine or threonine occurs onEGF domains within the sequence Cys-Xaa-Xaa-Gly-Gly-Ser/Thr-Cys.','Transfers an alpha-L-fucosyl residue from GDP-beta-L-fucose to the serinehydroxy group of a protein acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.222','O-fucosylpeptide 3-beta-N-acetylglucosaminyltransferase','O-fucosylpeptide beta-1,3-N-acetylglucosaminyltransferase','NULL','O-fucosylpeptide 3-beta-N-acetylglucosaminyltransferases are theproducts of fringe genes.O-linked fucose is an unusual form of glycosylation where the fucoseis attached directly to proteins through the hydroxy groups of Ser orThr residues.','Transfers a beta-D-GlcNAc residue from UDP-D-GlcNAc to the fucose residueof a fucosylated protein acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.223','Glucuronyl-galactosyl-proteoglycan 4-alpha-N-acetylglucosaminyltransferase','Alpha-1,4-N-acetylglucosaminyltransferase','NULL','Enzyme involved in the initiation of heparin and heparan sulfatesynthesis, transferring GlcNAc to the (GlcA-Gal-Gal-Xyl-)Ser core.Apparently products of both the human EXTL2 and EXTL3 genes cancatalyze this reaction.In Caenorhabditis elegans, the product of the rib-2 gene displaysthis activity as well as that of EC 2.4.1.224.','UDP-N-acetyl-D-glucosamine + beta-D-glucuronosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.224','Glucuronosyl-N-acetylglucosaminyl-proteoglycan 4-alpha-N-acetylglucosaminyltransferase','Alpha-N-acetylglucosaminyltransferase II','NULL','Involved in the biosynthesis of heparin and heparan sulfate.Some forms of the enzyme from human (particularly the enzyme complexencoded by the EXT1 and EXT2 genes) act as bifunctionalglycosyltransferases, which also have the EC 2.4.1.225 activityrequired for the synthesis of the heparan sulfate disacchariderepeats.Other human forms of this enzyme (e.g. the product of the EXTL1 gene)have only the 4-alpha-N-acetylglucosaminyltransferase activity.In Caenorhabditis elegans, the product of the rib-2 gene displays theactivities of this enzyme as well as EC 2.4.1.223.','UDP-N-acetyl-D-glucosamine + beta-D-glucuronosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.225','N-acetylglucosaminyl-proteoglycan 4-beta-glucuronosyltransferase','Heparan glucuronyltransferase II','NULL','Involved in the biosynthesis of heparin and heparan sulfate.Some forms of the human enzyme (particularly the enzyme complexencoded by the EXT1 and EXT2 genes) act as bifunctionalglycosyltransferases, which also have the EC 2.4.1.244 activityrequired for the synthesis of the heparan sulfate disacchariderepeats.','UDP-alpha-D-glucuronate + N-acetyl-alpha-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.226','N-acetylgalactosaminyl-proteoglycan 3-beta-glucuronosyltransferase','Chondroitin glucuronyltransferase II','NULL','Involved in the biosynthesis of chondroitin and dermatan sulfate.The human chondroitin synthetase is a bifunctionalglycosyltransferase, which also has the EC 2.4.1.175 activityrequired for the synthesis of the chondroitin sulfate disacchariderepeats.Similar chondroitin synthase copolymerases can be found inPasteurella multocida and Escherichia coli.There is also another human protein with apparently only the 3-beta-glucuronosyltransferase activity.','UDP-alpha-D-glucuronate + N-acetyl-beta-D-galactosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.227','Undecaprenyldiphospho-muramoylpentapeptide beta-N-acetylglucosaminyltransferase','MurG transferase','NULL','The enzyme also works when the lysine residue is replaced by meso-2,6-diaminoheptanedioate (meso-2,6-diaminopimelate, A2pm) combinedwith adjacent residues through its L-center, as it is in Gram-negative and some Gram-positive organisms.The undecaprenol involved is ditrans,octacis-undecaprenol.','UDP-N-acetylglucosamine + Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol = UDP + GlcNAc-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.228','Lactosylceramide 4-alpha-galactosyltransferase','Gal-beta-1-4Glc-beta-1-Cer alpha-1,4-galactosyltransferase','NULL','NULL','UDP-alpha-D-galactose + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.229','[Skp1-protein]-hydroxyproline N-acetylglucosaminyltransferase','Skp1-HyPro GlcNAc-transferase','Divalent metal cation.','Skp1 is a cytoplasmic and nuclear protein required for theubiquitination of cell cycle regulatory proteins and transcriptionalfactors.In Dictyostelium Skp1 is modified by the linear pentasaccharide Gal-alpha-1-6Gal-alpha-1-L-Fuc-alpha-1-2Gal-beta-1-3GlcNAc, which isattached to a hydroxyproline residue at position 143.This enzymecatalyzes the first step in the building up of the pentasaccharide byattaching an N-acetylglucosaminyl group to the hydroxyprolineresidue.','UDP-N-acetyl-alpha-D-glucosamine + [Skp1-protein]-trans-4-hydroxy-L-proline = UDP + [Skp1-protein]-O-(N-acetyl-alpha-D-glucosaminyl)-trans-4-hydroxy-L-proline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.230','Kojibiose phosphorylase','NULL','NULL','The enzyme from Thermoanaerobacter brockii can act with alpha-1,2-oligoglucans, such as selaginose, as substrate, but more slowly.The enzyme is inactive when dissaccharides with linkages other thanalpha-1,2 linkages, such as sophorose, trehalose, neotrehalose,nigerose, laminaribiose, maltose, cellobiose, isomaltose,gentiobiose, sucrose and lactose, are used as substrates.','2-alpha-D-glucosyl-D-glucose + phosphate = D-glucose + beta-D-glucose1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.231','Alpha,alpha-trehalose phosphorylase (configuration-retaining)','Trehalose phosphorylase','NULL','Unlike EC 2.4.1.64, this enzyme retains its anomeric configuration.Vanadate is a strong competitive inhibitor of this reversiblereaction.','Alpha,alpha-trehalose + phosphate = alpha-D-glucose + alpha-D-glucose1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.232','Initiation-specific alpha-1,6-mannosyltransferase','Alpha-1,6-mannosyltransferase','Manganese.','In Saccharomyces cerevisiae, this enzyme catalyzes an essential stepin the outer chain elongation of N-linked oligosaccharides.Man(8)GlcNAc and Man(9)GlcNAc are equally good substrates.','Transfers an alpha-D-mannosyl residue from GDP-mannose into lipid-linkedoligosaccharide, forming an alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.233','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.234','Kaempferol 3-O-galactosyltransferase','F3GalTase','NULL','Acts on the endogenous flavonols kaempferol and quercetin, to alesser extent on myricetin and fisetin, and weakly on galangin andisorhamnetin.The reaction can occur equally well in both directions.','UDP-alpha-D-galactose + kaempferol = UDP + kaempferol 3-O-beta-D-galactoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.235','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.236','Flavanone 7-O-glucoside 2-O-beta-L-rhamnosyltransferase','1-','NULL','Acts on the 7-O-glucoside of naringenin and hesperetin, also theflavone 7-O-glucosides of luteolin and apigenin.','UDP-L-rhamnose + a flavanone 7-O-glucoside = UDP + a flavanone 7-O-(beta-L-rhamnosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.237','Flavonol 7-O-beta-glucosyltransferase','UDP-glucose:flavonol 7-O-glucosyltransferase','NULL','Acts on the flavonols gossypetin (8-hydroxyquercetin) and to a lesserextent on quercetin, kaempferol and myricetin.','UDP-glucose + a flavonol = UDP + a flavonol 7-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.238','Delphinidin 3,5-di-O-glucoside 3-O-glucosyltransferase','3GT','NULL','Isolated from the plant Gentiana triflora (clustered gentian).','UDP-alpha-D-glucose + delphinidin 3,5-di-O-beta-D-glucoside = UDP +delphinidin 3,3,5-tri-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.239','Flavonol-3-O-glucoside glucosyltransferase','NULL','NULL','One of three specific glucosyltransferases in pea (Pisum sativum)that successively add a beta-D-glucosyl group first to O-3 ofkaempferol, and then to O-2 of the previously added glucosyl groupgiving the 3-O-sophoroside and then the 3-O-sophorotrioside (see alsoEC 2.4.1.91 and EC 2.4.1.240).TDP-glucose can replace UDP-glucose as the glucose donor but thereaction proceeds more slowly.','UDP-glucose + a flavonol 3-O-beta-D-glucoside = UDP + a flavonol 3-O-beta-D-glucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.240','Flavonol-3-O-glycoside glucosyltransferase','NULL','NULL','One of three specific glucosyltransferases in pea (Pisum sativum)that successively add a beta-D-glucosyl group first to O-3 ofkaempferol, and then to O-2 of the previously added glucosyl groupgiving the 3-O-sophoroside and then the 3-O-sophorotrioside (see alsoEC 2.4.1.91 and EC 2.4.1.239).','UDP-glucose + a flavonol 3-O-beta-D-glucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.241','Digalactosyldiacylglycerol synthase','DGDG synthase','Magnesium.','Diacylglycerol cannot serve as an acceptor molecule forgalactosylation as in the reaction catalyzed by EC 2.4.1.46.When phosphate is limiting, phospholipids in plant membranes arereduced but these are replaced, at least in part, by the glycolipidsdigalactosyldiacylglycerol (DGDG) and sulfoquinovosyldiacylglycerol.While both DGD1 and DGD2 are increased under phosphate-limitingconditions, DGD2 does not contribute significantly under optimalgrowth conditions.DGD2 is responsible for the synthesis of DGDG molecular species thatare rich in C16 fatty acids at sn-1 of diacylglycerol whereas DGD1leads to molecular species rich in C18 fatty acids.The enzyme has been localized to the outer side of chloroplastenvelope membranes.','UDP-alpha-D-galactose + 3-(beta-D-galactosyl)-1,2-diacyl-sn-glycerol =UDP + 3-(alpha-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.242','NDP-glucose--starch glucosyltransferase','GBSS','NULL','Unlike EC 2.4.1.11 and EC 2.4.1.21 which use UDP-glucose and ADP-glucose, respectively, this enzyme can use either UDP- or ADP-glucose.Mutants that lack the Wx (waxy) allele cannot produce this enzyme,which plays an important role in the normal synthesis of amylose.In such mutants, only amylopectin is produced in the endosperm orpollen.','NDP-glucose + (1,4-alpha-D-glucosyl)(n) = NDP + (1,4-alpha-D-glucosyl)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.243','6(G)-fructosyltransferase','6(G)-FFT','NULL','This enzyme catalyzes the transfer of the terminal (2-','(1-beta-D-fructofuranosyl-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.244','N-acetyl-beta-glucosaminyl-glycoprotein 4-beta-N-acetylgalactosaminyltransferase','Beta-1,4-N-acetylgalactosaminyltransferase','NULL','The enzyme from human can transfer N-acetyl-D-galactosamine (GalNAc)to N-glycan and O-glycan substrates that have N-acetyl-D-glucosamine(GlcNAc) but not D-glucuronic acid (GlcUA) at their non-reducing end.The N-acetyl-beta-D-glucosaminyl group is normally on a coreoligosaccharide although benzyl glycosides have been used in enzyme-characterization experiments.Some glycohormones, e.g. lutropin and thyrotropin contain theN-glycan structure containing the N-acetyl-beta-D-galactosaminyl-(1-','UDP-N-acetyl-alpha-D-galactosamine + N-acetyl-beta-D-glucosaminyl group =UDP + N-acetyl-beta-D-galactosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.245','Alpha,alpha-trehalose synthase','Trehalose synthase','Magnesium.','The enzyme-catalyzed reaction is reversible.In the reverse direction to that shown above, the enzyme is specificfor alpha,alpha-trehalose as substrate, as it cannot use alpha- orbeta-paranitrophenyl glucosides, maltose, sucrose, lactose orcellobiose.While the enzymes from the thermophilic bacterium Rubrobacterxylanophilus and the hyperthermophilic archaeon Pyrococcus horikoshiican use ADP-, UDP- and GDP-alpha-D-glucose to the same extent, thatfrom the hyperthermophilic archaeon Thermococcus litoralis has amarked preference for ADP-alpha-D-glucose and that from thehyperthermophilic archaeon Thermoproteus tenax has a markedpreference for UDP-alpha-D-glucose.Formerly EC 2.4.1.n1.','NDP-alpha-D-glucose + D-glucose = alpha,alpha-trehalose + NDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.246','Mannosylfructose-phosphate synthase','Mannosylfructose-6-phosphate synthase','Manganese or magnesium.','GDP-mannose can be replaced by ADP-mannose but with a concomitantdecrease in activity.The product of this reaction is dephosphorylated by EC 3.1.3.79 toform the non-reducing disaccharide mannosylfructose, which is themajor endogenous osmolyte produced by several alpha-proteobacteria inresponse to osmotic stress.The F in the product name is used to indicate that the fructoseresidue of sucrose carries the substituent.','GDP-mannose + D-fructose 6-phosphate = GDP + beta-D-fructofuranosyl-alpha-D-mannopyranoside 6(F)-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.247','Beta-D-galactosyl-(1-','D-galactosyl-beta-1-','NULL','The enzyme from Clostridium phytofermentans is also active towardtoward beta-D-galactosyl derivatives of L-mannose, L-lyxose,D-glucose, 2-deoxy-D-glucose, and D-galactose in this order.Differs from 1,3-beta-galactosyl-N-acetylhexosamine phosphorylase(EC 2.4.1.211) in being active toward L-rhamnose and inactive towardN-acetyl hexosamine derivatives.','Beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.248','Cycloisomaltooligosaccharide glucanotransferase','NULL','NULL','Specific for (1-','Cyclizes part of a (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.249','Delphinidin 3,5-O-glucosyltransferase','UDP-glucose:anthocyanin 3,5-O-glucosyltransferase','NULL','Ternatins are a group of polyacetylated delphinidin glucosides thatconfer blue color to the petals of butterfly pea (Clitoria ternatea).This enzyme catalyzes two reactions in the biosynthesis of ternatinC5: the conversion of delphinidin 3-O-(6-O-malonyl)-beta-D-glucoside to delphinidin 3-O-(6-O-malonyl)-beta-D-glucoside-3-O-beta-D-glucoside, followed by the conversion of the later to ternatinC5, by transferring two glucosyl groups in a stepwise manner.','2 UDP-glucose + delphinidin 3-O-(6-O-malonyl)-beta-D-glucoside = 2 UDP+ delphinidin 3-O-(6-O-malonyl)-beta-D-glucoside-3,5-di-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.250','D-inositol-3-phosphate glycosyltransferase','Mycothiol glycosyltransferase','NULL','The enzyme, which belongs to the GT-B fold superfamily, catalyzes thefirst dedicated reaction in the biosynthesis of mycothiol.The substrate was initially believed to be inositol, but eventuallyshown to be D-myo-inositol 3-phosphate.A substantial conformational change occurs upon UDP binding, whichgenerates the binding site for D-myo-inositol 3-phosphate.Formerly EC 2.4.1.n3.','UDP-N-acetyl-D-glucosamine + 1D-myo-inositol 3-phosphate = UDP + 1-O-(2-acetamido-2-deoxy-alpha-D-glucopyranosyl)-1D-myo-inositol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.251','GlcA-beta-(1-','NULL','NULL','The enzyme is involved in the biosynthesis of the exopolysaccharidexanthan.','GDP-mannose + GlcA-beta-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.252','GDP-mannose:cellobiosyl-diphosphopolyprenol alpha-mannosyltransferase','NULL','NULL','In the bacterium Gluconacetobacter xylinus (previously known asAcetobacter xylinum) the enzyme is involved in the biosynthesis ofthe exopolysaccharide acetan.In Xanthomonas campestris the enzyme is involved in the biosynthesisof the exopolysaccharide xanthan.','GDP-mannose + D-Glc-beta-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.253','Baicalein 7-O-glucuronosyltransferase','UBGAT','NULL','The enzyme is specific for UDP-D-glucuronate as a sugar donor andflavones with substitution ortho- to the 7-OH group such as baicalein(6-OH), scutellarein (6-OH) and wogonin (8-OMe).','UDP-D-glucuronate + baicalein = UDP + baicalin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.254','Cyanidin-3-O-glucoside 2-O-glucuronosyltransferase','NULL','NULL','The enzyme is highly specific for cyanidin 3-O-glucosides and UDP-alpha-D-glucuronate.Involved in the production of glucuronosylated anthocyanins that arethe origin of the red coloration of flowers of Bellis perennis.','UDP-alpha-D-glucuronate + cyanidin 3-O-beta-D-glucoside = UDP + cyanidin3-O-(2-O-beta-D-glucuronosyl)-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.255','Protein O-GlcNAc transferase','O-linked N-acetylglucosaminyltransferase','NULL','Within higher eukaryotes post-translational modification of proteinserines/threonines with N-acetylglucosamine (O-GlcNAc) is dynamic,inducible and abundant, regulating many cellular processes byinterfering with protein phosphorylation.EC 2.4.1.255 transfers GlcNAc onto substrate proteins andEC 3.2.1.169 cleaves GlcNAc from the modified proteins.','(1) UDP-N-acetyl-D-glucosamine + [protein]-L-serine = UDP + [protein]-3-O-(N-acetyl-D-glucosaminyl)-L-serine.(2) UDP-N-acetyl-D-glucosamine + [protein]-L-threonine = UDP + [protein]-3-O-(N-acetyl-D-glucosaminyl)-L-threonine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.256','Dolichyl-P-Glc:Glc(2)Man(9)GlcNAc(2)-PP-dolichol alpha-1,2-glucosyltransferase','Dol-P-Glc:Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase','NULL','This eukaryotic enzyme performs the final step in the synthesis ofthe lipid-linked oligosaccharide, attaching D-glucose in an alpha-1,2-linkage to the outermost D-glucose in the long branch.The lipid-linked oligosaccharide is involved in N-linked proteinglycosylation of selected asparagine residues of nascent polypeptidechains in eukaryotic cells.','Dolichyl beta-D-glucosyl phosphate + D-Glc-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.257','GDP-Man:Man(2)GlcNAc(2)-PP-dolichol alpha-1,6-mannosyltransferase','GDP-Man:Man(2)GlcNAc(2)-PP-Dol alpha-1,6-mannosyltransferase','NULL','The biosynthesis of asparagine-linked glycoproteins utilizes adolichyl diphosphate-linked glycosyl donor, which is assembled by theseries of membrane-bound glycosyltransferases that comprise thedolichol pathway.Alg2 mannosyltransferase from Saccharomyces cerevisiae carries out analpha-1,3-mannosylation (cf. EC 2.4.1.132) of D-Man-beta-(1-','GDP-D-mannose + D-Man-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.258','Dolichyl-P-Man:Man(5)GlcNAc(2)-PP-dolichol alpha-1,3-mannosyltransferase','Alg3 alpha-1,3-mannosyl transferase','NULL','The formation of N-glycosidic linkages of glycoproteins involves theordered assembly of the common Glc(3)Man(9)GlcNAc(2) core-oligosaccharide on the lipid carrier dolichyl diphosphate.Early mannosylation steps occur on the cytoplasmic side of theendoplasmic reticulum with GDP-Man as donor, the final reactions fromMan(5)GlcNAc(2)-PP-dolichyl to Man(9)Glc-NAc(2)-PP-dolichyl on thelumenal side use dolichyl beta-D-mannosyl phosphate.The first step of this assembly pathway on the luminal side of theendoplasmic reticulum is catalyzed by ALG3.Formerly EC 2.4.1.130.','Dolichyl beta-D-mannosyl phosphate + D-Man-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.259','Dolichyl-P-Man:Man(6)GlcNAc(2)-PP-dolichol alpha-1,2-mannosyltransferase','Dol-P-Man:Man(6)GlcNAc(2)-PP-Dol alpha-1,2-mannosyltransferase','NULL','The formation of N-glycosidic linkages of glycoproteins involves theordered assembly of the common Glc(3)Man(9)GlcNAc(2) core-oligosaccharide on the lipid carrier dolichyl diphosphate.Early mannosylation steps occur on the cytoplasmic side of theendoplasmic reticulum with GDP-Man as donor, the final reactions fromMan(5)GlcNAc(2)-PP-dolichol to Man(9)Glc-NAc(2)-PP-dolichol on thelumenal side use dolichyl beta-D-mannosyl phosphate.ALG9 mannosyltransferase catalyzes the addition of two differentalpha-1,2-mannose residues - the addition of alpha-1,2-mannose toMan(6)GlcNAc(2)-PP-dolichol (EC 2.4.1.259) and the addition of alpha-1,2-mannose to Man(8)GlcNAc(2)-PP-dolichol (EC 2.4.1.261).Formerly EC 2.4.1.130.','Dolichyl beta-D-mannosyl phosphate + D-Man-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.260','Dolichyl-P-Man:Man(7)GlcNAc(2)-PP-dolichol alpha-1,6-mannosyltransferase','Dol-P-Man:Man(7)GlcNAc(2)-PP-Dol alpha-1,6-mannosyltransferase','NULL','The formation of N-glycosidic linkages of glycoproteins involves theordered assembly of the common Glc(3)Man(9)GlcNAc(2) core-oligosaccharide on the lipid carrier dolichyl diphosphate.Early mannosylation steps occur on the cytoplasmic side of theendoplasmic reticulum with GDP-Man as donor, the final reactions fromMan(5)GlcNAc(2)-PP-dolichol to Man(9)Glc-NAc(2)-PP-dolichol on thelumenal side use dolichyl beta-D-mannosyl phosphate.Formerly EC 2.4.1.130.','Dolichyl beta-D-mannosyl phosphate + D-Man-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.261','Dolichyl-P-Man:Man(8)GlcNAc(2)-PP-dolichol alpha-1,2-mannosyltransferase','Dol-P-Man:Man(8)GlcNAc(2)-PP-Dol alpha-1,2-mannosyltransferase','NULL','The formation of N-glycosidic linkages of glycoproteins involves theordered assembly of the common Glc(3)Man(9)GlcNAc(2) core-oligosaccharide on the lipid carrier dolichyl diphosphate.Early mannosylation steps occur on the cytoplasmic side of theendoplasmic reticulum with GDP-Man as donor, the final reactions fromMan(5)GlcNAc(2)-PP-Dol to Man(9)Glc-NAc(2)-PP-Dol on the lumenal sideuse dolichyl beta-D-mannosyl phosphate.ALG9 mannosyltransferase catalyzes the addition of two differentalpha-1,2-mannose residues - the addition of alpha-1,2-mannose toMan(6)GlcNAc(2)-PP-Dol (EC 2.4.1.259) and the addition of alpha-1,2-mannose to Man(8)GlcNAc(2)-PP-Dol (EC 2.4.1.261).Formerly EC 2.4.1.130.','Dolichyl beta-D-mannosyl phosphate + D-Man-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.262','Soyasapogenol glucuronosyltransferase','NULL','Magnesium.','Requires a divalent ion, Mg(2+) better than Mn(2+), better thanCa(2+).Also acts on soysapogenol A and E.','UDP-glucuronate + soyasapogenol B = UDP + soyasapogenol B 3-O-D-glucuronide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.263','Abscisate beta-glucosyltransferase','ABA-glucosyltransferase','NULL','The enzyme acts better on (S)-2-trans-abscisate than the natural (S)-2-cis isomer, abscisate, or its enantiomer, the (R)-2-cis isomer.','UDP-D-glucose + abscisate = UDP + beta-D-glucopyranosyl abscisate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.264','D-man-alpha-(1-','NULL','NULL','The enzyme is involved in the biosynthesis of the exopolysaccharidesxanthan (in the bacterium Xanthomonas campestris) and acetan (in thebacterium Gluconacetobacter xylinus).','UDP-glucuronate + D-Man-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.265','Dolichyl-P-Glc:Glc(1)Man(9)GlcNAc(2)-PP-dolichol alpha-1-','Dol-P-Glc:Glc(1)Man(9)GlcNAc(2)-PP-Dol alpha-1-','NULL','The successive addition of three glucose residues by EC 2.4.1.267,2.4.1.265 and EC 2.4.1.256 represents the final stage of the lipid-linked oligosaccharide assembly.','Dolichyl beta-D-glucosyl phosphate + D-Glc-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.266','Glucosyl-3-phosphoglycerate synthase','Glucosylphosphoglycerate synthase','NULL','The enzyme is involved in biosynthesis of 2-O-(alpha-D-glucopyranosyl)-D-glycerate via the two-step pathway in whichglucosyl-3-phosphoglycerate synthase catalyzes the conversion of GDP-glucose and 3-phospho-D-glycerate into 2-O-(alpha-D-glucopyranosyl)-3-phospho-D-glycerate, which is then converted to 2-O-(alpha-D-glucopyranosyl)-D-glycerate by EC 3.1.3.85.The activity is dependent on divalent cations (Mn(2+), Co(2+),or Mg(2+)).The enzyme from Persephonella marina shows moderate flexibility onthe sugar donor concerning the nucleotide moiety (UDP-glucose, ADP-glucose, GDP-glucose) but is strictly specific for glucose.The enzyme is also strictly specific for 3-phospho-D-glycerate asacceptor.The enzyme from Methanococcoides burtonii is strictly specific forGDP-glucose and 3-phospho-D-glycerate.This enzyme catalyzes the first glucosylation step in methylglucoselipopolysaccharide biosynthesis in mycobacteria.','NDP-glucose + 3-phospho-D-glycerate = NDP + 2-O-(alpha-D-glucopyranosyl)-3-phospho-D-glycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.267','Dolichyl-P-Glc:Man(9)GlcNAc(2)-PP-dolichol alpha-1-','Dol-P-Glc:Man(9)GlcNAc(2)-PP-Dol alpha-1-','NULL','The successive addition of three glucose residues by EC 2.4.1.267,EC 2.4.1.265 and EC 2.4.1.256 represents the final stage of thelipid-linked oligosaccharide assembly.','Dolichyl beta-D-glucosyl phosphate + D-Man-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.268','Glucosylglycerate synthase','NULL','NULL','Persephonella marina possesses two enzymatic systems for thesynthesis of glucosylglycerate.The first one is a single-step pathway in which glucosylglyceratesynthase catalyzes the synthesis of 2-O-(alpha-D-glucopyranosyl)-D-glycerate in one-step from ADP-glucose and D-glycerate.The second system is a two-step pathway in which EC 2.4.1.266catalyzes the conversion of NDP-glucose and 3-phospho-D-glycerateinto 2-O-(alpha-D-glucopyranosyl)-3-phospho-D-glycerate, which isthen converted to 2-O-(alpha-D-glucopyranosyl)-D-glycerate byEC 3.1.3.85.','ADP-glucose + D-glycerate = 2-O-(alpha-D-glucopyranosyl)-D-glycerate +ADP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.269','Mannosylglycerate synthase','NULL','NULL','Rhodothermus marinus can also form mannosylglycerate via a two-steppathway catalyzed by EC 2.4.1.217 and EC 3.1.3.70.Depending on experimental conditions mannosylglycerate synthase ismore or less specific for the GDP-mannose and D-glycerate.','GDP-mannose + D-glycerate = GDP + 2-O-(alpha-D-mannopyranosyl)-D-glycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.270','Mannosylglucosyl-3-phosphoglycerate synthase','NULL','NULL','The enzyme is involved in synthesis of 2-(2-O-(alpha-D-mannopranosyl)-alpha-D-glucopyranosyl)-D-glycerate.Petrotoga miotherma and Petrotoga mobilis accumulate this compound inresponse to water stress imposed by salt.','GDP-mannose + 2-O-(alpha-D-glucopyranosyl)-3-phospho-D-glycerate = GDP +2-O-(2-O-(alpha-D-mannopyranosyl)-alpha-D-glucopyranosyl)-3-phospho-D-glycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.271','Crocetin glucosyltransferase','Crocetin GTase','NULL','In Crocus sativus this enzyme esterifies a free carboxyl group ofcrocetin or crocetin glycosyl ester.There are two isoenzymes, UGTCs2, which is mainly expressed in fullydeveloped stigmas, and UGTCs3, which is mainly expressed in stamens.','(1) UDP-glucose + crocetin = UDP + beta-D-glucosyl crocetin.(2) UDP-glucose + beta-D-glucosyl crocetin = UDP + bis(beta-D-glucosyl)crocetin.(3) UDP-glucose + beta-D-gentiobiosyl crocetin = UDP + beta-D-gentiobiosyl beta-D-glucosyl crocetin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.272','Soyasapogenol B glucuronide galactosyltransferase','NULL','NULL','Part of the biosynthetic pathway for soyasaponins.','UDP-alpha-D-galactose + soyasapogenol B 3-O-beta-D-glucuronide = UDP +soyasaponin III.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.273','Soyasaponin III rhamnosyltransferase','NULL','NULL','Part of the biosynthetic pathway for soyasaponins.','UDP-rhamnose + soyasaponin III = UDP + soyasaponin I.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.274','Glucosylceramide beta-1,4-galactosyltransferase','Lactosylceramide synthase','NULL','Involved in the synthesis of several different major classes ofglycosphingolipids.','UDP-alpha-D-galactose + beta-D-glucosyl-(1', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.275','Neolactotriaosylceramide beta-1,4-galactosyltransferase','Lactotriaosylceramide beta-1,4-galactosyltransferase','NULL','NULL','UDP-alpha-D-galactose + N-acetyl-beta-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.276','Zeaxanthin glucosyltransferase','NULL','NULL','The reaction proceeds in two steps with the monoglucoside as anintermediate.','2 UDP-glucose + zeaxanthin = 2 UDP + zeaxanthin bis(beta-D-glucoside).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.277','10-deoxymethynolide desosaminyltransferase','Glycosyltransferase DesVII','NULL','DesVII is the glycosyltransferase responsible for the attachment ofdTDP-D-desosamine to 10-deoxymethynolide or narbonolide during thebiosynthesis of methymycin, neomethymycin, narbomycin, and pikromycinin the bacterium Streptomyces venezuelae.Activity requires an additional protein partner, DesVIII.','dTDP-3-dimethylamino-3,4,6-trideoxy-alpha-D-glucopyranose +10-deoxymethynolide = dTDP + 10-deoxymethymycin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.278','3-alpha-mycarosylerythronolide B desosaminyl transferase','Desosaminyl transferase EryCIII','NULL','The enzyme is involved in erythromycin biosynthesis.','dTDP-D-desosamine + 3-alpha-L-mycarosylerythronolide B = dTDP +erythromycin D.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.279','Nigerose phosphorylase','NULL','NULL','The enzymes from Clostridium phytofermentans is specific fornigerose.and shows only 0.5% relative activity with kojibiose (cf.EC 2.4.1.230).','3-O-alpha-D-glucopyranosyl-D-glucopyranose + phosphate = D-glucose +beta-D-glucose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.280','N,N-diacetylchitobiose phosphorylase','NULL','NULL','The enzyme is specific for N,N-diacetylchitobiose and does notphosphorylyze other N-acetylchitooligosaccharides, cellobiose,trehalose, lactose, maltose or sucrose.','N,N-diacetylchitobiose + phosphate = N-acetyl-D-glucosamine + N-acetyl-alpha-D-glucosamine 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.281','4-O-beta-D-mannosyl-D-glucose phosphorylase','Mannosylglucose phosphorylase','NULL','This enzyme forms part of a mannan catabolic pathway in the anaerobicbacterium Bacteroides fragilis NCTC 9343.','4-O-beta-D-mannopyranosyl-D-glucopyranose + phosphate = D-glucose +alpha-D-mannose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.282','3-O-alpha-D-glucosyl-L-rhamnose phosphorylase','NULL','NULL','The enzyme does not phosphorylate alpha,alpha-trehalose, kojibiose,nigerose, or maltose.In the reverse phosphorolysis reaction the enzyme is specific forL-rhamnose as acceptor and beta-D-glucose 1-phosphate as donor.','3-O-alpha-D-glucopyranosyl-L-rhamnopyranose + phosphate =L-rhamnopyranose + beta-D-glucose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.283','2-deoxystreptamine N-acetyl-D-glucosaminyltransferase','NULL','NULL','Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including kanamycin, butirosin, neomycinand ribostamycin.Unlike the enzyme from the bacterium Streptomyces kanamyceticus,which can also accept UDP-D-glucose (cf. EC 2.4.1.284), the enzymefrom Bacillus circulans can only accept UDP-N-acetyl-alpha-D-glucosamine.','UDP-N-acetyl-alpha-D-glucosamine + 2-deoxystreptamine = UDP + 2-N-acetylparomamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.284','2-deoxystreptamine glucosyltransferase','NULL','NULL','Involved in the biosynthesis of kanamycin B and kanamycin C.Also catalyzes EC 2.4.1.283, but activity is only one fifth of thatwith UDP-alpha-D-glucose.','UDP-alpha-D-glucose + 2-deoxystreptamine = UDP + 2-deamino-2-hydroxyparomamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.285','UDP-GlcNAc:ribostamycin N-acetylglucosaminyltransferase','NULL','Divalent cation.','Involved in biosynthesis of the aminoglycoside antibiotic neomycin.','UDP-N-acetyl-alpha-D-glucosamine + ribostamycin = UDP + 2-acetyl-6-hydroxyneomycin C.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.286','Chalcone 4-O-glucosyltransferase','4CGT','NULL','Isolated from the plant Antirrhinum majus (snapdragon).Involved in the biosynthesis of aurones, plant flavonoids thatprovide yellow color to the flowers.','(1) UDP-alpha-D-glucose + 2,4,4,6-tetrahydroxychalcone = UDP +2,4,4,6-tetrahydroxychalcone 4-O-beta-D-glucoside.(2) UDP-alpha-D-glucose + 2,3,4,4,6-pentahydroxychalcone = UDP +2,3,4,4,6-pentahydroxychalcone 4-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.287','Rhamnopyranosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,3/1,4-galactofuranosyltransferase','Arabinogalactan galactofuranosyl transferase 1','NULL','Isolated from Mycobacterium tuberculosis and M.smegmatis, the enzymehas dual beta-(1-','2 UDP-alpha-D-galactofuranose + alpha-L-rhamnopyranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.288','Galactofuranosylgalactofuranosylrhamnosyl-N-acetylglucosaminyl-diphospho-decaprenol beta-1,5/1,6-galactofuranosyltransferase','NULL','NULL','Isolated from Mycobacterium tuberculosis.The enzyme adds approximately twenty-eight galactofuranosyl residueswith alternating 1-','28 UDP-alpha-D-galactofuranose + beta-D-galactofuranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.289','N-acetylglucosaminyl-diphospho-decaprenol L-rhamnosyltransferase','NULL','Manganese or magnesium.','Isolated from Mycobacterium smegmatis and Mycobacterium tuberculosis.The enzyme catalyzes the addition of a rhamnosyl unit to N-acetyl-alpha-D-glucosaminyl-diphospho-trans,octacis-decaprenol, completingthe synthesis of the linkage unit that attaches the arabinogalactanmoiety to the peptidoglycan moiety in Mycobacterial cell wall.','dTDP-6-deoxy-beta-L-mannose + N-acetyl-alpha-D-glucosaminyl-diphospho-trans,octacis-decaprenol = dTDP + alpha-L-rhamnopyranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.290','N,N-diacetylbacillosaminyl-diphospho-undecaprenol alpha-1,3-N-acetylgalactosaminyltransferase','NULL','NULL','Isolated from Campylobacter jejuni.Part of a bacterial N-linked glycosylation pathway.','UDP-N-acetyl-alpha-D-galactosamine + N,N-diacetyl-alpha-D-bacillosaminyl-diphospho-tritrans,heptacis-undecaprenol = UDP + N-acetyl-D-galactosaminyl-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.291','N-acetylgalactosamine-N,N-diacetylbacillosaminyl-diphospho-undecaprenol4-alpha-N-acetylgalactosaminyltransferase','NULL','NULL','Isolated from Campylobacter jejuni.Part of a bacterial N-linked glycosylation pathway.','UDP-N-acetyl-alpha-D-galactosamine + N-acetyl-D-galactosaminyl-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.292','GalNAc-alpha-(1-','NULL','NULL','Isolated from Campylobacter jejuni.Part of a bacterial N-linked glycosylation pathway.','3 UDP-N-acetyl-alpha-D-galactosamine + GalNAc-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.293','GalNAc(5)-diNAcBac-PP-undecaprenol beta-1,3-glucosyltransferase','NULL','NULL','Isolated from the bacterium Campylobacter jejuni.Part of a bacterial N-linked glycosylation pathway.','UDP-alpha-D-glucose + (GalNAc-alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.294','Cyanidin 3-O-galactosyltransferase','NULL','NULL','Isolated from the plant Daucus carota (Afghan cultivar carrot).','UDP-alpha-D-galactose + cyanidin = UDP + cyanidin 3-O-beta-D-galactoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.295','Anthocyanin 3-O-sambubioside 5-O-glucosyltransferase','NULL','NULL','Isolated from the plant Matthiola incana (stock).No activity with anthocyanidin 3-O-glucosides.','UDP-alpha-D-glucose + an anthocyanidin 3-O-beta-D-sambubioside = UDP +an anthocyanidin 5-O-beta-D-glucoside 3-O-beta-D-sambubioside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.296','Anthocyanidin 3-O-coumaroylrutinoside 5-O-glucosyltransferase','NULL','NULL','Isolated from the plant Petunia hybrida.It does not act on an anthocyanidin 3-O-rutinoside.','UDP-alpha-D-glucose + an anthocyanidin 3-O-(2-O-(4-coumaroyl)-alpha-L-rhamnosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.297','Anthocyanidin 3-O-glucoside 2-O-glucosyltransferase','NULL','NULL','Isolated from Ipomoea nil (Japanese morning glory).','UDP-alpha-D-glucose + an anthocyanidin 3-O-beta-D-glucoside = UDP +an anthocyanidin 3-O-sophoroside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.298','Anthocyanidin 3-O-glucoside 5-O-glucosyltransferase','NULL','NULL','Isolated from the plants Perilla frutescens var. crispa, Verbenahybrida, Dahlia variabilis and Gentiana triflora (clustered gentian).It will also act on anthocyanidin 3-O-(6-O-malonylglucoside) and ismuch less active with hydroxycinnamoylglucose derivatives.There is no activity in the absence of the 3-O-glucoside group.','UDP-alpha-D-glucose + an anthocyanidin 3-O-beta-D-glucoside = UDP +an anthocyanidin 3,5-di-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.299','Cyanidin 3-O-glucoside 5-O-glucosyltransferase (acyl-glucose)','NULL','NULL','Isolated from the plant Dianthus caryophyllus (carnation).Also acts on other anthocyanidins and with other acyl-glucose donors.Cf. EC 2.4.1.298.','1-O-sinapoyl-beta-D-glucose + cyanidin 3-O-beta-D-glucoside = sinapate +cyanidin 3,5-di-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.300','Cyanidin 3-O-glucoside 7-O-glucosyltransferase (acyl-glucose)','NULL','NULL','Isolated from the plant Delphinium grandiflorum (delphinium).Also acts on other anthocyanidins and with other acyl-glucosederivatives.','1-O-vanilloyl-beta-D-glucose + cyanidin 3-O-beta-D-glucoside = vanillate+ cyanidin 3,7-di-O-beta-D-glucoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.301','2-deamino-2-hydroxyneamine 1-alpha-D-kanosaminyltransferase','NULL','NULL','Involved in the biosynthetic pathway of kanamycins.The enzyme characterized from the bacterium Streptomyceskanamyceticus can also accept UDP-alpha-D-glucose with lowerefficiency.','(1) UDP-alpha-D-kanosamine + 2-deamino-2-hydroxyneamine = UDP +kanamycin A.(2) UDP-alpha-D-kanosamine + neamine = UDP + kanamycin B.(3) UDP-alpha-D-kanosamine + paromamine = UDP + kanamycin C.(4) UDP-alpha-D-kanosamine + 2-deamino-2-hydroxyparomamine = UDP +kanamycin X.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.302','L-noviosyl transferase','NULL','NULL','The enzyme is involved in the biosynthesis of the aminocoumarinantibiotic, novobiocin.','dTDP-beta-L-noviose + novobiocic acid = dTDP + demethyldecarbamoylnovobiocin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.303','UDP-Gal:alpha-D-GlcNAc-diphosphoundecaprenol beta-1,3-galactosyltransferase','NULL','Manganese.','The enzyme is involved in the the biosynthesis of theO-polysaccharide repeating unit of Escherichia coli O7:K1 (VW187).','UDP-alpha-D-galactose + N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol = UDP + beta-D-Gal-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.304','UDP-Gal:alpha-D-GlcNAc-diphosphoundecaprenol beta-1,4-galactosyltransferase','NULL','Manganese.','The enzyme is involved in the the biosynthesis of theO-polysaccharide repeating unit of the bacterium Shigella boydii B14.','UDP-alpha-D-galactose + N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol = UDP + beta-D-Gal-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.305','UDP-Glc:alpha-D-GlcNAc-glucosaminyl-diphosphoundecaprenol beta-1,3-glucosyltransferase','NULL','NULL','The enzyme is involved in the the biosynthesis of theO-polysaccharide repeating unit of the bacterium Escherichia coliserotype O56 and serotype O152.','UDP-alpha-D-glucose + N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol = UDP + beta-D-Glc-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.306','UDP-GalNAc:alpha-D-GalNAc-diphosphoundecaprenol alpha-1,3-N-acetylgalactosaminyltransferase','NULL','NULL','The enzyme is involved in the the biosynthesis of theO-polysaccharide repeating unit of Escherichia coli serotype O86.','UDP-N-acetyl-alpha-D-galactosamine + N-acetyl-alpha-D-galactosaminyl-diphospho-ditrans,octacis-undecaprenol = UDP + alpha-D-GalNAc-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.307','UDP-Gal:alpha-D-GalNAc-1,3-alpha-D-GalNAc-diphosphoundecaprenol beta-1,3-galactosyltransferase','NULL','NULL','The enzyme is involved in the the biosynthesis of theO-polysaccharide repeating unit of the bacterium Escherichia coliserotype O86.','UDP-alpha-D-galactose + alpha-D-GalNAc-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.308','GDP-Fuc:beta-D-Gal-1,3-alpha-D-GalNAc-1,3-alpha-GalNAc-diphosphoundecaprenol alpha-1,2-fucosyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of the O-polysacchariderepeating unit of the bacterium Escherichia coli serotype O86.','GDP-beta-L-fucose + beta-D-Gal-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.309','UDP-Gal:alpha-L-Fuc-1,2-beta-Gal-1,3-alpha-GalNAc-1,3-alpha-GalNAc-diphosphoundecaprenol alpha-1,3-galactosyltransferase','NULL','NULL','The enzyme is involved in the the biosynthesis of theO-polysaccharide repeating unit of the bacterium Escherichia coliserotype O86.','UDP-alpha-D-galactose + alpha-L-Fuc-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.310','Vancomycin aglycone glucosyltransferase','NULL','NULL','The enzyme from the bacterium Amycolatopsis orientalis is involved inthe biosynthesis of the glycopeptide antibiotic chloroeremomycin.','UDP-alpha-D-glucose + vancomycin aglycone = UDP + devancoaminyl-vancomycin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.311','dTDP-epi-vancosaminyltransferase','NULL','NULL','The enzyme from the bacterium Amycolatopsis orientalis is involved inthe biosynthesis of the glycopeptide antibiotic chloroeremomycin.','dTDP-beta-L-4-epi-vancosamine + devancoaminyl-vancomycin = dTDP +chloroorienticin B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.312','Protein O-mannose beta-1,4-N-acetylglucosaminyltransferase','NULL','NULL','The human protein is involved in the formation of a phosphorylatedtrisaccharide on a threonine residue of alpha-dystroglycan,an extracellular peripheral glycoprotein that acts as a receptor forextracellular matrix proteins containing laminin-G domains.','UDP-N-acetyl-alpha-D-glucosamine + 3-O-(alpha-D-mannosyl)-L-threonyl-[protein] = UDP + 3-O-(N-acetyl-beta-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.313','Protein O-mannose beta-1,3-N-acetylgalactosaminyltransferase','NULL','NULL','The human protein is specific for UDP-N-acetyl-alpha-D-galactosamineas donor.The enzyme is involved in the formation of a phosphorylatedtrisaccharide on a threonine residue of alpha-dystroglycan,an extracellular peripheral glycoprotein that acts as a receptor forextracellular matrix proteins containing laminin-G domains.','UDP-N-acetyl-alpha-D-galactosamine + 3-O-(N-acetyl-beta-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.314','Ginsenoside Rd glucosyltransferase','NULL','NULL','The glucosyl group forms a 1-','UDP-alpha-D-glucose + ginsenoside Rd = UDP + ginsenoside Rb1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.315','Diglucosyl diacylglycerol synthase (1,6-linking)','DGlcDAG synthase','NULL','The enzyme is found in several bacterial species.The enzyme from Bacillus subtilis is specific for glucose and canform tetraglucosyl diacylglycerol in vitro.The enzyme from Mycoplasma genitalium can incoporate galactose withsimilar efficiency, but forms mainly 1,2-diacyl-diglucopyranosyl-sn-glycerol in vivo.The enzyme from Staphylococcus aureus can also form glucosyl-glycero-3-phospho-(1-sn-glycerol).','(1) UDP-alpha-D-glucose + 1,2-diacyl-3-O-(alpha-D-glucopyranosyl)-sn-glycerol = 1,2-diacyl-3-O-(alpha-D-glucopyranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.316','Tylactone mycaminosyltransferase','NULL','NULL','The enzyme participates in the biosynthetic pathway of the macrolideantibiotic tylosin, which is produced by several species ofStreptomyces bacteria.','Tylactone + dTDP-alpha-D-mycaminose = dTDP + 5-O-beta-D-mycaminosyltylactone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.317','O-mycaminosyltylonolide 6-deoxyallosyltransferase','NULL','NULL','The enzyme participates in the biosynthetic pathway of the macrolideantibiotic tylosin, which is produced by several species ofStreptomyces bacteria.','5-O-beta-D-mycaminosyltylonolide + dTDP-6-deoxy-alpha-D-allose = dTDP +demethyllactenocin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.318','Demethyllactenocin mycarosyltransferase','NULL','NULL','The enzyme participates in the biosynthetic pathway of the macrolideantibiotic tylosin, which is produced by several species ofStreptomyces bacteria.','dTDP-beta-L-mycarose + demethyllactenocin = dTDP + demethylmacrocin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.319','Beta-1,4-mannooligosaccharide phosphorylase','NULL','NULL','The enzyme, isolated from the ruminal bacterium Ruminococcus albus,catalyzes the reversible phosphorolysis of beta-1,4-mannooligosaccharide with a minimum size of three monomers.','((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.320','1,4-beta-mannosyl-N-acetylglucosamine phosphorylase','NULL','NULL','The enzyme isolated from the anaerobic bacterium Bacteroidesthetaiotaomicron is involved in the degradation of host-derivedN-glycans.','4-O-beta-D-mannopyranosyl-N-acetyl-D-glucosamine + phosphate = N-acetyl-D-glucosamine + alpha-D-mannose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.321','Cellobionic acid phosphorylase','NULL','NULL','The enzyme occurs in cellulolytic bacteria and fungi.It catalyzes the reversible phosphorolysis of cellobionic acid.In the synthetic direction it produces 4-O-beta-D-glucopyranosyl-D-glucuronate from alpha-D-glucose 1-phosphate and D-glucuronate withlow activity.','4-O-beta-D-glucopyranosyl-D-gluconate + phosphate = alpha-D-glucose1-phosphate + D-gluconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.322','Desvancosaminyl-vancomycin vancosaminetransferase','Desvancosamine-vancomycin TDP-vancosaminyltransferase','NULL','The enzyme, isolated from the bacterium Amycolatopsis orientalis,catalyzes the ultimate step in the biosynthesis of the antibioticvancomycin.','dTDP-beta-L-vancosamine + desvancosaminyl-vancomycin = dTDP + vancomycin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.323','7-deoxyloganetic acid glucosyltransferase','NULL','NULL','Isolated from the plant Catharanthus roseus (Madagascar periwinkle).Involved in loganin and secologanin biosynthesis.Does not react with 7-deoxyloganetin.Cf. EC 2.4.1.324.','UDP-alpha-D-glucose + 7-deoxyloganetate = UDP + 7-deoxyloganate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.324','7-deoxyloganetin glucosyltransferase','UGT85A24','NULL','Isolated from the plants Catharanthus roseus (Madagascar periwinkle)and Gardenia jasminoides (cape jasmine).With Gardenia it also acts on genipin.Involved in loganin and secologanin biosynthesis.Does not react with 7-deoxyloganetate.Cf. EC 2.4.1.323.','UDP-alpha-D-glucose + 7-deoxyloganetin = UDP + 7-deoxyloganin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.n2','Loliose synthase','NULL','NULL','NULL','UDP-galactose + sucrose = UDP + loliose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.n3','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.n4','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.1.n5','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.1','Purine-nucleoside phosphorylase','Inosine phosphorylase','NULL','Specificity not completely determined.Can also catalyze ribosyltransferase reactions of the type catalyzedby EC 2.4.2.5.','(1) Purine nucleoside + phosphate = purine + alpha-D-ribose 1-phosphate.(2) Purine deoxynucleoside + phosphate = purine + 2-deoxy-alpha-D-ribose1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.2','Pyrimidine-nucleoside phosphorylase','Py-NPase','NULL','Unlike EC 2.4.2.3, and EC 2.4.2.4, this enzyme can accept both theribonucleoside uridine and the 2-deoxyribonucleosides2-deoxyuridine and thymidine.The reaction is reversible, and the enzyme does not distinguishbetween alpha-D-ribose 1-phosphate and 2-deoxy-alpha-D-ribose1-phosphate in the synthetic direction.Formerly EC 2.4.2.23.','(1) Uridine + phosphate = uracil + alpha-D-ribose 1-phosphate.(2) Thymidine + phosphate = thymine + 2-deoxy-alpha-D-ribose 1-phosphate.(3) 2-deoxyuridine + phosphate = uracil + 2-deoxy-alpha-D-ribose1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.3','Uridine phosphorylase','Pyrimidine phosphorylase','NULL','The enzyme participates the the pathways of pyrimidineribonucleosides degradation and salvage.The mammalian enzyme also accepts 2-deoxyuridine.Formerly EC 2.4.2.23.','Uridine + phosphate = uracil + alpha-D-ribose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.4','Thymidine phosphorylase','Pyrimidine phosphorylase','NULL','In some tissues also catalyzes deoxyribosyltransferase reactions ofthe type catalyzed by EC 2.4.2.6.Formerly EC 2.4.2.23.','Thymidine + phosphate = thymine + 2-deoxy-alpha-D-ribose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.5','Nucleoside ribosyltransferase','NULL','NULL','Base(1) and base(2) represent various purines and pyrimidines.','D-ribosyl-base(1) + base(2) = D-ribosyl-base(2) + base(1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.6','Nucleoside deoxyribosyltransferase','NULL','NULL','Base(1) and base(2) represent various purines and pyrimidines.','2-deoxy-D-ribosyl-base(1) + base(2) = 2-deoxy-D-ribosyl-base(2) +base(1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.7','Adenine phosphoribosyltransferase','AMP diphosphorylase','NULL','5-amino-4-imidazolecarboxamide can replace adenine.','AMP + diphosphate = adenine + 5-phospho-alpha-D-ribose 1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.8','Hypoxanthine phosphoribosyltransferase','Guanine phosphoribosyltransferase','NULL','Guanine and 6-mercaptopurine can replace hypoxanthine.','IMP + diphosphate = hypoxanthine + 5-phospho-alpha-D-ribose1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.9','Uracil phosphoribosyltransferase','UMP diphosphorylase','NULL','NULL','UMP + diphosphate = uracil + 5-phospho-alpha-D-ribose 1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.10','Orotate phosphoribosyltransferase','OPRT','NULL','The enzyme from higher eukaryotes also catalyzes the reaction listedas EC 4.1.1.23.','Orotidine 5-phosphate + diphosphate = orotate + 5-phospho-alpha-D-ribose1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.11','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.12','Nicotinamide phosphoribosyltransferase','NMN diphosphorylase','NULL','NULL','Nicotinamide D-ribonucleotide + diphosphate = nicotinamide + 5-phospho-alpha-D-ribose 1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.13','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.14','Amidophosphoribosyltransferase','Glutamine phosphoribosylpyrophosphate amidotransferase','NULL','NULL','5-phospho-beta-D-ribosylamine + diphosphate + L-glutamate = L-glutamine +5-phospho-alpha-D-ribose 1-diphosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.15','Guanosine phosphorylase','NULL','NULL','Also acts on deoxyguanosine.','Guanosine + phosphate = guanine + alpha-D-ribose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.16','Urate-ribonucleotide phosphorylase','NULL','NULL','NULL','Urate D-ribonucleotide + phosphate = urate + alpha-D-ribose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.17','ATP phosphoribosyltransferase','Phosphoribosyl-ATP diphosphorylase','NULL','Involved in histidine biosynthesis.','1-(5-phospho-beta-D-ribosyl)-ATP + diphosphate = ATP + 5-phospho-alpha-D-ribose 1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.18','Anthranilate phosphoribosyltransferase','Phosphoribosyl-anthranilate diphosphorylase','NULL','In some organisms, this enzyme is part of a multifunctional proteintogether with one or more components of the system for biosynthesisof tryptophan (EC 4.1.1.48, EC 4.1.3.27, EC 4.2.1.20,and EC 5.3.1.24).','N-(5-phospho-D-ribosyl)-anthranilate + diphosphate = anthranilate +5-phospho-alpha-D-ribose 1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.19','Nicotinate-nucleotide diphosphorylase (carboxylating)','Nicotinate-nucleotide pyrophosphorylase (carboxylating)','NULL','The reaction is catalyzed in the opposite direction.Since quinolinate is synthesized from L-tryptophan in eukaryotes,but from L-aspartate in some prokaryotes, this is the first NAD(+)biosynthesis enzyme shared by both eukaryotes and prokaryotes.','Beta-nicotinate D-ribonucleotide + diphosphate + CO(2) = pyridine-2,3-dicarboxylate + 5-phospho-alpha-D-ribose 1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.20','Dioxotetrahydropyrimidine phosphoribosyltransferase','Dioxotetrahydropyrimidine-ribonucleotide diphosphorylase','NULL','Acts, in the reverse direction, on uracil and other pyrimidines andpteridines containing a 2,4-diketo structure.','A 2,4-dioxotetrahydropyrimidine D-ribonucleotide + diphosphate = a 2,4-dioxotetrahydropyrimidine + 5-phospho-alpha-D-ribose 1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.21','Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase','N(1)-alpha-phosphoribosyltransferase','NULL','Also acts on benzimidazole, and the clostridial enzyme acts onadenine to form 7-alpha-D-ribosyladenine 5-phosphate.The product of the reaction, alpha-ribazole 5-phosphate, forms partof the corrin biosynthesis pathway and is a substrate for EC 2.7.8.2.It can also be dephosphorylated to form alpha-ribazole by the actionof EC 3.1.3.73.','Beta-nicotinate D-ribonucleotide + 5,6-dimethylbenzimidazole = nicotinate+ alpha-ribazole 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.22','Xanthine phosphoribosyltransferase','Xan phosphoribosyltransferase','NULL','NULL','XMP + diphosphate = 5-phospho-alpha-D-ribose 1-diphosphate + xanthine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.23','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.24','1,4-beta-D-xylan synthase','NULL','NULL','Formerly EC 2.4.1.72.','UDP-D-xylose + ((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.25','Flavone apiosyltransferase','NULL','NULL','7-O-beta-D-glucosides of a number of flavonoids and of 4-substitutedphenols can act as acceptors.','UDP-alpha-D-apiose + apigenin 7-O-beta-D-glucoside = UDP + apigenin 7-O-(beta-D-apiosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.26','Protein xylosyltransferase','UDP-D-xylose:core protein beta-D-xylosyltransferase','NULL','Involved in the biosynthesis of the linkage region ofglycosaminoglycan chains as part of proteoglycan biosynthesis(chondroitin, dermatan and heparan sulfates).','Transfers a beta-D-xylosyl residue from UDP-D-xylose to the serinehydroxy group of an acceptor protein substrate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.27','dTDP-dihydrostreptose--streptidine-6-phosphatedihydrostreptosyltransferase','NULL','NULL','NULL','dTDP-L-dihydrostreptose + streptidine 6-phosphate = dTDP + O-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.28','S-methyl-5-thioadenosine phosphorylase','5-deoxy-5-methylthioadenosine phosphorylase','NULL','Also acts on 5-deoxyadenosine and other analogs having 5-deoxygroups.','S-methyl-5-thioadenosine + phosphate = adenine + S-methyl-5-thio-alpha-D-ribose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.29','tRNA-guanine(34) transglycosylase','Guanine insertion enzyme','NULL','Certain prokaryotic and eukaryotic tRNAs contain the modified basequeuine at position 34.In eukaryotes queuine is salvaged from food and incorporated intotRNA directly via a base-exchange reaction, replacing guanine.In eubacteria, which produce queuine de novo, the enzyme catalyzesthe exchange of guanine with the queuine precursor preQ(1), which isultimately modified to queuine.The eubacterial enzyme can also use an earlier intermediate, preQ(0),to replace guanine in unmodified tRNA(Tyr) and tRNA(Asn).This enzyme acts after EC 1.7.1.13 in the queuine-biosynthesispathway.','(1) Guanine(34) in tRNA + queuine = queuosine(34) in tRNA + guanine.(2) Guanine(34) in tRNA + 7-aminomethyl-7-carbaguanine = 7-aminomethyl-7-carbaguanine(34) in tRNA + guanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.30','NAD(+) ADP-ribosyltransferase','ADP-ribosyltransferase (polymerizing)','NULL','The ADP-D-ribosyl group of NAD(+) is transferred to an acceptorcarboxy group on a histone or the enzyme itself, and further ADP-ribosyl groups are transferred to the 2-position of the terminaladenosine moiety, building up a polymer with an average chain lengthof 20-30 units.','NAD(+) + (ADP-D-ribosyl)(n)-acceptor = nicotinamide + (ADP-D-ribosyl)(n+1)-acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.31','NAD(+)--protein-arginine ADP-ribosyltransferase','ADP-ribosyltransferase','NULL','Arginine residues in proteins act as acceptors.Free arginine, agmatine ((4-aminobutyl)guanidine), arginine methylester and guanidine can also do so.Catalyzes the NAD(+)-dependent activation of EC 4.6.1.1.Some bacterial enterotoxins possess similar enzymatic activities (cf.EC 2.4.2.36).','NAD(+) + protein-L-arginine = nicotinamide + N(omega)-(ADP-D-ribosyl)-protein-L-arginine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.32','Dolichyl-phosphate D-xylosyltransferase','NULL','NULL','NULL','UDP-D-xylose + dolichyl phosphate = UDP + dolichyl D-xylosyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.33','Dolichyl-xylosyl-phosphate--protein xylosyltransferase','NULL','NULL','NULL','Dolichyl D-xylosyl phosphate + protein = dolichyl phosphate +D-xylosylprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.34','Indolylacetylinositol arabinosyltransferase','Arabinosylindolylacetylinositol synthase','NULL','The position of acylation is indeterminate because of the ease ofacyl transfer between hydroxy groups.','UDP-L-arabinose + (indol-3-yl)acetyl-1D-myo-inositol = UDP + (indol-3-yl)acetyl-myo-inositol 3-L-arabinoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.35','Flavonol-3-O-glycoside xylosyltransferase','NULL','NULL','Flavonol 3-O-glucoside, flavonol 3-O-galactoside and, more slowly,rutin, can act as acceptors.','UDP-alpha-D-xylose + a flavonol 3-O-glycoside = UDP + a flavonol 3-(beta-D-xylosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.36','NAD(+)--diphthamide ADP-ribosyltransferase','ADP-ribosyltransferase','NULL','Dihtheria toxin and some other bacterial toxins catalyze thisreaction, which inactivates translation elongation factor 2 (EF2).The acceptor is diphthamide, a unique modification of a histidineresidue in the elongation factor found in archaebacteria and alleukaryotes, but not in eubacteria.Cf. EC 2.4.2.31.The relevant histidine of EF2 is His(715) in mammals, His(699) inyeast and His(600) in Pyrococcus horikoshii.','NAD(+) + diphthamide-[translation elongation factor 2] = nicotinamide +N-(ADP-D-ribosyl)diphthamide-[translation elongation factor 2].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.37','NAD(+)--dinitrogen-reductase ADP-D-ribosyltransferase','NAD--azoferredoxin (ADPribose)transferase','NULL','Together with EC 3.2.2.24, controls the level of activity ofEC 1.18.6.1.','NAD(+) + [dinitrogen reductase] = nicotinamide + ADP-D-ribosyl-[dinitrogen reductase].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.38','Glycoprotein 2-beta-D-xylosyltransferase','Beta-1,2-xylosyltransferase','NULL','Specific for N-linked oligosaccharides (N-glycans).','UDP-D-xylose + N(4)-(N-acetyl-beta-D-glucosaminyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.39','Xyloglucan 6-xylosyltransferase','Uridine diphosphoxylose-xyloglucan 6-alpha-xylosyltransferase','NULL','In association with EC 2.4.1.168 this enzyme brings about thesynthesis of xyloglucan.Concurrent transfers of glucose and xylose are necessary for thissynthesis.Formerly EC 2.4.1.169.','Transfers an alpha-D-xylosyl residue from UDP-D-xylose to a glucoseresidue in xyloglucan, forming an alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.40','Zeatin O-beta-D-xylosyltransferase','Uridine diphosphoxylose-zeatin xylosyltransferase','NULL','Does not act on UDP-glucose (cf. EC 2.4.1.103).Formerly EC 2.4.1.204.','UDP-D-xylose + zeatin = UDP + O-beta-D-xylosylzeatin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.41','Xylogalacturonan beta-1,3-xylosyltransferase','XGA xylosyltransferase','NULL','Involved in plant cell wall synthesis.The enzyme from Arabidopsis thaliana could transfer D-xylose fromUDP-D-xylose onto oligogalacturonide acceptors.The enzyme did not show significant activity with UDP-glucose, UDP-galactose, or UDP-N-acetyl-D-glucosamine as sugar donors.','Transfers a xylosyl residue from UDP-D-xylose to a D-galactose residue inxylogalacturonan, forming a beta-1,3-D-xylosyl-D-galactose linkage.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.42','UDP-D-xylose:beta-D-glucoside alpha-1,3-D-xylosyltransferase','Beta-glucoside alpha-1,3-xylosyltransferase','NULL','The enzyme is involved in the biosynthesis of the Xyl-alpha(1-3)Xyl-alpha(1-3)Glc-beta-1-O-Ser on epidermal growth factor-like domains.','UDP-alpha-D-xylose + Glc-beta-Ser(53)-EGF-like domain of bovine factorIX(45-87) = UDP + Xyl-alpha(1-3)Glc-beta-Ser(53)-EGF-like domain ofbovine factor IX(45-87).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.43','Lipid IV(A) 4-amino-4-deoxy-L-arabinosyltransferase','4-amino-4-deoxy-L-arabinose lipid A transferase','NULL','Integral membrane protein present in the inner membrane of certainGram negative endobacteria.In strains that do not produce 3-deoxy-D-manno-octulosonic acid(Kdo), the enzyme adds a single arabinose unit to the 1-phosphatemoiety of the tetra-acylated lipid A precursor, lipid IV(A).In the presence of a Kdo disaccharide, the enzyme primarily adds anarabinose unit to the 4-phosphate of lipid A molecules.The Salmonella typhimurium enzyme can add arabinose units to bothpositions.Formerly EC 2.4.2.n1.','4-amino-4-deoxy-alpha-L-arabinopyranosyl di-trans,octa-cis-undecaprenylphosphate + lipid IV(A) = lipid II(A) + di-trans,octa-cis-undecaprenylphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.44','S-methyl-5-thioinosine phosphorylase','Methylthioinosine phosphorylase','NULL','No activity with S-methyl-5-thioadenosine.The catabolism of of 5-methylthioadenosine in Pseudomonas aeruginosainvolves deamination to S-methyl-5-thioinosine (EC 3.5.4.31) andphosphorolysis to hypoxanthine.','S-methyl-5-thioinosine + phosphate = hypoxanthine + S-methyl-5-thio-alpha-D-ribose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.45','Decaprenyl-phosphate phosphoribosyltransferase','5-phospho-alpha-D-ribose 1-pyrophosphate:decaprenyl phosphate5-phosphoribosyltransferase','Magnesium.','Isolated from Mycobacterium tuberculosis.Has some activity with other polyprenyl phosphates.','Trans,octacis-decaprenyl phosphate + 5-phospho-alpha-D-ribose1-diphosphate = trans,octacis-decaprenylphospho-beta-D-ribofuranose5-phosphate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.46','Galactan 5-O-arabinofuranosyltransferase','NULL','NULL','Isolated from Mycobacterium tuberculosis and Corynebacteriumglutamicum.These arabinofuranosyl groups form the start of an arabinofurananchain as part of the of the cell wall in mycobacteria.','Adds an alpha-D-arabinofuranosyl group from trans,octacis-decaprenylphospho-beta-D-arabinofuranose at the 5-O-position of theeighth, tenth and twelfth galactofuranose unit of the galactofurananchain of (beta-D-galactofuranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.47','Arabinofuranan 3-O-arabinosyltransferase','NULL','NULL','Isolated from Mycobacterium smegmatis.Involved in the formation of the cell wall in mycobacteria.','Adds an alpha-D-arabinofuranosyl group from trans,octacis-decaprenylphospho-beta-D-arabinofuranose at the 3-O-position of an alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.48','tRNA-guanine(15) transglycosylase','Transfer ribonucleic acid guanine(15) transglycosylase','NULL','Archaeal tRNAs contain the modified nucleoside archaeosine atposition 15.This archaeal enzyme catalyzes the exchange of guanine at position 15of tRNA with the base preQ(0), which is ultimately modified to formthe nucleoside archaeosine (cf. EC 2.6.1.97).','Guanine(15) in tRNA + 7-cyano-7-carbaguanine = 7-cyano-7-carbaguanine(15)in tRNA + guanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.49','Neamine phosphoribosyltransferase','NULL','Divalent metal cation.','Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including ribostamycin, neomycin andbutirosin.','Neamine + 5-phospho-alpha-D-ribose 1-diphosphate =5-phosphoribostamycin + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.50','Cyanidin 3-O-galactoside 2-O-xylosyltransferase','NULL','NULL','Isolated from the plant Daucus carota (Afghan cultivar carrot).','UDP-alpha-D-xylose + cyanidin 3-O-beta-D-galactoside = UDP + cyanidin3-O-(beta-D-xylosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.51','Anthocyanidin 3-O-glucoside 2-O-xylosyltransferase','NULL','NULL','Isolated from the plants Matthiola incana (stock) and Arabidopsisthaliana (mouse-eared cress).The enzyme has similar activity with the 3-glucosides ofpelargonidin, cyanidin, delphinidin, quercetin and kaempferol as wellas with cyanidin 3-O-rhamnosyl-(1-','UDP-alpha-D-xylose + an anthocyanidin 3-O-beta-D-glucoside = UDP +an anthocyanidin 3-O-beta-D-sambubioside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.52','Triphosphoribosyl-dephospho-CoA synthase','2-(5-triphosphoribosyl)-3-dephospho-CoA synthase','NULL','ATP cannot be replaced by GTP, CTP, UTP, ADP or AMP.The reaction involves the formation of a new alpha (1-','ATP + 3-dephospho-CoA = 2-(5-triphospho-alpha-D-ribosyl)-3-dephospho-CoA + adenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.53','Undecaprenyl-phosphate 4-deoxy-4-formamido-L-arabinose transferase','Ara4FN transferase','NULL','The enzyme shows no activity with UDP-4-amino-4-deoxy-beta-L-arabinose.Formerly EC 2.7.8.30 and EC 2.7.8.n1.','UDP-4-deoxy-4-formamido-beta-L-arabinose + ditrans,octacis-undecaprenylphosphate = UDP + 4-deoxy-4-formamido-alpha-L-arabinose ditrans,octacis-undecaprenyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.54','Beta-ribofuranosylaminobenzene 5-phosphate synthase','4-(beta-D-ribofuranosyl)aminobenzene 5-phosphate synthase','Manganese or magnesium.','Isolated from the archaea Methanosarcina thermophila andArchaeoglobus fulgidus.The enzyme is involved in biosynthesis of tetrahydromethanopterin inarchaea.','5-phospho-alpha-D-ribose 1-diphosphate + 4-aminobenzoate = 4-(beta-D-ribofuranosyl)aniline 5-phosphate + CO(2) + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.55','Nicotinate D-ribonucleotide:phenol phospho-D-ribosyltransferase','NULL','NULL','The enzyme is involved in the biosynthesis of phenolic cobamides inthe Gram-positive bacterium Sporomusa ovata.It can also transfer the phospho-D-ribosyl group to 4-methylphenoland 5,6-dimethylbenzimidazole.The related EC 2.4.2.21 also transfers the phospho-D-ribosyl groupfrom nicotinate D-ribonucleotide to 5,6-dimethylbenzimidazole,but shows no activity with 4-methylphenol or phenol.','Nicotinate D-ribonucleotide + phenol = nicotinate + phenyl 5-phospho-alpha-D-ribofuranoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.56','Kaempferol 3-O-xylosyltransferase','Flavonol 3-O-xylosyltransferase','NULL','The enzyme from the plant Euonymus alatus also catalyzes the 3-O-D-xylosylation of other flavonols (e.g. quercetin, isorhamnetin,rhamnetin, myricetin, fisetin) with lower activity.','UDP-alpha-D-xylose + kaempferol = UDP + kaempferol 3-O-beta-D-xyloside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.57','AMP phosphorylase','AMPpase','NULL','The enzyme from archaea is involved in AMP metabolism and CO(2)fixation through type III RubisCO enzymes.The activity with CMP and UMP requires activation by cAMP.','(1) AMP + phosphate = adenine + alpha-D-ribose 1,5-bisphosphate.(2) CMP + phosphate = cytosine + alpha-D-ribose 1,5-bisphosphate.(3) UMP + phosphate = uracil + alpha-D-ribose 1,5-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.2.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.1','Beta-galactoside alpha-2,6-sialyltransferase','Beta-galactosamide alpha-2,6-sialyltransferase','NULL','The terminal beta-D-galactosyl residue of the oligosaccharide ofglycoproteins, as well as lactose, can act as acceptor.','CMP-N-acetylneuraminate + beta-D-galactosyl-1,4-N-acetyl-beta-D-glucosamine = CMP + alpha-N-acetylneuraminyl-2,6-beta-D-galactosyl-1,4-N-acetyl-beta-D-glucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.2','Monosialoganglioside sialyltransferase','NULL','NULL','May be identical with EC 2.4.99.4.','CMP-N-acetylneuraminate + D-galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosyl-(1', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.3','Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase','GalNAc alpha-2,6-sialyltransferase I','NULL','Alpha-N-acetylgalactosamine linked to threonine or serine is also anacceptor, when substituted at the 3-position.','CMP-N-acetylneuraminate + glycano-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.4','Beta-galactoside alpha-2,3-sialyltransferase','CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase','NULL','The acceptor is Gal-beta-1,3-GalNAc-R, where R is H, a threonine orserine residue in a glycoprotein, or a glycolipid.Lactose can also act as acceptor.May be identical with EC 2.4.99.2.','CMP-N-acetylneuraminate + beta-D-galactosyl-1,3-N-acetyl-alpha-D-galactosaminyl-R = CMP + alpha-N-acetylneuraminyl-2,3-beta-D-galactosyl-1,3-N-acetyl-alpha-D-galactosaminyl-R.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.5','Galactosyldiacylglycerol alpha-2,3-sialyltransferase','NULL','NULL','The beta-D-galactosyl residue of the oligosaccharide of glycoproteinsmay also act as acceptor.','CMP-N-acetyl-beta-neuraminate + 1,2-diacyl-3-beta-D-galactosyl-sn-glycerol = CMP + 1,2-diacyl-3-(3-(N-acetyl-alpha-D-neuraminyl)-beta-D-galactosyl)-sn-glycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.6','N-acetyllactosaminide alpha-2,3-sialyltransferase','NULL','NULL','Acts on beta-D-galactosyl-1,4-N-acetyl-D-glucosaminyl termini onglycoprotein.','CMP-N-acetylneuraminate + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.7','Alpha-N-acetylneuraminyl-2,3-beta-galactosyl-1,3-N-acetylgalactosaminide6-alpha-sialyltransferase','Alpha-N-acetylneuraminyl-2,3-beta-galactosyl-1,3-N-acetyl-galactosaminidealpha-2,6-sialyltransferase','NULL','Attaches N-acetylneuraminic acid in alpha-2,6-linkage to N-acetyl-galactosamine only when present in the structure of alpha-N-acetyl-neuraminyl-(2-','CMP-N-acetylneuraminate + N-acetyl-alpha-neuraminyl-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.8','Alpha-N-acetylneuraminate alpha-2,8-sialyltransferase','Alpha-2,8-sialyltransferase','NULL','Gangliosides act as acceptors.','CMP-N-acetylneuraminate + alpha-N-acetylneuraminyl-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.9','Lactosylceramide alpha-2,3-sialyltransferase','Ganglioside GM3 synthase','NULL','Lactose cannot act as acceptor.','CMP-N-acetylneuraminate + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.10','Neolactotetraosylceramide alpha-2,3-sialyltransferase','Sialyltransferase','NULL','NULL','CMP-N-acetylneuraminate + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.11','Lactosylceramide alpha-2,6-N-sialyltransferase','NULL','NULL','NULL','CMP-N-acetylneuraminate + beta-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.12','Lipid IV(A) 3-deoxy-D-manno-octulosonic acid transferase','3-deoxy-D-manno-oct-2-ulosonic acid transferase','NULL','The bifunctional enzyme from Escherichia coli transfers two 3-deoxy-D-manno-octulosonate residues to lipid IV(A) (cf. EC 2.4.99.13).The monofunctional enzymes from Aquifex aeolicus and Haemophilusinfluenzae catalyze the transfer of a single 3-deoxy-D-manno-octulosonate residue from CMP-3-deoxy-D-manno-octulosonate to lipidIV(A).The enzymes from Chlamydia transfer three or more 3-deoxy-D-manno-octulosonate residues and generate genus-specific epitopes.','Lipid IV(A) + CMP-alpha-Kdo = alpha-Kdo-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.13','(KDO)-lipid IV(A) 3-deoxy-D-manno-octulosonic acid transferase','3-deoxy-D-manno-oct-2-ulosonic acid transferase','NULL','The bifunctional enzyme from Escherichia coli transfers two 3-deoxy-D-manno-octulosonate residues to lipid IV(A) (cf. EC 2.4.99.12).The enzymes from Chlamydia transfer three or more 3-deoxy-D-manno-octulosonate residues and generate genus-specific epitopes.','Alpha-Kdo-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.14','(KDO)(2)-lipid IV(A) (2-8) 3-deoxy-D-manno-octulosonic acid transferase','3-deoxy-D-manno-oct-2-ulosonic acid transferase','NULL','The enzymes from Chlamydia transfer three or more 3-deoxy-D-manno-octulosonate residues and generate genus-specific epitopes.','Alpha-Kdo-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.15','(KDO)(3)-lipid IV(A) (2-4) 3-deoxy-D-manno-octulosonic acid transferase','3-deoxy-D-manno-oct-2-ulosonic acid transferase','NULL','The enzyme from Chlamydia psittaci transfers four KDO residues tolipid A, forming a branched tetrasaccharide with the structure alpha-KDO-(2,8)-(alpha-KDO-(2,4))-alpha-KDO-(2,4)-alpha-KDO (cf.EC 2.4.99.12, EC 2.4.99.13, and EC 2.4.99.14).','Alpha-Kdo-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.16','Starch synthase (maltosyl-transferring)','Alpha-1,4-glucan:maltose-1-P maltosyltransferase','NULL','The enzyme from from the bacterium Mycobacterium smegmatis isspecific for maltose.It has no activity with alpha-D-glucose.Formerly EC 2.4.1.n5.','Alpha-maltose 1-phosphate + ((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.17','S-adenosylmethionine:tRNA ribosyltransferase-isomerase','Queuosine biosynthesis protein QueA','NULL','The reaction is a combined transfer and isomerization of the ribosemoiety of S-adenosyl-L-methionine to the modified guanosine base inthe wobble position in tRNAs specific for Tyr, His, Asp or Asn.It is part of the queuosine biosynthesis pathway.','S-adenosyl-L-methionine + 7-aminomethyl-7-carbaguanosine(34) in tRNA =L-methionine + adenine + epoxyqueuosine(34) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.18','Dolichyl-diphosphooligosaccharide--protein glycotransferase','NULL','NULL','Occurs in eukaryotes that form a glycoprotein by the transfer of aglucosyl-mannosyl-glucosamine polysaccharide to the side-chain of anL-asparagine residue in the sequence -Asn-Xaa-Ser- or -Asn-Xaa-Thr-(Xaa not Pro) in nascent polypeptide chains.The basic oligosaccharide is the tetradecasaccharideGlc(3)Man(9)GlcNAc(2).However, smaller oligosaccharides derived from it andoligosaccharides with additional monosaccharide units attached may beinvolved.Man(3)GlcNAc(2) seems to be common for all of the oligosaccharidesinvolved with the terminal N-acetylglucosamine linked to the proteinL-asparagine.Occurs on the cytosolic face of the endoplasmic reticulum.The dolichol involved normally has 14-21 isoprenoid units with twotrans double-bonds at the omega end, and the rest of the double-bondsin cis form.Formerly EC 2.4.1.119.','Dolichyl diphosphooligosaccharide + [protein]-L-asparagine = dolichyldiphosphate + a glycoprotein with the oligosaccharide chain attached byN-beta-D-glycosyl linkage to a protein L-asparagine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.19','Undecaprenyl-diphosphooligosaccharide--protein glycotransferase','NULL','Manganese or magnesium.','A bacterial enzyme that has been isolated from Campylobacter jejuniand Campylobacter lari.It forms a glycoprotein by the transfer of a glucosyl-N-acetylgalactosaminyl-N,N-diacetylbacillosamine(GalNAc(2)(Glc)GalNAc(3)diNAcBac) polysaccharide and relatedoligosaccharides to the side-chain of an L-asparagine residue in thesequence -Asp/Glu-Xaa-Asn-Xaa-Ser/Thr- (Xaa and Xaa not Pro) innascent polypeptide chains.Occurs on the external face of the plasma membrane.The polyprenol involved is normally tritrans,heptacis-undecaprenolbut a decaprenol is used by some species.','Tritrans,heptacis-undecaprenyl diphosphooligosaccharide + [protein]-L-asparagine = tritrans,heptacis-undecaprenyl diphosphate + a glycoproteinwith the oligosaccharide chain attached by N-beta-D-glycosyl linkage toprotein L-asparagine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.4.99.20','2-phospho-ADP-ribosyl cyclase/2-phospho-cyclic-ADP-ribose transferase','Diphosphopyridine nucleosidase','NULL','This multiunctional enzyme catalyzes both the removal of nicotinamidefrom NADP(+), forming 2-phospho-cyclic ADP-ribose, and the additionof nicotinate to the cyclic product, forming NAADP(+), a calciummessenger that can mobilize intracellular Ca(2+) stores and activateCa(2+) influx to regulate a wide range of physiological processes.In addition, the enzyme also catalyzes EC 3.2.2.6.','NADP(+) + nicotinate = nicotinate-adenine dinucleotide phosphate +nicotinamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.1','Dimethylallyltranstransferase','Dimethylallyltransferase','NULL','Will not accept larger prenyl diphosphates as efficient donors.','Dimethylallyl diphosphate + isopentenyl diphosphate = diphosphate +geranyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.2','Thiamine pyridinylase','Pyrimidine transferase','NULL','Various bases and thiol compounds can act instead of pyridine.','Thiamine + pyridine = 1-((4-amino-2-methylpyrimidin-5-yl)methyl)pyridinium + 4-methyl-5-(2-hydroxyethyl)thiazole.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.3','Thiamine-phosphate diphosphorylase','Thiamine-phosphate pyrophosphorylase','NULL','NULL','2-methyl-4-amino-5-hydroxymethylpyrimidine diphosphate + 4-methyl-5-(2-phosphono-oxyethyl)thiazole = diphosphate + thiamine phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.4','Adenosylmethionine cyclotransferase','NULL','NULL','NULL','S-adenosyl-L-methionine = S-methyl-5-thioadenosine + 2-aminobutan-4-olide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.5','Galactose-6-sulfurylase','Galactose-6-sulfatase','NULL','NULL','Eliminates sulfate from the D-galactose 6-sulfate residues of porphyran,producing 3,6-anhydrogalactose residues.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.6','Methionine adenosyltransferase','AdoMet synthetase','NULL','Formerly EC 2.4.2.13.','ATP + L-methionine + H(2)O = phosphate + diphosphate + S-adenosyl-L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.7','UDP-N-acetylglucosamine 1-carboxyvinyltransferase','Enoylpyruvate transferase','NULL','NULL','Phosphoenolpyruvate + UDP-N-acetyl-alpha-D-glucosamine = phosphate + UDP-N-acetyl-3-O-(1-carboxyvinyl)-alpha-D-glucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.8','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.9','Riboflavin synthase','NULL','NULL','NULL','2 6,7-dimethyl-8-(1-D-ribityl)lumazine = riboflavin + 4-(1-D-ribitylamino)-5-amino-2,6-dihydroxypyrimidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.10','(2E,6E)-farnesyl diphosphate synthase','Farnesyl pyrophosphate synthetase','NULL','Some forms of this enzyme will also use dimethylallyl diphosphate asa substrate.The enzyme will not accept larger prenyl diphosphates as efficientdonors.','Geranyl diphosphate + isopentenyl diphosphate = diphosphate + (2E,6E)-farnesyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.11','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.12','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.13','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.14','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.15','Dihydropteroate synthase','DHPS','NULL','NULL','(2-amino-4-hydroxy-7,8-dihydropteridin-6-yl)methyl diphosphate +4-aminobenzoate = diphosphate + dihydropteroate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.16','Spermidine synthase','Aminopropyltransferase','NULL','The enzymes from the plant Glycine max and from mammalia are highlyspecific for putrescine as the amine acceptor.The enzymes from the bacteria Escherichia coli and Thermotogamaritima prefer putrescine but are more tolerant toward other amineacceptors, such as spermidine and cadaverine.Cf. EC 2.5.1.22 and EC 2.5.1.23.','S-adenosyl 3-(methylthio)propylamine + putrescine = 5-S-methyl-5-thioadenosine + spermidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.17','Cob(I)yrinic acid a,c-diamide adenosyltransferase','Aquacob(I)alamin adenosyltransferase','NULL','The corrinoid adenosylation pathway comprises three steps:(1) Reduction of Co(III) to Co(II) by a one-electron transfer; thiscan be carried out by EC 1.16.1.3, or non-enzymically in the presenceof dihydroflavin nucleotides.(2) Co(II) is reduced to Co(I) in a second single-electron transferby EC 1.16.1.4.(3) The Co(I) conducts a nucleophilic attack on the adenosyl moietyof ATP to leave the cobalt atom in a Co(III) state (EC 2.5.1.17).The enzyme responsible for the adenosylation reaction is the productof the gene cobO in the aerobic bacterium Pseudomonas denitrificansand of the gene cobA in the anaerobic bacterium Salmonellatyphimurium.In P.denitrificans, the enzyme shows specificity for cobyrinic acida,c-diamide and the corrinoids that occur later in the biosyntheticpathway whereas CobA seems to have broader specificity.While CobA has a preference for ATP and Mn(2+), it is able totransfer a variety of nucleosides to the cobalt, including CTP,UTP and GTP, in decreasing order of preference and to use Mg(2+)instead of Mn(2+).','(1) ATP + cob(I)yrinic acid a,c-diamide = triphosphate +adenosylcob(III)yrinic acid a,c-diamide.(2) ATP + cobinamide = triphosphate + adenosylcobinamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.18','Glutathione transferase','Glutathione S-alkyltransferase','NULL','A group of enzymes of broad specificity.R may be an aliphatic, aromatic or heterocyclic group; X may be asulfate, nitrile or halide group.Also catalyzes the addition of aliphatic epoxides and arene oxides toglutathione, the reduction of polyol nitrate by glutathione to polyoland nitrile, certain isomerization reactions and disulfideinterchange.Formerly EC 1.8.6.1, EC 2.5.1.12, EC 2.5.1.13, EC 2.5.1.14 andEC 4.4.1.7.','RX + glutathione = HX + R-S-glutathione.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.19','3-phosphoshikimate 1-carboxyvinyltransferase','3-enol-pyruvoylshikimate-5-phosphate synthase','NULL','NULL','Phosphoenolpyruvate + 3-phosphoshikimate = phosphate + 5-O-(1-carboxyvinyl)-3-phosphoshikimate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.20','Rubber cis-polyprenylcistransferase','Rubber allyltransferase','NULL','Rubber particles act as acceptor.','Polycis-polyprenyl diphosphate + isopentenyl diphosphate = diphosphate +a polycis-polyprenyl diphosphate longer by one C(5) unit.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.21','Squalene synthase','Farnesyl-diphosphate farnesyltransferase','Manganese or magnesium.','This microsomal enzyme catalyzes the first committed step in thebiosynthesis of sterols.In the absence of NAD(P)H, presqualene diphosphate (PSPP) isaccumulated.When NAD(P)H is present, presqualene diphosphate does not dissociatefrom the enzyme during the synthesis of squalene from farnesyldiphosphate (FPP).High concentrations of FPP inhibit the production of squalene but notof PSPP.','2 farnesyl diphosphate + NAD(P)H = squalene + 2 diphosphate + NAD(P)(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.22','Spermine synthase','Spermidine aminopropyltransferase','NULL','The enzyme from mammalia is highly specific for spermidine,  cf.EC 2.5.1.16 and EC 2.5.1.23.','S-adenosyl 3-(methylthio)propylamine + spermidine = S-methyl-5-thioadenosine + spermine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.23','Sym-norspermidine synthase','NULL','NULL','The enzyme has been originally characterized from the protist Euglenagracilis.The enzyme from the archaeon Sulfolobus solfataricus can transfer thepropylamine moiety from S-adenosyl 3-(methylthio)propylamine toputrescine, sym-norspermidine and spermidine with lower efficiency.Cf. EC 2.5.1.16 and EC 2.5.1.22.','S-adenosyl 3-(methylthio)propylamine + propane-1,3-diamine = S-methyl-5-thioadenosine + bis(3-aminopropyl)amine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.24','Discadenine synthase','Discadenine synthetase','NULL','NULL','S-adenosyl-L-methionine + N(6)-(Delta(2)-isopentenyl)-adenine = S-methyl-5-thioadenosine + discadenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.25','tRNA-uridine aminocarboxypropyltransferase','NULL','NULL','NULL','S-adenosyl-L-methionine + tRNA uridine = S-methyl-5-thioadenosine + tRNA3-(3-amino-3-carboxypropyl)-uridine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.26','Alkylglycerone-phosphate synthase','Alkyl-DHAP synthase','NULL','The ester-linked fatty acid of the substrate is cleaved and replacedby a long-chain alcohol in an ether linkage.','1-acyl-glycerone 3-phosphate + a long-chain alcohol = an alkyl-glycerone3-phosphate + a long-chain acid anion.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.27','Adenylate dimethylallyltransferase (AMP-dependent)','Adenylate isopentenyltransferase','NULL','Involved in the biosynthesis of cytokinins in plants.Some isoforms from the plant Arabidopsis thaliana are specific forAMP while others also have the activity of EC 2.5.1.112.','Dimethylallyl diphosphate + AMP = diphosphate + N(6)-(dimethylallyl)adenosine 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.28','Dimethylallylcistransferase','Neryl-diphosphate synthase','NULL','Will not use larger prenyl diphosphates as efficient donors.','Dimethylallyl diphosphate + isopentenyl diphosphate = diphosphate +neryl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.29','Geranylgeranyl diphosphate synthase','Farnesyltransferase','NULL','Some forms of this enzyme will also use geranyl diphosphate anddimethylallyl diphosphate as donors; it will not use larger prenyldiphosphates as efficient donors.','(2E,6E)-farnesyl diphosphate + isopentenyl diphosphate = diphosphate +geranylgeranyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.30','Heptaprenyl diphosphate synthase','All-trans-heptaprenyl-diphosphate synthase','NULL','This enzyme catalyzes the condensation reactions resulting in theformation of all-trans-heptaprenyl diphosphate, the isoprenoid sidechain of ubiquinone-7 and menaquinone-7.The enzyme adds four isopentenyl diphosphate molecules sequentiallyto farnesyl diphosphate with trans stereochemistry.','(2E,6E)-farnesyl diphosphate + 4 isopentenyl diphosphate = 4 diphosphate+ all-trans-heptaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.31','Ditrans,polycis-undecaprenyl-diphosphate synthase ((2E,6E)-farnesyl-diphosphate specific)','Bactoprenyl-diphosphate synthase','NULL','Undecaprenyl pyrophosphate synthase catalyzes the consecutivecondensation reactions of a farnesyl diphosphate with eightisopentenyl diphosphates, in which new cis-double bonds are formed,to generate undecaprenyl diphosphate that serves as a lipid carrierfor peptidoglycan synthesis of bacterial cell wall.','(2E,6E)-farnesyl diphosphate + 8 isopentenyl diphosphate = 8 diphosphate+ di-trans,octa-cis-undecaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.32','15-cis-phytoene synthase','Geranylgeranyl-diphosphate geranylgeranyltransferase','Manganese.','The enzyme produces 15-cis-phytoene.Cf. EC 2.5.1.99.','2 geranylgeranyl diphosphate = 15-cis-phytoene + 2 diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.33','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.34','4-dimethylallyltryptophan synthase','Tryptophan dimethylallyltransferase','NULL','NULL','Dimethylallyl diphosphate + L-tryptophan = diphosphate + 4-(3-methylbut-2-enyl)-L-tryptophan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.35','Aspulvinone dimethylallyltransferase','NULL','NULL','Will also use as acceptor aspulvinone G, a hydroxylated derivative ofthe complex phenolic pigment aspulvinone E.','2 dimethylallyl diphosphate + aspulvinone E = 2 diphosphate +aspulvinone H.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.36','Trihydroxypterocarpan dimethylallyltransferase','Dimethylallyl-diphosphate:(6aS,11aS)-3,6a,9-trihydroxypterocarpandimethyltransferase','NULL','Part of the glyceollin biosynthesis system in soybean.','(1) Dimethylallyl diphosphate + (6aS,11aS)-3,6a,9-trihydroxypterocarpan =diphosphate + 2-dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan.(2) Dimethylallyl diphosphate + (6aS,11aS)-3,6a,9-trihydroxypterocarpan =diphosphate + 4-dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.37','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.38','Isonocardicin synthase','NULL','NULL','Involved in the biosynthesis of the beta-lactam antibiotic nocardicinA.','S-adenosyl-L-methionine + nocardicin E = S-methyl-5-thioadenosine +isonocardicin A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.39','4-hydroxybenzoate polyprenyltransferase','4-hydroxybenzoate nonaprenyltransferase','NULL','This enzyme, involved in the biosynthesis of ubiquinone, attaches apolyprenyl side chain to a 4-hydroxybenzoate ring, producing thefirst ubiquinone intermediate that is membrane bound.The number of isoprenoid subunits in the side chain varies indifferent species.The enzyme does not have any specificity concerning the length of thepolyprenyl tail, and accepts tails of various lengths with similarefficiency.','A polyprenyl diphosphate + 4-hydroxybenzoate = diphosphate + a 4-hydroxy-3-polyprenylbenzoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.40','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.41','Phosphoglycerol geranylgeranyltransferase','(S)-3-O-geranylgeranylglyceryl phosphate synthase','Magnesium.','Catalyzes the first pathway-specific step in the biosynthesis of thecore membrane diether lipids in archaebacteria.It catalyzes the alkylation of the primary hydroxy group insn-glycerol 1-phosphate by geranylgeranyl diphosphate (GGPP) in aprenyltransfer reaction where a hydroxy group is the nucleophile inthe acceptor substrate.The other enzymes involved in the biosynthesis of polar lipids inarchaea are EC 1.1.1.261, EC 2.5.1.42 and EC 2.7.7.67, which lead tothe formation of CDP-unsaturated archaeol.The final step in the pathway involves the addition of L-serine, withconcomitant removal of CMP, leading to the production of unsaturatedarchaetidylserine.','Geranylgeranyl diphosphate + sn-glycerol 1-phosphate = diphosphate +sn-3-O-(geranylgeranyl)glycerol 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.42','Geranylgeranylglycerol-phosphate geranylgeranyltransferase','(S)-2,3-di-O-geranylgeranylglyceryl phosphate synthase','Magnesium.','Carries out the second prenyltransfer reaction involved in theformation of polar membrane lipids in archaea.4-hydroxybenzoate, 1,4-dihydroxy 2-naphthoate, homogentisate andalpha-glycerophosphate cannot act as prenyl-acceptor substrates.The other enzymes involved in the biosynthesis of polar lipids inarchaea are EC 1.1.1.261, EC 2.5.1.41, which, together with thisenzyme, alkylates the hydroxy groups of glycerol 1-phosphate to yieldunsaturated archaetidic acid, which is acted upon by EC 2.7.7.67 toform CDP-unsaturated archaeol.The final step in the pathway involves the addition of L-serine, withconcomitant removal of CMP, leading to the production of unsaturatedarchaetidylserine.','Geranylgeranyl diphosphate + sn-3-O-(geranylgeranyl)glycerol 1-phosphate= diphosphate + 2,3-bis-O-(geranylgeranyl)glycerol 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.43','Nicotianamine synthase','NULL','NULL','NULL','3 S-adenosyl-L-methionine = 3 S-methyl-5-thioadenosine + nicotianamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.44','Homospermidine synthase','NULL','NAD(+).','The reaction of this enzyme occurs in three steps, with some of theintermediates presumably remaining enzyme-bound: NAD(+)-dependentdehydrogenation of putrescine, transfer of the 4-aminobutylidenegroup from dehydroputrescine to a second molecule of putrescine andreduction of the imine intermediate to form homospermidine.Hence the overall reaction is transfer of a 4-aminobutyl group.Differs from EC 2.5.1.45, which cannot use putrescine as donor of theaminobutyl group.','(1) 2 putrescine = sym-homospermidine + NH(3) + H(+).(2) Putrescine + spermidine = sym-homospermidine + propane-1,3-diamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.45','Homospermidine synthase (spermidine-specific)','NULL','NAD(+).','The reaction of this enzyme occurs in three steps:(1) NAD(+)-dependent dehydrogenation of spermidine.(2) Transfer of the 4-aminobutylidene group from dehydrospermidine toa second molecule of putrescine.(3) Reduction of the imine intermediate to form homospermidine.Hence the overall reaction is transfer of a 4-aminobutyl group.This enzyme is more specific than EC 2.5.1.44 which is found inbacteria, as it cannot use putrescine as donor of the 4-aminobutylgroup.Forms part of the biosynthetic pathway of the poisonous pyrrolizidinealkaloids of the ragworts (Senecio).','Spermidine + putrescine = sym-homospermidine + propane-1,3-diamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.46','Deoxyhypusine synthase','(4-aminobutyl)lysine synthase','NAD(+).','The eukaryotic initiation factor eIF5A contains a hypusine residuethat is essential for activity.Catalyzes the first reaction of hypusine formation from one specificlysine residue of the eIF5A precursor.The reaction occurs in four steps: NAD(+)-dependent dehydrogenationof spermidine (1a), formation of an enzyme-imine intermediate bytransfer of the 4-aminobutylidene group from dehydrospermidine to theactive site lysine residue (1b), transfer of the same4-aminobutylidene group from the enzyme intermediate to the e1F5Aprecursor (1c), reduction of the e1F5A-imine intermediate to form adeoxyhypusine residue (1d).Hence the overall reaction is transfer of a 4-aminobutyl group.For the plant enzyme, homospermidine can substitute for spermidineand putrescine can substitute for the lysine residue of the eIF5Aprecursor.Hypusine is formed from deoxyhypusine by the action of EC 1.14.99.29.Formerly EC 1.1.1.249.','[eIF5A-precursor]-lysine + spermidine = [eIF5A-precursor]-deoxyhypusine +propane-1,3-diamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.47','Cysteine synthase','Acetylserine sulfhydrylase','Pyridoxal 5-phosphate.','Some alkyl thiols, cyanide, pyrazole and some other heterocycliccompounds can act as acceptors.Not identical with EC 2.5.1.51, EC 2.5.1.52 and EC 2.5.1.53.Formerly EC 4.2.99.8.','O(3)-acetyl-L-serine + H(2)S = L-cysteine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.48','Cystathionine gamma-synthase','Cystathionine synthase','Pyridoxal 5-phosphate.','Also reacts with H(2)S and methanethiol as replacing agents,producing homocysteine and methionine, respectively.In the absence of thiol, can also catalyze beta,gamma-elimination toform 2-oxobutanoate, succinate and ammonia.Formerly EC 4.2.99.9.','O(4)-succinyl-L-homoserine + L-cysteine = L-cystathionine + succinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.49','O-acetylhomoserine aminocarboxypropyltransferase','Methionine synthase','NULL','Also reacts with other thiols and H(2)S, producing homocysteine orthioethers.The name methionine synthase is more commonly applied to EC 2.1.1.13.The enzyme from Saccharomyces cerevisiae also catalyzes the reactionof EC 2.5.1.47, but more slowly.Formerly EC 4.2.99.10.','O-acetyl-L-homoserine + methanethiol = L-methionine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.50','Zeatin 9-aminocarboxyethyltransferase','Beta-(9-cytokinin)-alanine synthase','NULL','The enzyme acts not only on zeatin but also on other N(6)-substitutedadenines.The reaction destroys their cytokinin activity and forms thecorresponding 3-(adenin-9-yl)-L-alanine.Formerly EC 4.2.99.13.','O-acetyl-L-serine + zeatin = lupinate + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.51','Beta-pyrazolylalanine synthase','Beta-(1-pyrazolyl)alanine synthase','NULL','Highly specific for acetylserine and pyrazole.Not identical with EC 2.5.1.52.Formerly EC 4.2.99.14 and EC 4.2.99.17.','O(3)-acetyl-L-serine + pyrazole = 3-(pyrazol-1-yl)-L-alanine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.52','L-mimosine synthase','O(3)-acetyl-L-serine acetate-lyase (adding 3,4-dihydroxypyridin-1-yl)','NULL','Brings about the biosynthesis of L-mimosine in Leucaena sp. andMimosa.Not identical with EC 2.5.1.51.Formerly EC 4.2.99.15.','O(3)-acetyl-L-serine + 3,4-dihydroxypyridine = 3-(3,4-dihydroxypyridin-1-yl)-L-alanine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.53','Uracilylalanine synthase','Isowillardiine synthase','NULL','Both L-willardiine and L-isowillardiine are produced in the reaction.Not identical with EC 2.5.1.47.Formerly EC 4.2.99.16.','O(3)-acetyl-L-serine + uracil = 3-(uracil-1-yl)-L-alanine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.54','3-deoxy-7-phosphoheptulonate synthase','2-dehydro-3-deoxy-phosphoheptonate aldolase','NULL','Formerly EC 4.1.2.15.','Phosphoenolpyruvate + D-erythrose 4-phosphate + H(2)O = 3-deoxy-D-arabino-hept-2-ulosonate 7-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.55','3-deoxy-8-phosphooctulonate synthase','2-dehydro-3-deoxy-D-octonate-8-phosphate D-arabinose-5-phosphate-lyase(pyruvate-phosphorylating)','NULL','Formerly EC 4.1.2.16.','Phosphoenolpyruvate + D-arabinose 5-phosphate + H(2)O = 2-dehydro-3-deoxy-D-octonate 8-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.56','N-acetylneuraminate synthase','(NANA)condensing enzyme','NULL','Formerly EC 4.1.3.19.','Phosphoenolpyruvate + N-acetyl-D-mannosamine + H(2)O = phosphate +N-acetylneuraminate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.57','N-acylneuraminate-9-phosphate synthase','N-acetylneuraminate 9-phosphate lyase','NULL','Acts on N-glycoloyl and N-acetyl-derivatives.Formerly EC 4.1.3.20.','Phosphoenolpyruvate + N-acyl-D-mannosamine 6-phosphate + H(2)O =N-acylneuraminate 9-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.58','Protein farnesyltransferase','CAAX farnesyltransferase','Magnesium; Zinc.','This enzyme, along with EC 2.5.1.59 and EC 2.5.1.60, constitutes theprotein prenyltransferase family of enzymes.Catalyzes the formation of a thioether linkage between the C-1 of anisoprenyl group and a cysteine residue fourth from the C-terminus ofthe protein.These protein acceptors have the C-terminal sequence CA(1)A(2)X,where the terminal residue, X, is preferably serine, methionine,alanine or glutamine; leucine makes the protein a substrate forEC 2.5.1.59.The enzymes are relaxed in specificity for A(1), but cannot act ifA(2) is aromatic.Substrates of the prenyltransferases include Ras, Rho, Rab, otherRas-related small GTP-binding proteins, gamma-subunits ofheterotrimeric G-proteins, nuclear lamins, centromeric proteins andmany proteins involved in visual signal transduction.','Farnesyl diphosphate + protein-cysteine = S-farnesyl protein +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.59','Protein geranylgeranyltransferase type I','GGTase-I','Zinc.','This enzyme, along with EC 2.5.1.58 and EC 2.5.1.60, constitutes theprotein prenyltransferase family of enzymes.Catalyzes the formation of a thioether linkage between the C-1 atomof the geranylgeranyl group and a cysteine residue fourth from theC-terminus of the protein.These protein acceptors have the C-terminal sequence CA(1)A(2)X,where the terminal residue, X, is preferably leucine; serine,methionine, alanine or glutamine makes the protein a substrate forEC 2.5.1.58.The enzymes are relaxed in specificity for A(1), but cannot act ifA(2) is aromatic.Known targets of this enzyme include most gamma-subunits ofheterotrimeric G proteins and Ras-related GTPases such as members ofthe Ras and Rac/Rho families.The Zn(2+) is required for peptide, but not for isoprenoid, substratebinding.','Geranylgeranyl diphosphate + protein-cysteine = S-geranylgeranyl-protein + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.60','Protein geranylgeranyltransferase type II','GGTaseII','NULL','This enzyme, along with EC 2.5.1.58 and EC 2.5.1.59, constitutes theprotein prenyltransferase family of enzymes.Attaches geranylgeranyl groups to two C-terminal cysteines in Ras-related GTPases of a single family, the Rab family (Ypt/Sec4 in lowereukaryotes) that terminate in XXCC, XCXC and CCXX motifs.Reaction is entirely dependent on the Rab substrate being bound toRab escort protein (REP).Post-translational modification with the geranylgeranyl moiety isessential for Rab GTPases to be able to control the processes ofmembrane docking and fusion.','Geranylgeranyl diphosphate + protein-cysteine = S-geranylgeranyl-protein + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.61','Hydroxymethylbilane synthase','(4-(2-carboxyethyl)-3-(carboxymethyl)pyrrol-2-yl)methyltransferase(hydrolyzing)','Dipyrromethane.','The enzyme works by stepwise addition of pyrrolylmethyl groups untila hexapyrrole is present at the active center.The terminal tetrapyrrole is then hydrolyzed to yield the product,leaving a cysteine-bound dipyrrole on which assembly continues.In the presence of a second enzyme, EC 4.2.1.75, which is oftencalled cosynthase, the product is cyclized to form uroporphyrinogenIII.If EC 4.2.1.75 is absent, the hydroxymethylbilane cyclizesspontaneously to form uroporphyrinogen I.Formerly EC 4.3.1.8.','4 porphobilinogen + H(2)O = hydroxymethylbilane + 4 NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.62','Chlorophyll synthase','NULL','Magnesium.','The enzyme is modified by binding of the first substrate, phytyldiphosphate, before reaction of the modified enzyme with the secondsubstrate, chlorophyllide a, can occur.The reaction also occurs when phytyl diphosphate is replaced bygeranylgeranyl diphosphate.','Chlorophyllide a + phytyl diphosphate = chlorophyll a + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.63','Adenosyl-fluoride synthase','Fluorinase','NULL','NULL','S-adenosyl-L-methionine + fluoride = 5-deoxy-5-fluoroadenosine +L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.64','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.65','O-phosphoserine sulfhydrylase','O-phosphoserine(thiol)-lyase','Pyridoxal 5-phosphate.','The enzyme from Aeropyrum pernix acts on both O-phospho-L-serine andO(3)-acetyl-L-serine, in contrast with EC 2.5.1.47, which acts onlyon O(3)-acetyl-L-serine.','O-phospho-L-serine + H(2)S = L-cysteine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.66','N(2)-(2-carboxyethyl)arginine synthase','CEA synthetase','NULL','Requires thiamine diphosphate and catalyzes the first step in theclavulanic-acid biosynthesis pathway.The 2-hydroxy-3-oxo group transferred from glyceraldehyde 3-phosphateis isomerized during transfer to form the 2-carboxyethyl group.','D-glyceraldehyde 3-phosphate + L-arginine = N(2)-(2-carboxyethyl)-L-arginine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.67','Chrysanthemyl diphosphate synthase','CPPase','Manganese or magnesium.','Chrysanthemyl diphosphate is a monoterpene with a non-head-to-taillinkage; it is unlike most monoterpenoids, which are derived fromgeranyl diphosphate and have isoprene units that are linked head-to-tail.The mechanism of its formation is similar to that of the early stepsof squalene and phytoene biosynthesis.Chrysanthemyl diphosphate is the precursor of chrysanthemic acid,the acid half of the pyrethroid insecticides found in chrysanthemums.','2 dimethylallyl diphosphate = diphosphate + chrysanthemyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.68','(2Z,6E)-farnesyl diphosphate synthase','(Z)-farnesyl diphosphate synthase','Manganese or magnesium.','The product of this reaction is an intermediate in the synthesis ofdecaprenyl phosphate, which plays a central role in the biosynthesisof most features of the mycobacterial cell wall, includingpeptidoglycan, linker unit galactan and arabinan.Neryl diphosphate can also act as substrate.','Geranyl diphosphate + isopentenyl diphosphate = diphosphate + (2Z,6E)-farnesyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.69','Lavandulyl diphosphate synthase','FDS-5','NULL','Lavandulyl diphosphate is a monoterpene with a non-head-to-taillinkage; it is unlike most monoterpenoids, which are derived fromgeranyl diphosphate and have isoprene units that are linked head-to-tail.When this enzyme is incubated with dimethylallyl diphosphate andisopentenyl diphosphate, it also forms the regular monoterpenegeranyl diphosphate.The enzyme from Artemisia tridentata (big sagebrush) forms bothlavandulyl diphosphate and chrysanthemyl diphosphate (seeEC 2.5.1.67) when dimethylally diphosphate is the sole substrate.','2 dimethylallyl diphosphate = diphosphate + lavandulyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.70','Naringenin 8-dimethylallyltransferase','N8DT','Magnesium.','In addition to naringenin, the enzyme can prenylate several otherflavanones at the C-8 position, but more slowly.Along with EC 1.14.13.103 and EC 2.5.1.71, this enzyme forms part ofthe sophoraflavanone-G biosynthesis pathway.','Dimethylallyl diphosphate + (-)-(2S)-naringenin = diphosphate +sophoraflavanone B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.71','Leachianone-G 2-dimethylallyltransferase','Leachianone G 2-dimethylallyltransferase','NULL','The reaction forms the lavandulyl sidechain of sophoraflavanone G bytransferring a dimethylallyl group to the 2 position of anotherdimethylallyl group attached at postiion 8 of leachianone G.The enzyme is specific for dimethylallyl diphosphate as the prenyldonor, as it cannot be replaced by isopentenyl diphosphate or geranyldiphosphate.Euchrenone a(7) (a 5-deoxy derivative of leachianone G) andkenusanone I (a 7-methoxy derivative of leachianone G) can also actas substrates, but more slowly.Along with EC 1.14.13.103 and EC 2.5.1.70, this enzyme forms part ofthe sophoraflavanone-G biosynthesis pathway.','Dimethylallyl diphosphate + leachianone G = diphosphate +sophoraflavanone G.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.72','Quinolinate synthase','Quinolinate synthetase','Iron-sulfur.','Quinolinate synthase catalyzes the second step in the de novobiosynthesis of NAD(+) from aspartate in some bacteria, withEC 1.4.3.16 catalyzing the first step and EC 2.4.2.19 the third step.In Escherichia coli, two of the residues that are involved in the[4Fe-4S] cluster binding appear to undergo reversible disulfide-bondformation that regulates the activity of the enzyme.','Glycerone phosphate + iminosuccinate = pyridine-2,3-dicarboxylate +2 H(2)O + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.73','O-phospho-L-seryl-tRNA:Cys-tRNA synthase','Sep-tRNA:Cys-tRNA synthase','NULL','In organisms like Archaeoglobus fulgidus lacking EC 6.1.1.16 for thedirect Cys-tRNA(Cys) formation, Cys-tRNA(Cys) is produced by anindirect pathway, in which EC 6.1.1.27 ligates O-phosphoserine totRNA(Cys), and EC 2.5.1.73 converts the produced O-phospho-L-seryl-tRNA(Cys) to Cys-tRNA(Cys).The SepRS/SepCysS pathway is the sole route for cysteine biosynthesisin the organism.Methanosarcina mazei can use both pathways, the direct route usingEC 6.1.1.16 and the indirect pathway with EC 6.1.1.27 andEC 2.5.1.73.Formerly EC 2.5.1.n3.','O-phospho-L-seryl-tRNA(Cys) + sulfide = L-cysteinyl-tRNA(Cys) +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.74','1,4-dihydroxy-2-naphthoate polyprenyltransferase','NULL','NULL','This enzyme catalyzes a step in the synthesis of menaquinone,in which the prenyl chain synthesized by polyprenyl diphosphatesynthase is transferred to 1,4-dihydroxy-2-naphthoate (DHNA).The bacterial enzyme is an inner membrane protein, with theC-terminus located in the periplasm.It is highly specific for DHNA but not for a specific length of theprenyl chain.','An all-trans-polyprenyl diphosphate + 1,4-dihydroxy-2-naphthoate =a demethylmenaquinol + diphosphate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.75','tRNA dimethylallyltransferase','IPP transferase','NULL','Formerly known as tRNA isopentenyltransferase (EC 2.5.1.8), but it isnow known that dimethylallyl diphosphate, rather than isopentenyldiphosphate, is the substrate.Formerly EC 1.8.6.1 and EC 2.5.1.8.','Dimethylallyl diphosphate + adenine(37) in tRNA = diphosphate + N(6)-dimethylallyladenine(37) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.76','Cysteate synthase','NULL','Pyridoxal 5-phosphate.','It is highly specific for O-phospho-L-serine and sulfite.The reaction proceeds through a dehydroalanine (2-aminoacrylic acid)intermediate.The enzyme from Methanosarcina acetivorans is evolutionarily relatedto EC 4.2.3.1, but the reaction is more similar to that ofEC 2.5.1.65.','O-phospho-L-serine + sulfite = L-cysteate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.77','7,8-didemethyl-8-hydroxy-5-deazariboflavin synthase','NULL','Iron-sulfur.','The [4Fe-4S] cluster is coordinated by 3 cysteines and anexchangeable AdoMet molecule.The first stage of catalysis is reduction of the 2 AdoMet to produce2 methionine and 2 5-deoxyadenosin-5-yl radicals that extract ahydrogen from each of the substrates permitting the condensation ofthe two.The overall reaction catalyzed is the transfer of the hydroxybenzylgroup from 4-hydroxyphenylpyruvate (HPP) to 5-amino-6-ribitylaminopyrimidine-2,4(1H,3H)-dione to form 7,8-didemethyl-8-hydroxy-5-deazariboflavin (FO).7,8-didemethyl-8-hydroxy-5-deazariboflavin is the chromophore of thehydride carrier coenzyme F420.','5-amino-6-(D-ribitylamino)uracil + 3-(4-hydroxyphenyl)pyruvate +2 S-adenosyl-L-methionine + H(2)O = 7,8-didemethyl-8-hydroxy-5-deazariboflavin + 2 L-methionine + 2 5-deoxyadenosine + oxalate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.78','6,7-dimethyl-8-ribityllumazine synthase','Lumazine synthase','NULL','Involved in riboflavin biosynthesis.','1-deoxy-L-glycero-tetrulose 4-phosphate + 5-amino-6-(D-ribitylamino)uracil = 6,7-dimethyl-8-(D-ribityl)lumazine + 2 H(2)O +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.79','Thermospermine synthase','NULL','NULL','This enzyme is required for correct xylem specification throughregulation of the lifetime of the xylem elements.','S-adenosyl 3-(methylthio)propylamine + spermidine = S-methyl-5-thioadenosine + thermospermine + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.80','7-dimethylallyltryptophan synthase','7-DMATS','NULL','This enzyme is more flexible toward the aromatic substrate thanEC 2.5.1.34, but similar to that enzyme, accepts only dimethylallyldiphosphate as the prenyl donor.','Dimethylallyl diphosphate + L-tryptophan = diphosphate + 7-(3-methylbut-2-enyl)-L-tryptophan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.81','Geranylfarnesyl diphosphate synthase','FGPP synthase','NULL','The enzyme from Methanosarcina mazei is involved in biosynthesis ofthe polyprenyl side-chain of methanophenazine, an electron carrierutilized for methanogenesis.It prefers geranylgeranyl diphosphate and farnesyl diphosphate asallylic substrate.The enzyme from Aeropyrum pernix prefers farnesyl diphosphate asallylic substrate.The enzyme is involved in the biosynthesis of C(25)-C(25) membranelipids.Formerly EC 2.5.1.n2.','Geranylgeranyl diphosphate + isopentenyl diphosphate = (2E,6E,10E,14E)-geranylfarnesyl diphosphate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.82','Hexaprenyl diphosphate synthase (geranylgeranyl-diphosphate specific)','(All-trans)-hexaprenyl-diphosphate synthase','Magnesium.','The enzyme prefers geranylgeranyl diphosphate to farnesyl diphosphateas an allylic substrate and does not show activity for geranyldiphosphate and dimethylallyl diphosphate.Formerly EC 2.5.1.33.','Geranylgeranyl diphosphate + 2 isopentenyl diphosphate = 2 diphosphate +all-trans-hexaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.83','Hexaprenyl-diphosphate synthase ((2E,6E)-farnesyl-diphosphate specific)','Hexaprenyl diphosphate synthase','NULL','The enzyme prefers farnesyl diphosphate to geranylgeranyl diphosphateas an allylic substrate and does not show activity for geranyldiphosphate and dimethylallyl diphosphate.Formerly EC 2.5.1.33.','(2E,6E)-farnesyl diphosphate + 3 isopentenyl diphosphate = 3 diphosphate+ all-trans-hexaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.84','All-trans-nonaprenyl-diphosphate synthase (geranyl-diphosphate specific)','Nonaprenyl diphosphate synthase','NULL','(2E,6E)-Farnesyl diphosphate and geranylgeranyl diphosphate are lesseffective as substrates than geranyl diphosphate.The enzyme is involved in the synthesis of the side chain ofmenaquinone-9.In Oryza sativa the enzyme SPS2 is involved in providing solanesyldiphosphate for plastoquinone-9 formation.Formerly EC 2.5.1.11.','Geranyl diphosphate + 7 isopentenyl diphosphate = 7 diphosphate + all-trans-nonaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.85','All-trans-nonaprenyl-diphosphate synthase (geranylgeranyl-diphosphatespecific)','Nonaprenyl diphosphate synthase','NULL','Geranylgeranyl diphosphate is preferred over farnesyl diphosphate asallylic substrate.The plant Arabidopsis thaliana has two different enzymes thatcatalyze this reaction.SPS1 contributes to the biosynthesis of the ubiquinone side-chainwhile SPS2 supplies the precursor of the plastoquinone side-chains.Formerly EC 2.5.1.11.','Geranylgeranyl diphosphate + 5 isopentenyl diphosphate = 5 diphosphate +all-trans-nonaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.86','Trans,polycis-decaprenyl diphosphate synthase','(2Z,6Z,10Z,14Z,18Z,22Z,26Z,30Z,34E)-decaprenyl diphosphate synthase','NULL','The enzyme is involved in the biosynthesis of decaprenyl phosphate,which plays a central role in the biosynthesis of essentialmycobacterial cell wall components, such as the mycolyl-arabinogalactan-peptidoglycan complex and lipoarabinomannan.','(2Z,6E)-farnesyl diphosphate + 7 isopentenyl diphosphate = 7 diphosphate+ (2Z,6Z,10Z,14Z,18Z,22Z,26Z,30Z,34E)-decaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.87','Ditrans,polycis-polyprenyl diphosphate synthase ((2E,6E)-farnesyldiphosphate specific)','Dehydrodolichyl diphosphate synthase 1','NULL','The enzyme is involved in biosynthesis of dolichol (a long-chainpolyprenol) with a saturated alpha-isoprene unit, which serves as aglycosyl carrier in protein glycosylation.The yeast Saccharomyces cerevisiae has two different enzymes thatcatalyze this reaction.Rer2p synthesizes a well-defined family of polyprenols of 13-18isoprene residues with dominating C(80) (16 isoprene residues)extending to C(120), while Srt1p synthesizes mainly polyprenol with22 isoprene subunits.Largest Srt1p products reach C(290).The enzyme from Arabidopsis thaliana catalyzes the formation ofpolyprenyl diphosphates with predominant carbon number C(120).','(2E,6E)-farnesyl diphosphate + n isopentenyl diphosphate = n diphosphate+ ditrans,polycis-polyprenyl diphosphate (n = 10-55).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.88','Trans,polycis-polyprenyl diphosphate synthase ((2Z,6E)-farnesyldiphosphate specific)','NULL','NULL','Highest activity with (2Z,6E)-farnesyl diphosphate as allylicsubstrate.Broad product specificity with the major product being dodecaprenyldiphosphate.Synthesizes even C(70) prenyl diphosphate as the maximum chain-lengthproduct.','(2Z,6E)-farnesyl diphosphate + n isopentenyl diphosphate = n diphosphate+ trans,polycis-polyprenyl diphosphate (n = 9-11).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.89','Tritrans,polycis-undecaprenyl-diphosphate synthase (geranylgeranyl-diphosphate specific)','NULL','NULL','This enzyme is involved in the biosynthesis of the glycosyl carrierlipid in some archaebacteria.Unlike EC 2.5.1.31, its counterpart in most bacteria, it prefersgeranylgeranyl diphosphate to farnesyl diphosphate as the allylicsubstrate, resulting in production of a tritrans,polycis variant ofundecaprenyl diphosphate.','Geranylgeranyl diphosphate + 7 isopentenyl diphosphate = 7 diphosphate +tritrans,heptacis-undecaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.90','All-trans-octaprenyl-diphosphate synthase','Octaprenyl pyrophosphate synthetase','NULL','This enzyme catalyzes the condensation reactions resulting in theformation of all-trans-octaprenyl diphosphate, the isoprenoid sidechain of ubiquinone-8 and menaquinone-8.The enzyme adds five isopentenyl diphosphate molecules sequentiallyto farnesyl diphosphate with trans stereochemistry.','(2E,6E)-farnesyl diphosphate + 5 isopentenyl diphosphate = 5 diphosphate+ all-trans-octaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.91','All-trans-decaprenyl-diphosphate synthase','Decaprenyl pyrophosphate synthetase','NULL','This enzyme catalyzes the condensation reactions resulting in theformation of all-trans-decaprenyl diphosphate, the isoprenoid sidechain of ubiquinone-10 and menaquinone-10.The enzyme adds seven isopentenyl diphosphate molecules sequentiallyto farnesyl diphosphate with trans stereochemistry.','(2E,6E)-farnesyl diphosphate + 7 isopentenyl diphosphate = 7 diphosphate+ all-trans-decaprenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.92','(2Z,6Z)-farnesyl diphosphate synthase','Cis,cis-farnesyl diphosphate synthase','NULL','This enzyme, originally characterized from wild tomato, specificallyforms (2Z,6Z)-farnesyl diphosphate via neryl diphosphate andisopentenyl diphosphate.In wild tomato it is involved in the biosynthesis of severalsesquiterpenes.Cf. EC 2.5.1.68 and EC 2.5.1.10.','Dimethylallyl diphosphate + 2 isopentenyl diphosphate = 2 diphosphate +(2Z,6Z)-farnesyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.93','4-hydroxybenzoate geranyltransferase','4HB geranyltransferase','Magnesium.','The enzyme is involved in shikonin biosynthesis.It has a strict substrate specificity for geranyl diphosphate.','Geranyl diphosphate + 4-hydroxybenzoate = 3-geranyl-4-hydroxybenzoate +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.94','Adenosyl-chloride synthase','5-chloro-5-deoxyadenosine synthase','NULL','This enzyme, isolated from the marine bacterium Salinispora tropica,catalyzes an early step in the pathway leading to biosynthesis of theproteosome inhibitor salinosporamide A.The enzyme is very similar to EC 2.5.1.63 but does not acceptfluoride.','S-adenosyl-L-methionine + chloride = 5-deoxy-5-chloroadenosine +L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.95','Xanthan pyruvate transferase','NULL','NULL','Involved in the biosynthesis of the polysaccharide xanthan.30-40% of terminal mannose residues of xanthan have a 4,6-O-(1-carboxyethan-1,1-diyl) ketal group.It also acts on the 6-O-acetyl derivative of the inner mannose unit.','Phosphoenolpyruvate + D-Man-beta-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.96','4,4-diapophytoene synthase','C(30) carotene synthase','Manganese.','Typical of Staphylococcus aureus and some other bacteria such asHeliobacillus sp.','2 (2E,6E)-farnesyl diphosphate = 4,4-diapophytoene + 2 diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.97','Pseudaminic acid synthase','NULL','NULL','The enzyme requires a divalent metal ion, the highest activity valuesare observed in the presence of Mn(2+) and CO(2+) (10 mM).','Phosphoenolpyruvate + 2,4-bis(acetylamino)-2,4,6-trideoxy-beta-L-altropyranose + H(2)O = 5,7-bis(acetylamino)-3,5,7,9-tetradeoxy-L-glycero-alpha-L-manno-2-nonulopyranosonic acid + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.98','Rhizobium leguminosarum exopolysaccharide glucosyl ketal-pyruvate-transferase','NULL','NULL','The enzyme is responsible for pyruvylation of subterminal glucose inthe acidic octasaccharide repeating unit of the exopolysaccharide ofRhizobium leguminosarum (bv. viciae strain VF39) which is necessaryto establish nitrogen-fixing symbiosis with Pisum sativum, Viciafaba, and Vicia sativa.','Phosphoenolpyruvate + [D-GlcA-beta-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.99','All-trans-phytoene synthase','15-trans-phytoene synthase','Manganese.','The enzyme from the bacterium Pantoea agglomerans (previously knownas Erwinia herbicola) produces the all-trans isomer of phytoene.Cf. EC 2.5.1.32.','2 geranylgeranyl diphosphate = all-trans-phytoene + 2 diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.100','Fumigaclavine A dimethylallyltransferase','NULL','NULL','Fumigaclavine C is an ergot alkaloid produced by some fungi of theTrichocomaceae family.Activity does not require any metal ions.','Fumigaclavine A + dimethylallyl diphosphate = fumigaclavine C +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.101','N,N-diacetyllegionaminate synthase','NULL','Manganese.','NULL','2,4-diacetamido-2,4,6-trideoxy-alpha-D-mannopyranose +phosphoenolpyruvate + H(2)O = N,N-diacetyllegionaminate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.102','Geranylpyrophosphate--olivetolic acid geranyltransferase','NULL','NULL','Part of the cannabinoids biosynthetic pathway of the plant Cannabissativa.The enzyme can also use neryl diphosphate as substrate, formingcannabinerolate.','Geranyl diphosphate + 2,4-dihydroxy-6-pentylbenzoate = diphosphate +cannabigerolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.103','Presqualene diphosphate synthase','NULL','NULL','Isolated from the green alga Botryococcus braunii BOT22.Unlike EC 2.5.1.21, where squalene is formed in one step fromfarnesyl diphosphate, in this alga the intermediate presqualenediphosphate is generated and released by this enzyme.This compound is then converted into either squalene (by EC 1.3.1.96)or botryococcene (EC 1.3.1.97).','2 (2E,6E)-farnesyl diphosphate = presqualene diphosphate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.104','N(1)-aminopropylagmatine synthase','ACAPT','NULL','The enzyme is involved in the biosynthesis of spermidine fromagmatine in some archaea and bacteria.The enzyme from the Gram-negative bacterium Thermus thermophilusaccepts agmatine, spermidine and norspermidine with similar catalyticefficiency.The enzymes from the archaea Pyrococcus furiosus and Thermococcuskodakarensis prefer agmatine, but can utilize cadaverine, putrescineand propane-1,3-diamine with much lower catalytic efficiency.Cf. EC 2.5.1.16 and EC 2.5.1.23.','S-adenosylmethioninamine + agmatine = S-methyl-5-thioadenosine + N(1)-(3-aminopropyl)agmatine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.105','7,8-dihydropterin-6-yl-methyl-4-(beta-D-ribofuranosyl)aminobenzene5-phosphate synthase','Dihydropteroate synthase','NULL','The enzyme, which has been studied in the archaeon Methanocaldococcusjannaschii, is involved in the biosynthesis oftetrahydromethanopterin.Formerly EC 2.5.1.n6.','(7,8-dihydropterin-6-yl)methyl diphosphate + 4-(beta-D-ribofuranosyl)aniline 5-phosphate = N-((7,8-dihydropterin-6-yl)methyl)-4-(beta-D-ribofuranosyl)aniline 5-phosphate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.106','Tryprostatin B synthase','Brevianamide F prenyltransferase','NULL','The enzyme from the fungus Aspergillus fumigatus can also prenylateother tryptophan-containing cyclic dipeptides.Prenylation occurs mainly at C-2, but also at C-3.Involved in the biosynthetic pathways of several indole alkaloidssuch as tryprostatins, cyclotryprostatins, spirotryprostatins,fumitremorgins and verruculogen.','Dimethylallyl diphosphate + brevianamide F = diphosphate +tryprostatin B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.107','Verruculogen prenyltransferase','NULL','NULL','Found in a number of fungi.Catalyzes the last step in the biosynthetic pathway of the indolealkaloid fumitremorgin A.The enzyme from the fungus Neosartorya fischeri is also active withfumitremorgin B and 12-alpha,13-alpha-dihydroxyfumitremorgin C.','Dimethylallyl diphosphate + verruculogen = diphosphate + fumitremorgin A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.108','2-(3-amino-3-carboxypropyl)histidine synthase','NULL','Iron-sulfur.','Modifies a histidine residue of the translation elongation factor 2(EF2) via a 3-amino-3-carboxypropyl radical.The enzyme is present in archae and eukaryotes but not in eubacteria.Part of diphthamide biosynthesis.Formerly EC 2.5.1.n7.','L-histidine-[translation elongation factor 2] + S-adenosyl-L-methionine =2-((3S)-3-amino-3-carboxypropyl)-L-histidine-[translation elongationfactor 2] + S-methyl-5-thioadenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.109','Brevianamide F prenyltransferase (deoxybrevianamide E-forming)','Brevianamide F reverse prenyltransferase','NULL','The enzyme from the fungus Aspergilus sp. MF297-2 is specific forbrevianamide F, while the enzyme from Aspergillus versicolor acceptsa broad range of trytophan-containing cyclic dipeptides.Involved in the biosynthetic pathways of several indole alkaloidssuch as paraherquamides and malbrancheamides.','Dimethylallyl diphosphate + brevianamide F = diphosphate +deoxybrevianamide E.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.110','12-alpha,13-alpha-dihydroxyfumitremorgin C prenyltransferase','NULL','NULL','The enzyme from the fungus Aspergillus fumigatus also shows someactivity with fumitremorgin C.Involved in the biosynthetic pathways of several indole alkaloidssuch as fumitremorgins and verruculogen.','Dimethylallyl diphosphate + 12-alpha,13-alpha-dihydroxyfumitremorgin C =diphosphate + fumitremorgin B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.111','4-hydroxyphenylpyruvate 3-dimethylallyltransferase','4HPP dimethylallyltransferase','NULL','The enzyme is involved in the biosynthesis of the 3-dimethylallyl-4-hydroxyphenylpyruvate moiety of the aminocoumarin antibioticsclorobiocin and novobiocin.','Dimethylallyl diphosphate + 4-hydroxyphenylpyruvate = diphosphate +3-dimethylallyl-4-hydroxyphenylpyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.112','Adenylate dimethylallyltransferase (ADP/ATP-dependent)','Adenylate isopentenyltransferase','NULL','Involved in the biosynthesis of cytokinins in plants.The IPT4 isoform from the plant Arabidopsis thaliana is specific forADP and ATP.Other isoforms, such as IPT1 from Arabidopsis thaliana and the enzymefrom the common hop, Humulus lupulus, also have a lower activity withAMP (cf. EC 2.5.1.27).Formerly EC 2.5.1.n4.','(1) Dimethylallyl diphosphate + ADP = diphosphate +N6-(dimethylallyl)adenosine diphosphate.(2) Dimethylallyl diphosphate + ATP = diphosphate +N6-(dimethylallyl)adenosine triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.113','[CysO sulfur-carrier protein]-thiocarboxylate-dependent cysteinesynthase','CysO-thiocarboxylate-dependent cysteine synthase','Pyridoxal 5-phosphate.','The enzyme participates in an alternative pathway for L-cysteinebiosynthesis that involves a protein-bound thiocarboxylate as thesulfide donor.The enzyme from the bacterium Mycobacterium tuberculosis also hasvery low activity with O(3)-acetyl-L-serine (cf. EC 2.5.1.65).Formerly EC 2.5.1.n5.','O-phospho-L-serine + [CysO sulfur-carrier protein]-Gly-NH-CH(2)-C(O)SH =[CysO sulfur-carrier protein]-Gly-NH-CH(2)-C(O)-S-L-cysteine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.114','tRNA(Phe) (4-demethylwyosine(37)-C(7)) aminocarboxypropyltransferase','tRNA-yW synthesizing enzyme-2','NULL','The enzyme, which is found in all eukaryotes and in the majority ofEuryarchaeota (but not in the Crenarchaeota), is involved in thehypermodification of the guanine nucleoside at position 37 of tRNAleading to formation of assorted wye bases.This modification is essential for translational reading-framemaintenance.The eukaryotic enzyme is involved in biosynthesis of thetricyclic base wybutosine, which is found only in tRNA(Phe).','S-adenosyl-L-methionine + 4-demethylwyosine(37) in tRNA(Phe) = S-methyl-5-thioadenosine + 7-((3S)-3-amino-3-carboxypropyl)-4-demethylwyosine(37)in tRNA(Phe).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.115','Homogentisate phytyltransferase','HPT','Magnesium.','Involved in the biosynthesis of the vitamin E tocopherols.While the enzyme from the cyanobacterium Synechocystis PCC 6803 hasan appreciable activity with geranylgeranyl diphosphate(EC 2.5.1.116), the enzyme from the plant Arabidopsis thaliana hasonly a low activity with that substrate.Formerly EC 2.5.1.n8.','Phytyl diphosphate + homogentisate = diphosphate + 2-methyl-6-phytylbenzene-1,4-diol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.116','Homogentisate geranylgeranyltransferase','HGGT','Magnesium.','Involved in the biosynthesis of the vitamin E, tocotrienols.While the enzyme from the bacterium Synechocystis PCC 6803 has higheractivity with phytyl diphosphate (EC 2.5.1.115) the enzymes frombarley, rice and wheat have only a low activity with that substrate.','Geranylgeranyl diphosphate + homogentisate = diphosphate +6-geranylgeranyl-2-methylbenzene-1,4-diol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.117','Homogentisate solanesyltransferase','HST','Magnesium.','Part of the biosynthesis pathway of plastoquinol-9.The enzymes purified from the plant Arabidopsis thaliana and the algaChlamydomonas reinhardtii are also active in vitro with unsaturatedC(10)-C(20) prenyl diphosphates, producing main products that are notdecarboxylated.','All-trans-nonaprenyl diphosphate + homogentisate = diphosphate +2-methyl-6-all-trans-nonaprenylbenzene-1,4-diol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.118','Beta-(isoxazolin-5-on-2-yl)-L-alanine synthase','NULL','NULL','The enzyme from the plants Lathyrus odoratus (sweet pea) andL.sativus (grass pea) also forms 3-(5-oxoisoxazolin-4-yl)-L-alaninein vitro (cf. EC 2.5.1.119).However, only 3-(5-oxoisoxazolin-2-yl)-L-alanine is formed in vivo.3-(5-oxoisoxazolin-2-yl)-L-alanine is the biosynthetic precursor ofthe neurotoxin N(3)-oxalyl-L-2,3-diaminopropanoic acid, the cause oflathyrism.Closely related and possibly identical to EC 2.5.1.47 andEC 2.5.1.51.','O(3)-acetyl-L-serine + isoxazolin-5-one = 3-(5-oxoisoxazolin-2-yl)-L-alanine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.119','Beta-(isoxazolin-5-on-4-yl)-L-alanine synthase','NULL','NULL','3-(5-oxoisoxazolin-4-yl)-L-alanine is an antifungal antibioticproduced by the bacterium Streptomyces platensis.The enzymes from the plants Lathyrus odoratus (sweet pea), L.sativus(grass pea) and Citrullus vulgaris (watermelon) that catalyzeEC 2.5.1.118 also catalyze this reaction in vitro, but not in vivo.Closely related and possibly identical to EC 2.5.1.47 andEC 2.5.1.51.','O(3)-acetyl-L-serine + isoxazolin-5-one = 3-(5-oxoisoxazolin-4-yl)-L-alanine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.120','Aminodeoxyfutalosine synthase','AFL synthase','NULL','This enzyme is a member of the AdoMet radical (radical SAM) family.S-adenosyl-L-methionine acts as both a radical generator and as thesource of the transferred adenosyl group.The enzyme, found in several bacterial species, is part of thefutalosine pathway for menaquinone biosynthesis.','S-adenosyl-L-methionine + 3-((1-carboxyvinyl)oxy)benzoate + H(2)O =6-amino-6-deoxyfutalosine + L-methionine + bicarbonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.n2','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.n3','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.n4','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.n5','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.n6','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.n7','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.5.1.n8','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.1','Aspartate transaminase','Aspartate aminotransferase','Pyridoxal 5-phosphate.','Also acts on L-tyrosine, L-phenylalanine and L-tryptophan.This activity can be formed from EC 2.6.1.57 by controlledproteolysis.','L-aspartate + 2-oxoglutarate = oxaloacetate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.2','Alanine transaminase','Alanine aminotransferase','Pyridoxal 5-phosphate.','2-aminobutanoate acts slowly instead of alanine.','L-alanine + 2-oxoglutarate = pyruvate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.3','Cysteine transaminase','Cysteine aminotransferase','Pyridoxal 5-phosphate.','NULL','L-cysteine + 2-oxoglutarate = mercaptopyruvate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.4','Glycine transaminase','Glycine aminotransferase','Pyridoxal 5-phosphate.','NULL','Glycine + 2-oxoglutarate = glyoxylate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.5','Tyrosine transaminase','TyrAT','Pyridoxal 5-phosphate.','L-phenylalanine can act instead of L-tyrosine.The mitochondrial enzyme may be identical with EC 2.6.1.1.The three isoenzymic forms are interconverted by EC 3.4.22.32 andEC 3.4.22.33.The enzyme can also catalyze the final step in the methionine-salvagepathway of Klebsiella pneumoniae.','L-tyrosine + 2-oxoglutarate = 4-hydroxyphenylpyruvate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.6','Leucine transaminase','Leucine aminotransferase','Pyridoxal 5-phosphate.','Differs from EC 2.6.1.42 in that it does not act on L-valine orL-isoleucine, although it does act on L-methionine.','L-leucine + 2-oxoglutarate = 4-methyl-2-oxopentanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.7','Kynurenine--oxoglutarate transaminase','Kynurenine aminotransferase','Pyridoxal 5-phosphate.','Also acts on 3-hydroxykynurenine.The product 4-(2-aminophenyl)-2,4-dioxobutanoate is converted intokynurenate by a spontaneous reaction.','L-kynurenine + 2-oxoglutarate = 4-(2-aminophenyl)-2,4-dioxobutanoate +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.8','2,5-diaminovalerate transaminase','2,5-diaminovalerate aminotransferase','Pyridoxal 5-phosphate.','2,5-diaminoglutarate can act instead of diaminopentanoate.','2,5-diaminopentanoate + 2-oxoglutarate = 5-amino-2-oxopentanoate +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.9','Histidinol-phosphate transaminase','Histidinol-phosphate aminotransferase','Pyridoxal 5-phosphate.','NULL','L-histidinol phosphate + 2-oxoglutarate = 3-(imidazol-4-yl)-2-oxopropylphosphate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.10','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.11','Acetylornithine transaminase','Acetylornithine 5-aminotransferase','Pyridoxal 5-phosphate.','Also acts on L-ornithine and N(2)-succinyl-L-ornithine.','N(2)-acetyl-L-ornithine + 2-oxoglutarate = N-acetyl-L-glutamate5-semialdehyde + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.12','Alanine--oxo-acid transaminase','Alanine--oxo-acid aminotransferase','Pyridoxal 5-phosphate.','NULL','L-alanine + a 2-oxo acid = pyruvate + an L-amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.13','Ornithine aminotransferase','L-ornithine 5-aminotransferase','Pyridoxal 5-phosphate.','NULL','L-ornithine + a 2-oxo acid = L-glutamate 5-semialdehyde + an L-aminoacid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.14','Asparagine--oxo-acid transaminase','Asparagine--oxo-acid aminotransferase','Pyridoxal 5-phosphate.','NULL','L-asparagine + a 2-oxo acid = 2-oxosuccinamate + an amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.15','Glutamine--pyruvate transaminase','Glutaminase II','Pyridoxal 5-phosphate.','L-methionine can act as donor.Glyoxylate can act as acceptor.','L-glutamine + pyruvate = 2-oxoglutaramate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.16','Glutamine--fructose-6-phosphate transaminase (isomerizing)','D-fructose-6-phosphate amidotransferase','NULL','Although the overall reaction is that of a transferase, the mechanisminvolves the formation of ketimine between fructose 6-phosphate and a6-amino group from a lysine residue at the active site, which issubsequently displaced by ammonia (transamidination).Formerly EC 5.3.1.19.','L-glutamine + D-fructose 6-phosphate = L-glutamate + D-glucosamine6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.17','Succinyldiaminopimelate transaminase','Succinyldiaminopimelate aminotransferase','Pyridoxal 5-phosphate.','NULL','N-succinyl-L-2,6-diaminoheptanedioate + 2-oxoglutarate = N-succinyl-2-L-amino-6-oxoheptanedioate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.18','Beta-alanine--pyruvate transaminase','Beta-alanine--pyruvate aminotransferase','Pyridoxal 5-phosphate.','NULL','L-alanine + 3-oxopropanoate = pyruvate + beta-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.19','4-aminobutyrate--2-oxoglutarate transaminase','4-aminobutyrate aminotransferase','Pyridoxal 5-phosphate.','Some preparations also act on beta-alanine, 5-aminopentanoate and(R,S)-3-amino-2-methylpropanoate.','4-aminobutanoate + 2-oxoglutarate = succinate semialdehyde + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.20','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.21','D-amino-acid transaminase','D-alanine aminotransferase','Pyridoxal 5-phosphate.','The enzyme from thermophilic Bacillus species acts on many D-aminoacids with D-alanine and D-2-aminobutyrate as the best amino donors.It can similarly use any of several 2-oxo acids as amino acceptor,with 2-oxoglutarate and 2-oxobutyrate among the best.The enzyme from some other sources has a broader specificity.Formerly EC 2.6.1.10.','D-alanine + 2-oxoglutarate = pyruvate + D-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.22','(S)-3-amino-2-methylpropionate transaminase','(S)-3-amino-2-methylpropionate aminotransferase','NULL','Also acts on beta-alanine and other omega-amino acids having carbonchains between 2 and 5.The two enantiomers of the 2-methyl-3-oxopropanoate formed by theenzyme interconvert by enolization, so that this enzyme, togetherwith EC 2.6.1.40, provide a route for interconversion of theenantiomers of 3-amino-2-methylpropanoate.','(S)-3-amino-2-methylpropanoate + 2-oxoglutarate = 2-methyl-3-oxopropanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.23','4-hydroxyglutamate transaminase','4-hydroxyglutamate aminotransferase','NULL','Oxaloacetate can replace 2-oxoglutarate.May be identical with EC 2.6.1.1.','4-hydroxy-L-glutamate + 2-oxoglutarate = 4-hydroxy-2-oxoglutarate +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.24','Diiodotyrosine transaminase','Diiodotyrosine aminotransferase','Pyridoxal 5-phosphate.','Also acts on 3,5-dichloro-, 3,5-dibromo- and 3-iodo-L-tyrosine,thyroxine and triiodothyronine.Formerly EC 2.6.1.25.','3,5-diiodo-L-tyrosine + 2-oxoglutarate = 4-hydroxy-3,5-diiodophenylpyruvate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.25','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.26','Thyroid-hormone transaminase','3,5-dinitrotyrosine aminotransferase','Pyridoxal 5-phosphate.','Acts on monoiodotyrosine, diiodotyrosine, triiodothyronine, thyroxineand dinitrotyrosine (unlike EC 2.6.1.24, which does not act ondinitrotyrosine).Pyruvate or oxaloacetate can act as acceptor.','L-3,5,3-triiodothyronine + 2-oxoglutarate = 3-(4-(4-hydroxy-3-iodophenoxy)-3,5-diiodophenyl)-2-oxopropanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.27','Tryptophan transaminase','Tryptophan aminotransferase','Pyridoxal 5-phosphate.','Also acts on 5-hydroxytryptophan and, to a lesser extent, on thephenyl amino acids.','L-tryptophan + 2-oxoglutarate = (indol-3-yl)pyruvate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.28','Tryptophan--phenylpyruvate transaminase','Tryptophan--phenylpyruvate aminotransferase','NULL','Valine, leucine and isoleucine can replace tryptophan as amino donor.','L-tryptophan + phenylpyruvate = (indol-3-yl)pyruvate + L-phenylalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.29','Diamine transaminase','Diamine aminotransferase','NULL','NULL','An alpha,omega-diamine + 2-oxoglutarate = an omega-aminoaldehyde +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.30','Pyridoxamine--pyruvate transaminase','Pyridoxamine--pyruvate aminotransferase','Pyridoxal 5-phosphate.','NULL','Pyridoxamine + pyruvate = pyridoxal + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.31','Pyridoxamine--oxaloacetate transaminase','Pyridoxamine--oxaloacetate aminotransferase','NULL','NULL','Pyridoxamine + oxaloacetate = pyridoxal + L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.32','Valine--3-methyl-2-oxovalerate transaminase','Valine--3-methyl-2-oxovalerate aminotransferase','NULL','NULL','L-valine + (S)-3-methyl-2-oxopentanoate = 3-methyl-2-oxobutanoate +L-isoleucine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.33','dTDP-4-amino-4,6-dideoxy-D-glucose transaminase','dTDP-4-amino-4,6-dideoxy-D-glucose aminotransferase','Pyridoxal 5-phosphate.','NULL','dTDP-4-amino-4,6-dideoxy-alpha-D-glucose + 2-oxoglutarate = dTDP-4-dehydro-6-deoxy-alpha-D-glucose + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.34','UDP-N-acetylbacillosamine transaminase','UDP-4-amino-2-acetamido-2,4,6-trideoxyglucose aminotransferase','Pyridoxal 5-phosphate.','The enzyme is involved in biosynthesis of UDP-N,N-diacetylbacillosamine, an intermediate in protein glycosylationpathways in several bacterial species, including N-linkedglycosylation of certain L-asparagine residues in Campylobacterspecies and O-linked glycosylation of certain L-serine residues inNeisseria species.Formerly EC 2.6.1.91.','UDP-N-acetylbacillosamine + 2-oxoglutarate = UDP-2-acetamido-2,6-dideoxy-alpha-D-xylo-hex-4-ulose + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.35','Glycine--oxaloacetate transaminase','Glycine--oxaloacetate aminotransferase','Pyridoxal 5-phosphate.','NULL','Glycine + oxaloacetate = glyoxylate + L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.36','L-lysine 6-transaminase','L-lysine aminotransferase','Pyridoxal 5-phosphate.','The product, (S)-2-amino-6-oxohexanoate, is converted into theintramolecularly dehydrated form, 1-piperideine 6-carboxylate.','L-lysine + 2-oxoglutarate = (S)-2-amino-6-oxohexanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.37','2-aminoethylphosphonate--pyruvate transaminase','(2-aminoethyl)phosphonate transaminase','Pyridoxal 5-phosphate.','2-aminoethylarsonate can replace 2-aminoethylphosphonate as asubstrate.','(2-aminoethyl)phosphonate + pyruvate = 2-phosphonoacetaldehyde +L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.38','Histidine transaminase','Histidine aminotransferase','NULL','NULL','L-histidine + 2-oxoglutarate = (imidazol-5-yl)pyruvate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.39','2-aminoadipate transaminase','2-aminoadipate aminotransferase','Pyridoxal 5-phosphate.','NULL','L-2-aminoadipate + 2-oxoglutarate = 2-oxoadipate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.40','(R)-3-amino-2-methylpropionate--pyruvate transaminase','(R)-3-amino-2-methylpropionate transaminase','NULL','The two enantiomers of the 2-methyl-3-oxopropanoate formed by theenzyme interconvert by enolization, so that this enzyme, togetherwith EC 2.6.1.22, provide a route for interconversion of theenantiomers of 3-amino-2-methylpropanoate.Formerly EC 2.6.1.61.','(R)-3-amino-2-methylpropanoate + pyruvate = 2-methyl-3-oxopropanoate +L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.41','D-methionine--pyruvate transaminase','D-methionine aminotransferase','NULL','Oxaloacetate can replace pyruvate.','D-methionine + pyruvate = 4-methylthio-2-oxobutanoate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.42','Branched-chain-amino-acid transaminase','Branched-chain amino acid aminotransferase','Pyridoxal 5-phosphate.','Also acts on L-isoleucine and L-valine.Different from EC 2.6.1.66.','(1) L-leucine + 2-oxoglutarate = 4-methyl-2-oxopentanoate + L-glutamate.(2) L-isoleucine + 2-oxoglutarate = (S)-3-methyl-2-oxopentanoate +L-glutamate.(3) L-valine + 2-oxoglutarate = 3-methyl-2-oxobutanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.43','Aminolevulinate transaminase','Aminolevulinate aminotransferase','Pyridoxal 5-phosphate.','NULL','5-aminolevulinate + pyruvate = 4,5-dioxopentanoate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.44','Alanine--glyoxylate transaminase','AGT','Pyridoxal 5-phosphate.','With one component of the animal enzyme, 2-oxobutanoate can replaceglyoxylate.A second component also catalyzes the reaction of EC 2.6.1.51.','L-alanine + glyoxylate = pyruvate + glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.45','Serine--glyoxylate transaminase','Serine--glyoxylate aminotransferase','Pyridoxal 5-phosphate.','NULL','L-serine + glyoxylate = 3-hydroxypyruvate + glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.46','Diaminobutyrate--pyruvate transaminase','Diaminobutyrate--pyruvate aminotransferase','Pyridoxal 5-phosphate.','NULL','L-2,4-diaminobutanoate + pyruvate = L-aspartate 4-semialdehyde +L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.47','Alanine--oxomalonate transaminase','Alanine--oxomalonate aminotransferase','Pyridoxal 5-phosphate.','NULL','L-alanine + oxomalonate = pyruvate + aminomalonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.48','5-aminovalerate transaminase','5-aminovalerate aminotransferase','Pyridoxal 5-phosphate.','NULL','5-aminopentanoate + 2-oxoglutarate = 5-oxopentanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.49','Dihydroxyphenylalanine transaminase','Dihydroxyphenylalanine aminotransferase','Pyridoxal 5-phosphate.','NULL','L-dopa + 2-oxoglutarate = 3,4-dihydroxyphenylpyruvate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.50','Glutamine--scyllo-inositol transaminase','Glutamine--scyllo-inosose aminotransferase','Pyridoxal 5-phosphate.','NULL','L-glutamine + 2,4,6/3,5-pentahydroxycyclohexanone = 2-oxoglutaramate +1-amino-1-deoxy-scyllo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.51','Serine--pyruvate transaminase','Serine--pyruvate aminotransferase','Pyridoxal 5-phosphate.','The liver enzyme may be identical with EC 2.6.1.44.','L-serine + pyruvate = 3-hydroxypyruvate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.52','Phosphoserine transaminase','3-phosphoserine aminotransferase','Pyridoxal 5-phosphate.','Catalyzes the second step in the phosphorylated pathway of serinebiosynthesis in Escherichia coli.Also catalyzes the third step in the biosynthesis of the coenzymepyridoxal 5-phosphate in E.coli (using reaction 2 above).In E.coli, pyridoxal 5-phosphate is synthesized de novo by a pathwaythat involves EC 1.2.1.72, EC 1.1.1.290, EC 2.6.1.52, EC 1.1.1.262,EC 2.6.99.2 and EC 1.4.3.5 (with pyridoxine 5-phosphate assubstrate).Pyridoxal phosphate is the cofactor for both activities and thereforeseems to be involved in its own biosynthesis.Non-phosphorylated forms of serine and threonine are not substrates.','(1) O-phospho-L-serine + 2-oxoglutarate = 3-phosphonooxypyruvate +L-glutamate.(2) 4-phosphonooxy-L-threonine + 2-oxoglutarate = (3R)-3-hydroxy-2-oxo-4-phosphonooxybutanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.53','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.54','Pyridoxamine-phosphate transaminase','Pyridoxamine-phosphate aminotransferase','NULL','Also acts, more slowly, on pyridoxamine.','Pyridoxamine 5-phosphate + 2-oxoglutarate = pyridoxal 5-phosphate +D-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.55','Taurine--2-oxoglutarate transaminase','Taurine aminotransferase','Pyridoxal 5-phosphate.','Also acts on D,L-3-amino-isobutanoate, beta-alanine and3-aminopropanesulfonate.Involved in the microbial utilization of beta-alanine.','Taurine + 2-oxoglutarate = sulfoacetaldehyde + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.56','1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol transaminase','1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol aminotransferase','NULL','L-glutamate and L-glutamine can also act as amino donors.','1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol + pyruvate = 1D-1-guanidino-1-deoxy-3-dehydro-scyllo-inositol + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.57','Aromatic-amino-acid transaminase','NULL','Pyridoxal 5-phosphate.','L-methionine can also act as donor, more slowly.Oxaloacetate can act as acceptor.Controlled proteolysis converts the enzyme to EC 2.6.1.1.','An aromatic amino acid + 2-oxoglutarate = an aromatic oxo acid +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.58','Phenylalanine(histidine) transaminase','Phenylalanine(histidine) aminotransferase','NULL','L-histidine and L-tyrosine can act instead of L-phenylalanine; in thereverse reaction, L-methionine, L-serine and L-glutamine can replaceL-alanine.','L-phenylalanine + pyruvate = phenylpyruvate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.59','dTDP-4-amino-4,6-dideoxygalactose transaminase','dTDP-4-amino-4,6-dideoxygalactose aminotransferase','Pyridoxal 5-phosphate.','NULL','dTDP-4-amino-4,6-dideoxy-alpha-D-galactose + 2-oxoglutarate = dTDP-4-dehydro-6-deoxy-alpha-D-galactose + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.60','Aromatic-amino-acid--glyoxylate transaminase','Aromatic-amino-acid--glyoxylate aminotransferase','NULL','Phenylalanine, kynurenine, tyrosine and histidine can act as aminodonors.Glyoxylate, pyruvate and hydroxypyruvate can act as amino acceptors.','An aromatic amino acid + glyoxylate = an aromatic oxo acid + glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.61','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.62','Adenosylmethionine--8-amino-7-oxononanoate transaminase','7,8-diamino-pelargonic acid aminotransferase','Pyridoxal 5-phosphate.','S-adenosylhomocysteine can also act as donor.','S-adenosyl-L-methionine + 8-amino-7-oxononanoate = S-adenosyl-4-methylthio-2-oxobutanoate + 7,8-diaminononanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.63','Kynurenine--glyoxylate transaminase','Kynurenine--glyoxylate aminotransferase','NULL','Acts, more slowly, on L-phenylalanine, L-histidine and L-tyrosine.','L-kynurenine + glyoxylate = 4-(2-aminophenyl)-2,4-dioxobutanoate +glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.64','Glutamine--phenylpyruvate transaminase','Glutamine transaminase K','Pyridoxal 5-phosphate.','L-methionine, L-histidine and L-tyrosine can act as donors.Has little activity on pyruvate and glyoxylate (cf. EC 2.6.1.15).','L-glutamine + phenylpyruvate = 2-oxoglutaramate + L-phenylalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.65','N(6)-acetyl-beta-lysine transaminase','N(6)-acetyl-beta-lysine aminotransferase','Pyridoxal 5-phosphate.','NULL','6-acetamido-3-aminohexanoate + 2-oxoglutarate = 6-acetamido-3-oxohexanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.66','Valine--pyruvate transaminase','Alanine--valine transaminase','Pyridoxal 5-phosphate.','Different from EC 2.6.1.42.','L-valine + pyruvate = 3-methyl-2-oxobutanoate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.67','2-aminohexanoate transaminase','2-aminohexanoate aminotransferase','Pyridoxal 5-phosphate.','Also acts on L-leucine and, more slowly, on L-isoleucine, L-2-aminopentanoate and L-aspartate.','L-2-aminohexanoate + 2-oxoglutarate = 2-oxohexanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.68','Ornithine(lysine) transaminase','Ornithine(lysine) aminotransferase','NULL','The enzyme from Trichomonas vaginalis also acts on L-lysine,producing 2,3,4,5-tetrahydropyridine-2-carboxylate.','L-ornithine + 2-oxoglutarate = 3,4-dihydro-2H-pyrrole-2-carboxylate +L-glutamate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.69','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.70','Aspartate--phenylpyruvate transaminase','Aspartate--phenylpyruvate aminotransferase','NULL','The enzyme from Pseudomonas putida also acts on4-hydroxyphenylpyruvate and, more slowly, on L-glutamate andL-histidine.','L-aspartate + phenylpyruvate = oxaloacetate + L-phenylalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.71','Lysine--pyruvate 6-transaminase','Lysine--pyruvate 6-aminotransferase','NULL','NULL','L-lysine + pyruvate = (S)-2-amino-6-oxohexanoate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.72','D-4-hydroxyphenylglycine transaminase','D-4-hydroxyphenylglycine aminotransferase','Pyridoxal 5-phosphate.','NULL','D-4-hydroxyphenylglycine + 2-oxoglutarate = 4-hydroxyphenylglyoxylate +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.73','Methionine--glyoxylate transaminase','MGAT','NULL','L-glutamate can also act as donor.','L-methionine + glyoxylate = 4-methylthio-2-oxobutanoate + glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.74','Cephalosporin-C transaminase','Cephalosporin C aminotransferase','NULL','A number of D-amino acids, including D-alanine, D-aspartate andD-methionine can also act as amino-group donors.Although this enzyme acts on several free D-amino acids, it differsfrom EC 2.6.1.21 in that it can use cephalosporin C as an aminodonor.','(7R)-7-(5-carboxy-5-oxopentanoyl)aminocephalosporinate + D-glutamate =cephalosporin C + 2-oxoglutarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.75','Cysteine-conjugate transaminase','NULL','NULL','A number of cysteine conjugates can also act.','S-(4-bromophenyl)-L-cysteine + 2-oxoglutarate = S-(4-bromophenyl)-mercaptopyruvate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.76','Diaminobutyrate--2-oxoglutarate transaminase','2,4-diaminobutyrate 4-aminotransferase','Potassium; Pyridoxal 5-phosphate.','Differs from EC 2.6.1.46, which has pyruvate as the amino-groupacceptor.This is the first enzyme in the ectoine biosynthesis pathway,the other enzymes involved being EC 2.3.1.178 and EC 4.2.1.108.','L-2,4-diaminobutanoate + 2-oxoglutarate = L-aspartate 4-semialdehyde +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.77','Taurine--pyruvate aminotransferase','NULL','Pyridoxal 5-phosphate.','The enzyme from Bilophila wadsworthia catalyzes a reversible reactionthat starts an anaerobic taurine degradation pathway.beta-alanine is also a significant amino group donor.The enzyme from the bacterium Pseudomonas denitrificans PD1222 canalso use hypotaurine, producing 2-sulfinoacetaldehyde, whichspontaneously hydrolyzes to sulfite and acetaldehyde.Unlike EC 2.6.1.55, 2-oxoglutarate cannot serve as an acceptor forthe amino group.','Taurine + pyruvate = L-alanine + 2-sulfoacetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.78','Aspartate--prephenate aminotransferase','L-aspartate:prephenate aminotransferase','Pyridoxal 5-phosphate.','Glutamate can also act as the amino donor, but more slowly (cf.EC 2.6.1.79).','L-arogenate + oxaloacetate = prephenate + L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.79','Glutamate--prephenate aminotransferase','L-glutamate:prephenate aminotransferase','Pyridoxal 5-phosphate.','Aspartate can also act as the amino donor, but more slowly (cf.EC 2.6.1.78).The enzyme from higher plants shows a marked preference forprephenate as substrate compared to pyruvate, phenylpyruvate or4-hydroxyphenylpyruvate.','L-arogenate + 2-oxoglutarate = prephenate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.80','Nicotianamine aminotransferase','NAAT','Pyridoxal 5-phosphate.','This enzyme is produced by grasses.They secrete both the nicotianamine and the transaminated productinto the soil around them.Both compounds chelate Fe(2+) and Fe(3+); these chelators, calledmugineic acid family phytosiderophores, are taken up by the grass,which is thereby supplied with iron.','Nicotianamine + 2-oxoglutarate = 3-deamino-3-oxonicotianamine +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.81','Succinylornithine transaminase','N(2)-succinylornithine 5-aminotransferase','Pyridoxal 5-phosphate.','Also acts on N(2)-acetyl-L-ornithine and L-ornithine, but moreslowly.In Pseudomonas aeruginosa, the arginine-inducible succinylornithinetransaminase, EC 2.6.1.11 and EC 2.6.1.13 activities are catalyzed bythe same enzyme, but this is not the case in all species.This is the third enzyme in the arginine succinyltransferase (AST)pathway for the catabolism of arginine.This pathway converts the carbon skeleton of arginine into glutamate,with the concomitant production of ammonia and conversion ofsuccinyl-CoA into succinate and CoA.The five enzymes involved in this pathway are EC 2.3.1.109,EC 3.5.3.23, EC 2.6.1.81, EC 1.2.1.71 and EC 3.5.1.96.','N(2)-succinyl-L-ornithine + 2-oxoglutarate = N-succinyl-L-glutamate5-semialdehyde + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.82','Putrescine aminotransferase','PAT','Pyridoxal 5-phosphate.','The enzymatic part of the reaction produces 4-aminobutanal thatspontaneously cyclizes to form 1-pyrroline, which is a substrate forEC 1.2.1.19.Cadaverine and spermidine can also act as substrates.Forms part of the arginine catabolism pathway.','Putrescine + 2-oxoglutarate = L-glutamate + 1-pyrroline + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.83','LL-diaminopimelate aminotransferase','LL-DAP aminotransferase','Pyridoxal 5-phosphate.','In vivo, the reaction occurs in the opposite direction to that shownabove.This is one of the final steps in the lysine biosynthesis pathway ofplants (ranging from mosses to flowering plants).Meso-diaminoheptanedioate, an isomer of LL-2,6-diaminoheptanedioate,and the structurally related compounds lysine and ornithine are notsubstrates.2-oxoglutarate cannot be replaced by oxaloacetate or pyruvate.It is not yet known if the substrate of the biosynthetic reaction isthe cyclic or acyclic form of tetrahydropyridine-2,6-dicarboxylate.','LL-2,6-diaminoheptanedioate + 2-oxoglutarate = (S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate + L-glutamate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.84','Arginine--pyruvate transaminase','Arginine:pyruvate transaminase','Pyridoxal 5-phosphate.','While L-arginine is the best substrate, the enzyme exhibits broadsubstrate specificity, with L-lysine, L-methionine, L-leucine,L-ornithine and L-glutamine also able to act as substrates, but moreslowly.Pyruvate cannot be replaced by 2-oxoglutarate as amino-groupacceptor.This is the first catalytic enzyme of the arginine transaminasepathway for L-arginine utilization in Pseudomonas aeruginosa.This pathway is only used when the major route of argininecatabolism, i.e. the arginine succinyltransferase pathway,is blocked.','L-arginine + pyruvate = 5-guanidino-2-oxopentanoate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.85','Aminodeoxychorismate synthase','4-amino-4-deoxychorismate synthase','Magnesium.','Composed of two subunits, PabA and PabB.In the absence of PabA and glutamine, PabB converts ammonia andchorismate into 4-amino-4-deoxychorismate.PabA converts glutamine into glutamate only in the presence ofstoichiometric amounts of PabB.Coupled with EC 4.1.3.38, to form 4-aminobenzoate.Formerly EC 6.3.5.8.','Chorismate + L-glutamine = 4-amino-4-deoxychorismate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.86','2-amino-4-deoxychorismate synthase','2-amino-2-deoxyisochorismate synthase','Magnesium.','The reaction occurs in the reverse direction to that shown above.In contrast to most anthranilate-synthase I (ASI) homologs, thisenzyme is not inhibited by tryptophan.In Streptomyces globisporus, the sequential action of this enzyme andEC 1.3.99.24, 2-amino-4-deoxychorismate dehydrogenase, leads to theformation of the benzoxazolinate moiety of the enediyne antitumorantibiotic C-1027.In certain Pseudomonads the enzyme participates in the biosynthesisof phenazine, a precursor for several compounds with antibioticactivity.','(2S)-2-amino-4-deoxychorismate + L-glutamate = chorismate + L-glutamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.87','UDP-4-amino-4-deoxy-L-arabinose aminotransferase','UDP-(beta-L-threo-pentapyranosyl-4-ulose diphosphate) aminotransferase','Pyridoxal 5-phosphate.','Formerly EC 2.6.1.n1.','UDP-4-amino-4-deoxy-beta-L-arabinose + 2-oxoglutarate = UDP-beta-L-threo-pentapyranos-4-ulose + glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.88','Methionine transaminase','Methionine-oxo-acid transaminase','NULL','The enzyme is most active with L-methionine.It participates in the L-methionine salvage pathway from S-methyl-5-thioadenosine, a by-product of polyamine biosynthesis.The enzyme from the bacterium Klebsiella pneumoniae can use severaldifferent amino acids as amino donor, with aromatic amino acids beingthe most effective.The enzyme from the plant Arabidopsis thaliana is also a part of thechain elongation pathway in the biosynthesis of methionine-derivedglucosinolates.','L-methionine + a 2-oxo acid = 2-oxo-4-methylthiobutanoate + an L-aminoacid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.89','dTDP-3-amino-3,6-dideoxy-alpha-D-glucopyranose transaminase','dTDP-3-dehydro-6-deoxy-D-glucose 3-aminotransferase','Pyridoxal 5-phosphate.','The reaction occurs in the reverse direction.The enzyme is involved in biosynthesis of D-mycaminose.','dTDP-3-amino-3,6-dideoxy-alpha-D-glucopyranose + 2-oxoglutarate = dTDP-3-dehydro-6-deoxy-alpha-D-glucopyranose + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.90','dTDP-3-amino-3,6-dideoxy-alpha-D-galactopyranose transaminase','dTDP-6-deoxy-D-xylohex-3-uloseaminase','Pyridoxal 5-phosphate.','The enzyme is involved in the biosynthesis of dTDP-3-acetamido-3,6-dideoxy-alpha-D-galactose.The reaction occurs in the reverse direction.','dTDP-3-amino-3,6-dideoxy-alpha-D-galactopyranose + 2-oxoglutarate = dTDP-3-dehydro-6-deoxy-alpha-D-galactopyranose + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.91','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.92','UDP-4-amino-4,6-dideoxy-N-acetyl-beta-L-altrosamine transaminase','NULL','Pyridoxal 5-phosphate.','The enzyme is involved in biosynthesis of pseudaminic acid.','UDP-4-amino-4,6-dideoxy-N-acetyl-beta-L-altrosamine + 2-oxoglutarate =UDP-2-acetamido-2,6-dideoxy-beta-L-arabino-hex-4-ulose + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.93','Neamine transaminase','Glutamate--6-dehydroparomamine aminotransferase','NULL','The reaction occurs in vivo in the opposite direction.Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including kanamycin B, butirosin, neomycinand ribostamycin.Works in combination with EC 1.1.3.43 to replace the 6-hydroxy groupof paromamine with an amino group.The enzyme from the bacterium Streptomyces kanamyceticus can alsocatalyze EC 2.6.1.94, which leads to production of kanamycin A.The enzyme from the bacterium Streptomyces fradiae can also catalyzeEC 2.6.1.95, leading to production of neomycin C.','Neamine + 2-oxoglutarate = 6-dehydroparomamine + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.94','2-deamino-2-hydroxyneamine transaminase','NULL','NULL','The reaction occurs in vivo in the opposite direction.Involved in the biosynthetic pathway of kanamycin A and kanamycin D.The enzyme, characterized from the bacterium Streptomyceskanamyceticus, can also catalyze EC 2.6.1.93.','2-deamino-2-hydroxyneamine + 2-oxoglutarate = 2-deamino-2-hydroxy-6-dehydroparomamine + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.95','Neomycin C transaminase','NULL','NULL','The reaction occurs in vivo in the opposite direction.Involved in the biosynthetic pathway of aminoglycoside antibiotics ofthe neomycin family.Works in combination with EC 1.1.3.44 to replace the 6-hydroxygroup of 6-deamino-6-hydroxyneomycin C with an amino group.The enzyme, characterized from the bacterium Streptomyces fradiae,can also catalyze EC 2.6.1.93.','Neomycin C + 2-oxoglutarate = 6-deamino-6-oxoneomycin C +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.96','4-aminobutyrate--pyruvate transaminase','4-aminobutyrate aminotransferase','Pyridoxal 5-phosphate.','The enzyme is found in plants that do not have the 2-oxoglutaratedependent enzyme (cf. EC 2.6.1.19).The reaction with pyruvate is reversible while the reaction withglyoxylate only takes place in the forward direction.','(1) 4-aminobutanoate + pyruvate = succinate semialdehyde + L-alanine.(2) 4-aminobutanoate + glyoxylate = succinate semialdehyde + glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.97','Archaeosine synthase','NULL','NULL','In Euryarchaeota the reaction is catalyzed by ArcS.In Crenarchaeota, which do not have an ArcS homolog, the reaction iscatalyzed either by a homolog of EC 6.3.4.20 that includes aglutaminase domain (cf. EC 3.5.1.2), or by a homolog of EC 1.7.1.13.The enzyme from the Euryarchaeon Methanocaldococcus jannaschii canalso use arginine and ammonium as amino donors.','L-glutamine + 7-cyano-7-carbaguanine(15) in tRNA + H(2)O = L-glutamate +archaeine(15) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.98','UDP-2-acetamido-2-deoxy-ribo-hexuluronate aminotransferase','NULL','Pyridoxal 5-phosphate.','This enzyme participates in the biosynthetic pathway for UDP-alpha-D-ManNAc3NAcA (UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-mannuronicacid), an important precursor of B-band lipopolysaccharide.The enzymes from Pseudomonas aeruginosa serotype O5 and Thermusthermophilus form a complex with the previous enzyme in the pathway,EC 1.1.1.335.','UDP-2-acetamido-3-amino-2,3-dideoxy-alpha-D-glucuronate + 2-oxoglutarate= UDP-2-acetamido-2-deoxy-D-ribo-hex-3-uluronate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.99','L-tryptophan--pyruvate aminotransferase','NULL','NULL','This plant enzyme, along with EC 1.14.13.168, is responsible for thebiosynthesis of the plant hormone indole-3-acetate from L-tryptophan.','L-tryptophan + pyruvate = indole-3-pyruvate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.100','L-glutamine:2-deoxy-scyllo-inosose aminotransferase','NULL','NULL','Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including kanamycin, butirosin, neomycinand ribostamycin.Also catalyzes EC 2.6.1.101.','L-glutamine + 2-deoxy-scyllo-inosose = 2-oxoglutaramate + 2-deoxy-scyllo-inosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.101','L-glutamine:3-amino-2,3-dideoxy-scyllo-inosose aminotransferase','NULL','NULL','Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including kanamycin, butirosin, neomycinand ribostamycin.Also catalyzes EC 2.6.1.100.','L-glutamine + 3-amino-2,3-dideoxy-scyllo-inosose = 2-oxoglutaramate +2-deoxystreptamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.102','GDP-perosamine synthase','GDP-4-keto-6-deoxy-D-mannose-4-aminotransferase','Pyridoxal 5-phosphate.','D-perosamine is one of several dideoxy sugars found in the O-specificpolysaccharide of the lipopolysaccharide component of the outermembrane of Gram-negative bacteria.The enzyme catalyzes the final step in GDP-perosamine synthesis.','GDP-4-amino-4,6-dideoxy-alpha-D-mannose + 2-oxoglutarate = GDP-4-dehydro-6-deoxy-alpha-D-mannose + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.103','(S)-3,5-dihydroxyphenylglycine transaminase','NULL','Pyridoxal 5-phosphate.','The enzyme from the bacterium Amycolatopsis orientalis catalyzes thereaction in the reverse direction as part of the biosynthesis of the(S)-3,5-dihydroxyphenylglycine constituent of the glycopeptideantibiotic chloroeremomycin.','(S)-3,5-dihydroxyphenylglycine + 2-oxoglutarate = 2-(3,5-dihydroxyphenyl)-2-oxoacetate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.104','3-dehydro-glucose-6-phosphate--glutamate transaminase','3-oxo-glucose-6-phosphate:glutamate aminotransferase','Pyridoxal 5-phosphate.','The enzyme, found in the bacterium Bacillus subtilis, is involved ina kanosamine biosynthesis pathway.','Kanosamine 6-phosphate + 2-oxoglutarate = 3-dehydro-D-glucose 6-phosphate+ L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.105','Lysine--8-amino-7-oxononanoate transaminase','DAPA aminotransferase','Pyridoxal 5-phosphate.','Participates in the pathway for biotin biosynthesis.The enzyme from the bacterium Bacillus subtilis cannot useS-adenosyl-L-methionine as amino donor and catalyzes an alternativereaction for the conversion of 8-amino-7-oxononanoate to 7,8-diaminononanoate (cf. EC 2.6.1.62).Formerly EC 2.6.1.n2.','L-lysine + 8-amino-7-oxononanoate = (S)-2-amino-6-oxohexanoate + 7,8-diaminononanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.106','dTDP-3-amino-3,4,6-trideoxy-alpha-D-glucose transaminase','NULL','Pyridoxal 5-phosphate.','The enzyme is involved in the biosynthesis of dTDP-alpha-D-desosamine, a sugar found in several bacterial macrolide antibioticsincluding erythromycin, megalomicin A, mycinamicin II,and oleandomycin.The reaction occurs in the reverse direction.','dTDP-3-amino-3,4,6-trideoxy-alpha-D-glucose + 2-oxoglutarate = dTDP-3-dehydro-4,6-deoxy-alpha-D-glucose + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.107','Beta-methylphenylalanine transaminase','NULL','Pyridoxal 5-phosphate.','Isolated from the bacterium Streptomyces hygroscopicus NRRL3085.It is involved in the biosynthesis of the glycopeptide antibioticmannopeptimycin.','(2S,3S)-3-methylphenylalanine + 2-oxoglutarate = (3S)-2-oxo-3-phenylbutanoate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.1.n2','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.2.1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.3.1','Oximinotransferase','Oximase','NULL','Acetaldehyde can act instead of acetone; D-glucose oxime can actinstead of pyruvate oxime.','Pyruvate oxime + acetone = pyruvate + acetone oxime.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.99.1','dATP(dGTP)--DNA purinetransferase','NULL','NULL','The purine residue is transferred on to the apurinic site forming anormal glycosylic bond.dATP reacts at sites of the double-stranded depurinated DNA that lackadenine, and dGTP at sites that lack guanine.','(1) dATP + depurinated DNA = deoxyribose triphosphate + DNA.(2) dGTP + depurinated DNA = deoxyribose triphosphate + DNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.99.2','Pyridoxine 5-phosphate synthase','PNP synthase','NULL','In Escherichia coli, the coenzyme pyridoxal 5-phosphate issynthesized de novo by a pathway that involves EC 1.2.1.72,EC 1.1.1.290, EC 2.6.1.52, EC 1.1.1.262, EC 2.6.99.2 and EC 1.4.3.5.1-deoxy-D-xylulose cannot replace 1-deoxy-D-xylulose 5-phosphate as asubstrate.','1-deoxy-D-xylulose 5-phosphate + 3-amino-2-oxopropyl phosphate =pyridoxine 5-phosphate + phosphate + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.99.3','O-ureido-L-serine synthase','NULL','NULL','The enzyme participates in the biosynthetic pathway of D-cycloserine,an antibiotic substance produced by several Streptomyces species.Also catalyzes EC 2.5.1.47.','O-acetyl-L-serine + hydroxyurea = O-ureido-L-serine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.6.99.4','N(6)-L-threonylcarbamoyladenine synthase','T6A synthase','NULL','The enzyme is involved in the synthesis of N(6)-threonylcarbamoyladenosine(37) in tRNAs, which is found in tRNAs withthe anticodon NNU, i.e. tRNA(Ile), tRNA(Thr), tRNA(Asn), tRNA(Lys),tRNA(Ser) and tRNA(Arg).','L-threonylcarbamoyladenylate + adenine(37) in tRNA = AMP + N(6)-L-threonylcarbamoyladenine(37) in tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.1','Hexokinase','Hexokinase type I','NULL','D-glucose, D-mannose, D-fructose, sorbitol and D-glucosamine can actas acceptors.ITP and dATP can act as donors.','ATP + D-hexose = ADP + D-hexose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.2','Glucokinase','Glucose kinase','NULL','A group of enzymes found in invertebrates and microorganisms highlyspecific for glucose.','ATP + D-glucose = ADP + D-glucose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.3','Ketohexokinase','Hepatic fructokinase','NULL','D-sorbose, D-agatose and 5-dehydro-D-fructose also act as acceptors.','ATP + D-fructose = ADP + D-fructose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.4','Fructokinase','D-fructokinase','NULL','NULL','ATP + D-fructose = ADP + D-fructose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.5','Rhamnulokinase','NULL','NULL','NULL','ATP + L-rhamnulose = ADP + L-rhamnulose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.6','Galactokinase','NULL','NULL','Part of the Leloir pathway for galactose metabolism.The enzymes from mammals and from the bacterium Escherichia coli haveno activity with N-acetyl-alpha-D-galactosamine.','ATP + alpha-D-galactose = ADP + alpha-D-galactose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.7','Mannokinase','NULL','NULL','NULL','ATP + D-mannose = ADP + D-mannose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.8','Glucosamine kinase','NULL','NULL','NULL','ATP + D-glucosamine = ADP + D-glucosamine phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.9','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.10','Phosphoglucokinase','Glucose-phosphate kinase','NULL','NULL','ATP + alpha-D-glucose 1-phosphate = ADP + alpha-D-glucose 1,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.11','6-phosphofructokinase','Phosphofructokinase I','NULL','D-tagatose 6-phosphate and sedoheptulose 7-phosphate can act asacceptors.UTP, CTP and ITP can act as donors.Not identical with EC 2.7.1.105.','ATP + D-fructose 6-phosphate = ADP + D-fructose 1,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.12','Gluconokinase','Gluconate kinase','NULL','NULL','ATP + D-gluconate = ADP + 6-phospho-D-gluconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.13','Dehydrogluconokinase','Ketogluconokinase','NULL','NULL','ATP + 2-dehydro-D-gluconate = ADP + 6-phospho-2-dehydro-D-gluconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.14','Sedoheptulokinase','Heptulokinase','NULL','NULL','ATP + sedoheptulose = ADP + sedoheptulose 7-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.15','Ribokinase','NULL','NULL','2-deoxy-D-ribose can also act as acceptor.','ATP + D-ribose = ADP + D-ribose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.16','Ribulokinase','L-ribulokinase','NULL','Ribitol and L-arabinitol can also act as acceptors.','ATP + L(or D)-ribulose = ADP + L(or D)-ribulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.17','Xylulokinase','Xylulose kinase','NULL','NULL','ATP + D-xylulose = ADP + D-xylulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.18','Phosphoribokinase','NULL','NULL','NULL','ATP + D-ribose 5-phosphate = ADP + alpha-D-ribose 1,5-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.19','Phosphoribulokinase','Phosphopentokinase','NULL','NULL','ATP + D-ribulose 5-phosphate = ADP + D-ribulose 1,5-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.20','Adenosine kinase','NULL','NULL','2-aminoadenosine can also act as acceptor.','ATP + adenosine = ADP + AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.21','Thymidine kinase','NULL','NULL','Deoxyuridine can also act as acceptor.dGTP can also act as donor.The deoxypyrimidine kinase complex induced by Herpes simplex viruscatalyzes this reaction as well as those of EC 2.7.1.114,EC 2.7.1.118, and EC 2.7.4.9.Formerly EC 2.7.1.75.','ATP + thymidine = ADP + thymidine 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.22','Ribosylnicotinamide kinase','NULL','NULL','NULL','ATP + 1-(beta-D-ribofuranosyl)-nicotinamide = ADP + beta-nicotinamideD-ribonucleotide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.23','NAD(+) kinase','DPN kinase','NULL','NULL','ATP + NAD(+) = ADP + NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.24','Dephospho-CoA kinase','Dephosphocoenzyme A kinase','NULL','NULL','ATP + 3-dephospho-CoA = ADP + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.25','Adenylyl-sulfate kinase','Adenosine 5-phosphosulfate kinase','NULL','The human phosphoadenosine-phosphosulfate synthase (PAPS) system is abifunctional enzyme: ATP sulfurylase, which catalyzes the formationof adenosine 5-phosphosulfate (APS) from ATP and inorganic sulfateand the second step is catalyzed by the APS kinase portion of3-phosphoadenosine 5-phosphosulfate (PAPS) synthase, which involvesthe formation of PAPS from enzyme bound APS and ATP.This is in contrast to what is found in bacteria, yeasts, fungi andplants, where the formation of PAPS is carried out by two individualpolypeptides, EC 2.7.7.4 and EC 2.7.1.25.','ATP + adenylyl sulfate = ADP + 3-phosphoadenylyl sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.26','Riboflavin kinase','Flavokinase','Magnesium or zinc or manganese.','The cofactors FMN and FAD participate in numerous processes in allorganisms, including mitochondrial electron transport,photosynthesis, fatty-acid oxidation, and metabolism of vitamin B(6),vitamin B12 and folates.While monofunctional riboflavin kinase is found in eukaryotes, somebacteria have a bifunctional enzyme that exhibits both this activityand that of EC 2.7.7.2.In Bacillus subtilis, ATP can be replaced by other phosphate donorsbut with decreasing enzyme activity in the order ATP ','ATP + riboflavin = ADP + FMN.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.27','Erythritol kinase','NULL','NULL','NULL','ATP + erythritol = ADP + D-erythritol 4-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.28','Triokinase','Triose kinase','NULL','NULL','ATP + D-glyceraldehyde = ADP + D-glyceraldehyde 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.29','Glycerone kinase','Dihydroxyacetone kinase','NULL','NULL','ATP + glycerone = ADP + glycerone phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.30','Glycerol kinase','ATP:glycerol 3-phosphotransferase','NULL','Glycerone and L-glyceraldehyde can act as acceptors.UTP (and, in the case of the Saccharomyces cerevisiae enzyme, ITP andGTP) can act as donors.','ATP + glycerol = ADP + sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.31','Glycerate 3-kinase','NULL','NULL','NULL','ATP + D-glycerate = ADP + 3-phospho-D-glycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.32','Choline kinase','NULL','NULL','Ethanolamine and its methyl and ethyl derivatives can also act asacceptors.','ATP + choline = ADP + phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.33','Pantothenate kinase','ATP:pantothenate 4-phosphotransferase','NULL','NULL','ATP + (R)-pantothenate = ADP + (R)-4-phosphopantothenate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.34','Pantetheine kinase','NULL','NULL','NULL','ATP + pantetheine = ADP + pantetheine 4-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.35','Pyridoxal kinase','Pyridoxamine kinase','NULL','Pyridoxine, pyridoxamine and various derivatives can also act asacceptors.','ATP + pyridoxal = ADP + pyridoxal 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.36','Mevalonate kinase','NULL','NULL','CTP, GTP and UTP can also act as donors.','ATP + (R)-mevalonate = ADP + (R)-5-phosphomevalonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.37','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.38','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.39','Homoserine kinase','NULL','NULL','NULL','ATP + L-homoserine = ADP + O-phospho-L-homoserine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.40','Pyruvate kinase','Phosphoenol transphosphorylase','NULL','UTP, GTP, CTP, ITP and dATP can also act as donors.Also phosphorylates hydroxylamine and fluoride in the presence ofCO(2).','ATP + pyruvate = ADP + phosphoenolpyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.41','Glucose-1-phosphate phosphodismutase','NULL','NULL','NULL','2 D-glucose 1-phosphate = D-glucose + D-glucose 1,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.42','Riboflavin phosphotransferase','NULL','NULL','NULL','Alpha-D-glucose 1-phosphate + riboflavin = D-glucose + FMN.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.43','Glucuronokinase','NULL','NULL','NULL','ATP + D-glucuronate = ADP + 1-phospho-alpha-D-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.44','Galacturonokinase','NULL','NULL','NULL','ATP + D-galacturonate = ADP + 1-phospho-alpha-D-galacturonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.45','2-dehydro-3-deoxygluconokinase','2-keto-3-deoxygluconokinase','NULL','The enzyme shows no activity with 2-dehydro-3-deoxy-D-galactonate.Cf. EC 2.7.1.178.','ATP + 2-dehydro-3-deoxy-D-gluconate = ADP + 2-dehydro-3-deoxy-6-phospho-D-gluconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.46','L-arabinokinase','NULL','NULL','NULL','ATP + L-arabinose = ADP + beta-L-arabinose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.47','D-ribulokinase','NULL','NULL','NULL','ATP + D-ribulose = ADP + D-ribulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.48','Uridine kinase','Uridine monophosphokinase','NULL','Cytidine can act as acceptor.GTP and ITP can act as donors.','(1) ATP + uridine = ADP + UMP.(2) ATP + cytidine = ADP + CMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.49','Hydroxymethylpyrimidine kinase','NULL','NULL','CTP, UTP and GTP can act as donors.','ATP + 4-amino-5-hydroxymethyl-2-methylpyrimidine = ADP + 4-amino-5-phosphonooxymethyl-2-methylpyrimidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.50','Hydroxyethylthiazole kinase','NULL','NULL','NULL','ATP + 4-methyl-5-(2-hydroxyethyl)thiazole = ADP + 4-methyl-5-(2-phosphonooxyethyl)thiazole.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.51','L-fuculokinase','NULL','NULL','NULL','ATP + L-fuculose = ADP + L-fuculose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.52','Fucokinase','ATP:6-deoxy-L-galactose 1-phosphotransferase','Divalent cation.','Forms part of a salvage pathway for reutilization of L-fucose.Can also phosphorylate D-arabinose, but more slowly.','ATP + L-fucose = ADP + beta-L-fucose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.53','L-xylulokinase','L-xylulose kinase','NULL','NULL','ATP + L-xylulose = ADP + L-xylulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.54','D-arabinokinase','NULL','NULL','NULL','ATP + D-arabinose = ADP + D-arabinose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.55','Allose kinase','Allokinase','NULL','NULL','ATP + D-allose = ADP + D-allose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.56','1-phosphofructokinase','Fructose 1-phosphate kinase','NULL','ITP, GTP or UTP can replace ATP.','ATP + D-fructose 1-phosphate = ADP + D-fructose 1,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.57','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.58','2-dehydro-3-deoxygalactonokinase','2-keto-3-deoxy-galactonokinase','NULL','NULL','ATP + 2-dehydro-3-deoxy-D-galactonate = ADP + 2-dehydro-3-deoxy-D-galactonate 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.59','N-acetylglucosamine kinase','GlcNAc kinase','NULL','The bacterial enzyme also acts on D-glucose.','ATP + N-acetyl-D-glucosamine = ADP + N-acetyl-D-glucosamine 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.60','N-acylmannosamine kinase','NULL','NULL','Acts on the acetyl and glycolyl derivatives.','ATP + N-acyl-D-mannosamine = ADP + N-acyl-D-mannosamine 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.61','Acyl-phosphate--hexose phosphotransferase','NULL','NULL','Phosphorylates D-glucose and D-mannose on O-6, and D-fructose on O-1or O-6.','Acyl phosphate + D-hexose = a carboxylate + D-hexose phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.62','Phosphoramidate--hexose phosphotransferase','NULL','NULL','Activity is observed with several hexoses; of these glucose is thebest substrate and the product from it is alpha-D-glucose1-phosphate.The phosphoramidate donor can be replaced by N-phosphoglycine and byan N-phosphohistidine.May be identical with EC 3.1.3.9.','Phosphoramidate + D-hexose = NH(3) + alpha-D-hexose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.63','Polyphosphate--glucose phosphotransferase','Polyphosphate glucokinase','NULL','Requires a neutral salt, e.g. KCl, for maximum activity.Also acts on glucosamine.','(Phosphate)(n) + D-glucose = (phosphate)(n-1) + D-glucose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.64','Inositol 3-kinase','Inositol 1-kinase','NULL','NULL','ATP + myo-inositol = ADP + 1D-myo-inositol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.65','Scyllo-inosamine 4-kinase','NULL','NULL','Also acts on streptamine, 2-deoxystreptamine and 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol.','ATP + 1-amino-1-deoxy-scyllo-inositol = ADP + 1-amino-1-deoxy-scyllo-inositol 4-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.66','Undecaprenol kinase','Isoprenoid-alcohol kinase','NULL','NULL','ATP + undecaprenol = ADP + undecaprenyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.67','1-phosphatidylinositol 4-kinase','Phosphatidylinositol 4-kinase','NULL','NULL','ATP + 1-phosphatidyl-1D-myo-inositol = ADP + 1-phosphatidyl-1D-myo-inositol 4-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.68','1-phosphatidylinositol-4-phosphate 5-kinase','Diphosphoinositide kinase','NULL','This enzyme can also phosphorylate PtdIns3P in the 4-position,and PtdIns, PtdIns3P and PtdIns(3,4)P(2) in the 5-position in vitro,but to a lesser extent.The last of these reactions occurs in vivo and is physiologicallyrelevant.','ATP + 1-phosphatidyl-1D-myo-inositol 4-phosphate = ADP + 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.69','Protein-N(pi)-phosphohistidine--sugar phosphotransferase','Enzyme II of the phosphotransferase system','NULL','Comprises a group of related enzymes.The protein substrate is a phosphocarrier protein of low molecularmass (9.5 kDa).The protein is phosphorylated in a reaction catalyzed by EC 2.7.3.9and this acts as the phosphate donor for the above reaction.The enzyme translocates the sugar it phosphorylates into bacteria.Aldohexoses and their glycosides and alditols are phosphorylated onO-6; fructose and sorbose on O-1.Glycerone and disaccharides are also substrates.','Protein EIIB N(pi)-phospho-L-histidine/cysteine + sugar = protein EIIB +sugar phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.70','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.71','Shikimate kinase','NULL','NULL','NULL','ATP + shikimate = ADP + shikimate 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.72','Streptomycin 6-kinase','APH(6)','NULL','dATP can replace ATP.Dihydrostreptomycin, streptidine and 2-deoxystreptidine can act asacceptors.','ATP + streptomycin = ADP + streptomycin 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.73','Inosine kinase','Inosine-guanosine kinase','NULL','NULL','ATP + inosine = ADP + IMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.74','Deoxycytidine kinase','NULL','NULL','Cytosine arabinoside can act as acceptor; all natural nucleosidetriphosphates (except dCTP) can act as donors.','NTP + deoxycytidine = NDP + dCMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.75','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.76','Deoxyadenosine kinase','NULL','NULL','Deoxyguanosine can also act as acceptor.Possibly identical with EC 2.7.1.74.','ATP + deoxyadenosine = ADP + dAMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.77','Nucleoside phosphotransferase','NULL','NULL','Phenyl phosphate and nucleoside 3-phosphates can act as donors,although not so well as nucleoside 5-phosphates.Nucleosides as well as 2-deoxynucleosides can act as acceptors.','A nucleotide + a 2-deoxynucleoside = a nucleoside + a 2-deoxynucleoside5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.78','Polynucleotide 5-hydroxyl-kinase','NULL','NULL','Also acts on 5-dephospho-RNA 3-mononucleotides.','ATP + 5-dephospho-DNA = ADP + 5-phospho-DNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.79','Diphosphate--glycerol phosphotransferase','Pyrophosphate--glycerol phosphotransferase','NULL','May be identical with EC 3.1.3.9.','Diphosphate + glycerol = phosphate + glycerol 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.80','Diphosphate--serine phosphotransferase','Pyrophosphate--serine phosphotransferase','NULL','NULL','Diphosphate + L-serine = phosphate + O-phospho-L-serine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.81','Hydroxylysine kinase','NULL','NULL','Both the natural 5-hydroxy-L-lysine and its 5-epimer act asacceptors.','GTP + 5-hydroxy-L-lysine = GDP + 5-phosphonooxy-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.82','Ethanolamine kinase','NULL','NULL','NULL','ATP + ethanolamine = ADP + O-phosphoethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.83','Pseudouridine kinase','NULL','NULL','NULL','ATP + pseudouridine = ADP + pseudouridine 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.84','Alkylglycerone kinase','NULL','NULL','NULL','ATP + O-alkylglycerone = ADP + O-alkylglycerone phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.85','Beta-glucoside kinase','NULL','NULL','Phosphorylates a number of beta-D-glucosides.GTP, CTP, ITP and UTP can also act as donors.','ATP + cellobiose = ADP + 6-phospho-beta-D-glucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.86','NADH kinase','NULL','NULL','CTP, ITP, UTP and GTP can also act as phosphate donors (in decreasingorder of activity).Specific for NADH.Activated by acetate.Formerly EC 2.7.1.96.','ATP + NADH = ADP + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.87','Streptomycin 3-kinase','Streptomycin 3-phosphotransferase','NULL','Also phosphorylates dihydrostreptomycin, 3-deoxydihydrostreptomycinand their 6-phosphates.','ATP + streptomycin = ADP + streptomycin 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.88','Dihydrostreptomycin-6-phosphate 3-alpha-kinase','NULL','NULL','3-deoxydihydrostreptomycin 6-phosphate can also act as acceptor.','ATP + dihydrostreptomycin 6-phosphate = ADP + dihydrostreptomycin3-alpha,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.89','Thiamine kinase','NULL','NULL','NULL','ATP + thiamine = ADP + thiamine phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.90','Diphosphate--fructose-6-phosphate 1-phosphotransferase','6-phosphofructokinase (diphosphate)','NULL','NULL','Diphosphate + D-fructose 6-phosphate = phosphate + D-fructose 1,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.91','Sphinganine kinase','Dihydrosphingosine kinase','NULL','NULL','ATP + sphinganine = ADP + sphinganine 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.92','5-dehydro-2-deoxygluconokinase','5-keto-2-deoxygluconokinase','NULL','NULL','ATP + 5-dehydro-2-deoxy-D-gluconate = ADP + 6-phospho-5-dehydro-2-deoxy-D-gluconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.93','Alkylglycerol kinase','NULL','NULL','NULL','ATP + 1-O-alkyl-sn-glycerol = ADP + 1-O-alkyl-sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.94','Acylglycerol kinase','Monoacylglycerol kinase','NULL','Acts on both 1- and 2-acylglycerols.','ATP + acylglycerol = ADP + acyl-sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.95','Kanamycin kinase','Aminoglycoside 3-phosphotransferase','NULL','Also acts on the antibiotics neomycin, paromomycin, neamine,paromamine, vistamycin and gentamicin A.An enzyme from Pseudomonas aeruginosa also acts on butirosin.','ATP + kanamycin = ADP + kanamycin 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.96','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.97','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.98','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.99','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.100','S-methyl-5-thioribose kinase','5-methylthioribose kinase','NULL','Also acts, more slowly, on CTP.','ATP + S-methyl-5-thio-D-ribose = ADP + S-methyl-5-thio-alpha-D-ribose1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.101','Tagatose kinase','D-tagatose 6-phosphate kinase','NULL','NULL','ATP + D-tagatose = ADP + D-tagatose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.102','Hamamelose kinase','NULL','NULL','Also acts, more slowly, on D-hamamelitol.','ATP + D-hamamelose = ADP + D-hamamelose 2-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.103','Viomycin kinase','Capreomycin phosphotransferase','NULL','Also acts on capreomycins.A serine residue in the peptide antibiotics acts as phosphate-acceptor.','ATP + viomycin = ADP + O-phosphoviomycin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.104','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.105','6-phosphofructo-2-kinase','Phosphofructokinase 2','NULL','Not identical with EC 2.7.1.11.The enzyme copurifies with EC 3.1.3.46.','ATP + D-fructose 6-phosphate = ADP + beta-D-fructose 2,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.106','Glucose-1,6-bisphosphate synthase','Glucose 1,6-diphosphate synthase','NULL','Alpha-D-glucose 6-phosphate can act as acceptor, forming alpha-D-glucose 1,6-bisphosphate.','3-phospho-D-glyceroyl phosphate + alpha-D-glucose 1-phosphate =3-phospho-D-glycerate + alpha-D-glucose 1,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.107','Diacylglycerol kinase (ATP)','DGK','NULL','Involved in synthesis of membrane phospholipids and the neutral lipidtriacylglycerol.Activity is stimulated by certain phospholipids.In plants and animals the product 1,2-diacyl-sn-glycerol 3-phosphateis an important second messenger.cf. EC 2.7.1.174.','ATP + 1,2-diacyl-sn-glycerol = ADP + 1,2-diacyl-sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.108','Dolichol kinase','NULL','NULL','NULL','CTP + dolichol = CDP + dolichyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.109','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.110','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.111','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.112','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.113','Deoxyguanosine kinase','NULL','NULL','Deoxyinosine can also act as acceptor.','ATP + deoxyguanosine = ADP + dGMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.114','AMP--thymidine kinase','Adenylate-nucleoside phosphotransferase','NULL','The deoxypyrimidine kinase complex induced by Herpes simplex viruscatalyzes this reaction as well as those of EC 2.7.1.21,EC 2.7.1.128, and EC 2.7.4.9.','AMP + thymidine = adenosine + thymidine 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.115','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.116','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.117','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.118','ADP--thymidine kinase','NULL','NULL','The deoxypyrimidine kinase complex induced by Herpes simplex viruscatalyzes this reaction as well as those of EC 2.7.1.21, EC 2.7.1.114and EC 2.7.4.9.','ADP + thymidine = AMP + thymidine 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.119','Hygromycin-B 7-O-kinase','APH(7)','NULL','Phosphorylates the antibiotics hygromycin B, 1-N-hygromycin B anddestomycin, but not hygromycin B2, at the 7-hydroxy group in thedestomic acid ring.','ATP + hygromycin B = ADP + 7-O-phosphohygromycin B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.120','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.121','Phosphoenolpyruvate--glycerone phosphotransferase','NULL','NULL','NULL','Phosphoenolpyruvate + glycerone = pyruvate + glycerone phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.122','Xylitol kinase','Xylitol phosphotransferase','NULL','NULL','ATP + xylitol = ADP + xylitol 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.123','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.124','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.125','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.126','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.127','Inositol-trisphosphate 3-kinase','1D-myo-inositol-trisphosphate 3-kinase','Calcium.','NULL','ATP + 1D-myo-inositol 1,4,5-trisphosphate = ADP + 1D-myo-inositol1,3,4,5-tetrakisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.128','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.129','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.130','Tetraacyldisaccharide 4-kinase','Lipid-A 4-kinase','NULL','Involved with EC 2.3.1.129 and 2.4.1.182 in the biosynthesis of thephosphorylated glycolipid, lipid A, in the outer membrane ofEscherichia coli.','ATP + (2-N,3-O-bis(3-hydroxytetradecanoyl)-beta-D-glucosaminyl)-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.131','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.132','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.133','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.134','Inositol-tetrakisphosphate 1-kinase','1D-myo-inositol-tetrakisphosphate 1-kinase','NULL','This enzyme also phosphorylates Ins(1,3,4)P(3) on O-5 and O-6.The phosphotransfer from ATP to either inositol 1,3,4-trisphosphateor inositol 3,4,5,6-tetrakisphosphate appears to be freely reversibleto the extent that the enzyme can act like an inositol polyphosphatephosphatase in the presence of ADP.It can also catalyze an isomerization between Ins(1,3,4,5)P(4) andIns(1,3,4,6)P(4) in the presence of ADP.Formerly EC 2.7.1.133 and EC 2.7.1.139.','ATP + 1D-myo-inositol 3,4,5,6-tetrakisphosphate = ADP + 1D-myo-inositol1,3,4,5,6-pentakisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.135','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.136','Macrolide 2-kinase','NULL','NULL','Erythromycin, spiramycin and some other macrolide antibiotics canalso act as acceptors.','ATP + oleandomycin = ADP + oleandomycin 2-O-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.137','Phosphatidylinositol 3-kinase','1-phosphatidylinositol 3-kinase','NULL','NULL','ATP + 1-phosphatidyl-1D-myo-inositol = ADP + 1-phosphatidyl-1D-myo-inositol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.138','Ceramide kinase','Acylsphingosine kinase','NULL','NULL','ATP + ceramide = ADP + ceramide 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.139','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.140','Inositol-tetrakisphosphate 5-kinase','1D-myo-inositol-tetrakisphosphate 5-kinase','NULL','The enzyme from plants and yeast can also use Ins(1,2,3,4,6)P(5) as asubstrate.','ATP + 1D-myo-inositol 1,3,4,6-tetrakisphosphate = ADP + 1D-myo-inositol1,3,4,5,6-pentakisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.141','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.142','Glycerol-3-phosphate--glucose phosphotransferase','NULL','NULL','Involved in the anaerobic metabolism of sugars in the bloodstream oftrypanosomes.','sn-glycerol 3-phosphate + D-glucose = glycerol + D-glucose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.143','Diphosphate-purine nucleoside kinase','Diphosphate-dependent nucleoside kinase','NULL','The enzyme from the Acholeplasma class of Mollicutes catalyzes theconversion of adenosine, guanosine and inosine to AMP, GMP and IMP.ATP cannot substitute for diphosphate as a substrate.','Diphosphate + a purine nucleoside = phosphate + a purine mononucleotide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.144','Tagatose-6-phosphate kinase','Phosphotagatokinase','NULL','NULL','ATP + D-tagatose 6-phosphate = ADP + D-tagatose 1,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.145','Deoxynucleoside kinase','Ms-dNK','NULL','The enzyme from embryonic cells of Drosophila melanogaster differsfrom other deoxynucleoside kinases (EC 2.7.1.76 and EC 2.7.1.113) inits broad specificity for all four common deoxynucleosides.','ATP + 2-deoxynucleoside = ADP + 2-deoxynucleoside 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.146','ADP-specific phosphofructokinase','ADP-6-phosphofructokinase','Magnesium or cobalt.','ADP can be replaced by GDP, ATP and GTP, to a limited extent.','ADP + D-fructose 6-phosphate = AMP + D-fructose 1,6-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.147','ADP-specific glucokinase','ADP-dependent glucokinase','Magnesium.','The enzyme from Pyrococcus furiosus is highly specific for D-glucose;there is some activity with 2-deoxy-D-glucose, but no activity withD-fructose, D-mannose or D-galactose as the substrate.No activity is detected when ADP is replaced by ATP, GDP,phosphoenolpyruvate, diphosphate or polyphosphate.','ADP + D-glucose = AMP + D-glucose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.148','4-(cytidine 5-diphospho)-2-C-methyl-D-erythritol kinase','4-diphosphocytidyl-2-C-methyl-D-erythritol kinase','Manganese or magnesium.','Forms part of an alternative nonmevalonate pathway for terpenoidbiosynthesis.','ATP + 4-(cytidine 5-diphospho)-2-C-methyl-D-erythritol = ADP +2-phospho-4-(cytidine 5-diphospho)-2-C-methyl-D-erythritol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.149','1-phosphatidylinositol-5-phosphate 4-kinase','Type II PIP kinase','NULL','NULL','ATP + 1-phosphatidyl-1D-myo-inositol 5-phosphate = ADP + 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.150','1-phosphatidylinositol-3-phosphate 5-kinase','Phosphatidylinositol 3-phosphate 5-kinase','NULL','NULL','ATP + 1-phosphatidyl-1D-myo-inositol 3-phosphate = ADP + 1-phosphatidyl-1D-myo-inositol 3,5-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.151','Inositol-polyphosphate multikinase','Inositol polyphosphate 6-/3-/5-kinase','NULL','This enzyme also phosphorylates Ins(1,4,5)P(3) to Ins(1,3,4,5)P(4),Ins(1,3,4,5)P(4) to Ins(1,3,4,5,6)P(5), and Ins(1,3,4,5,6)P(4) toIns(PP)P(4), isomer unknown.The enzyme from the plant Arabidopsis thaliana can also phosphorylateIns(1,3,4,6)P(4) and Ins(1,2,3,4,6)P(5) at the D-5-position toproduce 1,3,4,5,6-pentakisphosphate and inositol hexakisphosphate(InsP(6)), respectively.Yeast produces InsP(6) from Ins(1,4,5)P(3) by the actions of thisenzyme and EC 2.7.1.158.','2 ATP + 1D-myo-inositol 1,4,5-trisphosphate = 2 ADP + 1D-myo-inositol1,3,4,5,6-pentakisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.152','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.153','Phosphatidylinositol-4,5-bisphosphate 3-kinase','Type I phosphoinositide 3-kinase','NULL','This enzyme also catalyzes the phosphorylation of PtdIns4P toPtdIns(3,4)P(2), and of PtdIns to PtdIns3P.','ATP + 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate = ADP +1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.154','Phosphatidylinositol-4-phosphate 3-kinase','C2-domain-containing phosphoinositide 3-kinase','NULL','This enzyme also phosphorylates PtdIns to PtdIns3P.','ATP + 1-phosphatidyl-1D-myo-inositol 4-phosphate = ADP + 1-phosphatidyl-1D-myo-inositol 3,4-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.155','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.156','Adenosylcobinamide kinase','Adenosylcobinamide kinase/adenosylcobinamide-phosphateguanylyltransferase','NULL','In Salmonella typhimurium LT2, under anaerobic conditions, CobU(EC 2.7.7.62 and EC 2.7.1.156), CobT (EC 2.4.2.21),CobC (EC 3.1.3.73) and CobS (EC 2.7.8.26) catalyze reactions in thenucleotide loop assembly pathway, which convert adenosylcobinamide(AdoCbi) into adenosylcobalamin (AdoCbl).CobT and CobC are involved in 5,6-dimethylbenzimidazole activationwhereby 5,6-dimethylbenzimidazole is converted to its riboside,alpha-ribazole.The second branch of the nucleotide loop assembly pathway is thecobinamide (Cbi) activation branch where AdoCbi oradenosylcobinamide-phosphate is converted to the activatedintermediate AdoCbi-GDP by CobU.The final step in adenosylcobalamin biosynthesis is the condensationof AdoCbi-GDP with alpha-ribazole, which is catalyzed by CobS(EC 2.7.8.26), to yield adenosylcobalamin.CobU is a bifunctional enzyme that has both kinase (EC 2.7.1.156) andguanylyltransferase (EC 2.7.7.62) activities.However, both activities are not required at all times.The kinase activity has been proposed to function only whenS.typhimurium is assimilating cobinamide whereas theguanylyltransferase activity is required for both assimilation ofexogenous cobinamide and for de novo synthesis of adenosylcobalamin.','ATP or GTP + adenosylcobinamide = adenosylcobinamide phosphate +ADP or GDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.157','N-acetylgalactosamine kinase','GALK2','Magnesium.','The enzyme is highly specific for GalNAc as substrate, but has slightactivity with D-galactose.','ATP + N-acetyl-alpha-D-galactosamine = ADP + N-acetyl-alpha-D-galactosamine 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.158','Inositol-pentakisphosphate 2-kinase','Inositol 1,3,4,5,6-pentakisphosphate 2-kinase','NULL','The enzyme can also use Ins(1,4,5,6)P(4) and Ins(1,4,5)P(3) assubstrate.Inositol hexakisphosphate (phytate) accumulates in storage proteinbodies during seed development and, when hydrolyzed, releases storednutrients to the developing seedling before the plant is capable ofabsorbing nutrients from its environment.','ATP + 1D-myo-inositol 1,3,4,5,6-pentakisphosphate = ADP + 1D-myo-inositolhexakisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.159','Inositol-1,3,4-trisphosphate 5/6-kinase','Inositol trisphosphate 5/6-kinase','NULL','In humans, this enzyme, along with EC 2.7.1.127, EC 2.7.1.140 andEC 2.7.1.158 is involved in the production of inositolhexakisphosphate (InsP(6)).InsP(6) is involved in many cellular processes, including mRNA exportfrom the nucleus.Yeast do not have this enzyme, so produce InsP(6) from Ins(1,4,5)P(3)by the actions of EC 2.7.1.151 and EC 2.7.1.158.','(1) ATP + 1D-myo-inositol 1,3,4-trisphosphate = ADP + 1D-myo-inositol1,3,4,5-tetrakisphosphate.(2) ATP + 1D-myo-inositol 1,3,4-trisphosphate = ADP + 1D-myo-inositol1,3,4,6-tetrakisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.160','2-phosphotransferase','Tpt1p','NULL','Catalyzes the final step of tRNA splicing in the yeast Saccharomycescerevisiae.The reaction takes place in two steps: in the first step,the 2-phosphate on the RNA substrate is ADP-ribosylated, causing therelase of nicotinamide and the formation of the reactionintermediate, ADP-ribosylated tRNA.In the second step, dephosphorylated (mature) tRNA is formed alongwith ADP ribose 1,2-cyclic phosphate.Highly specific for oligonucleotide substrates bearing an internal2-phosphate.Oligonucleotides with only a terminal 5- or 3-phosphate are notsubstrates.','2-phospho-[ligated tRNA] + NAD(+) = mature tRNA + ADP ribose1,2-phosphate + nicotinamide + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.161','CTP-dependent riboflavin kinase','NULL','NULL','This archaeal enzyme differs from EC 2.7.1.26 in using CTP as thedonor nucleotide.UTP, but not ATP or GTP, can also act as a phosphate donor but it isat least an order of magnitude less efficient than CTP.Formerly EC 2.7.1.n2.','CTP + riboflavin = CDP + FMN.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.162','N-acetylhexosamine 1-kinase','N-acetylgalactosamine/N-acetylglucosamine 1-kinase','Magnesium.','Involved in the lacto-N-biose I/galacto-N-biose degradation pathwayin the probiotic bacterium Bifidobacterium longum.Differs from EC 2.7.1.157 as it can phosphorylate bothN-acetylgalactosamine and N-acetylglucosamine at similar rates.Also has some activity with N-acetyl-D-mannosamine, D-talose andD-mannose as substrate.ATP can be replaced by GTP or ITP but with decreased enzyme activity.','ATP + N-acetyl-D-hexosamine = ADP + N-acetyl-alpha-D-hexosamine1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.163','Hygromycin B 4-O-kinase','Hygromycin-B kinase','NULL','Phosphorylates the antibiotic hygromycin B.Whereas the enzyme from Streptomyces hygroscopicus (EC 2.7.1.119)catalyzes the formation of 7-O-phosphohygromycin B, this enzyme,found in Escherichia coli carrying a plasmid conferring resistance tohygromycin-B, forms 4-O-phosphohygromycin B.Formerly EC 2.7.1.n6.','ATP + hygromycin B = ADP + 4-O-phosphohygromycin B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.164','O-phosphoseryl-tRNA(Sec) kinase','L-seryl-tRNA(Sec) kinase','NULL','In archaea and eukarya selenocysteine formation is achieved by a two-step process: O-phosphoseryl-tRNA(Sec) kinase (PSTK) phosphorylatesthe endogenous L-seryl-tRNA(Sec) to O-phospho-L-seryl-tRNA(Sec),and then this misacylated amino acid-tRNA species is converted toL-selenocysteinyl-tRNA(Sec) by EC 2.9.1.2.Formerly EC 2.7.1.n3.','ATP + L-seryl-tRNA(Sec) = ADP + O-phospho-L-seryl-tRNA(Sec).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.165','Glycerate 2-kinase','D-glycerate-2-kinase','Divalent cation.','A key enzyme in the nonphosphorylative Entner-Doudoroff pathway inarchaea.In Hyphomicrobium methylovorum GM2 the enzyme is involved informaldehyde assaimilation I (serine pathway).In Escherichia coli the enzyme is involved inD-glucarate/D-galactarate degradation.','ATP + D-glycerate = ADP + 2-phospho-D-glycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.166','3-deoxy-D-manno-octulosonic acid kinase','Kdo kinase','NULL','The enzyme phosphorylates the 4-OH position of KDO in (KDO)-lipidIV(A).','Alpha-Kdo-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.167','D-glycero-beta-D-manno-heptose-7-phosphate kinase','D-beta-D-heptose 7-phosphate kinase/D-beta-D-heptose 1-phosphateadenylyltransferase','NULL','The bifunctional protein hldE includes D-glycero-beta-D-manno-heptose-7-phosphate kinase and D-glycero-beta-D-manno-heptose1-phosphate adenylyltransferase activity (cf. EC 2.7.7.70).The enzyme is involved in biosynthesis of ADP-L-glycero-beta-D-manno-heptose, which is utilized for assembly of the lipopolysaccharideinner core in Gram-negative bacteria.The enzyme selectively produces D-glycero-beta-D-manno-heptose 1,7-bisphosphate.','D-glycero-beta-D-manno-heptose 7-phosphate + ATP = D-glycero-beta-D-manno-heptose 1,7-bisphosphate + ADP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.168','D-glycero-alpha-D-manno-heptose-7-phosphate kinase','D-alpha-D-heptose-7-phosphate kinase','NULL','The enzyme is involved in biosynthesis of GDP-D-glycero-alpha-D-manno-heptose, which is required for assembly of S-layer glycoproteinin Gram-positive bacteria.The enzyme is specific for the alpha-anomer.','D-glycero-alpha-D-manno-heptose 7-phosphate + ATP = D-glycero-alpha-D-manno-heptose 1,7-bisphosphate + ADP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.169','Pantoate kinase','NULL','NULL','The conversion of (R)-pantoate to (R)-4-phosphopantothenate is partof the pathway leading to biosynthesis of 4-phosphopantetheine,an essential cofactor of coenzyme A and acyl-carrier protein.In bacteria and eukaryotes this conversion is performed bycondensation with beta-alanine, followed by phosphorylation(EC 6.3.2.1 and EC 2.7.1.33, respectively).In archaea the order of these two steps is reversed,and phosphorylation precedes condensation with beta-alanine.','ATP + (R)-pantoate = ADP + (R)-4-phosphopantoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.170','Anhydro-N-acetylmuramic acid kinase','AnhMurNAc kinase','NULL','This enzyme, along with EC 4.2.1.126, is required for the utilizationof anhydro-N-acetylmuramic acid in proteobacteria.The substrate is either imported from the medium or derived from thebacteriums own cell wall murein during cell wall recycling.The product N-acetylmuramate 6-phosphate is produced as a 7:1 mixtureof the alpha- and beta-anomers.Formerly EC 2.7.1.n1.','ATP + 1,6-anhydro-N-acetyl-beta-muramate + H(2)O = ADP + N-acetylmuramate6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.171','Protein-fructosamine 3-kinase','Fructosamine 3-kinase','NULL','Nonenzymatic glycation is an important factor in the pathogenesis ofdiabetic complications.Key early intermediates in this process are fructosamines, such as[protein]-N(6)-D-fructosyl-L-lysine.Fructosamine-3-kinase is part of an ATP-dependent system for removingcarbohydrates from nonenzymatically glycated proteins.The phosphorylation destablilizes the [protein]-N(6)-D-fructosyl-L-lysine adduct and leads to its spontaneous decomposition.Cf. EC 2.7.1.172.','ATP + [protein]-N(6)-D-fructosyl-L-lysine = ADP + [protein]-N(6)-(3-O-phospho-D-fructosyl)-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.172','Protein-ribulosamine 3-kinase','FN3K-related protein','NULL','This enzyme plays a role in freeing proteins from ribulosamines orpsicosamines, which might arise from the reaction of amines withglucose and/or glycolytic intermediates.This role is shared by EC 2.7.1.171, which has, in addition,the unique capacity to phosphorylate fructosamines.The plant enzyme also phosphorylates [protein]-N(6)-D-erythrulosyl-L-lysine.No activity with [protein]-N(6)-D-fructosyl-L-lysine.','ATP + [protein]-N(6)-D-ribulosyl-L-lysine = ADP + [protein]-N(6)-(3-O-phospho-D-ribulosyl)-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.173','Nicotinate riboside kinase','Ribosylnicotinic acid kinase','NULL','The enzyme from yeast and human also has the activity of EC 2.7.1.22.Formerly EC 2.7.1.n4.','ATP + beta-D-ribosylnicotinate = ADP + nicotinate beta-D-ribonucleotide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.174','Diacylglycerol kinase (CTP)','CTP-dependent diacylglycerol kinase','Magnesium or calcium.','Involved in synthesis of membrane phospholipids and the neutral lipidtriacylglycerol.Unlike the diacylglycerol kinases from bacteria, plants, and animals(cf. EC 2.7.1.107), the enzyme from Saccharomyces cerevisiae utilizesCTP.The enzyme can also use dCTP, but not ATP, GTP or UTP.Formerly EC 2.7.1.n5.','CTP + 1,2-diacyl-sn-glycerol = CDP + 1,2-diacyl-sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.175','Maltokinase','NULL','Magnesium.','Formerly EC 2.7.1.n8.','ATP + maltose = ADP + alpha-maltose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.176','UDP-N-acetylglucosamine kinase','UNAG kinase','NULL','Toxic component of a toxin-antitoxin (TA) module.The phosphorylation of UDP-N-acetyl-alpha-D-glucosamine results inthe inhibition of EC 2.5.1.7, which is then blocked.Phosphorylation is inhibited by cognate epsilon antitoxin.Formerly EC 2.7.1.n7.','ATP + UDP-N-acetyl-alpha-D-glucosamine = ADP + UDP-N-acetyl-alpha-D-glucosamine 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.177','L-threonine kinase','NULL','NULL','The enzyme is involved in the de novo synthesis of adenosylcobalamin.It is specific for ATP and free L-threonine.In the bacterium Salmonella enterica the activity with CTP, GTP,or UTP is 6, 11, and 3% of the activity with ATP.','ATP + L-threonine = ADP + O-phospho-L-threonine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.178','2-dehydro-3-deoxyglucono/galactono-kinase','2-keto-3-deoxy-D-gluconate kinase','NULL','The enzyme from the archaeon Sulfolobus solfataricus is involved inglucose and galactose catabolism via the branched variant of theEntner-Doudoroff pathway.It phosphorylates 2-dehydro-3-deoxy-D-gluconate and 2-dehydro-3-deoxy-D-galactonate with similar catalytic efficiency.Cf. EC 2.7.1.45 and EC 2.7.1.58.','(1) ATP + 2-dehydro-3-deoxy-D-gluconate = ADP + 2-dehydro-3-deoxy-6-phospho-D-gluconate.(2) ATP + 2-dehydro-3-deoxy-D-galactonate = ADP + 2-dehydro-3-deoxy-6-phospho-D-galactonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.179','Kanosamine kinase','NULL','NULL','The enzyme from the bacterium Amycolatopsis mediterranei is specificfor kanosamine.','ATP + kanosamine = ADP + kanosamine 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.180','FAD:protein FMN transferase','Flavin transferase','Magnesium.','The enzyme catalyzes the transfer of the FMN portion of FAD and itscovalent binding to the hydroxyl group of an L-threonine residue in atarget flavin-binding protein such as the B and C subunits ofEC 1.6.5.8.','FAD + [protein]-L-threonine = [protein]-FMN-L-threonine + AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.181','Polymannosyl GlcNAc-diphospho-ditrans,octacis-undecaprenol kinase','NULL','NULL','The enzyme is involved in the biosynthesis of the polymannoseO-polysaccharide in the outer leaflet of the membrane of Escherichiacoli serotype O9a.O-polysaccharide structures vary extensively because of differencesin the number and type of sugars in the repeat unit.The dual kinase/methylase WbdD also catalyzes the methylation of3-phospho-alpha-D-Man-(1-','ATP + alpha-D-Man-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.182','Phytol kinase','NULL','NULL','The enzyme is found in plants and photosynthetic algae and isinvolved in phytol salvage.It can use UTP as an alternative phosphate donor with lower activity.','CTP + phytol = CDP + phytyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.n2','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.n3','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.n4','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.n5','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.n6','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.n7','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.1.n8','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.1','Acetate kinase','Acetate kinase (phosphorylating)','Magnesium.','While purified enzyme from Escherichia coli is specific for acetate,others have found that the enzyme can also use propanoate as asubstrate, but more slowly.Acetate can be converted into the key metabolic intermediate acetyl-CoA by coupling acetate kinase with EC 2.3.1.8.Both this enzyme and EC 2.7.2.15 play important roles in theproduction of propanoate.','ATP + acetate = ADP + acetyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.2','Carbamate kinase','NULL','NULL','NULL','ATP + NH(3) + CO(2) = ADP + carbamoyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.3','Phosphoglycerate kinase','NULL','NULL','NULL','ATP + 3-phospho-D-glycerate = ADP + 3-phospho-D-glyceroyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.4','Aspartate kinase','Aspartokinase','NULL','The enzyme from Escherichia coli is a multifunctional protein, whichalso catalyzes the reaction of EC 1.1.1.3.This is also the case for two of the four isoenzymes in Arabidopsisthaliana.The equilibrium constant strongly favors the reaction from right toleft, i.e. the non-physiological direction of reaction.','ATP + L-aspartate = ADP + 4-phospho-L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.5','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.6','Formate kinase','NULL','NULL','NULL','ATP + formate = ADP + formyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.7','Butyrate kinase','NULL','NULL','The enzyme from Clostridium sp. also acts, more slowly on pentanoateand propanoate, and on some branched-chain fatty acids (cf.EC 2.7.1.14).','ATP + butanoate = ADP + butanoyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.8','Acetylglutamate kinase','NULL','NULL','NULL','ATP + N-acetyl-L-glutamate = ADP + N-acetyl-L-glutamate 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.9','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.10','Phosphoglycerate kinase (GTP)','NULL','NULL','NULL','GTP + 3-phospho-D-glycerate = GDP + 3-phospho-D-glyceroyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.11','Glutamate 5-kinase','Gamma-glutamyl kinase','NULL','The product rapidly cyclizes to 5-oxoproline and phosphate.','ATP + L-glutamate = ADP + L-glutamate 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.12','Acetate kinase (diphosphate)','Acetate kinase (pyrophosphate)','NULL','NULL','Diphosphate + acetate = phosphate + acetyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.13','Glutamate 1-kinase','NULL','NULL','NULL','ATP + L-glutamate = ADP + alpha-L-glutamyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.14','Branched-chain-fatty-acid kinase','Isobutyrate kinase','NULL','3-methylbutanoate, 2-methylbutanoate, pentanoate, butanoate,and propanoate can also act as acceptors (cf. EC 2.7.2.7).','ATP + 2-methylpropanoate = ADP + 2-methylpropanoyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.2.15','Propionate kinase','Propionate/acetate kinase','Magnesium.','Both propanoate and acetate can act as a substrate.Involved in the anaerobic degradation of L-threonine in bacteria.Both this enzyme and EC 2.7.2.1 play important roles in theproduction of propanoate.','ATP + propanoate = ADP + propanoyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.1','Guanidinoacetate kinase','Glycocyamine kinase','NULL','NULL','ATP + guanidinoacetate = ADP + phosphoguanidinoacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.2','Creatine kinase','NULL','NULL','N-ethylglycocyamine can also act as acceptor.','ATP + creatine = ADP + phosphocreatine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.3','Arginine kinase','Arginine phosphokinase','NULL','NULL','ATP + L-arginine = ADP + N(omega)-phospho-L-arginine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.4','Taurocyamine kinase','NULL','NULL','NULL','ATP + taurocyamine = ADP + N-phosphotaurocyamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.5','Lombricine kinase','NULL','NULL','Also acts on methylated lombricines such as thalassemine; thespecificity varies with the source species.','ATP + lombricine = ADP + N-phospholombricine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.6','Hypotaurocyamine kinase','NULL','NULL','Also acts, more slowly, on taurocyamine.','ATP + hypotaurocyamine = ADP + N(omega)-phosphohypotaurocyamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.7','Opheline kinase','NULL','NULL','Has a little activity on taurocyamine, lombricine andphosphotaurocyamine.','ATP + guanidinoethyl methyl phosphate = ADP +N-phosphoguanidinoethyl methylphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.8','Ammonia kinase','NULL','NULL','Has a wide specificity.In the reverse direction, N-phosphoglycine and N-phosphohistidine canalso act as phosphate donors, and ADP, dADP, GDP, CDP, dTDP, dCDP,IDP and UDP can act as phosphate acceptors (in decreasing order ofactivity).','ATP + NH(3) = ADP + phosphoramide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.9','Phosphoenolpyruvate--protein phosphotransferase','Enzyme I of the phosphotransferase system','NULL','Acts only on histidine residues in specific phosphocarrier proteinsof low molecular mass (9.5 kDa) involved in bacterial sugartransport.A similar reaction where the protein is the enzyme EC 2.7.9.2 is partof the mechanism of that enzyme.','Phosphoenolpyruvate + protein L-histidine = pyruvate + protein N(pi)-phospho-L-histidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.10','Agmatine kinase','Phosphagen phosphokinase','NULL','L-arginine can act as acceptor, more slowly.','ATP + agmatine = ADP + N(4)-phosphoagmatine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.11','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.3.12','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.1','Polyphosphate kinase','ATP-polyphosphate phosphotransferase','NULL','NULL','ATP + (phosphate)(n) = ADP + (phosphate)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.2','Phosphomevalonate kinase','NULL','NULL','NULL','ATP + (R)-5-phosphomevalonate = ADP + (R)-5-diphosphomevalonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.3','Adenylate kinase','Adenylic kinase','NULL','Inorganic triphosphate can also act as donor.','ATP + AMP = 2 ADP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.4','Nucleoside-phosphate kinase','NMP-kinase','NULL','Many nucleotides can act as acceptors.Other nucleoside triphosphates can act instead of ATP.','ATP + nucleoside phosphate = ADP + nucleoside diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.5','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.6','Nucleoside-diphosphate kinase','NDK','NULL','Many nucleoside diphosphates can act as acceptors.Many ribo- and deoxyribonucleoside triphosphates can act as donors.','ATP + nucleoside diphosphate = ADP + nucleoside triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.7','Phosphomethylpyrimidine kinase','Hydroxymethylpyrimidine phosphokinase','NULL','NULL','ATP + 4-amino-2-methyl-5-phosphomethylpyrimidine = ADP + 4-amino-2-methyl-5-diphosphomethylpyrimidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.8','Guanylate kinase','Deoxyguanylate kinase','NULL','dGMP can also act as acceptor.dATP can act as donor.','ATP + GMP = ADP + GDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.9','dTMP kinase','Thymidylate kinase','NULL','NULL','ATP + dTMP = ADP + dTDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.10','Nucleoside-triphosphate--adenylate kinase','NULL','NULL','Many nucleoside triphosphates can act as donors.','NTP + AMP = NDP + ADP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.11','(Deoxy)adenylate kinase','NULL','NULL','AMP can also act as acceptor.','ATP + dAMP = ADP + dADP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.12','T(2)-induced deoxynucleotide kinase','NULL','NULL','dTMP and dAMP can act as acceptors.dATP can act as donor.','ATP + dGMP (or dTMP) = ADP + dGDP (or dTDP).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.13','(Deoxy)nucleoside-phosphate kinase','NULL','NULL','dATP can substitute for ATP.','ATP + deoxynucleoside phosphate = ADP + deoxynucleoside diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.14','UMP/CMP kinase','CMP kinase','NULL','This eukaryotic enzyme is a bifunctional enzyme that catalyzes thephosphorylation of both CMP and UMP with similar efficiency.dCMP can also act as acceptor.Different from the monofunctional prokaryotic enzymes EC 2.7.4.25,CMP kinase and EC 2.7.4.22, UMP kinase.This eukaryotic enzyme is a bifunctional enzyme that catalyzes thephosphorylation of both CMP and UMP with similar efficiency.dCMP can also act as acceptor.Different from the monofunctional prokaryotic enzymes EC 2.7.4.25 andEC 2.7.4.22.Formerly EC 2.7.4.5.','(1) ATP + (d)CMP = ADP + (d)CDP.(2) ATP + UMP = ADP + UDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.15','Thiamine-diphosphate kinase','NULL','NULL','NULL','ATP + thiamine diphosphate = ADP + thiamine triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.16','Thiamine-phosphate kinase','Thiamine-monophosphate kinase','NULL','NULL','ATP + thiamine phosphate = ADP + thiamine diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.17','3-phosphoglyceroyl-phosphate--polyphosphate phosphotransferase','NULL','NULL','NULL','3-phospho-D-glyceroyl phosphate + (phosphate)(n) = 3-phosphoglycerate +(phosphate)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.18','Farnesyl-diphosphate kinase','NULL','NULL','ADP can also act as donor.','ATP + farnesyl diphosphate = ADP + farnesyl triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.19','5-methyldeoxycytidine-5-phosphate kinase','NULL','NULL','The enzyme, from phage XP-12-infected Xanthomonas oryzae, convertsm(5)dCMP into m(5)dCDP and then into m(5)dCTP.','ATP + 5-methyldeoxycytidine 5-phosphate = ADP + 5-methyldeoxycytidinediphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.20','Dolichyl-diphosphate--polyphosphate phosphotransferase','NULL','NULL','NULL','Dolichyl diphosphate + (phosphate)(n) = dolichyl phosphate +(phosphate)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.21','Inositol-hexakisphosphate kinase','ATP:1D-myo-inositol-hexakisphosphate phosphotransferase','NULL','Three mammalian isoforms are known to exist.Formerly EC 2.7.1.152.','(1) ATP + 1D-myo-inositol hexakisphosphate = ADP + 1D-myo-inositol5-diphosphate 1,2,3,4,6-pentakisphosphate.(2) ATP + 1D-myo-inositol 1-diphosphate 2,3,4,5,6-pentakisphosphate =ADP + 1D-myo-inositol 1,5-bis(diphosphate) 2,3,4,6-tetrakisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.22','UMP kinase','UMP-kinase','NULL','Strictly specific for UMP as substrate and is used by prokaryotes inthe de novo synthesis of pyrimidines, in contrast to eukaryotes,which use the dual-specificity enzyme EC 2.7.4.14 for the samepurpose.Subject of feedback regulation, being inhibited by UTP and activatedby GTP.','ATP + UMP = ADP + UDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.23','Ribose 1,5-bisphosphate phosphokinase','Ribose 1,5-bisphosphokinase','NULL','This enzyme, found in NAD suppression mutants of Escherichia coli,synthesizes 5-phospho-alpha-D-ribose 1-diphosphate (PRPP) without theparticipation of EC 2.7.6.1.Ribose, ribose 1-phosphate and ribose 5-phosphate are not substrates,and GTP cannot act as a phosphate donor.','ATP + alpha-D-ribose 1,5-bisphosphate = ADP + 5-phospho-alpha-D-ribose1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.24','Diphosphoinositol-pentakisphosphate kinase','Diphosphoinositol pentakisphosphate kinase','NULL','This enzyme is activated by osmotic shock.Ins(1,3,4,5,6)P(5), 1D-myo-inositol diphosphate tetrakisphosphate and1D-myo-inositol bisdiphosphate triphosphate are not substrates.Formerly EC 2.7.1.155.','(1) ATP + 1D-myo-inositol 5-diphosphate 1,2,3,4,6-pentakisphosphate =ADP + 1D-myo-inositol 1,5-bis(diphosphate) 2,3,4,6-tetrakisphosphate.(2) ATP + 1D-myo-inositol hexakisphosphate = ADP + 1D-myo-inositol1-diphosphate 2,3,4,5,6-pentakisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.25','(d)CMP kinase','dCMP kinase','NULL','The prokaryotic cytidine monophosphate kinase specificallyphosphorylates CMP (or dCMP), using ATP as the preferred phosphoryldonor.Unlike EC 2.7.4.14, a eukaryotic enzyme that phosphorylates UMP andCMP with similar efficiency, the prokaryotic enzyme phosphorylatesUMP with very low rates, and this function is catalyzed inprokaryotes by EC 2.7.4.22.The enzyme phosphorylates dCMP nearly as well as it does CMP.','ATP + (d)CMP = ADP + (d)CDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.26','Isopentenyl phosphate kinase','NULL','NULL','The enzyme is involved in the mevalonate pathway in Archaea.The activity has also been identified in the plant Mentha piperita(peppermint).It is strictly specific for ATP but can use other phosphate acceptorssuch as dimethylallyl phosphate, geranyl phosphate, or fosfomycin.','ATP + isopentenyl phosphate = ADP + isopentenyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.27','([Pyruvate, phosphate dikinase] phosphate) phosphotransferase','Bifunctional dikinase regulatory protein','NULL','The enzyme from the plants maize and Arabidopsis is bifunctional andalso catalyzes the phosphorylation of pyruvate, phosphate dikinase(EC 2.7.9.1), cf. EC 2.7.11.32.Formerly EC 2.7.4.n1.','[Pyruvate, phosphate dikinase] phosphate + phosphate = [pyruvate,phosphate dikinase] + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.4.n2','([Pyruvate, water dikinase] phosphate) phosphotransferase','PSRP','NULL','The enzyme from the bacterium Escherichia coli is bifunctional andalso catalyzes the phosphorylation of pyruvate, water dikinase(EC 2.7.9.2), cf. EC 2.7.11.33.Formerly EC 2.7.4.n2.','[Pyruvate, water dikinase] phosphate + phosphate = [pyruvate, waterdikinase] + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.5.1','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.5.2','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.5.3','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.5.4','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.5.5','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.5.6','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.5.7','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.6.1','Ribose-phosphate diphosphokinase','Phosphoribosyl diphosphate synthetase','NULL','dATP can also act as donor.','ATP + D-ribose 5-phosphate = AMP + 5-phospho-alpha-D-ribose1-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.6.2','Thiamine diphosphokinase','Thiamine kinase','NULL','NULL','ATP + thiamine = AMP + thiamine diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.6.3','2-amino-4-hydroxy-6-hydroxymethyldihydropteridine diphosphokinase','2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase','NULL','NULL','ATP + 2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine = AMP +(2-amino-4-hydroxy-7,8-dihydropteridin-6-yl)methyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.6.4','Nucleotide diphosphokinase','Nucleotide pyrophosphokinase','NULL','Acts on the 5-mono-, di- and triphosphate derivatives of purinenucleosides.','ATP + nucleoside 5-phosphate = AMP + 5-phosphonucleoside3-diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.6.5','GTP diphosphokinase','GTP pyrophosphokinase','NULL','GDP can also act as acceptor.','ATP + GTP = AMP + guanosine 3-diphosphate 5-triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.1','Nicotinamide-nucleotide adenylyltransferase','Adenosine triphosphate-nicotinamide mononucleotide transadenylase','NULL','Nicotinate nucleotide can also act as acceptor.See also EC 2.7.7.18.','ATP + nicotinamide ribonucleotide = diphosphate + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.2','FAD synthetase','FAD diphosphorylase','Magnesium.','Highly specific for ATP as phosphate donor.The cofactors FMN and FAD participate in numerous processes in allorganisms, including mitochondrial electron transport,photosynthesis, fatty-acid oxidation, and metabolism of vitamin B6,vitamin B12 and folates.While monofunctional FAD synthetase is found in eukaryotes and insome prokaryotes, most prokaryotes have a bifunctional enzyme thatexhibits both this activity and that of EC 2.7.1.26.','ATP + FMN = diphosphate + FAD.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.3','Pantetheine-phosphate adenylyltransferase','3-dephospho-CoA pyrophosphorylase','NULL','The enzyme from several bacteria (e.g. Escherichia coli, Bacillussubtilis and Haemophilus influenzae) has been shown to bebifunctional and also to possess the activity of EC 2.3.1.157.','ATP + pantetheine 4-phosphate = diphosphate + 3-dephospho-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.4','Sulfate adenylyltransferase','ATP-sulfurylase','NULL','The human phosphoadenosine-phosphosulfate synthase (PAPS) system is abifunctional enzyme: ATP sulfurylase, which catalyzes the formationof adenosine 5-phosphosulfate (APS) from ATP and inorganic sulfateand the second step is catalyzed by the APS kinase portion of3-phosphoadenosine 5-phosphosulfate (PAPS) synthase, which involvesthe formation of PAPS from enzyme bound APS and ATP.This is in contrast to what is found in bacteria, yeasts, fungi andplants, where the formation of PAPS is carried out by two individualpolypeptides, EC 2.7.7.4 and EC 2.7.1.25.','ATP + sulfate = diphosphate + adenylyl sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.5','Sulfate adenylyltransferase (ADP)','Adenosine diphosphate sulfurylase','NULL','NULL','ADP + sulfate = phosphate + adenylyl sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.6','DNA-directed RNA polymerase','DNA-dependent RNA polymerase','NULL','Catalyzes DNA-template-directed extension of the 3-end of an RNAstrand by one nucleotide at a time.Can initiate a chain de novo.In eukaryotes three forms of the enzyme have been distinguished onthe basis of sensitivity of alpha-amanitin, and the type of RNAsynthesized.See also EC 2.7.7.19 and EC 2.7.7.48.','Nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.7','DNA-directed DNA polymerase','DNA nucleotidyltransferase (DNA-directed)','NULL','Catalyzes DNA-template-directed extension of the 3-end of a DNAstrand by one nucleotide at a time.Cannot initiate a chain de novo.Requires a primer which may be DNA or RNA.See also EC 2.7.7.49.','Deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.8','Polyribonucleotide nucleotidyltransferase','Polynucleotide phosphorylase','NULL','ADP, IDP, GDP, UDP and CDP can act as donors.','RNA(n+1) + phosphate = RNA(n) + a nucleoside diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.9','UTP--glucose-1-phosphate uridylyltransferase','Glucose-1-phosphate uridylyltransferase','NULL','NULL','UTP + alpha-D-glucose 1-phosphate = diphosphate + UDP-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.10','UTP--hexose-1-phosphate uridylyltransferase','Gal-1-P uridylyltransferase','NULL','Alpha-D-glucose 1-phosphate can also act as acceptor, more slowly.','UTP + alpha-D-galactose 1-phosphate = diphosphate + UDP-alpha-D-galactose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.11','UTP--xylose-1-phosphate uridylyltransferase','Xylose-1-phosphate uridylyltransferase','NULL','NULL','UTP + alpha-D-xylose 1-phosphate = diphosphate + UDP-D-xylose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.12','UDP-glucose--hexose-1-phosphate uridylyltransferase','Gal-1-P uridylyltransferase','NULL','NULL','UDP-alpha-D-glucose + alpha-D-galactose 1-phosphate = alpha-D-glucose1-phosphate + UDP-alpha-D-galactose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.13','Mannose-1-phosphate guanylyltransferase','GDP-mannose pyrophosphorylase','NULL','The bacterial enzyme can also use ITP and dGTP as donors.','GTP + alpha-D-mannose 1-phosphate = diphosphate + GDP-mannose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.14','Ethanolamine-phosphate cytidylyltransferase','CTP:phosphoethanolamine cytidylyltransferase','NULL','NULL','CTP + ethanolamine phosphate = diphosphate + CDP-ethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.15','Choline-phosphate cytidylyltransferase','CTP:phosphocholine cytidylyltransferase','NULL','NULL','CTP + phosphocholine = diphosphate + CDP-choline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.16','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.17','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.18','Nicotinate-nucleotide adenylyltransferase','Deamido-NAD(+) diphosphorylase','NULL','NULL','ATP + beta-nicotinate-D-ribonucleotide = diphosphate + deamido-NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.19','Polynucleotide adenylyltransferase','NTP polymerase','NULL','Also acts slowly with CTP.Catalyzes template-independent extension of the 3-end of a DNAstrand by one nucleotide at a time.Cannot initiate a chain de novo.The primer, depending on the source of the enzyme, may be an RNA orDNA fragment or oligo(A) bearing a 3-OH terminal group.See also EC 2.7.7.6.','ATP + RNA(n) = diphosphate + RNA(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.20','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.21','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.22','Mannose-1-phosphate guanylyltransferase (GDP)','GDP-mannose phosphorylase','NULL','NULL','GDP + alpha-D-mannose 1-phosphate = phosphate + GDP-mannose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.23','UDP-N-acetylglucosamine diphosphorylase','N-acetylglucosamine-1-phosphate uridyltransferase','NULL','Part of the pathway for acetamido sugar biosynthesis in bacteria andarchaea.The enzyme from several bacteria (e.g., Escherichia coli, Bacillussubtilis and Hemophilus influenzae) has been shown to be bifunctionaland also to possess the activity of EC 2.3.1.157.The enzyme from plants and animals is also active toward N-acetyl-alpha-D-galactosamine 1-phosphate (cf. EC 2.7.7.83), while thebacterial enzyme shows low activity toward that substrate.','UTP + N-acetyl-alpha-D-glucosamine 1-phosphate = diphosphate + UDP-N-acetyl-alpha-D-glucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.24','Glucose-1-phosphate thymidylyltransferase','dTDP-glucose diphosphorylase','NULL','NULL','dTTP + alpha-D-glucose 1-phosphate = diphosphate + dTDP-alpha-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.25','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.26','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.27','Glucose-1-phosphate adenylyltransferase','ADP-glucose diphosphorylase','NULL','NULL','ATP + alpha-D-glucose 1-phosphate = diphosphate + ADP-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.28','Nucleoside-triphosphate-aldose-1-phosphate nucleotidyltransferase','GDP hexose pyrophosphorylase','NULL','In decreasing order of activity, guanosine, inosine and adenosinediphosphate hexoses are substrates in the reverse reaction, witheither glucose or mannose as the sugar.Formerly EC 2.7.7.29.','Nucleoside triphosphate + alpha-D-aldose 1-phosphate = diphosphate + NDP-hexose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.29','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.30','Fucose-1-phosphate guanylyltransferase','GDP-fucose diphosphorylase','NULL','NULL','GTP + beta-L-fucose 1-phosphate = diphosphate + GDP-L-fucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.31','DNA nucleotidylexotransferase','Terminal addition enzyme','NULL','Catalyzes template-independent extension of the 3-end of a DNAstrand by one nucleotide at a time.Cannot initiate a chain de novo.Nucleoside may be ribo- or deoxyribo-.','Deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.32','Galactose-1-phosphate thymidylyltransferase','dTDP-galactose diphosphorylase','NULL','NULL','dTTP + alpha-D-galactose 1-phosphate = diphosphate + dTDP-galactose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.33','Glucose-1-phosphate cytidylyltransferase','CDP-glucose diphosphorylase','NULL','NULL','CTP + alpha-D-glucose 1-phosphate = diphosphate + CDP-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.34','Glucose-1-phosphate guanylyltransferase','GDP-glucose diphosphorylase','NULL','Also acts, more slowly, on D-mannose 1-phosphate.','GTP + alpha-D-glucose 1-phosphate = diphosphate + GDP-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.35','Ribose-5-phosphate adenylyltransferase','ADP-ribose phosphorylase','NULL','NULL','ADP + D-ribose 5-phosphate = phosphate + ADP-D-ribose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.36','Aldose-1-phosphate adenylyltransferase','ADP-aldose phosphorylase','NULL','NULL','ADP + alpha-D-aldose 1-phosphate = phosphate + ADP-aldose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.37','Aldose-1-phosphate nucleotidyltransferase','NDP-aldose phosphorylase','NULL','The enzyme works on a variety of alpha-D-aldose 1-phosphates andbeta-L-aldose 1-phosphates (which have the same anomericconfiguration as the former).','NDP + alpha-D-aldose 1-phosphate = phosphate + NDP-aldose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.38','3-deoxy-manno-octulosonate cytidylyltransferase','CMP-2-keto-3-deoxyoctulosonic acid synthetase','NULL','NULL','CTP + 3-deoxy-D-manno-octulosonate = diphosphate + CMP-3-deoxy-D-manno-octulosonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.39','Glycerol-3-phosphate cytidylyltransferase','CDP-glycerol diphosphorylase','NULL','NULL','CTP + sn-glycerol 3-phosphate = diphosphate + CDP-glycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.40','D-ribitol-5-phosphate cytidylyltransferase','CDP-ribitol diphosphorylase','NULL','NULL','CTP + D-ribitol 5-phosphate = diphosphate + CDP-ribitol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.41','Phosphatidate cytidylyltransferase','CDP-diacylglycerol synthase','NULL','NULL','CTP + phosphatidate = diphosphate + CDP-diacylglycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.42','[Glutamate--ammonia-ligase] adenylyltransferase','Glutamate-ammonia-ligase adenylyltransferase','NULL','NULL','ATP + [L-glutamate:ammonia ligase (ADP-forming)] = diphosphate +adenylyl-[L-glutamate:ammonia ligase (ADP-forming)].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.43','N-acylneuraminate cytidylyltransferase','CMP-N-acetylneuraminic acid synthetase','NULL','Acts on N-acetyl- and N-glycolyl- derivatives.','CTP + N-acylneuraminate = diphosphate + CMP-N-acylneuraminate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.44','Glucuronate-1-phosphate uridylyltransferase','NULL','NULL','Also acts slowly with CTP.','UTP + 1-phospho-alpha-D-glucuronate = diphosphate + UDP-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.45','Guanosine-triphosphate guanylyltransferase','NULL','NULL','Also acts, more slowly, on GDP to form P(1),P(3)-bis(5-guanosyl)triphosphate.','2 GTP = diphosphate + P(1),P(4)-bis(5-guanosyl) tetraphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.46','Gentamicin 2-nucleotidyltransferase','2-aminoglycoside nucleotidyltransferase','NULL','ATP, dATP, CTP, ITP and GTP can act as donors.Kanamycin, tobramycin and sisomicin can also act as acceptors.The nucleotidyl residue is transferred to the 2-hydroxy group of the3-amino-3-deoxy-D-glucose moiety in the antibiotic.','Nucleoside triphosphate + gentamicin = diphosphate +2-nucleotidylgentamicin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.47','Streptomycin 3-adenylyltransferase','NULL','NULL','Also acts on spectinomycin.','ATP + streptomycin = diphosphate + 3-adenylylstreptomycin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.48','RNA-directed RNA polymerase','RNA nucleotidyltransferase (RNA-directed)','NULL','Catalyzes RNA-template-directed extension of the 3-end of an RNAstrand by one nucleotide at a time.Can initiate a chain de novo.See also EC 2.7.7.6.','Nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.49','RNA-directed DNA polymerase','DNA nucleotidyltransferase (RNA-directed)','NULL','Catalyzes RNA-template-directed extension of the 3-end of a DNAstrand by one deoxynucleotide at a time.Cannot initiate a chain de novo.Requires a RNA or DNA primer.DNA can also serve as template.See also EC 2.7.7.7.','Deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.50','mRNA guanylyltransferase','GTP--RNA guanylyltransferase','NULL','Can modify also synthetic poly(A) and poly(G) to form the structuresm(7)G(5)pppAn and m(7)G(5)pppGn.','GTP + (5)pp-Pur-mRNA = diphosphate + G(5)ppp-Pur-mRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.51','Adenylylsulfate--ammonia adenylyltransferase','NULL','NULL','NULL','Adenylyl sulfate + NH(3) = adenosine 5-phosphoramidate + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.52','RNA uridylyltransferase','Terminal uridylyltransferase','NULL','Requires an oligoribonucleotide or polyribonucleotide with a freeterminal 3-OH as a primer.','UTP + RNA(n) = diphosphate + RNA(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.53','ATP adenylyltransferase','AP-4-A phosphorylase','NULL','GTP and adenosine tetraphosphate can also act as adenylyl acceptors.','ADP + ATP = phosphate + P(1),P(4)-bis(5-adenosyl) tetraphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.54','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.55','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.56','tRNA nucleotidyltransferase','Phosphate-dependent exonuclease','NULL','Brings about the final exonucleolytic trimming of the 3-terminus oftRNA precursors in Escherichia coli by a phosphorolysis, producing amature 3-terminus in tRNA and nucleoside diphosphate.Not identical with EC 2.7.7.8.','tRNA(n+1) + phosphate = tRNA(n) + a nucleoside diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.57','N-methylphosphoethanolamine cytidylyltransferase','NULL','NULL','NULL','CTP + N-methylethanolamine phosphate = diphosphate + CDP-N-methylethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.58','(2,3-dihydroxybenzoyl)adenylate synthase','2,3-dihydroxybenzoate-AMP ligase','NULL','NULL','ATP + 2,3-dihydroxybenzoate = diphosphate + (2,3-dihydroxybenzoyl)adenylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.59','[Protein-PII] uridylyltransferase','PII uridylyl-transferase','NULL','The enzyme uridylylates and de-uridylylates the small trimericprotein PII.','UTP + [protein-PII] = diphosphate + uridylyl-[protein-PII].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.60','2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase','4-diphosphocytidyl-2-C-methyl-D-erythritol synthase','Manganese or magnesium.','ATP or UTP can replace CTP, but both are less effective.GTP and TTP are not substrates.Forms part of an alternative nonmevalonate pathway for terpenoidbiosynthesis.','CTP + 2-C-methyl-D-erythritol 4-phosphate = diphosphate + 4-(cytidine5-diphospho)-2-C-methyl-D-erythritol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.61','Citrate lyase holo-[acyl-carrier-protein] synthase','2-(5-phosphoribosyl)-3-dephospho-CoA transferase','NULL','The gamma-subunit of EC 4.1.3.6 serves as an acyl-carrier protein(ACP) and contains the prosthetic group 2-(5-triphosphoribosyl)-3-dephospho-CoA.Synthesis and attachment of the prosthetic group requires theconcerted action of this enzyme and EC 2.4.2.52.In the enzyme from Escherichia coli, the prosthetic group is attachedto serine-14 of the ACP via a phosphodiester bond.','2-(5-triphosphoribosyl)-3-dephospho-CoA + citrate lyase apo-[acyl-carrier-protein] = citrate lyase holo-[acyl-carrier-protein] +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.62','Adenosylcobinamide-phosphate guanylyltransferase','Adenosylcobinamide kinase/adenosylcobinamide-phosphateguanylyltransferase','NULL','In Salmonella typhimurium LT2, under anaerobic conditions, CobU(EC 2.7.7.62 and EC 2.7.1.156), CobT (EC 2.4.2.21),CobC (EC 3.1.3.73) and CobS (EC 2.7.8.26) catalyze reactions in thenucleotide loop assembly pathway, which convert adenosylcobinamide(AdoCbi) into adenosylcobalamin (AdoCbl).CobT and CobC are involved in 5,6-dimethylbenzimidazole activationwhereby 5,6-dimethylbenzimidazole is converted to its riboside,alpha-ribazole.The second branch of the nuclotide loop assembly pathway is thecobinamide (Cbi) activation branch where AdoCbi oradenosylcobinamide-phosphate is converted to the activatedintermediate AdoCbi-GDP by the bifunctional enzyme CobU.The final step in adenosylcobalamin biosynthesis is the condensationof AdoCbi-GDP with alpha-ribazole, which is catalyzed by CobS(EC 2.7.8.26), to yield adenosylcobalamin.CobU is a bifunctional enzyme that has both kinase (EC 2.7.1.156) andguanylyltransferase (EC 2.7.7.62) activities.However, both activities are not required at all times.The kinase activity has been proposed to function only whenS.typhimurium is assimilating cobinamide whereas theguanylyltransferase activity is required for both assimilation ofexogenous cobinamide and for de novo synthesis of adenosylcobalamin.The guanylyltransferase reaction is a two-stage reaction withformation of a CobU-GMP intermediate.','GTP + adenosylcobinamide phosphate = diphosphate + adenosylcobinamide-GDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.63','Lipoate--protein ligase','Lipoate protein ligase','Magnesium.','Both 6S- and 6R-lipoates can act as substrates but there is apreference for the naturally occurring R-form.Selenolipoate, i.e. 5-(1,2-diselenolan-3-yl)pentanoic acid,and 6-sulfanyloctanoate can also act as substrates, but more slowly.Responsible for lipoylation in the presence of exogenous lipoic acid.Lipoylation is essential for the function of several key enzymesinvolved in oxidative metabolism, including pyruvate dehydrogenase(E(2) domain), 2-oxoglutarate dehydrogenase (E(2) domain),the branched-chain 2-oxoacid dehydrogenases and the glycine cleavagesystem (H protein).Attaches lipoic acid to the lipoyl domains of these proteins.It is likely that an alternative pathway, involving EC 2.3.1.181 andEC 2.8.1.8 is the normal route for lipoylation.','(1) ATP + lipoate = diphosphate + lipoyl-AMP.(2) Lipoyl-AMP + protein = protein N(6)-(lipoyl)lysine + AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.64','UTP-monosaccharide-1-phosphate uridylyltransferase','UDP-sugar pyrophosphorylase','Manganese or magnesium.','The reaction can occur in either direction and it has been postulatedthat MgUTP and Mg-diphosphate are the actual substrates.Catalyzes the formation of UDP-Glc, UDP-Gal, UDP-GlcA, UDP-L-Ara andUDP-Xyl, showing broad substrate specificity toward monosaccharide1-phosphates.Mannose 1-phosphate, L-fucose 1-phosphate and glucose 6-phosphate arenot substrates and UTP cannot be replaced by other nucleotidetriphosphates.','UTP + a monosaccharide 1-phosphate = diphosphate + UDP-monosaccharide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.65','Diguanylate cyclase','DGC','Magnesium; Manganese.','Can be activated by BeF3, a phosphoryl mimic, which results indimerization.Dimerization is required but is not sufficient for diguanylate-cyclase activity.Cyclic di-3,5-guanylate is an intracellular signaling molecule thatcontrols motility and adhesion in bacterial cells; it was firstidentified as having a positive allosteric effect on EC 2.4.1.12.','2 GTP = 2 diphosphate + cyclic di-3,5-guanylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.66','Malonate decarboxylase holo-[acyl-carrier-protein] synthase','Holo ACP synthase','NULL','The delta subunit of malonate decarboxylase serves as an an acyl-carrier protein (ACP) and contains the prosthetic group2-(5-triphosphoribosyl)-3-dephospho-CoA.Two reactions are involved in the production of the holo-ACP form ofthis enzyme.The first reaction is catalyzed by EC 2.4.2.52.The resulting prosthetic group is then attached to the ACP subunitvia a phosphodiester linkage to a serine residue, thus forming theholo form of the enzyme, in a manner analogous to that ofEC 2.7.7.61.','2-(5-triphosphoribosyl)-3-dephospho-CoA + malonate decarboxylase apo-[acyl-carrier-protein] = malonate decarboxylase holo-[acyl-carrier-protein] + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.67','CDP-archaeol synthase','NULL','Magnesium; Potassium.','Catalyzes one of the steps in the biosynthesis of polar lipids inarchaea, which are characterized by having an sn-glycero-1-phosphatebackbone rather than an sn-glycero-3-phosphate backbone as is foundin bacteria and eukaryotes.The other enzymes involved in the biosynthesis of polar lipids inarchaea are EC 1.1.1.261, EC 2.5.1.41 and EC 2.5.1.42, which togetheralkylate the hydroxy groups of glycerol 1-phosphate to giveunsaturated archaetidic acid, which is acted upon by this enzyme toform CDP-unsaturated archaeol.The final step in the pathway involves the addition of L-serine, withconcomitant removal of CMP, leading to the production of unsaturatedarchaetidylserine.Formerly EC 2.7.7.n2.','CTP + 2,3-bis-O-(geranylgeranyl)-sn-glycero-1-phosphate = diphosphate +CDP-2,3-bis-O-(geranylgeranyl)-sn-glycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.68','2-phospho-L-lactate guanylyltransferase','NULL','NULL','This enzyme is involved in the biosynthesis of coenzyme F420,a redox-active cofactor found in all methanogenic archaea, as well assome eubacteria.','(2S)-2-phospholactate + GTP = (2S)-lactyl-2-diphospho-5-guanosine +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.69','GDP-L-galactose phosphorylase','NULL','NULL','The enzyme catalyzes a reaction of the Smirnoff-Wheeler pathway,the major route to ascorbate biosynthesis in plants.','GDP-L-galactose + phosphate = alpha-L-galactose 1-phosphate + GDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.70','D-glycero-beta-D-manno-heptose 1-phosphate adenylyltransferase','D-glycero-D-manno-heptose-1-beta-phosphate adenylyltransferase','NULL','The bifunctional protein hldE includes D-glycero-beta-D-manno-heptose-7-phosphate kinase and D-glycero-beta-D-manno-heptose1-phosphate adenylyltransferase activity (cf. EC 2.7.1.167).The enzyme is involved in biosynthesis of ADP-L-glycero-beta-D-manno-heptose, which is utilized for assembly of the lipopolysaccharideinner core in Gram-negative bacteria.','D-glycero-beta-D-manno-heptose 1-phosphate + ATP = ADP-D-glycero-beta-D-manno-heptose + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.71','D-glycero-alpha-D-manno-heptose 1-phosphate guanylyltransferase','NULL','NULL','The enzyme is involved in biosynthesis of GDP-D-glycero-alpha-D-manno-heptose, which is required for assembly of S-layer glycoproteinin some Gram-positive bacteria.','D-glycero-alpha-D-manno-heptose 1-phosphate + GTP = GDP-D-glycero-alpha-D-manno-heptose + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.72','CCA tRNA nucleotidyltransferase','CCA-adding enzyme','NULL','The acylation of all tRNAs with an amino acid occurs at the terminalribose of a 3 CCA sequence.The CCA sequence is added to the tRNA precursor by stepwisenucleotide addition performed by a single enzyme that is ubiquitousin all living organisms.Although the enzyme has the option of releasing the product aftereach addition, it prefers to stay bound to the product and proceedwith the next addition.Formerly EC 2.7.7.21 and EC 2.7.7.25.','A tRNA precursor + 2 CTP + ATP = a tRNA with a 3 CCA end +3 diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.73','Sulfur carrier protein ThiS adenylyltransferase','NULL','Zinc.','The enzyme catalyzes the adenylation of ThiS, a sulfur carrierprotein involved in the biosynthesis of thiamine.The enzyme shows significant structural similarity to ubiquitin-activating enzyme.In Escherichia coli, but not in Bacillus subtilis, the enzyme forms across link from Cys-184 to the ThiS carboxy terminus (the positionthat is also thiolated) via an acyldisulfide.Formerly EC 2.7.7.n3.','ATP + [ThiS] = diphosphate + adenylyl-[ThiS].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.74','1L-myo-inositol 1-phosphate cytidylyltransferase','IPCT','NULL','In many organisms this activity is catalyzed by a bifunctionalenzyme.The cytidylyltransferase domain of the bifunctionalEC 2.7.7.74/EC 2.7.8.34 (CTP:inositol-1-phosphatecytidylyltransferase/CDP-inositol:inositol-1-phosphate transferase)is absolutely specific for CTP and 1L-myo-inositol 1-phosphate.The enzyme is involved in biosynthesis of bis(1L-myo-inositol)1,3-phosphate, a widespread organic solute in microorganisms adaptedto hot environments.In many organisms this activity is catalyzed by a bifunctionalenzyme.The cytidylyltransferase domain of the bifunctionalEC 2.7.7.74/EC 2.7.8.34 is absolutely specific for CTP and 1L-myo-inositol 1-phosphate.The enzyme is involved in biosynthesis of bis(1L-myo-inositol)1,3-phosphate, a widespread organic solute in microorganisms adaptedto hot environments.','CTP + 1L-myo-inositol 1-phosphate = diphosphate + CDP-1L-myo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.75','Molybdopterin adenylyltransferase','NULL','Manganese or magnesium.','Catalyzes the activation of molybdopterin for molybdenum insertion.In eukaryotes, this reaction is catalyzed by the C-terminal domain ofa fusion protein that also includes molybdopterin molybdotransferase(EC 2.10.1.1).The reaction requires a divalent cation such as Mg(2+) or Mn(2+).Catalyzes the activation of molybdopterin for molybdenum insertion.In eukaryotes, this reaction is catalyzed by the C-terminal domain ofa fusion protein that also includes EC 2.10.1.1.Formerly EC 2.7.7.n5.','ATP + molybdopterin = diphosphate + adenylyl-molybdopterin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.76','Molybdenum cofactor cytidylyltransferase','Mo-MPT cytidyltransferase','Manganese or magnesium.','Catalyzes the cytidylation of the molybdenum cofactor.This modification occurs only in prokaryotes.ATP or GTP cannot replace CTP.','CTP + molybdenum cofactor = diphosphate + cytidylyl molybdenum cofactor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.77','Molybdenum cofactor guanylyltransferase','MoCo guanylyltransferase','NULL','Catalyzes the guanylation of the molybdenum cofactor.This modification occurs only in prokaryotes.','GTP + molybdenum cofactor = diphosphate + guanylyl molybdenum cofactor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.78','GDP-D-glucose phosphorylase','NULL','NULL','The enzyme may be involved in prevention of misincorporation ofglucose in place of mannose residues into glycoconjugates i.e. toremove accidentally produced GDP-alpha-D-glucose.Activities with GDP-L-galactose, GDP-D-mannose and UDP-D-glucose areall less than 3% that with GDP-D-glucose.','GDP-alpha-D-glucose + phosphate = alpha-D-glucose 1-phosphate + GDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.79','tRNA(His) guanylyltransferase','Histidine tRNA guanylyltransferase','NULL','In eukarya an additional guanosine residue is added post-transcriptionally to the 5-end of tRNA(His) molecules.The addition occurs opposite a universally conserved adenosine(73)and is thus the result of a non-templated 3-5 addition reaction.The additional guanosine residue is an important determinant foraminoacylation by EC 6.1.1.21.The enzyme requires a divalent cation for activity.ATP activation is not required when the substrate contains a5-triphosphate (ppp-tRNA(His)).','p-tRNA(His) + ATP + GTP = pppG-P-tRNA(His) + AMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.80','Molybdopterin-synthase adenylyltransferase','NULL','NULL','Adenylates the C-terminus of the small subunit of the molybdopterinsynthase.This activation is required to form the thiocarboxylated C-terminusof the active molybdopterin synthase small subunit.The reaction occurs in prokaryotes and eukaryotes.In the human, the reaction is catalyzed by the N-terminal domain ofthe protein MOCS3, which also includes a molybdopterin-synthasesulfurtransferase (EC 2.8.1.11) C-terminal domain.Formerly EC 2.7.7.n4.','ATP + [molybdopterin-synthase sulfur-carrier protein]-Gly-Gly =diphosphate + [molybdopterin-synthase sulfur-carrier protein]-Gly-Gly-AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.81','Pseudaminic acid cytidylyltransferase','NULL','Magnesium.','NULL','CTP + 5,7-diacetamido-3,5,7,9-tetradeoxy-L-glycero-alpha-L-manno-2-nonulopyranosonic acid = diphosphate + CMP-5,7-diacetamido-3,5,7,9-tetradeoxy-L-glycero-alpha-L-manno-2-nonulopyranosonic acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.82','CMP-N,N-diacetyllegionaminic acid synthase','NULL','NULL','Isolated from the bacteria Legionella pneumophila and Campylobacterjejuni.Involved in biosynthesis of legionaminic acid, a sialic acid-likederivative that is incorporated into virulence-associated cellsurface glycoconjugates which may include lipopolysaccharide (LPS),capsular polysaccharide, pili and flagella.','CTP + N,N-diacetyllegionaminate = CMP-N,N-diacetyllegionaminate +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.83','UDP-N-acetylgalactosamine diphosphorylase','NULL','NULL','The enzyme from plants and animals also has activity toward N-acetyl-alpha-D-glucosamine 1-phosphate (cf. EC 2.7.7.23).','UTP + N-acetyl-alpha-D-galactosamine 1-phosphate = diphosphate + UDP-N-acetyl-alpha-D-galactosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.84','2-5 oligoadenylate synthase','NULL','NULL','The enzyme is activated by binding to double-stranded RNA.The resulting product binds to and activates RNase L, whichsubsequently degrades the RNA.Oligoadenylates of chain lengths 2, 4 and 5 are also produced.The dimer does not have any known biological activity.','3 ATP = pppA2p5A2p5A + 2 diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.85','Diadenylate cyclase','Cyclic-di-AMP synthase','NULL','Cyclic di-3,5-adenylate is a bioactive molecule produced by somebacteria and archaea, which may function as a secondary signalingmolecule.The intracellular bacterial pathogen Listeria monocytogenes secretesit into the hosts cytosol, where it triggers a cytosolic pathway ofinnate immunity.','2 ATP = 2 diphosphate + cyclic di-3,5-adenylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.86','Cyclic GMP-AMP synthase','NULL','NULL','Cyclic 3,5-AMP-GMP is a signaling molecule in mammalian cells thattriggers the production of type I interferons and other cytokines.','ATP + GTP = 2 diphosphate + cyclic 3,5-AMP-GMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.87','L-threonylcarbamoyladenylate synthase','NULL','NULL','The enzyme is involved in the synthesis of N(6)-threonylcarbamoyladenosine(37) in tRNAs with the anticodon NNU, i.e.tRNA(Ile), tRNA(Thr), tRNA(Asn), tRNA(Lys), tRNA(Ser) and tRNA(Arg).','L-threonine + ATP + bicarbonate = L-threonylcarbamoyladenylate +diphosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.n1','Adenosine monophosphate-protein transferase','NULL','NULL','NULL','ATP + [protein] = diphosphate + [protein]-AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.n2','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.n3','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.n4','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.7.n5','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.1','Ethanolaminephosphotransferase','NULL','NULL','NULL','CDP-ethanolamine + 1,2-diacyl-sn-glycerol = CMP +a phosphatidylethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.2','Diacylglycerol cholinephosphotransferase','1-alkyl-2-acetylglycerol cholinephosphotransferase','NULL','1-alkyl-2-acylglycerol can act as acceptor.Formerly EC 2.7.8.16.','CDP-choline + 1,2-diacyl-sn-glycerol = CMP + a phosphatidylcholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.3','Ceramide cholinephosphotransferase','NULL','NULL','NULL','CDP-choline + N-acylsphingosine = CMP + sphingomyelin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.4','Serine-phosphoethanolamine synthase','NULL','NULL','NULL','CDP-ethanolamine + L-serine = CMP + L-serine-phosphoethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.5','CDP-diacylglycerol--glycerol-3-phosphate 3-phosphatidyltransferase','3-phosphatidyl-1-glycerol-3-phosphate synthase','NULL','NULL','CDP-diacylglycerol + sn-glycerol 3-phosphate = CMP +3(3-sn-phosphatidyl)-sn-glycerol 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.6','Undecaprenyl-phosphate galactose phosphotransferase','Poly(isoprenol)-phosphate galactosephosphotransferase','NULL','NULL','UDP-alpha-D-galactose + undecaprenyl phosphate = UMP + alpha-D-galactosyl-diphosphoundecaprenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.7','Holo-[acyl-carrier-protein] synthase','4-phosphopantetheinyl transferase','Magnesium.','All polyketide synthases, fatty-acid synthases and non-ribosomalpeptide synthases require post-translational modification of theirconstituent acyl-carrier-protein (ACP) domains to becomecatalytically active.The inactive apo-proteins are converted into their active holo-formsby transfer of the 4-phosphopantetheinyl moiety of CoA to thesidechain hydroxy group of a conserved serine residue in each ACPdomain.The enzyme from human can activate both the ACP domain of the humancytosolic multifunctional fatty acid synthase and that associatedwith human mitochondria as well as peptidyl-carrier and acyl-carrier-proteins from prokaryotes.Removal of the 4-phosphopantetheinyl moiety from holo-ACP is carriedout by EC 3.1.4.14.','CoA-(4-phosphopantetheine) + apo-[acyl-carrier-protein] = adenosine3,5-bisphosphate + holo-[acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.8','CDP-diacylglycerol--serine O-phosphatidyltransferase','CDP-diglycerine-serine O-phosphatidyltransferase','NULL','NULL','CDP-diacylglycerol + L-serine = CMP + (3-sn-phosphatidyl)-L-serine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.9','Phosphomannan mannosephosphotransferase','NULL','NULL','NULL','GDP-mannose + (phosphomannan)(n) = GMP + (phosphomannan)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.10','Sphingosine cholinephosphotransferase','NULL','NULL','NULL','CDP-choline + sphingosine = CMP + sphingosyl-phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.11','CDP-diacylglycerol--inositol 3-phosphatidyltransferase','CDP-diacylglycerol--inositol phosphatidyltransferase','NULL','NULL','CDP-diacylglycerol + myo-inositol = CMP + phosphatidyl-1D-myo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.12','CDP-glycerol glycerophosphotransferase','Teichoic-acid synthase','NULL','NULL','CDP-glycerol + (glycerophosphate)(n) = CMP + (glycerophosphate)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.13','Phospho-N-acetylmuramoyl-pentapeptide-transferase','MraY transferase','NULL','In Gram-negative and some Gram-positive organisms the L-lysine isreplaced by meso-2,6-diaminoheptanedioate (meso-2,6-diaminopimelate,A2pm), which is combined with adjacent residues through its L-center.The undecaprenol involved is ditrans,octacis-undecaprenol.','UDP-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala) + undecaprenylphosphate = UMP + Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.14','CDP-ribitol ribitolphosphotransferase','Teichoic-acid synthase','NULL','Formerly EC 2.4.1.55.','CDP-ribitol + (ribitol phosphate)(n) = CMP + (ribitol phosphate)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.15','UDP-N-acetylglucosamine--dolichyl-phosphateN-acetylglucosaminephosphotransferase','GlcNAc-1-P transferase','NULL','NULL','UDP-N-acetyl-D-glucosamine + dolichyl phosphate = UMP + N-acetyl-D-glucosaminyl-diphosphodolichol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.16','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.17','UDP-N-acetylglucosamine--lysosomal-enzymeN-acetylglucosaminephosphotransferase','N-acetylglucosaminylphosphotransferase','NULL','Some other glycoproteins with high-mannose can act as acceptors,but much more slowly than lysosomal enzymes.','UDP-N-acetyl-D-glucosamine + lysosomal-enzyme D-mannose = UMP +lysosomal-enzyme N-acetyl-D-glucosaminyl-phospho-D-mannose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.18','UDP-galactose--UDP-N-acetylglucosamine galactose phosphotransferase','NULL','NULL','N-acetylglucosamine end-groups in glycoproteins can also act asacceptors.','UDP-alpha-D-galactose + UDP-N-acetyl-D-glucosamine = UMP + UDP-N-acetyl-6-(D-galactose-1-phospho)-D-glucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.19','UDP-glucose--glycoprotein glucose phosphotransferase','GlcPTase','NULL','Penultimate mannose residues on oligomannose type glycoproteins canact as acceptors.','UDP-glucose + glycoprotein D-mannose = UMP + glycoprotein 6-(D-glucose-1-phospho)-D-mannose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.20','Phosphatidylglycerol--membrane-oligosaccharide glycerophosphotransferase','Phosphoglycerol transferase','NULL','1,2-beta- and 1,6-beta-linked glucose residues in membranepolysaccharides and in synthetic glucosides can act as acceptors.','Phosphatidylglycerol + membrane-derived-oligosaccharide D-glucose = 1,2-diacyl-sn-glycerol + membrane-derived-oligosaccharide 6-(glycerophospho)-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.21','Membrane-oligosaccharide glycerophosphotransferase','Periplasmic phosphoglycerotransferase','NULL','Beta-linked glucose residues in simple glucosides, such asgentiobiose, can act as acceptors.In the presence of low concentrations of acceptor, free cyclic 1,2-phosphoglycerol is formed.','Transfer of a glycerophospho group from one membrane-derivedoligosaccharide to another.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.22','1-alkenyl-2-acylglycerol choline phosphotransferase','NULL','NULL','NULL','CDP-choline + 1-alkenyl-2-acylglycerol = CMP + plasmenylcholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.23','Carboxyvinyl-carboxyphosphonate phosphorylmutase','Carboxyphosphonoenolpyruvate phosphonomutase','NULL','Catalyzes the transfer and decarboxylation of thecarboxy(hydroxy)phosphoryl group, HOOC-P(O)(OH)- (phosphoryl being a3-valent group), in the formation of an unusual C-P bond that isinvolved in the biosynthesis of the antibiotic bialaphos.','1-carboxyvinyl carboxyphosphonate = 3-(hydrohydroxyphosphoryl)pyruvate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.24','Phosphatidylcholine synthase','CDP-diglyceride-choline O-phosphatidyltransferase','Manganese or magnesium.','NULL','CDP-diacylglycerol + choline = CMP + phosphatidylcholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.25','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.26','Adenosylcobinamide-GDP ribazoletransferase','Cobalamin (5-phosphate) synthase','NULL','In Salmonella typhimurium LT2, under anaerobic conditions, CobU(EC 2.7.7.62 and EC 2.7.1.156), CobT (EC 2.4.2.21),CobC (EC 3.1.3.73) and CobS (EC 2.7.8.26) catalyze reactions in thenucleotide loop assembly pathway, which convert adenosylcobinamide(AdoCbi) into adenosylcobalamin (AdoCbl).CobT and CobC are involved in 5,6-dimethylbenzimidazole activationwhereby 5,6-dimethylbenzimidazole is converted to its riboside,alpha-ribazole.The second branch of the nucleotide loop assembly pathway is thecobinamide (Cbi) activation branch where AdoCbi oradenosylcobinamide-phosphate is converted to the activatedintermediate AdoCbi-GDP by the bifunctional enzyme CobU.CobS catalyzes the final step in adenosylcobalamin biosynthesis,which is the condensation of AdoCbi-GDP with alpha-ribazole to yieldadenosylcobalamin.','(1) Adenosylcobinamide-GDP + alpha-ribazole = GMP + adenosylcobalamin.(2) Adenosylcobinamide-GDP + alpha-ribazole 5-phosphate = GMP +adenosylcobalamin 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.27','Sphingomyelin synthase','Phosphatidylcholine:ceramide cholinephosphotransferase','NULL','The reaction can occur in both directions.Occupies a central position in sphingolipid and glycerophospholipidmetabolism.Up- and down-regulation of its activity has been linked to mitogenicand pro-apoptotic signaling in a variety of mammalian cell types.Unlike EC 2.7.8.3 CDP-choline cannot replace phosphatidylcholine asthe donor of the phosphocholine moiety of sphingomyelin.','A ceramide + a phosphatidylcholine = a sphingomyelin + a 1,2-diacyl-sn-glycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.28','2-phospho-L-lactate transferase','NULL','NULL','This enzyme is involved in the biosynthesis of coenzyme F420,a redox-active cofactor found in all methanogenic archaea, as well assome eubacteria.','(2S)-lactyl-2-diphospho-5-guanosine + 7,8-didemethyl-8-hydroxy-5-deazariboflavin = guanosine 5-phosphate + coenzyme F420-0.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.29','L-serine-phosphatidylethanolamine phosphatidyltransferase','Phosphatidylserine synthase 2','NULL','This mammalian enzyme catalyzes an exchange reaction in which thepolar head group of phosphatidylethanolamine is replaced by L-serine.','L-1-phosphatidylethanolamine + L-serine = L-1-phosphatidylserine +ethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.30','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.31','Undecaprenyl-phosphate glucose phosphotransferase','Undecaprenylphosphate glucosylphosphate transferase','NULL','The enzyme is involved in biosynthesis of xanthan.','UDP-glucose + ditrans,octacis-undecaprenyl phosphate = UMP + alpha-D-glucopyranosyl-diphospho-ditrans,octacis-undecaprenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.32','3-O-alpha-D-mannopyranosyl-alpha-D-mannopyranosexylosylphosphotransferase','XPT1','Manganese.','The enzyme is specific for mannose as an acceptor but is flexible asto the structural context of the mannosyl disaccharide.','UDP-xylose + 3-O-alpha-D-mannopyranosyl-alpha-D-mannopyranose = UMP +3-O-(6-O-alpha-D-xylosylphospho-alpha-D-mannopyranosyl)-alpha-D-mannopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.33','UDP-N-acetylglucosamine--undecaprenyl-phosphateN-acetylglucosaminephosphotransferase','UDP-GlcNAc:undecaprenyl-phosphate GlcNAc-1-phosphate transferase','NULL','This enzyme catalyzes the synthesis of N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol, an essential lipidintermediate for the biosynthesis of various bacterial cell envelopecomponents.The enzyme also initiates the biosynthesis of enterobacterial commonantigen and O-antigen lipopolysaccharide in certain Escherichia colistrains, including K-12 and of teichoic acid in certain Gram-positivebacteria.Formerly EC 2.7.8.n2.','UDP-N-acetyl-alpha-D-glucosamine + ditrans,octacis-undecaprenyl phosphate= UMP + N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.34','CDP-L-myo-inositol myo-inositolphosphotransferase','DIPPS','NULL','In many organisms this activity is catalyzed by a bifunctionalenzyme.The di-myo-inositol-1,3-phosphate-1-phosphate synthase domain ofthe bifunctional EC 2.7.7.74/EC 2.7.8.34 uses only 1L-myo-inositol1-phosphate as an alcohol acceptor, but CDP-glycerol, as well as CDP-1L-myo-inositol and CDP-D-myo-inositol, are recognized as alcoholdonors.The enzyme is involved in biosynthesis of bis(1L-myo-inositol) 1,3-phosphate, a widespread organic solute in microorganisms adapted tohot environments.','CDP-1L-myo-inositol + 1L-myo-inositol 1-phosphate = CMP + bis(1L-myo-inositol) 3,1-phosphate 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.35','UDP-N-acetylglucosamine--decaprenyl-phosphateN-acetylglucosaminephosphotransferase','GlcNAc-1-phosphate transferase','NULL','Isolated from Mycobacterium tuberculosis and Mycobacterium smegmatis.This enzyme catalyzes the synthesis of monotrans,octacis-decaprenyl-N-acetyl-alpha-D-glucosaminyl diphosphate (mycobacterial lipid I),an essential lipid intermediate for the biosynthesis of variousbacterial cell envelope components.cf. EC 2.7.8.33.','UDP-N-acetyl-alpha-D-glucosamine + trans,octacis-decaprenyl phosphate =UMP + N-acetyl-alpha-D-glucosaminyl-diphospho-trans,octacis-decaprenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.36','Undecaprenyl phosphate N,N-diacetylbacillosamine 1-phosphatetransferase','NULL','NULL','Isolated from Campylobacter jejuni.Part of a bacterial N-linked glycosylation pathway.','UDP-N,N-diacetylbacillosamine + tritrans,heptacis-undecaprenyl phosphate= UMP + N,N-diacetyl-alpha-D-bacillosaminyl-diphospho-tritrans,heptacis-undecaprenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.37','Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase','NULL','NULL','Isolated from the bacterium Escherichia coli.','ATP + methylphosphonate = alpha-D-ribose 1-methylphosphonate5-triphosphate + adenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.38','Archaetidylserine synthase','NULL','Manganese.','Isolated from the archaeon Methanothermobacter thermautotrophicus.','(1) CDP-2,3-bis-(O-geranylgeranyl)-sn-glycerol + L-serine = CMP + 2,3-bis-(O-geranylgeranyl)-sn-glycero-1-phospho-L-serine.(2) CDP-2,3-bis-(O-phytanyl)-sn-glycerol + L-serine = CMP + 2,3-bis-(O-phytanyl)-sn-glycero-1-phospho-L-serine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.39','Archaetidylinositol phosphate synthase','AIP synthase','Manganese or magnesium.','The enzyme is involved in biosynthesis of archaetidyl-myo-inositol,a compound essential for glycolipid biosynthesis in archaea.','CDP-2,3-bis-(O-phytanyl)-sn-glycerol + 1L-myo-inositol 1-phosphate =CMP + 1-archaetidyl-1D-myo-inositol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.40','UDP-N-acetylgalactosamine-undecaprenyl-phosphateN-acetylgalactosaminephosphotransferase','NULL','NULL','The enzyme catalyzes a step in the assembly of the repeating-unit ofthe O-antigen of the Gram-negative bacterium Aeromonas hydrophilaAH-3.The enzyme shows no activity with UDP-N-acetyl-alpha-D-glucosamine(Cf. EC 2.7.8.33).','UDP-N-acetyl-alpha-D-galactosamine + ditrans,octacis-undecaprenylphosphate = UMP + N-acetyl-alpha-D-galactosaminyl-diphospho-ditrans,octacis-undecaprenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.8.n2','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.9.1','Pyruvate, phosphate dikinase','PPDK','NULL','NULL','ATP + pyruvate + phosphate = AMP + phosphoenolpyruvate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.9.2','Pyruvate, water dikinase','Phosphoenolpyruvate synthase','Manganese.','NULL','ATP + pyruvate + H(2)O = AMP + phosphoenolpyruvate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.9.3','Selenide, water dikinase','Selenide,water dikinase','Magnesium.','NULL','ATP + selenide + H(2)O = AMP + selenophosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.9.4','Alpha-glucan, water dikinase','Alpha-glucan,water dikinase','Magnesium.','ATP appears to be the only phosphate donor.No activity could be detected using GTP, UTP, phosphoenolpyruvate ordiphosphate.The protein phosphorylates glucans exclusively at O-6 of glucosylresidues.The protein phosphorylates itself with the beta-phosphate of ATP,which is then transferred to the glucan.','ATP + alpha-glucan + H(2)O = AMP + phospho-alpha-glucan + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.9.5','Phosphoglucan, water dikinase','NULL','NULL','The enzyme phosphorylates granular starch that has previously beenphosphorylated by EC 2.7.9.4; there is no activity withunphosphorylated glucans.It transfers the beta-phosphate of ATP to the phosphoglucan, whereasthe gamma-phosphate is transferred to water.In contrast to EC 2.7.9.4, which phosphorylates the glucose groups inglucans predominantly on O-6, this enzyme phosphorylates glucosegroups in phosphorylated starch predominantly on O-3.The protein phosphorylates itself with the beta-phosphate of ATP,which is then transferred to the glucan.','ATP + (phospho-alpha-glucan) + H(2)O = AMP + O-phospho-(phospho-alpha-glucan) + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.10.1','Receptor protein-tyrosine kinase','Receptor protein tyrosine kinase','NULL','The receptor protein-tyrosine kinases, which can be defined as havinga transmembrane domain, are a large and diverse multigene familyfound only in metazoans.In the human genome, 58 receptor-type protein-tyrosine kinases havebeen identified and these are distributed into 20 subfamilies.Formerly EC 2.7.1.112.','ATP + a [protein]-L-tyrosine = ADP + a [protein]-L-tyrosine phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.10.2','Non-specific protein-tyrosine kinase','Cytoplasmic protein tyrosine kinase','NULL','Unlike EC 2.7.10.1, this protein-tyrosine kinase does not have atransmembrane domain.In the human genome, 32 non-specific protein-tyrosine kinases havebeen identified and these can be divided into 10 families.Formerly EC 2.7.1.112.','ATP + a [protein]-L-tyrosine = ADP + a [protein]-L-tyrosine phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.1','Non-specific serine/threonine protein kinase','Protein phosphokinase','NULL','This is a heterogeneous group of serine/threonine protein kinasesthat do not have an activating compound and are either non-specificor their specificity has not been analyzed to date.Formerly EC 2.7.1.37 and EC 2.7.1.70.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.2','[Pyruvate dehydrogenase (acetyl-transferring)] kinase','PDH kinase','NULL','Has no activating compound but is specific for its substrate.A mitochondrial enzyme associated with the pyruvate dehydrogenasecomplex in mammals.Phosphorylation inactivates EC 1.2.4.1.Formerly EC 2.7.1.99.','ATP + [pyruvate dehydrogenase (acetyl-transferring)] = ADP + [pyruvatedehydrogenase (acetyl-transferring)] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.3','Dephospho-[reductase kinase] kinase','AMP-activated kinase','NULL','Activated by AMP and is specific for its substrate.Phosphorylates and activates EC 2.7.11.31 which has been inactivatedby EC 3.1.3.16.Formerly EC 2.7.1.110.','ATP + dephospho-[[hydroxymethylglutaryl-CoA reductase (NADPH)] kinase] =ADP + [[hydroxymethylglutaryl-CoA reductase (NADPH)] kinase].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.4','[3-methyl-2-oxobutanoate dehydrogenase (acetyl-transferring)] kinase','BCK','NULL','Has no activating compound but is specific for its substrate.A mitochondrial enzyme associated with the branched-chain 2-oxoaciddehydrogenase complex.Phosphorylation inactivates EC 1.2.4.4.Formerly EC 2.7.1.115.','ATP + [3-methyl-2-oxobutanoate dehydrogenase (acetyl-transferring)] =ADP + [3-methyl-2-oxobutanoate dehydrogenase (acetyl-transferring)]phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.5','[Isocitrate dehydrogenase (NADP(+))] kinase','ICDH kinase/phosphatase','NULL','Has no activating compound but is specific for its substrate.Phosphorylates and inactivates EC 1.1.1.42.Formerly EC 2.7.1.116.','ATP + [isocitrate dehydrogenase (NADP(+))] = ADP + [isocitratedehydrogenase (NADP(+))] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.6','[Tyrosine 3-monooxygenase] kinase','Pheochromocytoma tyrosine hydroxylase-associated kinase','Magnesium.','Has no activating compound but is specific for its substrate, withwhich it copurifies.Activates EC 1.14.16.2 by phosphorylation.Formerly EC 2.7.1.124.','ATP + [tyrosine-3-monooxygenase] = ADP + [tyrosine-3-monooxygenase]phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.7','[Myosin heavy-chain] kinase','Calmodulin-dependent myosin heavy chain kinase','NULL','The enzyme from Dictyostelium sp. (slime molds) brings aboutphosphorylation of the heavy chains of Dictyostelium myosin,inhibiting the actin-activated ATPase activity of the myosin.One threonine residue in each heavy chain acts as acceptor.While the enzyme from some species is activated by actin, in othercases Ca(2+)/calmodulin are required for activity.Formerly EC 2.7.1.129.','ATP + [myosin heavy-chain] = ADP + [myosin heavy-chain] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.8','Fas-activated serine/threonine kinase','FAST','NULL','This enzyme is activated during Fas-mediated apoptosis.Following Fas ligation, the enzyme, which is constitutivelyphosphorylated, is dephosphorylated, and it is the dephosphorylatedform that causes phosphorylation of TIA-1, a nuclear RNA-bindingprotein.Phosphorylation of TIA-1 precedes the onset of DNA fragmentation.Formerly EC 2.7.1.37.','ATP + [Fas-activated serine/threonine protein] = ADP + [Fas-activatedserine/threonine phosphoprotein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.9','[Goodpasture-antigen-binding protein] kinase','Goodpasture antigen-binding protein kinase','NULL','This serine/threonine kinase specifically binds to and phosphorylatesthe N-terminal region of the human Goodpasture antigen, which islocated on the alpha(3) chain of collagen IV and is involved inautoimmune disease.Formerly EC 2.7.1.37.','ATP + [Goodpasture antigen-binding protein] = ADP + [Goodpasture antigen-binding phosphoprotein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.10','I-kappa-B kinase','CHUK','NULL','Phosphorylates I-kappa-B proteins at specific serine residues, whichmarks them for destruction via the ubiquitination pathway.Subsequent degradation of the I-kappa-B complex (IKK) activatesNF-kappa-B, a translation factor that plays an important role ininflammation, immunity, cell proliferation and apoptosis.If the serine residues are replaced by threonine residues,the activity of the enzyme is decreased considerably.Formerly EC 2.7.1.37.','ATP + [I-kappa-B protein] = ADP + [I-kappa-B phosphoprotein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.11','cAMP-dependent protein kinase','PKA','NULL','cAMP is required to activate this enzyme.The inactive holoenzyme of cAMP-dependent protein kinase is atetramer composed of two regulatory (R) and two catalytic (C)subunits.cAMP causes the dissociation of the inactive holoenzyme into a dimerof regulatory subunits bound to four cAMP molecules and two freemonomeric catalytic subunits (i.e. R(2)C(2) + 4 cAMP = R(2)(cAMP)(4)+ 2 C).Formerly EC 2.7.1.37.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.12','cGMP-dependent protein kinase','3:5-cyclic GMP-dependent protein kinase','NULL','cGMP is required to activate this enzyme.The enzyme occurs as a dimer in higher eukaryotes.The C-terminal region of each polypeptide chain contains thecatalytic domain that includes the ATP and protein substrate bindingsites.This domain catalyzes the phosphorylation by ATP to specific serineor threonine residues in protein substrates.The enzyme also has two allosteric cGMP-binding sites (sites A andB).Binding of cGMP causes a conformational change that is associatedwith activation of the kinase.Formerly EC 2.7.1.37.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.13','Protein kinase C','Calcium-dependent protein kinase C','NULL','A family of serine- and threonine-specific protein kinases thatdepend on lipids for activity.They can be activated by calcium but have a requirement for thesecond messenger diacylglycerol.Members of this group of enzymes phosphorylate a wide variety ofprotein targets and are known to be involved in diverse cell-signaling pathways.Members of the protein kinase C family also serve as major receptorsfor phorbol esters, a class of tumor promoters.Formerly EC 2.7.1.37.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.14','Rhodopsin kinase','Cone opsin kinase','NULL','Acts on the bleached or activated form of rhodopsin.Also phosphorylates the beta-adrenergic receptor, but more slowly.Does not act on casein, histones or phosphovitin.Inhibited by Zn(2+) and digitonin (cf. EC 2.7.11.15 andEC 2.7.11.16).Formerly EC 2.7.1.97 and EC 2.7.1.125.','ATP + [rhodopsin] = ADP + [rhodopsin] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.15','[Beta-adrenergic-receptor] kinase','ADRBK1','NULL','Acts on the agonist-occupied form of the receptor; alsophosphorylates rhodopsin, but more slowly.Does not act on casein or histones.Inhibited by Zn(2+) and digitonin but is unaffected by cyclic-AMP(cf. EC 2.7.11.14 and EC 2.7.11.16).Formerly EC 2.7.1.126.','ATP + [beta-adrenergic receptor] = ADP + [beta-adrenergic receptor]phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.16','[G-protein-coupled receptor] kinase','G protein-coupled receptor kinase','NULL','All members of this enzyme subfamily possess a highly conservedbinding site for 1-phosphatidylinositol 4,5-bisphosphate.(cf. EC 2.7.11.14 and EC 2.7.11.15).','ATP + [G-protein-coupled receptor] = ADP + [G-protein-coupled receptor]phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.17','Calcium/calmodulin-dependent protein kinase','Ca(2+)/calmodulin-dependent microtubule-associated protein 2 kinase','Calcium.','Requires calmodulin.A wide range of proteins can act as acceptor, including vimentin,synapsin, glycogen synthase, myosin light-chains and the microtubule-associated tau protein.Not identical with EC EC 2.7.11.18 or EC 2.7.11.26.Formerly EC 2.7.1.120 and EC 2.7.1.123.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.18','[Myosin light-chain] kinase','[Myosin-light-chain] kinase','Calcium.','Requires calmodulin for activity.The 20 kDa light chain from smooth muscle myosin is phosphorylatedmore rapidly than any other acceptor, but light chains from othermyosins and myosin itself can act as acceptors, more slowly.Formerly EC 2.7.1.117.','ATP + [myosin light-chain] = ADP + [myosin light-chain] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.19','Phosphorylase kinase','Dephosphophosphorylase kinase','Calcium.','Requires calmodulin for activity.The enzyme phosphorylates a specific serine residue in each of thesubunits of the dimeric phosphorylase b.For muscle phosphorylase but not liver phosphorylase, this isaccompanied by a further dimerization to form a tetramericphosphorylase.The enzyme couples muscle contraction with energy production viaglycogenolysis--glycolysis by catalyzing the Ca(2+)-dependentphosphorylation and activation of glycogen phosphorylase b.The gamma subunit of the tetrameric alpha-beta-gamma-delta enzyme isthe catalytic subunit.Formerly EC 2.7.1.38.','2 ATP + phosphorylase b = 2 ADP + phosphorylase a.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.20','[Elongation factor 2] kinase','Ca/CaM-kinase III','Calcium.','Requires calmodulin for activity.Can also be phosphorylated by the catalytic subunit of EC 2.7.11.11.Elongation factor 2 is phosphorylated in several cell types inresponse to various growth factors, hormones and other stimuli thatraise intracellular Ca(2+).','ATP + [elongation factor 2] = ADP + [elongation factor 2] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.21','Polo kinase','PLK','NULL','The enzyme associates with the spindle pole during mitosis and isthought to play an important role in the dynamic function of themitotic spindle during chromosome segregation.The human form of the enzyme, Plk1, does not phosphorylate histoneH1, enolase and phosvitin but it can phosphorylate myelin basicprotein and microtubule-associated protein MAP-2, although to alesser extent than casein.Formerly EC 2.7.1.37.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.22','Cyclin-dependent kinase','Cdc2 kinase','NULL','Activation of cyclin-dependent kinases requires association of theenzyme with a regulatory subunit referred to as a cyclin.It is the sequential activation and inactivation of cyclin-dependentkinases, through the periodic synthesis and destruction of cyclins,that provides the primary means of cell-cycle regulation.Formerly EC 2.7.1.37.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.23','[RNA-polymerase]-subunit kinase','CTD kinase','NULL','Appears to be distinct from other protein phosphokinases.Brings about multiple phosphorylation of the unique C-terminal repeatdomain of the largest subunit of eukaryotic EC 2.7.7.6.Does not phosphorylate casein, phosvitin or histone.Formerly EC 2.7.1.141.','ATP + [DNA-directed RNA polymerase] = ADP + [DNA-directed RNA polymerase]phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.24','Mitogen-activated protein kinase','C-Jun N-terminal kinase','NULL','Phosphorylation of specific tyrosine and threonine residues in theactivation loop of this enzyme by EC 2.7.12.2 is necessary for enzymeactivation.Once activated, the enzyme phosphorylates target substrates on serineor threonine residues followed by a proline.A distinguishing feature of all MAPKs is the conserved sequence Thr-Xaa-Tyr (TXY).Mitogen-activated protein kinase (MAPK) signal transduction pathwaysare among the most widespread mechanisms of cellular regulation.Mammalian MAPK pathways can be recruited by a wide variety of stimuliincluding hormones (e.g. insulin and growth hormone), mitogens (e.g.epidermal growth factor and platelet-derived growth factor),vasoactive peptides (e.g. angiotensin-II and endothelin),inflammatory cytokines of the tumor necrosis factor (TNF) family andenvironmental stresses such as osmotic shock, ionizing radiation andischemeic injury.Formerly EC 2.7.1.37.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.25','Mitogen-activated protein kinase kinase kinase','MAP kinase kinase kinase','NULL','This enzyme phosphorylates and activates its downstream proteinkinase, EC 2.7.12.2, but requires MAPKKKK for activation.Some members of this family can be activated by p21-activated kinases(PAK/STE20) or Ras.While c-Raf and c-Mos activate the classical MAPK/ERK pathway, MEKK1and MEKK2 preferentially activate the c-Jun N-terminal proteinkinase(JNK)/stress-activated protein kinase (SAPK) pathway.Mitogen-activated protein kinase (MAPK) signal transduction pathwaysare among the most widespread mechanisms of cellular regulation.Mammalian MAPK pathways can be recruited by a wide variety of stimuliincluding hormones (e.g. insulin and growth hormone), mitogens (e.g.epidermal growth factor and platelet-derived growth factor),vasoactive peptides (e.g. angiotensin-II and endothelin),inflammatory cytokines of the tumor necrosis factor (TNF) family andenvironmental stresses such as osmotic shock, ionizing radiation andischemeic injury.Formerly EC 2.7.1.37.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.26','[Tau protein] kinase','[Tau-protein] kinase','NULL','Activated by tubulin.Involved in the formation of paired helical filaments, which are themain fibrous component of all fibrillary lesions in brain and areassociated with Alzheimers disease.Formerly EC 2.7.1.135.','ATP + [tau protein] = ADP + [tau protein] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.27','[Acetyl-CoA carboxylase] kinase','Acetyl coenzyme A carboxylase kinase (phosphorylating)','NULL','Phosphorylates and inactivates EC 6.4.1.2, which can bedephosphorylated and reactivated by EC 3.1.3.17.More active toward the dimeric form of acetyl-CoA carboxylase thanthe polymeric form.Phosphorylates serine residues.Formerly EC 2.7.1.111 and EC 2.7.1.128.','ATP + [acetyl-CoA carboxylase] = ADP + [acetyl-CoA carboxylase]phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.28','Tropomyosin kinase','Tropomyosin kinase (phosphorylating)','NULL','Phosphorylates casein equally well, and histone and phosvitin to alesser extent.Formerly EC 2.7.1.132.','ATP + [tropomyosin] = ADP + [tropomyosin] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.29','[Low-density-lipoprotein receptor] kinase','LDL receptor kinase','NULL','Phosphorylates the last serine residue (Ser-833) in the cytoplasmicdomain of the low-density lipoprotein receptor.Casein can also act as a substrate but with lower affinity.GTP can act instead of ATP.Formerly EC 2.7.1.131.','ATP + [low-density-lipoprotein receptor] = ADP + [low-density-lipoproteinreceptor] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.30','Receptor protein serine/threonine kinase','Activin receptor kinase','NULL','The transforming growth factor beta (TGF-beta) family of cytokinesregulates cell proliferation, differentiation, recognition and death.Signaling occurs by the binding of ligand to the type II receptor,which is the constitutively active kinase.Bound TGF-beta is then recognized by receptor I, which isphosphorylated and can propagate the signal to downstream substrates.Formerly EC 2.7.1.37.','ATP + [receptor-protein] = ADP + [receptor-protein] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.31','[Hydroxymethylglutaryl-CoA reductase (NADPH)] kinase','3-hydroxy-3-methylglutaryl coenzyme A reductase kinase','NULL','Activated by AMP.EC 1.1.1.34 is inactivated by the phosphorylation of the enzymeprotein.Histones can also act as acceptors.Can also phosphorylate EC 6.4.1.2 and EC 3.1.1.79.Thr-172 within the catalytic subunit (alpha-subunit) is the majorsite phosphorylated by the AMP-activated protein kinase kinase.GTP can act instead of ATP.Formerly EC 2.7.1.109.','ATP + [hydroxymethylglutaryl-CoA reductase (NADPH)] = ADP +[hydroxymethylglutaryl-CoA reductase (NADPH)] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.32','[Pyruvate, phosphate dikinase] kinase','Bifunctional dikinase regulatory protein','NULL','The enzyme from the plants Zea mays (maize) and Arabidopsis isbifunctional and catalyzes both the phosphorylation anddephosphorylation of pyruvate, phosphate dikinase (EC 2.7.9.1),cf. EC 2.7.4.27.The enzyme is specific for a reaction intermediate form ofEC 2.7.9.1, and phosphorylates an active site histidine.Formerly EC 2.7.11.n1.','ADP + [pyruvate, phosphate dikinase] = AMP + [pyruvate, phosphatedikinase] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.33','[Pyruvate, water dikinase] kinase','PEPS kinase','NULL','The enzyme from the bacterium Escherichia coli is bifunctional andcatalyzes both the phosphorylation and dephosphorylation ofEC 2.7.9.2; cf. EC 2.7.4.28.The enzyme is specific for a reaction intermediate form ofEC 2.7.9.2, where it phosphorylates an active site histidine.It has no activity toward EC 2.7.9.1, (cf. EC 2.7.11.32).Formerly EC 2.7.11.n2.','ADP + [pyruvate, water dikinase] = AMP + [pyruvate, water dikinase]phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.11.n2','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.12.1','Dual-specificity kinase','NULL','NULL','This family of enzymes can phosphorylate both Ser/Thr and Tyrresidues.Formerly EC 2.7.1.37.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.12.2','Mitogen-activated protein kinase kinase','MAP kinase kinase','NULL','A dual-specific protein kinase and requires mitogen-activated proteinkinase kinase kinase (MAPKKK) for activation.It is required for activation of EC 2.7.11.24.Phosphorylation of MEK1 by Raf involves phosphorylation of two serineresidues.Mitogen-activated protein kinase (MAPK) signal transduction pathwaysare among the most widespread mechanisms of cellular regulation.Mammalian MAPK pathways can be recruited by a wide variety of stimuliincluding hormones (e.g. insulin and growth hormone), mitogens (e.g.epidermal growth factor and platelet-derived growth factor),vasoactive peptides (e.g. angiotensin-II and endothelin),inflammatory cytokines of the tumor necrosis factor (TNF) family andenvironmental stresses such as osmotic shock, ionizing radiation andischemeic injury.','ATP + a protein = ADP + a phosphoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.13.1','Protein-histidine pros-kinase','Histidine kinase','NULL','A number of histones can act as acceptor.Formerly EC 2.7.3.11.','ATP + protein L-histidine = ADP + protein N(pi)-phospho-L-histidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.13.2','Protein-histidine tele-kinase','Histidine kinase','NULL','A number of histones can act as acceptor.Formerly EC 2.7.3.12.','ATP + protein L-histidine = ADP + protein N(tau)-phospho-L-histidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.13.3','Histidine kinase','Histidine protein kinase','NULL','This entry has been included to accommodate those protein-histidinekinases for which the phosphorylation site has not been established(i.e. either the pros- or tele-nitrogen of histidine).A number of histones can act as acceptor.','ATP + protein L-histidine = ADP + protein N-phospho-L-histidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.7.99.1','Triphosphate--protein phosphotransferase','Diphosphate--protein phosphotransferase','NULL','This enzyme was originally thought to use diphosphate as substratebut this has since been disproved.The activity is observed as the second part of a biphasic reactionafter depletion of ATP.Tripolyphosphate is a contaminant of (gamma-(32)P)ATP.Formerly EC 2.7.1.104.','Triphosphate + [microsomal-membrane protein] = diphosphate + [microsomal-membrane protein] phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.1','Thiosulfate sulfurtransferase','Rhodanese','NULL','A few other sulfur compounds can act as donors.','Thiosulfate + cyanide = sulfite + thiocyanate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.2','3-mercaptopyruvate sulfurtransferase','NULL','Zinc.','Sulfite, sulfinates, mercaptoethanol and mercaptopyruvate can alsoact as acceptors.','3-mercaptopyruvate + cyanide = pyruvate + thiocyanate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.3','Thiosulfate--thiol sulfurtransferase','Glutathione-dependent thiosulfate reductase','NULL','The primary product is glutathione hydrodisulfide, which reacts withglutathione to give glutathione disulfide and sulfide.L-cysteine can also act as acceptor.','Thiosulfate + 2 glutathione = sulfite + glutathione disulfide + sulfide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.4','tRNA sulfurtransferase','NULL','NULL','A group of enzymes transferring sulfur to various nucleotides in atRNA chain, producing residues such as 4-thiouridylate.With some enzymes mercaptopyruvate can act as sulfur donor.','L-cysteine + activated tRNA = L-serine + tRNA containing athionucleotide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.5','Thiosulfate--dithiol sulfurtransferase','Thiosulfate reductase','NULL','The enzyme from Chlorella shows very little activity towardmonothiols such as glutathione and cysteine (cf. EC 2.8.1.3).The enzyme probably transfers the sulfur atom onto one thiol group toform -S-S(-), and sulfide is spontaneously expelled from this byreaction with the other thiol group.May be identical with EC 2.8.1.1.','Thiosulfate + dithioerythritol = sulfite + 4,5-cis-dihydroxy-1,2-dithiacyclohexane + sulfide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.6','Biotin synthase','NULL','Iron-sulfur.','The enzyme binds a [4Fe-4S] and a [2Fe-2S] cluster.In every reaction cycle, the enzyme consumes two molecules of AdoMet,each producing 5-deoxyadenosine and a putative dethiobiotinyl carbonradical.Reaction with another equivalent of AdoMet results in abstraction ofthe C6 methylene pro-(S) hydrogen atom from 9-mercaptodethiobiotin,and the resulting carbon radical is quenched via formation of anintramolecular C-S bond, thus closing the biotin thiophane ring.The sulfur donor (S) is believed to be the [2Fe-2S] cluster, which issacrificed in the process, so that in vitro the reaction is a singleturnover.In vivo, the [2Fe-2S] cluster can be reassembled by the Isc or Sufiron-sulfur cluster assembly systems, to allow further catalysis.','Dethiobiotin + sulfur-(sulfur carrier) + 2 S-adenosyl-L-methionine =biotin + (sulfur carrier) + 2 L-methionine + 2 5-deoxyadenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.7','Cysteine desulfurase','Cysteine desulfurylase','Pyridoxal 5-phosphate.','The sulfur from free L-cysteine is first transferred to a cysteineresidue in the active site, and then passed on to various otheracceptors.The enzyme is involved in the biosynthesis of iron-sulfur clusters,thio-nucleosides in tRNA, thiamine, biotin, lipoate and pyranopterin(molybdopterin).In Azotobacter vinelandii, this sulfur provides the inorganic sulfiderequired for nitrogenous metallocluster formation.','L-cysteine + acceptor = L-alanine + S-sulfanyl-acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.8','Lipoyl synthase','Lipoate synthase','Iron-sulfur.','Member of the AdoMet radical (radical SAM) family, all members ofwhich produce the 5-deoxyadenosin-5-yl radical and methionine fromAdoMet (i.e. S-adenosylmethionine, or S-(5-deoxyadenosin-5-yl)methionine), by the addition of an electron from an iron-sulfurcenter.The radical is converted into 5-deoxyadenosine when it abstracts ahydrogen atom from C-6 and C-8, leaving reactive radicals at thesepositions so that they can add sulfur, with inversion ofconfiguration.Catalyzes the final step in the de-novo biosynthesis of the lipoylcofactor, with the other enzyme involved being EC 2.3.1.181.Lipoylation is essential for the function of several key enzymesinvolved in oxidative metabolism, as it converts apoprotein into thebiologically active holoprotein.Examples of such lipoylated proteins include pyruvate dehydrogenase(E(2) domain), 2-oxoglutarate dehydrogenase (E(2) domain),the branched-chain 2-oxoacid dehydrogenases and the glycine cleavagesystem.An alternative lipoylation pathway involves EC 2.7.7.63, which canlipoylate apoproteins using exogenous lipoic acid (or its analogs).','Protein N(6)-(octanoyl)lysine + 2 sulfur-(sulfur carrier) + 2 S-adenosyl-L-methionine = protein N(6)-(lipoyl)lysine + 2 (sulfur carrier) +2 L-methionine + 2 5-deoxyadenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.9','Molybdenum cofactor sulfurtransferase','MoCo sulfurase','Pyridoxal 5-phosphate.','Replaces the equatorial oxo ligand of the molybdenum by sulfur via anenzyme-bound persulfide.The reaction occurs in prokaryotes and eukaryotes but MoCosulfurtransferases are only found in eukaryotes.In prokaryotes the reaction is catalyzed by two enzymes: EC 2.8.1.7,which is homologous to the N-terminus of eukaryotic MoCosulfurtransferases, and a molybdo-enzyme specific chaperone whichbinds the MoCo and acts as an adapter protein.','Molybdenum cofactor + L-cysteine + 2 H(+) = thio-molybdenum cofactor +L-alanine + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.10','Thiazole synthase','NULL','NULL','H(2)S can provide the sulfur in vitro.Part of the pathway for thiamine biosynthesis.','1-deoxy-D-xylulose 5-phosphate + 2-iminoacetate + thiocarboxy-adenylate-[sulfur-carrier protein ThiS] = 2-((2R,5Z)-2-carboxy-4-methylthiazol-5(2H)-ylidene)ethyl phosphate + [sulfur-carrier protein ThiS] + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.11','Molybdopterin-synthase sulfurtransferase','Molybdopterin synthase sulfurylase','NULL','The enzyme transfers sulfur to form a thiocarboxylate moiety on theC-terminal glycine of the small subunit of EC 2.8.1.12.In the human, the reaction is catalyzed by the rhodanese-likeC-terminal domain (cf. EC 2.8.1.1) of the MOCS3 protein,a bifunctional protein that also contains EC 2.7.7.80 at theN-terminal domain.Formerly EC 2.8.1.n1.','[Molybdopterin-synthase sulfur-carrier protein]-Gly-Gly-AMP + [cysteinedesulfurase]-S-sulfanyl-L-cysteine = AMP + [molybdopterin-synthasesulfur-carrier protein]-Gly-NH-CH(2)-C(O)SH + cysteine desulfurase.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.12','Molybdopterin synthase','MPT synthase','NULL','Catalyzes the synthesis of molybdopterin from cyclic pyranopterinphosphate.Two sulfur atoms are transferred to cyclic pyranopterin phosphate inorder to form the characteristic ene-dithiol group found in themolybdenum cofactor.Molybdopterin synthase consists of two large subunits forming acentral dimer and two small subunits (molybdopterin-synthase sulfur-carrier proteins) that are thiocarboxylated at the C-terminus byEC 2.8.1.11, molybdopterin synthase sulfurtransferase.The reaction occurs in prokaryotes and eukaryotes.','Cyclic pyranopterin phosphate + 2 [molybdopterin-synthase sulfur-carrierprotein]-Gly-NH-CH(2)-C(O)SH + H(2)O = molybdopterin + 2 [molybdopterin-synthase sulfur-carrier protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.1.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.1','Aryl sulfotransferase','Phenol sulfotransferase','NULL','A number of aromatic compounds can act as acceptors.Organic hydroxylamines are not substrates (cf. EC 2.8.2.9).','3-phosphoadenylyl sulfate + a phenol = adenosine 3,5-bisphosphate +an aryl sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.2','Alcohol sulfotransferase','Hydroxysteroid sulfotransferase','NULL','Primary and secondary alcohols, including aliphatic alcohols,ascorbate, chloramphenicol, ephedrine and hydroxysteroids, but notphenolic steroids, can act as acceptors (cf. EC 2.8.2.15).','3-phosphoadenylyl sulfate + an alcohol = adenosine 3,5-bisphosphate +an alkyl sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.3','Amine sulfotransferase','Amine N-sulfotransferase','NULL','A large number of primary and secondary amines can act as acceptors,including aniline, 2-naphthylamine, cyclohexylamine and octylamine.','3-phosphoadenylyl sulfate + an amine = adenosine 3,5-bisphosphate +a sulfamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.4','Estrone sulfotransferase','NULL','NULL','NULL','3-phosphoadenylyl sulfate + estrone = adenosine 3,5-bisphosphate +estrone 3-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.5','Chondroitin 4-sulfotransferase','NULL','NULL','The sulfation is at the 4-position of N-acetylgalactosamine residuesof chondroitin.Not identical with EC 2.8.2.17.','3-phosphoadenylyl sulfate + chondroitin = adenosine 3,5-bisphosphate +chondroitin 4-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.6','Choline sulfotransferase','NULL','NULL','NULL','3-phosphoadenylyl sulfate + choline = adenosine 3,5-bisphosphate +choline sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.7','UDP-N-acetylgalactosamine-4-sulfate sulfotransferase','NULL','NULL','NULL','3-phosphoadenylyl sulfate + UDP-N-acetyl-D-galactosamine 4-sulfate =adenosine 3,5-bisphosphate + UDP-N-acetyl-D-galactosamine 4,6-bissulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.8','[Heparan sulfate]-glucosamine N-sulfotransferase','3-phosphoadenylyl-sulfate:heparitin N-sulfotransferase','NULL','The enzyme also catalyzes the sulfation of chondroitin 4-sulfate anddermatan sulfate, but to a much more limited extent.Formerly EC 2.8.2.12.','3-phosphoadenylyl sulfate + [heparan sulfate]-glucosamine = adenosine3,5-bisphosphate + [heparan sulfate]-N-sulfoglucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.9','Tyrosine-ester sulfotransferase','NULL','NULL','Phenols and organic hydroxylamines can act as acceptors (cf.EC 2.8.2.1).','3-phosphoadenylyl sulfate + L-tyrosine methyl ester = adenosine3,5-bisphosphate + L-tyrosine methyl ester 4-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.10','Renilla-luciferin sulfotransferase','Luciferin sulfotransferase','NULL','The product may be identical with Watasenia luciferin.','3-phosphoadenylyl sulfate + Renilla luciferin = adenosine3,5-bisphosphate + luciferyl sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.11','Galactosylceramide sulfotransferase','NULL','NULL','Also acts on lactosylceramide and monogalactosylalkylacylglycerol.','3-phosphoadenylyl sulfate + a galactosylceramide = adenosine3,5-bisphosphate + a galactosylceramidesulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.12','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.13','Psychosine sulfotransferase','NULL','NULL','NULL','3-phosphoadenylyl sulfate + galactosylsphingosine = adenosine3,5-bisphosphate + psychosine sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.14','Bile-salt sulfotransferase','BAST I','NULL','The formation of sulfate esters of bile acids is an essential step inthe prevention of toxicity by monohydroxy bile acids in many species.This enzyme is both a bile salt and a 3-hydroxysteroidsulfotransferase.In addition to the 5-beta-bile acid glycolithocholate, deoxycholate,3-beta-hydroxy-5-cholenoate and dehydroepiandrosterone (3-beta-hydroxyandrost-5-en-17-one) also act as substrates (see alsoEC 2.8.2.2 and EC 2.8.2.34).May be identical to EC 2.8.2.2.','(1) 3-phosphoadenylyl sulfate + glycolithocholate = adenosine3,5-bisphosphate + glycolithocholate 3-sulfate.(2) 3-phosphoadenylyl sulfate + taurolithocholate = adenosine3,5-bisphosphate + taurolithocholate sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.15','Steroid sulfotransferase','NULL','NULL','Broad specificity resembling EC 2.8.2.2, but also acts on estrone.','3-phosphoadenylyl sulfate + a phenolic steroid = adenosine3,5-bisphosphate + steroid O-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.16','Thiol sulfotransferase','NULL','NULL','Also acts on dithiols; substrates include glutathione,dithioerythritol and 2,3-mercaptopropanol.','3-phosphoadenylyl sulfate + a thiol = adenosine 3,5-bisphosphate +an S-alkyl thiosulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.17','Chondroitin 6-sulfotransferase','NULL','NULL','The sulfation is at the 6-position of N-acetylgalactosamine residuesof chondroitin.Not identical with EC 2.8.2.5.','3-phosphoadenylyl sulfate + chondroitin = adenosine 3,5-bisphosphate +chondroitin 6-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.18','Cortisol sulfotransferase','Glucocorticosteroid sulfotransferase','NULL','NULL','3-phosphoadenylyl sulfate + cortisol = adenosine 3,5-bisphosphate +cortisol 21-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.19','Triglucosylalkylacylglycerol sulfotransferase','NULL','NULL','NULL','3-phosphoadenylyl sulfate + alpha-D-glucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.20','Protein-tyrosine sulfotransferase','Tyrosylprotein sulfotransferase','NULL','The tyrosine residues of some specific proteins of ratpheochromocytoma cells act as acceptors.Formerly EC 2.8.2.n1.','3-phosphoadenylyl sulfate + protein tyrosine = adenosine3,5-bisphosphate + protein tyrosine-O-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.21','Keratan sulfotransferase','NULL','NULL','Sulfation takes place at the 6-position of galactosyl andN-acetylglucosaminyl residues in keratan, a proteoglycan.Not identical with EC 2.8.2.5, EC 2.8.2.6 or EC 2.8.2.17.','3-phosphoadenylyl sulfate + keratan = adenosine 3,5-bisphosphate +keratan 6-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.22','Aryl-sulfate sulfotransferase','Arylsulfate sulfotransferase','NULL','Hydroxy groups of tyrosine residues in peptides such as angiotensincan act as acceptors.Does not act on 3-phosphoadenylyl sulfate or adenosine3,5-bisphosphate.','An aryl sulfate + a phenol = a phenol + an aryl sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.23','[Heparan sulfate]-glucosamine 3-sulfotransferase 1','3-phosphoadenylyl-sulfate:heparin-glucosamine 3-O-sulfotransferase','NULL','This enzyme differs from EC 2.8.2.29 and EC 2.8.2.30 by being themost selective for a precursor of the antithrombin-binding site.It has a minimal acceptor sequence of: -','3-phosphoadenylyl sulfate + [heparan sulfate]-glucosamine = adenosine3,5-bisphosphate + [heparan sulfate]-glucosamine 3-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.24','Desulfoglucosinolate sulfotransferase','NULL','NULL','Involved with EC 2.4.1.195 in the biosynthesis of thioglycosides incruciferous plants.','3-phosphoadenylyl sulfate + desulfoglucotropeolin = adenosine3,5-bisphosphate + glucotropeolin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.25','Flavonol 3-sulfotransferase','NULL','NULL','Also acts on some other flavonol aglycones.','3-phosphoadenylyl sulfate + quercetin = adenosine 3,5-bisphosphate +quercetin 3-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.26','Quercetin-3-sulfate 3-sulfotransferase','NULL','NULL','NULL','3-phosphoadenylyl sulfate + quercetin 3-sulfate = adenosine3,5-bisphosphate + quercetin 3,3-bissulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.27','Quercetin-3-sulfate 4-sulfotransferase','NULL','NULL','NULL','3-phosphoadenylyl sulfate + quercetin 3-sulfate = adenosine3,5-bisphosphate + quercetin 3,4-bissulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.28','Quercetin-3,3-bissulfate 7-sulfotransferase','NULL','NULL','Quercetin 3,4-bissulfate can also act as acceptor.','3-phosphoadenylyl sulfate + quercetin 3,3-bissulfate = adenosine3,5-bisphosphate + quercetin 3,3,7-trissulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.29','[Heparan sulfate]-glucosamine 3-sulfotransferase 2','3-OST-2','NULL','This enzyme sulfates the residues marked with an asterisk insequences containing at least -','3-phosphoadenylyl sulfate + [heparan sulfate]-glucosamine = adenosine3,5-bisphosphate + [heparan sulfate]-glucosamine 3-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.30','[Heparan sulfate]-glucosamine 3-sulfotransferase 3','3-OST-3','NULL','Two major substrates contain the tetrasaccharides: -','3-phosphoadenylyl sulfate + [heparan sulfate]-glucosamine = adenosine3,5-bisphosphate + [heparan sulfate]-glucosamine 3-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.31','Petromyzonol sulfotransferase','PZ-SULT','NULL','The enzyme from the lamprey Petromyzon marinus can also use thecorresponding 3-ketone as a substrate.It is stereoselective (5-alpha-cholane) and regioselective,exhibiting a preference for an hydroxy group at C-24.The enzyme is inactive when allocholic acid, which has a carboxygroup at C-24, is used as a substrate.','3-phosphoadenylyl sulfate + 5-alpha-cholan-3-alpha,7-alpha,12-alpha,24-tetrol = adenosine 3,5-bisphosphate + 5-alpha-cholan-3-alpha,7-alpha,12-alpha-triol 24-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.32','Scymnol sulfotransferase','NULL','NULL','The enzyme from the shark Heterodontus portusjacksoni is able tosulfate the C(27) bile salts 5-beta-scymnol (the natural bile salt)and 5-alpha-cyprinol (the carp bile salt).Enzyme activity is activated by Mg(2+) but inhibited by the product5-beta-scymnol sulfate.','3-phosphoadenosine 5-phosphosulfate + 5-beta-scymnol = adenosine3,5-bisphosphate + 5-beta-scymnol sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.33','N-acetylgalactosamine 4-sulfate 6-O-sulfotransferase','GalNAc4S-6ST','NULL','The enzyme from human transfers sulfate to position 6 of bothinternal residues and non-reducing terminal GalNAc 4-sulfate residuesof chondroitin sulfate.Oligosaccharides derived from chondroitin sulfate also serve asacceptors but chondroitin sulfate E, keratan sulfate and heparansulfate do not.Differs from EC 2.8.2.17 in being able to use both chondroitin anddermatan as effective substrates.','(1) 3-phospho-5-adenylyl sulfate + [dermatan]-4-O-sulfo-N-acetylgalactosamine = adenosine 3,5-bisphosphate + [dermatan]-4,6-di-O-sulfo-N-acetyl-D-galactosamine.(2) 3-phospho-5-adenylyl sulfate + [chondroitin]-4-O-sulfo-N-acetylgalactosamine = adenosine 3,5-bisphosphate + [chondroitin]-4,6-di-O-sulfo-N-acetyl-D-galactosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.34','Glycochenodeoxycholate sulfotransferase','BAST','NULL','Specifically sulfates glycochenodeoxycholate at the 7-alpha-position(see also EC 2.8.2.14).The monohydroxy bile acids glycolithocholate, chenodeoxycholate andursodeoxycholate act as inhibitors.','3-phosphoadenylyl sulfate + glycochenodeoxycholate = adenosine3,5-bisphosphate + glycochenodeoxycholate 7-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.35','Dermatan 4-sulfotransferase','CHST14 protein','NULL','The sulfation takes place at the 4-position of N-acetylgalactosamineresidues of dermatan.D4ST-1 shows a strong preference in vitro for sulfate transfer toIdoUA-alpha(1,3)GalNAc-beta(1,4) that is flanked by GlcUA-beta(1,3)GalNAc-beta(1,4) as compared with IdoUA-alpha(1,3)GalNAc-beta(1,4) flanked by IdoUA-alpha(1,3)GalNAc-beta(1,4).','3-phospho-5-adenylyl sulfate + [dermatan]-N-acetyl-D-galactosamine =adenosine 3,5-bisphosphate + [dermatan]- 4-O-sulfo-N-acetyl-D-galactosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.36','Desulfo-A47934 sulfotransferase','NULL','NULL','The enzyme from the bacterium Streptomyces toyocaensis catalyzes thefinal step in the biosynthesis of the glycopeptide antibiotic A47934,a naturally occurring antibiotic of the vancomycin group.','3-phosphoadenylyl sulfate + desulfo-A47934 = adenosine3,5-bisphosphate + A47934.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.2.n2','Thyroxine sulfotransferase','Sulfotransferase 6B1','NULL','Found in mammals.The enzyme is involved in the metabolism of the hormone thyroxine.','3-phosphoadenylyl sulfate + thyroxine = adenosine 3,5-bisphosphate +thyroxine-sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.1','Propionate CoA-transferase','NULL','NULL','Butanoate and lactate can also act as acceptors.','Acetyl-CoA + propanoate = acetate + propanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.2','Oxalate CoA-transferase','Succinyl--beta-ketoacyl-CoA transferase','NULL','NULL','Succinyl-CoA + oxalate = succinate + oxalyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.3','Malonate CoA-transferase','NULL','NULL','The enzyme from Pseudomonas ovalis also catalyzes the reaction ofEC 4.1.1.9.','Acetyl-CoA + malonate = acetate + malonyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.4','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.5','3-oxoacid CoA-transferase','Succinyl-CoA:3-ketoacid-CoA transferase','NULL','Acetoacetate or, more slowly, 3-oxopropanoate, 3-oxopentanoate,3-oxo-4-methylpentanoate or 3-oxohexanoate can act as acceptor.Malonyl-CoA can act instead of succinyl-CoA.','Succinyl-CoA + a 3-oxo acid = succinate + a 3-oxoacyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.6','3-oxoadipate CoA-transferase','Beta-ketoadipate:succinyl-CoA transferase','NULL','NULL','Succinyl-CoA + 3-oxoadipate = succinate + 3-oxoadipyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.7','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.8','Acetate CoA-transferase','NULL','NULL','Acts on butanoyl-CoA and pentanoyl-CoA.','Acyl-CoA + acetate = a fatty acid anion + acetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.9','Butyrate--acetoacetate CoA-transferase','NULL','NULL','Butanoate, acetoacetate and their CoA thioesters are the preferredsubstrates, but the enzyme also acts, more slowly, on the derivativesof a number of C(2) to C(6) monocarboxylic acids.','Butanoyl-CoA + acetoacetate = butanoate + acetoacetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.10','Citrate CoA-transferase','NULL','NULL','Component of EC 4.1.3.6.Also catalyzes the transfer of thioacyl carrier protein from itsacetyl thioester to citrate.','Acetyl-CoA + citrate = acetate + (3S)-citryl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.11','Citramalate CoA-transferase','NULL','NULL','Component of EC 4.1.3.22.Also catalyzes the transfer of thioacyl carrier protein from itsacetyl thioester to citramalate.','Acetyl-CoA + citramalate = acetate + (3S)-citramalyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.12','Glutaconate CoA-transferase','NULL','NULL','Glutarate, (R)-2-hydroxyglutarate, propenoate and propanoate, but not(Z)-glutaconate, can also act as acceptors.','Acetyl-CoA + (E)-glutaconate = acetate + glutaconyl-1-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.13','Succinate--hydroxymethylglutarate CoA-transferase','NULL','NULL','Malonyl-CoA can also act as donor, more slowly.','Succinyl-CoA + 3-hydroxy-3-methylglutarate = succinate + (S)-3-hydroxy-3-methylglutaryl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.14','5-hydroxypentanoate CoA-transferase','5-hydroxyvalerate CoA-transferase','NULL','Propanoyl-CoA, acetyl-CoA, butanoyl-CoA and some other acyl-CoAs canact as substrates, more slowly than 5-hydroxypentanoyl-CoA.','Acetyl-CoA + 5-hydroxypentanoate = acetate + 5-hydroxypentanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.15','Succinyl-CoA:(R)-benzylsuccinate CoA-transferase','Benzylsuccinate CoA-transferase','NULL','Involved in anaerobic catabolism of toluene and is a strictlytoluene-induced enzyme that catalyzes the reversible regio- andenantio-selective synthesis of (R)-2-benzylsuccinyl-CoA.The enzyme from Thauera aromatica is inactive when (R)-benzylsuccinate is replaced by (S)-benzylsuccinate.','Succinyl-CoA + (R)-2-benzylsuccinate = succinate + (R)-2-benzylsuccinyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.16','Formyl-CoA transferase','Formyl-CoA oxalate CoA-transferase','NULL','The enzyme from Oxalobacter formigenes can also catalyze the transferof CoA from formyl-CoA to succinate.','Formyl-CoA + oxalate = formate + oxalyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.17','Cinnamoyl-CoA:phenyllactate CoA-transferase','FldA','NULL','3-phenylpropranoate is a better CoA acceptor than (R)-phenyllactatein vitro.The enzyme from Clostridium sporogenes is specific for derivatives of3-phenylpropanoate and 4-phenylbutyrate.','(E)-cinnamoyl-CoA + (R)-phenyllactate = (E)-cinnamate + (R)-phenyllactyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.18','Succinyl-CoA:acetate CoA-transferase','SCACT','NULL','In acetic acid bacteria the enzyme, which is highly specific,catalyzes the conversion of toxic acetate to acetyl-CoA.In the hydrogenosomes of some trichomonads the enzyme catalyzes theproduction of acetate.','Succinyl-CoA + acetate = acetyl-CoA + succinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.19','CoA:oxalate CoA-transferase','Acetyl-CoA oxalate CoA-transferase','NULL','The enzymes characterized from the bacteria Escherichia coli andAcetobacter aceti can also use formyl-CoA and oxalate (EC 2.8.3.16)or formyl-CoA and acetate, with significantly reduced specificactivities.','Acetyl-CoA + oxalate = acetate + oxalyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.20','Succinyl-CoA--D-citramalate CoA-transferase','NULL','NULL','The enzyme, purified from the bacterium Clostridium tetanomorphum,can also accept itaconate as acceptor, with lower efficiency.','(1) Succinyl-CoA + (R)-citramalate = succinate + (R)-citramalyl-CoA.(2) Succinyl-CoA + (R)-malate = succinate + (R)-malyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.21','L-carnitine CoA-transferase','NULL','NULL','The enzyme is found in gammaproteobacteria such as Proteus sp. andEscherichia coli.It has similar activity with both substrates.Formerly EC 4.2.1.89.','(1) (E)-4-(trimethylammonio)but-2-enoyl-CoA + L-carnitine = (E)-4-(trimethylammonio)but-2-enoate + L-carnitinyl-CoA.(2) 4-trimethylammoniobutanoyl-CoA + L-carnitine =4-trimethylammoniobutanoate + L-carnitinyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.3.22','Succinyl-CoA--L-malate CoA-transferase','NULL','NULL','The enzyme, purified from the bacterium Chloroflexus aurantiacus,can also accept itaconate as acceptor, with lower efficiency.It is part of the 3-hydroxypropanoate cycle for carbon assimilation.','(1) Succinyl-CoA + (S)-malate = succinate + (S)-malyl-CoA.(2) Succinyl-CoA + (S)-citramalate = succinate + (S)-citramalyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.4.1','Coenzyme-B sulfoethylthiotransferase','Methyl coenzyme M reductase','Coenzyme F430.','This enzyme catalyzes the final step in methanogenesis,the biological production of methane.This important anaerobic process is carried out only by methanogenicarchaea.The enzyme can also function in reverse, for anaerobic oxidation ofmethane.The enzyme requires the hydroporphinoid nickel complex coenzymeF(430).Highly specific for coenzyme B with a heptanoyl chain; ethyl CoM anddifluoromethyl CoM are poor substrates.The sulfide sulfur can be replaced by selenium but not by oxygen.','Methyl-CoM + CoB = CoM-S-S-CoB + methane.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.4.2','Arsenate-mycothiol transferase','NULL','NULL','Reduction of arsenate is part of a defense mechanism of the cellagainst toxic arsenate.The product arseno-mycothiol is reduced by EC 1.20.4.3 (mycoredoxin)to arsenite and mycothiol-mycoredoxin disulfide.Finally, a second mycothiol recycles mycoredoxin and formsmycothione.','Arsenate + mycothiol = arseno-mycothiol + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.4.3','tRNA-2-methylthio-N(6)-dimethylallyladenosine synthase','2-methylthio-N-6-isopentenyl adenosine synthase','Iron-sulfur.','This bacterial enzyme binds two [4Fe-4S] clusters as well as thetransferred sulfur.The enzyme is a member of the superfamily of S-adenosyl-L-methionine-dependent radical (radical AdoMet) enzymes.The sulfur donor is believed to be one of the [4Fe-4S] clusters,which is sacrificed in the process, so that in vitro the reaction isa single turnover.','N(6)-dimethylallyladenine(37) in tRNA + sulfur-(sulfur carrier) +2 S-adenosyl-L-methionine = 2-methylthio-N(6)-dimethylallyladenine(37) intRNA + S-adenosyl-L-homocysteine + (sulfur carrier) + L-methionine +5-deoxyadenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.4.4','[Ribosomal protein S12] (aspartate(89)-C(3))-methylthiotransferase','NULL','Iron-sulfur.','This bacterial enzyme binds two [4Fe-4S] clusters as well as thetransferred sulfur.The enzyme is a member of the superfamily of S-adenosyl-L-methionine-dependent radical (radical AdoMet) enzymes.The sulfur donor is believed to be one of the [4Fe-4S] clusters,which is sacrificed in the process, so that in vitro the reaction isa single turnover.','Asp(89)-[ribosomal protein S12] + sulfur-(sulfur carrier) + 2 S-adenosyl-L-methionine = 3-methylthioaspartate(89)-[ribosomal protein S12] +S-adenosyl-L-homocysteine + (sulfur carrier) + L-methionine +5-deoxyadenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.8.4.5','tRNA (N(6)-L-threonylcarbamoyladenosine(37)-C(2))-methylthiotransferase','Methylthio-threonylcarbamoyl-adenosine transferase B','Iron-sulfur.','The enzyme, which is a member of the S-adenosyl-L-methionine-dependent radical (radical AdoMet) enzymes superfamily, binds two[4Fe-4S] clusters as well as the transferred sulfur.The sulfur donor is believed to be one of the [4Fe-4S] clusters,which is sacrificed in the process, so that in vitro the reaction isa single turnover.','N(6)-L-threonylcarbamoyladenine(37) in tRNA + sulfur-(sulfur carrier) +2 S-adenosyl-L-methionine = 2-methylthio-N(6)-L-threonylcarbamoyladenine(37) in tRNA + S-adenosyl-L-homocysteine +(sulfur carrier) + L-methionine + 5-deoxyadenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.9.1.1','L-seryl-tRNA(Sec) selenium transferase','Cysteinyl-tRNA(Sec)-selenium transferase','Pyridoxal 5-phosphate.','Recognizes specifically tRNA(Sec)-species.Binding of tRNA(Sec) also occurs in the absence of the seryl group.2-aminoacryloyl-tRNA, bound to the enzyme as an imine with thepyridoxal phosphate, is an intermediate in the reaction.Since the selenium atom replaces oxygen in serine, the product mayalso be referred to as L-selenoseryl-tRNA(Sel).The abbreviation Sel has also been used for selenocysteine but Sec ispreferred.','L-seryl-tRNA(Sec) + selenophosphate = L-selenocysteinyl-tRNA(Sec) +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.9.1.2','O-phospho-L-seryl-tRNA(Sec):L-selenocysteinyl-tRNA synthase','O-phosphoseryl-tRNA(Sec) selenium transferase','Pyridoxal 5-phosphate.','In archaea and eukarya selenocysteine formation is achieved by a two-step process: EC 2.7.1.164 phosphorylates the endogenous L-seryl-tRNA(Sec) to O-phospho-L-seryl-tRNA(Sec), and then this misacylatedamino acid-tRNA species is converted to L-selenocysteinyl-tRNA(Sec)by Sep-tRNA:Sec-tRNA synthase.Formerly EC 2.9.1.n1.','O-phospho-L-seryl-tRNA(Sec) + selenophosphate = L-selenocysteinyl-tRNA(Sec) + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.9.1.n1','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 2.10.1.1','Molybdopterin molybdotransferase','NULL','Zinc or magnesium.','Catalyzes the insertion of molybdenum into the ene-dithiol group ofmolybdopterin.In eukaryotes this reaction is catalyzed by the N-terminal domain ofa fusion protein whose C-terminal domain catalyzes EC 2.7.7.75.','Adenylyl-molybdopterin + molybdate = molybdenum cofactor + AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.1','Carboxylesterase','Ali-esterase','NULL','Wide specificity; also hydrolyzes vitamin A esters.The enzymes from microsomes also catalyze the reactions ofEC 3.1.1.2, EC 3.1.1.5, EC 3.1.1.6, EC 3.1.1.23, EC 3.1.1.28,EC 3.1.2.2, EC 3.5.1.4, and EC 3.5.1.13.Formerly EC 3.1.1.12.','A carboxylic ester + H(2)O = an alcohol + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.2','Arylesterase','A-esterase','NULL','Acts on many phenolic esters.It is likely that the three forms of human paraoxonase are lactonasesrather than aromatic esterases.The natural substrates of the paraoxonases are lactones, with (+-)-5-hydroxy-6E,8Z,11Z,4Z-eicostetraenoic-acid 1,5-lactone being the bestsubstrate.','A phenyl acetate + H(2)O = a phenol + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.3','Triacylglycerol lipase','Lipase','NULL','The pancreatic enzyme acts only on an ester-water interface; theouter ester links are preferentially hydrolyzed.','Triacylglycerol + H(2)O = diacylglycerol + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.4','Phospholipase A(2)','Lecithinase A','Calcium.','Also acts on phosphatidylethanolamine, choline plasmalogen andphosphatides, removing the fatty acid attached to the 2-position.','Phosphatidylcholine + H(2)O = 1-acylglycerophosphocholine +a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.5','Lysophospholipase','Lecithinase B','NULL','NULL','2-lysophosphatidylcholine + H(2)O = glycerophosphocholine +a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.6','Acetylesterase','C-esterase (in animal tissues)','NULL','NULL','An acetic ester + H(2)O = an alcohol + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.7','Acetylcholinesterase','Choline esterase I','NULL','Acts on a variety of acetic esters.Also catalyzes transacetylations.','Acetylcholine + H(2)O = choline + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.8','Cholinesterase','Acylcholine acylhydrolase','NULL','Acts on a variety of choline esters and a few other compounds.','An acylcholine + H(2)O = choline + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.9','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.10','Tropinesterase','Atropine acylhydrolase','NULL','Also acts on cocaine and other tropine esters.','Atropine + H(2)O = tropine + tropate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.11','Pectinesterase','Pectin demethoxylase','NULL','NULL','Pectin + n H(2)O = n methanol + pectate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.12','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.13','Sterol esterase','Cholesterol ester synthase','NULL','A group of enzymes of broad specificity, acting on esters of sterolsand long-chain fatty acids, that may also bring about theesterification of sterols.Activated by bile salts.','A steryl ester + H(2)O = a sterol + a fatty acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.14','Chlorophyllase','NULL','NULL','Chlorophyllase has been found in higher plants, diatoms, and in thegreen algae Chlorella.Forms part of the chlorophyll degradation pathway and is thought totake part in de-greening processes such as fruit ripening, leafsenescence and flowering, as well as in the turnover and homeostasisof chlorophyll.Acts preferentially on chlorophyll a but will also accept chlorophyllb and pheophytins as substrates.Ethylene and methyl jasmonate, which are known to acceleratesenescence in many species, can enhance the activity of the hormone-inducible form of this enzyme.','Chlorophyll + H(2)O = phytol + chlorophyllide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.15','L-arabinonolactonase','NULL','NULL','NULL','L-arabinono-1,4-lactone + H(2)O = L-arabinonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.16','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.17','Gluconolactonase','Aldonolactonase','NULL','Acts on a wide range of hexono-1,5-lactones.Formerly EC 3.1.1.18.','D-glucono-1,5-lactone + H(2)O = D-gluconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.18','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.19','Uronolactonase','NULL','NULL','NULL','D-glucurono-6,2-lactone + H(2)O = D-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.20','Tannase','NULL','NULL','Also hydrolyzes ester links in other tannins.','Digallate + H(2)O = 2 gallate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.21','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.22','Hydroxybutyrate-dimer hydrolase','NULL','NULL','NULL','(R)-3-((R)-3-hydroxybutanoyloxy)butanoate + H(2)O = 2 (R)-3-hydroxybutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.23','Acylglycerol lipase','Monoacylglycerol lipase','NULL','NULL','Hydrolyzes glycerol monoesters of long-chain fatty acids.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.24','3-oxoadipate enol-lactonase','3-ketoadipate enol-lactonase','NULL','Acts on the product of EC 4.1.1.44.Formerly EC 3.1.1.16.','3-oxoadipate enol-lactone + H(2)O = 3-oxoadipate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.25','1,4-lactonase','Gamma-lactonase','Calcium.','Specific for 1,4-lactones with 4-8 carbon atoms.Does not hydrolyze simple aliphatic esters, acetylcholine, sugarlactones or substituted aliphatic lactones, e.g. 3-hydroxy-4-butyrolactone.','A 1,4-lactone + H(2)O = a 4-hydroxyacid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.26','Galactolipase','NULL','NULL','Also acts on 2,3-di-O-acyl-1-O-(6-O-alpha-D-galactosyl-beta-D-galactosyl)-D-glycerol, and phosphatidylcholine and otherphospholipids.','1,2-diacyl-3-beta-D-galactosyl-sn-glycerol + 2 H(2)O = 3-beta-D-galactosyl-sn-glycerol + 2 carboxylates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.27','4-pyridoxolactonase','NULL','NULL','NULL','4-pyridoxolactone + H(2)O = 4-pyridoxate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.28','Acylcarnitine hydrolase','NULL','NULL','Acts on higher fatty acid (C(6) to C(18)) esters of L-carnitine.Highest activity is with O-decanoyl-L-carnitine.','O-acylcarnitine + H(2)O = a fatty acid + L-carnitine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.29','Aminoacyl-tRNA hydrolase','Peptidyl-tRNA hydrolase','NULL','NULL','N-substituted aminoacyl-tRNA + H(2)O = N-substituted amino acid + tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.30','D-arabinonolactonase','NULL','NULL','NULL','D-arabinono-1,4-lactone + H(2)O = D-arabinonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.31','6-phosphogluconolactonase','NULL','NULL','NULL','6-phospho-D-glucono-1,5-lactone + H(2)O = 6-phospho-D-gluconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.32','Phospholipase A(1)','Phospholipase A1','Calcium.','Has a much broader specificity than EC 3.1.1.4.','Phosphatidylcholine + H(2)O = 2-acylglycerophosphocholine +a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.33','6-acetylglucose deacetylase','NULL','NULL','NULL','6-acetyl-D-glucose + H(2)O = D-glucose + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.34','Lipoprotein lipase','Clearing factor lipase','NULL','Hydrolyzes triacylglycerols in chylomicrons and very low-densitylipoproteins (VLDL).Also hydrolyzes diacylglycerol.','Triacylglycerol + H(2)O = diacylglycerol + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.35','Dihydrocoumarin hydrolase','NULL','NULL','Also hydrolyzes some other benzenoid 1,4-lactones.','Dihydrocoumarin + H(2)O = melilotate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.36','Limonin-D-ring-lactonase','NULL','NULL','Limonoate is a triterpenoid.','Limonoate D-ring-lactone + H(2)O = limonoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.37','Steroid-lactonase','NULL','NULL','NULL','Testololactone + H(2)O = testolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.38','Triacetate-lactonase','NULL','NULL','NULL','Triacetate lactone + H(2)O = triacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.39','Actinomycin lactonase','NULL','NULL','NULL','Actinomycin + H(2)O = actinomycinic monolactone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.40','Orsellinate-depside hydrolase','Lecanorate hydrolase','NULL','Will only hydrolyze those substrates based on the 2,4-dihydroxy-6-methylbenzoate structure that also have a free hydroxyl ortho to thedepside linkage.','Orsellinate depside + H(2)O = 2 orsellinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.41','Cephalosporin-C deacetylase','NULL','NULL','Hydrolyzes the acetyl ester bond on the 10-position of the antibioticcephalosporin C.','Cephalosporin C + H(2)O = deacetylcephalosporin C + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.42','Chlorogenate hydrolase','Chlorogenase','NULL','Also acts, more slowly, on isochlorogenate.No other substrates are known.','Chlorogenate + H(2)O = caffeate + quinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.43','Alpha-amino-acid esterase','Alpha-amino-acid ester hydrolase','NULL','Also catalyzes alpha-aminoacyl transfer to a number of aminenucleophiles.','An alpha-amino acid ester + H(2)O = an alpha-amino acid + an alcohol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.44','4-methyloxaloacetate esterase','NULL','NULL','NULL','Oxaloacetate 4-methyl ester + H(2)O = oxaloacetate + methanol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.45','Carboxymethylenebutenolidase','Dienelactone hydrolase','NULL','NULL','4-carboxymethylenebut-2-en-4-olide + H(2)O = 4-oxohex-2-enedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.46','Deoxylimonate A-ring-lactonase','NULL','NULL','Opens the A-ring-lactone of the triterpenoid deoxylimonic acid,leaving the D-ring-lactone intact.','Deoxylimonate + H(2)O = deoxylimononic acid D-ring-lactone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.47','1-alkyl-2-acetylglycerophosphocholine esterase','2-acetyl-1-alkylglycerophosphocholine esterase','NULL','NULL','1-alkyl-2-acetyl-sn-glycero-3-phosphocholine + H(2)O = 1-alkyl-sn-glycero-3-phosphocholine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.48','Fusarinine-C ornithinesterase','Ornithine esterase','NULL','Hydrolyzes the three ornithine ester bonds in fusarinine C.Also acts on N(5)-dinitrophenyl-L-ornithine methyl ester.','N(5)-acyl-L-ornithine ester + H(2)O = N(5)-acyl-L-ornithine + an alcohol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.49','Sinapine esterase','NULL','NULL','NULL','Sinapoylcholine + H(2)O = sinapate + choline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.50','Wax-ester hydrolase','NULL','NULL','Also acts on long-chain acylglycerol, but not diacyl- ortriacylglycerols.','A wax ester + H(2)O = a long-chain alcohol + a long-chain carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.51','Phorbol-diester hydrolase','Diacylphorbate 12-hydrolase','NULL','Hydrolyzes the 12-ester bond in a variety of 12,13-diacylphorbols(phorbol is a diterpenoid).This reaction inactivates tumor promoter 12-O-tetradecanoylphorbol-13-acetate from croton oil.','Phorbol 12,13-dibutanoate + H(2)O = phorbol 13-butanoate + butanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.52','Phosphatidylinositol deacylase','Phosphatidylinositol phospholipase A2','NULL','NULL','1-phosphatidyl-D-myo-inositol + H(2)O = 1-acylglycerophosphoinositol +a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.53','Sialate O-acetylesterase','NULL','NULL','Acts on free and glycosidically-bound N-acetyl- orN-glycoloylneuraminic acid.Acts mainly on 4-O- and 9-O-acetyl groups.Also acts on some other O-acetyl esters, both cyclic and acycliccompounds, which are not sialic acids.','N-acetyl-O-acetylneuraminate + H(2)O = N-acetylneuraminate + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.54','Acetoxybutynylbithiophene deacetylase','NULL','NULL','Highly specific.','5-(4-acetoxybut-1-ynyl)-2,2-bithiophene + H(2)O = 5-(4-hydroxybut-1-ynyl)-2,2-bithiophene + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.55','Acetylsalicylate deacetylase','Aspirin esterase','NULL','Not identical with EC 3.1.1.1, EC 3.1.1.2, EC 3.1.1.7 or EC 3.1.1.8.The activity of the liver cytosol enzyme is highest with acetylesters of aryl alcohols, and thioesters are also hydrolyzed.The microsomal enzyme also hydrolyzes some other negatively chargedesters.Best substrates were esters of salicylate with long-chain alcohols.','Acetylsalicylate + H(2)O = salicylate + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.56','Methylumbelliferyl-acetate deacetylase','Esterase D','NULL','Acts on short-chain acyl esters of 4-methylumbelliferone, but not onnaphthyl, indoxyl or thiocholine esters.','4-methylumbelliferyl acetate + H(2)O = 4-methylumbelliferone + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.57','2-pyrone-4,6-dicarboxylate lactonase','2-pyrone-4,6-dicarboxylate hydrolase','NULL','The product is most likely the keto-form of 4-oxalmesaconate (shown).It can be converted to the enol-form, 4-hydroxybuta-1,3-diene-1,2,4-trioate, either spontaneously or by EC 5.3.2.8.','2-oxo-2H-pyran-4,6-dicarboxylate + H(2)O = (1E)-4-oxobut-1-ene-1,2,4-tricarboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.58','N-acetylgalactosaminoglycan deacetylase','NULL','NULL','NULL','N-acetyl-D-galactosaminoglycan + H(2)O = D-galactosaminoglycan + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.59','Juvenile-hormone esterase','JH esterase','NULL','Demethylates the insect juvenile hormones, JH(1) and JH(3), but doesnot hydrolyze the analogous ethyl or isopropyl esters.','Methyl (2E,6E)-(10R,11S)-10,11-epoxy-3,7,11-trimethyltrideca-2,6-dienoate+ H(2)O = (2E,6E)-(10R,11S)-10,11-epoxy-3,7,11-trimethyltrideca-2,6-dienoate + methanol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.60','Bis(2-ethylhexyl)phthalate esterase','NULL','NULL','Also acts on 4-nitrophenyl esters, with optimum acyl chain lengthC(6) to C(8).','Bis(2-ethylhexyl)phthalate + H(2)O = 2-ethylhexyl phthalate +2-ethylhexan-1-ol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.61','Protein-glutamate methylesterase','CheB methylesterase','NULL','Hydrolyzes the products of EC 2.1.1.77, EC 2.1.1.78, EC 2.1.1.80 andEC 2.1.1.100.','Protein L-glutamate O(5)-methyl ester + H(2)O = protein L-glutamate +methanol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.62','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.63','11-cis-retinyl-palmitate hydrolase','NULL','NULL','Activated by bile salts.','11-cis-retinyl palmitate + H(2)O = 11-cis-retinol + palmitate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.64','Retinoid isomerohydrolase','All-trans-retinyl-palmitate hydrolase','NULL','This enzyme, which operates in the retinal pigment epithelium (RPE),catalyzes the cleavage and isomerization of all-trans-retinyl fattyacid esters to 11-cis-retinol, a key step in the regeneration of thevisual chromophore in the vertebrate visual cycle.Interaction of the enzyme with the membrane is critical for itsenzymatic activity.Formerly EC 5.2.1.7.','An all-trans-retinyl ester + H(2)O = 11-cis-retinol + a fatty acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.65','L-rhamnono-1,4-lactonase','NULL','NULL','NULL','L-rhamnono-1,4-lactone + H(2)O = L-rhamnonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.66','5-(3,4-diacetoxybut-1-ynyl)-2,2-bithiophene deacetylase','NULL','NULL','A highly specific enzyme from Tagetes patula.','5-(3,4-diacetoxybut-1-ynyl)-2,2-bithiophene + H(2)O = 5-(3-hydroxy-4-acetoxybut-1-ynyl)-2,2-bithiophene + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.67','Fatty-acyl-ethyl-ester synthase','FAEE synthase','NULL','In the reverse reaction, forms ethyl esters from fatty acids andethanol in the absence of coenzyme A or ATP.Best substrates are unsaturated octadecanoic acids; palmitate,stearate and arachidonate also act, more slowly.','A long-chain-fatty-acyl ethyl ester + H(2)O = a long-chain-fatty acid +ethanol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.68','Xylono-1,4-lactonase','NULL','NULL','NULL','D-xylono-1,4-lactone + H(2)O = D-xylonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.69','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.70','Cetraxate benzylesterase','NULL','NULL','Acts on a number of benzyl esters of substituted phenyl propanoates,and on the benzyl esters of phenylalanine and tyrosine.','Cetraxate benzyl ester + H(2)O = cetraxate + benzyl alcohol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.71','Acetylalkylglycerol acetylhydrolase','Alkylacetylglycerol acetylhydrolase','NULL','Hydrolysis of the acetyl group from the 1-alkyl-2-acetyl and 1-alkyl-3-acetyl substrates occurs at apparently identical rates.Differs from EC 3.1.1.34 since 1,2-diacetyl-sn-glycerols are notsubstrates.It also differs from EC 3.1.1.47.','2-acetyl-1-alkyl-sn-glycerol + H(2)O = 1-alkyl-sn-glycerol + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.72','Acetylxylan esterase','NULL','NULL','Catalyzes the hydrolysis of acetyl groups from polymeric xylan,acetylated xylose, acetylated glucose, alpha-napthyl acetate,p-nitrophenyl acetate but not from triacetylglycerol.Does not act on acetylated mannan or pectin.','Deacetylation of xylans and xylo-oligosaccharides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.73','Feruloyl esterase','Cinnamoyl ester hydrolase','NULL','Catalyzes the hydrolysis of the 4-hydroxy-3-methoxycinnamoyl(feruloyl) group from an esterified sugar, which is usually arabinosein natural substrates.p-nitrophenol acetate and methyl ferulate are poorer substrates.All microbial ferulate esterases are secreted into the culturemedium.They are sometimes called hemicellulase accessory enzymes, since theyhelp xylanases and pectinases to break down plant cell wallhemicellulose.','Feruloyl-polysaccharide + H(2)O = ferulate + polysaccharide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.74','Cutinase','Cutin hydrolase','NULL','Cutin, a polymeric structural component of plant cuticles, is anhydroxy fatty acid polymer, usually C(16) or C(18) and that containsone to three hydroxyl groups.The enzyme from several fungal sources also hydrolyzes thep-nitrophenyl esters of hexadecanoic acid.It is however inactive toward several esters that are substrates fornon-specific esterases.','Cutin + H(2)O = cutin monomers.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.75','Poly(3-hydroxybutyrate) depolymerase','PHB depolymerase','NULL','Reaction also occurs with esters of other short-chain-length (C(1)-C(5)) hydroxyalkanoic acids (HA).There are two types of polymers: native (intracellular) granules areamorphous and have an intact surface layer; denatured (extracellular)granules either have no surface layer or a damaged surface layer andare partially crystalline.','((R)-3-hydroxybutanoate)(n) + H(2)O = ((R)-3-hydroxybutanoate)(n-1 to 5)+ ((R)-3-hydroxybutanoate)(1-5).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.76','Poly(3-hydroxyoctanoate) depolymerase','PHA depolymerase','NULL','The main product after prolonged incubation is the dimer.Besides hydrolyzing polymers of 3-hydroxyoctanoic acid, the enzymealso hydrolyzes other polymers derived from medium-chain-length(C(6)-C(12)) hydroxyalkanoic acids and copolymers of mixtures ofthese.It also hydrolyzes p-nitrophenyl esters of fatty acids.Polymers of short-chain-length hydroxyalkanoic acids such aspoly((R)-3-hydroxybutanoic acid) and poly((R)-3-hydroxypentanoicacid) are not hydrolyzed.','Hydrolyzes the polyester poly(oxycarbonyl((R)-2-pentylethylene)) tooligomers.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.77','Acyloxyacyl hydrolase','NULL','NULL','The substrate is lipid A on the reducing end of the toxiclipopolysaccharide (LPS) of Salmonella typhimurium and relatedorganisms.It consists of diglucosamine, beta-D-GlcN-(1-','3-(acyloxy)acyl group of bacterial toxin = 3-hydroxyacyl group ofbacterial toxin + a fatty acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.78','Polyneuridine-aldehyde esterase','PNAE','NULL','Following hydrolysis of this indole alkaloid ester, the carboxylicacid decarboxylates spontaneously giving the sarpagan skeleton.Also acts on akuammidine aldehyde (the 16-epimer of polyneuridinealdehyde).','Polyneuridine aldehyde + H(2)O = 16-epivellosimine + CO(2) + methanol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.79','Hormone-sensitive lipase','HSL','NULL','This enzyme is a serine hydrolase.Compared with other lipases, hormone-sensitive lipase has a uniquelybroad substrate specificity.It hydrolyzes all acylglycerols (triacylglycerol, diacylglycerol andmonoacylglycerol) as well as cholesteryl esters, steroid fatty acidesters, retinyl esters and p-nitrophenyl esters.It exhibits a preference for the 1- or 3-ester bond of itsacylglycerol substrate compared with the 2-ester bond.The enzyme shows little preference for the fatty acids in thetriacylglycerol, although there is some increase in activity withdecreasing chain length.The enzyme activity is increased in response to hormones that elevateintracellular levels of cAMP.','(1) Diacylglycerol + H(2)O = monoacylglycerol + a carboxylate.(2) Triacylglycerol + H(2)O = diacylglycerol + a carboxylate.(3) Monoacylglycerol + H(2)O = glycerol + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.80','Acetylajmaline esterase','2-beta-(R)-17-O-acetylajmalan:acetylesterase','NULL','Responsible for the last stages in the biosynthesis of the indolealkaloid ajmaline.Highly specific for the substrates 17-O-acetylajmaline and 17-O-acetylnorajmaline as the structurally related acetylated alkaloidsvinorine, vomilenine, 1,2-dihydrovomilenine and 1,2-dihydroraucaffricine cannot act as substrates.','(1) 17-O-acetylajmaline + H(2)O = ajmaline + acetate.(2) 17-O-acetylnorajmaline + H(2)O = norajmaline + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.81','Quorum-quenching N-acyl-homoserine lactonase','Acyl homoserine degrading enzyme','NULL','Acyl-homoserine lactones (AHLs) are produced by a number of bacterialspecies and are used by them to regulate the expression of virulencegenes in a process known as quorum-sensing.Each bacterial cell has a basal level of AHL and, once the populationdensity reaches a critical level, it triggers AHL-signaling which,in turn, initiates the expression of particular virulence genes.Plants or animals capable of degrading AHLs would have a therapeuticadvantage in avoiding bacterial infection as they could prevent AHL-signaling and the expression of virulence genes in quorum-sensingbacteria.N-(3-oxohexanoyl)-L-homoserine lactone, N-(3-oxododecanoyl)-L-homoserine lactone, N-butanoyl-L-homoserine lactone andN-(3-oxooctanoyl)-L-homoserine lactone can act as substrates.','An N-acyl-L-homoserine lactone + H(2)O = an N-acyl-L-homoserine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.82','Pheophorbidase','Phedase','NULL','Forms part of the chlorophyll degradation pathway, and is found inhigher plants and in algae.In higher plants it participates in de-greening processes such asfruit ripening, leaf senescence, and flowering.Exists in two forms: type 1 is induced by senescence whereas type 2is constitutively expressed.Highly specific for pheophorbide as substrate (with a preference forpheophorbide a over pheophorbide b) as other chlorophyll derivativessuch as protochlorophyllide a, pheophytin a and c, chlorophyll a andb, and chlorophyllide a cannot act as substrates.Another enzyme, called pheophorbide demethoxycarbonylase (PDC),produces pyropheophorbide a from pheophorbide a without forming anintermediate although the precise reaction is not yet known.','Pheophorbide a + H(2)O = pyropheophorbide a + methanol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.83','Monoterpene epsilon-lactone hydrolase','MLH','NULL','Catalyzes the ring opening of epsilon-lactones which are formedduring degradation of dihydrocarveol by the Gram-positive bacteriumRhodococcus erythropolis DCL14.Also acts on ethyl caproate, indicating that it is an esterase with apreference for lactones (internal cyclic esters).Not stereoselective.','Isoprop(en)ylmethyloxepan-2-one + H(2)O =6-hydroxyisoprop(en)ylmethylhexanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.84','Cocaine esterase','Human carboxylesterase 2','NULL','Rhodococcus sp. strain MB1 and Pseudomonas maltophilia strain MB11Lcan utilize cocaine as sole source of carbon and energy.','Cocaine + H(2)O = ecgonine methyl ester + benzoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.85','Pimeloyl-[acyl-carrier protein] methyl ester esterase','NULL','NULL','Involved in biotin biosynthesis in Gram-negative bacteria.The enzyme exhibits carboxylesterase activity, particularly towardsubstrates with short acyl chains.Even though the enzyme can interact with coenzyme A thioesters,the in vivo role of the enzyme is to hydrolyze the methyl ester ofpimeloyl-[acyl carrier protein], terminating the part of the biotinbiosynthesis pathway that is catalyzed by the fatty acid elongationenzymes.','Pimeloyl-[acyl-carrier protein] methyl ester + H(2)O = pimeloyl-[acyl-carrier protein] + methanol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.86','Rhamnogalacturonan acetylesterase','RGAE','NULL','The degradation of rhamnogalacturonan by rhamnogalacturonases dependson the removal of the acetyl esters from the substrate.','Hydrolytic cleavage of 2-O-acetyl- or 3-O-acetyl groups of alpha-D-galacturonic acid in rhamnogalacturonan I.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.87','Fumonisin B1 esterase','NULL','NULL','The enzyme is involved in degradation of fumonisin B1.','Fumonisin B1 + 2 H(2)O = aminopentol + 2 propane-1,2,3-tricarboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.88','Pyrethroid hydrolase','Permethrinase','NULL','The enzyme is involved in degradation of pyrethroid pesticides.The enzymes from Sphingobium sp., Klebsiella sp. and Aspergillusniger hydrolyze cis-permethrin at approximately equal rate to trans-permethrin.The enzyme from mouse hydrolyzes trans-permethrin at a rate about22-fold higher than cis-permethrin.','Trans-permethrin + H(2)O = (3-phenoxyphenyl)methanol + (1S,3R)-3-(2,2-dichloroethenyl)-2,2-dimethylcyclopropanecarboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.89','Protein phosphatase methylesterase-1','NULL','NULL','A key regulator of protein phosphatase 2A.The methyl ester is formed by EC 2.1.1.233.Occurs mainly in the nucleus.','[Phosphatase 2A protein]-leucine methyl ester + H(2)O = [phosphatase 2Aprotein]-leucine + methanol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.90','All-trans-retinyl ester 13-cis isomerohydrolase','NULL','NULL','All-trans-retinyl esters, which are a storage form of vitamin A,are generated by the activity of EC 2.3.1.135.They can be hydrolyzed to 11-cis-retinol by EC 3.1.1.64, or to13-cis-retinol by this enzyme.','An all-trans-retinyl ester + H(2)O = 13-cis-retinol + a fatty acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.91','2-oxo-3-(5-oxofuran-2-ylidene)propanoate lactonase','NULL','NULL','This enzyme, characterized from the soil bacterium Bradyrhizobium sp.JS329, is involved in the pathway of 5-nitroanthranilate degradation.','2-oxo-3-(5-oxofuran-2-ylidene)propanoate + H(2)O = maleylpyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.92','4-sulfomuconolactone hydrolase','NULL','NULL','The enzyme was isolated from the bacteria Hydrogenophaga intermediaand Agrobacterium radiobacter S2.It catalyzes a step in the degradation of 4-sulfocatechol.','4-sulfomuconolactone + H(2)O = maleylacetate + sulfite.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.93','Mycophenolic acid acyl-glucuronide esterase','AcMPAG deglucuronidase','NULL','This liver enzyme deglucuronidates mycophenolic acid O-acyl-glucuronide, a metabolite of the immunosuppressant drug mycophenolatethat is thought to be immunotoxic.','Mycophenolic acid O-acyl-glucuronide + H(2)O = mycophenolate +D-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.94','Versiconal hemiacetal acetate esterase','VHA esterase','NULL','Isolated from the mold Aspergillus parasiticus.Involved in a metabolic grid that leads to aflatoxin biosynthesis.','(1) Versiconal hemiacetal acetate + H(2)O = versiconal + acetate.(2) Versiconol acetate + H(2)O = versiconol + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.95','Aclacinomycin methylesterase','Aclacinomycin methyl esterase','NULL','The enzyme is involved in the modification of the aklavinone skeletonin the biosynthesis of anthracyclines in Streptomyces species.','Aclacinomycin T + H(2)O = 15-demethylaclacinomycin T + methanol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.1.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.1','Acetyl-CoA hydrolase','Acetyl-CoA acylase','NULL','NULL','Acetyl-CoA + H(2)O = CoA + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.2','Palmitoyl-CoA hydrolase','Long-chain fatty-acyl-CoA hydrolase','NULL','Also hydrolyzes CoA thioesters of other long-chain fatty acids.','Palmitoyl-CoA + H(2)O = CoA + palmitate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.3','Succinyl-CoA hydrolase','Succinyl-CoA acylase','NULL','NULL','Succinyl-CoA + H(2)O = CoA + succinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.4','3-hydroxyisobutyryl-CoA hydrolase','NULL','NULL','Also hydrolyzes 3-hydroxypropanoyl-CoA.','3-hydroxy-2-methylpropanoyl-CoA + H(2)O = CoA + 3-hydroxy-2-methylpropanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.5','Hydroxymethylglutaryl-CoA hydrolase','NULL','NULL','NULL','(S)-3-hydroxy-3-methylglutaryl-CoA + H(2)O = CoA + 3-hydroxy-3-methylglutarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.6','Hydroxyacylglutathione hydrolase','Glyoxalase II','NULL','Also hydrolyzes S-acetoacetylglutathione, more slowly.Formerly EC 3.1.2.8.','S-(2-hydroxyacyl)glutathione + H(2)O = glutathione + a 2-hydroxycarboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.7','Glutathione thiolesterase','Glutathione thioesterase','NULL','NULL','S-acylglutathione + H(2)O = glutathione + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.9','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.10','Formyl-CoA hydrolase','NULL','NULL','NULL','Formyl-CoA + H(2)O = CoA + formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.11','Acetoacetyl-CoA hydrolase','NULL','NULL','NULL','Acetoacetyl-CoA + H(2)O = CoA + acetoacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.12','S-formylglutathione hydrolase','NULL','NULL','Also hydrolyzes S-acetylglutathione, more slowly.','S-formylglutathione + H(2)O = glutathione + formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.13','S-succinylglutathione hydrolase','NULL','NULL','NULL','S-succinylglutathione + H(2)O = glutathione + succinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.14','Oleoyl-[acyl-carrier-protein] hydrolase','Acyl-[acyl-carrier-protein] hydrolase','NULL','Acts on [acyl-carrier-protein] thioesters of fatty acids from C(12)to C(18), but the derivative of oleic acid is hydrolyzed much morerapidly than any other compound tested.','Oleoyl-[acyl-carrier-protein] + H(2)O = [acyl-carrier-protein] + oleate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.15','Ubiquitin thiolesterase','Ubiquitin carboxy-terminal esterase','NULL','Acts on esters formed between thiols such as dithiothreitol orglutathione and the C-terminal glycine residue of the polypeptideubiquitin.Also acts on AMP-ubiquitin.','Ubiquitin C-terminal thioester + H(2)O = ubiquitin + a thiol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.16','Citrate-lyase deacetylase','[Citrate-(pro-3S)-lyase] thioesterase','NULL','In the proteobacterium Rubrivivax gelatinosus, this enzyme modulatesthe activity of EC 4.1.3.6 by converting it from its active acetylform into its inactive thiol form by removal of its acetyl groups.The activity of citrate-lyase deacetylase is itself inhibited byL-glutamate.','[Citrate (pro-3S)-lyase](acetyl form) + 6 H(2)O = [citrate (pro-3S)-lyase](thiol form) + 6 acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.17','(S)-methylmalonyl-CoA hydrolase','NULL','NULL','NULL','(S)-methylmalonyl-CoA + H(2)O = methylmalonate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.18','ADP-dependent short-chain-acyl-CoA hydrolase','NULL','NULL','Requires ADP; inhibited by NADH.Maximum activity is shown with propanoyl-CoA.','Acyl-CoA + H(2)O = CoA + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.19','ADP-dependent medium-chain-acyl-CoA hydrolase','NULL','NULL','Requires ADP; inhibited by NADH.Maximum activity is shown with nonanoyl-CoA.','Acyl-CoA + H(2)O = CoA + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.20','Acyl-CoA hydrolase','NULL','NULL','Broad specificity for medium- to long-chain acyl-CoA.Insensitive to NAD(+) (cf. EC 3.1.2.19).','Acyl-CoA + H(2)O = CoA + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.21','Dodecanoyl-[acyl-carrier-protein] hydrolase','Dodecanoyl-acyl-carrier-protein hydrolase','NULL','Acts on the acyl-carrier protein thioester of C(12) and, with a muchlower activity, C(14) fatty acids.The derivative of oleic acid is hydrolyzed very slowly.','Dodecanoyl-[acyl-carrier-protein] + H(2)O = [acyl-carrier-protein] +dodecanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.22','Palmitoyl-protein hydrolase','Palmitoyl-protein thioesterase','NULL','Specific for long-chain thioesters of fatty acids.Hydrolyzes fatty acids from S-acylated cysteine residues in proteins,palmitoyl cysteine and palmitoyl-CoA.','Palmitoyl-[protein] + H(2)O = palmitate + [protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.23','4-hydroxybenzoyl-CoA thioesterase','NULL','NULL','This enzyme is part of the bacterial 2,4-dichlorobenzoate degradationpathway.','4-hydroxybenzoyl-CoA + H(2)O = 4-hydroxybenzoate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.24','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.25','Phenylacetyl-CoA hydrolase','NULL','NULL','This is the second step in the conversion of phenylacetyl-CoA tophenylglyoxylate, the first step being carried out by EC 1.17.5.1.','Phenylglyoxylyl-CoA + H(2)O = phenylglyoxylate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.26','Bile-acid-CoA hydrolase','Bile acid-coenzyme A hydrolase','NULL','Choloyl-CoA, 3-dehydrocholoyl-CoA and chenodeoxycholoyl-CoA can alsoact as substrates, but acetyl-CoA, isovaleryl-CoA, palmitoyl-CoA andphenylacetyl-CoA cannot.','Deoxycholoyl-CoA + H(2)O = CoA + deoxycholate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.27','Choloyl-CoA hydrolase','Chenodeoxycholoyl-coenzyme A thioesterase','NULL','Also acts on chenodeoxycholoyl-CoA and to a lesser extent on short-and medium- to long-chain acyl-CoAs, and other substrates, includingtrihydroxycoprostanoyl-CoA, hydroxymethylglutaryl-CoA and branchedchain acyl-CoAs, all of which are present in peroxisomes.The enzyme is strongly inhibited by CoA and may be involved incontrolling CoA levels in the peroxisome.','Choloyl-CoA + H(2)O = cholate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.28','1,4-dihydroxy-2-naphthoyl-CoA hydrolase','NULL','NULL','This enzyme participates in the synthesis of menaquinones,phyloquinone, as well as several plant pigments.The enzyme from the cyanobacterium Synechocystis sp. PCC 6803 doesnot accept benzoyl-CoA or phenylacetyl-CoA as substrates.Formerly EC 3.1.2.n1.','1,4-dihydroxy-2-naphthoyl-CoA + H(2)O = 1,4-dihydroxy-2-naphthoate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.29','Fluoroacetyl-CoA thioesterase','NULL','NULL','Fluoroacetate is extremely toxic.It reacts with CoA to form fluoroacetyl-CoA, which substitutes foracetyl CoA and reacts with EC 2.3.3.1 to produce fluorocitrate,a metabolite of which binds very tightly to EC 4.2.1.3 and halts theTCA cycle.This enzyme hydrolyzes fluoroacetyl-CoA before it can react withcitrate synthase, and thus confers fluoroacetate resistance on theorganisms that produce it.It has been described in the poisonous plant Dichapetalum cymosum andthe bacterium Streptomyces cattleya, both of which are fluoroacetateproducers.','Fluoroacetyl-CoA + H(2)O = fluoroacetate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.30','(3S)-malyl-CoA thioesterase','NULL','NULL','The enzyme has no activity with (2R,3S)-2-methylmalyl-CoA (cf.EC 4.1.3.24) or other CoA esters.Formerly EC 3.1.2.n2.','(S)-malyl-CoA + H(2)O = (S)-malate + CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.2.n2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.1','Alkaline phosphatase','Alkaline phosphomonoesterase','Magnesium; Zinc.','Active at a high pH optimum.Wide specificity.Also catalyzes transphosphorylations.Some enzymes hydrolase diphosphate (cf. EC 3.6.1.1).','A phosphate monoester + H(2)O = an alcohol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.2','Acid phosphatase','Acid phosphomonoesterase','NULL','Wide specificity.Also catalyzes transphosphorylations.','A phosphate monoester + H(2)O = an alcohol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.3','Phosphoserine phosphatase','NULL','NULL','NULL','O-phospho-L(or D)-serine + H(2)O = L(or D)-serine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.4','Phosphatidate phosphatase','Lipin','Magnesium.','This enzyme catalyzes the Mg(2+)-dependent dephosphorylation of a1,2-diacylglycerol-3-phosphate, yielding a 1,2-diacyl-sn-glycerol(DAG), the substrate for de novo lipid synthesis via the Kennedypathway and for the synthesis of triacylglycerol.In lipid signaling, the enzyme generates a pool of DAG to be used forprotein kinase C activation.The mammalian enzymes are known as lipins.','A 1,2-diacylglycerol 3-phosphate + H(2)O = a 1,2-diacyl-sn-glycerol +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.5','5-nucleotidase','NULL','NULL','Wide specificity for 5-nucleotides.','A 5-ribonucleotide + H(2)O = a ribonucleoside + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.6','3-nucleotidase','NULL','NULL','Wide specificity for 3-nucleotides.','A 3-ribonucleotide + H(2)O = a ribonucleoside + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.7','3(2),5-bisphosphate nucleotidase','3(2),5-bisphosphonucleoside 3(2)-phosphohydrolase','NULL','Also acts on 3-phosphoadenylylsulfate, and on the corresponding2-phosphates.','Adenosine 3,5-bisphosphate + H(2)O = adenosine 5-phosphate +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.8','3-phytase','1-phytase','NULL','NULL','Myo-inositol hexakisphosphate + H(2)O = 1D-myo-inositol 1,2,4,5,6-pentakisphosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.9','Glucose-6-phosphatase','NULL','NULL','Wide distribution in animal tissues.Also catalyzes potent transphosphorylations from carbamoyl phosphate,hexose phosphates, diphosphate, phosphoenolpyruvate and nucleosidedi- and triphosphates, to D-glucose, D-mannose, 3-methyl-D-glucose,or 2-deoxy-D-glucose (cf. EC 2.7.1.62, EC 2.7.1.79, and EC 3.9.1.1).','D-glucose 6-phosphate + H(2)O = D-glucose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.10','Glucose-1-phosphatase','NULL','NULL','Also acts, more slowly, on alpha-D-galactose 1-phosphate.','Alpha-D-glucose 1-phosphate + H(2)O = D-glucose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.11','Fructose-bisphosphatase','Fructose 1,6-bisphosphatase','NULL','The animal enzyme also acts on sedoheptulose 1,7-bisphosphate.','D-fructose 1,6-bisphosphate + H(2)O = D-fructose 6-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.12','Trehalose-phosphatase','Trehalose 6-phosphate phosphatase','NULL','NULL','Trehalose 6-phosphate + H(2)O = trehalose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.13','Bisphosphoglycerate phosphatase','NULL','NULL','NULL','2,3-bisphospho-D-glycerate + H(2)O = 3-phospho-D-glycerate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.14','Methylphosphothioglycerate phosphatase','NULL','NULL','NULL','S-methyl-3-phospho-1-thio-D-glycerate + H(2)O = S-methyl-1-thio-D-glycerate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.15','Histidinol-phosphatase','NULL','NULL','NULL','L-histidinol phosphate + H(2)O = L-histidinol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.16','Protein-serine/threonine phosphatase','Protein phosphatase-1','NULL','A group of enzymes removing the serine- or threonine-bound phosphategroup from a wide range of phosphoproteins, including a number ofenzymes which have been phosphorylated under the action of a kinase(cf. EC 3.1.3.48).The spleen enzyme also acts on phenolic phosphates and phosphamides(cf. EC 3.9.1.1).','[a protein]-serine/threonine phosphate + H(2)O = [a protein]-serine/threonine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.17','[Phosphorylase] phosphatase','NULL','NULL','NULL','[Phosphorylase a] + 4 H(2)O = 2 [phosphorylase b] + 4 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.18','Phosphoglycolate phosphatase','NULL','NULL','NULL','2-phosphoglycolate + H(2)O = glycolate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.19','Glycerol-2-phosphatase','NULL','NULL','NULL','Glycerol 2-phosphate + H(2)O = glycerol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.20','Phosphoglycerate phosphatase','NULL','NULL','NULL','D-glycerate 2-phosphate + H(2)O = D-glycerate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.21','Glycerol-1-phosphatase','NULL','NULL','The Dunaliella enzyme acts more rapidly on sn-glycerol 1-phosphatethan on the 3-phosphate.The enzyme from Saccharomyces cerevisiae also acts on propane-1,2-diol 1-phosphate, but not on a variety of other phosphate esters.','Glycerol 1-phosphate + H(2)O = glycerol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.22','Mannitol-1-phosphatase','NULL','NULL','NULL','D-mannitol 1-phosphate + H(2)O = D-mannitol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.23','Sugar-phosphatase','NULL','NULL','Has a wide specificity, acting on aldohexose 1-phosphates, ketohexose1-phosphates, aldohexose 6-phosphates, ketohexose 6-phosphates, bothphosphate ester bonds of fructose 1,6-bisphosphate, phosphoric estersof disaccharides, and on pentose and triose phosphates, but at aslower rate.','Sugar phosphate + H(2)O = sugar + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.24','Sucrose-phosphate phosphatase','SPP','Magnesium.','This is the final step in the sucrose-biosynthesis pathway.Highly specific for sucrose 6-phosphate, with fructose 6-phosphateunable to act as a substrate.The F in sucrose 6(F)-phosphate is used to indicate that the fructoseresidue of sucrose carries the substituent.','Sucrose 6(F)-phosphate + H(2)O = sucrose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.25','Inositol-phosphate phosphatase','Inositol 1-phosphatase','NULL','Acts on five of the six isomers of myo-inositol phosphate, all exceptmyo-inositol 2-phosphate, but does not act on myo-inositol bearingmore than one phosphate group.It also acts on adenosine 2-phosphate (but not the 3- or5-phosphates), sn-glycerol 3-phosphate and glycerol 2-phosphate,but does not act on inositol bisphosphates or more phosphorylatedinositols.','Myo-inositol phosphate + H(2)O = myo-inositol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.26','4-phytase','6-phytase (name based on 1L-numbering system and not 1D-numbering)','NULL','NULL','Myo-inositol hexakisphosphate + H(2)O = 1D-myo-inositol 1,2,3,5,6-pentakisphosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.27','Phosphatidylglycerophosphatase','NULL','NULL','NULL','Phosphatidylglycerophosphate + H(2)O = phosphatidylglycerol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.28','ADP-phosphoglycerate phosphatase','NULL','NULL','Also acts on 2,3-diphosphoglycerate.','3-(ADP)-2-phosphoglycerate + H(2)O = 3-(ADP)-glycerate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.29','N-acylneuraminate-9-phosphatase','NULL','NULL','NULL','N-acylneuraminate 9-phosphate + H(2)O = N-acylneuraminate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.30','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.31','Nucleotidase','NULL','NULL','A wide specificity for 2, 3- and 5- nucleotides; also hydrolyzesglycerol phosphate and 4-nitrophenyl phosphate.Formerly EC 3.1.3.30.','A nucleotide + H(2)O = a nucleoside + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.32','Polynucleotide 3-phosphatase','2(3)-polynucleotidase','NULL','Also hydrolyzes nucleoside 2-, 3- and 5-monophosphates, but only2- and 3-phosphopolynucleotides.','A 3-phosphopolynucleotide + H(2)O = a polynucleotide + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.33','Polynucleotide 5-phosphatase','5-polynucleotidase','NULL','Does not act on nucleoside monophosphates.Induced in Escherichia coli by T-even phages.','A 5-phosphopolynucleotide + H(2)O = a polynucleotide + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.34','Deoxynucleotide 3-phosphatase','3-deoxynucleotidase','NULL','Also catalyzes the selective removal of 3-phosphate groups from DNAand oligodeoxynucleotides.Induced in Escherichia coli by T-even phages.','A deoxynucleoside 3-phosphate + H(2)O = a deoxynucleoside + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.35','Thymidylate 5-phosphatase','Thymidylate 5-nucleotidase','NULL','Acts on 5-methyl-dCMP and on TMP, but more slowly than on dTMP.','Thymidylate + H(2)O = thymidine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.36','Phosphoinositide 5-phosphatase','Diphosphoinositide phosphatase','NULL','These enzymes can also remove the 5-phosphate from Ins(1,4,5)P(3)and/or Ins(1,3,4,5)P(4).They are a diverse family of enzymes, with differing abilities tocatalyze two or more of the four reactions listed.They are thought to use inositol lipids rather than inositolphosphates as substrates in vivo.All of them can use either or both of PtdIns(4,5)P(2) andPtdIns(3,4,5)P(3) as substrates; this is the main property thatdistinguishes them from EC 3.1.3.56.','1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate + H(2)O = 1-phosphatidyl-1D-myo-inositol 4-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.37','Sedoheptulose-bisphosphatase','Sedoheptulose-1,7-bisphosphatase','NULL','NULL','Sedoheptulose 1,7-bisphosphate + H(2)O = sedoheptulose 7-phosphate +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.38','3-phosphoglycerate phosphatase','NULL','NULL','Wide specificity, but 3-phosphoglycerate is the best substrate.','D-glycerate 3-phosphate + H(2)O = D-glycerate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.39','Streptomycin-6-phosphatase','NULL','NULL','Also acts on dihydrostreptomycin 3-alpha,6-bisphosphate andstreptidine 6-phosphate.','Streptomycin 6-phosphate + H(2)O = streptomycin + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.40','Guanidinodeoxy-scyllo-inositol-4-phosphatase','NULL','NULL','NULL','1-guanidino-1-deoxy-scyllo-inositol 4-phosphate + H(2)O = 1-guanidino-1-deoxy-scyllo-inositol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.41','4-nitrophenylphosphatase','NULL','NULL','A number of other substances, including phenyl phosphate,4-nitrophenyl sulfate, acetyl phosphate and glycerol phosphate,are not substrates.','4-nitrophenyl phosphate + H(2)O = 4-nitrophenol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.42','[Glycogen-synthase-D] phosphatase','NULL','NULL','The product is EC 2.4.1.11.','[Glycogen-synthase D] + H(2)O = [glycogen-synthase I] + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.43','[Pyruvate dehydrogenase (acetyl-transferring)]-phosphatase','NULL','NULL','A mitochondrial enzyme associated with EC 1.2.4.1 in the pyruvatedehydrogenase complex.','[Pyruvate dehydrogenase (acetyl-transferring)] phosphate + H(2)O =[pyruvate dehydrogenase (acetyl-transferring)] + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.44','[Acetyl-CoA carboxylase]-phosphatase','NULL','NULL','Simultaneously dephosphorylates and activates EC 6.4.1.2.Acts similarly on EC 1.1.1.88, EC 2.4.1.1 and EC 2.4.1.11,and dephosphorylates phosphoprotamine and 4-nitrophenyl phosphate.Not identical with EC 3.1.3.17 or EC 3.1.3.43.','[Acetyl-CoA carboxylase] phosphate + H(2)O = [acetyl-CoA carboxylase] +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.45','3-deoxy-manno-octulosonate-8-phosphatase','NULL','NULL','NULL','3-deoxy-D-manno-octulosonate 8-phosphate + H(2)O = 3-deoxy-D-manno-octulosonate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.46','Fructose-2,6-bisphosphate 2-phosphatase','Fructose-2,6-bisphosphatase','NULL','The enzyme copurifies with EC 2.7.1.105 (cf. EC 3.1.3.54).','Beta-D-fructose 2,6-bisphosphate + H(2)O = D-fructose 6-phosphate +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.47','[Hydroxymethylglutaryl-CoA reductase (NADPH)]-phosphatase','Reductase phosphatase','NULL','Acts on the product of the reaction catalyzed by EC 2.7.11.31,simultaneously dephosphorylating and activating EC 1.1.1.34.','[Hydroxymethylglutaryl-CoA reductase (NADPH)] phosphate + H(2)O =[hydroxymethylglutaryl-CoA reductase (NADPH)] + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.48','Protein-tyrosine-phosphatase','Phosphotyrosine phosphatase','NULL','Dephosphorylates O-phosphotyrosine groups in phosphoproteins, such asthe products of EC 2.7.10.2.','Protein tyrosine phosphate + H(2)O = protein tyrosine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.49','[Pyruvate kinase]-phosphatase','NULL','NULL','Simultaneously dephosphorylates and activates EC 2.7.1.40 which hasbeen inactivated by protein kinase.','[Pyruvate kinase] phosphate + H(2)O = [pyruvate kinase] + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.50','Sorbitol-6-phosphatase','NULL','NULL','Acts, very slowly, on hexose 6-phosphates.','Sorbitol 6-phosphate + H(2)O = sorbitol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.51','Dolichyl-phosphatase','Dolichol phosphate phosphatase','NULL','NULL','Dolichyl phosphate + H(2)O = dolichol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.52','[3-methyl-2-oxobutanoate dehydrogenase (2-methylpropanoyl-transferring)]-phosphatase','Branched-chain oxo-acid dehydrogenase phosphatase','NULL','A mitochondrial enzyme associated with the 3-methyl-2-oxobutanoatedehydrogenase complex.Simultaneously dephosphorylates and activates EC 1.2.4.4 which hasbeen inactivated by phosphorylation.','[3-methyl-2-oxobutanoate dehydrogenase (2-methylpropanoyl-transferring)]phosphate + H(2)O = [3-methyl-2-oxobutanoate dehydrogenase(2-methylpropanoyl-transferring)] + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.53','[Myosin-light-chain] phosphatase','Myosin light-chain kinase phosphatase','NULL','Composed of three subunits.The holoenzyme dephosphorylates myosin light chains and EC 2.7.11.31,but not myosin; the catalytic subunit acts on all three substrates.','[Myosin light-chain] phosphate + H(2)O = [myosin light-chain] +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.54','Fructose-2,6-bisphosphate 6-phosphatase','NULL','NULL','Cf. EC 3.1.3.46.','Beta-D-fructose 2,6-bisphosphate + H(2)O = beta-D-fructofuranose2-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.55','Caldesmon-phosphatase','NULL','NULL','Dephosphorylation activates the calmodulin- and actin-binding abilityof caldesmon.','[Caldesmon] phosphate + H(2)O = [caldesmon] + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.56','Inositol-polyphosphate 5-phosphatase','5PTase','NULL','This enzyme is distinguished from the family of enzymes classifiedunder EC 3.1.3.36 by its inability to dephosphorylate inositollipids.','(1) D-myo-inositol 1,4,5-trisphosphate + H(2)O = myo-inositol 1,4-bisphosphate + phosphate.(2) 1D-myo-inositol 1,3,4,5-tetrakisphosphate + H(2)O = 1D-myo-inositol1,3,4-trisphosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.57','Inositol-1,4-bisphosphate 1-phosphatase','Inositol polyphosphate 1-phosphatase','NULL','Acts on inositol 1,4-bisphosphate and inositol 1,3,4-trisphosphate(forming inositol 3,4-bisphosphate) with similar V(max) for bothsubstrates, but with a 5 times higher affinity for the bisphosphate.Does not act on inositol 1-phosphate, inositol 1,4,5-trisphosphate orinositol 1,3,4,5-tetrakisphosphate.','1D-myo-inositol 1,4-bisphosphate + H(2)O = 1D-myo-inositol 4-phosphate +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.58','Sugar-terminal-phosphatase','Xylitol-5-phosphatase','NULL','Acts on sugars and polyols phosphorylated on the terminal carbon,with a preference for sugars with a D-erythro-configuration, e.g.good substrates are glucose 6-phosphate, mannose 6-phosphate,6-phosphogluconate, erythrose 4-phosphate and xylitol 5-phosphate.','D-glucose 6-phosphate + H(2)O = D-glucose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.59','Alkylacetylglycerophosphatase','NULL','NULL','Involved in the biosynthesis of thrombocyte activating factor inanimal tissues.','1-alkyl-2-acetyl-sn-glycero-3-phosphate + H(2)O = 1-alkyl-2-acetyl-sn-glycerol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.60','Phosphoenolpyruvate phosphatase','NULL','NULL','Also acts, more slowly, on a wide range of other monophosphates.','Phosphoenolpyruvate + H(2)O = pyruvate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.61','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.62','Multiple inositol-polyphosphate phosphatase','Inositol (1,3,4,5)-tetrakisphosphate 3-phosphatase','NULL','Also acts on Ins(1,3,4,5)P(4) to yield Ins(1,4,5)P(3).','Myo-inositol hexakisphosphate + H(2)O = myo-inositol pentakisphosphate(mixed isomers) + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.63','2-carboxy-D-arabinitol-1-phosphatase','NULL','NULL','NULL','2-carboxy-D-arabinitol 1-phosphate + H(2)O = 2-carboxy-D-arabinitol +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.64','Phosphatidylinositol-3-phosphatase','D-myo-inositol-1,3-bisphosphate 3-phosphohydrolase','NULL','This enzyme still works when the 2,3-bis(acyloxy)propyl group isremoved, i.e., it hydrolyzes Ins(1,3)P(2) to Ins-1-P.Formerly EC 3.1.3.65.','1-phosphatidyl-1D-myo-inositol 3-phosphate + H(2)O = 1-phosphatidyl-1D-myo-inositol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.65','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.66','Phosphatidylinositol-3,4-bisphosphate 4-phosphatase','D-myo-inositol-3,4-bisphosphate 4-phosphohydrolase','NULL','Magnesium-independent.This enzyme still works when the 2,3-bis(acyloxy)propyl group isremoved, i.e., it hydrolyzes Ins(1,3,4)P(3) to Ins(1,3)P(2).It also converts Ins(3,4)P(2) into Ins-3-P.','1-phosphatidyl-myo-inositol 3,4-bisphosphate + H(2)O = 1-phosphatidyl-1D-myo-inositol 3-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.67','Phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase','Phosphatidylinositol-3,4,5-trisphosphate 3-phosphohydrolase','Magnesium.','Does not dephosphorylate inositol 4,5-bisphosphate.This enzyme still works when the 2,3-bis(acyloxy)propyl group isremoved, i.e., it hydrolyzes Ins(1,3,4,5)P(4) to Ins(1,4,5)P(3).','Phosphatidylinositol 3,4,5-trisphosphate + H(2)O = phosphatidylinositol4,5-bisphosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.68','2-deoxyglucose-6-phosphatase','2-deoxyglucose-6-phosphate phosphatase','NULL','Also active toward fructose 1-phosphate.','2-deoxy-D-glucose 6-phosphate + H(2)O = 2-deoxy-D-glucose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.69','Glucosylglycerol 3-phosphatase','NULL','NULL','Acts with EC 2.4.1.213 to form glucosylglycerol, an osmolyte thatendows cyanobacteria with resistance to salt.','2-(beta-D-glucosyl)-sn-glycerol-3-phosphate + H(2)O = 2-(beta-D-glucosyl)-sn-glycerol + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.70','Mannosyl-3-phosphoglycerate phosphatase','NULL','Magnesium.','The enzyme from Pyrococcus horikoshii is specific for alpha-D-mannosyl-3-phosphoglycerate and forms part of the pathway for thesynthesis of mannosylglycerate.','2(alpha-D-mannosyl)-3-phosphoglycerate + H(2)O = 2(alpha-D-mannosyl)-D-glycerate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.71','2-phosphosulfolactate phosphatase','(2R)-phosphosulfolactate phosphohydrolase','Magnesium.','The enzyme from Methanococcus jannaschii acts on both stereoisoimersof the substrate and also hydrolyzes a number of phosphate monoestersof (S)-2-hydroxycarboxylic acids, including 2-phosphomalate,2-phospholactate and 2-phosphoglycolate.This enzyme can also hydrolyze phosphate monoesters of (R)-2-hydroxycarboxylic acids such as (S)-2-phospho-3-sulfolactate and (R)-2-phosphomalate, which, presumably, bind to the enzyme in oppositeorientations.','(2R)-2-phospho-3-sulfolactate + H(2)O = (2R)-3-sulfolactate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.72','5-phytase','NULL','NULL','The enzyme attacks the product of the above reaction more slowly toyield Ins(1,2,3)P(3).','Myo-inositol hexakisphosphate + H(2)O = 1L-myo-inositol 1,2,3,4,6-pentakisphosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.73','Adenosylcobalamin/alpha-ribazole phosphatase','Adenosylcobalamin phosphatase','NULL','This enzyme catalyzes the last step in the anaerobic (early cobaltinsertion) pathway of adenosylcobalamin biosynthesis, characterizedin Salmonella enterica.It also participates in a salvage pathway that recycles cobinamideinto adenosylcobalamin.','(1) Adenosylcobalamin 5-phosphate + H(2)O = coenzyme B12 + phosphate.(2) Alpha-ribazole 5-phosphate + H(2)O = alpha-ribazole + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.74','Pyridoxal phosphatase','PLP phosphatase','Magnesium.','Specific for phosphorylated vitamin B(6) compounds: it acts not onlyon pyridoxal phosphate (PLP), but also on pyridoxine phosphate (PNP),pyridoxamine phosphate (PMP), 4-pyridoxic acid phosphate and4-deoxypyridoxine phosphate.This reaction can also be carried out by EC 3.1.3.1 and EC 3.1.3.2,but these enzymes have very broad substrate specificities.','Pyridoxal 5-phosphate + H(2)O = pyridoxal + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.75','Phosphoethanolamine/phosphocholine phosphatase','3X11A','Cobalt or manganese or magnesium.','The enzyme is highly specific for phosphoethanolamine andphosphocholine.','(1) O-phosphoethanolamine + H(2)O = ethanolamine + phosphate.(2) Phosphocholine + H(2)O = choline + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.76','Lipid-phosphate phosphatase','Dihydroxy fatty acid phosphatase','Magnesium.','The enzyme from mammals is a bifunctional enzyme: the N-terminaldomain exhibits lipid-phosphate-phosphatase activity and theC-terminal domain has the activity of EC 3.3.2.10.The best substrates for this enzyme are 10-hydroxy-9-(phosphonooxy)octadecanoates, with the threo- form being a bettersubstrate than the erythro- form.The phosphatase activity is not found in plant EC 3.3.2.10 or inmammalian EC 3.3.2.9.','(9S,10S)-10-hydroxy-9-(phosphonooxy)octadecanoate + H(2)O = (9S,10S)-9,10-dihydroxyoctadecanoate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.77','Acireductone synthase','E-1 enolase-phosphatase','NULL','This bifunctional enzyme first enolizes the substrate to form theintermediate 2-hydroxy-5-(methylthio)-3-oxopent-1-enyl phosphate,which is then dephosphorylated to form the acireductone 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one.The acireductone represents a branch point in the methione salvagepathway as it is used in the formation of formate, CO and3-(methylthio)propanoate by EC 1.13.11.53 and of formate and4-methylthio-2-oxobutanoate either by a spontaneous reaction underaerobic conditions or by EC 1.13.11.54.','5-(methylthio)-2,3-dioxopentyl phosphate + H(2)O = 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.78','Phosphatidylinositol-4,5-bisphosphate 4-phosphatase','PtdIns-4,5-P(2) 4-phosphatase','NULL','Two pathways exist in mammalian cells to degrade 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate (PtdIns(4,5)P(2)).One is catalyzed by this enzyme and the other by EC 3.1.3.36, wherethe product is PtdIns4P.The enzyme from human is specific for PtdIns(4,5)P(2) as substrate,as it cannot use PtdIns(3,4,5)P(3), PtdIns(3,4)P(2), PtdIns(3,5)P(2),PtdIns5P, PtdIns4P or PtdIns3P.In humans, the enzyme is localized to late endosomal/lysosomalmembranes.It can control nuclear levels of PtdIns5P and thereby control p53-dependent apoptosis.Formerly EC 3.1.3.n3.','1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate + H(2)O = 1-phosphatidyl-1D-myo-inositol 5-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.79','Mannosylfructose-phosphate phosphatase','Mannosylfructose-6-phosphate phosphatase','Magnesium.','Mannosylfructose is the major endogenous osmolyte produced by severalalpha-proteobacteria in response to osmotic stress and is synthesizedby the sequential action of EC 2.4.1.246 followed by this enzyme.While mannosylfructose 6-phosphate is the physiological substrate,the enzyme can use sucrose 6-phosphate very efficiently.The F in mannosylfructose 6(F)-phosphate is used to indicate that thefructose residue of sucrose carries the substituent.','Beta-D-fructofuranosyl-alpha-D-mannopyranoside 6(F)-phosphate + H(2)O =alpha-D-fructofuranosyl-alpha-D-mannopyranoside + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.80','2,3-bisphosphoglycerate 3-phosphatase','2,3-BPG 3-phosphatase','NULL','This reaction is a shortcut in the Rapoport-Luebering shunt.It bypasses the reactions of EC 3.1.3.13/EC 5.4.2.12 and directlyforms 2-phospho-Dglycerate by removing the 3-phospho-group of 2,3-diphospho-Dglycerate.The MIPP1 protein also catalyzes the reaction of EC 3.1.3.62.','2,3-bisphospho-D-glycerate + H(2)O = 2-phospho-D-glycerate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.81','Diacylglycerol diphosphate phosphatase','DGPP phosphatase','NULL','The bifunctional enzyme catalyzes the dephosphorylation ofdiacylglycerol diphosphate to phosphatidate and the subsequentdephosphorylation of phosphatidate to diacylglycerol (cf.EC 3.1.3.4).It regulates intracellular levels of diacylglycerol diphosphate andphosphatidate, phospholipid molecules believed to play a signalingrole in stress response.The phosphatase activity of the bifunctional enzyme is Mg(2+)-independent and N-ethylmaleimide-insensitive and is distinct from theMg(2+)-dependent and N-ethylmaleimide-sensitive enzyme EC 3.1.3.4.The diacylglycerol pyrophosphate phosphatase activity inSaccharomyces cerevisiae is induced by zinc depletion, by inositolsupplementation, and when cells enter the stationary phase.','1,2-diacyl-sn-glycerol 3-diphosphate + H(2)O = 1,2-diacyl-sn-glycerol3-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.82','D-glycero-beta-D-manno-heptose 1,7-bisphosphate 7-phosphatase','NULL','NULL','The enzyme is involved in biosynthesis of ADP-L-glycero-beta-D-manno-heptose, which is utilized for assembly of the lipopolysaccharideinner core in Gram-negative bacteria.In vitro the catalytic efficiency with the beta-anomer is 100-200-fold higher than with the alpha-anomer.','D-glycero-beta-D-manno-heptose 1,7-bisphosphate + H(2)O = D-glycero-beta-D-manno-heptose 1-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.83','D-glycero-alpha-D-manno-heptose-1,7-bisphosphate 7-phosphatase','NULL','NULL','The enzyme is involved in biosynthesis of GDP-D-glycero-alpha-D-manno-heptose, which is required for assembly of S-layer glycoproteinin some Gram-positive bacteria.The in vitro catalytic efficiency of the enzyme from Bacteroidesthetaiotaomicron is 6-fold higher with the alpha-anomer than with thebeta-anomer.','D-glycero-alpha-D-manno-heptose-1,7-bisphosphate + H(2)O =  D-glycero-alpha-D-manno-heptose-1-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.84','ADP-ribose 1-phosphate phosphatase','Appr1p phosphatase','NULL','The enzyme is highly specific for ADP-D-ribose 1-phosphate.Involved together with EC 3.1.4.37 in the breakdown of adenosinediphosphate ribose 1,2-cyclic phosphate (Appr','ADP-D-ribose 1-phosphate + H(2)O = ADP-D-ribose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.85','Glucosyl-3-phosphoglycerate phosphatase','NULL','NULL','The enzyme is involved in biosynthesis of 2-O-(alpha-D-glucopyranosyl)-D-glycerate via the two-step pathway in whichEC 2.4.1.266 catalyzes the conversion of GDP-glucose and 3-phospho-D-glycerate into 2-O-(alpha-D-glucopyranosyl)-3-phospho-D-glycerate,which is then converted to 2-O-(alpha-D-glucopyranosyl)-D-glycerateby glucosyl-3-phosphoglycerate phosphatase.In vivo the enzyme catalyzes the dephosphorylation of 2-O-(alpha-D-mannopyranosyl)-3-phospho-D-glycerate with lower efficiency.Divalent metal ions (Mg(2+), Mn(2+) or Co(2+)) stimulate activity.','2-O-(alpha-D-glucopyranosyl)-3-phospho-D-glycerate + H(2)O = 2-O-(alpha-D-glucopyranosyl)-D-glycerate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.86','Phosphatidylinositol-3,4,5-trisphosphate 5-phosphatase','SHIP','NULL','This enzyme hydrolyzes phosphatidylinositol-3,4,5-trisphosphate(PtdIns(3,4,5)P(3)) to produce PtdIns(3,4)P(2), thereby negativelyregulating the PI3K (phosphoinositide 3-kinase) pathways.The enzyme also shows activity toward PtdIns(1,3,4,5)P(4).The enzyme is involved in several signal transduction pathways in theimmune system leading to an adverse range of effects.Formerly EC 3.1.3.n1.','1-phosphatidyl-1D-myo-inositol 3,4,5-triphosphate + H(2)O =1-phosphatidyl-1D-myo-inositol 3,4-diphosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.87','2-hydroxy-3-keto-5-methylthiopentenyl-1-phosphate phosphatase','HK-MTPenyl-1-P phosphatase','NULL','The enzyme participates in the methionine salvage pathway in Bacillussubtilis.In some species a single bifunctional enzyme, EC 3.1.3.77, catalyzesboth this reaction and that of EC 5.3.2.5.Formerly EC 3.1.3.n4.','2-hydroxy-5-(methylthio)-3-oxopent-1-enyl phosphate + H(2)O = 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.88','5-phosphoribostamycin phosphatase','NULL','NULL','Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including ribostamycin, neomycin andbutirosin.No metal is required for activity.','5-phosphoribostamycin + H(2)O = ribostamycin + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.89','5-deoxynucleotidase','NULL','Cobalt.','The enzyme, characterized from the bacterium Escherichia coli, showsstrict specificity toward deoxyribonucleoside 5-monophosphates anddoes not dephosphorylate 5-ribonucleotides or ribonucleoside3-monophosphates.','A 2-deoxyribonucleoside 5-monophosphate + H(2)O =a 2-deoxyribonucleoside + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.90','Maltose 6-phosphate phosphatase','Maltose 6-P phosphatase','NULL','The enzyme from the bacterium Enterococcus faecalis also has activitywith the sucrose isomer turanose 6-phosphate (alpha-D-glucopyranosyl-(1-','Maltose 6-phosphate + H(2)O = maltose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.91','7-methylguanosine nucleotidase','N(7)-methylguanylate 5-phosphatase','NULL','The enzyme also has low activity with N(7)-methyl-GDP, producingN(7)-methyl-GMP.Does not accept AMP or GMP, and has low activity with UMP.','(1) N(7)-methyl-GMP + H(2)O = N(7)-methyl-guanosine + phosphate.(2) CMP + H(2)O = cytidine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.92','Kanosamine-6-phosphate phosphatase','NULL','NULL','The enzyme, found in the bacterium Bacillus subtilis, is involved ina kanosamine biosynthesis pathway.','Kanosamine 6-phosphate + H(2)O = kanosamine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.93','L-galactose 1-phosphate phosphatase','NULL','NULL','The enzyme from plants also has the activity of EC 3.1.3.25.The enzymes have very low activity with D-galactose 1-phosphate (cf.EC 3.1.3.94).Formerly EC 3.1.3.n5.','Beta-L-galactose 1-phosphate + H(2)O = L-galactose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.94','D-galactose 1-phosphate phosphatase','NULL','NULL','The human enzyme also has the activity of EC 3.1.3.25.The enzyme has very low activity with L-galactose 1-phosphate (cf.EC 3.1.3.93).','Alpha-D-galactose 1-phosphate + H(2)O = D-galactose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.95','Phosphatidylinositol-3,5-bisphosphate 3-phosphatase','PtdIns-3,5-P(2) 3-Ptase','NULL','The enzyme is found in both plants and animals.It also has the activity of EC 3.1.3.64.','1-phosphatidyl-1D-myo-inositol 3,5-bisphosphate + H(2)O = 1-phosphatidyl-1D-myo-inositol 5-phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.n2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.n3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.n4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.3.n5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.1','Phosphodiesterase I','5 nucleotide phosphodiesterase/alkaline phosphodiesterase I','NULL','Hydrolyzes both ribonucleotides and deoxyribonucleotides.Has low activity toward polynucleotides.A 3-phosphate terminus on the substrate inhibits hydrolysis.','Hydrolytically removes 5-nucleotides successively from the 3-hydroxytermini of 3-hydroxy-terminated oligonucleotides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.2','Glycerophosphocholine phosphodiesterase','NULL','NULL','Also acts on sn-glycero-3-phosphoethanolamine.','sn-glycero-3-phosphocholine + H(2)O = choline + sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.3','Phospholipase C','Clostridium oedematiens beta- and gamma-toxins','Zinc.','The bacterial enzyme also acts on sphingomyelin andphosphatidylinositol.The enzyme from seminal plasma does not act on phosphatidylinositol.','A phosphatidylcholine + H(2)O = 1,2-diacyl-sn-glycerol + phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.4','Phospholipase D','Choline phosphatase','NULL','Also acts on other phosphatidyl esters.','A phosphatidylcholine + H(2)O = choline + a phosphatidate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.6','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.7','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.9','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.10','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.11','Phosphoinositide phospholipase C','1-phosphatidyl-D-myo-inositol-4,5-bisphosphateinositoltrisphosphohydrolase','NULL','These enzymes form some of the cyclic phosphate Ins(cyclic1,2)P(4,5)P(2) as well as Ins(1,4,5)P(3).They show activity toward phosphatidylinositol, i.e., the activity ofEC 4.6.1.13 in vitro at high calcium concentration.','1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate + H(2)O = 1D-myo-inositol1,4,5-trisphosphate + diacylglycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.12','Sphingomyelin phosphodiesterase','Acid sphingomyelinase','NULL','Has very little activity on phosphatidylcholine.','Sphingomyelin + H(2)O = N-acylsphingosine + phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.13','Serine-ethanolaminephosphate phosphodiesterase','NULL','NULL','Acts only on those phosphodiesters that have ethanolamine as acomponent part of the molecule.','Serine phosphoethanolamine + H(2)O = serine + ethanolamine phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.14','[Acyl-carrier-protein] phosphodiesterase','[Acyl-carrier-protein] 4-pantetheine-phosphohydrolase','NULL','The enzyme cleaves acyl-[acyl-carrier-protein] species with acylchains of 6-16 carbon atoms although it appears to demonstrate apreference for the unacylated acyl-carrier-protein (ACP) and short-chain ACPs over the medium- and long-chain species.Deletion of the gene encoding this enzyme abolishes ACP prosthetic-group turnover in vivo.Activation of apo-ACP to form the holoenzyme is carried out byEC 2.7.8.7.','Holo-[acyl-carrier-protein] + H(2)O = 4-phosphopantetheine + apo-[acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.15','Adenylyl-[glutamate--ammonia ligase] hydrolase','NULL','NULL','NULL','Adenylyl-[L-glutamate:ammonia ligase (ADP-forming)] + H(2)O = adenylate +[L-glutamate:ammonia ligase (ADP-forming)].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.16','2,3-cyclic-nucleotide 2-phosphodiesterase','NULL','NULL','Also hydrolyzes 3-nucleoside monophosphates and bis-4-nitrophenylphosphate, but not 3-deoxynucleotides.Similar reactions are carried out by EC 3.1.27.3 and EC 3.1.27.5.','Nucleoside 2,3-cyclic phosphate + H(2)O = nucleoside 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.17','3,5-cyclic-nucleotide phosphodiesterase','Cyclic AMP phosphodiesterase','NULL','Acts on 3,5-cyclic AMP, 3,5-cyclic dAMP, 3,5-cyclic IMP,3,5-cyclic GMP and 3,5-cyclic CMP.','Nucleoside 3,5-cyclic phosphate + H(2)O = nucleoside 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.18','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.19','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.20','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.21','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.22','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.23','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.24','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.25','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.26','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.27','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.28','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.29','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.30','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.31','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.32','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.33','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.34','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.35','3,5-cyclic-GMP phosphodiesterase','NULL','NULL','NULL','Guanosine 3,5-cyclic phosphate + H(2)O = guanosine 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.36','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.37','2,3-cyclic-nucleotide 3-phosphodiesterase','Cyclic-CMP phosphodiesterase','NULL','The brain enzyme acts on 2,3-cyclic AMP more rapidly than on theUMP or CMP derivatives.An enzyme from liver acts on 2,3-cyclic CMP more rapidly than onthe purine derivatives; it also hydrolyzes the corresponding3,5-cyclic phosphates, more slowly.This latter enzyme has been called cyclic-CMP phosphodiesterase.','Nucleoside 2,3-cyclic phosphate + H(2)O = nucleoside 2-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.38','Glycerophosphocholine cholinephosphodiesterase','NULL','NULL','No activity on sn-3-glycerophosphoethanolamine.','sn-glycero-3-phosphocholine + H(2)O = glycerol + phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.39','Alkylglycerophosphoethanolamine phosphodiesterase','Lysophospholipase D','NULL','Also acts on acyl and choline analogs.','1-alkyl-sn-glycero-3-phosphoethanolamine + H(2)O = 1-alkyl-sn-glycerol3-phosphate + ethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.40','CMP-N-acylneuraminate phosphodiesterase','CMP-sialate hydrolase','NULL','NULL','CMP-N-acylneuraminate + H(2)O = CMP + N-acylneuraminate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.41','Sphingomyelin phosphodiesterase D','NULL','NULL','Does not act on phosphatidylcholine, but hydrolyzes2-lysophosphatidylcholine to choline and 2-lysophosphatidate.','Sphingomyelin + H(2)O = ceramide phosphate + choline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.42','Glycerol-1,2-cyclic-phosphate 2-phosphodiesterase','NULL','NULL','Acts on both stereoisomers of the substrate and also, more slowly,on 3,5-cyclic AMP and on 2,3-cyclic AMP.','Glycerol 1,2-cyclic phosphate + H(2)O = glycerol 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.43','Glycerophosphoinositol inositolphosphodiesterase','1,2-cyclic-inositol-phosphate phosphodiesterase','NULL','This enzyme also hydrolyzes Ins(cyclic 1,2)P to Ins-1-P.Formerly EC 3.1.4.36.','1-(sn-glycero-3-phospho)-1D-myo-inositol + H(2)O = glycerol + 1D-myo-inositol 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.44','Glycerophosphoinositol glycerophosphodiesterase','sn-glycero(3)phosphoinositol glycerophosphohydrolase','NULL','NULL','1-(sn-glycero-3-phospho)-1D-myo-inositol + H(2)O = myo-inositol +sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.45','N-acetylglucosamine-1-phosphodiester alpha-N-acetylglucosaminidase','Alpha-N-acetylglucosaminyl phosphodiesterase','NULL','Acts on a variety of compounds in which N-acetyl-D-glucosamine isalpha-linked to a phosphate group, including the biosynthetic inintermediates of the high mannose oligosaccharide components of somelysosomal enzymes and the product of EC 2.7.8.17.','Glycoprotein N-acetyl-D-glucosaminyl-phospho-D-mannose + H(2)O =N-acetyl-D-glucosamine + glycoprotein phospho-D-mannose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.46','Glycerophosphodiester phosphodiesterase','Glycerophosphoryl diester phosphodiesterase','NULL','Broad specificity for glycerophosphodiesters; glycerophosphocholine,glycerophosphoethanolamine, glycerophosphoglycerol andbis(glycerophospho)-glycerol are hydrolyzed.','A glycerophosphodiester + H(2)O = an alcohol + sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.47','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.48','Dolichylphosphate-glucose phosphodiesterase','NULL','NULL','NULL','Dolichyl beta-D-glucosyl phosphate + H(2)O = dolichyl phosphate +D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.49','Dolichylphosphate-mannose phosphodiesterase','Mannosylphosphodolichol phosphodiesterase','NULL','NULL','Dolichyl beta-D-mannosyl phosphate + H(2)O = dolichyl phosphate +D-mannose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.50','Glycosylphosphatidylinositol phospholipase D','Glycoprotein phospholipase D','NULL','This enzyme is also active when O-4 of the glucosamine is substitutedby carrying the oligosaccharide that can link a protein to thestructure.It therefore cleaves proteins from the lipid part of theglycosylphosphatidylinositol (GPI) anchors, but does so byhydrolysis, whereas EC 4.6.1.14 does so by elimination.It acts on plasma membranes only after solubilization of thesubstrate with detergents or solvents, but it may act onintracellular membranes.','6-(alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol + H(2)O =6-(alpha-D-glucosaminyl)-1D-myo-inositol + phosphatidate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.51','Glucose-1-phospho-D-mannosylglycoprotein phosphodiesterase','Alpha-glucose-1-phosphate phosphodiesterase','NULL','This enzyme is specific for the product of EC 2.7.8.19.','6-(D-glucose-1-phospho)-D-mannosylglycoprotein + H(2)O = alpha-D-glucose1-phosphate + D-mannosylglycoprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.52','Cyclic-guanylate-specific phosphodiesterase','C-di-GMP-specific phosphodiesterase','Heme; Magnesium; Manganese.','Inhibited by Ca(2+) and Zn(2+).Linearizes cyclic di-3,5-guanylate, the product of EC 2.7.7.65 andan allosteric activator of EC 2.4.1.12 rendering it inactive.It is the balance between these two enzymes that determines thecellular level of cyclic di-3,5-guanylate.','Cyclic di-3,5-guanylate + H(2)O = 5-phosphoguanylyl(3-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.53','3,5-cyclic-AMP phosphodiesterase','cAMP-specific PDE','NULL','Specific for 3,5-cAMP and does not hydrolyze other nucleoside3,5-cyclic phosphates such as cGMP (cf. EC 3.1.4.17 andEC 3.1.4.35).It is involved in modulation of the levels of cAMP, which is amediator in the processes of cell transformation and proliferation.Formerly EC 3.1.4.n1.','Adenosine 3,5-cyclic phosphate + H(2)O = adenosine 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.54','N-acetylphosphatidylethanolamine-hydrolyzing phospholipase D','Anandamide-generating phospholipase D','Zinc.','This enzyme is involved in the biosynthesis of anandamide.It does not hydrolyze phosphatidylcholine andphosphatidylethanolamine.No transphosphatidation.','An N-acylphosphatidylethanolamine + H(2)O = an N-acylethanolamine +a 1,2-diacylglycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.55','Phosphoribosyl 1,2-cyclic phosphate phosphodiesterase','NULL','Manganese; Zinc.','Isolated from the bacterium Escherichia coli, where it participatesin the degradation of methylphosphonate.','5-phospho-alpha-D-ribose 1,2-cyclic phosphate + H(2)O = alpha-D-ribose1,5-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.56','7,8-dihydroneopterin 2,3-cyclic phosphate phosphodiesterase','NULL','Fe(2+); Zinc.','Hydrolyzes 7,8-dihydroneopterin 2,3-cyclic phosphate, a step intetrahydromethanopterin biosynthesis.In vitro the enzyme forms 7,8-dihydroneopterin 2-phosphate and 7,8-dihydroneopterin 3-phosphate at a ratio of 4:1.','(1) 7,8-dihydroneopterin 2,3-cyclic phosphate + H(2)O = 7,8-dihydroneopterin 3-phosphate.(2) 7,8-dihydroneopterin 2,3-cyclic phosphate + H(2)O = 7,8-dihydroneopterin 2-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.4.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.5.1','dGTPase','Deoxy-GTPase','NULL','Also acts on GTP.','dGTP + H(2)O = deoxyguanosine + triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.1','Arylsulfatase','Aryl-sulfate sulphohydrolase','NULL','A group of enzymes with rather similar specificities.','A phenol sulfate + H(2)O = a phenol + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.2','Steryl-sulfatase','Arylsulfatase C','NULL','Also acts on some related steryl sulfates.','3-beta-hydroxyandrost-5-en-17-one 3-sulfate + H(2)O = 3-beta-hydroxyandrost-5-en-17-one + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.3','Glycosulfatase','Glucosulfatase','NULL','Also acts on other sulfates of mono- and disaccharides and onadenosine 5-sulfate.','D-glucose 6-sulfate + H(2)O = D-glucose + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.4','N-acetylgalactosamine-6-sulfatase','Chondroitinase','NULL','NULL','Hydrolysis of the 6-sulfate groups of the N-acetyl-D-galactosamine6-sulfate units of chondroitin sulfate and of the D-galactose 6-sulfateunits of keratan sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.5','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.6','Choline-sulfatase','NULL','NULL','NULL','Choline sulfate + H(2)O = choline + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.7','Cellulose-polysulfatase','NULL','NULL','NULL','Hydrolysis of the 2- and 3-sulfate groups of the polysulfates ofcellulose and charonin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.8','Cerebroside-sulfatase','Arylsulfatase A','NULL','Hydrolyzes galactose-3-sulfate residues in a number of lipids.Also hydrolyzes ascorbate 2-sulfate and many phenol sulfates.','A cerebroside 3-sulfate + H(2)O = a cerebroside + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.9','Chondro-4-sulfatase','NULL','NULL','Also acts on the saturated analog but not on higher oligosaccharides,nor any 6-sulfates.','4-deoxy-beta-D-gluc-4-enuronosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.10','Chondro-6-sulfatase','NULL','NULL','Also acts on the saturated analog and N-acetyl-D-galactosamine 4,6-disulfate, but not higher oligosaccharides, nor any 4-sulfate.','4-deoxy-beta-D-gluc-4-enuronosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.11','Disulfoglucosamine-6-sulfatase','N-sulfoglucosamine-6-sulfatase','NULL','May be identical with EC 3.1.6.14.','2-N,6-O-disulfo-D-glucosamine + H(2)O = 2-N-sulfo-D-glucosamine +sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.12','N-acetylgalactosamine-4-sulfatase','Arylsulfatase B','NULL','Also acts on N-acetylglucosamine 4-sulfate.','Hydrolysis of the 4-sulfate groups of the N-acetyl-D-galactosamine4-sulfate units of chondroitin sulfate and dermatan sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.13','Iduronate-2-sulfatase','Chondroitinsulfatase','NULL','NULL','Hydrolysis of the 2-sulfate groups of the L-iduronate 2-sulfate units ofdermatan sulfate, heparan sulfate and heparin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.14','N-acetylglucosamine-6-sulfatase','Chondroitinsulfatase','NULL','May be identical with EC 3.1.6.11.','Hydrolysis of the 6-sulfate groups of the N-acetyl-D-glucosamine6-sulfate units of heparan sulfate and keratan sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.15','N-sulfoglucosamine-3-sulfatase','Chondroitinsulfatase','NULL','The enzyme from Flavobacterium heparinum also hydrolyzes N-acetyl-D-glucosamine 3-O-sulfate; the mammalian enzyme acts only on thedisulfated residue.','Hydrolysis of the 3-sulfate groups of the N-sulfo-D-glucosamine 3-O-sulfate units of heparin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.16','Monomethyl-sulfatase','NULL','NULL','Highly specific; does not act on monoethyl sulfate, monoisopropylsulfate or monododecyl sulfate.','Monomethyl sulfate + H(2)O = methanol + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.17','D-lactate-2-sulfatase','NULL','NULL','Highly specific.','(R)-2-O-sulfolactate + H(2)O = (R)-lactate + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.18','Glucuronate-2-sulfatase','Chondro-2-sulfatase','NULL','Does not act on iduronate 2-sulfate residues (cf. EC 3.1.6.13).','Hydrolysis of the 2-sulfate groups of the 2-O-sulfo-D-glucuronateresidues of chondroitin sulfate, heparin and heparitin sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.6.19','(R)-specific secondary-alkylsulfatase','(R)-specific sec-alkylsulfatase','Zn(2+).','The enzyme from Rhodococcus ruber is involved in the biodegradationof alkyl sulfate esters used as detergents and released into theenvironment.The preferred substrates are linear secondary-alkyl sulfate esters,particularly octan-2-yl, octan-3-yl, and octan-4-yl sulfates.The enzyme from Pseudomonas sp. DSM6611 utilizes a range ofsecondary-alkyl sulfate esters bearing aromatic, olefinic andacetylenic moieties.Perfect enantioselectivities are obtained for substrates bearinggroups of different size adjacent to the sulfate moiety.The enzymatic hydrolysis proceeds through inversion of theconfiguration at the stereogenic carbon atom.','An (R)-secondary-alkyl sulfate + H(2)O = an (S)-secondary-alcohol +sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.1','Prenyl-diphosphatase','Prenyl-pyrophosphatase','NULL','Farnesyl diphosphate is the best substrate so far tested.','Prenyl diphosphate + H(2)O = prenol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.2','Guanosine-3,5-bis(diphosphate) 3-diphosphatase','(ppGpp)ase','NULL','NULL','Guanosine 3,5-bis(diphosphate) + H(2)O = guanosine 5-diphosphate +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.3','Monoterpenyl-diphosphatase','Bornyl diphosphate hydrolase','NULL','A group of enzymes with varying specificity for the monoterpenolmoiety.One has highest activity on sterically-hindered compounds such as(+)-bornyl diphosphate; another has highest activity on thediphosphates of primary alylic alcohols such as geraniol.','A monoterpenyl diphosphate + H(2)O = a monoterpenol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.4','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.5','Geranylgeranyl diphosphate diphosphatase','Geranylgeranyl diphosphate phosphatase','NULL','Involved in the biosynthesis of plaunotol.There are two isoenzymes with different ion requirements.Neither require Mg(2+) but in addition PII is inhibited by Zn(2+),Mn(2+) and Co(2+).It is not known which isoenzyme is involved in plaunotolbiosynthesis.','Geranylgeranyl diphosphate + H(2)O = geranylgeraniol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.6','Farnesyl diphosphatase','FPP phosphatase','NULL','The enzyme is involved in the biosynthesis of acyclicsesquiterpenoids.','(2E,6E)-farnesyl diphosphate + H(2)O = (2E,6E)-farnesol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.7','Drimenol cyclase','Farnesyl pyrophosphate:drimenol cyclase','NULL','NULL','(2E,6E)-farnesyl diphosphate + H(2)O = drimenol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.8','Tuberculosinol synthase','NULL','NULL','Only found in species of Mycobacterium that cause tuberculosis.In addition, it also gives isotuberculosinol in 1:1 mixture,cf. EC 3.1.7.9.','Tuberculosinyl diphosphate + H(2)O = tuberculosinol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.9','Isotuberculosinol synthase','NULL','NULL','Only found in species of Mycobacterium that cause tuberculosis.In addition, it also gives tuberculosinol in 1:1 mixture,cf EC 3.1.7.8.The isotuberculosinol form was a 3:1 mixture of the 13S and 13Rforms, respectively.','Tuberculosinyl diphosphate + H(2)O = (13S)-isotuberculosinol +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.10','(13E)-labda-7,13-dien-15-ol synthase','Labda-7,13E-dien-15-ol synthase','NULL','The enzyme from the lycophyte Selaginella moellendorffii isbifunctional, initially forming (13E)-labda-7,13-dien-15-yldiphosphate, which is hydrolyzed to the alcohol.','Geranylgeranyl diphosphate + H(2)O = (13E)-labda-7,13-dien-15-ol +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.7.11','Geranyl diphosphate diphosphatase','Geraniol synthase','Manganese or magnesium.','Isolated from Ocimum basilicum (basil) and Cinnamomum tenuipile(camphor tree).Geraniol is labelled when formed in the presence of ((18)O)H2O.Thus mechanism involves a geranyl cation.Neryl diphosphate is hydrolyzed more slowly.May be the same as EC 3.1.7.3.','Geranyl diphosphate + H(2)O = geraniol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.8.1','Aryldialkylphosphatase','A-esterase','Divalent cation.','Acts on organophosphorus compounds (such as paraoxon) includingesters of phosphonic and phosphinic acids.Inhibited by chelating agents.Previously regarded as identical with EC 3.1.1.2.','An aryl dialkyl phosphate + H(2)O = dialkyl phosphate + an aryl alcohol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.8.2','Diisopropyl-fluorophosphatase','DFPase','Divalent cation.','Acts on phosphorus anhydride bonds (such as phosphorus-halide andphosphorus-cyanide) in organophosphorus compounds (including nervegases).Inhibited by chelating agents.Related to EC 3.1.8.1.Formerly EC 3.8.2.1.','Diisopropyl fluorophosphate + H(2)O = diisopropyl phosphate + fluoride.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.11.1','Exodeoxyribonuclease I','E','NULL','Preference for single-stranded DNA.The Escherichia coli enzyme hydrolyzes glucosylated DNA.Similar enzymes; mammalian DNase III, exonuclease IV, T2- andT4-induced exodeoxyribonucleases.Formerly EC 3.1.4.25.','Exonucleolytic cleavage in the 3- to 5-direction to yield nucleoside5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.11.2','Exodeoxyribonuclease III','E','NULL','Preference for double-stranded DNA.Has endonucleolytic activity near apurinic sites on DNA.Similar enzyme: Haemophilus influenzae exonuclease.Formerly EC 3.1.4.27.','Exonucleolytic cleavage in the 3- to 5-direction to yield nucleoside5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.11.3','Exodeoxyribonuclease (lambda-induced)','Lambda exonuclease','NULL','Preference for double-stranded DNA.Does not attack single-strand breaks.Similar enzymes: T4, T5 and T7 exonucleases, mammalian DNase IV.Formerly EC 3.1.4.28.','Exonucleolytic cleavage in the 5- to 3-direction to yield nucleoside5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.11.4','Exodeoxyribonuclease (phage SP3-induced)','DNA 5-dinucleotidohydrolase','NULL','Preference for single-stranded DNA.Formerly EC 3.1.4.31.','Exonucleolytic cleavage in the 5- to 3-direction to yield nucleoside5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.11.5','Exodeoxyribonuclease V','E','NULL','Preference for double-stranded DNA.Possesses DNA-dependent ATPase activity.Acts endonucleolytically on single-stranded circular DNA.Similar enzyme: Haemophilus influenzae ATP-dependent DNase.','Exonucleolytic cleavage (in the presence of ATP) in either 5- to 3- or3- to 5-direction to yield 5-phosphooligonucleotides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.11.6','Exodeoxyribonuclease VII','E','NULL','Preference for single-stranded DNA.Similar enzyme: Micrococcus luteus exonuclease.','Exonucleolytic cleavage in either 5- to 3- or 3- to 5-direction toyield nucleoside 5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.13.1','Exoribonuclease II','Ribonuclease II','NULL','Preference for single-stranded RNA.The enzyme processes 3-terminal extra-nucleotides of monomeric tRNAprecursors, following the action of EC 3.1.26.5.Similar enzymes: RNase Q, RNase BN, RNase PIII, RNase Y.Formerly EC 3.1.4.20.','Exonucleolytic cleavage in the 3- to 5-direction to yield nucleoside5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.13.2','Exoribonuclease H','NULL','NULL','This is a secondary reaction to the RNA 5-end directed cleavage13-19 nucleotides from the RNA end performed by EC 3.1.26.13.','3-end directed exonucleolytic cleavage of viral RNA-DNA hybrid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.13.3','Oligonucleotidase','NULL','NULL','Also hydrolyzes NAD(+) to NMN and AMP.Formerly EC 3.1.4.19.','Exonucleolytic cleavage of oligonucleotides to yield nucleoside5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.13.4','Poly(A)-specific ribonuclease','NULL','NULL','Cleaves poly(A) in either the single- or double-stranded form.','Exonucleolytic cleavage of poly(A) to 5-AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.13.5','Ribonuclease D','RNase D','Cobalt or manganese or magnesium.','Alteration of the 3-terminal base has no effect on the rate ofhydrolysis whereas modification of the 3-terminal sugar has a majoreffect.tRNA terminating with a 3-phosphate is completely inactive.This enzyme can convert a tRNA precursor into a mature tRNA.','Exonucleolytic cleavage that removes extra residues from the 3-terminusof tRNA to produce 5-mononucleotides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.14.1','Yeast ribonuclease','NULL','NULL','Similar enzyme: RNase U4.','Exonucleolytic cleavage to nucleoside 3-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.15.1','Venom exonuclease','Venom phosphodiesterase','NULL','Preference for single-stranded substrate.Similar enzymes: hog kidney phosphodiesterase, Lactobacillusexonuclease.','Exonucleolytic cleavage in the 3- to 5- direction to yield nucleoside5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.16.1','Spleen exonuclease','3-exonuclease','NULL','Preference for single-stranded substrate.Similar enzymes: Lactobacillus acidophilus nuclease, Bacillussubtilis nuclease, salmon testis nuclease.Formerly EC 3.1.4.18.','Exonucleolytic cleavage in the 5- to 3-direction to yield nucleoside3-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.21.1','Deoxyribonuclease I','DNase','NULL','Preference for double-stranded DNA.Similar enzymes: streptococcal DNase (streptodornase),T4 endonuclease II, T7 endonuclease II, Escherichia coli endonucleaseI, nicking nuclease of calf thymus, colicin E2 and E3.Formerly EC 3.1.4.5.','Endonucleolytic cleavage to 5-phosphodinucleotide and5-phosphooligonucleotide end-products.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.21.2','Deoxyribonuclease IV','Deoxyribonuclease IV (phage T4-induced)','NULL','The enzyme is an apurinic/apyrimidinic (AP) site endonuclease thatprimes DNA repair synthesis at AP sites.It specifically cleaves the DNA backbone at AP sites and also removes3 DNA-blocking groups such as 3 phosphates, 3 phosphoglycolates,and 3 alpha,beta-unsaturated aldehydes that arise from oxidativebase damage and the activity of combined glycosylase/lyase enzymes.It is also the only known repair enzyme that is able to cleave theDNA backbone 5 of the oxidative lesion alpha-deoxyadenosine.The enzyme has a strong preference for single-stranded DNA.Formerly EC 3.1.4.30.','Endonucleolytic cleavage to 5-phosphooligonucleotide end-products.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.21.3','Type I site-specific deoxyribonuclease','Type I restriction enzyme','NULL','Large group of enzymes that have an absolute requirement for ATP (ordATP) and S-adenosyl-L-methionine.They recognize specific short DNA sequences and cleave at sitesremote from the recognition sequence.Multifunctional proteins which also catalyze the reactions ofEC 2.1.1.72 and EC 2.1.1.73, with similar site specificity.Formerly EC 3.1.24.1 and EC 3.1.24.2.See the REBASE database for a complete list of these enzymes:http://rebase.neb.com/rebase/','Endonucleolytic cleavage of DNA to give random double-stranded fragmentswith terminal 5-phosphates; ATP is simultaneously hydrolyzed.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.21.4','Type II site-specific deoxyribonuclease','Type II restriction enzyme','Magnesium.','Large group of enzymes which recognize specific short DNA sequencesand cleave either within, or at a short specific distance from,the recognition site.See the REBASE database for a complete list of these enzymes:http://rebase.neb.com/rebase/','Endonucleolytic cleavage of DNA to give specific double-strandedfragments with terminal 5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.21.5','Type III site-specific deoxyribonuclease','Type III restriction enzyme','NULL','Has an absolute requirement for ATP, but does not hydrolyze it;S-adenosyl-L-methionine stimulates the reaction, but is notabsolutely required.Recognize specific short DNA sequences and cleave a short distanceaway from the recognition sequence.Exist as complexes with enzymes of similar specificity listed underEC 2.1.1.72 or EC 2.1.1.73.Formerly EC 3.1.24.3 and EC 3.1.24.4.See the REBASE database for a complete list of these enzymes:http://rebase.neb.com/rebase/','Endonucleolytic cleavage of DNA to give specific double-strandedfragments with terminal 5-phosphates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.21.6','CC-preferring endodeoxyribonuclease','5-CC-3-preferring endodeoxyribonuclease','Magnesium.','Prefers CC sites in double-stranded circular and linear DNA.Greater affinity for double-stranded than single-stranded DNA.Produces nicks, generating double-stranded fragments with 5- and/or3-protruding single-stranded tails.','Endonucleolytic cleavage to give 5-phosphooligonucleotide end-products,with a preference for cleavage within the sequence CC.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.21.7','Deoxyribonuclease V','DNase V','NULL','Formerly EC 3.1.22.3.','Endonucleolytic cleavage at apurinic or apyrimidinic sites to productswith a 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.21.8','T(4) deoxyribonuclease II','T(4) endonuclease II','Magnesium.','This phage T(4) enzyme is involved in degradation of host DNA.The enzyme primarily catalyzes nicking of the bottom strand of doublestranded DNA between the first and second base pair to the right of atop-strand CCGC motif.Double-stranded breaks are produced 5- to 10-fold less frequently.It does not cleave the T4 native DNA, which contains5-hydroxymethylcytosine instead of cytosine.','Endonucleolytic nicking and cleavage of cytosine-containing double-stranded DNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.21.9','T(4) deoxyribonuclease IV','T(4) endonuclease IV','NULL','This phage T(4) enzyme is involved in degradation of host DNA.The enzyme does not cleave double-stranded DNA or native T4 DNA,which contains 5-hydroxymethylcytosine instead of cytosine.','Endonucleolytic cleavage of the 5 phosphodiester bond of deoxycytidinein single-stranded DNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.22.1','Deoxyribonuclease II','DNase II','NULL','Preference for double-stranded DNA.Similar enzymes: crab testis DNase, snail DNase, salmon testis DNase,liver acid DNase, human acid DNases of gastric mucosa and cervix.Formerly EC 3.1.4.6.','Endonucleolytic cleavage to nucleoside 3-phosphates and3-phosphooligonucleotide end-products.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.22.2','Aspergillus deoxyribonuclease K(1)','Aspergillus DNase K1','NULL','Preference for single-stranded DNA.','Endonucleolytic cleavage to nucleoside 3-phosphates and3-phosphooligonucleotide end-products.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.22.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.22.4','Crossover junction endodeoxyribonuclease','Crossover junction endoribonuclease','NULL','The enzyme from Saccharomyces cerevisiae has no endonuclease orexonuclease activity on single-stranded or double-stranded DNAmolecules that do not contain Holliday junctions.','Endonucleolytic cleavage at a junction such as a reciprocal single-stranded crossover between two homologous DNA duplexes (Hollidayjunction).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.22.5','Deoxyribonuclease X','NULL','NULL','Preference for supercoiled DNA; little activity on linear double-stranded DNA.Inhibited by single-stranded DNA, ATP and AMP.','Endonucleolytic cleavage of supercoiled plasma DNA to linear DNAduplexes.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.6','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.7','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.9','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.10','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.11','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.12','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.13','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.14','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.15','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.16','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.17','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.18','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.19','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.20','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.21','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.22','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.23','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.24','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.25','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.26','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.27','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.28','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.29','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.30','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.31','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.32','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.33','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.34','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.35','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.36','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.37','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.38','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.39','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.40','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.41','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.42','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.43','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.44','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.45','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.46','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.47','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.48','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.49','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.50','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.51','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.52','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.53','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.54','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.55','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.56','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.57','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.58','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.23.59','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.24.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.24.2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.24.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.24.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.25.1','Deoxyribonuclease (pyrimidine dimer)','Endodeoxyribonuclease (pyrimidine dimer)','NULL','Acts on a damaged strand, 5 from the damaged site.Similar enzymes: T4 endonuclease V, Escherichia coli endonuclease IIIand V, correndonuclease II.','Endonucleolytic cleavage near pyrimidine dimers to products with5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.25.2','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.1','Physarum polycephalum ribonuclease','NULL','NULL','NULL','Endonucleolytic cleavage to 5-phosphomonoester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.2','Ribonuclease alpha','NULL','NULL','Specific for O-methylated RNA.','Endonucleolytic cleavage to 5-phosphomonoester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.3','Ribonuclease III','Ribonuclease 3','NULL','An endoribonuclease that cleaves double-stranded RNA molecules.The cleavage can be either a single-stranded nick or double-strandedbreak in the RNA, depending in part upon the degree of base-pairingin the region of the cleavage site.Specificity is conferred by negative determinants, i.e., the presenceof certain Watson-Crick base-pairs at specific positions thatstrongly inhibit cleavage.RNase III is involved in both rRNA processing and mRNA processing anddecay.','Endonucleolytic cleavage to 5-phosphomonoester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.4','Ribonuclease H','Endoribonuclease H','NULL','Acts on RNA-DNA hybrids.','Endonucleolytic cleavage to 5-phosphomonoester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.5','Ribonuclease P','NULL','NULL','Essential for tRNA processing; generates 5-termini of mature tRNAmolecules.','Endonucleolytic cleavage of RNA, removing 5-extranucleotides from tRNAprecursor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.6','Ribonuclease IV','Endoribonuclease IV','NULL','Forms oligonucleotides with an average chain length of 10.','Endonucleolytic cleavage of poly(A) to fragments terminated by 3-hydroxyand 5-phosphate groups.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.7','Ribonuclease P4','NULL','NULL','NULL','Endonucleolytic cleavage of RNA, removing 3-extranucleotides from tRNAprecursor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.8','Ribonuclease M5','5S ribosomal maturation nuclease','NULL','Converts the 5S-rRNA precursor from Bacillus subtilis into 5S-rRNA,with 5-phosphate and 3-hydroxy groups.','Endonucleolytic cleavage of RNA, removing 21 and 42 nucleotides,respectively, from the 5- and 3-termini of a 5S-rRNA precursor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.9','Ribonuclease (poly-(U)-specific)','NULL','NULL','Forms oligonucleotides with chain length 6-12.','Endonucleolytic cleavage of poly(U) to fragments terminated by 3-hydroxyand 5-phosphate groups.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.10','Ribonuclease IX','NULL','NULL','Acts on poly(U) and poly(C), with a higher affinity for poly(C),but does not act on poly(A) or poly(G).','Endonucleolytic cleavage of poly(U) or poly(C) to fragments terminated by3-hydroxy and 5-phosphate groups.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.11','Ribonuclease Z','3 tRNase','NULL','No cofactor requirements.','Endonucleolytic cleavage of RNA, removing extra 3 nucleotides from tRNAprecursor, generating 3 termini of tRNAs. A 3-hydroxy group is left atthe tRNA terminus and a 5-phosphoryl group is left at the trailermolecule.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.12','Ribonuclease E','Endoribonuclease E','NULL','RNase E is a bacterial ribonuclease that plays a role in theprocessing of ribosomal RNA (9S to 5S rRNA), the chemical degradationof bulk cellular RNA, the decay of specific regulatory, messenger andstructural RNAs and the control of plasmid DNA replication.The enzyme binds to monophosphorylated 5 ends of substrates butexhibits sequential cleavages in the 3 to 5 direction.2-O-Methyl nucleotide substitutions at RNase E binding sites do notprevent binding but do prevent cleavage of non-modified targetsequences 5 to that locus.In Escherichia coli, the enzyme is found in the RNA degradosome.The C-terminal half of the protein contains binding sites for thethree other major degradosomal components, the DEAD-box RNA helicaseRh1B, EC 4.1.1.11 and EC 2.7.7.8.Formerly EC 3.1.26.n1.','Endonucleolytic cleavage of single-stranded RNA in A- and U-rich regions.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.13','Retroviral ribonuclease H','NULL','NULL','Retroviral reverse transcriptase is a multifunctional enzymeresponsible for viral replication.To perform this task the enzyme combines two distinct activities.The polymerase domain (EC 2.7.7.49) occupies the N-terminal two-thirds of the reverse transcriptase whereas the ribonuclease H domaincomprises the C-terminal remaining one-third.The RNase H domain of Moloney murine leukemia virus and Humanimmunodeficiency virus display two metal binding sites.','Endohydrolysis of RNA in RNA/DNA hybrids. Three different cleavage modes:1. sequence-specific internal cleavage of RNA. Human immunodeficiencyvirus type 1 and Moloney murine leukemia virus enzymes prefer to cleavethe RNA strand one nucleotide away from the RNA-DNA junction. 2. RNA5-end directed cleavage 13-19 nucleotides from the RNA end. 3. DNA3-end directed cleavage 15-20 nucleotides away from the primer terminus.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.26.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.1','Ribonuclease T(2)','Ribonuclease II','NULL','Formerly EC 2.7.7.17 and EC 3.1.4.23.','Two-stage endonucleolytic cleavage to nucleoside 3-phosphates and3-phosphooligonucleotides with 2,3-cyclic phosphate intermediates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.2','Bacillus subtilis ribonuclease','NULL','NULL','Similar enzymes: Azotobacter agilis RNase, Proteus mirabilis RNase.','Endonucleolytic cleavage to 2,3-cyclic nucleotides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.3','Ribonuclease T(1)','Aspergillus oryzae ribonuclease','NULL','Formerly EC 2.7.7.26 and EC 3.1.4.8.','Two-stage endonucleolytic cleavage to nucleoside 3-phosphates and3-phosphooligonucleotides ending in G-P with 2,3-cyclic phosphateintermediates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.4','Ribonuclease U(2)','Ribonuclease U2','NULL','NULL','Two-stage endonucleolytic cleavage to nucleoside 3-phosphates and3-phosphooligonucleotides ending in A-P or G-P with 2,3-cyclicphosphate intermediates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.5','Pancreatic ribonuclease','Pancreatic RNase','NULL','Similar enzymes: venom RNase, Thiobacillus thioparus RNase, Xenopuslaevis RNase, Rhizopus oligosporus RNase, ribonuclease M.Formerly EC 2.7.7.16 and EC 3.1.4.22.','Endonucleolytic cleavage to nucleoside 3-phosphates and3-phosphooligonucleotides ending in Cp or Up with 2,3-cyclic phosphateintermediates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.6','Enterobacter ribonuclease','NULL','NULL','Preference for cleavage at CpA.Homopolymers of A, U or G are not hydrolyzed.','Endonucleolytic cleavage to nucleoside 3-phosphates and3-phosphooligonucleotides with 2,3-cyclic phosphate intermediates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.7','Ribonuclease F','RNase F','NULL','Cleavage takes place between a cytosine and an adenine.','Endonucleolytic cleavage of RNA precursor into two, leaving 5-hydroxyand 3-phosphate groups.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.8','Ribonuclease V','Endoribonuclease V','NULL','Also hydrolyzes poly(U).','Hydrolysis of poly(A), forming oligoribonucleotides and ultimately3-AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.9','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.27.10','rRNA endonuclease','Alpha-sarcin','NULL','Also acts on bacterial rRNA.','Hydrolysis of the phosphodiester linkage between guanosine and adenosineresidues at one specific position in 28S rRNA from rat ribosomes.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.30.1','Aspergillus nuclease S(1)','Aspergillus nuclease S1','Zinc.','Preference for single-stranded substrate.Similar enzymes: Neurospora crassa nuclease, mung bean nuclease,Penicillium citrinum nuclease P1.Formerly EC 3.1.4.21.','Endonucleolytic cleavage to 5-phosphomononucleotide and5-phosphooligonucleotide end-products.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.30.2','Serratia marcescens nuclease','Endonuclease (Serratia marcescens)','NULL','Hydrolyzes double- or single-stranded substrate.Similar enzymes: silkworm nuclease, potato nuclease, Azotobacternuclease.Formerly EC 3.1.4.9.','Endonucleolytic cleavage to 5-phosphomononucleotide and5-phosphooligonucleotide end-products.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.1.31.1','Micrococcal nuclease','Micrococcal endonuclease','NULL','Hydrolyzes double- or single-stranded substrate.Similar enzymes: Chlamydomonas nuclease, spleen phosphodiesterase,spleen endonuclease.Formerly EC 3.1.4.7.','Endonucleolytic cleavage to nucleoside 3-phosphates and3-phosphooligonucleotide end-products.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.1','Alpha-amylase','Glycogenase','NULL','Acts on starch, glycogen and related polysaccharides andoligosaccharides in a random manner; reducing groups are liberated inthe alpha-configuration.The term alpha relates to the initial anomeric configuration of thefree sugar group released and not to the configuration of the linkagehydrolyzed.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.2','Beta-amylase','4-alpha-D-glucan maltohydrolase','NULL','Acts on starch, glycogen and related polysaccharides andoligosaccharides producing beta-maltose by an inversion.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.3','Glucan 1,4-alpha-glucosidase','4-alpha-D-glucan glucohydrolase','NULL','Most forms of the enzyme can rapidly hydrolyze 1,6-alpha-D-glucosidicbonds when the next bond in the sequence is 1,4, and somepreparations of this enzyme hydrolyze 1,6- and 1,3-alpha-D-glucosidicbonds in other polysaccharides.This entry covers all such enzymes acting on polysaccharides morerapidly than on oligosaccharides.EC 3.2.1.20 from mammalian intestine can catalyze similar reactions.','Hydrolysis of terminal (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.4','Cellulase','Avicelase','NULL','Will also hydrolyze 1,4-linkages in beta-D-glucans also containing1,3-linkages.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.5','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.6','Endo-1,3(4)-beta-glucanase','Endo-1,3-beta-glucanase','NULL','Substrates include laminarin, lichenin and cereal D-glucans.Different from EC 3.2.1.39 and EC 3.2.1.52.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.7','Inulinase','Inulase','NULL','NULL','Endohydrolysis of (2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.8','Endo-1,4-beta-xylanase','NULL','NULL','NULL','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.9','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.10','Oligo-1,6-glucosidase','Alpha-methylglucosidase','NULL','This enzyme, like EC 3.2.1.33, can release an alpha-1-','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.11','Dextranase','Alpha-1,6-glucan-6-glucanohydrolase','NULL','NULL','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.12','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.13','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.14','Chitinase','1,4-beta-poly-N-acetylglucosaminidase','NULL','Some chitinases also display the activity defined in EC 3.2.1.17.','Random hydrolysis of N-acetyl-beta-D-glucosaminide (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.15','Polygalacturonase','Pectin depolymerase','NULL','NULL','Random hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.16','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.17','Lysozyme','Muramidase','NULL','Cf. EC 3.2.1.14.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.18','Exo-alpha-sialidase','Alpha-neuraminidase','NULL','The enzyme does not act on 4-O-acetylated sialic acids.An endo-alpha-sialidase activity is listed as EC 3.2.1.129.See also EC 4.2.2.15.','Hydrolysis of alpha-(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.19','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.20','Alpha-glucosidase','Acid maltase','NULL','Group of enzymes whose specificity is directed mainly toward theexohydrolysis of 1,4-alpha-glucosidic linkages, and that hydrolyzeoligosaccharides rapidly, relative to polysaccharides, which arehydrolyzed relatively slowly, or not at all.The intestinal enzyme also hydrolyzes polysaccharides, catalyzing thereactions of EC 3.2.1.3, and, more slowly, hydrolyzes 1,6-alpha-D-glucose links.','Hydrolysis of terminal, non-reducing (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.21','Beta-glucosidase','Amygdalase','NULL','Wide specificity for beta-D-glucosides.Some examples also hydrolyze one or more of the following: beta-D-galactosides, alpha-L-arabinosides, beta-D-xylosides and beta-D-fucosides.','Hydrolysis of terminal, non-reducing beta-D-glucosyl residues withrelease of beta-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.22','Alpha-galactosidase','Melibiase','Magnesium; NAD(+).','Also hydrolyzes alpha-D-fucosides.','Hydrolysis of terminal, non-reducing alpha-D-galactose residues in alpha-D-galactosides, including galactose oligosaccharides, galactomannans andgalactolipids.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.23','Beta-galactosidase','Exo-(1-','NULL','Some enzymes in this group hydrolyze alpha-L-arabinosides; someanimal enzymes also hydrolyze beta-D-fucosides and beta-D-glucosides(cf. EC 3.2.1.108).','Hydrolysis of terminal non-reducing beta-D-galactose residues in beta-D-galactosides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.24','Alpha-mannosidase','NULL','NULL','Also hydrolyzes alpha-D-lyxosides and heptopyranosides with the sameconfiguration at C-2, C-3 and C-4 as mannose.','Hydrolysis of terminal, non-reducing alpha-D-mannose residues in alpha-D-mannosides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.25','Beta-mannosidase','Mannanase','NULL','NULL','Hydrolysis of terminal, non-reducing beta-D-mannose residues in beta-D-mannosides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.26','Beta-fructofuranosidase','Beta-fructosidase','NULL','Substrates include sucrose.Also catalyzes fructotransferase reactions.','Hydrolysis of terminal non-reducing beta-D-fructofuranoside residues inbeta-D-fructofuranosides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.27','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.28','Alpha,alpha-trehalase','Trehalase','NULL','The enzyme is an anomer-inverting glucosidase that catalyzes thehydrolysis of the alpha-glucosidic O-linkage of alpha,alpha-trehalose, releasing initially equimolar amounts of alpha- and beta-D-glucose.It is widely distributed in microorganisms, plants, invertebrates andvertebrates.','Alpha,alpha-trehalose + H(2)O = beta-D-glucose + alpha-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.29','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.30','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.31','Beta-glucuronidase','NULL','NULL','NULL','A beta-D-glucuronoside + H(2)O = D-glucuronate + an alcohol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.32','Endo-1,3-beta-xylanase','Endo-1,3-beta-xylosidase','NULL','This enzyme is found mostly in marine bacteria, which break down thebeta(1,3)-xylan found in the cell wall of some green and red algae.The enzyme produces mainly xylobiose, xylotriose and xylotetraose.','Random hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.33','Amylo-alpha-1,6-glucosidase','Amylo-1,6-glucosidase','NULL','This enzyme hydrolyzes an unsubstituted (1-','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.34','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.35','Hyaluronoglucosaminidase','Chondroitinase','NULL','Also hydrolyzes 1,4-beta-D-glycosidic linkages between N-acetyl-galactosamine or N-acetylgalactosamine sulfate and glucuronic acid inchondroitin, chondroitin 4- and 6-sulfates, and dermatan.Formerly EC 3.2.1.34.','Random hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.36','Hyaluronoglucuronidase','Hyaluronidase','NULL','NULL','Random hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.37','Xylan 1,4-beta-xylosidase','1,4-beta-D-xylan xylohydrolase','NULL','Also hydrolyzes xylobiose.Some other exoglycosidase activities have been found associated withthis enzyme in sheep liver.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.38','Beta-D-fucosidase','NULL','NULL','Enzymes from some sources also hydrolyze beta-D-galactosides and/orbeta-D-glucosides and/or alpha-L-arabinosides.The activity of EC 3.2.1.37 is an associated activity found in somesources (e.g. liver).','Hydrolysis of terminal non-reducing beta-D-fucose residues in beta-D-fucosides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.39','Glucan endo-1,3-beta-D-glucosidase','(1-','NULL','Very limited action on mixed-link (1-','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.40','Alpha-L-rhamnosidase','NULL','NULL','NULL','Hydrolysis of terminal non-reducing alpha-L-rhamnose residues in alpha-L-rhamnosides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.41','Pullulanase','Alpha-dextrin endo-1,6-alpha-glucosidase','NULL','Different from EC 3.2.1.142 in its action on glycogen, and its rateof hydrolysis of limit dextrins.Its action on amylopectin is complete.Maltose is the smallest sugar that it can release from an alpha-(1-','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.42','GDP-glucosidase','NULL','NULL','NULL','GDP-glucose + H(2)O = D-glucose + GDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.43','Beta-L-rhamnosidase','NULL','NULL','NULL','Hydrolysis of terminal, non-reducing beta-L-rhamnose residues in beta-L-rhamnosides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.44','Fucoidanase','NULL','NULL','NULL','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.45','Glucosylceramidase','Acid beta-glucosidase','NULL','Also acts on glucosylsphingosine (cf. EC 3.2.1.62).','D-glucosyl-N-acylsphingosine + H(2)O = D-glucose + N-acylsphingosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.46','Galactosylceramidase','Galactocerebrosidase','NULL','Cf. EC 3.2.1.62.','D-galactosyl-N-acylsphingosine + H(2)O = D-galactose + N-acylsphingosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.47','Galactosylgalactosylglucosylceramidase','NULL','NULL','NULL','Alpha-D-galactosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.48','Sucrose alpha-glucosidase','Sucrase','NULL','Isolated from intestinal mucosa as a single polypeptide chain alsodisplaying activity toward isomaltose (EC 3.2.1.10).','Hydrolysis of sucrose and maltose by an alpha-D-glucosidase-type action.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.49','Alpha-N-acetylgalactosaminidase','Alpha-galactosidase B','NULL','The human lysosomal enzyme is involved in the degradation of bloodtype A epitope.','Cleavage of non-reducing alpha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.50','Alpha-N-acetylglucosaminidase','N-acetyl-alpha-glucosaminidase','NULL','Hydrolyzes UDP-N-acetylglucosamine.','Hydrolysis of terminal non-reducing N-acetyl-D-glucosamine residues inN-acetyl-alpha-D-glucosaminides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.51','Alpha-L-fucosidase','Alpha-L-fucoside fucohydrolase','NULL','NULL','An alpha-L-fucoside + H(2)O = L-fucose + an alcohol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.52','Beta-N-acetylhexosaminidase','Beta-hexosaminidase','NULL','Acts on N-acetylglucosides and N-acetylgalactosides.Formerly EC 3.2.1.29 and EC 3.2.1.30.','Hydrolysis of terminal non-reducing N-acetyl-D-hexosamine residues inN-acetyl-beta-D-hexosaminides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.53','Beta-N-acetylgalactosaminidase','NULL','NULL','NULL','Hydrolysis of terminal non-reducing N-acetyl-D-galactosamine residues inN-acetyl-beta-D-galactosaminides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.54','Cyclomaltodextrinase','NULL','NULL','Also hydrolyzes linear maltodextrin.Formerly EC 3.2.1.12 and EC 3.2.1.13.','Cyclomaltodextrin + H(2)O = linear maltodextrin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.55','Non-reducing end alpha-L-arabinofuranosidase','Alpha-L-arabinofuranosidase','NULL','Acts on alpha-L-arabinofuranosides, alpha-L-arabinans containing(1,3)- and/or (1,5)-linkages, arabinoxylans and arabinogalactans.Some EC 3.2.1.23 and EC 3.2.1.38 enzymes also hydrolyze alpha-L-arabinosides.cf. EC 3.2.1.185.Formerly EC 3.2.1.79.','Hydrolysis of terminal non-reducing alpha-L-arabinofuranoside residues inalpha-L-arabinosides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.56','Glucuronosyl-disulfoglucosamine glucuronidase','NULL','NULL','NULL','3-D-glucuronosyl-N(2),6-disulfo-beta-D-glucosamine + H(2)O =D-glucuronate + N(2),6-disulfo-D-glucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.57','Isopullulanase','NULL','NULL','The enzyme has practically no action on starch.Panose (4-alpha-isomaltosylglucose) is hydrolyzed to isomaltose andglucose.Cf. EC 3.2.1.41 and EC 3.2.1.135.','Hydrolysis of pullulan to isopanose (6-alpha-maltosylglucose).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.58','Glucan 1,3-beta-glucosidase','Exo-1,3-beta-glucanase','NULL','Acts on oligosaccharides, but very slowly on laminaribiose.','Successive hydrolysis of beta-D-glucose units from the non-reducing endsof (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.59','Glucan endo-1,3-alpha-glucosidase','Endo-1,3-alpha-glucanase','NULL','Products from pseudonigeran (1,3-alpha-D-glucan) are nigerose andalpha-D-glucose.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.60','Glucan 1,4-alpha-maltotetraohydrolase','Exo-maltotetraohydrolase','NULL','Compare EC 3.2.1.2, which removes successive maltose residues,and EC 3.2.1.98 and EC 3.2.1.116.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.61','Mycodextranase','NULL','NULL','Products are nigerose and 4-alpha-D-nigerosylglucose.No hydrolysis of alpha-D-glucans containing only 1,3- or 1,4-bonds.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.62','Glycosylceramidase','Phloretin-glucosidase','NULL','Broad specificity (cf. EC 3.2.1.45 and EC 3.2.1.46).Also hydrolyzes phlorizin to phloretin and glucose.The intestinal enzyme is a complex which also catalyzes the reactionof EC 3.2.1.108.','Glycosyl-N-acylsphingosine + H(2)O = N-acylsphingosine + a sugar.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.63','1,2-alpha-L-fucosidase','Almond emulsin fucosidase II','NULL','Highly specific for non-reducing terminal L-fucose residues linked toD-galactose residues by 1,2-alpha-linkage.Not identical with EC 3.2.1.111.','Methyl-2-alpha-L-fucopyranosyl-beta-D-galactoside + H(2)O = L-fucose +methyl beta-D-galactoside.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.64','2,6-beta-fructan 6-levanbiohydrolase','2,6-beta-D-fructan 6-beta-D-fructofuranosylfructohydrolase','NULL','NULL','Hydrolysis of (2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.65','Levanase','NULL','NULL','NULL','Random hydrolysis of (2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.66','Quercitrinase','NULL','NULL','Quercitrin is quercetin 3-L-rhamnoside.','Quercitrin + H(2)O = L-rhamnose + quercetin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.67','Galacturan 1,4-alpha-galacturonidase','Exopolygalacturonase','NULL','NULL','((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.68','Isoamylase','Debranching enzyme','NULL','Also readily hydrolyzes amylopectin.Differs from EC 3.2.1.41 and EC 3.2.1.142 by its inability tohydrolyze pullulan, and by limited action on alpha-limit dextrins.Maltose is the smallest sugar it can release from an alpha-(1-','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.69','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.70','Glucan 1,6-alpha-glucosidase','Exo-1,6-alpha-glucosidase','NULL','Hydrolysis is accompanied by inversion at C-1, so that new reducingends are released in the beta-configuration.Dextrans and isomaltosaccharides are hydrolyzed, as is isomaltose,but very slowly.The enzyme from some sources also possesses the activity ofEC 3.2.1.59.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.71','Glucan endo-1,2-beta-glucosidase','Endo-1,2-beta-glucanase','NULL','NULL','Random hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.72','Xylan 1,3-beta-xylosidase','Exo-1,3-beta-xylosidase','NULL','NULL','Hydrolysis of successive xylose residues from the non-reducing termini of(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.73','Licheninase','1,3-1,4-beta-D-glucan 4-glucanohydrolase','NULL','Acts on lichenin and cereal beta-D-glucans, but not on beta-D-glucanscontaining only 1,3- or 1,4-bonds.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.74','Glucan 1,4-beta-glucosidase','1,4-beta-D-glucan glucohydrolase','NULL','Acts on 1,4-beta-D-glucans and related oligosaccharides.Cellobiose is hydrolyzed, but very slowly.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.75','Glucan endo-1,6-beta-glucosidase','Beta-1,6-glucanase','NULL','Acts on lutean, pustulan and 1,6-oligo-beta-D-glucosides.','Random hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.76','L-iduronidase','NULL','NULL','NULL','Hydrolysis of unsulfated alpha-L-iduronosidic linkages in dermatansulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.77','Mannan 1,2-(1,3)-alpha-mannosidase','Exo-1,2-1,3-alpha-mannosidase','NULL','A 1,6-alpha-D-mannan backbone remains after action on Saccharomycescerevisiae mannan.This is further attacked, but slowly.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.78','Mannan endo-1,4-beta-mannosidase','Beta-mannanase','NULL','NULL','Random hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.79','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.80','Fructan beta-fructosidase','Exo-beta-D-fructosidase','NULL','Hydrolyzes inulin and levan, and also sucrose.','Hydrolysis of terminal, non-reducing (2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.81','Beta-agarase','Agarase','NULL','Also acts on porphyran, but more slowly.Cleaves the beta-(1-','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.82','Exo-poly-alpha-galacturonosidase','Exopolygalacturanosidase','NULL','NULL','Hydrolysis of pectic acid from the non-reducing end, releasingdigalacturonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.83','Kappa-carrageenase','Kappa-carrageenan 4-beta-D-glycanohydrolase','NULL','The main products of hydrolysis are neocarrabiose-sulfate andneocarratetraose-sulfate.Unlike EC 3.2.1.157, but similar to EC 3.2.1.81, this enzyme proceedswith retention of the anomeric configuration.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.84','Glucan 1,3-alpha-glucosidase','Exo-1,3-alpha-glucanase','NULL','Does not act on nigeran.','Hydrolysis of terminal (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.85','6-phospho-beta-galactosidase','6-Phospho-beta-D-galactosidase','NULL','NULL','A 6-phospho-beta-D-galactoside + H(2)O = 6-phospho-D-galactose +an alcohol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.86','6-phospho-beta-glucosidase','Phospho-beta-glucosidase','NULL','Also hydrolyzes several other phospho-beta-D-glucosides, but nottheir non-phosphorylated forms.','6-phospho-beta-D-glucosyl-(1,4)-D-glucose + H(2)O = D-glucose + D-glucose6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.87','Capsular-polysaccharide endo-1,3-alpha-galactosidase','Polysaccharide depolymerase','NULL','Hydrolyzes the galactosyl-alpha-1,3-D-galactose linkages only in thecomplex substrate, bringing about depolymerization.','Random hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.88','Non-reducing end beta-L-arabinopyranosidase','Beta-L-arabinosidase','NULL','The enzyme, which was characterized from dormant seeds of the plantCajanus cajan (pigeon pea), has been shown to remove the terminalnon-reducing beta-L-arabinopyranoside residue from the artificialsubstrate p-nitrophenyl-beta-L-arabinopyranose.In the presence of methanol the enzyme demonstrates transglycosylaseactivity, transferring the arabinose moiety to methanol whileretaining the anomeric configuration, generating 1-O-methyl-beta-L-arabinopyranose.','Removal of a terminal beta-L-arabinopyranose residue from the non-reducing end of its substrate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.89','Arabinogalactan endo-beta-1,4-galactanase','Arabinogalactanase','NULL','This enzyme, isolated from the bacterium Bacillus subtilis,hydrolyzes the beta(1-','The enzyme specifically hydrolyzes (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.90','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.91','Cellulose 1,4-beta-cellobiosidase (non-reducing end)','1,4-beta-cellobiohydrolase','NULL','NULL','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.92','Peptidoglycan beta-N-acetylmuramidase','Exo-beta-N-acetylmuramidase','NULL','NULL','Hydrolysis of terminal, non-reducing N-acetylmuramic residues.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.93','Alpha,alpha-phosphotrehalase','Trehalose-6-phosphate hydrolase','NULL','NULL','Alpha,alpha-trehalose 6-phosphate + H(2)O = D-glucose + D-glucose6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.94','Glucan 1,6-alpha-isomaltosidase','Exo-isomaltohydrolase','NULL','Optimum activity is on those 1,6-alpha-D-glucans containing 6, 7 and8 glucose units; those containing 3, 4 and 5 glucose units arehydrolyzed at slower rates.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.95','Dextran 1,6-alpha-isomaltotriosidase','Exo-isomaltotriohydrolase','NULL','NULL','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.96','Mannosyl-glycoprotein endo-beta-N-acetylglucosaminidase','Di-N-acetylchitobiosyl beta-N-acetylglucosaminidase','NULL','NULL','Endohydrolysis of the N,N-diacetylchitobiosyl unit in high-mannoseglycopeptides and glycoproteins containing the -(Man(GlcNAc)(2))Asn-structure. One N-acetyl-D-glucosamine residue remains attached to theprotein; the rest of the oligosaccharide is released intact.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.97','Endo-alpha-N-acetylgalactosaminidase','D-galactosyl-3-(N-acetyl-alpha-D-galactosaminyl)-L-serinemucinaminohydrolase','NULL','The enzyme catalyzes the liberation of Gal-(1-','3-O-beta-D-galactosyl-N-acetyl-alpha-D-galactosaminyl-L-serine-[protein]+ H(2)O = 3-O-beta-D-galactosyl-N-acetyl-alpha-D-galactosamine +L-serine-[protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.98','Glucan 1,4-alpha-maltohexaosidase','Exo-maltohexaohydrolase','NULL','Cf. EC 3.2.1.3, which removes successive glucose residues.Cf. EC 3.2.1.2, which removes successive maltose residues.Cf. EC 3.2.1.116, which removes successive maltotriose residues.Cf. EC 3.2.1.60, which removes successive maltotetraose residues.The products have the alpha-configuration.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.99','Arabinan endo-1,5-alpha-L-arabinosidase','Endo-1,5-alpha-L-arabinanase','NULL','Acts best on linear 1,5-alpha-L-arabinan.Also acts on branched arabinan, but more slowly.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.100','Mannan 1,4-mannobiosidase','Exo-1,4-beta-mannobiohydrolase','NULL','NULL','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.101','Mannan endo-1,6-alpha-mannosidase','1,6-beta-D-mannan mannanohydrolase','NULL','NULL','Random hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.102','Blood-group-substance endo-1,4-beta-galactosidase','Endo-beta-galactosidase','NULL','Hydrolyzes the 1,4-beta-D-galactosyl linkages adjacent to a 1,3-alpha-D-galactosyl or N-acetylgalactosaminyl residues and a 1,2-alpha-D-fucosyl residue.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.103','Keratan-sulfate endo-1,4-beta-galactosidase','Endo-beta-galactosidase','NULL','Hydrolyzes the 1,4-beta-D-galactosyl linkages adjacent to 1,3-N-acetyl-alpha-D-glucosaminyl residues.Also acts on some non-sulfated oligosaccharides, but only acts onblood group substances when the 1,2-linked fucosyl residues have beenremoved (cf. EC 3.2.1.102).','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.104','Steryl-beta-glucosidase','NULL','NULL','Acts on glucosides of cholesterol and sitosterol, but not on somerelated sterols such as coprostanol.','Cholesteryl-beta-D-glucoside + H(2)O = D-glucose + cholesterol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.105','3-alpha-(S)-strictosidine beta-glucosidase','Strictosidine beta-glucosidase','NULL','Does not act on a number of closely related glycosides.Strictosidine is a precursor of indole alkaloids.','Strictosidine + H(2)O = D-glucose + strictosidine aglycone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.106','Mannosyl-oligosaccharide glucosidase','Processing A-glucosidase I','NULL','Also acts, more slowly, on the corresponding glycolipids andglycopeptides.Involved in the formation of high-mannose and complex glycoproteins.','Exohydrolysis of the non-reducing terminal glucose residues in themannosyl-oligosaccharide Glc(3)Man(9)GlcNAc(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.107','Protein-glucosylgalactosylhydroxylysine glucosidase','NULL','NULL','Requires free, positively charged epsilon-amino group ofhydroxylysine.','Protein alpha-D-glucosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.108','Lactase','NULL','NULL','The enzyme from intestinal mucosa is isolated as a complex which alsocatalyzes the reaction of EC 3.2.1.62 (cf. EC 3.2.1.23).','Lactose + H(2)O = D-galactose + D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.109','Endogalactosaminidase','NULL','NULL','NULL','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.110','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.111','1,3-alpha-L-fucosidase','Almond emulsin fucosidase I','NULL','Not identical with EC 3.2.1.63.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.112','2-deoxyglucosidase','NULL','NULL','NULL','A 2-deoxy-alpha-D-glucoside + H(2)O = 2-deoxy-D-glucose + an alcohol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.113','Mannosyl-oligosaccharide 1,2-alpha-mannosidase','Man9-mannosidase','NULL','Involved in the synthesis of glycoproteins.','Hydrolysis of the terminal (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.114','Mannosyl-oligosaccharide 1,3-1,6-alpha-mannosidase','ManII','NULL','Involved in the synthesis of glycoproteins.','Hydrolysis of the terminal (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.115','Branched-dextran exo-1,2-alpha-glucosidase','Dextran alpha-1,2 debranching enzyme','NULL','Does not hydrolyze disaccharides or oligosaccharides containinglinear 1,2-alpha-glucosidic linkages.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.116','Glucan 1,4-alpha-maltotriohydrolase','Exo-maltotriohydrolase','NULL','Cf. EC 3.2.1.2, EC 3.2.1.60, and EC 3.2.1.98.The products of all these enzymes have the alpha-configuration.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.117','Amygdalin beta-glucosidase','NULL','NULL','Highly specific; does not act on prunasin, linamarin, gentiobiose orcellobiose (cf. EC 3.2.1.21).','(R)-amygdalin + H(2)O = (R)-prunasin + D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.118','Prunasin beta-glucosidase','NULL','NULL','Highly specific; does not act on amygdalin, linamarin or gentiobiose(cf. EC 3.2.1.21).','(R)-prunasin + H(2)O = D-glucose + mandelonitrile.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.119','Vicianin beta-glucosidase','NULL','NULL','Also hydrolyzes, more slowly, (R)-amygdalin and (R)-prunasin, but notgentiobiose, linamarin or cellobiose.','(R)-vicianin + H(2)O = mandelonitrile + vicianose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.120','Oligoxyloglucan beta-glycosidase','Isoprimeverose-producing oligoxyloglucan hydrolase','NULL','NULL','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.121','Polymannuronate hydrolase','NULL','NULL','Does not act on alginic acid, which is a copolymer ofpolymannuronate.','Endohydrolysis of the D-mannuronide linkages of polymannuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.122','Maltose-6-phosphate glucosidase','Phospho-alpha-glucosidase','NULL','Hydrolyzes a variety of 6-phospho-alpha-D-glucosides, includingalpha-maltose 6-phosphate, alpha,alpha-trehalose 6-phosphate, sucrose6-phosphate and p-nitrophenyl-alpha-D-glucopyranoside 6-phosphate (asa chromogenic substrate).The enzyme is activated by Fe(2+), Mn(2+), Co(2+) and Ni(2+).It is rapidly inactivated in air.','Alpha-maltose 6-phosphate + H(2)O = D-glucose + D-glucose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.123','Endoglycosylceramidase','Endo-glucosylceramidase','NULL','An enzyme from Rhodococcus sp. which degrades various acidic andneutral glycosphingolipids to oligosaccharides and ceramides,by cleaving a glucosyl bond.Does not act on monoglycosylceramides (cf. EC 3.2.1.62).','Oligoglycosylglucosyl-(1', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.124','3-deoxy-2-octulosonidase','NULL','NULL','The enzyme from a bacteriophage catalyzes the depolymerization ofcapsular polysaccharides containing 3-deoxy-2-octulosonide in thecell wall of Escherichia coli.','Endohydrolysis of the beta-ketopyranosidic linkages of 3-deoxy-D-manno-2-octulosonate in capsular polysaccharides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.125','Raucaffricine beta-glucosidase','NULL','NULL','Highly specific; some other ajmalan glucoside alkaloids arehydrolyzed, more slowly.','Raucaffricine + H(2)O = D-glucose + vomilenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.126','Coniferin beta-glucosidase','NULL','NULL','Also hydrolyzes syringin, 4-cinnamyl alcohol beta-glucoside and, moreslowly, some other aryl beta-glycosides.A plant cell-wall enzyme involved in the biosynthesis of lignin.','Coniferin + H(2)O = D-glucose + coniferol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.127','1,6-alpha-L-fucosidase','NULL','NULL','The enzyme from Aspergillus niger does not act on 1,2-, 1,3-, or 1,4-L-fucosyl linkages.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.128','Glycyrrhizinate beta-glucuronidase','NULL','NULL','The enzyme from Aspergillus niger is specific for the hydrolysis ofthe triterpenoid glycoside glycyrrhizinate from roots of Glycyrrhizasp.','Glycyrrhizinate + H(2)O = (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.129','Endo-alpha-sialidase','Endo-N-acetylneuraminidase','NULL','Although the name endo-N-acetylneuraminidase has also been used forthis enzyme, this is misleading since its activity is not restrictedto acetyl-substituted substrates.An exo-alpha-sialidase activity is listed as EC 3.2.1.18.See also 4.2.2.15.','Endohydrolysis of (2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.130','Glycoprotein endo-alpha-1,2-mannosidase','NULL','NULL','Involved in the synthesis of glycoproteins.','Hydrolysis of the terminal alpha-D-glucosyl-(1,3)-D-mannosyl unit fromthe GlcMan(9)(GlcNAc)(2) oligosaccharide component of the glycoproteinproduced in the Golgi membrane.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.131','Xylan alpha-1,2-glucuronosidase','NULL','NULL','NULL','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.132','Chitosanase','NULL','NULL','A whole spectrum of chitosanases are now known that can hydrolyzevarious types of links in chitosan.The only constant property is the endohydrolysis of GlcN-GlcN links,which is common to all known chitosanases.One known chitosanase is limited to this link recognition, while themajority can also recognize GlcN-GlcNAc links or GlcNAc-GlcN linksbut not both.They also do not recognize GlcNAc-GlcNAc links in partly acetylatedchitosan.','Endohydrolysis of beta-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.133','Glucan 1,4-alpha-maltohydrolase','Maltogenic alpha-amylase','NULL','Acts on starch and related polysaccharides and oligosaccharides.The product is alpha-maltose; cf. EC 3.2.1.2.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.134','Difructose-anhydride synthase','NULL','NULL','Produces difructose anhydride by the reverse reaction of partialhydrolysis, forming an alpha-fructosidic linkage.','Bis-D-fructose 2,1:2,1-dianhydride + H(2)O = inulobiose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.135','Neopullulanase','NULL','NULL','Cf. EC 3.2.1.41 and EC 3.2.1.57.','Hydrolysis of pullulan to panose (6-alpha-D-glucosylmaltose).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.136','Glucuronoarabinoxylan endo-1,4-beta-xylanase','Feraxan endoxylanase','NULL','High activity toward feruloylated arabinoxylans from cereal plantcell walls.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.137','Mannan exo-1,2-1,6-alpha-mannosidase','NULL','NULL','Mannose residues linked alpha-D-1,3- are also released very slowly.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.138','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.139','Alpha-glucuronidase','Alpha-glucosiduronase','NULL','Considerable differences in the specificities of the enzymes fromdifferent fungi for alpha-D-glucosiduronates have been reported.','An alpha-D-glucuronoside + H(2)O = an alcohol + D-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.140','Lacto-N-biosidase','Oligosaccharide lacto-N-biosylhydrolase','NULL','The enzyme from Streptomyces specifically hydrolyzes the terminallacto-N-biosyl residue (beta-D-Gal-(1-','Beta-D-Gal-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.141','4-alpha-D-((1-','Malto-oligosyltrehalose trehalohydrolase','NULL','NULL','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.142','Limit dextrinase','Amylopectin-1,6-glucosidase','NULL','Plant enzymes with little or no action on glycogen.Action on amylopectin is incomplete, but action on alpha-limitdextrins is complete.Maltose is the smallest sugar it can release from an alpha-(1-','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.143','Poly(ADP-ribose) glycohydrolase','NULL','NULL','Specific to (1-2) linkage of ribose-ribose bond of poly(ADP-D-ribose).','Hydrolyzes poly(ADP-D-ribose) at glycosidic (1-2) linkage of ribose-ribose bond to produce free ADP-D-ribose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.144','3-deoxyoctulosonase','Alpha-Kdo-ase','NULL','Releases Kdo (alpha- and beta-linked 3-deoxy-D-manno-octulosonicacid) from different lipopolysaccharides, including Re-LPS fromEscherichia coli and Salmonella, Rd-LPS from S.minnesota, and de-O-acyl-re-LPS.4-methylumbelliferyl-alpha-Kdo (alpha-Kdo-OMec) is also a substrate.','3-deoxyoctulosonyl-lipopolysaccharide + H(2)O = 3-deoxyoctulosonic acid +lipopolysaccharide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.145','Galactan 1,3-beta-galactosidase','NULL','NULL','This enzyme removes not only free galactose, but also 6-glycosylatedresidues, e.g., (1-','Hydrolysis of terminal, non-reducing beta-D-galactose residues in (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.146','Beta-galactofuranosidase','Beta-D-galactofuranosidase','NULL','The enzyme from Helminthosporium sacchari detoxifieshelminthosporoside, a bis(digalactosyl)terpene produced by thisfungus, by releasing its four molecules of bound galactose.','Hydrolysis of terminal non-reducing beta-D-galactofuranosides, releasinggalactose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.147','Thioglucosidase','Myrosinase','NULL','Has a wide specificity for thioglycosides.Formerly EC 3.2.3.1.','A thioglucoside + H(2)O = a sugar + a thiol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.148','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.149','Beta-primeverosidase','NULL','NULL','The enzyme is responsible for the formation of the alcoholic aroma inoolong and black tea.In addition to beta-primeverosides (i.e. 6-O-(beta-D-xylopyranosyl)-beta-D-glucopyranosides), it also hydrolyzes 6-O-(beta-D-apiofuranosyl)-beta-D-glucopyranosides and, less rapidly, beta-vicianosides and 6-O-(alpha-L-arabinofuranosyl)-beta-D-glucopyranosides, but not beta-glucosides.Geranyl-, linaloyl-, benzyl- and p-nitrophenol glycosides are allhydrolyzed.','A 6-O-(beta-D-xylopyranosyl)-beta-D-glucopyranoside + H(2)O = 6-O-(beta-D-xylopyranosyl)-beta-D-glucopyranose + an alcohol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.150','Oligoxyloglucan reducing-end-specific cellobiohydrolase','NULL','NULL','The enzyme is found in the fungus Geotrichum sp. M128.The substrate is a hemicellulose found in plant cell walls.','Hydrolysis of cellobiose from the reducing end of xyloglucans consistingof a beta-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.151','Xyloglucan-specific endo-beta-1,4-glucanase','1,4-beta-D-glucan glucanohydrolase','NULL','The enzyme from Aspergillus aculeatus is specific for xyloglucan anddoes not hydrolyze other cell-wall components.The reaction involves endohydrolysis of 1,4-beta-D-glucosidiclinkages in xyloglucan with retention of the beta-configuration ofthe glycosyl residues.','Xyloglucan + H(2)O = xyloglucan oligosaccharides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.152','Mannosylglycoprotein endo-beta-mannosidase','Endo-beta-mannosidase','NULL','The substrate group is a substituent on N-4 of an asparagine residuein the glycoprotein.The mannose residue at the non-reducing end of the sequence may carryfurther alpha-D-mannosyl groups on O-3 or O-6, but such a substituenton O-3 of the beta-D-mannosyl group prevents the action of theenzyme.The enzyme was obtained from the lily, Lilium longiflorum.','Hydrolysis of the alpha-D-mannosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.153','Fructan beta-(2,1)-fructosidase','1-FEH II','NULL','Possesses one of the activities of EC 3.2.1.80.While the best substrates are the inulin-type fructans, such as1-kestose (beta-D-fructofuranosyl-(2-','Hydrolysis of terminal, non-reducing (2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.154','Fructan beta-(2,6)-fructosidase','6-FEH','NULL','Possesses one of the activities of EC 3.2.1.80.While the best substrates are the levan-type fructans such as6-kestotriose (beta-D-fructofuranosyl-(2-','Hydrolysis of terminal, non-reducing (2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.155','Xyloglucan-specific exo-beta-1,4-glucanase','NULL','NULL','The enzyme removes XXXG heptasaccharides, XXLG/XLXG octasaccharidesand XLLG nonasaccharides from the end of tamarind seed xyloglucanpolymers in a processive manner.Hydrolysis occurs at the unsubstituted D-glucopyranose residue in themain backbone.It is not known whether the cleavage takes place at the reducing ornon-reducing end of the polymer.Very low activity with beta-D-glucans.The enzyme from Chrysosporium lucknowense shifts to an endoglucanasemode when acting on linear substrates without bulky substituents onthe polymeric backbone such as barley beta-glucan.Formerly EC 3.2.1.160.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.156','Oligosaccharide reducing-end xylanase','Reducing end xylose-releasing exo-oligoxylanase','NULL','The enzyme, originally isolated from the bacterium Bacillushalodurans C-125, releases the xylose unit at the reducing end ofoligosaccharides ending with the structure beta-D-xylopyranosyl-(1-','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.157','Iota-carrageenase','NULL','NULL','The main products of hydrolysis are iota-neocarratetraose sulfate andiota-neocarrahexaose sulfate.Iota-neocarraoctaose is the shortest substrate oligomer that can becleaved.Unlike EC 3.2.1.81 and EC 3.2.1.83, this enzyme proceeds withinversion of the anomeric configuration.Iota-carrageenan differs from kappa-carrageenan by possessing a sulfogroup on O-2 of the 3,6-anhydro-D-galactose residues, in addition tothat present in the kappa-compound on O-4 of the D-galactoseresidues.','Endohydrolysis of 1,4-beta-D-linkages between D-galactose 4-sulfate and3,6-anhydro-D-galactose-2-sulfate in iota-carrageenans.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.158','Alpha-agarase','Agarase A33','Calcium.','The enzyme from Thalassomonas sp. can use agarose, agarohexaose andneoagarohexaose as substrate.The products of agarohexaose hydrolysis are dimers and tetramers,with agarotetraose being the predominant product, whereas hydrolysisof neoagarohexaose gives rise to two types of trimer.While the enzyme can also hydrolyze the highly sulfated agaroseporphyran very efficiently, it cannot hydrolyze kappa-carrageenan andiota-carrageenan.','Endohydrolysis of 1,3-alpha-L-galactosidic linkages in agarose, yieldingagarotetraose as the major product.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.159','Alpha-neoagaro-oligosaccharide hydrolase','Alpha-NAOS hydrolase','NULL','When neoagarohexaose is used as a substrate, the oligosaccharide iscleaved at the non-reducing end to produce 3,6-anhydro-L-galactoseand agaropentaose, which is further hydrolyzed to agarobiose andagarotriose.With neoagarotetraose as substrate, the products are predominantlyagarotriose and 3,6-anhydro-L-galactose.In Vibrio sp. the actions of EC 3.2.1.81 and EC 3.2.1.159 can be usedto degrade agarose to 3,6-anhydro-L-galactose and D-galactose.','Hydrolysis of the 1,3-alpha-L-galactosidic linkages of neoagaro-oligosaccharides that are smaller than a hexamer, yielding 3,6-anhydro-L-galactose and D-galactose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.160','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.161','Beta-apiosyl-beta-glucosidase','Furcatin hydrolase','NULL','Removes disaccharides from the natural substrates dalpatein 7-O-beta-D-apiofuranosyl-(1-','7-(beta-D-apiofuranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.162','Lambda-carrageenase','Endo-beta-1,4-carrageenose 2,6,2-trisulfate-hydrolase','NULL','The enzyme from Pseudoalteromonas sp. is specific for lambda-carrageenan.Iota-carrageenan (see EC 3.2.1.157), kappa-carrageenan (seeEC 3.2.1.83), agarose and porphyran are not substrates.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.163','1,6-alpha-D-mannosidase','NULL','NULL','Specific for (1-','Hydrolysis of the 1,6-linked alpha-D-mannose residues in alpha-D-Manp-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.164','Galactan endo-1,6-beta-galactosidase','Endo-1,6-beta-galactanase','NULL','Specifically hydrolyzes 1,6-beta-D-galactooligosaccharides with adegree of polymerization (DP) higher than 3, and their acidicderivatives with 4-O-methylglucosyluronate or glucosyluronate groupsat the non-reducing terminals.1,3-beta-D- and 1,4-beta-D-galactosyl residues cannot act assubstrates.Can also hydrolyze alpha-L-arabinofuranosidase-treatedarabinogalactan protein (AGP) extracted from radish roots.AGPs are thought to be involved in many physiological events, such ascell division, cell expansion and cell death.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.165','Exo-1,4-beta-D-glucosaminidase','Exo-beta-D-glucosaminidase','NULL','Chitosan is a partially or totally N-deacetylated chitin derivativethat is found in the cell walls of some phytopathogenic fungi andcomprises D-glucosamine residues with a variable content of GlcNAcresidues.Acts specifically on chitooligosaccharides and chitosan, havingmaximal activity on chitotetraose, chitopentaose and theircorresponding alcohols.Can degrade GlcN-GlcNAc but not GlcNAc-GlcNAc.','Hydrolysis of chitosan or chitosan oligosaccharides to remove successiveD-glucosamine residues from the non-reducing termini.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.166','Heparanase','Hpa1 heparanase','NULL','Heparanase cleaves the linkage between a glucuronic acid unit and anN-sulfo glucosamine unit carrying either a 3-O-sulfo or a 6-O-sulfogroup.Heparanase-1 cuts macromolecular heparin into fragments of 5000-20000Da.The enzyme cleaves the heparan sulfate glycosaminoglycans fromproteoglycan core proteins and degrades them to smalloligosaccharides.Inside cells, the enzyme is important for the normal catabolism ofheparan sulfate proteoglycans, generating glycosaminoglycan fragmentsthat are then transported to lysosomes and completely degraded.When secreted, heparanase degrades basement membrane heparan sulfateglycosaminoglycans at sites of injury or inflammation, allowingextravasion of immune cells into nonvascular spaces and releasingfactors that regulate cell proliferation and angiogenesis.','Endohydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.167','Baicalin-beta-D-glucuronidase','Baicalinase','NULL','The enzyme also hydrolyzes wogonin 7-O-beta-D-glucuronide andoroxylin 7-O-beta-D-glucuronide with lower efficiency.Neglegible activity with p-nitrophenyl-beta-D-glucuronide.','Baicalin + H(2)O = baicalein + D-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.168','Hesperidin 6-O-alpha-L-rhamnosyl-beta-D-glucosidase','NULL','NULL','The enzyme exhibits high specificity toward 7-O-linked flavonoidbeta-rutinosides.','Hesperidin + H(2)O = hesperetin + rutinose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.169','Protein O-GlcNAcase','Glycoside hydrolase O-GlcNAcase','NULL','Within higher eukaryotes post-translational modification of proteinserines/threonines with N-acetylglucosamine (O-GlcNAc) is dynamic,inducible and abundant, regulating many cellular processes byinterfering with protein phosphorylation.EC 2.4.1.155 transfers GlcNAc onto substrate proteins andEC 3.2.1.169 cleaves GlcNAc from the modified proteins.','(1) [Protein]-3-O-(N-acetyl-D-glucosaminyl)-L-serine + H(2)O = [protein]-L-serine + N-acetyl-D-glucosamine.(2) [Protein]-3-O-(N-acetyl-D-glucosaminyl)-L-threonine + H(2)O =[protein]-L-threonine + N-acetyl-D-glucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.170','Mannosylglycerate hydrolase','2-O-(6-phospho-mannosyl)-D-glycerate hydrolase','NULL','The enzyme participates in the mannosylglycerate degradation pathwayof some bacteria.Mannosylglycerate is phosphorylated during transport into the cell,and the phosphorylated form is hydrolyzed by this enzyme.','2-O-(6-phospho-alpha-D-mannosyl)-D-glycerate + H(2)O = D-mannose6-phosphate + D-glycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.171','Rhamnogalacturonan hydrolase','RG-hydrolase','NULL','The enzyme is part of the degradation system for rhamnogalacturonan Iin Aspergillus aculeatus.','Endohydrolysis of alpha-D-GalA-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.172','Unsaturated rhamnogalacturonyl hydrolase','NULL','NULL','The enzyme is part of the degradation system for rhamnogalacturonan Iin Bacillus subtilis strain 168.','2-O-(4-deoxy-beta-L-threo-hex-4-enopyranuronosyl)-alpha-L-rhamnopyranose+ H(2)O = 5-dehydro-4-deoxy-D-glucuronate + L-rhamnopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.173','Rhamnogalacturonan galacturonohydrolase','RG-galacturonohydrolase','NULL','The enzyme is part of the degradation system for rhamnogalacturonan Iin Aspergillus aculeatus.','Exohydrolysis of the alpha-D-GalA-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.174','Rhamnogalacturonan rhamnohydrolase','RG alpha-L-rhamnopyranohydrolase','NULL','The enzyme is part of the degradation system for rhamnogalacturonan Iin Aspergillus aculeatus.','Exohydrolysis of the alpha-L-Rha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.175','Beta-D-glucopyranosyl abscisate beta-glucosidase','ABA-beta-D-glucosidase','NULL','The enzyme hydrolzes the biologically inactive beta-D-glucopyranosylester of abscisic acid to produce active abscisate.Abscisate is a phytohormone critical for plant growth, developmentand adaption to various stress conditions.The enzyme does not hydrolyze beta-D-glucopyranosyl zeatin.','D-glucopyranosyl abscisate + H(2)O = D-glucose + abscisate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.176','Cellulose 1,4-beta-cellobiosidase (reducing end)','Cellulase SS','NULL','Some exocellulases, most of which belong to the glycoside hydrolasefamily 48 (GH48), act at the reducing ends of cellulose and similarsubstrates.The CelS enzyme from Clostridium thermocellum is the most abundantsubunit of the cellulosome formed by the organism.It liberates cellobiose units from the reducing end by hydrolysis ofthe glycosidic bond, employing an inverting reaction mechanism.Different from EC 3.2.1.91, which attacks cellulose from the non-reducing end.','Hydrolysis of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.177','Alpha-D-xyloside xylohydrolase','Alpha-xylosidase','NULL','The enzyme catalyzes hydrolysis of a terminal, unsubstituted xylosideat the extreme reducing end of a xylogluco-oligosaccharide.Representative alpha-xylosidases from glycoside hydrolase family 31utilize a two-step (double-displacement) mechanism involving acovalent glycosyl-enzyme intermediate, and retain the anomericconfiguration of the product.','Hydrolysis of terminal, non-reducing alpha-D-xylose residues with releaseof alpha-D-xylose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.178','Beta-porphyranase','Endo-beta-porphyranase','NULL','The backbone of porphyran consists largely (approximately 70%) of(1-','Hydrolysis of beta-D-galactopyranose-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.179','Gellan tetrasaccharide unsaturated glucuronyl hydrolase','Unsaturated glucuronyl hydrolase','NULL','The enzyme releases 4-deoxy-4(5)-unsaturated D-glucuronic acid fromoligosaccharides produced by polysaccharide lyases, e.g. thetetrasaccharide beta-D-4-deoxy-Delta(4)-GlcAp-(1-','Beta-D-4-deoxy-Delta(4)-GlcAp-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.180','Unsaturated chondroitin disaccharide hydrolase','Unsaturated glucuronyl hydrolase','NULL','The enzyme releases 4-deoxy-4,5-didehydro D-glucuronic acid or4-deoxy-4,5-didehydro L-iduronic acid from chondroitin disaccharides,hyaluronan disaccharides and heparin disaccharides and cleaves bothglycosidic (1-','Beta-D-4-deoxy-Delta(4)-GlcAp-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.181','Galactan endo-beta-1,3-galactanase','Endo-beta-1,3-galactanase','NULL','The enzyme from the fungus Flammulina velutipes (winter mushroom)hydrolyzes the beta(1-','The enzyme specifically hydrolyzes beta-1,3-galactan and beta-1,3-galactooligosaccharides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.182','4-hydroxy-7-methoxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl glucosidebeta-D-glucosidase','DIMBOA glucosidase','NULL','The enzyme from Triticum aestivum (wheat) has a higher affinity forDIMBOA glucoside than DIBOA glucoside.With Secale cereale (rye) the preference is reversed.','(1) (2R)-4-hydroxy-7-methoxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-ylbeta-D-glucopyranoside + H(2)O = 2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one + D-glucose.(2) (2R)-4-hydroxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside + H(2)O = 2,4-dihydroxy-2H-1,4-benzoxazin-3(4H)-one +D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.183','UDP-N-acetylglucosamine 2-epimerase (hydrolyzing)','UDP-N-acetylglucosamine 2-epimerase','NULL','The enzyme is found in mammalian liver, as well as in some pathogenicbacteria including Neisseria meningitidis and Staphylococcus aureus.It catalyzes the first step of sialic acid (N-acetylneuraminic acid)biosynthesis.The initial product formed is the alpha anomer, which rapidlymutarotates to a mixture of anomers.The mammalian enzyme is bifunctional and also catalyzes EC 2.7.1.60.Cf. EC 5.1.3.14.','UDP-N-acetyl-alpha-D-glucosamine + H(2)O = N-acetyl-D-mannosamine + UDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.184','UDP-N,N-diacetylbacillosamine 2-epimerase (hydrolyzing)','UDP-Bac2Ac(4)Ac 2-epimerase','Magnesium.','Involved in biosynthesis of legionaminic acid, a nonulosonatederivative that is incorporated by some bacteria into assortedvirulence-associated cell surface glycoconjugates.The initial product formed by the enzyme from Legionella pneumophila,which incorporates legionaminic acid into the O-antigen moiety of itslipopolysaccharide, is 2,4-diacetamido-2,4,6-trideoxy-alpha-D-mannopyranose, which rapidly mutarotates to a mixture of anomers.The enzyme from Campylobacter jejuni, which incorporates legionaminicacid into flagellin, prefers GDP-N,N-diacetylbacillosamine.','UDP-N,N-diacetylbacillosamine + H(2)O = UDP + 2,4-diacetamido-2,4,6-trideoxy-D-mannopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.185','Non-reducing end beta-L-arabinofuranosidase','NULL','NULL','The enzyme, which was identified in the bacterium Bifidobacteriumlongum JCM1217, removes the beta-L-arabinofuranose residue from thenon-reducing end of multiple substrates, including beta-L-arabinofuranosyl-hydroxyproline (Ara-Hyp), Ara(2)-Hyp, Ara(3)-Hyp,and beta-L-arabinofuranosyl-(1-','Beta-L-arabinofuranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.186','Protodioscin 26-O-beta-D-glucosidase','Furostanol 26-O-beta-D-glucoside glucohydrolase','NULL','The enzyme has been characterized from the plants Cheilocostusspeciosus and Solanum torvum.It also hydrolyzes the 26-beta-D-glucose group from related steroidglucosides such as protogracillin, torvoside A and torvoside H.','Protodioscin + H(2)O = 26-deglucoprotodiscin + D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.187','(Ara-f)(3)-Hyp beta-L-arabinobiosidase','Beta-L-arabinobiosidase','NULL','The enzyme, which was identified in the bacterium Bifidobacteriumlongum JCM1217, is specific for (Ara-f)(3)-Hyp, a sugar chain foundin hydroxyproline-rich glyoproteins such as extensin and lectin.The enzyme was not able to accept (Ara-f)(2)-Hyp or (Ara-f)(4)-Hyp assubstrates.In the presence of 1-alkanols, the enzyme demonstratestransglycosylation activity, retaining the anomeric configuration ofthe arabinofuranose residue.','4-O-(beta-L-arabinofuranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.188','Avenacosidase','NULL','NULL','Isolated from oat (Avena sativa) seedlings.The product acts as a defense system against fungal infection.Also acts on avenacoside A.','Avenacoside B + H(2)O = 26-desgluco-avenacoside B + D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.189','Dioscin glycosidase (diosgenin-forming)','Dioscin glycosidase (aglycone-forming)','NULL','The enzyme is involved in degradation of the steroid saponin dioscinby some fungi of the Absidia genus.The enzyme can also hydrolyze 3-O-[alpha-L-Ara-(1-','3-O-[alpha-L-Rha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.190','Dioscin glycosidase (3-O-beta-D-Glc-diosgenin-forming)','Dioscin-alpha-L-rhamnosidase','NULL','The enzyme is involved in the hydrolysis of the steroid saponindioscin by the digestive system of Sus scrofa (pig).Cf. EC 3.2.1.189.','3-O-[alpha-L-Rha-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.191','Ginsenosidase type III','NULL','NULL','Ginsenosidase type III catalyzes the sequential hydrolysis of the3-O-beta-D-(1-','A protopanaxadiol-type ginsenoside with two glucosyl residues at position3 + 2 H(2)O = a protopanaxadiol-type ginsenoside with no glycosidicmodification at position 3 + 2 D-glucopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.192','Ginsenoside Rb1 beta-glucosidase','NULL','NULL','Ginsenosidases catalyze the hydrolysis of glycosyl moieties attachedto the C-3, C-6 or C-20 position of ginsenosides.They are specific with respect to the nature of the glycosidiclinkage, the position and the order in which the linkages arecleaved.Ginsenoside Rb1 beta-glucosidase specifically and sequentiallyhydrolyzes the 20-(beta-D-glucopyranosyl-(1-','Ginsenoside Rb1 + 2 H(2)O = ginsenoside Rg3 + 2 D-glucopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.193','Ginsenosidase type I','NULL','NULL','The enzyme hydrolyzes the 3-O-beta-D-(1-','(1) A protopanaxadiol-type ginsenoside with two glucosyl residues atposition 3 + H(2)O = a protopanaxadiol-type ginsenoside with one glucosylresidue at position 3 + D-glucopyranose.(2) A protopanaxadiol-type ginsenoside with one glucosyl residue atposition 3 + H(2)O = a protopanaxadiol-type ginsenoside with noglycosidic modifications at position 3 + D-glucopyranose.(3) A protopanaxadiol-type ginsenoside with two glycosyl residues atposition 20 + H(2)O = a protopanaxadiol-type ginsenoside with a singleglucosyl residue at position 20 + a monosaccharide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.194','Ginsenosidase type IV','NULL','NULL','Ginsenosidase type IV catalyzes the sequential hydrolysis of the 6-O-beta-D-(1-','A protopanaxatriol-type ginsenoside with two glycosyl residues atposition 6 + 2 H(2)O = a protopanaxatriol-type ginsenoside with noglycosidic modification at position 6 + D-glucopyranose +a monosaccharide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.195','20-O-multi-glycoside ginsenosidase','NULL','NULL','The 20-O-multi-glycoside ginsenosidase catalyzes the hydrolysis ofthe 20-O-alpha-(1-','A protopanaxadiol-type ginsenoside with two glycosyl residues at position20 + H(2)O = a protopanaxadiol-type ginsenoside with a single glucosylresidue at position 20 + a monosaccharide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.n1','Blood group B branched chain alpha-1,3-galactosidase','NULL','NULL','The enzyme has exclusive specificity for the branched blood group Bstructures.','Hydrolysis of terminal, non-reducing branched (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.n2','Blood group B linear chain alpha-1,3-galactosidase','NULL','NULL','NULL','Hydrolysis of terminal, non-reducing linear (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.1.n3','Dictyostelium lysozyme A','NULL','NULL','Differs from EC 3.2.1.17, lysozyme, as it does not hydrolyze linkagesbetween N-acetyl-D-glucosamine residues.','Hydrolysis of 1,4-beta-linkages between N-acetylmuramic acid andN-acetyl-D-glucosamine residues in a peptidoglycan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.1','Purine nucleosidase','IAG-NH','NULL','The enzyme from the bacterium Ochrobactrum anthropi specificallycatalyzes the irreversible N-riboside hydrolysis of purinenucleosides.Pyrimidine nucleosides, purine and pyrimidine nucleotides, NAD(+),NADP(+) and nicotinaminde mononucleotide are not substrates.','A purine nucleoside + H(2)O = D-ribose + a purine base.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.2','Inosine nucleosidase','Inosinase','NULL','NULL','Inosine + H(2)O = D-ribose + hypoxanthine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.3','Uridine nucleosidase','Uridine ribohydrolase','NULL','NULL','Uridine + H(2)O = D-ribose + uracil.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.4','AMP nucleosidase','NULL','NULL','NULL','AMP + H(2)O = D-ribose 5-phosphate + adenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.5','NAD(+) glycohydrolase','ADP-ribosyl cyclase','NULL','This enzyme catalyzes the hydrolysis of NAD(+), without associatedADP-ribosyl transferase or ADP-ribosyl cyclase activities (unlike themetazoan enzyme EC 3.2.2.6, bifunctional ADP-ribosyl cyclase/cyclicADP-ribose hydrolase).The enzyme from Group A streptococci has been implicated in thepathogenesis of diseases such as streptococcal toxic shock-likesyndrome (STSS) and necrotizing fasciitis.The enzyme from the venom of the snake Agkistrodon acutus alsocatalyzes EC 3.6.1.5.','NAD(+) + H(2)O = ADP-D-ribose + nicotinamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.6','ADP-ribosyl cyclase/cyclic ADP-ribose hydrolase','NAD hydrolase','NULL','This multiunctional enzyme catalyzes both the synthesis andhydrolysis of cyclic ADP-ribose, a calcium messenger that canmobilize intracellular Ca(2+) stores and activate Ca(2+) influx toregulate a wide range of physiological processes.In addition, the enzyme also catalyzes EC 2.4.99.20.Cf. EC 3.2.2.5.','NAD(+) + H(2)O = ADP-D-ribose + nicotinamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.7','Adenosine nucleosidase','Adenosinase','NULL','Also acts on adenosine N-oxide.','Adenosine + H(2)O = D-ribose + adenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.8','Ribosylpyrimidine nucleosidase','N-ribosylpyrimidine nucleosidase','NULL','Also hydrolyzes purine D-ribonucleosides, more slowly.2-, 3- and 5-deoxynucleosides are not substrates.','A pyrimidine nucleoside + H(2)O = D-ribose + a pyrimidine base.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.9','Adenosylhomocysteine nucleosidase','5-methyladenosine nucleosidase','NULL','NULL','(1) S-adenosyl-L-homocysteine + H(2)O = S-(5-deoxy-D-ribos-5-yl)-L-homocysteine + adenine.(2) S-methyl-5-thioadenosine + H(2)O = 5-methyl-5-thio-D-ribose +adenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.10','Pyrimidine-5-nucleotide nucleosidase','Pyr5N','NULL','Also acts on dUMP, dTMP and dCMP.','A pyrimidine 5-nucleotide + H(2)O = D-ribose 5-phosphate + a pyrimidinebase.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.11','Beta-aspartyl-N-acetylglucosaminidase','NULL','NULL','NULL','1-beta-aspartyl-N-acetyl-D-glucosaminylamine + H(2)O = L-asparagine +N-acetyl-D-glucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.12','Inosinate nucleosidase','NULL','NULL','NULL','5-inosinate + H(2)O = D-ribose 5-phosphate + hypoxanthine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.13','1-methyladenosine nucleosidase','NULL','NULL','NULL','1-methyladenosine + H(2)O = 1-methyladenine + D-ribose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.14','NMN nucleosidase','NMNase','NULL','NULL','Beta-nicotinamide D-ribonucleotide + H(2)O = D-ribose 5-phosphate +nicotinamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.15','DNA-deoxyinosine glycosylase','DNA(hypoxanthine) glycohydrolase','NULL','NULL','Hydrolyzes DNA and polynucleotides, releasing free hypoxanthine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.16','Methylthioadenosine nucleosidase','5-methylthioadenosine nucleosidase','NULL','Does not act on S-adenosylhomocysteine (cf. EC 3.2.2.9).','S-methyl-5-thioadenosine + H(2)O = S-methyl-5-thio-D-ribose + adenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.17','Deoxyribodipyrimidine endonucleosidase','Pyrimidine dimer DNA-glycosylase','NULL','NULL','Cleaves the N-glycosidic bond between the 5-pyrimidine residue incyclobutadipyrimidine (in DNA) and the corresponding deoxy-D-riboseresidue.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.18','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.19','[Protein ADP-ribosylarginine] hydrolase','ADP-ribose-L-arginine cleavage enzyme','NULL','The enzyme will remove ADP-D-ribose from arginine residues in ADP-ribosylated proteins.','(1) Protein-N(omega)-(ADP-D-ribosyl)-L-arginine + H(2)O = ADP-D-ribose +protein-L-arginine.(2) N(omega)-(ADP-D-ribosyl)-L-arginine + H(2)O = ADP-D-ribose +L-arginine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.20','DNA-3-methyladenine glycosylase I','DNA glycosidase I','NULL','NULL','Hydrolysis of alkylated DNA, releasing 3-methyladenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.21','DNA-3-methyladenine glycosylase II','DNA glycosidase II','NULL','NULL','Hydrolysis of alkylated DNA, releasing 3-methyladenine, 3-methylguanine,7-methylguanine and 7-methyladenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.22','rRNA N-glycosylase','rRNA N-glycosidase','NULL','Ricin A-chain and related toxins show this activity.Naked rRNA is attacked more slowly than rRNA in intact ribosomes.Naked rRNA from Escherichia coli is cleaved at a correspondingposition.','Endohydrolysis of the N-glycosidic bond at one specific adenosine on the28S rRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.23','DNA-formamidopyrimidine glycosylase','Fapy-DNA glycosylase','NULL','May play a significant role in processes leading to recovery frommutagenesis and/or cell death by alkylating agents.Also involved in the GO system responsible for removing anoxidatively damaged form of guanine (7,8-dihydro-8-oxoguanine) fromDNA.','Hydrolysis of DNA containing ring-opened 7-methylguanine residues,releasing 2,6-diamino-4-hydroxy-5-(N-methyl)formamidopyrimidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.24','ADP-ribosyl-[dinitrogen reductase] hydrolase','ADP-ribosyl glycohydrolase','NULL','Together with EC 2.4.2.37, controls the level of activity ofEC 1.18.6.1.','ADP-D-ribosyl-[dinitrogen reductase] = ADP-D-ribose + [dinitrogenreductase].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.25','N-methyl nucleosidase','7-methylxanthosine nucleosidase','NULL','Preferentially hydrolyzes 3- and 7-methylpurine nucleosides, such as3-methylxanthosine, 3-methyladenosine and 7-methylguanosine.Hydrolysis of 7-methylxanthosine to form 7-methylxanthine is thesecond step in the caffeine biosynthesis pathway.','7-methylxanthosine + H(2)O = 7-methylxanthine + D-ribose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.26','Futalosine hydrolase','Futalosine nucleosidase','NULL','This enzyme, which is specific for futalosine, catalyzes the secondstep of a novel menaquinone biosynthetic pathway that is found insome prokaryotes.','Futalosine + H(2)O = dehypoxanthine futalosine + hypoxanthine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.27','Uracil-DNA glycosylase','Uracil DNA glycohydrolase','NULL','Uracil-DNA glycosylases are widespread enzymes that are found in allliving organisms.Uracil-DNA glycosylase (EC 3.2.2.27) and double-stranded uracil-DNAglycosylase (EC 3.2.2.28) form a central part of the DNA-repairmachinery since they initiate the DNA base-excision repair pathway byhydrolyzing the N-glycosidic bond between uracil and the deoxyribosesugar thereby catalyzing the removal of mis-incorporated uracil fromDNA.','Hydrolyzes single-stranded DNA or mismatched double-stranded DNA andpolynucleotides, releasing free uracil.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.28','Double-stranded uracil-DNA glycosylase','DsUDG','NULL','No activity on DNA containing a T/G mispair or single-stranded DNAcontaining either a site-specific uracil or 3,N(4)-ethenocytosineresidue, significant role for double-stranded uracil-DNA glycosylasein mutation avoidance in non-dividing Escherichia coli.Uracil-DNA glycosylases are widespread enzymes that are found in allliving organisms.Uracil-DNA glycosylase (EC 3.2.2.27) and double-stranded uracil-DNAglycosylase (EC 3.2.2.28) form a central part of the DNA-repairmachinery since they initiate the DNA base-excision repair pathway byhydrolyzing the N-glycosidic bond between uracil and the deoxyribosesugar thereby catalyzing the removal of mis-incorporated uracil fromDNA.','Specifically hydrolyzes mismatched double-stranded DNA andpolynucleotides, releasing free uracil.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.2.29','Thymine-DNA glycosylase','Mismatch-specific thymine-DNA glycosylase','NULL','Thymine-DNA glycosylase is part of the DNA-repair machinery.Thymine removal is fastest when it is from a G/T mismatch with a5-flanking C/G pair.The glycosylase removes uracil from G/U, C/U, and T/U base pairsfaster than it removes thymine from G/T.','Hydrolyzes mismatched double-stranded DNA and polynucleotides, releasingfree thymine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.2.3.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.1.1','Adenosylhomocysteinase','Adenosylhomocysteine hydrolase','NAD(+).','The NAD(+) cofactor appears to bring about a transient oxidation atC-3 of the 5-deoxyadenosine residue, thus labilizing the thioetherbond cf. EC 5.5.1.4.','S-adenosyl-L-homocysteine + H(2)O = L-homocysteine + adenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.1.2','Adenosylmethionine hydrolase','Methylmethionine-sulfonium-salt hydrolase','NULL','Also hydrolyzes methylmethionine sulfonium salt to dimethyl sulfideand homoserine.','S-adenosyl-L-methionine + H(2)O = L-homoserine + methylthioadenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.1.3','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.1','Isochorismatase','2,3-dihydro-2,3-dihydroxybenzoate synthase','NULL','NULL','Isochorismate + H(2)O = (2S,3S)-2,3-dihydroxy-2,3-dihydrobenzoate +pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.2','Alkenylglycerophosphocholine hydrolase','NULL','NULL','NULL','1-(1-alkenyl)-sn-glycero-3-phosphocholine + H(2)O = an aldehyde +sn-glycero-3-phosphocholine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.4','Trans-epoxysuccinate hydrolase','Trans-epoxysuccinate hydratase','NULL','Acts on both optical isomers of the substrate.Formerly EC 4.2.1.37.','Trans-2,3-epoxysuccinate + H(2)O = meso-tartrate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.5','Alkenylglycerophosphoethanolamine hydrolase','NULL','NULL','NULL','1-(1-alkenyl)-sn-glycero-3-phosphoethanolamine + H(2)O = an aldehyde +sn-glycero-3-phosphoethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.6','Leukotriene-A(4) hydrolase','Leukotriene A(4) hydrolase','Zinc.','A bifunctional zinc metalloprotease that displays both epoxidehydrolase and aminopeptidase activities.It preferentially cleaves tripeptides at an arginyl bond, withdipeptides and tetrapeptides being poorer substrates.It also converts leukotriene A(4) into leukotriene B(4), unlikeEC 3.2.2.10 which converts leukotriene A(4) into 5,6-dihydroxy-7,9,11,14-eicosatetraenoic acid.','(7E,9E,11Z,14Z)-(5S,6S)-5,6-epoxyicosa-7,9,11,14-tetraenoate + H(2)O =(6Z,8E,10E,14Z)-(5S,12R)-5,12-dihydroxyicosa-6,8,10,14-tetraenoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.7','Hepoxilin-epoxide hydrolase','Hepoxilin A(3) hydrolase','NULL','Converts hepoxilin A(3) into trioxilin A(3).Highly specific for the substrate, having only slight activity withother epoxides such as leukotriene A(4) and styrene oxide.Probably plays a modulatory role in inflammation, vascularphysiology, systemic glucose metabolism and neurological function.','(5Z,9E,14Z)-(8-xi,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate +H(2)O = (5Z,9E,14Z)-(8-xi,11-xi,12S)-8,11,12-trihydroxyicosa-5,9,14-trienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.8','Limonene-1,2-epoxide hydrolase','Limonene oxide hydrolase','NULL','Involved in the monoterpene degradation pathway of the actinomyceteRhodococcus erythropolis.The enzyme hydrolyzes several alicyclic and 1-methyl-substitutedepoxides, such as 1-methylcyclohexene oxide, indene oxide andcyclohexene oxide.It differs from the previously described epoxide hydrolases(EC 3.3.2.4, EC 3.3.2.6, EC 3.3.2.7, EC 3.3.2.9 and EC 3.3.2.10) asit is not inhibited by 2-bromo-4-nitroacetophenone, diethylpyrocarbonate, 4-fluorochalcone oxide or 1,10-phenanthroline.','1,2-epoxymenth-8-ene + H(2)O = menth-8-ene-1,2-diol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.9','Microsomal epoxide hydrolase','Arene-oxide hydratase','NULL','Involved in the metabolism of numerous xenobiotics, such as 1,3-butadiene oxide, styrene oxide and the polycyclic aromatichydrocarbon benzo[a]pyrene 4,5-oxide.In a series of oxiranes with a lipophilic substituent of sufficientsize (styrene oxides), monosubstituted as well as 1,1- and cis-1,2-disubstituted oxiranes serve as substrates or inhibitors of theenzyme, while trans-1,2-disubstituted, tri- and tetra-substitutedoxiranes are not substrates.The reaction involves the formation of an hydroxyalkyl-enzymeintermediate.Formerly EC 3.3.2.3, EC 4.2.1.63 and EC 4.2.1.64.','Cis-stilbene oxide + H(2)O = (+)-(1R,2R)-1,2-diphenylethane-1,2-diol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.10','Soluble epoxide hydrolase','Arene-oxide hydratase','NULL','Catalyzes the hydrolysis of trans-substituted epoxides, such astrans-stilbene oxide, as well as various aliphatic epoxides derivedfrom fatty-acid metabolism.It is involved in the metabolism of arachidonic epoxides(epoxyeicosatrienoic acids; EETs) and linoleic acid epoxides.The enzyme from mammals is a bifunctional enzyme: the C-terminaldomain exhibits epoxide-hydrolase activity and the N-terminal domainhas the activity of EC 3.1.3.76.Like EC 3.3.2.9, it is probable that the reaction involves theformation of an hydroxyalkyl-enzyme intermediate.The enzyme can also use leukotriene A(4), the substrate ofEC 3.3.2.6, but it forms 5,6-dihydroxy-7,9,11,14-eicosatetraenoicacid rather than leukotriene B(4) as the product.Formerly EC 3.3.2.3, EC 4.2.1.63 and EC 4.2.1.64.','An epoxide + H(2)O = a glycol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.11','Cholesterol-5,6-oxide hydrolase','ChEH','NULL','The enzyme appears to work equally well with both diastereoisomers ofcholesterol 5,6-epoxide.The product is a competitive inhibitor of the reaction.','(1) 5,6-alpha-epoxy-5-alpha-cholestan-3-beta-ol + H(2)O = cholestane-3-beta,5-alpha,6-beta-triol.(2) 5,6-beta-epoxy-5-beta-cholestan-3-beta-ol + H(2)O = cholestane-3-beta,5-alpha,6-beta-triol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.12','Oxepin-CoA hydrolase','NULL','NULL','The enzyme from Escherichia coli is a bifunctional fusion proteinthat also catalyzes EC 1.17.1.7.Combined the two activities result in a two-step conversion ofoxepin-CoA to 3-oxo-5,6-dehydrosuberyl-CoA, part of an aerobicphenylacetate degradation pathway.The enzyme from Escherichia coli also exhibits enoyl-CoA hydrataseactivity utilizing crotonyl-CoA as a substrate.Formerly EC 3.7.1.16.','2-oxepin-2(3H)-ylideneacetyl-CoA + H(2)O = 3-oxo-5,6-dehydrosuberyl-CoAsemialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.13','Chorismatase','Chorismate/3,4-dihydroxycyclohexa-1,5-dienoate synthase','NULL','The enzyme found in several bacterial species is involved in thebiosynthesis of macrocyclic polyketides.','Chorismate + H(2)O = (3R,4R)-3,4-dihydroxy-3,4-dihydrobenzoate +pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.3.2.30','Aminodeoxyfutalosine nucleosidase','AFL nucleosidase','NULL','The enzyme, found in several bacterial species, also has the activityof EC 3.2.2.9.It is part of a modified futalosine pathway for menaquinonebiosynthesis.','6-amino-6-deoxyfutalosine + H(2)O = dehypoxanthine futalosine + adenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.1.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.1.2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.1.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.1.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.2.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.2.2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.2.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.3.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.3.2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.3.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.3.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.3.5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.3.6','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.3.7','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.6','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.7','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.9','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.10','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.11','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.12','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.13','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.14','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.15','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.16','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.17','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.18','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.19','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.20','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.21','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.22','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.23','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.24','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.4.25','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.1','Leucyl aminopeptidase','Cytosol aminopeptidase','Zinc.','Is activated by heavy metal ions.Belongs to peptidase family M17.Formerly EC 3.4.1.1.','Release of an N-terminal amino acid, Xaa-|-Yaa-, in which Xaa ispreferably Leu, but may be other amino acids including Pro although notArg or Lys, and Yaa may be Pro. Amino acid amides and methyl esters arealso readily hydrolyzed, but rates on arylamides are exceedingly low.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.2','Membrane alanyl aminopeptidase','Amino-oligopeptidase','Zinc.','Is not activated by heavy metal ions.Belongs to peptidase family M1.Formerly EC 3.4.1.2, EC 3.4.3.5 and EC 3.4.13.6.','Release of an N-terminal amino acid, Xaa-|-Yaa- from a peptide, amide orarylamide. Xaa is preferably Ala, but may be most amino acids includingPro (slow action). When a terminal hydrophobic residue is followed by aprolyl residue, the two may be released as an intact Xaa-Pro dipeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.3','Cystinyl aminopeptidase','Cystine aminopeptidase','Zinc.','Belongs to peptidase family M1.','Release of an N-terminal amino acid, Cys-|-Xaa-, in which the half-cystine residue is involved in a disulfide loop, notably in oxytocin orvasopressin. Hydrolysis rates on a range of aminoacyl arylamides exceedthat for the cystinyl derivative, however.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.4','Tripeptide aminopeptidase','Aminoexotripeptidase','Zinc.','Widely distributed in mammalian tissues.Belongs to peptidase family M9.Formerly EC 3.4.1.3.','Release of the N-terminal residue from a tripeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.5','Prolyl aminopeptidase','Cytosol aminopeptidase V','Manganese.','Present in the cytosol of mammalian and microbial cells.In contrast to the mammalian form, the bacterial form of the enzymehydrolyzes both polyproline and prolyl-2-naphthylamide.The mammalian enzyme, which is not specific for prolyl bonds,is possibly identical with EC 3.4.11.1.Belongs to peptidase family S33.Formerly EC 3.4.1.4.','Release of N-terminal proline from a peptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.6','Aminopeptidase B','Arginine aminopeptidase','Zinc.','Cytosolic or membrane-associated enzyme from mammalian tissues,activated by chloride ions and low concentrations of thiol compounds.One of the activities of the bifunctional enzyme EC 3.3.2.6.Belongs to peptidase family M1.','Release of N-terminal Arg and Lys from oligopeptides when P1 is not Pro.Also acts on arylamides of Arg and Lys.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.7','Glutamyl aminopeptidase','Aminopeptidase A','Zinc.','Also acts on L-alpha-glutamyl-peptides.Calcium-activated and generally membrane-bound.Belongs to peptidase family M1.','Release of N-terminal glutamate (and to a lesser extent aspartate) from apeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.9','Xaa-Pro aminopeptidase','Aminoacylproline aminopeptidase','Manganese or cobalt.','Generally membrane bound enzyme present in both mammalian andbacterial cells.Belongs to peptidase family M24B.','Release of any N-terminal amino acid, including proline, that is linkedto proline, even from a dipeptide or tripeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.10','Bacterial leucyl aminopeptidase','Aeromonas proteolytica aminopeptidase','Zinc.','Similar aminopeptidases were isolated from Escherichia coli andStaphylococcus thermophilus.Belongs to peptidase families M17 and M28.','Release of an N-terminal amino acid, preferentially leucine, but notglutamic or aspartic acids.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.11','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.12','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.13','Clostridial aminopeptidase','Clostridium histolyticum aminopeptidase','Manganese or cobalt.','A secreted enzyme from Clostridium histolyticum.','Release of any N-terminal amino acid, including proline andhydroxyproline, but no cleavage of Xaa-|-Pro.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.14','Cytosol alanyl aminopeptidase','Aminopolypeptidase','Zinc or cobalt.','A puromycin-sensitive, cobalt-activated zinc-sialoglycoprotein thatis generally cytosolic.Multiple forms are widely distributed in mammalian tissues and bodyfluids.','Release of an N-terminal amino acid, preferentially alanine, from a widerange of peptides, amides and arylamides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.15','Aminopeptidase Y','Aminopeptidase (cobalt-activated)','Cobalt.','Inhibited by Zn(2+) and Mn(2+).Hydrolyzes L-lysyl-4-nitroanilide and, more slowly, L-arginyl-4-nitroanilide.Belongs to peptidase family M28B.','Preferentially, release of N-terminal lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.16','Xaa-Trp aminopeptidase','Aminopeptidase W','Zinc.','NULL','Release of a variety of N-terminal residues (especially glutamate andleucine) from peptides, provided tryptophan (or at least phenylalanine ortyrosine) is the penultimate residue. Also acts on Glu-|-Trp, Leu-|-Trpand a number of other dipeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.17','Tryptophanyl aminopeptidase','Tryptophan aminopeptidase','Manganese.','Also acts on L-tryptophanamide.From Trichosporon cutaneum.','Preferential release of N-terminal tryptophan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.18','Methionyl aminopeptidase','Methionine aminopeptidase','Cobalt.','Removes the amino-terminal methionine from nascent proteins.Membrane-bound enzyme present in both prokaryotes and eukaryotes.Belongs to peptidase family M24A.','Release of N-terminal amino acids, preferentially methionine, frompeptides and arylamides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.19','D-stereospecific aminopeptidase','NULL','NULL','A thiol-dependent peptidase known from the bacterium Ochrobactrumanthropi.A homolog of EC 3.4.16.4.Not metal dependent; probably has serine at the active center.Belongs to peptidase family S12.','Release of an N-terminal D-amino acid from a peptide, Xaa-|-Yaa-,in which Xaa is preferably D-Ala, D-Ser or D-Thr. D-amino acid amides andmethyl esters also are hydrolyzed, as is glycine amide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.20','Aminopeptidase Ey','NULL','Zinc.','From the plasma fraction of chicken egg yolk.Belongs to peptidase family M1.','Differs from other aminopeptidases in broad specificity for amino acidsin the P1 position and the ability to hydrolyze peptides of four or fiveresidues that contain Pro in the P1 position.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.21','Aspartyl aminopeptidase','NULL','NULL','Aminoacyl-arylamides are poor substrates.Belongs to peptidase family M18.','Release of an N-terminal aspartate or glutamate from a peptide, with apreference for aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.22','Aminopeptidase I','Aminopeptidase III','NULL','Belongs to peptidase family M18.','Release of an N-terminal amino acid, preferably a neutral or hydrophobicone, from a polypeptide. Aminoacyl-arylamides are poor substrates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.23','PepB aminopeptidase','Aminopeptidase B','NULL','A 270-kDa protein composed of six 46.3-kDa subunits.The pH optimum is in the alkaline range and activity is stimulated byKCl.Belongs to peptidase family M17.','Release of an N-terminal amino acid, Xaa, from a peptide or arylamide.Xaa is preferably Glu or Asp but may be other amino acids, including Leu,Met, His, Cys and Gln.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.24','Aminopeptidase S','Double-zinc aminopeptidase','Zinc.','Activated by Ca(2+).In the presence of Ca(2+), the best substrates are Leu-Phe, Leu-Ser,Leu-pNA (aminoacyl-p-nitroanilide), Phe-Phe-Phe and Phe-Phe.Peptides with proline in the P1 position are not substrates.Belongs to peptidase family M28.Formerly EC 3.4.11.n1.','Release of an N-terminal amino acid with a preference for largehydrophobic amino-terminus residues.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.25','Beta-peptidyl aminopeptidase','NULL','NULL','Sphingosinicella xenopeptidilytica strain 3-2W4 is able to utilizethe beta-peptides beta-homoVal-beta-homoAla-beta-homoLeu and beta-homoAla-beta-homoLeu as sole carbon and energy sources.','Cleaves N-terminal beta-homoamino acids from peptides composed of 2 to 6amino acids.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.26','Intermediate cleaving peptidase 55','NULL','NULL','Icp55 removes the destabilizing N-terminal amino acid residues thatare left after cleavage by the mitochondrial processing peptidase,leading to the stabilization of the substrate.The enzyme can remove single amino acids or a short peptide, as inthe case of cysteine desulfurase (EC 2.8.1.7), where three aminoacids are removed.','The enzyme cleaves the Pro(36)-Pro(37) bond of cysteine desulfurase(EC 2.8.1.7) removing three amino acid residues (Tyr-Ser-Pro) from theN-terminus after cleavage by mitochondrial processing peptidase.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.11.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.6','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.7','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.9','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.10','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.11','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.12','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.12.13','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.4','Xaa-Arg dipeptidase','Aminoacyl-lysine dipeptidase','NULL','Widely distributed in mammals.','Preferential hydrolysis of Xaa-|-Arg, Xaa-|-Lys or Xaa-|-ornithinedipeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.5','Xaa-methyl-His dipeptidase','Aminoacyl-methylhistidine dipeptidase','NULL','Identical with acetylhistidine deacetylase.Formerly EC 3.4.3.4 and EC 3.5.1.34.','Hydrolysis of anserine (beta-alanyl-|-N(pi)-methyl-L-histidine),carnosine, homocarnosine, glycyl-|-leucine and other dipeptides withbroad specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.6','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.7','Glu-Glu dipeptidase','Alpha-glutamyl-glutamate dipeptidase','NULL','It is unclear whether the specificity of this enzyme extends to otheralpha-glutamyl dipeptides.','Hydrolysis of the Glu-|-Glu dipeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.9','Xaa-Pro dipeptidase','Gamma-peptidase','Manganese.','Possibly thiol dependent.Cytosolic from most animal tissues.Belongs to peptidase family M24A.Formerly EC 3.4.3.7.','Hydrolysis of Xaa-|-Pro dipeptides; also acts on aminoacyl-hydroxyprolineanalogs. No action on Pro-|-Pro.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.10','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.11','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.12','Met-Xaa dipeptidase','Dipeptidase M','Manganese.','Escherichia coli enzyme with thiol dependence.','Hydrolysis of Met-|-Xaa dipeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.13','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.14','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.15','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.16','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.17','Non-stereospecific dipeptidase','D-(or L-)aminoacyl-dipeptidase','NULL','A digestive enzyme of cephalopods.','Hydrolysis of dipeptides containing either D- or L-amino acids or both.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.18','Cytosol nonspecific dipeptidase','Cytosol non-specific dipeptidase','Zinc.','Broad specificity, varying somewhat with source species.Activated and stabilized by dithiothreitol and manganese.Inhibited by bestatin and leucine.Belongs to peptidase family M20.Formerly EC 3.4.3.1, EC 3.4.3.2, EC 3.4.3.3, EC 3.4.3.6, EC 3.4.13.1,EC 3.4.13.2, EC 3.4.13.3, EC 3.4.13.8, EC 3.4.13.11, EC 3.4.13.13 andEC 3.4.13.15.','Hydrolysis of dipeptides, preferentially hydrophobic dipeptides includingprolyl amino acids.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.19','Membrane dipeptidase','Dehydropeptidase I','Zinc.','Membrane bound, with broad specificity.Abundant in the kidney cortex.Inhibited by bestatin and cilastatin.Belongs to peptidase family M19.Formerly EC 3.4.3.1, EC 3.4.3.2, EC 3.4.3.6, EC 3.4.13.1,EC 3.4.13.2, EC 3.4.13.8, EC 3.4.13.11 and EC 3.4.13.15.','Hydrolysis of dipeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.20','Beta-Ala-His dipeptidase','Serum carnosinase','NULL','Present in the serum of humans and higher primates, but not in theserum of other mammals.Activated by Cd(2+) and citrate.Belongs to peptidase family M20.Formerly EC 3.4.3.3, EC 3.4.13.3 and EC 3.4.13.13.','Preferential hydrolysis of the beta-Ala-|-His dipeptide (carnosine),and also anserine, Xaa-|-His dipeptides and other dipeptides includinghomocarnosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.21','Dipeptidase E','Aspartyl dipeptidase','NULL','A free carboxy group is not absolutely required in the substratesince Asp-Phe-NH(2) and Asp-Phe-OMe are hydrolyzed somewhat moreslowly than dipeptides with free C-termini.No peptide larger than a C-blocked dipeptide is known to be asubstrate.Asp-NH-Np is hydrolyzed and is a convenient substrate for routineassay.The enzyme is most active near pH 7.0, and is not inhibited bydi-isopropylfluorophosphate or phenylmethanesulfonyl fluoride.Belongs to peptidase family S51.','Dipeptidase E catalyzes the hydrolysis of dipeptides Asp-|-Xaa. It doesnot act on peptides with N-terminal Glu, Asn or Gln, nor does it cleaveisoaspartyl peptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.13.22','D-Ala-D-Ala dipeptidase','D-alanyl-D-alanine dipeptidase','Zinc.','Protects Enterococcus faecium from the antibiotic vancomycin, whichcan bind to the -D-Ala-D-Ala sequence at the C-terminus of thepeptidoglycan pentapeptide.Reduces the availability of the free dipeptide D-Ala-D-Ala, which isthe precursor for this pentapeptide sequence, allowing D-Ala-(R)-lactate (for which vancomycin has much less affinity) to be added tothe cell wall instead.The enzyme is stereospecific, as L-Ala-L-Ala, D-Ala-L-Ala and L-Ala-D-Ala are not substrates.Belongs to peptidase family M15.','D-Ala-D-Ala + H(2)O = 2 D-Ala.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.1','Dipeptidyl-peptidase I','Cathepsin C','NULL','Also polymerizes dipeptide amides, arylamides and esters at neutralpH.Cl(-)-dependent lysosomal cysteine-type peptidase.Belongs to peptidase family C1.Formerly EC 3.4.4.9.','Release of an N-terminal dipeptide, Xaa-Yaa-|-Zaa-, except when Xaa isArg or Lys, or Yaa or Zaa is Pro.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.2','Dipeptidyl-peptidase II','Carboxytripeptidase','NULL','A lysosomal serine-type peptidase maximally active at acidic pH.Cleaves Lys-Ala-naphthylamide.Belongs to peptidase family S28.','Release of an N-terminal dipeptide, Xaa-Yaa-|-, preferentially when Yaais Ala or Pro. Substrates are oligopeptides, preferentially tripeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.4','Dipeptidyl-peptidase III','Dipeptidyl aminopeptidase III','NULL','A cytosolic peptidase that is active at neutral pH.It has broad activity on peptides, although it is highly selectivefor Arg-Arg-2-naphthylamide, at pH 9.2.Active in the hydrolysis of enkephalins.Belongs to peptidase family M49.','Release of an N-terminal dipeptide from a peptide comprising four or moreresidues, with broad specificity. Also acts on dipeptidyl2-naphthylamides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.5','Dipeptidyl-peptidase IV','Dipeptidyl aminopeptidase IV','NULL','A membrane-bound serine-type peptidase in mammals.EC 3.4.14.11 catalyzes a similar reaction.Belongs to peptidase family S9B.','Release of an N-terminal dipeptide, Xaa-Yaa-|-Zaa-, from a polypeptide,preferentially when Yaa is Pro, provided Zaa is neither Pro norhydroxyproline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.6','Dipeptidyl-dipeptidase','Dipeptidyl ligase','NULL','Tetrapeptides are formed from Ala-Ala, Gly-Gly, Ala-Gly and Gly-Alawithout prior activation of the carboxy group.Activated by thiols.From cabbage (Brassica oleracea).','Preferential release of dipeptides from a tetrapeptide, e.g. Ala-Gly-|-Ala-Gly. Acts more slowly on Ala-Ala-|-Ala-Ala and Gly-Gly-|-Gly-Gly.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.7','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.9','Tripeptidyl-peptidase I','Tripeptidyl aminopeptidase','NULL','A lysosomal enzyme that is active at acidic pH.Belongs to peptidase family S53.Formerly included in EC 3.4.14.8.','Release of an N-terminal tripeptide from a polypeptide, but also hasendopeptidase activity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.10','Tripeptidyl-peptidase II','Tripeptidyl aminopeptidase','NULL','A cytosolic enzyme active at neutral pH.Inhibited by diisopropyl fluorophosphate.Belongs to peptidase family S8.Formerly EC 3.4.14.8.','Release of an N-terminal tripeptide from a polypeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.11','Xaa-Pro dipeptidyl-peptidase','X-Pro dipeptidyl-peptidase','NULL','EC 3.4.14.5 catalyzes a similar reaction.Belongs to peptidase family S15.','Hydrolyzes Xaa-Pro-|- bonds to release unblocked, N-terminal dipeptidesfrom substrates including Ala-Pro-|-p-nitroanilide and (sequentially)Tyr-Pro-|-Phe-Pro-|-Gly-Pro-|-Ile.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.14.12','Xaa-Xaa-Pro tripeptidyl-peptidase','Prolyl tripeptidyl peptidase','NULL','This cell-surface-associated serine exopeptidase is found in theGram-negative, anaerobic bacterium Porphyromonas gingivalis, whichhas been implicated in adult periodontal disease.The enzyme releases tripeptides from the free amino terminus ofpeptides and small proteins, such as interleukin-6.The enzyme possesses an absolute requirement for a proline residue atthe P1 position but is completely inactivated by a proline residue atthe P1 position.The size of the peptide does not affect the rate of reaction.','Hydrolysis of Xaa-Xaa-Pro-|-Yaa- releasing the N-terminal tripeptide of apeptide with Pro as the third residue (position P1) and where Yaa is notproline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.15.1','Peptidyl-dipeptidase A','ACE','Zinc.','Cl(-)-dependent.A glycoprotein that is generally membrane-bound.Active at neutral pH.Only single dipeptides are released from angiotensin I and bradykininbecause of the lack of activity on bonds involving proline.Belongs to peptidase family M2.','Release of a C-terminal dipeptide, oligopeptide-|-Xaa-Yaa, when Xaa isnot Pro, and Yaa is neither Asp nor Glu. Thus, conversion of angiotensinI to angiotensin II, with increase in vasoconstrictor activity, but noaction on angiotensin II.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.15.2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.15.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.15.4','Peptidyl-dipeptidase B','Atrial di-(tri-)peptidyl carboxylase','Zinc.','A membrane-bound, zinc metallopeptidase located in mammalian atrial,but not ventricular, myocytes.Although it is capable of converting the 126-residue atriopeptin IIIdirectly to atriopeptin I by releasing a C-terminal tripeptide Phe-Arg-Tyr, it is generally restricted to the release of dipeptides.In contrast to EC 3.4.15.1 it displays no chlorine dependence andshows no action on angiotensin I.Conversely, peptidyl-dipeptidase a is unable to release Phe-Arg fromthe C-terminus of atriopeptin II.','Release of a C-terminal dipeptide or exceptionally a tripeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.15.5','Peptidyl-dipeptidase Dcp','Dipeptidyl carboxypeptidase','Zinc.','Inhibited by captopril, as is peptidyl-dipeptidase A.Belongs to peptidase family M3.Formerly EC 3.4.15.3.','Hydrolysis of unblocked, C-terminal dipeptides from oligopeptides, withbroad specificity. Does not hydrolyze bonds in which P1 is Pro, or bothP1 and P1 are Gly.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.15.6','Cyanophycinase','Beta-asp-Arg hydrolyzing enzyme','NULL','Highly specific for the branched polypeptide cyanophycin and does nothydrolyze poly-L-aspartate or poly-L-arginine.A serine-type exopeptidase.Belongs to peptidase family S51.','(L-Asp(4-L-Arg))(n) + H(2)O = (L-Asp(4-L-Arg))(n-1) + L-Asp(4-L-Arg).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.16.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.16.2','Lysosomal Pro-Xaa carboxypeptidase','Angiotensinase C','NULL','A lysosomal peptidase active at acidic pH that inactivatesangiotensin II.Inhibited by diisopropyl fluorophosphate.Belongs to peptidase family S28.Formerly EC 3.4.12.4.','Cleavage of a -Pro-|-Xaa bond to release a C-terminal amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.16.3','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.16.4','Serine-type D-Ala-D-Ala carboxypeptidase','D-alanyl-D-alanine carboxypeptidase','NULL','A group of bacterial enzymes, membrane-bound.Inhibited by beta-lactam antibiotics, which acylate the active siteserine in the enzyme.Distinct from EC 3.4.17.14.Belongs to peptidase families S11, S12 and S13.','Preferential cleavage: (Ac)(2)-L-Lys-D-Ala-|-D-Ala. Also transpeptidationof peptidyl-alanyl moieties that are N-acyl substituents of D-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.16.5','Carboxypeptidase C','Carboxypeptidase Y','NULL','Carboxypeptidases of broad specificity with optimum pH 4.5-6.0,inhibited by the action of diisopropyl fluorophosphate, and sensitiveto thiol-blocking reagents.This enzyme is probably also identical to lysosomal tyrosinecarboxypeptidase.Belongs to peptidase family S10.Formerly EC 3.4.12.1, EC 3.4.12.12, EC 3.4.16.1, EC 3.4.16.3 andEC 3.4.21.13.','Release of a C-terminal amino acid with broad specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.16.6','Carboxypeptidase D','Carboxypeptidase KEX1','NULL','Carboxypeptidases of broad specificity with optimum pH 4.5-6.0,inhibited by the action of diisopropyl fluorophosphate, and sensitiveto thiol-blocking reagents.Belongs to peptidase family S10.Formerly EC 3.4.12.1, EC 3.4.12.12, EC 3.4.16.1, EC 3.4.16.3 andEC 3.4.21.13.','Preferential release of a C-terminal arginine or lysine residue.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.1','Carboxypeptidase A','Carboxypolypeptidase','Zinc.','Isolated from cattle, pig and dogfish pancreas, and other sourcesincluding mast cells and skeletal muscle.Belongs to peptidase family M14.Formerly EC 3.4.2.1 and EC 3.4.12.2.','Release of a C-terminal amino acid, but little or no action with -Asp,-Glu, -Arg, -Lys or -Pro.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.2','Carboxypeptidase B','Protaminase','Zinc.','Isolated from cattle, pig and dogfish pancreas, and other sources,including skin fibroblasts and adrenal medulla.Belongs to peptidase family M14.Formerly EC 3.4.2.2 and EC 3.4.12.3.','Preferential release of a C-terminal lysine or arginine amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.3','Lysine carboxypeptidase','Anaphylatoxin inactivator','Zinc.','Inactivates bradykinin and anaphylatoxins in blood plasma.Belongs to peptidase family M14.Formerly EC 3.4.12.7.','Release of a C-terminal basic amino acid, preferentially lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.4','Gly-Xaa carboxypeptidase','Carboxypeptidase A','NULL','Belongs to peptidase family M20A.Formerly EC 3.4.2.3, EC 3.4.12.8 and EC 3.4.17.9.','Release of a C-terminal amino acid from a peptide in which glycine is thepenultimate amino acid, e.g. Z-Gly-|-Leu.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.5','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.6','Alanine carboxypeptidase','NULL','NULL','Peptide can be replaced by a pteroyl group or variety of acyl groups.The enzyme from Corynebacterium equi also hydrolyzes N-benzoylglycineand N-benzoyl-L-aminobutyric acid.Formerly EC 3.4.12.11.','Release of a C-terminal alanine from a peptide or a variety of pteroyl oracyl groups.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.7','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.8','Muramoylpentapeptide carboxypeptidase','Carboxypeptidase D-alanyl-D-alanine','Zinc.','Does not cleave the C-terminal D-alanine from the product of theabove reaction, UDP-N-acetyl-muramoyl-L-alanyl-D-gamma-glutamyl-6-carboxy-L-lysyl-D-alanine.Competitively inhibited by penicillins and cephalosporins.Belongs to peptidase family M15.Formerly EC 3.4.12.6.','Cleavage of the bond UDP-N-acetylmuramoyl-L-alanyl-D-gamma-glutamyl-6-carboxy-L-lysyl-D-alanyl-|-D-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.9','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.10','Carboxypeptidase E','Carboxypeptidase H','Zinc.','Activated by Co(2+).Inhibited by 1,10-phenanthroline and other chelating agents.pH optimum 5.6.Located in storage granules of secretory cells, and active inprocessing of protein hormones and bioactive peptides.Distinct from EC 3.4.17.2 and EC 3.4.17.3.Belongs to peptidase family M14.','Release of C-terminal arginine or lysine residues from polypeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.11','Glutamate carboxypeptidase','Carboxypeptidase G','Zinc.','Produced by pseudomonads, Flavobacterium sp. and Acinetobacter sp.Its ability to hydrolyze pteroyl-L-glutamate (folic acid) has led toits use as a folate-depleting, antitumor agent.Belongs to peptidase family M20A.','Release of C-terminal glutamate residues from a wide range of N-acylatingmoieties, including peptidyl, aminoacyl, benzoyl, benzyloxycarbonyl,folyl and pteroyl groups.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.12','Carboxypeptidase M','NULL','Zinc.','A membrane-bound enzyme optimally active at neutral pH.Distinct from EC 3.4.17.3 and EC 3.4.17.10.Belongs to peptidase family M14.','Cleavage of C-terminal arginine or lysine residues from polypeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.13','Muramoyltetrapeptide carboxypeptidase','Carboxypeptidase IIW','NULL','Variants are known from various microorganisms.Involved in peptidoglycan synthesis, catalyzing both decarboxylationand transpeptidation.Stimulated by divalent cations such as magnesium and calcium, but notzinc.Inhibited by thiol-blocking reagents, but unaffected by penicillin.','GlcNAc-MurNAc-L-alanyl-gamma-D-glutamyl-meso-diaminopimelyl-D-alanine +H(2)O = GlcNAc-MurNAc-L-alanyl-gamma-D-glutamyl-meso-diaminopimelate +D-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.14','Zinc D-Ala-D-Ala carboxypeptidase','D-alanyl-D-alanine hydrolase','Zinc.','Catalyzes carboxypeptidation but not transpeptidation reactionsinvolved in bacterial cell wall metabolism.Weakly inhibited by beta-lactams.Distinct from EC 3.4.16.4.','Cleavage of the bond: (Ac)(2)-L-lysyl-D-alanyl-|-D-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.15','Carboxypeptidase A2','NULL','Zinc.','Isolated from rat pancreas but not present in cattle pancreas.Belongs to peptidase family M14.','Similar to that of carboxypeptidase A (EC 3.4.17.1), but with apreference for bulkier C-terminal residues.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.16','Membrane Pro-Xaa carboxypeptidase','Carboxypeptidase P','NULL','One of the renal brush border exopeptidases.','Release of a C-terminal residue other than proline, by preferentialcleavage of a prolyl bond.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.17','Tubulinyl-Tyr carboxypeptidase','Carboxypeptidase-tubulin','NULL','Active at neutral pH, from brain.','Cleavage of the -Glu-|-Tyr bond to release the C-terminal tyrosineresidue from the native tyrosinated tubulin. Inactive on Z-Glu-Tyr.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.18','Carboxypeptidase T','NULL','Zinc.','Belongs to peptidase family M14.','Releases a C-terminal residue, which may be hydrophobic or positivelycharged.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.19','Carboxypeptidase Taq','Carboxypeptidase 1','Zinc.','Most active at 80 degrees Celsius.Belongs to peptidase family M32.','Release of a C-terminal amino acid with broad specificity, except for-Pro.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.20','Carboxypeptidase U','Arginine carboxypeptidase','Zinc.','Pro-carboxypeptidase U in plasma is activated by thrombin or plasminduring clotting to form the unstable carboxypeptidase U.Activity similar to that of the more stable lysine carboxypeptidase,except no preference is shown for Lys over Arg.Belongs to peptidase family M14.','Release of C-terminal Arg and Lys from a polypeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.21','Glutamate carboxypeptidase II','Folate hydrolase','Zinc.','Hydrolyzes alpha-peptide bonds in Ac-Asp-Glu, Asp-Glu, and Glu-Glu,but also gamma-glutamyl bonds in gamma-Glu-Glu and folylpoly-gamma-glutamates.With folylpoly-gamma-glutamates, shows processive carboxypeptidaseactivity to produce pteroylmonoglutamate.Does not hydrolyze Ac-beta-Asp-Glu.Inhibited by quisqualic acid, Ac-beta-Asp-Glu, and 2-phosphonomethyl-pentanedioate.The release of C-terminal glutamate from folylpoly-gamma-glutamatesis also catalyzed by EC 3.4.17.11 and EC 3.4.19.9.Belongs to peptidase family M28.Formerly EC 3.4.19.8.','Release of an unsubstituted, C-terminal glutamyl residue, typically fromAc-Asp-Glu or folylpoly-gamma-glutamates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.22','Metallocarboxypeptidase D','Carboxypeptidase D','Zinc.','Activated by cobalt.Inhibited by guanidinoethylmercaptosuccinic acid.Belongs to peptidase family M14.','Releases C-terminal Arg and Lys from polypeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.17.23','Angiotensin-converting enzyme 2','ACE2','NULL','A transmembrane glycoprotein with an extracellular catalytic domain.Angiotensin-converting enzyme 2 functions as a carboxypeptidase,cleaving a single C-terminal residue from a distinct range ofsubstrates.Catalytic efficiency is 400-fold higher with angiotensin II (1-8) asa substrate than with angiotensin I (1-10).Angiotensin-converting enzyme 2 also efficiently hydrolyzes des-Arg(9)-bradykinin, but it does not hydrolyze bradykinin.Belongs to peptidase family M2.','Angiotensin II + H(2)O = angiotensin-(1-7) + L-phenylalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.18.1','Cathepsin X','Acid carboxypeptidase','NULL','Deamidates Z-Arg-NH(2), but shows no action on the arylamidederivatives.Inhibited by thiol-blocking reagents.Belongs to peptidase family C1.','Release of C-terminal amino acid residues with broad specificity,but lacks action on C-terminal proline. Shows weak endopeptidaseactivity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.1','Acylaminoacyl-peptidase','Acylamino-acid-releasing enzyme','NULL','Best if P1 = Ser, Ala, Met; poor if P1 = Gly, Tyr, Asp, Asn or Pro.Group of similar enzymes liberating N-acetyl or N-formyl amino acidfrom proteins and peptides.Active at neutral pH.Several variants of this enzyme exist; the human erythrocyte enzymeis relatively specific for removal of N-acetylalanine from peptides.Display dipeptidyl-peptidase activity on glycyl-peptides, perhaps asa result of misrecognition of the glycyl residue as an unchargedN-acyl group.Inhibited by diisopropyl fluorophosphate.Belongs to peptidase family S9C.Formerly EC 3.4.14.3.','Cleavage of an N-acetyl or N-formyl amino acid from the N-terminus of apolypeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.2','Peptidyl-glycinamidase','Carboxyamidase','NULL','Inactivates vasopressin and oxytocin by splitting off glycinamide.Also cleaves ester substrates of trypsin and chymotrypsin.Although glycinamide is by far the preferred leaving group, otheraminoacylamides may also be released, e.g. phenylalaninamide.The toad skin enzyme is inhibited by diisopropyl fluorophosphate.Formerly EC 3.4.15.2.','Cleavage of C-terminal glycinamide from polypeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.3','Pyroglutamyl-peptidase I','5-oxoprolyl-peptidase','NULL','Removes 5-oxoproline from various penultimate amino acid residuesexcept L-proline.Inhibited by thiol-blocking reagents.Belongs to peptidase family C15.Formerly EC 3.4.11.8.','Release of an N-terminal pyroglutamyl group from a polypeptide,the second amino acid generally not being Pro.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.4','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.5','Beta-aspartyl-peptidase','NULL','NULL','Other isopeptide bonds, e.g. gamma-glutamyl and beta-alanyl, are nothydrolyzed.Formerly EC 3.4.13.10.','Cleavage of a beta-linked Asp residue from the N-terminus of apolypeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.6','Pyroglutamyl-peptidase II','PAP-II','Zinc.','Highly specific for thyrotropin releasing hormone (pyroglutamyl-histidyl-prolylamide).Will not cleave the pyroglutamyl-histidyl bond of luteinizing hormonereleasing hormone.Found in serum and brain.Inhibited by metal chelators.Belongs to peptidase family M1.','Release of the N-terminal pyroglutamyl group from pGlu-|-His-Xaatripeptides and pGlu-|-His-Xaa-Gly tetrapeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.7','N-formylmethionyl-peptidase','(fMet)-releasing enzyme','NULL','Highly specific for N-formylmethionyl-peptides.Will not cleave methionyl peptides or N-formyl derivatives of aminoacids other than methionine.Isolated from liver.Inhibited by heavy metals and activated by chloride.','Release of an N-terminal, formyl-methionyl residue from a polypeptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.9','Gamma-glutamyl hydrolase','Carboxypeptidase G','NULL','A lysosomal or secreted, thiol-dependent peptidase, most active atacidic pH.Commonly studied with folylpoly-gamma-glutamate as substrate, withwhich the initial cleavage may release glutamate or poly-gamma-glutamate of two or more residues, according to the species of originof the enzyme.Final products are pteroyl-alpha-glutamate (folic acid) and freeglutamate.Highly specific for the gamma-glutamyl bond, but not for theC-terminal amino acid (leaving group).Action on gamma-glutamyl bonds is independent of an N-terminalpteroyl moiety, but it is not known whether an N-terminal gamma-Gluresidue can be hydrolyzed.Belongs to peptidase family C26.Formerly EC 3.4.12.10 and EC 3.4.22.12.','Hydrolysis of a gamma-glutamyl bond.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.10','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.11','Gamma-D-glutamyl-meso-diaminopimelate peptidase','Endopeptidase I','NULL','Metallopeptidase from Bacillus sphaericus, the substrates beingcomponents of the bacterial spore wall.Belongs to peptidase family M14.','Hydrolysis of gamma-D-glutamyl bonds to the L-terminus (position 7) ofmeso-diaminopimelic acid (meso-A(2)pm) in 7-(L-Ala-gamma-D-Glu)-meso-A(2)pm and 7-(L-Ala-gamma-D-Glu)-7-(D-Ala)-meso-A(2)pm. It is requiredthat the D-terminal amino and carboxy groups of meso-A(2)pm areunsubstituted.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.12','Ubiquitinyl hydrolase 1','Ubiquitin C-terminal hydrolase','NULL','Links to polypeptides smaller than 60 residues are hydrolyzed morereadily than those to larger polypeptides.Isoforms exist with quantitatively different specificities among thebest known being UCH-L1 and UCH-L3, major proteins of the brain ofmammals.Inhibited by ubiquitin aldehyde (in which Gly76 is replaced byaminoacetaldehyde).Belongs to peptidase family C12.','Thiol-dependent hydrolysis of ester, thioester, amide, peptide andisopeptide bonds formed by the C-terminal Gly of ubiquitin (a 76-residueprotein attached to proteins as an intracellular targeting signal).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.13','Glutathione hydrolase','Gamma-glutamyltranspeptidase','NULL','This protein also has EC 2.3.2.2 activity.The enzyme consists of two chains that are created by the proteolyticcleavage of a single precursor polypeptide.The N-terminal L-threonine of the C-terminal subunit functions as theactive site for both the cleavage and the hydrolysis reactions.The human enzyme also hydrolyzes oxidized glutathione and leukotrieneC4 with similar efficiency, while the mouse enzyme does not.','Glutathione + H(2)O = L-cysteinylglycine + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.19.14','Leukotriene-C(4) hydrolase','Gamma-glutamyl leukotrienase','NULL','The mouse enzyme is specific for leukotriene C(4), while the humanenzyme also has considerable activity toward glutathione and oxidizedglutathione (cf. EC 3.4.19.13).','Leukotriene C(4) + H(2)O = leukotriene D(4) + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.1','Chymotrypsin','Alpha-chymotrypsin','NULL','Belongs to peptidase family S1.Formerly EC 3.4.4.5 and EC 3.4.4.6.','Preferential cleavage: Tyr-|-Xaa, Trp-|-Xaa, Phe-|-Xaa, Leu-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.2','Chymotrypsin C','Caldecrin','NULL','Reacts more readily with Tos-Leu-CH(2)Cl than Tos-Phe-CH(2)Cl,in contrast to chymotrypsin.Belongs to peptidase family S1.','Preferential cleavage: Leu-|-Xaa, Tyr-|-Xaa, Phe-|-Xaa, Met-|-Xaa,Trp-|-Xaa, Gln-|-Xaa, Asn-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.3','Metridin','Metridium proteinase A','NULL','Digestive enzyme from the sea anemone Metridium senile.','Preferential cleavage: Tyr-|-Xaa, Phe-|-Xaa, Leu-|-Xaa; little action onTrp-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.4','Trypsin','Alpha-trypsin','NULL','Belongs to peptidase family S1.Formerly EC 3.4.4.4.','Preferential cleavage: Arg-|-Xaa, Lys-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.5','Thrombin','Fibrinogenase','NULL','More selective than trypsin and plasmin.Belongs to peptidase family S1.Formerly EC 3.4.4.13.','Selective cleavage of Arg-|-Gly bonds in fibrinogen to form fibrin andrelease fibrinopeptides A and B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.6','Coagulation factor Xa','Prothrombase','NULL','A blood coagulation factor formed from the proenzyme factor X bylimited proteolysis.Factor X is a glycoprotein composed of a heavy chain and a lightchain, which are generated from a precursor protein by the excisionof the tripeptide RKR and held together by one or more disulfidebonds.The activated factor Xa converts prothrombin to thrombin in thepresence of factor Va, Ca(2+) and phospholipids.Scutelarin (EC 3.4.21.60) has similar specificity, but does notrequire factor Va.Belongs to peptidase family S1.','Selective cleavage of Arg-|-Thr and then Arg-|-Ile bonds in prothrombinto form thrombin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.7','Plasmin','Fibrinase','NULL','Formed from plasminogen by proteolysis which results in multipleforms of the active plasmin.Belongs to peptidase family S1.Formerly EC 3.4.4.14.','Preferential cleavage: Lys-|-Xaa ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.9','Enteropeptidase','Enterokinase','NULL','Activates trypsinogen.Not inhibited by protein inhibitors of trypsin.Belongs to peptidase family S1.Formerly EC 3.4.4.8.','Activation of trypsinogen by selective cleavage of 6-Lys-|-Ile-7 bond.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.10','Acrosin','NULL','NULL','Inhibited by naturally occurring trypsin inhibitors.Occurs in spermatozoa.Belongs to peptidase family S1.','Preferential cleavage: Arg-|-Xaa, Lys-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.11','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.12','Alpha-lytic endopeptidase','Alpha-lytic protease','NULL','Belongs to peptidase family S1E.','Preferential cleavage: Ala-|-Xaa, Val-|-Xaa in bacterial cell walls,elastin and other proteins.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.13','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.14','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.15','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.16','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.17','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.18','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.19','Glutamyl endopeptidase','Endoproteinase Glu-C','NULL','In appropriate buffer the specificity is restricted to Glu-|-Xaa.Bonds involving bulky side-chains of hydrophobic amino acids arecleaved at a lower rate.Belongs to peptidase family S1B.','Preferential cleavage: Glu-|-Xaa, Asp-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.20','Cathepsin G','NULL','NULL','From azurophil granules of polymorphonuclear leukocytes.Belongs to peptidase family S1.','Specificity similar to chymotrypsin C.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.21','Coagulation factor VIIa','NULL','NULL','Formed from the precursor factor VII.The cattle enzyme is more readily inhibited by diisopropylfluorophosphate than the human.Belongs to peptidase family S1.','Selective cleavage of Arg-|-Ile bond in factor X to form factor Xa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.22','Coagulation factor IXa','Christmas factor','NULL','The proenzyme factor IX is activated by EC 3.4.21.27.Belongs to peptidase family S1.','Selective cleavage of Arg-|-Ile bond in factor X to form factor Xa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.23','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.24','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.25','Cucumisin','NULL','NULL','From the sarcocarp of the musk melon (Cucumis melo).Other endopeptidases from plants, which are less well characterizedbut presumably of serine-type, include euphorbain from Euphorbiacerifera, solanain from horse-nettle Solanum elaeagnifolium, hurainfrom Hura crepitans and tabernamontanain from Tabernamontanagrandiflora.Belongs to peptidase family S8.','Hydrolysis of proteins with broad specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.26','Prolyl oligopeptidase','Post-proline cleaving enzyme','NULL','Found in vertebrates, plants and Flavobacterium.Generally cytosolic, commonly activated by thiol compounds.Belongs to peptidase family S9A.Formerly EC 3.4.22.18.','Hydrolysis of Pro-|-Xaa ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.27','Coagulation factor XIa','Plasma thromboplastin antecedent','NULL','The proenzyme XI is activated by EC 3.4.21.38.Belongs to peptidase family S1.','Selective cleavage of Arg-|-Ala and Arg-|-Val bonds in factor IX to formfactor IXa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.28','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.29','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.30','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.31','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.32','Brachyurin','Collagenolytic protease','NULL','From hepatopancreas of the fiddler crab (Uca pugilator).Other serine endopeptidases that degrade collagen, but are not listedseparately here, include a second endopeptidase from U.pugilator,digestive enzymes from other decapod crustacea, and an enzyme fromthe fungus Entomophthora coronata.Belongs to peptidase family S1.','Hydrolysis of proteins, with broad specificity for peptide bonds. Nativecollagen is cleaved about 75% of the length of the molecule from theN-terminus. Low activity on small molecule substrates of both trypsin andchymotrypsin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.33','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.34','Plasma kallikrein','Kininogenin','NULL','Formed from plasma prokallikrein (Fletcher factor) by factor XIIa.Activates coagulation factors XII, VII and plasminogen.Selective for Arg ','Cleaves selectively Arg-|-Xaa and Lys-|-Xaa bonds, including Lys-|-Argand Arg-|-Ser bonds in (human) kininogen to release bradykinin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.35','Tissue kallikrein','Glandular kallikrein','NULL','Formed from tissue prokallikrein by activation with trypsin.A large number of tissue kallikrein-related sequences have beenreported for rats and mice, though fewer seem to exist in othermammals.The few that have been isolated and tested on substrates includemouse EC 3.4.21.54, submandibular proteinase a, epidermal growth-factor-binding protein, nerve growth factor gamma-subunit, rat tonin,submaxillary proteinases A and B, T-kininogenase, kallikreins K7 andK8.The rat enzyme is unusual in liberating bradykinin directly fromautologous kininogens by cleavage at two Arg-|-Xaa bonds.Belongs to peptidase family S1.Formerly EC 3.4.21.8.','Preferential cleavage of Arg-|-Xaa bonds in small molecule substrates.Highly selective action to release kallidin (lysyl-bradykinin) fromkininogen involves hydrolysis of Met-|-Xaa or Leu-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.36','Pancreatic elastase','Pancreatic elastase I','NULL','Formed by activation of proelastase from mammalian pancreas bytrypsin.Belongs to peptidase family S1.Formerly EC 3.4.4.7 and EC 3.4.21.11.','Hydrolysis of proteins, including elastin. Preferential cleavage:Ala-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.37','Leukocyte elastase','Bone marrow serine protease','NULL','Differs from pancreatic elastase in specificity on syntheticsubstrates and in inhibitor sensitivity.Belongs to peptidase family S1.Formerly EC 3.4.4.7 and EC 3.4.21.11.','Hydrolysis of proteins, including elastin. Preferential cleavage:Val-|-Xaa ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.38','Coagulation factor XIIa','Hageman factor','NULL','Also activates plasma prekallikrein and plasminogen.Formed from the proenzyme, factor XII, by plasma kallikrein or factorXIIa.Belongs to peptidase family S1.','Selective cleavage of Arg-|-Ile bonds in factor VII to form factor VIIaand factor XI to form factor XIa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.39','Chymase','Mast cell protease I','NULL','In mast cell granules.Belongs to peptidase family S1.','Preferential cleavage: Phe-|-Xaa ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.40','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.41','Complement subcomponent C1r','NULL','NULL','Activates C1s by proteolytic cleavage so that it can, in turn,activate C2 and C4.Activated from proenzyme C1r in plasma by autocatalytic mechanismduring activation of the complement system by the classical route.Belongs to peptidase family S1.','Selective cleavage of Lys(or Arg)-|-Ile bond in complement subcomponentC1s to form the active form of C1s (EC 3.4.21.42).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.42','Complement subcomponent C1s','C1s esterase','NULL','Activated from proenzyme C1s in plasma by the activated complementsubcomponent C1r.Belongs to peptidase family S1.','Cleavage of Arg-|-Ala bond in complement component C4 to form C4a andC4b, and Lys(or Arg)-|-Lys bond in complement component C2 to form C2aand C2b: the classical pathway C3 convertase.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.43','Classical-complement-pathway C3/C5 convertase','C3 convertase','NULL','A complex of complement fragments C4b, C2a and C2b.Belongs to peptidase family S1.Formerly EC 3.4.21.44.','Selective cleavage of Arg-|-Ser bond in complement component C3 alpha-chain to form C3a and C3b, and Arg-|-Xaa bond in complement component C5alpha-chain to form C5a and C5b.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.44','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.45','Complement factor I','C3b inactivator','NULL','Cleavage of complement subcomponent C3b requires its binding tocofactor H or complement receptor CR1; cleavage of iC3b requirescomplement receptor CR1; cleavage of C4b requires C4b-bindingprotein.Belongs to peptidase family S1.','Inactivates complement subcomponents C3b, iC3b and C4b by proteolyticcleavage.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.46','Complement factor D','C3 convertase activator','NULL','A component of the alternative pathway of complement activation.This reaction is analogous to the activation of complement componentC2 by activated complement subcomponent C1s.Belongs to peptidase family S1.','Selective cleavage of Arg-|-Lys bond in complement factor B when incomplex with complement subcomponent C3b or with cobra venom factor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.47','Alternative-complement-pathway C3/C5 convertase','C3 convertase','NULL','A bimolecular complex of complement fragment Bb with either C3b orcobra venom factor; Bb contains the active site.Bb is formed by cleavage of proenzyme factor B by factor D.Cleavage of complement component C5 requires additional C3b whichbinds C5 and renders it susceptible to cleavage by C3b,Bb complex.C3b,Bb is stabilized in plasma by factor P.Belongs to peptidase family S1.','Cleavage of Arg-|-Ser bond in complement component C3 alpha-chain toyield C3a and C3b, and Arg-|-Xaa bond in complement component C5 alpha-chain to yield C5a and C5b.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.48','Cerevisin','Proteinase yscB','NULL','Contains a Cys residue near the active site His, and is inhibited bymercurials.Proteinase ycaB is a similar enzyme from the yeast Candida albicans.Belongs to peptidase family S8.Formerly EC 3.4.22.9.','Hydrolysis of proteins with broad specificity, and of Bz-Arg-OEt ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.49','Hypodermin C','Hypoderma collagenase','NULL','Little action on small molecule substrates of trypsin, chymotrypsin,elastase or microbial collagenases.Belongs to peptidase family S1.','Hydrolysis of proteins including native collagen at Xaa-|-Ala bondleaving an N-terminal (75%) and a C-terminal (25%) fragment.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.50','Lysyl endopeptidase','Achromobacter proteinase I','NULL','Isolated from Achromobacter lyticus.Enzymes with similar specificity are produced by Lysobacterenzymogenes (endoproteinase Lys-C) and Pseudomonas aeruginosa (Ps-1).Belongs to peptidase family S1.','Preferential cleavage: Lys-|-Xaa, including Lys-|-Pro.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.51','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.52','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.53','Endopeptidase La','ATP-dependent protease La','NULL','ATP hydrolysis is linked with peptide bond hydrolysis.Vanadate inhibits both reactions.A similar enzyme occurs in animal mitochondria.Belongs to peptidase family S16.','Hydrolysis of proteins in presence of ATP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.54','Gamma-renin','NULL','NULL','Unlike EC 3.4.23.15, does not act on natural angiotensinogen.A member of the tissue kallikrein family, from submandibular glandsof male mice.Belongs to peptidase family S1.','Cleavage of the Leu-|-Leu bond in synthetic tetradecapeptide reninsubstrate, to produce angiotensin I, but not active on naturalangiotensinogen. Also hydrolyzes Bz-Arg-p-nitroanilide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.55','Venombin AB','Bitis gabonica serine proteinase','NULL','From the venom of the Gaboon viper Bitis gabonica.Activates factor XIII.Not inhibited by antithrombin III/heparin or hirudin, unlikeEC 3.4.21.5.','Preferential cleavage: Arg-|-Xaa bonds in fibrinogen to form fibrin andrelease fibrinopeptides A and B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.56','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.57','Leucyl endopeptidase','Leucine-specific serine proteinase','NULL','The enzyme from spinach leaves shows high specificity for leucine inpeptides and 4-nitroanilides.','Hydrolysis of proteins. Preferential cleavage: Leu-|-Xaa in smallmolecule substrates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.58','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.59','Tryptase','Lung tryptase','NULL','Occurs as a tetrameric molecule with high affinity for heparin,in mast cell granules.Not inhibited by alpha-1-proteinase inhibitor of alpha-2-macroglobulin.Belongs to peptidase family S1.','Preferential cleavage: Arg-|-Xaa, Lys-|-Xaa, but with more restrictedspecificity than trypsin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.60','Scutelarin','Taipan activator','NULL','From the venom of the Taipan snake (Oxyuranus scutellatus).Converts prothrombin to thrombin.Specificity is similar to that of Factor Xa (EC 3.4.21.6).However, unlike Factor Xa this enzyme can cleave its target in theabsence of coagulation Factor Va.Activity is potentiated by phospholipid and Ca(2+) which binds viagamma-carboxyglutamic acid residues.Similar enzymes are known from the venom of other Australian elapidsnakes, including Pseudonaja textilis textilis, Oxyuranusmicrolepidotus and Demansia nuchalis affinis.Formerly EC 3.4.99.28.','Selective cleavage of Arg-|-Thr and Arg-|-Ile in prothrombin to formthrombin and two inactive fragments.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.61','Kexin','Paired-basic endopeptidase','NULL','Calcium activated.Inhibited by p-mercuribenzoate.Belongs to peptidase family S8.Formerly EC 3.4.22.23.','Cleavage of -Lys-Arg-|-Xaa- and -Arg-Arg-|-Xaa- bonds to process yeastalpha-factor pheromone and killer toxin precursors.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.62','Subtilisin','NULL','NULL','Subtilisin is a serine endopeptidase that evolved independently ofchymotrypsin.It contains no cysteine residues (although these are found inhomologous enzymes).Species variants include subtilisin BPN (also subtilisin B,subtilopeptidase C, nagarse, nagarse proteinase, subtilisin Novo,bacterial proteinase Novo) and subtilisin Carlsberg (subtilisin A,subtilopeptidase A, alcalase Novo).Similar enzymes are produced by various Bacillus subtilis strains andother Bacillus species.Belongs to peptidase family S8.Formerly EC 3.4.4.16 and EC 3.4.21.14.','Hydrolysis of proteins with broad specificity for peptide bonds, and apreference for a large uncharged residue in P1. Hydrolyzes peptideamides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.63','Oryzin','Aspergillopeptidase B','NULL','Predominant extracellular alkaline endopeptidase of the moldAspergillus oryzae.Identical or closely related enzymes are produced by A.flavus andA.sojae.Belongs to peptidase family S8.Formerly EC 3.4.4.16, EC 3.4.21.14 and EC 3.4.21.15.','Hydrolysis of proteins with broad specificity, and of Bz-Arg-OEt ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.64','Peptidase K','Endopeptidase K','NULL','From the mold Tritirachium album Limber.Contains two disulfide bridges and one free Cys near the active siteHis.Belongs to peptidase family S8.Formerly EC 3.4.4.16 and EC 3.4.21.14.','Hydrolysis of keratin, and of other proteins with subtilisin-likespecificity. Hydrolyzes peptide amides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.65','Thermomycolin','Thermomycolase','NULL','From the thermophilic fungus Malbranchea pulchella var. sulfurea.Contains Cys, but not inhibited by p-mercuribenzoate.Very thermostable.Belongs to peptidase family S8.Formerly EC 3.4.4.16 and EC 3.4.21.14.','Rather non-specific hydrolysis of proteins. Preferential cleavage:-Ala-|-Xaa-, -Tyr-|-Xaa-, -Phe-|-Xaa- in small molecular substrates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.66','Thermitase','Thermophilic Streptomyces serine protease','NULL','From Thermoactinomyces vulgaris containing a single Cys, near theactive site His, and inhibited by p-mercuribenzoate.The N-terminal extension of the polypeptide chain relative tosubtilisin contributes to calcium-binding and the highthermostability.The amino acid composition and properties of the thermostable enzymefrom Streptomyces rectus var. proteolyticus are closely similar.Belongs to peptidase family S8.Formerly EC 3.4.4.16.','Hydrolysis of proteins, including collagen.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.67','Endopeptidase So','E','NULL','Cytoplasmic peptidase.Inhibited by Tos-Phe-CH(2)Cl, but not by Tos-Lys-CH(2)Cl.Formerly EC 3.4.4.16 and EC 3.4.21.14.','Hydrolysis of proteins, but not Bz-Tyr-OEt, Ac-Phe-beta-naphthylester,or Bz-Arg-OEt.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.68','T-plasminogen activator','Tissue plasminogen activator','NULL','From a wide variety of mammalian tissues, especially endothelialcells.Secreted as a single chain precursor which is cleaved to a two-chainform by plasmin.Activity is considerably enhanced by fibrin.Belongs to peptidase family S1.Formerly EC 3.4.21.31 and EC 3.4.99.26.','Specific cleavage of Arg-|-Val bond in plasminogen to form plasmin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.69','Protein C (activated)','Activated blood coagulation factor XIV','NULL','One of the gamma-carboxyglutamic acid-containing coagulation factors.Formed from protein C, the proenzyme that circulates in plasma,by the action of a complex of thrombin with thrombomodulin, or byserine endopeptidases present in several snake venoms.Belongs to peptidase family S1.','Degradation of blood coagulation factors Va and VIIIa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.70','Pancreatic endopeptidase E','Cholesterol-binding proteinase','NULL','From pancreatic juice.Unlike elastases, has an acidic pI.Binds cholesterol.Belongs to peptidase family S1.','Preferential cleavage: Ala-|-Xaa. Does not hydrolyze elastin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.71','Pancreatic elastase II','NULL','NULL','Formed by activation of proelastase II from mammalian pancreas bytrypsin.Usually, only one of the pancreatic elastases (see also EC 3.4.21.36)is expressed in a given species; human pancreatic elastase is of typeII.Belongs to peptidase family S1.','Preferential cleavage: Leu-|-Xaa, Met-|-Xaa and Phe-|-Xaa. Hydrolyzeselastin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.72','IgA-specific serine endopeptidase','IgA protease','NULL','Species variants differing slightly in specificity are secreted byGram-negative bacteria Neisseria gonorrhoeae and Haemophilusinfluenzae.Some other bacterial endopeptidases with similar specificity are ofmetallotype (see EC 3.4.24.13).Belongs to peptidase family S6.','Cleavage of immunoglobulin A molecules at certain Pro-|-Xaa bonds in thehinge region. No small molecule substrates are known.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.73','U-plasminogen activator','Cellular plasminogen activator','NULL','Formed from the inactive precursor by action of plasmin or plasmakallikrein.Differs in structure from EC 3.4.21.68, and does not bind to fibrin.Belongs to peptidase family S1.Formerly EC 3.4.21.31 and EC 3.4.99.26.','Specific cleavage of Arg-|-Val bond in plasminogen to form plasmin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.74','Venombin A','NULL','NULL','A somewhat thrombin-like enzyme from venoms of snakes of theviper/rattlesnake group.Species variants of the enzyme include ancrod from Agkistrodonrhodostoma (Malayan pit viper), batroxobin from Bothrops atrox (SouthAmerican pit viper) and crotalase from Crotalus adamanteus (Easterndiamondback rattlesnake).Does not require activation by calcium.Belongs to peptidase family S1.Formerly EC 3.4.21.28, EC 3.4.21.29 and EC 3.4.21.30.','Selective cleavage of Arg-|-Xaa bond in fibrinogen, to form fibrin,and release fibrinopeptide A. The specificity of further degradation offibrinogen varies with species origin of the enzyme.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.75','Furin','Dibasic processing enzyme','NULL','Activated by calcium.Belongs to peptidase family S8.','Release of mature proteins from their proproteins by cleavage of -Arg-Xaa-Yaa-Arg-|-Zaa- bonds, where Xaa can be any amino acid and Yaa is Argor Lys. Releases albumin, complement component C3 and von Willebrandfactor from their respective precursors.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.76','Myeloblastin','Leukocyte proteinase 3','NULL','Not inhibited by secretory leukocyte proteinase inhibitor.Belongs to peptidase family S1.','Hydrolysis of proteins, including elastin, by preferential cleavage:-Ala-|-Xaa- ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.77','Semenogelase','Gamma-seminoprotein','NULL','Slowly inhibited by alpha-1-antichymotrypsin.Belongs to peptidase family S1.','Preferential cleavage: -Tyr-|-Xaa-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.78','Granzyme A','Cytotoxic T-lymphocyte proteinase 1','NULL','From cytotoxic T lymphocyte granules.The human enzyme does not cleave Phe-|-Xaa.Belongs to peptidase family S1.','Hydrolysis of proteins, including fibronectin, type IV collagen andnucleolin. Preferential cleavage: -Arg-|-Xaa-, -Lys-|-Xaa- ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.79','Granzyme B','Cytotoxic t-lymphocyte proteinase 2','NULL','From cytotoxic T lymphocyte granules.Belongs to peptidase family S1.','Preferential cleavage: -Asp-|-Xaa- ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.80','Streptogrisin A','Pronase enzyme A','NULL','Not inhibited by Tos-Phe-CH(2)Cl or ovomucoid.Belongs to peptidase family S1E.','Hydrolysis of proteins with specificity similar to chymotrypsin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.81','Streptogrisin B','Pronase enzyme B','NULL','Belongs to peptidase family S1E.','Hydrolysis of proteins with trypsin-like specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.82','Glutamyl endopeptidase II','GluSGP','NULL','Inhibited by (18-Leu-','Preferential cleavage: -Glu-|-Xaa- ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.83','Oligopeptidase B','Protease II','NULL','Inhibited by Tos-Lys-CH(2)Cl.Belongs to peptidase family S9A.','Hydrolysis of -Arg-|-Xaa- and -Lys-|-Xaa- bonds in oligopeptides, evenwhen P1 residue is proline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.84','Limulus clotting factor C','NULL','NULL','From the hemocyte granules of the horseshoe crabs Limulus andTachypleus.Factor C is activated by Gram-negative bacterial lipopolysaccharidesand chymotrypsin.Inhibited by antithrombin III.Belongs to peptidase family S1.','Selective cleavage of 103-Arg-|-Ser-104 and 124-Ile-|-Ile-125 bonds inLimulus clotting factor B to form activated factor B. Cleavage of -Pro-Arg-|-Xaa- bonds in synthetic substrates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.85','Limulus clotting factor B','NULL','NULL','From the hemocyte granules of the horseshoe crabs Limulus andTachypleus.Factor B is activated by factor C.','Selective cleavage of 98-Arg-|-Ile-99 bond in Limulus proclotting enzymeto form active clotting enzyme.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.86','Limulus clotting enzyme','NULL','NULL','From the hemocyte granules of the horseshoe crabs Limulus andTachypleus.Limulus clotting enzyme is activated by factor B.Belongs to peptidase family S1.','Selective cleavage of 18-Arg-|- and 47-Arg-|- bonds in coagulogen to formcoagulin and fragments.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.87','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.88','Repressor LexA','NULL','NULL','In the presence of single-stranded DNA, the RecA protein interactswith LexA causing an autocatalytic cleavage which disrupts the DNA-binding part of LexA, leading to derepression of the SOS regulon andeventually DNA repair.The activity was previously attributed to the RecA protein (formerlyEC 3.4.99.37).Belongs to peptidase family S24.','Hydrolysis of Ala-|-Gly bond in repressor LexA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.89','Signal peptidase I','Bacterial leader peptidase I','NULL','Unaffected by inhibitors of most serine peptidases, but site-directedmutagenesis implicates a Ser/Lys catalytic dyad in activity.Cleaves a single bond -Ala-|-Ala- in M13 phage procoat protein,producing free signal peptide and coat protein.Eukaryote signal peptidases that may have somewhat differentspecificity are known from the endoplasmic reticulum membrane andmitochondrial inner membrane.Belongs to peptidase family S26.Formerly EC 3.4.99.36.','Cleavage of hydrophobic, N-terminal signal or leader sequences fromsecreted and periplasmic proteins.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.90','Togavirin','NsP2 proteinase','NULL','Once released, the core protein does not retain catalytic activity.Belongs to peptidase family S3.','Autocatalytic release of the core protein from the N-terminus of thetogavirus structural polyprotein by hydrolysis of a -Trp-|-Ser- bond.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.91','Flavivirin','NS2B-3 proteinase','NULL','NULL','Selective hydrolysis of -Xaa-Xaa-|-Yaa- bonds in which each of the Xaacan be either Arg or Lys and Yaa can be either Ser or Ala.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.92','Endopeptidase Clp','Caseinolytic protease','NULL','Belongs to peptidase family S14.','Hydrolysis of proteins to small peptides in the presence of ATP andmagnesium. Alpha-casein is the usual test substrate. In the absence ofATP, only oligopeptides shorter than five residues are hydrolyzed (suchas succinyl-Leu-Tyr-|-NHMec; and Leu-Tyr-Leu-|-Tyr-Trp, in which cleavageof the -Tyr-|-Leu- and -Tyr-|-Trp bonds also occurs).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.93','Proprotein convertase 1','NEC 1','NULL','A calcium dependent enzyme, maximally active at about pH 5.5.Substrates include pro-opiomelanocortin, prorenin, proenkephalin,prodynorphin, prosomatostatin and proinsulin.Unlike prohormone convertase 2, does not cleave proluteinizing-hormone-releasing hormone.Usually processing of prodynorphin occurs at a bond in which P2 isThr.Present in the regulated secretory pathway of neuroendocrine cells,commonly actin co-operatively with prohormone convertase 2.Belongs to peptidase family S8.','Release of protein hormones, neuropeptides and renin from theirprecursors, generally by hydrolysis of -Lys-Arg-|- bonds.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.94','Proprotein convertase 2','NEC 2','NULL','A calcium dependent enzyme, maximally active at about pH 5.5.Specificity is broader than that of prohormone convertase 1.Substrates include pro-opiomelanocortin, proenkephalin, prodynorphin,proinsulin and proluteinizing-hormone-releasing hormone.Unlike prohormone convertase 1, does not cleave prorenin orprosomatostatin.Usually processing of prodynorphin occurs at a bond in which P2 isThr.Present in the regulated secretory pathway of neuroendocrine cells,commonly actin co-operatively with prohormone convertase 1.Belongs to peptidase family S8.','Release of protein hormones and neuropeptides from their precursors,generally by hydrolysis of -Lys-Arg-|- bonds.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.95','Snake venom factor V activator','Factor V-activating proteinase','NULL','Inhibited by diisopropyl fluorophosphate.Belongs to peptidase family S1.','Fully activates human clotting factor V by a single cleavage at the 1545-Trp-Tyr-Leu-Arg-|-Ser-Asn-Asn-Gly-1552 bond. Cattle, but not rabbit,factor V is cleaved, and no other proteins of the clotting system areattacked. Esterase activity is observed on Bz-Arg-OEt and Tos-Arg-OMe,and amidase activity on Phe-pipecolyl-Arg-NHPhNO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.96','Lactocepin','Extracellular lactococcal proteinase','NULL','Responsible for the hydrolysis of casein in milk and the provision ofpeptides essential for cell growth.Specificity differences between lactocepins from different starterstrains may be partly responsible for imparating different flavorqualities to cheese.Belongs to peptidase family S8.','Endopeptidase activity with very broad specificity, although some subsitepreference have been noted, e.g. large hydrophobic residues in the P1 andP4 positions, and Pro in the P2 position. Best known for its action oncaseins, although it has been shown to hydrolyze hemoglobin and oxidizedinsulin B-chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.97','Assemblin','NULL','NULL','Involved in the breakdown of the scaffold protein during the latestages of assembly of the herpes-virus virion.Inhibited by diisopropyl fluorophosphate.Belongs to peptidase family S21.','Cleaves -Ala-|-Ser- and -Ala-|-Ala- bonds in the scaffold protein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.98','Hepacivirin','Cpro-2','NULL','Encoded by the genome of the viruses of the hepatitis C group,and contributes to the maturation of the precursor polyproteins.The enzyme is greatly activated by binding of the 54-residue NS4Acofactor protein also derived from the viral polyprotein.Belongs to peptidase family S29.','Hydrolysis of four peptide bonds in the viral precursor polyprotein,commonly with Asp or Glu in the P6 position, Cys or Thr in P1 and Ser orAla in P1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.99','Spermosin','NULL','NULL','The enzyme from the ascidian (Prochordate) Halocynthia roretzi islocalized in the sperm head, and released during sperm activation.A proline-rich region is involved in binding to the vitelline coat ofthe egg.Belongs to peptidase family S1.','Hydrolyzes arginyl bonds, preferably with Pro in the P2 position.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.100','Sedolisin','Pseudomonalisin','NULL','An enzyme secreted by Pseudomonas sp. No. 101.Optimum pH 4.It is distinguished from EC 3.4.21.101 by its insensitivity to EPNPand from EC 3.4.23.31 by this property and by its unrelated amino-acid sequence.Inhibited by tyrostatin, a peptide aldehyde.Belongs to peptidase family S53.Formerly EC 3.4.23.37.','Hydrolysis of the B chain of insulin at 13-Glu-|-Ala-14, 15-Leu-|-Tyr-16and 25-Phe-|-Tyr-26 and angiotensin I at 4-Tyr-|-Ile-5. A good syntheticsubstrate is Lys-Pro-Ile-Glu-Phe-|-Phe(NO(2))-Arg-Leu.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.101','Xanthomonalisin','Sedolisin-B','NULL','Secreted by the bacterium Xanthomonas sp.Belongs to peptidase family S53.Formerly EC 3.4.23.33.','Cleavage of casein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.102','C-terminal processing peptidase','Photosystem II D1 protein processing peptidase','NULL','In the plant chloroplast, the enzyme removes the C-terminal extensionof the D1 polypeptide of photosystem II; this proteolytic processingis necessary to allow the light-driven assembly of the tetranuclearmanganese cluster, which is responsible for photosynthetic wateroxidation.Belongs to peptidase family S41.','The enzyme shows specific recognition of a C-terminal tripeptide, Xaa-Yaa-Zaa, in which Xaa is preferably Ala or Leu, Yaa is preferably Ala orTyr, and Zaa is preferably Ala, but then cleaves at a variable distancefrom the C-terminus. A typical cleavage is -Ala-Ala-|-Arg-Ala-Ala-Lys-Glu-Asn-Tyr-Ala-Leu-Ala-Ala.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.103','Physarolisin','Dictyostelium discoideum aspartic proteinase','NULL','From the slime mold Physarum polycephalum.Is not inhibited by pepstatin, but is blocked by methyl2-diazoacetamidohexanoate.Closely similar enzymes are found in Dictyostelium discoideum andP.flavicomum.Belongs to peptidase family S53.Formerly EC 3.4.4.17, EC 3.4.23.6 and EC 3.4.23.27.','Milk clotting activity. Preferential cleavage of 8-Gly-|-Ser-9 in B chainof insulin most rapidly, followed by 11-Leu-|-Val-12,19-Cys(SO(3)H)-|-Gly and 24-Phe-|-Phe-25. No action on Ac-Phe-Tyr(I)(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.104','Mannan-binding lectin-associated serine protease-2','Mannan-binding lectin-associated serine peptidase 2','NULL','Mannan-binding lectin (MBL) recognizes patterns of neutralcarbohydrates, such as mannose and N-acetylglucosamine, on a widerange of microbial surfaces and is able to initiate activation of thelectin pathway of complement.Displays C1s-like esterolytic activity (cf. EC 3.4.21.42).It also cleaves C4 and C2 with efficiencies that are relativelyhigher than those of EC 3.4.21.42.Belongs to peptidase family S1A.','Selective cleavage after Arg-223 in complement component C2 (-Ser-Leu-Gly-Arg-|-Lys-Ile-Gln-Ile) and after Arg-76 in complement componentC4 (-Gly-Leu-Gln-Arg-|-Ala-Leu-Glu-Ile).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.105','Rhomboid protease','NULL','NULL','These endopeptidases are multi-spanning membrane proteins.Their catalytic site is embedded within the membrane and they cleavetype-1 transmembrane domains.Important for embryo development in Drosophila melanogaster.Rhomboid is a key regulator of EGF receptor signaling and isresponsible for cleaving Spitz, the main ligand of the Drosophila EGFreceptor pathway.Parasite-encoded rhomboid enzymes are also important for invasion ofhost cells by Toxoplasma and the malaria parasite.Belongs to peptidase family S54.','Cleaves type-1 transmembrane domains using a catalytic dyad composed ofserine and histidine that are contributed by different transmembranedomains.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.106','Hepsin','NULL','NULL','This type-II membrane-associated serine peptidase has been implicatedin cell growth and development.The enzyme has been shown to activate blood coagulation factor VII bycleavage of the 152-Arg-|-Ile-153 peptide bound in BHK cells, thusindicating a possible role in the initiation of blood coagulation.There is no cleavage after aromatic or aliphatic residues.The occupancy of the S2 site is an absolute requirement for catalysisand a basic residue at that site is preferred to an aliphaticresidue.The nature of the residue at S3 also affects hydrolysis, with Glnbeing much more favorable than Ala.Belongs to peptidase family S1A.','Cleavage after basic amino-acid residues, with Arg strongly preferred toLys.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.107','Peptidase Do','High temperature requirement protease A','NULL','This serine endopeptidase is essential for the clearance of denaturedor aggregated proteins from the inner-membrane and periplasmic spacein Escherichia coli.Natural substrates of the enzyme include colicin A lysis protein,pilin subunits and MalS from E.coli.The enzyme has weak peptidase activity with casein and other non-native substrates.The peptidase acts as a chaperone at low temperatures but switches toa peptidase (heat shock protein) at higher temperatures.Molecular chaperones and peptidases control the folded state ofproteins by recognizing hydrophobic stretches of polypeptide thatbecome exposed by misfolding or unfolding.They then bind these hydrophobic substrates to prevent aggregation orassist in protein refolding.If attempts at refolding fail, then irreversibly damaged proteins aredegraded by peptidases such as this enzyme.Belongs to peptidase family S1B.','Acts on substrates that are at least partially unfolded. The cleavagesite P1 residue is normally between a pair of hydrophobic residues, suchas Val-|-Val.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.108','HtrA2 peptidase','High temperature requirement protein A2','NULL','Up-regulated in mammalian cells in response to stress induced by bothheat shock and tunicamycin treatment.Can induce apoptosis in a caspase-independent manner through itspeptidase activity and in a caspase-dependent manner by disruptingthe interaction between caspase and the inhibitor of apoptosis (IAP).Belongs to peptidase family S1B.','Cleavage of non-polar aliphatic amino-acids at the P1 position, with apreference for Val, Ile and Met. At the P2 and P3 positions, Arg isselected most strongly with a secondary preference for other hydrophilicresidues.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.109','Matriptase','Epithin','NULL','This trypsin-like integral-membrane serine peptidase has beenimplicated in breast cancer invasion and metastasis.Can activate hepatocyte growth factor/scattering factor (HGF/SF) bycleavage of the two-chain form at an Arg residue to give activealpha- and beta-HGF, but it does not activate plasminogen, whichshares high homology with HGF.Can also activate urokinase plasminogen activator (uPA), whichinitiates the matrix-degrading peptidase cascade.Belongs to peptidase family S1A.','Cleaves various synthetic substrates with Arg or Lys at the P1 positionand prefers small side-chain amino acids, such as Ala and Gly, at the P2position.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.110','C5a peptidase','SCP','NULL','Surface-associated subtilisin-like serine peptidase with veryspecific substrate specificity.Virulent strains of streptococci, including Streptococcus pyogenes,can evade human detection and phagocytosis by destroying thecomplement chemotaxin C5a.Cleavage of human C5a by this enzyme reduces the ability of C5a tobind receptors on the surface of polymorphonuclear neutrophilleukocytes (PMNLs) and thereby abolishes its chemotactic properties.Belongs to peptidase family S8A.','The primary cleavage site is at 67-His-|-Lys-68 in human C5a with a minorsecondary cleavage site at 58-Ala-|-Ser-59.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.111','Aqualysin 1','Caldolysin','NULL','This enzyme from the extreme thermophile, Thermus aquaticus, is analkaline serine peptidase.It has three subsites, S1, S2, and S3, in the substrate binding site.The preferred amino acids at the S1 site are Ala and Phe, at the S2site are Ala and norleucine and at the S3 site are Phe and Ile.These specificities are similar to those of EC 3.4.21.62 andEC 3.4.21.64.The enzyme displays broad specificity for cleavage of insulin B-chainand hydrolyzes elastin substrates such as succinyl-(Ala)(n)-p-nitroanilide (n = 1,2,3) and some peptide esters.Belongs to peptidase family S8A.','Exhibits low specificity toward esters of amino acids with smallhydrophobic or aromatic residues at the P1 position.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.112','Site-1 protease','Mammalian subtilisin/kexin isozyme 1','NULL','Cleaves sterol regulatory element-binding proteins (SREBPs) andthereby initiates a process by which the active fragments of theSREBPs translocate to the nucleus and activate genes controlling thesynthesis and uptake of cholesterol and unsaturated fatty acids intothe bloodstream.The enzyme also processes pro-brain-derived neurotrophic factor andundergoes autocatalytic activation in the endoplasmic reticulumthrough sequential cleavages.The enzyme can also process the unfolded protein response stressfactor ATF6 at an Arg-His-Lys-Lys-|- site, and the envelopeglycoprotein of the highly infectious Lassa virus and Crimean Congohemorrhagic fever virus at Arg-Arg-Lys-Lys-|-.Belongs to peptidase family S8A.','Processes precursors containing basic and hydrophobic/aliphatic residuesat P4 and P2, respectively, with a relatively relaxed acceptance of aminoacids at P1 and P3.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.113','Pestivirus NS3 polyprotein peptidase','BDV NS3 endopeptidase','NULL','The polyprotein of noncytopathogenic pestiviruses is cleaved co- andpost-translationally into at least 11 proteins (N(pro), C, E(rns),E1, E2, p7, NS2-3, NS4A, NS4B, NS5A, and NS5B).The genomes of cytopathogenic pestivirus strains express at least oneadditional protein, called NS3 (p80).This enzyme, which resides in the N-terminal region of NS3(nonstructural protein 3), is essential for generation of its ownC-terminus and for processing of the downstream cleavage sites,leading to the release of the pestivirus nonstructural proteins NS4A,NS4B, NS5A and NS5B.Belongs to peptidase family S31.','Leu is conserved at position P1 for all four cleavage sites. Alanine isfound at position P1 of the NS4A-NS4B cleavage site, whereas serine isfound at position P1 of the NS3-NS4A, NS4B-NS5A and NS5A-NS5B cleavagesites.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.114','Equine arterivirus serine peptidase','3C-like Ser protease','NULL','In the equine arterivirus (EAV), the replicase gene is translatedinto open reading frame 1a (ORF1a) and ORF1ab polyproteins.This enzyme is the main viral proteinase and processes five cleavagesites in the ORF1a protein and three in the ORF1b-encoded part of theORF1ab protein to yield nonstructural proteins (nsp5-nsp12).It combines the catalytic system of a chymotrypsin-like serinepeptidase (His-Asp-Ser catalytic triad) with the substratespecificity of a 3C-like serine peptidase (Glu or Gln) at the P1position and a small amino-acid residue (Gly, Ser or Ala) at the P1position.Cleavage of ORF1ab by this enzyme is essential for viral replication.Belongs to peptidase family S32.','Cleavage of (Glu/Gln)-|-(Gly/Ser/Ala) in arterivirus replicasetranslation products ORF1a and ORF1ab.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.115','Infectious pancreatic necrosis birnavirus Vp4 peptidase','Infectious pancreatic necrosis virus protease','NULL','Infectious pancreatic necrosis virus (IPNV) is a birnavirus thatcauses an acute, contagious disease in young salmonid fish.As with most viruses that infect eukaryotic cells, the proteolyticprocessing of viral precursor proteins is a crucial step in the lifecycle of this virus.pVP2 is converted into VP2 by cleavage near the carboxy end of pVP2;this cleavage is most likely due to host-cell proteases rather thanVP4.Differs from most serine peptidases in not having the catalytic triadSer-His-Asp.Belongs to peptidase family S50.','Cleaves the (Ser/Thr)-Xaa-Ala-|-(Ser/Ala)-Gly motif in the polyproteinNH(2)-pVP2-VP4-VP3-COOH of infectious pancreatic necrosis virus at thepVP2-VP4 and VP4-VP3 junctions.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.116','SpoIVB peptidase','Sporulation factor IV B protease','NULL','This enzyme plays a central role in a regulatory checkpoint (thesigma(K) checkpoint), which coordinates gene expression during thelater stages of spore formation in Bacillus subtilis.The enzyme activates proteolytic processing of a sporulation-specificsigma factor, pro-sigma(K), to its mature and active form, sigma(K),by self-cleavage.The enzyme is also subject to secondary proteolysis, which presumablyinactivates SpoIVB.The enzyme is also essential for the formation of heat-resistantspores.Belongs to peptidase family S55.','Self-cleaves 52-Val-|-Asn-53, 62-Ala-|-Phe-63 and 74-Val-|-Thr-75 at theN-terminus of SpoIVB.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.117','Stratum corneum chymotryptic enzyme','Kallikrein 7','NULL','This enzyme has wide substrate specificity, being able to degradeheat-denatured bovine casein and the alpha-chain of native humanfibrinogen.It cleaves the B chain of bovine insulin at 6-Leu-|-Cya-7 (cysteicacid), 16-Tyr-|-Leu-17, 25-Phe-|-Tyr-26 and 26-Tyr-|-Thr-27.It is thought to play a role in the desquamation (skin-shedding) ofthe outer layer of skin, the stratum corneum, by degradingintercellular cohesive structures.Belongs to peptidase family S1A.','Cleavage of proteins with aromatic side chains in the P1 position.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.118','Kallikrein 8','Neuropsin','NULL','Activated by removal of an N-terminal prepropeptide.The highest amidolytic activity is observed using Boc-Val-Pro-Arg-|-7-amido-4-methylcoumarin, which is a substrate of alpha-thrombin.Substrates lacking basic amino acids in the P1 position are notcleaved.Degrades casein, fibronectin, gelatin, collagen type IV, fibrinogen,and high-molecular-mass kininogen and is associated with diseasessuch as ovarian cancer and Alzheimers disease.Belongs to peptidase family S1A.','Cleavage of amide substrates following the basic amino acids Arg or Lysat the P1 position, with a preference for Arg over Lys.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.119','Kallikrein 13','Epidermal growth factor-binding protein type B','NULL','Specific for prorenin from the mouse submandibular gland, as proreninfrom the mouse kidney (Ren1) and human prorenin are not substrates.Site-directed mutagenesis studies have shown that the enzyme willalso cleave prorenin when Lys-Arg is replaced by Arg-Arg or Gln-Argbut the rate of reaction is much slower when Lys-Lys is used.Also able to process pro-interleukin-1-beta (pro-IL-1-beta) in mousesubmandibular gland to form IL-1-beta.Belongs to peptidase family S1A.','Hydrolyzes mouse Ren2 protein (a species of prorenin present in thesubmandibular gland) on the carboxy side of the arginine residue at theLys-Arg-|- pair in the N-terminus, to yield mature renin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.120','Oviductin','Oviductal protease','NULL','The egg envelope of Xenopus laevis is modified during transit of theegg through the pars rectus oviduct, changing the egg envelope froman unfertilizable form to a fertilizable form.This process involves the conversion of glycoprotein gp43 to gp41(ZPC) by the pars recta protease oviductin.It is thought that the enzymatically active protease moleculecomprises the N-terminal protease domain coupled to two C-terminalCUB domains, which are related to the mammalian spermadhesinmolecules implicated in mediating sperm-envelope interactions.Also found in Bufo japonicus.Belongs to peptidase family S1.','Preferential cleavage at 371-Gly-Ser-Arg-|-Trp-374 of glycoprotein gp43in Xenopus laevis coelemic egg envelope to yield gp41.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.21.121','Lys-Lys/Arg-Xaa endopeptidase','Aeromonas extracellular serine protease','NULL','The enzyme is a serine peptidase, which has been shown to cleaveprothrombin and prekallikrein.It hydrolyzes the complement component C5 releasing complementcomponent C5a.','Cleavage of -Lys-Lys-|-Xaa and -Lys-Arg-|-Xaa bonds.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.1','Cathepsin B','Cathepsin B1','NULL','An intracellular (lysosomal) enzyme.Belongs to peptidase family C1.','Hydrolysis of proteins with broad specificity for peptide bonds.Preferentially cleaves -Arg-Arg-|-Xaa bonds in small molecule substrates(thus differing from cathepsin L). In addition to being an endopeptidase,shows peptidyl-dipeptidase activity, liberating C-terminal dipeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.2','Papain','Papaya peptidase I','NULL','From latex of the papaya (Carica papaya).Inhibited by compound E-64 and proteins of the cystatin family.Belongs to peptidase family C1.Formerly EC 3.4.4.10.','Hydrolysis of proteins with broad specificity for peptide bonds,but preference for an amino acid bearing a large hydrophobic side chainat the P2 position. Does not accept Val in P1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.3','Ficain','Ficin','NULL','The major proteolytic component of the latex of fig, Ficus glabrata.Cysteine endopeptidases with similar properties are present in othermembers of the large genus Ficus.Belongs to peptidase family C1.Formerly EC 3.4.4.12.','Specificity similar to that of papain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.6','Chymopapain','Papaya proteinase II','NULL','The major endopeptidase of papaya (Carica papaya) latex.It has multiple chromatographic forms.Belongs to peptidase family C1.Formerly EC 3.4.4.11.','Specificity similar to that of papain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.7','Asclepain','NULL','NULL','From the latex of milkweed, Asclepias syriaca.It has multiple forms.Belongs to peptidase family C1.','Specificity similar to that of papain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.8','Clostripain','Clostridiopeptidase B','NULL','From the bacterium Clostridium histolyticum.Activated by calcium; inhibited by EDTA.Belongs to peptidase family C11.Formerly EC 3.4.4.20.','Preferential cleavage: Arg-|-Xaa, including Arg-|-Pro bond, but notLys-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.9','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.10','Streptopain','Streptococcal cysteine proteinase','NULL','From the bacterium, group A Streptococcus.Formed from the proenzyme by limited proteolysis.Belongs to peptidase family C10.Formerly EC 3.4.4.18.','Preferential cleavage with hydrophobic residues at P2, P1 and P1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.11','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.12','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.13','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.14','Actinidain','Actinidia anionic protease','NULL','From the kiwi fruit or Chinese gooseberry (Actinidia chinensis).Belongs to peptidase family C1.','Specificity close to that of papain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.15','Cathepsin L','NULL','NULL','A lysosomal enzyme that is readily inhibited by the diazomethaneinhibitor Z-Phe-Phe-CHN(2) or the epoxide inhibitor E-64.Belongs to peptidase family C1.','Specificity close to that of papain. As compared to cathepsin B,cathepsin L exhibits higher activity toward protein substrates, but haslittle activity on Z-Arg-Arg-NHMec, and no peptidyl-dipeptidase activity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.16','Cathepsin H','Aleurain','NULL','Present in lysosomes of mammalian cells.Because of the combination of endopeptidase and aminopeptidaseactivity it can be considered as endoaminopeptidase.Belongs to peptidase family C1.','Hydrolysis of proteins, acting as an aminopeptidase (notably, cleavingArg-|-Xaa bonds) as well as an endopeptidase.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.17','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.18','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.19','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.20','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.21','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.22','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.23','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.24','Cathepsin T','NULL','NULL','Degrades azocasein and denatured hemoglobin; the only native proteinon which it has been shown to act is EC 2.6.1.5.','Interconversion of the three forms of EC 2.6.1.5.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.25','Glycyl endopeptidase','Papaya peptidase B','NULL','From the papaya plant, Carica papaya.Not inhibited by chicken cystatin, unlike most other homologs ofpapain.Belongs to peptidase family C1.','Preferential cleavage: Gly-|-Xaa, in proteins and in small moleculesubstrates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.26','Cancer procoagulant','NULL','NULL','Apparently produced only by malignant and fetal cells.','Specific cleavage of Arg-|-Ile bond in Factor X to form Factor Xa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.27','Cathepsin S','NULL','NULL','A lysosomal cysteine endopeptidase that is unusual among such enzymesfor its stability to neutral pH.Belongs to peptidase family C1.','Similar to cathepsin L, but with much less activity on Z-Phe-Arg-|-NHMec,and more activity on the Z-Val-Val-Arg-|-Xaa compound.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.28','Picornain 3C','Foot-and-mouth protease 3C','NULL','From entero-, rhino-, aphto- and cardioviruses.Larger than the homologous virus picornain 2A.Belongs to peptidase family C3.','Selective cleavage of Gln-|-Gly bond in the poliovirus polyprotein. Inother picornavirus reactions Glu may be substituted for Gln, and Ser orThr for Gly.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.29','Picornain 2A','Picornavirus endopeptidase 2A','NULL','From entero- and rhinoviruses.Smaller than the homologous virus picornain 3C.Belongs to peptidase family C3.','Selective cleavage of Tyr-|-Gly bond in the picornavirus polyprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.30','Caricain','Papaya peptidase A','NULL','From the papaya plant, Carica papaya.Belongs to peptidase family C1.','Hydrolysis of proteins with broad specificity for peptide bonds, similarto those of papain and chymopapain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.31','Ananain','NULL','NULL','From stem of pineapple plant, Ananas comosus.Differs from stem and fruit bromelains in being inhibited by chickencystatin.Belongs to peptidase family C1.','Hydrolysis of proteins with broad specificity for peptide bonds. Bestreported small molecule substrate Bz-Phe-Val-Arg-|-NHMec, but broaderspecificity than fruit bromelain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.32','Stem bromelain','Bromelain','NULL','The most abundant of cysteine endopeptidases of the stem of thepineapple plant, Ananas comosus.Distinct from the bromelain found in the pineapple fruit(EC 3.4.22.33).Scarcely inhibited by chicken cystatin and also very slowlyinactivated by E-64.Belongs to peptidase family C1.Formerly EC 3.4.4.24 and EC 3.4.22.4.','Broad specificity for cleavage of proteins, but strong preference forZ-Arg-Arg-|-NHMec among small molecule substrates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.33','Fruit bromelain','NULL','NULL','From the pineapple plant, Ananas comosus.Scarcely inhibited by chicken cystatin.Another cysteine endopeptidase, with similar action on small moleculesubstrates, pinguinain (formerly EC 3.4.99.18), is obtained from therelated plant, Bromelia pinguin, but pinguinain differs from fruitbromelain in being inhibited by chicken cystatin.Belongs to peptidase family C1.Formerly EC 3.4.4.24, EC 3.4.22.4 and EC 3.4.22.5.','Hydrolysis of proteins with broad specificity for peptide bonds. Bz-Phe-Val-Arg-|-NHMec is a good synthetic substrate, but there is no action onZ-Arg-Arg-|-NHMec (c.f. stem bromelain).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.34','Legumain','Asparaginyl endopeptidase','NULL','Best known from legume seeds, the trematode Schistosoma mansoni andmammalian lysosomes.Not inhibited by compound E-64.Belongs to peptidase family C13.','Hydrolysis of proteins and small molecule substrates at -Asn-|-Xaa-bonds.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.35','Histolysain','Histolysin','NULL','From the protozoan, Entamoeba histolytica.Belongs to peptidase family C1.','Hydrolysis of proteins, including basement membrane collagen andazocasein. Preferential cleavage: Arg-Arg-|-Xaa in small moleculesubstrates including Z-Arg-Arg-|-NHMec.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.36','Caspase-1','Interleukin 1-beta converting enzyme','NULL','Part of the family of inflammatory-caspases, which also includescaspase-4 (EC 3.4.22.57) and caspase-5 (EC 3.4.22.58) in humans andcaspase-11 (EC 3.4.22.64), caspase-12, caspase-13 and caspase-14 inmice.Contains a caspase-recruitment domain (CARD) in its N-terminalprodomain, which plays a role in procaspase activation.Cleaves pro-interleukin-1-beta (pro-IL-1-beta) to form mature IL-1-beta, a potent mediator of inflammation.Also activates the proinflammatory cytokine, IL-18, which is alsoknown as interferon-gamma-inducing factor.Inhibited by Ac-Tyr-Val-Ala-Asp-CHO.Caspase-11 plays a critical role in the activation of caspase-1 inmice whereas caspase-4 enhances its activation in humans.Belongs to peptidase family C14.','Strict requirement for an Asp residue at position P1 and has a preferredcleavage sequence of Tyr-Val-Ala-Asp-|-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.37','Gingipain R','Arg-gingipain','NULL','Strongly activated by glycine and stabilized by calcium.Belongs to peptidase family C25.','Hydrolysis of proteins and small molecule substrates, with a preferencefor Arg in P1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.38','Cathepsin K','Cathepsin O','NULL','Prominently expressed in mammalian osteoclasts, and believed to playa role in bone resorption.Belongs to peptidase family C1.','Broad proteolytic activity. With small-molecule substrates andinhibitors, the major determinant of specificity is P2, which ispreferably Leu, Met ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.39','Adenain','NULL','NULL','A cysteine endopeptidase from adenoviruses.Activity is greatly stimulated by the binding to the enzyme of an11-residue peptide from the adenovirus capsid protein pre-VI at asite separate from the active site.Belongs to peptidase family C5.','Cleaves proteins of the adenovirus and its host cell at two consensussites: -Yaa-Xaa-Gly-Gly-|-Xaa- and -Yaa-Xaa-Gly-Xaa-|-Gly- (in which Yaais Met, Ile or Leu, and Xaa is any amino acid).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.40','Bleomycin hydrolase','Aminopeptidase C (Lactococcus lactis)','NULL','The active sites are on the walls of a central channel through themolecule, and access of substrate molecules to them is obstructed bythis and by the C-terminus of each polypeptide chain.Bleomycin can scarcely be the natural substrate, and there arereports of limited endopeptidase activity.Known from bacteria as well as eukaryotic organisms.Hydrolase H from chicken muscle has many similarities to bleomycinhydrolase, but hydrolyzes Ph-CO-Arg-2-naphthylamine as well asaminopeptidase substrates.Belongs to peptidase family C1.','Inactivates bleomycin B2 (a cytotoxic glycometallopeptide) by hydrolysisof a carboxyamide bond of beta-aminoalanine, but also shows generalaminopeptidase activity. The specificity varies somewhat with source,but amino acid arylamides of Met, Leu and Ala are preferred.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.41','Cathepsin F','NULL','NULL','The enzyme is unstable at neutral pH values and is inhibited bycompound E-64.Belongs to peptidase family C1.','The recombinant enzyme cleaves synthetic substrates with Phe and Leu(better than Val) in P2, with high specificity constant (k(cat)/K(m))comparable to that of cathepsin L.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.42','Cathepsin O','NULL','NULL','The recombinant human enzyme is catalytically active at pH 6.0 and isinhibited by compound E-64.Belongs to peptidase family C1.','The recombinant human enzyme hydrolyzes synthetic endopeptidasesubstrates including Z-Phe-Arg-NHMec and Z-Arg-Arg-NHMec.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.43','Cathepsin V','Cathepsin L2','NULL','Maximally active at pH 5.7 and unstable at neutral pH.Compound E-64, leupeptin and chicken cystatin are inhibitors.Belongs to peptidase family C1.','The recombinant enzyme hydrolyzes proteins (serum albumin, collagen) andsynthetic substrates (Z-Phe-Arg-NHMec ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.44','Nuclear-inclusion-a endopeptidase','Potyvirus NIa protease','NULL','The potyviruses cause diseases in plants, and inclusion bodies appearin the host cell nuclei, protein a of these being the endopeptidase.The enzyme finds practical use when encoded in vectors for theartificial expression of recombinant fusion proteins, since it canconfer on them the capacity for autolytic cleavage.It is also reported that transgenic plants expressing the enzyme areresistant to viral infection.Belongs to peptidase family C4.','Hydrolyzes glutaminyl bonds, and activity is further restricted bypreferences for the amino acids in P6 - P1 that vary with the species ofpotyvirus, e.g. Glu-Xaa-Xaa-Tyr-Xaa-Gln-|-(Ser or Gly) for the enzymefrom tobacco etch virus. The natural substrate is the viral polyprotein,but other proteins and oligopeptides containing the appropriate consensussequence are also cleaved.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.45','Helper-component proteinase','HC-Pro','NULL','Known from many potyviruses.The helper component-proteinase of the tobacco etch virus is amultifunctional protein with several known activities.The N-terminal region is required for aphid transmission andefficient genome amplification, the central region is required forlong-distance movement in plants, and the C-terminal domain hascysteine endopeptidase activity.Belongs to peptidase family C6.','Hydrolyzes a Gly-|-Gly bond at its own C-terminus, commonly in thesequence -Tyr-Xaa-Val-Gly-|-Gly, in the processing of the potyviralpolyprotein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.46','L-peptidase','NULL','NULL','Best known from foot-and-mouth disease virus, but occurs in otheraphthoviruses and cardioviruses.Destruction of initiation factor eIF-4G has the effect of shuttingoff host-cell protein synthesis while allowing synthesis of viralproteins to continue.The tertiary structure reveals a distant relationship to papain and,consistent with this, compound E-64 is inhibitory.Belongs to peptidase family C28.','Autocatalytically cleaves itself from the polyprotein of the foot-and-mouth disease virus by hydrolysis of a Lys-|-Gly bond, but thencleaves host cell initiation factor eIF-4G at bonds -Gly-|-Arg- and-Lys-|-Arg-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.47','Gingipain K','Lys-gingipain','NULL','Activity is stimulated by glycine.Known from the bacterium Porphyromonas gingivalis and contributes tothe pathogenicity of the organism.Belongs to peptidase family C25.','Endopeptidase with strict specificity for lysyl bonds.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.48','Staphopain','Staphylopain','NULL','Known from species of Staphylococcus.Belongs to peptidase family C47.','Broad endopeptidase action on proteins including elastin, but ratherlimited hydrolysis of small-molecule substrates. Assays are convenientlymade with hemoglobin, casein or Z-Phe-Arg-NHMec as substrate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.49','Separase','Separin','NULL','In both Saccharomyces cerevisiae and human cells, cleavage of thecohesin subunit Scc1 by separase is required for sister chromatidseparation in mitosis.S.cerevisiae separase is also known to cleave the Rec8 subunit of ameiotic cohesin complex and the kinetochore protein Slk19.Belongs to peptidase family C50.','All bonds known to be hydrolyzed by this endopeptidase have arginine inP1 and an acidic residue in P4. P6 is often occupied by an acidic residueor by an hydroxy-amino-acid residue, the phosphorylation of whichenhances cleavage.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.50','V-cath endopeptidase','Baculovirus cathepsin','NULL','Contributes to the liquefaction of the tissues of the insect host inthe late stages of infection by the baculovirus.Belongs to peptidase family C1.','Endopeptidase of broad specificity, hydrolyzing substrates of bothcathepsin L and cathepsin B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.51','Cruzipain','Congopain','NULL','Is located in the digestive vacuoles of the parasitic trypanosome andcontributes to the nutrition of the organism by digestion of hostproteins.Belongs to peptidase family C1.','Broad endopeptidase specificity similar to that of cathepsin L.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.52','Calpain-1','Calcium-activated neutral protease I','Calcium.','Cytosolic in animal cells.The active enzyme molecule is a heterodimer in which the largesubunit contains the peptidase unit, and the small subunit is also acomponent of EC 3.4.22.53.Belongs to peptidase family C2.Formerly EC 3.4.22.17 and EC 3.4.24.5.','Broad endopeptidase specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.53','Calpain-2','Calcium-activated neutral protease II','Calcium.','Cytosolic in animal cells.The active enzyme molecule is a heterodimer in which the largesubunit contains the peptidase unit, and the small subunit is also acomponent of EC 3.4.22.52.Belongs to peptidase family C2.Formerly EC 3.4.22.17 and EC 3.4.24.5.','Broad endopeptidase specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.54','Calpain-3','Calcium-activated neutral proteinase 3','Calcium.','Activated by autoproteolytic cleavage of insertion sequence 1 (IS1),which allows substrates and inhibitors gain access to the activesite.Substrates include the protein itself and connectin/titin.Belongs to peptidase family C2.','Broad endopeptidase activity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.55','Caspase-2','CASP-2','NULL','Caspase-2 is an initiator caspase, as are caspases-8 (EC 3.4.22.61),caspase-9 (EC 3.4.22.62) and caspase-10 (EC 3.4.22.63).Contains a caspase-recruitment domain (CARD) in its N-terminalprodomain, which plays a role in procaspase activation.Two forms of caspase-2 exist that have antagonistic effects: caspase-2L induces programd cell death and caspase-2S suppresses cell death.Caspase-2 is activated by caspase-3 (EC 3.4.22.56), or by a caspase-3-like protease.Activation involves cleavage of the N-terminal prodomain, followed byself-proteolysis between the large and small subunits of pro-caspase-2 and further proteolysis into smaller fragments.Proteolysis occurs at Asp residues and the preferred substrate forthis enzyme is a pentapeptide rather than a tetrapeptide.Apart from itself, the enzyme can cleave golgin-16, which is presentin the Golgi complex and has a cleavage site that is unique forcaspase-2.alpha-II-spectrin, a component of the membrane cytoskeleton, is asubstrate of the large isoform of pro-caspase-2 (caspase-2L) but notof the short isoform (caspase-2S).Belongs to peptidase family C14.','Strict requirement for an Asp residue at P1, with 316-asp being essentialfor proteolytic activity and has a preferred cleavage sequence of Val-Asp-Val-Ala-Asp-|-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.56','Caspase-3','Apopain','NULL','Caspase-3 is an effector/executioner caspase, as are caspase-6(EC 3.4.22.59) and caspase-7 (EC 3.4.22.60).These caspases are responsible for the proteolysis of the majority ofcellular polypeptides, (e.g. poly(ADP-ribose) polymerase (PARP)),which leads to the apoptotic phenotype.Procaspase-3 can be activated by caspase-1 (EC 3.4.22.36), caspase-8(EC 3.4.22.61), caspase-9 (EC 3.4.22.62) and caspase-10(EC 3.4.22.63) as well as by the serine protease granzyme B.Caspase-3 can activate procaspase-2 (EC 3.4.22.55).Activation occurs by inter-domain cleavage followed by removal of theN-terminal prodomain.While Asp-Glu-(Val/Ile)-Asp is thought to be the preferred cleavagesequence, the enzyme can accommodate different residues at P2 and P3of the substrate.Like caspase-2, a hydrophobic residue at P5 of caspase-3 leads tomore efficient hydrolysis, e.g. (Val/Leu)-Asp-Val-Ala-Asp-|- is abetter substrate than Asp-Val-Ala-Asp-|-.This is not the case for caspase-7.Belongs to peptidase family C14.','Strict requirement for an Asp residue at positions P1 and P4. It has apreferred cleavage sequence of Asp-Xaa-Xaa-Asp-|- with a hydrophobicamino-acid residue at P2 and a hydrophilic amino-acid residue at P3,although Val or Ala are also accepted at this position.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.57','Caspase-4','CASP-4','NULL','Part of the family of inflammatory-caspases, which also includescaspase-1 (EC 3.4.22.36) and caspase-5 (EC 3.4.22.58) in humans andcaspase-11 (EC 3.4.22.64), caspase-12, caspase-13 and caspase-14 inmice.Contains a caspase-recruitment domain (CARD) in its N-terminalprodomain, which plays a role in procaspase activation.Able to cleave itself and the p30 caspase-1 precursor, but, unlikecaspase-1, it is very inefficient at generating mature interleukin-1-beta (IL-1-beta) from pro-IL-1-beta.Both this enzyme and caspase-5 can cleave pro-caspase-3 to releasethe small subunit (p12) but not the large subunit (p17).The caspase-1 inhibitor Ac-Tyr-Val-Ala-Asp-CHO can also inhibit thisenzyme, but more slowly.Belongs to peptidase family C14.','Strict requirement for Asp at the P1 position. It has a preferredcleavage sequence of Tyr-Val-Ala-Asp-|- but also cleaves at Asp-Glu-Val-Asp-|-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.58','Caspase-5','CASP-5','NULL','This enzyme is part of the family of inflammatory-caspases, whichalso includes caspase-1 (EC 3.4.22.36) and caspase-4 (EC 3.4.22.57)in humans and caspase-11 (EC 3.4.22.64), caspase-12, caspase-13 andcaspase-14 in mice.Contains a caspase-recruitment domain (CARD) in its N-terminalprodomain, which plays a role in procaspase activation.The enzyme is able to cleave itself and the p30 caspase-1 precursor,but is very inefficient at generating mature interleukin-1-beta(IL-1-beta) from pro-IL-1-beta.Both this enzyme and caspase-4 can cleave pro-caspase-3 to releasethe small subunit (p12) but not the large subunit (p17).Unlike caspase-4, this enzyme can be induced by lipopolysaccharide.Belongs to peptidase family C14.','Strict requirement for Asp at the P1 position. It has a preferredcleavage sequence of Tyr-Val-Ala-Asp-|- but also cleaves at Asp-Glu-Val-Asp-|-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.59','Caspase-6','Apoptotic protease Mch-2','NULL','Caspase-6 is an effector/executioner caspase, as are caspase-3(EC 3.4.22.56) and caspase-7 (EC 3.4.22.60).These caspases are responsible for the proteolysis of the majority ofcellular polypeptides, (e.g. poly(ADP-ribose) polymerase (PARP)),which lead to the apoptotic phenotype.Caspase-6 can cleave its prodomain to produce mature caspase-6, whichdirectly activates caspase-8 (EC 3.4.22.61) and leads to cytochrome crelease from the mitochondria; the release of cytochrome c, which isan essential component of the intrinsic apoptosis pathway.Can also cleave and inactivate lamins, the intermediate filamentscaffold proteins of the nuclear envelope, leading to nuclearfragementation in the final phases of apoptosis.Belongs to peptidase family C14.','Strict requirement for Asp at position P1 and has a preferred cleavagesequence of Val-Glu-His-Asp-|-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.60','Caspase-7','Apoptotic protease Mch-3','NULL','Caspase-7 is an effector/executioner caspase, as are caspase-3(EC 3.4.22.56) and caspase-6 (EC 3.4.22.59).These caspases are responsible for the proteolysis of the majority ofcellular polypeptides, (e.g. poly(ADP-ribose) polymerase (PARP)),which lead to the apoptotic phenotype.Although a hydrophobic residue at P5 of caspase-2 (EC 3.4.22.55) andcaspase-3 leads to more efficient hydrolysis, the amino-acid residueat this location in caspase-7 has no effect.Caspase-7 is activated by the initiator caspases (caspase-8(EC 3.4.22.61), caspase-9 (EC 3.4.22.62) and caspase-10(EC 3.4.22.63)).Removal of the N-terminal prodomain occurs before cleavage in thelinker region between the large and small subunits.Belongs to peptidase family C14.','Strict requirement for an Asp residue at position P1 and has a preferredcleavage sequence of Asp-Glu-Val-Asp-|-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.61','Caspase-8','Apoptotic cysteine protease','NULL','Caspase-8 is an initiator caspase, as are caspase-2 (EC 3.4.22.55),caspase-9 (EC 3.4.22.62) and caspase-10 (EC 3.4.22.63).Apical activator of the extrinsic (death receptor) apoptosis pathway,triggered by death receptor ligation.Contains two tandem death effector domains (DEDs) in its N-terminalprodomain, which play a role in procaspase activation.Linked to cell surface death receptors such as Fas.When Fas is aggregated by the Fas ligand, procaspase-8 is recruitedto the death receptor where it is activated.Has a preference for Glu at P3 and prefers small residues, such asGly, Ser and Ala at the P1 position.Has very broad P4 specificity, tolerating substrates with Asp, Val orLeu in this position.Endogenous substrates for caspase-8 include procaspase-3, the pro-apoptotic Bcl-2 family member Bid, RIP, PAK2 and the caspase-8activity modulator FLIP(L).Belongs to peptidase family C14.','Strict requirement for Asp at position P1 and has a preferred cleavagesequence of (Leu/Asp/Val)-Glu-Thr-Asp-|-(Gly/Ser/Ala).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.62','Caspase-9','APAF-3','NULL','Caspase-9 is an initiator caspase, as are caspase-2 (EC 3.4.22.55),caspase-8 (EC 3.4.22.61) and caspase-10 (EC 3.4.22.63).Contains a caspase-recruitment domain (CARD) in its N-terminalprodomain, which plays a role in procaspase activation.An alternatively spliced version of caspase-9 also exists, caspase-9S, that inhibits apoptosis, similar to the situation found withcaspase-2.Phosphorylation of caspase-9 from some species by Akt, a serine-threonine protein kinase, inhibits caspase activity in vitro andcaspase activation in vivo.The activity of caspase-9 is increased dramatically upon associationwith the apoptosome but the enzyme can be activated withoutproteolytic cleavage.Procaspase-3 is the enzymes physiological substrate.Belongs to peptidase family C14.','Strict requirement for an Asp residue at position P1 and with a markedpreference for His at position P2. It has a preferred cleavage sequenceof Leu-Gly-His-Asp-|-Xaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.63','Caspase-10','Apoptotic protease Mch-4','NULL','Caspase-10 is an initiator caspase, as are caspase-2 (EC 3.4.22.55),caspase-8 (EC 3.4.22.61) and caspase-9 (EC 3.4.22.62).Like caspase-8, it contains two tandem death effector domains (DEDs)in its N-terminal prodomain, which play a role in procaspaseactivation.Has many overlapping substrates in common with caspase-8, such as RIP(the cleavage of which impairs NF-kappa-B survival signaling andstarts the cell-death process) and PAK2 (associated with some of themorphological features of apoptosis, such as cell rounding andapoptotic body formation).Bid, a Bcl2 protein, can be cleaved by caspase-3 (EC 3.4.22.56),caspase-8 and caspase-10 at Lys-Gln-Thr-Asp-|- to yield the pro-apoptotic p15 fragment.The p15 fragment is N-myristoylated and enhances cytochrome c releasefrom mitochondria (which initiatiates the intrinsic apoptosispathway).Bid can be further cleaved by caspase-10 and granzyme B but not bycaspase-3 and caspase-8 at Ile-Glu-Thr-Asp-|- to yield a p13 fragmentthat is not N-myristoylated.Belongs to peptidase family C14.','Strict requirement for Asp at position P1 and has a preferred cleavagesequence of Leu-Gln-Thr-Asp-|-Gly.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.64','Caspase-11','CASP-11','NULL','Part of the family of inflammatory-caspases, which also includescaspase-1 (EC 3.4.22.36), caspase-4 (EC 3.4.22.57) and caspase-5(EC 3.4.22.58) in humans and caspase-12, caspase-13 and caspase-14 inmice.Contains a caspase-recruitment domain (CARD) in its N-terminalprodomain, which plays a role in procaspase activation.Like caspase-5, but unlike caspase-4, this enzyme can be induced bylipopolysaccharide.Not only activates caspase-1, which is required for the maturation ofproinflammatory cytokines such as interleukin-1-beta (IL-1-beta) andIL-18, but it also activates caspase-3 (EC 3.4.22.56), which leads tocellular apoptosis under pathological conditions.Belongs to peptidase family C14.','Strict requirement for Asp at the P1 position and has a preferredcleavage sequence of (Ile/Leu/Val/Phe)-Gly-His-Asp-|-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.65','Peptidase 1 (mite)','Allergen Der p 1','NULL','This enzyme, derived from the house dust mite, is a major componentof the allergic immune response.The substrate specificity of this enzyme is not altogether clear; itcleaves the low-affinity IgE receptor CD23 at 298-Glu-|-Ser-299 and155-Ser-|-Ser-156.It also cleaves the pulmonary structural proteins occludin andclaudin at Leu-|-Leu, Asp-|-Leu and at Gly-|-Thr bonds, and it canalso cleave the alpha subunit of the interleukin-2 (IL-2) receptor(CD25).Using a positional scanning combinatorial library, it was found thatthe major substrate-specificity determinant is for Ala in the P2position.The enzyme shows only a slight preference for basic amino acids inthe P1 and P3 positions and a preference for aliphatic amino acidssuch as Ile, Pro, Val, Leu and norleucine in the P4 position.Belongs to peptidase family C1.','Broad endopeptidase specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.66','Calicivirin','Calicivirus 3C-like protease','NULL','Viruses that are members of the Norovirus genus (Caliciviridaefamily) are a major cause of epidemic acute viral gastroenteritis.The nonstructural proteins of these viruses are produced byproteolytic cleavage of a large precursor polyprotein, performed by aprotease that is incorporated into the polyprotein.Cleavage sites are apparently defined by features based on bothsequence and structure since several sites in the polyproteinfulfilling the identified sequence requirements are not cleaved.The presence of acidic (Asp), basic (Arg), aromatic (Tyr) oraliphatic (Leu) amino acids at the P1 position results in only minordifferences in cleavage efficiency, suggesting that steric orconformational constraints may play a role in determiningspecificity.Changes to the amino acid at the P2 position do not alter cleavageefficiency.Belongs to peptidase family C37.','Endopeptidase with a preference for cleavage when the P1 position isoccupied by Glu-|-Xaa and the P1 position is occupied by Gly-|-Yaa.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.67','Zingipain','Ginger protease','NULL','Found in ginger (Zingiber officinale) rhizome and is a member of thepapain family.The enzyme is inhibited by some divalent metal ions, such as Hg(2+),Cu(2+), Cd(2+) and Zn(2+).Belongs to peptidase family C1.','Preferential cleavage of peptides with a proline residue at the P2position.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.68','Ulp1 peptidase','Smt3-protein conjugate proteinase','NULL','The enzyme from Saccharomyces cerevisiae can also recognize smallubiquitin-like modifier 1 (SUMO-1) from human as a substrate in bothSUMO-processing (alpha-linked peptide bonds) and SUMO-deconjugation(epsilon-linked peptide bonds) reactions.Ulp1 has several functions, including an essential role inchromosomal segregation and progression of the cell cycle through theG2/M phase of the cell cycle.Belongs to peptidase family C48.','Hydrolysis of the alpha-linked peptide bond in the sequence Gly-Gly-|-Ala-Thr-Tyr at the C-terminal end of the small ubiquitin-likemodifier (SUMO) propeptide, Smt3, leading to the mature form of theprotein. A second reaction involves the cleavage of an epsilon-linkedpeptide bond between the C-terminal glycine of the mature SUMO and thelysine epsilon-amino group of the target protein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.69','SARS coronavirus main proteinase','3C-like protease','NULL','SARS coronavirus main protease is the key enzyme in SARS coronavirusreplicase polyprotein processing.Belongs to peptidase family C30.','TSAVLQ-|-SGFRK-NH(2) and SGVTFQ-|-GKFKK the two peptides corresponding tothe two self-cleavage sites of the SARS 3C-like proteinase are the twomost reactive peptide substrates. The enzyme exhibits a strong preferencefor substrates containing Gln at P1 position and Leu at P2 position.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.70','Sortase A','NULL','NULL','Belongs to peptidase family C60.','The enzyme catalyzes a cell wall sorting reaction in which a surfaceprotein with a sorting signal containing a LPXTG motif is cleaved betweenthe Thr and Gly residue. The resulting threonine carboxyl end of theprotein is covalently attached to a pentaglycine cross-bridge ofpeptidoglycan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.22.71','Sortase B','SrtB','NULL','Belongs to peptidase family C60.','The enzyme catalyzes a cell wall sorting reaction in which a surfaceprotein with a sorting signal containing a NXTN motif is cleaved. Theresulting threonine carboxyl end of the protein is covalently attached toa pentaglycine cross-bridge of peptidoglycan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.1','Pepsin A','Pepsin','NULL','The predominant endopeptidase in the gastric juice of vertebrates,formed from pepsinogen A by limited proteolysis.Belongs to peptidase family A1.Formerly EC 3.4.4.1.','Preferential cleavage: hydrophobic, preferably aromatic, residues in P1and P1 positions. Cleaves 1-Phe-|-Val-2, 4-Gln-|-His-5, 13-Glu-|-Ala-14,14-Ala-|-Leu-15, 15-Leu-|-Tyr-16, 16-Tyr-|-Leu-17, 23-Gly-|-Phe-24,24-Phe-|-Phe-25 and 25-Phe-|-Tyr-26 bonds in the B chain of insulin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.2','Pepsin B','Parapepsin I','NULL','Belongs to peptidase family A1.Formerly EC 3.4.4.2.','Degradation of gelatin; little activity on hemoglobin. Specificity on Bchain of insulin more restricted than that of pepsin A; does not cleave1-Phe-|-Val-2, 4-Gln-|-His-5 or 23-Gly-|-Phe-24.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.3','Gastricsin','Pepsin C','NULL','Formed from progastricsin, apparently in the gastric juice of mostvertebrates.In addition to the fundus, progastricsin is also secreted in antrumand proximal duodenum.Seminal plasma contains a zymogen that is immunologically identicalwith progastricsin.Belongs to peptidase family A1.Formerly EC 3.4.4.22.','More restricted specificity than pepsin A, but shows preferentialcleavage at Tyr-|-Xaa bonds. High activity on hemoglobin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.4','Chymosin','Rennin','NULL','Neonatal gastric enzyme with high milk clotting and weak generalproteolytic activity, formed from prochymosin.Found among mammals with postnatal uptake of immunoglobins.Belongs to peptidase family A1.Formerly EC 3.4.4.3.','Broad specificity similar to that of pepsin A. Clots milk by cleavage ofa single 104-Ser-Phe-|-Met-Ala-107 bond in kappa-chain of casein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.5','Cathepsin D','NULL','NULL','Occurs intracellularly, in lysosomes.Belongs to peptidase family A1.Formerly EC 3.4.4.23.','Specificity similar to, but narrower than, that of pepsin A. Does notcleave the 4-Gln-|-His-5 bond in B chain of insulin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.6','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.7','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.9','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.10','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.11','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.12','Nepenthesin','Nepenthes aspartic proteinase','NULL','From the insectivorous plants Nepenthes sp. (secretions) and Droserapeltata (ground-up leaves).Aspartic endopeptidases are probably present in many other plants,including lotus and sorghum.Formerly EC 3.4.99.4.','Similar to pepsin, but also cleaves on either side of Asp and atLys-|-Arg.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.13','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.14','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.15','Renin','Angiotensin-forming enzyme','NULL','Belongs to peptidase family A1.Formerly EC 3.4.4.15 and EC 3.4.99.19.','Cleavage of Leu-|-Xaa bond in angiotensinogen to generate angiotensin I.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.16','HIV-1 retropepsin','Human immunodeficiency virus type 1 protease','NULL','Present in human immunodeficiency virus type 1.Contributes to the maturation of the viral particle, and is a targetof antiviral drugs.Active enzyme is a dimer of identical 11-kDa subunits.Similar enzymes occur in other retroviruses.Belongs to peptidase family A2.','Specific for a P1 residue that is hydrophobic, and P1 variable,but often Pro.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.17','Pro-opiomelanocortin converting enzyme','Prohormone converting enzyme','NULL','A 70 kDa membrane-bound enzyme isolated from cattle pituitarysecretory vesicle.Formerly EC 3.4.99.38.','Cleavage at paired basic residues in certain prohormones, either betweenthem, or on the carboxyl side.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.18','Aspergillopepsin I','Aspergillopepsin A','NULL','Found in a variety of Aspergillus species (imperfect fungi):A.awamori, A.foetidus, A.fumigatus, A.kawachii, A.niger, A.oryzae,A.saitoi, and A.sojae.Belongs to peptidase family A1.Formerly EC 3.4.23.6.','Hydrolysis of proteins with broad specificity. Generally favorshydrophobic residues in P1 and P1, but also accepts Lys in P1, whichleads to activation of trypsinogen. Does not clot milk.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.19','Aspergillopepsin II','Aspergillus niger var','NULL','Isolated from Aspergillus niger var. macrosporus, distinct fromaspergillopepsin I in specificity and insensitivity to pepstatin.Belongs to peptidase family A4.Formerly EC 3.4.23.6.','Preferential cleavage in B chain of insulin: 3-Asn-|-Gln-4, 13-Gly-|-Ala-14, and 26-Tyr-|-Thr-27.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.20','Penicillopepsin','Penicillium janthinellum aspartic protease','NULL','From the imperfect fungus Penicillium janthinellum.Closely related enzymes have been isolated from P.roqueforti andP.duponti.Belongs to peptidase family A1.Formerly EC 3.4.4.17, EC 3.4.23.6 and EC 3.4.23.7.','Hydrolysis of proteins with broad specificity similar to that of pepsinA, preferring hydrophobic residues at P1 and P1, but also cleaving20-Gly-|-Glu-21 in the B chain of insulin. Clots milk, and activatestrypsinogen.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.21','Rhizopuspepsin','Rhizopus aspartic proteinase','NULL','From the zygomycete fungi Rhizopus chinensis and R.niveus.Belongs to peptidase family A1.Formerly EC 3.4.4.17, EC 3.4.23.6, EC 3.4.23.9 and EC 3.4.99.25.','Hydrolysis of proteins with broad specificity similar to that of pepsinA, preferring hydrophobic residues at P1 and P1. Clots milk andactivates trypsinogen. Does not cleave 4-Gln-|-His-5, but does cleave10-His-|-Leu-11 and 12-Val-|-Glu-13 in B chain of insulin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.22','Endothiapepsin','Endothia aspartic proteinase','NULL','From the ascomycete Endothia parasitica.Belongs to peptidase family A1.Formerly EC 3.4.4.17, EC 3.4.23.6 and EC 3.4.23.10.','Hydrolysis of proteins with specificity similar to that of pepsin A,prefers hydrophobic residues at P1 and P1, but does not cleave14-Ala-|-Leu-15 in the B chain of insulin or Z-Glu-Tyr. Clots milk.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.23','Mucorpepsin','Mucor rennin','NULL','Isolated from the zygomycete fungi Mucor pusillus and M.miehei.Belongs to peptidase family A1.Formerly EC 3.4.4.17 and EC 3.4.23.6.','Hydrolysis of proteins, favoring hydrophobic residues at P1 and P1.Clots milk. Does not accept Lys at P1, and hence does not activatetrypsinogen.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.24','Candidapepsin','Candida albicans aspartic proteinase','NULL','This endopeptidase from the imperfect yeast Candida albicans isinhibited by pepstatin, but not by methyl 2-diazoacetamido-hexanoateor 1,2-epoxy-3-(p-nitrophenoxy)propane.Belongs to peptidase family A1.Formerly EC 3.4.4.17 and EC 3.4.23.6.','Preferential cleavage at the carboxyl of hydrophobic amino acids,but fails to cleave 15-Leu-|-Tyr-16, 16-Tyr-|-Leu-17 and 24-Phe-|-Phe-25of insulin B chain. Activates trypsinogen, and degrades keratin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.25','Saccharopepsin','Saccharomyces aspartic proteinase','NULL','Located in the vacuole.Belongs to peptidase family A1.Formerly EC 3.4.4.17, EC 3.4.23.6 and EC 3.4.23.8.','Hydrolysis of proteins with broad specificity for peptide bonds. Cleaves-Leu-Leu-|-Val-Tyr- bond in a synthetic substrate. Does not act on estersof Tyr or Arg.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.26','Rhodotorulapepsin','Rhodotorula aspartic proteinase','NULL','From the imperfect yeast Rhodotorula glutinis.Somewhat similar enzymes have been isolated from the imperfect yeast-like organism Cladosporium sp. and the imperfect fungus Paecilomycesvarioti.Formerly EC 3.4.4.17 and EC 3.4.23.6.','Specificity similar to that of pepsin A. Cleaves Z-Lys-|-Ala-Ala-Ala andactivates trypsinogen.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.27','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.28','Acrocylindropepsin','Acrocylindrium proteinase','NULL','From the imperfect fungus Acrocylindrium sp.Has a very low pH optimum on casein of 2.0.Formerly EC 3.4.4.17, EC 3.4.23.6 and EC 3.4.99.1.','Preference for hydrophobic residue at P1 and P1. Action on the B chainof insulin is generally similar to that of pepsin A, but it also cleaves6-Leu-|-Cys(SO(3)H)-7, 21-Glu-|-Arg-22 and 3-Asn-|-Gln-4, although not4-Gln-|-His-5.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.29','Polyporopepsin','Polyporus aspartic proteinase','NULL','From the basidiomycete Irpex lacteus (also known as Polyporustulipiferae).Belongs to peptidase family A1.Formerly EC 3.4.4.17.','Milk clotting activity, broad specificity, but fails to cleave15-Leu-|-Tyr-16 or 16-Tyr-|-Leu-17 of insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.30','Pycnoporopepsin','Proteinase Ia','NULL','From the basidiomycete Pycnoporus sanguineus, formerly known asP.coccineus and Trametes sanguinea.Formerly EC 3.4.4.17 and EC 3.4.23.6.','Specificity similar to pepsin A, but narrower, cleaving only three bondsin the B chain of insulin: 14-Ala-|-Leu-15, 16-Tyr-|-Leu-17,and 24-Phe-|-Phe-25.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.31','Scytalidopepsin A','Scytalidium aspartic proteinase A','NULL','Isolated from the imperfect fungus Scytalidium lignicolum.Not inhibited by pepstatin-Ac, methyl 2-diazoacetamidohexanoate or1,2-epoxy-3-(p-nitrophenyl)propane.A related enzyme from the same organism, proteinase C, is alsoinsensitive to these inhibitors and has molecular weight 406 kDa.','Hydrolysis of proteins with specificity similar to that of pepsin A,but also cleaves 7-Cys(SO(3)H)-|-Gly-8 and 17-Leu-|-Val-18 in the B chainof insulin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.32','Scytalidopepsin B','Scytalidium aspartic proteinase B','NULL','Isolated from the imperfect fungus Scytalidium lignicolum.A second endopeptidase from S.lignicolum (see scytalidopepsin A) thatis insensitive to pepstatin and methyl 2-diazoacetamidohexanoate.1,2-epoxy-3-(p-nitrophenoxy)propane reacts with Glu-53, whichreplaces one of the aspartic residues at the active center.One of the smallest aspartic endopeptidases active as the monomer,with molecular weight 22 kDa.Similarly inhibitor-resistant endopeptidases are found in thebasidiomycetes Lentinus edodes and Ganoderma lucidum, and inPolyporus tulipiferae (a second endopeptidase distinct frompolyporopepsin), but these are of typical aspartic endopeptidasesize.Belongs to peptidase family G1.','Hydrolysis of proteins with broad specificity, cleaving 24-Phe-|-Phe-25,but not 15-Leu-|-Tyr-16 and 25-Phe-|-Tyr-26 in the B chain of insulin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.33','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.34','Cathepsin E','Erythrocyte membrane aspartic proteinase','NULL','Found in stomach, spleen, erythrocyte membrane; not lysosomal.Pro-cathepsin E is an 86 kDa disulfide-linked dimer; activation orreduction produces monomers.Belongs to peptidase family A1.','Similar to cathepsin D, but slightly broader specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.35','Barrierpepsin','Bar proteinase','NULL','Belongs to peptidase family A1.','Selective cleavage of 6-Leu-|-Lys-7 bond in the pheromone alpha-matingfactor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.36','Signal peptidase II','Bacterial leader peptidase I','NULL','Inhibited by pepstatin and the antibiotic globomycin.Belongs to peptidase family A8.Formerly EC 3.4.99.35.','Release of signal peptides from bacterial membrane prolipoproteins.Hydrolyzes -Xaa-Yaa-Zaa-|-(S,diacylglyceryl)Cys-, in which Xaa ishydrophobic (preferably Leu), and Yaa (Ala or Ser) and Zaa (Gly or Ala)have small, neutral side chains.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.37','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.38','Plasmepsin I','Aspartic hemoglobinase I','NULL','Inhibited by pepstatin.Belongs to peptidase family A1.','Hydrolysis of the 33-Phe-|-Leu-34 bond in the alpha-chain of hemoglobin,leading to denaturation of molecule.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.39','Plasmepsin II','Aspartic hemoglobinase II','NULL','Inhibited by pepstatin.Belongs to peptidase family A1.','Hydrolysis of the bonds linking certain hydrophobic residues inhemoglobin or globin. Also cleaves small molecules substrates such asAla-Leu-Glu-Arg-Thr-Phe-|-Phe(NO(2))-Ser-Phe-Pro-Thr.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.40','Phytepsin','NULL','NULL','Known particularly fron barley grain, but present in other plantsalso.Belongs to peptidase family A1.','Prefers hydrophobic residues Phe, Val, Ile, Leu, and Ala at P1 and P1,but also cleaves -Phe-|-Asp- and -Asp-|-Asp- bonds in 2S albumin fromplant seeds.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.41','Yapsin 1','Aspartic proteinase 3','NULL','Weakly inhibited by pepstatin.Can partially substitute for kexin in a deficient strain ofSaccharomyces cerevisiae.The homologous product of the MKC7 gene (S.cerevisiae) has similarcatalytic activity and has been termed yapsin 2.Belongs to peptidase family A1.','Hydrolyzes various precursor proteins with Arg or Lys in P1, and commonlyArg or Lys also in P2. The P3 amino acid is usually non-polar,but otherwise additional basic amino acids are favorable in both non-prime and prime positions.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.42','Thermopsin','NULL','NULL','Isolated from cells and culture medium of the thermophilic archaeSulfolobus acidocaldarius, optimum digestion at pH 2 and 90 degreesCelsius, inhibited by pepstatin.Another thermostable bacterial endopeptidase has been isolated fromBacillus sp., but this is not inhibited by pepstatin.Belongs to peptidase family A5.Formerly EC 3.4.99.43.','Specificity similar to pepsin A, prefers bulky hydrophobic side-chains oneither side of the scissible bond.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.43','Prepilin peptidase','NULL','NULL','Many species of bacteria carry pili on their cell surfaces.These are virulence determinants in pathogenic strains, and areassembled biosynthetically from type IV prepilin subunits.Before assembly, the prepilin molecules require proteolyticprocessing, which is done by the prepilin peptidase.Prepilin peptidase and its homologs play a central role not only intype IV pilus biogenesis but also in transport of macromoleculesacross cell membranes.Although both peptide-bond hydrolysis and N-methylation are catalyzedby the same molecule, the methylation can be inhibited withoutaffecting peptidase activity, and it is believed that the enzyme hastwo separate catalytic sites.Belongs to peptidase family A24.','Typically cleaves a -Gly-|-Phe- bond to release an N-terminal, basicpeptide of 5-8 residues from type IV prepilin, and then N-methylates thenew N-terminal amino group, the methyl donor being S-adenosyl-L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.44','Nodavirus endopeptidase','Black Beetle virus endopeptidase','NULL','A single aspartic residue is critical for activity, and inhibition byEDTA indicates that a metal ion is also important.The enzyme is known from several nodaviruses that are pathogens ofinsects.Belongs to peptidase family A6.','Hydrolysis of an asparaginyl bond involved in the maturation of thestructural protein of the virus, typically -Asn-|-Ala- or -Asn-|-Phe-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.45','Memapsin 1','ASP1','NULL','Can cleave beta-amyloid precursor protein to form the amyloidogenicbeta-peptide that is implicated in the pathology of Alzheimersdisease, but is not significantly expressed in human brain.Belongs to peptidase family A1.','Broad endopeptidase specificity. Cleaves Glu-Val-Asn-Leu-|-Asp-Ala-Glu-Phe in the Swedish variant of Alzheimers amyloid precursor protein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.46','Memapsin 2','BACE1','NULL','Suggested to be the major beta-secretase responsible for thecleavage of the beta-amyloid precursor protein to form theamyloidogenic beta-peptide that is implicated in the pathology ofAlzheimers disease.Belongs to peptidase family A1.','Broad endopeptidase specificity. Cleaves Glu-Val-Asn-Leu-|-Asp-Ala-Glu-Phe in the Swedish variant of Alzheimers amyloid precursor protein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.47','HIV-2 retropepsin','NULL','NULL','Responsible for the post-translational processing of the humanimmunodeficiency virus polyprotein.Belongs to peptidase family A2.','Endopeptidase for which the P1 residue is preferably hydrophobic.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.48','Plasminogen activator Pla','Coagulase/fibrinolysin','NULL','From the bacterium Yersinia pestis that causes plague.Belongs to peptidase family A26.','Converts human Glu-plasminogen to plasmin by cleaving the 560-Arg-|-Val-561 peptide bond that is also hydrolyzed by the mammalian u-plasminogenactivator and t-plasminogen activator. Also cleaves arginyl bonds inother proteins.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.49','Omptin','Outer membrane protein 3B','NULL','Shows a preference for cleavage between consecutive basic aminoacids, but is capable of cleavage when P1 is a non-basic residue.Belongs to peptidase family A26.Formerly EC 3.4.21.87.','Has a virtual requirement for Arg in the P1 position and a slightly lessstringent preference for this residue in the P1 position, which can alsocontain Lys, Gly or Val.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.50','Human endogenous retrovirus K endopeptidase','HERV K10 endopeptidase','NULL','Belongs to peptidase family A2.','Processing at the authentic HIV-1 PR recognition site and release of themature p17 matrix and the p24 capsid protein, as a result of the cleavageof the -SQNY-|-PIVQ- cleavage site.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.51','HycI peptidase','NULL','NULL','The reaction requires nickel to be bound to the precursor of thelarge subunit of hydrogenase 3.The endopeptidase uses the metal in the large subunit of [NiFe]-hydrogenases as a recognition motif.Belongs to peptidase family A31.','This enzyme specifically removes a 32-amino acid peptide from theC-terminus of the precursor of the large subunit of E.coli hydrogenase 3by cleavage at the C-terminal side of Arg(537).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.23.52','Preflagellin peptidase','NULL','NULL','An aspartic peptidase from Archaea but not bacteria.In peptidase family A24 (type IV prepilin peptidase family).','Cleaves the signal peptide of 3 to 12 amino acids from the N-terminal ofpreflagellin, usually at Arg-Gly-|- or Lys-Gly-|-, to release flagellin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.1','Atrolysin A','Crotalus atrox alpha-proteinase','Zinc.','A hemorrhagic endopeptidase of 68 kDa, one of six hemorrhagic toxinsin the venom of the western diamondback rattlesnake.The 60 kDa hemorrhagic toxin 1 of Crotalus ruber ruber showsidentical specificity.Belongs to peptidase family M12B.','Cleavage of 3-Asn-|-Gln-4, 5-His-|-Leu-6, 10-His-|-Leu-11, 14-Ala-|-Leu-15 and 16-Tyr-|-Leu-17 in insulin B chain; removes C-terminal Leu fromsmall peptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.2','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.3','Microbial collagenase','Clostridiopeptidase A','Zinc.','Six species of metalloendopeptidase acting on native collagen can beisolated from the medium of Clostridium histolyticum.Class I has forms alpha (68 kDa), beta (115 kDa) and gamma (79 kDa);class II has delta (100 kDa), epsilon (110 kDa) and zeta (125 kDa).The two classes are immunologically crossreactive, but havesignificantly different sequences, and different specificities suchthat their actions on collagen are complementary.The enzymes also act as peptidyl-tripeptidases.Variants of the enzyme have been purified from Bacillus cereus,Empedobacter collagenolyticum, Pseudomonas marinoglutinosa,and species of Vibrio, Vibrio B-30 (ATCC 21250) and V.alginolyticus(previously Achromobacter iophagus).Also known from Streptomyces sp.Belongs to peptidase family M9.Formerly EC 3.4.4.19, EC 3.4.24.8 and EC 3.4.99.5.','Digestion of native collagen in the triple helical region at Xaa-|-Glybonds. With synthetic peptides, a preference is shown for Gly at P3 andP1; Pro and Ala at P2 and P2; and hydroxyproline, Ala or Arg at P3.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.6','Leucolysin','Leucostoma neutral proteinase','Calcium; Zinc.','From the venom of the western cottonmouth moccasin snake (Agkistrodonpiscivorus leucostoma).','Cleavage of 1-Phe-|-Val-2, 5-His-|-Leu-6, 14-Ala-|-Leu-15, 20-Gly-|-Glu-21, 23-Gly-|-Phe-24 and 24-Phe-|-Phe-25 bonds in insulin B chain as wellas N-blocked dipeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.7','Interstitial collagenase','Matrix metalloproteinase 1','Zinc.','The enzyme takes its name from substrates of the interstitialcollagen group - types I, II and III, all of which are cleaved in thehelical domain.Alpha-macroglobulins are cleaved much more rapidly.The enzyme is widely distributed in vertebrate animals.Belongs to peptidase family M10B.','Cleavage of the triple helix of collagen at about three-quarters of thelength of the molecule from the N-terminus, at 775-Gly-|-Ile-776 in thealpha-1(I) chain. Cleaves synthetic substrates and alpha-macroglobulinsat bonds where P1 is a hydrophobic residue.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.9','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.10','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.11','Neprilysin','Endopeptidase-2','Zinc.','A membrane-bound glycoprotein widely distributed in animal tissues.Inhibited by phosphoramidon and thiorphan.Belongs to peptidase family M13.','Preferential cleavage of polypeptides between hydrophobic residues,particularly with Phe or Tyr at P1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.12','Envelysin','Hatching enzyme','Calcium; Zinc.','Dissolves fertilization envelope in embryos of sea urchin.Belongs to peptidase family M10B.','Hydrolysis of proteins of the fertilization envelope and dimethylcasein.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.13','IgA-specific metalloendopeptidase','IgA protease','Zinc.','Found in several pathogenic species of Streptococcus such asS.sanguis and S.pneumoniae.There is also an IgA-specific prolyl endopeptidase of serine-type(see EC 3.4.21.72).Belongs to peptidase family M26.','Cleavage of Pro-|-Thr bond in the hinge region of the heavy chain ofhuman IgA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.14','Procollagen N-endopeptidase','Procollagen N-proteinase','Zinc.','Removes N-terminal propeptides of type I and II collagens prior tofibril assembly.Does not cleave type III procollagen.Belongs to peptidase family M12.','Cleaves the N-propeptide of collagen chain alpha-1(I) at Pro-|-Gln and ofalpha-1(II) and alpha-2(I) at Ala-|-Gln.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.15','Thimet oligopeptidase','Endo-oligopeptidase A','Zinc.','Thiol compounds activate at low concentrations.Belongs to peptidase family M3.Formerly EC 3.4.22.19 and EC 3.4.99.31.','Preferential cleavage of bonds with hydrophobic residues at P1, P2 andP3 and a small residue at P1 in substrates of 5 to 15 residues.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.16','Neurolysin','Neurotensin endopeptidase','Zinc.','No absolute requirement for a prolyl bond; the enzyme acts on somepeptides, such as dynorphin 1-8, that do not contain proline,and does not act on some others that do.Belongs to peptidase family M3.','Preferential cleavage in neurotensin: 10-Pro-|-Tyr-11.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.17','Stromelysin 1','Matrix metalloproteinase 3','Calcium; Zinc.','Extracellular endopeptidase of vertebrate tissues.Degradation of proteoglycans, fibronectin, collagen type III, IV,V and IX and activates procollagenase.Belongs to peptidase family M10B.','Preferential cleavage where P1, P2 and P3 are hydrophobic residues.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.18','Meprin A','Endopeptidase-2','Zinc.','A membrane-bound metalloendopeptidase of rat and mouse kidney andintestinal brush borders, and salivary ducts.Differences from EC 3.4.24.11 include insensitivity to phosphoramidonand thiorphan.PABA-peptide hydrolase is a very similar enzyme found in humanintestinal microvilli.Belongs to peptidase family M12A.','Hydrolysis of protein and peptide substrates preferentially on carboxylside of hydrophobic residues.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.19','Procollagen C-endopeptidase','NULL','Zinc.','The activity is increased by calcium and by an enhancer glycoprotein.','Cleavage of the C-terminal propeptide at Ala-|-Asp in type I and IIprocollagens and at Arg-|-Asp in type III.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.20','Peptidyl-Lys metalloendopeptidase','Armillaria mellea neutral proteinase','NULL','From the honey fungus Armillaria mellea.A similar specificity is shown by Myxobacter sp. AL-1 proteinase II(endoproteinase Lys-C).Formerly EC 3.4.99.30 and EC 3.4.99.32.','Preferential cleavage in proteins: -Xaa-|-Lys- (in which Xaa may be Pro).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.21','Astacin','Astacus proteinase','Zinc.','A digestive endopeptidase from the cardia of the crayfish Astacusfluviatilis.Belongs to peptidase family M12A.Formerly EC 3.4.99.6.','Hydrolysis of peptide bonds in substrates containing five or more aminoacids, preferentially with Ala in P1, and Pro in P2.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.22','Stromelysin 2','Matrix metalloproteinase 10','Calcium; Zinc.','Digests gelatin types I, III, IV, V, fibronectin and proteoglycan.Belongs to peptidase family M10B.','Similar to stromelysin 1, but action on collagen types III, IV and V isweak.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.23','Matrilysin','Matrin','Calcium; Zinc.','Similar in specificity to stromelysin, but more active on azocoll.Belongs to peptidase family M10B.','Cleavage of 14-Ala-|-Leu-15 and 16-Tyr-|-Leu-17 in B chain of insulin. Noaction on collagen types I, II, IV, V. Cleaves gelatin chain alpha-2(I) ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.24','Gelatinase A','72-kDa gelatinase','Calcium; Zinc.','Belongs to peptidase family M10B.','Cleavage of gelatin type I and collagen types IV, V, VII, X. Cleaves thecollagen-like sequence Pro-Gln-Gly-|-Ile-Ala-Gly-Gln.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.25','Vibriolysin','Aeromonas proteolytica neutral proteinase','Zinc.','Thermostable enzyme from Vibrio proteolyticus (formerly Aeromonasproteolytica).Specificity related to, but distinct from, those of the thermolysinand Bacillus subtilis endopeptidase.Belongs to peptidase family M4.Formerly EC 3.4.24.4.','Preferential cleavage of bonds with bulky hydrophobic groups in P2 andP1. Phe at P1 is the most favored residue, which distinguished thisenzyme from thermolysin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.26','Pseudolysin','Pseudomonas aeruginosa neutral metalloproteinase','Calcium; Zinc.','Causes tissue damage.Belongs to peptidase family M4.Formerly EC 3.4.24.4.','Hydrolysis of proteins including elastin, collagen types III and IV,fibronectin and immunoglobulin A, generally with bulky hydrophobic groupat P1. Insulin B chain cleavage pattern identical to that ofthermolysin, but specificity differs in other respects.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.27','Thermolysin','Bacillus thermoproteolyticus neutral proteinase','Calcium; Zinc.','A thermostable extracellular metalloendopeptidase containing fourcalcium ions.Enzymes that may be species variants of thermolysin are reported fromMicrococcus caseolyticus and Aspergillus oryzae.Closely related but distinct enzymes are aeromonolysin, pseudolysin,bacillolysin, aureolysin and mycolysin.Belongs to peptidase family M4.Formerly EC 3.4.24.4.','Preferential cleavage: Xaa-|-Leu ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.28','Bacillolysin','Bacillus metalloendopeptidase','Calcium; Zinc.','Variants of this enzyme have been found in species of Bacillusincluding B.subtilis, B.amyloliquefaciens, B.megaterium,B.mesentericus, B.cereus and B.stearothermophilus.Belongs to peptidase family M4.Formerly EC 3.4.24.4.','Similar, but not identical, to that of thermolysin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.29','Aureolysin','Staphylococcus aureus neutral proteinase','Calcium; Zinc.','Earlier confused with staphylokinase.Belongs to peptidase family M4.Formerly EC 3.4.24.4 and EC 3.4.99.22.','Cleavage of insulin B chain with specificity similar to that ofthermolysin, preferring hydrophobic P1 residues. Activates the glutamylendopeptidase (EC 3.4.21.19) of Staphylococcus aureus.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.30','Coccolysin','Streptococcus thermophilus intracellular proteinase','Zinc.','A 30 kDa endopeptidase found intracellularly in Streptococcusthermophilus and S.diacetilactis and in the medium of Enterococcusfaecalis.Belongs to peptidase family M4.Formerly included in EC 3.4.24.4.','Preferential cleavage: Xaa-|-Leu, Xaa-|-Phe, Xaa-|-Tyr, Xaa-|-Ala.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.31','Mycolysin','Pronase component','Zinc.','From Streptomyces griseus, S.naraensis, and S.cacaoi.Specificity similar to that of thermolysin, but much more sensitiveto inhibition by mercaptoacetyl-Phe-Leu.Belongs to peptidase family M5.Formerly EC 3.4.24.4.','Preferential cleavage of bonds with hydrophobic residues in P1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.32','Beta-lytic metalloendopeptidase','Achromopeptidase component','Zinc.','From Achromobacter lyticus and Lysobacter enzymogenes.Digests bacterial cell walls.Myxobacter sp. AL-1 proteinase I also has characteristics of thisenzyme.Belongs to peptidase family M23.Formerly EC 3.4.24.4 and EC 3.4.99.13.','Cleavage of N-acetylmuramoyl-|-Ala, and of the insulin B chain at23-Gly-|-Phe-24 ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.33','Peptidyl-Asp metalloendopeptidase','Endoproteinase Asp-N','NULL','A metalloenzyme isolated from Pseudomonas fragi.Useful in protein sequencing applications because of its limitedspecificity.Belongs to peptidase family M72.','Cleavage of Xaa-|-Asp, Xaa-|-Glu and Xaa-|-cysteic acid bonds.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.34','Neutrophil collagenase','Matrix metalloproteinase 8','Calcium; Zinc.','Similar to interstitial collagenase in specificity.Stored in the specific granules of neutrophil leukocytes.Belongs to peptidase family M10B.','Cleavage of interstitial collagens in the triple helical domain. UnlikeEC 3.4.24.7, this enzyme cleaves type III collagen more slowly than typeI.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.35','Gelatinase B','92-kDa gelatinase','Calcium; Zinc.','Belongs to peptidase family M10B.','Cleavage of gelatin types I and V and collagen types IV and V.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.36','Leishmanolysin','Glycoprotein gp63','Calcium; Zinc.','A membrane-bound glycoprotein found on the promastigote of variousspecies of Leishmania protozoans.Contains consensus sequence for a zinc binding site; Z-Tyr-Leu-NHOHis a strong inhibitor.The enzyme can activate its proenzyme by cleavage of the 100-Val-|-Val-101 bond.An acidic pH optimum is found with certain protein substrates.Belongs to peptidase family M8.','Preference for hydrophobic residues at P1 and P1 and basic residues atP2 and P3. A model nonapeptide is cleaved at -Ala-Tyr-|-Leu-Lys-Lys-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.37','Saccharolysin','Oligopeptidase yscD','Zinc.','Belongs to peptidase family M3.Formerly EC 3.4.22.22.','Cleavage of Pro-|-Phe and Ala-|-Ala bonds.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.38','Gametolysin','Autolysin','Zinc.','Inhibited by phosphoramidon, but also thiol-dependent.Belongs to peptidase family M11.','Cleavage of the proline- and hydroxyproline-rich proteins of theChlamydomonas cell wall; also cleaves azocasein, gelatin and Leu-Trp-Met-|-Arg-Phe-Ala.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.39','Deuterolysin','Acid metalloproteinase','Zinc.','Proteolytic activity found in Penicillium roqueforti, P.caseicolum,Aspergillus sojae and A.oryzae.Optimum pH of 5 for digesting various proteins.Strong action on protamine and histones.Insensitive to phosphoramidon.A distinct A.sojae endopeptidase is larger and has neutral pHoptimum.Belongs to peptidase family M35.Formerly EC 3.4.24.4.','Preferential cleavage of bonds with hydrophobic residues in P1; also3-Asn-|-Gln-4 and 8-Gly-|-Ser-9 bonds in insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.40','Serralysin','Escherichia freundii proteinase','Zinc.','An extracellular endopeptidase from Pseudomonas aeruginosa,Escherichia freundii, Serratia marcescens and Erwinia chrysanthemi.There is broad specificity in cleavage of the insulin B chain, withsome species variations.The pH optimum for digesting various proteins is about 9-10.Belongs to peptidase family M10A.Formerly EC 3.4.24.4.','Preferential cleavage of bonds with hydrophobic residues in P1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.41','Atrolysin B','Crotalus atrox metalloendopeptidase b','Zinc.','From the venom of the western diamondback rattlesnake (Crotalusatrox).Belongs to peptidase family M12B.','Cleavage of 5-His-|-Leu-6, 10-His-|-Leu-11, 14-Ala-|-Leu-15,16-Tyr-|-Leu-17 and 23-Gly-|-Phe-24 of insulin B chain; identical to thecleavage of insulin B chain by atrolysin C. Also cleaves Xaa-|-Ser bondsin glucagon.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.42','Atrolysin C','Crotalus atrox metalloendopeptidase c','Zinc.','Hemorrhagic endopeptidase from the venom of the western diamondbackrattlesnake (Crotalus atrox).Exist as two forms, c and d.Digests type IV collagen.Phosphoramidon inhibits in the 0.1 mM range.Hemorrhagic toxin-2 of C.ruber ruber has the same molecular weightand specificity.Belongs to peptidase family M12B.','Cleavage of 5-His-|-Leu-6, 10-His-|-Leu-11, 14-Ala-|-Leu-15,16-Tyr-|-Leu-17 and 23-Gly-|-Phe-24 of insulin B chain. With smallmolecule substrates prefers hydrophobic residue at P2 and small residuesuch as Ala, Gly at P1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.43','Atroxase','NULL','Zinc.','A nonhemorrhagic endopeptidase from the venom of the westerndiamondback rattlesnake (Crotalus atrox).Cleaves fibrinogen.Belongs to peptidase family M12B.','Cleavage of 5-His-|-Leu-6, 9-Ser-|-His-10, 10-His-|-Leu-11, 14-Ala-|-Leu-15 and 16-Tyr-|-Leu-17 in insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.44','Atrolysin E','Crotalus atrox metalloendopeptidase e','Zinc.','Hemorrhagic endopeptidase from the venom of the western diamondbackrattlesnake (Crotalus atrox).Digests basement membrane components, including the triple helix oftype IV collagen.Such action is believed to contribute to the hemorrhagic property byweakening capillary walls.Belongs to peptidase family M12B.','Cleavage of 3-Asn-|-Gln-4, 9-Ser-|-His-10 and 14-Ala-|-Leu-15 bonds ininsulin B chain and 14-Tyr-|-Gln-15 and 8-Thr-|-Ser-9 in A chain. Cleavestype IV collagen at 73-Ala-|-Gln-74 in alpha-1-(IV) and at 7-Gly-|-Leu-8in alpha-2-(IV).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.45','Atrolysin F','Crotalus atrox metalloendopeptidase f','Zinc.','Hemorrhagic endopeptidase from the venom of the western diamondbackrattlesnake (Crotalus atrox).Digests the gamma chain of fibrinogen.Belongs to peptidase family M12B.','Cleavage of 2-Val-|-Asn-3, 4-Gln-|-His-5, 6-Leu-|-Cys-7, 10-His-|-Leu-11,14-Ala-|-Leu-15 and 16-Tyr-|-Leu-17 bonds in insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.46','Adamalysin','Crotalus adamanteus metalloendopeptidase','Zinc.','From the venom of the eastern diamondback rattlesnake (Crotalusadamanteus).Two isoenzymes of approximately 24 kDa that inactivate alpha-1-proteinase inhibitor by a single cleavage.Belongs to peptidase family M12B.','Cleavage of 1-Phe-|-Val-2, 5-His-|-Leu-6, 14-Ala-|-Leu-15, 15-Leu-|-Tyr-16, and 16-Tyr-|-Leu-17 of insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.47','Horrilysin','Crotalus horridus metalloendopeptidase','Zinc.','Cleaves basement membrane, hide powder and fibrinogen.Hemorrhagic endopeptidase from the venom of the timber rattlesnake(Crotalus horridus horridus).Belongs to peptidase family M12B.','Cleavage of only the single bond 14-Ala-|-Leu-15 in the insulin B chain,12-Ser-|-Leu-13 in the A chain, and Ile-|-Gly, Pro-|-Ala, and Ser-|-Trpin mellitin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.48','Ruberlysin','Crotalus ruber metalloendopeptidase II','Zinc.','Hemorrhagic endopeptidase from the venom of the red rattlesnake(Crotalus ruber ruber).Cleaves fibrinogen.Belongs to peptidase family M12B.','Cleavage of 10-His-|-Leu-11, 14-Ala-|-Leu-15, 16-Tyr-|-Leu-17 and23-Gly-|-Phe-24 bonds of the B chain of insulin; His-|-Pro, Pro-|-Phe,and Trp-|-Ser of angiotensin I; and Gly-|-Phe of Met enkephalin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.49','Bothropasin','NULL','Zinc.','Caseinolytic endopeptidase of jararaca snake (Bothrops jararaca)venom.Belongs to peptidase family M12B.','Cleavage of 5-His-|-Leu-6, 10-His-|-Leu-11, 14-Ala-|-Leu-15,16-Tyr-|-Leu-17 and 24-Phe-|-Phe-25 in insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.50','Bothrolysin','Bothrops metalloendopeptidase J','Zinc.','Endopeptidase from the venom of the jararaca snake (Bothropsjararaca).Insensitive to phosphoramidon at 0.5 mM.Belongs to peptidase family M12B.','Cleavage of 4-Gln-|-His-5, 9-Ser-|-His-10 and 14-Ala-|-Leu-15 of insulinB chain and Pro-|-Phe of angiotensin I.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.51','Ophiolysin','Ophiophagus metalloendopeptidase','Zinc.','Endopeptidase from the venom of the king cobra (Ophiophagus hannah).Belongs to peptidase family M12B.','Cleavage of 3-Asn-|-Gln-4, 4-Gln-|-His-5, 10-His-|-Leu-11, 14-Ala-|-Leu-15, and 16-Tyr-|-Leu-17 in insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.52','Trimerelysin I','Hemorrhagic proteinase HR1a','Zinc.','Hemorrhagic endopeptidase of pI 4.4 from the venom of the habu snake(Trimeresurus flavoviridis).Belongs to peptidase family M12B.','Cleavage of only two bonds: 10-His-|-Leu-11 and 14-Ala-|-Leu-15, in theinsulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.53','Trimerelysin II','Proteinase H(2)','Zinc.','Nonhemorrhagic endopeptidase from the venom of the habu snake(Trimeresurus flavoviridis).Belongs to peptidase family M12B.','Cleavage of 3-Asn-|-Gln-4, 10-His-|-Leu-11 and 14-Ala-|-Leu-15 in theinsulin B chain, and the bond Z-Gly-Pro-|-Leu-Gly-Pro in a small moleculesubstrate of microbial collagenase.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.54','Mucrolysin','Mucrotoxin A','Zinc.','Hemorrhagic and fibrinogenolytic endopeptidase from the Chinese habusnake (Trimeresurus mucrosquamatus) venom.Belongs to peptidase family M12B.','Cleavage of 9-Ser-|-His-10, 10-His-|-Leu-11, 14-Ala-|-Leu-15,15-Leu-|-Tyr-16 and 16-Tyr-|-Leu-17 bonds in the insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.55','Pitrilysin','Protease III','Zinc.','From the periplasmic space of Escherichia coli.Inhibited by EDTA and 1,10-phenanthroline; not thiol dependent.Belongs to peptidase family M16.Formerly EC 3.4.99.44.','Preferential cleavage of 16-Tyr-|-Leu-17 and 25-Phe-|-Tyr-26 bonds ofoxidized insulin B chain. Also acts on other substrates of Mw less than 7kDa such as insulin and glucagon.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.56','Insulysin','Insulin protease','Zinc.','Cytosolic enzyme, known from mammals and Drosophila melanogaster.Inhibited by bacitracin, chelating agents EDTA and 1,10-phenanthroline, and by thiol-blocking reagents such asN-ethylmaleimide, but not by phosphoramidon.Belongs to peptidase family M16.Formerly EC 3.4.22.11, EC 3.4.99.10 and EC 3.4.99.45.','Degradation of insulin, glucagon and other polypeptides. No action onproteins.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.57','O-sialoglycoprotein endopeptidase','Glycoprotease','NULL','Inhibited by EDTA (100 mM) and 1,10-phenanthroline.Belongs to peptidase family M22.','Hydrolysis of O-sialoglycoproteins; cleaves 31-Arg-|-Asp-32 bond inglycophorin A. Does not cleave unglycosylated proteins, desialylatedglycoproteins or glycoproteins that are only N-glycosylated.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.58','Russellysin','Blood-coagulation factor X-activating enzyme','Calcium; Zinc.','Belongs to peptidase family M12B.','Specifically activates several components of the blood clotting system,including coagulation factor X, coagulation factor IX and protein C bycleavage of Arg-|-Xaa bonds. Has no action on insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.59','Mitochondrial intermediate peptidase','NULL','Zinc.','Natural substrates are precursor proteins that have already beenprocessed by mitochondrial processing peptidase.Belongs to peptidase family M3.','Release of an N-terminal octapeptide as second stage of processing ofsome proteins imported into the mitochondrion.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.60','Dactylysin','Peptide hormone inactivating endopeptidase','NULL','An endopeptidase of 100 kDa secreted from the skin of Xenopus laevis.Resembles neprilysin in insensitivity to 1 uM captopril, but differsin being insensitive to thiorphan and unable to digest (5-Met)-enkephalin, (5-Leu)-enkephalin, oxytocin and substance P-(7-11)-peptide.','Hydrolysis of peptides of at least six residues, with bulky hydrophobicresidues in the P1 position. Shows a preference for hydrophobic doubletssuch as -Phe-|-Phe- and -Phe-|-Leu- in somatostatin-(1-14)-peptide anddynorphin A-(1-6)-peptide, respectively.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.61','Nardilysin','N-arginine dibasic convertase','Zinc.','Unusually for a metallopeptidase, inhibited by bestatin, amastin andethylmaleimide.Belongs to peptidase family M16.','Hydrolysis of polypeptides, preferably at -Xaa-|-Arg-Lys-, and lesscommonly at -Arg-|-Arg-Xaa-, in which Xaa is not Arg or Lys.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.62','Magnolysin','Neurosecretory granule protease cleaving pro-ocytocin/neurophysin','NULL','Inhibited by EDTA.','Hydrolysis of polypeptides with Arg or Lys in P1 and P2, e.g. tohydrolyze pro-oxytocin at -Lys-Arg-|-Ala-Val-. The specificity furtherdepends on the organization of a beta-turn-alpha-helix of nine or moreresidues containing the paired basic amino acids near the center.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.63','Meprin B','NULL','Zinc.','Inhibited by EDTA and 1,10-phenanthroline, but not by phosphoramidon,captopril or thiorphan.Belongs to peptidase family M12A.','Hydrolysis of proteins, including azocasein, and peptides. Hydrolysis of5-His-|-Leu-6, 6-Leu-|-Cys-7, 14-Ala-|-Leu-15 and 19-Cys-|-Gly-20 bondsin insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.64','Mitochondrial processing peptidase','Processing enhancing peptidase','Zinc.','Known from the mitochondrial matrix of fungi and mammals.Belongs to peptidase family M16.Formerly EC 3.4.99.41.','Release of N-terminal transit peptides from precursor proteins importedinto the mitochondrion, typically with Arg in position P2.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.65','Macrophage elastase','Matrix metalloproteinase 12','Calcium; Zinc.','Belongs to peptidase family M10B.','Hydrolysis of soluble and insoluble elastin. Specific cleavages are alsoproduced at 14-Ala-|-Leu-15 and 16-Tyr-|-Leu-17 in the B chain ofinsulin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.66','Choriolysin L','Low choriolytic enzyme','Zinc.','Belongs to peptidase family M12A.','Hydrolysis of the inner layer of fish egg envelope. Also hydrolysis ofcasein and small molecule substrates such as succinyl-Leu-Leu-Val-Tyr-|-7-(4-methyl)coumarylamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.67','Choriolysin H','High choriolytic enzyme','Zinc.','Belongs to peptidase family M12A.','Hydrolysis of the inner layer of fish egg envelope. Also hydrolysis ofcasein and small molecule substrates such as succinyl-Leu-Leu-Val-Tyr-|-7-(4-methyl)coumarylamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.68','Tentoxilysin','Tetanus neurotoxin','Zinc.','Belongs to peptidase family M27.','Hydrolysis of 76-Gln-|-Phe-77 bond in synaptobrevin 2.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.69','Bontoxilysin','Botulinum neurotoxin','Zinc.','Belongs to peptidase family M27.','Limited hydrolysis of proteins of the neuroexocytosis apparatus,synaptobrevins, SNAP25 or syntaxin. No detected action on small moleculesubstrates.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.70','Oligopeptidase A','NULL','Zinc.','Differs from thimet oligopeptidase in lack of thiol activation.Belongs to peptidase family M3.','Hydrolysis of oligopeptides, with broad specificity. Gly or Ala commonlyoccur as P1 or P1 residues, but more distant residues are alsoimportant, as is shown by the fact that Z-Gly-Pro-Gly-|-Gly-Pro-Ala iscleaved, but not Z-(Gly)(5).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.71','Endothelin-converting enzyme 1','ECE-1','Zinc.','Phosphoramidon-sensitive.Belongs to peptidase family M13.','Hydrolysis of the 21-Trp-|-Val-22 bond in big endothelin to formendothelin 1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.72','Fibrolase','Fibrinolytic proteinase','Zinc.','Belongs to peptidase family M12.','Hydrolysis of 14-Ala-|-Leu-15 in insulin B chain and 413-Lys-|-Leu-414 inalpha-chain of fibrinogen.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.73','Jararhagin','HF2-proteinase','Zinc.','Hemorrhagic endopeptidase from the venom of the jararaca snake.Belongs to peptidase family M12.','Cleavage of 10-His-|-Leu-11, 14-Ala-|-Leu-15, 16-Tyr-|-Leu-17 and24-Phe-|-Phe-25 bonds in insulin B chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.74','Fragilysin','Bacteroides fragilis enterotoxin','Zinc.','Thought to be a cause of diarrhea.Hydrolyzes extracellular matrix proteins, and disrupts tightjunctions of intstinal cells.Also degrades intracellular, cytoskeletal proteins actin, myosin andothers.Belongs to peptidase family M10.','Broad proteolytic specificity, bonds hydrolyzed includes -Gly-|-Leu-,-Met-|-Leu-, -Phe-|-Leu-, -Cys-|-Leu-, -Leu-|-Gly-.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.75','Lysostaphin','Glycyl-glycine endopeptidase','Zinc.','Lyses cells of Staphylococcus aureus in particular, by its action onthe cross-bridge of the cell wall.Belongs to peptidase family M23.','Hydrolysis of the -Gly-|-Gly- bond in the pentaglycine inter-peptide linkjoining staphylococcal cell wall peptidoglycans.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.76','Flavastacin','NULL','Zinc.','The specificity is similar to that of EC 3.4.24.33, but the two arereported to be structurally dissimilar.Belongs to peptidase family M12.','Hydrolyzes polypeptides on the amino-side of Asp in -Xaa-|-Asp-. Actsvery slowly on -Xaa-|-Glu.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.77','Snapalysin','Small neutral protease','Zinc.','Belongs to peptidase family M7.','Hydrolyzes proteins with a preference for Tyr or Phe in the P1 position.Has no action on amino-acid p-nitroanilides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.78','GPR endopeptidase','Germination protease','NULL','Initiates the degradation of small, acid-soluble proteins duringspore germination in Bacillus megaterium.Belongs to peptidase family M63.','Endopeptidase action with P4 Glu or Asp, P1 preferably Glu ', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.79','Pappalysin-1','Insulin-like growth factor binding protein-4 protease','Zinc.','Circulates in human pregnancy mainly as a complex with the proform ofeosinophil major basic protein, which acts as an inhibitor of thepeptidase.The rate of hydrolysis of IGFBP-4 is increased about 20-fold by thepresence of insulin-like growth factor (IGF), whereas that of IGFBP-5is decreased about two-fold.Belongs to peptidase family M43.','Cleavage of the 135-Met-|-Lys-136 bond in insulin-like growth factorbinding protein (IGFBP)-4, and the 143-Ser-|-Lys-144 bond in IGFBP-5.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.80','Membrane-type matrix metalloproteinase-1','Matrix metalloproteinase 14','Zinc.','Believed to play an important role in the activation of progelatinaseA at cell surfaces.Belongs to peptidase family M10.','Endopeptidase activity. Activates progelatinase A by cleavage of thepropeptide at 37-Asn-|-Leu-38. Other bonds hydrolyzed include35-Gly-|-Ile-36 in the propeptide of collagenase 3, and 341-Asn-|-Phe-342, 441-Asp-|-Leu-442 and 354-Gln-|-Thr-355 in the aggrecaninterglobular domain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.81','ADAM10 endopeptidase','Kuzbanian protein','Zinc.','Partially responsible for the alpha-secretase activity in brainthat degrades the potentially harmful beta-amyloid peptide.Work with ADAM10-deficient mice supports a role in Notch signaling.Belongs to peptidase family M12.','Endopeptidase of broad specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.82','ADAMTS-4 endopeptidase','Aggrecanase-1','Zinc.','Thought to be biologically significant for the degradation of theaggrecan component of cartilage matrix.Belongs to peptidase family M12.','Glutamyl endopeptidase; bonds cleaved include 370-Thr-Glu-Gly-Glu-|-Ala-Arg-Gly-Ser-377 in the interglobular domain of mammalian aggrecan.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.83','Anthrax lethal factor endopeptidase','Lethal toxin','Zinc.','From the bacterium Bacilus anthracis that causes anthrax.One of three proteins that are collectively termed anthrax toxin.Cleaves several MAP kinase kinases near their N-termini, preventingthem from phosphorylating the downstream mitogen-activated proteinkinases.Belongs to peptidase family M34.','Preferred amino acids around the cleavage site can be denotedBBBBxHx-|-H, in which B denotes Arg or Lys, H denotes a hydrophobic aminoacid, and x is any amino acid. The only known protein substrates aremitogen-activated protein (MAP) kinase kinases.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.84','Ste24 endopeptidase','CAAX prenyl protease 1','Zinc.','One of two enzymes that can catalyze this processing step for matinga-factor in Saccharomyces cerevisiae.Subsequently, the S-isoprenylated cysteine residue that forms the newC-terminus is methyl-esterified and forms a hydrophobic membrane-anchor.Belongs to peptidase family M48.','The peptide bond hydrolyzed can be designated -C-|-A-A-X in which C is anS-isoprenylated cysteine residue, A is usually aliphatic and X is theC-terminal residue of the substrate protein, and may be any of severalamino acids.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.85','S2P endopeptidase','Membrane-bound transcription factor site 2 protease','Zinc.','The transcription factors SREBP-1 and -2 are synthesized as precursorproteins that are attached to the membranes of the endoplasmicreticulum and two cleavages are needed to release the active factorso that it can move to the nucleus.This enzyme cleaves the second of these, and is thus the site 2protease, S2P.Belongs to peptidase family M50.','Cleaves several transcription factors that are type-2 transmembraneproteins within membrane-spanning domains. Known substrates includesterol regulatory element-binding protein (SREBP) -1, SREBP-2 and formsof the transcriptional activator ATF6. SREBP-2 is cleaved at the site477-DRSRILL-|-CVLTFLCLSFNPLTSLLQWGGA-505. The residues Asn-Pro,11 residues distal to the site of cleavage in the membrane-spanningdomain, are important for cleavage by S2P endopeptidase. Replacement ofeither of these residues does not prevent cleavage, but there is nocleavage if both of these residues are replaced.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.86','ADAM 17 endopeptidase','TACE','Zinc.','In vivo, the cleavage of tumor necrosis factor alpha precursorreleases the soluble, 17-kDa TNF-alpha, which induces inflammation.Belongs to peptidase family M12.','Narrow endopeptidase specificity. Cleaves Pro-Leu-Ala-Gln-Ala-|-Val-Arg-Ser-Ser-Ser in the membrane-bound, 26-kDa form of tumor necrosisfactor alpha (TNF-alpha). Similarly cleaves other membrane-anchored,cell-surface proteins to shed the extracellular domains.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.24.87','ADAMTS13 endopeptidase','ADAMTS VWF cleaving metalloprotease','NULL','Belongs to peptidase family M12.','The enzyme cleaves the von Willebrand factor at bond Tyr(842)-|-Met(843)within the A2 domain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.25.1','Proteasome endopeptidase complex','Ingensin','NULL','A 20-S protein composed of 28 subunits arranged in four rings ofseven.The outer rings are composed of alpha subunits, but the beta subunitsforming the inner rings are responsible for peptidase activity.In eukaryotic organisms there are up to seven different types of betasubunits, three of which may carry the N-terminal threonine residuesthat are the nucleophiles in catalysis, and show differentspecificities.The molecule is barrel-shaped, and the active sites are on the innersurfaces.Terminal apertures restrict access of substrates to the active sites.Inhibited by mercurial reagents and some inhibitors of serineendopeptidases.Belongs to peptidase family T1.Formerly EC 3.4.22.21, EC 3.4.24.5 and EC 3.4.99.46.','Cleavage of peptide bonds with very broad specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.25.2','HslU--HslV peptidase','NULL','NULL','The HslU subunit of the HslU--HslV complex functions as an ATPdependent unfoldase.The binding of ATP and its subsequent hydrolysis by HslU areessential for unfolding of protein substrates subsequently hydrolyzedby HslV.HslU recognizes the N-terminal part of its protein substrates andunfolds these before they are guided to HslV for hydrolysis.Belongs to peptidase family T1.','ATP-dependent cleavage of peptide bonds with broad specificity.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.2','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.3','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.6','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.7','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.8','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.9','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.10','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.11','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.12','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.13','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.14','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.15','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.16','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.17','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.18','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.19','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.20','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.21','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.22','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.23','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.24','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.25','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.26','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.27','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.28','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.29','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.30','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.31','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.32','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.33','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.34','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.35','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.36','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.37','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.38','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.39','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.40','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.41','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.42','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.43','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.44','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.45','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.4.99.46','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.1','Asparaginase','L-asparaginase','NULL','NULL','L-asparagine + H(2)O = L-aspartate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.2','Glutaminase','L-glutamine amidohydrolase','NULL','NULL','L-glutamine + H(2)O = L-glutamate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.3','Omega-amidase','NULL','NULL','Acts on glutaramate, succinamate and their 2-oxo derivatives.','A monoamide of a dicarboxylic acid + H(2)O = a dicarboxylate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.4','Amidase','Acylamidase','NULL','NULL','A monocarboxylic acid amide + H(2)O = a monocarboxylate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.5','Urease','NULL','Nickel.','NULL','Urea + H(2)O = CO(2) + 2 NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.6','Beta-ureidopropionase','NULL','NULL','The animal enzyme also acts on beta-ureidoisobutyrate.','N-carbamoyl-beta-alanine + H(2)O = beta-alanine + CO(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.7','Ureidosuccinase','NULL','NULL','NULL','N-carbamoyl-L-aspartate + H(2)O = L-aspartate + CO(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.8','Formylaspartate deformylase','NULL','NULL','NULL','N-formyl-L-aspartate + H(2)O = formate + L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.9','Arylformamidase','Formylase','NULL','Also acts on other aromatic formylamines.','N-formyl-L-kynurenine + H(2)O = formate + L-kynurenine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.10','Formyltetrahydrofolate deformylase','Formyl-FH(4) hydrolase','NULL','NULL','10-formyltetrahydrofolate + H(2)O = formate + tetrahydrofolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.11','Penicillin amidase','Penicillin acylase','NULL','NULL','Penicillin + H(2)O = a carboxylate + 6-aminopenicillanate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.12','Biotinidase','Biotinase','NULL','Also acts on biotin esters.','Biotin amide + H(2)O = biotin + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.13','Aryl-acylamidase','NULL','NULL','Also acts on 4-substituted anilides.','An anilide + H(2)O = a carboxylate + aniline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.14','N-acyl-aliphatic-L-amino acid amidohydrolase','Acylase I','Zn(2+).','The enzyme is found in animals and is involved in the hydrolysis ofN-acylated or N-acetylated amino acids (except L-aspartate).It acts on mercapturic acids (S-conjugates of N-acetyl-L-cysteine)and neutral aliphatic N-acyl-alpha-amino acids.Some bacterial aminoacylases demonstrate substrate specificity ofboth EC 3.5.1.14 and EC 3.5.1.114.Cf. EC 3.5.1.15 and EC 3.5.1.114.','(1) An N-acyl-aliphatic-L-amino acid + H(2)O = an aliphatic L-amino acid+ a carboxylate.(2) An N-acetyl-L-cysteine-S-conjugate + H(2)O = an L-cysteine-S-conjugate + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.15','Aspartoacylase','Aminoacylase II','NULL','NULL','N-acyl-L-aspartate + H(2)O = a carboxylate + L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.16','Acetylornithine deacetylase','Acetylornithinase','NULL','Also hydrolyzes N-acetylmethionine.','N(2)-acetyl-L-ornithine + H(2)O = acetate + L-ornithine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.17','Acyl-lysine deacylase','Epsilon-lysine acylase','NULL','NULL','N(6)-acyl-L-lysine + H(2)O = a carboxylate + L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.18','Succinyl-diaminopimelate desuccinylase','NULL','NULL','NULL','N-succinyl-LL-2,6-diaminoheptanedioate + H(2)O = succinate + LL-2,6-diaminoheptanedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.19','Nicotinamidase','NAMase','NULL','NULL','Nicotinamide + H(2)O = nicotinate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.20','Citrullinase','NULL','NULL','NULL','L-citrulline + H(2)O = L-ornithine + CO(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.21','N-acetyl-beta-alanine deacetylase','NULL','NULL','NULL','N-acetyl-beta-alanine + H(2)O = acetate + beta-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.22','Pantothenase','NULL','NULL','NULL','(R)-pantothenate + H(2)O = (R)-pantoate + beta-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.23','Ceramidase','Acylsphingosine deacylase','NULL','NULL','N-acylsphingosine + H(2)O = a carboxylate + sphingosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.24','Choloylglycine hydrolase','Bile salt hydrolase','NULL','Also acts on the 3-alpha,12-alpha-dihydroxy-derivative, and oncholoyl-taurine.','3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholan-24-oylglycine + H(2)O =3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholanate + glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.25','N-acetylglucosamine-6-phosphate deacetylase','NULL','NULL','Formerly EC 3.5.1.80.','N-acetyl-D-glucosamine 6-phosphate + H(2)O = D-glucosamine 6-phosphate +acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.26','N(4)-(beta-N-acetylglucosaminyl)-L-asparaginase','Aspartylglucosaminidase','NULL','Acts only on asparagine-oligosaccharides containing one amino acid,i.e. the asparagine has free alpha-amino and alpha-carboxyl groups(cf. EC 3.5.1.52).Formerly EC 3.5.1.37.','N(4)-(beta-N-acetyl-D-glucosaminyl)-L-asparagine + H(2)O = N-acetyl-beta-D-glucosaminylamine + L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.27','N-formylmethionylaminoacyl-tRNA deformylase','NULL','NULL','NULL','N-formyl-L-methionylaminoacyl-tRNA + H(2)O = formate +L-methionylaminoacyl-tRNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.28','N-acetylmuramoyl-L-alanine amidase','NULL','NULL','Formerly EC 3.4.12.5, EC 3.4.17.7 and EC 3.4.19.10.','Hydrolyzes the link between N-acetylmuramoyl residues and L-amino acidresidues in certain cell-wall glycopeptides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.29','2-(acetamidomethylene)succinate hydrolase','NULL','NULL','Involved in the degradation of pyridoxin in Pseudomonas.','2-(acetamidomethylene)succinate + 2 H(2)O = acetate + succinatesemialdehyde + NH(3) + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.30','5-aminopentanamidase','5-aminovaleramidase','NULL','The enzyme from Pseudomonas putida also acts on 4-aminobutanamideand, more slowly, on 6-aminohexanamide.','5-aminopentanamide + H(2)O = 5-aminopentanoate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.31','Formylmethionine deformylase','NULL','NULL','NULL','N-formyl-L-methionine + H(2)O = formate + L-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.32','Hippurate hydrolase','Benzoylglycine amidohydrolase','NULL','Acts on various N-benzoylamino acids.','Hippurate + H(2)O = benzoate + glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.33','N-acetylglucosamine deacetylase','NULL','NULL','NULL','N-acetyl-D-glucosamine + H(2)O = D-glucosamine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.34','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.35','D-glutaminase','NULL','NULL','NULL','D-glutamine + H(2)O = D-glutamate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.36','N-methyl-2-oxoglutaramate hydrolase','5-hydroxy-N-methylpyroglutamate synthase','NULL','In the reverse reaction, the product cyclizes non-enzymically to2-hydroxy-N-methyl-5-oxoproline.','N-methyl-2-oxoglutaramate + H(2)O = 2-oxoglutarate + methylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.37','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.38','Glutamin-(asparagin-)ase','Glutaminase-asparaginase','NULL','L-asparagine is hydrolyzed at 0.8 of the rate of L-glutamine.The D-isomers are also hydrolyzed, more slowly.Cf. EC 3.5.1.1 and EC 3.5.1.2.','(1) L-glutamine + H(2)O = L-glutamate + NH(3).(2) L-asparagine + H(2)O = L-aspartate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.39','Alkylamidase','NULL','NULL','The enzyme hydrolyzes N-monosubstituted and N,N-disubstituted amides,and there is some activity toward primary amides.It has little or no activity toward short-chain substrates.','N-methylhexanamide + H(2)O = hexanoate + methylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.40','Acylagmatine amidase','NULL','NULL','Also acts on acetylagmatine, propanoylagmatine and bleomycin B(2).','Benzoylagmatine + H(2)O = benzoate + agmatine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.41','Chitin deacetylase','NULL','NULL','Hydrolyzes the N-acetamido groups of N-acetyl-D-glucosamine residuesin chitin.','Chitin + H(2)O = chitosan + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.42','Nicotinamide-nucleotide amidase','NULL','NULL','Also acts more slowly on beta-nicotinamide D-ribonucleoside.','Beta-nicotinamide D-ribonucleotide + H(2)O = beta-nicotinateD-ribonucleotide + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.43','Peptidyl-glutaminase','Peptidoglutaminase I','NULL','Specific for the hydrolysis of the gamma-amide of glutaminesubstituted at the alpha-amino group, e.g., glycyl-L-glutamine,N-acetyl-L-glutamine and L-leucylglycyl-L-glutamine.','Alpha-N-peptidyl-L-glutamine + H(2)O = alpha-N-peptidyl-L-glutamate +NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.44','Protein-glutamine glutaminase','Glutaminylpeptide glutaminase','NULL','Specific for the hydrolysis of the gamma-amide of glutaminesubstituted at the carboxyl position or both the alpha-amino andcarboxyl positions, e.g., L-glutaminylglycine and L-phenylalanyl-L-glutaminylglycine.','Protein L-glutamine + H(2)O = protein L-glutamate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.45','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.46','6-aminohexanoate-dimer hydrolase','NULL','NULL','Also hydrolyzes, more slowly, oligomers of 6-aminohexanoatecontaining up to 6 residues.The residues are removed sequentially from the N-terminus.','N-(6-aminohexanoyl)-6-aminohexanoate + H(2)O = 2 6-aminohexanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.47','N-acetyldiaminopimelate deacetylase','NULL','NULL','Formerly EC 3.1.1.62.','N-acetyl-LL-2,6-diaminoheptanedioate + H(2)O = acetate + LL-2,6-diaminoheptanedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.48','Acetylspermidine deacetylase','N(1)-acetylspermidine amidohydrolase','NULL','It was initially thought that N(1)-acetylspermidine was the substratefor this deacetylase reaction but this has since been disproved.','N(8)-acetylspermidine + H(2)O = acetate + spermidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.49','Formamidase','Formamide amidohydrolase','NULL','Also acts, more slowly, on acetamide, propanamide and butanamide.','Formamide + H(2)O = formate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.50','Pentanamidase','Valeramidase','NULL','Also acts, more slowly, on other short-chain aliphatic amides.Different from EC 3.5.1.49.','Pentanamide + H(2)O = pentanoate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.51','4-acetamidobutyryl-CoA deacetylase','Aminobutyryl-CoA thiolesterase','NULL','The enzyme also hydrolyzes 4-aminobutanoyl-CoA to aminobutanoate andcoenzyme A.Formerly EC 3.2.2.18.','4-acetamidobutanoyl-CoA + H(2)O = acetate + 4-aminobutanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.52','Peptide-N(4)-(N-acetyl-beta-glucosaminyl)asparagine amidase','Glycopeptidase','NULL','Does not act on (GlcNAc)-Asn, because it requires the presence ofmore than two amino-acid residues in the substrate (cf. EC 3.5.1.26).Formerly EC 3.2.2.18.','Hydrolysis of an N(4)-(acetyl-beta-D-glucosaminyl)asparagine residue inwhich the glucosamine residue may be further glycosylated, to yield a(substituted) N-acetyl-beta-D-glucosaminylamine and a peptide containingan aspartate residue.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.53','N-carbamoylputrescine amidase','NULL','NULL','NULL','N-carbamoylputrescine + H(2)O = putrescine + CO(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.54','Allophanate hydrolase','NULL','NULL','Along with EC 3.5.2.15 and EC 3.5.1.84, forms part of the cyanuric-acid metabolism pathway, which degrades s-triazide herbicides, suchas atrazine (2-chloro-4-(ethylamino)-6-(isopropylamino)-1,3,5-triazine), in bacteria.The Saccharomyces cerevisiae enzyme (but not that from green algae)also catalyzes the reaction of EC 6.3.4.6, thus bringing about thehydrolysis of urea to CO(2) and NH(3) in the presence of ATP andbicarbonate.The enzyme from Pseudomonas sp. strain ADP has a narrow substratespecificity, being unable to use the structurally analogous compoundsurea, hydroxyurea or methylcarbamate as substrate.','Urea-1-carboxylate + H(2)O = 2 CO(2) + 2 NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.55','Long-chain-fatty-acyl-glutamate deacylase','Long-chain aminoacylase','NULL','Does not act on acyl derivatives of other amino acids.Optimum chain length of acyl residue is 12 to 16.','N-long-chain-fatty-acyl-L-glutamate + H(2)O = a long-chain carboxylate +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.56','N,N-dimethylformamidase','NULL','Iron.','Also acts on N-ethylformamide and N-methylformamide and, more slowly,on N,N-diethylformamide, N,N-dimethylacetamide and unsubstituted acylamides.','N,N-dimethylformamide + H(2)O = dimethylamine + formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.57','Tryptophanamidase','NULL','Manganese.','Also acts on N-ethylformamide and L-tyrosinamide, and on sometryptophan dipeptides.','L-tryptophanamide + H(2)O = L-tryptophan + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.58','N-benzyloxycarbonylglycine hydrolase','NULL','Cobalt; Zinc.','Also acts, more slowly, on N-benzyloxycarbonylalanine, but not on thecorresponding derivatives of other amino acids or onN-benzyloxycarbonylpeptides (cf. EC 3.5.1.64).','N-benzyloxycarbonylglycine + H(2)O = benzyl alcohol + CO(2) + glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.59','N-carbamoylsarcosine amidase','CSHase','NULL','NULL','N-carbamoylsarcosine + H(2)O = sarcosine + CO(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.60','N-(long-chain-acyl)ethanolamine deacylase','N-acylethanolamine amidohydrolase','NULL','Does not act on N-acylsphingosine or N,O-diacylethanolamine.','N-(long-chain-acyl)ethanolamine + H(2)O = a long-chain carboxylate +ethanolamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.61','Mimosinase','Mimosine amidohydrolase','NULL','An enzyme from Leucaena leucocephala leaf, which also contains thetoxic amino acid, mimosine.','(S)-2-amino-3-(3-hydroxy-4-oxo-4H-pyridin-1-yl)propanoate + H(2)O =3-hydroxy-4H-pyrid-4-one + L-serine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.62','Acetylputrescine deacetylase','NULL','NULL','The enzyme from Micrococcus luteus also acts on N(8)-acetylspermidineand acetylcadaverine, but more slowly.','N-acetylputrescine + H(2)O = acetate + putrescine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.63','4-acetamidobutyrate deacetylase','NULL','NULL','Also acts on N-acetyl-beta-alanine and 5-acetamidopentanoate.','4-acetamidobutanoate + H(2)O = acetate + 4-aminobutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.64','N(alpha)-benzyloxycarbonylleucine hydrolase','NULL','NULL','Also acts on N(alpha)-t-butoxycarbonyl-L-leucine, and, more slowly,on the corresponding derivatives of L-aspartate, L-methionine,L-glutamate and L-alanine (cf. EC 3.5.1.58).','N(alpha)-benzyloxycarbonyl-L-leucine + H(2)O = benzyl alcohol + CO(2) +L-leucine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.65','Theanine hydrolase','NULL','NULL','Also acts on other N-alkyl-L-glutamines.','N(5)-ethyl-L-glutamine + H(2)O = L-glutamate + ethylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.66','2-(hydroxymethyl)-3-(acetamidomethylene)succinate hydrolase','NULL','NULL','Involved in the degradation of pyridoxin by Pseudomonas andArthrobacter.','2-(hydroxymethyl)-3-(acetamidomethylene)succinate + 2 H(2)O = acetate +2-(hydroxymethyl)-4-oxobutanoate + NH(3) + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.67','4-methyleneglutaminase','4-methyleneglutamine deamidase','NULL','NULL','4-methylene-L-glutamine + H(2)O = 4-methylene-L-glutamate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.68','N-formylglutamate deformylase','Beta-citryl-L-glutamate hydrolase','NULL','The animal enzyme also acts on beta-citryl-L-glutamate and beta-citryl-L-glutamine.','N-formyl-L-glutamate + H(2)O = formate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.69','Glycosphingolipid deacylase','Glycosphingolipid ceramide deacylase','NULL','Does not act on sphingolipids such as ceramide.Not identical with EC 3.5.1.23.','Hydrolysis of gangliosides and neutral glycosphingolipids, releasingfatty acids to form the lyso-derivatives.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.70','Aculeacin-A deacylase','NULL','NULL','NULL','Hydrolysis of the amide bond in aculeacin A and related neutrallipopeptide antibiotics, releasing the long-chain fatty acid side-chain.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.71','N-feruloylglycine deacylase','NULL','NULL','Hydrolyzes a range of L-amino acids from the cinnamoyl andsubstituted cinnamoyl series.Not identical with EC 3.5.1.14.','N-feruloylglycine + H(2)O = ferulate + glycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.72','D-benzoylarginine-4-nitroanilide amidase','NULL','NULL','NULL','N-benzoyl-D-arginine-4-nitroanilide + H(2)O = N-benzoyl-D-arginine +4-nitroaniline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.73','Carnitinamidase','NULL','NULL','Does not act on D-carnitinamide.','L-carnitinamide + H(2)O = L-carnitine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.74','Chenodeoxycholoyltaurine hydrolase','NULL','NULL','Some other taurine conjugates are hydrolyzed, but not glycineconjugates of bile acids.','Chenodeoxycholoyltaurine + H(2)O = chenodeoxycholate + taurine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.75','Urethanase','Urethane hydrolase','NULL','NULL','Urethane + H(2)O = ethanol + CO(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.76','Arylalkyl acylamidase','Aralkyl acylamidase','NULL','Strict specificity for N-acetyl arylalkylamines, including N-acetyl-2-phenylethylamine, N-acetyl-3-phenylpropylamine, N-acetyldopamine,N-acetyl-serotonin and melatonin.It also accepts arylalkyl acetates but not acetanilide derivatives,which are common substrates of EC 3.5.1.13.','N-acetylarylalkylamine + H(2)O = arylalkylamine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.77','N-carbamoyl-D-amino-acid hydrolase','D-N-carbamoylase','NULL','This enzyme, along with EC 3.5.1.87, EC 5.1.99.5 and EC 3.5.2.2,forms part of the reaction cascade known as the hydantoinaseprocess, which allows the total conversion of D,L-5-monosubstitutedhydantoins into optically pure D- or L-amino acids.Has strict stereospecificity for N-carbamoyl-D-amino acids and doesnot act upon the corresponding L-amino acids or N-formyl amino acids,N-carbamoyl-sarcosine, -citrulline, -allantoin and -ureidopropanoate,which are substrates for other amidohydrolases.','N-carbamoyl-D-amino acid + H(2)O = D-amino acid + NH(3) + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.78','Glutathionylspermidine amidase','Glutathionylspermidine amidohydrolase (spermidine-forming)','NULL','Transforms glutathionylspermidine into glutathione and spermidine.The enzyme from Escherichia coli is bifunctional and also catalyzesthe reaction of EC 6.3.1.8, resulting in a net hydrolysis of ATP.','Glutathionylspermidine + H(2)O = glutathione + spermidine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.79','Phthalyl amidase','o-phthalyl amidase','NULL','Phthalyl is used to mean 2-carboxybenzoyl.The enzyme from Xanthobacter agilis hydrolyzes phthalylated aminoacids, peptides, beta-lactams, aromatic and aliphatic amines.The substituent on nitrogen may be an alkyl group, but may also becomplex, giving an amino acid or peptide derivative.Substitutions on the phthalyl ring include 6-F, 6-NH(2), 3-OH, and anitrogen in the aromatic ring ortho to the carboxy group attached tothe amine.','A phthalylamide + H(2)O = phthalic acid + a substituted amine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.80','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.81','N-acyl-D-amino-acid deacylase','D-aminoacylase','Zinc.','Has a wide specificity; hydrolyzes N-acyl derivative of neutralD-amino acids.Used in separating D- and L-amino acids.','N-acyl-D-amino acid + H(2)O = a carboxylate + D-amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.82','N-acyl-D-glutamate deacylase','N-acyl-D-glutamate amidohydrolase','Zinc.','NULL','N-acyl-D-glutamate + H(2)O = a carboxylate + D-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.83','N-acyl-D-aspartate deacylase','N-acyl-D-aspartate amidohydrolase','Zinc.','NULL','N-acyl-D-aspartate + H(2)O = a carboxylate + D-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.84','Biuret amidohydrolase','NULL','NULL','Along with EC 3.5.2.15 and EC 3.5.1.54, this enzyme forms part of thecyanuric-acid metabolism pathway, which degrades s-triazideherbicides, such as atrazine (2-chloro-4-(ethylamino)-6-(isopropylamino)-1,3,5-triazine), in bacteria.Urea-1-carboxylate rather than urea (as was thought previously) isthe 2-nitrogen intermediate in cyanuric-acid metabolism in bacteria.The product, urea-1-carboxylate, can spontaneously decarboxylateunder acidic conditions to form urea but, under physiologicalconditions, it can be converted into CO(2) and ammonia by the actionof EC 3.5.1.54.','Biuret + H(2)O = urea-1-carboxylate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.85','(S)-N-acetyl-1-phenylethylamine hydrolase','(S)-N-acetyl-1-phenylethylamine amidohydrolase','NULL','Inhibited by phenylmethanesulfonyl fluoride.Some related acetylated compounds are hydrolyzed with variableenantiomeric selectivities.','N-acetylphenylethylamine + H(2)O = phenylethylamine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.86','Mandelamide amidase','Mandelamide hydrolase','NULL','NULL','(R)-mandelamide + H(2)O = (R)-mandelate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.87','N-carbamoyl-L-amino-acid hydrolase','L-carbamoylase','Cobalt or nickel or manganese.','This enzyme, along with EC 3.5.1.77, EC 5.1.99.5 and EC 3.5.2.2,forms part of the reaction cascade known as the hydantoinaseprocess, which allows the total conversion of D,L-5-monosubstitutedhydantoins into optically pure D- or L-amino acids.The enzyme from Alcaligenes xylosoxidans has broad specificity forcarbamoyl-L-amino acids, although it is inactive on the carbamoylderivatives of glutamate, aspartate, arginine, tyrosine ortryptophan.The enzyme from Sinorhizobium meliloti requires a divalent cation foractivity and can hydrolyze N-carbamoyl-L-tryptophan as well asN-carbamoyl L-amino acids with aliphatic substituents.The enzyme is inactive on derivatives of D-amino acids.In addition to N-carbamoyl L-amino acids, the enzyme can alsohydrolyze formyl and acetyl derivatives to varying degrees.','N-carbamoyl-L-2-amino acid (a 2-ureido carboxylate) + H(2)O = L-2-aminoacid + NH(3) + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.88','Peptide deformylase','PDF','Fe(2+).','Requires at least a dipeptide for an efficient rate of reaction.N-terminal L-methionine is a prerequisite for activity but the enzymehas broad specificity at other positions.Differs in substrate specificity from EC 3.5.1.27 and EC 3.5.1.31.','Formyl-L-methionyl peptide + H(2)O = formate + methionyl peptide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.89','N-acetylglucosaminylphosphatidylinositol deacetylase','Acetylglucosaminylphosphatidylinositol deacetylase','NULL','Involved in the second step of glycosylphosphatidylinositol (GPI)anchor formation in all eukaryotes.The enzyme appears to be composed of a single subunit (PIG-L inmammalian cells and GPI12 in Saccharomyces cerevisiae).In some species, the long-chain sn-1-acyl group of the phosphatidylgroup is replaced by a long-chain alkyl or alk-1-enyl group.Formerly EC 3.1.1.69.','6-(N-acetyl-alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol + H(2)O= 6-(alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.90','Adenosylcobinamide hydrolase','AdoCbi amidohydrolase','NULL','Involved in the salvage pathway of cobinamide in archaea.Archaea convert adenosylcobinamide (AdoCbi) into adenosylcobinamidephosphate (AdoCbi-P) in two steps.First, the amidohydrolase activity of cbiZ cleaves off theaminopropanol moiety of AdoCbi yielding adenosylcobyric acid(AdoCby); second, AdoCby is converted into AdoCbi-P by the action ofEC 6.3.1.10 (CbiB).','Adenosylcobinamide + H(2)O = adenosylcobyric acid + (R)-1-aminopropan-2-ol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.91','N-substituted formamide deformylase','NULL','Zinc.','While N-benzylformamide is the best substrate, the enzyme fromArthrobacter pascens can also act on the N-substituted formamidesN-butylformamide, N-allylformamide, N-(2-(cyclohex-1-enyl)ethyl)formamide and N-(1-phenylethyl)formamide, but much moreslowly.Amides of other acids do not act as substrates.','N-benzylformamide + H(2)O = formate + benzylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.92','Pantetheine hydrolase','Pantetheinase','NULL','Hydrolyzes only one of the amide bonds of pantetheine.The substrate analogs phosphopantetheine and CoA are not substrates.Recycles pantothenate (vitamin B(5)) and produces cysteamine,a potent anti-oxidant.','(R)-pantetheine + H(2)O = (R)-pantothenate + 2-aminoethanethiol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.93','Glutaryl-7-aminocephalosporanic-acid acylase','7-beta-(4-carboxybutanamido)cephalosporanic acid acylase','NULL','Forms 7-aminocephalosporanic acid, a key intermediate in thesynthesis of cephem antibiotics.It reacts only weakly with cephalosporin C.','(7R)-7-(4-carboxybutanamido)cephalosporanate + H(2)O = (7R)-7-aminocephalosporanate + glutarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.94','Gamma-glutamyl-gamma-aminobutyrate hydrolase','Gamma-glutamyl-GABA hydrolase','NULL','Forms part of a novel putrescine-utilizing pathway in Escherichiacoli, in which it has been hypothesized that putrescine is firstglutamylated to form gamma-glutamylputrescine, which is oxidized to4-(gamma-glutamylamino)butanal and then to 4-(gamma-glutamylamino)butanoate.Also can catalyze the reactions of EC 3.5.1.35 and EC 3.5.1.65.','4-(L-gamma-glutamylamino)butanoate + H(2)O = 4-aminobutanoate +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.95','N-malonylurea hydrolase','Ureidomalonase','NULL','Forms part of the oxidative pyrimidine-degrading pathway in somemicroorganisms, along with EC 1.17.99.4 and EC 3.5.2.1.','3-oxo-3-ureidopropanoate + H(2)O = malonate + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.96','Succinylglutamate desuccinylase','N(2)-succinylglutamate desuccinylase','Cobalt.','N(2)-acetylglutamate is not a substrate.The final enzyme in the arginine succinyltransferase (AST) pathwayfor the catabolism of arginine.This pathway converts the carbon skeleton of arginine into glutamate,with the concomitant production of ammonia and conversion ofsuccinyl-CoA into succinate and CoA.The five enzymes involved in this pathway are EC 2.3.1.109,EC 3.5.3.23, EC 2.6.1.11, EC 1.2.1.71 and EC 3.5.1.96.','N-succinyl-L-glutamate + H(2)O = succinate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.97','Acyl-homoserine-lactone acylase','Acyl-homoserine lactone acylase','NULL','Acyl-homoserine lactones (AHLs) are produced by a number of bacterialspecies and are used by them to regulate the expression of virulencegenes in a process known as quorum-sensing.Each bacterial cell has a basal level of AHL and, once the populationdensity reaches a critical level, it triggers AHL-signaling which,in turn, initiates the expression of particular virulence genes.Plants or animals capable of degrading AHLs would have a therapeuticadvantage in avoiding bacterial infection as they could prevent AHL-signaling and the expression of virulence genes in quorum-sensingbacteria.This quorum-quenching enzyme removes the fatty-acid side chain fromthe homoserine lactone ring of AHL-dependent quorum-sensing signalmolecules.It has broad specificity for AHLs with side changes ranging in lengthfrom 11 to 14 carbons.Substituents at the 3-position, as found in N-(3-oxododecanoyl)-L-homoserine lactone, do not affect this activity.','An N-acyl-L-homoserine lactone + H(2)O = L-homoserine lactone +a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.98','Histone deacetylase','HDAC','NULL','A class of enzymes that remove acetyl groups from N(6)-acetyl-lysineresidues on a histone.The reaction of this enzyme is opposite to that of EC 2.3.1.48.Histone deacetylases (HDACs) can be organized into three classesdepending on sequence similarity and domain organization.Histone acetylation plays an important role in regulation of geneexpression.In eukaryotes, HDACs play a key role in the regulation oftranscription and cell proliferation.May be identical to EC 3.5.1.17.','Hydrolysis of an N(6)-acetyl-lysine residue of a histone to yield adeacetylated histone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.99','Fatty acid amide hydrolase','Anandamide amidohydrolase','NULL','Integral membrane protein, the enzyme is responsible for thecatabolism of neuromodulatory fatty acid amides, including anandamideand oleamide, occurs in mammalia.Formerly EC 3.5.1.n2.','(1) Anandamide + H(2)O = arachidonic acid + ethanolamine.(2) Oleamide + H(2)O = oleic acid + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.100','(R)-amidase','R-amidase','NULL','In addition (R)-piperidine-3-carboxamide is hydrolyzed to (R)-piperidine-3-carboxylic acid and NH(3), and (R)-N-tert-butylpiperazine-2-carboxamide is hydrolyzed to (R)-piperazine-2-carboxylic acid and tert-butylamine with lower activity.The enzyme does not act on the other amide substrates which arehydrolyzed by EC 3.5.1.4 (amidase).','(1) (R)-piperazine-2-carboxamide + H(2)O = (R)-piperazine-2-carboxylicacid + NH(3).(2) Beta-alaninamide + H(2)O = beta-alanine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.101','L-proline amide hydrolase','L-amino acid amidase','NULL','NULL','(1) (S)-piperidine-2-carboxamide + H(2)O = (S)-piperidine-2-carboxylicacid + NH(3).(2) L-prolinamide + H(2)O = L-proline + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.102','2-amino-5-formylamino-6-ribosylaminopyrimidin-4(3H)-one 5-monophosphatedeformylase','NULL','NULL','The enzyme catalyzes the second step in archaeal riboflavin and 7,8-didemethyl-8-hydroxy-5-deazariboflavin biosynthesis.The first step is catalyzed by EC 3.5.4.29.The bacterial enzyme, EC 3.5.4.25 catalyzes both reactions.','2-amino-5-formylamino-6-(5-phospho-D-ribosylamino)pyrimidin-4(3H)-one +H(2)O = 2,5-diamino-6-(5-phospho-D-ribosylamino)pyrimidin-4(3H)-one +formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.103','N-acetyl-1-D-myo-inositol-2-amino-2-deoxy-alpha-D-glucopyranosidedeacetylase','1D-myo-inositol 2-acetamido-2-deoxy-alpha-D-glucopyranoside deacetylase','Zinc.','This enzyme is considered the key enzyme and rate limiting step inthe mycothiol biosynthesis pathway.In addition to acetylase activity, the enzyme possesses weakmycothiol conjugate amidase activity, and shares sequence similaritywith mycothiol S-conjugate amidase.Formerly EC 3.1.1.n1.','1-O-(2-acetamido-2-deoxy-alpha-D-glucopyranosyl)-1D-myo-inositol + H(2)O= 1-O-(2-amino-2-deoxy-alpha-D-glucopyranosyl)-1D-myo-inositol + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.104','Peptidoglycan-N-acetylglucosamine deacetylase','N-acetylglucosamine deacetylase','NULL','Modification of peptidoglycan by N-deacetylation is an importantfactor in virulence of Helicobacter pylori, Listeria monocytogenesand Streptococcus suis.The enzyme from Streptococcus pneumoniae is a metalloenzyme using aHis-His-Asp zinc-binding triad with a nearby aspartic acid andhistidine acting as the catalytic base and acid, respectively.','Peptidoglycan-N-acetyl-D-glucosamine + H(2)O = peptidoglycan-D-glucosamine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.105','Chitin disaccharide deacetylase','Chitin oligosaccharide amidohydolase','NULL','Chitin oligosaccharide deacetylase is a key enzyme in the chitincatabolic cascade of chitinolytic Vibrio strains.Besides being a nutrient, the heterodisaccharide product 4-O-(N-acetyl-beta-D-glucosaminyl)-D-glucosamine is a unique inducer ofchitinase production in Vibrio parahemolyticus.In contrast to EC 3.5.1.41 this enzyme is specific for the chitindisaccharide.It also deacetylates the chitin trisaccharide with lower efficiency.No activity with higher polymers of GlcNAc.','2-(acetylamino)-4-O-(2-(acetylamino)-2-deoxy-beta-D-glucopyranosyl)-2-deoxy-beta-D-glucopyranose + H(2)O = 2-(acetylamino)-4-O-(2-amino-2-deoxy-beta-D-glucopyranosyl)-2-deoxy-beta-D-glucopyranose + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.106','N-formylmaleamate deformylase','NULL','NULL','The reaction is involved in the aerobic catabolism of nicotinic acid.','N-formylmaleamic acid + H(2)O = maleamate + formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.107','Maleamate amidohydrolase','NULL','NULL','The reaction is involved in the aerobic catabolism of nicotinic acid.','Maleamate + H(2)O = maleate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.108','UDP-3-O-acyl-N-acetylglucosamine deacetylase','UDP-3-O-((R)-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase','Zinc.','The enzyme catalyzes a committed step in the biosynthesis of lipid A.Formerly EC 3.5.1.n1.','UDP-3-O-((3R)-3-hydroxytetradecanoyl)-N-acetyl-alpha-D-glucosamine +H(2)O = UDP-3-O-((3R)-3-hydroxytetradecanoyl)-alpha-D-glucosamine +acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.109','Sphingomyelin deacylase','Glucosylceramide sphingomyelin deacylase','NULL','The enzyme is involved in the sphingolipid metabolism in theepidermis.','(1) An N-acyl-sphingosylphosphorylcholine + H(2)O = a fatty acid +sphingosylphosphorylcholine.(2) A D-glucosyl-N-acylsphingosine + H(2)O = a fatty acid + D-glucosyl-sphingosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.110','Peroxyureidoacrylate/ureidoacrylate amidohydrolase','NULL','NULL','The enzyme also shows activity toward ureidoacrylate.Part of the Rut pyrimidine catabolic pathway.Formerly EC 3.5.1.n4.','(1) (Z)-3-ureidoacrylate peracid + H(2)O = (Z)-3-peroxyaminoacrylate +NH(3).(2) (Z)-3-ureidoacrylate + H(2)O = (Z)-3-aminoacrylate + CO(2) + NH(3).(3) (Z)-2-methyl-ureidoacrylate peracid + H(2)O = (Z)-2-methyl-peroxyaminoacrylate + CO(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.111','2-oxoglutaramate amidase','Omega-amidase','NULL','The enzyme, which is highly specific for its substrate, participatesin the nicotine degradation pathway of several Gram-positivebacteria.','2-oxoglutaramate + H(2)O = 2-oxoglutarate + ammonia.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.112','2-N-acetylparomamine deacetylase','NULL','NULL','Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including kanamycin, butirosin, neomycinand ribostamycin.The enzyme from the bacterium Streptomyces fradiae can also accept2-acetyl-6-hydroxyneomycin C as substrate, cf. EC 3.5.1.113.','2-N-acetylparomamine + H(2)O = paromamine + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.113','2-acetyl-6-hydroxyneomycin C deacetylase','NULL','NULL','Involved in the biosynthetic pathway of aminoglycoside antibiotics ofthe neomycin family.The enzyme from the bacterium Streptomyces fradiae also catalyzesEC 3.5.1.112.','2-acetyl-6-deamino-6-hydroxyneomycin C + H(2)O = 6-deamino-6-hydroxyneomycin C + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.114','N-acyl-aromatic-L-amino acid amidohydrolase','Aminoacylase 3','Zinc.','This enzyme is found in animals and is involved in the hydrolysis ofN-acylated or N-acetylated amino acids (except L-aspartate).It preferentially deacetylates N(alpha)-acetylated aromatic aminoacids and mercapturic acids (S-conjugates of N-acetyl-L-cysteine)that are usually not deacetylated by EC 3.5.1.14.Some bacterial aminoacylases demonstrate substrate specificity forboth EC 3.5.1.14 and EC 3.5.1.114.Cf. EC 3.5.1.14 and EC 3.5.1.15.','(1) An N-acyl-aromatic-L-amino acid + H(2)O = an aromatic-L-amino acid +a carboxylate.(2) An N-acetyl-L-cysteine-S-conjugate + H(2)O = an L-cysteine-S-conjugate + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.115','Mycothiol S-conjugate amidase','NULL','NULL','The enzyme that is found in actinomycetes is involved in thedetoxification of oxidizing agents and electrophilic antibiotics.The enzyme has low activity with 1-O-(2-acetamido-2-deoxy-alpha-D-glucopyranosyl)-1D-myo-inositol as substrate (cf. EC 3.5.1.103).','A mycothiol S-conjugate + H(2)O = an N-acetyl L-cysteine-S-conjugate +1-O-(2-amino-2-deoxy-alpha-D-glucopyranosyl)-1D-myo-inositol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.116','Ureidoglycolate amidohydrolase','Ureidoglycolate hydrolase','NULL','This plant enzyme is involved in the degradation of ureidoglycolate,an intermediate of purine degradation.Not to be confused with EC 4.3.2.3, which releases urea rather thanammonia.Formerly EC 3.5.3.19.','(S)-ureidoglycolate + H(2)O = glyoxylate + 2 NH(3) + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.n2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.n3','4-deoxy-4-formamido-L-arabinose-phosphoundecaprenol deformylase','Polymyxin resistance protein PmrJ','NULL','NULL','4-deoxy-4-formamido-beta-L-arabinose di-trans,poly-cis-undecaprenylphosphate + H(2)O = 4-amino-4-deoxy-alpha-L-arabinose di-trans,poly-cis-undecaprenyl phosphate + formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.1.n4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.1','Barbiturase','NULL','Zinc.','Specific for barbiturate as substrate.Forms part of the oxidative pyrimidine-degrading pathway in somemicroorganisms, along with EC 1.17.99.4 and EC 3.5.1.95.It was previously thought that the end-products of the reaction weremalonate and urea.May be involved in the regulation of pyrimidine metabolism, alongwith EC 2.4.2.9.','Barbiturate + H(2)O = 3-oxo-3-ureidopropanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.2','Dihydropyrimidinase','Hydantoinase','NULL','Also acts on dihydrothymine and hydantoin.','5,6-dihydrouracil + H(2)O = 3-ureidopropanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.3','Dihydroorotase','Carbamoylaspartic dehydrase','NULL','NULL','(S)-dihydroorotate + H(2)O = N-carbamoyl-L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.4','Carboxymethylhydantoinase','NULL','NULL','NULL','L-5-carboxymethylhydantoin + H(2)O = N-carbamoyl-L-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.5','Allantoinase','NULL','NULL','NULL','(S)-allantoin + H(2)O = allantoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.6','Beta-lactamase','Cephalosporinase','Zinc.','Zinc is only requires in class-B enzymes.A group of enzymes of varying specificity hydrolyzing beta-lactams;some act more rapidly on penicillins, some more rapidly oncephalosporins.Formerly EC 3.5.2.8.','A beta-lactam + H(2)O = a substituted beta-amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.7','Imidazolonepropionase','4(5)-imidazolone-5(4)-propionic acid hydrolase','NULL','NULL','(S)-3-(5-oxo-4,5-dihydro-3H-imidazol-4-yl)propanoate + H(2)O =N-formimidoyl-L-glutamate + H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.8','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.9','5-oxoprolinase (ATP-hydrolyzing)','5-OPase','NULL','NULL','ATP + 5-oxo-L-proline + 2 H(2)O = ADP + phosphate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.10','Creatininase','Creatinine amidohydrolase','NULL','NULL','Creatinine + H(2)O = creatine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.11','L-lysine-lactamase','L-alpha-aminocaprolactam hydrolase','NULL','Also hydrolyzes L-lysinamide.','(S)-2-aminohexano-6-lactam + H(2)O = L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.12','6-aminohexanoate-cyclic-dimer hydrolase','NULL','NULL','The cyclic dimer of 6-aminohexanoate is converted to the lineardimer.','1,8-diazacyclotetradecane-2,9-dione + H(2)O = N-(6-aminohexanoyl)-6-aminohexanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.13','2,5-dioxopiperazine hydrolase','Cyclo(Gly-Gly) hydrolase','NULL','Highly specific; does not hydrolyze other dioxopiperazines,glycylglycine, proteins or barbiturates.','2,5-dioxopiperazine + H(2)O = glycylglycine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.14','N-methylhydantoinase (ATP-hydrolyzing)','N-methylhydantoin amidohydrolase','NULL','NULL','ATP + N-methylimidazolidine-2,4-dione + 2 H(2)O = ADP + phosphate +N-carbamoylsarcosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.15','Cyanuric acid amidohydrolase','NULL','NULL','Along with EC 3.5.1.54 and EC 3.5.1.84, forms part of the cyanuric-acid metabolism pathway, which degrades s-triazide herbicides, suchas atrazine (2-chloro-4-(ethylamino)-6-(isopropylamino)-1,3,5-triazine), in bacteria.A key enzyme in the pathway, catalyzing the ring cleavage of cyanuricacid.Specific for cyanuric acid as substrate as neither the structurallyrelated compounds ammeline (2,4-diamino-6-hydroxy-s-triazine) andammelide (2-amino-4,6-dihydroxy-s-triazine) nor a number ofpyrimidine compounds, such as uracil and cytosine, can act assubstrates.','Cyanuric acid + H(2)O = biuret + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.16','Maleimide hydrolase','Cyclic imide hydrolase','NULL','Succinimide and glutarimide, and sulfur-containing cyclic imides,such as rhodanine, can also act as substrates for the enzyme fromBlastobacter sp. A17p-4.The reverse, cyclization, reaction is also catalyzed, but much moreslowly.It has lower activity toward cyclic ureides, which are the substratesof EC 3.5.2.2.','Maleimide + H(2)O = maleamic acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.17','Hydroxyisourate hydrolase','5-hydroxyisourate hydrolase','NULL','The reaction is the first stage in the conversion of5-hydroxyisourate into (S)-allantoin.This reaction will also occur spontaneously, but more slowly.','5-hydroxyisourate + H(2)O = 5-hydroxy-2-oxo-4-ureido-2,5-dihydro-1H-imidazole-5-carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.18','Enamidase','NULL','Iron; Zinc.','Forms part of the nicotinate-fermentation catabolism pathway inEubacterium barkeri.Other enzymes involved in this pathway are EC 1.17.1.5, EC 1.3.7.1,EC 1.1.1.291, EC 5.4.99.4, EC 5.3.3.6, EC 4.2.1.85 and EC 4.1.3.32.','6-oxo-1,4,5,6-tetrahydronicotinate + 2 H(2)O = 2-formylglutarate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.2.19','Streptothricin hydrolase','NULL','NULL','The enzyme also catalyzes the hydrolysis of streptothricin-D tostreptothricin-D acid.The enzyme is responsible for streptothricin resistance inStreptomyces albulus and Streptomyces noursei.','Streptothricin-F + H(2)O = streptothricin-F acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.1','Arginase','Arginine amidinase','Manganese.','Also hydrolyzes alpha-N-substituted L-arginines and canavanine.','L-arginine + H(2)O = L-ornithine + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.2','Guanidinoacetase','Glycocyaminase','Manganese.','NULL','Guanidinoacetate + H(2)O = glycine + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.3','Creatinase','Creatine amidinohydrolase','NULL','NULL','Creatine + H(2)O = sarcosine + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.4','Allantoicase','Allantoate amidinohydrolase','NULL','Also hydrolyzes (R)-ureidoglycolate to glyoxylate and urea.','Allantoate + H(2)O = (S)-ureidoglycolate + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.5','Formimidoylaspartate deiminase','Formiminoaspartate deiminase','NULL','NULL','N-formimidoyl-L-aspartate + H(2)O = N-formyl-L-aspartate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.6','Arginine deiminase','Arginine dihydrolase','NULL','Also acts on canavanine.','L-arginine + H(2)O = L-citrulline + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.7','Guanidinobutyrase','NULL','Manganese.','Also acts, very slowly, on 5-guanidinopentanoate and 6-guanidino-hexanoate.','4-guanidinobutanoate + H(2)O = 4-aminobutanoate + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.8','Formimidoylglutamase','Formiminoglutamase','NULL','NULL','N-formimidoyl-L-glutamate + H(2)O = L-glutamate + formamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.9','Allantoate deiminase','NULL','Manganese.','This enzyme is part of the ureide pathway, which permits certainorganisms to recycle the nitrogen in purine compounds.This enzyme, which liberates ammonia from allantoate, is present inplants and bacteria.In plants it is localized in the endoplasmic reticulum.Requires manganese.','Allantoate + H(2)O = (S)-ureidoglycine + NH(3) + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.10','D-arginase','NULL','NULL','NULL','D-arginine + H(2)O = D-ornithine + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.11','Agmatinase','Agmatine ureohydrolase','NULL','NULL','Agmatine + H(2)O = putrescine + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.12','Agmatine deiminase','NULL','NULL','The plant enzyme also catalyzes the reactions of EC 2.1.3.3,EC 2.1.3.6 and EC 2.7.2.2, thus functioning as a putrescine synthase,converting agmatine and ornithine into putrescine and citrulline,respectively.','Agmatine + H(2)O = N-carbamoylputrescine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.13','Formimidoylglutamate deiminase','Formiminoglutamate deiminase','NULL','NULL','N-formimidoyl-L-glutamate + H(2)O = N-formyl-L-glutamate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.14','Amidinoaspartase','NULL','NULL','Also acts slowly on N-amidino-L-glutamate.','N-amidino-L-aspartate + H(2)O = L-aspartate + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.15','Protein-arginine deiminase','NULL','NULL','Also acts on N-acyl-L-arginine and, more slowly, on L-arginineesters.','Protein L-arginine + H(2)O = protein L-citrulline + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.16','Methylguanidinase','NULL','NULL','Acts on other alkylguanidines, very slowly.','Methylguanidine + H(2)O = methylamine + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.17','Guanidinopropionase','NULL','Manganese.','Also acts, more slowly, on taurocyamine and 4-guanidinobutanoate.','3-guanidinopropanoate + H(2)O = beta-alanine + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.18','Dimethylargininase','Dimethylarginine dimethylaminohydrolase','NULL','Also acts on N(omega)-methyl-L-arginine.','N(omega),N(omega)-dimethyl-L-arginine + H(2)O = dimethylamine +L-citrulline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.19','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.20','Diguanidinobutanase','NULL','NULL','Other diguanidinoalkanes with 3 to 10 methylene groups can act, moreslowly.','1,4-diguanidinobutane + H(2)O = agmatine + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.21','Methylenediurea deaminase','Methylenediurease','NULL','The methylenediurea is hydrolyzed and decarboxylated to give anaminated methylurea, which then spontaneously hydrolyzes tohydroxymethylurea.The enzyme from Ochrobactrum anthropi also hydrolyzesdimethylenetriurea and trimethylenetetraurea as well asureidoglycolate, which is hydrolyzed to urea and glyoxylate,and allantoate, which is hydrolyzed to ureidoglycolate, ammonia andcarbon dioxide.','NH(2)-CO-NH-CH(2)-NH-CO-NH(2) + 2 H(2)O = N-(hydroxymethyl)urea + 2 NH(3)+ CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.22','Proclavaminate amidinohydrolase','PAH','Manganese.','Forms part of the pathway for the biosythesis of the beta-lactamaseinhibitor clavulanate in Streptomyces clavuligerus.It carries out an intermediary reaction between the first reaction ofEC 1.14.11.21, and the second and third reactions of that enzyme.','Amidinoproclavaminate + H(2)O = proclavaminate + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.23','N-succinylarginine dihydrolase','Arginine succinylhydrolase','NULL','Arginine, N(2)-acetylarginine and N(2)-glutamylarginine do not act assubstrates.The second enzyme in the arginine succinyltransferase (AST) pathwayfor the catabolism of arginine.This pathway converts the carbon skeleton of arginine into glutamate,with the concomitant production of ammonia and conversion ofsuccinyl-CoA into succinate and CoA.The five enzymes involved in this pathway are EC 2.3.1.109,EC 3.5.3.23, EC 2.6.1.11, EC 1.2.1.71 and EC 3.5.1.96.','N(2)-succinyl-L-arginine + 2 H(2)O = N(2)-succinyl-L-ornithine + 2 NH(3)+ CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.24','N(1)-aminopropylagmatine ureohydrolase','NULL','NULL','The enzyme, which has been characterized from the hyperthermophilicarchaeon Pyrococcus kodakarensis and the thermophilic Gram-negativebacterium Thermus thermophilus, is involved in the biosynthesis ofspermidine.','N(1)-aminopropylagmatine + H(2)O = spermidine + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.25','N(omega)-hydroxy-L-arginine amidinohydrolase','NULL','NULL','The enzyme participates in the biosynthetic pathway of D-cycloserine,an antibiotic substance produced by several Streptomyces species.','N(omega)-hydroxy-L-arginine + H(2)O = L-ornithine + hydroxyurea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.3.26','(S)-ureidoglycine aminohydrolase','NULL','Manganese.','This enzyme, found in plants and bacteria, is part of the ureidepathway, which enables the recycling of the nitrogen in purinecompounds.In plants it is localized in the endoplasmic reticulum.','(S)-2-ureidoglycine + H(2)O = (S)-ureidoglycolate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.1','Cytosine deaminase','Cytosine aminohydrolase','NULL','Also acts on 5-methylcytosine.','Cytosine + H(2)O = uracil + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.2','Adenine deaminase','Adenase','NULL','NULL','Adenine + H(2)O = hypoxanthine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.3','Guanine deaminase','GAH','NULL','NULL','Guanine + H(2)O = xanthine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.4','Adenosine deaminase','Adenosine aminohydrolase','NULL','NULL','Adenosine + H(2)O = inosine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.5','Cytidine deaminase','(Deoxy)cytidine deaminase','Zinc.','Catalyzes the deamination of cytidine and 2-deoxycytidine withsimilar efficiencies.The enzyme, which is widely distributed among organisms, is involvedin salvage of both exogenous and endogenous cytidine and2-deoxycytidine for UMP synthesis.Formerly EC 3.5.4.14.','(1) Cytidine + H(2)O = uridine + NH(3).(2) 2-deoxycytidine + H(2)O = 2-deoxyuridine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.6','AMP deaminase','5-AMP deaminase','NULL','Cf. EC 3.5.4.17.','AMP + H(2)O = IMP + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.7','ADP deaminase','NULL','NULL','NULL','ADP + H(2)O = IDP + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.8','Aminoimidazolase','4-aminoimidazole deaminase','Fe(2+).','Forms part of the xanthine-degradation pathway in some bacteria.The product of the reaction, imidazol-4-one, can be converted non-enzymically into formiminoglycine.','4-aminoimidazole + H(2)O = imidazol-4-one + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.9','Methenyltetrahydrofolate cyclohydrolase','NULL','NULL','NULL','5,10-methenyltetrahydrofolate + H(2)O = 10-formyltetrahydrofolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.10','IMP cyclohydrolase','IMP synthetase','NULL','NULL','IMP + H(2)O = 5-formamido-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.11','Pterin deaminase','NULL','NULL','The animal enzyme is specific for pterin, isoxanthopterin andtetrahydropterin.','A 2-amino-4-hydroxypteridine + H(2)O = a 2,4-dihydroxypteridine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.12','dCMP deaminase','Deoxycytidylate deaminase','NULL','Also acts on some 5-substituted dCMPs.','dCMP + H(2)O = dUMP + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.13','dCTP deaminase','Deoxycytidine triphosphate deaminase','NULL','NULL','dCTP + H(2)O = dUTP + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.14','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.15','Guanosine deaminase','Guanosine aminase','NULL','NULL','Guanosine + H(2)O = xanthosine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.16','GTP cyclohydrolase I','NULL','NULL','The reaction involves hydrolysis of two C-N bonds and isomerizationof the pentose unit; the recyclization may be non-enzymic.Involved in the de novo synthesis of tetrahydrobiopterin from GTP,with the other enzymes involved being EC 1.1.1.153 and EC 4.2.3.12.','GTP + H(2)O = formate + 2-amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)-dihydropteridine triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.17','Adenosine-phosphate deaminase','NULL','NULL','Acts on 5-AMP, ADP, ATP, NAD(+) and adenosine, in decreasing orderof activity.The bacterial enzyme also acts on the deoxy derivatives (cf.EC 3.5.4.6).','5-AMP + H(2)O = 5-IMP + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.18','ATP deaminase','NULL','NULL','NULL','ATP + H(2)O = ITP + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.19','Phosphoribosyl-AMP cyclohydrolase','NULL','NULL','The Neurospora crassa enzyme also catalyzes the reactions ofEC 1.1.1.23 and EC 3.6.1.31.','1-(5-phospho-beta-D-ribosyl)-AMP + H(2)O = 1-(5-phospho-beta-D-ribosyl)-5-((5-phospho-beta-D-ribosylamino)methylideneamino)imidazole-4-carboxamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.20','Pyrithiamine deaminase','NULL','NULL','NULL','1-(4-amino-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium bromide + H(2)O = 1-(4-hydroxy-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium bromide + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.21','Creatinine deaminase','NULL','NULL','NULL','Creatinine + H(2)O = N-methylhydantoin + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.22','1-pyrroline-4-hydroxy-2-carboxylate deaminase','NULL','NULL','NULL','1-pyrroline-4-hydroxy-2-carboxylate + H(2)O = 2,5-dioxopentanoate +NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.23','Blasticidin-S deaminase','NULL','NULL','Catalyzes the deamination of the cytosine group of the antibioticsblasticidin S, cytomycin and acetylblasticidin S.','Blasticidin S + H(2)O = deaminohydroxyblasticidin S + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.24','Sepiapterin deaminase','NULL','NULL','Also acts on isosepiapterin, more slowly.','Sepiapterin + H(2)O = xanthopterin-B(2) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.25','GTP cyclohydrolase II','NULL','NULL','Two C-N bonds are hydrolyzed, releasing formate, with simultaneousremoval of the terminal diphosphate.','GTP + 3 H(2)O = formate + 2,5-diamino-6-hydroxy-4-(5-phospho-D-ribosylamino)pyrimidine + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.26','Diaminohydroxyphosphoribosylaminopyrimidine deaminase','NULL','NULL','The substrate is the product of EC 3.5.4.25.','2,5-diamino-6-hydroxy-4-(5-phosphoribosylamino)pyrimidine + H(2)O =5-amino-6-(5-phosphoribosylamino)uracil + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.27','Methenyltetrahydromethanopterin cyclohydrolase','Methenyl-H(4)MPT cyclohydrolase','NULL','Methanopterin is a pterin analog.Involved in the formation of methane from CO(2) in Methanobacteriumthermoautotrophicum.','5,10-methenyl-5,6,7,8-tetrahydromethanopterin + H(2)O = 5-formyl-5,6,7,8-tetrahydromethanopterin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.28','S-adenosylhomocysteine deaminase','NULL','NULL','NULL','S-adenosyl-L-homocysteine + H(2)O = S-inosyl-L-homocysteine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.29','GTP cyclohydrolase IIa','GTP cyclohydrolase III','Magnesium.','This enzyme catalyzes the hydrolysis of the imidazole ring ofguanosine 5-triphosphate, N(7)-methylguanosine 5-triphosphate orinosine 5-triphosphate.Xanthosine 5-triphosphate and ATP are not substrates.It also catalyzes the hydrolysis of diphosphate to form twoequivalents of phosphate.Unlike EC 3.5.4.25, this enzyme does not release formate, but doeshydrolyze the diphosphate from GTP to phosphate.','GTP + 3 H(2)O = 2-amino-5-formylamino-6-(5-phospho-D-ribosylamino)pyrimidin-4(3H)-one + 2 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.30','dCTP deaminase (dUMP-forming)','NULL','Magnesium.','Is highly specific for dCTP as substrate as dCMP, CTP, CDP, CMP,cytosine or deoxycytosine are not deaminated.While most bacteria require two enzymes to form dUMP from dCTP(EC 3.5.4.13 and EC 3.6.1.23), the archaeon Methanocaldococcusjannaschii uses a single enzyme to carry out both functions.This enzyme can also act as a dUTP diphosphatase, but more slowly.','dCTP + 2 H(2)O = dUMP + diphosphate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.31','S-methyl-5-thioadenosine deaminase','5-methylthioadenosine deaminase','NULL','The enzyme from Thermotoga maritima also functions asS-adenosylhomocysteine deaminase (EC 3.5.4.28) and has some activityagainst adenosine.Adenosine-5-monophosphate (AMP) and S-adenosyl-L-methionine (SAM) arenot substrates.Formerly EC 3.5.4.n1.','S-methyl-5-thioadenosine + H(2)O = S-methyl-5-thioinosine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.32','8-oxoguanine deaminase','NULL','Zinc.','8-Oxoguanine is formed via the oxidation of guanine within DNA byreactive oxygen species.If uncorrected, this modification leads to the incorporation of8-oxoG:A mismatches and eventually to G:C to T:A transversions.','8-oxoguanine + H(2)O = urate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.33','tRNA(adenine(34)) deaminase','NULL','Zinc.','The enzyme is involved in editing of tRNA.','Adenine(34) in tRNA + H(2)O = hypoxanthine(34) in tRNA + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.34','tRNA(Ala)(adenine(37)) deaminase','NULL','Magnesium.','The enzyme deaminates adenosine(37) to inosine in eukaryotictRNA(Ala).','Adenine(37) in tRNA(Ala) + H(2)O = hypoxanthine(37) in tRNA(Ala) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.35','tRNA(cytosine(8)) deaminase','NULL','NULL','The enzyme from Methanopyrus kandleri specifically catalyzes thedeamimation of cytosine at poition 8 of tRNA in 30 different tRNAs.This cytosine-to-uracil editing guarantees the proper folding andfunctionality of the tRNAs.','Cytosine(8) in tRNA + H(2)O = uracil(8) in tRNA + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.36','mRNA(cytosine(6666)) deaminase','Apolipoprotein B mRNA editing enzyme','NULL','The apolipoprotein B mRNA editing enzyme complex catalyzes theediting of apolipoprotein B mRNA at cytidine(6666) to uridine,thereby transforming the codon for glutamine-2153 to a terminationcodon.Editing results in translation of a truncated apolipoprotein Bisoform (apoB-48) with distinct functions in lipid transport.The catalytic component (APOBEC-1) contains zinc at the active site.','Cytosine(6666) in apolipoprotein B mRNA + H(2)O = uracil(6666) inapolipoprotein B mRNA + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.37','Double-stranded RNA adenine deaminase','Double-stranded RNA adenosine deaminase','NULL','This eukaryotic enzyme is involved in RNA editing.It destabilizes double-stranded RNA through conversion of adenosineto inosine.Inositol hexakisphosphate is required for activity.','Adenine in double-stranded RNA + H(2)O = hypoxanthine in double-strandedRNA + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.38','Single-stranded DNA cytosine deaminase','Activation-induced cytidine deaminase','NULL','The enzyme exclusively catalyzes deamination of cytosine in single-stranded DNA.It preferentially deaminates five-nucleotide bubbles.The optimal target consists of a single-stranded NWRCN motif (W =A or T, R = A or G).The enzyme initiates antibody diversification processes bydeaminating immunoglobulin sequences.','Cytosine in single-stranded DNA + H(2)O = uracil in single-stranded DNA +NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.39','GTP cyclohydrolase IV','NULL','Fe(2+).','The enzyme is involved in methanopterin biosynthesis in methanogenicarchaea.Cf. EC 3.5.4.16, EC 3.5.4.25 and EC 3.5.4.29.Formerly EC 3.5.4.n2.','GTP + H(2)O = formate + 7,8-dihydro-D-neopterin 2,3-cyclic phosphate +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.40','Aminodeoxyfutalosine deaminase','AFL deaminase','NULL','The enzyme, found in several bacterial species, is part of thefutalosine pathway for menaquinone biosynthesis.','6-amino-6-deoxyfutalosine + H(2)O = futalosine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.4.n2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.5.1','Nitrilase','NULL','NULL','Acts on a wide range of aromatic nitriles including (indol-3-yl)acetonitrile, and also on some aliphatic nitriles, and on thecorresponding acid amides (cf. EC 4.2.1.84).','A nitrile + 2 H(2)O = a carboxylate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.5.2','Ricinine nitrilase','NULL','NULL','NULL','Ricinine + 2 H(2)O = 3-carboxy-4-methoxy-N-methyl-2-pyridone + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.5.3','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.5.4','Cyanoalanine nitrilase','NULL','NULL','L-asparagine is formed as an intermediate.Cf. EC 4.2.1.65 and EC 3.5.1.1.','3-cyano-L-alanine + 2 H(2)O = L-aspartate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.5.5','Arylacetonitrilase','NULL','NULL','Requires thiol compounds.Also hydrolyzes other 4-substituted phenylacetonitriles, thien-2-ylacetonitrile, tolylacetonitriles, and, more slowly, benzyl cyanide.','4-chlorophenylacetonitrile + 2 H(2)O = 4-chlorophenylacetate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.5.6','Bromoxynil nitrilase','NULL','NULL','Involved in the bacterial degradation of the herbicide bromoxynil.Highly specific.','3,5-dibromo-4-hydroxybenzonitrile + 2 H(2)O = 3,5-dibromo-4-hydroxy-benzoate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.5.7','Aliphatic nitrilase','NULL','NULL','Preferentially hydrolyzes aliphatic nitriles, some of which areapparently not substrates for other known EC 3.5.5.1.Substrates include crotononitrile, acrylonitrile and glutaronitrile.','R-CN + 2 H(2)O = R-COOH + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.5.8','Thiocyanate hydrolase','NULL','NULL','The enzyme from Thiobacillus thioparus catalyzes the first step inthe degradation of thiocyanate.','Thiocyanate + 2 H(2)O = carbonyl sulfide + NH(3) + HO(-).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.1','Riboflavinase','NULL','NULL','NULL','Riboflavin + H(2)O = ribitol + lumichrome.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.2','Aminopyrimidine aminohydrolase','Thiaminase','NULL','Previously known as thiaminase II, this enzyme is involved in theregeneration of the thiamine pyrimidine from degraded products,rather than in thiamine degradation, and participates in thiaminesalvage pathways.','(1) 4-amino-5-aminomethyl-2-methylpyrimidine + H(2)O = 4-amino-5-hydroxymethyl-2-methylpyrimidine + ammonia.(2) Thiamine + H(2)O = 4-amino-5-hydroxymethyl-2-methylpyrimidine +5-(2-hydroxyethyl)-4-methylthiazole.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.3','Hydroxydechloroatrazine ethylaminohydrolase','Hydroxyatrazine ethylaminohydrolase','NULL','Involved in a pathway by which the herbicide atrazine, 2-chloro-4-(ethylamino)-6-(isopropylamino)-1,3,5-triazine, is degraded inbacteria via N-isopropylammelide, 2,4-dihydroxy-6-(isopropylamino)-1,3,5-triazine.','4-(ethylamino)-2-hydroxy-6-(isopropylamino)-1,3,5-triazine + H(2)O =N-isopropylammelide + ethylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.4','N-isopropylammelide isopropylaminohydrolase','NULL','NULL','Involved in a pathway by which the herbicide atrazine, 2-chloro-4-(ethylamino)-6-(isopopylamino)-1,3,5-triazine, is degraded inbacteria via N-isopropylammelide, 2,4-dihydroxy-6-(isopropylamino)-1,3,5-triazine.','N-isopropylammelide + H(2)O = cyanuric acid + isopropylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.5','2-aminomuconate deaminase','NULL','NULL','2-aminomuconate is an intermediate in the bacterial biodegradation ofnitrobenzene.The enzyme has been isolated from several species, includingPseudomonas pseudocaligenes JS45, Pseudomonas fluorescens KU-7,Pseudomonas sp. AP3 and Burkholderia cenocepacia J2315.The reaction is spontaneous in acid conditions.','2-aminomuconate + H(2)O = (3E)-2-oxohex-3-enedioate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.6','Glucosamine-6-phosphate deaminase','GlcN6P deaminase','NULL','Isomerization of the aldose-ketose type converts the -CH(-NH(2))-CH=Ogroup of glucosamine 6-phosphate into -C(=NH)-CH(2)-OH, forming2-deoxy-2-imino-D-arabino-hexitol which then hydrolyzes to yieldfructose 6-phosphate and ammonia.N-acetyl-D-glucosamine 6-phosphate, which is not broken down,activates the enzyme.Formerly EC 5.3.1.10.','Alpha-D-glucosamine 6-phosphate + H(2)O = D-fructose 6-phosphate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.7','1-aminocyclopropane-1-carboxylate deaminase','1-aminocyclopropane carboxylic acid deaminase','Pyridoxal 5-phosphate.','Its introduction has been used to make fruit ripening dependent onexternally added ethylene, as it removes the substrate for endogenousethylene formation.Formerly EC 4.1.99.4.','1-aminocyclopropane-1-carboxylate + H(2)O = 2-oxobutanoate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.8','5-nitroanthranilic acid aminohydrolase','5NAA deaminase','NULL','The enzyme catalyzes the initial step in biodegradation of5-nitroanthranilic acid by Bradyrhizobium sp. strain JS329.','5-nitroanthranilate + H(2)O = 5-nitrosalicylate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.9','2-nitroimidazole nitrohydrolase','2NI denitrase','NULL','The enzyme catalyzes the initial step in the biodegradation of2-nitroimidazole by the soil bacterium Mycobacterium sp. JS330.','2-nitroimidazole + H(2)O = imidazol-2-one + nitrite.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.5.99.10','2-iminobutanoate/2-iminopropanoate deaminase','Enamine/imine deaminase','NULL','This enzyme, which has been found in all species and tissuesexamined, catalyzes the hydrolytic deamination of imine intermediatesformed by several types of pyridoxal-5-phosphate-dependentdehydratases, such as EC 4.3.1.19 and EC 4.3.1.17.The reactions, which can occur spontaneously, are accelerated tominimize the cellular damage that could be caused by these reactiveintermediates.','(1) 2-iminobutanoate + H(2)O = 2-oxobutanoate + NH(3).(2) 2-iminopropanoate + H(2)O = pyruvate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.1','Inorganic diphosphatase','Diphosphate phosphohydrolase','NULL','Specificity varies with the source and with the activating metal ion.The enzyme from some sources may be identical with EC 3.1.3.1 orEC 3.1.3.9.','Diphosphate + H(2)O = 2 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.2','Trimetaphosphatase','NULL','NULL','NULL','Trimetaphosphate + H(2)O = triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.3','Adenosinetriphosphatase','Adenylpyrophosphatase','NULL','Many enzymes previously listed under this number are now listedseparately as EC 3.6.1.32 to EC 3.6.1.39.The remaining enzymes, not separately listed on the basis of somefunction coupled with hydrolyzes of ATP, include enzymes dependent onCa(2+), Mg(2+), anions, H(+) or DNA.Formerly EC 3.6.1.4.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.5','Apyrase','Adenosine diphosphatase','Calcium.','Apyrases are active against both di- and triphosphate nucleotides(NDPs and NTPs) and hydrolyze NTPs to nucleotide monophosphates(NMPs) in two distinct successive phosphate-releasing steps, withNDPs as intermediates.They differ from ATPases, which specifically hydrolyze ATP,by hydrolyzing both ATP and ADP.Most of the ecto-ATPases that occur on the cell surface and hydrolyzeextracellular nucleotides belong to this enzyme family.','A nucleoside 5-triphosphate + 2 H(2)O = a nucleoside 5-phosphate +2 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.6','Nucleoside diphosphate phosphatase','Nucleoside-diphosphatase','NULL','The enzyme, which appears to be limited to metazoa, acts on multiplenucleoside diphosphates as well as on D-ribose 5-diphosphate.Specificity depends on species and isoform.','A nucleoside diphosphate + H(2)O = a nucleoside phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.7','Acylphosphatase','Acylphosphate phosphohydrolase','NULL','NULL','An acylphosphate + H(2)O = a carboxylate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.8','ATP diphosphatase','ATP pyrophosphatase','NULL','Also acts on ITP, GTP, CTP and UTP.','ATP + H(2)O = AMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.9','Nucleotide diphosphatase','Nucleotide pyrophosphatase','NULL','Substrates include NAD(+), NADP(+), FAD, CoA and also ATP and ADP.','A dinucleotide + H(2)O = 2 mononucleotides.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.10','Endopolyphosphatase','Metaphosphatase','NULL','The product contains 4 or 5 phosphate residues.','Polyphosphate + n H(2)O = (n+1) oligophosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.11','Exopolyphosphatase','Exopolypase','NULL','NULL','(Polyphosphate)(n) + H(2)O = (polyphosphate)(n-1) + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.12','dCTP diphosphatase','dCTP pyrophosphatase','NULL','Also hydrolyzes dCDP to dCMP and phosphate.','dCTP + H(2)O = dCMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.13','ADP-ribose diphosphatase','Adenosine diphosphoribose pyrophosphatase','NULL','NULL','ADP-D-ribose + H(2)O = AMP + D-ribose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.14','Adenosine-tetraphosphatase','NULL','NULL','Also acts on inosine tetraphosphate and tripolyphosphate but showslittle or no activity with other nucleotides or polyphosphates.','Adenosine 5-tetraphosphate + H(2)O = ATP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.15','Nucleoside-triphosphate phosphatase','NTPase','NULL','The enzyme is found in eukaryotes and thermophilic bacteria,but appears to be absent from mesophilic bacteria.Also hydrolyzes nucleoside diphosphates, thiamine diphosphate andFAD.The enzyme from the plant Pisum sativum (garden pea) is regulated bycalmodulin.','NTP + H(2)O = NDP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.16','CDP-glycerol diphosphatase','CDP-glycerol pyrophosphatase','NULL','NULL','CDP-glycerol + H(2)O = CMP + sn-glycerol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.17','Bis(5-nucleosyl)-tetraphosphatase (asymmetrical)','Ap(4)A hydrolase','NULL','Also acts on bis(5-xanthosyl)-tetraphosphate and, more slowly,on bis(5-adenosyl)-tetraphosphate and bis(5-uridyl)-tetraphosphate(cf. EC 3.6.1.41).','P(1),P(4)-bis(5-guanosyl) tetraphosphate + H(2)O = GTP + GMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.18','FAD diphosphatase','FAD pyrophosphatase','NULL','The plant enzyme also hydrolyzes NAD(+) and NADH; the animal enzymehydrolyzes NAD(+) and CoA at about half of the rate of hydrolysis ofFAD.May be identical with EC 3.6.1.9.','FAD + H(2)O = AMP + FMN.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.19','Nucleoside-triphosphate diphosphatase','Nucleoside-triphosphate pyrophosphatase','NULL','May be identical with EC 3.6.1.9.','A nucleoside triphosphate + H(2)O = a nucleotide + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.20','5-acylphosphoadenosine hydrolase','NULL','NULL','Also acts on inosine and uridine compounds.','5-acylphosphoadenosine + H(2)O = AMP + a carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.21','ADP-sugar diphosphatase','ADP-sugar pyrophosphatase','NULL','Has a distinct specificity from EC 3.6.1.45.','ADP-sugar + H(2)O = AMP + alpha-D-aldose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.22','NAD(+) diphosphatase','NAD(+) pyrophosphatase','NULL','Also acts on NADP(+), 3-acetylpyridine and the thionicotinamideanalogs of NAD(+) and NADP(+).','NAD(+) + H(2)O = AMP + NMN.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.23','dUTP diphosphatase','Deoxyuridine-triphosphatase','NULL','NULL','dUTP + H(2)O = dUMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.24','Nucleoside phosphoacylhydrolase','NULL','NULL','Attacks ribonucleoside 5-nitrophenylphosphates, but is inactiveagainst phosphodiesters.','Hydrolyzes mixed phospho-anhydride bonds.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.25','Triphosphatase','NULL','NULL','NULL','Triphosphate + H(2)O = diphosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.26','CDP-diacylglycerol diphosphatase','CDP-diacylglycerol phosphatidylhydrolase','NULL','NULL','CDP-diacylglycerol + H(2)O = CMP + phosphatidate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.27','Undecaprenyl-diphosphate phosphatase','C(55)-isoprenyl diphosphatase','NULL','Isolated from the bacteria Micrococcus lysodeikticus, Escherichiacoli and Bacillus subtilis.The product of the reaction, ditrans,octacis-undecaprenyl phosphate,is essential for cell wall polysaccharide biosynthesis in thesestrains.','Ditrans,octacis-undecaprenyl diphosphate + H(2)O = ditrans,octacis-undecaprenyl phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.28','Thiamine-triphosphatase','ThTPase','NULL','NULL','Thiamine triphosphate + H(2)O = thiamine diphosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.29','Bis(5-adenosyl)-triphosphatase','AP(3)A hydrolase','NULL','NULL','P(1)-P(3)-bis(5-adenosyl) triphosphate + H(2)O = ADP + AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.30','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.31','Phosphoribosyl-ATP diphosphatase','Phosphoribosyl-ATP pyrophosphatase','NULL','The Neurospora crassa enzyme also catalyzes the reactions ofEC 1.1.1.23 and EC 3.5.4.19.','1-(5-phospho-beta-D-ribosyl)-ATP + H(2)O = 1-(5-phospho-beta-D-ribosyl)-AMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.32','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.33','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.34','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.35','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.36','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.37','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.38','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.39','Thymidine-triphosphatase','NULL','NULL','Also acts, more slowly, on dUTP and UTP.','dTTP + H(2)O = dTDP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.40','Guanosine-5-triphosphate,3-diphosphate diphosphatase','Guanosine pentaphosphate phosphohydrolase','NULL','Also hydrolyzes other guanosine 5-triphosphate derivatives with atleast one unsubstituted phosphate group on the 3-position, but notGTP, ATP or adenosine 5-triphosphate,3-diphosphate.','Guanosine 5-triphosphate,3-diphosphate + H(2)O = guanosine3,5-bis(diphosphate) + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.41','Bis(5-nucleosyl)-tetraphosphatase (symmetrical)','Ap(4)A hydrolase','NULL','Also acts on bis(5-guanosyl) tetraphosphate and bis(5-adenosyl)pentaphosphate, and, more slowly, on some other polyphosphates,forming a nucleoside bisphosphate as one product in all cases (cf.EC 3.6.1.17).','P(1),P(4)-bis(5-adenosyl) tetraphosphate + H(2)O = 2 ADP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.42','Guanosine-diphosphatase','GDPase','NULL','Also acts on UDP but not on other nucleoside diphosphates andtriphosphates.','GDP + H(2)O = GMP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.43','Dolichyldiphosphatase','NULL','NULL','NULL','Dolichyl diphosphate + H(2)O = dolichyl phosphate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.44','Oligosaccharide-diphosphodolichol diphosphatase','Oligosaccharide-diphosphodolichol pyrophosphatase','NULL','NULL','Oligosaccharide-diphosphodolichol + H(2)O = oligosaccharide phosphate +dolichyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.45','UDP-sugar diphosphatase','Nucleosidediphosphate-sugar diphosphatase','Divalent cation.','UDP-sugar is the best substrate, although other nucleoside-sugardiphosphates are used as substrates with similar Km values but muchlower maximum velocities.Thus, this enzyme has a specificity distinct from that ofEC 3.6.1.21.Some but not all enzymes of this class also appear to have EC 3.1.3.5activity.','UDP-sugar + H(2)O = UMP + alpha-D-aldose 1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.46','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.47','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.48','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.49','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.50','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.51','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.52','Diphosphoinositol-polyphosphate diphosphatase','Diphosphoinositol-polyphosphate phosphohydrolase','NULL','This enzyme hydrolyzes the diphosphate bond, leaving a phospho groupwhere a diphospho group had been.It can also act on bis(adenosine) diphosphate.','Diphospho-myo-inositol polyphosphate + H(2)O = myo-inositol polyphosphate+ phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.53','Mn(2+)-dependent ADP-ribose/CDP-alcohol diphosphatase','ADPRibase-Mn','Manganese.','ADP-D-ribose, CDP-choline, CDP-ethanolamine and ADP are substratesfor this enzyme but ADP-D-glucose, UDP-D-glucose, CDP-D-glucose, CDP,CMP and AMP are not hydrolyzed.','(1) CDP-choline + H(2)O = CMP + phosphocholine.(2) ADP-D-ribose + H(2)O = AMP + D-ribose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.54','UDP-2,3-diacylglucosamine diphosphatase','UDP-2,3-bis(3-hydroxytetradecanoyl)glucosamine diphosphatase','NULL','The enzyme catalyzes a step in the biosynthesis of lipid A.Formerly EC 3.6.1.n5.','UDP-2,3-bis((3R)-3-hydroxymyristoyl)-alpha-D-glucosamine + H(2)O = 2,3-bis((3R)-3-hydroxymyristoyl)-beta-D-glucosaminyl 1-phosphate + UMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.55','8-oxo-dGTP diphosphatase','7,8-dihydro-8-oxo-dGTP pyrophosphohydrolase','Magnesium.','This enzyme hydrolyzes the phosphoanhydride bond between the alphaand beta phosphate of 8-oxoguanine-containing nucleoside di- andtriphosphates thereby preventing misincorporation of the oxidizedpurine nucleoside triphosphates into DNA.It does not hydrolyze 2-hydroxy-dATP (cf. EC 3.6.1.56).','8-oxo-dGTP + H(2)O = 8-oxo-dGMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.56','2-hydroxy-dATP diphosphatase','(2-deoxy) ribonucleoside 5-triphosphate pyrophosphohydrolase','NULL','The enzyme hydrolyzes oxidized purine nucleoside triphosphates suchas 2-hydroxy-dATP, thereby preventing their misincorporation intoDNA.It can also recognize 8-oxo-dGTP and 8-oxo-dATP, but with lowerefficiency (cf. EC 3.6.1.55).','2-hydroxy-dATP + H(2)O = 2-hydroxy-dAMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.57','UDP-2,4-diacetamido-2,4,6-trideoxy-beta-L-altropyranose hydrolase','UDP-6-deoxy-AltdiNAc hydrolase','NULL','The enzyme is involved in biosynthesis of pseudaminic acid.','UDP-2,4-diacetamido-2,4,6-trideoxy-beta-L-altropyranose + H(2)O = 2,4-diacetamido-2,4,6-trideoxy-beta-L-altropyranose + UDP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.58','8-oxo-dGDP phosphatase','NULL','NULL','The enzyme catalyzes the hydrolysis of both 8-oxo-dGDP and 8-oxo-GDPthereby preventing translational errors caused by oxidative damage.The preferred in vivo substrate is not known.The enzyme does not degrade 8-oxo-dGTP and 8-oxo-GTP to themonophosphates (cf. EC 3.6.1.55).Ribonucleotide diphosphates and deoxyribonucleotide diphosphates arehydrolyzed with broad specificity.The bifunctional enzyme NUDT5 also hydrolyzes ADP-ribose to AMP andD-ribose 5-phosphate (cf. EC 3.6.1.13).The human enzyme NUDT18 also hydrolyzes 8-oxo-dADP and 2-hydroxy-dADP, the latter at a slower rate.','8-oxo-dGDP + H(2)O = 8-oxo-dGMP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.59','5-(N(7)-methyl 5-triphosphoguanosine)-(mRNA) diphosphatase','M(7)GpppN m(7)GMP phosphohydrolase','NULL','The enzyme removes (decaps) the N(7)-methylguanosine 5-phosphate capfrom an mRNA degraded to a maximal length of 10 nucleotides.Decapping is an important process in the control of eukaryotic mRNAdegradation.The enzyme functions to clear the cell of cap structure followingdecay of the RNA body.The nematode enzyme can also decap triply methylated substrates,5-(N(2),N(2),N(7)-trimethyl 5-triphosphoguanosine)-(mRNA).Formerly EC 3.6.1.30.','A 5-(N(7)-methyl 5-triphosphoguanosine)-(mRNA) + H(2)O = N(7)-methylguanosine 5-phosphate + a 5-diphospho-(mRNA).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.60','Diadenosine hexaphosphate hydrolase (AMP-forming)','NULL','NULL','A divalent cation is essential for activity.Mn(2+) (2-6 mM) is most effective.The enzyme controls intracellular levels of P(1),P(5)-bis(5-adenosyl)pentaphosphate and P(1),P(6)-bis(5-adenosyl)hexaphosphate.Weak activity with P(1),P(4)-bis(5-adenosyl)tetraphosphate.Marked preference for adenine over guanine nucleotides.','(1) P(1),P(6)-bis(5-adenosyl)hexaphosphate + H(2)O = adenosine5-pentaphosphate + AMP.(2) P(1),P(5)-bis(5-adenosyl)pentaphosphate + H(2)O = adenosine5-tetraphosphate + AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.61','Diadenosine hexaphosphate hydrolase (ATP-forming)','NULL','Divalent cation.','The enzyme hydrolyzes P(1),P(4)-bis(5-guanosyl) tetraphosphate veryslowly (cf. EC 3.6.1.17).','(1) P(1),P(6)-bis(5-adenosyl)hexaphosphate + H(2)O = 2 ATP.(2) P(1),P(5)-bis(5-adenosyl)pentaphosphate + H(2)O = ATP + ADP.(3) P(1),P(4)-bis(5-adenosyl)tetraphosphate + H(2)O = ATP + AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.62','5-(N(7)-methylguanosine 5-triphospho)-(mRNA) hydrolase','M(7)GpppN-mRNA hydrolase','NULL','Decapping of mRNA is a critical step in eukaryotic mRNA turnover.The enzyme is unable to cleave a free cap structure (m(7)GpppG).The enzyme from Vaccinia virus is synergistically activated in thepresence of Mg(2+) and Mn(2+).Formerly EC 3.6.1.30.','5-(N(7)-methylguanosine 5-triphospho)-(mRNA) + H(2)O = N(7)-methylguanosine 5-diphosphate + 5-phospho-(mRNA).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.63','Alpha-D-ribose 1-methylphosphonate 5-triphosphate diphosphatase','NULL','NULL','Isolated from the bacterium Escherichia coli.','Alpha-D-ribose 1-methylphosphonate 5-triphosphate + H(2)O = alpha-D-ribose 1-methylphosphonate 5-phosphate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.64','Inosine diphosphate phosphatase','(Deoxy)inosine diphosphatase','NULL','The human enzyme also hydrolyzes GDP and dGDP, and to a lesser extentITP, dITP and XTP.','(1) IDP + H(2)O = IMP + phosphate.(2) dIDP + H(2)O = dIMP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.65','(d)CTP diphosphatase','(d)CTP diphosphohydrolase','NULL','The enzyme, characterized from the bacterium Escherichia coli,is specific for the pyrimidine nucleotides CTP and dCTP.It also acts on 5-methyl-dCTP, 5-hydroxy-dCTP and 8-hydroxy-dGTP.','(1) CTP + H(2)O = CMP + diphosphate.(2) dCTP + H(2)O = dCMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.66','XTP/dITP diphosphatase','Hypoxanthine/xanthine dNTP pyrophosphatase','Magnesium.','The enzymes from the bacterium Escherichia coli and the archaeaMethanococcus jannaschii and Archaeoglobus fulgidus are highlyspecific for XTP and dITP.','(1) XTP + H(2)O = XMP + diphosphate.(2) dITP + H(2)O = dIMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.n1','D-tyrosyl-tRNA(Tyr) hydrolase','D-tyrosyl-tRNA(Tyr) deacylase','NULL','NULL','D-tyrosyl-tRNA(Tyr) + H(2)O = tRNA(Tyr) + D-tyrosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.n2','L-cysteinyl-tRNA(Pro)','Cys-tRNA(Pro) deacylase','NULL','The enzyme from Escherichia coli also catalyzes the reaction ofEC 3.6.1.n3.','L-cysteinyl-tRNA(Pro) + H(2)O = tRNA(Pro) + L-cysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.n3','L-cysteinyl-tRNA(Cys) hydrolase','Cys-tRNA(Cys) deacylase','NULL','The enzyme from Escherichia coli also catalyzes the reaction ofEC 3.6.1.n2.','L-cysteinyl-tRNA(Cys) + H(2)O = tRNA(Cys) + L-cysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.n4','Dihydroneopterin triphosphate diphosphatase','Dihydroneopterin triphosphate pyrophosphatase','Magnesium.','Part of the folate biosynthesis pathway in bacteria, together withEC 4.1.2.25, EC 2.7.6.3, EC 2.5.1.15 and EC 6.3.2.12.Member of the NUDIX family of diphosphatases.','7,8-dihydroneopterin 3-triphosphate + H(2)O = 7,8-dihydroneopterin3-phosphate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.1.n5','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.2.1','Adenylylsulfatase','NULL','NULL','NULL','Adenylyl sulfate + H(2)O = AMP + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.2.2','Phosphoadenylylsulfatase','3-phosphoadenylyl sulfatase','Manganese.','NULL','3-phosphoadenylyl sulfate + H(2)O = adenosine 3,5-bisphosphate +sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.1','Phospholipid-translocating ATPase','Flippase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.The enzyme apparently has several activities, one of them being themovement of phospholipids from one membrane face to the other.Formerly EC 3.6.3.13.','ATP + H(2)O + phospholipid(Side 1) = ADP + phosphate + phospholipid(Side2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.2','Magnesium-importing ATPase','Magnesium-translocating P-type ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.','ATP + H(2)O + Mg(2+)(Out) = ADP + phosphate + Mg(2+)(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.3','Cadmium-exporting ATPase','Cadmium-translocating P-type ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.','ATP + H(2)O + Cd(2+)(In) = ADP + phosphate + Cd(2+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.4','Cu(2+)-exporting ATPase','NULL','Magnesium.','A P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.The enzyme from the termophilic bacterium Archaeoglobus fulgidus isinvolved in copper extrusion from the cell.','ATP + H(2)O + Cu(2+)(Side 1) = ADP + phosphate + Cu(2+)(Side 2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.5','Zinc-exporting ATPase','P(1B)-type ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.This enzyme also exports Cd(2+) and Co(2+).','ATP + H(2)O + Zn(2+)(In) = ADP + phosphate + Zn(2+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.6','Proton-exporting ATPase','H(+)-exporting ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.Generates an electrochemical potential gradient of protons across theplasma membrane.Formerly EC 3.6.1.35.','ATP + H(2)O + H(+)(In) = ADP + phosphate + H(+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.7','Sodium-exporting ATPase','Na(+)-exporting ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.Involved in the efflux of Na(+), with one ion being exported per ATPhydrolyzed.','ATP + H(2)O + Na(+)(In) = ADP + phosphate + Na(+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.8','Calcium-transporting ATPase','Ca(2+)-transporting ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.This enzyme family comprises three types of Ca(2+)-transportingenzymes that are found in the plasma membrane, the sarcoplasmicreticulum and in Saccharomyces cerevisiae.The first and third transport one ion per ATP hydrolyzed, whereas thesecond transports two ions.Formerly EC 3.6.1.38.','ATP + H(2)O + Ca(2+)(Side 1) = ADP + phosphate + Ca(2+)(Side 2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.9','Sodium/potassium-exchanging ATPase','Na(+)/K(+)-ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.An enzyme from the plasma membrane of animal cells that catalyzes theefflux of three Na(+) and influx of two K(+) per ATP hydrolyzed.It is involved in generating the plasma membrane electricalpotential.Formerly EC 3.6.1.37.','ATP + H(2)O + Na(+)(In) + K(+)(Out) = ADP + phosphate + Na(+)(Out) +K(+)(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.10','Hydrogen/potassium-exchanging ATPase','Gastric H(+)/K(+) ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.Hydrolysis of ATP is coupled with the exchange of H(+) and K(+) ions.Formerly EC 3.6.1.36.','ATP + H(2)O + H(+)(In) + K(+)(Out) = ADP + phosphate + H(+)(Out) +K(+)(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.11','Chloride-transporting ATPase','Chloride-motive ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.','ATP + H(2)O + Cl(-)(Out) = ADP + phosphate + Cl(-)(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.12','Potassium-transporting ATPase','K(+)-importing ATPase','Magnesium.','P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.A bacterial enzyme of di(heterotetrameric) structure that is involvedin K(+) import.The probable stoichiometry is one ion per ATP hydrolyzed.','ATP + H(2)O + K(+)(Out) = ADP + phosphate + K(+)(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.13','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.14','H(+)-transporting two-sector ATPase','ATP synthase','NULL','A multisubunit non-phosphorylated ATPase that is involved in thetransport of ions.Large enzymes of mitochondria, chloroplasts and bacteria with amembrane sector (F(o), V(o), A(o)) and a cytoplasmic-compartmentsector (F(1), V(1), A(1)).The F-type enzymes of the inner mitochondrial and thylakoid membranesact as ATP synthases.All of the enzymes included here operate in a rotational mode, wherethe extramembrane sector (containing 3 alpha- and 3 beta-subunits) isconnected via the delta-subunit to the membrane sector by severalsmaller subunits.Within this complex, the gamma- and epsilon-subunits, as well as the9-12 c subunits rotate by consecutive 120 degree angles and performparts of ATP synthesis.This movement is driven by the H(+) electrochemical potentialgradient.The V-type (in vacuoles and clathrin-coated vesicles) and A-type(archaeal) enzymes have a similar structure but, under physiologicalconditions, they pump H(+) rather than synthesize ATP.Formerly EC 3.6.1.34.','ATP + H(2)O + H(+)(In) = ADP + phosphate + H(+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.15','Sodium-transporting two-sector ATPase','ATP synthase, sodium ion specific','NULL','A multisubunit non-phosphorylated ATPase that is involved in thetransport of ions.An enzyme found in alkaliphilic bacteria that is similar toEC 3.6.3.14.','ATP + H(2)O + Na(+)(In) = ADP + phosphate + Na(+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.16','Arsenite-transporting ATPase','Arsenical pump-driving ATPase','NULL','A bacterial multisubunit non-phosphorylated ATPase that is involvedin the export of arsenite and antimonite anions.','ATP + H(2)O + arsenite(In) = ADP + phosphate + arsenite(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.17','Monosaccharide-transporting ATPase','NULL','NULL','Family of bacterial enzymes importing ribose, xylose, arabinose,galactose and methylgalactoside.','ATP + H(2)O + monosaccharide(Out) = ADP + phosphate + monosaccharide(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.18','Oligosaccharide-transporting ATPase','NULL','NULL','A bacterial enzyme that imports lactose, melibiose and raffinose.','ATP + H(2)O + oligosaccharide(Out) = ADP + phosphate +oligosaccharide(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.19','Maltose-transporting ATPase','NULL','NULL','Comprises bacterial enzymes that import maltose and maltoseoligosaccharides.','ATP + H(2)O + maltose(Out) = ADP + phosphate + maltose(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.20','Glycerol-3-phosphate-transporting ATPase','NULL','NULL','A bacterial enzyme that imports phosphorylated glycerol.','ATP + H(2)O + glycerol-3-phosphate(Out) = ADP + phosphate + glycerol-3-phosphate(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.21','Polar-amino-acid-transporting ATPase','Histidine permease','NULL','Comprises bacterial enzymes that import His, Arg, Lys, Glu, Gln, Asp,ornithine, octopine and nopaline.','ATP + H(2)O + polar amino acid(Out) = ADP + phosphate + polar aminoacid(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.22','Nonpolar-amino-acid-transporting ATPase','NULL','NULL','Comprises bacterial enzymes that import Leu, Ile and Val.','ATP + H(2)O + nonpolar amino acid(Out) = ADP + phosphate + nonpolar aminoacid(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.23','Oligopeptide-transporting ATPase','Oligopeptide permease','NULL','A bacterial enzyme that imports di- and oligopeptides.','ATP + H(2)O + oligopeptide(Out) = ADP + phosphate + oligopeptide(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.24','Nickel-transporting ATPase','NULL','NULL','A bacterial enzyme that imports Ni(2+).','ATP + H(2)O + Ni(2+)(Out) = ADP + phosphate + Ni(2+)(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.25','Sulfate-transporting ATPase','NULL','NULL','A bacterial enzyme that imports sulfate and thiosulfate anions.','ATP + H(2)O + sulfate(Out) = ADP + phosphate + sulfate(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.26','Nitrate-transporting ATPase','NULL','NULL','A bacterial enzyme that imports NO(3)(-), NO(2)(-) and OCN(-).','ATP + H(2)O + nitrate(Out) = ADP + phosphate + nitrate(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.27','Phosphate-transporting ATPase','ABC phosphate transporter','NULL','A bacterial enzyme that imports phosphate anions.','ATP + H(2)O + phosphate(Out) = ADP + phosphate + phosphate(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.28','Phosphonate-transporting ATPase','NULL','NULL','A bacterial enzyme that imports phosphonate and organophosphateanions.','ATP + H(2)O + phosphonate(Out) = ADP + phosphate + phosphonate(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.29','Molybdate-transporting ATPase','NULL','NULL','A bacterial enzyme that imports molybdate anions.','ATP + H(2)O + molybdate(Out) = ADP + phosphate + molybdate(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.30','Fe(3+)-transporting ATPase','NULL','NULL','A bacterial enzyme that imports ferric cations.','ATP + H(2)O + Fe(3+)(Out) = ADP + phosphate + Fe(3+)(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.31','Polyamine-transporting ATPase','NULL','NULL','A bacterial enzyme that imports putrescine and spermidine.','ATP + H(2)O + polyamine(Out) = ADP + phosphate + polyamine(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.32','Quaternary-amine-transporting ATPase','NULL','NULL','A bacterial enzyme that imports betaine and glycine.','ATP + H(2)O + quaternary amine(Out) = ADP + phosphate + quaternaryamine(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.33','Vitamin B12-transporting ATPase','NULL','NULL','A bacterial enzyme that imports cobalamin derivatives.','ATP + H(2)O + vitamin B12(Out) = ADP + phosphate + vitamin B12(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.34','Iron-chelate-transporting ATPase','NULL','NULL','A bacterial enzyme that imports Fe-enterobactin, Fe-dicitrate,Fe-hydroxamate and other siderophores.','ATP + H(2)O + iron chelate(Out) = ADP + phosphate + iron chelate(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.35','Manganese-transporting ATPase','ABC-type manganese permease complex','NULL','A bacterial enzyme that imports Mn(2+), Zn(2+) and iron chelates.','ATP + H(2)O + Mn(2+)(Out) = ADP + phosphate + Mn(2+)(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.36','Taurine-transporting ATPase','NULL','NULL','A bacterial enzyme that imports taurine.','ATP + H(2)O + taurine(Out) = ADP + phosphate + taurine(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.37','Guanine-transporting ATPase','NULL','NULL','A eukaryotic enzyme that imports guanine and tryptophan (it containsa single ATP-binding site).','ATP + H(2)O + guanine(Out) = ADP + phosphate + guanine(In).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.38','Capsular-polysaccharide-transporting ATPase','NULL','NULL','An enzyme that exports capsular polysaccharide from Gram-negativebacteria.','ATP + H(2)O + capsular polysaccharide(In) = ADP + phosphate + capsularpolysaccharide(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.39','Lipopolysaccharide-transporting ATPase','NULL','NULL','Enzymes of Gram-negative bacteria that export lipo-oligosaccharidesand lipopolysaccharides.','ATP + H(2)O + lipopolysaccharide(In) = ADP + phosphate +lipopolysaccharide(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.40','Teichoic-acid-transporting ATPase','NULL','NULL','An enzyme found in Gram-positive bacteria that exports teichoic acid.','ATP + H(2)O + teichoic acid(In) = ADP + phosphate + teichoic acid(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.41','Heme-transporting ATPase','NULL','NULL','An enzyme found in Gram-negative bacteria that exports heme.','ATP + H(2)O + heme(In) = ADP + phosphate + heme(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.42','Beta-glucan-transporting ATPase','NULL','NULL','An enzyme found in Gram-negative bacteria that exports beta-glucan.','ATP + H(2)O + beta-glucan(In) = ADP + phosphate + beta-glucan(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.43','Peptide-transporting ATPase','NULL','NULL','A family of enzymes that exports alpha-hemolysin, cyclolysin, colicinV and siderophores from Gram-negative bacteria, and bacteriocin,subtilin, competence factor and pediocin from Gram-positive bacteria.','ATP + H(2)O + peptide(In) = ADP + phosphate + peptide(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.44','Xenobiotic-transporting ATPase','ATP phosphohydrolase (steroid-exporting)','NULL','Enzymes of Gram-positive bacteria and eukaryotic cells that export anumber of drugs, with unusual specificity, covering various groups ofunrelated substances, while ignoring some that are closely relatedstructurally.Several distinct enzymes may be present in a single eukaryotic cell.Many of them transport glutathione conjugates with drugs.Some also show some flippase (EC 3.6.3.1) activity.Formerly EC 3.6.3.45.','ATP + H(2)O + xenobiotic(In) = ADP + phosphate + xenobiotic(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.45','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.46','Cadmium-transporting ATPase','Yeast cadmium factor','NULL','A Saccharomyces cerevisiae enzyme that exports some heavy metals,especially Cd(2+), from the cytosol into the vacuole.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.47','Fatty-acyl-CoA-transporting ATPase','NULL','NULL','An animal and Saccharomyces cerevisiae enzyme that transports fattyacyl CoA into and out of peroxisomes.','ATP + H(2)O + fatty acyl CoA(Cis) = ADP + phosphate + fatty acylCoA(Trans).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.48','Alpha-factor-transporting ATPase','NULL','NULL','A Saccharomyces cerevisiae enzyme that exports the alpha-factor sexpheromone.','ATP + H(2)O + alpha-factor(In) = ADP + phosphate + alpha-factor(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.49','Channel-conductance-controlling ATPase','Cystic-fibrosis membrane-conductance-regulating protein','NULL','An animal enzyme that is active in forming a chloride channel,the absence of which brings about cystic fibrosis.It is also involved in the functioning of other transmembranechannels.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.50','Protein-secreting ATPase','NULL','NULL','A non-phosphorylated, non-ABC (ATP-binding cassette) ATPase that isinvolved in protein transport.There are several families of enzymes included here, e.g. ATP-hydrolyzing enzymes of the general secretory pathway (Sec or TypeII), of the virulence-related secretory pathway (Type III) and of theconjugal DNA-protein transfer pathway (Type IV).Type II enzymes occur in bacteria, archaea and eukaryotes, whereastype III and type IV enzymes occur in bacteria where they formcomponents of a multi-subunit complex.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.51','Mitochondrial protein-transporting ATPase','NULL','NULL','A non-phosphorylated, non-ABC (ATP-binding cassette) ATPase involvedin the transport of proteins or preproteins into mitochondria usingthe TIM protein complex.TIM is the protein transport machinery of the inner mitochondrialmembrane that contains three essential Tim proteins: Tim17 and Tim23are thought to build a preprotein translocation channel while Tim44interacts transiently with the matrix heat-shock protein Hsp70 toform an ATP-driven import motor.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.52','Chloroplast protein-transporting ATPase','NULL','NULL','A non-phosphorylated, non-ABC (ATP-binding cassette) ATPase that isinvolved in protein transport.Involved in the transport of proteins or preproteins into chloroplaststroma (several ATPases may participate in this process).','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.53','Ag(+)-exporting ATPase','NULL','NULL','P-type ATPase that exports Ag(+) ions from pathogenic microorganismsas well as from some animal tissues.','ATP + H(2)O + Ag(+)(In) = ADP + phosphate + Ag(+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.54','Cu(+) exporting ATPase','NULL','NULL','A P-type ATPase that undergoes covalent phosphorylation during thetransport cycle.This enzyme transports Cu(+) or Ag(+), and cannot transport thedivalent ions, contrary to EC 3.6.3.4, which mainly transports thedivalent copper ion.Formerly EC 3.6.3.n1.','ATP + H(2)O + Cu(+)(Side 1) = ADP + phosphate + Cu(+)(Side 2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.55','Tungstate-importing ATPase','Tungstate transporter','NULL','NULL','ATP + H(2)O + tungstate(Side 1) = ADP + phosphate + tungstate(Side 2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.3.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.1','Myosin ATPase','Actomyosin','NULL','Involved in muscle contraction.In the absence of actin, myosin shows only low ATPase activity, whichrequires Ca(2+).Formerly EC 3.6.1.32.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.2','Dynein ATPase','NULL','NULL','A multisubunit protein complex associated with microtubules.Hydrolysis of ATP provides energy for the movement of organelles(endosomes, lysosomes, mitochondria) along microtubules to thecentrosome toward the microtubules minus end.It also functions in the movement of eukaryotic flagella and cilia.Formerly EC 3.6.1.33.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.3','Microtubule-severing ATPase','Katanin','NULL','Another member of the AAA-ATPase family, active in splittingmicrotubules into tubulin dimers in the centrosome.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.4','Plus-end-directed kinesin ATPase','Kinesin','NULL','Microtubular motor protein, involved in organelle movement,in mitosis and meiosis.In contrast to dynein, it moves along microtubules toward the plusend.Also hydrolyzes GTP.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.5','Minus-end-directed kinesin ATPase','NULL','NULL','Structurally almost identical to EC 3.6.4.3 but the movement itcatalyzes is toward the minus end of microtubules.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.6','Vesicle-fusing ATPase','NULL','NULL','A large family of ATP-hydrolyzing enzymes involved in the heterotypicfusion of membrane vesicles with target membranes and the homotypicfusion of various membrane compartments.They belong to the AAA-type (ATPase associated with a variety of cellactivities) ATPase superfamily.They include peroxin, which apparently is involved in Zellwegerssyndrome.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.7','Peroxisome-assembly ATPase','Peroxisome assembly factor-2','NULL','An extremely diversified group of enzymes that use the energy of ATPhydrolysis to import and assemble peroxisome components into theorganelle.Their molecular masses range from 25 to 600 kDa.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.8','Proteasome ATPase','RP triphosphatase','NULL','Belongs to the AAA-type superfamily and, like EC 3.6.4.5, is involvedin channel gating and polypeptide unfolding before proteolysis in theproteasome.Six ATPase subunits are present in the regulatory particle (RP) of26S proteasome.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.9','Chaperonin ATPase','Chaperonin','NULL','Multisubunit proteins with 2x7 (Type I, in most cells) or 2x8 (TypeII, in archaea) ATP-binding sites involved in maintaining an unfoldedpolypeptide structure before folding or to entry into mitochondriaand chloroplasts.Molecular masses of subunits ranges from 10-90 kDa.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.10','Non-chaperonin molecular chaperone ATPase','Molecular chaperone Hsc70 ATPase','NULL','This is a highly diverse group of enzymes that perform many functionsthat are similar to those of chaperonins.They comprise a number of heat-shock-cognate proteins.They are also active in clathrin uncoating and in the oligomerizationof actin.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.11','Nucleoplasmin ATPase','NULL','NULL','An acidic nuclear protein that is active in the ATP-dependentassembly of nucleosome cores, in decondensation of sperm chromatinand in other histone-involving processes.','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.12','DNA helicase','NULL','NULL','DNA helicases utilize the energy from ATP hydrolysis to unwinddouble-stranded DNA.Some of them unwind duplex DNA with a 3 to 5 polarity (1,3,5,8),other show 5 to 3 polarity (10,11,12,13) or unwind DNA in bothdirections (14,15).Some helicases unwind DNA as well as RNA (4,9).May be identical with EC 3.6.4.13 (RNA helicase).','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.4.13','RNA helicase','NULL','NULL','RNA helicases utilize the energy from ATP hydrolysis to unwind RNA.Some of them unwind RNA with a 3 to 5 polarity, other show 5 to 3polarity.Some helicases unwind DNA as well as RNA.May be identical with EC 3.6.4.12 (DNA helicase).','ATP + H(2)O = ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.5.1','Heterotrimeric G-protein GTPase','NULL','NULL','This group comprises GTP-hydrolyzing systems, where GTP and GDPalternate in binding.This group includes stimulatory and inhibitory G-proteins such asG(s), G(i), G(o) and G(olf), targeting adenylate cyclase and/or K(+)and Ca(2+) channels; G(q) stimulating phospholipase C; transducinactivating cGMP phosphodiesterase; gustducin activating cAMPphosphodiesterase.G(olf) is instrumental in odor perception, transducin in vision andgustducin in taste recognition.At least 16 different alpha subunits (39-52 kDa), 5 beta subunits (36kDa) and 12 gamma subunits (6-9 kDa) are known.Formerly EC 3.6.1.46.','GTP + H(2)O = GDP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.5.2','Small monomeric GTPase','NULL','NULL','A family of about 50 enzymes with a molecular mass of 21 kDa that aredistantly related to the alpha-subunit of EC 3.6.1.46.Involved in cell-growth regulation (Ras subfamily), membrane vesicletraffic and uncoating (Rab and ARF subfamilies), nuclear proteinimport (Ran subfamily) and organization of the cytoskeleton (Rho andRac subfamilies).Formerly EC 3.6.1.47.','GTP + H(2)O = GDP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.5.3','Protein-synthesizing GTPase','Elongation factor (EF)','NULL','This enzyme comprises a family of proteins involved in prokaryotic aswell as eukaryotic protein synthesis.In the initiation factor complex, it is IF-2b (98 kDa) that binds GTPand subsequently hydrolyzes it in prokaryotes.In eukaryotes, it is eIF-2 (150 kDa) that binds GTP.In the elongation phase, the GTP-hydrolyzing proteins are the EF-Tupolypeptide of the prokaryotic transfer factor (43 kDa),the eukaryotic elongation factor EF-1-alpha (53 kDa), the prokaryoticEF-G (77 kDa), the eukaryotic EF-2 (70-110 kDa) and the signalrecognition particle that play a role in endoplasmic reticulumprotein synthesis (325 kDa).EF-Tu and EF-1-alpha catalyze binding of aminoacyl-tRNA to theribosomal A-site, while EF-G and EF-2 catalyze the translocation ofpeptidyl-tRNA from the A-site to the P-site.GTPase activity is also involved in polypeptide release from theribosome with the aid of the pRFs and eRFs.Formerly EC 3.6.1.48.','GTP + H(2)O = GDP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.5.4','Signal-recognition-particle GTPase','NULL','NULL','Activity is associated with the signal-recognition particle (aprotein- and RNA-containing structure involved in endoplasmic-reticulum-associated protein synthesis).Formerly EC 3.6.1.49.','GTP + H(2)O = GDP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.5.5','Dynamin GTPase','NULL','NULL','An enzyme with a molecular mass of about 100 kDa that is involved inendocytosis and is instrumental in pinching off membrane vesicles.Formerly EC 3.6.1.50.','GTP + H(2)O = GDP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.5.6','Tubulin GTPase','NULL','NULL','An intrinsic activity of alpha-tubulin involved in tubulin folding,division plane formation in prokaryotic cells and others.Formerly EC 3.6.1.51.','GTP + H(2)O = GDP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.6.5.n1','Elongation factor 4','NULL','NULL','The enzyme is required for accurate and efficient protein synthesisunder certain stress conditions.May act as a fidelity factor of the translation reaction,by catalyzing a one-codon backward translocation of tRNAs onimproperly translocated ribosomes.Back-translocation proceeds from a post-translocation (POST) complexto a pre-translocation (PRE) complex, thus giving elongation factor Ga second chance to translocate the tRNAs correctly.Binds to ribosomes in a GTP-dependent manner.','GTP + H(2)O = GDP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.1','Oxaloacetase','Oxalacetic hydrolase','NULL','NULL','Oxaloacetate + H(2)O = oxalate + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.2','Fumarylacetoacetase','Beta-diketonase','NULL','Also acts on other 3,5- and 2,4-dioxo acids.','4-fumarylacetoacetate + H(2)O = acetoacetate + fumarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.3','Kynureninase','L-kynurenine hydrolase','Pyridoxal 5-phosphate.','Also acts on 3-hydroxykynurenine and some other (3-arylcarbonyl)-alanines.','L-kynurenine + H(2)O = anthranilate + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.4','Phloretin hydrolase','NULL','NULL','Also hydrolyzes other C-acylated phenols related to phloretin.','Phloretin + H(2)O = phloretate + phloroglucinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.5','Acylpyruvate hydrolase','NULL','NULL','Acts on formylpyruvate, 2,4-dioxopentanoate, 2,4-dioxohexanoate and2,4-dioxoheptanoate.','A 3-acylpyruvate + H(2)O = a carboxylate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.6','Acetylpyruvate hydrolase','NULL','NULL','Highly specific; does not act on pyruvate, oxaloacetate,maleylpyruvate, fumarylpyruvate or acetylacetone.','Acetylpyruvate + H(2)O = acetate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.7','Beta-diketone hydrolase','Oxidized PVA hydrolase','NULL','Also acts on the product of the action of EC 1.1.3.18 on polyvinylalcohols.Involved in the bacterial degradation of polyvinyl alcohol.','Nonane-4,6-dione + H(2)O = pentan-2-one + butanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.8','2,6-dioxo-6-phenylhexa-3-enoate hydrolase','HOHPDA hydrolase','NULL','Cleaves the products from biphenol, 3-isopropylcatechol and3-methylcatechol produced by EC 1.13.11.39 by ring-fission at a-CO-C bond.Involved in the breakdown of biphenyl-related compounds byPseudomonas sp.','2,6-dioxo-6-phenylhexa-3-enoate + H(2)O = benzoate + 2-oxopent-4-enoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.9','2-hydroxymuconate-6-semialdehyde hydrolase','2-hydroxymuconate-semialdehyde hydrolase','NULL','The enzyme is involved in the degradation of catechols.','2-hydroxymuconate-6-semialdehyde + H(2)O = formate + 2-oxopent-4-enoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.10','Cyclohexane-1,3-dione hydrolase','NULL','NULL','Highly specific; does not act on other dione derivatives ofcyclohexane, cyclopentane or cycloheptane.','Cyclohexane-1,3-dione + H(2)O = 5-oxohexanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.11','Cyclohexane-1,2-dione hydrolase','NULL','NULL','Highly specific; does not act on cyclohexanone or cyclohexane-1,3-dione as substrate.','Cyclohexane-1,2-dione + H(2)O = 6-oxohexanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.12','Cobalt-precorrin 5A hydrolase','NULL','NULL','This enzyme hydrolyzes the ring A acetate delta-lactone of cobalt-precorrin-5A resulting in the loss of the C-20 carbon and itsattached methyl group in the form of acetaldehyde.This is a key reaction in the contraction of the porphyrin-typetetrapyrrole ring and its conversion to a corrin ring in theanaerobic (early cobalt insertion) adenosylcobalamin biosynthesispathway.','Cobalt-precorrin-5A + H(2)O = cobalt-precorrin-5B + acetaldehyde +2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.13','2-hydroxy-6-oxo-6-(2-aminophenyl)hexa-2,4-dienoate hydrolase','NULL','NULL','This enzyme catalyzes the third step in the aerobic degradationpathway of carbazole.The effect of the presence of an amino group or hydroxyl group at the2-position of the substrate is small.The enzyme has no cofactor requirement.','(2E,4E)-6-(2-aminophenyl)-2-hydroxy-6-oxohexa-2,4-dienoate + H(2)O =anthranilate + (2E)-2-hydroxypenta-2,4-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.14','2-hydroxy-6-oxonona-2,4-dienedioate hydrolase','NULL','NULL','This enzyme catalyzes a step in a pathway of phenylpropanoidcompounds degradation.The first step of the enzyme mechanism involves a reversible keto-enol tautomerization.Formerly EC 3.7.1.n1.','(1) (2Z,4E)-2-hydroxy-6-oxonona-2,4-diene-1,9-dioate + H(2)O = (2Z)-2-hydroxypenta-2,4-dienoate + succinate.(2) (2Z,4E,7E)-2-hydroxy-6-oxonona-2,4,7-triene-1,9-dioate + H(2)O =(2Z)-2-hydroxypenta-2,4-dienoate + fumarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.15','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.16','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.17','4,5-9,10-diseco-3-hydroxy-5,9,17-trioxoandrosta-1(10),2-diene-4-oatehydrolase','NULL','NULL','The enzyme is involved in the bacterial degradation of the steroidring structure, and is involved in degradation of multiple steroids,such as testosterone, cholesterol, and sitosterol.','(1E,2Z)-3-hydroxy-5,9,17-trioxo-4,5:9,10-disecoandrosta-1(10),2-dien-4-oate + H(2)O = 3-((3aS,4S,7aS)-7a-methyl-1,5-dioxo-octahydro-1H-inden-4-yl)propanoate + (2Z,4Z)-2-hydroxyhexa-2,4-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.18','6-oxocamphor hydrolase','NULL','NULL','Isolated from Rhodococcus sp.The bornane ring system is cleaved by a retro-Claisen reaction togive the enol of alpha-campholonate.When separate from the enzyme the enol is tautomerised to the ketoform as a 6:1 mixture of ((1S,3R)-2,2,3-trimethyl-4-oxocyclopentyl)acetate and ((1S,3S)-2,2,3-trimethyl-4-oxocyclopentyl)acetate.','Bornane-2,6-dione + H(2)O = ((1S)-4-hydroxy-2,2,3-trimethylcyclopent-3-enyl)acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.19','2,6-dihydroxypseudooxynicotine hydrolase','NULL','NULL','The enzyme, characterized from the soil bacterium Arthrobacternicotinovorans, participates in nicotine degradation.','1-(2,6-dihydroxypyridin-3-yl)-4-(methylamino)butan-1-one + H(2)O = 2,6-dihydroxypyridine + 4-methylaminobutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.20','3-fumarylpyruvate hydrolase','NULL','NULL','The enzyme is involved in bacterial degradation of 5-substitutedsalicylates, including gentisate (5-hydroxysalicylate),5-nitrosalicylate and 5-halosalicylates.','3-fumarylpyruvate + H(2)O = fumarate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.21','6-oxocyclohex-1-ene-1-carbonyl-CoA hydratase','6-oxocyclohex-1-ene-1-carbonyl-CoA hydrolase','NULL','The enzyme, which participates in the anaerobic benzoyl-CoAdegradation pathway in certain organisms, catalyzes the addition ofone molecule of water to the double bound of 6-oxocyclohex-1-ene-1-carbonyl-CoA followed by the hydrolytic C-C cleavage of the alicyclicring.','6-oxocyclohex-1-ene-1-carbonyl-CoA + 2 H(2)O = 3-hydroxypimeloyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.22','3D-(3,5/4)-trihydroxycyclohexane-1,2-dione acylhydrolase (decyclizing)','THcHDO hydrolase','Magnesium; Thiamine diphosphate.','The enzyme, found in the bacterium Bacillus subtilis, is part of themyo-inositol degradation pathway leading to acetyl-CoA.Formerly EC 3.7.1.n2.','3D-3,5/4-trihydroxycyclohexa-1,2-dione + H(2)O = 5-deoxy-D-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.n1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.7.1.n2','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.1','Alkylhalidase','Halogenase','NULL','NULL','Bromochloromethane + H(2)O = formaldehyde + bromide + chloride.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.2','(S)-2-haloacid dehalogenase','2-haloacid dehalogenase','NULL','Acts on acids of short chain lengths, C(2) to C(4), with inversion ofconfiguration at C-2 (see also EC 3.8.1.9, EC 3.8.1.10 andEC 3.8.1.11).','(S)-2-haloacid + H(2)O = (R)-2-hydroxyacid + halide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.3','Haloacetate dehalogenase','NULL','NULL','NULL','Haloacetate + H(2)O = glycolate + halide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.4','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.5','Haloalkane dehalogenase','1-chlorohexane halidohydrolase','NULL','Acts on a wide range of 1-haloalkanes, haloalcohols, haloalkenes andsome haloaromatic compounds.','1-haloalkane + H(2)O = a primary alcohol + halide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.6','4-chlorobenzoate dehalogenase','NULL','NULL','Catalyzes the first step in the degradation of chlorobenzoate inPseudomonas.In many microorganisms, this activity comprises three separateenzymes, EC 6.2.1.33, EC 3.8.1.7 and EC 3.1.2.23.','4-chlorobenzoate + H(2)O = 4-hydroxybenzoate + chloride.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.7','4-chlorobenzoyl-CoA dehalogenase','NULL','NULL','Specific for dehalogenation at the 4-position.Can dehalogenate substrates bearing fluorine, chlorine, bromine andiodine in the 4-position.This enzyme is part of the bacterial 2,4-dichlorobenzoate degradationpathway.','4-chlorobenzoyl-CoA + H(2)O = 4-hydroxybenzoyl CoA + chloride.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.8','Atrazine chlorohydrolase','NULL','NULL','Involved in the degradation of the herbicide atrazine, 2-chloro-4-(ethylamino)-6-(isopropylamino)-1,3,5-triazine, in bacteria.','Atrazine + H(2)O = 4-(ethylamino)-2-hydroxy-6-(isopropylamino)-1,3,5-triazine + HCl.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.9','(R)-2-haloacid dehalogenase','2-haloalkanoic acid dehalogenase','NULL','Acts on acids of short chain lengths, C(2) to C(4), with inversion ofconfiguration at C-2 (see also EC 3.8.1.2, EC 3.8.1.10 andEC 3.8.1.11).','(R)-2-haloacid + H(2)O = (S)-2-hydroxyacid + halide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.10','2-haloacid dehalogenase (configuration-inverting)','2-haloalkanoic acid dehalogenase','NULL','Dehalogenates both (S)- and (R)-2-haloalkanoic acids to thecorresponding (R)- and (S)-hydroxyalkanoic acids, respectively, withinversion of configuration at C-2.The enzyme from Pseudomonas sp. 113 acts on 2-haloalkanoic acidswhose carbon chain lengths are five or less (see also EC 3.8.1.2,EC 3.8.1.9 and EC 3.8.1.11).','(1) (S)-2-haloacid + H(2)O = (R)-2-hydroxyacid + halide.(2) (R)-2-haloacid + H(2)O = (S)-2-hydroxyacid + halide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.1.11','2-haloacid dehalogenase (configuration-retaining)','2-haloalkanoic acid dehalogenase','NULL','Dehalogenates both (S)- and (R)-2-haloalkanoic acids to thecorresponding (S)- and (R)-hydroxyalkanoic acids, respectively, withretention of configuration at C-2 (see also EC 3.8.1.2, EC 3.8.1.9and EC 3.8.1.10).','(1) (S)-2-haloacid + H(2)O = (S)-2-hydroxyacid + halide.(2) (R)-2-haloacid + H(2)O = (R)-2-hydroxyacid + halide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.8.2.1','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.9.1.1','Phosphoamidase','NULL','NULL','Also acts on N-phospho-arginine and other phosphoamides.Possibly identical with EC 3.1.3.9 or EC 3.1.3.16.','N-phosphocreatine + H(2)O = creatine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.10.1.1','N-sulfoglucosamine sulfohydrolase','Sulfoglucosamine sulfamidase','NULL','NULL','N-sulfo-D-glucosamine + H(2)O = D-glucosamine + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.10.1.2','Cyclamate sulfohydrolase','Cyclamate sulfamatase','NULL','Also readily hydrolyzes aliphatic sulfamates with 3 to 8 carbons.','Cyclohexylsulfamate + H(2)O = cyclohexylamine + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.11.1.1','Phosphonoacetaldehyde hydrolase','2-phosphonoacetylaldehyde phosphonohydrolase','Magnesium.','This enzyme destabilizes the C-P bond, by forming an imine betweenone of its lysine residues and the carbonyl group of the substrate,thus allowing this, normally stable, bond to be broken.The mechanism is similar to that used by EC 4.1.2.13 to break aC-C bond.','Phosphonoacetaldehyde + H(2)O = acetaldehyde + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.11.1.2','Phosphonoacetate hydrolase','NULL','Zinc.','NULL','Phosphonoacetate + H(2)O = acetate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.11.1.3','Phosphonopyruvate hydrolase','PPH','Cobalt or manganese or magnesium.','Highly specific for phosphonopyruvate as substrate.The reaction is not inhibited by phosphate but is inhibited by thephosphonates phosphonoformic acid, hydroxymethylphosphonic acid and3-phosphonopropanoic acid.','3-phosphonopyruvate + H(2)O = pyruvate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.12.1.1','Trithionate hydrolase','NULL','NULL','NULL','Trithionate + H(2)O = thiosulfate + sulfate + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.13.1.1','UDP-sulfoquinovose synthase','Sulfite:UDP-glucose sulfotransferase','NAD(+).','NAD(+) appears to oxidize the substrate to UDP-4-dehydroglucose,which dehydrates to UDP-4-dehydro-6-deoxygluc-5-enose, to whichsulfite can add; the reaction is completed when the substrate isrehydrogenated at C-4.The enzyme from Arabidopsis thaliana is specific for UDP-Glc andsulfite.','UDP-alpha-D-glucose + sulfite = UDP-alpha-D-sulfoquinovose + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.13.1.2','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 3.13.1.3','2-hydroxybiphenyl-2-sulfinate desulfinase','2-(2-hydroxyphenyl)benzenesulfinate desulfinase','FMN.','The enzyme has a narrow substrate specificity with biphenyl-2-sulfinate being the only other substrate known to date.Formerly EC 3.1.2.24.','2-hydroxybiphenyl-2-sulfinate + H(2)O = 2-hydroxybiphenyl + sulfite.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.1','Pyruvate decarboxylase','2-oxo-acid carboxy-lyase','Thiamine diphosphate.','Also catalyzes acyloin formation.','A 2-oxo acid = an aldehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.2','Oxalate decarboxylase','Oxalate carboxy-lyase','Manganese.','NULL','Oxalate = formate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.3','Oxaloacetate decarboxylase','Oxalate beta-decarboxylase','Biotin; Sodium or manganese.','The enzyme from Klebsiella aerogenes is a biotinyl protein and alsodecarboxylates glutaconyl-CoA and methylmalonyl-CoA.The process is accompanied by the extrusion of two sodium ions fromcells.Some animal enzymes require manganese.','Oxaloacetate = pyruvate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.4','Acetoacetate decarboxylase','Acetoacetate carboxy-lyase','NULL','NULL','Acetoacetate = acetone + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.5','Acetolactate decarboxylase','(S)-2-hydroxy-2-methyl-3-oxobutanoate carboxy-lyase','NULL','NULL','(2S)-2-hydroxy-2-methyl-3-oxobutanoate = (3R)-3-hydroxybutan-2-one +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.6','Aconitate decarboxylase','CAD','NULL','NULL','Cis-aconitate = itaconate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.7','Benzoylformate decarboxylase','Benzoylformate carboxy-lyase','Thiamine diphosphate.','NULL','Benzoylformate = benzaldehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.8','Oxalyl-CoA decarboxylase','Oxalyl-CoA carboxy-lyase','Thiamine diphosphate.','NULL','Oxalyl-CoA = formyl-CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.9','Malonyl-CoA decarboxylase','Malonyl-CoA carboxy-lyase','NULL','Specific for malonyl-CoA.The enzyme from Pseudomonas ovalis also catalyzes the reaction ofEC 2.8.3.3.','Malonyl-CoA = acetyl-CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.10','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.11','Aspartate 1-decarboxylase','Aspartate alpha-decarboxylase','Pyruvate.','NULL','L-aspartate = beta-alanine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.12','Aspartate 4-decarboxylase','Desulfinase','Pyridoxal 5-phosphate.','Also catalyzes the decarboxylation of aminomalonate and thedesulfination of 3-sulfino-L-alanine to sulfite and alanine.Formerly EC 4.1.1.10.','L-aspartate = L-alanine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.13','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.14','Valine decarboxylase','L-valine carboxy-lyase','Pyridoxal 5-phosphate.','Also acts on L-leucine.','L-valine = 2-methylpropanamine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.15','Glutamate decarboxylase','L-glutamate 1-carboxy-lyase','Pyridoxal 5-phosphate.','The brain enzyme also acts on L-cysteate, 3-sulfino-L-alanine andL-aspartate.','L-glutamate = 4-aminobutanoate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.16','Hydroxyglutamate decarboxylase','3-hydroxy-L-glutamate 1-carboxy-lyase','Pyridoxal 5-phosphate.','NULL','3-hydroxy-L-glutamate = 4-amino-3-hydroxybutanoate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.17','Ornithine decarboxylase','L-ornithine carboxy-lyase','Pyridoxal 5-phosphate.','NULL','L-ornithine = putrescine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.18','Lysine decarboxylase','L-lysine carboxy-lyase','Pyridoxal 5-phosphate.','Also acts on 5-hydroxy-L-lysine.','L-lysine = cadaverine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.19','Arginine decarboxylase','L-arginine carboxy-lyase','Pyridoxal 5-phosphate.','NULL','L-arginine = agmatine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.20','Diaminopimelate decarboxylase','DAP decarboxylase','Pyridoxal 5-phosphate.','NULL','Meso-2,6-diaminoheptanedioate = L-lysine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.21','Phosphoribosylaminoimidazole carboxylase','1-(5-phosphoribosyl)-5-amino-4-imidazolecarboxylate carboxy-lyase','NULL','While this is the reaction that occurs in vertebrates during purinebiosynthesis, two enzymes are required to carry out the same reactionin Escherichia coli, namely EC 6.3.4.18 and EC 5.4.99.18.5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole is not a substrate.','5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate = 5-amino-1-(5-phospho-D-ribosyl)imidazole + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.22','Histidine decarboxylase','L-histidine carboxy-lyase','Pyruvate or pyridoxal 5-phosphate.','NULL','L-histidine = histamine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.23','Orotidine-5-phosphate decarboxylase','OMP decarboxylase','NULL','The enzyme from higher eukaryotes is identical with EC 2.4.2.10.','Orotidine 5-phosphate = UMP + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.24','Aminobenzoate decarboxylase','Aminobenzoate carboxy-lyase','Pyridoxal 5-phosphate.','NULL','4(or 2)-aminobenzoate = aniline + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.25','Tyrosine decarboxylase','L-tyrosine carboxy-lyase','Pyridoxal 5-phosphate.','The bacterial enzyme also acts on 3-hydroxytyrosine and, more slowly,on 3-hydroxyphenylalanine.','L-tyrosine = tyramine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.26','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.27','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.28','Aromatic-L-amino-acid decarboxylase','5-hydroxytryptophan decarboxylase','Pyridoxal 5-phosphate.','Also acts on other aromatic L-amino acids, including L-tryptophan.Formerly EC 4.1.1.26 and EC 4.1.1.27.','(1) L-dopa = dopamine + CO(2).(2) 5-hydroxy-L-tryptophan = 5-hydroxytryptamine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.29','Sulfinoalanine decarboxylase','3-sulfino-L-alanine carboxy-lyase','Pyridoxal 5-phosphate.','Also acts on L-cysteate.','3-sulfino-L-alanine = hypotaurine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.30','Pantothenoylcysteine decarboxylase','N-((R)-pantothenoyl)-L-cysteine carboxy-lyase','NULL','NULL','N-((R)-pantothenoyl)-L-cysteine = pantetheine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.31','Phosphoenolpyruvate carboxylase','PEP carboxylase','NULL','This enzyme replenishes oxaloacetate in the tricarboxylic acid cyclewhen operating in the reverse direction.The reaction proceeds in two steps: formation of carboxyphosphate andthe enolate form of pyruvate, followed by carboxylation of theenolate and release of phosphate.','Phosphate + oxaloacetate = H(2)O + phosphoenolpyruvate + HCO(3)(-).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.32','Phosphoenolpyruvate carboxykinase (GTP)','PEP carboxykinase','NULL','ITP can act as phosphate donor.','GTP + oxaloacetate = GDP + phosphoenolpyruvate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.33','Diphosphomevalonate decarboxylase','Mevalonate diphosphate decarboxylase','NULL','NULL','ATP + (R)-5-diphosphomevalonate = ADP + phosphate + isopentenyldiphosphate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.34','Dehydro-L-gulonate decarboxylase','3-dehydro-L-gulonate carboxy-lyase','NULL','NULL','3-dehydro-L-gulonate = L-xylulose + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.35','UDP-glucuronate decarboxylase','UDP-D-glucuronate carboxy-lyase','NAD(+).','NULL','UDP-D-glucuronate = UDP-D-xylose + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.36','Phosphopantothenoylcysteine decarboxylase','N-((R)-4-phosphopantothenoyl)-L-cysteine carboxy-lyase','FMN.','NULL','N-((R)-4-phosphopantothenoyl)-L-cysteine = pantotheine 4-phosphate +CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.37','Uroporphyrinogen decarboxylase','Uroporphyrinogen III decarboxylase','NULL','Acts on a number of porphyrinogens.','Uroporphyrinogen III = coproporphyrinogen + 4 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.38','Phosphoenolpyruvate carboxykinase (diphosphate)','PEP carboxyphosphotransferase','NULL','Also catalyzes the reaction: phosphoenolpyruvate + phosphate =pyruvate + diphosphate.','Diphosphate + oxaloacetate = phosphate + phosphoenolpyruvate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.39','Ribulose-bisphosphate carboxylase','Carboxydismutase','NULL','Will utilize O(2) instead of CO(2), forming 3-phospho-D-glycerate and2-phosphoglycolate.','2 3-phospho-D-glycerate + 2 H(+) = D-ribulose 1,5-bisphosphate + CO(2) +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.40','Hydroxypyruvate decarboxylase','Hydroxypyruvate carboxy-lyase','NULL','NULL','Hydroxypyruvate = glycolaldehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.41','Methylmalonyl-CoA decarboxylase','(S)-2-methyl-3-oxopropanoyl-CoA carboxy-lyase','Biotin.','The enzyme from Veillonella alcalescens is a biotinyl-protein,requires sodium and acts as a sodium pump.','(S)-methylmalonyl-CoA = propanoyl-CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.42','Carnitine decarboxylase','Carnitine carboxy-lyase','NULL','Requires ATP.','Carnitine = 2-methylcholine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.43','Phenylpyruvate decarboxylase','Phenylpyruvate carboxy-lyase','NULL','Also acts on (indol-3-yl)pyruvate.','Phenylpyruvate = phenylacetaldehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.44','4-carboxymuconolactone decarboxylase','4-carboxymuconolactone carboxy-lyase','NULL','NULL','2-carboxy-2,5-dihydro-5-oxofuran-2-acetate = 4,5-dihydro-5-oxofuran-2-acetate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.45','Aminocarboxymuconate-semialdehyde decarboxylase','Picolinic acid decarboxylase','NULL','The product rearranges non-enzymically to picolinate.','2-amino-3-(3-oxoprop-1-en-1-yl)but-2-enedioate = 2-aminomuconatesemialdehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.46','o-pyrocatechuate decarboxylase','2,3-DHBA decarboxylase','NULL','NULL','2,3-dihydroxybenzoate = catechol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.47','Tartronate-semialdehyde synthase','Glyoxylate carboligase','Thiamine diphosphate.','NULL','2 glyoxylate = tartronate semialdehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.48','Indole-3-glycerol-phosphate synthase','Indoleglycerol phosphate synthetase','NULL','In some organisms, this enzyme is part of a multifunctional proteintogether with one or more components of the system for biosynthesisof tryptophan (EC 2.4.2.18, EC 4.1.3.27, EC 4.2.1.20,and EC 5.3.1.24).','1-(2-carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate = 1-C-(3-indolyl)-glycerol 3-phosphate + CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.49','Phosphoenolpyruvate carboxykinase (ATP)','PEP carboxykinase','NULL','NULL','ATP + oxaloacetate = ADP + phosphoenolpyruvate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.50','Adenosylmethionine decarboxylase','S-adenosyl-L-methionine carboxy-lyase','Pyruvate.','NULL','S-adenosyl-L-methionine = S-adenosyl 3-(methylthio)propylamine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.51','3-hydroxy-2-methylpyridine-4,5-dicarboxylate 4-decarboxylase','3-hydroxy-2-methylpyridine-4,5-dicarboxylate 4-carboxy-lyase','NULL','NULL','3-hydroxy-2-methylpyridine-4,5-dicarboxylate = 3-hydroxy-2-methylpyridine-5-carboxylate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.52','6-methylsalicylate decarboxylase','6-methylsalicylate carboxy-lyase','NULL','NULL','6-methylsalicylate = 3-methylphenol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.53','Phenylalanine decarboxylase','L-phenylalanine carboxy-lyase','Pyridoxal 5-phosphate.','Also acts on tyrosine and other aromatic amino acids.','L-phenylalanine = phenylethylamine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.54','Dihydroxyfumarate decarboxylase','Dihydroxyfumarate carboxy-lyase','NULL','NULL','Dihydroxyfumarate = tartronate semialdehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.55','4,5-dihydroxyphthalate decarboxylase','4,5-dihydroxyphthalate carboxy-lyase','NULL','NULL','4,5-dihydroxyphthalate = 3,4-dihydroxybenzoate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.56','3-oxolaurate decarboxylase','3-oxododecanoate carboxy-lyase','NULL','Also decarboxylates other C(14) to C(16) oxo acids.','3-oxododecanoate = 2-undecanone + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.57','Methionine decarboxylase','L-methionine carboxy-lyase','NULL','NULL','L-methionine = 3-methylthiopropanamine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.58','Orsellinate decarboxylase','Orsellinate carboxy-lyase','NULL','NULL','2,4-dihydroxy-6-methylbenzoate = orcinol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.59','Gallate decarboxylase','Gallate carboxy-lyase','NULL','NULL','3,4,5-trihydroxybenzoate = pyrogallol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.60','Stipitatonate decarboxylase','NULL','NULL','NULL','Stipitatonate = stipitatate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.61','4-hydroxybenzoate decarboxylase','4-hydroxybenzoate carboxy-lyase','NULL','NULL','4-hydroxybenzoate = phenol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.62','Gentisate decarboxylase','2,5-dihydroxybenzoate decarboxylase','NULL','NULL','2,5-dihydroxybenzoate = hydroquinone + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.63','Protocatechuate decarboxylase','3,4-dihydroxybenzoate decarboxylase','NULL','NULL','3,4-dihydroxybenzoate = catechol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.64','2,2-dialkylglycine decarboxylase (pyruvate)','L-alanine-alpha-ketobutyrate aminotransferase','Pyridoxal 5-phosphate.','Acts on 2-amino-2-methylpropanoate (i.e. 2-methylalanine), 2-amino-2-methylbutanoate and 1-aminocyclopentanecarboxylate.','2,2-dialkylglycine + pyruvate = dialkyl ketone + CO(2) + L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.65','Phosphatidylserine decarboxylase','Phosphatidyl-L-serine carboxy-lyase','Pyruvate or pyridoxal 5-phosphate.','NULL','Phosphatidyl-L-serine = phosphatidylethanolamine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.66','Uracil-5-carboxylate decarboxylase','Uracil-5-carboxylate carboxy-lyase','NULL','NULL','Uracil 5-carboxylate = uracil + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.67','UDP-galacturonate decarboxylase','UDP-D-galacturonate carboxy-lyase','NULL','NULL','UDP-D-galacturonate = UDP-L-arabinose + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.68','5-oxopent-3-ene-1,2,5-tricarboxylate decarboxylase','5-oxopent-3-ene-1,2,5-tricarboxylate carboxy-lyase','NULL','NULL','5-oxopent-3-ene-1,2,5-tricarboxylate = 2-oxohept-3-enedioate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.69','3,4-dihydroxyphthalate decarboxylase','3,4-dihydroxyphthalate carboxy-lyase','NULL','NULL','3,4-dihydroxyphthalate = 3,4-dihydroxybenzoate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.70','Glutaconyl-CoA decarboxylase','4-carboxybut-2-enoyl-CoA carboxy-lyase','Biotin.','The enzyme from Acidaminococcus fermentans is a biotinyl-protein,requires sodium, and acts as a sodium pump.Prior to the sodium dependent decarboxylation, the carboxylate istransferred to biotin in a sodium independent manner.','4-carboxybut-2-enoyl-CoA = but-2-enoyl-CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.71','2-oxoglutarate decarboxylase','2-oxoglutarate carboxy-lyase','Thiamine diphosphate.','Highly specific.','2-oxoglutarate = succinate semialdehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.72','Branched-chain-2-oxoacid decarboxylase','(3S)-3-methyl-2-oxopentanoate carboxy-lyase','NULL','Acts on a number of 2-oxo acids, with a high affinity towardbranched-chain substrates.The aldehyde formed may be enzyme-bound, and may be an intermediatein the bacterial system for biosynthesis of branched-chain fattyacids.','(3S)-3-methyl-2-oxopentanoate = 2-methylbutanal + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.73','Tartrate decarboxylase','(R,R)-tartrate carboxy-lyase','NULL','NULL','(R,R)-tartrate = D-glycerate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.74','Indolepyruvate decarboxylase','3-(indol-3-yl)pyruvate carboxy-lyase','Magnesium; Thiamine diphosphate.','More specific than EC 4.1.1.1.','3-(indol-3-yl)pyruvate = 2-(indol-3-yl)acetaldehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.75','5-guanidino-2-oxopentanoate decarboxylase','2-oxo-5-guanidinopentanoate carboxy-lyase','Divalent cation; Thiamine diphosphate.','NULL','5-guanidino-2-oxo-pentanoate = 4-guanidinobutanal + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.76','Arylmalonate decarboxylase','2-aryl-2-methylmalonate carboxy-lyase','NULL','NULL','2-aryl-2-methylmalonate = 2-arylpropanoate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.77','2-oxo-3-hexenedioate decarboxylase','4-oxalocrotonate carboxy-lyase','NULL','Involved in the meta-cleavage pathway for the degradation of phenols,modified phenols and catechols.The enzyme has been reported to accept multiple tautomeric forms.However, careful analysis of the stability of the differenttautomers, as well as characterization of the enzyme that producesits substrate, EC 5.3.2.6, showed that the actual substrate for theenzyme is (3E)-2-oxohex-3-enedioate.','(3E)-2-oxohex-3-enedioate = 2-oxopent-4-enoate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.78','Acetylenedicarboxylate decarboxylase','Acetylenedicarboxylate carboxy-lyase','NULL','The mechanism appears to involve hydration of the acetylene anddecarboxylation of the oxaloacetic acid formed, although freeoxaloacetate is not an intermediate.It is thus analogous to EC 4.2.1.71 in its mechanism.Formerly EC 4.2.1.72.','Acetylenedicarboxylate + H(2)O = pyruvate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.79','Sulfopyruvate decarboxylase','Sulfopyruvate carboxy-lyase','Thiamine diphosphate.','Does not decarboxylate pyruvate or phosphonopyruvate.The enzyme appears to be oxygen-sensitive.','3-sulfopyruvate = 2-sulfoacetaldehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.80','4-hydroxyphenylpyruvate decarboxylase','4-hydroxyphenylpyruvate carboxy-lyase','NULL','Reacts with dopamine to give the benzylisoquinoline alkaloidskeleton.','4-hydroxyphenylpyruvate = 4-hydroxyphenylacetaldehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.81','Threonine-phosphate decarboxylase','L-threonine O-3-phosphate carboxy-lyase','Pyridoxal 5-phosphate.','Unable to decarboxylate the D-isomer of threonine O-3-phosphate.The product of this reaction, (R)-1-aminopropan-2-yl phosphate,is the substrate of EC 6.3.1.10, which converts adenosylcobyric acidinto adenosylcobinamide phosphate in the anaerobic cobalaminbiosynthesis pathway.','L-threonine O-3-phosphate = (R)-1-aminopropan-2-yl phosphate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.82','Phosphonopyruvate decarboxylase','3-phosphonopyruvate carboxy-lyase','Magnesium; Thiamine diphosphate.','Catalyzes a step in the biosynthetic pathway of2-aminoethylphosphonate, a component of the capsular polysaccharidecomplex of Bacteroides fragilis.Activated by the divalent cations Mg(2+), Ca(2+) and Mn(2+).Pyruvate and sulfopyruvate can also act as substrates, but moreslowly.','3-phosphonopyruvate = 2-phosphonoacetaldehyde + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.83','4-hydroxyphenylacetate decarboxylase','4-Hpd','NULL','The enzyme, from the strict anaerobe Clostridium difficile, can alsouse (3,4-dihydroxyphenyl)acetate as a substrate, yielding4-methylcatechol as a product.The enzyme is a glycyl radical enzyme.','(4-hydroxyphenyl)acetate = 4-methylphenol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.84','D-dopachrome decarboxylase','D-dopachrome carboxy-lyase','NULL','Specific for D-dopachrome as substrate.L-dopachrome, L- or D-alpha-methyldopachrome and dopaminochrome donot act as substrates (see also EC 5.3.3.12).','D-dopachrome = 5,6-dihydroxyindole + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.85','3-dehydro-L-gulonate-6-phosphate decarboxylase','3-dehydro-L-gulonate-6-phosphate carboxy-lyase','Magnesium.','Along with EC 5.1.3.22, this enzyme is involved in a pathway for theutilization of L-ascorbate by Escherichia coli.','3-dehydro-L-gulonate 6-phosphate = L-xylulose 5-phosphate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.86','Diaminobutyrate decarboxylase','DABA DC','Divalent cation; Pyridoxal 5-phosphate.','N(4)-acetyl-L-2,4-diaminobutanoate, 2,3-diaminopropanoate, ornithineand lysine are not substrates.In Acinetobacter baumannii it is cotranscribed with the neighboringdat gene that encodes EC 2.6.1.76, which can supply the substrate forthe decarboxylase.','L-2,4-diaminobutanoate = propane-1,3-diamine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.87','Malonyl-S-ACP decarboxylase','Malonyl-S-acyl-carrier protein decarboxylase','NULL','This enzyme comprises the beta and gamma subunits of EC 4.1.1.88 butis not present in EC 4.1.1.89.It follows on from EC 2.3.1.187 and results in the regeneration ofthe acetylated form of the acyl-carrier-protein subunit of malonatedecarboxylase.The carboxy group is lost with retention of configuration.','A malonyl-[acyl-carrier protein] + H(+) = an acetyl-[acyl-carrierprotein] + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.88','Biotin-independent malonate decarboxylase','Malonate carboxy-lyase (biotin-independent)','NULL','Two types of malonate decarboxylase are currently known, both ofwhich form multienzyme complexes.This enzyme is a cytosolic protein that is biotin-independent.The other type is a biotin-dependent, Na(+)-translocating enzyme thatincludes both soluble and membrane-bound components (cf.EC 4.1.1.89).As free malonate is chemically rather inert, it has to be activatedprior to decarboxylation.In both enzymes, this is achieved by exchanging malonate with anacetyl group bound to an acyl-carrier protiein (ACP), to formmalonyl-ACP and acetate, with subsequent decarboxylation regeneratingthe acetyl-ACP.The ACP subunit of both enzymes differs from that found in fatty-acidbiosynthesis by having phosphopantethine attached to a serine side-chain as 2-(5-triphosphoribosyl)-3-dephospho-CoA rather than asphosphopantetheine 4-phosphate.The individual enzymes involved in carrying out the reaction of thisenzyme complex are EC 2.3.1.187, EC 2.3.1.39 and EC 4.1.1.87.The carboxy group is lost with retention of configuration.','Malonate + H(+) = acetate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.89','Biotin-dependent malonate decarboxylase','Malonate carboxy-lyase (biotin-dependent)','NULL','Two types of malonate decarboxylase are currently known, both ofwhich form multienzyme complexes.The enzyme described here is a biotin-dependent, Na(+)-translocatingenzyme that includes both soluble and membrane-bound components.The other type is a biotin-independent cytosolic protein (cf.EC 4.1.1.88).As free malonate is chemically rather inert, it has to be activatedprior to decarboxylation.Both enzymes achieve this by exchanging malonate with an acetyl groupbound to an acyl-carrier protiein (ACP), to form malonyl-ACP andacetate, with subsequent decarboxylation regenerating the acetyl-bound form of the enzyme.The ACP subunit of both enzymes differs from that found in fatty-acidbiosynthesis by having phosphopantethine attached to a serine side-chain as 2-(5-triphosphoribosyl)-3-dephospho-CoA rather than asphosphopantetheine 4-phosphate.In the anaerobic bacterium Malonomonas rubra, the components of themultienzyme complex/enzymes involved in carrying out the reactions ofthis enzyme are as follows: MadA (EC 2.3.1.187), MadB (EC 4.3.99.2),MadC/MadD (EC 2.1.3.10) and MadH (EC 6.2.1.35).Two other components that are involved are MadE, the acyl-carrierprotein and MadF, the biotin protein.The carboxy group is lost with retention of configuration.','Malonate + H(+) = acetate + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.90','Peptidyl-glutamate 4-carboxylase','Gamma-glutamyl carboxylase','NULL','The enzyme can use various vitamin-K derivatives, includingmenaquinone, but does not contain iron.In the reverse direction the mechanism appears to involve thegeneration of a strong base by oxygenation of vitamin K.It catalyzes the post-translational modification of several proteinsof the blood-clotting system.9-12 glutamate residues are converted to 4-carboxyglutamate (Gla) ina specific domain of the target protein.','[Peptidyl]-4-carboxyglutamate + 2,3-epoxyphylloquinone + H(2)O =[peptidyl]-glutamate + CO(2) + O(2) + phylloquinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.91','Salicylate decarboxylase','Salicylic acid decarboxylase','NULL','In the reverse direction the enzyme catalyzes the regioselectivecarboxylation of phenol into stoichiometric amounts of salicylate.The enzyme also catalyzes the reversible decarboxylation of 2,4-dihydroxybenzoate, 2,6-dihydroxybenzoate, 2,3-dihydroxybenzoate and4-aminosalicylate.','Salicylate = phenol + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.92','Indole-3-carboxylate decarboxylase','NULL','NULL','Activated by Zn(2+), Mn(2+) or Mg(2+).','Indole-3-carboxylate = indole + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.93','Pyrrole-2-carboxylate decarboxylase','NULL','NULL','The enzyme catalyzes both the carboxylation and decarboxylationreactions.However, while bicarbonate is the preferred substrate for thecarboxylation reaction, decarboxylation produces carbon dioxide.The enzyme is activated by carboxylic acids.','(1) Pyrrole-2-carboxylate = pyrrole + CO(2).(2) Pyrrole-2-carboxylate + H(2)O = pyrrole + HCO(3)(-).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.94','Ethylmalonyl-CoA decarboxylase','NULL','NULL','The enzyme, which exists in all vertebrates, decarboxylatesethylmalonyl-CoA, a potentially toxic compound that is formed in lowamounts by the activity of EC 6.4.1.2 and EC 6.4.1.3.It prefers the S isomer, and can decarboxylate (R)-methylmalonyl-CoAwith lower efficiency.Cf. EC 4.1.1.41.Formerly EC 4.1.1.n2.','(S)-ethylmalonyl-CoA = butanoyl-CoA + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.95','L-glutamyl-[BtrI acyl-carrier protein] decarboxylase','NULL','Pyridoxal 5-phosphate.','Catalyzes a step in the biosynthesis of the side chain of theaminoglycoside antibiotics of the butirosin family.Has very low activity with substrates not bound to an acyl-carrierprotein.','L-glutamyl-[BtrI acyl-carrier protein] = 4-amino butanoyl-[BtrI acyl-carrier protein] + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.96','Carboxynorspermidine decarboxylase','Carboxyspermidine decarboxylase','NULL','Part of a bacterial polyamine biosynthesis pathway.The enzyme is essential for biofilm formation in the bacterium Vibriocholerae.The enzyme from Campylobacter jejuni only produces spermidine in vivoeven though it shows activity with carboxynorspermidine in vitro.','(1) Carboxynorspermidine = bis(3-aminopropyl)amine + CO(2).(2) Carboxyspermidine = spermidine + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.97','2-oxo-4-hydroxy-4-carboxy-5-ureidoimidazoline decarboxylase','OHCU decarboxylase','NULL','This enzyme is part of the pathway from urate to (S)-allantoin, whichis present in bacteria, plants and animals (but not in humans).Formerly EC 4.1.1.n1.','5-hydroxy-2-oxo-4-ureido-2,5-dihydro-1H-imidazole-5-carboxylate = (S)-allantoin + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.n1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.1.n2','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.2','Ketotetrose-phosphate aldolase','Erythrulose-1-phosphate formaldehyde-lyase','NULL','NULL','Erythrulose 1-phosphate = glycerone phosphate + formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.3','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.4','Deoxyribose-phosphate aldolase','2-deoxy-D-ribose-5-phosphate acetaldehyde-lyase','NULL','NULL','2-deoxy-D-ribose 5-phosphate = D-glyceraldehyde 3-phosphate +acetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.5','L-threonine aldolase','L-threonine acetaldehyde-lyase','Pyridoxal 5-phosphate.','This enzyme is specific for L-threonine and can not utilize L-allo-threonine.Different from EC 4.1.2.49 and EC 4.1.2.48.','L-threonine = glycine + acetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.6','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.7','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.8','Indole-3-glycerol-phosphate lyase','(1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate D-glyceraldehyde-3-phosphate-lyase','NULL','Forms part of the defense mechanism against insects and microbialpathogens in the grass family, Gramineae, where it catalyzes one ofthe steps in the formation of the cyclic hydroxamic acids 2,4-dihydroxy-2H-1,4-benzoxazin-3(4H)-one (DIBOA) and 2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one (DIMBOA).Resembles the alpha-subunit of EC 4.2.1.20 but, unlike tryptophansynthase, its activity is independent of the beta-subunit and freeindole is released.','(1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate = indole + D-glyceraldehyde3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.9','Phosphoketolase','D-xylulose-5-phosphate D-glyceraldehyde-3-phosphate-lyase (phosphate-acetylating)','Thiamine diphosphate.','NULL','D-xylulose 5-phosphate + phosphate = acetyl phosphate + D-glyceraldehyde3-phosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.10','(R)-mandelonitrile lyase','(R)-oxynitrilase','Flavoprotein.','A variety of enzymes from different sources and with differentproperties.Some are flavoproteins, others are not.Active toward a number of aromatic and aliphatic hydroxynitriles(cyanohydrins).','(R)-mandelonitrile = cyanide + benzaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.11','Hydroxymandelonitrile lyase','(S)-4-hydroxymandelonitrile hydroxybenzaldehyde-lyase','NULL','Does not accept aliphatic hydroxynitriles, unlike EC 4.1.2.10,EC 4.1.2.46 and EC 4.1.2.47.','(S)-4-hydroxymandelonitrile = cyanide + 4-hydroxybenzaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.12','2-dehydropantoate aldolase','2-dehydropantoate formaldehyde-lyase','NULL','NULL','2-dehydropantoate = 3-methyl-2-oxobutanoate + formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.13','Fructose-bisphosphate aldolase','Aldolase','Zinc.','Also acts on (3S,4R)-ketose 1-phosphates.The enzymes increase electron-attraction by the carbonyl group, some(Class I) forming a protonated imine with it, others (Class II),mainly of microbial origin, polarizing it with a metal ion, e.g.zinc.Formerly EC 4.1.2.7.','D-fructose 1,6-bisphosphate = glycerone phosphate + D-glyceraldehyde3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.14','2-dehydro-3-deoxy-phosphogluconate aldolase','2-dehydro-3-deoxy-D-gluconate-6-phosphate D-glyceraldehyde-3-phosphate-lyase','NULL','The enzyme shows no activity with 2-dehydro-3-deoxy-6-phosphate-D-galactonate (cf. EC 4.1.2.55).Also acts on 2-oxobutanoate.','2-dehydro-3-deoxy-6-phosphate-D-gluconate = pyruvate + D-glyceraldehyde3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.15','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.16','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.17','L-fuculose-phosphate aldolase','Fuculose aldolase','NULL','NULL','L-fuculose 1-phosphate = glycerone phosphate + (S)-lactaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.18','2-dehydro-3-deoxy-L-pentonate aldolase','2-dehydro-3-deoxy-L-pentonate glycolaldehyde-lyase','NULL','NULL','2-dehydro-3-deoxy-L-pentonate = pyruvate + glycolaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.19','Rhamnulose-1-phosphate aldolase','L-rhamnulose 1-phosphate aldolase','NULL','NULL','L-rhamnulose 1-phosphate = glycerone phosphate + (S)-lactaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.20','2-dehydro-3-deoxyglucarate aldolase','2-dehydro-3-deoxy-D-glucarate tartronate-semialdehyde-lyase','NULL','NULL','2-dehydro-3-deoxy-D-glucarate = pyruvate + tartronate semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.21','2-dehydro-3-deoxy-6-phosphogalactonate aldolase','2-dehydro-3-deoxy-D-galactonate-6-phosphate D-glyceraldehyde-3-phosphate-lyase','NULL','The enzyme catalyzes the last reaction in a D-galactose degradationpathway.Cf. EC 4.1.2.55.','2-dehydro-3-deoxy-6-phospho-D-galactonate = pyruvate + D-glyceraldehyde3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.22','Fructose-6-phosphate phosphoketolase','D-fructose-6-phosphate D-erythrose-4-phosphate-lyase (phosphate-acetylating)','Thiamine diphosphate.','Also acts on D-xylulose 5-phosphate.','D-fructose 6-phosphate + phosphate = acetyl phosphate + D-erythrose4-phosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.23','3-deoxy-D-manno-octulosonate aldolase','3-deoxy-D-manno-octulosonate D-arabinose-lyase','NULL','NULL','3-deoxy-D-manno-octulosonate = pyruvate + D-arabinose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.24','Dimethylaniline-N-oxide aldolase','N,N-dimethylaniline-N-oxide formaldehyde-lyase','NULL','Acts on various N,N-dialkylarylamides.','N,N-dimethylaniline N-oxide = N-methylaniline + formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.25','Dihydroneopterin aldolase','2-amino-4-hydroxy-6-(D-erythro-1,2,3-trihydroxypropyl)-7,8-dihydropteridine glycolaldehyde-lyase','NULL','NULL','2-amino-4-hydroxy-6-(D-erythro-1,2,3-trihydroxypropyl)-7,8-dihydropteridine = 2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine+ glycolaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.26','Phenylserine aldolase','L-threo-3-phenylserine benzaldehyde-lyase','Pyridoxal 5-phosphate.','NULL','L-threo-3-phenylserine = glycine + benzaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.27','Sphinganine-1-phosphate aldolase','Sphinganine-1-phosphate palmitaldehyde-lyase','Pyridoxal 5-phosphate.','NULL','Sphinganine 1-phosphate = phosphoethanolamine + palmitaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.28','2-dehydro-3-deoxy-D-pentonate aldolase','2-keto-3-deoxy-D-pentonate aldolase','NULL','NULL','2-dehydro-3-deoxy-D-pentonate = pyruvate + glycolaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.29','5-dehydro-2-deoxyphosphogluconate aldolase','5-dehydro-2-deoxy-D-gluconate-6-phosphate malonate-semialdehyde-lyase','NULL','NULL','5-dehydro-2-deoxy-D-gluconate 6-phosphate = glycerone phosphate +malonate semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.30','17-alpha-hydroxyprogesterone aldolase','17-alpha-hydroxyprogesterone acetaldehyde-lyase','NULL','A microsomal hemeprotein that catalyzes two independent reactions atthe same active site - the 17-hydroxylation of pregnenolone andprogesterone, which is part of glucocorticoid hormones biosynthesis(EC 1.14.99.9), and the conversion of the 17-hydroxylated productsvia a 17,20-lyase reaction to form androstenedione anddehydroepiandrosterone, leading to sex hormone biosynthesis.The enzymes from different organisms differ in their substratespecificity.While the enzymes from pig, hamster, and rat accept both 17-alpha-hydroxyprogesterone and 17-alpha-hydroxypregnenolone, the enzymesfrom human, bovine, sheep, goat, and bison do not accept the former,and the enzyme from guinea pig does not accept the latter.','17-alpha-hydroxyprogesterone = androst-4-ene-3,17-dione + acetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.31','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.32','Trimethylamine-oxide aldolase','Trimethylamine-N-oxide formaldehyde-lyase','NULL','NULL','Trimethylamine N-oxide = dimethylamine + formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.33','Fucosterol-epoxide lyase','(24R,24R)-fucosterol-epoxide acetaldehyde-lyase','NULL','The insect enzyme is involved in the conversion of sitosterol intocholesterol.','(24R,24(1)R)-fucosterol epoxide = desmosterol + acetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.34','4-(2-carboxyphenyl)-2-oxobut-3-enoate aldolase','(3E)-4-(2-carboxyphenyl)-2-oxobut-3-enoate 2-carboxybenzaldehyde-lyase','NULL','Involved, with EC 1.13.11.38, in the metabolism of phenanthrene inbacteria.','(3Z)-4-(2-carboxyphenyl)-2-oxobut-3-enoate + H(2)O = 2-formylbenzoate +pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.35','Propioin synthase','4-hydroxy-3-hexanone aldolase','NULL','NULL','4-hydroxy-3-hexanone = 2 propanal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.36','Lactate aldolase','(S)-lactate acetaldehyde-lyase','NULL','NULL','(S)-lactate = formate + acetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.37','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.38','Benzoin aldolase','2-hydroxy-1,2-diphenylethanone benzaldehyde-lyase','Thiamine diphosphate.','NULL','Benzoin = 2 benzaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.39','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.40','Tagatose-bisphosphate aldolase','D-tagatose-1,6-bisphosphate aldolase','NULL','Enzyme activity is stimulated by certain divalent cations.Involved in the tagatose 6-phosphate pathway of lactose catabolism inbacteria.','D-tagatose 1,6-bisphosphate = glycerone phosphate + D-glyceraldehyde3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.41','Vanillin synthase','NULL','NULL','Involved, together with EC 4.2.1.101 in the production of vanillinfrom trans-ferulic acid.Vanillin is converted to vanillate by EC 1.2.1.67.','3-hydroxy-3-(4-hydroxy-3-methoxyphenyl)propanoyl-CoA = vanillin + acetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.42','D-threonine aldolase','DTA','Pyridoxal 5-phosphate.','Activated by divalent metal cations (e.g. Co(2+), Ni(2+), Mn(2+) orMg(2+)).The reaction is reversible, which can lead to the interconversion ofD-threonine and D-allothreonine.Several other D-beta-hydroxy-alpha-amino acids, such as D-beta-phenylserine, D-beta-hydroxy-alpha-aminovaleric acid and D-beta-3,4-dihydroxyphenylserine, can also act as substrate.','(1) D-threonine = glycine + acetaldehyde.(2) D-allothreonine = glycine + acetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.43','3-hexulose-6-phosphate synthase','3-hexulose phosphate synthase','Manganese or magnesium.','Specific for formaldehyde and D-ribulose 5-phosphate as substrates.Ribose 5-phosphate, xylulose 5-phosphate, allulose 6-phosphate andfructose 6-phosphate cannot act as substrate.This enzyme, along with EC 5.3.1.27, plays a key role in theribulose-monophosphate cycle of formaldehyde fixation, which ispresent in many microorganisms that are capable of utilizingC1-compounds.The hyperthermophilic and anaerobic archaeon Pyrococcus horikoshiiOT3 constitutively produces a bifunctional enzyme that sequentiallycatalyzes the reactions of this enzyme and EC 5.3.1.27.','D-arabino-hex-3-ulose 6-phosphate = D-ribulose 5-phosphate +formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.44','Benzoyl-CoA-dihydrodiol lyase','Dihydrodiol transforming enzyme','NULL','The enzyme is involved in the aerobic benzoyl-CoA catabolic pathwayof the bacterium Azoarcus evansii.Formerly EC 4.1.2.n1.','2,3-dihydro-2,3-dihydroxybenzoyl-CoA + H(2)O = 3,4-dehydroadipyl-CoAsemialdehyde + formate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.45','Trans-o-hydroxybenzylidenepyruvate hydratase-aldolase','2-hydroxybenzalpyruvate aldolase','NULL','This enzyme is involved in naphthalene degradation.The enzyme catalyzes a retro-aldol reaction in vitro, and it acceptsa broad range of aldehydes and 4-substituted 2-oxobut-3-enoates assubstrates.','(3E)-4-(2-hydroxyphenyl)-2-oxobut-3-enoate + H(2)O = salicylaldehyde +pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.46','Aliphatic (R)-hydroxynitrile lyase','(R)-hydroxynitrile lyase','Zinc.','The enzyme catalyzes the stereoselective synthesis of aliphatic (R)-cyanohydrins.No activity toward mandelonitrile and 4-hydroxymandelonitrile.Natural substrates for the (R)-oxynitrilase from Linum usitatissimumare acetone and butan-2-one, which are the building blocks of thecyanogen glycosides in Linum, linamarin and lotaustralin,or linustatin and neolinustatin, respectively.Formerly EC 4.1.2.37 and EC 4.1.2.39.','(2R)-2-hydroxy-2-methylbutanenitrile = cyanide + butan-2-one.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.47','(S)-hydroxynitrile lyase','(S)-cyanohydrin producing hydroxynitrile lyase','NULL','Hydroxynitrile lyases catalyzes the the cleavage of hydroxynitrilesinto cyanide and the corresponding aldehyde or ketone.In nature the liberation of cyanide serves as a defense mechanismagainst herbivores and microbial attack in plants.In vitro the enzymes from Manihot esculenta and Hevea brasiliensisaccept a broad range of aliphatic and aromatic carbonyl compounds assubstrates and catalyze the formation of (S)-hydroxynitriles.Formerly EC 4.1.2.37 and EC 4.1.2.39.','(1) An aliphatic (S)-hydroxynitrile = cyanide + an aliphatic aldehyde orketone.(2) An aromatic (S)-hydroxynitrile = cyanide + an aromatic aldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.48','Low-specificity L-threonine aldolase','NULL','Pyridoxal 5-phosphate.','The low-specificity L-threonine aldolase can act on both L-threonineand L-allo-threonine.The enzyme from Escherichia coli can also act on L-threo-phenylserineand L-erythro-phenylserine.The enzyme can also catalyze the aldol condensation of glycolaldehydeand glycine to form 4-hydroxy-L-threonine, an intermediate ofpyridoxal phosphate biosynthesis.Different from EC 4.1.2.5 and EC 4.1.2.49.','(1) L-threonine = glycine + acetaldehyde.(2) L-allo-threonine = glycine + acetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.49','L-allo-threonine aldolase','NULL','Pyridoxal 5-phosphate.','This enzyme, characterized from the bacterium Aeromonas jandaei,is specific for L-allo-threonine and can not act on eitherL-threonine or L-serine.Different from EC 4.1.2.5 and EC 4.1.2.48.A previously listed enzyme with this name, EC 4.1.2.6, was deleted in1971 after it was found to be identical to EC 2.1.2.1.','L-allo-threonine = glycine + acetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.50','6-carboxytetrahydropterin synthase','CPH4 synthase','Zinc.','Isolated from the bacteria Bacillus subtilis and Escherichia coli.The reaction is part of the biosynthesis pathway of queuosine.The enzyme from Escherichia coli can also convert 6-pyruvoyl-5,6,7,8-tetrahydropterin and sepiapterin to 6-carboxy-5,6,7,8-tetrahydropterin.','7,8-dihydroneopterin 3-triphosphate + H(2)O = 6-carboxy-5,6,7,8-tetrahydropterin + acetaldehyde + triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.51','2-dehydro-3-deoxy-D-gluconate aldolase','KDG-specific aldolase','NULL','The enzyme from the archaeon Picrophilus torridus is involved inD-glucose and D-galactose catabolism via the nonphosphorylativevariant of the Entner-Doudoroff pathway.In the direction of aldol synthesis the enzyme catalyzes theformation of 2-dehydro-3-deoxy-D-gluconate and 2-dehydro-3-deoxy-D-galactonate at a similar ratio.It shows no activity with 2-dehydro-3-deoxy-D-gluconate 6-phosphate.Cf. EC 4.1.2.14.','2-dehydro-3-deoxy-D-gluconate = pyruvate + D-glyceraldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.52','4-hydroxy-2-oxoheptanedioate aldolase','2,4-dihydroxyhept-2-enedioate aldolase','Manganese or cobalt.','The enzyme is also able to catalyze the aldol cleavage of 4-hydroxy-2-oxopentanoate and 4-hydroxy-2-oxohexanoate, and can use2-oxobutanoate as carbonyl donor, with lower efficiency.In the reverse direction, is able to condense a range of aldehydeacceptors with pyruvate.The enzyme from the bacterium Escherichia coli produces a racemicmixture of (4R)- and (4S)-hydroxy-2-oxoheptanedioate.Formerly EC 4.1.2.n4.','4-hydroxy-2-oxoheptanedioate = pyruvate + succinate semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.53','2-keto-3-deoxy-L-rhamnonate aldolase','2-dehydro-3-deoxyrhamnonate aldolase','Magnesium.','The enzyme can also use 2-oxo-3-deoxy-L-mannonate, 2-oxo-3-deoxy-L-lyxonate and 4-hydroxy-2-ketoheptane-1,7-dioate (HKHD) as substrates.Formerly EC 4.1.2.n3.','2-dehydro-3-deoxy-L-rhamnonate = pyruvate + (S)-lactaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.54','L-threo-3-deoxy-hexylosonate aldolase','NULL','NULL','The enzyme takes part in a D-galacturonate degradation pathway in thefungi Aspergillus niger and Trichoderma reesei (Hypocrea jecorina).','2-dehydro-3-deoxy-L-galactonate = pyruvate + L-glyceraldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.55','2-dehydro-3-deoxy-phosphogluconate/2-dehydro-3-deoxy-6-phosphogalactonatealdolase','2-keto-3-deoxygluconate aldolase','NULL','In the archaeon Sulfolobus solfataricus the enzyme is involved inglucose and galactose catabolism via the branched variant of theEntner-Doudoroff pathway.It utilizes 2-dehydro-3-deoxy-6-phosphate-D-gluconate and 2-dehydro-3-deoxy-6-phosphate-D-galactonate with similar catalytic efficiency.In vitro the enzyme can also catalyze the cleavage of the non-phosphorylated forms 2-dehydro-3-deoxy-D-gluconate and 2-dehydro-3-deoxy-D-galactonate with much lower catalytic efficiency.Cf. EC 4.1.2.21 and EC 4.1.2.14.','(1) 2-dehydro-3-deoxy-6-phospho-D-gluconate = pyruvate + D-glyceraldehyde3-phosphate.(2) 2-dehydro-3-deoxy-6-phospho-D-galactonate = pyruvate +D-glyceraldehyde 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.56','2-amino-4,5-dihydroxy-6-oxo-7-(phosphonooxy)heptanoate synthase','NULL','NULL','Part of the pathway for the biosynthesis of grixazone, a mixture ofyellow pigments produced by the bacterium Streptomyces griseus.Formerly EC 4.1.2.n6.','2-amino-4,5-dihydroxy-6-oxo-7-(phosphonooxy)heptanoate = glyceronephosphate + L-aspartate 4-semialdehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.n1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.n2','2-hydroxyphytanoyl-CoA lyase','2-HPCL','Magnesium; Thiamine diphosphate.','The enzyme can also use other 3-methyl-branched fatty acids assubstrates.','2-hydroxyphytanoyl-CoA = formyl-CoA + pristanal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.n3','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.n4','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.n5','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.2.n6','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.1','Isocitrate lyase','ICL','NULL','The isomer of isocitrate involved is (1R,2S)-1-hydroxypropane-1,2,3-tricarboxylate.','Isocitrate = succinate + glyoxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.2','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.3','N-acetylneuraminate lyase','N-acetylneuraminate pyruvate-lyase','NULL','Also acts on N-glycoloylneuraminate, and on O-acetylated sialicacids, other than 4-O-acetylated derivatives.','N-acetylneuraminate = N-acetyl-D-mannosamine + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.4','Hydroxymethylglutaryl-CoA lyase','(S)-3-hydroxy-3-methylglutaryl-CoA acetoacetate-lyase','NULL','NULL','(S)-3-hydroxy-3-methylglutaryl-CoA = acetyl-CoA + acetoacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.5','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.6','Citrate (pro-3S)-lyase','Citrase','NULL','Can be dissociated into components, two of which are identical withEC 2.8.3.10 and EC 4.1.3.34.EC 3.1.2.16 deacetylates and inactivates the enzyme.','Citrate = acetate + oxaloacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.7','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.8','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.9','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.10','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.11','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.12','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.13','Oxalomalate lyase','3-oxalomalate glyoxylate-lyase','NULL','NULL','3-oxalomalate = oxaloacetate + glyoxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.14','L-erythro-3-hydroxyaspartate aldolase','Erythro-3-hydroxy-L(s)-aspartate glyoxylate-lyase','NULL','NULL','L-erythro-3-hydroxy-aspartate = glycine + glyoxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.15','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.16','4-hydroxy-2-oxoglutarate aldolase','2-keto-4-hydroxyglutarate aldolase','NULL','The enzymes from rat liver and bovine liver act on both enantiomersof 4-hydroxy-2-oxoglutarate.Formerly EC 4.1.2.1 and EC 4.1.2.31.','4-hydroxy-2-oxoglutarate = pyruvate + glyoxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.17','4-hydroxy-4-methyl-2-oxoglutarate aldolase','4-carboxy-4-hydroxy-2-oxoadipate aldolase','Divalent cation.','This enzyme participates in the degradation of protocatechuate (viathe meta-cleavage pathway), phthalate, syringate and gallate.The enzyme from Pseudomonas ochraceae can also cleave 4-hydroxy-2-oxoglutarate to glyoxylate and pyruvate, and also catalyzes thereaction of EC 4.1.1.3.','(1) 4-hydroxy-4-methyl-2-oxoglutarate = 2 pyruvate.(2) 2-hydroxy-4-oxobutane-1,2,4-tricarboxylate = oxaloacetate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.18','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.19','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.20','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.21','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.22','Citramalate lyase','(3S)-citramalate pyruvate-lyase','NULL','Can be dissociated into components, two of which are identical withEC 2.8.3.11 and EC 4.1.3.25.','(3S)-citramalate = acetate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.23','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.24','Malyl-CoA lyase','(3S)-3-carboxy-3-hydroxypropanoyl-CoA glyoxylate-lyase','NULL','The enzyme from the bacterium Chloroflexus aurantiacus, whichparticipates in the 3-hydroxypropanoate cycle for carbonassimilation, also has the activity of EC 4.1.3.25.The enzymes from Rhodobacter species are part of acetate assimilationpathways.The reactions are reversible.','(1) (S)-malyl-CoA = acetyl-CoA + glyoxylate.(2) (2R,3S)-2-methylmalyl-CoA = propanoyl-CoA + glyoxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.25','(S)-citramalyl-CoA lyase','(3S)-citramalyl-CoA pyruvate-lyase','Magnesium.','The enzyme from the bacterium Clostridium tetanomorphum is acomponent of EC 4.1.3.22.It also acts on (3S)-citramalyl thioacyl-carrier protein.The enzyme from the bacterium Chloroflexus aurantiacus also has theactivity of EC 4.1.3.24.It has no activity with (3R)-citramalyl-CoA (cf. EC 4.1.3.46).','(3S)-citramalyl-CoA = acetyl-CoA + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.26','3-hydroxy-3-isohexenylglutaryl-CoA lyase','3-hydroxy-3-(4-methylpent-3-en-1-yl)glutaryl-CoA acetate-lyase','NULL','Also acts on the hydroxy derivative of farnesoyl-CoA.','3-hydroxy-3-(4-methylpent-3-en-1-yl)glutaryl-CoA = 7-methyl-3-oxooct-6-enoyl-CoA + acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.27','Anthranilate synthase','NULL','NULL','In some organisms, this enzyme is part of a multifunctional proteintogether with one or more components of the system for biosynthesisof tryptophan (EC 2.4.2.18, EC 4.1.1.48, EC 4.2.1.20,and EC 5.3.1.24).The native enzyme in the complex with uses either glutamine or (lessefficiently) NH(3).The enzyme separated from the complex uses NH(3) only.','Chorismate + L-glutamine = anthranilate + pyruvate + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.28','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.29','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.30','Methylisocitrate lyase','(2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate pyruvate-lyase','NULL','Acts on threo-D(s)-2-methylisocitrate, but not on threo-D(s)-isocitrate, threo-DL-isocitrate or erythro-L(s)-isocitrate.','(2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate = pyruvate + succinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.31','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.32','2,3-dimethylmalate lyase','(2R,3S)-2,3-dimethylmalate pyruvate-lyase','NULL','Forms part of the nicotinate-fermentation catabolism pathway inEubacterium barkeri.Other enzymes involved in this pathway are EC 1.17.5.1, EC 1.3.7.1,EC 3.5.2.18, EC 1.1.1.291, EC 5.4.99.4, EC 5.3.3.6 and EC 4.2.1.85.','(2R,3S)-2,3-dimethylmalate = propanoate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.33','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.34','Citryl-CoA lyase','(3S)-citryl-CoA oxaloacetate-lyase','NULL','Component of EC 4.1.3.6 and EC 4.1.3.8.Also acts on (3S)-citryl thioacyl-carrier protein.','(3S)-citryl-CoA = acetyl-CoA + oxaloacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.35','(1-hydroxycyclohexan-1-yl)acetyl-CoA lyase','(1-hydroxycyclohexan-1-yl)acetyl-CoA cyclohexanone-lyase','NULL','NULL','(1-hydroxycyclohexan-1-yl)acetyl-CoA = acetyl-CoA + cyclohexanone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.36','1,4-dihydroxy-2-naphthoyl-CoA synthase','DHNA synthetase','NULL','This enzyme is involved in the synthesis of 1,4-dihydroxy-2-naphthoate, a branch point metabolite leading to the biosynthesis ofmenaquinone (vitamin K(2), in bacteria), phylloquinone (vitamin K(1)in plants), and many plant pigments.The coenzyme A group is subsequently removed from the product byEC 3.1.2.28.','4-(2-carboxyphenyl)-4-oxobutanoyl-CoA = 1,4-dihydroxy-2-naphthoyl-CoA +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.37','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.38','Aminodeoxychorismate lyase','4-amino-4-deoxychorismate lyase','Pyridoxal 5-phosphate.','Forms part of the folate biosynthesis pathway.Acts on 4-amino-4-deoxychorismate, the product of EC 2.6.1.85,to form p-aminobenzoate.','4-amino-4-deoxychorismate = 4-aminobenzoate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.39','4-hydroxy-2-oxovalerate aldolase','4-hydroxy-2-ketovalerate aldolase','Manganese.','The enzyme from the bacterium Pseudomonas putida is also stimulatedby NADH.In some bacterial species, the enzyme forms a bifunctional complexwith EC 1.2.1.10.The enzymes from the bacteria Burkholderia xenovorans and Thermusthermophilus also perform the reaction of EC 4.1.3.43.','(S)-4-hydroxy-2-oxopentanoate = acetaldehyde + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.40','Chorismate lyase','CL','NULL','Catalyzes the first step in the biosynthesis of ubiquinone inEscherichia coli and other Gram-negative bacteria.The yeast Saccharomyces cerevisiae can synthesize ubiquinone fromeither chorismate or tyrosine.','Chorismate = 4-hydroxybenzoate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.41','3-hydroxy-D-aspartate aldolase','D-3-hydroxyaspartate aldolase','Pyridoxal 5-phosphate.','The enzyme, purified from the bacterium Paracoccus denitrificansy IFO13301, is strictly D-specific as to the alpha-position of thesubstrate, but accepts both the threo and erythro forms at the beta-position.The erythro form is a far better substrate (about 100 fold).The enzyme can also accept D-allothreonine, D-threonine, erythro-3-phenyl-D-serine and threo-3-phenyl-D-serine.Different from EC 4.1.3.14.Requires a divalent cation, such as Mg(2+), Mn(2+) or Co(2+).','(1) Threo-3-hydroxy-D-aspartate = glycine + glyoxylate.(2) D-erythro-3-hydroxyaspartate = glycine + glyoxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.42','(4S)-4-hydroxy-2-oxoglutarate aldolase','2-keto-4-hydroxyglutaric aldolase','NULL','The enzyme from the bacterium Escherichia coli is specific for the(S) enantiomer.That enzyme is trifunctional, and also catalyzes the reactions ofEC 4.1.1.3 and EC 4.1.2.14.Cf. EC 4.1.3.16.','(4S)-4-hydroxy-2-oxoglutarate = pyruvate + glyoxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.43','4-hydroxy-2-oxohexanoate aldolase','NULL','Manganese.','The enzyme from the bacteria Burkholderia xenovorans and Thermusthermophilus also perform the reaction of EC 4.1.3.39.The enzymes form a bifunctional complex with EC 1.2.1.89 with a tightchannel connecting the two subunits.Formerly EC 4.1.3.n1.','(S)-4-hydroxy-2-oxohexanoate = propanal + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.44','tRNA 4-demethylwyosine synthase (AdoMet-dependent)','NULL','Iron-sulfur.','This enzyme, which is a member of the superfamily of S-adenosyl-L-methionine-dependent radical (radical AdoMet) enzymes, binds two[4Fe-4S] clusters.Carbons C(2) and C(3) from pyruvate are incorporated into4-demethylwyosine.The enzyme is found in eukaryotes where it is part of the pathway forwybutosine synthesis, and in archaea, where it is involved in thebiosynthesis of archaeal wye bases, such as wyosine, isowyosine,and methylwyosine.','N(1)-methylguanine(37) in tRNA(Phe) + pyruvate + S-adenosyl-L-methionine= 4-demethylwyosine(37) in tRNA(Phe) + L-methionine + 5-deoxyadenosine +CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.45','3-hydroxybenzoate synthase','Chorismatase/3-hydroxybenzoate synthase','NULL','The enzyme, found in several bacterial species is involved inbiosynthesis of secondary products.The enzyme from the bacterium Xanthomonas campestris pv. campestrisalso has the activity of EC 4.1.3.40.','Chorismate = 3-hydroxybenzoate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.46','(R)-citramalyl-CoA lyase','NULL','Manganese.','The enzyme, purified from the bacterium Chloroflexus aurantiacus,has no activity with (3S)-citramalyl-CoA (cf. EC 4.1.3.25).','(3R)-citramalyl-CoA = acetyl-CoA + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.3.n1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.1','Tryptophanase','L-tryptophan indole-lyase','Potassium; Pyridoxal 5-phosphate.','Also catalyzes 2,3-elimination and beta-replacement reactions of someindole-substituted tryptophan analogs of L-cysteine, L-serine andother 3-substituted amino acids.','L-tryptophan + H(2)O = indole + pyruvate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.2','Tyrosine phenol-lyase','Beta-tyrosinase','Pyridoxal 5-phosphate.','Also slowly catalyzes pyruvate formation from D-tyrosine, S-methyl-L-cysteine, L-cysteine, L-serine and D-serine.','L-tyrosine + H(2)O = phenol + pyruvate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.3','Deoxyribodipyrimidine photo-lyase','DNA photolyase','5,10-methenyltetrahydrofolate; FAD.','A flavoprotein containing a second chromophore group.Catalyzes the reactivation by light of irradiated DNA.A similar reactivation of irradiated RNA is probably due to aseparate enzyme.','Cyclobutadipyrimidine (in DNA) = 2 pyrimidine residues (in DNA).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.4','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.5','Aldehyde oxygenase (deformylating)','Aldehyde decarbonylase','Iron.','Involved in the biosynthesis of alkanes.The enzyme from the cyanobacterium Nostoc punctiforme PCC 73102 isonly active in vitro in the presence of ferredoxin, ferredoxinreductase and NADPH, and produces mostly C(15) to C(17) alkanes.The enzyme from pea (Pisum sativum) produces alkanes of chain lengthC18 to C32 and is inhibited by metal-chelating agents.The substrate for this enzyme is formed by EC 1.2.1.80.','A long-chain aldehyde + O(2) + 2 NADPH = an alkane + formate + H(2)O +2 NADP(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.6','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.7','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.8','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.9','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.10','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.11','Benzylsuccinate synthase','NULL','NULL','A glycyl radical enzyme that is inhibited by benzyl alcohol,benzaldehyde, phenylhydrazine and is inactivated by oxygen.','Toluene + fumarate = benzylsuccinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.12','3,4-dihydroxy-2-butanone-4-phosphate synthase','DHBP synthase','Magnesium.','The reaction involves an intramolecular skeletal rearrangement, withthe bonds in D-ribulose 5-phosphate that connect C-3 and C-5 to C-4being broken, C-4 being removed as formate and reconnection of C-3and C-5.The phosphorylated four-carbon product (L-3,4-dihydroxybutan-2-one4-phosphate) is an intermediate in the biosynthesis of riboflavin.','D-ribulose 5-phosphate = formate + L-3,4-dihydroxybutan-2-one4-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.13','(6-4)DNA photolyase','DNA photolyase','FAD.','The overall repair reaction consists of two distinct steps, one ofwhich is light-independent and the other one light-dependent.In the initial light-independent step, a 6-iminium ion is thought tobe generated via proton transfer induced by two histidines highlyconserved among the (6-4) photolyases.This intermediate spontaneously rearranges to form an oxetaneintermediate by intramolecular nucleophilic attack.In the subsequent light-driven reaction, one electron is believed tobe transferred from the fully reduced FAD cofactor (FADH(-)) to theoxetane intermediate thus forming a neutral FADH radical and ananionic oxetane radical, which spontaneously fractures.The excess electron is then back-transferred to the flavin radicalrestoring the fully reduced flavin cofactor and a pair of pyrimidinebases.','(6-4) photoproduct (in DNA) = 2 pyrimidine residues (in DNA).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.14','Spore photoproduct lyase','SP lyase','Iron-sulfur; S-adenosyl-L-methionine.','This enzyme is a member of the AdoMet radical (radical SAM) family.The enzyme binds a [4Fe-4S]  cluster.The cluster is coordinated by 3 cysteines and an exchangeable SAMmolecule.The 5-deoxy-adenosine radical formed after electron transfer fromthe [4Fe-4S] cluster to the S-adenosyl-L-methionine, initiates therepair by abstracting the C-6 hydrogen of the spore photoproductlesion.During the second part of the repair process the SAM molecule isregenerated.','(5R)-5,6-dihydro-5-(thymidin-7-yl)thymidine (in double-helical DNA) =thymidylyl-(3-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.15','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.16','Geosmin synthase','NULL','Magnesium.','Geosmin is the cause of the characteristic smell of moist soil.It is a bifunctional enzyme.The N-terminal part of the enzyme is EC 4.2.3.22, germacradienolsynthase, and forms germacradienol from FPP.The C-terminal part of the enzyme catalyzes the conversion ofgermacradienol to geosmin via (1S,4aS,8aS)-8,10-dimethyl-1,2,3,4,4a,5,6,8a-octahydronaphthalene.','(1E,4S,5E,7R)-germacra-1(10),5-dien-11-ol + H(2)O = (-)-geosmin +acetone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.17','Phosphomethylpyrimidine synthase','NULL','Iron-sulfur.','Binds a [4Fe-4S] cluster that is coordinated by 3 cysteines and anexchangeable S-adenosyl-L-methionine molecule.The first stage of catalysis is reduction of the S-adenosyl-L-methionine to produce L-methionine and a 5-deoxyadenosin-5-ylradical that is crucial for the conversion of the substrate.Part of the pathway for thiamine biosynthesis.','5-amino-1-(5-phospho-D-ribosyl)imidazole + S-adenosyl-L-methionine =4-amino-2-methyl-5-phosphomethylpyrimidine + 5-deoxyadenosine +L-methionine + formate + CO.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.18','Cyclic pyranopterin phosphate synthase','Molybdenum cofactor biosynthesis protein 1','Iron-sulfur; S-adenosyl-L-methionine.','The enzyme catalyzes an early step in the biosynthesis of themolybdenum cofactor (MoCo).The enzyme MoaA from bacteria and the human enzyme MOCS1A eachcontain two oxygen-sensitive FeS clusters.The enzyme is a member of the superfamily of S-adenosyl-L-methionine-dependent radical (radical AdoMet) enzymes.In bacteria, the reaction is catalyzed by MoaA and requires theaction of MoaC.The latter protein is equivalent to the C-terminal domain of theeukaryotic enzyme MOCS1A which does not need further proteincomponents to perform the reaction.','GTP = cyclic pyranopterin phosphate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.19','2-iminoacetate synthase','NULL','Iron-sulfur.','Binds a [4Fe-4S] cluster that is coordinated by 3 cysteines and anexchangeable S-adenosyl-L-methionine molecule.The first stage of catalysis is reduction of the S-adenosyl-L-methionine to produce methionine and a 5-deoxyadenosin-5-yl radicalthat is crucial for the conversion of the substrate.Part of the pathway for thiamine biosynthesis.Formerly EC 1.3.99.n2.','L-tyrosine + S-adenosyl-L-methionine + reduced acceptor = 2-iminoacetate+ 4-methylphenol + 5-deoxyadenosine + L-methionine + acceptor + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.1.99.20','3-amino-4-hydroxybenzoate synthase','3,4-AHBA synthase','Manganese.','The reaction is suggested to take place in several steps.Schiff base formation, double bond migration and dephosphorylationfollowed by ring opening and closing to form a pyrrolidine ring,and finally dehydration to form the product 3-amino-4-hydroxybenzoate.In the bacterium Streptomyces griseus the enzyme is involved inbiosynthesis of grixazone, a yellow pigment that contains aphenoxazinone chromophore.Formerly EC 4.3.3.n1.','2-amino-4,5-dihydroxy-6-oxo-7-(phosphonooxy)heptanoate = 3-amino-4-hydroxybenzoate + phosphate + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.1','Carbonate dehydratase','Carbonate hydro-lyase','Zinc.','NULL','H(2)CO(3) = CO(2) + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.2','Fumarate hydratase','Fumarase','NULL','NULL','(S)-malate = fumarate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.3','Aconitate hydratase','Aconitase','Iron-sulfur.','Besides interconverting citrate and cis-aconitate, it alsointerconverts cis-aconitate with isocitrate and, hence, interconvertscitrate and isocitrate.The equilibrium mixture is 91% citrate, 6% isocitrate and 3%aconitate.Cis-aconitate is used to designate the isomer (Z)-prop-1-ene-1,2,3-tricarboxylate.Formerly EC 4.2.1.4.','Citrate = isocitrate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.4','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.5','Arabinonate dehydratase','D-arabinonate hydro-lyase','NULL','NULL','D-arabinonate = 2-dehydro-3-deoxy-D-arabinonate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.6','Galactonate dehydratase','NULL','NULL','The enzyme shows no activity with D-gluconate.Cf. EC 4.2.1.140.','D-galactonate = 2-dehydro-3-deoxy-D-galactonate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.7','Altronate dehydratase','NULL','NULL','NULL','D-altronate = 2-dehydro-3-deoxy-D-gluconate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.8','Mannonate dehydratase','Altronate hydrolase','NULL','NULL','D-mannonate = 2-dehydro-3-deoxy-D-gluconate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.9','Dihydroxy-acid dehydratase','NULL','NULL','NULL','2,3-dihydroxy-3-methylbutanoate = 3-methyl-2-oxobutanoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.10','3-dehydroquinate dehydratase','NULL','NULL','NULL','3-dehydroquinate = 3-dehydroshikimate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.11','Phosphopyruvate hydratase','2-phosphoglycerate dehydratase','Magnesium.','Also acts on 3-phospho-D-erythronate.','2-phospho-D-glycerate = phosphoenolpyruvate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.12','Phosphogluconate dehydratase','NULL','NULL','NULL','6-phospho-D-gluconate = 2-dehydro-3-deoxy-6-phospho-D-gluconate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.13','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.14','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.15','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.16','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.17','Enoyl-CoA hydratase','Enoyl hydrase','NULL','Acts in the reverse direction.With cis-compounds, yields (3R)-3-hydroxyacyl-CoA (cf. EC 4.2.1.74).','(3S)-3-hydroxyacyl-CoA = trans-2(or 3)-enoyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.18','Methylglutaconyl-CoA hydratase','NULL','NULL','NULL','(S)-3-hydroxy-3-methylglutaryl-CoA = trans-3-methylglutaconyl-CoA +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.19','Imidazoleglycerol-phosphate dehydratase','NULL','NULL','NULL','D-erythro-1-(imidazol-4-yl)glycerol 3-phosphate = 3-(imidazol-4-yl)-2-oxopropyl phosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.20','Tryptophan synthase','Indoleglycerol phosphate aldolase','Pyridoxal 5-phosphate.','The alpha-subunit catalyzes the conversion of 1-C-(indol-3-yl)glycerol 3-phosphate to indole and D-glyceraldehyde 3-phosphate(this reaction was listed formerly as EC 4.1.2.8).The indole migrates to the beta-subunit where, in the presence ofpyridoxal 5-phosphate, it is combined with L-serine to formL-tryptophan.In some organisms this enzyme is part of a multifunctional proteinthat also includes one or more of the enzymes EC 2.4.2.18,EC 4.1.1.48, EC 4.1.3.27 and EC 5.3.1.24.In thermophilic organisms, where the high temperature enhancesdiffusion and causes the loss of indole, a protein similar to thebeta subunit can be found (EC 4.2.1.122).That enzyme cannot combine with the alpha unit of EC 4.2.1.20 to forma complex.','L-serine + 1-C-(indol-3-yl)glycerol 3-phosphate = L-tryptophan +D-glyceraldehyde 3-phosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.21','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.22','Cystathionine beta-synthase','Beta-thionase','Pyridoxal 5-phosphate.','A multifunctional enzyme: catalyzes beta-replacement reaction betweenL-serine, L-cysteine, cysteine thioethers or some other beta-substituted alpha-L-amino acids and a variety of mercaptans.Formerly EC 4.2.1.21.','L-serine + L-homocysteine = L-cystathionine + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.23','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.24','Porphobilinogen synthase','Aminolevulinate dehydratase','Zinc.','The enzyme catalyzes the asymmetric condensation and cyclization oftwo 5-aminolevulinate molecules, which is the first common step inthe biosynthesis of tetrapyrrole pigments such as porphyrin,chlorophyll, vitamin B12, siroheme, phycobilin, and cofactor F430.The enzyme is widespread, being essential in organisms that carry outrespiration, photosynthesis, or methanogenesis.In humans, the enzyme is a primary target for the environmental toxinPb.The enzymes from some organisms utilize a dynamic equilibrium betweenarchitecturally distinct multimeric assemblies as a means forallosteric regulation.','2 5-aminolevulinate = porphobilinogen + 2 H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.25','L-arabinonate dehydratase','L-arabinonate hydro-lyase','NULL','NULL','L-arabinonate = 2-dehydro-3-deoxy-L-arabinonate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.26','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.27','Acetylenecarboxylate hydratase','Acetylenemonocarboxylate hydratase','NULL','The reaction is effectively irreversible, favoring oxopropanoate(malonic semialdehyde) and its tautomers.Also acts on but-3-ynoate forming acetoacetate.The mechanism appears to involve hydration of the acetylene to3-hydroxypropenoate, which will spontaneously tautomerize to3-oxopropanoate.It is thus analogous to EC 4.1.1.78 in its mechanism.Formerly EC 4.2.1.71.','3-oxopropanoate = propynoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.28','Propanediol dehydratase','NULL','Cobalamin.','Also dehydrates ethylene glycol to acetaldehyde.','Propane-1,2-diol = propanal + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.29','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.30','Glycerol dehydratase','NULL','Cobalamin.','NULL','Glycerol = 3-hydroxypropanal + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.31','Maleate hydratase','NULL','NULL','NULL','(R)-malate = maleate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.32','L(+)-tartrate dehydratase','NULL','Iron.','Exists in an inactive low-molecular form, which is converted intoactive enzyme in the presence of Fe(2+) and thiol (cf. EC 4.2.1.81).','(R,R)-tartrate = oxaloacetate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.33','3-isopropylmalate dehydratase','(2R,3S)-3-isopropylmalate hydro-lyase','Iron-sulfur.','Forms part of the leucine-biosynthesis pathway.Brings about the interconversion of the two isomers ofisopropylmalate.','(2R,3S)-3-isopropylmalate = (2S)-2-isopropylmalate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.34','(S)-2-methylmalate dehydratase','Mesaconate hydratase','NULL','Also hydrates fumarate to (S)-malate.','(S)-2-methylmalate = 2-methylfumarate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.35','(R)-2-methylmalate dehydratase','Citraconate hydratase','Iron.','NULL','(R)-2-methylmalate = 2-methylmaleate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.36','Homoaconitate hydratase','2-hydroxybutane-1,2,4-tricarboxylate hydro-lyase','Iron-sulfur.','The enzyme from the hyperthermophilic eubacterium Thermusthermophilus can catalyze the reaction shown above but cannotcatalyze the previously described reaction, i.e. formation ofhomocitrate by hydration of cis-homoaconitate.The enzyme responsible for the conversion of cis-homoaconitate intohomocitrate in T.thermophilus is unknown at present but the reactioncan be catalyzed in vitro using EC 4.2.1.3 from pig.','(1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate = (Z)-but-1-ene-1,2,4-tricarboxylate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.37','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.38','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.39','Gluconate dehydratase','NULL','NULL','The enzyme shows no activity with D-galactonate.Cf. EC 4.2.1.140.','D-gluconate = 2-dehydro-3-deoxy-D-gluconate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.40','Glucarate dehydratase','NULL','NULL','NULL','D-glucarate = 5-dehydro-4-deoxy-D-glucarate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.41','5-dehydro-4-deoxyglucarate dehydratase','5-keto-4-deoxy-glucarate dehydratase','NULL','NULL','5-dehydro-4-deoxy-D-glucarate = 2,5-dioxopentanoate + H(2)O + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.42','Galactarate dehydratase','NULL','NULL','NULL','D-galactarate = 5-dehydro-4-deoxy-D-glucarate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.43','2-dehydro-3-deoxy-L-arabinonate dehydratase','2-keto-3-deoxy-L-arabinonate dehydratase','NULL','NULL','2-dehydro-3-deoxy-L-arabinonate = 2,5-dioxopentanoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.44','Myo-inosose-2 dehydratase','NULL','Manganese or cobalt.','NULL','2,4,6/3,5-pentahydroxycyclohexanone = 3,5/4-trihydroxycyclohexa-1,2-dione+ H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.45','CDP-glucose 4,6-dehydratase','NULL','NAD(+).','NULL','CDP-glucose = CDP-4-dehydro-6-deoxy-D-glucose + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.46','dTDP-glucose 4,6-dehydratase','NULL','NAD(+).','NULL','dTDP-alpha-D-glucose = dTDP-4-dehydro-6-deoxy-alpha-D-glucose + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.47','GDP-mannose 4,6-dehydratase','GDP-D-mannose 4,6-dehydratase','NAD(+).','Forms the first step in the biosynthesis of GDP-alpha-D-rhamnose andGDP-alpha-L-fucose.In Aneurinibacillus thermoaerophilus L420-91T, this enzyme acts as abifunctional enzyme, catalyzing the above reaction as well as thereaction catalyzed by EC 1.1.1.281.','GDP-alpha-D-mannose = GDP-4-dehydro-alpha-D-rhamnose + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.48','D-glutamate cyclase','NULL','NULL','Also acts on various derivatives of D-glutamate.','D-glutamate = 5-oxo-D-proline + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.49','Urocanate hydratase','Imidazolonepropionate hydrolase','NAD(+).','NULL','3-(5-oxo-4,5-dihydro-3H-imidazol-4-yl)propanoate = urocanate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.50','Pyrazolylalanine synthase','NULL','Pyridoxal 5-phosphate.','NULL','L-serine + pyrazole = 3-(pyrazol-1-yl)-L-alanine + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.51','Prephenate dehydratase','NULL','NULL','This enzyme in the enteric bacteria also possesses EC 5.4.99.5activity and converts chorismate into prephenate.','Prephenate = phenylpyruvate + H(2)O + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.52','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.53','Oleate hydratase','NULL','NULL','Acts on a number of 10-hydroxy acids.','(R)-10-hydroxystearate = oleate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.54','Lactoyl-CoA dehydratase','NULL','NULL','A bacterial enzyme that is involved in propanoate fermentation (alsoknown as the acrylate pathway).','(R)-lactoyl-CoA = acryloyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.55','3-hydroxybutyryl-CoA dehydratase','Crotonase','NULL','Also acts on crotonoyl thioesters of pantetheine and acyl-carrierprotein.','(3R)-3-hydroxybutanoyl-CoA = crotonoyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.56','Itaconyl-CoA hydratase','NULL','NULL','NULL','Citramalyl-CoA = itaconyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.57','Isohexenylglutaconyl-CoA hydratase','NULL','NULL','Also acts on dimethylacryloyl-CoA and farnesoyl-CoA.','3-hydroxy-3-(4-methylpent-3-en-1-yl)glutaryl-CoA = 3-(4-methylpent-3-en-1-yl)pent-2-enedioyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.58','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.59','3-hydroxyacyl-[acyl-carrier-protein] dehydratase','(3R)-3-hydroxyoctanoyl-[acyl-carrier-protein] hydro-lyase','NULL','This enzyme is responsible for the dehydration step of thedissociated (type II) fatty-acid biosynthesis system that occurs inplants and bacteria.The enzyme uses fatty acyl thioesters of ACP in vivo.Different forms of the enzyme may have preferences for substrateswith different chain length.For example, the activity of FabZ, the ubiquitous enzyme in bacteria,decreases with increasing chain length.Gram-negative bacteria that produce unsaturated fatty acids, such asEscherichia coli, have another form (FabA) that prefers intermediatechain length, and also catalyzes EC 5.3.3.14.Despite the differences both forms can catalyze all steps leading tothe synthesis of palmitate (C16:0).FabZ, but not FabA, can also accept unsaturated substrates.Formerly EC 4.2.1.58, EC 4.2.1.60 and EC 4.2.1.61.','A (3R)-3-hydroxyacyl-[acyl-carrier protein] = a trans-2-enoyl-[acyl-carrier protein] + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.60','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.61','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.62','5-alpha-hydroxysteroid dehydratase','NULL','NULL','NULL','5-alpha-ergosta-7,22-diene-3-beta,5-diol = ergosterol + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.63','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.64','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.65','3-cyanoalanine hydratase','NULL','NULL','NULL','L-asparagine = 3-cyanoalanine + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.66','Cyanide hydratase','Formamide hydro-lyase','NULL','NULL','Formamide = cyanide + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.67','D-fuconate dehydratase','NULL','NULL','Also acts on L-arabinonate.','D-fuconate = 2-dehydro-3-deoxy-D-fuconate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.68','L-fuconate dehydratase','NULL','NULL','Also acts, slowly, on D-arabinonate.','L-fuconate = 2-dehydro-3-deoxy-L-fuconate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.69','Cyanamide hydratase','NULL','NULL','NULL','Urea = cyanamide + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.70','Pseudouridylate synthase','Pseudouridine synthase','NULL','Families of enzymes that forms pseudouridine in all kinds of RNA fromeubacteria, archaea and eukaryotes.','Uracil + D-ribose 5-phosphate = pseudouridine 5-phosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.71','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.72','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.73','Protoaphin-aglucone dehydratase (cyclizing)','NULL','NULL','The product is converted non-enzymically to erythroaphin, an aphidpigment.','Protoaphin aglucone = xanthoaphin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.74','Long-chain-enoyl-CoA hydratase','NULL','NULL','Acts in the reverse direction.The best substrate is oct-3-enoyl-CoA.Unlike EC 4.2.1.17 does not act on crotonoyl-CoA.','(3S)-3-hydroxyacyl-CoA = trans-2-enoyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.75','Uroporphyrinogen-III synthase','Hydroxymethylbilane hydro-lyase (cyclizing)','NULL','In the presence of EC 2.5.1.61, forms uroporphyrinogen III fromporphobilinogen.','Hydroxymethylbilane = uroporphyrinogen III + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.76','UDP-glucose 4,6-dehydratase','NULL','NULL','NULL','UDP-glucose = UDP-4-dehydro-6-deoxy-D-glucose + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.77','Trans-L-3-hydroxyproline dehydratase','NULL','NULL','Highly specific.','Trans-3-hydroxy-L-proline = 1-pyrroline 2-carboxylate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.78','(S)-norcoclaurine synthase','(S)-norlaudanosoline synthase','NULL','The reaction makes a 6-membered ring by forming a bond between C-6 ofthe 3,4-dihydroxyphenyl group of the dopamine and C-1 of the aldehydein the imine formed between the substrates.The product is the precursor of the benzylisoquinoline alkaloids inplants.Will also catalyze the reaction of 4-(2-aminoethyl)benzene-1,2-diol +(3,4-dihydroxyphenyl)acetaldehyde to form (S)-norlaudanosoline,but this alkaloid has not been found to occur in plants.','Dopamine + 4-hydroxyphenylacetaldehyde = (S)-norcoclaurine + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.79','2-methylcitrate dehydratase','2-hydroxybutane-1,2,3-tricarboxylate hydro-lyase','NULL','Not identical with EC 4.2.1.4.Specific for (2S,3S)-methylcitrate, showing no activity with (2R,3S)-methylcitrate.Can also use cis-aconitate as a substrate but more slowly.Both this enzyme and EC 4.2.1.3 are required to complete theisomerization of (2S,3S)-methylcitrate to (2R,3S)-2-methylisocitrate.','(2S,3S)-2-hydroxybutane-1,2,3-tricarboxylate = (Z)-but-2-ene-1,2,3-tricarboxylate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.80','2-oxopent-4-enoate hydratase','2-keto-4-pentenoate hydratase','NULL','Also acts, more slowly, on cis-2-oxohex-4-enoate, but not on thetrans-isomer.','4-hydroxy-2-oxopentanoate = 2-oxopent-4-enoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.81','D(-)-tartrate dehydratase','NULL','Iron.','NULL','(S,S)-tartrate = oxaloacetate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.82','Xylonate dehydratase','D-xylo-aldonate dehydratase','NULL','NULL','D-xylonate = 2-dehydro-3-deoxy-D-xylonate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.83','4-oxalmesaconate hydratase','4-carboxy-2-oxohexenedioate hydratase','NULL','This enzyme participates in the degradation of protocatechuate (viathe meta-cleavage pathway), syringate and gallate, catalyzing thereaction in the opposite direction.It accepts the enol-form of 4-oxalomesaconate, 2-hydroxy-4-carboxy-hexa-2,4-dienedioate.','2-hydroxy-4-oxobutane-1,2,4-tricarboxylate = (1E,3E)-4-hydroxybuta-1,3-diene-1,2,4-tricarboxylate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.84','Nitrile hydratase','Nitrilase','NULL','Acts on short-chain aliphatic nitriles, converting them into thecorresponding acid amides.Does not act on these amides or on aromatic nitriles (cf.EC 3.5.5.1).','An aliphatic amide = a nitrile + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.85','Dimethylmaleate hydratase','NULL','Iron.','Inhibited by oxygen.Forms part of the nicotinate-fermentation catabolism pathway inEubacterium barkeri.Other enzymes involved in this pathway are EC 1.17.5.1, EC 1.3.7.1,EC 3.5.2.18, EC 1.1.1.291, EC 5.4.99.4, EC 5.3.3.6 and EC 4.1.3.32.','(2R,3S)-2,3-dimethylmalate = dimethylmaleate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.86','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.87','Octopamine dehydratase','NULL','NULL','The enzyme-catalyzed reaction is believed to be dehydration to anenamine, which is spontaneously hydrolyzed to an aldehyde andammonia.','1-(4-hydroxyphenyl)-2-aminoethanol = (4-hydroxyphenyl)acetaldehyde +NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.88','(R)-synephrine','NULL','NULL','Removal of H(2)O from (R)-synephrine produces a 2,3-enamine, whichhydrolyzes to the products shown in the reaction above.The enzyme from Arthrobacter synephrinum is highly specific.','(R)-synephrine = (4-hydroxyphenyl)acetaldehyde + methylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.89','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.90','L-rhamnonate dehydratase','NULL','NULL','NULL','L-rhamnonate = 2-dehydro-3-deoxy-L-rhamnonate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.91','Arogenate dehydratase','Carboxycyclohexadienyl dehydratase','NULL','Also acts on prephenate and D-prephenyllactate.Cf. EC 4.2.1.51.','L-arogenate = L-phenylalanine + H(2)O + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.92','Hydroperoxide dehydratase','Allene oxide synthase','Heme-thiolate.','Acts on a number of unsaturated fatty-acid hydroperoxides, formingthe corresponding allene oxides.The product of the above reaction is unstable and is acted upon byEC 5.3.99.6 to form the cyclopentenone derivative (15Z)-12-oxophyto-10,15-dienoate (OPDA), which is the first cyclic and biologicallyactive metabolite in the jasmonate biosynthesis pathway.','(9Z,11E,15Z)-(13S)-hydroperoxyoctadeca-9,11,15-trienoate = (9Z,15Z)-(13S)-12,13-epoxyoctadeca-9,11,15-trienoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.93','ATP-dependent NAD(P)H-hydrate dehydratase','ATP-dependent H(4)NAD(P)OH dehydratase','NULL','Acts equally well on hydrated NADH and hydrated NADPH.NAD(P)H spontaneously hydrates to both the (6S)- and (6R)- isomers,and these are interconverted by EC 5.1.99.6, to a 60:40 ratio.Hence EC 4.2.1.93 together with EC 5.1.99.6 can restore the mixtureof hydrates into NAD(P)H.The enzyme from eukaryotes has no activity with ADP, contrary to theenzyme from bacteria (cf. EC 4.2.1.136).','(1) ATP + (6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adeninedinucleotide = ADP + phosphate + NADH.(2) ATP + (6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adeninedinucleotide phosphate = ADP + phosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.94','Scytalone dehydratase','NULL','NULL','Involved with EC 1.1.1.252 in the biosynthesis of melanin inpathogenic fungi.','Scytalone = 1,3,8-trihydroxynaphthalene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.95','Kievitone hydratase','NULL','NULL','The enzyme from Fusarium sp. hydrates the methylbutenyl side chain ofthe isoflavonoid phytoalexins, thus reducing their toxicity.','Kievitone hydrate = kievitone + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.96','4a-hydroxytetrahydrobiopterin dehydratase','4-alpha-hydroxy-tetrahydropterin dehydratase','NULL','Catalyzes the dehydration of 4a-hydroxytetrahydrobiopterins.','(6R)-6-(L-erythro-1,2-dihydroxypropyl)-5,6,7,8-tetrahydro-4a-hydroxypterin = (6R)-6-(L-erythro-1,2-dihydroxypropyl)-7,8-dihydro-6H-pterin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.97','Phaseollidin hydratase','NULL','NULL','The enzyme from Fusarium solani, which is distinct from EC 4.2.1.95,hydrates the methylbutenyl side-chain of the isoflavonoidphytoalexin, phaseollidin.','Phaseollidin hydrate = phaseollidin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.98','16-alpha-hydroxyprogesterone dehydratase','16-alpha-dehydroxylase','NULL','16-alpha-hydroxypregnenolone is also a substrate.Formerly EC 4.2.1.86.','16-alpha-hydroxyprogesterone = 16,17-didehydroprogesterone + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.99','2-methylisocitrate dehydratase','(2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate hydro-lyase','NULL','The enzyme from the fungus Yarrowia lipolytica does not act onisocitrate.','(2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate = (Z)-but-2-ene-1,2,3-tricarboxylate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.100','Cyclohexa-1,5-dienecarbonyl-CoA hydratase','Cyclohexa-1,5-diene-1-carbonyl-CoA hydratase','NULL','Forms part of the anaerobic benzoate degradation pathway, which alsoincludes EC 1.3.8.6, EC 1.3.7.8 and EC 4.2.1.55.Formerly EC 4.2.1.102.','6-hydroxycyclohex-1-enecarbonyl-CoA = cyclohexa-1,5-dienecarbonyl-CoA +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.101','Trans-feruloyl-CoA hydratase','4-hydroxy-3-methoxyphenyl-beta-hydroxypropanoyl-CoA hydro-lyase (trans-feruloyl-CoA-forming)','NULL','NULL','4-hydroxy-3-methoxyphenyl-beta-hydroxypropanoyl-CoA = feruloyl-CoA +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.102','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.103','Cyclohexyl-isocyanide hydratase','Isonitrile hydratase','NULL','The enzyme from Pseudomonas putida strain N19-2 can also catalyze thehydration of other isonitriles to the corresponding N-substitutedformamides.The enzyme has no metal requirements.','N-cyclohexylformamide = cyclohexyl isocyanide + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.104','Cyanase','Cyanate aminohydrolase','NULL','This enzyme, which is found in bacteria and plants, is used todecompose cyanate, which can be used as the sole source of nitrogen.The reaction can be considered as the reverse of carbamate = cyanate+ H(2)O, where this is assisted by reaction with bicarbonate andcarbon dioxide, and hence is classified in sub-subclass 4.2.1.Bicarbonate functions as a recycling substrate.Formerly EC 3.5.5.3 and EC 4.3.99.1.','Cyanate + HCO(3)(-) + 2 H(+) = NH(3) + 2 CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.105','2-hydroxyisoflavanone dehydratase','NULL','NULL','Catalyzes the final step in the formation of the isoflavonoidskeleton.The reaction also occurs spontaneously.','(1) 2,4,7-trihydroxyisoflavanone = daidzein + H(2)O.(2) 2,4,5,7-tetrahydroxyisoflavanone = genistein + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.106','Bile-acid 7-alpha-dehydratase','BA7 alpha dehydratase','NULL','The enzyme from Eubacterium sp. strain VPI 12708 can also use7-alpha-hydroxy-3-oxochol-4-enoate as a substrate but not 7-alpha,12-alpha-dihydroxy-3-oxochol-5-beta-anoate, 3-alpha,7-alpha,12-alpha-trihydroxychol-5-beta-anoate or 7-beta-hydroxy-3-oxochol-4-enoate.','7-alpha,12-alpha-dihydroxy-3-oxochol-4-enoate = 12-alpha-hydroxy-3-oxochola-4,6-dienoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.107','3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholest-24-enoyl-CoAhydratase','D-3-hydroxyacyl-CoA dehydratase','NULL','Forms part of the rat peroxisomal multifunctional enzyme perMFE-2,which also exhibits a dehydrogenase activity.Involved in the beta-oxidation of the cholesterol side chain in thecholic-acid biosynthesis pathway.','(24R,25R)-3-alpha,7-alpha,12-alpha,24-tetrahydroxy-5-beta-cholestanoyl-CoA = (24E)-3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholest-24-enoyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.108','Ectoine synthase','L-ectoine synthase','NULL','Ectoine is an osmoprotectant that is found in halophilic eubacteria.This is the third enzyme in the ectoine biosynthesis pathway,the other enzymes involved being EC 2.6.1.76 and EC 2.3.1.178.','N(4)-acetyl-L-2,4-diaminobutanoate = L-ectoine + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.109','Methylthioribulose 1-phosphate dehydratase','1-PMT-ribulose dehydratase','NULL','This enzyme forms part of the methionine salvage pathway.','S-methyl-5-thio-D-ribulose 1-phosphate = 5-(methylthio)-2,3-dioxopentylphosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.110','Aldos-2-ulose dehydratase','1,5-anhydro-D-fructose dehydratase (microthecin-forming)','NULL','This enzyme catalyzes two of the steps in the anhydrofructosepathway, which leads to the degradation of glycogen and starch via1,5-anhydro-D-fructose.Aldose-2-uloses such as 2-dehydroglucose can also act as substrates,but more slowly.This is a bifunctional enzyme that acts as both a lyase and as anisomerase.Differs from EC 4.2.1.111, which can carry out only reaction 1,is inhibited by its product and requires metal ions for activity.','(1) 1,5-anhydro-D-fructose = 1,5-anhydro-4-deoxy-D-glycero-hex-3-en-2-ulose + H(2)O.(2) 1,5-anhydro-4-deoxy-D-glycero-hex-3-en-2-ulose = 2-hydroxy-2-(hydroxymethyl)-2H-pyran-3(6H)-one.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.111','1,5-anhydro-D-fructose dehydratase','1,5-anhydro-D-arabino-hex-2-ulose dehydratase','Calcium or sodium or magnesium.','Catalyzes one of the steps in the anhydrofructose pathway, whichleads to the degradation of glycogen and starch via 1,5-anhydro-D-fructose.The other enzymes involved in this pathway are EC 4.2.1.110,EC 4.2.2.13 and EC 5.3.2.7.Unlike EC 4.2.1.110 the enzyme is specific for 1,5-anhydro-D-fructoseas substrate and shows no activity toward aldose-2-uloses such as2-dehydroglucose; in addition, it is inhibited by its end-productascopyrone M and it cannot convert the end-product ascopyrone M intomicrothecin, as can EC 4.2.1.110.','1,5-anhydro-D-fructose = 1,5-anhydro-4-deoxy-D-glycero-hex-3-en-2-ulose +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.112','Acetylene hydratase','AHy','Bis(molybdopterin guanine dinucleotide)molybdenum cofactor; Iron-sulfur;Tungsten.','The tungsten center binds a water molecule that is activated by anadjacent aspartate residue, enabling it to attack acetylene bound ina distinct hydrophobic pocket.Ethylene cannot act as a substrate.','Acetaldehyde = acetylene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.113','o-succinylbenzoate synthase','2-succinylbenzoate synthase','Manganese or magnesium.','Forms part of the vitamin-K biosynthesis pathway.','(1R,6R)-6-hydroxy-2-succinylcyclohexa-2,4-diene-1-carboxylate =2-succinylbenzoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.114','Methanogen homoaconitase','Methanogen HACN','Iron-sulfur.','Catalyzes several reactions in the pathway of coenzyme-B biosynthesisin methanogenic archaea.In contrast to EC 4.2.1.36, this enzyme can catalyze both thedehydration of (R)-homocitrate to form cis-homoaconitate and thesubsequent hydration reaction that forms homoisocitrate.In addition to cis-homoaconitate, the enzyme can also catalyze thehydration of the physiological substrates dihomocitrate andtrihomocitrate as well as the non-physiological substratetetrahomocitrate.Cis-aconitate and threo-DL-isocitrate cannot act as substrates,and (S)-homocitrate and trans-homoaconitate act as inhibitors of theenzyme.','(R)-2-hydroxybutane-1,2,4-tricarboxylate = (1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.115','UDP-N-acetylglucosamine 4,6-dehydratase (inverting)','UDP-N-acetylglucosamine 5-inverting 4,6-dehydratase','NADPH.','The first enzyme in the biosynthetic pathway of pseudaminic acid,a sialic-acid-like sugar that is unique to bacteria and is used byHelicobacter pylori to modify its flagellin.Plays a critical role in H.pyloris pathogenesis, being involved inthe synthesis of both functional flagella and lipopolysaccharides.Completely inhibited by UDP-alpha-D-galactose.The reaction results in the chirality of the C-5 atom being inverted.It is thought that Lys-133 acts sequentially as a catalytic acid,protonating the C-6 hydroxy group and as a catalytic base,abstracting the C-5 proton, resulting in the elimination of water.','UDP-N-acetyl-alpha-D-glucosamine = UDP-2-acetamido-2,6-dideoxy-beta-L-arabino-hex-4-ulose + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.116','3-hydroxypropionyl-CoA dehydratase','NULL','NULL','Catalyzes a step in the 3-hydroxypropanoate/4-hydroxybutyrate cycle,an autotrophic CO(2) fixation pathway found in some thermoacidophilicarchaea.The enzyme from Metallosphaera sedula acts nearly equally as well on(S)-3-hydroxybutanoyl-CoA but not (R)-3-hydroxybutanoyl-CoA.','3-hydroxypropanoyl-CoA = acryloyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.117','2-methylcitrate dehydratase (2-methyl-trans-aconitate forming)','NULL','Iron-sulfur.','Catalyzes the dehydration of (2S,3S)-2-methylcitrate, forming thetrans isomer of 2-methyl-aconitate (unlike EC 4.2.1.79, which formsonly the cis isomer).Part of a propanoate degradation pathway.The enzyme from Shewanella oneidensis can also accept citrate andcis-aconitate, but activity with (2S,3S)-2-methylcitrate wasapproximately 2.5-fold higher.2-methylisocitrate and isocitrate were not substrates.','(2S,3S)-2-methylcitrate = 2-methyl-trans-aconitate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.118','3-dehydroshikimate dehydratase','NULL','Manganese.','Catalyzes an early step in the biosynthesis of petrobactin,a siderophore produced by many bacteria, including the human pathogenBacillus anthracis.','3-dehydro-shikimate = protocatechuate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.119','Enoyl-CoA hydratase 2','2-enoyl-CoA hydratase 2','NULL','This enzyme catalyzes a hydration step in peroxisomal beta-oxidation.The human multifunctional enzyme type 2 (MFE-2) is a 79000 Da enzymecomposed of three functional units: (3R)-hydroxyacyl-CoAdehydrogenase, 2-enoyl-CoA hydratase 2 and sterol carrier protein2-like units.The enzymes from Aeromonas caviae and Arabidopsis thaliana aremonofunctional enzymes.2-Enoyl-CoA hydratase 3 from Candida tropicalis is a part frommultifunctional enzyme type 2.','(3R)-3-hydroxyacyl-CoA = (2E)-2-enoyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.120','4-hydroxybutanoyl-CoA dehydratase','NULL','FAD; Iron-sulfur.','The enzyme is often present as a bifunctional enzyme, catalyzing thedehydration of 4-hydroxybutanoyl-CoA to but-3-enoyl-CoA followed byisomerization of the later to crotonyl-CoA (EC 5.3.3.3).The enzyme has been characterized from several microorganisms,including Clostridium kluyveri, where it participates in succinatefermentation, Clostridium aminobutyricum, where it participates in4-aminobutyrate degradation, and Metallosphaera sedula, where itparticipates in the 3-hydroxypropanoate/4-hydroxybutyrate cycle,an autotrophic CO(2) fixation pathway found in some thermoacidophilicarchaea.','4-hydroxybutanoyl-CoA = but-3-enoyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.121','Colneleate synthase','9-DES','Heme-thiolate.','It catalyzes the selective removal of pro-R hydrogen at C-8 in thebiosynthesis of colneleic acid.It forms also (8E)-9-((1E,3Z,6Z)-nona-1,3,6-trien-1-yloxy)non-8-enoicacid (i.e. colnelenate) from (9Z,10E,12Z,15Z)-9-hydroperoxy-10,12,15-octadecatrienoate.The corresponding 13-hydroperoxides are poor substrates.The divinyl ethers colneleate and colnelenate have antimicrobialactivity.','(9S,10E,12Z)-9-hydroperoxyoctadeca-10,12-dienoate = (8E)-9-((1E,3Z)-nona-1,3-dien-1-yloxy)non-8-enoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.122','Tryptophan synthase (indole-salvaging)','Tryptophan synthase beta-2','NULL','Most mesophilic bacteria have a multimeric tryptophan synthasecomplex (EC 4.2.1.20) that forms L-tryptophan from L-serine and 1-C-(indol-3-yl)glycerol 3-phosphate via an indole intermediate.This intermediate, which is formed by the alpha subunits,is transferred in an internal tunnel to the beta units, which convertit to tryptophan.In thermophilic organisms the high temperature enhances diffusion andcauses the loss of indole.This enzyme, which does not combine with the alpha unit to form acomplex, salvages the lost indole back to L-tryptophan.It has a much lower Km for indole than the beta subunit ofEC 4.2.1.20.','L-serine + indole = L-tryptophan + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.123','Tetrahymanol synthase','Squalene--tetrahymanol cyclase','NULL','The reaction occurs in the reverse direction.','Tetrahymanol = squalene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.124','Arabidiol synthase','NULL','NULL','The reaction occurs in the reverse direction.','Arabidiol = (3S)-2,3-epoxy-2,3-dihydrosqualene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.125','Dammarenediol II synthase','2,3-oxidosqualene (20S)-dammarenediol cyclase','NULL','The reaction occurs in the reverse direction.','Dammarenediol II = (3S)-2,3-epoxy-2,3-dihydrosqualene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.126','N-acetylmuramic acid 6-phosphate etherase','MurNAc-6-P etherase','NULL','This enzyme, along with EC 2.7.1.170, is required for the utilizationof anhydro-N-acetylmuramic acid in proteobacteria.The substrate is either imported from the medium or derived from thebacteriums own cell wall murein during cell wall recycling.Formerly EC 4.2.1.n1.','(R)-lactate + N-acetyl-D-glucosamine 6-phosphate = N-acetylmuramate6-phosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.127','Linalool dehydratase','NULL','NULL','In absence of oxygen the bifunctional linalool dehydratase-isomerasecan catalyze in vitro two reactions, the dehydration of linalool tomyrcene and the isomerization of geraniol to linalool (the latteractivity is classified as EC 5.4.4.4).','Linalool = myrcene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.128','Lupan-3-beta,20-diol synthase','NULL','NULL','The reaction occurs in the reverse direction.The recombinant enzyme from Arabidopsis thaliana gives a 1:1 mixtureof lupeol and lupan-3-beta,20-diol with small amounts of beta-amyrin,germanicol, taraxasterol and psi-taraxasterol.See EC 5.4.99.41.','Lupan-3-beta,20-diol = (3S)-2,3-epoxy-2,3-dihydrosqualene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.129','Squalene--hopanol cyclase','Squalene--hopene cyclase','NULL','The enzyme produces the cyclization products hopene (cf.EC 5.4.99.17) and hopanol from squalene at a constant ratio of 5:1.','Hopan-22-ol = squalene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.130','D-lactate dehydratase','Glyoxylase III','NULL','The enzyme converts methylglyoxal to D-lactate in a singleglutathione (GSH)-independent step.The other known route for this conversion is the two-step GSH-dependent pathway catalyzed by EC 4.4.1.5 and EC 3.1.2.6.','(R)-lactate = methylglyoxal + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.131','Carotenoid 1,2-hydratase','NULL','NULL','In Rubrivivax gelatinosus and Thiocapsa roseopersicina both productsare formed, whereas Rhodobacter capsulatus only gives 1-hydroxy-1,2-dihydrolycopene.Also acts on neurosporene giving 1-hydroxy-1,2-dihydroneurosporenewith both organisms but 1,1-dihydroxy-1,1,2,2-tetrahydroneurosporene only with Rubrivivax gelatinosus.','(1) 1-hydroxy-1,2-dihydrolycopene = lycopene + H(2)O.(2) 1,1-dihydroxy-1,1,2,2-tetrahydrolycopene = 1-hydroxy-1,2-dihydrolycopene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.132','2-hydroxyhexa-2,4-dienoate hydratase','NULL','NULL','This enzyme catalyzes a late step in the bacterial steroiddegradation pathway.The product, 4-hydroxy-2-oxohexanoate, forms a 2-hydroxy-4-hex-2-enolactone under acidic conditions.','4-hydroxy-2-oxohexanoate = (2Z,4Z)-2-hydroxyhexa-2,4-dienoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.133','Copal-8-ol diphosphate hydratase','NULL','Magnesium.','The enzyme was characterized from the plant Cistus creticus subsp.creticus.Formerly EC 3.1.7.4 and EC 5.5.1.21.','(13E)-8-alpha-hydroxylabd-13-en-15-yl diphosphate = geranylgeranyldiphosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.134','Very-long-chain (3R)-3-hydroxyacyl-[acyl-carrier protein] dehydratase','NULL','NULL','This is the third component of the elongase, a microsomal proteincomplex responsible for extending palmitoyl-CoA and stearoyl-CoA (andmodified forms thereof) to very-long chain acyl CoAs.Cf. EC 2.3.1.199, EC 1.1.1.330, and EC 1.3.1.93.Formerly EC 4.2.1.n2.','A very-long-chain (3R)-3-hydroxyacyl-[acyl-carrier protein] = a very-long-chain trans-2,3-dehydroacyl-[acyl-carrier protein] + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.135','UDP-N-acetylglucosamine 4,6-dehydratase (configuration-retaining)','NULL','NAD(+).','This is the first enzyme in the biosynthetic pathway ofN,N-diacetylbacillosamine, the first carbohydrate in theglycoprotein N-linked heptasaccharide in Campylobacter jejuni.This enzyme belongs to the short-chain dehydrogenase/reductase familyof enzymes.','UDP-N-acetyl-alpha-D-glucosamine = UDP-2-acetamido-2,6-dideoxy-alpha-D-xylo-hex-4-ulose + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.136','ADP-dependent NAD(P)H-hydrate dehydratase','(6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adenine-dinucleotidehydro-lyase (ADP-hydrolyzing; NADH-forming)','NULL','Acts equally well on hydrated NADH and hydrated NADPH.NAD(P)H spontaneously hydrates to both the (6S)- and (6R)- isomers.The enzyme from bacteria consists of two domains, one of which actsas an NAD(P)H-hydrate epimerase that interconverts the two isomers toa 60:40 ratio (cf. EC 5.1.99.6), while the other catalyzes thedehydration.Hence the enzyme can restore the complete mixture of isomers intoNAD(P)H.The enzyme has no activity with ATP, contrary to the enzyme fromeukaryotes (cf. EC 4.2.1.93).','(1) ADP + (6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide adenine-dinucleotide = AMP + phosphate + NADH.(2) ADP + (6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide adenine-dinucleotide phosphate = AMP + phosphate + NADPH.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.137','Sporulenol synthase','NULL','NULL','The reaction occurs in the reverse direction.Isolated from Bacillus subtilis.Similar sesquarterpenoids are present in a number of Bacillusspecies.','Sporulenol = tetraprenyl-beta-curcumene + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.138','(+)-caryolan-1-ol synthase','NULL','NULL','A multifunctional enzyme which also forms (+)-beta-caryophyllene fromfarnesyl diphosphate (EC 4.2.3.89).Formerly EC 3.7.1.15.','(+)-beta-caryophyllene + H(2)O = (+)-caryolan-1-ol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.139','Medicarpin synthase','2,7-dihydroxy-4-methoxyisoflavanol dehydratase','NULL','Isolated from the plant Medicago sativa (alfalfa).Catalyzes the final step in the biosynthesis of medicarpin, the mainpterocarpan phytoalexin in alfalfa.Formerly EC 1.1.1.246.','4-methoxyisoflavan-2,4,7-triol = (-)-medicarpin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.140','Gluconate/galactonate dehydratase','Gluconate dehydratase','Magnesium.','The enzyme is involved in glucose and galactose catabolism via thenonphosphorylative variant of the Entner-Doudoroff pathway inPicrophilus torridus and via the branched variant of the Entner-Doudoroff pathway in Sulfolobus solfataricus.In vitro it utilizes D-gluconate with 6-10 fold higher catalyticefficiency than D-galactonate.Cf. EC 4.2.1.6 and EC 4.2.1.39.','(1) D-gluconate = 2-dehydro-3-deoxy-D-gluconate + H(2)O.(2) D-galactonate = 2-dehydro-3-deoxy-D-galactonate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.141','2-dehydro-3-deoxy-D-arabinonate dehydratase','NULL','NULL','The enzyme participates in pentose oxidation pathways that convertpentose sugars to the tricarboxylic acid cycle intermediate2-oxoglutarate.','2-dehydro-3-deoxy-D-arabinonate = 2,5-dioxopentanoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.142','5-oxoaverantin cyclase','OAVN cyclase','NULL','Isolated from the aflatoxin-producing mold Aspergillus parasiticus.The enzyme also catalyzes the conversion of versiconal toversicolorin B (EC 4.2.1.143).Involved in aflatoxin biosynthesis.','5-oxoaverantin = (2S,5S)-averufin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.143','Versicolorin B synthase','Versiconal cyclase','NULL','Isolated from the aflatoxin-producing mold Aspergillus parasiticus.Involved in aflatoxin biosynthesis.','Versiconal = versicolorin B + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.144','3-amino-5-hydroxybenzoate synthase','AHBA synthase','Pyridoxal 5-phosphate.','The enzyme from the bacterium Amycolatopsis mediterranei participatesin the pathway for rifamycin B biosynthesis.The enzyme also functions as a transaminase earlier in the pathway,producing UDP-alpha-D-kanosamine.','5-amino-5-deoxy-3-dehydroshikimate = 3-amino-5-hydroxybenzoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.145','Capreomycidine synthase','NULL','Pyridoxal 5-phosphate.','The enzyme is involved in the biosynthesis of the cyclic pentapeptideantibiotic viomycin.','(2S,3S)-3-hydroxyarginine = (2S,3R)-capreomycidine + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.146','L-galactonate dehydratase','NULL','NULL','The enzyme takes part in a D-galacturonate degradation pathway in thefungi Trichoderma reesei (Hypocrea jecorina) and Aspergillus niger.','L-galactonate = 2-dehydro-3-deoxy-L-galactonate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.147','5,6,7,8-tetrahydromethanopterin hydro-lyase','Formaldehyde-activating enzyme','NULL','Found in methylotrophic bacteria and methanogenic archaea.','5,6,7,8-tetrahydromethanopterin + formaldehyde = 5,10-methylenetetrahydromethanopterin + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.148','2-methylfumaryl-CoA hydratase','Erythro-beta-methylmalonyl-CoA hydrolyase','NULL','The enzyme from the bacterium Chloroflexus aurantiacus is part of the3-hydroxypropanoate cycle for carbon assimilation.','(2R,3S)-2-methylmalyl-CoA = 2-methylfumaryl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.149','Crotonobetainyl-CoA hydratase','L-carnityl-CoA dehydratase','NULL','The enzyme is also able to use crotonyl-CoA as substrate, with lowefficiency.Formerly EC 4.2.1.89.','L-carnitinyl-CoA = (E)-4-(trimethylammonio)but-2-enoyl-CoA + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.150','Short-chain-enoyl-CoA hydratase','3-hydroxybutyryl-CoA dehydratase','NULL','The enzyme from the bacterium Clostridium acetobutylicum is part ofthe central fermentation pathway and plays a key role in theproduction of both acids and solvents.It is specific for short, C(4)-C(6), chain length substrates andexhibits an extremely high turnover number for crotonyl-CoA.Cf. EC 4.2.1.17 and EC 4.2.1.74.','A short-chain (3S)-3-hydroxyacyl-CoA = a short-chain trans-2-enoyl-CoA +H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.151','Chorismate dehydratase','NULL','NULL','The enzyme, found in several bacterial species, is part of thefutalosine pathway for menaquinone biosynthesis.','Chorismate = 3-((1-carboxyvinyl)oxy)benzoate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.n1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.1.n2','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.1','Hyaluronate lyase','Hyaluronidase','NULL','Also acts on chondroitin.The product is more systematically known as 3-(4-deoxy-alpha-L-threo-hex-4-enopyranosyluronic acid)-2-acetamido-2-deoxy-D-glucose.Formerly EC 4.2.99.1.','Cleaves hyaluronate chains at a beta-D-GalNAc-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.2','Pectate lyase','Alpha-1,4-D-endopolygalacturonic acid lyase','NULL','Favors pectate, the anion, over pectin, the methyl ester (which isthe preferred substrate of EC 4.2.2.10).Formerly EC 4.2.99.3.','Eliminative cleavage of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.3','Poly(beta-D-mannuronate) lyase','Alginate lyase I','NULL','The enzyme from the bacterium Sphingomonas sp. strain A1 cleavesalginate producing di- and trisaccharides with an unsaturated uronylresidue at the non-reducing terminus.Formerly EC 4.2.99.4.','Eliminative cleavage of polysaccharides containing beta-D-mannuronateresidues to give oligosaccharides with 4-deoxy-alpha-L-erythro-hex-4-enopyranuronosyl groups at their non-reducing ends.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.4','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.5','Chondroitin AC lyase','Chondroitin AC eliminase','NULL','Acts on chondroitin 4-sulfate and chondroitin 6-sulfate, but lesswell on hyaluronate.In general, chondroitin sulfate (CS) and dermatan sulfate (DS) chainscomprise a linkage region, a chain cap and a repeat region.The repeat region of CS is a repeating disaccharide of glucuronicacid (GlcA) and N-acetylgalactosamine (GalNAc) [-4)GlcA(beta-1-3)GalNAc(beta-1-](n), which may be O-sulfated on the C-4 and/or C-6of GalNAc and C-2 of GlcA.GlcA residues of CS may be epimerized to iduronic acid (IdoA) formingthe repeating disaccharide [-4)IdoA(alpha-1-3)GalNAc(beta-1-](n) ofDS.Both the concentrations and locations of sulfate-ester substituentsvary with glucosaminoglycan source.Formerly EC 4.2.99.6.','Eliminative degradation of polysaccharides containing 1,4-beta-D-hexosaminyl and 1,3-beta-D-glucuronosyl linkages to disaccharidescontaining 4-deoxy-beta-D-gluc-4-enuronosyl groups.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.6','Oligogalacturonide lyase','NULL','NULL','Also catalyzes eliminative removal of unsaturated terminal residuesfrom oligosaccharides of D-galacturonate.','4-(4-deoxy-beta-D-galact-4-enuronosyl)-D-galacturonate = 2 5-dehydro-4-deoxy-D-glucuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.7','Heparin lyase','Heparin eliminase','NULL','NULL','Eliminative cleavage of polysaccharides containing (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.8','Heparin-sulfate lyase','Heparin-sulfate eliminase','NULL','Does not act on N,O-desulfated glucosamine or N-acetyl-O-sulfatedglucosamine linkages.','Elimination of sulfate; appears to act on linkages between N-acetyl-D-glucosamine and uronate. Product is an unsaturated sugar.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.9','Pectate disaccharide-lyase','Exo-PATE','NULL','NULL','Eliminative cleavage of 4-(4-deoxy-alpha-D-galact-4-enuronosyl)-D-galacturonate from the reducing end of pectate, i.e. de-esterifiedpectin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.10','Pectin lyase','Endo-pectin lyase','NULL','Favors pectin, the methyl ester, over pectate, the anion (which isthe preferred substrate of EC 4.2.2.2).','Eliminative cleavage of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.11','Poly(alpha-L-guluronate) lyase','Alginase II','NULL','NULL','Eliminative cleavage of polysaccharides containing a terminal alpha-L-guluronate group, to give oligosaccharides with 4-deoxy-alpha-L-erythro-hex-4-enuronosyl groups at their non-reducing ends.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.12','Xanthan lyase','NULL','NULL','NULL','Eliminative cleavage of the terminal beta-D-mannosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.13','Exo-(1-','Alpha-1,4-glucan 1,5-anhydro-D-fructose eliminase','NULL','Catalyzes the sequential degradation of (1-','Linear alpha-glucan = (n-1) 1,5-anhydro-D-fructose + D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.14','Glucuronan lyase','NULL','NULL','NULL','Eliminative cleavage of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.15','Anhydrosialidase','Anhydroneuraminidase','NULL','Also acts on N-glycolylneuraminate glycosides.Cf. EC 3.2.1.18 and EC 3.2.1.129.Formerly EC 3.2.1.138.','Elimination of alpha-sialyl groups in N-acetylneuraminic acid glycosides,releasing 2,7-anhydro-alpha-N-acetylneuraminate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.16','Levan fructotransferase (DFA-IV-forming)','2,6-beta-D-fructan D-fructosyl-D-fructosyltransferase (forming di-beta-D-fructofuranose 2,6:2,6-dianhydride)','NULL','This enzyme, like EC 4.2.2.17 and EC 4.2.2.18 eliminates the fructanchain from the terminal disaccharide leaving a difructosedianhydride.These enzymes have long been known as fructotransferases, so this isretained in the description (official name).Since the transfer is intramolecular, the reaction is an eliminationand, hence, the enzyme is a lyase, belonging in enzyme class 4.-.-.-.','Produces di-beta-D-fructofuranose 2,6:2,6-dianhydride (DFA IV) bysuccessively eliminating the diminishing (2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.17','Inulin fructotransferase (DFA-I-forming)','Inulin D-fructosyl-D-fructosyltransferase (1,2:1,2-dianhydride-forming)','NULL','This enzyme, like EC 4.2.2.16 and EC 4.2.2.18 eliminates the fructanchain from the terminal disaccharide leaving a difructosedianhydride.These enzymes have long been known as fructotransferases, so this isretained in the description (official name).Since the transfer is intramolecular, the reaction is an eliminationand, hence, the enzyme is a lyase, belonging in enzyme class 4.-.-.-.Formerly EC 2.4.1.200.','Produces alpha-D-fructofuranose beta-D-fructofuranose1,2:2,1-dianhydride (DFA I) by successively eliminating the diminishing(2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.18','Inulin fructotransferase (DFA-III-forming)','Inulase II','NULL','This enzyme, like EC 4.2.2.16 and EC 4.2.2.17 eliminates the fructanchain from the terminal disaccharide leaving a difructosedianhydride.These enzymes have long been known as fructotransferases, so this isretained in the description (official name).Since the transfer is intramolecular, the reaction is an eliminationand, hence, the enzyme is a lyase, belonging in enzyme class 4.-.-.-.Formerly EC 2.4.1.93.','Produces alpha-D-fructofuranose beta-D-fructofuranose1,2:2,3-dianhydride (DFA III) by successively eliminating thediminishing (2-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.19','Chondroitin B lyase','ChonB','NULL','This is the only lyase that is known to be specific for dermatansulfate as substrate.The minimum substrate length required for catalysis is atetrasaccharide.In general, chondroitin sulfate (CS) and dermatan sulfate (DS) chainscomprise a linkage region, a chain cap and a repeat region.The repeat region of CS is a repeating disaccharide of glucuronicacid (GlcA) and N-acetylgalactosamine (GalNAc) [-4)GlcA(beta-1-3)GalNAc(beta-1-](n), which may be O-sulfated on the C-4 and/or C-6of GalNAc and C-2 of GlcA.GlcA residues of CS may be epimerized to iduronic acid (IdoA) formingthe repeating disaccharide [-4)IdoA(alpha-1-3)GalNAc(beta-1-](n) ofDS.Both the concentrations and locations of sulfate-ester substituentsvary with glucosaminoglycan source.','Eliminative cleavage of dermatan sulfate containing (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.20','Chondroitin-sulfate-ABC endolyase','Chondroitin ABC eliminase','NULL','Degrades a variety of glycosaminoglycans of the chondroitin-sulfate-and dermatan-sulfate type.Chondroitin sulfate, chondroitin-sulfate proteoglycan and dermatansulfate are the best substrates but the enzyme can also act onhyaluronan at a much lower rate.Keratan sulfate, heparan sulfate and heparin are not substrates.In general, chondroitin sulfate (CS) and dermatan sulfate (DS) chainscomprise a linkage region, a chain cap and a repeat region.The repeat region of CS is a repeating disaccharide of glucuronicacid (GlcA) and N-acetylgalactosamine (GalNAc) [-4)GlcA(beta-1-3)GalNAc(beta-1-](n), which may be O-sulfated on the C-4 and/or C-6of GalNAc and C-2 of GlcA.GlcA residues of CS may be epimerized to iduronic acid (IdoA) formingthe repeating disaccharide [-4)IdoA(alpha-1-3)GalNAc(beta-1-](n) ofDS.Both the concentrations and locations of sulfate-ester substituentsvary with glucosaminoglycan source.The related enzyme EC 4.2.2.21 has the same substrate specificity butremoves disaccharide residues from the non-reducing ends of bothpolymeric chondroitin sulfates and their oligosaccharide fragmentsproduced by EC 4.2.2.20.Formerly EC 4.2.2.4 and EC 4.2.99.6.','Endolytic cleavage of (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.21','Chondroitin-sulfate-ABC exolyase','Chondroitin ABC eliminase','NULL','Degrades a variety of glycosaminoglycans of the chondroitin-sulfate-and dermatan-sulfate type.Chondroitin sulfate, chondroitin-sulfate proteoglycan and dermatansulfate are the best substrates but the enzyme can also act onhyaluronan at a much lower rate.The related enzyme EC 4.2.2.20 has the same substrate specificity butproduces a mixture of oligosaccharides of different sizes that areultimately degraded to tetra- and disaccharides.Both enzymes act by the removal of a relatively acidic C-5 proton ofthe uronic acid followed by the elimination of a 4-linked hexosamine,resulting in the formation of an unsaturated C4-C5 bond on thehexuronic acid moiety of the products.Formerly EC 4.2.2.4 and EC 4.2.99.6.','Exolytic removal of Delta(4)-unsaturated disaccharide residues from thenon-reducing ends of both polymeric chondroitin/dermatan sulfates andtheir oligosaccharide fragments.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.22','Pectate trisaccharide-lyase','Exopectate-lyase','NULL','Differs in specificity from EC 4.2.2.9 as the predominant action isremoval of a trisaccharide rather than a disaccharide from thereducing end.Disaccharides and tetrasaccharides may also be removed.','Eliminative cleavage of unsaturated trigalacturonate as the major productfrom the reducing end of polygalacturonic acid/pectate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.23','Rhamnogalacturonan endolyase','Alpha-L-rhamnopyranosyl-(1-','NULL','The enzyme is part of the degradation system for rhamnogalacturonan Iin Bacillus subtilis strain 168 and Aspergillus aculeatus.','Endotype eliminative cleavage of L-alpha-rhamnopyranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.24','Rhamnogalacturonan exolyase','NULL','NULL','The enzyme is part of the degradation system for rhamnogalacturonan Iin Bacillus subtilis strain 168.','Exotype eliminative cleavage of alpha-L-rhamnopyranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.25','Gellan lyase','NULL','NULL','The enzyme is highly specific to gellan, especially deacetylatedgellan.','Eliminative cleavage of beta-D-glucopyranosyl-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.n1','Peptidoglycan lytic exotransglycosylase','Exomuramidase','NULL','NULL','Exolytic cleavage of the (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.2.n2','Peptidoglycan lytic endotransglycosylase','NULL','NULL','NULL','Endolytic cleavage of the (1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.1','Threonine synthase','NULL','Pyridoxal 5-phosphate.','Formerly EC 4.2.99.2.','O-phospho-L-homoserine + H(2)O = L-threonine + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.2','Ethanolamine-phosphate phospho-lyase','Amino alcohol O-phosphate phospholyase','Pyridoxal 5-phosphate.','Also acts on D- (or L)-1-aminopropan-2-ol O-phosphate.Formerly EC 4.2.99.7.','Ethanolamine phosphate + H(2)O = acetaldehyde + NH(3) + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.3','Methylglyoxal synthase','NULL','NULL','Does not act on D-glyceraldehyde 3-phosphate.Formerly EC 4.2.99.11.','Glycerone phosphate = methylglyoxal + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.4','3-dehydroquinate synthase','3-dehydroquinate synthetase','Cobalt; NAD(+).','The hydrogen atoms on C-7 of the substrate are retained on C-2 of theproducts.Formerly EC 4.6.1.3.','3-deoxy-D-arabino-hept-2-ulosonate 7-phosphate = 3-dehydroquinate +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.5','Chorismate synthase','5-enolpyruvylshikimate-3-phosphate phospholyase','FMN.','The reaction goes via a radical mechanism that involves reduced FMNand its semiquinone (FMNH.).Shikimate is numbered so that the double-bond is between C-1 and C-2,but some earlier papers numbered the ring in the reverse direction.Formerly EC 4.6.1.4.','5-O-(1-carboxyvinyl)-3-phosphoshikimate = chorismate + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.6','Trichodiene synthase','Sesquiterpene cyclase','NULL','Formerly EC 4.1.99.6.','(2E,6E)-farnesyl diphosphate = trichodiene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.7','Pentalenene synthase','Pentalenene synthetase','NULL','Isolated from Streptomyces avermitilis.The enzyme is involved in the biosynthesis of pentalenolactone andrelated antibiotics.The 9si hydrogen of farnesyl diphosphate undergoes a 1,2-hydrideshift where it becomes the 1-alpha hydrogen of pentalenene.Formerly EC 4.6.1.5.','(2E,6E)-farnesyl diphosphate = pentalenene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.8','Casbene synthase','Casbene synthetase','NULL','The enzyme from castor bean (Ricinus communis) produces theantifungal diterpene casbene.Formerly EC 4.6.1.7.','Geranylgeranyl diphosphate = casbene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.9','Aristolochene synthase','Sesquiterpene cyclase','Magnesium; Manganese.','The initial internal cyclization produces the monocyclic intermediategermacrene A; further cyclization and methyl transfer converts theintermediate into aristolochene.While in some species germacrene A remains as an enzyme-boundintermediate, it has been shown to be a minor product of the reactionin Penicillium roqueforti (see also EC 4.2.3.23).Aristolochene is the likely parent compound for a number ofsesquiterpenes produced by filamentous fungi.Formerly EC 2.5.1.40 and EC 4.1.99.7.','(2E,6E)-farnesyl diphosphate = aristolochene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.10','(-)-endo-fenchol synthase','NULL','NULL','(3R)-linalyl diphosphate is an intermediate in the reaction.Formerly EC 4.6.1.8.','Geranyl diphosphate = (-)-endo-fenchol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.11','Sabinene-hydrate synthase','Sabinene hydrate cyclase','NULL','Both cis- and trans- isomers of sabinene hydrate are formed.(3R)-linalyl diphosphate is an intermediate in the reaction.Formerly EC 4.6.1.9.','Geranyl diphosphate + H(2)O = sabinene hydrate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.12','6-pyruvoyltetrahydropterin synthase','6-pyruvoyl tetrahydrobiopterin synthase','Magnesium.','Catalyzes triphosphate elimination and an intramolecular redoxreaction.The product is 6-pyruvoyltetrahydrobiopterin.Involved in the de novo synthesis of tetrahydrobiopterin from GTP,with the other enzymes involved being EC 1.1.1.153 and EC 3.5.4.16.Formerly EC 4.6.1.10.','7,8-dihydroneopterin 3-triphosphate = 6-pyruvoyl-5,6,7,8-tetrahydropterin + triphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.13','(+)-delta-cadinene synthase','D-cadinene synthase','Magnesium.','The sesquiterpenoid (+)-delta-cadinene is an intermediate inphytoalexin biosynthesis.Formerly EC 4.6.1.11.','(2E,6E)-farnesyl diphosphate = (+)-delta-cadinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.14','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.15','Myrcene synthase','NULL','Manganese; Potassium.','Mg(2+) is essentially ineffective as the divalent metal ion cofactor.Formerly EC 4.1.99.9.','Geranyl diphosphate = myrcene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.16','(4S)-limonene synthase','(-)-(4S)-limonene synthase','Manganese; Potassium.','Mg(2+) is essentially ineffective as the divalent metal ion cofactor.Formerly EC 4.1.99.10.','Geranyl diphosphate = (-)-(4S)-limonene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.17','Taxadiene synthase','Taxa-4(5),11(12)-diene synthase','NULL','This is the committed step in the biosynthesis of the diterpenoidantineoplastic drug Taxol (paclitaxel).The cyclization involves a 1,5-hydride shift.','Geranylgeranyl diphosphate = taxa-4,11-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.18','Abieta-7,13-diene synthase','Abietadiene cyclase','Magnesium.','Part of a bifunctional enzyme involved in the biosynthesis ofabietadiene.See also EC 5.5.1.12.','(+)-copalyl diphosphate = abieta-7,13-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.19','Ent-kaurene synthase','Ent-kaurene synthase B','NULL','Part of a bifunctional enzyme involved in the biosynthesis of ent-kaurene.See also EC 5.5.1.13.','Ent-copalyl diphosphate = ent-kaurene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.20','(R)-limonene synthase','(+)-limonene synthase','Manganese.','Forms the first step of carvone biosynthesis in caraway.','Geranyl diphosphate = (+)-(4R)-limonene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.21','Vetispiradiene synthase','HVS','NULL','The initial internal cyclization produces the monocyclic intermediategermacrene A.','(2E,6E)-farnesyl diphosphate = vetispiradiene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.22','Germacradienol synthase','Germacradienol/germacrene-D synthase','Magnesium.','H-1si of farnesyl diphosphate is lost in the formation of(1E,4S,5E,7R)-germacra-1(10),5-dien-11-ol.Formation of (-)-germacrene D involves a stereospecific 1,3-hydrideshift of H-1si of farnesyl diphosphate.Both products are formed from a common intermediate.The enzyme mediates a key step in the biosynthesis of geosmin (seeEC 4.1.99.16), a widely occurring metabolite of many streptomycetes,bacteria and fungi.Also catalyzes the reaction of EC 4.2.3.75.','(2E,6E)-farnesyl diphosphate + H(2)O = (1E,4S,5E,7R)-germacra-1(10),5-dien-11-ol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.23','Germacrene-A synthase','(+)-(10R)-germacrene A synthase','Magnesium.','While germacrene A is an enzyme-bound intermediate in thebiosynthesis of a number of phytoalexins, e.g. EC 4.2.3.9 from somespecies and EC 4.2.3.21, it is the sole sesquiterpenoid productformed in chicory.','(2E,6E)-farnesyl diphosphate = (+)-(R)-germacrene A + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.24','Amorpha-4,11-diene synthase','Amorphadiene synthase','Magnesium; Manganese.','A key enzyme in the biosynthesis of the antimalarial endoperoxideartemisinin.Catalyzes the formation of both olefinic and oxygenatedsesquiterpenes, with amorpha-4,11-diene being the major product.When geranyl diphosphate is used as a substrate, no monoterpenes areproduced.','(2E,6E)-farnesyl diphosphate = amorpha-4,11-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.25','S-linalool synthase','3S-linalool synthase','Manganese or magnesium.','Neither (S)- nor (R)-linalyl diphosphate can act as substrate for theenzyme from Clarkia breweri.Unlike many other monoterpene synthases, only a single product, (3S)-linalool, is formed.','Geranyl diphosphate + H(2)O = (3S)-linalool + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.26','R-linalool synthase','(-)-3R-linalool synthase','Manganese or magnesium.','Geranyl diphosphate cannot be replaced by isopentenyl diphosphate,dimethylallyl diphosphate, farnesyl diphosphate or geranylgeranyldiphosphate as substrate.Unlike many other monoterpene synthases, only a single product, (3R)-linalool, is formed.','Geranyl diphosphate + H(2)O = (3R)-linalool + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.27','Isoprene synthase','ISPC','Manganese or magnesium.','Located in the chloroplast of isoprene-emitting plants, such aspoplar and aspen, and may be activitated by light-dependent changesin chloroplast pH and Mg(2+) concentration.','Dimethylallyl diphosphate = isoprene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.28','Ent-cassa-12,15-diene synthase','NULL','NULL','This class I diterpene cyclase produces ent-cassa-12,15-diene,a precursor of the rice phytoalexins (-)-phytocassanes A-E.Phytoalexins are diterpenoid secondary metabolites that are involvedin the defense mechanism of the plant, and are produced in responseto pathogen attack through the perception of elicitor signalmolecules such as chitin oligosaccharide, or after exposure to UVirradiation.','Ent-copalyl diphosphate = ent-cassa-12,15-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.29','Ent-sandaracopimaradiene synthase','Ent-sandaracopimara-8(14),15-diene synthase','NULL','Ent-sandaracopimaradiene is a precursor of the rice oryzalexins A-F.Phytoalexins are diterpenoid secondary metabolites that are involvedin the defense mechanism of the plant, and are produced in responseto pathogen attack through the perception of elicitor signalmolecules such as chitin oligosaccharide, or after exposure to UVirradiation.As a minor product, this enzyme also forms ent-pimara-8(14),15-diene,which is the sole product of EC 4.2.3.30.Ent-pimara-8(14),15-diene is not a precursor in the biosynthesis ofeither gibberellins or phytoalexins.','Ent-copalyl diphosphate = ent-sandaracopimara-8(14),15-diene +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.30','Ent-pimara-8(14),15-diene synthase','NULL','NULL','Unlike EC 4.2.3.29 which can produce both ent-sandaracopimaradieneand ent-pimara-8(14),15-diene, this diterpene cyclase produces onlyent-pimara-8(14),15-diene.Ent-pimara-8(14),15-diene is not a precursor in the biosynthesis ofeither gibberellins or phytoalexins.','Ent-copalyl diphosphate = ent-pimara-8(14),15-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.31','Ent-pimara-9(11),15-diene synthase','PMD synthase','Divalent cation.','Involved in the biosynthesis of the diterpenoid viguiepinol.','Ent-copalyl diphosphate = ent-pimara-9(11),15-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.32','Levopimaradiene synthase','NULL','NULL','In Ginkgo, the enzyme catalyzes the initial cyclization step in thebiosynthesis of ginkgolides, a structurally unique family ofditerpenoids that are highly specific platelet-activating-factorreceptor antagonists.Levopimaradiene is widely distributed in higher plants.In some species the enzyme also forms abietadiene, palustradiene,and neoabietadiene.','(+)-copalyl diphosphate = abieta-8(14),12-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.33','Stemar-13-ene synthase','Stemarene synthase','NULL','This diterpene cyclase produces stemar-13-ene, a putative precursorof the rice phytoalexin oryzalexin S.Phytoalexins are diterpenoid secondary metabolites that are involvedin the defense mechanism of the plant, and are produced in responseto pathogen attack through the perception of elicitor signalmolecules such as chitin oligosaccharide, or after exposure to UVirradiation.','9-alpha-copalyl diphosphate = stemar-13-ene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.34','Stemod-13(17)-ene synthase','Stemodene synthase','NULL','Catalyzes the committed step in the biosynthesis of the stemodanefamily of diterpenoid secondary metabolites, some of which possessmild antiviral activity.Also produces stemod-12-ene and stemar-13-ene as minor products.','9-alpha-copalyl diphosphate = stemod-13(17)-ene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.35','Syn-pimara-7,15-diene synthase','9-beta-pimara-7,15-diene synthase','NULL','A class I terpene synthase.9-beta-pimara-7,15-diene is a precursor of momilactones A and B, ricediterpenoid phytoalexins that are produced in response to attack (bya pathogen, elicitor or UV irradiation) and are involved in thedefense mechanism of the plant.Momilactone B can also act as an allochemical, being constitutivelyproduced in the root of the plant and secreted to the rhizospherewhere it suppresses the growth of neighboring plants and soilmicroorganisms.','9-alpha-copalyl diphosphate = 9-beta-pimara-7,15-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.36','Terpentetriene synthase','NULL','Magnesium.','Following on from EC 5.5.1.15, this enzyme completes thetransformation of geranylgeranyl diphosphate (GGDP) intoterpentetriene, which is a precursor of the diterpenoid antibioticterpentecin.Farnesyl diphosphate can also act as a substrate.','Terpentedienyl diphosphate = terpentetriene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.37','Epi-isozizaene synthase','NULL','Magnesium.','The displacement of the diphosphate group of farnesyl diphosphateoccurs with retention of configuration.In the soil-dwelling bacterium Streptomyces coelicolor A3(2),the product of this reaction is used by EC 1.14.13.106 to produce thesesquiterpene antibiotic albaflavenone.','(2E,6E)-farnesyl diphosphate = (+)-epi-isozizaene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.38','Alpha-bisabolene synthase','NULL','Magnesium.','While unlikely to encounter geranyl diphosphate (GDP) in vivo as itis localized to plastids, the enzyme can use GDP as a substrate invitro to produce (+)-(4R)-limonene (cf. EC 4.2.3.20).Induced as part of a defense mechanism in the grand fir Abies grandisas a response to stem wounding.Formerly EC 4.2.3.n1.','(2E,6E)-farnesyl diphosphate = (E)-alpha-bisabolene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.39','Epi-cedrol synthase','8-epicedrol synthase','Magnesium.','Similar to many other plant terpenoid synthases, this enzyme producesmany products from a single substrate.The predominant product is the cyclic sesquiterpenoid alcohol, 8-epi-cedrol, with minor products including cedrol and the olefins alpha-cedrene, beta-cedrene, (E)-beta-farnesene and (E)-alpha-bisabolene.','(2E,6E)-farnesyl diphosphate + H(2)O = 8-epi-cedrol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.40','(Z)-gamma-bisabolene synthase','NULL','NULL','This sesquiterpenoid enzyme is constitutively expressed in the root,hydathodes and stigma of the plant Arabidopsis thaliana.If the leaves of the plant are wounded, e.g. by cutting, the enzymeis also induced close to the wound site.The sesquiterpenoids (E)-nerolidol and alpha-bisabolol are alsoproduced by this enzyme as minor products.','(2E,6E)-farnesyl diphosphate = (Z)-gamma-bisabolene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.41','Elisabethatriene synthase','Elisabethatriene cyclase','Magnesium.','Also able to use farnesyl diphosphate and geranyl diphosphate.','Geranylgeranyl diphosphate = elisabethatriene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.42','Aphidicolan-16-beta-ol synthase','ACS','NULL','A bifunctional enzyme which also has EC 5.5.1.14 activity.Aphidicolan-16-beta-ol is a precursor of aphidicolin, a specificinhibitor of EC 2.7.7.7.','9-alpha-copalyl diphosphate + H(2) = aphidicolan-16-beta-ol +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.43','Fusicocca-2,10(14)-diene synthase','Fusicoccadiene synthase','NULL','A multifunctional enzyme with EC 2.5.1.29 activity.','Geranylgeranyl diphosphate = fusicocca-2,10(14)-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.44','Isopimara-7,15-diene synthase','NULL','NULL','The enzyme only gave isopimara-7,15-diene.Formerly EC 4.2.3.n9.','(+)-copalyl diphosphate = isopimara-7,15-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.45','Phyllocladan-16-alpha-ol synthase','PaDC1','NULL','NULL','(+)-copalyl diphosphate + H(2)O = phyllocladan-16-alpha-ol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.46','Alpha-farnesene synthase','(E,E)-alpha-farnesene synthase','NULL','NULL','(2E,6E)-farnesyl diphosphate = (3E,6E)-alpha-farnesene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.47','Beta-farnesene synthase','Farnesene synthase','NULL','NULL','(2E,6E)-farnesyl diphosphate = (E)-beta-farnesene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.48','(3S,6E)-nerolidol synthase','(3S)-(E)-nerolidol synthase','NULL','The enzyme catalyzes a step in the formation of 4,8-dimethyl-1,3(E),7-nonatriene, a key signal molecule in induced plant defensemediated by the attraction of enemies of herbivores (2).Nerolidol is a naturally occurring sesquiterpene found in theessential oils of many types of plants.','(2E,6E)-farnesyl diphosphate + H(2)O = (3S,6E)-nerolidol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.49','(3R,6E)-nerolidol synthase','NULL','NULL','The enzyme catalyzes a step in the formation of (3E)-4,8-dimethyl-1,3,7-nonatriene, a key signal molecule in induced plant defensemediated by the attraction of enemies of herbivores (1).Nerolidol is a naturally occurring sesquiterpene found in theessential oils of many types of plants.','(2E,6E)-farnesyl diphosphate + H(2)O = (3R,6E)-nerolidol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.50','(+)-alpha-santalene synthase ((2Z,6Z)-farnesyl diphosphate cyclizing)','NULL','NULL','The enzyme synthesizes a mixture of sesquiterpenoids from (2Z,6Z)-farnesyl diphosphate.Following dephosphorylation of (2Z,6Z)-farnesyl diphosphate,the (2Z,6Z)-farnesyl carbocation is converted to either the (6R)- orthe (6S)-bisabolyl cations depending on the stereochemistry of the6,1 closure.The (6R)-bisabolyl cation will then lead to the formation of (+)-alpha-santalene (EC 4.2.3.50), while the (6S)-bisabolyl cation willgive rise to (+)-endo-beta-bergamotene (see EC 4.2.3.53) as well as(-)-endo-alpha-bergamotene (see EC 4.2.3.54).Small amounts of (-)-epi-beta-santalene are also formed from the(6R)-bisabolyl cation and small amounts of (-)-exo-alpha-bergamoteneare formed from the (6S)-bisabolyl cation.','(2Z,6Z)-farnesyl diphosphate = (+)-alpha-santalene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.51','Beta-phellandrene synthase (neryl-diphosphate-cyclizing)','Phellandrene synthase 1','NULL','The enzyme from Solanum lycopersicum has very poor affinity withgeranyl diphosphate as substrate.Catalyzes the formation of the acyclic myrcene and ocimene as majorproducts in addition to beta-phellandrene.','Neryl diphosphate = beta-phellandrene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.52','(4S)-beta-phellandrene synthase (geranyl-diphosphate-cyclizing)','(-)-(4S)-beta-phellandrene synthase','Manganese.','Mg(2+) is not effective.Some (-)-alpha-phellandrene is also formed.The reaction involves a 1,3-hydride shift.Formerly EC 4.2.3.n5.','Geranyl diphosphate = (4S)-beta-phellandrene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.53','(+)-endo-beta-bergamotene synthase ((2Z,6Z)-farnesyl diphosphatecyclizing)','NULL','NULL','The enzyme synthesizes a mixture of sesquiterpenoids from (2Z,6Z)-farnesyl diphosphate.Following dephosphorylation of (2Z,6Z)-farnesyl diphosphate,the (2Z,6Z)-farnesyl carbocation is converted to either the (6R)- orthe (6S)-bisabolyl cations depending on the stereochemistry of the6,1 closure.The (6R)-bisabolyl cation will then lead to the formation of (+)-alpha-santalene (see EC 4.2.3.50), while the (6S)-bisabolyl cationwill give rise to (-)-endo-alpha-bergamotene (see EC 4.2.3.54),as well as (+)-endo-beta-bergamotene.Small amounts of (-)-epi-beta-santalene are also formed from the(6R)-bisabolyl cation and small amounts of (-)-exo-alpha-bergamoteneare formed from the (6S)-bisabolyl cation.','(2Z,6Z)-farnesyl diphosphate = (+)-endo-beta-bergamotene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.54','(-)-endo-alpha-bergamotene synthase ((2Z,6Z)-farnesyl diphosphatecyclizing)','NULL','NULL','The enzyme synthesizes a mixture of sesquiterpenoids from (2Z,6Z)-farnesyl diphosphate.Following dephosphorylation of (2Z,6Z)-farnesyl diphosphate,the (2Z,6Z)-farnesyl carbocation is converted to either the (6R)- orthe (6S)-bisabolyl cations depending on the stereochemistry of the6,1 closure.The (6R)-bisabolyl cation will then lead to the formation of (+)-alpha-santalene (see EC 4.2.3.50), while the (6S)-bisabolyl cationwill give rise to (+)-endo-beta-bergamotene (EC 4.2.3.53) as well as(-)-endo-alpha-bergamotene.Small amounts of (-)-epi-beta-santalene are also formed from the(6R)-bisabolyl cation and small amounts of (-)-exo-alpha-bergamoteneare formed from the (6S)-bisabolyl cation (1).','(2Z,6Z)-farnesyl diphosphate = (-)-endo-alpha-bergamotene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.55','(S)-beta-bisabolene synthase','NULL','Manganese or magnesium.','The synthesis of (S)-beta-macrocarpene from (2E,6E)-farnesyldiphosphate proceeds in two steps.The first step is the cyclization to (S)-beta-bisabolene.The second step is the isomerization to (S)-beta-macrocarpene (cf.EC 5.5.1.17, (S)-beta-macrocarpene synthase).','(2E,6E)-farnesyl diphosphate = (S)-beta-bisabolene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.56','Gamma-humulene synthase','NULL','Manganese or magnesium.','Has a broad product range.While gamma-humulene is the major product, sibirene, longifolene,beta-himachalene, gamma-himachalene and alpha-himachalene areproduced in appreciable amounts and further 47 other minor productscan also be detected.The enzyme can use geranyl diphosphate as a substrate, producingmainly limonene and terpinolene as the major products.Formerly EC 4.2.3.n3.','Farnesyl-diphosphate = gamma-humulene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.57','(-)-beta-caryophyllene synthase','NULL','NULL','Widely distributed in higher plants, cf. EC 4.2.3.89.','(2E,6E)-farnesyl diphosphate = (-)-beta-caryophyllene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.58','Longifolene synthase','NULL','NULL','As well as 61% longifolene the enzyme gives 15% of alpha-longipinene,6% longicyclene and traces of other sesquiterpenoids.','(2E,6E)-farnesyl diphosphate = longifolene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.59','(E)-gamma-bisabolene synthase','NULL','NULL','NULL','(2E,6E)-farnesyl diphosphate = (E)-gamma-bisabolene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.60','Germacrene C synthase','NULL','NULL','NULL','(2E,6E)-farnesyl diphosphate = germacrene C + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.61','5-epiaristolochene synthase','5-epi-aristolochene synthase','NULL','Initial cyclization gives (+)-germacrene A in an enzyme bound formwhich is not released to the medium.','(2E,6E)-farnesyl diphosphate = (+)-5-epiaristolochene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.62','(-)-gamma-cadinene synthase ((2Z,6E)-farnesyl diphosphate cyclizing)','(-)-gamma-cadinene cyclase','NULL','Formerly EC 4.2.3.n10.','(2Z,6E)-farnesyl diphosphate = (-)-gamma-cadinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.63','(+)-cubenene synthase','NULL','Magnesium.','NULL','(2E,6E)-farnesyl diphosphate = (+)-cubenene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.64','(+)-epicubenol synthase','Farnesyl pyrophosphate cyclase','Magnesium.','In the bacteria Streptomyces and the liverwort Heteroscyphus the (+)-isomer is formed in contrast to higher plants where the (-)-isomer isformed.','(2E,6E)-farnesyl diphosphate + H(2)O = (+)-epicubenol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.65','Zingiberene synthase','Alpha-zingiberene synthase','NULL','Formerly EC 4.2.3.n12.','(2E,6E)-farnesyl diphosphate = zingiberene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.66','Beta-selinene cyclase','NULL','NULL','NULL','(2E,6E)-farnesyl diphosphate = beta-selinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.67','Cis-muuroladiene synthase','NULL','NULL','The recombinant enzyme from black peppermint (Mentha x piperita) gavea mixture of cis-muurola-3,5-diene (45%) and cis-muurola-4(14),5-diene (43%).','(1) (2E,6E)-farnesyl diphosphate = cis-muurola-3,5-diene + diphosphate.(2) (2E,6E)-farnesyl diphosphate = cis-muurola-4(14),5-diene +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.68','Beta-eudesmol synthase','NULL','NULL','The recombinant enzyme from ginger (Zingiber zerumbet) gives 62.6%beta-eudesmol, 16.8% 10-epi-gamma-eudesmol (cf. EC 4.2.3.84),10% alpha-eudesmol (cf. EC 4.2.3.85), and 5.6% aristolene.','(2E,6E)-farnesyl diphosphate + H(2)O = beta-eudesmol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.69','(+)-alpha-barbatene synthase','NULL','NULL','The recombinant enzyme from the plant Arabidopsis thaliana produces27.3% alpha-barbatene, 17.8% thujopsene (cf. EC 4.2.3.79) and 9.9%beta-chamigrene (cf. 4.2.3.78) plus traces of other sesquiterpenoids.','(2E,6E)-farnesyl diphosphate = (+)-alpha-barbatene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.70','Patchoulol synthase','NULL','NULL','NULL','(2E,6E)-farnesyl diphosphate + H(2)O = patchoulol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.71','(E,E)-germacrene B synthase','NULL','NULL','NULL','(2E,6E)-farnesyl diphosphate = (E,E)-germacrene B + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.72','Alpha-gurjunene synthase','NULL','NULL','Initial cyclization probably gives biyclogermacrene in an enzymebound form which is not released to the medium.The enzyme from Solidago canadensis also forms a small amount of (+)-gamma-gurjunene.','(2E,6E)-farnesyl diphosphate = (-)-alpha-gurjunene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.73','Valencene synthase','NULL','NULL','The recombinant enzyme from Vitis vinifera gave 49.5% (+)-valenceneand 35.5% (-)-7-epi-alpha-selinene.Initial cyclization gives (+)-germacrene A in an enzyme bound formwhich is not released to the medium.','(2E,6E)-farnesyl diphosphate = (+)-valencene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.74','Presilphiperfolanol synthase','NULL','Magnesium.','Presilphiperfolan-8-beta-ol is the precursor of botrydial,a phytotoxic sesquiterpene metabolite secreted by the fungusBotryotinia fuckeliana (Botrytis cinerea), the causal agent of graymold disease in plants.','(2E,6E)-farnesyl diphosphate + H(2)O = presilphiperfolan-8-beta-ol +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.75','(-)-germacrene D synthase','NULL','NULL','In Solidago canadensis the biosynthesis results in the pro-R hydrogenat C-1 of the farnesy diphosphate ending up at C-11 of the (-)-germacrene D.With Streptomyces coelicolor the pro-S hydrogen at C-1 ends up atC-11 of the (-)-germacrene D.Formerly EC 4.2.3.n13.','(2E,6E)-farnesyl diphosphate = (-)-germacrene D + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.76','(+)-delta-selinene synthase','NULL','NULL','Initial cyclization gives germacrene C in an enzyme bound form whichis not released to the medium.Formerly EC 4.2.3.n2.','(2E,6E)-farnesyl diphosphate = (+)-delta-selinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.77','(+)-germacrene D synthase','NULL','Manganese or magnesium.','The formation of (+)-germacrene D involves a 1,2-hydride shiftwhereas for (-)-germacrene D there is a 1,3-hydride shift (seeEC 4.2.3.75).','(2E,6E)-farnesyl diphosphate = (+)-germacrene D + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.78','Beta-chamigrene synthase','NULL','NULL','The recombinant enzyme from the plant Arabidopsis thaliana produces27.3% (+)-alpha-barbatene, 17.8% (+)-thujopsene and 9.9% (+)-beta-chamigrene plus traces of other sesquiterpenoids.See EC 4.2.3.69 and EC 4.2.3.79.','(2E,6E)-farnesyl diphosphate = (+)-beta-chamigrene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.79','Thujopsene synthase','NULL','NULL','The recombinant enzyme from the plant Arabidopsis thaliana produces27.3% (+)-alpha-barbatene, 17.8% (+)-thujopsene and 9.9% (+)-beta-chamigrene plus traces of other sesquiterpenoids.See EC 4.2.3.69 and EC 4.2.3.78.','(2E,6E)-farnesyl diphosphate = (+)-thujopsene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.80','Alpha-longipinene synthase','NULL','NULL','The enzyme from Norway spruce produces longifolene as the mainproduct (cf. EC 4.2.3.58).Alpha-longipinene constitutes about 15% of the total products.','(2E,6E)-farnesyl diphosphate = alpha-longipinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.81','Exo-alpha-bergamotene synthase','Trans-alpha-bergamotene synthase','NULL','The enzyme synthesizes a mixture of sesquiterpenoids from (2E,6E)-farnesyl diphosphate.As well as (-)-exo-alpha-bergamotene (74%) there were (E)-nerolidol(10%), (Z)-alpha-bisabolene (6%), (E)-beta-farnesene (5%) and beta-sesquiphellandrene (1%).','(2E,6E)-farnesyl diphosphate = (-)-exo-alpha-bergamotene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.82','Alpha-santalene synthase','NULL','NULL','The enzyme synthesizes a mixture of sesquiterpenoids from (2E,6E)-farnesyl diphosphate.As well as (+)-alpha-santalene, (-)-beta-santalene and (-)-exo-alpha-bergamotene are formed with traces of (+)-epi-beta-santalene.See EC 4.2.3.83 and EC 4.2.3.81.Cf. EC 4.2.3.50.','(2E,6E)-farnesyl diphosphate = (+)-alpha-santalene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.83','Beta-santalene synthase','NULL','NULL','The enzyme synthesizes a mixture of sesquiterpenoids from (2E,6E)-farnesyl diphosphate.As well as (-)-beta-santalene (+)-alpha-santalene and (-)-exo-alpha-bergamotene are formed with traces of (+)-epi-beta-santalene.See EC 4.2.3.82 and EC 4.2.3.81.','(2E,6E)-farnesyl diphosphate = (-)-beta-santalene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.84','10-epi-gamma-eudesmol synthase','NULL','NULL','The recombinant enzyme from ginger (Zingiber zerumbet) gives 62.6%beta-eudesmol, 16.8% 10-epi-gamma-eudesmol, 10% alpha-eudesmol,and 5.6% aristolene.Cf. EC 4.2.3.68 and EC 4.2.3.85.','(2E,6E)-farnesyl diphosphate + H(2)O = 10-epi-gamma-eudesmol +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.85','Alpha-eudesmol synthase','NULL','NULL','The recombinant enzyme from ginger (Zingiber zerumbet) gives 62.6%beta-eudesmol, 16.8% 10-epi-gamma-eudesmol, 10% alpha-eudesmol,and 5.6% aristolene.Cf. EC 4.2.3.68 and EC 4.2.3.84.','(2E,6E)-farnesyl diphosphate + H(2)O = alpha-eudesmol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.86','7-epi-alpha-selinene synthase','NULL','NULL','The recombinant enzyme from Vitis vinifera forms 49.5% (+)-valencene(cf. EC 4.2.3.73) and 35.5% (-)-7-epi-alpha-selinene.Initial cyclization gives (+)-germacrene A in an enzyme bound formwhich is not released to the medium.','(2E,6E)-farnesyl diphosphate = 7-epi-alpha-selinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.87','Alpha-guaiene synthase','NULL','Magnesium.','The enzyme from Pogostemon cablin gives 13% alpha-guaiene as well as37% (-)-patchoulol (see EC 4.2.3.70), 13% delta-guaiene (seeEC 4.2.3.93), and traces of at least ten other sesquiterpenoids.In Aquilaria crassna three clones of the enzyme gave about 80% delta-guaiene and 20% alpha-guaiene, with traces of alpha-humulene.A fourth clone gave 54% delta-guaiene and 45% alpha-guaiene.','(2E,6E)-farnesyl diphosphate = alpha-guaiene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.88','Viridiflorene synthase','TPS31','NULL','Viridiflorene is the only product of this enzyme from Solanumlycopersicum.','(2E,6E)-farnesyl diphosphate = viridiflorene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.89','(+)-beta-caryophyllene synthase','NULL','NULL','A multifunctional enzyme which also converts the (+)-beta-caryophyllene to (+)-caryolan-1-ol (see EC 4.2.1.138).Cf. EC 4.2.3.57.','(2E,6E)-farnesyl diphosphate + H(2)O = (+)-beta-caryophyllene +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.90','5-epi-alpha-selinene synthase','8a-epi-alpha-selinene synthase','Magnesium.','The enzyme forms 5-epi-alpha-selinene possibly via germecrene A or a1,6-hydride shift mechanism.','(2E,6E)-farnesyl diphosphate = 5-epi-alpha-selinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.91','Cubebol synthase','NULL','Magnesium.','The enzyme gives 28% cubebol, 29% (-)-germacrene D, 10% (+)-delta-cadinene and traces of several other sesquiterpenoids.See also EC 4.2.3.75 and EC 4.2.3.13.','(2E,6E)-farnesyl diphosphate + H(2)O = cubebol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.92','(+)-gamma-cadinene synthase','NULL','NULL','The cloned enzyme from the melon, Cucumis melo, gave mainly delta-and gamma-cadinene with traces of several other sesquiterpenoids cf.EC 4.2.3.62 and EC 4.2.3.13.','(2E,6E)-farnesyl diphosphate = (+)-gamma-cadinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.93','Delta-guaiene synthase','NULL','Magnesium.','In Aquilaria crassna three clones of the enzyme gave about 80% delta-guaiene and 20% alpha-guaiene (see also EC 4.2.3.87).A fourth clone gave 54% delta-guaiene and 45% alpha-guaiene.The enzyme from Pogostemon cablin gives 13% delta-guaiene as well as37% (-)-patchoulol (see EC 4.2.3.70), 13% alpha-guaiene (seeEC 4.2.3.87), and traces of at least ten other sesquiterpenoids.','(2E,6E)-farnesyl diphosphate = delta-guaiene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.94','Gamma-curcumene synthase','NULL','NULL','One of five sesquiterpenoid synthases in Pogostemon cablin(patchouli).','(2E,6E)-farnesyl diphosphate = gamma-curcumene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.95','(-)-alpha-cuprenene synthase','NULL','NULL','The enzyme from Coprinus cinereus produces (-)-alpha-cuprenene withhigh selectivity.','(2E,6E)-farnesyl diphosphate = (-)-alpha-cuprenene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.96','Avermitilol synthase','NULL','Magnesium.','The recombinent enzyme gives avermitilol (85%) plus traces ofgermacrene A, germacrene B and viridiflorol.The (1S)-hydrogen of farnesyl diphosphate is retained.','(2E,6E)-farnesyl diphosphate + H(2)O = avermitilol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.97','(-)-delta-cadinene synthase','NULL','NULL','The cyclization mechanism involves an intermediate nerolidyldiphosphate leading to a helminthogermacradienyl cation.Following a 1,3-hydride shift of the original 1-pro-S hydrogen of(2E,6E)-farnesyl diphosphate, cyclization and deprotonation gives(-)-delta-cadinene.','(2E,6E)-farnesyl diphosphate = (-)-delta-cadinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.98','(+)-T-muurolol synthase','NULL','NULL','The cyclization mechanism involves an intermediate nerolidyldiphosphate leading to a helminthogermacradienyl cation.After a 1,3-hydride shift of the original 1-pro-(S) hydrogen offarnesyl diphosphate, cyclization and deprotonation result in (+)-T-muurolol.','(2E,6E)-farnesyl diphosphate + H(2)O = (+)-T-muurolol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.99','Labdatriene synthase','NULL','NULL','The enzyme from rice (Oryza sativa), expressed in Escherichia coli,also produces ent-sandaracopimara-8(14),15-diene from ent-copalyldiphosphate, another naturally occurring copalyl isomer in rice (cf.EC 4.2.3.29).','9-alpha-copalyl diphosphate = (12E)-9-alpha-labda-8(17),12,14-triene +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.100','Bicyclogermacrene synthase','NULL','Manganese or magnesium.','The enzyme from oregano (Origanum vulgare) gives mainlybicyclogermacrene with Mn(2+) as a cofactor.With Mg(2+) a more complex mixture is produced.','(2E,6E)-farnesyl diphosphate = bicyclogermacrene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.101','7-epi-sesquithujene synthase','TPS4-B73','Manganese or magnesium.','The enzyme from Zea mays, variety B73, gives mainly 7-epi-sesquithujene with (S)-beta-bisabolene and traces of othersesquiterpenoids, cf. EC 4.2.3.55.The product ratio is dependent on which metal ion is present.7-epi-sesquithujene is an attractant for the emerald ash borerbeetle.','(2E,6E)-farnesyl diphosphate = 7-epi-sesquithujene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.102','Sesquithujene synthase','TPS5-del1','Manganese or magnesium.','The enzyme from Zea mays, variety Delprim, gives mainly sesquithujenewith (S)-beta-bisabolene and (E)-beta-farnesene plus traces of othersesquiterpenoids, cf. EC 4.2.3.55 and EC 4.2.3.47.The exact product ratio is dependent on which metal ion is present.','(2E,6E)-farnesyl diphosphate = sesquithujene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.103','Ent-isokaurene synthase','Ent-isokaur-15-ene synthase','NULL','Two enzymes of the rice sub-species Oryza sativa subsp. indica,OsKSL5 and OsKSL6, produce ent-isokaurene.A variant of OsKSL5 from the sub-species Oryza sativa subsp. japonicaproduces ent-pimara-8(14),15-diene instead (cf. EC 4.2.3.30).Formerly EC 4.2.3.n8.','Ent-copalyl diphosphate = ent-isokaurene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.104','Alpha-humulene synthase','ZSS1','NULL','The enzyme from Zingiber zerumbet, shampoo ginger, also gives tracesof beta-caryophyllene.','(2E,6E)-farnesyl diphosphate = alpha-humulene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.105','Tricyclene synthase','NULL','NULL','The enzyme from Solanum lycopersicum (tomato) gives a mixture oftricyclene, camphene, beta-myrcene, limonene, and traces of severalother monoterpenoids.See EC 4.2.3.117, EC 4.2.3.15 and EC 4.2.3.16.','Geranyl diphosphate = tricyclene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.106','(E)-beta-ocimene synthase','Beta-ocimene synthase','NULL','Widely distributed in plants, which release beta-ocimene whenattacked by herbivorous insects.','Geranyl diphosphate = (E)-beta-ocimene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.107','(+)-car-3-ene synthase','3-carene cyclase','NULL','The enzyme reacts with (3S)-linalyl diphosphate twice as rapidly asgeranyl diphosphate, but 25 times as rapidly as (3R)-linalyldiphosphate.It is assumed that (3S)-linalyl diphosphate is normally formed as anenzyme bound intermediate in the reaction.In the reaction the 5-pro-R hydrogen of geranyl diphosphate iseliminated during cyclopropane ring formation.In Picea abies (Norway spruce) and Picea sitchensis (Sitka spruce)terpinolene is also formed.See EC 4.2.3.113.(+)-car-3-ene is associated with resistance of Picea sitchensis(Sitka spruce) to white pine weevil.','Geranyl diphosphate = (+)-car-3-ene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.108','1,8-cineole synthase','1,8-cineole cyclase','Zinc or manganese.','1,8-cineole is the main product from the enzyme with just traces ofother monoterpenoids.The oxygen atom is derived from water.The reaction proceeds via linalyl diphosphate and alpha-terpineol,the stereochemistry of both depends on the organism.However neither intermediate can substitute for geranyl diphosphate.The reaction in Salvia officinalis (sage) proceeds via (-)-(3R)-linalyl diphosphate while that in Arabidopsis (rock cress) proceedsvia (+)-(3S)-linalyl diphosphate.','Geranyl diphosphate + H(2)O = 1,8-cineole + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.109','(-)-sabinene synthase','NULL','Magnesium.','Isolated from Pinus contorta (lodgepole pine) as cyclase I and fromConocephalum conicum (liverwort).','Geranyl diphosphate = (-)-sabinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.110','(+)-sabinene synthase','NULL','NULL','Isolated from Salvia officinalis (sage).The recombinant enzyme gave 63% (+)-sabinene, 21% gamma-terpinene,and traces of other monoterpenoids.See EC 4.2.3.114.','Geranyl diphosphate = (+)-sabinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.111','(-)-alpha-terpineol synthase','NULL','NULL','The enzyme has been characterized from Vitis vinifera (grape).Also forms some 1,8-cineole and traces of other monoterpenoids.','Geranyl diphosphate + H(2)O = (-)-alpha-terpineol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.112','(+)-alpha-terpineol synthase','NULL','NULL','The enzyme has been characterized from Santalum album (sandalwood).Also forms some (-)-limonene and traces of other monoterpenoids.See EC 4.2.3.16.','Geranyl diphosphate + H(2)O = (+)-alpha-terpineol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.113','Terpinolene synthase','NULL','Manganese or magnesium.','Mn(2+) is less effective as cofactor than Mg(2+) and product ratiochanges.Forms traces of other monoterpenoids.Formerly EC 4.2.3.n6.','Geranyl-diphosphate = terpinolene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.114','Gamma-terpinene synthase','NULL','Magnesium.','Isolated from Thymus vulgaris (thyme), Citrus limon (lemon), Citrusunshiu (satsuma) and Origanum vulgare (oregano).The reaction involves a 1,2-hydride shift.The 5-pro-S hydrogen of geranyl diphosphate is lost.Traces of several other monoterpenoids are formed in addition togamma-terpinene.','Geranyl diphosphate = gamma-terpinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.115','Alpha-terpinene synthase','NULL','NULL','The enzyme has been characterized from Dysphania ambrosioides(American wormseed).The enzyme will also use (3R)-linalyl diphosphate.The reaction involves a 1,2-hydride shift.The 1-pro-(S) hydrogen of geranyl diphosphate is lost.','Geranyl diphosphate = alpha-terpinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.116','(+)-camphene synthase','NULL','NULL','Cyclase I of Salvia officinalis (sage) gives about equal parts (+)-camphene and (+)-alpha-pinene.(3R)-linalyl diphosphate can also be used by the enzyme in preferenceto (3S)-linalyl diphosphate.See also EC 4.2.3.121.','Geranyl diphosphate = (+)-camphene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.117','(-)-camphene synthase','NULL','NULL','(-)-Camphene is the major product in Abies grandis (grand fir) withtraces of other monoterpenoids.In Pseudotsuga menziesii (Douglas-fir) there are about equal parts of(-)-camphene and (-)-alpha-pinene with traces of four othermonoterpenoids.In Solanum lycopersicum (tomato) tricyclene, beta-myrcene, limonene,and traces of several other monoterpenoids are also formed.See also EC 4.2.3.15, EC 4.2.3.16, EC 4.2.3.119 and EC 4.2.3.105.Formerly EC 4.2.3.n4.','Geranyl-diphosphate = (-)-camphene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.118','2-methylisoborneol synthase','2-MIB cyclase','Magnesium.','The product, 2-methylisoborneol, is a characteristc odiferouscompound with a musty smell produced by soil microorganisms.Formerly EC 4.2.3.n14.','(E)-2-methylgeranyl diphosphate + H(2)O = 2-methylisoborneol +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.119','(-)-alpha-pinene synthase','(-)-alpha-pinene cyclase','NULL','Cyclase II of Salvia officinalis (sage) gives about equal parts (-)-alpha-pinene, (-)-beta-pinene and (-)-camphene, plus traces of othermonoterpenoids.(3S)-Linalyl diphosphate can also be used by the enzyme in preferenceto (3R)-linalyl diphosphate.The 4-pro-S-hydrogen of geranyl diphosphate is lost.The enzyme from Abies grandis (grand fir) gives roughly equal parts(-)-alpha-pinene and (-)-beta-pinene.However the clone ag11 gave 35% (-)-limonene, 24% (-)-alpha-pineneand 20% (-)-beta-phellandrene.Synthase I from Pinus taeda (loblolly pine) produces (-)-alpha-pinenewith traces of (-)-beta-pinene.The enzyme from Picea sitchensis (Sika spruce) forms 70% (-)-alpha-pinene and 30% (-)-beta-pinene.The recombinant PmeTPS1 enzyme from Pseudotsuga menziesii (Douglasfir) gave roughly equal proportions of (-)-alpha-pinene and (-)-camphene plus traces of other monoterpenoids.See also EC 4.2.3.120, EC 4.2.3.117, EC 4.2.3.16, and EC 4.2.3.52.Formerly EC 4.1.99.8, EC 4.2.3.14, and EC 4.2.3.n7.','Geranyl-diphosphate = (-)-alpha-pinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.120','(-)-beta-pinene synthase','(-)-(1S,5S)-pinene synthase','NULL','Cyclase II of Salvia officinalis (sage) produces about equal parts(-)-alpha-pinene, (-)-beta-pinene and (-)-camphene, plus traces ofother monoterpenoids.The enzyme can also use (3S)-Linalyl diphosphate (preferred to (3R)-linalyl diphosphate).The enzyme from Abies grandis (grand fir) produces roughly equalparts of (-)-alpha-pinene and (-)-beta-pinene.Cyclase IV from Pinus contorta (lodgepole pine) produces 63% (-)-beta-pinene, 26% 3-carene, and traces of alpha-pinene.Synthase III from Pinus taeda (loblolly pine) forms (-)-beta-pinenewith traces of alpha-pinene.A cloned enzyme from Artemisia annua (sweet wormwood) gave (-)-beta-pinene with traces of (-)-alpha-pinene.The enzyme from Picea sitchensis (Sika spruce) forms 30% (-)-beta-pinene and 70% (-)-alpha-pinene.See also EC 4.2.3.119, EC 4.2.3.117 and EC 4.2.3.107.Formerly EC 4.1.99.8 and EC 4.2.3.14.','Geranyl diphosphate = (-)-beta-pinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.121','(+)-alpha-pinene synthase','(+)-alpha-pinene cyclase','NULL','Cyclase I of Salvia officinalis (sage) gives about equal parts (+)-alpha-pinene and (+)-camphene, whereas cyclase III gives about equalparts of (+)-alpha-pinene and (+)-beta-pinene.(3R)-linalyl diphosphate can also be used by the enzyme in preferenceto (3S)-linalyl diphosphate.The 4-pro-R-hydrogen of geranyl diphosphate is lost.With synthase II of Pinus taeda (loblolly pine) (+)-beta-pinene wasthe only product.See also EC 4.2.3.122 and EC 4.2.3.116.','Geranyl diphosphate = (+)-alpha-pinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.122','(+)-beta-pinene synthase','(+)-pinene cyclase','NULL','Cyclase III from Salvia officinalis (sage) gives roughly equal partsof (+)-beta-pinene and (+)-alpha-pinene.See EC 4.2.3.121.','Geranyl diphosphate = (+)-beta-pinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.123','Beta-sesquiphellandrene synthase','Tps1','NULL','NULL','(2E,6E)-farnesyl diphosphate = beta-sesquiphellandrene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.124','2-deoxy-scyllo-inosose synthase','NULL','Cobalt.','Involved in the biosynthetic pathways of several clinically importantaminocyclitol antibiotics, including kanamycin, butirosin, neomycinand ribostamycin.Requires an NAD(+) cofactor, which is transiently reduced during thereaction.The enzyme from the bacterium Bacillus circulans forms a complex withthe glutamine amidotransferase subunit of pyridoxal 5-phosphatesynthase (EC 4.3.3.6), which appears to stabilize the complex.','D-glucose-6-phosphate = 2-deoxy-L-scyllo-inosose + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.125','Alpha-muurolene synthase','NULL','NULL','The enzyme has been characterized from the fungus Coprinus cinereus.Also gives germacrene A and gamma-muurolene, see EC 4.2.3.23 andEC 4.2.3.126.','(2E,6E)-farnesyl diphosphate = alpha-muurolene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.126','Gamma-muurolene synthase','NULL','NULL','The enzyme has been characterized from the fungus Coprinus cinereus.Also gives germacrene A and alpha-muurolene, see EC 4.2.3.23 andEC 4.2.3.125.','(2E,6E)-farnesyl diphosphate = gamma-muurolene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.127','Beta-copaene synthase','Cop4','Magnesium.','Isolated from the fungus Coprinus cinereus.The enzyme also forms (+)-delta-cadinene, beta-cubebene, (+)-sativeneand traces of several other sequiterpenoids.beta-Copaene is formed in the presence of Mg(2+) but not Mn(2+).See EC 4.2.3.13 EC 4.2.3.128 and EC 4.2.3.129.','(2E,6E)-farnesyl diphosphate = beta-copaene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.128','Beta-cubebene synthase','NULL','NULL','Isolated from the fungus Coprinus cinereus.The enzyme also forms (+)-delta-cadinene, beta-copaene, (+)-sativeneand traces of several other sequiterpenoids.It is found in many higher plants such as Magnolia grandiflora(Southern Magnolia) together with germacrene A.See EC 4.2.3.13 EC 4.2.3.127 EC 4.2.3.129 and EC 4.2.3.23.','(2E,6E)-farnesyl diphosphate = beta-cubebene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.129','(+)-sativene synthase','Cop4','NULL','Isolated from the fungus Coprinus cinereus.The enzyme also forms (+)-delta-cadinene, beta-copaene, beta-cubebene, and traces of several other sequiterpenoids.See EC 4.2.3.13 EC 4.2.3.127 and EC 4.2.3.128.','(2E,6E)-farnesyl diphosphate = (+)-sativene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.130','Tetraprenyl-beta-curcumene synthase','NULL','NULL','Isolated from Bacillus subtilis.This sesquarterpene is present in a number of Bacillus species.','All-trans-heptaprenyl diphosphate = tetraprenyl-beta-curcumene +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.131','Miltiradiene synthase','NULL','NULL','Isolated from the plant Selaginella moellendorffii as amutifunctional enzyme that also catalyzes EC 5.5.1.12.In the plant Salvia miltiorrhiza the two enzymes are separateentities.','(+)-copalyl diphosphate = miltiradiene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.132','Neoabietadiene synthase','NULL','NULL','Isolated from Abies grandis (grand fir).This class I enzyme forms about equal proportions of abietadiene,levopimaradiene and neoabietadiene; see also EC 4.2.3.18 andEC 4.2.3.32.An X-ray study of this multifunctional enzyme showed that the class Iactivity is in the alpha domain, while EC 5.5.1.12 activity (a classII activity) is in the beta and gamma domains.In Pinus taeda (loblolly pine) the major product is levopimaradiene,with less abietadiene and neoabietadiene.','(+)-copalyl diphosphate = neoabietadiene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.133','Alpha-copaene synthase','NULL','NULL','Isolated from Helianthus annuus (sunflower).The enzyme also produces beta-caryophyllene, delta-cadinene andtraces of other sesquiterpenoids.See EC 4.2.3.13 and EC 4.2.3.57.','(2E,6E)-farnesyl diphosphate = alpha-copaene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.134','5-phosphonooxy-L-lysine phospho-lyase','5-phosphohydroxy-L-lysine ammoniophospholyase','Pyridoxal 5-phosphate.','Has no activity with phosphoethanolamine (cf. EC 4.2.3.2).','(5R)-5-phosphonooxy-L-lysine + H(2)O = (S)-2-amino-6-oxohexanoate + NH(3)+ phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.135','Delta(6)-protoilludene synthase','6-protoilludene synthase','NULL','Isolated from the fungus Armillaria gallica.Delta(6)-Protoilludene is the first step in the biosynthesis of themelleolides.','(2E,6E)-farnesyl diphosphate = Delta(6)-protoilludene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.136','Alpha-isocomene synthase','NULL','NULL','Isolated from the roots of the plant Matricaria chamomilla var.recutita (chamomile).The enzyme also produced traces of five other sesquiterpenoids.','(2E,6E)-farnesyl diphosphate = (-)-alpha-isocomene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.137','(E)-2-epi-beta-caryophyllene synthase','2-epi-(E)-beta-caryophyllene synthase','NULL','Isolated from the plant Selaginella moellendorfii.The enzyme also gives two other sesquiterpenoids.','(2E,6E)-farnesyl diphosphate = (E)-2-epi-beta-caryophyllene +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.138','(+)-epi-alpha-bisabolol synthase','NULL','NULL','Isolated from the plant Phyla dulcis (Aztec sweet herb).(+)-epi-alpha-bisabolol is the precursor of the sweetenerhernandulcin.','(2E,6E)-farnesyl diphosphate + H(2)O = (+)-epi-alpha-bisabolol +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.139','Valerena-4,7(11)-diene synthase','NULL','NULL','Isolated from the plant Valeriana officinalis (valerian).','(2E,6E)-farnesyl diphosphate = valerena-4,7(11)-diene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.140','Cis-abienol synthase','Z-abienol synthase','NULL','Isolated from the plants Abies balsamea (balsam fir) and Nicotianatabacum (tobacco).','(13E)-8-alpha-hydroxylabd-13-en-15-yl diphosphate = cis-abienol +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.141','Sclareol synthase','NULL','Manganese or magnesium.','Isolated from the plant Salvia sclarea (clary sage).Originally thought to be synthesized in one step from geranylgeranyldiphosphate it is now known to require two enzymes, EC 4.2.1.133 andEC 4.2.3.141.Sclareol is used in perfumery.Formerly EC 3.1.7.4.','(13E)-8-alpha-hydroxylabd-13-en-15-yl diphosphate + H(2)O = sclareol +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.142','7-epizingiberene synthase ((2Z,6Z)-farnesyl diphosphate cyclizing)','NULL','NULL','Isolated from the plant Solanum habrochaites.7-Epizingiberene is a whitefly repellant.','(2Z,6Z)-farnesyl diphosphate = 7-epizingiberene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.143','Kunzeaol synthase','NULL','NULL','Isolated from the root of the plant Thapsia garganica.The enzyme also produces germacrene D, bicyclogermacrene and tracesof other sesquiterpenoids.See EC 4.2.3.77 and EC 4.2.3.100.','(2E,6E)-farnesyl diphosphate + H(2)O = kunzeaol + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.144','Geranyllinalool synthase','NULL','NULL','The enzyme is a component of the herbivore-induced indirect defensesystem.The product, (E,E)-geranyllinalool, is a precursor to the volatilecompound 4,8,12-trimethyl-1,3,7,11-tridecatetraene (TMTT), which isreleased by many plants in response to damage.','Geranylgeranyl diphosphate + H(2)O = (E,E)-geranyllinalool + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n2','Delta-selinene synthase','NULL','Magnesium.','Has a broad product range.While delta-selinene is the major product, (E,E)-germacrene B, guaia-6,9-diene, germacrene A and delta-cadinene are produced inappreciable amounts and further 29 other minor products can also bedetected.The enzyme can use geranyl diphosphate as a substrate, producingmainly limonene, (Z)-ocimene and myrcene as the major products.','(1) Farnesyl-diphosphate = delta-selinene + diphosphate.(2) Farnesyl-diphosphate = (E,E)-germacrene B + diphosphate.(3) Farnesyl-diphosphate = guaia-6,9-diene + diphosphate.(4) Farnesyl-diphosphate = germacrene A + diphosphate.(5) Farnesyl-diphosphate = (+)-delta-cadinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n3','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n4','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n5','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n6','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n7','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n8','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n9','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n10','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n11','Selinene synthase','NULL','NULL','NULL','(1) (2E,6E)-farnesyl diphosphate = alpha-selinene + diphosphate.(2) (2E,6E)-farnesyl diphosphate = beta-selinene + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n12','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n13','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.3.n14','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.2','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.3','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.4','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.5','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.6','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.7','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.8','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.9','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.10','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.11','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.12','Carboxymethyloxysuccinate lyase','NULL','NULL','NULL','Carboxymethyloxysuccinate = fumarate + glycolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.13','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.14','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.15','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.16','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.17','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.18','DNA-(apurinic or apyrimidinic site) lyase','AP endonuclease class I','NULL','Nicking of the phosphodiester bond is due to a lyase-type reaction,not hydrolysis.This group of enzymes was previously listed as endonucleases, underthe number EC 3.1.25.2.','The C-O-P bond 3 to the apurinic or apyrimidinic site in DNA is brokenby a beta-elimination reaction, leaving a 3-terminal unsaturated sugarand a product with a terminal 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.19','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.20','2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate synthase','2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylic acid synthase','NULL','Involved in the biosynthesis of vitamin K2 (menaquinone).In most anaerobes and all Gram-positive aerobes, menaquinone is thesole electron transporter in the respiratory chain and is essentialfor their survival.It had previously been thought that the reactions carried out by thisenzyme and EC 2.2.1.9 were carried out by a single enzyme but thishas since been disproved.Formerly EC 2.5.1.64 and EC 4.2.99.n1.','5-enolpyruvoyl-6-hydroxy-2-succinylcyclohex-3-ene-1-carboxylate =(1R,6R)-6-hydroxy-2-succinylcyclohexa-2,4-diene-1-carboxylate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.21','Isochorismate lyase','Isochorismate pyruvate lyase','NULL','This enzyme is part of the pathway of salicylate formation fromchorismate, and forms an integral part of pathways that producesalicylate-derived siderophores, such as pyochelin andyersiniabactin.','Isochorismate = salicylate + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.22','Tuliposide A-converting enzyme','6-O-(4 -hydroxy-2 -methylenebutyryl)-D-glucose acyltransferase (lactone-forming)','NULL','Isolated from the plant Tulipa gesneriana (tulip).The reaction is an intramolecular transesterification producing thelactone.The enzyme also has a weak activity with 6-tuliposide B and 6-O-benzoyl-D-glucose.','6-tuliposide A = tulipalin A + D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.2.99.n1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.1','Aspartate ammonia-lyase','Aspartase','NULL','NULL','L-aspartate = fumarate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.2','Methylaspartate ammonia-lyase','Beta-methylaspartase','Cobalamin.','NULL','L-threo-3-methylaspartate = mesaconate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.3','Histidine ammonia-lyase','Histidase','NULL','Member of the aromatic amino acid lyase family, other members ofwhich are EC 4.3.1.23, EC 4.3.1.24 and EC 4.3.1.25.Catalyzes the first step in the degradation of histidine and theproduct, urocanic acid, is further metabolized to glutamate.','L-histidine = urocanate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.4','Formimidoyltetrahydrofolate cyclodeaminase','Formiminotetrahydrofolate cyclodeaminase','NULL','In eukaroytes, occurs as a bifunctional enzyme that also hasEC 2.1.2.5 activity.','5-formimidoyltetrahydrofolate = 5,10-methenyltetrahydrofolate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.5','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.6','Beta-alanyl-CoA ammonia-lyase','NULL','NULL','NULL','Beta-alanyl-CoA = acryloyl-CoA + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.7','Ethanolamine ammonia-lyase','NULL','Cobalamin.','NULL','Ethanolamine = acetaldehyde + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.8','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.9','Glucosaminate ammonia-lyase','2-amino-2-deoxy-D-gluconate ammonia-lyase','Pyridoxal 5-phosphate.','Formerly EC 4.2.1.26 and EC 4.3.1.21.','D-glucosaminate = 2-dehydro-3-deoxy-D-gluconate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.10','Serine-sulfate ammonia-lyase','NULL','NULL','NULL','L-serine O-sulfate + H(2)O = pyruvate + NH(3) + sulfate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.11','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.12','Ornithine cyclodeaminase','OCD','NAD(+).','The enzyme is a member of the mu-crystallin protein family.The reaction is stimulated by the presence of ADP or ATP and isinhibited by O(2).','L-ornithine = L-proline + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.13','Carbamoyl-serine ammonia-lyase','O-carbamoyl-L-serine deaminase','Pyridoxal 5-phosphate.','NULL','O-carbamoyl-L-serine + H(2)O = pyruvate + 2 NH(3) + CO(2).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.14','3-aminobutyryl-CoA ammonia-lyase','L-3-aminobutyryl-CoA deaminase','NULL','Hydroxylamine can replace ammonia as a substrate.Crotonoyl-pantetheine can replace crotonoyl-CoA but it is a poorersubstrate.','L-3-aminobutyryl-CoA = crotonoyl-CoA + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.15','Diaminopropionate ammonia-lyase','2,3-diaminopropanoate ammonia-lyase','Pyridoxal 5-phosphate.','Active toward both D- and L-diaminopropionate.D- and L-serine are poor substrates.','2,3-diaminopropionate + H(2)O = pyruvate + 2 NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.16','Threo-3-hydroxy-L-aspartate ammonia-lyase','L-threo-3-hydroxyaspartate dehydratase','Pyridoxal 5-phosphate.','The enzyme, purified from the bacterium Pseudomonas sp. T62,is highly specific, and does not accept any other stereoisomer of3-hydroxyaspartate.Different from EC 4.3.1.20 and EC 4.3.1.27.Requires a divalent cation such as Mn(2+), Mg(2+), or Ca(2+).','Threo-3-hydroxy-L-aspartate = oxaloacetate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.17','L-serine ammonia-lyase','L-hydroxyaminoacid dehydratase','Pyridoxal 5-phosphate or iron-sulfur.','The reaction catalyzed by both types of enzymes involves the initialelimination of water to form an enamine intermediate (hence theenzymes original classification as EC 4.2.1.13) followed bytautomerization to an imine form and hydrolysis of the C-N bond.The latter reaction, which can occur spontaneously, is also becatalyzed by EC 3.5.99.10.This reaction is also carried out by EC 4.3.1.19 from a number ofsources.','L-serine = pyruvate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.18','D-serine ammonia-lyase','D-hydroxy amino acid dehydratase','Pyridoxal 5-phosphate.','Also acts, slowly, on D-threonine.The reaction catalyzed probably involves initial elimination of water(hence the enzymes original classification as EC 4.2.1.14), followedby isomerization and hydrolysis of the product with C-N bondbreakage.Formerly EC 4.2.1.14.','D-serine = pyruvate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.19','Threonine ammonia-lyase','L-serine dehydratase','Pyridoxal 5-phosphate or iron-sulfur.','The reaction catalyzed by both types of enzymes involves the initialelimination of water to form an enamine intermediate (hence theenzymes original classification as EC 4.2.1.16), followed bytautomerization to an imine form and hydrolysis of the C-N bond.The latter reaction, which can occur spontaneously, is also becatalyzed by EC 3.5.99.10.The enzymes from a number of sources also act on L-serine,Cf. EC 4.3.1.17.','L-threonine = 2-oxobutanoate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.20','Erythro-3-hydroxy-L-aspartate ammonia-lyase','3-hydroxyaspartate dehydratase','Pyridoxal 5-phosphate.','The enzyme, which was characterized from the bacterium Paracoccusdenitrificans NCIMB 8944, is highly specific for the L-isomer oferythro-3-hydroxyaspartate.Different from EC 4.3.1.16 and EC 4.3.1.27.Requires a divalent cation such as Mn(2+), Mg(2+), or Ca(2+).Formerly EC 4.2.1.38.','Erythro-3-hydroxy-L-aspartate = oxaloacetate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.21','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.22','3,4-dihydroxyphenylalanine reductive deaminase','DOPA-reductive deaminase','NULL','Forms part of the L-phenylalanine-catabolism pathway in theanoxygenic phototrophic bacterium Rhodobacter sphaeroides OU5.NADPH is oxidized more slowly than NADH.','L-dopa + NADH = 3,4-dihydroxyphenylpropanoate + NAD(+) + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.23','Tyrosine ammonia-lyase','L-tyrosine ammonia-lyase','MIO.','Member of the aromatic amino acid lyase family, other members ofwhich are EC 4.3.1.3, EC 4.3.1.24 and EC 4.3.1.25.Far more active with tyrosine than with phenylalanine as substrate,but the substrate specificity can be switched by mutation of a singleamino acid (H89F) in the enzyme from the bacterium Rhodobactersphaeroides.Formerly EC 4.3.1.5.','L-tyrosine = trans-p-hydroxycinnamate + ammonia.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.24','Phenylalanine ammonia-lyase','NULL','MIO.','Member of the aromatic amino acid lyase family, other members ofwhich are EC 4.3.1.3 and EC 4.3.1.23 and EC 4.3.1.25.The enzyme from some species is highly specific for phenylalanine.Formerly EC 4.3.1.5.','L-phenylalanine = trans-cinnamate + ammonia.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.25','Phenylalanine/tyrosine ammonia-lyase','Bifunctional PAL','MIO.','Member of the aromatic amino acid lyase family, other members ofwhich are EC 4.3.1.3, EC 4.3.1.23 and EC 4.3.1.24.The enzyme from some monocots, including maize, and from the yeastRhodosporidium toruloides, deaminate L-phenylalanine and L-tyrosinewith similar catalytic efficiency.Formerly EC 4.3.1.5.','(1) L-phenylalanine = trans-cinnamate + ammonia.(2) L-tyrosine = trans-p-hydroxycinnamate + ammonia.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.26','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.27','Threo-3-hydroxy-D-aspartate ammonia-lyase','D-threo-3-hydroxyaspartate dehydratase','Pyridoxal 5-phosphate.','The enzyme, purified from the bacterium Delftia sp. HT23, also hasactivity against L-threo-3-hydroxyaspartate, L-erythro-3-hydroxyaspartate, and D-serine.Different from EC 4.3.1.20 and EC 4.3.1.16.Requires a divalent cation such as Mn(2+), Co(2+) or Ni(2+).','Threo-3-hydroxy-D-aspartate = oxaloacetate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.28','L-lysine cyclodeaminase','NULL','NAD(+).','The enzyme produces the non-proteinogenic amino acid L-pipecolate,which is incorporated into multiple secondary metabolite products,including rapamycin, tobulysin, virginiamycin and pristinamycin.','L-lysine = L-pipecolate + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.29','D-glucosaminate-6-phosphate ammonia lyase','NULL','NULL','The enzyme, from the bacterium Salmonella typhimurium, is involved inthe degradation pathway of D-glucosaminate.','6-phospho-D-glucosaminate = 2-dehydro-3-deoxy-6-phospho-D-gluconate +NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.1.30','dTDP-4-amino-4,6-dideoxy-D-glucose ammonia-lyase','NULL','Iron-sulfur.','The enzyme, which is a member of the AdoMet radical (radical SAM)family, is involved in biosynthesis of TDP-alpha-D-desosamine.The reaction starts by the transfer of an electron from the reducedform of the enzymes [4Fe-4S] cluster to S-adenosyl-L-methionine,spliting it into methionine and the radical 5-deoxyadenosin-5-yl,which attacks the sugar substrate.','dTDP-4-amino-4,6-dideoxy-alpha-D-glucopyranose + S-adenosyl-L-methionine+ reduced acceptor = dTDP-3-dehydro-4,6-dideoxy-alpha-D-glucopyranose +NH(3) + L-methionine + 5-deoxyadenosine + acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.2.1','Argininosuccinate lyase','Arginosuccinase','NULL','NULL','2-(N(omega)-L-arginino)succinate = fumarate + L-arginine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.2.2','Adenylosuccinate lyase','Adenylosuccinase','NULL','Also acts on 1-(5-phosphoribosyl)-4-(N-succinocarboxamide)-5-aminoimidazole.','(1) N(6)-(1,2-dicarboxyethyl)AMP = fumarate + AMP.(2) (S)-2-(5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamido)succinate = fumarate + 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.2.3','Ureidoglycolate lyase','Ureidoglycolase','NULL','This microbial enzyme is involved in the degradation ofureidoglycolate, an intermediate of purine degradation.Not to be confused with EC 3.5.1.116 which releases ammonia ratherthan urea.','(S)-ureidoglycolate = glyoxylate + urea.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.2.4','Purine imidazole-ring cyclase','NULL','NULL','Also acts on 2,6-diamino-5-formamido-3,4-dihydro-4-oxopyrimidineresidues.Brings about the reclosure of the imidazole rings of purine residuesdamaged by gamma-rays.','DNA 4,6-diamino-5-formamidopyrimidine = DNA adenine + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.2.5','Peptidylamidoglycolate lyase','Alpha-hydroxyglycine amidating dealkylase','NULL','Acts on the product of the reaction catalyzed by EC 1.14.17.3, thusremoving a terminal glycine residue and leaving a des-glycine peptideamide.','Peptidylamidoglycolate = peptidyl amide + glyoxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.2.6','Gamma-L-glutamyl-butirosin B gamma-glutamyl cyclotransferase','NULL','NULL','The enzyme catalyzes the last step in the biosynthesis of theaminoglycoside antibiotic butirosin B.The enzyme acts as a cyclotransferase, cleaving the amide bond viatransamidation using the alpha-amine of the terminal gamma-L-glutamate of the side chain, releasing it as the cyclic 5-oxoproline.','Gamma-L-glutamyl-butirosin B = butirosin B + 5-oxoproline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.3.1','3-ketovalidoxylamine C-N-lyase','NULL','Calcium.','Eliminates 4-nitroaniline from 4-nitrophenyl-3-ketovalidamine or4-nitrophenol from 4-nitrophenyl-alpha-D-3-dehydroglucoside.Involved in the degradation of the fungicide validamycin A byFlavobacterium saccharophilum.','4-nitrophenyl-3-ketovalidamine = 4-nitroaniline + 5-D-(5/6)-5-C-(hydroxymethyl)-2,6-dihydroxycyclohex-2-en-1-one.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.3.2','Strictosidine synthase','STR','NULL','Catalyzes a a Pictet-Spengler reaction between the aldehyde group ofsecologanin and the amino group of tryptamine.Involved in the biosynthesis of the monoterpenoid indole alkaloids.','3-alpha-(S)-strictosidine + H(2)O = tryptamine + secologanin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.3.3','Deacetylisoipecoside synthase','NULL','NULL','The enzyme from the leaves of Alangium lamarckii differs inenantiomeric specificity from EC 4.3.3.4.The product is rapidly converted to demethylisoalangiside.','Deacetylisoipecoside + H(2)O = dopamine + secologanin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.3.4','Deacetylipecoside synthase','NULL','NULL','The enzyme from the leaves of Alangium lamarckii differs inenantiomeric specificity from EC 4.3.3.3.The product is rapidly converted to demethylalangiside.','Deacetylipecoside + H(2)O = dopamine + secologanin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.3.5','4-demethylrebeccamycin synthase','Arcyriaflavin A N-glycosyltransferase','NULL','This enzyme catalyzes a step in the biosynthesis of rebeccamycin,an indolocarbazole alkaloid produced by the actinobacteriumLechevalieria aerocolonigenes.The enzyme is a glycosylase, and acts in the reverse direction tothat shown.It has a wide substrate range, and was shown to glycosylate severalsubstrates, including the staurosporine aglycone, EJG-III-108A,J-104303, 6-N-methyl-arcyriaflavin C and indolo-[2,3-a]-carbazole.','4-O-demethylrebeccamycin + H(2)O = dichloro-arcyriaflavin A + beta-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.3.6','Pyridoxal 5-phosphate synthase (glutamine hydrolyzing)','NULL','NULL','The ammonia is provided by the glutaminase subunit and channeled tothe active site of the lyase subunit by a 100 A tunnel.The enzyme can also use ribulose 5-phosphate and dihydroxyacetonephosphate.The enzyme complex is found in aerobic bacteria, archeae, fungi andplants.','D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + L-glutamine =pyridoxal 5-phosphate + L-glutamate + 3 H(2)O + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.3.7','4-hydroxy-tetrahydrodipicolinate synthase','DHDPS','NULL','Formerly EC 4.2.1.52.','Pyruvate + L-aspartate-4-semialdehyde = (4S)-4-hydroxy-2,3,4,5-tetrahydro-(2S)-dipicolinate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.3.n1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.99.1','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.99.2','Carboxybiotin decarboxylase','Carboxybiotin protein decarboxylase','NULL','The integral membrane protein MadB from the anaerobic bacteriumMalonomonas rubra is a component of the multienzyme complexEC 4.1.1.89.The free energy of the decarboxylation reaction is used to pump Na(+)out of the cell.','A carboxybiotinyl-[protein] + 1-2 Na(+)(In) + H(+)(Out) = CO(2) +a biotinyl-[protein] + 1-2 Na(+)(Out).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.99.3','7-carboxy-7-deazaguanine synthase','7-carboxy-7-carbaguanine synthase','Iron-sulfur; Magnesium; S-adenosyl-L-methionine.','The enzyme is a member of the superfamily of S-adenosyl-L-methionine-dependent radical (radical AdoMet) enzymes.Binds a [4Fe-4S] cluster that is coordinated by 3 cysteines and anexchangeable S-adenosyl-L-methionine molecule.The S-adenosyl-L-methionine is catalytic as it is regenerated at theend of the reaction.The reaction is part of the biosynthesis pathway of queuosine.','6-carboxy-5,6,7,8-tetrahydropterin = 7-carboxy-7-carbaguanine + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.3.99.4','Choline trimethylamine-lyase','NULL','NULL','The enzyme utilizes a glycine radical to break the C-N bond incholine.Found in choline-degrading anaerobic bacteria.','Choline = trimethylamine + acetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.1','Cystathionine gamma-lyase','Cystathionase','Pyridoxal 5-phosphate.','Also catalyzes elimination reactions of L-homoserine to form H(2)O,NH(3) and 2-oxobutanoate, of L-cystine, producing thiocysteine,pyruvate and NH(3), and of L-cysteine producing pyruvate, NH(3) andH(2)S.Formerly EC 4.2.1.15.','L-cystathionine + H(2)O = L-cysteine + NH(3) + 2-oxobutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.2','Homocysteine desulfhydrase','Homocysteine desulfurase','Pyridoxal 5-phosphate.','Formerly EC 1.2.1.1.','L-homocysteine + H(2)O = H(2)S + NH(3) + 2-oxobutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.3','Dimethylpropiothetin dethiomethylase','NULL','NULL','NULL','S,S-dimethyl-beta-propiothetin = dimethyl sulfide + acrylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.4','Alliin lyase','Alliinase','Pyridoxal 5-phosphate.','NULL','An S-alkyl-L-cysteine S-oxide = an alkyl sulfenate + 2-aminoacrylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.5','Lactoylglutathione lyase','(R)-S-lactoylglutathione methylglyoxal-lyase (isomerizing)','NULL','Also acts on 3-phosphoglycerol-glutathione.','(R)-S-lactoylglutathione = glutathione + methylglyoxal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.6','S-alkylcysteine lyase','S-alkyl-L-cysteinase','Pyridoxal 5-phosphate.','Decomposes S-alkyl-L-cysteines by alpha,beta-elimination.Possibly identical, in Saccharomyces cerevisiae, with EC 4.4.1.8.','An S-alkyl-L-cysteine + H(2)O = an alkyl thiol + NH(3) + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.7','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.8','Cystathionine beta-lyase','Beta C-S lyase','Pyridoxal 5-phosphate.','The enzyme from some sources also acts on L-cystine, formingpyruvate, ammonia and cysteine persulfide, and a number of relatedcompounds.Possibly identical, in Saccharomyces cerevisiae, with EC 4.4.1.6.','L-cystathionine + H(2)O = L-homocysteine + NH(3) + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.9','L-3-cyanoalanine synthase','Beta-cyano-L-alanine synthase','Pyridoxal 5-phosphate.','NULL','L-cysteine + hydrogen cyanide = L-3-cyanoalanine + hydrogen sulfide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.10','Cysteine lyase','NULL','Pyridoxal 5-phosphate.','Can use a second molecule of cysteine (producing lanthionine) orother alkyl thiols as replacing agents.','L-cysteine + sulfite = L-cysteate + H(2)S.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.11','Methionine gamma-lyase','L-methioninase','Pyridoxal 5-phosphate.','NULL','L-methionine + H(2)O = methanethiol + NH(3) + 2-oxobutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.12','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.13','Cysteine-S-conjugate beta-lyase','L-cysteine-S-conjugate thiol-lyase (deaminating)','Pyridoxal 5-phosphate.','R may represent aromatic compounds such as 4-bromobenzene and 2,4-dinitrobenzene.','RS-CH(2)-CH(NH(3)(+))COO(-) + H(2)O = RSH + NH(3) + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.14','1-aminocyclopropane-1-carboxylate synthase','1-aminocyclopropane-1-carboxylic acid synthase','Pyridoxal 5-phosphate.','Catalyzes an alpha,gamma-elimination.','S-adenosyl-L-methionine = 1-aminocyclopropane-1-carboxylate +methylthioadenosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.15','D-cysteine desulfhydrase','NULL','NULL','NULL','D-cysteine + H(2)O = H(2)S + NH(3) + pyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.16','Selenocysteine lyase','Selenocysteine beta-lyase','Pyridoxal 5-phosphate.','Dithiothreitol or 2-mercaptoethanol can act as the reducing agent inthe reaction.The enzyme from animals does not act on cysteine, serine orchloroalanine, while the enzyme from bacteria shows activity withcysteine (cf. EC 2.8.1.7).','L-selenocysteine + reduced acceptor = selenide + L-alanine + acceptor.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.17','Holocytochrome-c synthase','Cytochrome c heme-lyase','NULL','In the reverse direction, the enzyme catalyzes the attachment of hemeto two cysteine residues in the protein, forming thioether links.','Holocytochrome c = apocytochrome c + heme.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.18','Transferred entry: 1','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.19','Phosphosulfolactate synthase','(2R)-phospho-3-sulfolactate synthase','Magnesium.','The enzyme from Methanococcus jannaschii catalyzes the Michaeladdition of sulfite to phosphoenolpyruvate.It specifically requires phosphoenolpyruvate and its broad alkalinepH optimum suggests that it uses sulfite rather than bisulfite.','(2R)-2-O-phospho-3-sulfolactate = phosphoenolpyruvate + hydrogen sulfite.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.20','Leukotriene-C(4) synthase','Leukotriene A(4):glutathione S-leukotrienyltransferase','NULL','The reaction proceeds in the direction of addition.Not identical with EC 2.5.1.18.Formerly EC 2.5.1.37.','Leukotriene C(4) = leukotriene A(4) + glutathione.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.21','S-ribosylhomocysteine lyase','S-ribosylhomocysteinase','Fe(2+).','The 4,5-dihydroxypentan-2,3-dione formed spontaneously cyclizes andcombines with borate to form an autoinducer (AI-2) in the bacterialquorum-sensing mechanism, which is used by many bacteria to controlgene expression in response to cell density.Formerly EC 3.2.1.148 and EC 3.3.1.3.','S-(5-deoxy-D-ribos-5-yl)-L-homocysteine = L-homocysteine + (4S)-4,5-dihydroxypentan-2,3-dione.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.22','S-(hydroxymethyl)glutathione synthase','Glutathione-dependent formaldehyde-activating enzyme','NULL','The enzyme from Paracoccus denitrificans accelerates the spontaneousreaction in which the adduct of formaldehyde and glutathione isformed, i.e. the substrate for EC 1.1.1.284, in the formaldehyde-detoxification pathway.Formerly EC 1.2.1.1.','S-(hydroxymethyl)glutathione = glutathione + formaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.23','2-hydroxypropyl-CoM lyase','Coenzyme M-epoxyalkane ligase','Zinc.','Acts on both enantiomers of chiral epoxyalkanes to form thecorresponding (R)- and (S)-2-hydroxyalkyl-CoM adducts.The enzyme will function with some other thiols (e.g.,2-sulfanylethanol) as the nucleophile.Uses short-chain epoxyalkanes from C(2) (epoxyethane) to C(6) (1,2-epoxyhexane).This enzyme forms component I of a four-component enzyme system(comprising EC 4.4.1.23 (component I), EC 1.8.1.5 (component II),EC 1.1.1.268 (component III) and EC 1.1.1.269 (component IV)) that isinvolved in epoxyalkane carboxylation in Xanthobacter sp. strain Py2.Formerly EC 4.2.99.19.','(1) (R)-2-hydroxypropyl-CoM = (R)-1,2-epoxypropane + HS-CoM.(2) (S)-2-hydroxypropyl-CoM = (S)-1,2-epoxypropane + HS-CoM.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.24','(2R)-sulfolactate sulfo-lyase','Sulfolactate sulfo-lyase','Fe(2+).','This inducible enzyme participates in cysteate degradation inParacoccus pantotrophus NKNCYSA and in 3-sulfolactate degradation inChromohalobacter salexigens.The enzyme is specific for the (R) isomer.','(2R)-3-sulfolactate = pyruvate + bisulfite.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.25','L-cysteate sulfo-lyase','NULL','Pyridoxal 5-phosphate.','D-cysteine can also act as a substrate, but more slowly.It is converted into pyruvate, sulfide and NH(3).The inducible enzyme from Silicibacter pomeroyi DSS-3 forms part ofthe cysteate-degradation pathway.','L-cysteate + H(2)O = pyruvate + bisulfite + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.26','Olivetolic acid cyclase','NULL','NULL','Part of the cannabinoids biosynthetic pathway in the plant Cannabissativa.','3,5,7-trioxododecanoyl-CoA = CoA + 2,4-dihydroxy-6-pentylbenzoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.4.1.27','Carbon disulfide lyase','Carbon disulfide hydrolase','Zn(2+).','The hyperthermophilic archaeon Acidianus sp. A1-3 obtains energy bythe conversion of carbon disulfide to hydrogen sulfide, with carbonylsulfide as an intermediate.','Carbon disulfide + 2 H(2)O = CO(2) + 2 hydrogen sulfide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.5.1.1','DDT-dehydrochlorinase','NULL','NULL','NULL','1,1,1-trichloro-2,2-bis(4-chlorophenyl)ethane = 1,1-dichloro-2,2-bis(4-chlorophenyl)ethylene + chloride.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.5.1.2','3-chloro-D-alanine dehydrochlorinase','NULL','Pyridoxal 5-phosphate.','Also catalyzes beta-replacement reactions, e.g. converts 3-chloro-D-alanine and H(2)S into D-cysteine and HCl.','3-chloro-D-alanine + H(2)O = pyruvate + HCl + NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.5.1.3','Dichloromethane dehalogenase','NULL','Glutathione.','NULL','Dichloromethane + H(2)O = formaldehyde + 2 HCl.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.5.1.4','L-2-amino-4-chloropent-4-enoate dehydrochlorinase','NULL','NULL','NULL','L-2-amino-4-chloropent-4-enoate + H(2)O = 2-oxopent-4-enoate + HCl +NH(3).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.5.1.5','S-carboxymethylcysteine synthase','NULL','Pyridoxal 5-phosphate.','NULL','3-chloro-L-alanine + thioglycolate = S-carboxymethyl-L-cysteine +chloride.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.1','Adenylate cyclase','3,5-cyclic AMP synthetase','NULL','Also acts on dATP to form 3,5-cyclic dAMP.Requires pyruvate.Activated by NAD(+) in presence of EC 2.4.2.31.','ATP = 3,5-cyclic AMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.2','Guanylate cyclase','Guanyl cyclase','NULL','Also acts on ITP and dGTP.','GTP = 3,5-cyclic GMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.3','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.4','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.5','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.6','Cytidylate cyclase','35-cyclic-CMP synthase','NULL','NULL','CTP = 3,5-cyclic CMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.7','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.8','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.9','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.10','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.11','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.12','2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase','MECDP-synthase','Manganese or magnesium.','Forms part of an alternative nonmevalonate pathway for terpenoidbiosynthesis.','2-phospho-4-(cytidine 5-diphospho)-2-C-methyl-D-erythritol = 2-C-methyl-D-erythritol 2,4-cyclodiphosphate + CMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.13','Phosphatidylinositol diacylglycerol-lyase','1-phosphatidylinositol phosphodiesterase','NULL','This enzyme is bacterial.Activity is also found in animals, but this activity is due to thepresence of EC 3.1.4.11.Formerly EC 3.1.4.10.','1-phosphatidyl-1D-myo-inositol = 1D-myo-inositol 1,2-cyclic phosphate +1,2-diacyl-sn-glycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.14','Glycosylphosphatidylinositol diacylglycerol-lyase','(Glycosyl)phosphatidylinositol-specific phospholipase C','NULL','This enzyme is also active when O-4 of the glucosamine is substitutedby carrying the oligosaccharide that can link a protein to thestructure.It therefore cleaves proteins from the lipid part of theglycosylphostphatidylinositol (GPI) anchors.In some cases, the long-chain acyl group at the sn-1 position ofglycerol is replaced by an alkyl or alk-1-enyl group.In other cases, the diacylglycerol is replaced by ceramide.The only characterized enzyme with this specificity is fromTrypanosoma brucei, where the acyl groups are myristoyl, but thefunction of the trypanosome enzyme is unknown.Substitution on O-2 of the inositol blocks action of this enzyme.It is not identical with EC 3.1.4.50.Formerly EC 3.1.4.47.','6-(alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol = 6-(alpha-D-glucosaminyl)-1D-myo-inositol 1,2-cyclic phosphate + 1,2-diacyl-sn-glycerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.15','FAD-AMP lyase (cyclizing)','FMN cyclase','Manganese or cobalt.','While FAD was the best substrate tested the enzyme also splitsribonucleoside diphosphate-X compounds in which X is an acyclic orcyclic monosaccharide or derivative bearing an X-OH group that isable to attack internally the proximal phosphorus with the geometrynecessary to form a P=X product; either a five-atom monocyclicphosphodiester or a cis-bicyclic phosphodiester-pyranose fusion.The reaction is strongly inhibited by ADP or ATP but is unaffected bythe presence of the product, cFMN.','FAD = AMP + riboflavin cyclic-4,5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.6.1.16','tRNA-intron lyase','PretRNA lyase (intron-removing; cyclic-2,3-phosphate-forming)','NULL','The enzyme catalyzes the final stage in the maturation of tRNAmolecules.Formerly EC 3.1.27.9.','PretRNA = a 3-half-tRNA molecule with a 5-OH end + a 5-half-tRNAmolecule with a 2,3-cyclic phosphate end + an intron with a2,3-cyclic phosphate and a 5-hydroxyl terminus.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.7.1.1','Alpha-D-ribose 1-methylphosphonate 5-phosphate C-P-lyase','NULL','Iron-sulfur; S-adenosyl-L-methionine.','Isolated from Escherichia coli.Involves a [4Fe-4S] cluster and a S-adenosyl-L-methionine (SAM)radical.','Alpha-D-ribose 1-methylphosphonate 5-phosphate = alpha-D-ribose 1,2-cyclic phosphate 5-phosphate + methane.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.99.1.1','Ferrochelatase','Heme synthase','NULL','NULL','Protoheme + 2 H(+) = protoporphyrin + Fe(2+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.99.1.2','Alkylmercury lyase','Organomercurial lyase','NULL','Acts on CH(3)Hg(+) and a number of other alkylmercury compounds,in the presence of cysteine or other thiols, liberating mercury as amercaptide.Some arylmercury compounds can also act as substrates.','An alkylmercury + H(+) = an alkane + Hg(2+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.99.1.3','Sirohydrochlorin cobaltochelatase','Anaerobic cobalt chelatase','NULL','This enzyme is a type II chelatase, being either a monomer (CbiX) ora homodimer (CibK) and being ATP-independent.CbiK from Salmonella enterica uses precorrin-2 as the substrate toyield cobalt-precorrin-2.The enzyme contains two histidines at the active site that arethought to be involved in the deprotonation of the tetrapyrrolesubstrate as well as in metal binding.CbiX from Bacillus megaterium inserts cobalt at the level ofsirohydrochlorin (factor-II) rather than precorrin-2.','Cobalt-sirohydrochlorin + 2 H(+) = sirohydrochlorin + Co(2+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.99.1.4','Sirohydrochlorin ferrochelatase','NULL','NULL','This enzyme catalyzes the third of three steps leading to theformation of siroheme from uroporphyrinogen III.The first step involves the donation of two S-adenosyl-L-methionine-derived methyl groups to carbons 2 and 7 of uroporphyrinogen III toform precorrin-2 (EC 2.1.1.107) and the second step involves anNAD(+)-dependent dehydrogenation to form sirohydrochlorin fromprecorrin-2 (EC 1.3.1.76).In Saccharomyces cerevisiae, the last two steps are carried out by asingle bifunctional enzyme, Met8p.In some bacteria, steps 1-3 are catalyzed by a single multifunctionalprotein called CysG, whereas in Bacillus megaterium, three separateenzymes carry out each of the steps, with SirB being responsible forthe above reaction.','Siroheme + 2 H(+) = sirohydrochlorin + Fe(2+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.99.1.5','Aliphatic aldoxime dehydratase','NULL','Calcium; Protoheme IX.','Exhibits a strong preference for aliphatic aldoximes, such asbutyraldoxime and acetaldoxime, over aromatic aldoximes, such aspyridine-2-aldoxime, which is a poor substrate.No activity was found with the aromatic aldoximes benzaldoxime andpyridine-4-aldoxime.','An aliphatic aldoxime = an aliphatic nitrile + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.99.1.6','Indoleacetaldoxime dehydratase','3-indoleacetaldoxime hydro-lyase','NULL','Formerly EC 4.2.1.29.','(Indol-3-yl)acetaldehyde oxime = (indol-3-yl)acetonitrile + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.99.1.7','Phenylacetaldoxime dehydratase','Arylacetaldoxime dehydratase','Heme.','The enzyme from Bacillus sp. OxB-1 contains protoheme IX, the iron ofwhich must be in the form Fe(2+) for activity.(Z)-phenylacetaldoxime binds to ferric heme (the Fe(3+) form) via theoxygen atom whereas it binds to the active ferrous form via thenitrogen atom.In this way, the oxidation state of the heme controls thecoordination stucture of the substrate--heme complex, which regulatesenzyme activity.The enzyme is active toward several (Z)-arylacetaldoximes and (E/Z)-alkylaldoximes as well as toward arylalkylaldoximes such as3-phenylpropionaldoxime and 4-phenylbutyraldoxime.However, it is inactive with phenylacetaldoximes that have asubstituent group at an alpha-site of an oxime group, for example,with (E/Z)-2-phenylpropionaldoxime and (E/Z)-mandelaldoxime.The activity of the enzyme is inhibited completely by the heavy-metalcations Cu(+), Cu(2+), Ag(+) and Hg(+) whereas Fe(2+) and Sn(2+) havean activatory effect.','(Z)-phenylacetaldehyde oxime = phenylacetonitrile + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 4.99.1.8','Heme ligase','Hemozoin synthase','NULL','This heme detoxifying enzyme is found in Plasmodium parasites andconverts toxic heme to crystalline hemozoin.These organisms lack the mammalian heme oxygenase for elimination ofheme.','2 ferriprotoporphyrin IX = beta-hematin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.1','Alanine racemase','NULL','Pyridoxal 5-phosphate.','NULL','L-alanine = D-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.2','Methionine racemase','NULL','Pyridoxal 5-phosphate.','NULL','L-methionine = D-methionine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.3','Glutamate racemase','NULL','Pyridoxal 5-phosphate.','NULL','L-glutamate = D-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.4','Proline racemase','NULL','NULL','NULL','L-proline = D-proline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.5','Lysine racemase','NULL','NULL','NULL','L-lysine = D-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.6','Threonine racemase','NULL','NULL','Inverts both chiral centers.','L-threonine = D-threonine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.7','Diaminopimelate epimerase','NULL','NULL','NULL','LL-2,6-diaminoheptanedioate = meso-diaminoheptanedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.8','4-hydroxyproline epimerase','Hydroxyproline 2-epimerase','NULL','Also interconverts trans-4-hydroxy-D-proline and cis-4-hydroxy-L-proline.','Trans-4-hydroxy-L-proline = cis-4-hydroxy-D-proline.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.9','Arginine racemase','NULL','Pyridoxal 5-phosphate.','NULL','L-arginine = D-arginine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.10','Amino-acid racemase','NULL','Pyridoxal 5-phosphate.','NULL','An L-amino acid = a D-amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.11','Phenylalanine racemase (ATP-hydrolyzing)','NULL','NULL','NULL','ATP + L-phenylalanine + H(2)O = AMP + diphosphate + D-phenylalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.12','Ornithine racemase','NULL','NULL','NULL','L-ornithine = D-ornithine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.13','Aspartate racemase','NULL','NULL','Also acts, at half the rate, on L-alanine.','L-aspartate = D-aspartate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.14','Nocardicin-A epimerase','NULL','NULL','The (9S) configuration of isonocardicin A is converted into the(9R) configuration.','Isonocardicin A = nocardicin A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.15','2-aminohexano-6-lactam racemase','2-amino-hexano-6-lactam racemase','Pyridoxal 5-phosphate.','Also interconverts 2-aminopentano-5-lactam (alpha-amino-delta-valerolactam) and 2-amino-4-thiahexano-6-lactam (where S replacesCH(2) of C-4).It does not catalyze the racemisation of alpha-amino acids but hassome transaminase activity with them.','(S)-2-aminohexano-6-lactam = (R)-2-aminohexano-6-lactam.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.16','Protein-serine epimerase','Protein-serine racemase','NULL','The enzyme specifically interconverts the configuration of Ser-46 ofthe peptide omega-agatoxin-KT, found in the venom of the funnel webspider, Agelenopsis aperta, but not that of the other serine residue,Ser-28.','[Protein]-L-serine = [protein]-D-serine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.17','Isopenicillin-N epimerase','NULL','Pyridoxal 5-phosphate.','Epimerization at C-5 of the 5-amino-5-carboxypentanoyl group to formpenicillin N is required to make a substrate for EC 1.14.20.1,to produce cephalosporins.Forms part of the penicillin biosynthesis pathway.','Isopenicillin N = penicillin N.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.18','Serine racemase','NULL','Pyridoxal 5-phosphate.','Highly selective for L-serine as substrate.D-serine is found in type-II astrocytes in mammalian brain, where itappears to be an endogenous ligand of the glycine site of N-methyl-D-aspartate (NMDA) receptors.The reaction can also occur in the reverse direction but does so moreslowly at physiological serine concentrations.','L-serine = D-serine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.19','O-ureido-serine racemase','NULL','NULL','The enzyme employs a two-base mechanism, with a thiolate-thiol pairin the active site.It participates in the biosynthetic pathway of D-cycloserine,an antibiotic substance produced by several Streptomyces species.','O-ureido-L-serine = O-ureido-D-serine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.1.n1','L-Ala-D/L-Glu epimerase','NULL','NULL','NULL','L-alanyl-D-glutamate = L-alanyl-L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.2.1','Lactate racemase','Hydroxyacid racemase','NULL','NULL','(S)-lactate = (R)-lactate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.2.2','Mandelate racemase','NULL','NULL','NULL','(S)-mandelate = (R)-mandelate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.2.3','3-hydroxybutyryl-CoA epimerase','NULL','NULL','NULL','(S)-3-hydroxybutanoyl-CoA = (R)-3-hydroxybutanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.2.4','Acetoin racemase','Acetylmethylcarbinol racemase','NULL','NULL','(S)-acetoin = (R)-acetoin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.2.5','Tartrate epimerase','NULL','NULL','NULL','(R,R)-tartrate = meso-tartrate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.2.6','Isocitrate epimerase','NULL','NULL','(1R,2S)-1-hydroxypropane-1,2,3-tricarboxylate is the commonlyoccurring isomer of isocitrate.','(1R,2S)-1-hydroxypropane-1,2,3-tricarboxylate = (1S,2S)-1-hydroxypropane-1,2,3-tricarboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.1','Ribulose-phosphate 3-epimerase','D-ribulose phosphate-3-epimerase','NULL','Also converts D-erythrose 4-phosphate into D-erythrulose 4-phosphateand D-threose 4-phosphate.','D-ribulose 5-phosphate = D-xylulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.2','UDP-glucose 4-epimerase','Galactowaldenase','NAD(+).','Also acts on UDP-2-deoxyglucose.','UDP-alpha-D-glucose = UDP-alpha-D-galactose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.3','Aldose 1-epimerase','Aldose mutarotase','NULL','Also acts on L-arabinose, D-xylose, D-galactose, maltose and lactose.This enzyme catalyzes the first step in galactose metabolism byconverting beta-D-galactose into alpha-D-galactose, which is thesubstrate for EC 2.7.1.6.','Alpha-D-glucose = beta-D-glucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.4','L-ribulose-5-phosphate 4-epimerase','L-ribulose 5-phosphate 4-epimerase','Divalent cation.','NULL','L-ribulose 5-phosphate = D-xylulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.5','UDP-arabinose 4-epimerase','NULL','NULL','NULL','UDP-L-arabinose = UDP-D-xylose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.6','UDP-glucuronate 4-epimerase','UDP glucuronic epimerase','NULL','NULL','UDP-glucuronate = UDP-D-galacturonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.7','UDP-N-acetylglucosamine 4-epimerase','UDP-GlcNAc 4-epimerase','NULL','NULL','UDP-N-acetyl-alpha-D-glucosamine = UDP-N-acetyl-alpha-D-galactosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.8','N-acylglucosamine 2-epimerase','GlcNAc 2-epimerase','NULL','Requires catalytic amounts of ATP.','N-acyl-D-glucosamine = N-acyl-D-mannosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.9','N-acylglucosamine-6-phosphate 2-epimerase','Acylglucosamine phosphate 2-epimerase','NULL','NULL','N-acyl-D-glucosamine 6-phosphate = N-acyl-D-mannosamine 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.10','CDP-paratose 2-epimerase','CDP-abequose epimerase','NAD(+).','CDP-paratose (CDP-3,6-dideoxy-D-glucose), is more systematicallycalled CDP-alpha-3,6-dideoxy-D-ribo-hexose.CDP-tyvelose (CDP-3,6-dideoxy-D-mannose) is systematically calledCDP-3,6-dideoxy-D-arabino-hexose.','CDP-3,6-dideoxy-D-glucose = CDP-3,6-dideoxy-D-mannose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.11','Cellobiose epimerase','NULL','NULL','NULL','Cellobiose = D-glucosyl-D-mannose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.12','UDP-glucuronate 5-epimerase','UDP-glucuronic acid 5-epimerase','NAD(+).','NULL','UDP-glucuronate = UDP-L-iduronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.13','dTDP-4-dehydrorhamnose 3,5-epimerase','dTDP-4-keto-6-deoxyglucose 3,5-epimerase','NULL','Occurs in a complex with EC 1.1.1.133.','dTDP-4-dehydro-6-deoxy-alpha-D-glucose = dTDP-4-dehydro-beta-L-rhamnose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.14','UDP-N-acetylglucosamine 2-epimerase (non-hydrolyzing)','UDP-GlcNAc-2-epimerase','NULL','This bacterial enzyme catalyzes the reversible interconversion ofUDP-GlcNAc and UDP-ManNAc.The latter is used in a variety of bacterial polysaccharidebiosyntheses.Cf. EC 3.2.1.183.','UDP-N-acetyl-alpha-D-glucosamine = UDP-N-acetyl-alpha-D-mannosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.15','Glucose-6-phosphate 1-epimerase','NULL','NULL','NULL','Alpha-D-glucose 6-phosphate = beta-D-glucose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.16','UDP-glucosamine 4-epimerase','NULL','NULL','NULL','UDP-glucosamine = UDP-galactosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.17','Heparosan-N-sulfate-glucuronate 5-epimerase','NULL','NULL','Acts on D-glucuronosyl residues adjacent to sulfated D-glucosamineunits in the heparin precursor.Not identical with EC 5.1.3.19.','Heparosan-N-sulfate D-glucuronate = heparosan-N-sulfate L-iduronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.18','GDP-mannose 3,5-epimerase','NULL','NULL','NULL','GDP-alpha-D-mannose = GDP-beta-L-galactose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.19','Chondroitin-glucuronate 5-epimerase','Dermatan-sulfate 5-epimerase','NULL','Not identical with EC 5.1.3.17.','Chondroitin D-glucuronate = dermatan L-iduronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.20','ADP-glyceromanno-heptose 6-epimerase','ADP-L-glycero-D-manno-heptose 6-epimerase','NAD(+).','NULL','ADP-D-glycero-D-manno-heptose = ADP-L-glycero-D-manno-heptose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.21','Maltose epimerase','NULL','NULL','The enzyme catalyzes the interconversion of alpha and beta anomers ofmaltose more effectively than those of disaccharides such as lactoseand cellobiose.','Alpha-maltose = beta-maltose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.22','L-ribulose-5-phosphate 3-epimerase','L-xylulose 5-phosphate 3-epimerase','NULL','Along with EC 4.1.1.85, this enzyme is involved in a pathway for theutilization of L-ascorbate by Escherichia coli.','L-ribulose 5-phosphate = L-xylulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.23','UDP-2,3-diacetamido-2,3-dideoxyglucuronic acid 2-epimerase','2,3-diacetamido-2,3-dideoxy-alpha-D-glucuronic acid 2-epimerase','NULL','Participates in the biosynthetic pathway for UDP-alpha-D-ManNAc3NAcA(UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-mannuronic acid),an important precursor of the B-band lipopolysaccharide ofPseudomonas aeroginosa serotype O5 and of the band-A trisaccharide ofBordetella pertussis, both important respiratory pathogens.Highly specific as UDP-alpha-D-GlcNAc, UDP-alpha-D-GlcNAcA (UDP-2-acetamido-2-deoxy-alpha-D-glucuronic acid) and UDP-alpha-D-GlcNAc3NAc(UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-glucose) cannot act assubstrates.','UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-glucuronate = UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-mannuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.24','N-acetylneuraminate epimerase','Alpha-N-acetylneuraminic acid epimerase','NULL','Sialoglycoconjugates present in vertebrates are linked exclusively byalpha-linkages and are released in alpha form during degradation.This enzyme accelerates maturotation to the beta form (which alsooccurs as a slow spontaneous reaction), which is necessary forfurther metabolism by the bacteria.Formerly EC 5.1.3.n1.','N-acetyl-alpha-neuraminate = N-acetyl-beta-neuraminate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.25','dTDP-L-rhamnose 4-epimerase','dTDP-4-L-rhamnose 4-epimerase','NULL','The equilibrium is strongly toward dTDP-beta-L-rhamnose.','dTDP-6-deoxy-beta-L-talose = dTDP-beta-L-rhamnose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.26','N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol4-epimerase','GlcNAc-P-P-Und 4-epimerase','NULL','The enzyme is involved in biosynthesis of the repeatingtetrasaccharide unit of the O-antigen produced by some Gram-negativebacteria.','N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol =N-acetyl-alpha-D-galactosaminyl-diphospho-ditrans,octacis-undecaprenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.27','dTDP-4-dehydro-6-deoxy-D-glucose 3-epimerase','dTDP-4-keto-6-deoxy-D-glucose 3-epimerase','NULL','The enzyme is involved in the biosynthetic pathway of dTDP-6-deoxy-alpha-D-allose, which is converted to mycinose after attachment ogthe aglycones of several macrolide antibiotics, including tylosin,chalcomycin, dihydrochalcomycin, and mycinamicin II.','dTDP-4-dehydro-6-deoxy-alpha-D-glucose = dTDP-4-dehydro-6-deoxy-alpha-D-allose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.28','UDP-N-acetyl-L-fucosamine synthase','NULL','NULL','Isolated from the bacteria Pseudomonas aeruginosa and Staphylococcusaureus.Involved in bacterial polysaccharide biosynthesis.','UDP-2-acetamido-2,6-dideoxy-beta-L-talose = UDP-N-acetyl-beta-L-fucosamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.n1','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.n2','L-fucose mutarotase','Fucose 1-epimerase','NULL','NULL','Alpha-L-fucose = beta-L-fucose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.3.n3','L-rhamnose mutarotase','Rhamnose 1-epimerase','NULL','NULL','Alpha-L-rhamnose = beta-L-rhamnose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.99.1','Methylmalonyl-CoA epimerase','2-methyl-3-oxopropanoyl-CoA 2-epimerase','NULL','NULL','(R)-methylmalonyl-CoA = (S)-methylmalonyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.99.2','16-hydroxysteroid epimerase','NULL','NULL','NULL','16-alpha-hydroxysteroid = 16-beta-hydroxysteroid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.99.3','Allantoin racemase','NULL','NULL','NULL','(S)(+)-allantoin = (R)(-)-allantoin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.99.4','Alpha-methylacyl-CoA racemase','NULL','NULL','Alpha-methyl-branched acyl-CoA derivatives with chain lengths of morethan C(10) are substrates.Also active toward some aromatic compounds (e.g. ibuprofen) and bileacid intermediates, such as trihydroxycoprostanoyl-CoA.Not active toward free acids.','(2S)-2-methylacyl-CoA = (2R)-2-methylacyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.99.5','Hydantoin racemase','5-monosubstituted-hydantoin racemase','NULL','This enzyme, along with EC 3.5.1.77, EC 3.5.1.87 and EC 3.5.2.2,forms part of the reaction cascade known as the hydantoinaseprocess, which allows the total conversion of D,L-5-monosubstitutedhydantoins into optically pure D- or L-amino acids.The enzyme from Pseudomonas sp. (HyuE) has a preference forhydantoins with aliphatic substituents, e.g. D- and L-5-(2-methylthioethyl)hydantoin, whereas that from Arthrobacteraurescens shows highest activity with arylalkyl substituents,especially 5-benzylhydantoin, at the 5-position.In the enzyme from Sinorhizobium meliloti, Cys-76 is responsible forrecognition and proton retrieval of D-isomers, while Cys-181 isresponsible for L-isomer recognition and racemization.','D-5-monosubstituted hydantoin = L-5-monosubstituted hydantoin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.1.99.6','NAD(P)H-hydrate epimerase','NAD(P)HX epimerase','NULL','The enzyme can use either (R)-NADH-hydrate or (R)-NADPH-hydrate as asubstrate.Its physiological role is to convert the (R) forms to the (S) forms,which could then be restored to active dinucleotides by EC 4.2.1.93.','(1) (6R)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adeninedinucleotide = (6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adeninedinucleotide.(2) (6R)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adeninedinucleotide phosphate = (6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adenine dinucleotide phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.1','Maleate isomerase','NULL','NULL','NULL','Maleate = fumarate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.2','Maleylacetoacetate isomerase','Maleylacetone isomerase','NULL','Also acts on maleylpyruvate.','4-maleylacetoacetate = 4-fumarylacetoacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.3','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.4','Maleylpyruvate isomerase','NULL','NULL','NULL','3-maleylpyruvate = 3-fumarylpyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.5','Linoleate isomerase','NULL','NULL','NULL','9-cis,12-cis-octadecadienoate = 9-cis,11-trans-octadecadienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.6','Furylfuramide isomerase','NULL','NAD(+).','NULL','(E)-2-(2-furyl)-3-(5-nitro-2-furyl)acrylamide = (Z)-2-(2-furyl)-3-(5-nitro-2-furyl)acrylamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.7','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.8','Peptidylprolyl isomerase','Cyclophilin','NULL','The first type of this enzyme found proved to be the proteincyclophilin, which binds the immunosuppressant cyclosporin A.Other distinct families of the enzyme exist, one being FK-506 bindingproteins (FKBP) and another that includes parvulin from Escherichiacoli.The three families are structurally unrelated and can bedistinguished by being inhibited by cyclosporin A, FK-506 and5-hydroxy-1,4-naphthoquinone, respectively.','Peptidylproline (omega=180) = peptidylproline (omega=0).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.9','Farnesol 2-isomerase','NULL','NULL','NULL','(2E,6E)-farnesol = (2Z,6E)-farnesol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.10','2-chloro-4-carboxymethylenebut-2-en-1,4-olide isomerase','2-chlorocarboxymethylenebutenolide isomerase','NULL','NULL','Cis-2-chloro-4-carboxymethylenebut-2-en-1,4-olide = trans-2-chloro-4-carboxymethylenebut-2-en-1,4-olide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.11','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.12','Zeta-carotene isomerase','15-cis-zeta-carotene isomerase','NULL','Involved in carotenoid biosynthesis.','9,15,9-tri-cis-zeta-carotene = 9,9-di-cis-zeta-carotene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.13','Prolycopene isomerase','Carotene cis-trans isomerase','FADH(2).','The enzyme is involved in carotenoid biosynthesis.','7,9,7,9-tetra-cis-lycopene = all-trans-lycopene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.2.1.14','Beta-carotene isomerase','NULL','NULL','The enzyme participates in a pathway leading to biosynthesis ofstrigolactones, plant hormones involved in promotion of symbioticassociations known as arbuscular mycorrhiza.','All-trans-beta-carotene = 9-cis-beta-carotene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.1','Triose-phosphate isomerase','Phosphotriose isomerase','NULL','NULL','D-glyceraldehyde 3-phosphate = glycerone phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.2','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.3','D-arabinose isomerase','Arabinose isomerase','Manganese.','The enzyme binds the closed form of the sugar and catalyzes ringopening to generate a form of open-chain conformation thatfacilitates the isomerization reaction, which proceeds via an ene-diol mechanism.The enzyme catalyzes the aldose-ketose isomerization of severalsugars.Most enzymes also catalyze the reaction of EC 5.3.1.25.The enzyme from the bacterium Falsibacillus pallidus also convertsD-altrose to D-psicose.Cf. EC 5.3.1.4.','D-arabinose = D-ribulose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.4','L-arabinose isomerase','NULL','Divalent cation.','The enzyme binds the closed form of the sugar and catalyzes ringopening to generate a form of open-chain conformation thatfacilitates the isomerization reaction, which proceeds via an ene-diol mechanism.The enzyme can also convert D-galactose to D-tagatose with lowerefficiency.','L-arabinose = L-ribulose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.5','Xylose isomerase','D-xylose ketoisomerase','Magnesium.','The enzyme catalyzes the interconversion of aldose and ketose sugarswith broad substrate specificity.The enzyme binds the closed form of its sugar substrate (in the caseof glucose, only the alpha anomer) and catalyzes ring opening togenerate a form of open-chain conformation that is coordinated to oneof the metal sites.Isomerization proceeds via a hydride-shift mechanism.','D-xylopyranose = D-xylulose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.6','Ribose-5-phosphate isomerase','5-phosphoribose isomerase','NULL','Also acts on D-ribose 5-diphosphate and D-ribose 5-triphosphate.','D-ribose 5-phosphate = D-ribulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.7','Mannose isomerase','D-mannose isomerase','NULL','Also acts on D-lyxose and D-rhamnose.','D-mannose = D-fructose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.8','Mannose-6-phosphate isomerase','Phosphohexoisomerase','Zinc.','NULL','D-mannose 6-phosphate = D-fructose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.9','Glucose-6-phosphate isomerase','Hexose monophosphate isomerase','NULL','Also catalyzes the anomerization of D-glucose 6-phosphate.','D-glucose 6-phosphate = D-fructose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.10','Transferred entry: 3','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.11','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.12','Glucuronate isomerase','D-glucuronate isomerase','NULL','Also converts D-galacturonate to D-tagaturonate.','D-glucuronate = D-fructuronate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.13','Arabinose-5-phosphate isomerase','Arabinose phosphate isomerase','NULL','NULL','D-arabinose 5-phosphate = D-ribulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.14','L-rhamnose isomerase','Rhamnose isomerase','Divalent cation.','The enzyme binds the closed ring form of the substrate and catalyzesring opening to generate a form of open-chain conformation that iscoordinated to one of the metal sites.Isomerization proceeds via a hydride-shift mechanism.While the enzyme from the bacterium Escherichia coli is specific forL-rhamnose, the enzyme from the bacterium Pseudomonas stutzeri hasbroad substrate specificity and catalyzes the interconversion ofL-mannose and L-fructose, L-lyxose and L-xylulose, D-ribose andD-ribulose, and D-allose and D-psicose.','L-rhamnopyranose = L-rhamnulose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.15','D-lyxose ketol-isomerase','D-lyxose isomerase','NULL','NULL','D-lyxose = D-xylulose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.16','1-(5-phosphoribosyl)-5-((5-phosphoribosylamino)methylideneamino)imidazole-4-carboxamideisomerase','N-(5-phospho-D-ribosylformimino)-5-amino-1-(5-phosphoribosyl)-4-imidazolecarboxamide isomerase','NULL','Involved in histidine biosynthesis.','1-(5-phospho-beta-D-ribosyl)-5-((5-phospho-beta-D-ribosylamino)methylideneamino)imidazole-4-carboxamide = 5-((5-phospho-1-deoxy-D-ribulos-1-ylamino)methylideneamino)-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.17','5-dehydro-4-deoxy-D-glucuronate isomerase','4-deoxy-L-threo-5-hexosulose-uronate ketol-isomerase','NULL','The enzyme is involved in the degradation of polygalacturonate,a later stage in the degradation of pectin by many microorganisms.','5-dehydro-4-deoxy-D-glucuronate = 3-deoxy-D-glycero-2,5-hexodiulosonate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.18','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.19','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.20','Ribose isomerase','D-ribose isomerase','NULL','Also acts on L-lyxose and L-rhamnose.','D-ribose = D-ribulose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.21','Corticosteroid side-chain-isomerase','NULL','NULL','An epimerization at C-20 and C-21 is probably catalyzed by the sameenzyme.','11-deoxycorticosterone = 20-hydroxy-3-oxopregn-4-en-21-al.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.22','Hydroxypyruvate isomerase','NULL','NULL','NULL','Hydroxypyruvate = 2-hydroxy-3-oxopropanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.23','S-methyl-5-thioribose-1-phosphate isomerase','1-phospho-5-S-methylthioribose isomerase','NULL','NULL','S-methyl-5-thio-alpha-D-ribose 1-phosphate = S-methyl-5-thio-D-ribulose1-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.24','Phosphoribosylanthranilate isomerase','N-(5-phosphoribosyl)anthranilate isomerase','NULL','In some organisms, this enzyme is part of a multifunctional proteintogether with one or more components of the system for biosynthesisof tryptophan (EC 2.4.2.18, EC 4.1.1.48, EC 4.1.3.27,and EC 4.2.1.20).','N-(5-phospho-beta-D-ribosyl)anthranilate = 1-(2-carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.25','L-fucose isomerase','NULL','Manganese.','The enzyme binds the closed form of the sugar and catalyzes ringopening to generate a form of open-chain conformation thatfacilitates the isomerization reaction, which proceeds via an ene-diol mechanism.The enzyme from Escherichia coli can also convert D-arabinose toD-ribulose.The enzyme from the thermophilic bacterium Caldicellulosiruptorsaccharolyticus also converts D-altrose to D-psicose and L-galactoseto L-tagatose.','L-fucopyranose = L-fuculose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.26','Galactose-6-phosphate isomerase','NULL','NULL','Involved in the tagatose 6-phosphate pathway of lactose catabolism inbacteria.','D-galactose 6-phosphate = D-tagatose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.27','6-phospho-3-hexuloisomerase','3-hexulose-6-phosphate isomerase','NULL','This enzyme, along with EC 4.1.2.43, plays a key role in theribulose-monophosphate cycle of formaldehyde fixation, which ispresent in many microorganisms that are capable of utilizingC1-compounds.The hyperthermophilic and anaerobic archaeon Pyrococcus horikoshiiOT3 constitutively produces a bifunctional enzyme that sequentiallycatalyzes the reactions of EC 4.1.2.43 and this enzyme.','D-arabino-hex-3-ulose 6-phosphate = D-fructose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.28','D-sedoheptulose 7-phosphate isomerase','Phosphoheptose isomerase','NULL','In Gram-negative bacteria the enzyme is involved in biosynthesis ofADP-L-glycero-beta-D-manno-heptose, which is utilized for assembly ofthe lipopolysaccharide inner core.In Gram-positive bacteria the enzyme is involved in biosynthesis ofGDP-D-glycero-alpha-D-manno-heptose, which is required for assemblyof S-layer glycoprotein.','D-sedoheptulose 7-phosphate = D-glycero-D-manno-heptose 7-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.29','Ribose 1,5-bisphosphate isomerase','Ribulose 1,5-bisphosphate synthase','NULL','This archaeal enzyme is involved in AMP metabolism and CO(2) fixationthrough type III RubisCO enzymes.The enzyme is activated by cAMP.Formerly EC 5.3.1.n2.','Alpha-D-ribose 1,5-bisphosphate = D-ribulose 1,5-bisphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.30','5-deoxy-glucuronate isomerase','5DG isomerase','NULL','The enzyme, found in the bacterium Bacillus subtilis, is part of amyo-inositol degradation pathway leading to acetyl-CoA.Formerly EC 5.3.1.n1.','5-deoxy-D-glucuronate = 5-dehydro-2-deoxy-D-gluconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.n1','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.1.n2','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.2.1','Phenylpyruvate tautomerase','Phenylpyruvate keto--enol tautomerase','NULL','Also acts on other arylpyruvates.','Keto-phenylpyruvate = enol-phenylpyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.2.2','Oxaloacetate tautomerase','Oxalacetic keto--enol isomerase','NULL','NULL','Keto-oxaloacetate = enol-oxaloacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.2.3','TDP-4-oxo-6-deoxy-alpha-D-glucose-3,4-oxoisomerase (dTDP-3-dehydro-6-deoxy-alpha-D-galactopyranose-forming)','dTDP-6-deoxy-hex-4-ulose isomerase','NULL','The enzyme is involved in the biosynthesis of dTDP-3-acetamido-3,6-dideoxy-alpha-D-galactose.Four moieties of alpha-D-rhamnose and two moities of 3-acetamido-3,6-dideoxy-alpha-D-galactose form the repeating unit of the glycan chainin the S-layer of the bacterium Aneurinibacillus thermoaerophilus.','dTDP-4-dehydro-6-deoxy-alpha-D-glucopyranose = dTDP-3-dehydro-6-deoxy-alpha-D-galactopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.2.4','TDP-4-oxo-6-deoxy-alpha-D-glucose-3,4-oxoisomerase (dTDP-3-dehydro-6-deoxy-alpha-D-glucopyranose-forming)','dTDP-4-keto-6-deoxy-D-glucose-3,4-ketoisomerase','NULL','The enzyme is involved in biosynthesis of D-mycaminose.','dTDP-4-dehydro-6-deoxy-alpha-D-glucopyranose = dTDP-3-dehydro-6-deoxy-alpha-D-glucopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.2.5','2,3-diketo-5-methylthiopentyl-1-phosphate enolase','DK-MTP-1-P enolase','NULL','The enzyme participates in the methionine salvage pathway in Bacillussubtilis.In some species a single bifunctional enzyme, EC 3.1.3.77, catalyzesboth this reaction and that of EC 3.1.3.87.Formerly EC 5.3.2.n1.','5-(methylthio)-2,3-dioxopentyl phosphate = 2-hydroxy-5-(methylthio)-3-oxopent-1-enyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.2.6','2-hydroxymuconate tautomerase','4-oxalocrotonate isomerase','NULL','Involved in the meta-cleavage pathway for the degradation of phenols,modified phenols and catechols.The enol form (2Z,4E)-2-hydroxyhexa-2,4-dienedioate is produced aspart of this pathway and is converted to the keto form (3E)-2-oxohex-3-enedioate by the enzyme.Another keto form, (4E)-2-oxohex-4-enedioate (4-oxalocrotonate),was originally thought to be produced by the enzyme but later shownto be produced non-enzymatically.','(2Z,4E)-2-hydroxyhexa-2,4-dienedioate = (3E)-2-oxohex-3-enedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.2.7','Ascopyrone tautomerase','1,5-anhydro-D-glycero-hex-3-en-2-ulose tautomerase','NULL','Catalyzes one of the steps in the anhydrofructose pathway, whichleads to the degradation of glycogen and starch via 1,5-anhydro-D-fructose.The other enzymes involved in this pathway are EC 4.2.1.110,EC 4.2.1.111 and EC 4.2.2.13.Ascopyrone P is an anti-oxidant.Formerly EC 5.3.3.15.','1,5-anhydro-4-deoxy-D-glycero-hex-3-en-2-ulose = 1,5-anhydro-4-deoxy-D-glycero-hex-1-en-3-ulose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.2.8','4-oxalomesaconate tautomerase','NULL','NULL','This enzyme has been characterized from the bacterium Pseudomonasputida KT2440 and is involved in the degradation pathway of syringateand gallate.It catalyzes the interconversion of two of the tautomers of4-oxalomesaconate, a reaction that can also occur spontaneously.Formerly EC 5.3.3.16.','(1E)-4-oxobut-1-ene-1,2,4-tricarboxylate = (1E,3E)-4-hydroxybuta-1,3-diene-1,2,4-tricarboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.2.n1','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.1','Steroid Delta-isomerase','3-oxosteroid isomerase','NULL','NULL','A 3-oxo-Delta(5)-steroid = a 3-oxo-Delta(4)-steroid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.2','Isopentenyl-diphosphate Delta-isomerase','IPP isomerase','Calcium or magnesium or manganese; FMN or FAD.','NULL','Isopentenyl diphosphate = dimethylallyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.3','Vinylacetyl-CoA Delta-isomerase','Vinylacetyl coenzyme A isomerase','NULL','Also acts on 3-methyl-vinylacetyl-CoA.','Vinylacetyl-CoA = (E)-but-2-enoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.4','Muconolactone Delta-isomerase','Muconolactone isomerase','NULL','Formerly EC 3.1.1.16.','(S)-5-oxo-2,5-dihydrofuran-2-acetate = 5-oxo-4,5-dihydrofuran-2-acetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.5','Cholestenol Delta-isomerase','NULL','NULL','NULL','5-alpha-cholest-7-en-3-beta-ol = 5-alpha-cholest-8-en-3-beta-ol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.6','Methylitaconate Delta-isomerase','NULL','NULL','Forms part of the nicotinate-fermentation catabolism pathway inEubacterium barkeri.Other enzymes involved in this pathway are EC 1.17.1.5, EC 1.3.7.1,EC 3.5.2.18, EC 1.1.1.291, EC 5.4.99.4, EC 4.2.1.85 and EC 4.1.3.32.','Methylitaconate = 2,3-dimethylmaleate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.7','Aconitate Delta-isomerase','Aconitate isomerase','NULL','Cis-aconitate is used to designate the isomer (Z)-prop-1-ene-1,2,3-tricarboxylate.This isomerization could take place either in a direct cis-transinterconversion or by an allylic rearrangement; the enzyme has beenshown to catalyze the latter change.','Trans-aconitate = cis-aconitate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.8','Dodecenoyl-CoA isomerase','3,2-trans-enoyl-CoA isomerase','NULL','Also catalyzes the interconversion of 3-acetylenic fatty acylthioesters and (+)-2,3-dienoyl fatty acyl thioesters, with fatty acidchain lengths C(6) to C(12).','(3Z)-dodec-3-enoyl-CoA = (2E)-dodec-2-enoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.9','Prostaglandin-A(1) Delta-isomerase','Prostaglandin A isomerase','NULL','Interconverts prostaglandin A(1) and prostaglandin C(1).','(13E)-(15S)-15-hydroxy-9-oxoprosta-10,13-dienoate = (13E)-(15S)-15-hydroxy-9-oxoprosta-11,13-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.10','5-carboxymethyl-2-hydroxymuconate Delta-isomerase','5-carboxymethyl-2-hydroxymuconic acid isomerase','NULL','NULL','5-carboxymethyl-2-hydroxymuconate = 5-carboxy-2-oxohept-3-enedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.11','Isopiperitenone Delta-isomerase','NULL','NULL','Involved in the biosynthesis of menthol and related monoterpenes inpeppermint (Mentha piperita) leaves.','Isopiperitenone = piperitenone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.12','L-dopachrome isomerase','DCF','Zinc.','Stereospecific for L-dopachrome (5,6-dioxo-2,3-5,6-tetrahydroindole-2-carboxylate).Dopachrome methyl ester is a substrate, but dopaminochrome (2,3-dihydroxyindolyl-5,6-quinone) is not (see also EC 4.1.1.84).','L-dopachrome = 5,6-dihydroxyindole-2-carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.13','Polyenoic fatty acid isomerase','Eicosapentaenoate cis-Delta(5,8,11,14,17)-eicosapentaenoate cis-Delta(5)-trans-Delta(7,9)-cis-Delta(14,17) isomerase','NULL','The enzyme from the red alga Ptilota filicina catalyzes theisomerization of skip dienes (methylene-interrupted double bonds) ina broad range of fatty acids and fatty-acid analogs, such asarachidonate and gamma-linolenate, to yield a conjugated triene.','(5Z,8Z,11Z,14Z,17Z)-icosapentaenoate = (5Z,7E,9E,14Z,17Z)-icosapentaenoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.14','Trans-2-decenoyl-[acyl-carrier-protein] isomerase','Beta-hydroxydecanoyl thioester dehydrase','NULL','While the enzyme from Escherichia coli is highly specific for the10-carbon enoyl-ACP, the enzyme from Streptococcus pneumoniae canalso use the 12-carbon enoyl-ACP as substrate in vitro but not 14- or16-carbon enoyl-ACPs.ACP can be replaced by either CoA or N-acetylcysteamine thioesters.The cis-3-enoyl product is required to form unsaturated fatty acids,such as palmitoleic acid and cis-vaccenic acid, in dissociated (ortype II) fatty-acid biosynthesis.','Trans-dec-2-enoyl-[acyl-carrier-protein] = cis-dec-3-enoyl-[acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.15','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.16','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.17','Trans-2,3-dihydro-3-hydroxyanthranilate isomerase','NULL','NULL','The enzyme is involved in phenazine biosynthesis.','(5S,6S)-6-amino-5-hydroxycyclohexane-1,3-diene-1-carboxyate = (1R,6S)-6-amino-5-oxocyclohex-2-ene-1-carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.3.18','2-(1,2-epoxy-1,2-dihydrophenyl)acetyl-CoA isomerase','1,2-epoxyphenylacetyl-CoA isomerase','NULL','The enzyme catalyzes the reversible isomerization of 2-(1,2-epoxy-1,2-dihydrophenyl)acetyl-CoA to the unusual unsaturated, oxygen-containing, seven-member heterocyclic enol ether 2-oxepin-2(3H)-ylideneacetyl-CoA, as part of an aerobic phenylacetate degradationpathway.','2-(1,2-epoxy-1,2-dihydrophenyl)acetyl-CoA = 2-oxepin-2(3H)-ylideneacetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.4.1','Protein disulfide-isomerase','S-S rearrangase','NULL','Needs reducing agents or partly reduced enzyme; the reaction dependson sulfhydryl-disulfide interchange.','Catalyzes the rearrangement of -S-S- bonds in proteins.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.1','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.2','Prostaglandin-D synthase','PGD2 synthase','Glutathione.','Brings about the opening of the epidioxy bridge.','(5Z,13E,15S)-9-alpha,11-alpha-epidioxy-15-hydroxyprosta-5,13-dienoate =(5Z,13E,15S)-9-alpha,15-dihydroxy-11-oxoprosta-5,13-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.3','Prostaglandin-E synthase','Endoperoxide isomerase','Glutathione.','Brings about the opening of the epidioxy bridge.','(5Z,13E)-(15S)-9-alpha,11-alpha-epidioxy-15-hydroxyprosta-5,13-dienoate =(5Z,13E)-(15S)-11-alpha,15-dihydroxy-9-oxoprosta-5,13-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.4','Prostaglandin-I synthase','PGI(2) synthase','Heme-thiolate.','Converts prostaglandin H(2) into prostaglandin I(2) (prostacyclin).','(5Z,13E)-(15S)-9-alpha,11-alpha-epidioxy-15-hydroxyprosta-5,13-dienoate =(5Z,13E)-(15S)-6,9-alpha-epoxy-11-alpha,15-dihydroxyprosta-5,13-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.5','Thromboxane-A synthase','Thromboxane synthase','Heme-thiolate.','Converts prostaglandin H(2) into thromboxane A(2).','(5Z,13E)-(15S)-9-alpha,11-alpha-epidioxy-15-hydroxyprosta-5,13-dienoate =(5Z,13E)-(15S)-9-alpha,11-alpha-epoxy-15-hydroxythromboxa-5,13-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.6','Allene-oxide cyclase','NULL','NULL','Allene oxides formed by the action of EC 4.2.1.92 are converted intocyclopentenone derivatives.','(9Z)-(13S)-12,13-epoxyoctadeca-9,11,15-trienoate = (15Z)-12-oxophyto-10,15-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.7','Styrene-oxide isomerase','SOI','NULL','Highly specific.','Styrene oxide = phenylacetaldehyde.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.8','Capsanthin/capsorubin synthase','Capsanthin-capsorubin synthase','NULL','Induced during chromoplast differentiation in plants.Isomerization of the epoxide ring of violaxanthin gives thecyclopentyl-ketone of capsorubin or capsanthin.','(1) Violaxanthin = capsorubin.(2) Antheraxanthin = capsanthin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.9','Neoxanthin synthase','NSY','NULL','The opening of the epoxide ring of violaxanthin generates a chiralallene.Neoxanthin is a precursor of the plant hormone abscisic acid and thelast product of carotenoid synthesis in green plants.','Violaxanthin = neoxanthin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.10','Thiazole tautomerase','NULL','NULL','The enzyme catalyzes the irreversible aromatization of the thiazolemoiety of 2-((2R,5Z)-2-carboxy-4-methylthiazol-5(2H)-ylidene)ethylphosphate.','2-((2R,5Z)-2-carboxy-4-methylthiazol-5(2H)-ylidene)ethyl phosphate =2-(2-carboxy-4-methylthiazol-5-yl)ethyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.11','2-keto-myo-inositol isomerase','2KMI isomerase','Divalent cation.','The enzyme, found in the bacterium Bacillus subtilis, is part of themyo-inositol/D-chiro-inositol degradation pathway leading to acetyl-CoA.Formerly EC 5.3.99.n1.','2,4,6/3,5-pentahydroxycyclohexanone = 2D-2,3,5/4,6-pentahydroxycyclohexanone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.3.99.n1','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.1.1','Lysolecithin acylmutase','Lysolecithin migratase','NULL','NULL','2-lysolecithin = 3-lysolecithin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.1.2','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.1.3','2-methylfumaryl-CoA isomerase','Mesaconyl-CoA C(1)-C(4) CoA transferase','NULL','The enzyme, purified from the bacterium Chloroflexus aurantiacus,acts as an intramolecular CoA transferase and does not transfer CoAto free mesaconate.It is part of the 3-hydroxypropanoate cycle for carbon assimilation.','2-methylfumaryl-CoA = 3-methylfumaryl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.1','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.2','Phosphoglucomutase (alpha-D-glucose-1,6-bisphosphate-dependent)','Glucose phosphomutase','NULL','Maximum activity is only obtained in the presence of alpha-D-glucose1,6-bisphosphate.This bisphosphate is an intermediate in the reaction, being formed bytransfer of a phosphate residue from the enzyme to the substrate,but the dissociation of bisphosphate from the enzyme complex is muchslower than the overall isomerization.Also, more slowly, catalyzes the interconversion of 1-phosphate and6-phosphate isomers of many other alpha-D-hexoses, and theinterconversion of alpha-D-ribose 1-phosphate and 5-phosphate.Cf. EC 5.4.2.5.Formerly EC 2.7.5.1.','Alpha-D-glucose 1-phosphate = alpha-D-glucose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.3','Phosphoacetylglucosamine mutase','Acetylglucosamine phosphomutase','NULL','Activated by N-acetyl-alpha-D-glucosamine 1,6-bisphosphate.Formerly EC 2.7.5.2.','N-acetyl-alpha-D-glucosamine 1-phosphate = N-acetyl-D-glucosamine6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.4','Bisphosphoglycerate mutase','2,3-bisphosphoglycerate mutase','NULL','In the direction shown, the enzyme is phosphorylated by3-phosphoglyceroyl phosphate to give phosphoenzyme and3-phosphoglycerate.The latter is rephosphorylated by the enzyme to yield 2,3-diphosphoglycerate, but this reaction is slowed down by dissociationof 3-phosphoglycerate from the enzyme, which is therefore more activein the presence of added 3-phosphoglycerate.Also catalyzes, slowly, the reactions of EC 3.1.3.13 and EC 5.4.2.12.Formerly EC 2.7.5.4.','3-phospho-D-glyceroyl phosphate = 2,3-bisphospho-D-glycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.5','Phosphoglucomutase (glucose-cofactor)','Glucose phosphomutase','NULL','Activated by D-glucose, which probably acts as an acceptor for aphosphate residue from the substrate, thus being itself convertedinto the product.Cf. EC 5.4.2.2.Formerly EC 2.7.5.5.','Alpha-D-glucose 1-phosphate = alpha-D-glucose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.6','Beta-phosphoglucomutase','NULL','NULL','NULL','Beta-D-glucose 1-phosphate = beta-D-glucose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.7','Phosphopentomutase','Deoxyribomutase','NULL','Also converts 2-deoxy-alpha-D-ribose 1-phosphate into 2-deoxy-D-ribose 5-phosphate.Alpha-D-ribose 1,5-bisphosphate, 2-deoxy-alpha-D-ribose 1,5-bisphosphate, or alpha-D-glucose 1,6-bisphosphate can act ascofactor.Formerly EC 2.7.5.6.','Alpha-D-ribose 1-phosphate = D-ribose 5-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.8','Phosphomannomutase','Phosphomannose mutase','NULL','Alpha-D-mannose 1,6-bisphosphate or alpha-D-glucose 1,6-bisphosphatecan act as cofactor.Formerly EC 2.7.5.7.','Alpha-D-mannose 1-phosphate = D-mannose 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.9','Phosphoenolpyruvate mutase','PEP mutase','NULL','Involved in the biosynthesis of the C-P bond, although theequilibrium greatly favors phosphoenolpyruvate.','Phosphoenolpyruvate = 3-phosphonopyruvate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.10','Phosphoglucosamine mutase','NULL','NULL','The enzyme is involved in the pathway for bacterial cell-wallpeptidoglycan and lipopolysaccharide biosyntheses, being an essentialstep in the pathway for UDP-N-acetylglucosamine biosynthesis.The enzyme from Escherichia coli is activated by phosphorylation andcan be autophosphorylated in vitro by alpha-D-glucosamine 1,6-bisphosphate, which is an intermediate in the reaction, alpha-D-glucose 1,6-bisphosphate or ATP.It can also catalyze the interconversion of alpha-D-glucose1-phosphate and alpha-D-glucose 6-phosphate, although at a much lowerrate.','Alpha-D-glucosamine 1-phosphate = D-glucosamine 6-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.11','Phosphoglycerate mutase (2,3-diphosphoglycerate-dependent)','2,3-diphosphoglycerate dependent phosphoglycerate mutase','NULL','The enzymes from vertebrates, platyhelminths, mollusks, annelids,crustaceans, insects, algae, fungi, yeast and some bacteria(particularly Gram-negative) require 2,3-bisphospho-D-glycerate as acofactor.The enzyme is activated by 2,3-bisphospho-D-glycerate by transferringa phosphate to histidine (His(10) in man and Escherichia coli, His(8)in Saccharomyces cerevisiae).This phosphate can be transferred to the free OH of phosphoglycerate,followed by transfer of the phosphate already on the phosphoglycerateback to the histidine.Cf. EC 5.4.2.12.The enzyme has no requirement for metal ions.This enzyme also catalyze, slowly, the reactions of EC 5.4.2.4.Formerly EC 2.7.5.3 and EC 5.4.2.1.','2-phospho-D-glycerate = 3-phospho-D-glycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.2.12','Phosphoglycerate mutase (2,3-diphosphoglycerate-independent)','2,3-diphosphoglycerate-independent phosphoglycerate mutase','Manganese or cobalt.','The enzymes from higher plants, algae, fungi, nematodes, sponges,coelenterates, myriapods, arachnids, echinoderms, archaea and somebacteria (particularly Gram-positive) have maximum activity in theabsence of 2,3-bisphospho-D-glycerate.Cf. EC 5.4.2.11.The reaction involves a phosphotransferase reaction to serinefollowed by transfer back to the glycerate at the other position.Both metal ions are involved in the reaction.Formerly EC 2.7.5.3 and EC 5.4.2.1.','2-phospho-D-glycerate = 3-phospho-D-glycerate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.1','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.2','Lysine 2,3-aminomutase','NULL','NULL','Activity is stimulated by S-adenosyl-L-methionine and pyridoxalphosphate.','L-lysine = (3S)-3,6-diaminohexanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.3','Beta-lysine 5,6-aminomutase','Beta-lysine mutase','Cobalamin.','NULL','(3S)-3,6-diaminohexanoate = (3S,5S)-3,5-diaminohexanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.4','D-lysine 5,6-aminomutase','Adenosylcobalamin-dependent D-lysine 5,6-aminomutase','Cobalamin.','NULL','D-lysine = 2,5-diaminohexanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.5','D-ornithine 4,5-aminomutase','D-alpha-ornithine 5,4-aminomutase','Cobalamin; Pyridoxal 5-phosphate.','NULL','D-ornithine = (2R,4S)-2,4-diaminopentanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.6','Tyrosine 2,3-aminomutase','Tyrosine alpha,beta-mutase','MIO.','NULL','L-tyrosine = 3-amino-3-(4-hydroxyphenyl)propanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.7','Leucine 2,3-aminomutase','NULL','Cobalamin.','NULL','(2S)-alpha-leucine = (3R)-beta-leucine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.8','Glutamate-1-semialdehyde 2,1-aminomutase','Glutamate-1-semialdehyde aminotransferase','Pyridoxal 5-phosphate.','NULL','(S)-4-amino-5-oxopentanoate = 5-aminolevulinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.9','Glutamate 2,3-aminomutase','NULL','Iron-sulfur; Pyridoxal 5-phosphate; S-adenosyl-L-methionine.','This enzyme is a member of the AdoMet radical (radical SAM) family.It contains pyridoxal phosphate and a [4Fe-4S] cluster, which iscoordinated by 3 cysteines and binds an exchangeable S-adenosyl-L-methionine molecule.During the reaction cycle, the AdoMet forms a 5-deoxyadenosylradical, which is regenerated at the end of the reaction.','L-glutamate = 3-aminopentanedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.10','Phenylalanine aminomutase (L-beta-phenylalanine forming)','NULL','MIO.','Cf. EC 5.4.3.11.','L-phenylalanine = L-beta-phenylalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.3.11','Phenylalanine aminomutase (D-beta-phenylalanine forming)','NULL','MIO.','The enzyme from the bacterium Pantoea agglomerans produces D-beta-phenylalanine, an intermediate in the biosynthesis of the polyketidenon-ribosomal antibiotic andrimid.Cf. EC 5.4.3.10.','L-phenylalanine = D-beta-phenylalanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.4.1','(Hydroxyamino)benzene mutase','HAB mutase','NULL','NULL','(Hydroxyamino)benzene = 2-aminophenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.4.2','Isochorismate synthase','Isochorismate mutase','Magnesium.','The reaction is reversible.Formerly EC 5.4.99.6.','Chorismate = isochorismate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.4.3','3-(hydroxyamino)phenol mutase','3-hydroxylaminophenol mutase','NULL','NULL','3-hydroxyaminophenol = aminohydroquinone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.4.4','Geraniol isomerase','NULL','NULL','In absence of oxygen the bifunctional linalool dehydratase-isomerasecan catalyze in vitro two reactions, the isomerization of (3S)-linalool to geraniol and the hydration of myrcene to (3S)-linalool,the latter activity being classified as EC 4.2.1.127.','Geraniol = linalool.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.4.5','9,12-octadecadienoate 8-hydroperoxide 8R-isomerase','(8R,9Z,12Z)-8-hydroperoxy-9,12-octadecadienoate mutase ((5S,8R,9Z,12Z)-5,8-dihydroxy-9,12-octadecadienoate-forming)','Heme.','The bifunctional enzyme from Aspergillus nidulans uses different hemedomains to catalyze two separate reactions.Linoleic acid is oxidized within the N-terminal heme peroxidasedomain to (8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate (cf.EC 1.13.11.60), which is subsequently isomerized to (5S,8R,9Z,12Z)-5,8-dihydroxyoctadeca-9,12-dienoate within the C-terminal P450 hemethiolate domain.','(8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate = (5S,8R,9Z,12Z)-5,8-dihydroxyoctadeca-9,12-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.4.6','9,12-octadecadienoate 8-hydroperoxide 8S-isomerase','(8R,9Z,12Z)-8-hydroperoxy-9,12-octadecadienoate mutase ((7S,8S,9Z,12Z)-5,8-dihydroxy-9,12-octadecadienoate-forming)','Heme.','The bifunctional enzyme from Gaeumannomyces graminis catalyzes theoxidation of linoleic acid to (8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate (cf. EC 1.13.11.60), which is then isomerized to(7S,8S,9Z,12Z)-5,8-dihydroxyoctadeca-9,12-dienoate.Formerly EC 1.13.11.44.','(8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate = (7S,8S,9Z,12Z)-7,8-dihydroxyoctadeca-9,12-dienoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.1','Methylaspartate mutase','Beta-methylaspartate-glutamate mutase','Cobalamin.','NULL','L-threo-3-methylaspartate = L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.2','Methylmalonyl-CoA mutase','(R)-2-methyl-3-oxopropanoyl-CoA CoA-carbonylmutase','Cobalamin.','NULL','(R)-methylmalonyl-CoA = succinyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.3','2-acetolactate mutase','Acetohydroxy acid isomerase','L-ascorbate.','Also converts 2-aceto-2-hydroxybutanoate to 3-hydroxy-3-methyl-2-oxopentanoate.','2-acetolactate = 3-hydroxy-3-methyl-2-oxobutanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.4','2-methyleneglutarate mutase','Alpha-methyleneglutarate mutase','Cobalamin.','Forms part of the nicotinate-fermentation catabolism pathway inEubacterium barkeri.Other enzymes involved in this pathway are EC 1.17.1.5, EC 1.3.7.1,EC 3.5.2.18, EC 1.1.1.291, EC 5.3.3.6, EC 4.2.1.85 and EC 4.1.3.32.','2-methyleneglutarate = 2-methylene-3-methylsuccinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.5','Chorismate mutase','Hydroxyphenylpyruvate synthase','NULL','NULL','Chorismate = prephenate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.6','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.7','Lanosterol synthase','2,3-epoxysqualene--lanosterol cyclase','NULL','Formerly EC 1.14.1.3 and EC 1.99.1.13.','(3S)-2,3-epoxy-2,3-dihydrosqualene = lanosterol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.8','Cycloartenol synthase','2,3-epoxysqualene--cycloartenol cyclase','NULL','NULL','(3S)-2,3-epoxy-2,3-dihydrosqualene = cycloartenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.9','UDP-galactopyranose mutase','NULL','NULL','NULL','UDP-alpha-D-galactopyranose = UDP-alpha-D-galactofuranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.10','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.11','Isomaltulose synthase','Isomaltulose synthetase','NULL','Simultaneously produces isomaltulose (6-O-alpha-D-glucopyranosyl-D-fructose) and smaller amounts of trehalulose (1-O-alpha-D-glucopyranosyl-beta-D-fructose) from sucrose.Formerly EC 5.4.99.10.','Sucrose = 6-O-alpha-D-glucopyranosyl-D-fructofuranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.12','tRNA pseudouridine(38-40) synthase','PSUI','NULL','The uridylate residues at positions 38, 39 and 40 of nearly all tRNAsare isomerized to pseudouridine.TruA specifically modifies uridines at positions 38, 39, and/or 40 inthe anticodon stem loop of tRNAs with highly divergent sequences andstructures.','tRNA uridine(38-40) = tRNA pseudouridine(38-40).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.13','Isobutyryl-CoA mutase','NULL','Cobalt.','NULL','2-methylpropanoyl-CoA = butanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.14','4-carboxymethyl-4-methylbutenolide mutase','4-methyl-2-enelactone methyl-isomerase','NULL','NULL','4-carboxymethyl-4-methylbut-2-en-1,4-olide = 4-carboxymethyl-3-methylbut-2-en-1,4-olide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.15','(1-','(1,4)-alpha-D-glucan 1-alpha-D-glucosylmutase','NULL','Acts on (1-','4-((1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.16','Maltose alpha-D-glucosyltransferase','Maltose glucosylmutase','NULL','NULL','Maltose = alpha,alpha-trehalose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.17','Squalene--hopene cyclase','NULL','NULL','The enzyme also produces the cyclization product hopan-22-ol byaddition of water (cf. EC 4.2.1.129).Hopene and hopanol are formed at a constant ratio of 5:1.','Squalene = hop-22(29)-ene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.18','5-(carboxyamino)imidazole ribonucleotide mutase','N(5)-CAIR mutase','NULL','In eubacteria, fungi and plants, this enzyme, along with EC 6.3.4.18,is required to carry out the single reaction catalyzed by EC 4.1.1.21in vertebrates.In the absence of EC 6.3.2.6, the reaction is reversible.The substrate is readily converted into 5-amino-1-(5-phospho-D-ribosyl)imidazole by non-enzymic decarboxylation.','5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole = 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.19','16S rRNA pseudouridine(516) synthase','16S RNA pseudouridine 516 synthase','NULL','The enzyme is specific for uridine(516) in 16S rRNA.In vitro, the enzyme does not modify free 16S rRNA.The preferred substrate is a 5-terminal fragment of 16S rRNAcomplexed with 30S ribosomal proteins.','16S rRNA uridine(516) = 16S rRNA pseudouridine(516).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.20','23S rRNA pseudouridine(2457) synthase','NULL','NULL','The enzyme modifies uridine(2457) in a stem of 23S RNA in Escherichiacoli.','23S rRNA uridine(2457) = 23S rRNA pseudouridine(2457).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.21','23S rRNA pseudouridine(2604) synthase','NULL','NULL','The enzyme is not completely specific for uridine(2604) and can, to asmall extent, also react with uridine(2605).','23S rRNA uridine(2604) = 23S rRNA pseudouridine(2604).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.22','23S rRNA pseudouridine(2605) synthase','NULL','NULL','Pseudouridine synthase RluB converts uridine(2605) of 23S rRNA topseudouridine.','23S rRNA uridine(2605) = 23S rRNA pseudouridine(2605).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.23','23S rRNA pseudouridine(1911/1915/1917) synthase','NULL','NULL','Pseudouridine synthase RluD converts uridines at positions 1911,1915, and 1917 of 23S rRNA to pseudouridines.These nucleotides are located in the functionally important helix-loop 69 of 23S rRNA.','23S rRNA uridine(1911)/uridine(1915)/uridine(1917) = 23S rRNApseudouridine(1911)/pseudouridine(1915)/pseudouridine(1917).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.24','23S rRNA pseudouridine(955/2504/2580) synthase','NULL','NULL','The enzyme converts uridines at position 955, 2504 and 2580 of 23SrRNA to pseudouridines.','23S rRNA uridine(955)/uridine(2504)/uridine(2580) = 23S rRNApseudouridine(955)/pseudouridine(2504)/pseudouridine(2580).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.25','tRNA pseudouridine(55) synthase','Psi(55) synthase','NULL','Pseudourine synthase TruB from Escherichia coli specifically modifiesuridine(55) in tRNA molecules.The bifunctional archaeal enzyme also catalyzes the pseudouridylationof uridine(54).It is not known whether the enzyme from Escherichia coli can also acton position 54 in vitro, since this position is occupied in E.colitRNAs by thymine.','tRNA uridine(55) = tRNA pseudouridine(55).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.26','tRNA pseudouridine(65) synthase','NULL','NULL','TruC specifically modifies uridines at positions 65 in tRNA.','tRNA uridine(65) = tRNA pseudouridine(65).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.27','tRNA pseudouridine(13) synthase','tRNA Psi(13) synthase','NULL','Pseudouridine synthase PusS from Escherichia coli specifically actson uridine(13) in tRNA.The Pus7 protein from Saccharomyces cerevisiae is a multisite-multisubstrate pseudouridine synthase that is able to modifyuridine(13) in several yeast tRNAs, uridine(35) in the pre-tRNA(Tyr),uridine(35) in U2 small nuclear RNA, and uridine(50) in 5S rRNA.','tRNA uridine(13) = tRNA pseudouridine(13).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.28','tRNA pseudouridine(32) synthase','Pseudouridine synthase RluA','NULL','The dual-specificity enzyme also catalyzes the formation ofpseudouridine(746) in 23S rRNA.Cf. EC 5.4.99.29.','tRNA uridine(32) = tRNA pseudouridine(32).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.29','23S rRNA pseudouridine(746) synthase','23S RNA Psi(746) synthase','NULL','RluA is the sole protein responsible for the in vivo formation of 23SRNA pseudouridine(746).The dual-specificity enzyme also catalyzes the formation ofuridine(32) in tRNA.Cf. EC 5.4.99.28.','23S rRNA uridine(746) = 23S rRNA pseudouridine(746).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.30','UDP-arabinopyranose mutase','NULL','NULL','The reaction is reversible and at thermodynamic equilibrium thepyranose form is favored over the furanose form (90:10).','UDP-beta-L-arabinofuranose = UDP-beta-L-arabinopyranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.31','Thalianol synthase','NULL','NULL','NULL','(3S)-2,3-epoxy-2,3-dihydrosqualene = thalianol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.32','Protostadienol synthase','NULL','NULL','(17Z)-protosta-17(20),24-dien-3-beta-ol is a precursor of thesteroidal antibiotic helvolic acid.','(3S)-2,3-epoxy-2,3-dihydrosqualene = (17Z)-protosta-17(20),24-dien-3-beta-ol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.33','Cucurbitadienol synthase','NULL','NULL','NULL','(3S)-2,3-epoxy-2,3-dihydrosqualene = cucurbitadienol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.34','Germanicol synthase','NULL','NULL','The enzyme produces germanicol, beta-amyrin and lupeol in the ratio63:33:4.','(3S)-2,3-epoxy-2,3-dihydrosqualene = germanicol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.35','Taraxerol synthase','NULL','NULL','The enzyme gives taraxerol, beta-amyrin and lupeol in the ratio70:17:13.','(3S)-2,3-epoxy-2,3-dihydrosqualene = taraxerol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.36','Isomultiflorenol synthase','NULL','NULL','NULL','(3S)-2,3-epoxy-2,3-dihydrosqualene = isomultiflorenol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.37','Dammaradiene synthase','NULL','NULL','NULL','Squalene = dammara-20,24-diene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.38','Camelliol C synthase','NULL','Iron-sulfur; S-adenosyl-L-methionine.','The product is 97% camelliol, 2% achilleol A and 0.2% beta-amyrin.Achilleol is an isomer of camelliol C with a 4-methylenecyclohexanolring system.This enzyme probably evolved from EC 5.4.99.39.','(3S)-2,3-epoxy-2,3-dihydrosqualene = camelliol C.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.39','Beta-amyrin synthase','2,3-oxidosqualene beta-amyrin cyclase','NULL','Some organism possess a monofunctional beta-amyrin synthase, otherhave a multifunctional enzyme that also catalyzes the synthesis ofalpha-amyrin (EC 5.4.99.40) or lupeol (EC 5.4.99.41).Formerly EC 5.4.99.n1.','(3S)-2,3-epoxy-2,3-dihydrosqualene = beta-amyrin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.40','Alpha-amyrin synthase','2,3-oxidosqualene alpha-amyrin cyclase','NULL','A multifunctional enzyme which produces both alpha- and beta-amyrin(see EC 5.4.99.39).','(3S)-2,3-epoxy-2,3-dihydrosqualene = alpha-amyrin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.41','Lupeol synthase','NULL','NULL','Also forms some beta-amyrin.The recombinant enzyme from Arabidopsis thaliana gives a 1:1 mixtureof lupeol and lupan-3-beta,20-diol with small amounts of beta-amyrin,germanicol, taraxasterol and psi-taraxasterol.See EC 4.2.1.128.Formerly EC 5.4.99.n2.','(3S)-2,3-epoxy-2,3-dihydrosqualene = lupeol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.42','tRNA pseudouridine(31) synthase','NULL','NULL','The enzyme specifically acts on uridine(31) in tRNA.','tRNA uridine(31) = tRNA pseudouridine(31).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.43','21S rRNA pseudouridine(2819) synthase','NULL','NULL','The enzyme specifically acts on uridine(2819) in 21S rRNA.','21S rRNA uridine(2819) = 21S rRNA pseudouridine(2819).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.44','Mitochondrial tRNA pseudouridine(27/28) synthase','NULL','NULL','The mitochondrial enzyme Pus2p is specific for position 27 or 28 inmitochondrial tRNA.','Mitochondrial tRNA uridine(27/28) = mitochondrial tRNApseudouridine(27/28).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.45','tRNA pseudouridine(38/39) synthase','Pseudouridine synthase 3','NULL','The enzyme from Saccharomyces cerevisiae is active only towarduridine(38) and uridine(39), and shows no activity with uridine(40)(cf. EC 5.4.99.12).In vitro the enzyme from mouse is active on uridine(39) and veryslightly on uridine(38) (human tRNA(Leu)).','tRNA uridine(38/39) = tRNA pseudouridine(38/39).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.46','Shionone synthase','NULL','NULL','The enzyme gives traces of four other triterpenoids.','(3S)-2,3-epoxy-2,3-dihydrosqualene = shionone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.47','Parkeol synthase','NULL','NULL','The enzyme from rice (Oryza sativa) produces parkeol as a singleproduct.','(3S)-2,3-epoxy-2,3-dihydrosqualene = parkeol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.48','Achilleol B synthase','NULL','NULL','Achilleol B is probably formed by cleavage of the 8-14 and 9-10 bondsof (3S)-2,3-epoxy-2,3-dihydrosqualene as part of the cyclizationreaction, after formation of the oleanane skeleton.','(3S)-2,3-epoxy-2,3-dihydrosqualene = achilleol B.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.49','Glutinol synthase','NULL','NULL','The enzyme from Kalanchoe daigremontiana also gives traces of othertriterpenoids.','(3S)-2,3-epoxy-2,3-dihydrosqualene = glutinol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.50','Friedelin synthase','NULL','NULL','The enzyme from Kalanchoe daigremontiana also gives traces of othertriterpenoids.','(3S)-2,3-epoxy-2,3-dihydrosqualene = friedelin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.51','Baccharis oxide synthase','NULL','NULL','The enzyme from Stevia rebaudiana also gives traces of othertriterpenoids.','(3S)-2,3-epoxy-2,3-dihydrosqualene = baccharis oxide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.52','Alpha-seco-amyrin synthase','NULL','NULL','The enzyme from Arabidopsis thaliana is multifunctional and producesabout equal amounts of alpha- and beta-seco-amyrin.See EC 5.4.99.54.','(3S)-2,3-epoxy-2,3-dihydrosqualene = alpha-seco-amyrin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.53','Marneral synthase','NULL','NULL','Marneral is a triterpenoid formed by Grob fragmentation of the A ringof 2,3-epoxy-2,3-dihydrosqualene during cyclization.','(3S)-2,3-epoxy-2,3-dihydrosqualene = marneral.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.54','Beta-seco-amyrin synthase','NULL','NULL','The enzyme from Arabidopsis thaliana is multifunctional and producesabout equal amounts of alpha- and beta-seco-amyrin.See EC 5.4.99.52.','(3S)-2,3-epoxy-2,3-dihydrosqualene = beta-seco-amyrin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.55','Delta-amyrin synthase','NULL','NULL','The enzyme from tomato (Solanum lycopersicum) gives 48% delta-amyrin,18% alpha-amyrin, 13% beta-amyrin and traces of three or four othertriterpenoid alcohols.See also EC 5.4.99.40 and EC 5.4.99.39.','(3S)-2,3-epoxy-2,3-dihydrosqualene = delta-amyrin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.56','Tirucalladienol synthase','NULL','NULL','The product from Arabidopsis thaliana is 85% tirucalla-7,24-dien-3-beta-ol with trace amounts of other triterpenoids.','(3S)-2,3-epoxy-2,3-dihydrosqualene = tirucalla-7,24-dien-3-beta-ol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.57','Baruol synthase','BARS1','NULL','The enzyme from Arabidopsis thaliana also produces traces of 22 othertriterpenoids.','(3S)-2,3-epoxy-2,3-dihydrosqualene = baruol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.58','Methylornithine synthase','NULL','NULL','The enzyme is a member of the superfamily of S-adenosyl-L-methionine-dependent radical (radical AdoMet) enzymes.Binds a [4Fe-4S] cluster that is coordinated by 3 cysteines and anexchangeable S-adenosyl-L-methionine molecule.The reaction is part of the biosynthesis pathway of pyrrolysine,a naturally occurring amino acid found in some archaealmethyltransferases.','L-lysine = (3R)-3-methyl-D-ornithine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.59','dTDP-fucopyranose mutase','NULL','NULL','The enzyme is involved in the biosynthesis of the Escherichia coliO52 O antigen.','dTDP-alpha-D-fucopyranose = dTDP-alpha-D-fucofuranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.60','Cobalt-precorrin-8 methylmutase','NULL','NULL','The enzyme catalyzes the the conversion of cobalt-precorrin-8 tocobyrinate by methyl rearrangement, a step in the anaerobic (earlycobalt insertion) pathway of adenosylcobalamin biosynthesis.The equivalent enzyme in the aerobic pathway is EC 5.4.99.61.','Cobalt-precorrin-8 = cobyrinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.61','Precorrin-8X methylmutase','HBA synthase','NULL','The enzyme catalyzes the the conversion of precorrin-8X tohydrogenobyrinate by methyl rearrangement, a step in the aerobic(late cobalt insertion) pathway of adenosylcobalamin biosynthesis.The equivalent enzyme in the anaerobic pathway is EC 5.4.99.60.Formerly EC 5.4.1.2.','Precorrin-8X = hydrogenobyrinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.n1','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.4.99.n2','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.1','Muconate cycloisomerase','Cis,cis-muconate lactonizing enzyme I','Manganese.','Also acts, in the reverse reaction, on 3-methyl-cis,cis-hexa-dienedioate and, very slowly, on cis,trans-hexadienedioate.Not identical with EC 5.5.1.7 or EC 5.5.1.11.','2,5-dihydro-5-oxofuran-2-acetate = cis,cis-hexadienedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.2','3-carboxy-cis,cis-muconate cycloisomerase','3-carboxymuconate lactonizing enzyme','NULL','NULL','2-carboxy-2,5-dihydro-5-oxofuran-2-acetate = cis,cis-butadiene-1,2,4-tricarboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.3','Tetrahydroxypteridine cycloisomerase','NULL','NULL','NULL','Tetrahydroxypteridine = xanthine-8-carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.4','Inositol-3-phosphate synthase','1L-myo-inositol-1-phosphate lyase (isomerizing)','NAD(+).','The NAD(+) dehydrogenates the -CHOH- group to -CO- at C-5 of theglucose 6-phosphate, making C-6 into an active methylene, able tocondense with the -CHO at C-1; finally, the enzyme-bound NADHreconverts C-5 into the -CHOH- form.','D-glucose 6-phosphate = 1D-myo-inositol 3-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.5','Carboxy-cis,cis-muconate cyclase','3-carboxy-cis,cis-muconate lactonizing enzyme','NULL','NULL','3-carboxy-2,5-dihydro-5-oxofuran-2-acetate = 3-carboxy-cis,cis-muconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.6','Chalcone isomerase','Chalcone--flavonone isomerase','NULL','NULL','A chalcone = a flavanone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.7','Chloromuconate cycloisomerase','Muconate cycloisomerase II','Manganese.','Spontaneous elimination of HCl produces cis-4-carboxymethylenebut-2-en-4-olide.Also acts in reverse direction on 2-chloro-cis,cis-muconate.Not identical with EC 5.5.1.1 or EC 5.5.1.11.','2-chloro-2,5-dihydro-5-oxofuran-2-acetate = 3-chloro-cis,cis-muconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.8','(+)-bornyl diphosphate synthase','Bornyl pyrophosphate synthase','NULL','The enzyme from Salvia officinalis (sage) can also use (3R)-linalyldiphosphate or more slowly neryl diphosphate in vitro.The reaction proceeds via isomeration of geranyl diphosphate to (3R)-linalyl diphosphate.The oxygen and phosphorus originally linked to C-1 of geranyldiphosphate end up linked to C-2 of (+)-bornyl diphosphate.cf. EC 5.5.1.22.','Geranyl diphosphate = (+)-bornyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.9','Cycloeucalenol cycloisomerase','Cycloeucalenol--obtusifoliol isomerase','NULL','Opens the cyclopropane ring of a number of related 4-alpha-methyl-9-beta-19-cyclosterols, but not those with a 4-beta-methyl group, withformation of an 8(9) double bond.Involved in the synthesis of plant sterols.','Cycloeucalenol = obtusifoliol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.10','Alpha-pinene-oxide decyclase','Alpha-pinene oxide lyase','NULL','Both rings of pinene are cleaved in the reaction.','Alpha-pinene oxide = (Z)-2-methyl-5-isopropylhexa-2,5-dienal.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.11','Dichloromuconate cycloisomerase','NULL','Manganese.','The product of cyclisomerization of dichloro-cis,cis-muconatespontaneously eliminates chloride to produce cis-4-carboxymethylene-3-chlorobut-2-en-4-olide.Also acts, in the reverse direction, on cis,cis-muconate and itsmonochloro-derivatives, but with lower affinity.Not identical with EC 5.5.1.1 or EC 5.5.1.7.','2,4-dichloro-2,5-dihydro-5-oxofuran-2-acetate = 2,4-dichloro-cis,cis-muconate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.12','Copalyl diphosphate synthase','NULL','NULL','In some plants, such as Salvia miltiorrhiza, this enzyme ismonofunctional.In other plants this activity is often a part of a bifunctionalenzyme.For example, in Selaginella moellendorffii this activity is catalyzedby a bifunctional enzyme that also catalyzes EC 4.2.3.131, while inthe tree Abies grandis (grand fir) it is catalyzed by a bifunctionalenzyme that also catalyzes EC 4.2.3.18.','Geranylgeranyl diphosphate = (+)-copalyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.13','Ent-copalyl diphosphate synthase','Ent-CDP synthase','NULL','Part of a bifunctional enzyme involved in the biosynthesis ofkaurene.See also EC 4.2.3.19.','Geranylgeranyl diphosphate = ent-copalyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.14','Syn-copalyl-diphosphate synthase','Syn-copalyl diphosphate synthase','Magnesium.','This class II terpene synthase produces syn-copalyl diphosphate,a precursor of several rice phytoalexins, including oryzalexin S andmomilactones A and B.Phytoalexins are diterpenoid secondary metabolites that are involvedin the defense mechanism of the plant, and are produced in responseto pathogen attack through the perception of elicitor signalmolecules such as chitin oligosaccharide, or after exposure to UVirradiation.Constitutively expressed in the roots of plants where one of itsproducts, momilactone B, acts as an allelochemical (a moleculereleased into the environment to suppress the growth of neighboringplants).In other tissues the enzyme is up-regulated by conditions thatstimulate the biosynthesis of phytoalexins.','Geranylgeranyl diphosphate = 9-alpha-copalyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.15','Terpentedienyl-diphosphate synthase','Clerodadienyl diphosphate synthase','Magnesium.','Contains a DXDD motif, which is a characteristic of diterpene cylaseswhose reactions are initiated by protonation at the 14,15-double bondof geranylgeranyl diphosphate (GGDP).The triggering proton is lost at the end of the cyclization reaction.The product of the reaction, terpentedienyl diphosphate, is thesubstrate for EC 4.2.3.36 and is a precursor of the diterpenoidantibiotic terpentecin.','Geranylgeranyl diphosphate = terpentedienyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.16','Halimadienyl-diphosphate synthase','Tuberculosinol diphosphate synthase','Magnesium.','Found in pathogenic prokaryotes such as Mycobacterium tuberculosisbut not in non-pathogens such as Mycobacterium smegmatis so may playa role in pathogenicity.The product of the reaction is subsequently dephosphorylated yieldingtuberculosinol (halima-5,13-dien-15-ol).','Geranylgeranyl diphosphate = tuberculosinyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.17','(S)-beta-macrocarpene synthase','NULL','NULL','NULL','(S)-beta-bisabolene = (S)-beta-macrocarpene.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.18','Lycopene epsilon-cyclase','NULL','NULL','The carotenoid lycopene has the psi-end group at both ends.When acting on one end, this enzyme forms delta-carotene.When acting on both ends, it forms epsilon-carotene.','Carotenoid psi-end group = carotenoid epsilon-end group.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.19','Lycopene beta-cyclase','NULL','FAD.','Lycopene has a psi-end group at both ends.When acting on one end, this enzyme forms gamma-carotene.When acting on both ends it forms beta-carotene.It also acts on neurosporene to give beta-zeacarotene.Formerly EC 5.5.1.n2.','Carotenoid psi-end group = carotenoid beta-end group.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.20','Prosolanapyrone-III cycloisomerase','Prosolanapyrone II oxidase/prosolanapyrone III cyclosiomerase','NULL','The enzyme is involved in the biosynthesis of the phytotoxinsolanapyrone in some fungi.The bifunctional enzyme catalyzes the oxidation of prosolanapyrone IIand the subsequent Diels Alder cycloisomerization of the productprosolanapyrone III to (-)-solanapyrone A (cf. EC 1.1.3.42).','Prosolanapyrone III = (-)-solanapyrone A.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.21','Transferred entry: 4','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.22','(-)-bornyl diphosphate synthase','(-)-bornyl pyrophosphate cyclase','NULL','The enzyme from Tanacetum vulgare (tansey) can also use (3S)-linalyldiphosphate or more slowly neryl diphosphate in vitro.The reaction proceeds via isomeration of geranyl diphosphate to (3S)-linalyl diphosphate.The oxygen and phosphorus originally linked to C-1 of geranyldiphosphate end up linked to C-2 of (-)-bornyl diphosphate.Cf. EC 5.5.1.8.','Geranyl diphosphate = (-)-bornyl diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.23','Aklanonic acid methyl ester cyclase','Methyl aklanonate cyclase','NULL','The enzyme is involved in the biosynthesis of aklaviketone,an intermediate in the biosynthetic pathways leading to formation ofseveral anthracycline antibiotics, including aclacinomycin,daunorubicin and doxorubicin.','Methyl aklanonate = aklaviketone.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.24','Tocopherol cyclase','NULL','NULL','The enzyme has been described from plants and cyanobacteria.It has similar activity with all four listed benzoquinol substrates.Involved in the biosynthesis of vitamin E tocopherols andtocotrienols.','(1) Delta-tocopherol = 2-methyl-6-phytylbenzene-1,4-diol.(2) Gamma-tocopherol = 2,3-dimethyl-6-phytylbenzene-1,4-diol.(3) Delta-tocotrienol = 2-methyl-6-geranylgeranylbenzene-1,4-diol.(4) Gamma-tocotrienol = 2,3-dimethyl-6-geranylgeranylbenzene-1,4-diol.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.n1','D-ribose pyranase','NULL','NULL','The enzyme also catalyzes the conversion between beta-allofuranoseand beta-allopyranose.','Beta-D-ribopyranose = beta-D-ribofuranose.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.5.1.n2','Transferred entry: 5','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.99.1.1','Thiocyanate isomerase','Isothiocyanate isomerase','NULL','NULL','Benzyl isothiocyanate = benzyl thiocyanate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.99.1.2','DNA topoisomerase','DNA topoisomerase I','NULL','Brings about the conversion of one topological isomer of DNA intoanother, e.g. the relaxation of superhelical turns in DNA,the interconversion of simple and knotted rings of single-strandedDNA, and the intertwisting of single-stranded rings of complementarysequences (cf. EC 5.99.1.3).','ATP-independent breakage of single-stranded DNA, followed by passage andrejoining.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.99.1.3','DNA topoisomerase (ATP-hydrolyzing)','DNA gyrase','NULL','Can introduce negative superhelical turns into double-strandedcircular DNA.One unit has nicking-closing activity, and another catalyzes super-twisting and hydrolysis of ATP (cf. EC 5.99.1.2).','ATP-dependent breakage, passage and rejoining of double-stranded DNA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 5.99.1.4','2-hydroxychromene-2-carboxylate isomerase','2HC2CA isomerase','NULL','This enzyme is involved in naphthalene degradation.','2-hydroxy-2H-chromene-2-carboxylate = (3E)-4-(2-hydroxyphenyl)-2-oxobut-3-enoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.1','Tyrosine--tRNA ligase','L-tyrosine-tRNA(Tyr) ligase (AMP-forming)','NULL','NULL','ATP + L-tyrosine + tRNA(Tyr) = AMP + diphosphate + L-tyrosyl-tRNA(Tyr).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.2','Tryptophan--tRNA ligase','L-tryptophan-tRNA(Trp) ligase (AMP-forming)','NULL','NULL','ATP + L-tryptophan + tRNA(Trp) = AMP + diphosphate + L-tryptophyl-tRNA(Trp).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.3','Threonine--tRNA ligase','Threonine translase','NULL','NULL','ATP + L-threonine + tRNA(Thr) = AMP + diphosphate + L-threonyl-tRNA(Thr).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.4','Leucine--tRNA ligase','Leucine translase','NULL','NULL','ATP + L-leucine + tRNA(Leu) = AMP + diphosphate + L-leucyl-tRNA(Leu).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.5','Isoleucine--tRNA ligase','Isoleucine translase','NULL','NULL','ATP + L-isoleucine + tRNA(Ile) = AMP + diphosphate + L-isoleucyl-tRNA(Ile).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.6','Lysine--tRNA ligase','Lysine translase','NULL','NULL','ATP + L-lysine + tRNA(Lys) = AMP + diphosphate + L-lysyl-tRNA(Lys).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.7','Alanine--tRNA ligase','Alanine translase','NULL','NULL','ATP + L-alanine + tRNA(Ala) = AMP + diphosphate + L-alanyl-tRNA(Ala).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.8','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.9','Valine--tRNA ligase','Valine translase','NULL','NULL','ATP + L-valine + tRNA(Val) = AMP + diphosphate + L-valyl-tRNA(Val).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.10','Methionine--tRNA ligase','Methionine translase','NULL','In those organisms producing N-formylmethionyl-tRNA(fMet) fortranslation initiation, this enzyme also recognizes the initiatortRNA(fMet) and catalyzes the formation of L-methionyl-tRNA(fMet),the substrate for EC 2.1.2.9.','ATP + L-methionine + tRNA(Met) = AMP + diphosphate + L-methionyl-tRNA(Met).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.11','Serine--tRNA ligase','Serine translase','NULL','This enzyme also recognizes tRNA(Sec), the special tRNA forselenocysteine, and catalyzes the formation of L-seryl-tRNA(Sec),the substrate for EC 2.9.1.1.','ATP + L-serine + tRNA(Ser) = AMP + diphosphate + L-seryl-tRNA(Ser).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.12','Aspartate--tRNA ligase','Aspartic acid translase','NULL','NULL','ATP + L-aspartate + tRNA(Asp) = AMP + diphosphate + L-aspartyl-tRNA(Asp).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.13','D-alanine--poly(phosphoribitol) ligase','D-alanine-activating enzyme','NULL','A thioester bond is formed transiently between D-alanine and thesulfhydryl group of the 4-phosphopantetheine prosthetic group ofD-alanyl carrier protein during the activation of the alanine.Involved in the synthesis of teichoic acids.','ATP + D-alanine + poly(ribitol phosphate) = AMP + diphosphate + O-D-alanyl-poly(ribitol phosphate).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.14','Glycine--tRNA ligase','Glycyl translase','NULL','NULL','ATP + glycine + tRNA(Gly) = AMP + diphosphate + glycyl-tRNA(Gly).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.15','Proline--tRNA ligase','Proline translase','NULL','NULL','ATP + L-proline + tRNA(Pro) = AMP + diphosphate + L-prolyl-tRNA(Pro).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.16','Cysteine--tRNA ligase','Cysteine translase','NULL','NULL','ATP + L-cysteine + tRNA(Cys) = AMP + diphosphate + L-cysteinyl-tRNA(Cys).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.17','Glutamate--tRNA ligase','Glutamic acid translase','NULL','NULL','ATP + L-glutamate + tRNA(Glu) = AMP + diphosphate + L-glutamyl-tRNA(Glu).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.18','Glutamine--tRNA ligase','Glutamine translase','NULL','NULL','ATP + L-glutamine + tRNA(Gln) = AMP + diphosphate + L-glutaminyl-tRNA(Gln).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.19','Arginine--tRNA ligase','Arginine translase','NULL','NULL','ATP + L-arginine + tRNA(Arg) = AMP + diphosphate + L-arginyl-tRNA(Arg).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.20','Phenylalanine--tRNA ligase','Phenylalanine translase','NULL','NULL','ATP + L-phenylalanine + tRNA(Phe) = AMP + diphosphate + L-phenylalanyl-tRNA(Phe).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.21','Histidine--tRNA ligase','Histidine translase','NULL','NULL','ATP + L-histidine + tRNA(His) = AMP + diphosphate + L-histidyl-tRNA(His).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.22','Asparagine--tRNA ligase','Asparagine translase','NULL','NULL','ATP + L-asparagine + tRNA(Asn) = AMP + diphosphate + L-asparaginyl-tRNA(Asn).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.23','Aspartate--tRNA(Asn) ligase','Nondiscriminating aspartyl-tRNA synthetase','NULL','When this enzyme acts on tRNA(Asp), it catalyzes the same reaction asEC 6.1.1.12.It has, however, diminished discrimination, so that it can also formaspartyl-tRNA(Asn).This relaxation of specificity has been found to result from theabsence of a loop in the tRNA that specifically recognizes the thirdposition of the anticodon.This accounts for the ability of this enzyme in, for example, Thermusthermophilus, to recognize both tRNA(Asp) (GUC anticodon) andtRNA(Asn) (GUU anticodon).The aspartate-tRNA(Asn) is not used in protein synthesis until it isconverted by EC 6.3.5.6 into asparaginyl-tRNA(Asn).','ATP + L-aspartate + tRNA(Asx) = AMP + diphosphate + L-aspartyl-tRNA(Asx).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.24','Glutamate--tRNA(Gln) ligase','Nondiscriminating glutamyl-tRNA synthetase','NULL','When this enzyme acts on tRNA(Glu), it catalyzes the same reaction asEC 6.1.1.17.It has, however, diminished discrimination, so that it can also formglutamate-tRNA(Gln).This relaxation of specificity has been found to result from theabsence of a loop in the tRNA that specifically recognizes the thirdposition of the anticodon.This accounts for the ability of this enzyme in, for example,Bacillus subtilis, to recognize both tRNA(1)(Gln) (UUG anticodon) andtRNA(Glu) (UUC anticodon) but not tRNA(2)(Gln) (CUG anticodon).The ability of this enzyme to recognize both tRNA(Glu) and one of thetRNA(Gln) isoacceptors derives from their sharing a major identityelement, a hypermodified derivative of U34 (5-methylaminomethyl-2-thiouridine).The glutamyl-tRNA(Gln) is not used in protein synthesis until it isconverted by EC 6.3.5.7 into glutaminyl-tRNA(Gln).','ATP + L-glutamate + tRNA(Glx) = AMP + diphosphate + L-glutamyl-tRNA(Glx).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.25','Deleted entry','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.26','Pyrrolysine--tRNA(Pyl) ligase','PylRS','NULL','In organisms such as Methanosarcina barkeri that incorporate themodified amino acid pyrrolysine (Pyl) into certain methylaminemethyltransferases, an unusual tRNA(Pyl), with a CUA anticodon,can be charged directly with pyrrolysine by this class II aminoacyl--tRNA ligase.Specific for pyrrolysine as substrate as it cannot be replaced bylysine or any of the other natural amino acids.','ATP + L-pyrrolysine + tRNA(Pyl) = AMP + diphosphate + L-pyrrolysyl-tRNA(Pyl).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.27','O-phosphoserine--tRNA ligase','O-phosphoseryl-tRNA ligase','NULL','In organisms like Archaeoglobus fulgidus lacking EC 6.1.1.16 for thedirect Cys-tRNA(Cys) formation, Cys-tRNA(Cys) is produced by anindirect pathway, in which EC 6.1.1.27 ligates O-phosphoserine totRNA(Cys), and EC 2.5.1.73 converts the produced O-phospho-L-seryl-tRNA(Cys) to Cys-tRNA(Cys).The SepRS/SepCysS pathway is the sole route for cysteine biosynthesisin the organism.Methanosarcina mazei can use both pathways, the direct route usingEC 6.1.1.16 and the indirect pathway with EC 6.1.1.27 andEC 2.5.1.73.Formerly EC 6.1.1.n2.','ATP + O-phospho-L-serine + tRNA(Cys) = AMP + diphosphate + O-phospho-L-seryl-tRNA(Cys).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.n1','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.1.n2','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.1.2.1','D-alanine--(R)-lactate ligase','NULL','NULL','The product of this enzyme, the depsipeptide D-alanyl-(R)-lactate,can be incorporated into the peptidoglycan pentapeptide instead ofthe usual D-alanyl-D-alanine dipeptide, which is formed byEC 6.3.2.4.The resulting peptidoglycan does not bind the glycopeptideantibiotics vancomycin and teicoplanin, conferring resistance on thebacteria.','D-alanine + (R)-lactate + ATP = D-alanyl-(R)-lactate + ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.1','Acetate--CoA ligase','Acetate thiokinase','NULL','Also acts on propanoate and propenoate.','ATP + acetate + CoA = AMP + diphosphate + acetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.2','Butyrate--CoA ligase','Acyl-activating enzyme','NULL','Acts on acids from C(4) to C(11) and on the corresponding 3-hydroxy-and 2,3- or 3,4-unsaturated acids.','ATP + a carboxylate + CoA = AMP + diphosphate + an acyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.3','Long-chain-fatty-acid--CoA ligase','Acyl-activating enzyme','NULL','Acts on a wide range of long-chain saturated and unsaturated fattyacids, but the enzymes from different tissues show some variation inspecificity.The liver enzyme acts on acids from C(6) to C(20); that from brainshows high activity up to C(24).','ATP + a long-chain fatty acid + CoA = AMP + diphosphate + an acyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.4','Succinate--CoA ligase (GDP-forming)','Succinyl-CoA synthetase (GDP-forming)','NULL','Itaconate can act instead of succinate and ITP instead of GTP.','GTP + succinate + CoA = GDP + phosphate + succinyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.5','Succinate--CoA ligase (ADP-forming)','Succinate thiokinase','NULL','NULL','ATP + succinate + CoA = ADP + phosphate + succinyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.6','Glutarate--CoA ligase','Glutaryl-CoA synthetase','NULL','GTP or ITP can act instead of ATP.','ATP + glutarate + CoA = ADP + phosphate + glutaryl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.7','Cholate--CoA ligase','3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestanate--CoA ligase','Magnesium.','This membrane-bound enzyme catalyzes the first step in theconjugation of bile acids with amino acids, converting bile acidsinto their acyl-CoA thioesters.The second step involves EC 2.3.1.65 and converts the acyl-CoAthioester into the corresponding N-acyl amidate by conjugation withglycine or taurine.Chenodeoxycholate, deoxycholate, lithocholate andtrihydroxycoprostanoate can also act as substrates.Formerly EC 6.2.1.29.','(1) ATP + cholate + CoA = AMP + diphosphate + choloyl-CoA.(2) ATP + (25R)-3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestan-26-oate + CoA = AMP + diphosphate + (25R)-3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.8','Oxalate--CoA ligase','Oxalyl-CoA synthetase','NULL','NULL','ATP + oxalate + CoA = AMP + diphosphate + oxalyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.9','Malate--CoA ligase','Malate thiokinase','NULL','NULL','ATP + malate + CoA = ADP + phosphate + malyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.10','Acid--CoA ligase (GDP-forming)','Acyl-CoA synthetase (GDP-forming)','NULL','NULL','GTP + a carboxylate + CoA = GDP + phosphate + acyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.11','Biotin--CoA ligase','Biotinyl-CoA synthetase','NULL','NULL','ATP + biotin + CoA = AMP + diphosphate + biotinyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.12','4-coumarate--CoA ligase','4-coumaroyl-CoA synthase','NULL','NULL','ATP + 4-coumarate + CoA = AMP + diphosphate + 4-coumaroyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.13','Acetate--CoA ligase (ADP-forming)','Acetate thiokinase','NULL','Also acts on propanoate and, very slowly, on butanoate.','ATP + acetate + CoA = ADP + phosphate + acetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.14','6-carboxyhexanoate--CoA ligase','6-carboxyhexanoyl-CoA synthetase','NULL','NULL','ATP + 6-carboxyhexanoate + CoA = AMP + diphosphate + 6-carboxyhexanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.15','Arachidonate--CoA ligase','Arachidonoyl-CoA synthetase','NULL','8,11,14-icosatrienoate, but not the other long-chain fatty acids,can act in place of arachidonate.Not identical with EC 6.2.1.3.','ATP + arachidonate + CoA = AMP + diphosphate + arachidonoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.16','Acetoacetate--CoA ligase','Acetoacetyl-CoA synthetase','NULL','Also acts, more slowly, on L-3-hydroxybutanoate.','ATP + acetoacetate + CoA = AMP + diphosphate + acetoacetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.17','Propionate--CoA ligase','Propionyl-CoA synthetase','NULL','Propenoate can act instead of propanoate.Not identical with EC 6.2.1.1 or EC 6.2.1.2.','ATP + propanoate + CoA = AMP + diphosphate + propanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.18','Citrate--CoA ligase','NULL','NULL','Component of EC 4.1.3.8.','ATP + citrate + CoA = ADP + phosphate + (3S)-citryl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.19','Long-chain-fatty-acid--luciferin-component ligase','Acyl-protein synthetase','NULL','Together with EC 1.2.1.50 forms a fatty acid reductase system whichproduces the substrate of EC 1.14.14.3, thus being a component of thebacterial luciferase system.','ATP + a long-chain fatty acid + protein = AMP + diphosphate + an acyl-protein thioester.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.20','Long-chain-fatty-acid--[acyl-carrier-protein] ligase','Acyl-[acyl-carrier-protein] synthetase','NULL','Not identical with EC 6.2.1.3.','ATP + an acid + [acyl-carrier-protein] = AMP + diphosphate + acyl-[acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.21','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.22','[Citrate (pro-3S)-lyase] ligase','Citrate lyase ligase','NULL','Converts the inactive thiol form of EC 4.1.3.6 into the active form.','ATP + acetate + [citrate (pro-3S)-lyase](thiol form) = AMP + diphosphate+ [citrate (pro-3S)-lyase](acetyl form).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.23','Dicarboxylate--CoA ligase','NULL','NULL','Acts on dicarboxylic acids of chain length C(5) to C(16); the bestsubstrate is dodecanedioic acid.','ATP + an alpha-omega-dicarboxylate + CoA = AMP + diphosphate + an omega-carboxyacyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.24','Phytanate--CoA ligase','NULL','NULL','Not identical with EC 6.2.1.20.','ATP + phytanate + CoA = AMP + diphosphate + phytanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.25','Benzoate--CoA ligase','NULL','NULL','Also acts on 2-, 3- and 4-fluorobenzoate, but only very slowly on thecorresponding chlorobenzoates.','ATP + benzoate + CoA = AMP + diphosphate + benzoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.26','o-succinylbenzoate--CoA ligase','o-succinylbenzoyl-CoA synthetase','NULL','NULL','ATP + 2-succinylbenzoate + CoA = AMP + diphosphate + 2-succinylbenzoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.27','4-hydroxybenzoate--CoA ligase','4-hydroxybenzoate-CoA synthetase','NULL','NULL','ATP + 4-hydroxybenzoate + CoA = AMP + diphosphate + 4-hydroxybenzoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.28','3-alpha,7-alpha-dihydroxy-5-beta-cholestanate--CoA ligase','DHCA-CoA ligase','NULL','NULL','ATP + 3-alpha,7-alpha-dihydroxy-5-beta-cholestanate + CoA = AMP +diphosphate + 3-alpha,7-alpha-dihydroxy-5-beta-cholestanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.29','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.30','Phenylacetate--CoA ligase','NULL','NULL','Also acts, more slowly, on acetate, propanoate and butanoate, but noton hydroxy derivatives of phenylacetate and related compounds.Formerly EC 6.2.1.21.','ATP + phenylacetate + CoA = AMP + diphosphate + phenylacetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.31','2-furoate--CoA ligase','NULL','NULL','NULL','ATP + 2-furoate + CoA = AMP + diphosphate + 2-furoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.32','Anthranilate--CoA ligase','NULL','NULL','NULL','ATP + anthranilate + CoA = AMP + diphosphate + anthraniloyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.33','4-chlorobenzoate--CoA ligase','NULL','Magnesium.','Part of the bacterial 2,4-dichlorobenzoate degradation pathway.','4-chlorobenzoate + CoA + ATP = 4-chlorobenzoyl-CoA + AMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.34','Trans-feruloyl-CoA synthase','Trans-feruloyl-CoA synthetase','Magnesium.','It has not yet been established whether AMP + diphosphate or ADP +phosphate are formed in this reaction.','Ferulic acid + CoA + ATP = feruloyl-CoA + products of ATP breakdown.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.35','ACP-SH:acetate ligase','[Acyl-carrier protein]:acetate ligase','NULL','This enzyme, from the anaerobic bacterium Malonomonas rubra, is acomponent of the multienzyme complex EC 4.1.1.89.The enzyme uses the energy from hydrolysis of ATP to convert thethiol group of the acyl-carrier-protein-bound 2-(5-phosphoribosyl)-3-dephospho-CoA prosthetic group into its acetyl thioester.','ATP + acetate + an [acyl-carrier protein] = AMP + diphosphate +an acetyl-[acyl-carrier protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.36','3-hydroxypropionyl-CoA synthase','3-hydroxypropionate--CoA ligase','NULL','Catalyzes a step in the 3-hydroxypropanoate/4-hydroxybutyrate cycle,an autotrophic CO(2) fixation pathway found in some thermoacidophilicarchaea.The enzymes from Metallosphaera sedula and Sulfolobus tokodaii canalso use propanoate, acrylate, acetate, and butanoate as substrates,and are thus different from EC 6.2.1.17, which does not acceptacetate or butanoate.','3-hydroxypropanoate + ATP + CoA = 3-hydroxypropionyl-CoA + AMP +diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.37','3-hydroxybenzoate--CoA ligase','3-hydroxybenzoate--coenzyme A ligase (AMP-forming)','NULL','The enzyme works equally well with 4-hydroxybenzoate but shows lowactivity toward benzoate, 4-aminobenzoate, 3-aminobenzoate,3-fluorobenzoate, 4-fluorobenzoate, 3-chlorobenzoate,and 4-chlorobenzoate.There is no activity with 3,4-dihydroxybenzoate, 2,3-dihydroxybenzoate, and 2-hydroxybenzoate as substrates.Formerly EC 6.2.1.n1.','ATP + 3-hydroxybenzoate + CoA = AMP + diphosphate + 3-hydroxybenzoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.38','(2,2,3-trimethyl-5-oxocyclopent-3-enyl)acetyl-CoA synthase','2-oxo-Delta(3)-4,5,5-trimethylcyclopentenylacetyl-CoA synthetase','NULL','Isolated from Pseudomonas putida.Forms part of the pathway of camphor catabolism.','((1R)-2,2,3-trimethyl-5-oxocyclopent-3-enyl)acetate + ATP + CoA = AMP +diphosphate + ((1R)-2,2,3-trimethyl-5-oxocyclopent-3-enyl)acetyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.39','[Butirosin acyl-carrier protein]--L-glutamate ligase','[BtrI acyl-carrier protein]--L-glutamate ligase','Manganese or magnesium.','Catalyzes two steps in the biosynthesis of the side chain of theaminoglycoside antibiotics of the butirosin family.The enzyme adds one molecule of L-glutamate to a dedicated acyl-carrier protein, and following decarboxylation of the product byEC 4.1.1.95, adds a second L-glutamate molecule.','(1) ATP + L-glutamate + [BtrI acyl-carrier protein] = ADP + phosphate +L-glutamyl-[BtrI acyl-carrier protein].(2) ATP + L-glutamate + 4-amino butanoyl-[BtrI acyl-carrier protein] =ADP + phosphate + 4-(L-gamma-glutamylamino)butanoyl-[BtrI acyl-carrierprotein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.40','4-hydroxybutyrate--CoA ligase','4-hydroxybutyrate-CoA synthetase','NULL','Isolated from the archaeon Metallosphaera sedula.Involved in the 3-hydroxypropionate/4-hydroxybutyrate cycle.','ATP + 4-hydroxybutanoate + CoA = AMP + diphosphate + 4-hydroxybutanoyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.n1','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.n2','Amino acid--[acyl-carrier-protein] ligase','NULL','NULL','Catalyzes the ATP-dependent activation of L-alanine, and to a lesserextent of L-glycine and L-serine, and their transfer to thephosphopantetheine prosthetic group covalently attached to a vicinalcarrier protein.May participate in nonribosomal peptide synthesis or relatedprocesses.L-proline and L-glutamate are very poor substrates whereas D-aminoacids are not substrates for ATP-dependent activation.Does not display tRNA aminoacylation activity.','ATP + an amino acid + [acyl-carrier-protein] = AMP + diphosphate +an aminoacyl-[acyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.2.1.n3','Malonate--CoA ligase','NULL','NULL','NULL','ATP + malonate + CoA = AMP + diphosphate + malonyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.1','Aspartate--ammonia ligase','Asparagine synthetase','NULL','NULL','ATP + L-aspartate + NH(3) = AMP + diphosphate + L-asparagine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.2','Glutamate--ammonia ligase','Glutamine synthetase','NULL','Also acts, more slowly, on 4-methylene-L-glutamate (cf. EC 6.3.1.7).','ATP + L-glutamate + NH(3) = ADP + phosphate + L-glutamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.3','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.4','Aspartate--ammonia ligase (ADP-forming)','Asparagine synthetase (ADP-forming)','NULL','NULL','ATP + L-aspartate + NH(3) = ADP + phosphate + L-asparagine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.5','NAD(+) synthase','NAD(+) synthetase','NULL','L-glutamine also acts, more slowly, as amido-donor (cf. EC 6.3.5.1).','ATP + deamido-NAD(+) + NH(3) = AMP + diphosphate + NAD(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.6','Glutamate--ethylamine ligase','N(5)-ethyl-L-glutamine synthetase','NULL','NULL','ATP + L-glutamate + ethylamine = ADP + phosphate + N(5)-ethyl-L-glutamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.7','4-methyleneglutamate--ammonia ligase','4-methyleneglutamine synthetase','NULL','Glutamine can act instead of NH(3), more slowly.','ATP + 4-methylene-L-glutamate + NH(3) = AMP + diphosphate + 4-methylene-L-glutamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.8','Glutathionylspermidine synthase','Gamma-L-glutamyl-L-cysteinyl-glycine:spermidine ligase (ADP-forming)','Magnesium.','Involved in the synthesis of trypanothione in trypanosomatids.The enzyme from Escherichia coli is bifunctional and also catalyzesthe EC 3.5.1.78 reaction, resulting in a net hydrolysis of ATP.','Glutathione + spermidine + ATP = glutathionylspermidine + ADP +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.9','Trypanothione synthase','TSR synthetase','Magnesium.','Involved in the synthesis of trypanothione in trypanosomatids.','Glutathione + glutathionylspermidine + ATP = N(1),N(8)-bis-(glutathionyl)spermidine + ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.10','Adenosylcobinamide-phosphate synthase','NULL','NULL','This enzyme forms part of the anaerobic cobalamin biosynthesispathway.One of the substrates for this reaction, (R)-1-aminopropan-2-ylphosphate, is produced by CobD (EC 4.1.1.81).','(1) ATP + adenosylcobyric acid + (R)-1-aminopropan-2-yl phosphate = ADP +phosphate + adenosylcobinamide phosphate.(2) ATP + adenosylcobyric acid + (R)-1-aminopropan-2-ol = ADP + phosphate+ adenosylcobinamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.11','Glutamate--putrescine ligase','Gamma-glutamylputrescine synthetase','NULL','Forms part of a bacterial putrescine utilization pathway.','ATP + L-glutamate + putrescine = ADP + phosphate + gamma-L-glutamylputrescine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.12','D-aspartate ligase','Asl(fm)','NULL','Forms part of the peptidoglycan assembly pathway of Gram-positivebacteria grown in medium containing D-Asp.Normally, the side chains the acylate the 6-amino group of theL-lysine residue contain L-Ala-L-Ala but these amino acids arereplaced by D-Asp when D-Asp is included in the medium.Hybrid chains containing L-Ala-D-Asp, L-Ala-L-Ala-D-Asp or D-Asp-L-Ala are not formed.Highly specific for D-aspartate, as L-aspartate, D-glutamate,D-alanine, D-iso-asparagine and D-malic acid are not substrates.In Enterococcus faecium, the substrate D-aspartate is produced byEC 5.1.1.13.','ATP + D-aspartate + (beta-GlcNAc-(1-', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.13','L-cysteine:1D-myo-inositol 2-amino-2-deoxy-alpha-D-glucopyranosideligase','MshC ligase','NULL','This enzyme is a key enzyme in the biosynthesis of mycothiol, a smallmolecular weight thiol found in Mycobacteria spp. and otheractinomycetes.Mycothiol plays a fundamental role in these organisms by helping toprovide protection from the effects of reactive oxygen species andelectrophiles, including many antibiotics.The enzyme may represent a novel target for new classes ofantituberculars.Formerly EC 6.1.1.n1.','1-O-(2-amino-2-deoxy-alpha-D-glucopyranosyl)-1D-myo-inositol + L-cysteine+ ATP = 1-O-(2-(L-cysteinamido)-2-deoxy-alpha-D-glucopyranosyl)-1D-myo-inositol + AMP + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.14','Diphthine--ammonia ligase','Diphthamide synthase','NULL','This amidase catalyzes the last step in the conversion of anL-histidine residue in the translation elongation factor translationelongation factor 2 to diphthamide.This factor is found in all archaebacteria and eukaryotes, but not ineubacteria, and is the target of bacterial toxins such as thediphtheria toxin and the Pseudomonas exotoxin A (see EC 2.4.2.36).The substrate of the enzyme, diphthine, is produced by EC 2.1.1.98.Formerly EC 6.3.2.22.','ATP + diphthine-[translation elongation factor 2] + NH(3) = AMP +diphosphate + diphthamide-[translation elongation factor 2].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.15','8-demethylnovobiocic acid synthase','8-demethylnovobiocic acid synthetase','NULL','The enzyme is involved in the biosynthesis of the aminocoumarinantibiotic novobiocin.','ATP + 3-dimethylallyl-4-hydroxybenzoate + 3-amino-4,7-dihydroxycoumarin =AMP + diphosphate + 8-demethylnovobiocic acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.16','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.1.17','Beta-citrylglutamate synthase','NAAG synthetase I','NULL','The enzyme, found in animals, also has the activity of EC 6.3.2.41.','ATP + citrate + L-glutamate = ADP + phosphate + beta-citryl-L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.1','Pantoate--beta-alanine ligase','Pantoate-activating enzyme','NULL','NULL','ATP + (R)-pantoate + beta-alanine = AMP + diphosphate + (R)-pantothenate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.2','Glutamate--cysteine ligase','Gamma-glutamyl-L-cysteine synthetase','NULL','Can use L-aminohexanoate in place of glutamate.','ATP + L-glutamate + L-cysteine = ADP + phosphate + gamma-L-glutamyl-L-cysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.3','Glutathione synthase','Glutathione synthetase','NULL','NULL','ATP + gamma-L-glutamyl-L-cysteine + glycine = ADP + phosphate +glutathione.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.4','D-alanine--D-alanine ligase','Alanine:alanine ligase (ADP-forming)','NULL','Involved with EC 6.3.2.7 or EC 6.3.2.13, EC 6.3.2.8, EC 6.3.2.9 andEC 6.3.2.10 in the synthesis of a cell-wall peptide.','ATP + 2 D-alanine = ADP + phosphate + D-alanyl-D-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.5','Phosphopantothenate--cysteine ligase','Phosphopantothenoylcysteine synthetase','NULL','Cysteine can be replaced by some of its derivatives.','CTP + (R)-4-phosphopantothenate + L-cysteine = CMP + diphosphate +N-((R)-4-phosphopantothenoyl)-L-cysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.6','Phosphoribosylaminoimidazolesuccinocarboxamide synthase','4-((N-succinylamino)carbonyl)-5-aminoimidazole ribonucleotide synthetase','NULL','Forms part of the purine biosynthesis pathway.','ATP + 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate +L-aspartate = ADP + phosphate + (S)-2-(5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamido)succinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.7','UDP-N-acetylmuramoyl-L-alanyl-D-glutamate--L-lysine ligase','L-lysine-adding enzyme','NULL','Involved with EC 6.3.2.4, EC 6.3.2.8, EC 6.3.2.9 and EC 6.3.2.10 inthe synthesis of a cell-wall peptide.This enzyme adds lysine in some Gram-positive organisms; in othersand in Gram-negative organisms EC 6.3.2.13 adds 2,6-diaminopimelateinstead.','ATP + UDP-N-acetylmuramoyl-L-alanyl-D-glutamate + L-lysine = ADP +phosphate + UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.8','UDP-N-acetylmuramate--L-alanine ligase','Alanine-adding enzyme','NULL','Involved with EC 6.3.2.4, EC 6.3.2.7 or EC 6.3.2.13, EC 6.3.2.9 andEC 6.3.2.10 in the synthesis of a cell-wall peptide.','ATP + UDP-N-acetylmuramate + L-alanine = ADP + phosphate + UDP-N-acetylmuramoyl-L-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.9','UDP-N-acetylmuramoyl-L-alanine--D-glutamate ligase','D-glutamate ligase','NULL','Involved with EC 6.3.2.4, EC 6.3.2.7 or EC 6.3.2.13, EC 6.3.2.8 andEC 6.3.2.10 in the synthesis of a cell-wall peptide.','ATP + UDP-N-acetylmuramoyl-L-alanine + glutamate = ADP + phosphate + UDP-N-acetylmuramoyl-L-alanyl-D-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.10','UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase','MurF synthetase','NULL','Involved with EC 6.3.2.4, EC 6.3.2.7 or EC 6.3.2.13, EC 6.3.2.8 andEC 6.3.2.9 in the synthesis of a cell-wall peptide.Also catalyzes the reaction when the C-terminal residue of thetripeptide is meso-2,4-diaminoheptanedioate (acylated at itsL-center), linking the D-Ala--D-Ala to the carboxy group of theL-center.Formerly EC 6.3.2.15.','ATP + UDP-N-acetylmuramoyl-L-alanyl-gamma-D-glutamyl-L-lysine + D-alanyl-D-alanine = ADP + phosphate + UDP-N-acetylmuramoyl-L-alanyl-gamma-D-glutamyl-L-lysyl-D-alanyl-D-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.11','Carnosine synthase','Carnosine synthetase','NULL','This enzyme was thought to form AMP, but studies with highly purifiedenzyme proved that it forms ADP.Carnosine is a dipeptide that is present at high concentrations inskeletal muscle and the olfactory bulb of vertebrates.It is also found in the skeletal muscle of some invertebrates.The enzyme can also catalyze the formation of homocarnosine from4-aminobutyrate and L-histidine, with much lower activity.','ATP + L-histidine + beta-alanine = ADP + phosphate + carnosine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.12','Dihydrofolate synthase','7,8-dihydrofolate synthetase','NULL','In some bacteria, a single protein catalyzes both this activity andthat of EC 6.3.2.17, the combined activity of which leads to theformation of the coenzyme polyglutamated tetrahydropteroate(H(4)PteGlu(n)), i.e. various tetrahydrofolates.In contrast, the activities are located on separate proteins in mosteukaryotes studied to date.This enzyme is reponsible for attaching the first glutamate residueto dihydropteroate to form dihydrofolate and is present only in thoseorganisms that have the ability to synthesize tetrahydrofolate denovo, e.g. plants, most bacteria, fungi and protozoa.','ATP + 7,8-dihydropteroate + L-glutamate = ADP + phosphate + 7,8-dihydropteroylglutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.13','UDP-N-acetylmuramoyl-L-alanyl-D-glutamate--2,6-diaminopimelate ligase','MurE synthetase','NULL','Involved with EC 6.3.2.4, EC 6.3.2.8, EC 6.3.2.9 and EC 6.3.2.10 inthe synthesis of a cell-wall peptide.This enzyme adds diaminopimelate in Gram-negative organisms and insome Gram-positive organisms; in others EC 6.3.2.7 adds lysineinstead.It is the amino group of the L-center of the diaminopimelate that isacylated.','ATP + UDP-N-acetylmuramoyl-L-alanyl-D-glutamate + meso-2,6-diaminoheptanedioate = ADP + phosphate + UDP-N-acetylmuramoyl-L-alanyl-D-gamma-glutamyl-meso-2,6-diamino-heptanedioate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.14','Enterobactin synthase','2,3-dihydroxybenzoate--serine ligase','NULL','This enzyme complex catalyzes the conversion of three molecules eachof 2,3-dihydroxybenzoate and L-serine to form the siderophoreenterobactin.In Escherichia coli the complex is formed by EntB (an aryl carrierprotein that has to be activated by 4-phosphopantetheine), EntD (aphosphopantetheinyl transferase that activates EntB), EntE (catalyzesthe ATP-dependent condensation of 2,3-dihydroxybenzoate and holo-EntBto form the covalently arylated form of EntB), and EntF (a fourdomain protein that catalyzes the activation of L-serine by ATP,the condensation of the activated L-serine with the activated 2,3-dihydroxybenzoate, and the trimerization of three such moieties to asingle enterobactin molecule).','6 ATP + 3 2,3-dihydroxybenzoate + 3 L-serine = enterobactin + 6 AMP +6 diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.15','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.16','D-alanine--alanyl-poly(glycerolphosphate) ligase','D-alanine:membrane-acceptor ligase','NULL','Involved in the synthesis of teichoic acids.','ATP + D-alanine + alanyl-poly(glycerolphosphate) = ADP + phosphate +D-alanyl-alanyl-poly(glycerolphosphate).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.17','Tetrahydrofolate synthase','Folate polyglutamate synthetase','NULL','In some bacteria, a single protein catalyzes both this activity andthat of EC 6.3.2.12, the combined activity of which leads to theformation of the coenzyme polyglutamated tetrahydropteroate(H(4)PteGlu(n)), i.e. various tetrahydrofolates (H(4)folate).In contrast, the activities are located on separate proteins in mosteukaryotes studied to date.In Arabidopsis thaliana, this enzyme is present as distinct isoformsin the mitochondria, the cytosol and the chloroplast.Each isoform is encoded by a separate gene, a situation that isunique among eukaryotes.As the affinity of folate-dependent enzymes increases markedly withthe number of glutamic residues, the tetrahydropteroyl polyglutamatesare the preferred coenzymes of C(1) metabolism.The enzymes from different sources (particularly eukaryotes versusprokaryotes) have different substrate specificities with regard toone-carbon substituents and the number of glutamate residues presenton the tetrahydrofolates.','ATP + tetrahydropteroyl-(gamma-Glu)(n) + L-glutamate = ADP + phosphate +tetrahydropteroyl-(gamma-Glu)(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.18','Gamma-glutamylhistamine synthase','Gamma-GHA synthetase','NULL','NULL','ATP + L-glutamate + histamine = products of ATP breakdown + N(alpha)-gamma-L-glutamylhistamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.19','Ubiquitin--protein ligase','Ubiquitin-activating enzyme','NULL','Ubiquitin is coupled to protein by a peptide bond between theC-terminal glycine of ubiquitin and epsilon-amino groups of lysineresidues in the protein.An intermediate in the reaction contains one ubiquitin residue boundas a thioester to the enzyme, and a residue of ubiquitin adenylatenon-covalently bound to the enzyme.','ATP + ubiquitin + protein lysine = AMP + diphosphate + proteinN-ubiquityllysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.20','Indoleacetate--lysine synthetase','IAA-lysine synthetase','NULL','NULL','ATP + (indol-3-yl)acetate + L-lysine = ADP + phosphate + N(6)-((indol-3-yl)acetyl)-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.21','Ubiquitin--calmodulin ligase','Ubiquityl-calmodulin synthase','NULL','Specific for Ca(2+)-calmodulin from vertebrates.At least 3 ubiquitin molecules can be coupled to lysine residues incalmodulin.','n ATP + calmodulin + n ubiquitin = n AMP + n diphosphate +(ubiquitin)(n)-calmodulin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.22','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.23','Homoglutathione synthase','Beta-alanine specific hGSH synthetase','NULL','Not identical with EC 6.3.2.3.','ATP + gamma-L-glutamyl-L-cysteine + beta-alanine = ADP + phosphate +gamma-L-glutamyl-L-cysteinyl-beta-alanine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.24','Tyrosine--arginine ligase','Kyotorphin synthase','NULL','NULL','ATP + L-tyrosine + L-arginine = AMP + diphosphate + L-tyrosyl-L-arginine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.25','Tubulin--tyrosine ligase','TTL','NULL','L-tyrosine is linked via a peptide bond to the C-terminus ofde-tyrosinated alpha-tubulin (des-Tyr(omega)-alpha-tubulin).The enzyme is highly specific for alpha-tubulin and moderatelyspecific for ATP and L-tyrosine.L-phenylalanine and 3,4-dihydroxy-L-phenylalanine are transferred butwith higher Km values.','ATP + detyrosinated alpha-tubulin + L-tyrosine = alpha-tubulin + ADP +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.26','N-(5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine synthase','ACV synthetase','Magnesium.','The enzyme contains 4-phosphopantetheine, which may be involved inthe mechanism of the reaction.Forms part of the penicillin biosynthesis pathway.','L-2-aminohexanedioate + L-cysteine + L-valine + 3 ATP + H(2)O = N-(L-5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine + 3 AMP + 3 diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.27','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.28','L-amino-acid alpha-ligase','Bacilysin synthetase','Manganese or magnesium.','While the enzyme has extremely broad substrate specificity, it doesnot accept highly charged amino acids, such as Lys, Arg, Glu and Asp,nor does it react with secondary amines such as Pro.The N-terminal residue of the alpha-dipeptide formed seems to belimited to Ala, Gly, Ser, Thr and Met (with Ala and Ser being themost preferred), whereas the C-terminal residue seems to allow for awider variety of amino acids (but with a preference for Met and Phe).However, not all combinations or dipeptides are formed; for example,while Ser is acceptable for the N-terminus and Thr for theC-terminus, a Ser-Thr dipeptide is not formed.D-Ala, D-Ser and D-Phe are not substrates.','ATP + an L-amino acid + an L-amino acid = ADP + phosphate + L-aminoacyl-L-amino acid.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.29','Cyanophycin synthase (L-aspartate-adding)','Cyanophycin synthetase','Magnesium.','Both this enzyme and EC 6.3.2.30 are required for the elongation ofcyanophycin, which is a protein-like cell inclusion that is unique tocyanobacteria and acts as a temporary nitrogen store.Both enzymes are found in the same protein but have different activesites.Both L-Asp and L-Arg must be present before either enzyme willdisplay significant activity.','ATP + (L-Asp(4-L-Arg))(n) + L-Asp = ADP + phosphate + (L-Asp(4-L-Arg))(n)-L-Asp.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.30','Cyanophycin synthase (L-arginine-adding)','Cyanophycin synthetase','Magnesium.','Both this enzyme and EC 6.3.2.29 are required for the elongation ofcyanophycin, which is a protein-like cell inclusion that is unique tocyanobacteria and acts as a temporary nitrogen store.Both enzymes are found in the same protein but have different activesites.Both L-Asp and L-Arg must be present before either enzyme willdisplay significant activity.Canavanine and lysine can be incoporated into the polymer instead ofarginine.','ATP + (L-Asp(4-L-Arg))(n)-L-Asp + L-Arg = ADP + phosphate + (L-Asp(4-L-Arg))(n+1).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.31','Coenzyme F420-0:L-glutamate ligase','NULL','NULL','This protein catalyzes the successive addition of two glutamateresidues to cofactor F420 by two distinct and independent reactions.In the reaction described here the enzyme attaches a glutamate viaits alpha-amine group to F420-0.In the second reaction (EC 6.3.2.34) it catalyzes the addition of asecond L-glutamate residue to the gamma-carboxyl of the firstglutamate.','GTP + coenzyme F420-0 + L-glutamate = GDP + phosphate + coenzyme F420-1.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.32','Coenzyme gamma-F420-2:alpha-L-glutamate ligase','NULL','NULL','The enzyme caps the gamma-glutamyl tail of the hydride carriercoenzyme F420.','ATP + coenzyme gamma-F420-2 + L-glutamate = ADP + phosphate + coenzymealpha-F420-3.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.33','Tetrahydrosarcinapterin synthase','NULL','NULL','This enzyme catalyzes the biosynthesis of 5,6,7,8-tetrahydrosarcinapterin, a modified form of tetrahydromethanopterinfound in the Methanosarcinales.It does not require K(+), and does not discriminate between ATP andGTP.','ATP + tetrahydromethanopterin + L-glutamate = ADP + phosphate + 5,6,7,8-tetrahydrosarcinapterin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.34','Coenzyme F420-1:gamma-L-glutamate ligase','NULL','NULL','This protein catalyzes the successive addition of two glutamateresidues to cofactor F420 by two distinct and independent reactions.In the first reaction (EC 6.3.2.31) the enzyme attaches a glutamatevia its alpha-amine group to F420-0.In the second reaction, which is described here, the enzyme catalyzesthe addition of a second L-glutamate residue to the gamma-carboxyl ofthe first glutamate.','GTP + coenzyme F420-1 + L-glutamate = GDP + phosphate + coenzyme gamma-F420-2.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.35','D-alanine--D-serine ligase','NULL','NULL','The product of this enzyme, D-alanyl-D-serine, can be incorporatedinto the peptidoglycan pentapeptide instead of the usual D-alanyl-D-alanine dipeptide, which is formed by EC 6.3.2.4, D-alanine--D-alanine ligase.The resulting peptidoglycan does not bind the glycopeptideantibiotics vancomycin and teicoplanin, conferring resistance on thebacteria.','D-alanine + D-serine + ATP = D-alanyl-D-serine + ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.36','4-phosphopantoate--beta-alanine ligase','Phosphopantothenate synthetase','NULL','The conversion of (R)-pantoate to (R)-4-phosphopantothenate is partof the pathway leading to biosynthesis of 4-phosphopantetheine,an essential cofactor of coenzyme A and acyl-carrier protein.In bacteria and eukaryotes this conversion is performed bycondensation with beta-alanine, followed by phosphorylation(EC 6.3.2.1 and EC 2.7.1.33, respectively).In archaea the order of these two steps is reversed,and phosphorylation precedes condensation with beta-alanine.The two archaeal enzymes that catalyze this conversion areEC 2.7.1.169 and this enzyme.','ATP + (R)-4-phosphopantoate + beta-alanine = AMP + diphosphate + (R)-4-phosphopantothenate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.37','UDP-N-acetylmuramoyl-L-alanyl-D-glutamate--D-lysine ligase','D-lysine-adding enzyme','NULL','The enzyme from Thermotoga maritima also performs the reaction ofEC 6.3.2.7.Involved in the synthesis of cell-wall peptidoglycan.Formerly EC 6.3.2.n1.','ATP + UDP-N-acetylmuramoyl-L-alanyl-D-glutamate + D-lysine = ADP +phosphate + UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-D-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.38','N(2)-citryl-N(6)-acetyl-N(6)-hydroxylysine synthase','N(alpha)-citryl-N(epsilon)-acetyl-N(epsilon)-hydroxylysine synthase','Magnesium.','Aerobactin is one of a group of high-affinity iron chelators known assiderophores and is produced under conditions of iron deprivation.It is a dihydroxamate comprising two molecules of N(6)-acetyl-N(6)-hydroxy-L-lysine and one molecule of citrate.This enzyme catalyzes the first of two synthase reactions to linkN(6)-acetyl-N(6)-hydroxy-L-lysine and citrate.Formerly EC 6.3.2.27.','2 ATP + citrate + N(6)-acetyl-N(6)-hydroxy-L-lysine + H(2)O = 2 ADP +2 phosphate + N(2)-citryl-N(6)-acetyl-N(6)-hydroxy-L-lysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.39','Aerobactin synthase','NULL','Magnesium.','Aerobactin is one of a group of high-affinity iron chelators known assiderophores and is produced under conditions of iron deprivation.It is a dihydroxamate comprising two molecules of N(6)-acetyl-N(6)-hydroxy-L-lysine and one molecule of citric acid.This enzyme catalyzes the second of two synthase reactions to linkN(6)-acetyl-N(6)-hydroxy-L-lysine and citrate.Formerly EC 6.3.2.27.','2 ATP + N(2)-citryl-N(6)-acetyl-N(6)-hydroxy-L-lysine + N(6)-acetyl-N(6)-hydroxy-L-lysine + H(2)O = 2 ADP + 2 phosphate + aerobactin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.40','Cyclopeptine synthase','NULL','NULL','Cyclopeptine synthase is the key enzyme of benzodiazepine alkaloidbiosynthesis in the fungus Penicillium cyclopium.The enzyme is a non-ribosomal peptide synthase.Formerly EC 2.7.7.54 and EC 2.7.7.55.','2 ATP + S-adenosyl-L-methionine + anthranilate + L-phenylalanine =cyclopeptine + 2 AMP + 2 diphosphate + S-adenosyl-L-homocysteine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.41','N-acetylaspartylglutamate synthase','N-acetylaspartylglutamate synthetase','NULL','The enzyme, found in animals, produces the neurotransmitter N-acetyl-L-aspartyl-L-glutamate.One isoform also has the activity of EC 6.3.1.17, while anotherisoform has the activity of EC 6.3.2.42.Formerly EC 6.3.2.n6.','ATP + N-acetyl-L-aspartate + L-glutamate = ADP + phosphate + N-acetyl-L-aspartyl-L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.42','N-acetylaspartylglutamylglutamate synthase','N-acetylaspartylglutamylglutamate synthetase','NULL','The enzyme, found in mammals, also has the activity of EC 6.3.2.41.','2 ATP + N-acetyl-L-aspartate + 2 L-glutamate = 2 ADP + 2 phosphate +N-acetyl-L-aspartyl-L-glutamyl-L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.n1','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.n2','Pup--protein ligase','Pup-conjugating enzyme','NULL','The enzyme catalyzes the covalent attachment of the prokaryoticubiquitin-like protein modifier Pup to the proteasomal substrateproteins, thereby targeting them for proteasomal degradation.This tagging system is termed pupylation.The ligation reaction involves the side-chain carboxylate of theC-terminal glutamate of Pup and the side-chain amino group of asubstrate lysine.PafA is required to confer resistance against the lethal effects ofreactive nitrogen intermediates (RNI), antimicrobial moleculesproduced by activated macrophages and other cell types.','ATP + [Pup] + [protein]-lysine = ADP + phosphate + [protein]-N-pupyllysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.n3','ISG15--protein ligase','ISG15-conjugating enzyme','NULL','The enzyme from human also functions as ubiquitin--protein ligase(EC 6.3.2.19).','ATP + [ISG15] + [protein]-lysine = AMP + diphosphate + [protein]-N-ISGyllysine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.n4','Alpha-aminoadipate--LysW ligase','AAA--LysW ligase','NULL','The enzyme ligates the L-2-aminoadipate to the C-terminal glutamateresidue of LysW.This is the first step in the AAA pathway of L-lysine biosynthesis.','ATP + [LysW] + L-2-aminoadipate = ADP + phosphate + [LysW]-L-2-aminoadipate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.n5','Pantoate--beta-alanine ligase (ADP-forming)','Pantothenate synthetase','NULL','The enzyme is found in Archaea.','ATP + (R)-pantoate + beta-alanine = ADP + phosphate + (R)-pantothenate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.2.n6','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.3.1','Phosphoribosylformylglycinamidine cyclo-ligase','AIR synthase','NULL','NULL','ATP + 2-(formamido)-N(1)-(5-phospho-D-ribosyl)acetamidine = ADP +phosphate + 5-amino-1-(5-phospho-D-ribosyl)imidazole.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.3.2','5-formyltetrahydrofolate cyclo-ligase','5,10-methenyltetrahydrofolate synthetase','NULL','NULL','ATP + 5-formyltetrahydrofolate = ADP + phosphate + 5,10-methenyltetrahydrofolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.3.3','Dethiobiotin synthase','DTB synthetase','NULL','CTP has half the activity of ATP.','ATP + 7,8-diaminononanoate + CO(2) = ADP + phosphate + dethiobiotin.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.3.4','(Carboxyethyl)arginine beta-lactam-synthase','Beta-lactam synthetase','NULL','Forms part of the pathway for the biosythesis of the beta-lactamaseinhibitor clavulanate in Streptomyces clavuligerus.It has been proposed that L-N(2)-(2-carboxyethyl)arginine is firstconverted into an acyl-AMP by reaction with ATP and loss ofdiphosphate, and that the beta-lactam ring is then formed by theintramolecular attack of the beta-nitrogen on the activated carboxygroup.','ATP + L-N(2)-(2-carboxyethyl)arginine = AMP + diphosphate +deoxyamidinoproclavaminate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.3.5','O-ureido-D-serine cyclo-ligase','NULL','NULL','The enzyme participates in the biosynthetic pathway of D-cycloserine,an antibiotic substance produced by several Streptomyces species.','O-ureido-D-serine + ATP + H(2)O = D-cycloserine + CO(2) + ammonia + ADP +phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.3.6','Carbapenam-3-carboxylate synthase','Carbapenam-3-carboxylate ligase','NULL','The enzyme is involved in the biosynthesis of the carbapenem beta-lactam antibiotic (5R)-carbapen-2-em-3-carboxylate in the bacteriumPectobacterium carotovorum.Formerly EC 6.3.1.16.','ATP + (2S,5S)-5-carboxymethylproline = AMP + diphosphate + (3S,5S)-carbapenam 3-carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.1','Transferred entry: 6','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.2','CTP synthase (glutamine hydrolyzing)','CTP synthetase','NULL','The enzyme contains three functionally distinct sites: an allostericGTP-binding site, a glutaminase site where glutamine hydrolysisoccurs (cf. EC 3.5.1.2), and the active site where CTP synthesistakes place.The reaction proceeds via phosphorylation of UTP by ATP to give anactivated intermediate 4-phosphoryl UTP and ADP.Ammonia then reacts with this intermediate generating CTP and aphosphate.The enzyme can also use ammonia from the surrounding solution.','ATP + UTP + L-glutamine = ADP + phosphate + CTP + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.3','Formate--tetrahydrofolate ligase','Formyltetrahydrofolate synthetase','NULL','In eukaryotes occurs as a trifunctional enzyme also having EC 1.5.1.5and EC 3.5.4.9 activity.','ATP + formate + tetrahydrofolate = ADP + phosphate +10-formyltetrahydrofolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.4','Adenylosuccinate synthase','Adenylosuccinate synthetase','NULL','NULL','GTP + IMP + L-aspartate = GDP + phosphate + N(6)-(1,2-dicarboxyethyl)-AMP.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.5','Argininosuccinate synthase','Arginine succinate synthetase','NULL','NULL','ATP + L-citrulline + L-aspartate = AMP + diphosphate + N(omega)-(L-arginino)succinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.6','Urea carboxylase','ATP--urea amidolyase','Biotin.','The Saccharomyces cerevisiae enzyme (but not that from green algae)also catalyzes the reaction of EC 3.5.1.54, thus bringing about thehydrolysis of urea to CO(2) and NH(3).The enzyme from the prokaryotic bacterium Oleomonas sagaranensis canalso use acetamide and formamide as substrates.Formerly EC 3.5.1.45.','ATP + urea + HCO(3)(-) = ADP + phosphate + urea-1-carboxylate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.7','Ribose-5-phosphate--ammonia ligase','5-phosphoribosylamine synthetase','NULL','NULL','ATP + ribose 5-phosphate + NH(3) = ADP + phosphate +5-phosphoribosylamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.8','Imidazoleacetate--phosphoribosyldiphosphate ligase','5-phosphoribosylimidazoleacetate synthetase','NULL','NULL','ATP + imidazole-4-acetate + 5-phosphoribosyl diphosphate = ADP +phosphate + 1-(5-phosphoribosyl)imidazole-4-acetate + diphosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.9','Biotin--[methylmalonyl-CoA-carboxytransferase] ligase','Biotin--[methylmalonyl-CoA-carboxyltransferase] ligase','NULL','NULL','ATP + biotin + apo-[methylmalonyl-CoA:pyruvate carboxytransferase] =AMP + diphosphate + [methylmalonyl-CoA:pyruvate carboxytransferase].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.10','Biotin--[propionyl-CoA-carboxylase (ATP-hydrolyzing)] ligase','Biotin-[propionyl-CoA-carboxylase (ATP-hydrolyzing)] synthetase','NULL','NULL','ATP + biotin + apo-[propionyl-CoA:carbon-dioxide ligase (ADP-forming)] =AMP + diphosphate + [propionyl-CoA:carbon-dioxide ligase (ADP-forming)].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.11','Biotin--[methylcrotonoyl-CoA-carboxylase] ligase','Biotin--[methylcrotonoyl-CoA-carboxylase] synthetase','NULL','NULL','ATP + biotin + apo-[3-methylcrotonoyl-CoA:carbon-dioxide ligase (ADP-forming)] = AMP + diphosphate + [3-methylcrotonoyl-CoA:carbon-dioxideligase (ADP-forming)].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.12','Glutamate--methylamine ligase','Gamma-glutamylmethylamide synthetase','NULL','NULL','ATP + L-glutamate + methylamine = ADP + phosphate + N(5)-methyl-L-glutamine.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.13','Phosphoribosylamine--glycine ligase','GAR synthetase','NULL','Formerly EC 6.3.1.3.','ATP + 5-phospho-D-ribosylamine + glycine = ADP + phosphate + N(1)-(5-phospho-D-ribosyl)glycinamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.14','Biotin carboxylase','NULL','NULL','NULL','ATP + biotin-[carboxyl-carrier-protein] + CO(2) = ADP + phosphate +carboxy-biotin-[carboxyl-carrier-protein].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.15','Biotin--[acetyl-CoA-carboxylase] ligase','Acetyl-CoA carboxylase biotin holoenzyme synthetase','NULL','NULL','ATP + biotin + apo-[acetyl-CoA:carbon-dioxide ligase (ADP-forming)] =AMP + diphosphate + [acetyl-CoA:carbon-dioxide ligase (ADP-forming)].', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.16','Carbamoyl-phosphate synthase (ammonia)','Carbamoyl-phosphate synthetase (ammonia)','NULL','Formerly EC 2.7.2.5.','2 ATP + NH(3) + CO(2) + H(2)O = 2 ADP + phosphate + carbamoyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.17','Formate--dihydrofolate ligase','Dihydrofolate formyltransferase','NULL','NULL','ATP + formate + dihydrofolate = ADP + phosphate + 10-formyldihydrofolate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.18','5-(carboxyamino)imidazole ribonucleotide synthase','N(5)-CAIR synthetase','NULL','In Escherichia coli, this enzyme, along with EC 5.4.99.18,is required to carry out the single reaction catalyzed by EC 4.1.1.21in vertebrates.Carboxyphosphate is the putative acyl phosphate intermediate.Involved in the late stages of purine biosynthesis.','ATP + 5-amino-1-(5-phospho-D-ribosyl)imidazole + HCO(3)(-) = ADP +phosphate + 5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.19','tRNA(Ile)-lysidine synthetase','Isoleucine-specific transfer ribonucleate lysidine synthetase','NULL','The bacterial enzyme modifies the wobble base of the CAU anticodon oftRNA(Ile) at the oxo group in position 2 of cytidine(34).This modification determines both codon and amino acid specificitiesof tRNA(Ile).','(tRNA(Ile2))-cytidine(34) + L-lysine + ATP = (tRNA(Ile2))-lysidine(34) +AMP + diphosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.20','7-cyano-7-deazaguanine synthase','7-cyano-7-carbaguanine synthase','NULL','The reaction is part of the biosynthesis pathway of queuosine.','7-carboxy-7-carbaguanine + NH(3) + ATP = 7-cyano-7-carbaguanine + ADP +phosphate + H(2)O.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.21','Nicotinate phosphoribosyltransferase','Niacin ribonucleotidase','NULL','The enzyme, which is involved in pyridine nucleotide recycling,can form beta-nicotinate D-ribonucleotide and diphosphate fromnicotinate and 5-phospho-alpha-D-ribose 1-diphosphate (PRPP) in theabsence of ATP.However, when ATP is available the enzyme is phosphorylated resultingin a much lower K(m) for nicotinate.The phospho-enzyme is hydrolyzed during the transferase reaction,regenerating the low affinity form.The presence of ATP shifts the products/substrates equilibrium from0.67 to 1100.Formerly EC 2.4.2.11.','Nicotinate + 5-phospho-alpha-D-ribose 1-diphosphate + ATP + H(2)O = beta-nicotinate D-ribonucleotide + diphosphate + ADP + phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.22','tRNA(Ile)(2)-agmatinylcytidine synthase','tRNA(Ile)-2-agmatinylcytidine synthetase','NULL','The enzyme from the archaeon Archaeoglobus fulgidus modifies thewobble base of the CAU anticodon of the archaeal tRNA(Ile)(2) at theoxo group in position 2 of cytidine(34).This modification is crucial for accurate decoding of the geneticcode.In bacteria EC 6.3.4.19, catalyzes the modification of(tRNA(Ile)(2))-cytidine(34) to (tRNA(Ile)(2))-lysidine(34).','ATP + agmatine + (tRNA(Ile)(2))-cytidine(34) + H(2)O = (tRNA(Ile)(2))-2-agmatinylcytidine(34) + AMP + 2 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.4.23','Formate--phosphoribosylaminoimidazolecarboxamide ligase','5-formaminoimidazole-4-carboxamide ribonucleotide synthetase','NULL','This archaeal enzyme, characterized from the methanogenMethanocaldococcus jannaschii, catalyzes a step in the synthesis ofpurine nucleotides.It differs from the orthologous bacterial/eukaryotic enzymes, whichutilize 10-formyltetrahydrofolate rather than formate and ATP.Cf. EC 2.1.2.3.','ATP + formate + 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide =ADP + phosphate + 5-formamido-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.1','NAD(+) synthase (glutamine-hydrolyzing)','NAD(+) synthetase (glutamine-hydrolyzing)','NULL','NH(3) can act instead of glutamine (cf. EC 6.3.1.5).','ATP + deamido-NAD(+) + L-glutamine + H(2)O = AMP + diphosphate + NAD(+) +L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.2','GMP synthase (glutamine-hydrolyzing)','GMP synthetase (glutamine-hydrolyzing)','NULL','Involved in the de novo biosynthesis of guanosine nucleotides.An N-terminal glutaminase domain binds L-glutamine and generatesammonia, which is transferred by a substrate-protective tunnel to theATP-pyrophosphatase domain.The enzyme can catalyze the second reaction alone in the presence ofammonia.Formerly EC 6.3.4.1.','ATP + XMP + L-glutamine + H(2)O = AMP + diphosphate + GMP + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.3','Phosphoribosylformylglycinamidine synthase','FGAM synthase','NULL','NULL','ATP + N(2)-formyl-N(1)-(5-phospho-D-ribosyl)glycinamide + L-glutamine +H(2)O = ADP + phosphate + 2-(formamido)-N(1)-(5-phospho-D-ribosyl)acetamidine + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.4','Asparagine synthase (glutamine-hydrolyzing)','AS-B','NULL','The enzyme from Escherichia coli has two active sites that areconnected by an intramolecular ammonia tunnel.The enzyme catalyzes three distinct chemical reactions: glutaminehydrolysis to yield ammonia takes place in the N-terminal domain.The C-terminal active site mediates both the synthesis of a beta-aspartyl-AMP intermediate and its subsequent reaction with ammonia.The ammonia released is channeled to the other active site to yieldasparagine.','ATP + L-aspartate + L-glutamine + H(2)O = AMP + diphosphate +L-asparagine + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.5','Carbamoyl-phosphate synthase (glutamine-hydrolyzing)','Carbamoyl phosphate synthetase','NULL','The product carbamoyl phosphate is an intermediate in thebiosynthesis of arginine and the pyrimidine nucleotides.The enzyme from Escherichia coli has three separate active sites,which are connected by a molecular tunnel that is almost 100 A inlength.The amidotransferase domain within the small subunit of the enzymehydrolyzes glutamine to ammonia via a thioester intermediate.The ammonia migrates through the interior of the protein, where itreacts with carboxy phosphate to produce the carbamate intermediate.The carboxy-phosphate intermediate is formed by the phosphorylationof bicarbonate by ATP at a site contained within the N-terminal halfof the large subunit.The carbamate intermediate is transported through the interior of theprotein to a second site within the C-terminal half of the largesubunit, where it is phosphorylated by another ATP to yield the finalproduct, carbamoyl phosphate.Formerly EC 2.7.2.9.','2 ATP + L-glutamine + HCO(3)(-) + H(2)O = 2 ADP + phosphate + L-glutamate+ carbamoyl phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.6','Asparaginyl-tRNA synthase (glutamine-hydrolyzing)','Asp-AdT','NULL','This reaction forms part of a two-reaction system for producingasparaginyl-tRNA in Deinococcus radiodurans and other organismslacking a specific enzyme for asparagine synthesis.In the first step, a non-discriminating ligase (EC 6.1.1.23)mischarges tRNA(Asn) with aspartate, leading to the formation ofaspartyl-tRNA(Asn).The aspartyl-tRNA(Asn) is not used in protein synthesis until thepresent enzyme converts it into asparaginyl-tRNA(Asn) (aspartyl-tRNA(Asp) is not a substrate for this reaction).Ammonia or asparagine can substitute for the preferred substrateglutamine.','ATP + L-aspartyl-tRNA(Asn) + L-glutamine + H(2)O = ADP + phosphate +L-asparaginyl-tRNA(Asn) + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.7','Glutaminyl-tRNA synthase (glutamine-hydrolyzing)','Glu-AdT','NULL','In systems lacking discernible EC 6.1.1.18, glutaminyl-tRNA(Gln) isformed by a two-enzyme system.In the first step, a nondiscriminating ligase (EC 6.1.1.24)mischarges tRNA(Gln) with glutamate, forming glutamyl-tRNA(Gln).The glutamyl-tRNA(Gln) is not used in protein synthesis until thepresent enzyme converts it into glutaminyl-tRNA(Gln) (glutamyl-tRNA(Glu) is not a substrate for this reaction).Ammonia or asparagine can substitute for the preferred substrateglutamine.','ATP + L-glutamyl-tRNA(Gln) + L-glutamine = ADP + phosphate +L-glutaminyl-tRNA(Gln) + L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.8','Transferred entry: 2','NULL','NULL','NULL','NULL', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.9','Hydrogenobyrinic acid a,c-diamide synthase (glutamine-hydrolyzing)','NULL','NULL','This step in the aerobic biosynthesis of cobalamin generateshydrogenobyrinic acid a,c-diamide, the substrate required byEC 6.6.1.2, which adds cobalt to the macrocycle.','2 ATP + hydrogenobyrinic acid + 2 L-glutamine + 2 H(2)O = 2 ADP +2 phosphate + hydrogenobyrinic acid a,c-diamide + 2 L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.10','Adenosylcobyric acid synthase (glutamine-hydrolyzing)','5-deoxy-5-adenosylcobyrinic-acid-a,c-diamide:L-glutamine amido-ligase','Magnesium.','NH(3) can act instead of glutamine.This enzyme catalyzes the four-step amidation sequence from cobyrinicacid a,c-diamide to cobyric acid via the formation of cobyrinic acidtriamide, tetraamide and pentaamide intermediates.','4 ATP + adenosylcobyrinic acid a,c-diamide + 4 L-glutamine + 4 H(2)O =4 ADP + 4 phosphate + adenosylcobyric acid + 4 L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.3.5.11','Cobyrinate a,c-diamide synthase (glutamine-hydrolyzing)','Cobyrinic acid a,c-diamide synthetase','NULL','This enzyme is the first glutamine amidotransferase that participatesin the anaerobic (early cobalt insertion) biosynthetic pathway ofadenosylcobalamin, and catalyzes the ATP-dependent synthesis ofcobyrinate a,c-diamide from cobyrinate using either L-glutamine orammonia as the nitrogen source.It is proposed that the enzyme first catalyzes the amidation of thec-carboxylate, and then the intermediate is released into solutionand binds to the same catalytic site for the amidation of thea-carboxylate.The Km for ammonia is substantially higher than that for L-glutamine.','2 ATP + cobyrinate + 2 L-glutamine + 2 H(2)O = 2 ADP + 2 phosphate +cobyrinate a,c-diamide + 2 L-glutamate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.4.1.1','Pyruvate carboxylase','Pyruvic carboxylase','Biotin; Zinc or manganese.','The animal enzyme requires acetyl-CoA.','ATP + pyruvate + HCO(3)(-) = ADP + phosphate + oxaloacetate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.4.1.2','Acetyl-CoA carboxylase','NULL','Biotin.','Also catalyzes transcarboxylation; the plant enzyme also carboxylatespropanoyl-CoA and butanoyl-CoA.','ATP + acetyl-CoA + HCO(3)(-) = ADP + phosphate + malonyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.4.1.3','Propionyl-CoA carboxylase','PCCase','Biotin.','Also carboxylates butanoyl-CoA and catalyzes transcarboxylation.','ATP + propanoyl-CoA + HCO(3)(-) = ADP + phosphate + (S)-methylmalonyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.4.1.4','Methylcrotonoyl-CoA carboxylase','Beta-methylcrotonyl-CoA carboxylase','Biotin.','NULL','ATP + 3-methylcrotonoyl-CoA + HCO(3)(-) = ADP + phosphate +3-methylglutaconyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.4.1.5','Geranoyl-CoA carboxylase','Geranyl-CoA carboxylase','Biotin.','Also carboxylates dimethylpropenoyl-CoA and farnesoyl-CoA.','ATP + geranoyl-CoA + HCO(3)(-) = ADP + phosphate + 3-(4-methylpent-3-en-1-yl)pent-2-enedioyl-CoA.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.4.1.6','Acetone carboxylase','NULL','Magnesium.','The enzyme from Xanthobacter sp. strain Py2 also carboxylates butan-2-one to 3-oxopentanoate.','Acetone + CO(2) + ATP + 2 H(2)O = acetoacetate + AMP + 2 phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.4.1.7','2-oxoglutarate carboxylase','Carboxylating factor for ICDH','Biotin; Magnesium.','It was originally thought that this enzyme was a promoting factor forthe carboxylation of 2-oxoglutarate by EC 1.1.1.41, but this hassince been disproved.The product of the reaction is unstable and is quickly converted intoisocitrate by the action of EC 1.1.1.41.','ATP + 2-oxoglutarate + HCO(3)(-) = ADP + phosphate + oxalosuccinate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.4.1.8','Acetophenone carboxylase','NULL','NULL','The enzyme is involved in anaerobic degradation of ethylbenzene.No activity with acetone, butanone, 4-hydroxy-acetophenone or4-amino-acetophenone.','2 ATP + acetophenone + HCO(3)(-) + H(2)O + H(+) = 2 ADP + 2 phosphate +3-oxo-3-phenylpropanoate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.5.1.1','DNA ligase (ATP)','DNA joinase','NULL','Catalyzes the formation of a phosphodiester at the site of a single-strand break in duplex DNA.RNA can also act as substrate, to some extent.','ATP + (deoxyribonucleotide)(n) + (deoxyribonucleotide)(m) = AMP +diphosphate + (deoxyribonucleotide)(n+m).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.5.1.2','DNA ligase (NAD(+))','DNA joinase','NULL','Catalyzes the formation of a phosphodiester at the site of a single-strand break in duplex DNA.RNA can also act as substrate, to some extent.','NAD(+) + (deoxyribonucleotide)(n) + (deoxyribonucleotide)(m) = AMP +beta-nicotinamide D-ribonucleotide + (deoxyribonucleotide)(n+m).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.5.1.3','RNA ligase (ATP)','Polyribonucleotide synthase (ATP)','NULL','Converts linear RNA to a circular form by transfer of the5-phosphate to the 3-hydroxy terminus.','ATP + (ribonucleotide)(n) + (ribonucleotide)(m) = AMP + diphosphate +(ribonucleotide)(n+m).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.5.1.4','RNA 3-terminal-phosphate cyclase (ATP)','RNA cyclase','NULL','Adenosine 5-(gamma-thio)triphosphate can act instead of ATP (cf.EC 6.5.1.5).','ATP + RNA 3-terminal-phosphate = AMP + diphosphate + RNA terminal-2,3-cyclic-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.5.1.5','RNA 3-terminal-phosphate cyclase (GTP)','RNA-3-phosphate cyclase (GTP)','NULL','Cf. EC 6.5.1.4.','GTP + RNA 3-terminal-phosphate = GMP + diphosphate + RNA terminal-2,3-cyclic-phosphate.', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.6.1.1','Magnesium chelatase','Magnesium-chelatase','NULL','This is the first committed step of chlorophyll biosynthesis and is abranchpoint of two major routes in the tetrapyrrole pathway.','ATP + protoporphyrin IX + Mg(2+) + H(2)O = ADP + phosphate +Mg-protoporphyrin IX + 2 H(+).', 'NULL', 'NULL');
INSERT INTO enzyme VALUES ( 'EC 6.6.1.2','Cobaltochelatase','Hydrogenobyrinic acid a,c-diamide cobaltochelatase','NULL','This enzyme, which forms part of the aerobic cobalamin biosynthesispathway, is a type I chelatase, being heterotrimeric and ATP-dependent.It comprises two components, one of which corresponds to CobN and theother is composed of two polypeptides, specified by cobS and cobT inPseudomonas denitrificans, and named CobST.Hydrogenobyrinic acid is a very poor substrate.ATP can be replaced by dATP or CTP but the reaction proceeds moreslowly.CobN exhibits a high affinity for hydrogenobyrinic acid a,c-diamide.The oligomeric protein CobST possesses at least one sulfhydryl groupthat is essential for ATP-binding.Once the Co(2+) is inserted, the next step in the pathway ensuresthat the cobalt is ligated securely by reducing Co(II) to Co(I); thisstep is carried out by EC 1.16.8.1.','ATP + hydrogenobyrinic acid a,c-diamide + Co(2+) + H(2)O = ADP +phosphate + cob(II)yrinic acid a,c-diamide + H(2).', 'NULL', 'NULL');
UPDATE enzyme SET sys_name = 'Alcohol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.1';
UPDATE enzyme SET history = 'EC 1.1.1.1 created 1961' WHERE ec = 'EC 1.1.1.1';
UPDATE enzyme SET sys_name = 'Alcohol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.2';
UPDATE enzyme SET history = 'EC 1.1.1.2 created 1961' WHERE ec = 'EC 1.1.1.2';
UPDATE enzyme SET sys_name = 'L-homoserine:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.3';
UPDATE enzyme SET history = 'EC 1.1.1.3 created 1961, modified 1976' WHERE ec = 'EC 1.1.1.3';
UPDATE enzyme SET sys_name = '(R,R)-butane-2,3-diol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.4';
UPDATE enzyme SET history = 'EC 1.1.1.4 created 1961 (EC 1.1.1.74 created 1972, incorporated 1976)' WHERE ec = 'EC 1.1.1.4';
UPDATE enzyme SET history = 'EC 1.1.1.5 created 1961, modified 1976, deleted 2010' WHERE ec = 'EC 1.1.1.5';
UPDATE enzyme SET sys_name = 'Glycerol:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.6';
UPDATE enzyme SET history = 'EC 1.1.1.6 created 1961' WHERE ec = 'EC 1.1.1.6';
UPDATE enzyme SET sys_name = 'Propane-1,2-diol-1-phosphate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.7';
UPDATE enzyme SET history = 'EC 1.1.1.7 created 1961' WHERE ec = 'EC 1.1.1.7';
UPDATE enzyme SET sys_name = 'sn-glycerol-3-phosphate:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.8';
UPDATE enzyme SET history = 'EC 1.1.1.8 created 1961, modified 2005' WHERE ec = 'EC 1.1.1.8';
UPDATE enzyme SET sys_name = 'Xylitol:NAD(+) 2-oxidoreductase (D-xylulose-forming)' WHERE ec = 'EC 1.1.1.9';
UPDATE enzyme SET history = 'EC 1.1.1.9 created 1961' WHERE ec = 'EC 1.1.1.9';
UPDATE enzyme SET sys_name = 'Xylitol:NADP(+) 4-oxidoreductase (L-xylulose-forming)' WHERE ec = 'EC 1.1.1.10';
UPDATE enzyme SET history = 'EC 1.1.1.10 created 1961' WHERE ec = 'EC 1.1.1.10';
UPDATE enzyme SET sys_name = 'D-arabinitol:NAD(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.11';
UPDATE enzyme SET history = 'EC 1.1.1.11 created 1961' WHERE ec = 'EC 1.1.1.11';
UPDATE enzyme SET sys_name = 'L-arabinitol:NAD(+) 4-oxidoreductase (L-xylulose-forming)' WHERE ec = 'EC 1.1.1.12';
UPDATE enzyme SET history = 'EC 1.1.1.12 created 1961' WHERE ec = 'EC 1.1.1.12';
UPDATE enzyme SET sys_name = 'L-arabinitol:NAD(+) 2-oxidoreductase (L-ribulose-forming)' WHERE ec = 'EC 1.1.1.13';
UPDATE enzyme SET history = 'EC 1.1.1.13 created 1961' WHERE ec = 'EC 1.1.1.13';
UPDATE enzyme SET sys_name = 'L-iditol:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.14';
UPDATE enzyme SET history = 'EC 1.1.1.14 created 1961, modified 2011' WHERE ec = 'EC 1.1.1.14';
UPDATE enzyme SET sys_name = 'D-iditol:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.15';
UPDATE enzyme SET history = 'EC 1.1.1.15 created 1961' WHERE ec = 'EC 1.1.1.15';
UPDATE enzyme SET sys_name = 'Galactitol:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.16';
UPDATE enzyme SET history = 'EC 1.1.1.16 created 1961' WHERE ec = 'EC 1.1.1.16';
UPDATE enzyme SET sys_name = 'D-mannitol-1-phosphate:NAD(+) 5-oxidoreductase' WHERE ec = 'EC 1.1.1.17';
UPDATE enzyme SET history = 'EC 1.1.1.17 created 1961' WHERE ec = 'EC 1.1.1.17';
UPDATE enzyme SET sys_name = 'Myo-inositol:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.18';
UPDATE enzyme SET history = 'EC 1.1.1.18 created 1961' WHERE ec = 'EC 1.1.1.18';
UPDATE enzyme SET sys_name = 'L-gulonate:NADP(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.19';
UPDATE enzyme SET history = 'EC 1.1.1.19 created 1961' WHERE ec = 'EC 1.1.1.19';
UPDATE enzyme SET sys_name = 'L-gulono-1,4-lactone:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.20';
UPDATE enzyme SET history = 'EC 1.1.1.20 created 1961' WHERE ec = 'EC 1.1.1.20';
UPDATE enzyme SET sys_name = 'Alditol:NAD(P)(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.21';
UPDATE enzyme SET history = 'EC 1.1.1.21 created 1961 (EC 1.1.1.139 created 1972, incorporated 1978)' WHERE ec = 'EC 1.1.1.21';
UPDATE enzyme SET sys_name = 'UDP-glucose:NAD(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.22';
UPDATE enzyme SET history = 'EC 1.1.1.22 created 1961' WHERE ec = 'EC 1.1.1.22';
UPDATE enzyme SET sys_name = 'L-histidinol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.23';
UPDATE enzyme SET history = 'EC 1.1.1.23 created 1961' WHERE ec = 'EC 1.1.1.23';
UPDATE enzyme SET sys_name = 'L-quinate:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.24';
UPDATE enzyme SET history = 'EC 1.1.1.24 created 1961, modified 1976, modified 2004' WHERE ec = 'EC 1.1.1.24';
UPDATE enzyme SET sys_name = 'Shikimate:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.25';
UPDATE enzyme SET history = 'EC 1.1.1.25 created 1961, modified 1976, modified 2004' WHERE ec = 'EC 1.1.1.25';
UPDATE enzyme SET sys_name = 'Glycolate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.26';
UPDATE enzyme SET history = 'EC 1.1.1.26 created 1961' WHERE ec = 'EC 1.1.1.26';
UPDATE enzyme SET sys_name = '(S)-lactate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.27';
UPDATE enzyme SET history = 'EC 1.1.1.27 created 1961' WHERE ec = 'EC 1.1.1.27';
UPDATE enzyme SET sys_name = '(R)-lactate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.28';
UPDATE enzyme SET history = 'EC 1.1.1.28 created 1961' WHERE ec = 'EC 1.1.1.28';
UPDATE enzyme SET sys_name = 'D-glycerate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.29';
UPDATE enzyme SET history = 'EC 1.1.1.29 created 1961' WHERE ec = 'EC 1.1.1.29';
UPDATE enzyme SET sys_name = '(R)-3-hydroxybutanoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.30';
UPDATE enzyme SET history = 'EC 1.1.1.30 created 1961' WHERE ec = 'EC 1.1.1.30';
UPDATE enzyme SET sys_name = '3-hydroxy-2-methylpropanoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.31';
UPDATE enzyme SET history = 'EC 1.1.1.31 created 1961' WHERE ec = 'EC 1.1.1.31';
UPDATE enzyme SET sys_name = '(R)-mevalonate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.32';
UPDATE enzyme SET history = 'EC 1.1.1.32 created 1961' WHERE ec = 'EC 1.1.1.32';
UPDATE enzyme SET sys_name = '(R)-mevalonate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.33';
UPDATE enzyme SET history = 'EC 1.1.1.33 created 1961' WHERE ec = 'EC 1.1.1.33';
UPDATE enzyme SET sys_name = '(R)-mevalonate:NADP(+) oxidoreductase (CoA-acylating)' WHERE ec = 'EC 1.1.1.34';
UPDATE enzyme SET history = 'EC 1.1.1.34 created 1961' WHERE ec = 'EC 1.1.1.34';
UPDATE enzyme SET sys_name = '(S)-3-hydroxyacyl-CoA:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.35';
UPDATE enzyme SET history = 'EC 1.1.1.35 created 1961' WHERE ec = 'EC 1.1.1.35';
UPDATE enzyme SET sys_name = '(R)-3-hydroxyacyl-CoA:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.36';
UPDATE enzyme SET history = 'EC 1.1.1.36 created 1961' WHERE ec = 'EC 1.1.1.36';
UPDATE enzyme SET sys_name = '(S)-malate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.37';
UPDATE enzyme SET history = 'EC 1.1.1.37 created 1961' WHERE ec = 'EC 1.1.1.37';
UPDATE enzyme SET sys_name = '(S)-malate:NAD(+) oxidoreductase (oxaloacetate-decarboxylating)' WHERE ec = 'EC 1.1.1.38';
UPDATE enzyme SET history = 'EC 1.1.1.38 created 1961' WHERE ec = 'EC 1.1.1.38';
UPDATE enzyme SET sys_name = '(S)-malate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.39';
UPDATE enzyme SET history = 'EC 1.1.1.39 created 1961' WHERE ec = 'EC 1.1.1.39';
UPDATE enzyme SET sys_name = '(S)-malate:NADP(+) oxidoreductase (oxaloacetate-decarboxylating)' WHERE ec = 'EC 1.1.1.40';
UPDATE enzyme SET history = 'EC 1.1.1.40 created 1961, modified 1976' WHERE ec = 'EC 1.1.1.40';
UPDATE enzyme SET sys_name = 'Isocitrate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.41';
UPDATE enzyme SET history = 'EC 1.1.1.41 created 1961, modified 2005' WHERE ec = 'EC 1.1.1.41';
UPDATE enzyme SET sys_name = 'Isocitrate:NADP(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.42';
UPDATE enzyme SET history = 'EC 1.1.1.42 created 1961, modified 2005' WHERE ec = 'EC 1.1.1.42';
UPDATE enzyme SET sys_name = '6-phospho-D-gluconate:NAD(P)(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.43';
UPDATE enzyme SET history = 'EC 1.1.1.43 created 1961' WHERE ec = 'EC 1.1.1.43';
UPDATE enzyme SET sys_name = '6-phospho-D-gluconate:NADP(+) 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.44';
UPDATE enzyme SET history = 'EC 1.1.1.44 created 1961, modified 2013' WHERE ec = 'EC 1.1.1.44';
UPDATE enzyme SET sys_name = 'L-gulonate:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.45';
UPDATE enzyme SET history = 'EC 1.1.1.45 created 1961' WHERE ec = 'EC 1.1.1.45';
UPDATE enzyme SET sys_name = 'L-arabinose:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.46';
UPDATE enzyme SET history = 'EC 1.1.1.46 created 1961' WHERE ec = 'EC 1.1.1.46';
UPDATE enzyme SET sys_name = 'D-glucose:NAD(P)(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.47';
UPDATE enzyme SET history = 'EC 1.1.1.47 created 1961, modified 2013' WHERE ec = 'EC 1.1.1.47';
UPDATE enzyme SET sys_name = 'D-galactose:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.48';
UPDATE enzyme SET history = 'EC 1.1.1.48 created 1961,modified 2011' WHERE ec = 'EC 1.1.1.48';
UPDATE enzyme SET sys_name = 'D-glucose-6-phosphate:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.49';
UPDATE enzyme SET history = 'EC 1.1.1.49 created 1961, modified 2013' WHERE ec = 'EC 1.1.1.49';
UPDATE enzyme SET sys_name = '3-alpha-hydroxysteroid:NAD(P)(+) 3-oxidoreductase (Si-specific)' WHERE ec = 'EC 1.1.1.50';
UPDATE enzyme SET history = 'EC 1.1.1.50 created 1961, modified 1986, modified 1990, modified 2012, modified 2013' WHERE ec = 'EC 1.1.1.50';
UPDATE enzyme SET sys_name = '3(or 17)-beta-hydroxysteroid:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.51';
UPDATE enzyme SET history = 'EC 1.1.1.51 created 1961' WHERE ec = 'EC 1.1.1.51';
UPDATE enzyme SET sys_name = 'Lithocholate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.52';
UPDATE enzyme SET history = 'EC 1.1.1.52 created 1961, modified 1976' WHERE ec = 'EC 1.1.1.52';
UPDATE enzyme SET sys_name = '3-alpha-(or 20-beta)-hydroxysteroid:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.53';
UPDATE enzyme SET history = 'EC 1.1.1.53 created 1961, modified 1986' WHERE ec = 'EC 1.1.1.53';
UPDATE enzyme SET sys_name = 'Allyl-alcohol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.54';
UPDATE enzyme SET history = 'EC 1.1.1.54 created 1965' WHERE ec = 'EC 1.1.1.54';
UPDATE enzyme SET sys_name = 'Propane-1,2-diol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.55';
UPDATE enzyme SET history = 'EC 1.1.1.55 created 1965' WHERE ec = 'EC 1.1.1.55';
UPDATE enzyme SET sys_name = 'Ribitol:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.56';
UPDATE enzyme SET history = 'EC 1.1.1.56 created 1965' WHERE ec = 'EC 1.1.1.56';
UPDATE enzyme SET sys_name = 'D-mannonate:NAD(+) 5-oxidoreductase' WHERE ec = 'EC 1.1.1.57';
UPDATE enzyme SET history = 'EC 1.1.1.57 created 1965' WHERE ec = 'EC 1.1.1.57';
UPDATE enzyme SET sys_name = 'D-altronate:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.58';
UPDATE enzyme SET history = 'EC 1.1.1.58 created 1965' WHERE ec = 'EC 1.1.1.58';
UPDATE enzyme SET sys_name = '3-hydroxypropanoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.59';
UPDATE enzyme SET history = 'EC 1.1.1.59 created 1965' WHERE ec = 'EC 1.1.1.59';
UPDATE enzyme SET sys_name = 'D-glycerate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.60';
UPDATE enzyme SET history = 'EC 1.1.1.60 created 1965' WHERE ec = 'EC 1.1.1.60';
UPDATE enzyme SET sys_name = '4-hydroxybutanoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.61';
UPDATE enzyme SET history = 'EC 1.1.1.61 created 1965' WHERE ec = 'EC 1.1.1.61';
UPDATE enzyme SET sys_name = '17-beta-estradiol:NAD(P)(+) 17-oxidoreductase' WHERE ec = 'EC 1.1.1.62';
UPDATE enzyme SET history = 'EC 1.1.1.62 created 1965, modified 1983, modified 1986, modified 2012' WHERE ec = 'EC 1.1.1.62';
UPDATE enzyme SET history = 'EC 1.1.1.63 created 1965' WHERE ec = 'EC 1.1.1.63';
UPDATE enzyme SET sys_name = '17-beta-hydroxysteroid:NADP(+) 17-oxidoreductase' WHERE ec = 'EC 1.1.1.64';
UPDATE enzyme SET history = 'EC 1.1.1.64 created 1965' WHERE ec = 'EC 1.1.1.64';
UPDATE enzyme SET sys_name = 'Pyridoxine:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.65';
UPDATE enzyme SET history = 'EC 1.1.1.65 created 1965, modified 1976' WHERE ec = 'EC 1.1.1.65';
UPDATE enzyme SET sys_name = '10-hydroxydecanoate:NAD(+) 10-oxidoreductase' WHERE ec = 'EC 1.1.1.66';
UPDATE enzyme SET history = 'EC 1.1.1.66 created 1965' WHERE ec = 'EC 1.1.1.66';
UPDATE enzyme SET sys_name = 'D-mannitol:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.67';
UPDATE enzyme SET history = 'EC 1.1.1.67 created 1965' WHERE ec = 'EC 1.1.1.67';
UPDATE enzyme SET history = 'EC 1.1.1.68 created 1965, deleted 1978 (transferred to EC 1.1.99.15, deleted 1980)' WHERE ec = 'EC 1.1.1.68';
UPDATE enzyme SET sys_name = 'D-gluconate:NAD(P)(+) 5-oxidoreductase' WHERE ec = 'EC 1.1.1.69';
UPDATE enzyme SET history = 'EC 1.1.1.69 created 1965, modified 1976' WHERE ec = 'EC 1.1.1.69';
UPDATE enzyme SET history = 'EC 1.1.1.70 created 1965, deleted 1978' WHERE ec = 'EC 1.1.1.70';
UPDATE enzyme SET sys_name = 'Alcohol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.71';
UPDATE enzyme SET history = 'EC 1.1.1.71 created 1972' WHERE ec = 'EC 1.1.1.71';
UPDATE enzyme SET sys_name = 'Glycerol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.72';
UPDATE enzyme SET history = 'EC 1.1.1.72 created 1972' WHERE ec = 'EC 1.1.1.72';
UPDATE enzyme SET sys_name = 'Octanol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.73';
UPDATE enzyme SET history = 'EC 1.1.1.73 created 1972' WHERE ec = 'EC 1.1.1.73';
UPDATE enzyme SET history = 'EC 1.1.1.74 created 1972, deleted 1976' WHERE ec = 'EC 1.1.1.74';
UPDATE enzyme SET sys_name = '(R)-1-aminopropan-2-ol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.75';
UPDATE enzyme SET history = 'EC 1.1.1.75 created 1972' WHERE ec = 'EC 1.1.1.75';
UPDATE enzyme SET sys_name = '(S,S)-butane-2,3-diol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.76';
UPDATE enzyme SET history = 'EC 1.1.1.76 created 1972, modified 2010' WHERE ec = 'EC 1.1.1.76';
UPDATE enzyme SET sys_name = '(R)(or (S))-propane-1,2-diol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.77';
UPDATE enzyme SET history = 'EC 1.1.1.77 created 1972' WHERE ec = 'EC 1.1.1.77';
UPDATE enzyme SET sys_name = '(R)-lactaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.78';
UPDATE enzyme SET history = 'EC 1.1.1.78 created 1972, modified 2005, modified 2013' WHERE ec = 'EC 1.1.1.78';
UPDATE enzyme SET sys_name = 'Glycolate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.79';
UPDATE enzyme SET history = 'EC 1.1.1.79 created 1972' WHERE ec = 'EC 1.1.1.79';
UPDATE enzyme SET sys_name = 'Propan-2-ol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.80';
UPDATE enzyme SET history = 'EC 1.1.1.80 created 1972' WHERE ec = 'EC 1.1.1.80';
UPDATE enzyme SET sys_name = 'D-glycerate:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.81';
UPDATE enzyme SET history = 'EC 1.1.1.81 created 1972' WHERE ec = 'EC 1.1.1.81';
UPDATE enzyme SET sys_name = '(S)-malate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.82';
UPDATE enzyme SET history = 'EC 1.1.1.82 created 1972' WHERE ec = 'EC 1.1.1.82';
UPDATE enzyme SET sys_name = '(R)-malate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.83';
UPDATE enzyme SET history = 'EC 1.1.1.83 created 1972' WHERE ec = 'EC 1.1.1.83';
UPDATE enzyme SET sys_name = '(R)-3,3-dimethylmalate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.84';
UPDATE enzyme SET history = 'EC 1.1.1.84 created 1972' WHERE ec = 'EC 1.1.1.84';
UPDATE enzyme SET sys_name = '(2R,3S)-3-isopropylmalate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.85';
UPDATE enzyme SET history = 'EC 1.1.1.85 created 1972, modified 1976' WHERE ec = 'EC 1.1.1.85';
UPDATE enzyme SET sys_name = '(R)-2,3-dihydroxy-3-methylbutanoate:NADP(+) oxidoreductase (isomerizing)' WHERE ec = 'EC 1.1.1.86';
UPDATE enzyme SET history = 'EC 1.1.1.86 created 1972, modified 1976, modified 1981 (EC 1.1.1.89 created 1972, incorporated 1976)' WHERE ec = 'EC 1.1.1.86';
UPDATE enzyme SET sys_name = '(1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.87';
UPDATE enzyme SET history = 'EC 1.1.1.87 created 1972 (EC 1.1.1.155 created 1976, incorporated 2004)' WHERE ec = 'EC 1.1.1.87';
UPDATE enzyme SET sys_name = '(R)-mevalonate:NAD(+) oxidoreductase (CoA-acylating)' WHERE ec = 'EC 1.1.1.88';
UPDATE enzyme SET history = 'EC 1.1.1.88 created 1972, modified 2002' WHERE ec = 'EC 1.1.1.88';
UPDATE enzyme SET history = 'EC 1.1.1.89 created 1972, deleted 1976' WHERE ec = 'EC 1.1.1.89';
UPDATE enzyme SET sys_name = 'Aryl-alcohol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.90';
UPDATE enzyme SET history = 'EC 1.1.1.90 created 1972, modified 1989' WHERE ec = 'EC 1.1.1.90';
UPDATE enzyme SET sys_name = 'Aryl-alcohol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.91';
UPDATE enzyme SET history = 'EC 1.1.1.91 created 1972' WHERE ec = 'EC 1.1.1.91';
UPDATE enzyme SET sys_name = 'D-glycerate:NAD(P)(+) oxidoreductase (carboxylating)' WHERE ec = 'EC 1.1.1.92';
UPDATE enzyme SET history = 'EC 1.1.1.92 created 1972' WHERE ec = 'EC 1.1.1.92';
UPDATE enzyme SET sys_name = 'Tartrate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.93';
UPDATE enzyme SET history = 'EC 1.1.1.93 created 1972' WHERE ec = 'EC 1.1.1.93';
UPDATE enzyme SET sys_name = 'sn-glycerol-3-phosphate:NAD(P)(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.94';
UPDATE enzyme SET history = 'EC 1.1.1.94 created 1972, modified 2005' WHERE ec = 'EC 1.1.1.94';
UPDATE enzyme SET sys_name = '3-phospho-D-glycerate:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.95';
UPDATE enzyme SET history = 'EC 1.1.1.95 created 1972, modified 2006' WHERE ec = 'EC 1.1.1.95';
UPDATE enzyme SET sys_name = '3-(3,5-diiodo-4-hydroxyphenyl)lactate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.96';
UPDATE enzyme SET history = 'EC 1.1.1.96 created 1972' WHERE ec = 'EC 1.1.1.96';
UPDATE enzyme SET sys_name = '3-hydroxybenzyl-alcohol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.97';
UPDATE enzyme SET history = 'EC 1.1.1.97 created 1972' WHERE ec = 'EC 1.1.1.97';
UPDATE enzyme SET sys_name = '(R)-2-hydroxystearate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.98';
UPDATE enzyme SET history = 'EC 1.1.1.98 created 1972' WHERE ec = 'EC 1.1.1.98';
UPDATE enzyme SET sys_name = '(S)-2-hydroxystearate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.99';
UPDATE enzyme SET history = 'EC 1.1.1.99 created 1972' WHERE ec = 'EC 1.1.1.99';
UPDATE enzyme SET sys_name = '(3R)-3-hydroxyacyl-[acyl-carrier-protein]:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.100';
UPDATE enzyme SET history = 'EC 1.1.1.100 created 1972, modified 1976' WHERE ec = 'EC 1.1.1.100';
UPDATE enzyme SET sys_name = '1-palmitoylglycerol-3-phosphate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.101';
UPDATE enzyme SET history = 'EC 1.1.1.101 created 1972, modified 1976' WHERE ec = 'EC 1.1.1.101';
UPDATE enzyme SET sys_name = 'D-erythro-dihydrosphingosine:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.102';
UPDATE enzyme SET history = 'EC 1.1.1.102 created 1972' WHERE ec = 'EC 1.1.1.102';
UPDATE enzyme SET sys_name = 'L-threonine:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.103';
UPDATE enzyme SET history = 'EC 1.1.1.103 created 1972' WHERE ec = 'EC 1.1.1.103';
UPDATE enzyme SET sys_name = '4-hydroxy-L-proline:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.104';
UPDATE enzyme SET history = 'EC 1.1.1.104 created 1972' WHERE ec = 'EC 1.1.1.104';
UPDATE enzyme SET sys_name = 'All-trans retinol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.105';
UPDATE enzyme SET history = 'EC 1.1.1.105 created 1972, modified 2011' WHERE ec = 'EC 1.1.1.105';
UPDATE enzyme SET sys_name = '(R)-pantoate:NAD(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.106';
UPDATE enzyme SET history = 'EC 1.1.1.106 created 1972, modified 1976' WHERE ec = 'EC 1.1.1.106';
UPDATE enzyme SET sys_name = 'Pyridoxal:NAD(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.107';
UPDATE enzyme SET history = 'EC 1.1.1.107 created 1972' WHERE ec = 'EC 1.1.1.107';
UPDATE enzyme SET sys_name = 'Carnitine:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.108';
UPDATE enzyme SET history = 'EC 1.1.1.108 created 1972' WHERE ec = 'EC 1.1.1.108';
UPDATE enzyme SET history = 'EC 1.1.1.109 created 1972, deleted 1976' WHERE ec = 'EC 1.1.1.109';
UPDATE enzyme SET sys_name = '(Indol-3-yl)lactate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.110';
UPDATE enzyme SET history = 'EC 1.1.1.110 created 1972' WHERE ec = 'EC 1.1.1.110';
UPDATE enzyme SET sys_name = '(S)-3-(imidazol-5-yl)lactate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.111';
UPDATE enzyme SET history = 'EC 1.1.1.111 created 1972' WHERE ec = 'EC 1.1.1.111';
UPDATE enzyme SET sys_name = 'Indan-1-ol:NAD(P)(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.112';
UPDATE enzyme SET history = 'EC 1.1.1.112 created 1972' WHERE ec = 'EC 1.1.1.112';
UPDATE enzyme SET sys_name = 'L-xylose:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.113';
UPDATE enzyme SET history = 'EC 1.1.1.113 created 1972' WHERE ec = 'EC 1.1.1.113';
UPDATE enzyme SET sys_name = 'D-apiitol:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.114';
UPDATE enzyme SET history = 'EC 1.1.1.114 created 1972' WHERE ec = 'EC 1.1.1.114';
UPDATE enzyme SET sys_name = 'D-ribose:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.115';
UPDATE enzyme SET history = 'EC 1.1.1.115 created 1972' WHERE ec = 'EC 1.1.1.115';
UPDATE enzyme SET sys_name = 'D-arabinose:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.116';
UPDATE enzyme SET history = 'EC 1.1.1.116 created 1972' WHERE ec = 'EC 1.1.1.116';
UPDATE enzyme SET sys_name = 'D-arabinose:NAD(P)(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.117';
UPDATE enzyme SET history = 'EC 1.1.1.117 created 1972' WHERE ec = 'EC 1.1.1.117';
UPDATE enzyme SET sys_name = 'D-glucose:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.118';
UPDATE enzyme SET history = 'EC 1.1.1.118 created 1972, modified 1976' WHERE ec = 'EC 1.1.1.118';
UPDATE enzyme SET sys_name = 'D-glucose:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.119';
UPDATE enzyme SET history = 'EC 1.1.1.119 created 1972' WHERE ec = 'EC 1.1.1.119';
UPDATE enzyme SET sys_name = 'D-galactose:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.120';
UPDATE enzyme SET history = 'EC 1.1.1.120 created 1972' WHERE ec = 'EC 1.1.1.120';
UPDATE enzyme SET sys_name = 'D-aldose:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.121';
UPDATE enzyme SET history = 'EC 1.1.1.121 created 1972' WHERE ec = 'EC 1.1.1.121';
UPDATE enzyme SET sys_name = 'D-threo-aldose:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.122';
UPDATE enzyme SET history = 'EC 1.1.1.122 created 1972' WHERE ec = 'EC 1.1.1.122';
UPDATE enzyme SET sys_name = 'L-sorbose:NADP(+) 5-oxidoreductase' WHERE ec = 'EC 1.1.1.123';
UPDATE enzyme SET history = 'EC 1.1.1.123 created 1972, modified 1976' WHERE ec = 'EC 1.1.1.123';
UPDATE enzyme SET sys_name = 'D-fructose:NADP(+) 5-oxidoreductase' WHERE ec = 'EC 1.1.1.124';
UPDATE enzyme SET history = 'EC 1.1.1.124 created 1972, modified 1976' WHERE ec = 'EC 1.1.1.124';
UPDATE enzyme SET sys_name = '2-deoxy-D-gluconate:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.125';
UPDATE enzyme SET history = 'EC 1.1.1.125 created 1972' WHERE ec = 'EC 1.1.1.125';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-D-gluconate:NADP(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.126';
UPDATE enzyme SET history = 'EC 1.1.1.126 created 1972' WHERE ec = 'EC 1.1.1.126';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-D-gluconate:NAD(+) 5-oxidoreductase' WHERE ec = 'EC 1.1.1.127';
UPDATE enzyme SET history = 'EC 1.1.1.127 created 1972, modified 1976, modified 1989' WHERE ec = 'EC 1.1.1.127';
UPDATE enzyme SET history = 'EC 1.1.1.128 created 1972, modified 1976, deleted 2012' WHERE ec = 'EC 1.1.1.128';
UPDATE enzyme SET sys_name = 'L-threonate:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.129';
UPDATE enzyme SET history = 'EC 1.1.1.129 created 1972' WHERE ec = 'EC 1.1.1.129';
UPDATE enzyme SET sys_name = '3-dehydro-L-gulonate:NAD(P)(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.130';
UPDATE enzyme SET history = 'EC 1.1.1.130 created 1972' WHERE ec = 'EC 1.1.1.130';
UPDATE enzyme SET sys_name = 'D-mannonate:NAD(P)(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.131';
UPDATE enzyme SET history = 'EC 1.1.1.131 created 1972 (EC 1.2.1.34 created 1972, incorporated 1983; EC 1.1.1.180 created 1983, incorporated 1984)' WHERE ec = 'EC 1.1.1.131';
UPDATE enzyme SET sys_name = 'GDP-D-mannose:NAD(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.132';
UPDATE enzyme SET history = 'EC 1.1.1.132 created 1972' WHERE ec = 'EC 1.1.1.132';
UPDATE enzyme SET sys_name = 'dTDP-beta-L-rhamnose:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.133';
UPDATE enzyme SET history = 'EC 1.1.1.133 created 1972' WHERE ec = 'EC 1.1.1.133';
UPDATE enzyme SET sys_name = 'dTDP-6-deoxy-beta-L-talose:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.134';
UPDATE enzyme SET history = 'EC 1.1.1.134 created 1972' WHERE ec = 'EC 1.1.1.134';
UPDATE enzyme SET sys_name = 'GDP-6-deoxy-alpha-D-talose:NAD(P)(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.135';
UPDATE enzyme SET history = 'EC 1.1.1.135 created 1972, modified 1976' WHERE ec = 'EC 1.1.1.135';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:NAD(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.136';
UPDATE enzyme SET history = 'EC 1.1.1.136 created 1972, modified 2012' WHERE ec = 'EC 1.1.1.136';
UPDATE enzyme SET sys_name = 'D-ribitol-5-phosphate:NAD(P)(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.137';
UPDATE enzyme SET history = 'EC 1.1.1.137 created 1972' WHERE ec = 'EC 1.1.1.137';
UPDATE enzyme SET sys_name = 'D-mannitol:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.138';
UPDATE enzyme SET history = 'EC 1.1.1.138 created 1972' WHERE ec = 'EC 1.1.1.138';
UPDATE enzyme SET history = 'EC 1.1.1.139 created 1972, deleted 1978' WHERE ec = 'EC 1.1.1.139';
UPDATE enzyme SET sys_name = 'D-sorbitol-6-phosphate:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.140';
UPDATE enzyme SET history = 'EC 1.1.1.140 created 1972' WHERE ec = 'EC 1.1.1.140';
UPDATE enzyme SET sys_name = '(5Z,13E,15S)-11-alpha,15-dihydroxy-9-oxoprost-5,13-dienoate:NAD(+) 15-oxidoreductase' WHERE ec = 'EC 1.1.1.141';
UPDATE enzyme SET history = 'EC 1.1.1.141 created 1972' WHERE ec = 'EC 1.1.1.141';
UPDATE enzyme SET sys_name = '1D-3-O-methyl-chiro-inositol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.142';
UPDATE enzyme SET history = 'EC 1.1.1.142 created 1972' WHERE ec = 'EC 1.1.1.142';
UPDATE enzyme SET sys_name = '5-O-methyl-myo-inositol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.143';
UPDATE enzyme SET history = 'EC 1.1.1.143 created 1972' WHERE ec = 'EC 1.1.1.143';
UPDATE enzyme SET sys_name = 'Perillyl-alcohol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.144';
UPDATE enzyme SET history = 'EC 1.1.1.144 created 1972' WHERE ec = 'EC 1.1.1.144';
UPDATE enzyme SET sys_name = '3-beta-hydroxy-Delta(5)-steroid:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.145';
UPDATE enzyme SET history = 'EC 1.1.1.145 created 1972' WHERE ec = 'EC 1.1.1.145';
UPDATE enzyme SET sys_name = '11-beta-hydroxysteroid:NADP(+) 11-oxidoreductase' WHERE ec = 'EC 1.1.1.146';
UPDATE enzyme SET history = 'EC 1.1.1.146 created 1972' WHERE ec = 'EC 1.1.1.146';
UPDATE enzyme SET sys_name = '16-alpha-hydroxysteroid:NAD(P)(+) 16-oxidoreductase' WHERE ec = 'EC 1.1.1.147';
UPDATE enzyme SET history = 'EC 1.1.1.147 created 1972' WHERE ec = 'EC 1.1.1.147';
UPDATE enzyme SET sys_name = '17-alpha-hydroxysteroid:NAD(P)(+) 17-oxidoreductase' WHERE ec = 'EC 1.1.1.148';
UPDATE enzyme SET history = 'EC 1.1.1.148 created 1972' WHERE ec = 'EC 1.1.1.148';
UPDATE enzyme SET sys_name = '20-alpha-hydroxysteroid:NAD(P)(+) 20-oxidoreductase' WHERE ec = 'EC 1.1.1.149';
UPDATE enzyme SET history = 'EC 1.1.1.149 created 1972, deleted 1983, reinstated 1986' WHERE ec = 'EC 1.1.1.149';
UPDATE enzyme SET sys_name = '21-hydroxysteroid:NAD(+) 21-oxidoreductase' WHERE ec = 'EC 1.1.1.150';
UPDATE enzyme SET history = 'EC 1.1.1.150 created 1972' WHERE ec = 'EC 1.1.1.150';
UPDATE enzyme SET sys_name = '21-hydroxysteroid:NADP(+) 21-oxidoreductase' WHERE ec = 'EC 1.1.1.151';
UPDATE enzyme SET history = 'EC 1.1.1.151 created 1972' WHERE ec = 'EC 1.1.1.151';
UPDATE enzyme SET sys_name = '3-alpha-hydroxy-5-beta-steroid:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.152';
UPDATE enzyme SET history = 'EC 1.1.1.152 created 1972' WHERE ec = 'EC 1.1.1.152';
UPDATE enzyme SET sys_name = 'L-erythro-7,8-dihydrobiopterin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.153';
UPDATE enzyme SET history = 'EC 1.1.1.153 created 1972, modified 2012' WHERE ec = 'EC 1.1.1.153';
UPDATE enzyme SET sys_name = '(S)-ureidoglycolate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.154';
UPDATE enzyme SET history = 'EC 1.1.1.154 created 1976' WHERE ec = 'EC 1.1.1.154';
UPDATE enzyme SET history = 'EC 1.1.1.155 created 1976, deleted 2004' WHERE ec = 'EC 1.1.1.155';
UPDATE enzyme SET sys_name = 'Glycerol:NADP(+) 2-oxidoreductase (glycerone-forming)' WHERE ec = 'EC 1.1.1.156';
UPDATE enzyme SET history = 'EC 1.1.1.156 created 1976' WHERE ec = 'EC 1.1.1.156';
UPDATE enzyme SET sys_name = '(S)-3-hydroxybutanoyl-CoA:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.157';
UPDATE enzyme SET history = 'EC 1.1.1.157 created 1976' WHERE ec = 'EC 1.1.1.157';
UPDATE enzyme SET history = 'EC 1.1.1.158 created 1976, modified 1983, modified 2002, deleted 2013' WHERE ec = 'EC 1.1.1.158';
UPDATE enzyme SET sys_name = '7-alpha-hydroxysteroid:NAD(+) 7-oxidoreductase' WHERE ec = 'EC 1.1.1.159';
UPDATE enzyme SET history = 'EC 1.1.1.159 created 1976, modified 1980' WHERE ec = 'EC 1.1.1.159';
UPDATE enzyme SET sys_name = '(+-)-5-((tert-butylamino)-2-hydroxypropoxy)-1,2,3,4-tetrahydro-1-naphthol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.160';
UPDATE enzyme SET history = 'EC 1.1.1.160 created 1976' WHERE ec = 'EC 1.1.1.160';
UPDATE enzyme SET history = 'EC 1.1.1.161 created 1976, deleted 2012' WHERE ec = 'EC 1.1.1.161';
UPDATE enzyme SET sys_name = 'D-threitol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.162';
UPDATE enzyme SET history = 'EC 1.1.1.162 created 1976' WHERE ec = 'EC 1.1.1.162';
UPDATE enzyme SET sys_name = 'Cyclopentanol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.163';
UPDATE enzyme SET history = 'EC 1.1.1.163 created 1976' WHERE ec = 'EC 1.1.1.163';
UPDATE enzyme SET sys_name = 'Hexadecanol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.164';
UPDATE enzyme SET history = 'EC 1.1.1.164 created 1976' WHERE ec = 'EC 1.1.1.164';
UPDATE enzyme SET sys_name = '2-butyne-1,4-diol:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.165';
UPDATE enzyme SET history = 'EC 1.1.1.165 created 1976' WHERE ec = 'EC 1.1.1.165';
UPDATE enzyme SET sys_name = '(1S,3R,4S)-3,4-dihydroxycyclohexane-1-carboxylate:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.166';
UPDATE enzyme SET history = 'EC 1.1.1.166 created 1976' WHERE ec = 'EC 1.1.1.166';
UPDATE enzyme SET sys_name = 'Hydroxymalonate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.167';
UPDATE enzyme SET history = 'EC 1.1.1.167 created 1976' WHERE ec = 'EC 1.1.1.167';
UPDATE enzyme SET sys_name = '(R)-pantolactone:NADP(+) oxidoreductase (Re-specific)' WHERE ec = 'EC 1.1.1.168';
UPDATE enzyme SET history = 'EC 1.1.1.168 created 1976, modified 1986, modified 1999, modified 2013' WHERE ec = 'EC 1.1.1.168';
UPDATE enzyme SET sys_name = '(R)-pantoate:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.169';
UPDATE enzyme SET history = 'EC 1.1.1.169 created 1976' WHERE ec = 'EC 1.1.1.169';
UPDATE enzyme SET sys_name = '3-beta-hydroxysteroid-4-alpha-carboxylate:NAD(P)(+) 3-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.170';
UPDATE enzyme SET history = 'EC 1.1.1.170 created 1978, modified 2002, modified 2012' WHERE ec = 'EC 1.1.1.170';
UPDATE enzyme SET history = 'EC 1.1.1.171 created 1978, deleted 1984' WHERE ec = 'EC 1.1.1.171';
UPDATE enzyme SET sys_name = '2-hydroxyadipate:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.172';
UPDATE enzyme SET history = 'EC 1.1.1.172 created 1978' WHERE ec = 'EC 1.1.1.172';
UPDATE enzyme SET sys_name = 'L-rhamnofuranose:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.173';
UPDATE enzyme SET history = 'EC 1.1.1.173 created 1978' WHERE ec = 'EC 1.1.1.173';
UPDATE enzyme SET sys_name = 'Trans-cyclohexane-1,2-diol:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.174';
UPDATE enzyme SET history = 'EC 1.1.1.174 created 1978' WHERE ec = 'EC 1.1.1.174';
UPDATE enzyme SET sys_name = 'D-xylose:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.175';
UPDATE enzyme SET history = 'EC 1.1.1.175 created 1978' WHERE ec = 'EC 1.1.1.175';
UPDATE enzyme SET sys_name = '12-alpha-hydroxysteroid:NADP(+) 12-oxidoreductase' WHERE ec = 'EC 1.1.1.176';
UPDATE enzyme SET history = 'EC 1.1.1.176 created 1978' WHERE ec = 'EC 1.1.1.176';
UPDATE enzyme SET sys_name = 'sn-glycerol-3-phosphate:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.177';
UPDATE enzyme SET history = 'EC 1.1.1.177 created 1980, modified 1980' WHERE ec = 'EC 1.1.1.177';
UPDATE enzyme SET sys_name = '(2S,3S)-3-hydroxy-2-methylbutanoyl-CoA:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.178';
UPDATE enzyme SET history = 'EC 1.1.1.178 created 1981' WHERE ec = 'EC 1.1.1.178';
UPDATE enzyme SET sys_name = 'D-xylose:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.179';
UPDATE enzyme SET history = 'EC 1.1.1.179 created 1982' WHERE ec = 'EC 1.1.1.179';
UPDATE enzyme SET history = 'EC 1.1.1.180 created 1983, deleted 1984' WHERE ec = 'EC 1.1.1.180';
UPDATE enzyme SET sys_name = 'Cholest-5-ene-3-beta,7-alpha-diol:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.181';
UPDATE enzyme SET history = 'EC 1.1.1.181 created 1983' WHERE ec = 'EC 1.1.1.181';
UPDATE enzyme SET history = 'EC 1.1.1.182 created 1983, deleted 1990' WHERE ec = 'EC 1.1.1.182';
UPDATE enzyme SET sys_name = 'Geraniol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.183';
UPDATE enzyme SET history = 'EC 1.1.1.183 created 1983' WHERE ec = 'EC 1.1.1.183';
UPDATE enzyme SET sys_name = 'Secondary-alcohol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.184';
UPDATE enzyme SET history = 'EC 1.1.1.184 created 1983' WHERE ec = 'EC 1.1.1.184';
UPDATE enzyme SET sys_name = 'L-glycol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.185';
UPDATE enzyme SET history = 'EC 1.1.1.185 created 1984' WHERE ec = 'EC 1.1.1.185';
UPDATE enzyme SET sys_name = 'dTDP-D-galactose:NADP(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.186';
UPDATE enzyme SET history = 'EC 1.1.1.186 created 1984, modified 2002' WHERE ec = 'EC 1.1.1.186';
UPDATE enzyme SET sys_name = 'GDP-alpha-D-rhamnose:NAD(P)(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.187';
UPDATE enzyme SET history = 'EC 1.1.1.187 created 1984' WHERE ec = 'EC 1.1.1.187';
UPDATE enzyme SET sys_name = '(5Z,13E)-(15S)-9-alpha,11-alpha,15-trihydroxyprosta-5,13-dienoate:NADP(+) 11-oxidoreductase' WHERE ec = 'EC 1.1.1.188';
UPDATE enzyme SET history = 'EC 1.1.1.188 created 1984, modified 1989, modified 1990' WHERE ec = 'EC 1.1.1.188';
UPDATE enzyme SET sys_name = '(5Z,13E)-(15S)-9-alpha,11-alpha,15-trihydroxyprosta-5,13-dienoate:NADP(+) 9-oxidoreductase' WHERE ec = 'EC 1.1.1.189';
UPDATE enzyme SET history = 'EC 1.1.1.189 created 1984, modified 1989' WHERE ec = 'EC 1.1.1.189';
UPDATE enzyme SET sys_name = '(Indol-3-yl)ethanol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.190';
UPDATE enzyme SET history = 'EC 1.1.1.190 created 1984' WHERE ec = 'EC 1.1.1.190';
UPDATE enzyme SET sys_name = '(Indol-3-yl)ethanol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.191';
UPDATE enzyme SET history = 'EC 1.1.1.191 created 1984' WHERE ec = 'EC 1.1.1.191';
UPDATE enzyme SET sys_name = 'Long-chain-alcohol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.192';
UPDATE enzyme SET history = 'EC 1.1.1.192 created 1984' WHERE ec = 'EC 1.1.1.192';
UPDATE enzyme SET sys_name = '5-amino-6-(5-phospho-D-ribitylamino)uracil:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.193';
UPDATE enzyme SET history = 'EC 1.1.1.193 created 1984, modified 2010' WHERE ec = 'EC 1.1.1.193';
UPDATE enzyme SET sys_name = 'Coniferyl-alcohol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.194';
UPDATE enzyme SET history = 'EC 1.1.1.194 created 1984' WHERE ec = 'EC 1.1.1.194';
UPDATE enzyme SET sys_name = 'Cinnamyl-alcohol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.195';
UPDATE enzyme SET history = 'EC 1.1.1.195 created 1984' WHERE ec = 'EC 1.1.1.195';
UPDATE enzyme SET sys_name = '(5Z,13E)-(15S)-9-alpha,15-dihydroxy-11-oxoprosta-5,13-dienoate:NADP(+) 15-oxidoreductase' WHERE ec = 'EC 1.1.1.196';
UPDATE enzyme SET history = 'EC 1.1.1.196 created 1984, modified 1990' WHERE ec = 'EC 1.1.1.196';
UPDATE enzyme SET sys_name = '(13E)-(15S)-11-alpha,15-dihydroxy-9-oxoprost-13-enoate:NADP(+) 15-oxidoreductase' WHERE ec = 'EC 1.1.1.197';
UPDATE enzyme SET history = 'EC 1.1.1.197 created 1984' WHERE ec = 'EC 1.1.1.197';
UPDATE enzyme SET sys_name = '(+)-borneol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.198';
UPDATE enzyme SET history = 'EC 1.1.1.198 created 1984, modified 1990 (EC 1.1.1.182 created 1983, part incorporated 1990)' WHERE ec = 'EC 1.1.1.198';
UPDATE enzyme SET sys_name = 'Reduced-(S)-usnate:NAD(+) oxidoreductase (ether-bond-forming)' WHERE ec = 'EC 1.1.1.199';
UPDATE enzyme SET history = 'EC 1.1.1.199 created 1984' WHERE ec = 'EC 1.1.1.199';
UPDATE enzyme SET sys_name = 'D-aldose-6-phosphate:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.200';
UPDATE enzyme SET history = 'EC 1.1.1.200 created 1984' WHERE ec = 'EC 1.1.1.200';
UPDATE enzyme SET sys_name = '7-beta-hydroxysteroid:NADP(+) 7-oxidoreductase' WHERE ec = 'EC 1.1.1.201';
UPDATE enzyme SET history = 'EC 1.1.1.201 created 1984' WHERE ec = 'EC 1.1.1.201';
UPDATE enzyme SET sys_name = 'Propane-1,3-diol:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.202';
UPDATE enzyme SET history = 'EC 1.1.1.202 created 1984' WHERE ec = 'EC 1.1.1.202';
UPDATE enzyme SET sys_name = 'Uronate:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.203';
UPDATE enzyme SET history = 'EC 1.1.1.203 created 1972 as EC 1.2.1.35, transferred 1984 to EC 1.1.1.203' WHERE ec = 'EC 1.1.1.203';
UPDATE enzyme SET history = 'EC 1.1.1.204 created 1972 as EC 1.2.1.37, transferred 1984 to EC 1.1.1.204, modified 1989, deleted 2004' WHERE ec = 'EC 1.1.1.204';
UPDATE enzyme SET sys_name = 'IMP:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.205';
UPDATE enzyme SET history = 'EC 1.1.1.205 created 1961 as EC 1.2.1.14, transferred 1984 to EC 1.1.1.205' WHERE ec = 'EC 1.1.1.205';
UPDATE enzyme SET sys_name = 'Tropine:NADP(+) 3-alpha-oxidoreductase' WHERE ec = 'EC 1.1.1.206';
UPDATE enzyme SET history = 'EC 1.1.1.206 created 1984, modified 2007' WHERE ec = 'EC 1.1.1.206';
UPDATE enzyme SET sys_name = '(-)-menthol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.207';
UPDATE enzyme SET history = 'EC 1.1.1.207 created 1984' WHERE ec = 'EC 1.1.1.207';
UPDATE enzyme SET sys_name = '(+)-neomenthol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.208';
UPDATE enzyme SET history = 'EC 1.1.1.208 created 1984' WHERE ec = 'EC 1.1.1.208';
UPDATE enzyme SET sys_name = '3(or 17)-alpha-hydroxysteroid:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.209';
UPDATE enzyme SET history = 'EC 1.1.1.209 created 1986' WHERE ec = 'EC 1.1.1.209';
UPDATE enzyme SET sys_name = '3-beta-(or 20-alpha)-hydroxysteroid:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.210';
UPDATE enzyme SET history = 'EC 1.1.1.210 created 1986' WHERE ec = 'EC 1.1.1.210';
UPDATE enzyme SET sys_name = 'Long-chain-(S)-3-hydroxyacyl-CoA:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.211';
UPDATE enzyme SET history = 'EC 1.1.1.211 created 1986' WHERE ec = 'EC 1.1.1.211';
UPDATE enzyme SET sys_name = '(3R)-3-hydroxyacyl-[acyl-carrier-protein]:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.212';
UPDATE enzyme SET history = 'EC 1.1.1.212 created 1986' WHERE ec = 'EC 1.1.1.212';
UPDATE enzyme SET sys_name = '3-alpha-hydroxysteroid:NAD(P)(+) 3-oxidoreductase (Re-specific)' WHERE ec = 'EC 1.1.1.213';
UPDATE enzyme SET history = 'EC 1.1.1.213 created 1986, modified 2012, modified 2013' WHERE ec = 'EC 1.1.1.213';
UPDATE enzyme SET sys_name = '(R)-pantolactone:NADP(+) oxidoreductase (Si-specific)' WHERE ec = 'EC 1.1.1.214';
UPDATE enzyme SET history = 'EC 1.1.1.214 created 1986, modified 1999, modified 2013' WHERE ec = 'EC 1.1.1.214';
UPDATE enzyme SET sys_name = 'D-gluconate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.215';
UPDATE enzyme SET history = 'EC 1.1.1.215 created 1989' WHERE ec = 'EC 1.1.1.215';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesol:NADP(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.216';
UPDATE enzyme SET history = 'EC 1.1.1.216 created 1989' WHERE ec = 'EC 1.1.1.216';
UPDATE enzyme SET sys_name = 'Benzyl-(2R,3S)-2-methyl-3-hydroxybutanoate:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.217';
UPDATE enzyme SET history = 'EC 1.1.1.217 created 1989' WHERE ec = 'EC 1.1.1.217';
UPDATE enzyme SET sys_name = 'Morphine:NAD(P)(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.218';
UPDATE enzyme SET history = 'EC 1.1.1.218 created 1989, modified 1990' WHERE ec = 'EC 1.1.1.218';
UPDATE enzyme SET sys_name = 'Cis-3,4-leucopelargonidin:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.219';
UPDATE enzyme SET history = 'EC 1.1.1.219 created 1989' WHERE ec = 'EC 1.1.1.219';
UPDATE enzyme SET sys_name = '6-lactoyl-5,6,7,8-tetrahydropterin:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.220';
UPDATE enzyme SET history = 'EC 1.1.1.220 created 1989' WHERE ec = 'EC 1.1.1.220';
UPDATE enzyme SET sys_name = 'Vomifoliol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.221';
UPDATE enzyme SET history = 'EC 1.1.1.221 created 1989' WHERE ec = 'EC 1.1.1.221';
UPDATE enzyme SET sys_name = '(R)-3-(4-hydroxyphenyl)lactate:NAD(P)(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.222';
UPDATE enzyme SET history = 'EC 1.1.1.222 created 1989' WHERE ec = 'EC 1.1.1.222';
UPDATE enzyme SET sys_name = '(-)-trans-isopiperitenol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.223';
UPDATE enzyme SET history = 'EC 1.1.1.223 created 1989' WHERE ec = 'EC 1.1.1.223';
UPDATE enzyme SET sys_name = 'D-mannitol-1-phosphate:NADP(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.224';
UPDATE enzyme SET history = 'EC 1.1.1.224 created 1989' WHERE ec = 'EC 1.1.1.224';
UPDATE enzyme SET sys_name = 'Chlordecone-alcohol:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.225';
UPDATE enzyme SET history = 'EC 1.1.1.225 created 1989' WHERE ec = 'EC 1.1.1.225';
UPDATE enzyme SET sys_name = 'Trans-4-hydroxycyclohexanecarboxylate:NAD(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.226';
UPDATE enzyme SET history = 'EC 1.1.1.226 created 1990' WHERE ec = 'EC 1.1.1.226';
UPDATE enzyme SET sys_name = '(-)-borneol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.227';
UPDATE enzyme SET history = 'EC 1.1.1.227 created 1990 (EC 1.1.1.182 created 1983, part incorporated 1990)' WHERE ec = 'EC 1.1.1.227';
UPDATE enzyme SET sys_name = '(+)-cis-sabinol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.228';
UPDATE enzyme SET history = 'EC 1.1.1.228 created 1990 (EC 1.1.1.182 created 1983, part incorporated 1990)' WHERE ec = 'EC 1.1.1.228';
UPDATE enzyme SET sys_name = 'Diethyl-(2R,3R)-2-methyl-3-hydroxysuccinate:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.229';
UPDATE enzyme SET history = 'EC 1.1.1.229 created 1990' WHERE ec = 'EC 1.1.1.229';
UPDATE enzyme SET sys_name = '3-alpha-hydroxyglycyrrhetinate:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.230';
UPDATE enzyme SET history = 'EC 1.1.1.230 created 1990' WHERE ec = 'EC 1.1.1.230';
UPDATE enzyme SET sys_name = '(5Z,13E)-(15S)-6,9-alpha-epoxy-11-alpha,15-dihydroxyprosta-5,13-dienoate:NADP(+) 15-oxidoreductase' WHERE ec = 'EC 1.1.1.231';
UPDATE enzyme SET history = 'EC 1.1.1.231 created 1990' WHERE ec = 'EC 1.1.1.231';
UPDATE enzyme SET sys_name = '(15S)-15-hydroxy-5,8,11-cis-13-trans-icosatetraenoate:NAD(P)(+) 15-oxidoreductase' WHERE ec = 'EC 1.1.1.232';
UPDATE enzyme SET history = 'EC 1.1.1.232 created 1992' WHERE ec = 'EC 1.1.1.232';
UPDATE enzyme SET sys_name = 'N-acyl-D-mannosamine:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.233';
UPDATE enzyme SET history = 'EC 1.1.1.233 created 1992' WHERE ec = 'EC 1.1.1.233';
UPDATE enzyme SET sys_name = '(2S)-flavan-4-ol:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.234';
UPDATE enzyme SET history = 'EC 1.1.1.234 created 1992' WHERE ec = 'EC 1.1.1.234';
UPDATE enzyme SET sys_name = 'Coformycin:NADP(+) 8-oxidoreductase' WHERE ec = 'EC 1.1.1.235';
UPDATE enzyme SET history = 'EC 1.1.1.235 created 1992' WHERE ec = 'EC 1.1.1.235';
UPDATE enzyme SET sys_name = 'Pseudotropine:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.236';
UPDATE enzyme SET history = 'EC 1.1.1.236 created 1992, modified 2007' WHERE ec = 'EC 1.1.1.236';
UPDATE enzyme SET sys_name = '4-hydroxyphenyllactate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.237';
UPDATE enzyme SET history = 'EC 1.1.1.237 created 1992' WHERE ec = 'EC 1.1.1.237';
UPDATE enzyme SET sys_name = '12-beta-hydroxysteroid:NADP(+) 12-oxidoreductase' WHERE ec = 'EC 1.1.1.238';
UPDATE enzyme SET history = 'EC 1.1.1.238 created 1992' WHERE ec = 'EC 1.1.1.238';
UPDATE enzyme SET sys_name = '3-alpha(or 17-beta)-hydroxysteroid:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.239';
UPDATE enzyme SET history = 'EC 1.1.1.239 created 1992,  (EC 1.1.1.63 created 1965, incorporated 2012)' WHERE ec = 'EC 1.1.1.239';
UPDATE enzyme SET sys_name = 'N-acetyl-D-hexosamine:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.240';
UPDATE enzyme SET history = 'EC 1.1.1.240 created 1992' WHERE ec = 'EC 1.1.1.240';
UPDATE enzyme SET sys_name = '6-endo-hydroxycineole:NAD(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.241';
UPDATE enzyme SET history = 'EC 1.1.1.241 created 1992' WHERE ec = 'EC 1.1.1.241';
UPDATE enzyme SET history = 'EC 1.1.1.242 created 1992, deleted 2001' WHERE ec = 'EC 1.1.1.242';
UPDATE enzyme SET sys_name = '(-)-trans-carveol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.243';
UPDATE enzyme SET history = 'EC 1.1.1.243 created 1992' WHERE ec = 'EC 1.1.1.243';
UPDATE enzyme SET sys_name = 'Methanol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.244';
UPDATE enzyme SET history = 'EC 1.1.1.244 created 1992' WHERE ec = 'EC 1.1.1.244';
UPDATE enzyme SET sys_name = 'Cyclohexanol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.245';
UPDATE enzyme SET history = 'EC 1.1.1.245 created 1992' WHERE ec = 'EC 1.1.1.245';
UPDATE enzyme SET history = 'EC 1.1.1.246 created 1992, deleted 2013' WHERE ec = 'EC 1.1.1.246';
UPDATE enzyme SET sys_name = 'Codeine:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.247';
UPDATE enzyme SET history = 'EC 1.1.1.247 created 1999, modified 2001' WHERE ec = 'EC 1.1.1.247';
UPDATE enzyme SET sys_name = 'Salutaridinol:NADP(+) 7-oxidoreductase' WHERE ec = 'EC 1.1.1.248';
UPDATE enzyme SET history = 'EC 1.1.1.248 created 1999, modified 2001' WHERE ec = 'EC 1.1.1.248';
UPDATE enzyme SET history = 'EC 1.1.1.249 provisional version created 1999, deleted 1999 (reinstated 2001 as EC 2.5.1.46)' WHERE ec = 'EC 1.1.1.249';
UPDATE enzyme SET sys_name = 'D-arabinitol:NAD(+) 2-oxidoreductase (D-ribulose-forming)' WHERE ec = 'EC 1.1.1.250';
UPDATE enzyme SET history = 'EC 1.1.1.250 created 1999' WHERE ec = 'EC 1.1.1.250';
UPDATE enzyme SET sys_name = 'Galactitol-1-phosphate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.251';
UPDATE enzyme SET history = 'EC 1.1.1.251 created 1999' WHERE ec = 'EC 1.1.1.251';
UPDATE enzyme SET sys_name = 'Scytalone:NADP(+) Delta(5)-oxidoreductase' WHERE ec = 'EC 1.1.1.252';
UPDATE enzyme SET history = 'EC 1.1.1.252 created 1992 as EC 1.3.1.50, transferred 1999 to EC 1.1.1.252' WHERE ec = 'EC 1.1.1.252';
UPDATE enzyme SET history = 'EC 1.1.1.253 created 1999, deleted 2003' WHERE ec = 'EC 1.1.1.253';
UPDATE enzyme SET sys_name = '(S)-carnitine:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.254';
UPDATE enzyme SET history = 'EC 1.1.1.254 created 1999' WHERE ec = 'EC 1.1.1.254';
UPDATE enzyme SET sys_name = 'Mannitol:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.255';
UPDATE enzyme SET history = 'EC 1.1.1.255 created 2000' WHERE ec = 'EC 1.1.1.255';
UPDATE enzyme SET sys_name = 'Fluoren-9-ol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.256';
UPDATE enzyme SET history = 'EC 1.1.1.256 created 2000' WHERE ec = 'EC 1.1.1.256';
UPDATE enzyme SET sys_name = '4-(hydroxymethyl)benzenesulfonate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.257';
UPDATE enzyme SET history = 'EC 1.1.1.257 created 2000' WHERE ec = 'EC 1.1.1.257';
UPDATE enzyme SET sys_name = '6-hydroxyhexanoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.258';
UPDATE enzyme SET history = 'EC 1.1.1.258 created 2000' WHERE ec = 'EC 1.1.1.258';
UPDATE enzyme SET sys_name = '3-hydroxypimeloyl-CoA:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.259';
UPDATE enzyme SET history = 'EC 1.1.1.259 created 2000' WHERE ec = 'EC 1.1.1.259';
UPDATE enzyme SET sys_name = 'Sulcatol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.260';
UPDATE enzyme SET history = 'EC 1.1.1.260 created 2000, modified 2001' WHERE ec = 'EC 1.1.1.260';
UPDATE enzyme SET sys_name = 'sn-glycerol-1-phosphate:NAD(P)(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.261';
UPDATE enzyme SET history = 'EC 1.1.1.261 created 2000, modified 2009' WHERE ec = 'EC 1.1.1.261';
UPDATE enzyme SET sys_name = '4-phosphonooxy-L-threonine:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.262';
UPDATE enzyme SET history = 'EC 1.1.1.262 created 2000, modified 2006' WHERE ec = 'EC 1.1.1.262';
UPDATE enzyme SET sys_name = '1,5-anhydro-D-glucitol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.263';
UPDATE enzyme SET history = 'EC 1.1.1.263 created 2000' WHERE ec = 'EC 1.1.1.263';
UPDATE enzyme SET sys_name = 'L-idonate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.264';
UPDATE enzyme SET history = 'EC 1.1.1.264 created 2000, modified 2013' WHERE ec = 'EC 1.1.1.264';
UPDATE enzyme SET sys_name = '3-methylbutanol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.265';
UPDATE enzyme SET history = 'EC 1.1.1.265 created 2000' WHERE ec = 'EC 1.1.1.265';
UPDATE enzyme SET sys_name = 'dTDP-alpha-D-fucopyranose:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.266';
UPDATE enzyme SET history = 'EC 1.1.1.266 created 2001, modified 2013' WHERE ec = 'EC 1.1.1.266';
UPDATE enzyme SET sys_name = '2-C-methyl-D-erythritol-4-phosphate:NADP(+) oxidoreductase (isomerizing)' WHERE ec = 'EC 1.1.1.267';
UPDATE enzyme SET history = 'EC 1.1.1.267 created 2001' WHERE ec = 'EC 1.1.1.267';
UPDATE enzyme SET sys_name = '2-(2-(R)-hydroxypropylthio)ethanesulfonate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.268';
UPDATE enzyme SET history = 'EC 1.1.1.268 created 2001' WHERE ec = 'EC 1.1.1.268';
UPDATE enzyme SET sys_name = '2-(2-(S)-hydroxypropylthio)ethanesulfonate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.269';
UPDATE enzyme SET history = 'EC 1.1.1.269 created 2001' WHERE ec = 'EC 1.1.1.269';
UPDATE enzyme SET sys_name = '3-beta-hydroxysteroid:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.270';
UPDATE enzyme SET history = 'EC 1.1.1.270 created 2002, modified 2012' WHERE ec = 'EC 1.1.1.270';
UPDATE enzyme SET sys_name = 'GDP-beta-L-fucose:NADP(+) 4-oxidoreductase (3,5-epimerizing)' WHERE ec = 'EC 1.1.1.271';
UPDATE enzyme SET history = 'EC 1.1.1.271 created 2002, modified 2003' WHERE ec = 'EC 1.1.1.271';
UPDATE enzyme SET sys_name = '(R)-2-hydroxycarboxylate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.272';
UPDATE enzyme SET history = 'EC 1.1.1.272 created 2002, modified 2013' WHERE ec = 'EC 1.1.1.272';
UPDATE enzyme SET sys_name = '10-deoxysarpagine:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.273';
UPDATE enzyme SET history = 'EC 1.1.1.273 created 2002' WHERE ec = 'EC 1.1.1.273';
UPDATE enzyme SET sys_name = '2-dehydro-D-gluconate:NADP(+) 2-oxidoreductase (2-dehydro-D-gluconate-forming)' WHERE ec = 'EC 1.1.1.274';
UPDATE enzyme SET history = 'EC 1.1.1.274 created 2002, modified 2013' WHERE ec = 'EC 1.1.1.274';
UPDATE enzyme SET sys_name = '(+)-trans-carveol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.275';
UPDATE enzyme SET history = 'EC 1.1.1.275 created 2003' WHERE ec = 'EC 1.1.1.275';
UPDATE enzyme SET sys_name = 'L-serine:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.276';
UPDATE enzyme SET history = 'EC 1.1.1.276 created 2003' WHERE ec = 'EC 1.1.1.276';
UPDATE enzyme SET sys_name = '3-beta-hydroxy-5-beta-steroid:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.277';
UPDATE enzyme SET history = 'EC 1.1.1.277 created 2003' WHERE ec = 'EC 1.1.1.277';
UPDATE enzyme SET sys_name = '3-beta-hydroxy-5-alpha-steroid:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.278';
UPDATE enzyme SET history = 'EC 1.1.1.278 created 2003' WHERE ec = 'EC 1.1.1.278';
UPDATE enzyme SET sys_name = 'Ethyl-(R)-3-hydroxyhexanoate:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.279';
UPDATE enzyme SET history = 'EC 1.1.1.279 created 1990 as EC 1.2.1.55, transferred 2003 to EC 1.1.1.279' WHERE ec = 'EC 1.1.1.279';
UPDATE enzyme SET sys_name = 'Ethyl-(S)-3-hydroxyhexanoate:NADP(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.280';
UPDATE enzyme SET history = 'EC 1.1.1.280 created 1990 as EC 1.2.1.56, transferred 2003 to EC 1.1.1.280' WHERE ec = 'EC 1.1.1.280';
UPDATE enzyme SET sys_name = 'GDP-alpha-D-rhamnose:NAD(P)(+) 4-oxidoreductase (D-rhamnose-forming)' WHERE ec = 'EC 1.1.1.281';
UPDATE enzyme SET history = 'EC 1.1.1.281 created 2004' WHERE ec = 'EC 1.1.1.281';
UPDATE enzyme SET sys_name = 'L-quinate:NAD(P)(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.282';
UPDATE enzyme SET history = 'EC 1.1.1.282 created 2004' WHERE ec = 'EC 1.1.1.282';
UPDATE enzyme SET sys_name = '(S)-lactaldehyde:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.283';
UPDATE enzyme SET history = 'EC 1.1.1.283 created 2005, modified 2013' WHERE ec = 'EC 1.1.1.283';
UPDATE enzyme SET sys_name = 'S-(hydroxymethyl)glutathione:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.284';
UPDATE enzyme SET history = 'EC 1.1.1.284 created 2005 (EC 1.2.1.1 created 1961, modified 1982, modified 2002, part-transferred 2005 to EC 1.1.1.284)' WHERE ec = 'EC 1.1.1.284';
UPDATE enzyme SET sys_name = '2-deoxymugineic acid:NAD(P)(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.285';
UPDATE enzyme SET history = 'EC 1.1.1.285 created 2005' WHERE ec = 'EC 1.1.1.285';
UPDATE enzyme SET sys_name = 'Isocitrate(homoisocitrate):NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.286';
UPDATE enzyme SET history = 'EC 1.1.1.286 created 2005' WHERE ec = 'EC 1.1.1.286';
UPDATE enzyme SET sys_name = 'D-arabinitol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.287';
UPDATE enzyme SET history = 'EC 1.1.1.287 created 2005' WHERE ec = 'EC 1.1.1.287';
UPDATE enzyme SET sys_name = 'Xanthoxin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.288';
UPDATE enzyme SET history = 'EC 1.1.1.288 created 2005' WHERE ec = 'EC 1.1.1.288';
UPDATE enzyme SET sys_name = 'D-glucitol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.289';
UPDATE enzyme SET history = 'EC 1.1.1.289 created 2006' WHERE ec = 'EC 1.1.1.289';
UPDATE enzyme SET sys_name = '4-phospho-D-erythronate:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.290';
UPDATE enzyme SET history = 'EC 1.1.1.290 created 2006' WHERE ec = 'EC 1.1.1.290';
UPDATE enzyme SET sys_name = '(S)-2-hydroxymethylglutarate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.291';
UPDATE enzyme SET history = 'EC 1.1.1.291 created 2006' WHERE ec = 'EC 1.1.1.291';
UPDATE enzyme SET sys_name = '1,5-anhydro-D-mannitol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.292';
UPDATE enzyme SET history = 'EC 1.1.1.292 created 2007' WHERE ec = 'EC 1.1.1.292';
UPDATE enzyme SET history = 'EC 1.1.1.293 created 2007, withdrawn while undergoing public review, deleted 2007' WHERE ec = 'EC 1.1.1.293';
UPDATE enzyme SET sys_name = '7(1)-hydroxychlorophyllide-a:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.294';
UPDATE enzyme SET history = 'EC 1.1.1.294 created 2007' WHERE ec = 'EC 1.1.1.294';
UPDATE enzyme SET sys_name = '3-beta-hydroxy-9-beta-pimara-7,15-diene-19,6-beta-olide:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.295';
UPDATE enzyme SET history = 'EC 1.1.1.295 created 2008' WHERE ec = 'EC 1.1.1.295';
UPDATE enzyme SET sys_name = 'Menth-8-en-2-ol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.296';
UPDATE enzyme SET history = 'EC 1.1.1.296 created 2008' WHERE ec = 'EC 1.1.1.296';
UPDATE enzyme SET sys_name = 'Menth-8-ene-1,2-diol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.297';
UPDATE enzyme SET history = 'EC 1.1.1.297 created 2008' WHERE ec = 'EC 1.1.1.297';
UPDATE enzyme SET sys_name = '3-hydroxypropionate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.298';
UPDATE enzyme SET history = 'EC 1.1.1.298 created 2009' WHERE ec = 'EC 1.1.1.298';
UPDATE enzyme SET sys_name = '(S)-malate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.299';
UPDATE enzyme SET history = 'EC 1.1.1.299 created 2009' WHERE ec = 'EC 1.1.1.299';
UPDATE enzyme SET sys_name = 'Retinol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.300';
UPDATE enzyme SET history = 'EC 1.1.1.300 created 2009' WHERE ec = 'EC 1.1.1.300';
UPDATE enzyme SET sys_name = 'D-arabinitol-phosphate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.301';
UPDATE enzyme SET history = 'EC 1.1.1.301 created 2010' WHERE ec = 'EC 1.1.1.301';
UPDATE enzyme SET sys_name = '2,5-diamino-6-(1-D-ribosylamino)pyrimidin-4(3H)-one 5-phosphate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.302';
UPDATE enzyme SET history = 'EC 1.1.1.302 created 2010, modified 2011' WHERE ec = 'EC 1.1.1.302';
UPDATE enzyme SET sys_name = '(R)-acetoin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.303';
UPDATE enzyme SET history = 'EC 1.1.1.303 created 1961 as EC 1.1.1.5, transferred 2010 to EC 1.1.1.303' WHERE ec = 'EC 1.1.1.303';
UPDATE enzyme SET sys_name = '(S)-acetoin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.304';
UPDATE enzyme SET history = 'EC 1.1.1.304 created 2010' WHERE ec = 'EC 1.1.1.304';
UPDATE enzyme SET sys_name = 'UDP-glucuronate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.305';
UPDATE enzyme SET history = 'EC 1.1.1.305 created 2010' WHERE ec = 'EC 1.1.1.305';
UPDATE enzyme SET sys_name = 'S-(hydroxymethyl)mycothiol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.306';
UPDATE enzyme SET history = 'EC 1.1.1.306 created 2010 as EC 1.2.1.66, transferred 2010 to EC 1.1.1.306' WHERE ec = 'EC 1.1.1.306';
UPDATE enzyme SET sys_name = 'Xylitol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.307';
UPDATE enzyme SET history = 'EC 1.1.1.307 created 2010' WHERE ec = 'EC 1.1.1.307';
UPDATE enzyme SET sys_name = '(R)-2,3-dihydroxypropane-1-sulfonate:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.308';
UPDATE enzyme SET history = 'EC 1.1.1.308 created 2010' WHERE ec = 'EC 1.1.1.308';
UPDATE enzyme SET sys_name = '2-hydroxyethylphosphonate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.309';
UPDATE enzyme SET history = 'EC 1.1.1.309 created 2010' WHERE ec = 'EC 1.1.1.309';
UPDATE enzyme SET sys_name = '(2S)-sulfolactate:NAD+ oxidoreductase' WHERE ec = 'EC 1.1.1.310';
UPDATE enzyme SET history = 'EC 1.1.1.310 created 2011, modified 2013' WHERE ec = 'EC 1.1.1.310';
UPDATE enzyme SET sys_name = '(S)-1-phenylethanol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.311';
UPDATE enzyme SET history = 'EC 1.1.1.311 created 2011' WHERE ec = 'EC 1.1.1.311';
UPDATE enzyme SET sys_name = '4-carboxy-2-hydroxymuconate semialdehyde hemiaceta:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.312';
UPDATE enzyme SET history = 'EC 1.1.1.312 created 1978 as EC 1.2.1.45, transferred 2011 to EC 1.1.1.312' WHERE ec = 'EC 1.1.1.312';
UPDATE enzyme SET sys_name = 'Isethionate:NADP+ oxidoreductase' WHERE ec = 'EC 1.1.1.313';
UPDATE enzyme SET history = 'EC 1.1.1.313 created 2011' WHERE ec = 'EC 1.1.1.313';
UPDATE enzyme SET sys_name = 'Germacra-1(10),4,11(13)-trien-12-ol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.314';
UPDATE enzyme SET history = 'EC 1.1.1.314 created 2011' WHERE ec = 'EC 1.1.1.314';
UPDATE enzyme SET sys_name = '11-cis-retinol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.315';
UPDATE enzyme SET history = 'EC 1.1.1.315 created 2011' WHERE ec = 'EC 1.1.1.315';
UPDATE enzyme SET sys_name = 'L-galactose:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.316';
UPDATE enzyme SET history = 'EC 1.1.1.316 created 2011' WHERE ec = 'EC 1.1.1.316';
UPDATE enzyme SET sys_name = 'Raucaffrinoline:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.317';
UPDATE enzyme SET history = 'EC 1.1.1.317 cfreated 2011' WHERE ec = 'EC 1.1.1.317';
UPDATE enzyme SET sys_name = 'Eugenol:NADP(+) oxidoreductase (coniferyl ester reducing)' WHERE ec = 'EC 1.1.1.318';
UPDATE enzyme SET history = 'EC 1.1.1.318 created 2012' WHERE ec = 'EC 1.1.1.318';
UPDATE enzyme SET sys_name = 'Eugenol:NADP(+) oxidoreductase (coniferyl acetate reducing)' WHERE ec = 'EC 1.1.1.319';
UPDATE enzyme SET history = 'EC 1.1.1.319 created 2012' WHERE ec = 'EC 1.1.1.319';
UPDATE enzyme SET sys_name = '(S)-benzoin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.320';
UPDATE enzyme SET history = 'EC 1.1.1.320 created 2012' WHERE ec = 'EC 1.1.1.320';
UPDATE enzyme SET sys_name = '(R)-benzoin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.321';
UPDATE enzyme SET history = 'EC 1.1.1.321 created 2012' WHERE ec = 'EC 1.1.1.321';
UPDATE enzyme SET sys_name = '(-)-endo-fenchol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.322';
UPDATE enzyme SET history = 'EC 1.1.1.322 created 2012' WHERE ec = 'EC 1.1.1.322';
UPDATE enzyme SET sys_name = '(+)-thujan-3-ol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.323';
UPDATE enzyme SET history = 'EC 1.1.1.323 created 2012' WHERE ec = 'EC 1.1.1.323';
UPDATE enzyme SET sys_name = '(6E)-8-hydroxygeraniol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.324';
UPDATE enzyme SET history = 'EC 1.1.1.324 created 2012' WHERE ec = 'EC 1.1.1.324';
UPDATE enzyme SET sys_name = 'L-threo-7,8-dihydrobiopterin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.325';
UPDATE enzyme SET history = 'EC 1.1.1.325 created 2012' WHERE ec = 'EC 1.1.1.325';
UPDATE enzyme SET sys_name = '10-hydroxy-alpha-humulene:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.326';
UPDATE enzyme SET history = 'EC 1.1.1.326 created 2012' WHERE ec = 'EC 1.1.1.326';
UPDATE enzyme SET sys_name = '5-exo-hydroxycamphor:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.327';
UPDATE enzyme SET history = 'EC 1.1.1.327 created 2012' WHERE ec = 'EC 1.1.1.327';
UPDATE enzyme SET sys_name = '3,3-bipyridine-2,2,5,5,6,6-hexol:NADP(+) 11-oxidoreductase' WHERE ec = 'EC 1.1.1.328';
UPDATE enzyme SET history = 'EC 1.1.1.328 created 2012' WHERE ec = 'EC 1.1.1.328';
UPDATE enzyme SET sys_name = '2-deoxy-scyllo-inosamine:NAD(P)(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.329';
UPDATE enzyme SET history = 'EC 1.1.1.329 created 2012' WHERE ec = 'EC 1.1.1.329';
UPDATE enzyme SET sys_name = '(3R)-3-hydroxyacyl-CoA:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.330';
UPDATE enzyme SET history = 'EC 1.1.1.330 created 2012' WHERE ec = 'EC 1.1.1.330';
UPDATE enzyme SET sys_name = '(-)-secoisolariciresinol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.331';
UPDATE enzyme SET history = 'EC 1.1.1.331 created 2012' WHERE ec = 'EC 1.1.1.331';
UPDATE enzyme SET sys_name = 'Chanoclavine-I:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.332';
UPDATE enzyme SET history = 'EC 1.1.1.332 created 2012' WHERE ec = 'EC 1.1.1.332';
UPDATE enzyme SET sys_name = 'Trans,octacis-decaprenylphospho-beta-D-arabinofuranose:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.333';
UPDATE enzyme SET history = 'EC 1.1.1.333 created 2012' WHERE ec = 'EC 1.1.1.333';
UPDATE enzyme SET sys_name = 'Ecgonine methyl ester:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.334';
UPDATE enzyme SET history = 'EC 1.1.1.334 created 2012' WHERE ec = 'EC 1.1.1.334';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-2-amino-2-deoxy-alpha-D-glucuronate:NAD(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.335';
UPDATE enzyme SET history = 'EC 1.1.1.335 created 2012' WHERE ec = 'EC 1.1.1.335';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-mannosamine:NAD(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.336';
UPDATE enzyme SET history = 'EC 1.1.1.336 created 2012' WHERE ec = 'EC 1.1.1.336';
UPDATE enzyme SET sys_name = '(2S)-2-hydroxycarboxylate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.337';
UPDATE enzyme SET history = 'EC 1.1.1.337 created 2012' WHERE ec = 'EC 1.1.1.337';
UPDATE enzyme SET sys_name = '(2R)-3-sulfolactate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.338';
UPDATE enzyme SET history = 'EC 1.1.1.338 created 2012' WHERE ec = 'EC 1.1.1.338';
UPDATE enzyme SET sys_name = 'dTDP-6-deoxy-beta-L-talose:NAD(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.339';
UPDATE enzyme SET history = 'EC 1.1.1.339 created 2012' WHERE ec = 'EC 1.1.1.339';
UPDATE enzyme SET sys_name = '1-deoxy-11-beta-hydroxypentalenate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.340';
UPDATE enzyme SET history = 'EC 1.1.1.340 created 2012' WHERE ec = 'EC 1.1.1.340';
UPDATE enzyme SET sys_name = 'CDP-alpha-D-abequose:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.341';
UPDATE enzyme SET history = 'EC 1.1.1.341 created 2012' WHERE ec = 'EC 1.1.1.341';
UPDATE enzyme SET sys_name = 'CDP-alpha-D-paratose:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.342';
UPDATE enzyme SET history = 'EC 1.1.1.342 created 2012' WHERE ec = 'EC 1.1.1.342';
UPDATE enzyme SET sys_name = '6-phospho-D-gluconate:NAD(+) 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.343';
UPDATE enzyme SET history = 'EC 1.1.1.343 created 2013' WHERE ec = 'EC 1.1.1.343';
UPDATE enzyme SET sys_name = 'dTDP-6-deoxy-beta-L-talose:NAD(P)(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.344';
UPDATE enzyme SET history = 'EC 1.1.1.344 created 2013' WHERE ec = 'EC 1.1.1.344';
UPDATE enzyme SET sys_name = '(R)-2-hydroxycarboxylate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.345';
UPDATE enzyme SET history = 'EC 1.1.1.345 created 2013' WHERE ec = 'EC 1.1.1.345';
UPDATE enzyme SET sys_name = '2-dehydro-D-gluconate:NADP(+) 2-oxidoreductase (2-dehydro-L-gulonate-forming)' WHERE ec = 'EC 1.1.1.346';
UPDATE enzyme SET history = 'EC 1.1.1.346 created 2013' WHERE ec = 'EC 1.1.1.346';
UPDATE enzyme SET sys_name = 'Geraniol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.347';
UPDATE enzyme SET history = 'EC 1.1.1.347 created 2013' WHERE ec = 'EC 1.1.1.347';
UPDATE enzyme SET sys_name = '(3R,4R)-4-methoxyisoflavan-2,4,7-triol:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.1.1.348';
UPDATE enzyme SET history = 'EC 1.1.1.348 created 1992 as EC 1.1.1.246, part transferred 2013 to EC 1.1.1.348' WHERE ec = 'EC 1.1.1.348';
UPDATE enzyme SET sys_name = '(1S)-averantin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.349';
UPDATE enzyme SET history = 'EC 1.1.1.349 created 2013' WHERE ec = 'EC 1.1.1.349';
UPDATE enzyme SET sys_name = '(S)-ureidoglycolate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.350';
UPDATE enzyme SET history = 'EC 1.1.1.350 created 2013' WHERE ec = 'EC 1.1.1.350';
UPDATE enzyme SET sys_name = '6-phospho-D-gluconate:NAD(P)(+) 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.1.1.351';
UPDATE enzyme SET history = 'EC 1.1.1.351 created 2013' WHERE ec = 'EC 1.1.1.351';
UPDATE enzyme SET sys_name = '(1S,5S)-hydroxyaverantin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.352';
UPDATE enzyme SET history = 'EC 1.1.1.352 created 2013' WHERE ec = 'EC 1.1.1.352';
UPDATE enzyme SET sys_name = 'Versiconol-acetate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.353';
UPDATE enzyme SET history = 'EC 1.1.1.353 created 2013' WHERE ec = 'EC 1.1.1.353';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesol:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.354';
UPDATE enzyme SET history = 'EC 1.1.1.354 created 2013' WHERE ec = 'EC 1.1.1.354';
UPDATE enzyme SET sys_name = 'Kanamycin A:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.355';
UPDATE enzyme SET history = 'EC 1.1.1.355 created 2013' WHERE ec = 'EC 1.1.1.355';
UPDATE enzyme SET sys_name = 'GDP-beta-L-colitose:NAD(P)(+) 4-oxidoreductase (5-epimerizing)' WHERE ec = 'EC 1.1.1.356';
UPDATE enzyme SET history = 'EC 1.1.1.356 created 2013' WHERE ec = 'EC 1.1.1.356';
UPDATE enzyme SET sys_name = '3-alpha-hydroxysteroid:NAD(P)(+) 3-oxidoreductase' WHERE ec = 'EC 1.1.1.357';
UPDATE enzyme SET history = 'EC 1.1.1.357 created 2013' WHERE ec = 'EC 1.1.1.357';
UPDATE enzyme SET sys_name = '(R)-pantolactone:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.358';
UPDATE enzyme SET history = 'EC 1.1.1.358 created 2013' WHERE ec = 'EC 1.1.1.358';
UPDATE enzyme SET sys_name = 'An aldopyranose:NAD(P)(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.359';
UPDATE enzyme SET history = 'EC 1.2.1.87 created 2013' WHERE ec = 'EC 1.1.1.359';
UPDATE enzyme SET sys_name = 'D-glucose/D-galactose 1-dehydrogenase (NADPH)' WHERE ec = 'EC 1.1.1.360';
UPDATE enzyme SET history = 'EC 1.2.1.88 created 2013' WHERE ec = 'EC 1.1.1.360';
UPDATE enzyme SET sys_name = 'D-glucose-6-phosphate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.361';
UPDATE enzyme SET history = 'EC 1.1.1.361 created 2013' WHERE ec = 'EC 1.1.1.361';
UPDATE enzyme SET sys_name = 'Aklavinone:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.362';
UPDATE enzyme SET history = 'EC 1.1.1.362 created 2013' WHERE ec = 'EC 1.1.1.362';
UPDATE enzyme SET sys_name = 'D-glucose-6-phosphate:NAD(P)(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.363';
UPDATE enzyme SET history = 'EC 1.1.1.363 created 2013' WHERE ec = 'EC 1.1.1.363';
UPDATE enzyme SET sys_name = 'dTDP-6-deoxy-alpha-D-allose:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.1.1.364';
UPDATE enzyme SET history = 'EC 1.1.1.364 created 2013' WHERE ec = 'EC 1.1.1.364';
UPDATE enzyme SET sys_name = 'L-galactonate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.365';
UPDATE enzyme SET history = 'EC 1.1.1.365 created 2013' WHERE ec = 'EC 1.1.1.365';
UPDATE enzyme SET sys_name = 'L-idonate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.366';
UPDATE enzyme SET history = 'EC 1.1.1.366 created 2013' WHERE ec = 'EC 1.1.1.366';
UPDATE enzyme SET sys_name = 'UDP-2-acetamido-2,6-dideoxy-L-talose:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.367';
UPDATE enzyme SET history = 'EC 1.1.1.367 created 2014' WHERE ec = 'EC 1.1.1.367';
UPDATE enzyme SET sys_name = '6-hydroxycyclohex-1-ene-1-carbonyl-CoA:NAD(+) 6-oxidoreductase' WHERE ec = 'EC 1.1.1.368';
UPDATE enzyme SET history = 'EC 1.1.1.368 created 2014' WHERE ec = 'EC 1.1.1.368';
UPDATE enzyme SET sys_name = '1D-chiro-inositol:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.1.1.369';
UPDATE enzyme SET history = 'EC 1.1.1.369 created 2014' WHERE ec = 'EC 1.1.1.369';
UPDATE enzyme SET sys_name = 'Scyllo-inositol:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.370';
UPDATE enzyme SET history = 'EC 1.1.1.370 created 2014' WHERE ec = 'EC 1.1.1.370';
UPDATE enzyme SET sys_name = 'Scyllo-inositol:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.1.1.371';
UPDATE enzyme SET history = 'EC 1.1.1.371 created 2014' WHERE ec = 'EC 1.1.1.371';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.1.1.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.1.1.n3';
UPDATE enzyme SET sys_name = '(-)-(4R,6S)-trans-carveol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.n4';
UPDATE enzyme SET history = 'EC 1.1.1.n4 created 2010' WHERE ec = 'EC 1.1.1.n4';
UPDATE enzyme SET sys_name = '(2R,3S)-3-methylmalate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.1.1.n5';
UPDATE enzyme SET history = 'EC 1.1.1.n5 created 2010' WHERE ec = 'EC 1.1.1.n5';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.1.1.n6';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.1.1.n7';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.1.1.n8';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.1.1.n9';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.1.1.n10';
UPDATE enzyme SET sys_name = '4-hydroxybutanoate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.1.1.n11';
UPDATE enzyme SET history = 'EC 1.1.1.n11 created 2010' WHERE ec = 'EC 1.1.1.n11';
UPDATE enzyme SET sys_name = '(R)-3-hydroxyacyl-CoA:NAD+ oxidoreductase' WHERE ec = 'EC 1.1.1.n12';
UPDATE enzyme SET history = 'EC 1.1.1.n12 created 2012' WHERE ec = 'EC 1.1.1.n12';
UPDATE enzyme SET history = 'EC 1.1.1.n13 created 2012' WHERE ec = 'EC 1.1.1.n13';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.1.1.n14';
UPDATE enzyme SET history = 'EC 1.1.2.1 created 1961, deleted 1965' WHERE ec = 'EC 1.1.2.1';
UPDATE enzyme SET sys_name = 'D-mannitol:ferricytochrome-c 2-oxidoreductase' WHERE ec = 'EC 1.1.2.2';
UPDATE enzyme SET history = 'EC 1.1.2.2 created 1961' WHERE ec = 'EC 1.1.2.2';
UPDATE enzyme SET sys_name = '(S)-lactate:ferricytochrome-c 2-oxidoreductase' WHERE ec = 'EC 1.1.2.3';
UPDATE enzyme SET history = 'EC 1.1.2.3 created 1961' WHERE ec = 'EC 1.1.2.3';
UPDATE enzyme SET sys_name = '(R)-lactate:ferricytochrome-c 2-oxidoreductase' WHERE ec = 'EC 1.1.2.4';
UPDATE enzyme SET history = 'EC 1.1.2.4 created 1961' WHERE ec = 'EC 1.1.2.4';
UPDATE enzyme SET sys_name = '(R)-lactate:ferricytochrome-c-553 2-oxidoreductase' WHERE ec = 'EC 1.1.2.5';
UPDATE enzyme SET history = 'EC 1.1.2.5 created 1989' WHERE ec = 'EC 1.1.2.5';
UPDATE enzyme SET sys_name = 'Polyvinyl alcohol:ferricytochrome-c oxidoreductase' WHERE ec = 'EC 1.1.2.6';
UPDATE enzyme SET history = 'EC 1.1.2.6 created 1989 as EC 1.1.99.23, transferred 2010 to EC 1.1.2.6' WHERE ec = 'EC 1.1.2.6';
UPDATE enzyme SET sys_name = 'Methanol:cytochrome c oxidoreductase' WHERE ec = 'EC 1.1.2.7';
UPDATE enzyme SET history = 'EC 1.1.2.7 created 1972 as 1.1.99.8, modified 1982, part transferred 2010 to EC 1.1.2.7' WHERE ec = 'EC 1.1.2.7';
UPDATE enzyme SET sys_name = 'Alcohol:cytochrome c oxidoreductase' WHERE ec = 'EC 1.1.2.8';
UPDATE enzyme SET history = 'EC 1.1.2.8 created 1972 as 1.1.99.8, modified 1982, part transferred 2010 to EC 1.1.2.8' WHERE ec = 'EC 1.1.2.8';
UPDATE enzyme SET history = 'EC 1.1.3.1 created 1961, deleted 1984' WHERE ec = 'EC 1.1.3.1';
UPDATE enzyme SET history = 'EC 1.1.3.2 created 1961, deleted 1972 (transferred to EC 1.13.2.4, deleted 1976)' WHERE ec = 'EC 1.1.3.2';
UPDATE enzyme SET sys_name = '(S)-malate:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.3';
UPDATE enzyme SET history = 'EC 1.1.3.3 created 1961' WHERE ec = 'EC 1.1.3.3';
UPDATE enzyme SET sys_name = 'Beta-D-glucose:oxygen 1-oxidoreductase' WHERE ec = 'EC 1.1.3.4';
UPDATE enzyme SET history = 'EC 1.1.3.4 created 1961' WHERE ec = 'EC 1.1.3.4';
UPDATE enzyme SET sys_name = 'D-hexose:oxygen 1-oxidoreductase' WHERE ec = 'EC 1.1.3.5';
UPDATE enzyme SET history = 'EC 1.1.3.5 created 1961, modified 1976' WHERE ec = 'EC 1.1.3.5';
UPDATE enzyme SET sys_name = 'Cholesterol:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.6';
UPDATE enzyme SET history = 'EC 1.1.3.6 created 1961, modified 1982, modified 2012' WHERE ec = 'EC 1.1.3.6';
UPDATE enzyme SET sys_name = 'Aryl-alcohol:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.7';
UPDATE enzyme SET history = 'EC 1.1.3.7 created 1965' WHERE ec = 'EC 1.1.3.7';
UPDATE enzyme SET sys_name = 'L-gulono-1,4-lactone:oxygen 3-oxidoreductase' WHERE ec = 'EC 1.1.3.8';
UPDATE enzyme SET history = 'EC 1.1.3.8 created 1965, modified 2001, modified 2006' WHERE ec = 'EC 1.1.3.8';
UPDATE enzyme SET sys_name = 'D-galactose:oxygen 6-oxidoreductase' WHERE ec = 'EC 1.1.3.9';
UPDATE enzyme SET history = 'EC 1.1.3.9 created 1965' WHERE ec = 'EC 1.1.3.9';
UPDATE enzyme SET sys_name = 'Pyranose:oxygen 2-oxidoreductase' WHERE ec = 'EC 1.1.3.10';
UPDATE enzyme SET history = 'EC 1.1.3.10 created 1972' WHERE ec = 'EC 1.1.3.10';
UPDATE enzyme SET sys_name = 'L-sorbose:oxygen 5-oxidoreductase' WHERE ec = 'EC 1.1.3.11';
UPDATE enzyme SET history = 'EC 1.1.3.11 created 1972' WHERE ec = 'EC 1.1.3.11';
UPDATE enzyme SET sys_name = 'Pyridoxine:oxygen 4-oxidoreductase' WHERE ec = 'EC 1.1.3.12';
UPDATE enzyme SET history = 'EC 1.1.3.12 created 1972, modified 1976' WHERE ec = 'EC 1.1.3.12';
UPDATE enzyme SET sys_name = 'Alcohol:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.13';
UPDATE enzyme SET history = 'EC 1.1.3.13 created 1972' WHERE ec = 'EC 1.1.3.13';
UPDATE enzyme SET sys_name = 'Catechol:oxygen oxidoreductase (dimerizing)' WHERE ec = 'EC 1.1.3.14';
UPDATE enzyme SET history = 'EC 1.1.3.14 created 1972' WHERE ec = 'EC 1.1.3.14';
UPDATE enzyme SET sys_name = '(S)-2-hydroxy-acid:oxygen 2-oxidoreductase' WHERE ec = 'EC 1.1.3.15';
UPDATE enzyme SET history = 'EC 1.1.3.15 created 1972 (EC 1.1.3.1 created 1961, incorporated 1984)' WHERE ec = 'EC 1.1.3.15';
UPDATE enzyme SET sys_name = 'Ecdysone:oxygen 3-oxidoreductase' WHERE ec = 'EC 1.1.3.16';
UPDATE enzyme SET history = 'EC 1.1.3.16 created 1976' WHERE ec = 'EC 1.1.3.16';
UPDATE enzyme SET sys_name = 'Choline:oxygen 1-oxidoreductase' WHERE ec = 'EC 1.1.3.17';
UPDATE enzyme SET history = 'EC 1.1.3.17 created 1978, modified 2005, modified 2007' WHERE ec = 'EC 1.1.3.17';
UPDATE enzyme SET sys_name = 'Secondary-alcohol:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.18';
UPDATE enzyme SET history = 'EC 1.1.3.18 created 1981' WHERE ec = 'EC 1.1.3.18';
UPDATE enzyme SET sys_name = '(S)-2-hydroxy-2-(4-hydroxyphenyl)acetate:oxygen 1-oxidoreductase' WHERE ec = 'EC 1.1.3.19';
UPDATE enzyme SET history = 'EC 1.1.3.19 created 1984' WHERE ec = 'EC 1.1.3.19';
UPDATE enzyme SET sys_name = 'Long-chain-alcohol:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.20';
UPDATE enzyme SET history = 'EC 1.1.3.20 created 1984, modified 2010' WHERE ec = 'EC 1.1.3.20';
UPDATE enzyme SET sys_name = 'sn-glycerol-3-phosphate:oxygen 2-oxidoreductase' WHERE ec = 'EC 1.1.3.21';
UPDATE enzyme SET history = 'EC 1.1.3.21 created 1984' WHERE ec = 'EC 1.1.3.21';
UPDATE enzyme SET history = 'EC 1.1.3.22 created 1961 as EC 1.2.3.2, transferred 1984 to EC 1.1.3.22, modified 1989, deleted 2004' WHERE ec = 'EC 1.1.3.22';
UPDATE enzyme SET sys_name = 'Thiamine:oxygen 5-oxidoreductase' WHERE ec = 'EC 1.1.3.23';
UPDATE enzyme SET history = 'EC 1.1.3.23 created 1984' WHERE ec = 'EC 1.1.3.23';
UPDATE enzyme SET history = 'EC 1.1.3.24 created 1984, deleted 2007' WHERE ec = 'EC 1.1.3.24';
UPDATE enzyme SET history = 'EC 1.1.3.25 created 1986, deleted 2005' WHERE ec = 'EC 1.1.3.25';
UPDATE enzyme SET history = 'EC 1.1.3.26 created 1989, deleted 2002' WHERE ec = 'EC 1.1.3.26';
UPDATE enzyme SET sys_name = 'L-2-hydroxyphytanate:oxygen 2-oxidoreductase' WHERE ec = 'EC 1.1.3.27';
UPDATE enzyme SET history = 'EC 1.1.3.27 created 1990' WHERE ec = 'EC 1.1.3.27';
UPDATE enzyme SET sys_name = 'Nucleoside:oxygen 5-oxidoreductase' WHERE ec = 'EC 1.1.3.28';
UPDATE enzyme SET history = 'EC 1.1.3.28 created 1992, modified 2001' WHERE ec = 'EC 1.1.3.28';
UPDATE enzyme SET sys_name = 'N-acyl-D-hexosamine:oxygen 1-oxidoreductase' WHERE ec = 'EC 1.1.3.29';
UPDATE enzyme SET history = 'EC 1.1.3.29 created 1992' WHERE ec = 'EC 1.1.3.29';
UPDATE enzyme SET sys_name = 'Polyvinyl-alcohol:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.30';
UPDATE enzyme SET history = 'EC 1.1.3.30 created 1992' WHERE ec = 'EC 1.1.3.30';
UPDATE enzyme SET history = 'EC 1.1.3.31 created 1992, deleted 2003' WHERE ec = 'EC 1.1.3.31';
UPDATE enzyme SET history = 'EC 1.1.3.32 created 1999, deleted 2002' WHERE ec = 'EC 1.1.3.32';
UPDATE enzyme SET history = 'EC 1.1.3.33 created 1999, deleted 2002' WHERE ec = 'EC 1.1.3.33';
UPDATE enzyme SET history = 'EC 1.1.3.34 created 1999, deleted 2002' WHERE ec = 'EC 1.1.3.34';
UPDATE enzyme SET history = 'EC 1.1.3.35 created 1999, deleted 2002' WHERE ec = 'EC 1.1.3.35';
UPDATE enzyme SET history = 'EC 1.1.3.36 created 1999, deleted 2002' WHERE ec = 'EC 1.1.3.36';
UPDATE enzyme SET sys_name = 'D-arabinono-1,4-lactone:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.37';
UPDATE enzyme SET history = 'EC 1.1.3.37 created 1999' WHERE ec = 'EC 1.1.3.37';
UPDATE enzyme SET sys_name = 'Vanillyl alcohol:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.38';
UPDATE enzyme SET history = 'EC 1.1.3.38 created 1999' WHERE ec = 'EC 1.1.3.38';
UPDATE enzyme SET sys_name = 'Nucleoside:oxygen 5-oxidoreductase (H(2)O(2)-forming)' WHERE ec = 'EC 1.1.3.39';
UPDATE enzyme SET history = 'EC 1.1.3.39 created 2001' WHERE ec = 'EC 1.1.3.39';
UPDATE enzyme SET sys_name = 'Mannitol:oxygen oxidoreductase (cyclizing)' WHERE ec = 'EC 1.1.3.40';
UPDATE enzyme SET history = 'EC 1.1.3.40 created 2001' WHERE ec = 'EC 1.1.3.40';
UPDATE enzyme SET sys_name = 'Alditol:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.41';
UPDATE enzyme SET history = 'EC 1.1.3.41 created 2002, modified 2008' WHERE ec = 'EC 1.1.3.41';
UPDATE enzyme SET sys_name = 'Prosolanapyrone-II:oxygen 3-oxidoreductase' WHERE ec = 'EC 1.1.3.42';
UPDATE enzyme SET history = 'EC 1.1.3.42 created 2011' WHERE ec = 'EC 1.1.3.42';
UPDATE enzyme SET sys_name = 'Paromamine:oxygen 6-oxidoreductase' WHERE ec = 'EC 1.1.3.43';
UPDATE enzyme SET history = 'EC 1.1.3.43 created 2012' WHERE ec = 'EC 1.1.3.43';
UPDATE enzyme SET sys_name = '6-deamino-6-hydroxyneomycin C:oxygen 6-oxidoreductase' WHERE ec = 'EC 1.1.3.44';
UPDATE enzyme SET history = 'EC 1.1.3.44 created 2012' WHERE ec = 'EC 1.1.3.44';
UPDATE enzyme SET sys_name = 'Aclacinomycin-N:oxygen oxidoreductase' WHERE ec = 'EC 1.1.3.45';
UPDATE enzyme SET history = 'EC 1.1.3.45 created 2013' WHERE ec = 'EC 1.1.3.45';
UPDATE enzyme SET sys_name = '2-methyl-3-phytyl-1,4-naphthoquinone:oxidized-dithiothreitol oxidoreductase' WHERE ec = 'EC 1.1.4.1';
UPDATE enzyme SET history = 'EC 1.1.4.1 created 1989' WHERE ec = 'EC 1.1.4.1';
UPDATE enzyme SET sys_name = '3-hydroxy-2-methyl-3-phytyl-2,3-dihydronaphthoquinone:oxidized-dithiothreitol oxidoreductase' WHERE ec = 'EC 1.1.4.2';
UPDATE enzyme SET history = 'EC 1.1.4.2 created 1989' WHERE ec = 'EC 1.1.4.2';
UPDATE enzyme SET history = 'EC 1.1.5.1 created 1983, deleted 2002' WHERE ec = 'EC 1.1.5.1';
UPDATE enzyme SET sys_name = 'D-glucose:ubiquinone oxidoreductase' WHERE ec = 'EC 1.1.5.2';
UPDATE enzyme SET history = 'EC 1.1.5.2 created 1982 as 1.1.99.17, transferred 2003 to EC 1.1.5.2, modified 2010' WHERE ec = 'EC 1.1.5.2';
UPDATE enzyme SET sys_name = 'sn-glycerol-3-phosphate:quinone oxidoreductase' WHERE ec = 'EC 1.1.5.3';
UPDATE enzyme SET history = 'EC 1.1.5.3 created 1961 as EC 1.1.2.1, transferred 1965 to EC 1.1.99.5, transferred 2009 to EC 1.1.5.3' WHERE ec = 'EC 1.1.5.3';
UPDATE enzyme SET sys_name = '(S)-malate:quinone oxidoreductase' WHERE ec = 'EC 1.1.5.4';
UPDATE enzyme SET history = 'EC 1.1.5.4, created 1978 as EC 1.1.99.16, transferred 2009 to EC 1.1.5.4' WHERE ec = 'EC 1.1.5.4';
UPDATE enzyme SET sys_name = 'Alcohol:quinone oxidoreductase' WHERE ec = 'EC 1.1.5.5';
UPDATE enzyme SET history = 'EC 1.1.5.5 created 2009, modified 2010' WHERE ec = 'EC 1.1.5.5';
UPDATE enzyme SET sys_name = 'Formate:quinone oxidoreductase' WHERE ec = 'EC 1.1.5.6';
UPDATE enzyme SET history = 'EC 1.1.5.6 created 2010' WHERE ec = 'EC 1.1.5.6';
UPDATE enzyme SET sys_name = 'Cyclic alcohol:quinone oxidoreductase' WHERE ec = 'EC 1.1.5.7';
UPDATE enzyme SET history = 'EC 1.1.5.7 created 2010' WHERE ec = 'EC 1.1.5.7';
UPDATE enzyme SET sys_name = 'Quinate:quinone 3-oxidoreductase' WHERE ec = 'EC 1.1.5.8';
UPDATE enzyme SET history = 'EC 1.1.5.8 created 1992 as EC 1.1.99.25, modified 2004, transferred 2010 to EC 1.1.5.8' WHERE ec = 'EC 1.1.5.8';
UPDATE enzyme SET sys_name = 'D-glucose:quinone 1-oxidoreductase' WHERE ec = 'EC 1.1.5.9';
UPDATE enzyme SET history = 'EC 1.1.5.9 created 1972 as 1.1.99.10, modified 1976, transferred 2013 to EC 1.1.5.9' WHERE ec = 'EC 1.1.5.9';
UPDATE enzyme SET sys_name = '(R)-2-hydroxyacid:quinone oxidoreductase' WHERE ec = 'EC 1.1.5.10';
UPDATE enzyme SET history = 'EC 1.1.5.10 created 2014' WHERE ec = 'EC 1.1.5.10';
UPDATE enzyme SET sys_name = 'Myo-inositol:quinone 2-oxidoreductase' WHERE ec = 'EC 1.1.5.n1';
UPDATE enzyme SET history = 'EC 1.1.5.n1 created 2010' WHERE ec = 'EC 1.1.5.n1';
UPDATE enzyme SET sys_name = 'Alcohol:azurin oxidoreductase' WHERE ec = 'EC 1.1.9.1';
UPDATE enzyme SET history = 'EC 1.1.9.1 created 2010 as EC 1.1.98.1; transferred 2011 to EC 1.1.9.1' WHERE ec = 'EC 1.1.9.1';
UPDATE enzyme SET history = 'EC 1.1.98.1 created 2010, deleted 2012' WHERE ec = 'EC 1.1.98.1';
UPDATE enzyme SET sys_name = 'D-glucose-6-phosphate:F420 1-oxidoreductase' WHERE ec = 'EC 1.1.98.2';
UPDATE enzyme SET history = 'EC 1.1.98.2 created 2010 as EC 1.1.99.34, transferred 2011 to EC 1.1.98.2' WHERE ec = 'EC 1.1.98.2';
UPDATE enzyme SET sys_name = 'Trans,octacis-decaprenylphospho-beta-D-ribofuranose:FAD 2-oxidoreductase' WHERE ec = 'EC 1.1.98.3';
UPDATE enzyme SET history = 'EC 1.1.98.3 created 2012' WHERE ec = 'EC 1.1.98.3';
UPDATE enzyme SET sys_name = 'Quinol:coenzyme-F420 oxidoreductase' WHERE ec = 'EC 1.1.98.4';
UPDATE enzyme SET history = 'EC 1.1.98.4 created 2013' WHERE ec = 'EC 1.1.98.4';
UPDATE enzyme SET sys_name = 'Secondary-alcohol:coenzyme F420 oxidoreductase' WHERE ec = 'EC 1.1.98.5';
UPDATE enzyme SET history = 'EC 1.1.98.5 created 2013' WHERE ec = 'EC 1.1.98.5';
UPDATE enzyme SET sys_name = 'Choline:acceptor 1-oxidoreductase' WHERE ec = 'EC 1.1.99.1';
UPDATE enzyme SET history = 'EC 1.1.99.1 created 1961, modified 1989, modified 2005' WHERE ec = 'EC 1.1.99.1';
UPDATE enzyme SET sys_name = '(S)-2-hydroxyglutarate:acceptor 2-oxidoreductase' WHERE ec = 'EC 1.1.99.2';
UPDATE enzyme SET history = 'EC 1.1.99.2 created 1961, modified 2013' WHERE ec = 'EC 1.1.99.2';
UPDATE enzyme SET sys_name = 'D-gluconate:acceptor 2-oxidoreductase' WHERE ec = 'EC 1.1.99.3';
UPDATE enzyme SET history = 'EC 1.1.99.3 created 1961, modified 1976, modified 1989' WHERE ec = 'EC 1.1.99.3';
UPDATE enzyme SET sys_name = '2-dehydro-D-gluconate:acceptor 2-oxidoreductase' WHERE ec = 'EC 1.1.99.4';
UPDATE enzyme SET history = 'EC 1.1.99.4 created 1961, modified 1989' WHERE ec = 'EC 1.1.99.4';
UPDATE enzyme SET history = 'EC 1.1.99.5 created 1961 as EC 1.1.2.1, transferred 1965 to EC 1.1.99.5, deleted 2009' WHERE ec = 'EC 1.1.99.5';
UPDATE enzyme SET sys_name = '(R)-lactate:acceptor 2-oxidoreductase' WHERE ec = 'EC 1.1.99.6';
UPDATE enzyme SET history = 'EC 1.1.99.6 created 1965, modified 2013' WHERE ec = 'EC 1.1.99.6';
UPDATE enzyme SET sys_name = '(S)-lactate:oxaloacetate oxidoreductase' WHERE ec = 'EC 1.1.99.7';
UPDATE enzyme SET history = 'EC 1.1.99.7 created 1972' WHERE ec = 'EC 1.1.99.7';
UPDATE enzyme SET history = 'EC 1.1.99.8 created 1972, modified 1982, deleted 2010' WHERE ec = 'EC 1.1.99.8';
UPDATE enzyme SET sys_name = 'Pyridoxine:acceptor 5-oxidoreductase' WHERE ec = 'EC 1.1.99.9';
UPDATE enzyme SET history = 'EC 1.1.99.9 created 1972, modified 1976' WHERE ec = 'EC 1.1.99.9';
UPDATE enzyme SET history = 'EC 1.1.99.10 created 1972, modified 1976, deleted 2013' WHERE ec = 'EC 1.1.99.10';
UPDATE enzyme SET sys_name = 'D-fructose:acceptor 5-oxidoreductase' WHERE ec = 'EC 1.1.99.11';
UPDATE enzyme SET history = 'EC 1.1.99.11 created 1972' WHERE ec = 'EC 1.1.99.11';
UPDATE enzyme SET sys_name = 'L-sorbose:acceptor 5-oxidoreductase' WHERE ec = 'EC 1.1.99.12';
UPDATE enzyme SET history = 'EC 1.1.99.12 created 1972' WHERE ec = 'EC 1.1.99.12';
UPDATE enzyme SET sys_name = 'D-aldohexoside:acceptor 3-oxidoreductase' WHERE ec = 'EC 1.1.99.13';
UPDATE enzyme SET history = 'EC 1.1.99.13 created 1972' WHERE ec = 'EC 1.1.99.13';
UPDATE enzyme SET sys_name = 'Glycolate:acceptor 2-oxidoreductase' WHERE ec = 'EC 1.1.99.14';
UPDATE enzyme SET history = 'EC 1.1.99.14 created 1978' WHERE ec = 'EC 1.1.99.14';
UPDATE enzyme SET history = 'EC 1.1.99.15 created 1978, deleted 1980' WHERE ec = 'EC 1.1.99.15';
UPDATE enzyme SET history = 'EC 1.1.99.16 created 1978, deleted 2010' WHERE ec = 'EC 1.1.99.16';
UPDATE enzyme SET history = 'EC 1.1.99.17 created 1982, deleted 2004' WHERE ec = 'EC 1.1.99.17';
UPDATE enzyme SET sys_name = 'Cellobiose:acceptor 1-oxidoreductase' WHERE ec = 'EC 1.1.99.18';
UPDATE enzyme SET history = 'EC 1.1.99.18 created 1983, modified 2002 (EC 1.1.5.1 created 1983, incorporated 2002; EC 1.1.3.25 created 1986, incorporated 2005)' WHERE ec = 'EC 1.1.99.18';
UPDATE enzyme SET history = 'EC 1.1.99.19 created 1961 as EC 1.2.99.1, transferred 1984 to EC 1.1.99.19, deleted 2006' WHERE ec = 'EC 1.1.99.19';
UPDATE enzyme SET sys_name = 'Alkan-1-ol:acceptor oxidoreductase' WHERE ec = 'EC 1.1.99.20';
UPDATE enzyme SET history = 'EC 1.1.99.20 created 1989' WHERE ec = 'EC 1.1.99.20';
UPDATE enzyme SET sys_name = 'D-sorbitol:acceptor 1-oxidoreductase' WHERE ec = 'EC 1.1.99.21';
UPDATE enzyme SET history = 'EC 1.1.99.21 created 1989' WHERE ec = 'EC 1.1.99.21';
UPDATE enzyme SET sys_name = 'Glycerol:acceptor 1-oxidoreductase' WHERE ec = 'EC 1.1.99.22';
UPDATE enzyme SET history = 'EC 1.1.99.22 created 1989' WHERE ec = 'EC 1.1.99.22';
UPDATE enzyme SET history = 'EC 1.1.99.23 created 1989, deleted 2010' WHERE ec = 'EC 1.1.99.23';
UPDATE enzyme SET sys_name = '(S)-3-hydroxybutanoate:2-oxoglutarate oxidoreductase' WHERE ec = 'EC 1.1.99.24';
UPDATE enzyme SET history = 'EC 1.1.99.24 created 1992' WHERE ec = 'EC 1.1.99.24';
UPDATE enzyme SET history = 'EC 1.1.99.25 created 1992, modified 2004, deleted 2011' WHERE ec = 'EC 1.1.99.25';
UPDATE enzyme SET sys_name = '3-hydroxycyclohexanone:acceptor 1-oxidoreductase' WHERE ec = 'EC 1.1.99.26';
UPDATE enzyme SET history = 'EC 1.1.99.26 created 1992' WHERE ec = 'EC 1.1.99.26';
UPDATE enzyme SET sys_name = '(R)-pantolactone:acceptor oxidoreductase (flavin-containing)' WHERE ec = 'EC 1.1.99.27';
UPDATE enzyme SET history = 'EC 1.1.99.27 created 1999' WHERE ec = 'EC 1.1.99.27';
UPDATE enzyme SET sys_name = 'D-glucose:D-fructose oxidoreductase' WHERE ec = 'EC 1.1.99.28';
UPDATE enzyme SET history = 'EC 1.1.99.28 created 1999' WHERE ec = 'EC 1.1.99.28';
UPDATE enzyme SET sys_name = 'Pyranose:acceptor oxidoreductase' WHERE ec = 'EC 1.1.99.29';
UPDATE enzyme SET history = 'EC 1.1.99.29 created 2004' WHERE ec = 'EC 1.1.99.29';
UPDATE enzyme SET sys_name = '(2R)-hydroxy-carboxylate:acceptor oxidoreductase' WHERE ec = 'EC 1.1.99.30';
UPDATE enzyme SET history = 'EC 1.1.99.30 created 2004' WHERE ec = 'EC 1.1.99.30';
UPDATE enzyme SET sys_name = '(S)-2-hydroxy-2-phenylacetate:acceptor 2-oxidoreductase' WHERE ec = 'EC 1.1.99.31';
UPDATE enzyme SET history = 'EC 1.1.99.31 created 2006' WHERE ec = 'EC 1.1.99.31';
UPDATE enzyme SET sys_name = 'L-sorbose:acceptor 1-oxidoreductase' WHERE ec = 'EC 1.1.99.32';
UPDATE enzyme SET history = 'EC 1.1.99.32 created 2008' WHERE ec = 'EC 1.1.99.32';
UPDATE enzyme SET sys_name = 'Formate:acceptor oxidoreductase' WHERE ec = 'EC 1.1.99.33';
UPDATE enzyme SET history = 'EC 1.1.99.33 created 2010' WHERE ec = 'EC 1.1.99.33';
UPDATE enzyme SET history = 'EC 1.1.99.34 created 2010, deleted 2011' WHERE ec = 'EC 1.1.99.34';
UPDATE enzyme SET sys_name = 'D-glucose:acceptor oxidoreductase' WHERE ec = 'EC 1.1.99.35';
UPDATE enzyme SET history = 'EC 1.1.99.35 created 2010' WHERE ec = 'EC 1.1.99.35';
UPDATE enzyme SET sys_name = 'Ethanol:acceptor oxidoreductase' WHERE ec = 'EC 1.1.99.36';
UPDATE enzyme SET history = 'EC 1.1.99.36 created 2010' WHERE ec = 'EC 1.1.99.36';
UPDATE enzyme SET sys_name = 'Methanol:acceptor oxidoreductase' WHERE ec = 'EC 1.1.99.37';
UPDATE enzyme SET history = 'EC 1.1.99.37 created 2010' WHERE ec = 'EC 1.1.99.37';
UPDATE enzyme SET sys_name = '2-deoxy-scyllo-inosamine:S-adenosyl-L-methionine 1-oxidoreductase' WHERE ec = 'EC 1.1.99.38';
UPDATE enzyme SET history = 'EC 1.1.99.38 created 2012, modified 2013' WHERE ec = 'EC 1.1.99.38';
UPDATE enzyme SET sys_name = '(R)-2-hydroxyglutarate:acceptor 2-oxidoreductase' WHERE ec = 'EC 1.1.99.39';
UPDATE enzyme SET history = 'EC 1.1.99.39 created 2013' WHERE ec = 'EC 1.1.99.39';
UPDATE enzyme SET history = 'EC 1.2.1.1 created 1961, modified 1982, modified 2002, deleted 2005' WHERE ec = 'EC 1.2.1.1';
UPDATE enzyme SET sys_name = 'Formate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.2';
UPDATE enzyme SET history = 'EC 1.2.1.2 created 1961' WHERE ec = 'EC 1.2.1.2';
UPDATE enzyme SET sys_name = 'Aldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.3';
UPDATE enzyme SET history = 'EC 1.2.1.3 created 1961 (EC 1.1.1.70 created 1965, incorporated 1978)' WHERE ec = 'EC 1.2.1.3';
UPDATE enzyme SET sys_name = 'Aldehyde:NADP(+) oxidoreductase' WHERE ec = 'EC 1.2.1.4';
UPDATE enzyme SET history = 'EC 1.2.1.4 created 1961' WHERE ec = 'EC 1.2.1.4';
UPDATE enzyme SET sys_name = 'Aldehyde:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.2.1.5';
UPDATE enzyme SET history = 'EC 1.2.1.5 created 1961' WHERE ec = 'EC 1.2.1.5';
UPDATE enzyme SET history = 'EC 1.2.1.6 created 1961, deleted 1965' WHERE ec = 'EC 1.2.1.6';
UPDATE enzyme SET sys_name = 'Benzaldehyde:NADP(+) oxidoreductase' WHERE ec = 'EC 1.2.1.7';
UPDATE enzyme SET history = 'EC 1.2.1.7 created 1961' WHERE ec = 'EC 1.2.1.7';
UPDATE enzyme SET sys_name = 'Betaine-aldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.8';
UPDATE enzyme SET history = 'EC 1.2.1.8 created 1961, modified 2005, modified 2011' WHERE ec = 'EC 1.2.1.8';
UPDATE enzyme SET sys_name = 'D-glyceraldehyde-3-phosphate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.2.1.9';
UPDATE enzyme SET history = 'EC 1.2.1.9 created 1961' WHERE ec = 'EC 1.2.1.9';
UPDATE enzyme SET sys_name = 'Acetaldehyde:NAD(+) oxidoreductase (CoA-acetylating)' WHERE ec = 'EC 1.2.1.10';
UPDATE enzyme SET history = 'EC 1.2.1.10 created 1961, modified 2006, modified 2011' WHERE ec = 'EC 1.2.1.10';
UPDATE enzyme SET sys_name = 'L-aspartate-4-semialdehyde:NADP(+) oxidoreductase (phosphorylating)' WHERE ec = 'EC 1.2.1.11';
UPDATE enzyme SET history = 'EC 1.2.1.11 created 1961' WHERE ec = 'EC 1.2.1.11';
UPDATE enzyme SET sys_name = 'D-glyceraldehyde-3-phosphate:NAD(+) oxidoreductase (phosphorylating)' WHERE ec = 'EC 1.2.1.12';
UPDATE enzyme SET history = 'EC 1.2.1.12 created 1961' WHERE ec = 'EC 1.2.1.12';
UPDATE enzyme SET sys_name = 'D-glyceraldehyde-3-phosphate:NADP(+) oxidoreductase (phosphorylating)' WHERE ec = 'EC 1.2.1.13';
UPDATE enzyme SET history = 'EC 1.2.1.13 created 1961' WHERE ec = 'EC 1.2.1.13';
UPDATE enzyme SET history = 'EC 1.2.1.14 created 1961, deleted 1984' WHERE ec = 'EC 1.2.1.14';
UPDATE enzyme SET sys_name = '3-oxopropanoate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.2.1.15';
UPDATE enzyme SET history = 'EC 1.2.1.15 created 1965' WHERE ec = 'EC 1.2.1.15';
UPDATE enzyme SET sys_name = 'Succinate-semialdehyde:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.2.1.16';
UPDATE enzyme SET history = 'EC 1.2.1.16 created 1965' WHERE ec = 'EC 1.2.1.16';
UPDATE enzyme SET sys_name = 'Glyoxylate:NADP(+) oxidoreductase (CoA-oxalylating)' WHERE ec = 'EC 1.2.1.17';
UPDATE enzyme SET history = 'EC 1.2.1.17 created 1965' WHERE ec = 'EC 1.2.1.17';
UPDATE enzyme SET sys_name = '3-oxopropanoate:NAD(P)(+) oxidoreductase (decarboxylating, CoA-acetylating)' WHERE ec = 'EC 1.2.1.18';
UPDATE enzyme SET history = 'EC 1.2.1.18 created 1965' WHERE ec = 'EC 1.2.1.18';
UPDATE enzyme SET sys_name = '4-aminobutanal:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.2.1.19';
UPDATE enzyme SET history = 'EC 1.2.1.19 created 1965, modified 1989  (EC 1.5.1.35 created 2006, incorporated 2007)' WHERE ec = 'EC 1.2.1.19';
UPDATE enzyme SET sys_name = 'Glutarate-semialdehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.20';
UPDATE enzyme SET history = 'EC 1.2.1.20 created 1965' WHERE ec = 'EC 1.2.1.20';
UPDATE enzyme SET sys_name = 'Glycolaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.21';
UPDATE enzyme SET history = 'EC 1.2.1.21 created 1972' WHERE ec = 'EC 1.2.1.21';
UPDATE enzyme SET sys_name = '(S)-lactaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.22';
UPDATE enzyme SET history = 'EC 1.2.1.22 created 1972' WHERE ec = 'EC 1.2.1.22';
UPDATE enzyme SET sys_name = '2-oxoaldehyde:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.2.1.23';
UPDATE enzyme SET history = 'EC 1.2.1.23 created 1972, modified 1986' WHERE ec = 'EC 1.2.1.23';
UPDATE enzyme SET sys_name = 'Succinate-semialdehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.24';
UPDATE enzyme SET history = 'EC 1.2.1.24 created 1972, modified 2010' WHERE ec = 'EC 1.2.1.24';
UPDATE enzyme SET sys_name = '3-methyl-2-oxobutanoate:NAD(+) 2-oxidoreductase (CoA-methyl-propanoylating)' WHERE ec = 'EC 1.2.1.25';
UPDATE enzyme SET history = 'EC 1.2.1.25 created 1972' WHERE ec = 'EC 1.2.1.25';
UPDATE enzyme SET sys_name = '2,5-dioxopentanoate:NADP(+) 5-oxidoreductase' WHERE ec = 'EC 1.2.1.26';
UPDATE enzyme SET history = 'EC 1.2.1.26 created 1972' WHERE ec = 'EC 1.2.1.26';
UPDATE enzyme SET sys_name = '2-methyl-3-oxopropanoate:NAD(+) 3-oxidoreductase (CoA-propanoylating)' WHERE ec = 'EC 1.2.1.27';
UPDATE enzyme SET history = 'EC 1.2.1.27 created 1972' WHERE ec = 'EC 1.2.1.27';
UPDATE enzyme SET sys_name = 'Benzaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.28';
UPDATE enzyme SET history = 'EC 1.2.1.28 created 1972' WHERE ec = 'EC 1.2.1.28';
UPDATE enzyme SET sys_name = 'Aryl-aldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.29';
UPDATE enzyme SET history = 'EC 1.2.1.29 created 1972' WHERE ec = 'EC 1.2.1.29';
UPDATE enzyme SET sys_name = 'Aryl-aldehyde:NADP(+) oxidoreductase (ATP-forming)' WHERE ec = 'EC 1.2.1.30';
UPDATE enzyme SET history = 'EC 1.2.1.30 created 1972' WHERE ec = 'EC 1.2.1.30';
UPDATE enzyme SET sys_name = 'L-2-aminoadipate-6-semialdehyde:NAD(P)(+) 6-oxidoreductase' WHERE ec = 'EC 1.2.1.31';
UPDATE enzyme SET history = 'EC 1.2.1.31 created 1972, modified 2011' WHERE ec = 'EC 1.2.1.31';
UPDATE enzyme SET sys_name = '2-aminomuconate-6-semialdehyde:NAD(+) 6-oxidoreductase' WHERE ec = 'EC 1.2.1.32';
UPDATE enzyme SET history = 'EC 1.2.1.32 created 1972' WHERE ec = 'EC 1.2.1.32';
UPDATE enzyme SET sys_name = '(R)-4-dehydropantoate:NAD(+) 4-oxidoreductase' WHERE ec = 'EC 1.2.1.33';
UPDATE enzyme SET history = 'EC 1.2.1.33 created 1972' WHERE ec = 'EC 1.2.1.33';
UPDATE enzyme SET history = 'EC 1.2.1.34 created 1972, deleted 1983 (transferred to EC 1.1.1.180, deleted 1984)' WHERE ec = 'EC 1.2.1.34';
UPDATE enzyme SET history = 'EC 1.2.1.35 created 1972, deleted 1984' WHERE ec = 'EC 1.2.1.35';
UPDATE enzyme SET sys_name = 'Retinal:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.36';
UPDATE enzyme SET history = 'EC 1.2.1.36 created 1972' WHERE ec = 'EC 1.2.1.36';
UPDATE enzyme SET history = 'EC 1.2.1.37 created 1972, deleted 1984' WHERE ec = 'EC 1.2.1.37';
UPDATE enzyme SET sys_name = 'N-acetyl-L-glutamate-5-semialdehyde:NADP(+) 5-oxidoreductase (phosphorylating)' WHERE ec = 'EC 1.2.1.38';
UPDATE enzyme SET history = 'EC 1.2.1.38 created 1972' WHERE ec = 'EC 1.2.1.38';
UPDATE enzyme SET sys_name = 'Phenylacetaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.39';
UPDATE enzyme SET history = 'EC 1.2.1.39 created 1976' WHERE ec = 'EC 1.2.1.39';
UPDATE enzyme SET history = 'EC 1.2.1.40 created 1976, deleted 2012' WHERE ec = 'EC 1.2.1.40';
UPDATE enzyme SET sys_name = 'L-glutamate-5-semialdehyde:NADP(+) 5-oxidoreductase (phosphorylating)' WHERE ec = 'EC 1.2.1.41';
UPDATE enzyme SET history = 'EC 1.2.1.41 created 1976' WHERE ec = 'EC 1.2.1.41';
UPDATE enzyme SET sys_name = 'Hexadecanal:NAD(+) oxidoreductase (CoA-acylating)' WHERE ec = 'EC 1.2.1.42';
UPDATE enzyme SET history = 'EC 1.2.1.42 created 1978' WHERE ec = 'EC 1.2.1.42';
UPDATE enzyme SET sys_name = 'Formate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.2.1.43';
UPDATE enzyme SET history = 'EC 1.2.1.43 created 1978' WHERE ec = 'EC 1.2.1.43';
UPDATE enzyme SET sys_name = 'Cinnamaldehyde:NADP(+) oxidoreductase (CoA-cinnamoylating)' WHERE ec = 'EC 1.2.1.44';
UPDATE enzyme SET history = 'EC 1.2.1.44 created 1978' WHERE ec = 'EC 1.2.1.44';
UPDATE enzyme SET history = 'EC 1.2.1.45 created 1978, deleted 2011' WHERE ec = 'EC 1.2.1.45';
UPDATE enzyme SET sys_name = 'Formaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.46';
UPDATE enzyme SET history = 'EC 1.2.1.46 created 1982' WHERE ec = 'EC 1.2.1.46';
UPDATE enzyme SET sys_name = '4-trimethylammoniobutanal:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.2.1.47';
UPDATE enzyme SET history = 'EC 1.2.1.47 created 1983' WHERE ec = 'EC 1.2.1.47';
UPDATE enzyme SET sys_name = 'Long-chain-aldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.48';
UPDATE enzyme SET history = 'EC 1.2.1.48 created 1984' WHERE ec = 'EC 1.2.1.48';
UPDATE enzyme SET sys_name = '2-oxoaldehyde:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.2.1.49';
UPDATE enzyme SET history = 'EC 1.2.1.49 created 1986' WHERE ec = 'EC 1.2.1.49';
UPDATE enzyme SET sys_name = 'Long-chain-aldehyde:NADP(+) oxidoreductase (acyl-CoA-forming)' WHERE ec = 'EC 1.2.1.50';
UPDATE enzyme SET history = 'EC 1.2.1.50 created 1986' WHERE ec = 'EC 1.2.1.50';
UPDATE enzyme SET sys_name = 'Pyruvate:NADP(+) 2-oxidoreductase (CoA-acetylating)' WHERE ec = 'EC 1.2.1.51';
UPDATE enzyme SET history = 'EC 1.2.1.51 created 1989' WHERE ec = 'EC 1.2.1.51';
UPDATE enzyme SET sys_name = '2-oxoglutarate:NADP(+) 2-oxidoreductase (CoA-succinylating)' WHERE ec = 'EC 1.2.1.52';
UPDATE enzyme SET history = 'EC 1.2.1.52 created 1989' WHERE ec = 'EC 1.2.1.52';
UPDATE enzyme SET sys_name = '4-hydroxyphenylacetaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.53';
UPDATE enzyme SET history = 'EC 1.2.1.53 created 1989' WHERE ec = 'EC 1.2.1.53';
UPDATE enzyme SET sys_name = '4-guanidinobutanal:NAD(+) 1-oxidoreductase' WHERE ec = 'EC 1.2.1.54';
UPDATE enzyme SET history = 'EC 1.2.1.54 created 1989' WHERE ec = 'EC 1.2.1.54';
UPDATE enzyme SET history = 'EC 1.2.1.55 created 1990, deleted 2003' WHERE ec = 'EC 1.2.1.55';
UPDATE enzyme SET history = 'EC 1.2.1.56 created 1990, deleted 2003' WHERE ec = 'EC 1.2.1.56';
UPDATE enzyme SET sys_name = 'Butanal:NAD(P)(+) oxidoreductase (CoA-acylating)' WHERE ec = 'EC 1.2.1.57';
UPDATE enzyme SET history = 'EC 1.2.1.57 created 1992' WHERE ec = 'EC 1.2.1.57';
UPDATE enzyme SET sys_name = 'Phenylglyoxylate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.58';
UPDATE enzyme SET history = 'EC 1.2.1.58 created 1999' WHERE ec = 'EC 1.2.1.58';
UPDATE enzyme SET sys_name = 'D-glyceraldehyde 3-phosphate:NAD(P)(+) oxidoreductase (phosphorylating)' WHERE ec = 'EC 1.2.1.59';
UPDATE enzyme SET history = 'EC 1.2.1.59 created 1999' WHERE ec = 'EC 1.2.1.59';
UPDATE enzyme SET sys_name = '5-carboxymethyl-2-hydroxymuconic-semialdehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.60';
UPDATE enzyme SET history = 'EC 1.2.1.60 created 2000' WHERE ec = 'EC 1.2.1.60';
UPDATE enzyme SET sys_name = '4-hydroxymuconic-semialdehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.61';
UPDATE enzyme SET history = 'EC 1.2.1.61 created 2000' WHERE ec = 'EC 1.2.1.61';
UPDATE enzyme SET sys_name = '4-formylbenzenesulfonate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.62';
UPDATE enzyme SET history = 'EC 1.2.1.62 created 2000' WHERE ec = 'EC 1.2.1.62';
UPDATE enzyme SET sys_name = '6-oxohexanoate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.2.1.63';
UPDATE enzyme SET history = 'EC 1.2.1.63 created 2000' WHERE ec = 'EC 1.2.1.63';
UPDATE enzyme SET sys_name = '3-hydroxybenzaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.64';
UPDATE enzyme SET history = 'EC 1.2.1.64 created 2000' WHERE ec = 'EC 1.2.1.64';
UPDATE enzyme SET sys_name = 'Salicylaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.65';
UPDATE enzyme SET history = 'EC 1.2.1.65 created 2000' WHERE ec = 'EC 1.2.1.65';
UPDATE enzyme SET history = 'EC 1.2.1.66 created 2000, deleted 2010' WHERE ec = 'EC 1.2.1.66';
UPDATE enzyme SET sys_name = 'Vanillin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.67';
UPDATE enzyme SET history = 'EC 1.2.1.67 created 2000' WHERE ec = 'EC 1.2.1.67';
UPDATE enzyme SET sys_name = 'Coniferyl aldehyde:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.2.1.68';
UPDATE enzyme SET history = 'EC 1.2.1.68 created 2000' WHERE ec = 'EC 1.2.1.68';
UPDATE enzyme SET sys_name = 'Fluoroacetaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.69';
UPDATE enzyme SET history = 'EC 1.2.1.69 created 2003' WHERE ec = 'EC 1.2.1.69';
UPDATE enzyme SET sys_name = 'L-glutamate-semialdehyde: NADP(+) oxidoreductase (L-glutamyl-tRNA(Glu)-forming)' WHERE ec = 'EC 1.2.1.70';
UPDATE enzyme SET history = 'EC 1.2.1.70 created 2004' WHERE ec = 'EC 1.2.1.70';
UPDATE enzyme SET sys_name = 'N-succinyl-L-glutamate 5-semialdehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.71';
UPDATE enzyme SET history = 'EC 1.2.1.71 created 2006' WHERE ec = 'EC 1.2.1.71';
UPDATE enzyme SET sys_name = 'D-erythrose 4-phosphate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.72';
UPDATE enzyme SET history = 'EC 1.2.1.72 created 2006' WHERE ec = 'EC 1.2.1.72';
UPDATE enzyme SET sys_name = '2-sulfoacetaldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.73';
UPDATE enzyme SET history = 'EC 1.2.1.73 created 2008' WHERE ec = 'EC 1.2.1.73';
UPDATE enzyme SET sys_name = 'Abieta-7,13-dien-18-al:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.74';
UPDATE enzyme SET history = 'EC 1.2.1.74 created 2009, modified 2012' WHERE ec = 'EC 1.2.1.74';
UPDATE enzyme SET sys_name = 'Malonate semialdehyde:NADP(+) oxidoreductase (malonate semialdehyde-forming)' WHERE ec = 'EC 1.2.1.75';
UPDATE enzyme SET history = 'EC 1.2.1.75 created 2009' WHERE ec = 'EC 1.2.1.75';
UPDATE enzyme SET sys_name = 'Succinate semialdehyde:NADP(+) oxidoreductase (CoA-acetylating)' WHERE ec = 'EC 1.2.1.76';
UPDATE enzyme SET history = 'EC 1.2.1.76 created 2009' WHERE ec = 'EC 1.2.1.76';
UPDATE enzyme SET sys_name = '3,4-dehydroadipyl-CoA semialdehyde:NADP(+) oxidoreductase' WHERE ec = 'EC 1.2.1.77';
UPDATE enzyme SET history = 'EC 1.2.1.77 created 2010' WHERE ec = 'EC 1.2.1.77';
UPDATE enzyme SET sys_name = '2-formylbenzoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.78';
UPDATE enzyme SET history = 'EC 1.2.1.78 created 2010' WHERE ec = 'EC 1.2.1.78';
UPDATE enzyme SET sys_name = 'Succinate-semialdehyde:NADP(+) oxidoreductase' WHERE ec = 'EC 1.2.1.79';
UPDATE enzyme SET history = 'EC 1.2.1.79 created 2010' WHERE ec = 'EC 1.2.1.79';
UPDATE enzyme SET sys_name = 'Long-chain-aldehyde:NAD(P)(+) oxidoreductase (acyl-[acyl-carrier protein]-forming)' WHERE ec = 'EC 1.2.1.80';
UPDATE enzyme SET history = 'EC 2.1.2.80 created 2010' WHERE ec = 'EC 1.2.1.80';
UPDATE enzyme SET sys_name = '2-sulfoacetaldehyde:NADP(+) oxidoreductase (CoA-acetylating)' WHERE ec = 'EC 1.2.1.81';
UPDATE enzyme SET history = 'EC 1.2.1.81 created 2011' WHERE ec = 'EC 1.2.1.81';
UPDATE enzyme SET sys_name = '4-apo-beta,psi-carotenal:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.82';
UPDATE enzyme SET history = 'EC 1.2.1.82 created 2011' WHERE ec = 'EC 1.2.1.82';
UPDATE enzyme SET sys_name = '4-oxo-4-(pyridin-3-yl)butanal:NADP(+) oxidoreductase' WHERE ec = 'EC 1.2.1.83';
UPDATE enzyme SET history = 'EC 1.2.1.83 created 2012' WHERE ec = 'EC 1.2.1.83';
UPDATE enzyme SET sys_name = 'Long-chain acyl-CoA:NADPH reductase' WHERE ec = 'EC 1.2.1.84';
UPDATE enzyme SET history = 'EC 1.2.1.84 created 2012' WHERE ec = 'EC 1.2.1.84';
UPDATE enzyme SET sys_name = '2-hydroxymuconate-6-semialdehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.85';
UPDATE enzyme SET history = 'EC 1.2.1.85 created 2012' WHERE ec = 'EC 1.2.1.85';
UPDATE enzyme SET sys_name = 'Geranial:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.86';
UPDATE enzyme SET history = 'EC 1.2.1.86 created 2012' WHERE ec = 'EC 1.2.1.86';
UPDATE enzyme SET sys_name = 'Propanal:NAD(+) oxidoreductase (CoA-propanoylating)' WHERE ec = 'EC 1.2.1.87';
UPDATE enzyme SET history = 'EC 1.2.1.87 created 2013' WHERE ec = 'EC 1.2.1.87';
UPDATE enzyme SET sys_name = 'L-glutamate gamma-semialdehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.2.1.88';
UPDATE enzyme SET history = 'EC 1.2.1.88 created 1972 as 1.5.1.12, modified 2008, transferred 2013 to EC 1.2.1.88' WHERE ec = 'EC 1.2.1.88';
UPDATE enzyme SET sys_name = 'D-glyceraldehyde:NADP(+) oxidoreductase' WHERE ec = 'EC 1.2.1.89';
UPDATE enzyme SET history = 'EC 1.2.1.89 created 2014' WHERE ec = 'EC 1.2.1.89';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.2.1.n1';
UPDATE enzyme SET sys_name = 'Hexadecanal:NADP(+) oxidoreductase (CoA-acylating)' WHERE ec = 'EC 1.2.1.n2';
UPDATE enzyme SET history = 'EC 1.2.1.n2 created 2010' WHERE ec = 'EC 1.2.1.n2';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.2.1.n3';
UPDATE enzyme SET sys_name = 'Formate:ferricytochrome-b1 oxidoreductase' WHERE ec = 'EC 1.2.2.1';
UPDATE enzyme SET history = 'EC 1.2.2.1 created 1961' WHERE ec = 'EC 1.2.2.1';
UPDATE enzyme SET history = 'EC 1.2.2.2 created 1961, deleted 2010' WHERE ec = 'EC 1.2.2.2';
UPDATE enzyme SET sys_name = 'Formate:ferricytochrome-c-553 oxidoreductase' WHERE ec = 'EC 1.2.2.3';
UPDATE enzyme SET history = 'EC 1.2.2.3 created 1981' WHERE ec = 'EC 1.2.2.3';
UPDATE enzyme SET sys_name = 'Carbon monoxide,water:cytochrome b-561 oxidoreductase' WHERE ec = 'EC 1.2.2.4';
UPDATE enzyme SET history = 'EC 1.2.2.4 created 1999, modified 2003' WHERE ec = 'EC 1.2.2.4';
UPDATE enzyme SET sys_name = 'Aldehyde:oxygen oxidoreductase' WHERE ec = 'EC 1.2.3.1';
UPDATE enzyme SET history = 'EC 1.2.3.1 created 1961, modified 2002, modified 2004, modified 2012' WHERE ec = 'EC 1.2.3.1';
UPDATE enzyme SET history = 'EC 1.2.3.2 created 1961, deleted 1984' WHERE ec = 'EC 1.2.3.2';
UPDATE enzyme SET sys_name = 'Pyruvate:oxygen 2-oxidoreductase (phosphorylating)' WHERE ec = 'EC 1.2.3.3';
UPDATE enzyme SET history = 'EC 1.2.3.3 created 1961, modified 2006' WHERE ec = 'EC 1.2.3.3';
UPDATE enzyme SET sys_name = 'Oxalate:oxygen oxidoreductase' WHERE ec = 'EC 1.2.3.4';
UPDATE enzyme SET history = 'EC 1.2.3.4 created 1961' WHERE ec = 'EC 1.2.3.4';
UPDATE enzyme SET sys_name = 'Glyoxylate:oxygen oxidoreductase' WHERE ec = 'EC 1.2.3.5';
UPDATE enzyme SET history = 'EC 1.2.3.5 created 1972' WHERE ec = 'EC 1.2.3.5';
UPDATE enzyme SET sys_name = 'Pyruvate:oxygen 2-oxidoreductase (CoA-acetylating)' WHERE ec = 'EC 1.2.3.6';
UPDATE enzyme SET history = 'EC 1.2.3.6 created 1976' WHERE ec = 'EC 1.2.3.6';
UPDATE enzyme SET sys_name = '(Indol-3-yl)acetaldehyde:oxygen oxidoreductase' WHERE ec = 'EC 1.2.3.7';
UPDATE enzyme SET history = 'EC 1.2.3.7 created 1984, modified 2004, modified 2006' WHERE ec = 'EC 1.2.3.7';
UPDATE enzyme SET sys_name = 'Pyridoxal:oxygen 4-oxidoreductase' WHERE ec = 'EC 1.2.3.8';
UPDATE enzyme SET history = 'EC 1.2.3.8 created 1984' WHERE ec = 'EC 1.2.3.8';
UPDATE enzyme SET sys_name = 'Aryl-aldehyde:oxygen oxidoreductase' WHERE ec = 'EC 1.2.3.9';
UPDATE enzyme SET history = 'EC 1.2.3.9 created 1986, modified 2002' WHERE ec = 'EC 1.2.3.9';
UPDATE enzyme SET history = 'EC 1.2.3.10 created 1990, deleted 2003' WHERE ec = 'EC 1.2.3.10';
UPDATE enzyme SET history = 'EC 1.2.3.11 created 1990, modified 2002, deleted 2012' WHERE ec = 'EC 1.2.3.11';
UPDATE enzyme SET history = 'EC 1.2.3.12 created 2000, deleted 2003' WHERE ec = 'EC 1.2.3.12';
UPDATE enzyme SET sys_name = '4-hydroxyphenylpyruvate:oxygen oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.2.3.13';
UPDATE enzyme SET history = 'EC 1.2.3.13 created 2000' WHERE ec = 'EC 1.2.3.13';
UPDATE enzyme SET sys_name = 'Abscisic-aldehyde:oxygen oxidoreductase' WHERE ec = 'EC 1.2.3.14';
UPDATE enzyme SET history = 'EC 1.2.3.14 created 2005' WHERE ec = 'EC 1.2.3.14';
UPDATE enzyme SET sys_name = 'Pyruvate:[dihydrolipoyllysine-residue acetyltransferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-acetylating)' WHERE ec = 'EC 1.2.4.1';
UPDATE enzyme SET history = 'EC 1.2.4.1 created 1961, modified 2003' WHERE ec = 'EC 1.2.4.1';
UPDATE enzyme SET sys_name = '2-oxoglutarate:[dihydrolipoyllysine-residue succinyltransferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-succinylating)' WHERE ec = 'EC 1.2.4.2';
UPDATE enzyme SET history = 'EC 1.2.4.2 created 1961, modified 1980, modified 1986, modified 2003' WHERE ec = 'EC 1.2.4.2';
UPDATE enzyme SET history = 'EC 1.2.4.3 created 1972, deleted 1978' WHERE ec = 'EC 1.2.4.3';
UPDATE enzyme SET sys_name = '3-methyl-2-oxobutanoate:[dihydrolipoyllysine-residue (2-methylpropanoyl)transferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-2-methylpropanoylating)' WHERE ec = 'EC 1.2.4.4';
UPDATE enzyme SET history = 'EC 1.2.4.4 created 1972 (EC 1.2.4.3 created 1972, incorporated 1978), modified 2003' WHERE ec = 'EC 1.2.4.4';
UPDATE enzyme SET sys_name = 'Pyruvate:ubiquinone oxidoreductase' WHERE ec = 'EC 1.2.5.1';
UPDATE enzyme SET history = 'EC 1.2.5.1 created 2010' WHERE ec = 'EC 1.2.5.1';
UPDATE enzyme SET sys_name = 'Pyruvate:ferredoxin 2-oxidoreductase (CoA-acetylating)' WHERE ec = 'EC 1.2.7.1';
UPDATE enzyme SET history = 'EC 1.2.7.1 created 1972, modified 2003, modified 2013' WHERE ec = 'EC 1.2.7.1';
UPDATE enzyme SET history = 'EC 1.2.7.2 created 1972, deleted 2013' WHERE ec = 'EC 1.2.7.2';
UPDATE enzyme SET sys_name = '2-oxoglutarate:ferredoxin oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.2.7.3';
UPDATE enzyme SET history = 'EC 1.2.7.3 created 1972, modified 2005' WHERE ec = 'EC 1.2.7.3';
UPDATE enzyme SET sys_name = 'Carbon-monoxide,water:ferredoxin oxidoreductase' WHERE ec = 'EC 1.2.7.4';
UPDATE enzyme SET history = 'EC 1.2.7.4 created 2003' WHERE ec = 'EC 1.2.7.4';
UPDATE enzyme SET sys_name = 'Aldehyde:ferredoxin oxidoreductase' WHERE ec = 'EC 1.2.7.5';
UPDATE enzyme SET history = 'EC 1.2.7.5 created 2003' WHERE ec = 'EC 1.2.7.5';
UPDATE enzyme SET sys_name = 'D-glyceraldehyde-3-phosphate:ferredoxin oxidoreductase' WHERE ec = 'EC 1.2.7.6';
UPDATE enzyme SET history = 'EC 1.2.7.6 created 2003' WHERE ec = 'EC 1.2.7.6';
UPDATE enzyme SET sys_name = '3-methyl-2-oxobutanoate:ferredoxin oxidoreductase (decarboxylating; CoA-2-methylpropanoylating)' WHERE ec = 'EC 1.2.7.7';
UPDATE enzyme SET history = 'EC 1.2.7.7 created 2003' WHERE ec = 'EC 1.2.7.7';
UPDATE enzyme SET sys_name = '3-(indol-3-yl)pyruvate:ferredoxin oxidoreductase (decarboxylating, CoA-indole-acetylating)' WHERE ec = 'EC 1.2.7.8';
UPDATE enzyme SET history = 'EC 1.2.7.8 created 2003' WHERE ec = 'EC 1.2.7.8';
UPDATE enzyme SET history = 'EC 1.2.7.9 created 2003, deleted 2005' WHERE ec = 'EC 1.2.7.9';
UPDATE enzyme SET sys_name = 'Oxalate:ferredoxin oxidoreductase' WHERE ec = 'EC 1.2.7.10';
UPDATE enzyme SET history = 'EC 1.2.7.10 created 2011' WHERE ec = 'EC 1.2.7.10';
UPDATE enzyme SET sys_name = '2-oxocarboxylate:ferredoxin 2-oxidoreductase (decarboxylating, CoA-acylating)' WHERE ec = 'EC 1.2.7.11';
UPDATE enzyme SET history = 'EC 1.2.7.11 created 2013' WHERE ec = 'EC 1.2.7.11';
UPDATE enzyme SET history = 'EC 1.2.99.1 created 1961, deleted 1984' WHERE ec = 'EC 1.2.99.1';
UPDATE enzyme SET sys_name = 'Carbon-monoxide:acceptor oxidoreductase' WHERE ec = 'EC 1.2.99.2';
UPDATE enzyme SET history = 'EC 1.2.99.2 created 1982, modified 1990, modified 2003' WHERE ec = 'EC 1.2.99.2';
UPDATE enzyme SET sys_name = 'Aldehyde:(pyrroloquinoline-quinone) oxidoreductase' WHERE ec = 'EC 1.2.99.3';
UPDATE enzyme SET history = 'EC 1.2.99.3 created 1983, modified 1989' WHERE ec = 'EC 1.2.99.3';
UPDATE enzyme SET sys_name = 'Formaldehyde:formaldehyde oxidoreductase' WHERE ec = 'EC 1.2.99.4';
UPDATE enzyme SET history = 'EC 1.2.99.4 created 1986, modified 2012' WHERE ec = 'EC 1.2.99.4';
UPDATE enzyme SET sys_name = 'Formylmethanofuran:acceptor oxidoreductase' WHERE ec = 'EC 1.2.99.5';
UPDATE enzyme SET history = 'EC 1.2.99.5 created 1992' WHERE ec = 'EC 1.2.99.5';
UPDATE enzyme SET sys_name = 'Aldehyde:acceptor oxidoreductase' WHERE ec = 'EC 1.2.99.6';
UPDATE enzyme SET history = 'EC 1.2.99.6 created 1992' WHERE ec = 'EC 1.2.99.6';
UPDATE enzyme SET sys_name = 'Aldehyde:acceptor oxidoreductase (FAD-independent)' WHERE ec = 'EC 1.2.99.7';
UPDATE enzyme SET history = 'EC 1.2.99.7 created 2004' WHERE ec = 'EC 1.2.99.7';
UPDATE enzyme SET sys_name = 'D-glyceraldehyde:acceptor oxidoreductase (FAD-containing)' WHERE ec = 'EC 1.2.99.8';
UPDATE enzyme SET history = 'EC 1.2.99.8 created 2013' WHERE ec = 'EC 1.2.99.8';
UPDATE enzyme SET sys_name = '5,6-dihydrouracil:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.1';
UPDATE enzyme SET history = 'EC 1.3.1.1 created 1961, modified 2011' WHERE ec = 'EC 1.3.1.1';
UPDATE enzyme SET sys_name = '5,6-dihydrouracil:NADP(+) 5-oxidoreductase' WHERE ec = 'EC 1.3.1.2';
UPDATE enzyme SET history = 'EC 1.3.1.2 created 1961, modified 1986' WHERE ec = 'EC 1.3.1.2';
UPDATE enzyme SET sys_name = '5-beta-cholestan-3-one:NADP(+) 4,5-oxidoreductase' WHERE ec = 'EC 1.3.1.3';
UPDATE enzyme SET history = 'EC 1.3.1.3 created 1961 (EC 1.3.1.23 created 1972, incoporated 2005), modified 2005' WHERE ec = 'EC 1.3.1.3';
UPDATE enzyme SET history = 'EC 1.3.1.4 created 1965' WHERE ec = 'EC 1.3.1.4';
UPDATE enzyme SET sys_name = '23,24-dihydrocucurbitacin:NAD(P)(+) Delta(23)-oxidoreductase' WHERE ec = 'EC 1.3.1.5';
UPDATE enzyme SET history = 'EC 1.3.1.5 created 1965' WHERE ec = 'EC 1.3.1.5';
UPDATE enzyme SET sys_name = 'Succinate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.6';
UPDATE enzyme SET history = 'EC 1.3.1.6 created 1972' WHERE ec = 'EC 1.3.1.6';
UPDATE enzyme SET sys_name = 'Meso-tartrate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.7';
UPDATE enzyme SET history = 'EC 1.3.1.7 created 1972' WHERE ec = 'EC 1.3.1.7';
UPDATE enzyme SET sys_name = 'Acyl-CoA:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.3.1.8';
UPDATE enzyme SET history = 'EC 1.3.1.8 created 1972, modified 1986' WHERE ec = 'EC 1.3.1.8';
UPDATE enzyme SET sys_name = 'Acyl-[acyl-carrier protein]:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.9';
UPDATE enzyme SET history = 'EC 1.3.1.9 created 1972, modified 2013' WHERE ec = 'EC 1.3.1.9';
UPDATE enzyme SET sys_name = 'Acyl-[acyl-carrier-protein]:NADP(+) oxidoreductase (Si-specific)' WHERE ec = 'EC 1.3.1.10';
UPDATE enzyme SET history = 'EC 1.3.1.10 created 1972, modified 1986, modified 2013' WHERE ec = 'EC 1.3.1.10';
UPDATE enzyme SET sys_name = '3-(2-hydroxyphenyl)propanoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.11';
UPDATE enzyme SET history = 'EC 1.3.1.11 created 1972' WHERE ec = 'EC 1.3.1.11';
UPDATE enzyme SET sys_name = 'Prephenate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.1.12';
UPDATE enzyme SET history = 'EC 1.3.1.12 created 1972' WHERE ec = 'EC 1.3.1.12';
UPDATE enzyme SET sys_name = 'Prephenate:NADP(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.1.13';
UPDATE enzyme SET history = 'EC 1.3.1.13 created 1972' WHERE ec = 'EC 1.3.1.13';
UPDATE enzyme SET sys_name = '(S)-dihydroorotate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.14';
UPDATE enzyme SET history = 'EC 1.3.1.14 created 1972, modified 2011' WHERE ec = 'EC 1.3.1.14';
UPDATE enzyme SET sys_name = '(S)-dihydroorotate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.15';
UPDATE enzyme SET history = 'EC 1.3.1.15 created 1972, modified 2011' WHERE ec = 'EC 1.3.1.15';
UPDATE enzyme SET sys_name = '3-nitropropanoate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.16';
UPDATE enzyme SET history = 'EC 1.3.1.16 created 1972' WHERE ec = 'EC 1.3.1.16';
UPDATE enzyme SET sys_name = '3-methyl-1,3-dihydroindol-2-one:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.17';
UPDATE enzyme SET history = 'EC 1.3.1.17 created 1972' WHERE ec = 'EC 1.3.1.17';
UPDATE enzyme SET sys_name = '7,8-dihydro-7,8-dihydroxykynurenate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.18';
UPDATE enzyme SET history = 'EC 1.3.1.18 created 1972' WHERE ec = 'EC 1.3.1.18';
UPDATE enzyme SET sys_name = 'Cis-1,2-dihydrobenzene-1,2-diol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.19';
UPDATE enzyme SET history = 'EC 1.3.1.19 created 1972' WHERE ec = 'EC 1.3.1.19';
UPDATE enzyme SET sys_name = 'Trans-1,2-dihydrobenzene-1,2-diol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.20';
UPDATE enzyme SET history = 'EC 1.3.1.20 created 1972' WHERE ec = 'EC 1.3.1.20';
UPDATE enzyme SET sys_name = 'Cholesterol:NADP(+) Delta(7)-oxidoreductase' WHERE ec = 'EC 1.3.1.21';
UPDATE enzyme SET history = 'EC 1.3.1.21 created 1972, modified 2013' WHERE ec = 'EC 1.3.1.21';
UPDATE enzyme SET sys_name = '3-oxo-5-alpha-steroid:NADP(+) Delta(4)-oxidoreductase' WHERE ec = 'EC 1.3.1.22';
UPDATE enzyme SET history = 'EC 1.3.1.22 created 1972, modified 2012' WHERE ec = 'EC 1.3.1.22';
UPDATE enzyme SET history = 'EC 1.3.1.23 created 1972, deleted 2005' WHERE ec = 'EC 1.3.1.23';
UPDATE enzyme SET sys_name = 'Bilirubin:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.3.1.24';
UPDATE enzyme SET history = 'EC 1.3.1.24 created 1972' WHERE ec = 'EC 1.3.1.24';
UPDATE enzyme SET sys_name = '(1R,6S)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.1.25';
UPDATE enzyme SET history = 'EC 1.3.1.25 created 1976, modified 2004 (EC 1.3.1.55 created 1999, incorporated 2004)' WHERE ec = 'EC 1.3.1.25';
UPDATE enzyme SET history = 'EC 1.3.1.26 created 1976, modified 2011, deleted 2013' WHERE ec = 'EC 1.3.1.26';
UPDATE enzyme SET sys_name = 'Hexadecanal:NADP(+) Delta(2)-oxidoreductase' WHERE ec = 'EC 1.3.1.27';
UPDATE enzyme SET history = 'EC 1.3.1.27 created 1976' WHERE ec = 'EC 1.3.1.27';
UPDATE enzyme SET sys_name = '(2S, 3S)-2,3-dihydro-2,3-dihydroxybenzoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.28';
UPDATE enzyme SET history = 'EC 1.3.1.28 created 1972 as EC 1.1.1.109, transferred 1976 to EC 1.3.1.28' WHERE ec = 'EC 1.3.1.28';
UPDATE enzyme SET sys_name = 'Cis-1,2-dihydronaphthalene-1,2-diol:NAD(+) 1,2-oxidoreductase' WHERE ec = 'EC 1.3.1.29';
UPDATE enzyme SET history = 'EC 1.3.1.29 created 1976' WHERE ec = 'EC 1.3.1.29';
UPDATE enzyme SET history = 'EC 1.3.1.30 created 1978' WHERE ec = 'EC 1.3.1.30';
UPDATE enzyme SET sys_name = 'Butanoate:NAD(+) Delta(2)-oxidoreductase' WHERE ec = 'EC 1.3.1.31';
UPDATE enzyme SET history = 'EC 1.3.1.31 created 1982' WHERE ec = 'EC 1.3.1.31';
UPDATE enzyme SET sys_name = '3-oxoadipate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.3.1.32';
UPDATE enzyme SET history = 'EC 1.3.1.32 created 1983' WHERE ec = 'EC 1.3.1.32';
UPDATE enzyme SET sys_name = 'Chlorophyllide-a:NADP(+) 7,8-oxidoreductase' WHERE ec = 'EC 1.3.1.33';
UPDATE enzyme SET history = 'EC 1.3.1.33 created 1984' WHERE ec = 'EC 1.3.1.33';
UPDATE enzyme SET sys_name = 'Trans-2,3-didehydroacyl-CoA:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.3.1.34';
UPDATE enzyme SET history = 'EC 1.3.1.34 created 1984, modified 1986' WHERE ec = 'EC 1.3.1.34';
UPDATE enzyme SET sys_name = '1-acyl-2-oleoyl-sn-glycero-3-phosphocholine:NAD(+) Delta(12)-oxidoreductase' WHERE ec = 'EC 1.3.1.35';
UPDATE enzyme SET history = 'EC 1.3.1.35 created 1984' WHERE ec = 'EC 1.3.1.35';
UPDATE enzyme SET sys_name = 'Geissoschizine:NADP(+) 4,21-oxidoreductase' WHERE ec = 'EC 1.3.1.36';
UPDATE enzyme SET history = 'EC 1.3.1.36 created 1986' WHERE ec = 'EC 1.3.1.36';
UPDATE enzyme SET sys_name = 'Acyl-CoA:NADP(+) cis-2-oxidoreductase' WHERE ec = 'EC 1.3.1.37';
UPDATE enzyme SET history = 'EC 1.3.1.37 created 1986' WHERE ec = 'EC 1.3.1.37';
UPDATE enzyme SET sys_name = 'Acyl-CoA:NADP(+) trans-2-oxidoreductase' WHERE ec = 'EC 1.3.1.38';
UPDATE enzyme SET history = 'EC 1.3.1.38 created 1986' WHERE ec = 'EC 1.3.1.38';
UPDATE enzyme SET sys_name = 'Acyl-[acyl-carrier protein]:NADP(+) oxidoreductase (Re-specific)' WHERE ec = 'EC 1.3.1.39';
UPDATE enzyme SET history = 'EC 1.3.1.39 created 1986, modified 2013' WHERE ec = 'EC 1.3.1.39';
UPDATE enzyme SET sys_name = '2,6-dioxo-6-phenylhexanoate:NADP(+) Delta(2)-oxidoreductase' WHERE ec = 'EC 1.3.1.40';
UPDATE enzyme SET history = 'EC 1.3.1.40 created 1989' WHERE ec = 'EC 1.3.1.40';
UPDATE enzyme SET sys_name = '5,12-dihydroxanthommatin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.41';
UPDATE enzyme SET history = 'EC 1.3.1.41 created 1989' WHERE ec = 'EC 1.3.1.41';
UPDATE enzyme SET sys_name = '8-((1R,2R)-3-oxo-2-((Z)-pent-2-enyl)cyclopentyl)octanoate:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.3.1.42';
UPDATE enzyme SET history = 'EC 1.3.1.42 created 1989' WHERE ec = 'EC 1.3.1.42';
UPDATE enzyme SET sys_name = 'L-arogenate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.1.43';
UPDATE enzyme SET history = 'EC 1.3.1.43 created 1989, modified 2003, modified 2005' WHERE ec = 'EC 1.3.1.43';
UPDATE enzyme SET sys_name = 'Acyl-CoA:NAD(+) trans-2-oxidoreductase' WHERE ec = 'EC 1.3.1.44';
UPDATE enzyme SET history = 'EC 1.3.1.44 created 1989' WHERE ec = 'EC 1.3.1.44';
UPDATE enzyme SET sys_name = 'Vestitone:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.45';
UPDATE enzyme SET history = 'EC 1.3.1.45 created 1990' WHERE ec = 'EC 1.3.1.45';
UPDATE enzyme SET sys_name = 'Dihydrobiochanin-A:NADP(+) Delta(2)-oxidoreductase' WHERE ec = 'EC 1.3.1.46';
UPDATE enzyme SET history = 'EC 1.3.1.46 created 1990' WHERE ec = 'EC 1.3.1.46';
UPDATE enzyme SET sys_name = '1,2-dihydrosantonin:NAD(P)(+) 1,2-oxidoreductase' WHERE ec = 'EC 1.3.1.47';
UPDATE enzyme SET history = 'EC 1.3.1.47 created 1990' WHERE ec = 'EC 1.3.1.47';
UPDATE enzyme SET sys_name = '11-alpha-hydroxy-9,15-dioxoprostanoate:NAD(P)(+) Delta(13)-oxidoreductase' WHERE ec = 'EC 1.3.1.48';
UPDATE enzyme SET history = 'EC 1.3.1.48 created 1990' WHERE ec = 'EC 1.3.1.48';
UPDATE enzyme SET sys_name = '(+)-cis-3,4-dihydrophenanthrene-3,4-diol:NAD(+) 3,4-oxidoreductase' WHERE ec = 'EC 1.3.1.49';
UPDATE enzyme SET history = 'EC 1.3.1.49 created 1992' WHERE ec = 'EC 1.3.1.49';
UPDATE enzyme SET history = 'EC 1.3.1.50 created 1992, deleted 1999' WHERE ec = 'EC 1.3.1.50';
UPDATE enzyme SET sys_name = '2-hydroxy-2,3-dihydrodaidzein:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.3.1.51';
UPDATE enzyme SET history = 'EC 1.3.1.51 created 1992, modified 2004' WHERE ec = 'EC 1.3.1.51';
UPDATE enzyme SET history = 'EC 1.3.1.52 created 1992, deleted 2012' WHERE ec = 'EC 1.3.1.52';
UPDATE enzyme SET sys_name = '(3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.53';
UPDATE enzyme SET history = 'EC 1.3.1.53 created 1999 (EC 1.3.1.61 created 2000, incorporated 2007)' WHERE ec = 'EC 1.3.1.53';
UPDATE enzyme SET sys_name = 'Precorrin-6B:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.54';
UPDATE enzyme SET history = 'EC 1.3.1.54 created 1999, modified 2004' WHERE ec = 'EC 1.3.1.54';
UPDATE enzyme SET history = 'EC 1.3.1.55 created 1999, deleted 2004' WHERE ec = 'EC 1.3.1.55';
UPDATE enzyme SET sys_name = 'Cis-3-phenylcyclohexa-3,5-diene-1,2-diol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.56';
UPDATE enzyme SET history = 'EC 1.3.1.56 created 2000' WHERE ec = 'EC 1.3.1.56';
UPDATE enzyme SET sys_name = 'Dihydrophloroglucinol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.57';
UPDATE enzyme SET history = 'EC 1.3.1.57 created 2000' WHERE ec = 'EC 1.3.1.57';
UPDATE enzyme SET sys_name = 'Cis-2,3-dihydroxy-2,3-dihydro-p-cumate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.58';
UPDATE enzyme SET history = 'EC 1.3.1.58 created 2000' WHERE ec = 'EC 1.3.1.58';
UPDATE enzyme SET history = 'EC 1.3.1.59 created 2000, modified 2005, deleted 2006' WHERE ec = 'EC 1.3.1.59';
UPDATE enzyme SET sys_name = 'Cis-1,2-dihydroxy-1,2-dihydrodibenzothiophene:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.60';
UPDATE enzyme SET history = 'EC 1.3.1.60 created 2000' WHERE ec = 'EC 1.3.1.60';
UPDATE enzyme SET history = 'EC 1.3.1.61 created 2000, deleted 2007' WHERE ec = 'EC 1.3.1.61';
UPDATE enzyme SET sys_name = 'Pimeloyl-CoA:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.62';
UPDATE enzyme SET history = 'EC 1.3.1.62 created 2000' WHERE ec = 'EC 1.3.1.62';
UPDATE enzyme SET sys_name = '4-chlorobenzoyl-CoA:NADP(+) oxidoreductase (halogenating)' WHERE ec = 'EC 1.3.1.63';
UPDATE enzyme SET history = 'EC 1.3.1.63 created 2000, modified 2011' WHERE ec = 'EC 1.3.1.63';
UPDATE enzyme SET sys_name = 'Cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,2-dicarboxylate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.64';
UPDATE enzyme SET history = 'EC 1.3.1.64 created 2000' WHERE ec = 'EC 1.3.1.64';
UPDATE enzyme SET sys_name = '5,6-dihydroxy-3-methyl-2-oxo-1,2,5,6-tetrahydroquinoline:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.65';
UPDATE enzyme SET history = 'EC 1.3.1.65 created 2000' WHERE ec = 'EC 1.3.1.65';
UPDATE enzyme SET sys_name = 'Cis-1,2-dihydro-3-ethylcatechol:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.66';
UPDATE enzyme SET history = 'EC 1.3.1.66 created 2000' WHERE ec = 'EC 1.3.1.66';
UPDATE enzyme SET sys_name = 'Cis-1,2-dihydroxy-4-methylcyclohexa-3,5-diene-1-carboxylate:NAD(P)(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.1.67';
UPDATE enzyme SET history = 'EC 1.3.1.67 created 2000' WHERE ec = 'EC 1.3.1.67';
UPDATE enzyme SET sys_name = '1,2-dihydroxy-6-methylcyclohexa-3,5-dienecarboxylate:NAD(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.1.68';
UPDATE enzyme SET history = 'EC 1.3.1.68 created 2000' WHERE ec = 'EC 1.3.1.68';
UPDATE enzyme SET sys_name = 'Dihydrozeatin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.69';
UPDATE enzyme SET history = 'EC 1.3.1.69 created 1992 as EC 1.1.1.242, transferred 2001 to EC 1.3.1.69' WHERE ec = 'EC 1.3.1.69';
UPDATE enzyme SET sys_name = '4,4-dimethyl-5-alpha-cholesta-8,24-dien-3-beta-ol:NADP(+) Delta(14)-oxidoreductase' WHERE ec = 'EC 1.3.1.70';
UPDATE enzyme SET history = 'EC 1.3.1.70 created 2001' WHERE ec = 'EC 1.3.1.70';
UPDATE enzyme SET sys_name = 'Ergosterol:NADP(+) Delta(24(24(1)))-oxidoreductase' WHERE ec = 'EC 1.3.1.71';
UPDATE enzyme SET history = 'EC 1.3.1.71 created 2001, modified 2002' WHERE ec = 'EC 1.3.1.71';
UPDATE enzyme SET sys_name = 'Sterol:NADP(+) Delta(24)-oxidoreductase' WHERE ec = 'EC 1.3.1.72';
UPDATE enzyme SET history = 'EC 1.3.1.72 created 2001' WHERE ec = 'EC 1.3.1.72';
UPDATE enzyme SET sys_name = '17-O-acetylnorajmaline:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.73';
UPDATE enzyme SET history = 'EC 1.3.1.73 created 2002' WHERE ec = 'EC 1.3.1.73';
UPDATE enzyme SET sys_name = 'n-alkanal:NAD(P)(+) 2-oxidoreductase' WHERE ec = 'EC 1.3.1.74';
UPDATE enzyme SET history = 'EC 1.3.1.74 created 2003, modified 2014' WHERE ec = 'EC 1.3.1.74';
UPDATE enzyme SET sys_name = 'Chlorophyllide-a:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.75';
UPDATE enzyme SET history = 'EC 1.3.1.75 created 2003' WHERE ec = 'EC 1.3.1.75';
UPDATE enzyme SET sys_name = 'Precorrin-2:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.76';
UPDATE enzyme SET history = 'EC 1.3.1.76 created 2004' WHERE ec = 'EC 1.3.1.76';
UPDATE enzyme SET sys_name = 'Flavan-3-ol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.3.1.77';
UPDATE enzyme SET history = 'EC 1.3.1.77 created 2004' WHERE ec = 'EC 1.3.1.77';
UPDATE enzyme SET sys_name = 'L-arogenate:NADP(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.1.78';
UPDATE enzyme SET history = 'EC 1.3.1.78 created 2005' WHERE ec = 'EC 1.3.1.78';
UPDATE enzyme SET sys_name = 'L-arogenate:NAD(P)(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.1.79';
UPDATE enzyme SET history = 'EC 1.3.1.79 created 2005' WHERE ec = 'EC 1.3.1.79';
UPDATE enzyme SET sys_name = 'Primary fluorescent chlorophyll catabolite:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.80';
UPDATE enzyme SET history = 'EC 1.3.1.80 created 2007' WHERE ec = 'EC 1.3.1.80';
UPDATE enzyme SET sys_name = '(-)-menthone:NADP+ oxidoreductase' WHERE ec = 'EC 1.3.1.81';
UPDATE enzyme SET history = 'EC 1.3.1.81 created 2008' WHERE ec = 'EC 1.3.1.81';
UPDATE enzyme SET sys_name = '(+)-cis-isopulegone:NADP+ oxidoreductase' WHERE ec = 'EC 1.3.1.82';
UPDATE enzyme SET history = 'EC 1.3.1.82 created 2008' WHERE ec = 'EC 1.3.1.82';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.83';
UPDATE enzyme SET history = 'EC 1.3.1.83 created 2009' WHERE ec = 'EC 1.3.1.83';
UPDATE enzyme SET sys_name = 'Propanoyl-CoA:NADP+ oxidoreductase' WHERE ec = 'EC 1.3.1.84';
UPDATE enzyme SET history = 'EC 1.3.1.84 created 2009' WHERE ec = 'EC 1.3.1.84';
UPDATE enzyme SET sys_name = '(2S)-ethylmalonyl-CoA:NADP(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.1.85';
UPDATE enzyme SET history = 'EC 1.3.1.85 created 2010' WHERE ec = 'EC 1.3.1.85';
UPDATE enzyme SET sys_name = 'Butanoyl-CoA:NADP(+) 2,3-oxidoreductase' WHERE ec = 'EC 1.3.1.86';
UPDATE enzyme SET history = 'EC 1.3.1.86 created 2010' WHERE ec = 'EC 1.3.1.86';
UPDATE enzyme SET sys_name = '3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)propanoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.87';
UPDATE enzyme SET history = 'EC 1.3.1.87 created 2011' WHERE ec = 'EC 1.3.1.87';
UPDATE enzyme SET sys_name = 'tRNA-5,6-dihydrouracil(16/17):NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.3.1.88';
UPDATE enzyme SET history = 'EC 1.3.1.88 created 2011' WHERE ec = 'EC 1.3.1.88';
UPDATE enzyme SET sys_name = 'tRNA-5,6-dihydrouracil(47):NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.3.1.89';
UPDATE enzyme SET history = 'EC 1.3.1.89 created 2011' WHERE ec = 'EC 1.3.1.89';
UPDATE enzyme SET sys_name = 'tRNA-5,6-dihydrouracil(20a/20b):NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.3.1.90';
UPDATE enzyme SET history = 'EC 1.3.1.90 created 2011' WHERE ec = 'EC 1.3.1.90';
UPDATE enzyme SET sys_name = 'tRNA-5,6-dihydrouracil(20):NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.3.1.91';
UPDATE enzyme SET history = 'EC 1.3.1.91 created 2011' WHERE ec = 'EC 1.3.1.91';
UPDATE enzyme SET sys_name = 'Artemisinic aldehyde:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.92';
UPDATE enzyme SET history = 'EC 1.3.1.92 created 2012' WHERE ec = 'EC 1.3.1.92';
UPDATE enzyme SET sys_name = 'Very-long-chain acyl-CoA:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.93';
UPDATE enzyme SET history = 'EC 1.3.1.93 created 2012' WHERE ec = 'EC 1.3.1.93';
UPDATE enzyme SET sys_name = 'Ditrans,polycis-dolichol:NADP(+) 2,3-oxidoreductase' WHERE ec = 'EC 1.3.1.94';
UPDATE enzyme SET history = 'EC 1.3.1.94 created 2012' WHERE ec = 'EC 1.3.1.94';
UPDATE enzyme SET sys_name = 'Propanoyl-CoA:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.95';
UPDATE enzyme SET history = 'EC 1.3.1.95 created 2012' WHERE ec = 'EC 1.3.1.95';
UPDATE enzyme SET sys_name = 'Squalene:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.96';
UPDATE enzyme SET history = 'EC 1.3.1.96 created 2012' WHERE ec = 'EC 1.3.1.96';
UPDATE enzyme SET sys_name = 'C(30) botryococcene:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.97';
UPDATE enzyme SET history = 'EC 1.3.1.97 created 2012' WHERE ec = 'EC 1.3.1.97';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-muramate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.98';
UPDATE enzyme SET history = 'EC 1.3.1.98 created 1976 as EC 1.1.1.158, modified 1983, modified 2002, transferred 2013 to EC 1.3.1.98' WHERE ec = 'EC 1.3.1.98';
UPDATE enzyme SET sys_name = '8-oxogeranial:NAD(P)(+) oxidoreductase (cyclizing, cis-trans-nepetalactol forming)' WHERE ec = 'EC 1.3.1.99';
UPDATE enzyme SET history = 'EC 1.3.1.99 created 2013' WHERE ec = 'EC 1.3.1.99';
UPDATE enzyme SET sys_name = 'Chanoclavine-I aldehyde:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.100';
UPDATE enzyme SET history = 'EC 1.3.1.100 created 2013' WHERE ec = 'EC 1.3.1.100';
UPDATE enzyme SET sys_name = '2,3-bis-(O-phytany)l-sn-glycerol 1-phosphate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.3.1.101';
UPDATE enzyme SET history = 'EC 1.3.1.101 created 2013' WHERE ec = 'EC 1.3.1.101';
UPDATE enzyme SET sys_name = 'n-alkanal:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.3.1.102';
UPDATE enzyme SET history = 'EC 1.3.1.102 created 2013' WHERE ec = 'EC 1.3.1.102';
UPDATE enzyme SET sys_name = '(S)-2-chloropropanoate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.103';
UPDATE enzyme SET history = 'EC 1.3.1.103 created 2013' WHERE ec = 'EC 1.3.1.103';
UPDATE enzyme SET sys_name = 'Acyl-[acyl-carrier protein]:NADP(+) oxidoreductase' WHERE ec = 'EC 1.3.1.104';
UPDATE enzyme SET history = 'EC 1.3.1.104 created 2013' WHERE ec = 'EC 1.3.1.104';
UPDATE enzyme SET sys_name = '4-hydroxy-2,5-dimethylfuran-3(2H)-one:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.3.1.105';
UPDATE enzyme SET history = 'EC 1.3.1.105 created 2013' WHERE ec = 'EC 1.3.1.105';
UPDATE enzyme SET sys_name = 'Cobalt-precorrin-6B:NAD(+) oxidoreductase' WHERE ec = 'EC 1.3.1.106';
UPDATE enzyme SET history = 'EC 1.3.1.106 created 2014' WHERE ec = 'EC 1.3.1.106';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.3.1.n1';
UPDATE enzyme SET history = 'EC 1.3.2.1 created 1961, deleted 1964' WHERE ec = 'EC 1.3.2.1';
UPDATE enzyme SET history = 'EC 1.3.2.2 created 1961, deleted 1964' WHERE ec = 'EC 1.3.2.2';
UPDATE enzyme SET sys_name = 'L-galactono-1,4-lactone:ferricytochrome-c oxidoreductase' WHERE ec = 'EC 1.3.2.3';
UPDATE enzyme SET history = 'EC 1.3.2.3 created 1961, modified 2006' WHERE ec = 'EC 1.3.2.3';
UPDATE enzyme SET history = 'EC 1.3.3.1 created 1961, deleted 2011' WHERE ec = 'EC 1.3.3.1';
UPDATE enzyme SET history = 'EC 1.3.3.2 created 1972, deleted 2005' WHERE ec = 'EC 1.3.3.2';
UPDATE enzyme SET sys_name = 'Coproporphyrinogen:oxygen oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.3.3';
UPDATE enzyme SET history = 'EC 1.3.3.3 created 1972, modified 2003' WHERE ec = 'EC 1.3.3.3';
UPDATE enzyme SET sys_name = 'Protoporphyrinogen-IX:oxygen oxidoreductase' WHERE ec = 'EC 1.3.3.4';
UPDATE enzyme SET history = 'EC 1.3.3.4 created 1978, modified 2003' WHERE ec = 'EC 1.3.3.4';
UPDATE enzyme SET sys_name = 'Bilirubin:oxygen oxidoreductase' WHERE ec = 'EC 1.3.3.5';
UPDATE enzyme SET history = 'EC 1.3.3.5 created 1984' WHERE ec = 'EC 1.3.3.5';
UPDATE enzyme SET sys_name = 'Acyl-CoA:oxygen 2-oxidoreductase' WHERE ec = 'EC 1.3.3.6';
UPDATE enzyme SET history = 'EC 1.3.3.6 created 1986' WHERE ec = 'EC 1.3.3.6';
UPDATE enzyme SET sys_name = '5,6-dihydrouracil:oxygen oxidoreductase' WHERE ec = 'EC 1.3.3.7';
UPDATE enzyme SET history = 'EC 1.3.3.7 created 1989' WHERE ec = 'EC 1.3.3.7';
UPDATE enzyme SET sys_name = '(S)-tetrahydroberberine:oxygen oxidoreductase' WHERE ec = 'EC 1.3.3.8';
UPDATE enzyme SET history = 'EC 1.3.3.8 created 1990 (EC 1.5.3.8 created 1989, incorporated 1992)' WHERE ec = 'EC 1.3.3.8';
UPDATE enzyme SET sys_name = 'Loganin:oxygen oxidoreductase (ring-cleaving)' WHERE ec = 'EC 1.3.3.9';
UPDATE enzyme SET history = 'EC 1.3.3.9 created 2002' WHERE ec = 'EC 1.3.3.9';
UPDATE enzyme SET sys_name = 'L-tryptophan:oxygen alpha,beta-oxidoreductase' WHERE ec = 'EC 1.3.3.10';
UPDATE enzyme SET history = 'EC 1.3.3.10 created 2000 as EC 1.4.3.17, transferred 2003 to EC 1.3.3.10' WHERE ec = 'EC 1.3.3.10';
UPDATE enzyme SET sys_name = '6-(2-amino-2-carboxyethyl)-7,8-dioxo-1,2,3,4,7,8-hexahydroquinoline-2,4-dicarboxylate:oxygen oxidoreductase (cyclizing)' WHERE ec = 'EC 1.3.3.11';
UPDATE enzyme SET history = 'EC 1.3.3.11 created 2005' WHERE ec = 'EC 1.3.3.11';
UPDATE enzyme SET sys_name = 'L-galactono-1,4-lactone:oxygen 3-oxidoreductase' WHERE ec = 'EC 1.3.3.12';
UPDATE enzyme SET history = 'EC 1.3.3.12 created 1984 as EC 1.1.3.24, transferred 2006 to EC 1.3.3.12' WHERE ec = 'EC 1.3.3.12';
UPDATE enzyme SET sys_name = 'Cyclo(L-leucyl-L-phenylalanyl):oxygen oxidoreductase' WHERE ec = 'EC 1.3.3.13';
UPDATE enzyme SET history = 'EC 1.3.3.13 created 2013' WHERE ec = 'EC 1.3.3.13';
UPDATE enzyme SET sys_name = 'Aclacinomycin-A:oxygen oxidoreductase' WHERE ec = 'EC 1.3.3.14';
UPDATE enzyme SET history = 'EC 1.3.3.14 created 2013' WHERE ec = 'EC 1.3.3.14';
UPDATE enzyme SET sys_name = 'Succinate:quinone oxidoreductase' WHERE ec = 'EC 1.3.5.1';
UPDATE enzyme SET history = 'EC 1.3.5.1 created 1983, modified 2014' WHERE ec = 'EC 1.3.5.1';
UPDATE enzyme SET sys_name = '(S)-dihydroorotate:quinone oxidoreductase' WHERE ec = 'EC 1.3.5.2';
UPDATE enzyme SET history = 'EC 1.3.99.11 created 1983, transferred 2009 to EC 1.3.5.2, modified 2011' WHERE ec = 'EC 1.3.5.2';
UPDATE enzyme SET sys_name = 'Protoporphyrinogen IX:menaquinone oxidoreductase' WHERE ec = 'EC 1.3.5.3';
UPDATE enzyme SET history = 'EC 1.3.5.3 created 2010' WHERE ec = 'EC 1.3.5.3';
UPDATE enzyme SET sys_name = 'Succinate:quinone oxidoreductase' WHERE ec = 'EC 1.3.5.4';
UPDATE enzyme SET history = 'EC 1.3.5.4 created 2010, modified 2013' WHERE ec = 'EC 1.3.5.4';
UPDATE enzyme SET sys_name = '15-cis-phytoene:plastoquinone oxidoreductase' WHERE ec = 'EC 1.3.5.5';
UPDATE enzyme SET history = 'EC 1.3.5.5 created 2011' WHERE ec = 'EC 1.3.5.5';
UPDATE enzyme SET sys_name = '9,9-di-cis-zeta-carotene:quinone oxidoreductase' WHERE ec = 'EC 1.3.5.6';
UPDATE enzyme SET history = 'EC 1.3.5.6 created 1999 as EC 1.14.99.30, transferred 2011 to EC 1.3.5.6' WHERE ec = 'EC 1.3.5.6';
UPDATE enzyme SET sys_name = '6-oxo-1,4,5,6-tetrahydronicotinate:ferredoxin oxidoreductase' WHERE ec = 'EC 1.3.7.1';
UPDATE enzyme SET history = 'EC 1.3.7.1 created 1972' WHERE ec = 'EC 1.3.7.1';
UPDATE enzyme SET sys_name = '15,16-dihydrobiliverdin:ferredoxin oxidoreductase' WHERE ec = 'EC 1.3.7.2';
UPDATE enzyme SET history = 'EC 1.3.7.2 created 2002' WHERE ec = 'EC 1.3.7.2';
UPDATE enzyme SET sys_name = '(3Z)-phycoerythrobilin:ferredoxin oxidoreductase' WHERE ec = 'EC 1.3.7.3';
UPDATE enzyme SET history = 'EC 1.3.7.3 created 2002' WHERE ec = 'EC 1.3.7.3';
UPDATE enzyme SET sys_name = '(3Z)-phytochromobilin:ferredoxin oxidoreductase' WHERE ec = 'EC 1.3.7.4';
UPDATE enzyme SET history = 'EC 1.3.7.4 created 2002' WHERE ec = 'EC 1.3.7.4';
UPDATE enzyme SET sys_name = '(3Z)-phycocyanobilin:ferredoxin oxidoreductase' WHERE ec = 'EC 1.3.7.5';
UPDATE enzyme SET history = 'EC 1.3.7.5 created 2002' WHERE ec = 'EC 1.3.7.5';
UPDATE enzyme SET sys_name = '(3Z)-phycoerythrobilin:ferredoxin oxidoreductase (from biliverdin IX-alpha)' WHERE ec = 'EC 1.3.7.6';
UPDATE enzyme SET history = 'EC 1.3.7.6 created 2008' WHERE ec = 'EC 1.3.7.6';
UPDATE enzyme SET sys_name = 'ATP-dependent ferredoxin:protochlorophyllide-a 7,8-oxidoreductase' WHERE ec = 'EC 1.3.7.7';
UPDATE enzyme SET history = 'EC 1.3.7.7 created 2011, modified 2013' WHERE ec = 'EC 1.3.7.7';
UPDATE enzyme SET sys_name = 'Cyclohexa-1,5-diene-1-carbonyl-CoA:ferredoxin oxidoreductase (aromatizing, ATP-forming)' WHERE ec = 'EC 1.3.7.8';
UPDATE enzyme SET history = 'EC 1.3.7.8 created 1999 as EC 1.3.99.15, transferred 2011 to EC 1.3.7.8' WHERE ec = 'EC 1.3.7.8';
UPDATE enzyme SET sys_name = 'Benzoyl-CoA:acceptor oxidoreductase' WHERE ec = 'EC 1.3.7.9';
UPDATE enzyme SET history = 'EC 1.3.7.9 created 2000 as EC 1.3.99.20, transferred 2011 to EC 1.3.7.9' WHERE ec = 'EC 1.3.7.9';
UPDATE enzyme SET history = 'EC 1.3.7.10 created 2012, deleted 2013' WHERE ec = 'EC 1.3.7.10';
UPDATE enzyme SET sys_name = 'Short-chain acyl-CoA:electron-transfer flavoprotein 2,3-oxidoreductase' WHERE ec = 'EC 1.3.8.1';
UPDATE enzyme SET history = 'EC 1.3.8.1 created 1961 as EC 1.3.2.1, transferred 1964 to EC 1.3.99.2, transferred 2011 to EC 1.3.8.1, modified 2012' WHERE ec = 'EC 1.3.8.1';
UPDATE enzyme SET sys_name = '4,4-diapophytoene:FAD oxidoreductase' WHERE ec = 'EC 1.3.8.2';
UPDATE enzyme SET history = 'EC 1.3.8.2 created 2011' WHERE ec = 'EC 1.3.8.2';
UPDATE enzyme SET sys_name = '(R)-benzylsuccinyl-CoA:electron transfer flavoprotein oxidoreductase' WHERE ec = 'EC 1.3.8.3';
UPDATE enzyme SET history = 'EC 1.3.8.3 created 2003 as EC 1.3.99.21, transferred 2012 to EC 1.3.8.3' WHERE ec = 'EC 1.3.8.3';
UPDATE enzyme SET sys_name = '3-methylbutanoyl-CoA:electron-transfer flavoprotein oxidoreductase' WHERE ec = 'EC 1.3.8.4';
UPDATE enzyme SET history = 'EC 1.3.8.4 created 1978 as EC 1.3.99.10, modified 1986, transferred 2012 to EC 1.3.8.4' WHERE ec = 'EC 1.3.8.4';
UPDATE enzyme SET sys_name = '2-methyl-branched-chain-acyl-CoA:electron-transfer flavoprotein 2-oxidoreductase' WHERE ec = 'EC 1.3.8.5';
UPDATE enzyme SET history = 'EC 1.3.8.5 created 1992 as EC 1.3.1.52, transferred 2012 to EC 1.3.8.5' WHERE ec = 'EC 1.3.8.5';
UPDATE enzyme SET sys_name = 'Glutaryl-CoA:electron-transfer flavoprotein 2,3-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.8.6';
UPDATE enzyme SET history = 'EC 1.3.8.6 created 1972 as EC 1.3.99.7, transferred 2012 to EC 1.3.8.6, modified 2013' WHERE ec = 'EC 1.3.8.6';
UPDATE enzyme SET sys_name = 'Medium-chain acyl-CoA:electron-transfer flavoprotein 2,3-oxidoreductase' WHERE ec = 'EC 1.3.8.7';
UPDATE enzyme SET history = 'EC 1.3.8.7 created 1961 as EC 1.3.2.2, transferred 1964 to EC 1.3.99.3, part transferred 2012 to EC 1.3.8.7' WHERE ec = 'EC 1.3.8.7';
UPDATE enzyme SET sys_name = 'Long-chain-acyl-CoA:electron-transfer flavoprotein 2,3-oxidoreductase' WHERE ec = 'EC 1.3.8.8';
UPDATE enzyme SET history = 'EC 1.3.8.8 created 1989 as EC 1.3.99.13, part transferred 2012 to EC 1.3.8.8' WHERE ec = 'EC 1.3.8.8';
UPDATE enzyme SET sys_name = 'Very-long-chain acyl-CoA:electron-transfer flavoprotein 2,3-oxidoreductase' WHERE ec = 'EC 1.3.8.9';
UPDATE enzyme SET history = 'EC 1.3.8.9 created 1961 as EC 1.3.2.2, transferred 1964 to EC 1.3.99.3, part transferred 2012 to EC 1.3.8.9' WHERE ec = 'EC 1.3.8.9';
UPDATE enzyme SET sys_name = 'Cyclohex-1-ene-1-carbonyl-CoA:electron transfer flavoprotein oxidoreductase' WHERE ec = 'EC 1.3.8.10';
UPDATE enzyme SET history = 'EC 1.3.8.10 created 2013' WHERE ec = 'EC 1.3.8.10';
UPDATE enzyme SET sys_name = 'Cyclohexane-1-carbonyl-CoA:electron transfer flavoprotein oxidoreductase' WHERE ec = 'EC 1.3.8.11';
UPDATE enzyme SET history = 'EC 1.3.8.11 created 2013' WHERE ec = 'EC 1.3.8.11';
UPDATE enzyme SET sys_name = '(S)-dihydroorotate:fumarate oxidoreductase' WHERE ec = 'EC 1.3.98.1';
UPDATE enzyme SET history = 'EC 1.3.98.1 created 1961 as EC 1.3.3.1, transferred 2011 to EC 1.3.98.1' WHERE ec = 'EC 1.3.98.1';
UPDATE enzyme SET sys_name = 'Fumarate CoM:CoB oxidoreductase (succinate forming)' WHERE ec = 'EC 1.3.98.2';
UPDATE enzyme SET history = 'EC 1.3.98.2 created 2014' WHERE ec = 'EC 1.3.98.2';
UPDATE enzyme SET history = 'EC 1.3.99.1 created 1961, deleted 2014' WHERE ec = 'EC 1.3.99.1';
UPDATE enzyme SET history = 'EC 1.3.99.2 created 1961 as EC 1.3.2.1, transferred 1964 to EC 1.3.99.2, deleted 2011' WHERE ec = 'EC 1.3.99.2';
UPDATE enzyme SET history = 'EC 1.3.99.3 created 1961 as EC 1.3.2.2, transferred 1964 to EC 1.3.99.3' WHERE ec = 'EC 1.3.99.3';
UPDATE enzyme SET sys_name = '3-oxosteroid:acceptor Delta(1)-oxidoreductase' WHERE ec = 'EC 1.3.99.4';
UPDATE enzyme SET history = 'EC 1.3.99.4 created 1965' WHERE ec = 'EC 1.3.99.4';
UPDATE enzyme SET sys_name = '3-oxo-5-alpha-steroid:acceptor Delta(4)-oxidoreductase' WHERE ec = 'EC 1.3.99.5';
UPDATE enzyme SET history = 'EC 1.3.99.5 created 1965, deleted 2012, modified 2012' WHERE ec = 'EC 1.3.99.5';
UPDATE enzyme SET sys_name = '3-oxo-5-beta-steroid:acceptor Delta(4)-oxidoreductase' WHERE ec = 'EC 1.3.99.6';
UPDATE enzyme SET history = 'EC 1.3.99.6 created 1972' WHERE ec = 'EC 1.3.99.6';
UPDATE enzyme SET history = 'EC 1.3.99.7 created 1972, deleted 2012' WHERE ec = 'EC 1.3.99.7';
UPDATE enzyme SET sys_name = '2-furoyl-CoA:acceptor 5-oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.3.99.8';
UPDATE enzyme SET history = 'EC 1.3.99.8 created 1976' WHERE ec = 'EC 1.3.99.8';
UPDATE enzyme SET history = 'EC 1.3.99.9 created 1976, deleted 2002' WHERE ec = 'EC 1.3.99.9';
UPDATE enzyme SET history = 'EC 1.3.99.10 created 1978, modified 1986, deleted 2012' WHERE ec = 'EC 1.3.99.10';
UPDATE enzyme SET history = 'EC 1.3.99.11 created 1983, deleted 2009' WHERE ec = 'EC 1.3.99.11';
UPDATE enzyme SET sys_name = '2-methylbutanoyl-CoA:acceptor oxidoreductase' WHERE ec = 'EC 1.3.99.12';
UPDATE enzyme SET history = 'EC 1.3.99.12 created 1986' WHERE ec = 'EC 1.3.99.12';
UPDATE enzyme SET history = 'EC 1.3.99.13 created 1989, deleted 2012' WHERE ec = 'EC 1.3.99.13';
UPDATE enzyme SET sys_name = 'Cyclohexanone:acceptor 2-oxidoreductase' WHERE ec = 'EC 1.3.99.14';
UPDATE enzyme SET history = 'EC 1.3.99.14 created 1992' WHERE ec = 'EC 1.3.99.14';
UPDATE enzyme SET history = 'EC 1.3.99.15 created 1999, deleted 2011' WHERE ec = 'EC 1.3.99.15';
UPDATE enzyme SET sys_name = 'Isoquinoline:acceptor 1-oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.3.99.16';
UPDATE enzyme SET history = 'EC 1.3.99.16 created 1999' WHERE ec = 'EC 1.3.99.16';
UPDATE enzyme SET sys_name = 'Quinoline:acceptor 2-oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.3.99.17';
UPDATE enzyme SET history = 'EC 1.3.99.17 created 1999' WHERE ec = 'EC 1.3.99.17';
UPDATE enzyme SET sys_name = 'Quinoline-2-carboxylate:acceptor 4-oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.3.99.18';
UPDATE enzyme SET history = 'EC 1.3.99.18 created 1999' WHERE ec = 'EC 1.3.99.18';
UPDATE enzyme SET sys_name = 'Quinoline-4-carboxylate:acceptor 2-oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.3.99.19';
UPDATE enzyme SET history = 'EC 1.3.99.19 created 1999, modified 2006' WHERE ec = 'EC 1.3.99.19';
UPDATE enzyme SET history = 'EC 1.3.99.20 created 2000, deleted 2011' WHERE ec = 'EC 1.3.99.20';
UPDATE enzyme SET history = 'EC 1.3.99.21 created 2003, deleted 2012' WHERE ec = 'EC 1.3.99.21';
UPDATE enzyme SET sys_name = 'Coproporphyrinogen-III:S-adenosyl-L-methionine oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.99.22';
UPDATE enzyme SET history = 'EC 1.3.99.22 created 2004' WHERE ec = 'EC 1.3.99.22';
UPDATE enzyme SET sys_name = 'All-trans-13,14-dihydroretinol:acceptor 13,14-oxidoreductase' WHERE ec = 'EC 1.3.99.23';
UPDATE enzyme SET history = 'EC 1.3.99.23 created 2005' WHERE ec = 'EC 1.3.99.23';
UPDATE enzyme SET sys_name = '(2S)-2-amino-4-deoxychorismate:FMN oxidoreductase' WHERE ec = 'EC 1.3.99.24';
UPDATE enzyme SET history = 'EC 1.3.99.24 created 2008' WHERE ec = 'EC 1.3.99.24';
UPDATE enzyme SET sys_name = '(+)-dihydrocarvone:acceptor 1,6-oxidoreductase' WHERE ec = 'EC 1.3.99.25';
UPDATE enzyme SET history = 'EC 1.3.99.25 created 2008' WHERE ec = 'EC 1.3.99.25';
UPDATE enzyme SET sys_name = 'All-trans-zeta-carotene:acceptor oxidoreductase' WHERE ec = 'EC 1.3.99.26';
UPDATE enzyme SET history = 'EC 1.3.99.26 created 2011' WHERE ec = 'EC 1.3.99.26';
UPDATE enzyme SET sys_name = '1-hydroxy-1,2-dihydrolycopene:acceptor oxidoreductase' WHERE ec = 'EC 1.3.99.27';
UPDATE enzyme SET history = 'EC 1.3.99.27 created 2011' WHERE ec = 'EC 1.3.99.27';
UPDATE enzyme SET sys_name = '15-cis-phytoene:acceptor oxidoreductase (neurosporene-forming)' WHERE ec = 'EC 1.3.99.28';
UPDATE enzyme SET history = 'EC 1.3.99.28 created 2011' WHERE ec = 'EC 1.3.99.28';
UPDATE enzyme SET sys_name = '15-cis-phytoene:acceptor oxidoreductase (zeta-carotene-forming)' WHERE ec = 'EC 1.3.99.29';
UPDATE enzyme SET history = 'EC 1.3.99.29 created 2011' WHERE ec = 'EC 1.3.99.29';
UPDATE enzyme SET sys_name = '15-cis-phytoene:acceptor oxidoreductase (3,4-didehydrolycopene-forming)' WHERE ec = 'EC 1.3.99.30';
UPDATE enzyme SET history = 'EC 1.3.99.30 created 2011' WHERE ec = 'EC 1.3.99.30';
UPDATE enzyme SET sys_name = '15-cis-phytoene:acceptor oxidoreductase (lycopene-forming)' WHERE ec = 'EC 1.3.99.31';
UPDATE enzyme SET history = 'EC 1.3.99.31 created 2011' WHERE ec = 'EC 1.3.99.31';
UPDATE enzyme SET sys_name = 'Glutaryl-CoA:acceptor 2,3-oxidoreductase (non-decarboxylating)' WHERE ec = 'EC 1.3.99.32';
UPDATE enzyme SET history = 'EC 1.3.99.32 created 2012, modified 2013' WHERE ec = 'EC 1.3.99.32';
UPDATE enzyme SET sys_name = 'Dihydrourocanate:acceptor oxidoreductase' WHERE ec = 'EC 1.3.99.33';
UPDATE enzyme SET history = 'EC 1.3.99.33 created 2013' WHERE ec = 'EC 1.3.99.33';
UPDATE enzyme SET sys_name = '2,3-bis-(O-phytanyl)-sn-glycerol 1-phosphate:acceptor oxidoreductase' WHERE ec = 'EC 1.3.99.34';
UPDATE enzyme SET history = 'EC 1.3.99.34 created 2013' WHERE ec = 'EC 1.3.99.34';
UPDATE enzyme SET sys_name = 'ATP-dependent acceptor:chlorophyllide-a 7,8-oxidoreductase' WHERE ec = 'EC 1.3.99.35';
UPDATE enzyme SET history = 'EC 1.3.99.35 created 2014' WHERE ec = 'EC 1.3.99.35';
UPDATE enzyme SET sys_name = '3-hydroxycyclohexa-1,5-diene-1-carbonyl-CoA:acceptor oxidoreductase (aromatising, ATP-forming)' WHERE ec = 'EC 1.3.99.n1';
UPDATE enzyme SET history = 'EC 1.3.99.n1 created 2010' WHERE ec = 'EC 1.3.99.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.3.99.n2';
UPDATE enzyme SET sys_name = '[Cypemycin]-Cys-C-term:acceptor oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.3.99.n3';
UPDATE enzyme SET history = 'EC 1.3.99.n3 created 2011' WHERE ec = 'EC 1.3.99.n3';
UPDATE enzyme SET sys_name = 'L-alanine:NAD(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.1';
UPDATE enzyme SET history = 'EC 1.4.1.1 created 1961' WHERE ec = 'EC 1.4.1.1';
UPDATE enzyme SET sys_name = 'L-glutamate:NAD(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.2';
UPDATE enzyme SET history = 'EC 1.4.1.2 created 1961' WHERE ec = 'EC 1.4.1.2';
UPDATE enzyme SET sys_name = 'L-glutamate:NAD(P)(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.3';
UPDATE enzyme SET history = 'EC 1.4.1.3 created 1961' WHERE ec = 'EC 1.4.1.3';
UPDATE enzyme SET sys_name = 'L-glutamate:NADP(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.4';
UPDATE enzyme SET history = 'EC 1.4.1.4 created 1961' WHERE ec = 'EC 1.4.1.4';
UPDATE enzyme SET sys_name = 'L-amino-acid:NAD(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.5';
UPDATE enzyme SET history = 'EC 1.4.1.5 created 1961' WHERE ec = 'EC 1.4.1.5';
UPDATE enzyme SET history = 'EC 1.4.1.6 created 1961, deleted 1982' WHERE ec = 'EC 1.4.1.6';
UPDATE enzyme SET sys_name = 'L-serine:NAD(+) 2-oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.7';
UPDATE enzyme SET history = 'EC 1.4.1.7 created 1972, modified 2003' WHERE ec = 'EC 1.4.1.7';
UPDATE enzyme SET sys_name = 'L-valine:NADP(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.8';
UPDATE enzyme SET history = 'EC 1.4.1.8 created 1972' WHERE ec = 'EC 1.4.1.8';
UPDATE enzyme SET sys_name = 'L-leucine:NAD(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.9';
UPDATE enzyme SET history = 'EC 1.4.1.9 created 1972' WHERE ec = 'EC 1.4.1.9';
UPDATE enzyme SET sys_name = 'Glycine:NAD(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.10';
UPDATE enzyme SET history = 'EC 1.4.1.10 created 1972' WHERE ec = 'EC 1.4.1.10';
UPDATE enzyme SET sys_name = 'L-erythro-3,5-diaminohexanoate:NAD(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.11';
UPDATE enzyme SET history = 'EC 1.4.1.11 created 1976' WHERE ec = 'EC 1.4.1.11';
UPDATE enzyme SET sys_name = '2,4-diaminopentanoate:NAD(P)(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.12';
UPDATE enzyme SET history = 'EC 1.4.1.12 created 1976' WHERE ec = 'EC 1.4.1.12';
UPDATE enzyme SET sys_name = 'L-glutamate:NADP(+) oxidoreductase (transaminating)' WHERE ec = 'EC 1.4.1.13';
UPDATE enzyme SET history = 'EC 1.4.1.13 created 1972 as EC 2.6.1.53, transferred 1976 to EC 1.4.1.13, modified 2001, modified 2010' WHERE ec = 'EC 1.4.1.13';
UPDATE enzyme SET sys_name = 'L-glutamate:NAD(+) oxidoreductase (transaminating)' WHERE ec = 'EC 1.4.1.14';
UPDATE enzyme SET history = 'EC 1.4.1.14 created 1978' WHERE ec = 'EC 1.4.1.14';
UPDATE enzyme SET sys_name = 'L-lysine:NAD(+) oxidoreductase (deaminating, cyclizing)' WHERE ec = 'EC 1.4.1.15';
UPDATE enzyme SET history = 'EC 1.4.1.15 created 1978' WHERE ec = 'EC 1.4.1.15';
UPDATE enzyme SET sys_name = 'Meso-2,6-diaminoheptanedioate:NADP(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.16';
UPDATE enzyme SET history = 'EC 1.4.1.16 created 1981' WHERE ec = 'EC 1.4.1.16';
UPDATE enzyme SET sys_name = 'N-methyl-L-alanine:NADP(+) oxidoreductase (demethylating, deaminating)' WHERE ec = 'EC 1.4.1.17';
UPDATE enzyme SET history = 'EC 1.4.1.17 created 1984' WHERE ec = 'EC 1.4.1.17';
UPDATE enzyme SET sys_name = 'L-lysine:NAD(+) 6-oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.18';
UPDATE enzyme SET history = 'EC 1.4.1.18 created 1989, modified 2006, modified 2011' WHERE ec = 'EC 1.4.1.18';
UPDATE enzyme SET sys_name = 'L-tryptophan:NAD(P)(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.19';
UPDATE enzyme SET history = 'EC 1.4.1.19 created 1989' WHERE ec = 'EC 1.4.1.19';
UPDATE enzyme SET sys_name = 'L-phenylalanine:NAD(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.20';
UPDATE enzyme SET history = 'EC 1.4.1.20 created 1989' WHERE ec = 'EC 1.4.1.20';
UPDATE enzyme SET sys_name = 'L-aspartate:NAD(P)(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.21';
UPDATE enzyme SET history = 'EC 1.14.1.21 created 2005' WHERE ec = 'EC 1.4.1.21';
UPDATE enzyme SET sys_name = 'L-valine:NAD(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.23';
UPDATE enzyme SET history = 'EC 1.4.1.23 created 2012' WHERE ec = 'EC 1.4.1.23';
UPDATE enzyme SET sys_name = '2-amino-3,7-dideoxy-D-threo-hept-6-ulosonate:NAD(+) oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.1.24';
UPDATE enzyme SET history = 'EC 1.4.1.24 created 2012' WHERE ec = 'EC 1.4.1.24';
UPDATE enzyme SET sys_name = 'Glycine:ferricytochrome-c oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.2.1';
UPDATE enzyme SET history = 'EC 1.4.2.1 created 1976' WHERE ec = 'EC 1.4.2.1';
UPDATE enzyme SET sys_name = 'D-aspartate:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.1';
UPDATE enzyme SET history = 'EC 1.4.3.1 created 1961' WHERE ec = 'EC 1.4.3.1';
UPDATE enzyme SET sys_name = 'L-amino-acid:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.2';
UPDATE enzyme SET history = 'EC 1.4.3.2 created 1961' WHERE ec = 'EC 1.4.3.2';
UPDATE enzyme SET sys_name = 'D-amino-acid:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.3';
UPDATE enzyme SET history = 'EC 1.4.3.3 created 1961' WHERE ec = 'EC 1.4.3.3';
UPDATE enzyme SET sys_name = 'Amine:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.4';
UPDATE enzyme SET history = 'EC 1.4.3.4 created 1961, modified 1983 (EC 1.4.3.9 created 1972, incorporated 1984), modified 2008' WHERE ec = 'EC 1.4.3.4';
UPDATE enzyme SET sys_name = 'Pyridoxamine-5-phosphate:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.5';
UPDATE enzyme SET history = 'EC 1.4.3.5 created 1961, modified 2006' WHERE ec = 'EC 1.4.3.5';
UPDATE enzyme SET history = 'EC 1.4.3.6 created 1961, modified 1983, modified 1989, deleted 2008' WHERE ec = 'EC 1.4.3.6';
UPDATE enzyme SET sys_name = 'D-glutamate:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.7';
UPDATE enzyme SET history = 'EC 1.4.3.7 created 1972' WHERE ec = 'EC 1.4.3.7';
UPDATE enzyme SET sys_name = 'Ethanolamine:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.8';
UPDATE enzyme SET history = 'EC 1.4.3.8 created 1972' WHERE ec = 'EC 1.4.3.8';
UPDATE enzyme SET history = 'EC 1.4.3.9 created 1972, deleted 1984' WHERE ec = 'EC 1.4.3.9';
UPDATE enzyme SET sys_name = 'Putrescine:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.10';
UPDATE enzyme SET history = 'EC 1.4.3.10 created 1976' WHERE ec = 'EC 1.4.3.10';
UPDATE enzyme SET sys_name = 'L-glutamate:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.11';
UPDATE enzyme SET history = 'EC 1.4.3.11 created 1976, modified 1989' WHERE ec = 'EC 1.4.3.11';
UPDATE enzyme SET sys_name = 'Cyclohexylamine:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.12';
UPDATE enzyme SET history = 'EC 1.4.3.12 created 1978' WHERE ec = 'EC 1.4.3.12';
UPDATE enzyme SET sys_name = 'Protein-L-lysine:oxygen 6-oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.13';
UPDATE enzyme SET history = 'EC 1.4.3.13 created 1980, modified 1983' WHERE ec = 'EC 1.4.3.13';
UPDATE enzyme SET sys_name = 'L-lysine:oxygen 2-oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.14';
UPDATE enzyme SET history = 'EC 1.4.3.14 created 1981' WHERE ec = 'EC 1.4.3.14';
UPDATE enzyme SET sys_name = 'D-glutamate(D-aspartate):oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.15';
UPDATE enzyme SET history = 'EC 1.4.3.15 created 1983, modified 2012' WHERE ec = 'EC 1.4.3.15';
UPDATE enzyme SET sys_name = 'L-aspartate:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.16';
UPDATE enzyme SET history = 'EC 1.4.3.16 created 1984, modified 2008' WHERE ec = 'EC 1.4.3.16';
UPDATE enzyme SET history = 'EC 1.4.3.17 created 2000, deleted 2003' WHERE ec = 'EC 1.4.3.17';
UPDATE enzyme SET sys_name = 'Glycine:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.19';
UPDATE enzyme SET history = 'EC 1.4.3.19 created 2002, modified 2012' WHERE ec = 'EC 1.4.3.19';
UPDATE enzyme SET sys_name = 'L-lysine:oxygen 6-oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.20';
UPDATE enzyme SET history = 'EC 1.4.3.20 created 2006, modified 2011' WHERE ec = 'EC 1.4.3.20';
UPDATE enzyme SET sys_name = 'Primary-amine:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.21';
UPDATE enzyme SET history = 'EC 1.4.3.21 created 2008' WHERE ec = 'EC 1.4.3.21';
UPDATE enzyme SET sys_name = 'Histamine:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.3.22';
UPDATE enzyme SET history = 'EC 1.4.3.22 created 2007 (EC 1.4.3.6 created 1961, part-incorporated 2008)' WHERE ec = 'EC 1.4.3.22';
UPDATE enzyme SET sys_name = '7-chloro-L-tryptophan:oxygen oxidoreductase' WHERE ec = 'EC 1.4.3.23';
UPDATE enzyme SET history = 'EC 1.4.3.23 created 2010' WHERE ec = 'EC 1.4.3.23';
UPDATE enzyme SET sys_name = '4-(methylamino)-1-(pyridin-3-yl)butan-1-one:oxygen oxidoreductase (methylamine releasing)' WHERE ec = 'EC 1.4.3.24';
UPDATE enzyme SET history = 'EC 1.4.3.24 created 2012' WHERE ec = 'EC 1.4.3.24';
UPDATE enzyme SET history = 'EC 1.4.4.1 created 1972, modified 1982 (EC 1.4.1.6 created 1961, incorporated 1982), deleted 2003' WHERE ec = 'EC 1.4.4.1';
UPDATE enzyme SET sys_name = 'Glycine:H-protein-lipoyllysine oxidoreductase (decarboxylating, acceptor-amino-methylating)' WHERE ec = 'EC 1.4.4.2';
UPDATE enzyme SET history = 'EC 1.4.4.2 created 1984, modified 2003, modified 2006, modified 2013' WHERE ec = 'EC 1.4.4.2';
UPDATE enzyme SET sys_name = 'D-amino acid:quinone oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.5.1';
UPDATE enzyme SET history = 'EC 1.4.5.1 created 2010' WHERE ec = 'EC 1.4.5.1';
UPDATE enzyme SET sys_name = 'L-glutamate:ferredoxin oxidoreductase (transaminating)' WHERE ec = 'EC 1.4.7.1';
UPDATE enzyme SET history = 'EC 1.4.7.1 created 1976, modified 2012' WHERE ec = 'EC 1.4.7.1';
UPDATE enzyme SET sys_name = 'Methylamine:amicyanin oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.9.1';
UPDATE enzyme SET history = 'EC 1.4.98.1 created 1978 as EC 1.4.99.3, modified 1986, transferred 2011 to EC 1.4.98.1, transferred 2011 to EC 1.4.9.1' WHERE ec = 'EC 1.4.9.1';
UPDATE enzyme SET sys_name = 'Aralkylamine:azurin oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.9.2';
UPDATE enzyme SET history = 'EC 1.4.9.2 created 1986 as EC 1.4.99.4, transferred 2011 to 1.4.9.2' WHERE ec = 'EC 1.4.9.2';
UPDATE enzyme SET history = 'EC 1.4.98.1 created 1978 as EC 1.4.99.3, modified 1986, transferred 2011 to EC 1.4.98.1, deleted 2011' WHERE ec = 'EC 1.4.98.1';
UPDATE enzyme SET sys_name = 'D-amino-acid:acceptor oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.99.1';
UPDATE enzyme SET history = 'EC 1.4.99.1 created 1972' WHERE ec = 'EC 1.4.99.1';
UPDATE enzyme SET sys_name = 'Taurine:acceptor oxidoreductase (deaminating)' WHERE ec = 'EC 1.4.99.2';
UPDATE enzyme SET history = 'EC 1.4.99.2 created 1976' WHERE ec = 'EC 1.4.99.2';
UPDATE enzyme SET history = 'EC 1.4.99.3 created 1978, modified 1986, deleted 2011' WHERE ec = 'EC 1.4.99.3';
UPDATE enzyme SET history = 'EC 1.4.99.4 created 1986, deleted 2011' WHERE ec = 'EC 1.4.99.4';
UPDATE enzyme SET sys_name = 'Glycine:acceptor oxidoreductase (hydrogen-cyanide-forming)' WHERE ec = 'EC 1.4.99.5';
UPDATE enzyme SET history = 'EC 1.4.99.5 created 2002' WHERE ec = 'EC 1.4.99.5';
UPDATE enzyme SET sys_name = 'L-proline:NAD(P)(+) 2-oxidoreductase' WHERE ec = 'EC 1.5.1.1';
UPDATE enzyme SET history = 'EC 1.5.1.1 created 1961' WHERE ec = 'EC 1.5.1.1';
UPDATE enzyme SET sys_name = 'L-proline:NAD(P)(+) 5-oxidoreductase' WHERE ec = 'EC 1.5.1.2';
UPDATE enzyme SET history = 'EC 1.5.1.2 created 1961' WHERE ec = 'EC 1.5.1.2';
UPDATE enzyme SET sys_name = '5,6,7,8-tetrahydrofolate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.3';
UPDATE enzyme SET history = 'EC 1.5.1.3 created 1961, modified 1976 (EC 1.5.1.4 created 1961, incorporated 1976)' WHERE ec = 'EC 1.5.1.3';
UPDATE enzyme SET history = 'EC 1.5.1.4 created 1961, deleted 1976' WHERE ec = 'EC 1.5.1.4';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.5';
UPDATE enzyme SET history = 'EC 1.5.1.5 created 1961' WHERE ec = 'EC 1.5.1.5';
UPDATE enzyme SET sys_name = '10-formyltetrahydrofolate:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.6';
UPDATE enzyme SET history = 'EC 1.5.1.6 created 1972' WHERE ec = 'EC 1.5.1.6';
UPDATE enzyme SET sys_name = 'N(6)-(L-1,3-dicarboxypropyl)-L-lysine:NAD(+) oxidoreductase (L-lysine-forming)' WHERE ec = 'EC 1.5.1.7';
UPDATE enzyme SET history = 'EC 1.5.1.7 created 1972' WHERE ec = 'EC 1.5.1.7';
UPDATE enzyme SET sys_name = 'N(6)-(L-1,3-dicarboxypropyl)-L-lysine:NADP(+) oxidoreductase (L-lysine-forming)' WHERE ec = 'EC 1.5.1.8';
UPDATE enzyme SET history = 'EC 1.5.1.8 created 1972' WHERE ec = 'EC 1.5.1.8';
UPDATE enzyme SET sys_name = 'N(6)-(L-1,3-dicarboxypropyl)-L-lysine:NAD(+) oxidoreductase (L-glutamate-forming)' WHERE ec = 'EC 1.5.1.9';
UPDATE enzyme SET history = 'EC 1.5.1.9 created 1972, modified 2011' WHERE ec = 'EC 1.5.1.9';
UPDATE enzyme SET sys_name = 'N(6)-(L-1,3-dicarboxypropyl)-L-lysine:NADP(+) oxidoreductase (L-glutamate-forming)' WHERE ec = 'EC 1.5.1.10';
UPDATE enzyme SET history = 'EC 1.5.1.10 created 1972, modified 2011' WHERE ec = 'EC 1.5.1.10';
UPDATE enzyme SET sys_name = 'N(2)-(D-1-carboxyethyl)-L-arginine:NAD(+) oxidoreductase (L-arginine-forming)' WHERE ec = 'EC 1.5.1.11';
UPDATE enzyme SET history = 'EC 1.5.1.11 created 1972' WHERE ec = 'EC 1.5.1.11';
UPDATE enzyme SET history = 'EC 1.5.1.12 created 1972, modified 2008, deleted 2013' WHERE ec = 'EC 1.5.1.12';
UPDATE enzyme SET history = 'EC 1.5.1.13 created 1972, deleted 2004' WHERE ec = 'EC 1.5.1.13';
UPDATE enzyme SET history = 'EC 1.5.1.14 created 1976, deleted 1989' WHERE ec = 'EC 1.5.1.14';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.5.1.15';
UPDATE enzyme SET history = 'EC 1.5.1.15 created 1978' WHERE ec = 'EC 1.5.1.15';
UPDATE enzyme SET sys_name = 'N(2)-(D-1-carboxyethyl)-L-lysine:NADP(+) oxidoreductase (L-lysine-forming)' WHERE ec = 'EC 1.5.1.16';
UPDATE enzyme SET history = 'EC 1.5.1.16 created 1978' WHERE ec = 'EC 1.5.1.16';
UPDATE enzyme SET sys_name = '2,2-iminodipropanoate:NAD(+) oxidoreductase (L-alanine-forming)' WHERE ec = 'EC 1.5.1.17';
UPDATE enzyme SET history = 'EC 1.5.1.17 created 1983, modified 1986' WHERE ec = 'EC 1.5.1.17';
UPDATE enzyme SET sys_name = '(-)-ephedrine:NAD(+) 2-oxidoreductase' WHERE ec = 'EC 1.5.1.18';
UPDATE enzyme SET history = 'EC 1.5.1.18 created 1984' WHERE ec = 'EC 1.5.1.18';
UPDATE enzyme SET sys_name = 'N(2)-(D-1,3-dicarboxypropyl)-L-arginine:NADP(+) oxidoreductase (L-arginine-forming)' WHERE ec = 'EC 1.5.1.19';
UPDATE enzyme SET history = 'EC 1.5.1.19 created 1984' WHERE ec = 'EC 1.5.1.19';
UPDATE enzyme SET sys_name = '5-methyltetrahydrofolate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.5.1.20';
UPDATE enzyme SET history = 'EC 1.5.1.20 created 1978 as EC 1.1.1.171, transferred 1984 to EC 1.5.1.20 (EC 1.7.99.5 incorporated 2005), modified 2005' WHERE ec = 'EC 1.5.1.20';
UPDATE enzyme SET sys_name = 'L-pipecolate:NADP(+) 2-oxidoreductase' WHERE ec = 'EC 1.5.1.21';
UPDATE enzyme SET history = 'EC 1.5.1.21 created 1984 (EC 1.5.1.14 created 1976, incorporated 1989)' WHERE ec = 'EC 1.5.1.21';
UPDATE enzyme SET sys_name = 'N-(carboxymethyl)-D-alanine:NAD(+) oxidoreductase (glycine-forming)' WHERE ec = 'EC 1.5.1.22';
UPDATE enzyme SET history = 'EC 1.5.1.22 created 1986' WHERE ec = 'EC 1.5.1.22';
UPDATE enzyme SET sys_name = 'N(2)-(D-1-carboxyethyl)taurine:NAD(+) oxidoreductase (taurine-forming)' WHERE ec = 'EC 1.5.1.23';
UPDATE enzyme SET history = 'EC 1.5.1.23 created 1989' WHERE ec = 'EC 1.5.1.23';
UPDATE enzyme SET sys_name = 'N(5)-(L-1-carboxyethyl)-L-ornithine:NADP(+) oxidoreductase (L-ornithine-forming)' WHERE ec = 'EC 1.5.1.24';
UPDATE enzyme SET history = 'EC 1.5.1.24 created 1990' WHERE ec = 'EC 1.5.1.24';
UPDATE enzyme SET sys_name = 'Thiomorpholine-3-carboxylate:NAD(P)(+) 5,6-oxidoreductase' WHERE ec = 'EC 1.5.1.25';
UPDATE enzyme SET history = 'EC 1.5.1.25 created 1990' WHERE ec = 'EC 1.5.1.25';
UPDATE enzyme SET sys_name = 'N-(D-1-carboxyethyl)-beta-alanine:NAD(+) oxidoreductase (beta-alanine-forming)' WHERE ec = 'EC 1.5.1.26';
UPDATE enzyme SET history = 'EC 1.5.1.26 created 1990' WHERE ec = 'EC 1.5.1.26';
UPDATE enzyme SET sys_name = '(R)-reticuline:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.27';
UPDATE enzyme SET history = 'EC 1.5.1.27 created 1999, modified 2004' WHERE ec = 'EC 1.5.1.27';
UPDATE enzyme SET sys_name = '(2S)-2-((1-(R)-carboxyethyl)amino)pentanoate:NAD(+) oxidoreductase (L-aminopentanoate-forming)' WHERE ec = 'EC 1.5.1.28';
UPDATE enzyme SET history = 'EC 1.5.1.28 created 1999' WHERE ec = 'EC 1.5.1.28';
UPDATE enzyme SET history = 'EC 1.5.1.29 created 1981 as EC 1.6.8.1, transferred 2002 to EC 1.5.1.29, modified 2002, deleted 2011' WHERE ec = 'EC 1.5.1.29';
UPDATE enzyme SET sys_name = 'Reduced-riboflavin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.30';
UPDATE enzyme SET history = 'EC 1.5.1.30 created 1982 as EC 1.6.8.2, transferred 2002 to EC 1.5.1.30, modified 2011' WHERE ec = 'EC 1.5.1.30';
UPDATE enzyme SET sys_name = '(R)-tetrahydroberberine:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.31';
UPDATE enzyme SET history = 'EC 1.5.1.31 created 2002' WHERE ec = 'EC 1.5.1.31';
UPDATE enzyme SET sys_name = '1,2-dihydrovomilenine:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.32';
UPDATE enzyme SET history = 'EC 1.5.1.32 created 2002' WHERE ec = 'EC 1.5.1.32';
UPDATE enzyme SET sys_name = '5,6,7,8-tetrahydrobiopterin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.33';
UPDATE enzyme SET history = 'EC 1.5.1.33 created 1999 as EC 1.1.1.253, transferred 2003 to EC 1.5.1.33' WHERE ec = 'EC 1.5.1.33';
UPDATE enzyme SET sys_name = '5,6,7,8-tetrahydropteridine:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.5.1.34';
UPDATE enzyme SET history = 'EC 1.5.1.34 created 1972 as EC 1.6.99.7, modified 1981 (EC 1.6.99.10 created 1978, incorporated 1981), transferred 2003 to EC 1.5.1.34' WHERE ec = 'EC 1.5.1.34';
UPDATE enzyme SET history = 'EC 1.5.1.35 created 2006, deleted 2007' WHERE ec = 'EC 1.5.1.35';
UPDATE enzyme SET sys_name = 'Flavin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.5.1.36';
UPDATE enzyme SET history = 'EC 1.5.1.36 created 2011' WHERE ec = 'EC 1.5.1.36';
UPDATE enzyme SET sys_name = 'FADH(2):NAD(+) oxidoreductase' WHERE ec = 'EC 1.5.1.37';
UPDATE enzyme SET history = 'EC 1.5.1.37 created 2011' WHERE ec = 'EC 1.5.1.37';
UPDATE enzyme SET sys_name = 'FMNH(2):NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.38';
UPDATE enzyme SET history = 'EC 1.5.1.38 created 2011' WHERE ec = 'EC 1.5.1.38';
UPDATE enzyme SET sys_name = 'FMNH(2):NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.5.1.39';
UPDATE enzyme SET history = 'EC 1.5.1.39 created 2011' WHERE ec = 'EC 1.5.1.39';
UPDATE enzyme SET sys_name = 'Reduced coenzyme F420:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.40';
UPDATE enzyme SET history = 'EC 1.5.1.40 created 2011' WHERE ec = 'EC 1.5.1.40';
UPDATE enzyme SET sys_name = 'Riboflavin:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.5.1.41';
UPDATE enzyme SET history = 'EC 1.5.1.41 created 2011' WHERE ec = 'EC 1.5.1.41';
UPDATE enzyme SET sys_name = 'FMNH(2):NAD(+) oxidoreductase' WHERE ec = 'EC 1.5.1.42';
UPDATE enzyme SET history = 'EC 1.5.1.42 created 2011' WHERE ec = 'EC 1.5.1.42';
UPDATE enzyme SET sys_name = 'Carboxynorspermidine:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.43';
UPDATE enzyme SET history = 'EC 1.5.1.43 created 2012' WHERE ec = 'EC 1.5.1.43';
UPDATE enzyme SET sys_name = 'Festuclavine:NAD(+) oxidoreductase' WHERE ec = 'EC 1.5.1.44';
UPDATE enzyme SET history = 'EC 1.5.1.44 created 2012' WHERE ec = 'EC 1.5.1.44';
UPDATE enzyme SET sys_name = 'FADH(2):NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.5.1.45';
UPDATE enzyme SET history = 'EC 1.5.1.45 created 2012' WHERE ec = 'EC 1.5.1.45';
UPDATE enzyme SET sys_name = 'Agroclavine:NADP(+) oxidoreductase' WHERE ec = 'EC 1.5.1.46';
UPDATE enzyme SET history = 'EC 1.5.1.46 created 2013' WHERE ec = 'EC 1.5.1.46';
UPDATE enzyme SET sys_name = '5,6,7,8-tetrahydromethanopterin 5,6-oxidoreductase' WHERE ec = 'EC 1.5.1.47';
UPDATE enzyme SET history = 'EC 1.5.1.47 created 2013' WHERE ec = 'EC 1.5.1.47';
UPDATE enzyme SET sys_name = 'Sarcosine:oxygen oxidoreductase (demethylating)' WHERE ec = 'EC 1.5.3.1';
UPDATE enzyme SET history = 'EC 1.5.3.1 created 1961' WHERE ec = 'EC 1.5.3.1';
UPDATE enzyme SET sys_name = 'N-methyl-L-amino-acid:oxygen oxidoreductase (demethylating)' WHERE ec = 'EC 1.5.3.2';
UPDATE enzyme SET history = 'EC 1.5.3.2 created 1961' WHERE ec = 'EC 1.5.3.2';
UPDATE enzyme SET history = 'EC 1.5.3.3 created 1961, deleted 1972' WHERE ec = 'EC 1.5.3.3';
UPDATE enzyme SET sys_name = 'N(6)-methyl-L-lysine:oxygen oxidoreductase (demethylating)' WHERE ec = 'EC 1.5.3.4';
UPDATE enzyme SET history = 'EC 1.5.3.4 created 1972' WHERE ec = 'EC 1.5.3.4';
UPDATE enzyme SET sys_name = '(S)-6-hydroxynicotine:oxygen oxidoreductase' WHERE ec = 'EC 1.5.3.5';
UPDATE enzyme SET history = 'EC 1.5.3.5 created 1972' WHERE ec = 'EC 1.5.3.5';
UPDATE enzyme SET sys_name = '(R)-6-hydroxynicotine:oxygen oxidoreductase' WHERE ec = 'EC 1.5.3.6';
UPDATE enzyme SET history = 'EC 1.5.3.6 created 1972' WHERE ec = 'EC 1.5.3.6';
UPDATE enzyme SET sys_name = 'L-pipecolate:oxygen 1,6-oxidoreductase' WHERE ec = 'EC 1.5.3.7';
UPDATE enzyme SET history = 'EC 1.5.3.7 created 1986, modified 2011' WHERE ec = 'EC 1.5.3.7';
UPDATE enzyme SET history = 'EC 1.5.3.8 created 1989, deleted 1992' WHERE ec = 'EC 1.5.3.8';
UPDATE enzyme SET history = 'EC 1.5.3.9 created 1989, modified 1999, deleted 2003' WHERE ec = 'EC 1.5.3.9';
UPDATE enzyme SET sys_name = 'N,N-dimethylglycine:oxygen oxidoreductase (demethylating)' WHERE ec = 'EC 1.5.3.10';
UPDATE enzyme SET history = 'EC 1.5.3.10 created 1992' WHERE ec = 'EC 1.5.3.10';
UPDATE enzyme SET history = 'EC 1.5.3.11 created 1992, deleted 2009' WHERE ec = 'EC 1.5.3.11';
UPDATE enzyme SET sys_name = 'Dihydrobenzophenanthridine:oxygen oxidoreductase' WHERE ec = 'EC 1.5.3.12';
UPDATE enzyme SET history = 'EC 1.5.3.12 created 1999' WHERE ec = 'EC 1.5.3.12';
UPDATE enzyme SET sys_name = 'N(1)-acetylpolyamine:oxygen oxidoreductase (3-acetamidopropanal-forming)' WHERE ec = 'EC 1.5.3.13';
UPDATE enzyme SET history = 'EC 1.5.3.13 created 2009' WHERE ec = 'EC 1.5.3.13';
UPDATE enzyme SET sys_name = 'Spermidine:oxygen oxidoreductase (propane-1,3-diamine-forming)' WHERE ec = 'EC 1.5.3.14';
UPDATE enzyme SET history = 'EC 1.5.3.14 created 2009' WHERE ec = 'EC 1.5.3.14';
UPDATE enzyme SET sys_name = 'N(8)-acetylspermidine:oxygen oxidoreductase (propane-1,3-diamine-forming)' WHERE ec = 'EC 1.5.3.15';
UPDATE enzyme SET history = 'EC 1.5.3.15 created 2009' WHERE ec = 'EC 1.5.3.15';
UPDATE enzyme SET sys_name = 'Spermidine:oxygen oxidoreductase (spermidine-forming)' WHERE ec = 'EC 1.5.3.16';
UPDATE enzyme SET history = 'EC 1.5.3.16 created 2009' WHERE ec = 'EC 1.5.3.16';
UPDATE enzyme SET sys_name = 'Polyamine:oxygen oxidoreductase (3-aminopropanal or 3-acetamidopropanal-forming)' WHERE ec = 'EC 1.5.3.17';
UPDATE enzyme SET history = 'EC 1.5.3.17 created 2009' WHERE ec = 'EC 1.5.3.17';
UPDATE enzyme SET sys_name = 'L-saccharopine:oxygen oxidoreductase (L-glutamate forming)' WHERE ec = 'EC 1.5.3.18';
UPDATE enzyme SET history = 'EC 1.5.3.18 created 2011' WHERE ec = 'EC 1.5.3.18';
UPDATE enzyme SET sys_name = '4-methylaminobutanoate:oxygen oxidoreductase (formaldehyde-forming)' WHERE ec = 'EC 1.5.3.19';
UPDATE enzyme SET history = 'EC 1.5.3.19 created 2012' WHERE ec = 'EC 1.5.3.19';
UPDATE enzyme SET sys_name = 'N-alkylglycine:oxygen oxidoreductase (alkylamine forming)' WHERE ec = 'EC 1.5.3.20';
UPDATE enzyme SET history = 'EC 1.5.3.20 created 2012' WHERE ec = 'EC 1.5.3.20';
UPDATE enzyme SET sys_name = '4-methylaminobutanoate methylamidohydrolase' WHERE ec = 'EC 1.5.3.21';
UPDATE enzyme SET history = 'EC 1.5.3.21 created 2012' WHERE ec = 'EC 1.5.3.21';
UPDATE enzyme SET sys_name = 'Reduced coenzyme F420:oxygen oxidoreductase' WHERE ec = 'EC 1.5.3.22';
UPDATE enzyme SET history = 'EC 1.5.3.22 created 2013' WHERE ec = 'EC 1.5.3.22';
UPDATE enzyme SET sys_name = 'Pyrimidodiazepine:glutathione-disulfide oxidoreductase (ring-opening, cyclizing)' WHERE ec = 'EC 1.5.4.1';
UPDATE enzyme SET history = 'EC 1.5.4.1 created 1990' WHERE ec = 'EC 1.5.4.1';
UPDATE enzyme SET sys_name = 'Electron-transferring-flavoprotein:ubiquinone oxidoreductase' WHERE ec = 'EC 1.5.5.1';
UPDATE enzyme SET history = 'EC 1.5.5.1 created 1986' WHERE ec = 'EC 1.5.5.1';
UPDATE enzyme SET sys_name = 'L-proline:quinone oxidoreductase' WHERE ec = 'EC 1.5.5.2';
UPDATE enzyme SET history = 'EC 1.5.5.2 created 1980 as EC 1.5.99.8, modified 2008, transferred 2013 to EC 1.5.5.2' WHERE ec = 'EC 1.5.5.2';
UPDATE enzyme SET sys_name = '5-methyltetrahydrofolate:ferredoxin oxidoreductase' WHERE ec = 'EC 1.5.7.1';
UPDATE enzyme SET history = 'EC 1.5.7.1 created 2005' WHERE ec = 'EC 1.5.7.1';
UPDATE enzyme SET sys_name = 'Coenzyme F420:ferredoxin oxidoreductase' WHERE ec = 'EC 1.5.7.2';
UPDATE enzyme SET history = 'EC 1.5.7.2 created 2013' WHERE ec = 'EC 1.5.7.2';
UPDATE enzyme SET sys_name = 'Dimethylamine:electron-transferring flavoprotein oxidoreductase' WHERE ec = 'EC 1.5.8.1';
UPDATE enzyme SET history = 'EC 1.5.8.1 created 1999 as EC 1.5.99.10, transferred 2002 to EC1.5.8.1' WHERE ec = 'EC 1.5.8.1';
UPDATE enzyme SET sys_name = 'Trimethylamine:electron-transferring flavoprotein oxidoreductase (demethylating)' WHERE ec = 'EC 1.5.8.2';
UPDATE enzyme SET history = 'EC 1.5.8.2 created 1976 as EC 1.5.99.7, transferred 2002 to EC 1.5.8.2' WHERE ec = 'EC 1.5.8.2';
UPDATE enzyme SET sys_name = 'Sarcosine:electron-transfer flavoprotein oxidoreductase (demethylating)' WHERE ec = 'EC 1.5.8.3';
UPDATE enzyme SET history = 'EC 1.5.8.3 created 1972 as EC 1.5.99.1, transferred 2012 to EC 1.5.8.3' WHERE ec = 'EC 1.5.8.3';
UPDATE enzyme SET sys_name = 'N,N-dimethylglycine:electron-transfer flavoprotein oxidoreductase (demethylating)' WHERE ec = 'EC 1.5.8.4';
UPDATE enzyme SET history = 'EC 1.5.8.4 created 1972 as EC 1.5.99.2, transferred 2012 to EC 1.5.8.4' WHERE ec = 'EC 1.5.8.4';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydromethanopterin:coenzyme-F420 oxidoreductase' WHERE ec = 'EC 1.5.98.1';
UPDATE enzyme SET history = 'EC 1.5.98.1 created 1989 as EC 1.5.99.9, modified 2004, transferred 2014 to EC 1.5.98.1' WHERE ec = 'EC 1.5.98.1';
UPDATE enzyme SET sys_name = '5-methyltetrahydromethanopterin:coenzyme-F420 oxidoreductase' WHERE ec = 'EC 1.5.98.2';
UPDATE enzyme SET history = 'EC 1.5.98.2 created 2000 as EC 1.5.99.11, modified 2004, transferred 2014 to EC 1.5.98.2' WHERE ec = 'EC 1.5.98.2';
UPDATE enzyme SET history = 'EC 1.5.99.1 created 1972, deleted 2012' WHERE ec = 'EC 1.5.99.1';
UPDATE enzyme SET history = 'EC 1.5.99.2 created 1972, deleted 2012' WHERE ec = 'EC 1.5.99.2';
UPDATE enzyme SET sys_name = 'L-pipecolate:acceptor 1,6-oxidoreductase' WHERE ec = 'EC 1.5.99.3';
UPDATE enzyme SET history = 'EC 1.5.99.3 created 1972, modified 1986, modified 2011' WHERE ec = 'EC 1.5.99.3';
UPDATE enzyme SET sys_name = 'Nicotine:acceptor 6-oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.5.99.4';
UPDATE enzyme SET history = 'EC 1.5.99.4 created 1972' WHERE ec = 'EC 1.5.99.4';
UPDATE enzyme SET sys_name = 'N-methyl-L-glutamate:acceptor oxidoreductase (demethylating)' WHERE ec = 'EC 1.5.99.5';
UPDATE enzyme SET history = 'EC 1.5.99.5 created 1976' WHERE ec = 'EC 1.5.99.5';
UPDATE enzyme SET sys_name = 'Spermidine:acceptor oxidoreductase' WHERE ec = 'EC 1.5.99.6';
UPDATE enzyme SET history = 'EC 1.5.99.6 created 1976' WHERE ec = 'EC 1.5.99.6';
UPDATE enzyme SET history = 'EC 1.5.99.7 created 1976, deleted 2002' WHERE ec = 'EC 1.5.99.7';
UPDATE enzyme SET history = 'EC 1.5.99.8 created 1980, modified 2008, deleted 2013' WHERE ec = 'EC 1.5.99.8';
UPDATE enzyme SET history = 'EC 1.5.99.9 created 1989, modified 2004, deleted 2014' WHERE ec = 'EC 1.5.99.9';
UPDATE enzyme SET history = 'EC 1.5.99.10 created 1999, deleted 2002' WHERE ec = 'EC 1.5.99.10';
UPDATE enzyme SET history = 'EC 1.5.99.11 created 2000,  modified 2004, deleted 2014' WHERE ec = 'EC 1.5.99.11';
UPDATE enzyme SET sys_name = 'N(6)-dimethylallyladenine:acceptor oxidoreductase' WHERE ec = 'EC 1.5.99.12';
UPDATE enzyme SET history = 'EC 1.5.99.12 created 2001' WHERE ec = 'EC 1.5.99.12';
UPDATE enzyme SET sys_name = 'D-proline:acceptor oxidoreductase' WHERE ec = 'EC 1.5.99.13';
UPDATE enzyme SET history = 'EC 1.5.99.13 created 2010, modified 2011' WHERE ec = 'EC 1.5.99.13';
UPDATE enzyme SET sys_name = '1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one:acceptor 6-oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.5.99.14';
UPDATE enzyme SET history = 'EC 1.5.99.14 created 2012' WHERE ec = 'EC 1.5.99.14';
UPDATE enzyme SET sys_name = 'NADPH:NAD(+) oxidoreductase (Si-specific)' WHERE ec = 'EC 1.6.1.1';
UPDATE enzyme SET history = 'EC 1.6.1.1 created 1961, modified 1986, modified 2013' WHERE ec = 'EC 1.6.1.1';
UPDATE enzyme SET sys_name = 'NADPH:NAD(+) oxidoreductase (Re/Si-specific)' WHERE ec = 'EC 1.6.1.2';
UPDATE enzyme SET history = 'EC 1.6.1.2 created 1986, modified 2013' WHERE ec = 'EC 1.6.1.2';
UPDATE enzyme SET sys_name = 'NADPH:NAD(+) oxidoreductase' WHERE ec = 'EC 1.6.1.3';
UPDATE enzyme SET history = 'EC 1.6.1.3 created 2013' WHERE ec = 'EC 1.6.1.3';
UPDATE enzyme SET history = 'EC 1.6.2.1 created 1961, deleted 1965' WHERE ec = 'EC 1.6.2.1';
UPDATE enzyme SET sys_name = 'NADH:ferricytochrome-b5 oxidoreductase' WHERE ec = 'EC 1.6.2.2';
UPDATE enzyme SET history = 'EC 1.6.2.2 created 1961' WHERE ec = 'EC 1.6.2.2';
UPDATE enzyme SET history = 'EC 1.6.2.3 created 1972, deleted 1965' WHERE ec = 'EC 1.6.2.3';
UPDATE enzyme SET sys_name = 'NADPH:hemoprotein oxidoreductase' WHERE ec = 'EC 1.6.2.4';
UPDATE enzyme SET history = 'EC 1.6.2.4 created 1972, modified 2003' WHERE ec = 'EC 1.6.2.4';
UPDATE enzyme SET sys_name = 'NADPH:ferricytochrome-c2 oxidoreductase' WHERE ec = 'EC 1.6.2.5';
UPDATE enzyme SET history = 'EC 1.6.2.5 created 1972' WHERE ec = 'EC 1.6.2.5';
UPDATE enzyme SET sys_name = 'NAD(P)H:ferrileghemoglobin oxidoreductase' WHERE ec = 'EC 1.6.2.6';
UPDATE enzyme SET history = 'EC 1.6.2.6 created 1989' WHERE ec = 'EC 1.6.2.6';
UPDATE enzyme SET sys_name = 'NAD(P)H:oxygen oxidoreductase (H(2)O(2)-forming)' WHERE ec = 'EC 1.6.3.1';
UPDATE enzyme SET history = 'EC 1.6.3.1 created 2003, modified 2012' WHERE ec = 'EC 1.6.3.1';
UPDATE enzyme SET sys_name = 'NAD(P)H:oxygen oxidoreductase (H(2)O-forming)' WHERE ec = 'EC 1.6.3.2';
UPDATE enzyme SET history = 'EC 1.6.3.2 created 2013' WHERE ec = 'EC 1.6.3.2';
UPDATE enzyme SET sys_name = 'NADH:oxygen oxidoreductase (H(2)O(2)-forming)' WHERE ec = 'EC 1.6.3.3';
UPDATE enzyme SET history = 'EC 1.6.3.3 created 2013' WHERE ec = 'EC 1.6.3.3';
UPDATE enzyme SET sys_name = 'NADH:oxygen oxidoreductase (H(2)O-forming)' WHERE ec = 'EC 1.6.3.4';
UPDATE enzyme SET history = 'EC 1.6.3.4 created 2013' WHERE ec = 'EC 1.6.3.4';
UPDATE enzyme SET sys_name = 'NAD(P)H:oxygen oxidoreductase (H(2)O(2)-forming, epimerising)' WHERE ec = 'EC 1.6.3.5';
UPDATE enzyme SET history = 'EC 1.6.3.5 created 2014' WHERE ec = 'EC 1.6.3.5';
UPDATE enzyme SET history = 'EC 1.6.4.1 created 1961, deleted 2002' WHERE ec = 'EC 1.6.4.1';
UPDATE enzyme SET history = 'EC 1.6.4.2 created 1961, modified 1989, deleted 2002' WHERE ec = 'EC 1.6.4.2';
UPDATE enzyme SET history = 'EC 1.6.4.3 created 1961, modified 1976, deleted 1983' WHERE ec = 'EC 1.6.4.3';
UPDATE enzyme SET history = 'EC 1.6.4.4 created 1965, deleted 2002' WHERE ec = 'EC 1.6.4.4';
UPDATE enzyme SET history = 'EC 1.6.4.5 created 1972, deleted 2002' WHERE ec = 'EC 1.6.4.5';
UPDATE enzyme SET history = 'EC 1.6.4.6 created 1972, deleted 2002' WHERE ec = 'EC 1.6.4.6';
UPDATE enzyme SET history = 'EC 1.6.4.7 created 1978, deleted 2002' WHERE ec = 'EC 1.6.4.7';
UPDATE enzyme SET history = 'EC 1.6.4.8 created 1989, deleted 2002' WHERE ec = 'EC 1.6.4.8';
UPDATE enzyme SET history = 'EC 1.6.4.9 created 1992, deleted 2002' WHERE ec = 'EC 1.6.4.9';
UPDATE enzyme SET history = 'EC 1.6.4.10 created 1992, deleted 2002' WHERE ec = 'EC 1.6.4.10';
UPDATE enzyme SET history = 'EC 1.6.5.1 created 1961, deleted 1965' WHERE ec = 'EC 1.6.5.1';
UPDATE enzyme SET sys_name = 'NAD(P)H:quinone oxidoreductase' WHERE ec = 'EC 1.6.5.2';
UPDATE enzyme SET history = 'EC 1.6.5.2 created 1961, transferred 1965 to EC 1.6.99.2, transferred 2005 to EC 1.6.5.2' WHERE ec = 'EC 1.6.5.2';
UPDATE enzyme SET sys_name = 'NADH:ubiquinone oxidoreductase' WHERE ec = 'EC 1.6.5.3';
UPDATE enzyme SET history = 'EC 1.6.5.3 created 1961, deleted 1965, reinstated 1983, modified 2011' WHERE ec = 'EC 1.6.5.3';
UPDATE enzyme SET sys_name = 'NADH:monodehydroascorbate oxidoreductase' WHERE ec = 'EC 1.6.5.4';
UPDATE enzyme SET history = 'EC 1.6.5.4 created 1961' WHERE ec = 'EC 1.6.5.4';
UPDATE enzyme SET sys_name = 'NADPH:quinone oxidoreductase' WHERE ec = 'EC 1.6.5.5';
UPDATE enzyme SET history = 'EC 1.6.5.5 created 1999' WHERE ec = 'EC 1.6.5.5';
UPDATE enzyme SET sys_name = 'NADPH:p-benzoquinone oxidoreductase' WHERE ec = 'EC 1.6.5.6';
UPDATE enzyme SET history = 'EC 1.6.5.6 created 2000' WHERE ec = 'EC 1.6.5.6';
UPDATE enzyme SET sys_name = '2-hydroxy-1,4-benzoquinone:NADH oxidoreductase' WHERE ec = 'EC 1.6.5.7';
UPDATE enzyme SET history = 'EC 1.6.5.7 created 2000, modified 2004' WHERE ec = 'EC 1.6.5.7';
UPDATE enzyme SET sys_name = 'NADH:ubiquinone oxidoreductase (Na(+)-translocating)' WHERE ec = 'EC 1.6.5.8';
UPDATE enzyme SET history = 'EC 1.6.5.8 created 2011' WHERE ec = 'EC 1.6.5.8';
UPDATE enzyme SET sys_name = 'NADH:ubiquinone oxidoreductase' WHERE ec = 'EC 1.6.5.9';
UPDATE enzyme SET history = 'EC 1.6.5.9 created 2011' WHERE ec = 'EC 1.6.5.9';
UPDATE enzyme SET sys_name = 'NADPH:(quinone-acceptor) oxidoreductase' WHERE ec = 'EC 1.6.5.10';
UPDATE enzyme SET history = 'EC 1.6.5.10 created 1972 as EC 1.6.99.6, transferred 2011 to EC 1.6.5.10' WHERE ec = 'EC 1.6.5.10';
UPDATE enzyme SET history = 'EC 1.6.6.1 created 1961, deleted 2002' WHERE ec = 'EC 1.6.6.1';
UPDATE enzyme SET history = 'EC 1.6.6.2 created 1961, deleted 2002' WHERE ec = 'EC 1.6.6.2';
UPDATE enzyme SET history = 'EC 1.6.6.3 created 1961, deleted 2002' WHERE ec = 'EC 1.6.6.3';
UPDATE enzyme SET history = 'EC 1.6.6.4 created 1961, deleted 2002' WHERE ec = 'EC 1.6.6.4';
UPDATE enzyme SET history = 'EC 1.6.6.5 created 1961, deleted 1964' WHERE ec = 'EC 1.6.6.5';
UPDATE enzyme SET history = 'EC 1.6.6.6 created 1961, deleted 2002' WHERE ec = 'EC 1.6.6.6';
UPDATE enzyme SET history = 'EC 1.6.6.7 created 1961, deleted 2002' WHERE ec = 'EC 1.6.6.7';
UPDATE enzyme SET history = 'EC 1.6.6.8 created 1965, deleted 2002' WHERE ec = 'EC 1.6.6.8';
UPDATE enzyme SET sys_name = 'NADH:trimethylamine-N-oxide oxidoreductase' WHERE ec = 'EC 1.6.6.9';
UPDATE enzyme SET history = 'EC 1.6.6.9 created 1972' WHERE ec = 'EC 1.6.6.9';
UPDATE enzyme SET history = 'EC 1.6.6.10 created 1972, deleted 2002' WHERE ec = 'EC 1.6.6.10';
UPDATE enzyme SET history = 'EC 1.6.6.11 created 1972, deleted 2002' WHERE ec = 'EC 1.6.6.11';
UPDATE enzyme SET history = 'EC 1.6.6.12 created 1989, deleted 2002' WHERE ec = 'EC 1.6.6.12';
UPDATE enzyme SET history = 'EC 1.6.6.13 created 1989, deleted 2002' WHERE ec = 'EC 1.6.6.13';
UPDATE enzyme SET history = 'EC 1.6.7.1 created 1972, deleted 1978' WHERE ec = 'EC 1.6.7.1';
UPDATE enzyme SET history = 'EC 1.6.7.2 created 1972, deleted 1978' WHERE ec = 'EC 1.6.7.2';
UPDATE enzyme SET history = 'EC 1.6.8.1 created 1981, deleted 2002' WHERE ec = 'EC 1.6.8.1';
UPDATE enzyme SET history = 'EC 1.6.8.2 created 1982, deleted 2002' WHERE ec = 'EC 1.6.8.2';
UPDATE enzyme SET sys_name = 'NADPH:acceptor oxidoreductase' WHERE ec = 'EC 1.6.99.1';
UPDATE enzyme SET history = 'EC 1.6.99.1 created 1961, modified 1976' WHERE ec = 'EC 1.6.99.1';
UPDATE enzyme SET history = 'EC 1.6.99.2 created 1961 as EC 1.6.5.2, transferred 1965 to EC 1.6.99.2, transferred 2005 to EC 1.6.5.2, deleted 2005' WHERE ec = 'EC 1.6.99.2';
UPDATE enzyme SET sys_name = 'NADH:acceptor oxidoreductase' WHERE ec = 'EC 1.6.99.3';
UPDATE enzyme SET history = 'EC 1.6.99.3 created 1961 as EC 1.6.2.1, transferred 1965 to EC 1.6.99.3' WHERE ec = 'EC 1.6.99.3';
UPDATE enzyme SET history = 'EC 1.6.99.4 created 1965, deleted 1972' WHERE ec = 'EC 1.6.99.4';
UPDATE enzyme SET sys_name = 'NADH:(quinone-acceptor) oxidoreductase' WHERE ec = 'EC 1.6.99.5';
UPDATE enzyme SET history = 'EC 1.6.99.5 created 1972' WHERE ec = 'EC 1.6.99.5';
UPDATE enzyme SET history = 'EC 1.6.99.6 created 1972, deleted 2011' WHERE ec = 'EC 1.6.99.6';
UPDATE enzyme SET history = 'EC 1.6.99.7 created 1972, modified 1981 (EC 1.6.99.10 created 1978, incorporated 1981), deleted 2003' WHERE ec = 'EC 1.6.99.7';
UPDATE enzyme SET history = 'EC 1.6.99.8 created 1972, deleted 2002' WHERE ec = 'EC 1.6.99.8';
UPDATE enzyme SET history = 'EC 1.6.99.9 created 1972, deleted 2002' WHERE ec = 'EC 1.6.99.9';
UPDATE enzyme SET history = 'EC 1.6.99.10 created 1978, deleted 1981' WHERE ec = 'EC 1.6.99.10';
UPDATE enzyme SET history = 'EC 1.6.99.11 created 1989, deleted 2002' WHERE ec = 'EC 1.6.99.11';
UPDATE enzyme SET history = 'EC 1.6.99.12 created 1989, deleted 2002' WHERE ec = 'EC 1.6.99.12';
UPDATE enzyme SET history = 'EC 1.6.99.13 created 1992, deleted 2002' WHERE ec = 'EC 1.6.99.13';
UPDATE enzyme SET sys_name = 'NADPH:F420 oxidoreductase' WHERE ec = 'EC 1.6.99.n1';
UPDATE enzyme SET history = 'EC 1.6.99.n1 created 2012' WHERE ec = 'EC 1.6.99.n1';
UPDATE enzyme SET sys_name = 'Nitrite:NAD(+) oxidoreductase' WHERE ec = 'EC 1.7.1.1';
UPDATE enzyme SET history = 'EC 1.7.1.1 created 1961 as EC 1.6.6.1, transferred 2002 to EC 1.7.1.1' WHERE ec = 'EC 1.7.1.1';
UPDATE enzyme SET sys_name = 'Nitrite:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.7.1.2';
UPDATE enzyme SET history = 'EC 1.7.1.2 created 1961 as EC 1.6.6.2, transferred 2002 to EC 1.7.1.2' WHERE ec = 'EC 1.7.1.2';
UPDATE enzyme SET sys_name = 'Nitrite:NADP(+) oxidoreductase' WHERE ec = 'EC 1.7.1.3';
UPDATE enzyme SET history = 'EC 1.7.1.3 created 1961 as EC 1.6.6.3, transferred 2002 to EC 1.7.1.3' WHERE ec = 'EC 1.7.1.3';
UPDATE enzyme SET sys_name = 'Ammonia:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.7.1.4';
UPDATE enzyme SET history = 'EC 1.7.1.4 created 1961 as EC 1.6.6.4, transferred 2002 to EC 1.7.1.4, modified 2013' WHERE ec = 'EC 1.7.1.4';
UPDATE enzyme SET sys_name = 'Hydroxylamine:NAD(+) oxidoreductase' WHERE ec = 'EC 1.7.1.5';
UPDATE enzyme SET history = 'EC 1.7.1.5 created 1961 as EC 1.6.6.6, transferred 2002 to EC 1.7.1.5' WHERE ec = 'EC 1.7.1.5';
UPDATE enzyme SET sys_name = 'N,N-dimethyl-1,4-phenylenediamine, aniline:NADP(+) oxidoreductase' WHERE ec = 'EC 1.7.1.6';
UPDATE enzyme SET history = 'EC 1.7.1.6 created 1961 as EC 1.6.6.7, transferred 2002 to EC 1.7.1.6' WHERE ec = 'EC 1.7.1.6';
UPDATE enzyme SET sys_name = 'Inosine-5-phosphate:NADP(+) oxidoreductase (aminating)' WHERE ec = 'EC 1.7.1.7';
UPDATE enzyme SET history = 'EC 1.7.1.7 created 1965 as EC 1.6.6.8, transferred 2002 to EC 1.7.1.7' WHERE ec = 'EC 1.7.1.7';
UPDATE enzyme SET history = 'EC 1.7.1.8 created 2002, deleted 2002' WHERE ec = 'EC 1.7.1.8';
UPDATE enzyme SET sys_name = '4-(hydroxyamino)quinoline N-oxide:NADP(+) oxidoreductase' WHERE ec = 'EC 1.7.1.9';
UPDATE enzyme SET history = 'EC 1.7.1.9 created 1972 as EC 1.6.6.10, transferred 2002 to EC 1.7.1.9' WHERE ec = 'EC 1.7.1.9';
UPDATE enzyme SET sys_name = 'Ammonium:NAD(+) oxidoreductase' WHERE ec = 'EC 1.7.1.10';
UPDATE enzyme SET history = 'EC 1.7.1.10 created 1972 as EC 1.6.6.11, transferred 2002 to EC 1.7.1.10' WHERE ec = 'EC 1.7.1.10';
UPDATE enzyme SET sys_name = '4-(dimethylamino)phenylazobenzene:NADP(+) oxidoreductase' WHERE ec = 'EC 1.7.1.11';
UPDATE enzyme SET history = 'EC 1.7.1.11 created 1989 as EC 1.6.6.12, transferred 2002 to EC 1.7.1.11' WHERE ec = 'EC 1.7.1.11';
UPDATE enzyme SET sys_name = '2-acetamidofluorene:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.7.1.12';
UPDATE enzyme SET history = 'EC 1.7.1.12 created 1989 as EC 1.6.6.13, transferred 2002 to EC 1.7.1.12' WHERE ec = 'EC 1.7.1.12';
UPDATE enzyme SET sys_name = '7-aminomethyl-7-carbaguanine:NADP(+) oxidoreductase' WHERE ec = 'EC 1.7.1.13';
UPDATE enzyme SET history = 'EC 1.7.1.13 created 2006' WHERE ec = 'EC 1.7.1.13';
UPDATE enzyme SET sys_name = 'Nitrous oxide:NAD(P) oxidoreductase' WHERE ec = 'EC 1.7.1.14';
UPDATE enzyme SET history = 'EC 1.7.1.14 created 2011' WHERE ec = 'EC 1.7.1.14';
UPDATE enzyme SET sys_name = 'Ammonia:NAD(+) oxidoreductase' WHERE ec = 'EC 1.7.1.15';
UPDATE enzyme SET history = 'EC 1.7.1.15 created 2013' WHERE ec = 'EC 1.7.1.15';
UPDATE enzyme SET sys_name = 'Nitric-oxide:ferricytochrome-c oxidoreductase' WHERE ec = 'EC 1.7.2.1';
UPDATE enzyme SET history = 'EC 1.7.2.1 created 1961, modified 1976, modified 2001, modified 2002 (EC 1.7.99.3 created 1961 as EC 1.6.6.5, transferred 1964 to EC 1.7.99.3, modified 1976, incorporated 2002) (EC 1.9.3.2 created 1965, incorporated 2002)' WHERE ec = 'EC 1.7.2.1';
UPDATE enzyme SET sys_name = 'Ammonia:ferricytochrome-c oxidoreductase' WHERE ec = 'EC 1.7.2.2';
UPDATE enzyme SET history = 'EC 1.7.2.2 created 2001' WHERE ec = 'EC 1.7.2.2';
UPDATE enzyme SET sys_name = 'Trimethylamine:cytochrome c oxidoreductase' WHERE ec = 'EC 1.7.2.3';
UPDATE enzyme SET history = 'EC 1.7.2.3 created 2002' WHERE ec = 'EC 1.7.2.3';
UPDATE enzyme SET sys_name = 'Nitrogen:cytochrome c oxidoreductase (N(2)O-forming)' WHERE ec = 'EC 1.7.2.4';
UPDATE enzyme SET history = 'EC 1.7.2.4 created 1989 as EC 1.7.99.6, modified 1999, transferred 2011 to EC 1.7.2.4' WHERE ec = 'EC 1.7.2.4';
UPDATE enzyme SET sys_name = 'Nitrous oxide:ferricytochrome-c oxidoreductase' WHERE ec = 'EC 1.7.2.5';
UPDATE enzyme SET history = 'EC 1.7.2.5 created 1992 as EC 1.7.99.7, transferred 2011 to EC 1.7.2.5' WHERE ec = 'EC 1.7.2.5';
UPDATE enzyme SET sys_name = 'Hydroxylamine:ferricytochrome-c oxidoreductase' WHERE ec = 'EC 1.7.2.6';
UPDATE enzyme SET history = 'EC 1.7.2.6 created 1972 as EC 1.7.3.4, transferred 2012 to EC 1.7.2.6' WHERE ec = 'EC 1.7.2.6';
UPDATE enzyme SET sys_name = 'Nitroalkane:oxygen oxidoreductase' WHERE ec = 'EC 1.7.3.1';
UPDATE enzyme SET history = 'EC 1.7.3.1 created 1961, modified 2006, modified 2009' WHERE ec = 'EC 1.7.3.1';
UPDATE enzyme SET sys_name = 'N-acetylindoxyl:oxygen oxidoreductase' WHERE ec = 'EC 1.7.3.2';
UPDATE enzyme SET history = 'EC 1.7.3.2 created 1961' WHERE ec = 'EC 1.7.3.2';
UPDATE enzyme SET sys_name = 'Urate:oxygen oxidoreductase' WHERE ec = 'EC 1.7.3.3';
UPDATE enzyme SET history = 'EC 1.7.3.3 created 1961, modified 2002, modified 2005, modified 2010' WHERE ec = 'EC 1.7.3.3';
UPDATE enzyme SET history = 'EC 1.7.3.4 created 1972, deleted 2013' WHERE ec = 'EC 1.7.3.4';
UPDATE enzyme SET sys_name = '3-aci-nitropropanoate:oxygen oxidoreductase' WHERE ec = 'EC 1.7.3.5';
UPDATE enzyme SET history = 'EC 1.7.3.5 created 1990' WHERE ec = 'EC 1.7.3.5';
UPDATE enzyme SET sys_name = 'Hydroxylamine:oxygen oxidoreductase' WHERE ec = 'EC 1.7.3.6';
UPDATE enzyme SET history = 'EC 1.7.3.6 created 1972 as EC 1.7.3.4, part transferred 2013 to EC 1.7.3.6' WHERE ec = 'EC 1.7.3.6';
UPDATE enzyme SET sys_name = 'Nitrite:quinone oxidoreductase' WHERE ec = 'EC 1.7.5.1';
UPDATE enzyme SET history = 'EC 1.7.5.1 created 2010' WHERE ec = 'EC 1.7.5.1';
UPDATE enzyme SET sys_name = 'Nitrous oxide:menaquinol oxidoreductase' WHERE ec = 'EC 1.7.5.2';
UPDATE enzyme SET history = 'EC 1.7.5.2 created 2011' WHERE ec = 'EC 1.7.5.2';
UPDATE enzyme SET sys_name = 'Nitrite:nitrite oxidoreductase' WHERE ec = 'EC 1.7.6.1';
UPDATE enzyme SET history = 'EC 1.7.6.1 created 2011' WHERE ec = 'EC 1.7.6.1';
UPDATE enzyme SET sys_name = 'Ammonia:ferredoxin oxidoreductase' WHERE ec = 'EC 1.7.7.1';
UPDATE enzyme SET history = 'EC 1.7.7.1 created 1972, modified 1999' WHERE ec = 'EC 1.7.7.1';
UPDATE enzyme SET sys_name = 'Nitrite:ferredoxin oxidoreductase' WHERE ec = 'EC 1.7.7.2';
UPDATE enzyme SET history = 'EC 1.7.7.2 created 1986' WHERE ec = 'EC 1.7.7.2';
UPDATE enzyme SET sys_name = 'Ammonia:acceptor oxidoreductase' WHERE ec = 'EC 1.7.99.1';
UPDATE enzyme SET history = 'EC 1.7.99.1 created 1961, modified 1999, modified 2002' WHERE ec = 'EC 1.7.99.1';
UPDATE enzyme SET history = 'EC 1.7.99.2 created 1961, modified 1976, deleted 1992' WHERE ec = 'EC 1.7.99.2';
UPDATE enzyme SET history = 'EC 1.7.99.3 created 1961 as EC 1.6.6.5, transferred 1964 to EC 1.7.99.3, modified 1976, deleted 2002' WHERE ec = 'EC 1.7.99.3';
UPDATE enzyme SET sys_name = 'Nitrite:acceptor oxidoreductase' WHERE ec = 'EC 1.7.99.4';
UPDATE enzyme SET history = 'EC 1.7.99.4 created 1972, modified 1976' WHERE ec = 'EC 1.7.99.4';
UPDATE enzyme SET history = 'EC 1.7.99.5 created 1965 as EC 1.1.1.68, transferred 1978 to EC 1.1.99.15, transferred 1980 to EC 1.7.99.5, deleted 2005' WHERE ec = 'EC 1.7.99.5';
UPDATE enzyme SET history = 'EC 1.7.99.6 created 1989, modified 1999, deleted 2011' WHERE ec = 'EC 1.7.99.6';
UPDATE enzyme SET history = 'EC 1.7.99.7 created 1992, modified 1999, deleted 2011' WHERE ec = 'EC 1.7.99.7';
UPDATE enzyme SET sys_name = 'Hydrazine:acceptor oxidoreductase' WHERE ec = 'EC 1.7.99.8';
UPDATE enzyme SET history = 'EC 1.7.99.8 created 2003, modified 2010' WHERE ec = 'EC 1.7.99.8';
UPDATE enzyme SET history = 'EC 1.8.1.1 created 1961, deleted 1972' WHERE ec = 'EC 1.8.1.1';
UPDATE enzyme SET sys_name = 'Hydrogen-sulfide:NADP(+) oxidoreductase' WHERE ec = 'EC 1.8.1.2';
UPDATE enzyme SET history = 'EC 1.8.1.2 created 1961' WHERE ec = 'EC 1.8.1.2';
UPDATE enzyme SET sys_name = 'Hypotaurine:NAD(+) oxidoreductase' WHERE ec = 'EC 1.8.1.3';
UPDATE enzyme SET history = 'EC 1.8.1.3 created 1972' WHERE ec = 'EC 1.8.1.3';
UPDATE enzyme SET sys_name = 'Protein-N(6)-(dihydrolipoyl)lysine:NAD(+) oxidoreductase' WHERE ec = 'EC 1.8.1.4';
UPDATE enzyme SET history = 'EC 1.8.1.4 created 1961 as EC 1.6.4.3, modified 1976, transferred 1983 to EC 1.8.1.4, modified 2003, modified 2006' WHERE ec = 'EC 1.8.1.4';
UPDATE enzyme SET sys_name = '2-mercaptoethanesulfonate,acetoacetate:NADP(+) oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.8.1.5';
UPDATE enzyme SET history = 'EC 1.8.1.5 created 2001' WHERE ec = 'EC 1.8.1.5';
UPDATE enzyme SET sys_name = 'L-cysteine:NAD(+) oxidoreductase' WHERE ec = 'EC 1.8.1.6';
UPDATE enzyme SET history = 'EC 1.8.1.6 created 1961 as EC 1.6.4.1, transferred 2002 to EC 1.8.1.6' WHERE ec = 'EC 1.8.1.6';
UPDATE enzyme SET sys_name = 'Glutathione:NADP(+) oxidoreductase' WHERE ec = 'EC 1.8.1.7';
UPDATE enzyme SET history = 'EC 1.8.1.7 created 1961 as EC 1.6.4.2, modified 1989, transferred 2002 to EC 1.8.1.7' WHERE ec = 'EC 1.8.1.7';
UPDATE enzyme SET sys_name = 'Protein-dithiol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.8.1.8';
UPDATE enzyme SET history = 'EC 1.8.1.8 created 1965 as EC 1.6.4.4, transferred 2002 to EC 1.8.1.8' WHERE ec = 'EC 1.8.1.8';
UPDATE enzyme SET sys_name = 'Thioredoxin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.8.1.9';
UPDATE enzyme SET history = 'EC 1.8.1.9 created 1972 as EC 1.6.4.5, transferred 2002 to EC 1.8.1.9' WHERE ec = 'EC 1.8.1.9';
UPDATE enzyme SET sys_name = 'Glutathione:NADP(+) oxidoreductase (CoA-acylating)' WHERE ec = 'EC 1.8.1.10';
UPDATE enzyme SET history = 'EC 1.8.1.10 created 1972 as EC 1.6.4.6, transferred 2002 to EC 1.8.1.10' WHERE ec = 'EC 1.8.1.10';
UPDATE enzyme SET sys_name = '3-mercapto-2-mercaptomethylpropanoate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.8.1.11';
UPDATE enzyme SET history = 'EC 1.8.1.11 created 1978 as EC 1.6.4.7, transferred 2002 to EC 1.8.1.11' WHERE ec = 'EC 1.8.1.11';
UPDATE enzyme SET sys_name = 'Trypanothione:NADP(+) oxidoreductase' WHERE ec = 'EC 1.8.1.12';
UPDATE enzyme SET history = 'EC 1.8.1.12 created 1989 as EC 1.6.4.8, transferred 2002 to EC 1.8.1.12' WHERE ec = 'EC 1.8.1.12';
UPDATE enzyme SET sys_name = 'Gamma-glutamylcysteine:NADP(+) oxidoreductase' WHERE ec = 'EC 1.8.1.13';
UPDATE enzyme SET history = 'EC 1.8.1.13 created 1992 as EC 1.6.4.9, transferred 2002 to EC 1.8.1.13' WHERE ec = 'EC 1.8.1.13';
UPDATE enzyme SET sys_name = 'CoA:NADP(+) oxidoreductase' WHERE ec = 'EC 1.8.1.14';
UPDATE enzyme SET history = 'EC 1.8.1.14 created 1992 as EC 1.6.4.10, transferred 2002 to EC 1.8.1.14, modified 2005, modified 2013' WHERE ec = 'EC 1.8.1.14';
UPDATE enzyme SET sys_name = 'Mycothiol:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.8.1.15';
UPDATE enzyme SET history = 'EC 1.8.1.15 created 2002' WHERE ec = 'EC 1.8.1.15';
UPDATE enzyme SET sys_name = 'Glutathione amide:NAD(+) oxidoreductase' WHERE ec = 'EC 1.8.1.16';
UPDATE enzyme SET history = 'EC 1.8.1.16 created 2010' WHERE ec = 'EC 1.8.1.16';
UPDATE enzyme SET sys_name = 'Dimethylsulfoxide:NAD(+) oxidoreductase' WHERE ec = 'EC 1.8.1.17';
UPDATE enzyme SET history = 'EC 1.8.1.17 created 2011' WHERE ec = 'EC 1.8.1.17';
UPDATE enzyme SET sys_name = 'Hydrogen sulfide:NAD(P)(+) oxidoreductase (CoA-dependent)' WHERE ec = 'EC 1.8.1.18';
UPDATE enzyme SET history = 'EC 1.8.1.18 created 2013' WHERE ec = 'EC 1.8.1.18';
UPDATE enzyme SET sys_name = 'Hydrogen sulfide,polysulfide:NADP(+) oxidoreductase' WHERE ec = 'EC 1.8.1.19';
UPDATE enzyme SET history = 'EC 1.8.1.19 created 2013' WHERE ec = 'EC 1.8.1.19';
UPDATE enzyme SET sys_name = 'Sulfite:ferricytochrome-c oxidoreductase' WHERE ec = 'EC 1.8.2.1';
UPDATE enzyme SET history = 'EC 1.8.2.1 created 1972' WHERE ec = 'EC 1.8.2.1';
UPDATE enzyme SET sys_name = 'Thiosulfate:ferricytochrome-c oxidoreductase' WHERE ec = 'EC 1.8.2.2';
UPDATE enzyme SET history = 'EC 1.8.2.2 created 1990' WHERE ec = 'EC 1.8.2.2';
UPDATE enzyme SET sys_name = 'Hydrogen-sulfide:flavocytochrome c oxidoreductase' WHERE ec = 'EC 1.8.2.3';
UPDATE enzyme SET history = 'EC 1.8.2.3 created 2011' WHERE ec = 'EC 1.8.2.3';
UPDATE enzyme SET sys_name = 'Dimethyl sulfide:ferricytochrome-c2 oxidoreductase' WHERE ec = 'EC 1.8.2.4';
UPDATE enzyme SET history = 'EC 1.8.2.4 created 2011' WHERE ec = 'EC 1.8.2.4';
UPDATE enzyme SET sys_name = 'Sulfite:oxygen oxidoreductase' WHERE ec = 'EC 1.8.3.1';
UPDATE enzyme SET history = 'EC 1.8.3.1 created 1961' WHERE ec = 'EC 1.8.3.1';
UPDATE enzyme SET sys_name = 'Thiol:oxygen oxidoreductase' WHERE ec = 'EC 1.8.3.2';
UPDATE enzyme SET history = 'EC 1.8.3.2 created 1961, modified 2011' WHERE ec = 'EC 1.8.3.2';
UPDATE enzyme SET sys_name = 'Glutathione:oxygen oxidoreductase' WHERE ec = 'EC 1.8.3.3';
UPDATE enzyme SET history = 'EC 1.8.3.3 created 1989' WHERE ec = 'EC 1.8.3.3';
UPDATE enzyme SET sys_name = 'Methanethiol:oxygen oxidoreductase' WHERE ec = 'EC 1.8.3.4';
UPDATE enzyme SET history = 'EC 1.8.3.4 created 1990' WHERE ec = 'EC 1.8.3.4';
UPDATE enzyme SET sys_name = 'S-prenyl-L-cysteine:oxygen oxidoreductase' WHERE ec = 'EC 1.8.3.5';
UPDATE enzyme SET history = 'EC 1.8.3.5 created 2000 as EC 4.4.1.18, transferred 2002 to EC 1.8.3.5' WHERE ec = 'EC 1.8.3.5';
UPDATE enzyme SET sys_name = 'S-(2E,6E)-farnesyl-L-cysteine oxidase' WHERE ec = 'EC 1.8.3.6';
UPDATE enzyme SET history = 'EC 1.8.3.6 created 2011' WHERE ec = 'EC 1.8.3.6';
UPDATE enzyme SET sys_name = 'Glutathione:homocystine oxidoreductase' WHERE ec = 'EC 1.8.4.1';
UPDATE enzyme SET history = 'EC 1.8.4.1 created 1961' WHERE ec = 'EC 1.8.4.1';
UPDATE enzyme SET sys_name = 'Glutathione:protein-disulfide oxidoreductase' WHERE ec = 'EC 1.8.4.2';
UPDATE enzyme SET history = 'EC 1.8.4.2 created 1965' WHERE ec = 'EC 1.8.4.2';
UPDATE enzyme SET sys_name = 'CoA:glutathione-disulfide oxidoreductase' WHERE ec = 'EC 1.8.4.3';
UPDATE enzyme SET history = 'EC 1.8.4.3 created 1972' WHERE ec = 'EC 1.8.4.3';
UPDATE enzyme SET sys_name = 'Glutathione:cystine oxidoreductase' WHERE ec = 'EC 1.8.4.4';
UPDATE enzyme SET history = 'EC 1.8.4.4 created 1972' WHERE ec = 'EC 1.8.4.4';
UPDATE enzyme SET history = 'EC 1.8.4.5 created 1984, deleted 2006' WHERE ec = 'EC 1.8.4.5';
UPDATE enzyme SET history = 'EC 1.8.4.6 created 1984, deleted 2006' WHERE ec = 'EC 1.8.4.6';
UPDATE enzyme SET sys_name = '[Xanthine-dehydrogenase]:glutathione-disulfide S-oxidoreductase' WHERE ec = 'EC 1.8.4.7';
UPDATE enzyme SET history = 'EC 1.8.4.7 created 1989, modified 2002' WHERE ec = 'EC 1.8.4.7';
UPDATE enzyme SET sys_name = 'Adenosine 3,5-bisphosphate,sulfite:thioredoxin-disulfide oxidoreductase (3-phosphoadenosine-5-phosphosulfate-forming)' WHERE ec = 'EC 1.8.4.8';
UPDATE enzyme SET history = 'EC 1.8.4.8 created 1999 as EC 1.8.99.4, transferred 2000 to EC 1.8.4.8' WHERE ec = 'EC 1.8.4.8';
UPDATE enzyme SET sys_name = 'AMP,sulfite:glutathione-disulfide oxidoreductase (adenosine-5-phosphosulfate-forming)' WHERE ec = 'EC 1.8.4.9';
UPDATE enzyme SET history = 'EC 1.8.4.9 created 2000, modified 2002' WHERE ec = 'EC 1.8.4.9';
UPDATE enzyme SET sys_name = 'AMP,sulfite:thioredoxin-disulfide oxidoreductase (adenosine-5-phosphosulfate-forming)' WHERE ec = 'EC 1.8.4.10';
UPDATE enzyme SET history = 'EC 1.8.4.10 created 2003' WHERE ec = 'EC 1.8.4.10';
UPDATE enzyme SET sys_name = 'Peptide-L-methionine:thioredoxin-disulfide S-oxidoreductase (L-methionine (S)-S-oxide-forming)' WHERE ec = 'EC 1.8.4.11';
UPDATE enzyme SET history = 'EC 1.8.4.11 created 2006' WHERE ec = 'EC 1.8.4.11';
UPDATE enzyme SET sys_name = 'Peptide-methionine:thioredoxin-disulfide S-oxidoreductase (methionine (R)-S-oxide-forming)' WHERE ec = 'EC 1.8.4.12';
UPDATE enzyme SET history = 'EC 1.8.4.12 created 2006' WHERE ec = 'EC 1.8.4.12';
UPDATE enzyme SET sys_name = 'L-methionine:thioredoxin-disulfide S-oxidoreductase' WHERE ec = 'EC 1.8.4.13';
UPDATE enzyme SET history = 'EC 1.8.4.13, created 1984 as EC 1.8.4.5, part-transferred 2006 to EC 1.8.4.13' WHERE ec = 'EC 1.8.4.13';
UPDATE enzyme SET sys_name = 'L-methionine:thioredoxin-disulfide S-oxidoreductase (L-methionine (R)-S-oxide-forming)' WHERE ec = 'EC 1.8.4.14';
UPDATE enzyme SET history = 'EC 1.8.4.14 created 1984 as EC 1.8.4.5, part-transferred 2006 to EC 1.8.4.14' WHERE ec = 'EC 1.8.4.14';
UPDATE enzyme SET sys_name = 'Glutathione:dehydroascorbate oxidoreductase' WHERE ec = 'EC 1.8.5.1';
UPDATE enzyme SET history = 'EC 1.8.5.1 created 1961' WHERE ec = 'EC 1.8.5.1';
UPDATE enzyme SET sys_name = 'Thiosulfate:6-decylubiquinone oxidoreductase' WHERE ec = 'EC 1.8.5.2';
UPDATE enzyme SET history = 'EC 1.8.5.2 created 2004' WHERE ec = 'EC 1.8.5.2';
UPDATE enzyme SET sys_name = 'Dimethylsulfide:menaquinone oxidoreductase' WHERE ec = 'EC 1.8.5.3';
UPDATE enzyme SET history = 'EC 1.8.5.3 created 2011' WHERE ec = 'EC 1.8.5.3';
UPDATE enzyme SET sys_name = 'Sulfide:quinone oxidoreductase' WHERE ec = 'EC 1.8.5.4';
UPDATE enzyme SET history = 'EC 1.8.5.4 created 2011' WHERE ec = 'EC 1.8.5.4';
UPDATE enzyme SET history = 'EC 1.8.6.1 created 1961, deleted 1976' WHERE ec = 'EC 1.8.6.1';
UPDATE enzyme SET sys_name = 'Hydrogen-sulfide:ferredoxin oxidoreductase' WHERE ec = 'EC 1.8.7.1';
UPDATE enzyme SET history = 'EC 1.8.7.1 created 1972' WHERE ec = 'EC 1.8.7.1';
UPDATE enzyme SET sys_name = 'Ferredoxin:thioredoxin disulfide oxidoreductase' WHERE ec = 'EC 1.8.7.2';
UPDATE enzyme SET history = 'EC 1.8.7.2 created 2010' WHERE ec = 'EC 1.8.7.2';
UPDATE enzyme SET sys_name = 'Coenzyme B:coenzyme M:methanophenazine oxidoreductase' WHERE ec = 'EC 1.8.98.1';
UPDATE enzyme SET history = 'EC 1.8.98.1 created 2003' WHERE ec = 'EC 1.8.98.1';
UPDATE enzyme SET sys_name = 'Peroxiredoxin-(S-hydroxy-S-oxocysteine):thiol oxidoreductase (ATP-hydrolyzing; peroxiredoxin-(S-hydroxycysteine)-forming)' WHERE ec = 'EC 1.8.98.2';
UPDATE enzyme SET history = '1.8.98.2 created 2005' WHERE ec = 'EC 1.8.98.2';
UPDATE enzyme SET sys_name = 'Hydrogen-sulfide:acceptor oxidoreductase' WHERE ec = 'EC 1.8.99.1';
UPDATE enzyme SET history = 'EC 1.8.99.1 created 1972' WHERE ec = 'EC 1.8.99.1';
UPDATE enzyme SET sys_name = 'AMP,sulfite:acceptor oxidoreductase (adenosine-5-phosphosulfate-forming)' WHERE ec = 'EC 1.8.99.2';
UPDATE enzyme SET history = 'EC 1.8.99.2 created 1972' WHERE ec = 'EC 1.8.99.2';
UPDATE enzyme SET sys_name = 'Trithionate:acceptor oxidoreductase' WHERE ec = 'EC 1.8.99.3';
UPDATE enzyme SET history = 'EC 1.8.99.3 created 1986' WHERE ec = 'EC 1.8.99.3';
UPDATE enzyme SET history = 'EC 1.8.99.4 created 1999, deleted 2000' WHERE ec = 'EC 1.8.99.4';
UPDATE enzyme SET sys_name = 'Ferrocytochrome-c:oxygen oxidoreductase' WHERE ec = 'EC 1.9.3.1';
UPDATE enzyme SET history = 'EC 1.9.3.1 created 1961, modified 2000' WHERE ec = 'EC 1.9.3.1';
UPDATE enzyme SET history = 'EC 1.9.3.2 created 1965, deleted 2002' WHERE ec = 'EC 1.9.3.2';
UPDATE enzyme SET sys_name = 'Ferrocytochrome:nitrate oxidoreductase' WHERE ec = 'EC 1.9.6.1';
UPDATE enzyme SET history = 'EC 1.9.6.1 created 1961' WHERE ec = 'EC 1.9.6.1';
UPDATE enzyme SET sys_name = 'Ferrocytochrome-c:Fe(3+) oxidoreductase' WHERE ec = 'EC 1.9.99.1';
UPDATE enzyme SET history = 'EC 1.9.99.1 created 1972' WHERE ec = 'EC 1.9.99.1';
UPDATE enzyme SET sys_name = '(+-)-trans-acenaphthene-1,2-diol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.10.1.1';
UPDATE enzyme SET history = 'EC 1.10.1.1 created 1976' WHERE ec = 'EC 1.10.1.1';
UPDATE enzyme SET sys_name = 'L-ascorbate:ferricytochrome-b5 oxidoreductase' WHERE ec = 'EC 1.10.2.1';
UPDATE enzyme SET history = 'EC 1.10.2.1 created 1972, modified 2000' WHERE ec = 'EC 1.10.2.1';
UPDATE enzyme SET sys_name = 'Quinol:ferricytochrome-c oxidoreductase' WHERE ec = 'EC 1.10.2.2';
UPDATE enzyme SET history = 'EC 1.10.2.2 created 1978, modified 2013' WHERE ec = 'EC 1.10.2.2';
UPDATE enzyme SET sys_name = '1,2-benzenediol:oxygen oxidoreductase' WHERE ec = 'EC 1.10.3.1';
UPDATE enzyme SET history = 'EC 1.10.3.1 created 1961, deleted 1972, reinstated 1978' WHERE ec = 'EC 1.10.3.1';
UPDATE enzyme SET sys_name = 'Benzenediol:oxygen oxidoreductase' WHERE ec = 'EC 1.10.3.2';
UPDATE enzyme SET history = 'EC 1.10.3.2 created 1961, deleted 1972, reinstated 1978' WHERE ec = 'EC 1.10.3.2';
UPDATE enzyme SET sys_name = 'L-ascorbate:oxygen oxidoreductase' WHERE ec = 'EC 1.10.3.3';
UPDATE enzyme SET history = 'EC 1.10.3.3 created 1961, modified 2011' WHERE ec = 'EC 1.10.3.3';
UPDATE enzyme SET sys_name = '2-aminophenol:oxygen oxidoreductase' WHERE ec = 'EC 1.10.3.4';
UPDATE enzyme SET history = 'EC 1.10.3.4 created 1972, modified 2006' WHERE ec = 'EC 1.10.3.4';
UPDATE enzyme SET sys_name = '3-hydroxyanthranilate:oxygen oxidoreductase' WHERE ec = 'EC 1.10.3.5';
UPDATE enzyme SET history = 'EC 1.10.3.5 created 1972' WHERE ec = 'EC 1.10.3.5';
UPDATE enzyme SET sys_name = 'Rifamycin-B:oxygen oxidoreductase' WHERE ec = 'EC 1.10.3.6';
UPDATE enzyme SET history = 'EC 1.10.3.6 created 1986' WHERE ec = 'EC 1.10.3.6';
UPDATE enzyme SET history = 'EC 1.10.3.7 created 1986, deleted 2003' WHERE ec = 'EC 1.10.3.7';
UPDATE enzyme SET history = 'EC 1.10.3.8 created 1986, deleted 2003' WHERE ec = 'EC 1.10.3.8';
UPDATE enzyme SET sys_name = 'H(2)O:plastoquinone reductase (light-dependent)' WHERE ec = 'EC 1.10.3.9';
UPDATE enzyme SET history = 'EC 1.10.3.9 created 2011' WHERE ec = 'EC 1.10.3.9';
UPDATE enzyme SET sys_name = 'Ubiquinol:O2 oxidoreductase (H(+)-transporting)' WHERE ec = 'EC 1.10.3.10';
UPDATE enzyme SET history = 'EC 1.10.3.10 created 2011' WHERE ec = 'EC 1.10.3.10';
UPDATE enzyme SET sys_name = 'Ubiquinol:O(2) oxidoreductase (non-electrogenic)' WHERE ec = 'EC 1.10.3.11';
UPDATE enzyme SET history = 'EC 1.10.3.11 created 2011, modified 2014' WHERE ec = 'EC 1.10.3.11';
UPDATE enzyme SET sys_name = 'Menaquinol:O(2) oxidoreductase (H(+)-transporting)' WHERE ec = 'EC 1.10.3.12';
UPDATE enzyme SET history = 'EC 1.10.3.12 created 2011' WHERE ec = 'EC 1.10.3.12';
UPDATE enzyme SET sys_name = 'Caldariellaquinol:O(2) oxidoreductase (H(+)-transporting)' WHERE ec = 'EC 1.10.3.13';
UPDATE enzyme SET history = 'EC 1.10.3.13 created 2013' WHERE ec = 'EC 1.10.3.13';
UPDATE enzyme SET sys_name = 'Ubiquinol:O(2) oxidoreductase (electrogenic, non H(+)-transporting)' WHERE ec = 'EC 1.10.3.14';
UPDATE enzyme SET history = 'EC 1.10.3.14 created 2014' WHERE ec = 'EC 1.10.3.14';
UPDATE enzyme SET sys_name = 'Plastoquinol:oxidized-plastocyanin oxidoreductase' WHERE ec = 'EC 1.10.9.1';
UPDATE enzyme SET history = 'EC 1.10.9.1 created 1984 as EC 1.10.99.1, transferred 2011 to EC 1.10.9.1' WHERE ec = 'EC 1.10.9.1';
UPDATE enzyme SET history = 'EC 1.10.99.1 created 1984, deleted 2012' WHERE ec = 'EC 1.10.99.1';
UPDATE enzyme SET sys_name = '1-(beta-D-ribofuranosyl)-1,4-dihydronicotinamide:quinone oxidoreductase' WHERE ec = 'EC 1.10.99.2';
UPDATE enzyme SET history = 'EC 1.10.99.2 created 2005' WHERE ec = 'EC 1.10.99.2';
UPDATE enzyme SET sys_name = 'Violaxanthin:ascorbate oxidoreductase' WHERE ec = 'EC 1.10.99.3';
UPDATE enzyme SET history = 'EC 1.10.99.3 created 2005' WHERE ec = 'EC 1.10.99.3';
UPDATE enzyme SET sys_name = 'NADH:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.1';
UPDATE enzyme SET history = 'EC 1.11.1.1 created 1961' WHERE ec = 'EC 1.11.1.1';
UPDATE enzyme SET sys_name = 'NADPH:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.2';
UPDATE enzyme SET history = 'EC 1.11.1.2 created 1961' WHERE ec = 'EC 1.11.1.2';
UPDATE enzyme SET sys_name = 'Hexadecanoate:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.3';
UPDATE enzyme SET history = 'EC 1.11.1.3 created 1961' WHERE ec = 'EC 1.11.1.3';
UPDATE enzyme SET history = 'EC 1.11.1.4 created 1961, deleted 1964, reinstated 1965 as EC 1.13.1.12, deleted 1972' WHERE ec = 'EC 1.11.1.4';
UPDATE enzyme SET sys_name = 'Ferrocytochrome-c:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.5';
UPDATE enzyme SET history = 'EC 1.11.1.5 created 1961' WHERE ec = 'EC 1.11.1.5';
UPDATE enzyme SET sys_name = 'Hydrogen-peroxide:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.6';
UPDATE enzyme SET history = 'EC 1.11.1.6 created 1961, modified 1986, modified 1999, modified 2013' WHERE ec = 'EC 1.11.1.6';
UPDATE enzyme SET sys_name = 'Phenolic donor:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.7';
UPDATE enzyme SET history = 'EC 1.11.1.7 created 1961, modified 2011' WHERE ec = 'EC 1.11.1.7';
UPDATE enzyme SET sys_name = 'Iodide:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.8';
UPDATE enzyme SET history = 'EC 1.11.1.8 created 1961, modified 2012' WHERE ec = 'EC 1.11.1.8';
UPDATE enzyme SET sys_name = 'Glutathione:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.9';
UPDATE enzyme SET history = 'EC 1.11.1.9 created 1965, modified 1989' WHERE ec = 'EC 1.11.1.9';
UPDATE enzyme SET sys_name = 'Chloride:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.10';
UPDATE enzyme SET history = 'EC 1.11.1.10 created 1972, modified 2011' WHERE ec = 'EC 1.11.1.10';
UPDATE enzyme SET sys_name = 'L-ascorbate:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.11';
UPDATE enzyme SET history = 'EC 1.11.1.11 created 1983, modified 2011' WHERE ec = 'EC 1.11.1.11';
UPDATE enzyme SET sys_name = 'Glutathione:lipid-hydroperoxide oxidoreductase' WHERE ec = 'EC 1.11.1.12';
UPDATE enzyme SET history = 'EC 1.11.1.12 created 1989' WHERE ec = 'EC 1.11.1.12';
UPDATE enzyme SET sys_name = 'Mn(II):hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.13';
UPDATE enzyme SET history = 'EC 1.11.1.13 created 1992' WHERE ec = 'EC 1.11.1.13';
UPDATE enzyme SET sys_name = '1,2-bis(3,4-dimethoxyphenyl)propane-1,3-diol:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.14';
UPDATE enzyme SET history = 'EC 1.11.1.14 created 1992, modified 2006, modified 2011' WHERE ec = 'EC 1.11.1.14';
UPDATE enzyme SET sys_name = 'Thiol-containing-reductant:hydroperoxide oxidoreductase' WHERE ec = 'EC 1.11.1.15';
UPDATE enzyme SET history = 'EC 1.11.1.15 created 2004' WHERE ec = 'EC 1.11.1.15';
UPDATE enzyme SET sys_name = 'Reactive-black-5:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.16';
UPDATE enzyme SET history = 'EC 1.11.1.16 created 2006' WHERE ec = 'EC 1.11.1.16';
UPDATE enzyme SET sys_name = 'Glutathione amide:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.17';
UPDATE enzyme SET history = 'EC 1.11.1.17 created 2010' WHERE ec = 'EC 1.11.1.17';
UPDATE enzyme SET sys_name = 'Bromide:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.18';
UPDATE enzyme SET history = 'EC 1.11.1.18 created 2010' WHERE ec = 'EC 1.11.1.18';
UPDATE enzyme SET sys_name = 'Reactive-Blue-5:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.19';
UPDATE enzyme SET history = 'EC 1.11.1.19 created 2010' WHERE ec = 'EC 1.11.1.19';
UPDATE enzyme SET sys_name = 'Thioredoxin:(5Z,9-alpha,11-alpha,13E,15S)-9,11-epidioxy-15-hydroxy-prosta-5,13-dienoate oxidoreductase' WHERE ec = 'EC 1.11.1.20';
UPDATE enzyme SET history = 'EC 1.11.1.20 created 2011' WHERE ec = 'EC 1.11.1.20';
UPDATE enzyme SET sys_name = 'Donor:hydrogen-peroxide oxidoreductase' WHERE ec = 'EC 1.11.1.21';
UPDATE enzyme SET history = 'EC 1.11.1.21 created 2011' WHERE ec = 'EC 1.11.1.21';
UPDATE enzyme SET sys_name = 'Hydroperoxy fatty acid:NADPH oxidorductase' WHERE ec = 'EC 1.11.1.22';
UPDATE enzyme SET history = 'EC 1.11.1.22 created 2013' WHERE ec = 'EC 1.11.1.22';
UPDATE enzyme SET sys_name = 'Substrate:hydrogen peroxide oxidoreductase (RH-hydroxylating or-epoxidizing)' WHERE ec = 'EC 1.11.2.1';
UPDATE enzyme SET history = 'EC 1.11.2.1 created 2010' WHERE ec = 'EC 1.11.2.1';
UPDATE enzyme SET sys_name = 'Chloride:hydrogen-peroxide oxidoreductase (hypochlorite-forming)' WHERE ec = 'EC 1.11.2.2';
UPDATE enzyme SET history = 'EC 1.11.2.2 created 2011' WHERE ec = 'EC 1.11.2.2';
UPDATE enzyme SET sys_name = 'Substrate:hydroperoxide oxidoreductase (RH-hydroxylating or epoxidizing)' WHERE ec = 'EC 1.11.2.3';
UPDATE enzyme SET history = 'EC 1.11.2.3 created 2011' WHERE ec = 'EC 1.11.2.3';
UPDATE enzyme SET sys_name = 'Fatty acid:hydroperoxide oxidoreductase (RH-hydroxylating)' WHERE ec = 'EC 1.11.2.4';
UPDATE enzyme SET history = 'EC 1.11.2.4 created 2011' WHERE ec = 'EC 1.11.2.4';
UPDATE enzyme SET history = 'EC 1.12.1.1 created 1965, deleted 1972' WHERE ec = 'EC 1.12.1.1';
UPDATE enzyme SET sys_name = 'Hydrogen:NAD(+) oxidoreductase' WHERE ec = 'EC 1.12.1.2';
UPDATE enzyme SET history = 'EC 1.12.1.2 created 1972, modified 2002' WHERE ec = 'EC 1.12.1.2';
UPDATE enzyme SET sys_name = 'Hydrogen:NADP(+) oxidoreductase' WHERE ec = 'EC 1.12.1.3';
UPDATE enzyme SET history = 'EC 1.12.1.3 created 2002, modified 2013' WHERE ec = 'EC 1.12.1.3';
UPDATE enzyme SET sys_name = 'Hydrogen:NAD(+), ferredoxin oxidoreductase' WHERE ec = 'EC 1.12.1.4';
UPDATE enzyme SET history = 'EC 1.12.1.4 created 2011' WHERE ec = 'EC 1.12.1.4';
UPDATE enzyme SET sys_name = 'Hydrogen:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.12.1.5';
UPDATE enzyme SET history = 'EC 1.12.1.5 created 2013' WHERE ec = 'EC 1.12.1.5';
UPDATE enzyme SET sys_name = 'Hydrogen:ferricytochrome-c3 oxidoreductase' WHERE ec = 'EC 1.12.2.1';
UPDATE enzyme SET history = 'EC 1.12.2.1 created 1972, modified 2002' WHERE ec = 'EC 1.12.2.1';
UPDATE enzyme SET sys_name = 'Hydrogen:quinone oxidoreductase' WHERE ec = 'EC 1.12.5.1';
UPDATE enzyme SET history = 'EC 1.12.5.1 created 1999 as EC 1.12.99.3, transferred 2002 to EC 1.12.5.1' WHERE ec = 'EC 1.12.5.1';
UPDATE enzyme SET history = 'EC 1.12.7.1 created 1972, deleted 1978' WHERE ec = 'EC 1.12.7.1';
UPDATE enzyme SET sys_name = 'Hydrogen:ferredoxin oxidoreductase' WHERE ec = 'EC 1.12.7.2';
UPDATE enzyme SET history = 'EC 1.12.7.2 created 1961 as EC 1.98.1.1, transferred 1965 to EC 1.12.1.1, transferred 1972 to EC 1.12.7.1, transferred 1978 to EC 1.18.3.1, transferred 1984 to EC 1.18.99.1, transferred 2002 to EC 1.12.7.2' WHERE ec = 'EC 1.12.7.2';
UPDATE enzyme SET sys_name = 'Hydrogen:coenzyme F420 oxidoreductase' WHERE ec = 'EC 1.12.98.1';
UPDATE enzyme SET history = 'EC 1.12.98.1 created 1989 as EC 1.12.99.1, transferred 2002 to EC 1.12.98.1' WHERE ec = 'EC 1.12.98.1';
UPDATE enzyme SET sys_name = 'Hydrogen:5,10-methenyltetrahydromethanopterin oxidoreductase' WHERE ec = 'EC 1.12.98.2';
UPDATE enzyme SET history = 'EC 1.12.98.2 created 1999 as EC 1.12.99.4, transferred 2002 to EC 1.12.98.2, modified 2004' WHERE ec = 'EC 1.12.98.2';
UPDATE enzyme SET sys_name = 'Hydrogen:2-(2,3-dihydropentaprenyloxy)phenazine oxidoreductase' WHERE ec = 'EC 1.12.98.3';
UPDATE enzyme SET history = 'EC 1.12.98.3 created 2002' WHERE ec = 'EC 1.12.98.3';
UPDATE enzyme SET sys_name = 'H(2):polysulfide oxidoreductase' WHERE ec = 'EC 1.12.98.4';
UPDATE enzyme SET history = 'EC 1.12.98.4 created 1992 as EC 1.97.1.3, transferred 2013 to EC 1.12.98.4' WHERE ec = 'EC 1.12.98.4';
UPDATE enzyme SET history = 'EC 1.12.99.1 created 1989, deleted 2002' WHERE ec = 'EC 1.12.99.1';
UPDATE enzyme SET history = 'EC 1.12.99.2 created 1992, deleted 2002' WHERE ec = 'EC 1.12.99.2';
UPDATE enzyme SET history = 'EC 1.12.99.3 created 1999, deleted 2002' WHERE ec = 'EC 1.12.99.3';
UPDATE enzyme SET history = 'EC 1.12.99.4 created 1999, deleted 2002' WHERE ec = 'EC 1.12.99.4';
UPDATE enzyme SET history = 'EC 1.12.99.5 created 1999, deleted 2001' WHERE ec = 'EC 1.12.99.5';
UPDATE enzyme SET sys_name = 'Hydrogen:acceptor oxidoreductase' WHERE ec = 'EC 1.12.99.6';
UPDATE enzyme SET history = 'EC 1.12.99.6 created 2002, modified 2003' WHERE ec = 'EC 1.12.99.6';
UPDATE enzyme SET history = 'EC 1.13.1.1 created 1961 as EC 1.99.2.2, transferred 1965 to EC 1.13.1.1, deleted 1972' WHERE ec = 'EC 1.13.1.1';
UPDATE enzyme SET history = 'EC 1.13.1.2 created 1965, deleted 1972' WHERE ec = 'EC 1.13.1.2';
UPDATE enzyme SET history = 'EC 1.13.1.3 created 1961 as EC 1.99.2.3, transferred 1965 to EC 1.13.1.3, deleted 1972' WHERE ec = 'EC 1.13.1.3';
UPDATE enzyme SET history = 'EC 1.13.1.4 created 1961 as EC 1.99.2.4, transferred 1965 to EC 1.13.1.4, deleted 1972' WHERE ec = 'EC 1.13.1.4';
UPDATE enzyme SET history = 'EC 1.13.1.5 created 1961 as EC 1.99.2.5, transferred 1965 to EC 1.13.1.5, deleted 1972' WHERE ec = 'EC 1.13.1.5';
UPDATE enzyme SET history = 'EC 1.13.1.6 created 1965, deleted 1972' WHERE ec = 'EC 1.13.1.6';
UPDATE enzyme SET history = 'EC 1.13.1.7 created 1965, transferred 1972 to EC 1.13.11.7, deleted 1980' WHERE ec = 'EC 1.13.1.7';
UPDATE enzyme SET history = 'EC 1.13.1.8 created 1965, deleted 1972' WHERE ec = 'EC 1.13.1.8';
UPDATE enzyme SET history = 'EC 1.13.1.9 created 1965, deleted 1972' WHERE ec = 'EC 1.13.1.9';
UPDATE enzyme SET history = 'EC 1.13.1.10 created 1965, deleted 1972' WHERE ec = 'EC 1.13.1.10';
UPDATE enzyme SET history = 'EC 1.13.1.11 created 1961 as EC 1.99.2.6, transferred1965 to EC 1.13.1.11, deleted 1972' WHERE ec = 'EC 1.13.1.11';
UPDATE enzyme SET history = 'EC 1.13.1.12 created 1961 as EC 1.11.1.4, deleted 1964, reinstated 1965 as EC 1.13.1.12, deleted 1972' WHERE ec = 'EC 1.13.1.12';
UPDATE enzyme SET history = 'EC 1.13.1.13 created 1961 as EC 1.99.2.1, transferred 1965 to EC 1.13.1.13, deleted 1972' WHERE ec = 'EC 1.13.1.13';
UPDATE enzyme SET sys_name = 'Catechol:oxygen 1,2-oxidoreductase' WHERE ec = 'EC 1.13.11.1';
UPDATE enzyme SET history = 'EC 1.13.11.1 created 1961 as EC 1.99.2.2, transferred 1965 to EC 1.13.1.1, transferred 1972 to EC 1.13.11.1' WHERE ec = 'EC 1.13.11.1';
UPDATE enzyme SET sys_name = 'Catechol:oxygen 2,3-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.2';
UPDATE enzyme SET history = 'EC 1.13.11.2 created 1965 as EC 1.13.1.2, transferred 1972 to EC 1.13.11.2, modified 1999, modified 2013' WHERE ec = 'EC 1.13.11.2';
UPDATE enzyme SET sys_name = 'Protocatechuate:oxygen 3,4-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.3';
UPDATE enzyme SET history = 'EC 1.13.11.3 created 1961 as EC 1.99.2.3, transferred 1965 to EC 1.13.1.3, transferred 1972 to EC 1.13.11.3' WHERE ec = 'EC 1.13.11.3';
UPDATE enzyme SET sys_name = 'Gentisate:oxygen 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.4';
UPDATE enzyme SET history = 'EC 1.13.11.4 created 1961 as EC 1.99.2.4, transferred 1965 to EC 1.13.1.4, transferred 1972 to EC 1.13.11.4' WHERE ec = 'EC 1.13.11.4';
UPDATE enzyme SET sys_name = 'Homogentisate:oxygen 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.5';
UPDATE enzyme SET history = 'EC 1.13.11.5 created 1961 as EC 1.99.2.5, transferred 1965 to EC 1.13.1.5, transferred 1972 to EC 1.13.11.5' WHERE ec = 'EC 1.13.11.5';
UPDATE enzyme SET sys_name = '3-hydroxyanthranilate:oxygen 3,4-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.6';
UPDATE enzyme SET history = 'EC 1.13.11.6 created 1965 as EC 1.13.1.6, transferred 1972 to EC 1.13.11.6' WHERE ec = 'EC 1.13.11.6';
UPDATE enzyme SET history = 'EC 1.13.11.7 created 1965 as EC 1.13.1.7, transferred 1972 to EC 1.13.11.7, deleted 1980' WHERE ec = 'EC 1.13.11.7';
UPDATE enzyme SET sys_name = 'Protocatechuate:oxygen 4,5-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.8';
UPDATE enzyme SET history = 'EC 1.13.11.8 created 1965 as EC 1.13.1.8, transferred 1972 to EC 1.13.11.8' WHERE ec = 'EC 1.13.11.8';
UPDATE enzyme SET sys_name = '2,5-dihydroxypyridine:oxygen 5,6-oxidoreductase' WHERE ec = 'EC 1.13.11.9';
UPDATE enzyme SET history = 'EC 1.13.11.9 created 1965 as EC 1.13.1.9, transferred 1972 to EC 1.13.11.9, modified 2010' WHERE ec = 'EC 1.13.11.9';
UPDATE enzyme SET sys_name = '7,8-dihydroxykynurenate:oxygen 8,8a-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.10';
UPDATE enzyme SET history = 'EC 1.13.11.10 created 1965 as EC 1.13.1.10, transferred 1972 to EC 1.13.11.10' WHERE ec = 'EC 1.13.11.10';
UPDATE enzyme SET sys_name = 'L-tryptophan:oxygen 2,3-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.11';
UPDATE enzyme SET history = 'EC 1.13.11.11 created 1961 as EC 1.11.1.4, deleted 1964, reinstated 1965 as EC 1.13.1.12, transferred 1972 to EC 1.13.11.11, modified 1989, modified 2006' WHERE ec = 'EC 1.13.11.11';
UPDATE enzyme SET sys_name = 'Linoleate:oxygen 13-oxidoreductase' WHERE ec = 'EC 1.13.11.12';
UPDATE enzyme SET history = 'EC 1.13.11.12 created 1961 as EC 1.99.2.1, transferred 1965 to EC 1.13.1.13, transferred 1972 to EC 1.13.11.12, modified 2011, modified 2012' WHERE ec = 'EC 1.13.11.12';
UPDATE enzyme SET history = 'EC 1.13.11.13 created 1972, deleted 2012' WHERE ec = 'EC 1.13.11.13';
UPDATE enzyme SET sys_name = '2,3-dihydroxybenzoate:oxygen 3,4-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.14';
UPDATE enzyme SET history = 'EC 1.13.11.14 created 1972, modified 1976' WHERE ec = 'EC 1.13.11.14';
UPDATE enzyme SET sys_name = '3,4-dihydroxyphenylacetate:oxygen 2,3-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.15';
UPDATE enzyme SET history = 'EC 1.13.11.15 created 1972' WHERE ec = 'EC 1.13.11.15';
UPDATE enzyme SET sys_name = '3-(2,3-dihydroxyphenyl)propanoate:oxygen 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.16';
UPDATE enzyme SET history = 'EC 1.13.11.16 created 1972, modified 2011, modified 2012' WHERE ec = 'EC 1.13.11.16';
UPDATE enzyme SET sys_name = 'Indole:oxygen 2,3-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.17';
UPDATE enzyme SET history = 'EC 1.13.11.17 created 1972, modified 1986' WHERE ec = 'EC 1.13.11.17';
UPDATE enzyme SET sys_name = 'S-sulfanylglutathione:oxygen oxidoreductase' WHERE ec = 'EC 1.13.11.18';
UPDATE enzyme SET history = 'EC 1.13.11.18 created 1972' WHERE ec = 'EC 1.13.11.18';
UPDATE enzyme SET sys_name = '2-aminoethanethiol:oxygen oxidoreductase' WHERE ec = 'EC 1.13.11.19';
UPDATE enzyme SET history = 'EC 1.13.11.19 created 1972, modified 2006' WHERE ec = 'EC 1.13.11.19';
UPDATE enzyme SET sys_name = 'L-cysteine:oxygen oxidoreductase' WHERE ec = 'EC 1.13.11.20';
UPDATE enzyme SET history = 'EC 1.13.11.20 created 1972, modified 1976' WHERE ec = 'EC 1.13.11.20';
UPDATE enzyme SET history = 'EC 1.13.11.21 created 1972, deleted 2001' WHERE ec = 'EC 1.13.11.21';
UPDATE enzyme SET sys_name = '3,4-dihydroxy-trans-cinnamate:oxygen 3,4-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.22';
UPDATE enzyme SET history = 'EC 1.13.11.22 created 1972' WHERE ec = 'EC 1.13.11.22';
UPDATE enzyme SET sys_name = '2,3-dihydroxyindole:oxygen 2,3-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.23';
UPDATE enzyme SET history = 'EC 1.13.11.23 created 1972' WHERE ec = 'EC 1.13.11.23';
UPDATE enzyme SET sys_name = 'Quercetin:oxygen 2,3-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.24';
UPDATE enzyme SET history = 'EC 1.13.11.24 created 1972' WHERE ec = 'EC 1.13.11.24';
UPDATE enzyme SET sys_name = '3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione:oxygen 4,5-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.25';
UPDATE enzyme SET history = 'EC 1.13.11.25 created 1972' WHERE ec = 'EC 1.13.11.25';
UPDATE enzyme SET sys_name = '[Protein]-L-tryptophan:oxygen 2,3-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.26';
UPDATE enzyme SET history = 'EC 1.13.11.26 created 1972, modified 2011' WHERE ec = 'EC 1.13.11.26';
UPDATE enzyme SET sys_name = '4-hydroxyphenylpyruvate:oxygen oxidoreductase (hydroxylating, decarboxylating)' WHERE ec = 'EC 1.13.11.27';
UPDATE enzyme SET history = 'EC 1.13.11.27 created 1961 as EC 1.99.1.14, transferred 1965 to EC 1.14.2.2, transferred 1972 to EC 1.13.11.27' WHERE ec = 'EC 1.13.11.27';
UPDATE enzyme SET sys_name = '2,3-dihydroxybenzoate:oxygen 2,3-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.28';
UPDATE enzyme SET history = 'EC 1.13.11.28 created 1978' WHERE ec = 'EC 1.13.11.28';
UPDATE enzyme SET sys_name = '3,4-dihydroxy-L-phenylalanine:oxygen 4,5-oxidoreductase (recyclizing)' WHERE ec = 'EC 1.13.11.29';
UPDATE enzyme SET history = 'EC 1.13.11.29 created 1978' WHERE ec = 'EC 1.13.11.29';
UPDATE enzyme SET sys_name = '3,4-dihydroxy-L-phenylalanine:oxygen 2,3-oxidoreductase (recyclizing)' WHERE ec = 'EC 1.13.11.30';
UPDATE enzyme SET history = 'EC 1.13.11.30 created 1978' WHERE ec = 'EC 1.13.11.30';
UPDATE enzyme SET sys_name = 'Arachidonate:oxygen 12-oxidoreductase' WHERE ec = 'EC 1.13.11.31';
UPDATE enzyme SET history = 'EC 1.13.11.31 created 1983' WHERE ec = 'EC 1.13.11.31';
UPDATE enzyme SET history = 'EC 1.13.11.32 created 1984, modified 2006, deleted 2010' WHERE ec = 'EC 1.13.11.32';
UPDATE enzyme SET sys_name = 'Arachidonate:oxygen 15-oxidoreductase' WHERE ec = 'EC 1.13.11.33';
UPDATE enzyme SET history = 'EC 1.13.11.33 created 1984' WHERE ec = 'EC 1.13.11.33';
UPDATE enzyme SET sys_name = 'Arachidonate:oxygen 5-oxidoreductase' WHERE ec = 'EC 1.13.11.34';
UPDATE enzyme SET history = 'EC 1.13.11.34 created 1984, modified 1990' WHERE ec = 'EC 1.13.11.34';
UPDATE enzyme SET sys_name = '1,2,3-trihydroxybenzene:oxygen 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.35';
UPDATE enzyme SET history = 'EC 1.13.11.35 created 1984, modified 2012' WHERE ec = 'EC 1.13.11.35';
UPDATE enzyme SET sys_name = '5-amino-4-chloro-2-(2,3-dihydroxyphenyl)-3(2H)-pyridazinone 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.36';
UPDATE enzyme SET history = 'EC 1.13.11.36 created 1984' WHERE ec = 'EC 1.13.11.36';
UPDATE enzyme SET sys_name = 'Benzene-1,2,4-triol:oxygen 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.37';
UPDATE enzyme SET history = 'EC 1.13.11.37 created 1989, modified 2013' WHERE ec = 'EC 1.13.11.37';
UPDATE enzyme SET sys_name = '1-hydroxy-2-naphthoate:oxygen 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.38';
UPDATE enzyme SET history = 'EC 1.13.11.38 created 1989' WHERE ec = 'EC 1.13.11.38';
UPDATE enzyme SET sys_name = 'Biphenyl-2,3-diol:oxygen 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.39';
UPDATE enzyme SET history = 'EC 1.13.11.39 created 1989, modified 2010' WHERE ec = 'EC 1.13.11.39';
UPDATE enzyme SET sys_name = 'Arachidonate:oxygen 8-oxidoreductase' WHERE ec = 'EC 1.13.11.40';
UPDATE enzyme SET history = 'EC 1.13.11.40 created 1989' WHERE ec = 'EC 1.13.11.40';
UPDATE enzyme SET sys_name = '2,4-dihydroxyacetophenone oxidoreductase (C-C-bond-cleaving)' WHERE ec = 'EC 1.13.11.41';
UPDATE enzyme SET history = 'EC 1.13.11.41 created 1989' WHERE ec = 'EC 1.13.11.41';
UPDATE enzyme SET history = 'EC 1.13.11.42 created 1992, deleted 2006' WHERE ec = 'EC 1.13.11.42';
UPDATE enzyme SET sys_name = '1,2-bis(4-hydroxy-3-methoxyphenyl)ethylene:oxygen oxidoreductase (alpha-beta-bond-cleaving)' WHERE ec = 'EC 1.13.11.43';
UPDATE enzyme SET history = 'EC 1.13.11.43 created 1992' WHERE ec = 'EC 1.13.11.43';
UPDATE enzyme SET history = 'EC 1.13.11.44 created 2000, deleted 2012' WHERE ec = 'EC 1.13.11.44';
UPDATE enzyme SET sys_name = 'Linoleate:oxygen 11S-oxidoreductase' WHERE ec = 'EC 1.13.11.45';
UPDATE enzyme SET history = 'EC 1.13.11.45 created 2000' WHERE ec = 'EC 1.13.11.45';
UPDATE enzyme SET sys_name = '4-hydroxyphenylpyruvate:oxygen oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.11.46';
UPDATE enzyme SET history = 'EC 1.13.11.46 created 2001' WHERE ec = 'EC 1.13.11.46';
UPDATE enzyme SET sys_name = '3-hydroxy-1H-quinolin-4-one 2,4-dioxygenase (CO-forming)' WHERE ec = 'EC 1.13.11.47';
UPDATE enzyme SET history = 'EC 1.13.11.47 created 1999 as EC 1.13.99.5, transferred 2001 to EC 1.13.11.47 (EC 1.12.99.5 created 1999 deleted 2001 as identical)' WHERE ec = 'EC 1.13.11.47';
UPDATE enzyme SET sys_name = '3-hydroxy-2-methyl-1H-quinolin-4-one 2,4-dioxygenase (CO-forming)' WHERE ec = 'EC 1.13.11.48';
UPDATE enzyme SET history = 'EC 1.13.11.48 created 2001' WHERE ec = 'EC 1.13.11.48';
UPDATE enzyme SET sys_name = 'Chloride:oxygen oxidoreductase' WHERE ec = 'EC 1.13.11.49';
UPDATE enzyme SET history = 'EC 1.13.11.49 created 2001' WHERE ec = 'EC 1.13.11.49';
UPDATE enzyme SET sys_name = 'Acetylacetone:oxygen oxidoreductase' WHERE ec = 'EC 1.13.11.50';
UPDATE enzyme SET history = 'EC 1.13.11.50 created 2003' WHERE ec = 'EC 1.13.11.50';
UPDATE enzyme SET sys_name = '9-cis-epoxycarotenoid 11,12-dioxygenase' WHERE ec = 'EC 1.13.11.51';
UPDATE enzyme SET history = 'EC 1.13.11.51 created 2005' WHERE ec = 'EC 1.13.11.51';
UPDATE enzyme SET sys_name = 'D-tryptophan:oxygen 2,3-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.52';
UPDATE enzyme SET history = 'EC 1.13.11.52 created 2006' WHERE ec = 'EC 1.13.11.52';
UPDATE enzyme SET sys_name = '1,2-dihydroxy-5-(methylthio)pent-1-en-3-one:oxygen oxidoreductase (formate- and CO-forming)' WHERE ec = 'EC 1.13.11.53';
UPDATE enzyme SET history = 'EC 1.13.11.53 created 2006' WHERE ec = 'EC 1.13.11.53';
UPDATE enzyme SET sys_name = '1,2-dihydroxy-5-(methylthio)pent-1-en-3-one:oxygen oxidoreductase (formate-forming)' WHERE ec = 'EC 1.13.11.54';
UPDATE enzyme SET history = 'EC 1.13.11.54 created 2006' WHERE ec = 'EC 1.13.11.54';
UPDATE enzyme SET sys_name = 'Sulfur:oxygen oxidoreductase (hydrogen-sulfide- and sulfite-forming)' WHERE ec = 'EC 1.13.11.55';
UPDATE enzyme SET history = 'EC 1.13.11.55 created 2006' WHERE ec = 'EC 1.13.11.55';
UPDATE enzyme SET sys_name = '1,2-dihydroxynaphthalene:oxygen oxidoreductase' WHERE ec = 'EC 1.13.11.56';
UPDATE enzyme SET history = 'EC 1.13.11.56 created 2010' WHERE ec = 'EC 1.13.11.56';
UPDATE enzyme SET sys_name = 'Gallate:oxygen oxidoreductase' WHERE ec = 'EC 1.13.11.57';
UPDATE enzyme SET history = 'EC 1.13.11.57 created 2011' WHERE ec = 'EC 1.13.11.57';
UPDATE enzyme SET sys_name = 'Linoleate:oxygen 9S-oxidoreductase' WHERE ec = 'EC 1.13.11.58';
UPDATE enzyme SET history = 'EC 1.13.11.58 created 2011' WHERE ec = 'EC 1.13.11.58';
UPDATE enzyme SET sys_name = 'Torulene:oxygen oxidoreductase' WHERE ec = 'EC 1.13.11.59';
UPDATE enzyme SET history = 'EC 1.13.11.59 created 2011' WHERE ec = 'EC 1.13.11.59';
UPDATE enzyme SET sys_name = 'Linoleate:oxygen (8R)-oxidoreductase' WHERE ec = 'EC 1.13.11.60';
UPDATE enzyme SET history = 'EC 1.13.11.60 created 2011' WHERE ec = 'EC 1.13.11.60';
UPDATE enzyme SET sys_name = 'Alpha-linolenate:oxygen (9R)-oxidoreductase' WHERE ec = 'EC 1.13.11.61';
UPDATE enzyme SET history = 'EC 1.13.11.61 created 2011' WHERE ec = 'EC 1.13.11.61';
UPDATE enzyme SET sys_name = 'Linoleate:oxygen (10R)-oxidoreductase' WHERE ec = 'EC 1.13.11.62';
UPDATE enzyme SET history = 'EC 1.13.11.62 created 2011' WHERE ec = 'EC 1.13.11.62';
UPDATE enzyme SET sys_name = 'Beta-carotene:oxygen 15,15-dioxygenase (bond-cleaving)' WHERE ec = 'EC 1.13.11.63';
UPDATE enzyme SET history = 'EC 1.13.11.63 created 2012' WHERE ec = 'EC 1.13.11.63';
UPDATE enzyme SET sys_name = '5-nitrosalicylate:oxygen 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.64';
UPDATE enzyme SET history = 'EC 1.13.11.64 created 2012' WHERE ec = 'EC 1.13.11.64';
UPDATE enzyme SET sys_name = 'Zeaxanthin:oxygen 15,15-oxidoreductase (bond-cleaving, cis-isomerizing)' WHERE ec = 'EC 1.13.11.65';
UPDATE enzyme SET history = 'EC 1.13.11.65 created 2012 as EC 1.14.13.164, transferred 2012 to EC 1.13.11.65' WHERE ec = 'EC 1.13.11.65';
UPDATE enzyme SET sys_name = 'Benzene-1,4-diol:oxygen 1,2-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.66';
UPDATE enzyme SET history = 'EC 1.13.11.66 created 2012' WHERE ec = 'EC 1.13.11.66';
UPDATE enzyme SET sys_name = '8-apo-beta-carotenol:O(2) oxidoreductase (14,13-cleaving)' WHERE ec = 'EC 1.13.11.67';
UPDATE enzyme SET history = 'EC 1.13.11.67 created 2000 as EC 1.13.12.12, transferred 2012 to EC 1.13.11.67' WHERE ec = 'EC 1.13.11.67';
UPDATE enzyme SET sys_name = '9-cis-beta-carotene:O(2) oxidoreductase (9,10-cleaving)' WHERE ec = 'EC 1.13.11.68';
UPDATE enzyme SET history = 'EC 1.13.11.68 created 2012' WHERE ec = 'EC 1.13.11.68';
UPDATE enzyme SET sys_name = '9-cis-10-apo-beta-carotenal:O(2) oxidoreductase (14,15-cleaving, carlactone-forming)' WHERE ec = 'EC 1.13.11.69';
UPDATE enzyme SET history = 'EC 1.13.11.69 created 2012' WHERE ec = 'EC 1.13.11.69';
UPDATE enzyme SET sys_name = 'All-trans-10-apo-beta-carotenal:O(2) oxidoreductase (13,14-cleaving)' WHERE ec = 'EC 1.13.11.70';
UPDATE enzyme SET history = 'EC 1.13.11.70 created 2012' WHERE ec = 'EC 1.13.11.70';
UPDATE enzyme SET sys_name = 'All-trans-beta-carotene:O(2) oxidoreductase (9,10-cleaving)' WHERE ec = 'EC 1.13.11.71';
UPDATE enzyme SET history = 'EC 1.13.11.71 created 2012' WHERE ec = 'EC 1.13.11.71';
UPDATE enzyme SET sys_name = '2-hydroxyethylphosphonate:O(2) 1,2-oxidoreductase (hydroxymethylphosphonate forming)' WHERE ec = 'EC 1.13.11.72';
UPDATE enzyme SET history = 'EC 1.13.11.72 created 2012' WHERE ec = 'EC 1.13.11.72';
UPDATE enzyme SET sys_name = '2-hydroxyethylphosphonate:O(2) 1,2-oxidoreductase (methylphosphonate forming)' WHERE ec = 'EC 1.13.11.73';
UPDATE enzyme SET history = 'EC 1.13.11.73 created 2012' WHERE ec = 'EC 1.13.11.73';
UPDATE enzyme SET sys_name = '2-aminophenol:oxygen 1,6-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.74';
UPDATE enzyme SET history = 'EC 1.13.11.74 created 2013' WHERE ec = 'EC 1.13.11.74';
UPDATE enzyme SET sys_name = 'All-trans-8-apo-beta-carotenal:oxygen 15,15-oxidoreductase (bond-cleaving)' WHERE ec = 'EC 1.13.11.75';
UPDATE enzyme SET history = 'EC 1.13.11.75 created 2010 as EC 1.14.99.41, transferred 2013 to EC 1.13.11.75' WHERE ec = 'EC 1.13.11.75';
UPDATE enzyme SET sys_name = '2-amino-5-chlorophenol:oxygen 1,6-oxidoreductase (decyclizing)' WHERE ec = 'EC 1.13.11.76';
UPDATE enzyme SET history = 'EC 1.13.11.76 created 2013' WHERE ec = 'EC 1.13.11.76';
UPDATE enzyme SET sys_name = 'Oleate:oxygen (10S)-oxidoreductase' WHERE ec = 'EC 1.13.11.77';
UPDATE enzyme SET history = 'EC 1.13.11.77 created 2013' WHERE ec = 'EC 1.13.11.77';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.13.11.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.13.11.n2';
UPDATE enzyme SET sys_name = 'L-arginine:oxygen 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.12.1';
UPDATE enzyme SET history = 'EC 1.13.12.1 created 1972' WHERE ec = 'EC 1.13.12.1';
UPDATE enzyme SET sys_name = 'L-lysine:oxygen 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.12.2';
UPDATE enzyme SET history = 'EC 1.13.12.2 created 1972' WHERE ec = 'EC 1.13.12.2';
UPDATE enzyme SET sys_name = 'L-tryptophan:oxygen 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.12.3';
UPDATE enzyme SET history = 'EC 1.13.12.3 created 1972' WHERE ec = 'EC 1.13.12.3';
UPDATE enzyme SET sys_name = '(S)-lactate:oxygen 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.12.4';
UPDATE enzyme SET history = 'EC 1.13.12.4 created 1961 as EC 1.1.3.2, transferred 1972 to EC 1.13.12.4' WHERE ec = 'EC 1.13.12.4';
UPDATE enzyme SET sys_name = 'Renilla-luciferin:oxygen 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.12.5';
UPDATE enzyme SET history = 'EC 1.13.12.5 created 1976, modified 1981, modified 1982, modified 2004' WHERE ec = 'EC 1.13.12.5';
UPDATE enzyme SET sys_name = 'Cypridina-luciferin:oxygen 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.12.6';
UPDATE enzyme SET history = 'EC 1.13.12.6 created 1976, modified 1982' WHERE ec = 'EC 1.13.12.6';
UPDATE enzyme SET sys_name = 'Photinus-luciferin:oxygen 4-oxidoreductase (decarboxylating, ATP-hydrolyzing)' WHERE ec = 'EC 1.13.12.7';
UPDATE enzyme SET history = 'EC 1.13.12.7 created 1976, modified 1981, modified 1982' WHERE ec = 'EC 1.13.12.7';
UPDATE enzyme SET sys_name = 'Watasenia-luciferin:oxygen 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.12.8';
UPDATE enzyme SET history = 'EC 1.13.12.8 created 1982' WHERE ec = 'EC 1.13.12.8';
UPDATE enzyme SET sys_name = 'L-phenylalanine:oxygen 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.12.9';
UPDATE enzyme SET history = 'EC 1.13.12.9 created 1986, modified 2003' WHERE ec = 'EC 1.13.12.9';
UPDATE enzyme SET history = 'EC 1.13.12.10 created 1989, modified 1999, deleted 2001' WHERE ec = 'EC 1.13.12.10';
UPDATE enzyme SET history = 'EC 1.13.12.11 created 1992, deleted 2006' WHERE ec = 'EC 1.13.12.11';
UPDATE enzyme SET history = 'EC 1.13.12.12 created 2000, modified 2001, deleted 2012' WHERE ec = 'EC 1.13.12.12';
UPDATE enzyme SET sys_name = 'Oplophorus-luciferin:oxygen 2-oxidoreductase (decarboxylating)' WHERE ec = 'EC 1.13.12.13';
UPDATE enzyme SET history = 'EC 1.13.12.13 created 2004' WHERE ec = 'EC 1.13.12.13';
UPDATE enzyme SET history = 'EC 1.13.12.14 created 2006, deleted 2011' WHERE ec = 'EC 1.13.12.14';
UPDATE enzyme SET sys_name = '3,4-dihydroxy-L-phenylalanine:oxygen oxidoreductase (deaminating)' WHERE ec = 'EC 1.13.12.15';
UPDATE enzyme SET history = 'EC 1.13.12.15 created 2008' WHERE ec = 'EC 1.13.12.15';
UPDATE enzyme SET sys_name = 'Nitronate:oxygen 2-oxidoreductase (nitrite-forming)' WHERE ec = 'EC 1.13.12.16';
UPDATE enzyme SET history = 'EC 1.13.12.16 created 1984 as EC 1.13.11.32, transferred 2009 to EC 1.13.12.16, modified 2011' WHERE ec = 'EC 1.13.12.16';
UPDATE enzyme SET sys_name = 'Dichlorochromopyrrolate,NADH:oxygen 2,5-oxidoreductase (dichloroarcyriaflavin A-forming)' WHERE ec = 'EC 1.13.12.17';
UPDATE enzyme SET history = 'EC 1.13.12.17 created 2010' WHERE ec = 'EC 1.13.12.17';
UPDATE enzyme SET sys_name = 'Gonyaulax-luciferin:oxygen 13(2)-oxidoreductase' WHERE ec = 'EC 1.13.12.18';
UPDATE enzyme SET history = 'EC 1.13.12.18 created 2011' WHERE ec = 'EC 1.13.12.18';
UPDATE enzyme SET sys_name = '2-oxuglutarate:oxygen oxidoreductase (decarboxylating, ethylene-forming)' WHERE ec = 'EC 1.13.12.19';
UPDATE enzyme SET history = 'EC 1.13.12.19 created 2011' WHERE ec = 'EC 1.13.12.19';
UPDATE enzyme SET sys_name = 'Norsolorinic acid anthrone:oxygen 9-oxidoreductase (norsolorinic acid-forming)' WHERE ec = 'EC 1.13.12.20';
UPDATE enzyme SET history = 'EC 1.13.12.20 created 2013' WHERE ec = 'EC 1.13.12.20';
UPDATE enzyme SET sys_name = 'Tetracenomycin-F1:oxygen C5-monooxygenase' WHERE ec = 'EC 1.13.12.21';
UPDATE enzyme SET history = 'EC 1.13.12.21 created 2013' WHERE ec = 'EC 1.13.12.21';
UPDATE enzyme SET sys_name = 'Myo-inositol:oxygen oxidoreductase' WHERE ec = 'EC 1.13.99.1';
UPDATE enzyme SET history = 'EC 1.13.99.1 created 1961 as EC 1.99.2.6, transferred 1965 to EC 1.13.1.11, transferred 1972 to EC 1.13.99.1, modified 2002' WHERE ec = 'EC 1.13.99.1';
UPDATE enzyme SET history = 'EC 1.13.99.2 created 1972, deleted 1992' WHERE ec = 'EC 1.13.99.2';
UPDATE enzyme SET sys_name = 'L-tryptophan:oxygen 2-oxidoreductase (side-chain-cleaving)' WHERE ec = 'EC 1.13.99.3';
UPDATE enzyme SET history = 'EC 1.13.99.3 created 1984' WHERE ec = 'EC 1.13.99.3';
UPDATE enzyme SET history = 'EC 1.13.99.4 created 1989, deleted 1992' WHERE ec = 'EC 1.13.99.4';
UPDATE enzyme SET history = 'EC 1.13.99.5 created 1999, deleted 2001' WHERE ec = 'EC 1.13.99.5';
UPDATE enzyme SET history = 'EC 1.14.1.1 created 1961 as EC 1.99.1.1, transferred 1965, deleted 1972' WHERE ec = 'EC 1.14.1.1';
UPDATE enzyme SET history = 'EC 1.14.1.2 created 1965, deleted 1972' WHERE ec = 'EC 1.14.1.2';
UPDATE enzyme SET history = 'EC 1.14.1.3 created 1961 as EC 1.99.1.13, transferred 1965 to EC 1.14.1.3, deleted 1972' WHERE ec = 'EC 1.14.1.3';
UPDATE enzyme SET history = 'EC 1.14.1.4 created 1965, deleted 1972' WHERE ec = 'EC 1.14.1.4';
UPDATE enzyme SET history = 'EC 1.14.1.5 created 1965, deleted 1972' WHERE ec = 'EC 1.14.1.5';
UPDATE enzyme SET history = 'EC 1.14.1.6 created 1961 as EC 1.99.1.7, transferred 1965 to EC 1.14.1.6, deleted 1972' WHERE ec = 'EC 1.14.1.6';
UPDATE enzyme SET history = 'EC 1.14.1.7 created 1965, deleted 1972' WHERE ec = 'EC 1.14.1.7';
UPDATE enzyme SET history = 'EC 1.14.1.8 created 1965, deleted 1972' WHERE ec = 'EC 1.14.1.8';
UPDATE enzyme SET history = 'EC 1.14.1.9 created 1965, deleted 1972' WHERE ec = 'EC 1.14.1.9';
UPDATE enzyme SET history = 'EC 1.14.1.10 created 1965, deleted 1972' WHERE ec = 'EC 1.14.1.10';
UPDATE enzyme SET history = 'EC 1.14.1.11 created 1965, deleted 1972' WHERE ec = 'EC 1.14.1.11';
UPDATE enzyme SET history = 'EC 1.14.2.1 created 1965, deleted 1972' WHERE ec = 'EC 1.14.2.1';
UPDATE enzyme SET history = 'EC 1.14.2.2 created 1961 as EC 1.99.1.14, transferred 1965 to EC 1.14.2.2, deleted 1972' WHERE ec = 'EC 1.14.2.2';
UPDATE enzyme SET history = 'EC 1.14.3.1 created 1961 as EC 1.99.1.2, transferred 1965, deleted 1972' WHERE ec = 'EC 1.14.3.1';
UPDATE enzyme SET sys_name = '4-trimethylammoniobutanoate,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.1';
UPDATE enzyme SET history = 'EC 1.14.11.1 created 1972' WHERE ec = 'EC 1.14.11.1';
UPDATE enzyme SET sys_name = 'Procollagen-L-proline,2-oxoglutarate:oxygen oxidoreductase (4-hydroxylating)' WHERE ec = 'EC 1.14.11.2';
UPDATE enzyme SET history = 'EC 1.14.11.2 created 1972, modified 1981, modified 1983' WHERE ec = 'EC 1.14.11.2';
UPDATE enzyme SET sys_name = '2-deoxyuridine,2-oxoglutarate:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.11.3';
UPDATE enzyme SET history = 'EC 1.14.11.3 created 1972, modified 1976, modified 1989, modified 2002' WHERE ec = 'EC 1.14.11.3';
UPDATE enzyme SET sys_name = 'L-lysine-[procollagen],2-oxoglutarate:oxygen oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.11.4';
UPDATE enzyme SET history = 'EC 1.14.11.4 created 1972, modified 1983' WHERE ec = 'EC 1.14.11.4';
UPDATE enzyme SET history = 'EC 1.14.11.5 created 1972, deleted 1976' WHERE ec = 'EC 1.14.11.5';
UPDATE enzyme SET sys_name = 'Thymine,2-oxoglutarate:oxygen oxidoreductase (7-hydroxylating)' WHERE ec = 'EC 1.14.11.6';
UPDATE enzyme SET history = 'EC 1.14.11.6 created 1972, modified 1976 (EC 1.14.11.5 created 1972, incorporated 1976)' WHERE ec = 'EC 1.14.11.6';
UPDATE enzyme SET sys_name = 'Procollagen-L-proline,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.7';
UPDATE enzyme SET history = 'EC 1.14.11.7 created 1981, modified 1983' WHERE ec = 'EC 1.14.11.7';
UPDATE enzyme SET sys_name = 'N(6),N(6),N(6)-trimethyl-L-lysine,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.8';
UPDATE enzyme SET history = 'EC 1.14.11.8 created 1983' WHERE ec = 'EC 1.14.11.8';
UPDATE enzyme SET sys_name = 'Flavanone,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.9';
UPDATE enzyme SET history = 'EC 1.14.11.9 created 1983, modified 1989, modified 2004' WHERE ec = 'EC 1.14.11.9';
UPDATE enzyme SET sys_name = '2-deoxyuridine,2-oxoglutarate:oxygen oxidoreductase (1-hydroxylating)' WHERE ec = 'EC 1.14.11.10';
UPDATE enzyme SET history = 'EC 1.14.11.10 created 1989, modified 2002' WHERE ec = 'EC 1.14.11.10';
UPDATE enzyme SET sys_name = 'L-hyoscyamine,2-oxoglutarate:oxygen oxidoreductase ((6S)-hydroxylating)' WHERE ec = 'EC 1.14.11.11';
UPDATE enzyme SET history = 'EC 1.14.11.11 created 1989' WHERE ec = 'EC 1.14.11.11';
UPDATE enzyme SET sys_name = '(Gibberellin-44),2-oxoglutarate:oxygen oxidoreductase' WHERE ec = 'EC 1.14.11.12';
UPDATE enzyme SET history = 'EC 1.14.11.12 created 1990' WHERE ec = 'EC 1.14.11.12';
UPDATE enzyme SET sys_name = '(Gibberellin-1),2-oxoglutarate:oxygen oxidoreductase (2-beta-hydroxylating)' WHERE ec = 'EC 1.14.11.13';
UPDATE enzyme SET history = 'EC 1.14.11.13 created 1990' WHERE ec = 'EC 1.14.11.13';
UPDATE enzyme SET sys_name = '(6S)-6-hydroxyhyoscyamine,2-oxoglutarate:oxygen oxidoreductase (epoxide-forming)' WHERE ec = 'EC 1.14.11.14';
UPDATE enzyme SET history = 'EC 1.14.11.14 created 1992' WHERE ec = 'EC 1.14.11.14';
UPDATE enzyme SET sys_name = '(Gibberellin-20),2-oxoglutarate:oxygen oxidoreductase (3-beta-hydroxylating)' WHERE ec = 'EC 1.14.11.15';
UPDATE enzyme SET history = 'EC 1.14.11.15 created 1992' WHERE ec = 'EC 1.14.11.15';
UPDATE enzyme SET sys_name = 'Peptide-L-aspartate,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.16';
UPDATE enzyme SET history = 'EC 1.14.11.16 created 1992' WHERE ec = 'EC 1.14.11.16';
UPDATE enzyme SET sys_name = 'Taurine, 2-oxoglutarate:oxygen oxidoreductase (sulfite-forming)' WHERE ec = 'EC 1.14.11.17';
UPDATE enzyme SET history = 'EC 1.14.11.17 created 2000' WHERE ec = 'EC 1.14.11.17';
UPDATE enzyme SET sys_name = 'Phytanoyl-CoA, 2-oxoglutarate:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.11.18';
UPDATE enzyme SET history = 'EC 1.14.11.18 created 2000' WHERE ec = 'EC 1.14.11.18';
UPDATE enzyme SET sys_name = 'Leucocyanidin,2-oxoglutarate:oxygen oxidoreductase' WHERE ec = 'EC 1.14.11.19';
UPDATE enzyme SET history = 'EC 1.14.11.19 created 2001' WHERE ec = 'EC 1.14.11.19';
UPDATE enzyme SET sys_name = 'Deacetoxyvindoline,2-oxoglutarate:oxygen oxidoreductase (4-beta-hydroxylating)' WHERE ec = 'EC 1.14.11.20';
UPDATE enzyme SET history = 'EC 1.14.11.20 created 2002, modified 2005' WHERE ec = 'EC 1.14.11.20';
UPDATE enzyme SET sys_name = 'Deoxyamidinoproclavaminate,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.21';
UPDATE enzyme SET history = 'EC 1.14.11.21 created 2003' WHERE ec = 'EC 1.14.11.21';
UPDATE enzyme SET sys_name = 'Flavanone,2-oxoglutarate:oxygen oxidoreductase (dehydrating)' WHERE ec = 'EC 1.14.11.22';
UPDATE enzyme SET history = 'EC 1.14.11.22 created 2004' WHERE ec = 'EC 1.14.11.22';
UPDATE enzyme SET sys_name = 'Dihydroflavonol,2-oxoglutarate:oxygen oxidoreductase' WHERE ec = 'EC 1.14.11.23';
UPDATE enzyme SET history = 'EC 1.14.11.23 created 2004' WHERE ec = 'EC 1.14.11.23';
UPDATE enzyme SET sys_name = '2-deoxymugineic acid,2-oxoglutarate:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.11.24';
UPDATE enzyme SET history = 'EC 1.14.11.24 created 2005' WHERE ec = 'EC 1.14.11.24';
UPDATE enzyme SET sys_name = 'Mugineic acid,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.25';
UPDATE enzyme SET history = 'EC 1.14.11.25 created 2005' WHERE ec = 'EC 1.14.11.25';
UPDATE enzyme SET sys_name = 'Deacetoxycephalosporin-C,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.26';
UPDATE enzyme SET history = 'EC 1.14.11.26 created 2005' WHERE ec = 'EC 1.14.11.26';
UPDATE enzyme SET sys_name = 'Protein-N(6),N(6)-dimethyl-L-lysine,2-oxoglutarate:oxygen oxidoreductase' WHERE ec = 'EC 1.14.11.27';
UPDATE enzyme SET history = 'EC 1.14.11.27 created 2006' WHERE ec = 'EC 1.14.11.27';
UPDATE enzyme SET sys_name = 'L-proline,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.28';
UPDATE enzyme SET history = 'EC 1.14.11.28 created 2006' WHERE ec = 'EC 1.14.11.28';
UPDATE enzyme SET sys_name = 'Hypoxia-inducible factor-L-proline, 2-oxoglutarate:oxygen oxidoreductase (4-hydroxylating)' WHERE ec = 'EC 1.14.11.29';
UPDATE enzyme SET history = 'EC 1.14.11.29 created 2010' WHERE ec = 'EC 1.14.11.29';
UPDATE enzyme SET sys_name = 'Hypoxia-inducible factor-L-asparagine, 2-oxoglutarate:oxygen oxidoreductase (4-hydroxylating)' WHERE ec = 'EC 1.14.11.30';
UPDATE enzyme SET history = 'EC 1.14.11.30 created 2010' WHERE ec = 'EC 1.14.11.30';
UPDATE enzyme SET sys_name = 'Thebaine,2-oxoglutarate:oxygen oxidoreductase (6-O-demethylating)' WHERE ec = 'EC 1.14.11.31';
UPDATE enzyme SET history = 'EC 1.14.11.31 created 2010' WHERE ec = 'EC 1.14.11.31';
UPDATE enzyme SET sys_name = 'Codeine,2-oxoglutarate:oxygen oxidoreductase (3-O-demethylating)' WHERE ec = 'EC 1.14.11.32';
UPDATE enzyme SET history = 'EC 1.14.11.32 created 2010' WHERE ec = 'EC 1.14.11.32';
UPDATE enzyme SET sys_name = 'Methyl DNA-base, 2-oxoglutarate:oxygen oxidoreductase (formaldehyde-forming)' WHERE ec = 'EC 1.14.11.33';
UPDATE enzyme SET history = 'EC 1.14.11.33 created 2011' WHERE ec = 'EC 1.14.11.33';
UPDATE enzyme SET sys_name = 'L-arginine,2-oxoglutarate:oxygen oxidoreductase (succinate-forming)' WHERE ec = 'EC 1.14.11.34';
UPDATE enzyme SET history = 'EC 1.14.11.34 created 2011' WHERE ec = 'EC 1.14.11.34';
UPDATE enzyme SET sys_name = '1-deoxypentalenic acid,2-oxoglutarate:oxygen oxidoreductase' WHERE ec = 'EC 1.14.11.35';
UPDATE enzyme SET history = 'EC 1.14.11.35 created 2012' WHERE ec = 'EC 1.14.11.35';
UPDATE enzyme SET sys_name = 'Pentalenolactone-D,2-oxoglutarate:oxygen oxidoreductase' WHERE ec = 'EC 1.14.11.36';
UPDATE enzyme SET history = 'EC 1.14.11.36 created 2012' WHERE ec = 'EC 1.14.11.36';
UPDATE enzyme SET sys_name = 'Kanamycin B,2-oxoglutarate:oxygen oxidoreductase (deaminating, 2-hydroxylating)' WHERE ec = 'EC 1.14.11.37';
UPDATE enzyme SET history = 'EC 1.14.11.37 created 2013, modified 2013' WHERE ec = 'EC 1.14.11.37';
UPDATE enzyme SET sys_name = 'Fumitremorgin B,2-oxoglutarate:oxygen oxidoreductase (verruculogen-forming)' WHERE ec = 'EC 1.14.11.38';
UPDATE enzyme SET history = 'EC 1.14.11.38 created 2013' WHERE ec = 'EC 1.14.11.38';
UPDATE enzyme SET sys_name = 'L-asparagine,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.39';
UPDATE enzyme SET history = 'EC 1.14.11.39 created 2013' WHERE ec = 'EC 1.14.11.39';
UPDATE enzyme SET sys_name = 'L-enduracididine,2-oxoglutarate:O(2) oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.40';
UPDATE enzyme SET history = 'EC 1.14.11.40 created 2013' WHERE ec = 'EC 1.14.11.40';
UPDATE enzyme SET sys_name = 'L-arginine,2-oxoglutarate:O(2) oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.41';
UPDATE enzyme SET history = 'EC 1.14.11.41 created 2013' WHERE ec = 'EC 1.14.11.41';
UPDATE enzyme SET sys_name = 'tRNA(Phe) 7-(3-amino-3-carboxypropyl)wyosine(37),2-oxoglutarate:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.11.42';
UPDATE enzyme SET history = 'EC 1.14.11.42 created 2013' WHERE ec = 'EC 1.14.11.42';
UPDATE enzyme SET sys_name = '(S)-2-(4-chloro-2-methylphenoxy)propanoate,2-oxoglutarate:oxygen oxidoreductase (pyruvate-forming)' WHERE ec = 'EC 1.14.11.43';
UPDATE enzyme SET history = 'EC 1.14.11.43 created 2013' WHERE ec = 'EC 1.14.11.43';
UPDATE enzyme SET sys_name = '(R)-2-(4-chloro-2-methylphenoxy)propanoate,2-oxoglutarate:oxygen oxidoreductase (pyruvate-forming)' WHERE ec = 'EC 1.14.11.44';
UPDATE enzyme SET history = 'EC 1.14.11.44 created 2013' WHERE ec = 'EC 1.14.11.44';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.11.n1';
UPDATE enzyme SET sys_name = 'DNA 5-methylcytosine,2-oxoglutarate:oxygen oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.11.n2';
UPDATE enzyme SET history = 'EC 1.14.11.n2 created 2012' WHERE ec = 'EC 1.14.11.n2';
UPDATE enzyme SET sys_name = 'L-proline,2-oxoglutarate:oxygen oxidoreductase (cis-4-hydroxylating)' WHERE ec = 'EC 1.14.11.n3';
UPDATE enzyme SET history = 'EC 1.14.11.n3 created 2012' WHERE ec = 'EC 1.14.11.n3';
UPDATE enzyme SET sys_name = 'Ankyrin-repeat-L-histidine,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.11.n4';
UPDATE enzyme SET history = 'EC 1.14.11.n4 created 2012' WHERE ec = 'EC 1.14.11.n4';
UPDATE enzyme SET sys_name = 'Anthranilate,NAD(P)H:oxygen oxidoreductase (1,2-hydroxylating, deaminating, decarboxylating)' WHERE ec = 'EC 1.14.12.1';
UPDATE enzyme SET history = 'EC 1.14.12.1 created 1972' WHERE ec = 'EC 1.14.12.1';
UPDATE enzyme SET history = 'EC 1.14.12.2 created 1972, deleted 1990' WHERE ec = 'EC 1.14.12.2';
UPDATE enzyme SET sys_name = 'Benzene,NADH:oxygen oxidoreductase (1,2-hydroxylating)' WHERE ec = 'EC 1.14.12.3';
UPDATE enzyme SET history = 'EC 1.14.12.3 created 1972' WHERE ec = 'EC 1.14.12.3';
UPDATE enzyme SET sys_name = '3-hydroxy-2-methylpyridine-5-carboxylate,NAD(P)H:oxygen oxidoreductase (decyclizing)' WHERE ec = 'EC 1.14.12.4';
UPDATE enzyme SET history = 'EC 1.14.12.4 created 1972' WHERE ec = 'EC 1.14.12.4';
UPDATE enzyme SET sys_name = '5-pyridoxate,NADPH:oxygen oxidoreductase (decyclizing)' WHERE ec = 'EC 1.14.12.5';
UPDATE enzyme SET history = 'EC 1.14.12.5 created 1972' WHERE ec = 'EC 1.14.12.5';
UPDATE enzyme SET history = 'EC 1.14.12.6 created 1978, deleted 1999' WHERE ec = 'EC 1.14.12.6';
UPDATE enzyme SET sys_name = 'Phthalate,NADH:oxygen oxidoreductase (4,5-hydroxylating)' WHERE ec = 'EC 1.14.12.7';
UPDATE enzyme SET history = 'EC 1.14.12.7 created 1990' WHERE ec = 'EC 1.14.12.7';
UPDATE enzyme SET sys_name = '4-sulfobenzoate,NADH:oxygen oxidoreductase (3,4-hydroxylating, sulfite-forming)' WHERE ec = 'EC 1.14.12.8';
UPDATE enzyme SET history = 'EC 1.14.12.8 created 1992' WHERE ec = 'EC 1.14.12.8';
UPDATE enzyme SET sys_name = '4-chlorophenylacetate,NADH:oxygen oxidoreductase (3,4-hydroxylating, dechlorinating)' WHERE ec = 'EC 1.14.12.9';
UPDATE enzyme SET history = 'EC 1.14.12.9 created 1989 as EC 1.13.99.4, transferred 1992 to EC 1.14.12.9' WHERE ec = 'EC 1.14.12.9';
UPDATE enzyme SET sys_name = 'Benzoate,NADH:oxygen oxidoreductase (1,2-hydroxylating)' WHERE ec = 'EC 1.14.12.10';
UPDATE enzyme SET history = 'EC 1.14.12.10 created 1972 as EC 1.13.99.2, transferred 1992 to EC 1.14.12.10' WHERE ec = 'EC 1.14.12.10';
UPDATE enzyme SET sys_name = 'Toluene,NADH:oxygen oxidoreductase (1,2-hydroxylating)' WHERE ec = 'EC 1.14.12.11';
UPDATE enzyme SET history = 'EC 1.14.12.11 created 1992' WHERE ec = 'EC 1.14.12.11';
UPDATE enzyme SET sys_name = 'Naphthalene,NADH:oxygen oxidoreductase (1,2-hydroxylating)' WHERE ec = 'EC 1.14.12.12';
UPDATE enzyme SET history = 'EC 1.14.12.12 created 1992' WHERE ec = 'EC 1.14.12.12';
UPDATE enzyme SET sys_name = '2-halobenzoate,NADH:oxygen oxidoreductase (1,2-hydroxylating, dehalogenating, decarboxylating)' WHERE ec = 'EC 1.14.12.13';
UPDATE enzyme SET history = 'EC 1.14.12.13 created 1992, modified 2012' WHERE ec = 'EC 1.14.12.13';
UPDATE enzyme SET sys_name = '2-aminobenzenesulfonate,NADH:oxygen oxidoreductase (2,3-hydroxylating, ammonia-forming)' WHERE ec = 'EC 1.14.12.14';
UPDATE enzyme SET history = 'EC 1.14.12.14 created 1999' WHERE ec = 'EC 1.14.12.14';
UPDATE enzyme SET sys_name = 'Benzene-1,4-dicarboxylate,NADH:oxygen oxidoreductase (1,2-hydroxylating)' WHERE ec = 'EC 1.14.12.15';
UPDATE enzyme SET history = 'EC 1.14.12.15 created 1999' WHERE ec = 'EC 1.14.12.15';
UPDATE enzyme SET sys_name = 'Quinolin-2-ol,NADH:oxygen oxidoreductase (5,6-hydroxylating)' WHERE ec = 'EC 1.14.12.16';
UPDATE enzyme SET history = 'EC 1.14.12.16 created 1999' WHERE ec = 'EC 1.14.12.16';
UPDATE enzyme SET sys_name = 'Nitric oxide,NAD(P)H:oxygen oxidoreductase' WHERE ec = 'EC 1.14.12.17';
UPDATE enzyme SET history = 'EC 1.14.12.17 created 2000' WHERE ec = 'EC 1.14.12.17';
UPDATE enzyme SET sys_name = 'Biphenyl,NADH:oxygen oxidoreductase (2,3-hydroxylating)' WHERE ec = 'EC 1.14.12.18';
UPDATE enzyme SET history = 'EC 1.14.12.18 created 2001' WHERE ec = 'EC 1.14.12.18';
UPDATE enzyme SET sys_name = '3-phenylpropanoate,NADH:oxygen oxidoreductase (2,3-hydroxylating)' WHERE ec = 'EC 1.14.12.19';
UPDATE enzyme SET history = 'EC 1.14.12.19 created 2005, modified 2011' WHERE ec = 'EC 1.14.12.19';
UPDATE enzyme SET sys_name = 'Pheophorbide-a,NADPH:oxygen oxidoreductase (biladiene-forming)' WHERE ec = 'EC 1.14.12.20';
UPDATE enzyme SET history = 'EC 1.14.12.20 created 2007' WHERE ec = 'EC 1.14.12.20';
UPDATE enzyme SET sys_name = 'Benzoyl-CoA,NADPH:oxygen oxidoreductase (2,3-hydroxylating)' WHERE ec = 'EC 1.14.12.21';
UPDATE enzyme SET history = 'EC 1.14.12.21 created 2010' WHERE ec = 'EC 1.14.12.21';
UPDATE enzyme SET sys_name = '9H-carbazole,NAD(P)H:oxygen oxidoreductase (2,3-hydroxylating)' WHERE ec = 'EC 1.14.12.22';
UPDATE enzyme SET history = 'EC 1.14.12.22 created 2010' WHERE ec = 'EC 1.14.12.22';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.12.n1';
UPDATE enzyme SET sys_name = 'Salicylate,NADH:oxygen oxidoreductase (1-hydroxylating, decarboxylating)' WHERE ec = 'EC 1.14.13.1';
UPDATE enzyme SET history = 'EC 1.14.13.1 created 1972' WHERE ec = 'EC 1.14.13.1';
UPDATE enzyme SET sys_name = '4-hydroxybenzoate,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.2';
UPDATE enzyme SET history = 'EC 1.14.13.2 created 1972, modified 1999' WHERE ec = 'EC 1.14.13.2';
UPDATE enzyme SET history = 'EC 1.14.13.3 created 1972, deleted 2011' WHERE ec = 'EC 1.14.13.3';
UPDATE enzyme SET sys_name = '3-(2-hydroxyphenyl)propanoate,NADH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.4';
UPDATE enzyme SET history = 'EC 1.14.13.4 created 1972' WHERE ec = 'EC 1.14.13.4';
UPDATE enzyme SET sys_name = '4-imidazoleacetate,NADH:oxygen oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.13.5';
UPDATE enzyme SET history = 'EC 1.14.13.5 created 1965 as EC 1.14.1.5, transferred 1972 to EC 1.14.13.5' WHERE ec = 'EC 1.14.13.5';
UPDATE enzyme SET sys_name = 'Orcinol,NADH:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.6';
UPDATE enzyme SET history = 'EC 1.14.13.6 created 1972' WHERE ec = 'EC 1.14.13.6';
UPDATE enzyme SET sys_name = 'Phenol,NADPH:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.7';
UPDATE enzyme SET history = 'EC 1.14.13.7 created 1972, modified 2011' WHERE ec = 'EC 1.14.13.7';
UPDATE enzyme SET sys_name = 'N,N-dimethylaniline,NADPH:oxygen oxidoreductase (N-oxide-forming)' WHERE ec = 'EC 1.14.13.8';
UPDATE enzyme SET history = 'EC 1.14.13.8 created 1972 (EC 1.13.12.11 created 1992, part-incorporated 2006), modified 2006' WHERE ec = 'EC 1.14.13.8';
UPDATE enzyme SET sys_name = 'L-kynurenine,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.9';
UPDATE enzyme SET history = 'EC 1.14.13.9 created 1961 as EC 1.99.1.5, transferred 1965 to EC 1.14.1.2, transferred 1972 to EC 1.14.13.9' WHERE ec = 'EC 1.14.13.9';
UPDATE enzyme SET sys_name = '2,6-dihydroxypyridine,NADH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.10';
UPDATE enzyme SET history = 'EC 1.14.13.10 created 1976' WHERE ec = 'EC 1.14.13.10';
UPDATE enzyme SET sys_name = 'Trans-cinnamate,NADPH:oxygen oxidoreductase (4-hydroxylating)' WHERE ec = 'EC 1.14.13.11';
UPDATE enzyme SET history = 'EC 1.14.13.11 created 1976' WHERE ec = 'EC 1.14.13.11';
UPDATE enzyme SET sys_name = 'Benzoate,NADPH:oxygen oxidoreductase (4-hydroxylating)' WHERE ec = 'EC 1.14.13.12';
UPDATE enzyme SET history = 'EC 1.14.13.12 created 1976' WHERE ec = 'EC 1.14.13.12';
UPDATE enzyme SET sys_name = 'Calcidiol,NADPH:oxygen oxidoreductase (1-hydroxylating)' WHERE ec = 'EC 1.14.13.13';
UPDATE enzyme SET history = 'EC 1.14.13.13 created 1976' WHERE ec = 'EC 1.14.13.13';
UPDATE enzyme SET sys_name = 'Trans-cinnamate,NADPH:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.14';
UPDATE enzyme SET history = 'EC 1.14.13.14 created 1976' WHERE ec = 'EC 1.14.13.14';
UPDATE enzyme SET sys_name = '5-beta-cholestane-3-alpha,7-alpha,12-alpha-triol,NADPH:oxygen oxidoreductase (26-hydroxylating)' WHERE ec = 'EC 1.14.13.15';
UPDATE enzyme SET history = 'EC 1.14.13.15 created 1976, modified 2005, modified 2012' WHERE ec = 'EC 1.14.13.15';
UPDATE enzyme SET sys_name = 'Cyclopentanone,NADPH:oxygen oxidoreductase (5-hydroxylating, lactonizing)' WHERE ec = 'EC 1.14.13.16';
UPDATE enzyme SET history = 'EC 1.14.13.16 created 1976' WHERE ec = 'EC 1.14.13.16';
UPDATE enzyme SET sys_name = 'Cholesterol,NADPH:oxygen oxidoreductase (7-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.17';
UPDATE enzyme SET history = 'EC 1.14.13.17 created 1976' WHERE ec = 'EC 1.14.13.17';
UPDATE enzyme SET sys_name = '4-hydroxyphenylacetate,NAD(P)H:oxygen oxidoreductase (1-hydroxylating)' WHERE ec = 'EC 1.14.13.18';
UPDATE enzyme SET history = 'EC 1.14.13.18 created 1976' WHERE ec = 'EC 1.14.13.18';
UPDATE enzyme SET sys_name = 'Taxifolin,NAD(P)H:oxygen oxidoreductase (8-hydroxylating)' WHERE ec = 'EC 1.14.13.19';
UPDATE enzyme SET history = 'EC 1.14.13.19 created 1976' WHERE ec = 'EC 1.14.13.19';
UPDATE enzyme SET sys_name = '2,4-dichlorophenol,NADPH:oxygen oxidoreductase (6-hydroxylating)' WHERE ec = 'EC 1.14.13.20';
UPDATE enzyme SET history = 'EC 1.14.13.20 created 1983' WHERE ec = 'EC 1.14.13.20';
UPDATE enzyme SET sys_name = 'Flavonoid,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.21';
UPDATE enzyme SET history = 'EC 1.14.13.21 created 1983' WHERE ec = 'EC 1.14.13.21';
UPDATE enzyme SET sys_name = 'Cyclohexanone:NADPH:oxygen oxidoreductase (lactone-forming)' WHERE ec = 'EC 1.14.13.22';
UPDATE enzyme SET history = 'EC 1.14.13.22 created 1984, modified 2004' WHERE ec = 'EC 1.14.13.22';
UPDATE enzyme SET sys_name = '3-hydroxybenzoate,NADPH:oxygen oxidoreductase (4-hydroxylating)' WHERE ec = 'EC 1.14.13.23';
UPDATE enzyme SET history = 'EC 1.14.13.23 created 1972 as EC 1.14.99.13, transferred 1984 to EC 1.14.13.23' WHERE ec = 'EC 1.14.13.23';
UPDATE enzyme SET sys_name = '3-hydroxybenzoate,NADH:oxygen oxidoreductase (6-hydroxylating)' WHERE ec = 'EC 1.14.13.24';
UPDATE enzyme SET history = 'EC 1.14.13.24 created 1984' WHERE ec = 'EC 1.14.13.24';
UPDATE enzyme SET sys_name = 'Methane,NAD(P)H:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.13.25';
UPDATE enzyme SET history = 'EC 1.14.13.25 created 1984, modified 2011' WHERE ec = 'EC 1.14.13.25';
UPDATE enzyme SET sys_name = '1-acyl-2-oleoyl-sn-glycero-3-phosphocholine,NADH:oxygen oxidoreductase (12-hydroxylating)' WHERE ec = 'EC 1.14.13.26';
UPDATE enzyme SET history = 'EC 1.14.13.26 created 1984' WHERE ec = 'EC 1.14.13.26';
UPDATE enzyme SET sys_name = '4-aminobenzoate,NAD(P)H:oxygen oxidoreductase (1-hydroxylating, decarboxylating)' WHERE ec = 'EC 1.14.13.27';
UPDATE enzyme SET history = 'EC 1.14.13.27 created 1989' WHERE ec = 'EC 1.14.13.27';
UPDATE enzyme SET sys_name = '(6aR,11aR)-3,9-dihydroxypterocarpan,NADPH:oxygen oxidoreductase (6a-hydroxylating)' WHERE ec = 'EC 1.14.13.28';
UPDATE enzyme SET history = 'EC 1.14.13.28 created 1989' WHERE ec = 'EC 1.14.13.28';
UPDATE enzyme SET sys_name = '4-nitrophenol,NADH:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.29';
UPDATE enzyme SET history = 'EC 1.14.13.29 created 1989' WHERE ec = 'EC 1.14.13.29';
UPDATE enzyme SET sys_name = '(6Z,8E,10E,14Z)-(5S,12R)-5,12-dihydroxyicosa-6,8,10,14-tetraenoate,NADPH:oxygen oxidoreductase (20-hydroxylating)' WHERE ec = 'EC 1.14.13.30';
UPDATE enzyme SET history = 'EC 1.14.13.30 created 1989' WHERE ec = 'EC 1.14.13.30';
UPDATE enzyme SET sys_name = '2-nitrophenol,NADPH:oxygen 2-oxidoreductase (2-hydroxylating, nitrite-forming)' WHERE ec = 'EC 1.14.13.31';
UPDATE enzyme SET history = 'EC 1.14.13.31 created 1989' WHERE ec = 'EC 1.14.13.31';
UPDATE enzyme SET sys_name = 'Albendazole,NADPH:oxygen oxidoreductase (sulfoxide-forming)' WHERE ec = 'EC 1.14.13.32';
UPDATE enzyme SET history = 'EC 1.14.13.32 created 1989' WHERE ec = 'EC 1.14.13.32';
UPDATE enzyme SET sys_name = '4-hydroxybenzoate,NAD(P)H:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.33';
UPDATE enzyme SET history = 'EC 1.14.13.33 created 1989, modified 1999' WHERE ec = 'EC 1.14.13.33';
UPDATE enzyme SET sys_name = '(7E,9E,11Z,14Z)-(5S,6R)-6-(cystein-S-yl)-5-hydroxyicosa-7,9,11,14-tetraenoate,NADPH:oxygen oxidoreductase (20-hydroxylating)' WHERE ec = 'EC 1.14.13.34';
UPDATE enzyme SET history = 'EC 1.14.13.34 created 1989' WHERE ec = 'EC 1.14.13.34';
UPDATE enzyme SET sys_name = 'Anthranilate,NADPH:oxygen oxidoreductase (3-hydroxylating, deaminating)' WHERE ec = 'EC 1.14.13.35';
UPDATE enzyme SET history = 'EC 1.14.13.35 created 1972 as EC 1.14.12.2, transferred 1990 to EC 1.14.13.35' WHERE ec = 'EC 1.14.13.35';
UPDATE enzyme SET sys_name = 'Trans-5-O-(4-coumaroyl)-D-quinate,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.36';
UPDATE enzyme SET history = 'EC 1.14.13.36 created 1990' WHERE ec = 'EC 1.14.13.36';
UPDATE enzyme SET sys_name = '(S)-N-methylcanadine,NADPH:oxygen oxidoreductase (14-hydroxylating)' WHERE ec = 'EC 1.14.13.37';
UPDATE enzyme SET history = 'EC 1.14.13.37 created 1990' WHERE ec = 'EC 1.14.13.37';
UPDATE enzyme SET sys_name = 'Anhydrotetracycline,NADPH:oxygen oxidoreductase (6-hydroxylating)' WHERE ec = 'EC 1.14.13.38';
UPDATE enzyme SET history = 'EC 1.14.13.38 created 1990' WHERE ec = 'EC 1.14.13.38';
UPDATE enzyme SET sys_name = 'L-arginine,NADPH:oxygen oxidoreductase (nitric-oxide-forming)' WHERE ec = 'EC 1.14.13.39';
UPDATE enzyme SET history = 'EC 1.14.13.39 created 1992, modified 2012' WHERE ec = 'EC 1.14.13.39';
UPDATE enzyme SET sys_name = '2-aminobenzoyl-CoA,NAD(P)H:oxygen oxidoreductase (de-aromatizing)' WHERE ec = 'EC 1.14.13.40';
UPDATE enzyme SET history = 'EC 1.14.13.40 created 1992' WHERE ec = 'EC 1.14.13.40';
UPDATE enzyme SET sys_name = 'L-tyrosine,NADPH:oxygen oxidoreductase (N-hydroxylating)' WHERE ec = 'EC 1.14.13.41';
UPDATE enzyme SET history = 'EC 1.14.13.41 created 1992, modified 2001, modified 2005' WHERE ec = 'EC 1.14.13.41';
UPDATE enzyme SET history = 'EC 1.14.13.42 created 1992, deleted 2011' WHERE ec = 'EC 1.14.13.42';
UPDATE enzyme SET sys_name = 'Questin,NADPH:oxygen oxidoreductase (hydroxylating, anthraquinone-ring-opening)' WHERE ec = 'EC 1.14.13.43';
UPDATE enzyme SET history = 'EC 1.14.13.43 created 1992' WHERE ec = 'EC 1.14.13.43';
UPDATE enzyme SET sys_name = '2-hydroxybiphenyl,NADH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.44';
UPDATE enzyme SET history = 'EC 1.14.13.44 created 1992' WHERE ec = 'EC 1.14.13.44';
UPDATE enzyme SET history = 'EC 1.14.13.45 created 1992, deleted 2003' WHERE ec = 'EC 1.14.13.45';
UPDATE enzyme SET sys_name = '(-)-menthol,NADPH:oxygen oxidoreductase (8-hydroxylating)' WHERE ec = 'EC 1.14.13.46';
UPDATE enzyme SET history = 'EC 1.14.13.46 created 1992' WHERE ec = 'EC 1.14.13.46';
UPDATE enzyme SET sys_name = '(S)-limonene,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.47';
UPDATE enzyme SET history = 'EC 1.14.13.47 created 1992, modified 2003' WHERE ec = 'EC 1.14.13.47';
UPDATE enzyme SET sys_name = '(S)-limonene,NADPH:oxygen oxidoreductase (6-hydroxylating)' WHERE ec = 'EC 1.14.13.48';
UPDATE enzyme SET history = 'EC 1.14.13.48 created 1992, modified 2003' WHERE ec = 'EC 1.14.13.48';
UPDATE enzyme SET sys_name = '(S)-limonene,NADPH:oxygen oxidoreductase (7-hydroxylating)' WHERE ec = 'EC 1.14.13.49';
UPDATE enzyme SET history = 'EC 1.14.13.49 created 1992, modified 2003' WHERE ec = 'EC 1.14.13.49';
UPDATE enzyme SET sys_name = 'Pentachlorophenol,NADPH:oxygen oxidoreductase (hydroxylating, dechlorinating)' WHERE ec = 'EC 1.14.13.50';
UPDATE enzyme SET history = 'EC 1.14.13.50 created 1992, modified 2005' WHERE ec = 'EC 1.14.13.50';
UPDATE enzyme SET sys_name = '6-oxocineole,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.51';
UPDATE enzyme SET history = 'EC 1.14.13.51 created 1992' WHERE ec = 'EC 1.14.13.51';
UPDATE enzyme SET sys_name = 'Formononetin,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.52';
UPDATE enzyme SET history = 'EC 1.14.13.52 created 1992' WHERE ec = 'EC 1.14.13.52';
UPDATE enzyme SET sys_name = 'Formononetin,NADPH:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.53';
UPDATE enzyme SET history = 'EC 1.14.13.53 created 1992, modified 2005' WHERE ec = 'EC 1.14.13.53';
UPDATE enzyme SET sys_name = 'Ketosteroid,NADPH:oxygen oxidoreductase (20-hydroxylating, ester-producing/20-hydroxylating, side-chain cleaving/17-hydroxylating, lactonizing)' WHERE ec = 'EC 1.14.13.54';
UPDATE enzyme SET history = 'EC 1.14.13.54 created 1999' WHERE ec = 'EC 1.14.13.54';
UPDATE enzyme SET sys_name = 'Protopine,NADPH:oxygen oxidoreductase (6-hydroxylating)' WHERE ec = 'EC 1.14.13.55';
UPDATE enzyme SET history = 'EC 1.14.13.55 created 1999' WHERE ec = 'EC 1.14.13.55';
UPDATE enzyme SET sys_name = 'Dihydrosanguinarine,NADPH:oxygen oxidoreductase (10-hydroxylating)' WHERE ec = 'EC 1.14.13.56';
UPDATE enzyme SET history = 'EC 1.14.13.56 created 1999' WHERE ec = 'EC 1.14.13.56';
UPDATE enzyme SET sys_name = 'Dihydrochelirubine,NADPH:oxygen oxidoreductase (12-hydroxylating)' WHERE ec = 'EC 1.14.13.57';
UPDATE enzyme SET history = 'EC 1.14.13.57 created 1999' WHERE ec = 'EC 1.14.13.57';
UPDATE enzyme SET sys_name = 'Benzoyl-CoA,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.58';
UPDATE enzyme SET history = 'EC 1.14.13.58 created 1999' WHERE ec = 'EC 1.14.13.58';
UPDATE enzyme SET sys_name = 'L-lysine,NADPH:oxygen oxidoreductase (6-hydroxylating)' WHERE ec = 'EC 1.14.13.59';
UPDATE enzyme SET history = 'EC 1.14.13.59 created 1999, modified 2001, modified 2012' WHERE ec = 'EC 1.14.13.59';
UPDATE enzyme SET history = 'EC 1.14.13.60 created 1999, deleted 2013' WHERE ec = 'EC 1.14.13.60';
UPDATE enzyme SET sys_name = 'Quinolin-2(1H)-one,NADH:oxygen oxidoreductase (8-oxygenating)' WHERE ec = 'EC 1.14.13.61';
UPDATE enzyme SET history = 'EC 1.14.13.61 created 1999' WHERE ec = 'EC 1.14.13.61';
UPDATE enzyme SET sys_name = 'Quinolin-4(1H)-one,NADH:oxygen oxidoreductase (3-oxygenating)' WHERE ec = 'EC 1.14.13.62';
UPDATE enzyme SET history = 'EC 1.14.13.62 created 1999' WHERE ec = 'EC 1.14.13.62';
UPDATE enzyme SET sys_name = '3-hydroxyphenylacetate,NAD(P)H:oxygen oxidoreductase (6-hydroxylating)' WHERE ec = 'EC 1.14.13.63';
UPDATE enzyme SET history = 'EC 1.14.13.63 created 1999' WHERE ec = 'EC 1.14.13.63';
UPDATE enzyme SET sys_name = '4-hydroxybenzoate,NAD(P)H:oxygen oxidoreductase (1-hydroxylating, decarboxylating)' WHERE ec = 'EC 1.14.13.64';
UPDATE enzyme SET history = 'EC 1.14.13.64 created 1999' WHERE ec = 'EC 1.14.13.64';
UPDATE enzyme SET history = 'EC 1.14.13.65 created 1999, deleted 2006' WHERE ec = 'EC 1.14.13.65';
UPDATE enzyme SET sys_name = '2-hydroxycyclohexan-1-one,NADPH:oxygen 2-oxidoreductase (1,2-lactonizing)' WHERE ec = 'EC 1.14.13.66';
UPDATE enzyme SET history = 'EC 1.14.13.66 created 1978 as EC 1.14.12.6, transferred 1999 to EC 1.14.13.66' WHERE ec = 'EC 1.14.13.66';
UPDATE enzyme SET sys_name = 'Quinine,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.67';
UPDATE enzyme SET history = 'EC 1.14.13.67 created 2000' WHERE ec = 'EC 1.14.13.67';
UPDATE enzyme SET sys_name = '(Z)-4-hydroxyphenylacetaldehyde oxime,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.68';
UPDATE enzyme SET history = 'EC 1.14.13.68 created 2000, modified 2005' WHERE ec = 'EC 1.14.13.68';
UPDATE enzyme SET sys_name = 'Alkene,NADH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.69';
UPDATE enzyme SET history = 'EC 1.14.13.69 created 2001' WHERE ec = 'EC 1.14.13.69';
UPDATE enzyme SET sys_name = 'Sterol,NADPH:oxygen oxidoreductase (14-methyl cleaving)' WHERE ec = 'EC 1.14.13.70';
UPDATE enzyme SET history = 'EC 1.14.13.70 created 2001, modified 2013' WHERE ec = 'EC 1.14.13.70';
UPDATE enzyme SET sys_name = '(S)-N-methylcoclaurine,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.71';
UPDATE enzyme SET history = 'EC 1.14.13.71 created 2001' WHERE ec = 'EC 1.14.13.71';
UPDATE enzyme SET sys_name = '4,4-dimethyl-5-alpha-cholest-7-en-3-beta-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.13.72';
UPDATE enzyme SET history = 'EC 1.14.13.72 created 1972 as EC 1.14.99.16, transferred 2002 to EC 1.14.13.72' WHERE ec = 'EC 1.14.13.72';
UPDATE enzyme SET sys_name = 'Tabersonine,NADPH:oxygen oxidoreductase (16-hydroxylating)' WHERE ec = 'EC 1.14.13.73';
UPDATE enzyme SET history = 'EC 1.14.13.73 created 2002' WHERE ec = 'EC 1.14.13.73';
UPDATE enzyme SET sys_name = '7-deoxyloganin,NADPH:oxygen oxidoreductase (7-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.74';
UPDATE enzyme SET history = 'EC 1.14.13.74 created 2002' WHERE ec = 'EC 1.14.13.74';
UPDATE enzyme SET sys_name = 'Vinorine,NADPH:oxygen oxidoreductase (21-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.75';
UPDATE enzyme SET history = 'EC 1.14.13.75 created 2002' WHERE ec = 'EC 1.14.13.75';
UPDATE enzyme SET sys_name = 'Taxa-4(20),11-dien-5-alpha-yl acetate,NADPH:oxygen oxidoreductase (10-beta-hydroxylating)' WHERE ec = 'EC 1.14.13.76';
UPDATE enzyme SET history = 'EC 1.14.13.76 created 2002' WHERE ec = 'EC 1.14.13.76';
UPDATE enzyme SET sys_name = 'Taxa-4(20),11-dien-5-alpha-ol,NADPH:oxygen oxidoreductase (13-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.77';
UPDATE enzyme SET history = 'EC 1.14.13.77 created 2002' WHERE ec = 'EC 1.14.13.77';
UPDATE enzyme SET sys_name = 'Ent-kaur-16-ene,NADPH:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.13.78';
UPDATE enzyme SET history = 'EC 1.14.13.78 created 2002' WHERE ec = 'EC 1.14.13.78';
UPDATE enzyme SET sys_name = 'Ent-kaur-16-en-19-oate,NADPH:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.13.79';
UPDATE enzyme SET history = 'EC 1.14.13.79 created 2002' WHERE ec = 'EC 1.14.13.79';
UPDATE enzyme SET sys_name = '(R)-limonene,NADPH:oxygen oxidoreductase (6-hydroxylating)' WHERE ec = 'EC 1.14.13.80';
UPDATE enzyme SET history = 'EC 1.14.13.80 created 2003' WHERE ec = 'EC 1.14.13.80';
UPDATE enzyme SET sys_name = 'Magnesium-protoporphyrin-IX 13-monomethyl ester,NADPH:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.13.81';
UPDATE enzyme SET history = 'EC 1.14.13.81 created 2003' WHERE ec = 'EC 1.14.13.81';
UPDATE enzyme SET sys_name = 'Vanillate:oxygen oxidoreductase (demethylating)' WHERE ec = 'EC 1.14.13.82';
UPDATE enzyme SET history = 'EC 1.14.13.82 created 2000 as EC 1.2.3.12, transferred 2003 to EC 1.14.13.82' WHERE ec = 'EC 1.14.13.82';
UPDATE enzyme SET sys_name = 'Precorrin-3A,NADH:oxygen oxidoreductase (20-hydroxylating)' WHERE ec = 'EC 1.14.13.83';
UPDATE enzyme SET history = 'EC 1.14.13.83 created 2004' WHERE ec = 'EC 1.14.13.83';
UPDATE enzyme SET sys_name = '(4-hydroxyphenyl)ethan-1-one,NADPH:oxygen oxidoreductase (ester-forming)' WHERE ec = 'EC 1.14.13.84';
UPDATE enzyme SET history = 'EC 1.14.13.84 created 2004' WHERE ec = 'EC 1.14.13.84';
UPDATE enzyme SET sys_name = '2-(or 4-)dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan,NADPH:oxygen oxidoreductase (cyclizing)' WHERE ec = 'EC 1.14.13.85';
UPDATE enzyme SET history = 'EC 1.14.13.85 created 2004' WHERE ec = 'EC 1.14.13.85';
UPDATE enzyme SET history = 'EC 1.14.13.86 created 2004, deleted 2013' WHERE ec = 'EC 1.14.13.86';
UPDATE enzyme SET sys_name = 'Liquiritigenin,NADPH:oxygen oxidoreductase (licodione-forming)' WHERE ec = 'EC 1.14.13.87';
UPDATE enzyme SET history = 'EC 1.14.13.87 created 2004' WHERE ec = 'EC 1.14.13.87';
UPDATE enzyme SET sys_name = 'Flavanone,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.88';
UPDATE enzyme SET history = 'EC 1.14.13.88 created 2004' WHERE ec = 'EC 1.14.13.88';
UPDATE enzyme SET sys_name = 'Isoflavone,NADPH:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.89';
UPDATE enzyme SET history = 'EC 1.14.13.89 created 2005' WHERE ec = 'EC 1.14.13.89';
UPDATE enzyme SET sys_name = 'Zeaxanthin,NAD(P)H:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.90';
UPDATE enzyme SET history = 'EC 1.14.13.90 created 2005' WHERE ec = 'EC 1.14.13.90';
UPDATE enzyme SET sys_name = '10-deoxysarpagine,NADPH:oxygen oxidoreductase (10-hydroxylating)' WHERE ec = 'EC 1.14.13.91';
UPDATE enzyme SET history = 'EC 1.14.13.91 created 2005' WHERE ec = 'EC 1.14.13.91';
UPDATE enzyme SET sys_name = 'Phenylacetone,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.92';
UPDATE enzyme SET history = 'EC 1.14.13.92 created 2005' WHERE ec = 'EC 1.14.13.92';
UPDATE enzyme SET sys_name = 'Abscisate,NADPH:oxygen oxidoreductase (8-hydroxylating)' WHERE ec = 'EC 1.14.13.93';
UPDATE enzyme SET history = 'EC 1.14.13.93 created 2005' WHERE ec = 'EC 1.14.13.93';
UPDATE enzyme SET sys_name = 'Lithocholate,NADPH:oxygen oxidoreductase (6-beta-hydroxylating)' WHERE ec = 'EC 1.14.13.94';
UPDATE enzyme SET history = 'EC 1.14.13.94 created 2005' WHERE ec = 'EC 1.14.13.94';
UPDATE enzyme SET sys_name = '7-alpha-hydroxycholest-4-en-3-one,NADPH:oxygen oxidoreductase (12-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.95';
UPDATE enzyme SET history = 'EC 1.14.13.95 created 2005' WHERE ec = 'EC 1.14.13.95';
UPDATE enzyme SET sys_name = '5-beta-cholestane-3-alpha,7-alpha-diol,NADPH:oxygen oxidoreductase (12-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.96';
UPDATE enzyme SET history = 'EC 1.14.13.96 created 2005' WHERE ec = 'EC 1.14.13.96';
UPDATE enzyme SET sys_name = 'Taurochenodeoxycholate,NADPH:oxygen oxidoreductase (6-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.97';
UPDATE enzyme SET history = 'EC 1.14.13.97 created 2005' WHERE ec = 'EC 1.14.13.97';
UPDATE enzyme SET sys_name = 'Cholesterol,NADPH:oxygen oxidoreductase (24-hydroxylating)' WHERE ec = 'EC 1.14.13.98';
UPDATE enzyme SET history = 'EC 1.14.13.98 created 2005' WHERE ec = 'EC 1.14.13.98';
UPDATE enzyme SET sys_name = '(24R)-cholest-5-ene-3-beta,24-diol,NADPH:oxygen oxidoreductase (7-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.99';
UPDATE enzyme SET history = 'EC 1.14.13.99 created 2005' WHERE ec = 'EC 1.14.13.99';
UPDATE enzyme SET sys_name = 'Cholest-5-ene-3-beta,25/26-diol,NADPH:oxygen oxidoreductase (7-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.100';
UPDATE enzyme SET history = 'EC 1.14.13.100 created 2005, modified 2013 (EC 1.14.13.60 created 1999, incorporated 2013)' WHERE ec = 'EC 1.14.13.100';
UPDATE enzyme SET sys_name = 'Senecionine,NADPH:oxygen oxidoreductase (N-oxide-forming)' WHERE ec = 'EC 1.14.13.101';
UPDATE enzyme SET history = 'EC 1.14.13.101 created 2006' WHERE ec = 'EC 1.14.13.101';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 1.14.13.102';
UPDATE enzyme SET history = 'EC 1.14.13.102 created 2007' WHERE ec = 'EC 1.14.13.102';
UPDATE enzyme SET sys_name = 'Sophoraflavanone-B,NADPH:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.103';
UPDATE enzyme SET history = 'EC 1.14.13.103 created 2007' WHERE ec = 'EC 1.14.13.103';
UPDATE enzyme SET sys_name = '(+)-pulegone,NADPH:oxygen oxidoreductase (9-hydroxylating)' WHERE ec = 'EC 1.14.13.104';
UPDATE enzyme SET history = 'EC 1.14.13.104 created 2008' WHERE ec = 'EC 1.14.13.104';
UPDATE enzyme SET sys_name = '(-)-menthone,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.105';
UPDATE enzyme SET history = 'EC 1.14.13.105 created 2008' WHERE ec = 'EC 1.14.13.105';
UPDATE enzyme SET sys_name = '(+)-epi-isozizaene,NADPH:oxygen oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.13.106';
UPDATE enzyme SET history = 'EC 1.14.13.106 created 2008' WHERE ec = 'EC 1.14.13.106';
UPDATE enzyme SET sys_name = 'Limonene,NAD(P)H:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.107';
UPDATE enzyme SET history = 'EC 1.14.13.107 created 2009' WHERE ec = 'EC 1.14.13.107';
UPDATE enzyme SET sys_name = 'Abieta-7,13-diene,NADPH:oxygen oxidoreductase (18-hydroxylating)' WHERE ec = 'EC 1.14.13.108';
UPDATE enzyme SET history = 'EC 1.14.13.108 created 2009, modified 2012' WHERE ec = 'EC 1.14.13.108';
UPDATE enzyme SET sys_name = 'Abieta-7,13-dien-18-ol,NADPH:oxygen oxidoreductase (18-hydroxylating)' WHERE ec = 'EC 1.14.13.109';
UPDATE enzyme SET history = 'EC 1.14.13.109 created 2009, modified 2012' WHERE ec = 'EC 1.14.13.109';
UPDATE enzyme SET sys_name = 'Geranylgeraniol,NADPH:oxygen oxidoreductase (18-hydroxylating)' WHERE ec = 'EC 1.14.13.110';
UPDATE enzyme SET history = 'EC 1.14.13.110 created 2009' WHERE ec = 'EC 1.14.13.110';
UPDATE enzyme SET sys_name = 'Methanesulfonate,NADH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.111';
UPDATE enzyme SET history = 'EC 1.14.13.111 created 2009 as EC 1.14.14.6, transferred 2010 to EC 1.14.13.111' WHERE ec = 'EC 1.14.13.111';
UPDATE enzyme SET sys_name = '3-epi-6-deoxocathasterone,NADPH:oxygen oxidoreductase (C-23-hydroxylating)' WHERE ec = 'EC 1.14.13.112';
UPDATE enzyme SET history = 'EC 1.14.13.112 created 2010' WHERE ec = 'EC 1.14.13.112';
UPDATE enzyme SET sys_name = 'Urate,NADH:oxygen oxidoreductase (5-hydroxyisourate forming)' WHERE ec = 'EC 1.14.13.113';
UPDATE enzyme SET history = 'EC 1.14.13.113 created 2010' WHERE ec = 'EC 1.14.13.113';
UPDATE enzyme SET sys_name = '6-hydroxynicotinate,NADH:oxygen oxidoreductase (3-hydroxylating, decarboxylating)' WHERE ec = 'EC 1.14.13.114';
UPDATE enzyme SET history = 'EC 1.14.13.114 created 2010' WHERE ec = 'EC 1.14.13.114';
UPDATE enzyme SET sys_name = '(+)-columbianetin,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.115';
UPDATE enzyme SET history = 'EC 1.14.13.115 created 2010' WHERE ec = 'EC 1.14.13.115';
UPDATE enzyme SET sys_name = 'Geranylhydroquinone,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.116';
UPDATE enzyme SET history = 'EC 1.14.13.116 created 2010' WHERE ec = 'EC 1.14.13.116';
UPDATE enzyme SET sys_name = 'L-isoleucine,NADPH:oxygen oxidoreductase (N-hydroxylating)' WHERE ec = 'EC 1.14.13.117';
UPDATE enzyme SET history = 'EC 1.14.13.117 created 2010' WHERE ec = 'EC 1.14.13.117';
UPDATE enzyme SET sys_name = 'L-valine,NADPH:oxygen oxidoreductase (N-hydroxylating)' WHERE ec = 'EC 1.14.13.118';
UPDATE enzyme SET history = 'EC 1.14.13.118 created 2010' WHERE ec = 'EC 1.14.13.118';
UPDATE enzyme SET sys_name = '5-epiaristolochene,NADPH:oxygen oxidoreductase (1- and 3-hydroxylating)' WHERE ec = 'EC 1.14.13.119';
UPDATE enzyme SET history = 'EC 1.14.13.119 created 2011' WHERE ec = 'EC 1.14.13.119';
UPDATE enzyme SET sys_name = 'Germacra-1(10),4,11(13)-trien-12-oate,NADPH:oxygen oxidoreductase (6-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.120';
UPDATE enzyme SET history = 'EC 1.14.13.120 created 2011' WHERE ec = 'EC 1.14.13.120';
UPDATE enzyme SET sys_name = '(-)-vetispiradiene,NADPH:oxygen 2-alpha-oxidoreductase' WHERE ec = 'EC 1.14.13.121';
UPDATE enzyme SET history = 'EC 1.14.13.121 created 2011' WHERE ec = 'EC 1.14.13.121';
UPDATE enzyme SET sys_name = 'Chlorophyllide-a:oxygen 7-oxidoreductase' WHERE ec = 'EC 1.14.13.122';
UPDATE enzyme SET history = 'EC 1.14.13.122 created 2006 as EC 1.13.12.14, transferred 2011 to EC 1.14.13.122' WHERE ec = 'EC 1.14.13.122';
UPDATE enzyme SET sys_name = '(+)-germacrene-A,NADPH:oxygen oxidoreductase (12-hydroxylating)' WHERE ec = 'EC 1.14.13.123';
UPDATE enzyme SET history = 'EC 1.14.13.123 created 2011' WHERE ec = 'EC 1.14.13.123';
UPDATE enzyme SET sys_name = 'L-phenylalanine,NADPH:oxygen oxidoreductase (N-hydroxylating)' WHERE ec = 'EC 1.14.13.124';
UPDATE enzyme SET history = 'EC 1.14.13.124 created 2011' WHERE ec = 'EC 1.14.13.124';
UPDATE enzyme SET sys_name = 'L-tryptophan,NADPH:oxygen oxidoreductase (N-hydroxylating)' WHERE ec = 'EC 1.14.13.125';
UPDATE enzyme SET history = 'EC 1.14.13.125 created 2011' WHERE ec = 'EC 1.14.13.125';
UPDATE enzyme SET sys_name = 'Calcitriol,NADPH:oxygen oxidoreductase (24-hydroxylating)' WHERE ec = 'EC 1.14.13.126';
UPDATE enzyme SET history = 'EC 1.14.13.126 created 2011' WHERE ec = 'EC 1.14.13.126';
UPDATE enzyme SET sys_name = '3-(3-hydroxyphenyl)propanoic acid,NADH:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.127';
UPDATE enzyme SET history = 'EC 1.14.13.127 created 2011' WHERE ec = 'EC 1.14.13.127';
UPDATE enzyme SET sys_name = '7-methylxanthine:oxygen oxidoreductase (demethylating)' WHERE ec = 'EC 1.14.13.128';
UPDATE enzyme SET history = 'EC 1.14.13.128 created 2011' WHERE ec = 'EC 1.14.13.128';
UPDATE enzyme SET sys_name = 'Beta-carotene,NADH:oxygen 3-oxidoreductase' WHERE ec = 'EC 1.14.13.129';
UPDATE enzyme SET history = 'EC 1.14.13.129 created 2011' WHERE ec = 'EC 1.14.13.129';
UPDATE enzyme SET sys_name = 'Pyrrole-2-carboxylate,NADH:oxygen oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.13.130';
UPDATE enzyme SET history = 'EC 1.14.13.130 created 2011' WHERE ec = 'EC 1.14.13.130';
UPDATE enzyme SET sys_name = 'Dimethyl sulfide,NADH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.131';
UPDATE enzyme SET history = 'EC 1.14.13.131 created 2011' WHERE ec = 'EC 1.14.13.131';
UPDATE enzyme SET sys_name = 'Squalene,NADPH:oxygen oxidoreductase (2,3-epoxidizing)' WHERE ec = 'EC 1.14.13.132';
UPDATE enzyme SET history = 'EC 1.14.13.132 created 1961 as EC 1.99.1.13, transferred 1965 to EC 1.14.1.3, part transferred 1972 to EC 1.14.99.7, transferred 2011 to EC 1.14.13.132' WHERE ec = 'EC 1.14.13.132';
UPDATE enzyme SET sys_name = 'Pentalenene,NADPH:oxygen 13-oxidoreductase' WHERE ec = 'EC 1.14.13.133';
UPDATE enzyme SET history = 'EC 1.14.13.133 created 2011' WHERE ec = 'EC 1.14.13.133';
UPDATE enzyme SET sys_name = 'Beta-amyrin,NADPH:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.13.134';
UPDATE enzyme SET history = 'EC 1.14.13.134 created 2011' WHERE ec = 'EC 1.14.13.134';
UPDATE enzyme SET sys_name = '1-hydroxy-2-naphthoate,NAD(P)H:oxygen oxidoreductase (2-hydroxylating, decarboxylating)' WHERE ec = 'EC 1.14.13.135';
UPDATE enzyme SET history = 'EC 1.14.13.135 created 2011' WHERE ec = 'EC 1.14.13.135';
UPDATE enzyme SET sys_name = 'Liquiritigenin,NADPH:oxygen oxidoreductase (hydroxylating, aryl migration)' WHERE ec = 'EC 1.14.13.136';
UPDATE enzyme SET history = 'EC 1.14.13.136 created 2011, modified 2013' WHERE ec = 'EC 1.14.13.136';
UPDATE enzyme SET sys_name = 'Indole,NAD(P)H:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.137';
UPDATE enzyme SET history = 'EC 1.14.13.137 created 2012' WHERE ec = 'EC 1.14.13.137';
UPDATE enzyme SET sys_name = 'Indolin-2-one,NAD(P)H:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.138';
UPDATE enzyme SET history = 'EC 1.14.13.138 created 2012' WHERE ec = 'EC 1.14.13.138';
UPDATE enzyme SET sys_name = '3-hydroxyindolin-2-one,NAD(P)H:oxygen oxidoreductase (2-hydroxy-2H-1,4-benzoxazin-3(4H)-one-forming)' WHERE ec = 'EC 1.14.13.139';
UPDATE enzyme SET history = 'EC 1.14.13.139 created 2012' WHERE ec = 'EC 1.14.13.139';
UPDATE enzyme SET sys_name = '2-hydroxy-2H-1,4-benzoxazin-3(4H)-one,NAD(P)H:oxygen oxidoreductase (N-hydroxylating)' WHERE ec = 'EC 1.14.13.140';
UPDATE enzyme SET history = 'EC 1.14.13.140 created 2012' WHERE ec = 'EC 1.14.13.140';
UPDATE enzyme SET sys_name = 'Cholest-4-en-3-one,NADH:oxygen oxidoreductase (26-hydroxylating)' WHERE ec = 'EC 1.14.13.141';
UPDATE enzyme SET history = 'EC 1.14.13.141 created 2012' WHERE ec = 'EC 1.14.13.141';
UPDATE enzyme SET sys_name = 'Androsta-1,4-diene-3,17-dione,NADH:oxygen oxidoreductase (9-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.142';
UPDATE enzyme SET history = 'EC 1.14.13.142 created 2012' WHERE ec = 'EC 1.14.13.142';
UPDATE enzyme SET sys_name = 'Ent-isokaurene,NADPH:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.13.143';
UPDATE enzyme SET history = 'EC 1.14.13.143 created 2012' WHERE ec = 'EC 1.14.13.143';
UPDATE enzyme SET sys_name = '9-beta-pimara-7,15-diene,NADPH:oxygen 19-oxidoreductase' WHERE ec = 'EC 1.14.13.144';
UPDATE enzyme SET history = 'EC 1.14.13.144 created 2012' WHERE ec = 'EC 1.14.13.144';
UPDATE enzyme SET sys_name = 'Ent-cassa-12,15-diene,NADPH:oxygen 11-oxidoreductase' WHERE ec = 'EC 1.14.13.145';
UPDATE enzyme SET history = 'EC 1.14.13.145 created 2012' WHERE ec = 'EC 1.14.13.145';
UPDATE enzyme SET sys_name = '10-beta-hydroxytaxa-4(20),11-dien-5-alpha-yl-acetate,NADPH:oxygen 14-oxidoreductase' WHERE ec = 'EC 1.14.13.146';
UPDATE enzyme SET history = 'EC 1.14.13.146 created 2012' WHERE ec = 'EC 1.14.13.146';
UPDATE enzyme SET sys_name = 'Taxusin,NADPH:oxygen 7-oxidoreductase' WHERE ec = 'EC 1.14.13.147';
UPDATE enzyme SET history = 'EC 1.14.13.147 created 2012' WHERE ec = 'EC 1.14.13.147';
UPDATE enzyme SET sys_name = 'N,N,N-trimethylamine,NADPH:oxygen oxidoreductase (N-oxide-forming)' WHERE ec = 'EC 1.14.13.148';
UPDATE enzyme SET history = 'EC 1.14.13.148 created 2012' WHERE ec = 'EC 1.14.13.148';
UPDATE enzyme SET sys_name = 'Phenylacetyl-CoA:oxygen oxidoreductase (1,2-epoxidizing)' WHERE ec = 'EC 1.14.13.149';
UPDATE enzyme SET history = 'EC 1.14.13.149 created 2012' WHERE ec = 'EC 1.14.13.149';
UPDATE enzyme SET sys_name = 'Alpha-humulene,NADPH:oxygen 10-oxidoreductase' WHERE ec = 'EC 1.14.13.150';
UPDATE enzyme SET history = 'EC 1.14.13.150 created 2012' WHERE ec = 'EC 1.14.13.150';
UPDATE enzyme SET sys_name = 'Linalool,NADH:oxygen oxidoreductase (8-hydroxylating)' WHERE ec = 'EC 1.14.13.151';
UPDATE enzyme SET history = 'EC 1.14.13.151 created 1989 as EC 1.14.99.28, transferred 2012 to EC 1.14.13.151' WHERE ec = 'EC 1.14.13.151';
UPDATE enzyme SET sys_name = 'Geraniol,NADPH:oxygen oxidoreductase (8-hydroxylating)' WHERE ec = 'EC 1.14.13.152';
UPDATE enzyme SET history = 'EC 1.14.13.152 created 2012' WHERE ec = 'EC 1.14.13.152';
UPDATE enzyme SET sys_name = '(+)-sabinene,NADPH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.153';
UPDATE enzyme SET history = 'EC 1.14.13.153 created 2012' WHERE ec = 'EC 1.14.13.153';
UPDATE enzyme SET sys_name = 'Erythromycin-D,NADPH:oxygen oxidoreductase (12-hydroxylating)' WHERE ec = 'EC 1.14.13.154';
UPDATE enzyme SET history = 'EC 1.14.13.154 created 2012' WHERE ec = 'EC 1.14.13.154';
UPDATE enzyme SET sys_name = '(-)-alpha-pinene,NADH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.155';
UPDATE enzyme SET history = 'EC 1.14.13.155 created 2012' WHERE ec = 'EC 1.14.13.155';
UPDATE enzyme SET sys_name = '1,8-cineole,NADPH:oxygen oxidoreductase (2-endo-hydroxylating)' WHERE ec = 'EC 1.14.13.156';
UPDATE enzyme SET history = 'EC 1.14.13.156 created 2012' WHERE ec = 'EC 1.14.13.156';
UPDATE enzyme SET sys_name = '1,8-cineole,NADPH:oxygen oxidoreductase (2-exo-hydroxylating)' WHERE ec = 'EC 1.14.13.157';
UPDATE enzyme SET history = 'EC 1.14.13.157 created 2012' WHERE ec = 'EC 1.14.13.157';
UPDATE enzyme SET sys_name = 'Amorpha-4,11-diene,NADPH:oxygen oxidoreductase (12-hydroxylating)' WHERE ec = 'EC 1.14.13.158';
UPDATE enzyme SET history = 'EC 1.14.13.158 created 2012' WHERE ec = 'EC 1.14.13.158';
UPDATE enzyme SET sys_name = 'Calciol,NADPH:oxygen oxidoreductase (25-hydroxylating)' WHERE ec = 'EC 1.14.13.159';
UPDATE enzyme SET history = 'EC 1.14.13.159 created 2012' WHERE ec = 'EC 1.14.13.159';
UPDATE enzyme SET sys_name = '((1R)-2,2,3-trimethyl-5-oxocyclopent-3-enyl)acetyl-CoA,NADPH:oxygen oxidoreductase (1,5-lactonizing)' WHERE ec = 'EC 1.14.13.160';
UPDATE enzyme SET history = 'EC 1.14.13.160 created 2012' WHERE ec = 'EC 1.14.13.160';
UPDATE enzyme SET sys_name = '(+)-camphor,NADPH:oxygen oxidoreductase (6-exo-hydroxylating)' WHERE ec = 'EC 1.14.13.161';
UPDATE enzyme SET history = 'EC 1.14.13.161 created 2012' WHERE ec = 'EC 1.14.13.161';
UPDATE enzyme SET sys_name = '(+)-bornane-2,5-dione,NADH:oxygen oxidoreductase (1,2-lactonizing)' WHERE ec = 'EC 1.14.13.162';
UPDATE enzyme SET history = 'EC 1.14.13.162 created 1972 as EC 1.14.15.2, transferred 2012 to EC 1.14.13.162' WHERE ec = 'EC 1.14.13.162';
UPDATE enzyme SET sys_name = '4-(6-hydroxypyridin-3-yl)-4-oxobutanoate,NADH:oxygen oxidoreductase (3-hydroxylating, succinate semialdehyde releasing)' WHERE ec = 'EC 1.14.13.163';
UPDATE enzyme SET history = 'EC 1.14.13.163 created 2012' WHERE ec = 'EC 1.14.13.163';
UPDATE enzyme SET history = 'EC 1.14.13.164 created 2012, deleted 2012' WHERE ec = 'EC 1.14.13.164';
UPDATE enzyme SET sys_name = 'L-arginine,NAD(P)H:oxygen oxidoreductase (nitric-oxide-forming)' WHERE ec = 'EC 1.14.13.165';
UPDATE enzyme SET history = 'EC 1.14.13.165 created 2012' WHERE ec = 'EC 1.14.13.165';
UPDATE enzyme SET sys_name = '4-nitrocatechol,NAD(P)H:oxygen 4-oxidoreductase (4-hydroxylating, nitrite-forming)' WHERE ec = 'EC 1.14.13.166';
UPDATE enzyme SET history = 'EC 1.14.13.166 created 2012' WHERE ec = 'EC 1.14.13.166';
UPDATE enzyme SET sys_name = '4-nitrophenol,NAD(P)H:oxygen 4-oxidoreductase (4-hydroxylating, nitrite-forming)' WHERE ec = 'EC 1.14.13.167';
UPDATE enzyme SET history = 'EC 1.14.13.167 created 2012' WHERE ec = 'EC 1.14.13.167';
UPDATE enzyme SET sys_name = 'Indole-3-pyruvate,NADPH:oxygen oxidoreductase (1-hydroxylating, decarboxylating)' WHERE ec = 'EC 1.14.13.168';
UPDATE enzyme SET history = 'EC 1.14.13.168 created 2012' WHERE ec = 'EC 1.14.13.168';
UPDATE enzyme SET sys_name = 'Sphinganine,NADPH:oxygen oxidoreductase (C(4)-hydroxylating)' WHERE ec = 'EC 1.14.13.169';
UPDATE enzyme SET history = 'EC 1.14.13.169 created 2012' WHERE ec = 'EC 1.14.13.169';
UPDATE enzyme SET sys_name = '1-deoxy-11-oxopentalenate,NADH:oxygen oxidoreductase (pentalenolactone-D forming)' WHERE ec = 'EC 1.14.13.170';
UPDATE enzyme SET history = 'EC 1.14.13.170 created 2012' WHERE ec = 'EC 1.14.13.170';
UPDATE enzyme SET sys_name = '1-deoxy-11-oxopentalenate,NADH:oxygen oxidoreductase (neopentalenolactone-D forming)' WHERE ec = 'EC 1.14.13.171';
UPDATE enzyme SET history = 'EC 1.14.13.171 created 2012' WHERE ec = 'EC 1.14.13.171';
UPDATE enzyme SET sys_name = 'Salicylate,NADH:oxygen oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.13.172';
UPDATE enzyme SET history = 'EC 1.14.13.172 created 2013' WHERE ec = 'EC 1.14.13.172';
UPDATE enzyme SET sys_name = '11-oxo-beta-amyrin,NADPH:oxygen oxidoreductase (30-hydroxylating)' WHERE ec = 'EC 1.14.13.173';
UPDATE enzyme SET history = 'EC 1.14.13.173 created 2013' WHERE ec = 'EC 1.14.13.173';
UPDATE enzyme SET sys_name = '(1S)-averantin,NADPH:O(2) oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.13.174';
UPDATE enzyme SET history = 'EC 1.14.13.174 created 2013' WHERE ec = 'EC 1.14.13.174';
UPDATE enzyme SET sys_name = '8-O-methylsterigmatocystin,NADPH:O(2) oxidoreductase (aflatoxin-B forming)' WHERE ec = 'EC 1.14.13.175';
UPDATE enzyme SET history = 'EC 1.14.13.175 created 2013' WHERE ec = 'EC 1.14.13.175';
UPDATE enzyme SET sys_name = 'Tryprostatin B,NADPH:oxygen oxidoreductase (6-hydroxytryprostatin B-forming)' WHERE ec = 'EC 1.14.13.176';
UPDATE enzyme SET history = 'EC 1.14.13.176 created 2013' WHERE ec = 'EC 1.14.13.176';
UPDATE enzyme SET sys_name = 'Fumitremorgin C,NADPH:oxygen oxidoreductase (12-alpha,13-alpha-dihydroxyfumitremorgin C-forming)' WHERE ec = 'EC 1.14.13.177';
UPDATE enzyme SET history = 'EC 1.14.13.177 created 2013' WHERE ec = 'EC 1.14.13.177';
UPDATE enzyme SET sys_name = 'Caffeine:oxygen oxidoreductase (N(1)-demethylating)' WHERE ec = 'EC 1.14.13.178';
UPDATE enzyme SET history = 'EC 1.14.13.178 created 2013' WHERE ec = 'EC 1.14.13.178';
UPDATE enzyme SET sys_name = 'Theobromine:oxygen oxidoreductase (N(3)-demethylating)' WHERE ec = 'EC 1.14.13.179';
UPDATE enzyme SET history = 'EC 1.14.13.179 created 2013' WHERE ec = 'EC 1.14.13.179';
UPDATE enzyme SET sys_name = 'Aklavinone,NADPH:oxygen oxidoreductase (12-hydroxylating)' WHERE ec = 'EC 1.14.13.180';
UPDATE enzyme SET history = 'EC 1.14.13.180 created 2013' WHERE ec = 'EC 1.14.13.180';
UPDATE enzyme SET sys_name = '13-deoxydaunorubicin,NADPH:oxygen oxidoreductase (13-hydroxylating)' WHERE ec = 'EC 1.14.13.181';
UPDATE enzyme SET history = 'EC 1.14.13.181 created 2013' WHERE ec = 'EC 1.14.13.181';
UPDATE enzyme SET sys_name = '2-heptyl-4(1H)-quinolone,NADH:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.13.182';
UPDATE enzyme SET history = 'EC 1.14.13.182 created 2013' WHERE ec = 'EC 1.14.13.182';
UPDATE enzyme SET sys_name = 'Dammarenediol-II,NADPH:O(2) oxidoreductase (12-beta-hydroxylating)' WHERE ec = 'EC 1.14.13.183';
UPDATE enzyme SET history = 'EC 1.14.13.183 created 2013' WHERE ec = 'EC 1.14.13.183';
UPDATE enzyme SET sys_name = 'Protopanaxadiol,NADPH:O(2) oxidoreductase (6-alpha-hydroxylating)' WHERE ec = 'EC 1.14.13.184';
UPDATE enzyme SET history = 'EC 1.14.13.184 created 2013' WHERE ec = 'EC 1.14.13.184';
UPDATE enzyme SET sys_name = 'Narbomycin,NADH:oxygen oxidoreductase (pikromycin-forming)' WHERE ec = 'EC 1.14.13.185';
UPDATE enzyme SET history = 'EC 1.14.13.185 created 2014' WHERE ec = 'EC 1.14.13.185';
UPDATE enzyme SET sys_name = '20-oxo-5-O-beta-mycaminosyltylactone,NADPH:oxygen oxidoreductase (23-hydroxylating)' WHERE ec = 'EC 1.14.13.186';
UPDATE enzyme SET history = 'EC 1.14.13.186 created 2014' WHERE ec = 'EC 1.14.13.186';
UPDATE enzyme SET sys_name = 'dTDP-beta-L-evernosamine,NADPH:oxygen oxidoreductase (N-hydroxylating)' WHERE ec = 'EC 1.14.13.187';
UPDATE enzyme SET history = 'EC 1.14.13.187 created 2014' WHERE ec = 'EC 1.14.13.187';
UPDATE enzyme SET sys_name = '6-deoxyerythronolide-B,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.188';
UPDATE enzyme SET history = 'EC 1.14.13.188 created 2014' WHERE ec = 'EC 1.14.13.188';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.13.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.13.n2';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.13.n3';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.13.n4';
UPDATE enzyme SET sys_name = 'Dihomomethionine,NADPH:oxygen oxidoreductase (N-hydroxylating)' WHERE ec = 'EC 1.14.13.n5';
UPDATE enzyme SET history = 'EC 1.14.13.n5 created 2010' WHERE ec = 'EC 1.14.13.n5';
UPDATE enzyme SET sys_name = 'Hexahomomethionine,NADPH:oxygen oxidoreductase (N-hydroxylating)' WHERE ec = 'EC 1.14.13.n6';
UPDATE enzyme SET history = 'EC 1.14.13.n6 created 2010' WHERE ec = 'EC 1.14.13.n6';
UPDATE enzyme SET sys_name = '4-nitrophenol,NADPH:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.13.n7';
UPDATE enzyme SET history = 'EC 1.14.13.n7 created 2010' WHERE ec = 'EC 1.14.13.n7';
UPDATE enzyme SET sys_name = '(Z)-2-methylbutanal oxime,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.13.n8';
UPDATE enzyme SET history = 'EC 1.14.13.n8 created 2012' WHERE ec = 'EC 1.14.13.n8';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.13.n9';
UPDATE enzyme SET sys_name = 'Substrate,reduced-flavoprotein:oxygen oxidoreductase (RH-hydroxylating or -epoxidizing)' WHERE ec = 'EC 1.14.14.1';
UPDATE enzyme SET history = 'EC 1.14.14.1 created 1961 as EC 1.99.1.1, transferred 1965 to EC 1.14.1.1, transferred 1972 to EC 1.14.14.1 (EC 1.14.14.2 created 1972, incorporated 1976; EC 1.14.99.8 created 1972, incorporated 1984)' WHERE ec = 'EC 1.14.14.1';
UPDATE enzyme SET history = 'EC 1.14.14.2 created 1972, deleted 1976' WHERE ec = 'EC 1.14.14.2';
UPDATE enzyme SET sys_name = 'Alkanal,reduced-FMN:oxygen oxidoreductase (1-hydroxylating, luminescing)' WHERE ec = 'EC 1.14.14.3';
UPDATE enzyme SET history = 'EC 1.14.14.3 created 1981' WHERE ec = 'EC 1.14.14.3';
UPDATE enzyme SET history = 'EC 1.14.14.4 created 2000, deleted 2002' WHERE ec = 'EC 1.14.14.4';
UPDATE enzyme SET sys_name = 'Alkanesulfonate, reduced-FMN:oxygen oxidoreductase' WHERE ec = 'EC 1.14.14.5';
UPDATE enzyme SET history = 'EC 1.14.14.5 created 2002' WHERE ec = 'EC 1.14.14.5';
UPDATE enzyme SET history = 'EC 1.14.14.6 created 2009, deleted 2010' WHERE ec = 'EC 1.14.14.6';
UPDATE enzyme SET sys_name = 'L-tryptophan:FADH(2) oxidoreductase (7-halogenating)' WHERE ec = 'EC 1.14.14.7';
UPDATE enzyme SET history = 'EC 1.14.14.7 created 2009' WHERE ec = 'EC 1.14.14.7';
UPDATE enzyme SET sys_name = 'Anthranilate,FAD:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.14.8';
UPDATE enzyme SET history = 'EC 1.14.14.8 created 2010' WHERE ec = 'EC 1.14.14.8';
UPDATE enzyme SET sys_name = '4-hydroxyphenylacetate,FAD:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.14.9';
UPDATE enzyme SET history = 'EC 1.14.14.9 created 1972 as EC 1.14.13.3, transferred 2011 to EC 1.14.14.9' WHERE ec = 'EC 1.14.14.9';
UPDATE enzyme SET sys_name = 'Nitrilotriacetate,FMNH(2):oxygen oxidoreductase (glyoxylate-forming)' WHERE ec = 'EC 1.14.14.10';
UPDATE enzyme SET history = 'EC 1.14.14.10 created 2011' WHERE ec = 'EC 1.14.14.10';
UPDATE enzyme SET sys_name = 'Styrene,FADH(2):oxygen oxidoreductase' WHERE ec = 'EC 1.14.14.11';
UPDATE enzyme SET history = 'EC 1.14.14.11 created 2011' WHERE ec = 'EC 1.14.14.11';
UPDATE enzyme SET sys_name = '3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione,FADH(2):oxygen oxidoreductase' WHERE ec = 'EC 1.14.14.12';
UPDATE enzyme SET history = 'EC 1.14.14.12 created 2011' WHERE ec = 'EC 1.14.14.12';
UPDATE enzyme SET sys_name = '4-(gamma-L-glutamylamino)butanoyl-[BtrI acyl-carrier protein],FMNH(2):oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.14.13';
UPDATE enzyme SET history = 'EC 1.14.14.13 created 2012' WHERE ec = 'EC 1.14.14.13';
UPDATE enzyme SET sys_name = 'Testosterone monooxygenase (17-beta-estradiol-forming)' WHERE ec = 'EC 1.14.14.14';
UPDATE enzyme SET history = 'EC 1.14.14.14 created 2013' WHERE ec = 'EC 1.14.14.14';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.14.n1';
UPDATE enzyme SET sys_name = '(+)-camphor,reduced putidaredoxin:oxygen oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.15.1';
UPDATE enzyme SET history = 'EC 1.14.15.1 created 1972, modified 1986' WHERE ec = 'EC 1.14.15.1';
UPDATE enzyme SET history = 'EC 1.14.15.2 created 1972, deleted 2012' WHERE ec = 'EC 1.14.15.2';
UPDATE enzyme SET sys_name = 'Alkane,reduced-rubredoxin:oxygen 1-oxidoreductase' WHERE ec = 'EC 1.14.15.3';
UPDATE enzyme SET history = 'EC 1.14.15.3 created 1972' WHERE ec = 'EC 1.14.15.3';
UPDATE enzyme SET sys_name = 'Steroid,reduced-adrenodoxin:oxygen oxidoreductase (11-beta-hydroxylating)' WHERE ec = 'EC 1.14.15.4';
UPDATE enzyme SET history = 'EC 1.14.15.4 created 1961 as EC 1.99.1.7, transferred 1965 to EC 1.14.1.6, transferred 1972 to EC 1.14.15.4, modified 1989' WHERE ec = 'EC 1.14.15.4';
UPDATE enzyme SET sys_name = 'Corticosterone,reduced-adrenodoxin:oxygen oxidoreductase (18-hydroxylating)' WHERE ec = 'EC 1.14.15.5';
UPDATE enzyme SET history = 'EC 1.14.15.5 created 1972' WHERE ec = 'EC 1.14.15.5';
UPDATE enzyme SET sys_name = 'Cholesterol,reduced-adrenodoxin:oxygen oxidoreductase (side-chain-cleaving)' WHERE ec = 'EC 1.14.15.6';
UPDATE enzyme SET history = 'EC 1.14.15.6 created 1983, modified 2013' WHERE ec = 'EC 1.14.15.6';
UPDATE enzyme SET sys_name = 'Choline,reduced-ferredoxin:oxygen oxidoreductase' WHERE ec = 'EC 1.14.15.7';
UPDATE enzyme SET history = 'EC 1.14.15.7 created 2001, modified 2002 (EC 1.14.14.4 created 2000, incorporated 2002), modified 2005, modified 2011' WHERE ec = 'EC 1.14.15.7';
UPDATE enzyme SET sys_name = 'Progesterone,reduced-ferredoxin:oxygen oxidoreductase (15-beta-hydroxylating)' WHERE ec = 'EC 1.14.15.8';
UPDATE enzyme SET history = 'EC 1.14.15.8 created 2010' WHERE ec = 'EC 1.14.15.8';
UPDATE enzyme SET sys_name = 'Spheroidene,reduced-ferredoxin:oxygen oxidoreductase (spheroiden-2-one-forming)' WHERE ec = 'EC 1.14.15.9';
UPDATE enzyme SET history = 'EC 1.14.15.9 created 2012' WHERE ec = 'EC 1.14.15.9';
UPDATE enzyme SET sys_name = '(+)-camphor,reduced putidaredoxin:oxygen oxidoreductase (6-endo-hydroxylating)' WHERE ec = 'EC 1.14.15.10';
UPDATE enzyme SET history = 'EC 1.14.15.10 created 2012' WHERE ec = 'EC 1.14.15.10';
UPDATE enzyme SET sys_name = '1-deoxypentalenate,reduced ferredoxin:O(2) oxidoreductase' WHERE ec = 'EC 1.14.15.11';
UPDATE enzyme SET history = 'EC 1.14.15.11 created 2012' WHERE ec = 'EC 1.14.15.11';
UPDATE enzyme SET sys_name = 'Acyl-[acyl-carrier protein],reduced-flavodoxin:oxygen oxidoreductase (pimeloyl-[acyl-carrier protein] forming)' WHERE ec = 'EC 1.14.15.12';
UPDATE enzyme SET history = 'EC 1.14.15.12 created 2013' WHERE ec = 'EC 1.14.15.12';
UPDATE enzyme SET sys_name = 'Cyclo(L-leucyl-L-leucyl),reduced-ferredoxin:oxygen oxidoreductase (N-hydroxylating,aromatizing)' WHERE ec = 'EC 1.14.15.13';
UPDATE enzyme SET history = 'EC 1.14.15.13 created 2013' WHERE ec = 'EC 1.14.15.13';
UPDATE enzyme SET sys_name = 'L-phenylalanine,tetrahydrobiopterin:oxygen oxidoreductase (4-hydroxylating)' WHERE ec = 'EC 1.14.16.1';
UPDATE enzyme SET history = 'EC 1.14.16.1 created 1961 as EC 1.99.1.2, transferred 1965 to EC 1.14.3.1, transferred 1972 to EC 1.14.16.1, modified 2002, modified 2003' WHERE ec = 'EC 1.14.16.1';
UPDATE enzyme SET sys_name = 'L-tyrosine,tetrahydrobiopterin:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.16.2';
UPDATE enzyme SET history = 'EC 1.14.16.2 created 1972, modified 2003' WHERE ec = 'EC 1.14.16.2';
UPDATE enzyme SET sys_name = 'Anthranilate,tetrahydrobiopterin:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.16.3';
UPDATE enzyme SET history = 'EC 1.14.16.3 created 1972' WHERE ec = 'EC 1.14.16.3';
UPDATE enzyme SET sys_name = 'L-tryptophan,tetrahydrobiopterin:oxygen oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.16.4';
UPDATE enzyme SET history = 'EC 1.14.16.4 created 1972, modified 2003' WHERE ec = 'EC 1.14.16.4';
UPDATE enzyme SET sys_name = '1-alkyl-sn-glycerol,tetrahydrobiopterin:oxygen oxidoreductase' WHERE ec = 'EC 1.14.16.5';
UPDATE enzyme SET history = 'EC 1.14.16.5 created 1972 as EC 1.14.99.17, transferred 1976 to EC 1.14.16.5, modified 2010' WHERE ec = 'EC 1.14.16.5';
UPDATE enzyme SET sys_name = '(S)-2-hydroxy-2-phenylacetate,tetrahydrobiopterin:oxygen oxidoreductase (4-hydroxylating)' WHERE ec = 'EC 1.14.16.6';
UPDATE enzyme SET history = 'EC 1.14.16.6 created 1984' WHERE ec = 'EC 1.14.16.6';
UPDATE enzyme SET sys_name = '3,4-dihydroxyphenethylamine,ascorbate:oxygen oxidoreductase (beta-hydroxylating)' WHERE ec = 'EC 1.14.17.1';
UPDATE enzyme SET history = 'EC 1.14.17.1 created 1965 as EC 1.14.2.1, transferred 1972 to EC 1.14.17.1' WHERE ec = 'EC 1.14.17.1';
UPDATE enzyme SET history = 'EC 1.14.17.2 created 1972, deleted 1984' WHERE ec = 'EC 1.14.17.2';
UPDATE enzyme SET sys_name = 'Peptidylglycine,ascorbate:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.17.3';
UPDATE enzyme SET history = 'EC 1.14.17.3 created 1989' WHERE ec = 'EC 1.14.17.3';
UPDATE enzyme SET sys_name = '1-aminocyclopropane-1-carboxylate oxygenase (ethylene-forming)' WHERE ec = 'EC 1.14.17.4';
UPDATE enzyme SET history = 'EC 1.14.17.4 created 2003' WHERE ec = 'EC 1.14.17.4';
UPDATE enzyme SET sys_name = 'L-tyrosine,L-dopa:oxygen oxidoreductase' WHERE ec = 'EC 1.14.18.1';
UPDATE enzyme SET history = 'EC 1.14.18.1 created 1972, modified 1976, modified 1980 (EC 1.14.17.2 created 1972, incorporated 1984), modified 2012' WHERE ec = 'EC 1.14.18.1';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate,ferrocytochrome-b5:oxygen oxidoreductase (N-acetyl-hydroxylating)' WHERE ec = 'EC 1.14.18.2';
UPDATE enzyme SET history = 'EC 1.14.18.2 created 1992 as EC 1.14.13.45, transferred 2003 to EC 1.14.18.2' WHERE ec = 'EC 1.14.18.2';
UPDATE enzyme SET sys_name = 'Methane,quinol:oxygen oxidoreductase' WHERE ec = 'EC 1.14.18.3';
UPDATE enzyme SET history = 'EC 1.14.18.3 created 2011' WHERE ec = 'EC 1.14.18.3';
UPDATE enzyme SET sys_name = 'Stearoyl-CoA,ferrocytochrome-b5:oxygen oxidoreductase (9,10-dehydrogenating)' WHERE ec = 'EC 1.14.19.1';
UPDATE enzyme SET history = 'EC 1.14.19.1 created 1972 as EC 1.14.99.5, modified 1986, modified 2000, transferred 2000 to EC 1.14.19.1, modified 2003' WHERE ec = 'EC 1.14.19.1';
UPDATE enzyme SET sys_name = 'Acyl-[acyl-carrier-protein], hydrogen-donor:oxygen oxidoreductase' WHERE ec = 'EC 1.14.19.2';
UPDATE enzyme SET history = 'EC 1.14.19.2 created 1972 as EC 1.14.99.6, modified 2000, transferred 2000 to EC 1.14.19.2' WHERE ec = 'EC 1.14.19.2';
UPDATE enzyme SET sys_name = 'Linoleoyl-CoA,hydrogen-donor:oxygen oxidoreductase' WHERE ec = 'EC 1.14.19.3';
UPDATE enzyme SET history = 'EC 1.14.19.3 created 1986 as EC 1.14.99.25, transferred 2000 to EC 1.14.19.3' WHERE ec = 'EC 1.14.19.3';
UPDATE enzyme SET sys_name = 'Phytosphinganine,hydrogen donor:oxygen Delta(8)-oxidoreductase' WHERE ec = 'EC 1.14.19.4';
UPDATE enzyme SET history = 'EC 1.14.19.4 created 2008' WHERE ec = 'EC 1.14.19.4';
UPDATE enzyme SET sys_name = 'Acyl-CoA,hydrogen donor:oxygen Delta(11)-oxidoreductase' WHERE ec = 'EC 1.14.19.5';
UPDATE enzyme SET history = 'EC 1.14.19.5 created 2008' WHERE ec = 'EC 1.14.19.5';
UPDATE enzyme SET sys_name = 'Acyl-CoA,hydrogen donor:oxygen Delta(12)-oxidoreductase' WHERE ec = 'EC 1.14.19.6';
UPDATE enzyme SET history = 'EC 1.14.19.6 created 2008' WHERE ec = 'EC 1.14.19.6';
UPDATE enzyme SET sys_name = '(S)-2-hydroxypropylphosphonate,NADH:oxygen epoxidase' WHERE ec = 'EC 1.14.19.7';
UPDATE enzyme SET history = 'EC 1.14.19.7 created 2011' WHERE ec = 'EC 1.14.19.7';
UPDATE enzyme SET sys_name = 'Pentalenolactone-reduced-ferredoxin:oxygen oxidoreductase (pentalenolactone forming)' WHERE ec = 'EC 1.14.19.8';
UPDATE enzyme SET history = 'EC 1.14.19.8 created 2012 as EC 1.3.7.10, transferred to EC 1.14.19.8 2013' WHERE ec = 'EC 1.14.19.8';
UPDATE enzyme SET sys_name = 'Stearoyl-CoA,NADPH:oxygen Delta(8)-oxidoreductase' WHERE ec = 'EC 1.14.19.n4';
UPDATE enzyme SET history = 'EC 1.14.19.n4 created 2013' WHERE ec = 'EC 1.14.19.n4';
UPDATE enzyme SET sys_name = 'Versicolorin B,NADPH:oxygen Delta(1)-oxidoreductase' WHERE ec = 'EC 1.14.19.n5';
UPDATE enzyme SET history = 'EC 1.14.19.n5 created 2013' WHERE ec = 'EC 1.14.19.n5';
UPDATE enzyme SET sys_name = 'Penicillin-N,2-oxoglutarate:oxygen oxidoreductase (ring-expanding)' WHERE ec = 'EC 1.14.20.1';
UPDATE enzyme SET history = 'EC 1.14.20.1 created 2002' WHERE ec = 'EC 1.14.20.1';
UPDATE enzyme SET sys_name = '(2R)-4-hydroxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside:oxygen oxidoreductase (7-hydroxylating)' WHERE ec = 'EC 1.14.20.2';
UPDATE enzyme SET history = 'EC 1.14.20.2 created 2012' WHERE ec = 'EC 1.14.20.2';
UPDATE enzyme SET sys_name = '(3S,5S)-carbapenam-3-carboxylate,2-oxoglutarate:oxygen oxidoreductase (dehydrating)' WHERE ec = 'EC 1.14.20.3';
UPDATE enzyme SET history = 'EC 1.14.20.3 created 2013' WHERE ec = 'EC 1.14.20.3';
UPDATE enzyme SET sys_name = '(S)-cheilanthifoline,NADPH:oxygen oxidoreductase (methylenedioxy-bridge-forming)' WHERE ec = 'EC 1.14.21.1';
UPDATE enzyme SET history = 'EC 1.14.21.1 created 1999 as EC 1.1.3.32, transferred 2002 to EC 1.14.21.1' WHERE ec = 'EC 1.14.21.1';
UPDATE enzyme SET sys_name = '(S)-scoulerine,NADPH:oxygen oxidoreductase (methylenedioxy-bridge-forming)' WHERE ec = 'EC 1.14.21.2';
UPDATE enzyme SET history = 'EC 1.14.21.2 created 1999 as EC 1.1.3.33, transferred 2002 to EC 1.14.21.2' WHERE ec = 'EC 1.14.21.2';
UPDATE enzyme SET sys_name = '(S)-N-methylcoclaurine,NADPH:oxygen oxidoreductase (C-O phenol-coupling)' WHERE ec = 'EC 1.14.21.3';
UPDATE enzyme SET history = 'EC 1.14.21.3 created 1999 as EC 1.1.3.34, transferred 2002 to EC 1.14.21.3' WHERE ec = 'EC 1.14.21.3';
UPDATE enzyme SET sys_name = '(R)-reticuline,NADPH:oxygen oxidoreductase (C-C phenol-coupling)' WHERE ec = 'EC 1.14.21.4';
UPDATE enzyme SET history = 'EC 1.14.21.4 created 1999 as EC 1.1.3.35, transferred 2002 to EC 1.14.21.4' WHERE ec = 'EC 1.14.21.4';
UPDATE enzyme SET sys_name = '(S)-tetrahydrocolumbamine,NADPH:oxygen oxidoreductase (methylenedioxy-bridge-forming)' WHERE ec = 'EC 1.14.21.5';
UPDATE enzyme SET history = 'EC 1.14.21.5 created 1999 as EC 1.1.3.36, transferred 2002 to EC 1.14.21.5' WHERE ec = 'EC 1.14.21.5';
UPDATE enzyme SET sys_name = '5-alpha-cholest-7-en-3-beta-ol,NAD(P)H:oxygen 5-oxidoreductase' WHERE ec = 'EC 1.14.21.6';
UPDATE enzyme SET history = 'EC 1.14.21.6 created 1972 as EC 1.3.3.2, transferred 2005 to EC 1.14.21.6' WHERE ec = 'EC 1.14.21.6';
UPDATE enzyme SET sys_name = 'Flaviolin,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.21.7';
UPDATE enzyme SET history = 'EC 1.14.21.7 created 2008' WHERE ec = 'EC 1.14.21.7';
UPDATE enzyme SET sys_name = 'Calycosin,NADPH:oxygen oxidoreductase (methylenedioxy-bridge-forming)' WHERE ec = 'EC 1.14.21.8';
UPDATE enzyme SET history = 'EC 1.14.21.8 created 2011' WHERE ec = 'EC 1.14.21.8';
UPDATE enzyme SET sys_name = 'Cyclo(L-tyrosyl-L-tyrosyl),NADPH:oxygen oxidoreductase (diarylbridge-forming)' WHERE ec = 'EC 1.14.21.9';
UPDATE enzyme SET history = 'EC 1.14.21.9 created 2013' WHERE ec = 'EC 1.14.21.9';
UPDATE enzyme SET sys_name = 'Tryprostatin A,NADPH:oxygen oxidoreductase' WHERE ec = 'EC 1.14.21.10';
UPDATE enzyme SET history = 'EC 1.14.21.10 created 2013' WHERE ec = 'EC 1.14.21.10';
UPDATE enzyme SET sys_name = '(5Z,8Z,11Z,14Z)-icosa-5,8,11,14-tetraenoate,hydrogen-donor:oxygen oxidoreductase' WHERE ec = 'EC 1.14.99.1';
UPDATE enzyme SET history = 'EC 1.14.99.1 created 1972, modified 1990' WHERE ec = 'EC 1.14.99.1';
UPDATE enzyme SET sys_name = 'Kynurenate,hydrogen-donor:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.99.2';
UPDATE enzyme SET history = 'EC 1.14.99.2 created 1965 as EC 1.14.1.4, transferred 1972 to EC 1.14.99.2' WHERE ec = 'EC 1.14.99.2';
UPDATE enzyme SET sys_name = 'Protoheme,hydrogen-donor:oxygen oxidoreductase (alpha-methene-oxidizing, hydroxylating)' WHERE ec = 'EC 1.14.99.3';
UPDATE enzyme SET history = 'EC 1.14.99.3 created 1972, modified 2006' WHERE ec = 'EC 1.14.99.3';
UPDATE enzyme SET sys_name = 'Progesterone,hydrogen-donor:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.99.4';
UPDATE enzyme SET history = 'EC 1.14.99.4 created 1972, modified 1999' WHERE ec = 'EC 1.14.99.4';
UPDATE enzyme SET history = 'EC 1.14.99.5 created 1972, modified 1986, modified 2000, deleted 2000' WHERE ec = 'EC 1.14.99.5';
UPDATE enzyme SET history = 'EC 1.14.99.6 created 1972, modified 2000, deleted 2000' WHERE ec = 'EC 1.14.99.6';
UPDATE enzyme SET history = 'EC 1.14.99.7 created 1961 as EC 1.99.1.13, transferred 1965 to EC 1.14.1.3, part transferred 1972 to EC 1.14.99.7 rest to EC 5.4.99.7, deleted 2011' WHERE ec = 'EC 1.14.99.7';
UPDATE enzyme SET history = 'EC 1.14.99.8 created 1972, deleted 1984' WHERE ec = 'EC 1.14.99.8';
UPDATE enzyme SET sys_name = 'Steroid,NADPH--hemoprotein reductase:oxygen oxidoreductase (17-alpha-hydroxylating)' WHERE ec = 'EC 1.14.99.9';
UPDATE enzyme SET history = 'EC 1.14.99.9 created 1961 as EC 1.99.1.9, transferred 1965 to EC 1.14.1.7, transferred 1972 to EC 1.14.99.9, modified 2013' WHERE ec = 'EC 1.14.99.9';
UPDATE enzyme SET sys_name = 'Steroid,NADPH--hemoprotein reductase:oxygen oxidoreductase (21-hydroxylating)' WHERE ec = 'EC 1.14.99.10';
UPDATE enzyme SET history = 'EC 1.14.99.10 created 1961 as EC 1.99.1.11, transferred 1965 to EC 1.14.1.8, transferred 1972 to EC 1.14.99.10, modified 2013' WHERE ec = 'EC 1.14.99.10';
UPDATE enzyme SET sys_name = 'Estradiol-17-beta,hydrogen-donor:oxygen oxidoreductase (6-beta-hydroxylating)' WHERE ec = 'EC 1.14.99.11';
UPDATE enzyme SET history = 'EC 1.14.99.11 created 1965 as EC 1.14.1.10, transferred 1972 to EC 1.14.99.11' WHERE ec = 'EC 1.14.99.11';
UPDATE enzyme SET sys_name = 'Androst-4-ene-3,17-dione-hydrogen-donor:oxygen oxidoreductase (13-hydroxylating, lactonizing)' WHERE ec = 'EC 1.14.99.12';
UPDATE enzyme SET history = 'EC 1.14.99.12 created 1972, modified 1999' WHERE ec = 'EC 1.14.99.12';
UPDATE enzyme SET history = 'EC 1.14.99.13 created 1972, deleted 1984' WHERE ec = 'EC 1.14.99.13';
UPDATE enzyme SET sys_name = 'Progesterone,hydrogen-donor:oxygen oxidoreductase (11-alpha-hydroxylating)' WHERE ec = 'EC 1.14.99.14';
UPDATE enzyme SET history = 'EC 1.14.99.14 created 1972' WHERE ec = 'EC 1.14.99.14';
UPDATE enzyme SET sys_name = '4-methoxybenzoate,hydrogen-donor:oxygen oxidoreductase (O-demethylating)' WHERE ec = 'EC 1.14.99.15';
UPDATE enzyme SET history = 'EC 1.14.99.15 created 1972' WHERE ec = 'EC 1.14.99.15';
UPDATE enzyme SET history = 'EC 1.14.99.164 created 1972, deleted 2002' WHERE ec = 'EC 1.14.99.16';
UPDATE enzyme SET history = 'EC 1.14.99.17 created 1972, deleted 1976' WHERE ec = 'EC 1.14.99.17';
UPDATE enzyme SET history = 'EC 1.14.99.18 created 1976, modified 1999, deleted 2003' WHERE ec = 'EC 1.14.99.18';
UPDATE enzyme SET sys_name = 'O-1-alkyl-2-acyl-sn-glycero-3-phosphoethanolamine,hydrogen-donor:oxygen oxidoreductase' WHERE ec = 'EC 1.14.99.19';
UPDATE enzyme SET history = 'EC 1.14.99.19 created 1976' WHERE ec = 'EC 1.14.99.19';
UPDATE enzyme SET sys_name = 'Phylloquinone,hydrogen-donor:oxygen oxidoreductase (2,3-epoxidizing)' WHERE ec = 'EC 1.14.99.20';
UPDATE enzyme SET history = 'EC 1.14.99.20 created 1976' WHERE ec = 'EC 1.14.99.20';
UPDATE enzyme SET sys_name = 'Latia-luciferin,hydrogen-donor:oxygen oxidoreductase (demethylating)' WHERE ec = 'EC 1.14.99.21';
UPDATE enzyme SET history = 'EC 1.14.99.21 created 1976, modified 1982' WHERE ec = 'EC 1.14.99.21';
UPDATE enzyme SET sys_name = 'Ecdysone,hydrogen-donor:oxygen oxidoreductase (20-hydroxylating)' WHERE ec = 'EC 1.14.99.22';
UPDATE enzyme SET history = 'EC 1.14.99.22 created 1978' WHERE ec = 'EC 1.14.99.22';
UPDATE enzyme SET sys_name = '3-hydroxybenzoate,hydrogen-donor:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.99.23';
UPDATE enzyme SET history = 'EC 1.14.99.23 created 1984' WHERE ec = 'EC 1.14.99.23';
UPDATE enzyme SET sys_name = 'Steroid,hydrogen-donor:oxygen oxidoreductase (9-epoxidizing)' WHERE ec = 'EC 1.14.99.24';
UPDATE enzyme SET history = 'EC 1.14.99.24 created 1986' WHERE ec = 'EC 1.14.99.24';
UPDATE enzyme SET history = 'EC 1.14.99.25 created 1986, deleted 2000' WHERE ec = 'EC 1.14.99.25';
UPDATE enzyme SET sys_name = '2-hydroxypyridine,hydrogen-donor:oxygen oxidoreductase (5-hydroxylating)' WHERE ec = 'EC 1.14.99.26';
UPDATE enzyme SET history = 'EC 1.14.99.26 created 1989' WHERE ec = 'EC 1.14.99.26';
UPDATE enzyme SET sys_name = '5-hydroxy-1,4-naphthoquinone,hydrogen-donor:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.99.27';
UPDATE enzyme SET history = 'EC 1.14.99.27 created 1989' WHERE ec = 'EC 1.14.99.27';
UPDATE enzyme SET history = 'EC 1.14.99.28 created 1989, deleted 2012' WHERE ec = 'EC 1.14.99.28';
UPDATE enzyme SET sys_name = 'Deoxyhypusine,hydrogen-donor:oxygen oxidoreductase (2-hydroxylating)' WHERE ec = 'EC 1.14.99.29';
UPDATE enzyme SET history = 'EC 1.14.99.29 created 1989' WHERE ec = 'EC 1.14.99.29';
UPDATE enzyme SET history = 'EC 1.14.99.30 created 1999, deleted 2011' WHERE ec = 'EC 1.14.99.30';
UPDATE enzyme SET sys_name = 'n-tetradecanoyl-CoA,NAD(P)H:O(2) oxidoreductase (11-(E) desaturating)' WHERE ec = 'EC 1.14.99.31';
UPDATE enzyme SET history = 'EC 1.14.99.31 created 2000' WHERE ec = 'EC 1.14.99.31';
UPDATE enzyme SET sys_name = 'n-tetradecanoyl-CoA,NAD(P)H:O(2) oxidoreductase (11-(Z) desaturating)' WHERE ec = 'EC 1.14.99.32';
UPDATE enzyme SET history = 'EC 1.14.99.32 created 2000' WHERE ec = 'EC 1.14.99.32';
UPDATE enzyme SET sys_name = 'Linoleate, hydrogen-donor:oxygen oxidoreductase (Delta(12)-unsaturating)' WHERE ec = 'EC 1.14.99.33';
UPDATE enzyme SET history = 'EC 1.14.99.33 created 2000' WHERE ec = 'EC 1.14.99.33';
UPDATE enzyme SET sys_name = '7-O-methylluteone,NADPH:O(2) oxidoreductase' WHERE ec = 'EC 1.14.99.34';
UPDATE enzyme SET history = 'EC 1.14.99.34 created 2000' WHERE ec = 'EC 1.14.99.34';
UPDATE enzyme SET sys_name = 'Thiophene-2-carbonyl-CoA, hydrogen-donor:oxygen oxidoreductase' WHERE ec = 'EC 1.14.99.35';
UPDATE enzyme SET history = 'EC 1.14.99.35 created 2000' WHERE ec = 'EC 1.14.99.35';
UPDATE enzyme SET sys_name = 'Beta-carotene:oxygen 15,15-oxidoreductase (bond-cleaving)' WHERE ec = 'EC 1.14.99.36';
UPDATE enzyme SET history = 'EC 1.14.99.36 created 1972 as EC 1.13.11.21, transferred 2001 to EC 1.14.99.36' WHERE ec = 'EC 1.14.99.36';
UPDATE enzyme SET sys_name = 'Taxa-4,11-diene,hydrogen-donor:oxygen oxidoreductase (5-alpha-hydroxylating)' WHERE ec = 'EC 1.14.99.37';
UPDATE enzyme SET history = 'EC 1.14.99.37 created 2002' WHERE ec = 'EC 1.14.99.37';
UPDATE enzyme SET sys_name = 'Cholesterol,hydrogen-donor:oxygen oxidoreductase (25-hydroxylating)' WHERE ec = 'EC 1.14.99.38';
UPDATE enzyme SET history = 'EC 1.14.99.38 created 2005' WHERE ec = 'EC 1.14.99.38';
UPDATE enzyme SET sys_name = 'Ammonia,hydrogen-donor:oxygen oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.14.99.39';
UPDATE enzyme SET history = 'EC 1.14.99.39 created 2010' WHERE ec = 'EC 1.14.99.39';
UPDATE enzyme SET sys_name = 'FMNH(2) oxidoreductase (5,6-dimethylbenzimidazole forming)' WHERE ec = 'EC 1.14.99.40';
UPDATE enzyme SET history = 'EC 1.14.99.40 created 2010' WHERE ec = 'EC 1.14.99.40';
UPDATE enzyme SET history = 'EC 1.14.99.41 created 2010, deleted 2013' WHERE ec = 'EC 1.14.99.41';
UPDATE enzyme SET sys_name = 'Zeaxanthin:oxygen oxidoreductase (7,8-cleaving)' WHERE ec = 'EC 1.14.99.42';
UPDATE enzyme SET history = 'EC 1.14.99.42 created 2011' WHERE ec = 'EC 1.14.99.42';
UPDATE enzyme SET sys_name = 'Beta-amyrin,AH(2):oxygen oxidoreductase (24-hydroxylating)' WHERE ec = 'EC 1.14.99.43';
UPDATE enzyme SET history = 'EC 1.14.99.43 created 2011' WHERE ec = 'EC 1.14.99.43';
UPDATE enzyme SET sys_name = '4,4-diapolycopene,AH(2):oxygen oxidoreductase (4,4-hydroxylating)' WHERE ec = 'EC 1.14.99.44';
UPDATE enzyme SET history = 'EC 1.14.99.44 created 2011' WHERE ec = 'EC 1.14.99.44';
UPDATE enzyme SET sys_name = 'Alpha-carotene:oxygen oxidoreductase (3-hydroxylating)' WHERE ec = 'EC 1.14.99.45';
UPDATE enzyme SET history = 'EC 1.14.99.45 created 2011' WHERE ec = 'EC 1.14.99.45';
UPDATE enzyme SET sys_name = 'Uracil,FMNH(2):oxygen oxidoreductase (uracil hydroxylating, ring-opening)' WHERE ec = 'EC 1.14.99.46';
UPDATE enzyme SET history = 'EC 1.14.99.46 created 2012' WHERE ec = 'EC 1.14.99.46';
UPDATE enzyme SET sys_name = '(+)-larreatricin:oxygen 3-hydroxylase' WHERE ec = 'EC 1.14.99.47';
UPDATE enzyme SET history = 'EC 1.14.99.47 created 2012' WHERE ec = 'EC 1.14.99.47';
UPDATE enzyme SET sys_name = 'Protoheme,hydrogen-donor:oxygen oxidoreductase (delta/beta-methene-oxidizing, hydroxylating)' WHERE ec = 'EC 1.14.99.48';
UPDATE enzyme SET history = 'EC 1.14.99.48 created 2013' WHERE ec = 'EC 1.14.99.48';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.99.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.99.n2';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.99.n3';
UPDATE enzyme SET sys_name = 'Zeaxanthin:oxygen oxidoreductase (9,10-cleaving)' WHERE ec = 'EC 1.14.99.n4';
UPDATE enzyme SET history = 'EC 1.14.99.n4 created 2012' WHERE ec = 'EC 1.14.99.n4';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 1.14.99.n5';
UPDATE enzyme SET sys_name = 'Superoxide:superoxide oxidoreductase' WHERE ec = 'EC 1.15.1.1';
UPDATE enzyme SET history = 'EC 1.15.1.1 created 1972' WHERE ec = 'EC 1.15.1.1';
UPDATE enzyme SET sys_name = 'Rubredoxin:superoxide oxidoreductase' WHERE ec = 'EC 1.15.1.2';
UPDATE enzyme SET history = 'EC 1.15.1.2 created 2001 as EC 1.18.96.1, transferred 2001 to EC 1.15.1.2' WHERE ec = 'EC 1.15.1.2';
UPDATE enzyme SET sys_name = 'Hg:NADP(+) oxidoreductase' WHERE ec = 'EC 1.16.1.1';
UPDATE enzyme SET history = 'EC 1.16.1.1 created 1984' WHERE ec = 'EC 1.16.1.1';
UPDATE enzyme SET sys_name = 'Transferrin(Fe(II))(2):NAD(+) oxidoreductase' WHERE ec = 'EC 1.16.1.2';
UPDATE enzyme SET history = 'EC 1.16.1.2 created 1989' WHERE ec = 'EC 1.16.1.2';
UPDATE enzyme SET sys_name = 'Cob(II)alamin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.16.1.3';
UPDATE enzyme SET history = 'EC 1.16.1.3 created 1972 as EC 1.6.99.8, transferred 2002 to EC 1.16.1.3' WHERE ec = 'EC 1.16.1.3';
UPDATE enzyme SET sys_name = 'Cob(I)alamin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.16.1.4';
UPDATE enzyme SET history = 'EC 1.16.1.4 created 1972 as EC 1.6.99.9, transferred 2002 to EC 1.16.1.4' WHERE ec = 'EC 1.16.1.4';
UPDATE enzyme SET sys_name = 'Cob(II)alamin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.16.1.5';
UPDATE enzyme SET history = 'EC 1.16.1.5 created 1989 as EC 1.6.99.11, transferred 2002 to EC 1.16.1.5' WHERE ec = 'EC 1.16.1.5';
UPDATE enzyme SET sys_name = 'Cob(I)alamin, cyanide:NADP(+) oxidoreductase' WHERE ec = 'EC 1.16.1.6';
UPDATE enzyme SET history = 'EC 1.16.1.6 created 1989 as EC 1.6.99.12, transferred 2002 to EC 1.16.1.6' WHERE ec = 'EC 1.16.1.6';
UPDATE enzyme SET sys_name = 'Fe(II)-siderophore:NAD(+) oxidoreductase' WHERE ec = 'EC 1.16.1.7';
UPDATE enzyme SET history = 'EC 1.16.1.7 created 1992 as EC 1.6.99.13, transferred 2002 to EC 1.16.1.7, modified 2011, modified 2014' WHERE ec = 'EC 1.16.1.7';
UPDATE enzyme SET sys_name = '[Methionine synthase]-methylcob(I)alamin,S-adenosylhomocysteine:NADP(+) oxidoreductase' WHERE ec = 'EC 1.16.1.8';
UPDATE enzyme SET history = 'EC 1.16.1.8 created 1999 as EC 2.1.1.135, transferred 2003 to EC 1.16.1.8' WHERE ec = 'EC 1.16.1.8';
UPDATE enzyme SET sys_name = 'Fe(II)-siderophore:NADP(+) oxidoreductase' WHERE ec = 'EC 1.16.1.9';
UPDATE enzyme SET history = 'EC 1.16.1.9 created 1992 as EC 1.6.99.13, transferred 2002 to EC 1.16.1.7, transferred 2011 to EC 1.16.1.9, modified 2012, modified 2014' WHERE ec = 'EC 1.16.1.9';
UPDATE enzyme SET sys_name = 'Fe(II)-siderophore:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.16.1.10';
UPDATE enzyme SET history = 'EC 1.16.1.10 created 2014' WHERE ec = 'EC 1.16.1.10';
UPDATE enzyme SET sys_name = 'Fe(II):oxygen oxidoreductase' WHERE ec = 'EC 1.16.3.1';
UPDATE enzyme SET history = 'EC 1.16.3.1 created 1972, modified 2011' WHERE ec = 'EC 1.16.3.1';
UPDATE enzyme SET sys_name = 'Ascorbate:ferric chelate reductase  (transmembrane)' WHERE ec = 'EC 1.16.5.1';
UPDATE enzyme SET history = 'EC 1.16.5.1 created 2011' WHERE ec = 'EC 1.16.5.1';
UPDATE enzyme SET sys_name = 'Cob(I)yrinic acid-a,c-diamide:FMN oxidoreductase' WHERE ec = 'EC 1.16.8.1';
UPDATE enzyme SET history = 'EC 1.16.8.1 created 2004' WHERE ec = 'EC 1.16.8.1';
UPDATE enzyme SET sys_name = 'Fe(II):rusticyanin oxidoreductase' WHERE ec = 'EC 1.16.9.1';
UPDATE enzyme SET history = 'EC 1.16.9.1 created 2011 as EC 1.16.98.1, transferred 2011' WHERE ec = 'EC 1.16.9.1';
UPDATE enzyme SET history = 'EC 1.16.98.1 created 2011, deleted 2011' WHERE ec = 'EC 1.16.98.1';
UPDATE enzyme SET sys_name = 'CDP-4-dehydro-3,6-dideoxy-D-glucose:NAD(P)(+) 3-oxidoreductase' WHERE ec = 'EC 1.17.1.1';
UPDATE enzyme SET history = 'EC 1.17.1.1 created 1972, modified 2005' WHERE ec = 'EC 1.17.1.1';
UPDATE enzyme SET sys_name = 'Isopentenyl-diphosphate:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.17.1.2';
UPDATE enzyme SET history = 'EC 1.17.1.2 created 2003, modified 2009' WHERE ec = 'EC 1.17.1.2';
UPDATE enzyme SET sys_name = '(2R,3S)-catechin:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.17.1.3';
UPDATE enzyme SET history = 'EC 1.17.1.3 created 2003' WHERE ec = 'EC 1.17.1.3';
UPDATE enzyme SET sys_name = 'Xanthine:NAD(+) oxidoreductase' WHERE ec = 'EC 1.17.1.4';
UPDATE enzyme SET history = 'EC 1.17.1.4 created 1972 as EC 1.2.1.37, transferred 1984 to EC 1.1.1.204, modified 1989, transferred 2004 to EC 1.17.1.4, modified 2011' WHERE ec = 'EC 1.17.1.4';
UPDATE enzyme SET sys_name = 'Nicotinate:NADP(+) 6-oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.17.1.5';
UPDATE enzyme SET history = 'EC 1.17.1.5 created 1972 as EC 1.5.1.13, transferred 2004 to EC 1.17.1.5' WHERE ec = 'EC 1.17.1.5';
UPDATE enzyme SET history = 'EC 1.17.1.6 created 2005, deleted 2006' WHERE ec = 'EC 1.17.1.6';
UPDATE enzyme SET sys_name = '3-oxo-5,6-dehydrosuberyl-CoA semialdehyde:NADP(+) oxidoreductase' WHERE ec = 'EC 1.17.1.7';
UPDATE enzyme SET history = 'EC 1.17.1.7 created 2011' WHERE ec = 'EC 1.17.1.7';
UPDATE enzyme SET sys_name = '(S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate:NADP(+) 4-oxidoreductase' WHERE ec = 'EC 1.17.1.8';
UPDATE enzyme SET history = 'EC 1.17.1.8 created 1976 as EC 1.3.1.26, transferred 2013 to EC 1.17.1.8' WHERE ec = 'EC 1.17.1.8';
UPDATE enzyme SET sys_name = 'Nicotinate:cytochrome 6-oxidoreductase (hydroxylating)' WHERE ec = 'EC 1.17.2.1';
UPDATE enzyme SET history = 'EC 1.17.2.1 created 2010' WHERE ec = 'EC 1.17.2.1';
UPDATE enzyme SET sys_name = 'Lupanine:cytochrome c-oxidoreductase (17-hydroxylating)' WHERE ec = 'EC 1.17.2.2';
UPDATE enzyme SET history = 'EC 1.17.2.2 created 2012' WHERE ec = 'EC 1.17.2.2';
UPDATE enzyme SET sys_name = '2-amino-4-hydroxypteridine:oxygen oxidoreductase (7-hydroxylating)' WHERE ec = 'EC 1.17.3.1';
UPDATE enzyme SET history = 'EC 1.17.3.1 created 1983' WHERE ec = 'EC 1.17.3.1';
UPDATE enzyme SET sys_name = 'Xanthine:oxygen oxidoreductase' WHERE ec = 'EC 1.17.3.2';
UPDATE enzyme SET history = 'EC 1.17.3.2 created 1961 as EC 1.2.3.2, transferred 1984 to EC 1.1.3.22, modified 1989, transferred 2004 to EC 1.17.3.2, modified 2011' WHERE ec = 'EC 1.17.3.2';
UPDATE enzyme SET sys_name = '6-hydroxynicotinate:O(2) oxidoreductase' WHERE ec = 'EC 1.17.3.3';
UPDATE enzyme SET history = 'EC 1.17.3.3 created 2004' WHERE ec = 'EC 1.17.3.3';
UPDATE enzyme SET sys_name = '2-deoxyribonucleoside-diphosphate:thioredoxin-disulfide 2-oxidoreductase' WHERE ec = 'EC 1.17.4.1';
UPDATE enzyme SET history = 'EC 1.17.4.1 created 1972' WHERE ec = 'EC 1.17.4.1';
UPDATE enzyme SET sys_name = '2-deoxyribonucleoside-triphosphate:thioredoxin-disulfide 2-oxidoreductase' WHERE ec = 'EC 1.17.4.2';
UPDATE enzyme SET history = 'EC 1.17.4.2 created 1972' WHERE ec = 'EC 1.17.4.2';
UPDATE enzyme SET history = 'EC 1.17.4.3 created 2003, deleted 2009' WHERE ec = 'EC 1.17.4.3';
UPDATE enzyme SET sys_name = 'Phenylacetyl-CoA:quinone oxidoreductase' WHERE ec = 'EC 1.17.5.1';
UPDATE enzyme SET history = 'EC 1.17.5.1 created 2004' WHERE ec = 'EC 1.17.5.1';
UPDATE enzyme SET sys_name = 'Caffeine:ubiquinone oxidoreductase' WHERE ec = 'EC 1.17.5.2';
UPDATE enzyme SET history = 'EC 1.17.5.2 created 2010' WHERE ec = 'EC 1.17.5.2';
UPDATE enzyme SET sys_name = '(E)-4-hydroxy-3-methylbut-2-en-1-yl-diphosphate:oxidized ferredoxin oxidoreductase' WHERE ec = 'EC 1.17.7.1';
UPDATE enzyme SET history = 'EC 1.17.7.1 created 2003 as EC 1.17.4.3, transferred 2009 to EC 1.17.7.1' WHERE ec = 'EC 1.17.7.1';
UPDATE enzyme SET sys_name = '7(1)-hydroxychlorophyllide a:ferredoxin oxidoreductase' WHERE ec = 'EC 1.17.7.2';
UPDATE enzyme SET history = 'EC 1.17.7.2 created 2011' WHERE ec = 'EC 1.17.7.2';
UPDATE enzyme SET sys_name = '4-methylphenol:acceptor oxidoreductase (methyl-hydroxylating)' WHERE ec = 'EC 1.17.99.1';
UPDATE enzyme SET history = 'EC 1.17.99.1 created 1983, modified 2001, modified 2011' WHERE ec = 'EC 1.17.99.1';
UPDATE enzyme SET sys_name = 'Ethylbenzene:acceptor oxidoreductase' WHERE ec = 'EC 1.17.99.2';
UPDATE enzyme SET history = 'EC 1.17.99.2 created 2001' WHERE ec = 'EC 1.17.99.2';
UPDATE enzyme SET sys_name = '(25R)-3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestan-26-oyl-CoA:acceptor 24-oxidoreductase (24R-hydroxylating)' WHERE ec = 'EC 1.17.99.3';
UPDATE enzyme SET history = 'EC 1.17.99.3 created 2005' WHERE ec = 'EC 1.17.99.3';
UPDATE enzyme SET sys_name = 'Uracil:acceptor oxidoreductase' WHERE ec = 'EC 1.17.99.4';
UPDATE enzyme SET history = 'EC 1.1.99.19 created 1961 as EC 1.2.99.1, transferred 1984 to EC 1.1.99.19, transferred 2006 to EC 1.17.99.4' WHERE ec = 'EC 1.17.99.4';
UPDATE enzyme SET sys_name = 'Deoxycholate:FAD oxidoreductase (7-alpha-dehydroxylating)' WHERE ec = 'EC 1.17.99.5';
UPDATE enzyme SET history = 'EC 1.17.99.5, created 2005 as EC 1.17.1.6, transferred 2006 to EC 1.17.99.5' WHERE ec = 'EC 1.17.99.5';
UPDATE enzyme SET sys_name = 'Rubredoxin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.18.1.1';
UPDATE enzyme SET history = 'EC 1.18.1.1 created 1972 as EC 1.6.7.2, transferred 1978 to EC 1.18.1.1, modified 2001' WHERE ec = 'EC 1.18.1.1';
UPDATE enzyme SET sys_name = 'Ferredoxin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.18.1.2';
UPDATE enzyme SET history = 'EC 1.18.1.2 created 1965 as EC 1.6.99.4, transferred 1972 as EC 1.6.7.1, transferred 1978 to EC 1.18.1.2, modified 2012' WHERE ec = 'EC 1.18.1.2';
UPDATE enzyme SET sys_name = 'Ferredoxin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.18.1.3';
UPDATE enzyme SET history = 'EC 1.18.1.3 created 1976 as EC 1.6.7.3, transferred 1978 to EC 1.18.1.3, modified 2011' WHERE ec = 'EC 1.18.1.3';
UPDATE enzyme SET sys_name = 'Rubredoxin:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.18.1.4';
UPDATE enzyme SET history = 'EC 1.18.1.4 created 1984, modified 2001' WHERE ec = 'EC 1.18.1.4';
UPDATE enzyme SET sys_name = 'Putidaredoxin:NAD(+) oxidoreductase' WHERE ec = 'EC 1.18.1.5';
UPDATE enzyme SET history = 'EC 1.18.1.5 created 2012' WHERE ec = 'EC 1.18.1.5';
UPDATE enzyme SET sys_name = 'Adrenodoxin:NADP(+) oxidoreductase' WHERE ec = 'EC 1.18.1.6';
UPDATE enzyme SET history = 'EC 1.18.1.6 created 1965 as EC 1.6.99.4, transferred 1972 as EC 1.6.7.1, transferred 1978 to EC 1.18.1.2, part transferred 2012 to 1.18.1.6' WHERE ec = 'EC 1.18.1.6';
UPDATE enzyme SET sys_name = 'Ferredoxin:NAD(P)(+) oxidoreductase' WHERE ec = 'EC 1.18.1.7';
UPDATE enzyme SET history = 'EC 1.18.1.7 created 2013' WHERE ec = 'EC 1.18.1.7';
UPDATE enzyme SET history = 'EC 1.18.2.1 created 1978, deleted 1984' WHERE ec = 'EC 1.18.2.1';
UPDATE enzyme SET history = 'EC 1.18.3.1 created 1978, deleted 1984' WHERE ec = 'EC 1.18.3.1';
UPDATE enzyme SET sys_name = 'Reduced ferredoxin:dinitrogen oxidoreductase (ATP-hydrolyzing)' WHERE ec = 'EC 1.18.6.1';
UPDATE enzyme SET history = 'EC 1.18.6.1 created 1978 as EC 1.18.2.1, transferred 1984 to EC 1.18.6.1, modified 2005' WHERE ec = 'EC 1.18.6.1';
UPDATE enzyme SET history = 'EC 1.18.96.1 created 2001, deleted 2001' WHERE ec = 'EC 1.18.96.1';
UPDATE enzyme SET history = 'EC 1.18.99.1 created 1961 as EC 1.98.1.1, transferred 1965 to EC 1.12.1.1, transferred 1972 to EC 1.12.7.1, transferred 1978 to EC 1.18.3.1, transferred 1984 to EC 1.18.99.1, deleted 2002' WHERE ec = 'EC 1.18.99.1';
UPDATE enzyme SET sys_name = 'Reduced flavodoxin:dinitrogen oxidoreductase (ATP-hydrolyzing)' WHERE ec = 'EC 1.19.6.1';
UPDATE enzyme SET history = 'EC 1.19.6.1 created 1984' WHERE ec = 'EC 1.19.6.1';
UPDATE enzyme SET sys_name = 'Phosphonate:NAD(+) oxidoreductase' WHERE ec = 'EC 1.20.1.1';
UPDATE enzyme SET history = 'EC 1.20.1.1 created 2001' WHERE ec = 'EC 1.20.1.1';
UPDATE enzyme SET sys_name = 'Arsenite:cytochrome c oxidoreductase' WHERE ec = 'EC 1.20.2.1';
UPDATE enzyme SET history = 'EC 1.20.2.1 created 2011' WHERE ec = 'EC 1.20.2.1';
UPDATE enzyme SET sys_name = 'Glutharedoxin:arsenate oxidoreductase' WHERE ec = 'EC 1.20.4.1';
UPDATE enzyme SET history = 'EC 1.20.4.1 created 2000 as EC 1.97.1.5, transferred 2001 to EC 1.20.4.1' WHERE ec = 'EC 1.20.4.1';
UPDATE enzyme SET sys_name = 'Gluthathione:methylarsonate oxidoreductase' WHERE ec = 'EC 1.20.4.2';
UPDATE enzyme SET history = 'EC 1.20.4.2 created 2000 as EC 1.97.1.7, transfered 2001 to EC 1.20.4.2, modified 2003' WHERE ec = 'EC 1.20.4.2';
UPDATE enzyme SET sys_name = 'Arseno-mycothiol:mycoredoxin oxidoreductase' WHERE ec = 'EC 1.20.4.3';
UPDATE enzyme SET history = 'EC 1.20.4.3 created 2010' WHERE ec = 'EC 1.20.4.3';
UPDATE enzyme SET sys_name = 'Arsenite:azurin oxidoreductase' WHERE ec = 'EC 1.20.9.1';
UPDATE enzyme SET history = 'EC 1.20.9.1 created 2001 as EC 1.20.98.1, transferred 2011 to EC 1.20.9.1' WHERE ec = 'EC 1.20.9.1';
UPDATE enzyme SET history = 'EC 1.20.98.1 created 2001, deleted 2011' WHERE ec = 'EC 1.20.98.1';
UPDATE enzyme SET sys_name = 'Arsenate:acceptor oxidoreductase' WHERE ec = 'EC 1.20.99.1';
UPDATE enzyme SET history = 'EC 1.20.99.1 created 2000 as EC 1.97.1.6, transferred 2001 to EC 1.20.99.1' WHERE ec = 'EC 1.20.99.1';
UPDATE enzyme SET sys_name = 'N-((5S)-5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine:oxygen oxidoreductase (cyclizing)' WHERE ec = 'EC 1.21.3.1';
UPDATE enzyme SET history = 'EC 1.21.3.1 created 2002' WHERE ec = 'EC 1.21.3.1';
UPDATE enzyme SET sys_name = 'Columbamine:oxygen oxidoreductase (cyclizing)' WHERE ec = 'EC 1.21.3.2';
UPDATE enzyme SET history = 'EC 1.21.3.2 created 1989 as EC 1.1.3.26, transferred 2002 to EC 1.21.3.2' WHERE ec = 'EC 1.21.3.2';
UPDATE enzyme SET sys_name = '(S)-reticuline:oxygen oxidoreductase (methylene-bridge-forming)' WHERE ec = 'EC 1.21.3.3';
UPDATE enzyme SET history = 'EC 1.21.3.3 created 1989 as EC 1.5.3.9, transferred 2002 to EC 1.21.3.3' WHERE ec = 'EC 1.21.3.3';
UPDATE enzyme SET sys_name = 'Sulochrin:oxygen oxidoreductase (cyclizing, (+)-specific)' WHERE ec = 'EC 1.21.3.4';
UPDATE enzyme SET history = 'EC 1.21.3.4 created 1986 as EC 1.10.3.7, transferred 2002 to EC 1.21.3.4' WHERE ec = 'EC 1.21.3.4';
UPDATE enzyme SET sys_name = 'Sulochrin:oxygen oxidoreductase (cyclizing, (-)-specific)' WHERE ec = 'EC 1.21.3.5';
UPDATE enzyme SET history = 'EC 1.21.3.5 created 1986 as EC 1.10.3.8, transferred 2002 to EC 1.21.3.5' WHERE ec = 'EC 1.21.3.5';
UPDATE enzyme SET sys_name = '2,4,4,6-tetrahydroxychalcone 4-O-beta-glucoside:oxygen oxidoreductase' WHERE ec = 'EC 1.21.3.6';
UPDATE enzyme SET history = 'EC 1.21.3.6 created 2003, modified 2012' WHERE ec = 'EC 1.21.3.6';
UPDATE enzyme SET sys_name = 'Cannabigerolate:oxygen oxidoreductase (cyclizing, Delta(9)-tetrahydrocannabinolate-forming)' WHERE ec = 'EC 1.21.3.7';
UPDATE enzyme SET history = 'EC 1.21.3.7 created 2012' WHERE ec = 'EC 1.21.3.7';
UPDATE enzyme SET sys_name = 'Cannabigerolate:oxygen oxidoreductase (cyclizing, cannabidiolate-forming)' WHERE ec = 'EC 1.21.3.8';
UPDATE enzyme SET history = 'EC 1.21.3.8 created 2012' WHERE ec = 'EC 1.21.3.8';
UPDATE enzyme SET sys_name = '2-imino-3-(7-chloroindol-3-yl)propanoate ammonia-lyase (dichlorochromopyrrolate-forming)' WHERE ec = 'EC 1.21.3.9';
UPDATE enzyme SET history = 'EC 1.21.3.9 created 2010 as 4.3.1.26, transferred 2013 to EC 1.21.3.9' WHERE ec = 'EC 1.21.3.9';
UPDATE enzyme SET sys_name = '5-aminopentanoate:lipoate oxidoreductase (cyclizing)' WHERE ec = 'EC 1.21.4.1';
UPDATE enzyme SET history = 'EC 1.21.4.1 created 1972 as EC 1.4.4.1, modified 1982 (EC 1.4.1.6 created 1961, incorporated 1982), transferred 2003 to EC 1.21.4.1' WHERE ec = 'EC 1.21.4.1';
UPDATE enzyme SET sys_name = 'Acetyl-phosphate ammonia:thioredoxin disulfide oxidoreductase (glycine-forming)' WHERE ec = 'EC 1.21.4.2';
UPDATE enzyme SET history = 'EC 1.21.4.2 created 2003' WHERE ec = 'EC 1.21.4.2';
UPDATE enzyme SET sys_name = 'Acetyl-phosphate methylamine:thioredoxin disulfide oxidoreductase (N-methylglycine-forming)' WHERE ec = 'EC 1.21.4.3';
UPDATE enzyme SET history = 'EC 1.21.4.3 created 2003' WHERE ec = 'EC 1.21.4.3';
UPDATE enzyme SET sys_name = 'Acetyl-phosphate trimethylamine:thioredoxin disulfide oxidoreductase (betaine-forming)' WHERE ec = 'EC 1.21.4.4';
UPDATE enzyme SET history = 'EC 1.21.4.4 created 2003' WHERE ec = 'EC 1.21.4.4';
UPDATE enzyme SET sys_name = 'Beta-cyclopiazonate:acceptor oxidoreductase (cyclizing)' WHERE ec = 'EC 1.21.99.1';
UPDATE enzyme SET history = 'EC 1.21.99.1 created 1976 as EC 1.3.99.9, transferred 2002 to EC 1.21.99.1' WHERE ec = 'EC 1.21.99.1';
UPDATE enzyme SET sys_name = 'Dehypoxanthine futalosine:S-adenosyl-L-methionine oxidoreductase (cyclizing)' WHERE ec = 'EC 1.21.99.2';
UPDATE enzyme SET history = 'EC 1.21.99.2 created 2014' WHERE ec = 'EC 1.21.99.2';
UPDATE enzyme SET sys_name = 'NADP(+):L-tyrosine oxidoreductase (iodinating)' WHERE ec = 'EC 1.22.1.1';
UPDATE enzyme SET history = 'EC 1.22.1.1 created 2010' WHERE ec = 'EC 1.22.1.1';
UPDATE enzyme SET sys_name = '(+)-lariciresinol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.23.1.1';
UPDATE enzyme SET history = 'EC 1.23.1.1 created 2013' WHERE ec = 'EC 1.23.1.1';
UPDATE enzyme SET sys_name = '(-)-secoisolariciresinol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.23.1.2';
UPDATE enzyme SET history = 'EC 1.23.1.2 created 2013' WHERE ec = 'EC 1.23.1.2';
UPDATE enzyme SET sys_name = '(-)-lariciresinol:NADP(+) oxidoreductase' WHERE ec = 'EC 1.23.1.3';
UPDATE enzyme SET history = 'EC 1.23.1.3 created 2013' WHERE ec = 'EC 1.23.1.3';
UPDATE enzyme SET sys_name = '(+)-secoisolariciresinol:NADPH(+) oxidoreductase' WHERE ec = 'EC 1.23.1.4';
UPDATE enzyme SET history = 'EC 1.23.1.4 created 2013' WHERE ec = 'EC 1.23.1.4';
UPDATE enzyme SET sys_name = 'Chlorite:acceptor oxidoreductase' WHERE ec = 'EC 1.97.1.1';
UPDATE enzyme SET history = 'EC 1.97.1.1 created 1978' WHERE ec = 'EC 1.97.1.1';
UPDATE enzyme SET sys_name = '1,2,3,5-tetrahydroxybenzene:1,2,3-trihydroxybenzene hydroxytransferase' WHERE ec = 'EC 1.97.1.2';
UPDATE enzyme SET history = 'EC 1.97.1.2 created 1992' WHERE ec = 'EC 1.97.1.2';
UPDATE enzyme SET history = 'EC 1.97.1.3 created 1992, deleted 2013' WHERE ec = 'EC 1.97.1.3';
UPDATE enzyme SET sys_name = '[Formate C-acetyltransferase]-glycine dihydroflavodoxin:S-adenosyl-L-methionine oxidoreductase (S-adenosyl-L-methionine cleaving)' WHERE ec = 'EC 1.97.1.4';
UPDATE enzyme SET history = 'EC 1.97.1.4 created 1999, modified 2004' WHERE ec = 'EC 1.97.1.4';
UPDATE enzyme SET history = 'EC 1.97.1.5 created 2000 deleted 2001' WHERE ec = 'EC 1.97.1.5';
UPDATE enzyme SET history = 'EC 1.97.1.6 created 2000 deleted 2001' WHERE ec = 'EC 1.97.1.6';
UPDATE enzyme SET history = 'EC 1.97.1.7 created 2000, deleted 2001' WHERE ec = 'EC 1.97.1.7';
UPDATE enzyme SET sys_name = 'Acceptor:trichloroethene oxidoreductase (chlorinating)' WHERE ec = 'EC 1.97.1.8';
UPDATE enzyme SET history = 'EC 1.97.1.8 created 2001' WHERE ec = 'EC 1.97.1.8';
UPDATE enzyme SET sys_name = 'Selenite:reduced acceptor oxidoreductase' WHERE ec = 'EC 1.97.1.9';
UPDATE enzyme SET history = 'EC 1.97.1.9 created 2003' WHERE ec = 'EC 1.97.1.9';
UPDATE enzyme SET sys_name = 'Acceptor:3,5,3-triiodo-L-thyronine oxidoreductase (iodinating)' WHERE ec = 'EC 1.97.1.10';
UPDATE enzyme SET history = 'EC 1.97.1.10 created 1984 as EC 3.8.1.4, transferred 2003 to EC 1.97.1.10' WHERE ec = 'EC 1.97.1.10';
UPDATE enzyme SET sys_name = 'Acceptor:3,3,5-triiodo-L-thyronine oxidoreductase (iodinating)' WHERE ec = 'EC 1.97.1.11';
UPDATE enzyme SET history = 'EC 1.97.1.11 created 2003' WHERE ec = 'EC 1.97.1.11';
UPDATE enzyme SET sys_name = 'Plastocyanin:ferredoxin oxidoreductase (light-dependent)' WHERE ec = 'EC 1.97.1.12';
UPDATE enzyme SET history = 'EC 1.97.1.12 created 2011' WHERE ec = 'EC 1.97.1.12';
UPDATE enzyme SET history = 'EC 1.98.1.1 created 1961, deleted 1965' WHERE ec = 'EC 1.98.1.1';
UPDATE enzyme SET history = 'EC 1.99.1.1 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.1';
UPDATE enzyme SET history = 'EC 1.99.1.2 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.2';
UPDATE enzyme SET history = 'EC 1.99.1.3 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.3';
UPDATE enzyme SET history = 'EC 1.99.1.4 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.4';
UPDATE enzyme SET history = 'EC 1.99.1.5 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.5';
UPDATE enzyme SET history = 'EC 1.99.1.6 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.6';
UPDATE enzyme SET history = 'EC 1.99.1.7 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.7';
UPDATE enzyme SET history = 'EC 1.99.1.8 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.8';
UPDATE enzyme SET history = 'EC 1.99.1.9 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.9';
UPDATE enzyme SET history = 'EC 1.99.1.10 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.10';
UPDATE enzyme SET history = 'EC 1.99.1.11 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.11';
UPDATE enzyme SET history = 'EC 1.99.1.12 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.12';
UPDATE enzyme SET history = 'EC 1.99.1.13 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.13';
UPDATE enzyme SET history = 'EC 1.99.1.14 created 1961, deleted 1965' WHERE ec = 'EC 1.99.1.14';
UPDATE enzyme SET history = 'EC 1.99.2.1 created 1961, deleted 1965' WHERE ec = 'EC 1.99.2.1';
UPDATE enzyme SET history = 'EC 1.99.2.2 created 1961, deleted 1965' WHERE ec = 'EC 1.99.2.2';
UPDATE enzyme SET history = 'EC 1.99.2.3 created 1961, deleted 1965' WHERE ec = 'EC 1.99.2.3';
UPDATE enzyme SET history = 'EC 1.99.2.4 created 1961, deleted 1965' WHERE ec = 'EC 1.99.2.4';
UPDATE enzyme SET history = 'EC 1.99.2.5 created 1961, deleted 1965' WHERE ec = 'EC 1.99.2.5';
UPDATE enzyme SET history = 'EC 1.99.2.6 created 1961, deleted 1965' WHERE ec = 'EC 1.99.2.6';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:nicotinamide N-methyltransferase' WHERE ec = 'EC 2.1.1.1';
UPDATE enzyme SET history = 'EC 2.1.1.1 created 1961' WHERE ec = 'EC 2.1.1.1';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:N-guanidinoacetate methyltransferase' WHERE ec = 'EC 2.1.1.2';
UPDATE enzyme SET history = 'EC 2.1.1.2 created 1961' WHERE ec = 'EC 2.1.1.2';
UPDATE enzyme SET sys_name = 'Dimethylsulfonioacetate:L-homocysteine S-methyltransferase' WHERE ec = 'EC 2.1.1.3';
UPDATE enzyme SET history = 'EC 2.1.1.3 created 1961' WHERE ec = 'EC 2.1.1.3';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:N-acetylserotonin O-methyltransferase' WHERE ec = 'EC 2.1.1.4';
UPDATE enzyme SET history = 'EC 2.1.1.4 created 1961' WHERE ec = 'EC 2.1.1.4';
UPDATE enzyme SET sys_name = 'Trimethylammonioacetate:L-homocysteine S-methyltransferase' WHERE ec = 'EC 2.1.1.5';
UPDATE enzyme SET history = 'EC 2.1.1.5 created 1961' WHERE ec = 'EC 2.1.1.5';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:catechol O-methyltransferase' WHERE ec = 'EC 2.1.1.6';
UPDATE enzyme SET history = 'EC 2.1.1.6 created 1965' WHERE ec = 'EC 2.1.1.6';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:nicotinate N-methyltransferase' WHERE ec = 'EC 2.1.1.7';
UPDATE enzyme SET history = 'EC 2.1.1.7 created 1965' WHERE ec = 'EC 2.1.1.7';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:histamine N-tele-methyltransferase' WHERE ec = 'EC 2.1.1.8';
UPDATE enzyme SET history = 'EC 2.1.1.8 created 1965' WHERE ec = 'EC 2.1.1.8';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:thiol S-methyltransferase' WHERE ec = 'EC 2.1.1.9';
UPDATE enzyme SET history = 'EC 2.1.1.9 created 1965' WHERE ec = 'EC 2.1.1.9';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:L-homocysteine S-methyltransferase' WHERE ec = 'EC 2.1.1.10';
UPDATE enzyme SET history = 'EC 2.1.1.10 created 1965, modified 2010' WHERE ec = 'EC 2.1.1.10';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:magnesium-protoporphyrin-IX O-methyltransferase' WHERE ec = 'EC 2.1.1.11';
UPDATE enzyme SET history = 'EC 2.1.1.11 created 1965, modified 2003' WHERE ec = 'EC 2.1.1.11';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:L-methionine S-methyltransferase' WHERE ec = 'EC 2.1.1.12';
UPDATE enzyme SET history = 'EC 2.1.1.12 created 1972' WHERE ec = 'EC 2.1.1.12';
UPDATE enzyme SET sys_name = '5-methyltetrahydrofolate:L-homocysteine S-methyltransferase' WHERE ec = 'EC 2.1.1.13';
UPDATE enzyme SET history = 'EC 2.1.1.13 created 1972, modified 2003' WHERE ec = 'EC 2.1.1.13';
UPDATE enzyme SET sys_name = '5-methyltetrahydropteroyltri-L-glutamate:L-homocysteine S-methyltransferase' WHERE ec = 'EC 2.1.1.14';
UPDATE enzyme SET history = 'EC 2.1.1.14 created 1972, modified 2003' WHERE ec = 'EC 2.1.1.14';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:fatty-acid O-methyltransferase' WHERE ec = 'EC 2.1.1.15';
UPDATE enzyme SET history = 'EC 2.1.1.15 created 1972' WHERE ec = 'EC 2.1.1.15';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:unsaturated-phospholipid methyltransferase (methenylating)' WHERE ec = 'EC 2.1.1.16';
UPDATE enzyme SET history = 'EC 2.1.1.16 created 1972, modified 1986' WHERE ec = 'EC 2.1.1.16';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:phosphatidylethanolamine N-methyltransferase' WHERE ec = 'EC 2.1.1.17';
UPDATE enzyme SET history = 'EC 2.1.1.17 created 1972' WHERE ec = 'EC 2.1.1.17';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(1-&gt;4)-alpha-D-glucan 6-O-methyltransferase' WHERE ec = 'EC 2.1.1.18';
UPDATE enzyme SET history = 'EC 2.1.1.18 created 1972' WHERE ec = 'EC 2.1.1.18';
UPDATE enzyme SET sys_name = 'Trimethylsulfonium:tetrahydrofolate N-methyltransferase' WHERE ec = 'EC 2.1.1.19';
UPDATE enzyme SET history = 'EC 2.1.1.19 created 1972' WHERE ec = 'EC 2.1.1.19';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:glycine N-methyltransferase' WHERE ec = 'EC 2.1.1.20';
UPDATE enzyme SET history = 'EC 2.1.1.20 created 1972, modified 2005' WHERE ec = 'EC 2.1.1.20';
UPDATE enzyme SET sys_name = 'Methylamine:L-glutamate N-methyltransferase' WHERE ec = 'EC 2.1.1.21';
UPDATE enzyme SET history = 'EC 2.1.1.21 created 1972' WHERE ec = 'EC 2.1.1.21';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:carnosine N-methyltransferase' WHERE ec = 'EC 2.1.1.22';
UPDATE enzyme SET history = 'EC 2.1.1.22 created 1972' WHERE ec = 'EC 2.1.1.22';
UPDATE enzyme SET history = 'EC 2.1.1.23 created 1972, modified 1976, modified 1983, deleted 1999' WHERE ec = 'EC 2.1.1.23';
UPDATE enzyme SET history = 'EC 2.1.1.24 created 1972, modified 1983, modified 1989, deleted 1992' WHERE ec = 'EC 2.1.1.24';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:phenol O-methyltransferase' WHERE ec = 'EC 2.1.1.25';
UPDATE enzyme SET history = 'EC 2.1.1.25 created 1972' WHERE ec = 'EC 2.1.1.25';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:2-iodophenol O-methyltransferase' WHERE ec = 'EC 2.1.1.26';
UPDATE enzyme SET history = 'EC 2.1.1.26 created 1972' WHERE ec = 'EC 2.1.1.26';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tyramine N-methyltransferase' WHERE ec = 'EC 2.1.1.27';
UPDATE enzyme SET history = 'EC 2.1.1.27 created 1972' WHERE ec = 'EC 2.1.1.27';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:phenylethanolamine N-methyltransferase' WHERE ec = 'EC 2.1.1.28';
UPDATE enzyme SET history = 'EC 2.1.1.28 created 1972' WHERE ec = 'EC 2.1.1.28';
UPDATE enzyme SET history = 'EC 2.1.1.29 created 1972, deleted 2011' WHERE ec = 'EC 2.1.1.29';
UPDATE enzyme SET history = 'EC 2.1.1.30 created 1972, deleted 1981' WHERE ec = 'EC 2.1.1.30';
UPDATE enzyme SET history = 'EC 2.1.1.31 created 1972, deleted 2011' WHERE ec = 'EC 2.1.1.31';
UPDATE enzyme SET history = 'EC 2.1.1.32 created 1972, deleted 2011' WHERE ec = 'EC 2.1.1.32';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (guanine-N(7))-methyltransferase' WHERE ec = 'EC 2.1.1.33';
UPDATE enzyme SET history = 'EC 2.1.1.33 created 1972, modified 2011' WHERE ec = 'EC 2.1.1.33';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (guanosine(18)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.34';
UPDATE enzyme SET history = 'EC 2.1.1.34 created 1972, modified 2005, modified 2011' WHERE ec = 'EC 2.1.1.34';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (uracil(54)-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.35';
UPDATE enzyme SET history = 'EC 2.1.1.35 created 1972, modified 2011' WHERE ec = 'EC 2.1.1.35';
UPDATE enzyme SET history = 'EC 2.1.1.36 created 1972, deleted 2011' WHERE ec = 'EC 2.1.1.36';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:DNA (cytosine-5-)-methyltransferase' WHERE ec = 'EC 2.1.1.37';
UPDATE enzyme SET history = 'EC 2.1.1.37 created 1972, modified 2003' WHERE ec = 'EC 2.1.1.37';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:O-demethylpuromycin O-methyltransferase' WHERE ec = 'EC 2.1.1.38';
UPDATE enzyme SET history = 'EC 2.1.1.38 created 1972' WHERE ec = 'EC 2.1.1.38';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:1D-myo-inositol 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.39';
UPDATE enzyme SET history = 'EC 2.1.1.39 created 1972, modified 2002' WHERE ec = 'EC 2.1.1.39';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:1D-myo-inositol 1-O-methyltransferase' WHERE ec = 'EC 2.1.1.40';
UPDATE enzyme SET history = 'EC 2.1.1.40 created 1972, modified 2002' WHERE ec = 'EC 2.1.1.40';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:zymosterol 24-C-methyltransferase' WHERE ec = 'EC 2.1.1.41';
UPDATE enzyme SET history = 'EC 2.1.1.41 created 1972, modfied 2001' WHERE ec = 'EC 2.1.1.41';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3-hydroxyflavone 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.42';
UPDATE enzyme SET history = 'EC 2.1.1.42 created 1976, modified 2011' WHERE ec = 'EC 2.1.1.42';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:histone-L-lysine N(6)-methyltransferase' WHERE ec = 'EC 2.1.1.43';
UPDATE enzyme SET history = 'EC 2.1.1.43 created 1976, modified 1982, modified 1983' WHERE ec = 'EC 2.1.1.43';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:L-histidine N(alpha)-methyltransferase (hercynine-forming)' WHERE ec = 'EC 2.1.1.44';
UPDATE enzyme SET history = 'EC 2.1.1.44 created 1976, modified 2013' WHERE ec = 'EC 2.1.1.44';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate:dUMP C-methyltransferase' WHERE ec = 'EC 2.1.1.45';
UPDATE enzyme SET history = 'EC 2.1.1.45 created 1976' WHERE ec = 'EC 2.1.1.45';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:4-hydroxyisoflavone 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.46';
UPDATE enzyme SET history = 'EC 2.1.1.46 created 1976, modified 2011' WHERE ec = 'EC 2.1.1.46';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(indol-3-yl)pyruvate C-methyltransferase' WHERE ec = 'EC 2.1.1.47';
UPDATE enzyme SET history = 'EC 2.1.1.47 created 1976' WHERE ec = 'EC 2.1.1.47';
UPDATE enzyme SET history = 'EC 2.1.1.48 created 1976, deleted 2010' WHERE ec = 'EC 2.1.1.48';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:amine N-methyltransferase' WHERE ec = 'EC 2.1.1.49';
UPDATE enzyme SET history = 'EC 2.1.1.49 created 1976, modified 1990 (EC 2.1.1.81 created 1989, incorporated 1990)' WHERE ec = 'EC 2.1.1.49';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:loganate 11-O-methyltransferase' WHERE ec = 'EC 2.1.1.50';
UPDATE enzyme SET history = 'EC 2.1.1.50 created 1976' WHERE ec = 'EC 2.1.1.50';
UPDATE enzyme SET history = 'EC 2.1.1.51 created 1976, deleted 2010' WHERE ec = 'EC 2.1.1.51';
UPDATE enzyme SET history = 'EC 2.1.1.52 created 1976, deleted 2010' WHERE ec = 'EC 2.1.1.52';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:putrescine N-methyltransferase' WHERE ec = 'EC 2.1.1.53';
UPDATE enzyme SET history = 'EC 2.1.1.53 created 1976' WHERE ec = 'EC 2.1.1.53';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate:dCMP C-methyltransferase' WHERE ec = 'EC 2.1.1.54';
UPDATE enzyme SET history = 'EC 2.1.1.54 created 1978' WHERE ec = 'EC 2.1.1.54';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (adenine-N(6)-)-methyltransferase' WHERE ec = 'EC 2.1.1.55';
UPDATE enzyme SET history = 'EC 2.1.1.55 created 1981' WHERE ec = 'EC 2.1.1.55';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:mRNA (guanine-N(7)-)-methyltransferase' WHERE ec = 'EC 2.1.1.56';
UPDATE enzyme SET history = 'EC 2.1.1.56 created 1981' WHERE ec = 'EC 2.1.1.56';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:5-(N(7)-methyl 5-triphosphoguanosine)-(purine-ribonucleotide)-(mRNA) 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.57';
UPDATE enzyme SET history = 'EC 2.1.1.57 created 1981 (EC 2.1.1.58 created 1981, incorporated 1984), modified 2014' WHERE ec = 'EC 2.1.1.57';
UPDATE enzyme SET history = 'EC 2.1.1.58 created 1981, deleted 1984' WHERE ec = 'EC 2.1.1.58';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[cytochrome c]-L-lysine N(6)-methyltransferase' WHERE ec = 'EC 2.1.1.59';
UPDATE enzyme SET history = 'EC 2.1.1.59 created 1982, modified 1983' WHERE ec = 'EC 2.1.1.59';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:calmodulin-L-lysine N(6)-methyltransferase' WHERE ec = 'EC 2.1.1.60';
UPDATE enzyme SET history = 'EC 2.1.1.60 created 1982, modified 1983' WHERE ec = 'EC 2.1.1.60';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (5-methylaminomethyl-2-thio-uridylate)-methyltransferase' WHERE ec = 'EC 2.1.1.61';
UPDATE enzyme SET history = 'EC 2.1.1.61 created 1982, modified 2012' WHERE ec = 'EC 2.1.1.61';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:mRNA (2-O-methyladenosine-N(6)-)-methyltransferase' WHERE ec = 'EC 2.1.1.62';
UPDATE enzyme SET history = 'EC 2.1.1.62 created 1982' WHERE ec = 'EC 2.1.1.62';
UPDATE enzyme SET sys_name = 'DNA-6-O-methylguanine:[protein]-L-cysteine S-methyltransferase' WHERE ec = 'EC 2.1.1.63';
UPDATE enzyme SET history = 'EC 2.1.1.63 created 1982, modified 1983, modified 1999, modified 2003' WHERE ec = 'EC 2.1.1.63';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3-hydroxy-2-methoxy-5-methyl-6-(all-trans-polyprenyl)-1,4-benzoquinol 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.64';
UPDATE enzyme SET history = 'EC 2.1.1.64 created 1982, modified 2011' WHERE ec = 'EC 2.1.1.64';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:licodione 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.65';
UPDATE enzyme SET history = 'EC 2.1.1.65 created 1983' WHERE ec = 'EC 2.1.1.65';
UPDATE enzyme SET history = 'EC 2.1.1.66 created 1984, deleted 2013' WHERE ec = 'EC 2.1.1.66';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:thiopurine S-methyltransferase' WHERE ec = 'EC 2.1.1.67';
UPDATE enzyme SET history = 'EC 2.1.1.67 created 1984' WHERE ec = 'EC 2.1.1.67';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3,4-dihydroxy-trans-cinnamate 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.68';
UPDATE enzyme SET history = 'EC 2.1.1.68 created 1984' WHERE ec = 'EC 2.1.1.68';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:5-hydroxyfurocoumarin 5-O-methyltransferase' WHERE ec = 'EC 2.1.1.69';
UPDATE enzyme SET history = 'EC 2.1.1.69 created 1984 (EC 2.1.1.92 created 1989, incorporated 2006), modified 2006' WHERE ec = 'EC 2.1.1.69';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:8-hydroxyfurocoumarin 8-O-methyltransferase' WHERE ec = 'EC 2.1.1.70';
UPDATE enzyme SET history = 'EC 2.1.1.70 created 1984, modified 2006 (EC 2.1.1.93 created 2006, incorporated 2008)' WHERE ec = 'EC 2.1.1.70';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:phosphatidyl-N-methylethanolamine N-methyltransferase' WHERE ec = 'EC 2.1.1.71';
UPDATE enzyme SET history = 'EC 2.1.1.71 created 1984' WHERE ec = 'EC 2.1.1.71';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 2.1.1.72';
UPDATE enzyme SET history = 'EC 2.1.1.72 created 1984' WHERE ec = 'EC 2.1.1.72';
UPDATE enzyme SET history = 'EC 2.1.1.73 created 1984, deleted 2003' WHERE ec = 'EC 2.1.1.73';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate:tRNA (uracil(54)-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.74';
UPDATE enzyme SET history = 'EC 2.1.1.74 created 1983, as EC 2.1.2.12 transferred 1984 to EC 2.1.1.74, modified 2011' WHERE ec = 'EC 2.1.1.74';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:apigenin 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.75';
UPDATE enzyme SET history = 'EC 2.1.1.75 created 1984' WHERE ec = 'EC 2.1.1.75';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3,5,7,3,4-pentahydroxyflavone 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.76';
UPDATE enzyme SET history = 'EC 2.1.1.76 created 1984' WHERE ec = 'EC 2.1.1.76';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:protein-L-isoaspartate O-methyltransferase' WHERE ec = 'EC 2.1.1.77';
UPDATE enzyme SET history = 'EC 2.1.1.77 created 1984, modified 1989 (EC 2.1.1.24 created 1972, modified 1983, modified 1989, part incorporated 1992)' WHERE ec = 'EC 2.1.1.77';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:isoorientin 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.78';
UPDATE enzyme SET history = 'EC 2.1.1.78 created 1986' WHERE ec = 'EC 2.1.1.78';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:unsaturated-phospholipid methyltransferase (cyclizing)' WHERE ec = 'EC 2.1.1.79';
UPDATE enzyme SET history = 'EC 2.1.1.79 created 1986' WHERE ec = 'EC 2.1.1.79';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:protein-L-glutamate O-methyltransferase' WHERE ec = 'EC 2.1.1.80';
UPDATE enzyme SET history = 'EC 2.1.1.80 created 1989 (EC 2.1.1.24 created 1972, modified 1983, modified 1989, part incorporated 1992)' WHERE ec = 'EC 2.1.1.80';
UPDATE enzyme SET history = 'EC 2.1.1.81 created 1989, deleted 1990' WHERE ec = 'EC 2.1.1.81';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:5,7,3,4-tetrahydroxy-3-methoxyflavone 7-O-methyltransferase' WHERE ec = 'EC 2.1.1.82';
UPDATE enzyme SET history = 'EC 2.1.1.82 created 1989' WHERE ec = 'EC 2.1.1.82';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:5,3,4-trihydroxy-3,7-dimethoxyflavone 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.83';
UPDATE enzyme SET history = 'EC 2.1.1.83 created 1989' WHERE ec = 'EC 2.1.1.83';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:5,6,3,4-tetrahydroxy-3,7-dimethoxyflavone 6-O-methyltransferase' WHERE ec = 'EC 2.1.1.84';
UPDATE enzyme SET history = 'EC 2.1.1.84 created 1989' WHERE ec = 'EC 2.1.1.84';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:protein-L-histidine N-tele-methyltransferase' WHERE ec = 'EC 2.1.1.85';
UPDATE enzyme SET history = 'EC 2.1.1.85 created 1989' WHERE ec = 'EC 2.1.1.85';
UPDATE enzyme SET sys_name = '5-methyl-5,6,7,8-tetrahydromethanopterin:2-mercaptoethanesulfonate 2-methyltransferase' WHERE ec = 'EC 2.1.1.86';
UPDATE enzyme SET history = 'EC 2.1.1.86 created 1989, modified 2000' WHERE ec = 'EC 2.1.1.86';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:pyridine N-methyltransferase' WHERE ec = 'EC 2.1.1.87';
UPDATE enzyme SET history = 'EC 2.1.1.87 created 1989' WHERE ec = 'EC 2.1.1.87';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3,3,4,5,7,8-hexahydroxyflavone 8-O-methyltransferase' WHERE ec = 'EC 2.1.1.88';
UPDATE enzyme SET history = 'EC 2.1.1.88 created 1989' WHERE ec = 'EC 2.1.1.88';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:5,8,13,13a-tetrahydrocolumbamine 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.89';
UPDATE enzyme SET history = 'EC 2.1.1.89 created 1989' WHERE ec = 'EC 2.1.1.89';
UPDATE enzyme SET sys_name = 'Methanol:5-hydroxybenzimidazolylcobamide Co-methyltransferase' WHERE ec = 'EC 2.1.1.90';
UPDATE enzyme SET history = 'EC 2.1.1.90 created 1989, modified 2012' WHERE ec = 'EC 2.1.1.90';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:2-methylpropanal-oxime O-methyltransferase' WHERE ec = 'EC 2.1.1.91';
UPDATE enzyme SET history = 'EC 2.1.1.91 created 1989' WHERE ec = 'EC 2.1.1.91';
UPDATE enzyme SET history = 'EC 2.1.1.92 created 1989, deleted 2006' WHERE ec = 'EC 2.1.1.92';
UPDATE enzyme SET history = 'EC 2.1.1.93 created 1989, deleted 2008' WHERE ec = 'EC 2.1.1.93';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16-hydroxytabersonine 16-O-methyltransferase' WHERE ec = 'EC 2.1.1.94';
UPDATE enzyme SET history = 'EC 2.1.1.94 created 1989, modified 2005' WHERE ec = 'EC 2.1.1.94';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:gamma-tocopherol 5-O-methyltransferase' WHERE ec = 'EC 2.1.1.95';
UPDATE enzyme SET history = 'EC 2.1.1.95 created 1989' WHERE ec = 'EC 2.1.1.95';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:dimethyl-sulfide S-methyltransferase' WHERE ec = 'EC 2.1.1.96';
UPDATE enzyme SET history = 'EC 2.1.1.96 created 1990' WHERE ec = 'EC 2.1.1.96';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3-hydroxyanthranilate 4-C-methyltransferase' WHERE ec = 'EC 2.1.1.97';
UPDATE enzyme SET history = 'EC 2.1.1.97 created 1990' WHERE ec = 'EC 2.1.1.97';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:2-(3-carboxy-3-aminopropyl)-L-histidine-[translation elongation factor 2] methyltransferase' WHERE ec = 'EC 2.1.1.98';
UPDATE enzyme SET history = 'EC 2.1.1.98 created 1990, modified 2013' WHERE ec = 'EC 2.1.1.98';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3-hydroxy-16-methoxy-2,3-dihydrotabersonine N-methyltransferase' WHERE ec = 'EC 2.1.1.99';
UPDATE enzyme SET history = 'EC 2.1.1.99 created 1990, modified 2005' WHERE ec = 'EC 2.1.1.99';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:protein-C-terminal-S-farnesyl-L-cysteine O-methyltransferase' WHERE ec = 'EC 2.1.1.100';
UPDATE enzyme SET history = 'EC 2.1.1.100 created 1992 (EC 2.1.1.24 created 1972, modified 1983, modified 1989, part incorporated 1992)' WHERE ec = 'EC 2.1.1.100';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:macrocin 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.101';
UPDATE enzyme SET history = 'EC 2.1.1.101 created 1992' WHERE ec = 'EC 2.1.1.101';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:demethylmacrocin 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.102';
UPDATE enzyme SET history = 'EC 2.1.1.102 created 1992' WHERE ec = 'EC 2.1.1.102';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:ethanolamine-phosphate N-methyltransferase' WHERE ec = 'EC 2.1.1.103';
UPDATE enzyme SET history = 'EC 2.1.1.103 created 1992' WHERE ec = 'EC 2.1.1.103';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:caffeoyl-CoA 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.104';
UPDATE enzyme SET history = 'EC 2.1.1.104 created 1992' WHERE ec = 'EC 2.1.1.104';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:N-benzoyl-4-O-hydroxyanthranilate 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.105';
UPDATE enzyme SET history = 'EC 2.1.1.105 created 1992' WHERE ec = 'EC 2.1.1.105';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:L-tryptophan 2-C-methyltransferase' WHERE ec = 'EC 2.1.1.106';
UPDATE enzyme SET history = 'EC 2.1.1.106 created 1992' WHERE ec = 'EC 2.1.1.106';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:uroporphyrinogen-III C-methyltransferase' WHERE ec = 'EC 2.1.1.107';
UPDATE enzyme SET history = 'EC 2.1.1.107 created 1992, modified 2004' WHERE ec = 'EC 2.1.1.107';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:6-hydroxymellein 6-O-methyltransferase' WHERE ec = 'EC 2.1.1.108';
UPDATE enzyme SET history = 'EC 2.1.1.108 created 1992' WHERE ec = 'EC 2.1.1.108';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:6-demethylsterigmatocystin 6-O-methyltransferase' WHERE ec = 'EC 2.1.1.109';
UPDATE enzyme SET history = 'EC 2.1.1.109 created 1992' WHERE ec = 'EC 2.1.1.109';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:sterigmatocystin 8-O-methyltransferase' WHERE ec = 'EC 2.1.1.110';
UPDATE enzyme SET history = 'EC 2.1.1.110 created 1992, modified 2005, modified 2013' WHERE ec = 'EC 2.1.1.110';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:anthranilate N-methyltransferase' WHERE ec = 'EC 2.1.1.111';
UPDATE enzyme SET history = 'EC 2.1.1.111 created 1992' WHERE ec = 'EC 2.1.1.111';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:glucuronoxylan-D-glucuronate 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.112';
UPDATE enzyme SET history = 'EC 2.1.1.112 created 1992' WHERE ec = 'EC 2.1.1.112';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:DNA-cytosine N(4)-methyltransferase' WHERE ec = 'EC 2.1.1.113';
UPDATE enzyme SET history = 'EC 2.1.1.113 created 1992' WHERE ec = 'EC 2.1.1.113';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3,4-dihydroxy-5-all-trans-polyprenylbenzoate 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.114';
UPDATE enzyme SET history = 'EC 2.1.1.114 created 1999' WHERE ec = 'EC 2.1.1.114';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(RS)-1-benzyl-1,2,3,4-tetrahydroisoquinoline N-methyltransferase' WHERE ec = 'EC 2.1.1.115';
UPDATE enzyme SET history = 'EC 2.1.1.115 created 1999' WHERE ec = 'EC 2.1.1.115';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3-hydroxy-N-methyl-(S)-coclaurine 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.116';
UPDATE enzyme SET history = 'EC 2.1.1.116 created 1999' WHERE ec = 'EC 2.1.1.116';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(S)-scoulerine 9-O-methyltransferase' WHERE ec = 'EC 2.1.1.117';
UPDATE enzyme SET history = 'EC 2.1.1.117 created 1999' WHERE ec = 'EC 2.1.1.117';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:columbamine O-methyltransferase' WHERE ec = 'EC 2.1.1.118';
UPDATE enzyme SET history = 'EC 2.1.1.118 created 1999' WHERE ec = 'EC 2.1.1.118';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:10-hydroxydihydrosanguinarine 10-O-methyltransferase' WHERE ec = 'EC 2.1.1.119';
UPDATE enzyme SET history = 'EC 2.1.1.119 created 1999' WHERE ec = 'EC 2.1.1.119';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:12-hydroxydihydrochelirubine 12-O-methyltransferase' WHERE ec = 'EC 2.1.1.120';
UPDATE enzyme SET history = 'EC 2.1.1.120 created 1999' WHERE ec = 'EC 2.1.1.120';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:6-O-methylnorlaudanosoline 5-O-methyltransferase' WHERE ec = 'EC 2.1.1.121';
UPDATE enzyme SET history = 'EC 2.1.1.121 created 1999' WHERE ec = 'EC 2.1.1.121';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(S)-7,8,13,14-tetrahydroprotoberberine cis-N-methyltransferase' WHERE ec = 'EC 2.1.1.122';
UPDATE enzyme SET history = 'EC 2.1.1.122 created 1999' WHERE ec = 'EC 2.1.1.122';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[cytochrome c]-methionine S-methyltransferase' WHERE ec = 'EC 2.1.1.123';
UPDATE enzyme SET history = 'EC 2.1.1.123 created 1999' WHERE ec = 'EC 2.1.1.123';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[cytochrome c]-arginine N(omega)-methyltransferase' WHERE ec = 'EC 2.1.1.124';
UPDATE enzyme SET history = 'EC 2.1.1.124 created 1999 (EC 2.1.1.23 created 1972, modified 1976, modified 1983, part incorporated 1999)' WHERE ec = 'EC 2.1.1.124';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:histone-arginine N(omega)-methyltransferase' WHERE ec = 'EC 2.1.1.125';
UPDATE enzyme SET history = 'EC 2.1.1.125 created 1999 (EC 2.1.1.23 created 1972, modified 1976, modified 1983, part incorporated 1999)' WHERE ec = 'EC 2.1.1.125';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[myelin-basic-protein]-arginine N(omega)-methyltransferase' WHERE ec = 'EC 2.1.1.126';
UPDATE enzyme SET history = 'EC 2.1.1.126 created 1999 (EC 2.1.1.23 created 1972, modified 1976, modified 1983, part incorporated 1999)' WHERE ec = 'EC 2.1.1.126';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[3-phospho-D-glycerate-carboxy-lyase (dimerizing)]-lysine N(6)-methyltransferase' WHERE ec = 'EC 2.1.1.127';
UPDATE enzyme SET history = 'EC 2.1.1.127 created 1999, modified 2012' WHERE ec = 'EC 2.1.1.127';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(RS)-norcoclaurine 6-O-methyltransferase' WHERE ec = 'EC 2.1.1.128';
UPDATE enzyme SET history = 'EC 2.1.1.128 created 1999' WHERE ec = 'EC 2.1.1.128';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:1D-myo-inositol 4-methyltransferase' WHERE ec = 'EC 2.1.1.129';
UPDATE enzyme SET history = 'EC 2.1.1.129 created 1999 (EC 2.1.1.134 created 1999, incorporated 2002), modified 2002' WHERE ec = 'EC 2.1.1.129';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:precorrin-2 C(20)-methyltransferase' WHERE ec = 'EC 2.1.1.130';
UPDATE enzyme SET history = 'EC 2.1.1.130 created 1999' WHERE ec = 'EC 2.1.1.130';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:precorrin-3B C(17)-methyltransferase' WHERE ec = 'EC 2.1.1.131';
UPDATE enzyme SET history = 'EC 2.1.1.131 created 1999' WHERE ec = 'EC 2.1.1.131';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:1-precorrin-6B C(5,15)-methyltransferase (C-12-decarboxylating)' WHERE ec = 'EC 2.1.1.132';
UPDATE enzyme SET history = 'EC 2.1.1.132 created 1999, modified 2013' WHERE ec = 'EC 2.1.1.132';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:precorrin-4 C(11) methyltransferase' WHERE ec = 'EC 2.1.1.133';
UPDATE enzyme SET history = 'EC 2.1.1.133 created 1999' WHERE ec = 'EC 2.1.1.133';
UPDATE enzyme SET history = 'EC 2.1.1.134 created 1999, deleted 2002' WHERE ec = 'EC 2.1.1.134';
UPDATE enzyme SET history = 'EC 2.1.1.135 created 1999, deleted 2003' WHERE ec = 'EC 2.1.1.135';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:trichlorophenol O-methyltransferase' WHERE ec = 'EC 2.1.1.136';
UPDATE enzyme SET history = 'EC 2.1.1.136 created 2000' WHERE ec = 'EC 2.1.1.136';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:arsenite As-methyltransferase' WHERE ec = 'EC 2.1.1.137';
UPDATE enzyme SET history = 'EC 2.1.1.137 created 2000, modified 2003' WHERE ec = 'EC 2.1.1.137';
UPDATE enzyme SET history = 'EC 2.1.1.138 created 2000, deleted 2003' WHERE ec = 'EC 2.1.1.138';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3-demethylstaurosporine O-methyltransferase' WHERE ec = 'EC 2.1.1.139';
UPDATE enzyme SET history = 'EC 2.1.1.139 created 2000' WHERE ec = 'EC 2.1.1.139';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(S)-coclaurine-N-methyltransferase' WHERE ec = 'EC 2.1.1.140';
UPDATE enzyme SET history = 'EC 2.1.1.140 created 2001' WHERE ec = 'EC 2.1.1.140';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:jasmonate O-methyltransferase' WHERE ec = 'EC 2.1.1.141';
UPDATE enzyme SET history = 'EC 2.1.1.141 created 2001' WHERE ec = 'EC 2.1.1.141';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:cycloartenol 24-C-methyltransferase' WHERE ec = 'EC 2.1.1.142';
UPDATE enzyme SET history = 'EC 2.1.1.142 created 2001' WHERE ec = 'EC 2.1.1.142';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:24-methylenelophenol C-methyltransferase' WHERE ec = 'EC 2.1.1.143';
UPDATE enzyme SET history = 'EC 2.1.1.143 created 2001' WHERE ec = 'EC 2.1.1.143';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(E)-prop-1-ene-1,2,3-tricarboxylate 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.144';
UPDATE enzyme SET history = 'EC 2.1.1.144 created 2002' WHERE ec = 'EC 2.1.1.144';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(E)-prop-1-ene-1,2,3-tricarboxylate 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.145';
UPDATE enzyme SET history = 'EC 2.1.1.145 created 2002' WHERE ec = 'EC 2.1.1.145';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:isoeugenol O-methyltransferase' WHERE ec = 'EC 2.1.1.146';
UPDATE enzyme SET history = 'EC 2.1.1.146 created 2002' WHERE ec = 'EC 2.1.1.146';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:protoberberine 13-C-methyltransferase' WHERE ec = 'EC 2.1.1.147';
UPDATE enzyme SET history = 'EC 2.1.1.147 created 2002' WHERE ec = 'EC 2.1.1.147';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate,FADH(2):dUMP C-methyltransferase' WHERE ec = 'EC 2.1.1.148';
UPDATE enzyme SET history = 'EC 2.1.1.148 created 2003, modified 2010' WHERE ec = 'EC 2.1.1.148';
UPDATE enzyme SET history = 'EC 2.1.1.149 created 2003, modified 2011, deleted 2013' WHERE ec = 'EC 2.1.1.149';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:hydroxyisoflavone 7-O-methyltransferase' WHERE ec = 'EC 2.1.1.150';
UPDATE enzyme SET history = 'EC 2.1.1.150 created 2003' WHERE ec = 'EC 2.1.1.150';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:cobalt-factor-II C(20)-methyltransferase' WHERE ec = 'EC 2.1.1.151';
UPDATE enzyme SET history = 'EC 2.1.1.151 created 2004' WHERE ec = 'EC 2.1.1.151';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:precorrin-5 C(1)-methyltransferase (deacetylating)' WHERE ec = 'EC 2.1.1.152';
UPDATE enzyme SET history = 'EC 2.1.1.152 created 2004' WHERE ec = 'EC 2.1.1.152';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:vitexin-2-O-beta-L-rhamnoside 7-O-methyltransferase' WHERE ec = 'EC 2.1.1.153';
UPDATE enzyme SET history = 'EC 2.1.1.153 created 2004' WHERE ec = 'EC 2.1.1.153';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:isoliquiritigenin 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.154';
UPDATE enzyme SET history = 'EC 2.1.1.154 created 2004' WHERE ec = 'EC 2.1.1.154';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:kaempferol 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.155';
UPDATE enzyme SET history = 'EC 2.1.1.155 created 2004' WHERE ec = 'EC 2.1.1.155';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:glycine(or sarcosine) N-methyltransferase (sarcosine(or N,N-dimethylglycine)-forming)' WHERE ec = 'EC 2.1.1.156';
UPDATE enzyme SET history = 'EC 2.1.1.156 created 2005' WHERE ec = 'EC 2.1.1.156';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:sarcosine(or N,N-dimethylglycine) N-methyltransferase (N,N-dimethylglycine(or betaine)-forming)' WHERE ec = 'EC 2.1.1.157';
UPDATE enzyme SET history = 'EC 2.1.1.157 created 2005, modified 2010' WHERE ec = 'EC 2.1.1.157';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:xanthosine N(7)-methyltransferase' WHERE ec = 'EC 2.1.1.158';
UPDATE enzyme SET history = 'EC 2.1.1.158 created 2007' WHERE ec = 'EC 2.1.1.158';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:7-methylxanthine N(3)-methyltransferase' WHERE ec = 'EC 2.1.1.159';
UPDATE enzyme SET history = 'EC 2.1.1.159 created 2007' WHERE ec = 'EC 2.1.1.159';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3,7-dimethylxanthine N(1)-methyltransferase' WHERE ec = 'EC 2.1.1.160';
UPDATE enzyme SET history = 'EC 2.1.1.160 created 2007' WHERE ec = 'EC 2.1.1.160';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:N,N-dimethylglycine N-methyltransferase (betaine-forming)' WHERE ec = 'EC 2.1.1.161';
UPDATE enzyme SET history = 'EC 2.1.1.161 created 2007' WHERE ec = 'EC 2.1.1.161';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:glycine(or sarcosine or N,N-dimethylglycine) N-methyltransferase (sarcosine(or N,N-dimethylglycine or betaine)-forming)' WHERE ec = 'EC 2.1.1.162';
UPDATE enzyme SET history = 'EC 2.1.1.162 created 2007' WHERE ec = 'EC 2.1.1.162';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methione:demethylmenaquinone methyltransferase' WHERE ec = 'EC 2.1.1.163';
UPDATE enzyme SET history = 'EC 2.1.1.163 created 2009' WHERE ec = 'EC 2.1.1.163';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:demethylrebeccamycin-D-glucose O-methyltransferase' WHERE ec = 'EC 2.1.1.164';
UPDATE enzyme SET history = 'EC 2.1.1.164 created 2010' WHERE ec = 'EC 2.1.1.164';
UPDATE enzyme SET sys_name = 'S-adenosylmethionine:iodide methyltransferase' WHERE ec = 'EC 2.1.1.165';
UPDATE enzyme SET history = 'EC 2.1.1.165 created 2010' WHERE ec = 'EC 2.1.1.165';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (uridine(2552)-2-O-)-methyltransferase' WHERE ec = 'EC 2.1.1.166';
UPDATE enzyme SET history = 'EC 2.1.1.166 created 2010' WHERE ec = 'EC 2.1.1.166';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:27S pre-rRNA (guanosine(2922)-2-O-)-methyltransferase' WHERE ec = 'EC 2.1.1.167';
UPDATE enzyme SET history = 'EC 2.1.1.167 created 2010' WHERE ec = 'EC 2.1.1.167';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:21S rRNA (uridine(2791)-2-O-)-methyltransferase' WHERE ec = 'EC 2.1.1.168';
UPDATE enzyme SET history = 'EC 2.1.1.168 created 2010' WHERE ec = 'EC 2.1.1.168';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tricetin 3,4,5-O-trimethyltransferase' WHERE ec = 'EC 2.1.1.169';
UPDATE enzyme SET history = 'EC 2.1.1.169 cfreated 2010' WHERE ec = 'EC 2.1.1.169';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (guanine(527)-N(7))-methyltransferase' WHERE ec = 'EC 2.1.1.170';
UPDATE enzyme SET history = 'EC 2.1.1.170 created 2010' WHERE ec = 'EC 2.1.1.170';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (guanine(966)-N(2))-methyltransferase' WHERE ec = 'EC 2.1.1.171';
UPDATE enzyme SET history = 'EC 2.1.1.171 created 1976 as EC 2.1.1.52, part-transferred 2010 to EC 2.1.1.171' WHERE ec = 'EC 2.1.1.171';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (guanine(1207)-N(2))-methyltransferase' WHERE ec = 'EC 2.1.1.172';
UPDATE enzyme SET history = 'EC 2.1.1.172 created 1976 as EC 2.1.1.52, part-transferred 2010 to EC 2.1.1.172' WHERE ec = 'EC 2.1.1.172';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (guanine(2445)-N(2))-methyltransferase' WHERE ec = 'EC 2.1.1.173';
UPDATE enzyme SET history = 'EC 2.1.1.173 created 1976 as EC 2.1.1.52, part-transferred 2010 to EC 2.1.1.173' WHERE ec = 'EC 2.1.1.173';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (guanine(1835)-N(2))-methyltransferase' WHERE ec = 'EC 2.1.1.174';
UPDATE enzyme SET history = 'EC 2.1.1.174 created 1976 as EC 2.1.1.52, part-transferred 2010 to EC 2.1.1.174' WHERE ec = 'EC 2.1.1.174';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tricetin 3,5-O-dimethyltransferase' WHERE ec = 'EC 2.1.1.175';
UPDATE enzyme SET history = 'EC 2.1.1.175 created 2010' WHERE ec = 'EC 2.1.1.175';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (cytosine(967)-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.176';
UPDATE enzyme SET history = 'EC 2.1.1.176 created 2010' WHERE ec = 'EC 2.1.1.176';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (pseudouridine(1915)-N(3))-methyltransferase' WHERE ec = 'EC 2.1.1.177';
UPDATE enzyme SET history = 'EC 2.1.1.177 created 2010' WHERE ec = 'EC 2.1.1.177';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (cytosine(91407)-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.178';
UPDATE enzyme SET history = 'EC 2.1.1.178 created 2010' WHERE ec = 'EC 2.1.1.178';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (guanine(1405)-N(7))-methyltransferase' WHERE ec = 'EC 2.1.1.179';
UPDATE enzyme SET history = 'EC 2.1.1.179 created 2010' WHERE ec = 'EC 2.1.1.179';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (adenine(1408)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.180';
UPDATE enzyme SET history = 'EC 2.1.1.180 created 2010' WHERE ec = 'EC 2.1.1.180';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(adenine(1618)-N(6))-methyltransferase' WHERE ec = 'EC 2.1.1.181';
UPDATE enzyme SET history = 'EC 2.1.1.181 created 1976 as EC 2.1.1.48, part-transferred 2010 to EC 2.1.1.181' WHERE ec = 'EC 2.1.1.181';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase' WHERE ec = 'EC 2.1.1.182';
UPDATE enzyme SET history = 'EC 2.1.1.182 created 1976 as EC 2.1.1.48, part-transferred 2010 to EC 2.1.1.182' WHERE ec = 'EC 2.1.1.182';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:18S rRNA (adenine(1779)-N(6)/adenine(1780)-N(6))-dimethyltransferase' WHERE ec = 'EC 2.1.1.183';
UPDATE enzyme SET history = 'EC 2.1.1.183 created 1976 as EC 2.1.1.48, part-transferred 2010 to EC 2.1.1.183' WHERE ec = 'EC 2.1.1.183';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (adenine(2085)-N(6))-dimethyltransferase' WHERE ec = 'EC 2.1.1.184';
UPDATE enzyme SET history = 'EC 2.1.1.184 created 1976 as EC 2.1.1.48, part-transferred 2010 to EC 2.1.1.184' WHERE ec = 'EC 2.1.1.184';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (guanosine(2251)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.185';
UPDATE enzyme SET history = 'EC 2.1.1.185 created 2010' WHERE ec = 'EC 2.1.1.185';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (cytidine(2498)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.186';
UPDATE enzyme SET history = 'EC 2.1.1.186 created 2010' WHERE ec = 'EC 2.1.1.186';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (guanine(745)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.187';
UPDATE enzyme SET history = 'EC 2.1.1.187 created 1976 as EC 2.1.1.51, part-transferred 2010 to EC 2.1.1.187' WHERE ec = 'EC 2.1.1.187';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (guanine(748)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.188';
UPDATE enzyme SET history = 'EC 2.1.1.188 created 1976 as EC 2.1.1.51, part-transferred 2010 to EC 2.1.1.188' WHERE ec = 'EC 2.1.1.188';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (uracil(747)-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.189';
UPDATE enzyme SET history = 'EC 2.1.1.189 created 2010' WHERE ec = 'EC 2.1.1.189';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (uracil(1939)-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.190';
UPDATE enzyme SET history = 'EC 2.1.1.190 created 2010' WHERE ec = 'EC 2.1.1.190';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (cytosine(1962)-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.191';
UPDATE enzyme SET history = 'EC 2.1.1.191 created 2010' WHERE ec = 'EC 2.1.1.191';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (adenine(2503)-C(2))-methyltransferase' WHERE ec = 'EC 2.1.1.192';
UPDATE enzyme SET history = 'EC 2.1.1.192 created 2010, modified 2011' WHERE ec = 'EC 2.1.1.192';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (uracil(1498)-N(3))-methyltransferase' WHERE ec = 'EC 2.1.1.193';
UPDATE enzyme SET history = 'EC 2.1.1.193 created 2010' WHERE ec = 'EC 2.1.1.193';
UPDATE enzyme SET history = 'EC 2.1.1.194 created 2010, deleted 2011' WHERE ec = 'EC 2.1.1.194';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:cobalt-precorrin-5B (C(1))-methyltransferase' WHERE ec = 'EC 2.1.1.195';
UPDATE enzyme SET history = 'EC 2.1.1.195 created 2010' WHERE ec = 'EC 2.1.1.195';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:precorrin-6B C(15)-methyltransferase (C-12-decarboxylating)' WHERE ec = 'EC 2.1.1.196';
UPDATE enzyme SET history = 'EC 2.1.1.196 created 2010, modified 2013' WHERE ec = 'EC 2.1.1.196';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:malonyl-[acyl-carrier protein] O-methyltransferase' WHERE ec = 'EC 2.1.1.197';
UPDATE enzyme SET history = 'EC 2.1.1.197 created 2010, modified 2013' WHERE ec = 'EC 2.1.1.197';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (cytidine(1402)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.198';
UPDATE enzyme SET history = 'EC 2.1.1.198 created 2010' WHERE ec = 'EC 2.1.1.198';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (cytosine(1402)-N(4))-methyltransferase' WHERE ec = 'EC 2.1.1.199';
UPDATE enzyme SET history = 'EC 2.1.1.199 created 2010' WHERE ec = 'EC 2.1.1.199';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (cytidine(32)/uridine(32)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.200';
UPDATE enzyme SET history = 'EC 2.1.1.200 created 2011' WHERE ec = 'EC 2.1.1.200';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:2-methoxy-6-all-trans-polyprenyl-1,4-benzoquinol 5-C-methyltransferase' WHERE ec = 'EC 2.1.1.201';
UPDATE enzyme SET history = 'EC 2.1.1.201 created 2011' WHERE ec = 'EC 2.1.1.201';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (cytosine-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.202';
UPDATE enzyme SET history = 'EC 2.1.1.202 created 1976 as EC 2.1.1.29, part-transferred 2011 to EC 2.1.1.202' WHERE ec = 'EC 2.1.1.202';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (cytosine(34)-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.203';
UPDATE enzyme SET history = 'EC 2.1.1.203 created 2011' WHERE ec = 'EC 2.1.1.203';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (cytosine(38)-C(5))-methyltransferase' WHERE ec = 'EC 2.1.1.204';
UPDATE enzyme SET history = 'EC 2.1.1.204 created 2011' WHERE ec = 'EC 2.1.1.204';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (cytidine(32)/guanosine(34)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.205';
UPDATE enzyme SET history = 'EC 2.1.1.205 created 2011' WHERE ec = 'EC 2.1.1.205';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (cytidine(56)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.206';
UPDATE enzyme SET history = 'EC 2.1.1.206 created 2011' WHERE ec = 'EC 2.1.1.206';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (cytidine(34)/5-carboxymethylaminomethyluridine(34)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.207';
UPDATE enzyme SET history = 'EC 2.1.1.207 created 2011' WHERE ec = 'EC 2.1.1.207';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (uridine(2479)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.208';
UPDATE enzyme SET history = 'EC 2.1.1.208 created 2011' WHERE ec = 'EC 2.1.1.208';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (guanine(2535)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.209';
UPDATE enzyme SET history = 'EC 2.1.1.209 created 2011' WHERE ec = 'EC 2.1.1.209';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:demethylspheroidene O-methyltransferase' WHERE ec = 'EC 2.1.1.210';
UPDATE enzyme SET history = 'EC 2.1.1.210 created 2011' WHERE ec = 'EC 2.1.1.210';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA(Ser) (uridine(44)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.211';
UPDATE enzyme SET history = 'EC 2.1.1.211 created 2011' WHERE ec = 'EC 2.1.1.211';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:2,4,7-trihydroxyisoflavanone 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.212';
UPDATE enzyme SET history = 'EC 2.1.1.212 created 2011' WHERE ec = 'EC 2.1.1.212';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (guanine(10)-N(2))-dimethyltransferase' WHERE ec = 'EC 2.1.1.213';
UPDATE enzyme SET history = 'EC 2.1.1.213, created 1972 as EC 2.1.1.32, part-transferred 2011 to EC 2.1.1.213' WHERE ec = 'EC 2.1.1.213';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (guanine(10)-N(2))-methyltransferase' WHERE ec = 'EC 2.1.1.214';
UPDATE enzyme SET history = 'EC 2.1.1.214, created 1972 as EC 2.1.1.32, part-transferred 2011 to EC 2.1.1.214' WHERE ec = 'EC 2.1.1.214';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (guanine(26)-N(2)/guanine(27)-N(2))-dimethyltransferase' WHERE ec = 'EC 2.1.1.215';
UPDATE enzyme SET history = 'EC 2.1.1.215, created 1972 as EC 2.1.1.32, part-transferred 2011 to EC 2.1.1.215' WHERE ec = 'EC 2.1.1.215';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (guanine(26)-N(2))-dimethyltransferase' WHERE ec = 'EC 2.1.1.216';
UPDATE enzyme SET history = 'EC 2.1.1.216, created 1972 as EC 2.1.1.32, part-transferred 2011 to EC 2.1.1.216' WHERE ec = 'EC 2.1.1.216';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (adenine(22)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.217';
UPDATE enzyme SET history = 'EC 2.1.1.217 created 1972 as EC 2.1.1.36, part-transferred 2011 to EC 2.1.1.217' WHERE ec = 'EC 2.1.1.217';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (adenine(9)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.218';
UPDATE enzyme SET history = 'EC 2.1.1.218 created 1972 as EC 2.1.1.36, part-transferred 2011 to EC 2.1.1.218' WHERE ec = 'EC 2.1.1.218';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (adenine(57)/adenine(58)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.219';
UPDATE enzyme SET history = 'EC 2.1.1.219 created 1972 as EC 2.1.1.36, part-transferred 2011 to EC 2.1.1.219' WHERE ec = 'EC 2.1.1.219';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (adenine(58)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.220';
UPDATE enzyme SET history = 'EC 2.1.1.220 created 1972 as EC 2.1.1.36, part-transferred 2011 to EC 2.1.1.220' WHERE ec = 'EC 2.1.1.220';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (guanine(9)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.221';
UPDATE enzyme SET history = 'EC 2.1.1.221,  created 1972 as EC 2.1.1.31, part-transferred 2011 to EC 2.1.1.221' WHERE ec = 'EC 2.1.1.221';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3-(all-trans-polyprenyl)benzene-1,2-diol 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.222';
UPDATE enzyme SET history = 'EC 2.1.1.222 created 2011, modified 2013' WHERE ec = 'EC 2.1.1.222';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA(1)(Val) (adenine(37)-N(6))-methyltransferase' WHERE ec = 'EC 2.1.1.223';
UPDATE enzyme SET history = 'EC 2.1.1.223 created 2011' WHERE ec = 'EC 2.1.1.223';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (adenine(2503)-C(8))-methyltransferase' WHERE ec = 'EC 2.1.1.224';
UPDATE enzyme SET history = 'EC 2.1.1.224 created 2011' WHERE ec = 'EC 2.1.1.224';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA(Pro/His/Gly)(GCC) (cytidine/adenosine(4)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.225';
UPDATE enzyme SET history = 'EC 2.1.1.225 created 2011' WHERE ec = 'EC 2.1.1.225';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (cytidine(1920)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.226';
UPDATE enzyme SET history = 'EC 2.1.1.226 created 2011' WHERE ec = 'EC 2.1.1.226';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (cytidine(1409)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.227';
UPDATE enzyme SET history = 'EC 2.1.1.227 created 2011' WHERE ec = 'EC 2.1.1.227';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (guanine(37)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.228';
UPDATE enzyme SET history = 'EC 2.1.1.228, created 1972 as EC 2.1.1.31, part-transferred 2011 to EC 2.1.1.228' WHERE ec = 'EC 2.1.1.228';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (carboxymethyluridine(34)-5-O)-methyltransferase' WHERE ec = 'EC 2.1.1.229';
UPDATE enzyme SET history = 'EC 2.1.1.229 created 2011' WHERE ec = 'EC 2.1.1.229';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (adenosine(1067)-2-O)-methyltransferase' WHERE ec = 'EC 2.1.1.230';
UPDATE enzyme SET history = 'EC 2.1.1.230 created 2011' WHERE ec = 'EC 2.1.1.230';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:flavonoid 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.231';
UPDATE enzyme SET history = 'EC 2.1.1.231 created 2011' WHERE ec = 'EC 2.1.1.231';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(2S)-5,7,4-trihydroxyflavanone 7-O-methyltransferase' WHERE ec = 'EC 2.1.1.232';
UPDATE enzyme SET history = 'EC 2.1.1.232 created 2011' WHERE ec = 'EC 2.1.1.232';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[phosphatase 2A protein]-leucine O-methyltransferase' WHERE ec = 'EC 2.1.1.233';
UPDATE enzyme SET history = 'EC 2.1.1.233 created 2011' WHERE ec = 'EC 2.1.1.233';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:dTDP-3-amino-3,4,6-trideoxy-alpha-D-glucopyranose 3-N,N-dimethyltransferase' WHERE ec = 'EC 2.1.1.234';
UPDATE enzyme SET history = 'EC 2.1.1.234 created 2011' WHERE ec = 'EC 2.1.1.234';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:dTDP-3-amino-3,6-dideoxy-alpha-D-glucopyranose 3-N,N-dimethyltransferase' WHERE ec = 'EC 2.1.1.235';
UPDATE enzyme SET history = 'EC 2.1.1.235 created 2011' WHERE ec = 'EC 2.1.1.235';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:dTDP-3-amino-3,6-dideoxy-alpha-D-galactopyranose 3-N,N-dimethyltransferase' WHERE ec = 'EC 2.1.1.236';
UPDATE enzyme SET history = 'EC 2.1.1.236 created 2011' WHERE ec = 'EC 2.1.1.236';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:mycinamicin III 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.237';
UPDATE enzyme SET history = 'EC 2.1.1.237 created 2011' WHERE ec = 'EC 2.1.1.237';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:mycinamicin VI 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.238';
UPDATE enzyme SET history = 'EC 2.1.1.238 created 2011' WHERE ec = 'EC 2.1.1.238';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:L-olivosyl-oleandolide B 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.239';
UPDATE enzyme SET history = 'EC 2.1.1.239 created 2012' WHERE ec = 'EC 2.1.1.239';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:trans-resveratrol 3,5-O-dimethyltransferase' WHERE ec = 'EC 2.1.1.240';
UPDATE enzyme SET history = 'EC 2.1.1.240 created 2012' WHERE ec = 'EC 2.1.1.240';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(2R)-4,7-dihydroxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside 7-O-methyltransferase' WHERE ec = 'EC 2.1.1.241';
UPDATE enzyme SET history = 'EC 2.1.1.241 created 2012' WHERE ec = 'EC 2.1.1.241';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:16S rRNA (guanine(1516)-N(2))-methyltransferase' WHERE ec = 'EC 2.1.1.242';
UPDATE enzyme SET history = 'EC 2.1.1.242 created 2012' WHERE ec = 'EC 2.1.1.242';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:5-carbamimidamido-2-oxopentanoate S-methyltransferase' WHERE ec = 'EC 2.1.1.243';
UPDATE enzyme SET history = 'EC 2.1.1.243 created 2012' WHERE ec = 'EC 2.1.1.243';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:N-terminal-(A,P,S)PK-(protein) methyltransferase' WHERE ec = 'EC 2.1.1.244';
UPDATE enzyme SET history = 'EC 2.1.1.244 created 2012' WHERE ec = 'EC 2.1.1.244';
UPDATE enzyme SET sys_name = '5-methyltetrahydrosarcinapterin:corrinoid/iron-sulfur protein methyltransferase' WHERE ec = 'EC 2.1.1.245';
UPDATE enzyme SET history = 'EC 2.1.1.245 created 2012' WHERE ec = 'EC 2.1.1.245';
UPDATE enzyme SET sys_name = 'Methylated methanol-specific corrinoid protein:coenzyme M methyltransferase' WHERE ec = 'EC 2.1.1.246';
UPDATE enzyme SET history = 'EC 2.1.1.246 created 2012' WHERE ec = 'EC 2.1.1.246';
UPDATE enzyme SET sys_name = 'Methylated monomethylamine-specific corrinoid protein:coenzyme M methyltransferase' WHERE ec = 'EC 2.1.1.247';
UPDATE enzyme SET history = 'EC 2.1.1.247 created 2012' WHERE ec = 'EC 2.1.1.247';
UPDATE enzyme SET sys_name = 'Monomethylamine:5-hydroxybenzimidazolylcobamide Co-methyltransferase' WHERE ec = 'EC 2.1.1.248';
UPDATE enzyme SET history = 'EC 2.1.1.248 created 2012' WHERE ec = 'EC 2.1.1.248';
UPDATE enzyme SET sys_name = 'Dimethylamine:5-hydroxybenzimidazolylcobamide Co-methyltransferase' WHERE ec = 'EC 2.1.1.249';
UPDATE enzyme SET history = 'EC 2.1.1.249 created 2012' WHERE ec = 'EC 2.1.1.249';
UPDATE enzyme SET sys_name = 'Trimethylamine:5-hydroxybenzimidazolylcobamide Co-methyltransferase' WHERE ec = 'EC 2.1.1.250';
UPDATE enzyme SET history = 'EC 2.1.1.250 created 2012' WHERE ec = 'EC 2.1.1.250';
UPDATE enzyme SET sys_name = 'Methylated-thiol:coenzyme M methyltransferase' WHERE ec = 'EC 2.1.1.251';
UPDATE enzyme SET history = 'EC 2.1.1.251 created 2012' WHERE ec = 'EC 2.1.1.251';
UPDATE enzyme SET sys_name = 'Tetramethylammonium:5-hydroxybenzimidazolylcobamide Co-methyltransferase' WHERE ec = 'EC 2.1.1.252';
UPDATE enzyme SET history = 'EC 2.1.1.252 created 2012' WHERE ec = 'EC 2.1.1.252';
UPDATE enzyme SET sys_name = 'Methylated tetramethylammonium-specific corrinoid protein:coenzyme M methyltransferase' WHERE ec = 'EC 2.1.1.253';
UPDATE enzyme SET history = 'EC 2.1.1.253 created 2012' WHERE ec = 'EC 2.1.1.253';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:erythromycin C 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.254';
UPDATE enzyme SET history = 'EC 2.1.1.254 created 2012' WHERE ec = 'EC 2.1.1.254';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:geranyl-diphosphate 2-C-methyltransferase' WHERE ec = 'EC 2.1.1.255';
UPDATE enzyme SET history = 'EC 2.1.1.255 created 2012' WHERE ec = 'EC 2.1.1.255';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (guanine(6)-N(2))-methyltransferase' WHERE ec = 'EC 2.1.1.256';
UPDATE enzyme SET history = 'EC 2.1.1.256 created 2012' WHERE ec = 'EC 2.1.1.256';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA (pseudouridine(54)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.257';
UPDATE enzyme SET history = 'EC 2.1.1.257 created 2012' WHERE ec = 'EC 2.1.1.257';
UPDATE enzyme SET sys_name = '5-methyltetrahydrofolate:corrinoid/iron-sulfur protein methyltransferase' WHERE ec = 'EC 2.1.1.258';
UPDATE enzyme SET history = 'EC 2.1.1.258 created 2012' WHERE ec = 'EC 2.1.1.258';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[fructose-bisphosphate aldolase]-lysine N(6)-methyltransferase' WHERE ec = 'EC 2.1.1.259';
UPDATE enzyme SET history = 'EC 2.1.1.259 created 2012' WHERE ec = 'EC 2.1.1.259';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:18S rRNA (pseudouridine(1191)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.260';
UPDATE enzyme SET history = 'EC 2.1.1.260 created 2012' WHERE ec = 'EC 2.1.1.260';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:4-(3-methylbut-2-enyl)-L-tryptophan N-methyltransferase' WHERE ec = 'EC 2.1.1.261';
UPDATE enzyme SET history = 'EC 2.1.1.261 created 2012' WHERE ec = 'EC 2.1.1.261';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:squalene C-methyltransferase' WHERE ec = 'EC 2.1.1.262';
UPDATE enzyme SET history = 'EC 2.1.1.262 created 2012' WHERE ec = 'EC 2.1.1.262';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:botryococcene C-methyltransferase' WHERE ec = 'EC 2.1.1.263';
UPDATE enzyme SET history = 'EC 2.1.1.263 created 2012' WHERE ec = 'EC 2.1.1.263';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (guanine(2069)-N(7))-methyltransferase' WHERE ec = 'EC 2.1.1.264';
UPDATE enzyme SET history = 'EC 2.1.1.264 created 2012' WHERE ec = 'EC 2.1.1.264';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tellurite methyltransferase' WHERE ec = 'EC 2.1.1.265';
UPDATE enzyme SET history = 'EC 2.1.1.265 created 2012' WHERE ec = 'EC 2.1.1.265';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:23S rRNA (adenine(2030)-N(6))-methyltransferase' WHERE ec = 'EC 2.1.1.266';
UPDATE enzyme SET history = 'EC 2.1.1.266 created 2013' WHERE ec = 'EC 2.1.1.266';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:flavonoid 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.267';
UPDATE enzyme SET history = 'EC 2.1.1.267 created 2013, modified 2014' WHERE ec = 'EC 2.1.1.267';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA(Thr) (cytosine(32)-N(3))-methyltransferase' WHERE ec = 'EC 2.1.1.268';
UPDATE enzyme SET history = 'EC 2.1.1.268 created 2013' WHERE ec = 'EC 2.1.1.268';
UPDATE enzyme SET sys_name = 'S,S-dimethyl-beta-propiothetin:tetrahydrofolate S-methyltransferase' WHERE ec = 'EC 2.1.1.269';
UPDATE enzyme SET history = 'EC 2.1.1.269 created 2013' WHERE ec = 'EC 2.1.1.269';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(+)-6a-hydroxymaackiain 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.270';
UPDATE enzyme SET history = 'EC 2.1.1.270 created 2013' WHERE ec = 'EC 2.1.1.270';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:cobalt-precorrin-4 11-methyltransferase' WHERE ec = 'EC 2.1.1.271';
UPDATE enzyme SET history = 'EC 2.1.1.271 created 2013' WHERE ec = 'EC 2.1.1.271';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:cobalt-factor III 17-methyltransferase (ring contracting)' WHERE ec = 'EC 2.1.1.272';
UPDATE enzyme SET history = 'EC 2.1.1.272 created 2013' WHERE ec = 'EC 2.1.1.272';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:benzoate O-methyltransferase' WHERE ec = 'EC 2.1.1.273';
UPDATE enzyme SET history = 'EC 2.1.1.273 created 2013' WHERE ec = 'EC 2.1.1.273';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:salicylate carboxymethyltransferase' WHERE ec = 'EC 2.1.1.274';
UPDATE enzyme SET history = 'EC 2.1.1.274 created 2013' WHERE ec = 'EC 2.1.1.274';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:gibberellin A(9) O-methyltransferase' WHERE ec = 'EC 2.1.1.275';
UPDATE enzyme SET history = 'EC 2.1.1.275 created 2013' WHERE ec = 'EC 2.1.1.275';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:gibberellin A(4) O-methyltransferase' WHERE ec = 'EC 2.1.1.276';
UPDATE enzyme SET history = 'EC 2.1.1.276 created 2013' WHERE ec = 'EC 2.1.1.276';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:anthranilate O-methyltransferase' WHERE ec = 'EC 2.1.1.277';
UPDATE enzyme SET history = 'EC 2.1.1.277 created 2013' WHERE ec = 'EC 2.1.1.277';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(indol-3-yl)acetate O-methyltransferase' WHERE ec = 'EC 2.1.1.278';
UPDATE enzyme SET history = 'EC 2.1.1.278 created 2013' WHERE ec = 'EC 2.1.1.278';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:trans-anol O-methyltransferase' WHERE ec = 'EC 2.1.1.279';
UPDATE enzyme SET history = 'EC 2.1.1.279 created 2013' WHERE ec = 'EC 2.1.1.279';
UPDATE enzyme SET sys_name = 'S-methyl-L-methionine:L-selenocysteine Se-methyltransferase' WHERE ec = 'EC 2.1.1.280';
UPDATE enzyme SET history = 'EC 2.1.1.280 created 2013' WHERE ec = 'EC 2.1.1.280';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:2-oxo-3-phenylpropanoate C(3)-methyltransferase' WHERE ec = 'EC 2.1.1.281';
UPDATE enzyme SET history = 'EC 2.1.1.281 created 2013' WHERE ec = 'EC 2.1.1.281';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA(Phe) 7-((3S)-(3-amino-3-carboxypropyl)-4-demethylwyosine-N(4))-methyltransferase' WHERE ec = 'EC 2.1.1.282';
UPDATE enzyme SET history = 'EC 2.1.1.282 created 2013, modified 2014' WHERE ec = 'EC 2.1.1.282';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:emodin 8-O-methyltransferase' WHERE ec = 'EC 2.1.1.283';
UPDATE enzyme SET history = 'EC 2.1.1.283 created 2013' WHERE ec = 'EC 2.1.1.283';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:8-demethylnovobiocic acid C(8)-methyltransferase' WHERE ec = 'EC 2.1.1.284';
UPDATE enzyme SET history = 'EC 2.1.1.284 created 2013' WHERE ec = 'EC 2.1.1.284';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:demethyldecarbamoylnovobiocin 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.285';
UPDATE enzyme SET history = 'EC 2.1.1.285 created 2013' WHERE ec = 'EC 2.1.1.285';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:25S rRNA (adenine(2142)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.286';
UPDATE enzyme SET history = 'EC 2.1.1.286 created 2013' WHERE ec = 'EC 2.1.1.286';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:25S rRNA (adenine(645)-N(1))-methyltransferase' WHERE ec = 'EC 2.1.1.287';
UPDATE enzyme SET history = 'EC 2.1.1.287 created 2013' WHERE ec = 'EC 2.1.1.287';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:aklanonate O-methyltransferase' WHERE ec = 'EC 2.1.1.288';
UPDATE enzyme SET history = 'EC 2.1.1.288 created 2013' WHERE ec = 'EC 2.1.1.288';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:precorrin-7 C(5)-methyltransferase' WHERE ec = 'EC 2.1.1.289';
UPDATE enzyme SET history = 'EC 2.1.1.289 created 2013' WHERE ec = 'EC 2.1.1.289';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA(Phe) (7-((3S)-3-amino-3-carboxypropyl)wyosine(37)-O)-methyltransferase' WHERE ec = 'EC 2.1.1.290';
UPDATE enzyme SET history = 'EC 2.1.1.290 created 2013' WHERE ec = 'EC 2.1.1.290';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(R,S)-reticuline 7-O-methyltransferase' WHERE ec = 'EC 2.1.1.291';
UPDATE enzyme SET history = 'EC 2.1.1.291 created 2013' WHERE ec = 'EC 2.1.1.291';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:carminomycin 4-O-methyltransferase' WHERE ec = 'EC 2.1.1.292';
UPDATE enzyme SET history = 'EC 2.1.1.292 created 2013' WHERE ec = 'EC 2.1.1.292';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:6-hydroxytryprostatin B O-methyltransferase' WHERE ec = 'EC 2.1.1.293';
UPDATE enzyme SET history = 'EC 2.1.1.293 created 2013' WHERE ec = 'EC 2.1.1.293';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3-O-phospho-alpha-D-Man-(1-&gt;2)-alpha-D-Man-(1-&gt;2)-alpha-D-Man-(1-&gt;3)-alpha-D-Man-(1-&gt;3)-(alpha-D-Man-(1-&gt;2)-alpha-D-Man-(1-&gt;2)-alpha-D-Man-(1-&gt;3)-alpha-D-Man-(1-&gt;3))(n)-alpha-D-Man-(1-&gt;3)-alpha-D-Man-(1-&gt;3)-alpha-D-GlcNAc-alpha-diphospho-ditrans,octacis-undecaprenol 3-phospho-methyltransferase' WHERE ec = 'EC 2.1.1.294';
UPDATE enzyme SET history = 'EC 2.1.1.294 created 2014' WHERE ec = 'EC 2.1.1.294';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:2-methyl-6-phytyl-1,4-benzoquinol C(3)-methyltransferase' WHERE ec = 'EC 2.1.1.295';
UPDATE enzyme SET history = 'EC 2.1.1.295 created 2014' WHERE ec = 'EC 2.1.1.295';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:5-(N(7)-methyl 5-triphosphoguanosine)-(2-O-methyl-purine-ribonucleotide)-ribonucleotide-(mRNA) 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.296';
UPDATE enzyme SET history = 'EC 2.1.1.296 created 2014' WHERE ec = 'EC 2.1.1.296';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[peptide chain release factor 1 or 2]-L-glutamine (N(5)-glutamine)-methyltransferase' WHERE ec = 'EC 2.1.1.297';
UPDATE enzyme SET history = 'EC 2.1.1.297 created 2014' WHERE ec = 'EC 2.1.1.297';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[ribosomal protein L3]-L-glutamine (N(5)-glutamine)-methyltransferase' WHERE ec = 'EC 2.1.1.298';
UPDATE enzyme SET history = 'EC 2.1.1.298 created 2014' WHERE ec = 'EC 2.1.1.298';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:N-terminal-(A,P,S)PK-[protein] monomethyltransferase' WHERE ec = 'EC 2.1.1.299';
UPDATE enzyme SET history = 'EC 2.1.1.299 created 2014' WHERE ec = 'EC 2.1.1.299';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(+-)-pavine N-methyltransferase' WHERE ec = 'EC 2.1.1.300';
UPDATE enzyme SET history = 'EC 2.1.1.300 created 2014' WHERE ec = 'EC 2.1.1.300';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:resorcinol 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.n1';
UPDATE enzyme SET history = 'EC 2.1.1.n1 created 2011' WHERE ec = 'EC 2.1.1.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.1.n2';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.1.n3';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:thiocyanate methyltransferase' WHERE ec = 'EC 2.1.1.n4';
UPDATE enzyme SET history = 'EC 2.1.1.n4 creagted 2011' WHERE ec = 'EC 2.1.1.n4';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.1.n5';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.1.n6';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:(8Z,11Z)-5-(pentadeca-8,11,14-trien-1-yl)resorcinol 3-O-methyltransferase' WHERE ec = 'EC 2.1.1.n7';
UPDATE enzyme SET history = 'EC 2.1.1.n7 created 2011' WHERE ec = 'EC 2.1.1.n7';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:small RNA 2-O-methyltransferase' WHERE ec = 'EC 2.1.1.n8';
UPDATE enzyme SET history = 'EC 2.1.1.n8 created 2012' WHERE ec = 'EC 2.1.1.n8';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.1.n9';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.1.n10';
UPDATE enzyme SET sys_name = 'DNA (containing Sp-methylphosphotriester):[protein] L-cysteine methyltransferase' WHERE ec = 'EC 2.1.1.n11';
UPDATE enzyme SET history = 'EC 2.1.1.n11 created 2013' WHERE ec = 'EC 2.1.1.n11';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:demethylphylloquinone methyltransferase' WHERE ec = 'EC 2.1.1.n12';
UPDATE enzyme SET history = 'EC 2.1.1.n12 created 2012' WHERE ec = 'EC 2.1.1.n12';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:[cypemycin]-Ala-N-term N-methyltransferase' WHERE ec = 'EC 2.1.1.n13';
UPDATE enzyme SET history = 'EC 2.1.1.n13 created 2011' WHERE ec = 'EC 2.1.1.n13';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.1.n14';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.1.n15';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.1.n16';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate:glycine hydroxymethyltransferase' WHERE ec = 'EC 2.1.2.1';
UPDATE enzyme SET history = 'EC 2.1.2.1 created 1961, modified 1983' WHERE ec = 'EC 2.1.2.1';
UPDATE enzyme SET sys_name = '10-formyltetrahydrofolate:5-phosphoribosylglycinamide N-formyltransferase' WHERE ec = 'EC 2.1.2.2';
UPDATE enzyme SET history = 'EC 2.1.2.2 created 1961, modified 2000' WHERE ec = 'EC 2.1.2.2';
UPDATE enzyme SET sys_name = '10-formyltetrahydrofolate:5-phosphoribosyl-5-amino-4-imidazole-carboxamide N-formyltransferase' WHERE ec = 'EC 2.1.2.3';
UPDATE enzyme SET history = 'EC 2.1.2.3 created 1961, modified 2000' WHERE ec = 'EC 2.1.2.3';
UPDATE enzyme SET sys_name = '5-formimidoyltetrahydrofolate:glycine N-formimidoyltransferase' WHERE ec = 'EC 2.1.2.4';
UPDATE enzyme SET history = 'EC 2.1.2.4 created 1961, modified 2000' WHERE ec = 'EC 2.1.2.4';
UPDATE enzyme SET sys_name = '5-formimidoyltetrahydrofolate:L-glutamate N-formimidoyltransferase' WHERE ec = 'EC 2.1.2.5';
UPDATE enzyme SET history = 'EC 2.1.2.5 created 1961, modified 2000 (EC 2.1.2.6 created 1965, incorporated 1984)' WHERE ec = 'EC 2.1.2.5';
UPDATE enzyme SET history = 'EC 2.1.2.6 created 1965, deleted 1984' WHERE ec = 'EC 2.1.2.6';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate:D-alanine 2-hydroxymethyltransferase' WHERE ec = 'EC 2.1.2.7';
UPDATE enzyme SET history = 'EC 2.1.2.7 created 1972' WHERE ec = 'EC 2.1.2.7';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate:deoxycytidylate 5-hydroxymethyltransferase' WHERE ec = 'EC 2.1.2.8';
UPDATE enzyme SET history = 'EC 2.1.2.8 created 1972' WHERE ec = 'EC 2.1.2.8';
UPDATE enzyme SET sys_name = '10-formyltetrahydrofolate:L-methionyl-tRNA N-formyltransferase' WHERE ec = 'EC 2.1.2.9';
UPDATE enzyme SET history = 'EC 2.1.2.9 created 1972, modified 2002, modified 2012' WHERE ec = 'EC 2.1.2.9';
UPDATE enzyme SET sys_name = '[Protein]-S(8)-aminomethyldihydrolipoyllysine:tetrahydrofolate aminomethyltransferase (ammonia-forming)' WHERE ec = 'EC 2.1.2.10';
UPDATE enzyme SET history = 'EC 2.1.2.10 created 1972, modified 2003, modified 2006' WHERE ec = 'EC 2.1.2.10';
UPDATE enzyme SET sys_name = '5,10-methylenetetrahydrofolate:3-methyl-2-oxobutanoate hydroxymethyltransferase' WHERE ec = 'EC 2.1.2.11';
UPDATE enzyme SET history = 'EC 2.1.2.11 created 1982' WHERE ec = 'EC 2.1.2.11';
UPDATE enzyme SET history = 'EC 2.1.2.12 created 1983, deleted 1984' WHERE ec = 'EC 2.1.2.12';
UPDATE enzyme SET sys_name = '10-formyltetrahydrofolate:UDP-4-amino-4-deoxy-beta-L-arabinose N-formyltransferase' WHERE ec = 'EC 2.1.2.13';
UPDATE enzyme SET history = 'EC 2.1.2.13 created 2010' WHERE ec = 'EC 2.1.2.13';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.1.2.n1';
UPDATE enzyme SET sys_name = '(S)-methylmalonyl-CoA:pyruvate carboxytransferase' WHERE ec = 'EC 2.1.3.1';
UPDATE enzyme SET history = 'EC 2.1.3.1 created 1961, modified 2004' WHERE ec = 'EC 2.1.3.1';
UPDATE enzyme SET sys_name = 'Carbamoyl-phosphate:L-aspartate carbamoyltransferase' WHERE ec = 'EC 2.1.3.2';
UPDATE enzyme SET history = 'EC 2.1.3.2 created 1961' WHERE ec = 'EC 2.1.3.2';
UPDATE enzyme SET sys_name = 'Carbamoyl-phosphate:L-ornithine carbamoyltransferase' WHERE ec = 'EC 2.1.3.3';
UPDATE enzyme SET history = 'EC 2.1.3.3 created 1961' WHERE ec = 'EC 2.1.3.3';
UPDATE enzyme SET history = 'EC 2.1.3.4 created 1965, deleted 1972' WHERE ec = 'EC 2.1.3.4';
UPDATE enzyme SET sys_name = 'Carbamoyl-phosphate:oxamate carbamoyltransferase' WHERE ec = 'EC 2.1.3.5';
UPDATE enzyme SET history = 'EC 2.1.3.5 created 1976' WHERE ec = 'EC 2.1.3.5';
UPDATE enzyme SET sys_name = 'Carbamoyl-phosphate:putrescine carbamoyltransferase' WHERE ec = 'EC 2.1.3.6';
UPDATE enzyme SET history = 'EC 2.1.3.6 created 1976' WHERE ec = 'EC 2.1.3.6';
UPDATE enzyme SET sys_name = 'Carbamoyl-phosphate:3-hydroxymethylceph-3-em-4-carboxylate carbamoyltransferase' WHERE ec = 'EC 2.1.3.7';
UPDATE enzyme SET history = 'EC 2.1.3.7 created 1983' WHERE ec = 'EC 2.1.3.7';
UPDATE enzyme SET sys_name = 'Carbamoyl-phosphate:L-lysine carbamoyltransferase' WHERE ec = 'EC 2.1.3.8';
UPDATE enzyme SET history = 'EC 2.1.3.8 created 1986' WHERE ec = 'EC 2.1.3.8';
UPDATE enzyme SET sys_name = 'Carbamoyl-phosphate:N(2)-acetyl-L-ornithine carbamoyltransferase' WHERE ec = 'EC 2.1.3.9';
UPDATE enzyme SET history = 'EC 2.1.3.9 created 2005' WHERE ec = 'EC 2.1.3.9';
UPDATE enzyme SET sys_name = 'Malonyl-[acyl-carrier protein]:biotinyl-[protein] carboxytransferase' WHERE ec = 'EC 2.1.3.10';
UPDATE enzyme SET history = 'EC 2.1.3.10 created 2008' WHERE ec = 'EC 2.1.3.10';
UPDATE enzyme SET sys_name = 'Carbamoyl phosphate:N(2)-succinyl-L-ornithine carbamoyltransferase' WHERE ec = 'EC 2.1.3.11';
UPDATE enzyme SET history = 'EC 2.1.3.11 created 2008' WHERE ec = 'EC 2.1.3.11';
UPDATE enzyme SET sys_name = 'Carbamoyl phosphate:decarbamoylnovobiocin 3-O-carbamoyltransferase' WHERE ec = 'EC 2.1.3.12';
UPDATE enzyme SET history = 'EC 2.1.3.12 created 2013' WHERE ec = 'EC 2.1.3.12';
UPDATE enzyme SET sys_name = 'Carbamoyl phosphate:ATP carbamoyl-transferase' WHERE ec = 'EC 2.1.3.13';
UPDATE enzyme SET history = 'EC 2.1.3.13 created 2013' WHERE ec = 'EC 2.1.3.13';
UPDATE enzyme SET sys_name = 'Carbamoyl phosphate:tobramycin 6-O-carbamoyl-transferase' WHERE ec = 'EC 2.1.3.14';
UPDATE enzyme SET history = 'EC 2.1.3.14 created 2013' WHERE ec = 'EC 2.1.3.14';
UPDATE enzyme SET sys_name = 'L-arginine:glycine amidinotransferase' WHERE ec = 'EC 2.1.4.1';
UPDATE enzyme SET history = 'EC 2.1.4.1 created 1961 as EC 2.6.2.1, transferred 1965 to EC 2.1.4.1' WHERE ec = 'EC 2.1.4.1';
UPDATE enzyme SET sys_name = 'L-arginine:1-amino-1-deoxy-scyllo-inositol-4-phosphate amidinotransferase' WHERE ec = 'EC 2.1.4.2';
UPDATE enzyme SET history = 'EC 2.1.4.2 created 1976, modified 2001' WHERE ec = 'EC 2.1.4.2';
UPDATE enzyme SET sys_name = 'Sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glycolaldehydetransferase' WHERE ec = 'EC 2.2.1.1';
UPDATE enzyme SET history = 'EC 2.2.1.1 created 1961' WHERE ec = 'EC 2.2.1.1';
UPDATE enzyme SET sys_name = 'Sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase' WHERE ec = 'EC 2.2.1.2';
UPDATE enzyme SET history = 'EC 2.2.1.2 created 1961' WHERE ec = 'EC 2.2.1.2';
UPDATE enzyme SET sys_name = 'D-xylulose-5-phosphate:formaldehyde glycolaldehydetransferase' WHERE ec = 'EC 2.2.1.3';
UPDATE enzyme SET history = 'EC 2.2.1.3 created 1984' WHERE ec = 'EC 2.2.1.3';
UPDATE enzyme SET sys_name = '3-hydroxybutan-2-one:D-ribose-5-phosphate aldehydetransferase' WHERE ec = 'EC 2.2.1.4';
UPDATE enzyme SET history = 'EC 2.2.1.4 created 1989' WHERE ec = 'EC 2.2.1.4';
UPDATE enzyme SET sys_name = '2-oxoglutarate:glyoxylate succinaldehydetransferase (decarboxylating)' WHERE ec = 'EC 2.2.1.5';
UPDATE enzyme SET history = 'EC 2.2.1.5 created 1972 as EC 4.1.3.15, transferred 2002 to EC 2.2.1.5' WHERE ec = 'EC 2.2.1.5';
UPDATE enzyme SET sys_name = 'Pyruvate:pyruvate acetaldehydetransferase (decarboxylating)' WHERE ec = 'EC 2.2.1.6';
UPDATE enzyme SET history = 'EC 2.2.1.6 created 1972 as EC 4.1.3.18, transferred 2002 to EC 2.2.1.6' WHERE ec = 'EC 2.2.1.6';
UPDATE enzyme SET sys_name = 'Pyruvate:D-glyceraldehyde-3-phosphate acetaldehydetransferase (decarboxylating)' WHERE ec = 'EC 2.2.1.7';
UPDATE enzyme SET history = 'EC 2.2.1.7 created 2001 as EC 4.1.3.37, transferred 2002 to EC 2.2.1.7' WHERE ec = 'EC 2.2.1.7';
UPDATE enzyme SET sys_name = 'Fluoroacetaldehyde:L-threonine aldehydetransferase' WHERE ec = 'EC 2.2.1.8';
UPDATE enzyme SET history = 'EC 2.2.1.8 created 2003' WHERE ec = 'EC 2.2.1.8';
UPDATE enzyme SET sys_name = 'Isochorismate:2-oxoglutarate 4-oxopentanoatetransferase (decarboxylating)' WHERE ec = 'EC 2.2.1.9';
UPDATE enzyme SET history = 'EC 2.2.1.9 created 2008 (EC 2.5.1.64 created 2003, part-incorporated 2008)' WHERE ec = 'EC 2.2.1.9';
UPDATE enzyme SET sys_name = 'L-aspartate 4-semialdehyde:1-deoxy-D-threo-hexo-2,5-diulose 6-phosphate methylglyoxaltransferase' WHERE ec = 'EC 2.2.1.10';
UPDATE enzyme SET history = 'EC 2.2.1.10 created 2012' WHERE ec = 'EC 2.2.1.10';
UPDATE enzyme SET sys_name = '2-oxopropanal:D-fructose 1,6-bisphosphate glycerone-phosphatetransferase' WHERE ec = 'EC 2.2.1.11';
UPDATE enzyme SET history = 'EC 2.2.1.11 created 2012' WHERE ec = 'EC 2.2.1.11';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:L-glutamate N-acetyltransferase' WHERE ec = 'EC 2.3.1.1';
UPDATE enzyme SET history = 'EC 2.3.1.1 created 1961' WHERE ec = 'EC 2.3.1.1';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:imidazole N-acetyltransferase' WHERE ec = 'EC 2.3.1.2';
UPDATE enzyme SET history = 'EC 2.3.1.2 created 1961' WHERE ec = 'EC 2.3.1.2';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:D-glucosamine N-acetyltransferase' WHERE ec = 'EC 2.3.1.3';
UPDATE enzyme SET history = 'EC 2.3.1.3 created 1961' WHERE ec = 'EC 2.3.1.3';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:D-glucosamine-6-phosphate N-acetyltransferase' WHERE ec = 'EC 2.3.1.4';
UPDATE enzyme SET history = 'EC 2.3.1.4 created 1961, modified 2002' WHERE ec = 'EC 2.3.1.4';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:arylamine N-acetyltransferase' WHERE ec = 'EC 2.3.1.5';
UPDATE enzyme SET history = 'EC 2.3.1.5 created 1961' WHERE ec = 'EC 2.3.1.5';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:choline O-acetyltransferase' WHERE ec = 'EC 2.3.1.6';
UPDATE enzyme SET history = 'EC 2.3.1.6 created 1961' WHERE ec = 'EC 2.3.1.6';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:carnitine O-acetyltransferase' WHERE ec = 'EC 2.3.1.7';
UPDATE enzyme SET history = 'EC 2.3.1.7 created 1961' WHERE ec = 'EC 2.3.1.7';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:phosphate acetyltransferase' WHERE ec = 'EC 2.3.1.8';
UPDATE enzyme SET history = 'EC 2.3.1.8 created 1961, modified 1976' WHERE ec = 'EC 2.3.1.8';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:acetyl-CoA C-acetyltransferase' WHERE ec = 'EC 2.3.1.9';
UPDATE enzyme SET history = 'EC 2.3.1.9 created 1961' WHERE ec = 'EC 2.3.1.9';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:hydrogen-sulfide S-acetyltransferase' WHERE ec = 'EC 2.3.1.10';
UPDATE enzyme SET history = 'EC 2.3.1.10 created 1961' WHERE ec = 'EC 2.3.1.10';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:2-aminoethanethiol S-acetyltransferase' WHERE ec = 'EC 2.3.1.11';
UPDATE enzyme SET history = 'EC 2.3.1.11 created 1961, modified 2006' WHERE ec = 'EC 2.3.1.11';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:enzyme-N(6)-(dihydrolipoyl)lysine S-acetyltransferase' WHERE ec = 'EC 2.3.1.12';
UPDATE enzyme SET history = 'EC 2.3.1.12 created 1961, modified 2003' WHERE ec = 'EC 2.3.1.12';
UPDATE enzyme SET sys_name = 'Acyl-CoA:glycine N-acyltransferase' WHERE ec = 'EC 2.3.1.13';
UPDATE enzyme SET history = 'EC 2.3.1.13 created 1961' WHERE ec = 'EC 2.3.1.13';
UPDATE enzyme SET sys_name = 'Phenylacetyl-CoA:L-glutamine alpha-N-phenylacetyltransferase' WHERE ec = 'EC 2.3.1.14';
UPDATE enzyme SET history = 'EC 2.3.1.14 created 1961' WHERE ec = 'EC 2.3.1.14';
UPDATE enzyme SET sys_name = 'Acyl-CoA:sn-glycerol-3-phosphate 1-O-acyltransferase' WHERE ec = 'EC 2.3.1.15';
UPDATE enzyme SET history = 'EC 2.3.1.15 created 1961, modified 1976, modified 1990' WHERE ec = 'EC 2.3.1.15';
UPDATE enzyme SET sys_name = 'Acyl-CoA:acetyl-CoA C-acyltransferase' WHERE ec = 'EC 2.3.1.16';
UPDATE enzyme SET history = 'EC 2.3.1.16 created 1961' WHERE ec = 'EC 2.3.1.16';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:L-aspartate N-acetyltransferase' WHERE ec = 'EC 2.3.1.17';
UPDATE enzyme SET history = 'EC 2.3.1.17 created 1965' WHERE ec = 'EC 2.3.1.17';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:beta-D-galactoside 6-acetyltransferase' WHERE ec = 'EC 2.3.1.18';
UPDATE enzyme SET history = 'EC 2.3.1.18 created 1965' WHERE ec = 'EC 2.3.1.18';
UPDATE enzyme SET sys_name = 'Butanoyl-CoA:phosphate butanoyltransferase' WHERE ec = 'EC 2.3.1.19';
UPDATE enzyme SET history = 'EC 2.3.1.19 created 1965' WHERE ec = 'EC 2.3.1.19';
UPDATE enzyme SET sys_name = 'Acyl-CoA:1,2-diacyl-sn-glycerol O-acyltransferase' WHERE ec = 'EC 2.3.1.20';
UPDATE enzyme SET history = 'EC 2.3.1.20 created 1965' WHERE ec = 'EC 2.3.1.20';
UPDATE enzyme SET sys_name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase' WHERE ec = 'EC 2.3.1.21';
UPDATE enzyme SET history = 'EC 2.3.1.21 created 1972' WHERE ec = 'EC 2.3.1.21';
UPDATE enzyme SET sys_name = 'Acyl-CoA:2-acylglycerol O-acyltransferase' WHERE ec = 'EC 2.3.1.22';
UPDATE enzyme SET history = 'EC 2.3.1.22 created 1972, modified 1986, modified 1989' WHERE ec = 'EC 2.3.1.22';
UPDATE enzyme SET sys_name = 'Acyl-CoA:1-acyl-sn-glycero-3-phosphocholine O-acyltransferase' WHERE ec = 'EC 2.3.1.23';
UPDATE enzyme SET history = 'EC 2.3.1.23 created 1972' WHERE ec = 'EC 2.3.1.23';
UPDATE enzyme SET sys_name = 'Acyl-CoA:sphingosine N-acyltransferase' WHERE ec = 'EC 2.3.1.24';
UPDATE enzyme SET history = 'EC 2.3.1.24 created 1972' WHERE ec = 'EC 2.3.1.24';
UPDATE enzyme SET sys_name = 'Acyl-CoA:1-O-(alk-1-enyl)-glycero-3-phosphocholine 2-O-acyltransferase' WHERE ec = 'EC 2.3.1.25';
UPDATE enzyme SET history = 'EC 2.3.1.25 created 1972, modified 2013' WHERE ec = 'EC 2.3.1.25';
UPDATE enzyme SET sys_name = 'Acyl-CoA:cholesterol O-acyltransferase' WHERE ec = 'EC 2.3.1.26';
UPDATE enzyme SET history = 'EC 2.3.1.26 created 1972' WHERE ec = 'EC 2.3.1.26';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:cortisol O-acetyltransferase' WHERE ec = 'EC 2.3.1.27';
UPDATE enzyme SET history = 'EC 2.3.1.27 created 1972' WHERE ec = 'EC 2.3.1.27';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:chloramphenicol 3-O-acetyltransferase' WHERE ec = 'EC 2.3.1.28';
UPDATE enzyme SET history = 'EC 2.3.1.28 created 1972' WHERE ec = 'EC 2.3.1.28';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:glycine C-acetyltransferase' WHERE ec = 'EC 2.3.1.29';
UPDATE enzyme SET history = 'EC 2.3.1.29 created 1972' WHERE ec = 'EC 2.3.1.29';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:L-serine O-acetyltransferase' WHERE ec = 'EC 2.3.1.30';
UPDATE enzyme SET history = 'EC 2.3.1.30 created 1972' WHERE ec = 'EC 2.3.1.30';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:L-homoserine O-acetyltransferase' WHERE ec = 'EC 2.3.1.31';
UPDATE enzyme SET history = 'EC 2.3.1.31 created 1972' WHERE ec = 'EC 2.3.1.31';
UPDATE enzyme SET sys_name = 'Acetyl-phosphate:L-lysine N(6)-acetyltransferase' WHERE ec = 'EC 2.3.1.32';
UPDATE enzyme SET history = 'EC 2.3.1.32 created 1972' WHERE ec = 'EC 2.3.1.32';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:L-histidine N-acetyltransferase' WHERE ec = 'EC 2.3.1.33';
UPDATE enzyme SET history = 'EC 2.3.1.33 created 1972' WHERE ec = 'EC 2.3.1.33';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:D-tryptophan N-acetyltransferase' WHERE ec = 'EC 2.3.1.34';
UPDATE enzyme SET history = 'EC 2.3.1.34 created 1972' WHERE ec = 'EC 2.3.1.34';
UPDATE enzyme SET sys_name = 'N(2)-acetyl-L-ornithine:L-glutamate N-acetyltransferase' WHERE ec = 'EC 2.3.1.35';
UPDATE enzyme SET history = 'EC 2.3.1.35 created 1972' WHERE ec = 'EC 2.3.1.35';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:D-amino-acid N-acetyltransferase' WHERE ec = 'EC 2.3.1.36';
UPDATE enzyme SET history = 'EC 2.3.1.36 created 1972' WHERE ec = 'EC 2.3.1.36';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:glycine C-succinyltransferase (decarboxylating)' WHERE ec = 'EC 2.3.1.37';
UPDATE enzyme SET history = 'EC 2.3.1.37 created 1972' WHERE ec = 'EC 2.3.1.37';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:[acyl-carrier-protein] S-acetyltransferase' WHERE ec = 'EC 2.3.1.38';
UPDATE enzyme SET history = 'EC 2.3.1.38 created 1972, modified 2006' WHERE ec = 'EC 2.3.1.38';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:[acyl-carrier protein] S-malonyltransferase' WHERE ec = 'EC 2.3.1.39';
UPDATE enzyme SET history = 'EC 2.3.1.39 created 1972, modified 2006, modified 2008' WHERE ec = 'EC 2.3.1.39';
UPDATE enzyme SET sys_name = 'Acyl-[acyl-carrier-protein]:O-(2-acyl-sn-glycero-3-phospho)ethanolamine O-acyltransferase' WHERE ec = 'EC 2.3.1.40';
UPDATE enzyme SET history = 'EC 2.3.1.40 created 1972' WHERE ec = 'EC 2.3.1.40';
UPDATE enzyme SET sys_name = 'Acyl-[acyl-carrier protein]:malonyl-[acyl-carrier protein] C-acyltransferase (decarboxylating)' WHERE ec = 'EC 2.3.1.41';
UPDATE enzyme SET history = 'EC 2.3.1.41 created 1972, modified 2006' WHERE ec = 'EC 2.3.1.41';
UPDATE enzyme SET sys_name = 'Acyl-CoA:glycerone-phosphate O-acyltransferase' WHERE ec = 'EC 2.3.1.42';
UPDATE enzyme SET history = 'EC 2.3.1.42 created 1972' WHERE ec = 'EC 2.3.1.42';
UPDATE enzyme SET sys_name = 'Phosphatidylcholine:sterol O-acyltransferase' WHERE ec = 'EC 2.3.1.43';
UPDATE enzyme SET history = 'EC 2.3.1.43 created 1972, modified 1976' WHERE ec = 'EC 2.3.1.43';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:N-acetylneuraminate 4-O-acetyltransferase' WHERE ec = 'EC 2.3.1.44';
UPDATE enzyme SET history = 'EC 2.3.1.44 created 1972' WHERE ec = 'EC 2.3.1.44';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:N-acetylneuraminate 7-O(or 9-O)-acetyltransferase' WHERE ec = 'EC 2.3.1.45';
UPDATE enzyme SET history = 'EC 2.3.1.45 created 1972' WHERE ec = 'EC 2.3.1.45';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:L-homoserine O-succinyltransferase' WHERE ec = 'EC 2.3.1.46';
UPDATE enzyme SET history = 'EC 2.3.1.46 created 1976' WHERE ec = 'EC 2.3.1.46';
UPDATE enzyme SET sys_name = '6-carboxyhexanoyl-[acyl-carrier protein]:L-alanine C-carboxyhexanoyltransferase (decarboxylating)' WHERE ec = 'EC 2.3.1.47';
UPDATE enzyme SET history = 'EC 2.3.1.47 created 1976, modified 2013' WHERE ec = 'EC 2.3.1.47';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:histone acetyltransferase' WHERE ec = 'EC 2.3.1.48';
UPDATE enzyme SET history = 'EC 2.3.1.48 created 1976' WHERE ec = 'EC 2.3.1.48';
UPDATE enzyme SET sys_name = 'S-acetylphosphopantetheine:deacetyl-[citrate-oxaloacetate-lyase((pro-3S)-CH(2)COO(-)-&gt;acetate)] S-acetyltransferase' WHERE ec = 'EC 2.3.1.49';
UPDATE enzyme SET history = 'EC 2.3.1.49 created 1976' WHERE ec = 'EC 2.3.1.49';
UPDATE enzyme SET sys_name = 'Palmitoyl-CoA:L-serine C-palmitoyltransferase (decarboxylating)' WHERE ec = 'EC 2.3.1.50';
UPDATE enzyme SET history = 'EC 2.3.1.50 created 1976, modified 1982' WHERE ec = 'EC 2.3.1.50';
UPDATE enzyme SET sys_name = 'Acyl-CoA:1-acyl-sn-glycerol-3-phosphate 2-O-acyltransferase' WHERE ec = 'EC 2.3.1.51';
UPDATE enzyme SET history = 'EC 2.3.1.51 created 1976, modified 1990' WHERE ec = 'EC 2.3.1.51';
UPDATE enzyme SET sys_name = 'Acyl-CoA:2-acyl-sn-glycerol 3-phosphate O-acyltransferase' WHERE ec = 'EC 2.3.1.52';
UPDATE enzyme SET history = 'EC 2.3.1.52 created 1976' WHERE ec = 'EC 2.3.1.52';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:L-phenylalanine N-acetyltransferase' WHERE ec = 'EC 2.3.1.53';
UPDATE enzyme SET history = 'EC 2.3.1.53 created 1976' WHERE ec = 'EC 2.3.1.53';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:formate C-acetyltransferase' WHERE ec = 'EC 2.3.1.54';
UPDATE enzyme SET history = 'EC 2.3.1.54 created 1976' WHERE ec = 'EC 2.3.1.54';
UPDATE enzyme SET history = 'EC 2.3.1.55 created 1976, deleted 1999' WHERE ec = 'EC 2.3.1.55';
UPDATE enzyme SET sys_name = 'N-hydroxy-4-acetylaminobiphenyl:N-hydroxy-4-aminobiphenyl O-acetyltransferase' WHERE ec = 'EC 2.3.1.56';
UPDATE enzyme SET history = 'EC 2.3.1.56 created 1976' WHERE ec = 'EC 2.3.1.56';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:alkane-alpha,omega-diamine N-acetyltransferase' WHERE ec = 'EC 2.3.1.57';
UPDATE enzyme SET history = 'EC 2.3.1.57 created 1976, modified 1989' WHERE ec = 'EC 2.3.1.57';
UPDATE enzyme SET sys_name = 'Oxalyl-CoA:L-2,3-diaminopropanoate N(3)-oxalyltransferase' WHERE ec = 'EC 2.3.1.58';
UPDATE enzyme SET history = 'EC 2.3.1.58 created 1976' WHERE ec = 'EC 2.3.1.58';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:gentamicin-C(1a) N(2)-acetyltransferase' WHERE ec = 'EC 2.3.1.59';
UPDATE enzyme SET history = 'EC 2.3.1.59 created 1976' WHERE ec = 'EC 2.3.1.59';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:gentamicin-C N(3)-acetyltransferase' WHERE ec = 'EC 2.3.1.60';
UPDATE enzyme SET history = 'EC 2.3.1.60 created 1976' WHERE ec = 'EC 2.3.1.60';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:enzyme-N(6)-(dihydrolipoyl)lysine S-succinyltransferase' WHERE ec = 'EC 2.3.1.61';
UPDATE enzyme SET history = 'EC 2.3.1.61 created 1978, modified 2003' WHERE ec = 'EC 2.3.1.61';
UPDATE enzyme SET sys_name = 'Acyl-CoA:2-acyl-sn-glycero-3-phosphocholine O-acyltransferase' WHERE ec = 'EC 2.3.1.62';
UPDATE enzyme SET history = 'EC 2.3.1.62 created 1978' WHERE ec = 'EC 2.3.1.62';
UPDATE enzyme SET sys_name = 'Acyl-CoA:1-alkyl-sn-glycero-3-phosphocholine O-acyltransferase' WHERE ec = 'EC 2.3.1.63';
UPDATE enzyme SET history = 'EC 2.3.1.63 created 1978' WHERE ec = 'EC 2.3.1.63';
UPDATE enzyme SET sys_name = '4-coumaroyl-CoA:agmatine N(4)-coumaroyltransferase' WHERE ec = 'EC 2.3.1.64';
UPDATE enzyme SET history = 'EC 2.3.1.64 created 1983' WHERE ec = 'EC 2.3.1.64';
UPDATE enzyme SET sys_name = 'Choloyl-CoA:glycine N-choloyltransferase' WHERE ec = 'EC 2.3.1.65';
UPDATE enzyme SET history = 'EC 2.3.1.65 created 1983, modified 2005' WHERE ec = 'EC 2.3.1.65';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:L-leucine N-acetyltransferase' WHERE ec = 'EC 2.3.1.66';
UPDATE enzyme SET history = 'EC 2.3.1.66 created 1983' WHERE ec = 'EC 2.3.1.66';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:1-alkyl-sn-glycero-3-phosphocholine 2-O-acetyltransferase' WHERE ec = 'EC 2.3.1.67';
UPDATE enzyme SET history = 'EC 2.3.1.67 created 1984' WHERE ec = 'EC 2.3.1.67';
UPDATE enzyme SET sys_name = 'Acyl-CoA:L-glutamine N-acyltransferase' WHERE ec = 'EC 2.3.1.68';
UPDATE enzyme SET history = 'EC 2.3.1.68 created 1984' WHERE ec = 'EC 2.3.1.68';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:monoterpenol O-acetyltransferase' WHERE ec = 'EC 2.3.1.69';
UPDATE enzyme SET history = 'EC 2.3.1.69 created 1984' WHERE ec = 'EC 2.3.1.69';
UPDATE enzyme SET history = 'EC 2.3.1.70 created 1984, deleted 2009' WHERE ec = 'EC 2.3.1.70';
UPDATE enzyme SET sys_name = 'Benzoyl-CoA:glycine N-benzoyltransferase' WHERE ec = 'EC 2.3.1.71';
UPDATE enzyme SET history = 'EC 2.3.1.71 created 1984' WHERE ec = 'EC 2.3.1.71';
UPDATE enzyme SET sys_name = '1-O-(indol-3-yl)acetyl-beta-D-glucose:myo-inositol (indol-3-yl)acetyltransferase' WHERE ec = 'EC 2.3.1.72';
UPDATE enzyme SET history = 'EC 2.3.1.72 created 1984, modified 2003' WHERE ec = 'EC 2.3.1.72';
UPDATE enzyme SET sys_name = '1,2-diacyl-sn-glycerol:sterol O-acyltransferase' WHERE ec = 'EC 2.3.1.73';
UPDATE enzyme SET history = 'EC 2.3.1.73 created 1984' WHERE ec = 'EC 2.3.1.73';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:4-coumaroyl-CoA malonyltransferase (cyclizing)' WHERE ec = 'EC 2.3.1.74';
UPDATE enzyme SET history = 'EC 2.3.1.74 created 1984' WHERE ec = 'EC 2.3.1.74';
UPDATE enzyme SET sys_name = 'Acyl-CoA:long-chain-alcohol O-acyltransferase' WHERE ec = 'EC 2.3.1.75';
UPDATE enzyme SET history = 'EC 2.3.1.75 created 1984' WHERE ec = 'EC 2.3.1.75';
UPDATE enzyme SET sys_name = 'Acyl-CoA:retinol O-acyltransferase' WHERE ec = 'EC 2.3.1.76';
UPDATE enzyme SET history = 'EC 2.3.1.76 created 1984' WHERE ec = 'EC 2.3.1.76';
UPDATE enzyme SET sys_name = 'Triacylglycerol:3-beta-hydroxysteroid O-acyltransferase' WHERE ec = 'EC 2.3.1.77';
UPDATE enzyme SET history = 'EC 2.3.1.77 created 1984' WHERE ec = 'EC 2.3.1.77';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:heparan-alpha-D-glucosaminide N-acetyltransferase' WHERE ec = 'EC 2.3.1.78';
UPDATE enzyme SET history = 'EC 2.3.1.78 created 1984' WHERE ec = 'EC 2.3.1.78';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:maltose O-acetyltransferase' WHERE ec = 'EC 2.3.1.79';
UPDATE enzyme SET history = 'EC 2.3.1.79 created 1984' WHERE ec = 'EC 2.3.1.79';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:S-substituted L-cysteine N-acetyltransferase' WHERE ec = 'EC 2.3.1.80';
UPDATE enzyme SET history = 'EC 2.3.1.80 created 1984' WHERE ec = 'EC 2.3.1.80';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:2-deoxystreptamine-antibiotic N(3)-acetyltransferase' WHERE ec = 'EC 2.3.1.81';
UPDATE enzyme SET history = 'EC 2.3.1.81 created 1984' WHERE ec = 'EC 2.3.1.81';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:kanamycin-B N(6)-acetyltransferase' WHERE ec = 'EC 2.3.1.82';
UPDATE enzyme SET history = 'EC 2.3.1.82 created 1976 as EC 2.3.1.55, transferred 1999 to EC 2.3.1.82, modified 1999' WHERE ec = 'EC 2.3.1.82';
UPDATE enzyme SET sys_name = '3-sn-phosphatidylcholine:dolichol O-acyltransferase' WHERE ec = 'EC 2.3.1.83';
UPDATE enzyme SET history = 'EC 2.3.1.83 created 1984' WHERE ec = 'EC 2.3.1.83';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:alcohol O-acetyltransferase' WHERE ec = 'EC 2.3.1.84';
UPDATE enzyme SET history = 'EC 2.3.1.84 created 1984' WHERE ec = 'EC 2.3.1.84';
UPDATE enzyme SET sys_name = 'Acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolyzing)' WHERE ec = 'EC 2.3.1.85';
UPDATE enzyme SET history = 'EC 2.3.1.85 created 1984' WHERE ec = 'EC 2.3.1.85';
UPDATE enzyme SET sys_name = 'Acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl- reducing)' WHERE ec = 'EC 2.3.1.86';
UPDATE enzyme SET history = 'EC 2.3.1.86 created 1984, modified 2003, modified 2013' WHERE ec = 'EC 2.3.1.86';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:2-arylethylamine N-acetyltransferase' WHERE ec = 'EC 2.3.1.87';
UPDATE enzyme SET history = 'EC 2.3.1.87 created 1986, modified 2005' WHERE ec = 'EC 2.3.1.87';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:peptide N(alpha)-acetyltransferase' WHERE ec = 'EC 2.3.1.88';
UPDATE enzyme SET history = 'EC 2.3.1.88 created 1986, modified 1989' WHERE ec = 'EC 2.3.1.88';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:(S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N(2)-acetyltransferase' WHERE ec = 'EC 2.3.1.89';
UPDATE enzyme SET history = 'EC 2.3.1.89 created 1986' WHERE ec = 'EC 2.3.1.89';
UPDATE enzyme SET sys_name = '1-O-galloyl-beta-D-glucose:1-O-galloyl-beta-D-glucose O-galloyltransferase' WHERE ec = 'EC 2.3.1.90';
UPDATE enzyme SET history = 'EC 2.3.1.90 created 1986' WHERE ec = 'EC 2.3.1.90';
UPDATE enzyme SET sys_name = '1-O-sinnapoyl-beta-D-glucose:choline 1-O-sinnapoyltransferase' WHERE ec = 'EC 2.3.1.91';
UPDATE enzyme SET history = 'EC 2.3.1.91 created 1986' WHERE ec = 'EC 2.3.1.91';
UPDATE enzyme SET sys_name = '1-O-sinapoyl-beta-D-glucose:(S)-malate O-sinapoyltransferase' WHERE ec = 'EC 2.3.1.92';
UPDATE enzyme SET history = 'EC 2.3.1.92 created 1986' WHERE ec = 'EC 2.3.1.92';
UPDATE enzyme SET sys_name = '(E)-2-methylcrotonoyl-CoA:13-hydroxylupanine O-2-methylcrotonoyltransferase' WHERE ec = 'EC 2.3.1.93';
UPDATE enzyme SET history = 'EC 2.3.1.93 created 1986, modified 2011' WHERE ec = 'EC 2.3.1.93';
UPDATE enzyme SET sys_name = 'Propanoyl-CoA:(2S)-methylmalonyl-CoA malonyltransferase (cyclizing)' WHERE ec = 'EC 2.3.1.94';
UPDATE enzyme SET history = 'EC 2.3.1.94 created 1989, modified 2008' WHERE ec = 'EC 2.3.1.94';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:4-coumaroyl-CoA malonyltransferase (cyclizing)' WHERE ec = 'EC 2.3.1.95';
UPDATE enzyme SET history = 'EC 2.3.1.95 created 1989' WHERE ec = 'EC 2.3.1.95';
UPDATE enzyme SET sys_name = 'Palmitoyl-CoA:glycoprotein N-palmitoyltransferase' WHERE ec = 'EC 2.3.1.96';
UPDATE enzyme SET history = 'EC 2.3.1.96 created 1989' WHERE ec = 'EC 2.3.1.96';
UPDATE enzyme SET sys_name = 'Tetradecanoyl-CoA:glycylpeptide N-tetradecanoyltransferase' WHERE ec = 'EC 2.3.1.97';
UPDATE enzyme SET history = 'EC 2.3.1.97 created 1989, modified 1990' WHERE ec = 'EC 2.3.1.97';
UPDATE enzyme SET sys_name = 'Chlorogenate:glucarate O-(hydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.98';
UPDATE enzyme SET history = 'EC 2.3.1.98 created 1989, modified 1990' WHERE ec = 'EC 2.3.1.98';
UPDATE enzyme SET sys_name = 'Feruloyl-CoA:quinate O-(hydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.99';
UPDATE enzyme SET history = 'EC 2.3.1.99 created 1989, modified 1990' WHERE ec = 'EC 2.3.1.99';
UPDATE enzyme SET sys_name = 'Palmitoyl-CoA:[myelin-proteolipid] O-palmitoyltransferase' WHERE ec = 'EC 2.3.1.100';
UPDATE enzyme SET history = 'EC 2.3.1.100 created 1989' WHERE ec = 'EC 2.3.1.100';
UPDATE enzyme SET sys_name = 'Formylmethanofuran:5,6,7,8-tetrahydromethanopterin 5-formyltransferase' WHERE ec = 'EC 2.3.1.101';
UPDATE enzyme SET history = 'EC 2.3.1.101 created 1989' WHERE ec = 'EC 2.3.1.101';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:N(6)-hydroxy-L-lysine 6-acetyltransferase' WHERE ec = 'EC 2.3.1.102';
UPDATE enzyme SET history = 'EC 2.3.1.102 created 1989' WHERE ec = 'EC 2.3.1.102';
UPDATE enzyme SET sys_name = '1-O-(4-hydroxy-3,5-dimethoxycinnamoyl)-beta-D-glucoside:1-O-(4-hydroxy-3,5-dimethoxycinnamoyl-beta-D-glucoside 1-O-sinapoyltransferase' WHERE ec = 'EC 2.3.1.103';
UPDATE enzyme SET history = 'EC 2.3.1.103 created 1989' WHERE ec = 'EC 2.3.1.103';
UPDATE enzyme SET history = 'EC 2.3.1.104 created 1989, deleted 2013' WHERE ec = 'EC 2.3.1.104';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:1-alkyl-sn-glycero-3-phosphate 2-O-acetyltransferase' WHERE ec = 'EC 2.3.1.105';
UPDATE enzyme SET history = 'EC 2.3.1.105 created 1989' WHERE ec = 'EC 2.3.1.105';
UPDATE enzyme SET sys_name = 'Sinapoyl-CoA:2-hydroxymalonate O-(hydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.106';
UPDATE enzyme SET history = 'EC 2.3.1.106 created 1989, modified 1990, modified 2002' WHERE ec = 'EC 2.3.1.106';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:deacetylvindoline 4-O-acetyltransferase' WHERE ec = 'EC 2.3.1.107';
UPDATE enzyme SET history = 'EC 2.3.1.107 created 1989, modified 2005' WHERE ec = 'EC 2.3.1.107';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:[alpha-tubulin]-L-lysine N(6)-acetyltransferase' WHERE ec = 'EC 2.3.1.108';
UPDATE enzyme SET history = 'EC 2.3.1.108 created 1989' WHERE ec = 'EC 2.3.1.108';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:L-arginine N(2)-succinyltransferase' WHERE ec = 'EC 2.3.1.109';
UPDATE enzyme SET history = 'EC 2.3.1.109 created 1989, modified 2006' WHERE ec = 'EC 2.3.1.109';
UPDATE enzyme SET sys_name = 'Feruloyl-CoA:tyramine N-(hydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.110';
UPDATE enzyme SET history = 'EC 2.3.1.110 created 1989' WHERE ec = 'EC 2.3.1.110';
UPDATE enzyme SET sys_name = 'Acyl-CoA:methylmalonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing)' WHERE ec = 'EC 2.3.1.111';
UPDATE enzyme SET history = 'EC 2.3.1.111 created 1989' WHERE ec = 'EC 2.3.1.111';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:D-tryptophan N-malonyltransferase' WHERE ec = 'EC 2.3.1.112';
UPDATE enzyme SET history = 'EC 2.3.1.112 created 1989' WHERE ec = 'EC 2.3.1.112';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:anthranilate N-malonyltransferase' WHERE ec = 'EC 2.3.1.113';
UPDATE enzyme SET history = 'EC 2.3.1.113 created 1989' WHERE ec = 'EC 2.3.1.113';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:3,4-dichloroaniline N-malonyltransferase' WHERE ec = 'EC 2.3.1.114';
UPDATE enzyme SET history = 'EC 2.3.1.114 created 1989' WHERE ec = 'EC 2.3.1.114';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:isoflavone-7-O-beta-D-glucoside 6-O-malonyltransferase' WHERE ec = 'EC 2.3.1.115';
UPDATE enzyme SET history = 'EC 2.3.1.115 created 1989' WHERE ec = 'EC 2.3.1.115';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:flavonol-3-O-beta-D-glucoside 6-O-malonyltransferase' WHERE ec = 'EC 2.3.1.116';
UPDATE enzyme SET history = 'EC 2.3.1.116 created 1989' WHERE ec = 'EC 2.3.1.116';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:(S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase' WHERE ec = 'EC 2.3.1.117';
UPDATE enzyme SET history = 'EC 2.3.1.117 created 1989, modified 2001' WHERE ec = 'EC 2.3.1.117';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:N-hydroxyarylamine O-acetyltransferase' WHERE ec = 'EC 2.3.1.118';
UPDATE enzyme SET history = 'EC 2.3.1.118 created 1989' WHERE ec = 'EC 2.3.1.118';
UPDATE enzyme SET sys_name = 'Stearoyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing)' WHERE ec = 'EC 2.3.1.119';
UPDATE enzyme SET history = 'EC 2.3.1.119 created 1990' WHERE ec = 'EC 2.3.1.119';
UPDATE enzyme SET history = 'EC 2.3.1.120 created 1990, deleted 1992' WHERE ec = 'EC 2.3.1.120';
UPDATE enzyme SET sys_name = 'Acyl-CoA:1-alkenylglycerophosphoethanolamine O-acyltransferase' WHERE ec = 'EC 2.3.1.121';
UPDATE enzyme SET history = 'EC 2.3.1.121 created 1990' WHERE ec = 'EC 2.3.1.121';
UPDATE enzyme SET sys_name = 'Alpha,alpha-trehalose-6-mycolate:alpha,alpha-trehalose-6-mycolate 6-mycolyltransferase' WHERE ec = 'EC 2.3.1.122';
UPDATE enzyme SET history = 'EC 2.3.1.122 created 1990' WHERE ec = 'EC 2.3.1.122';
UPDATE enzyme SET sys_name = 'Palmitoyl-CoA:dolichol O-palmitoyltransferase' WHERE ec = 'EC 2.3.1.123';
UPDATE enzyme SET history = 'EC 2.3.1.123 created 1990' WHERE ec = 'EC 2.3.1.123';
UPDATE enzyme SET history = 'EC 2.3.1.124 created 1990, deleted 1992' WHERE ec = 'EC 2.3.1.124';
UPDATE enzyme SET sys_name = 'Acyl-CoA:1-O-alkyl-2-acetyl-sn-glycerol O-acyltransferase' WHERE ec = 'EC 2.3.1.125';
UPDATE enzyme SET history = 'EC 2.3.1.125 created 1990' WHERE ec = 'EC 2.3.1.125';
UPDATE enzyme SET sys_name = 'Caffeoyl-CoA:isocitrate 2-O-(3,4-dihydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.126';
UPDATE enzyme SET history = 'EC 2.3.1.126 created 1990' WHERE ec = 'EC 2.3.1.126';
UPDATE enzyme SET sys_name = 'Benzoyl-CoA:L-ornithine N-benzoyltransferase' WHERE ec = 'EC 2.3.1.127';
UPDATE enzyme SET history = 'EC 2.3.1.127 created 1990' WHERE ec = 'EC 2.3.1.127';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:ribosomal-protein-L-alanine N-acetyltransferase' WHERE ec = 'EC 2.3.1.128';
UPDATE enzyme SET history = 'EC 2.3.1.128 created 1990' WHERE ec = 'EC 2.3.1.128';
UPDATE enzyme SET sys_name = '(R)-3-hydroxymyristoyl-[acyl-carrier-protein]:UDP-N-acetyl-alpha-D-glucosamine 3-O-(3-hydroxymyristoyl)transferase' WHERE ec = 'EC 2.3.1.129';
UPDATE enzyme SET history = 'EC 2.3.1.129 created 1990' WHERE ec = 'EC 2.3.1.129';
UPDATE enzyme SET sys_name = 'Feruloyl-CoA:galactarate O-(hydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.130';
UPDATE enzyme SET history = 'EC 2.3.1.130 created 1990' WHERE ec = 'EC 2.3.1.130';
UPDATE enzyme SET sys_name = 'Sinapoyl-CoA:glucarate O-(hydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.131';
UPDATE enzyme SET history = 'EC 2.3.1.131 created 1990' WHERE ec = 'EC 2.3.1.131';
UPDATE enzyme SET sys_name = 'Sinapoyl-CoA:glucarolactone O-(hydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.132';
UPDATE enzyme SET history = 'EC 2.3.1.132 created 1990' WHERE ec = 'EC 2.3.1.132';
UPDATE enzyme SET sys_name = '4-coumaroyl-CoA:shikimate O-(hydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.133';
UPDATE enzyme SET history = 'EC 2.3.1.133 created 1990' WHERE ec = 'EC 2.3.1.133';
UPDATE enzyme SET sys_name = 'Mono-beta-D-galactosyldiacylglycerol:mono-beta-D-galactosyldiacylglycerol acyltransferase' WHERE ec = 'EC 2.3.1.134';
UPDATE enzyme SET history = 'EC 2.3.1.134 created 1990' WHERE ec = 'EC 2.3.1.134';
UPDATE enzyme SET sys_name = 'Phosphatidylcholine:retinol--(cellular-retinol-binding-protein) O-acyltransferase' WHERE ec = 'EC 2.3.1.135';
UPDATE enzyme SET history = 'EC 2.3.1.135 created 1992, modified 2011' WHERE ec = 'EC 2.3.1.135';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:polysialic-acid O-acetyltransferase' WHERE ec = 'EC 2.3.1.136';
UPDATE enzyme SET history = 'EC 2.3.1.136 created 1992' WHERE ec = 'EC 2.3.1.136';
UPDATE enzyme SET sys_name = 'Octanoyl-CoA:L-carnitine O-octanoyltransferase' WHERE ec = 'EC 2.3.1.137';
UPDATE enzyme SET history = 'EC 2.3.1.137 created 1992' WHERE ec = 'EC 2.3.1.137';
UPDATE enzyme SET sys_name = 'Caffeoyl-CoA:putrescine N-(3,4-dihydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.138';
UPDATE enzyme SET history = 'EC 2.3.1.138 created 1992' WHERE ec = 'EC 2.3.1.138';
UPDATE enzyme SET sys_name = 'Palmitoyl-CoA:ecdysone palmitoyltransferase' WHERE ec = 'EC 2.3.1.139';
UPDATE enzyme SET history = 'EC 2.3.1.139 created 1992' WHERE ec = 'EC 2.3.1.139';
UPDATE enzyme SET sys_name = 'Caffeoyl-CoA:(R)-3-(3,4-dihydroxyphenyl)lactate 2-O-caffeoyl-transferase' WHERE ec = 'EC 2.3.1.140';
UPDATE enzyme SET history = 'EC 2.3.1.140 created 1992, modified 2013' WHERE ec = 'EC 2.3.1.140';
UPDATE enzyme SET sys_name = 'Acyl-[acyl-carrier protein]:D-galactosylacylglycerol O-acyltransferase' WHERE ec = 'EC 2.3.1.141';
UPDATE enzyme SET history = 'EC 2.3.1.141 created 1992' WHERE ec = 'EC 2.3.1.141';
UPDATE enzyme SET sys_name = 'Fatty-acyl-CoA:mucus-glycoprotein fatty-acyltransferase' WHERE ec = 'EC 2.3.1.142';
UPDATE enzyme SET history = 'EC 2.3.1.142 created 1992' WHERE ec = 'EC 2.3.1.142';
UPDATE enzyme SET sys_name = '1-O-galloyl-beta-D-glucose:1,2,3,6-tetrakis-O-galloyl-beta-D-glucose 4-O-galloyltransferase' WHERE ec = 'EC 2.3.1.143';
UPDATE enzyme SET history = 'EC 2.3.1.143 created 1992' WHERE ec = 'EC 2.3.1.143';
UPDATE enzyme SET sys_name = 'Benzoyl-CoA:anthranilate N-benzoyltransferase' WHERE ec = 'EC 2.3.1.144';
UPDATE enzyme SET history = 'EC 2.3.1.144 created 1992' WHERE ec = 'EC 2.3.1.144';
UPDATE enzyme SET sys_name = '(E,E)-piperoyl-CoA:piperidine N-piperoyltransferase' WHERE ec = 'EC 2.3.1.145';
UPDATE enzyme SET history = 'EC 2.3.1.145 created 1992' WHERE ec = 'EC 2.3.1.145';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:cinnamoyl-CoA malonyltransferase (cyclizing)' WHERE ec = 'EC 2.3.1.146';
UPDATE enzyme SET history = 'EC 2.3.1.146 created 1992' WHERE ec = 'EC 2.3.1.146';
UPDATE enzyme SET sys_name = '1-organyl-2-arachidonoyl-sn-glycero-3-phosphocholine:1-organyl-2-lyso-sn-glycero-3-phosphoethanolamine arachidonoyltransferase (CoA-independent)' WHERE ec = 'EC 2.3.1.147';
UPDATE enzyme SET history = 'EC 2.3.1.147 created 1999' WHERE ec = 'EC 2.3.1.147';
UPDATE enzyme SET sys_name = '1-organyl-2-acyl-sn-glycero-3-phosphocholine:1-organyl-2-lyso-sn-glycero-3-phosphoethanolamine acyltransferase (CoA-dependent)' WHERE ec = 'EC 2.3.1.148';
UPDATE enzyme SET history = 'EC 2.3.1.148 created 1999' WHERE ec = 'EC 2.3.1.148';
UPDATE enzyme SET sys_name = '1-alkyl-2-acyl-sn-glycero-3-phosphocholine:1-organyl-2-lyso-sn-glycero-3-phospholipid acetyltransferase' WHERE ec = 'EC 2.3.1.149';
UPDATE enzyme SET history = 'EC 2.3.1.149 created 1999' WHERE ec = 'EC 2.3.1.149';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:salutaridinol 7-O-acetyltransferase' WHERE ec = 'EC 2.3.1.150';
UPDATE enzyme SET history = 'EC 2.3.1.150 created 1999' WHERE ec = 'EC 2.3.1.150';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:3-hydroxybenzoyl-CoA malonyltransferase (decarboxylating, 2,3,4,6-tetrahydroxybenzophenone-forming)' WHERE ec = 'EC 2.3.1.151';
UPDATE enzyme SET history = 'EC 2.3.1.151 created 1999, modified 2013' WHERE ec = 'EC 2.3.1.151';
UPDATE enzyme SET sys_name = '1-O-trans-cinnamoyl-beta-D-glucopyranose:alcohol O-cinnamoyltransferase' WHERE ec = 'EC 2.3.1.152';
UPDATE enzyme SET history = 'EC 2.3.1.152 created 1999' WHERE ec = 'EC 2.3.1.152';
UPDATE enzyme SET sys_name = '4-hydroxycinnamoyl-CoA:anthocyanidin 3,5-diglucoside 5-O-glucoside-6-O-4-hydroxycinnamoyltransferase' WHERE ec = 'EC 2.3.1.153';
UPDATE enzyme SET history = 'EC 2.3.1.153 created 1999, modified 2013' WHERE ec = 'EC 2.3.1.153';
UPDATE enzyme SET sys_name = '4,8,12-trimethyltridecanoyl-CoA:propanoyl-CoA C(2)-4,8,12-trimethyltridecanoyltransferase' WHERE ec = 'EC 2.3.1.154';
UPDATE enzyme SET history = 'EC 2.3.1.154 created 2000' WHERE ec = 'EC 2.3.1.154';
UPDATE enzyme SET sys_name = 'Myristoyl-CoA:acetyl-CoA C-myristoyltransferase' WHERE ec = 'EC 2.3.1.155';
UPDATE enzyme SET history = 'EC 2.3.1.155 created 2000' WHERE ec = 'EC 2.3.1.155';
UPDATE enzyme SET sys_name = 'Isovaleryl-CoA:malonyl-CoA acyltransferase' WHERE ec = 'EC 2.3.1.156';
UPDATE enzyme SET history = 'EC 2.3.1.156 created 2000' WHERE ec = 'EC 2.3.1.156';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:alpha-D-glucosamine-1-phosphate N-acetyltransferase' WHERE ec = 'EC 2.3.1.157';
UPDATE enzyme SET history = 'EC 2.3.1.157 created 2001' WHERE ec = 'EC 2.3.1.157';
UPDATE enzyme SET sys_name = 'Phospholipid:1,2-diacyl-sn-glycerol O-acyltransferase' WHERE ec = 'EC 2.3.1.158';
UPDATE enzyme SET history = 'EC 2.3.1.158 created 2001' WHERE ec = 'EC 2.3.1.158';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:N-methylanthraniloyl-CoA malonyltransferase (cyclizing)' WHERE ec = 'EC 2.3.1.159';
UPDATE enzyme SET history = 'EC 2.3.1.159 created 2002' WHERE ec = 'EC 2.3.1.159';
UPDATE enzyme SET sys_name = 'Acyl-CoA:16-epivellosimine O-acetyltransferase (cyclizing)' WHERE ec = 'EC 2.3.1.160';
UPDATE enzyme SET history = 'EC 2.3.1.160 created 2002' WHERE ec = 'EC 2.3.1.160';
UPDATE enzyme SET sys_name = 'Acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing, thioester-hydrolyzing)' WHERE ec = 'EC 2.3.1.161';
UPDATE enzyme SET history = 'EC 2.3.1.161 created 2002' WHERE ec = 'EC 2.3.1.161';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:taxa-4(20),11-dien-5-alpha-ol O-acetyltransferase' WHERE ec = 'EC 2.3.1.162';
UPDATE enzyme SET history = 'EC 2.3.1.162 created 2002' WHERE ec = 'EC 2.3.1.162';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:taxan-10-beta-ol O-acetyltransferase' WHERE ec = 'EC 2.3.1.163';
UPDATE enzyme SET history = 'EC 2.3.1.163 created 2002' WHERE ec = 'EC 2.3.1.163';
UPDATE enzyme SET sys_name = 'Acyl-CoA:isopenicillin N N-acyltransferase' WHERE ec = 'EC 2.3.1.164';
UPDATE enzyme SET history = 'EC 2.3.1.164 created 2002' WHERE ec = 'EC 2.3.1.164';
UPDATE enzyme SET sys_name = 'Acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl-reducing, thioester-hydrolyzing and cyclizing)' WHERE ec = 'EC 2.3.1.165';
UPDATE enzyme SET history = 'EC 2.3.1.165 created 2002' WHERE ec = 'EC 2.3.1.165';
UPDATE enzyme SET sys_name = 'Benzoyl-CoA:taxan-2-alpha-ol O-benzoyltransferase' WHERE ec = 'EC 2.3.1.166';
UPDATE enzyme SET history = 'EC 2.3.1.166 created 2002' WHERE ec = 'EC 2.3.1.166';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:taxan-10-beta-ol O-acetyltransferase' WHERE ec = 'EC 2.3.1.167';
UPDATE enzyme SET history = 'EC 2.3.1.167 created 2002' WHERE ec = 'EC 2.3.1.167';
UPDATE enzyme SET sys_name = '2-methylpropanoyl-CoA:enzyme-N(6)-(dihydrolipoyl)lysine S-(2-methylpropanoyl)transferase' WHERE ec = 'EC 2.3.1.168';
UPDATE enzyme SET history = 'EC 2.3.1.168 created 2003' WHERE ec = 'EC 2.3.1.168';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:corrinoid protein O-acetyltransferase' WHERE ec = 'EC 2.3.1.169';
UPDATE enzyme SET history = 'EC 2.3.1.169 created 2003' WHERE ec = 'EC 2.3.1.169';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:4-coumaroyl-CoA malonyltransferase (cyclizing, reducing)' WHERE ec = 'EC 2.3.1.170';
UPDATE enzyme SET history = 'EC 2.3.1.170 created 2004' WHERE ec = 'EC 2.3.1.170';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:anthocyanidin-3-O-beta-D-glucoside 6-O-malonyltransferase' WHERE ec = 'EC 2.3.1.171';
UPDATE enzyme SET history = 'EC 2.3.1.171 created 2004' WHERE ec = 'EC 2.3.1.171';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:pelargonidin-3-O-(6-caffeoyl-beta-D-glucoside)-5-O-beta-D-glucoside 6-O-malonyltransferase' WHERE ec = 'EC 2.3.1.172';
UPDATE enzyme SET history = 'EC 2.3.1.172 created 2004' WHERE ec = 'EC 2.3.1.172';
UPDATE enzyme SET sys_name = '4-coumaroyl-CoA:flavonol 3-O-(beta-D-glucosyl-(1-&gt;2)-beta-D-glucosyl-(1-&gt;2)-beta-D-glucoside) 6-O-4-coumaroyltransferase' WHERE ec = 'EC 2.3.1.173';
UPDATE enzyme SET history = 'EC 2.3.1.173 created 2004' WHERE ec = 'EC 2.3.1.173';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:acetyl-CoA C-succinyltransferase' WHERE ec = 'EC 2.3.1.174';
UPDATE enzyme SET history = 'EC 2.3.1.174 created 2005, modified 2013' WHERE ec = 'EC 2.3.1.174';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:deacetylcephalosporin-C O-acetyltransferase' WHERE ec = 'EC 2.3.1.175';
UPDATE enzyme SET history = 'EC 2.3.1.175 created 2005' WHERE ec = 'EC 2.3.1.175';
UPDATE enzyme SET sys_name = '3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholanoyl-CoA:propanoyl-CoA C-acyltransferase' WHERE ec = 'EC 2.3.1.176';
UPDATE enzyme SET history = 'EC 2.3.1.176 created 2005' WHERE ec = 'EC 2.3.1.176';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:benzoyl-CoA malonyltransferase' WHERE ec = 'EC 2.3.1.177';
UPDATE enzyme SET history = 'EC 2.3.1.177 created 2006, modified 2012' WHERE ec = 'EC 2.3.1.177';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:L-2,4-diaminobutanoate N(4)-acetyltransferase' WHERE ec = 'EC 2.3.1.178';
UPDATE enzyme SET history = 'EC 2.3.1.178 created 2006' WHERE ec = 'EC 2.3.1.178';
UPDATE enzyme SET sys_name = '(Z)-hexadec-11-enoyl-[acyl-carrier protein]:malonyl-[acyl-carrier protein] C-acyltransferase (decarboxylating)' WHERE ec = 'EC 2.3.1.179';
UPDATE enzyme SET history = 'EC 2.3.1.179 created 2006' WHERE ec = 'EC 2.3.1.179';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:malonyl-[acyl-carrier-protein] C-acyltransferase' WHERE ec = 'EC 2.3.1.180';
UPDATE enzyme SET history = 'EC 2.3.1.180 created 2006' WHERE ec = 'EC 2.3.1.180';
UPDATE enzyme SET sys_name = 'Octanoyl-[acyl-carrier protein]:protein N-octanoyltransferase' WHERE ec = 'EC 2.3.1.181';
UPDATE enzyme SET history = 'EC 2.3.1.181 created 2006' WHERE ec = 'EC 2.3.1.181';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 2.3.1.182';
UPDATE enzyme SET history = 'EC 2.3.1.182 created 2007' WHERE ec = 'EC 2.3.1.182';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:phosphinothricin N-acetyltransferase' WHERE ec = 'EC 2.3.1.183';
UPDATE enzyme SET history = 'EC 2.3.1.183 created 2007' WHERE ec = 'EC 2.3.1.183';
UPDATE enzyme SET sys_name = 'Acyl-[acy-carrier protein]:S-adenosyl-L-methionine acyltranserase (lactone-forming, methylthioadenosine-releasing)' WHERE ec = 'EC 2.3.1.184';
UPDATE enzyme SET history = 'EC 2.3.1.184 created 2007' WHERE ec = 'EC 2.3.1.184';
UPDATE enzyme SET sys_name = 'Acyl-CoA:tropine O-acyltransferase' WHERE ec = 'EC 2.3.1.185';
UPDATE enzyme SET history = 'EC 2.3.1.185 created 2008' WHERE ec = 'EC 2.3.1.185';
UPDATE enzyme SET sys_name = 'Acyl-CoA:pseudotropine O-acyltransferase' WHERE ec = 'EC 2.3.1.186';
UPDATE enzyme SET history = 'EC 2.3.1.186 created 2008' WHERE ec = 'EC 2.3.1.186';
UPDATE enzyme SET sys_name = 'Acetyl-[acyl-carrier-protein]:malonate S-[acyl-carrier-protein]transferase' WHERE ec = 'EC 2.3.1.187';
UPDATE enzyme SET history = 'EC 2.3.1.187 created 2008' WHERE ec = 'EC 2.3.1.187';
UPDATE enzyme SET sys_name = 'Feruloyl-CoA:16-hydroxypalmitate feruloyltransferase' WHERE ec = 'EC 2.3.1.188';
UPDATE enzyme SET history = 'EC 2.3.1.188 created 2009' WHERE ec = 'EC 2.3.1.188';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:desacetylmycothiol O-acetyltransferase' WHERE ec = 'EC 2.3.1.189';
UPDATE enzyme SET history = 'EC 2.3.1.189 created 2010' WHERE ec = 'EC 2.3.1.189';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:acetoin O-acetyltransferase' WHERE ec = 'EC 2.3.1.190';
UPDATE enzyme SET history = 'EC 2.3.1.190 created 2010' WHERE ec = 'EC 2.3.1.190';
UPDATE enzyme SET sys_name = '(3R)-3-hydroxymyristoyl-[acyl-carrier protein]:UDP-3-O-((3R)-hydroxymyristoyl)-alpha-D-glucosamine N-acyltransferase' WHERE ec = 'EC 2.3.1.191';
UPDATE enzyme SET history = 'EC 2.3.1.191 created 2010' WHERE ec = 'EC 2.3.1.191';
UPDATE enzyme SET sys_name = 'Phenylacetyl-CoA:glycine N-phenylacetyltransferase' WHERE ec = 'EC 2.3.1.192';
UPDATE enzyme SET history = 'EC 2.3.1.192 created 2010' WHERE ec = 'EC 2.3.1.192';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:(elongator tRNA(Met))-cytidine(34) N(4)-acetyltransferase (ATP-hydrolyzing)' WHERE ec = 'EC 2.3.1.193';
UPDATE enzyme SET history = 'EC 2.3.1.193 created 2011' WHERE ec = 'EC 2.3.1.193';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:malonyl-CoA C-acetyltransferase (decarboxylating)' WHERE ec = 'EC 2.3.1.194';
UPDATE enzyme SET history = 'EC 2.3.1.194 created 2011' WHERE ec = 'EC 2.3.1.194';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:(3Z)-hex-3-en-ol acetyltransferase' WHERE ec = 'EC 2.3.1.195';
UPDATE enzyme SET history = 'EC 2.3.1.195 created 2011' WHERE ec = 'EC 2.3.1.195';
UPDATE enzyme SET sys_name = 'Benzoyl-CoA:benzyl alcohol O-benzoyltransferase' WHERE ec = 'EC 2.3.1.196';
UPDATE enzyme SET history = 'EC 2.3.1.196 created 2011' WHERE ec = 'EC 2.3.1.196';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:dTDP-3-amino-3,6-dideoxy-alpha-D-galactopyranose 3-N-acetyltransferase' WHERE ec = 'EC 2.3.1.197';
UPDATE enzyme SET history = 'EC 2.3.1.197 created 2012' WHERE ec = 'EC 2.3.1.197';
UPDATE enzyme SET sys_name = 'Acyl-CoA:sn-glycerol 3-phosphate 2-O-acyltransferase' WHERE ec = 'EC 2.3.1.198';
UPDATE enzyme SET history = 'EC 2.3.1.198 created 2012' WHERE ec = 'EC 2.3.1.198';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:very-long-chain acyl-CoA malonyltransferase (decarboxylating and thioester-hydrolyzing)' WHERE ec = 'EC 2.3.1.199';
UPDATE enzyme SET history = 'EC 2.3.1.199 created 2012' WHERE ec = 'EC 2.3.1.199';
UPDATE enzyme SET sys_name = '[Glycine cleavage system H]-N(6)-lipoyl-L-lysine:[lipoyl-carrier protein]-N(6)-L-lysine lipoyltransferase' WHERE ec = 'EC 2.3.1.200';
UPDATE enzyme SET history = 'EC 2.3.1.200 created 2012' WHERE ec = 'EC 2.3.1.200';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:UDP-2-acetamido-3-amino-2,3-dideoxy-alpha-D-glucuronate N-acetyltransferase' WHERE ec = 'EC 2.3.1.201';
UPDATE enzyme SET history = 'EC 2.3.1.201 created 2012' WHERE ec = 'EC 2.3.1.201';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:UDP-4-amino-4,6-dideoxy-N-acetyl-beta-L-altrosamine N-acetyltransferase' WHERE ec = 'EC 2.3.1.202';
UPDATE enzyme SET history = 'EC 2.3.1.202 created 2012' WHERE ec = 'EC 2.3.1.202';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:UDP-4-amino-4,6-dideoxy-N-acetyl-alpha-D-glucosamine N-acetyltransferase' WHERE ec = 'EC 2.3.1.203';
UPDATE enzyme SET history = 'EC 2.3.1.203 created 2012, modified 2013' WHERE ec = 'EC 2.3.1.203';
UPDATE enzyme SET sys_name = '[Glycine cleavage system H]-N(6)-octanoyl-L-lysine:[lipoyl-carrier protein]-N(6)-L-lysine octanoyltransferase' WHERE ec = 'EC 2.3.1.204';
UPDATE enzyme SET history = 'EC 2.3.1.204 created 2012' WHERE ec = 'EC 2.3.1.204';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:fumigaclavine B O-acetyltransferase' WHERE ec = 'EC 2.3.1.205';
UPDATE enzyme SET history = 'EC 2.3.1.205 created 2012' WHERE ec = 'EC 2.3.1.205';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:hexanoyl-CoA malonyltransferase (3,5,7-trioxododecanoyl-CoA-forming)' WHERE ec = 'EC 2.3.1.206';
UPDATE enzyme SET history = 'EC 2.3.1.206 created 2012' WHERE ec = 'EC 2.3.1.206';
UPDATE enzyme SET sys_name = 'Octanoyl-CoA:malonyl-[acyl-carrier protein] C-heptanoylltransferase (decarboxylating, CoA-forming)' WHERE ec = 'EC 2.3.1.207';
UPDATE enzyme SET history = 'EC 2.3.1.207 created 2012' WHERE ec = 'EC 2.3.1.207';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:2-hydroxybenzoyl-CoA malonyltransferase' WHERE ec = 'EC 2.3.1.208';
UPDATE enzyme SET history = 'EC 2.3.1.208 created 2012' WHERE ec = 'EC 2.3.1.208';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:dTDP-4-amino-4,6-dideoxy-alpha-D-glucose N-acetyltransferase' WHERE ec = 'EC 2.3.1.209';
UPDATE enzyme SET history = 'EC 2.3.1.209 created 2012' WHERE ec = 'EC 2.3.1.209';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:dTDP-4-amino-4,6-dideoxy-alpha-D-galactose N-acetyltransferase' WHERE ec = 'EC 2.3.1.210';
UPDATE enzyme SET history = 'EC 2.3.1.210 created 2012' WHERE ec = 'EC 2.3.1.210';
UPDATE enzyme SET sys_name = '4-coumaroyl-CoA:malonyl-CoA 4-coumaryltransferase (bisdemethoxycurcumin-forming)' WHERE ec = 'EC 2.3.1.211';
UPDATE enzyme SET history = 'EC 2.3.1.211 created 2013' WHERE ec = 'EC 2.3.1.211';
UPDATE enzyme SET sys_name = '4-coumaroyl-CoA:malonyl-CoA 4-coumaryltransferase (4-hydroxybenzalacetone-forming)' WHERE ec = 'EC 2.3.1.212';
UPDATE enzyme SET history = 'EC 2.3.1.212 created 2013' WHERE ec = 'EC 2.3.1.212';
UPDATE enzyme SET sys_name = '1-O-(4-hydroxycinnamoyl)-beta-D-glucose:cyanidin 3-O-(6-O-beta-D-glucosyl-2-O-beta-D-xylosyl-beta-D-galactoside) 6-O-(4-hydroxycinnamoyl)transferase' WHERE ec = 'EC 2.3.1.213';
UPDATE enzyme SET history = 'EC 2.3.1.213 created 2013' WHERE ec = 'EC 2.3.1.213';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:4-demalonylsalvianin 4-O-malonyltransferase' WHERE ec = 'EC 2.3.1.214';
UPDATE enzyme SET history = 'EC 2.3.1.214 created 2013' WHERE ec = 'EC 2.3.1.214';
UPDATE enzyme SET sys_name = '4-hydroxycinnamoyl-CoA:anthocyanin-3-O-glucoside 6-O-acyltransferase' WHERE ec = 'EC 2.3.1.215';
UPDATE enzyme SET history = 'EC 2.3.1.215 created 2013' WHERE ec = 'EC 2.3.1.215';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:malonyl-CoA malonyltransferase (5,7-dihydroxy-2-methyl-4H-chromen-4-one-forming)' WHERE ec = 'EC 2.3.1.216';
UPDATE enzyme SET history = 'EC 2.3.1.216 created 2013' WHERE ec = 'EC 2.3.1.216';
UPDATE enzyme SET sys_name = 'Feruloyl-CoA:feruloylacetyl-CoA feruloyltransferase (curcumin-forming)' WHERE ec = 'EC 2.3.1.217';
UPDATE enzyme SET history = 'EC 2.3.1.217 created 2013' WHERE ec = 'EC 2.3.1.217';
UPDATE enzyme SET sys_name = 'Phenylpropanoyl-CoA:malonyl-CoA phenylpropanoyl-transferase (decarboxylating)' WHERE ec = 'EC 2.3.1.218';
UPDATE enzyme SET history = 'EC 2.3.1.218 created 2013' WHERE ec = 'EC 2.3.1.218';
UPDATE enzyme SET sys_name = '4-coumaroyl-CoA:feruloylacetyl-CoA feruloyltransferase (demethoxycurcumin-forming)' WHERE ec = 'EC 2.3.1.219';
UPDATE enzyme SET history = 'EC 2.3.1.219 created 2013' WHERE ec = 'EC 2.3.1.219';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:benzoyl-CoA malonyltransferase (2,4,6-trihydroxybenzophenone-forming)' WHERE ec = 'EC 2.3.1.220';
UPDATE enzyme SET history = 'EC 2.3.1.220 created 2013' WHERE ec = 'EC 2.3.1.220';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:hexanoate malonyltransferase (norsolorinate anthrone-forming)' WHERE ec = 'EC 2.3.1.221';
UPDATE enzyme SET history = 'EC 2.3.1.221 created 2013' WHERE ec = 'EC 2.3.1.221';
UPDATE enzyme SET sys_name = 'Propanoyl-CoA:phosphate propanoyltransferase' WHERE ec = 'EC 2.3.1.222';
UPDATE enzyme SET history = 'EC 2.3.1.222 created 2013' WHERE ec = 'EC 2.3.1.222';
UPDATE enzyme SET sys_name = '2,3-didehydroadipoyl-CoA:acetyl-CoA C-didehydroadipoyltransferase (double bond migration)' WHERE ec = 'EC 2.3.1.223';
UPDATE enzyme SET history = 'EC 2.3.1.223 created 2013' WHERE ec = 'EC 2.3.1.223';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:benzylalcohol O-acetyltransferase' WHERE ec = 'EC 2.3.1.224';
UPDATE enzyme SET history = 'EC 2.3.1.224 created 2013' WHERE ec = 'EC 2.3.1.224';
UPDATE enzyme SET sys_name = 'Palmitoyl-CoA:[protein]-L-cysteine S-palmitoyltransferase' WHERE ec = 'EC 2.3.1.225';
UPDATE enzyme SET history = 'EC 2.3.1.225 created 2013' WHERE ec = 'EC 2.3.1.225';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:(S)-1-pyrroline-5-carboxylate malonyltransferase (cyclizing)' WHERE ec = 'EC 2.3.1.226';
UPDATE enzyme SET history = 'EC 2.3.1.226 created 2013' WHERE ec = 'EC 2.3.1.226';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:GDP-4-amino-4,6-dideoxy-alpha-D-mannose N-acetyltransferase' WHERE ec = 'EC 2.3.1.227';
UPDATE enzyme SET history = 'EC 2.3.1.227 created 2013' WHERE ec = 'EC 2.3.1.227';
UPDATE enzyme SET sys_name = 'Isovaleryl-CoA:S-adenosyl-L-methionine isovaleryltranserase (lactone-forming, methylthioadenosine-releasing)' WHERE ec = 'EC 2.3.1.228';
UPDATE enzyme SET history = 'EC 2.3.1.228 created 2013' WHERE ec = 'EC 2.3.1.228';
UPDATE enzyme SET sys_name = '4-coumaroyl-CoA:S-adenosyl-L-methionine trans-4-coumaroyltranserase (lactone-forming, methylthioadenosine-releasing)' WHERE ec = 'EC 2.3.1.229';
UPDATE enzyme SET history = 'EC 2.3.1.229 created 2013' WHERE ec = 'EC 2.3.1.229';
UPDATE enzyme SET sys_name = 'Malonyl-CoA:anthraniloyl-CoA C-acetyltransferase (decarboxylating)' WHERE ec = 'EC 2.3.1.230';
UPDATE enzyme SET history = 'EC 2.3.1.230 created 2013' WHERE ec = 'EC 2.3.1.230';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA(Phe) (7-((3S)-3-amino-3-(methoxycarbonyl)propyl)wyosine(37)-N)-methyltransferase (carbon dioxide-adding)' WHERE ec = 'EC 2.3.1.231';
UPDATE enzyme SET history = 'EC 2.3.1.231 created 2013' WHERE ec = 'EC 2.3.1.231';
UPDATE enzyme SET sys_name = 'Anthraniloyl-CoA:methanol O-anthraniloyltransferase' WHERE ec = 'EC 2.3.1.232';
UPDATE enzyme SET history = 'EC 2.3.1.232 created 2014' WHERE ec = 'EC 2.3.1.232';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.3.1.n1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 2.3.1.n2';
UPDATE enzyme SET history = 'EC 2.3.1.n2 created 2012' WHERE ec = 'EC 2.3.1.n2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 2.3.1.n3';
UPDATE enzyme SET history = 'EC 2.3.1.n2 created 2011' WHERE ec = 'EC 2.3.1.n3';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 2.3.1.n4';
UPDATE enzyme SET history = 'EC 2.3.1.n4 created 2012' WHERE ec = 'EC 2.3.1.n4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 2.3.1.n5';
UPDATE enzyme SET history = 'EC 2.3.1.n5 created 2012' WHERE ec = 'EC 2.3.1.n5';
UPDATE enzyme SET sys_name = 'Acyl-CoA:1-acyl-sn-glycero-3-phosphatidylserine acyltransferase' WHERE ec = 'EC 2.3.1.n6';
UPDATE enzyme SET history = 'EC 2.3.1.n6 created 2012' WHERE ec = 'EC 2.3.1.n6';
UPDATE enzyme SET sys_name = 'Acyl-CoA:1-acyl-sn-glycero-3-phosphoethanolamine acyltransferase' WHERE ec = 'EC 2.3.1.n7';
UPDATE enzyme SET history = 'EC 2.3.1.n7 created 2012' WHERE ec = 'EC 2.3.1.n7';
UPDATE enzyme SET history = 'EC 2.3.1.n8 created 2012' WHERE ec = 'EC 2.3.1.n8';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.3.1.n9';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.3.1.n10';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.3.1.n11';
UPDATE enzyme SET sys_name = '1-O-sinapoyl-beta-D-glucose:cyanidin 3-O-(6-O-(4-O-beta-D-glucosyl-p-coumaroyl)-2-O-(beta-D-xylosyl)-beta-D-glucosyl)-5-O-(6-O-malonyl-beta-D-glucoside) 1-O-sinapoyltransferase' WHERE ec = 'EC 2.3.1.n12';
UPDATE enzyme SET history = 'EC 2.3.1.n12 created 2013' WHERE ec = 'EC 2.3.1.n12';
UPDATE enzyme SET sys_name = 'Glutamine:(gamma-D-glutamyl)(N)-[peptide] 5-glutamyltransferase' WHERE ec = 'EC 2.3.2.1';
UPDATE enzyme SET history = 'EC 2.3.2.1 created 1961, modified 1976, modified 2013' WHERE ec = 'EC 2.3.2.1';
UPDATE enzyme SET sys_name = '(5-L-glutamyl)-peptide:amino-acid 5-glutamyltransferase' WHERE ec = 'EC 2.3.2.2';
UPDATE enzyme SET history = 'EC 2.3.2.2 created 1972, modified 1976, modified 2011' WHERE ec = 'EC 2.3.2.2';
UPDATE enzyme SET sys_name = 'L-lysyl-tRNA(Lys):phosphatidylglycerol 3-O-lysyltransferase' WHERE ec = 'EC 2.3.2.3';
UPDATE enzyme SET history = 'EC 2.3.2.3 created 1972, modified 2013' WHERE ec = 'EC 2.3.2.3';
UPDATE enzyme SET sys_name = '(Gamma-L-glutamyl)-L-amino-acid gamma-glutamyltransferase (cyclizing)' WHERE ec = 'EC 2.3.2.4';
UPDATE enzyme SET history = 'EC 2.3.2.4 created 1972' WHERE ec = 'EC 2.3.2.4';
UPDATE enzyme SET sys_name = 'L-glutaminyl-peptide gamma-glutamyltransferase (cyclizing)' WHERE ec = 'EC 2.3.2.5';
UPDATE enzyme SET history = 'EC 2.3.2.5 created 1972, modified 1990' WHERE ec = 'EC 2.3.2.5';
UPDATE enzyme SET sys_name = 'L-leucyl-tRNA(Leu):protein leucyltransferase' WHERE ec = 'EC 2.3.2.6';
UPDATE enzyme SET history = 'EC 2.3.2.6 created 1972, modified 1976, modified 2013' WHERE ec = 'EC 2.3.2.6';
UPDATE enzyme SET sys_name = 'L-asparagine:hydroxylamine gamma-aspartyltransferase' WHERE ec = 'EC 2.3.2.7';
UPDATE enzyme SET history = 'EC 2.3.2.7 created 1972' WHERE ec = 'EC 2.3.2.7';
UPDATE enzyme SET sys_name = 'L-arginyl-tRNA(Arg):protein arginyltransferase' WHERE ec = 'EC 2.3.2.8';
UPDATE enzyme SET history = 'EC 2.3.2.8 created 1972, modified 1976, modified 2013' WHERE ec = 'EC 2.3.2.8';
UPDATE enzyme SET sys_name = '(Gamma-L-glutamyl)-N(1)-(4-hydroxymethylphenyl)hydrazine:acceptor gamma-glutamyltransferase' WHERE ec = 'EC 2.3.2.9';
UPDATE enzyme SET history = 'EC 2.3.2.9 created 1972' WHERE ec = 'EC 2.3.2.9';
UPDATE enzyme SET sys_name = 'L-alanyl-tRNA(Ala):UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamyl-L-lysyl-D-alanyl-D-alanine N(6)-alanyltransferase' WHERE ec = 'EC 2.3.2.10';
UPDATE enzyme SET history = 'EC 2.3.2.10 created 1972, modified 2013' WHERE ec = 'EC 2.3.2.10';
UPDATE enzyme SET sys_name = 'L-alanyl-tRNA(Ala):phosphatidylglycerol alanyltransferase' WHERE ec = 'EC 2.3.2.11';
UPDATE enzyme SET history = 'EC 2.3.2.11 created 1972, modified 2013' WHERE ec = 'EC 2.3.2.11';
UPDATE enzyme SET sys_name = 'Peptidyl-tRNA:aminoacyl-tRNA N-peptidyltransferase' WHERE ec = 'EC 2.3.2.12';
UPDATE enzyme SET history = 'EC 2.3.2.12 created 1976, modified 2010' WHERE ec = 'EC 2.3.2.12';
UPDATE enzyme SET sys_name = 'Protein-glutamine:amine gamma-glutamyltransferase' WHERE ec = 'EC 2.3.2.13';
UPDATE enzyme SET history = 'EC 2.3.2.13 created 1978, modified 1981, modified 1983' WHERE ec = 'EC 2.3.2.13';
UPDATE enzyme SET sys_name = 'L-glutamine:D-alanine gamma-glutamyltransferase' WHERE ec = 'EC 2.3.2.14';
UPDATE enzyme SET history = 'EC 2.3.2.14 created 1989' WHERE ec = 'EC 2.3.2.14';
UPDATE enzyme SET sys_name = 'Glutathione:poly(4-glutamyl-cysteinyl)glycine 4-glutamylcysteinyltransferase' WHERE ec = 'EC 2.3.2.15';
UPDATE enzyme SET history = 'EC 2.3.2.15 created 1992' WHERE ec = 'EC 2.3.2.15';
UPDATE enzyme SET sys_name = 'MurNAc-L-Ala-D-isoglutaminyl-L-Lys-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc:glycine N(6)-glycyltransferase' WHERE ec = 'EC 2.3.2.16';
UPDATE enzyme SET history = 'EC 2.3.2.16 created 2010' WHERE ec = 'EC 2.3.2.16';
UPDATE enzyme SET sys_name = 'MurNAc-L-Ala-D-isoglutaminyl-L-Lys-(N(6)-Gly)-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc:glycine glycyltransferase' WHERE ec = 'EC 2.3.2.17';
UPDATE enzyme SET history = 'EC 2.3.2.17 created 2010' WHERE ec = 'EC 2.3.2.17';
UPDATE enzyme SET sys_name = 'MurNAc-L-Ala-D-isoglutaminyl-L-Lys-(N(6)-tri-Gly)-D-Ala-D-Ala-diphospho-ditrans,octacis-undecaprenyl-GlcNAc:glycine glycyltransferase' WHERE ec = 'EC 2.3.2.18';
UPDATE enzyme SET history = 'EC 2.3.2.18 created 2010' WHERE ec = 'EC 2.3.2.18';
UPDATE enzyme SET sys_name = 'Ribostamycin:4-(gamma-L-glutamylamino)-(S)-2-hydroxybutanoyl-[BtrI acyl-carrier protein] 4-(gamma-L-glutamylamino)-(S)-2-hydroxybutanoate transferase' WHERE ec = 'EC 2.3.2.19';
UPDATE enzyme SET history = 'EC 2.3.2.19 created 2012' WHERE ec = 'EC 2.3.2.19';
UPDATE enzyme SET sys_name = 'L-leucyl-tRNA(Leu):L-phenylalanyl-tRNA(Phe) leucyltransferase (cyclizing)' WHERE ec = 'EC 2.3.2.20';
UPDATE enzyme SET history = 'EC 2.3.2.20 created 2013' WHERE ec = 'EC 2.3.2.20';
UPDATE enzyme SET sys_name = 'L-tyrosyl-tRNA(Tyr):L-tyrosyl-tRNA(Tyr) tyrosyltransferase (cyclizing)' WHERE ec = 'EC 2.3.2.21';
UPDATE enzyme SET history = 'EC 2.3.2.21 created 2013' WHERE ec = 'EC 2.3.2.21';
UPDATE enzyme SET sys_name = 'L-leucyl-tRNA(Leu):L-leucyl-tRNA(Leu) leucyltransferase (cyclizing)' WHERE ec = 'EC 2.3.2.22';
UPDATE enzyme SET history = 'EC 2.3.2.22 created 2013' WHERE ec = 'EC 2.3.2.22';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:oxaloacetate C-acetyltransferase (thioester-hydrolyzing, (pro-S)-carboxymethyl forming)' WHERE ec = 'EC 2.3.3.1';
UPDATE enzyme SET history = 'EC 2.3.3.1 created 1961 as EC 4.1.3.7, transferred 2002 to EC 2.3.3.1, modified 2014' WHERE ec = 'EC 2.3.3.1';
UPDATE enzyme SET sys_name = 'Dodecanoyl-CoA:oxaloacetate C-dodecanoyltransferase (thioester-hydrolyzing, 1-carboxyundecyl-forming)' WHERE ec = 'EC 2.3.3.2';
UPDATE enzyme SET history = 'EC 2.3.3.2 created 1972 as EC 4.1.3.23, transferred 2002 to EC 2.3.3.2' WHERE ec = 'EC 2.3.3.2';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:oxaloacetate C-acetyltransferase (thioester-hydrolyzing, (pro-R)-carboxymethyl-forming)' WHERE ec = 'EC 2.3.3.3';
UPDATE enzyme SET history = 'EC 2.3.3.3 created 1972 as EC 4.1.3.28, transferred 2002 to EC 2.3.3.3' WHERE ec = 'EC 2.3.3.3';
UPDATE enzyme SET sys_name = 'Dodecanoyl-CoA:2-oxoglutarate C-dodecanoyltransferase (thioester-hydrolyzing, 1-carboxyundecyl-forming)' WHERE ec = 'EC 2.3.3.4';
UPDATE enzyme SET history = 'EC 2.3.3.4 created 1976 as EC 4.1.3.29, transferred 2002 to EC 2.3.3.4' WHERE ec = 'EC 2.3.3.4';
UPDATE enzyme SET sys_name = 'Propanoyl-CoA:oxaloacetate C-propanoyltransferase (thioester-hydrolyzing, 1-carboxyethyl-forming)' WHERE ec = 'EC 2.3.3.5';
UPDATE enzyme SET history = 'EC 2.3.3.5 created 1978 as EC 4.1.3.31, transferred 2002 to EC 2.3.3.5' WHERE ec = 'EC 2.3.3.5';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:2-oxobutanoate C-acetyltransferase (thioester-hydrolyzing, carboxymethyl-forming)' WHERE ec = 'EC 2.3.3.6';
UPDATE enzyme SET history = 'EC 2.3.3.6 created 1983 as EC 4.1.3.33, transferred 2002 to EC 2.3.3.6' WHERE ec = 'EC 2.3.3.6';
UPDATE enzyme SET sys_name = 'Butanoyl-CoA:glyoxylate C-butanoyltransferase (thioester-hydrolyzing, 1-carboxypropyl-forming)' WHERE ec = 'EC 2.3.3.7';
UPDATE enzyme SET history = 'EC 2.3.3.7 created 1965 as EC 4.1.3.10, modified 1983, transferred 2002 to EC 2.3.3.10' WHERE ec = 'EC 2.3.3.7';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:oxaloacetate C-acetyltransferase ((pro-S)-carboxymethyl-forming, ADP-phosphorylating)' WHERE ec = 'EC 2.3.3.8';
UPDATE enzyme SET history = 'EC 2.3.3.8 created 1965 as EC 4.1.3.8, modified 1986, transferred 2002 to EC 2.3.3.8' WHERE ec = 'EC 2.3.3.8';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:glyoxylate C-acetyltransferase (thioester-hydrolyzing, carboxymethyl-forming)' WHERE ec = 'EC 2.3.3.9';
UPDATE enzyme SET history = 'EC 2.3.3.9 created 1961 as EC 4.1.3.2, transferred 2002 to EC 2.3.3.9' WHERE ec = 'EC 2.3.3.9';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:acetoacetyl-CoA C-acetyltransferase (thioester-hydrolyzing, carboxymethyl-forming)' WHERE ec = 'EC 2.3.3.10';
UPDATE enzyme SET history = 'EC 2.3.3.10 created 1961 as EC 4.1.3.5, transferred 2002 to EC 2.3.3.10' WHERE ec = 'EC 2.3.3.10';
UPDATE enzyme SET sys_name = 'Propanoyl-CoA:glyoxylate C-propanoyltransferase (thioester-hydrolyzing, 2-carboxyethyl-forming)' WHERE ec = 'EC 2.3.3.11';
UPDATE enzyme SET history = 'EC 2.3.3.11 created 1965 as EC 4.1.3.9, transferred 2002 to EC 2.3.3.11' WHERE ec = 'EC 2.3.3.11';
UPDATE enzyme SET sys_name = 'Pentanoyl-CoA:glyoxylate C-pentanoyltransferase (thioester-hydrolyzing, 1-carboxybutyl-forming)' WHERE ec = 'EC 2.3.3.12';
UPDATE enzyme SET history = 'EC 2.3.3.12 created 1972 as EC 4.1.3.11, transferred 2002 to EC 2.3.3.12' WHERE ec = 'EC 2.3.3.12';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:3-methyl-2-oxobutanoate C-acetyltransferase (thioester-hydrolyzing, carboxymethyl-forming)' WHERE ec = 'EC 2.3.3.13';
UPDATE enzyme SET history = 'EC 2.3.3.13 created 1972 as EC 4.1.3.12, transferred 2002 to EC 2.3.3.13' WHERE ec = 'EC 2.3.3.13';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:2-oxoglutarate C-acetyltransferase (thioester-hydrolyzing, carboxymethyl forming)' WHERE ec = 'EC 2.3.3.14';
UPDATE enzyme SET history = 'EC 2.3.3.14 created 1972 as EC 4.1.3.21, transferred 2002 to EC 2.3.3.14' WHERE ec = 'EC 2.3.3.14';
UPDATE enzyme SET sys_name = 'Acetyl-phosphate:sulfite S-acetyltransferase (acyl-phosphate hydrolyzing, 2-oxoethyl-forming)' WHERE ec = 'EC 2.3.3.15';
UPDATE enzyme SET history = 'EC 2.3.3.15 created 2003' WHERE ec = 'EC 2.3.3.15';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:oxaloacetate C-acetyltransferase (thioester-hydrolyzing)' WHERE ec = 'EC 2.3.3.16';
UPDATE enzyme SET history = 'EC 2.3.3.16 created 2014' WHERE ec = 'EC 2.3.3.16';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-glucan:phosphate alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.1';
UPDATE enzyme SET history = 'EC 2.4.1.1 created 1961, modified 2013' WHERE ec = 'EC 2.4.1.1';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-glucan:(1-&gt;6)-alpha-D-glucan 6-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.2';
UPDATE enzyme SET history = 'EC 2.4.1.2 created 1961' WHERE ec = 'EC 2.4.1.2';
UPDATE enzyme SET history = 'EC 2.4.1.3 created 1961, deleted 1972' WHERE ec = 'EC 2.4.1.3';
UPDATE enzyme SET sys_name = 'Sucrose:(1-&gt;4)-alpha-D-glucan 4-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.4';
UPDATE enzyme SET history = 'EC 2.4.1.4 created 1961' WHERE ec = 'EC 2.4.1.4';
UPDATE enzyme SET sys_name = 'Sucrose:(1-&gt;6)-alpha-D-glucan 6-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.5';
UPDATE enzyme SET history = 'EC 2.4.1.5 created 1961' WHERE ec = 'EC 2.4.1.5';
UPDATE enzyme SET history = 'EC 2.4.1.6 created 1961, deleted 1972' WHERE ec = 'EC 2.4.1.6';
UPDATE enzyme SET sys_name = 'Sucrose:phosphate alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.7';
UPDATE enzyme SET history = 'EC 2.4.1.7 created 1961' WHERE ec = 'EC 2.4.1.7';
UPDATE enzyme SET sys_name = 'Maltose:phosphate 1-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.8';
UPDATE enzyme SET history = 'EC 2.4.1.8 created 1961' WHERE ec = 'EC 2.4.1.8';
UPDATE enzyme SET sys_name = 'Sucrose:(2-&gt;1)-beta-D-fructan 1-beta-D-fructosyltransferase' WHERE ec = 'EC 2.4.1.9';
UPDATE enzyme SET history = 'EC 2.4.1.9 created 1961' WHERE ec = 'EC 2.4.1.9';
UPDATE enzyme SET sys_name = 'Sucrose:(6)-beta-D-fructofuranosyl-(2-&gt;)(n) alpha-D-glucopyranoside  6-beta-D-fructosyltransferase' WHERE ec = 'EC 2.4.1.10';
UPDATE enzyme SET history = 'EC 2.4.1.10 created 1961' WHERE ec = 'EC 2.4.1.10';
UPDATE enzyme SET sys_name = 'UDP-glucose:glycogen 4-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.11';
UPDATE enzyme SET history = 'EC 2.4.1.11 created 1961' WHERE ec = 'EC 2.4.1.11';
UPDATE enzyme SET sys_name = 'UDP-glucose:(1-&gt;4)-beta-D-glucan 4-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.12';
UPDATE enzyme SET history = 'EC 2.4.1.12 created 1961' WHERE ec = 'EC 2.4.1.12';
UPDATE enzyme SET sys_name = 'NDP-glucose:D-fructose 2-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.13';
UPDATE enzyme SET history = 'EC 2.4.1.13 created 1961, modified 2003' WHERE ec = 'EC 2.4.1.13';
UPDATE enzyme SET sys_name = 'UDP-glucose:D-fructose-6-phosphate 2-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.14';
UPDATE enzyme SET history = 'EC 2.4.1.14 created 1961, modified 2008' WHERE ec = 'EC 2.4.1.14';
UPDATE enzyme SET sys_name = 'UDP-glucose:D-glucose-6-phosphate 1-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.15';
UPDATE enzyme SET history = 'EC 2.4.1.15 created 1961' WHERE ec = 'EC 2.4.1.15';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:chitin 4-beta-N-acetylglucosaminyl-transferase' WHERE ec = 'EC 2.4.1.16';
UPDATE enzyme SET history = 'EC 2.4.1.16 created 1961' WHERE ec = 'EC 2.4.1.16';
UPDATE enzyme SET sys_name = 'UDP-glucuronate beta-D-glucuronosyltransferase (acceptor-unspecific)' WHERE ec = 'EC 2.4.1.17';
UPDATE enzyme SET history = 'EC 2.4.1.17 created 1961 (EC 2.4.1.42, EC 2.4.1.59 and EC 2.4.1.61 all created 1972; EC 2.4.1.76, EC 2.4.1.77 and EC 2.4.1.84 all created 1976; EC 2.4.1.107 and EC 2.4.1.108 both created 1983, all incorporated 1984)' WHERE ec = 'EC 2.4.1.17';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-glucan:(1-&gt;4)-alpha-D-glucan 6-alpha-D-((1-&gt;4)-alpha-D-glucano)-transferase' WHERE ec = 'EC 2.4.1.18';
UPDATE enzyme SET history = 'EC 2.4.1.18 created 1961' WHERE ec = 'EC 2.4.1.18';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-glucan:(1-&gt;4)-alpha-D-glucan 4-alpha-D-((1-&gt;4)-alpha-D-glucano)-transferase (cyclizing)' WHERE ec = 'EC 2.4.1.19';
UPDATE enzyme SET history = 'EC 2.4.1.19 created 1961' WHERE ec = 'EC 2.4.1.19';
UPDATE enzyme SET sys_name = 'Cellobiose:phosphate alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.20';
UPDATE enzyme SET history = 'EC 2.4.1.20 created 1965' WHERE ec = 'EC 2.4.1.20';
UPDATE enzyme SET sys_name = 'ADP-glucose:(1-&gt;4)-alpha-D-glucan 4-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.21';
UPDATE enzyme SET history = 'EC 2.4.1.21 created 1965' WHERE ec = 'EC 2.4.1.21';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:D-glucose 4-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.22';
UPDATE enzyme SET history = 'EC 2.4.1.22 created 1965' WHERE ec = 'EC 2.4.1.22';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:sphingosine 1-beta-galactosyltransferase' WHERE ec = 'EC 2.4.1.23';
UPDATE enzyme SET history = 'EC 2.4.1.23 created 1965' WHERE ec = 'EC 2.4.1.23';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-glucan:(1-&gt;4)-alpha-D-glucan(D-glucose) 6-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.24';
UPDATE enzyme SET history = 'EC 2.4.1.24 created 1965' WHERE ec = 'EC 2.4.1.24';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-glucan:(1-&gt;4)-alpha-D-glucan 4-alpha-D-glycosyltransferase' WHERE ec = 'EC 2.4.1.25';
UPDATE enzyme SET history = 'EC 2.4.1.25 created 1965 (EC 2.4.1.3 created 1961, incorporated 1972)' WHERE ec = 'EC 2.4.1.25';
UPDATE enzyme SET sys_name = 'UDP-glucose:DNA alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.26';
UPDATE enzyme SET history = 'EC 2.4.1.26 created 1965' WHERE ec = 'EC 2.4.1.26';
UPDATE enzyme SET sys_name = 'UDP-glucose:DNA beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.27';
UPDATE enzyme SET history = 'EC 2.4.1.27 created 1965' WHERE ec = 'EC 2.4.1.27';
UPDATE enzyme SET sys_name = 'UDP-glucose:D-glucosyl-DNA beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.28';
UPDATE enzyme SET history = 'EC 2.4.1.28 created 1965' WHERE ec = 'EC 2.4.1.28';
UPDATE enzyme SET sys_name = 'GDP-glucose:(1-&gt;4)-beta-D-glucan 4-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.29';
UPDATE enzyme SET history = 'EC 2.4.1.29 created 1965' WHERE ec = 'EC 2.4.1.29';
UPDATE enzyme SET sys_name = '(1-&gt;3)-beta-D-oligoglucan:phosphate alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.30';
UPDATE enzyme SET history = 'EC 2.4.1.30 created 1972' WHERE ec = 'EC 2.4.1.30';
UPDATE enzyme SET sys_name = '3-beta-D-glucosyl-D-glucose:phosphate alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.31';
UPDATE enzyme SET history = 'EC 2.4.1.31 created 1972' WHERE ec = 'EC 2.4.1.31';
UPDATE enzyme SET sys_name = 'GDP-mannose:glucomannan 4-beta-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.32';
UPDATE enzyme SET history = 'EC 2.4.1.32 created 1972' WHERE ec = 'EC 2.4.1.32';
UPDATE enzyme SET sys_name = 'GDP-D-mannuronate:alginate D-mannuronyltransferase' WHERE ec = 'EC 2.4.1.33';
UPDATE enzyme SET history = 'EC 2.4.1.33 created 1972' WHERE ec = 'EC 2.4.1.33';
UPDATE enzyme SET sys_name = 'UDP-glucose:(1-&gt;3)-beta-D-glucan 3-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.34';
UPDATE enzyme SET history = 'EC 2.4.1.34 created 1972' WHERE ec = 'EC 2.4.1.34';
UPDATE enzyme SET sys_name = 'UDP-glucose:phenol beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.35';
UPDATE enzyme SET history = 'EC 2.4.1.35 created 1972' WHERE ec = 'EC 2.4.1.35';
UPDATE enzyme SET sys_name = 'GDP-glucose:D-glucose-6-phosphate 1-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.36';
UPDATE enzyme SET history = 'EC 2.4.1.36 created 1972' WHERE ec = 'EC 2.4.1.36';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:alpha-L-fucosyl-(1-&gt;2)-D-galactoside 3-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.37';
UPDATE enzyme SET history = 'EC 2.4.1.37 created 1972, modified 1999, modified 2002' WHERE ec = 'EC 2.4.1.37';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:N-acetyl-beta-D-glucosaminylglycopeptide 4-beta-galactosyltransferase' WHERE ec = 'EC 2.4.1.38';
UPDATE enzyme SET history = 'EC 2.4.1.38 created 1972, modified 1976, modified 1980, modified 1986' WHERE ec = 'EC 2.4.1.38';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:estradiol-17-alpha-3-D-glucuronoside 17-alpha-N-acetylglucosaminyltransferase' WHERE ec = 'EC 2.4.1.39';
UPDATE enzyme SET history = 'EC 2.4.1.39 created 1972' WHERE ec = 'EC 2.4.1.39';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-galactosamine:glycoprotein-alpha-L-fucosyl-(1-&gt;2)-D-galactose 3-N-acetyl-D-galactosaminyltransferase' WHERE ec = 'EC 2.4.1.40';
UPDATE enzyme SET history = 'EC 2.4.1.40 created 1972, modified 1999' WHERE ec = 'EC 2.4.1.40';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:polypeptide N-acetylgalactosaminyl-transferase' WHERE ec = 'EC 2.4.1.41';
UPDATE enzyme SET history = 'EC 2.4.1.41 created 1972, modified 1989' WHERE ec = 'EC 2.4.1.41';
UPDATE enzyme SET history = 'EC 2.4.1.42 created 1972, deleted 1984' WHERE ec = 'EC 2.4.1.42';
UPDATE enzyme SET sys_name = 'UDP-D-galacturonate:(1-&gt;4)-alpha-poly-D-galacturonate 4-alpha-D-galacturonosyltransferase' WHERE ec = 'EC 2.4.1.43';
UPDATE enzyme SET history = 'EC 2.4.1.43 created 1972' WHERE ec = 'EC 2.4.1.43';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:lipopolysaccharide 3-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.44';
UPDATE enzyme SET history = 'EC 2.4.1.44 created 1972, modified 2002' WHERE ec = 'EC 2.4.1.44';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:2-(2-hydroxyacyl)sphingosine 1-beta-D-galactosyl-transferase' WHERE ec = 'EC 2.4.1.45';
UPDATE enzyme SET history = 'EC 2.4.1.45 created 1972' WHERE ec = 'EC 2.4.1.45';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:1,2-diacyl-sn-glycerol 3-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.46';
UPDATE enzyme SET history = 'EC 2.4.1.46 created 1972, modified 2005' WHERE ec = 'EC 2.4.1.46';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:N-acylsphingosine D-galactosyltransferase' WHERE ec = 'EC 2.4.1.47';
UPDATE enzyme SET history = 'EC 2.4.1.47 created 1972' WHERE ec = 'EC 2.4.1.47';
UPDATE enzyme SET sys_name = 'GDP-mannose:heteroglycan 2-(or 3-)-alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.48';
UPDATE enzyme SET history = 'EC 2.4.1.48 created 1972' WHERE ec = 'EC 2.4.1.48';
UPDATE enzyme SET sys_name = '(1-&gt;4)-beta-D-oligo-D-glucan:phosphate alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.49';
UPDATE enzyme SET history = 'EC 2.4.1.49 created 1972' WHERE ec = 'EC 2.4.1.49';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:procollagen-5-hydroxy-L-lysine D-galactosyltransferase' WHERE ec = 'EC 2.4.1.50';
UPDATE enzyme SET history = 'EC 2.4.1.50 created 1972, modified 1983' WHERE ec = 'EC 2.4.1.50';
UPDATE enzyme SET history = 'EC 2.4.1.51 created 1972, deleted 1984' WHERE ec = 'EC 2.4.1.51';
UPDATE enzyme SET sys_name = 'UDP-glucose:poly(glycerol-phosphate) alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.52';
UPDATE enzyme SET history = 'EC 2.4.1.52 created 1972' WHERE ec = 'EC 2.4.1.52';
UPDATE enzyme SET sys_name = 'UDP-glucose:poly(ribitol-phosphate) beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.53';
UPDATE enzyme SET history = 'EC 2.4.1.53 created 1972' WHERE ec = 'EC 2.4.1.53';
UPDATE enzyme SET sys_name = 'GDP-mannose:undecaprenyl-phosphate D-mannosyltransferase' WHERE ec = 'EC 2.4.1.54';
UPDATE enzyme SET history = 'EC 2.4.1.54 created 1972' WHERE ec = 'EC 2.4.1.54';
UPDATE enzyme SET history = 'EC 2.4.1.55 created 1972, deleted 1982' WHERE ec = 'EC 2.4.1.55';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:lipopolysaccharide N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.56';
UPDATE enzyme SET history = 'EC 2.4.1.56 created 1972' WHERE ec = 'EC 2.4.1.56';
UPDATE enzyme SET sys_name = 'GDP-mannose:1-phosphatidyl-1D-myo-inositol alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.57';
UPDATE enzyme SET history = 'EC 2.4.1.57 created 1972, modified 2003' WHERE ec = 'EC 2.4.1.57';
UPDATE enzyme SET sys_name = 'UDP-glucose:lipopolysaccharide glucosyltransferase' WHERE ec = 'EC 2.4.1.58';
UPDATE enzyme SET history = 'EC 2.4.1.58 created 1972' WHERE ec = 'EC 2.4.1.58';
UPDATE enzyme SET history = 'EC 2.4.1.59 created 1972, deleted 1984' WHERE ec = 'EC 2.4.1.59';
UPDATE enzyme SET sys_name = 'CDP-D-abequose:Man(alpha-1-&gt;4)Rha(alpha-1-&gt;3)Gal(beta-1)-diphospholipid D-abequosyltransferase' WHERE ec = 'EC 2.4.1.60';
UPDATE enzyme SET history = 'EC 2.4.1.60 created 1972, modified 2012' WHERE ec = 'EC 2.4.1.60';
UPDATE enzyme SET history = 'EC 2.4.1.61 created 1972, deleted 1984' WHERE ec = 'EC 2.4.1.61';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:N-acetyl-beta-D-galactosaminyl-(1-&gt;4)-(alpha-N-acetylneuraminyl-(2-&gt;3))-beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide 3-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.62';
UPDATE enzyme SET history = 'EC 2.4.1.62 created 1972, modified 2013' WHERE ec = 'EC 2.4.1.62';
UPDATE enzyme SET sys_name = 'UDP-glucose:2-hydroxy-2-methylpropanenitrile beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.63';
UPDATE enzyme SET history = 'EC 2.4.1.63 created 1972' WHERE ec = 'EC 2.4.1.63';
UPDATE enzyme SET sys_name = 'Alpha,alpha-trehalose:phosphate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.64';
UPDATE enzyme SET history = 'EC 2.4.1.64 created 1972' WHERE ec = 'EC 2.4.1.64';
UPDATE enzyme SET sys_name = 'GDP-beta-L-fucose:beta-D-galactosyl-(1-&gt;3)-N-acetyl-D-glucosaminyl-R 4(I)-alpha-L-fucosyltransferase' WHERE ec = 'EC 2.4.1.65';
UPDATE enzyme SET history = 'EC 2.4.1.65 created 1972, modified 2001, modified twice 2002' WHERE ec = 'EC 2.4.1.65';
UPDATE enzyme SET sys_name = 'UDP-glucose:(2S,5R)-5-O-(beta-D-galactosyl)-5-hydroxy-lysine-[procollagen] D-glucosyltransferase' WHERE ec = 'EC 2.4.1.66';
UPDATE enzyme SET history = 'EC 2.4.1.66 created 1972' WHERE ec = 'EC 2.4.1.66';
UPDATE enzyme SET sys_name = 'Alpha-D-galactosyl-(1-&gt;3)-1D-myo-inositol:raffinose galactosyltransferase' WHERE ec = 'EC 2.4.1.67';
UPDATE enzyme SET history = 'EC 2.4.1.67 created 1972, modified 2003' WHERE ec = 'EC 2.4.1.67';
UPDATE enzyme SET sys_name = 'GDP-beta-L-fucose:glycoprotein (L-fucose to asparagine-linked N-acetylglucosamine of N(4)-(N-acetyl-beta-D-glucosaminyl-(1-&gt;2)-alpha-D-mannosyl-(1-&gt;3)-(N-acetyl-beta-D-glucosaminyl-(1-&gt;2)-alpha-D-mannosyl-(1-&gt;6))-beta-D-mannosyl-(1-&gt;4)-N-acetyl-beta-D-glucosaminyl-(1-&gt;4)-N-acetyl-beta-D-glucosaminyl)asparagine) 6-alpha-L-fucosyltransferase' WHERE ec = 'EC 2.4.1.68';
UPDATE enzyme SET history = 'EC 2.4.1.68 created 1972, modified 2002' WHERE ec = 'EC 2.4.1.68';
UPDATE enzyme SET sys_name = 'GDP-beta-L-fucose:beta-D-galactosyl-(1-&gt;3)-N-acetyl-beta-D-glucosaminyl-(1-&gt;3)-beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide 2-alpha-L-fucosyltransferase' WHERE ec = 'EC 2.4.1.69';
UPDATE enzyme SET history = 'EC 2.4.1.69 created 1972 (EC 2.4.1.89 created 1976, incorporated 1984)' WHERE ec = 'EC 2.4.1.69';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:poly(ribitol-phosphate) N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.70';
UPDATE enzyme SET history = 'EC 2.4.1.70 created 1972' WHERE ec = 'EC 2.4.1.70';
UPDATE enzyme SET sys_name = 'UDP-glucose:arylamine N-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.71';
UPDATE enzyme SET history = 'EC 2.4.1.71 created 1972' WHERE ec = 'EC 2.4.1.71';
UPDATE enzyme SET history = 'EC 2.4.1.72 created 1972, deleted 1976' WHERE ec = 'EC 2.4.1.72';
UPDATE enzyme SET sys_name = 'UDP-glucose:galactosyl-lipopolysaccharide alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.73';
UPDATE enzyme SET history = 'EC 2.4.1.73 created 1972' WHERE ec = 'EC 2.4.1.73';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:glycosaminoglycan D-galactosyltransferase' WHERE ec = 'EC 2.4.1.74';
UPDATE enzyme SET history = 'EC 2.4.1.74 created 1972, modified 1980' WHERE ec = 'EC 2.4.1.74';
UPDATE enzyme SET history = 'EC 2.4.1.75 created 1976, deleted 2005' WHERE ec = 'EC 2.4.1.75';
UPDATE enzyme SET history = 'EC 2.4.1.76 created 1976, deleted 1984' WHERE ec = 'EC 2.4.1.76';
UPDATE enzyme SET history = 'EC 2.4.1.77 created 1976, deleted 1984' WHERE ec = 'EC 2.4.1.77';
UPDATE enzyme SET sys_name = 'UDP-glucose:phosphopolyprenol D-glucosyltransferase' WHERE ec = 'EC 2.4.1.78';
UPDATE enzyme SET history = 'EC 2.4.1.78 created 1976' WHERE ec = 'EC 2.4.1.78';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:alpha-D-galactosyl-(1-&gt;4)-beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide III(3)-beta-N-acetyl-D-galactosaminyltransferase' WHERE ec = 'EC 2.4.1.79';
UPDATE enzyme SET history = 'EC 2.4.1.79 created 1976, modified 2006' WHERE ec = 'EC 2.4.1.79';
UPDATE enzyme SET sys_name = 'UDP-glucose:N-acylsphingosine D-glucosyltransferase' WHERE ec = 'EC 2.4.1.80';
UPDATE enzyme SET history = 'EC 2.4.1.80 created 1976' WHERE ec = 'EC 2.4.1.80';
UPDATE enzyme SET sys_name = 'UDP-glucose:5,7,3,4-tetrahydroxyflavone 7-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.81';
UPDATE enzyme SET history = 'EC 2.4.1.81 created 1976' WHERE ec = 'EC 2.4.1.81';
UPDATE enzyme SET sys_name = 'Alpha-D-galactosyl-(1-&gt;3)-1D-myo-inositol:sucrose 6-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.82';
UPDATE enzyme SET history = 'EC 2.4.1.82 created 1976, modified 2003' WHERE ec = 'EC 2.4.1.82';
UPDATE enzyme SET sys_name = 'GDP-mannose:dolichyl-phosphate beta-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.83';
UPDATE enzyme SET history = 'EC 2.4.1.83 created 1976, modified 1983' WHERE ec = 'EC 2.4.1.83';
UPDATE enzyme SET history = 'EC 2.4.1.84 created 1976, deleted 1984' WHERE ec = 'EC 2.4.1.84';
UPDATE enzyme SET sys_name = 'UDP-D-glucose:(S)-4-hydroxymandelonitrile beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.85';
UPDATE enzyme SET history = 'EC 2.4.1.85 created 1976, modified 2005' WHERE ec = 'EC 2.4.1.85';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:N-acetyl-beta-D-glucosaminyl-(1-&gt;3)-beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide 3-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.86';
UPDATE enzyme SET history = 'EC 2.4.1.86 created 1976' WHERE ec = 'EC 2.4.1.86';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:beta-D-galactosyl-(1-&gt;4)-beta-N-acetyl-D-glucosaminyl-R 3-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.87';
UPDATE enzyme SET history = 'EC 2.4.1.87 created 1976, modified 1989, modified 2002 (EC 2.4.1.124 created 1984, incorporated 2002; EC 2.4.1.151 created 1984, incorporated 2002)' WHERE ec = 'EC 2.4.1.87';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:N-acetyl-D-galactosaminyl-(1-&gt;3)-D-galactosyl-(1-&gt;4)-D-galactosyl-(1-&gt;4)-D-glucosyl-(1&lt;-&gt;1)-ceramide alpha-N-acetyl-D-galactosaminyltransferase' WHERE ec = 'EC 2.4.1.88';
UPDATE enzyme SET history = 'EC 2.4.1.88 created 1976' WHERE ec = 'EC 2.4.1.88';
UPDATE enzyme SET history = 'EC 2.4.1.89 created 1976, deleted 1984' WHERE ec = 'EC 2.4.1.89';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:N-acetyl-D-glucosamine 4-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.90';
UPDATE enzyme SET history = 'EC 2.4.1.90 created 1976 (EC 2.4.1.98 created 1980, incorporated 1984)' WHERE ec = 'EC 2.4.1.90';
UPDATE enzyme SET sys_name = 'UDP-glucose:flavonol 3-O-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.91';
UPDATE enzyme SET history = 'EC 2.4.1.91 created 1976' WHERE ec = 'EC 2.4.1.91';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:1-O-(O-(N-acetyl-alpha-neuraminosyl)-(2-&gt;3)-O-beta-D-galactopyranosyl-(1-&gt;4)-beta-D-glucopyranosyl)-ceramide 4-beta-N-acetyl-D-galactosaminyltransferase' WHERE ec = 'EC 2.4.1.92';
UPDATE enzyme SET history = 'EC 2.4.1.92 created 1976, modified 2006' WHERE ec = 'EC 2.4.1.92';
UPDATE enzyme SET history = 'EC 2.4.1.93 created 1976, deleted 2004' WHERE ec = 'EC 2.4.1.93';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:[protein]-L-asparagine beta-N-acetyl-D-glucosaminyl-transferase' WHERE ec = 'EC 2.4.1.94';
UPDATE enzyme SET history = 'EC 2.4.1.94 created 1978, modified 2010' WHERE ec = 'EC 2.4.1.94';
UPDATE enzyme SET sys_name = 'Bilirubin-glucuronoside:bilirubin-glucuronoside D-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.95';
UPDATE enzyme SET history = 'EC 2.4.1.95 created 1978' WHERE ec = 'EC 2.4.1.95';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:sn-glycerol-3-phosphate 1-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.96';
UPDATE enzyme SET history = 'EC 2.4.1.96 created 1978' WHERE ec = 'EC 2.4.1.96';
UPDATE enzyme SET sys_name = '(1-&gt;3)-beta-D-glucan:phosphate alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.97';
UPDATE enzyme SET history = 'EC 2.4.1.97 created 1978' WHERE ec = 'EC 2.4.1.97';
UPDATE enzyme SET history = 'EC 2.4.1.98 created 1980, deleted 1984' WHERE ec = 'EC 2.4.1.98';
UPDATE enzyme SET sys_name = 'Sucrose:sucrose 1-beta-D-fructosyltransferase' WHERE ec = 'EC 2.4.1.99';
UPDATE enzyme SET history = 'EC 2.4.1.99 created 1981, modified 2004' WHERE ec = 'EC 2.4.1.99';
UPDATE enzyme SET sys_name = '(2-&gt;1)-beta-D-fructan:(2-&gt;1)-beta-D-fructan 1-beta-D-fructosyltransferase' WHERE ec = 'EC 2.4.1.100';
UPDATE enzyme SET history = 'EC 2.4.1.100 created 1981, modified 2004' WHERE ec = 'EC 2.4.1.100';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:3-(alpha-D-mannosyl)-beta-D-mannosyl-glycoprotein 2-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.101';
UPDATE enzyme SET history = 'EC 2.4.1.101 created 1983, modified 2001 (EC 2.4.1.51 created 1972, part incorporated 1984)' WHERE ec = 'EC 2.4.1.101';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:O-glycosyl-glycoprotein (N-acetyl-D-glucosamine to N-acetyl-D-galactosamine of beta-D-galactosyl-(1-&gt;3)-N-acetyl-D-galactosaminyl-R) 6-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.102';
UPDATE enzyme SET history = 'EC 2.4.1.102 created 1983' WHERE ec = 'EC 2.4.1.102';
UPDATE enzyme SET sys_name = 'UDP-glucose:1,2-dihydroxy-9,10-anthraquinone 2-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.103';
UPDATE enzyme SET history = 'EC 2.4.1.103 created 1983' WHERE ec = 'EC 2.4.1.103';
UPDATE enzyme SET sys_name = 'UDP-glucose:7,8-dihydroxycoumarin 7-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.104';
UPDATE enzyme SET history = 'EC 2.4.1.104 created 1983' WHERE ec = 'EC 2.4.1.104';
UPDATE enzyme SET sys_name = 'UDP-glucose:vitexin 2-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.105';
UPDATE enzyme SET history = 'EC 2.4.1.105 created 1983' WHERE ec = 'EC 2.4.1.105';
UPDATE enzyme SET sys_name = 'UDP-glucose:isovitexin 2-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.106';
UPDATE enzyme SET history = 'EC 2.4.1.106 created 1983' WHERE ec = 'EC 2.4.1.106';
UPDATE enzyme SET history = 'EC 2.4.1.107 created 1983, deleted 1984' WHERE ec = 'EC 2.4.1.107';
UPDATE enzyme SET history = 'EC 2.4.1.108 created 1983, deleted 1984' WHERE ec = 'EC 2.4.1.108';
UPDATE enzyme SET sys_name = 'Dolichyl-phosphate-D-mannose:protein O-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.109';
UPDATE enzyme SET history = 'EC 2.4.1.109 created 1983' WHERE ec = 'EC 2.4.1.109';
UPDATE enzyme SET sys_name = 'GDP-mannose:tRNA(Asp)-queuosine O-5-beta-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.110';
UPDATE enzyme SET history = 'EC 2.4.1.110 created 1984' WHERE ec = 'EC 2.4.1.110';
UPDATE enzyme SET sys_name = 'UDP-glucose:coniferyl-alcohol 4-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.111';
UPDATE enzyme SET history = 'EC 2.4.1.111 created 1984' WHERE ec = 'EC 2.4.1.111';
UPDATE enzyme SET history = 'EC 2.4.1.112 created 1984, deleted 2007' WHERE ec = 'EC 2.4.1.112';
UPDATE enzyme SET sys_name = 'ADP-glucose:protein 4-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.113';
UPDATE enzyme SET history = 'EC 2.4.1.113 created 1984' WHERE ec = 'EC 2.4.1.113';
UPDATE enzyme SET sys_name = 'UDP-glucose:trans-2-hydroxycinnamate O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.114';
UPDATE enzyme SET history = 'EC 2.4.1.114 created 1984' WHERE ec = 'EC 2.4.1.114';
UPDATE enzyme SET sys_name = 'UDP-D-glucose:anthocyanidin 3-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.115';
UPDATE enzyme SET history = 'EC 2.4.1.115 created 1984 (EC 2.4.1.233 created 2004, incorporated 2005), modified 2005' WHERE ec = 'EC 2.4.1.115';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:cyanidin-3-O-alpha-L-rhamnosyl-(1-&gt;6)-beta-D-glucoside 5-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.116';
UPDATE enzyme SET history = 'EC 2.4.1.116 created 1984  (EC 2.4.1.235 created 2004, incorporated 2006), modified 2006, modified 2013' WHERE ec = 'EC 2.4.1.116';
UPDATE enzyme SET sys_name = 'UDP-glucose:dolichyl-phosphate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.117';
UPDATE enzyme SET history = 'EC 2.4.1.117 created 1984' WHERE ec = 'EC 2.4.1.117';
UPDATE enzyme SET sys_name = 'UDP-glucose:zeatin 7-glucosyltransferase' WHERE ec = 'EC 2.4.1.118';
UPDATE enzyme SET history = 'EC 2.4.1.118 created 1984' WHERE ec = 'EC 2.4.1.118';
UPDATE enzyme SET history = 'EC 2.4.1.119 created 1984, deleted 2012' WHERE ec = 'EC 2.4.1.119';
UPDATE enzyme SET sys_name = 'UDP-glucose:sinapate D-glucosyltransferase' WHERE ec = 'EC 2.4.1.120';
UPDATE enzyme SET history = 'EC 2.4.1.120 created 1984' WHERE ec = 'EC 2.4.1.120';
UPDATE enzyme SET sys_name = 'UDP-glucose:(indol-3-yl)acetate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.121';
UPDATE enzyme SET history = 'EC 2.4.1.121 created 1984' WHERE ec = 'EC 2.4.1.121';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:glycoprotein-N-acetyl-D-galactosamine 3-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.122';
UPDATE enzyme SET history = 'EC 2.4.1.122 created 1984' WHERE ec = 'EC 2.4.1.122';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:myo-inositol 3-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.123';
UPDATE enzyme SET history = 'EC 2.4.1.123 created 1984, modified 2003' WHERE ec = 'EC 2.4.1.123';
UPDATE enzyme SET history = 'EC 2.4.1.124 created 1984, deleted 2002' WHERE ec = 'EC 2.4.1.124';
UPDATE enzyme SET sys_name = 'Sucrose:(1-&gt;6)-alpha-D-glucan 3(6)-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.125';
UPDATE enzyme SET history = 'EC 2.4.1.125 created 1984' WHERE ec = 'EC 2.4.1.125';
UPDATE enzyme SET sys_name = 'UDP-glucose:trans-4-hydroxycinnamate 4-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.126';
UPDATE enzyme SET history = 'EC 2.4.1.126 created 1984' WHERE ec = 'EC 2.4.1.126';
UPDATE enzyme SET sys_name = 'UDP-glucose:(-)-menthol O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.127';
UPDATE enzyme SET history = 'EC 2.4.1.127 created 1984' WHERE ec = 'EC 2.4.1.127';
UPDATE enzyme SET sys_name = 'UDP-glucose:scopoletin O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.128';
UPDATE enzyme SET history = 'EC 2.4.1.128 created 1984' WHERE ec = 'EC 2.4.1.128';
UPDATE enzyme SET sys_name = '(Poly-N-acetyl-D-glucosaminyl-(1-&gt;4)-(N-acetyl-D-muramoylpentapeptide))-diphosphoundecaprenol:(N-acetyl-D-glucosaminyl-(1-&gt;4)-N-acetyl-D-muramoylpentapeptide)-diphosphoundecaprenol disaccharidetransferase' WHERE ec = 'EC 2.4.1.129';
UPDATE enzyme SET history = 'EC 2.4.1.129 created 1984, modified 2002' WHERE ec = 'EC 2.4.1.129';
UPDATE enzyme SET history = 'EC 2.4.1.130 created 1984, deleted 2011' WHERE ec = 'EC 2.4.1.130';
UPDATE enzyme SET sys_name = 'GDP-alpha-D-mannose:D-Man-alpha-(1-&gt;3)-(D-Man-alpha-(1-&gt;6))-D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol 2-alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.131';
UPDATE enzyme SET history = 'EC 2.4.1.131 created 1984, modified 2011, modified 2012' WHERE ec = 'EC 2.4.1.131';
UPDATE enzyme SET sys_name = 'GDP-D-mannose:D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol 3-alpha-mannosyltransferase' WHERE ec = 'EC 2.4.1.132';
UPDATE enzyme SET history = 'EC 2.4.1.132 created 1984, modified 2011, modified 2012' WHERE ec = 'EC 2.4.1.132';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:O-beta-D-xylosylprotein 4-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.133';
UPDATE enzyme SET history = 'EC 2.4.1.133 created 1984, modified 2002' WHERE ec = 'EC 2.4.1.133';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:4-beta-D-galactosyl-O-beta-D-xylosylprotein 3-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.134';
UPDATE enzyme SET history = 'EC 2.4.1.134 created 1984, modified 2002' WHERE ec = 'EC 2.4.1.134';
UPDATE enzyme SET sys_name = 'UDP-glucuronate:3-beta-D-galactosyl-4-beta-D-galactosyl-O-beta-D-xylosylprotein D-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.135';
UPDATE enzyme SET history = 'EC 2.4.1.135 created 1984, modified 2002' WHERE ec = 'EC 2.4.1.135';
UPDATE enzyme SET sys_name = 'UDP-glucose:gallate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.136';
UPDATE enzyme SET history = 'EC 2.4.1.136 created 1984' WHERE ec = 'EC 2.4.1.136';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:sn-glycerol-3-phosphate 2-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.137';
UPDATE enzyme SET history = 'EC 2.4.1.137 created 1984' WHERE ec = 'EC 2.4.1.137';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:alpha-D-mannosyl-(1-&gt;3)-alpha-D-mannosyl-(1-&gt;2)-alpha-D-mannosyl-(1-&gt;2)-D-mannose alpha-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.138';
UPDATE enzyme SET history = 'EC 2.4.1.138 created 1984' WHERE ec = 'EC 2.4.1.138';
UPDATE enzyme SET sys_name = 'Alpha-D-glucose-1-phosphate:alpha-D-glucose-1-phosphate 4-alpha-D-glucosyltransferase (dephosphorylating)' WHERE ec = 'EC 2.4.1.139';
UPDATE enzyme SET history = 'EC 2.4.1.139 created 1984' WHERE ec = 'EC 2.4.1.139';
UPDATE enzyme SET sys_name = 'Sucrose:(1-&gt;6)((1-&gt;3))-alpha-D-glucan 6(3)-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.140';
UPDATE enzyme SET history = 'EC 2.4.1.140 created 1984, modified 2003' WHERE ec = 'EC 2.4.1.140';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:N-acetyl-D-glucosaminyl-diphosphodolichol N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.141';
UPDATE enzyme SET history = 'EC 2.4.1.141 created 1984' WHERE ec = 'EC 2.4.1.141';
UPDATE enzyme SET sys_name = 'GDP-mannose:chitobiosyldiphosphodolichol beta-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.142';
UPDATE enzyme SET history = 'EC 2.4.1.142 created 1984, modified 2001' WHERE ec = 'EC 2.4.1.142';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:6-(alpha-D-mannosyl)-beta-D-mannosyl-glycoprotein 2-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.143';
UPDATE enzyme SET history = 'EC 2.4.1.143 created 1984, modified 2001 (EC 2.4.1.51 created 1972, part incorporated 1984)' WHERE ec = 'EC 2.4.1.143';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:beta-D-mannosyl-glycoprotein 4-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.144';
UPDATE enzyme SET history = 'EC 2.4.1.144 created 1984, modified 2001 (EC 2.4.1.51 created 1972, part incorporated 1984)' WHERE ec = 'EC 2.4.1.144';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:3-(2-(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl)-glycoprotein 4-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.145';
UPDATE enzyme SET history = 'EC 2.4.1.145 created 1984, modified 2001 (EC 2.4.1.51 created 1972, part incorporated 1984)' WHERE ec = 'EC 2.4.1.145';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:beta-D-galactosyl-(1-&gt;3)-(N-acetyl-D-glucosaminyl-(1-&gt;6))-N-acetyl-D-galactosaminyl-R 3-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.146';
UPDATE enzyme SET history = 'EC 2.4.1.146 created 1984' WHERE ec = 'EC 2.4.1.146';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:N-acetyl-D-galactosaminyl-R 3-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.147';
UPDATE enzyme SET history = 'EC 2.4.1.147 created 1984' WHERE ec = 'EC 2.4.1.147';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:N-acetyl-beta-D-glucosaminyl-(1-&gt;3)-N-acetyl--D-glucosaminyl-R 6-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.148';
UPDATE enzyme SET history = 'EC 2.4.1.148 created 1984' WHERE ec = 'EC 2.4.1.148';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:beta-D-galactosyl-(1-&gt;4)-N-acetyl-D-glucosamine 3-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.149';
UPDATE enzyme SET history = 'EC 2.4.1.149 created 1984' WHERE ec = 'EC 2.4.1.149';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:beta-D-galactosyl-(1-&gt;4)-N-acetyl-D-glucosaminide 6-beta-6-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.150';
UPDATE enzyme SET history = 'EC 2.4.1.150 created 1984' WHERE ec = 'EC 2.4.1.150';
UPDATE enzyme SET history = 'EC 2.4.1.124 created 1984, deleted 2002' WHERE ec = 'EC 2.4.1.151';
UPDATE enzyme SET sys_name = 'GDP-beta-L-fucose:(1-&gt;4)-beta-D-galactosyl-N-acetyl-D-glucosaminyl-R 3-alpha-L-fucosyltransferase' WHERE ec = 'EC 2.4.1.152';
UPDATE enzyme SET history = 'EC 2.4.1.152 created 1984, modified 2002' WHERE ec = 'EC 2.4.1.152';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:dolichyl-phosphate alpha-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.153';
UPDATE enzyme SET history = 'EC 2.4.1.153 created 1984' WHERE ec = 'EC 2.4.1.153';
UPDATE enzyme SET history = 'EC 2.4.1.154 created 1986, deleted 2006' WHERE ec = 'EC 2.4.1.154';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:6-(2-(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl)-glycoprotein 6-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.155';
UPDATE enzyme SET history = 'EC 2.4.1.155 created 1986, modified 2001' WHERE ec = 'EC 2.4.1.155';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:(indol-3-yl)acetyl-myo-inositol 5-O-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.156';
UPDATE enzyme SET history = 'EC 2.4.1.156 created 1986' WHERE ec = 'EC 2.4.1.156';
UPDATE enzyme SET sys_name = 'UDP-glucose:1,2-diacyl-sn-glycerol 3-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.157';
UPDATE enzyme SET history = 'EC 2.4.1.157 created 1986' WHERE ec = 'EC 2.4.1.157';
UPDATE enzyme SET sys_name = 'UDP-glucose:13-hydroxydocosanoate 13-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.158';
UPDATE enzyme SET history = 'EC 2.4.1.158 created 1986' WHERE ec = 'EC 2.4.1.158';
UPDATE enzyme SET sys_name = 'UDP-L-rhamnose:flavonol-3-O-D-glucoside 6-O-L-rhamnosyltransferase' WHERE ec = 'EC 2.4.1.159';
UPDATE enzyme SET history = 'EC 2.4.1.159 created 1986' WHERE ec = 'EC 2.4.1.159';
UPDATE enzyme SET sys_name = 'UDP-glucose:pyridoxine 5-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.160';
UPDATE enzyme SET history = 'EC 2.4.1.160 created 1986' WHERE ec = 'EC 2.4.1.160';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-glucan:(1-&gt;4)-alpha-D-glucan 4-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.161';
UPDATE enzyme SET history = 'EC 2.4.1.161 created 1989, modified 2013' WHERE ec = 'EC 2.4.1.161';
UPDATE enzyme SET sys_name = 'Alpha-D-aldosyl-beta-D-fructoside:aldose 1-beta-D-fructosyltransferase' WHERE ec = 'EC 2.4.1.162';
UPDATE enzyme SET history = 'EC 2.4.1.162 created 1989, modified 1999' WHERE ec = 'EC 2.4.1.162';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:beta-D-galactosyl-(1-&gt;4)-N-acetyl-beta-D-glucosaminyl-(1-&gt;3)-beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide 3-beta-N-acetylglucosaminyltransferase' WHERE ec = 'EC 2.4.1.163';
UPDATE enzyme SET history = 'EC 2.4.1.163 created 1989' WHERE ec = 'EC 2.4.1.163';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:D-galactosyl-(1-&gt;4)-N-acetyl-beta-D-glucosaminyl-(1-&gt;3)-beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide 6-beta-N-acetylglucosaminyltransferase' WHERE ec = 'EC 2.4.1.164';
UPDATE enzyme SET history = 'EC 2.4.1.164 created 1989' WHERE ec = 'EC 2.4.1.164';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:N-acetylneuraminyl-(2-&gt;3)-alpha-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide 4-betaN-acetylgalactosaminyltransferase' WHERE ec = 'EC 2.4.1.165';
UPDATE enzyme SET history = 'EC 2.4.1.165 created 1989' WHERE ec = 'EC 2.4.1.165';
UPDATE enzyme SET sys_name = 'Raffinose:raffinose alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.166';
UPDATE enzyme SET history = 'EC 2.4.1.166 created 1989' WHERE ec = 'EC 2.4.1.166';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:sucrose 6(F)-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.167';
UPDATE enzyme SET history = 'EC 2.4.1.167 created 1989' WHERE ec = 'EC 2.4.1.167';
UPDATE enzyme SET sys_name = 'UDP-glucose:xyloglucan 4-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.168';
UPDATE enzyme SET history = 'EC 2.4.1.168 created 1989' WHERE ec = 'EC 2.4.1.168';
UPDATE enzyme SET history = 'EC 2.4.1.169 created 1989, deleted 2003' WHERE ec = 'EC 2.4.1.169';
UPDATE enzyme SET sys_name = 'UDP-glucose:isoflavone 7-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.170';
UPDATE enzyme SET history = 'EC 2.4.1.170 created 1989' WHERE ec = 'EC 2.4.1.170';
UPDATE enzyme SET sys_name = 'UDP-glucose:methyl-ONN-azoxymethanol beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.171';
UPDATE enzyme SET history = 'EC 2.4.1.171 created 1989' WHERE ec = 'EC 2.4.1.171';
UPDATE enzyme SET sys_name = 'UDP-glucose:salicyl-alcohol beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.172';
UPDATE enzyme SET history = 'EC 2.4.1.172 created 1989' WHERE ec = 'EC 2.4.1.172';
UPDATE enzyme SET sys_name = 'UDP-glucose:sterol 3-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.173';
UPDATE enzyme SET history = 'EC 2.4.1.173 created 1989' WHERE ec = 'EC 2.4.1.173';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-galactosamine:D-glucuronyl-(1-&gt;3)-beta-D-galactosyl-proteoglycan 4-beta-N-acetylgalactosaminyltransferase' WHERE ec = 'EC 2.4.1.174';
UPDATE enzyme SET history = 'EC 2.4.1.174 created 1989, modified 2002' WHERE ec = 'EC 2.4.1.174';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:beta-D-glucuronosyl-(1-&gt;3)-N-acetyl-beta-D-galactosaminyl-proteoglycan 4-beta-N-acetylgalactosaminyltransferase' WHERE ec = 'EC 2.4.1.175';
UPDATE enzyme SET history = 'EC 2.4.1.175 created 1989, modified 2002' WHERE ec = 'EC 2.4.1.175';
UPDATE enzyme SET sys_name = 'UDP-glucose:gibberellin 2-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.176';
UPDATE enzyme SET history = 'EC 2.4.1.176 created 1989' WHERE ec = 'EC 2.4.1.176';
UPDATE enzyme SET sys_name = 'UDP-glucose:trans-cinnamate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.177';
UPDATE enzyme SET history = 'EC 2.4.1.177 created 1989' WHERE ec = 'EC 2.4.1.177';
UPDATE enzyme SET sys_name = 'UDP-glucose:4-hydroxymandelonitrile glucosyltransferase' WHERE ec = 'EC 2.4.1.178';
UPDATE enzyme SET history = 'EC 2.4.1.178 created 1989' WHERE ec = 'EC 2.4.1.178';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:D-galactosyl-(1-&gt;4)-beta-D-glucosyl-R 3-beta-galactosyltransferase' WHERE ec = 'EC 2.4.1.179';
UPDATE enzyme SET history = 'EC 2.4.1.179 created 1989' WHERE ec = 'EC 2.4.1.179';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-mannosaminouronate:lipid I N-acetyl-beta-D-mannosaminouronosyltransferase' WHERE ec = 'EC 2.4.1.180';
UPDATE enzyme SET history = 'EC 2.4.1.180 created 1990, modified 2011' WHERE ec = 'EC 2.4.1.180';
UPDATE enzyme SET sys_name = 'UDP-glucose:hydroxyanthraquinone O-glucosyltransferase' WHERE ec = 'EC 2.4.1.181';
UPDATE enzyme SET history = 'EC 2.4.1.181 created 1990' WHERE ec = 'EC 2.4.1.181';
UPDATE enzyme SET sys_name = 'UDP-2,3-bis(3-hydroxytetradecanoyl)glucosamine:2,3-bis-(3-hydroxytetradecanoyl)-beta-D-glucosaminyl-1-phosphate 2,3-bis(3-hydroxytetradecanoyl)-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.182';
UPDATE enzyme SET history = 'EC 2.4.1.182 created 1990' WHERE ec = 'EC 2.4.1.182';
UPDATE enzyme SET sys_name = 'UDP-glucose:alpha-D-(1-&gt;3)-glucan 3-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.183';
UPDATE enzyme SET history = 'EC 2.4.1.183 created 1990' WHERE ec = 'EC 2.4.1.183';
UPDATE enzyme SET sys_name = '3-(beta-D-galactosyl)-1,2-diacyl-sn-glycerol:mono-3-(beta-D-galactosyl)-1,2-diacyl-sn-glycerol beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.184';
UPDATE enzyme SET history = 'EC 2.4.1.184 created 1990, modified 2005' WHERE ec = 'EC 2.4.1.184';
UPDATE enzyme SET sys_name = 'UDP-glucose:flavanone 7-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.185';
UPDATE enzyme SET history = 'EC 2.4.1.185 created 1992' WHERE ec = 'EC 2.4.1.185';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:glycogenin alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.186';
UPDATE enzyme SET history = 'EC 2.4.1.186 created 1992 (EC 2.4.1.112 created 1984, incorporated 2007)' WHERE ec = 'EC 2.4.1.186';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-mannosamine:N-acetyl-beta-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol 4-beta-4-N-acetylmannosaminyltransferase' WHERE ec = 'EC 2.4.1.187';
UPDATE enzyme SET history = 'EC 2.4.1.187 created 1992' WHERE ec = 'EC 2.4.1.187';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:N-acetyl-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol 4-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.188';
UPDATE enzyme SET history = 'EC 2.4.1.188 created 1992' WHERE ec = 'EC 2.4.1.188';
UPDATE enzyme SET sys_name = 'UDP-glucuronate:luteolin 7-O-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.189';
UPDATE enzyme SET history = 'EC 2.4.1.189 created 1992' WHERE ec = 'EC 2.4.1.189';
UPDATE enzyme SET sys_name = 'UDP-glucuronate:luteolin-7-O-beta-D-glucuronide 2-O-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.190';
UPDATE enzyme SET history = 'EC 2.4.1.190 created 1992' WHERE ec = 'EC 2.4.1.190';
UPDATE enzyme SET sys_name = 'UDP-glucuronate:luteolin-7-O-beta-D-diglucuronide 4-O-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.191';
UPDATE enzyme SET history = 'EC 2.4.1.191 created 1992, modified 2011' WHERE ec = 'EC 2.4.1.191';
UPDATE enzyme SET sys_name = 'UDP-glucose:(20S,22S,25S)-22,25-epoxyfurost-5-ene-3-beta,26-diol 3-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.192';
UPDATE enzyme SET history = 'EC 2.4.1.192 created 1992' WHERE ec = 'EC 2.4.1.192';
UPDATE enzyme SET sys_name = 'UDP-glucose:(25S)-5-beta-spirostan-3-beta-ol 3-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.193';
UPDATE enzyme SET history = 'EC 2.4.1.193 created 1992' WHERE ec = 'EC 2.4.1.193';
UPDATE enzyme SET sys_name = 'UDP-glucose:4-hydroxybenzoate 4-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.194';
UPDATE enzyme SET history = 'EC 2.4.1.194 created 1992' WHERE ec = 'EC 2.4.1.194';
UPDATE enzyme SET sys_name = 'UDP-glucose:N-hydroxy-2-phenylethanethioamide S-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.195';
UPDATE enzyme SET history = 'EC 2.4.1.195 created 1992, modified 2006' WHERE ec = 'EC 2.4.1.195';
UPDATE enzyme SET sys_name = 'UDP-glucose:nicotinate N-glucosyltransferase' WHERE ec = 'EC 2.4.1.196';
UPDATE enzyme SET history = 'EC 2.4.1.196 created 1992' WHERE ec = 'EC 2.4.1.196';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:high-mannose-oligosaccharide 4-beta-N-acetylglucosaminyltransferase' WHERE ec = 'EC 2.4.1.197';
UPDATE enzyme SET history = 'EC 2.4.1.197 created 1992' WHERE ec = 'EC 2.4.1.197';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:1-phosphatidyl-1D-myo-inositol 6-(N-acetyl-alpha-D-glucosaminyl)transferase' WHERE ec = 'EC 2.4.1.198';
UPDATE enzyme SET history = 'EC 2.4.1.198 created 1992, modified 2002' WHERE ec = 'EC 2.4.1.198';
UPDATE enzyme SET sys_name = 'Beta-D-mannosylphosphodecaprenol:(1-&gt;6)-alpha-D-mannosyloligosaccharide 1,6-alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.199';
UPDATE enzyme SET history = 'EC 2.4.1.199 created 1992' WHERE ec = 'EC 2.4.1.199';
UPDATE enzyme SET history = 'EC 2.4.1.200 created 1992, deleted 2004' WHERE ec = 'EC 2.4.1.200';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:2,6-bis(N-acetyl-beta-D-glucosaminyl)-alpha-D-mannosyl-glycoprotein 4-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.201';
UPDATE enzyme SET history = 'EC 2.4.1.201 created 1992, modified 2001' WHERE ec = 'EC 2.4.1.201';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one 2-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.202';
UPDATE enzyme SET history = 'EC 2.4.1.202 created 1992, modified 2012' WHERE ec = 'EC 2.4.1.202';
UPDATE enzyme SET sys_name = 'UDP-glucose:trans-zeatin O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.203';
UPDATE enzyme SET history = 'EC 2.4.1.203 created 1992, modified 2001' WHERE ec = 'EC 2.4.1.203';
UPDATE enzyme SET history = 'EC 2.4.1.204 created 1992, deleted 2003' WHERE ec = 'EC 2.4.1.204';
UPDATE enzyme SET sys_name = 'UDP-galactose:galactogen 6-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.205';
UPDATE enzyme SET history = 'EC 2.4.1.205 created 1992' WHERE ec = 'EC 2.4.1.205';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide 3-beta-N-acetylglucosaminyltransferase' WHERE ec = 'EC 2.4.1.206';
UPDATE enzyme SET history = 'EC 2.4.1.206 created 1992' WHERE ec = 'EC 2.4.1.206';
UPDATE enzyme SET sys_name = 'Xyloglucan:xyloglucan xyloglucanotransferase' WHERE ec = 'EC 2.4.1.207';
UPDATE enzyme SET history = 'EC 2.4.1.207 created 1999' WHERE ec = 'EC 2.4.1.207';
UPDATE enzyme SET sys_name = 'UDP-glucose:1,2-diacyl-3-O-(alpha-D-glucopyranosyl)-sn-glycerol (1-&gt;2) glucosyltransferase' WHERE ec = 'EC 2.4.1.208';
UPDATE enzyme SET history = 'EC 2.4.1.208 created 1999' WHERE ec = 'EC 2.4.1.208';
UPDATE enzyme SET sys_name = 'UDP-glucose:cis-p-coumarate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.209';
UPDATE enzyme SET history = 'EC 2.4.1.209 created 2000' WHERE ec = 'EC 2.4.1.209';
UPDATE enzyme SET sys_name = 'UDP-glucose:limonin glucosyltransferase' WHERE ec = 'EC 2.4.1.210';
UPDATE enzyme SET history = 'EC 2.4.1.210 created 2000' WHERE ec = 'EC 2.4.1.210';
UPDATE enzyme SET sys_name = 'Beta-D-galactopyranosyl-(1-&gt;3)-N-acetyl-D-hexosamine:phosphate galactosyltransferase' WHERE ec = 'EC 2.4.1.211';
UPDATE enzyme SET history = 'EC 2.4.1.211 created 2001' WHERE ec = 'EC 2.4.1.211';
UPDATE enzyme SET sys_name = 'Alternating UDP-alpha-N-acetyl-D-glucosamine:beta-D-glucuronosyl-(1-&gt;3)-(nascent hyaluronan) 4-N-acetyl-beta-D-glucosaminyltransferase and UDP-alpha-D-glucuronate:N-acetyl-beta-D-glucosaminyl-(1-&gt;4)-(nascent hyaluronan) 3-beta-D-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.212';
UPDATE enzyme SET history = 'EC 2.4.1.212 created 2001, modified 2007' WHERE ec = 'EC 2.4.1.212';
UPDATE enzyme SET sys_name = 'ADP-glucose:sn-glycerol-3-phosphate 2-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.213';
UPDATE enzyme SET history = 'EC 2.4.1.213 created 2001' WHERE ec = 'EC 2.4.1.213';
UPDATE enzyme SET sys_name = 'GDP-beta-L-fucose:glycoprotein (L-fucose to asparagine-linked N-acetylglucosamine of N(4)-(N-acetyl-beta-D-glucosaminyl-(1-&gt;2)-alpha-D-mannosyl-(1-&gt;3)-(N-acetyl-beta-D-glucosaminyl-(1-&gt;2)-alpha-D-mannosyl-(1-&gt;6))-beta-D-mannosyl-(1-&gt;4)-N-acetyl-beta-D-glucosaminyl-(1-&gt;4)-N-acetyl-beta-D-glucosaminyl)asparagine) 3-alpha-L-fucosyl-transferase' WHERE ec = 'EC 2.4.1.214';
UPDATE enzyme SET history = 'EC 2.4.1.214 created 2001' WHERE ec = 'EC 2.4.1.214';
UPDATE enzyme SET sys_name = 'UDP-glucose:cis-zeatin O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.215';
UPDATE enzyme SET history = 'EC 2.4.1.215 created 2001' WHERE ec = 'EC 2.4.1.215';
UPDATE enzyme SET sys_name = 'Alpha,alpha-trehalose 6-phosphate:phosphate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.216';
UPDATE enzyme SET history = 'EC 2.4.1.216 created 2001' WHERE ec = 'EC 2.4.1.216';
UPDATE enzyme SET sys_name = 'GDP-mannose:3-phospho-D-glycerate 3-alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.217';
UPDATE enzyme SET history = 'EC 2.4.1.217 created 2002' WHERE ec = 'EC 2.4.1.217';
UPDATE enzyme SET sys_name = 'UDP-glucose:hydroquinone-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.218';
UPDATE enzyme SET history = 'EC 2.4.1.218 created 2002' WHERE ec = 'EC 2.4.1.218';
UPDATE enzyme SET sys_name = 'UDP-glucose:vomilenine 21-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.219';
UPDATE enzyme SET history = 'EC 2.4.1.219 created 2002' WHERE ec = 'EC 2.4.1.219';
UPDATE enzyme SET sys_name = 'UDP-glucose:indoxyl 3-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.220';
UPDATE enzyme SET history = 'EC 2.4.1.220 created 2002' WHERE ec = 'EC 2.4.1.220';
UPDATE enzyme SET sys_name = 'GDP-beta-L-fucose:polypeptide O-alpha-L-fucosyltransferase' WHERE ec = 'EC 2.4.1.221';
UPDATE enzyme SET history = 'EC 2.4.1.221 created 2002' WHERE ec = 'EC 2.4.1.221';
UPDATE enzyme SET sys_name = 'UDP-D-GlcNAc:O-L-fucosylpeptide 3-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.222';
UPDATE enzyme SET history = 'EC 2.4.1.222 created 2002' WHERE ec = 'EC 2.4.1.222';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:beta-D-glucuronosyl-(1-&gt;3)-beta-D-galactosyl-(1-&gt;3)-beta-D-galactosyl-(1-&gt;4)-beta-D-xylosyl-proteoglycan 4(IV)-alpha-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.223';
UPDATE enzyme SET history = 'EC 2.4.1.223 created 2002' WHERE ec = 'EC 2.4.1.223';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:beta-D-glucuronosyl-(1-&gt;4)-N-acetyl-alpha-D-glucosaminyl-proteoglycan 4-alpha-N-acetylglucosaminyltransferase' WHERE ec = 'EC 2.4.1.224';
UPDATE enzyme SET history = 'EC 2.4.1.224 created 2002' WHERE ec = 'EC 2.4.1.224';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucuronate:N-acetyl-alpha-D-glucosaminyl-(1-&gt;4)-beta-D-glucuronosyl-proteoglycan 4-beta-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.225';
UPDATE enzyme SET history = 'EC 2.4.1.225 created 2002' WHERE ec = 'EC 2.4.1.225';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucuronate:N-acetyl-beta-D-galactosaminyl-(1-&gt;4)-beta-D-glucuronosyl-proteoglycan 3-beta-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.226';
UPDATE enzyme SET history = 'EC 2.4.1.226 created 2002' WHERE ec = 'EC 2.4.1.226';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:N-acetyl-alpha-D-muramyl(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol 4-beta-N-acetylglucosaminlytransferase' WHERE ec = 'EC 2.4.1.227';
UPDATE enzyme SET history = 'EC 2.4.1.227 created 2002' WHERE ec = 'EC 2.4.1.227';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:beta-D-galactosyl-(1-&gt;4)-D-glucosyl-(1&lt;-&gt;1)-ceramide 4(II)-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.228';
UPDATE enzyme SET history = 'EC 2.4.1.228 created 2002' WHERE ec = 'EC 2.4.1.228';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:[Skp1-protein]-trans-4-hydroxy-L-proline N-acetyl-alpha-D-glucosaminyl-transferase' WHERE ec = 'EC 2.4.1.229';
UPDATE enzyme SET history = 'EC 2.4.1.229 created 2003, modified 2013' WHERE ec = 'EC 2.4.1.229';
UPDATE enzyme SET sys_name = '2-alpha-D-glucosyl-D-glucose:phosphate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.230';
UPDATE enzyme SET history = 'EC 2.4.1.230 created 2003' WHERE ec = 'EC 2.4.1.230';
UPDATE enzyme SET sys_name = 'Alpha,alpha-trehalose:phosphate alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.231';
UPDATE enzyme SET history = 'EC 2.4.1.231 created 2003' WHERE ec = 'EC 2.4.1.231';
UPDATE enzyme SET sys_name = 'GDP-mannose:oligosaccharide 6-alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.232';
UPDATE enzyme SET history = 'EC 2.4.1.232 created 2004' WHERE ec = 'EC 2.4.1.232';
UPDATE enzyme SET history = 'EC 2.4.1.233 created 2004, deleted 2005' WHERE ec = 'EC 2.4.1.233';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:kaempferol 3-O-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.234';
UPDATE enzyme SET history = 'EC 2.4.1.234 created 2004' WHERE ec = 'EC 2.4.1.234';
UPDATE enzyme SET history = 'EC 2.4.1.235 created 2004, deleted 2006' WHERE ec = 'EC 2.4.1.235';
UPDATE enzyme SET sys_name = 'UDP-L-rhamnose:flavanone-7-O-glucoside 2-O-beta-L-rhamnosyltransferase' WHERE ec = 'EC 2.4.1.236';
UPDATE enzyme SET history = 'EC 2.4.1.236 created 2004' WHERE ec = 'EC 2.4.1.236';
UPDATE enzyme SET sys_name = 'UDP-glucose:flavonol 7-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.237';
UPDATE enzyme SET history = 'EC 2.4.1.237 created 2004' WHERE ec = 'EC 2.4.1.237';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:delphinidin-3,5-di-O-beta-D-glucoside 3-O-glucosyltransferase' WHERE ec = 'EC 2.4.1.238';
UPDATE enzyme SET history = 'EC 2.4.1.238 created 2004, modified 2013' WHERE ec = 'EC 2.4.1.238';
UPDATE enzyme SET sys_name = 'UDP-glucose:flavonol-3-O-beta-D-glucoside 2-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.239';
UPDATE enzyme SET history = 'EC 2.4.1.239 created 2004' WHERE ec = 'EC 2.4.1.239';
UPDATE enzyme SET sys_name = 'UDP-glucose:flavonol-3-O-beta-D-glucosyl-(1-&gt;2)-beta-D-glucoside 2-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.240';
UPDATE enzyme SET history = 'EC 2.4.1.240 created 2004' WHERE ec = 'EC 2.4.1.240';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:3-(beta-D-galactosyl)-1,2-diacyl-sn-glycerol 6-alpha-galactosyltransferase' WHERE ec = 'EC 2.4.1.241';
UPDATE enzyme SET history = 'EC 2.4.1.241 created 2005' WHERE ec = 'EC 2.4.1.241';
UPDATE enzyme SET sys_name = 'NDP-glucose:(1-&gt;4)-alpha-D-glucan 4-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.242';
UPDATE enzyme SET history = 'EC 2.4.1.242 created 2005' WHERE ec = 'EC 2.4.1.242';
UPDATE enzyme SET sys_name = '1(F)-oligo(beta-D-fructofuranosyl-(2-&gt;1)-)sucrose 6(G)-beta-D-fructotransferase' WHERE ec = 'EC 2.4.1.243';
UPDATE enzyme SET history = 'EC 2.4.1.243 created 2006' WHERE ec = 'EC 2.4.1.243';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:N-acetyl-beta-D-glucosaminyl-group 4-beta-N-acetylgalactosaminyltransferase' WHERE ec = 'EC 2.4.1.244';
UPDATE enzyme SET history = 'EC 2.4.1.244 created 2006' WHERE ec = 'EC 2.4.1.244';
UPDATE enzyme SET sys_name = 'NDP-alpha-D-glucose:D-glucose 1-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.245';
UPDATE enzyme SET history = 'EC 2.4.1.245 created 2008, modified 2013' WHERE ec = 'EC 2.4.1.245';
UPDATE enzyme SET sys_name = 'GDP-mannose:D-fructose-6-phosphate 2-alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.246';
UPDATE enzyme SET history = 'EC 2.4.1.246 created 2009' WHERE ec = 'EC 2.4.1.246';
UPDATE enzyme SET sys_name = 'Beta-D-galactosyl-(1-&gt;4)-L-rhamnose:phosphate 1-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.247';
UPDATE enzyme SET history = 'EC 2.4.1.247 created 2009' WHERE ec = 'EC 2.4.1.247';
UPDATE enzyme SET sys_name = '(1-&gt;6)-alpha-D-glucan:(1-&gt;6)-alpha-D-glucan 6-alpha-D-(1-&gt;6-alpha-D-glucano)-transferase (cyclizing)' WHERE ec = 'EC 2.4.1.248';
UPDATE enzyme SET history = 'EC 2.4.1.248 created 2009' WHERE ec = 'EC 2.4.1.248';
UPDATE enzyme SET sys_name = 'UDP-glucose:delphinidin 3-O-(6-O-malonyl)-beta-D-glucoside 3-O-glucosyltransferase' WHERE ec = 'EC 2.4.1.249';
UPDATE enzyme SET history = 'EC 2.4.1.249 created 2009' WHERE ec = 'EC 2.4.1.249';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:1D-myo-inositol 3-phosphate alpha-D-glycosyltransferase' WHERE ec = 'EC 2.4.1.250';
UPDATE enzyme SET history = 'EC 2.4.1.250 created 2010' WHERE ec = 'EC 2.4.1.250';
UPDATE enzyme SET sys_name = 'GDP-mannose:GlcA-beta-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-D-Glc-beta-(1-&gt;4)-D-Glc-alpha-1-diphospho-ditrans,octacis-undecaprenol 4-beta-mannosyltransferase' WHERE ec = 'EC 2.4.1.251';
UPDATE enzyme SET history = 'EC 2.4.1.251 created 2010' WHERE ec = 'EC 2.4.1.251';
UPDATE enzyme SET sys_name = 'GDP-mannose:D-Glc-beta-(1-&gt;4)-Glc-alpha-1-diphosphoundecaprenol 3-alpha-mannosyltransferase' WHERE ec = 'EC 2.4.1.252';
UPDATE enzyme SET history = 'EC 2.4.1.252 created 2010' WHERE ec = 'EC 2.4.1.252';
UPDATE enzyme SET sys_name = 'UDP-D-glucuronate:5,6,7-trihydroxyflavone 7-O-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.253';
UPDATE enzyme SET history = 'EC 2.4.1.253 created 2010' WHERE ec = 'EC 2.4.1.253';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucuronate:cyanidin-3-O-beta-D-glucoside 2-O-beta-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.254';
UPDATE enzyme SET history = 'EC 2.4.1.254 created 2011' WHERE ec = 'EC 2.4.1.254';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:protein-O-beta-N-acetyl-D-glucosaminyl transferase' WHERE ec = 'EC 2.4.1.255';
UPDATE enzyme SET history = 'EC 2.4.1.255 created 2011' WHERE ec = 'EC 2.4.1.255';
UPDATE enzyme SET sys_name = 'Dolichyl beta-D-glucosyl phosphate:D-Glc-alpha-(1-&gt;3)-D-Glc-alpha-(1-&gt;3)-D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-[D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-(D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;6))-D-Man-alpha-(1-&gt;6)]-D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol' WHERE ec = 'EC 2.4.1.256';
UPDATE enzyme SET history = 'EC 2.4.1.256 created 2011, modified 2012' WHERE ec = 'EC 2.4.1.256';
UPDATE enzyme SET sys_name = 'GDP-D-mannose:D-Man-alpha-(1-&gt;3)-D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol alpha-6-mannosyltransferase' WHERE ec = 'EC 2.4.1.257';
UPDATE enzyme SET history = 'EC 2.4.1.257 created 2011, modified 2012' WHERE ec = 'EC 2.4.1.257';
UPDATE enzyme SET sys_name = 'Dolichyl beta-D-mannosyl phosphate:D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-(D-Man-alpha-(1-&gt;6))-D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol alpha-1,3-mannosyltransferase' WHERE ec = 'EC 2.4.1.258';
UPDATE enzyme SET history = 'EC 2.4.1.258 created 1976 as EC 2.4.1.130, part-transferred 2011 to EC 2.4.1.258, modified 2012' WHERE ec = 'EC 2.4.1.258';
UPDATE enzyme SET sys_name = 'Dolichyl beta-D-mannosyl phosphate: D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-(D-Man-alpha-(1-&gt;3)-D-Man-alpha-(1-&gt;6))-D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol alpha-1,2-mannosyltransferase' WHERE ec = 'EC 2.4.1.259';
UPDATE enzyme SET history = 'EC 2.4.1.259 created 1976 as EC 2.4.1.130, part-transferred 2011 to EC 2.4.1.259, modified 2012' WHERE ec = 'EC 2.4.1.259';
UPDATE enzyme SET sys_name = 'Dolichyl beta-D-mannosyl phosphate:D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-(D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-D-Man-alpha-(1-&gt;6))-D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol alpha-1,6-mannosyltransferase' WHERE ec = 'EC 2.4.1.260';
UPDATE enzyme SET history = 'EC 2.4.1.260 created 1976 as EC 2.4.1.130, part-transferred 2011 to EC 2.4.1.160, modified 2012' WHERE ec = 'EC 2.4.1.260';
UPDATE enzyme SET sys_name = 'Dolichyl beta-D-mannosyl phosphate:D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-(D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-(D-Man-alpha-(1-&gt;6))-D-Man-alpha-(1-&gt;6))-D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol 2-alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.261';
UPDATE enzyme SET history = 'EC 2.4.1.261 created 1976 as EC 2.4.1.130, part transferred 2011 to EC 2.4.1.261, modified 2012' WHERE ec = 'EC 2.4.1.261';
UPDATE enzyme SET sys_name = 'UDP-D-glucuronate:soyasapogenol 3-O-D-glucuronosyltransferase' WHERE ec = 'EC 2.4.1.262';
UPDATE enzyme SET history = 'EC 2.4.1.262 created 2011' WHERE ec = 'EC 2.4.1.262';
UPDATE enzyme SET sys_name = 'UDP-D-glucose:abscisate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.263';
UPDATE enzyme SET history = 'EC 2.4.1.263 created 2011' WHERE ec = 'EC 2.4.1.263';
UPDATE enzyme SET sys_name = 'UDP-glucuronate:D-Man-alpha-(1-&gt;3)-D-Glc-beta-(1-&gt;4)-D-Glc-alpha-1-diphospho-ditrans,octacis-undecaprenol beta-1,2-glucuronyltransferase' WHERE ec = 'EC 2.4.1.264';
UPDATE enzyme SET history = 'EC 2.4.1.264 created 2011' WHERE ec = 'EC 2.4.1.264';
UPDATE enzyme SET sys_name = 'Dolichyl beta-D-glucosyl phosphate:D-Glc-alpha-(1-&gt;3)-D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-[D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-(D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;6))-D-Man-alpha-(1-&gt;6)]-D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol alpha-1,3-glucosyltransferase' WHERE ec = 'EC 2.4.1.265';
UPDATE enzyme SET history = 'EC 2.4.1.265 created 2011, modified 2012' WHERE ec = 'EC 2.4.1.265';
UPDATE enzyme SET sys_name = 'NDP-glucose:3-phospho-D-glycerate 2-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.266';
UPDATE enzyme SET history = 'EC 2.4.1.266 created 2011' WHERE ec = 'EC 2.4.1.266';
UPDATE enzyme SET sys_name = 'Dolichyl alpha-D-glucosyl phosphate:D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-[D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-(D-Man-alpha-(1-&gt;2)-D-Man-alpha-(1-&gt;6))-D-Man-alpha-(1-&gt;6)]-D-Man-beta-(1-&gt;4)-D-GlcNAc-beta-(1-&gt;4)-D-GlcNAc-diphosphodolichol beta-1,3-glucosyltransferase' WHERE ec = 'EC 2.4.1.267';
UPDATE enzyme SET history = 'EC 2.4.1.267 created 2011, modified 2012' WHERE ec = 'EC 2.4.1.267';
UPDATE enzyme SET sys_name = 'ADP-glucose:D-glycerate 2-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.268';
UPDATE enzyme SET history = 'EC 2.4.1.268 created 2011' WHERE ec = 'EC 2.4.1.268';
UPDATE enzyme SET sys_name = 'GDP-mannose:D-glycerate 2-alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.269';
UPDATE enzyme SET history = 'EC 2.4.1.269 created 2011' WHERE ec = 'EC 2.4.1.269';
UPDATE enzyme SET sys_name = 'GDP-mannose:2-O-(alpha-D-glucosyl)-3-phospho-D-glycerate 2-O-alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.270';
UPDATE enzyme SET history = 'EC 2.4.1.270 created 2011' WHERE ec = 'EC 2.4.1.270';
UPDATE enzyme SET sys_name = 'UDP-glucose:crocetin 8-O-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.271';
UPDATE enzyme SET history = 'EC 2.4.1.271 created 2011' WHERE ec = 'EC 2.4.1.271';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:soyasapogenol B 3-O-glucuronide beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.272';
UPDATE enzyme SET history = 'EC 2.4.1.272 created 2011' WHERE ec = 'EC 2.4.1.272';
UPDATE enzyme SET sys_name = 'UDP-rhamnose:soyasaponin III rhamnosyltransferase' WHERE ec = 'EC 2.4.1.273';
UPDATE enzyme SET history = 'EC 2.4.1.273 created 2011' WHERE ec = 'EC 2.4.1.273';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:beta-D-glucosyl-(1&lt;-&gt;1)-ceramide 4-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.274';
UPDATE enzyme SET history = 'EC 2.4.1.274 created 2011' WHERE ec = 'EC 2.4.1.274';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:N-acetyl-beta-D-glucosaminyl-(1-&gt;3)-beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide 4-beta-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.275';
UPDATE enzyme SET history = 'EC 2.4.1.275 created 2011, modified 2013' WHERE ec = 'EC 2.4.1.275';
UPDATE enzyme SET sys_name = 'UDP-glucose:zeaxanthin beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.276';
UPDATE enzyme SET history = 'EC 2.4.1.276 created 2011' WHERE ec = 'EC 2.4.1.276';
UPDATE enzyme SET sys_name = 'dTDP-3-dimethylamino-3,4,6-trideoxy-alpha-D-glucopyranose:10-deoxymethynolide 3-dimethylamino-4,6-dideoxy-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.277';
UPDATE enzyme SET history = 'EC 2.4.1.277 created 2011, modified 2014' WHERE ec = 'EC 2.4.1.277';
UPDATE enzyme SET sys_name = 'dTDP-3-dimethylamino-3,4,6-trideoxy-alpha-D-glucopyranose:3-alpha-mycarosylerythronolide B 3-dimethylamino-3,4,6-trideoxy-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.278';
UPDATE enzyme SET history = 'EC 2.4.1.278 created 2012, modified 2014' WHERE ec = 'EC 2.4.1.278';
UPDATE enzyme SET sys_name = '3-O-alpha-D-glucopyranosyl-D-glucopyranose:phosphate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.279';
UPDATE enzyme SET history = 'EC 2.4.1.279 created 2012' WHERE ec = 'EC 2.4.1.279';
UPDATE enzyme SET sys_name = 'N,N-diacetylchitobiose:phosphate N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.280';
UPDATE enzyme SET history = 'EC 2.4.1.280 created 2012' WHERE ec = 'EC 2.4.1.280';
UPDATE enzyme SET sys_name = '4-O-beta-D-mannopyranosyl-D-glucopyranose:phosphate alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.281';
UPDATE enzyme SET history = 'EC 2.4.1.281 created 2012' WHERE ec = 'EC 2.4.1.281';
UPDATE enzyme SET sys_name = '3-O-alpha-D-glucopyranosyl-L-rhamnopyranose:phosphate beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.282';
UPDATE enzyme SET history = 'EC 2.4.1.282 created 2012' WHERE ec = 'EC 2.4.1.282';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:2-deoxystreptamine N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.283';
UPDATE enzyme SET history = 'EC 2.4.1.283 created 2012' WHERE ec = 'EC 2.4.1.283';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:2-deoxystreptamine 6-alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.284';
UPDATE enzyme SET history = 'EC 2.4.1.284 created 2012' WHERE ec = 'EC 2.4.1.284';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:ribostamycin N-acetylglucosaminyltransferase' WHERE ec = 'EC 2.4.1.285';
UPDATE enzyme SET history = 'EC 2.4.1.285 created 2012' WHERE ec = 'EC 2.4.1.285';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:2,4,4,6-tetrahydroxychalcone 4-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.286';
UPDATE enzyme SET history = 'EC 2.4.1.286 created 2012' WHERE ec = 'EC 2.4.1.286';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactofuranose:alpha-L-rhamnopyranosyl-(1-&gt;3)-N-acetyl-alpha-D-glucosaminyl-diphospho-trans,octacis-decaprenol 3-beta/4-beta-galactofuranosyltransferase' WHERE ec = 'EC 2.4.1.287';
UPDATE enzyme SET history = 'EC 2.4.1.287 created 2012' WHERE ec = 'EC 2.4.1.287';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactofuranose:beta-D-galactofuranosyl-(1-&gt;5)-beta-D-galactofuranosyl-(1-&gt;4)-alpha-L-rhamnopyranosyl-(1-&gt;3)-N-acetyl-alpha-D-glucosaminyl-diphospho-trans,octacis-decaprenol 4-beta/5-beta-galactofuranosyltransferase' WHERE ec = 'EC 2.4.1.288';
UPDATE enzyme SET history = 'EC 2.4.1.288 created 2012' WHERE ec = 'EC 2.4.1.288';
UPDATE enzyme SET sys_name = 'dTDP-6-deoxy-beta-L-mannose:N-acetyl-alpha-D-glucosaminyl-diphospho-trans,octacis-decaprenol 3-alpha-L-rhamnosyltransferase' WHERE ec = 'EC 2.4.1.289';
UPDATE enzyme SET history = 'EC 2.4.1.289 created 2012' WHERE ec = 'EC 2.4.1.289';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:N,N-diacetyl-alpha-D-bacillosaminyl-diphospho-tritrans,heptacis-undecaprenol 3-alpha-N-acetyl-D-galactosaminyltransferase' WHERE ec = 'EC 2.4.1.290';
UPDATE enzyme SET history = 'EC 2.4.1.290 created 2012' WHERE ec = 'EC 2.4.1.290';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:N-acetylgalactosaminyl-alpha-(1-&gt;3)-N,N-diacetyl-alpha-D-bacillosaminyl-diphospho-tritrans,heptacis-undecaprenol 3-alpha-N-acetyl-D-galactosaminyltransferase' WHERE ec = 'EC 2.4.1.291';
UPDATE enzyme SET history = 'EC 2.4.1.291 created 2012' WHERE ec = 'EC 2.4.1.291';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:GalNAc-alpha-(1-&gt;4)-GalNAc-alpha-(1-&gt;3)-diNAcBac-PP-tritrans,heptacis-undecaprenol 4-alpha-N-acetyl-D-galactosaminyltransferase' WHERE ec = 'EC 2.4.1.292';
UPDATE enzyme SET history = 'EC 2.4.1.292 created 2012' WHERE ec = 'EC 2.4.1.292';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:(GalNAc-alpha-(1-&gt;4))4-GalNAc-alpha-(1-&gt;3)-diNAcBac-diphospho-tritrans,heptacis-undecaprenol 3-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.293';
UPDATE enzyme SET history = 'EC 2.4.1.293 created 2012' WHERE ec = 'EC 2.4.1.293';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:cyanidin 3-O-galactosyltransferase' WHERE ec = 'EC 2.4.1.294';
UPDATE enzyme SET history = 'EC 2.4.1.294 created 2013' WHERE ec = 'EC 2.4.1.294';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:anthocyanidin-3-O-beta-D-sambubioside 5-O-glucosyltransferase' WHERE ec = 'EC 2.4.1.295';
UPDATE enzyme SET history = 'EC 2.4.1.295 created 2013' WHERE ec = 'EC 2.4.1.295';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:anthocyanidin-3-O-(3-O-(4-coumaroyl)-alpha-L-rhamnosyl-(1-&gt;6)-beta-D-glucoside) 5-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.296';
UPDATE enzyme SET history = 'EC 2.4.1.296 created 2013' WHERE ec = 'EC 2.4.1.296';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:anthocyanidin-3-O-glucoside 2-O-glucosyltransferase' WHERE ec = 'EC 2.4.1.297';
UPDATE enzyme SET history = 'EC 2.4.1.297 created 2013' WHERE ec = 'EC 2.4.1.297';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:anthocyanidin-3-O-beta-D-glucoside 5-O-glucosyltransferase' WHERE ec = 'EC 2.4.1.298';
UPDATE enzyme SET history = 'EC 2.4.1.298 created 2013' WHERE ec = 'EC 2.4.1.298';
UPDATE enzyme SET sys_name = '1-O-sinapoyl-beta-D-glucose:cyanidin-3-O-beta-D-glucoside 5-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.299';
UPDATE enzyme SET history = 'EC 2.4.1.299 created 2013' WHERE ec = 'EC 2.4.1.299';
UPDATE enzyme SET sys_name = '1-O-vanilloyl-beta-D-glucose:cyanidin-3-O-beta-D-glucoside 7-O-beta-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.300';
UPDATE enzyme SET history = 'EC 2.4.1.300 created 2013' WHERE ec = 'EC 2.4.1.300';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-kanosamine:2-deamino-2-hydroxyneamine 1-alpha-D-kanosaminyltransferase' WHERE ec = 'EC 2.4.1.301';
UPDATE enzyme SET history = 'EC 2.4.1.301 created 2013' WHERE ec = 'EC 2.4.1.301';
UPDATE enzyme SET sys_name = 'dTDP-beta-L-noviose:novobiocic acid 7-O-noviosyltransferase' WHERE ec = 'EC 2.4.1.302';
UPDATE enzyme SET history = 'EC 2.4.1.302 created 2013' WHERE ec = 'EC 2.4.1.302';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol 3-beta-galactosyltransferase' WHERE ec = 'EC 2.4.1.303';
UPDATE enzyme SET history = 'EC 2.4.1.303 created 2013' WHERE ec = 'EC 2.4.1.303';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol beta-1,4-galactosyltransferase' WHERE ec = 'EC 2.4.1.304';
UPDATE enzyme SET history = 'EC 2.4.1.304 created 2013' WHERE ec = 'EC 2.4.1.304';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol beta-1,3-glucosyltransferase' WHERE ec = 'EC 2.4.1.305';
UPDATE enzyme SET history = 'EC 2.4.1.305 created 2013' WHERE ec = 'EC 2.4.1.305';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:N-acetyl-alpha-D-galactosaminyl-diphospho-ditrans,octacis-undecaprenol alpha-1,3-N-acetyl-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.306';
UPDATE enzyme SET history = 'EC 2.4.1.306 created 2013' WHERE ec = 'EC 2.4.1.306';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:alpha-D-GalNAc-(1-&gt;3)-alpha-D-GalNAc-diphospho-ditrans,octacis-undecaprenol beta-1,3-galactosyltransferase' WHERE ec = 'EC 2.4.1.307';
UPDATE enzyme SET history = 'EC 2.4.1.307 created 2013' WHERE ec = 'EC 2.4.1.307';
UPDATE enzyme SET sys_name = 'GDP-beta-L-fucose:beta-D-Gal-(1-&gt;3)-alpha-D-GalNAc-(1-&gt;3)-alpha-D-GalNAc-diphospho-ditrans,octacis-undecaprenol alpha-1,2-fucosyltransferase' WHERE ec = 'EC 2.4.1.308';
UPDATE enzyme SET history = 'EC 2.4.1.308 created 2013' WHERE ec = 'EC 2.4.1.308';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:alpha-L-Fuc-(1-&gt;2)-beta-D-Gal-(1-&gt;3)-alpha-D-GalNAc-(1-&gt;3)-alpha-D-GalNAc-diphospho-ditrans,octacis-undecaprenol alpha-1,3-galactosyltransferase' WHERE ec = 'EC 2.4.1.309';
UPDATE enzyme SET history = 'EC 2.4.1.309 created 2013' WHERE ec = 'EC 2.4.1.309';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:vancomycin aglycone 48-O-beta-glucosyltransferase' WHERE ec = 'EC 2.4.1.310';
UPDATE enzyme SET history = 'EC 2.4.1.310 created 2013' WHERE ec = 'EC 2.4.1.310';
UPDATE enzyme SET sys_name = 'dTDP-L-4-epi-vancosamine:devancoaminyl-vancomycin vancosaminyltransferase' WHERE ec = 'EC 2.4.1.311';
UPDATE enzyme SET history = 'EC 2.4.1.311 created 2013' WHERE ec = 'EC 2.4.1.311';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:alpha-D-mannosyl-threonyl-[protein] 4-beta-N-acetyl-D-glucosaminyltransferase' WHERE ec = 'EC 2.4.1.312';
UPDATE enzyme SET history = 'EC 2.4.1.312 created 2013' WHERE ec = 'EC 2.4.1.312';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:N-acetyl-beta-D-glucosaminyl-(1-&gt;4)-alpha-D-mannosyl-threonyl-[protein] 3-beta-N-acetyl-D-galactosaminyltransferase' WHERE ec = 'EC 2.4.1.313';
UPDATE enzyme SET history = 'EC 2.4.1.313 created 2013' WHERE ec = 'EC 2.4.1.313';
UPDATE enzyme SET sys_name = 'UDP-glucose:ginsenoside-Rd beta-1,6-glucosyltransferase' WHERE ec = 'EC 2.4.1.314';
UPDATE enzyme SET history = 'EC 2.4.1.314 created 2013' WHERE ec = 'EC 2.4.1.314';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:1,2-diacyl-3-O-(alpha-D-glucopyranosyl)-sn-glycerol 6-glucosyltransferase' WHERE ec = 'EC 2.4.1.315';
UPDATE enzyme SET history = 'EC 2.4.1.315 created 2014' WHERE ec = 'EC 2.4.1.315';
UPDATE enzyme SET sys_name = 'dTDP-alpha-D-mycaminose:tylactone 5-O-beta-D-mycaminosyltransferase' WHERE ec = 'EC 2.4.1.316';
UPDATE enzyme SET history = 'EC 2.4.1.316 created 2014' WHERE ec = 'EC 2.4.1.316';
UPDATE enzyme SET sys_name = 'dTDP-6-deoxy-alpha-D-allose:5-O-beta-D-mycaminosyltylonolide 23-O-6-deoxy-alpha-D-allosyltransferase' WHERE ec = 'EC 2.4.1.317';
UPDATE enzyme SET history = 'EC 2.4.1.317 created 2014' WHERE ec = 'EC 2.4.1.317';
UPDATE enzyme SET sys_name = 'dTDP-beta-L-mycarose:demethyllactenocin 4-O-alpha-L-mycarosyltransferase' WHERE ec = 'EC 2.4.1.318';
UPDATE enzyme SET history = 'EC 2.4.1.318 created 2014' WHERE ec = 'EC 2.4.1.318';
UPDATE enzyme SET sys_name = '1,4-beta-D-mannooligosaccharide:phosphate alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.319';
UPDATE enzyme SET history = 'EC 2.4.1.319 created 2014' WHERE ec = 'EC 2.4.1.319';
UPDATE enzyme SET sys_name = '4-O-beta-D-mannopyranosyl-N-acetyl-D-glucosamine:phosphate alpha-D-mannosyltransferase' WHERE ec = 'EC 2.4.1.320';
UPDATE enzyme SET history = 'EC 2.4.1.320 created 2014' WHERE ec = 'EC 2.4.1.320';
UPDATE enzyme SET sys_name = '4-O-beta-D-glucopyranosyl-D-gluconate:phosphate alpha-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.321';
UPDATE enzyme SET history = 'EC 2.4.1.321 created 2014' WHERE ec = 'EC 2.4.1.321';
UPDATE enzyme SET sys_name = 'dTDP-beta-L-vancomycin:desvancosaminyl-vancomycin beta-L-vancosaminetransferase' WHERE ec = 'EC 2.4.1.322';
UPDATE enzyme SET history = 'EC 2.4.1.322 created 2014' WHERE ec = 'EC 2.4.1.322';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:7-deoxyloganetate O-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.323';
UPDATE enzyme SET history = 'EC 2.4.1.323 created 2014' WHERE ec = 'EC 2.4.1.323';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:7-deoxyloganetin O-D-glucosyltransferase' WHERE ec = 'EC 2.4.1.324';
UPDATE enzyme SET history = 'EC 2.4.1.324 created 2014' WHERE ec = 'EC 2.4.1.324';
UPDATE enzyme SET sys_name = 'UDP-galactose:sucrose 3(G)-alpha-D-galactosyltransferase' WHERE ec = 'EC 2.4.1.n2';
UPDATE enzyme SET history = 'EC 2.4.1.n2 created 2010' WHERE ec = 'EC 2.4.1.n2';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.4.1.n3';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.4.1.n5';
UPDATE enzyme SET sys_name = 'Purine-nucleoside:phosphate ribosyltransferase' WHERE ec = 'EC 2.4.2.1';
UPDATE enzyme SET history = 'EC 2.4.2.1 created 1961' WHERE ec = 'EC 2.4.2.1';
UPDATE enzyme SET sys_name = 'Pyrimidine-nucleoside:phosphate (2-deoxy)-alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.2';
UPDATE enzyme SET history = 'EC 2.4.2.2 created 1961, modified 2013' WHERE ec = 'EC 2.4.2.2';
UPDATE enzyme SET sys_name = 'Uridine:phosphate alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.3';
UPDATE enzyme SET history = 'EC 2.4.2.3 created 1961' WHERE ec = 'EC 2.4.2.3';
UPDATE enzyme SET sys_name = 'Thymidine:phosphate deoxy-alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.4';
UPDATE enzyme SET history = 'EC 2.4.2.4 created 1961' WHERE ec = 'EC 2.4.2.4';
UPDATE enzyme SET sys_name = 'Nucleoside:purine(pyrimidine) D-ribosyltransferase' WHERE ec = 'EC 2.4.2.5';
UPDATE enzyme SET history = 'EC 2.4.2.5 created 1961' WHERE ec = 'EC 2.4.2.5';
UPDATE enzyme SET sys_name = 'Nucleoside:purine(pyrimidine) deoxy-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.6';
UPDATE enzyme SET history = 'EC 2.4.2.6 created 1961' WHERE ec = 'EC 2.4.2.6';
UPDATE enzyme SET sys_name = 'AMP:diphosphate phospho-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.7';
UPDATE enzyme SET history = 'EC 2.4.2.7 created 1961' WHERE ec = 'EC 2.4.2.7';
UPDATE enzyme SET sys_name = 'IMP:diphosphate phospho-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.8';
UPDATE enzyme SET history = 'EC 2.4.2.8 created 1961, modified 1982' WHERE ec = 'EC 2.4.2.8';
UPDATE enzyme SET sys_name = 'UMP:diphosphate phospho-alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.9';
UPDATE enzyme SET history = 'EC 2.4.2.9 created 1961' WHERE ec = 'EC 2.4.2.9';
UPDATE enzyme SET sys_name = 'Orotidine-5-phosphate:diphosphate phospho-alpha-D-ribosyl-transferase' WHERE ec = 'EC 2.4.2.10';
UPDATE enzyme SET history = 'EC 2.4.2.10 created 1961, modified 1986' WHERE ec = 'EC 2.4.2.10';
UPDATE enzyme SET history = 'EC 2.4.2.11 created 1961, deleted 2013' WHERE ec = 'EC 2.4.2.11';
UPDATE enzyme SET sys_name = 'Nicotinamide-D-nucleotide:diphosphate phospho-alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.12';
UPDATE enzyme SET history = 'EC 2.4.2.12 created 1961' WHERE ec = 'EC 2.4.2.12';
UPDATE enzyme SET history = 'EC 2.4.2.13 created 1961, deleted 1965' WHERE ec = 'EC 2.4.2.13';
UPDATE enzyme SET sys_name = '5-phospho-beta-D-ribosylamine:diphosphate phospho-alpha-D-ribosyltransferase (glutamate-amidating)' WHERE ec = 'EC 2.4.2.14';
UPDATE enzyme SET history = 'EC 2.4.2.14 created 1961' WHERE ec = 'EC 2.4.2.14';
UPDATE enzyme SET sys_name = 'Guanosine:phosphate alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.15';
UPDATE enzyme SET history = 'EC 2.4.2.15 created 1965' WHERE ec = 'EC 2.4.2.15';
UPDATE enzyme SET sys_name = 'Urate-D-ribonucleotide:phosphate alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.16';
UPDATE enzyme SET history = 'EC 2.4.2.16 created 1965' WHERE ec = 'EC 2.4.2.16';
UPDATE enzyme SET sys_name = '1-(5-phospho-beta-D-ribosyl)-ATP:diphosphate phospho-alpha-D-ribosyl-transferase' WHERE ec = 'EC 2.4.2.17';
UPDATE enzyme SET history = 'EC 2.4.2.17 created 1972' WHERE ec = 'EC 2.4.2.17';
UPDATE enzyme SET sys_name = 'N-(5-phospho-D-ribosyl)-anthranilate:diphosphate phospho-alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.18';
UPDATE enzyme SET history = 'EC 2.4.2.18 created 1972' WHERE ec = 'EC 2.4.2.18';
UPDATE enzyme SET sys_name = 'Beta-nicotinate-D-ribonucleotide:diphosphate phospho-alpha-D-ribosyltransferase (carboxylating)' WHERE ec = 'EC 2.4.2.19';
UPDATE enzyme SET history = 'EC 2.4.2.19 created 1972' WHERE ec = 'EC 2.4.2.19';
UPDATE enzyme SET sys_name = '2,4-dioxotetrahydropyrimidine-D-ribonucleotide:diphosphate phospho-alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.20';
UPDATE enzyme SET history = 'EC 2.4.2.20 created 1972' WHERE ec = 'EC 2.4.2.20';
UPDATE enzyme SET sys_name = 'Nicotinate-nucleotide:5,6-dimethylbenzimidazole phospho-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.21';
UPDATE enzyme SET history = 'EC 2.4.2.21 created 1972' WHERE ec = 'EC 2.4.2.21';
UPDATE enzyme SET sys_name = 'XMP:diphosphate 5-phospho-alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.22';
UPDATE enzyme SET history = 'EC 2.4.2.22 created 1972' WHERE ec = 'EC 2.4.2.22';
UPDATE enzyme SET history = 'EC 2.4.2.23 created 1972' WHERE ec = 'EC 2.4.2.23';
UPDATE enzyme SET sys_name = 'UDP-D-xylose:(1-&gt;4)-beta-D-xylan 4-beta-D-xylosyltransferase' WHERE ec = 'EC 2.4.2.24';
UPDATE enzyme SET history = 'EC 2.4.2.24 created 1972 as EC  2.4.1.72, transferred 1976 to EC 2.4.2.24' WHERE ec = 'EC 2.4.2.24';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-apiose:5,4-dihydroxyflavone 7-O-beta-D-glucoside 2-O-beta-D-apiofuranosyltransferase' WHERE ec = 'EC 2.4.2.25';
UPDATE enzyme SET history = 'EC 2.4.2.25 created 1976' WHERE ec = 'EC 2.4.2.25';
UPDATE enzyme SET sys_name = 'UDP-D-xylose:protein beta-D-xylosyltransferase' WHERE ec = 'EC 2.4.2.26';
UPDATE enzyme SET history = 'EC 2.4.2.26 created 1976, modified 2002' WHERE ec = 'EC 2.4.2.26';
UPDATE enzyme SET sys_name = 'dTDP-L-dihydrostreptose:streptidine-6-phosphate dihydrostreptosyltransferase' WHERE ec = 'EC 2.4.2.27';
UPDATE enzyme SET history = 'EC 2.4.2.27 created 1982' WHERE ec = 'EC 2.4.2.27';
UPDATE enzyme SET sys_name = 'S-methyl-5-thioadenosine:phosphate S-methyl-5-thio-alpha-D-ribosyl-transferase' WHERE ec = 'EC 2.4.2.28';
UPDATE enzyme SET history = 'EC 2.4.2.28 created 1983' WHERE ec = 'EC 2.4.2.28';
UPDATE enzyme SET sys_name = 'tRNA-guanine(34):queuine tRNA-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.29';
UPDATE enzyme SET history = 'EC 2.4.2.29 created 1984, modified 2007, modified 2012' WHERE ec = 'EC 2.4.2.29';
UPDATE enzyme SET sys_name = 'NAD(+):poly(ADP-D-ribosyl)-acceptor ADP-D-ribosyl-transferase' WHERE ec = 'EC 2.4.2.30';
UPDATE enzyme SET history = 'EC 2.4.2.30 created 1984, modified 1990' WHERE ec = 'EC 2.4.2.30';
UPDATE enzyme SET sys_name = 'NAD(+):protein-L-arginine ADP-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.31';
UPDATE enzyme SET history = 'EC 2.4.2.31 created 1984, modified 1990, modified 2006' WHERE ec = 'EC 2.4.2.31';
UPDATE enzyme SET sys_name = 'UDP-D-xylose:dolichyl-phosphate D-xylosyltransferase' WHERE ec = 'EC 2.4.2.32';
UPDATE enzyme SET history = 'EC 2.4.2.32 created 1984, modified 2003' WHERE ec = 'EC 2.4.2.32';
UPDATE enzyme SET sys_name = 'Dolichyl-D-xylosyl-phosphate:protein D-xylosyltransferase' WHERE ec = 'EC 2.4.2.33';
UPDATE enzyme SET history = 'EC 2.4.2.33 created 1984' WHERE ec = 'EC 2.4.2.33';
UPDATE enzyme SET sys_name = 'UDP-L-arabinose:(indol-3-yl)acetyl-1D-myo-inositol L-arabinosyltransferase' WHERE ec = 'EC 2.4.2.34';
UPDATE enzyme SET history = 'EC 2.4.2.34 created 1986, modified 2003' WHERE ec = 'EC 2.4.2.34';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-xylose:flavonol-3-O-glycoside 2-O-beta-D-xylosyltransferase' WHERE ec = 'EC 2.4.2.35';
UPDATE enzyme SET history = 'EC 2.4.2.35 created 1986, modified 2014' WHERE ec = 'EC 2.4.2.35';
UPDATE enzyme SET sys_name = 'NAD(+):diphthamide-[translation elongation factor 2] N-(ADP-D-ribosyl)transferase' WHERE ec = 'EC 2.4.2.36';
UPDATE enzyme SET history = 'EC 2.4.2.36 created 1990, modified 2013' WHERE ec = 'EC 2.4.2.36';
UPDATE enzyme SET sys_name = 'NAD(+):(dinitrogen reductase) (ADP-D-ribosyl)transferase' WHERE ec = 'EC 2.4.2.37';
UPDATE enzyme SET history = 'EC 2.4.2.37 created 1992' WHERE ec = 'EC 2.4.2.37';
UPDATE enzyme SET sys_name = 'UDP-D-xylose:glycoprotein (D-xylose to the 3,6-disubstituted mannose of N(4)-(N-acetyl-beta-D-glucosaminyl-(1-&gt;2)-alpha-D-mannosyl-(1-&gt;3)-(N-acetyl-beta-D-glucosaminyl-(1-&gt;2)-alpha-D-mannosyl-(1-&gt;6))-beta-D-mannosyl-(1-&gt;4)-N-acetyl-beta-D-glucosaminyl-(1-&gt;4)-N-acetyl-beta-D-glucosaminyl)asparagine) 2-beta-D-xylosyltransferase' WHERE ec = 'EC 2.4.2.38';
UPDATE enzyme SET history = 'EC 2.4.2.38 created 2001' WHERE ec = 'EC 2.4.2.38';
UPDATE enzyme SET sys_name = 'UDP-D-xylose:xyloglucan 6-alpha-D-xylosyltransferase' WHERE ec = 'EC 2.4.2.39';
UPDATE enzyme SET history = 'EC 2.4.2.39 created 1989 as EC 2.4.1.169, transferred 2003 to EC 2.4.2.39' WHERE ec = 'EC 2.4.2.39';
UPDATE enzyme SET sys_name = 'UDP-D-xylose:zeatin O-beta-D-xylosyltransferase' WHERE ec = 'EC 2.4.2.40';
UPDATE enzyme SET history = 'EC 2.4.2.40 created 1992 as EC 2.4.1.204, transferred 2003 to EC 2.4.2.40' WHERE ec = 'EC 2.4.2.40';
UPDATE enzyme SET sys_name = 'UDP-D-xylose:xylogalacturonan 3-beta-D-xylosyltransferase' WHERE ec = 'EC 2.4.2.41';
UPDATE enzyme SET history = 'EC 2.4.2.41 created 2009' WHERE ec = 'EC 2.4.2.41';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-xylose:beta-D-glucoside 3-alpha-D-D-xylosyltransferase' WHERE ec = 'EC 2.4.2.42';
UPDATE enzyme SET history = 'EC 2.4.2.42 created 2010' WHERE ec = 'EC 2.4.2.42';
UPDATE enzyme SET sys_name = '4-amino-4-deoxy-alpha-L-arabinopyranosyl ditrans,octacis-undecaprenyl phosphate:lipid IV(A)  di-trans,octa-cis-undecaprenyl phosphate:lipid IV(A) L-2-aminoarabinopyranosyltransferasetransferase' WHERE ec = 'EC 2.4.2.43';
UPDATE enzyme SET history = 'EC 2.4.1.43 created in 2010, modified 2011' WHERE ec = 'EC 2.4.2.43';
UPDATE enzyme SET sys_name = 'S-methyl-5-thioinosine:phosphate S-methyl-5-thio-alpha-D-ribosyl-transferase' WHERE ec = 'EC 2.4.2.44';
UPDATE enzyme SET history = 'EC 2.4.2.44 created 2011' WHERE ec = 'EC 2.4.2.44';
UPDATE enzyme SET sys_name = 'Trans,octacis-decaprenylphospho-beta-D-ribofuranose 5-phosphate:diphosphate phospho-alpha-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.45';
UPDATE enzyme SET history = 'EC 2.4.2.45 created 2012' WHERE ec = 'EC 2.4.2.45';
UPDATE enzyme SET sys_name = 'Galactofuranan:trans,octacis-decaprenylphospho-beta-D-arabinofuranose 5-O-alpha-D-arabinofuranosyltransferase' WHERE ec = 'EC 2.4.2.46';
UPDATE enzyme SET history = 'EC 2.4.2.46 created 2012' WHERE ec = 'EC 2.4.2.46';
UPDATE enzyme SET sys_name = 'Alpha-(1-&gt;5)-arabinofuranan:trans,octacis-decaprenylphospho-beta-D-arabinofuranose 3-O-alpha-D-arabinofuranosyltransferase' WHERE ec = 'EC 2.4.2.47';
UPDATE enzyme SET history = 'EC 2.4.2.47 created 2012' WHERE ec = 'EC 2.4.2.47';
UPDATE enzyme SET sys_name = 'tRNA-guanine(15):7-cyano-7-carbaguanine tRNA-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.48';
UPDATE enzyme SET history = 'EC 2.4.2.48 created 2012' WHERE ec = 'EC 2.4.2.48';
UPDATE enzyme SET sys_name = 'Neamine:5-phospho-alpha-D-ribose 1-diphosphate phosphoribosyltransferase' WHERE ec = 'EC 2.4.2.49';
UPDATE enzyme SET history = 'EC 2.4.2.49 created 2013' WHERE ec = 'EC 2.4.2.49';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-xylose:cyanidin-3-O-beta-D-galactoside 2-O-xylosyltransferase' WHERE ec = 'EC 2.4.2.50';
UPDATE enzyme SET history = 'EC 2.4.2.50 created 2013' WHERE ec = 'EC 2.4.2.50';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-xylose:anthocyanidin-3-O-beta-D-glucoside 2-O-xylosyltransferase' WHERE ec = 'EC 2.4.2.51';
UPDATE enzyme SET history = 'EC 2.4.2.51 created 2013' WHERE ec = 'EC 2.4.2.51';
UPDATE enzyme SET sys_name = 'ATP:3-dephospho-CoA 5-triphosphoribosyltransferase' WHERE ec = 'EC 2.4.2.52';
UPDATE enzyme SET history = 'EC 2.4.2.52 created 2002 as EC 2.7.8.25, modified 2008, transferred 2013 to EC 2.4.2.52' WHERE ec = 'EC 2.4.2.52';
UPDATE enzyme SET sys_name = 'UDP-4-amino-4-deoxy-alpha-L-arabinose:ditrans-octacis-undecaprenyl phosphate 4-amino-4-deoxy-alpha-L-arabinosyltransferase' WHERE ec = 'EC 2.4.2.53';
UPDATE enzyme SET history = 'EC 2.4.2.53 created 2010 as EC 2.7.8.30, modified 2011, transferred 2013 to EC 2.4.2.53' WHERE ec = 'EC 2.4.2.53';
UPDATE enzyme SET sys_name = '5-phospho-alpha-D-ribose 1-diphosphate:4-aminobenzoate 5-phospho-beta-D-ribofuranosyltransferase (decarboxylating)' WHERE ec = 'EC 2.4.2.54';
UPDATE enzyme SET history = 'EC 2.4.2.54 created 2013' WHERE ec = 'EC 2.4.2.54';
UPDATE enzyme SET sys_name = 'Nicotinate D-ribonucleotide:phenol phospho-D-ribosyltransferase' WHERE ec = 'EC 2.4.2.55';
UPDATE enzyme SET history = 'EC 2.4.2.55 created 2013' WHERE ec = 'EC 2.4.2.55';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-xylose:kaempferol 3-O-D-xylosyltransferase' WHERE ec = 'EC 2.4.2.56';
UPDATE enzyme SET history = 'EC 2.4.2.56 created 2013' WHERE ec = 'EC 2.4.2.56';
UPDATE enzyme SET sys_name = 'AMP:phosphate alpha-D-ribosyl 5-phosphate-transferase' WHERE ec = 'EC 2.4.2.57';
UPDATE enzyme SET history = 'EC 2.4.2.57 created 2014' WHERE ec = 'EC 2.4.2.57';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.4.2.n1';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:beta-D-galactosyl-(1-&gt;4)-N-acetyl-beta-D-glucosamine alpha-(2-&gt;6)-N-acetylneuraminyltransferase' WHERE ec = 'EC 2.4.99.1';
UPDATE enzyme SET history = 'EC 2.4.99.1 created 1972, modified 1976, modified 1986' WHERE ec = 'EC 2.4.99.1';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:D-galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosyl-(1&lt;-&gt;1)-ceramide N-acetylneuraminyltransferase' WHERE ec = 'EC 2.4.99.2';
UPDATE enzyme SET history = 'EC 2.4.99.2 created 1976, modified 1986' WHERE ec = 'EC 2.4.99.2';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:glycano-1,3-(N-acetyl-alpha-D-galactosaminyl)-glycoprotein alpha-2,6-N-acetylneuraminyltransferase' WHERE ec = 'EC 2.4.99.3';
UPDATE enzyme SET history = 'EC 2.4.99.3 created 1984, modified 1986' WHERE ec = 'EC 2.4.99.3';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:beta-D-galactoside alpha-(2-&gt;3)-N-acetylneuraminyl-transferase' WHERE ec = 'EC 2.4.99.4';
UPDATE enzyme SET history = 'EC 2.4.99.4 created 1984, modified 1986' WHERE ec = 'EC 2.4.99.4';
UPDATE enzyme SET sys_name = 'CMP-N-acetyl-beta-neuraminate:1,2-diacyl-3-beta-D-galactosyl-sn-glycerol N-acetylneuraminyltransferase' WHERE ec = 'EC 2.4.99.5';
UPDATE enzyme SET history = 'EC 2.4.99.5 created 1984, modified 1986' WHERE ec = 'EC 2.4.99.5';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:beta-D-galactosyl-(1-&gt;4)-N-acetyl-D-glucosaminyl-glycoprotein alpha-(2-&gt;3)-N-acetylneuraminyltransferase' WHERE ec = 'EC 2.4.99.6';
UPDATE enzyme SET history = 'EC 2.4.99.6 created 1984, modified 1986' WHERE ec = 'EC 2.4.99.6';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:(alpha-N-acetylneuraminyl-2,3-beta-D-galactosyl-1,3)-N-acetyl-D-galactosaminide galactosamine-6-alpha-N-acetylneuraminyltransferase' WHERE ec = 'EC 2.4.99.7';
UPDATE enzyme SET history = 'EC 2.4.99.7 created 1984, modified 1986, modified 2004' WHERE ec = 'EC 2.4.99.7';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:alpha-N-acetylneuraminyl-(2-&gt;3)-beta-D-galactoside alpha-(2-&gt;8)-N-acetylneuraminyltransferase' WHERE ec = 'EC 2.4.99.8';
UPDATE enzyme SET history = 'EC 2.4.99.8 created 1984, modified 1986' WHERE ec = 'EC 2.4.99.8';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide alpha-(2-&gt;3)-N-acetylneuraminyltransferase' WHERE ec = 'EC 2.4.99.9';
UPDATE enzyme SET history = 'EC 2.4.99.9 created 1984, modified 1986' WHERE ec = 'EC 2.4.99.9';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:beta-D-galactosyl-(1-&gt;4)-N-acetyl-beta-D-glucosaminyl-(1-&gt;3)-beta-D-galactosyl-(1-&gt;4)-D-glucosyl-(1&lt;-&gt;1)-ceramide alpha-(2-&gt;3)-sialyltransferase' WHERE ec = 'EC 2.4.99.10';
UPDATE enzyme SET history = 'EC 2.4.99.10 created 1986' WHERE ec = 'EC 2.4.99.10';
UPDATE enzyme SET sys_name = 'CMP-N-acetylneuraminate:beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide alpha-(2-&gt;6)-N-acetylneuraminyltransferase' WHERE ec = 'EC 2.4.99.11';
UPDATE enzyme SET history = 'EC 2.4.99.11 created 1992' WHERE ec = 'EC 2.4.99.11';
UPDATE enzyme SET sys_name = 'CMP-3-deoxy-D-manno-octulosonate:lipid IV(A)  3-deoxy-D-manno-octulosonate transferase' WHERE ec = 'EC 2.4.99.12';
UPDATE enzyme SET history = 'EC 2.4.99.12 created 2010, updated 2011' WHERE ec = 'EC 2.4.99.12';
UPDATE enzyme SET sys_name = 'CMP-3-deoxy-D-manno-octulosonate:(KDO)-lipid IV(A)  3-deoxy-D-manno-octulosonate transferase' WHERE ec = 'EC 2.4.99.13';
UPDATE enzyme SET history = 'EC 2.4.99.13 created 2010, modified 2011' WHERE ec = 'EC 2.4.99.13';
UPDATE enzyme SET sys_name = 'CMP-3-deoxy-D-manno-octulosonate:(KDO)(2)-lipid IV(A) 3-deoxy-D-manno-octulosonate transferase ((2-&gt;8) glycosidic bond-forming)' WHERE ec = 'EC 2.4.99.14';
UPDATE enzyme SET history = 'EC 2.4.99.14 created 2010, modified 2011' WHERE ec = 'EC 2.4.99.14';
UPDATE enzyme SET sys_name = 'CMP-3-deoxy-D-manno-octulosonate:(KDO)(3)-lipid IV(A) 3-deoxy-D-manno-octulosonate transferase ((2-&gt;4) glycosidic bond-forming)' WHERE ec = 'EC 2.4.99.15';
UPDATE enzyme SET history = 'EC 2.4.99.15 created 2010, modified 2011 ' WHERE ec = 'EC 2.4.99.15';
UPDATE enzyme SET sys_name = 'Alpha-maltose 1-phosphate:(1-&gt;4)-alpha-D-glucan 4-alpha-D-maltosyltransferase' WHERE ec = 'EC 2.4.99.16';
UPDATE enzyme SET history = 'EC 2.4.99.16 created 2012' WHERE ec = 'EC 2.4.99.16';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:7-aminomethyl-7-deazaguanosine ribosyltransferase (ribosyl isomerizing L-methionine, adenine releasing)' WHERE ec = 'EC 2.4.99.17';
UPDATE enzyme SET history = 'EC 2.4.99.17 created 2012' WHERE ec = 'EC 2.4.99.17';
UPDATE enzyme SET sys_name = 'Dolichyl-diphosphooligosaccharide:protein-L-asparagine N-beta-D-oligopolysaccharidotransferase' WHERE ec = 'EC 2.4.99.18';
UPDATE enzyme SET history = 'EC 2.4.99.18 created 1984 as EC 2.4.1.119, transferred 2012 to EC 2.4.99.18' WHERE ec = 'EC 2.4.99.18';
UPDATE enzyme SET sys_name = 'Tritrans,heptacis-undecaprenyl-diphosphooligosaccharide:protein-L-asparagine N-beta-D-oligosaccharidotransferase' WHERE ec = 'EC 2.4.99.19';
UPDATE enzyme SET history = 'EC 2.4.99.19 created 2012' WHERE ec = 'EC 2.4.99.19';
UPDATE enzyme SET sys_name = 'NADP(+):nicotinate ADP-ribosyltransferase' WHERE ec = 'EC 2.4.99.20';
UPDATE enzyme SET history = 'EC 2.4.99.20 created 2014' WHERE ec = 'EC 2.4.99.20';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:isopentenyl-diphosphate dimethylallyltranstransferase' WHERE ec = 'EC 2.5.1.1';
UPDATE enzyme SET history = 'EC 2.5.1.1 created 1961' WHERE ec = 'EC 2.5.1.1';
UPDATE enzyme SET sys_name = 'Thiamine:base 2-methyl-4-aminopyrimidine-5-methenyltransferase' WHERE ec = 'EC 2.5.1.2';
UPDATE enzyme SET history = 'EC 2.5.1.2 created 1961, modified 1976, modified 2001' WHERE ec = 'EC 2.5.1.2';
UPDATE enzyme SET sys_name = '2-methyl-4-amino-5-hydroxymethylpyrimidine-diphosphate:4-methyl-5-(2-phosphoethyl)thiazole 2-methyl-4-aminopyrimidine-5-methenyltransferase' WHERE ec = 'EC 2.5.1.3';
UPDATE enzyme SET history = 'EC 2.5.1.3 created 1965' WHERE ec = 'EC 2.5.1.3';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine alkyltransferase (cyclizing)' WHERE ec = 'EC 2.5.1.4';
UPDATE enzyme SET history = 'EC 2.5.1.4 created 1965' WHERE ec = 'EC 2.5.1.4';
UPDATE enzyme SET sys_name = 'D-galactose-6-sulfate:alkyltransferase (cyclizing)' WHERE ec = 'EC 2.5.1.5';
UPDATE enzyme SET history = 'EC 2.5.1.5 created 1965' WHERE ec = 'EC 2.5.1.5';
UPDATE enzyme SET sys_name = 'ATP:L-methionine S-adenosyltransferase' WHERE ec = 'EC 2.5.1.6';
UPDATE enzyme SET history = 'EC 2.5.1.6 created 1961 as EC 2.4.2.13, transferred 1965 to EC 2.5.1.6' WHERE ec = 'EC 2.5.1.6';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:UDP-N-acetyl-alpha-D-glucosamine 1-carboxyvinyltransferase' WHERE ec = 'EC 2.5.1.7';
UPDATE enzyme SET history = 'EC 2.5.1.7 created 1972, modified 1983, modified 2002' WHERE ec = 'EC 2.5.1.7';
UPDATE enzyme SET history = 'EC 2.5.1.8 created 1972, deleted 2010' WHERE ec = 'EC 2.5.1.8';
UPDATE enzyme SET sys_name = '6,7-dimethyl-8-(1-D-ribityl)lumazine:6,7-dimethyl-8-(1-D-ribityl)lumazine 2,3-butanediyltransferase' WHERE ec = 'EC 2.5.1.9';
UPDATE enzyme SET history = 'EC 2.5.1.9 created 1972' WHERE ec = 'EC 2.5.1.9';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate:isopentenyl-diphosphate geranyltranstransferase' WHERE ec = 'EC 2.5.1.10';
UPDATE enzyme SET history = 'EC 2.5.1.10 created 1972, modified 2010' WHERE ec = 'EC 2.5.1.10';
UPDATE enzyme SET history = 'EC 2.5.1.11 created 1972, deleted 2010' WHERE ec = 'EC 2.5.1.11';
UPDATE enzyme SET history = 'EC 2.5.1.12 created 1972, deleted 1976' WHERE ec = 'EC 2.5.1.12';
UPDATE enzyme SET history = 'EC 2.5.1.13 created 1972, deleted 1976' WHERE ec = 'EC 2.5.1.13';
UPDATE enzyme SET history = 'EC 2.5.1.14 created 1972, deleted 1976' WHERE ec = 'EC 2.5.1.14';
UPDATE enzyme SET sys_name = '(2-amino-4-hydroxy-7,8-dihydropteridin-6-yl)methyl-diphosphate:4-aminobenzoate 2-amino-4-hydroxydihydropteridine-6-methenyltransferase' WHERE ec = 'EC 2.5.1.15';
UPDATE enzyme SET history = 'EC 2.5.1.15 created 1972' WHERE ec = 'EC 2.5.1.15';
UPDATE enzyme SET sys_name = 'S-adenosyl 3-(methylthio)propylamine:putrescine 3-aminopropyltransferase' WHERE ec = 'EC 2.5.1.16';
UPDATE enzyme SET history = 'EC 2.5.1.16 created 1972, modified 1982, modified 2013' WHERE ec = 'EC 2.5.1.16';
UPDATE enzyme SET sys_name = 'ATP:cob(I)yrinic acid-a,c-diamide Co-beta-adenosyltransferase' WHERE ec = 'EC 2.5.1.17';
UPDATE enzyme SET history = 'EC 2.5.1.17 created 1972, modified 2004' WHERE ec = 'EC 2.5.1.17';
UPDATE enzyme SET sys_name = 'RX:glutathione R-transferase' WHERE ec = 'EC 2.5.1.18';
UPDATE enzyme SET history = 'EC 2.5.1.18 created 1976 (EC 2.5.1.12, EC 2.5.1.13, EC 2.5.1.14 and EC 4.4.1.7 created 1972, incorporated 1976)' WHERE ec = 'EC 2.5.1.18';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:3-phosphoshikimate 5-O-(1-carboxyvinyl)-transferase' WHERE ec = 'EC 2.5.1.19';
UPDATE enzyme SET history = 'EC 2.5.1.19 created 1976, modified 1983, modified 1989' WHERE ec = 'EC 2.5.1.19';
UPDATE enzyme SET sys_name = 'Polycis-polyprenyl-diphosphate:isopentenyl-diphosphate polyprenylcistransferase' WHERE ec = 'EC 2.5.1.20';
UPDATE enzyme SET history = 'EC 2.5.1.20 created 1976' WHERE ec = 'EC 2.5.1.20';
UPDATE enzyme SET sys_name = 'Farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase' WHERE ec = 'EC 2.5.1.21';
UPDATE enzyme SET history = 'EC 2.5.1.21 created 1976, modified 2005' WHERE ec = 'EC 2.5.1.21';
UPDATE enzyme SET sys_name = 'S-adenosyl 3-(methylthio)propylamine:spermidine 3-aminopropyltransferase' WHERE ec = 'EC 2.5.1.22';
UPDATE enzyme SET history = 'EC 2.5.1.22 created 1982, modified 2013' WHERE ec = 'EC 2.5.1.22';
UPDATE enzyme SET sys_name = 'S-adenosyl 3-(methylthio)propylamine:propane-1,3-diamine 3-aminopropyltransferase' WHERE ec = 'EC 2.5.1.23';
UPDATE enzyme SET history = 'EC 2.5.1.23 created 1983, modified 2013' WHERE ec = 'EC 2.5.1.23';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:N(6)-(Delta(2)-isopentenyl)-adenine 3-(3-amino-3-carboxypropyl)-transferase' WHERE ec = 'EC 2.5.1.24';
UPDATE enzyme SET history = 'EC 2.5.1.24 created 1984' WHERE ec = 'EC 2.5.1.24';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA-uridine 3-(3-amino-3-carboxypropyl)transferase' WHERE ec = 'EC 2.5.1.25';
UPDATE enzyme SET history = 'EC 2.5.1.25 created 1984' WHERE ec = 'EC 2.5.1.25';
UPDATE enzyme SET sys_name = '1-acyl-glycerone-3-phosphate:long-chain-alcohol O-3-phospho-2-oxopropanyltransferase' WHERE ec = 'EC 2.5.1.26';
UPDATE enzyme SET history = 'EC 2.5.1.26 created 1984' WHERE ec = 'EC 2.5.1.26';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:AMP dimethylallyltransferase' WHERE ec = 'EC 2.5.1.27';
UPDATE enzyme SET history = 'EC 2.5.1.27 created 1984, modified 2002, modified 2013' WHERE ec = 'EC 2.5.1.27';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:isopentenyl-diphosphate dimethylallylcistransferase' WHERE ec = 'EC 2.5.1.28';
UPDATE enzyme SET history = 'EC 2.5.1.28 created 1984' WHERE ec = 'EC 2.5.1.28';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase' WHERE ec = 'EC 2.5.1.29';
UPDATE enzyme SET history = 'EC 2.5.1.29 created 1984, modifed 2011' WHERE ec = 'EC 2.5.1.29';
UPDATE enzyme SET sys_name = 'All-trans-hexaprenyl-diphosphate:isopentenyl-diphosphate hexaprenyltranstransferase' WHERE ec = 'EC 2.5.1.30';
UPDATE enzyme SET history = 'EC 2.5.1.30 created 1984, modified 2010' WHERE ec = 'EC 2.5.1.30';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate cistransferase (adding 8 isopentenyl units)' WHERE ec = 'EC 2.5.1.31';
UPDATE enzyme SET history = 'EC 2.5.1.31 created 1984, modified 2010, modified 2011' WHERE ec = 'EC 2.5.1.31';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:geranylgeranyl-diphosphate geranylgeranyltransferase (15-cis-phytoene forming)' WHERE ec = 'EC 2.5.1.32';
UPDATE enzyme SET history = 'EC 2.5.1.32 created 1984, modified 2005, modified 2012' WHERE ec = 'EC 2.5.1.32';
UPDATE enzyme SET history = 'EC 2.5.1.33 created 1984, deleted 2010' WHERE ec = 'EC 2.5.1.33';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:L-tryptophan dimethylallyltransferase' WHERE ec = 'EC 2.5.1.34';
UPDATE enzyme SET history = 'EC 2.5.1.34 created 1984, modified 2010' WHERE ec = 'EC 2.5.1.34';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:aspulvinone-E dimethylallyltransferase' WHERE ec = 'EC 2.5.1.35';
UPDATE enzyme SET history = 'EC 2.5.1.35 created 1984' WHERE ec = 'EC 2.5.1.35';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:(6aS,11aS)-3,6a,9-trihydroxypterocarpan dimethylallyltransferase' WHERE ec = 'EC 2.5.1.36';
UPDATE enzyme SET history = 'EC 2.5.1.36 created 1989' WHERE ec = 'EC 2.5.1.36';
UPDATE enzyme SET history = 'EC 2.5.1.37 created 1989, deleted 2004' WHERE ec = 'EC 2.5.1.37';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:nocardicin-E 3-amino-3-carboxypropyltransferase' WHERE ec = 'EC 2.5.1.38';
UPDATE enzyme SET history = 'EC 2.5.1.38 created 1992' WHERE ec = 'EC 2.5.1.38';
UPDATE enzyme SET sys_name = 'Polyprenyl-diphosphate:4-hydroxybenzoate polyprenyltransferase' WHERE ec = 'EC 2.5.1.39';
UPDATE enzyme SET history = 'EC 2.5.1.39 created 1992, modified 2010' WHERE ec = 'EC 2.5.1.39';
UPDATE enzyme SET history = 'EC 2.5.1.40 created 1992, deleted 1999' WHERE ec = 'EC 2.5.1.40';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:sn-glycerol-1-phosphate geranylgeranyltransferase' WHERE ec = 'EC 2.5.1.41';
UPDATE enzyme SET history = 'EC 2.5.1.41 created 1992, modified 2009' WHERE ec = 'EC 2.5.1.41';
UPDATE enzyme SET sys_name = 'Geranylgeranyl diphosphate:sn-3-O-(geranylgeranyl)glycerol 1-phosphate geranylgeranyltransferase' WHERE ec = 'EC 2.5.1.42';
UPDATE enzyme SET history = 'EC 2.5.1.42 created 1992, modified 2009' WHERE ec = 'EC 2.5.1.42';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:S-adenosyl-L-methionine:S-adenosyl-L-methionine 3-amino-3-carboxypropyltransferase' WHERE ec = 'EC 2.5.1.43';
UPDATE enzyme SET history = 'EC 2.5.1.43 created 1999' WHERE ec = 'EC 2.5.1.43';
UPDATE enzyme SET sys_name = 'Putrescine:putrescine 4-aminobutyltransferase (ammonia-forming)' WHERE ec = 'EC 2.5.1.44';
UPDATE enzyme SET history = 'EC 2.5.1.44 created 1999, modified 2001' WHERE ec = 'EC 2.5.1.44';
UPDATE enzyme SET sys_name = 'Spermidine:putrescine 4-aminobutyltransferase (propane-1,3-diamine-forming)' WHERE ec = 'EC 2.5.1.45';
UPDATE enzyme SET history = 'EC 2.5.1.45 created 2001' WHERE ec = 'EC 2.5.1.45';
UPDATE enzyme SET sys_name = '[eIF5A-precursor]-lysine:spermidine 4-aminobutyltransferase (propane-1,3-diamine-forming)' WHERE ec = 'EC 2.5.1.46';
UPDATE enzyme SET history = 'EC 2.5.1.46 provisional version created 1999 as EC 1.1.1.249 deleted 1999, revised and reinstated 2001 as EC 2.5.1.46' WHERE ec = 'EC 2.5.1.46';
UPDATE enzyme SET sys_name = 'O(3)-acetyl-L-serine:hydrogen-sulfide 2-amino-2-carboxyethyltransferase' WHERE ec = 'EC 2.5.1.47';
UPDATE enzyme SET history = 'EC 2.5.1.47 created 1972 as 4.2.99.8, modified 1976, modified 1990, transferred 2002 to EC 2.5.1.47' WHERE ec = 'EC 2.5.1.47';
UPDATE enzyme SET sys_name = 'O(4)-succinyl-L-homoserine:L-cysteine S-(3-amino-3-carboxypropyl)transferase' WHERE ec = 'EC 2.5.1.48';
UPDATE enzyme SET history = 'EC 2.5.1.48 created 1972 as EC 4.2.99.9, transferred 2002 to EC 2.5.1.48' WHERE ec = 'EC 2.5.1.48';
UPDATE enzyme SET sys_name = 'O-acetyl-L-homoserine:methanethiol 3-amino-3-carboxypropyltransferase' WHERE ec = 'EC 2.5.1.49';
UPDATE enzyme SET history = 'EC 2.5.1.49 created 1972 as EC 4.2.99.10, transferred 2002 to EC 2.5.1.49' WHERE ec = 'EC 2.5.1.49';
UPDATE enzyme SET sys_name = 'O(3)-acetyl-L-serine:zeatin 2-amino-2-carboxyethyltransferase' WHERE ec = 'EC 2.5.1.50';
UPDATE enzyme SET history = 'EC 2.5.1.50 created 1984 as EC 4.2.99.13, transferred 2002 to EC 2.5.1.50' WHERE ec = 'EC 2.5.1.50';
UPDATE enzyme SET sys_name = 'O(3)-acetyl-L-serine:pyrazole 1-(2-amino-2-carboxyethyl)transferase' WHERE ec = 'EC 2.5.1.51';
UPDATE enzyme SET history = 'EC 2.5.1.51 created 1989 as EC 4.2.99.14 (EC 4.2.99.17 incorporated 1992), transferred 2002 to EC 2.5.1.51' WHERE ec = 'EC 2.5.1.51';
UPDATE enzyme SET sys_name = 'O(3)-acetyl-L-serine:3,4-dihydroxypyridine 1-(2-amino-2-carboxyethyl)transferase' WHERE ec = 'EC 2.5.1.52';
UPDATE enzyme SET history = 'EC 2.5.1.52 created 1989 as EC 4.2.99.15, transferred 2002 to EC 2.5.1.52' WHERE ec = 'EC 2.5.1.52';
UPDATE enzyme SET sys_name = 'O(3)-acetyl-L-serine:uracil 1-(2-amino-2-carboxyethyl)transferase' WHERE ec = 'EC 2.5.1.53';
UPDATE enzyme SET history = 'EC 2.5.1.53 created 1990 as EC 4.2.99.16, transferred 2002 to EC 2.5.1.53' WHERE ec = 'EC 2.5.1.53';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:D-erythrose-4-phosphate C-(1-carboxyvinyl)transferase (phosphate-hydrolyzing, 2-carboxy-2-oxoethyl-forming)' WHERE ec = 'EC 2.5.1.54';
UPDATE enzyme SET history = 'EC 2.5.1.54 created 1965 as EC 4.1.2.15, modified 1976, transferred 2002 to EC 2.5.1.54' WHERE ec = 'EC 2.5.1.54';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:D-arabinose-5-phosphate C-(1-carboxyvinyl)transferase (phosphate-hydrolyzing, 2-carboxy-2-oxoethyl-forming)' WHERE ec = 'EC 2.5.1.55';
UPDATE enzyme SET history = 'EC 2.5.1.55 created 1965 as EC 4.1.2.16, transferred 2002 to EC 2.5.1.55' WHERE ec = 'EC 2.5.1.55';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:N-acetyl-D-mannosamine C-(1-carboxyvinyl)transferase (phosphate-hydrolyzing, 2-carboxy-2-oxoethyl-forming)' WHERE ec = 'EC 2.5.1.56';
UPDATE enzyme SET history = 'EC 2.5.1.56 created 1972 as EC 4.1.3.19, transferred 2002 to EC 2.5.1.56' WHERE ec = 'EC 2.5.1.56';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:N-acyl-D-mannosamine-6-phosphate 1-(2-carboxy-2-oxoethyl)transferase' WHERE ec = 'EC 2.5.1.57';
UPDATE enzyme SET history = 'EC 2.5.1.57 created 1972 as EC 4.1.3.20, transferred 2002 to EC 2.5.1.57' WHERE ec = 'EC 2.5.1.57';
UPDATE enzyme SET sys_name = 'Farnesyl-diphosphate:protein-cysteine farnesyltransferase' WHERE ec = 'EC 2.5.1.58';
UPDATE enzyme SET history = 'EC 2.5.1.58 created 2003' WHERE ec = 'EC 2.5.1.58';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:protein-cysteine geranyltransferase' WHERE ec = 'EC 2.5.1.59';
UPDATE enzyme SET history = 'EC 2.5.1.59 created 2003' WHERE ec = 'EC 2.5.1.59';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:protein-cysteine geranyltransferase' WHERE ec = 'EC 2.5.1.60';
UPDATE enzyme SET history = 'EC 2.5.1.60 created 2003' WHERE ec = 'EC 2.5.1.60';
UPDATE enzyme SET sys_name = 'Porphobilinogen:(4-(2-carboxyethyl)-3-(carboxymethyl)pyrrol-2-yl)methyltransferase (hydrolyzing)' WHERE ec = 'EC 2.5.1.61';
UPDATE enzyme SET history = 'EC 2.5.1.61 created 1972 as EC 4.3.1.8, transferred 2003 to EC 2.5.1.61' WHERE ec = 'EC 2.5.1.61';
UPDATE enzyme SET sys_name = 'Chlorophyllide-a:phytyl-diphosphate phytyltransferase' WHERE ec = 'EC 2.5.1.62';
UPDATE enzyme SET history = 'EC 2.5.1.62 created 2003' WHERE ec = 'EC 2.5.1.62';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:fluoride adenosyltransferase' WHERE ec = 'EC 2.5.1.63';
UPDATE enzyme SET history = 'EC 2.5.1.63 created 2003' WHERE ec = 'EC 2.5.1.63';
UPDATE enzyme SET history = 'EC 2.5.1.64 created 2003, deleted 2008' WHERE ec = 'EC 2.5.1.64';
UPDATE enzyme SET sys_name = 'O-phospho-L-serine:hydrogen-sulfide 2-amino-2-carboxyethyltransferase' WHERE ec = 'EC 2.5.1.65';
UPDATE enzyme SET history = 'EC 2.5.1.65 created 2004' WHERE ec = 'EC 2.5.1.65';
UPDATE enzyme SET sys_name = 'Glyceraldehyde-3-phosphate:L-arginine N(2)-(2-hydroxy-3-oxopropyl) transferase (2-carboxyethyl-forming)' WHERE ec = 'EC 2.5.1.66';
UPDATE enzyme SET history = 'EC 2.5.1.66 created 2004' WHERE ec = 'EC 2.5.1.66';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:dimethylallyl-diphosphate dimethylallyltransferase (chrysanthemyl-diphosphate-forming)' WHERE ec = 'EC 2.5.1.67';
UPDATE enzyme SET history = 'EC 2.5.1.67 created 2007' WHERE ec = 'EC 2.5.1.67';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate:isopentenyl-diphosphate geranylcistransferase' WHERE ec = 'EC 2.5.1.68';
UPDATE enzyme SET history = 'EC 2.5.1.68 created 2007, modified 2010' WHERE ec = 'EC 2.5.1.68';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:dimethylallyl-diphosphate dimethylallyltransferase (lavandulyl-diphosphate-forming)' WHERE ec = 'EC 2.5.1.69';
UPDATE enzyme SET history = 'EC 2.5.1.69 created 2007' WHERE ec = 'EC 2.5.1.69';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:naringenin 8-dimethylallyltransferase' WHERE ec = 'EC 2.5.1.70';
UPDATE enzyme SET history = 'EC 2.5.1.70 created 2007' WHERE ec = 'EC 2.5.1.70';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:leachianone-G 2-dimethylallyltransferase' WHERE ec = 'EC 2.5.1.71';
UPDATE enzyme SET history = 'EC 2.5.1.71 created 2007' WHERE ec = 'EC 2.5.1.71';
UPDATE enzyme SET sys_name = 'Glycerone phosphate:iminosuccinate alkyltransferase (cyclizing)' WHERE ec = 'EC 2.5.1.72';
UPDATE enzyme SET history = 'EC 2.5.1.72 created 2008' WHERE ec = 'EC 2.5.1.72';
UPDATE enzyme SET sys_name = 'O-phospho-L-seryl-tRNA(Cys):hydrogen sulfide 2-aminopropanoate transferase' WHERE ec = 'EC 2.5.1.73';
UPDATE enzyme SET history = 'EC 2.5.1.73 created 2009' WHERE ec = 'EC 2.5.1.73';
UPDATE enzyme SET sys_name = 'All-transall-trans-polyprenyl diphosphate:1,4-dihydroxy-2-naphthoate polyprenyltransferase' WHERE ec = 'EC 2.5.1.74';
UPDATE enzyme SET history = 'EC 2.5.1.74 created 2009' WHERE ec = 'EC 2.5.1.74';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:tRNA dimethylallyltransferase' WHERE ec = 'EC 2.5.1.75';
UPDATE enzyme SET history = 'EC 2.5.1.75 created 1972 as EC 2.5.1.8, transferred 2009 to EC 2.5.1.75' WHERE ec = 'EC 2.5.1.75';
UPDATE enzyme SET sys_name = 'Sulfite:O-phospho-L-serine sulfotransferase (phosphate-hydrolyzing, L-cysteate-forming)' WHERE ec = 'EC 2.5.1.76';
UPDATE enzyme SET history = 'EC 2.5.1.76 created 2009' WHERE ec = 'EC 2.5.1.76';
UPDATE enzyme SET sys_name = '5-amino-6-(D-ribitylamino)uracil:4-hydroxyphenylpyruvate, 4-methylphenol transferase' WHERE ec = 'EC 2.5.1.77';
UPDATE enzyme SET history = 'EC 2.5.1.77 created 2010' WHERE ec = 'EC 2.5.1.77';
UPDATE enzyme SET sys_name = '5-amino-6-(D-ribitylamino)uracil butanedionetransferase' WHERE ec = 'EC 2.5.1.78';
UPDATE enzyme SET history = 'EC 2.5.1.78 created 2010' WHERE ec = 'EC 2.5.1.78';
UPDATE enzyme SET sys_name = 'S-adenosyl 3-(methylthio)propylamine:spermidine 3-aminopropyltransferase (thermospermine synthesizing)' WHERE ec = 'EC 2.5.1.79';
UPDATE enzyme SET history = 'EC 2.5.1.79 created 2010, modified 2013' WHERE ec = 'EC 2.5.1.79';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:L-tryptophan 7-dimethylallyltransferase' WHERE ec = 'EC 2.5.1.80';
UPDATE enzyme SET history = 'EC 2.5.1.80 created 2010' WHERE ec = 'EC 2.5.1.80';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:isopentenyl-diphosphate transtransferase (adding 1 isopentenyl unit)' WHERE ec = 'EC 2.5.1.81';
UPDATE enzyme SET history = 'EC 2.5.1.81 created 2010' WHERE ec = 'EC 2.5.1.81';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:isopentenyl-diphosphate transferase (adding 2 isopentenyl units)' WHERE ec = 'EC 2.5.1.82';
UPDATE enzyme SET history = 'EC 2.5.1.82 created 1984 as EC 2.5.1.33, part-transferred 2010 to EC 2.5.1.82' WHERE ec = 'EC 2.5.1.82';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase (adding 3 isopentenyl units)' WHERE ec = 'EC 2.5.1.83';
UPDATE enzyme SET history = 'EC 2.5.1.83 created 1984 as EC 2.5.1.33, part-transferred 2010 to EC 2.5.1.83' WHERE ec = 'EC 2.5.1.83';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate:isopentenyl-diphosphate transtransferase (adding 7 isopentenyl units)' WHERE ec = 'EC 2.5.1.84';
UPDATE enzyme SET history = 'EC 2.5.1.84, created 1972 as EC 2.5.1.11, part-transferred 2010 to EC 2.5.1.84' WHERE ec = 'EC 2.5.1.84';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:isopentenyl-diphosphate transtransferase (adding 5 isopentenyl units)' WHERE ec = 'EC 2.5.1.85';
UPDATE enzyme SET history = 'EC 2.5.1.85 created 1972 as EC 2.5.1.11, part-transferred 2010 to EC 2.5.1.85' WHERE ec = 'EC 2.5.1.85';
UPDATE enzyme SET sys_name = '(2Z,6E)-farnesyl-diphosphate:isopentenyl-diphosphate farnesylcistransferase (adding 7 isopentenyl units)' WHERE ec = 'EC 2.5.1.86';
UPDATE enzyme SET history = 'EC 2.5.1.86 created 2010' WHERE ec = 'EC 2.5.1.86';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate cistransferase (adding 10-55 isopentenyl units)' WHERE ec = 'EC 2.5.1.87';
UPDATE enzyme SET history = 'EC 2.5.1.87 created 2010' WHERE ec = 'EC 2.5.1.87';
UPDATE enzyme SET sys_name = '(2Z,6E)-farnesyl-diphosphate:isopentenyl-diphosphate cistransferase (adding 9-11 isopentenyl units)' WHERE ec = 'EC 2.5.1.88';
UPDATE enzyme SET history = 'EC 2.5.1.88 created 2010' WHERE ec = 'EC 2.5.1.88';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:isopentenyl-diphosphate cistransferase (adding 7 isopentenyl units)' WHERE ec = 'EC 2.5.1.89';
UPDATE enzyme SET history = 'EC 2.5.1.89 created 2010, modified 2011' WHERE ec = 'EC 2.5.1.89';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase (adding 5 isopentenyl units)' WHERE ec = 'EC 2.5.1.90';
UPDATE enzyme SET history = 'EC 2.5.1.90 created 2010' WHERE ec = 'EC 2.5.1.90';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase (adding 7 isopentenyl units)' WHERE ec = 'EC 2.5.1.91';
UPDATE enzyme SET history = 'EC 2.5.1.91 created 2010' WHERE ec = 'EC 2.5.1.91';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:isopentenyl-diphosphate cistransferase (adding 2 isopentenyl units)' WHERE ec = 'EC 2.5.1.92';
UPDATE enzyme SET history = 'EC 2.5.1.92 created 2010, modified 2011' WHERE ec = 'EC 2.5.1.92';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate:4-hydroxybenzoate 3-geranyltransferase' WHERE ec = 'EC 2.5.1.93';
UPDATE enzyme SET history = 'EC 2.5.1.93 created 2010' WHERE ec = 'EC 2.5.1.93';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:chloride adenosyltransferase' WHERE ec = 'EC 2.5.1.94';
UPDATE enzyme SET history = 'EC 2.5.1.94 created 2010' WHERE ec = 'EC 2.5.1.94';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:D-Man-beta-(1-&gt;4)-GlcA-beta-(1-&gt;2)-D-Man-alpha-(1-&gt;3)-D-Glc-beta-(1-&gt;4)-D-Glc-alpha-1-diphospho-ditrans,octacis-undecaprenol 4,6-O-(1-carboxyethan-1,1-diyl)transferase' WHERE ec = 'EC 2.5.1.95';
UPDATE enzyme SET history = 'EC 2.5.1.95 created 2011, modified 2012' WHERE ec = 'EC 2.5.1.95';
UPDATE enzyme SET sys_name = 'Farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase (4,4-diapophytoene forming)' WHERE ec = 'EC 2.5.1.96';
UPDATE enzyme SET history = 'EC 2.5.1.96 created 2011' WHERE ec = 'EC 2.5.1.96';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:2,4-bis(acetylamino)-2,4,6-trideoxy-beta-L-altropyranose transferase (phosphate-hydrolyzing, 2,7-acetylamino-transfering, 2-carboxy-2-oxoethyl-forming)' WHERE ec = 'EC 2.5.1.97';
UPDATE enzyme SET history = 'EC 2.5.1.97 created 2011' WHERE ec = 'EC 2.5.1.97';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:[D-GlcA-beta-(1-&gt;4)-2-O-Ac-D-GlcA-beta-(1-&gt;4)-D-Glc-beta-(1-&gt;4)-(3-O-CH(3)-CH(2)CH(OH)C(O)-D-Gal-beta-(1-&gt;4)-D-Glc-beta-(1-&gt;4)-D-Glc-beta-(1-&gt;4)-D-Glc-beta-(1-&gt;6))-2(or3)-O-Ac-D-Glc-alpha-(1-&gt;6)](n) 4,6-O-(1-carboxyethan-1,1-diyl)transferase' WHERE ec = 'EC 2.5.1.98';
UPDATE enzyme SET history = 'EC 2.5.1.98 created 2012' WHERE ec = 'EC 2.5.1.98';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate:geranylgeranyl-diphosphate geranylgeranyltransferase (all-trans-phytoene forming)' WHERE ec = 'EC 2.5.1.99';
UPDATE enzyme SET history = 'EC 2.5.1.99 created 2012' WHERE ec = 'EC 2.5.1.99';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:fumigaclavine A dimethylallyltransferase' WHERE ec = 'EC 2.5.1.100';
UPDATE enzyme SET history = 'EC 2.5.1.100 created 2012' WHERE ec = 'EC 2.5.1.100';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:2,4-diacetamido-2,4,6-trideoxy-alpha-D-mannopyranose 1-(2-carboxy-2-oxoethyl)transferase' WHERE ec = 'EC 2.5.1.101';
UPDATE enzyme SET history = 'EC 2.5.1.101 created 2012' WHERE ec = 'EC 2.5.1.101';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate:olivetolate geranyltransferase' WHERE ec = 'EC 2.5.1.102';
UPDATE enzyme SET history = 'EC 2.5.1.102 created 2012' WHERE ec = 'EC 2.5.1.102';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate:(2E,6E)-farnesyl-diphosphate farnesyltransferase (presqualene diphosphate forming)' WHERE ec = 'EC 2.5.1.103';
UPDATE enzyme SET history = 'EC 2.5.1.103 created 2012' WHERE ec = 'EC 2.5.1.103';
UPDATE enzyme SET sys_name = 'S-adenosylmethioninamine:agmatine 3-aminopropyltransferase' WHERE ec = 'EC 2.5.1.104';
UPDATE enzyme SET history = 'EC 2.5.1.104 created 2013' WHERE ec = 'EC 2.5.1.104';
UPDATE enzyme SET sys_name = '(7,8-dihydropterin-6-yl)methyl diphosphate:4-(beta-D-ribofuranosyl)aniline 5-phosphate 6-hydroxymethyl-7,8-dihydropterintransferase' WHERE ec = 'EC 2.5.1.105';
UPDATE enzyme SET history = 'EC 2.5.1.105 created 2013' WHERE ec = 'EC 2.5.1.105';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:brevianamide-F dimethylallyl-C-2-transferase' WHERE ec = 'EC 2.5.1.106';
UPDATE enzyme SET history = 'EC 2.5.1.106 created 2013' WHERE ec = 'EC 2.5.1.106';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:verruculogen dimethylallyl-O-transferase' WHERE ec = 'EC 2.5.1.107';
UPDATE enzyme SET history = 'EC 2.5.1.107 created 2013' WHERE ec = 'EC 2.5.1.107';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:L-histidine-[translation elongation factor 2] 2-((3S)-3-amino-3-carboxypropyl)transferase' WHERE ec = 'EC 2.5.1.108';
UPDATE enzyme SET history = 'EC 2.5.1.108 created 2013' WHERE ec = 'EC 2.5.1.108';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:brevianamide-F tert-dimethylallyl-C-2-transferase' WHERE ec = 'EC 2.5.1.109';
UPDATE enzyme SET history = 'EC 2.5.1.109 created 2013' WHERE ec = 'EC 2.5.1.109';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:12-alpha,13-alpha-dihydroxyfumitremorgin C dimethylallyl-N-1-transferase' WHERE ec = 'EC 2.5.1.110';
UPDATE enzyme SET history = 'EC 2.5.1.110 created 2013' WHERE ec = 'EC 2.5.1.110';
UPDATE enzyme SET sys_name = 'Dimethylallyl diphosphate:4-hydroxyphenylpyruvate 3-dimethylallyltransferase' WHERE ec = 'EC 2.5.1.111';
UPDATE enzyme SET history = 'EC 2.5.1.111 created 2013' WHERE ec = 'EC 2.5.1.111';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate:ADP/ATP dimethylallyltransferase' WHERE ec = 'EC 2.5.1.112';
UPDATE enzyme SET history = 'EC 2.5.1.112 created 2013' WHERE ec = 'EC 2.5.1.112';
UPDATE enzyme SET sys_name = 'O-phospho-L-serine:thiocarboxylated [CysO sulfur-carrier protein] 2-amino-2-carboxyethyltransferase' WHERE ec = 'EC 2.5.1.113';
UPDATE enzyme SET history = 'EC 2.5.1.113 created 2013' WHERE ec = 'EC 2.5.1.113';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:tRNA(Phe) (4-demethylwyosine(37)-C(7))-((3S)-3-amino-3-carboxypropyl)transferase' WHERE ec = 'EC 2.5.1.114';
UPDATE enzyme SET history = 'EC 2.5.1.114 created 2013' WHERE ec = 'EC 2.5.1.114';
UPDATE enzyme SET sys_name = 'Phytyl diphosphate:homogentisate phytyltransferase' WHERE ec = 'EC 2.5.1.115';
UPDATE enzyme SET history = 'EC 2.5.1.115 created 2014' WHERE ec = 'EC 2.5.1.115';
UPDATE enzyme SET sys_name = 'Geranylgeranyl diphosphate:homogentisate geranylgeranyltransferase' WHERE ec = 'EC 2.5.1.116';
UPDATE enzyme SET history = 'EC 2.5.1.116 created 2014' WHERE ec = 'EC 2.5.1.116';
UPDATE enzyme SET sys_name = 'All-trans-nonaprenyl diphosphate:homogentisate nonaprenyltransferase' WHERE ec = 'EC 2.5.1.117';
UPDATE enzyme SET history = 'EC 2.5.1.117 created 2014' WHERE ec = 'EC 2.5.1.117';
UPDATE enzyme SET sys_name = 'O(3)-acetyl-L-serine:isoxazolin-5-one 2-(2-amino-2-carboxyethyl)transferase' WHERE ec = 'EC 2.5.1.118';
UPDATE enzyme SET history = 'EC 2.5.1.118 created 2014' WHERE ec = 'EC 2.5.1.118';
UPDATE enzyme SET sys_name = 'O(3)-acetyl-L-serine:isoxazolin-5-one 4-(2-amino-2-carboxyethyl)transferase' WHERE ec = 'EC 2.5.1.119';
UPDATE enzyme SET history = 'EC 2.5.1.119 created 2014' WHERE ec = 'EC 2.5.1.119';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:3-((1-carboxyvinyl)-oxy)benzoate adenosyltransferase (bicarbonate-hydrolyzing, 6-amino-6-deoxyfutalosine-forming)' WHERE ec = 'EC 2.5.1.120';
UPDATE enzyme SET history = 'EC 2.5.1.120 created 2014' WHERE ec = 'EC 2.5.1.120';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.5.1.n2';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.5.1.n4';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.5.1.n5';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.5.1.n6';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.5.1.n7';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.5.1.n8';
UPDATE enzyme SET sys_name = 'L-aspartate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.1';
UPDATE enzyme SET history = 'EC 2.6.1.1 created 1961, modified 1976' WHERE ec = 'EC 2.6.1.1';
UPDATE enzyme SET sys_name = 'L-alanine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.2';
UPDATE enzyme SET history = 'EC 2.6.1.2 created 1961' WHERE ec = 'EC 2.6.1.2';
UPDATE enzyme SET sys_name = 'L-cysteine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.3';
UPDATE enzyme SET history = 'EC 2.6.1.3 created 1961' WHERE ec = 'EC 2.6.1.3';
UPDATE enzyme SET sys_name = 'Glycine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.4';
UPDATE enzyme SET history = 'EC 2.6.1.4 created 1961, modified 1982' WHERE ec = 'EC 2.6.1.4';
UPDATE enzyme SET sys_name = 'L-tyrosine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.5';
UPDATE enzyme SET history = 'EC 2.6.1.5 created 1961' WHERE ec = 'EC 2.6.1.5';
UPDATE enzyme SET sys_name = 'L-leucine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.6';
UPDATE enzyme SET history = 'EC 2.6.1.6 created 1961, modified 1982' WHERE ec = 'EC 2.6.1.6';
UPDATE enzyme SET sys_name = 'L-kynurenine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.7';
UPDATE enzyme SET history = 'EC 2.6.1.7 created 1961, modified 1983' WHERE ec = 'EC 2.6.1.7';
UPDATE enzyme SET sys_name = '2,5-diaminopentanoate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.8';
UPDATE enzyme SET history = 'EC 2.6.1.8 created 1961, modified 1982' WHERE ec = 'EC 2.6.1.8';
UPDATE enzyme SET sys_name = 'L-histidinol-phosphate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.9';
UPDATE enzyme SET history = 'EC 2.6.1.9 created 1961' WHERE ec = 'EC 2.6.1.9';
UPDATE enzyme SET history = 'EC 2.6.1.10 created 1961, deleted 1972' WHERE ec = 'EC 2.6.1.10';
UPDATE enzyme SET sys_name = 'N(2)-acetyl-L-ornithine:2-oxoglutarate 5-aminotransferase' WHERE ec = 'EC 2.6.1.11';
UPDATE enzyme SET history = 'EC 2.6.1.11 created 1961, modified 2004 (EC 2.6.1.69 created 1989, incorporated 2004)' WHERE ec = 'EC 2.6.1.11';
UPDATE enzyme SET sys_name = 'L-alanine:2-oxo-acid aminotransferase' WHERE ec = 'EC 2.6.1.12';
UPDATE enzyme SET history = 'EC 2.6.1.12 created 1961' WHERE ec = 'EC 2.6.1.12';
UPDATE enzyme SET sys_name = 'L-ornithine:2-oxo-acid aminotransferase' WHERE ec = 'EC 2.6.1.13';
UPDATE enzyme SET history = 'EC 2.6.1.13 created 1961' WHERE ec = 'EC 2.6.1.13';
UPDATE enzyme SET sys_name = 'L-asparagine:2-oxo-acid aminotransferase' WHERE ec = 'EC 2.6.1.14';
UPDATE enzyme SET history = 'EC 2.6.1.14 created 1961' WHERE ec = 'EC 2.6.1.14';
UPDATE enzyme SET sys_name = 'L-glutamine:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.15';
UPDATE enzyme SET history = 'EC 2.6.1.15 created 1961' WHERE ec = 'EC 2.6.1.15';
UPDATE enzyme SET sys_name = 'L-glutamine:D-fructose-6-phosphate isomerase (deaminating)' WHERE ec = 'EC 2.6.1.16';
UPDATE enzyme SET history = 'EC 2.6.1.16 created 1961, deleted 1972, reinstated 1984, modified 2000 (EC 5.3.1.19 created 1972, incorporated 1984)' WHERE ec = 'EC 2.6.1.16';
UPDATE enzyme SET sys_name = 'N-succinyl-L-2,6-diaminoheptanedioate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.17';
UPDATE enzyme SET history = 'EC 2.6.1.17 created 1965' WHERE ec = 'EC 2.6.1.17';
UPDATE enzyme SET sys_name = 'L-alanine:3-oxopropanoate aminotransferase' WHERE ec = 'EC 2.6.1.18';
UPDATE enzyme SET history = 'EC 2.6.1.18 created 1965' WHERE ec = 'EC 2.6.1.18';
UPDATE enzyme SET sys_name = '4-aminobutanoate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.19';
UPDATE enzyme SET history = 'EC 2.6.1.19 created 1965, modified 1982, modified 2012' WHERE ec = 'EC 2.6.1.19';
UPDATE enzyme SET history = 'EC 2.6.1.20 created 1965, deleted 1972' WHERE ec = 'EC 2.6.1.20';
UPDATE enzyme SET sys_name = 'D-alanine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.21';
UPDATE enzyme SET history = 'EC 2.6.1.21 created 1972 (EC 2.6.1.10 created 1961, incorporated 1972), modified 2005' WHERE ec = 'EC 2.6.1.21';
UPDATE enzyme SET sys_name = '(S)-3-amino-2-methylpropanoate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.22';
UPDATE enzyme SET history = 'EC 2.6.1.22 created 1972, modified 1982, modified 2004' WHERE ec = 'EC 2.6.1.22';
UPDATE enzyme SET sys_name = '4-hydroxy-L-glutamate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.23';
UPDATE enzyme SET history = 'EC 2.6.1.23 created 1972, modified 1982' WHERE ec = 'EC 2.6.1.23';
UPDATE enzyme SET sys_name = '3,5-diiodo-L-tyrosine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.24';
UPDATE enzyme SET history = 'EC 2.6.1.24 created 1972 (EC 2.6.1.25 created 1972, incorporated 1972)' WHERE ec = 'EC 2.6.1.24';
UPDATE enzyme SET history = 'EC 2.6.1.25 created 1972, deleted 1984' WHERE ec = 'EC 2.6.1.25';
UPDATE enzyme SET sys_name = 'L-3,5,3-triiodothyronine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.26';
UPDATE enzyme SET history = 'EC 2.6.1.26 created 1972' WHERE ec = 'EC 2.6.1.26';
UPDATE enzyme SET sys_name = 'L-tryptophan:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.27';
UPDATE enzyme SET history = 'EC 2.6.1.27 created 1972' WHERE ec = 'EC 2.6.1.27';
UPDATE enzyme SET sys_name = 'L-tryptophan:phenylpyruvate aminotransferase' WHERE ec = 'EC 2.6.1.28';
UPDATE enzyme SET history = 'EC 2.6.1.28 created 1972' WHERE ec = 'EC 2.6.1.28';
UPDATE enzyme SET sys_name = 'Diamine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.29';
UPDATE enzyme SET history = 'EC 2.6.1.29 created 1972' WHERE ec = 'EC 2.6.1.29';
UPDATE enzyme SET sys_name = 'Pyridoxamine:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.30';
UPDATE enzyme SET history = 'EC 2.6.1.30 created 1972' WHERE ec = 'EC 2.6.1.30';
UPDATE enzyme SET sys_name = 'Pyridoxamine:oxaloacetate aminotransferase' WHERE ec = 'EC 2.6.1.31';
UPDATE enzyme SET history = 'EC 2.6.1.31 created 1972' WHERE ec = 'EC 2.6.1.31';
UPDATE enzyme SET sys_name = 'L-valine:(S)-3-methyl-2-oxopentanoate aminotransferase' WHERE ec = 'EC 2.6.1.32';
UPDATE enzyme SET history = 'EC 2.6.1.32 created 1972, modified 1976' WHERE ec = 'EC 2.6.1.32';
UPDATE enzyme SET sys_name = 'dTDP-4-amino-4,6-dideoxy-alpha-D-glucose:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.33';
UPDATE enzyme SET history = 'EC 2.6.1.33 created 1972' WHERE ec = 'EC 2.6.1.33';
UPDATE enzyme SET sys_name = 'UDP-4-amino-4,6-dideoxy-N-acetyl-alpha-D-glucosamine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.34';
UPDATE enzyme SET history = 'EC 2.6.1.34 created 1972, modified 2013' WHERE ec = 'EC 2.6.1.34';
UPDATE enzyme SET sys_name = 'Glycine:oxaloacetate aminotransferase' WHERE ec = 'EC 2.6.1.35';
UPDATE enzyme SET history = 'EC 2.6.1.35 created 1972' WHERE ec = 'EC 2.6.1.35';
UPDATE enzyme SET sys_name = 'L-lysine:2-oxoglutarate 6-aminotransferase' WHERE ec = 'EC 2.6.1.36';
UPDATE enzyme SET history = 'EC 2.6.1.36 created 1972, modified 2011' WHERE ec = 'EC 2.6.1.36';
UPDATE enzyme SET sys_name = '(2-aminoethyl)phosphonate:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.37';
UPDATE enzyme SET history = 'EC 2.6.1.37 created 1972, modified 1982, modified 2001' WHERE ec = 'EC 2.6.1.37';
UPDATE enzyme SET sys_name = 'L-histidine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.38';
UPDATE enzyme SET history = 'EC 2.6.1.38 created 1972' WHERE ec = 'EC 2.6.1.38';
UPDATE enzyme SET sys_name = 'L-2-aminoadipate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.39';
UPDATE enzyme SET history = 'EC 2.6.1.39 created 1972' WHERE ec = 'EC 2.6.1.39';
UPDATE enzyme SET sys_name = '(R)-3-amino-2-methylpropanoate:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.40';
UPDATE enzyme SET history = 'EC 2.6.1.40 created 1972 (EC 2.6.1.61 created 1982, incorporated 2004) modified 2004' WHERE ec = 'EC 2.6.1.40';
UPDATE enzyme SET sys_name = 'D-methionine:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.41';
UPDATE enzyme SET history = 'EC 2.6.1.41 created 1972, modified 1982' WHERE ec = 'EC 2.6.1.41';
UPDATE enzyme SET sys_name = 'Branched-chain-amino-acid:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.42';
UPDATE enzyme SET history = 'EC 2.6.1.42 created 1972' WHERE ec = 'EC 2.6.1.42';
UPDATE enzyme SET sys_name = '5-aminolevulinate:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.43';
UPDATE enzyme SET history = 'EC 2.6.1.43 created 1972' WHERE ec = 'EC 2.6.1.43';
UPDATE enzyme SET sys_name = 'L-alanine:glyoxylate aminotransferase' WHERE ec = 'EC 2.6.1.44';
UPDATE enzyme SET history = 'EC 2.6.1.44 created 1972, modified 1982' WHERE ec = 'EC 2.6.1.44';
UPDATE enzyme SET sys_name = 'L-serine:glyoxylate aminotransferase' WHERE ec = 'EC 2.6.1.45';
UPDATE enzyme SET history = 'EC 2.6.1.45 created 1972' WHERE ec = 'EC 2.6.1.45';
UPDATE enzyme SET sys_name = 'L-2,4-diaminobutanoate:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.46';
UPDATE enzyme SET history = 'EC 2.6.1.46 created 1972' WHERE ec = 'EC 2.6.1.46';
UPDATE enzyme SET sys_name = 'L-alanine:oxomalonate aminotransferase' WHERE ec = 'EC 2.6.1.47';
UPDATE enzyme SET history = 'EC 2.6.1.47 created 1972' WHERE ec = 'EC 2.6.1.47';
UPDATE enzyme SET sys_name = '5-aminopentanoate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.48';
UPDATE enzyme SET history = 'EC 2.6.1.48 created 1972' WHERE ec = 'EC 2.6.1.48';
UPDATE enzyme SET sys_name = '3,4-dihydroxy-L-phenylalanine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.49';
UPDATE enzyme SET history = 'EC 2.6.1.49 created 1972' WHERE ec = 'EC 2.6.1.49';
UPDATE enzyme SET sys_name = 'L-glutamine:2,4,6/3,5-pentahydroxycyclohexanone aminotransferase' WHERE ec = 'EC 2.6.1.50';
UPDATE enzyme SET history = 'EC 2.6.1.50 created 1972' WHERE ec = 'EC 2.6.1.50';
UPDATE enzyme SET sys_name = 'L-serine:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.51';
UPDATE enzyme SET history = 'EC 2.6.1.51 created 1972' WHERE ec = 'EC 2.6.1.51';
UPDATE enzyme SET sys_name = 'O-phospho-L-serine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.52';
UPDATE enzyme SET history = 'EC 2.6.1.52 created 1972, modified 2006' WHERE ec = 'EC 2.6.1.52';
UPDATE enzyme SET history = 'EC 2.6.1.53 created 1972, deleted 1976' WHERE ec = 'EC 2.6.1.53';
UPDATE enzyme SET sys_name = 'Pyridoxamine-5-phosphate:2-oxoglutarate aminotransferase (D-glutamate-forming)' WHERE ec = 'EC 2.6.1.54';
UPDATE enzyme SET history = 'EC 2.6.1.54 created 1976' WHERE ec = 'EC 2.6.1.54';
UPDATE enzyme SET sys_name = 'Taurine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.55';
UPDATE enzyme SET history = 'EC 2.6.1.55 created 1976, modified 2003' WHERE ec = 'EC 2.6.1.55';
UPDATE enzyme SET sys_name = '1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.56';
UPDATE enzyme SET history = 'EC 2.6.1.56 created 1976' WHERE ec = 'EC 2.6.1.56';
UPDATE enzyme SET sys_name = 'Aromatic-amino-acid:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.57';
UPDATE enzyme SET history = 'EC 2.6.1.57 created 1976' WHERE ec = 'EC 2.6.1.57';
UPDATE enzyme SET sys_name = 'L-phenylalanine:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.58';
UPDATE enzyme SET history = 'EC 2.6.1.58 created 1978' WHERE ec = 'EC 2.6.1.58';
UPDATE enzyme SET sys_name = 'dTDP-4-amino-4,6-dideoxy-alpha-D-galactose:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.59';
UPDATE enzyme SET history = 'EC 2.6.1.59 created 1978' WHERE ec = 'EC 2.6.1.59';
UPDATE enzyme SET sys_name = 'Aromatic-amino-acid:glyoxylate aminotransferase' WHERE ec = 'EC 2.6.1.60';
UPDATE enzyme SET history = 'EC 2.6.1.60 created 1978' WHERE ec = 'EC 2.6.1.60';
UPDATE enzyme SET history = 'EC 2.6.1.61 created 1982, deleted 2004' WHERE ec = 'EC 2.6.1.61';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:8-amino-7-oxononanoate aminotransferase' WHERE ec = 'EC 2.6.1.62';
UPDATE enzyme SET history = 'EC 2.6.1.62 created 1983' WHERE ec = 'EC 2.6.1.62';
UPDATE enzyme SET sys_name = 'L-kynurenine:glyoxylate aminotransferase (cyclizing)' WHERE ec = 'EC 2.6.1.63';
UPDATE enzyme SET history = 'EC 2.6.1.63 created 1983' WHERE ec = 'EC 2.6.1.63';
UPDATE enzyme SET sys_name = 'L-glutamine:phenylpyruvate aminotransferase' WHERE ec = 'EC 2.6.1.64';
UPDATE enzyme SET history = 'EC 2.6.1.64 created 1984' WHERE ec = 'EC 2.6.1.64';
UPDATE enzyme SET sys_name = '6-acetamido-3-aminohexanoate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.65';
UPDATE enzyme SET history = 'EC 2.6.1.65 created 1984' WHERE ec = 'EC 2.6.1.65';
UPDATE enzyme SET sys_name = 'L-valine:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.66';
UPDATE enzyme SET history = 'EC 2.6.1.66 created 1984' WHERE ec = 'EC 2.6.1.66';
UPDATE enzyme SET sys_name = 'L-2-aminohexanoate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.67';
UPDATE enzyme SET history = 'EC 2.6.1.67 created 1989' WHERE ec = 'EC 2.6.1.67';
UPDATE enzyme SET sys_name = 'L-ornithine:2-oxoglutarate-aminotransferase' WHERE ec = 'EC 2.6.1.68';
UPDATE enzyme SET history = 'EC 2.6.1.68 created 1989' WHERE ec = 'EC 2.6.1.68';
UPDATE enzyme SET history = 'EC 2.6.1.69 created 1989, deleted 2004' WHERE ec = 'EC 2.6.1.69';
UPDATE enzyme SET sys_name = 'L-aspartate:phenylpyruvate aminotransferase' WHERE ec = 'EC 2.6.1.70';
UPDATE enzyme SET history = 'EC 2.6.1.70 created 1989' WHERE ec = 'EC 2.6.1.70';
UPDATE enzyme SET sys_name = 'L-lysine:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.71';
UPDATE enzyme SET history = 'EC 2.6.1.71 created 1990, modified 2011' WHERE ec = 'EC 2.6.1.71';
UPDATE enzyme SET sys_name = 'D-4-hydroxyphenylglycine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.72';
UPDATE enzyme SET history = 'EC 2.6.1.72 created 1990' WHERE ec = 'EC 2.6.1.72';
UPDATE enzyme SET sys_name = 'L-methionine:glyoxylate aminotransferase' WHERE ec = 'EC 2.6.1.73';
UPDATE enzyme SET history = 'EC 2.6.1.73 created 1992' WHERE ec = 'EC 2.6.1.73';
UPDATE enzyme SET sys_name = 'Cephalosporin-C:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.74';
UPDATE enzyme SET history = 'EC 2.6.1.74 created 1992, modified 2005' WHERE ec = 'EC 2.6.1.74';
UPDATE enzyme SET sys_name = 'S-(4-bromophenyl)-L-cysteine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.75';
UPDATE enzyme SET history = 'EC 2.6.1.75 created 1992' WHERE ec = 'EC 2.6.1.75';
UPDATE enzyme SET sys_name = 'L-2,4-diaminobutanoate:2-oxoglutarate 4-aminotransferase' WHERE ec = 'EC 2.6.1.76';
UPDATE enzyme SET history = 'EC 2.6.1.76 created 2000, modified 2006' WHERE ec = 'EC 2.6.1.76';
UPDATE enzyme SET sys_name = 'Taurine:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.77';
UPDATE enzyme SET history = 'EC 2.6.1.77 created 2003' WHERE ec = 'EC 2.6.1.77';
UPDATE enzyme SET sys_name = 'L-arogenate:oxaloacetate aminotransferase' WHERE ec = 'EC 2.6.1.78';
UPDATE enzyme SET history = 'EC 2.6.1.78 created 2005' WHERE ec = 'EC 2.6.1.78';
UPDATE enzyme SET sys_name = 'L-arogenate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.79';
UPDATE enzyme SET history = 'EC 2.6.1.79 created 2005' WHERE ec = 'EC 2.6.1.79';
UPDATE enzyme SET sys_name = 'Nicotianamine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.80';
UPDATE enzyme SET history = 'EC 2.6.1.80 created 2005' WHERE ec = 'EC 2.6.1.80';
UPDATE enzyme SET sys_name = 'N(2)-succinyl-L-ornithine:2-oxoglutarate 5-aminotransferase' WHERE ec = 'EC 2.6.1.81';
UPDATE enzyme SET history = 'EC 2.6.1.81 created 2006' WHERE ec = 'EC 2.6.1.81';
UPDATE enzyme SET sys_name = 'Butane-1,4-diamine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.82';
UPDATE enzyme SET history = 'EC 2.6.1.82 created 2006' WHERE ec = 'EC 2.6.1.82';
UPDATE enzyme SET sys_name = 'LL-2,6-diaminoheptanedioate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.83';
UPDATE enzyme SET history = 'EC 2.6.1.83 created 2006' WHERE ec = 'EC 2.6.1.83';
UPDATE enzyme SET sys_name = 'L-arginine:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.84';
UPDATE enzyme SET history = 'EC 2.6.1.84 created 2007' WHERE ec = 'EC 2.6.1.84';
UPDATE enzyme SET sys_name = 'Chorismate:L-glutamine aminotransferase' WHERE ec = 'EC 2.6.1.85';
UPDATE enzyme SET history = 'EC 2.6.1.85 created 2003 as EC 6.3.5.8, transferred 2007 to EC 2.6.1.85' WHERE ec = 'EC 2.6.1.85';
UPDATE enzyme SET sys_name = '(2S)-2-amino-4-deoxychorismate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.86';
UPDATE enzyme SET history = 'EC 2.6.1.86 created 2008' WHERE ec = 'EC 2.6.1.86';
UPDATE enzyme SET sys_name = 'UDP-4-amino-4-deoxy-beta-L-arabinose:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.87';
UPDATE enzyme SET history = 'EC 2.6.1.87 created 2010' WHERE ec = 'EC 2.6.1.87';
UPDATE enzyme SET sys_name = 'L-methionine:2-oxo-acid aminotransferase' WHERE ec = 'EC 2.6.1.88';
UPDATE enzyme SET history = 'EC 2.6.1.88 created 2011' WHERE ec = 'EC 2.6.1.88';
UPDATE enzyme SET sys_name = 'dTDP-3-amino-3,6-dideoxy-alpha-D-glucopyranose:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.89';
UPDATE enzyme SET history = 'EC 2.6.1.89 created 2011' WHERE ec = 'EC 2.6.1.89';
UPDATE enzyme SET sys_name = 'dTDP-3-amino-3,6-dideoxy-alpha-D-galactopyranose:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.90';
UPDATE enzyme SET history = 'EC 2.6.1.90 created 2011' WHERE ec = 'EC 2.6.1.90';
UPDATE enzyme SET history = 'EC 2.6.1.91 created 2011, deleted 2013' WHERE ec = 'EC 2.6.1.91';
UPDATE enzyme SET sys_name = 'UDP-4-amino-4,6-dideoxy-N-acetyl-beta-L-altrosamine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.92';
UPDATE enzyme SET history = 'EC 2.6.1.92 created 2011' WHERE ec = 'EC 2.6.1.92';
UPDATE enzyme SET sys_name = 'Neamine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.93';
UPDATE enzyme SET history = 'EC 2.6.1.93 created 2012' WHERE ec = 'EC 2.6.1.93';
UPDATE enzyme SET sys_name = '2-deamino-2-hydroxyneamine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.94';
UPDATE enzyme SET history = 'EC 2.6.1.94 created 2012' WHERE ec = 'EC 2.6.1.94';
UPDATE enzyme SET sys_name = '2-oxoglutarate:neomycin C aminotransferase' WHERE ec = 'EC 2.6.1.95';
UPDATE enzyme SET history = 'EC 2.6.1.95 created 2012' WHERE ec = 'EC 2.6.1.95';
UPDATE enzyme SET sys_name = '4-aminobutanoate:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.96';
UPDATE enzyme SET history = 'EC 2.6.1.96 created 2012' WHERE ec = 'EC 2.6.1.96';
UPDATE enzyme SET sys_name = 'L-glutamine:7-cyano-7-carbaguanine aminotransferase' WHERE ec = 'EC 2.6.1.97';
UPDATE enzyme SET history = 'EC 2.6.1.97 created 2012' WHERE ec = 'EC 2.6.1.97';
UPDATE enzyme SET sys_name = 'UDP-2-acetamido-3-amino-2,3-dideoxy-alpha-D-glucuronate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.98';
UPDATE enzyme SET history = 'EC 2.6.1.98 created 2012' WHERE ec = 'EC 2.6.1.98';
UPDATE enzyme SET sys_name = 'L-tryptophan:pyruvate aminotransferase' WHERE ec = 'EC 2.6.1.99';
UPDATE enzyme SET history = 'EC 2.6.1.99 created 2012' WHERE ec = 'EC 2.6.1.99';
UPDATE enzyme SET sys_name = 'L-glutamine:2-deoxy-scyllo-inosose aminotransferase' WHERE ec = 'EC 2.6.1.100';
UPDATE enzyme SET history = 'EC 2.6.1.100 created 2013' WHERE ec = 'EC 2.6.1.100';
UPDATE enzyme SET sys_name = 'L-glutamine:5-amino-2,3,4-trihydroxycyclohexanone aminotransferase' WHERE ec = 'EC 2.6.1.101';
UPDATE enzyme SET history = 'EC 2.6.1.101 created 2013' WHERE ec = 'EC 2.6.1.101';
UPDATE enzyme SET sys_name = 'GDP-4-amino-4,6-dideoxy-alpha-D-mannose:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.102';
UPDATE enzyme SET history = 'EC 2.6.1.102 created 2013' WHERE ec = 'EC 2.6.1.102';
UPDATE enzyme SET sys_name = '(S)-3,5-dihydroxyphenylglycine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.103';
UPDATE enzyme SET history = 'EC 2.6.1.103 created 2013' WHERE ec = 'EC 2.6.1.103';
UPDATE enzyme SET sys_name = 'Kanosamine 6-phosphate:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.104';
UPDATE enzyme SET history = 'EC 2.6.1.104 created 2014' WHERE ec = 'EC 2.6.1.104';
UPDATE enzyme SET sys_name = 'L-lysine:8-amino-7-oxononanoate aminotransferase' WHERE ec = 'EC 2.6.1.105';
UPDATE enzyme SET history = 'EC 2.6.1.105 created 2014' WHERE ec = 'EC 2.6.1.105';
UPDATE enzyme SET sys_name = 'dTDP-3-amino-3,4,6-trideoxy-alpha-D-glucose:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.106';
UPDATE enzyme SET history = 'EC 2.6.1.106 created 2014' WHERE ec = 'EC 2.6.1.106';
UPDATE enzyme SET sys_name = '(2S,3S)-3-methylphenylalanine:2-oxoglutarate aminotransferase' WHERE ec = 'EC 2.6.1.107';
UPDATE enzyme SET history = 'EC 2.6.1.107 created 2014' WHERE ec = 'EC 2.6.1.107';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.6.1.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.6.1.n2';
UPDATE enzyme SET history = 'EC 2.6.2.1 created 1961, deleted 1965' WHERE ec = 'EC 2.6.2.1';
UPDATE enzyme SET sys_name = 'Pyruvate-oxime:acetone oximinotransferase' WHERE ec = 'EC 2.6.3.1';
UPDATE enzyme SET history = 'EC 2.6.3.1 created 1961' WHERE ec = 'EC 2.6.3.1';
UPDATE enzyme SET sys_name = 'dATP(dGTP):depurinated-DNA purine transferase' WHERE ec = 'EC 2.6.99.1';
UPDATE enzyme SET history = 'EC 2.6.99.1 created 1984' WHERE ec = 'EC 2.6.99.1';
UPDATE enzyme SET sys_name = '1-deoxy-D-xylulose-5-phosphate:3-amino-2-oxopropyl phosphate 3-amino-2-oxopropyltransferase (phosphate-hydrolyzing; cyclizing)' WHERE ec = 'EC 2.6.99.2';
UPDATE enzyme SET history = 'EC 2.6.99.2 created 2006' WHERE ec = 'EC 2.6.99.2';
UPDATE enzyme SET sys_name = 'O-acetyl-L-serine:hydroxyurea 2-amino-2-carboxyethyltransferase' WHERE ec = 'EC 2.6.99.3';
UPDATE enzyme SET history = 'EC 2.6.99.3 created 2013' WHERE ec = 'EC 2.6.99.3';
UPDATE enzyme SET sys_name = 'L-threonylcarbamoyladenylate:adenine(37) in tRNA N(6)-L-threonylcarbamoyltransferase' WHERE ec = 'EC 2.6.99.4';
UPDATE enzyme SET history = 'EC 2.6.99.4 created 2014' WHERE ec = 'EC 2.6.99.4';
UPDATE enzyme SET sys_name = 'ATP:D-hexose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.1';
UPDATE enzyme SET history = 'EC 2.7.1.1 created 1961' WHERE ec = 'EC 2.7.1.1';
UPDATE enzyme SET sys_name = 'ATP:D-glucose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.2';
UPDATE enzyme SET history = 'EC 2.7.1.2 created 1961' WHERE ec = 'EC 2.7.1.2';
UPDATE enzyme SET sys_name = 'ATP:D-fructose 1-phosphotransferase' WHERE ec = 'EC 2.7.1.3';
UPDATE enzyme SET history = 'EC 2.7.1.3 created 1961' WHERE ec = 'EC 2.7.1.3';
UPDATE enzyme SET sys_name = 'ATP:D-fructose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.4';
UPDATE enzyme SET history = 'EC 2.7.1.4 created 1961' WHERE ec = 'EC 2.7.1.4';
UPDATE enzyme SET sys_name = 'ATP:L-rhamnulose 1-phosphotransferase' WHERE ec = 'EC 2.7.1.5';
UPDATE enzyme SET history = 'EC 2.7.1.5 created 1961' WHERE ec = 'EC 2.7.1.5';
UPDATE enzyme SET sys_name = 'ATP:alpha-D-galactose 1-phosphotransferase' WHERE ec = 'EC 2.7.1.6';
UPDATE enzyme SET history = 'EC 2.7.1.6 created 1961' WHERE ec = 'EC 2.7.1.6';
UPDATE enzyme SET sys_name = 'ATP:D-mannose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.7';
UPDATE enzyme SET history = 'EC 2.7.1.7 created 1961' WHERE ec = 'EC 2.7.1.7';
UPDATE enzyme SET sys_name = 'ATP:D-glucosamine phosphotransferase' WHERE ec = 'EC 2.7.1.8';
UPDATE enzyme SET history = 'EC 2.7.1.8 created 1961' WHERE ec = 'EC 2.7.1.8';
UPDATE enzyme SET history = 'EC 2.7.1.9 created 1961, deleted 1965' WHERE ec = 'EC 2.7.1.9';
UPDATE enzyme SET sys_name = 'ATP:alpha-D-glucose-1-phosphate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.10';
UPDATE enzyme SET history = 'EC 2.7.1.10 created 1961' WHERE ec = 'EC 2.7.1.10';
UPDATE enzyme SET sys_name = 'ATP:D-fructose-6-phosphate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.11';
UPDATE enzyme SET history = 'EC 2.7.1.11 created 1961' WHERE ec = 'EC 2.7.1.11';
UPDATE enzyme SET sys_name = 'ATP:D-gluconate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.12';
UPDATE enzyme SET history = 'EC 2.7.1.12 created 1961' WHERE ec = 'EC 2.7.1.12';
UPDATE enzyme SET sys_name = 'ATP:2-dehydro-D-gluconate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.13';
UPDATE enzyme SET history = 'EC 2.7.1.13 created 1961' WHERE ec = 'EC 2.7.1.13';
UPDATE enzyme SET sys_name = 'ATP:sedoheptulose 7-phosphotransferase' WHERE ec = 'EC 2.7.1.14';
UPDATE enzyme SET history = 'EC 2.7.1.14 created 1961' WHERE ec = 'EC 2.7.1.14';
UPDATE enzyme SET sys_name = 'ATP:D-ribose 5-phosphotransferase' WHERE ec = 'EC 2.7.1.15';
UPDATE enzyme SET history = 'EC 2.7.1.15 created 1961' WHERE ec = 'EC 2.7.1.15';
UPDATE enzyme SET sys_name = 'ATP:L(or D)-ribulose 5-phosphotransferase' WHERE ec = 'EC 2.7.1.16';
UPDATE enzyme SET history = 'EC 2.7.1.16 created 1961' WHERE ec = 'EC 2.7.1.16';
UPDATE enzyme SET sys_name = 'ATP:D-xylulose 5-phosphotransferase' WHERE ec = 'EC 2.7.1.17';
UPDATE enzyme SET history = 'EC 2.7.1.17 created 1961' WHERE ec = 'EC 2.7.1.17';
UPDATE enzyme SET sys_name = 'ATP:D-ribose-5-phosphate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.18';
UPDATE enzyme SET history = 'EC 2.7.1.18 created 1961' WHERE ec = 'EC 2.7.1.18';
UPDATE enzyme SET sys_name = 'ATP:D-ribulose-5-phosphate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.19';
UPDATE enzyme SET history = 'EC 2.7.1.19 created 1961' WHERE ec = 'EC 2.7.1.19';
UPDATE enzyme SET sys_name = 'ATP:adenosine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.20';
UPDATE enzyme SET history = 'EC 2.7.1.20 created 1961' WHERE ec = 'EC 2.7.1.20';
UPDATE enzyme SET sys_name = 'ATP:thymidine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.21';
UPDATE enzyme SET history = 'EC 2.7.1.21 created 1961, deleted 1972, reinstated 1976 (EC 2.7.1.75 created 1972, incorporated 1976)' WHERE ec = 'EC 2.7.1.21';
UPDATE enzyme SET sys_name = 'ATP:N-ribosylnicotinamide 5-phosphotransferase' WHERE ec = 'EC 2.7.1.22';
UPDATE enzyme SET history = 'EC 2.7.1.22 created 1961' WHERE ec = 'EC 2.7.1.22';
UPDATE enzyme SET sys_name = 'ATP:NAD(+) 2-phosphotransferase' WHERE ec = 'EC 2.7.1.23';
UPDATE enzyme SET history = 'EC 2.7.1.23 created 1961' WHERE ec = 'EC 2.7.1.23';
UPDATE enzyme SET sys_name = 'ATP:3-dephospho-CoA 3-phosphotransferase' WHERE ec = 'EC 2.7.1.24';
UPDATE enzyme SET history = 'EC 2.7.1.24 created 1961' WHERE ec = 'EC 2.7.1.24';
UPDATE enzyme SET sys_name = 'ATP:adenylyl-sulfate 3-phosphotransferase' WHERE ec = 'EC 2.7.1.25';
UPDATE enzyme SET history = 'EC 2.7.1.25 created 1961, modified 1999' WHERE ec = 'EC 2.7.1.25';
UPDATE enzyme SET sys_name = 'ATP:riboflavin 5-phosphotransferase' WHERE ec = 'EC 2.7.1.26';
UPDATE enzyme SET history = 'EC 2.7.1.26 created 1961, modified 2007' WHERE ec = 'EC 2.7.1.26';
UPDATE enzyme SET sys_name = 'ATP:erythritol 4-phosphotransferase' WHERE ec = 'EC 2.7.1.27';
UPDATE enzyme SET history = 'EC 2.7.1.27 created 1961' WHERE ec = 'EC 2.7.1.27';
UPDATE enzyme SET sys_name = 'ATP:D-glyceraldehyde 3-phosphotransferase' WHERE ec = 'EC 2.7.1.28';
UPDATE enzyme SET history = 'EC 2.7.1.28 created 1961' WHERE ec = 'EC 2.7.1.28';
UPDATE enzyme SET sys_name = 'ATP:glycerone phosphotransferase' WHERE ec = 'EC 2.7.1.29';
UPDATE enzyme SET history = 'EC 2.7.1.29 created 1961' WHERE ec = 'EC 2.7.1.29';
UPDATE enzyme SET sys_name = 'ATP:glycerol 3-phosphotransferase' WHERE ec = 'EC 2.7.1.30';
UPDATE enzyme SET history = 'EC 2.7.1.30 created 1961' WHERE ec = 'EC 2.7.1.30';
UPDATE enzyme SET sys_name = 'ATP:D-glycerate 3-phosphotransferase' WHERE ec = 'EC 2.7.1.31';
UPDATE enzyme SET history = 'EC 2.7.1.31 created 1961, modified 2012' WHERE ec = 'EC 2.7.1.31';
UPDATE enzyme SET sys_name = 'ATP:choline phosphotransferase' WHERE ec = 'EC 2.7.1.32';
UPDATE enzyme SET history = 'EC 2.7.1.32 created 1961' WHERE ec = 'EC 2.7.1.32';
UPDATE enzyme SET sys_name = 'ATP:(R)-pantothenate 4-phosphotransferase' WHERE ec = 'EC 2.7.1.33';
UPDATE enzyme SET history = 'EC 2.7.1.33 created 1961' WHERE ec = 'EC 2.7.1.33';
UPDATE enzyme SET sys_name = 'ATP:pantetheine 4-phosphotransferase' WHERE ec = 'EC 2.7.1.34';
UPDATE enzyme SET history = 'EC 2.7.1.34 created 1961' WHERE ec = 'EC 2.7.1.34';
UPDATE enzyme SET sys_name = 'ATP:pyridoxal 5-phosphotransferase' WHERE ec = 'EC 2.7.1.35';
UPDATE enzyme SET history = 'EC 2.7.1.35 created 1961' WHERE ec = 'EC 2.7.1.35';
UPDATE enzyme SET sys_name = 'ATP:(R)-mevalonate 5-phosphotransferase' WHERE ec = 'EC 2.7.1.36';
UPDATE enzyme SET history = 'EC 2.7.1.36 created 1961' WHERE ec = 'EC 2.7.1.36';
UPDATE enzyme SET history = 'EC 2.7.1.37 created 1961, deleted 2005' WHERE ec = 'EC 2.7.1.37';
UPDATE enzyme SET history = 'EC 2.7.1.38 created 1961, deleted 2005' WHERE ec = 'EC 2.7.1.38';
UPDATE enzyme SET sys_name = 'ATP:L-homoserine O-phosphotransferase' WHERE ec = 'EC 2.7.1.39';
UPDATE enzyme SET history = 'EC 2.7.1.39 created 1961' WHERE ec = 'EC 2.7.1.39';
UPDATE enzyme SET sys_name = 'ATP:pyruvate 2-O-phosphotransferase' WHERE ec = 'EC 2.7.1.40';
UPDATE enzyme SET history = 'EC 2.7.1.40 created 1961' WHERE ec = 'EC 2.7.1.40';
UPDATE enzyme SET sys_name = 'D-glucose-1-phosphate:D-glucose-1-phosphate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.41';
UPDATE enzyme SET history = 'EC 2.7.1.41 created 1961' WHERE ec = 'EC 2.7.1.41';
UPDATE enzyme SET sys_name = 'Alpha-D-glucose-1-phosphate:riboflavin 5-phosphotransferase' WHERE ec = 'EC 2.7.1.42';
UPDATE enzyme SET history = 'EC 2.7.1.42 created 1961' WHERE ec = 'EC 2.7.1.42';
UPDATE enzyme SET sys_name = 'ATP:D-glucuronate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.43';
UPDATE enzyme SET history = 'EC 2.7.1.43 created 1965' WHERE ec = 'EC 2.7.1.43';
UPDATE enzyme SET sys_name = 'ATP:D-galacturonate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.44';
UPDATE enzyme SET history = 'EC 2.7.1.44 created 1965' WHERE ec = 'EC 2.7.1.44';
UPDATE enzyme SET sys_name = 'ATP:2-dehydro-3-deoxy-D-gluconate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.45';
UPDATE enzyme SET history = 'EC 2.7.1.45 created 1965, modified 1976' WHERE ec = 'EC 2.7.1.45';
UPDATE enzyme SET sys_name = 'ATP:L-arabinose 1-phosphotransferase' WHERE ec = 'EC 2.7.1.46';
UPDATE enzyme SET history = 'EC 2.7.1.46 created 1965' WHERE ec = 'EC 2.7.1.46';
UPDATE enzyme SET sys_name = 'ATP:D-ribulose 5-phosphotransferase' WHERE ec = 'EC 2.7.1.47';
UPDATE enzyme SET history = 'EC 2.7.1.47 created 1965' WHERE ec = 'EC 2.7.1.47';
UPDATE enzyme SET sys_name = 'ATP:uridine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.48';
UPDATE enzyme SET history = 'EC 2.7.1.48 created 1965' WHERE ec = 'EC 2.7.1.48';
UPDATE enzyme SET sys_name = 'ATP:4-amino-5-hydroxymethyl-2-methylpyrimidine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.49';
UPDATE enzyme SET history = 'EC 2.7.1.49 created 1965' WHERE ec = 'EC 2.7.1.49';
UPDATE enzyme SET sys_name = 'ATP:4-methyl-5-(2-hydroxyethyl)thiazole 2-phosphotransferase' WHERE ec = 'EC 2.7.1.50';
UPDATE enzyme SET history = 'EC 2.7.1.50 created 1965' WHERE ec = 'EC 2.7.1.50';
UPDATE enzyme SET sys_name = 'ATP:L-fuculose 1-phosphotransferase' WHERE ec = 'EC 2.7.1.51';
UPDATE enzyme SET history = 'EC 2.7.1.51 created 1965' WHERE ec = 'EC 2.7.1.51';
UPDATE enzyme SET sys_name = 'ATP:beta-L-fucose 1-phosphotransferase' WHERE ec = 'EC 2.7.1.52';
UPDATE enzyme SET history = 'EC 2.7.1.52 created 1972, modified 2004' WHERE ec = 'EC 2.7.1.52';
UPDATE enzyme SET sys_name = 'ATP:L-xylulose 5-phosphotransferase' WHERE ec = 'EC 2.7.1.53';
UPDATE enzyme SET history = 'EC 2.7.1.53 created 1972' WHERE ec = 'EC 2.7.1.53';
UPDATE enzyme SET sys_name = 'ATP:D-arabinose 5-phosphotransferase' WHERE ec = 'EC 2.7.1.54';
UPDATE enzyme SET history = 'EC 2.7.1.54 created 1972' WHERE ec = 'EC 2.7.1.54';
UPDATE enzyme SET sys_name = 'ATP:D-allose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.55';
UPDATE enzyme SET history = 'EC 2.7.1.55 created 1972' WHERE ec = 'EC 2.7.1.55';
UPDATE enzyme SET sys_name = 'ATP:D-fructose-phosphate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.56';
UPDATE enzyme SET history = 'EC 2.7.1.56 created 1972' WHERE ec = 'EC 2.7.1.56';
UPDATE enzyme SET history = 'EC 2.7.1.57 created 1972, deleted 1984' WHERE ec = 'EC 2.7.1.57';
UPDATE enzyme SET sys_name = 'ATP:2-dehydro-3-deoxy-D-galactonate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.58';
UPDATE enzyme SET history = 'EC 2.7.1.58 created 1972' WHERE ec = 'EC 2.7.1.58';
UPDATE enzyme SET sys_name = 'ATP:N-acetyl-D-glucosamine 6-phosphotransferase' WHERE ec = 'EC 2.7.1.59';
UPDATE enzyme SET history = 'EC 2.7.1.59 created 1972' WHERE ec = 'EC 2.7.1.59';
UPDATE enzyme SET sys_name = 'ATP:N-acyl-D-mannosamine 6-phosphotransferase' WHERE ec = 'EC 2.7.1.60';
UPDATE enzyme SET history = 'EC 2.7.1.60 created 1972' WHERE ec = 'EC 2.7.1.60';
UPDATE enzyme SET sys_name = 'Acyl-phosphate:D-hexose phosphotransferase' WHERE ec = 'EC 2.7.1.61';
UPDATE enzyme SET history = 'EC 2.7.1.61 created 1972, modified 2011' WHERE ec = 'EC 2.7.1.61';
UPDATE enzyme SET sys_name = 'Phosphoramidate:D-hexose 1-phosphotransferase' WHERE ec = 'EC 2.7.1.62';
UPDATE enzyme SET history = 'EC 2.7.1.62 created 1972' WHERE ec = 'EC 2.7.1.62';
UPDATE enzyme SET sys_name = 'Polyphosphate:D-glucose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.63';
UPDATE enzyme SET history = 'EC 2.7.1.63 created 1972' WHERE ec = 'EC 2.7.1.63';
UPDATE enzyme SET sys_name = 'ATP:myo-inositol 1-phosphotransferase' WHERE ec = 'EC 2.7.1.64';
UPDATE enzyme SET history = 'EC 2.7.1.64 created 1972, modified 2001' WHERE ec = 'EC 2.7.1.64';
UPDATE enzyme SET sys_name = 'ATP:1-amino-1-deoxy-scyllo-inositol 4-phosphotransferase' WHERE ec = 'EC 2.7.1.65';
UPDATE enzyme SET history = 'EC 2.7.1.65 created 1972, modified 1976' WHERE ec = 'EC 2.7.1.65';
UPDATE enzyme SET sys_name = 'ATP:undecaprenol phosphotransferase' WHERE ec = 'EC 2.7.1.66';
UPDATE enzyme SET history = 'EC 2.7.1.66 created 1972' WHERE ec = 'EC 2.7.1.66';
UPDATE enzyme SET sys_name = 'ATP:1-phosphatidyl-1D-myo-inositol 4-phosphotransferase' WHERE ec = 'EC 2.7.1.67';
UPDATE enzyme SET history = 'EC 2.7.1.67 created 1972, modified 1982, modified 2002' WHERE ec = 'EC 2.7.1.67';
UPDATE enzyme SET sys_name = 'ATP:1-phosphatidyl-1D-myo-inositol-4-phosphate 5-phosphotransferase' WHERE ec = 'EC 2.7.1.68';
UPDATE enzyme SET history = 'EC 2.7.1.68 created 1972, modified 1980, modified 1982, modified 2002' WHERE ec = 'EC 2.7.1.68';
UPDATE enzyme SET sys_name = 'Protein-N(pros)-phospho-L-histidine:sugar N(pros)-phosphotransferase' WHERE ec = 'EC 2.7.1.69';
UPDATE enzyme SET history = 'EC 2.7.1.69 created 1972, modified 2000' WHERE ec = 'EC 2.7.1.69';
UPDATE enzyme SET history = 'EC 2.7.1.70 created 1972, deleted 2004' WHERE ec = 'EC 2.7.1.70';
UPDATE enzyme SET sys_name = 'ATP:shikimate 3-phosphotransferase' WHERE ec = 'EC 2.7.1.71';
UPDATE enzyme SET history = 'EC 2.7.1.71 created 1972' WHERE ec = 'EC 2.7.1.71';
UPDATE enzyme SET sys_name = 'ATP:streptomycin 6-phosphotransferase' WHERE ec = 'EC 2.7.1.72';
UPDATE enzyme SET history = 'EC 2.7.1.72 created 1972, modified 1976' WHERE ec = 'EC 2.7.1.72';
UPDATE enzyme SET sys_name = 'ATP:inosine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.73';
UPDATE enzyme SET history = 'EC 2.7.1.73 created 1972' WHERE ec = 'EC 2.7.1.73';
UPDATE enzyme SET sys_name = 'NTP:deoxycytidine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.74';
UPDATE enzyme SET history = 'EC 2.7.1.74 created 1972' WHERE ec = 'EC 2.7.1.74';
UPDATE enzyme SET history = 'EC 2.7.1.75 created 1972, deleted 1976' WHERE ec = 'EC 2.7.1.75';
UPDATE enzyme SET sys_name = 'ATP:deoxyadenosine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.76';
UPDATE enzyme SET history = 'EC 2.7.1.76 created 1972' WHERE ec = 'EC 2.7.1.76';
UPDATE enzyme SET sys_name = 'Nucleotide:nucleoside 5-phosphotransferase' WHERE ec = 'EC 2.7.1.77';
UPDATE enzyme SET history = 'EC 2.7.1.77 created 1972' WHERE ec = 'EC 2.7.1.77';
UPDATE enzyme SET sys_name = 'ATP:5-dephosphopolynucleotide 5-phosphotransferase' WHERE ec = 'EC 2.7.1.78';
UPDATE enzyme SET history = 'EC 2.7.1.78 created 1972' WHERE ec = 'EC 2.7.1.78';
UPDATE enzyme SET sys_name = 'Diphosphate:glycerol 1-phosphotransferase' WHERE ec = 'EC 2.7.1.79';
UPDATE enzyme SET history = 'EC 2.7.1.79 created 1972' WHERE ec = 'EC 2.7.1.79';
UPDATE enzyme SET sys_name = 'Diphosphate:L-serine O-phosphotransferase' WHERE ec = 'EC 2.7.1.80';
UPDATE enzyme SET history = 'EC 2.7.1.80 created 1972' WHERE ec = 'EC 2.7.1.80';
UPDATE enzyme SET sys_name = 'GTP:5-hydroxy-L-lysine O-phosphotransferase' WHERE ec = 'EC 2.7.1.81';
UPDATE enzyme SET history = 'EC 2.7.1.81 created 1972' WHERE ec = 'EC 2.7.1.81';
UPDATE enzyme SET sys_name = 'ATP:ethanolamine O-phosphotransferase' WHERE ec = 'EC 2.7.1.82';
UPDATE enzyme SET history = 'EC 2.7.1.82 created 1976' WHERE ec = 'EC 2.7.1.82';
UPDATE enzyme SET sys_name = 'ATP:pseudouridine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.83';
UPDATE enzyme SET history = 'EC 2.7.1.83 created 1976' WHERE ec = 'EC 2.7.1.83';
UPDATE enzyme SET sys_name = 'ATP:O-alkylglycerone phosphotransferase' WHERE ec = 'EC 2.7.1.84';
UPDATE enzyme SET history = 'EC 2.7.1.84 created 1976' WHERE ec = 'EC 2.7.1.84';
UPDATE enzyme SET sys_name = 'ATP:cellobiose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.85';
UPDATE enzyme SET history = 'EC 2.7.1.85 created 1976' WHERE ec = 'EC 2.7.1.85';
UPDATE enzyme SET sys_name = 'ATP:NADH 2-phosphotransferase' WHERE ec = 'EC 2.7.1.86';
UPDATE enzyme SET history = 'EC 2.7.1.86 created 1976 (EC 2.7.1.96 created 1978, incorporated 1978)' WHERE ec = 'EC 2.7.1.86';
UPDATE enzyme SET sys_name = 'ATP:streptomycin 3-phosphotransferase' WHERE ec = 'EC 2.7.1.87';
UPDATE enzyme SET history = 'EC 2.7.1.87 created 1976' WHERE ec = 'EC 2.7.1.87';
UPDATE enzyme SET sys_name = 'ATP:dihydrostreptomycin-6-phosphate 3-alpha-phosphotransferase' WHERE ec = 'EC 2.7.1.88';
UPDATE enzyme SET history = 'EC 2.7.1.88 created 1976' WHERE ec = 'EC 2.7.1.88';
UPDATE enzyme SET sys_name = 'ATP:thiamine phosphotransferase' WHERE ec = 'EC 2.7.1.89';
UPDATE enzyme SET history = 'EC 2.7.1.89 created 1976' WHERE ec = 'EC 2.7.1.89';
UPDATE enzyme SET sys_name = 'Diphosphate:D-fructose-6-phosphate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.90';
UPDATE enzyme SET history = 'EC 2.7.1.90 created 1976' WHERE ec = 'EC 2.7.1.90';
UPDATE enzyme SET sys_name = 'ATP:sphinganine 1-phosphotransferase' WHERE ec = 'EC 2.7.1.91';
UPDATE enzyme SET history = 'EC 2.7.1.91 created 1976, modified 1980' WHERE ec = 'EC 2.7.1.91';
UPDATE enzyme SET sys_name = 'ATP:5-dehydro-2-deoxy-D-gluconate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.92';
UPDATE enzyme SET history = 'EC 2.7.1.92 created 1976' WHERE ec = 'EC 2.7.1.92';
UPDATE enzyme SET sys_name = 'ATP:1-O-alkyl-sn-glycerol 3-phosphotransferase' WHERE ec = 'EC 2.7.1.93';
UPDATE enzyme SET history = 'EC 2.7.1.93 created 1976' WHERE ec = 'EC 2.7.1.93';
UPDATE enzyme SET sys_name = 'ATP:acylglycerol 3-phosphotransferase' WHERE ec = 'EC 2.7.1.94';
UPDATE enzyme SET history = 'EC 2.7.1.94 created 1976' WHERE ec = 'EC 2.7.1.94';
UPDATE enzyme SET sys_name = 'ATP:kanamycin 3-O-phosphotransferase' WHERE ec = 'EC 2.7.1.95';
UPDATE enzyme SET history = 'EC 2.7.1.95 created 1976' WHERE ec = 'EC 2.7.1.95';
UPDATE enzyme SET history = 'EC 2.7.1.96 created 1978, deleted 1978' WHERE ec = 'EC 2.7.1.96';
UPDATE enzyme SET history = 'EC 2.7.1.97 created 1978, deleted 1992' WHERE ec = 'EC 2.7.1.97';
UPDATE enzyme SET history = 'EC 2.7.1.98 created 1978, deleted 1984' WHERE ec = 'EC 2.7.1.98';
UPDATE enzyme SET history = 'EC 2.7.1.99 created 1978, deleted 2005' WHERE ec = 'EC 2.7.1.99';
UPDATE enzyme SET sys_name = 'ATP:S-methyl-5-thio-D-ribose 1-phosphotransferase' WHERE ec = 'EC 2.7.1.100';
UPDATE enzyme SET history = 'EC 2.7.1.100 created 1980' WHERE ec = 'EC 2.7.1.100';
UPDATE enzyme SET sys_name = 'ATP:D-tagatose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.101';
UPDATE enzyme SET history = 'EC 2.7.1.101 created 1983' WHERE ec = 'EC 2.7.1.101';
UPDATE enzyme SET sys_name = 'ATP:D-hamamelose 2-phosphotransferase' WHERE ec = 'EC 2.7.1.102';
UPDATE enzyme SET history = 'EC 2.7.1.102 created 1983' WHERE ec = 'EC 2.7.1.102';
UPDATE enzyme SET sys_name = 'ATP:viomycin O-phosphotransferase' WHERE ec = 'EC 2.7.1.103';
UPDATE enzyme SET history = 'EC 2.7.1.103 created 1983' WHERE ec = 'EC 2.7.1.103';
UPDATE enzyme SET history = 'EC 2.7.1.104 created 1983, deleted 2005' WHERE ec = 'EC 2.7.1.104';
UPDATE enzyme SET sys_name = 'ATP:beta-D-fructose-6-phosphate 2-phosphotransferase' WHERE ec = 'EC 2.7.1.105';
UPDATE enzyme SET history = 'EC 2.7.1.105 created 1984' WHERE ec = 'EC 2.7.1.105';
UPDATE enzyme SET sys_name = '3-phospho-D-glyceroyl-phosphate:alpha-D-glucose-1-phosphate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.106';
UPDATE enzyme SET history = 'EC 2.7.1.106 created 1984' WHERE ec = 'EC 2.7.1.106';
UPDATE enzyme SET sys_name = 'ATP:1,2-diacyl-sn-glycerol 3-phosphotransferase' WHERE ec = 'EC 2.7.1.107';
UPDATE enzyme SET history = 'EC 2.7.1.107 created 1984, modified 2013' WHERE ec = 'EC 2.7.1.107';
UPDATE enzyme SET sys_name = 'CTP:dolichol O-phosphotransferase' WHERE ec = 'EC 2.7.1.108';
UPDATE enzyme SET history = 'EC 2.7.1.108 created 1984' WHERE ec = 'EC 2.7.1.108';
UPDATE enzyme SET history = 'EC 2.7.1.109 created 1984, deleted 2005' WHERE ec = 'EC 2.7.1.109';
UPDATE enzyme SET history = 'EC 2.7.1.110 created 1984, deleted 2005' WHERE ec = 'EC 2.7.1.110';
UPDATE enzyme SET history = 'EC 2.7.1.111 created 1984, deleted 1992' WHERE ec = 'EC 2.7.1.111';
UPDATE enzyme SET history = 'EC 2.7.1.112 created 1984, deleted 2005' WHERE ec = 'EC 2.7.1.112';
UPDATE enzyme SET sys_name = 'ATP:deoxyguanosine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.113';
UPDATE enzyme SET history = 'EC 2.7.1.113 created 1984' WHERE ec = 'EC 2.7.1.113';
UPDATE enzyme SET sys_name = 'AMP:thymidine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.114';
UPDATE enzyme SET history = 'EC 2.7.1.114 created 1984' WHERE ec = 'EC 2.7.1.114';
UPDATE enzyme SET history = 'EC 2.7.1.115 created 1986, deleted 2005' WHERE ec = 'EC 2.7.1.115';
UPDATE enzyme SET history = 'EC 2.7.1.116 created 1986, deleted 2005' WHERE ec = 'EC 2.7.1.116';
UPDATE enzyme SET history = 'EC 2.7.1.117 created 1986, deleted 2005' WHERE ec = 'EC 2.7.1.117';
UPDATE enzyme SET sys_name = 'ADP:thymidine 5-phosphotransferase' WHERE ec = 'EC 2.7.1.118';
UPDATE enzyme SET history = 'EC 2.7.1.118 created 1986' WHERE ec = 'EC 2.7.1.118';
UPDATE enzyme SET sys_name = 'ATP:hygromycin-B 7-O-phosphotransferase' WHERE ec = 'EC 2.7.1.119';
UPDATE enzyme SET history = 'EC 2.7.1.119 created 1989, modified 2009, modified 2011' WHERE ec = 'EC 2.7.1.119';
UPDATE enzyme SET history = 'EC 2.7.1.120 created 1989, modified 1990, deleted 2005' WHERE ec = 'EC 2.7.1.120';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:glycerone phosphotransferase' WHERE ec = 'EC 2.7.1.121';
UPDATE enzyme SET history = 'EC 2.7.1.121 created 1989' WHERE ec = 'EC 2.7.1.121';
UPDATE enzyme SET sys_name = 'ATP:xylitol 5-phosphotransferase' WHERE ec = 'EC 2.7.1.122';
UPDATE enzyme SET history = 'EC 2.7.1.122 created 1989' WHERE ec = 'EC 2.7.1.122';
UPDATE enzyme SET history = 'EC 2.7.1.123 created 1989, deleted 2005' WHERE ec = 'EC 2.7.1.123';
UPDATE enzyme SET history = 'EC 2.7.1.124 created 1989, deleted 2005' WHERE ec = 'EC 2.7.1.124';
UPDATE enzyme SET history = 'EC 2.7.1.125 created 1989 (EC 2.7.1.97 created 1978, incorporated 1992), deleted 2005' WHERE ec = 'EC 2.7.1.125';
UPDATE enzyme SET history = 'EC 2.7.1.126 created 1989, deleted 2005' WHERE ec = 'EC 2.7.1.126';
UPDATE enzyme SET sys_name = 'ATP:1D-myo-inositol-1,4,5-trisphosphate 3-phosphotransferase' WHERE ec = 'EC 2.7.1.127';
UPDATE enzyme SET history = 'EC 2.7.1.127 created 1989, modified 2002' WHERE ec = 'EC 2.7.1.127';
UPDATE enzyme SET history = 'EC 2.7.1.128 created 1990 (EC 2.7.1.111 created 1984, incorporated 1992), deleted 2005' WHERE ec = 'EC 2.7.1.128';
UPDATE enzyme SET history = 'EC 2.7.1.129 created 1990, deleted 2005' WHERE ec = 'EC 2.7.1.129';
UPDATE enzyme SET sys_name = 'ATP:2,3,2,3-tetrakis(3-hydroxytetradecanoyl)-D-glucosaminyl-beta-D-1,6-glucosaminyl-beta-phosphate 4-O-phosphotransferase' WHERE ec = 'EC 2.7.1.130';
UPDATE enzyme SET history = 'EC 2.7.1.130 created 1990' WHERE ec = 'EC 2.7.1.130';
UPDATE enzyme SET history = 'EC 2.7.1.131 created 1990, deleted 2005' WHERE ec = 'EC 2.7.1.131';
UPDATE enzyme SET history = 'EC 2.7.1.132 created 1990, deleted 2005' WHERE ec = 'EC 2.7.1.132';
UPDATE enzyme SET history = 'EC 2.7.1.133 created 1990, deleted 2002' WHERE ec = 'EC 2.7.1.133';
UPDATE enzyme SET sys_name = 'ATP:1D-myo-inositol-3,4,5,6-tetrakisphosphate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.134';
UPDATE enzyme SET history = 'EC 2.7.1.134 created 1990, (EC 2.7.1.133 created 1989, incorporated 2002; EC 2.7.1.139 created 1992, incorporated 2002), modified 2002' WHERE ec = 'EC 2.7.1.134';
UPDATE enzyme SET history = 'EC 2.7.1.135 created 1990, deleted 2005' WHERE ec = 'EC 2.7.1.135';
UPDATE enzyme SET sys_name = 'ATP:macrolide 2-O-phosphotransferase' WHERE ec = 'EC 2.7.1.136';
UPDATE enzyme SET history = 'EC 2.7.1.136 created 1992' WHERE ec = 'EC 2.7.1.136';
UPDATE enzyme SET sys_name = 'ATP:1-phosphatidyl-1D-myo-inositol 3-phosphotransferase' WHERE ec = 'EC 2.7.1.137';
UPDATE enzyme SET history = 'EC 2.7.1.137 created 1992, modified 2002' WHERE ec = 'EC 2.7.1.137';
UPDATE enzyme SET sys_name = 'ATP:ceramide 1-phosphotransferase' WHERE ec = 'EC 2.7.1.138';
UPDATE enzyme SET history = 'EC 2.7.1.138 created 1992' WHERE ec = 'EC 2.7.1.138';
UPDATE enzyme SET history = 'EC 2.7.1.139 created 1992, deleted 2002' WHERE ec = 'EC 2.7.1.139';
UPDATE enzyme SET sys_name = 'ATP:1D-myo-inositol-1,3,4,6-tetrakisphosphate 5-phosphotransferase' WHERE ec = 'EC 2.7.1.140';
UPDATE enzyme SET history = 'EC 2.7.1.140 created 1992' WHERE ec = 'EC 2.7.1.140';
UPDATE enzyme SET history = 'EC 2.7.1.141 created 1992, deleted 2005' WHERE ec = 'EC 2.7.1.141';
UPDATE enzyme SET sys_name = 'sn-glycerol-3-phosphate:D-glucose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.142';
UPDATE enzyme SET history = 'EC 2.7.1.142 created 1992' WHERE ec = 'EC 2.7.1.142';
UPDATE enzyme SET sys_name = 'Diphosphate:purine nucleoside phosphotransferase' WHERE ec = 'EC 2.7.1.143';
UPDATE enzyme SET history = 'EC 2.7.1.143 created 1999' WHERE ec = 'EC 2.7.1.143';
UPDATE enzyme SET sys_name = 'ATP:D-tagatose-6-phosphate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.144';
UPDATE enzyme SET history = 'EC 2.7.1.144 created 1999' WHERE ec = 'EC 2.7.1.144';
UPDATE enzyme SET sys_name = 'ATP:deoxynucleoside 5-phosphotransferase' WHERE ec = 'EC 2.7.1.145';
UPDATE enzyme SET history = 'EC 2.7.1.145 created 2001' WHERE ec = 'EC 2.7.1.145';
UPDATE enzyme SET sys_name = 'ADP:D-fructose-6-phosphate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.146';
UPDATE enzyme SET history = 'EC 2.7.1.146 created 2001' WHERE ec = 'EC 2.7.1.146';
UPDATE enzyme SET sys_name = 'ADP:D-glucose 6-phosphotransferase' WHERE ec = 'EC 2.7.1.147';
UPDATE enzyme SET history = 'EC 2.7.1.147 created 2001' WHERE ec = 'EC 2.7.1.147';
UPDATE enzyme SET sys_name = 'ATP:4-(cytidine 5-diphospho)-2-C-methyl-D-erythritol 2-phosphotransferase' WHERE ec = 'EC 2.7.1.148';
UPDATE enzyme SET history = 'EC 2.7.1.148 created 2001' WHERE ec = 'EC 2.7.1.148';
UPDATE enzyme SET sys_name = 'ATP:1-phosphatidyl-1D-myo-inositol-5-phosphate 4-phosphotransferase' WHERE ec = 'EC 2.7.1.149';
UPDATE enzyme SET history = 'EC 2.7.1.149 created 2002' WHERE ec = 'EC 2.7.1.149';
UPDATE enzyme SET sys_name = 'ATP:1-phosphatidyl-1D-myo-inositol-3-phosphate 5-phosphotransferase' WHERE ec = 'EC 2.7.1.150';
UPDATE enzyme SET history = 'EC 2.7.1.150 created 2002' WHERE ec = 'EC 2.7.1.150';
UPDATE enzyme SET sys_name = 'ATP:1D-myo-inositol-1,4,5-trisphosphate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.151';
UPDATE enzyme SET history = 'EC 2.7.1.151 created 2002, modified 2006' WHERE ec = 'EC 2.7.1.151';
UPDATE enzyme SET history = 'EC 2.7.1.152 created 2002, deleted 2003' WHERE ec = 'EC 2.7.1.152';
UPDATE enzyme SET sys_name = 'ATP:1-phosphatidyl-1D-myo-inositol-4,5-bisphosphate 3-phosphotransferase' WHERE ec = 'EC 2.7.1.153';
UPDATE enzyme SET history = 'EC 2.7.1.153 created 2002' WHERE ec = 'EC 2.7.1.153';
UPDATE enzyme SET sys_name = 'ATP:1-phosphatidyl-1D-myo-inositol-4-phosphate 3-phosphotransferase' WHERE ec = 'EC 2.7.1.154';
UPDATE enzyme SET history = 'EC 2.7.1.154 created 2002' WHERE ec = 'EC 2.7.1.154';
UPDATE enzyme SET history = 'EC 2.7.1.155 created 2003, deleted 2007' WHERE ec = 'EC 2.7.1.155';
UPDATE enzyme SET sys_name = 'RTP:adenosylcobinamide phosphotransferase' WHERE ec = 'EC 2.7.1.156';
UPDATE enzyme SET history = 'EC 2.7.1.156 created 2004' WHERE ec = 'EC 2.7.1.156';
UPDATE enzyme SET sys_name = 'ATP:N-acetyl-alpha-D-galactosamine 1-phosphotransferase' WHERE ec = 'EC 2.7.1.157';
UPDATE enzyme SET history = 'EC 2.7.1.157 created 2005' WHERE ec = 'EC 2.7.1.157';
UPDATE enzyme SET sys_name = 'ATP:1D-myo-inositol 1,3,4,5,6-pentakisphosphate 2-phosphotransferase' WHERE ec = 'EC 2.7.1.158';
UPDATE enzyme SET history = 'EC 2.7.1.158 created 2006' WHERE ec = 'EC 2.7.1.158';
UPDATE enzyme SET sys_name = 'ATP:1D-myo-inositol 1,3,4-trisphosphate 5-phosphotransferase' WHERE ec = 'EC 2.7.1.159';
UPDATE enzyme SET history = 'EC 2.7.1.159 created 2006' WHERE ec = 'EC 2.7.1.159';
UPDATE enzyme SET sys_name = '2-phospho-[ligated tRNA]:NAD(+) phosphotransferase' WHERE ec = 'EC 2.7.1.160';
UPDATE enzyme SET history = 'EC 2.7.1.160 created 2006' WHERE ec = 'EC 2.7.1.160';
UPDATE enzyme SET sys_name = 'CTP:riboflavin 5-phosphotransferase' WHERE ec = 'EC 2.7.1.161';
UPDATE enzyme SET history = 'EC 2.7.1.161 created 2008' WHERE ec = 'EC 2.7.1.161';
UPDATE enzyme SET sys_name = 'ATP:N-acetyl-D-hexosamine 1-phosphotransferase' WHERE ec = 'EC 2.7.1.162';
UPDATE enzyme SET history = 'EC 2.7.1.162 created 2008' WHERE ec = 'EC 2.7.1.162';
UPDATE enzyme SET sys_name = 'ATP:hygromycin-B 4-O-phosphotransferase' WHERE ec = 'EC 2.7.1.163';
UPDATE enzyme SET history = 'EC 2.7.1.163 created 2009' WHERE ec = 'EC 2.7.1.163';
UPDATE enzyme SET sys_name = 'ATP:L-seryl-tRNA(Sec) O-phosphotransferase' WHERE ec = 'EC 2.7.1.164';
UPDATE enzyme SET history = 'EC 2.7.1.164 created 2009' WHERE ec = 'EC 2.7.1.164';
UPDATE enzyme SET sys_name = 'ATP:D-glycerate 2-phosphotransferase' WHERE ec = 'EC 2.7.1.165';
UPDATE enzyme SET history = 'EC 2.7.1.165 created 2010' WHERE ec = 'EC 2.7.1.165';
UPDATE enzyme SET sys_name = 'ATP:(KDO)-lipid IV(A) 3-deoxy-alpha-D-manno-oct-2-ulopyranose 4-phosphotransferase' WHERE ec = 'EC 2.7.1.166';
UPDATE enzyme SET history = 'EC 2.7.1.166 created 2010, modified 2011' WHERE ec = 'EC 2.7.1.166';
UPDATE enzyme SET sys_name = 'ATP:D-glycero-beta-D-manno-heptose 7-phosphate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.167';
UPDATE enzyme SET history = 'EC 2.7.1.167 created 2010' WHERE ec = 'EC 2.7.1.167';
UPDATE enzyme SET sys_name = 'ATP:D-glycero-alpha-D-manno-heptose 7-phosphate 1-phosphotransferase' WHERE ec = 'EC 2.7.1.168';
UPDATE enzyme SET history = 'EC 2.7.1.168 created 2010' WHERE ec = 'EC 2.7.1.168';
UPDATE enzyme SET sys_name = 'ATP:(R)-pantoate 4-phosphotransferase' WHERE ec = 'EC 2.7.1.169';
UPDATE enzyme SET history = 'EC 2.7.1.169 created 2010' WHERE ec = 'EC 2.7.1.169';
UPDATE enzyme SET sys_name = 'ATP:1,6-anhydro-N-acetyl-beta-muramate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.170';
UPDATE enzyme SET history = 'EC 2.7.1.170 created 2011' WHERE ec = 'EC 2.7.1.170';
UPDATE enzyme SET sys_name = 'ATP:[protein]-N(6)-D-fructosyl-L-lysine 3-phosphotransferase' WHERE ec = 'EC 2.7.1.171';
UPDATE enzyme SET history = 'EC 2.7.1.171 created 2011' WHERE ec = 'EC 2.7.1.171';
UPDATE enzyme SET sys_name = 'ATP:[protein]-N(6)-D-ribulosyl-L-lysine 3-phosphotransferase' WHERE ec = 'EC 2.7.1.172';
UPDATE enzyme SET history = 'EC 2.7.1.172 created 2011' WHERE ec = 'EC 2.7.1.172';
UPDATE enzyme SET sys_name = 'ATP:beta-D-ribosylnicotinate 5-phosphotransferase' WHERE ec = 'EC 2.7.1.173';
UPDATE enzyme SET history = 'EC 2.7.1.173 created 2012' WHERE ec = 'EC 2.7.1.173';
UPDATE enzyme SET sys_name = 'CTP:1,2-diacyl-sn-glycerol 3-phosphotransferase' WHERE ec = 'EC 2.7.1.174';
UPDATE enzyme SET history = 'EC 2.7.1.174 created 2012, modified 2013' WHERE ec = 'EC 2.7.1.174';
UPDATE enzyme SET sys_name = 'ATP:alpha-maltose 1-phosphotransferase' WHERE ec = 'EC 2.7.1.175';
UPDATE enzyme SET history = 'EC 2.7.1.175 created 2012' WHERE ec = 'EC 2.7.1.175';
UPDATE enzyme SET sys_name = 'ATP:UDP-N-acetyl-alpha-D-glucosamine 3-phosphotransferase' WHERE ec = 'EC 2.7.1.176';
UPDATE enzyme SET history = 'EC 2.7.1.176 created 2012' WHERE ec = 'EC 2.7.1.176';
UPDATE enzyme SET sys_name = 'ATP:L-threonine O(3)-phosphotransferase' WHERE ec = 'EC 2.7.1.177';
UPDATE enzyme SET history = 'EC 2.7.1.177 created 2012' WHERE ec = 'EC 2.7.1.177';
UPDATE enzyme SET sys_name = 'ATP:2-dehydro-3-deoxy-D-gluconate/2-dehydro-3-deoxy-D-galactonate 6-phosphotransferase' WHERE ec = 'EC 2.7.1.178';
UPDATE enzyme SET history = 'EC 2.7.1.178 created 2013' WHERE ec = 'EC 2.7.1.178';
UPDATE enzyme SET sys_name = 'ATP:kanosamine 6-phosphotransferase' WHERE ec = 'EC 2.7.1.179';
UPDATE enzyme SET history = 'EC 2.7.1.179 created 2013' WHERE ec = 'EC 2.7.1.179';
UPDATE enzyme SET sys_name = 'FAD:protein riboflavin-5-phosphate transferase' WHERE ec = 'EC 2.7.1.180';
UPDATE enzyme SET history = 'EC 2.7.1.180 created 2013' WHERE ec = 'EC 2.7.1.180';
UPDATE enzyme SET sys_name = 'ATP:alpha-D-Man-(1-&gt;2)-alpha-D-Man-(1-&gt;2)-alpha-D-Man-(1-&gt;3)-alpha-D-Man-(1-&gt;3)-(alpha-D-Man-(1-&gt;2)-alpha-D-Man-(1-&gt;2)-alpha-D-Man-(1-&gt;3)-alpha-D-Man-(1-&gt;3))(n)-alpha-D-Man-(1-&gt;3)-alpha-D-Man-(1-&gt;3)-alpha-D-GlcNAc-diphospho-ditrans,octacis-undecaprenol 3-phosphotransferase' WHERE ec = 'EC 2.7.1.181';
UPDATE enzyme SET history = 'EC 2.7.1.181 created 2014' WHERE ec = 'EC 2.7.1.181';
UPDATE enzyme SET sys_name = 'CTP:phytol O-phosphotransferase' WHERE ec = 'EC 2.7.1.182';
UPDATE enzyme SET history = 'EC 2.7.1.182 created 2014' WHERE ec = 'EC 2.7.1.182';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.1.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.1.n4';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.1.n5';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.1.n7';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.1.n8';
UPDATE enzyme SET sys_name = 'ATP:acetate phosphotransferase' WHERE ec = 'EC 2.7.2.1';
UPDATE enzyme SET history = 'EC 2.7.2.1 created 1961, modified 2005' WHERE ec = 'EC 2.7.2.1';
UPDATE enzyme SET sys_name = 'ATP:carbamate phosphotransferase' WHERE ec = 'EC 2.7.2.2';
UPDATE enzyme SET history = 'EC 2.7.2.2 created 1961' WHERE ec = 'EC 2.7.2.2';
UPDATE enzyme SET sys_name = 'ATP:3-phospho-D-glycerate 1-phosphotransferase' WHERE ec = 'EC 2.7.2.3';
UPDATE enzyme SET history = 'EC 2.7.2.3 created 1961' WHERE ec = 'EC 2.7.2.3';
UPDATE enzyme SET sys_name = 'ATP:L-aspartate 4-phosphotransferase' WHERE ec = 'EC 2.7.2.4';
UPDATE enzyme SET history = 'EC 2.7.2.4 created 1961' WHERE ec = 'EC 2.7.2.4';
UPDATE enzyme SET history = 'EC 2.7.2.5 created 1965, deleted 1978' WHERE ec = 'EC 2.7.2.5';
UPDATE enzyme SET sys_name = 'ATP:formate phosphotransferase' WHERE ec = 'EC 2.7.2.6';
UPDATE enzyme SET history = 'EC 2.7.2.6 created 1965' WHERE ec = 'EC 2.7.2.6';
UPDATE enzyme SET sys_name = 'ATP:butanoate 1-phosphotransferase' WHERE ec = 'EC 2.7.2.7';
UPDATE enzyme SET history = 'EC 2.7.2.7 created 1972, modified 1986, modified 1990' WHERE ec = 'EC 2.7.2.7';
UPDATE enzyme SET sys_name = 'ATP:N-acetyl-L-glutamate 5-phosphotransferase' WHERE ec = 'EC 2.7.2.8';
UPDATE enzyme SET history = 'EC 2.7.2.8 created 1972' WHERE ec = 'EC 2.7.2.8';
UPDATE enzyme SET history = 'EC 2.7.2.9 created 1972, deleted 1978' WHERE ec = 'EC 2.7.2.9';
UPDATE enzyme SET sys_name = 'GTP:3-phospho-D-glycerate 1-phosphotransferase' WHERE ec = 'EC 2.7.2.10';
UPDATE enzyme SET history = 'EC 2.7.2.10 created 1976' WHERE ec = 'EC 2.7.2.10';
UPDATE enzyme SET sys_name = 'ATP:L-glutamate 5-phosphotransferase' WHERE ec = 'EC 2.7.2.11';
UPDATE enzyme SET history = 'EC 2.7.2.11 created 1976' WHERE ec = 'EC 2.7.2.11';
UPDATE enzyme SET sys_name = 'Diphosphate:acetate phosphotransferase' WHERE ec = 'EC 2.7.2.12';
UPDATE enzyme SET history = 'EC 2.7.2.12 created 1976' WHERE ec = 'EC 2.7.2.12';
UPDATE enzyme SET sys_name = 'ATP:L-glutamate 1-phosphotransferase' WHERE ec = 'EC 2.7.2.13';
UPDATE enzyme SET history = 'EC 2.7.2.13 created 1984' WHERE ec = 'EC 2.7.2.13';
UPDATE enzyme SET sys_name = 'ATP:branched-chain-fatty-acid 1-phosphotransferase' WHERE ec = 'EC 2.7.2.14';
UPDATE enzyme SET history = 'EC 2.7.2.14 created 1990' WHERE ec = 'EC 2.7.2.14';
UPDATE enzyme SET sys_name = 'ATP:propanoate phosphotransferase' WHERE ec = 'EC 2.7.2.15';
UPDATE enzyme SET history = 'EC 2.7.2.15 created 2005' WHERE ec = 'EC 2.7.2.15';
UPDATE enzyme SET sys_name = 'ATP:guanidinoacetate N-phosphotransferase' WHERE ec = 'EC 2.7.3.1';
UPDATE enzyme SET history = 'EC 2.7.3.1 created 1961' WHERE ec = 'EC 2.7.3.1';
UPDATE enzyme SET sys_name = 'ATP:creatine N-phosphotransferase' WHERE ec = 'EC 2.7.3.2';
UPDATE enzyme SET history = 'EC 2.7.3.2 created 1961' WHERE ec = 'EC 2.7.3.2';
UPDATE enzyme SET sys_name = 'ATP:L-arginine N(omega)-phosphotransferase' WHERE ec = 'EC 2.7.3.3';
UPDATE enzyme SET history = 'EC 2.7.3.3 created 1961' WHERE ec = 'EC 2.7.3.3';
UPDATE enzyme SET sys_name = 'ATP:taurocyamine N-phosphotransferase' WHERE ec = 'EC 2.7.3.4';
UPDATE enzyme SET history = 'EC 2.7.3.4 created 1965' WHERE ec = 'EC 2.7.3.4';
UPDATE enzyme SET sys_name = 'ATP:lombricine N-phosphotransferase' WHERE ec = 'EC 2.7.3.5';
UPDATE enzyme SET history = 'EC 2.7.3.5 created 1965, modified 1976' WHERE ec = 'EC 2.7.3.5';
UPDATE enzyme SET sys_name = 'ATP:hypotaurocyamine N-phosphotransferase' WHERE ec = 'EC 2.7.3.6';
UPDATE enzyme SET history = 'EC 2.7.3.6 created 1965' WHERE ec = 'EC 2.7.3.6';
UPDATE enzyme SET sys_name = 'ATP:guanidinoethyl-methyl-phosphate phosphotransferase' WHERE ec = 'EC 2.7.3.7';
UPDATE enzyme SET history = 'EC 2.7.3.7 created 1972' WHERE ec = 'EC 2.7.3.7';
UPDATE enzyme SET sys_name = 'ATP:ammonia phosphotransferase' WHERE ec = 'EC 2.7.3.8';
UPDATE enzyme SET history = 'EC 2.7.3.8 created 1972' WHERE ec = 'EC 2.7.3.8';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate:protein-L-histidine N(pi)-phosphotransferase' WHERE ec = 'EC 2.7.3.9';
UPDATE enzyme SET history = 'EC 2.7.3.9 created 1972' WHERE ec = 'EC 2.7.3.9';
UPDATE enzyme SET sys_name = 'ATP:agmatine N(4)-phosphotransferase' WHERE ec = 'EC 2.7.3.10';
UPDATE enzyme SET history = 'EC 2.7.3.10 created 1984' WHERE ec = 'EC 2.7.3.10';
UPDATE enzyme SET history = 'EC 2.7.3.11 created 1989, deleted 2005' WHERE ec = 'EC 2.7.3.11';
UPDATE enzyme SET history = 'EC 2.7.3.12 created 1989, deleted 2005' WHERE ec = 'EC 2.7.3.12';
UPDATE enzyme SET sys_name = 'ATP:polyphosphate phosphotransferase' WHERE ec = 'EC 2.7.4.1';
UPDATE enzyme SET history = 'EC 2.7.4.1 created 1961' WHERE ec = 'EC 2.7.4.1';
UPDATE enzyme SET sys_name = 'ATP:(R)-5-phosphomevalonate phosphotransferase' WHERE ec = 'EC 2.7.4.2';
UPDATE enzyme SET history = 'EC 2.7.4.2 created 1961' WHERE ec = 'EC 2.7.4.2';
UPDATE enzyme SET sys_name = 'ATP:AMP phosphotransferase' WHERE ec = 'EC 2.7.4.3';
UPDATE enzyme SET history = 'EC 2.7.4.3 created 1961' WHERE ec = 'EC 2.7.4.3';
UPDATE enzyme SET sys_name = 'ATP:nucleoside-phosphate phosphotransferase' WHERE ec = 'EC 2.7.4.4';
UPDATE enzyme SET history = 'EC 2.7.4.4 created 1961' WHERE ec = 'EC 2.7.4.4';
UPDATE enzyme SET history = 'EC 2.7.4.5 created 1961, deleted 1972' WHERE ec = 'EC 2.7.4.5';
UPDATE enzyme SET sys_name = 'ATP:nucleoside-diphosphate phosphotransferase' WHERE ec = 'EC 2.7.4.6';
UPDATE enzyme SET history = 'EC 2.7.4.6 created 1961' WHERE ec = 'EC 2.7.4.6';
UPDATE enzyme SET sys_name = 'ATP:(4-amino-2-methylpyrimidin-5-yl)methyl-phosphate phosphotransferase' WHERE ec = 'EC 2.7.4.7';
UPDATE enzyme SET history = 'EC 2.7.4.7 created 1965' WHERE ec = 'EC 2.7.4.7';
UPDATE enzyme SET sys_name = 'ATP:(d)GMP phosphotransferase' WHERE ec = 'EC 2.7.4.8';
UPDATE enzyme SET history = 'EC 2.7.4.8 created 1965' WHERE ec = 'EC 2.7.4.8';
UPDATE enzyme SET sys_name = 'ATP:dTMP phosphotransferase' WHERE ec = 'EC 2.7.4.9';
UPDATE enzyme SET history = 'EC 2.7.4.9 created 1965' WHERE ec = 'EC 2.7.4.9';
UPDATE enzyme SET sys_name = 'Nucleoside-triphosphate:AMP phosphotransferase' WHERE ec = 'EC 2.7.4.10';
UPDATE enzyme SET history = 'EC 2.7.4.10 created 1965' WHERE ec = 'EC 2.7.4.10';
UPDATE enzyme SET sys_name = 'ATP:(d)AMP phosphotransferase' WHERE ec = 'EC 2.7.4.11';
UPDATE enzyme SET history = 'EC 2.7.4.11 created 1972' WHERE ec = 'EC 2.7.4.11';
UPDATE enzyme SET sys_name = 'ATP:(d)NMP phosphotransferase' WHERE ec = 'EC 2.7.4.12';
UPDATE enzyme SET history = 'EC 2.7.4.12 created 1972' WHERE ec = 'EC 2.7.4.12';
UPDATE enzyme SET sys_name = 'ATP:deoxynucleoside-phosphate phosphotransferase' WHERE ec = 'EC 2.7.4.13';
UPDATE enzyme SET history = 'EC 2.7.4.13 created 1972' WHERE ec = 'EC 2.7.4.13';
UPDATE enzyme SET sys_name = 'ATP:CMP(UMP) phosphotransferase' WHERE ec = 'EC 2.7.4.14';
UPDATE enzyme SET history = 'EC 2.7.4.14 created 1961 as EC 2.7.4.5, transferred 1972 to EC 2.7.4.14, modified 1980, modified 2010, modified 2011' WHERE ec = 'EC 2.7.4.14';
UPDATE enzyme SET sys_name = 'ATP:thiamine-diphosphate phosphotransferase' WHERE ec = 'EC 2.7.4.15';
UPDATE enzyme SET history = 'EC 2.7.4.15 created 1972' WHERE ec = 'EC 2.7.4.15';
UPDATE enzyme SET sys_name = 'ATP:thiamine-phosphate phosphotransferase' WHERE ec = 'EC 2.7.4.16';
UPDATE enzyme SET history = 'EC 2.7.4.16 created 1976' WHERE ec = 'EC 2.7.4.16';
UPDATE enzyme SET sys_name = '3-phospho-D-glyceroyl-phosphate:polyphosphate phosphotransferase' WHERE ec = 'EC 2.7.4.17';
UPDATE enzyme SET history = 'EC 2.7.4.17 created 1976' WHERE ec = 'EC 2.7.4.17';
UPDATE enzyme SET sys_name = 'ATP:farnesyl-diphosphate phosphotransferase' WHERE ec = 'EC 2.7.4.18';
UPDATE enzyme SET history = 'EC 2.7.4.18 created 1978' WHERE ec = 'EC 2.7.4.18';
UPDATE enzyme SET sys_name = 'ATP:5-methyldeoxycytidine-5-phosphate phosphotransferase' WHERE ec = 'EC 2.7.4.19';
UPDATE enzyme SET history = 'EC 2.7.4.19 created 1984' WHERE ec = 'EC 2.7.4.19';
UPDATE enzyme SET sys_name = 'Dolichyl-diphosphate:polyphosphate phosphotransferase' WHERE ec = 'EC 2.7.4.20';
UPDATE enzyme SET history = 'EC 2.7.4.20 created 1989' WHERE ec = 'EC 2.7.4.20';
UPDATE enzyme SET sys_name = 'ATP:1D-myo-inositol-hexakisphosphate 5-phosphotransferase' WHERE ec = 'EC 2.7.4.21';
UPDATE enzyme SET history = 'EC 2.7.4.21 created 2002 as EC 2.7.1.152, transferred 2003 to EC 2.7.4.21, modified 2013' WHERE ec = 'EC 2.7.4.21';
UPDATE enzyme SET sys_name = 'ATP:UMP phosphotransferase' WHERE ec = 'EC 2.7.4.22';
UPDATE enzyme SET history = 'EC 2.7.4.22 created 2006' WHERE ec = 'EC 2.7.4.22';
UPDATE enzyme SET sys_name = 'ATP:alpha-D-ribose-1,5-bisphosphate phosphotransferase' WHERE ec = 'EC 2.7.4.23';
UPDATE enzyme SET history = 'EC 2.7.4.23 created 2006' WHERE ec = 'EC 2.7.4.23';
UPDATE enzyme SET sys_name = 'ATP:1D-myo-inositol-5-diphosphate-pentakisphosphate phosphotransferase' WHERE ec = 'EC 2.7.4.24';
UPDATE enzyme SET history = 'EC 2.7.4.24 created 2003 as EC 2.7.1.155, transferred 2007 to EC 2.7.4.24, modified 2014' WHERE ec = 'EC 2.7.4.24';
UPDATE enzyme SET sys_name = 'ATP:(d)CMP phosphotransferase' WHERE ec = 'EC 2.7.4.25';
UPDATE enzyme SET history = 'EC 2.7.4.25 created 2010' WHERE ec = 'EC 2.7.4.25';
UPDATE enzyme SET sys_name = 'ATP:isopentenyl phosphate phosphotransferase' WHERE ec = 'EC 2.7.4.26';
UPDATE enzyme SET history = 'EC 2.7.4.26 created 2012' WHERE ec = 'EC 2.7.4.26';
UPDATE enzyme SET sys_name = '[Pyruvate, phosphate dikinase] phosphate:phosphate phosphotransferase' WHERE ec = 'EC 2.7.4.27';
UPDATE enzyme SET history = 'EC 2.7.4.27 created 2012' WHERE ec = 'EC 2.7.4.27';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.4.n1';
UPDATE enzyme SET sys_name = '[Pyruvate, water dikinase] phosphate:phosphate phosphotransferase' WHERE ec = 'EC 2.7.4.n2';
UPDATE enzyme SET history = 'EC 2.7.4.28 created 2012' WHERE ec = 'EC 2.7.4.n2';
UPDATE enzyme SET history = 'EC 2.7.5.1 created 1961, deleted 1984' WHERE ec = 'EC 2.7.5.1';
UPDATE enzyme SET history = 'EC 2.7.5.2 created 1961, deleted 1984' WHERE ec = 'EC 2.7.5.2';
UPDATE enzyme SET history = 'EC 2.7.5.3 created 1961, deleted 1984' WHERE ec = 'EC 2.7.5.3';
UPDATE enzyme SET history = 'EC 2.7.5.4 created 1961, deleted 1984' WHERE ec = 'EC 2.7.5.4';
UPDATE enzyme SET history = 'EC 2.7.5.5 created 1972, deleted 1984' WHERE ec = 'EC 2.7.5.5';
UPDATE enzyme SET history = 'EC 2.7.5.6 created 1972, deleted 1984' WHERE ec = 'EC 2.7.5.6';
UPDATE enzyme SET history = 'EC 2.7.5.7 created 1981, deleted 1984' WHERE ec = 'EC 2.7.5.7';
UPDATE enzyme SET sys_name = 'ATP:D-ribose-5-phosphate diphosphotransferase' WHERE ec = 'EC 2.7.6.1';
UPDATE enzyme SET history = 'EC 2.7.6.1 created 1961' WHERE ec = 'EC 2.7.6.1';
UPDATE enzyme SET sys_name = 'ATP:thiamine diphosphotransferase' WHERE ec = 'EC 2.7.6.2';
UPDATE enzyme SET history = 'EC 2.7.6.2 created 1961' WHERE ec = 'EC 2.7.6.2';
UPDATE enzyme SET sys_name = 'ATP:2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine 6-diphosphotransferase' WHERE ec = 'EC 2.7.6.3';
UPDATE enzyme SET history = 'EC 2.7.6.3 created 1972' WHERE ec = 'EC 2.7.6.3';
UPDATE enzyme SET sys_name = 'ATP:nucleoside-5-phosphate diphosphotransferase' WHERE ec = 'EC 2.7.6.4';
UPDATE enzyme SET history = 'EC 2.7.6.4 created 1976' WHERE ec = 'EC 2.7.6.4';
UPDATE enzyme SET sys_name = 'ATP:GTP 3-diphosphotransferase' WHERE ec = 'EC 2.7.6.5';
UPDATE enzyme SET history = 'EC 2.7.6.5 created 1981' WHERE ec = 'EC 2.7.6.5';
UPDATE enzyme SET sys_name = 'ATP:nicotinamide-nucleotide adenylyltransferase' WHERE ec = 'EC 2.7.7.1';
UPDATE enzyme SET history = 'EC 2.7.7.1 created 1961' WHERE ec = 'EC 2.7.7.1';
UPDATE enzyme SET sys_name = 'ATP:FMN adenylyltransferase' WHERE ec = 'EC 2.7.7.2';
UPDATE enzyme SET history = 'EC 2.7.7.2 created 1961, modified 2007' WHERE ec = 'EC 2.7.7.2';
UPDATE enzyme SET sys_name = 'ATP:pantetheine-4-phosphate adenylyltransferase' WHERE ec = 'EC 2.7.7.3';
UPDATE enzyme SET history = 'EC 2.7.7.3 created 1961, modified 2002' WHERE ec = 'EC 2.7.7.3';
UPDATE enzyme SET sys_name = 'ATP:sulfate adenylyltransferase' WHERE ec = 'EC 2.7.7.4';
UPDATE enzyme SET history = 'EC 2.7.7.4 created 1961, modified 1999' WHERE ec = 'EC 2.7.7.4';
UPDATE enzyme SET sys_name = 'ADP:sulfate adenylyltransferase' WHERE ec = 'EC 2.7.7.5';
UPDATE enzyme SET history = 'EC 2.7.7.5 created 1961' WHERE ec = 'EC 2.7.7.5';
UPDATE enzyme SET sys_name = 'Nucleoside-triphosphate:RNA nucleotidyltransferase (DNA-directed)' WHERE ec = 'EC 2.7.7.6';
UPDATE enzyme SET history = 'EC 2.7.7.6 created 1961, modified 1981, modified 1982, modified 1989' WHERE ec = 'EC 2.7.7.6';
UPDATE enzyme SET sys_name = 'Deoxynucleoside-triphosphate:DNA deoxynucleotidyltransferase (DNA-directed)' WHERE ec = 'EC 2.7.7.7';
UPDATE enzyme SET history = 'EC 2.7.7.7 created 1961, modified 1981, modified 1982' WHERE ec = 'EC 2.7.7.7';
UPDATE enzyme SET sys_name = 'Polyribonucleotide:phosphate nucleotidyltransferase' WHERE ec = 'EC 2.7.7.8';
UPDATE enzyme SET history = 'EC 2.7.7.8 created 1961' WHERE ec = 'EC 2.7.7.8';
UPDATE enzyme SET sys_name = 'UTP:alpha-D-glucose-1-phosphate uridylyltransferase' WHERE ec = 'EC 2.7.7.9';
UPDATE enzyme SET history = 'EC 2.7.7.9 created 1961' WHERE ec = 'EC 2.7.7.9';
UPDATE enzyme SET sys_name = 'UTP:alpha-D-hexose-1-phosphate uridylyltransferase' WHERE ec = 'EC 2.7.7.10';
UPDATE enzyme SET history = 'EC 2.7.7.10 created 1961' WHERE ec = 'EC 2.7.7.10';
UPDATE enzyme SET sys_name = 'UTP:alpha-D-xylose-1-phosphate uridylyltransferase' WHERE ec = 'EC 2.7.7.11';
UPDATE enzyme SET history = 'EC 2.7.7.11 created 1961' WHERE ec = 'EC 2.7.7.11';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose:alpha-D-galactose-1-phosphate uridylyltransferase' WHERE ec = 'EC 2.7.7.12';
UPDATE enzyme SET history = 'EC 2.7.7.12 created 1961' WHERE ec = 'EC 2.7.7.12';
UPDATE enzyme SET sys_name = 'GTP:alpha-D-mannose-1-phosphate guanylyltransferase' WHERE ec = 'EC 2.7.7.13';
UPDATE enzyme SET history = 'EC 2.7.7.13 created 1961, modified 1976' WHERE ec = 'EC 2.7.7.13';
UPDATE enzyme SET sys_name = 'CTP:ethanolamine-phosphate cytidylyltransferase' WHERE ec = 'EC 2.7.7.14';
UPDATE enzyme SET history = 'EC 2.7.7.14 created 1961' WHERE ec = 'EC 2.7.7.14';
UPDATE enzyme SET sys_name = 'CTP:choline-phosphate cytidylyltransferase' WHERE ec = 'EC 2.7.7.15';
UPDATE enzyme SET history = 'EC 2.7.7.15 created 1961' WHERE ec = 'EC 2.7.7.15';
UPDATE enzyme SET history = 'EC 2.7.7.16 created 1961, deleted 1972, (transferred to EC 3.1.4.22, deleted 1980)' WHERE ec = 'EC 2.7.7.16';
UPDATE enzyme SET history = 'EC 2.7.7.17 created 1965, deleted 1972, (transferred to EC 3.1.4.23, deleted 1980)' WHERE ec = 'EC 2.7.7.17';
UPDATE enzyme SET sys_name = 'ATP:beta-nicotinate-D-ribonucleotide adenylyltransferase' WHERE ec = 'EC 2.7.7.18';
UPDATE enzyme SET history = 'EC 2.7.7.18 created 1965' WHERE ec = 'EC 2.7.7.18';
UPDATE enzyme SET sys_name = 'ATP:polynucleotide adenylyltransferase' WHERE ec = 'EC 2.7.7.19';
UPDATE enzyme SET history = 'EC 2.7.7.19 created 1965' WHERE ec = 'EC 2.7.7.19';
UPDATE enzyme SET history = 'EC 2.7.7.20 created 1965, deleted 1972' WHERE ec = 'EC 2.7.7.20';
UPDATE enzyme SET history = 'EC 2.7.7.21 created 1965, deleted 2010' WHERE ec = 'EC 2.7.7.21';
UPDATE enzyme SET sys_name = 'GDP:alpha-D-mannose-1-phosphate guanylyltransferase' WHERE ec = 'EC 2.7.7.22';
UPDATE enzyme SET history = 'EC 2.7.7.22 created 1965, modified 1976' WHERE ec = 'EC 2.7.7.22';
UPDATE enzyme SET sys_name = 'UTP:N-acetyl-alpha-D-glucosamine-1-phosphate uridylyltransferase' WHERE ec = 'EC 2.7.7.23';
UPDATE enzyme SET history = 'EC 2.7.7.23 created 1965, modified 2012' WHERE ec = 'EC 2.7.7.23';
UPDATE enzyme SET sys_name = 'dTTP:alpha-D-glucose-1-phosphate thymidylyltransferase' WHERE ec = 'EC 2.7.7.24';
UPDATE enzyme SET history = 'EC 2.7.7.24 created 1965' WHERE ec = 'EC 2.7.7.24';
UPDATE enzyme SET history = 'EC 2.7.7.25 created 1965, deleted 2010' WHERE ec = 'EC 2.7.7.25';
UPDATE enzyme SET history = 'EC 2.7.7.26 created 1965, deleted 1972, (transferred to EC 3.1.4.8, deleted 1980)' WHERE ec = 'EC 2.7.7.26';
UPDATE enzyme SET sys_name = 'ATP:alpha-D-glucose-1-phosphate adenylyltransferase' WHERE ec = 'EC 2.7.7.27';
UPDATE enzyme SET history = 'EC 2.7.7.27 created 1972' WHERE ec = 'EC 2.7.7.27';
UPDATE enzyme SET sys_name = 'NTP:alpha-D-aldose-1-phosphate nucleotidyltransferase' WHERE ec = 'EC 2.7.7.28';
UPDATE enzyme SET history = 'EC 2.7.7.28 created 1972, modified 2004 (EC 2.7.7.29 created 1972, incorporated 2004)' WHERE ec = 'EC 2.7.7.28';
UPDATE enzyme SET history = 'EC 2.7.7.29 created 1972, deleted 2004' WHERE ec = 'EC 2.7.7.29';
UPDATE enzyme SET sys_name = 'GTP:beta-L-fucose-1-phosphate guanylyltransferase' WHERE ec = 'EC 2.7.7.30';
UPDATE enzyme SET history = 'EC 2.7.7.30 created 1972' WHERE ec = 'EC 2.7.7.30';
UPDATE enzyme SET sys_name = 'Nucleoside-triphosphate:DNA deoxynucleotidylexotransferase' WHERE ec = 'EC 2.7.7.31';
UPDATE enzyme SET history = 'EC 2.7.7.31 created 1972' WHERE ec = 'EC 2.7.7.31';
UPDATE enzyme SET sys_name = 'dTTP:alpha-D-galactose-1-phosphate thymidylyltransferase' WHERE ec = 'EC 2.7.7.32';
UPDATE enzyme SET history = 'EC 2.7.7.32 created 1972' WHERE ec = 'EC 2.7.7.32';
UPDATE enzyme SET sys_name = 'CTP:alpha-D-glucose-1-phosphate cytidylyltransferase' WHERE ec = 'EC 2.7.7.33';
UPDATE enzyme SET history = 'EC 2.7.7.33 created 1972' WHERE ec = 'EC 2.7.7.33';
UPDATE enzyme SET sys_name = 'GTP:alpha-D-glucose-1-phosphate guanylyltransferase' WHERE ec = 'EC 2.7.7.34';
UPDATE enzyme SET history = 'EC 2.7.7.34 created 1972' WHERE ec = 'EC 2.7.7.34';
UPDATE enzyme SET sys_name = 'ADP:D-ribose-5-phosphate adenylyltransferase' WHERE ec = 'EC 2.7.7.35';
UPDATE enzyme SET history = 'EC 2.7.7.35 created 1972' WHERE ec = 'EC 2.7.7.35';
UPDATE enzyme SET sys_name = 'ADP:alpha-D-aldose-1-phosphate adenylyltransferase' WHERE ec = 'EC 2.7.7.36';
UPDATE enzyme SET history = 'EC 2.7.7.36 created 1972, modified 1986' WHERE ec = 'EC 2.7.7.36';
UPDATE enzyme SET sys_name = 'NDP:alpha-D-aldose-1-phosphate nucleotidyltransferase' WHERE ec = 'EC 2.7.7.37';
UPDATE enzyme SET history = 'EC 2.7.7.37 created 1972, modified 1986' WHERE ec = 'EC 2.7.7.37';
UPDATE enzyme SET sys_name = 'CTP:3-deoxy-D-manno-octulosonate cytidylyltransferase' WHERE ec = 'EC 2.7.7.38';
UPDATE enzyme SET history = 'EC 2.7.7.38 created 1972' WHERE ec = 'EC 2.7.7.38';
UPDATE enzyme SET sys_name = 'CTP:sn-glycerol-3-phosphate cytidylyltransferase' WHERE ec = 'EC 2.7.7.39';
UPDATE enzyme SET history = 'EC 2.7.7.39 created 1972' WHERE ec = 'EC 2.7.7.39';
UPDATE enzyme SET sys_name = 'CTP:D-ribitol-5-phosphate cytidylyltransferase' WHERE ec = 'EC 2.7.7.40';
UPDATE enzyme SET history = 'EC 2.7.7.40 created 1972' WHERE ec = 'EC 2.7.7.40';
UPDATE enzyme SET sys_name = 'CTP:phosphatidate cytidylyltransferase' WHERE ec = 'EC 2.7.7.41';
UPDATE enzyme SET history = 'EC 2.7.7.41 created 1972' WHERE ec = 'EC 2.7.7.41';
UPDATE enzyme SET sys_name = 'ATP:[L-glutamate:ammonia ligase (ADP-forming)] adenylyltransferase' WHERE ec = 'EC 2.7.7.42';
UPDATE enzyme SET history = 'EC 2.7.7.42 created 1972' WHERE ec = 'EC 2.7.7.42';
UPDATE enzyme SET sys_name = 'CTP:N-acylneuraminate cytidylyltransferase' WHERE ec = 'EC 2.7.7.43';
UPDATE enzyme SET history = 'EC 2.7.7.43 created 1972' WHERE ec = 'EC 2.7.7.43';
UPDATE enzyme SET sys_name = 'UTP:1-phospho-alpha-D-glucuronate uridylyltransferase' WHERE ec = 'EC 2.7.7.44';
UPDATE enzyme SET history = 'EC 2.7.7.44 created 1976' WHERE ec = 'EC 2.7.7.44';
UPDATE enzyme SET sys_name = 'GTP:GTP guanylyltransferase' WHERE ec = 'EC 2.7.7.45';
UPDATE enzyme SET history = 'EC 2.7.7.45 created 1976' WHERE ec = 'EC 2.7.7.45';
UPDATE enzyme SET sys_name = 'NTP:gentamicin 2-nucleotidyltransferase' WHERE ec = 'EC 2.7.7.46';
UPDATE enzyme SET history = 'EC 2.7.7.46 created 1976' WHERE ec = 'EC 2.7.7.46';
UPDATE enzyme SET sys_name = 'ATP:streptomycin 3-adenylyltransferase' WHERE ec = 'EC 2.7.7.47';
UPDATE enzyme SET history = 'EC 2.7.7.47 created 1976' WHERE ec = 'EC 2.7.7.47';
UPDATE enzyme SET sys_name = 'Nucleoside-triphosphate:RNA nucleotidyltransferase (RNA-directed)' WHERE ec = 'EC 2.7.7.48';
UPDATE enzyme SET history = 'EC 2.7.7.48 created 1981, modified 1982' WHERE ec = 'EC 2.7.7.48';
UPDATE enzyme SET sys_name = 'Deoxynucleoside-triphosphate:DNA deoxynucleotidyltransferase (RNA-directed)' WHERE ec = 'EC 2.7.7.49';
UPDATE enzyme SET history = 'EC 2.7.7.49 created 1981, modified 1982' WHERE ec = 'EC 2.7.7.49';
UPDATE enzyme SET sys_name = 'GTP:mRNA guanylyltransferase' WHERE ec = 'EC 2.7.7.50';
UPDATE enzyme SET history = 'EC 2.7.7.50 created 1981' WHERE ec = 'EC 2.7.7.50';
UPDATE enzyme SET sys_name = 'Adenylyl-sulfate:ammonia adenylyltransferase' WHERE ec = 'EC 2.7.7.51';
UPDATE enzyme SET history = 'EC 2.7.7.51 created 1982' WHERE ec = 'EC 2.7.7.51';
UPDATE enzyme SET sys_name = 'UTP:RNA uridylyltransferase' WHERE ec = 'EC 2.7.7.52';
UPDATE enzyme SET history = 'EC 2.7.7.52 created 1983' WHERE ec = 'EC 2.7.7.52';
UPDATE enzyme SET sys_name = 'ADP:ATP adenylyltransferase' WHERE ec = 'EC 2.7.7.53';
UPDATE enzyme SET history = 'EC 2.7.7.53 created 1986' WHERE ec = 'EC 2.7.7.53';
UPDATE enzyme SET history = 'EC 2.7.7.54 created 1989, deleted 2013' WHERE ec = 'EC 2.7.7.54';
UPDATE enzyme SET history = 'EC 2.7.7.55 created 1989, deleted 2013' WHERE ec = 'EC 2.7.7.55';
UPDATE enzyme SET sys_name = 'tRNA:phosphate nucleotidyltransferase' WHERE ec = 'EC 2.7.7.56';
UPDATE enzyme SET history = 'EC 2.7.7.56 created 1992' WHERE ec = 'EC 2.7.7.56';
UPDATE enzyme SET sys_name = 'CTP:N-methylethanolamine-phosphate cytidylyltransferase' WHERE ec = 'EC 2.7.7.57';
UPDATE enzyme SET history = 'EC 2.7.7.57 created 1992' WHERE ec = 'EC 2.7.7.57';
UPDATE enzyme SET sys_name = 'ATP:2,3-dihydroxybenzoate adenylyltransferase' WHERE ec = 'EC 2.7.7.58';
UPDATE enzyme SET history = 'EC 2.7.7.58 created 1992' WHERE ec = 'EC 2.7.7.58';
UPDATE enzyme SET sys_name = 'UTP:[protein-PII] uridylyltransferase' WHERE ec = 'EC 2.7.7.59';
UPDATE enzyme SET history = 'EC 2.7.7.59 created 1999' WHERE ec = 'EC 2.7.7.59';
UPDATE enzyme SET sys_name = 'CTP:2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase' WHERE ec = 'EC 2.7.7.60';
UPDATE enzyme SET history = 'EC 2.7.7.60 created 2001' WHERE ec = 'EC 2.7.7.60';
UPDATE enzyme SET sys_name = '2-(5-triphosphoribosyl)-3-dephospho-CoA:apo-citrate-lyase 2-(5-phosphoribosyl)-3-dephospho-CoA-transferase' WHERE ec = 'EC 2.7.7.61';
UPDATE enzyme SET history = 'EC 2.7.7.61 created 2002, modified 2008' WHERE ec = 'EC 2.7.7.61';
UPDATE enzyme SET sys_name = 'GTP:adenosylcobinamide-phosphate guanylyltransferase' WHERE ec = 'EC 2.7.7.62';
UPDATE enzyme SET history = 'EC 2.7.7.62 created 2004' WHERE ec = 'EC 2.7.7.62';
UPDATE enzyme SET sys_name = 'ATP:lipoate adenylyltransferase' WHERE ec = 'EC 2.7.7.63';
UPDATE enzyme SET history = 'EC 2.7.7.63 created 2006' WHERE ec = 'EC 2.7.7.63';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 2.7.7.64';
UPDATE enzyme SET history = 'EC 2.7.7.64 created 2006' WHERE ec = 'EC 2.7.7.64';
UPDATE enzyme SET sys_name = 'GTP:GTP guanylyltransferase (cyclizing)' WHERE ec = 'EC 2.7.7.65';
UPDATE enzyme SET history = 'EC 2.7.7.65 created 2008' WHERE ec = 'EC 2.7.7.65';
UPDATE enzyme SET sys_name = '2-(5-triphosphoribosyl)-3-dephospho-CoA:apo-malonate-decarboxylase 2-(5-phosphoribosyl)-3-dephospho-CoA-transferase' WHERE ec = 'EC 2.7.7.66';
UPDATE enzyme SET history = 'EC 2.7.7.66 created 2008' WHERE ec = 'EC 2.7.7.66';
UPDATE enzyme SET sys_name = 'CTP:2,3-bis-O-(geranylgeranyl)-sn-glycero-1-phosphate cytidylyltransferase' WHERE ec = 'EC 2.7.7.67';
UPDATE enzyme SET history = 'EC 2.7.7.67 created 2009' WHERE ec = 'EC 2.7.7.67';
UPDATE enzyme SET sys_name = 'GTP:2-phospho-L-lactate guanylyltransferase' WHERE ec = 'EC 2.7.7.68';
UPDATE enzyme SET history = 'EC 2.7.7.68 created 2010' WHERE ec = 'EC 2.7.7.68';
UPDATE enzyme SET sys_name = 'GDP:alpha-L-galactose 1-phosphate guanylyltransferase' WHERE ec = 'EC 2.7.7.69';
UPDATE enzyme SET history = 'EC 2.7.7.69 created 2010' WHERE ec = 'EC 2.7.7.69';
UPDATE enzyme SET sys_name = 'ATP:D-glycero-beta-D-manno-heptose 1-phosphate adenylyltransferase' WHERE ec = 'EC 2.7.7.70';
UPDATE enzyme SET history = 'EC 2.7.7.70 created 2010' WHERE ec = 'EC 2.7.7.70';
UPDATE enzyme SET sys_name = 'GTP:D-glycero-alpha-D-manno-heptose 1-phosphate guanylyltransferase' WHERE ec = 'EC 2.7.7.71';
UPDATE enzyme SET history = 'EC 2.7.7.71 created 2010' WHERE ec = 'EC 2.7.7.71';
UPDATE enzyme SET sys_name = 'CTP,CTP,ATP:tRNA cytidylyl,cytidylyl,adenylyltransferase' WHERE ec = 'EC 2.7.7.72';
UPDATE enzyme SET history = 'EC 2.7.7.72 created 2010' WHERE ec = 'EC 2.7.7.72';
UPDATE enzyme SET sys_name = 'ATP:[ThiS] adenylyltransferase' WHERE ec = 'EC 2.7.7.73';
UPDATE enzyme SET history = 'EC 2.7.7.73 created 2011' WHERE ec = 'EC 2.7.7.73';
UPDATE enzyme SET sys_name = 'CTP:1L-myo-inositol 1-phosphate cytidylyltransferase' WHERE ec = 'EC 2.7.7.74';
UPDATE enzyme SET history = 'EC 2.7.7.74 created 2011' WHERE ec = 'EC 2.7.7.74';
UPDATE enzyme SET sys_name = 'ATP:molybdopterin adenylyltransferase' WHERE ec = 'EC 2.7.7.75';
UPDATE enzyme SET history = 'EC 2.7.7.75 created 2011' WHERE ec = 'EC 2.7.7.75';
UPDATE enzyme SET sys_name = 'CTP:molybdenum cofactor cytidylyltransferase' WHERE ec = 'EC 2.7.7.76';
UPDATE enzyme SET history = 'EC 2.7.7.76 created 2011' WHERE ec = 'EC 2.7.7.76';
UPDATE enzyme SET sys_name = 'GTP:molybdenum cofactor guanylyltransferase' WHERE ec = 'EC 2.7.7.77';
UPDATE enzyme SET history = 'EC 2.7.7.77 created 2011' WHERE ec = 'EC 2.7.7.77';
UPDATE enzyme SET sys_name = 'GDP:alpha-D-glucose 1-phosphate guanylyltransferase' WHERE ec = 'EC 2.7.7.78';
UPDATE enzyme SET history = 'EC 2.7.7.78 created 2011' WHERE ec = 'EC 2.7.7.78';
UPDATE enzyme SET sys_name = 'p-tRNA(His):GTP guanylyltransferase (ATP-hydrolyzing)' WHERE ec = 'EC 2.7.7.79';
UPDATE enzyme SET history = 'EC 2.7.7.79 created 2011' WHERE ec = 'EC 2.7.7.79';
UPDATE enzyme SET sys_name = 'ATP:molybdopterin-synthase adenylyltransferase' WHERE ec = 'EC 2.7.7.80';
UPDATE enzyme SET history = 'EC 2.7.7.80 created 2011' WHERE ec = 'EC 2.7.7.80';
UPDATE enzyme SET sys_name = 'CTP:5,7-diacetamido-3,5,7,9-tetradeoxy-L-glycero-alpha-L-manno-nonulosonic acid cytidylyltransferase' WHERE ec = 'EC 2.7.7.81';
UPDATE enzyme SET history = 'EC 2.7.7.81 created 2012' WHERE ec = 'EC 2.7.7.81';
UPDATE enzyme SET sys_name = 'CTP:N,N-diacetyllegionaminate cytidylyltransferase' WHERE ec = 'EC 2.7.7.82';
UPDATE enzyme SET history = 'EC 2.7.7.82 created 2012' WHERE ec = 'EC 2.7.7.82';
UPDATE enzyme SET sys_name = 'UTP:N-acetyl-alpha-D-galactosamine-1-phosphate uridylyltransferase' WHERE ec = 'EC 2.7.7.83';
UPDATE enzyme SET history = 'EC 2.7.7.83 created 2012' WHERE ec = 'EC 2.7.7.83';
UPDATE enzyme SET sys_name = 'ATP:ATP adenylyltransferase (2-5 linkages-forming)' WHERE ec = 'EC 2.7.7.84';
UPDATE enzyme SET history = 'EC 2.7.7.84 created 2013' WHERE ec = 'EC 2.7.7.84';
UPDATE enzyme SET sys_name = 'ATP:ATP adenylyltransferase (cyclizing)' WHERE ec = 'EC 2.7.7.85';
UPDATE enzyme SET history = 'EC 2.7.7.85 created 2013' WHERE ec = 'EC 2.7.7.85';
UPDATE enzyme SET sys_name = 'ATP:GTP adenylyltransferase (cyclizing)' WHERE ec = 'EC 2.7.7.86';
UPDATE enzyme SET history = 'EC 2.7.7.86 created 2013' WHERE ec = 'EC 2.7.7.86';
UPDATE enzyme SET sys_name = 'ATP:L-threonyl,bicarbonate adenylyltransferase' WHERE ec = 'EC 2.7.7.87';
UPDATE enzyme SET history = 'EC 2.7.7.87 created 2013' WHERE ec = 'EC 2.7.7.87';
UPDATE enzyme SET sys_name = 'ATP:[protein] adenylyltransferase' WHERE ec = 'EC 2.7.7.n1';
UPDATE enzyme SET history = 'EC 2.7.7.n1 created 2010' WHERE ec = 'EC 2.7.7.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.7.n3';
UPDATE enzyme SET history = 'EC 2.7.7.n4 created 2011' WHERE ec = 'EC 2.7.7.n4';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.7.n5';
UPDATE enzyme SET sys_name = 'CDP-ethanolamine:1,2-diacyl-sn-glycerol ethanolaminephosphotransferase' WHERE ec = 'EC 2.7.8.1';
UPDATE enzyme SET history = 'EC 2.7.8.1 created 1961' WHERE ec = 'EC 2.7.8.1';
UPDATE enzyme SET sys_name = 'CDP-choline:1,2-diacyl-sn-glycerol cholinephosphotransferase' WHERE ec = 'EC 2.7.8.2';
UPDATE enzyme SET history = 'EC 2.7.8.2 created 1961, modified 1986 (EC 2.7.8.16 created 1983, incorporated 1986)' WHERE ec = 'EC 2.7.8.2';
UPDATE enzyme SET sys_name = 'CDP-choline:N-acylsphingosine cholinephosphotransferase' WHERE ec = 'EC 2.7.8.3';
UPDATE enzyme SET history = 'EC 2.7.8.3 created 1965' WHERE ec = 'EC 2.7.8.3';
UPDATE enzyme SET sys_name = 'CDP-ethanolamine:L-serine ethanolamine phosphotransferase' WHERE ec = 'EC 2.7.8.4';
UPDATE enzyme SET history = 'EC 2.7.8.4 created 1972, modified 1976' WHERE ec = 'EC 2.7.8.4';
UPDATE enzyme SET sys_name = 'CDP-diacylglycerol:sn-glycerol-3-phosphate 3-phosphatidyltransferase' WHERE ec = 'EC 2.7.8.5';
UPDATE enzyme SET history = 'EC 2.7.8.5 created 1972, modified 1976' WHERE ec = 'EC 2.7.8.5';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:undecaprenyl-phosphate galactose phosphotransferase' WHERE ec = 'EC 2.7.8.6';
UPDATE enzyme SET history = 'EC 2.7.8.6 created 1972' WHERE ec = 'EC 2.7.8.6';
UPDATE enzyme SET sys_name = 'CoA-(4-phosphopantetheine):apo-[acyl-carrier-protein] 4-pantetheinephosphotransferase' WHERE ec = 'EC 2.7.8.7';
UPDATE enzyme SET history = 'EC 2.7.8.7 created 1972, modified 2006' WHERE ec = 'EC 2.7.8.7';
UPDATE enzyme SET sys_name = 'CDP-diacylglycerol:L-serine 3-sn-phosphatidyltransferase' WHERE ec = 'EC 2.7.8.8';
UPDATE enzyme SET history = 'EC 2.7.8.8 created 1972, modified 1976' WHERE ec = 'EC 2.7.8.8';
UPDATE enzyme SET sys_name = 'GDP-mannose:phosphomannan mannose phosphotransferase' WHERE ec = 'EC 2.7.8.9';
UPDATE enzyme SET history = 'EC 2.7.8.9 created 1972' WHERE ec = 'EC 2.7.8.9';
UPDATE enzyme SET sys_name = 'CDP-choline:sphingosine cholinephosphotransferase' WHERE ec = 'EC 2.7.8.10';
UPDATE enzyme SET history = 'EC 2.7.8.10 created 1972, modified 1976' WHERE ec = 'EC 2.7.8.10';
UPDATE enzyme SET sys_name = 'CDP-diacylglycerol:myo-inositol 3-phosphatidyltransferase' WHERE ec = 'EC 2.7.8.11';
UPDATE enzyme SET history = 'EC 2.7.8.11 created 1972, modified 1976' WHERE ec = 'EC 2.7.8.11';
UPDATE enzyme SET sys_name = 'CDP-glycerol:poly(glycerophosphate) glycerophosphotransferase' WHERE ec = 'EC 2.7.8.12';
UPDATE enzyme SET history = 'EC 2.7.8.12 created 1972, modified 1982' WHERE ec = 'EC 2.7.8.12';
UPDATE enzyme SET sys_name = 'UDP-MurAc(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala):undecaprenyl-phosphate phospho-N-acetylmuramoyl-pentapeptide-transferase' WHERE ec = 'EC 2.7.8.13';
UPDATE enzyme SET history = 'EC 2.7.8.13 created 1972, modified 2002' WHERE ec = 'EC 2.7.8.13';
UPDATE enzyme SET sys_name = 'CDP-ribitol:poly(ribitol phosphate) ribitolphosphotransferase' WHERE ec = 'EC 2.7.8.14';
UPDATE enzyme SET history = 'EC 2.7.8.14 created 1972 as EC 2.4.1.55, transferred 1982 to EC 2.7.8.14' WHERE ec = 'EC 2.7.8.14';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:dolichyl-phosphate N-acetyl-D-glucosaminephosphotransferase' WHERE ec = 'EC 2.7.8.15';
UPDATE enzyme SET history = 'EC 2.7.8.15 created 1983' WHERE ec = 'EC 2.7.8.15';
UPDATE enzyme SET history = 'EC 2.7.8.16 created 1983, deleted 1986' WHERE ec = 'EC 2.7.8.16';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-D-glucosamine:lysosomal-enzyme N-acetylglucosaminephosphotransferase' WHERE ec = 'EC 2.7.8.17';
UPDATE enzyme SET history = 'EC 2.7.8.17 created 1984' WHERE ec = 'EC 2.7.8.17';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactose:UDP-N-acetyl-D-glucosamine galactose phosphotransferase' WHERE ec = 'EC 2.7.8.18';
UPDATE enzyme SET history = 'EC 2.7.8.18 created 1986' WHERE ec = 'EC 2.7.8.18';
UPDATE enzyme SET sys_name = 'UDP-glucose:glycoprotein-D-mannose glucosephosphotransferase' WHERE ec = 'EC 2.7.8.19';
UPDATE enzyme SET history = 'EC 2.7.8.19 created 1986' WHERE ec = 'EC 2.7.8.19';
UPDATE enzyme SET sys_name = 'Phosphatidylglycerol:membrane-derived-oligosaccharide-D-glucose glycerophosphotransferase' WHERE ec = 'EC 2.7.8.20';
UPDATE enzyme SET history = 'EC 2.7.8.20 created 1986' WHERE ec = 'EC 2.7.8.20';
UPDATE enzyme SET sys_name = 'Membrane-derived-oligosaccharide-6-(glycerophospho)-D-glucose:membrane-derived-oligosaccharide-D-glucose glycerophosphotransferase' WHERE ec = 'EC 2.7.8.21';
UPDATE enzyme SET history = 'EC 2.7.8.21 created 1986' WHERE ec = 'EC 2.7.8.21';
UPDATE enzyme SET sys_name = 'CDP-choline:1-alkenyl-2-acylglycerol cholinephosphotransferase' WHERE ec = 'EC 2.7.8.22';
UPDATE enzyme SET history = 'EC 2.7.8.22 created 1990' WHERE ec = 'EC 2.7.8.22';
UPDATE enzyme SET sys_name = '1-carboxyvinyl carboxyphosphonate phosphorylmutase (decarboxylating)' WHERE ec = 'EC 2.7.8.23';
UPDATE enzyme SET history = 'EC 2.7.8.23 created 1999' WHERE ec = 'EC 2.7.8.23';
UPDATE enzyme SET sys_name = 'CDP-diacylglycerol:choline O-phosphatidyltransferase' WHERE ec = 'EC 2.7.8.24';
UPDATE enzyme SET history = 'EC 2.7.8.24 created 2001' WHERE ec = 'EC 2.7.8.24';
UPDATE enzyme SET history = 'EC 2.7.8.25 created 2002, modified 2008, deleted 2013' WHERE ec = 'EC 2.7.8.25';
UPDATE enzyme SET sys_name = 'Adenosylcobinamide-GDP:alpha-ribazole ribazoletransferase' WHERE ec = 'EC 2.7.8.26';
UPDATE enzyme SET history = 'EC 2.7.8.26 created 2004' WHERE ec = 'EC 2.7.8.26';
UPDATE enzyme SET sys_name = 'Ceramide:phosphatidylcholine cholinephosphotransferase' WHERE ec = 'EC 2.7.8.27';
UPDATE enzyme SET history = 'EC 2.7.8.27 created 2006' WHERE ec = 'EC 2.7.8.27';
UPDATE enzyme SET sys_name = '(2S)-lactyl-2-diphospho-5-guanosine:7,8-didemethyl-8-hydroxy-5-deazariboflavin 2-phospho-L-lactate transferase' WHERE ec = 'EC 2.7.8.28';
UPDATE enzyme SET history = 'EC 2.7.8.28 created 2010' WHERE ec = 'EC 2.7.8.28';
UPDATE enzyme SET sys_name = 'L-1-phosphatidylethanolamine:L-serine phosphatidyltransferase' WHERE ec = 'EC 2.7.8.29';
UPDATE enzyme SET history = 'EC 2.7.8.29 created 2010' WHERE ec = 'EC 2.7.8.29';
UPDATE enzyme SET history = 'EC 2.7.8.30 created 2010, modified 2011, deleted 2013' WHERE ec = 'EC 2.7.8.30';
UPDATE enzyme SET sys_name = 'UDP-glucose:ditrans,octacis-undecaprenyl-phosphate glucose phosphotransferase' WHERE ec = 'EC 2.7.8.31';
UPDATE enzyme SET history = 'EC 2.7.8.31 created 2010' WHERE ec = 'EC 2.7.8.31';
UPDATE enzyme SET sys_name = 'UDP-D-xylose:3-O-alpha-D-mannopyranosyl-alpha-D-mannopyranose xylosylphosphotransferase' WHERE ec = 'EC 2.7.8.32';
UPDATE enzyme SET history = 'EC 2.7.8.32 created 2011' WHERE ec = 'EC 2.7.8.32';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:ditrans,octacis-undecaprenyl phosphate N-acetyl-alpha-D-glucosaminephosphotransferase' WHERE ec = 'EC 2.7.8.33';
UPDATE enzyme SET history = 'EC 2.7.8.33 created 2011' WHERE ec = 'EC 2.7.8.33';
UPDATE enzyme SET sys_name = 'CDP-1L-myo-inositol:1L-myo-inositol 1-phosphate myo-inositolphosphotransferase' WHERE ec = 'EC 2.7.8.34';
UPDATE enzyme SET history = 'EC 2.7.8.34 created 2011' WHERE ec = 'EC 2.7.8.34';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine:trans,octacis-decaprenyl-phosphate N-acetylglucosaminephosphotransferase' WHERE ec = 'EC 2.7.8.35';
UPDATE enzyme SET history = 'EC 2.7.8.35 created 2012' WHERE ec = 'EC 2.7.8.35';
UPDATE enzyme SET sys_name = 'UDP-N,N-diacetylbacillosamine:tritrans,heptacis-undecaprenyl-phosphate N,N-diacetylbacillosamine transferase' WHERE ec = 'EC 2.7.8.36';
UPDATE enzyme SET history = 'EC 2.7.8.36 created 2012' WHERE ec = 'EC 2.7.8.36';
UPDATE enzyme SET sys_name = 'ATP:methylphosphonate 5-triphosphoribosyltransferase' WHERE ec = 'EC 2.7.8.37';
UPDATE enzyme SET history = 'EC 2.7.8.37 created 2012' WHERE ec = 'EC 2.7.8.37';
UPDATE enzyme SET sys_name = 'CDP-2,3-bis-(O-geranylgeranyl)-sn-glycerol:L-serine 2,3-bis-(O-geranylgeranyl)-sn-glycerol phosphotransferase' WHERE ec = 'EC 2.7.8.38';
UPDATE enzyme SET history = 'EC 2.7.8.38 created 2013, modified 2013' WHERE ec = 'EC 2.7.8.38';
UPDATE enzyme SET sys_name = 'CDP-2,3-bis-(O-phytanyl)-sn-glycerol:1L-myo-inositol 1-phosphate 1-sn-archaetidyltransferase' WHERE ec = 'EC 2.7.8.39';
UPDATE enzyme SET history = 'EC 2.7.8.39 created 2013' WHERE ec = 'EC 2.7.8.39';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-galactosamine:ditrans,octacis-undecaprenyl phosphate N-acetyl-D-galactosaminephosphotransferase' WHERE ec = 'EC 2.7.8.40';
UPDATE enzyme SET history = 'EC 2.7.8.40 created 2013' WHERE ec = 'EC 2.7.8.40';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.8.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.8.n2';
UPDATE enzyme SET sys_name = 'ATP:pyruvate, phosphate phosphotransferase' WHERE ec = 'EC 2.7.9.1';
UPDATE enzyme SET history = 'EC 2.7.9.1 created 1972' WHERE ec = 'EC 2.7.9.1';
UPDATE enzyme SET sys_name = 'ATP:pyruvate, water phosphotransferase' WHERE ec = 'EC 2.7.9.2';
UPDATE enzyme SET history = 'EC 2.7.9.2 created 1976' WHERE ec = 'EC 2.7.9.2';
UPDATE enzyme SET sys_name = 'ATP:selenide, water phosphotransferase' WHERE ec = 'EC 2.7.9.3';
UPDATE enzyme SET history = 'EC 2.7.9.3 created 1999' WHERE ec = 'EC 2.7.9.3';
UPDATE enzyme SET sys_name = 'ATP:alpha-glucan, water phosphotransferase' WHERE ec = 'EC 2.7.9.4';
UPDATE enzyme SET history = 'EC 2.7.9.4 created 2002' WHERE ec = 'EC 2.7.9.4';
UPDATE enzyme SET sys_name = 'ATP:phospho-alpha-glucan, water phosphotransferase' WHERE ec = 'EC 2.7.9.5';
UPDATE enzyme SET history = 'EC 2.7.9.5 created 2005' WHERE ec = 'EC 2.7.9.5';
UPDATE enzyme SET sys_name = 'ATP:[protein]-L-tyrosine O-phosphotransferase (receptor-type)' WHERE ec = 'EC 2.7.10.1';
UPDATE enzyme SET history = 'EC 2.7.10.1 created 1984 as EC 2.7.1.112, part-transferred 2005 to EC 2.7.10.1' WHERE ec = 'EC 2.7.10.1';
UPDATE enzyme SET sys_name = 'ATP:[protein]-L-tyrosine O-phosphotransferase (non-specific)' WHERE ec = 'EC 2.7.10.2';
UPDATE enzyme SET history = 'EC 2.7.10.2 created 1984 as EC 2.7.1.112, part-transferred 2005 to EC 2.7.10.2' WHERE ec = 'EC 2.7.10.2';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (non-specific)' WHERE ec = 'EC 2.7.11.1';
UPDATE enzyme SET history = 'EC 2.7.11.1 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.1';
UPDATE enzyme SET sys_name = 'ATP:[pyruvate dehydrogenase (acetyl-transferring)] phosphotransferase' WHERE ec = 'EC 2.7.11.2';
UPDATE enzyme SET history = 'EC 2.7.11.2 created 1978 as EC 2.7.1.99, transferred 2005 to EC 2.7.11.2' WHERE ec = 'EC 2.7.11.2';
UPDATE enzyme SET sys_name = 'ATP:dephospho-([hydroxymethylglutaryl-CoA reductase (NADPH)] kinase) phosphotransferase' WHERE ec = 'EC 2.7.11.3';
UPDATE enzyme SET history = 'EC 2.7.11.3 created 1984 as EC 2.7.1.110, transferred 2005 to EC 2.7.11.3' WHERE ec = 'EC 2.7.11.3';
UPDATE enzyme SET sys_name = 'ATP:[3-methyl-2-oxobutanoate dehydrogenase (acetyl-transferring)] phosphotransferase' WHERE ec = 'EC 2.7.11.4';
UPDATE enzyme SET history = 'EC 2.7.11.3 created 1986 as EC 2.7.1.115, transferred 2005 to EC 2.7.11.3' WHERE ec = 'EC 2.7.11.4';
UPDATE enzyme SET sys_name = 'ATP:[isocitrate dehydrogenase (NADP(+))] phosphotransferase' WHERE ec = 'EC 2.7.11.5';
UPDATE enzyme SET history = 'EC 2.7.11.5 created 1986 as EC 2.7.1.116, transferred  2005 to EC 2.7.11.5' WHERE ec = 'EC 2.7.11.5';
UPDATE enzyme SET sys_name = 'ATP:[tyrosine-3-monoxygenase] phosphotransferase' WHERE ec = 'EC 2.7.11.6';
UPDATE enzyme SET history = 'EC 2.7.11.6 created 1989 as EC 2.7.1.124, transferred 2005 to EC 2.7.11.6' WHERE ec = 'EC 2.7.11.6';
UPDATE enzyme SET sys_name = 'ATP:[myosin heavy-chain] O-phosphotransferase' WHERE ec = 'EC 2.7.11.7';
UPDATE enzyme SET history = 'EC 2.7.11.7 created 1990 as EC 2.7.1.129, transferred 2005 to EC 2.7.11.7' WHERE ec = 'EC 2.7.11.7';
UPDATE enzyme SET sys_name = 'ATP:[Fas-activated serine/threonine protein] phosphotransferase' WHERE ec = 'EC 2.7.11.8';
UPDATE enzyme SET history = 'EC 2.7.11.8 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.8';
UPDATE enzyme SET sys_name = 'ATP:[Goodpasture antigen-binding protein] phosphotransferase' WHERE ec = 'EC 2.7.11.9';
UPDATE enzyme SET history = 'EC 2.7.11.9 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.9';
UPDATE enzyme SET sys_name = 'ATP:[I-kappa-B protein] phosphotransferase' WHERE ec = 'EC 2.7.11.10';
UPDATE enzyme SET history = 'EC 2.7.11.10 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.10';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (cAMP-dependent)' WHERE ec = 'EC 2.7.11.11';
UPDATE enzyme SET history = 'EC 2.7.11.11 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.11';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (cGMP-dependent)' WHERE ec = 'EC 2.7.11.12';
UPDATE enzyme SET history = 'EC 2.7.11.12 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.12';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (diacylglycerol-dependent)' WHERE ec = 'EC 2.7.11.13';
UPDATE enzyme SET history = 'EC 2.7.11.13 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.13';
UPDATE enzyme SET sys_name = 'ATP:rhodopsin phosphotransferase' WHERE ec = 'EC 2.7.11.14';
UPDATE enzyme SET history = 'EC 2.7.11.14 created 1989 as EC 2.7.1.125 (EC 2.7.1.97 created 1978, incorporated 1992), transferred 2005 to EC 2.7.11.14' WHERE ec = 'EC 2.7.11.14';
UPDATE enzyme SET sys_name = 'ATP:[beta-adrenergic-receptor] phosphotransferase' WHERE ec = 'EC 2.7.11.15';
UPDATE enzyme SET history = 'EC 2.7.11.15 created 1989 as EC 2.7.1.126, transferred 2005 to EC 2.7.11.15' WHERE ec = 'EC 2.7.11.15';
UPDATE enzyme SET sys_name = 'ATP:[G-protein-coupled receptor] phosphotransferase' WHERE ec = 'EC 2.7.11.16';
UPDATE enzyme SET history = 'EC 2.7.11.16 created 2005' WHERE ec = 'EC 2.7.11.16';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (Ca(2+)/calmodulin-dependent)' WHERE ec = 'EC 2.7.11.17';
UPDATE enzyme SET history = 'EC 2.7.11.17 created 1989 as EC 2.7.1.123, transferred 2005 to EC 2.7.11.17 (EC 2.7.1.120 incorporated 2005)' WHERE ec = 'EC 2.7.11.17';
UPDATE enzyme SET sys_name = 'ATP:[myosin-light-chain] O-phosphotransferase' WHERE ec = 'EC 2.7.11.18';
UPDATE enzyme SET history = 'EC 2.7.11.18 created 1986 as EC 2.7.1.117, transferred 2005 to EC 2.7.11.18' WHERE ec = 'EC 2.7.11.18';
UPDATE enzyme SET sys_name = 'ATP:phosphorylase-b phosphotransferase' WHERE ec = 'EC 2.7.11.19';
UPDATE enzyme SET history = 'EC 2.7.11.9 created 1961 as EC 2.7.1.38, transferred 2005 to EC 2.7.11.9' WHERE ec = 'EC 2.7.11.19';
UPDATE enzyme SET sys_name = 'ATP:[elongation factor 2] phosphotransferase' WHERE ec = 'EC 2.7.11.20';
UPDATE enzyme SET history = 'EC 2.7.11.20 created 2005' WHERE ec = 'EC 2.7.11.20';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (spindle-pole-dependent)' WHERE ec = 'EC 2.7.11.21';
UPDATE enzyme SET history = 'EC 2.7.11.21 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.21';
UPDATE enzyme SET sys_name = 'ATP:cyclin phosphotransferase' WHERE ec = 'EC 2.7.11.22';
UPDATE enzyme SET history = 'EC 2.7.11.22 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.22';
UPDATE enzyme SET sys_name = 'ATP:[DNA-directed RNA polymerase] phosphotransferase' WHERE ec = 'EC 2.7.11.23';
UPDATE enzyme SET history = 'EC 2.7.11.23 created 1992 as EC 2.7.1.141, transferred 2005 to EC 2.7.11.23' WHERE ec = 'EC 2.7.11.23';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (MAPKK-activated)' WHERE ec = 'EC 2.7.11.24';
UPDATE enzyme SET history = 'EC 2.7.11.24 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.24';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (MAPKKKK-activated)' WHERE ec = 'EC 2.7.11.25';
UPDATE enzyme SET history = 'EC 2.7.11.25 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.25';
UPDATE enzyme SET sys_name = 'ATP:[tau-protein] O-phosphotransferase' WHERE ec = 'EC 2.7.11.26';
UPDATE enzyme SET history = 'EC 2.7.11.26 created 1990 as EC 2.7.1.135, transferred 2005 to EC 2.7.11.26' WHERE ec = 'EC 2.7.11.26';
UPDATE enzyme SET sys_name = 'ATP:[acetyl-CoA carboxylase] phosphotransferase' WHERE ec = 'EC 2.7.11.27';
UPDATE enzyme SET history = 'EC 2.7.11.27 created 1990 as EC 2.7.1.128 (EC 2.7.1.111 created 1984, incorporated 1992), transferred 2005 to EC 2.7.11.27' WHERE ec = 'EC 2.7.11.27';
UPDATE enzyme SET sys_name = 'ATP:tropomyosin O-phosphotransferase' WHERE ec = 'EC 2.7.11.28';
UPDATE enzyme SET history = 'EC 2.7.11.28 created 1990 as EC 2.7.1.132, transferred 2005 to EC 2.7.11.28' WHERE ec = 'EC 2.7.11.28';
UPDATE enzyme SET sys_name = 'ATP:[low-density-lipoprotein receptor]-L-serine O-phosphotransferase' WHERE ec = 'EC 2.7.11.29';
UPDATE enzyme SET history = 'EC 2.7.11.29 created 1990 as EC 2.7.1.131, transfered 2005 to EC 2.7.11.29' WHERE ec = 'EC 2.7.11.29';
UPDATE enzyme SET sys_name = 'ATP:[receptor-protein] phosphotransferase' WHERE ec = 'EC 2.7.11.30';
UPDATE enzyme SET history = 'EC 2.7.11.30 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.11.30';
UPDATE enzyme SET sys_name = 'ATP:[hydroxymethylglutaryl-CoA reductase (NADPH)] phosphotransferase' WHERE ec = 'EC 2.7.11.31';
UPDATE enzyme SET history = 'EC 2.7.11.31 created 1984 as EC 2.7.1.109, transferred 2005 to EC 2.7.11.31' WHERE ec = 'EC 2.7.11.31';
UPDATE enzyme SET sys_name = 'ADP:[pyruvate, phosphate dikinase] phosphotransferase' WHERE ec = 'EC 2.7.11.32';
UPDATE enzyme SET history = 'EC 2.7.11.32 created 2012' WHERE ec = 'EC 2.7.11.32';
UPDATE enzyme SET sys_name = 'ADP:[pyruvate, water dikinase] phosphotransferase' WHERE ec = 'EC 2.7.11.33';
UPDATE enzyme SET history = 'EC 2.7.11.33 created 2012' WHERE ec = 'EC 2.7.11.33';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.11.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.7.11.n2';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (Ser/Thr- and Tyr-phosphorylating)' WHERE ec = 'EC 2.7.12.1';
UPDATE enzyme SET history = 'EC 2.7.12.1 created 2005 (EC 2.7.1.37 part-incorporated 2005)' WHERE ec = 'EC 2.7.12.1';
UPDATE enzyme SET sys_name = 'ATP:protein phosphotransferase (MAPKKK-activated)' WHERE ec = 'EC 2.7.12.2';
UPDATE enzyme SET history = 'EC 2.7.12.2 created 2005' WHERE ec = 'EC 2.7.12.2';
UPDATE enzyme SET sys_name = 'ATP:protein-L-histidine N(pros)-phosphotransferase' WHERE ec = 'EC 2.7.13.1';
UPDATE enzyme SET history = 'EC 2.7.13.1 created 1989 as EC 2.7.3.11, transferred 2005 to EC 2.7.13.1' WHERE ec = 'EC 2.7.13.1';
UPDATE enzyme SET sys_name = 'ATP:protein-L-histidine N(tele)-phosphotransferase' WHERE ec = 'EC 2.7.13.2';
UPDATE enzyme SET history = 'EC 2.7.13.2 created 1989 as EC 2.7.3.12, transferred 2005 to EC 2.7.13.2' WHERE ec = 'EC 2.7.13.2';
UPDATE enzyme SET sys_name = 'ATP:protein-L-histidine N-phosphotransferase' WHERE ec = 'EC 2.7.13.3';
UPDATE enzyme SET history = 'EC 2.7.13.3 created 2005' WHERE ec = 'EC 2.7.13.3';
UPDATE enzyme SET sys_name = 'Triphosphate:[microsomal-membrane-protein] phosphotransferase' WHERE ec = 'EC 2.7.99.1';
UPDATE enzyme SET history = 'EC 2.7.99.1 created 1983 as EC 2.7.1.104, transferred 2005 to EC 2.7.99.1' WHERE ec = 'EC 2.7.99.1';
UPDATE enzyme SET sys_name = 'Thiosulfate:cyanide sulfurtransferase' WHERE ec = 'EC 2.8.1.1';
UPDATE enzyme SET history = 'EC 2.8.1.1 created 1961' WHERE ec = 'EC 2.8.1.1';
UPDATE enzyme SET sys_name = '3-mercaptopyruvate:cyanide sulfurtransferase' WHERE ec = 'EC 2.8.1.2';
UPDATE enzyme SET history = 'EC 2.8.1.2 created 1961' WHERE ec = 'EC 2.8.1.2';
UPDATE enzyme SET sys_name = 'Thiosulfate:thiol sulfurtransferase' WHERE ec = 'EC 2.8.1.3';
UPDATE enzyme SET history = 'EC 2.8.1.3 created 1982' WHERE ec = 'EC 2.8.1.3';
UPDATE enzyme SET sys_name = 'L-cysteine:tRNA sulfurtransferase' WHERE ec = 'EC 2.8.1.4';
UPDATE enzyme SET history = 'EC 2.8.1.4 created 1984' WHERE ec = 'EC 2.8.1.4';
UPDATE enzyme SET sys_name = 'Thiosulfate:dithioerythritol sulfurtransferase' WHERE ec = 'EC 2.8.1.5';
UPDATE enzyme SET history = 'EC 2.8.1.5 created 1989, modified 1999' WHERE ec = 'EC 2.8.1.5';
UPDATE enzyme SET sys_name = 'Dethiobiotin:sulfur-(sulfur carrier) sulfurtransferase' WHERE ec = 'EC 2.8.1.6';
UPDATE enzyme SET history = 'EC 2.8.1.6 created 1999, modified 2006, modified 2011' WHERE ec = 'EC 2.8.1.6';
UPDATE enzyme SET sys_name = 'L-cysteine:acceptor sulfurtransferase' WHERE ec = 'EC 2.8.1.7';
UPDATE enzyme SET history = 'EC 2.8.1.7 created 2003, modified 2011' WHERE ec = 'EC 2.8.1.7';
UPDATE enzyme SET sys_name = 'Protein N(6)-(octanoyl)lysine:sulfur-(sulfur carrier) sulfurtransferase' WHERE ec = 'EC 2.8.1.8';
UPDATE enzyme SET history = 'EC 2.8.1.8 created 2006' WHERE ec = 'EC 2.8.1.8';
UPDATE enzyme SET sys_name = 'L-cysteine:molybdenum cofactor sulfurtransferase' WHERE ec = 'EC 2.8.1.9';
UPDATE enzyme SET history = 'EC 2.8.1.9 created 2011' WHERE ec = 'EC 2.8.1.9';
UPDATE enzyme SET sys_name = '1-deoxy-D-xylulose 5-phosphate:thiol sulfurtransferase' WHERE ec = 'EC 2.8.1.10';
UPDATE enzyme SET history = 'EC 2.8.1.10 created 2011' WHERE ec = 'EC 2.8.1.10';
UPDATE enzyme SET sys_name = 'Persulfurated L-cysteine desulfurase:[molybdopterin-synthase sulfur-carrier protein]-Gly-Gly sulfurtransferase' WHERE ec = 'EC 2.8.1.11';
UPDATE enzyme SET history = 'EC 2.8.1.11 created 2011' WHERE ec = 'EC 2.8.1.11';
UPDATE enzyme SET sys_name = 'Thiocarboxylated molybdopterin synthase:cyclic pyranopterin phosphate sulfurtransferase' WHERE ec = 'EC 2.8.1.12';
UPDATE enzyme SET history = 'EC 2.8.1.12 created 2011' WHERE ec = 'EC 2.8.1.12';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.8.1.n1';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:phenol sulfotransferase' WHERE ec = 'EC 2.8.2.1';
UPDATE enzyme SET history = 'EC 2.8.2.1 created 1961, modified 1980' WHERE ec = 'EC 2.8.2.1';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:alcohol sulfotransferase' WHERE ec = 'EC 2.8.2.2';
UPDATE enzyme SET history = 'EC 2.8.2.2 created 1961, modified 1980' WHERE ec = 'EC 2.8.2.2';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:amine N-sulfotransferase' WHERE ec = 'EC 2.8.2.3';
UPDATE enzyme SET history = 'EC 2.8.2.3 created 1965' WHERE ec = 'EC 2.8.2.3';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:estrone 3-sulfotransferase' WHERE ec = 'EC 2.8.2.4';
UPDATE enzyme SET history = 'EC 2.8.2.4 created 1965' WHERE ec = 'EC 2.8.2.4';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:chondroitin 4-sulfotransferase' WHERE ec = 'EC 2.8.2.5';
UPDATE enzyme SET history = 'EC 2.8.2.5 created 1965, modified 1986' WHERE ec = 'EC 2.8.2.5';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:choline sulfotransferase' WHERE ec = 'EC 2.8.2.6';
UPDATE enzyme SET history = 'EC 2.8.2.6 created 1972' WHERE ec = 'EC 2.8.2.6';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:UDP-N-acetyl-D-galactosamine-4-sulfate 6-sulfotransferase' WHERE ec = 'EC 2.8.2.7';
UPDATE enzyme SET history = 'EC 2.8.2.7 created 1972' WHERE ec = 'EC 2.8.2.7';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:[heparan sulfate]-glucosamine N-sulfotransferase' WHERE ec = 'EC 2.8.2.8';
UPDATE enzyme SET history = 'EC 2.8.2.8 created 1972, modified 2001 (EC 2.8.2.12 created 1972, incorporated 2001)' WHERE ec = 'EC 2.8.2.8';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:L-tyrosine-methyl-ester sulfotransferase' WHERE ec = 'EC 2.8.2.9';
UPDATE enzyme SET history = 'EC 2.8.2.9 created 1972, deleted 1980, reinstated 1984' WHERE ec = 'EC 2.8.2.9';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:Renilla luciferin sulfotransferase' WHERE ec = 'EC 2.8.2.10';
UPDATE enzyme SET history = 'EC 2.8.2.10 created 1972, modified 1982' WHERE ec = 'EC 2.8.2.10';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:galactosylceramide 3-sulfotransferase' WHERE ec = 'EC 2.8.2.11';
UPDATE enzyme SET history = 'EC 2.8.2.11 created 1972, modified 1976' WHERE ec = 'EC 2.8.2.11';
UPDATE enzyme SET history = 'EC 2.8.2.12 created 1972, deleted 2001' WHERE ec = 'EC 2.8.2.12';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:galactosylsphingosine sulfotransferase' WHERE ec = 'EC 2.8.2.13';
UPDATE enzyme SET history = 'EC 2.8.2.13 created 1976' WHERE ec = 'EC 2.8.2.13';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:glycolithocholate sulfotransferase' WHERE ec = 'EC 2.8.2.14';
UPDATE enzyme SET history = 'EC 2.8.2.14 created 1978, modified 2005' WHERE ec = 'EC 2.8.2.14';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:phenolic-steroid sulfotransferase' WHERE ec = 'EC 2.8.2.15';
UPDATE enzyme SET history = 'EC 2.8.2.15 created 1984' WHERE ec = 'EC 2.8.2.15';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:thiol S-sulfotransferase' WHERE ec = 'EC 2.8.2.16';
UPDATE enzyme SET history = 'EC 2.8.2.16 created 1984' WHERE ec = 'EC 2.8.2.16';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:chondroitin 6-sulfotransferase' WHERE ec = 'EC 2.8.2.17';
UPDATE enzyme SET history = 'EC 2.8.2.17 created 1986' WHERE ec = 'EC 2.8.2.17';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:cortisol 21-sulfotransferase' WHERE ec = 'EC 2.8.2.18';
UPDATE enzyme SET history = 'EC 2.8.2.18 created 1986' WHERE ec = 'EC 2.8.2.18';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:triglucosyl-1-O-alkyl-2-O-acylglycerol 6-sulfotransferase' WHERE ec = 'EC 2.8.2.19';
UPDATE enzyme SET history = 'EC 2.8.2.19 created 1986' WHERE ec = 'EC 2.8.2.19';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:protein-tyrosine O-sulfotransferase' WHERE ec = 'EC 2.8.2.20';
UPDATE enzyme SET history = 'EC 2.8.2.20 created 1986' WHERE ec = 'EC 2.8.2.20';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:keratan 6-sulfotransferase' WHERE ec = 'EC 2.8.2.21';
UPDATE enzyme SET history = 'EC 2.8.2.21 created 1989' WHERE ec = 'EC 2.8.2.21';
UPDATE enzyme SET sys_name = 'Aryl-sulfate:phenol sulfotransferase' WHERE ec = 'EC 2.8.2.22';
UPDATE enzyme SET history = 'EC 2.8.2.22 created 1990' WHERE ec = 'EC 2.8.2.22';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:[heparan sulfate]-glucosamine 3-sulfotransferase' WHERE ec = 'EC 2.8.2.23';
UPDATE enzyme SET history = 'EC 2.8.2.23 created 1992, modified 2001' WHERE ec = 'EC 2.8.2.23';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:desulfoglucosinolate sulfotransferase' WHERE ec = 'EC 2.8.2.24';
UPDATE enzyme SET history = 'EC 2.8.2.24 created 1992' WHERE ec = 'EC 2.8.2.24';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:quercetin 3-sulfotransferase' WHERE ec = 'EC 2.8.2.25';
UPDATE enzyme SET history = 'EC 2.8.2.25 created 1992' WHERE ec = 'EC 2.8.2.25';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:quercetin-3-sulfate 3-sulfotransferase' WHERE ec = 'EC 2.8.2.26';
UPDATE enzyme SET history = 'EC 2.8.2.26 created 1992' WHERE ec = 'EC 2.8.2.26';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:quercetin-3-sulfate 4-sulfotransferase' WHERE ec = 'EC 2.8.2.27';
UPDATE enzyme SET history = 'EC 2.8.2.27 created 1992' WHERE ec = 'EC 2.8.2.27';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:quercetin-3,3-bissulfate 7-sulfotransferase' WHERE ec = 'EC 2.8.2.28';
UPDATE enzyme SET history = 'EC 2.8.2.28 created 1992' WHERE ec = 'EC 2.8.2.28';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:[heparan sulfate]-glucosamine 3-sulfotransferase' WHERE ec = 'EC 2.8.2.29';
UPDATE enzyme SET history = 'EC 2.8.2.29 created 2001' WHERE ec = 'EC 2.8.2.29';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:[heparan sulfate]-glucosamine 3-sulfotransferase' WHERE ec = 'EC 2.8.2.30';
UPDATE enzyme SET history = 'EC 2.8.2.30 created 2001' WHERE ec = 'EC 2.8.2.30';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:5-alpha-cholan-3-alpha,7-alpha,12-alpha,24-tetrol sulfotransferase' WHERE ec = 'EC 2.8.2.31';
UPDATE enzyme SET history = 'EC 2.8.2.31 created 2004' WHERE ec = 'EC 2.8.2.31';
UPDATE enzyme SET sys_name = '3-phosphoadenosine 5-phosphosulfate:5-beta-scymnol sulfotransferase' WHERE ec = 'EC 2.8.2.32';
UPDATE enzyme SET history = 'EC 2.8.2.32 created 2005' WHERE ec = 'EC 2.8.2.32';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:[dermatan]-4-O-sulfo-N-acetylgalactosamine 6-O-sulfotransferase' WHERE ec = 'EC 2.8.2.33';
UPDATE enzyme SET history = 'EC 2.8.2.33 created 2005, modified 2010' WHERE ec = 'EC 2.8.2.33';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:glycochenodeoxycholate 7-sulfotransferase' WHERE ec = 'EC 2.8.2.34';
UPDATE enzyme SET history = 'EC 2.8.2.34 created 2005' WHERE ec = 'EC 2.8.2.34';
UPDATE enzyme SET sys_name = '3-phospho-5-adenylyl sulfate:[dermatan]-N-acetylgalactosamine 4-sulfotransferase' WHERE ec = 'EC 2.8.2.35';
UPDATE enzyme SET history = 'EC 2.8.2.35 created 2010' WHERE ec = 'EC 2.8.2.35';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl-sulfate:desulfo-A47934 sulfotransferase' WHERE ec = 'EC 2.8.2.36';
UPDATE enzyme SET history = 'EC 2.8.2.36 created 2014' WHERE ec = 'EC 2.8.2.36';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 2.8.2.n1';
UPDATE enzyme SET sys_name = '3-phosphoadenylyl sulfate:thyroxine sulfotransferase' WHERE ec = 'EC 2.8.2.n2';
UPDATE enzyme SET history = 'EC 2.8.2.n2' WHERE ec = 'EC 2.8.2.n2';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:propanoate CoA-transferase' WHERE ec = 'EC 2.8.3.1';
UPDATE enzyme SET history = 'EC 2.8.3.1 created 1961' WHERE ec = 'EC 2.8.3.1';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:oxalate CoA-transferase' WHERE ec = 'EC 2.8.3.2';
UPDATE enzyme SET history = 'EC 2.8.3.2 created 1961' WHERE ec = 'EC 2.8.3.2';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:malonate CoA-transferase' WHERE ec = 'EC 2.8.3.3';
UPDATE enzyme SET history = 'EC 2.8.3.3 created 1961' WHERE ec = 'EC 2.8.3.3';
UPDATE enzyme SET history = 'EC 2.8.3.4 created 1961, deleted 1964' WHERE ec = 'EC 2.8.3.4';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:3-oxo-acid CoA-transferase' WHERE ec = 'EC 2.8.3.5';
UPDATE enzyme SET history = 'EC 2.8.3.5 created 1961, modified 1980' WHERE ec = 'EC 2.8.3.5';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:3-oxoadipate CoA-transferase' WHERE ec = 'EC 2.8.3.6';
UPDATE enzyme SET history = 'EC 2.8.3.6 created 1961' WHERE ec = 'EC 2.8.3.6';
UPDATE enzyme SET history = 'EC 2.8.3.7 created 1972, deleted 2014' WHERE ec = 'EC 2.8.3.7';
UPDATE enzyme SET sys_name = 'Acyl-CoA:acetate CoA-transferase' WHERE ec = 'EC 2.8.3.8';
UPDATE enzyme SET history = 'EC 2.8.3.8 created 1972' WHERE ec = 'EC 2.8.3.8';
UPDATE enzyme SET sys_name = 'Butanoyl-CoA:acetoacetate CoA-transferase' WHERE ec = 'EC 2.8.3.9';
UPDATE enzyme SET history = 'EC 2.8.3.9 created 1984' WHERE ec = 'EC 2.8.3.9';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:citrate CoA-transferase' WHERE ec = 'EC 2.8.3.10';
UPDATE enzyme SET history = 'EC 2.8.3.10 created 1984' WHERE ec = 'EC 2.8.3.10';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:citramalate CoA-transferase' WHERE ec = 'EC 2.8.3.11';
UPDATE enzyme SET history = 'EC 2.8.3.11 created 1984' WHERE ec = 'EC 2.8.3.11';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:(E)-glutaconate CoA-transferase' WHERE ec = 'EC 2.8.3.12';
UPDATE enzyme SET history = 'EC 2.8.3.12 created 1984, modified 2002' WHERE ec = 'EC 2.8.3.12';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:3-hydroxy-3-methylglutarate CoA-transferase' WHERE ec = 'EC 2.8.3.13';
UPDATE enzyme SET history = 'EC 2.8.3.13 created 1984' WHERE ec = 'EC 2.8.3.13';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:5-hydroxypentanoate CoA-transferase' WHERE ec = 'EC 2.8.3.14';
UPDATE enzyme SET history = 'EC 2.8.3.14 created 1992' WHERE ec = 'EC 2.8.3.14';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:(R)-2-benzylsuccinate CoA-transferase' WHERE ec = 'EC 2.8.3.15';
UPDATE enzyme SET history = 'EC 2.8.3.15 created 2003' WHERE ec = 'EC 2.8.3.15';
UPDATE enzyme SET sys_name = 'Formyl-CoA:oxalate CoA-transferase' WHERE ec = 'EC 2.8.3.16';
UPDATE enzyme SET history = 'EC 2.8.3.16 created 2003' WHERE ec = 'EC 2.8.3.16';
UPDATE enzyme SET sys_name = '(E)-cinnamoyl-CoA:(R)-phenyllactate CoA-transferase' WHERE ec = 'EC 2.8.3.17';
UPDATE enzyme SET history = 'EC 2.8.3.17 created 2003' WHERE ec = 'EC 2.8.3.17';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:acetate CoA-transferase' WHERE ec = 'EC 2.8.3.18';
UPDATE enzyme SET history = 'EC 2.8.3.18 created 2013' WHERE ec = 'EC 2.8.3.18';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:oxalate CoA-transferase' WHERE ec = 'EC 2.8.3.19';
UPDATE enzyme SET history = 'EC 2.8.3.19 created 2013' WHERE ec = 'EC 2.8.3.19';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:(R)-citramalate CoA-transferase' WHERE ec = 'EC 2.8.3.20';
UPDATE enzyme SET history = 'EC 2.8.3.20 created 2014' WHERE ec = 'EC 2.8.3.20';
UPDATE enzyme SET sys_name = '(E)-4-(trimethylammonio)but-2-enoyl-CoA:L-carnitine CoA-transferase' WHERE ec = 'EC 2.8.3.21';
UPDATE enzyme SET history = 'EC 2.8.3.21 created 2014' WHERE ec = 'EC 2.8.3.21';
UPDATE enzyme SET sys_name = 'Succinyl-CoA:(S)-malate CoA-transferase' WHERE ec = 'EC 2.8.3.22';
UPDATE enzyme SET history = 'EC 2.8.3.22 created 2014' WHERE ec = 'EC 2.8.3.22';
UPDATE enzyme SET sys_name = 'Methyl-CoM:CoB S-(2-sulfoethyl)thiotransferase' WHERE ec = 'EC 2.8.4.1';
UPDATE enzyme SET history = 'EC 2.8.4.1 created 2001, modified 2011' WHERE ec = 'EC 2.8.4.1';
UPDATE enzyme SET sys_name = 'Mycothiol:arsenate S-arsenotransferase' WHERE ec = 'EC 2.8.4.2';
UPDATE enzyme SET history = 'EC 2.8.4.2.created 2010' WHERE ec = 'EC 2.8.4.2';
UPDATE enzyme SET sys_name = 'tRNA (N(6)-dimethylallyladenosine(37)):sulfur-(sulfur carrier),S-adenosyl-L-methionine C(2)-methylthiotransferase' WHERE ec = 'EC 2.8.4.3';
UPDATE enzyme SET history = 'EC 2.8.4.3 created 2014' WHERE ec = 'EC 2.8.4.3';
UPDATE enzyme SET sys_name = '[Ribosomal protein S12]-Asp(89):sulfur-(sulfur carrier),S-adenosyl-L-methionine C(3)-methylthiotransferase' WHERE ec = 'EC 2.8.4.4';
UPDATE enzyme SET history = 'EC 2.8.4.4 created 2014' WHERE ec = 'EC 2.8.4.4';
UPDATE enzyme SET sys_name = 'tRNA (N(6)-L-threonylcarbamoyladenosine(37)):sulfur-(sulfur carrier),S-adenosyl-L-methionine C(2)-methylthiotransferase' WHERE ec = 'EC 2.8.4.5';
UPDATE enzyme SET history = 'EC 2.8.4.5 created 2014' WHERE ec = 'EC 2.8.4.5';
UPDATE enzyme SET sys_name = 'Selenophosphate:L-seryl-tRNA(Sec) selenium transferase' WHERE ec = 'EC 2.9.1.1';
UPDATE enzyme SET history = 'EC 2.9.1.1 created 1999' WHERE ec = 'EC 2.9.1.1';
UPDATE enzyme SET sys_name = 'Selenophosphate:O-phospho-L-seryl-tRNA(Sec) selenium transferase' WHERE ec = 'EC 2.9.1.2';
UPDATE enzyme SET history = 'EC 2.9.1.2 created 2009' WHERE ec = 'EC 2.9.1.2';
UPDATE enzyme SET sys_name = 'Adenylyl-molybdopterin:molybdate molybdate transferase (AMP-forming)' WHERE ec = 'EC 2.10.1.1';
UPDATE enzyme SET history = 'EC 2.10.1.1 created 2011' WHERE ec = 'EC 2.10.1.1';
UPDATE enzyme SET sys_name = 'Carboxylic-ester hydrolase' WHERE ec = 'EC 3.1.1.1';
UPDATE enzyme SET history = 'EC 3.1.1.1 created 1961' WHERE ec = 'EC 3.1.1.1';
UPDATE enzyme SET sys_name = 'Aryl-ester hydrolase' WHERE ec = 'EC 3.1.1.2';
UPDATE enzyme SET history = 'EC 3.1.1.2 created 1961, modified 1989' WHERE ec = 'EC 3.1.1.2';
UPDATE enzyme SET sys_name = 'Triacylglycerol acylhydrolase' WHERE ec = 'EC 3.1.1.3';
UPDATE enzyme SET history = 'EC 3.1.1.3 created 1961' WHERE ec = 'EC 3.1.1.3';
UPDATE enzyme SET sys_name = 'Phosphatidylcholine 2-acylhydrolase' WHERE ec = 'EC 3.1.1.4';
UPDATE enzyme SET history = 'EC 3.1.1.4 created 1961, modified 1976, modified 1983' WHERE ec = 'EC 3.1.1.4';
UPDATE enzyme SET sys_name = '2-lysophosphatidylcholine acylhydrolase' WHERE ec = 'EC 3.1.1.5';
UPDATE enzyme SET history = 'EC 3.1.1.5 created 1961, modified 1976, modified 1983' WHERE ec = 'EC 3.1.1.5';
UPDATE enzyme SET sys_name = 'Acetic-ester acetylhydrolase' WHERE ec = 'EC 3.1.1.6';
UPDATE enzyme SET history = 'EC 3.1.1.6 created 1961' WHERE ec = 'EC 3.1.1.6';
UPDATE enzyme SET sys_name = 'Acetylcholine acetylhydrolase' WHERE ec = 'EC 3.1.1.7';
UPDATE enzyme SET history = 'EC 3.1.1.7 created 1961' WHERE ec = 'EC 3.1.1.7';
UPDATE enzyme SET sys_name = 'Acylcholine acylhydrolase' WHERE ec = 'EC 3.1.1.8';
UPDATE enzyme SET history = 'EC 3.1.1.8 created 1961' WHERE ec = 'EC 3.1.1.8';
UPDATE enzyme SET history = 'EC 3.1.1.9 created 1961, deleted 1972' WHERE ec = 'EC 3.1.1.9';
UPDATE enzyme SET sys_name = 'Atropine acylhydrolase' WHERE ec = 'EC 3.1.1.10';
UPDATE enzyme SET history = 'EC 3.1.1.10 created 1961, deleted 1972, reinstated 1976' WHERE ec = 'EC 3.1.1.10';
UPDATE enzyme SET sys_name = 'Pectin pectylhydrolase' WHERE ec = 'EC 3.1.1.11';
UPDATE enzyme SET history = 'EC 3.1.1.11 created 1961' WHERE ec = 'EC 3.1.1.11';
UPDATE enzyme SET history = 'EC 3.1.1.12 created 1961, deleted 1972' WHERE ec = 'EC 3.1.1.12';
UPDATE enzyme SET sys_name = 'Steryl-ester acylhydrolase' WHERE ec = 'EC 3.1.1.13';
UPDATE enzyme SET history = 'EC 3.1.1.13 created 1961, modified 1990' WHERE ec = 'EC 3.1.1.13';
UPDATE enzyme SET sys_name = 'Chlorophyll chlorophyllidohydrolase' WHERE ec = 'EC 3.1.1.14';
UPDATE enzyme SET history = 'EC 3.1.1.14 created 1961, modified 2007' WHERE ec = 'EC 3.1.1.14';
UPDATE enzyme SET sys_name = 'L-arabinono-1,4-lactone lactonohydrolase' WHERE ec = 'EC 3.1.1.15';
UPDATE enzyme SET history = 'EC 3.1.1.15 created 1961' WHERE ec = 'EC 3.1.1.15';
UPDATE enzyme SET history = 'EC 3.1.1.16 created 1961, deleted 1972' WHERE ec = 'EC 3.1.1.16';
UPDATE enzyme SET sys_name = 'D-glucono-1,5-lactone lactonohydrolase' WHERE ec = 'EC 3.1.1.17';
UPDATE enzyme SET history = 'EC 3.1.1.17 created 1961 (EC 3.1.1.18 created 1961, incorporated 1982)' WHERE ec = 'EC 3.1.1.17';
UPDATE enzyme SET history = 'EC 3.1.1.18 created 1961, deleted 1982' WHERE ec = 'EC 3.1.1.18';
UPDATE enzyme SET sys_name = 'D-glucurono-6,2-lactone lactonohydrolase' WHERE ec = 'EC 3.1.1.19';
UPDATE enzyme SET history = 'EC 3.1.1.19 created 1961' WHERE ec = 'EC 3.1.1.19';
UPDATE enzyme SET sys_name = 'Tannin acylhydrolase' WHERE ec = 'EC 3.1.1.20';
UPDATE enzyme SET history = 'EC 3.1.1.20 created 1961' WHERE ec = 'EC 3.1.1.20';
UPDATE enzyme SET history = 'EC 3.1.1.21 created 1972, deleted 2011' WHERE ec = 'EC 3.1.1.21';
UPDATE enzyme SET sys_name = '(R)-3-((R)-3-hydroxybutanoyloxy)butanoate hydroxybutanoylhydrolase' WHERE ec = 'EC 3.1.1.22';
UPDATE enzyme SET history = 'EC 3.1.1.22 created 1972' WHERE ec = 'EC 3.1.1.22';
UPDATE enzyme SET sys_name = 'Glycerol-ester acylhydrolase' WHERE ec = 'EC 3.1.1.23';
UPDATE enzyme SET history = 'EC 3.1.1.23 created 1972' WHERE ec = 'EC 3.1.1.23';
UPDATE enzyme SET sys_name = '4-carboxymethylbut-3-en-4-olide enol-lactonohydrolase' WHERE ec = 'EC 3.1.1.24';
UPDATE enzyme SET history = 'EC 3.1.1.24 created 1961 as EC 3.1.1.16, part transferred 1972 to EC 3.1.1.24' WHERE ec = 'EC 3.1.1.24';
UPDATE enzyme SET sys_name = '1,4-lactone hydroxyacylhydrolase' WHERE ec = 'EC 3.1.1.25';
UPDATE enzyme SET history = 'EC 3.1.1.25 created 1972, modified 1981' WHERE ec = 'EC 3.1.1.25';
UPDATE enzyme SET sys_name = '1,2-diacyl-3-beta-D-galactosyl-sn-glycerol acylhydrolase' WHERE ec = 'EC 3.1.1.26';
UPDATE enzyme SET history = 'EC 3.1.1.26 created 1972' WHERE ec = 'EC 3.1.1.26';
UPDATE enzyme SET sys_name = '4-pyridoxolactone lactonohydrolase' WHERE ec = 'EC 3.1.1.27';
UPDATE enzyme SET history = 'EC 3.1.1.27 created 1972' WHERE ec = 'EC 3.1.1.27';
UPDATE enzyme SET sys_name = 'O-acylcarnitine acylhydrolase' WHERE ec = 'EC 3.1.1.28';
UPDATE enzyme SET history = 'EC 3.1.1.28 created 1972' WHERE ec = 'EC 3.1.1.28';
UPDATE enzyme SET sys_name = 'Aminoacyl-tRNA aminoacylhydrolase' WHERE ec = 'EC 3.1.1.29';
UPDATE enzyme SET history = 'EC 3.1.1.29 created 1972' WHERE ec = 'EC 3.1.1.29';
UPDATE enzyme SET sys_name = 'D-arabinono-1,4-lactone lactonohydrolase' WHERE ec = 'EC 3.1.1.30';
UPDATE enzyme SET history = 'EC 3.1.1.30 created 1972' WHERE ec = 'EC 3.1.1.30';
UPDATE enzyme SET sys_name = '6-phospho-D-glucono-1,5-lactone lactonohydrolase' WHERE ec = 'EC 3.1.1.31';
UPDATE enzyme SET history = 'EC 3.1.1.31 created 1972' WHERE ec = 'EC 3.1.1.31';
UPDATE enzyme SET sys_name = 'Phosphatidylcholine 1-acylhydrolase' WHERE ec = 'EC 3.1.1.32';
UPDATE enzyme SET history = 'EC 3.1.1.32 created 1972, modified 1976' WHERE ec = 'EC 3.1.1.32';
UPDATE enzyme SET sys_name = '6-acetyl-D-glucose acetylhydrolase' WHERE ec = 'EC 3.1.1.33';
UPDATE enzyme SET history = 'EC 3.1.1.33 created 1972' WHERE ec = 'EC 3.1.1.33';
UPDATE enzyme SET sys_name = 'Triacylglycero-protein acylhydrolase' WHERE ec = 'EC 3.1.1.34';
UPDATE enzyme SET history = 'EC 3.1.1.34 created 1972, modified 1976' WHERE ec = 'EC 3.1.1.34';
UPDATE enzyme SET sys_name = 'Dihydrocoumarin lactonohydrolase' WHERE ec = 'EC 3.1.1.35';
UPDATE enzyme SET history = 'EC 3.1.1.35 created 1972' WHERE ec = 'EC 3.1.1.35';
UPDATE enzyme SET sys_name = 'Limonoate-D-ring-lactone lactonohydrolase' WHERE ec = 'EC 3.1.1.36';
UPDATE enzyme SET history = 'EC 3.1.1.36 created 1972' WHERE ec = 'EC 3.1.1.36';
UPDATE enzyme SET sys_name = 'Testololactone lactonohydrolase' WHERE ec = 'EC 3.1.1.37';
UPDATE enzyme SET history = 'EC 3.1.1.37 created 1972' WHERE ec = 'EC 3.1.1.37';
UPDATE enzyme SET sys_name = 'Triacetolactone lactonohydrolase' WHERE ec = 'EC 3.1.1.38';
UPDATE enzyme SET history = 'EC 3.1.1.38 created 1972' WHERE ec = 'EC 3.1.1.38';
UPDATE enzyme SET sys_name = 'Actinomycin lactonohydrolase' WHERE ec = 'EC 3.1.1.39';
UPDATE enzyme SET history = 'EC 3.1.1.39 created 1972' WHERE ec = 'EC 3.1.1.39';
UPDATE enzyme SET sys_name = 'Orsellinate-depside hydrolase' WHERE ec = 'EC 3.1.1.40';
UPDATE enzyme SET history = 'EC 3.1.1.40 created 1976' WHERE ec = 'EC 3.1.1.40';
UPDATE enzyme SET sys_name = 'Cephalosporin-C acetylhydrolase' WHERE ec = 'EC 3.1.1.41';
UPDATE enzyme SET history = 'EC 3.1.1.41 created 1976' WHERE ec = 'EC 3.1.1.41';
UPDATE enzyme SET sys_name = 'Chlorogenate hydrolase' WHERE ec = 'EC 3.1.1.42';
UPDATE enzyme SET history = 'EC 3.1.1.42 created 1981' WHERE ec = 'EC 3.1.1.42';
UPDATE enzyme SET sys_name = 'Alpha-amino-acid-ester aminoacylhydrolase' WHERE ec = 'EC 3.1.1.43';
UPDATE enzyme SET history = 'EC 3.1.1.43 created 1983' WHERE ec = 'EC 3.1.1.43';
UPDATE enzyme SET sys_name = 'Oxaloacetate-4-methyl-ester oxaloacetohydrolase' WHERE ec = 'EC 3.1.1.44';
UPDATE enzyme SET history = 'EC 3.1.1.44 created 1983' WHERE ec = 'EC 3.1.1.44';
UPDATE enzyme SET sys_name = '4-carboxymethylenebut-2-en-4-olide lactonohydrolase' WHERE ec = 'EC 3.1.1.45';
UPDATE enzyme SET history = 'EC 3.1.1.45 created 1983' WHERE ec = 'EC 3.1.1.45';
UPDATE enzyme SET sys_name = 'Deoxylimonate A-ring-lactonohydrolase' WHERE ec = 'EC 3.1.1.46';
UPDATE enzyme SET history = 'EC 3.1.1.46 created 1983' WHERE ec = 'EC 3.1.1.46';
UPDATE enzyme SET sys_name = '1-alkyl-2-acetyl-sn-glycero-3-phosphocholine acetohydrolase' WHERE ec = 'EC 3.1.1.47';
UPDATE enzyme SET history = 'EC 3.1.1.47 created 1984' WHERE ec = 'EC 3.1.1.47';
UPDATE enzyme SET sys_name = 'N(5)-acyl-L-ornithine-ester hydrolase' WHERE ec = 'EC 3.1.1.48';
UPDATE enzyme SET history = 'EC 3.1.1.48 created 1984' WHERE ec = 'EC 3.1.1.48';
UPDATE enzyme SET sys_name = 'Sinapoylcholine sinapohydrolase' WHERE ec = 'EC 3.1.1.49';
UPDATE enzyme SET history = 'EC 3.1.1.49 created 1984' WHERE ec = 'EC 3.1.1.49';
UPDATE enzyme SET sys_name = 'Wax-ester acylhydrolase' WHERE ec = 'EC 3.1.1.50';
UPDATE enzyme SET history = 'EC 3.1.1.50 created 1984' WHERE ec = 'EC 3.1.1.50';
UPDATE enzyme SET sys_name = '12,13-diacylphorbate 12-acylhydrolase' WHERE ec = 'EC 3.1.1.51';
UPDATE enzyme SET history = 'EC 3.1.1.51 created 1984' WHERE ec = 'EC 3.1.1.51';
UPDATE enzyme SET sys_name = '1-phosphatidyl-D-myo-inositol 2-acylhydrolase' WHERE ec = 'EC 3.1.1.52';
UPDATE enzyme SET history = 'EC 3.1.1.52 created 1984' WHERE ec = 'EC 3.1.1.52';
UPDATE enzyme SET sys_name = 'N-acyl-O-acetylneuraminate O-acetylhydrolase' WHERE ec = 'EC 3.1.1.53';
UPDATE enzyme SET history = 'EC 3.1.1.53 created 1984' WHERE ec = 'EC 3.1.1.53';
UPDATE enzyme SET sys_name = '5-(4-acetoxybut-1-ynyl)-2,2-bithiophene O-acetylhydrolase' WHERE ec = 'EC 3.1.1.54';
UPDATE enzyme SET history = 'EC 3.1.1.54 created 1986' WHERE ec = 'EC 3.1.1.54';
UPDATE enzyme SET sys_name = 'Acetylsalicylate O-acetylhydrolase' WHERE ec = 'EC 3.1.1.55';
UPDATE enzyme SET history = 'EC 3.1.1.55 created 1986, modified 1989' WHERE ec = 'EC 3.1.1.55';
UPDATE enzyme SET sys_name = '4-methylumbelliferyl-acetate acylhydrolase' WHERE ec = 'EC 3.1.1.56';
UPDATE enzyme SET history = 'EC 3.1.1.56 created 1986' WHERE ec = 'EC 3.1.1.56';
UPDATE enzyme SET sys_name = '2-oxo-2H-pyran-4,6-dicarboxylate lactonohydrolase' WHERE ec = 'EC 3.1.1.57';
UPDATE enzyme SET history = 'EC 3.1.1.57 created 1986' WHERE ec = 'EC 3.1.1.57';
UPDATE enzyme SET sys_name = 'N-acetyl-D-galactosaminoglycan acetylhydrolase' WHERE ec = 'EC 3.1.1.58';
UPDATE enzyme SET history = 'EC 3.1.1.58 created 1986' WHERE ec = 'EC 3.1.1.58';
UPDATE enzyme SET sys_name = 'Methyl-(2E,6E)-(10R,11S)-10,11-epoxy-3,7,11-trimethyltrideca-2,6-dienoate acylhydrolase' WHERE ec = 'EC 3.1.1.59';
UPDATE enzyme SET history = 'EC 3.1.1.59 created 1989' WHERE ec = 'EC 3.1.1.59';
UPDATE enzyme SET sys_name = 'Bis(2-ethylhexyl)phthalate acylhydrolase' WHERE ec = 'EC 3.1.1.60';
UPDATE enzyme SET history = 'EC 3.1.1.60 created 1989' WHERE ec = 'EC 3.1.1.60';
UPDATE enzyme SET sys_name = 'Protein-L-glutamate-O(5)-methyl-ester acylhydrolase' WHERE ec = 'EC 3.1.1.61';
UPDATE enzyme SET history = 'EC 3.1.1.61 created 1989, modified 2002' WHERE ec = 'EC 3.1.1.61';
UPDATE enzyme SET history = 'EC 3.1.1.62 created 1989, deleted 1992' WHERE ec = 'EC 3.1.1.62';
UPDATE enzyme SET sys_name = '11-cis-retinyl-palmitate acylhydrolase' WHERE ec = 'EC 3.1.1.63';
UPDATE enzyme SET history = 'EC 3.1.1.63 created 1989' WHERE ec = 'EC 3.1.1.63';
UPDATE enzyme SET sys_name = 'All-trans-retinyl ester acylhydrolase, 11-cis retinol forming' WHERE ec = 'EC 3.1.1.64';
UPDATE enzyme SET history = 'EC 3.1.1.64 created 1989 (EC 5.2.1.7 created 1989, incorporated 2011), modified 2011' WHERE ec = 'EC 3.1.1.64';
UPDATE enzyme SET sys_name = 'L-rhamnono-1,4-lactone lactonohydrolase' WHERE ec = 'EC 3.1.1.65';
UPDATE enzyme SET history = 'EC 3.1.1.65 created 1989' WHERE ec = 'EC 3.1.1.65';
UPDATE enzyme SET sys_name = '5-(3,4-diacetoxybut-1-ynyl)-2,2-bithiophene acetylhydrolase' WHERE ec = 'EC 3.1.1.66';
UPDATE enzyme SET history = 'EC 3.1.1.66 created 1989' WHERE ec = 'EC 3.1.1.66';
UPDATE enzyme SET sys_name = 'Long-chain-fatty-acyl-ethyl-ester acylhydrolase' WHERE ec = 'EC 3.1.1.67';
UPDATE enzyme SET history = 'EC 3.1.1.67 created 1989' WHERE ec = 'EC 3.1.1.67';
UPDATE enzyme SET sys_name = 'D-xylono-1,4-lactone lactonohydrolase' WHERE ec = 'EC 3.1.1.68';
UPDATE enzyme SET history = 'EC 3.1.1.68 created 1990' WHERE ec = 'EC 3.1.1.68';
UPDATE enzyme SET history = 'EC 3.1.1.69 created 1992, deleted 2002' WHERE ec = 'EC 3.1.1.69';
UPDATE enzyme SET sys_name = 'Cetraxate-benzyl-ester benzylhydrolase' WHERE ec = 'EC 3.1.1.70';
UPDATE enzyme SET history = 'EC 3.1.1.70 created 1992' WHERE ec = 'EC 3.1.1.70';
UPDATE enzyme SET sys_name = '2-acetyl-1-alkyl-sn-glycerol acetylhydrolase' WHERE ec = 'EC 3.1.1.71';
UPDATE enzyme SET history = 'EC 3.1.1.71 created 1999' WHERE ec = 'EC 3.1.1.71';
UPDATE enzyme SET sys_name = 'Acetylxylan esterase' WHERE ec = 'EC 3.1.1.72';
UPDATE enzyme SET history = 'EC 3.1.1.72 created 1999' WHERE ec = 'EC 3.1.1.72';
UPDATE enzyme SET sys_name = '4-hydroxy-3-methoxycinnamoyl-sugar hydrolase' WHERE ec = 'EC 3.1.1.73';
UPDATE enzyme SET history = 'EC 3.1.1.73 created 2000' WHERE ec = 'EC 3.1.1.73';
UPDATE enzyme SET sys_name = 'Cutin hydrolase' WHERE ec = 'EC 3.1.1.74';
UPDATE enzyme SET history = 'EC 3.1.1.74 created 2000' WHERE ec = 'EC 3.1.1.74';
UPDATE enzyme SET sys_name = 'Poly((R)-3-hydroxybutanoate) hydrolase' WHERE ec = 'EC 3.1.1.75';
UPDATE enzyme SET history = 'EC 3.1.1.75 created 2001' WHERE ec = 'EC 3.1.1.75';
UPDATE enzyme SET sys_name = 'Poly(oxycarbonyl((R)-2-pentylethylene)) hydrolase' WHERE ec = 'EC 3.1.1.76';
UPDATE enzyme SET history = 'EC 3.1.1.76 created 2001, modified 2005' WHERE ec = 'EC 3.1.1.76';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.1.77';
UPDATE enzyme SET history = 'EC 3.1.1.77 created 2001' WHERE ec = 'EC 3.1.1.77';
UPDATE enzyme SET sys_name = 'Polyneuridine aldehyde hydrolase (decarboxylating)' WHERE ec = 'EC 3.1.1.78';
UPDATE enzyme SET history = 'EC 3.1.1.78 created 2002' WHERE ec = 'EC 3.1.1.78';
UPDATE enzyme SET sys_name = 'Diacylglycerol acylhydrolase' WHERE ec = 'EC 3.1.1.79';
UPDATE enzyme SET history = 'EC 3.1.1.79 created 2004' WHERE ec = 'EC 3.1.1.79';
UPDATE enzyme SET sys_name = '17-O-acetylajmaline O-acetylhydrolase' WHERE ec = 'EC 3.1.1.80';
UPDATE enzyme SET history = 'EC 3.1.1.80 created 2006' WHERE ec = 'EC 3.1.1.80';
UPDATE enzyme SET sys_name = 'N-acyl-L-homoserine-lactone lactonohydrolase' WHERE ec = 'EC 3.1.1.81';
UPDATE enzyme SET history = 'EC 3.1.1.81 created 2007' WHERE ec = 'EC 3.1.1.81';
UPDATE enzyme SET sys_name = 'Pheophorbide-a hydrolase' WHERE ec = 'EC 3.1.1.82';
UPDATE enzyme SET history = 'EC 3.1.1.82 created 2007' WHERE ec = 'EC 3.1.1.82';
UPDATE enzyme SET sys_name = 'Isoprop(en)ylmethyloxepan-2-one lactonohydrolase' WHERE ec = 'EC 3.1.1.83';
UPDATE enzyme SET history = 'EC 3.1.1.83 created 2008' WHERE ec = 'EC 3.1.1.83';
UPDATE enzyme SET sys_name = 'Cocaine benzoylhydrolase' WHERE ec = 'EC 3.1.1.84';
UPDATE enzyme SET history = 'EC 3.1.1.84 created 2010' WHERE ec = 'EC 3.1.1.84';
UPDATE enzyme SET sys_name = 'Pimeloyl-[acyl-carrier protein] methyl ester hydrolase' WHERE ec = 'EC 3.1.1.85';
UPDATE enzyme SET history = 'EC 3.1.1.85 created 2011' WHERE ec = 'EC 3.1.1.85';
UPDATE enzyme SET sys_name = 'Rhamnogalacturonan 2/3-O-acetyl-alpha-D-galacturonate O-acetylhydrolase' WHERE ec = 'EC 3.1.1.86';
UPDATE enzyme SET history = 'EC 3.1.1.86 created 2011' WHERE ec = 'EC 3.1.1.86';
UPDATE enzyme SET sys_name = 'Fumonisin B1 acylhydrolase' WHERE ec = 'EC 3.1.1.87';
UPDATE enzyme SET history = 'EC 3.1.1.87 created 2011' WHERE ec = 'EC 3.1.1.87';
UPDATE enzyme SET sys_name = 'Pyrethroid-ester hydrolase' WHERE ec = 'EC 3.1.1.88';
UPDATE enzyme SET history = 'EC 3.1.1.88 created 2011' WHERE ec = 'EC 3.1.1.88';
UPDATE enzyme SET sys_name = '[Phosphatase 2A protein]-leucine ester acylhydrolase' WHERE ec = 'EC 3.1.1.89';
UPDATE enzyme SET history = 'EC 3.1.1.89 created 2011' WHERE ec = 'EC 3.1.1.89';
UPDATE enzyme SET sys_name = 'All-trans-retinyl ester acylhydrolase, 13-cis retinol forming' WHERE ec = 'EC 3.1.1.90';
UPDATE enzyme SET history = 'EC 3.1.1.90 created 2011' WHERE ec = 'EC 3.1.1.90';
UPDATE enzyme SET sys_name = '2-oxo-3-(5-oxofuran-2-ylidene)propanoate lactonohydrolase' WHERE ec = 'EC 3.1.1.91';
UPDATE enzyme SET history = 'EC 3.1.1.91 created 2012' WHERE ec = 'EC 3.1.1.91';
UPDATE enzyme SET sys_name = '4-sulfomuconolactone sulfohydrolase' WHERE ec = 'EC 3.1.1.92';
UPDATE enzyme SET history = 'EC 3.1.1.92 created 2012' WHERE ec = 'EC 3.1.1.92';
UPDATE enzyme SET sys_name = 'Mycophenolic acid O-acyl-glucuronide-ester hydrolase' WHERE ec = 'EC 3.1.1.93';
UPDATE enzyme SET history = 'EC 3.1.1.93 created 2012' WHERE ec = 'EC 3.1.1.93';
UPDATE enzyme SET sys_name = 'Versiconal-hemiacetal-acetate O-acetylhydrolase' WHERE ec = 'EC 3.1.1.94';
UPDATE enzyme SET history = 'EC 3.1.1.94 created 2013' WHERE ec = 'EC 3.1.1.94';
UPDATE enzyme SET sys_name = 'Aclacinomycin T acylhydrolase' WHERE ec = 'EC 3.1.1.95';
UPDATE enzyme SET history = 'EC 3.1.1.95 created 2013' WHERE ec = 'EC 3.1.1.95';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.1.1.n1';
UPDATE enzyme SET sys_name = 'Acetyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.1';
UPDATE enzyme SET history = 'EC 3.1.2.1 created 1961' WHERE ec = 'EC 3.1.2.1';
UPDATE enzyme SET sys_name = 'Palmitoyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.2';
UPDATE enzyme SET history = 'EC 3.1.2.2 created 1961' WHERE ec = 'EC 3.1.2.2';
UPDATE enzyme SET sys_name = 'Succinyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.3';
UPDATE enzyme SET history = 'EC 3.1.2.3 created 1961' WHERE ec = 'EC 3.1.2.3';
UPDATE enzyme SET sys_name = '3-hydroxy-2-methylpropanoyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.4';
UPDATE enzyme SET history = 'EC 3.1.2.4 created 1961' WHERE ec = 'EC 3.1.2.4';
UPDATE enzyme SET sys_name = '(S)-3-hydroxy-3-methylglutaryl-CoA hydrolase' WHERE ec = 'EC 3.1.2.5';
UPDATE enzyme SET history = 'EC 3.1.2.5 created 1961' WHERE ec = 'EC 3.1.2.5';
UPDATE enzyme SET sys_name = 'S-(2-hydroxyacyl)glutathione hydrolase' WHERE ec = 'EC 3.1.2.6';
UPDATE enzyme SET history = 'EC 3.1.2.6 created 1961 (EC 3.1.2.8 created 1961, incorporated 1978)' WHERE ec = 'EC 3.1.2.6';
UPDATE enzyme SET sys_name = 'S-acylglutathione hydrolase' WHERE ec = 'EC 3.1.2.7';
UPDATE enzyme SET history = 'EC 3.1.2.7 created 1961' WHERE ec = 'EC 3.1.2.7';
UPDATE enzyme SET history = 'EC 3.1.2.8 created 1961, deleted 1978' WHERE ec = 'EC 3.1.2.8';
UPDATE enzyme SET history = 'EC 3.1.2.9 created 1961, deleted 1964' WHERE ec = 'EC 3.1.2.9';
UPDATE enzyme SET sys_name = 'Formyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.10';
UPDATE enzyme SET history = 'EC 3.1.2.10 created 1965' WHERE ec = 'EC 3.1.2.10';
UPDATE enzyme SET sys_name = 'Acetoacetyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.11';
UPDATE enzyme SET history = 'EC 3.1.2.11 created 1972' WHERE ec = 'EC 3.1.2.11';
UPDATE enzyme SET sys_name = 'S-formylglutathione hydrolase' WHERE ec = 'EC 3.1.2.12';
UPDATE enzyme SET history = 'EC 3.1.2.12 created 1978' WHERE ec = 'EC 3.1.2.12';
UPDATE enzyme SET sys_name = 'S-succinylglutathione hydrolase' WHERE ec = 'EC 3.1.2.13';
UPDATE enzyme SET history = 'EC 3.1.2.13 created 1978' WHERE ec = 'EC 3.1.2.13';
UPDATE enzyme SET sys_name = 'Oleoyl-[acyl-carrier-protein] hydrolase' WHERE ec = 'EC 3.1.2.14';
UPDATE enzyme SET history = 'EC 3.1.2.14 created 1984' WHERE ec = 'EC 3.1.2.14';
UPDATE enzyme SET sys_name = 'Ubiquitin-C-terminal-thioester hydrolase' WHERE ec = 'EC 3.1.2.15';
UPDATE enzyme SET history = 'EC 3.1.2.15 created 1986' WHERE ec = 'EC 3.1.2.15';
UPDATE enzyme SET sys_name = '[Citrate-(pro-3S)-lyase](acetyl-form) hydrolase' WHERE ec = 'EC 3.1.2.16';
UPDATE enzyme SET history = 'EC 3.1.2.16 created 1989' WHERE ec = 'EC 3.1.2.16';
UPDATE enzyme SET sys_name = '(S)-methylmalonyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.17';
UPDATE enzyme SET history = 'EC 3.1.2.17 created 1989' WHERE ec = 'EC 3.1.2.17';
UPDATE enzyme SET sys_name = 'ADP-dependent-short-chain-acyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.18';
UPDATE enzyme SET history = 'EC 3.1.2.18 created 1992' WHERE ec = 'EC 3.1.2.18';
UPDATE enzyme SET sys_name = 'ADP-dependent-medium-chain-acyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.19';
UPDATE enzyme SET history = 'EC 3.1.2.19 created 1992' WHERE ec = 'EC 3.1.2.19';
UPDATE enzyme SET sys_name = 'Acyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.20';
UPDATE enzyme SET history = 'EC 3.1.2.20 created 1992' WHERE ec = 'EC 3.1.2.20';
UPDATE enzyme SET sys_name = 'Dodecanoyl-[acyl-carrier-protein] hydrolase' WHERE ec = 'EC 3.1.2.21';
UPDATE enzyme SET history = 'EC 3.1.2.21 created 1999' WHERE ec = 'EC 3.1.2.21';
UPDATE enzyme SET sys_name = 'Palmitoyl-(protein) hydrolase' WHERE ec = 'EC 3.1.2.22';
UPDATE enzyme SET history = 'EC 3.1.2.22 created 1999' WHERE ec = 'EC 3.1.2.22';
UPDATE enzyme SET sys_name = '4-hydroxybenzoyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.23';
UPDATE enzyme SET history = 'EC 3.1.2.23 created 1999' WHERE ec = 'EC 3.1.2.23';
UPDATE enzyme SET history = 'EC 3.1.2.24 created 2000, deleted 2005' WHERE ec = 'EC 3.1.2.24';
UPDATE enzyme SET sys_name = 'Phenylglyoxylyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.25';
UPDATE enzyme SET history = 'EC 3.1.2.25 created 2004' WHERE ec = 'EC 3.1.2.25';
UPDATE enzyme SET sys_name = 'Deoxycholoyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.26';
UPDATE enzyme SET history = 'EC 3.1.2.26 created 2005' WHERE ec = 'EC 3.1.2.26';
UPDATE enzyme SET sys_name = 'Choloyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.27';
UPDATE enzyme SET history = 'EC 3.1.2.27 created 2005' WHERE ec = 'EC 3.1.2.27';
UPDATE enzyme SET sys_name = '1,4-dihydroxy-2-naphthoyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.28';
UPDATE enzyme SET history = 'EC 3.1.2.28 created 2010' WHERE ec = 'EC 3.1.2.28';
UPDATE enzyme SET sys_name = 'Fluoroacetyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.29';
UPDATE enzyme SET history = 'EC 3.1.2.29 created 2011' WHERE ec = 'EC 3.1.2.29';
UPDATE enzyme SET sys_name = '(S)-malyl-CoA hydrolase' WHERE ec = 'EC 3.1.2.30';
UPDATE enzyme SET history = 'EC 3.1.2.30 created 2014' WHERE ec = 'EC 3.1.2.30';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.1.2.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.1.2.n2';
UPDATE enzyme SET sys_name = 'Phosphate-monoester phosphohydrolase (alkaline optimum)' WHERE ec = 'EC 3.1.3.1';
UPDATE enzyme SET history = 'EC 3.1.3.1 created 1961' WHERE ec = 'EC 3.1.3.1';
UPDATE enzyme SET sys_name = 'Phosphate-monoester phosphohydrolase (acid optimum)' WHERE ec = 'EC 3.1.3.2';
UPDATE enzyme SET history = 'EC 3.1.3.2 created 1961' WHERE ec = 'EC 3.1.3.2';
UPDATE enzyme SET sys_name = 'O-phosphoserine phosphohydrolase' WHERE ec = 'EC 3.1.3.3';
UPDATE enzyme SET history = 'EC 3.1.3.3 created 1961' WHERE ec = 'EC 3.1.3.3';
UPDATE enzyme SET sys_name = 'Diacylglycerol-3-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.4';
UPDATE enzyme SET history = 'EC 3.1.3.4 created 1961, modified 2010' WHERE ec = 'EC 3.1.3.4';
UPDATE enzyme SET sys_name = '5-ribonucleotide phosphohydrolase' WHERE ec = 'EC 3.1.3.5';
UPDATE enzyme SET history = 'EC 3.1.3.5 created 1961' WHERE ec = 'EC 3.1.3.5';
UPDATE enzyme SET sys_name = '3-ribonucleotide phosphohydrolase' WHERE ec = 'EC 3.1.3.6';
UPDATE enzyme SET history = 'EC 3.1.3.6 created 1961' WHERE ec = 'EC 3.1.3.6';
UPDATE enzyme SET sys_name = 'Adenosine-3(2),5-bisphosphate 3(2)-phosphohydrolase' WHERE ec = 'EC 3.1.3.7';
UPDATE enzyme SET history = 'EC 3.1.3.7 created 1961' WHERE ec = 'EC 3.1.3.7';
UPDATE enzyme SET sys_name = 'Myo-inositol-hexakisphosphate 3-phosphohydrolase' WHERE ec = 'EC 3.1.3.8';
UPDATE enzyme SET history = 'EC 3.1.3.8 created 1961, modified 1976, modified 2002' WHERE ec = 'EC 3.1.3.8';
UPDATE enzyme SET sys_name = 'D-glucose-6-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.9';
UPDATE enzyme SET history = 'EC 3.1.3.9 created 1961' WHERE ec = 'EC 3.1.3.9';
UPDATE enzyme SET sys_name = 'Alpha-D-glucose-1-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.10';
UPDATE enzyme SET history = 'EC 3.1.3.10 created 1961' WHERE ec = 'EC 3.1.3.10';
UPDATE enzyme SET sys_name = 'D-fructose-1,6-bisphosphate 1-phosphohydrolase' WHERE ec = 'EC 3.1.3.11';
UPDATE enzyme SET history = 'EC 3.1.3.11 created 1961, modified 1976' WHERE ec = 'EC 3.1.3.11';
UPDATE enzyme SET sys_name = 'Alpha,alpha-trehalose-6-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.12';
UPDATE enzyme SET history = 'EC 3.1.3.12 created 1961' WHERE ec = 'EC 3.1.3.12';
UPDATE enzyme SET sys_name = '2,3-bisphospho-D-glycerate 2-phosphohydrolase' WHERE ec = 'EC 3.1.3.13';
UPDATE enzyme SET history = 'EC 3.1.3.13 created 1961' WHERE ec = 'EC 3.1.3.13';
UPDATE enzyme SET sys_name = 'S-methyl-3-phospho-1-thio-D-glycerate phosphohydrolase' WHERE ec = 'EC 3.1.3.14';
UPDATE enzyme SET history = 'EC 3.1.3.14 created 1961' WHERE ec = 'EC 3.1.3.14';
UPDATE enzyme SET sys_name = 'L-histidinol-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.15';
UPDATE enzyme SET history = 'EC 3.1.3.15 created 1961' WHERE ec = 'EC 3.1.3.15';
UPDATE enzyme SET sys_name = 'Phosphoprotein phosphohydrolase' WHERE ec = 'EC 3.1.3.16';
UPDATE enzyme SET history = 'EC 3.1.3.16 created 1961, modified 1989, modified 2013' WHERE ec = 'EC 3.1.3.16';
UPDATE enzyme SET sys_name = '[Phosphorylase a] phosphohydrolase' WHERE ec = 'EC 3.1.3.17';
UPDATE enzyme SET history = 'EC 3.1.3.17 created 1961' WHERE ec = 'EC 3.1.3.17';
UPDATE enzyme SET sys_name = '2-phosphoglycolate phosphohydrolase' WHERE ec = 'EC 3.1.3.18';
UPDATE enzyme SET history = 'EC 3.1.3.18 created 1965' WHERE ec = 'EC 3.1.3.18';
UPDATE enzyme SET sys_name = 'Glycerol-2-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.19';
UPDATE enzyme SET history = 'EC 3.1.3.19 created 1965' WHERE ec = 'EC 3.1.3.19';
UPDATE enzyme SET sys_name = 'D-glycerate-2-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.20';
UPDATE enzyme SET history = 'EC 3.1.3.20 created 1972' WHERE ec = 'EC 3.1.3.20';
UPDATE enzyme SET sys_name = 'Glycerol-1-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.21';
UPDATE enzyme SET history = 'EC 3.1.3.21 created 1972, modified 1986' WHERE ec = 'EC 3.1.3.21';
UPDATE enzyme SET sys_name = 'D-mannitol-1-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.22';
UPDATE enzyme SET history = 'EC 3.1.3.22 created 1972' WHERE ec = 'EC 3.1.3.22';
UPDATE enzyme SET sys_name = 'Sugar-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.23';
UPDATE enzyme SET history = 'EC 3.1.3.23 created 1972' WHERE ec = 'EC 3.1.3.23';
UPDATE enzyme SET sys_name = 'Sucrose-6(F)-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.24';
UPDATE enzyme SET history = 'EC 3.1.3.24 created 1972, modified 2008' WHERE ec = 'EC 3.1.3.24';
UPDATE enzyme SET sys_name = 'Myo-inositol-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.25';
UPDATE enzyme SET history = 'EC 3.1.3.25 created 1972, modified 1990, modified 2002, modified 2004' WHERE ec = 'EC 3.1.3.25';
UPDATE enzyme SET sys_name = 'Myo-inositol-hexakisphosphate 4-phosphohydrolase' WHERE ec = 'EC 3.1.3.26';
UPDATE enzyme SET history = 'EC 3.1.3.26 created 1972, modified 1976, modified 2002' WHERE ec = 'EC 3.1.3.26';
UPDATE enzyme SET sys_name = 'Phosphatidylglycerophosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.27';
UPDATE enzyme SET history = 'EC 3.1.3.27 created 1972' WHERE ec = 'EC 3.1.3.27';
UPDATE enzyme SET sys_name = '3-(ADP)-2-phosphoglycerate phosphohydrolase' WHERE ec = 'EC 3.1.3.28';
UPDATE enzyme SET history = 'EC 3.1.3.28 created 1972' WHERE ec = 'EC 3.1.3.28';
UPDATE enzyme SET sys_name = 'N-acylneuraminate-9-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.29';
UPDATE enzyme SET history = 'EC 3.1.3.29 created 1972' WHERE ec = 'EC 3.1.3.29';
UPDATE enzyme SET history = 'EC 3.1.3.30 created 1972, deleted 1992' WHERE ec = 'EC 3.1.3.30';
UPDATE enzyme SET sys_name = 'Nucleotide phosphohydrolase' WHERE ec = 'EC 3.1.3.31';
UPDATE enzyme SET history = 'EC 3.1.3.31 created 1972 (EC 3.1.3.30 created 1972, incorporated 1992)' WHERE ec = 'EC 3.1.3.31';
UPDATE enzyme SET sys_name = 'Polynucleotide 3-phosphohydrolase' WHERE ec = 'EC 3.1.3.32';
UPDATE enzyme SET history = 'EC 3.1.3.32 created 1972' WHERE ec = 'EC 3.1.3.32';
UPDATE enzyme SET sys_name = 'Polynucleotide 5-phosphohydrolase' WHERE ec = 'EC 3.1.3.33';
UPDATE enzyme SET history = 'EC 3.1.3.33 created 1972' WHERE ec = 'EC 3.1.3.33';
UPDATE enzyme SET sys_name = 'Deoxyribonucleotide 3-phosphohydrolase' WHERE ec = 'EC 3.1.3.34';
UPDATE enzyme SET history = 'EC 3.1.3.34 created 1972' WHERE ec = 'EC 3.1.3.34';
UPDATE enzyme SET sys_name = 'Thymidylate 5-phosphohydrolase' WHERE ec = 'EC 3.1.3.35';
UPDATE enzyme SET history = 'EC 3.1.3.35 created 1972' WHERE ec = 'EC 3.1.3.35';
UPDATE enzyme SET sys_name = 'Phosphatidyl-myo-inositol-4,5-bisphosphate 4-phosphohydrolase' WHERE ec = 'EC 3.1.3.36';
UPDATE enzyme SET history = 'EC 3.1.3.36 created 1972, modified 2002' WHERE ec = 'EC 3.1.3.36';
UPDATE enzyme SET sys_name = 'Sedoheptulose-1,7-bisphosphate 1-phosphohydrolase' WHERE ec = 'EC 3.1.3.37';
UPDATE enzyme SET history = 'EC 3.1.3.37 created 1976' WHERE ec = 'EC 3.1.3.37';
UPDATE enzyme SET sys_name = 'D-glycerate-3-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.38';
UPDATE enzyme SET history = 'EC 3.1.3.38 created 1976' WHERE ec = 'EC 3.1.3.38';
UPDATE enzyme SET sys_name = 'Streptomycin-6-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.39';
UPDATE enzyme SET history = 'EC 3.1.3.39 created 1976' WHERE ec = 'EC 3.1.3.39';
UPDATE enzyme SET sys_name = '1-guanidino-1-deoxy-scyllo-inositol-4-phosphate 4-phosphohydrolase' WHERE ec = 'EC 3.1.3.40';
UPDATE enzyme SET history = 'EC 3.1.3.40 created 1976' WHERE ec = 'EC 3.1.3.40';
UPDATE enzyme SET sys_name = '4-nitrophenylphosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.41';
UPDATE enzyme SET history = 'EC 3.1.3.41 created 1976' WHERE ec = 'EC 3.1.3.41';
UPDATE enzyme SET sys_name = '[UDP-glucose:glycogen 4-alpha-D-glucosyltransferase-D] phosphohydrolase' WHERE ec = 'EC 3.1.3.42';
UPDATE enzyme SET history = 'EC 3.1.3.42 created 1976' WHERE ec = 'EC 3.1.3.42';
UPDATE enzyme SET sys_name = '[Pyruvate dehydrogenase (acetyl-transferring)]-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.43';
UPDATE enzyme SET history = 'EC 3.1.3.43 created 1978' WHERE ec = 'EC 3.1.3.43';
UPDATE enzyme SET sys_name = '[Acetyl-CoA:carbon-dioxide ligase (ADP-forming)]-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.44';
UPDATE enzyme SET history = 'EC 3.1.3.44 created 1983' WHERE ec = 'EC 3.1.3.44';
UPDATE enzyme SET sys_name = '3-deoxy-D-manno-octulosonate-8-phosphate 8-phosphohydrolase' WHERE ec = 'EC 3.1.3.45';
UPDATE enzyme SET history = 'EC 3.1.3.45 created 1983' WHERE ec = 'EC 3.1.3.45';
UPDATE enzyme SET sys_name = 'Beta-D-fructose-2,6-bisphosphate 2-phosphohydrolase' WHERE ec = 'EC 3.1.3.46';
UPDATE enzyme SET history = 'EC 3.1.3.46 created 1984' WHERE ec = 'EC 3.1.3.46';
UPDATE enzyme SET sys_name = '[Hydroxymethylglutaryl-CoA reductase (NADPH)]-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.47';
UPDATE enzyme SET history = 'EC 3.1.3.47 created 1984' WHERE ec = 'EC 3.1.3.47';
UPDATE enzyme SET sys_name = 'Protein-tyrosine-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.48';
UPDATE enzyme SET history = 'EC 3.1.3.48 created 1984' WHERE ec = 'EC 3.1.3.48';
UPDATE enzyme SET sys_name = '[ATP:pyruvate 2-O-phosphotransferase]-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.49';
UPDATE enzyme SET history = 'EC 3.1.3.49 created 1984' WHERE ec = 'EC 3.1.3.49';
UPDATE enzyme SET sys_name = 'Sorbitol-6-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.50';
UPDATE enzyme SET history = 'EC 3.1.3.50 created 1984' WHERE ec = 'EC 3.1.3.50';
UPDATE enzyme SET sys_name = 'Dolichyl-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.51';
UPDATE enzyme SET history = 'EC 3.1.3.51 created 1984' WHERE ec = 'EC 3.1.3.51';
UPDATE enzyme SET sys_name = '[3-methyl-2-oxobutanoate dehydrogenase (2-methylpropanoyl-transferring)]-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.52';
UPDATE enzyme SET history = 'EC 3.1.3.52 created 1986' WHERE ec = 'EC 3.1.3.52';
UPDATE enzyme SET sys_name = '[Myosin-light-chain]-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.53';
UPDATE enzyme SET history = 'EC 3.1.3.53 created 1986' WHERE ec = 'EC 3.1.3.53';
UPDATE enzyme SET sys_name = 'Beta-D-fructose-2,6-bisphosphate 6-phosphohydrolase' WHERE ec = 'EC 3.1.3.54';
UPDATE enzyme SET history = 'EC 3.1.3.54 created 1989' WHERE ec = 'EC 3.1.3.54';
UPDATE enzyme SET sys_name = 'Caldesmon-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.55';
UPDATE enzyme SET history = 'EC 3.1.3.55 created 1989' WHERE ec = 'EC 3.1.3.55';
UPDATE enzyme SET sys_name = '1D-myo-inositol-1,4,5-trisphosphate 5-phosphohydrolase' WHERE ec = 'EC 3.1.3.56';
UPDATE enzyme SET history = 'EC 3.1.3.56 created 1989, modified 2002' WHERE ec = 'EC 3.1.3.56';
UPDATE enzyme SET sys_name = '1D-myo-inositol-1,4-bisphosphate 1-phosphohydrolase' WHERE ec = 'EC 3.1.3.57';
UPDATE enzyme SET history = 'EC 3.1.3.57 created 1989, modified 2002' WHERE ec = 'EC 3.1.3.57';
UPDATE enzyme SET sys_name = 'Sugar-omega-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.58';
UPDATE enzyme SET history = 'EC 3.1.3.58 created 1989' WHERE ec = 'EC 3.1.3.58';
UPDATE enzyme SET sys_name = '1-alkyl-2-acetyl-sn-glycero-3-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.59';
UPDATE enzyme SET history = 'EC 3.1.3.59 created 1989' WHERE ec = 'EC 3.1.3.59';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate phosphohydrolase' WHERE ec = 'EC 3.1.3.60';
UPDATE enzyme SET history = 'EC 3.1.3.60 created 1992' WHERE ec = 'EC 3.1.3.60';
UPDATE enzyme SET history = 'EC 3.1.3.61 created 1992, deleted 2002' WHERE ec = 'EC 3.1.3.61';
UPDATE enzyme SET sys_name = '1D-myo-inositol-hexakisphosphate 5-phosphohydrolase' WHERE ec = 'EC 3.1.3.62';
UPDATE enzyme SET history = 'EC 3.1.3.62 created 1992, modified 2002' WHERE ec = 'EC 3.1.3.62';
UPDATE enzyme SET sys_name = '2-carboxy-D-arabinitol-1-phosphate 1-phosphohydrolase' WHERE ec = 'EC 3.1.3.63';
UPDATE enzyme SET history = 'EC 3.1.3.63 created 1992' WHERE ec = 'EC 3.1.3.63';
UPDATE enzyme SET sys_name = '1-phosphatidyl-1D-myo-inositol-3-phosphate 3-phosphohydrolase' WHERE ec = 'EC 3.1.3.64';
UPDATE enzyme SET history = 'EC 3.1.3.64 created 1992, [EC 3.1.3.65 created 1992, incorporated 2002], modified 2002' WHERE ec = 'EC 3.1.3.64';
UPDATE enzyme SET history = 'EC 3.1.3.65 created 1992, deleted 2002' WHERE ec = 'EC 3.1.3.65';
UPDATE enzyme SET sys_name = '1-phosphatidyl-1D-myo-inositol-3,4-bisphosphate 4-phosphohydrolase' WHERE ec = 'EC 3.1.3.66';
UPDATE enzyme SET history = 'EC 3.1.3.66 created 1992, modified 2002' WHERE ec = 'EC 3.1.3.66';
UPDATE enzyme SET sys_name = '1-phosphatidyl-1D-myo-inositol-3,4,5-trisphosphate 3-phosphohydrolase' WHERE ec = 'EC 3.1.3.67';
UPDATE enzyme SET history = 'EC 3.1.3.67 created 1999, modified 2002' WHERE ec = 'EC 3.1.3.67';
UPDATE enzyme SET sys_name = '2-deoxy-D-glucose-6-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.68';
UPDATE enzyme SET history = 'EC 3.1.3.68 created 1999' WHERE ec = 'EC 3.1.3.68';
UPDATE enzyme SET sys_name = '2-(beta-D-glucosyl)-sn-glycerol-3-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.69';
UPDATE enzyme SET history = 'EC 3.1.3.69 created 2001' WHERE ec = 'EC 3.1.3.69';
UPDATE enzyme SET sys_name = 'Alpha-D-mannosyl-3-phosphoglycerate phosphohydrolase' WHERE ec = 'EC 3.1.3.70';
UPDATE enzyme SET history = 'EC 3.1.3.70 created 2002' WHERE ec = 'EC 3.1.3.70';
UPDATE enzyme SET sys_name = '(R)-2-phospho-3-sulfolactate phosphohydrolase' WHERE ec = 'EC 3.1.3.71';
UPDATE enzyme SET history = 'EC 3.1.3.71 created 2002' WHERE ec = 'EC 3.1.3.71';
UPDATE enzyme SET sys_name = 'Myo-inositol-hexakisphosphate 5-phosphohydrolase' WHERE ec = 'EC 3.1.3.72';
UPDATE enzyme SET history = 'EC 3.1.3.72 created 2002' WHERE ec = 'EC 3.1.3.72';
UPDATE enzyme SET sys_name = 'Adenosylcobalamin/alpha-ribazole-5-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.73';
UPDATE enzyme SET history = 'EC 3.1.3.73 created 2004, modified 2011' WHERE ec = 'EC 3.1.3.73';
UPDATE enzyme SET sys_name = 'Pyridoxal-5-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.74';
UPDATE enzyme SET history = 'EC 3.1.3.74 created 2004' WHERE ec = 'EC 3.1.3.74';
UPDATE enzyme SET sys_name = 'Phosphoethanolamine phosphohydrolase' WHERE ec = 'EC 3.1.3.75';
UPDATE enzyme SET history = 'EC 3.1.3.75 created 2004' WHERE ec = 'EC 3.1.3.75';
UPDATE enzyme SET sys_name = '(9S,10S)-10-hydroxy-9-(phosphonooxy)octadecanoate phosphohydrolase' WHERE ec = 'EC 3.1.3.76';
UPDATE enzyme SET history = 'EC 3.1.3.76 created 2006' WHERE ec = 'EC 3.1.3.76';
UPDATE enzyme SET sys_name = '5-(methylthio)-2,3-dioxopentyl-phosphate phosphohydrolase (isomerizing)' WHERE ec = 'EC 3.1.3.77';
UPDATE enzyme SET history = 'EC 3.1.3.77 created 2006' WHERE ec = 'EC 3.1.3.77';
UPDATE enzyme SET sys_name = '1-phosphatidyl-1D-myo-inositol-4,5-bisphosphate 4-phosphohydrolase' WHERE ec = 'EC 3.1.3.78';
UPDATE enzyme SET history = 'EC 3.1.3.78 created 2008' WHERE ec = 'EC 3.1.3.78';
UPDATE enzyme SET sys_name = 'Beta-D-fructofuranosyl-alpha-D-mannopyranoside-6(F)-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.79';
UPDATE enzyme SET history = 'EC 3.1.3.79 created 2009' WHERE ec = 'EC 3.1.3.79';
UPDATE enzyme SET sys_name = '2,3-bisphospho-D-glycerate 3-phosphohydrolase' WHERE ec = 'EC 3.1.3.80';
UPDATE enzyme SET history = 'EC 3.1.3.80 created 2010' WHERE ec = 'EC 3.1.3.80';
UPDATE enzyme SET sys_name = '1,2-diacyl-sn-glycerol 3-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.81';
UPDATE enzyme SET history = 'EC 3.1.3.81 created 2010' WHERE ec = 'EC 3.1.3.81';
UPDATE enzyme SET sys_name = 'D-glycero-beta-D-manno-heptose 1,7-bisphosphate 7-phosphohydrolase' WHERE ec = 'EC 3.1.3.82';
UPDATE enzyme SET history = 'EC 3.1.3.82 created 2010' WHERE ec = 'EC 3.1.3.82';
UPDATE enzyme SET sys_name = 'D-glycero-alpha-D-manno-heptose-1,7-bisphosphate 7-phosphohydrolase' WHERE ec = 'EC 3.1.3.83';
UPDATE enzyme SET history = 'EC 3.1.3.83 created 2010' WHERE ec = 'EC 3.1.3.83';
UPDATE enzyme SET sys_name = 'ADP-D-ribose 1-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.84';
UPDATE enzyme SET history = 'EC 3.1.3.84 created 2011' WHERE ec = 'EC 3.1.3.84';
UPDATE enzyme SET sys_name = 'Alpha-D-glucosyl-3-phospho-D-glycerate phosphohydrolase' WHERE ec = 'EC 3.1.3.85';
UPDATE enzyme SET history = 'EC 3.1.3.85 created 2011' WHERE ec = 'EC 3.1.3.85';
UPDATE enzyme SET sys_name = '1-phosphatidyl-1D-myo-inositol-3,4,5-trisphosphate 5-phosphohydrolase' WHERE ec = 'EC 3.1.3.86';
UPDATE enzyme SET history = 'EC 3.1.3.86 created 2011' WHERE ec = 'EC 3.1.3.86';
UPDATE enzyme SET sys_name = '2-hydroxy-5-(methylthio)-3-oxopent-1-enyl phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.87';
UPDATE enzyme SET history = 'EC 3.1.3.87 created 2012' WHERE ec = 'EC 3.1.3.87';
UPDATE enzyme SET sys_name = '5-phosphoribostamycin phosphohydrolase' WHERE ec = 'EC 3.1.3.88';
UPDATE enzyme SET history = 'EC 3.1.3.88 created 2012' WHERE ec = 'EC 3.1.3.88';
UPDATE enzyme SET sys_name = '2-deoxyribonucleoside 5-monophosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.89';
UPDATE enzyme SET history = 'EC 3.1.3.89 created 2013' WHERE ec = 'EC 3.1.3.89';
UPDATE enzyme SET sys_name = 'Maltose 6-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.90';
UPDATE enzyme SET history = 'EC 3.1.3.90 created 2013' WHERE ec = 'EC 3.1.3.90';
UPDATE enzyme SET sys_name = 'N(7)-methyl-GMP phosphohydrolase' WHERE ec = 'EC 3.1.3.91';
UPDATE enzyme SET history = 'EC 3.1.3.91 created 2013' WHERE ec = 'EC 3.1.3.91';
UPDATE enzyme SET sys_name = 'Kanosamine-6-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.92';
UPDATE enzyme SET history = 'EC 3.1.3.92 created 2013' WHERE ec = 'EC 3.1.3.92';
UPDATE enzyme SET sys_name = 'Beta-L-galactose-1-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.93';
UPDATE enzyme SET history = 'EC 3.1.3.93, created 2014' WHERE ec = 'EC 3.1.3.93';
UPDATE enzyme SET sys_name = 'Alpha-D-galactose-1-phosphate phosphohydrolase' WHERE ec = 'EC 3.1.3.94';
UPDATE enzyme SET history = 'EC 3.1.3.94 created 2014' WHERE ec = 'EC 3.1.3.94';
UPDATE enzyme SET sys_name = '1-phosphatidyl-1D-myo-inositol-3,5-bisphosphate 3-phosphohydrolase' WHERE ec = 'EC 3.1.3.95';
UPDATE enzyme SET history = 'EC 3.1.3.95 created 2014' WHERE ec = 'EC 3.1.3.95';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.1.3.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.1.3.n2';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.1.3.n4';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.1.3.n5';
UPDATE enzyme SET sys_name = 'Oligonucleotide 5-nucleotidohydrolase' WHERE ec = 'EC 3.1.4.1';
UPDATE enzyme SET history = 'EC 3.1.4.1 created 1961' WHERE ec = 'EC 3.1.4.1';
UPDATE enzyme SET sys_name = 'sn-glycero-3-phosphocholine glycerophosphohydrolase' WHERE ec = 'EC 3.1.4.2';
UPDATE enzyme SET history = 'EC 3.1.4.2 created 1961, modified 1976' WHERE ec = 'EC 3.1.4.2';
UPDATE enzyme SET sys_name = 'Phosphatidylcholine cholinephosphohydrolase' WHERE ec = 'EC 3.1.4.3';
UPDATE enzyme SET history = 'EC 3.1.4.3 created 1961' WHERE ec = 'EC 3.1.4.3';
UPDATE enzyme SET sys_name = 'Phosphatidylcholine phosphatidohydrolase' WHERE ec = 'EC 3.1.4.4';
UPDATE enzyme SET history = 'EC 3.1.4.4 created 1961' WHERE ec = 'EC 3.1.4.4';
UPDATE enzyme SET history = 'EC 3.1.4.5 created 1961, deleted 1978' WHERE ec = 'EC 3.1.4.5';
UPDATE enzyme SET history = 'EC 3.1.4.6 created 1961, deleted 1978' WHERE ec = 'EC 3.1.4.6';
UPDATE enzyme SET history = 'EC 3.1.4.7 created 1961, deleted 1978' WHERE ec = 'EC 3.1.4.7';
UPDATE enzyme SET history = 'EC 3.1.4.8 created 1961, deleted 1965, reinstated 1972, deleted 1978' WHERE ec = 'EC 3.1.4.8';
UPDATE enzyme SET history = 'EC 3.1.4.9 created 1965, deleted 1978' WHERE ec = 'EC 3.1.4.9';
UPDATE enzyme SET history = 'EC 3.1.4.10 created 1972, modified 1976, deleted 2002' WHERE ec = 'EC 3.1.4.10';
UPDATE enzyme SET sys_name = '1-phosphatidyl-1D-myo-inositol-4,5-bisphosphate inositoltrisphosphohydrolase' WHERE ec = 'EC 3.1.4.11';
UPDATE enzyme SET history = 'EC 3.1.4.11 created 1972, modified 2002' WHERE ec = 'EC 3.1.4.11';
UPDATE enzyme SET sys_name = 'Sphingomyelin cholinephosphohydrolase' WHERE ec = 'EC 3.1.4.12';
UPDATE enzyme SET history = 'EC 3.1.4.12 created 1972' WHERE ec = 'EC 3.1.4.12';
UPDATE enzyme SET sys_name = 'Serine-phosphoethanolamine ethanolaminephosphohydrolase' WHERE ec = 'EC 3.1.4.13';
UPDATE enzyme SET history = 'EC 3.1.4.13 created 1972, modified 1976' WHERE ec = 'EC 3.1.4.13';
UPDATE enzyme SET sys_name = 'Holo-[acyl-carrier-protein] 4-pantetheine-phosphohydrolase' WHERE ec = 'EC 3.1.4.14';
UPDATE enzyme SET history = 'EC 3.1.4.14 created 1972, modified 2006' WHERE ec = 'EC 3.1.4.14';
UPDATE enzyme SET sys_name = 'Adenylyl-[L-glutamate:ammonia ligase (ADP-forming)] adenylylhydrolase' WHERE ec = 'EC 3.1.4.15';
UPDATE enzyme SET history = 'EC 3.1.4.15 created 1972' WHERE ec = 'EC 3.1.4.15';
UPDATE enzyme SET sys_name = 'Nucleoside-2,3-cyclic-phosphate 3-nucleotidohydrolase' WHERE ec = 'EC 3.1.4.16';
UPDATE enzyme SET history = 'EC 3.1.4.16 created 1972, modified 1976' WHERE ec = 'EC 3.1.4.16';
UPDATE enzyme SET sys_name = '3,5-cyclic-nucleotide 5-nucleotidohydrolase' WHERE ec = 'EC 3.1.4.17';
UPDATE enzyme SET history = 'EC 3.1.4.17 created 1972, modified 1976' WHERE ec = 'EC 3.1.4.17';
UPDATE enzyme SET history = 'EC 3.1.4.18 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.18';
UPDATE enzyme SET history = 'EC 3.1.4.19 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.19';
UPDATE enzyme SET history = 'EC 3.1.4.20 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.20';
UPDATE enzyme SET history = 'EC 3.1.4.21 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.21';
UPDATE enzyme SET history = 'EC 3.1.4.22 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.22';
UPDATE enzyme SET history = 'EC 3.1.4.24 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.24';
UPDATE enzyme SET history = 'EC 3.1.4.25 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.25';
UPDATE enzyme SET history = 'EC 3.1.4.26 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.26';
UPDATE enzyme SET history = 'EC 3.1.4.27 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.27';
UPDATE enzyme SET history = 'EC 3.1.4.28 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.28';
UPDATE enzyme SET history = 'EC 3.1.4.29 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.29';
UPDATE enzyme SET history = 'EC 3.1.4.30 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.30';
UPDATE enzyme SET history = 'EC 3.1.4.31 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.31';
UPDATE enzyme SET history = 'EC 3.1.4.32 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.32';
UPDATE enzyme SET history = 'EC 3.1.4.33 created 1972, deleted 1978' WHERE ec = 'EC 3.1.4.33';
UPDATE enzyme SET sys_name = '3,5-cyclic-GMP 5-nucleotidohydrolase' WHERE ec = 'EC 3.1.4.35';
UPDATE enzyme SET history = 'EC 3.1.4.35 created 1976' WHERE ec = 'EC 3.1.4.35';
UPDATE enzyme SET history = 'EC 3.1.4.36 created 1976, deleted 2002' WHERE ec = 'EC 3.1.4.36';
UPDATE enzyme SET sys_name = 'Nucleoside-2,3-cyclic-phosphate 2-nucleotidohydrolase' WHERE ec = 'EC 3.1.4.37';
UPDATE enzyme SET history = 'EC 3.1.4.37 created 1976' WHERE ec = 'EC 3.1.4.37';
UPDATE enzyme SET sys_name = 'sn-glycero-3-phosphocholine cholinephosphohydrolase' WHERE ec = 'EC 3.1.4.38';
UPDATE enzyme SET history = 'EC 3.1.4.38 created 1976' WHERE ec = 'EC 3.1.4.38';
UPDATE enzyme SET sys_name = '1-alkyl-sn-glycero-3-phosphoethanolamine ethanolaminehydrolase' WHERE ec = 'EC 3.1.4.39';
UPDATE enzyme SET history = 'EC 3.1.4.39 created 1976' WHERE ec = 'EC 3.1.4.39';
UPDATE enzyme SET sys_name = 'CMP-N-acylneuraminate N-acylneuraminohydrolase' WHERE ec = 'EC 3.1.4.40';
UPDATE enzyme SET history = 'EC 3.1.4.40 created 1976' WHERE ec = 'EC 3.1.4.40';
UPDATE enzyme SET sys_name = 'Sphingomyelin ceramide-phosphohydrolase' WHERE ec = 'EC 3.1.4.41';
UPDATE enzyme SET history = 'EC 3.1.4.41 created 1978' WHERE ec = 'EC 3.1.4.41';
UPDATE enzyme SET sys_name = 'Rac-glycerol-1,2-cyclic-phosphate 2-glycerophosphohydrolase' WHERE ec = 'EC 3.1.4.42';
UPDATE enzyme SET history = 'EC 3.1.4.42 created 1984' WHERE ec = 'EC 3.1.4.42';
UPDATE enzyme SET sys_name = '1-(sn-glycero-3-phospho)-1D-myo-inositol inositolphosphohydrolase' WHERE ec = 'EC 3.1.4.43';
UPDATE enzyme SET history = 'EC 3.1.4.43 created 1984, (EC 3.1.4.36 created 1976, incorporated 2002), modified 2002' WHERE ec = 'EC 3.1.4.43';
UPDATE enzyme SET sys_name = '1-(sn-glycero-3-phospho)-1D-myo-inositol glycerophosphohydrolase' WHERE ec = 'EC 3.1.4.44';
UPDATE enzyme SET history = 'EC 3.1.4.44 created 1984, modified 2002' WHERE ec = 'EC 3.1.4.44';
UPDATE enzyme SET sys_name = 'Glycoprotein-N-acetyl-D-glucosaminyl-phospho-D-mannose N-acetyl-D-glucosaminylphosphohydrolase' WHERE ec = 'EC 3.1.4.45';
UPDATE enzyme SET history = 'EC 3.1.4.45 created 1984' WHERE ec = 'EC 3.1.4.45';
UPDATE enzyme SET sys_name = 'Glycerophosphodiester glycerophosphohydrolase' WHERE ec = 'EC 3.1.4.46';
UPDATE enzyme SET history = 'EC 3.1.4.46 created 1986' WHERE ec = 'EC 3.1.4.46';
UPDATE enzyme SET history = 'EC 3.1.4.47 created 1989, deleted 2002' WHERE ec = 'EC 3.1.4.47';
UPDATE enzyme SET sys_name = 'Dolichyl-beta-D-glucosyl-phosphate dolichylphosphohydrolase' WHERE ec = 'EC 3.1.4.48';
UPDATE enzyme SET history = 'EC 3.1.4.48 created 1989' WHERE ec = 'EC 3.1.4.48';
UPDATE enzyme SET sys_name = 'Dolichyl-beta-D-mannosyl-phosphate dolichylphosphohydrolase' WHERE ec = 'EC 3.1.4.49';
UPDATE enzyme SET history = 'EC 3.1.4.49 created 1990' WHERE ec = 'EC 3.1.4.49';
UPDATE enzyme SET sys_name = 'Glycoprotein-phosphatidylinositol phosphatidohydrolase' WHERE ec = 'EC 3.1.4.50';
UPDATE enzyme SET history = 'EC 3.1.4.50 created 1990, modified 2002' WHERE ec = 'EC 3.1.4.50';
UPDATE enzyme SET sys_name = '6-(D-glucose-1-phospho)-D-mannosylglycoprotein glucose-1-phosphohydrolase' WHERE ec = 'EC 3.1.4.51';
UPDATE enzyme SET history = 'EC 3.1.4.51 created 1992' WHERE ec = 'EC 3.1.4.51';
UPDATE enzyme SET sys_name = 'Cyclic bis(3-&gt;5)diguanylate 3-guanylylhydrolase' WHERE ec = 'EC 3.1.4.52';
UPDATE enzyme SET history = 'EC 3.1.4.52 created 2008' WHERE ec = 'EC 3.1.4.52';
UPDATE enzyme SET sys_name = '3,5-cyclic-AMP 5-nucleotidohydrolase' WHERE ec = 'EC 3.1.4.53';
UPDATE enzyme SET history = 'EC 3.1.4.53 created 2008, modifed 2011' WHERE ec = 'EC 3.1.4.53';
UPDATE enzyme SET sys_name = 'N-acetylphosphatidylethanolamine phosphatidohydrolase' WHERE ec = 'EC 3.1.4.54';
UPDATE enzyme SET history = 'EC 3.1.4.54 created 2011' WHERE ec = 'EC 3.1.4.54';
UPDATE enzyme SET sys_name = '5-phospho-alpha-D-ribose 1,2-cyclic phosphate 2-phosphohydrolase (alpha-D-ribose 1,5-bisphosphate-forming)' WHERE ec = 'EC 3.1.4.55';
UPDATE enzyme SET history = 'EC 3.1.4.55 created 2013' WHERE ec = 'EC 3.1.4.55';
UPDATE enzyme SET sys_name = '7,8-dihydroneopterin 2,3-cyclic phosphate 2/3-phosphodiesterase' WHERE ec = 'EC 3.1.4.56';
UPDATE enzyme SET history = 'EC 3.1.4.56 created 2013' WHERE ec = 'EC 3.1.4.56';
UPDATE enzyme SET sys_name = 'dGTP triphosphohydrolase' WHERE ec = 'EC 3.1.5.1';
UPDATE enzyme SET history = 'EC 3.1.5.1 created 1961' WHERE ec = 'EC 3.1.5.1';
UPDATE enzyme SET sys_name = 'Aryl-sulfate sulfohydrolase' WHERE ec = 'EC 3.1.6.1';
UPDATE enzyme SET history = 'EC 3.1.6.1 created 1961, modified 2011' WHERE ec = 'EC 3.1.6.1';
UPDATE enzyme SET sys_name = 'Steryl-sulfate sulfohydrolase' WHERE ec = 'EC 3.1.6.2';
UPDATE enzyme SET history = 'EC 3.1.6.2 created 1961' WHERE ec = 'EC 3.1.6.2';
UPDATE enzyme SET sys_name = 'Sugar-sulfate sulfohydrolase' WHERE ec = 'EC 3.1.6.3';
UPDATE enzyme SET history = 'EC 3.1.6.3 created 1961' WHERE ec = 'EC 3.1.6.3';
UPDATE enzyme SET sys_name = 'N-acetyl-D-galactosamine-6-sulfate 6-sulfohydrolase' WHERE ec = 'EC 3.1.6.4';
UPDATE enzyme SET history = 'EC 3.1.6.4 created 1961' WHERE ec = 'EC 3.1.6.4';
UPDATE enzyme SET history = 'EC 3.1.6.5 created 1961, deleted 1964' WHERE ec = 'EC 3.1.6.5';
UPDATE enzyme SET sys_name = 'Choline-sulfate sulfohydrolase' WHERE ec = 'EC 3.1.6.6';
UPDATE enzyme SET history = 'EC 3.1.6.6 created 1965' WHERE ec = 'EC 3.1.6.6';
UPDATE enzyme SET sys_name = 'Cellulose-sulfate sulfohydrolase' WHERE ec = 'EC 3.1.6.7';
UPDATE enzyme SET history = 'EC 3.1.6.7 created 1965' WHERE ec = 'EC 3.1.6.7';
UPDATE enzyme SET sys_name = 'Cerebroside-3-sulfate 3-sulfohydrolase' WHERE ec = 'EC 3.1.6.8';
UPDATE enzyme SET history = 'EC 3.1.6.8 created 1972' WHERE ec = 'EC 3.1.6.8';
UPDATE enzyme SET sys_name = '4-deoxy-beta-D-gluc-4-enuronosyl-(1-&gt;3)-N-acetyl-D-galactosamine-4-sulfate 4-sulfohydrolase' WHERE ec = 'EC 3.1.6.9';
UPDATE enzyme SET history = 'EC 3.1.6.9 created 1972' WHERE ec = 'EC 3.1.6.9';
UPDATE enzyme SET sys_name = '4-deoxy-beta-D-gluc-4-enuronosyl-(1-&gt;3)-N-acetyl-D-galactosamine-6-sulfate 6-sulfohydrolase' WHERE ec = 'EC 3.1.6.10';
UPDATE enzyme SET history = 'EC 3.1.6.10 created 1972' WHERE ec = 'EC 3.1.6.10';
UPDATE enzyme SET sys_name = '2-N,6-O-disulfo-D-glucosamine 6-sulfohydrolase' WHERE ec = 'EC 3.1.6.11';
UPDATE enzyme SET history = 'EC 3.1.6.11 created 1972, modified 1989' WHERE ec = 'EC 3.1.6.11';
UPDATE enzyme SET sys_name = 'N-acetyl-D-galactosamine-4-sulfate 4-sulfohydrolase' WHERE ec = 'EC 3.1.6.12';
UPDATE enzyme SET history = 'EC 3.1.6.12 created 1984' WHERE ec = 'EC 3.1.6.12';
UPDATE enzyme SET sys_name = 'L-iduronate-2-sulfate 2-sulfohydrolase' WHERE ec = 'EC 3.1.6.13';
UPDATE enzyme SET history = 'EC 3.1.6.13 created 1984' WHERE ec = 'EC 3.1.6.13';
UPDATE enzyme SET sys_name = 'N-acetyl-D-glucosamine-6-sulfate 6-sulfohydrolase' WHERE ec = 'EC 3.1.6.14';
UPDATE enzyme SET history = 'EC 3.1.6.14 created 1984' WHERE ec = 'EC 3.1.6.14';
UPDATE enzyme SET sys_name = 'N-sulfo-3-sulfoglucosamine 3-sulfohydrolase' WHERE ec = 'EC 3.1.6.15';
UPDATE enzyme SET history = 'EC 3.1.6.15 created 1984, modified 1989' WHERE ec = 'EC 3.1.6.15';
UPDATE enzyme SET sys_name = 'Monomethyl-sulfate sulfohydrolase' WHERE ec = 'EC 3.1.6.16';
UPDATE enzyme SET history = 'EC 3.1.6.16 created 1989' WHERE ec = 'EC 3.1.6.16';
UPDATE enzyme SET sys_name = '(R)-2-O-sulfolactate 2-sulfohydrolase' WHERE ec = 'EC 3.1.6.17';
UPDATE enzyme SET history = 'EC 3.1.6.17 created 1989' WHERE ec = 'EC 3.1.6.17';
UPDATE enzyme SET sys_name = 'Polysaccharide-2-O-sulfo-D-glucuronate 2-sulfohydrolase' WHERE ec = 'EC 3.1.6.18';
UPDATE enzyme SET history = 'EC 3.1.6.18 created 1989' WHERE ec = 'EC 3.1.6.18';
UPDATE enzyme SET sys_name = '(R)-secondary-alkyl sulfate sulfohydrolase ((S)-secondary-alcohol forming)' WHERE ec = 'EC 3.1.6.19';
UPDATE enzyme SET history = 'EC 3.1.6.19 created 2013' WHERE ec = 'EC 3.1.6.19';
UPDATE enzyme SET sys_name = 'Prenyl-diphosphate diphosphohydrolase' WHERE ec = 'EC 3.1.7.1';
UPDATE enzyme SET history = 'EC 3.1.7.1 created 1972' WHERE ec = 'EC 3.1.7.1';
UPDATE enzyme SET sys_name = 'Guanosine-3,5-bis(diphosphate) 3-diphosphohydrolase' WHERE ec = 'EC 3.1.7.2';
UPDATE enzyme SET history = 'EC 3.1.7.2 created 1980' WHERE ec = 'EC 3.1.7.2';
UPDATE enzyme SET sys_name = 'Monoterpenyl-diphosphate diphosphohydrolase' WHERE ec = 'EC 3.1.7.3';
UPDATE enzyme SET history = 'EC 3.1.7.3 created 1984' WHERE ec = 'EC 3.1.7.3';
UPDATE enzyme SET history = 'EC 3.1.7.4 created 2008, deleted 2013' WHERE ec = 'EC 3.1.7.4';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphohydrolase' WHERE ec = 'EC 3.1.7.5';
UPDATE enzyme SET history = 'EC 3.1.7.5 created 2009' WHERE ec = 'EC 3.1.7.5';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphohydrolase' WHERE ec = 'EC 3.1.7.6';
UPDATE enzyme SET history = 'EC 3.1.7.6 created 2010' WHERE ec = 'EC 3.1.7.6';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphohydrolase (drimenol-forming)' WHERE ec = 'EC 3.1.7.7';
UPDATE enzyme SET history = 'EC 3.1.7.7 created 2011' WHERE ec = 'EC 3.1.7.7';
UPDATE enzyme SET sys_name = 'Tuberculosinyl diphosphate diphosphohydrolase (tuberculosinol forming)' WHERE ec = 'EC 3.1.7.8';
UPDATE enzyme SET history = 'EC 3.1.7.8 created 2011' WHERE ec = 'EC 3.1.7.8';
UPDATE enzyme SET sys_name = 'Tuberculosinyl diphosphate diphosphohydrolase (isotuberculosinol forming)' WHERE ec = 'EC 3.1.7.9';
UPDATE enzyme SET history = 'EC 3.1.7.9 created 2011' WHERE ec = 'EC 3.1.7.9';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate diphosphohydrolase ((13E)-labda-7,13-dien-15-ol-forming)' WHERE ec = 'EC 3.1.7.10';
UPDATE enzyme SET history = 'EC 3.1.7.10 created 2012' WHERE ec = 'EC 3.1.7.10';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphohydrolase' WHERE ec = 'EC 3.1.7.11';
UPDATE enzyme SET history = 'EC 3.1.7.11 created 2012' WHERE ec = 'EC 3.1.7.11';
UPDATE enzyme SET sys_name = 'Aryltriphosphate dialkylphosphohydrolase' WHERE ec = 'EC 3.1.8.1';
UPDATE enzyme SET history = 'EC 3.1.8.1 created 1989' WHERE ec = 'EC 3.1.8.1';
UPDATE enzyme SET sys_name = 'Diisopropyl-fluorophosphate fluorohydrolase' WHERE ec = 'EC 3.1.8.2';
UPDATE enzyme SET history = 'EC 3.1.8.2 created 1961 as EC 3.8.2.1, transferred 1992 to EC 3.1.8.2' WHERE ec = 'EC 3.1.8.2';
UPDATE enzyme SET history = 'EC 3.1.11.1 created 1972 as EC 3.1.4.25, transferred 1978 to EC 3.1.11.1' WHERE ec = 'EC 3.1.11.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.11.2';
UPDATE enzyme SET history = 'EC 3.1.11.2 created 1972 as EC 3.1.4.27, transferred 1978 to EC 3.1.11.2' WHERE ec = 'EC 3.1.11.2';
UPDATE enzyme SET history = 'EC 3.1.11.3 created 1972 as EC 3.1.4.28, transferred 1978 to EC 3.1.11.3' WHERE ec = 'EC 3.1.11.3';
UPDATE enzyme SET history = 'EC 3.1.11.4 created 1972 as EC 3.1.4.31, transferred 1978 to EC 3.1.11.4' WHERE ec = 'EC 3.1.11.4';
UPDATE enzyme SET history = 'EC 3.1.11.5 created 1978' WHERE ec = 'EC 3.1.11.5';
UPDATE enzyme SET history = 'EC 3.1.11.6 created 1978' WHERE ec = 'EC 3.1.11.6';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.13.1';
UPDATE enzyme SET history = 'EC 3.1.13.1 created 1972 as EC 3.1.4.20, transferred 1978 to EC 3.1.13.1' WHERE ec = 'EC 3.1.13.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.13.2';
UPDATE enzyme SET history = 'EC 3.1.13.2 created 1978, modified 2010' WHERE ec = 'EC 3.1.13.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.13.3';
UPDATE enzyme SET history = 'EC 3.1.13.3 created 1972 as EC 3.1.4.19, transferred 1978 to EC 3.1.13.3' WHERE ec = 'EC 3.1.13.3';
UPDATE enzyme SET history = 'EC 3.1.13.4 created 1984' WHERE ec = 'EC 3.1.13.4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.13.5';
UPDATE enzyme SET history = 'EC 3.1.13.5 created 2006' WHERE ec = 'EC 3.1.13.5';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.14.1';
UPDATE enzyme SET history = 'EC 3.1.14.1 created 1978' WHERE ec = 'EC 3.1.14.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.15.1';
UPDATE enzyme SET history = 'EC 3.1.15.1 created 1978' WHERE ec = 'EC 3.1.15.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.16.1';
UPDATE enzyme SET history = 'EC 3.1.16.1 created 1972 as EC 3.1.4.18, transferred 1978 to EC 3.1.16.1' WHERE ec = 'EC 3.1.16.1';
UPDATE enzyme SET history = 'EC 3.1.21.1 created 1961 as EC 3.1.4.5, transferred 1978 to EC 3.1.21.1, modified 1981' WHERE ec = 'EC 3.1.21.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.21.2';
UPDATE enzyme SET history = 'EC 3.1.21.2 created 1972 as EC 3.1.4.30, transferred 1978 to EC 3.1.21.2, modified 2014' WHERE ec = 'EC 3.1.21.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.21.3';
UPDATE enzyme SET history = 'EC 3.1.21.3 created 1984 from EC 3.1.23 and EC 3.1.24' WHERE ec = 'EC 3.1.21.3';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.21.4';
UPDATE enzyme SET history = 'EC 3.1.21.4 created 1984 from EC 3.1.23 and EC 3.1.24' WHERE ec = 'EC 3.1.21.4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.21.5';
UPDATE enzyme SET history = 'EC 3.1.21.5 created 1984 from EC 3.1.23 and EC 3.1.24' WHERE ec = 'EC 3.1.21.5';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.21.6';
UPDATE enzyme SET history = 'EC 3.1.21.6 created 1999' WHERE ec = 'EC 3.1.21.6';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.21.7';
UPDATE enzyme SET history = 'EC 3.1.21.7 created 1978 as EC 3.1.22.3, transferred 2001 to EC 3.1.21.7' WHERE ec = 'EC 3.1.21.7';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.21.8';
UPDATE enzyme SET history = 'EC 3.1.21.8 created 2014' WHERE ec = 'EC 3.1.21.8';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.21.9';
UPDATE enzyme SET history = 'EC 3.1.21.9 created 2014' WHERE ec = 'EC 3.1.21.9';
UPDATE enzyme SET history = 'EC 3.1.22.1 created 1961 as EC 3.1.4.6, transferred 1978 to EC 3.1.22.1, modified 1981' WHERE ec = 'EC 3.1.22.1';
UPDATE enzyme SET history = 'EC 3.1.22.2 created 1978, modified 1981' WHERE ec = 'EC 3.1.22.2';
UPDATE enzyme SET history = 'EC 3.1.22.3 created 1978, deleted 2001' WHERE ec = 'EC 3.1.22.3';
UPDATE enzyme SET history = 'EC 3.1.22.4 created 1989, modified 2003' WHERE ec = 'EC 3.1.22.4';
UPDATE enzyme SET history = 'EC 3.1.22.5 created 1992' WHERE ec = 'EC 3.1.22.5';
UPDATE enzyme SET history = 'EC 3.1.23.1 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.1';
UPDATE enzyme SET history = 'EC 3.1.23.2 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.2';
UPDATE enzyme SET history = 'EC 3.1.23.3 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.3';
UPDATE enzyme SET history = 'EC 3.1.23.4 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.4';
UPDATE enzyme SET history = 'EC 3.1.23.5 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.5';
UPDATE enzyme SET history = 'EC 3.1.23.6 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.6';
UPDATE enzyme SET history = 'EC 3.1.23.7 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.7';
UPDATE enzyme SET history = 'EC 3.1.23.8 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.8';
UPDATE enzyme SET history = 'EC 3.1.23.9 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.23.9';
UPDATE enzyme SET history = 'EC 3.1.23.10 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.10';
UPDATE enzyme SET history = 'EC 3.1.23.11 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.11';
UPDATE enzyme SET history = 'EC 3.1.23.12 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.23.12';
UPDATE enzyme SET history = 'EC 3.1.23.13 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.13';
UPDATE enzyme SET history = 'EC 3.1.23.14 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.14';
UPDATE enzyme SET history = 'EC 3.1.23.15 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.15';
UPDATE enzyme SET history = 'EC 3.1.23.16 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.16';
UPDATE enzyme SET history = 'EC 3.1.23.17 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.17';
UPDATE enzyme SET history = 'EC 3.1.23.18 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.18';
UPDATE enzyme SET history = 'EC 3.1.23.19 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.19';
UPDATE enzyme SET history = 'EC 3.1.23.20 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.20';
UPDATE enzyme SET history = 'EC 3.1.23.21 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.21';
UPDATE enzyme SET history = 'EC 3.1.23.22 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.22';
UPDATE enzyme SET history = 'EC 3.1.23.23 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.23';
UPDATE enzyme SET history = 'EC 3.1.23.24 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.24';
UPDATE enzyme SET history = 'EC 3.1.23.25 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.25';
UPDATE enzyme SET history = 'EC 3.1.23.26 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.26';
UPDATE enzyme SET history = 'EC 3.1.23.27 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.27';
UPDATE enzyme SET history = 'EC 3.1.23.28 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.28';
UPDATE enzyme SET history = 'EC 3.1.23.29 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.29';
UPDATE enzyme SET history = 'EC 3.1.23.30 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.23.30';
UPDATE enzyme SET history = 'EC 3.1.23.31 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.31';
UPDATE enzyme SET history = 'EC 3.1.23.32 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.23.32';
UPDATE enzyme SET history = 'EC 3.1.23.33 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.33';
UPDATE enzyme SET history = 'EC 3.1.23.34 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.34';
UPDATE enzyme SET history = 'EC 3.1.23.35 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.35';
UPDATE enzyme SET history = 'EC 3.1.23.36 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.36';
UPDATE enzyme SET history = 'EC 3.1.23.37 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.37';
UPDATE enzyme SET history = 'EC 3.1.23.38 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.38';
UPDATE enzyme SET history = 'EC 3.1.23.39 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.39';
UPDATE enzyme SET history = 'EC 3.1.23.40 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.40';
UPDATE enzyme SET history = 'EC 3.1.23.41 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.41';
UPDATE enzyme SET history = 'EC 3.1.23.42 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.42';
UPDATE enzyme SET history = 'EC 3.1.23.43 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.23.43';
UPDATE enzyme SET history = 'EC 3.1.23.44 created 1978, deleted 1984' WHERE ec = 'EC 3.1.23.44';
UPDATE enzyme SET history = 'EC 3.1.23.45 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.23.45';
UPDATE enzyme SET history = 'EC 3.1.23.46 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.46';
UPDATE enzyme SET history = 'EC 3.1.23.47 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.47';
UPDATE enzyme SET history = 'EC 3.1.23.48 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.48';
UPDATE enzyme SET history = 'EC 3.1.23.49 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.49';
UPDATE enzyme SET history = 'EC 3.1.23.50 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.50';
UPDATE enzyme SET history = 'EC 3.1.23.51 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.51';
UPDATE enzyme SET history = 'EC 3.1.23.52 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.52';
UPDATE enzyme SET history = 'EC 3.1.23.53 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.53';
UPDATE enzyme SET history = 'EC 3.1.23.54 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.54';
UPDATE enzyme SET history = 'EC 3.1.23.55 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.55';
UPDATE enzyme SET history = 'EC 3.1.23.56 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.56';
UPDATE enzyme SET history = 'EC 3.1.23.57 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.57';
UPDATE enzyme SET history = 'EC 3.1.23.58 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.58';
UPDATE enzyme SET history = 'EC 3.1.23.59 created 1982, deleted 1984' WHERE ec = 'EC 3.1.23.59';
UPDATE enzyme SET history = 'EC 3.1.24.1 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.24.1';
UPDATE enzyme SET history = 'EC 3.1.24.2 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.24.2';
UPDATE enzyme SET history = 'EC 3.1.24.3 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.24.3';
UPDATE enzyme SET history = 'EC 3.1.24.4 created 1978, modified 1982, deleted 1984' WHERE ec = 'EC 3.1.24.4';
UPDATE enzyme SET history = 'EC 3.1.25.1 created 1978' WHERE ec = 'EC 3.1.25.1';
UPDATE enzyme SET history = 'EC 3.1.25.2 created 1978, deleted 1992' WHERE ec = 'EC 3.1.25.2';
UPDATE enzyme SET history = 'EC 3.1.26.1 created 1978' WHERE ec = 'EC 3.1.26.1';
UPDATE enzyme SET history = 'EC 3.1.26.2 created 1978' WHERE ec = 'EC 3.1.26.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.3';
UPDATE enzyme SET history = 'EC 3.1.26.3 created 1978, modified 2006' WHERE ec = 'EC 3.1.26.3';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.4';
UPDATE enzyme SET history = 'EC 3.1.26.4 created 1978, modified 2010' WHERE ec = 'EC 3.1.26.4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.5';
UPDATE enzyme SET history = 'EC 3.1.26.5 created 1978, modified 1982' WHERE ec = 'EC 3.1.26.5';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.6';
UPDATE enzyme SET history = 'EC 3.1.26.6 created 1984' WHERE ec = 'EC 3.1.26.6';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.7';
UPDATE enzyme SET history = 'EC 3.1.26.7 created 1984' WHERE ec = 'EC 3.1.26.7';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.8';
UPDATE enzyme SET history = 'EC 3.1.26.8 created 1986' WHERE ec = 'EC 3.1.26.8';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.9';
UPDATE enzyme SET history = 'EC 3.1.26.9 created 1986' WHERE ec = 'EC 3.1.26.9';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.10';
UPDATE enzyme SET history = 'EC 3.1.26.10 created 1992' WHERE ec = 'EC 3.1.26.10';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.11';
UPDATE enzyme SET history = 'EC 3.1.27.11 created 2002' WHERE ec = 'EC 3.1.26.11';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.12';
UPDATE enzyme SET history = 'EC 3.1.26.12 created 2008' WHERE ec = 'EC 3.1.26.12';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.26.13';
UPDATE enzyme SET history = 'EC 3.1.26.13 created 2009' WHERE ec = 'EC 3.1.26.13';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.27.1';
UPDATE enzyme SET history = 'EC 3.1.27.1 created 1972 as EC 3.1.4.23, transferred 1978 to EC 3.1.27.1, modified 1981' WHERE ec = 'EC 3.1.27.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.27.2';
UPDATE enzyme SET history = 'EC 3.1.27.2 created 1978' WHERE ec = 'EC 3.1.27.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.27.3';
UPDATE enzyme SET history = 'EC 3.1.27.3 created 1961 as EC 3.1.4.8, deleted 1965, reinstated 1972, , transferred 1978 to EC 3.1.27.3, modified 1981' WHERE ec = 'EC 3.1.27.3';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.27.4';
UPDATE enzyme SET history = 'EC 3.1.27.4 created 1978, modified 1981' WHERE ec = 'EC 3.1.27.4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.27.5';
UPDATE enzyme SET history = 'EC 3.1.27.5 created 1972 as EC 3.1.4.22, transferred 1978 to EC 3.1.27.5, modified 1981' WHERE ec = 'EC 3.1.27.5';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.27.6';
UPDATE enzyme SET history = 'EC 3.1.27.6 created 1978, modified 1981' WHERE ec = 'EC 3.1.27.6';
UPDATE enzyme SET history = 'EC 3.1.27.7 created 1984' WHERE ec = 'EC 3.1.27.7';
UPDATE enzyme SET history = 'EC 3.1.27.8 created 1984' WHERE ec = 'EC 3.1.27.8';
UPDATE enzyme SET history = 'EC 3.1.27.9 created 1992, deleted 2014' WHERE ec = 'EC 3.1.27.9';
UPDATE enzyme SET history = 'EC 3.1.27.10 created 1992' WHERE ec = 'EC 3.1.27.10';
UPDATE enzyme SET history = 'EC 3.1.30.1 created 1972 as EC 3.1.4.21, transferred 1978 to EC 3.1.30.1, modified 1981' WHERE ec = 'EC 3.1.30.1';
UPDATE enzyme SET history = 'EC 3.1.30.2 created 1965 as EC 3.1.4.9, transferred 1978 to EC 3.1.30.2, modified 1981' WHERE ec = 'EC 3.1.30.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.1.31.1';
UPDATE enzyme SET history = 'EC 3.1.31.1 created 1961 as EC 3.1.4.7, transferred 1978 to EC 3.1.31.1, modified 1981' WHERE ec = 'EC 3.1.31.1';
UPDATE enzyme SET sys_name = '4-alpha-D-glucan glucanohydrolase' WHERE ec = 'EC 3.2.1.1';
UPDATE enzyme SET history = 'EC 3.2.1.1 created 1961' WHERE ec = 'EC 3.2.1.1';
UPDATE enzyme SET sys_name = '4-alpha-D-glucan maltohydrolase' WHERE ec = 'EC 3.2.1.2';
UPDATE enzyme SET history = 'EC 3.2.1.2 created 1961' WHERE ec = 'EC 3.2.1.2';
UPDATE enzyme SET sys_name = '4-alpha-D-glucan glucohydrolase' WHERE ec = 'EC 3.2.1.3';
UPDATE enzyme SET history = 'EC 3.2.1.3 created 1961' WHERE ec = 'EC 3.2.1.3';
UPDATE enzyme SET sys_name = '4-beta-D-glucan 4-glucanohydrolase' WHERE ec = 'EC 3.2.1.4';
UPDATE enzyme SET history = 'EC 3.2.1.4 created 1961, modified 2001' WHERE ec = 'EC 3.2.1.4';
UPDATE enzyme SET history = 'EC 3.2.1.5 created 1961, deleted 1964' WHERE ec = 'EC 3.2.1.5';
UPDATE enzyme SET sys_name = '3(or 4)-beta-D-glucan 3(4)-glucanohydrolase' WHERE ec = 'EC 3.2.1.6';
UPDATE enzyme SET history = 'EC 3.2.1.6 created 1961, modified 1976' WHERE ec = 'EC 3.2.1.6';
UPDATE enzyme SET sys_name = '1-beta-D-fructan fructanohydrolase' WHERE ec = 'EC 3.2.1.7';
UPDATE enzyme SET history = 'EC 3.2.1.7 created 1961' WHERE ec = 'EC 3.2.1.7';
UPDATE enzyme SET sys_name = '4-beta-D-xylan xylanohydrolase' WHERE ec = 'EC 3.2.1.8';
UPDATE enzyme SET history = 'EC 3.2.1.8 created 1961' WHERE ec = 'EC 3.2.1.8';
UPDATE enzyme SET history = 'EC 3.2.1.9 created 1961, deleted 1972' WHERE ec = 'EC 3.2.1.9';
UPDATE enzyme SET sys_name = 'Oligosaccharide alpha-1,6-glucohydrolase' WHERE ec = 'EC 3.2.1.10';
UPDATE enzyme SET history = 'EC 3.2.1.10 created 1961, modified 2000' WHERE ec = 'EC 3.2.1.10';
UPDATE enzyme SET sys_name = '6-alpha-D-glucan 6-glucanohydrolase' WHERE ec = 'EC 3.2.1.11';
UPDATE enzyme SET history = 'EC 3.2.1.11 created 1961' WHERE ec = 'EC 3.2.1.11';
UPDATE enzyme SET history = 'EC 3.2.1.12 created 1961, deleted 1976' WHERE ec = 'EC 3.2.1.12';
UPDATE enzyme SET history = 'EC 3.2.1.13 created 1961, deleted 1976' WHERE ec = 'EC 3.2.1.13';
UPDATE enzyme SET sys_name = '(1-&gt;4)-2-acetamido-2-deoxy-beta-D-glucan glycanohydrolase' WHERE ec = 'EC 3.2.1.14';
UPDATE enzyme SET history = 'EC 3.2.1.14 created 1961' WHERE ec = 'EC 3.2.1.14';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-galacturonan glycanohydrolase' WHERE ec = 'EC 3.2.1.15';
UPDATE enzyme SET history = 'EC 3.2.1.15 created 1961' WHERE ec = 'EC 3.2.1.15';
UPDATE enzyme SET sys_name = 'Peptidoglycan N-acetylmuramoylhydrolase' WHERE ec = 'EC 3.2.1.17';
UPDATE enzyme SET history = 'EC 3.2.1.17 created 1961' WHERE ec = 'EC 3.2.1.17';
UPDATE enzyme SET sys_name = 'Acetylneuraminyl hydrolase' WHERE ec = 'EC 3.2.1.18';
UPDATE enzyme SET history = 'EC 3.2.1.18 created 1961, modified 1999' WHERE ec = 'EC 3.2.1.18';
UPDATE enzyme SET history = 'EC 3.2.1.19 created 1961, deleted 1978' WHERE ec = 'EC 3.2.1.19';
UPDATE enzyme SET sys_name = 'Alpha-D-glucoside glucohydrolase' WHERE ec = 'EC 3.2.1.20';
UPDATE enzyme SET history = 'EC 3.2.1.20 created 1961' WHERE ec = 'EC 3.2.1.20';
UPDATE enzyme SET sys_name = 'Beta-D-glucoside glucohydrolase' WHERE ec = 'EC 3.2.1.21';
UPDATE enzyme SET history = 'EC 3.2.1.21 created 1961' WHERE ec = 'EC 3.2.1.21';
UPDATE enzyme SET sys_name = 'Alpha-D-galactoside galactohydrolase' WHERE ec = 'EC 3.2.1.22';
UPDATE enzyme SET history = 'EC 3.2.1.22 created 1961' WHERE ec = 'EC 3.2.1.22';
UPDATE enzyme SET sys_name = 'Beta-D-galactoside galactohydrolase' WHERE ec = 'EC 3.2.1.23';
UPDATE enzyme SET history = 'EC 3.2.1.23 created 1961, modified 1980' WHERE ec = 'EC 3.2.1.23';
UPDATE enzyme SET sys_name = 'Alpha-D-mannoside mannohydrolase' WHERE ec = 'EC 3.2.1.24';
UPDATE enzyme SET history = 'EC 3.2.1.24 created 1961' WHERE ec = 'EC 3.2.1.24';
UPDATE enzyme SET sys_name = 'Beta-D-mannoside mannohydrolase' WHERE ec = 'EC 3.2.1.25';
UPDATE enzyme SET history = 'EC 3.2.1.25 created 1961' WHERE ec = 'EC 3.2.1.25';
UPDATE enzyme SET sys_name = 'Beta-D-fructofuranoside fructohydrolase' WHERE ec = 'EC 3.2.1.26';
UPDATE enzyme SET history = 'EC 3.2.1.26 created 1961' WHERE ec = 'EC 3.2.1.26';
UPDATE enzyme SET history = 'EC 3.2.1.27 created 1961, deleted 1972' WHERE ec = 'EC 3.2.1.27';
UPDATE enzyme SET sys_name = 'Alpha,alpha-trehalose glucohydrolase' WHERE ec = 'EC 3.2.1.28';
UPDATE enzyme SET history = 'EC 3.2.1.28 created 1961, modified 2012' WHERE ec = 'EC 3.2.1.28';
UPDATE enzyme SET history = 'EC 3.2.1.29 created 1961, deleted 1972' WHERE ec = 'EC 3.2.1.29';
UPDATE enzyme SET history = 'EC 3.2.1.30 created 1961, deleted 1992' WHERE ec = 'EC 3.2.1.30';
UPDATE enzyme SET sys_name = 'Beta-D-glucuronoside glucuronosohydrolase' WHERE ec = 'EC 3.2.1.31';
UPDATE enzyme SET history = 'EC 3.2.1.31 created 1961' WHERE ec = 'EC 3.2.1.31';
UPDATE enzyme SET sys_name = '3-beta-D-xylan xylanohydrolase' WHERE ec = 'EC 3.2.1.32';
UPDATE enzyme SET history = 'EC 3.2.1.32 created 1965, modified 2011' WHERE ec = 'EC 3.2.1.32';
UPDATE enzyme SET sys_name = 'Glycogen phosphorylase-limit dextrin 6-alpha-glucohydrolase' WHERE ec = 'EC 3.2.1.33';
UPDATE enzyme SET history = 'EC 3.2.1.33 created 1965, modified 2000' WHERE ec = 'EC 3.2.1.33';
UPDATE enzyme SET history = 'EC 3.2.1.34 created 1965, deleted 1972' WHERE ec = 'EC 3.2.1.34';
UPDATE enzyme SET sys_name = 'Hyaluronate 4-glycanohydrolase' WHERE ec = 'EC 3.2.1.35';
UPDATE enzyme SET history = 'EC 3.2.1.35 created 1965, modified 1976, modified 2001 (EC 3.2.1.34 created 1965, incorporated 1972)' WHERE ec = 'EC 3.2.1.35';
UPDATE enzyme SET sys_name = 'Hyaluronate 3-glycanohydrolase' WHERE ec = 'EC 3.2.1.36';
UPDATE enzyme SET history = 'EC 3.2.1.36 created 1965, modified 1980' WHERE ec = 'EC 3.2.1.36';
UPDATE enzyme SET sys_name = '4-beta-D-xylan xylohydrolase' WHERE ec = 'EC 3.2.1.37';
UPDATE enzyme SET history = 'EC 3.2.1.37 created 1965' WHERE ec = 'EC 3.2.1.37';
UPDATE enzyme SET sys_name = 'Beta-D-fucoside fucohydrolase' WHERE ec = 'EC 3.2.1.38';
UPDATE enzyme SET history = 'EC 3.2.1.38 created 1965, deleted 1972, reinstated 1978' WHERE ec = 'EC 3.2.1.38';
UPDATE enzyme SET sys_name = '3-beta-D-glucan glucanohydrolase' WHERE ec = 'EC 3.2.1.39';
UPDATE enzyme SET history = 'EC 3.2.1.39 created 1965' WHERE ec = 'EC 3.2.1.39';
UPDATE enzyme SET sys_name = 'Alpha-L-rhamnoside rhamnohydrolase' WHERE ec = 'EC 3.2.1.40';
UPDATE enzyme SET history = 'EC 3.2.1.40 created 1972' WHERE ec = 'EC 3.2.1.40';
UPDATE enzyme SET sys_name = 'Pullulan 6-alpha-glucanohydrolase' WHERE ec = 'EC 3.2.1.41';
UPDATE enzyme SET history = 'EC 3.2.1.41 created 1972, modified 1976, modified 2000 (EC 3.2.1.69 created 1972, incorporated 1976)' WHERE ec = 'EC 3.2.1.41';
UPDATE enzyme SET sys_name = 'GDP-glucose glucohydrolase' WHERE ec = 'EC 3.2.1.42';
UPDATE enzyme SET history = 'EC 3.2.1.42 created 1972' WHERE ec = 'EC 3.2.1.42';
UPDATE enzyme SET sys_name = 'Beta-L-rhamnoside rhamnohydrolase' WHERE ec = 'EC 3.2.1.43';
UPDATE enzyme SET history = 'EC 3.2.1.43 created 1972' WHERE ec = 'EC 3.2.1.43';
UPDATE enzyme SET sys_name = 'Poly((1-&gt;2)-alpha-L-fucoside-4-sulfate) glycanohydrolase' WHERE ec = 'EC 3.2.1.44';
UPDATE enzyme SET history = 'EC 3.2.1.44 created 1972' WHERE ec = 'EC 3.2.1.44';
UPDATE enzyme SET sys_name = 'D-glucosyl-N-acylsphingosine glucohydrolase' WHERE ec = 'EC 3.2.1.45';
UPDATE enzyme SET history = 'EC 3.2.1.45 created 1972' WHERE ec = 'EC 3.2.1.45';
UPDATE enzyme SET sys_name = 'D-galactosyl-N-acylsphingosine galactohydrolase' WHERE ec = 'EC 3.2.1.46';
UPDATE enzyme SET history = 'EC 3.2.1.46 created 1972' WHERE ec = 'EC 3.2.1.46';
UPDATE enzyme SET sys_name = 'Alpha-D-galactosyl-(1-&gt;4)-beta-D-galactosyl-(1-&gt;4)-beta-D-glucosyl-(1&lt;-&gt;1)-ceramide galactohydrolase' WHERE ec = 'EC 3.2.1.47';
UPDATE enzyme SET history = 'EC 3.2.1.47 created 1972, modified 2011' WHERE ec = 'EC 3.2.1.47';
UPDATE enzyme SET sys_name = 'Sucrose-alpha-D-glucohydrolase' WHERE ec = 'EC 3.2.1.48';
UPDATE enzyme SET history = 'EC 3.2.1.48 created 1972' WHERE ec = 'EC 3.2.1.48';
UPDATE enzyme SET sys_name = 'Alpha-N-acetyl-D-galactosaminide N-acetylgalactosaminohydrolase' WHERE ec = 'EC 3.2.1.49';
UPDATE enzyme SET history = 'EC 3.2.1.49 created 1972, modified 2011' WHERE ec = 'EC 3.2.1.49';
UPDATE enzyme SET sys_name = 'Alpha-N-acetyl-D-glucosaminide N-acetylglucosaminohydrolase' WHERE ec = 'EC 3.2.1.50';
UPDATE enzyme SET history = 'EC 3.2.1.50 created 1972' WHERE ec = 'EC 3.2.1.50';
UPDATE enzyme SET sys_name = 'Alpha-L-fucoside fucohydrolase' WHERE ec = 'EC 3.2.1.51';
UPDATE enzyme SET history = 'EC 3.2.1.51 created 1972' WHERE ec = 'EC 3.2.1.51';
UPDATE enzyme SET sys_name = 'Beta-N-acetyl-D-hexosaminide N-acetylhexosaminohydrolase' WHERE ec = 'EC 3.2.1.52';
UPDATE enzyme SET history = 'EC 3.2.1.52 created 1972 (EC 3.2.1.30 created 1961, incorporated 1992 [EC 3.2.1.29 created 1961, incorporated 1972])' WHERE ec = 'EC 3.2.1.52';
UPDATE enzyme SET sys_name = 'Beta-N-acetyl-D-galactosaminide N-acetylgalactosaminohydrolase' WHERE ec = 'EC 3.2.1.53';
UPDATE enzyme SET history = 'EC 3.2.1.53 created 1972' WHERE ec = 'EC 3.2.1.53';
UPDATE enzyme SET sys_name = 'Cyclomaltodextrin dextrin-hydrolase (decyclizing)' WHERE ec = 'EC 3.2.1.54';
UPDATE enzyme SET history = 'EC 3.2.1.54 created 1972 (EC 3.2.1.12 and EC 3.2.1.13 both created 1961 and incorporated 1976)' WHERE ec = 'EC 3.2.1.54';
UPDATE enzyme SET sys_name = 'Alpha-L-arabinofuranoside non-reducing end alpha-L-arabinofuranosidase' WHERE ec = 'EC 3.2.1.55';
UPDATE enzyme SET history = 'EC 3.2.1.55 created 1972, modified 1976 (EC 3.2.1.79 created 1972, incorporated 1976), modified 2013' WHERE ec = 'EC 3.2.1.55';
UPDATE enzyme SET sys_name = '3-D-glucuronsyl-N(2),6-disulfo-beta-D-glucosamine glucuronohydrolase' WHERE ec = 'EC 3.2.1.56';
UPDATE enzyme SET history = 'EC 3.2.1.56 created 1972' WHERE ec = 'EC 3.2.1.56';
UPDATE enzyme SET sys_name = 'Pullulan 4-glucanohydrolase (isopanose-forming)' WHERE ec = 'EC 3.2.1.57';
UPDATE enzyme SET history = 'EC 3.2.1.57 created 1972' WHERE ec = 'EC 3.2.1.57';
UPDATE enzyme SET sys_name = '3-beta-D-glucan glucohydrolase' WHERE ec = 'EC 3.2.1.58';
UPDATE enzyme SET history = 'EC 3.2.1.58 created 1972' WHERE ec = 'EC 3.2.1.58';
UPDATE enzyme SET sys_name = '3-alpha-D-glucan 3-glucanohydrolase' WHERE ec = 'EC 3.2.1.59';
UPDATE enzyme SET history = 'EC 3.2.1.59 created 1972' WHERE ec = 'EC 3.2.1.59';
UPDATE enzyme SET sys_name = '4-alpha-D-glucan maltotetraohydrolase' WHERE ec = 'EC 3.2.1.60';
UPDATE enzyme SET history = 'EC 3.2.1.60 created 1972' WHERE ec = 'EC 3.2.1.60';
UPDATE enzyme SET sys_name = '(1-&gt;3)-(1-&gt;4)-alpha-D-glucan 4-glucanohydrolase' WHERE ec = 'EC 3.2.1.61';
UPDATE enzyme SET history = 'EC 3.2.1.61 created 1972' WHERE ec = 'EC 3.2.1.61';
UPDATE enzyme SET sys_name = 'Glycosyl-N-acylsphingosine glycohydrolase' WHERE ec = 'EC 3.2.1.62';
UPDATE enzyme SET history = 'EC 3.2.1.62 created 1972, modified 1976' WHERE ec = 'EC 3.2.1.62';
UPDATE enzyme SET sys_name = '2-alpha-L-fucopyranosyl-beta-D-galactoside fucohydrolase' WHERE ec = 'EC 3.2.1.63';
UPDATE enzyme SET history = 'EC 3.2.1.63 created 1972' WHERE ec = 'EC 3.2.1.63';
UPDATE enzyme SET sys_name = '(2-&gt;6)-beta-D-fructofuranan 6-(beta-D-fructosyl)-D-fructose-hydrolase' WHERE ec = 'EC 3.2.1.64';
UPDATE enzyme SET history = 'EC 3.2.1.64 created 1972, modified 2004' WHERE ec = 'EC 3.2.1.64';
UPDATE enzyme SET sys_name = '(2-&gt;6)-beta-D-fructan fructanohydrolase' WHERE ec = 'EC 3.2.1.65';
UPDATE enzyme SET history = 'EC 3.2.1.65 created 1972' WHERE ec = 'EC 3.2.1.65';
UPDATE enzyme SET sys_name = 'Quercitrin 3-L-rhamnohydrolase' WHERE ec = 'EC 3.2.1.66';
UPDATE enzyme SET history = 'EC 3.2.1.66 created 1972' WHERE ec = 'EC 3.2.1.66';
UPDATE enzyme SET sys_name = 'Poly((1-&gt;4)-alpha-D-galacturonide) galacturonohydrolase' WHERE ec = 'EC 3.2.1.67';
UPDATE enzyme SET history = 'EC 3.2.1.67 created 1972' WHERE ec = 'EC 3.2.1.67';
UPDATE enzyme SET sys_name = 'Glycogen 6-alpha-D-glucanohydrolase' WHERE ec = 'EC 3.2.1.68';
UPDATE enzyme SET history = 'EC 3.2.1.68 created 1972, modified 1976, modified 2000' WHERE ec = 'EC 3.2.1.68';
UPDATE enzyme SET history = 'EC 3.2.1.69 created 1972, deleted 1976' WHERE ec = 'EC 3.2.1.69';
UPDATE enzyme SET sys_name = 'Glucan 6-alpha-D-glucohydrolase' WHERE ec = 'EC 3.2.1.70';
UPDATE enzyme SET history = 'EC 3.2.1.70 created 1972, modified 2001' WHERE ec = 'EC 3.2.1.70';
UPDATE enzyme SET sys_name = '2-beta-D-glucan glucanohydrolase' WHERE ec = 'EC 3.2.1.71';
UPDATE enzyme SET history = 'EC 3.2.1.71 created 1972' WHERE ec = 'EC 3.2.1.71';
UPDATE enzyme SET sys_name = '3-beta-D-xylan xylohydrolase' WHERE ec = 'EC 3.2.1.72';
UPDATE enzyme SET history = 'EC 3.2.1.72 created 1972' WHERE ec = 'EC 3.2.1.72';
UPDATE enzyme SET sys_name = '(1-&gt;3)-(1-&gt;4)-beta-D-glucan 4-glucanohydrolase' WHERE ec = 'EC 3.2.1.73';
UPDATE enzyme SET history = 'EC 3.2.1.73 created 1972' WHERE ec = 'EC 3.2.1.73';
UPDATE enzyme SET sys_name = '4-beta-D-glucan glucohydrolase' WHERE ec = 'EC 3.2.1.74';
UPDATE enzyme SET history = 'EC 3.2.1.74 created 1972' WHERE ec = 'EC 3.2.1.74';
UPDATE enzyme SET sys_name = '6-beta-D-glucan glucanohydrolase' WHERE ec = 'EC 3.2.1.75';
UPDATE enzyme SET history = 'EC 3.2.1.75 created 1972' WHERE ec = 'EC 3.2.1.75';
UPDATE enzyme SET sys_name = 'Glycosaminoglycan alpha-L-iduronohydrolase' WHERE ec = 'EC 3.2.1.76';
UPDATE enzyme SET history = 'EC 3.2.1.76 created 1972' WHERE ec = 'EC 3.2.1.76';
UPDATE enzyme SET sys_name = '(1-&gt;2)-(1-&gt;3)-alpha-D-mannan mannohydrolase' WHERE ec = 'EC 3.2.1.77';
UPDATE enzyme SET history = 'EC 3.2.1.77 created 1972' WHERE ec = 'EC 3.2.1.77';
UPDATE enzyme SET sys_name = '4-beta-D-mannan mannanohydrolase' WHERE ec = 'EC 3.2.1.78';
UPDATE enzyme SET history = 'EC 3.2.1.78 created 1972' WHERE ec = 'EC 3.2.1.78';
UPDATE enzyme SET history = 'EC 3.2.1.79 created 1972, deleted 1976' WHERE ec = 'EC 3.2.1.79';
UPDATE enzyme SET sys_name = 'Beta-D-fructan fructohydrolase' WHERE ec = 'EC 3.2.1.80';
UPDATE enzyme SET history = 'EC 3.2.1.80 created 1972' WHERE ec = 'EC 3.2.1.80';
UPDATE enzyme SET sys_name = 'Agarose 4-glycanohydrolase' WHERE ec = 'EC 3.2.1.81';
UPDATE enzyme SET history = 'EC 3.2.1.81 created 1972, modified 2006' WHERE ec = 'EC 3.2.1.81';
UPDATE enzyme SET sys_name = 'Poly((1-&gt;4)-alpha-D-galactosiduronate) digalacturonohydrolase' WHERE ec = 'EC 3.2.1.82';
UPDATE enzyme SET history = 'EC 3.2.1.82 created 1972' WHERE ec = 'EC 3.2.1.82';
UPDATE enzyme SET sys_name = 'Kappa-carrageenan 4-beta-D-glycanohydrolase (configuration-retaining)' WHERE ec = 'EC 3.2.1.83';
UPDATE enzyme SET history = 'EC 3.2.1.83 created 1972, modified 2006' WHERE ec = 'EC 3.2.1.83';
UPDATE enzyme SET sys_name = '3-alpha-D-glucan 3-glucohydrolase' WHERE ec = 'EC 3.2.1.84';
UPDATE enzyme SET history = 'EC 3.2.1.84 created 1972' WHERE ec = 'EC 3.2.1.84';
UPDATE enzyme SET sys_name = '6-phospho-beta-D-galactoside 6-phosphogalactohydrolase' WHERE ec = 'EC 3.2.1.85';
UPDATE enzyme SET history = 'EC 3.2.1.85 created 1976' WHERE ec = 'EC 3.2.1.85';
UPDATE enzyme SET sys_name = '6-phospho-beta-D-glucosyl-(1-&gt;4)-D-glucose glucohydrolase' WHERE ec = 'EC 3.2.1.86';
UPDATE enzyme SET history = 'EC 3.2.1.86 created 1976' WHERE ec = 'EC 3.2.1.86';
UPDATE enzyme SET sys_name = 'Aerobacter-capsular-polysaccharide galactohydrolase' WHERE ec = 'EC 3.2.1.87';
UPDATE enzyme SET history = 'EC 3.2.1.87 created 1976' WHERE ec = 'EC 3.2.1.87';
UPDATE enzyme SET sys_name = 'Beta-L-arabinopyranoside non-reducing end beta-L-arabinopyranosidase' WHERE ec = 'EC 3.2.1.88';
UPDATE enzyme SET history = 'EC 3.2.1.88 created 1976, modified 2013' WHERE ec = 'EC 3.2.1.88';
UPDATE enzyme SET sys_name = 'Arabinogalactan 4-beta-D-galactanohydrolase' WHERE ec = 'EC 3.2.1.89';
UPDATE enzyme SET history = 'EC 3.2.1.89 created 1976, modified 2012' WHERE ec = 'EC 3.2.1.89';
UPDATE enzyme SET history = 'EC 3.2.1.90 created 1976, deleted 2001' WHERE ec = 'EC 3.2.1.90';
UPDATE enzyme SET sys_name = '4-beta-D-glucan cellobiohydrolase (non-reducing end)' WHERE ec = 'EC 3.2.1.91';
UPDATE enzyme SET history = 'EC 3.2.1.91 created 1976, modified 2011' WHERE ec = 'EC 3.2.1.91';
UPDATE enzyme SET sys_name = 'Peptidoglycan beta-N-acetylmuramoylexohydrolase' WHERE ec = 'EC 3.2.1.92';
UPDATE enzyme SET history = 'EC 3.2.1.92 created 1976' WHERE ec = 'EC 3.2.1.92';
UPDATE enzyme SET sys_name = 'Alpha,alpha-trehalose-6-phosphate phosphoglucohydrolase' WHERE ec = 'EC 3.2.1.93';
UPDATE enzyme SET history = 'EC 3.2.1.93 created 1976' WHERE ec = 'EC 3.2.1.93';
UPDATE enzyme SET sys_name = '6-alpha-D-glucan isomaltohydrolase' WHERE ec = 'EC 3.2.1.94';
UPDATE enzyme SET history = 'EC 3.2.1.94 created 1976' WHERE ec = 'EC 3.2.1.94';
UPDATE enzyme SET sys_name = '1,6-alpha-D-glucan isomaltotriohydrolase' WHERE ec = 'EC 3.2.1.95';
UPDATE enzyme SET history = 'EC 3.2.1.95 created 1978' WHERE ec = 'EC 3.2.1.95';
UPDATE enzyme SET sys_name = 'Glycopeptide-D-mannosyl-N(4)-(N-acetyl-D-glucosaminyl)(2)-asparagine 1,4-N-acetyl-beta-glucosaminohydrolase' WHERE ec = 'EC 3.2.1.96';
UPDATE enzyme SET history = 'EC 3.2.1.96 created 1978' WHERE ec = 'EC 3.2.1.96';
UPDATE enzyme SET sys_name = 'Glycopeptide-D-galactosyl-N-acetyl-alpha-D-galactosamine D-galactosyl-N-acetyl-galactosaminohydrolase' WHERE ec = 'EC 3.2.1.97';
UPDATE enzyme SET history = 'EC 3.2.1.97 created 1978 (EC 3.2.1.110 created 1984, incorporated 2008), modified 2008, modified 2011' WHERE ec = 'EC 3.2.1.97';
UPDATE enzyme SET sys_name = '4-alpha-D-glucan maltohexaohydrolase' WHERE ec = 'EC 3.2.1.98';
UPDATE enzyme SET history = 'EC 3.2.1.98 created 1978' WHERE ec = 'EC 3.2.1.98';
UPDATE enzyme SET sys_name = '5-alpha-L-arabinan 5-alpha-L-arabinanohydrolase' WHERE ec = 'EC 3.2.1.99';
UPDATE enzyme SET history = 'EC 3.2.1.99 created 1981, modified 2011' WHERE ec = 'EC 3.2.1.99';
UPDATE enzyme SET sys_name = '4-beta-D-mannan mannobiohydrolase' WHERE ec = 'EC 3.2.1.100';
UPDATE enzyme SET history = 'EC 3.2.1.100 created 1983' WHERE ec = 'EC 3.2.1.100';
UPDATE enzyme SET sys_name = '6-alpha-D-mannan mannanohydrolase' WHERE ec = 'EC 3.2.1.101';
UPDATE enzyme SET history = 'EC 3.2.1.101 created 1984, modified 2001' WHERE ec = 'EC 3.2.1.101';
UPDATE enzyme SET sys_name = 'Blood-group-substance 4-beta-D-galactanohydrolase' WHERE ec = 'EC 3.2.1.102';
UPDATE enzyme SET history = 'EC 3.2.1.102 created 1984' WHERE ec = 'EC 3.2.1.102';
UPDATE enzyme SET sys_name = 'Keratan-sulfate 4-beta-D-galactanohydrolase' WHERE ec = 'EC 3.2.1.103';
UPDATE enzyme SET history = 'EC 3.2.1.103 created 1984' WHERE ec = 'EC 3.2.1.103';
UPDATE enzyme SET sys_name = 'Cholesteryl-beta-D-glucoside glucohydrolase' WHERE ec = 'EC 3.2.1.104';
UPDATE enzyme SET history = 'EC 3.2.1.104 created 1984' WHERE ec = 'EC 3.2.1.104';
UPDATE enzyme SET sys_name = 'Strictosidine beta-D-glucohydrolase' WHERE ec = 'EC 3.2.1.105';
UPDATE enzyme SET history = 'EC 3.2.1.105 created 1984' WHERE ec = 'EC 3.2.1.105';
UPDATE enzyme SET sys_name = 'Mannosyl-oligosaccharide glucohydrolase' WHERE ec = 'EC 3.2.1.106';
UPDATE enzyme SET history = 'EC 3.2.1.106 created 1984' WHERE ec = 'EC 3.2.1.106';
UPDATE enzyme SET sys_name = 'Protein-alpha-D-glucosyl-(1-&gt;2)-beta-D-galactosyl-L-hydroxylysine glucohydrolase' WHERE ec = 'EC 3.2.1.107';
UPDATE enzyme SET history = 'EC 3.2.1.107 created 1984' WHERE ec = 'EC 3.2.1.107';
UPDATE enzyme SET sys_name = 'Lactose galactohydrolase' WHERE ec = 'EC 3.2.1.108';
UPDATE enzyme SET history = 'EC 3.2.1.108 created 1984' WHERE ec = 'EC 3.2.1.108';
UPDATE enzyme SET sys_name = 'Galactosaminoglycan glycanohydrolase' WHERE ec = 'EC 3.2.1.109';
UPDATE enzyme SET history = 'EC 3.2.1.109 created 1984' WHERE ec = 'EC 3.2.1.109';
UPDATE enzyme SET history = 'EC 3.2.1.110 created 1984, deleted 2008' WHERE ec = 'EC 3.2.1.110';
UPDATE enzyme SET sys_name = '3-alpha-L-fucosyl-N-acetylglucosaminyl-glycoprotein fucohydrolase' WHERE ec = 'EC 3.2.1.111';
UPDATE enzyme SET history = 'EC 3.2.1.111 created 1986' WHERE ec = 'EC 3.2.1.111';
UPDATE enzyme SET sys_name = '2-deoxy-alpha-D-glucoside deoxyglucohydrolase' WHERE ec = 'EC 3.2.1.112';
UPDATE enzyme SET history = 'EC 3.2.1.112 created 1986' WHERE ec = 'EC 3.2.1.112';
UPDATE enzyme SET sys_name = '2-alpha-mannosyl-oligosaccharide alpha-D-mannohydrolase' WHERE ec = 'EC 3.2.1.113';
UPDATE enzyme SET history = 'EC 3.2.1.113 created 1986' WHERE ec = 'EC 3.2.1.113';
UPDATE enzyme SET sys_name = '(1-&gt;3)-(1-&gt;6)-mannosyl-oligosaccharide alpha-D-mannohydrolase' WHERE ec = 'EC 3.2.1.114';
UPDATE enzyme SET history = 'EC 3.2.1.114 created 1986' WHERE ec = 'EC 3.2.1.114';
UPDATE enzyme SET sys_name = '2-alpha-D-glucosyl-branched-dextran 2-glucohydrolase' WHERE ec = 'EC 3.2.1.115';
UPDATE enzyme SET history = 'EC 3.2.1.115 created 1989' WHERE ec = 'EC 3.2.1.115';
UPDATE enzyme SET sys_name = '4-alpha-D-glucan maltotriohydrolase' WHERE ec = 'EC 3.2.1.116';
UPDATE enzyme SET history = 'EC 3.2.1.116 created 1989' WHERE ec = 'EC 3.2.1.116';
UPDATE enzyme SET sys_name = 'Amygdalin beta-D-glucohydrolase' WHERE ec = 'EC 3.2.1.117';
UPDATE enzyme SET history = 'EC 3.2.1.117 created 1989' WHERE ec = 'EC 3.2.1.117';
UPDATE enzyme SET sys_name = 'Prunasin beta-D-glucohydrolase' WHERE ec = 'EC 3.2.1.118';
UPDATE enzyme SET history = 'EC 3.2.1.118 created 1989' WHERE ec = 'EC 3.2.1.118';
UPDATE enzyme SET sys_name = '(R)-vicianin beta-D-glucohydrolase' WHERE ec = 'EC 3.2.1.119';
UPDATE enzyme SET history = 'EC 3.2.1.119 created 1989' WHERE ec = 'EC 3.2.1.119';
UPDATE enzyme SET sys_name = 'Oligoxyloglucan xyloglucohydrolase' WHERE ec = 'EC 3.2.1.120';
UPDATE enzyme SET history = 'EC 3.2.1.120 created 1989' WHERE ec = 'EC 3.2.1.120';
UPDATE enzyme SET sys_name = 'Poly(mannuronide) mannuronohydrolase' WHERE ec = 'EC 3.2.1.121';
UPDATE enzyme SET history = 'EC 3.2.1.121 created 1989' WHERE ec = 'EC 3.2.1.121';
UPDATE enzyme SET sys_name = 'Maltose-6-phosphate 6-phosphoglucohydrolase' WHERE ec = 'EC 3.2.1.122';
UPDATE enzyme SET history = 'EC 3.2.1.122 created 1989, modified 1999' WHERE ec = 'EC 3.2.1.122';
UPDATE enzyme SET sys_name = 'Oligoglycosylglucosyl-(1&lt;-&gt;1)-ceramide glycohydrolase' WHERE ec = 'EC 3.2.1.123';
UPDATE enzyme SET history = 'EC 3.2.1.123 created 1989' WHERE ec = 'EC 3.2.1.123';
UPDATE enzyme SET sys_name = 'Capsular-polysaccharide 3-deoxy-D-manno-2-octulosonohydrolase' WHERE ec = 'EC 3.2.1.124';
UPDATE enzyme SET history = 'EC 3.2.1.124 created 1989' WHERE ec = 'EC 3.2.1.124';
UPDATE enzyme SET sys_name = 'Raucaffricine beta-D-glucohydrolase' WHERE ec = 'EC 3.2.1.125';
UPDATE enzyme SET history = 'EC 3.2.1.125 created 1989' WHERE ec = 'EC 3.2.1.125';
UPDATE enzyme SET sys_name = 'Coniferin beta-D-glucosidase' WHERE ec = 'EC 3.2.1.126';
UPDATE enzyme SET history = 'EC 3.2.1.126 created 1989' WHERE ec = 'EC 3.2.1.126';
UPDATE enzyme SET sys_name = '6-L-fucosyl-N-acetyl-D-glucosaminylglycopeptide fucohydrolase' WHERE ec = 'EC 3.2.1.127';
UPDATE enzyme SET history = 'EC 3.2.1.127 created 1989' WHERE ec = 'EC 3.2.1.127';
UPDATE enzyme SET sys_name = 'Glycyrrhizinate glucuronosylhydrolase' WHERE ec = 'EC 3.2.1.128';
UPDATE enzyme SET history = 'EC 3.2.1.128 created 1989' WHERE ec = 'EC 3.2.1.128';
UPDATE enzyme SET sys_name = 'Polysialoside (2-&gt;8)-alpha-sialosylhydrolase' WHERE ec = 'EC 3.2.1.129';
UPDATE enzyme SET history = 'EC 3.2.1.129 created 1990, modified 1999' WHERE ec = 'EC 3.2.1.129';
UPDATE enzyme SET sys_name = 'Glycoprotein glucosylmannohydrolase' WHERE ec = 'EC 3.2.1.130';
UPDATE enzyme SET history = 'EC 3.2.1.130 created 1990' WHERE ec = 'EC 3.2.1.130';
UPDATE enzyme SET sys_name = 'Xylan alpha-D-2-(4-O-methyl)glucuronohydrolase' WHERE ec = 'EC 3.2.1.131';
UPDATE enzyme SET history = 'EC 3.2.1.131 created 1990' WHERE ec = 'EC 3.2.1.131';
UPDATE enzyme SET sys_name = 'Chitosan N-acetylglucosaminohydrolase' WHERE ec = 'EC 3.2.1.132';
UPDATE enzyme SET history = 'EC 3.2.1.132 created 1990, modified 2004' WHERE ec = 'EC 3.2.1.132';
UPDATE enzyme SET sys_name = '4-alpha-D-glucan alpha-maltohydrolase' WHERE ec = 'EC 3.2.1.133';
UPDATE enzyme SET history = 'EC 3.2.1.133 created 1992, modified 1999' WHERE ec = 'EC 3.2.1.133';
UPDATE enzyme SET sys_name = 'Bis-D-fructose 2,1:2,1-dianhydride fructohydrolase' WHERE ec = 'EC 3.2.1.134';
UPDATE enzyme SET history = 'EC 3.2.1.134 created 1992' WHERE ec = 'EC 3.2.1.134';
UPDATE enzyme SET sys_name = 'Pullulan 4-D-glucanohydrolase (panose-forming)' WHERE ec = 'EC 3.2.1.135';
UPDATE enzyme SET history = 'EC 3.2.1.135 created 1992' WHERE ec = 'EC 3.2.1.135';
UPDATE enzyme SET sys_name = 'Glucuronoarabinoxylan 4-beta-D-xylanohydrolase' WHERE ec = 'EC 3.2.1.136';
UPDATE enzyme SET history = 'EC 3.2.1.136 created 1992' WHERE ec = 'EC 3.2.1.136';
UPDATE enzyme SET sys_name = '(1-&gt;2)-(1-&gt;6)-alpha-D-mannan D-mannohydrolase' WHERE ec = 'EC 3.2.1.137';
UPDATE enzyme SET history = 'EC 3.2.1.137 created 1992' WHERE ec = 'EC 3.2.1.137';
UPDATE enzyme SET history = 'EC 3.2.1.138 created 1992, deleted 2003' WHERE ec = 'EC 3.2.1.138';
UPDATE enzyme SET sys_name = 'Alpha-D-glucosiduronate glucuronohydrolase' WHERE ec = 'EC 3.2.1.139';
UPDATE enzyme SET history = 'EC 3.2.1.139 created 1999' WHERE ec = 'EC 3.2.1.139';
UPDATE enzyme SET sys_name = 'Oligosaccharide lacto-N-biosylhydrolase' WHERE ec = 'EC 3.2.1.140';
UPDATE enzyme SET history = 'EC 3.2.1.140 created 1999' WHERE ec = 'EC 3.2.1.140';
UPDATE enzyme SET sys_name = '4-alpha-D-((1-&gt;4)-alpha-D-glucano)trehalose glucanohydrolase (trehalose-producing)' WHERE ec = 'EC 3.2.1.141';
UPDATE enzyme SET history = 'EC 3.2.1.141 created 1999' WHERE ec = 'EC 3.2.1.141';
UPDATE enzyme SET sys_name = 'Dextrin alpha-6-glucanohydrolase' WHERE ec = 'EC 3.2.1.142';
UPDATE enzyme SET history = 'EC 3.2.1.142 created 2000' WHERE ec = 'EC 3.2.1.142';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.2.1.143';
UPDATE enzyme SET history = 'EC 3.2.1.143 created 2000' WHERE ec = 'EC 3.2.1.143';
UPDATE enzyme SET sys_name = '3-deoxyoctulosonyl-lipopolysaccharide hydrolase' WHERE ec = 'EC 3.2.1.144';
UPDATE enzyme SET history = 'EC 3.2.1.144 created 2000' WHERE ec = 'EC 3.2.1.144';
UPDATE enzyme SET sys_name = 'Galactan 3-beta-D-galactosidase' WHERE ec = 'EC 3.2.1.145';
UPDATE enzyme SET history = 'EC 3.2.1.145 created 2001' WHERE ec = 'EC 3.2.1.145';
UPDATE enzyme SET sys_name = 'Beta-D-galactofuranoside hydrolase' WHERE ec = 'EC 3.2.1.146';
UPDATE enzyme SET history = 'EC 3.2.1.146 created 2001' WHERE ec = 'EC 3.2.1.146';
UPDATE enzyme SET sys_name = 'Thioglucoside glucohydrolase' WHERE ec = 'EC 3.2.1.147';
UPDATE enzyme SET history = 'EC 3.2.1.147 created 1972 as EC 3.2.3.1, transferred 2001 to EC 3.2.1.147' WHERE ec = 'EC 3.2.1.147';
UPDATE enzyme SET history = 'EC 3.2.1.148 created 1972 as EC 3.3.1.3, transferred 2001 to EC 3.2.1.148, deleted 2004' WHERE ec = 'EC 3.2.1.148';
UPDATE enzyme SET sys_name = '6-O-(beta-D-xylopyranosyl)-beta-D-glucopyranoside 6-O-(beta-D-xylosyl)-beta-D-glucohydrolase' WHERE ec = 'EC 3.2.1.149';
UPDATE enzyme SET history = 'EC 3.2.1.149 created 2001' WHERE ec = 'EC 3.2.1.149';
UPDATE enzyme SET sys_name = 'Oligoxyloglucan reducing-end cellobiohydrolase' WHERE ec = 'EC 3.2.1.150';
UPDATE enzyme SET history = 'EC 3.2.1.150 created 2003' WHERE ec = 'EC 3.2.1.150';
UPDATE enzyme SET sys_name = '((1-&gt;6)-alpha-D-xylo)-(1-&gt;4)-beta-D-glucan glucanohydrolase' WHERE ec = 'EC 3.2.1.151';
UPDATE enzyme SET history = 'EC 3.2.1.151 created 2003' WHERE ec = 'EC 3.2.1.151';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.2.1.152';
UPDATE enzyme SET history = 'EC 3.2.1.152 created 2005' WHERE ec = 'EC 3.2.1.152';
UPDATE enzyme SET sys_name = 'Beta-(2-&gt;1)-D-fructan fructohydrolase' WHERE ec = 'EC 3.2.1.153';
UPDATE enzyme SET history = 'EC 3.2.1.153 created 2005' WHERE ec = 'EC 3.2.1.153';
UPDATE enzyme SET sys_name = '(2-&gt;6)-beta-D-fructan fructohydrolase' WHERE ec = 'EC 3.2.1.154';
UPDATE enzyme SET history = 'EC 3.2.1.154 created 2005' WHERE ec = 'EC 3.2.1.154';
UPDATE enzyme SET sys_name = '((1-&gt;6)-alpha-D-xylo)-(1-&gt;4)-beta-D-glucan exo-glucohydrolase' WHERE ec = 'EC 3.2.1.155';
UPDATE enzyme SET history = 'EC 3.2.1.155 created 2005, withdrawn at public-review stage, modified and reinstated 2006, modified 2011' WHERE ec = 'EC 3.2.1.155';
UPDATE enzyme SET sys_name = 'Beta-D-xylopyranosyl-(1-&gt;4)-beta-D-xylopyranose reducing-end xylanase' WHERE ec = 'EC 3.2.1.156';
UPDATE enzyme SET history = 'EC 3.2.1.156 created 2005' WHERE ec = 'EC 3.2.1.156';
UPDATE enzyme SET sys_name = 'Iota-carrageenan 4-beta-D-glycanohydrolase (configuration-inverting)' WHERE ec = 'EC 3.2.1.157';
UPDATE enzyme SET history = 'EC 3.2.1.157 created 2006' WHERE ec = 'EC 3.2.1.157';
UPDATE enzyme SET sys_name = 'Agarose 3-glycanohydrolase' WHERE ec = 'EC 3.2.1.158';
UPDATE enzyme SET history = 'EC 3.2.1.158 created 2006' WHERE ec = 'EC 3.2.1.158';
UPDATE enzyme SET sys_name = 'Alpha-neoagaro-oligosaccharide 3-glycohydrolase' WHERE ec = 'EC 3.2.1.159';
UPDATE enzyme SET history = 'EC 3.2.1.159 created 2006' WHERE ec = 'EC 3.2.1.159';
UPDATE enzyme SET history = 'EC 3.2.1.160 created 2006, deleted 2006' WHERE ec = 'EC 3.2.1.160';
UPDATE enzyme SET sys_name = '7-(beta-D-apiofuranosyl-(1-&gt;6)-beta-D-glucopyranosyloxy)isoflavonoid beta-D-apiofuranosyl-(1-&gt;6)-D-glucohydrolase' WHERE ec = 'EC 3.2.1.161';
UPDATE enzyme SET history = 'EC 3.2.1.161 created 2006' WHERE ec = 'EC 3.2.1.161';
UPDATE enzyme SET sys_name = 'Endo-(1-&gt;4)-beta-carrageenose 2,6,2-trisulfate-hydrolase' WHERE ec = 'EC 3.2.1.162';
UPDATE enzyme SET history = 'EC 3.2.1.162 created 2007' WHERE ec = 'EC 3.2.1.162';
UPDATE enzyme SET sys_name = '(1-&gt;6)-alpha-mannosyl alpha-D-mannohydrolase' WHERE ec = 'EC 3.2.1.163';
UPDATE enzyme SET history = 'EC 3.2.1.163 created 2007' WHERE ec = 'EC 3.2.1.163';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.2.1.164';
UPDATE enzyme SET history = 'EC 3.2.1.164 created 2007' WHERE ec = 'EC 3.2.1.164';
UPDATE enzyme SET sys_name = 'Chitosan exo-(1-&gt;4)-beta-D-glucosaminidase' WHERE ec = 'EC 3.2.1.165';
UPDATE enzyme SET history = 'EC 3.2.1.165 created 2008' WHERE ec = 'EC 3.2.1.165';
UPDATE enzyme SET sys_name = 'Heparan sulfate N-sulfo-D-glucosamine endoglucanase' WHERE ec = 'EC 3.2.1.166';
UPDATE enzyme SET history = 'EC 3.2.1.166 created 2010' WHERE ec = 'EC 3.2.1.166';
UPDATE enzyme SET sys_name = '5,6,7-trihydroxyflavone-7-O-beta-D-glucupyranosiduronate glucuronosylhydrolase' WHERE ec = 'EC 3.2.1.167';
UPDATE enzyme SET history = 'EC 3.2.1.167 created 2010' WHERE ec = 'EC 3.2.1.167';
UPDATE enzyme SET sys_name = 'Hesperetin 7-(6-O-alpha-L-rhamnopyranosyl-beta-D-glucopyranoside) 6-O-alpha-rhamnopyranosyl-beta-glucohydrolase' WHERE ec = 'EC 3.2.1.168';
UPDATE enzyme SET history = 'EC 3.2.1.168 created 2010' WHERE ec = 'EC 3.2.1.168';
UPDATE enzyme SET sys_name = '[Protein]-3-O-(N-acetyl-D-glucosaminyl)-L-serine/threonine N-acetylglucosaminyl hydrolase' WHERE ec = 'EC 3.2.1.169';
UPDATE enzyme SET history = 'EC3.2.1.169 created 2011' WHERE ec = 'EC 3.2.1.169';
UPDATE enzyme SET sys_name = '2-O-(6-phospho-alpha-D-mannosyl)-D-glycerate acylhydrolase' WHERE ec = 'EC 3.2.1.170';
UPDATE enzyme SET history = 'EC 3.2.1.170 created 2011' WHERE ec = 'EC 3.2.1.170';
UPDATE enzyme SET sys_name = 'Rhamnogalacturonan alpha-D-GalA-(1-&gt;2)-alpha-L-Rha hydrolase' WHERE ec = 'EC 3.2.1.171';
UPDATE enzyme SET history = 'EC 3.2.1.171 created 2011' WHERE ec = 'EC 3.2.1.171';
UPDATE enzyme SET sys_name = '2-O-(4-deoxy-beta-L-threo-hex-4-enopyranuronosyl)-alpha-L-rhamnopyranose hydrolase' WHERE ec = 'EC 3.2.1.172';
UPDATE enzyme SET history = 'EC 3.2.1.172 created 2011, modified 2012' WHERE ec = 'EC 3.2.1.172';
UPDATE enzyme SET sys_name = 'Rhamnogalacturonan oligosaccharide alpha-D-GalA-(1-&gt;2)-alpha-L-Rha galacturonohydrolase' WHERE ec = 'EC 3.2.1.173';
UPDATE enzyme SET history = 'EC 3.2.1.173 created 2011' WHERE ec = 'EC 3.2.1.173';
UPDATE enzyme SET sys_name = 'Rhamnogalacturonan oligosaccharide alpha-L-Rha-(1-&gt;4)-alpha-D-GalA rhamnohydrolase' WHERE ec = 'EC 3.2.1.174';
UPDATE enzyme SET history = 'EC 3.2.1.174 created 2011' WHERE ec = 'EC 3.2.1.174';
UPDATE enzyme SET sys_name = 'Beta-D-glucopyranosyl abscisate glucohydrolase' WHERE ec = 'EC 3.2.1.175';
UPDATE enzyme SET history = 'EC 3.2.1.175 created 2011' WHERE ec = 'EC 3.2.1.175';
UPDATE enzyme SET sys_name = '4-beta-D-glucan cellobiohydrolase (reducing end)' WHERE ec = 'EC 3.2.1.176';
UPDATE enzyme SET history = 'EC 3.2.1.176 created 2011' WHERE ec = 'EC 3.2.1.176';
UPDATE enzyme SET sys_name = 'Alpha-D-xyloside xylohydrolase' WHERE ec = 'EC 3.2.1.177';
UPDATE enzyme SET history = 'EC 3.2.1.177 created 2011' WHERE ec = 'EC 3.2.1.177';
UPDATE enzyme SET sys_name = 'Porphyran beta-D-galactopyranose-(1-&gt;4)-alpha-L-galactopyranose-6-sulfate 4-glycanohydrolase' WHERE ec = 'EC 3.2.1.178';
UPDATE enzyme SET history = 'EC 3.2.1.178 created 2011' WHERE ec = 'EC 3.2.1.178';
UPDATE enzyme SET sys_name = 'Beta-D-4-deoxy-Delta(4)-GlcAp-(1-&gt;4)-beta-D-Glcp-(1-&gt;4)-alpha-L-Rhap-(1-&gt;3)-beta-D-Glcp beta-D-4-deoxy-Delta(4)-GlcAp hydrolase' WHERE ec = 'EC 3.2.1.179';
UPDATE enzyme SET history = 'EC 3.2.1.179 created 2011' WHERE ec = 'EC 3.2.1.179';
UPDATE enzyme SET sys_name = 'Beta-D-4-deoxy-Delta(4)-GlcAp-(1-&gt;3)-beta-D-GalNAc6S hydrolase' WHERE ec = 'EC 3.2.1.180';
UPDATE enzyme SET history = 'EC 3.2.1.180 created 2011' WHERE ec = 'EC 3.2.1.180';
UPDATE enzyme SET sys_name = 'Arabinogalactan 3-beta-D-galactanohydrolase' WHERE ec = 'EC 3.2.1.181';
UPDATE enzyme SET history = 'EC 3.2.1.181 created 2012' WHERE ec = 'EC 3.2.1.181';
UPDATE enzyme SET sys_name = '(2R)-4-hydroxy-7-methoxy-3-oxo-3,4-dihydro-2H-1,4-benzoxazin-2-yl beta-D-glucopyranoside beta-D-glucosidase' WHERE ec = 'EC 3.2.1.182';
UPDATE enzyme SET history = 'EC 3.2.1.182 created 2012' WHERE ec = 'EC 3.2.1.182';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine hydrolase (2-epimerising)' WHERE ec = 'EC 3.2.1.183';
UPDATE enzyme SET history = 'EC 3.2.1.183 created 2012' WHERE ec = 'EC 3.2.1.183';
UPDATE enzyme SET sys_name = 'UDP-N,N-diacetylbacillosamine hydrolase (2-epimerising)' WHERE ec = 'EC 3.2.1.184';
UPDATE enzyme SET history = 'EC 3.2.1.184 created 2012' WHERE ec = 'EC 3.2.1.184';
UPDATE enzyme SET sys_name = 'Beta-L-arabinofuranoside non-reducing end beta-L-arabinofuranosidase' WHERE ec = 'EC 3.2.1.185';
UPDATE enzyme SET history = 'EC 3.2.1.185 created 2013' WHERE ec = 'EC 3.2.1.185';
UPDATE enzyme SET sys_name = 'Protodioscin glucohydrolase' WHERE ec = 'EC 3.2.1.186';
UPDATE enzyme SET history = 'EC 3.2.1.186 created 2013' WHERE ec = 'EC 3.2.1.186';
UPDATE enzyme SET sys_name = '4-O-(beta-L-arabinofuranosyl-(1-&gt;2)-beta-L-arabinofuranosyl-(1-&gt;2)-beta-L-arabinofuranosyl)-(2S,4S)-4-hydroxyproline beta-L-arabinofuranosyl-(1-&gt;2)-beta-L-arabinofuranose hydrolase' WHERE ec = 'EC 3.2.1.187';
UPDATE enzyme SET history = 'EC 3.2.1.187 created 2013' WHERE ec = 'EC 3.2.1.187';
UPDATE enzyme SET sys_name = 'Avenacoside B 26-beta-D-glucohydrolase' WHERE ec = 'EC 3.2.1.188';
UPDATE enzyme SET history = 'EC 3.2.1.188 created 2013' WHERE ec = 'EC 3.2.1.188';
UPDATE enzyme SET sys_name = '3-O-[alpha-L-Rha-(1-&gt;4)-(alpha-L-Rha-(1-&gt;2))-beta-D-Glc]diosgenin hydrolase (diosgenin-forming)' WHERE ec = 'EC 3.2.1.189';
UPDATE enzyme SET history = 'EC 3.2.1.189 created 2013' WHERE ec = 'EC 3.2.1.189';
UPDATE enzyme SET sys_name = '3-O-[alpha-L-Rha-(1-&gt;4)-(alpha-L-Rha-(1-&gt;2))-beta-D-Glc]diosgenin (3-O-beta-D-Glc-diosgenin-forming)' WHERE ec = 'EC 3.2.1.190';
UPDATE enzyme SET history = 'EC 3.2.1.190 created 2013' WHERE ec = 'EC 3.2.1.190';
UPDATE enzyme SET sys_name = 'Protopanaxadiol-type ginsenoside 3-beta-D-hydrolase' WHERE ec = 'EC 3.2.1.191';
UPDATE enzyme SET history = 'EC 3.2.1.191 created 2014' WHERE ec = 'EC 3.2.1.191';
UPDATE enzyme SET sys_name = 'Ginsenoside Rb1 glucohydrolase' WHERE ec = 'EC 3.2.1.192';
UPDATE enzyme SET history = 'EC 3.2.1.192 created 2014' WHERE ec = 'EC 3.2.1.192';
UPDATE enzyme SET sys_name = 'Ginsenoside glucohydrolase' WHERE ec = 'EC 3.2.1.193';
UPDATE enzyme SET history = 'EC 3.2.1.193 created 2014' WHERE ec = 'EC 3.2.1.193';
UPDATE enzyme SET sys_name = 'Protopanaxatriol-type ginsenoside 6-beta-D-glucohydrolase' WHERE ec = 'EC 3.2.1.194';
UPDATE enzyme SET history = 'EC 3.2.1.194 created 2014' WHERE ec = 'EC 3.2.1.194';
UPDATE enzyme SET sys_name = 'Protopanaxadiol-type ginsenoside 20-beta-D-glucohydrolase' WHERE ec = 'EC 3.2.1.195';
UPDATE enzyme SET history = 'EC 3.2.1.195 created 2014' WHERE ec = 'EC 3.2.1.195';
UPDATE enzyme SET sys_name = '(1-&gt;3)-alpha-D-galactose-branched-(blood group B structures) 3-galactohydrolase' WHERE ec = 'EC 3.2.1.n1';
UPDATE enzyme SET history = 'EC 3.2.1.n1 created 2010' WHERE ec = 'EC 3.2.1.n1';
UPDATE enzyme SET sys_name = '(1-&gt;3)-alpha-D-galactose-linear-(blood group B structures) 3-galactohydrolase' WHERE ec = 'EC 3.2.1.n2';
UPDATE enzyme SET history = 'EC 3.2.1.n2 created 2010' WHERE ec = 'EC 3.2.1.n2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.2.1.n3';
UPDATE enzyme SET history = 'EC 3.2.1.n3 created 2010' WHERE ec = 'EC 3.2.1.n3';
UPDATE enzyme SET sys_name = 'Purine-nucleoside ribohydrolase' WHERE ec = 'EC 3.2.2.1';
UPDATE enzyme SET history = 'EC 3.2.2.1 created 1961, modified 2006, modified 2011' WHERE ec = 'EC 3.2.2.1';
UPDATE enzyme SET sys_name = 'Inosine ribohydrolase' WHERE ec = 'EC 3.2.2.2';
UPDATE enzyme SET history = 'EC 3.2.2.2 created 1961' WHERE ec = 'EC 3.2.2.2';
UPDATE enzyme SET sys_name = 'Uridine ribohydrolase' WHERE ec = 'EC 3.2.2.3';
UPDATE enzyme SET history = 'EC 3.2.2.3 created 1961' WHERE ec = 'EC 3.2.2.3';
UPDATE enzyme SET sys_name = 'AMP phosphoribohydrolase' WHERE ec = 'EC 3.2.2.4';
UPDATE enzyme SET history = 'EC 3.2.2.4 created 1961' WHERE ec = 'EC 3.2.2.4';
UPDATE enzyme SET sys_name = 'NAD(+) glycohydrolase' WHERE ec = 'EC 3.2.2.5';
UPDATE enzyme SET history = 'EC 3.2.2.5 created 1961, modified 2004, modified 2013' WHERE ec = 'EC 3.2.2.5';
UPDATE enzyme SET sys_name = 'NAD(+) glycohydrolase (cyclic ADP-ribose-forming)' WHERE ec = 'EC 3.2.2.6';
UPDATE enzyme SET history = 'EC 3.2.2.6 created 1961, modified 2004, modified 2014' WHERE ec = 'EC 3.2.2.6';
UPDATE enzyme SET sys_name = 'Adenosine ribohydrolase' WHERE ec = 'EC 3.2.2.7';
UPDATE enzyme SET history = 'EC 3.2.2.7 created 1972' WHERE ec = 'EC 3.2.2.7';
UPDATE enzyme SET sys_name = 'Pyrimidine-nucleoside ribohydrolase' WHERE ec = 'EC 3.2.2.8';
UPDATE enzyme SET history = 'EC 3.2.2.8 created 1972' WHERE ec = 'EC 3.2.2.8';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-homocysteine homocysteinylribohydrolase' WHERE ec = 'EC 3.2.2.9';
UPDATE enzyme SET history = 'EC 3.2.2.9 created 1972, modified 2004' WHERE ec = 'EC 3.2.2.9';
UPDATE enzyme SET sys_name = 'Pyrimidine-5-nucleotide phosphoribo(deoxyribo)hydrolase' WHERE ec = 'EC 3.2.2.10';
UPDATE enzyme SET history = 'EC 3.2.2.10 created 1972' WHERE ec = 'EC 3.2.2.10';
UPDATE enzyme SET sys_name = '1-beta-aspartyl-N-acetyl-D-glucosaminylamine L-asparaginohydrolase' WHERE ec = 'EC 3.2.2.11';
UPDATE enzyme SET history = 'EC 3.2.2.11 created 1972' WHERE ec = 'EC 3.2.2.11';
UPDATE enzyme SET sys_name = '5-inosinate phosphoribohydrolase' WHERE ec = 'EC 3.2.2.12';
UPDATE enzyme SET history = 'EC 3.2.2.12 created 1972' WHERE ec = 'EC 3.2.2.12';
UPDATE enzyme SET sys_name = '1-methyladenosine ribohydrolase' WHERE ec = 'EC 3.2.2.13';
UPDATE enzyme SET history = 'EC 3.2.2.13 created 1976' WHERE ec = 'EC 3.2.2.13';
UPDATE enzyme SET sys_name = 'Nicotinamide-nucleotide phosphoribohydrolase' WHERE ec = 'EC 3.2.2.14';
UPDATE enzyme SET history = 'EC 3.2.2.14 created 1976' WHERE ec = 'EC 3.2.2.14';
UPDATE enzyme SET sys_name = 'DNA-deoxyinosine deoxyribohydrolase' WHERE ec = 'EC 3.2.2.15';
UPDATE enzyme SET history = 'EC 3.2.2.15 created 1980, modified 1982, modified 2000' WHERE ec = 'EC 3.2.2.15';
UPDATE enzyme SET sys_name = 'S-methyl-5-thioadenosine adeninehyrolase' WHERE ec = 'EC 3.2.2.16';
UPDATE enzyme SET history = 'EC 3.2.2.16 created 1983, modified 2003' WHERE ec = 'EC 3.2.2.16';
UPDATE enzyme SET sys_name = 'Deoxy-D-ribocyclobutadipyrimidine polynucleotidodeoxyribohydrolase' WHERE ec = 'EC 3.2.2.17';
UPDATE enzyme SET history = 'EC 3.2.2.17 created 1983' WHERE ec = 'EC 3.2.2.17';
UPDATE enzyme SET history = 'EC 3.2.2.18 created 1984, deleted 1989' WHERE ec = 'EC 3.2.2.18';
UPDATE enzyme SET sys_name = 'Protein-N(omega)-(ADP-D-ribosyl)-L-arginine ADP-ribosylhydrolase' WHERE ec = 'EC 3.2.2.19';
UPDATE enzyme SET history = 'EC 3.2.2.19 created 1989, modified 2004' WHERE ec = 'EC 3.2.2.19';
UPDATE enzyme SET sys_name = 'Alkylated-DNA glycohydrolase (releasing methyladenine and methylguanine)' WHERE ec = 'EC 3.2.2.20';
UPDATE enzyme SET history = 'EC 3.2.2.20 created 1990, modified 2000' WHERE ec = 'EC 3.2.2.20';
UPDATE enzyme SET sys_name = 'Alkylated-DNA glycohydrolase (releasing methyladenine and methylguanine)' WHERE ec = 'EC 3.2.2.21';
UPDATE enzyme SET history = 'EC 3.2.2.21 created 1990, modified 2000' WHERE ec = 'EC 3.2.2.21';
UPDATE enzyme SET sys_name = 'rRNA N-glycohydrolase' WHERE ec = 'EC 3.2.2.22';
UPDATE enzyme SET history = 'EC 3.2.2.22 created 1990, modified 2000' WHERE ec = 'EC 3.2.2.22';
UPDATE enzyme SET sys_name = 'DNA glycohydrolase (2,6-diamino-4-hydroxy-5-(N-methyl)formamidopyrimide releasing)' WHERE ec = 'EC 3.2.2.23';
UPDATE enzyme SET history = 'EC 3.2.2.23 created 1990, modified 2000' WHERE ec = 'EC 3.2.2.23';
UPDATE enzyme SET sys_name = 'ADP-D-ribosyl-[dinitrogen reductase] ADP-ribosylhydrolase' WHERE ec = 'EC 3.2.2.24';
UPDATE enzyme SET history = 'EC 3.2.2.24 created 1992' WHERE ec = 'EC 3.2.2.24';
UPDATE enzyme SET sys_name = '7-methylxanthosine ribohydrolase' WHERE ec = 'EC 3.2.2.25';
UPDATE enzyme SET history = 'EC 3.2.2.25 created 2007' WHERE ec = 'EC 3.2.2.25';
UPDATE enzyme SET sys_name = 'Futalosine ribohydrolase' WHERE ec = 'EC 3.2.2.26';
UPDATE enzyme SET history = 'EC 3.2.2.26 created 2008' WHERE ec = 'EC 3.2.2.26';
UPDATE enzyme SET sys_name = 'Uracil-DNA deoxyribohydrolase (uracil-releasing)' WHERE ec = 'EC 3.2.2.27';
UPDATE enzyme SET history = 'EC 3.2.2.27 created 2009' WHERE ec = 'EC 3.2.2.27';
UPDATE enzyme SET sys_name = 'Uracil-double-stranded DNA deoxyribohydrolase (uracil-releasing)' WHERE ec = 'EC 3.2.2.28';
UPDATE enzyme SET history = 'EC 3.2.2.28 created 2009' WHERE ec = 'EC 3.2.2.28';
UPDATE enzyme SET sys_name = 'Thymine-DNA deoxyribohydrolase (thymine-releasing)' WHERE ec = 'EC 3.2.2.29';
UPDATE enzyme SET history = 'EC 3.2.2.29 created 2009' WHERE ec = 'EC 3.2.2.29';
UPDATE enzyme SET history = 'EC 3.2.3.1 created 1972, deleted 2001' WHERE ec = 'EC 3.2.3.1';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-homocysteine hydrolase' WHERE ec = 'EC 3.3.1.1';
UPDATE enzyme SET history = 'EC 3.3.1.1 created 1961, modified 2004' WHERE ec = 'EC 3.3.1.1';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine hydrolase' WHERE ec = 'EC 3.3.1.2';
UPDATE enzyme SET history = 'EC 3.3.1.2 created 1972, modified 1976' WHERE ec = 'EC 3.3.1.2';
UPDATE enzyme SET history = 'EC 3.3.1.3 created 1972, deleted 2001' WHERE ec = 'EC 3.3.1.3';
UPDATE enzyme SET sys_name = 'Isochorismate pyruvate-hydrolase' WHERE ec = 'EC 3.3.2.1';
UPDATE enzyme SET history = 'EC 3.3.2.1 created 1972' WHERE ec = 'EC 3.3.2.1';
UPDATE enzyme SET sys_name = '1-(1-alkenyl)-sn-glycero-3-phosphocholine aldehydohydrolase' WHERE ec = 'EC 3.3.2.2';
UPDATE enzyme SET history = 'EC 3.3.2.2 created 1972, modified 1976' WHERE ec = 'EC 3.3.2.2';
UPDATE enzyme SET history = 'EC 3.3.2.3 created 1978, modified 1999, deleted 2006' WHERE ec = 'EC 3.3.2.3';
UPDATE enzyme SET sys_name = 'Trans-2,3-epoxysuccinate hydrolase' WHERE ec = 'EC 3.3.2.4';
UPDATE enzyme SET history = 'EC 3.3.2.4 created 1984' WHERE ec = 'EC 3.3.2.4';
UPDATE enzyme SET sys_name = '1-(1-alkenyl)-sn-glycero-3-phosphoethanolamine aldehydohydrolase' WHERE ec = 'EC 3.3.2.5';
UPDATE enzyme SET history = 'EC 3.3.2.5 created 1984' WHERE ec = 'EC 3.3.2.5';
UPDATE enzyme SET sys_name = '(7E,9E,11Z,14Z)-(5S,6S)-5,6-epoxyicosa-7,9,11,14-tetraenoate hydrolase' WHERE ec = 'EC 3.3.2.6';
UPDATE enzyme SET history = 'EC 3.3.2.6 created 1989, modified 2006' WHERE ec = 'EC 3.3.2.6';
UPDATE enzyme SET sys_name = '(5Z,9E,14Z)-(8-xi,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate hydrolase' WHERE ec = 'EC 3.3.2.7';
UPDATE enzyme SET history = 'EC 3.3.2.7 created 1992, modified 2006' WHERE ec = 'EC 3.3.2.7';
UPDATE enzyme SET sys_name = '1,2-epoxymenth-8-ene hydrolase' WHERE ec = 'EC 3.3.2.8';
UPDATE enzyme SET history = 'EC 3.3.2.8 created 2001' WHERE ec = 'EC 3.3.2.8';
UPDATE enzyme SET sys_name = 'Cis-stilbene-oxide hydrolase' WHERE ec = 'EC 3.3.2.9';
UPDATE enzyme SET history = 'EC 3.3.2.9 created 2006 (EC 3.3.2.3 part-incorporated 2006)' WHERE ec = 'EC 3.3.2.9';
UPDATE enzyme SET sys_name = 'Epoxide hydrolase' WHERE ec = 'EC 3.3.2.10';
UPDATE enzyme SET history = 'EC 3.3.2.10 created 2006 (EC 3.3.2.3 part-incorporated 2006)' WHERE ec = 'EC 3.3.2.10';
UPDATE enzyme SET sys_name = '5,6-alpha-epoxy-5-alpha-cholestan-3-beta-ol hydrolase' WHERE ec = 'EC 3.3.2.11';
UPDATE enzyme SET history = 'EC 3.3.2.11 created 2006' WHERE ec = 'EC 3.3.2.11';
UPDATE enzyme SET sys_name = '2-oxepin-2(3H)-ylideneacetyl-CoA hydrolyase' WHERE ec = 'EC 3.3.2.12';
UPDATE enzyme SET history = 'EC 3.3.2.12 created 2011 as EC 3.7.1.16, transferred 2012 to EC 3.3.2.12.' WHERE ec = 'EC 3.3.2.12';
UPDATE enzyme SET sys_name = 'Chorismate pyruvate-hydrolase' WHERE ec = 'EC 3.3.2.13';
UPDATE enzyme SET history = 'EC 3.3.2.13 created 2013' WHERE ec = 'EC 3.3.2.13';
UPDATE enzyme SET sys_name = '6-amino-6-deoxyfutalosine ribohydrolase' WHERE ec = 'EC 3.3.2.30';
UPDATE enzyme SET history = 'EC 3.2.2.30 created 2014' WHERE ec = 'EC 3.3.2.30';
UPDATE enzyme SET history = 'EC 3.4.1.1 created 1961, deleted 1972' WHERE ec = 'EC 3.4.1.1';
UPDATE enzyme SET history = 'EC 3.4.1.2 created 1961, deleted 1972' WHERE ec = 'EC 3.4.1.2';
UPDATE enzyme SET history = 'EC 3.4.1.3 created 1961, deleted 1972' WHERE ec = 'EC 3.4.1.3';
UPDATE enzyme SET history = 'EC 3.4.1.4 created 1965, deleted 1972' WHERE ec = 'EC 3.4.1.4';
UPDATE enzyme SET history = 'EC 3.4.2.1 created 1961, deleted 1972 [transferred to EC 3.4.12.2, deleted 1978)' WHERE ec = 'EC 3.4.2.1';
UPDATE enzyme SET history = 'EC 3.4.2.2 created 1961, deleted 1972 (transferred to EC 3.4.12.3, deleted 1978)' WHERE ec = 'EC 3.4.2.2';
UPDATE enzyme SET history = 'EC 3.4.2.3 created 1961, deleted 1972 (transferred to EC 3.4.12.8, deleted 1978)' WHERE ec = 'EC 3.4.2.3';
UPDATE enzyme SET history = 'EC 3.4.3.1 created 1961, deleted 1972 (transferred to EC 3.4.13.1, deleted 1978 and transferred to EC 3.4.13.11, deleted 1992)' WHERE ec = 'EC 3.4.3.1';
UPDATE enzyme SET history = 'EC 3.4.3.2 created 1961, deleted 1972 (transferred to EC 3.4.13.2, deleted 1978 and transferred to EC 3.4.13.11, deleted 1992)' WHERE ec = 'EC 3.4.3.2';
UPDATE enzyme SET history = 'EC 3.4.3.3 created 1961, deleted 1972' WHERE ec = 'EC 3.4.3.3';
UPDATE enzyme SET history = 'EC 3.4.3.4 created 1961, deleted 1972' WHERE ec = 'EC 3.4.3.4';
UPDATE enzyme SET history = 'EC 3.4.3.5 created 1961, deleted 1972' WHERE ec = 'EC 3.4.3.5';
UPDATE enzyme SET history = 'EC 3.4.3.6 created 1961, deleted 1972' WHERE ec = 'EC 3.4.3.6';
UPDATE enzyme SET history = 'EC 3.4.3.7 created 1961, deleted 1972' WHERE ec = 'EC 3.4.3.7';
UPDATE enzyme SET history = 'EC 3.4.4.1 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.1';
UPDATE enzyme SET history = 'EC 3.4.4.2 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.2';
UPDATE enzyme SET history = 'EC 3.4.4.3 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.3';
UPDATE enzyme SET history = 'EC 3.4.4.4 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.4';
UPDATE enzyme SET history = 'EC 3.4.4.5 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.5';
UPDATE enzyme SET history = 'EC 3.4.4.6 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.6';
UPDATE enzyme SET history = 'EC 3.4.4.7 created 1961, deleted 1972 (transferred to EC 3.4.21.11, deleted 1984)' WHERE ec = 'EC 3.4.4.7';
UPDATE enzyme SET history = 'EC 3.4.4.8 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.8';
UPDATE enzyme SET history = 'EC 3.4.4.9 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.9';
UPDATE enzyme SET history = 'EC 3.4.4.10 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.10';
UPDATE enzyme SET history = 'EC 3.4.4.11 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.11';
UPDATE enzyme SET history = 'EC 3.4.4.12 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.12';
UPDATE enzyme SET history = 'EC 3.4.4.13 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.13';
UPDATE enzyme SET history = 'EC 3.4.4.14 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.14';
UPDATE enzyme SET history = 'EC 3.4.4.15 created 1961, deleted 1972 (transferred to EC 3.4.99.19, deleted 1984)' WHERE ec = 'EC 3.4.4.15';
UPDATE enzyme SET history = 'EC 3.4.4.16 created 1961, deleted 1972 (transferred to EC 3.4.21.14, deleted 1992)' WHERE ec = 'EC 3.4.4.16';
UPDATE enzyme SET history = 'EC 3.4.4.17 created 1961, deleted 1972 (transferred to EC 3.4.23.6, deleted 1992)' WHERE ec = 'EC 3.4.4.17';
UPDATE enzyme SET history = 'EC 3.4.4.18 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.18';
UPDATE enzyme SET history = 'EC 3.4.4.19 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.19';
UPDATE enzyme SET history = 'EC 3.4.4.20 created 1961, deleted 1972' WHERE ec = 'EC 3.4.4.20';
UPDATE enzyme SET history = 'EC 3.4.4.21 created 1965, deleted 1981, deleted 1972 (transferred to EC 3.4.21.8, deleted 1981)' WHERE ec = 'EC 3.4.4.21';
UPDATE enzyme SET history = 'EC 3.4.4.22 created 1965, deleted 1972' WHERE ec = 'EC 3.4.4.22';
UPDATE enzyme SET history = 'EC 3.4.4.23 created 1965, deleted 1972' WHERE ec = 'EC 3.4.4.23';
UPDATE enzyme SET history = 'EC 3.4.4.24 created 1965, deleted 1972 (transferred to EC 3.4.22.4, deleted 1992)' WHERE ec = 'EC 3.4.4.24';
UPDATE enzyme SET history = 'EC 3.4.4.25 created 1965, deleted 1972' WHERE ec = 'EC 3.4.4.25';
UPDATE enzyme SET history = 'EC 3.4.11.1 created 1961 as EC 3.4.1.1, transferred 1972 to EC 3.4.11.1' WHERE ec = 'EC 3.4.11.1';
UPDATE enzyme SET history = 'EC 3.4.11.2 created 1961 as EC 3.4.1.2, transferred 1972 to EC 3.4.11.2 (EC 3.4.13.6 created 1961 as EC 3.4.3.5, transferred 1972 to EC 3.4.13.6, incorporated 1997)' WHERE ec = 'EC 3.4.11.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.3';
UPDATE enzyme SET history = 'EC 3.4.11.3 created 1972' WHERE ec = 'EC 3.4.11.3';
UPDATE enzyme SET history = 'EC 3.4.11.4 created 1961 as EC 3.4.1.3, transferred 1972 to EC 3.4.11.4' WHERE ec = 'EC 3.4.11.4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.5';
UPDATE enzyme SET history = 'EC 3.4.11.5 created 1965 as EC 3.4.1.4, transferred 1972 to EC 3.4.11.5' WHERE ec = 'EC 3.4.11.5';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.6';
UPDATE enzyme SET history = 'EC 3.4.11.6 created 1972, modified 1997' WHERE ec = 'EC 3.4.11.6';
UPDATE enzyme SET history = 'EC 3.4.11.7 created 1972' WHERE ec = 'EC 3.4.11.7';
UPDATE enzyme SET history = 'EC 3.4.11.8 created 1972, deleted 1981' WHERE ec = 'EC 3.4.11.8';
UPDATE enzyme SET history = 'EC 3.4.11.9 created 1972' WHERE ec = 'EC 3.4.11.9';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.10';
UPDATE enzyme SET history = 'EC 3.4.11.10 created 1972' WHERE ec = 'EC 3.4.11.10';
UPDATE enzyme SET history = 'EC 3.4.11.11 created 1978, deleted 1992' WHERE ec = 'EC 3.4.11.11';
UPDATE enzyme SET history = 'EC 3.4.11.12 created 1978, deleted 1997' WHERE ec = 'EC 3.4.11.12';
UPDATE enzyme SET history = 'EC 3.4.11.13 created 1978' WHERE ec = 'EC 3.4.11.13';
UPDATE enzyme SET history = 'EC 3.4.11.14 created 1978' WHERE ec = 'EC 3.4.11.14';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.15';
UPDATE enzyme SET history = 'EC 3.4.11.15 created 1989, modified 1997' WHERE ec = 'EC 3.4.11.15';
UPDATE enzyme SET history = 'EC 3.4.11.16 created 1989' WHERE ec = 'EC 3.4.11.16';
UPDATE enzyme SET history = 'EC 3.4.11.17 created 1989' WHERE ec = 'EC 3.4.11.17';
UPDATE enzyme SET history = 'EC 3.4.11.18 created 1990' WHERE ec = 'EC 3.4.11.18';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.19';
UPDATE enzyme SET history = 'EC 3.4.11.19 created 1993' WHERE ec = 'EC 3.4.11.19';
UPDATE enzyme SET history = 'EC 3.4.11.20 created 1995' WHERE ec = 'EC 3.4.11.20';
UPDATE enzyme SET history = 'EC 3.4.11.21 created 2000' WHERE ec = 'EC 3.4.11.21';
UPDATE enzyme SET history = 'EC 3.4.11.22 created 1997' WHERE ec = 'EC 3.4.11.22';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.23';
UPDATE enzyme SET history = 'EC 3.4.11.24 created 2003' WHERE ec = 'EC 3.4.11.23';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.24';
UPDATE enzyme SET history = 'EC 3.4.11.24 created 2008' WHERE ec = 'EC 3.4.11.24';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.25';
UPDATE enzyme SET history = 'EC 3.4.11.25 created 2011' WHERE ec = 'EC 3.4.11.25';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.11.26';
UPDATE enzyme SET history = 'EC 3.4.11.26 created 2011' WHERE ec = 'EC 3.4.11.26';
UPDATE enzyme SET history = 'EC 3.4.12.1 created 1972, deleted 1978' WHERE ec = 'EC 3.4.12.1';
UPDATE enzyme SET history = 'EC 3.4.12.2 created 1972, deleted 1978' WHERE ec = 'EC 3.4.12.2';
UPDATE enzyme SET history = 'EC 3.4.12.3 created 1972, deleted 1978' WHERE ec = 'EC 3.4.12.3';
UPDATE enzyme SET history = 'EC 3.4.12.4 created 1972, modified 1976, deleted 1978' WHERE ec = 'EC 3.4.12.4';
UPDATE enzyme SET history = 'EC 3.4.12.5 created 1972, deleted 1978 (transferred to EC 3.4.17.7, deleted 1992)' WHERE ec = 'EC 3.4.12.5';
UPDATE enzyme SET history = 'EC 3.4.12.6 created 1972, deleted 1978' WHERE ec = 'EC 3.4.12.6';
UPDATE enzyme SET history = 'EC 3.4.12.7 created 1972, deleted 1978' WHERE ec = 'EC 3.4.12.7';
UPDATE enzyme SET history = 'EC 3.4.12.8 created 1972, deleted 1978' WHERE ec = 'EC 3.4.12.8';
UPDATE enzyme SET history = 'EC 3.4.12.9 created 1972, deleted 1978 (transferred to EC 3.4.17.5, deleted 1992)' WHERE ec = 'EC 3.4.12.9';
UPDATE enzyme SET history = 'EC 3.4.12.10 created 1972, modified 1976, deleted 1978 [transferred to EC 3.4.22.12, deleted 1992)' WHERE ec = 'EC 3.4.12.10';
UPDATE enzyme SET history = 'EC 3.4.12.11 created 1972, deleted 1978' WHERE ec = 'EC 3.4.12.11';
UPDATE enzyme SET history = 'EC 3.4.12.12 created 1972, deleted 1978 (transferred to EC 3.4.16.3, transferred 1992 to EC 3.4.16.1, deleted 1993)' WHERE ec = 'EC 3.4.12.12';
UPDATE enzyme SET history = 'EC 3.4.12.13 created 1975, modified 1976, deleted 1978' WHERE ec = 'EC 3.4.12.13';
UPDATE enzyme SET history = 'EC 3.4.13.1 created 1972, deleted 1978 (transferred to EC 3.4.13.11, deleted 1992)' WHERE ec = 'EC 3.4.13.1';
UPDATE enzyme SET history = 'EC 3.4.13.2 created 1972, deleted 1978 (transferred to EC 3.4.13.11, deleted 1992)' WHERE ec = 'EC 3.4.13.2';
UPDATE enzyme SET history = 'EC 3.4.13.3 created 1961 as EC 3.4.3.3, transferred 1972 to EC 3.4.13.3, modified 1989 (EC 3.4.13.13 created 1981, incorporated 1992), deleted 2011, deleted 2011' WHERE ec = 'EC 3.4.13.3';
UPDATE enzyme SET history = 'EC 3.4.13.4 created 1972' WHERE ec = 'EC 3.4.13.4';
UPDATE enzyme SET history = 'EC 3.4.13.5 created 1961 as EC 3.4.3.4, transferred 1972 to EC 3.4.13.5, modified 1981 (EC 3.5.1.34 created 1972, incorporated 1981)' WHERE ec = 'EC 3.4.13.5';
UPDATE enzyme SET history = 'EC 3.4.13.6 created 1961 as EC 3.4.3.5, transferred 1972 to EC 3.4.13.6' WHERE ec = 'EC 3.4.13.6';
UPDATE enzyme SET history = 'EC 3.4.13.7 created 1972' WHERE ec = 'EC 3.4.13.7';
UPDATE enzyme SET history = 'EC 3.4.13.8 created 1961 as EC 3.4.3.6, transferred 1972 to EC 3.4.13.8' WHERE ec = 'EC 3.4.13.8';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.13.9';
UPDATE enzyme SET history = 'EC 3.4.13.9 created 1961 as EC 3.4.3.7, transferred 1972 to EC 3.4.13.9' WHERE ec = 'EC 3.4.13.9';
UPDATE enzyme SET history = 'EC 3.4.13.10 created 1972, deleted 1992' WHERE ec = 'EC 3.4.13.10';
UPDATE enzyme SET history = 'EC 3.4.13.11 created 1972, deleted 1992' WHERE ec = 'EC 3.4.13.11';
UPDATE enzyme SET history = 'EC 3.4.13.12 created 1976' WHERE ec = 'EC 3.4.13.12';
UPDATE enzyme SET history = 'EC 3.4.13.13 created 1981, deleted 1992' WHERE ec = 'EC 3.4.13.13';
UPDATE enzyme SET history = 'EC 3.4.13.14 created 1989, deleted 1992' WHERE ec = 'EC 3.4.13.14';
UPDATE enzyme SET history = 'EC 3.4.13.15 created 1989, deleted 1992' WHERE ec = 'EC 3.4.13.15';
UPDATE enzyme SET history = 'EC 3.4.13.16 created 1989, deleted 1992' WHERE ec = 'EC 3.4.13.16';
UPDATE enzyme SET history = 'EC 3.4.13.17 created 1990' WHERE ec = 'EC 3.4.13.17';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.13.18';
UPDATE enzyme SET history = 'EC 3.4.13.18 created 1961 as EC 3.4.3.1 and EC 3.4.3.2, transferred 1972 to EC 3.4.13.1 and EC 3.4.13.2, transferred 1978 to EC 3.4.13.11, transferred 1992 to EC 3.4.13.18, modified 2000 (EC 3.4.13.15 created 1989, incorporated 1992)' WHERE ec = 'EC 3.4.13.18';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.13.19';
UPDATE enzyme SET history = 'EC 3.4.13.19 created 1961 as EC 3.4.3.1 and EC 3.4.3.2, transferred 1972 to EC 3.4.13.1 and EC 3.4.13.2, transferred 1978 to EC 3.4.13.11, part transferred 1992 to EC 3.4.13.19, modified 2011' WHERE ec = 'EC 3.4.13.19';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.13.20';
UPDATE enzyme SET history = 'EC 3.4.13.20 created 1992' WHERE ec = 'EC 3.4.13.20';
UPDATE enzyme SET history = 'EC 3.4.13.21 created 2001' WHERE ec = 'EC 3.4.13.21';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.13.22';
UPDATE enzyme SET history = 'EC 3.4.13.22 created 2006' WHERE ec = 'EC 3.4.13.22';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.14.1';
UPDATE enzyme SET history = 'EC 3.4.14.1 created 1961 as EC 3.4.4.9, transferred 1972 to EC 3.4.14.1' WHERE ec = 'EC 3.4.14.1';
UPDATE enzyme SET history = 'EC 3.4.14.2 created 1978' WHERE ec = 'EC 3.4.14.2';
UPDATE enzyme SET history = 'EC 3.4.14.3 created 1978, deleted 1981' WHERE ec = 'EC 3.4.14.3';
UPDATE enzyme SET history = 'EC 3.4.14.4 created 1981, modified 2001' WHERE ec = 'EC 3.4.14.4';
UPDATE enzyme SET history = 'EC 3.4.14.5 created 1981, modified 1996' WHERE ec = 'EC 3.4.14.5';
UPDATE enzyme SET history = 'EC 3.4.14.6 created 1989' WHERE ec = 'EC 3.4.14.6';
UPDATE enzyme SET history = 'EC 3.4.14.7 created 1989, deleted 1992' WHERE ec = 'EC 3.4.14.7';
UPDATE enzyme SET history = 'EC 3.4.14.8 created 1989, deleted 1992' WHERE ec = 'EC 3.4.14.8';
UPDATE enzyme SET history = 'EC 3.4.14.9 created 1992 (part of EC 3.4.14.8 created 1989, incorporated 1992), modified 2000, modified 2001, modified 2003' WHERE ec = 'EC 3.4.14.9';
UPDATE enzyme SET history = 'EC 3.4.14.10 created 1992 (part of EC 3.4.14.8 created 1989, incorporated 1992)' WHERE ec = 'EC 3.4.14.10';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.14.11';
UPDATE enzyme SET history = 'EC 3.4.14.11 created 1996' WHERE ec = 'EC 3.4.14.11';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.14.12';
UPDATE enzyme SET history = 'EC 3.4.14.12 created 2006' WHERE ec = 'EC 3.4.14.12';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.15.1';
UPDATE enzyme SET history = 'EC 3.4.15.1 created 1972, modified 1981, modified 1989, modified 1996, modified 2011' WHERE ec = 'EC 3.4.15.1';
UPDATE enzyme SET history = 'EC 3.4.15.2 created 1978, deleted 1981' WHERE ec = 'EC 3.4.15.2';
UPDATE enzyme SET history = 'EC 3.4.15.3 created 1981, modified 1989, deleted 1996' WHERE ec = 'EC 3.4.15.3';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.15.4';
UPDATE enzyme SET history = 'EC 3.4.15.4 created 1992' WHERE ec = 'EC 3.4.15.4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.15.5';
UPDATE enzyme SET history = 'EC 3.4.15.5 created 1981 as EC 3.4.15.3, modified 1989, transferred 1996 to EC 3.4.15.5' WHERE ec = 'EC 3.4.15.5';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.15.6';
UPDATE enzyme SET history = 'EC 3.4.15.6 created 2007' WHERE ec = 'EC 3.4.15.6';
UPDATE enzyme SET history = 'EC 3.4.16.1 created 1972 as EC 3.4.12.1 and EC 3.4.21.13, both transferred 1978 to EC 3.4.16.1, deleted 1993' WHERE ec = 'EC 3.4.16.1';
UPDATE enzyme SET history = 'EC 3.4.16.2 created 1972 as EC 3.4.12.4, transferred 1978 to EC 3.4.16.2' WHERE ec = 'EC 3.4.16.2';
UPDATE enzyme SET history = 'EC 3.4.16.3 created 1972 as EC 3.4.12.12, transferred 1978 to EC 3.4.16.3, deleted 1992' WHERE ec = 'EC 3.4.16.3';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.16.4';
UPDATE enzyme SET history = 'EC 3.4.16.4 created 1989' WHERE ec = 'EC 3.4.16.4';
UPDATE enzyme SET history = 'EC 3.4.16.5 created 1972 as EC 3.4.12.1, transferred 1978 to EC 3.4.16.1, part transferred 1993 to EC 3.4.16.5 (EC 3.4.16.3 created 1972 as EC 3.4.12.12, transferred 1978 to EC 3.4.16.3, incorporated 1992)' WHERE ec = 'EC 3.4.16.5';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.16.6';
UPDATE enzyme SET history = 'EC 3.4.16.6 created 1972 as EC 3.4.12.1, transferred 1978 to EC 3.4.16.1, part transferred 1993 to EC 3.4.16.6, modified 2011' WHERE ec = 'EC 3.4.16.6';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.17.1';
UPDATE enzyme SET history = 'EC 3.4.17.1 created 1961 as EC 3.4.2.1, transferred 1972 to EC 3.4.12.2, transferred 1978 to EC 3.4.17.1' WHERE ec = 'EC 3.4.17.1';
UPDATE enzyme SET history = 'EC 3.4.17.2 created 1961 as EC 3.4.2.2, transferred 1972 to EC 3.4.12.3, transferred 1978 to EC 3.4.17.2' WHERE ec = 'EC 3.4.17.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.17.3';
UPDATE enzyme SET history = 'EC 3.4.17.3 created 1972 as EC 3.4.12.7, transferred 1978 to EC 3.4.17.3, modified 1989' WHERE ec = 'EC 3.4.17.3';
UPDATE enzyme SET history = 'EC 3.4.17.4 created 1961 as EC 3.4.2.3, transferred 1972 to EC 3.4.12.8, transferred 1978 to EC 3.4.17.4 (EC 3.4.17.9 created 1981, incorporated 1992)' WHERE ec = 'EC 3.4.17.4';
UPDATE enzyme SET history = 'EC 3.4.17.5 created 1972 as EC 3.4.12.9, transferred 1978 to EC 3.4.17.5, deleted 1992' WHERE ec = 'EC 3.4.17.5';
UPDATE enzyme SET history = 'EC 3.4.17.6 created 1972 as EC 3.4.12.11, transferred 1978 to EC 3.4.17.6' WHERE ec = 'EC 3.4.17.6';
UPDATE enzyme SET history = 'EC 3.4.17.7 created 1978, deleted 1992' WHERE ec = 'EC 3.4.17.7';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.17.8';
UPDATE enzyme SET history = 'EC 3.4.17.8 created 1972 as EC 3.4.12.6, transferred 1978 to EC 3.4.17.8, updated 2006' WHERE ec = 'EC 3.4.17.8';
UPDATE enzyme SET history = 'EC 3.4.17.9 created 1981, deleted 1992' WHERE ec = 'EC 3.4.17.9';
UPDATE enzyme SET history = 'EC 3.4.17.10 created 1986, modified 2000' WHERE ec = 'EC 3.4.17.10';
UPDATE enzyme SET history = 'EC 3.4.17.11 created 1992' WHERE ec = 'EC 3.4.17.11';
UPDATE enzyme SET history = 'EC 3.4.17.12 created 1992' WHERE ec = 'EC 3.4.17.12';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.17.13';
UPDATE enzyme SET history = 'EC 3.4.17.13 created 1992' WHERE ec = 'EC 3.4.17.13';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.17.14';
UPDATE enzyme SET history = 'EC 3.4.17.14 created 1992' WHERE ec = 'EC 3.4.17.14';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.17.15';
UPDATE enzyme SET history = 'EC 3.4.17.15 created 1992' WHERE ec = 'EC 3.4.17.15';
UPDATE enzyme SET history = 'EC 3.4.17.16 created 1992' WHERE ec = 'EC 3.4.17.16';
UPDATE enzyme SET history = 'EC 3.4.17.17 created 1992' WHERE ec = 'EC 3.4.17.17';
UPDATE enzyme SET history = 'EC 3.4.17.18 created 1993' WHERE ec = 'EC 3.4.17.18';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.17.19';
UPDATE enzyme SET history = 'EC 3.4.17.19 created 1996' WHERE ec = 'EC 3.4.17.19';
UPDATE enzyme SET history = 'EC 3.4.17.20 created 1997' WHERE ec = 'EC 3.4.17.20';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.17.21';
UPDATE enzyme SET history = 'EC 3.4.17.21 created 1997, modified 2000 (EC 3.4.13.8 created 1972 and EC 3.4.19.8 created 1992, incorporated 2000)' WHERE ec = 'EC 3.4.17.21';
UPDATE enzyme SET history = 'EC 3.4.17.22 created 1997' WHERE ec = 'EC 3.4.17.22';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.17.23';
UPDATE enzyme SET history = 'EC 3.4.17.23 created 2009' WHERE ec = 'EC 3.4.17.23';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.18.1';
UPDATE enzyme SET history = 'EC 3.4.18.1 created 1981, modified 2000' WHERE ec = 'EC 3.4.18.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.19.1';
UPDATE enzyme SET history = 'EC 3.4.19.1 created 1978 as EC 3.4.14.3, transferred 1981 to EC 3.4.19.1' WHERE ec = 'EC 3.4.19.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.19.2';
UPDATE enzyme SET history = 'EC 3.4.19.2 created 1978 as EC 3.4.15.2, transferred 1981 to EC 3.4.19.2' WHERE ec = 'EC 3.4.19.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.19.3';
UPDATE enzyme SET history = 'EC 3.4.19.3 created 1972 as EC 3.4.11.8, transferred 1981 to EC 3.4.19.3, modified 1997' WHERE ec = 'EC 3.4.19.3';
UPDATE enzyme SET history = 'EC 3.4.19.4 created 1989, deleted 1992' WHERE ec = 'EC 3.4.19.4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.19.5';
UPDATE enzyme SET history = 'EC 3.4.19.5 created 1972 as EC 3.4.13.10, transferred 1992 to EC 3.4.19.5, modified 1997' WHERE ec = 'EC 3.4.19.5';
UPDATE enzyme SET history = 'EC 3.4.19.6 created 1992' WHERE ec = 'EC 3.4.19.6';
UPDATE enzyme SET history = 'EC 3.4.19.7 created 1992' WHERE ec = 'EC 3.4.19.7';
UPDATE enzyme SET history = 'EC 3.4.19.8 created 1992, deleted 2000' WHERE ec = 'EC 3.4.19.8';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.19.9';
UPDATE enzyme SET history = 'EC 3.4.19.9 created 1972 as EC 3.4.12.10, transferred 1978 to EC 3.4.22.12, transferred 1992 to EC 3.4.19.9, modified 1997' WHERE ec = 'EC 3.4.19.9';
UPDATE enzyme SET history = 'EC 3.4.19.10 created 1972 as EC 3.4.12.5, transferred 1978 to EC 3.4.17.7, transferred 1992 to EC 3.4.19.10, deleted 1997' WHERE ec = 'EC 3.4.19.10';
UPDATE enzyme SET history = 'EC 3.4.19.11 created 1996' WHERE ec = 'EC 3.4.19.11';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.19.12';
UPDATE enzyme SET history = 'EC 3.4.19.12 created 2000' WHERE ec = 'EC 3.4.19.12';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.19.13';
UPDATE enzyme SET history = 'EC 3.4.19.13 created 2011' WHERE ec = 'EC 3.4.19.13';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.19.14';
UPDATE enzyme SET history = 'EC 3.4.19.14 created 2012' WHERE ec = 'EC 3.4.19.14';
UPDATE enzyme SET history = 'EC 3.4.21.1 created 1961 as EC 3.4.4.5 and EC 3.4.4.6, transferred 1972 to EC 3.4.21.1' WHERE ec = 'EC 3.4.21.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.2';
UPDATE enzyme SET history = 'EC 3.4.21.2 created 1972' WHERE ec = 'EC 3.4.21.2';
UPDATE enzyme SET history = 'EC 3.4.21.3 created 1972' WHERE ec = 'EC 3.4.21.3';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.4';
UPDATE enzyme SET history = 'EC 3.4.21.4 created 1961 as EC 3.4.4.4, transferred 1972 to EC 3.4.21.4' WHERE ec = 'EC 3.4.21.4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.5';
UPDATE enzyme SET history = 'EC 3.4.21.5 created 1961 as EC 3.4.4.13, transferred 1972 to EC 3.4.21.5' WHERE ec = 'EC 3.4.21.5';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.6';
UPDATE enzyme SET history = 'EC 3.4.21.6 created 1972, modified 2010' WHERE ec = 'EC 3.4.21.6';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.7';
UPDATE enzyme SET history = 'EC 3.4.21.7 created 1961 as EC 3.4.4.14, transferred 1972 to EC 3.4.21.7' WHERE ec = 'EC 3.4.21.7';
UPDATE enzyme SET history = 'EC 3.4.21.8 created 1972, deleted 1981' WHERE ec = 'EC 3.4.21.8';
UPDATE enzyme SET history = 'EC 3.4.21.9 created 1961 as EC 3.4.4.8, transferred 1972 to EC 3.4.21.9' WHERE ec = 'EC 3.4.21.9';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.10';
UPDATE enzyme SET history = 'EC 3.4.21.10 created 1972' WHERE ec = 'EC 3.4.21.10';
UPDATE enzyme SET history = 'EC 3.4.21.11 created 1972, deleted 1981' WHERE ec = 'EC 3.4.21.11';
UPDATE enzyme SET history = 'EC 3.4.21.12 created 1972' WHERE ec = 'EC 3.4.21.12';
UPDATE enzyme SET history = 'EC 3.4.21.13 created 1972, deleted 1978' WHERE ec = 'EC 3.4.21.13';
UPDATE enzyme SET history = 'EC 3.4.21.14 created 1961 as EC 3.4.4.16, transferred 1972 to EC 3.4.21.14, modified 1986, deleted 1992' WHERE ec = 'EC 3.4.21.14';
UPDATE enzyme SET history = 'EC 3.4.21.15 created 1972, deleted 1978 (transferred to EC 3.4.21.14, deleted 1992)' WHERE ec = 'EC 3.4.21.15';
UPDATE enzyme SET history = 'EC 3.4.21.16 created 1972, deleted 1992' WHERE ec = 'EC 3.4.21.16';
UPDATE enzyme SET history = 'EC 3.4.21.17 created 1972, deleted 1978 (transferred to EC 3.4.21.14, deleted 1992)' WHERE ec = 'EC 3.4.21.17';
UPDATE enzyme SET history = 'EC 3.4.21.18 created 1972 (EC 3.4.99.24 created 1972, incorporated 1978), deleted 1992' WHERE ec = 'EC 3.4.21.18';
UPDATE enzyme SET history = 'EC 3.4.21.19 created 1978' WHERE ec = 'EC 3.4.21.19';
UPDATE enzyme SET history = 'EC 3.4.21.20 created 1978' WHERE ec = 'EC 3.4.21.20';
UPDATE enzyme SET history = 'EC 3.4.21.21 created 1978' WHERE ec = 'EC 3.4.21.21';
UPDATE enzyme SET history = 'EC 3.4.21.22 created 1978' WHERE ec = 'EC 3.4.21.22';
UPDATE enzyme SET history = 'EC 3.4.21.23 created 1978, deleted 1992' WHERE ec = 'EC 3.4.21.23';
UPDATE enzyme SET history = 'EC 3.4.21.24 created 1978, deleted 1992' WHERE ec = 'EC 3.4.21.24';
UPDATE enzyme SET history = 'EC 3.4.21.25 created 1978' WHERE ec = 'EC 3.4.21.25';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.26';
UPDATE enzyme SET history = 'EC 3.4.21.26 created 1978, modified 1981 (EC 3.4.22.18 created 1981, incorporated 1992)' WHERE ec = 'EC 3.4.21.26';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.27';
UPDATE enzyme SET history = 'EC 3.4.21.27 created 1978' WHERE ec = 'EC 3.4.21.27';
UPDATE enzyme SET history = 'EC 3.4.21.28 created 1978, deleted 1992' WHERE ec = 'EC 3.4.21.28';
UPDATE enzyme SET history = 'EC 3.4.21.29 created 1978, deleted 1992' WHERE ec = 'EC 3.4.21.29';
UPDATE enzyme SET history = 'EC 3.4.21.30 created 1978, deleted 1992' WHERE ec = 'EC 3.4.21.30';
UPDATE enzyme SET history = 'EC 3.4.21.31 created 1972 as EC 3.4.99.26, transferred 1978 to EC 3.4.21.31, deleted 1992' WHERE ec = 'EC 3.4.21.31';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.32';
UPDATE enzyme SET history = 'EC 3.4.21.32 created 1978' WHERE ec = 'EC 3.4.21.32';
UPDATE enzyme SET history = 'EC 3.4.21.33 created 1978, deleted 1992' WHERE ec = 'EC 3.4.21.33';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.34';
UPDATE enzyme SET history = 'EC 3.4.21.34 created 1965 as EC 3.4.4.21, transferred 1972 to 3.4.21.8, part transferred 1981 to EC 3.4.21.34' WHERE ec = 'EC 3.4.21.34';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.35';
UPDATE enzyme SET history = 'EC 3.4.21.35 created 1965 as EC 3.4.4.21, transferred 1972 to 3.4.21.8, part transferred 1981 to EC 3.4.21.35' WHERE ec = 'EC 3.4.21.35';
UPDATE enzyme SET history = 'EC 3.4.21.36 created 1981 (EC 3.4.4.7 created 1961, transferred 1972 to EC 3.4.21.11 created 1972, part incorporated 1984)' WHERE ec = 'EC 3.4.21.36';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.37';
UPDATE enzyme SET history = 'EC 3.4.21.37 created 1981 (EC 3.4.4.7 created 1961, transferred 1972 to EC 3.4.21.11 created 1972, part incorporated 1984)' WHERE ec = 'EC 3.4.21.37';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.38';
UPDATE enzyme SET history = 'EC 3.4.21.38 created 1981' WHERE ec = 'EC 3.4.21.38';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.39';
UPDATE enzyme SET history = 'EC 3.4.21.39 created 1981' WHERE ec = 'EC 3.4.21.39';
UPDATE enzyme SET history = 'EC 3.4.21.40 created 1981, deleted 1992' WHERE ec = 'EC 3.4.21.40';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.41';
UPDATE enzyme SET history = 'EC 3.4.21.41 created 1981' WHERE ec = 'EC 3.4.21.41';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.42';
UPDATE enzyme SET history = 'EC 3.4.21.42 created 1981' WHERE ec = 'EC 3.4.21.42';
UPDATE enzyme SET history = 'EC 3.4.21.43 created 1981 (EC 3.4.21.44 created 1981, incorporated 1984)' WHERE ec = 'EC 3.4.21.43';
UPDATE enzyme SET history = 'EC 3.4.21.44 created 1981, deleted 1984' WHERE ec = 'EC 3.4.21.44';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.45';
UPDATE enzyme SET history = 'EC 3.4.21.45 created 1981' WHERE ec = 'EC 3.4.21.45';
UPDATE enzyme SET history = 'EC 3.4.21.46 created 1981' WHERE ec = 'EC 3.4.21.46';
UPDATE enzyme SET history = 'EC 3.4.21.47 created 1981' WHERE ec = 'EC 3.4.21.47';
UPDATE enzyme SET history = 'EC 3.4.21.48 created 1972 as EC 3.4.22.9, transferred 1981 to EC 3.4.21.48' WHERE ec = 'EC 3.4.21.48';
UPDATE enzyme SET history = 'EC 3.4.21.49 created 1981' WHERE ec = 'EC 3.4.21.49';
UPDATE enzyme SET history = 'EC 3.4.21.50 created 1983' WHERE ec = 'EC 3.4.21.50';
UPDATE enzyme SET history = 'EC 3.4.21.51 created 1984, deleted 1992' WHERE ec = 'EC 3.4.21.51';
UPDATE enzyme SET history = 'EC 3.4.21.52 created 1981 as EC 3.4.99.33, transferred 1984 to EC 3.4.21.52, deleted 1992' WHERE ec = 'EC 3.4.21.52';
UPDATE enzyme SET history = 'EC 3.4.21.53 created 1986' WHERE ec = 'EC 3.4.21.53';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.54';
UPDATE enzyme SET history = 'EC 3.4.21.54 created 1986' WHERE ec = 'EC 3.4.21.54';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.55';
UPDATE enzyme SET history = 'EC 3.4.21.55 created 1989' WHERE ec = 'EC 3.4.21.55';
UPDATE enzyme SET history = 'EC 3.4.21.56 created 1972 as EC 3.4.99.7, transferred 1989 to EC EC 3.4.21.56, deleted 1992' WHERE ec = 'EC 3.4.21.56';
UPDATE enzyme SET history = 'EC 3.4.21.57 created 1989' WHERE ec = 'EC 3.4.21.57';
UPDATE enzyme SET history = 'EC 3.4.21.58 created 1989, deleted 1992' WHERE ec = 'EC 3.4.21.58';
UPDATE enzyme SET history = 'EC 3.4.21.59 created 1992' WHERE ec = 'EC 3.4.21.59';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.60';
UPDATE enzyme SET history = 'EC 3.4.21.60 created 1978 as EC 3.4.99.28, transferred 1992 to EC 3.4.21.60, modified 2010' WHERE ec = 'EC 3.4.21.60';
UPDATE enzyme SET history = 'EC 3.4.21.61 created 1989 as EC 3.4.22.23, transferred 1992 to EC 3.4.21.61' WHERE ec = 'EC 3.4.21.61';
UPDATE enzyme SET history = 'EC 3.4.21.62 created 1992 (EC 3.4.21.14 created 1961 as EC 3.4.4.16, transferred 1972 to EC 3.4.21.14, modified 1986, part incorporated 1992)' WHERE ec = 'EC 3.4.21.62';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.63';
UPDATE enzyme SET history = 'EC 3.4.21.63 created 1992 (EC 3.4.21.14 created 1961 as EC 3.4.4.16, transferred 1972 to EC 3.4.21.14, modified 1986, part incorporated 1992)' WHERE ec = 'EC 3.4.21.63';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.64';
UPDATE enzyme SET history = 'EC 3.4.21.64 created 1992 (EC 3.4.21.14 created 1961 as EC 3.4.4.16, transferred 1972 to EC 3.4.21.14, modified 1986, part incorporated 1992)' WHERE ec = 'EC 3.4.21.64';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.65';
UPDATE enzyme SET history = 'EC 3.4.21.65 created 1992 (EC 3.4.21.14 created 1961 as EC 3.4.4.16, transferred 1972 to EC 3.4.21.14, modified 1986, part incorporated 1992)' WHERE ec = 'EC 3.4.21.65';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.66';
UPDATE enzyme SET history = 'EC 3.4.21.66 created 1992' WHERE ec = 'EC 3.4.21.66';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.67';
UPDATE enzyme SET history = 'EC 3.4.21.67 created 1992 (EC 3.4.21.14 created 1961 as EC 3.4.4.16, transferred 1972 to EC 3.4.21.14, modified 1986, part incorporated 1992)' WHERE ec = 'EC 3.4.21.67';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.68';
UPDATE enzyme SET history = 'EC 3.4.21.68 created 1972 as EC 3.4.99.26, transferred 1978 as EC 3.4.21.31, part transferred 1992 to EC 3.4.21.68' WHERE ec = 'EC 3.4.21.68';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.69';
UPDATE enzyme SET history = 'EC 3.4.21.69 created 1992' WHERE ec = 'EC 3.4.21.69';
UPDATE enzyme SET history = 'EC 3.4.21.70 created 1992' WHERE ec = 'EC 3.4.21.70';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.71';
UPDATE enzyme SET history = 'EC 3.4.21.71 created 1992' WHERE ec = 'EC 3.4.21.71';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.72';
UPDATE enzyme SET history = 'EC 3.4.21.72 created 1992' WHERE ec = 'EC 3.4.21.72';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.73';
UPDATE enzyme SET history = 'EC 3.4.21.73 created 1972 as EC 3.4.99.26, transferred 1978 as EC 3.4.21.31, part transferred 1992 to EC 3.4.21.73' WHERE ec = 'EC 3.4.21.73';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.74';
UPDATE enzyme SET history = 'EC 3.4.21.74 created 1992 (EC 3.4.21.28, EC 3.4.21.29 and 3.4.21.30 all created 1978 and incorporated 1992)' WHERE ec = 'EC 3.4.21.74';
UPDATE enzyme SET history = 'EC 3.4.21.75 created 1993' WHERE ec = 'EC 3.4.21.75';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.76';
UPDATE enzyme SET history = 'EC 3.4.21.76 created 1993' WHERE ec = 'EC 3.4.21.76';
UPDATE enzyme SET history = 'EC 3.4.21.77 created 1993' WHERE ec = 'EC 3.4.21.77';
UPDATE enzyme SET history = 'EC 3.4.21.78 created 1993' WHERE ec = 'EC 3.4.21.78';
UPDATE enzyme SET history = 'EC 3.4.21.79 created 1993' WHERE ec = 'EC 3.4.21.79';
UPDATE enzyme SET history = 'EC 3.4.21.80 created 1993' WHERE ec = 'EC 3.4.21.80';
UPDATE enzyme SET history = 'EC 3.4.21.81 created 1993' WHERE ec = 'EC 3.4.21.81';
UPDATE enzyme SET history = 'EC 3.4.21.82 created 1993' WHERE ec = 'EC 3.4.21.82';
UPDATE enzyme SET history = 'EC 3.4.21.83 created 1993' WHERE ec = 'EC 3.4.21.83';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.84';
UPDATE enzyme SET history = 'EC 3.4.21.84 created 1993' WHERE ec = 'EC 3.4.21.84';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.85';
UPDATE enzyme SET history = 'EC 3.4.21.85 created 1993' WHERE ec = 'EC 3.4.21.85';
UPDATE enzyme SET history = 'EC 3.4.21.86 created 1993' WHERE ec = 'EC 3.4.21.86';
UPDATE enzyme SET history = 'EC 3.4.21.87 created 1993, deleted 2006' WHERE ec = 'EC 3.4.21.87';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.88';
UPDATE enzyme SET history = 'EC 3.4.21.88 created 1995' WHERE ec = 'EC 3.4.21.88';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.89';
UPDATE enzyme SET history = 'EC 3.4.21.89 created 1984 as EC 3.4.99.36, transferred 1995 to EC 3.4.21.89' WHERE ec = 'EC 3.4.21.89';
UPDATE enzyme SET history = 'EC 3.4.21.90 created 1995' WHERE ec = 'EC 3.4.21.90';
UPDATE enzyme SET history = 'EC 3.4.21.91 created 1995' WHERE ec = 'EC 3.4.21.91';
UPDATE enzyme SET history = 'EC 3.4.21.92 created 1996' WHERE ec = 'EC 3.4.21.92';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.93';
UPDATE enzyme SET history = 'EC 3.4.21.93 created 1996' WHERE ec = 'EC 3.4.21.93';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.94';
UPDATE enzyme SET history = 'EC 3.4.21.94 created 1996' WHERE ec = 'EC 3.4.21.94';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.95';
UPDATE enzyme SET history = 'EC 3.4.21.95 created 1997' WHERE ec = 'EC 3.4.21.95';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.96';
UPDATE enzyme SET history = 'EC 3.4.21.96 created 1997' WHERE ec = 'EC 3.4.21.96';
UPDATE enzyme SET history = 'EC 3.4.21.97 created 2000' WHERE ec = 'EC 3.4.21.97';
UPDATE enzyme SET history = 'EC 3.4.21.98 created 2000' WHERE ec = 'EC 3.4.21.98';
UPDATE enzyme SET history = 'EC 3.4.21.99 created 2001' WHERE ec = 'EC 3.4.21.99';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.100';
UPDATE enzyme SET history = 'EC 3.4.21.100 created 1995 as EC 3.4.23.37, transferred 2001 to  EC 3.4.21.100, modified 2003' WHERE ec = 'EC 3.4.21.100';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.101';
UPDATE enzyme SET history = 'EC 3.4.21.101 created 1995 as EC 3.4.23.33, transferred 2001 to  EC 3.4.21.101, modified 2003' WHERE ec = 'EC 3.4.21.101';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.102';
UPDATE enzyme SET history = 'EC 3.4.21.102 created 2001' WHERE ec = 'EC 3.4.21.102';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.103';
UPDATE enzyme SET history = 'EC 3.4.21.103 created 1992 as EC 3.4.23.27 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992), transferred 2003 to EC 3.4.21.103' WHERE ec = 'EC 3.4.21.103';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.104';
UPDATE enzyme SET history = 'EC 3.4.21.104 created 2005' WHERE ec = 'EC 3.4.21.104';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.105';
UPDATE enzyme SET history = 'EC 3.4.21.105 created 2005' WHERE ec = 'EC 3.4.21.105';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.106';
UPDATE enzyme SET history = 'EC 3.4.21.106 created 2006' WHERE ec = 'EC 3.4.21.106';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.107';
UPDATE enzyme SET history = 'EC 3.4.21.107 created 2006' WHERE ec = 'EC 3.4.21.107';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.108';
UPDATE enzyme SET history = 'EC 3.4.21.108 created 2006' WHERE ec = 'EC 3.4.21.108';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.109';
UPDATE enzyme SET history = 'EC 3.4.21.109 created 2006' WHERE ec = 'EC 3.4.21.109';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.110';
UPDATE enzyme SET history = 'EC 3.4.21.110 created 2006' WHERE ec = 'EC 3.4.21.110';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.111';
UPDATE enzyme SET history = 'EC 3.4.21.111 created 2006' WHERE ec = 'EC 3.4.21.111';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.112';
UPDATE enzyme SET history = 'EC 3.4.21.112 created 2006' WHERE ec = 'EC 3.4.21.112';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.113';
UPDATE enzyme SET history = 'EC 3.4.21.113 created 2006' WHERE ec = 'EC 3.4.21.113';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.114';
UPDATE enzyme SET history = 'EC 3.4.21.114 created 2006' WHERE ec = 'EC 3.4.21.114';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.115';
UPDATE enzyme SET history = 'EC 3.4.21.115 created 2006' WHERE ec = 'EC 3.4.21.115';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.116';
UPDATE enzyme SET history = 'EC 3.4.21.116 created 2006' WHERE ec = 'EC 3.4.21.116';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.117';
UPDATE enzyme SET history = 'EC 3.4.21.117 created 2006' WHERE ec = 'EC 3.4.21.117';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.118';
UPDATE enzyme SET history = 'EC 3.4.21.118 created 2006' WHERE ec = 'EC 3.4.21.118';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.119';
UPDATE enzyme SET history = 'EC 3.4.21.119 created 2006' WHERE ec = 'EC 3.4.21.119';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.120';
UPDATE enzyme SET history = 'EC 3.4.21.120 created 2007' WHERE ec = 'EC 3.4.21.120';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.21.121';
UPDATE enzyme SET history = 'EC 3.4.21.121 created 2013' WHERE ec = 'EC 3.4.21.121';
UPDATE enzyme SET history = 'EC 3.4.22.1 created 1972' WHERE ec = 'EC 3.4.22.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.2';
UPDATE enzyme SET history = 'EC 3.4.22.2 created 1961 as EC 3.4.4.10, transferred 1972 to EC 3.4.22.2, modified 1976, modified 2000' WHERE ec = 'EC 3.4.22.2';
UPDATE enzyme SET history = 'EC 3.4.22.3 created 1961 as EC 3.4.4.12, transferred 1972 to EC 3.4.22.3' WHERE ec = 'EC 3.4.22.3';
UPDATE enzyme SET history = 'EC 3.4.22.4 created 1972, deleted 1992 (EC 3.4.22.5 created 1972, incorporated 1978)' WHERE ec = 'EC 3.4.22.4';
UPDATE enzyme SET history = 'EC 3.4.22.5 created 1972, deleted 1978' WHERE ec = 'EC 3.4.22.5';
UPDATE enzyme SET history = 'EC 3.4.22.6 created 1961 as EC 3.4.4.11, transferred 1972 to EC 3.4.22.6' WHERE ec = 'EC 3.4.22.6';
UPDATE enzyme SET history = 'EC 3.4.22.7 created 1972' WHERE ec = 'EC 3.4.22.7';
UPDATE enzyme SET history = 'EC 3.4.22.8 created 1961 as EC 3.4.4.20, transferred 1972 to EC 3.4.22.8' WHERE ec = 'EC 3.4.22.8';
UPDATE enzyme SET history = 'EC 3.4.22.9 created 1972, deleted 1981' WHERE ec = 'EC 3.4.22.9';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.10';
UPDATE enzyme SET history = 'EC 3.4.22.10 created 1961 as EC 3.4.4.18, transferred 1972 to EC 3.4.22.10' WHERE ec = 'EC 3.4.22.10';
UPDATE enzyme SET history = 'EC 3.4.22.11 created 1976, deleted 1978 (transferred to EC 3.4.99.45, deleted 1993)' WHERE ec = 'EC 3.4.22.11';
UPDATE enzyme SET history = 'EC 3.4.22.12 created 1978, deleted 1992' WHERE ec = 'EC 3.4.22.12';
UPDATE enzyme SET history = 'EC 3.4.22.13 created 1978, modified 1981, deleted 1992' WHERE ec = 'EC 3.4.22.13';
UPDATE enzyme SET history = 'EC 3.4.22.14 created 1978' WHERE ec = 'EC 3.4.22.14';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.15';
UPDATE enzyme SET history = 'EC 3.4.22.15 created 1978 (EC 3.4.99.19 created 1972, incorporated 1981)' WHERE ec = 'EC 3.4.22.15';
UPDATE enzyme SET history = 'EC 3.4.22.16 created 1981, modified 1989' WHERE ec = 'EC 3.4.22.16';
UPDATE enzyme SET history = 'EC 3.4.22.17 created 1981 (EC 3.4.24.5 created 1978, part incorporated 1989), deleted 2003' WHERE ec = 'EC 3.4.22.17';
UPDATE enzyme SET history = 'EC 3.4.22.18 created 1981, deleted 1992' WHERE ec = 'EC 3.4.22.18';
UPDATE enzyme SET history = 'EC 3.4.22.19 created 1989, deleted 1992' WHERE ec = 'EC 3.4.22.19';
UPDATE enzyme SET history = 'EC 3.4.22.20 created 1989, deleted 1992' WHERE ec = 'EC 3.4.22.20';
UPDATE enzyme SET history = 'EC 3.4.22.21 created 1989, deleted 1992' WHERE ec = 'EC 3.4.22.21';
UPDATE enzyme SET history = 'EC 3.4.22.22 created 1989, deleted 1992' WHERE ec = 'EC 3.4.22.22';
UPDATE enzyme SET history = 'EC 3.4.22.23 created 1989, deleted 1992' WHERE ec = 'EC 3.4.22.23';
UPDATE enzyme SET history = 'EC 3.4.22.24 created 1990' WHERE ec = 'EC 3.4.22.24';
UPDATE enzyme SET history = 'EC 3.4.22.25 created 1992' WHERE ec = 'EC 3.4.22.25';
UPDATE enzyme SET history = 'EC 3.4.22.26 created 1992' WHERE ec = 'EC 3.4.22.26';
UPDATE enzyme SET history = 'EC 3.4.22.27 created 1992' WHERE ec = 'EC 3.4.22.27';
UPDATE enzyme SET history = 'EC 3.4.22.28 created 1992' WHERE ec = 'EC 3.4.22.28';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.29';
UPDATE enzyme SET history = 'EC 3.4.22.29 created 1992' WHERE ec = 'EC 3.4.22.29';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.30';
UPDATE enzyme SET history = 'EC 3.4.22.30 created 1992' WHERE ec = 'EC 3.4.22.30';
UPDATE enzyme SET history = 'EC 3.4.22.31 created 1992' WHERE ec = 'EC 3.4.22.31';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.32';
UPDATE enzyme SET history = 'EC 3.4.22.32 created 1965 as EC 3.4.4.24, transferred 1972 to EC 3.4.22.4, part transferred 1992 to EC 3.4.22.32' WHERE ec = 'EC 3.4.22.32';
UPDATE enzyme SET history = 'EC 3.4.22.33 created 1965 as EC 3.4.4.24, transferred 1972 to EC 3.4.22.4, part transferred 1992 to EC 3.4.22.33' WHERE ec = 'EC 3.4.22.33';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.34';
UPDATE enzyme SET history = 'EC 3.4.22.34 created 1992, modified 2000' WHERE ec = 'EC 3.4.22.34';
UPDATE enzyme SET history = 'EC 3.4.22.35 created 1992' WHERE ec = 'EC 3.4.22.35';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.36';
UPDATE enzyme SET history = 'EC 3.4.22.36 created 1993, modified 1997, modified 2007' WHERE ec = 'EC 3.4.22.36';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.37';
UPDATE enzyme SET history = 'EC 3.4.22.37 created 1996' WHERE ec = 'EC 3.4.22.37';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.38';
UPDATE enzyme SET history = 'EC 3.4.22.38 created 1997' WHERE ec = 'EC 3.4.22.38';
UPDATE enzyme SET history = 'EC 3.4.22.39 created 2000' WHERE ec = 'EC 3.4.22.39';
UPDATE enzyme SET history = 'EC 3.4.22.40 created 2000' WHERE ec = 'EC 3.4.22.40';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.41';
UPDATE enzyme SET history = 'EC 3.4.22.41 created 2000' WHERE ec = 'EC 3.4.22.41';
UPDATE enzyme SET history = 'EC 3.4.22.42 created 2000' WHERE ec = 'EC 3.4.22.42';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.43';
UPDATE enzyme SET history = 'EC 3.4.22.43 created 2000' WHERE ec = 'EC 3.4.22.43';
UPDATE enzyme SET history = 'EC 3.4.22.44 created 2000' WHERE ec = 'EC 3.4.22.44';
UPDATE enzyme SET history = 'EC 3.4.22.45 created 2001' WHERE ec = 'EC 3.4.22.45';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.46';
UPDATE enzyme SET history = 'EC 3.4.22.46 created 2001' WHERE ec = 'EC 3.4.22.46';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.47';
UPDATE enzyme SET history = 'EC 3.4.22.47 created 2003' WHERE ec = 'EC 3.4.22.47';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.48';
UPDATE enzyme SET history = 'EC 3.4.22.48 created 2003' WHERE ec = 'EC 3.4.22.48';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.49';
UPDATE enzyme SET history = 'EC 3.4.22.49 created 2003' WHERE ec = 'EC 3.4.22.49';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.50';
UPDATE enzyme SET history = 'EC 3.4.22.50 created 2003' WHERE ec = 'EC 3.4.22.50';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.51';
UPDATE enzyme SET history = 'EC 3.4.23.51 created 2003' WHERE ec = 'EC 3.4.22.51';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.52';
UPDATE enzyme SET history = 'EC 3.4.22.52 created 1981 as EC 3.4.22.17, transferred 2003 to EC 3.4.22.52' WHERE ec = 'EC 3.4.22.52';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.53';
UPDATE enzyme SET history = 'EC 3.4.23.53 created 1981 as EC 3.4.22.17, transferred 2003 to EC 3.4.22.53' WHERE ec = 'EC 3.4.22.53';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.54';
UPDATE enzyme SET history = 'EC 3.4.22.54 created 2007' WHERE ec = 'EC 3.4.22.54';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.55';
UPDATE enzyme SET history = 'EC 3.4.22.55 created 2007' WHERE ec = 'EC 3.4.22.55';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.56';
UPDATE enzyme SET history = 'EC 3.4.22.56 created 2007' WHERE ec = 'EC 3.4.22.56';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.57';
UPDATE enzyme SET history = 'EC 3.4.22.57 created 2007' WHERE ec = 'EC 3.4.22.57';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.58';
UPDATE enzyme SET history = 'EC 3.4.22.58 created 2007' WHERE ec = 'EC 3.4.22.58';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.59';
UPDATE enzyme SET history = 'EC 2.4.22.59 created 2007' WHERE ec = 'EC 3.4.22.59';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.60';
UPDATE enzyme SET history = 'EC 3.4.22.60 created 2007' WHERE ec = 'EC 3.4.22.60';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.61';
UPDATE enzyme SET history = 'EC 3.4.22.61 created 2007' WHERE ec = 'EC 3.4.22.61';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.62';
UPDATE enzyme SET history = 'EC 3.4.22.62 created 2007' WHERE ec = 'EC 3.4.22.62';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.63';
UPDATE enzyme SET history = 'EC 3.4.22.63 created 2007' WHERE ec = 'EC 3.4.22.63';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.64';
UPDATE enzyme SET history = 'EC 3.4.22.64 created 2007' WHERE ec = 'EC 3.4.22.64';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.65';
UPDATE enzyme SET history = 'EC 3.4.22.65 created 2007' WHERE ec = 'EC 3.4.22.65';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.66';
UPDATE enzyme SET history = 'EC 3.4.22.66 created 2007' WHERE ec = 'EC 3.4.22.66';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.67';
UPDATE enzyme SET history = 'EC 3.4.22.67 created 2007' WHERE ec = 'EC 3.4.22.67';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.68';
UPDATE enzyme SET history = 'EC 3.4.22.68 created 2008, modified 2011' WHERE ec = 'EC 3.4.22.68';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.69';
UPDATE enzyme SET history = 'EC 3.4.22.69 created 2009' WHERE ec = 'EC 3.4.22.69';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.70';
UPDATE enzyme SET history = 'EC 3.4.22.70 created 2009' WHERE ec = 'EC 3.4.22.70';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.22.71';
UPDATE enzyme SET history = 'EC 3.4.22.71 created 2009' WHERE ec = 'EC 3.4.22.71';
UPDATE enzyme SET history = 'EC 3.4.23.1 created 1961 as EC 3.4.4.1, transferred 1972 to EC 3.4.23.1, modified 1986, modified 1989' WHERE ec = 'EC 3.4.23.1';
UPDATE enzyme SET history = 'EC 3.4.23.2 created 1961 as EC 3.4.4.2, transferred 1972 to EC 3.4.23.2, modified 1986' WHERE ec = 'EC 3.4.23.2';
UPDATE enzyme SET history = 'EC 3.4.23.3 created 1965 as EC 3.4.4.22, transferred 1972 to EC 3.4.23.3, modified 1986' WHERE ec = 'EC 3.4.23.3';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.4';
UPDATE enzyme SET history = 'EC 3.4.23.4 created 1961 as EC 3.4.4.3, transferred 1972 to EC 3.4.23.4, modified 1986' WHERE ec = 'EC 3.4.23.4';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.5';
UPDATE enzyme SET history = 'EC 3.4.23.5 created 1965 as EC 3.4.4.23, transferred 1972 to EC 3.4.23.5, modified 1986' WHERE ec = 'EC 3.4.23.5';
UPDATE enzyme SET history = 'EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981, deleted 1992 (EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978)' WHERE ec = 'EC 3.4.23.6';
UPDATE enzyme SET history = 'EC 3.4.23.7 created 1972, modified 1981, deleted 1978 (transferred to EC 3.4.23.6, deleted 1992)' WHERE ec = 'EC 3.4.23.7';
UPDATE enzyme SET history = 'EC 3.4.23.8 created 1972, modified 1981, deleted 1978 (transferred to EC 3.4.23.6, deleted 1992)' WHERE ec = 'EC 3.4.23.8';
UPDATE enzyme SET history = 'EC 3.4.23.9 created 1972, modified 1981, deleted 1978 (transferred to EC 3.4.23.6, deleted 1992)' WHERE ec = 'EC 3.4.23.9';
UPDATE enzyme SET history = 'EC 3.4.23.10 created 1972, modified 1981, deleted 1978 (transferred to EC 3.4.23.6, deleted 1992)' WHERE ec = 'EC 3.4.23.10';
UPDATE enzyme SET history = 'EC 3.4.23.11 created 1978, modified 1981, deleted 1992' WHERE ec = 'EC 3.4.23.11';
UPDATE enzyme SET history = 'EC 3.4.23.12 created 1972 as EC 3.4.99.4, transferred 1978 to EC 3.4.23.12, modified 1981' WHERE ec = 'EC 3.4.23.12';
UPDATE enzyme SET history = 'EC 3.4.23.13 created 1978, modified 1981, deleted 1992' WHERE ec = 'EC 3.4.23.13';
UPDATE enzyme SET history = 'EC 3.4.23.14 created 1978, modified 1981, deleted 1992' WHERE ec = 'EC 3.4.23.14';
UPDATE enzyme SET history = 'EC 3.4.23.15 created 1961 as EC 3.4.4.15, transferred 1972 to EC 3.4.99.19, transferred 1981 to EC 3.4.23.15' WHERE ec = 'EC 3.4.23.15';
UPDATE enzyme SET history = 'EC 3.4.23.16 created 1992, modified 2000' WHERE ec = 'EC 3.4.23.16';
UPDATE enzyme SET history = 'EC 3.4.23.17 created 1989 as EC 3.4.99.38, transferred 1992 to EC 3.4.23.17' WHERE ec = 'EC 3.4.23.17';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.18';
UPDATE enzyme SET history = 'EC 3.4.23.18 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.18';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.19';
UPDATE enzyme SET history = 'EC 3.4.23.19 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.19';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.20';
UPDATE enzyme SET history = 'EC 3.4.23.20 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.20';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.21';
UPDATE enzyme SET history = 'EC 3.4.23.21 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.21';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.22';
UPDATE enzyme SET history = 'EC 3.4.23.22 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.22';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.23';
UPDATE enzyme SET history = 'EC 3.4.23.23 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.23';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.24';
UPDATE enzyme SET history = 'EC 3.4.23.24 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.24';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.25';
UPDATE enzyme SET history = 'EC 3.4.23.25 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.25';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.26';
UPDATE enzyme SET history = 'EC 3.4.23.26 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.26';
UPDATE enzyme SET history = 'EC 3.4.23.27 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992), deleted 2003' WHERE ec = 'EC 3.4.23.27';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.28';
UPDATE enzyme SET history = 'EC 3.4.23.28 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.28';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.29';
UPDATE enzyme SET history = 'EC 3.4.23.29 created 1992' WHERE ec = 'EC 3.4.23.29';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.30';
UPDATE enzyme SET history = 'EC 3.4.23.30 created 1992 (EC 3.4.23.6 created 1992 (EC 3.4.23.6 created 1961 as EC 3.4.4.17, transferred 1972 to EC 3.4.23.6, modified 1981 [EC 3.4.23.7, EC 3.4.23.8, EC 3.4.23.9, EC 3.4.23.10, EC 3.4.99.1, EC 3.4.99.15 and EC 3.4.99.25 all created 1972 and incorporated 1978], part incorporated 1992)' WHERE ec = 'EC 3.4.23.30';
UPDATE enzyme SET history = 'EC 3.4.23.31 created 1992' WHERE ec = 'EC 3.4.23.31';
UPDATE enzyme SET history = 'EC 3.4.23.32 created 1992' WHERE ec = 'EC 3.4.23.32';
UPDATE enzyme SET history = 'EC 3.4.23.33 created 1992, deleted 2001' WHERE ec = 'EC 3.4.23.33';
UPDATE enzyme SET history = 'EC 3.4.23.34 created 1992' WHERE ec = 'EC 3.4.23.34';
UPDATE enzyme SET history = 'EC 3.4.23.35 created 1993' WHERE ec = 'EC 3.4.23.35';
UPDATE enzyme SET history = 'EC 3.4.23.36 created 1984 as EC 3.4.99.35, transferred 1995 to EC 3.4.23.36' WHERE ec = 'EC 3.4.23.36';
UPDATE enzyme SET history = 'EC 3.4.23.37 created 1995, deleted 2001' WHERE ec = 'EC 3.4.23.37';
UPDATE enzyme SET history = 'EC 3.4.23.38 created 1995' WHERE ec = 'EC 3.4.23.38';
UPDATE enzyme SET history = 'EC 3.4.23.39 created 1995' WHERE ec = 'EC 3.4.23.39';
UPDATE enzyme SET history = 'EC 3.4.23.40 created 1997' WHERE ec = 'EC 3.4.23.40';
UPDATE enzyme SET history = 'EC 3.4.23.41 created 2000' WHERE ec = 'EC 3.4.23.41';
UPDATE enzyme SET history = 'EC 3.4.23.42 created 1992 as EC 3.4.99.43, transferred 2000 to EC 3.4.23.42' WHERE ec = 'EC 3.4.23.42';
UPDATE enzyme SET history = 'EC 3.4.23.43 created 2001' WHERE ec = 'EC 3.4.23.43';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.44';
UPDATE enzyme SET history = 'EC 3.4.23.44 created 2001' WHERE ec = 'EC 3.4.23.44';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.45';
UPDATE enzyme SET history = 'EC 3.4.23.45 created 2003' WHERE ec = 'EC 3.4.23.45';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.46';
UPDATE enzyme SET history = 'EC 3.4.23.46 created 2003' WHERE ec = 'EC 3.4.23.46';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.47';
UPDATE enzyme SET history = 'EC 3.4.23.47 created 2003' WHERE ec = 'EC 3.4.23.47';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.48';
UPDATE enzyme SET history = 'EC 3.4.23.48 created 2003' WHERE ec = 'EC 3.4.23.48';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.49';
UPDATE enzyme SET history = 'EC 3.4.23.49 created 1993 as EC 3.4.21.87, transferred 2006 to EC 3.4.23.49' WHERE ec = 'EC 3.4.23.49';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.50';
UPDATE enzyme SET history = 'EC 3.4.23.50' WHERE ec = 'EC 3.4.23.50';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.51';
UPDATE enzyme SET history = 'EC 3.4.23.51 created 2009' WHERE ec = 'EC 3.4.23.51';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.23.52';
UPDATE enzyme SET history = 'EC 3.4.23.52 created 2011' WHERE ec = 'EC 3.4.23.52';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.1';
UPDATE enzyme SET history = 'EC 3.4.24.1 created 1972, modified 1986' WHERE ec = 'EC 3.4.24.1';
UPDATE enzyme SET history = 'EC 3.4.24.2 created 1972, deleted 1992' WHERE ec = 'EC 3.4.24.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.3';
UPDATE enzyme SET history = 'EC 3.4.24.3 created 1961 as EC 3.4.4.19, transferred 1972 to EC 3.4.24.3 (EC 3.4.24.8 created 1978, incorporated 1992, EC 3.4.99.5 created 1972, incorporated 1978)' WHERE ec = 'EC 3.4.24.3';
UPDATE enzyme SET history = 'EC 3.4.24.4 created 1972 (EC 3.4.99.13 and EC 3.4.99.22 both created 1972, incorporated 1978), deleted 1992' WHERE ec = 'EC 3.4.24.4';
UPDATE enzyme SET history = 'EC 3.4.24.5 created 1978, deleted 1989' WHERE ec = 'EC 3.4.24.5';
UPDATE enzyme SET history = 'EC 3.4.24.6 created 1978' WHERE ec = 'EC 3.4.24.6';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.7';
UPDATE enzyme SET history = 'EC 3.4.24.7 created 1978' WHERE ec = 'EC 3.4.24.7';
UPDATE enzyme SET history = 'EC 3.4.24.8 created 1978, deleted 1992' WHERE ec = 'EC 3.4.24.8';
UPDATE enzyme SET history = 'EC 3.4.24.9 created 1978, deleted 1992' WHERE ec = 'EC 3.4.24.9';
UPDATE enzyme SET history = 'EC 3.4.24.10 created 1972 as EC 3.4.99.12, transferred 1978 to EC 3.4.24.10, deleted 1992' WHERE ec = 'EC 3.4.24.10';
UPDATE enzyme SET history = 'EC 3.4.24.11 created 1978, modified 1989' WHERE ec = 'EC 3.4.24.11';
UPDATE enzyme SET history = 'EC 3.4.24.12 created 1978' WHERE ec = 'EC 3.4.24.12';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.13';
UPDATE enzyme SET history = 'EC 3.4.24.13 created 1984' WHERE ec = 'EC 3.4.24.13';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.14';
UPDATE enzyme SET history = 'EC 3.4.24.14 created 1984' WHERE ec = 'EC 3.4.24.14';
UPDATE enzyme SET history = 'EC 3.4.24.15 created 1984 (EC 3.4.22.19 created 1989 and EC 3.4.99.31 created 1978 both incorporated 1992)' WHERE ec = 'EC 3.4.24.15';
UPDATE enzyme SET history = 'EC 3.4.24.16 created 1989' WHERE ec = 'EC 3.4.24.16';
UPDATE enzyme SET history = 'EC 3.4.24.17 created 1990' WHERE ec = 'EC 3.4.24.17';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.18';
UPDATE enzyme SET history = 'EC 3.4.24.18 created 1992' WHERE ec = 'EC 3.4.24.18';
UPDATE enzyme SET history = 'EC 3.4.24.19 created 1992' WHERE ec = 'EC 3.4.24.19';
UPDATE enzyme SET history = 'EC 3.4.24.20 created 1978 as EC 3.4.99.32, transferred 1992 to EC 3.4.24.20 (EC 3.4.99.30 created 1978, incorporated 1992)' WHERE ec = 'EC 3.4.24.20';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.21';
UPDATE enzyme SET history = 'EC 3.4.24.21 created 1972 as EC 3.4.99.6, transferred 1992 to EC 3.4.24.21' WHERE ec = 'EC 3.4.24.21';
UPDATE enzyme SET history = 'EC 3.4.24.22 created 1992' WHERE ec = 'EC 3.4.24.22';
UPDATE enzyme SET history = 'EC 3.4.24.23 created 1992' WHERE ec = 'EC 3.4.24.23';
UPDATE enzyme SET history = 'EC 3.4.24.24 created 1992' WHERE ec = 'EC 3.4.24.24';
UPDATE enzyme SET history = 'EC 3.4.24.25 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.25, modified 1997' WHERE ec = 'EC 3.4.24.25';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.26';
UPDATE enzyme SET history = 'EC 3.4.24.26 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.26' WHERE ec = 'EC 3.4.24.26';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.27';
UPDATE enzyme SET history = 'EC 3.4.24.27 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.27' WHERE ec = 'EC 3.4.24.27';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.28';
UPDATE enzyme SET history = 'EC 3.4.24.28 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.28' WHERE ec = 'EC 3.4.24.28';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.29';
UPDATE enzyme SET history = 'EC 3.4.24.29 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.29' WHERE ec = 'EC 3.4.24.29';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.30';
UPDATE enzyme SET history = 'EC 3.4.24.30 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.30' WHERE ec = 'EC 3.4.24.30';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.31';
UPDATE enzyme SET history = 'EC 3.4.24.31 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.31' WHERE ec = 'EC 3.4.24.31';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.32';
UPDATE enzyme SET history = 'EC 3.4.24.32 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.32' WHERE ec = 'EC 3.4.24.32';
UPDATE enzyme SET history = 'EC 3.4.24.33 created 1992' WHERE ec = 'EC 3.4.24.33';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.34';
UPDATE enzyme SET history = 'EC 3.4.24.34 created 1992' WHERE ec = 'EC 3.4.24.34';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.35';
UPDATE enzyme SET history = 'EC 3.4.24.35 created 1992' WHERE ec = 'EC 3.4.24.35';
UPDATE enzyme SET history = 'EC 3.4.24.36 created 1992' WHERE ec = 'EC 3.4.24.36';
UPDATE enzyme SET history = 'EC 3.4.24.37 created 1989 as EC 3.4.22.22, transferred 1992 to EC 3.4.24.37' WHERE ec = 'EC 3.4.24.37';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.38';
UPDATE enzyme SET history = 'EC 3.4.24.38 created 1992, modified 2000' WHERE ec = 'EC 3.4.24.38';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.39';
UPDATE enzyme SET history = 'EC 3.4.24.39 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.39' WHERE ec = 'EC 3.4.24.39';
UPDATE enzyme SET history = 'EC 3.4.24.40 created 1972 as EC 3.4.24.4, part transferred 1992 to EC 3.4.24.40' WHERE ec = 'EC 3.4.24.40';
UPDATE enzyme SET history = 'EC 3.4.24.41 created 1992' WHERE ec = 'EC 3.4.24.41';
UPDATE enzyme SET history = 'EC 3.4.24.42 created 1992' WHERE ec = 'EC 3.4.24.42';
UPDATE enzyme SET history = 'EC 3.4.24.43 created 1992' WHERE ec = 'EC 3.4.24.43';
UPDATE enzyme SET history = 'EC 3.4.24.44 created 1992' WHERE ec = 'EC 3.4.24.44';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.45';
UPDATE enzyme SET history = 'EC 3.4.24.45 created 1992' WHERE ec = 'EC 3.4.24.45';
UPDATE enzyme SET history = 'EC 3.4.24.46 created 1992' WHERE ec = 'EC 3.4.24.46';
UPDATE enzyme SET history = 'EC 3.4.24.47 created 1992' WHERE ec = 'EC 3.4.24.47';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.48';
UPDATE enzyme SET history = 'EC 3.4.24.48 created 1992' WHERE ec = 'EC 3.4.24.48';
UPDATE enzyme SET history = 'EC 3.4.24.49 created 1992' WHERE ec = 'EC 3.4.24.49';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.50';
UPDATE enzyme SET history = 'EC 3.4.24.50 created 1992' WHERE ec = 'EC 3.4.24.50';
UPDATE enzyme SET history = 'EC 3.4.24.51 created 1992' WHERE ec = 'EC 3.4.24.51';
UPDATE enzyme SET history = 'EC 3.4.24.52 created 1992' WHERE ec = 'EC 3.4.24.52';
UPDATE enzyme SET history = 'EC 3.4.24.53 created 1992' WHERE ec = 'EC 3.4.24.53';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.54';
UPDATE enzyme SET history = 'EC 3.4.24.54 created 1992' WHERE ec = 'EC 3.4.24.54';
UPDATE enzyme SET history = 'EC 3.4.24.55 created 1992 as EC 3.4.99.44, transferred 1993 to EC 3.4.24.55 (EC 3.4.99.45 created 1992, incorporated 1993)' WHERE ec = 'EC 3.4.24.55';
UPDATE enzyme SET history = 'EC 3.4.24.56 created 1972 as EC 3.4.99.10, transferred 1976 EC 3.4.22.11, transferred 1978 to EC 3.4.99.45, transferred 1993 to to EC 3.4.24.56 (EC 3.4.99.46 created 1992, incorporated 2000)' WHERE ec = 'EC 3.4.24.56';
UPDATE enzyme SET history = 'EC 3.4.24.57 created 1993' WHERE ec = 'EC 3.4.24.57';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.58';
UPDATE enzyme SET history = 'EC 3.4.24.58 created 1993' WHERE ec = 'EC 3.4.24.58';
UPDATE enzyme SET history = 'EC 3.4.24.59 created 1993' WHERE ec = 'EC 3.4.24.59';
UPDATE enzyme SET history = 'EC 3.4.24.60 created 1995' WHERE ec = 'EC 3.4.24.60';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.61';
UPDATE enzyme SET history = 'EC 3.4.24.61 created 1995' WHERE ec = 'EC 3.4.24.61';
UPDATE enzyme SET history = 'EC 3.4.24.62 created 1995' WHERE ec = 'EC 3.4.24.62';
UPDATE enzyme SET history = 'EC 3.4.24.63 created 1995' WHERE ec = 'EC 3.4.24.63';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.64';
UPDATE enzyme SET history = 'EC 3.4.24.64 created 1989/90 as EC 3.4.99.41, transferred 1995 to EC 3.4.24.64' WHERE ec = 'EC 3.4.24.64';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.65';
UPDATE enzyme SET history = 'EC 3.4.24.65 created 1995' WHERE ec = 'EC 3.4.24.65';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.66';
UPDATE enzyme SET history = 'EC 3.4.24.66 created 1995' WHERE ec = 'EC 3.4.24.66';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.67';
UPDATE enzyme SET history = 'EC 3.4.24.67 created 1995' WHERE ec = 'EC 3.4.24.67';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.68';
UPDATE enzyme SET history = 'EC 3.4.24.68 created 1995' WHERE ec = 'EC 3.4.24.68';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.69';
UPDATE enzyme SET history = 'EC 3.4.24.69 created 1995' WHERE ec = 'EC 3.4.24.69';
UPDATE enzyme SET history = 'EC 3.4.24.70 created 1996' WHERE ec = 'EC 3.4.24.70';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.71';
UPDATE enzyme SET history = 'EC 3.4.24.71 created 1996' WHERE ec = 'EC 3.4.24.71';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.72';
UPDATE enzyme SET history = 'EC 3.4.24.72 created 1996' WHERE ec = 'EC 3.4.24.72';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.73';
UPDATE enzyme SET history = 'EC 3.4.24.73 created 1996' WHERE ec = 'EC 3.4.24.73';
UPDATE enzyme SET history = 'EC 3.4.24.74 created 1997' WHERE ec = 'EC 3.4.24.74';
UPDATE enzyme SET history = 'EC 3.4.24.75 created 1997' WHERE ec = 'EC 3.4.24.75';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.76';
UPDATE enzyme SET history = 'EC 3.4.24.76 created 2000' WHERE ec = 'EC 3.4.24.76';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.77';
UPDATE enzyme SET history = 'EC 3.4.24.77 created 2001' WHERE ec = 'EC 3.4.24.77';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.78';
UPDATE enzyme SET history = 'EC 3.4.24.78 created 2003' WHERE ec = 'EC 3.4.24.78';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.79';
UPDATE enzyme SET history = 'EC 3.4.24.79 created 2003' WHERE ec = 'EC 3.4.24.79';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.80';
UPDATE enzyme SET history = 'EC 3.4.24.80 created 2003' WHERE ec = 'EC 3.4.24.80';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.81';
UPDATE enzyme SET history = 'EC 3.4.24.81 created 2003' WHERE ec = 'EC 3.4.24.81';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.82';
UPDATE enzyme SET history = 'EC 3.4.24.82 created 2003' WHERE ec = 'EC 3.4.24.82';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.83';
UPDATE enzyme SET history = 'EC 3.4.24.83 created 2003' WHERE ec = 'EC 3.4.24.83';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.84';
UPDATE enzyme SET history = 'EC 3.4.24.84 created 2003' WHERE ec = 'EC 3.4.24.84';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.85';
UPDATE enzyme SET history = 'EC 3.4.24.85 created 2003' WHERE ec = 'EC 3.4.24.85';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.86';
UPDATE enzyme SET history = 'EC 3.4.24.86 created 2003' WHERE ec = 'EC 3.4.24.86';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.24.87';
UPDATE enzyme SET history = 'EC 3.4.24.87 created 2009' WHERE ec = 'EC 3.4.24.87';
UPDATE enzyme SET history = 'EC 3.4.25.1 created 1978 as EC 3.4.24.5, part transferred 1989 to EC 3.4.22.21, transferred 1992 to EC 3.4.99.46, transferred 2000 to EC 3.4.25.1' WHERE ec = 'EC 3.4.25.1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.4.25.2';
UPDATE enzyme SET history = 'EC 3.4.25.2 created 2009, modified 2010' WHERE ec = 'EC 3.4.25.2';
UPDATE enzyme SET history = 'EC 3.4.99.1 created 1972, deleted 1978 (transferred to EC 3.4.23.6, deleted 1992)' WHERE ec = 'EC 3.4.99.1';
UPDATE enzyme SET history = 'EC 3.4.99.2 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.2';
UPDATE enzyme SET history = 'EC 3.4.99.3 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.3';
UPDATE enzyme SET history = 'EC 3.4.99.4 created 1972, deleted 1978' WHERE ec = 'EC 3.4.99.4';
UPDATE enzyme SET history = 'EC 3.4.99.5 created 1972, deleted 1978' WHERE ec = 'EC 3.4.99.5';
UPDATE enzyme SET history = 'EC 3.4.99.6 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.6';
UPDATE enzyme SET history = 'EC 3.4.99.7 created 1972, deleted 1989' WHERE ec = 'EC 3.4.99.7';
UPDATE enzyme SET history = 'EC 3.4.99.8 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.8';
UPDATE enzyme SET history = 'EC 3.4.99.9 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.9';
UPDATE enzyme SET history = 'EC 3.4.99.10 created 1972, transferred 1976 to EC 3.4.22.11, transferred 1978 to EC 3.4.99.45, transferred 1993 to EC 3.4.24.56' WHERE ec = 'EC 3.4.99.10';
UPDATE enzyme SET history = 'EC 3.4.99.11 created 1965 as EC 3.4.4.25, transferred 1972 to EC 3.4.99.11, deleted 1992' WHERE ec = 'EC 3.4.99.11';
UPDATE enzyme SET history = 'EC 3.4.99.12 created 1972, deleted 1978 (transferred to EC 3.4.24.10, deleted 1992)' WHERE ec = 'EC 3.4.99.12';
UPDATE enzyme SET history = 'EC 3.4.99.13 created 1972, deleted 1978 (transferred to EC 3.4.24.4, deleted 1992)' WHERE ec = 'EC 3.4.99.13';
UPDATE enzyme SET history = 'EC 3.4.99.14 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.14';
UPDATE enzyme SET history = 'EC 3.4.99.15 created 1972, deleted 1978 (transferred to EC 3.4.23.6, deleted 1992)' WHERE ec = 'EC 3.4.99.15';
UPDATE enzyme SET history = 'EC 3.4.99.16 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.16';
UPDATE enzyme SET history = 'EC 3.4.99.17 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.17';
UPDATE enzyme SET history = 'EC 3.4.99.18 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.18';
UPDATE enzyme SET history = 'EC 3.4.99.19 created 1972, deleted 1981' WHERE ec = 'EC 3.4.99.19';
UPDATE enzyme SET history = 'EC 3.4.99.20 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.20';
UPDATE enzyme SET history = 'EC 3.4.99.21 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.21';
UPDATE enzyme SET history = 'EC 3.4.99.22 created 1972, modified 1976, deleted 1978 (transferred to EC 3.4.24.4, deleted 1992)' WHERE ec = 'EC 3.4.99.22';
UPDATE enzyme SET history = 'EC 3.4.99.23 created 1972, deleted 1992' WHERE ec = 'EC 3.4.99.23';
UPDATE enzyme SET history = 'EC 3.4.99.24 created 1972, deleted 1978 (transferred to EC 3.4.21.18, deleted 1992)' WHERE ec = 'EC 3.4.99.24';
UPDATE enzyme SET history = 'EC 3.4.99.25 created 1972, deleted 1978 (transferred to EC 3.4.23.6, deleted 1992)' WHERE ec = 'EC 3.4.99.25';
UPDATE enzyme SET history = 'EC 3.4.99.26 created 1972, deleted 1978 (transferred to EC 3.4.21.31, deleted 1992)' WHERE ec = 'EC 3.4.99.26';
UPDATE enzyme SET history = 'EC 3.4.99.27 created 1978, deleted 1992' WHERE ec = 'EC 3.4.99.27';
UPDATE enzyme SET history = 'EC 3.4.99.28 created 1978, deleted 1992' WHERE ec = 'EC 3.4.99.28';
UPDATE enzyme SET history = 'EC 3.4.99.29 created 1978, deleted 1992' WHERE ec = 'EC 3.4.99.29';
UPDATE enzyme SET history = 'EC 3.4.99.30 created 1978, deleted 1992' WHERE ec = 'EC 3.4.99.30';
UPDATE enzyme SET history = 'EC 3.4.99.31 created 1978, deleted 1992' WHERE ec = 'EC 3.4.99.31';
UPDATE enzyme SET history = 'EC 3.4.99.32 created 1978, deleted 1992' WHERE ec = 'EC 3.4.99.32';
UPDATE enzyme SET history = 'EC 3.4.99.33 created 1981, deleted 1984 (transferred to EC 3.4.21.52, deleted 1992)' WHERE ec = 'EC 3.4.99.33';
UPDATE enzyme SET history = 'EC 3.4.99.34 created 1981, deleted 1992' WHERE ec = 'EC 3.4.99.34';
UPDATE enzyme SET history = 'EC 3.4.99.35 created 1984, deleted 1995' WHERE ec = 'EC 3.4.99.35';
UPDATE enzyme SET history = 'EC 3.4.99.36 created 1984, deleted 1995' WHERE ec = 'EC 3.4.99.36';
UPDATE enzyme SET history = 'EC 3.4.99.37 created 1989, deleted 1992' WHERE ec = 'EC 3.4.99.37';
UPDATE enzyme SET history = 'EC 3.4.99.38 created 1989, deleted 1992' WHERE ec = 'EC 3.4.99.38';
UPDATE enzyme SET history = 'EC 3.4.99.39 created 1989, deleted 1992' WHERE ec = 'EC 3.4.99.39';
UPDATE enzyme SET history = 'EC 3.4.99.40 created 1989, deleted 1992' WHERE ec = 'EC 3.4.99.40';
UPDATE enzyme SET history = 'EC 3.4.99.41 created 1989/90, deleted 1995' WHERE ec = 'EC 3.4.99.41';
UPDATE enzyme SET history = 'EC 3.4.99.42 created 1990, deleted 1992' WHERE ec = 'EC 3.4.99.42';
UPDATE enzyme SET history = 'EC 3.4.99.43 created 1992, deleted 2000' WHERE ec = 'EC 3.4.99.43';
UPDATE enzyme SET history = 'EC 3.4.99.44 created 1992, deleted 1993' WHERE ec = 'EC 3.4.99.44';
UPDATE enzyme SET history = 'EC 3.4.99.45 created 1992, deleted 1993' WHERE ec = 'EC 3.4.99.45';
UPDATE enzyme SET history = 'EC 3.4.99.46 created 1992, deleted 2000' WHERE ec = 'EC 3.4.99.46';
UPDATE enzyme SET sys_name = 'L-asparagine amidohydrolase' WHERE ec = 'EC 3.5.1.1';
UPDATE enzyme SET history = 'EC 3.5.1.1 created 1961' WHERE ec = 'EC 3.5.1.1';
UPDATE enzyme SET sys_name = 'L-glutamine amidohydrolase' WHERE ec = 'EC 3.5.1.2';
UPDATE enzyme SET history = 'EC 3.5.1.2 created 1961' WHERE ec = 'EC 3.5.1.2';
UPDATE enzyme SET sys_name = 'Omega-amidodicarboxylate amidohydrolase' WHERE ec = 'EC 3.5.1.3';
UPDATE enzyme SET history = 'EC 3.5.1.3 created 1961' WHERE ec = 'EC 3.5.1.3';
UPDATE enzyme SET sys_name = 'Acylamide amidohydrolase' WHERE ec = 'EC 3.5.1.4';
UPDATE enzyme SET history = 'EC 3.5.1.4 created 1961, modified 2011' WHERE ec = 'EC 3.5.1.4';
UPDATE enzyme SET sys_name = 'Urea amidohydrolase' WHERE ec = 'EC 3.5.1.5';
UPDATE enzyme SET history = 'EC 3.5.1.5 created 1961' WHERE ec = 'EC 3.5.1.5';
UPDATE enzyme SET sys_name = 'N-carbamoyl-beta-alanine amidohydrolase' WHERE ec = 'EC 3.5.1.6';
UPDATE enzyme SET history = 'EC 3.5.1.6 created 1961' WHERE ec = 'EC 3.5.1.6';
UPDATE enzyme SET sys_name = 'N-carbamoyl-L-aspartate amidohydrolase' WHERE ec = 'EC 3.5.1.7';
UPDATE enzyme SET history = 'EC 3.5.1.7 created 1961' WHERE ec = 'EC 3.5.1.7';
UPDATE enzyme SET sys_name = 'N-formyl-L-aspartate amidohydrolase' WHERE ec = 'EC 3.5.1.8';
UPDATE enzyme SET history = 'EC 3.5.1.8 created 1961' WHERE ec = 'EC 3.5.1.8';
UPDATE enzyme SET sys_name = 'Aryl-formylamine amidohydrolase' WHERE ec = 'EC 3.5.1.9';
UPDATE enzyme SET history = 'EC 3.5.1.9 created 1961' WHERE ec = 'EC 3.5.1.9';
UPDATE enzyme SET sys_name = '10-formyltetrahydrofolate amidohydrolase' WHERE ec = 'EC 3.5.1.10';
UPDATE enzyme SET history = 'EC 3.5.1.10 created 1961' WHERE ec = 'EC 3.5.1.10';
UPDATE enzyme SET sys_name = 'Penicillin amidohydrolase' WHERE ec = 'EC 3.5.1.11';
UPDATE enzyme SET history = 'EC 3.5.1.11 created 1961' WHERE ec = 'EC 3.5.1.11';
UPDATE enzyme SET sys_name = 'Biotin-amide amidohydrolase' WHERE ec = 'EC 3.5.1.12';
UPDATE enzyme SET history = 'EC 3.5.1.12 created 1961' WHERE ec = 'EC 3.5.1.12';
UPDATE enzyme SET sys_name = 'Aryl-acylamide amidohydrolase' WHERE ec = 'EC 3.5.1.13';
UPDATE enzyme SET history = 'EC 3.5.1.13 created 1965' WHERE ec = 'EC 3.5.1.13';
UPDATE enzyme SET sys_name = 'N-acyl-aliphatic-L-amino acid amidohydrolase (carboxylate-forming)' WHERE ec = 'EC 3.5.1.14';
UPDATE enzyme SET history = 'EC 3.5.1.14 created 1965, modified 2013' WHERE ec = 'EC 3.5.1.14';
UPDATE enzyme SET sys_name = 'N-acyl-L-aspartate amidohydrolase' WHERE ec = 'EC 3.5.1.15';
UPDATE enzyme SET history = 'EC 3.5.1.15 created 1965' WHERE ec = 'EC 3.5.1.15';
UPDATE enzyme SET sys_name = 'N(2)-acetyl-L-ornithine amidohydrolase' WHERE ec = 'EC 3.5.1.16';
UPDATE enzyme SET history = 'EC 3.5.1.16 created 1965' WHERE ec = 'EC 3.5.1.16';
UPDATE enzyme SET sys_name = 'N(6)-acyl-L-lysine amidohydrolase' WHERE ec = 'EC 3.5.1.17';
UPDATE enzyme SET history = 'EC 3.5.1.17 created 1965' WHERE ec = 'EC 3.5.1.17';
UPDATE enzyme SET sys_name = 'N-succinyl-LL-2,6-diaminoheptanedioate amidohydrolase' WHERE ec = 'EC 3.5.1.18';
UPDATE enzyme SET history = 'EC 3.5.1.18 created 1965' WHERE ec = 'EC 3.5.1.18';
UPDATE enzyme SET sys_name = 'Nicotinamide amidohydrolase' WHERE ec = 'EC 3.5.1.19';
UPDATE enzyme SET history = 'EC 3.5.1.19 created 1972' WHERE ec = 'EC 3.5.1.19';
UPDATE enzyme SET sys_name = 'L-citrulline N(5)-carbamoyldihydrolase' WHERE ec = 'EC 3.5.1.20';
UPDATE enzyme SET history = 'EC 3.5.1.20 created 1972' WHERE ec = 'EC 3.5.1.20';
UPDATE enzyme SET sys_name = 'N-acetyl-beta-alanine amidohydrolase' WHERE ec = 'EC 3.5.1.21';
UPDATE enzyme SET history = 'EC 3.5.1.21 created 1972' WHERE ec = 'EC 3.5.1.21';
UPDATE enzyme SET sys_name = '(R)-pantothenate amidohydrolase' WHERE ec = 'EC 3.5.1.22';
UPDATE enzyme SET history = 'EC 3.5.1.22 created 1972' WHERE ec = 'EC 3.5.1.22';
UPDATE enzyme SET sys_name = 'N-acylsphingosine amidohydrolase' WHERE ec = 'EC 3.5.1.23';
UPDATE enzyme SET history = 'EC 3.5.1.23 created 1972, modified 1990' WHERE ec = 'EC 3.5.1.23';
UPDATE enzyme SET sys_name = '3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholan-24-oylglycine amidohydrolase' WHERE ec = 'EC 3.5.1.24';
UPDATE enzyme SET history = 'EC 3.5.1.24 created 1972' WHERE ec = 'EC 3.5.1.24';
UPDATE enzyme SET sys_name = 'N-acetyl-D-glucosamine-6-phosphate amidohydrolase' WHERE ec = 'EC 3.5.1.25';
UPDATE enzyme SET history = 'EC 3.5.1.25 created 1972 (EC 3.5.1.80 created 1999, incorporated 2002)' WHERE ec = 'EC 3.5.1.25';
UPDATE enzyme SET sys_name = 'N(4)-(beta-N-acetyl-D-glucosaminyl)-L-asparagine amidohydrolase' WHERE ec = 'EC 3.5.1.26';
UPDATE enzyme SET history = 'EC 3.5.1.26 created 1972 (EC 3.5.1.34 created 1972, incorporated 1981)' WHERE ec = 'EC 3.5.1.26';
UPDATE enzyme SET sys_name = 'N-formyl-L-methionylaminoacyl-tRNA amidohydrolase' WHERE ec = 'EC 3.5.1.27';
UPDATE enzyme SET history = 'EC 3.5.1.27 created 1972' WHERE ec = 'EC 3.5.1.27';
UPDATE enzyme SET sys_name = 'Peptidoglycan amidohydrolase' WHERE ec = 'EC 3.5.1.28';
UPDATE enzyme SET history = 'EC 3.5.1.28 created 1972 (EC 3.4.19.10 created 1992, incorporated 1997)' WHERE ec = 'EC 3.5.1.28';
UPDATE enzyme SET sys_name = '2-(acetamidomethylene)succinate amidohydrolase (deaminating, decarboxylating)' WHERE ec = 'EC 3.5.1.29';
UPDATE enzyme SET history = 'EC 3.5.1.29 created 1972' WHERE ec = 'EC 3.5.1.29';
UPDATE enzyme SET sys_name = '5-aminopentanamide amidohydrolase' WHERE ec = 'EC 3.5.1.30';
UPDATE enzyme SET history = 'EC 3.5.1.30 created 1972, modified 1976' WHERE ec = 'EC 3.5.1.30';
UPDATE enzyme SET sys_name = 'N-formyl-L-methionine amidohydrolase' WHERE ec = 'EC 3.5.1.31';
UPDATE enzyme SET history = 'EC 3.5.1.31 created 1972' WHERE ec = 'EC 3.5.1.31';
UPDATE enzyme SET sys_name = 'N-benzoylamino-acid amidohydrolase' WHERE ec = 'EC 3.5.1.32';
UPDATE enzyme SET history = 'EC 3.5.1.32 created 1972' WHERE ec = 'EC 3.5.1.32';
UPDATE enzyme SET sys_name = 'N-acetyl-D-glucosamine amidohydrolase' WHERE ec = 'EC 3.5.1.33';
UPDATE enzyme SET history = 'EC 3.5.1.33 created 1972' WHERE ec = 'EC 3.5.1.33';
UPDATE enzyme SET history = 'EC 3.5.1.34 created 1972, deleted 1981' WHERE ec = 'EC 3.5.1.34';
UPDATE enzyme SET sys_name = 'D-glutamine amidohydrolase' WHERE ec = 'EC 3.5.1.35';
UPDATE enzyme SET history = 'EC 3.5.1.35 created 1972' WHERE ec = 'EC 3.5.1.35';
UPDATE enzyme SET sys_name = 'N-methyl-2-oxoglutaramate methylamidohydrolase' WHERE ec = 'EC 3.5.1.36';
UPDATE enzyme SET history = 'EC 3.5.1.36 created 1972' WHERE ec = 'EC 3.5.1.36';
UPDATE enzyme SET history = 'EC 3.5.1.37 created 1972, deleted 1976' WHERE ec = 'EC 3.5.1.37';
UPDATE enzyme SET sys_name = 'L-glutamine(L-asparagine) amidohydrolase' WHERE ec = 'EC 3.5.1.38';
UPDATE enzyme SET history = 'EC 3.5.1.38 created 1976' WHERE ec = 'EC 3.5.1.38';
UPDATE enzyme SET sys_name = 'N-methylhexanamide amidohydrolase' WHERE ec = 'EC 3.5.1.39';
UPDATE enzyme SET history = 'EC 3.5.1.39 created 1976' WHERE ec = 'EC 3.5.1.39';
UPDATE enzyme SET sys_name = 'Benzoylagmatine amidohydrolase' WHERE ec = 'EC 3.5.1.40';
UPDATE enzyme SET history = 'EC 3.5.1.40 created 1976' WHERE ec = 'EC 3.5.1.40';
UPDATE enzyme SET sys_name = 'Chitin amidohydrolase' WHERE ec = 'EC 3.5.1.41';
UPDATE enzyme SET history = 'EC 3.5.1.41 created 1976' WHERE ec = 'EC 3.5.1.41';
UPDATE enzyme SET sys_name = 'Nicotinamide-D-ribonucleotide amidohydrolase' WHERE ec = 'EC 3.5.1.42';
UPDATE enzyme SET history = 'EC 3.5.1.42 created 1976' WHERE ec = 'EC 3.5.1.42';
UPDATE enzyme SET sys_name = 'Peptidyl-L-glutamine amidohydrolase' WHERE ec = 'EC 3.5.1.43';
UPDATE enzyme SET history = 'EC 3.5.1.43 created 1976' WHERE ec = 'EC 3.5.1.43';
UPDATE enzyme SET sys_name = 'Protein-L-glutamine amidohydrolase' WHERE ec = 'EC 3.5.1.44';
UPDATE enzyme SET history = 'EC 3.5.1.44 created 1976, modified 1983' WHERE ec = 'EC 3.5.1.44';
UPDATE enzyme SET history = 'EC 3.5.1.45 created 1978, deleted 1986' WHERE ec = 'EC 3.5.1.45';
UPDATE enzyme SET sys_name = 'N-(6-aminohexanoyl)-6-aminohexanoate amidohydrolase' WHERE ec = 'EC 3.5.1.46';
UPDATE enzyme SET history = 'EC 3.5.1.46 created 1983' WHERE ec = 'EC 3.5.1.46';
UPDATE enzyme SET sys_name = 'N(6)-acetyl-LL-2,6-diaminoheptanedioate amidohydrolase' WHERE ec = 'EC 3.5.1.47';
UPDATE enzyme SET history = 'EC 3.5.1.47 created 1984 (EC 3.1.1.62 created 1989, incorporated 1992)' WHERE ec = 'EC 3.5.1.47';
UPDATE enzyme SET sys_name = 'N(8)-acetylspermidine amidohydrolase' WHERE ec = 'EC 3.5.1.48';
UPDATE enzyme SET history = 'EC 3.5.1.48 created 1984, modified 2005' WHERE ec = 'EC 3.5.1.48';
UPDATE enzyme SET sys_name = 'Formamide amidohydrolase' WHERE ec = 'EC 3.5.1.49';
UPDATE enzyme SET history = 'EC 3.5.1.49 created 1984' WHERE ec = 'EC 3.5.1.49';
UPDATE enzyme SET sys_name = 'Pentanamide amidohydrolase' WHERE ec = 'EC 3.5.1.50';
UPDATE enzyme SET history = 'EC 3.5.1.50 created 1984' WHERE ec = 'EC 3.5.1.50';
UPDATE enzyme SET sys_name = '4-acetamidobutanoyl-CoA amidohydrolase' WHERE ec = 'EC 3.5.1.51';
UPDATE enzyme SET history = 'EC 3.5.1.51 created 1984' WHERE ec = 'EC 3.5.1.51';
UPDATE enzyme SET sys_name = 'N-linked-glycopeptide-(N-acetyl-beta-D-glucosaminyl)-L-asparagine amidohydrolase' WHERE ec = 'EC 3.5.1.52';
UPDATE enzyme SET history = 'EC 3.5.1.52 created 1984, modified 1989 (EC 3.2.2.18 created 1984, incorporated 1989)' WHERE ec = 'EC 3.5.1.52';
UPDATE enzyme SET sys_name = 'N-carbamoylputrescine amidohydrolase' WHERE ec = 'EC 3.5.1.53';
UPDATE enzyme SET history = 'EC 3.5.1.53 created 1986' WHERE ec = 'EC 3.5.1.53';
UPDATE enzyme SET sys_name = 'Urea-1-carboxylate amidohydrolase' WHERE ec = 'EC 3.5.1.54';
UPDATE enzyme SET history = 'EC 3.5.1.54 created 1986, modified 2008' WHERE ec = 'EC 3.5.1.54';
UPDATE enzyme SET sys_name = 'N-long-chain-fatty-acyl-L-glutamate amidohydrolase' WHERE ec = 'EC 3.5.1.55';
UPDATE enzyme SET history = 'EC 3.5.1.55 created 1986' WHERE ec = 'EC 3.5.1.55';
UPDATE enzyme SET sys_name = 'N,N-dimethylformamide amidohydrolase' WHERE ec = 'EC 3.5.1.56';
UPDATE enzyme SET history = 'EC 3.5.1.56 created 1989' WHERE ec = 'EC 3.5.1.56';
UPDATE enzyme SET sys_name = 'L-tryptophanamide amidohydrolase' WHERE ec = 'EC 3.5.1.57';
UPDATE enzyme SET history = 'EC 3.5.1.57 created 1989' WHERE ec = 'EC 3.5.1.57';
UPDATE enzyme SET sys_name = 'N-benzyloxycarbonylglycine urethanehydrolase' WHERE ec = 'EC 3.5.1.58';
UPDATE enzyme SET history = 'EC 3.5.1.58 created 1989' WHERE ec = 'EC 3.5.1.58';
UPDATE enzyme SET sys_name = 'N-carbamoylsarcosine amidohydrolase' WHERE ec = 'EC 3.5.1.59';
UPDATE enzyme SET history = 'EC 3.5.1.59 created 1989' WHERE ec = 'EC 3.5.1.59';
UPDATE enzyme SET sys_name = 'N-(long-chain-acyl)ethanolamine amidohydrolase' WHERE ec = 'EC 3.5.1.60';
UPDATE enzyme SET history = 'EC 3.5.1.60 created 1989' WHERE ec = 'EC 3.5.1.60';
UPDATE enzyme SET sys_name = 'Mimosine amidohydrolase' WHERE ec = 'EC 3.5.1.61';
UPDATE enzyme SET history = 'EC 3.5.1.61 created 1989' WHERE ec = 'EC 3.5.1.61';
UPDATE enzyme SET sys_name = 'N-acetylputrescine acetylhydrolase' WHERE ec = 'EC 3.5.1.62';
UPDATE enzyme SET history = 'EC 3.5.1.62 created 1989' WHERE ec = 'EC 3.5.1.62';
UPDATE enzyme SET sys_name = '4-acetamidobutanoate amidohydrolase' WHERE ec = 'EC 3.5.1.63';
UPDATE enzyme SET history = 'EC 3.5.1.63 created 1989' WHERE ec = 'EC 3.5.1.63';
UPDATE enzyme SET sys_name = 'N(alpha)-benzyloxycarbonyl-L-leucine urethanehydrolase' WHERE ec = 'EC 3.5.1.64';
UPDATE enzyme SET history = 'EC 3.5.1.64 created 1989' WHERE ec = 'EC 3.5.1.64';
UPDATE enzyme SET sys_name = 'N(5)-ethyl-L-glutamine amidohydrolase' WHERE ec = 'EC 3.5.1.65';
UPDATE enzyme SET history = 'EC 3.5.1.65 created 1989' WHERE ec = 'EC 3.5.1.65';
UPDATE enzyme SET sys_name = '2-(hydroxymethyl)-3-(acetamidomethylene)succinate amidohydrolase (deaminating, decarboxylating)' WHERE ec = 'EC 3.5.1.66';
UPDATE enzyme SET history = 'EC 3.5.1.66 created 1989' WHERE ec = 'EC 3.5.1.66';
UPDATE enzyme SET sys_name = '4-methylene-L-glutamine amidohydrolase' WHERE ec = 'EC 3.5.1.67';
UPDATE enzyme SET history = 'EC 3.5.1.67 created 1989' WHERE ec = 'EC 3.5.1.67';
UPDATE enzyme SET sys_name = 'N-formyl-L-glutamate amidohydrolase' WHERE ec = 'EC 3.5.1.68';
UPDATE enzyme SET history = 'EC 3.5.1.68 created 1989' WHERE ec = 'EC 3.5.1.68';
UPDATE enzyme SET sys_name = 'Glycosphingolipid amidohydrolase' WHERE ec = 'EC 3.5.1.69';
UPDATE enzyme SET history = 'EC 3.5.1.69 created 1990' WHERE ec = 'EC 3.5.1.69';
UPDATE enzyme SET sys_name = 'Aculeacin-A amidohydrolase' WHERE ec = 'EC 3.5.1.70';
UPDATE enzyme SET history = 'EC 3.5.1.70 created 1992' WHERE ec = 'EC 3.5.1.70';
UPDATE enzyme SET sys_name = 'N-feruloylglycine amidohydrolase' WHERE ec = 'EC 3.5.1.71';
UPDATE enzyme SET history = 'EC 3.5.1.71 created 1992' WHERE ec = 'EC 3.5.1.71';
UPDATE enzyme SET sys_name = 'N-benzoyl-D-arginine-4-nitroanilide amidohydrolase' WHERE ec = 'EC 3.5.1.72';
UPDATE enzyme SET history = 'EC 3.5.1.72 created 1992' WHERE ec = 'EC 3.5.1.72';
UPDATE enzyme SET sys_name = 'L-carnitinamide amidohydrolase' WHERE ec = 'EC 3.5.1.73';
UPDATE enzyme SET history = 'EC 3.5.1.73 created 1992' WHERE ec = 'EC 3.5.1.73';
UPDATE enzyme SET sys_name = 'Chenodeoxycholoyltaurine amidohydrolase' WHERE ec = 'EC 3.5.1.74';
UPDATE enzyme SET history = 'EC 3.5.1.74 created 1992' WHERE ec = 'EC 3.5.1.74';
UPDATE enzyme SET sys_name = 'Urethane amidohydrolase (decarboxylating)' WHERE ec = 'EC 3.5.1.75';
UPDATE enzyme SET history = 'EC 3.5.1.75 created 1992' WHERE ec = 'EC 3.5.1.75';
UPDATE enzyme SET sys_name = 'N-acetylarylalkylamine amidohydrolase' WHERE ec = 'EC 3.5.1.76';
UPDATE enzyme SET history = 'EC 3.5.1.76 created 1999' WHERE ec = 'EC 3.5.1.76';
UPDATE enzyme SET sys_name = 'N-carbamoyl-D-amino-acid amidohydrolase' WHERE ec = 'EC 3.5.1.77';
UPDATE enzyme SET history = 'EC 3.5.1.77 created 1999, modified 2008' WHERE ec = 'EC 3.5.1.77';
UPDATE enzyme SET sys_name = 'Gamma-L-glutamyl-L-cysteinyl-glycine:spermidine amidase' WHERE ec = 'EC 3.5.1.78';
UPDATE enzyme SET history = 'EC 3.5.1.78 created 1999' WHERE ec = 'EC 3.5.1.78';
UPDATE enzyme SET sys_name = 'Phthalyl-amide amidohydrolase' WHERE ec = 'EC 3.5.1.79';
UPDATE enzyme SET history = 'EC 3.5.1.79 created 1999' WHERE ec = 'EC 3.5.1.79';
UPDATE enzyme SET history = 'EC 3.5.1.80 created 1999, deleted 2002' WHERE ec = 'EC 3.5.1.80';
UPDATE enzyme SET sys_name = 'N-acyl-D-amino acid amidohydrolase' WHERE ec = 'EC 3.5.1.81';
UPDATE enzyme SET history = 'EC 3.5.1.81 created 1999' WHERE ec = 'EC 3.5.1.81';
UPDATE enzyme SET sys_name = 'N-acyl-D-glutamate amidohydrolase' WHERE ec = 'EC 3.5.1.82';
UPDATE enzyme SET history = 'EC 3.5.1.82 created 1999' WHERE ec = 'EC 3.5.1.82';
UPDATE enzyme SET sys_name = 'N-acyl-D-aspartate amidohydrolase' WHERE ec = 'EC 3.5.1.83';
UPDATE enzyme SET history = 'EC 3.5.1.83 created 1999' WHERE ec = 'EC 3.5.1.83';
UPDATE enzyme SET sys_name = 'Biuret amidohydrolase' WHERE ec = 'EC 3.5.1.84';
UPDATE enzyme SET history = 'EC 3.5.1.84 created 2000, modified 2008' WHERE ec = 'EC 3.5.1.84';
UPDATE enzyme SET sys_name = '(S)-N-acetylphenylethylamine:H(2)O hydrolase' WHERE ec = 'EC 3.5.1.85';
UPDATE enzyme SET history = 'EC 3.5.1.85 created 2000, modified 2002' WHERE ec = 'EC 3.5.1.85';
UPDATE enzyme SET sys_name = 'Mandelamide hydrolase' WHERE ec = 'EC 3.5.1.86';
UPDATE enzyme SET history = 'EC 3.5.1.86 created 2000' WHERE ec = 'EC 3.5.1.86';
UPDATE enzyme SET sys_name = 'N-carbamoyl-L-amino-acid amidohydrolase' WHERE ec = 'EC 3.5.1.87';
UPDATE enzyme SET history = 'EC 3.5.1.87 created 2001, modified 2008' WHERE ec = 'EC 3.5.1.87';
UPDATE enzyme SET sys_name = 'Formyl-L-methionyl peptide amidohydrolase' WHERE ec = 'EC 3.5.1.88';
UPDATE enzyme SET history = 'EC 3.5.1.88 created 2001' WHERE ec = 'EC 3.5.1.88';
UPDATE enzyme SET sys_name = '6-(N-acetyl-alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol acetylhydrolase' WHERE ec = 'EC 3.5.1.89';
UPDATE enzyme SET history = 'EC 3.5.1.89 created 1992 as EC 3.1.1.69, transferred 2002 to EC 3.5.1.89, modified 2002' WHERE ec = 'EC 3.5.1.89';
UPDATE enzyme SET sys_name = 'Adenosylcobinamide amidohydrolase' WHERE ec = 'EC 3.5.1.90';
UPDATE enzyme SET history = 'EC 3.5.1.90 created 2004' WHERE ec = 'EC 3.5.1.90';
UPDATE enzyme SET sys_name = 'N-benzylformamide amidohydrolase' WHERE ec = 'EC 3.5.1.91';
UPDATE enzyme SET history = 'EC 3.5.1.91 created 2005' WHERE ec = 'EC 3.5.1.91';
UPDATE enzyme SET sys_name = '(R)-pantetheine amidohydrolase' WHERE ec = 'EC 3.5.1.92';
UPDATE enzyme SET history = 'EC 3.5.1.92 created 2005' WHERE ec = 'EC 3.5.1.92';
UPDATE enzyme SET sys_name = '(7R)-7-(4-carboxybutanamido)cephalosporanate amidohydrolase' WHERE ec = 'EC 3.5.1.93';
UPDATE enzyme SET history = 'EC 3.5.1.93 created 2005' WHERE ec = 'EC 3.5.1.93';
UPDATE enzyme SET sys_name = '4-(L-gamma-glutamylamino)butanoate amidohydrolase' WHERE ec = 'EC 3.5.1.94';
UPDATE enzyme SET history = 'EC 3.5.1.94 created 2006, modified 2011' WHERE ec = 'EC 3.5.1.94';
UPDATE enzyme SET sys_name = '3-oxo-3-ureidopropanoate amidohydrolase (urea- and malonate-forming)' WHERE ec = 'EC 3.5.1.95';
UPDATE enzyme SET history = 'EC 3.5.1.95 created 2006' WHERE ec = 'EC 3.5.1.95';
UPDATE enzyme SET sys_name = 'N-succinyl-L-glutamate amidohydrolase' WHERE ec = 'EC 3.5.1.96';
UPDATE enzyme SET history = 'EC 3.5.1.96 created 2006' WHERE ec = 'EC 3.5.1.96';
UPDATE enzyme SET sys_name = 'N-acyl-L-homoserine-lactone amidohydrolase' WHERE ec = 'EC 3.5.1.97';
UPDATE enzyme SET history = 'EC 3.5.1.97 created 2007' WHERE ec = 'EC 3.5.1.97';
UPDATE enzyme SET sys_name = 'Histone amidohydrolase' WHERE ec = 'EC 3.5.1.98';
UPDATE enzyme SET history = 'EC 3.5.1.98 created 2008' WHERE ec = 'EC 3.5.1.98';
UPDATE enzyme SET sys_name = 'Fatty acylamide amidohydrolase' WHERE ec = 'EC 3.5.1.99';
UPDATE enzyme SET history = 'EC 3.5.1.99 created 2009' WHERE ec = 'EC 3.5.1.99';
UPDATE enzyme SET sys_name = '(R)-piperazine-2-carboxamide amidohydrolase' WHERE ec = 'EC 3.5.1.100';
UPDATE enzyme SET history = 'EC 3.5.1.100 created 2009, modified 2010' WHERE ec = 'EC 3.5.1.100';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.5.1.101';
UPDATE enzyme SET history = 'EC 3.5.1.101 created 2009' WHERE ec = 'EC 3.5.1.101';
UPDATE enzyme SET sys_name = '2-amino-5-formylamino-6-(5-phospho-D-ribosylamino)pyrimidin-4(3H)-one amidohydrolase' WHERE ec = 'EC 3.5.1.102';
UPDATE enzyme SET history = 'EC 3.5.1.102 created 2010, modified 2010' WHERE ec = 'EC 3.5.1.102';
UPDATE enzyme SET sys_name = '1-(2-acetamido-2-deoxy-alpha-D-glucopyranosyl)-1D-myo-inositol acetylhydrolase' WHERE ec = 'EC 3.5.1.103';
UPDATE enzyme SET history = 'EC 3.5.1.103 created 2010' WHERE ec = 'EC 3.5.1.103';
UPDATE enzyme SET sys_name = 'Peptidoglycan-N-acetylglucosamine amidohydrolase' WHERE ec = 'EC 3.5.1.104';
UPDATE enzyme SET history = 'EC 3.5.1.104 created 2010' WHERE ec = 'EC 3.5.1.104';
UPDATE enzyme SET sys_name = '2-(acetylamino)-4-O-(2-(acetylamino)-2-deoxy-beta-D-glucopyranosyl)-2-deoxy-D-glucopyranose acetylhydrolase' WHERE ec = 'EC 3.5.1.105';
UPDATE enzyme SET history = 'EC 3.5.1.105 created 2010' WHERE ec = 'EC 3.5.1.105';
UPDATE enzyme SET sys_name = 'N-formylmaleamic acid amidohydrolase' WHERE ec = 'EC 3.5.1.106';
UPDATE enzyme SET history = 'EC 3.5.1.106 created 2010' WHERE ec = 'EC 3.5.1.106';
UPDATE enzyme SET sys_name = 'Maleamate amidohydrolase' WHERE ec = 'EC 3.5.1.107';
UPDATE enzyme SET history = 'EC 3.5.1.107 created 2010' WHERE ec = 'EC 3.5.1.107';
UPDATE enzyme SET sys_name = 'UDP-3-O-((3R)-3-hydroxymyristoyl)-N-acetyl-alpha-D-glucosamine amidohydrolase' WHERE ec = 'EC 3.5.1.108';
UPDATE enzyme SET history = 'EC 3.5.1.108 created 2010' WHERE ec = 'EC 3.5.1.108';
UPDATE enzyme SET sys_name = 'N-acyl-sphingosylphosphorylcholine amidohydrolase' WHERE ec = 'EC 3.5.1.109';
UPDATE enzyme SET history = 'EC 3.5.1.109 created 2011' WHERE ec = 'EC 3.5.1.109';
UPDATE enzyme SET sys_name = '(Z)-3-ureidoacrylate peracid amidohydrolase' WHERE ec = 'EC 3.5.1.110';
UPDATE enzyme SET history = 'EC 3.5.1.110 created 2012' WHERE ec = 'EC 3.5.1.110';
UPDATE enzyme SET sys_name = '5-amino-2,5-dioxopentanoate amidohydrolase' WHERE ec = 'EC 3.5.1.111';
UPDATE enzyme SET history = 'EC 3.5.1.111 created 2012' WHERE ec = 'EC 3.5.1.111';
UPDATE enzyme SET sys_name = '2-N-acetylparomamine hydrolase (acetate-forming)' WHERE ec = 'EC 3.5.1.112';
UPDATE enzyme SET history = 'EC 3.5.1.112 created 2012' WHERE ec = 'EC 3.5.1.112';
UPDATE enzyme SET sys_name = '2-acetyl-6-hydroxyneomycin C hydrolase (acetate-forming)' WHERE ec = 'EC 3.5.1.113';
UPDATE enzyme SET history = 'EC 3.5.1.113 created 2012' WHERE ec = 'EC 3.5.1.113';
UPDATE enzyme SET sys_name = 'N-acyl-aromatic-L-amino acid amidohydrolase (carboxylate-forming)' WHERE ec = 'EC 3.5.1.114';
UPDATE enzyme SET history = 'EC 3.5.1.114 created 2013' WHERE ec = 'EC 3.5.1.114';
UPDATE enzyme SET sys_name = 'Mycothiol S-conjugate 1D-myo-inositol 2-amino-2-deoxy-alpha-D-glucopyranosyl-hydrolase' WHERE ec = 'EC 3.5.1.115';
UPDATE enzyme SET history = 'EC 3.5.1.115 created 2013' WHERE ec = 'EC 3.5.1.115';
UPDATE enzyme SET sys_name = '(S)-ureidoglycolate amidohydrolase (decarboxylating)' WHERE ec = 'EC 3.5.1.116';
UPDATE enzyme SET history = 'EC 3.5.1.116 created 1992 as EC 3.5.3.19, transferred 2014 to EC 3.5.1.116' WHERE ec = 'EC 3.5.1.116';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.5.1.n1';
UPDATE enzyme SET sys_name = '4-deoxy-4-formamido-L-arabinose-phosphoundecaprenol amidohydrolase' WHERE ec = 'EC 3.5.1.n3';
UPDATE enzyme SET history = 'EC 3.5.1.n3 created 2010' WHERE ec = 'EC 3.5.1.n3';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.5.1.n4';
UPDATE enzyme SET sys_name = 'Barbiturate amidohydrolase (3-oxo-3-ureidopropanoate-forming)' WHERE ec = 'EC 3.5.2.1';
UPDATE enzyme SET history = 'EC 3.5.2.1 created 1961, modified 2006' WHERE ec = 'EC 3.5.2.1';
UPDATE enzyme SET sys_name = '5,6-dihydropyrimidine amidohydrolase' WHERE ec = 'EC 3.5.2.2';
UPDATE enzyme SET history = 'EC 3.5.2.2 created 1961' WHERE ec = 'EC 3.5.2.2';
UPDATE enzyme SET sys_name = '(S)-dihydroorotate amidohydrolase' WHERE ec = 'EC 3.5.2.3';
UPDATE enzyme SET history = 'EC 3.5.2.3 created 1961' WHERE ec = 'EC 3.5.2.3';
UPDATE enzyme SET sys_name = 'L-5-carboxymethylhydantoin amidohydrolase' WHERE ec = 'EC 3.5.2.4';
UPDATE enzyme SET history = 'EC 3.5.2.4 created 1961' WHERE ec = 'EC 3.5.2.4';
UPDATE enzyme SET sys_name = '(S)-allantoin amidohydrolase' WHERE ec = 'EC 3.5.2.5';
UPDATE enzyme SET history = 'EC 3.5.2.5 created 1961' WHERE ec = 'EC 3.5.2.5';
UPDATE enzyme SET sys_name = 'Beta-lactam hydrolase' WHERE ec = 'EC 3.5.2.6';
UPDATE enzyme SET history = 'EC 3.5.2.6 created 1961, modified 1981 (EC 3.5.2.8 created 1972, incorporated 1978)' WHERE ec = 'EC 3.5.2.6';
UPDATE enzyme SET sys_name = '3-(5-oxo-4,5-dihydro-3H-imidazol-4-yl)propanoate amidohydrolase' WHERE ec = 'EC 3.5.2.7';
UPDATE enzyme SET history = 'EC 3.5.2.7 created 1965, modified 2001' WHERE ec = 'EC 3.5.2.7';
UPDATE enzyme SET history = 'EC 3.5.2.8 created 1972, deleted 1978' WHERE ec = 'EC 3.5.2.8';
UPDATE enzyme SET sys_name = '5-oxo-L-proline amidohydrolase (ATP-hydrolyzing)' WHERE ec = 'EC 3.5.2.9';
UPDATE enzyme SET history = 'EC 3.5.2.9 created 1976' WHERE ec = 'EC 3.5.2.9';
UPDATE enzyme SET sys_name = 'Creatinine amidohydrolase' WHERE ec = 'EC 3.5.2.10';
UPDATE enzyme SET history = 'EC 3.5.2.10 created 1978' WHERE ec = 'EC 3.5.2.10';
UPDATE enzyme SET sys_name = '(S)-2-aminohexano-6-lactam lactamhydrolase' WHERE ec = 'EC 3.5.2.11';
UPDATE enzyme SET history = 'EC 3.5.2.11 created 1981, modified 1989' WHERE ec = 'EC 3.5.2.11';
UPDATE enzyme SET sys_name = '1,8-diazacyclotetradecane-2,9-dione lactamhydrolase' WHERE ec = 'EC 3.5.2.12';
UPDATE enzyme SET history = 'EC 3.5.2.12 created 1983' WHERE ec = 'EC 3.5.2.12';
UPDATE enzyme SET sys_name = '2,5-dioxopiperazine amidohydrolase' WHERE ec = 'EC 3.5.2.13';
UPDATE enzyme SET history = 'EC 3.5.2.13 created 1989' WHERE ec = 'EC 3.5.2.13';
UPDATE enzyme SET sys_name = 'N-methylimidazolidine-2,4-dione amidohydrolase (ATP-hydrolyzing)' WHERE ec = 'EC 3.5.2.14';
UPDATE enzyme SET history = 'EC 3.5.2.14 created 1989' WHERE ec = 'EC 3.5.2.14';
UPDATE enzyme SET sys_name = 'Cyanuric acid amidohydrolase' WHERE ec = 'EC 3.5.2.15';
UPDATE enzyme SET history = 'EC 3.5.2.15 created 2000, modified 2008' WHERE ec = 'EC 3.5.2.15';
UPDATE enzyme SET sys_name = 'Cyclic-imide amidohydrolase (decyclizing)' WHERE ec = 'EC 3.5.2.16';
UPDATE enzyme SET history = 'EC 3.5.2.16 created 2001' WHERE ec = 'EC 3.5.2.16';
UPDATE enzyme SET sys_name = '5-hydroxyisourate amidohydrolase' WHERE ec = 'EC 3.5.2.17';
UPDATE enzyme SET history = 'EC 3.5.2.17 created 2004' WHERE ec = 'EC 3.5.2.17';
UPDATE enzyme SET sys_name = '6-oxo-1,4,5,6-tetrahydronicotinate amidohydrolase' WHERE ec = 'EC 3.5.2.18';
UPDATE enzyme SET history = 'EC 3.5.2.18 created 2006' WHERE ec = 'EC 3.5.2.18';
UPDATE enzyme SET sys_name = 'Streptothricin-F hydrolase' WHERE ec = 'EC 3.5.2.19';
UPDATE enzyme SET history = 'EC 3.5.2.19 created 2010' WHERE ec = 'EC 3.5.2.19';
UPDATE enzyme SET sys_name = 'L-arginine amidinohydrolase' WHERE ec = 'EC 3.5.3.1';
UPDATE enzyme SET history = 'EC 3.5.3.1 created 1961' WHERE ec = 'EC 3.5.3.1';
UPDATE enzyme SET sys_name = 'Guanidinoacetate amidinohydrolase' WHERE ec = 'EC 3.5.3.2';
UPDATE enzyme SET history = 'EC 3.5.3.2 created 1961' WHERE ec = 'EC 3.5.3.2';
UPDATE enzyme SET sys_name = 'Creatine amidinohydrolase' WHERE ec = 'EC 3.5.3.3';
UPDATE enzyme SET history = 'EC 3.5.3.3 created 1961' WHERE ec = 'EC 3.5.3.3';
UPDATE enzyme SET sys_name = 'Allantoate amidinohydrolase' WHERE ec = 'EC 3.5.3.4';
UPDATE enzyme SET history = 'EC 3.5.3.4 created 1961' WHERE ec = 'EC 3.5.3.4';
UPDATE enzyme SET sys_name = 'N-formimidoyl-L-aspartate iminohydrolase' WHERE ec = 'EC 3.5.3.5';
UPDATE enzyme SET history = 'EC 3.5.3.5 created 1961, modified 2000' WHERE ec = 'EC 3.5.3.5';
UPDATE enzyme SET sys_name = 'L-arginine iminohydrolase' WHERE ec = 'EC 3.5.3.6';
UPDATE enzyme SET history = 'EC 3.5.3.6 created 1961' WHERE ec = 'EC 3.5.3.6';
UPDATE enzyme SET sys_name = '4-guanidinobutanoate amidinohydrolase' WHERE ec = 'EC 3.5.3.7';
UPDATE enzyme SET history = 'EC 3.5.3.7 created 1972' WHERE ec = 'EC 3.5.3.7';
UPDATE enzyme SET sys_name = 'N-formimidoyl-L-glutamate formimidoylhydrolase' WHERE ec = 'EC 3.5.3.8';
UPDATE enzyme SET history = 'EC 3.5.3.8 created 1972, modified 2000, modified 2001' WHERE ec = 'EC 3.5.3.8';
UPDATE enzyme SET sys_name = 'Allantoate amidinohydrolase (decarboxylating)' WHERE ec = 'EC 3.5.3.9';
UPDATE enzyme SET history = 'EC 3.5.3.9 created 1972, modified 2010' WHERE ec = 'EC 3.5.3.9';
UPDATE enzyme SET sys_name = 'D-arginine amidinohydrolase' WHERE ec = 'EC 3.5.3.10';
UPDATE enzyme SET history = 'EC 3.5.3.10 created 1972' WHERE ec = 'EC 3.5.3.10';
UPDATE enzyme SET sys_name = 'Agmatine amidinohydrolase' WHERE ec = 'EC 3.5.3.11';
UPDATE enzyme SET history = 'EC 3.5.3.11 created 1972' WHERE ec = 'EC 3.5.3.11';
UPDATE enzyme SET sys_name = 'Agmatine iminohydrolase' WHERE ec = 'EC 3.5.3.12';
UPDATE enzyme SET history = 'EC 3.5.3.12 created 1972' WHERE ec = 'EC 3.5.3.12';
UPDATE enzyme SET sys_name = 'N-formimidoyl-L-glutamate iminohydrolase' WHERE ec = 'EC 3.5.3.13';
UPDATE enzyme SET history = 'EC 3.5.3.13 created 1975, modified 2000' WHERE ec = 'EC 3.5.3.13';
UPDATE enzyme SET sys_name = 'N-amidino-L-aspartate amidinohydrolase' WHERE ec = 'EC 3.5.3.14';
UPDATE enzyme SET history = 'EC 3.5.3.14 created 1976' WHERE ec = 'EC 3.5.3.14';
UPDATE enzyme SET sys_name = 'Protein-L-arginine iminohydrolase' WHERE ec = 'EC 3.5.3.15';
UPDATE enzyme SET history = 'EC 3.5.3.15 created 1983' WHERE ec = 'EC 3.5.3.15';
UPDATE enzyme SET sys_name = 'Methylguanidine amidinohydrolase' WHERE ec = 'EC 3.5.3.16';
UPDATE enzyme SET history = 'EC 3.5.3.16 created 1984' WHERE ec = 'EC 3.5.3.16';
UPDATE enzyme SET sys_name = '3-guanidinopropanoate amidinopropionase' WHERE ec = 'EC 3.5.3.17';
UPDATE enzyme SET history = 'EC 3.5.3.17 created 1989' WHERE ec = 'EC 3.5.3.17';
UPDATE enzyme SET sys_name = 'N(omega),N(omega)-dimethyl-L-arginine dimethylamidohydrolase' WHERE ec = 'EC 3.5.3.18';
UPDATE enzyme SET history = 'EC 3.5.3.18 created 1992' WHERE ec = 'EC 3.5.3.18';
UPDATE enzyme SET history = 'EC 3.5.3.19 created 1992, deleted 2014' WHERE ec = 'EC 3.5.3.19';
UPDATE enzyme SET sys_name = '1,4-diguanidinobutane amidinohydrolase' WHERE ec = 'EC 3.5.3.20';
UPDATE enzyme SET history = 'EC 3.5.3.20 created 1992' WHERE ec = 'EC 3.5.3.20';
UPDATE enzyme SET sys_name = 'Methylenediurea aminohydrolase' WHERE ec = 'EC 3.5.3.21';
UPDATE enzyme SET history = 'EC 3.5.3.21 created 1999' WHERE ec = 'EC 3.5.3.21';
UPDATE enzyme SET sys_name = 'Amidinoproclavaminate amidinohydrolase' WHERE ec = 'EC 3.5.3.22';
UPDATE enzyme SET history = 'EC 3.5.3.22 created 2003' WHERE ec = 'EC 3.5.3.22';
UPDATE enzyme SET sys_name = 'N(2)-succinyl-L-arginine iminohydrolase (decarboxylating)' WHERE ec = 'EC 3.5.3.23';
UPDATE enzyme SET history = 'EC 3.5.3.23 created 2006' WHERE ec = 'EC 3.5.3.23';
UPDATE enzyme SET sys_name = 'N(1)-aminopropylagmatine amidinohydrolase' WHERE ec = 'EC 3.5.3.24';
UPDATE enzyme SET history = 'EC 3.5.3.24 created 2013' WHERE ec = 'EC 3.5.3.24';
UPDATE enzyme SET sys_name = 'N(omega)-hydroxy-L-arginine amidinohydrolase' WHERE ec = 'EC 3.5.3.25';
UPDATE enzyme SET history = 'EC 3.5.3.25 created 2013' WHERE ec = 'EC 3.5.3.25';
UPDATE enzyme SET sys_name = '(S)-ureidoglycine aminohydrolase' WHERE ec = 'EC 3.5.3.26';
UPDATE enzyme SET history = 'EC 3.5.3.26 created 2013' WHERE ec = 'EC 3.5.3.26';
UPDATE enzyme SET sys_name = 'Cytosine aminohydrolase' WHERE ec = 'EC 3.5.4.1';
UPDATE enzyme SET history = 'EC 3.5.4.1 created 1961' WHERE ec = 'EC 3.5.4.1';
UPDATE enzyme SET sys_name = 'Adenine aminohydrolase' WHERE ec = 'EC 3.5.4.2';
UPDATE enzyme SET history = 'EC 3.5.4.2 created 1961' WHERE ec = 'EC 3.5.4.2';
UPDATE enzyme SET sys_name = 'Guanine aminohydrolase' WHERE ec = 'EC 3.5.4.3';
UPDATE enzyme SET history = 'EC 3.5.4.3 created 1961' WHERE ec = 'EC 3.5.4.3';
UPDATE enzyme SET sys_name = 'Adenosine aminohydrolase' WHERE ec = 'EC 3.5.4.4';
UPDATE enzyme SET history = 'EC 3.5.4.4 created 1961' WHERE ec = 'EC 3.5.4.4';
UPDATE enzyme SET sys_name = 'Cytidine/2-deoxycytidine aminohydrolase' WHERE ec = 'EC 3.5.4.5';
UPDATE enzyme SET history = 'EC 3.5.4.5 created 1961, modified 2013' WHERE ec = 'EC 3.5.4.5';
UPDATE enzyme SET sys_name = 'AMP aminohydrolase' WHERE ec = 'EC 3.5.4.6';
UPDATE enzyme SET history = 'EC 3.5.4.6 created 1961' WHERE ec = 'EC 3.5.4.6';
UPDATE enzyme SET sys_name = 'ADP aminohydrolase' WHERE ec = 'EC 3.5.4.7';
UPDATE enzyme SET history = 'EC 3.5.4.7 created 1961' WHERE ec = 'EC 3.5.4.7';
UPDATE enzyme SET sys_name = '4-aminoimidazole aminohydrolase' WHERE ec = 'EC 3.5.4.8';
UPDATE enzyme SET history = 'EC 3.5.4.8 created 1961' WHERE ec = 'EC 3.5.4.8';
UPDATE enzyme SET sys_name = '5,10-methenyltetrahydrofolate 5-hydrolase (decyclizing)' WHERE ec = 'EC 3.5.4.9';
UPDATE enzyme SET history = 'EC 3.5.4.9 created 1961' WHERE ec = 'EC 3.5.4.9';
UPDATE enzyme SET sys_name = 'IMP 1,2-hydrolase (decyclizing)' WHERE ec = 'EC 3.5.4.10';
UPDATE enzyme SET history = 'EC 3.5.4.10 created 1961, modified 2000' WHERE ec = 'EC 3.5.4.10';
UPDATE enzyme SET sys_name = '2-amino-4-hydroxypteridine aminohydrolase' WHERE ec = 'EC 3.5.4.11';
UPDATE enzyme SET history = 'EC 3.5.4.11 created 1965' WHERE ec = 'EC 3.5.4.11';
UPDATE enzyme SET sys_name = 'dCMP aminohydrolase' WHERE ec = 'EC 3.5.4.12';
UPDATE enzyme SET history = 'EC 3.5.4.12 created 1965' WHERE ec = 'EC 3.5.4.12';
UPDATE enzyme SET sys_name = 'dCTP aminohydrolase' WHERE ec = 'EC 3.5.4.13';
UPDATE enzyme SET history = 'EC 3.5.4.13 created 1972' WHERE ec = 'EC 3.5.4.13';
UPDATE enzyme SET history = 'EC 3.5.4.14 created 1972' WHERE ec = 'EC 3.5.4.14';
UPDATE enzyme SET sys_name = 'Guanosine aminohydrolase' WHERE ec = 'EC 3.5.4.15';
UPDATE enzyme SET history = 'EC 3.5.4.15 created 1972' WHERE ec = 'EC 3.5.4.15';
UPDATE enzyme SET sys_name = 'GTP 7,8-8,9-dihydrolase' WHERE ec = 'EC 3.5.4.16';
UPDATE enzyme SET history = 'EC 3.5.4.16 created 1972' WHERE ec = 'EC 3.5.4.16';
UPDATE enzyme SET sys_name = 'Adenosine-phosphate aminohydrolase' WHERE ec = 'EC 3.5.4.17';
UPDATE enzyme SET history = 'EC 3.5.4.17 created 1972, modified 1980' WHERE ec = 'EC 3.5.4.17';
UPDATE enzyme SET sys_name = 'ATP aminohydrolase' WHERE ec = 'EC 3.5.4.18';
UPDATE enzyme SET history = 'EC 3.5.4.18 created 1972' WHERE ec = 'EC 3.5.4.18';
UPDATE enzyme SET sys_name = '1-(5-phospho-beta-D-ribosyl)-AMP 1,6-hydrolase' WHERE ec = 'EC 3.5.4.19';
UPDATE enzyme SET history = 'EC 3.5.4.19 created 1972, modified 1976, modified 1981, modified 2000' WHERE ec = 'EC 3.5.4.19';
UPDATE enzyme SET sys_name = '1-(4-amino-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium-bromide aminohydrolase' WHERE ec = 'EC 3.5.4.20';
UPDATE enzyme SET history = 'EC 3.5.4.20 created 1972' WHERE ec = 'EC 3.5.4.20';
UPDATE enzyme SET sys_name = 'Creatinine iminohydrolase' WHERE ec = 'EC 3.5.4.21';
UPDATE enzyme SET history = 'EC 3.5.4.21 created 1972' WHERE ec = 'EC 3.5.4.21';
UPDATE enzyme SET sys_name = '1-pyrroline-4-hydroxy-2-carboxylate aminohydrolase (decyclizing)' WHERE ec = 'EC 3.5.4.22';
UPDATE enzyme SET history = 'EC 3.5.4.22 created 1976' WHERE ec = 'EC 3.5.4.22';
UPDATE enzyme SET sys_name = 'Blasticidin-S aminohydrolase' WHERE ec = 'EC 3.5.4.23';
UPDATE enzyme SET history = 'EC 3.5.4.23 created 1976' WHERE ec = 'EC 3.5.4.23';
UPDATE enzyme SET sys_name = 'Sepiapterin aminohydrolase' WHERE ec = 'EC 3.5.4.24';
UPDATE enzyme SET history = 'EC 3.5.4.24 created 1976' WHERE ec = 'EC 3.5.4.24';
UPDATE enzyme SET sys_name = 'GTP 7,8-8,9-dihydrolase (diphosphate-forming)' WHERE ec = 'EC 3.5.4.25';
UPDATE enzyme SET history = 'EC 3.5.4.25 created 1984' WHERE ec = 'EC 3.5.4.25';
UPDATE enzyme SET sys_name = '2,5-diamino-6-hydroxy-4-(5-phosphoribosylamino)pyrimidine 2-aminohydrolase' WHERE ec = 'EC 3.5.4.26';
UPDATE enzyme SET history = 'EC 3.5.4.26 created 1984' WHERE ec = 'EC 3.5.4.26';
UPDATE enzyme SET sys_name = '5,10-methenyltetrahydromethanopterin 10-hydrolase (decyclizing)' WHERE ec = 'EC 3.5.4.27';
UPDATE enzyme SET history = 'EC 3.5.4.27 created 1989' WHERE ec = 'EC 3.5.4.27';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-homocysteine aminohydrolase' WHERE ec = 'EC 3.5.4.28';
UPDATE enzyme SET history = 'EC 3.5.4.28 created 1992' WHERE ec = 'EC 3.5.4.28';
UPDATE enzyme SET sys_name = 'GTP 8,9-dihydrolase (phosphate-forming)' WHERE ec = 'EC 3.5.4.29';
UPDATE enzyme SET history = 'EC 3.5.4.29 created 2003' WHERE ec = 'EC 3.5.4.29';
UPDATE enzyme SET sys_name = 'dCTP aminohydrolase (dUMP-forming)' WHERE ec = 'EC 3.5.4.30';
UPDATE enzyme SET history = 'EC 3.5.4.30 created 2003' WHERE ec = 'EC 3.5.4.30';
UPDATE enzyme SET sys_name = 'S-methyl-5-thioadenosine amidohydrolase' WHERE ec = 'EC 3.5.4.31';
UPDATE enzyme SET history = 'EC 3.5.4.31 created 2011' WHERE ec = 'EC 3.5.4.31';
UPDATE enzyme SET sys_name = '8-oxoguanine aminohydrolase' WHERE ec = 'EC 3.5.4.32';
UPDATE enzyme SET history = 'EC 3.5.4.32 created 2012' WHERE ec = 'EC 3.5.4.32';
UPDATE enzyme SET sys_name = 'tRNA(adenine(34)) aminohydrolase' WHERE ec = 'EC 3.5.4.33';
UPDATE enzyme SET history = 'EC 3.5.4.33 created 2013' WHERE ec = 'EC 3.5.4.33';
UPDATE enzyme SET sys_name = 'tRNA(Ala)(adenine(37)) aminohydrolase' WHERE ec = 'EC 3.5.4.34';
UPDATE enzyme SET history = 'EC 3.5.4.34 created 2013' WHERE ec = 'EC 3.5.4.34';
UPDATE enzyme SET sys_name = 'tRNA(cytosine(8)) aminohydrolase' WHERE ec = 'EC 3.5.4.35';
UPDATE enzyme SET history = 'EC 3.5.4.35 created 2013' WHERE ec = 'EC 3.5.4.35';
UPDATE enzyme SET sys_name = 'mRNA(cytosine(6666)) aminohydrolase' WHERE ec = 'EC 3.5.4.36';
UPDATE enzyme SET history = 'EC 3.5.4.36 created 2013' WHERE ec = 'EC 3.5.4.36';
UPDATE enzyme SET sys_name = 'Double-stranded RNA adenine aminohydrolase' WHERE ec = 'EC 3.5.4.37';
UPDATE enzyme SET history = 'EC 3.5.4.37 created 2013' WHERE ec = 'EC 3.5.4.37';
UPDATE enzyme SET sys_name = 'Single-stranded DNA cytosine aminohydrolase' WHERE ec = 'EC 3.5.4.38';
UPDATE enzyme SET history = 'EC 3.5.4.38 created 2013' WHERE ec = 'EC 3.5.4.38';
UPDATE enzyme SET sys_name = 'GTP 7,8-8,9-dihydrolase (cyclizing, formate-releasing, diphosphate-releasing)' WHERE ec = 'EC 3.5.4.39';
UPDATE enzyme SET history = 'EC 3.5.4.39 created 2013' WHERE ec = 'EC 3.5.4.39';
UPDATE enzyme SET sys_name = '6-amino-6-deoxyfutalosine deaminase' WHERE ec = 'EC 3.5.4.40';
UPDATE enzyme SET history = 'EC 3.5.4.40 created 2014' WHERE ec = 'EC 3.5.4.40';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.5.4.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.5.4.n2';
UPDATE enzyme SET sys_name = 'Nitrile aminohydrolase' WHERE ec = 'EC 3.5.5.1';
UPDATE enzyme SET history = 'EC 3.5.5.1 created 1965, modified 1989' WHERE ec = 'EC 3.5.5.1';
UPDATE enzyme SET sys_name = 'Ricinine aminohydrolase' WHERE ec = 'EC 3.5.5.2';
UPDATE enzyme SET history = 'EC 3.5.5.2 created 1972' WHERE ec = 'EC 3.5.5.2';
UPDATE enzyme SET history = 'EC 3.5.5.3 created 1972, deleted 1990' WHERE ec = 'EC 3.5.5.3';
UPDATE enzyme SET sys_name = '3-cyano-L-alanine aminohydrolase' WHERE ec = 'EC 3.5.5.4';
UPDATE enzyme SET history = 'EC 3.5.5.4 created 1986' WHERE ec = 'EC 3.5.5.4';
UPDATE enzyme SET sys_name = 'Arylacetonitrile aminohydrolase' WHERE ec = 'EC 3.5.5.5';
UPDATE enzyme SET history = 'EC 3.5.5.5 created 1992' WHERE ec = 'EC 3.5.5.5';
UPDATE enzyme SET sys_name = '3,5-dibromo-4-hydroxybenzonitrile aminohydrolase' WHERE ec = 'EC 3.5.5.6';
UPDATE enzyme SET history = 'EC 3.5.5.6 created 1992' WHERE ec = 'EC 3.5.5.6';
UPDATE enzyme SET sys_name = 'Aliphatic nitrile aminohydrolase' WHERE ec = 'EC 3.5.5.7';
UPDATE enzyme SET history = 'EC 3.5.5.7 created 1999' WHERE ec = 'EC 3.5.5.7';
UPDATE enzyme SET sys_name = 'Thiocyanate aminohydrolase' WHERE ec = 'EC 3.5.5.8';
UPDATE enzyme SET history = 'EC 3.5.5.8 created 2000' WHERE ec = 'EC 3.5.5.8';
UPDATE enzyme SET sys_name = 'Riboflavin hydrolase' WHERE ec = 'EC 3.5.99.1';
UPDATE enzyme SET history = 'EC 3.5.99.1 created 1961' WHERE ec = 'EC 3.5.99.1';
UPDATE enzyme SET sys_name = '4-amino-5-aminomethyl-2-methylpyrimidine aminohydrolase' WHERE ec = 'EC 3.5.99.2';
UPDATE enzyme SET history = 'EC 3.5.99.2 created 1961, modified 2011' WHERE ec = 'EC 3.5.99.2';
UPDATE enzyme SET sys_name = '4-(ethylamino)-2-hydroxy-6-(isopropylamino)-1,3,5-triazine ethylaminohydrolase' WHERE ec = 'EC 3.5.99.3';
UPDATE enzyme SET history = 'EC 3.5.99.3 created 2000' WHERE ec = 'EC 3.5.99.3';
UPDATE enzyme SET sys_name = 'N-isopropylammelide isopropylaminohydrolase' WHERE ec = 'EC 3.5.99.4';
UPDATE enzyme SET history = 'EC 3.5.99.4 created 2000' WHERE ec = 'EC 3.5.99.4';
UPDATE enzyme SET sys_name = '2-aminomuconate aminohydrolase' WHERE ec = 'EC 3.5.99.5';
UPDATE enzyme SET history = 'EC 3.5.99.5 created 2000, modified 2012' WHERE ec = 'EC 3.5.99.5';
UPDATE enzyme SET sys_name = '2-amino-2-deoxy-D-glucose-6-phosphate aminohydrolase (ketol isomerizing)' WHERE ec = 'EC 3.5.99.6';
UPDATE enzyme SET history = 'EC 3.5.99.6 created 1961 as EC 5.3.1.10, transferred 2000 to EC 3.5.99.6' WHERE ec = 'EC 3.5.99.6';
UPDATE enzyme SET sys_name = '1-aminocyclopropane-1-carboxylate aminohydrolase (isomerizing)' WHERE ec = 'EC 3.5.99.7';
UPDATE enzyme SET history = 'EC 3.5.99.7 created 1981 as EC 4.1.99.4, transferred 2002 to EC 3.5.99.7' WHERE ec = 'EC 3.5.99.7';
UPDATE enzyme SET sys_name = '5-nitroanthranilate amidohydrolase' WHERE ec = 'EC 3.5.99.8';
UPDATE enzyme SET history = 'EC 3.5.99.8 created 2010' WHERE ec = 'EC 3.5.99.8';
UPDATE enzyme SET sys_name = '2-nitroimidazole nitrohydrolase' WHERE ec = 'EC 3.5.99.9';
UPDATE enzyme SET history = 'EC 3.5.99.9 created 2012' WHERE ec = 'EC 3.5.99.9';
UPDATE enzyme SET sys_name = '2-iminobutanoate aminohydrolase' WHERE ec = 'EC 3.5.99.10';
UPDATE enzyme SET history = 'EC 3.5.99.10 created 2014' WHERE ec = 'EC 3.5.99.10';
UPDATE enzyme SET sys_name = 'Diphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.1';
UPDATE enzyme SET history = 'EC 3.6.1.1 created 1961, modified 2000' WHERE ec = 'EC 3.6.1.1';
UPDATE enzyme SET sys_name = 'Trimetaphosphate hydrolase' WHERE ec = 'EC 3.6.1.2';
UPDATE enzyme SET history = 'EC 3.6.1.2 created 1961' WHERE ec = 'EC 3.6.1.2';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase' WHERE ec = 'EC 3.6.1.3';
UPDATE enzyme SET history = 'EC 3.6.1.3 created 1961 (EC 3.6.1.4 created 1961, incorporated 1965)' WHERE ec = 'EC 3.6.1.3';
UPDATE enzyme SET history = 'EC 3.6.1.4 created 1961, deleted 1965' WHERE ec = 'EC 3.6.1.4';
UPDATE enzyme SET sys_name = 'Nucleoside triphosphate phosphohydrolase (nucleoside monophosphoate-forming)' WHERE ec = 'EC 3.6.1.5';
UPDATE enzyme SET history = 'EC 3.6.1.5 created 1961, modified 1976, modified 2000, modified 2013' WHERE ec = 'EC 3.6.1.5';
UPDATE enzyme SET sys_name = 'Nucleoside-diphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.6';
UPDATE enzyme SET history = 'EC 3.6.1.6 created 1961' WHERE ec = 'EC 3.6.1.6';
UPDATE enzyme SET sys_name = 'Acylphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.7';
UPDATE enzyme SET history = 'EC 3.6.1.7 created 1961' WHERE ec = 'EC 3.6.1.7';
UPDATE enzyme SET sys_name = 'ATP diphosphohydrolase (diphosphate-forming)' WHERE ec = 'EC 3.6.1.8';
UPDATE enzyme SET history = 'EC 3.6.1.8 created 1961' WHERE ec = 'EC 3.6.1.8';
UPDATE enzyme SET sys_name = 'Dinucleotide nucleotidohydrolase' WHERE ec = 'EC 3.6.1.9';
UPDATE enzyme SET history = 'EC 3.6.1.9 created 1961' WHERE ec = 'EC 3.6.1.9';
UPDATE enzyme SET sys_name = 'Polyphosphate polyphosphohydrolase' WHERE ec = 'EC 3.6.1.10';
UPDATE enzyme SET history = 'EC 3.6.1.10 created 1961' WHERE ec = 'EC 3.6.1.10';
UPDATE enzyme SET sys_name = 'Polyphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.11';
UPDATE enzyme SET history = 'EC 3.6.1.11 created 1965' WHERE ec = 'EC 3.6.1.11';
UPDATE enzyme SET sys_name = 'dCTP nucleotidohydrolase' WHERE ec = 'EC 3.6.1.12';
UPDATE enzyme SET history = 'EC 3.6.1.12 created 1965' WHERE ec = 'EC 3.6.1.12';
UPDATE enzyme SET sys_name = 'ADP-D-ribose ribophosphohydrolase' WHERE ec = 'EC 3.6.1.13';
UPDATE enzyme SET history = 'EC 3.6.1.13 created 1965' WHERE ec = 'EC 3.6.1.13';
UPDATE enzyme SET sys_name = 'Adenosine-tetraphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.14';
UPDATE enzyme SET history = 'EC 3.6.1.14 created 1972' WHERE ec = 'EC 3.6.1.14';
UPDATE enzyme SET sys_name = 'Unspecific diphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.15';
UPDATE enzyme SET history = 'EC 3.6.1.15 created 1972' WHERE ec = 'EC 3.6.1.15';
UPDATE enzyme SET sys_name = 'CDP-glycerol phosphoglycerohydrolase' WHERE ec = 'EC 3.6.1.16';
UPDATE enzyme SET history = 'EC 3.6.1.16 created 1972' WHERE ec = 'EC 3.6.1.16';
UPDATE enzyme SET sys_name = 'P(1),P(4)-bis(5-nucleosyl)-tetraphosphate nucleotidohydrolase' WHERE ec = 'EC 3.6.1.17';
UPDATE enzyme SET history = 'EC 3.6.1.17 created 1972, modified 1976, modified 1986' WHERE ec = 'EC 3.6.1.17';
UPDATE enzyme SET sys_name = 'FAD nucleotidohydrolase' WHERE ec = 'EC 3.6.1.18';
UPDATE enzyme SET history = 'EC 3.6.1.18 created 1972' WHERE ec = 'EC 3.6.1.18';
UPDATE enzyme SET sys_name = 'Nucleoside-triphosphate diphosphohydrolase' WHERE ec = 'EC 3.6.1.19';
UPDATE enzyme SET history = 'EC 3.6.1.19 created 1972' WHERE ec = 'EC 3.6.1.19';
UPDATE enzyme SET sys_name = '5-acylphosphoadenosine acylhydrolase' WHERE ec = 'EC 3.6.1.20';
UPDATE enzyme SET history = 'EC 3.6.1.20 created 1972' WHERE ec = 'EC 3.6.1.20';
UPDATE enzyme SET sys_name = 'ADP-sugar sugarphosphohydrolase' WHERE ec = 'EC 3.6.1.21';
UPDATE enzyme SET history = 'EC 3.6.1.21 created 1972, modified 1999' WHERE ec = 'EC 3.6.1.21';
UPDATE enzyme SET sys_name = 'NAD(+) phosphohydrolase' WHERE ec = 'EC 3.6.1.22';
UPDATE enzyme SET history = 'EC 3.6.1.22 created 1972' WHERE ec = 'EC 3.6.1.22';
UPDATE enzyme SET sys_name = 'dUTP nucleotidohydrolase' WHERE ec = 'EC 3.6.1.23';
UPDATE enzyme SET history = 'EC 3.6.1.23 created 1972' WHERE ec = 'EC 3.6.1.23';
UPDATE enzyme SET sys_name = 'Nucleoside-5-phosphoacylate acylhydrolase' WHERE ec = 'EC 3.6.1.24';
UPDATE enzyme SET history = 'EC 3.6.1.24 created 1972' WHERE ec = 'EC 3.6.1.24';
UPDATE enzyme SET sys_name = 'Triphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.25';
UPDATE enzyme SET history = 'EC 3.6.1.25 created 1976' WHERE ec = 'EC 3.6.1.25';
UPDATE enzyme SET sys_name = 'CDP-diacylglycerol phosphatidylhydrolase' WHERE ec = 'EC 3.6.1.26';
UPDATE enzyme SET history = 'EC 3.6.1.26 created 1976' WHERE ec = 'EC 3.6.1.26';
UPDATE enzyme SET sys_name = 'Ditrans,octacis-undecaprenyl-diphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.27';
UPDATE enzyme SET history = 'EC 3.6.1.27 created 1978, modified 2002, modified 2012' WHERE ec = 'EC 3.6.1.27';
UPDATE enzyme SET sys_name = 'Thiamine-triphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.28';
UPDATE enzyme SET history = 'EC 3.6.1.28 created 1978' WHERE ec = 'EC 3.6.1.28';
UPDATE enzyme SET sys_name = 'P(1),P(3)-bis(5-adenosyl)-triphosphate adenylohydrolase' WHERE ec = 'EC 3.6.1.29';
UPDATE enzyme SET history = 'EC 3.6.1.29 created 1978' WHERE ec = 'EC 3.6.1.29';
UPDATE enzyme SET history = 'EC 3.6.1.30 created 1978, deleted 2012' WHERE ec = 'EC 3.6.1.30';
UPDATE enzyme SET sys_name = '1-(5-phospho-beta-D-ribosyl)-ATP diphosphohydrolase' WHERE ec = 'EC 3.6.1.31';
UPDATE enzyme SET history = 'EC 3.6.1.31 created 1981' WHERE ec = 'EC 3.6.1.31';
UPDATE enzyme SET history = 'EC 3.6.1.32 created 1984, deleted 2000' WHERE ec = 'EC 3.6.1.32';
UPDATE enzyme SET history = 'EC 3.6.1.33 created 1984, deleted 2000' WHERE ec = 'EC 3.6.1.33';
UPDATE enzyme SET history = 'EC 3.6.1.34 created 1984, deleted 2000' WHERE ec = 'EC 3.6.1.34';
UPDATE enzyme SET history = 'EC 3.6.1.35 created 1984, deleted 2000' WHERE ec = 'EC 3.6.1.35';
UPDATE enzyme SET history = 'EC 3.6.1.36 created 1984, deleted 2000' WHERE ec = 'EC 3.6.1.36';
UPDATE enzyme SET history = 'EC 3.6.1.37 created 1984, deleted 2000' WHERE ec = 'EC 3.6.1.37';
UPDATE enzyme SET history = 'EC 3.6.1.38 created 1984, deleted 2000' WHERE ec = 'EC 3.6.1.38';
UPDATE enzyme SET sys_name = 'dTTP nucleotidohydrolase' WHERE ec = 'EC 3.6.1.39';
UPDATE enzyme SET history = 'EC 3.6.1.39 created 1984' WHERE ec = 'EC 3.6.1.39';
UPDATE enzyme SET sys_name = 'Guanosine-5-triphosphate,3-diphosphate 5-phosphohydrolase' WHERE ec = 'EC 3.6.1.40';
UPDATE enzyme SET history = 'EC 3.6.1.40 created 1986, modified 2010' WHERE ec = 'EC 3.6.1.40';
UPDATE enzyme SET sys_name = 'P(1),P(4)-bis(5-nucleosyl)-tetraphosphate nucleosidebisphosphohydrolase' WHERE ec = 'EC 3.6.1.41';
UPDATE enzyme SET history = 'EC 3.6.1.41 created 1986' WHERE ec = 'EC 3.6.1.41';
UPDATE enzyme SET sys_name = 'GDP phosphohydrolase' WHERE ec = 'EC 3.6.1.42';
UPDATE enzyme SET history = 'EC 3.6.1.42 created 1989' WHERE ec = 'EC 3.6.1.42';
UPDATE enzyme SET sys_name = 'Dolichyl-diphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.43';
UPDATE enzyme SET history = 'EC 3.6.1.43 created 1989' WHERE ec = 'EC 3.6.1.43';
UPDATE enzyme SET sys_name = 'Oligosaccharide-diphosphodolichol phosphodolichohydrolase' WHERE ec = 'EC 3.6.1.44';
UPDATE enzyme SET history = 'EC 3.6.1.44 created 1992' WHERE ec = 'EC 3.6.1.44';
UPDATE enzyme SET sys_name = 'UDP-sugar sugarphosphohydrolase' WHERE ec = 'EC 3.6.1.45';
UPDATE enzyme SET history = 'EC 3.6.1.45 created 1999' WHERE ec = 'EC 3.6.1.45';
UPDATE enzyme SET history = 'EC 3.6.1.46 created 2000, deleted 2003' WHERE ec = 'EC 3.6.1.46';
UPDATE enzyme SET history = 'EC 3.6.1.47 created 2000, deleted 2003' WHERE ec = 'EC 3.6.1.47';
UPDATE enzyme SET history = 'EC 3.6.1.48 created 2000, deleted 2003' WHERE ec = 'EC 3.6.1.48';
UPDATE enzyme SET history = 'EC 3.6.1.49 created 2000, deleted 2003' WHERE ec = 'EC 3.6.1.49';
UPDATE enzyme SET history = 'EC 3.6.1.50 created 2000, deleted 2003' WHERE ec = 'EC 3.6.1.50';
UPDATE enzyme SET history = 'EC 3.6.1.51 created 2000, deleted 2003' WHERE ec = 'EC 3.6.1.51';
UPDATE enzyme SET sys_name = 'Diphospho-myo-inositol-polyphosphate diphosphohydrolase' WHERE ec = 'EC 3.6.1.52';
UPDATE enzyme SET history = 'EC 3.6.1.52 created 2002' WHERE ec = 'EC 3.6.1.52';
UPDATE enzyme SET sys_name = 'CDP-choline phosphohydrolase' WHERE ec = 'EC 3.6.1.53';
UPDATE enzyme SET history = 'EC 3.6.1.53 created 2008' WHERE ec = 'EC 3.6.1.53';
UPDATE enzyme SET sys_name = 'UDP-2,3-bis((3R)-3-hydroxymyristoyl)-alpha-D-glucosamine 2,3-bis((3R)-3-hydroxymyristoyl)-beta-D-glucosaminyl 1-phosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.54';
UPDATE enzyme SET history = 'EC 3.6.1.54 created 2010' WHERE ec = 'EC 3.6.1.54';
UPDATE enzyme SET sys_name = '8-oxo-dGTP diphosphohydrolase' WHERE ec = 'EC 3.6.1.55';
UPDATE enzyme SET history = 'EC 3.6.1.55 created 2011' WHERE ec = 'EC 3.6.1.55';
UPDATE enzyme SET sys_name = '2-hydroxy-dATP diphosphohydrolase' WHERE ec = 'EC 3.6.1.56';
UPDATE enzyme SET history = 'EC 3.6.1.56 created 2011' WHERE ec = 'EC 3.6.1.56';
UPDATE enzyme SET sys_name = 'UDP-2,4-diacetamido-2,4,6-trideoxy-beta-L-altropyranose hydrolase' WHERE ec = 'EC 3.6.1.57';
UPDATE enzyme SET history = 'EC 3.6.1.57 created 2011' WHERE ec = 'EC 3.6.1.57';
UPDATE enzyme SET sys_name = '8-oxo-dGDP phosphohydrolase' WHERE ec = 'EC 3.6.1.58';
UPDATE enzyme SET history = 'EC 3.6.1.58 created 2012' WHERE ec = 'EC 3.6.1.58';
UPDATE enzyme SET sys_name = '5-(N(7)-methyl 5-triphosphoguanosine)-(mRNA) N(7)-methylguanosine 5-phosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.59';
UPDATE enzyme SET history = 'EC 3.6.1.59 created 2012, modified 2013' WHERE ec = 'EC 3.6.1.59';
UPDATE enzyme SET sys_name = 'P(1),P(6)-bis(5-adenosyl)hexaphosphate nucleotidohydrolase (AMP-forming)' WHERE ec = 'EC 3.6.1.60';
UPDATE enzyme SET history = 'EC 3.6.1.60 created 2012' WHERE ec = 'EC 3.6.1.60';
UPDATE enzyme SET sys_name = 'P(1),P(6)-bis(5-adenosyl)hexaphosphate nucleotidohydrolase (ATP-forming)' WHERE ec = 'EC 3.6.1.61';
UPDATE enzyme SET history = 'EC 3.6.1.61 created 2012' WHERE ec = 'EC 3.6.1.61';
UPDATE enzyme SET sys_name = '5-(N(7)-methylguanosine 5-triphospho)-(mRNA) N(7)-methylguanosine-5-diphosphate phosphohydrolase' WHERE ec = 'EC 3.6.1.62';
UPDATE enzyme SET history = 'EC 3.6.1.62 created 2012, modified 2013' WHERE ec = 'EC 3.6.1.62';
UPDATE enzyme SET sys_name = 'Alpha-D-ribose-1-methylphosphonate-5-triphosphate diphosphohydrolase' WHERE ec = 'EC 3.6.1.63';
UPDATE enzyme SET history = 'EC 3.6.1.63 created 2012' WHERE ec = 'EC 3.6.1.63';
UPDATE enzyme SET sys_name = 'Inosine diphosphate phosphatase' WHERE ec = 'EC 3.6.1.64';
UPDATE enzyme SET history = 'EC 3.6.1.64 created 2013' WHERE ec = 'EC 3.6.1.64';
UPDATE enzyme SET sys_name = '(Deoxy)cytidine 5-triphosphate diphosphohydrolase' WHERE ec = 'EC 3.6.1.65';
UPDATE enzyme SET history = 'EC 3.6.1.65 created 2013' WHERE ec = 'EC 3.6.1.65';
UPDATE enzyme SET sys_name = 'XTP/dITP diphosphohydrolase (diphosphate-forming)' WHERE ec = 'EC 3.6.1.66';
UPDATE enzyme SET history = 'EC 3.6.1.66 created 2013' WHERE ec = 'EC 3.6.1.66';
UPDATE enzyme SET sys_name = 'D-tyrosyl-tRNA(Tyr) tyrosylhydrolase' WHERE ec = 'EC 3.6.1.n1';
UPDATE enzyme SET history = 'EC 3.6.1.n1 created 2010' WHERE ec = 'EC 3.6.1.n1';
UPDATE enzyme SET sys_name = 'L-cysteinyl-tRNA(Pro) cysteinylhydrolase' WHERE ec = 'EC 3.6.1.n2';
UPDATE enzyme SET history = 'EC 3.6.1.n2 created 2010' WHERE ec = 'EC 3.6.1.n2';
UPDATE enzyme SET sys_name = 'L-cysteinyl-tRNA(Cys)  cysteinylhydrolase' WHERE ec = 'EC 3.6.1.n3';
UPDATE enzyme SET history = 'EC 3.6.1.n3 created 2010' WHERE ec = 'EC 3.6.1.n3';
UPDATE enzyme SET sys_name = '7,8-dihydroneopterin 3-triphosphate diphosphohydrolase' WHERE ec = 'EC 3.6.1.n4';
UPDATE enzyme SET history = 'EC 3.6.1.n4 created 2010' WHERE ec = 'EC 3.6.1.n4';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.6.1.n5';
UPDATE enzyme SET sys_name = 'Adenylylsulfate sulfohydrolase' WHERE ec = 'EC 3.6.2.1';
UPDATE enzyme SET history = 'EC 3.6.2.1 created 1972' WHERE ec = 'EC 3.6.2.1';
UPDATE enzyme SET sys_name = '3-phosphoadenylylsulfate sulfohydrolase' WHERE ec = 'EC 3.6.2.2';
UPDATE enzyme SET history = 'EC 3.6.2.2 created 1972' WHERE ec = 'EC 3.6.2.2';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (phospholipid-flipping)' WHERE ec = 'EC 3.6.3.1';
UPDATE enzyme SET history = 'EC 3.6.3.1 created 2000 (EC 3.6.3.13 created 2000, incorporated 2001)' WHERE ec = 'EC 3.6.3.1';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Mg(2+)-importing)' WHERE ec = 'EC 3.6.3.2';
UPDATE enzyme SET history = 'EC 3.6.3.2 created 2000, modified 2001' WHERE ec = 'EC 3.6.3.2';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Cd(2+)-exporting)' WHERE ec = 'EC 3.6.3.3';
UPDATE enzyme SET history = 'EC 3.6.3.3 created 2000' WHERE ec = 'EC 3.6.3.3';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Cu(2+)-exporting)' WHERE ec = 'EC 3.6.3.4';
UPDATE enzyme SET history = 'EC 3.6.3.4 created 2000, modified 2013' WHERE ec = 'EC 3.6.3.4';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Zn(2+)-exporting)' WHERE ec = 'EC 3.6.3.5';
UPDATE enzyme SET history = 'EC 3.6.3.5 created 2000, modified 2001, modified 2006' WHERE ec = 'EC 3.6.3.5';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (H(+)-exporting)' WHERE ec = 'EC 3.6.3.6';
UPDATE enzyme SET history = 'EC 3.6.3.6 created 1984 as EC 3.6.1.35, transferred 2000 to EC 3.6.3.6' WHERE ec = 'EC 3.6.3.6';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Na(+)-exporting)' WHERE ec = 'EC 3.6.3.7';
UPDATE enzyme SET history = 'EC 3.6.3.7 created 2000, modified 2001' WHERE ec = 'EC 3.6.3.7';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Ca(2+)-transporting)' WHERE ec = 'EC 3.6.3.8';
UPDATE enzyme SET history = 'EC 3.6.3.8 created 1984 as EC 3.6.1.38, transferred 2000 to EC 3.6.3.8, modified 2001, modified 2011' WHERE ec = 'EC 3.6.3.8';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Na(+)/K(+)-exchanging)' WHERE ec = 'EC 3.6.3.9';
UPDATE enzyme SET history = 'EC 3.6.3.9 created 1984 as EC 3.6.1.37, transferred 2000 to EC 3.6.3.9, modified 2001' WHERE ec = 'EC 3.6.3.9';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (H(+)/K(+)-exchanging)' WHERE ec = 'EC 3.6.3.10';
UPDATE enzyme SET history = 'EC 3.6.3.10 created 1984 as EC 3.6.1.36, transferred 2000 to EC 3.6.3.10' WHERE ec = 'EC 3.6.3.10';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Cl(-)-importing)' WHERE ec = 'EC 3.6.3.11';
UPDATE enzyme SET history = 'EC 3.6.3.11 created 2000' WHERE ec = 'EC 3.6.3.11';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (K(+)-importing)' WHERE ec = 'EC 3.6.3.12';
UPDATE enzyme SET history = 'EC 3.6.3.12 created 2000' WHERE ec = 'EC 3.6.3.12';
UPDATE enzyme SET history = 'EC 3.6.3.13 created 2000, deleted 2001' WHERE ec = 'EC 3.6.3.13';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (H(+)-transporting)' WHERE ec = 'EC 3.6.3.14';
UPDATE enzyme SET history = 'EC 3.6.3.14 created 1984 as EC 3.6.1.34, transferred 2000 to EC 3.6.3.14' WHERE ec = 'EC 3.6.3.14';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Na(+)-transporting)' WHERE ec = 'EC 3.6.3.15';
UPDATE enzyme SET history = 'EC 3.6.3.15 created 2000' WHERE ec = 'EC 3.6.3.15';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (arsenite-exporting)' WHERE ec = 'EC 3.6.3.16';
UPDATE enzyme SET history = 'EC 3.6.3.16 created 2000' WHERE ec = 'EC 3.6.3.16';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (monosaccharide-importing)' WHERE ec = 'EC 3.6.3.17';
UPDATE enzyme SET history = 'EC 3.6.3.17 created 2000' WHERE ec = 'EC 3.6.3.17';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (disaccharide-importing)' WHERE ec = 'EC 3.6.3.18';
UPDATE enzyme SET history = 'EC 3.6.3.18 created 2000' WHERE ec = 'EC 3.6.3.18';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (maltose-importing)' WHERE ec = 'EC 3.6.3.19';
UPDATE enzyme SET history = 'EC 3.6.3.19 created 2000' WHERE ec = 'EC 3.6.3.19';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (glycerol-3-phosphate-importing)' WHERE ec = 'EC 3.6.3.20';
UPDATE enzyme SET history = 'EC 3.6.3.20 created 2000' WHERE ec = 'EC 3.6.3.20';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (polar-amino-acid-importing)' WHERE ec = 'EC 3.6.3.21';
UPDATE enzyme SET history = 'EC 3.6.3.21 created 2000' WHERE ec = 'EC 3.6.3.21';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (nonpolar-amino-acid-transporting)' WHERE ec = 'EC 3.6.3.22';
UPDATE enzyme SET history = 'EC 3.6.3.22 created 2000' WHERE ec = 'EC 3.6.3.22';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (oligopeptide-importing)' WHERE ec = 'EC 3.6.3.23';
UPDATE enzyme SET history = 'EC 3.6.3.23 created 2000' WHERE ec = 'EC 3.6.3.23';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (nickel-importing)' WHERE ec = 'EC 3.6.3.24';
UPDATE enzyme SET history = 'EC 3.6.3.24 created 2000' WHERE ec = 'EC 3.6.3.24';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (sulfate-importing)' WHERE ec = 'EC 3.6.3.25';
UPDATE enzyme SET history = 'EC 3.6.3.25 created 2000' WHERE ec = 'EC 3.6.3.25';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (nitrate-importing)' WHERE ec = 'EC 3.6.3.26';
UPDATE enzyme SET history = 'EC 3.6.3.26 created 2000' WHERE ec = 'EC 3.6.3.26';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (phosphate-importing)' WHERE ec = 'EC 3.6.3.27';
UPDATE enzyme SET history = 'EC 3.6.3.27 created 2000' WHERE ec = 'EC 3.6.3.27';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (phosphonate-transporting)' WHERE ec = 'EC 3.6.3.28';
UPDATE enzyme SET history = 'EC 3.6.3.28 created 2000' WHERE ec = 'EC 3.6.3.28';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (molybdate-importing)' WHERE ec = 'EC 3.6.3.29';
UPDATE enzyme SET history = 'EC 3.6.3.29 created 2000' WHERE ec = 'EC 3.6.3.29';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (ferric-ion-transporting)' WHERE ec = 'EC 3.6.3.30';
UPDATE enzyme SET history = 'EC 3.6.3.30 created 2000' WHERE ec = 'EC 3.6.3.30';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (polyamine-importing)' WHERE ec = 'EC 3.6.3.31';
UPDATE enzyme SET history = 'EC 3.6.3.31 created 2000' WHERE ec = 'EC 3.6.3.31';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (quaternary-amine-importing)' WHERE ec = 'EC 3.6.3.32';
UPDATE enzyme SET history = 'EC 3.6.3.32 created 2000' WHERE ec = 'EC 3.6.3.32';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (vitamin B12-importing)' WHERE ec = 'EC 3.6.3.33';
UPDATE enzyme SET history = 'EC 3.6.3.33 created 2000' WHERE ec = 'EC 3.6.3.33';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (iron-chelate-importing)' WHERE ec = 'EC 3.6.3.34';
UPDATE enzyme SET history = 'EC 3.6.3.34 created 2000' WHERE ec = 'EC 3.6.3.34';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (manganese-importing)' WHERE ec = 'EC 3.6.3.35';
UPDATE enzyme SET history = 'EC 3.6.3.35 created 2000' WHERE ec = 'EC 3.6.3.35';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (taurine-importing)' WHERE ec = 'EC 3.6.3.36';
UPDATE enzyme SET history = 'EC 3.6.3.36 created 2000' WHERE ec = 'EC 3.6.3.36';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (guanine-importing)' WHERE ec = 'EC 3.6.3.37';
UPDATE enzyme SET history = 'EC 3.6.3.37 created 2000' WHERE ec = 'EC 3.6.3.37';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (capsular-polysaccharide-exporting)' WHERE ec = 'EC 3.6.3.38';
UPDATE enzyme SET history = 'EC 3.6.3.38 created 2000' WHERE ec = 'EC 3.6.3.38';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (lipopolysaccharide-exporting)' WHERE ec = 'EC 3.6.3.39';
UPDATE enzyme SET history = 'EC 3.6.3.39 created 2000' WHERE ec = 'EC 3.6.3.39';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (teichoic-acid-exporting)' WHERE ec = 'EC 3.6.3.40';
UPDATE enzyme SET history = 'EC 3.6.3.40 created 2000' WHERE ec = 'EC 3.6.3.40';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (heme-exporting)' WHERE ec = 'EC 3.6.3.41';
UPDATE enzyme SET history = 'EC 3.6.3.41 created 2000' WHERE ec = 'EC 3.6.3.41';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (beta-glucan-exporting)' WHERE ec = 'EC 3.6.3.42';
UPDATE enzyme SET history = 'EC 3.6.3.42 created 2000' WHERE ec = 'EC 3.6.3.42';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (peptide-exporting)' WHERE ec = 'EC 3.6.3.43';
UPDATE enzyme SET history = 'EC 3.6.3.43 created 2000' WHERE ec = 'EC 3.6.3.43';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (xenobiotic-exporting)' WHERE ec = 'EC 3.6.3.44';
UPDATE enzyme SET history = 'EC 3.6.3.44 created 2000 (EC 3.6.3.45 incorporated 2006), modified 2006' WHERE ec = 'EC 3.6.3.44';
UPDATE enzyme SET history = 'EC 3.6.3.45 created 2000, deleted 2006' WHERE ec = 'EC 3.6.3.45';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (heavy-metal-exporting)' WHERE ec = 'EC 3.6.3.46';
UPDATE enzyme SET history = 'EC 3.6.3.46 created 2000' WHERE ec = 'EC 3.6.3.46';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (fatty-acyl-CoA-transporting)' WHERE ec = 'EC 3.6.3.47';
UPDATE enzyme SET history = 'EC 3.6.3.47 created 2000' WHERE ec = 'EC 3.6.3.47';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (alpha-factor-transporting)' WHERE ec = 'EC 3.6.3.48';
UPDATE enzyme SET history = 'EC 3.6.3.48 created 2000' WHERE ec = 'EC 3.6.3.48';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (channel-conductance-controlling)' WHERE ec = 'EC 3.6.3.49';
UPDATE enzyme SET history = 'EC 3.6.3.49 created 2000' WHERE ec = 'EC 3.6.3.49';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (protein-secreting)' WHERE ec = 'EC 3.6.3.50';
UPDATE enzyme SET history = 'EC 3.6.3.50 created 2000' WHERE ec = 'EC 3.6.3.50';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (mitochondrial protein-importing)' WHERE ec = 'EC 3.6.3.51';
UPDATE enzyme SET history = 'EC 3.6.3.51 created 2000' WHERE ec = 'EC 3.6.3.51';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (chloroplast protein-importing)' WHERE ec = 'EC 3.6.3.52';
UPDATE enzyme SET history = 'EC 3.6.3.52 created 2000' WHERE ec = 'EC 3.6.3.52';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Ag(+)-exporting)' WHERE ec = 'EC 3.6.3.53';
UPDATE enzyme SET history = 'EC 3.6.3.53 created 2000' WHERE ec = 'EC 3.6.3.53';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (Cu(+)-exporting)' WHERE ec = 'EC 3.6.3.54';
UPDATE enzyme SET history = 'EC 3.6.3.54 created 2013' WHERE ec = 'EC 3.6.3.54';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (tungstate-importing)' WHERE ec = 'EC 3.6.3.55';
UPDATE enzyme SET history = 'EC 3.6.3.55 created 2013' WHERE ec = 'EC 3.6.3.55';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.6.3.n1';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (actin-translocating)' WHERE ec = 'EC 3.6.4.1';
UPDATE enzyme SET history = 'EC 3.6.4.1 created 1984 as EC 3.6.1.32, transferred 2000 to EC 3.6.4.1' WHERE ec = 'EC 3.6.4.1';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (tubulin-translocating); dynein adenosine 5-triphosphatase' WHERE ec = 'EC 3.6.4.2';
UPDATE enzyme SET history = 'EC 3.6.4.2 created 1984 as EC 3.6.1.33, transferred 2000 to EC 3.6.4.2' WHERE ec = 'EC 3.6.4.2';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (tubulin-dimerizing)' WHERE ec = 'EC 3.6.4.3';
UPDATE enzyme SET history = 'EC 3.6.4.3 created 2000' WHERE ec = 'EC 3.6.4.3';
UPDATE enzyme SET sys_name = 'Kinesin ATP phosphohydrolase (plus-end-directed)' WHERE ec = 'EC 3.6.4.4';
UPDATE enzyme SET history = 'EC 3.6.4.4 created 2000' WHERE ec = 'EC 3.6.4.4';
UPDATE enzyme SET sys_name = 'Kinesin ATP phosphohydrolase (minus-end-directed)' WHERE ec = 'EC 3.6.4.5';
UPDATE enzyme SET history = 'EC 3.6.4.5 created 2000' WHERE ec = 'EC 3.6.4.5';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (vesicle-fusing)' WHERE ec = 'EC 3.6.4.6';
UPDATE enzyme SET history = 'EC 3.6.4.6 created 2000' WHERE ec = 'EC 3.6.4.6';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (peroxisome-assembling)' WHERE ec = 'EC 3.6.4.7';
UPDATE enzyme SET history = 'EC 3.6.4.7 created 2000' WHERE ec = 'EC 3.6.4.7';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (polypeptide-degrading)' WHERE ec = 'EC 3.6.4.8';
UPDATE enzyme SET history = 'EC 3.6.4.8 created 2000' WHERE ec = 'EC 3.6.4.8';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (polypeptide-unfolding)' WHERE ec = 'EC 3.6.4.9';
UPDATE enzyme SET history = 'EC 3.6.4.9 created 2000' WHERE ec = 'EC 3.6.4.9';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (polypeptide-polymerizing)' WHERE ec = 'EC 3.6.4.10';
UPDATE enzyme SET history = 'EC 3.6.4.10 created 2000' WHERE ec = 'EC 3.6.4.10';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (nucleosome-assembling)' WHERE ec = 'EC 3.6.4.11';
UPDATE enzyme SET history = 'EC 3.6.4.11 created 2000' WHERE ec = 'EC 3.6.4.11';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (DNA helix unwinding)' WHERE ec = 'EC 3.6.4.12';
UPDATE enzyme SET history = 'EC 3.6.4.12 created 2009' WHERE ec = 'EC 3.6.4.12';
UPDATE enzyme SET sys_name = 'ATP phosphohydrolase (RNA helix unwinding)' WHERE ec = 'EC 3.6.4.13';
UPDATE enzyme SET history = 'EC 3.6.4.13 created 2009' WHERE ec = 'EC 3.6.4.13';
UPDATE enzyme SET sys_name = 'GTP phosphohydrolase (signaling)' WHERE ec = 'EC 3.6.5.1';
UPDATE enzyme SET history = 'EC 3.6.5.1 created 2000 as EC 3.6.1.46, transferred 2003 to EC 3.6.5.1' WHERE ec = 'EC 3.6.5.1';
UPDATE enzyme SET sys_name = 'GTP phosphohydrolase (cell-regulating)' WHERE ec = 'EC 3.6.5.2';
UPDATE enzyme SET history = 'EC 3.6.5.2 created 2000 as EC 3.6.1.47, transferred 2003 to EC 3.6.5.2' WHERE ec = 'EC 3.6.5.2';
UPDATE enzyme SET sys_name = 'GTP phosphohydrolase (mRNA-translation-assisting)' WHERE ec = 'EC 3.6.5.3';
UPDATE enzyme SET history = 'EC 3.6.5.3 created 2000 as EC 3.6.1.48, transferred 2003 to EC 3.6.5.3' WHERE ec = 'EC 3.6.5.3';
UPDATE enzyme SET sys_name = 'GTP phosphohydrolase (protein-synthesis-assisting)' WHERE ec = 'EC 3.6.5.4';
UPDATE enzyme SET history = 'EC 3.6.5.4 created 2000 as EC 3.6.1.49, transferred 2003 to EC 3.6.5.4' WHERE ec = 'EC 3.6.5.4';
UPDATE enzyme SET sys_name = 'GTP phosphohydrolase (vesicle-releasing)' WHERE ec = 'EC 3.6.5.5';
UPDATE enzyme SET history = 'EC 3.6.5.5 created 2000 as EC 3.6.1.50, transferred 2003 to EC 3.6.5.5' WHERE ec = 'EC 3.6.5.5';
UPDATE enzyme SET sys_name = 'GTP phosphohydrolase (microtubule-releasing)' WHERE ec = 'EC 3.6.5.6';
UPDATE enzyme SET history = 'EC 3.6.5.6 created 2000 as EC 3.6.1.51, transferred 2003 to EC 3.6.5.6' WHERE ec = 'EC 3.6.5.6';
UPDATE enzyme SET sys_name = 'GTP phosphohydrolase (protein-synthesis-assisting, ribosomal back translocating)' WHERE ec = 'EC 3.6.5.n1';
UPDATE enzyme SET history = 'EC 3.6.5.n1 created 2011' WHERE ec = 'EC 3.6.5.n1';
UPDATE enzyme SET sys_name = 'Oxaloacetate acetylhydrolase' WHERE ec = 'EC 3.7.1.1';
UPDATE enzyme SET history = 'EC 3.7.1.1 created 1961' WHERE ec = 'EC 3.7.1.1';
UPDATE enzyme SET sys_name = '4-fumarylacetoacetate fumarylhydrolase' WHERE ec = 'EC 3.7.1.2';
UPDATE enzyme SET history = 'EC 3.7.1.2 created 1961' WHERE ec = 'EC 3.7.1.2';
UPDATE enzyme SET sys_name = 'L-kynurenine hydrolase' WHERE ec = 'EC 3.7.1.3';
UPDATE enzyme SET history = 'EC 3.7.1.3 created 1965' WHERE ec = 'EC 3.7.1.3';
UPDATE enzyme SET sys_name = '2,4,4,6-tetrahydroxydehydrochalcone 1,3,5-trihydroxybenzenehydrolase' WHERE ec = 'EC 3.7.1.4';
UPDATE enzyme SET history = 'EC 3.7.1.4 created 1972' WHERE ec = 'EC 3.7.1.4';
UPDATE enzyme SET sys_name = '3-acylpyruvate acylhydrolase' WHERE ec = 'EC 3.7.1.5';
UPDATE enzyme SET history = 'EC 3.7.1.5 created 1976' WHERE ec = 'EC 3.7.1.5';
UPDATE enzyme SET sys_name = '2,4-dioxopentanoate acetylhydrolase' WHERE ec = 'EC 3.7.1.6';
UPDATE enzyme SET history = 'EC 3.7.1.6 created 1984' WHERE ec = 'EC 3.7.1.6';
UPDATE enzyme SET sys_name = 'Nonane-4,6-dione acylhydrolase' WHERE ec = 'EC 3.7.1.7';
UPDATE enzyme SET history = 'EC 3.7.1.7 created 1989' WHERE ec = 'EC 3.7.1.7';
UPDATE enzyme SET sys_name = '2,6-dioxo-6-phenylhexa-3-enoate benzoylhydrolase' WHERE ec = 'EC 3.7.1.8';
UPDATE enzyme SET history = 'EC 3.7.1.8 created 1989' WHERE ec = 'EC 3.7.1.8';
UPDATE enzyme SET sys_name = '2-hydroxymuconate-6-semialdehyde formylhydrolase' WHERE ec = 'EC 3.7.1.9';
UPDATE enzyme SET history = 'EC 3.7.1.9 created 1990, modified 2013' WHERE ec = 'EC 3.7.1.9';
UPDATE enzyme SET sys_name = 'Cyclohexane-1,3-dione acylhydrolase (decyclizing)' WHERE ec = 'EC 3.7.1.10';
UPDATE enzyme SET history = 'EC 3.7.1.10 created 1992' WHERE ec = 'EC 3.7.1.10';
UPDATE enzyme SET sys_name = 'Cyclohexane-1,2-dione acylhydrolase (decyclizing)' WHERE ec = 'EC 3.7.1.11';
UPDATE enzyme SET history = 'EC 3.7.1.11 created 2009' WHERE ec = 'EC 3.7.1.11';
UPDATE enzyme SET sys_name = 'Cobalt-precorrin 5A acylhydrolase' WHERE ec = 'EC 3.7.1.12';
UPDATE enzyme SET history = 'EC 3.7.1.12 created 2010' WHERE ec = 'EC 3.7.1.12';
UPDATE enzyme SET sys_name = '(2E,4E)-6-(2-aminophenyl)-2-hydroxy-6-oxohexa-2,4-dienoate acylhydrolase' WHERE ec = 'EC 3.7.1.13';
UPDATE enzyme SET history = 'EC 3.7.1.13 created 2010' WHERE ec = 'EC 3.7.1.13';
UPDATE enzyme SET sys_name = '(2Z,4E)-2-hydroxy-6-oxona-2,4-dienedioate succinylhydrolase' WHERE ec = 'EC 3.7.1.14';
UPDATE enzyme SET history = 'EC 3.7.1.14 created 2011, modified 2012' WHERE ec = 'EC 3.7.1.14';
UPDATE enzyme SET history = 'EC 3.7.1.15 created 2011, deleted 2013' WHERE ec = 'EC 3.7.1.15';
UPDATE enzyme SET history = 'EC 3.7.1.16 created 2011, deleted 2013' WHERE ec = 'EC 3.7.1.16';
UPDATE enzyme SET sys_name = '4,5-9,10-diseco-3-hydroxy-5,9,17-trioxoandrosta-1(10),2-diene-4-oate hydrolase ( (2Z,4Z)-2-hydroxyhexa-2,4-dienoate-forming)' WHERE ec = 'EC 3.7.1.17';
UPDATE enzyme SET history = 'EC 3.7.1.17 created 2012' WHERE ec = 'EC 3.7.1.17';
UPDATE enzyme SET sys_name = 'Bornane-2,6-dione hydrolase' WHERE ec = 'EC 3.7.1.18';
UPDATE enzyme SET history = 'EC 3.7.1.18 created 2012' WHERE ec = 'EC 3.7.1.18';
UPDATE enzyme SET sys_name = '1-(2,6-dihydroxypyridin-3-yl)-4-(methylamino)butan-1-one hydrolase' WHERE ec = 'EC 3.7.1.19';
UPDATE enzyme SET history = 'EC 3.7.1.19 created 2012' WHERE ec = 'EC 3.7.1.19';
UPDATE enzyme SET sys_name = '3-fumarylpyruvate hydrolyase' WHERE ec = 'EC 3.7.1.20';
UPDATE enzyme SET history = 'EC 3.7.1.20 created 2012' WHERE ec = 'EC 3.7.1.20';
UPDATE enzyme SET sys_name = '6-oxocyclohex-1-ene-1-carbonyl-CoA hydrolase (decyclizing)' WHERE ec = 'EC 3.7.1.21';
UPDATE enzyme SET history = 'EC 3.7.1.21 created 2014' WHERE ec = 'EC 3.7.1.21';
UPDATE enzyme SET sys_name = '3D-3,5/4-trihydroxycyclohexa-1,2-dione hydrolase (decyclizing)' WHERE ec = 'EC 3.7.1.22';
UPDATE enzyme SET history = 'EC 3.7.1.22 created 2014' WHERE ec = 'EC 3.7.1.22';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.7.1.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 3.7.1.n2';
UPDATE enzyme SET sys_name = 'Alkyl-halide halidohydrolase' WHERE ec = 'EC 3.8.1.1';
UPDATE enzyme SET history = 'EC 3.8.1.1 created 1961' WHERE ec = 'EC 3.8.1.1';
UPDATE enzyme SET sys_name = '(S)-2-haloacid halidohydrolase' WHERE ec = 'EC 3.8.1.2';
UPDATE enzyme SET history = 'EC 3.8.1.2 created 1972, modified 2003' WHERE ec = 'EC 3.8.1.2';
UPDATE enzyme SET sys_name = 'Haloacetate halidohydrolase' WHERE ec = 'EC 3.8.1.3';
UPDATE enzyme SET history = 'EC 3.8.1.3 created 1972' WHERE ec = 'EC 3.8.1.3';
UPDATE enzyme SET history = 'EC 3.8.1.4 created 1984, deleted 2003' WHERE ec = 'EC 3.8.1.4';
UPDATE enzyme SET sys_name = '1-haloalkane halidohydrolase' WHERE ec = 'EC 3.8.1.5';
UPDATE enzyme SET history = 'EC 3.8.1.5 created 1989' WHERE ec = 'EC 3.8.1.5';
UPDATE enzyme SET sys_name = '4-chlorobenzoate chlorohydrolase' WHERE ec = 'EC 3.8.1.6';
UPDATE enzyme SET history = 'EC 3.8.1.6 created 1989, modified 1999' WHERE ec = 'EC 3.8.1.6';
UPDATE enzyme SET sys_name = '4-chlorobenzoyl CoA chlorohydrolase' WHERE ec = 'EC 3.8.1.7';
UPDATE enzyme SET history = 'EC 3.8.1.7 created 1999' WHERE ec = 'EC 3.8.1.7';
UPDATE enzyme SET sys_name = 'Atrazine chlorohydrolase' WHERE ec = 'EC 3.8.1.8';
UPDATE enzyme SET history = 'EC 3.8.1.8 created 2000, modified 2011' WHERE ec = 'EC 3.8.1.8';
UPDATE enzyme SET sys_name = '(R)-2-haloacid halidohydrolase' WHERE ec = 'EC 3.8.1.9';
UPDATE enzyme SET history = 'EC 3.8.1.9 created 2003' WHERE ec = 'EC 3.8.1.9';
UPDATE enzyme SET sys_name = '(R,S)-2-haloacid dehalogenase (configuration-inverting)' WHERE ec = 'EC 3.8.1.10';
UPDATE enzyme SET history = 'EC 3.8.1.10 created 2003' WHERE ec = 'EC 3.8.1.10';
UPDATE enzyme SET sys_name = '(S)-2-haloacid dehalogenase (configuration-retaining)' WHERE ec = 'EC 3.8.1.11';
UPDATE enzyme SET history = 'EC 3.8.1.11 created 2003' WHERE ec = 'EC 3.8.1.11';
UPDATE enzyme SET history = 'EC 3.8.2.1 created 1961, modified 1976, deleted 1992' WHERE ec = 'EC 3.8.2.1';
UPDATE enzyme SET sys_name = 'Phosphamide hydrolase' WHERE ec = 'EC 3.9.1.1';
UPDATE enzyme SET history = 'EC 3.9.1.1 created 1961' WHERE ec = 'EC 3.9.1.1';
UPDATE enzyme SET sys_name = 'N-sulfo-D-glucosamine sulfohydrolase' WHERE ec = 'EC 3.10.1.1';
UPDATE enzyme SET history = 'EC 3.10.1.1 created 1972, modified 1981, modified 1982' WHERE ec = 'EC 3.10.1.1';
UPDATE enzyme SET sys_name = 'Cyclohexylsulfamate sulfohydrolase' WHERE ec = 'EC 3.10.1.2';
UPDATE enzyme SET history = 'EC 3.10.1.2 created 1976, modified 1981' WHERE ec = 'EC 3.10.1.2';
UPDATE enzyme SET sys_name = '2-oxoethylphosphonate phosphonohydrolase' WHERE ec = 'EC 3.11.1.1';
UPDATE enzyme SET history = 'EC 3.11.1.1 created 1972, modified 1976, modified 2001' WHERE ec = 'EC 3.11.1.1';
UPDATE enzyme SET sys_name = 'Phosphonoacetate phosphonohydrolase' WHERE ec = 'EC 3.11.1.2';
UPDATE enzyme SET history = 'EC 3.11.1.2 created 1999' WHERE ec = 'EC 3.11.1.2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 3.11.1.3';
UPDATE enzyme SET history = 'EC 3.11.1.3 created 2007' WHERE ec = 'EC 3.11.1.3';
UPDATE enzyme SET sys_name = 'Trithionate thiosulfohydrolase' WHERE ec = 'EC 3.12.1.1';
UPDATE enzyme SET history = 'EC 3.12.1.1 created 1990' WHERE ec = 'EC 3.12.1.1';
UPDATE enzyme SET sys_name = 'UDP-6-sulfo-6-deoxyglucose sulfohydrolase' WHERE ec = 'EC 3.13.1.1';
UPDATE enzyme SET history = 'EC 3.13.1.1 created 2001, modified 2010' WHERE ec = 'EC 3.13.1.1';
UPDATE enzyme SET history = 'EC 3.13.1.2 created 1972 as EC 3.3.1.3, transferred 2001 to EC 3.2.1.148, transferred 2004 to EC 3.13.1.2, deleted 2005' WHERE ec = 'EC 3.13.1.2';
UPDATE enzyme SET sys_name = '2-hydroxybiphenyl-2-sulfinate sulfohydrolase' WHERE ec = 'EC 3.13.1.3';
UPDATE enzyme SET history = 'EC 3.13.1.3 created 2000 as EC 3.1.2.24, transferred 2005 to EC 3.13.1.3' WHERE ec = 'EC 3.13.1.3';
UPDATE enzyme SET sys_name = '2-oxo-acid carboxy-lyase (aldehyde-forming)' WHERE ec = 'EC 4.1.1.1';
UPDATE enzyme SET history = 'EC 4.1.1.1 created 1961' WHERE ec = 'EC 4.1.1.1';
UPDATE enzyme SET sys_name = 'Oxalate carboxy-lyase (formate-forming)' WHERE ec = 'EC 4.1.1.2';
UPDATE enzyme SET history = 'EC 4.1.1.2 created 1961' WHERE ec = 'EC 4.1.1.2';
UPDATE enzyme SET sys_name = 'Oxaloacetate carboxy-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.1.3';
UPDATE enzyme SET history = 'EC 4.1.1.3 created 1961, modified 1986, modified 2000' WHERE ec = 'EC 4.1.1.3';
UPDATE enzyme SET sys_name = 'Acetoacetate carboxy-lyase (acetone-forming)' WHERE ec = 'EC 4.1.1.4';
UPDATE enzyme SET history = 'EC 4.1.1.4 created 1961' WHERE ec = 'EC 4.1.1.4';
UPDATE enzyme SET sys_name = '(2S)-2-hydroxy-2-methyl-3-oxobutanoate carboxy-lyase ((3R)-3-hydroxybutan-2-one-forming)' WHERE ec = 'EC 4.1.1.5';
UPDATE enzyme SET history = 'EC 4.1.1.5 created 1961' WHERE ec = 'EC 4.1.1.5';
UPDATE enzyme SET sys_name = 'Cis-aconitate carboxy-lyase (itaconate-forming)' WHERE ec = 'EC 4.1.1.6';
UPDATE enzyme SET history = 'EC 4.1.1.6 created 1961' WHERE ec = 'EC 4.1.1.6';
UPDATE enzyme SET sys_name = 'Benzoylformate carboxy-lyase (benzaldehyde-forming)' WHERE ec = 'EC 4.1.1.7';
UPDATE enzyme SET history = 'EC 4.1.1.7 created 1961' WHERE ec = 'EC 4.1.1.7';
UPDATE enzyme SET sys_name = 'Oxalyl-CoA carboxy-lyase (formyl-CoA-forming)' WHERE ec = 'EC 4.1.1.8';
UPDATE enzyme SET history = 'EC 4.1.1.8 created 1961' WHERE ec = 'EC 4.1.1.8';
UPDATE enzyme SET sys_name = 'Malonyl-CoA carboxy-lyase (acetyl-CoA-forming)' WHERE ec = 'EC 4.1.1.9';
UPDATE enzyme SET history = 'EC 4.1.1.9 created 1961, deleted 1972, reinstated 1978' WHERE ec = 'EC 4.1.1.9';
UPDATE enzyme SET history = 'EC 4.1.1.10 created 1961, deleted 1972' WHERE ec = 'EC 4.1.1.10';
UPDATE enzyme SET sys_name = 'L-aspartate 1-carboxy-lyase (beta-alanine-forming)' WHERE ec = 'EC 4.1.1.11';
UPDATE enzyme SET history = 'EC 4.1.1.11 created 1961, deleted 1972, reinstated 1984' WHERE ec = 'EC 4.1.1.11';
UPDATE enzyme SET sys_name = 'L-aspartate 4-carboxy-lyase (L-alanine-forming)' WHERE ec = 'EC 4.1.1.12';
UPDATE enzyme SET history = 'EC 4.1.1.12 created 1961, modified 1976 (EC 4.1.1.10 created 1961, incorporated 1972)' WHERE ec = 'EC 4.1.1.12';
UPDATE enzyme SET history = 'EC 4.1.1.13 created 1961, deleted 1972' WHERE ec = 'EC 4.1.1.13';
UPDATE enzyme SET sys_name = 'L-valine carboxy-lyase (2-methylpropanamine-forming)' WHERE ec = 'EC 4.1.1.14';
UPDATE enzyme SET history = 'EC 4.1.1.14 created 1961' WHERE ec = 'EC 4.1.1.14';
UPDATE enzyme SET sys_name = 'L-glutamate 1-carboxy-lyase (4-aminobutanoate-forming)' WHERE ec = 'EC 4.1.1.15';
UPDATE enzyme SET history = 'EC 4.1.1.15 created 1961' WHERE ec = 'EC 4.1.1.15';
UPDATE enzyme SET sys_name = '3-hydroxy-L-glutamate 1-carboxy-lyase (4-amino-3-hydroxybutanoate-forming)' WHERE ec = 'EC 4.1.1.16';
UPDATE enzyme SET history = 'EC 4.1.1.16 created 1961' WHERE ec = 'EC 4.1.1.16';
UPDATE enzyme SET sys_name = 'L-ornithine carboxy-lyase (putrescine-forming)' WHERE ec = 'EC 4.1.1.17';
UPDATE enzyme SET history = 'EC 4.1.1.17 created 1961' WHERE ec = 'EC 4.1.1.17';
UPDATE enzyme SET sys_name = 'L-lysine carboxy-lyase (cadaverine-forming)' WHERE ec = 'EC 4.1.1.18';
UPDATE enzyme SET history = 'EC 4.1.1.18 created 1961' WHERE ec = 'EC 4.1.1.18';
UPDATE enzyme SET sys_name = 'L-arginine carboxy-lyase (agmatine-forming)' WHERE ec = 'EC 4.1.1.19';
UPDATE enzyme SET history = 'EC 4.1.1.19 created 1961' WHERE ec = 'EC 4.1.1.19';
UPDATE enzyme SET sys_name = 'Meso-2,6-diaminoheptanedioate carboxy-lyase (L-lysine-forming)' WHERE ec = 'EC 4.1.1.20';
UPDATE enzyme SET history = 'EC 4.1.1.20 created 1961' WHERE ec = 'EC 4.1.1.20';
UPDATE enzyme SET sys_name = '5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate carboxy-lyase (5-amino-1-(5-phospho-D-ribosyl)imidazole-forming)' WHERE ec = 'EC 4.1.1.21';
UPDATE enzyme SET history = 'EC 4.1.1.21 created 1961, modified 2000, modified 2006' WHERE ec = 'EC 4.1.1.21';
UPDATE enzyme SET sys_name = 'L-histidine carboxy-lyase (histamine-forming)' WHERE ec = 'EC 4.1.1.22';
UPDATE enzyme SET history = 'EC 4.1.1.22 created 1961' WHERE ec = 'EC 4.1.1.22';
UPDATE enzyme SET sys_name = 'Orotidine-5-phosphate carboxy-lyase (UMP-forming)' WHERE ec = 'EC 4.1.1.23';
UPDATE enzyme SET history = 'EC 4.1.1.23 created 1961, modified 1986' WHERE ec = 'EC 4.1.1.23';
UPDATE enzyme SET sys_name = 'Aminobenzoate carboxy-lyase (aniline-forming)' WHERE ec = 'EC 4.1.1.24';
UPDATE enzyme SET history = 'EC 4.1.1.24 created 1961' WHERE ec = 'EC 4.1.1.24';
UPDATE enzyme SET sys_name = 'L-tyrosine carboxy-lyase (tyramine-forming)' WHERE ec = 'EC 4.1.1.25';
UPDATE enzyme SET history = 'EC 4.1.1.25 created 1961' WHERE ec = 'EC 4.1.1.25';
UPDATE enzyme SET history = 'EC 4.1.1.26 created 1961, deleted 1972' WHERE ec = 'EC 4.1.1.26';
UPDATE enzyme SET history = 'EC 4.1.1.27 created 1961, deleted 1972' WHERE ec = 'EC 4.1.1.27';
UPDATE enzyme SET sys_name = 'Aromatic-L-amino-acid carboxy-lyase' WHERE ec = 'EC 4.1.1.28';
UPDATE enzyme SET history = 'EC 4.1.1.28 created 1961 (EC 4.1.1.26 and EC 4.1.1.27 both created 1961 and incorporated 1972)' WHERE ec = 'EC 4.1.1.28';
UPDATE enzyme SET sys_name = '3-sulfino-L-alanine carboxy-lyase (hypotaurine-forming)' WHERE ec = 'EC 4.1.1.29';
UPDATE enzyme SET history = 'EC 4.1.1.29 created 1961, deleted 1972, reinstated 1976, modified 1983, modified 1999' WHERE ec = 'EC 4.1.1.29';
UPDATE enzyme SET sys_name = 'N-((R)-pantothenoyl)-L-cysteine carboxy-lyase (pantetheine-forming)' WHERE ec = 'EC 4.1.1.30';
UPDATE enzyme SET history = 'EC 4.1.1.30 created 1961' WHERE ec = 'EC 4.1.1.30';
UPDATE enzyme SET sys_name = 'Phosphate:oxaloacetate carboxy-lyase (adding phosphate; phosphoenolpyruvate-forming)' WHERE ec = 'EC 4.1.1.31';
UPDATE enzyme SET history = 'EC 4.1.1.31 created 1961, modified 2011' WHERE ec = 'EC 4.1.1.31';
UPDATE enzyme SET sys_name = 'GTP:oxaloacetate carboxy-lyase (adding GTP; phosphoenolpyruvate-forming)' WHERE ec = 'EC 4.1.1.32';
UPDATE enzyme SET history = 'EC 4.1.1.32 created 1961' WHERE ec = 'EC 4.1.1.32';
UPDATE enzyme SET sys_name = 'ATP:(R)-5-diphosphomevalonate carboxy-lyase (adding ATP; isopentenyl-diphosphate-forming)' WHERE ec = 'EC 4.1.1.33';
UPDATE enzyme SET history = 'EC 4.1.1.33 created 1961' WHERE ec = 'EC 4.1.1.33';
UPDATE enzyme SET sys_name = '3-dehydro-L-gulonate carboxy-lyase (L-xylulose-forming)' WHERE ec = 'EC 4.1.1.34';
UPDATE enzyme SET history = 'EC 4.1.1.34 created 1965' WHERE ec = 'EC 4.1.1.34';
UPDATE enzyme SET sys_name = 'UDP-D-glucuronate carboxy-lyase (UDP-D-xylose-forming)' WHERE ec = 'EC 4.1.1.35';
UPDATE enzyme SET history = 'EC 4.1.1.35 created 1965' WHERE ec = 'EC 4.1.1.35';
UPDATE enzyme SET sys_name = 'N-((R)-4-phosphopantothenoyl)-L-cysteine carboxy-lyase (pantotheine-4-phosphate-forming)' WHERE ec = 'EC 4.1.1.36';
UPDATE enzyme SET history = 'EC 4.1.1.36 created 1965' WHERE ec = 'EC 4.1.1.36';
UPDATE enzyme SET sys_name = 'Uroporphyrinogen-III carboxy-lyase (coproporphyrinogen-III-forming)' WHERE ec = 'EC 4.1.1.37';
UPDATE enzyme SET history = 'EC 4.1.1.37 created 1965' WHERE ec = 'EC 4.1.1.37';
UPDATE enzyme SET sys_name = 'Diphosphate:oxaloacetate carboxy-lyase (transphosphorylating; phosphoenolpyruvate-forming)' WHERE ec = 'EC 4.1.1.38';
UPDATE enzyme SET history = 'EC 4.1.1.38 created 1965' WHERE ec = 'EC 4.1.1.38';
UPDATE enzyme SET sys_name = '3-phospho-D-glycerate carboxy-lyase (dimerizing; D-ribulose-1,5-bisphosphate-forming)' WHERE ec = 'EC 4.1.1.39';
UPDATE enzyme SET history = 'EC 4.1.1.39 created 1965, modified 2001, modified 2003' WHERE ec = 'EC 4.1.1.39';
UPDATE enzyme SET sys_name = 'Hydroxypyruvate carboxy-lyase (glycolaldehyde-forming)' WHERE ec = 'EC 4.1.1.40';
UPDATE enzyme SET history = 'EC 4.1.1.40 created 1972' WHERE ec = 'EC 4.1.1.40';
UPDATE enzyme SET sys_name = '(S)-methylmalonyl-CoA carboxy-lyase (propanoyl-CoA-forming)' WHERE ec = 'EC 4.1.1.41';
UPDATE enzyme SET history = 'EC 4.1.1.41 created 1972, modified 1983, modified 1986, modified 2004' WHERE ec = 'EC 4.1.1.41';
UPDATE enzyme SET sys_name = 'Carnitine carboxy-lyase (2-methylcholine-forming)' WHERE ec = 'EC 4.1.1.42';
UPDATE enzyme SET history = 'EC 4.1.1.42 created 1972' WHERE ec = 'EC 4.1.1.42';
UPDATE enzyme SET sys_name = 'Phenylpyruvate carboxy-lyase (phenylacetaldehyde-forming)' WHERE ec = 'EC 4.1.1.43';
UPDATE enzyme SET history = 'EC 4.1.1.43 created 1972' WHERE ec = 'EC 4.1.1.43';
UPDATE enzyme SET sys_name = '(R)-2-carboxy-2,5-dihydro-5-oxofuran-2-acetate carboxy-lyase (4,5-dihydro-5-oxofuran-2-acetate-forming)' WHERE ec = 'EC 4.1.1.44';
UPDATE enzyme SET history = 'EC 4.1.1.44 created 1972' WHERE ec = 'EC 4.1.1.44';
UPDATE enzyme SET sys_name = '2-amino-3-(3-oxoprop-1-en-1-yl)but-2-enedioate carboxy-lyase (2-aminomuconate-semialdehyde-forming)' WHERE ec = 'EC 4.1.1.45';
UPDATE enzyme SET history = 'EC 4.1.1.45 created 1972' WHERE ec = 'EC 4.1.1.45';
UPDATE enzyme SET sys_name = '2,3-dihydroxybenzoate carboxy-lyase (catechol-forming)' WHERE ec = 'EC 4.1.1.46';
UPDATE enzyme SET history = 'EC 4.1.1.46 created 1972' WHERE ec = 'EC 4.1.1.46';
UPDATE enzyme SET sys_name = 'Glyoxylate carboxy-lyase (dimerizing; tartronate-semialdehyde-forming)' WHERE ec = 'EC 4.1.1.47';
UPDATE enzyme SET history = 'EC 4.1.1.47 created 1972' WHERE ec = 'EC 4.1.1.47';
UPDATE enzyme SET sys_name = '1-(2-carboxyphenylamino)-1-deoxy-D-ribulose-5-phosphate carboxy-lyase (cyclizing; 1-C-(indol-3-yl)glycerol-3-phosphate-forming)' WHERE ec = 'EC 4.1.1.48';
UPDATE enzyme SET history = 'EC 4.1.1.48 created 1972' WHERE ec = 'EC 4.1.1.48';
UPDATE enzyme SET sys_name = 'ATP:oxaloacetate carboxy-lyase (transphosphorylating; phosphoenolpyruvate-forming)' WHERE ec = 'EC 4.1.1.49';
UPDATE enzyme SET history = 'EC 4.1.1.49 created 1972' WHERE ec = 'EC 4.1.1.49';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine carboxy-lyase (S-adenosyl 3-(methylthio)propylamine-forming)' WHERE ec = 'EC 4.1.1.50';
UPDATE enzyme SET history = 'EC 4.1.1.50 created 1972' WHERE ec = 'EC 4.1.1.50';
UPDATE enzyme SET sys_name = '3-hydroxy-2-methylpyridine-4,5-dicarboxylate 4-carboxy-lyase (3-hydroxy-2-methylpyridine-5-carboxylate-forming)' WHERE ec = 'EC 4.1.1.51';
UPDATE enzyme SET history = 'EC 4.1.1.51 created 1972' WHERE ec = 'EC 4.1.1.51';
UPDATE enzyme SET sys_name = '6-methylsalicylate carboxy-lyase (3-methylphenol-forming)' WHERE ec = 'EC 4.1.1.52';
UPDATE enzyme SET history = 'EC 4.1.1.52 created 1972, modified 2011' WHERE ec = 'EC 4.1.1.52';
UPDATE enzyme SET sys_name = 'L-phenylalanine carboxy-lyase (phenylethylamine-forming)' WHERE ec = 'EC 4.1.1.53';
UPDATE enzyme SET history = 'EC 4.1.1.53 created 1972' WHERE ec = 'EC 4.1.1.53';
UPDATE enzyme SET sys_name = 'Dihydroxyfumarate carboxy-lyase (tartronate-semialdehyde-forming)' WHERE ec = 'EC 4.1.1.54';
UPDATE enzyme SET history = 'EC 4.1.1.54 created 1972' WHERE ec = 'EC 4.1.1.54';
UPDATE enzyme SET sys_name = '4,5-dihydroxyphthalate carboxy-lyase (3,4-dihydroxybenzoate-forming)' WHERE ec = 'EC 4.1.1.55';
UPDATE enzyme SET history = 'EC 4.1.1.55 created 1972' WHERE ec = 'EC 4.1.1.55';
UPDATE enzyme SET sys_name = '3-oxododecanoate carboxy-lyase (2-undecanone-forming)' WHERE ec = 'EC 4.1.1.56';
UPDATE enzyme SET history = 'EC 4.1.1.56 created 1972' WHERE ec = 'EC 4.1.1.56';
UPDATE enzyme SET sys_name = 'L-methionine carboxy-lyase (3-methylthiopropanamine-forming)' WHERE ec = 'EC 4.1.1.57';
UPDATE enzyme SET history = 'EC 4.1.1.57 created 1972' WHERE ec = 'EC 4.1.1.57';
UPDATE enzyme SET sys_name = '2,4-dihydroxy-6-methylbenzoate carboxy-lyase (orcinol-forming)' WHERE ec = 'EC 4.1.1.58';
UPDATE enzyme SET history = 'EC 4.1.1.58 created 1972' WHERE ec = 'EC 4.1.1.58';
UPDATE enzyme SET sys_name = '3,4,5-trihydroxybenzoate carboxy-lyase (pyrogallol-forming)' WHERE ec = 'EC 4.1.1.59';
UPDATE enzyme SET history = 'EC 4.1.1.59 created 1972' WHERE ec = 'EC 4.1.1.59';
UPDATE enzyme SET sys_name = 'Stipitatonate carboxy-lyase (decyclizing, stipitatate-forming)' WHERE ec = 'EC 4.1.1.60';
UPDATE enzyme SET history = 'EC 4.1.1.60 created 1972' WHERE ec = 'EC 4.1.1.60';
UPDATE enzyme SET sys_name = '4-hydroxybenzoate carboxy-lyase (phenol-forming)' WHERE ec = 'EC 4.1.1.61';
UPDATE enzyme SET history = 'EC 4.1.1.61 created 1972' WHERE ec = 'EC 4.1.1.61';
UPDATE enzyme SET sys_name = '2,5-dihydroxybenzoate carboxy-lyase (hydroquinone-forming)' WHERE ec = 'EC 4.1.1.62';
UPDATE enzyme SET history = 'EC 4.1.1.62 created 1972' WHERE ec = 'EC 4.1.1.62';
UPDATE enzyme SET sys_name = '3,4-dihydroxybenzoate carboxy-lyase (catechol-forming)' WHERE ec = 'EC 4.1.1.63';
UPDATE enzyme SET history = 'EC 4.1.1.63 created 1972' WHERE ec = 'EC 4.1.1.63';
UPDATE enzyme SET sys_name = '2,2-dialkylglycine carboxy-lyase (amino-transferring; L-alanine-forming)' WHERE ec = 'EC 4.1.1.64';
UPDATE enzyme SET history = 'EC 4.1.1.64 created 1972' WHERE ec = 'EC 4.1.1.64';
UPDATE enzyme SET sys_name = 'Phosphatidyl-L-serine carboxy-lyase (phosphatidylethanolamine-forming)' WHERE ec = 'EC 4.1.1.65';
UPDATE enzyme SET history = 'EC 4.1.1.65 created 1976' WHERE ec = 'EC 4.1.1.65';
UPDATE enzyme SET sys_name = 'Uracil-5-carboxylate carboxy-lyase (uracil-forming)' WHERE ec = 'EC 4.1.1.66';
UPDATE enzyme SET history = 'EC 4.1.1.66 created 1976' WHERE ec = 'EC 4.1.1.66';
UPDATE enzyme SET sys_name = 'UDP-D-galacturonate carboxy-lyase (UDP-L-arabinose-forming)' WHERE ec = 'EC 4.1.1.67';
UPDATE enzyme SET history = 'EC 4.1.1.67 created 1984' WHERE ec = 'EC 4.1.1.67';
UPDATE enzyme SET sys_name = '5-oxopent-3-ene-1,2,5-tricarboxylate carboxy-lyase (2-oxohept-3-enedioate-forming)' WHERE ec = 'EC 4.1.1.68';
UPDATE enzyme SET history = 'EC 4.1.1.68 created 1984' WHERE ec = 'EC 4.1.1.68';
UPDATE enzyme SET sys_name = '3,4-dihydroxyphthalate carboxy-lyase (3,4-dihydroxybenzoate-forming)' WHERE ec = 'EC 4.1.1.69';
UPDATE enzyme SET history = 'EC 4.1.1.69 created 1986' WHERE ec = 'EC 4.1.1.69';
UPDATE enzyme SET sys_name = '4-carboxybut-2-enoyl-CoA carboxy-lyase (but-2-enoyl-CoA-forming)' WHERE ec = 'EC 4.1.1.70';
UPDATE enzyme SET history = 'EC 4.1.1.70 created 1986, modified 2003' WHERE ec = 'EC 4.1.1.70';
UPDATE enzyme SET sys_name = '2-oxoglutarate carboxy-lyase (succinate-semialdehyde-forming)' WHERE ec = 'EC 4.1.1.71';
UPDATE enzyme SET history = 'EC 4.1.1.71 created 1989' WHERE ec = 'EC 4.1.1.71';
UPDATE enzyme SET sys_name = '(3S)-3-methyl-2-oxopentanoate carboxy-lyase (2-methylbutanal-forming)' WHERE ec = 'EC 4.1.1.72';
UPDATE enzyme SET history = 'EC 4.1.1.72 created 1990' WHERE ec = 'EC 4.1.1.72';
UPDATE enzyme SET sys_name = '(R,R)-tartrate carboxy-lyase (D-glycerate-forming)' WHERE ec = 'EC 4.1.1.73';
UPDATE enzyme SET history = 'EC 4.1.1.73 created 1992' WHERE ec = 'EC 4.1.1.73';
UPDATE enzyme SET sys_name = '3-(indol-3-yl)pyruvate carboxy-lyase ((2-indol-3-yl)acetaldehyde-forming)' WHERE ec = 'EC 4.1.1.74';
UPDATE enzyme SET history = 'EC 4.1.1.74 created 1999' WHERE ec = 'EC 4.1.1.74';
UPDATE enzyme SET sys_name = '5-guanidino-2-oxo-pentanoate carboxy-lyase (4-guanidinobutanal-forming)' WHERE ec = 'EC 4.1.1.75';
UPDATE enzyme SET history = 'EC 4.1.1.75 created 1999' WHERE ec = 'EC 4.1.1.75';
UPDATE enzyme SET sys_name = '2-aryl-2-methylmalonate carboxy-lyase (2-arylpropanoate-forming)' WHERE ec = 'EC 4.1.1.76';
UPDATE enzyme SET history = 'EC 4.1.1.76 created 1999' WHERE ec = 'EC 4.1.1.76';
UPDATE enzyme SET sys_name = '(3E)-2-oxohex-3-enedioate carboxy-lyase (2-oxopent-4-enoate-forming)' WHERE ec = 'EC 4.1.1.77';
UPDATE enzyme SET history = 'EC 4.1.1.77 created 1999, modified 2011, modified 2012' WHERE ec = 'EC 4.1.1.77';
UPDATE enzyme SET sys_name = 'Acetylenedicarboxylate carboxy-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.1.78';
UPDATE enzyme SET history = 'EC 4.1.1.78 created 1978 as EC 4.2.1.72, transferred 2000 to EC 4.1.1.78' WHERE ec = 'EC 4.1.1.78';
UPDATE enzyme SET sys_name = '3-sulfopyruvate carboxy-lyase (2-sulfoacetaldehyde-forming)' WHERE ec = 'EC 4.1.1.79';
UPDATE enzyme SET history = 'EC 4.1.1.79 created 2002' WHERE ec = 'EC 4.1.1.79';
UPDATE enzyme SET sys_name = '4-hydroxyphenylpyruvate carboxy-lyase (4-hydroxyphenylacetaldehyde-forming)' WHERE ec = 'EC 4.1.1.80';
UPDATE enzyme SET history = 'EC 4.1.1.80 created 2002' WHERE ec = 'EC 4.1.1.80';
UPDATE enzyme SET sys_name = 'L-threonine-O-3-phosphate carboxy-lyase ((R)-1-aminopropan-2-yl-phosphate-forming)' WHERE ec = 'EC 4.1.1.81';
UPDATE enzyme SET history = 'EC 4.1.1.81 created 2004' WHERE ec = 'EC 4.1.1.81';
UPDATE enzyme SET sys_name = '3-phosphonopyruvate carboxy-lyase (2-phosphonoacetaldehyde-forming)' WHERE ec = 'EC 4.1.1.82';
UPDATE enzyme SET history = 'EC 4.1.1.82 created 2005' WHERE ec = 'EC 4.1.1.82';
UPDATE enzyme SET sys_name = '4-(hydroxyphenyl)acetate carboxy-lyase (4-methylphenol-forming)' WHERE ec = 'EC 4.1.1.83';
UPDATE enzyme SET history = 'EC 4.1.1.83 created 2005' WHERE ec = 'EC 4.1.1.83';
UPDATE enzyme SET sys_name = 'D-dopachrome carboxy-lyase (5,6-dihydroxyindole-forming)' WHERE ec = 'EC 4.1.1.84';
UPDATE enzyme SET history = 'EC 4.1.1.84 created 2005' WHERE ec = 'EC 4.1.1.84';
UPDATE enzyme SET sys_name = '3-dehydro-L-gulonate-6-phosphate carboxy-lyase (L-xylulose-5-phosphate-forming)' WHERE ec = 'EC 4.1.1.85';
UPDATE enzyme SET history = 'EC 4.1.1.85 created 2005' WHERE ec = 'EC 4.1.1.85';
UPDATE enzyme SET sys_name = 'L-2,4-diaminobutanoate carboxy-lyase (propane-1,3-diamine-forming)' WHERE ec = 'EC 4.1.1.86';
UPDATE enzyme SET history = 'EC 4.1.1.86 created 2006' WHERE ec = 'EC 4.1.1.86';
UPDATE enzyme SET sys_name = 'Malonyl-[acyl-carrier-protein] carboxy-lyase' WHERE ec = 'EC 4.1.1.87';
UPDATE enzyme SET history = 'EC 4.1.1.87 created 2008' WHERE ec = 'EC 4.1.1.87';
UPDATE enzyme SET sys_name = 'Malonate carboxy-lyase (biotin-independent)' WHERE ec = 'EC 4.1.1.88';
UPDATE enzyme SET history = 'EC 4.1.1.88 created 2008' WHERE ec = 'EC 4.1.1.88';
UPDATE enzyme SET sys_name = 'Malonate carboxy-lyase (biotin-dependent)' WHERE ec = 'EC 4.1.1.89';
UPDATE enzyme SET history = 'EC 4.1.1.89 created 2008' WHERE ec = 'EC 4.1.1.89';
UPDATE enzyme SET sys_name = 'Peptidyl-glutamate 4-carboxylase (2-methyl-3-phytyl-1,4-naphthoquinone-epoxidizing)' WHERE ec = 'EC 4.1.1.90';
UPDATE enzyme SET history = 'EC 4.1.1.90 created 2009' WHERE ec = 'EC 4.1.1.90';
UPDATE enzyme SET sys_name = 'Salicylate carboxy-lyase' WHERE ec = 'EC 4.1.1.91';
UPDATE enzyme SET history = 'EC 4.1.1.91 created 2011' WHERE ec = 'EC 4.1.1.91';
UPDATE enzyme SET sys_name = 'Indole-3-carboxylate carboxy-lyase' WHERE ec = 'EC 4.1.1.92';
UPDATE enzyme SET history = 'EC 4.1.1.92 created 2011' WHERE ec = 'EC 4.1.1.92';
UPDATE enzyme SET sys_name = 'Pyrrole-2-carboxylate carboxy-lyase' WHERE ec = 'EC 4.1.1.93';
UPDATE enzyme SET history = 'EC 4.1.1.93 created 2011' WHERE ec = 'EC 4.1.1.93';
UPDATE enzyme SET sys_name = '(S)-ethylmalonyl-CoA carboxy-lyase (butanoyl-CoA-forming)' WHERE ec = 'EC 4.1.1.94';
UPDATE enzyme SET history = 'EC 4.1.1.94 created 2012' WHERE ec = 'EC 4.1.1.94';
UPDATE enzyme SET sys_name = 'L-glutamyl-[BtrI acyl-carrier protein] carboxy-lyase' WHERE ec = 'EC 4.1.1.95';
UPDATE enzyme SET history = 'EC 4.1.1.95 created 2012' WHERE ec = 'EC 4.1.1.95';
UPDATE enzyme SET sys_name = 'Carboxynorspermidine carboxy-lyase (bis(3-aminopropyl)amine-forming)' WHERE ec = 'EC 4.1.1.96';
UPDATE enzyme SET history = 'EC 4.1.1.96 created 2012' WHERE ec = 'EC 4.1.1.96';
UPDATE enzyme SET sys_name = '5-hydroxy-2-oxo-4-ureido-2,5-dihydro-1H-imidazole-5-carboxylate carboxy-lyase (S-allantoin-forming)' WHERE ec = 'EC 4.1.1.97';
UPDATE enzyme SET history = 'EC 4.1.1.97 created 2014' WHERE ec = 'EC 4.1.1.97';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.1.1.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.1.1.n2';
UPDATE enzyme SET history = 'EC 4.1.2.1 created 1961, deleted 1972' WHERE ec = 'EC 4.1.2.1';
UPDATE enzyme SET sys_name = 'Erythrulose-1-phosphate formaldehyde-lyase (glycerone-phosphate-forming)' WHERE ec = 'EC 4.1.2.2';
UPDATE enzyme SET history = 'EC 4.1.2.2 created 1961' WHERE ec = 'EC 4.1.2.2';
UPDATE enzyme SET history = 'EC 4.1.2.3 created 1961, deleted 1972' WHERE ec = 'EC 4.1.2.3';
UPDATE enzyme SET sys_name = '2-deoxy-D-ribose-5-phosphate acetaldehyde-lyase (D-glyceraldehyde-3-phosphate-forming)' WHERE ec = 'EC 4.1.2.4';
UPDATE enzyme SET history = 'EC 4.1.2.4 created 1961' WHERE ec = 'EC 4.1.2.4';
UPDATE enzyme SET sys_name = 'L-threonine acetaldehyde-lyase (glycine-forming)' WHERE ec = 'EC 4.1.2.5';
UPDATE enzyme SET history = 'EC 4.1.2.5 created 1961, deleted 1972, reinstated 1976, modified 2011' WHERE ec = 'EC 4.1.2.5';
UPDATE enzyme SET history = 'EC 4.1.2.6 created 1961, deleted 1972' WHERE ec = 'EC 4.1.2.6';
UPDATE enzyme SET history = 'EC 4.1.2.7 created 1961, deleted 1972' WHERE ec = 'EC 4.1.2.7';
UPDATE enzyme SET sys_name = '(1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate D-glyceraldehyde-3-phosphate-lyase (indole-forming)' WHERE ec = 'EC 4.1.2.8';
UPDATE enzyme SET history = 'EC 4.1.2.8 created 1961, deleted 1972, reinstated 2006' WHERE ec = 'EC 4.1.2.8';
UPDATE enzyme SET sys_name = 'D-xylulose-5-phosphate D-glyceraldehyde-3-phosphate-lyase (adding phosphate; acetyl-phosphate-forming)' WHERE ec = 'EC 4.1.2.9';
UPDATE enzyme SET history = 'EC 4.1.2.9 created 1961' WHERE ec = 'EC 4.1.2.9';
UPDATE enzyme SET sys_name = '(R)-mandelonitrile benzaldehyde-lyase (cyanide-forming)' WHERE ec = 'EC 4.1.2.10';
UPDATE enzyme SET history = 'EC 4.1.2.10 created 1961, modified 1999, modified 2010' WHERE ec = 'EC 4.1.2.10';
UPDATE enzyme SET sys_name = '(S)-4-hydroxymandelonitrile 4-hydroxybenzaldehyde-lyase (cyanide-forming)' WHERE ec = 'EC 4.1.2.11';
UPDATE enzyme SET history = 'EC 4.1.2.11 created 1965, modified 1999' WHERE ec = 'EC 4.1.2.11';
UPDATE enzyme SET sys_name = '2-dehydropantoate formaldehyde-lyase (3-methyl-2-oxobutanoate-forming)' WHERE ec = 'EC 4.1.2.12';
UPDATE enzyme SET history = 'EC 4.1.2.12 created 1965, modified 2002' WHERE ec = 'EC 4.1.2.12';
UPDATE enzyme SET sys_name = 'D-fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming)' WHERE ec = 'EC 4.1.2.13';
UPDATE enzyme SET history = 'EC 4.1.2.13 created 1965, modified 1999 (EC 4.1.2.7 created 1961, incorporated 1972)' WHERE ec = 'EC 4.1.2.13';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-6-phosphate-D-gluconate D-glyceraldehyde-3-phosphate-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.14';
UPDATE enzyme SET history = 'EC 4.1.2.14 created 1965, modified 1976, modified 2014' WHERE ec = 'EC 4.1.2.14';
UPDATE enzyme SET history = 'EC 4.1.2.15 created 1965, modified 1976, deleted 2002' WHERE ec = 'EC 4.1.2.15';
UPDATE enzyme SET history = 'EC 4.1.2.16 created 1965, deleted 2002' WHERE ec = 'EC 4.1.2.16';
UPDATE enzyme SET sys_name = 'L-fuculose-1-phosphate (S)-lactaldehyde-lyase (glycerone-phosphate-forming)' WHERE ec = 'EC 4.1.2.17';
UPDATE enzyme SET history = 'EC 4.1.2.17 created 1965' WHERE ec = 'EC 4.1.2.17';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-L-pentonate glycolaldehyde-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.18';
UPDATE enzyme SET history = 'EC 4.1.2.18 created 1972, modified 1976' WHERE ec = 'EC 4.1.2.18';
UPDATE enzyme SET sys_name = 'L-rhamnulose-1-phosphate (S)-lactaldehyde-lyase (glycerone-phosphate-forming)' WHERE ec = 'EC 4.1.2.19';
UPDATE enzyme SET history = 'EC 4.1.2.19 created 1972' WHERE ec = 'EC 4.1.2.19';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-D-glucarate tartronate-semialdehyde-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.20';
UPDATE enzyme SET history = 'EC 4.1.2.20 created 1961 as EC 4.1.2.8, transferred 1972 to EC 4.1.2.20' WHERE ec = 'EC 4.1.2.20';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-6-phospho-D-galactonate D-glyceraldehyde-3-phosphate-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.21';
UPDATE enzyme SET history = 'EC 4.1.2.21 created 1972, modified 2014' WHERE ec = 'EC 4.1.2.21';
UPDATE enzyme SET sys_name = 'D-fructose-6-phosphate D-erythrose-4-phosphate-lyase (adding phosphate; acetyl-phosphate-forming)' WHERE ec = 'EC 4.1.2.22';
UPDATE enzyme SET history = 'EC 4.1.2.22 created 1972' WHERE ec = 'EC 4.1.2.22';
UPDATE enzyme SET sys_name = '3-deoxy-D-manno-octulosonate D-arabinose-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.23';
UPDATE enzyme SET history = 'EC 4.1.2.23 created 1972' WHERE ec = 'EC 4.1.2.23';
UPDATE enzyme SET sys_name = 'N,N-dimethylaniline-N-oxide formaldehyde-lyase (N-methylaniline-forming)' WHERE ec = 'EC 4.1.2.24';
UPDATE enzyme SET history = 'EC 4.1.2.24 created 1972' WHERE ec = 'EC 4.1.2.24';
UPDATE enzyme SET sys_name = '2-amino-4-hydroxy-6-(D-erythro-1,2,3-trihydroxypropyl)-7,8-dihydropteridine glycolaldehyde-lyase (2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine-forming)' WHERE ec = 'EC 4.1.2.25';
UPDATE enzyme SET history = 'EC 4.1.2.25 created 1972' WHERE ec = 'EC 4.1.2.25';
UPDATE enzyme SET sys_name = 'L-threo-3-phenylserine benzaldehyde-lyase (glycine-forming)' WHERE ec = 'EC 4.1.2.26';
UPDATE enzyme SET history = 'EC 4.1.2.26 created 1972' WHERE ec = 'EC 4.1.2.26';
UPDATE enzyme SET sys_name = 'Sphinganine-1-phosphate palmitaldehyde-lyase (phosphoethanolamine-forming)' WHERE ec = 'EC 4.1.2.27';
UPDATE enzyme SET history = 'EC 4.1.2.27 created 1972' WHERE ec = 'EC 4.1.2.27';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-D-pentonate glycolaldehyde-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.28';
UPDATE enzyme SET history = 'EC 4.1.2.28 created 1976' WHERE ec = 'EC 4.1.2.28';
UPDATE enzyme SET sys_name = '5-dehydro-2-deoxy-D-gluconate-6-phosphate malonate-semialdehyde-lyase (glycerone-phosphate-forming)' WHERE ec = 'EC 4.1.2.29';
UPDATE enzyme SET history = 'EC 4.1.2.29 created 1976' WHERE ec = 'EC 4.1.2.29';
UPDATE enzyme SET sys_name = '17-alpha-hydroxyprogesterone acetaldehyde-lyase (4-androstene-3,17-dione-forming)' WHERE ec = 'EC 4.1.2.30';
UPDATE enzyme SET history = 'EC 4.1.2.30 created 1976' WHERE ec = 'EC 4.1.2.30';
UPDATE enzyme SET history = 'EC 4.1.2.31 created 1978, deleted 1982' WHERE ec = 'EC 4.1.2.31';
UPDATE enzyme SET sys_name = 'Trimethylamine-N-oxide formaldehyde-lyase (dimethylamine-forming)' WHERE ec = 'EC 4.1.2.32';
UPDATE enzyme SET history = 'EC 4.1.2.32 created 1978' WHERE ec = 'EC 4.1.2.32';
UPDATE enzyme SET sys_name = '(24R,24(1)R)-fucosterol-epoxide acetaldehyde-lyase (desmosterol-forming)' WHERE ec = 'EC 4.1.2.33';
UPDATE enzyme SET history = 'EC 4.1.2.33 created 1989, modified 2013' WHERE ec = 'EC 4.1.2.33';
UPDATE enzyme SET sys_name = '(3Z)-4-(2-carboxyphenyl)-2-oxobut-3-enoate 2-formylbenzoate-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.34';
UPDATE enzyme SET history = 'EC 4.1.2.34 created 1989' WHERE ec = 'EC 4.1.2.34';
UPDATE enzyme SET sys_name = '4-hydroxy-3-hexanone propanal-lyase (propanal-forming)' WHERE ec = 'EC 4.1.2.35';
UPDATE enzyme SET history = 'EC 4.1.2.35 created 1990' WHERE ec = 'EC 4.1.2.35';
UPDATE enzyme SET sys_name = '(S)-lactate acetaldehyde-lyase (formate-forming)' WHERE ec = 'EC 4.1.2.36';
UPDATE enzyme SET history = 'EC 4.1.2.36 created 1990' WHERE ec = 'EC 4.1.2.36';
UPDATE enzyme SET history = 'EC 4.1.2.37 created 1992 (EC 4.1.2.39 created 1999, incorporated 2007), deleted 2011' WHERE ec = 'EC 4.1.2.37';
UPDATE enzyme SET sys_name = '2-hydroxy-1,2-diphenylethanone benzaldehyde-lyase (benzaldehyde-forming)' WHERE ec = 'EC 4.1.2.38';
UPDATE enzyme SET history = 'EC 4.1.2.38 created 1992' WHERE ec = 'EC 4.1.2.38';
UPDATE enzyme SET history = 'EC 4.1.2.39 created 1999, deleted 2007' WHERE ec = 'EC 4.1.2.39';
UPDATE enzyme SET sys_name = 'D-tagatose 1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming)' WHERE ec = 'EC 4.1.2.40';
UPDATE enzyme SET history = 'EC 4.1.2.40 created 1999' WHERE ec = 'EC 4.1.2.40';
UPDATE enzyme SET sys_name = '3-hydroxy-3-(4-hydroxy-3-methoxyphenyl)propanoyl-CoA vanillin-lyase (acetyl-CoA-forming)' WHERE ec = 'EC 4.1.2.41';
UPDATE enzyme SET history = 'EC 4.1.2.41 created 2000' WHERE ec = 'EC 4.1.2.41';
UPDATE enzyme SET sys_name = 'D-threonine acetaldehyde-lyase (glycine-forming)' WHERE ec = 'EC 4.1.2.42';
UPDATE enzyme SET history = 'EC 4.1.2.42 created 2007' WHERE ec = 'EC 4.1.2.42';
UPDATE enzyme SET sys_name = 'D-arabino-hex-3-ulose-6-phosphate formaldehyde-lyase (D-ribulose-5-phosphate-forming)' WHERE ec = 'EC 4.1.2.43';
UPDATE enzyme SET history = 'EC 4.1.2.43 created 2008' WHERE ec = 'EC 4.1.2.43';
UPDATE enzyme SET sys_name = '2,3-dihydro-2,3-dihydroxybenzoyl-CoA 3,4-didehydroadipyl-CoA-semialdehyde-lyase (formate-forming)' WHERE ec = 'EC 4.1.2.44';
UPDATE enzyme SET history = 'EC 4.1.2.44 created 2010' WHERE ec = 'EC 4.1.2.44';
UPDATE enzyme SET sys_name = '(3E)-4-(2-hydroxyphenyl)-2-oxobut-3-enoate hydro-lyase' WHERE ec = 'EC 4.1.2.45';
UPDATE enzyme SET history = 'EC 4.1.2.45 created 2010' WHERE ec = 'EC 4.1.2.45';
UPDATE enzyme SET sys_name = '(2R)-2-hydroxy-2-methylbutanenitrile butan-2-one-lyase (cyanide forming)' WHERE ec = 'EC 4.1.2.46';
UPDATE enzyme SET history = 'EC 4.1.2.46 created 2010' WHERE ec = 'EC 4.1.2.46';
UPDATE enzyme SET sys_name = '(S)-cyanohydrin lyase (cyanide forming)' WHERE ec = 'EC 4.1.2.47';
UPDATE enzyme SET history = 'EC 4.1.2.47 created 2010' WHERE ec = 'EC 4.1.2.47';
UPDATE enzyme SET sys_name = 'L-threonine/L-allo-threonine acetaldehyde-lyase (glycine-forming)' WHERE ec = 'EC 4.1.2.48';
UPDATE enzyme SET history = 'EC 4.1.2.48 created 2011' WHERE ec = 'EC 4.1.2.48';
UPDATE enzyme SET sys_name = 'L-allo-threonine acetaldehyde-lyase (glycine-forming)' WHERE ec = 'EC 4.1.2.49';
UPDATE enzyme SET history = 'EC 4.1.2.49 created 2011' WHERE ec = 'EC 4.1.2.49';
UPDATE enzyme SET sys_name = '7,8-dihydroneopterin 3-triphosphate acetaldehyde-lyase (6-carboxy-5,6,7,8-tetrahydropterin and triphosphate-forming)' WHERE ec = 'EC 4.1.2.50';
UPDATE enzyme SET history = 'EC 4.1.2.50 created 2012' WHERE ec = 'EC 4.1.2.50';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-D-gluconate D-glyceraldehyde-3-phosphate-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.51';
UPDATE enzyme SET history = 'EC 4.1.2.51 created 2013' WHERE ec = 'EC 4.1.2.51';
UPDATE enzyme SET sys_name = '4-hydroxy-2-oxoheptanedioate succinate-semialdehyde-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.52';
UPDATE enzyme SET history = 'EC 4.1.2.52 created 2013' WHERE ec = 'EC 4.1.2.52';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-L-rhamnonate (S)-lactaldehyde-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.53';
UPDATE enzyme SET history = 'EC 4.1.2.53 created 2013' WHERE ec = 'EC 4.1.2.53';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-L-galactonate L-glyceraldehyde-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.54';
UPDATE enzyme SET history = 'EC 4.1.2.54 created 2013' WHERE ec = 'EC 4.1.2.54';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-6-phosphate-D-gluconate/2-dehydro-3-deoxy-6-phosphate-D-galactonate D-glyceraldehyde-3-phosphate-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.2.55';
UPDATE enzyme SET history = 'EC 4.1.2.55 created 2014' WHERE ec = 'EC 4.1.2.55';
UPDATE enzyme SET sys_name = '2-amino-4,5-dihydroxy-6-oxo-7-(phosphoxy)heptanoate L-aspartate 4-semialdehyde-lyase (glycerone phosphate-forming)' WHERE ec = 'EC 4.1.2.56';
UPDATE enzyme SET history = 'EC 4.1.2.56 created 2014' WHERE ec = 'EC 4.1.2.56';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.1.2.n1';
UPDATE enzyme SET sys_name = '2-hydroxyphytanoyl-CoA pristanal-lyase (formyl-CoA-forming)' WHERE ec = 'EC 4.1.2.n2';
UPDATE enzyme SET history = 'EC 4.1.2.n2 created 2010' WHERE ec = 'EC 4.1.2.n2';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.1.2.n3';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.1.2.n4';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.1.2.n5';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.1.2.n6';
UPDATE enzyme SET sys_name = 'Isocitrate glyoxylate-lyase (succinate-forming)' WHERE ec = 'EC 4.1.3.1';
UPDATE enzyme SET history = 'EC 4.1.3.1 created 1961' WHERE ec = 'EC 4.1.3.1';
UPDATE enzyme SET history = 'EC 4.1.3.2 created 1961, deleted 2002' WHERE ec = 'EC 4.1.3.2';
UPDATE enzyme SET sys_name = 'N-acetylneuraminate pyruvate-lyase (N-acetyl-D-mannosamine-forming)' WHERE ec = 'EC 4.1.3.3';
UPDATE enzyme SET history = 'EC 4.1.3.3 created 1961' WHERE ec = 'EC 4.1.3.3';
UPDATE enzyme SET sys_name = '(S)-3-hydroxy-3-methylglutaryl-CoA acetoacetate-lyase (acetyl-CoA-forming)' WHERE ec = 'EC 4.1.3.4';
UPDATE enzyme SET history = 'EC 4.1.3.4 created 1961' WHERE ec = 'EC 4.1.3.4';
UPDATE enzyme SET history = 'EC 4.1.3.5 created 1961, deleted 2002' WHERE ec = 'EC 4.1.3.5';
UPDATE enzyme SET sys_name = 'Citrate oxaloacetate-lyase (forming acetate from the pro-S carboxymethyl group of citrate)' WHERE ec = 'EC 4.1.3.6';
UPDATE enzyme SET history = 'EC 4.1.3.6 created 1961' WHERE ec = 'EC 4.1.3.6';
UPDATE enzyme SET history = 'EC 4.1.3.7 created 1961, deleted 2002' WHERE ec = 'EC 4.1.3.7';
UPDATE enzyme SET history = 'EC 4.1.3.8 created 1965, modified 1986, deleted 2002' WHERE ec = 'EC 4.1.3.8';
UPDATE enzyme SET history = 'EC 4.1.3.9 created 1965, deleted 2002' WHERE ec = 'EC 4.1.3.9';
UPDATE enzyme SET history = 'EC 4.1.3.10 created 1965, modified 1983, deleted 2002' WHERE ec = 'EC 4.1.3.10';
UPDATE enzyme SET history = 'EC 4.1.3.11 created 1972, deleted 2002' WHERE ec = 'EC 4.1.3.11';
UPDATE enzyme SET history = 'EC 4.1.3.12 created 1972, deleted 2002' WHERE ec = 'EC 4.1.3.12';
UPDATE enzyme SET sys_name = '3-oxalomalate glyoxylate-lyase (oxaloacetate-forming)' WHERE ec = 'EC 4.1.3.13';
UPDATE enzyme SET history = 'EC 4.1.3.13 created 1972' WHERE ec = 'EC 4.1.3.13';
UPDATE enzyme SET sys_name = 'L-erythro-3-hydroxy-aspartate glyoxylate-lyase (glycine-forming)' WHERE ec = 'EC 4.1.3.14';
UPDATE enzyme SET history = 'EC 4.1.3.14 created 1972, modified 2010' WHERE ec = 'EC 4.1.3.14';
UPDATE enzyme SET history = 'EC 4.1.3.15 created 1972, deleted 2002' WHERE ec = 'EC 4.1.3.15';
UPDATE enzyme SET sys_name = '4-hydroxy-2-oxoglutarate glyoxylate-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.3.16';
UPDATE enzyme SET history = 'EC 4.1.3.16 created 1972 (EC 4.1.2.1 created 1961, incorporated 1972, EC 4.1.2.31 created 1978, incorporated 1982)' WHERE ec = 'EC 4.1.3.16';
UPDATE enzyme SET sys_name = '4-hydroxy-4-methyl-2-oxoglutarate pyruvate-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.3.17';
UPDATE enzyme SET history = 'EC 4.1.3.17 created 1972, modified 2012' WHERE ec = 'EC 4.1.3.17';
UPDATE enzyme SET history = 'EC 4.1.3.18 created 1972, deleted 2002' WHERE ec = 'EC 4.1.3.18';
UPDATE enzyme SET history = 'EC 4.1.3.19 created 1972, deleted 2002' WHERE ec = 'EC 4.1.3.19';
UPDATE enzyme SET history = 'EC 4.1.3.20 created 1972, deleted 2002' WHERE ec = 'EC 4.1.3.20';
UPDATE enzyme SET history = 'EC 4.1.3.21 created 1972, deleted 2002' WHERE ec = 'EC 4.1.3.21';
UPDATE enzyme SET sys_name = '(2S)-2-hydroxy-2-methylbutanedioate pyruvate-lyase (acetate-forming)' WHERE ec = 'EC 4.1.3.22';
UPDATE enzyme SET history = 'EC 4.1.3.22 created 1972' WHERE ec = 'EC 4.1.3.22';
UPDATE enzyme SET history = 'EC 4.1.3.23 created 1972, deleted 2002' WHERE ec = 'EC 4.1.3.23';
UPDATE enzyme SET sys_name = '(S)-malyl-CoA glyoxylate-lyase (acetyl-CoA-forming)' WHERE ec = 'EC 4.1.3.24';
UPDATE enzyme SET history = 'EC 4.1.3.24 created 1972, modified 2014' WHERE ec = 'EC 4.1.3.24';
UPDATE enzyme SET sys_name = '(3S)-citramalyl-CoA pyruvate-lyase (acetyl-CoA-forming)' WHERE ec = 'EC 4.1.3.25';
UPDATE enzyme SET history = 'EC 4.1.3.25 created 1972, modified 2014' WHERE ec = 'EC 4.1.3.25';
UPDATE enzyme SET sys_name = '3-hydroxy-3-(4-methylpent-3-en-1-yl)glutaryl-CoA acetate-lyase (7-methyl-3-oxooct-6-enoyl-CoA-forming)' WHERE ec = 'EC 4.1.3.26';
UPDATE enzyme SET history = 'EC 4.1.3.26 created 1972' WHERE ec = 'EC 4.1.3.26';
UPDATE enzyme SET sys_name = 'Chorismate pyruvate-lyase (amino-accepting; anthranilate-forming)' WHERE ec = 'EC 4.1.3.27';
UPDATE enzyme SET history = 'EC 4.1.3.27 created 1972' WHERE ec = 'EC 4.1.3.27';
UPDATE enzyme SET history = 'EC 4.1.3.28 created 1972, deleted 2002' WHERE ec = 'EC 4.1.3.28';
UPDATE enzyme SET history = 'EC 4.1.3.29 created 1976, deleted 2002' WHERE ec = 'EC 4.1.3.29';
UPDATE enzyme SET sys_name = '(2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate pyruvate-lyase (succinate-forming)' WHERE ec = 'EC 4.1.3.30';
UPDATE enzyme SET history = 'EC 4.1.3.30 created 1978' WHERE ec = 'EC 4.1.3.30';
UPDATE enzyme SET history = 'EC 4.1.3.31 created 1978, deleted 2002' WHERE ec = 'EC 4.1.3.31';
UPDATE enzyme SET sys_name = '(2R,3S)-2,3-dimethylmalate pyruvate-lyase (propanoate-forming)' WHERE ec = 'EC 4.1.3.32';
UPDATE enzyme SET history = 'EC 4.1.3.32 created 1981' WHERE ec = 'EC 4.1.3.32';
UPDATE enzyme SET history = 'EC 4.1.3.33 created 1983, deleted 2002' WHERE ec = 'EC 4.1.3.33';
UPDATE enzyme SET sys_name = '(3S)-citryl-CoA oxaloacetate-lyase (acetyl-CoA-forming)' WHERE ec = 'EC 4.1.3.34';
UPDATE enzyme SET history = 'EC 4.1.3.34 created 1984, modified 1986' WHERE ec = 'EC 4.1.3.34';
UPDATE enzyme SET sys_name = '(1-hydroxycyclohexan-1-yl)acetyl-CoA cyclohexanone-lyase (acetyl-CoA-forming)' WHERE ec = 'EC 4.1.3.35';
UPDATE enzyme SET history = 'EC 4.1.3.35 created 1986' WHERE ec = 'EC 4.1.3.35';
UPDATE enzyme SET sys_name = '4-(2-carboxyphenyl)-4-oxobutanoyl-CoA dihydratase (cyclizing)' WHERE ec = 'EC 4.1.3.36';
UPDATE enzyme SET history = 'EC 4.1.3.36 created 1992, modified 2010' WHERE ec = 'EC 4.1.3.36';
UPDATE enzyme SET history = 'EC 4.1.3.37 created 2001, deleted 2002' WHERE ec = 'EC 4.1.3.37';
UPDATE enzyme SET sys_name = '4-amino-4-deoxychorismate pyruvate-lyase (4-aminobenzoate-forming)' WHERE ec = 'EC 4.1.3.38';
UPDATE enzyme SET history = 'EC 4.1.3.38 created 2003' WHERE ec = 'EC 4.1.3.38';
UPDATE enzyme SET sys_name = '(S)-4-hydroxy-2-oxopentanoate pyruvate-lyase (acetaldehyde-forming)' WHERE ec = 'EC 4.1.3.39';
UPDATE enzyme SET history = 'EC 4.1.3.39 created 2006, modified 2011' WHERE ec = 'EC 4.1.3.39';
UPDATE enzyme SET sys_name = 'Chorismate pyruvate-lyase (4-hydroxybenzoate-forming)' WHERE ec = 'EC 4.1.3.40';
UPDATE enzyme SET history = 'EC 4.1.3.40 created 2007' WHERE ec = 'EC 4.1.3.40';
UPDATE enzyme SET sys_name = '3-hydroxy-D-aspartate glyoxylate-lyase (glycine-forming)' WHERE ec = 'EC 4.1.3.41';
UPDATE enzyme SET history = 'EC 4.1.3.41 created 2010' WHERE ec = 'EC 4.1.3.41';
UPDATE enzyme SET sys_name = '(4S)-4-hydroxy-2-oxoglutarate glyoxylate-lyase (pyruvate-forming)' WHERE ec = 'EC 4.1.3.42';
UPDATE enzyme SET history = 'EC 4.1.3.42 created 2013' WHERE ec = 'EC 4.1.3.42';
UPDATE enzyme SET sys_name = '(S)-4-hydroxy-2-oxohexanoate pyruvate-lyase (propanal-forming)' WHERE ec = 'EC 4.1.3.43';
UPDATE enzyme SET history = 'EC 4.1.3.43 created 2013' WHERE ec = 'EC 4.1.3.43';
UPDATE enzyme SET sys_name = 'tRNA(Phe) N(1)-methylguanine,pyruvate acetaldehyde-lyase (tRNA(Phe) 4-demethylwyosine-forming, decarboxylating, dehydrating)' WHERE ec = 'EC 4.1.3.44';
UPDATE enzyme SET history = 'EC 4.1.3.44 created 2013' WHERE ec = 'EC 4.1.3.44';
UPDATE enzyme SET sys_name = 'Chorismate pyruvate-lyase (3-hydroxybenzoate-forming)' WHERE ec = 'EC 4.1.3.45';
UPDATE enzyme SET history = 'EC 4.1.3.45 created 2013' WHERE ec = 'EC 4.1.3.45';
UPDATE enzyme SET sys_name = '(3R)-citramalyl-CoA pyruvate-lyase (acetyl-CoA-forming)' WHERE ec = 'EC 4.1.3.46';
UPDATE enzyme SET history = 'EC 4.1.3.46 created 2014' WHERE ec = 'EC 4.1.3.46';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.1.3.n1';
UPDATE enzyme SET sys_name = 'L-tryptophan indole-lyase (deaminating; pyruvate-forming)' WHERE ec = 'EC 4.1.99.1';
UPDATE enzyme SET history = 'EC 4.1.99.1 created 1972' WHERE ec = 'EC 4.1.99.1';
UPDATE enzyme SET sys_name = 'L-tyrosine phenol-lyase (deaminating; pyruvate-forming)' WHERE ec = 'EC 4.1.99.2';
UPDATE enzyme SET history = 'EC 4.1.99.2 created 1972' WHERE ec = 'EC 4.1.99.2';
UPDATE enzyme SET sys_name = 'Deoxyribocyclobutadipyrimidine pyrimidine-lyase' WHERE ec = 'EC 4.1.99.3';
UPDATE enzyme SET history = 'EC 4.1.99.3 created 1972' WHERE ec = 'EC 4.1.99.3';
UPDATE enzyme SET history = 'EC 4.1.99.4 created 1981, deleted 2002' WHERE ec = 'EC 4.1.99.4';
UPDATE enzyme SET sys_name = 'A long-chain aldehyde alkane-lyase' WHERE ec = 'EC 4.1.99.5';
UPDATE enzyme SET history = 'EC 4.1.99.5 created 1989, modified 2011, modified 2013' WHERE ec = 'EC 4.1.99.5';
UPDATE enzyme SET history = 'EC 4.1.99.6 created 1989, deleted 2000' WHERE ec = 'EC 4.1.99.6';
UPDATE enzyme SET history = 'EC 4.1.99.7 created 1992 as EC 2.5.1.40, transferred 1999 to EC 4.1.99.7, deleted 2000' WHERE ec = 'EC 4.1.99.7';
UPDATE enzyme SET history = 'EC 4.1.99.8 created 2000, deleted 2000' WHERE ec = 'EC 4.1.99.8';
UPDATE enzyme SET history = 'EC 4.1.99.9 created 2000, deleted 2000' WHERE ec = 'EC 4.1.99.9';
UPDATE enzyme SET history = 'EC 4.1.99.10 created 2000, deleted 2000' WHERE ec = 'EC 4.1.99.10';
UPDATE enzyme SET sys_name = 'Benzylsuccinate fumarate-lyase (toluene-forming)' WHERE ec = 'EC 4.1.99.11';
UPDATE enzyme SET history = 'EC 4.1.99.11 created 2000' WHERE ec = 'EC 4.1.99.11';
UPDATE enzyme SET sys_name = 'D-ribulose 5-phosphate formate-lyase (L-3,4-dihydroxybutan-2-one 4-phosphate-forming)' WHERE ec = 'EC 4.1.99.12';
UPDATE enzyme SET history = 'EC 4.1.99.12 created 2007' WHERE ec = 'EC 4.1.99.12';
UPDATE enzyme SET sys_name = '(6-4) photoproduct pyrimidine-lyase' WHERE ec = 'EC 4.1.99.13';
UPDATE enzyme SET history = 'EC 4.1.99.13 created 2009' WHERE ec = 'EC 4.1.99.13';
UPDATE enzyme SET sys_name = 'Spore photoproduct pyrimidine-lyase' WHERE ec = 'EC 4.1.99.14';
UPDATE enzyme SET history = 'EC 4.1.99.14 created 2009, modified 2010' WHERE ec = 'EC 4.1.99.14';
UPDATE enzyme SET history = 'EC 4.1.99.15 created 2009, deleted 2010' WHERE ec = 'EC 4.1.99.15';
UPDATE enzyme SET sys_name = 'Germacradienol geosmin-lyase (acetone forming)' WHERE ec = 'EC 4.1.99.16';
UPDATE enzyme SET history = 'EC 4.1.99.16 created 2011' WHERE ec = 'EC 4.1.99.16';
UPDATE enzyme SET sys_name = '5-amino-1-(5-phospho-D-ribosyl)imidazole formate-lyase (decarboxylating, 4-amino-2-methyl-5-phosphomethylpyrimidine-forming)' WHERE ec = 'EC 4.1.99.17';
UPDATE enzyme SET history = 'EC 4.1.99.17 created 2011' WHERE ec = 'EC 4.1.99.17';
UPDATE enzyme SET sys_name = 'GTP 8,9-lyase (cyclic pyranopterin phosphate-forming)' WHERE ec = 'EC 4.1.99.18';
UPDATE enzyme SET history = 'EC 4.1.99.18 created 2011' WHERE ec = 'EC 4.1.99.18';
UPDATE enzyme SET sys_name = 'L-tyrosine 4-methylphenol-lyase (2-iminoacetate-forming)' WHERE ec = 'EC 4.1.99.19';
UPDATE enzyme SET history = 'EC 4.1.99.19 created 2011' WHERE ec = 'EC 4.1.99.19';
UPDATE enzyme SET sys_name = '2-amino-4,5-dihydroxy-6-oxo-7-(phosphonooxy)heptanoate hydro-lyase (cyclizing, 3-amino-4-hydroxybenzoate-forming)' WHERE ec = 'EC 4.1.99.20';
UPDATE enzyme SET history = 'EC 4.1.99.20 created 2013' WHERE ec = 'EC 4.1.99.20';
UPDATE enzyme SET sys_name = 'Carbonate hydro-lyase (carbon-dioxide-forming)' WHERE ec = 'EC 4.2.1.1';
UPDATE enzyme SET history = 'EC 4.2.1.1 created 1961' WHERE ec = 'EC 4.2.1.1';
UPDATE enzyme SET sys_name = '(S)-malate hydro-lyase (fumarate-forming)' WHERE ec = 'EC 4.2.1.2';
UPDATE enzyme SET history = 'EC 4.2.1.2 created 1961' WHERE ec = 'EC 4.2.1.2';
UPDATE enzyme SET sys_name = 'Citrate(isocitrate) hydro-lyase (cis-aconitate-forming)' WHERE ec = 'EC 4.2.1.3';
UPDATE enzyme SET history = 'EC 4.2.1.3 created 1961, modified 2003' WHERE ec = 'EC 4.2.1.3';
UPDATE enzyme SET history = 'EC 4.2.1.4 created 1961, deleted 2013' WHERE ec = 'EC 4.2.1.4';
UPDATE enzyme SET sys_name = 'D-arabinonate hydro-lyase (2-dehydro-3-deoxy-D-arabinonate-forming)' WHERE ec = 'EC 4.2.1.5';
UPDATE enzyme SET history = 'EC 4.2.1.5 created 1961' WHERE ec = 'EC 4.2.1.5';
UPDATE enzyme SET sys_name = 'D-galactonate hydro-lyase (2-dehydro-3-deoxy-D-galactonate-forming)' WHERE ec = 'EC 4.2.1.6';
UPDATE enzyme SET history = 'EC 4.2.1.6 created 1961' WHERE ec = 'EC 4.2.1.6';
UPDATE enzyme SET sys_name = 'D-altronate hydro-lyase (2-dehydro-3-deoxy-D-gluconate-forming)' WHERE ec = 'EC 4.2.1.7';
UPDATE enzyme SET history = 'EC 4.2.1.7 created 1961, deleted 1972, reinstated 1976' WHERE ec = 'EC 4.2.1.7';
UPDATE enzyme SET sys_name = 'D-mannonate hydro-lyase (2-dehydro-3-deoxy-D-gluconate-forming)' WHERE ec = 'EC 4.2.1.8';
UPDATE enzyme SET history = 'EC 4.2.1.8 created 1961, modified 1976' WHERE ec = 'EC 4.2.1.8';
UPDATE enzyme SET sys_name = '2,3-dihydroxy-methylbutanoate hydro-lyase (3-methyl-2-oxobutanoate-forming)' WHERE ec = 'EC 4.2.1.9';
UPDATE enzyme SET history = 'EC 4.2.1.9 created 1961' WHERE ec = 'EC 4.2.1.9';
UPDATE enzyme SET sys_name = '3-dehydroquinate hydro-lyase (3-dehydroshikimate-forming)' WHERE ec = 'EC 4.2.1.10';
UPDATE enzyme SET history = 'EC 4.2.1.10 created 1961, modified 1976' WHERE ec = 'EC 4.2.1.10';
UPDATE enzyme SET sys_name = '2-phospho-D-glycerate hydro-lyase (phosphoenolpyruvate-forming)' WHERE ec = 'EC 4.2.1.11';
UPDATE enzyme SET history = 'EC 4.2.1.11 created 1961' WHERE ec = 'EC 4.2.1.11';
UPDATE enzyme SET sys_name = '6-phospho-D-gluconate hydro-lyase (2-dehydro-3-deoxy-6-phospho-D-gluconate-forming)' WHERE ec = 'EC 4.2.1.12';
UPDATE enzyme SET history = 'EC 4.2.1.12 created 1961' WHERE ec = 'EC 4.2.1.12';
UPDATE enzyme SET history = 'EC 4.2.1.13 created 1961, deleted 2001' WHERE ec = 'EC 4.2.1.13';
UPDATE enzyme SET history = 'EC 4.2.1.14 created 1961, deleted 2001' WHERE ec = 'EC 4.2.1.14';
UPDATE enzyme SET history = 'EC 4.2.1.15 created 1961, deleted 1972' WHERE ec = 'EC 4.2.1.15';
UPDATE enzyme SET history = 'EC 4.2.1.16 created 1961, deleted 2001' WHERE ec = 'EC 4.2.1.16';
UPDATE enzyme SET sys_name = '(3S)-3-hydroxyacyl-CoA hydro-lyase' WHERE ec = 'EC 4.2.1.17';
UPDATE enzyme SET history = 'EC 4.2.1.17 created 1961' WHERE ec = 'EC 4.2.1.17';
UPDATE enzyme SET sys_name = '(S)-3-hydroxy-3-methylglutaryl-CoA hydro-lyase (trans-3-methylglutaconyl-CoA-forming)' WHERE ec = 'EC 4.2.1.18';
UPDATE enzyme SET history = 'EC 4.2.1.18 created 1961' WHERE ec = 'EC 4.2.1.18';
UPDATE enzyme SET sys_name = 'D-erythro-1-(imidazol-4-yl)glycerol-3-phosphate hydro-lyase (3-(imidazol-4-yl)-2-oxopropyl-phosphate-forming)' WHERE ec = 'EC 4.2.1.19';
UPDATE enzyme SET history = 'EC 4.2.1.19 created 1961' WHERE ec = 'EC 4.2.1.19';
UPDATE enzyme SET sys_name = 'L-serine hydro-lyase (adding 1-C-(indol-3-yl)glycerol 3-phosphate; L-tryptophan and glyceraldehyde-3-phosphate-forming)' WHERE ec = 'EC 4.2.1.20';
UPDATE enzyme SET history = 'EC 4.2.1.20 created 1961, modified 1976, modified 2002, modified 2011' WHERE ec = 'EC 4.2.1.20';
UPDATE enzyme SET history = 'EC 4.2.1.21 created 1961, deleted 1964' WHERE ec = 'EC 4.2.1.21';
UPDATE enzyme SET sys_name = 'L-serine hydro-lyase (adding homocysteine; L-cystathionine-forming)' WHERE ec = 'EC 4.2.1.22';
UPDATE enzyme SET history = 'EC 4.2.1.22 created 1961 (EC 4.2.1.21 created 1961, incorporated 1964, EC 4.2.1.23 created 1961, incorporated 1972)' WHERE ec = 'EC 4.2.1.22';
UPDATE enzyme SET history = 'EC 4.2.1.23 created 1961, deleted 1972' WHERE ec = 'EC 4.2.1.23';
UPDATE enzyme SET sys_name = '5-aminolevulinate hydro-lyase (adding 5-aminolevulinate and cyclizing; porphobilinogen-forming)' WHERE ec = 'EC 4.2.1.24';
UPDATE enzyme SET history = 'EC 4.2.1.24 created 1961' WHERE ec = 'EC 4.2.1.24';
UPDATE enzyme SET sys_name = 'L-arabinonate hydro-lyase (2-dehydro-3-deoxy-L-arabinonate-forming)' WHERE ec = 'EC 4.2.1.25';
UPDATE enzyme SET history = 'EC 4.2.1.25 created 1965' WHERE ec = 'EC 4.2.1.25';
UPDATE enzyme SET history = 'EC 4.2.1.26 created 1965, deleted 2002' WHERE ec = 'EC 4.2.1.26';
UPDATE enzyme SET sys_name = '3-oxopropanoate hydro-lyase (propynoate-forming)' WHERE ec = 'EC 4.2.1.27';
UPDATE enzyme SET history = 'EC 4.2.1.27 created 1965, (EC 4.2.1.71 created 1978, modified 1989, modified 2000, incorporated 2004) modified 2004' WHERE ec = 'EC 4.2.1.27';
UPDATE enzyme SET sys_name = 'Propane-1,2-diol hydro-lyase (propanal-forming)' WHERE ec = 'EC 4.2.1.28';
UPDATE enzyme SET history = 'EC 4.2.1.28 created 1965' WHERE ec = 'EC 4.2.1.28';
UPDATE enzyme SET history = 'EC 4.2.1.29 created 1965, deleted 2004' WHERE ec = 'EC 4.2.1.29';
UPDATE enzyme SET sys_name = 'Glycerol hydro-lyase (3-hydroxypropanal-forming)' WHERE ec = 'EC 4.2.1.30';
UPDATE enzyme SET history = 'EC 4.2.1.30 created 1972' WHERE ec = 'EC 4.2.1.30';
UPDATE enzyme SET sys_name = '(R)-malate hydro-lyase (maleate-forming)' WHERE ec = 'EC 4.2.1.31';
UPDATE enzyme SET history = 'EC 4.2.1.31 created 1972' WHERE ec = 'EC 4.2.1.31';
UPDATE enzyme SET sys_name = '(R,R)-tartrate hydro-lyase (oxaloacetate-forming)' WHERE ec = 'EC 4.2.1.32';
UPDATE enzyme SET history = 'EC 4.2.1.32 created 1972, modified 1986' WHERE ec = 'EC 4.2.1.32';
UPDATE enzyme SET sys_name = '(2R,3S)-3-isopropylmalate hydro-lyase (2-isopropylmaleate-forming)' WHERE ec = 'EC 4.2.1.33';
UPDATE enzyme SET history = 'EC 4.2.1.33 created 1972, modified 1976, modified 2012' WHERE ec = 'EC 4.2.1.33';
UPDATE enzyme SET sys_name = '(S)-2-methylmalate hydro-lyase (2-methylfumarate-forming)' WHERE ec = 'EC 4.2.1.34';
UPDATE enzyme SET history = 'EC 4.2.1.34 created 1972' WHERE ec = 'EC 4.2.1.34';
UPDATE enzyme SET sys_name = '(R)-2-methylmalate hydro-lyase (2-methylmaleate-forming)' WHERE ec = 'EC 4.2.1.35';
UPDATE enzyme SET history = 'EC 4.2.1.35 created 1972' WHERE ec = 'EC 4.2.1.35';
UPDATE enzyme SET sys_name = '(1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate hydro-lyase ((Z)-but-1-ene-1,2,4-tricarboxylate-forming)' WHERE ec = 'EC 4.2.1.36';
UPDATE enzyme SET history = 'EC 4.2.1.36 created 1972, modified 2007' WHERE ec = 'EC 4.2.1.36';
UPDATE enzyme SET history = 'EC 4.2.1.37 created 1972, deleted 1992' WHERE ec = 'EC 4.2.1.37';
UPDATE enzyme SET history = 'EC 4.2.1.38 created 1972, deleted 2001' WHERE ec = 'EC 4.2.1.38';
UPDATE enzyme SET sys_name = 'D-gluconate hydro-lyase (2-dehydro-3-deoxy-D-gluconate-forming)' WHERE ec = 'EC 4.2.1.39';
UPDATE enzyme SET history = 'EC 4.2.1.39 created 1972' WHERE ec = 'EC 4.2.1.39';
UPDATE enzyme SET sys_name = 'D-glucarate hydro-lyase (5-dehydro-4-deoxy-D-glucarate-forming)' WHERE ec = 'EC 4.2.1.40';
UPDATE enzyme SET history = 'EC 4.2.1.40 created 1972' WHERE ec = 'EC 4.2.1.40';
UPDATE enzyme SET sys_name = '5-dehydro-4-deoxy-D-glucarate hydro-lyase (decarboxylating; 2,5-dioxopentanoate-forming)' WHERE ec = 'EC 4.2.1.41';
UPDATE enzyme SET history = 'EC 4.2.1.41 created 1972' WHERE ec = 'EC 4.2.1.41';
UPDATE enzyme SET sys_name = 'D-galactarate hydro-lyase (5-dehydro-4-deoxy-D-glucarate-forming)' WHERE ec = 'EC 4.2.1.42';
UPDATE enzyme SET history = 'EC 4.2.1.42 created 1972' WHERE ec = 'EC 4.2.1.42';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-L-arabinonate hydro-lyase (2,5-dioxopentanoate-forming)' WHERE ec = 'EC 4.2.1.43';
UPDATE enzyme SET history = 'EC 4.2.1.43 created 1972' WHERE ec = 'EC 4.2.1.43';
UPDATE enzyme SET sys_name = '2,4,6/3,5-pentahydroxycyclohexanone hydro-lyase (3,5/4-trihydroxycyclohexa-1,2-dione-forming)' WHERE ec = 'EC 4.2.1.44';
UPDATE enzyme SET history = 'EC 4.2.1.44 created 1972' WHERE ec = 'EC 4.2.1.44';
UPDATE enzyme SET sys_name = 'CDP-glucose 4,6-hydro-lyase (CDP-4-dehydro-6-deoxy-D-glucose-forming)' WHERE ec = 'EC 4.2.1.45';
UPDATE enzyme SET history = 'EC 4.2.1.45 created 1972' WHERE ec = 'EC 4.2.1.45';
UPDATE enzyme SET sys_name = 'dTDP-alpha-D-glucose 4,6-hydro-lyase (dTDP-4-dehydro-6-deoxy-D-glucose-forming)' WHERE ec = 'EC 4.2.1.46';
UPDATE enzyme SET history = 'EC 4.2.1.46 created 1972' WHERE ec = 'EC 4.2.1.46';
UPDATE enzyme SET sys_name = 'GDP-alpha-D-mannose 4,6-hydro-lyase (GDP-4-dehydro-6-deoxy-alpha-D-mannose-forming)' WHERE ec = 'EC 4.2.1.47';
UPDATE enzyme SET history = 'EC 4.2.1.47 created 1972, modified 2004' WHERE ec = 'EC 4.2.1.47';
UPDATE enzyme SET sys_name = 'D-glutamate hydro-lyase (cyclizing; 5-oxo-D-proline-forming)' WHERE ec = 'EC 4.2.1.48';
UPDATE enzyme SET history = 'EC 4.2.1.48 created 1972' WHERE ec = 'EC 4.2.1.48';
UPDATE enzyme SET sys_name = '3-(5-oxo-4,5-dihydro-3H-imidazol-4-yl)propanoate hydro-lyase (urocanate-forming)' WHERE ec = 'EC 4.2.1.49';
UPDATE enzyme SET history = 'EC 4.2.1.49 created 1972, modified 2001' WHERE ec = 'EC 4.2.1.49';
UPDATE enzyme SET sys_name = 'L-serine hydro-lyase (adding pyrazole; 3-(pyrazol-1-yl)-L-alanine-forming)' WHERE ec = 'EC 4.2.1.50';
UPDATE enzyme SET history = 'EC 4.2.1.50 created 1972' WHERE ec = 'EC 4.2.1.50';
UPDATE enzyme SET sys_name = 'Prephenate hydro-lyase (decarboxylating; phenylpyruvate-forming)' WHERE ec = 'EC 4.2.1.51';
UPDATE enzyme SET history = 'EC 4.2.1.51 created 1972' WHERE ec = 'EC 4.2.1.51';
UPDATE enzyme SET history = 'EC 4.2.1.52 created 1972, deleted 2012' WHERE ec = 'EC 4.2.1.52';
UPDATE enzyme SET sys_name = '(R)-10-hydroxystearate 10-hydro-lyase (oleate-forming)' WHERE ec = 'EC 4.2.1.53';
UPDATE enzyme SET history = 'EC 4.2.1.53 created 1972' WHERE ec = 'EC 4.2.1.53';
UPDATE enzyme SET sys_name = '(R)-lactoyl-CoA hydro-lyase (acryloyl-CoA-forming)' WHERE ec = 'EC 4.2.1.54';
UPDATE enzyme SET history = 'EC 4.2.1.54 created 1972, modified 2012' WHERE ec = 'EC 4.2.1.54';
UPDATE enzyme SET sys_name = '(3R)-3-hydroxybutanoyl-CoA hydro-lyase (crotonoyl-CoA-forming)' WHERE ec = 'EC 4.2.1.55';
UPDATE enzyme SET history = 'EC 4.2.1.55 created 1972' WHERE ec = 'EC 4.2.1.55';
UPDATE enzyme SET sys_name = 'Citramalyl-CoA hydro-lyase (itaconyl-CoA-forming)' WHERE ec = 'EC 4.2.1.56';
UPDATE enzyme SET history = 'EC 4.2.1.56 created 1972' WHERE ec = 'EC 4.2.1.56';
UPDATE enzyme SET sys_name = '3-hydroxy-3-(4-methylpent-3-en-1-yl)glutaryl-CoA hydro-lyase (3-(4-methylpent-3-en-1-yl)pent-2-enedioyl-CoA-forming)' WHERE ec = 'EC 4.2.1.57';
UPDATE enzyme SET history = 'EC 4.2.1.57 created 1972' WHERE ec = 'EC 4.2.1.57';
UPDATE enzyme SET history = 'EC 4.2.1.58 created 1972, deleted 2012' WHERE ec = 'EC 4.2.1.58';
UPDATE enzyme SET sys_name = '(3R)-3-hydroxyoctanoyl-[acyl-carrier protein] hydro-lyase (oct-2-enoyl-[acyl-carrier protein]-forming)' WHERE ec = 'EC 4.2.1.59';
UPDATE enzyme SET history = 'EC 4.2.1.59 created 1972, modified 2012' WHERE ec = 'EC 4.2.1.59';
UPDATE enzyme SET history = 'EC 4.2.1.60 created 1972, modified 2006, deleted 2012' WHERE ec = 'EC 4.2.1.60';
UPDATE enzyme SET history = 'EC 4.2.1.61 created 1972, deleted 2012' WHERE ec = 'EC 4.2.1.61';
UPDATE enzyme SET sys_name = '5-alpha-ergosta-7,22-diene-3-beta,5-diol 5,6-hydro-lyase (ergosterol-forming)' WHERE ec = 'EC 4.2.1.62';
UPDATE enzyme SET history = 'EC 4.2.1.62 created 1972' WHERE ec = 'EC 4.2.1.62';
UPDATE enzyme SET history = 'EC 4.2.1.63 created 1972, deleted 1978' WHERE ec = 'EC 4.2.1.63';
UPDATE enzyme SET history = 'EC 4.2.1.64 created 1972, deleted 1978' WHERE ec = 'EC 4.2.1.64';
UPDATE enzyme SET sys_name = 'L-asparagine hydro-lyase (3-cyanoalanine-forming)' WHERE ec = 'EC 4.2.1.65';
UPDATE enzyme SET history = 'EC 4.2.1.65 created 1976' WHERE ec = 'EC 4.2.1.65';
UPDATE enzyme SET sys_name = 'Formamide hydro-lyase (cyanide-forming)' WHERE ec = 'EC 4.2.1.66';
UPDATE enzyme SET history = 'EC 4.2.1.66 created 1976' WHERE ec = 'EC 4.2.1.66';
UPDATE enzyme SET sys_name = 'D-fuconate hydro-lyase (2-dehydro-3-deoxy-D-fuconate-forming)' WHERE ec = 'EC 4.2.1.67';
UPDATE enzyme SET history = 'EC 4.2.1.67 created 1976' WHERE ec = 'EC 4.2.1.67';
UPDATE enzyme SET sys_name = 'L-fuconate hydro-lyase (2-dehydro-3-deoxy-L-fuconate-forming)' WHERE ec = 'EC 4.2.1.68';
UPDATE enzyme SET history = 'EC 4.2.1.68 created 1976' WHERE ec = 'EC 4.2.1.68';
UPDATE enzyme SET sys_name = 'Urea hydro-lyase (cyanamide-forming)' WHERE ec = 'EC 4.2.1.69';
UPDATE enzyme SET history = 'EC 4.2.1.69 created 1976' WHERE ec = 'EC 4.2.1.69';
UPDATE enzyme SET sys_name = 'Uracil hydro-lyase (adding D-ribose 5-phosphate; pseudouridine-5-phosphate-forming)' WHERE ec = 'EC 4.2.1.70';
UPDATE enzyme SET history = 'EC 4.2.1.70 created 1978' WHERE ec = 'EC 4.2.1.70';
UPDATE enzyme SET history = 'EC 4.2.1.71 created 1978, modified 1989, modified 2000, deleted 2004' WHERE ec = 'EC 4.2.1.71';
UPDATE enzyme SET history = 'EC 4.2.1.72 created 1978, deleted 2000' WHERE ec = 'EC 4.2.1.72';
UPDATE enzyme SET sys_name = 'Protoaphin-aglucone hydro-lyase (cyclizing; xanthoaphin-forming)' WHERE ec = 'EC 4.2.1.73';
UPDATE enzyme SET history = 'EC 4.2.1.73 created 1978' WHERE ec = 'EC 4.2.1.73';
UPDATE enzyme SET sys_name = 'Long-chain-(3S)-3-hydroxyacyl-CoA hydro-lyase' WHERE ec = 'EC 4.2.1.74';
UPDATE enzyme SET history = 'EC 4.2.1.74 created 1981' WHERE ec = 'EC 4.2.1.74';
UPDATE enzyme SET sys_name = 'Hydroxymethylbilane hydro-lyase (cyclizing; uroporphyrinogen-III-forming)' WHERE ec = 'EC 4.2.1.75';
UPDATE enzyme SET history = 'EC 4.2.1.75 created 1982' WHERE ec = 'EC 4.2.1.75';
UPDATE enzyme SET sys_name = 'UDP-glucose 4,6-hydro-lyase (UDP-4-dehydro-6-deoxy-D-glucose-forming)' WHERE ec = 'EC 4.2.1.76';
UPDATE enzyme SET history = 'EC 4.2.1.76 created 1984' WHERE ec = 'EC 4.2.1.76';
UPDATE enzyme SET sys_name = 'Trans-3-hydroxy-Lproline hydro-lyase (1-pyrroline-2-carboxylate-forming)' WHERE ec = 'EC 4.2.1.77';
UPDATE enzyme SET history = 'EC 4.2.1.77 created 1984' WHERE ec = 'EC 4.2.1.77';
UPDATE enzyme SET sys_name = '4-hydroxyphenylacetaldehyde hydro-lyase (adding dopamine; (S)-norcoclaurine-forming)' WHERE ec = 'EC 4.2.1.78';
UPDATE enzyme SET history = 'EC 4.2.1.78 created 1984, modified 1999' WHERE ec = 'EC 4.2.1.78';
UPDATE enzyme SET sys_name = '(2S,3S)-2-hydroxybutane-1,2,3-tricarboxylate hydro-lyase ((Z)-but-2-ene-1,2,3-tricarboxylate-forming)' WHERE ec = 'EC 4.2.1.79';
UPDATE enzyme SET history = 'EC 4.2.1.79 created 1984, modified 2007' WHERE ec = 'EC 4.2.1.79';
UPDATE enzyme SET sys_name = '4-hydroxy-2-oxopentanoate hydro-lyase (2-oxopent-4-enoate-forming)' WHERE ec = 'EC 4.2.1.80';
UPDATE enzyme SET history = 'EC 4.2.1.80 created 1984' WHERE ec = 'EC 4.2.1.80';
UPDATE enzyme SET sys_name = '(S,S)-tartrate hydro-lyase (oxaloacetate-forming)' WHERE ec = 'EC 4.2.1.81';
UPDATE enzyme SET history = 'EC 4.2.1.81 created 1986' WHERE ec = 'EC 4.2.1.81';
UPDATE enzyme SET sys_name = 'D-xylonate hydro-lyase (2-dehydro-3-deoxy-D-xylonate-forming)' WHERE ec = 'EC 4.2.1.82';
UPDATE enzyme SET history = 'EC 4.2.1.82 created 1986' WHERE ec = 'EC 4.2.1.82';
UPDATE enzyme SET sys_name = '(1E,3E)-4-hydroxybuta-1,3-diene-1,2,4-tricarboxylate 1,2-hydro-lyase (2-hydroxy-4-oxobutane-1,2,4-tricarboxylate-forming)' WHERE ec = 'EC 4.2.1.83';
UPDATE enzyme SET history = 'EC 4.2.1.83 created 1986, modified 2011' WHERE ec = 'EC 4.2.1.83';
UPDATE enzyme SET sys_name = 'Aliphatic-amide hydro-lyase (nitrile-forming)' WHERE ec = 'EC 4.2.1.84';
UPDATE enzyme SET history = 'EC 4.2.1.84 created 1989' WHERE ec = 'EC 4.2.1.84';
UPDATE enzyme SET sys_name = '(2R,3S)-2,3-dimethylmalate hydro-lyase (dimethylmaleate-forming)' WHERE ec = 'EC 4.2.1.85';
UPDATE enzyme SET history = 'EC 4.2.1.85 created 1989' WHERE ec = 'EC 4.2.1.85';
UPDATE enzyme SET history = 'EC 4.2.1.86 created 1989, deleted 2004' WHERE ec = 'EC 4.2.1.86';
UPDATE enzyme SET sys_name = '1-(4-hydroxyphenyl)-2-aminoethanol hydro-lyase (deaminating; (4-hydroxyphenyl)acetaldehyde-forming)' WHERE ec = 'EC 4.2.1.87';
UPDATE enzyme SET history = 'EC 4.2.1.87 created 1989' WHERE ec = 'EC 4.2.1.87';
UPDATE enzyme SET sys_name = '(R)-synephrine hydro-lyase (methylamine-forming)' WHERE ec = 'EC 4.2.1.88';
UPDATE enzyme SET history = 'EC 4.2.1.88 created 1989, modified 2012' WHERE ec = 'EC 4.2.1.88';
UPDATE enzyme SET history = 'EC 4.2.1.89 created 1989, deleted 2014' WHERE ec = 'EC 4.2.1.89';
UPDATE enzyme SET sys_name = 'L-rhamnonate hydro-lyase (2-dehydro-3-deoxy-L-rhamnonate-forming)' WHERE ec = 'EC 4.2.1.90';
UPDATE enzyme SET history = 'EC 4.2.1.90 created 1989' WHERE ec = 'EC 4.2.1.90';
UPDATE enzyme SET sys_name = 'L-arogenate hydro-lyase (decarboxylating; L-phenylalanine-forming)' WHERE ec = 'EC 4.2.1.91';
UPDATE enzyme SET history = 'EC 4.2.1.91 created 1992, modified 2005' WHERE ec = 'EC 4.2.1.91';
UPDATE enzyme SET sys_name = '(9Z,11E,15Z)-(13S)-hydroperoxyoctadeca-9,11,15-trienoate 12,13-hydro-lyase ((9Z,15Z)-(13S)-12,13-epoxyoctadeca-9,11,15-trienoate-forming)' WHERE ec = 'EC 4.2.1.92';
UPDATE enzyme SET history = 'EC 4.2.1.92 created 1992, modified 2008' WHERE ec = 'EC 4.2.1.92';
UPDATE enzyme SET sys_name = '(6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adenine-dinucleotide hydro-lyase (ATP-hydrolyzing; NADH-forming)' WHERE ec = 'EC 4.2.1.93';
UPDATE enzyme SET history = 'EC 4.2.1.93 created 1992, modified 2012' WHERE ec = 'EC 4.2.1.93';
UPDATE enzyme SET sys_name = 'Scytalone 7,8-hydro-lyase (1,3,8-trihydroxynaphthalene-forming)' WHERE ec = 'EC 4.2.1.94';
UPDATE enzyme SET history = 'EC 4.2.1.94 created 1992' WHERE ec = 'EC 4.2.1.94';
UPDATE enzyme SET sys_name = 'Kievitone-hydrate hydro-lyase (kievitone-forming)' WHERE ec = 'EC 4.2.1.95';
UPDATE enzyme SET history = 'EC 4.2.1.95 created 1992' WHERE ec = 'EC 4.2.1.95';
UPDATE enzyme SET sys_name = '(6R)-6-(L-erythro-1,2-dihydroxypropyl)-5,6,7,8-tetrahydro-4a-hydroxypterin hydro-lyase ((6R)-6-(L-erythro-1,2-dihydroxypropyl)-7,8-dihydro-6H-pterin-forming)' WHERE ec = 'EC 4.2.1.96';
UPDATE enzyme SET history = 'EC 4.2.1.96 created 1999' WHERE ec = 'EC 4.2.1.96';
UPDATE enzyme SET sys_name = 'Phaseollidin-hydrate hydro-lyase (phaseollidin-forming)' WHERE ec = 'EC 4.2.1.97';
UPDATE enzyme SET history = 'EC 4.2.1.97 created 1999' WHERE ec = 'EC 4.2.1.97';
UPDATE enzyme SET sys_name = '16-alpha-hydroxyprogesterone hydro-lyase (16,17-didehydroprogesterone-forming)' WHERE ec = 'EC 4.2.1.98';
UPDATE enzyme SET history = 'EC 4.2.1.98 created 1999, modified 2004 (EC 4.2.1.86 created 1989, incorporated 2004)' WHERE ec = 'EC 4.2.1.98';
UPDATE enzyme SET sys_name = '(2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate hydro-lyase ((Z)-but-2-ene-1,2,3-tricarboxylate-forming)' WHERE ec = 'EC 4.2.1.99';
UPDATE enzyme SET history = 'EC 4.2.1.99 created 1999' WHERE ec = 'EC 4.2.1.99';
UPDATE enzyme SET sys_name = '6-hydroxycyclohex-1-enecarbonyl-CoA hydro-lyase (cyclohexa-1,5-dienecarbonyl-CoA-forming)' WHERE ec = 'EC 4.2.1.100';
UPDATE enzyme SET history = 'EC 4.2.1.100 created 2000, modified 2001' WHERE ec = 'EC 4.2.1.100';
UPDATE enzyme SET sys_name = '4-hydroxy-3-methoxyphenyl-beta-hydroxypropanoyl-CoA hydro-lyase (feruloyl-CoA-forming)' WHERE ec = 'EC 4.2.1.101';
UPDATE enzyme SET history = 'EC 4.2.1.101 created 2000' WHERE ec = 'EC 4.2.1.101';
UPDATE enzyme SET history = 'EC 4.2.1.102 created 2001, deleted 2001' WHERE ec = 'EC 4.2.1.102';
UPDATE enzyme SET sys_name = 'N-cyclohexylformamide hydro-lyase (cyclohexyl-isocyanide-forming)' WHERE ec = 'EC 4.2.1.103';
UPDATE enzyme SET history = 'EC 4.2.1.103 created 2001' WHERE ec = 'EC 4.2.1.103';
UPDATE enzyme SET sys_name = 'Carbamate hydro-lyase' WHERE ec = 'EC 4.2.1.104';
UPDATE enzyme SET history = 'EC 4.2.1.104 created 1972 as EC 3.5.5.3, transferred 1990 to EC 4.3.99.1, transferred 2001 to EC 4.2.1.104, modified 2007' WHERE ec = 'EC 4.2.1.104';
UPDATE enzyme SET sys_name = '2,4,7-trihydroxyisoflavanone hydro-lyase (daidzein-forming)' WHERE ec = 'EC 4.2.1.105';
UPDATE enzyme SET history = 'EC 4.2.1.105 created 2004, modified 2013' WHERE ec = 'EC 4.2.1.105';
UPDATE enzyme SET sys_name = '7-alpha,12-alpha-dihydroxy-3-oxochol-4-enoate hydro-lyase (12-alpha-hydroxy-3-oxochola-4,6-dienoate-forming)' WHERE ec = 'EC 4.2.1.106';
UPDATE enzyme SET history = 'EC 4.2.1.106 created 2005' WHERE ec = 'EC 4.2.1.106';
UPDATE enzyme SET sys_name = '(24R,25R)-3-alpha,7-alpha,12-alpha,24-tetrahydroxy-5-beta-cholestanoyl-CoA hydro-lyase ((24E)-3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholest-24-enoyl-CoA-forming)' WHERE ec = 'EC 4.2.1.107';
UPDATE enzyme SET history = 'EC 4.2.1.107 created 2005' WHERE ec = 'EC 4.2.1.107';
UPDATE enzyme SET sys_name = 'N(4)-acetyl-L-2,4-diaminobutanoate hydro-lyase (L-ectoine-forming)' WHERE ec = 'EC 4.2.1.108';
UPDATE enzyme SET history = 'EC 4.2.1.108 created 2006' WHERE ec = 'EC 4.2.1.108';
UPDATE enzyme SET sys_name = 'S-methyl-5-thio-D-ribulose-1-phosphate 4-hydro-lyase (5-(methylthio)-2,3-dioxopentyl-phosphate-forming)' WHERE ec = 'EC 4.2.1.109';
UPDATE enzyme SET history = 'EC 4.2.1.109 created 2006' WHERE ec = 'EC 4.2.1.109';
UPDATE enzyme SET sys_name = '1,5-anhydro-D-fructose hydro-lyase (microthecin-forming)' WHERE ec = 'EC 4.2.1.110';
UPDATE enzyme SET history = 'EC 4.2.1.110 created 2006' WHERE ec = 'EC 4.2.1.110';
UPDATE enzyme SET sys_name = '1,5-anhydro-D-fructose hydro-lyase (ascopyrone-M-forming)' WHERE ec = 'EC 4.2.1.111';
UPDATE enzyme SET history = 'EC 4.2.1.111 created 2006' WHERE ec = 'EC 4.2.1.111';
UPDATE enzyme SET sys_name = 'Acetaldehyde hydro-lyase (acetylene-forming)' WHERE ec = 'EC 4.2.1.112';
UPDATE enzyme SET history = 'EC 4.2.1.112 created 2007' WHERE ec = 'EC 4.2.1.112';
UPDATE enzyme SET sys_name = '(1R,6R)-6-hydroxy-2-succinylcyclohexa-2,4-diene-1-carboxylate hydrolyase (2-succinylbenzoate-forming)' WHERE ec = 'EC 4.2.1.113';
UPDATE enzyme SET history = 'EC 4.2.1.113 created 2007' WHERE ec = 'EC 4.2.1.113';
UPDATE enzyme SET sys_name = '(R)-2-hydroxybutane-1,2,4-tricarboxylate hydro-lyase ((1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate-forming)' WHERE ec = 'EC 4.2.1.114';
UPDATE enzyme SET history = 'EC 4.2.1.114 created 2009' WHERE ec = 'EC 4.2.1.114';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine hydro-lyase (inverting; UDP-2-acetamido-2,6-dideoxy-beta-L-arabino-hex-4-ulose-forming)' WHERE ec = 'EC 4.2.1.115';
UPDATE enzyme SET history = 'EC 4.2.1.115 created 2009' WHERE ec = 'EC 4.2.1.115';
UPDATE enzyme SET sys_name = '3-hydroxypropionyl-CoA hydro-lyase' WHERE ec = 'EC 4.2.1.116';
UPDATE enzyme SET history = 'EC 4.2.1.116 created 2009' WHERE ec = 'EC 4.2.1.116';
UPDATE enzyme SET sys_name = '(2S,3S)-2-hydroxybutane-1,2,3-tricarboxylate hydro-lyase (2-methyl-trans-aconitate forming)' WHERE ec = 'EC 4.2.1.117';
UPDATE enzyme SET history = 'EC 4.2.1.117 created 2009' WHERE ec = 'EC 4.2.1.117';
UPDATE enzyme SET sys_name = '3-dehydroshikimate hydro-lyase' WHERE ec = 'EC 4.2.1.118';
UPDATE enzyme SET history = 'EC 4.2.1.118 created 2009' WHERE ec = 'EC 4.2.1.118';
UPDATE enzyme SET sys_name = '(3R)-3-hydroxyacyl-CoA hydro-lyase' WHERE ec = 'EC 4.2.1.119';
UPDATE enzyme SET history = 'EC 4.2.1.119 created 2009' WHERE ec = 'EC 4.2.1.119';
UPDATE enzyme SET sys_name = '4-hydroxybutanoyl-CoA hydro-lyase' WHERE ec = 'EC 4.2.1.120';
UPDATE enzyme SET history = 'EC 4.2.1.120 created 2009' WHERE ec = 'EC 4.2.1.120';
UPDATE enzyme SET sys_name = '(8E)-9-((1E,3E)-nona-1,3-dien-1-yloxy)non-8-enoate synthase' WHERE ec = 'EC 4.2.1.121';
UPDATE enzyme SET history = 'EC 4.2.1.121 created 2010' WHERE ec = 'EC 4.2.1.121';
UPDATE enzyme SET sys_name = 'L-serine hydro-lyase (adding indole, L-tryptophan-forming)' WHERE ec = 'EC 4.2.1.122';
UPDATE enzyme SET history = 'EC 4.2.1.122 created 2011' WHERE ec = 'EC 4.2.1.122';
UPDATE enzyme SET sys_name = 'Squalene hydro-lyase (tetrahymanol forming)' WHERE ec = 'EC 4.2.1.123';
UPDATE enzyme SET history = 'EC 4.2.1.123 created 2011' WHERE ec = 'EC 4.2.1.123';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene hydro-lyase (arabidiol forming)' WHERE ec = 'EC 4.2.1.124';
UPDATE enzyme SET history = 'EC 4.2.1.124 created 2011' WHERE ec = 'EC 4.2.1.124';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene hydro-lyase (dammarenediol-II forming)' WHERE ec = 'EC 4.2.1.125';
UPDATE enzyme SET history = 'EC 4.2.1.125 created 2011' WHERE ec = 'EC 4.2.1.125';
UPDATE enzyme SET sys_name = '(R)-lactate hydro-lyase (adding N-acetyl-D-glucosamine 6-phosphate; N-acetylmuramate 6-phosphate-forming)' WHERE ec = 'EC 4.2.1.126';
UPDATE enzyme SET history = 'EC 4.2.1.126 created 2011' WHERE ec = 'EC 4.2.1.126';
UPDATE enzyme SET sys_name = 'Linalool hydro-lyase (myrcene-forming)' WHERE ec = 'EC 4.2.1.127';
UPDATE enzyme SET history = 'EC 4.2.1.127 created 2011' WHERE ec = 'EC 4.2.1.127';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene hydro-lyase (lupan-3-beta,20-diol forming)' WHERE ec = 'EC 4.2.1.128';
UPDATE enzyme SET history = 'EC 4.2.1.128 created 2011' WHERE ec = 'EC 4.2.1.128';
UPDATE enzyme SET sys_name = 'Hopan-22-ol hydro-lyase' WHERE ec = 'EC 4.2.1.129';
UPDATE enzyme SET history = 'EC 4.2.1.129 created 2011' WHERE ec = 'EC 4.2.1.129';
UPDATE enzyme SET sys_name = '(R)-lactate hydro-lyase' WHERE ec = 'EC 4.2.1.130';
UPDATE enzyme SET history = 'EC 4.2.1.130 created 2011' WHERE ec = 'EC 4.2.1.130';
UPDATE enzyme SET sys_name = 'Lycopene hydrO-lyase (1-hydroxy-1,2-dihydrolycopene-forming)' WHERE ec = 'EC 4.2.1.131';
UPDATE enzyme SET history = 'EC 4.2.1.131 created 2011' WHERE ec = 'EC 4.2.1.131';
UPDATE enzyme SET sys_name = '4-hydroxy-2-oxohexanoate hydro-layse ((2Z,4Z)-2-hydroxyhexa-2,4-dienoate-forming)' WHERE ec = 'EC 4.2.1.132';
UPDATE enzyme SET history = 'EC 4.2.1.132 created 2012' WHERE ec = 'EC 4.2.1.132';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate hydro-lyase ((13E)-8-alpha-hydroxylabd-13-en-15-yl diphosphate forming)' WHERE ec = 'EC 4.2.1.133';
UPDATE enzyme SET history = 'EC 4.2.1.133, created 2012 as EC 5.5.1.21, transferred 2012 to EC 4.2.1.133' WHERE ec = 'EC 4.2.1.133';
UPDATE enzyme SET sys_name = 'Very-long-chain (3R)-3-hydroxyacyl-[acyl-carrier protein] hydro-lyase' WHERE ec = 'EC 4.2.1.134';
UPDATE enzyme SET history = 'EC 4.2.1.134 created 2012' WHERE ec = 'EC 4.2.1.134';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine hydro-lyase (configuration-retaining; UDP-2-acetamido-2,6-dideoxy-alpha-D-xylo-hex-4-ulose-forming)' WHERE ec = 'EC 4.2.1.135';
UPDATE enzyme SET history = 'EC 4.2.1.135 created 2012' WHERE ec = 'EC 4.2.1.135';
UPDATE enzyme SET sys_name = '(6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adenine-dinucleotide hydro-lyase (ADP-hydrolyzing; NADH-forming)' WHERE ec = 'EC 4.2.1.136';
UPDATE enzyme SET history = 'EC 4.2.1.136 created 2012' WHERE ec = 'EC 4.2.1.136';
UPDATE enzyme SET sys_name = 'Tetraprenyl-beta-curcumene--sporulenol cyclase' WHERE ec = 'EC 4.2.1.137';
UPDATE enzyme SET history = 'EC 4.2.1.137 created 2012' WHERE ec = 'EC 4.2.1.137';
UPDATE enzyme SET sys_name = '(+)-beta-caryophyllene hydrolase (cyclizing, (+)-caryolan-1-ol-forming)' WHERE ec = 'EC 4.2.1.138';
UPDATE enzyme SET history = 'EC 4.2.1.138 created 2011 as EC 3.7.1.15, transferred 2013 to EC 4.2.1.138' WHERE ec = 'EC 4.2.1.138';
UPDATE enzyme SET sys_name = '4-methoxyisoflavan-2,4,7-triol hydro-lyase ((-)-medicarpin-forming)' WHERE ec = 'EC 4.2.1.139';
UPDATE enzyme SET history = 'EC 4.2.1.139 created 2013' WHERE ec = 'EC 4.2.1.139';
UPDATE enzyme SET sys_name = 'D-gluconate/D-galactonate hydro-lyase' WHERE ec = 'EC 4.2.1.140';
UPDATE enzyme SET history = 'EC 4.2.1.140 created 2013' WHERE ec = 'EC 4.2.1.140';
UPDATE enzyme SET sys_name = '2-dehydro-3-deoxy-D-arabinonate hydro-lyase (2,5-dioxopentanoate-forming)' WHERE ec = 'EC 4.2.1.141';
UPDATE enzyme SET history = 'EC 4.2.1.141 created 2013' WHERE ec = 'EC 4.2.1.141';
UPDATE enzyme SET sys_name = '5-oxoaverantin hydro-lyase ((2S,5S)-averufin forming)' WHERE ec = 'EC 4.2.1.142';
UPDATE enzyme SET history = 'EC 4.2.1.142 created 2013' WHERE ec = 'EC 4.2.1.142';
UPDATE enzyme SET sys_name = 'Versiconal hydro-lyase (versicolorin-B forming)' WHERE ec = 'EC 4.2.1.143';
UPDATE enzyme SET history = 'EC 4.2.1.143 created 2013' WHERE ec = 'EC 4.2.1.143';
UPDATE enzyme SET sys_name = '5-amino-5-deoxy-3-dehydroshikimate hydro-lyase (3-amino-5-hydroxybenzoate-forming)' WHERE ec = 'EC 4.2.1.144';
UPDATE enzyme SET history = 'EC 4.2.1.144 created 2013' WHERE ec = 'EC 4.2.1.144';
UPDATE enzyme SET sys_name = '(2S,3S)-3-hydroxyarginine hydrolyase (cyclizing, (2S,3R)-capreomycidine-forming)' WHERE ec = 'EC 4.2.1.145';
UPDATE enzyme SET history = 'EC 4.2.1.145 created 2013' WHERE ec = 'EC 4.2.1.145';
UPDATE enzyme SET sys_name = 'L-galactonate hydro-lyase (2-dehydro-3-deoxy-L-galactonate-forming)' WHERE ec = 'EC 4.2.1.146';
UPDATE enzyme SET history = 'EC 4.2.1.146 created 2013' WHERE ec = 'EC 4.2.1.146';
UPDATE enzyme SET sys_name = '5,6,7,8-tetrahydromethanopterin hydro-lyase (formaldehyde-adding, tetrahydromethanopterin-forming)' WHERE ec = 'EC 4.2.1.147';
UPDATE enzyme SET history = 'EC 4.2.1.147 created 2014' WHERE ec = 'EC 4.2.1.147';
UPDATE enzyme SET sys_name = '(2R,3S)-2-methylmalyl-CoA hydro-lyase (2-methylfumaryl-CoA-forming)' WHERE ec = 'EC 4.2.1.148';
UPDATE enzyme SET history = 'EC 4.2.1.148 created 2014' WHERE ec = 'EC 4.2.1.148';
UPDATE enzyme SET sys_name = 'L-carnitinyl-CoA hydro-lyase ((E)-4-(trimethylammonio)but-2-enoyl-CoA-forming)' WHERE ec = 'EC 4.2.1.149';
UPDATE enzyme SET history = 'EC 4.2.1.149 created 2014' WHERE ec = 'EC 4.2.1.149';
UPDATE enzyme SET sys_name = 'Short-chain-(3S)-3-hydroxyacyl-CoA hydro-lyase' WHERE ec = 'EC 4.2.1.150';
UPDATE enzyme SET history = 'EC 4.2.1.150 created 2014' WHERE ec = 'EC 4.2.1.150';
UPDATE enzyme SET sys_name = 'Chorismate hydro-lyase (3-((1-carboxyvinyl)oxy)benzoate-forming)' WHERE ec = 'EC 4.2.1.151';
UPDATE enzyme SET history = 'EC 4.2.1.151 created 2014' WHERE ec = 'EC 4.2.1.151';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.2.1.n1';
UPDATE enzyme SET history = 'EC 4.2.1.n2 created 2012' WHERE ec = 'EC 4.2.1.n2';
UPDATE enzyme SET sys_name = 'Hyaluronate lyase' WHERE ec = 'EC 4.2.2.1';
UPDATE enzyme SET history = 'EC 4.2.2.1 created 1961 as EC 4.2.99.1, transferred 1972 to EC 4.2.2.1, modified 2001' WHERE ec = 'EC 4.2.2.1';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-galacturonan lyase' WHERE ec = 'EC 4.2.2.2';
UPDATE enzyme SET history = 'EC 4.2.2.2 created 1965 as EC 4.2.99.3, transferred 1972 to EC 4.2.2.2, modified 2002' WHERE ec = 'EC 4.2.2.2';
UPDATE enzyme SET sys_name = 'Poly((1-&gt;4)-beta-D-mannuronide) lyase' WHERE ec = 'EC 4.2.2.3';
UPDATE enzyme SET history = 'EC 4.2.2.3 created 1965 as EC 4.2.99.4, transferred 1972 to EC 4.2.2.3, modified 1990' WHERE ec = 'EC 4.2.2.3';
UPDATE enzyme SET history = 'EC 4.2.2.4 created 1972 (EC 4.2.99.6 created 1965, part incorporated 1976), deleted 2006' WHERE ec = 'EC 4.2.2.4';
UPDATE enzyme SET sys_name = 'Chondroitin AC lyase' WHERE ec = 'EC 4.2.2.5';
UPDATE enzyme SET history = 'EC 4.2.2.5 created 1972 (EC 4.2.99.6 created 1965, part incorporated 1976)' WHERE ec = 'EC 4.2.2.5';
UPDATE enzyme SET sys_name = 'Oligogalacturonide lyase' WHERE ec = 'EC 4.2.2.6';
UPDATE enzyme SET history = 'EC 4.2.2.6 created 1972' WHERE ec = 'EC 4.2.2.6';
UPDATE enzyme SET sys_name = 'Heparin lyase' WHERE ec = 'EC 4.2.2.7';
UPDATE enzyme SET history = 'EC 4.2.2.7 created 1972' WHERE ec = 'EC 4.2.2.7';
UPDATE enzyme SET sys_name = 'Heparin-sulfate lyase' WHERE ec = 'EC 4.2.2.8';
UPDATE enzyme SET history = 'EC 4.2.2.8 created 1972' WHERE ec = 'EC 4.2.2.8';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-galacturonan reducing-end-disaccharide-lyase' WHERE ec = 'EC 4.2.2.9';
UPDATE enzyme SET history = 'EC 4.2.2.9 created 1972, modified 2002' WHERE ec = 'EC 4.2.2.9';
UPDATE enzyme SET sys_name = '(1-&gt;4)-6-O-methyl-alpha-D-galacturonan lyase' WHERE ec = 'EC 4.2.2.10';
UPDATE enzyme SET history = 'EC 4.2.2.10 created 1972, modified 2002' WHERE ec = 'EC 4.2.2.10';
UPDATE enzyme SET sys_name = 'Poly((1-&gt;4)-alpha-L-guluronide) exo-lyase' WHERE ec = 'EC 4.2.2.11';
UPDATE enzyme SET history = 'EC 4.2.2.11 created 1990' WHERE ec = 'EC 4.2.2.11';
UPDATE enzyme SET sys_name = 'Xanthan lyase' WHERE ec = 'EC 4.2.2.12';
UPDATE enzyme SET history = 'EC 4.2.2.12 created 1990' WHERE ec = 'EC 4.2.2.12';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-glucan exo-4-lyase (1,5-anhydro-D-fructose-forming)' WHERE ec = 'EC 4.2.2.13';
UPDATE enzyme SET history = 'EC 4.2.2.13 created 1999' WHERE ec = 'EC 4.2.2.13';
UPDATE enzyme SET sys_name = '(1-&gt;4)-beta-D-glucuronan lyase' WHERE ec = 'EC 4.2.2.14';
UPDATE enzyme SET history = 'EC 4.2.2.14 created 2000' WHERE ec = 'EC 4.2.2.14';
UPDATE enzyme SET sys_name = 'Glycoconjugate sialyl-lyase (2,7-cyclizing)' WHERE ec = 'EC 4.2.2.15';
UPDATE enzyme SET history = 'EC 4.2.2.15 created 1992 as EC 3.2.1.138, transferred 2003 to EC 4.2.2.15' WHERE ec = 'EC 4.2.2.15';
UPDATE enzyme SET sys_name = '(2-&gt;6)-beta-D-fructan lyase (di-beta-D-fructofuranose-2,6:2,6-dianhydride-forming)' WHERE ec = 'EC 4.2.2.16';
UPDATE enzyme SET history = 'EC 4.2.2.16 created 2004' WHERE ec = 'EC 4.2.2.16';
UPDATE enzyme SET sys_name = '(2-&gt;1)-beta-D-fructan lyase (alpha-D-fructofuranose-beta-D-fructofuranose-1,2:2,1-dianhydride-forming)' WHERE ec = 'EC 4.2.2.17';
UPDATE enzyme SET history = 'EC 4.2.2.17 created 1992 as EC 2.4.1.200, transferred 2004 to EC 4.2.2.17' WHERE ec = 'EC 4.2.2.17';
UPDATE enzyme SET sys_name = '(2-&gt;1)-beta-D-fructan lyase (alpha-D-fructofuranose-beta-D-fructofuranose-1,2:2,3-dianhydride-forming)' WHERE ec = 'EC 4.2.2.18';
UPDATE enzyme SET history = 'EC 4.2.2.18 created 1976 as EC 2.4.1.93, transferred 2004 to EC 4.2.2.18' WHERE ec = 'EC 4.2.2.18';
UPDATE enzyme SET sys_name = 'Chondroitin B lyase' WHERE ec = 'EC 4.2.2.19';
UPDATE enzyme SET history = 'EC 4.2.2.19 created 2005' WHERE ec = 'EC 4.2.2.19';
UPDATE enzyme SET sys_name = 'Chondroitin-sulfate-ABC endolyase' WHERE ec = 'EC 4.2.2.20';
UPDATE enzyme SET history = 'EC 4.2.2.20 created 2006 (EC 4.2.2.4 created 1972, part-incorporated 2006 (EC 4.2.99.6 created 1965, part-incorporated 1976)' WHERE ec = 'EC 4.2.2.20';
UPDATE enzyme SET sys_name = 'Chondroitin-sulfate-ABC exolyase' WHERE ec = 'EC 4.2.2.21';
UPDATE enzyme SET history = 'EC 4.2.2.21 created 2006 (EC 4.2.2.4 created 1972, part-incorporated 2006 (EC 4.2.99.6 created 1965, part incorporated 1976)), modified 2010' WHERE ec = 'EC 4.2.2.21';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-galacturonan reducing-end-trisaccharide-lyase' WHERE ec = 'EC 4.2.2.22';
UPDATE enzyme SET history = 'EC 4.2.2.22 created 2007' WHERE ec = 'EC 4.2.2.22';
UPDATE enzyme SET sys_name = 'Alpha-L-rhamnopyranosyl-(1-&gt;4)-alpha-D-galactopyranosyluronate endolyase' WHERE ec = 'EC 4.2.2.23';
UPDATE enzyme SET history = 'EC 4.2.2.23 created 2011' WHERE ec = 'EC 4.2.2.23';
UPDATE enzyme SET sys_name = 'Alpha-L-rhamnopyranosyl-(1-&gt;4)-alpha-D-galactopyranosyluronate exolyase' WHERE ec = 'EC 4.2.2.24';
UPDATE enzyme SET history = 'EC 4.2.2.24 created 2011' WHERE ec = 'EC 4.2.2.24';
UPDATE enzyme SET sys_name = 'Gellan beta-D-glucopyranosyl-(1-&gt;4)-D-glucopyranosyluronate lyase' WHERE ec = 'EC 4.2.2.25';
UPDATE enzyme SET history = 'EC 4.2.2.25 created 2011' WHERE ec = 'EC 4.2.2.25';
UPDATE enzyme SET sys_name = 'Peptidoglycan lytic exotransglycosylase' WHERE ec = 'EC 4.2.2.n1';
UPDATE enzyme SET history = 'EC 4.2.2.n1 created 2010' WHERE ec = 'EC 4.2.2.n1';
UPDATE enzyme SET sys_name = 'Peptidoglycan lytic endotransglycosylase' WHERE ec = 'EC 4.2.2.n2';
UPDATE enzyme SET history = 'EC 4.2.2.n2 created 2010' WHERE ec = 'EC 4.2.2.n2';
UPDATE enzyme SET sys_name = 'O-phosphate-L-homoserine phospho-lyase (adding water; L-threonine-forming)' WHERE ec = 'EC 4.2.3.1';
UPDATE enzyme SET history = 'EC 4.2.3.1 created 1961 as EC 4.2.99.2, transferred 2000 to EC 4.2.3.1' WHERE ec = 'EC 4.2.3.1';
UPDATE enzyme SET sys_name = 'Ethanolamine-phosphate phosphate-lyase (deaminating; acetaldehyde-forming)' WHERE ec = 'EC 4.2.3.2';
UPDATE enzyme SET history = 'EC 4.2.3.2 created 1972 as EC 4.2.99.7, transferred 2000 to EC 4.2.3.2' WHERE ec = 'EC 4.2.3.2';
UPDATE enzyme SET sys_name = 'Glycerone-phosphate phosphate-lyase (methylglyoxal-forming)' WHERE ec = 'EC 4.2.3.3';
UPDATE enzyme SET history = 'EC 4.2.3.3 created 1972 as EC 4.2.99.11, transferred 2000 to EC 4.2.3.3' WHERE ec = 'EC 4.2.3.3';
UPDATE enzyme SET sys_name = '3-deoxy-D-arabino-hept-2-ulosonate-7-phosphate phosphate-lyase (cyclizing; 3-dehydroquinate-forming)' WHERE ec = 'EC 4.2.3.4';
UPDATE enzyme SET history = 'EC 4.2.3.4 created 1978 as EC 4.6.1.3, transferred 2000 to EC 4.2.3.4, modified 2002' WHERE ec = 'EC 4.2.3.4';
UPDATE enzyme SET sys_name = '5-O-(1-carboxyvinyl)-3-phosphoshikimate phosphate-lyase (chorismate-forming)' WHERE ec = 'EC 4.2.3.5';
UPDATE enzyme SET history = 'EC 4.2.3.5 created 1978 as EC 4.6.1.4, modified 1983, transferred 2000 to EC 4.2.3.5, modified 2002' WHERE ec = 'EC 4.2.3.5';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, trichodiene-forming)' WHERE ec = 'EC 4.2.3.6';
UPDATE enzyme SET history = 'EC 4.2.3.6 created 1989 as EC 4.1.99.6, transferred 2000 to EC 4.2.3.6' WHERE ec = 'EC 4.2.3.6';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, pentalenene-forming)' WHERE ec = 'EC 4.2.3.7';
UPDATE enzyme SET history = 'EC 4.2.3.7 created 1989 as EC 4.6.1.5, transferred 2000 to EC 4.2.3.7' WHERE ec = 'EC 4.2.3.7';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate diphosphate-lyase (cyclizing, casbene-forming)' WHERE ec = 'EC 4.2.3.8';
UPDATE enzyme SET history = 'EC 4.2.3.8 created 1989 as EC 4.6.1.7, transferred 2000 to EC 4.2.3.8' WHERE ec = 'EC 4.2.3.8';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, aristolochene-forming)' WHERE ec = 'EC 4.2.3.9';
UPDATE enzyme SET history = 'EC 4.2.3.9 created 1992 as EC 2.5.1.40, transferred 1999 to EC 4.1.99.7, transferred 2000 to EC 4.2.3.9, modified 2006' WHERE ec = 'EC 4.2.3.9';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (-)-endo-fenchol-forming)' WHERE ec = 'EC 4.2.3.10';
UPDATE enzyme SET history = 'EC 4.2.3.10 created 1992 as EC 4.6.1.8, transferred 2000 to EC 4.2.3.10' WHERE ec = 'EC 4.2.3.10';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, sabinene-hydrate-forming)' WHERE ec = 'EC 4.2.3.11';
UPDATE enzyme SET history = 'EC 4.2.3.11 created 1992 as EC 4.6.1.9, transferred 2000 to EC 4.2.3.11' WHERE ec = 'EC 4.2.3.11';
UPDATE enzyme SET sys_name = '7,8-dihydroneopterin 3-triphosphate triphosphate-lyase (6-pyruvoyl-5,6,7,8-tetrahydropterin-forming)' WHERE ec = 'EC 4.2.3.12';
UPDATE enzyme SET history = 'EC 4.2.3.12 created 1999 as EC 4.6.1.10, transferred 2000 to EC 4.2.3.12, modified 2001' WHERE ec = 'EC 4.2.3.12';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, (+)-delta-cadinene-forming)' WHERE ec = 'EC 4.2.3.13';
UPDATE enzyme SET history = 'EC 4.2.3.13 created 1999 as EC 4.6.1.11, transferred 2000 to EC 4.2.3.13' WHERE ec = 'EC 4.2.3.13';
UPDATE enzyme SET history = 'EC 4.2.3.14 created 2000 as EC 4.1.99.8, transferred 2000 to EC 4.2.3.14, deleted 2012' WHERE ec = 'EC 4.2.3.14';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (myrcene-forming)' WHERE ec = 'EC 4.2.3.15';
UPDATE enzyme SET history = 'EC 4.2.3.15 created 2000 as EC 4.1.99.9, transferred 2000 to EC 4.2.3.15' WHERE ec = 'EC 4.2.3.15';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing; (-)-(4S)-limonene-forming)' WHERE ec = 'EC 4.2.3.16';
UPDATE enzyme SET history = 'EC 4.2.3.16 created 2000 as EC 4.1.99.10, transferred 2000 to EC 4.2.3.16, modified 2003' WHERE ec = 'EC 4.2.3.16';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate diphosphate-lyase (cyclizing; taxa-4,11-diene-forming)' WHERE ec = 'EC 4.2.3.17';
UPDATE enzyme SET history = 'EC 4.2.3.17 created 2002' WHERE ec = 'EC 4.2.3.17';
UPDATE enzyme SET sys_name = '(+)-copalyl-diphosphate diphosphate-lyase (cyclizing, abieta-7,13-diene-forming)' WHERE ec = 'EC 4.2.3.18';
UPDATE enzyme SET history = 'EC 4.2.3.18 created 2002, modified 2012' WHERE ec = 'EC 4.2.3.18';
UPDATE enzyme SET sys_name = 'Ent-copalyl-diphosphate diphosphate-lyase (cyclizing, ent-kaurene-forming)' WHERE ec = 'EC 4.2.3.19';
UPDATE enzyme SET history = 'EC 4.2.3.19 created 2002' WHERE ec = 'EC 4.2.3.19';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (+)-(4R)-limonene-forming)' WHERE ec = 'EC 4.2.3.20';
UPDATE enzyme SET history = 'EC 4.2.3.20 created 2003' WHERE ec = 'EC 4.2.3.20';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl diphosphate diphosphate-lyase (cyclizing, vetispiradiene-forming)' WHERE ec = 'EC 4.2.3.21';
UPDATE enzyme SET history = 'EC 4.2.3.21 created 2004, modified 2011' WHERE ec = 'EC 4.2.3.21';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((1E,4S,5E,7R)-germacra-1(10),5-dien-11-ol-forming)' WHERE ec = 'EC 4.2.3.22';
UPDATE enzyme SET history = 'EC 4.2.3.22 created 2006, modified 2011' WHERE ec = 'EC 4.2.3.22';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((+)-(R)-germacrene-A-forming)' WHERE ec = 'EC 4.2.3.23';
UPDATE enzyme SET history = 'EC 4.2.3.23 created 2006' WHERE ec = 'EC 4.2.3.23';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (amorpha-4,11-diene-forming)' WHERE ec = 'EC 4.2.3.24';
UPDATE enzyme SET history = 'EC 4.2.3.24 created 2006' WHERE ec = 'EC 4.2.3.24';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase ((3S)-linalool-forming)' WHERE ec = 'EC 4.2.3.25';
UPDATE enzyme SET history = 'EC 4.2.3.25 created 2006' WHERE ec = 'EC 4.2.3.25';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase ((3R)-linalool-forming)' WHERE ec = 'EC 4.2.3.26';
UPDATE enzyme SET history = 'EC 4.2.3.26 created 2006' WHERE ec = 'EC 4.2.3.26';
UPDATE enzyme SET sys_name = 'Dimethylallyl-diphosphate diphosphate-lyase (isoprene-forming)' WHERE ec = 'EC 4.2.3.27';
UPDATE enzyme SET history = 'EC 4.2.3.27 created 2007' WHERE ec = 'EC 4.2.3.27';
UPDATE enzyme SET sys_name = 'Ent-copalyl-diphosphate diphosphate-lyase (ent-cassa-12,15-diene-forming)' WHERE ec = 'EC 4.2.3.28';
UPDATE enzyme SET history = 'EC 4.2.3.28 created 2008' WHERE ec = 'EC 4.2.3.28';
UPDATE enzyme SET sys_name = 'Ent-copalyl-diphosphate diphosphate-lyase (ent-sandaracopimara-8(14),15-diene-forming)' WHERE ec = 'EC 4.2.3.29';
UPDATE enzyme SET history = 'EC 4.2.3.29 created 2008' WHERE ec = 'EC 4.2.3.29';
UPDATE enzyme SET sys_name = 'Ent-copalyl-diphosphate diphosphate-lyase (ent-pimara-8(14),15-diene-forming)' WHERE ec = 'EC 4.2.3.30';
UPDATE enzyme SET history = 'EC 4.2.3.30 created 2008' WHERE ec = 'EC 4.2.3.30';
UPDATE enzyme SET sys_name = 'Ent-copalyl-diphosphate diphosphate-lyase (ent-pimara-9(11),15-diene-forming)' WHERE ec = 'EC 4.2.3.31';
UPDATE enzyme SET history = 'EC 4.2.3.31 created 2008' WHERE ec = 'EC 4.2.3.31';
UPDATE enzyme SET sys_name = '(+)-copalyl-diphosphate diphosphate-lyase (abieta-8(14),12-diene-forming)' WHERE ec = 'EC 4.2.3.32';
UPDATE enzyme SET history = 'EC 4.2.3.32 created 2008, modified 2012' WHERE ec = 'EC 4.2.3.32';
UPDATE enzyme SET sys_name = '9-alpha-copalyl-diphosphate diphosphate-lyase (stemar-13-ene-forming)' WHERE ec = 'EC 4.2.3.33';
UPDATE enzyme SET history = 'EC 4.2.3.33 created 2008' WHERE ec = 'EC 4.2.3.33';
UPDATE enzyme SET sys_name = '9-alpha-copalyl-diphosphate diphosphate-lyase (stemod-13(17)-ene-forming)' WHERE ec = 'EC 4.2.3.34';
UPDATE enzyme SET history = 'EC 4.2.3.34 created 2008' WHERE ec = 'EC 4.2.3.34';
UPDATE enzyme SET sys_name = '9-alpha-copalyl-diphosphate diphosphate-lyase (9-beta-pimara-7,15-diene-forming)' WHERE ec = 'EC 4.2.3.35';
UPDATE enzyme SET history = 'EC 4.2.3.35 created 2008' WHERE ec = 'EC 4.2.3.35';
UPDATE enzyme SET sys_name = 'Terpentedienyl-diphosphate diphosphate-lyase (terpentetriene-forming)' WHERE ec = 'EC 4.2.3.36';
UPDATE enzyme SET history = 'EC 4.2.3.36 created 2008' WHERE ec = 'EC 4.2.3.36';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((+)-epi-isozizaene-forming)' WHERE ec = 'EC 4.2.3.37';
UPDATE enzyme SET history = 'EC 4.2.3.37 created 2008' WHERE ec = 'EC 4.2.3.37';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((E)-alpha-bisabolene-forming)' WHERE ec = 'EC 4.2.3.38';
UPDATE enzyme SET history = 'EC 4.2.3.38 created 2009' WHERE ec = 'EC 4.2.3.38';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (8-epi-cedrol-forming)' WHERE ec = 'EC 4.2.3.39';
UPDATE enzyme SET history = 'EC 4.2.3.39 created 2009' WHERE ec = 'EC 4.2.3.39';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((Z)-gamma-bisabolene-forming)' WHERE ec = 'EC 4.2.3.40';
UPDATE enzyme SET history = 'EC 4.2.3.40 created 2009' WHERE ec = 'EC 4.2.3.40';
UPDATE enzyme SET sys_name = 'Geranylgeranyl-diphosphate diphosphate-lyase (elisabethatriene-forming)' WHERE ec = 'EC 4.2.3.41';
UPDATE enzyme SET history = 'EC 4.2.3.41 created 2009' WHERE ec = 'EC 4.2.3.41';
UPDATE enzyme SET sys_name = '9-alpha-copalyl-diphosphate diphosphate-lyase (aphidicolan-16-beta-ol-forming)' WHERE ec = 'EC 4.2.3.42';
UPDATE enzyme SET history = 'EC 4.2.3.42 created 2009' WHERE ec = 'EC 4.2.3.42';
UPDATE enzyme SET sys_name = 'Geranylgeranyl diphosphate-lyase (fusicocca-2,10(14)-diene-forming)' WHERE ec = 'EC 4.2.3.43';
UPDATE enzyme SET history = 'EC 4.2.3.43 created 2009' WHERE ec = 'EC 4.2.3.43';
UPDATE enzyme SET sys_name = 'Copalyl diphosphate-lyase (isopimara-7,15-diene-forming)' WHERE ec = 'EC 4.2.3.44';
UPDATE enzyme SET history = 'EC 4.2.3.44 created 2009' WHERE ec = 'EC 4.2.3.44';
UPDATE enzyme SET sys_name = '(+)-copalyl-diphosphate diphosphate-lyase (phyllocladan-16-alpha-ol-forming)' WHERE ec = 'EC 4.2.3.45';
UPDATE enzyme SET history = 'EC 4.2.3.45 created 2009' WHERE ec = 'EC 4.2.3.45';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate lyase ((3E,6E)-alpha-farnesene-forming)' WHERE ec = 'EC 4.2.3.46';
UPDATE enzyme SET history = 'EC 4.2.3.46 created 2010' WHERE ec = 'EC 4.2.3.46';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((E)-beta-farnesene-forming)' WHERE ec = 'EC 4.2.3.47';
UPDATE enzyme SET history = 'EC 4.2.3.47 created 2010' WHERE ec = 'EC 4.2.3.47';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((3S,6E)-nerolidol-forming)' WHERE ec = 'EC 4.2.3.48';
UPDATE enzyme SET history = 'EC 4.2.3.48 created 2010' WHERE ec = 'EC 4.2.3.48';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((3R,6E)-nerolidol-forming)' WHERE ec = 'EC 4.2.3.49';
UPDATE enzyme SET history = 'EC 4.2.3.49 created 2010' WHERE ec = 'EC 4.2.3.49';
UPDATE enzyme SET sys_name = '(2Z,6Z)-farnesyl diphosphate lyase (cyclizing; (+)-alpha-santalene-forming)' WHERE ec = 'EC 4.2.3.50';
UPDATE enzyme SET history = 'EC 4.2.3.50 created 2010' WHERE ec = 'EC 4.2.3.50';
UPDATE enzyme SET sys_name = 'Neryl-diphosphate diphosphate-lyase (cyclizing; beta-phellandrene-forming)' WHERE ec = 'EC 4.2.3.51';
UPDATE enzyme SET history = 'EC 4.2.3.51 created 2010' WHERE ec = 'EC 4.2.3.51';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing; (4S)-beta-phellandrene-forming)' WHERE ec = 'EC 4.2.3.52';
UPDATE enzyme SET history = 'EC 4.2.3.52 created 2010' WHERE ec = 'EC 4.2.3.52';
UPDATE enzyme SET sys_name = '(2Z,6Z)-farnesyl diphosphate lyase (cyclizing; (+)-endo-beta-bergamotene-forming)' WHERE ec = 'EC 4.2.3.53';
UPDATE enzyme SET history = 'EC 4.2.3.53 created 2010' WHERE ec = 'EC 4.2.3.53';
UPDATE enzyme SET sys_name = '(2Z,6Z)-farnesyl diphosphate lyase (cyclizing; (-)-endo-alpha-bergamotene-forming)' WHERE ec = 'EC 4.2.3.54';
UPDATE enzyme SET history = 'EC 4.2.3.54 created 2010' WHERE ec = 'EC 4.2.3.54';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((S)-beta-bisabolene-forming)' WHERE ec = 'EC 4.2.3.55';
UPDATE enzyme SET history = 'EC 4.2.3.55 created 2011' WHERE ec = 'EC 4.2.3.55';
UPDATE enzyme SET sys_name = 'Farnesyl-diphosphate diphosphate-lyase (cyclizing, gamma-humulene-forming)' WHERE ec = 'EC 4.2.3.56';
UPDATE enzyme SET history = 'EC 4.2.3.56 created 2011' WHERE ec = 'EC 4.2.3.56';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((-)-beta-caryophyllene-forming)' WHERE ec = 'EC 4.2.3.57';
UPDATE enzyme SET history = 'EC 4.2.3.57 created 2011' WHERE ec = 'EC 4.2.3.57';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (longifolene-forming)' WHERE ec = 'EC 4.2.3.58';
UPDATE enzyme SET history = 'EC 4.2.3.58 created 2011' WHERE ec = 'EC 4.2.3.58';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((E)-gamma-bisabolene-forming)' WHERE ec = 'EC 4.2.3.59';
UPDATE enzyme SET history = 'EC 4.2.3.59 created 2011' WHERE ec = 'EC 4.2.3.59';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (germacrene-C-forming)' WHERE ec = 'EC 4.2.3.60';
UPDATE enzyme SET history = 'EC 4.2.3.60 created 2011' WHERE ec = 'EC 4.2.3.60';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((+)-5-epiaristolochene-forming)' WHERE ec = 'EC 4.2.3.61';
UPDATE enzyme SET history = 'EC 4.2.3.61 created 2011' WHERE ec = 'EC 4.2.3.61';
UPDATE enzyme SET sys_name = '(2Z,6E)-farnesyl-diphosphate diphosphate-lyase ((-)-gamma-cadinene-forming)' WHERE ec = 'EC 4.2.3.62';
UPDATE enzyme SET history = 'EC 4.2.3.62 created 2011' WHERE ec = 'EC 4.2.3.62';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((+)-cubenene-forming)' WHERE ec = 'EC 4.2.3.63';
UPDATE enzyme SET history = 'EC 4.2.3.63 created 2011' WHERE ec = 'EC 4.2.3.63';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((+)-epicubenol-forming)' WHERE ec = 'EC 4.2.3.64';
UPDATE enzyme SET history = 'EC 4.2.3.64 created 2011' WHERE ec = 'EC 4.2.3.64';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (zingiberene-forming)' WHERE ec = 'EC 4.2.3.65';
UPDATE enzyme SET history = 'EC 4.2.3.65 created 2011' WHERE ec = 'EC 4.2.3.65';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (beta-selinene-forming)' WHERE ec = 'EC 4.2.3.66';
UPDATE enzyme SET history = 'EC 4.2.3.66 created 2011' WHERE ec = 'EC 4.2.3.66';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cis-muuroladiene-forming)' WHERE ec = 'EC 4.2.3.67';
UPDATE enzyme SET history = 'EC 4.2.3.67 created 2011' WHERE ec = 'EC 4.2.3.67';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (beta-eudesmol-forming)' WHERE ec = 'EC 4.2.3.68';
UPDATE enzyme SET history = 'EC 4.2.3.68 created 2011, modified 2012' WHERE ec = 'EC 4.2.3.68';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((+)-alpha-barbatene-forming)' WHERE ec = 'EC 4.2.3.69';
UPDATE enzyme SET history = 'EC 4.2.3.69 created 2011, modified 2012' WHERE ec = 'EC 4.2.3.69';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (patchoulol-forming)' WHERE ec = 'EC 4.2.3.70';
UPDATE enzyme SET history = 'EC 4.2.3.70 created 2011' WHERE ec = 'EC 4.2.3.70';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((E,E)-germacrene-B-forming)' WHERE ec = 'EC 4.2.3.71';
UPDATE enzyme SET history = 'EC 4.2.3.71 created 2011' WHERE ec = 'EC 4.2.3.71';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((-)-alpha-gurjunene-forming)' WHERE ec = 'EC 4.2.3.72';
UPDATE enzyme SET history = 'EC 4.2.3.72 created 2011' WHERE ec = 'EC 4.2.3.72';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (valencene-forming)' WHERE ec = 'EC 4.2.3.73';
UPDATE enzyme SET history = 'EC 4.2.3.73 created 2011' WHERE ec = 'EC 4.2.3.73';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphohydrolase (presilphiperfolan-8-beta-ol-forming)' WHERE ec = 'EC 4.2.3.74';
UPDATE enzyme SET history = 'EC 4.2.3.74 created 2011' WHERE ec = 'EC 4.2.3.74';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((-)-germacrene-D-forming)' WHERE ec = 'EC 4.2.3.75';
UPDATE enzyme SET history = 'EC 4.2.3.75 created 2011' WHERE ec = 'EC 4.2.3.75';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((+)-delta-selinene-forming)' WHERE ec = 'EC 4.2.3.76';
UPDATE enzyme SET history = 'EC 4.2.3.76 created 2011' WHERE ec = 'EC 4.2.3.76';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((+)-germacrene-D-forming)' WHERE ec = 'EC 4.2.3.77';
UPDATE enzyme SET history = 'EC 4.2.3.77 created 2011' WHERE ec = 'EC 4.2.3.77';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl diphosphate lyase (cyclizing, (+)-beta-chamigrene-forming)' WHERE ec = 'EC 4.2.3.78';
UPDATE enzyme SET history = 'EC 4.2.3.78 created 2011' WHERE ec = 'EC 4.2.3.78';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl diphosphate lyase (cyclizing, (+)-thujopsene-forming)' WHERE ec = 'EC 4.2.3.79';
UPDATE enzyme SET history = 'EC 4.2.3.79 created 2011' WHERE ec = 'EC 4.2.3.79';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (alpha-longipinene-forming)' WHERE ec = 'EC 4.2.3.80';
UPDATE enzyme SET history = 'EC 4.2.3.80 created 2011' WHERE ec = 'EC 4.2.3.80';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl diphosphate lyase (cyclizing, (-)-exo-alpha-bergamotene-forming)' WHERE ec = 'EC 4.2.3.81';
UPDATE enzyme SET history = 'EC 4.2.3.81 created 2011' WHERE ec = 'EC 4.2.3.81';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl diphosphate lyase (cyclizing, (+)-alpha-santalene-forming)' WHERE ec = 'EC 4.2.3.82';
UPDATE enzyme SET history = 'EC 4.2.3.82 created 2011' WHERE ec = 'EC 4.2.3.82';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl diphosphate lyase (cyclizing, (-)-beta-santalene-forming)' WHERE ec = 'EC 4.2.3.83';
UPDATE enzyme SET history = 'EC 4.2.3.83 created 2011' WHERE ec = 'EC 4.2.3.83';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (10-epi-gamma-eudesmol-forming)' WHERE ec = 'EC 4.2.3.84';
UPDATE enzyme SET history = 'EC 4.2.3.84 created 2011' WHERE ec = 'EC 4.2.3.84';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (alpha-eudesmol-forming)' WHERE ec = 'EC 4.2.3.85';
UPDATE enzyme SET history = 'EC 4.2.3.85 created 2011' WHERE ec = 'EC 4.2.3.85';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (7-epi-alpha-selinene-forming)' WHERE ec = 'EC 4.2.3.86';
UPDATE enzyme SET history = 'EC 4.2.3.86 created 2011' WHERE ec = 'EC 4.2.3.86';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, alpha-guaiene-forming)' WHERE ec = 'EC 4.2.3.87';
UPDATE enzyme SET history = 'EC 4.2.3.87 created 2011' WHERE ec = 'EC 4.2.3.87';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (viridiflorene-forming)' WHERE ec = 'EC 4.2.3.88';
UPDATE enzyme SET history = 'EC 4.2.3.88 created 2011' WHERE ec = 'EC 4.2.3.88';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, (+)-beta-caryophyllene-forming)' WHERE ec = 'EC 4.2.3.89';
UPDATE enzyme SET history = 'EC 4.2.3.89 created 2011' WHERE ec = 'EC 4.2.3.89';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, 5-epi-alpha-selinene-forming)' WHERE ec = 'EC 4.2.3.90';
UPDATE enzyme SET history = 'EC 4.2.3.90 created 2011' WHERE ec = 'EC 4.2.3.90';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, cubebol-forming)' WHERE ec = 'EC 4.2.3.91';
UPDATE enzyme SET history = 'EC 4.2.3.91 created 2011' WHERE ec = 'EC 4.2.3.91';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase ((+)-gamma-cadinene-forming)' WHERE ec = 'EC 4.2.3.92';
UPDATE enzyme SET history = 'EC 4.2.3.92 created 2011' WHERE ec = 'EC 4.2.3.92';
UPDATE enzyme SET sys_name = 'Delta-guaiene hydrolase (cyclizing, delta-guaiene-forming)' WHERE ec = 'EC 4.2.3.93';
UPDATE enzyme SET history = 'EC 4.2.3.93 created 2011' WHERE ec = 'EC 4.2.3.93';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, gamma-curcumene-forming)' WHERE ec = 'EC 4.2.3.94';
UPDATE enzyme SET history = 'EC 4.2.3.94 created 2012' WHERE ec = 'EC 4.2.3.94';
UPDATE enzyme SET sys_name = '(-)-alpha-cuprenene hydrolase (cyclizing, (-)-alpha-cuprenene-forming)' WHERE ec = 'EC 4.2.3.95';
UPDATE enzyme SET history = 'EC 4.2.3.95 created 2012' WHERE ec = 'EC 4.2.3.95';
UPDATE enzyme SET sys_name = 'Avermitilol hydrolase (cyclizing, avermitilol-forming)' WHERE ec = 'EC 4.2.3.96';
UPDATE enzyme SET history = 'EC 4.2.3.96 created 2012' WHERE ec = 'EC 4.2.3.96';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl diphosphate diphosphate-lyase (cyclizing, (-)-delta-cadinene-forming)' WHERE ec = 'EC 4.2.3.97';
UPDATE enzyme SET history = 'EC 4.2.3.97 created 2012' WHERE ec = 'EC 4.2.3.97';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, (+)-T-muurolol-forming)' WHERE ec = 'EC 4.2.3.98';
UPDATE enzyme SET history = 'EC 4.2.3.98 created 2012' WHERE ec = 'EC 4.2.3.98';
UPDATE enzyme SET sys_name = '9-alpha-copalyl-diphosphate diphosphate-lyase ((12E)-9-alpha-labda-8(17),12,14-triene-forming)' WHERE ec = 'EC 4.2.3.99';
UPDATE enzyme SET history = 'EC 4.2.3.99 created 2012' WHERE ec = 'EC 4.2.3.99';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (bicyclogermacrene-forming)' WHERE ec = 'EC 4.2.3.100';
UPDATE enzyme SET history = 'EC 4.2.3.100 created 2012' WHERE ec = 'EC 4.2.3.100';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (7-epi-sesquithujene-forming)' WHERE ec = 'EC 4.2.3.101';
UPDATE enzyme SET history = 'EC 4.2.3.101 created 2012' WHERE ec = 'EC 4.2.3.101';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (sesquithujene-forming)' WHERE ec = 'EC 4.2.3.102';
UPDATE enzyme SET history = 'EC 4.2.3.102 created 2012' WHERE ec = 'EC 4.2.3.102';
UPDATE enzyme SET sys_name = 'Ent-copalyl-diphosphate diphosphate-lyase (cyclizing, ent-isokaurene-forming)' WHERE ec = 'EC 4.2.3.103';
UPDATE enzyme SET history = 'EC 4.2.3.103 created 2012' WHERE ec = 'EC 4.2.3.103';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (alpha-humulene-forming)' WHERE ec = 'EC 4.2.3.104';
UPDATE enzyme SET history = 'EC 4.2.3.104 created 2012' WHERE ec = 'EC 4.2.3.104';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing; tricyclene-forming)' WHERE ec = 'EC 4.2.3.105';
UPDATE enzyme SET history = 'EC 4.2.3.105 created 2012' WHERE ec = 'EC 4.2.3.105';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase ((E)-beta-ocimene-forming)' WHERE ec = 'EC 4.2.3.106';
UPDATE enzyme SET history = 'EC 4.2.3.106 created 2012' WHERE ec = 'EC 4.2.3.106';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (+)-car-3-ene-forming)' WHERE ec = 'EC 4.2.3.107';
UPDATE enzyme SET history = 'EC 4.2.3.107 created 2012' WHERE ec = 'EC 4.2.3.107';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, 1,8-cineole-forming)' WHERE ec = 'EC 4.2.3.108';
UPDATE enzyme SET history = 'EC 4.2.3.108 created 2012' WHERE ec = 'EC 4.2.3.108';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (-)-sabinene-forming)' WHERE ec = 'EC 4.2.3.109';
UPDATE enzyme SET history = 'EC 4.2.3.109 created 2012' WHERE ec = 'EC 4.2.3.109';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (+)-sabinene-forming)' WHERE ec = 'EC 4.2.3.110';
UPDATE enzyme SET history = 'EC 4.2.3.110 created 2012' WHERE ec = 'EC 4.2.3.110';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (-)-alpha-terpineol-forming)' WHERE ec = 'EC 4.2.3.111';
UPDATE enzyme SET history = 'EC 4.2.3.111 created 2012' WHERE ec = 'EC 4.2.3.111';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (+)-alpha-terpineol-forming)' WHERE ec = 'EC 4.2.3.112';
UPDATE enzyme SET history = 'EC 4.2.3.112 created 2012' WHERE ec = 'EC 4.2.3.112';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, terpinolene-forming)' WHERE ec = 'EC 4.2.3.113';
UPDATE enzyme SET history = 'EC 4.2.3.113 created 2012' WHERE ec = 'EC 4.2.3.113';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, gamma-terpinene-forming)' WHERE ec = 'EC 4.2.3.114';
UPDATE enzyme SET history = 'EC 4.2.3.114 created 2012' WHERE ec = 'EC 4.2.3.114';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, alpha-terpinene-forming)' WHERE ec = 'EC 4.2.3.115';
UPDATE enzyme SET history = 'EC 4.2.3.115 created 2012' WHERE ec = 'EC 4.2.3.115';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (+)-camphene-forming)' WHERE ec = 'EC 4.2.3.116';
UPDATE enzyme SET history = 'EC 4.2.3.116 created 2012' WHERE ec = 'EC 4.2.3.116';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (-)-camphene-forming)' WHERE ec = 'EC 4.2.3.117';
UPDATE enzyme SET history = 'EC 4.2.3.117 created 2012' WHERE ec = 'EC 4.2.3.117';
UPDATE enzyme SET sys_name = '(E)-2-methylgeranyl diphosphate lyase (cyclizing; 2-methylisoborneol-forming)' WHERE ec = 'EC 4.2.3.118';
UPDATE enzyme SET history = 'EC 4.2.3.118 created 2011' WHERE ec = 'EC 4.2.3.118';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (-)-(S)-limonene,(-)-alpha-pinene-forming)' WHERE ec = 'EC 4.2.3.119';
UPDATE enzyme SET history = 'EC 4.2.3.119 created 2012' WHERE ec = 'EC 4.2.3.119';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (-)-beta-pinene-forming)' WHERE ec = 'EC 4.2.3.120';
UPDATE enzyme SET history = 'EC 4.2.3.120 created 2012' WHERE ec = 'EC 4.2.3.120';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase (cyclizing, (+)-alpha-pinene-forming)' WHERE ec = 'EC 4.2.3.121';
UPDATE enzyme SET history = 'EC 4.2.3.121 created 2012' WHERE ec = 'EC 4.2.3.121';
UPDATE enzyme SET sys_name = 'Geranyl-diphosphate diphosphate-lyase ((+)-beta-pinene-forming)' WHERE ec = 'EC 4.2.3.122';
UPDATE enzyme SET history = 'EC 4.2.3.122 created 2012' WHERE ec = 'EC 4.2.3.122';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl diphosphate diphosphate-lyase (cyclizing, beta-sesquiphellandrene-forming)' WHERE ec = 'EC 4.2.3.123';
UPDATE enzyme SET history = 'EC 4.2.3.123 created 2012' WHERE ec = 'EC 4.2.3.123';
UPDATE enzyme SET sys_name = 'D-glucose-6-phosphate phosphate-lyase  (2-deoxy-scyllo-inosose-forming)' WHERE ec = 'EC 4.2.3.124';
UPDATE enzyme SET history = 'EC 4.2.3.124 created 2012' WHERE ec = 'EC 4.2.3.124';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl diphosphate-diphosphate-lyase (cyclizing, alpha-muurolene-forming)' WHERE ec = 'EC 4.2.3.125';
UPDATE enzyme SET history = 'EC 4.2.3.125 created 2012' WHERE ec = 'EC 4.2.3.125';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lase (cyclizing, gamma-muurolene-forming)' WHERE ec = 'EC 4.2.3.126';
UPDATE enzyme SET history = 'EC 4.2.3.126 created 2012' WHERE ec = 'EC 4.2.3.126';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, beta-copaene-forming)' WHERE ec = 'EC 4.2.3.127';
UPDATE enzyme SET history = 'EC 4.2.3.127 created 2012' WHERE ec = 'EC 4.2.3.127';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, beta-cubebene-forming)' WHERE ec = 'EC 4.2.3.128';
UPDATE enzyme SET history = 'EC 4.2.3.128 created 2012' WHERE ec = 'EC 4.2.3.128';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, (+)-sativene-forming)' WHERE ec = 'EC 4.2.3.129';
UPDATE enzyme SET history = 'EC 4.2.3.129 created 2012' WHERE ec = 'EC 4.2.3.129';
UPDATE enzyme SET sys_name = 'All-trans-heptaprenyl-diphosphate diphosphate-lyase (cyclizing, tetraprenyl-beta-curcumene-forming)' WHERE ec = 'EC 4.2.3.130';
UPDATE enzyme SET history = 'EC 4.2.3.130 created 2012' WHERE ec = 'EC 4.2.3.130';
UPDATE enzyme SET sys_name = '(+)-copaly-diphosphate diphosphate-lyase (cyclizing, miltiradiene-forming)' WHERE ec = 'EC 4.2.3.131';
UPDATE enzyme SET history = 'EC 4.2.3.131 created 2012' WHERE ec = 'EC 4.2.3.131';
UPDATE enzyme SET sys_name = '(+)-copaly-diphosphate diphosphate-lyase (cyclizing, neoabietadiene-forming)' WHERE ec = 'EC 4.2.3.132';
UPDATE enzyme SET history = 'EC 4.2.3.132 created 2012' WHERE ec = 'EC 4.2.3.132';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, alpha-copaene-forming)' WHERE ec = 'EC 4.2.3.133';
UPDATE enzyme SET history = 'EC 4.2.3.133 created 2012' WHERE ec = 'EC 4.2.3.133';
UPDATE enzyme SET sys_name = '(5R)-5-phosphonooxy-L-lysine phosphate-lyase (deaminating; (S)-2-amino-6-oxohexanoate-forming)' WHERE ec = 'EC 4.2.3.134';
UPDATE enzyme SET history = 'EC 4.2.3.134 created 2012' WHERE ec = 'EC 4.2.3.134';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, Delta(6)-protoilludene-forming)' WHERE ec = 'EC 4.2.3.135';
UPDATE enzyme SET history = 'EC 4.2.3.135 created 2012' WHERE ec = 'EC 4.2.3.135';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, (-)-alpha-isocomene-forming)' WHERE ec = 'EC 4.2.3.136';
UPDATE enzyme SET history = 'EC 4.2.3.136 created 2012' WHERE ec = 'EC 4.2.3.136';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, (E)-2-epi-beta-caryophyllene-forming)' WHERE ec = 'EC 4.2.3.137';
UPDATE enzyme SET history = 'EC 4.2.3.137 created 2012' WHERE ec = 'EC 4.2.3.137';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, (+)-epi-alpha-bisabolol-forming)' WHERE ec = 'EC 4.2.3.138';
UPDATE enzyme SET history = 'EC 4.2.3.138 created 2012' WHERE ec = 'EC 4.2.3.138';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (cyclizing, valerena-4,7(11)-diene-forming)' WHERE ec = 'EC 4.2.3.139';
UPDATE enzyme SET history = 'EC 4.2.3.139 created 2012' WHERE ec = 'EC 4.2.3.139';
UPDATE enzyme SET sys_name = '(13E)-8-alpha-hydroxylabd-13-en-15-yl-diphosphate-lyase (cis-abienol forming)' WHERE ec = 'EC 4.2.3.140';
UPDATE enzyme SET history = 'EC 4.2.3.140 created 2012' WHERE ec = 'EC 4.2.3.140';
UPDATE enzyme SET sys_name = '(13E)-8-alpha-hydroxylabd-13-en-15-yl-diphosphate-lyase (sclareol forming)' WHERE ec = 'EC 4.2.3.141';
UPDATE enzyme SET history = 'EC 4.2.3.141 created 2013' WHERE ec = 'EC 4.2.3.141';
UPDATE enzyme SET sys_name = '(2Z,6Z)-farnesyl-diphosphate lyase (cyclizing; 7-epizingiberene-forming)' WHERE ec = 'EC 4.2.3.142';
UPDATE enzyme SET history = 'EC 4.2.3.142 created 2013' WHERE ec = 'EC 4.2.3.142';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (kunzeaol forming)' WHERE ec = 'EC 4.2.3.143';
UPDATE enzyme SET history = 'EC 4.2.3.143 created 2013' WHERE ec = 'EC 4.2.3.143';
UPDATE enzyme SET sys_name = 'Geranylgeranyl diphosphate diphosphate-lyase ((E,E)-geranyllinalool-forming)' WHERE ec = 'EC 4.2.3.144';
UPDATE enzyme SET history = 'EC 4.2.3.144 created 2013' WHERE ec = 'EC 4.2.3.144';
UPDATE enzyme SET sys_name = 'Farnesyl-diphosphate diphosphate-lyase (cyclizing, delta-selinene-forming)' WHERE ec = 'EC 4.2.3.n2';
UPDATE enzyme SET history = 'EC 4.2.3.n2 created 2010' WHERE ec = 'EC 4.2.3.n2';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.2.3.n3';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.2.3.n4';
UPDATE enzyme SET history = 'EC 4.2.3.n5 created 2010' WHERE ec = 'EC 4.2.3.n5';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.2.3.n6';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.2.3.n7';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.2.3.n8';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.2.3.n10';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesyl-diphosphate diphosphate-lyase (selinene-forming)' WHERE ec = 'EC 4.2.3.n11';
UPDATE enzyme SET history = 'EC 4.2.3.n11 created 2011' WHERE ec = 'EC 4.2.3.n11';
UPDATE enzyme SET history = 'EC 4.2.3.n12 created 2011' WHERE ec = 'EC 4.2.3.n12';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.2.3.n13';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.2.3.n14';
UPDATE enzyme SET history = 'EC 4.2.99.1 created 1961, deleted 1972' WHERE ec = 'EC 4.2.99.1';
UPDATE enzyme SET history = 'EC 4.2.99.2 created 1961, deleted 2000' WHERE ec = 'EC 4.2.99.2';
UPDATE enzyme SET history = 'EC 4.2.99.3 created 1965, deleted 1972' WHERE ec = 'EC 4.2.99.3';
UPDATE enzyme SET history = 'EC 4.2.99.4 created 1965, deleted 1972' WHERE ec = 'EC 4.2.99.4';
UPDATE enzyme SET history = 'EC 4.2.99.5 created 1965, deleted 1972' WHERE ec = 'EC 4.2.99.5';
UPDATE enzyme SET history = 'EC 4.2.99.6 created 1965, deleted 1972' WHERE ec = 'EC 4.2.99.6';
UPDATE enzyme SET history = 'EC 4.2.99.7 created 1972, deleted 2000' WHERE ec = 'EC 4.2.99.7';
UPDATE enzyme SET history = 'EC 4.2.99.8 created 1972, modified 1976, modified 1990, deleted 2002' WHERE ec = 'EC 4.2.99.8';
UPDATE enzyme SET history = 'EC 4.2.99.9 created 1972, deleted 2002' WHERE ec = 'EC 4.2.99.9';
UPDATE enzyme SET history = 'EC 4.2.99.10 created 1972, deleted 2002' WHERE ec = 'EC 4.2.99.10';
UPDATE enzyme SET history = 'EC 4.2.99.11 created 1972, deleted 2000' WHERE ec = 'EC 4.2.99.11';
UPDATE enzyme SET sys_name = 'Carboxymethyloxysuccinate glycolate-lyase (fumarate-forming)' WHERE ec = 'EC 4.2.99.12';
UPDATE enzyme SET history = 'EC 4.2.99.12 created 1976' WHERE ec = 'EC 4.2.99.12';
UPDATE enzyme SET history = 'EC 4.2.99.13 created 1984, deleted 2002' WHERE ec = 'EC 4.2.99.13';
UPDATE enzyme SET history = 'EC 4.2.99.14 created 1989, deleted 2002' WHERE ec = 'EC 4.2.99.14';
UPDATE enzyme SET history = 'EC 4.2.99.15 created 1989, deleted 2002' WHERE ec = 'EC 4.2.99.15';
UPDATE enzyme SET history = 'EC 4.2.99.16 created 1990, deleted 2002' WHERE ec = 'EC 4.2.99.16';
UPDATE enzyme SET history = 'EC 4.2.99.17 created 1992, deleted 1992' WHERE ec = 'EC 4.2.99.17';
UPDATE enzyme SET sys_name = 'DNA-(apurinic or apyrimidinic site) 5-phosphomonoester-lyase' WHERE ec = 'EC 4.2.99.18';
UPDATE enzyme SET history = 'EC 4.2.99.18 created 1978 as EC 3.1.25.2, transferred 1992 to EC 4.2.99.18' WHERE ec = 'EC 4.2.99.18';
UPDATE enzyme SET history = 'EC 4.2.99.19 created 2001, deleted 2005' WHERE ec = 'EC 4.2.99.19';
UPDATE enzyme SET sys_name = '5-enolpyruvoyl-6-hydroxy-2-succinylcyclohex-3-ene-1-carboxylate pyruvate-lyase ((1R,6R)-6-hydroxy-2-succinylcyclohexa-2,4-diene-1-carboxylate-forming)' WHERE ec = 'EC 4.2.99.20';
UPDATE enzyme SET history = 'EC 4.2.99.20 created 2008 (EC 2.5.1.64 created 2003, part-incorporated 2008)' WHERE ec = 'EC 4.2.99.20';
UPDATE enzyme SET sys_name = 'Isochorismate pyruvate-lyase (salicylate-forming)' WHERE ec = 'EC 4.2.99.21';
UPDATE enzyme SET history = 'EC 4.2.99.1 created 2010' WHERE ec = 'EC 4.2.99.21';
UPDATE enzyme SET sys_name = '6-tuliposide A D-glucose-lyase (tulipalin A forming)' WHERE ec = 'EC 4.2.99.22';
UPDATE enzyme SET history = 'EC 4.2.99.22 created 2013' WHERE ec = 'EC 4.2.99.22';
UPDATE enzyme SET sys_name = 'L-aspartate ammonia-lyase (fumarate-forming)' WHERE ec = 'EC 4.3.1.1';
UPDATE enzyme SET history = 'EC 4.3.1.1 created 1961' WHERE ec = 'EC 4.3.1.1';
UPDATE enzyme SET sys_name = 'L-threo-3-methylaspartate ammonia-lyase (mesaconate-forming)' WHERE ec = 'EC 4.3.1.2';
UPDATE enzyme SET history = 'EC 4.3.1.2 created 1961' WHERE ec = 'EC 4.3.1.2';
UPDATE enzyme SET sys_name = 'L-histidine ammonia-lyase (urocanate-forming)' WHERE ec = 'EC 4.3.1.3';
UPDATE enzyme SET history = 'EC 4.3.1.3 created 1961, modified 2008' WHERE ec = 'EC 4.3.1.3';
UPDATE enzyme SET sys_name = '5-formimidoyltetrahydrofolate ammonia-lyase (cyclizing; 5,10-methenyltetrahydrofolate-forming)' WHERE ec = 'EC 4.3.1.4';
UPDATE enzyme SET history = 'EC 4.3.1.4 created 1961, modified 2000' WHERE ec = 'EC 4.3.1.4';
UPDATE enzyme SET history = 'EC 4.3.1.5 created 1965, deleted 2008' WHERE ec = 'EC 4.3.1.5';
UPDATE enzyme SET sys_name = 'Beta-alanyl-CoA ammonia-lyase (acryloyl-CoA-forming)' WHERE ec = 'EC 4.3.1.6';
UPDATE enzyme SET history = 'EC 4.3.1.6 created 1965' WHERE ec = 'EC 4.3.1.6';
UPDATE enzyme SET sys_name = 'Ethanolamine ammonia-lyase (acetaldehyde-forming)' WHERE ec = 'EC 4.3.1.7';
UPDATE enzyme SET history = 'EC 4.3.1.7 created 1972' WHERE ec = 'EC 4.3.1.7';
UPDATE enzyme SET history = 'EC 4.3.1.8 created 1972, modified 1982, modified 1989, deleted 2003' WHERE ec = 'EC 4.3.1.8';
UPDATE enzyme SET sys_name = 'D-glucosaminate ammonia-lyase (isomerizing; 2-dehydro-3-deoxy-D-gluconate-forming)' WHERE ec = 'EC 4.3.1.9';
UPDATE enzyme SET history = 'EC 4.3.1.9 created 1972 (EC 4.3.1.21 created 1965 as EC 4.2.1.26, transferred 2002 to EC 4.3.1.21, incorporated 2004)' WHERE ec = 'EC 4.3.1.9';
UPDATE enzyme SET sys_name = 'L-serine-O-sulfate ammonia-lyase (pyruvate-forming)' WHERE ec = 'EC 4.3.1.10';
UPDATE enzyme SET history = 'EC 4.3.1.10 created 1972' WHERE ec = 'EC 4.3.1.10';
UPDATE enzyme SET history = 'EC 4.3.1.11 created 1972, deleted 2007' WHERE ec = 'EC 4.3.1.11';
UPDATE enzyme SET sys_name = 'L-ornithine ammonia-lyase (cyclizing; L-proline-forming)' WHERE ec = 'EC 4.3.1.12';
UPDATE enzyme SET history = 'EC 4.3.1.12 created 1976' WHERE ec = 'EC 4.3.1.12';
UPDATE enzyme SET sys_name = 'O-carbamoyl-L-serine ammonia-lyase (decarboxylating; pyruvate-forming)' WHERE ec = 'EC 4.3.1.13';
UPDATE enzyme SET history = 'EC 4.3.1.13 created 1976' WHERE ec = 'EC 4.3.1.13';
UPDATE enzyme SET sys_name = 'L-3-aminobutyryl-CoA ammonia-lyase (crotonoyl-CoA-forming)' WHERE ec = 'EC 4.3.1.14';
UPDATE enzyme SET history = 'EC 4.3.1.14 created 1999' WHERE ec = 'EC 4.3.1.14';
UPDATE enzyme SET sys_name = '2,3-diaminopropanoate ammonia-lyase (adding water; pyruvate-forming)' WHERE ec = 'EC 4.3.1.15';
UPDATE enzyme SET history = 'EC 4.3.1.15 created 1999' WHERE ec = 'EC 4.3.1.15';
UPDATE enzyme SET sys_name = 'Threo-3-hydroxy-L-aspartate ammonia-lyase (oxaloacetate-forming)' WHERE ec = 'EC 4.3.1.16';
UPDATE enzyme SET history = 'EC 4.3.1.16 created 2001, modified 2010' WHERE ec = 'EC 4.3.1.16';
UPDATE enzyme SET sys_name = 'L-serine ammonia-lyase (pyruvate-forming)' WHERE ec = 'EC 4.3.1.17';
UPDATE enzyme SET history = 'EC 4.3.1.17 created 1961 as EC 4.2.1.13, transfered 2001 to EC 4.3.1.17, modified 2014' WHERE ec = 'EC 4.3.1.17';
UPDATE enzyme SET sys_name = 'D-serine ammonia-lyase (pyruvate-forming)' WHERE ec = 'EC 4.3.1.18';
UPDATE enzyme SET history = 'EC 4.3.1.18 created 1961 as EC 4.2.1.14, transfered 2001 to EC 4.3.1.18' WHERE ec = 'EC 4.3.1.18';
UPDATE enzyme SET sys_name = 'L-threonine ammonia-lyase (2-oxobutanoate-forming)' WHERE ec = 'EC 4.3.1.19';
UPDATE enzyme SET history = 'EC 4.3.1.19 created 1961 as EC 4.2.1.16, transferred 2001 to EC 4.3.1.19, modified 2014' WHERE ec = 'EC 4.3.1.19';
UPDATE enzyme SET sys_name = 'Erythro-3-hydroxy-L-aspartate ammonia-lyase (oxaloacetate-forming)' WHERE ec = 'EC 4.3.1.20';
UPDATE enzyme SET history = 'EC 4.3.1.20 created 1972 as EC 4.2.1.38, transfered 2001 to EC 4.3.1.20, modified 2010' WHERE ec = 'EC 4.3.1.20';
UPDATE enzyme SET history = 'EC 4.3.1.21 created 1965 as EC 4.2.1.26, transferred 2002 to EC 4.3.1.21, deleted 2004' WHERE ec = 'EC 4.3.1.21';
UPDATE enzyme SET sys_name = '3,4-dihydroxy-L-phenylalanine ammonia-lyase (3,4-dihydroxyphenylpropanoate-forming)' WHERE ec = 'EC 4.3.1.22';
UPDATE enzyme SET history = 'EC 4.3.1.22 created 2007' WHERE ec = 'EC 4.3.1.22';
UPDATE enzyme SET sys_name = 'L-tyrosine ammonia-lyase (trans-p-hydroxycinnamate-forming)' WHERE ec = 'EC 4.3.1.23';
UPDATE enzyme SET history = 'EC 4.3.1.23 created 2008 (EC 4.3.1.5 created 1965, part-incorporated 2008)' WHERE ec = 'EC 4.3.1.23';
UPDATE enzyme SET sys_name = 'L-phenylalanine ammonia-lyase (trans-cinnamate-forming)' WHERE ec = 'EC 4.3.1.24';
UPDATE enzyme SET history = 'EC 4.3.1.24 created 2008 (EC 4.3.1.5 created 1965, part-incorporated 2008)' WHERE ec = 'EC 4.3.1.24';
UPDATE enzyme SET sys_name = 'L-phenylalanine(or L-tyrosine):trans-cinnamate(or trans-p-hydroxycinnamate) ammonia-lyase' WHERE ec = 'EC 4.3.1.25';
UPDATE enzyme SET history = 'EC 4.3.1.25 created 2008 (EC 4.3.1.5 created 1965, part-incorporated 2008)' WHERE ec = 'EC 4.3.1.25';
UPDATE enzyme SET history = 'EC 4.3.1.26 created 2010, deleted 2013' WHERE ec = 'EC 4.3.1.26';
UPDATE enzyme SET sys_name = 'Threo-3-hydroxy-D-aspartate ammonia-lyase (oxaloacetate-forming)' WHERE ec = 'EC 4.3.1.27';
UPDATE enzyme SET history = 'EC 4.3.1.27 created 2010' WHERE ec = 'EC 4.3.1.27';
UPDATE enzyme SET sys_name = 'L-lysine ammonia-lyase (cyclizing; ammonia-forming)' WHERE ec = 'EC 4.3.1.28';
UPDATE enzyme SET history = 'EC 4.3.1.28 created 2012' WHERE ec = 'EC 4.3.1.28';
UPDATE enzyme SET sys_name = 'D-glucosaminate-6-phosphate ammonia-lyase (2-dehydro-3-deoxy-6-phospho-D-gluconate-forming)' WHERE ec = 'EC 4.3.1.29';
UPDATE enzyme SET history = 'EC 4.3.1.29 created 2013' WHERE ec = 'EC 4.3.1.29';
UPDATE enzyme SET sys_name = 'dTDP-4-amino-4,6-dideoxy-alpha-D-glucopyranose ammonia lyase (dTDP-3-dehydro-4,6-dideoxy-alpha-D-glucopyranose-forming)' WHERE ec = 'EC 4.3.1.30';
UPDATE enzyme SET history = 'EC 4.3.1.30 created 2011' WHERE ec = 'EC 4.3.1.30';
UPDATE enzyme SET sys_name = '2-(N(omega)-L-arginino)succinate arginine-lyase (fumarate-forming)' WHERE ec = 'EC 4.3.2.1';
UPDATE enzyme SET history = 'EC 4.3.2.1 created 1961' WHERE ec = 'EC 4.3.2.1';
UPDATE enzyme SET sys_name = 'N(6)-(1,2-dicarboxyethyl)AMP AMP-lyase (fumarate-forming)' WHERE ec = 'EC 4.3.2.2';
UPDATE enzyme SET history = 'EC 4.3.2.2 created 1961, modified 2000' WHERE ec = 'EC 4.3.2.2';
UPDATE enzyme SET sys_name = '(S)-ureidoglycolate urea-lyase (glyoxylate-forming)' WHERE ec = 'EC 4.3.2.3';
UPDATE enzyme SET history = 'EC 4.3.2.3 created 1972, modified 2014' WHERE ec = 'EC 4.3.2.3';
UPDATE enzyme SET sys_name = 'DNA-4,6-diamino-5-formamidopyrimidine C(8)-N(9)-lyase (cyclizing; DNA-adenine-forming)' WHERE ec = 'EC 4.3.2.4';
UPDATE enzyme SET history = 'EC 4.3.2.4 created 1989' WHERE ec = 'EC 4.3.2.4';
UPDATE enzyme SET sys_name = 'Peptidylamidoglycolate peptidylamide-lyase (glyoxylate-forming)' WHERE ec = 'EC 4.3.2.5';
UPDATE enzyme SET history = 'EC 4.3.2.5 created 1992' WHERE ec = 'EC 4.3.2.5';
UPDATE enzyme SET sys_name = 'Gamma-L-glutamyl-butirosin B gamma-glutamyl cyclotransferase (5-oxoproline producing)' WHERE ec = 'EC 4.3.2.6';
UPDATE enzyme SET history = 'EC 4.3.2.6 created 2012' WHERE ec = 'EC 4.3.2.6';
UPDATE enzyme SET sys_name = '4-nitrophenyl-3-ketovalidamine 4-nitroaniline-lyase (5-D-(5/6)-5-C-(hydroxymethyl)-2,6-dihydroxycyclohex-2-en-1-one-forming)' WHERE ec = 'EC 4.3.3.1';
UPDATE enzyme SET history = 'EC 4.3.3.1 created 1989' WHERE ec = 'EC 4.3.3.1';
UPDATE enzyme SET sys_name = '3-alpha-(S)-strictosidine tryptamine-lyase (secologanin-forming)' WHERE ec = 'EC 4.3.3.2';
UPDATE enzyme SET history = 'EC 4.3.3.2 created 1990' WHERE ec = 'EC 4.3.3.2';
UPDATE enzyme SET sys_name = 'Deacetylisoipecoside dopamine-lyase (secologanin-forming)' WHERE ec = 'EC 4.3.3.3';
UPDATE enzyme SET history = 'EC 4.3.3.3 created 2000' WHERE ec = 'EC 4.3.3.3';
UPDATE enzyme SET sys_name = 'Deacetylipecoside dopamine-lyase (secologanin-forming)' WHERE ec = 'EC 4.3.3.4';
UPDATE enzyme SET history = 'EC 4.3.3.4 created 2000' WHERE ec = 'EC 4.3.3.4';
UPDATE enzyme SET sys_name = '4-demethylrebeccamycin D-glucose-lyase' WHERE ec = 'EC 4.3.3.5';
UPDATE enzyme SET history = 'EC 4.3.3.5 created 2010' WHERE ec = 'EC 4.3.3.5';
UPDATE enzyme SET sys_name = 'D-ribose 5-phosphate,D-glyceraldehyde 3-phosphate pyridoxal 5-phosphate-lyase' WHERE ec = 'EC 4.3.3.6';
UPDATE enzyme SET history = 'EC 4.3.3.6 created 2011' WHERE ec = 'EC 4.3.3.6';
UPDATE enzyme SET sys_name = 'L-aspartate-4-semialdehyde hydro-lyase (adding pyruvate and cyclizing; (4S)-4-hydroxy-2,3,4,5-tetrahydro-(2S)-dipicolinate-forming)' WHERE ec = 'EC 4.3.3.7';
UPDATE enzyme SET history = 'EC 4.3.3.7 created 1972 as EC 4.2.1.52, transferred 2012 to EC 4.3.3.7' WHERE ec = 'EC 4.3.3.7';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 4.3.3.n1';
UPDATE enzyme SET history = 'EC 4.3.99.1 created 1972 as EC 3.5.5.3, transferred 1990 to EC 4.3.99.1, deleted 2001' WHERE ec = 'EC 4.3.99.1';
UPDATE enzyme SET sys_name = 'Carboxybiotinyl-[protein] carboxy-lyase' WHERE ec = 'EC 4.3.99.2';
UPDATE enzyme SET history = 'EC 4.3.99.2 created 2008' WHERE ec = 'EC 4.3.99.2';
UPDATE enzyme SET sys_name = '6-carboxy-5,6,7,8-tetrahydropterin ammonia-lyase' WHERE ec = 'EC 4.3.99.3';
UPDATE enzyme SET history = 'EC 4.3.99.3 created 2012' WHERE ec = 'EC 4.3.99.3';
UPDATE enzyme SET sys_name = 'Choline trimethylamine-lyase (acetaldehyde-forming)' WHERE ec = 'EC 4.3.99.4';
UPDATE enzyme SET history = 'EC 4.3.99.4 created 2013' WHERE ec = 'EC 4.3.99.4';
UPDATE enzyme SET sys_name = 'L-cystathionine cysteine-lyase (deaminating; 2-oxobutanoate-forming)' WHERE ec = 'EC 4.4.1.1';
UPDATE enzyme SET history = 'EC 4.4.1.1 created 1961 (EC 4.2.1.15 created 1961, incorporated 1972)' WHERE ec = 'EC 4.4.1.1';
UPDATE enzyme SET sys_name = 'L-homocysteine hydrogen-sulfide-lyase (deaminating; 2-oxobutanoate-forming)' WHERE ec = 'EC 4.4.1.2';
UPDATE enzyme SET history = 'EC 4.4.1.2 created 1961' WHERE ec = 'EC 4.4.1.2';
UPDATE enzyme SET sys_name = 'S,S-dimethyl-beta-propiothetin dimethyl-sulfide-lyase (acrylate-forming)' WHERE ec = 'EC 4.4.1.3';
UPDATE enzyme SET history = 'EC 4.4.1.3 created 1961' WHERE ec = 'EC 4.4.1.3';
UPDATE enzyme SET sys_name = 'S-alkyl-L-cysteine S-oxide alkyl-sulfenate-lyase (2-aminoacrylate-forming)' WHERE ec = 'EC 4.4.1.4';
UPDATE enzyme SET history = 'EC 4.4.1.4 created 1961' WHERE ec = 'EC 4.4.1.4';
UPDATE enzyme SET sys_name = '(R)-S-lactoylglutathione methylglyoxal-lyase (isomerizing; glutathione-forming)' WHERE ec = 'EC 4.4.1.5';
UPDATE enzyme SET history = 'EC 4.4.1.5 created 1961' WHERE ec = 'EC 4.4.1.5';
UPDATE enzyme SET sys_name = 'S-alkyl-L-cysteine alkylthiol-lyase (deaminating; pyruvate-forming)' WHERE ec = 'EC 4.4.1.6';
UPDATE enzyme SET history = 'EC 4.4.1.6 created 1965, deleted 1972, reinstated 1976' WHERE ec = 'EC 4.4.1.6';
UPDATE enzyme SET history = 'EC 4.4.1.7 created 1972, deleted 1976' WHERE ec = 'EC 4.4.1.7';
UPDATE enzyme SET sys_name = 'L-cystathionine L-homocysteine-lyase (deaminating; pyruvate-forming)' WHERE ec = 'EC 4.4.1.8';
UPDATE enzyme SET history = 'EC 4.4.1.8 created 1972' WHERE ec = 'EC 4.4.1.8';
UPDATE enzyme SET sys_name = 'L-cysteine hydrogen-sulfide-lyase (adding hydrogen cyanide; L-3-cyanoalanine-forming)' WHERE ec = 'EC 4.4.1.9';
UPDATE enzyme SET history = 'EC 4.4.1.9 created 1972, deleted 1976, reinstated 1978' WHERE ec = 'EC 4.4.1.9';
UPDATE enzyme SET sys_name = 'L-cysteine hydrogen-sulfide-lyase (adding sulfite; L-cysteate-forming)' WHERE ec = 'EC 4.4.1.10';
UPDATE enzyme SET history = 'EC 4.4.1.10 created 1972' WHERE ec = 'EC 4.4.1.10';
UPDATE enzyme SET sys_name = 'L-methionine methanethiol-lyase (deaminating; 2-oxobutanoate-forming)' WHERE ec = 'EC 4.4.1.11';
UPDATE enzyme SET history = 'EC 4.4.1.11 created 1976' WHERE ec = 'EC 4.4.1.11';
UPDATE enzyme SET history = 'EC 4.4.1.12 created 1976, deleted 2003' WHERE ec = 'EC 4.4.1.12';
UPDATE enzyme SET sys_name = 'L-cysteine-S-conjugate thiol-lyase (deaminating; pyruvate-forming)' WHERE ec = 'EC 4.4.1.13';
UPDATE enzyme SET history = 'EC 4.4.1.13 created 1981' WHERE ec = 'EC 4.4.1.13';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine methylthioadenosine-lyase (1-aminocyclopropane-1-carboxylate-forming)' WHERE ec = 'EC 4.4.1.14';
UPDATE enzyme SET history = 'EC 4.4.1.14 created 1984' WHERE ec = 'EC 4.4.1.14';
UPDATE enzyme SET sys_name = 'D-cysteine sulfide-lyase (deaminating; pyruvate-forming)' WHERE ec = 'EC 4.4.1.15';
UPDATE enzyme SET history = 'EC 4.4.1.15 created 1986' WHERE ec = 'EC 4.4.1.15';
UPDATE enzyme SET sys_name = 'L-selenocysteine selenide-lyase (L-alanine-forming)' WHERE ec = 'EC 4.4.1.16';
UPDATE enzyme SET history = 'EC 4.4.1.16 created 1986' WHERE ec = 'EC 4.4.1.16';
UPDATE enzyme SET sys_name = 'Holocytochrome-c apocytochrome-c-lyase (heme-forming)' WHERE ec = 'EC 4.4.1.17';
UPDATE enzyme SET history = 'EC 4.4.1.17 created 1990' WHERE ec = 'EC 4.4.1.17';
UPDATE enzyme SET history = 'EC 4.4.1.18 created 2000, deleted 2002' WHERE ec = 'EC 4.4.1.18';
UPDATE enzyme SET sys_name = '(2R)-2-O-phospho-3-sulfolactate hydrogen-sulfite-lyase (phosphoenolpyruvate-forming)' WHERE ec = 'EC 4.4.1.19';
UPDATE enzyme SET history = 'EC 4.4.1.19 created 2003' WHERE ec = 'EC 4.4.1.19';
UPDATE enzyme SET sys_name = 'Leukotriene-C(4) glutathione-lyase (leukotriene-A(4)-forming)' WHERE ec = 'EC 4.4.1.20';
UPDATE enzyme SET history = 'EC 4.4.1.20 created 1989 as EC 2.5.1.37, transferred 2004 to EC 4.4.1.20' WHERE ec = 'EC 4.4.1.20';
UPDATE enzyme SET sys_name = 'S-(5-deoxy-D-ribos-5-yl)-L-homocysteine L-homocysteine-lyase ((4S)-4,5-dihydroxypentan-2,3-dione-forming)' WHERE ec = 'EC 4.4.1.21';
UPDATE enzyme SET history = 'EC 4.4.1.21 created 2004' WHERE ec = 'EC 4.4.1.21';
UPDATE enzyme SET sys_name = 'S-(hydroxymethyl)glutathione formaldehyde-lyase (glutathione-forming)' WHERE ec = 'EC 4.4.1.22';
UPDATE enzyme SET history = 'EC 4.4.1.22 created 2005 (EC 1.2.1.1 created 1961, modified 1982, modified 2002, part transferred 2005 to EC 4.4.1.22)' WHERE ec = 'EC 4.4.1.22';
UPDATE enzyme SET sys_name = '(R)-(or (S)-)2-hydroxypropyl-CoM:2-mercaptoethanesulfonate lyase (epoxyalkane-ring-forming)' WHERE ec = 'EC 4.4.1.23';
UPDATE enzyme SET history = 'EC 4.4.1.23 created 2001 as EC 4.2.99.19, transferred 2005 to EC 4.4.1.23' WHERE ec = 'EC 4.4.1.23';
UPDATE enzyme SET sys_name = '(2R)-3-sulfolactate bisulfite-lyase (pyruvate-forming)' WHERE ec = 'EC 4.4.1.24';
UPDATE enzyme SET history = 'EC 4.4.1.24 created 2006, modified 2011' WHERE ec = 'EC 4.4.1.24';
UPDATE enzyme SET sys_name = 'L-cysteate bisulfite-lyase (deaminating; pyruvate-forming)' WHERE ec = 'EC 4.4.1.25';
UPDATE enzyme SET history = 'EC 4.4.1.25 created 2006' WHERE ec = 'EC 4.4.1.25';
UPDATE enzyme SET sys_name = '3,5,7-trioxododecanoyl-CoA CoA-lyase (2,4-dihydroxy-6-pentylbenzoate-forming)' WHERE ec = 'EC 4.4.1.26';
UPDATE enzyme SET history = 'EC 4.4.1.26 created 2012' WHERE ec = 'EC 4.4.1.26';
UPDATE enzyme SET sys_name = 'Carbon disulfide-lyase (hydrogen sulfide-producing)' WHERE ec = 'EC 4.4.1.27';
UPDATE enzyme SET history = 'EC 4.4.1.27 created 2013' WHERE ec = 'EC 4.4.1.27';
UPDATE enzyme SET sys_name = '1,1,1-trichloro-2,2-bis(4-chlorophenyl)ethane chloride-lyase (1,1-dichloro-2,2-bis(4-chlorophenyl)ethylene-forming)' WHERE ec = 'EC 4.5.1.1';
UPDATE enzyme SET history = 'EC 4.5.1.1 created 1961' WHERE ec = 'EC 4.5.1.1';
UPDATE enzyme SET sys_name = '3-chloro-D-alanine chloride-lyase (deaminating; pyruvate-forming)' WHERE ec = 'EC 4.5.1.2';
UPDATE enzyme SET history = 'EC 4.5.1.2 created 1984' WHERE ec = 'EC 4.5.1.2';
UPDATE enzyme SET sys_name = 'Dichloromethane chloride-lyase (adding H(2)O; chloride-hydrolyzing; formaldehyde-forming)' WHERE ec = 'EC 4.5.1.3';
UPDATE enzyme SET history = 'EC 4.5.1.3 created 1989' WHERE ec = 'EC 4.5.1.3';
UPDATE enzyme SET sys_name = 'L-2-amino-4-chloropent-4-enoate chloride-lyase (adding water; deaminating; 2-oxopent-4-enoate-forming)' WHERE ec = 'EC 4.5.1.4';
UPDATE enzyme SET history = 'EC 4.5.1.4 created 1990' WHERE ec = 'EC 4.5.1.4';
UPDATE enzyme SET sys_name = '3-chloro-L-alanine chloride-lyase (adding thioglycolate; S-carboxymethyl-L-cysteine-forming)' WHERE ec = 'EC 4.5.1.5';
UPDATE enzyme SET history = 'EC 4.5.1.5 created 1992' WHERE ec = 'EC 4.5.1.5';
UPDATE enzyme SET sys_name = 'ATP diphosphate-lyase (cyclizing; 3,5-cyclic-AMP-forming)' WHERE ec = 'EC 4.6.1.1';
UPDATE enzyme SET history = 'EC 4.6.1.1 created 1972' WHERE ec = 'EC 4.6.1.1';
UPDATE enzyme SET sys_name = 'GTP diphosphate-lyase (cyclizing; 3,5-cyclic-GMP-forming)' WHERE ec = 'EC 4.6.1.2';
UPDATE enzyme SET history = 'EC 4.6.1.2 created 1972' WHERE ec = 'EC 4.6.1.2';
UPDATE enzyme SET history = 'EC 4.6.1.3 created 1978, deleted 2000' WHERE ec = 'EC 4.6.1.3';
UPDATE enzyme SET history = 'EC 4.6.1.4 created 1978, modified 1983, deleted 2000' WHERE ec = 'EC 4.6.1.4';
UPDATE enzyme SET history = 'EC 4.6.1.5 created 1989, deleted 2000' WHERE ec = 'EC 4.6.1.5';
UPDATE enzyme SET sys_name = 'CTP diphosphate-lyase (cyclizing; 3,5-cyclic-CMP-forming)' WHERE ec = 'EC 4.6.1.6';
UPDATE enzyme SET history = 'EC 4.6.1.6 created 1989' WHERE ec = 'EC 4.6.1.6';
UPDATE enzyme SET history = 'EC 4.6.1.7 created 1989, deleted 2000' WHERE ec = 'EC 4.6.1.7';
UPDATE enzyme SET history = 'EC 4.6.1.8 created 1992, deleted 2000' WHERE ec = 'EC 4.6.1.8';
UPDATE enzyme SET history = 'EC 4.6.1.9 created 1992, deleted 2000' WHERE ec = 'EC 4.6.1.9';
UPDATE enzyme SET history = 'EC 4.6.1.10 created 1999, deleted 2000' WHERE ec = 'EC 4.6.1.10';
UPDATE enzyme SET history = 'EC 4.6.1.11 created 1999, deleted 2000' WHERE ec = 'EC 4.6.1.11';
UPDATE enzyme SET sys_name = '2-phospho-4-(cytidine 5-diphospho)-2-C-methyl-D-erythritol CMP-lyase (cyclizing; 2-C-methyl-D-erythritol 2,4-cyclodiphosphate-forming)' WHERE ec = 'EC 4.6.1.12';
UPDATE enzyme SET history = 'EC 4.6.1.12 created 2001' WHERE ec = 'EC 4.6.1.12';
UPDATE enzyme SET sys_name = '1-phosphatidyl-1D-myo-inositol 1,2-diacyl-sn-glycerol-lyase (1D-myo-inositol-1,2-cyclic-phosphate-forming)' WHERE ec = 'EC 4.6.1.13';
UPDATE enzyme SET history = 'EC 4.6.1.13 created 1972 as EC 3.1.4.10, modified 1976, transferred 2002 to EC 4.6.1.13' WHERE ec = 'EC 4.6.1.13';
UPDATE enzyme SET sys_name = '6-(alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol 1,2-diacyl-sn-glycerol-lyase (6-(alpha-D-glucosaminyl)-1D-myo-inositol 1,2-cyclic phosphate-forming)' WHERE ec = 'EC 4.6.1.14';
UPDATE enzyme SET history = 'EC 4.6.1.14 created 1989 as EC 3.1.4.47, transferred 2002 to EC 4.6.1.14' WHERE ec = 'EC 4.6.1.14';
UPDATE enzyme SET sys_name = 'FAD AMP-lyase (riboflavin-cyclic-4,5-phosphate-forming)' WHERE ec = 'EC 4.6.1.15';
UPDATE enzyme SET history = 'EC 4.6.1.15 created 2002' WHERE ec = 'EC 4.6.1.15';
UPDATE enzyme SET sys_name = 'PretRNA lyase (intron-removing; cyclic-2,3-phosphate-forming)' WHERE ec = 'EC 4.6.1.16';
UPDATE enzyme SET history = 'EC 4.6.1.16 created 1992 as EC 3.1.27.9, transferred 2014 to EC 4.6.1.16' WHERE ec = 'EC 4.6.1.16';
UPDATE enzyme SET sys_name = 'Alpha-D-ribose-1-methylphosphonate-5-phosphate C-P-lyase (methane forming)' WHERE ec = 'EC 4.7.1.1';
UPDATE enzyme SET history = 'EC 4.7.1.1 created 2013' WHERE ec = 'EC 4.7.1.1';
UPDATE enzyme SET sys_name = 'Protoheme ferro-lyase (protoporphyrin-forming)' WHERE ec = 'EC 4.99.1.1';
UPDATE enzyme SET history = 'EC 4.99.1.1 created 1965' WHERE ec = 'EC 4.99.1.1';
UPDATE enzyme SET sys_name = 'Alkylmercury mercuric-lyase (alkane-forming)' WHERE ec = 'EC 4.99.1.2';
UPDATE enzyme SET history = 'EC 4.99.1.2 created 1978' WHERE ec = 'EC 4.99.1.2';
UPDATE enzyme SET sys_name = 'Cobalt-sirohydrochlorin cobalt-lyase (sirohydrochlorin-forming)' WHERE ec = 'EC 4.99.1.3';
UPDATE enzyme SET history = 'EC 4.99.1.3 created 2004' WHERE ec = 'EC 4.99.1.3';
UPDATE enzyme SET sys_name = 'Siroheme ferro-lyase (sirohydrochlorin-forming)' WHERE ec = 'EC 4.99.1.4';
UPDATE enzyme SET history = 'EC 4.99.1.4 created 2004' WHERE ec = 'EC 4.99.1.4';
UPDATE enzyme SET sys_name = 'Aliphatic aldoxime hydro-lyase (aliphatic-nitrile-forming)' WHERE ec = 'EC 4.99.1.5';
UPDATE enzyme SET history = 'EC 4.99.1.5 created 2004' WHERE ec = 'EC 4.99.1.5';
UPDATE enzyme SET sys_name = '(Indol-3-yl)acetaldehyde-oxime hydro-lyase ((indol-3-yl)acetonitrile-forming)' WHERE ec = 'EC 4.99.1.6';
UPDATE enzyme SET history = 'EC 4.99.1.6 created 1965 as EC 4.2.1.29, transferred 2004 to EC 4.99.1.6' WHERE ec = 'EC 4.99.1.6';
UPDATE enzyme SET sys_name = '(Z)-phenylacetaldehyde-oxime hydro-lyase (phenylacetonitrile-forming)' WHERE ec = 'EC 4.99.1.7';
UPDATE enzyme SET history = 'EC 4.99.1.7 created 2005' WHERE ec = 'EC 4.99.1.7';
UPDATE enzyme SET sys_name = 'Fe(3+):ferriprotoporphyrin IX ligase (beta-hematin-forming)' WHERE ec = 'EC 4.99.1.8';
UPDATE enzyme SET history = 'EC 4.99.1.8 created 2009' WHERE ec = 'EC 4.99.1.8';
UPDATE enzyme SET sys_name = 'Alanine racemase' WHERE ec = 'EC 5.1.1.1';
UPDATE enzyme SET history = 'EC 5.1.1.1 created 1961' WHERE ec = 'EC 5.1.1.1';
UPDATE enzyme SET sys_name = 'Methionine racemase' WHERE ec = 'EC 5.1.1.2';
UPDATE enzyme SET history = 'EC 5.1.1.2 created 1961' WHERE ec = 'EC 5.1.1.2';
UPDATE enzyme SET sys_name = 'Glutamate racemase' WHERE ec = 'EC 5.1.1.3';
UPDATE enzyme SET history = 'EC 5.1.1.3 created 1961' WHERE ec = 'EC 5.1.1.3';
UPDATE enzyme SET sys_name = 'Proline racemase' WHERE ec = 'EC 5.1.1.4';
UPDATE enzyme SET history = 'EC 5.1.1.4 created 1961' WHERE ec = 'EC 5.1.1.4';
UPDATE enzyme SET sys_name = 'Lysine racemase' WHERE ec = 'EC 5.1.1.5';
UPDATE enzyme SET history = 'EC 5.1.1.5 created 1961' WHERE ec = 'EC 5.1.1.5';
UPDATE enzyme SET sys_name = 'Threonine racemase' WHERE ec = 'EC 5.1.1.6';
UPDATE enzyme SET history = 'EC 5.1.1.6 created 1961, modified 1981' WHERE ec = 'EC 5.1.1.6';
UPDATE enzyme SET sys_name = 'LL-2,6-diaminoheptanedioate 2-epimerase' WHERE ec = 'EC 5.1.1.7';
UPDATE enzyme SET history = 'EC 5.1.1.7 created 1961' WHERE ec = 'EC 5.1.1.7';
UPDATE enzyme SET sys_name = '4-hydroxyproline 2-epimerase' WHERE ec = 'EC 5.1.1.8';
UPDATE enzyme SET history = 'EC 5.1.1.8 created 1965, modified 1983' WHERE ec = 'EC 5.1.1.8';
UPDATE enzyme SET sys_name = 'Arginine racemase' WHERE ec = 'EC 5.1.1.9';
UPDATE enzyme SET history = 'EC 5.1.1.9 created 1972' WHERE ec = 'EC 5.1.1.9';
UPDATE enzyme SET sys_name = 'Amino-acid racemase' WHERE ec = 'EC 5.1.1.10';
UPDATE enzyme SET history = 'EC 5.1.1.10 created 1972' WHERE ec = 'EC 5.1.1.10';
UPDATE enzyme SET sys_name = 'Phenylalanine racemase (ATP-hydrolyzing)' WHERE ec = 'EC 5.1.1.11';
UPDATE enzyme SET history = 'EC 5.1.1.11 created 1972' WHERE ec = 'EC 5.1.1.11';
UPDATE enzyme SET sys_name = 'Ornithine racemase' WHERE ec = 'EC 5.1.1.12';
UPDATE enzyme SET history = 'EC 5.1.1.12 created 1972 as EC 5.4.3.1, transferred 1976 to EC 5.1.1.12' WHERE ec = 'EC 5.1.1.12';
UPDATE enzyme SET sys_name = 'Aspartate racemase' WHERE ec = 'EC 5.1.1.13';
UPDATE enzyme SET history = 'EC 5.1.1.13 created 1976' WHERE ec = 'EC 5.1.1.13';
UPDATE enzyme SET sys_name = 'Nocardicin-A epimerase' WHERE ec = 'EC 5.1.1.14';
UPDATE enzyme SET history = 'EC 5.1.1.14 created 1992' WHERE ec = 'EC 5.1.1.14';
UPDATE enzyme SET sys_name = '2-aminohexano-6-lactam racemase' WHERE ec = 'EC 5.1.1.15';
UPDATE enzyme SET history = 'EC 5.1.1.15 created 1999' WHERE ec = 'EC 5.1.1.15';
UPDATE enzyme SET sys_name = '[Protein]-serine epimerase' WHERE ec = 'EC 5.1.1.16';
UPDATE enzyme SET history = 'EC 5.1.1.16 created 1999' WHERE ec = 'EC 5.1.1.16';
UPDATE enzyme SET sys_name = 'Penicillin-N 5-amino-5-carboxypentanoyl-epimerase' WHERE ec = 'EC 5.1.1.17';
UPDATE enzyme SET history = 'EC 5.1.1.17 created 2002' WHERE ec = 'EC 5.1.1.17';
UPDATE enzyme SET sys_name = 'Serine racemase' WHERE ec = 'EC 5.1.1.18';
UPDATE enzyme SET history = 'EC 5.1.1.18 created 2007' WHERE ec = 'EC 5.1.1.18';
UPDATE enzyme SET sys_name = '(2S)-2-amino-3-((carbamoylamino)oxy)propanoate 2-epimerase' WHERE ec = 'EC 5.1.1.19';
UPDATE enzyme SET history = 'EC 5.1.1.19 created 2013' WHERE ec = 'EC 5.1.1.19';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 5.1.1.n1';
UPDATE enzyme SET history = 'EC 5.5.1.n1 created 2012' WHERE ec = 'EC 5.1.1.n1';
UPDATE enzyme SET sys_name = 'Lactate racemase' WHERE ec = 'EC 5.1.2.1';
UPDATE enzyme SET history = 'EC 5.1.2.1 created 1961' WHERE ec = 'EC 5.1.2.1';
UPDATE enzyme SET sys_name = 'Mandelate racemase' WHERE ec = 'EC 5.1.2.2';
UPDATE enzyme SET history = 'EC 5.1.2.2 created 1961' WHERE ec = 'EC 5.1.2.2';
UPDATE enzyme SET sys_name = '3-hydroxybutanoyl-CoA 3-epimerase' WHERE ec = 'EC 5.1.2.3';
UPDATE enzyme SET history = 'EC 5.1.2.3 created 1961' WHERE ec = 'EC 5.1.2.3';
UPDATE enzyme SET sys_name = 'Acetoin racemase' WHERE ec = 'EC 5.1.2.4';
UPDATE enzyme SET history = 'EC 5.1.2.4 created 1972' WHERE ec = 'EC 5.1.2.4';
UPDATE enzyme SET sys_name = 'Tartrate epimerase' WHERE ec = 'EC 5.1.2.5';
UPDATE enzyme SET history = 'EC 5.1.2.5 created 1972' WHERE ec = 'EC 5.1.2.5';
UPDATE enzyme SET sys_name = '(1R,2S)-1-hydroxypropane-1,2,3-tricarboxylate 1-epimerase' WHERE ec = 'EC 5.1.2.6';
UPDATE enzyme SET history = 'EC 5.1.2.6 created 1984' WHERE ec = 'EC 5.1.2.6';
UPDATE enzyme SET sys_name = 'D-ribulose-5-phosphate 3-epimerase' WHERE ec = 'EC 5.1.3.1';
UPDATE enzyme SET history = 'EC 5.1.3.1 created 1961, modified 1989' WHERE ec = 'EC 5.1.3.1';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-glucose 4-epimerase' WHERE ec = 'EC 5.1.3.2';
UPDATE enzyme SET history = 'EC 5.1.3.2 created 1961' WHERE ec = 'EC 5.1.3.2';
UPDATE enzyme SET sys_name = 'Aldose 1-epimerase' WHERE ec = 'EC 5.1.3.3';
UPDATE enzyme SET history = 'EC 5.1.3.3 created 1961' WHERE ec = 'EC 5.1.3.3';
UPDATE enzyme SET sys_name = 'L-ribulose-5-phosphate 4-epimerase' WHERE ec = 'EC 5.1.3.4';
UPDATE enzyme SET history = 'EC 5.1.3.4 created 1965, modified 2005' WHERE ec = 'EC 5.1.3.4';
UPDATE enzyme SET sys_name = 'UDP-L-arabinose 4-epimerase' WHERE ec = 'EC 5.1.3.5';
UPDATE enzyme SET history = 'EC 5.1.3.5 created 1965' WHERE ec = 'EC 5.1.3.5';
UPDATE enzyme SET sys_name = 'UDP-glucuronate 4-epimerase' WHERE ec = 'EC 5.1.3.6';
UPDATE enzyme SET history = 'EC 5.1.3.6 created 1965' WHERE ec = 'EC 5.1.3.6';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine 4-epimerase' WHERE ec = 'EC 5.1.3.7';
UPDATE enzyme SET history = 'EC 5.1.3.7 created 1965' WHERE ec = 'EC 5.1.3.7';
UPDATE enzyme SET sys_name = 'N-acyl-D-glucosamine 2-epimerase' WHERE ec = 'EC 5.1.3.8';
UPDATE enzyme SET history = 'EC 5.1.3.8 created 1972' WHERE ec = 'EC 5.1.3.8';
UPDATE enzyme SET sys_name = 'N-acyl-D-glucosamine-6-phosphate 2-epimerase' WHERE ec = 'EC 5.1.3.9';
UPDATE enzyme SET history = 'EC 5.1.3.9 created 1972' WHERE ec = 'EC 5.1.3.9';
UPDATE enzyme SET sys_name = 'CDP-3,6-dideoxy-D-glucose 2-epimerase' WHERE ec = 'EC 5.1.3.10';
UPDATE enzyme SET history = 'EC 5.1.3.10 created 1972, modified 2005' WHERE ec = 'EC 5.1.3.10';
UPDATE enzyme SET sys_name = 'Cellobiose 2-epimerase' WHERE ec = 'EC 5.1.3.11';
UPDATE enzyme SET history = 'EC 5.1.3.11 created 1972' WHERE ec = 'EC 5.1.3.11';
UPDATE enzyme SET sys_name = 'UDP-glucuronate 5-epimerase' WHERE ec = 'EC 5.1.3.12';
UPDATE enzyme SET history = 'EC 5.1.3.12 created 1972' WHERE ec = 'EC 5.1.3.12';
UPDATE enzyme SET sys_name = 'dTDP-4-dehydro-6-deoxy-alpha-D-glucose 3,5-epimerase' WHERE ec = 'EC 5.1.3.13';
UPDATE enzyme SET history = 'EC 5.1.3.13 created 1972' WHERE ec = 'EC 5.1.3.13';
UPDATE enzyme SET sys_name = 'UDP-N-acetyl-alpha-D-glucosamine 2-epimerase' WHERE ec = 'EC 5.1.3.14';
UPDATE enzyme SET history = 'EC 5.1.3.14 created 1976, modified 2012' WHERE ec = 'EC 5.1.3.14';
UPDATE enzyme SET sys_name = 'D-glucose-6-phosphate 1-epimerase' WHERE ec = 'EC 5.1.3.15';
UPDATE enzyme SET history = 'EC 5.1.3.15 created 1976' WHERE ec = 'EC 5.1.3.15';
UPDATE enzyme SET sys_name = 'UDP-glucosamine 4-epimerase' WHERE ec = 'EC 5.1.3.16';
UPDATE enzyme SET history = 'EC 5.1.3.16 created 1984' WHERE ec = 'EC 5.1.3.16';
UPDATE enzyme SET sys_name = 'Poly((1-&gt;4)-beta-D-glucuronosyl-(1-&gt;4)-N-sulfo-alpha-D-glucosaminyl) glucurono-5-epimerase' WHERE ec = 'EC 5.1.3.17';
UPDATE enzyme SET history = 'EC 5.1.3.17 created 1984' WHERE ec = 'EC 5.1.3.17';
UPDATE enzyme SET sys_name = 'GDP-alpha-D-mannose 3,5-epimerase' WHERE ec = 'EC 5.1.3.18';
UPDATE enzyme SET history = 'EC 5.1.3.18 created 1986' WHERE ec = 'EC 5.1.3.18';
UPDATE enzyme SET sys_name = 'Chondroitin-D-glucuronate 5-epimerase' WHERE ec = 'EC 5.1.3.19';
UPDATE enzyme SET history = 'EC 5.1.3.19 created 1986' WHERE ec = 'EC 5.1.3.19';
UPDATE enzyme SET sys_name = 'ADP-L-glycero-D-manno-heptose 6-epimerase' WHERE ec = 'EC 5.1.3.20';
UPDATE enzyme SET history = 'EC 5.1.3.20 created 1999' WHERE ec = 'EC 5.1.3.20';
UPDATE enzyme SET sys_name = 'Maltose 1-epimerase' WHERE ec = 'EC 5.1.3.21';
UPDATE enzyme SET history = 'EC 5.1.3.21 created 2002' WHERE ec = 'EC 5.1.3.21';
UPDATE enzyme SET sys_name = 'L-ribulose-5-phosphate 3-epimerase' WHERE ec = 'EC 5.1.3.22';
UPDATE enzyme SET history = 'EC 5.1.3.22 created 2005' WHERE ec = 'EC 5.1.3.22';
UPDATE enzyme SET sys_name = '2,3-diacetamido-2,3-dideoxy-alpha-D-glucuronate 2-epimerase' WHERE ec = 'EC 5.1.3.23';
UPDATE enzyme SET history = 'EC 5.1.3.23 created 2007' WHERE ec = 'EC 5.1.3.23';
UPDATE enzyme SET sys_name = 'N-acetyl-alpha-neuraminate 2-epimerase' WHERE ec = 'EC 5.1.3.24';
UPDATE enzyme SET history = 'EC 5.1.3.24 created 2011' WHERE ec = 'EC 5.1.3.24';
UPDATE enzyme SET sys_name = 'dTDP-6-deoxy-beta-L-talose 4-epimerase' WHERE ec = 'EC 5.1.3.25';
UPDATE enzyme SET history = 'EC 5.1.3.25 created 2012' WHERE ec = 'EC 5.1.3.25';
UPDATE enzyme SET sys_name = 'N-acetyl-alpha-D-glucosaminyl-diphospho-ditrans,octacis-undecaprenol 4-epimerase' WHERE ec = 'EC 5.1.3.26';
UPDATE enzyme SET history = 'EC 5.1.3.26 created 2013' WHERE ec = 'EC 5.1.3.26';
UPDATE enzyme SET sys_name = 'dTDP-4-dehydro-6-deoxy-alpha-D-glucose 3-epimerase' WHERE ec = 'EC 5.1.3.27';
UPDATE enzyme SET history = 'EC 5.1.3.27 created 2013' WHERE ec = 'EC 5.1.3.27';
UPDATE enzyme SET sys_name = 'UDP-2-acetamido-2,6-dideoxy-L-talose 2-epimerase' WHERE ec = 'EC 5.1.3.28';
UPDATE enzyme SET history = 'EC 5.1.3.28 created 2014' WHERE ec = 'EC 5.1.3.28';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 5.1.3.n1';
UPDATE enzyme SET sys_name = 'Alpha-L-fucose 1-epimerase' WHERE ec = 'EC 5.1.3.n2';
UPDATE enzyme SET history = 'EC 5.1.3.n2 created 2010' WHERE ec = 'EC 5.1.3.n2';
UPDATE enzyme SET sys_name = 'Alpha-L-rhamnose 1-epimerase' WHERE ec = 'EC 5.1.3.n3';
UPDATE enzyme SET history = 'EC 5.1.3.n3 created 2010' WHERE ec = 'EC 5.1.3.n3';
UPDATE enzyme SET sys_name = 'Methylmalonyl-CoA 2-epimerase' WHERE ec = 'EC 5.1.99.1';
UPDATE enzyme SET history = 'EC 5.1.99.1 created 1965, modified 1981, modified 2004' WHERE ec = 'EC 5.1.99.1';
UPDATE enzyme SET sys_name = '16-hydroxysteroid 16-epimerase' WHERE ec = 'EC 5.1.99.2';
UPDATE enzyme SET history = 'EC 5.1.99.2 created 1972' WHERE ec = 'EC 5.1.99.2';
UPDATE enzyme SET sys_name = 'Allantoin racemase' WHERE ec = 'EC 5.1.99.3';
UPDATE enzyme SET history = 'EC 5.1.99.3 created 1976' WHERE ec = 'EC 5.1.99.3';
UPDATE enzyme SET sys_name = '2-methylacyl-CoA 2-epimerase' WHERE ec = 'EC 5.1.99.4';
UPDATE enzyme SET history = 'EC 5.1.99.4 created 1999' WHERE ec = 'EC 5.1.99.4';
UPDATE enzyme SET sys_name = 'D-5-monosubstituted-hydantoin racemase' WHERE ec = 'EC 5.1.99.5';
UPDATE enzyme SET history = 'EC 5.1.99.5 created 2008' WHERE ec = 'EC 5.1.99.5';
UPDATE enzyme SET sys_name = '(6R)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adenine dinucleotide 6-epimerase' WHERE ec = 'EC 5.1.99.6';
UPDATE enzyme SET history = 'EC 5.1.99.6 created 2012' WHERE ec = 'EC 5.1.99.6';
UPDATE enzyme SET sys_name = 'Maleate cis-trans-isomerase' WHERE ec = 'EC 5.2.1.1';
UPDATE enzyme SET history = 'EC 5.2.1.1 created 1961' WHERE ec = 'EC 5.2.1.1';
UPDATE enzyme SET sys_name = '4-maleylacetoacetate cis-trans-isomerase' WHERE ec = 'EC 5.2.1.2';
UPDATE enzyme SET history = 'EC 5.2.1.2 created 1961' WHERE ec = 'EC 5.2.1.2';
UPDATE enzyme SET history = 'EC 5.2.1.3 created 1961, modified 1976, deleted 2011' WHERE ec = 'EC 5.2.1.3';
UPDATE enzyme SET sys_name = '3-maleylpyruvate cis-trans-isomerase' WHERE ec = 'EC 5.2.1.4';
UPDATE enzyme SET history = 'EC 5.2.1.4 created 1965' WHERE ec = 'EC 5.2.1.4';
UPDATE enzyme SET sys_name = 'Linoleate Delta(12)-cis-Delta(11)-trans-isomerase' WHERE ec = 'EC 5.2.1.5';
UPDATE enzyme SET history = 'EC 5.2.1.5 created 1972' WHERE ec = 'EC 5.2.1.5';
UPDATE enzyme SET sys_name = '2-(2-furyl)-3-(5-nitro-2-furyl)acrylamide cis-trans-isomerase' WHERE ec = 'EC 5.2.1.6';
UPDATE enzyme SET history = 'EC 5.2.1.6 created 1978' WHERE ec = 'EC 5.2.1.6';
UPDATE enzyme SET history = 'EC 5.2.1.7 created 1989, deleted 2011' WHERE ec = 'EC 5.2.1.7';
UPDATE enzyme SET sys_name = 'Peptidylproline cis-trans-isomerase' WHERE ec = 'EC 5.2.1.8';
UPDATE enzyme SET history = 'EC 5.2.1.8 created 1989, modified 2002' WHERE ec = 'EC 5.2.1.8';
UPDATE enzyme SET sys_name = '(2E,6E)-farnesol 2-cis-trans-isomerase' WHERE ec = 'EC 5.2.1.9';
UPDATE enzyme SET history = 'EC 5.2.1.9 created 1989' WHERE ec = 'EC 5.2.1.9';
UPDATE enzyme SET sys_name = '2-chloro-4-carboxymethylenebut-2-en-1,4-olide cis-trans-isomerase' WHERE ec = 'EC 5.2.1.10';
UPDATE enzyme SET history = 'EC 5.2.1.10 created 1992' WHERE ec = 'EC 5.2.1.10';
UPDATE enzyme SET history = 'EC 5.2.1.11 created 1992, deleted 2005' WHERE ec = 'EC 5.2.1.11';
UPDATE enzyme SET sys_name = '9,15,9-tri-cis-zeta-carotene cis-trans-isomerase' WHERE ec = 'EC 5.2.1.12';
UPDATE enzyme SET history = 'EC 5.2.1.12 created 2011' WHERE ec = 'EC 5.2.1.12';
UPDATE enzyme SET sys_name = '7,9,7,9-tetra-cis-lycopene cis-trans-isomerase' WHERE ec = 'EC 5.2.1.13';
UPDATE enzyme SET history = 'EC 5.2.1.13 created 2011' WHERE ec = 'EC 5.2.1.13';
UPDATE enzyme SET sys_name = 'Beta-carotene 9-cis-all-trans isomerase' WHERE ec = 'EC 5.2.1.14';
UPDATE enzyme SET history = 'EC 5.2.1.14 created 2012' WHERE ec = 'EC 5.2.1.14';
UPDATE enzyme SET sys_name = 'D-glyceraldehyde-3-phosphate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.1';
UPDATE enzyme SET history = 'EC 5.3.1.1 created 1961' WHERE ec = 'EC 5.3.1.1';
UPDATE enzyme SET history = 'EC 5.3.1.2 created 1961, deleted 1976' WHERE ec = 'EC 5.3.1.2';
UPDATE enzyme SET sys_name = 'D-arabinose aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.3';
UPDATE enzyme SET history = 'EC 5.3.1.3 created 1961, modified 2013' WHERE ec = 'EC 5.3.1.3';
UPDATE enzyme SET sys_name = 'L-arabinose aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.4';
UPDATE enzyme SET history = 'EC 5.3.1.4 created 1961' WHERE ec = 'EC 5.3.1.4';
UPDATE enzyme SET sys_name = 'D-xylose aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.5';
UPDATE enzyme SET history = 'EC 5.3.1.5 created 1961 (EC 5.3.1.18 created 1972, part incorporated 1978)' WHERE ec = 'EC 5.3.1.5';
UPDATE enzyme SET sys_name = 'D-ribose-5-phosphate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.6';
UPDATE enzyme SET history = 'EC 5.3.1.6 created 1961' WHERE ec = 'EC 5.3.1.6';
UPDATE enzyme SET sys_name = 'D-mannose aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.7';
UPDATE enzyme SET history = 'EC 5.3.1.7 created 1961' WHERE ec = 'EC 5.3.1.7';
UPDATE enzyme SET sys_name = 'D-mannose-6-phosphate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.8';
UPDATE enzyme SET history = 'EC 5.3.1.8 created 1961, modified 1976' WHERE ec = 'EC 5.3.1.8';
UPDATE enzyme SET sys_name = 'D-glucose-6-phosphate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.9';
UPDATE enzyme SET history = 'EC 5.3.1.9 created 1961, modified 1976 (EC 5.3.1.18 created part 1972, incorporated 1978)' WHERE ec = 'EC 5.3.1.9';
UPDATE enzyme SET history = 'EC 5.3.1.10 created 1961, deleted 2000' WHERE ec = 'EC 5.3.1.10';
UPDATE enzyme SET history = 'EC 5.3.1.11 created 1961, deleted 1978' WHERE ec = 'EC 5.3.1.11';
UPDATE enzyme SET sys_name = 'D-glucuronate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.12';
UPDATE enzyme SET history = 'EC 5.3.1.12 created 1961' WHERE ec = 'EC 5.3.1.12';
UPDATE enzyme SET sys_name = 'D-arabinose-5-phosphate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.13';
UPDATE enzyme SET history = 'EC 5.3.1.13 created 1965' WHERE ec = 'EC 5.3.1.13';
UPDATE enzyme SET sys_name = 'L-rhamnose aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.14';
UPDATE enzyme SET history = 'EC 5.3.1.14 created 1965' WHERE ec = 'EC 5.3.1.14';
UPDATE enzyme SET sys_name = 'D-lyxose aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.15';
UPDATE enzyme SET history = 'EC 5.3.1.15 created 1972' WHERE ec = 'EC 5.3.1.15';
UPDATE enzyme SET sys_name = '1-(5-phospho-beta-D-ribosyl)-5-((5-phospho-beta-D-ribosylamino)methylideneamino)imidazole-4-carboxamide aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.16';
UPDATE enzyme SET history = 'EC 5.3.1.16 created 1972, modified 2000' WHERE ec = 'EC 5.3.1.16';
UPDATE enzyme SET sys_name = '5-dehydro-4-deoxy-D-glucuronate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.17';
UPDATE enzyme SET history = 'EC 5.3.1.17 created 1972, modified 2012' WHERE ec = 'EC 5.3.1.17';
UPDATE enzyme SET history = 'EC 5.3.1.18 created 1972, deleted 1978' WHERE ec = 'EC 5.3.1.18';
UPDATE enzyme SET history = 'EC 5.3.1.19 created 1972, deleted 1984' WHERE ec = 'EC 5.3.1.19';
UPDATE enzyme SET sys_name = 'D-ribose aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.20';
UPDATE enzyme SET history = 'EC 5.3.1.20 created 1978' WHERE ec = 'EC 5.3.1.20';
UPDATE enzyme SET sys_name = '11-deoxycorticosterone aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.21';
UPDATE enzyme SET history = 'EC 5.3.1.21 created 1983' WHERE ec = 'EC 5.3.1.21';
UPDATE enzyme SET sys_name = 'Hydroxypyruvate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.22';
UPDATE enzyme SET history = 'EC 5.3.1.22 created 1983' WHERE ec = 'EC 5.3.1.22';
UPDATE enzyme SET sys_name = 'S-methyl-5-thio-alpha-D-ribose-1-phosphate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.23';
UPDATE enzyme SET history = 'EC 5.3.1.23 created 1989' WHERE ec = 'EC 5.3.1.23';
UPDATE enzyme SET sys_name = 'N-(5-phospho-beta-D-ribosyl)anthranilate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.24';
UPDATE enzyme SET history = 'EC 5.3.1.24 created 1990' WHERE ec = 'EC 5.3.1.24';
UPDATE enzyme SET sys_name = 'L-fucose aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.25';
UPDATE enzyme SET history = 'EC 5.3.1.25 created 1999' WHERE ec = 'EC 5.3.1.25';
UPDATE enzyme SET sys_name = 'D-galactose-6-phosphate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.26';
UPDATE enzyme SET history = 'EC 5.3.1.26 created 1999' WHERE ec = 'EC 5.3.1.26';
UPDATE enzyme SET sys_name = 'D-arabino-hex-3-ulose-6-phosphate isomerase' WHERE ec = 'EC 5.3.1.27';
UPDATE enzyme SET history = 'EC 5.3.1.27 created 2008' WHERE ec = 'EC 5.3.1.27';
UPDATE enzyme SET sys_name = 'D-glycero-D-manno-heptose 7-phosphate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.28';
UPDATE enzyme SET history = 'EC 5.3.1.28 created 2010' WHERE ec = 'EC 5.3.1.28';
UPDATE enzyme SET sys_name = 'Alpha-D-ribose 1,5-bisphosphate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.29';
UPDATE enzyme SET history = 'EC 5.3.1.29 created 2013' WHERE ec = 'EC 5.3.1.29';
UPDATE enzyme SET sys_name = '5-deoxy-D-glucuronate aldose-ketose-isomerase' WHERE ec = 'EC 5.3.1.30';
UPDATE enzyme SET history = 'EC 5.3.1.30 created 2014' WHERE ec = 'EC 5.3.1.30';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 5.3.1.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 5.3.1.n2';
UPDATE enzyme SET sys_name = 'Phenylpyruvate keto--enol-isomerase' WHERE ec = 'EC 5.3.2.1';
UPDATE enzyme SET history = 'EC 5.3.2.1 created 1961' WHERE ec = 'EC 5.3.2.1';
UPDATE enzyme SET sys_name = 'Oxaloacetate keto--enol-isomerase' WHERE ec = 'EC 5.3.2.2';
UPDATE enzyme SET history = 'EC 5.3.2.2 created 1972' WHERE ec = 'EC 5.3.2.2';
UPDATE enzyme SET sys_name = 'dTDP-4-dehydro-6-deoxy-alpha-D-glucopyranose:dTDP-3-dehydro-6-deoxy-alpha-D-galactopyranose isomerase' WHERE ec = 'EC 5.3.2.3';
UPDATE enzyme SET history = 'EC 5.3.2.3 created 2011' WHERE ec = 'EC 5.3.2.3';
UPDATE enzyme SET sys_name = 'dTDP-4-dehydro-6-deoxy-alpha-D-glucopyranose:dTDP-3-dehydro-6-deoxy-alpha-D-glucopyranose isomerase' WHERE ec = 'EC 5.3.2.4';
UPDATE enzyme SET history = 'EC 5.3.2.4 created 2011' WHERE ec = 'EC 5.3.2.4';
UPDATE enzyme SET sys_name = '2,3-diketo-5-methylthiopentyl-1-phosphate keto--enol-isomerase' WHERE ec = 'EC 5.3.2.5';
UPDATE enzyme SET history = 'EC 5.3.2.5 created 2012' WHERE ec = 'EC 5.3.2.5';
UPDATE enzyme SET sys_name = '(2Z,4E)-2-hydroxyhexa-2,4-dienedioate keto(-)-enol isomerase' WHERE ec = 'EC 5.3.2.6';
UPDATE enzyme SET history = 'EC 5.3.2.6 created 2012' WHERE ec = 'EC 5.3.2.6';
UPDATE enzyme SET sys_name = '1,5-anhydro-4-deoxy-D-glycero-hex-3-en-2-ulose Delta(3)-Delta(1)-isomerase' WHERE ec = 'EC 5.3.2.7';
UPDATE enzyme SET history = 'EC 5.3.2.7 created 2006 as EC 5.3.3.15, transferred 2012 to EC 5.3.2.7' WHERE ec = 'EC 5.3.2.7';
UPDATE enzyme SET sys_name = '4-oxalomesaconate keto--enol-isomerase' WHERE ec = 'EC 5.3.2.8';
UPDATE enzyme SET history = 'EC 5.3.2.8 created 2011 as 5.3.3.16, modified 2011, transferred 2012 to EC 5.3.2.8' WHERE ec = 'EC 5.3.2.8';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 5.3.2.n1';
UPDATE enzyme SET sys_name = '3-oxosteroid Delta(5)-Delta(4)-isomerase' WHERE ec = 'EC 5.3.3.1';
UPDATE enzyme SET history = 'EC 5.3.3.1 created 1961' WHERE ec = 'EC 5.3.3.1';
UPDATE enzyme SET sys_name = 'Isopentenyl-diphosphate Delta(3)-Delta(2)-isomerase' WHERE ec = 'EC 5.3.3.2';
UPDATE enzyme SET history = 'EC 5.3.3.2 created 1961, modified 2002' WHERE ec = 'EC 5.3.3.2';
UPDATE enzyme SET sys_name = 'Vinylacetyl-CoA Delta(3)-Delta(2)-isomerase' WHERE ec = 'EC 5.3.3.3';
UPDATE enzyme SET history = 'EC 5.3.3.3 created 1961, modified 2011' WHERE ec = 'EC 5.3.3.3';
UPDATE enzyme SET sys_name = '5-oxo-4,5-dihydrofuran-2-acetate Delta(3)-Delta(2)-isomerase' WHERE ec = 'EC 5.3.3.4';
UPDATE enzyme SET history = 'EC 5.3.3.4 created 1961 as EC 3.1.1.16, part transferred 1972 to EC 5.3.3.4 rest to EC 5.3.3.4' WHERE ec = 'EC 5.3.3.4';
UPDATE enzyme SET sys_name = 'Delta(7)-cholestenol Delta(7)-Delta(8)-isomerase' WHERE ec = 'EC 5.3.3.5';
UPDATE enzyme SET history = 'EC 5.3.3.5 created 1972' WHERE ec = 'EC 5.3.3.5';
UPDATE enzyme SET sys_name = 'Methylitaconate Delta(2)-Delta(3)-isomerase' WHERE ec = 'EC 5.3.3.6';
UPDATE enzyme SET history = 'EC 5.3.3.6 created 1972' WHERE ec = 'EC 5.3.3.6';
UPDATE enzyme SET sys_name = 'Aconitate Delta(2)-Delta(3)-isomerase' WHERE ec = 'EC 5.3.3.7';
UPDATE enzyme SET history = 'EC 5.3.3.7 created 1972' WHERE ec = 'EC 5.3.3.7';
UPDATE enzyme SET sys_name = 'Dodecenoyl-CoA (3Z)-(2E)-isomerase' WHERE ec = 'EC 5.3.3.8';
UPDATE enzyme SET history = 'EC 5.3.3.8 created 1978, modified 1980' WHERE ec = 'EC 5.3.3.8';
UPDATE enzyme SET sys_name = '(13E)-(15S)-15-hydroxy-9-oxoprosta-10,13-dienoate Delta(10)-Delta(11)-isomerase' WHERE ec = 'EC 5.3.3.9';
UPDATE enzyme SET history = 'EC 5.3.3.9 created 1978' WHERE ec = 'EC 5.3.3.9';
UPDATE enzyme SET sys_name = '5-carboxymethyl-2-hydroxymuconate Delta(2),Delta(4)-2-oxo,Delta(3)-isomerase' WHERE ec = 'EC 5.3.3.10';
UPDATE enzyme SET history = 'EC 5.3.3.10 created 1984' WHERE ec = 'EC 5.3.3.10';
UPDATE enzyme SET sys_name = 'Isopiperitenone Delta(8)-Delta(4)-isomerase' WHERE ec = 'EC 5.3.3.11';
UPDATE enzyme SET history = 'EC 5.3.3.11 created 1989' WHERE ec = 'EC 5.3.3.11';
UPDATE enzyme SET sys_name = 'L-dopachrome keto-enol isomerase' WHERE ec = 'EC 5.3.3.12';
UPDATE enzyme SET history = 'EC 5.3.3.12 created 1992, modified 1999, modified 2005' WHERE ec = 'EC 5.3.3.12';
UPDATE enzyme SET sys_name = '(5Z,8Z,11Z,14Z,17Z)-icosapentaenoate Delta(8,11)-Delta(7,9)-isomerase (trans-double-bond-forming)' WHERE ec = 'EC 5.3.3.13';
UPDATE enzyme SET history = 'EC 5.3.3.13 created 2004' WHERE ec = 'EC 5.3.3.13';
UPDATE enzyme SET sys_name = 'Decenoyl-[acyl-carrier-protein] Delta(2)-trans-Delta(3)-cis-isomerase' WHERE ec = 'EC 5.3.3.14';
UPDATE enzyme SET history = 'EC 5.3.3.14 created 2006' WHERE ec = 'EC 5.3.3.14';
UPDATE enzyme SET history = 'EC 5.3.3.15 created 2006, deleted 2013' WHERE ec = 'EC 5.3.3.15';
UPDATE enzyme SET history = 'EC 5.3.3.16 created 2011, modified 2011, deleted 2013' WHERE ec = 'EC 5.3.3.16';
UPDATE enzyme SET sys_name = '(5S,6S)-6-amino-5-hydroxycyclohexane-1,3-diene-1-carboxyate isomerase' WHERE ec = 'EC 5.3.3.17';
UPDATE enzyme SET history = 'EC 5.3.3.17 created 2011' WHERE ec = 'EC 5.3.3.17';
UPDATE enzyme SET sys_name = '2-(1,2-epoxy-1,2-dihydrophenyl)acetyl-CoA isomerase' WHERE ec = 'EC 5.3.3.18';
UPDATE enzyme SET history = 'EC 5.3.3.18 created 2011' WHERE ec = 'EC 5.3.3.18';
UPDATE enzyme SET sys_name = 'Protein disulfide-isomerase' WHERE ec = 'EC 5.3.4.1';
UPDATE enzyme SET history = 'EC 5.3.4.1 created 1972' WHERE ec = 'EC 5.3.4.1';
UPDATE enzyme SET history = 'EC 5.3.99.1 created 1972, deleted 1992' WHERE ec = 'EC 5.3.99.1';
UPDATE enzyme SET sys_name = '(5Z,13E,15S)-9-alpha,11-alpha-epidioxy-15-hydroxyprosta-5,13-dienoate D-isomerase' WHERE ec = 'EC 5.3.99.2';
UPDATE enzyme SET history = 'EC 5.3.99.2 created 1976, modified 1990' WHERE ec = 'EC 5.3.99.2';
UPDATE enzyme SET sys_name = '(5Z,13E)-(15S)-9-alpha,11-alpha-epidioxy-15-hydroxyprosta-5,13-dienoate E-isomerase' WHERE ec = 'EC 5.3.99.3';
UPDATE enzyme SET history = 'EC 5.3.99.3 created 1976, modified 1990' WHERE ec = 'EC 5.3.99.3';
UPDATE enzyme SET sys_name = '(5Z,13E)-(15S)-9-alpha,11-alpha-epidioxy-15-hydroxyprosta-5,13-dienoate 6-isomerase' WHERE ec = 'EC 5.3.99.4';
UPDATE enzyme SET history = 'EC 5.3.99.4 created 1984, modified 1990' WHERE ec = 'EC 5.3.99.4';
UPDATE enzyme SET sys_name = '(5Z,13E)-(15S)-9-alpha,11-alpha-epidioxy-15-hydroxyprosta-5,13-dienoate isomerase' WHERE ec = 'EC 5.3.99.5';
UPDATE enzyme SET history = 'EC 5.3.99.5 created 1984, modified 1990' WHERE ec = 'EC 5.3.99.5';
UPDATE enzyme SET sys_name = '(9Z)-(13S)-12,13-epoxyoctadeca-9,11,15-trienoate isomerase (cyclizing)' WHERE ec = 'EC 5.3.99.6';
UPDATE enzyme SET history = 'EC 5.3.99.6 created 1992' WHERE ec = 'EC 5.3.99.6';
UPDATE enzyme SET sys_name = 'Styrene-oxide isomerase (epoxide-cleaving)' WHERE ec = 'EC 5.3.99.7';
UPDATE enzyme SET history = 'EC 5.3.99.7 created 1992' WHERE ec = 'EC 5.3.99.7';
UPDATE enzyme SET sys_name = 'Violaxanthin--capsorubin isomerase (ketone-forming)' WHERE ec = 'EC 5.3.99.8';
UPDATE enzyme SET history = 'EC 5.3.99.8 created 2005' WHERE ec = 'EC 5.3.99.8';
UPDATE enzyme SET sys_name = 'Violaxanthin--neoxanthin isomerase (epoxide-opening)' WHERE ec = 'EC 5.3.99.9';
UPDATE enzyme SET history = 'EC 5.3.99.9 created 2005' WHERE ec = 'EC 5.3.99.9';
UPDATE enzyme SET sys_name = '2-(2-carboxy-4-methylthiazol-5-yl)ethyl phosphate isomerase' WHERE ec = 'EC 5.3.99.10';
UPDATE enzyme SET history = 'EC 5.3.99.10 created 2012' WHERE ec = 'EC 5.3.99.10';
UPDATE enzyme SET sys_name = '2,4,6/3,5-pentahydroxycyclohexanone 2-isomerase' WHERE ec = 'EC 5.3.99.11';
UPDATE enzyme SET history = 'EC 5.3.99.11 created 2014' WHERE ec = 'EC 5.3.99.11';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 5.3.99.n1';
UPDATE enzyme SET sys_name = 'Lysolecithin 2,3-acylmutase' WHERE ec = 'EC 5.4.1.1';
UPDATE enzyme SET history = 'EC 5.4.1.1 created 1961' WHERE ec = 'EC 5.4.1.1';
UPDATE enzyme SET history = 'EC 5.4.1.2 created 1999, deleted 2014' WHERE ec = 'EC 5.4.1.2';
UPDATE enzyme SET sys_name = '2-methylfumaryl-CoA 1,4-CoA-mutase' WHERE ec = 'EC 5.4.1.3';
UPDATE enzyme SET history = 'EC 5.4.1.3 created 2014' WHERE ec = 'EC 5.4.1.3';
UPDATE enzyme SET history = 'EC 5.4.2.1 created 1961 (EC 2.7.5.3 created 1961, incorporated 1984), deleted 2013' WHERE ec = 'EC 5.4.2.1';
UPDATE enzyme SET sys_name = 'Alpha-D-glucose 1,6-phosphomutase' WHERE ec = 'EC 5.4.2.2';
UPDATE enzyme SET history = 'EC 5.4.2.2 created 1961 as EC 2.7.5.1, transferred 1984 to EC 5.4.2.2' WHERE ec = 'EC 5.4.2.2';
UPDATE enzyme SET sys_name = 'N-acetyl-alpha-D-glucosamine 1,6-phosphomutase' WHERE ec = 'EC 5.4.2.3';
UPDATE enzyme SET history = 'EC 5.4.2.3 created 1961 as EC 2.7.5.2, transferred 1984 to EC 5.4.2.3' WHERE ec = 'EC 5.4.2.3';
UPDATE enzyme SET sys_name = '3-phospho-D-glycerate 1,2-phosphomutase' WHERE ec = 'EC 5.4.2.4';
UPDATE enzyme SET history = 'EC 5.4.2.4 created 1961 as EC 2.7.5.4, transferred 1984 to EC 5.4.2.4' WHERE ec = 'EC 5.4.2.4';
UPDATE enzyme SET sys_name = 'Alpha-D-glucose 1,6-phosphomutase (glucose-cofactor)' WHERE ec = 'EC 5.4.2.5';
UPDATE enzyme SET history = 'EC 5.4.2.5 created 1972 as EC 2.7.5.5, transferred 1984 to EC 5.4.2.5' WHERE ec = 'EC 5.4.2.5';
UPDATE enzyme SET sys_name = 'Beta-D-glucose 1,6-phosphomutase' WHERE ec = 'EC 5.4.2.6';
UPDATE enzyme SET history = 'EC 5.4.2.6 created 1984' WHERE ec = 'EC 5.4.2.6';
UPDATE enzyme SET sys_name = 'Alpha-D-ribose 1,5-phosphomutase' WHERE ec = 'EC 5.4.2.7';
UPDATE enzyme SET history = 'EC 5.4.2.7 created 1972 as EC 2.7.5.6, transferred 1984 to EC 5.4.2.7' WHERE ec = 'EC 5.4.2.7';
UPDATE enzyme SET sys_name = 'Alpha-D-mannose 1,6-phosphomutase' WHERE ec = 'EC 5.4.2.8';
UPDATE enzyme SET history = 'EC 5.4.2.8 created 1981 as EC 2.7.5.7, transferred 1984 to EC 5.4.2.8' WHERE ec = 'EC 5.4.2.8';
UPDATE enzyme SET sys_name = 'Phosphoenolpyruvate 2,3-phosphonomutase' WHERE ec = 'EC 5.4.2.9';
UPDATE enzyme SET history = 'EC 5.4.2.9 created 1990' WHERE ec = 'EC 5.4.2.9';
UPDATE enzyme SET sys_name = 'Alpha-D-glucosamine 1,6-phosphomutase' WHERE ec = 'EC 5.4.2.10';
UPDATE enzyme SET history = 'EC 5.4.2.10 created 2001' WHERE ec = 'EC 5.4.2.10';
UPDATE enzyme SET sys_name = 'D-phosphoglycerate 2,3-phosphomutase (2,3-diphosphoglycerate-dependent)' WHERE ec = 'EC 5.4.2.11';
UPDATE enzyme SET history = 'EC 5.4.2.11 created 1961 as EC 5.4.2.1 (EC 2.7.5.3 created 1961, incorporated 1984) transferred 2013 to 5.4.2.11' WHERE ec = 'EC 5.4.2.11';
UPDATE enzyme SET sys_name = 'D-phosphoglycerate 2,3-phosphomutase (2,3-diphosphoglycerate-independent)' WHERE ec = 'EC 5.4.2.12';
UPDATE enzyme SET history = 'EC 5.4.2.12 created 2013' WHERE ec = 'EC 5.4.2.12';
UPDATE enzyme SET history = 'EC 5.4.3.1 created 1972, deleted 1976' WHERE ec = 'EC 5.4.3.1';
UPDATE enzyme SET sys_name = 'L-lysine 2,3-aminomutase' WHERE ec = 'EC 5.4.3.2';
UPDATE enzyme SET history = 'EC 5.4.3.2 created 1972' WHERE ec = 'EC 5.4.3.2';
UPDATE enzyme SET sys_name = '(3S)-3,6-diaminohexanoate 5,6-aminomutase' WHERE ec = 'EC 5.4.3.3';
UPDATE enzyme SET history = 'EC 5.4.3.3 created 1972' WHERE ec = 'EC 5.4.3.3';
UPDATE enzyme SET sys_name = 'D-2,6-diaminohexanoate 5,6-aminomutase' WHERE ec = 'EC 5.4.3.4';
UPDATE enzyme SET history = 'EC 5.4.3.4 created 1972, modified 2003' WHERE ec = 'EC 5.4.3.4';
UPDATE enzyme SET sys_name = 'D-ornithine 4,5-aminomutase' WHERE ec = 'EC 5.4.3.5';
UPDATE enzyme SET history = 'EC 5.4.3.5 created 1972 as EC 5.4.3.1, transferred 1976 to EC 5.4.3.5, modified 2003' WHERE ec = 'EC 5.4.3.5';
UPDATE enzyme SET sys_name = 'L-tyrosine 2,3-aminomutase' WHERE ec = 'EC 5.4.3.6';
UPDATE enzyme SET history = 'EC 5.4.3.6 created 1976' WHERE ec = 'EC 5.4.3.6';
UPDATE enzyme SET sys_name = '(2S)-alpha-leucine 2,3-aminomutase' WHERE ec = 'EC 5.4.3.7';
UPDATE enzyme SET history = 'EC 5.4.3.7 created 1982' WHERE ec = 'EC 5.4.3.7';
UPDATE enzyme SET sys_name = '(S)-4-amino-5-oxopentanoate 4,5-aminomutase' WHERE ec = 'EC 5.4.3.8';
UPDATE enzyme SET history = 'EC 5.4.3.8 created 1983' WHERE ec = 'EC 5.4.3.8';
UPDATE enzyme SET sys_name = 'L-glutamate 2,3-aminomutase' WHERE ec = 'EC 5.4.3.9';
UPDATE enzyme SET history = 'EC 5.4.3.9 created 2012' WHERE ec = 'EC 5.4.3.9';
UPDATE enzyme SET sys_name = 'L-phenylalanine 2,3-aminomutase ((R)-3-amino-3-phenylpropanoate forming)' WHERE ec = 'EC 5.4.3.10';
UPDATE enzyme SET history = 'EC 5.4.3.10 created 2013' WHERE ec = 'EC 5.4.3.10';
UPDATE enzyme SET sys_name = 'L-phenylalanine 2,3-aminomutase ((S)-3-amino-3-phenylpropanoate-forming)' WHERE ec = 'EC 5.4.3.11';
UPDATE enzyme SET history = 'EC 5.4.3.11 created 2013' WHERE ec = 'EC 5.4.3.11';
UPDATE enzyme SET sys_name = '(Hydroxyamino)benzene hydroxymutase' WHERE ec = 'EC 5.4.4.1';
UPDATE enzyme SET history = 'EC 5.4.4.1 created 2003' WHERE ec = 'EC 5.4.4.1';
UPDATE enzyme SET sys_name = 'Isochorismate hydroxymutase' WHERE ec = 'EC 5.4.4.2';
UPDATE enzyme SET history = 'EC 5.4.4.2 created 1972 as EC 5.4.99.6, transferred 2003 to EC 5.4.4.2' WHERE ec = 'EC 5.4.4.2';
UPDATE enzyme SET sys_name = '3-(hydroxyamino)phenol hydroxymutase' WHERE ec = 'EC 5.4.4.3';
UPDATE enzyme SET history = 'EC 5.4.4.3 created 2003' WHERE ec = 'EC 5.4.4.3';
UPDATE enzyme SET sys_name = 'Geraniol hydroxymutase' WHERE ec = 'EC 5.4.4.4';
UPDATE enzyme SET history = 'EC 5.4.4.4 created 2011, modified 2012' WHERE ec = 'EC 5.4.4.4';
UPDATE enzyme SET sys_name = '(8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate hydroxymutase ((5S,8R,9Z,12Z)-5,8-dihydroxyoctadeca-9,12-dienoate-forming)' WHERE ec = 'EC 5.4.4.5';
UPDATE enzyme SET history = 'EC 5.4.4.5 created 2011' WHERE ec = 'EC 5.4.4.5';
UPDATE enzyme SET sys_name = '(8R,9Z,12Z)-8-hydroperoxy-9,12-octadecadienoate hydroxymutase ((7S,8S,9Z,12Z)-7,8-dihydroxyoctadeca-9,12-dienoate-forming)' WHERE ec = 'EC 5.4.4.6';
UPDATE enzyme SET history = 'EC 5.4.4.6 created 2011' WHERE ec = 'EC 5.4.4.6';
UPDATE enzyme SET sys_name = 'L-threo-3-methylaspartate carboxy-aminomethylmutase' WHERE ec = 'EC 5.4.99.1';
UPDATE enzyme SET history = 'EC 5.4.99.1 created 1961' WHERE ec = 'EC 5.4.99.1';
UPDATE enzyme SET sys_name = '(R)-methylmalonyl-CoA CoA-carbonylmutase' WHERE ec = 'EC 5.4.99.2';
UPDATE enzyme SET history = 'EC 5.4.99.2 created 1961, modified 1983, modified 2004' WHERE ec = 'EC 5.4.99.2';
UPDATE enzyme SET sys_name = '2-acetolactate methylmutase' WHERE ec = 'EC 5.4.99.3';
UPDATE enzyme SET history = 'EC 5.4.99.3 created 1972' WHERE ec = 'EC 5.4.99.3';
UPDATE enzyme SET sys_name = '2-methyleneglutarate carboxy-methylenemethylmutase' WHERE ec = 'EC 5.4.99.4';
UPDATE enzyme SET history = 'EC 5.4.99.4 created 1972' WHERE ec = 'EC 5.4.99.4';
UPDATE enzyme SET sys_name = 'Chorismate pyruvatemutase' WHERE ec = 'EC 5.4.99.5';
UPDATE enzyme SET history = 'EC 5.4.99.5 created 1972' WHERE ec = 'EC 5.4.99.5';
UPDATE enzyme SET history = 'EC 5.4.99.6 created 1972, deleted 2003' WHERE ec = 'EC 5.4.99.6';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, lanosterol-forming)' WHERE ec = 'EC 5.4.99.7';
UPDATE enzyme SET history = 'EC 5.4.99.7 created 1961 as EC 1.99.1.13, transferred 1965 to EC 1.14.1.3, part transferred 1972 to EC 5.4.99.7 rest to EC 1.14.99.7' WHERE ec = 'EC 5.4.99.7';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, cycloartenol-forming)' WHERE ec = 'EC 5.4.99.8';
UPDATE enzyme SET history = 'EC 5.4.99.8 created 1972' WHERE ec = 'EC 5.4.99.8';
UPDATE enzyme SET sys_name = 'UDP-alpha-D-galactopyranose furanomutase' WHERE ec = 'EC 5.4.99.9';
UPDATE enzyme SET history = 'EC 5.4.99.9 created 1984' WHERE ec = 'EC 5.4.99.9';
UPDATE enzyme SET history = 'EC 5.4.99.10 created 1984, deleted 1992' WHERE ec = 'EC 5.4.99.10';
UPDATE enzyme SET sys_name = 'Sucrose glucosylmutase' WHERE ec = 'EC 5.4.99.11';
UPDATE enzyme SET history = 'EC 5.4.99.11 created 1989 (EC 5.4.99.10 created 1984, incorporated 1992)' WHERE ec = 'EC 5.4.99.11';
UPDATE enzyme SET sys_name = 'tRNA-uridine(38-40) uracil mutase' WHERE ec = 'EC 5.4.99.12';
UPDATE enzyme SET history = 'EC 5.4.99.12 created 1990, modified 2011' WHERE ec = 'EC 5.4.99.12';
UPDATE enzyme SET sys_name = '2-methylpropanoyl-CoA CoA-carbonylmutase' WHERE ec = 'EC 5.4.99.13';
UPDATE enzyme SET history = 'EC 5.4.99.13 created 1992' WHERE ec = 'EC 5.4.99.13';
UPDATE enzyme SET sys_name = '4-carboxymethyl-4-methylbut-2-en-1,4-olide methylmutase' WHERE ec = 'EC 5.4.99.14';
UPDATE enzyme SET history = 'EC 5.4.99.14 created 1992' WHERE ec = 'EC 5.4.99.14';
UPDATE enzyme SET sys_name = '(1-&gt;4)-alpha-D-glucan 1-alpha-D-glucosylmutase' WHERE ec = 'EC 5.4.99.15';
UPDATE enzyme SET history = 'EC 5.4.99.15 created 1999' WHERE ec = 'EC 5.4.99.15';
UPDATE enzyme SET sys_name = 'Maltose alpha-D-glucosylmutase' WHERE ec = 'EC 5.4.99.16';
UPDATE enzyme SET history = 'EC 5.4.99.16 created 1999' WHERE ec = 'EC 5.4.99.16';
UPDATE enzyme SET sys_name = 'Squalene mutase (cyclizing, hop-22(29)-ene-forming)' WHERE ec = 'EC 5.4.99.17';
UPDATE enzyme SET history = 'EC 5.4.99.17 created 2002, modified 2011' WHERE ec = 'EC 5.4.99.17';
UPDATE enzyme SET sys_name = '5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole carboxymutase' WHERE ec = 'EC 5.4.99.18';
UPDATE enzyme SET history = 'EC 5.4.99.18 created 2006' WHERE ec = 'EC 5.4.99.18';
UPDATE enzyme SET sys_name = '16S rRNA-uridine(516) uracil mutase' WHERE ec = 'EC 5.4.99.19';
UPDATE enzyme SET history = 'EC 5.4.99.19 created 2011' WHERE ec = 'EC 5.4.99.19';
UPDATE enzyme SET sys_name = '23S rRNA-uridine(2457) uracil mutase' WHERE ec = 'EC 5.4.99.20';
UPDATE enzyme SET history = 'EC 5.4.99.20 created 2011' WHERE ec = 'EC 5.4.99.20';
UPDATE enzyme SET sys_name = '23S rRNA-uridine(2604) uracil mutase' WHERE ec = 'EC 5.4.99.21';
UPDATE enzyme SET history = 'EC 5.4.99.21 created 2011' WHERE ec = 'EC 5.4.99.21';
UPDATE enzyme SET sys_name = '23S rRNA-uridine(2605) uracil mutase' WHERE ec = 'EC 5.4.99.22';
UPDATE enzyme SET history = 'EC 5.4.99.22 created 2011' WHERE ec = 'EC 5.4.99.22';
UPDATE enzyme SET sys_name = '23S rRNA-uridine(1911/1915/1917) uracil mutase' WHERE ec = 'EC 5.4.99.23';
UPDATE enzyme SET history = 'EC 5.4.99.23 created 2011' WHERE ec = 'EC 5.4.99.23';
UPDATE enzyme SET sys_name = '23S rRNA-uridine(955/2504/2580) uracil mutase' WHERE ec = 'EC 5.4.99.24';
UPDATE enzyme SET history = 'EC 5.4.99.24 created 2011' WHERE ec = 'EC 5.4.99.24';
UPDATE enzyme SET sys_name = 'tRNA-uridine(55) uracil mutase' WHERE ec = 'EC 5.4.99.25';
UPDATE enzyme SET history = 'EC 5.4.99.25 created 2011, modified 2011' WHERE ec = 'EC 5.4.99.25';
UPDATE enzyme SET sys_name = 'tRNA-uridine65 uracil mutase' WHERE ec = 'EC 5.4.99.26';
UPDATE enzyme SET history = 'EC 5.4.99.26 created 2011' WHERE ec = 'EC 5.4.99.26';
UPDATE enzyme SET sys_name = 'tRNA-uridine(13) uracil mutase' WHERE ec = 'EC 5.4.99.27';
UPDATE enzyme SET history = 'EC 5.4.99.27 created 2011' WHERE ec = 'EC 5.4.99.27';
UPDATE enzyme SET sys_name = 'tRNA-uridine(32) uracil mutase' WHERE ec = 'EC 5.4.99.28';
UPDATE enzyme SET history = 'EC 5.4.99.28 created 2011' WHERE ec = 'EC 5.4.99.28';
UPDATE enzyme SET sys_name = '23S rRNA-uridine(746) uracil mutase' WHERE ec = 'EC 5.4.99.29';
UPDATE enzyme SET history = 'EC 5.4.99.29 created 2011' WHERE ec = 'EC 5.4.99.29';
UPDATE enzyme SET sys_name = 'UDP-arabinopyranose pyranomutase' WHERE ec = 'EC 5.4.99.30';
UPDATE enzyme SET history = 'EC 5.4.99.30 created 2011' WHERE ec = 'EC 5.4.99.30';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, thalianol-forming)' WHERE ec = 'EC 5.4.99.31';
UPDATE enzyme SET history = 'EC 5.4.99.31 created 2011' WHERE ec = 'EC 5.4.99.31';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, (17Z)-protosta-17(20),24-dien-3-beta-ol-forming)' WHERE ec = 'EC 5.4.99.32';
UPDATE enzyme SET history = 'EC 5.4.99.32 created 2011' WHERE ec = 'EC 5.4.99.32';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, cucurbitadienol-forming)' WHERE ec = 'EC 5.4.99.33';
UPDATE enzyme SET history = 'EC 5.4.99.33 created 2011' WHERE ec = 'EC 5.4.99.33';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, germanicol-forming)' WHERE ec = 'EC 5.4.99.34';
UPDATE enzyme SET history = 'EC 5.4.99.34 created 2011' WHERE ec = 'EC 5.4.99.34';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, taraxerol-forming)' WHERE ec = 'EC 5.4.99.35';
UPDATE enzyme SET history = 'EC 5.4.99.35 created 2011' WHERE ec = 'EC 5.4.99.35';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, isomultiflorenol-forming)' WHERE ec = 'EC 5.4.99.36';
UPDATE enzyme SET history = 'EC 5.4.99.36 created 2011' WHERE ec = 'EC 5.4.99.36';
UPDATE enzyme SET sys_name = 'Squalene mutase (cyclizing, dammara-20,24-diene-forming)' WHERE ec = 'EC 5.4.99.37';
UPDATE enzyme SET history = 'EC 5.4.99.37 created 2011' WHERE ec = 'EC 5.4.99.37';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, camelliol-C-forming)' WHERE ec = 'EC 5.4.99.38';
UPDATE enzyme SET history = 'EC 5.4.99.38 created 2011' WHERE ec = 'EC 5.4.99.38';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, beta-amyrin-forming)' WHERE ec = 'EC 5.4.99.39';
UPDATE enzyme SET history = 'EC 5.4.99.39 created 2011' WHERE ec = 'EC 5.4.99.39';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, alpha-amyrin-forming)' WHERE ec = 'EC 5.4.99.40';
UPDATE enzyme SET history = 'EC 5.4.99.40 created 2011' WHERE ec = 'EC 5.4.99.40';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, lupeol-forming)' WHERE ec = 'EC 5.4.99.41';
UPDATE enzyme SET history = 'EC 5.4.99.41 created 2011' WHERE ec = 'EC 5.4.99.41';
UPDATE enzyme SET sys_name = 'tRNA-uridine(31) uracil mutase' WHERE ec = 'EC 5.4.99.42';
UPDATE enzyme SET history = 'EC 5.4.99.42 created 2011' WHERE ec = 'EC 5.4.99.42';
UPDATE enzyme SET sys_name = '21S rRNA-uridine(2819) uracil mutase' WHERE ec = 'EC 5.4.99.43';
UPDATE enzyme SET history = 'EC 5.4.99.43 created 2011' WHERE ec = 'EC 5.4.99.43';
UPDATE enzyme SET sys_name = 'Mitochondrial tRNA-uridine(27/28) uracil mutase' WHERE ec = 'EC 5.4.99.44';
UPDATE enzyme SET history = 'EC 5.4.99.44 created 2011' WHERE ec = 'EC 5.4.99.44';
UPDATE enzyme SET sys_name = 'tRNA-uridine(38/39) uracil mutase' WHERE ec = 'EC 5.4.99.45';
UPDATE enzyme SET history = 'EC 5.4.99.45 created 2011' WHERE ec = 'EC 5.4.99.45';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, shionone-forming)' WHERE ec = 'EC 5.4.99.46';
UPDATE enzyme SET history = 'EC 5.4.99.46 created 2011' WHERE ec = 'EC 5.4.99.46';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, parkeol-forming)' WHERE ec = 'EC 5.4.99.47';
UPDATE enzyme SET history = 'EC 5.4.99.47 created 2011' WHERE ec = 'EC 5.4.99.47';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, achilleol-B-forming)' WHERE ec = 'EC 5.4.99.48';
UPDATE enzyme SET history = 'EC 5.4.99.48 created 2011' WHERE ec = 'EC 5.4.99.48';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, glutinol-forming)' WHERE ec = 'EC 5.4.99.49';
UPDATE enzyme SET history = 'EC 5.4.99.49 created 2011' WHERE ec = 'EC 5.4.99.49';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, friedelin-forming)' WHERE ec = 'EC 5.4.99.50';
UPDATE enzyme SET history = 'EC 5.4.99.50 created 2011' WHERE ec = 'EC 5.4.99.50';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, baccharis-oxide-forming)' WHERE ec = 'EC 5.4.99.51';
UPDATE enzyme SET history = 'EC 5.4.99.51 created 2011' WHERE ec = 'EC 5.4.99.51';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, alpha-seco-amyrin-forming)' WHERE ec = 'EC 5.4.99.52';
UPDATE enzyme SET history = 'EC 5.4.99.52 created 2011' WHERE ec = 'EC 5.4.99.52';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, marneral-forming)' WHERE ec = 'EC 5.4.99.53';
UPDATE enzyme SET history = 'EC 5.4.99.53 created 2011' WHERE ec = 'EC 5.4.99.53';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, beta-seco-amyrin-forming)' WHERE ec = 'EC 5.4.99.54';
UPDATE enzyme SET history = 'EC 5.4.99.54 created 2011' WHERE ec = 'EC 5.4.99.54';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, delta-amyrin-forming)' WHERE ec = 'EC 5.4.99.55';
UPDATE enzyme SET history = 'EC 5.4.99.55 created 2011' WHERE ec = 'EC 5.4.99.55';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, tirucalla-7,24-dien-3-beta-ol-forming)' WHERE ec = 'EC 5.4.99.56';
UPDATE enzyme SET history = 'EC 5.4.99.56 created 2011' WHERE ec = 'EC 5.4.99.56';
UPDATE enzyme SET sys_name = '(3S)-2,3-epoxy-2,3-dihydrosqualene mutase (cyclizing, baruol-forming)' WHERE ec = 'EC 5.4.99.57';
UPDATE enzyme SET history = 'EC 5.4.99.57 created 2012' WHERE ec = 'EC 5.4.99.57';
UPDATE enzyme SET sys_name = 'L-lysine carboxy-aminomethylmutase' WHERE ec = 'EC 5.4.99.58';
UPDATE enzyme SET history = 'EC 5.4.99.58 created 2012' WHERE ec = 'EC 5.4.99.58';
UPDATE enzyme SET sys_name = 'dTDP-alpha-D-fucopyranose furanomutase' WHERE ec = 'EC 5.4.99.59';
UPDATE enzyme SET history = 'EC 5.4.99.59 created 2013' WHERE ec = 'EC 5.4.99.59';
UPDATE enzyme SET sys_name = 'Precorrin-8 11,12-methylmutase' WHERE ec = 'EC 5.4.99.60';
UPDATE enzyme SET history = 'EC 5.4.99.60 created 2014' WHERE ec = 'EC 5.4.99.60';
UPDATE enzyme SET sys_name = 'Precorrin-8X 11,12-methylmutase' WHERE ec = 'EC 5.4.99.61';
UPDATE enzyme SET history = 'EC 5.4.99.61 created 1999 as EC 5.4.1.2, transferred 2014 to EC 5.4.99.61' WHERE ec = 'EC 5.4.99.61';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 5.4.99.n1';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 5.4.99.n2';
UPDATE enzyme SET sys_name = '2,5-dihydro-5-oxofuran-2-acetate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.1';
UPDATE enzyme SET history = 'EC 5.5.1.1 created 1961' WHERE ec = 'EC 5.5.1.1';
UPDATE enzyme SET sys_name = '2-carboxy-2,5-dihydro-5-oxofuran-2-acetate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.2';
UPDATE enzyme SET history = 'EC 5.5.1.2 created 1972' WHERE ec = 'EC 5.5.1.2';
UPDATE enzyme SET sys_name = 'Tetrahydroxypteridine lyase (isomerizing)' WHERE ec = 'EC 5.5.1.3';
UPDATE enzyme SET history = 'EC 5.5.1.3 created 1972' WHERE ec = 'EC 5.5.1.3';
UPDATE enzyme SET sys_name = '1D-myo-inositol-3-phosphate lyase (isomerizing)' WHERE ec = 'EC 5.5.1.4';
UPDATE enzyme SET history = 'EC 5.5.1.4 created 1972, modified 2001' WHERE ec = 'EC 5.5.1.4';
UPDATE enzyme SET sys_name = '3-carboxy-2,5-dihydro-5-oxofuran-2-acetate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.5';
UPDATE enzyme SET history = 'EC 5.5.1.5 created 1972' WHERE ec = 'EC 5.5.1.5';
UPDATE enzyme SET sys_name = 'Flavanone lyase (decyclizing)' WHERE ec = 'EC 5.5.1.6';
UPDATE enzyme SET history = 'EC 5.5.1.6 created 1972' WHERE ec = 'EC 5.5.1.6';
UPDATE enzyme SET sys_name = '2-chloro-2,5-dihydro-5-oxofuran-2-acetate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.7';
UPDATE enzyme SET history = 'EC 5.5.1.7 created 1983' WHERE ec = 'EC 5.5.1.7';
UPDATE enzyme SET sys_name = '(+)-bornyl-diphosphate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.8';
UPDATE enzyme SET history = 'EC 5.5.1.8 created 1984, modified 2012' WHERE ec = 'EC 5.5.1.8';
UPDATE enzyme SET sys_name = 'Cycloeucalenol lyase (cyclopropane-decyclizing)' WHERE ec = 'EC 5.5.1.9';
UPDATE enzyme SET history = 'EC 5.5.1.9 created 1986' WHERE ec = 'EC 5.5.1.9';
UPDATE enzyme SET sys_name = 'Alpha-pinene-oxide lyase (decyclizing)' WHERE ec = 'EC 5.5.1.10';
UPDATE enzyme SET history = 'EC 5.5.1.10 created 1990' WHERE ec = 'EC 5.5.1.10';
UPDATE enzyme SET sys_name = '2,4-dichloro-2,5-dihydro-5-oxofuran-2-acetate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.11';
UPDATE enzyme SET history = 'EC 5.5.1.11 created 1992' WHERE ec = 'EC 5.5.1.11';
UPDATE enzyme SET sys_name = '(+)-copalyl-diphosphate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.12';
UPDATE enzyme SET history = 'EC 5.5.1.12 created 2002, modified 2012' WHERE ec = 'EC 5.5.1.12';
UPDATE enzyme SET sys_name = 'Ent-copalyl-diphosphate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.13';
UPDATE enzyme SET history = 'EC 5.5.1.13 created 2002' WHERE ec = 'EC 5.5.1.13';
UPDATE enzyme SET sys_name = '9-alpha-copalyl-diphosphate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.14';
UPDATE enzyme SET history = 'EC 5.5.1.14 created 2008' WHERE ec = 'EC 5.5.1.14';
UPDATE enzyme SET sys_name = 'Terpentedienyl-diphosphate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.15';
UPDATE enzyme SET history = 'EC 5.5.1.15 crerated 2008' WHERE ec = 'EC 5.5.1.15';
UPDATE enzyme SET sys_name = 'Tuberculosinyl-diphosphate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.16';
UPDATE enzyme SET history = 'EC 5.5.1.16 created 2008, modified 2012' WHERE ec = 'EC 5.5.1.16';
UPDATE enzyme SET sys_name = '(S)-beta-macrocarpene lyase (decyclizing)' WHERE ec = 'EC 5.5.1.17';
UPDATE enzyme SET history = 'EC 5.5.1.17 created 2011' WHERE ec = 'EC 5.5.1.17';
UPDATE enzyme SET sys_name = 'Carotenoid psi-end group lyase (decyclizing)' WHERE ec = 'EC 5.5.1.18';
UPDATE enzyme SET history = 'EC 5.5.1.18 created 2011' WHERE ec = 'EC 5.5.1.18';
UPDATE enzyme SET sys_name = 'Carotenoid beta-end group lyase (decyclizing)' WHERE ec = 'EC 5.5.1.19';
UPDATE enzyme SET history = 'EC 5.5.1.19 created 2011' WHERE ec = 'EC 5.5.1.19';
UPDATE enzyme SET sys_name = 'Prosolanapyrone-III:(-)-solanapyrone A isomerase' WHERE ec = 'EC 5.5.1.20';
UPDATE enzyme SET history = 'EC 5.5.1.20 created 2011' WHERE ec = 'EC 5.5.1.20';
UPDATE enzyme SET history = 'EC 5.5.1.21 created 2012, deleted 2012' WHERE ec = 'EC 5.5.1.21';
UPDATE enzyme SET sys_name = '(-)-bornyl-diphosphate lyase (decyclizing)' WHERE ec = 'EC 5.5.1.22';
UPDATE enzyme SET history = 'EC 5.5.1.22 created 2012' WHERE ec = 'EC 5.5.1.22';
UPDATE enzyme SET sys_name = 'Methyl aklanonate-aklaviketone isomerase (cyclizing)' WHERE ec = 'EC 5.5.1.23';
UPDATE enzyme SET history = 'EC 5.5.1.23 created 2013' WHERE ec = 'EC 5.5.1.23';
UPDATE enzyme SET sys_name = 'Delta/gamma-tocopherol lyase (decyclizing)' WHERE ec = 'EC 5.5.1.24';
UPDATE enzyme SET history = 'EC 5.5.1.24 created 2013' WHERE ec = 'EC 5.5.1.24';
UPDATE enzyme SET sys_name = 'D-ribose lyase (isomerizing)' WHERE ec = 'EC 5.5.1.n1';
UPDATE enzyme SET history = 'EC 5.5.1.n1 created 2010' WHERE ec = 'EC 5.5.1.n1';
UPDATE enzyme SET history = 'EC 5.5.1.n2 created 2012' WHERE ec = 'EC 5.5.1.n2';
UPDATE enzyme SET sys_name = 'Benzyl-thiocyanate isomerase' WHERE ec = 'EC 5.99.1.1';
UPDATE enzyme SET history = 'EC 5.99.1.1 created 1965' WHERE ec = 'EC 5.99.1.1';
UPDATE enzyme SET sys_name = 'DNA topoisomerase' WHERE ec = 'EC 5.99.1.2';
UPDATE enzyme SET history = 'EC 5.99.1.2 created 1984' WHERE ec = 'EC 5.99.1.2';
UPDATE enzyme SET sys_name = 'DNA topoisomerase (ATP-hydrolyzing)' WHERE ec = 'EC 5.99.1.3';
UPDATE enzyme SET history = 'EC 5.99.1.3 created 1984' WHERE ec = 'EC 5.99.1.3';
UPDATE enzyme SET sys_name = '2-hydroxy-2H-chromene-2-carboxylate--(3E)-4-(2-hydroxyphenyl)-2-oxobut-3-enoate isomerase' WHERE ec = 'EC 5.99.1.4';
UPDATE enzyme SET history = 'EC 5.99.1.4 created 2010' WHERE ec = 'EC 5.99.1.4';
UPDATE enzyme SET sys_name = 'L-tyrosine:tRNA(Tyr) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.1';
UPDATE enzyme SET history = 'EC 6.1.1.1 created 1961, modified 2002' WHERE ec = 'EC 6.1.1.1';
UPDATE enzyme SET sys_name = 'L-tryptophan:tRNA(Trp) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.2';
UPDATE enzyme SET history = 'EC 6.1.1.2 created 1961, modified 2002' WHERE ec = 'EC 6.1.1.2';
UPDATE enzyme SET sys_name = 'L-threonine:tRNA(Thr) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.3';
UPDATE enzyme SET history = 'EC 6.1.1.3 created 1961' WHERE ec = 'EC 6.1.1.3';
UPDATE enzyme SET sys_name = 'L-leucine:tRNA(Leu) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.4';
UPDATE enzyme SET history = 'EC 6.1.1.4 created 1961' WHERE ec = 'EC 6.1.1.4';
UPDATE enzyme SET sys_name = 'L-isoleucine:tRNA(Ile) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.5';
UPDATE enzyme SET history = 'EC 6.1.1.5 created 1961' WHERE ec = 'EC 6.1.1.5';
UPDATE enzyme SET sys_name = 'L-lysine:tRNA(Lys) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.6';
UPDATE enzyme SET history = 'EC 6.1.1.6 created 1961' WHERE ec = 'EC 6.1.1.6';
UPDATE enzyme SET sys_name = 'L-alanine:tRNA(Ala) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.7';
UPDATE enzyme SET history = 'EC 6.1.1.7 created 1961' WHERE ec = 'EC 6.1.1.7';
UPDATE enzyme SET history = 'created 1961, deleted 1965' WHERE ec = 'EC 6.1.1.8';
UPDATE enzyme SET sys_name = 'L-valine:tRNA(Val) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.9';
UPDATE enzyme SET history = 'EC 6.1.1.9 created 1961' WHERE ec = 'EC 6.1.1.9';
UPDATE enzyme SET sys_name = 'L-methionine:tRNA(Met) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.10';
UPDATE enzyme SET history = 'EC 6.1.1.10 created 1961, modified 2002' WHERE ec = 'EC 6.1.1.10';
UPDATE enzyme SET sys_name = 'L-serine:tRNA(Ser) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.11';
UPDATE enzyme SET history = 'EC 6.1.1.11 created 1961, modified 2002' WHERE ec = 'EC 6.1.1.11';
UPDATE enzyme SET sys_name = 'L-aspartate:tRNA(Asp) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.12';
UPDATE enzyme SET history = 'EC 6.1.1.12 created 1965' WHERE ec = 'EC 6.1.1.12';
UPDATE enzyme SET sys_name = 'D-alanine:poly(phosphoribitol) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.13';
UPDATE enzyme SET history = 'EC 6.1.1.13 created 1965, modified 2001' WHERE ec = 'EC 6.1.1.13';
UPDATE enzyme SET sys_name = 'Glycine:tRNA(Gly) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.14';
UPDATE enzyme SET history = 'EC 6.1.1.14 created 1972' WHERE ec = 'EC 6.1.1.14';
UPDATE enzyme SET sys_name = 'L-proline:tRNA(Pro) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.15';
UPDATE enzyme SET history = 'EC 6.1.1.15 created 1972' WHERE ec = 'EC 6.1.1.15';
UPDATE enzyme SET sys_name = 'L-cysteine:tRNA(Cys) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.16';
UPDATE enzyme SET history = 'EC 6.1.1.16 created 1972' WHERE ec = 'EC 6.1.1.16';
UPDATE enzyme SET sys_name = 'L-glutamate:tRNA(Glu) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.17';
UPDATE enzyme SET history = 'EC 6.1.1.17 created 1972' WHERE ec = 'EC 6.1.1.17';
UPDATE enzyme SET sys_name = 'L-glutamine:tRNA(Gln) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.18';
UPDATE enzyme SET history = 'EC 6.1.1.18 created 1972' WHERE ec = 'EC 6.1.1.18';
UPDATE enzyme SET sys_name = 'L-arginine:tRNA(Arg) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.19';
UPDATE enzyme SET history = 'EC 6.1.1.19 created 1972' WHERE ec = 'EC 6.1.1.19';
UPDATE enzyme SET sys_name = 'L-phenylalanine:tRNA(Phe) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.20';
UPDATE enzyme SET history = 'EC 6.1.1.20 created 1972' WHERE ec = 'EC 6.1.1.20';
UPDATE enzyme SET sys_name = 'L-histidine:tRNA(His) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.21';
UPDATE enzyme SET history = 'EC 6.1.1.21 created 1972' WHERE ec = 'EC 6.1.1.21';
UPDATE enzyme SET sys_name = 'L-asparagine:tRNA(Asn) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.22';
UPDATE enzyme SET history = 'EC 6.1.1.22 created 1976' WHERE ec = 'EC 6.1.1.22';
UPDATE enzyme SET sys_name = 'L-aspartate:tRNA(Asx) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.23';
UPDATE enzyme SET history = 'EC 6.1.1.23 created 2002' WHERE ec = 'EC 6.1.1.23';
UPDATE enzyme SET sys_name = 'L-glutamate:tRNA(Glx) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.24';
UPDATE enzyme SET history = 'EC 6.1.1.24 created 2002' WHERE ec = 'EC 6.1.1.24';
UPDATE enzyme SET history = 'EC 6.1.1.25 created 2002, deleted 2012' WHERE ec = 'EC 6.1.1.25';
UPDATE enzyme SET sys_name = 'L-pyrrolysine:tRNA(Pyl) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.26';
UPDATE enzyme SET history = 'EC 6.1.1.26 created 2007' WHERE ec = 'EC 6.1.1.26';
UPDATE enzyme SET sys_name = 'O-phospho-L-serine:tRNA(Cys) ligase (AMP-forming)' WHERE ec = 'EC 6.1.1.27';
UPDATE enzyme SET history = 'EC 6.1.1.27 created 2009' WHERE ec = 'EC 6.1.1.27';
UPDATE enzyme SET sys_name = 'D-alanine:(R)-lactate ligase (ADP-forming)' WHERE ec = 'EC 6.1.2.1';
UPDATE enzyme SET history = 'EC 6.1.2.1 created 2010' WHERE ec = 'EC 6.1.2.1';
UPDATE enzyme SET sys_name = 'Acetate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.1';
UPDATE enzyme SET history = 'EC 6.2.1.1 created 1961' WHERE ec = 'EC 6.2.1.1';
UPDATE enzyme SET sys_name = 'Butanoate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.2';
UPDATE enzyme SET history = 'EC 6.2.1.2 created 1961, modified 2011' WHERE ec = 'EC 6.2.1.2';
UPDATE enzyme SET sys_name = 'Long-chain fatty acid:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.3';
UPDATE enzyme SET history = 'EC 6.2.1.3 created 1961, modified 1989, modified 2011' WHERE ec = 'EC 6.2.1.3';
UPDATE enzyme SET sys_name = 'Succinate:CoA ligase (GDP-forming)' WHERE ec = 'EC 6.2.1.4';
UPDATE enzyme SET history = 'EC 6.2.1.4 created 1961' WHERE ec = 'EC 6.2.1.4';
UPDATE enzyme SET sys_name = 'Succinate:CoA ligase (ADP-forming)' WHERE ec = 'EC 6.2.1.5';
UPDATE enzyme SET history = 'EC 6.2.1.5 created 1961' WHERE ec = 'EC 6.2.1.5';
UPDATE enzyme SET sys_name = 'Glutarate:CoA ligase (ADP-forming)' WHERE ec = 'EC 6.2.1.6';
UPDATE enzyme SET history = 'EC 6.2.1.6 created 1961' WHERE ec = 'EC 6.2.1.6';
UPDATE enzyme SET sys_name = 'Cholate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.7';
UPDATE enzyme SET history = 'EC 6.2.1.7 created 1961 (EC 6.2.1.29 created 1992, incorporated 2005), modified 2005' WHERE ec = 'EC 6.2.1.7';
UPDATE enzyme SET sys_name = 'Oxalate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.8';
UPDATE enzyme SET history = 'EC 6.2.1.8 created 1972' WHERE ec = 'EC 6.2.1.8';
UPDATE enzyme SET sys_name = 'Malate:CoA ligase (ADP-forming)' WHERE ec = 'EC 6.2.1.9';
UPDATE enzyme SET history = 'EC 6.2.1.9 created 1972' WHERE ec = 'EC 6.2.1.9';
UPDATE enzyme SET sys_name = 'Carboxylic acid:CoA ligase (GDP-forming)' WHERE ec = 'EC 6.2.1.10';
UPDATE enzyme SET history = 'EC 6.2.1.10 created 1972, modified 2011' WHERE ec = 'EC 6.2.1.10';
UPDATE enzyme SET sys_name = 'Biotin:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.11';
UPDATE enzyme SET history = 'EC 6.2.1.11 created 1972' WHERE ec = 'EC 6.2.1.11';
UPDATE enzyme SET sys_name = '4-coumarate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.12';
UPDATE enzyme SET history = 'EC 6.2.1.12 created 1976' WHERE ec = 'EC 6.2.1.12';
UPDATE enzyme SET sys_name = 'Acetate:CoA ligase (ADP-forming)' WHERE ec = 'EC 6.2.1.13';
UPDATE enzyme SET history = 'EC 6.2.1.13 created 1978' WHERE ec = 'EC 6.2.1.13';
UPDATE enzyme SET sys_name = '6-carboxyhexanoate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.14';
UPDATE enzyme SET history = 'EC 6.2.1.14 created 1983' WHERE ec = 'EC 6.2.1.14';
UPDATE enzyme SET sys_name = 'Arachidonate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.15';
UPDATE enzyme SET history = 'EC 6.2.1.15 created 1984' WHERE ec = 'EC 6.2.1.15';
UPDATE enzyme SET sys_name = 'Acetoacetate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.16';
UPDATE enzyme SET history = 'EC 6.2.1.16 created 1984' WHERE ec = 'EC 6.2.1.16';
UPDATE enzyme SET sys_name = 'Propanoate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.17';
UPDATE enzyme SET history = 'EC 6.2.1.17 created 1984' WHERE ec = 'EC 6.2.1.17';
UPDATE enzyme SET sys_name = 'Citrate:CoA ligase (ADP-forming)' WHERE ec = 'EC 6.2.1.18';
UPDATE enzyme SET history = 'EC 6.2.1.18 created 1986' WHERE ec = 'EC 6.2.1.18';
UPDATE enzyme SET sys_name = 'Long-chain-fatty-acid:protein ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.19';
UPDATE enzyme SET history = 'EC 6.2.1.19 created 1986, modified 2011' WHERE ec = 'EC 6.2.1.19';
UPDATE enzyme SET sys_name = 'Long-chain-fatty-acid:[acyl-carrier-protein] ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.20';
UPDATE enzyme SET history = 'EC 6.2.1.20 created 1986' WHERE ec = 'EC 6.2.1.20';
UPDATE enzyme SET history = 'EC 6.2.1.21 created 1986, deleted 2001' WHERE ec = 'EC 6.2.1.21';
UPDATE enzyme SET sys_name = 'Acetate:[citrate-(pro-3S)-lyase](thiol-form) ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.22';
UPDATE enzyme SET history = 'EC 6.2.1.22 created 1989' WHERE ec = 'EC 6.2.1.22';
UPDATE enzyme SET sys_name = 'Omega-dicarboxylate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.23';
UPDATE enzyme SET history = 'EC 6.2.1.23 created 1989, modified 2011' WHERE ec = 'EC 6.2.1.23';
UPDATE enzyme SET sys_name = 'Phytanate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.24';
UPDATE enzyme SET history = 'EC 6.2.1.24 created 1989' WHERE ec = 'EC 6.2.1.24';
UPDATE enzyme SET sys_name = 'Benzoate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.25';
UPDATE enzyme SET history = 'EC 6.2.1.25 created 1989' WHERE ec = 'EC 6.2.1.25';
UPDATE enzyme SET sys_name = '2-succinylbenzoate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.26';
UPDATE enzyme SET history = 'EC 6.2.1.26 created 1992' WHERE ec = 'EC 6.2.1.26';
UPDATE enzyme SET sys_name = '4-hydroxybenzoate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.27';
UPDATE enzyme SET history = 'EC 6.2.1.27 created 1992' WHERE ec = 'EC 6.2.1.27';
UPDATE enzyme SET sys_name = '(25R)-3-alpha,7-alpha-dihydroxy-5-beta-cholestan-26-oate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.28';
UPDATE enzyme SET history = 'EC 6.2.1.28 created 1992' WHERE ec = 'EC 6.2.1.28';
UPDATE enzyme SET history = 'EC 6.2.1.29 created 1992, deleted 2005' WHERE ec = 'EC 6.2.1.29';
UPDATE enzyme SET sys_name = 'Phenylacetate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.30';
UPDATE enzyme SET history = 'EC 6.2.1.30 created 1992 (EC 6.2.1.21 created 1986, incorporated 2001)' WHERE ec = 'EC 6.2.1.30';
UPDATE enzyme SET sys_name = '2-furoate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.31';
UPDATE enzyme SET history = 'EC 6.2.1.31 created 1992' WHERE ec = 'EC 6.2.1.31';
UPDATE enzyme SET sys_name = 'Anthranilate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.32';
UPDATE enzyme SET history = 'EC 6.2.1.32 created 1992' WHERE ec = 'EC 6.2.1.32';
UPDATE enzyme SET sys_name = '4-chlorobenzoate:CoA ligase' WHERE ec = 'EC 6.2.1.33';
UPDATE enzyme SET history = 'EC 6.2.1.33 created 1999' WHERE ec = 'EC 6.2.1.33';
UPDATE enzyme SET sys_name = 'Ferulate:CoASH ligase (ATP-hydrolyzing)' WHERE ec = 'EC 6.2.1.34';
UPDATE enzyme SET history = 'EC 6.2.1.34 created 2000' WHERE ec = 'EC 6.2.1.34';
UPDATE enzyme SET sys_name = 'Acetate:[acyl-carrier-protein] ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.35';
UPDATE enzyme SET history = 'EC 6.2.1.35 created 2008' WHERE ec = 'EC 6.2.1.35';
UPDATE enzyme SET sys_name = 'Hydroxypropionate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.36';
UPDATE enzyme SET history = 'EC 6.2.1.36 created 2009' WHERE ec = 'EC 6.2.1.36';
UPDATE enzyme SET sys_name = '3-hydroxybenzoate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.37';
UPDATE enzyme SET history = 'EC 6.2.1.37 created 2011' WHERE ec = 'EC 6.2.1.37';
UPDATE enzyme SET sys_name = '((1R)-2,2,3-trimethyl-5-oxocyclopent-3-enyl)acetate:CoA ligase (AMP-forming)' WHERE ec = 'EC 6.2.1.38';
UPDATE enzyme SET history = 'EC 6.2.1.38 created 2012' WHERE ec = 'EC 6.2.1.38';
UPDATE enzyme SET sys_name = '[BtrI acyl-carrier protein]:L-glutamate ligase (ADP-forming)' WHERE ec = 'EC 6.2.1.39';
UPDATE enzyme SET history = 'EC 6.2.1.39 created 2012' WHERE ec = 'EC 6.2.1.39';
UPDATE enzyme SET sys_name = '4-hydroxybutyrate:CoA ligase (AMP formimg)' WHERE ec = 'EC 6.2.1.40';
UPDATE enzyme SET history = 'EC 6.2.1.40 created 2014' WHERE ec = 'EC 6.2.1.40';
UPDATE enzyme SET history = 'EC 6.2.1.n1 created 2010' WHERE ec = 'EC 6.2.1.n1';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 6.2.1.n2';
UPDATE enzyme SET history = 'EC 6.2.1.n2 created 2012' WHERE ec = 'EC 6.2.1.n2';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 6.2.1.n3';
UPDATE enzyme SET history = 'EC 6.2.1.n3 created 2012' WHERE ec = 'EC 6.2.1.n3';
UPDATE enzyme SET sys_name = 'L-aspartate:ammonia ligase (AMP-forming)' WHERE ec = 'EC 6.3.1.1';
UPDATE enzyme SET history = 'EC 6.3.1.1 created 1961' WHERE ec = 'EC 6.3.1.1';
UPDATE enzyme SET sys_name = 'L-glutamate:ammonia ligase (ADP-forming)' WHERE ec = 'EC 6.3.1.2';
UPDATE enzyme SET history = 'EC 6.3.1.2 created 1961' WHERE ec = 'EC 6.3.1.2';
UPDATE enzyme SET history = 'EC 6.3.1.3 created 1961, deleted 1972' WHERE ec = 'EC 6.3.1.3';
UPDATE enzyme SET sys_name = 'L-aspartate:ammonia ligase (ADP-forming)' WHERE ec = 'EC 6.3.1.4';
UPDATE enzyme SET history = 'EC 6.3.1.4 created 1972' WHERE ec = 'EC 6.3.1.4';
UPDATE enzyme SET sys_name = 'Deamido-NAD(+):ammonia ligase (AMP-forming)' WHERE ec = 'EC 6.3.1.5';
UPDATE enzyme SET history = 'EC 6.3.1.5 created 1972' WHERE ec = 'EC 6.3.1.5';
UPDATE enzyme SET sys_name = 'L-glutamate:ethylamine ligase (ADP-forming)' WHERE ec = 'EC 6.3.1.6';
UPDATE enzyme SET history = 'EC 6.3.1.6 created 1976' WHERE ec = 'EC 6.3.1.6';
UPDATE enzyme SET sys_name = '4-methylene-L-glutamate:ammonia ligase (AMP-forming)' WHERE ec = 'EC 6.3.1.7';
UPDATE enzyme SET history = 'EC 6.3.1.7 created 1986' WHERE ec = 'EC 6.3.1.7';
UPDATE enzyme SET sys_name = 'Gamma-L-glutamyl-L-cysteinyl-glycine:spermidine ligase (ADP-forming) (spermidine is numbered so that atom N-1 is in the amino group of the aminopropyl part of the molecule)' WHERE ec = 'EC 6.3.1.8';
UPDATE enzyme SET history = 'EC 6.3.1.8 created 1999' WHERE ec = 'EC 6.3.1.8';
UPDATE enzyme SET sys_name = 'Glutathionylspermidine:glutathione ligase (ADP-forming)' WHERE ec = 'EC 6.3.1.9';
UPDATE enzyme SET history = 'EC 6.3.1.9 created 1999' WHERE ec = 'EC 6.3.1.9';
UPDATE enzyme SET sys_name = 'Adenosylcobyric acid:(R)-1-aminopropan-2-yl phosphate ligase (ADP-forming)' WHERE ec = 'EC 6.3.1.10';
UPDATE enzyme SET history = 'EC 6.3.1.10 created 2004' WHERE ec = 'EC 6.3.1.10';
UPDATE enzyme SET sys_name = 'L-glutamate:putrescine ligase (ADP-forming)' WHERE ec = 'EC 6.3.1.11';
UPDATE enzyme SET history = 'EC 6.3.1.11 created 2005' WHERE ec = 'EC 6.3.1.11';
UPDATE enzyme SET sys_name = 'D-aspartate:(beta-GlcNAc-(1-&gt;4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala))(n) ligase (ADP-forming)' WHERE ec = 'EC 6.3.1.12';
UPDATE enzyme SET history = 'EC 6.3.1.12 created 2006' WHERE ec = 'EC 6.3.1.12';
UPDATE enzyme SET sys_name = 'L-cysteine:1-O-(2-amino-2-deoxy-alpha-D-glucopyranosyl)-1D-myo-inositol ligase (AMP-forming)' WHERE ec = 'EC 6.3.1.13';
UPDATE enzyme SET history = 'EC 6.3.1.13 created 2009' WHERE ec = 'EC 6.3.1.13';
UPDATE enzyme SET sys_name = 'Diphthine-[translation elongation factor 2]:ammonia ligase (AMP-forming)' WHERE ec = 'EC 6.3.1.14';
UPDATE enzyme SET history = 'EC 6.3.1.14 created 1990 as EC 6.3.2.22, transferred 2010 to EC 6.3.1.14, modified 2013' WHERE ec = 'EC 6.3.1.14';
UPDATE enzyme SET sys_name = '3-dimethylallyl-4-hydroxybenzoate:3-amino-4,7-dihydroxycoumarin ligase (AMP-forming)' WHERE ec = 'EC 6.3.1.15';
UPDATE enzyme SET history = 'EC 6.3.1.15 created 2013' WHERE ec = 'EC 6.3.1.15';
UPDATE enzyme SET history = 'EC 6.3.1.16 created 2013, deleted 2013' WHERE ec = 'EC 6.3.1.16';
UPDATE enzyme SET sys_name = 'Citrate:L-glutamate ligase (ADP-forming)' WHERE ec = 'EC 6.3.1.17';
UPDATE enzyme SET history = 'EC 6.3.1.17 created 2014' WHERE ec = 'EC 6.3.1.17';
UPDATE enzyme SET sys_name = '(R)-pantoate:beta-alanine ligase (AMP-forming)' WHERE ec = 'EC 6.3.2.1';
UPDATE enzyme SET history = 'EC 6.3.2.1 created 1961' WHERE ec = 'EC 6.3.2.1';
UPDATE enzyme SET sys_name = 'L-glutamate:L-cysteine gamma-ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.2';
UPDATE enzyme SET history = 'EC 6.3.2.2 created 1961' WHERE ec = 'EC 6.3.2.2';
UPDATE enzyme SET sys_name = 'Gamma-L-glutamyl-L-cysteine:glycine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.3';
UPDATE enzyme SET history = 'EC 6.3.2.3 created 1961' WHERE ec = 'EC 6.3.2.3';
UPDATE enzyme SET sys_name = 'D-alanine:D-alanine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.4';
UPDATE enzyme SET history = 'EC 6.3.2.4 created 1961, modified 2002' WHERE ec = 'EC 6.3.2.4';
UPDATE enzyme SET sys_name = '(R)-4-phosphopantothenate:L-cysteine ligase' WHERE ec = 'EC 6.3.2.5';
UPDATE enzyme SET history = 'EC 6.3.2.5 created 1961, modified 2003' WHERE ec = 'EC 6.3.2.5';
UPDATE enzyme SET sys_name = '5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate:L-aspartate ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.6';
UPDATE enzyme SET history = 'EC 6.3.2.6 created 1961, modified 2000, modified 2006' WHERE ec = 'EC 6.3.2.6';
UPDATE enzyme SET sys_name = 'UDP-N-acetylmuramoyl-L-alanyl-D-glutamate:L-lysine gamma-ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.7';
UPDATE enzyme SET history = 'EC 6.3.2.7 created 1961, modified 2002' WHERE ec = 'EC 6.3.2.7';
UPDATE enzyme SET sys_name = 'UDP-N-acetylmuramate:L-alanine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.8';
UPDATE enzyme SET history = 'EC 6.3.2.8 created 1965, modified 2002' WHERE ec = 'EC 6.3.2.8';
UPDATE enzyme SET sys_name = 'UDP-N-acetylmuramoyl-L-alanine:D-glutamate ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.9';
UPDATE enzyme SET history = 'EC 6.3.2.9 created 1965, modified 2002' WHERE ec = 'EC 6.3.2.9';
UPDATE enzyme SET sys_name = 'UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysine:D-alanyl-D-alanine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.10';
UPDATE enzyme SET history = 'EC 6.3.2.10 created 1965, modified 2002' WHERE ec = 'EC 6.3.2.10';
UPDATE enzyme SET sys_name = 'L-histidine:beta-alanine ligase (AMP-forming)' WHERE ec = 'EC 6.3.2.11';
UPDATE enzyme SET history = 'EC 6.3.2.11 created 1965, modified 2010' WHERE ec = 'EC 6.3.2.11';
UPDATE enzyme SET sys_name = '7,8-dihydropteroate:L-glutamate ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.12';
UPDATE enzyme SET history = 'EC 6.3.2.12 created 1972, modified 2005' WHERE ec = 'EC 6.3.2.12';
UPDATE enzyme SET sys_name = 'UDP-N-acetylmuramoyl-L-alanyl-D-glutamate:meso-2,6-diaminoheptanedioate gamma-ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.13';
UPDATE enzyme SET history = 'EC 6.3.2.13 created 1972, modified 2002, modified 2010' WHERE ec = 'EC 6.3.2.13';
UPDATE enzyme SET sys_name = '2,3-dihydroxybenzoate:L-serine ligase' WHERE ec = 'EC 6.3.2.14';
UPDATE enzyme SET history = 'EC 6.3.2.14 created 1972, modified 2012' WHERE ec = 'EC 6.3.2.14';
UPDATE enzyme SET history = 'EC 6.3.2.15 created 1976, deleted 2003' WHERE ec = 'EC 6.3.2.15';
UPDATE enzyme SET sys_name = 'D-alanine:alanyl-poly(glycerolphosphate) ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.16';
UPDATE enzyme SET history = 'EC 6.3.2.16 created 1976' WHERE ec = 'EC 6.3.2.16';
UPDATE enzyme SET sys_name = 'Tetrahydropteroyl-(gamma-polyglutamate):L-glutamate gamma-ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.17';
UPDATE enzyme SET history = 'EC 6.3.2.17 created 1984, modified 2005' WHERE ec = 'EC 6.3.2.17';
UPDATE enzyme SET sys_name = 'L-glutamate:histamine ligase' WHERE ec = 'EC 6.3.2.18';
UPDATE enzyme SET history = 'EC 6.3.2.18 created 1986' WHERE ec = 'EC 6.3.2.18';
UPDATE enzyme SET sys_name = 'Ubiquitin:protein-lysine N-ligase (AMP-forming)' WHERE ec = 'EC 6.3.2.19';
UPDATE enzyme SET history = 'EC 6.3.2.19 created 1986' WHERE ec = 'EC 6.3.2.19';
UPDATE enzyme SET sys_name = '(Indol-3-yl)acetate:L-lysine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.20';
UPDATE enzyme SET history = 'EC 6.3.2.20 created 1989' WHERE ec = 'EC 6.3.2.20';
UPDATE enzyme SET sys_name = 'Calmodulin:ubiquitin ligase (AMP-forming)' WHERE ec = 'EC 6.3.2.21';
UPDATE enzyme SET history = 'EC 6.3.2.21 created 1990' WHERE ec = 'EC 6.3.2.21';
UPDATE enzyme SET history = 'EC 6.3.2.22 created 1990, deleted 2010' WHERE ec = 'EC 6.3.2.22';
UPDATE enzyme SET sys_name = 'Gamma-L-glutamyl-L-cysteine:beta-alanine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.23';
UPDATE enzyme SET history = 'EC 6.3.2.23 created 1990' WHERE ec = 'EC 6.3.2.23';
UPDATE enzyme SET sys_name = 'L-tyrosine:L-arginine ligase (AMP-forming)' WHERE ec = 'EC 6.3.2.24';
UPDATE enzyme SET history = 'EC 6.3.2.24 created 1992' WHERE ec = 'EC 6.3.2.24';
UPDATE enzyme SET sys_name = 'Alpha-tubulin:L-tyrosine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.25';
UPDATE enzyme SET history = 'EC 6.3.2.25 created 1999' WHERE ec = 'EC 6.3.2.25';
UPDATE enzyme SET sys_name = 'L-2-aminohexanedioate:L-cysteine:L-valine ligase (AMP-forming, valine-inverting)' WHERE ec = 'EC 6.3.2.26';
UPDATE enzyme SET history = 'EC 6.3.2.26 created 2002' WHERE ec = 'EC 6.3.2.26';
UPDATE enzyme SET history = 'EC 6.3.2.27 created 2002, modified 2006, deleted 2012' WHERE ec = 'EC 6.3.2.27';
UPDATE enzyme SET sys_name = 'L-amino acid:L-amino acid ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.28';
UPDATE enzyme SET history = 'EC 6.3.2.28 created 2006' WHERE ec = 'EC 6.3.2.28';
UPDATE enzyme SET sys_name = 'Cyanophycin:L-aspartate ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.29';
UPDATE enzyme SET history = 'EC 6.3.2.29 created 2007' WHERE ec = 'EC 6.3.2.29';
UPDATE enzyme SET sys_name = 'Cyanophycin:L-arginine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.30';
UPDATE enzyme SET history = 'EC 6.3.2.30 created 2007' WHERE ec = 'EC 6.3.2.30';
UPDATE enzyme SET sys_name = 'L-glutamate:coenzyme F420-0 ligase (GDP-forming)' WHERE ec = 'EC 6.3.2.31';
UPDATE enzyme SET history = 'EC 6.3.2.31 created 2010' WHERE ec = 'EC 6.3.2.31';
UPDATE enzyme SET sys_name = 'L-glutamate:coenzyme gamma-F420-2 (ADP-forming)' WHERE ec = 'EC 6.3.2.32';
UPDATE enzyme SET history = 'EC 6.3.2.32 created 2010' WHERE ec = 'EC 6.3.2.32';
UPDATE enzyme SET sys_name = 'Tetrahydromethanopterin:alpha-L-glutamate ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.33';
UPDATE enzyme SET history = 'EC 6.3.2.33 created 2010' WHERE ec = 'EC 6.3.2.33';
UPDATE enzyme SET sys_name = 'L-glutamate:coenzyme F420-1 ligase (GDP-forming)' WHERE ec = 'EC 6.3.2.34';
UPDATE enzyme SET history = 'EC 6.3.4.34 created 2010' WHERE ec = 'EC 6.3.2.34';
UPDATE enzyme SET sys_name = 'D-alanine:D-serine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.35';
UPDATE enzyme SET history = 'EC 6.3.2.35 created 2010' WHERE ec = 'EC 6.3.2.35';
UPDATE enzyme SET sys_name = '(R)-4-phosphopantoate:beta-alanine ligase (AMP-forming)' WHERE ec = 'EC 6.3.2.36';
UPDATE enzyme SET history = 'EC 6.3.2.36 created 2010' WHERE ec = 'EC 6.3.2.36';
UPDATE enzyme SET sys_name = 'UDP-N-acetylmuramoyl-L-alanyl-D-glutamate:L-lysine alpha-ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.37';
UPDATE enzyme SET history = 'EC 6.3.2.37 created 2011' WHERE ec = 'EC 6.3.2.37';
UPDATE enzyme SET sys_name = 'Citrate:N(6)-acetyl-N(6)-hydroxy-L-lysine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.38';
UPDATE enzyme SET history = 'EC 6.3.2.38 created 2012' WHERE ec = 'EC 6.3.2.38';
UPDATE enzyme SET sys_name = 'N(2)-citryl-N(6)-acetyl-N(6)-hydroxy-L-lysine:N(6)-acetyl-N(6)-hydroxy-L-lysine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.39';
UPDATE enzyme SET history = 'EC 6.3.2.39 created 2012' WHERE ec = 'EC 6.3.2.39';
UPDATE enzyme SET sys_name = 'S-adenosyl-L-methionine:anthranilate:L-phenylalanine ligase (cyclopeptine forming)' WHERE ec = 'EC 6.3.2.40';
UPDATE enzyme SET history = 'EC 6.3.2.40 created 2013' WHERE ec = 'EC 6.3.2.40';
UPDATE enzyme SET sys_name = 'N-acetyl-L-aspartate:L-glutamate ligase (ADP, N-acetyl-L-aspartyl-L-glutamate-forming)' WHERE ec = 'EC 6.3.2.41';
UPDATE enzyme SET history = 'EC 6.3.2.41 created 2014' WHERE ec = 'EC 6.3.2.41';
UPDATE enzyme SET sys_name = 'N-acetyl-L-aspartate:L-glutamate ligase (ADP, N-acetyl-L-aspartyl-L-glutamyl-L-glutamate-forming)' WHERE ec = 'EC 6.3.2.42';
UPDATE enzyme SET history = 'EC 6.3.2.42 created 2014' WHERE ec = 'EC 6.3.2.42';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 6.3.2.n1';
UPDATE enzyme SET sys_name = '[Pup]:[protein]-lysine N-ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.n2';
UPDATE enzyme SET history = 'EC 6.3.2.n2 created 2010' WHERE ec = 'EC 6.3.2.n2';
UPDATE enzyme SET sys_name = '[ISG15]:[protein]-lysine N-ligase (AMP-forming)' WHERE ec = 'EC 6.3.2.n3';
UPDATE enzyme SET history = 'EC 6.3.2.n3 created 2010' WHERE ec = 'EC 6.3.2.n3';
UPDATE enzyme SET sys_name = '[LysW] :L-2-aminoadipate gamma-ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.n4';
UPDATE enzyme SET history = 'EC 6.3.2.n4 created 2010' WHERE ec = 'EC 6.3.2.n4';
UPDATE enzyme SET sys_name = '(R)-pantoate:beta-alanine ligase (ADP-forming)' WHERE ec = 'EC 6.3.2.n5';
UPDATE enzyme SET history = 'EC 6.3.2.n5 created 2010' WHERE ec = 'EC 6.3.2.n5';
UPDATE enzyme SET history = 'Preliminary entry into NC-IUBMB' WHERE ec = 'EC 6.3.2.n6';
UPDATE enzyme SET sys_name = '2-(formamido)-N(1)-(5-phosphoribosyl)acetamidine cyclo-ligase (ADP-forming)' WHERE ec = 'EC 6.3.3.1';
UPDATE enzyme SET history = 'EC 6.3.3.1 created 1961, modified 2000' WHERE ec = 'EC 6.3.3.1';
UPDATE enzyme SET sys_name = '5-formyltetrahydrofolate cyclo-ligase (ADP-forming)' WHERE ec = 'EC 6.3.3.2';
UPDATE enzyme SET history = 'EC 6.3.3.2 created 1972' WHERE ec = 'EC 6.3.3.2';
UPDATE enzyme SET sys_name = '7,8-diaminononanoate:carbon-dioxide cyclo-ligase (ADP-forming)' WHERE ec = 'EC 6.3.3.3';
UPDATE enzyme SET history = 'EC 6.3.3.3 created 1976' WHERE ec = 'EC 6.3.3.3';
UPDATE enzyme SET sys_name = 'L-N(2)-(2-carboxyethyl)arginine cyclo-ligase (AMP-forming)' WHERE ec = 'EC 6.3.3.4';
UPDATE enzyme SET history = 'EC 6.3.3.4 created 2003' WHERE ec = 'EC 6.3.3.4';
UPDATE enzyme SET sys_name = 'O-ureido-D-serine cyclo-ligase (D-cycloserine-forming)' WHERE ec = 'EC 6.3.3.5';
UPDATE enzyme SET history = 'EC 6.3.3.5 created 2013' WHERE ec = 'EC 6.3.3.5';
UPDATE enzyme SET sys_name = '(2S,5S)-5-carboxymethylproline cyclo-ligase (AMP-forming)' WHERE ec = 'EC 6.3.3.6';
UPDATE enzyme SET history = 'EC 6.3.3.6 created 2013' WHERE ec = 'EC 6.3.3.6';
UPDATE enzyme SET history = 'EC 6.3.4.1 created 1961, deleted 2013' WHERE ec = 'EC 6.3.4.1';
UPDATE enzyme SET sys_name = 'UTP:L-glutamine amido-ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.2';
UPDATE enzyme SET history = 'EC 6.3.4.2 created 1961, modified 2013' WHERE ec = 'EC 6.3.4.2';
UPDATE enzyme SET sys_name = 'Formate:tetrahydrofolate ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.3';
UPDATE enzyme SET history = 'EC 6.3.4.3 created 1961' WHERE ec = 'EC 6.3.4.3';
UPDATE enzyme SET sys_name = 'IMP:L-aspartate ligase (GDP-forming)' WHERE ec = 'EC 6.3.4.4';
UPDATE enzyme SET history = 'EC 6.3.4.4 created 1961' WHERE ec = 'EC 6.3.4.4';
UPDATE enzyme SET sys_name = 'L-citrulline:L-aspartate ligase (AMP-forming)' WHERE ec = 'EC 6.3.4.5';
UPDATE enzyme SET history = 'EC 6.3.4.5 created 1961' WHERE ec = 'EC 6.3.4.5';
UPDATE enzyme SET sys_name = 'Urea:carbon-dioxide ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.6';
UPDATE enzyme SET history = 'EC 6.3.4.6 created 1972, modified 1986 (EC 3.5.1.45 created 1978, incorporated 1986)' WHERE ec = 'EC 6.3.4.6';
UPDATE enzyme SET sys_name = 'Ribose-5-phosphate:ammonia ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.7';
UPDATE enzyme SET history = 'EC 6.3.4.7 created 1972' WHERE ec = 'EC 6.3.4.7';
UPDATE enzyme SET sys_name = 'Imidazoleacetate:5-phosphoribosyl-diphosphate ligase (ADP- and diphosphate-forming)' WHERE ec = 'EC 6.3.4.8';
UPDATE enzyme SET history = 'EC 6.3.4.8 created 1972' WHERE ec = 'EC 6.3.4.8';
UPDATE enzyme SET sys_name = 'Biotin:apo[methylmalonyl-CoA:pyruvate carboxytransferase] ligase (AMP-forming)' WHERE ec = 'EC 6.3.4.9';
UPDATE enzyme SET history = 'EC 6.3.4.9 created 1972' WHERE ec = 'EC 6.3.4.9';
UPDATE enzyme SET sys_name = 'Biotin:apo-[propanoyl-CoA:carbon-dioxide ligase (ADP-forming)] ligase (AMP-forming)' WHERE ec = 'EC 6.3.4.10';
UPDATE enzyme SET history = 'EC 6.3.4.10 created 1972' WHERE ec = 'EC 6.3.4.10';
UPDATE enzyme SET sys_name = 'Biotin:apo-[3-methylcrotonoyl-CoA:carbon-dioxide ligase (ADP-forming)] ligase (AMP-forming)' WHERE ec = 'EC 6.3.4.11';
UPDATE enzyme SET history = 'EC 6.3.4.11 created 1972' WHERE ec = 'EC 6.3.4.11';
UPDATE enzyme SET sys_name = 'L-glutamate:methylamine ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.12';
UPDATE enzyme SET history = 'EC 6.3.4.12 created 1972' WHERE ec = 'EC 6.3.4.12';
UPDATE enzyme SET sys_name = '5-phospho-D-ribosylamine:glycine ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.13';
UPDATE enzyme SET history = 'EC 6.3.4.13 created 1961 as EC 6.3.1.3, transferred 1972 to EC 6.3.4.13, modified 2000' WHERE ec = 'EC 6.3.4.13';
UPDATE enzyme SET sys_name = 'Biotin-carboxyl-carrier-protein:carbon-dioxide ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.14';
UPDATE enzyme SET history = 'EC 6.3.4.14 created 1976' WHERE ec = 'EC 6.3.4.14';
UPDATE enzyme SET sys_name = 'Biotin:apo-[acetyl-CoA:carbon-dioxide ligase (ADP-forming)] ligase (AMP-forming)' WHERE ec = 'EC 6.3.4.15';
UPDATE enzyme SET history = 'EC 6.3.4.15 created 1978' WHERE ec = 'EC 6.3.4.15';
UPDATE enzyme SET sys_name = 'Carbon-dioxide:ammonia ligase (ADP-forming, carbamate-phosphorylating)' WHERE ec = 'EC 6.3.4.16';
UPDATE enzyme SET history = 'EC 6.3.4.16 created 1965 as EC 2.7.2.5, transferred 1978 to EC 6.3.4.16' WHERE ec = 'EC 6.3.4.16';
UPDATE enzyme SET sys_name = 'Formate:dihydrofolate ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.17';
UPDATE enzyme SET history = 'EC 6.3.4.17 created 1992' WHERE ec = 'EC 6.3.4.17';
UPDATE enzyme SET sys_name = '5-amino-1-(5-phospho-D-ribosyl)imidazole:carbon-dioxide ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.18';
UPDATE enzyme SET history = 'EC 6.3.4.18 created 2006' WHERE ec = 'EC 6.3.4.18';
UPDATE enzyme SET sys_name = 'L-lysine:(tRNA(Ile2))-cytidine(34) ligase (AMP-forming)' WHERE ec = 'EC 6.3.4.19';
UPDATE enzyme SET history = 'EC 6.3.4.19 created 2011' WHERE ec = 'EC 6.3.4.19';
UPDATE enzyme SET sys_name = '7-carboxy-7-carbaguanine:ammonia ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.20';
UPDATE enzyme SET history = 'EC 6.3.4.20 created 2012' WHERE ec = 'EC 6.3.4.20';
UPDATE enzyme SET sys_name = '5-phospho-alpha-D-ribose 1-diphosphate:nicotinate ligase (ADP, diphosphate-forming)' WHERE ec = 'EC 6.3.4.21';
UPDATE enzyme SET history = 'EC 6.3.4.21 created 1961 as EC 2.4.2.11, transferred 2013 to EC 6.3.4.21' WHERE ec = 'EC 6.3.4.21';
UPDATE enzyme SET sys_name = 'Agmatine:(tRNA(Ile))-cytidine(34) ligase' WHERE ec = 'EC 6.3.4.22';
UPDATE enzyme SET history = 'EC 6.3.4.22 created 2013' WHERE ec = 'EC 6.3.4.22';
UPDATE enzyme SET sys_name = 'Formate:5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide ligase (ADP-forming)' WHERE ec = 'EC 6.3.4.23';
UPDATE enzyme SET history = 'EC 6.3.4.23 created 2013' WHERE ec = 'EC 6.3.4.23';
UPDATE enzyme SET sys_name = 'Deamido-NAD(+):L-glutamine amido-ligase (AMP-forming)' WHERE ec = 'EC 6.3.5.1';
UPDATE enzyme SET history = 'EC 6.3.5.1 created 1961' WHERE ec = 'EC 6.3.5.1';
UPDATE enzyme SET sys_name = 'Xanthosine-5-phosphate:L-glutamine amido-ligase (AMP-forming)' WHERE ec = 'EC 6.3.5.2';
UPDATE enzyme SET history = 'EC 6.3.5.2 created 1961, modified 2013' WHERE ec = 'EC 6.3.5.2';
UPDATE enzyme SET sys_name = 'N(2)-formyl-N(1)-(5-phospho-D-ribosyl)glycinamide:L-glutamine amido-ligase (ADP-forming)' WHERE ec = 'EC 6.3.5.3';
UPDATE enzyme SET history = 'EC 6.3.5.3 created 1961, modified 2000' WHERE ec = 'EC 6.3.5.3';
UPDATE enzyme SET sys_name = 'L-aspartate:L-glutamine amido-ligase (AMP-forming)' WHERE ec = 'EC 6.3.5.4';
UPDATE enzyme SET history = 'EC 6.3.5.4 created 1972, modified 2006' WHERE ec = 'EC 6.3.5.4';
UPDATE enzyme SET sys_name = 'Hydrogen-carbonate:L-glutamine amido-ligase (ADP-forming, carbamate-phosphorylating)' WHERE ec = 'EC 6.3.5.5';
UPDATE enzyme SET history = 'EC 6.3.5.5 created 1972 as EC 2.7.2.9, transferred 1978 to EC 6.3.5.5, modified 2006' WHERE ec = 'EC 6.3.5.5';
UPDATE enzyme SET sys_name = 'L-aspartyl-tRNA(Asn):L-glutamine amido-ligase (ADP-forming)' WHERE ec = 'EC 6.3.5.6';
UPDATE enzyme SET history = 'EC 6.3.5.6 created 2002, modified 2012' WHERE ec = 'EC 6.3.5.6';
UPDATE enzyme SET sys_name = 'L-glutamyl-tRNA(Gln):L-glutamine amido-ligase (ADP-forming)' WHERE ec = 'EC 6.3.5.7';
UPDATE enzyme SET history = 'EC 6.3.5.7 created 2002' WHERE ec = 'EC 6.3.5.7';
UPDATE enzyme SET history = 'EC 6.3.5.8 created 2003, deleted 2007' WHERE ec = 'EC 6.3.5.8';
UPDATE enzyme SET sys_name = 'Hydrogenobyrinic-acid:L-glutamine amido-ligase (AMP-forming)' WHERE ec = 'EC 6.3.5.9';
UPDATE enzyme SET history = 'EC 6.3.5.9 created 2004' WHERE ec = 'EC 6.3.5.9';
UPDATE enzyme SET sys_name = 'Adenosylcobyrinic-acid-a,c-diamide:L-glutamine amido-ligase (ADP-forming)' WHERE ec = 'EC 6.3.5.10';
UPDATE enzyme SET history = 'EC 6.3.5.10 created 2004' WHERE ec = 'EC 6.3.5.10';
UPDATE enzyme SET sys_name = 'Cobyrinate:L-glutamine amido-ligase (ADP-forming)' WHERE ec = 'EC 6.3.5.11';
UPDATE enzyme SET history = 'EC 6.3.5.11 created 2010' WHERE ec = 'EC 6.3.5.11';
UPDATE enzyme SET sys_name = 'Pyruvate:carbon-dioxide ligase (ADP-forming)' WHERE ec = 'EC 6.4.1.1';
UPDATE enzyme SET history = 'EC 6.4.1.1 created 1961' WHERE ec = 'EC 6.4.1.1';
UPDATE enzyme SET sys_name = 'Acetyl-CoA:carbon-dioxide ligase (ADP-forming)' WHERE ec = 'EC 6.4.1.2';
UPDATE enzyme SET history = 'EC 6.4.1.2 created 1961' WHERE ec = 'EC 6.4.1.2';
UPDATE enzyme SET sys_name = 'Propanoyl-CoA:carbon-dioxide ligase (ADP-forming)' WHERE ec = 'EC 6.4.1.3';
UPDATE enzyme SET history = 'EC 6.4.1.3 created 1961, modified 1983' WHERE ec = 'EC 6.4.1.3';
UPDATE enzyme SET sys_name = '3-methylcrotonoyl-CoA:carbon-dioxide ligase (ADP-forming)' WHERE ec = 'EC 6.4.1.4';
UPDATE enzyme SET history = 'EC 6.4.1.4 created 1961' WHERE ec = 'EC 6.4.1.4';
UPDATE enzyme SET sys_name = 'Geranoyl-CoA:carbon-dioxide ligase (ADP-forming)' WHERE ec = 'EC 6.4.1.5';
UPDATE enzyme SET history = 'EC 6.4.1.5 created 1972' WHERE ec = 'EC 6.4.1.5';
UPDATE enzyme SET sys_name = 'Acetone:carbon-dioxide ligase (AMP-forming)' WHERE ec = 'EC 6.4.1.6';
UPDATE enzyme SET history = 'EC 6.4.1.6 created 2001' WHERE ec = 'EC 6.4.1.6';
UPDATE enzyme SET sys_name = 'NULL' WHERE ec = 'EC 6.4.1.7';
UPDATE enzyme SET history = 'EC 6.4.1.7 created 2006' WHERE ec = 'EC 6.4.1.7';
UPDATE enzyme SET sys_name = 'Acetophenone:carbon-dioxide ligase (ADP-forming)' WHERE ec = 'EC 6.4.1.8';
UPDATE enzyme SET history = 'EC 6.4.1.8 created 2011' WHERE ec = 'EC 6.4.1.8';
UPDATE enzyme SET sys_name = 'Poly(deoxyribonucleotide):poly(deoxyribonucleotide) ligase (AMP-forming)' WHERE ec = 'EC 6.5.1.1';
UPDATE enzyme SET history = 'EC 6.5.1.1 created 1972, modified 1976' WHERE ec = 'EC 6.5.1.1';
UPDATE enzyme SET sys_name = 'Poly(deoxyribonucleotide):poly(deoxyribonucleotide) ligase (AMP-forming, NMN-forming)' WHERE ec = 'EC 6.5.1.2';
UPDATE enzyme SET history = 'EC 6.5.1.2 created 1972, modified 1976' WHERE ec = 'EC 6.5.1.2';
UPDATE enzyme SET sys_name = 'Poly(ribonucleotide):poly(ribonucleotide) ligase (AMP-forming)' WHERE ec = 'EC 6.5.1.3';
UPDATE enzyme SET history = 'EC 6.5.1.3 created 1976' WHERE ec = 'EC 6.5.1.3';
UPDATE enzyme SET sys_name = 'RNA-3-phosphate:RNA ligase (cyclizing, AMP-forming)' WHERE ec = 'EC 6.5.1.4';
UPDATE enzyme SET history = 'EC 6.5.1.4 created 1986, modified 1989, modified 2013' WHERE ec = 'EC 6.5.1.4';
UPDATE enzyme SET sys_name = 'RNA-3-phosphate:RNA ligase (cyclizing, GMP-forming)' WHERE ec = 'EC 6.5.1.5';
UPDATE enzyme SET history = 'EC 6.5.1.5 created 2013' WHERE ec = 'EC 6.5.1.5';
UPDATE enzyme SET sys_name = 'Mg-protoporphyrin IX magnesium-lyase' WHERE ec = 'EC 6.6.1.1';
UPDATE enzyme SET history = 'EC 6.6.1.1 created 2003' WHERE ec = 'EC 6.6.1.1';
UPDATE enzyme SET sys_name = 'Hydrogenobyrinic-acid-a,c-diamide:cobalt cobalt-ligase (ADP-forming)' WHERE ec = 'EC 6.6.1.2';
UPDATE enzyme SET history = 'EC 6.6.1.2 created 2004' WHERE ec = 'EC 6.6.1.2';
